
<html lang="en"     class="pb-page"  data-request-id="d88511e6-92e4-4ec8-aaa9-8b75d9597f30"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.1c00146;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders" /></meta><meta name="dc.Creator" content="Anja  Dölle" /></meta><meta name="dc.Creator" content="Bikash  Adhikari" /></meta><meta name="dc.Creator" content="Andreas  Krämer" /></meta><meta name="dc.Creator" content="Janik  Weckesser" /></meta><meta name="dc.Creator" content="Nicola  Berner" /></meta><meta name="dc.Creator" content="Lena-Marie  Berger" /></meta><meta name="dc.Creator" content="Mathias  Diebold" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Jakob  Gebel" /></meta><meta name="dc.Creator" content="Frank  Löhr" /></meta><meta name="dc.Creator" content="Volker  Dötsch" /></meta><meta name="dc.Creator" content="Martin  Eilers" /></meta><meta name="dc.Creator" content="Stephanie  Heinzlmeir" /></meta><meta name="dc.Creator" content="Bernhard  Kuster" /></meta><meta name="dc.Creator" content="Christoph  Sotriffer" /></meta><meta name="dc.Creator" content="Elmar  Wolf" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Description" content="Histone H3K4 methylation serves as a post-translational hallmark of actively transcribed genes and is introduced by histone methyltransferase (HMT) and its regulatory scaffolding proteins. One of t..." /></meta><meta name="Description" content="Histone H3K4 methylation serves as a post-translational hallmark of actively transcribed genes and is introduced by histone methyltransferase (HMT) and its regulatory scaffolding proteins. One of t..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 13, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00146" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00146" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00146" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00146" /></link>
        
    
    

<title>Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00146" /></meta><meta property="og:title" content="Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0016.jpeg" /></meta><meta property="og:description" content="Histone H3K4 methylation serves as a post-translational hallmark of actively transcribed genes and is introduced by histone methyltransferase (HMT) and its regulatory scaffolding proteins. One of these is the WD-repeat-containing protein 5 (WDR5) that has also been associated with controlling long noncoding RNAs and transcription factors including MYC. The wide influence of dysfunctional HMT complexes and the typically upregulated MYC levels in diverse tumor types suggested WDR5 as an attractive drug target. Indeed, protein–protein interface inhibitors for two protein interaction interfaces on WDR5 have been developed. While such compounds only inhibit a subset of WDR5 interactions, chemically induced proteasomal degradation of WDR5 might represent an elegant way to target all oncogenic functions. This study presents the design, synthesis, and evaluation of two diverse WDR5 degrader series based on two WIN site binding scaffolds and shows that linker nature and length strongly influence degradation efficacy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00146"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00146">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00146&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00146&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00146&amp;href=/doi/10.1021/acs.jmedchem.1c00146" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00335" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.0c01773" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Anja Dölle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anja Dölle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institut für Pharmazeutische Chemie, Goethe University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anja++D%C3%B6lle">Anja Dölle</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1816-8982" title="Orcid link">https://orcid.org/0000-0002-1816-8982</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bikash Adhikari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bikash Adhikari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Systems Biology Group, Theodor Boveri Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bikash++Adhikari">Bikash Adhikari</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9064-0356" title="Orcid link">https://orcid.org/0000-0002-9064-0356</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Krämer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Krämer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institut für Pharmazeutische Chemie, Goethe University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Kr%C3%A4mer">Andreas Krämer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janik Weckesser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janik Weckesser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institut für Pharmazeutische Chemie, Goethe University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janik++Weckesser">Janik Weckesser</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5100-2776" title="Orcid link">https://orcid.org/0000-0001-5100-2776</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicola Berner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicola Berner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chair of Proteomics and Bioanalytics, Technical University of Munich, 85354 Freising, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicola++Berner">Nicola Berner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lena-Marie Berger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lena-Marie Berger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institut für Pharmazeutische Chemie, Goethe University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lena-Marie++Berger">Lena-Marie Berger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7835-8067" title="Orcid link">https://orcid.org/0000-0002-7835-8067</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mathias Diebold</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mathias Diebold</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut für Pharmazie und Lebensmittelchemie, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mathias++Diebold">Mathias Diebold</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7595-3901" title="Orcid link">https://orcid.org/0000-0002-7595-3901</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Magdalena M. Szewczyk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magdalena M. Szewczyk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dalia Barsyte-Lovejoy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dalia Barsyte-Lovejoy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, Toronto, ON M5G 2M9, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Princess Margaret Cancer Centre, University Health Network, Toronto, ON M5G 2C1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Biophysics, University of Toronto, Toronto, ON M5G 2M9, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-4971-3250" title="Orcid link">https://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jakob Gebel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jakob Gebel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institute of Biophysical Chemistry and Center for Biomolecular Magnetic Resonance and Cluster of Excellence Macromolecular Complexes (CEF), Goethe University Frankfurt am Main, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jakob++Gebel">Jakob Gebel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Frank Löhr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frank Löhr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institute of Biophysical Chemistry and Center for Biomolecular Magnetic Resonance and Cluster of Excellence Macromolecular Complexes (CEF), Goethe University Frankfurt am Main, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Frank++L%C3%B6hr">Frank Löhr</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-6399-9497" title="Orcid link">https://orcid.org/0000-0001-6399-9497</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Volker Dötsch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Volker Dötsch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Department of Medical Biophysics, University of Toronto, Toronto, ON M5G 2M9, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Volker++D%C3%B6tsch">Volker Dötsch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Eilers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Eilers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Theodor Boveri Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Eilers">Martin Eilers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie Heinzlmeir</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie Heinzlmeir</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chair of Proteomics and Bioanalytics, Technical University of Munich, 85354 Freising, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie++Heinzlmeir">Stephanie Heinzlmeir</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1066-6565" title="Orcid link">https://orcid.org/0000-0002-1066-6565</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bernhard Kuster</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bernhard Kuster</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chair of Proteomics and Bioanalytics, Technical University of Munich, 85354 Freising, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</div><div class="loa-info-affiliations-info">Bavarian Biomolecular Mass Spectrometry Center (BayBioMS), Technical University of Munich, 85354 Freising, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bernhard++Kuster">Bernhard Kuster</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9094-1677" title="Orcid link">https://orcid.org/0000-0002-9094-1677</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christoph Sotriffer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christoph Sotriffer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut für Pharmazie und Lebensmittelchemie, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christoph++Sotriffer">Christoph Sotriffer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4713-4068" title="Orcid link">https://orcid.org/0000-0003-4713-4068</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elmar Wolf</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elmar Wolf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Systems Biology Group, Theodor Boveri Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#187d7475796a366f77747e587a7177627d766c6a6d75366d7671356f6d7d6a627a6d6a7f367c7d"><span class="__cf_email__" data-cfemail="0560696864772b726a696345676c6a7f606b717770682b706b6c28727060777f677077622b6160">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elmar++Wolf">Elmar Wolf</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Stefan Knapp</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Knapp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Institut für Pharmazeutische Chemie, Goethe University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#03686d62737343736b62716e606b666e2d766d6a2e6571626d68657671772d6766"><span class="__cf_email__" data-cfemail="3b50555a4b4b7b4b535a495658535e56154e5552165d495a55505d4e494f155f5e">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5995-6494" title="Orcid link">https://orcid.org/0000-0001-5995-6494</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00146&amp;href=/doi/10.1021%2Facs.jmedchem.1c00146" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 13, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 January 2021</li><li><span class="item_label"><b>Published</b> online</span>13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00146" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00146</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D29%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnja%2BD%25C3%25B6lle%252C%2BBikash%2BAdhikari%252C%2BAndreas%2BKr%25C3%25A4mer%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00146%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2BWD-Repeat-Containing%2BProtein%2B5%2B%2528WDR5%2529%2BDegraders%26numPages%3D29%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D29%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00146"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1942</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00146" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Anja&quot;,&quot;last_name&quot;:&quot;Dölle&quot;},{&quot;first_name&quot;:&quot;Bikash&quot;,&quot;last_name&quot;:&quot;Adhikari&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Krämer&quot;},{&quot;first_name&quot;:&quot;Janik&quot;,&quot;last_name&quot;:&quot;Weckesser&quot;},{&quot;first_name&quot;:&quot;Nicola&quot;,&quot;last_name&quot;:&quot;Berner&quot;},{&quot;first_name&quot;:&quot;Lena-Marie&quot;,&quot;last_name&quot;:&quot;Berger&quot;},{&quot;first_name&quot;:&quot;Mathias&quot;,&quot;last_name&quot;:&quot;Diebold&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Jakob&quot;,&quot;last_name&quot;:&quot;Gebel&quot;},{&quot;first_name&quot;:&quot;Frank&quot;,&quot;last_name&quot;:&quot;Löhr&quot;},{&quot;first_name&quot;:&quot;Volker&quot;,&quot;last_name&quot;:&quot;Dötsch&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Eilers&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;Heinzlmeir&quot;},{&quot;first_name&quot;:&quot;Bernhard&quot;,&quot;last_name&quot;:&quot;Kuster&quot;},{&quot;first_name&quot;:&quot;Christoph&quot;,&quot;last_name&quot;:&quot;Sotriffer&quot;},{&quot;first_name&quot;:&quot;Elmar&quot;,&quot;last_name&quot;:&quot;Wolf&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00146&quot;},&quot;abstract&quot;:&quot;Histone H3K4 methylation serves as a post-translational hallmark of actively transcribed genes and is introduced by histone methyltransferase (HMT) and its regulatory scaffolding proteins. One of these is the WD-repeat-containing protein 5 (WDR5) that has also been associated with controlling long noncoding RNAs and transcription factors including MYC. The wide influence of dysfunctional HMT complexes and the typically upregulated MYC levels in diverse tumor types suggested WDR5 as an attractive drug target. Indeed, protein–protein interface inhibitors for two protein interaction interfaces on WDR5 have been developed. While such compounds only inhibit a subset of WDR5 interactions, chemically induced proteasomal degradation of WDR5 might represent an elegant way to target all oncogenic functions. This study presents the design, synthesis, and evaluation of two diverse WDR5 degrader series based on two WIN site binding scaffolds and shows that linker nature and length strongly influence degradation efficacy.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00146&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00146" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00146&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00146" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00146&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00146" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00146&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00146&amp;href=/doi/10.1021/acs.jmedchem.1c00146" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00146" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00146" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00146%26sid%3Dliteratum%253Aachs%26pmid%3D33980013%26genre%3Darticle%26aulast%3DD%25C3%25B6lle%26date%3D2021%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2BWD-Repeat-Containing%2BProtein%2B5%2B%2528WDR5%2529%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=290694" title="Bioluminescent probes">Bioluminescent probes</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Histone H3K4 methylation serves as a post-translational hallmark of actively transcribed genes and is introduced by histone methyltransferase (HMT) and its regulatory scaffolding proteins. One of these is the WD-repeat-containing protein 5 (WDR5) that has also been associated with controlling long noncoding RNAs and transcription factors including MYC. The wide influence of dysfunctional HMT complexes and the typically upregulated MYC levels in diverse tumor types suggested WDR5 as an attractive drug target. Indeed, protein–protein interface inhibitors for two protein interaction interfaces on WDR5 have been developed. While such compounds only inhibit a subset of WDR5 interactions, chemically induced proteasomal degradation of WDR5 might represent an elegant way to target all oncogenic functions. This study presents the design, synthesis, and evaluation of two diverse WDR5 degrader series based on two WIN site binding scaffolds and shows that linker nature and length strongly influence degradation efficacy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Eukaryotic transcription by all three nuclear RNA polymerases is controlled by chromatin organization. The accessibility of chromatin is influenced by the distribution of nucleosomes in the genome, the composition of histone variants within nucleosomes, and post-transcriptional modifications of histone proteins. Histone modifications present a crucial layer of epigenetic transcription control and have become a wide research field as their influence on disease development and progression is outstanding.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Tight control of target gene transcription is specifically important for multicellular organisms as deregulated transcription is often associated with aberrant cellular growth and proliferation and might ultimately induce the development of cancer.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7">(2−7)</a></div><div class="NLM_p">Among all possible histone modifications, mono-(me1), di-(me2), or tri-(me3) methylation of lysine residues in histone tails is considered as the key hallmark of epigenetic regulation. One important example is the methylation of histone H3 lysine 4 residues (H3K4), which defines regulatory elements such as promoters of RNA polymerase II and enhancer elements, and therefore plays a critical role in transcriptional regulation of most protein coding genes.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Cellular H3K4 methylation levels are determined by the balance between H3K4 demethylases and methyltransferases.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> The highly conserved class 2 lysine methyltransferase (KTM2) comprises the mixed-lineage leukemia family (MLL1, MLL2, MLL3, MLL4, MLL5) and SET1A/SET1B enzymes and are responsible for deposition of most of the H3K4 methylation marks associated with transcription.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> With the exception of MLL5, the catalytic activity of KMT2s is dependent on the assembly of further adaptor proteins. The so-called WRAD complex consists of WD-repeat-containing protein 5 (WDR5), DPY30, absent, small, or homeotic-2 like (ASH2L), and retinoblastoma binding protein 5 (RBBP5).<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a></div><div class="NLM_p">WDR5 is of particular importance: its propeller-shaped WD interaction domain interacts with a large diversity of proteins as well as some long noncoding RNAs. Both surfaces of the doughnut-shaped WD domain protein present docking sites, which are called WIN (<u class="uu">W</u>DR5-<u class="uu">in</u>teracting site) and WBM (<u class="uu">W</u>DR5-<u class="uu">b</u>inding <u class="uu">m</u>otif) sites, and both protein interaction sites have been targeted successfully by small molecules.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21">(16−21)</a></div><div class="NLM_p">Interestingly, WDR5 is not only an integral part of the WRAD complex but also directly binds to the MYC oncoprotein family (c-, L-, and N-Myc).<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> MYC proteins are essential transcription factors, and their expression is frequently enhanced and deregulated in human tumors.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Partial genetic inhibition of MYC is well tolerated in adult mice, and it induces tumor regression and long-term survival in several tumor models such as lung adenocarcinoma,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> but no clinical inhibitor of MYC function is available, so far.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> MYC binds to the WBM site of WDR5 with an evolutionary conserved N-terminal region, which is called Myc-box IIIb.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Interestingly, the interaction of MYC with WDR5 supports chromatin association of MYC and is required for the MYC-mediated induction of components of the ribosome and protein production.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Recently, first MYC/WDR5 protein–protein interface inhibitors have been developed.<a onclick="showRef(event, 'ref15 ref29'); return false;" href="javascript:void(0);" class="ref ref15 ref29">(15,29)</a></div><div class="NLM_p">The WDR5 WIN site, located opposite the MYC binding site, is required for WDR5 chromatin recruitment and interaction with KMT2 enzymes, and MLL1 is particularly dependent on this interaction. The small-molecule antagonist OICR-9429 for instance binds to the WIN site of WDR5 and efficiently disrupts its interaction with MLL1.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Its potential as an anticancer agent has been demonstrated in leukemia and some solid tumors.<a onclick="showRef(event, 'ref17 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref17 ref30 ref31">(17,30,31)</a> Interestingly, targeting the WIN site with small molecules evicts WDR5 as well as its interaction partners from the chromatin, resulting in changes in MLL1-dependent histone methylation.<a onclick="showRef(event, 'ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref30">(27,30)</a> Besides WDR5s’ well-established scaffolding function for KMT2 enzymes, long noncoding RNAs are also able to bind to WDR5, adding a further key role in MLL regulation of gene transcription and tumorigenesis.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a></div><div class="NLM_p last">Both main binding sites of WDR5 are important for key oncogenic functions. We hypothesized therefore that a WDR5 degrader molecule may be an efficient therapeutic agent for cancer treatment and designed a series of PROTACs (<u class="uu">PRO</u>teolysis <u class="uu">TA</u>rgeting <u class="uu">C</u>himeras) based on two known WIN ligands at two diverse attachment points for linkers and E3 binding moieties. Herein, we present the development of WDR5 degraders that are based on the scaffold of the antagonist OICR-9429 as well as a modified pyrroloimidazole scaffold published by Wang and co-workers.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> The most optimal degraders of both series result in rapid, selective, and robust degradation of WDR5, providing chemical tools for further studies on this interesting cancer target.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Synthesis Design of WDR5 Degraders</h3><div class="NLM_p">In this study, two established WDR5 antagonists were used as basic anchoring scaffolds for the PROTAC design: the (trifluoromethyl)-pyridine-2-one OICR-9429 as well as the pyrroloimidazole-based inhibitor that was published by Wang and co-workers.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> An overlay of the available crystal structures of the inhibitor complexes revealed (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) that the two ligands enter the WDR5-binding pocket at different angles. The attachment points to the E3 ligase binding moiety were chosen at carbonyl groups that were solvent-accessible and allowed the exit of linker moieties at different positions of the WIN pocket. We hypothesized that the diversity of linker attachment points might increase the possibility of efficient ternary complex formation and thus successful degradation of WDR5 by the developed degraders.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Synthesis design of WDR5 degraders based on available crystal structures. (a) Crystal structures of WDR5 in complex with the small-molecule antagonist OICR-9429 (pink; the chemical structure of OICR-9429 is displayed above) and (b) a small molecule published by Wang and co-workers (blue; chemical structure of the chemical probe is displayed above). The red-colored spheres indicate the solvent-exposed site and represent the attachment point for linkers. pdb entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DAK">6DAK</a>. (c) (Upper panel) Schematic illustration of a heterobifunctional degrader molecule (also called PROTAC) consisting of a ligand for the target protein (WDR5, herein colored in pink/blue) and a ligand binding an E3 ligase (orange). (Lower panel) Chemical structures of two developed degrader series targeting WDR5. The molecules contain either the OICR-9429 derivate (pink) or the pyrroloimidazole derivate (blue) and are connected via different linkers (red) to different E3 ligase ligands (orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of OICR-9429-Based Degraders <b>7a–e</b>, <b>8a–j</b>, and <b>9a–c</b></h3><div class="NLM_p">In the first series, the WDR5 antagonist OICR-9429 scaffold was used.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Educts <b>1</b> and <b>2</b> formed intermediate <b>3</b> in a nucleophilic aromatic substitution. The reduction reaction of the aromatic nitro group led to intermediate <b>4</b>, which was coupled via a Suzuki–Miyaura cross-coupling to obtain biaryl <b>5</b>. To maintain affinity for WDR5, an amide coupling of the primary amine <b>5</b> with the nicotinic acid was carried out to form amide <b>6</b>. The <i>in situ</i> deprotection of the carboxylic acid on the attached biaryl system pointed toward the solvent and served as an elongation point of several E3 ligase linkers <b>L1</b>–<b>L15</b> (synthesis described in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">the Supporting Information</a>). The E3 ligase linker molecules were synthesized either by attaching the Boc-protected linker to the VHL ligand in an amide formation reaction or by attaching the Boc-protected linker to pomalidomide in a nucleophilic aromatic substitution. Deprotection of intermediate <b>6</b> and amide coupling reactions to the corresponding E3 ligase linker resulted in degrader molecules <b>7a–e</b> and <b>8a–j</b>. The synthetic steps carried out for the synthesis of the OICR-9429-based degrader molecules are shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Route of WDR5 Degraders <b>7a–e</b> Addressing the E3 Ligase Cereblon (CRBN)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) <i>N</i>,<i>N-</i>Diisopropylethylamine (DIEA), EtOH, 80 °C, 16 h; (b) Zn, NH<sub>4</sub>Cl, dioxane/water (3/1), rt, 30 min; (c) (4-(<i>tert</i>-butoxycarbonyl)phenyl)boronic acid, XPhos PdG3, NaHCO<sub>3</sub>, dioxane/water (3/1), 85 °C, 16 h; (d) 1. carboxylic acid, SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/ACN (1/1), 50 °C, 3 h; 2. pyridine, CH<sub>2</sub>Cl<sub>2</sub>/ACN (1/1), 50 °C, 16 h; (e) 1. Trifluoroacetic acid (TFA)/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, <i>N</i>,<i>N</i>-dimethylformamide (DMF), CRBN ligase ligand linker <b>L1–L5</b>, rt, 3–5 h. The type and nature of the linker are indicated in red.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Chemical Structures of WDR5 Degraders <b>8a–j</b> Addressing the E3 Ligase Von–Hippel–Lindau (VHL)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The synthesis steps starting from intermediate <b>6</b> are shown: (a) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, linker <b>L6</b>–<b>L14</b>, rt, 3–5 h. The type and nature of the linker are indicated in red.</p></p></figure><div class="NLM_p">For the synthesis of the E3 ligase MDM2 targeting degraders, the linker was directly attached to the modified OICR-9429 scaffold <b>6</b> to form intermediates <b>6a–c</b>, and this molecule was subsequently coupled using an amide formation reaction to the MDM2 inhibitor idasanutlin, yielding degrader molecules <b>9a–c</b>, as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis Route of WDR5 Degraders <b>9a–c</b> Addressing the E3 Ligase MDM2<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The synthesis steps starting from intermediate <b>6</b> are shown: (a) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, linker, rt, 3–5 h; (b) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, idasanutlin, rt, 12 h. The type and nature of the linker are indicated in red.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Synthesis of Pyrroloimidazole-Based Degraders <b>17a–g</b></h3><div class="NLM_p">The modified scaffold of the second degrader series was derived from the antagonist published by Wang and co-workers.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The synthetic scheme for the pyrroloimidazole-based molecules is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Educt <b>10</b> underwent an intramolecular cyclization to form aryl bromide <b>11</b>. Intermediate <b>11</b> reacted by a Suzuki–Miyaura cross-coupling with (3-cyano-4-methoxyphenyl)boronic acid to form the biaryl system <b>12</b>. Reduction of the nitrile group led to amine <b>13</b>, which was then coupled with 2-(3,4-dichlorophenyl)acetic acid to yield amide <b>14</b>. Ether deprotection of intermediate <b>14</b> led to the free phenol <b>15</b>. Phenol <b>15</b> was reacted by a nucleophilic substitution reaction with the tosylated linker derivatives to yield intermediates <b>16a–g</b>. Deprotection of <b>16a–g</b> followed by amide-bond formation with the VHL E3 ligase ligand<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> resulted in the degrader molecules <b>17a–g</b>.</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of WDR5 degraders <b>17a–g</b> addressing the E3 ligase VHL<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) POBr<sub>3</sub>, MeCN, MW, 70 °C, 2 h; (b) (3-cyano-4-methoxyphenyl)boronic acid, XPhos PdG3, NaOH, THF/H<sub>2</sub>O, 80 °C, 21 h; (c) LiAlH<sub>4</sub>, THF, 60 °C–rt, 21 h; (d) 2-(3,4-dichlorophenyl)acetic acid, EDC, HOBt, DIEA, DMF, rt, 16 h; (e) 1. BBr<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C to rt, 21 h; 2. NaOH/H<sub>2</sub>O; (f) linker-OTs, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C, 16–22.5 h; (g) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1.5 h; 2. HATU, DIEA, DMF, VHL hydrochloride linker, rt, 3–18 h. The type and nature of the linker are indicated in red.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Evaluation of Degrader Binding to WDR5</h3><div class="NLM_p">To gain a first indication of the binding affinity of the degraders to WDR5, differential scanning fluorimetry (DSF)/temperature-shift measurements were carried out with recombinantly expressed WDR5, using the WDR5 antagonist OICR-9429 as a reference compound. The obtained temperature shifts Δ<i>T</i><sub>m</sub> are listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Unexpectedly, the inhibitor modification <b>6</b> increased the thermal stability significantly when compared to the reference OICR-9429, suggesting improved potency. Regarding the degrader molecules, temperature-shift data correlated with the nature of the introduced linker: mostly aliphatic linker <b>8e–i</b> showed weaker thermal stabilization compared to more polar linkers in <b>8a</b> and <b>8c</b>. However, this might be due to the limiting solubility of compounds containing an aliphatic linker. The heterobifunctional molecules <b>7a</b>, <b>8a</b>, and <b>9a</b> contained the shortest [PEG]-linker ([PEG]<sub>1</sub>), and they showed higher melting temperature shifts than degraders containing longer [PEG]-linkers, regardless of the E3 ligase ligand used. Besides the heterobifunctional molecules, the inhibitors idasanutlin, VHL ligand 1, and a modified thalidomide were included in the experiments to investigate their binding affinity toward WDR5 (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>). All E3 ligase ligands alone used did not result in significant temperature shifts of WDR5. The MDM2 ligand-containing degraders <b>9a–c</b> lost their affinity toward WDR5 possibly due to their size and associated less favorable physiochemical properties or steric constraints. Performing DSF measurements of MDM2 targeting degrader intermediates <b>6a–c</b>, a decrease in temperature shifts was observed.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>In Vitro</i> and <i>In Cellulo</i> Data of the WDR5 Antagonist OICR-9429, the Modified Inhibitor <b>6</b>, and Degraders <b>7a–e</b>, <b>8a–j</b>, and <b>9a–c</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0012.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Thermal shift Δ<i>T</i><sub>m</sub> values given are the mean of triplicate measurements. DSF assays were performed at 2 μM protein concentration and a final compound concentration of 10 μM.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><i>K</i><sub>d</sub> values were derived from ITC measurements (carried out as duplicate, except for <b>8i</b>, which was measured in a single measurement) and calculated by assuming a sigmoidal dose–response relationship (four parameters). The errors of the fits were calculated using standard deviation and a confidence interval of 68%.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">IC<sub>50</sub> values were derived from bioluminescence resonance energy transfer (BRET) duplicate measurements and calculated by assuming a normalized 3-parameter curve fit.</p></div><div class="footnote" id="t1fn4"><p class="first last">n.d., not determined.</p></div></div><div></div></div><div class="NLM_p">To determine the binding affinity for WDR5 in solution, isothermal titration calorimetry (ITC) measurements for selected degrader molecules <b>7a</b>, <b>8a</b>, and <b>8e–j</b> were performed (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; for binding curves, see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>). We chose CRBN-based degrader <b>7a</b> and VHL-based degrader <b>8a</b> for ITC measurements due to their comparably small molecular weight, their identical linker structure, and the high Δ<i>T</i><sub>m</sub> value. Their comparison showed that the binding affinity did not correlate well with Δ<i>T</i><sub>m</sub> shifts; <b>7a</b>, which showed less thermal stabilizing, had a three times higher affinity for WDR5 than <b>8a</b>, which showed a higher Δ<i>T</i><sub>m</sub> value. Also, DSF measurements revealed that <b>8e–i</b> showed a surprising diversity of thermal stabilization (ranging from 3.5 to 13 K shift), and these ligands were therefore chosen for further characterization by ITC. Interestingly, only a small difference in binding affinity was observed for degraders <b>8e–i</b>, which decrease with increasing linker length (ethyl- to hexyl-linker). Similar to <b>7a</b> and <b>8a</b>, we concluded that high thermal stabilization did not always correlate well with the binding affinity for these large ligands. For instance, <b>8g</b> and <b>8j</b> displayed the highest Δ<i>T</i><sub>m</sub> shifts, but in ITC measurements, they showed the lowest binding affinities in this series. Furthermore, a decrease in solubility was observed for degraders with a long aliphatic linker (especially <b>8h</b> and <b>8i</b>), which made data acquisition using ITC challenging. The limited solubility of some degraders also made comparison of thermodynamic parameters problematic, and particularly thermodynamic data measured on degraders harboring aliphatic and aromatic linkers such as <b>8h</b>, <b>8i</b>, and <b>8j</b> need to be treated with caution. Despite all of these technical challenges, we concluded that all heterobifunctional degrader molecules showed excellent affinity to WDR5 in the one- or two-digit nanomolar <i>K</i><sub>D</sub> range.</div><div class="NLM_p">For all PROTACs that we selected for ITC studies, large negative binding enthalpies were observed ranging from −10 to −4.9 kcal/mol (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Table S3</a>). Entropy changes (<i>T</i>Δ<i>S</i>) were also favorable, except for <b>8a</b> (−0.2 kcal/mol), ranging from +0.7 to +4.5 kcal/mol, but enthalpy–entropy compensation was observed for degraders with large favorable Δ<i>H</i> values. For instance, compounds <b>8a</b>, <b>8f</b>, and <b>8i</b> all showed large negative binding enthalpies of −10 kcal/mol with close-to-zero entropy changes (−0.2 to 1.1 kcal/mol), whereas compounds with large positive entropy changes showed small binding enthalpies (e.g., <b>8e</b>: Δ<i>H</i> = −6.3 kcal/mol, <i>T</i>Δ<i>S</i> = +4.5 kcal/mol). These compensation mechanisms could be due to water displacement as the polar [PEG]<sub>1</sub> linker in <b>8a</b> showed an unfavorable binding entropy change, while degraders with aliphatic linkers feature an increased binding entropy term.</div><div class="NLM_p">Overall, the ITC experiments highlighted the value of determining binding affinity in an orthogonal assay system as similar Δ<i>T</i><sub>m</sub> shifts resulted in some cases in very different binding affinities and degraders with high Δ<i>T</i><sub>m</sub> shift were not necessarily more affine than a degrader with a moderate Δ<i>T</i><sub>m</sub> shift.</div><div class="NLM_p">In analogy to degrader molecules based on the modified OICR-9429 scaffold, the highest Δ<i>T</i><sub>m</sub> shift of the pyrroloimidazole-based degraders <b>17a–g</b> was observed with the shortest linker moieties <b>17a</b> (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The subsequent decrease in temperature-shift values correlated also for this scaffold with the linker length. The Δ<i>T</i><sub>m</sub> shift did increase slightly for degraders with longer linker moieties, namely, <b>17e–g</b>; thus, solubility, solvatization in the large Win pocket, and interaction with the hydrophobic surface of WDR5 might be possible explanations for the observed effect. The affinity of <b>17b</b> for WDR5 was examined by ITC, showing that <b>17b</b> and the parent compound <b>14</b> had comparable dissociation constants (<i>K</i><sub>d</sub>s) of 97 and 125 nM, respectively. In accordance with the OICR-9429-derived degraders, the binding event was found to be enthalpically favored and also a gain in entropy was observed in the binding measurements. Once more, we assume that this observable effect might be related to the characteristic properties of the WDR5 system that have already been discussed in the context of the OICR-9429-derived degraders.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>In Vitro</i> and <i>In Cellulo</i> Data of the WDR5 Antagonist OICR-9429, the Modified Molecule <b>14</b>, and Degraders <b>17a–g</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0013.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Thermal shift Δ<i>T</i><sub>m</sub> values given are the mean of triplicate measurements. DSF assays were performed at 2 μM protein concentration and a final compound concentration of 10 μM.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last"><i>K</i><sub>d</sub> values were derived from ITC measurements as duplicate measurements and calculated by assuming a sigmoidal dose–response relationship (four parameters). The errors of the fits were calculated using standard deviation and a confidence interval of 68%.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">IC<sub>50</sub> values were derived from BRET duplicate measurements and calculated by assuming a normalized 3-parameter curve fit.</p></div><div class="footnote" id="t2fn4"><p class="first last">n.d., not determined.</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cellular Permeability and Target Engagement of Degrader Molecules</h3><div class="NLM_p">Due to the rather large molecular weight inherent of heterobifunctional molecules, cellular permeability can be a limiting factor. In this study, bioluminescence resonance energy transfer (BRET) experiments were used to determine cell permeability and potency in the cellular context and for comparison also in lysed cells. To establish this assay, three tracer molecules <b>19a–c</b> based on molecule <b>6</b> were synthesized using BODIPY fluorescent conjugates. The synthetic procedures of the tracer molecules <b>19a–c</b> via their intermediates <b>6a</b>, <b>S18a</b>, and <b>S18b</b> are summarized in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>. The obtained tracer molecules were titrated into cells transfected with N-terminally and C-terminally tagged WDR5 NanoLuc-fusion constructs to determine the assay performance (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). These experiments revealed <b>19c</b> (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) and the C-terminally tagged WDR5 construct as the most suitable Tracer-NanoLuc combination for cellular BRET assays.</div><figure id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cellular permeability and target engagement studies were performed with the BRET assay. (a) Tracer titration of all three synthesized tracer molecules and either C-terminal or N-terminal Nanoluc-tagged WDR5 (WDR5-C or WDR5-N). (b) Chemical structure of the most suitable tracer molecule <b>19c</b>. (c) and (d) NanoBRET dose–response curves of the degrader <b>8g</b>, ligand <b>6</b>, and WDR5 antagonist OICR-9429 in live cells (c) as well as in lysed cells (d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">BRET assays were performed in lysed (permeabilized) and living cells (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c,d), and both assay formats were used to assess cell penetration as well as cellular affinity for full-length WDR5 (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The parent compound of the degrader series <b>7a–e</b>, <b>8a–j</b>, and <b>9a–c</b> showed a cellular potency of 139 nM. We observed a significant drop in affinity when parent compound <b>6</b> was elongated to the final degrader molecules. Comparing the attached E3 ligase ligands as well as the nature of the linker, central changes could be observed. Most cereblon (CRBN) targeting degraders <b>7a–e</b> had similar cellular potencies. Furthermore, both formats of the assay system (intact and lysed modes) indicated a low micromolar affinity for WDR5. Interestingly, VHL targeting degraders showed a weaker cellular activity potentially due to the peptidelike nature of this ligand. While we were able to determine affinity values in lysed cells for all VHL degraders, no matter the linker nature, the only aliphatic linker containing degrader <b>8g</b> showed a two-digit micromolar affinity in the intact cellular experiment. Comparing <b>8e–i</b> to the PEG containing degraders <b>8a–c</b>, the directly linked degrader <b>8d</b>, and the aromatic linked molecule <b>8j</b>, weak solubility might be the limiting factor for this observation (as already shown in the ITC experiments of <b>8e–i</b>). As MDM2 targeting degraders <b>9a–c</b> already showed weak <i>in vitro</i> affinity, the BRET measurements confirmed the weak <i>in cellulo</i> activity of these molecules to WDR5. Taken together with the DSF results, the MDM2 targeting degraders were excluded from further experiments.</div><div class="NLM_p last">The generated tracer <b>19c</b> could not be displaced by the pyrroloimidazole-based degraders <b>17a–g</b>. A possible hypothesis for this observation might be the different binding modes of the used inhibitor. All in all, the herein-established BRET assay for WDR5 indicates that several degrader molecules are cell-permeable and bind to WDR5 in cells and lysates, demonstrating their potential as putative degraders <i>in vivo</i>.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> PROTAC-Mediated Degradation of Cellular WDR5</h3><div class="NLM_p">Targeted degradation of cellular proteins requires productive complex formation of the PROTAC-bound protein with the respective E3 ligase and proteasomal degradation of the ubiquitylated target protein. We therefore analyzed PROTAC-mediated degradation of WDR5 in cells. The open reading frame of WDR5 was fused with a luciferase peptide (called HiBiT) and stably transduced into the AML cell line MV4-11 (MV4-11<sup>WDR5-HiBiT</sup>). Immunoblots demonstrated that expression levels of WDR5-HiBiT are comparable to endogenously expressed WDR5 (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). We treated MV4-11<sup>WDR5-HiBiT</sup> cells with various concentrations of the two different degrader series for 24 h and estimated depletion of WDR5-HiBiT by measuring the luciferase activity.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cellular degradation studies on WDR5. (a) Immunoblot of WDR5. WDR5 was fused to the fragment of luciferase (HiBiT) and stably expressed in MV4-11 cells. Naive (-) MV4-11 cells and HiBiT-WDR5 (+) MV4-11 cells. Vinculin was used as a loading control. (b) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>8g</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (c) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of degraders <b>8a</b>, <b>8e</b>, <b>8f</b>, <b>8h</b>, and <b>8i</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (d) Quantification of WDR5 <i>D</i><sub>max</sub> (maximal degradation) from the HiBiT assay for degraders with different aliphatic linkers. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of degraders for 6 h or 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity of linkers comprising ethyl: <b>8e</b>, propyl: <b>8f</b>, butyl: <b>8g</b>, pentyl: <b>8h</b>, and hexyl: <b>8i</b>. (e) and (f) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>14</b>, <b>21</b>, and <b>17b</b> (e) and <b>6</b>, <b>20</b>, and <b>8g</b> (f) for 24 h. Cells were then lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Intriguingly, depletion of WDR5-HiBiT varied substantially between the different degrader classes (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>; additional data can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S3</a>). While none of the cereblon-based PROTACs showed significant depletion, many VHL ligand-containing degraders demonstrated cellular degrader efficacy. The most effective degrader of the OICR-9429-derived series <b>8g</b>, which linked both functional binding moieties by a butyl chain, showed a maximum depletion of 58 ± 3% and DC<sub>50</sub> values of 53 ± 10 nM (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). A shortening of the linker by implementing ethyl (<b>8e</b>) and propyl (<b>8f</b>) chains as well as an elongation with pentyl (<b>8h</b>) and hexyl (<b>8i</b>) chains significantly reduced the degradation efficiency (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c,d). Strikingly, degrader <b>8a</b>, a degrader resembling <b>8h</b>, but containing a [PEG]<sub>1</sub> moiety instead of an aliphatic chain, did not induce degradation of WDR5 (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). The most efficient pyrroloimidazole-based degrader was <b>17b</b>, which contained a [PEG]<sub>2</sub> linker (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). Both <b>8g</b> and <b>17b</b> did not induce maximal depletion of WDR5 at high concentrations, a phenomenon called the Hook effect, resulting from less efficient ternary complex formation at excess degrader levels due to binding site competition (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b,e).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Two negative controls—<b>20</b>, resembling molecule <b>8g</b>, and <b>21</b>, resembling molecule <b>17b</b>—with the inactive variant of the VHL ligand were synthesized to verify the effect of targeted protein degradation. Their biophysical properties can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>. Notably, neither negative controls <b>20</b> and <b>21</b> nor the ligands <b>6</b> and <b>14</b> alone induced degradation of WDR5 in cells (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e,f). The increase in WDR5 levels by <b>6</b> is most likely a direct effect on WDR5 stability induced by ligand binding.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. HiBiT Data of WDR5 Ligand <b>6</b> and Degraders <b>7a–e</b> and <b>8a–j</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0014.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">DC<sub>50</sub>, half-maximal degradation concentration, calculated with the dose–response (four parameters) equation.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">DC<sub>max</sub>, maximal degradation concentration.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last"><i>D</i><sub>max</sub>, maximal degradation.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">no, no degradation.</p></div><div class="footnote" id="t3fn5"><p class="first last">N/A, not applicable.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. HiBiT Data of WDR5 Ligand <b>14</b> and Degraders <b>17a–g</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0015.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">DC<sub>50</sub>, half-maximal degradation concentration, calculated with the dose–response (four parameters) equation.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">DC<sub>max</sub>, maximal degradation concentration.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last"><i>D</i><sub>max</sub>, maximal degradation.</p></div><div class="footnote" id="t4fn4"><p class="first last">no, no degradation.</p></div></div><div></div></div><div class="NLM_p">Next, we analyzed if the degrader molecules <b>8g</b> and <b>17b</b> also induced degradation of untagged and endogenously expressed WDR5. MV4-11<sup>WDR5-HiBiT</sup> cells were treated for 24 h with <b>8g</b> or <b>17b</b> and were analyzed by immunoblots stained with an anti-WDR5 antibody. Both degraders induced efficient depletion of endogenous WDR5, and the dose-dependency and gratifyingly degradation efficacy of the endogenous protein resembled the depletion of the HiBiT-tagged protein (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a><b>a</b> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">S4</a>). Degraders <b>8g</b> and <b>17b</b> showed a similar level of WDR5 depletion in naive MV4-11 cells even 72 h post treatment (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a><b>b</b> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">S5a</a>). Immunoblotting also confirmed the depletion efficiency of various other degraders of WDR5 in naive MV4-11 cells as seen in HiBiT data (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S5b–f</a>). Degraders induce degradation of their targets by inducing ubiquitylation and subsequently proteasomal degradation. We therefore tested if <b>8g</b> and <b>17b</b> decreased the protein stability of WDR5. We blocked protein translation by incubating cells with cycloheximide in addition to the degraders (or vehicle-treated cells) and estimated WDR5 levels by immunoblotting at several time points. Both degraders reduced the stability of the WDR5 protein substantially in comparison to the vehicle-treated cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a><b>c</b> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">S5g</a>).</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Degrader-induced depletion of WDR5 depends on the ubiquitin system. (a) Immunoblot of WDR5. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>8g</b> for 24 h and compared with dimethyl sulfoxide (DMSO)-treated or naive MV4-11 cells. Vinculin was used as a loading control (as in all further immunoblot experiments). Quantification is based on both protein bands (endogenous/HiBiT-tagged WDR5). (b) Immunoblot of WDR5. Naive MV4-11 cells were treated with different concentrations of <b>8g</b> for 72 h and compared with DMSO-treated cells. (c) Immunoblot and quantification of WDR5 levels. WDR5 protein stability was evaluated by treating 1 μM <b>8g</b> or DMSO-incubated MV4-11 cells for 0, 1, 2, 6, and 12 h with cycloheximide (CHX). The data is mean ± s.d from <i>n</i> = 2 biological replicates. (d) Quantitative reverse transcriptase polymerase chain reaction (RT-PCR) analysis of WDR5 mRNA levels. RNA was extracted from MV4-11 cells incubated with 1 μM <b>8g</b>, 1 μM <b>6</b>, 3 μM <b>17b</b>, and 3 μM <b>14</b> for 24 h. WDR5 expression levels were normalized with a reference gene (<i>B</i>2<i>M</i>). Bars represent mean ± s.d. of <i>n</i> = 3 technical replicates. (e) Immunoblot of WDR5. MV4-11 cells were treated for 6 h with 1 μM <b>8g</b>, 5 μM <b>6</b>, 10 μM VHL ligand, or a combination of them. (f) Immunoblot of WDR5. MV4-11 cells were treated for 6 h with 3 μM <b>17b</b>, 5 μM MLN4924, 10 μM MG132, or a combination of them. (g) Immunoblot of WDR5. HL-60 cells were treated with different concentrations of <b>8g</b> for 24 h. (h) and (i) Volcano plot exhibiting global proteomics change. MV4-11 cells were treated with 1 μM <b>8g</b> (h) or 5 μM <b>17b</b> (i) for 9 h, and lysates were analyzed by quantitative proteomics. WDR5 (blue) and other SET1/MLL complex core subunits: KMT2A, KTM2B, KTM2C, KTM2D, SETD1A, RBBP5, ASH2L, and DPY30 (orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To exclude effects of our compounds on WDR5 transcription, we treated MV4-11 cells with the most effective degraders <b>8g</b> and <b>17b</b> and their corresponding WDR5 ligands <b>6</b> and <b>14</b> and quantified mRNA by quantitative PCR (qPCR). As expected, even though both degraders reduced the WDR5 protein level, WDR5 mRNA levels were not decreased (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">S5h</a>). Similarly, <b>8g</b>-mediated WDR5 degradation was completely abolished by coincubation of MV4-11 cells with WDR5 ligand <b>6</b>, showing that WDR5 depletion requires binding of <b>8g</b> to WDR5 (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e). We also rescued WDR5 depletion by <b>17b</b> through proteasomal inhibition with MG132 and neddylation inhibition with MLN4924 (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>f). Finally, we also tested if the degradation of WDR5 was limited to MV4-11 cells or if the degrader compounds were also functional in other cell lines. We thus treated the human leukemia cell line HL-60 with <b>8g</b> and observed WDR5 depletion as seen in MV4-11 cells (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>g). We concluded that <b>8g</b> and <b>17b</b> mediate depletion of endogenous WDR5 in various cancer cell lines by inducing protein ubiquitylation and degradation.</div><div class="NLM_p">To determine whether <b>8g-</b> and <b>17b-</b>mediated degradation was specific to WDR5, we analyzed cellular protein levels by quantitative proteomics. To this end, we treated MV4-11 cells with <b>8g</b> and <b>17b</b> or with their corresponding ligands and compared their protein content to untreated cells by mass spectrometry. Remarkably, among 5805 proteins detected, only WDR5 was significantly and substantially depleted (−log<sub>10</sub> <i>p</i> > 3, log<sub>2</sub> FC < −0.5) after 9 h of treatment (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>h–i). WDR5 levels were not significantly altered by treatment with the ligands <b>6</b> and <b>14</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S6</a>).</div><div class="NLM_p">To determine the cellular consequences of WDR5 depletion, we analyzed MV4-11 cell proliferation. We treated MV4-11 cells with different concentrations of <b>17b</b>, its negative control (<b>21</b>), and its ligand (<b>14</b>) for 15 days. Only a higher concentration (∼10 μM) of <b>17b</b> induced a proliferation defect over the course of time, whereas neither a lower concentration of <b>17b</b> nor any incubation with <b>21</b> and <b>14</b> showed significant growth defect (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). We hypothesized that only high concentrations mediate a long-standing and sufficient depletion of WDR5, which is required to affect cell growth. In fact, all degraders studied here induce only partial degradation of WDR5 and we speculate that more efficient depletion is required to see stronger attenuation of cancer cell growth. As effective degradation via degraders requires a stoichiometric relation among the target protein, the degrader, and the E3 ligase, we wondered whether the expression of VHL might be the limiting factor for degrader efficacy. To test this hypothesis, we stably expressed exogenous VHL in MV4-11<sup>WDR5-HiBiT</sup> cells (MV4-11<sup>WDR5-HiBiT/VHL</sup>) (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S5i</a>). Strikingly, the HiBiT assay demonstrated the superior degradation of the WDR5 protein by both <b>8g</b> and <b>17b</b> in MV4-11<sup>WDR5-HiBiT/VHL</sup> cells when compared to control cells (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">S3</a>). For <b>17b</b>, the <i>D</i><sub>max</sub> increased from 52.5 to 77.8%, whereas DC<sub>50</sub> decreased from 1.01 to 0.155 μM. Immunoblotting confirmed the enhanced degradation of endogenous WDR5 after <b>17b</b> treatment in MV4-11<sup>WDR5-HiBiT/VHL</sup> cells (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S5j</a>). To test if enhanced degradation of WDR5 in VHL-overexpressing cells induced a more pronounced cell-cycle phenotype, we also generated naive MV4-11 cells with ectopic VHL expression (MV4-11<sup>VHL</sup>) (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Figure S5i</a>). We verified the degradation of endogenous WDR5 in these cells in comparison to control cells (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c). Finally, we repeated the cumulative growth analysis in MV4-11<sup>VHL</sup> cells by incubating the cells with different concentrations of <b>17b</b>, <b>21</b>, and <b>14</b> for 15 days. In line with the stronger WDR5 depletion, a high concentration (10 μM) of <b>17b</b> showed a stronger growth inhibition than that in the control cells (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a,d). Importantly, MV4-11<sup>VHL</sup> cells also showed growth defects even at lower concentrations of <b>17b</b> (5 μM), whereas none of the controls (<b>21</b> and <b>14)</b> showed notable effects (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d). We concluded that degrader-induced partial depletion of WDR5 shows moderate but statistically significant cell growth defects in MV4-11 cells.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. VHL overexpression increases PROTACs/degrader efficiency and cellular responses. (a) Cumulative growth curve in MV4-11 cells. MV4-11 cells were treated with 10, 5, and 3 μM of <b>17b</b>, <b>21</b>, and <b>14</b> and counted at indicated time points. To prevent the overgrowth, cells were reseeded to the original density every third day in fresh media with compounds and treatment was continued for 15 days. Data represent mean ± s.d. of <i>n</i> = 2 biological replicates. Asterisks indicate <i>P</i>-value calculated from the 15<sup>th</sup> day cumulative cell number (two-tailed unpaired <i>t</i>-test assuming equal variance against DMSO treatment). * <i>P</i> ≤ 0.05. (b) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells (Ctr) and MV4-11<sup>WDR5-HiBiT/VHL</sup> cells (VHL OE) were treated with different concentrations of <b>17b</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (c) Immunoblot of WDR5 and VHL. MV4-11 cells (Ctr) and MV4-11<sup>VHL</sup> (VHL OE) were treated for 24 h with various concentrations of <b>17b</b>. †, overexpressed VHL; ††, endogenous VHL. (d) Cumulative growth curve in MV4-11<sup>VHL</sup> cells. MV4-11<sup>VHL</sup> cells were treated with 10, 5, and 3 μM of <b>17b</b>, <b>21</b>, and <b>14</b> and counted at indicated time points. To prevent the overgrowth, cells were reseeded to the original density every third day in fresh media with compounds and treatment was continued for 15 days. Data represent mean ± s.d. of <i>n</i> = 2 biological replicates. Asterisks indicate <i>P</i>-values calculated from the 15<sup>th</sup> day cumulative cell number (two-tailed unpaired <i>t</i>-test assuming equal variance against DMSO treatment). * <i>P</i> ≤ 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Computational Studies on WDR5 Degraders</h3><div class="NLM_p">Computational docking studies were performed to obtain a structural model that was able to explain the binding of the active degraders <b>8e–j</b> to WDR5 and VHL. Therefore, crystal structures of WDR5 in complex with the WDR5 antagonist OICR-9429 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and VHL in complex with the ligand VH032 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H">4W9H</a>)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> were prepared for protein/protein docking, as described in the Methods section and the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>. In the second step, protein/protein docking was performed in the Molecular Operating Environment (MOE) to obtain 405 possible protein/protein complexes. For evaluation of the generated protein complexes, the distance between the two linkage sites served as the primary selection criterion. Since the shortest linker used in <b>8e</b> showed effective degradation, docking solutions capable of binding both OICR-9429 and VH032 in their known binding modes while maintaining a linkable distance for the short ethyl moiety were examined. Ten of the obtained protein/protein complexes showed a distance of less than 4 Å between the two critical carbon atoms, with five of these being ranked in the top 20% of the protein/protein docking solutions. None of the complexes stood out as clearly preferred, suggesting rather an ensemble of possible configurations.</div><div class="NLM_p">In the next step, degraders <b>8e–j</b> were docked with GOLD to the five best-ranked of the compatible protein/protein complexes (ranks 52, 53, 59, 78, and 79) and evaluated based on a rescoring with DrugscoreX and RMSD values with respect to the crystalized binding modes of the VHL ligand VH032 and the WDR5 antagonist OICR-9429. The docking results show that all active degraders can be placed in these complexes while still occupying their native binding sites in the individual proteins with RMSD values around or below 1 Å. Only degrader <b>8j</b> required rearrangements of the scaffolds in most top-ranked docking solutions (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>). The structural models obtained by a successive protein/protein- and small-molecule-docking suggest that WDR5 and VHL do not form a dominant ternary complex. Rather, it appears likely that an ensemble of multiple different configurations of the ternary complex may exist in solution, similar (but probably even more diverse) to that as illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Computational docking studies on WDR5 and VHL. (a) Structures of the selected protein/protein docking solutions compatible with a short linker distance. VHL (blue) and VH032 (orange) form a complex with WDR5 (gray) and a modified OICR-9429 ligand (pink), with the attachment points for linkers in close proximity (<4 Å). (b–f) DSX top-ranked docking pose of degrader <b>8g</b> docked to each of the selected protein/protein complexes. The functional parts of molecule <b>8g</b> are colored in orange for the interaction with VHL, in pink for the interaction with WDR5, and in red for the linker that connects both moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions and Outlook</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87241" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87241" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The MLL/SET HMT complexes as well as the transcription factor family MYC are attractive drug targets. In the strategy presented here, we aimed to modulate both oncogenic functions by degrading the scaffolding protein WDR5 using a PROTAC approach. We used two diverse inhibitor scaffolds allowing for diverse exit vectors from the WDR5 WIN binding site comprising OICR-9429 and a modified pyrroloimidazole-based inhibitor in combination with E3 ligase ligands targeting cereblon, VHL, and MDM2. Thus, the study provides interesting SAR developing degraders for this attractive cancer target. In BRET studies, VHL and cereblon-based degraders showed good cell permeability and on-target activity in cells. Surprisingly, degraders based on both inhibitor scaffolds led to successful degradation of the target protein, indicating good degradability of WDR5. However, only VHL-based degraders led to functional degradation, and small changes in linker length and linker type resulted in significant changes in degradation efficacy. Large MDM2-based PROTACs did not show significant binding affinity and were discarded early on in the study. The best degrader molecules showed low nM potency degrading WDR5 in a proteasome-and ubiquitin-dependent way. However, under the cell lines and condition tested, only partial degradation was observed. This could be a kinetic effect depending on the speed of resynthesis and degradation or potentially due to the location of WDR5 on the chromatin, which might protect a fraction of the protein from degradation. Nonetheless, the cells could be sensitized for better degradation of the protein via overexpression of one of the components of the ternary complex (here, VHL), which significantly increased the degradation efficiency of the PROTACs. These data suggest that using the PROTAC in cell lines that show higher VHL levels will also increase WDR5 degradation determining sensitivity to the degrader. We established an array of assay systems such as a BRET-based target engagement assay and a stable HiBiT cell line, which will allow further testing of future second-generation WDR5 degrader molecules with improved potency. However, the molecules presented here are versatile tools that will allow comprehensive evaluation of WDR5 degraders in diverse cancer types and the potential of this strategy for drug discovery.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Compound Synthesis</h3><div class="NLM_p last">The structures of the synthesized compounds were verified by <sup>1</sup>H NMR, <sup>13</sup>C NMR, and mass spectrometry (ESI/MALDI). Purity of the final compounds (254, 260, and 280 nm >95%) was determined by analytical high-performance liquid chromatography (HPLC). All commercial chemicals and solvents were used without further purification. The commercially available VHL ligand 1 hydrochloride was used for the degrader molecules <b>8a–j</b> and <b>17a–g</b>, while the VHL ligand for the negative controls <b>20</b> and <b>21</b> was obtained as described by Buckley and van Molle.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were measured in DMSO-<i>d</i><sub>6</sub>, MeOD, CD<sub>2</sub>Cl<sub>2</sub>, or CDCl<sub>3</sub> on a Bruker DPX250, AV300, AV400, AV500, DPX600, AV700, or AV800 spectrometer. Chemical shifts δ are reported in parts per million (ppm). Mass spectra were recorded by the mass spectrometry service team of the Goethe University using a Thermo Fisher Surveyor MSQ system (including TLC-MS for reaction control). High-resolution mass spectra were recorded on a MALDI LTQ ORBITRAP XL device from Thermo Fisher Scientific. Product purification was performed on a PuriFlash Flash Column Chromatography System from Interchim using prepacked silica or RP C18 columns. Product purification was also performed on an Agilent 1260 Infinity II LC System [Eclipse XDB-C18 column (7 μM, 21.2 mm × 250 mm)] using a gradient of water/MeCN + 0.1% TFA (98:2–5:95) over 40 min with a flow rate of 21 mL/min. Compound purity was analyzed on an Agilent 1260 Infinity II LC System [Eclipse XDB-C18 column (4.6 mm × 250 mm, 5 μm)] coupled to an Agilent InifinityLab LC/MSD using a gradient of water/MeCN + 0.1% TFA (98:2–5:95) over 25 min at a flow rate of 1 mL/min (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">SI</a> data for HPLC–MS traces of lead compounds). Final compounds <b>7a–e</b>, <b>8a–j</b>, and <b>9a–c</b> were synthesized from 5-bromo-2-fluoronitrobenzole <b>1</b> and 4-methylpiperazine <b>2</b> via intermediates <b>3–6</b> and <b>6a–c</b> as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Final compounds <b>17a–g</b> were synthesized from Piracetam <b>10</b> via intermediates <b>11–16a–g</b> as outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. More information can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Procedure A (Amide Coupling)</h3><div class="NLM_p last">First, 1.0 equiv of the Boc-protected carboxylic acid was dissolved in 1 mL of CH<sub>2</sub>Cl<sub>2</sub>, and 1 mL of TFA was added. The solution was stirred for 1 h at rt. Excess solvent was removed under reduced pressure. Then, 1.0 equiv of the Boc-protected amine was dissolved in 1 mL of CH<sub>2</sub>Cl<sub>2</sub> and 1 mL of TFA was added. The solution was stirred for 1 h at rt. Excess solvent was removed under reduced pressure. The crude species was dissolved in 0.5 mL of DMF, and DIEA was added until the pH of the solution was basic. Next, 1.2 equiv of HATU was added to the deprotected carboxylic acid. The reaction mixture was stirred for 20 min at rt. The crude species of the deprotected amine rt for 3–5 h. The reaction was stopped with 1 mL of water. Saturated NaHCO<sub>3</sub> solution and saturated NaCl solution were added, and the reaction mixture was extracted 4× with EA. The combined organic phases were washed with saturated NaHCO<sub>3</sub> solution, dried over MgSO<sub>4</sub>, and filtered. The solvent of the organic phase was evaporated under reduced pressure. The purification of the crude product was carried out on a preparative HPLC system.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Procedure B (Tosylation and Nucleophilic Substitution)</h3><div class="NLM_p">First, 1.4 equiv of tosylchloride was added portionwise (2×) to 1.0 equiv of commercially available alcohol, 0.3 equiv of 4-dimethylaminopyridine (DMAP), and 1.3 equiv of triethylamine in 5 mL of dichloromethane at −10 °C. The reaction mixture was stirred for 20 min, allowed to warm to room temperature, and stirred for another 22 h. The reaction was quenched by adding 3 mL of a saturated solution of NH<sub>4</sub>Cl in water. The organic layer was separated, and the remaining aqueous layer was extracted with dichloromethane (3×). The combined organic layers were washed with brine and dried with Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification was achieved by column chromatography (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>).</div><div class="NLM_p last">Then, 2.4 equiv of the tosylate linker species in 1.5 mL of DMF was added to a solution of the alcohol <b>15</b> and 3.2 equiv of K<sub>2</sub>CO<sub>3</sub> in 2 mL of DMF. The reaction mixture was stirred at 70 °C for 21 h. The reaction was quenched by adding 6 mL of water and ethyl acetate. The organic layer was separated, and the remaining aqueous layer was extracted with ethyl acetate (3×). The combined organic layers were dried with MgSO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification was carried out on a preparative HPLC system.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure C (Amide Coupling)</h3><div class="NLM_p">This reaction was not performed in an inert atmosphere. A solution of 1.0 equiv of ester in 4 mL of dichloromethane/TFA (1/1) was stirred for 1.5 h. All volatiles were removed under reduced pressure. Then, 1.2 equiv of HATU was added to a solution of the carboxylic acid and 10.0 equiv of DIPEA in 2 mL of DMF. The reaction mixture was stirred for 15 min, it turned orange, and 1.1 equiv of VHL amine hydrochloride was added. The mixture was stirred for another 3.5 h and quenched with water and ethyl acetate. The organic layer was separated, and the aqueous layer was extracted with ethyl acetate (3×). The combined organic layers were dried with MgSO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification was carried out on a preparative HPLC system. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthesis of 1-(4-Bromo-2-nitrophenyl)-4-methylpiperazine (<b>3</b>)</h4><div class="NLM_p last">First, 5 mL (39.7 mmol, 1.00 equiv) of 5-bromo-2-fluoro-nitrobenzene was dissolved in 20 mL of EtOH. Thn, 4 mL (39.7 mmol, 1.00 equiv) of <i>N</i>-methylpiperazine and 13.5 mL (79.4 mmol, 2.00 equiv) of DIEA were added and the reaction mixture was stirred for 5 h at 80 °C. The reaction was cooled to rt, and the solvent was removed under reduced pressure. The reaction mixture was diluted with water and extracted 8× with dichloromethane (DCM). The combined organic phases were washed with 1 M HCl and saturated NaCl solution, dried over MgSO<sub>4</sub>, and filtered. The crude product was purified via CC (gradient: 0–10% MeOH in DCM) to give 11.1 g (37 mmol, 95%) of an orange powder. <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>): 0.54. ESI: (calculated): [M + H<sup>+</sup>] 300.03 g/mol, (found): [M + H<sup>+</sup>] 299.98 g/mol. <sup>1</sup>H NMR (250 MHz, CDCl<sub>3</sub>) δ = 7.94 (d, <sup>4</sup><i>J</i> = 2.3 Hz, 1H), 7.65 (dd, <sup>3</sup><i>J</i> = 8.7 Hz, <sup>4</sup><i>J</i> = 2.4 Hz, 1H), 7.17 (d, <sup>3</sup><i>J</i> = 8.7 Hz, 1H), 3.43 (m, 4H), 3.25 (m, 4H), 2.80 (s, 3H) ppm.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Synthesis of 5-Bromo-2-(4-methylpiperazin-1-yl)aniline (<b>4</b>)</h4><div class="NLM_p last">First, 2.00 g (7.40 mmol, 1.00 equiv) of 1-(4-bromo-2-nitrophenyl)-4-methylpiperazine was suspended in a mixture of 1,4-dioxane and water (3:1). Then, 3.20 g (37 mmol, 7.50 equiv) of ammonium chloride was added, followed by a slow addition of 2.60 g (37 mmol, 7.50 equiv) of zinc dust. The reaction mixture was stirred until a color change from orange to light pink was observed. The solvent was removed under reduced pressure. The reaction mixture was diluted with saturated NaHCO<sub>3</sub> solution and extracted 6× with DCM. The combined organic phases were washed with saturated NaCl solution, dried over MgSO<sub>4</sub>, and filtered, and the solvent was removed under reduced pressure to give 1.57 mg (5.81 mmol, 79%) of a light-pink solid. <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>): 0.3. ESI: (calculated): [M + H<sup>+</sup>] 270.06 g/mol, (found): [M + H<sup>+</sup>] 270.03 g/mol. HPLC: RT = 10.9 min (254 nm, 99%). <sup>1</sup>H NMR (250 MHz, CDCl<sub>3</sub>) δ = 6.96–6.70 (m, 3H), 4.01 (s, 2H), 2.92 (t, <sup>3</sup><i>J</i> = 4.5 Hz, 4H), 2.60 (s, 4H), 2.39 (s, 3H) ppm.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Synthesis of <i>tert</i>-Butyl 3′-Amino-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate (<b>5</b>)</h4><div class="NLM_p last">First, 395 mg (1.78 mmol, 1.2 equiv) of (4-(<i>tert</i>-butoxycarbonyl)phenyl)boronic acid was dissolved in an argon-purged solvent solution (1,4-dioxane/water (3:1)) and 288 mg (7.40 mmol, 5.0 equiv) of sodium hydroxide was added. The reaction mixture was stirred for 5 min at rt under an argon atmosphere; then, 400 mg (1.48 mmol, 1.0 equiv) of 5-bromo-2-(4-methylpiperazin-1-yl)aniline and 171 mg (148 μmol, 0.1 equiv) of Pd(PPh<sub>3</sub>)<sub>4</sub> were added. The reaction was stirred at 90 °C for 18 h under an argon atmosphere. The reaction mixture was cooled to rt, filtered over celite, and washed with MeOH. The solvent was removed under reduced pressure. The reaction mixture was diluted with water and extracted 3× with DCM. The crude product was purified via FC (0–10% MeOH/DCM) to give 395 mg (1.08 mmol, 73%) of a white solid. <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>): 0.28. ESI: (calculated): [M + H<sup>+</sup>] 368.23 g/mol, (found): [M + H<sup>+</sup>] 368.13 g/mol. HPLC: RT = 11.9 min (254 nm, 94%). <sup>1</sup>H NMR (500 MHz, CD<sub>2</sub>Cl<sub>2</sub>) δ = 7.98 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.59 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.06 (d, <sup>3</sup><i>J</i> = 8.7 Hz, 1H), 6.99 (m, 2H), 3.97 (s, 1H), 2.97 (s, 4H), 2.61 (s, 4H), 2.35 (s, 3H), 1.58 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, CD<sub>2</sub>Cl<sub>2</sub>) δ = 165.9, 145.5, 142.3, 139.8, 136.5, 130.8, 130.1, 126.8, 120.4, 117.6, 113.8, 81.1, 55.0, 51.0, 46.1, 28.3 ppm.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Synthesis of <i>tert</i>-Butyl 3′-(6-Hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate (<b>6</b>)</h4><div class="NLM_p last">First, 56 mg (272 μmol, 1.00 equiv) of 6-hydroxy-4-(trifluoromethyl)nicotinic acid was dissolved in 1 mL of DCM and 228 μL (2.72 mmol, 10 equiv) of thionyl chloride. The reaction mixture was stirred for 3 h at 50 °C until a color change from clear to yellow was observed. Excess thionyl chloride was removed under reduced pressure, and the acyl chloride was evaporated on a high vacuum line for 5 min. The acyl chloride was diluted with 2 mL of DCM, and a 3 mL solution containing 100 mg (272 μmol, 1.00 equiv) of 3′-amino-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate and 44 μL (544 μmol, 2.00 equiv) of pyridine was added. The reaction mixture was stirred at 50 °C for 18 h. The reaction mixture was cooled to rt, diluted with water, and extracted 3× with DCM. The crude product was purified via FC (0–10% MeOH/DCM) to give 109 mg (139 μmol, 51%) of a yellow TFA salt with a stoichiometry of 1:2 (product: TFA). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>): 0.21. ESI: (calculated): [M + H<sup>+</sup>] 557.23 g/mol, (found): [M + H<sup>+</sup>] 557.08 g/mol. HPLC: RT = 12.9 min (254 nm, 93%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 8.97 (s, 1H), 8.69 (s, 1H), 8.04 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.89 (s, 1H), 7.66 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.48–7.30 (m, 2H), 6.95 (s, 1H), 3.05 (s, 4H), 2.78 (s, 4H), 2.51 (s, 3H), 1.61 (s, 9H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO) δ = 164.8, 163.1, 161.1, 143.7, 143.2, 139.5 (m), 138.6 (q, <sup>2</sup><i>J</i> = 32 Hz), 135.0, 134.1, 132.7, 130.1, 129.8, 126.4, 124.0, 122.1 (q, <sup>1</sup><i>J</i> = 273 Hz), 121.0, 118.9 (m), 111.6 (m), 80.7, 52.8, 48.1, 42.3, 27.8 ppm.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Synthesis of <i>tert</i>-Butyl (2-(2-(3′-(6-Hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxamido)ethoxy)ethyl)carbamate (<b>6a</b>)</h4><div class="NLM_p last">First, 20 mg (36 μmol, 1.0 equiv) of <i>tert</i>-butyl 3′-(6-hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate was dissolved in 0.5 mL of DCM and 0.5 mL of TFA and stirred at rt for 1 h. The excess solvent was evaporated. The solid was dissolved in 0.5 mL of DMF; then, 125 μL (720 μmol, 20 equiv) of DIEA and 16.4 mg (43 μmol, 1.2 equiv) of HATU were added. After 15 min, a solution of 7.7 mg (38 μmol, 1.05 equiv) of <i>tert</i>-butyl (2-(2-aminoethoxy)ethyl)carbamate in 0.5 mL of DMF was added. The solution was stirred for 3 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified using HPLC to obtain 18 mg of a TFA salt with an unknown stoichiometry as a white solid. ESI: (calculated) [M + H<sup>+</sup>] 687.31 g/mol, (found) [M + H<sup>+</sup>] 687.52 g/mol. HPLC: RT = 11.6 min (254 nm, 96%). <sup>1</sup>H NMR (400 MHz, DMSO) δ = 9.47 (s, 1H), 8.52 (t, <sup>3</sup><i>J</i> = 5.4 Hz, 1H), 8.13 (d, <sup>3</sup><i>J</i> = 2.1 Hz), 7.97 (s, 1H), 7.93 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.68 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.52 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>4</sup><i>J</i> = 2.2 Hz), 7.27 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.82 (s, 1H), 6.76 (s, 1H), 3.55–3.52 (m, 2H), 3.47–3.38 (m, 4H), 3.11–3.06 (m, 2H), 2.99–2.85 (m, 4H), 2.50 (s, 4H) 2.23 (s, 3H), 1.36 (s, 9H) ppm. <sup>13</sup>C NMR (101 MHz, DMSO) δ = 168.0, 166.0, 162.8, 161.2, 144.9, 142.2, 140.0, 139.5 (q, <sup>2</sup>J = 35 Hz), 134.2, 133.00 132.3, 132.2, 128.3, 127.9, 126.0, 123.9, 122.6 (q, <sup>1</sup>J = 240 Hz), 122.3, 121.0, 120.4, 119.1 (m), 110.3 (m), 77.59, 68.99, 68.7, 54.7, 51.0, 45.7, 39.1, 38.9, 28.2 ppm.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Synthesis of <i>tert</i>-Butyl (1-(3′-(6-Hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-yl)-1-oxo-5,8,11,14,17,20,23-heptaoxa-2-azapentacosan-25-yl)carbamate (<b>6b</b>)</h4><div class="NLM_p last">First, 20 mg (36 μmol, 1.0 equiv) of <i>tert</i>-butyl 3′-(6-hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate was dissolved in 0.5 mL of DCM and 0.5 mL of TFA and stirred at rt for 1 h. The excess solvent was evaporated. The solid was dissolved in 0.5 mL of DMF; then, 125 μL (720 μmol, 20 equiv) of DIEA and 16.4 mg (43 μmol, 1.2 equiv) of HATU were added. After 15 min, a solution of 17.7 mg (38 μmol, 1.05 equiv) of <i>tert</i>-butyl (23-amino-3,6,9,12,15,18,21-heptaoxatricosyl)carbamate in 0.5 mL of DMF was added. The solution was stirred for 3 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified using HPLC to obtain 10 mg of a TFA salt with an unknown stoichiometry as a white oil. ESI: (calculated) [M + H<sup>+</sup>] 951.47 g/mol, (found) [M + H<sup>+</sup>] 951.95 g/mol. HPLC: RT = 11.8 min (254 nm, 94%). <sup>1</sup>H NMR (400 MHz, DMSO) δ = 9.46 (s, 1H), 8.53 (t, <sup>3</sup><i>J</i> = 5.7 Hz, 1H), 8.12 (d, <sup>4</sup><i>J</i> = 2.2 Hz, 1H), 7.97 (s, 1H), 7.94 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.68 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.53 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>4</sup><i>J</i> = 2.2 Hz, 1H), 7.27 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.82 (s, 1H), 6.72 (t, <sup>3</sup><i>J</i> = 5.3 Hz, 1H), 3.61–3.47 (m, 28H), 3.44 (dd, <sup>2</sup><i>J</i> = 11.6 Hz, <sup>3</sup><i>J</i> = 5.8 Hz, 2H), 3.36 (t, <sup>3</sup><i>J</i> = 6.1 Hz, 4H), 3.05 (q, <sup>3</sup><i>J</i> = 6.0 Hz, 2H), 2.96–2.86 (m, 4H), 2.24 (s, 3H), 1.36 (s, 9H) ppm.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Synthesis of <i>tert</i>-Butyl (4-((3′-(6-Hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxamido)methyl)benzyl)carbamate (<b>6c</b>)</h4><div class="NLM_p last">First, 20 mg (36 μmol, 1.0 equiv) of <i>tert</i>-butyl 3′-(6-hydroxy-4-(trifluoromethyl)nicotinamido)-4′-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-4-carboxylate was dissolved in 0.5 mL of DCM and 0.5 mL of TFA and stirred at rt for 1 h. The excess solvent was evaporated. The solid was dissolved in 0.5 mL of DMF; then, 125 μL (720 μmol, 20 equiv) of DIEA and 16.4 mg (43 μmol, 1.2 equiv) of HATU were added. After 15 min, a solution of 8.90 mg (38 μmol, 1.05 equiv) of <i>tert</i>-butyl (4-(aminomethyl)benzyl)carbamate in 0.5 mL of DMF was added. The solution was stirred for 3 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified using HPLC to obtain 14 mg of a TFA salt with an unknown stoichiometry as a white solid. ESI: (calculated) [M + H<sup>+</sup>] 719.32 g/mol, (found) [M + H<sup>+</sup>] 719.54 g/mol. HPLC: RT = 12.2 min (254 nm, 91%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.29 (d, <sup>4</sup><i>J</i> = 1.8 Hz, 1H), 7.99 (s, 1H), 7.92 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.72 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.52 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 7.35–7.33 (m, 3H), 7.25 (d, <sup>3</sup><i>J</i> = 8.1 Hz, 2H), 6.92 (s, 1H), 4.57 (s, 2H), 4.21 (s, 2H), 3.01 (s, 4H), 2.69 (s, 4H), 2.39 (s, 3H), 1.44 (s, 9H).</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Synthesis of <i>N</i>-(4′-((2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>7a</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 17.2 mg (20.4 μmol, 71%) of a yellow solid. MALDI: (calculated) [M + H<sup>+</sup>] 843.31 g/mol, (found) [M + H<sup>+</sup>] 843.41 g/mol. HPLC: RT = 11.6 min (254 nm, 100%). HRMS: (calculated) [M + H<sup>+</sup>] 843.3072 g/mol, (found) [M + H<sup>+</sup>] 843.3067 g/mol, <sup>1</sup>H NMR (500 MHz, DMSO) δ = 12.54 (s, 1H), 11.08 (s, 1H), 9.52 (s, 1H), 8.53 (t, <sup>3</sup><i>J</i> = 5.6 Hz, 1H), 8.14 (d, <sup>4</sup><i>J</i> = 1.8 Hz, 1H), 8.00 (s, 1H), 7.91 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.66 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.60–7.48 (m, 2H), 7.28 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.15 (d, <sup>3</sup><i>J</i> = 8.6 Hz, 1H), 7.02 (d, <sup>3</sup><i>J</i> = 7.0 Hz, 1H), 6.83 (s, 1H), 6.63 (t, <sup>3</sup><i>J</i> = 5.7 Hz, 1H), 5.03 (dd, <sup>3</sup><i>J</i> = 12.8 Hz, <sup>4</sup>J = 5.4 Hz, 1H), 3.66 (t, <sup>3</sup><i>J</i> = 5.4 Hz, 2H), 3.61 (t, <sup>3</sup><i>J</i> = 6.0 Hz, 2H), 3.48 (dd, <sup>3</sup><i>J</i> = 13.3 Hz, <sup>4</sup><i>J</i> = 4.9 Hz, 4H), 2.99 (s, 4H), 2.91–2.80 (m, 1H), 2.75 (s, 4H), 2.58–2.57 (m, 1H), 2.52 (s, 1H), 2.42 (s, 3H), 2.01–1.98 (m, 1H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.7, 170.1, 168.9, 167.3, 166.0, 162.9, 161.1, 146.4, 144.3, 142.1, 139.3, 138.5 (q, <sup>2</sup><i>J</i> = 33 Hz), 136.2, 134.5, 133.0, 132.3, 132.1, 127.9, 126.0, 123.9, 122.3, 122.0 (q, <sup>1</sup><i>J</i> = 275 Hz), 120.6, 119.0, 117.5, 111.7, 110.7, 109.2, 68.8, 68.8, 54.1, 50.1, 48.5, 44.8, 41.7, 31.0, 25.5, 22.1 ppm.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Synthesis of <i>N</i>-(4′-((2-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethoxy)ethyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>7b</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 8.58 mg (8.95 μmol, 33%) of a light-yellow solid. MALDI: (calculated) [M + H<sup>+</sup>] 959.39 g/mol, (found) [M + H<sup>+</sup>] 959.47 g/mol. HPLC: RT = 11.8 min (254 nm, 98%). HRMS: (calculated) [M + H<sup>+</sup>] 959.3909 g/mol, (found) [M + H<sup>+</sup>] 959.3905 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 11.08 (s, 1H), 9.45 (s, 1H), 8.44 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.12 (d, <sup>4</sup><i>J</i> = 1.7 Hz, 1H), 7.98 (s, 1H), 7.91 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.67 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.61–7.54 (m, 1H), 7.51 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup>J = 2.2 Hz, 1H), 7.26 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.08 (d, <sup>3</sup><i>J</i> = 8.6 Hz, 1H), 7.01 (d, <sup>3</sup><i>J</i> = 7.0 Hz, 1H), 6.81 (s, 1H), 6.65 (t, <sup>3</sup><i>J</i> = 5.9 Hz, 1H), 5.04 (dd, <sup>3</sup><i>J</i> = 12.7 Hz, <sup>4</sup>J = 5.4 Hz, 1H), 3.61–3.35 (m, 21H), 2.95–2.89 (m, 3H), 2.91–2.84 (m, 1H), 2.62–2.55 (m, 1H), 2.23 (s, 3H), 2.07–1.95 (m, 1H), 1.85–1.72 (m, 5H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.8, 170.1, 168.8, 167.3, 165.8, 162.8, 162.3, 146.4, 144.8, 142.1, 139.0, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 136.2, 134.2, 133.2, 132.2, 132.2, 127.8, 126.0, 123.9, 122.2, 122.0 (q, <sup>1</sup><i>J</i> = 275 Hz), 120.4, 118.8, 117.1, 111.7, 110.3, 109.0, 69.8, 69.7, 69.7, 69.6, 68.3, 68.2, 54.7, 51.0, 48.5, 45.8, 40.4, 36.7, 35.8, 31.0, 29.4, 28.9, 22.1 ppm.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Synthesis of <i>N</i>-(4′-((17-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12,15-pentaoxaheptadecyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>7c</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 18 mg (14.4 μmol, 41%) of a yellow oil as a TFA salt (1:2/product: TFA.) MALDI: (calculated) [M + H<sup>+</sup>] 1019.41 g/mol, (found) [M + H<sup>+</sup>] 1019.52 g/mol. HPLC: RT = 11.6 min (254 nm, 100%). HRMS: (calculated) [M + H<sup>+</sup>] 1019.4121 g/mol, (found) [M + H<sup>+</sup>] 1019.4117 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 11.10 (s, 1H), 9.20 (s, 1H), 8.59 (d, <sup>3</sup><i>J</i> = 5.2 Hz, 1H), 8.32 (d, <sup>4</sup><i>J</i> = 1.7 Hz, 1H), 8.14 (s, 1H), 7.94 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 6H), 7.68 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 6H), 7.57 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>3</sup><i>J</i> = 7.3 Hz, 1H), 7.46 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.0 Hz, 1H), 7.29 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.13 (t, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.06–6.94 (m, 1H), 6.58 (dd, <sup>2</sup><i>J</i> = 12.2 Hz, <sup>3</sup><i>J</i> = 6.4 Hz, 1H), 6.46 (s, 1H), 5.05 (dd, <sup>3</sup><i>J</i> = 12.7 Hz, <sup>4</sup><i>J</i> = 5.4 Hz, 1H), 3.67–3.32 (m, 28H), 2.89 (t, <sup>3</sup><i>J</i> = 4.4 Hz, 4H), 2.60–2.56 (m, 1H), 2.48 (s, 1H), 2.28–2.25 (m, 1H), 2.22 (s, 3H), 2.08–1.95 (m, 1H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.8, 170.1, 168.9, 167.3, 165.9, 163.1, 161.1, 146.4, 142.9, 142.0, 139.5, 138.6 (q, <sup>2</sup><i>J</i> = 32 Hz), 136.2, 135.3, 133.2, 132.6, 132.1, 128.0, 126.1, 123.9, 122.3, 122.1 (q, <sup>1</sup><i>J</i> = 275 Hz), 121.0, 119.0, 117.4, 111.7, 110.7, 109.2, 69.8, 69.8, 69.7, 69.6, 68.9, 68.9, 52.8, 48.6, 48.1, 42.4, 41.7, 38.9, 31.0, 22.1 ppm.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Synthesis of <i>N</i>-(4′-((23-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12,15,18,21-heptaoxatricosyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>7d</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 20.5 mg (18.5 μmol, 69%) of a yellow oil. MALDI: (calculated) [M + H<sup>+</sup>] 1107.46 g/mol, (found) [M + H<sup>+</sup>] 1107.52 g/mol. HRMS: (calculated) [M + H<sup>+</sup>] 1107.4645 g/mol, (found) [M + H<sup>+</sup>] 1107.4638 g/mol. HPLC: RT = 11.7 min (254 nm, 97%). <sup>1</sup>H NMR (500 MHz, DMSO) δ = 12.51 (s, 1H), 11.08 (s, 1H), 9.47 (s, 1H), 8.54 (t, <sup>3</sup><i>J</i> = 5.6 Hz, 1H), 8.12 (d, <sup>4</sup><i>J</i> = 2.0 Hz, 1H), 7.98 (s, 1H), 7.94 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.68 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.60–7.55 (m, 1H), 7.52 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.2 Hz, 1H), 7.26 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.13 (d, <sup>3</sup><i>J</i> = 8.6 Hz, 1H), 7.03 (d, <sup>3</sup><i>J</i> = 7.0 Hz, 1H), 6.82 (s, 1H), 6.59 (t, <sup>3</sup><i>J</i> = 5.7 Hz, 1H), 5.05 (dd, <sup>3</sup><i>J</i> = 12.7 Hz, <sup>4</sup><i>J</i> = 5.4 Hz, 1H), 3.61 (t, <sup>3</sup><i>J</i> = 5.4 Hz, 2H), 3.58–3.41 (m, 32H), 3.33–3.25 (m, 2H), 2.95–2.89 (m, 4H), 2.89–2.83 (m, 1H), 2.61–2.51 (m, 2H), 2.23 (s, 3H), 2.06–1.96 (m, 1H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.7, 170.0, 168.9, 167.3, 165.9, 162.8, 161.1, 146.4, 144.9, 142.2, 139.2, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 136.2, 134.2, 132.9, 132.2, 132.1, 127.9, 126.0, 123.9, 122.3, 122.0 (q, <sup>1</sup><i>J</i> = 275 Hz), 120.4, 119.0, 117.4, 111.7, 110.64 109.2, 69.8, 69.8, 69.7, 69.6, 68.9, 68.9, 54.7, 51.0, 48.6, 45.7, 41.7, 31.0, 22.1 ppm.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Synthesis of <i>N</i>-(4′-((4-(((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)methyl)benzyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>7e</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 7.6 mg (6.9 μmol, 30%) of a yellow solid as a TFA salt (1:2/product: TFA). MALDI: (calculated) [M + Na<sup>+</sup>] 897.29 g/mol, [M + H<sup>+</sup>] 875.31 g/mol, (found) [M + Na<sup>+</sup>] 897.2018 g/mol (100), [M + H<sup>+</sup>] 875.2220 g/mol (70). HRMS: (calculated) [M + H<sup>+</sup>] 875.3123 g/mol, (found) [M + H<sup>+</sup>] 875.3120 g/mol. HPLC: RT = 12.1 min (254 nm, 95%). <sup>1</sup>H NMR (500 MHz, DMSO) δ = 11.09 (s, 1H), 10.11 (s, 1H), 9.61 (s, 1H), 9.08 (t, <sup>3</sup><i>J</i> = 5.9 Hz, 1H), 8.25–8. 22 (m, 1H), 8.04 (s, 1H), 7.95 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.69 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.56 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 7.450–7.47 (m, 1H), 7.38–7.28 (m, 3H), 7.28–7.15 (m, 3H), 7.00 (d, <sup>3</sup><i>J</i> = 7.0 Hz, 1H), 6.95 (d, <sup>3</sup><i>J</i> = 8.6 Hz, 1H), 6.84 (s, 1H), 5.08–5.04 (m, 1H), 4.56 (d, <sup>3</sup><i>J</i> = 4.4 Hz, 2H), 4.49 (d, <sup>3</sup><i>J</i> = 4.8 Hz, 2H), 3.55–3.52 (m, 2H), 3.33–3.19 (m, 4H), 3.08–3.04 (m, 2H), 2.92–2.82 (m, 4H), 2.59–2.54 (m, 2H), 2.04–2.00 (m, 1H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 173.3, 170.5, 169.2, 167.7, 166.3, 163.6, 161.6, 146.6, 143.4, 142.5, 140.6, 139.5, 139.0, 138.6, 136.6, 135.8, 133.6, 133.1, 132.6, 129.0, 128.5, 126.6, 126.4, 126.1, 125.9, 124.4, 122.9, 122.6 (d, <sup>1</sup><i>J</i> = 275 Hz), 121.5, 119.0, 118.1, 111.3, 111.2, 110.0, 53.3, 49.0, 48.6, 45.9, 43.0, 42.9, 31.4, 22.6 ppm.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((2-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)ethyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8a</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 27.4 mg, 26.6 μmol, 59% of a white solid. MALDI: (calculated): [M + Na<sup>+</sup>] 1050.41 g/mol, (found): [M + Na<sup>+</sup>] 1050.20 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1050.4129 g/mol, (found) [M + Na<sup>+</sup>] 1050.4121 g/mol. HPLC: RT = 11.3 min (254 nm, 97%). <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.47 (s, 1H), 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 6.1 Hz, 1H), 8.50 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.12 (d, <sup>3</sup><i>J</i> = 2.1 Hz, 1H), 8.02–7.91 (m, 4H), 7.68 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.52 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>4</sup><i>J</i> = 2.2 Hz, 1H), 7.45–7.33 (m, 4H), 7.26 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.82 (s, 1H), 5.13 (s, 1H), 4.56 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.46–4.38 (m, 2H), 4.35 (s, 1H), 4.24–4.20 (m, 1H), 3.71–3.58 (m, 4H), 3.58–3.48 (m, 2H), 3.44–3.41 (m, 4H), 2.98–2.85 (m, 4H), 2.61–2.52 (m, 1H), 2.44–2.34 (m, 2H), 2.23 (s, 3H), 2.07–1.99 (m, 1H), 1.96–1.84 (m, 1H), 0.92 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 170.0, 169.6, 165.9, 162.8, 161.2, 151.4, 147.7, 144.9, 142.2, 139.5, 139.2, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 134.2, 132.9, 132.2, 131.2, 129.6, 128.6, 127.9, 127.4, 126.0, 123.9, 122.3, 122.0 (d, <sup>1</sup><i>J</i> = 275 Hz), 120.4, 119.0, 111.7, 68.9, 68.6, 66.6, 58.7, 56.4, 56.3, 54.7, 51.0, 45.7, 41.7, 39.0, 37.9, 35.6, 35.4, 26.3, 15.9 ppm.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((2-(2-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)ethoxy)ethyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8b</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 14.2 mg (10 μmol, 23%) of a white solid as a TFA salt (1:3/product: TFA.) MALDI: (calculated) [M + Na<sup>+</sup>] 1094.44 g/mol, (found) [M + Na<sup>+</sup>] 1094.44 g/mol. HPLC: RT = 11.3 min (254 nm, 100%). HRMS: (calculated) [M + Na<sup>+</sup>] 1094.4392 g/mol, (found) [M + Na<sup>+</sup>] 1094.4384 g/mol. <sup>1</sup>H NMR (600 MHz, DMSO) δ = 9.92 (s, 1H), 9.59 (s, 1H), 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.6 Hz, 2H), 8.22 (s, 1H), 8.04 (s, 1H), 7.95–7.90 (m, 3H), 7.69 (d, <sup>3</sup><i>J</i> = 8.2 Hz, 2H), 7.55 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 1.4 Hz, 1H), 7.42–7.37 (m, 4H), 7.31 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 1H), 6.83 (s, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.49–4.38 (m, 2H), 4.35 (s, 1H), 4.22 (dd, <sup>3</sup><i>J</i> = 15.8 Hz, <sup>4</sup><i>J</i> = 5.5 Hz, 2H), 3.68–3.65 (m, 1H), 3.64–3.56 (m, 3H), 3.56–3.47 (m, 8H), 3.46–4.42 (m, 2H), 3.28–3.25 (m, 2H), 3.24–3.19 (m, 2H), 3.06–3.02 (m, 2H), 2.87 (s, 3H), 2.53 (s, 1H), 2.44 (s, 3H), 2.40–2.29 (m, 1H), 2.08–2.00 (m, 1H), 1.96–1.86 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 170.0, 169.6, 165.9, 163.1, 161.1, 151.5, 147.7, 142.9, 142.0, 139.5, 139.1, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 135.3, 133.2, 132.64, 131.2, 129.6, 128.6, 128.0, 127.4, 126.1, 123.9, 122.32, 122.1 (q, <sup>1</sup><i>J</i> = 275 Hz), 121.0, 119.0, 118.8, 111.6, 69.6, 69.5, 69.0, 68.9, 66.9, 58.7, 56.4, 56.3, 52.85, 48.1, 42.4, 41.7, 38.0, 35.7, 35.4, 26.3, 15.9 ppm.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-(((<i>S</i>)-17-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-18,18-dimethyl-15-oxo-3,6,9,12-tetraoxa-16-azanonadecyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8c</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 11.5 mg (7.11 μmol, 16%) of a white solid as a TFA salt (1:4/product: TFA.) MALDI: (calculated) [M + H<sup>+</sup>] 1160.51 g/mol, (found) [M + H<sup>+</sup>] 1160.58 g/mol. HPLC: RT = 11.4 min (254 nm, 99%). HRMS: (calculated) [M + Na<sup>+</sup>] 1182.4916 g/mol, (found) [M + Na<sup>+</sup>] 1182.4913 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.76 (s, 1H), 9.57 (s, 1H), 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.4 Hz, 2H), 8.21 (d, <sup>4</sup><i>J</i> = 1.8 Hz, 1H), 8.03 (s, 1H), 7.95 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.90 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 7.69 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.56 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 7.40 (q, <sup>3</sup><i>J</i> = 8.3 Hz, 5H), 7.31 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.84 (s, 1H), 5.12 (s, 1H), 4.54 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.48–4.39 (m, 2H), 4.35 (s, 1H), 4.21 (dd, <sup>3</sup><i>J</i> = 15.9 Hz, <sup>4</sup><i>J</i> = 5.5 Hz, 1H), 3.68–3.64 (m, 1H), 3.64–3.57 (m, 4H), 3.56–3.41 (m, 20H), 3.25–3.15 (m, 2H), 3.05–3.00 (m, 2H), 2.86 (s, 3H), 2.44 (s, 3H), 2.37–2.30 (m, 1H), 2.07–1.99 (m, 1H), 1.95–1.85 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 170.0, 169.5, 165.9, 163.1, 161.1, 151.5, 147.6, 142.9, 142.0, 139.5, 139.1, 138.5 (q, <sup>2</sup><i>J</i> = 33 Hz), 135.4, 133.2, 132.7, 131.2, 129.6, 128.6, 128.0, 127.4, 126.1, 123.9, 122.3, 122.1 (q, <sup>1</sup><i>J</i> = 277 Hz), 121.0, 118.9, 111.7, 69.8, 69.7, 69.7, 69.6, 69.5, 68.9, 68.9, 66.9, 58.7, 56.4, 56.3, 52.8, 48.1, 42.4, 41.7, 38.0, 35.7, 35.4, 26.3, 15.9 ppm.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8d</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 22.2 mg, 24.3 μmol, 67% of a white solid. ESI: (calculated) [M + H<sup>+</sup>] 913.37 g/mol, (found) [M + H<sup>+</sup>] 913.63 g/mol. MALDI: (calculated) [M + Na<sup>+</sup>] 935.3496 g/mol, (found) [M + Na<sup>+</sup>] 935.0969 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 935.3496 g/mol, (found) [M + Na<sup>+</sup>] 935.3486 g/mol. HPLC: RT = 11.8 min (254 nm, 96%). <sup>1</sup>H NMR (600 MHz, DMSO) δ = 9.94 (s, 1H), 9.59 (s, 1H), 8.99 (s, 1H), 8.59 (t, <sup>3</sup><i>J</i> = 5.7 Hz, 1H), 8.23 (s, 1H), 8.03 (d, <sup>3</sup><i>J</i> = 10.0 Hz, 2H), 7.98 (d, <sup>3</sup><i>J</i> = 7.8 Hz, 2H), 7.69 (d, <sup>3</sup><i>J</i> = 7.8 Hz, 2H), 7.56 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 1H), 7.41 (q, <sup>3</sup><i>J</i> = 8.1 Hz, 4H), 7.33 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 1H), 6.84 (s, 1H), 4.80 (d, <sup>3</sup><i>J</i> = 9.0 Hz, 1H), 4.47 (t, <sup>3</sup><i>J</i> = 8.1 Hz, 1H), 4.47–4.41 (m, 2H), 4.39 (s, 1H), 4.25 (dd, <sup>3</sup><i>J</i> = 15.7 Hz, <sup>4</sup><i>J</i> = 5.3 Hz, 1H), 3.75 (s, 2H), 3.54–3.52 (m, 2H), 3.29–3.17 (m, 2H), 3.23–3.19 (m, 2H), 3.06–3.02 (m, 2H), 2.87 (s, 3H), 2.45 (s, 3H), 2.10–2.03 (m, 1H), 1.96–1.90 (m, 1H), 1.05 (s, 9H) ppm. <sup>13</sup>C NMR (151 MHz, DMSO) δ = 171.9, 169.5, 166.2, 162.9, 161.1, 151.5, 147.8, 142.9, 142.4, 139.5, 139.4, 138.8 (q, <sup>2</sup><i>J</i> = 33 Hz), 134.5, 132.7, 132.4, 131.2, 129.7, 128.7, 128.5, 127.5, 126.0, 124.0, 122.3, 121.1 (q, <sup>1</sup><i>J</i> = 275 Hz), 120.6, 119.1, 111.6, 68.9, 58.8, 57.3, 56.5, 52.8, 48.1, 41.7, 40.4, 37.9, 35.6, 26.5, 15.9 ppm.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8e</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 26.8 mg, 18.6 μmol, 43% of a white solid as a TFA salt (1:4/product: TFA.) MALDI: (calculated) [M + Na<sup>+</sup>] 1006.39 g/mol, (found) [M + Na<sup>+</sup>] 1006.42 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1006.3867 g/mol, (found): [M + Na<sup>+</sup>] 1006.3870 g/mol. HPLC: RT = 11.2 min (254 nm, 100%). <sup>1</sup>H NMR (600 MHz, DMSO) δ = 10.24 (s, 1H), 9.92 (s, 1H), 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.50–8.48 (m, 1H), 8.34 (s, 1H), 8.16 (s, 1H), 8.01 (d, <sup>3</sup><i>J</i> = 9.2 Hz, 1H), 7.94 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.67 (d, <sup>3</sup><i>J</i> = 7.8 Hz, 2H), 7.59 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 1H), 7.40 (dd, <sup>2</sup><i>J</i> = 22.6 Hz, <sup>3</sup>J = 7.6 Hz, 4H), 7.34 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 1H), 7.31 (s, 1H), 4.57 (d, <sup>3</sup><i>J</i> = 9.2 Hz, 1H), 4.47–4.39 (m, 2H), 4.36 (s, 1H), 4.22 (dd, <sup>3</sup><i>J</i> = 15.9 Hz, <sup>4</sup><i>J</i> = 5.3 Hz, 1H), 3.72–3.64 (m, 2H), 3.59–3.52 (m, 2H), 3.49 (dd, <sup>3</sup><i>J</i> = 15.9 Hz, <sup>4</sup><i>J</i> = 7.6 Hz, 2H), 3.31 (d, <sup>3</sup><i>J</i> = 11.4 Hz, 2H), 3.06 (s, 4H), 2.86 (s, 3H), 2.62–2.54 (m, 2H), 2.44 (s, 3H), 2.09–1.99 (m, 1H), 1.95–1.87 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (151 MHz, DMSO) δ = 171.9, 170.4, 169.6, 165.8, 163.1, 161.1, 151.5, 147.6, 142.9, 142.0, 139.5, 139.0, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 135.3, 133.3, 132.6, 131.2, 129.6, 128.6, 127.9, 127.4, 126.1, 124.0, 122.4, 122.1 (q, <sup>1</sup><i>J</i> = 279 Hz), 121.0, 118.9, 111.7, 68.9, 58.7, 56.5, 56.4, 52.9, 48.1, 42.4, 41.7, 38.0, 36.3, 35.3, 34.9, 26.4, 15.9 ppm.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((4-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-4-oxobutyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8f</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 14.7 mg, 14.7 μmol, 41% of a white solid. ESI: (calculated) [M + H<sup>+</sup>] 998.42 g/mol, (found) [M + H<sup>+</sup>] 998.35 g/mol (70). MALDI: (calculated): [M + Na<sup>+</sup>] 1020.40 g/mol, (found) [M + Na<sup>+</sup>] 1020.34 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1020.4024 g/mol, (found) [M + Na<sup>+</sup>] 1020.4010 g/mol. HPLC: RT = 11.3 min (254 nm, 100%). <sup>1</sup>H NMR (600 MHz, DMSO) δ = 9.82 (s, 1H), 9.58 (s, 1H), 8.97 (s, 1H), 8.55 (t, <sup>3</sup><i>J</i> = 6.0 Hz, 1H), 8.52 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.21 (s, 1H), 8.03 (s, 1H), 7.95 (d, <sup>3</sup><i>J</i> = 8.2 Hz, 3H), 7.69 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.56 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 1.9 Hz, 1H), 7.40 (dd, <sup>2</sup><i>J</i> = 22.3 Hz, <sup>3</sup><i>J</i> = 8.1 Hz, 4H), 7.32 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.84 (s, 1H), 4.56 (d, <sup>3</sup><i>J</i> = 9.3 Hz, 1H), 4.43 (dd, <sup>3</sup><i>J</i> = 14.8 Hz, <sup>4</sup><i>J</i> = 6.9 Hz, 2H), 4.36 (s, 1H), 4.22 (dd, <sup>3</sup><i>J</i> = 15.8 Hz, <sup>4</sup><i>J</i> = 5.4 Hz, 1H), 3.71–3.63 (m, 2H), 3.52 (d, <sup>3</sup><i>J</i> = 11.2 Hz, 2H), 3.28 (d, <sup>3</sup><i>J</i> = 7.4 Hz, 4H), 3.22–3.18 (m, 2H), 3.05–3.01 (m, 2H), 2.87 (s, 3H), 2.44 (s, 3H), 2.37–2.30 (m, 1H), 2.25–2.20 (m, 1H), 2.05–2.00 (m, 1H), 1.95–1.87 (m, 1H), 1.81–1.72 (m, 2H), 0.95 (s, 9H) ppm. <sup>13</sup>C NMR (201 MHz, DMSO) δ = 171.9, 171.8, 169.6, 165.8, 162.8, 161.0, 151.4, 147.7, 144.8, 142.0, 139.5, 139.1, 138.7 (q, <sup>2</sup><i>J</i> = 33 Hz), 134.3, 133.2, 132.2, 131.1, 129.6, 128.6, 127.9, 127.4, 126.0, 124.0, 122.4, 122.0 (q, <sup>1</sup><i>J</i> = 276 Hz), 120.4, 119.0, 111.6, 68.9, 58.5, 56.2, 56.1, 52.6, 48.0, 42.1, 41.4, 38.7, 37.7, 35.1, 32.4, 26.2, 25.4, 15.6 ppm.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((5-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-5-oxopentyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8g</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 7.42 mg (7.33 μmol, 23%) of a white solid. ESI: (calculated) [M + H<sup>+</sup>] 1012.43 g/mol, (found) [M + H<sup>+</sup>] 1012.62 g/mol. HPLC: RT = 11.4 min (254 nm, 100%). HRMS: (calculated) [M + Na<sup>+</sup>] 1034.4180 g/mol, (found) [M + Na<sup>+</sup>] 1034.4171 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.77 (s, 1H), 9.57 (s, 1H), 8.97 (s, 1H), 8.55 (t, <sup>3</sup><i>J</i> = 6.1 Hz, 1H), 8.50 (t, <sup>3</sup><i>J</i> = 5.6 Hz, 1H), 8.21 (d, <sup>4</sup><i>J</i> = 1.8 Hz, 1H), 8.03 (s, 1H), 7.94 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.86 (d, <sup>3</sup><i>J</i> = 9.3 Hz, 1H), 7.68 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.55 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 7.42–7.39 (m, 4H), 7.32 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.84 (s, 1H), 5.12 (s, 1H), 4.54 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.48–4.38 (m, 2H), 4.35 (s, 1H), 4.21 (dd, <sup>3</sup><i>J</i> = 15.9 Hz, <sup>4</sup><i>J</i> = 5.5 Hz, 1H), 3.70–3.63 (m, 2H), 3.55–3.46 (m, 2H), 3.29–3.23 (m, 4H), 3.07–3.02 (m, 2H), 2.86 (s, 3H), 2.53–2.50 (m, 2H), 2.44 (s, 3H), 2.35–2.26 (m, 1H), 2.19–2.14 (m, 1H), 2.09–1.99 (m, 1H), 1.93–1.88 (m, 1H), 1.60–1.47 (m, 4H), 0.94 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.0, 172.0, 169.7, 165.8, 163.2, 151.4, 147.7, 144.6, 142.1, 139.5, 139.1, 138.1 (d, <sup>2</sup><i>J</i> = 32 Hz), 134.4, 133.2, 132.5, 131.2, 129.6, 128.6, 127.9, 127.4, 126.0, 123.7, 121.8, 122.3 (q, <sup>1</sup><i>J</i> = 276 Hz), 120.5, 117.7, 111.7, 68.9, 58.7, 56.3, 56.3, 54.8, 51.1, 45.8, 41.6, 38.7, 37.9, 35.2, 34.7, 28.9, 26.4, 23.1, 15.9 ppm.</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((6-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-6-oxohexyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8h</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 15.5 mg (11.3 μmol, 26%) of a white solid as a TFA salt (1:3/product: TFA.) MALDI: (calculated) [M + Na<sup>+</sup>] 1048.43 g/mol, [M + H<sup>+</sup>] 1026.45 g/mol, (found) [M + Na<sup>+</sup>] 1048.43 g/mol, [M + H<sup>+</sup>] 1026.44 g/mol. HPLC: RT = 11.4 min (254 nm, 100%). HRMS: (calculated) [M + Na<sup>+</sup>] 1048.4337 g/mol, (found) [M + Na<sup>+</sup>] 1048.4338 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.96 (s, 1H), 9.59 (s, 1H), 8.98 (s, 1H), 8.56 (t, <i>J</i> = 6.0 Hz, 1H), 8.48 (t, <i>J</i> = 5.6 Hz, 1H), 8.22 (d, <i>J</i> = 2.0 Hz, 1H), 8.04 (s, 1H), 7.94 (d, <i>J</i> = 8.5 Hz, 2H), 7.84 (d, <i>J</i> = 9.3 Hz, 1H), 7.68 (d, <i>J</i> = 8.4 Hz, 2H), 7.55 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.45–7.36 (m, 4H), 7.32 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (s, 1H), 4.54 (d, <i>J</i> = 9.4 Hz, 1H), 4.47–4.38 (m, 2H), 4.35 (s, 1H), 4.22 (dd, <i>J</i> = 15.9, 5.4 Hz, 1H), 3.68–3.65 (m, 2H), 3.53 (d, <i>J</i> = 11.2 Hz, 2H), 3.28–3.20 (m, 6H), 3.04 (t, <i>J</i> = 11.3 Hz, 2H), 2.87 (s, 3H), 2.44 (s, 3H), 2.31–2.22 (m, 1H), 2.17–2.10 (m, 1H), 2.05–2.01 (m, 1H), 1.93–1.87 (m, 1H), 1.56–1.50 (m, 4H), 1.35–1.25 (m, 2H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.1, 172.0, 169.7, 165.7, 163.1, 161.1, 151.5, 147.7, 142.9, 141.9, 139.5, 139.0, 138.5 (q, <sup>2</sup><i>J</i> = 33 Hz), 135.4, 133.5, 132.6, 131.2, 129.6, 128.6, 127.9, 127.4, 126.1, 123.9, 122.3, 122.1 (q, <sup>1</sup><i>J</i> = 276 Hz) 121.0, 118.9, 118.8, 111.6, 68.9, 58.7, 56.4, 56.3, 52.8, 48.1, 42.4, 41.7, 38.0, 35.2, 34.9, 29.0, 26.4, 26.2, 25.3, 15.9 ppm.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((7-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-7-oxoheptyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8i</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A to give 4.67 mg, 4.49 μmol, 14% of a yellow solid (TFA salt with a stoichiometry of 1:4/product: TFA). MALDI: (calculated) [M + H<sup>+</sup>] 1040.21 g/mol, (found) [M + H<sup>+</sup>] 1040.25 g/mol. HPLC: RT = 11.6 min (254 nm, 98%). HRMS: (calculated) [M + Na<sup>+</sup>] 1062.4494 g/mol, (found) [M + Na<sup>+</sup>] 1062.4483 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 12.55 (s, 1H), 9.52 (s, 1H), 8.97 (s, 1H), 8.55 (t, <sup>3</sup><i>J</i> = 6.0 Hz, 1H), 8.46 (t, <sup>3</sup><i>J</i> = 5.6 Hz, 1H), 8.17 (s, 1H), 8.01 (s, 1H), 7.93 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.84 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 7.68 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.54 (dd, <sup>3</sup><i>J</i> = 8.3 Hz, <sup>4</sup>J = 2.1 Hz, 1H), 7.40 (q, <sup>3</sup><i>J</i> = 8.3 Hz, 4H), 7.30 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.83 (s, 1H), 5.11 (d, <sup>3</sup><i>J</i> = 3.6 Hz, 1H), 4.54 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.49–4.38 (m, 2H), 4.35 (s, 1H), 4.21 (dd, <sup>2</sup><i>J</i> = 16.0 Hz, <sup>3</sup>J = 5.4 Hz, 1H), 3.72–3.58 (m, 2H), 3.26 (dd, <sup>3</sup><i>J</i> = 13.3 Hz, <sup>4</sup>J = 6.7 Hz, 4H), 3.05 (s, 4H), 2.44 (s, 3H), 2.32–2.28 (m, 1H), 2.19–2.07 (m, 1H), 2.07–1.98 (m, 1H), 1.93–1.88 (m, 1H), 1.52–1.47 (m, 4H), 1.31–1.23 (m, 3H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 172.1, 171.9, 169.7, 165.7, 163.1, 161.1, 151.4, 147.7, 142.9, 141.8, 139.5, 139.1, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 135.4, 133.5, 132.6, 131.2, 129.6, 128.6, 127.9, 127.4, 126.1, 123.9, 123.1, 122.3, 122.1 (q, <sup>1</sup><i>J</i> = 273 Hz), 121.0, 119.0, 111.6, 68.9, 58.7, 56.3, 56.3, 52.8, 48.1, 42.3, 41.6, 38.4, 38.0, 35.2, 34.8, 29.1, 26.4, 25.4, 15.9 ppm.</div></div><div id="sec4_4_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((4-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethyl)benzyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>8j</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 14.4 mg, 13.6 μmol, 30% of a white solid. MALDI: (calculated) [M + H<sup>+</sup>] 1060.44 g/mol, (found) [M + H<sup>+</sup>] 1060.17 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1082.4180 g/mol, (found) [M + Na<sup>+</sup>] 1082.4171 g/mol. HPLC: RT = 11.6 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.47 (s, 1H), 9.04 (t, <sup>3</sup><i>J</i> = 6.0 Hz, 1H), 8.98 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 6.1 Hz, 1H), 8.12 (d, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 8.09 (d, <sup>3</sup><i>J</i> = 9.3 Hz, 1H), 8.01–7.93 (m, 3H), 7.69 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.53 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>4</sup><i>J</i> = 2.2 Hz, 1H), 7.40 (q, <sup>3</sup><i>J</i> = 8.4 Hz, 4H), 7.27 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 7.24 (s, 4H), 6.82 (s, 1H), 5.11 (d, <sup>3</sup><i>J</i> = 3.1 Hz, 1H), 4.51 (d, <sup>3</sup><i>J</i> = 9.4 Hz, 1H), 4.47 (d, <sup>3</sup><i>J</i> = 5.8 Hz, 2H), 4.45–4.39 (m, 2H), 4.33 (s, 1H), 4.22 (dd, <sup>3</sup><i>J</i> = 15.9 Hz, <sup>4</sup><i>J</i> = 5.5 Hz, 1H), 3.71–3.58 (m, 3H), 3.44 (d, <sup>3</sup><i>J</i> = 13.9 Hz, 2H), 2.91 (t, <sup>3</sup><i>J</i> = 4.3 Hz, 4H), 2.50 (s, 4H), 2.44 (s, 3H), 2.24 (s, 3H), 2.07–1.97 (m, 1H), 1.92–1.86 (m, 1H), 0.92 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 170.0, 169.5, 165.8, 162.8, 161.1, 151.4, 147.72, 145.0, 142.3, 139.5, 139.2, 138.5 (q, <sup>2</sup><i>J</i> = 32 Hz), 137.6, 135.1, 134.2, 132.9, 132.2, 131.2, 129.6, 129.0, 128.6, 128.0, 127.4, 127.0, 126.1, 124.0, 122.4, 122.0 (d, <sup>1</sup><i>J</i> = 275 Hz), 120.4, 119.0, 111.6, 68.9, 58.7, 56.5, 56.4, 54.7, 51.0, 45.7, 42.4, 41.7, 41.5, 37.9, 35.4, 26.3, 15.9 ppm.</div></div><div id="sec4_4_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Synthesis of <i>N</i>-(4′-((2-(2-(4-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-3-(3-Chloro-2-fluorophenyl)-4-(4-chloro-2-fluorophenyl)-4-cyano-5-neopentylpyrrolidine-2-carboxamido)-3-methoxybenzamido)ethoxy)ethyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>9a</b>)</h4><div class="NLM_p last">First, 15.2 mg (21 μmol, 1.05 equiv) of intermediate <b>6a</b> was dissolved in 2 mL of TFA/CH<sub>2</sub>Cl<sub>2</sub> and stirred for 1 h at rt. The excess solvent was removed under reduced pressure. Then, 13 mg (21 μmol, 1.0 equiv) of idasanutlin, 9.6 mg (25 μmol, 1.2 equiv) of HATU, and 7 μL (42 μmol, 2.0 equiv) of DIEA were dissolved in 1 mL of DMF and stirred for 15 min at rt. Then, a solution of deprotected <b>6a</b> and 73 μL (420 μmol, 10 equiv) of DIEA in 1 mL of DMF were added to the solution and stirred for 12 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified by HPLC. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 3.0 mg, 2.5 μmol, 12% of a clear oil. MALDI: (calculated) [M + H<sup>+</sup>] 1184.49 g/mol, (found) [M + H<sup>+</sup>] 1184.40 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1206.3805 g/mol, (found) [M + Na<sup>+</sup>] 1206.3821 g/mol. HPLC: RT = 14.4 min (254 nm, 98%). <sup>1</sup>H NMR (400 MHz, DMSO) δ = 10.45 (s, 1H), 10.40 (s, 1H), 9.45 (s, 1H), 8.53 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.48 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.35 (s, 1H), 8.32 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 8.13 (d, <sup>4</sup><i>J</i> = 1.9 Hz, 1H), 7.92 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.73 (t, <sup>3</sup><i>J</i> = 7.3 Hz, 2H), 7.67 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 7.60–7.56 (m, 4H), 7.53–7.50 (m, 3H), 7.44–7.38 (m, 1H), 7.38–7.31 (m, 4H), 7.25 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.81 (s, 1H), 4.65–4.54 (m, 4H), 4.40–4.33 (m, 2H), 3.99–3.95 (m, 2H), 3.92 (s, 3H), 3.90 (s, 3H), 3.60–3.57 (m, 5H), 3.48–3.43 (m, 8H), 2.92–2.88 (m, 4H), 1.69–1.60 (m, 2H), 1.39–1.36 (m, 1H), 1.28–1.24 (m, 2H), 0.96 (s, 9H) ppm. Contains rotameres.</div></div><div id="sec4_4_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Synthesis of <i>N</i>-(4′-((1-(4-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-3-(3-Chloro-2-fluorophenyl)-4-(4-chloro-2-fluorophenyl)-4-cyano-5-neopentylpyrrolidine-2-carboxamido)-3-methoxyphenyl)-1-oxo-5,8,11,14,17,20,23-heptaoxa-2-azapentacosan-25-yl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>9b</b>)</h4><div class="NLM_p last">First, 8.9 mg (9 μmol, 1.0 equiv) of intermediate <b>6b</b> was dissolved in 2 mL of TFA/CH<sub>2</sub>Cl<sub>2</sub> and stirred for 1 h at rt. The excess solvent was removed under reduced pressure. Then, 5.5 mg (9 μmol, 1.0 equiv) of idasanutlin, 4 mg (11 μmol, 1.2 equiv) of HATU, and 3 μL (18 μmol, 2.0 equiv) of DIEA were dissolved in 1 mL of DMF and stirred for 15 min at rt. Then, a solution of deprotected <b>6b</b> and 31 μL (180 μmol, 10 equiv) of DIEA in 1 mL of DMF were added to the solution and stirred for 12 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified by HPLC. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 3.3 mg, 2.2 μmol, 25% of a clear oil. MALDI: (calculated) [M + H<sup>+</sup>] 1448.56 g/mol, (found) [M + H<sup>+</sup>] 1448.53 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1470.5378 g/mol, (found) [M + Na<sup>+</sup>] 1470.5392 g/mol. HPLC: RT = 14.6 min (254 nm, 96%). <sup>1</sup>H NMR (400 MHz, DMSO) δ = 10.47 (s, 1H), 10.40 (s, 1H), 9.46 (s, 1H), 8.53 (t, <sup>3</sup><i>J</i> = 5.5 Hz, 1H), 8.48 (t, <sup>3</sup><i>J</i> = 5.4 Hz, 1H), 8.36 (d, <sup>3</sup><i>J</i> = 8.8 Hz, 1H), 8.32 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 2H), 8.12 (d, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 7.92 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.73 (t, <sup>3</sup><i>J</i> = 7.2 Hz, 2H), 7.67 (d, <sup>3</sup><i>J</i> = 8.5 Hz, 2H), 7.60–7.56 (m, 5H), 7.54–7.51 (m, 2H), 7.50–7.48 (m, 1H), 7.40 (dd, <sup>3</sup><i>J</i> = 8.4 Hz, <sup>4</sup><i>J</i> = 2.4 Hz, 1H), 7.38–7.34 (m, 3H), 7.34–7.32 (m, 1H), 7.25 (d, <sup>3</sup><i>J</i> = 8.4 Hz, 1H), 6.82 (s, 1H), 4.65–4.54 (m, 4H), 4.41–4.33 (m, 2H), 3.99–3.95 (m, 2H), 3.92 (s, 3H), 3.90 (s, 3H), 3.60–3.57 (m, 5H), 3.49–3.41 (m, 8H), 2.91–2.90 (m, 4H), 1.68–1.61 (m, 2H), 1.44–1.33 (m, 2H), 0.96 (s, 9H) ppm. Contains rotameres.</div></div><div id="sec4_4_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Synthesis of <i>N</i>-(4′-((4-((4-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-3-(3-Chloro-2-fluorophenyl)-4-(4-chloro-2-fluorophenyl)-4-cyano-5-neopentylpyrrolidine-2-carboxamido)-3-methoxybenzamido)methyl)benzyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-hydroxy-4-(trifluoromethyl)nicotinamide (<b>9c</b>)</h4><div class="NLM_p last">First, 13.5 mg (24 μmol, 1.0 equiv) of intermediate <b>6c</b> was dissolved in 2 mL of TFA/CH<sub>2</sub>Cl<sub>2</sub> and stirred for 1 h at rt. The excess solvent was removed under reduced pressure. Then, 15.7 mg (25 μmol, 1.1 equiv) of idasanutlin, 11 mg (29 μmol, 1.2 equiv) of HATU, and 8 μL (49 μmol, 2.0 equiv) of DIEA were dissolved in 1 mL of DMF and stirred for 15 min at rt. Then, a solution of deprotected <b>6c</b> and 84 μL (490 μmol, 10 equiv) of DIEA in 1 mL of DMF were added to the solution and stirred for 12 h at rt. The reaction mixture was quenched with 2 mL of water and 2 mL of saturated NaHCO<sub>3</sub>; then, the reaction was extracted 3× with EA. The organic phase was dried over MgSO<sub>4</sub> and filtered, and the solvent was removed under reduced pressure. The crude product was purified by HPLC. The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 3.7 mg, 3.0 μmol, 13% of a clear oil. MALDI: (calculated) [M + K<sup>+</sup>] 1254.36 g/mol, (found) [M + K<sup>+</sup>] 1254.32 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1238.3856 g/mol, (found) [M + Na<sup>+</sup>] 1238.3894 g/mol. HPLC: RT = 14.7 min (254 nm, 98%). <sup>1</sup>H NMR (250 MHz, DMSO) δ = 10.40 (s, 1H), 9.13 (s, 1H), 9.04 (s, 1H), 8.96 (s, 1H), 8.49 (s, 3H), 8.44 (s, 1H), 8.31 (d, <sup>3</sup><i>J</i> = 9.1 Hz, 2H), 8.20 (d, <sup>3</sup><i>J</i> = 11.9 Hz, 1H), 7.98 (d, <sup>3</sup><i>J</i> = 7.9 Hz, 2H), 7.69 (d, <sup>3</sup><i>J</i> = 7.4 Hz, 2H), 7.61 (s, 2H), 7.57–7.52 (m, 2H), 7.52–7.47 (m, 1H), 7.39 (s, 1H), 7.36 (s, 2H), 7.29 (s, 4H), 6.52 (s, 1H), 6.13 (s, 1H), 4.59–4.58 (m, 2H), 4.51–4.41 (m, 4H), 3.91 (s, 4H), 3.57–3.50 (m, 6H), 3.47 (d, <sup>3</sup><i>J</i> = 4.8 Hz, 2H), 3.11 (d, <sup>3</sup><i>J</i> = 5.1 Hz, 2H), 3.03–2.99 (m, 2H), 2.44–2.37 (m, 1H), 0.97 (s, 9H) ppm. Contains rotameres.</div></div><div id="sec4_4_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Synthesis of 6-Hydroxy-<i>N</i>-(4′-((5-(((<i>S</i>)-1-((2<i>S</i>,4<i>S</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-5-oxopentyl)carbamoyl)-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-4-(trifluoromethyl)nicotinamide (<b>20</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure A with intermediate <b>6</b> and <i>tert</i>-butyl (5-(((<i>S</i>)-1-((2<i>S</i>,4<i>S</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-5-oxopentyl)carbamate (<b>L15</b>). The gained product was then dissolved in ethyl acetate and a solution of saturated NaHCO<sub>3</sub> and saturated NaCl solution and extracted with ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 9 mg, 8.9 μmol, 18% of a white oil. MALDI: (calculated) [M-CH<sub>3</sub>-<i>t</i>Bu+2H<sup>+</sup>] 941.35 g/mol, (found) [M-CH<sub>3</sub>-<i>t</i>Bu+2H<sup>+</sup>] 941.45 g/mol. HRMS: (calculated) [M + Na<sup>+</sup>] 1034.4180 g/mol, (found) [M + Na<sup>+</sup>] 1034.4169 g/mol. HPLC: RT = 11.4 min (254 nm, 100%). <sup>1</sup>H NMR (250 MHz, MeOD) δ = 9.03 (s, 1H), 8.26 (d, <sup>4</sup><i>J</i> = 1.9 Hz, 1H), 8.03 (s, 1H), 7.91 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.71 (d, <sup>3</sup><i>J</i> = 8.3 Hz, 2H), 7.56 (dd, <sup>3</sup><i>J</i> = 8.5 Hz, <sup>4</sup><i>J</i> = 2.0 Hz, 1H), 7.44 (dd, <sup>2</sup><i>J</i> = 15.1 Hz, <sup>3</sup><i>J</i> = 7.8 Hz, 4H), 7.37 (d, <sup>4</sup><i>J</i> = 2.1 Hz, 1H), 6.94 (s, 1H), 4.63 (s, 1H), 4.56 (s, 1H), 4.50 (s, 2H), 4.35 (d, <sup>3</sup><i>J</i> = 15.7 Hz, 1H), 3.91 (d, <sup>3</sup><i>J</i> = 10.7 Hz, 1H), 3.80 (dd, <sup>3</sup><i>J</i> = 10.9 Hz, <sup>4</sup><i>J</i> = 3.6 Hz, 1H), 3.64–3.60 (m, 2H), 3.44–3.39 (m, 2H), 3.36–3.24 (m, 4H), 3.20–3.15 (m, 2H), 2.97 (s, 3H), 2.47 (s, 3H), 2.39–2.33 (m, 2H), 2.28–2.15 (m, 1H), 2.13–2.08 (m, 1H), 1.79–1.59 (m, 4H), 1.04 (s, 9H) ppm.</div></div><div id="sec4_4_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Synthesis of 2-Bromo-6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazole (<b>11</b>)</h4><div class="NLM_p last">This reaction was performed in two equal batches. First, 1.00 g (7.03 mmol, 1.0 equiv) of piracetam <b>10</b> and 4.00 g (14.0 mmol, 2.0 equiv) of POBr<sub>3</sub> were dissolved in 20 mL of acetonitrile and heated in a microwave under stirring for 80 min at 5 W to 70 °C. Solid POBr<sub>3</sub> that had formed at the top of the vial was brought back into the reaction mixture with a spatula, and the reaction mixture was heated in a microwave for another 45 min. After cooling to rt, the batches were combined and quenched with 35 mL of water. K<sub>2</sub>CO<sub>3</sub> was added until no gas formation was observed, and the solvent was removed under reduced pressure. The aqueous mixture was extracted with dichloromethane (5 × 35 ml), washed with brine, and dried with MgSO<sub>4</sub>. The solvent was removed under reduced pressure. Purification was performed by flash column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH) to give 2.16 g, 5.76 mmol, 82% of a colorless solid. <i>R<sub>f</sub></i> (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.70. ESI: (calculated) [M + H<sup>+</sup>] 189.99 g/mol, (found) [M + H<sup>+</sup>] 189.09 g/mol; m.p. 95.7 °C. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ = 7.05 (s, 1H), 4.02 (t, <sup>3</sup><i>J</i> = 7.20 Hz, 2H), 2.82 (t, <sup>3</sup><i>J</i> = 7.40 Hz, 2H), 2.60–2.54 (m, 2H) ppm. <sup>13</sup>C NMR (125.8 MHz, CD<sub>3</sub>OD) δ = 155.4, 116.6, 116.0, 46.5, 26.4, 24.1 ppm.</div></div><div id="sec4_4_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Synthesis of 5-(6,7-Dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)-2-methoxybenzonitrile (<b>12</b>)</h4><div class="NLM_p last">A solution of 538 mg (13.5 mmol, 5.2 equiv) of NaOH and 693 mg (3.92 mmol, 1.5 equiv) of (3-cyano-4-methoxyphenyl)boronic acid in 24 mL of tetrahydrofuran/water (3/1) was stirred at 50 °C for 5 min. Bromide <b>11</b> (480 mg, 2.57 mmol, 1.0 equiv) and 113 mg (0.13 mmol. 0.1 equiv) of the XPhos PdG3 catalyst were added, and the reaction mixture was stirred for 21 h at 80 °C. The two batches were combined, and the organic solvent was removed under reduced pressure. The remaining aqueous layer was extracted with dichloromethane (3×). The combined organic layers were washed with brine and dried with MgSO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification by flash column chromatography (EA) gave 386 mg, 1.62 mmol, 63% of a colorless solid. <i>R<sub>f</sub></i> (EA) 0.43. ESI: (calculated) [M + H<sup>+</sup>] 240.12 g/mol, (found) [M + H<sup>+</sup>] 240.05 g/mol. M.p. 160.9 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ = 7.99 (dd, <sup>3</sup><i>J</i> = 8.82 Hz, <sup>4</sup><i>J</i> = 1.90 Hz, 1H), 7.87 (d, <sup>4</sup><i>J</i> = 1.90 Hz, 1H), 7.13 (s, 1H), 6.98 (d, <sup>3</sup><i>J</i> = 8.82 Hz, 1H), 4.05 (t, <sup>3</sup><i>J</i> = 6.98 Hz, 2H), 3.94 (s, 3H), 2.97 (t, <sup>3</sup><i>J</i> = 7.48 Hz, 2H), 2.68–2.62 (m, 2H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 160.1, 155.1, 143.6, 130.8, 129.8, 127.8, 116.6, 111.7, 110.3, 102.0, 56.3, 45.3, 26.2, 23.2 ppm.</div></div><div id="sec4_4_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Synthesis of (5-(6,7-Dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)-2-methoxyphenyl)methanamine (<b>13</b>)</h4><div class="NLM_p last">First, 16.1 mL (16.1 mmol, 5.0 equiv) of LiAlH<sub>4</sub> (1 M in THF) was added dropwise to a colorless solution of 772 mg (3.23 mmol, 1.0 equiv) of nitrile <b>12</b> in 32 mL of tetrahydrofuran via a syringe. The reaction mixture was stirred at 60 °C for 2.5 h, became orange, and was stirred overnight at room temperature. The reaction was quenched with an aqueous solution of 1 M NaOH (35 ml) and filtered, and the solvent was removed under reduced pressure. The remaining aqueous phase was extracted with dichloromethane (2×). The combined organic layers were washed with brine and dried with MgSO<sub>4</sub>. The solvent was removed under reduced pressure. The title compound <b>12</b> (691 mg, 2.84 mmol, 88%) was isolated as a sticky yellow solid and used without further purification. ESI (calculated) [M + H<sup>+</sup>] 244.15 g/mol, (found) [M + H<sup>+</sup>] 244.22 g/mol. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ = 7.59 (t, <sup>4</sup><i>J</i> = 1.28 Hz, 1H), 7.57 (dd, <sup>3</sup><i>J</i> = 8.26 Hz, <sup>4</sup><i>J</i> = 1.28 Hz, 1H), 7.07 (s, 1H), 6.84 (d, <sup>3</sup><i>J</i> = 8.26 Hz, 1H), 3–97 (t, <sup>3</sup><i>J</i> = 6.67 Hz, 2H), 3.84 (s, 3H), 3.81 (s, 2H), 2.89 (t, <sup>3</sup><i>J</i> = 7.56 Hz, 2H), 2.58 (quin, <sup>3</sup><i>J</i> = 7.23 Hz, 2H), 1.92 (br, s) ppm. The spectrum contains minor impurities. <sup>13</sup>C NMR (125.8 MHz, DMSO) δ = 155.8, 154.5, 145.2, 128.3, 127.9, 125.4, 124.5, 111.0, 110.5, 55.9, 44.7, 39.6, 26.1, 22.3 ppm.</div></div><div id="sec4_4_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Synthesis of 2-(3,4-Dichlorophenyl)-<i>N</i>-(5-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)-2-methoxybenzyl)acetamide (<b>14</b>)</h4><div class="NLM_p last">First, 118 mg of EDC hydrochloride (0.647 mmol, 1.5 equiv) was added to a solution of the amine <b>13</b> (100 mg, 0.411 mmol, 1.0 equiv), 126 mg (0.617 mmol, 1.5 equiv) of 2-(3,4-dichlorophenyl)acetic acid, 94 mg (0.65 μmol, 1.5 equiv) of HOBT monohydrate, and 0.11 mL (0.63 mmol, 1.5 equiv) of DIPEA in 1 mL of dimethylformamide at 0 °C. The reaction mixture was stirred at room temperature overnight. The reaction was quenched with water and extracted with ethyl acetate (4×). The combined organic layers were dried with MgSO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification by flash column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH) yielded the title compound <b>14;</b> 110 mg, 0.259 mmol, 62% of a colorless solid. <i>R<sub>f</sub></i> (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.59. ESI: (calculated) [M + H<sup>+</sup>] 430.11 g/mol, (found) [M + H<sup>+</sup>] 430.09 g/mol. HPLC: RT = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ = 7.64 (dd, <sup>3</sup><i>J</i> = 8.44 Hz, <sup>4</sup><i>J</i> = 1.79 Hz, 1H), 7.53 (s, 1H), 7.37 (s, 1H), 7.36 (d, <sup>3</sup><i>J</i> = 5.57 Hz, 1H), 7.11 (dd, <sup>3</sup><i>J</i> = 8.20 Hz, <sup>4</sup><i>J</i> = 1.51 Hz, 1H), 7.01 (s, 1H), 6.23 (t, <sup>3</sup><i>J</i> = 4.60 Hz, 1H), 4.42 (d, <sup>3</sup><i>J</i> = 5.60 Hz, 2H), 3.99 (t, <sup>3</sup><i>J</i> = 7.14 Hz, 2H), 3.76 (s, 3H), 7.49 (s, 2H), 2.90 (t, <sup>3</sup><i>J</i> = 7.46 Hz, 2H), 2.60 (quin, <sup>3</sup><i>J</i> = 7.46, 2H). <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 169.6, 156.6, 154.4, 145.0, 135.7, 132.6, 131.4, 131.2, 130.6, 129.0, 126.7, 126.1, 126.0, 125.2, 110.6, 109.7, 55.5, 45.3, 42.7, 39.7, 26.1, 23.2 ppm.</div></div><div id="sec4_4_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Synthesis of 2-(3,4-Dichlorophenyl)-<i>N</i>-(5-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)-2-hydroxybenzyl) Acetamide (<b>15</b>)</h4><div class="NLM_p last">First, 1.22 mL (1.22 mmol, 5.0 equiv) of a solution of 1 M BBr<sub>3</sub> in dichloromethane was added to a solution of 105 mg (0.244 mmol, 1.0 equiv) of the protected phenol <b>14</b> in 15 mL of dichloromethane at −78 °C via a syringe. After 30 min, the reaction mixture was allowed to warm to rt and turned orange while stirring for another 21 h. The reaction was quenched by adding 6.5 mL of an aqueous solution of 1 M NaOH and stirred for 4 h. The organic layer was separated, and the remaining aqueous layer was extracted with dichloromethane (2×). The combined organic layers were dried with MgSO<sub>4</sub>, and the solvent was removed under reduced pressure to give 101 mg, 0.242 mmol, 99% of a colorless solid without further purification. <i>R<sub>f</sub></i> (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.41. ESI: (calculated) [M + H<sup>+</sup>] 416.10 g/mol, (found) [M + H<sup>+</sup>] 416.09 g/mol. <sup>1</sup>H NMR (500 MHz, DMSO) δ = 9.45 (s, 1H), 8.50 (t, <sup>3</sup><i>J</i> = 5.70 Hz, 1H), 7.58–7.55 (m, 2H), 7.42–7.39 (m, 2H), 7.31 (dd, <sup>3</sup><i>J</i> = 8.31 Hz, <sup>4</sup><i>J</i> = 1.85 Hz, 1H), 7.15 (s, 1H), 6.77–6.74 (m, 1H), 4.23–4.20 (m, 2H), 3.94 (t, <sup>3</sup><i>J</i> = 7.01 Hz, 2H), 3.54 (s, 2H), 2.73 (t, <sup>3</sup><i>J</i> = 7.13 Hz), 2.54–2.47 (m, 2H) ppm. <sup>13</sup>C NMR (125.8 MHz, DMSO) δ = 169.6, 153.8, 153.4, 145.1, 137.6, 131.0, 130.7, 130.3, 129.5, 129.1, 126.4, 124.7, 124.4, 123.8, 115.0, 109.2, 44.3, 41.0, 37.8, 25.6, 22.4 ppm.</div></div><div id="sec4_4_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Synthesis of <i>tert</i>-Butyl 3-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)propanoate (<b>16a</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 3-(2-hydroxyethoxy)propanoate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (31 mg). MALDI: (calculated) [M + H<sup>±</sup>] 588.21 g/mol, (found) [M + H<sup>±</sup>] 588.25 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.72 (t, <sup>3</sup><i>J</i> = 6.00 Hz, 1H), 7.50 (d, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.45 (d, <sup>4</sup><i>J</i> = 1.98 Hz, 1H), 7.38 (dd, <sup>3</sup><i>J</i> = 8.44 Hz, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.31 (d, <sup>3</sup><i>J</i> = 8.22 Hz, 1H), 7.16 (dd, <sup>3</sup><i>J</i> = 8.22 Hz, <sup>4</sup><i>J</i> = 1.98 Hz, 1H), 7.14 (s, 1H) 6.78 (d, <sup>3</sup><i>J</i> = 8.57 Hz, 1H), 4.41 (d, <sup>3</sup><i>J</i> = 6.00 Hz, 2H), 4.16 (t, <sup>3</sup><i>J</i> = 7.25 Hz, 2H), 4.10–4.07 (m, 2H), 3.80–3.76 (m, 4H), 3.59 (s, 2H), 3.20 (t, <sup>3</sup><i>J</i> = 7.50 Hz, 2H), 2.77–2.70 (m, 2H), 2.50 (t, <sup>3</sup><i>J</i> = 6.12 Hz, 2H), 1.42 (s, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 171.2, 171.0, 157.4, 152.0, 139.2, 136.0, 132.3, 131.3, 130.9, 130.44, 129.1, 128.2, 125.9, 119.6, 112.2, 110.9, 81.0, 69.3, 68.1, 67.3, 47.6, 42.1, 38.1, 36.4, 28.2, 25.7, 23.5 ppm.</div></div><div id="sec4_4_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Synthesis of <i>tert</i>-Butyl 3-(2-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)ethoxy)propanoate (<b>16b</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 3-(2-(2-hydroxyethoxy)ethoxy)propanoate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (74 mg). MALDI: (calculated) [M + H]<sup>±</sup> 632.23 g/mol, (found) [M + H]<sup>±</sup> 632.19 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.63 (t, <sup>3</sup><i>J</i> = 6.00 Hz, 1H), 7.56 (d, <sup>4</sup><i>J</i> = 1.91 Hz, 1H), 7.47 (dd, <sup>3</sup><i>J</i> = 8.50 Hz, <sup>4</sup><i>J</i> = 1.91 Hz, 1H), 7.45 (d, <sup>4</sup><i>J</i> = 1.70 Hz, 1H), 7.32 (d, <sup>3</sup><i>J</i> = 8.17 Hz, 1H), 7.17 (dd, <sup>3</sup><i>J</i> = 8.17 Hz, <sup>4</sup><i>J</i> = 1.70 Hz, 1H), 7.12 (s, 1H), 6.86 (d, <sup>3</sup><i>J</i> = 8.50 Hz, 1H), 4.45 (d, <sup>3</sup><i>J</i> = 6.00 Hz, 2H), 4.21 (t, <sup>3</sup><i>J</i> = 7.24 Hz, 2H), 4.16–4.12 (m, 2H), 3.86–3.82 (m, 2H), 3.79–3.60 (m, 6H), 3.57 (s, 2H), 3.30 (t, <sup>3</sup><i>J</i> = 7.48 Hz, 2H), 2.83–2.75 (m, 2H), 2.48 (t, <sup>3</sup><i>J</i> = 6.42 Hz, 2H), 1.42 (s, 9H) ppm. Spectrum contains CH<sub>2</sub>Cl<sub>2</sub>. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>): δ = 171.1, 170.9, 157.5, 152.3, 139.7, 136.1, 132.3, 131.3, 130.9, 130.4, 129.1, 128.4, 126.1, 126.01, 119.7, 112.5, 110.6, 80.9, 70.8, 70.4, 69.6, 68.1, 67.0, 53.6 (CH<sub>2</sub>Cl<sub>2</sub>), 47.6, 42.2, 38.8, 36.36, 28.2, 25.8, 23.6 ppm.</div></div><div id="sec4_4_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Synthesis of <i>tert</i>-Butyl 3-(2-(2-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)ethoxy)ethoxy)propanoate (<b>16c</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 3-(2-(2-(2-hydroxyethoxy)ethoxy)ethoxy)propanoate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (43 mg). MALDI: (calculated) [M + H<sup>±</sup>] 676.26 g/mol, (found) [M + H<sup>±</sup>] 676.25 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.89 (t, <sup>3</sup><i>J</i> = 6.08 Hz, 1H), 7.51 (d, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.45 (d, <sup>4</sup><i>J</i> = 1.99 Hz, 1H), 7.38 (dd, <sup>3</sup><i>J</i> = 8.51 Hz, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.30 (d, <sup>3</sup><i>J</i> = 8.19 Hz, 1H), 7.17 (dd, <sup>3</sup><i>J</i> = 8.19 Hz, <sup>4</sup><i>J</i> = 1.99 Hz, 1H), 7.14 (s, 1H), 6.78 (d, <sup>3</sup><i>J</i> = 8.51 Hz, 1H), 4.41 (d, <sup>3</sup><i>J</i> = 6.08 Hz, 2H), 4.14 (t, <sup>3</sup><i>J</i> = 7.32 Hz, 2H), 4.12–4.09 (m, 2H), 3.84–3.81 (m, 2H), 3.72–3.53 (m, 12H), 3.18 (t, <sup>3</sup><i>J</i> = 7.66 Hz, 2H), 2.75–2.67 (m, 2H), 2.45 (t, <sup>3</sup><i>J</i> = 6.46 Hz, 2H), 1.41 (s, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 171.0, 171.0, 157.3, 152.0, 139.1, 136.2, 132.2, 131.2, 130.7, 130.4, 129.1, 128.3, 125.8, 119.7, 112.2, 111.0, 80.8, 70.8, 70.5, 70.5, 70.4, 69.6, 68.1, 67.0, 47.6, 42.1, 38.7, 36.3, 28.2, 25.7, 23.4 ppm.</div></div><div id="sec4_4_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Synthesis of <i>tert</i>-Butyl 1-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-3,6,9,12-tetraoxapentadecan-15-oate (<b>16d</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 1-hydroxy-3,6,9,12-tetraoxapentadecan-15-oate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (44 mg). MALDI: (calculated) [M + H]<sup>±</sup> 720.28 g/mol, (found) [M + H]<sup>±</sup> 720.33 g/mol; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.93 (t, <sup>3</sup><i>J</i> = 6.03 Hz, 1H), 7.47 (d, <sup>4</sup><i>J</i> = 2.19 Hz, 1H), 7.41 (d, <sup>4</sup><i>J</i> = 2.02 Hz, 1H), 7.39 (dd, <sup>3</sup><i>J</i> = 8.52 Hz, <sup>4</sup><i>J</i> = 2.19 Hz, 1H), 7.30 (d, <sup>3</sup><i>J</i> = 8.27 Hz, 1H), 7.19 (s, 1H), 7.13 (dd, <sup>3</sup><i>J</i> = 8.27 Hz, <sup>4</sup><i>J</i> = 2.02 Hz, 1H), 6.79 (d, <sup>3</sup><i>J</i> = 8.52 Hz, 1H), 4.41 (d, <sup>3</sup><i>J</i> = 6.03 Hz, 2H), 4.18 (t, <sup>3</sup><i>J</i> = 7.31 Hz, 2H), 4.13–4.08 (m, 2H), 3.85–3.81 (m, 2H), 3.76–3.52 (m, 16H), 3.20 (t, <sup>3</sup><i>J</i> = 7.53 Hz, 2H), 2.79–2.71 (m, 2H), 2.46 (t, <sup>3</sup><i>J</i> = 6.45 Hz, 2H), 1.41 (s, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 171.6, 171.1, 157.5, 152.0, 139.0, 135.8, 132.3, 131.2, 131.0, 130.5, 129.1, 127.9, 126.2, 126.0, 119.5, 112.4, 111.1, 80.8, 70.7, 70.6, 70.5, 70.4, 70.4, 70.3, 69.5, 68.1, 66.9, 47.7, 41.9, 39.1, 36.3, 28.1, 25.7, 23.4 ppm.</div></div><div id="sec4_4_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Synthesis of <i>tert</i>-Butyl 1-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-3,6,9,12,15-pentaoxaoctadecan-18-oate (<b>16e</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 1-hydroxy-3,6,9,12,15-pentaoxaoctadecan-18-oate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (39 mg). MALDI: (calculated) [M + H<sup>±</sup>] 764.31 g/mol, (found) [M + H<sup>±</sup>] 764.36 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.92 (t, <sup>3</sup><i>J</i> = 6.09 Hz, 1H), 7.53–7.49 (m, 1H), 7.44 (d, <sup>4</sup><i>J</i> = 1.95 Hz, 1H), 7.42–7.37 (m, 1H), 7.29 (d, <sup>3</sup><i>J</i> = 8.24 Hz, 1H), 7.18–7.15 (m, 2H), 6.81–6.77 (m, 1H), 4.40 (d, <sup>3</sup><i>J</i> = 6.09 Hz, 2H), 4.14 (t, <sup>3</sup><i>J</i> = 7.72 Hz, 2H), 4.12–4.08 (m, 2H), 3.84–3.81 (m, 2H), 3.75–3.53 (m, 20H), 3.21–3.15 (m, 2H), 2.74–2.67 (m, 2H), 2.46 (t, <sup>3</sup><i>J</i> = 6.46 Hz, 2H), 1.43–1.41 (m, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 171.1, 171.0, 157.3, 152.0, 139.1, 136.2, 132.1, 131.2, 130.7, 130.4, 129.1, 128.3, 125.8, 125.8, 119.7, 112.2, 110.9, 80.7, 70.8, 70.7, 70.7, 70.6, 70.5, 70.5, 70.5, 70.3, 69.5, 68.1, 66.9, 47.6, 42.0, 38.7, 36.3, 28.2, 25.7, 23.4 ppm.</div></div><div id="sec4_4_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Synthesis of <i>tert</i>-Butyl 1-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-3,6,9,12,15,18-hexaoxahenicosan-21-oate (<b>16f</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 1-hydroxy-3,6,9,12,15,18-hexaoxahenicosan-21-oate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (36 mg). MALDI: (calculated) [M + H<sup>±</sup>] 808.34 g/mol, (found) [M + H<sup>±</sup>] 808.39 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.91 (t, <sup>3</sup><i>J</i> = 5.71 Hz, 1H), 7.51–7.48 (m, 1H), 7.43 (d, <sup>4</sup><i>J</i> = 1.86 Hz, 1H), 7.42–7.38 (m, 1H), 7.32–7.28 (m, 1H), 7.17 (s, 1H), 7.15 (dd, (m, <sup>3</sup><i>J</i> = 8.23 Hz, <sup>4</sup><i>J</i> = 1.86 Hz, 1H)), 6.82–6.77 (m, 1H), 4.41 (d, <sup>3</sup><i>J</i> = 5.71 Hz, 2H), 4.16 (t, <sup>3</sup><i>J</i> = 6.88 Hz, 2H), 4.13–4.08 (m, 2H), 3.84–3.81 (m, 2H), 3.75–3.53 (m, 24H), 3.23–3.15 (m, 2H), 2.7–2.69 (m, 2H), 2.47 (t, <sup>3</sup><i>J</i> = 6.70 Hz, 2H), 1.44–1.38 (m, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>) δ = 171.2, 171.1, 157.4, 152.0, 139.1, 136.1, 132.2, 131.2, 130.8, 130.4, 129.1, 128.2, 126.0, 125.9, 119.6, 112.3, 111.0, 80.7, 70.7, 70.7, 70.7, 70.7, 70.6, 70.5, 70.5, 70.4, 70.4, 70.3, 69.5, 68.1, 66.9, 47.6, 42.0, 38.8, 36.3, 28.2, 25.7, 23.5 ppm.</div></div><div id="sec4_4_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Synthesis of <i>tert</i>-Butyl 1-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5<i>H</i>-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-3,6,9,12,15,18,21-heptaoxatetracosan-24-oate (<b>16g</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using <i>tert</i>-butyl 1-hydroxy-3,6,9,12,15,18,21-heptaoxatetracosan-24-oate as the commercial available starting material, obtaining the product as a TFA salt with an unknown stoichiometry as a colorless oil (59 mg). MALDI: (calculated) [M + H<sup>±</sup>] 852.36 g/mol, (found) [M + H<sup>±</sup>] 852.42 g/mol. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.98–7.89 (m, 1H), 7.50–7.46 (m, 1H), 7.43–7.41 (m, 1H), 7.40–7.36 (m, 1H), 7.30–7.27 (m, 1H), 7.17 (s, 1H), 7.16–7.12 (m, 1H), 6.80–6.76 (m, 1H), 4.40 (d, <sup>3</sup><i>J</i> = 5.38 Hz, 2H), 4.17–4.12 (m, 2H), 4.11–4.07 (m, 2H), 3.83–3.79 (m, 2H), 3.74–3.51 (m, 28H), 3.21–3.14 (m, 2H), 2.75–2.67 (m, 2H), 2.48–2.44 (m 2H), 1.42–1.40 (m, 9H) ppm. <sup>13</sup>C NMR (125.8 MHz, CDCl<sub>3</sub>): δ = 171.1, 171.0, 157.4, 152.0, 139.0, 136.1, 132.1, 131.2, 130.7, 130.4, 129.1, 128.2, 125.9, 125.8, 119.6, 112.2, 111.0, 80.7, 70.7, 70.7, 70.7, 70.6, 70.5, 70.5, 70.5, 70.5, 70.5, 70.4, 70.4, 70.3, 69.5, 68.0, 66.9, 47.6, 42.0, 38.8, 36.3, 28.1, 25.6, 23.4 ppm.</div></div><div id="sec4_4_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(3-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)propanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17a</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16a</b>, yielding in 17 mg, 18 μmol, 40% of a white solid. MALDI: (calculated) [M + 2H<sup>±</sup>] 945.34 g/mol, (found) [M + 2H<sup>±</sup>] 945.67 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 966.3153 g/mol, (found) [M + Na<sup>±</sup>] 966.3156 g/mol. HPLC: R<sub>t</sub> = 12.3 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.70 Hz, 1H), 8.42 (t, <sup>3</sup><i>J</i> = 5.85 Hz, 1H), 7.95 (d, <sup>3</sup><i>J</i> = 9.40 Hz, 1H), 7.58 (d, <sup>2</sup><i>J</i> = 2.11 Hz, 1H), 7.56 (d, <sup>3</sup><i>J</i> = 8.33 Hz, 1H), 7.53 (dd, <sup>3</sup><i>J</i> = 8.54 Hz, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.43 (d, <sup>4</sup><i>J</i> = 2.10 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.40 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.40 Hz, 2H), 7.32 (dd, <sup>3</sup><i>J</i> = 8.33 Hz, <sup>4</sup><i>J</i> = 2.11 Hz, 1H), 7.25 (s, 1H, H-3e), 6.94 (d, <sup>3</sup><i>J</i> = 8.54 Hz, 1H), 5.14 (bs, 1H), 4.56 (d, <sup>3</sup><i>J</i> = 9.40 Hz, 1H), 4.46–4.39 (m, 2H), 4.35 (br, s, 1H), 4.26 (d, <sup>3</sup><i>J</i> = 5.85 Hz, 2H), 4.22 (dd, <sup>2</sup><i>J</i> = 15.86 Hz, <sup>3</sup><i>J</i> = 5.70 Hz, 1H), 4.09 Hz, (t, <sup>3</sup><i>J</i> = 4.64 Hz, 2H), 3.98 (t, <sup>3</sup><i>J</i> = 7.16 Hz, 2H), 3.75–3.61 (m, 6H), 3.56 (s, 2H), 2.78 (t, <sup>3</sup><i>J</i> = 7.04 Hz, 2H), 2.62–2.55 (m, 1H), 2.50 (m, 2H), 2.44 (s, 3H), 2.42–2.36 (m, 1H), 2.07–2.01 (m, 1H), 1.94–1.87 (m, 1H), 0.92 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.3, 154.4, 154.0, 151.4, 147.7, 144.6, 139.5, 137.7, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.8, 128.6, 127.8, 127.4, 126.8, 123.6, 111.9, 109.7, 68.9, 68.6, 67.7, 67.2, 58.7, 56.3, 56.3, 44.3, 41.6, 41.2, 37.9, 37.5, 35.7, 35.3, 26.3, 25.6, 22.5, 15.9 ppm.</div></div><div id="sec4_4_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(3-(2-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)ethoxy)propanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17b</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16b</b>, yielding in 30 mg, 30 μmol, 48% of a white solid. MALDI: (calculated) [M + 2H<sup>±</sup>] 989.36 g/mol, (found) [M + 2H<sup>±</sup>] 989.89 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1010.3415 g/mol, (found) [M + Na<sup>±</sup>] 1010.3442 g/mol. HPLC: R<sub>t</sub> = 12.3 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.82 Hz, 1H), 8.41 (t, <sup>3</sup><i>J</i> = 5.64 Hz, 1H), 7.91 (d, <sup>3</sup><i>J</i> = 9.38 Hz, 1H), 7.58 (d, <sup>4</sup><i>J</i> = 2.04 Hz, 1H), 7.57 (d, <sup>3</sup><i>J</i> = 8.31 Hz, 1H), 7.54 (dd, <sup>3</sup><i>J</i> = 8.48 Hz, <sup>4</sup><i>J</i> = 2.25 Hz, 1H), 7.43 (d, <sup>4</sup><i>J</i> = 2.25 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.37 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.37 Hz, 2H), 7.32 (dd, <sup>3</sup><i>J</i> = 8.31 Hz, <sup>4</sup><i>J</i> = 2.04 Hz, 1H), 7.23 (s, 1H), 6.94 (d, <sup>3</sup><i>J</i> = 8.48 Hz, 1H), 5.14 (br, s, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.38 Hz, 1H), 4.46–4.40 (m, 2H), 4.35 (bs, 1H), 4.26 (d, <sup>3</sup><i>J</i> = 5.82 Hz, 2H), 4.22 (dd, <sup>2</sup><i>J</i> = 15.89 Hz, <sup>3</sup><i>J</i> = 5.64 Hz, 1H), 4.09 Hz, (t, <sup>3</sup><i>J</i> = 4.56 Hz, 2H), 3.97 (t, <sup>3</sup><i>J</i> = 7.02 Hz, 2H), 3.72 (t, <sup>3</sup><i>J</i> = 4.56 Hz, 2H), 3.69–3.46 (m, 10H), 2.78 (t, <sup>3</sup><i>J</i> = 6.62 Hz, 2H), 2.62–2.55 (m, 1H), 2.50 (m, 2H), 2.44 (s, 3H), 2.40–2.32 (m, 1H), 2.06–2.00 (m, 1H), 1.94–1.87 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.3, 154.4, 154.0, 151.4, 147.7, 144.7, 139.5, 137.7, 131.14, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.6, 127.9, 127.4, 126.8, 123.6, 111.9, 109.7, 69.9, 69.5, 69.0, 68.8, 67.7, 66.9, 59.7, 58.7, 56.3, 56.3, 44.3, 41.6, 41.2, 37.9, 37.5, 35.6, 35.3, 26.3, 25.6, 22.4, 15.9 ppm.</div></div><div id="sec4_4_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-14-(<i>tert</i>-Butyl)-1-(2-((2-(3,4-dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-12-oxo-3,6,9-trioxa-13-azapentadecan-15-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17c</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16c</b>, yielding in 24 mg, 23.4 μmol, 52% of a white solid. MALDI: (calculated) [M + 2H<sup>±</sup>] 1033.40 g/mol, (found) [M + 2H<sup>±</sup>] 1033.98 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1054.3677 g/mol, (found) [M + Na<sup>±</sup>] 1036.3689 g/mol. HPLC: R<sub>t</sub> = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.97 (s, 1H), 8.57 (t, <sup>3</sup><i>J</i> = 5.60 Hz, 1H), 8.42 (t, <sup>3</sup><i>J</i> = 5.80 Hz, 1H), 7.90 (d, <sup>3</sup><i>J</i> = 9.42 Hz, 1H), 7.58 (d, <sup>4</sup><i>J</i> = 2.08, 1H), 7.57 (d, <sup>3</sup><i>J</i> = 8.33 Hz, 1H), 7.54 (dd, <sup>3</sup><i>J</i> = 8.48 Hz, <sup>4</sup><i>J</i> = 2.20 Hz, 1H), 7.43 (d, <sup>4</sup><i>J</i> = 2.20 Hz, 1H), 7.41 (d, <sup>3</sup><i>J</i> = 8.39 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.39 Hz, 2H), 7.32 (dd, <sup>3</sup><i>J</i> = 8.33 Hz, <sup>4</sup><i>J</i> = 2.08 Hz, 1H), 7.24 (s, 1H), 6.95 (d, <sup>3</sup><i>J</i> = 8.48 Hz, 1H), 5.15 (br, s, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.42 Hz, 1H), 4.46–4.40 (m, 2H), 4.35 (br, s, 1H), 4.26 (d, <sup>3</sup><i>J</i> = 5.60 Hz, 2H), 4.22 (dd, <sup>2</sup><i>J</i> = 15.99 Hz, <sup>3</sup><i>J</i> = 5.80 Hz, 1H), 4.11–4.08 Hz, (m, 2H), 3.97 (t, <sup>3</sup><i>J</i> = 6.93 Hz, 2H), 3.74–3.70 (m, 2H), 3.69–3.42 (m, 14H), 2.77 (t, <sup>3</sup><i>J</i> = 6.93 Hz, 2H), 2.57–2.53 (m, 1H), 2.50 (m, 2H), 2.44 (s, 3H), 2.38–2.30 (m, 1H), 2.07–2.01 (m, 1H), 1.94–1.87 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.3, 154.5, 154.0, 151.4, 147.7, 144.6, 139.5, 137.7, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.1, 128.8, 128.6, 127.8, 127.4, 126.8, 123.6, 112.0, 109.8, 70.0, 69.8, 69.72, 69.46, 69.0, 68.9, 67.8, 66.9, 58.7, 56.3, 56.3, 44.3, 41.6, 41.2, 37.9, 37.6, 35.7, 35.3, 26.3, 25.6, 22.5, 15.9 ppm.</div></div><div id="sec4_4_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-17-(<i>tert</i>-Butyl)-1-(2-((2-(3,4-dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-15-oxo-3,6,9,12-tetraoxa-16-azaoctadecan-18-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17d</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16d</b>, yielding in 6.9 mg, 6.4 μmol, 12% of a yellow oil. MALDI: (calculated) [M + 2H<sup>±</sup>] 1076.41 g/mol, (found) [M + 2H<sup>±</sup>] 1076.10 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1098.3939 g/mol, (found) [M + Na<sup>±</sup>] 1098.3954 g/mol. HPLC: R<sub>t</sub> = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.97 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.55 Hz, 1H), 8.40 (t, <sup>3</sup><i>J</i> = 5.76 Hz, 1H), 7.90 (d, <sup>3</sup><i>J</i> = 9.38 Hz, 1H), 7.58–7.55 (m, 2H), 7.54 (dd, <sup>3</sup><i>J</i> = 8.48 Hz, <sup>4</sup><i>J</i> = 2.07 Hz, 1H), 7.43 (d, <sup>4</sup><i>J</i> = 2.07 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.31 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.31 Hz, 2H), 7.32 (dd, <sup>3</sup><i>J</i> = 8.17 Hz, <sup>4</sup><i>J</i> = 1.93 Hz, 1H), 7.24 (s, 1H), 6.95 (d, <sup>3</sup><i>J</i> = 8.48 Hz, 1H), 5.13 (br, s, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.38 Hz, 1H), 4.46–4.39 (m, 2H), 4.35 (br, s, 1H), 4.26 (d, <sup>3</sup><i>J</i> = 5.55 Hz, 2H), 4.22 (dd, <sup>2</sup><i>J</i> = 15.87 Hz, <sup>3</sup><i>J</i> = 5.76 Hz, 1H), 4.12–4.08 Hz, (m, 2H), 3.97 (t, <sup>3</sup><i>J</i> = 7.02 Hz, 2H), 3.75–3.71 (m, 2H), 3.69–3.42 (m, 18H), 2.76 (t, <sup>3</sup><i>J</i> = 7.29 Hz, 2H), 2.56–2.50 (m, 1H), 2.50 (m, 2H), 2.44 (s, 3H), 2.37–2.30 (m, 1H), 2.07–2.00 (m, 1H), 1.94–1.87 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.3, 154.5, 154.0, 151.4, 147.7, 144.6, 139.5, 137.7, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.8, 128.6, 127.8, 127.4, 126.8, 123.6, 112.0, 109.8, 70.0, 69.8, 69.7, 6978, 69.45, 69.0, 68.8, 67.8, 66.9, 58.7, 56.3, 56.3, 44.3, 41.6, 41.2, 37.9, 37.6, 35.6, 35.3, 26.3, 25.6, 22.5, 15.9 ppm.</div></div><div id="sec4_4_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(2-((2-(3,4-dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-18-oxo-3,6,9,12,15-pentaoxa-19-azahenicosan-21-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17e</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16e</b>, yielding in 17 mg, 15.2 μmol, 36% of a yellow oil. MALDI: (calculated) [M + 3H<sup>±</sup>] 1122.45 g/mol, (found) [M + 3H<sup>±</sup>] 1122.14 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1142.4202 g/mol, (found) [M + Na<sup>±</sup>] 1142.4229 g/mol. HPLC: R<sub>t</sub> = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.98 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 5.98 Hz, 1H), 8.40 (t, <sup>3</sup><i>J</i> = 5.68 Hz, 1H), 7.90 (d, <sup>3</sup><i>J</i> = 9.41 Hz, 1H), 7.82 (s, 1H), 7.59 (dd, 1H, <sup>3</sup><i>J</i> = 8.55, <sup>4</sup><i>J</i> = 2.05, 1H), 7.56 (d, <sup>3</sup><i>J</i> = 8.00 Hz, 1H), 7.55 (s, 1H), 7.45 (d, <sup>4</sup><i>J</i> = 2.05 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.32 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.32 Hz, 2H), 7.28 (dd, <sup>3</sup><i>J</i> = 8.00 Hz, <sup>4</sup><i>J</i> = 1.95 Hz, 1H), 7.15 (d, <sup>3</sup><i>J</i> = 8.55 Hz, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.41 Hz, 1H), 4.46–4.40 (m, 2H), 4.37–4.33 (m, 1H), 4.28 (d, <sup>3</sup><i>J</i> = 5.68 Hz, 2H), 4.25–4.19 (m, 3H), 4.18 (t, <sup>3</sup><i>J</i> = 4.42 Hz, 2H), 3.78–3.74 (m, 2H), 3.70–3.41 (m, 22H), 3.18 (t, <sup>3</sup><i>J</i> = 7.53 Hz, 2H), 2.73–2.65 (m, 2H), 2.57–2.51 (m, 1H), 2.44 (s, 3H), 2.38–2.31 (m, 1H), 2.07–2.01 (m, 1H), 1.94–1.88 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.2, 154.5, 154.0, 151.4, 147.70, 139.5, 137.68, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.8, 128.6, 128.0, 127.41, 126.81, 123.7, 123.6, 112.0, 109.8, 70.0, 69.8, 69.7, 69.7, 69.4, 69.0, 68.8, 67.8, 66.9, 58.7, 56.3, 44.4, 41.6, 41.2, 37.9, 37.6, 35.6, 35.3, 26.3, 22.5, 15.9 ppm.</div></div><div id="sec4_4_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-23-(<i>tert</i>-Butyl)-1-(2-((2-(3,4-dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-21-oxo-3,6,9,12,15,18-hexaoxa-22-azatetracosan-24-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17f</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16f</b>, yielding in 6.2 mg, 5.3 μmol, 27% of a yellow oil. MALDI: (calculated) [M + 3H<sup>±</sup>] 1166.48 g/mol, (found) [M + 3H<sup>±</sup>] 1166.20 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1186.4464 g/mol, (found) [M + Na<sup>±</sup>] 1186.4497 g/mol. HPLC: R<sub>t</sub> = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, water supp.) δ = 8.98 (s, 1H), 8.55 (t, <sup>3</sup><i>J</i> = 5.69 Hz, 1H), 8.38 (t, <sup>3</sup><i>J</i> = 5.52 Hz, 1H), 7.90 (d, <sup>3</sup><i>J</i> = 9.22 Hz, 1H), 7.82 (s, 1H), 7.61–7.53 (m, 3H), 7.44 (d, <sup>4</sup><i>J</i> = 1.64 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.27 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.27 Hz, 2H), 7.28 (dd, <sup>3</sup><i>J</i> = 8.18 Hz, <sup>4</sup><i>J</i> = 1.64 Hz, 1H), 7.15 (d, <sup>3</sup><i>J</i> = 8.62 Hz, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.22 Hz, 1H), 4.47–4.39 (m, 2H, H-10a), 4.35 (br, s, 1H), 4.27 (d, <sup>3</sup><i>J</i> = 5.52 Hz, 2H), 4.26–4.20 (m, 3H), 4.18 (t, <sup>3</sup><i>J</i> = 4.19 Hz, 2H), 3.76 (t, <sup>3</sup><i>J</i> = 4.19 Hz, 2H), 3.70–3.42 (m, 26H), 3.18 (t, <sup>3</sup><i>J</i> = 7.58 Hz, 2H), 2.74–2.64 (m, 2H), 2.58–2.50 (m, 1H), 2.44 (s, 3H), 2.39–2.30 (m, 1H), 2.06–1.99 (m, 1H), 1.95–1.86 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.2, 154.5, 154.0, 151.4, 147.7, 139.5, 137.7, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.8, 128.6, 128.0, 127.4, 126.8, 123.7, 123.6, 112.0, 109.8, 70.0, 69.8, 69.7, 69.7, 69.4, 69.0, 68.8, 67.8, 66.9, 58.7, 56.3, 44.4, 41.6, 41.2, 37.9, 37.5, 35.6, 35.3, 26.3, 25.6, 22.5, 15.9 ppm.</div></div><div id="sec4_4_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Synthesis of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-26-(<i>tert</i>-Butyl)-1-(2-((2-(3,4-dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)-24-oxo-3,6,9,12,15,18,21-heptaoxa-25-azaheptacosan-27-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17g</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B using intermediate <b>16g</b>, yielding in 18 mg, 15 μmol, 36% of a yellow oil. MALDI: (calculated) [M + 3H<sup>±</sup>] 1210.50 g/mol, (found) [M + 3H<sup>±</sup>] 1210.16 g/mol. HRMS: (calculated) [M + Na<sup>±</sup>] 1230.4726 g/mol, (found) [M + Na<sup>±</sup>] 1230.4747 g/mol. HPLC: R<sub>t</sub> = 12.4 min (254 nm, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, water supp.) δ = 8.98 (s, 1H), 8.56 (t, <sup>3</sup><i>J</i> = 6.03 Hz, 1H), 8.39 (t, <sup>3</sup><i>J</i> = 5.59 Hz, 1H), 7.90 (d, <sup>3</sup><i>J</i> = 9.34 Hz, 1H), 7.82 (s, 1H), 7.59 (dd, <sup>3</sup><i>J</i> = 8.50 Hz, <sup>4</sup><i>J</i> = 2.15 Hz, 1H), 7.57–7.54 (m, 2H), 7.45 (d, <sup>4</sup><i>J</i> = 1.83 Hz, 1H), 7.42 (d, <sup>3</sup><i>J</i> = 8.15 Hz, 2H), 7.38 (d, <sup>3</sup><i>J</i> = 8.29 Hz, 2H), 7.28 (dd, <sup>3</sup><i>J</i> = 8.18 Hz, <sup>4</sup><i>J</i> = 1.89 Hz, 1H), 7.15 (d, <sup>3</sup><i>J</i> = 8.63 Hz, 1H), 4.55 (d, <sup>3</sup><i>J</i> = 9.15 Hz, 1H), 4.47–4.39 (m, 2H), 4.38–4.33 (m, 1H), 4.28 (d, <sup>3</sup><i>J</i> = 5.38 Hz, 2H), 4.26–4.20 (m, 3H), 4.18 (t, <sup>3</sup><i>J</i> = 4.71 Hz, 2H), 3.76 (t, <sup>3</sup><i>J</i> = 4.46 Hz, 2H), 3.73–3.43 (m, 30H), 3.18 (t, <sup>3</sup><i>J</i> = 7.78 Hz, 2H), 2.73–2.64 (m, 2H), 2.58–2.50 (m, 1H), 2.44 (s, 3H), 2.39–2.30 (m, 1H), 2.08–2.00 (m, 1H), 1.95–1.86 (m, 1H), 0.93 (s, 9H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO) δ = 171.9, 169.9, 169.5, 169.3, 154.5, 154.02, 151.4, 147.7, 144.6, 139.5, 137.7, 131.1, 131.0, 130.7, 130.3, 129.6, 129.5, 129.0, 128.6, 127.9, 127.4, 126.8, 123.6, 123.6, 112.0, 109.8, 70.0, 69.8, 69.7, 69.7, 69.5, 69.0, 68.9, 67.8, 66.9, 58.7, 56.3, 56.3, 44.3, 41.6, 41.2, 37.9, 37.6, 35.7, 35.3, 26.3, 25.6, 22.5, 15.9 ppm.</div></div><div id="sec4_4_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Synthesis of (2<i>S</i>,4<i>S</i>)-1-((<i>S</i>)-2-(3-(2-(2-(2-((2-(3,4-Dichlorophenyl)acetamido)methyl)-4-(6,7-dihydro-5H-pyrrolo[1,2-<i>a</i>]imidazol-2-yl)phenoxy)ethoxy)ethoxy)propanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>21</b>)</h4><div class="NLM_p last">The reaction was carried out as described in General Procedure B, yielding in 17 mg, 17.2 μmol, 39% of a yellow solid. MALDI: (calculated) [M + H<sup>+</sup>] 988.36 g/mol, (found) [M + H<sup>+</sup>] 988.31 g/mol. HRMS: (calculated) [M + H<sup>+</sup>] 988.3596 g/mol, (found) [M + H<sup>+</sup>] 988.3242 g/mol. HPLC: R<sub>t</sub> = 11.8 min (254 nm, 100%). <sup>1</sup>H NMR (500 MHz, DMSO) δ = 8.98 (s, 1H), 8.44–8.37 (m, 2H), 7.81 (s, 1H), 7.59 (dd, <sup>3</sup><i>J</i> = 8.5 Hz, <sup>4</sup><i>J</i> = 2.3 Hz, 1H), 7.58–7.54 (m, 2H), 7.46–7.39 (m, 3H), 7.38–7.34 (m, 2H), 7.28 (dd, <sup>3</sup><i>J</i> = 8.2 Hz, <sup>4</sup><i>J</i> = 1.9 Hz, 1H), 7.16 (d, <sup>3</sup><i>J</i> = 8.7 Hz, 1H), 4.54 (t, <sup>3</sup><i>J</i> = 7.5 Hz, 1H), 4.51–4.42 (m, 1H), 4.39–4.30 (m, 4H), 4.28–4.26 (m 3H), 4.25–4.21 (m, 3H), 4.20–4.14 (m, 3H), 3.77–3.73 (m, 2H), 3.64–3.52 (m, 8H), 3.51–3.48 (m, 2H), 3.38–3.36 (m, 1H), 3.18 (t, <sup>3</sup><i>J</i> = 7.5 Hz, 2H), 2.74–2.65 (m, 2H), 2.44 (s, 3H), 2.31–2.28 (m, 1H), 2.24–2.20 (m, 1H), 2.07–2.01 (m, 1H), 1.97–1.87 (m, 1H), 1.81–1.77 (m, 1H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO) δ = 172.2, 170.6, 169.3, 168.7, 154.4, 154.0, 151.4, 147.8, 144.7, 139.2, 137.7, 131.1, 131.0, 130.7, 130.4, 129.8, 129.5, 129.1, 128.7, 128.6, 127.9, 127.4, 126.8, 123.6, 112.0, 109.8, 82.0, 69.9, 69.7, 69.3, 69.0, 68.7, 67.8, 66.2, 58.8, 56.5, 54.7, 44.3, 41.7, 41.2, 37.5, 36.9, 34.3, 25.6, 22.5, 15.9 ppm.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Protein Purification</h3><div class="NLM_p last">WDR5A was expressed and purified as described elsewhere.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Plasmids of WDR5A (aa 1-334 and aa 33-334) were a kind gift of M. Vedadi from the SGC Toronto. Briefly, WDR5 was overexpressed in <i>Escherichia coli</i> BL21 using TB media. Protein expression was induced by addition of 0.5 mM IPTG. Cells were grown overnight at 18 °C. Next morning, the cells were harvested and resuspended in Lysis buffer (50 mM HEPES buffer, pH 7.5, 500 mM NaCl, 20 mM imidazole, 0.5 mM TCEP, and 5% glycerol). For purification, the cells were lysed by sonication. After centrifugation, the supernatant was loaded onto a Nickel–Sepharose column equilibrated with 30 mL of lysis buffer. The column was washed with 100 mL of lysis buffer. WDR5 was eluted by an imidazole step gradient (50, 100, 200, 300 mM). Fractions containing WDR5 were pooled together, concentrated, and loaded onto a Superdex 200 16/60 HiLoad gel filtration column equilibrated with a final buffer (25 mM HEPES pH 7.5, 300 mM NaCl, and 0.5 mM TCEP). The protein was concentrated to approx. 400 μM. The buffer was kept and used for ITC experiments.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Differential Scanning Fluorimetry</h3><div class="NLM_p last">Ligand binding to protein was detected using DSF on an MX3005P qPCR system from Agilent Technologies as described elsewhere.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Briefly, protein was buffered in 25 mM HEPES (pH 7.5), 500 mM NaCl, and 0.5 mM TCEP and diluted to a final concentration of 2 μM, and the fluorescent dye SYPRO Orange was added at a dilution of 1:1000. Compounds were dissolved in DMSO (10 mM) and added at a final concentration of 10 μM to 20 μL of a protein–dye mix in a 96-well plate. Real-time melting curves were then recorded by heating the samples from 25 to 96 °C in 71 cycles (heating rate of 270 K/h, excitation/emission filters = 492/610 nm), and the melting point, <i>T</i><sub>m</sub>, was calculated using the Boltzmann equation.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Binding constant (<i>K</i><sub>d</sub>), stoichiometry (<i>n</i>), and thermodynamic binding parameters (Δ<i>H</i>, Δ<i>S</i>, and Δ<i>G</i>) of ligand–protein interactions were determined on a nano-ITC from TA Instruments as described elsewhere.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Briefly, compounds were diluted to a final concentration of 10–25 μM in buffer and placed into the sample cell. Proteins (80–120 μM) were added using an initial injection of 3 or 4 μL, followed by 12–30 injections of 6 or 8 μL at 22 °C. Collected data were corrected by subtraction of pure DMSO injection heats. Data were analyzed by assuming a sigmoidal dose–response relationship (four parameters). Errors of fits were calculated using standard deviation and a confidence interval of 68% in GraphPad Prism.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> NanoBRET</h3><div class="NLM_p last">The NanoBRET assay was performed as described previously.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Briefly, full-length WDR5 was cloned in frame as the N- or C-terminal NanoLuc-fusion pNLF1 vector using ligation-independent in-fusion cloning (Takara Bio) and sequence-verified. Plasmids were transfected into HEK293T cells using FuGENE HD (Promega, E2312), and proteins were allowed to express for 20 h. Serially diluted inhibitor and Tracer molecule <b>19c</b> at a concentration of 1 μM determined previously as the Tracer <b>19c</b><i>K</i><sub>D,app</sub> were pipetted into white 384-well plates (Greiner 781 207) using an Echo acoustic dispenser (Labcyte). The corresponding protein-transfected cells were added and reseeded at a density of 2 × 10<sup>5</sup> cells/mL after trypsinization and resuspending in Opti-MEM without phenol red (Life Technologies). The system was allowed to equilibrate for 2 h at 37 °C/5% CO<sub>2</sub> prior to BRET measurements. To measure BRET, NanoBRET Nano-Glo Substrate and Extracellular NanoLuc Inhibitor (Promega, N2540) was added as per the manufacturer’s protocol, and filtered luminescence was measured on a PHERAstar plate reader (BMG Labtech) equipped with a luminescence filter pair (450 nm BP filter (donor) and 610 nm LP filter (acceptor)). Competitive displacement data were then graphed using GraphPad Prism 8 software using a normalized 3-parameter curve fit.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Cell Culture</h3><div class="NLM_p last">Human MV4-11 (male) and human HL-60 (female) cells were cultured in RPMI-1640 medium, whereas human HEK293 (female) cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM) medium at 37 °C in 5% CO<sub>2</sub>. Both media were supplemented with 10% fetal bovine serum (FBS) and 1% penicillin/streptomycin solution.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Cloning</h3><div class="NLM_p last">WDR5-HiBiT was cloned by PCR amplification of the vector containing full-length WDR5 using the forward primer: CGCACCGGTATGGCGACGGAGGAGAAGAAGC and the reverse primer: CGCGACGCGTTTAGCTAATCTTCTTGAACAGCCGCCAGCCGCTCACACCGGAGCTCCCGCAGTCACTCTTCCACAGT. The PCR product was inserted into the pRRL-PGK vector using <i>Age</i>I/<i>Mlu</i>I restriction sites. HA-tagged VHL was cloned by PCR amplification of cDNA from MV4-11 cells as the template (forward primer: CGCACCGGTATGTACCCTTACGACGTGCCCGACTACGCCGGGAGCTCCGGTCCCCGGAGGGCGGAGAAC and reverse primer: GGACTAGTTCAATCTCCCATCCGTTGATGTG) and inserted into the pRRL-SFFV vector using <i>Age</i>I/<i>Spe</i>I sites. The sequence of the cloned VHL was identified as isoform 1 by sanger sequencing.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Cell-Line Generation</h3><div class="NLM_p last">Lentiviral infection was used to generate stable MV4-11<sup>WDR5-HiBiT</sup>, MV4-11<sup>VHL</sup>, and MV4-11<sup>WDR5-HiBiT/VHL</sup> cells. Lentivirus was produced using plasmids psPAX2, pMD2.G, and HiBiT-WDR5 or the HA-VHL plasmid in HEK293 cells. MV4-11 cells were infected with a filtered virus supernatant and selected after 72 h of infection for generation of MV4-11<sup>WDR5-HiBiT</sup> and MV4-11<sup>VHL</sup> cells. MV4-11<sup>WDR5-HiBiT</sup> cells were used to prepare MV4-11<sup>WDR5-HiBiT/VHL</sup> cells.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> HiBiT Assay</h3><div class="NLM_p last">The HiBiT assay was performed as described previously.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> MV4-11<sup>WDR5-HiBiT</sup> or MV4-11<sup>WDR5-HiBiT/VHL</sup> cells were seeded and treated with serial dilutions of compounds for 6 or 24 h. The Nano-Glo HiBiT Lytic Detection System (Promega) was used for the assay. Luminescence was measured on a GloMax 96 Microplate Luminometer (Promega). DC<sub>50</sub> was calculated using lower concentrations (showing sigmoidal behavior) with the dose–response (four parameters) equation.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Immunoblotting</h3><div class="NLM_p last">After the treatment, cells were lysed in RIPA lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium dodecyl sulfate (SDS), 0.1% sodium deoxycholate) supplemented with protease and phosphatase inhibitors (Sigma) for 20 min at 4 °C head-over-tail. Supernatants were collected after centrifugation. The bicinchoninic acid (BCA) assay was used for protein quantification. Per sample equal amounts of protein were separated by Bis-Tris-PAGE and transferred to poly(vinylidene fluoride) (PVDF) membranes (Millipore). The membranes were incubated with 5% (w/v) nonfat dry milk in TBS-T (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 0.1% (v/v) Tween-20) for 1 h at room temperature for blocking and then incubated with the primary antibody overnight at 4 °C. For visualization, horseradish peroxidase (HRP)-labeled secondary antibodies were used and detected using the chemiluminescent HRP substrate (Millipore) in LAS4000 Mini (Fuji). The signal was quantified using ImageJ (version 1.53g) or Image Studio Lite (LI-COR Biosciences, version 5.2.5). Vinculin was used as a loading control. Antibodies used in this study were WDR5 (Santa Cruz Biotechnology; sc-393080), HA (Santa Cruz Biotechnology; sc-805), VHL (Santa Cruz Biotechnology; sc-135657), and vinculin (Sigma; V9131).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Cycloheximide Chase Assay</h3><div class="NLM_p last">The cycloheximide chase assay was performed as described previously.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> MV4-11 cells were treated with 50 μg/mL CHX with or without PROTACs for different time points. The cells were harvested in RIPA buffer and probed for immunoblotting. The intensity of the WDR5 band at 0 h was set as 1. The mean ± SD from <i>n</i> = 2 biological experiments was plotted as log<sub>10</sub> values.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> RT-qPCR</h3><div class="NLM_p last">RT-qPCR was performed as described previously.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Total RNA was extracted using peqGOLD TriFast (Peqlab). cDNA synthesis was carried out, and cDNA were analyzed by qPCR on a StepOnePlus Real-Time PCR System (Thermo Fisher Scientific) using the SYBR Green Master Mix (Thermo Fisher Scientific). Equal amounts of cDNA and SYBR Green Master Mix were added along with WDR5 primers (forward: CCAGTCTCGGCCGTTCATTT and reverse: CGTTCGGGGAGAACTTCACA). For analysis, expression was normalized to the β2-microglobulin expression. qPCR was done in technical triplicates.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Cell Proliferation Assay</h3><div class="NLM_p last">MV4-11 or MV4-11<sup>VHL</sup> cells were seeded at a density of 1 × 10<sup>5</sup> cells per mL and treated with compounds. Cells were counted every third day and reseeded to the original density of 1 × 10<sup>5</sup> cells per mL in fresh media with compounds. The mean cumulative cell number ± SEM (<i>n</i> = 2 biological experiments) was plotted as a log<sub>10</sub> value over the course of time. <i>P</i>-values were calculated from the cumulative cell number from the endpoint using a two-tailed unpaired t-test assuming equal variance against DMSO-treated cells.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Quantitative Proteomics</h3><div class="NLM_p">In total, 4 million MV4-11 cells (in 10 mL) were seeded at least in triplicates for each treatment on the evening before the treatment. Cells were treated with 1 μM <b>6</b>, 1 μM <b>8g</b>, 5 μM <b>17b</b>, 5 μM 1<b>4</b>, or DMSO as a control for 9 h. After the treatment, cells were washed twice with ice-cold phosphate-buffered saline (PBS) supplemented with protease and phosphatase inhibitor and lysed in SDS lysis buffer (2% SDS in 40 mM Tris-HCl, pH 7.6). To reduce viscosity, the sample was sonicated using a sonication water bath (10 cycles, 15 s sonication, 15 s pause on ice), boiled at 95 °C for 10 min, and trifluoroacetic acid was added to a final concentration of 1%. To neutralize the sample (final pH 7.6–8.0), 300 mM <i>N</i>-methylmorpholin was added to a final concentration of 2%. The protein concentration in the cell lysate was determined using the Pierce BCA Protein Assay Kit (ThermoScientific) according to the protocol of the manufacturer. The bead suspension for the sp3 sample workup was prepared by mixing magnetic SeraMag-A and SeraMag-B beads (10 μL per sample of each type; Cytiva) in a ratio of 1:1, washing them three times with ddH<sub>2</sub>O, and resuspending them in 10 μL of ddH<sub>2</sub>O per sample. A total of 200 μg per sample was mixed with 10 μL of bead suspension. Acetonitrile (ACN) was added to a final concentration of 70% and incubated at room temperature, 18 min, 800 rpm. After discarding the supernatant, beads were washed twice using 200 μL of 80% ethanol. For reduction and alkylation, beads were resuspended in 70 μL of 2 mM CaCl<sub>2</sub> in 40 mM Tris pH 7.6. Proteins were reduced with 10 mM dithiothreitol (DTT) for 45 min at 37 °C and 800 rpm and alkylated with 55 mM chloroacetamide (CAA) at room temperature in the dark for 30 min. Proteins were digested (1:50 trypsin/substrate weight) overnight at 37 °C and 1000 rpm. Samples were centrifuged (5 min, 20 000 rcf) and sonicated 3 times for 30 s, and the supernatant was collected. Beads were washed once with 100 μL of ddH<sub>2</sub>O and sonicated 3 times for 30 s, and supernatants were combined with previous supernatants. Samples were acidified with formic acid (FA) to a final concentration of 1%. Peptides were desalted using tC18 RP solid-phase extraction cartridges (Waters Corp.; wash solvent: 0.1% FA; elution solvent: 0.1% FA in 50% acetonitrile (ACN)). Samples were frozen in a −80 °C freezer, dried in a SpeedVac, and reconstituted in 0.1% FA, and the peptide concentration was determined using a NanoDrop and stored at −20 °C until LC-MS<sup>2</sup> analysis.</div><div class="NLM_p">A microflow LC-MSMS setup with a Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) was used as described in detail in previous publications.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Peptides of weight 50 μg dissolved in 0.1% FA were directly injected onto the microflow LC system. Online chromatography was performed using a commercially available Thermo Fisher Scientific Acclaim PepMap 100 C18 LC column (2 μm particle size, 1 mm ID × 150 mm; catalog number 164711). Column temperature was maintained at 55 °C using the integrated column oven. Peptides were delivered at a flow rate of 50 μL/min and separated using a two-step linear gradient (120 min) ranging from 1–24% (105 min) and 24–35% (15 min) of LC solvent B (0.1% FA, 3% DMSO in ACN) in LC solvent A (0.1% FA, 3% DMSO).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The Q Exactive HF-X was operated as follows: positive polarity; spray voltage 4 kV; capillary temperature 320 °C; vaporizer temperature 200 °C. The flow rates of sheath gas, aux gas, and sweep gas were set to 40, 3, and 0, respectively. TopN was set to 50. Full MS was read out in the orbitrap, the resolution was set to 120 000, and the mass range was set to 360–1300. The full MS AGC target value was 3E6 with a maximum IT of 100 ms, and the RF lens value was set to 40. Peptide match was set to the preferred value, and the default charge state was set to 2. The dynamic exclusion duration was set to 40 s, and exclude isotopes were switched on. For readout of MS2 spectra, the orbitrap resolution was set to 15 000 and the mass range was set to 200–2000. The isolation width was set to 1.3 <i>m</i>/<i>z</i>, the first mass was fixed at 100 <i>m</i>/<i>z</i>, and NCE was 28. The AGC target value was set to 1E5 at a maximum IT of 22 ms.</div><div class="NLM_p">Protein and peptide identification and quantification were performed using MaxQuant<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (version 1.6.12.0) by searching the MS<sup>2</sup> data against all canonical protein sequences as annotated in the UniProt reference database (human proteins only, downloaded 24.08.2020) using the search engine Andromeda.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Carbamidomethylated cysteine was set as a fixed modification; oxidation of methionine and N-terminal protein acetylation were set as variable modifications. Trypsin/P was specified as a proteolytic enzyme, and up to two missed cleavage sites were allowed. The minimum peptide length was set to seven, and all data were adjusted to 1% peptide-spectrum-match (PSM) and 1% protein false discovery rate (FDR). LFQ-based quantification was enabled including the match between runs option and without normalization.</div><div class="NLM_p">Data analysis was performed using the Perseus software suite<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (version 1.6.14.0) and Microsoft Excel on identified and quantified protein groups as provided in the proteinGroups.txt file. Proteingroups.txt was filtered for contaminants and reverse hits, and median centric normalization and log 2 transformation were performed. The <b>14</b>-treated replicate 1 showed high differences from the other conditions and was not considered for further analysis. Entries were filtered for at least three valid values in one condition. Two-sample <i>t</i>-tests were performed (S0:0.1, permutation-based FDR: 5%; number of randomizations: 250). For principal component analysis (PCA), the remaining missing values were replaced from normal distribution (width 0.3, downshift: 1.8).</div><div class="NLM_p last">The mass spectrometry proteomics data and complete MaxQuant search results have been deposited to the ProteomeXchange Consortium (<a href="http://www.proteomexchange.org/" class="extLink">http://www.proteomexchange.org/</a>) via the PRIDE<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> partner repository with the data set identifier PXD025257.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Computational Studies on WDR5 Degraders</h3><div id="sec4_18_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Structure Preparation</h4><div class="NLM_p last">Preparation of protein structures started from the published crystal structures of WDR5 in complex with OICR-9429 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>; resolution: 1.50 Å)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and of VHL in complex with VH032 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H">4W9H</a>; resolution: 2.10 Å).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The structures were curated within the Molecular Operating Environment (MOE) 2019.01023<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> using its structure preparation utility. Chain F of structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H">4W9H</a>, containing VHL and VH032, was used, and the termini were capped. The dimethylarsenic-cystein in position 77 was mutated to cystein, side chains of unresolved amino acids were added automatically, and the system was protonated (Protonate3D4)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> at the pH of the crystallization buffer (6.3). Chain A of the WDR5 structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a> was similarly prepared; its termini were capped, and missing side chains were added automatically. Alternate conformations with occupancies of 0.5 were resolved based on visual inspection (conformation A was selected for residues 81, 84, 91, 154, 209, 256), and conformers with the highest occupancies were chosen where possible (conformation C for residue 325). The structure file contained an unknown atom, which was removed. Also, the partially resolved methylene-morpholino substituent of OICR-9429 was deleted and a para-positioned methyl group added to the terminal aromatic ring, where the amide bond to the linker in the studied PROTACs is formed. The structure was protonated at pH 6.5. All water molecules, ions, and other small molecules were removed from the structures.</div></div><div id="sec4_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Protein–Protein Docking</h4><div class="NLM_p last">The prepared structures of WDR5 with the modified OICR-9429 ligand and VHL with VH032 were docked onto each other using MOE’s Protein/Protein Docking (PPD) utility. VHL and VH032 were defined as “receptor”, whereas WDR5 and the modified OICR-9429 were used as “ligand”. Receptor- and ligand-sites were defined around the small molecules. Hydrophobic patch potentials were enabled, and antibody-specific options were turned off. Termination criteria were set to 800 iterations and a gradient of 0.001 kcal/(mol*Å) during the final minimization. The maximum numbers of returned poses after preplacement, placement, and refinement were set to 100 000, 10 000, and 1000, respectively. The obtained protein–protein complexes were evaluated based on the score assigned by MOE and the distance between the atoms in the modified OICR-9429 and VH032 ligands, which are linked in the final PROTACs. Ten protein/protein complexes showed a distance under 4 Å between the two critical carbon atoms, with five being ranked in the top 20% (ranks 52, 53, 59, 78, and 79). These five complexes were used as receptors for a subsequent small-molecule docking.</div></div><div id="sec4_18_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Small-Molecule Docking</h4><div class="NLM_p">A protocol for small-molecule docking was verified by redocking the modified OICR-9429 and VH032 ligands to their respective receptors. Small-molecule docking was performed with GOLD V 5.8.15<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> using the implemented scoring function ChemPLP. For each ligand, 100 docking solutions were generated without allowing for early termination. Default values for the genetic algorithm were kept but with the number of operations fixed at 100 000. Docking solutions were evaluated based on a rescoring with DrugscoreX6<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> (V0.90, CSD potentials), and RMSD values were calculated with fconv V 1.247.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The DSX top-ranked pose of VH032 showed an RMSD value of 0.27 Å, and the modified OICR-9429 ligand achieved 0.36 Å, indicating perfect reproduction of the crystallographically observed binding modes. Degraders <b>8e–j</b> were built and minimized (MMFF94×, gradient: 0.0001 kcal/(mol<sup>•</sup>Å)) using MOE and docked to the protein–protein complexes (obtained by PPD as described above) as receptor structures. Fifty docking solutions were created for each ligand–receptor pair. The non-hydrogen atoms of VH032 and the <i>p</i>-tolyl-phenyl-piperazinyl moiety of OICR-9429 were used as mild scaffold constraints during docking (constraint weight: 1.0) with the binding site defined as the region within 6 Å of these scaffolds.</div><div class="NLM_p">The small-molecule docking solutions were again evaluated based on a rescoring with DrugscoreX as well as the RMSD values with respect to the same scaffolds used as constraints.</div><div class="NLM_p last">Figures of structures were created using PyMOL Molecular Graphics System, Version 2.2.3. Schrödinger, LLC.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00146" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87352" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87352" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00146?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00146</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Thermal shift data; ITC data; BRET assay data; HiBiT data; immunoblotting data; proteomics data; computational data; experimental details for synthesis of all degraders, intermediates and negative controls; appendix with spectra for all degraders, intermediates, and negative controls (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Quantitative proteomics table (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf">jm1c00146_si_001.pdf (31.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_002.csv">jm1c00146_si_002.csv (3.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_003.csv">jm1c00146_si_003.csv (5.3 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00146" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elmar Wolf</span> - <span class="hlFld-Affiliation affiliation">Cancer
Systems Biology Group, Theodor Boveri Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#caafa6a7abb8e4bda5a6ac8aa8a3a5b0afa4beb8bfa7e4bfa4a3e7bdbfafb8b0a8bfb8ade4aeaf"><span class="__cf_email__" data-cfemail="3c5950515d4e124b53505a7c5e5553465952484e495112495255114b49594e465e494e5b125859">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institut
für Pharmazeutische Chemie, Goethe
University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Cancer Consortium (DKTK), German Cancer
Research Center (DKFZ), 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5995-6494" title="Orcid link">https://orcid.org/0000-0001-5995-6494</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dab1b4bbaaaa9aaab2bba8b7b9b2bfb7f4afb4b3f7bca8bbb4b1bcafa8aef4bebf"><span class="__cf_email__" data-cfemail="c9a2a7a8b9b989b9a1a8bba4aaa1aca4e7bca7a0e4afbba8a7a2afbcbbbde7adac">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anja Dölle</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institut
für Pharmazeutische Chemie, Goethe
University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1816-8982" title="Orcid link">https://orcid.org/0000-0002-1816-8982</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bikash Adhikari</span> - <span class="hlFld-Affiliation affiliation">Cancer
Systems Biology Group, Theodor Boveri Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9064-0356" title="Orcid link">https://orcid.org/0000-0002-9064-0356</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Krämer</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institut
für Pharmazeutische Chemie, Goethe
University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janik Weckesser</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institut
für Pharmazeutische Chemie, Goethe
University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5100-2776" title="Orcid link">https://orcid.org/0000-0001-5100-2776</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicola Berner</span> - <span class="hlFld-Affiliation affiliation">Chair
of Proteomics and Bioanalytics, Technical
University of Munich, 85354 Freising, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Cancer Consortium (DKTK), German Cancer
Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lena-Marie Berger</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institut
für Pharmazeutische Chemie, Goethe
University Frankfurt am Main, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7835-8067" title="Orcid link">https://orcid.org/0000-0002-7835-8067</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mathias Diebold</span> - <span class="hlFld-Affiliation affiliation">Institut
für Pharmazie und Lebensmittelchemie, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7595-3901" title="Orcid link">https://orcid.org/0000-0002-7595-3901</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, ON M5G 2M9, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre, University Health
Network, Toronto, ON M5G 2C1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Biophysics, University of Toronto, Toronto, ON M5G 2M9, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-4971-3250" title="Orcid link">https://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jakob Gebel</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Biophysical Chemistry and Center for Biomolecular Magnetic
Resonance and Cluster of Excellence Macromolecular Complexes (CEF), Goethe University Frankfurt am Main, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frank Löhr</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Biophysical Chemistry and Center for Biomolecular Magnetic
Resonance and Cluster of Excellence Macromolecular Complexes (CEF), Goethe University Frankfurt am Main, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-6399-9497" title="Orcid link">https://orcid.org/0000-0001-6399-9497</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Volker Dötsch</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences (BMLS), Goethe University Frankfurt am Main, Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Biophysics, University of Toronto, Toronto, ON M5G 2M9, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Eilers</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Molecular
Biology, Theodor Boveri
Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie Heinzlmeir</span> - <span class="hlFld-Affiliation affiliation">Chair
of Proteomics and Bioanalytics, Technical
University of Munich, 85354 Freising, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1066-6565" title="Orcid link">https://orcid.org/0000-0002-1066-6565</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernhard Kuster</span> - <span class="hlFld-Affiliation affiliation">Chair
of Proteomics and Bioanalytics, Technical
University of Munich, 85354 Freising, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Cancer Consortium (DKTK), German Cancer
Research Center (DKFZ), 69120 Heidelberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Bavarian Biomolecular Mass Spectrometry Center (BayBioMS), Technical University of Munich, 85354 Freising, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9094-1677" title="Orcid link">https://orcid.org/0000-0002-9094-1677</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christoph Sotriffer</span> - <span class="hlFld-Affiliation affiliation">Institut
für Pharmazie und Lebensmittelchemie, University of Würzburg, Am Hubland, 97074 Würzburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4713-4068" title="Orcid link">https://orcid.org/0000-0003-4713-4068</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.D. and B.A. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>A.D. synthesized all OICR-9429-based compounds, all E3 ligase linkers, and negative controls. A.D. evaluated all compounds in biophysical studies (DSF and calorimetry assays) with the help of J.W. and J.G. B.A. performed all cellular studies. J.W. synthesized all pyrroloimidazole-based compounds. J.G., F.L., and V.D. contributed to the chemical analysis of the degrader. A.K. expressed and purified the recombinant protein. L.-M.B. performed the BRET assays for all compounds. M.D. performed all computational modeling. C.S. supervised computational modeling studies. M.M.S. cloned the plasmids for the BRET assay. D.B.-L. supervised the cloning procedure for BRET assay clones. C.H.A. and M.E. gave critical input on the project and edited the manuscript. S.H. and N.B. performed the proteomic study and data analysis together with B.A. and B.K., who supervised the proteomic study. S.K. supervised the synthesis and biophysical experiments, and E.W. supervised the cell-based assays. E.W. and S.K. designed the study. A.D., B.A., E.W., and S.K. wrote the manuscript, which was approved by all authors.</p></li><li><span class="author-information-subsection-header">Funding</span><p>Research was supported by the European research council (TarMYC to E.W.), the Federal Ministry of Education and Research (CANCER to E.W., A.D., M.E., and S.K.), the DFG (WO 2108/1-1 and GRK 2243 to E.W.), and the Germain Cancer Aid (MSNZ Würzburg to E.W.). A.D., A.K., J.W., L.-M.B., M.M.S., D.B.-L., C.H.A., and S.K. are grateful for support by the SGC, a registered charity (no. 1097737) that receives funds from AbbVie, BayerAG, Boehringer Ingelheim, the Canada Foundation for Innovation, Eshelman Institute for Innovation, Genentech, Genome Canada through Ontario Genomics Institute [OGI-196], EU/EFPIA/OICR/McGill/KTH/Diamond, InnovativeMedicines Initiative 2 Joint Undertaking [EUbOPEN grant 875510], Janssen, Merck KGaA, Merck & Co, Pfizer, the São Paulo Research Foundation-FAPESP, Takeda, and Wellcome. A.K. and S.K. are also grateful for support from the German translational cancer network (DKTK) as well as the Frankfurt Cancer Institute (FCI).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7643-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Nina Krause and Lennart Laube from the University of Frankfurt for their synthesis of the pomalidomide linkers as well as Achim Glaesmann and Dominic Löw from the University of Frankfurt for their synthetic support in the lab. We thank Masoud Vedadi for donation of WDR5 plasmids.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ASH2L</td><td class="NLM_def"><p class="first last">absent, small, or homeotic-2 like</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">HMT</td><td class="NLM_def"><p class="first last">histone methyltransferase</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">KMT</td><td class="NLM_def"><p class="first last">lysine methyltransferase</p></td></tr><tr><td class="NLM_term">MbIIIb</td><td class="NLM_def"><p class="first last">Myc-box IIIb</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed-lineage leukemia</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">RBBP5</td><td class="NLM_def"><p class="first last">retinoblastoma binding protein 5</p></td></tr><tr><td class="NLM_term">SET</td><td class="NLM_def"><p class="first last">Su(var)3-9, Enhancer-of-zeste, and Trithorax</p></td></tr><tr><td class="NLM_term">WBM</td><td class="NLM_def"><p class="first last">WDR5-binding</p></td></tr><tr><td class="NLM_term">WDR5</td><td class="NLM_def"><p class="first last">WD-repeat-containing protein 5</p></td></tr><tr><td class="NLM_term">WIN</td><td class="NLM_def"><p class="first last">WDR5-interacting</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53887" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53887" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, B. F.</span></span> <span> </span><span class="NLM_article-title">Understanding Nucleosome Dynamics and their Links to Gene Expression and DNA Replication</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrm.2017.47" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=28537572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVahurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=548-562&author=W.+K.+M.+Laiauthor=B.+F.+Pugh&title=Understanding+Nucleosome+Dynamics+and+their+Links+to+Gene+Expression+and+DNA+Replication&doi=10.1038%2Fnrm.2017.47"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding nucleosome dynamics and their links to gene expression and DNA replication</span></div><div class="casAuthors">Lai, William K. M.; Pugh, B. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">548-562</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Advances in genomics technol. have provided the means to probe myriad chromatin interactions at unprecedented spatial and temporal resoln.  This has led to a profound understanding of nucleosome organization within the genome, revealing that nucleosomes are highly dynamic.  Nucleosome dynamics are governed by a complex interplay of histone compn., histone post-translational modifications, nucleosome occupancy and positioning within chromatin, which are influenced by numerous regulatory factors, including general regulatory factors, chromatin remodellers, chaperones and polymerases.  It is now known that these dynamics regulate diverse cellular processes ranging from gene transcription to DNA replication and repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqp-i8SEjuIrVg90H21EOLACvtfcHk0ljeoI1fhKdeaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVahurY%253D&md5=185a8ed8b369f77075d3dc602852d3b0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.47%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DW.%2BK.%2BM.%26aulast%3DPugh%26aufirst%3DB.%2BF.%26atitle%3DUnderstanding%2520Nucleosome%2520Dynamics%2520and%2520their%2520Links%2520to%2520Gene%2520Expression%2520and%2520DNA%2520Replication%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D548%26epage%3D562%26doi%3D10.1038%2Fnrm.2017.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbruzzese, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. A. G.</span></span> <span> </span><span class="NLM_article-title">The PI 3-kinase/Akt Signaling Pathway is activated due to aberrant Pten Expression and Targets Transcription Factors NF-kappaB and c-Myc in Pancreatic Cancer Cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">8571</span>– <span class="NLM_lpage">8580</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fsj.onc.1207902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=15467756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsF2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=8571-8580&author=T.+Asanoauthor=Y.+Yaoauthor=J.+Zhuauthor=D.+Liauthor=J.+L.+Abbruzzeseauthor=S.+A.+G.+Reddy&title=The+PI+3-kinase%2FAkt+Signaling+Pathway+is+activated+due+to+aberrant+Pten+Expression+and+Targets+Transcription+Factors+NF-kappaB+and+c-Myc+in+Pancreatic+Cancer+Cells&doi=10.1038%2Fsj.onc.1207902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The PI 3-kinase/Akt signaling pathway is activated due to aberrant Pten expression and targets transcription factors NF-κB and c-Myc in pancreatic cancer cells</span></div><div class="casAuthors">Asano, Takayuki; Yao, Yixin; Zhu, Jijiang; Li, Donghui; Abbruzzese, James L.; Reddy, Shrikanth A. G.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">53</span>),
    <span class="NLM_cas:pages">8571-8580</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The persistent activation of signaling cascades results in dramatic consequences that include loss of cellular growth control and neoplastic transformation.  We show here that phosphoinositide 3-kinase (PI 3-kinase) and its mediator Akt were constitutively activated in pancreatic cancer and that this might be due to the aberrant expression of their natural antagonist MMAC/PTEN.  Indeed, our results show that MMAC/PTEN expression was either lost or significantly reduced in five of eight cell lines and in twelve of seventeen tumor specimens examd.  That the poor expression of MMAC/PTEN in pancreatic cancer cells could be due to promoter methylation was indicated by methylation-specific PCR anal.  Our studies also indicated that PI 3-kinase targeted two important transcription factors in pancreatic cancer cells.  The ability of constitutively activated NF-κB to induce gene expression and the stabilization of c-MYC protein by decreased phosphorylation of Thr58 were both dependent on PI 3-kinase activity.  When pancreatic cancer cells were treated with a peptide antagonist of NF-κB nuclear translocation, or stably transfected with a dominant-neg. mutant of MYC, their proliferation was markedly inhibited. Taken together, these data indicate that the aberrant expression of MMAC/PTEN contributes to the activation of the PI 3-kinase/Akt pathway and its transcription factor mediators in pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNKTz7-bIWcbVg90H21EOLACvtfcHk0liU4C70-qTJ1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsF2msrk%253D&md5=49f2070b7000c93a96f1354327de9be5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207902%26sid%3Dliteratum%253Aachs%26aulast%3DAsano%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAbbruzzese%26aufirst%3DJ.%2BL.%26aulast%3DReddy%26aufirst%3DS.%2BA.%2BG.%26atitle%3DThe%2520PI%25203-kinase%252FAkt%2520Signaling%2520Pathway%2520is%2520activated%2520due%2520to%2520aberrant%2520Pten%2520Expression%2520and%2520Targets%2520Transcription%2520Factors%2520NF-kappaB%2520and%2520c-Myc%2520in%2520Pancreatic%2520Cancer%2520Cells%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D8571%26epage%3D8580%26doi%3D10.1038%2Fsj.onc.1207902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors as Targets for Cancer Therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1038/nrc906</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc906" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=12360277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=740-749&author=J.+E.+Darnell&title=Transcription+Factors+as+Targets+for+Cancer+Therapy&doi=10.1038%2Fnrc906"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors as targets for cancer therapy</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">740-749</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A limited list of transcription factors are overactive in most human cancer cells, which makes them targets for the development of anticancer drugs.  That they are the most direct and hopeful targets for treating cancer is proposed, and this is supported by the fact that there are many more human oncogenes in signaling pathways than there are oncogenic transcription factors.  But how could specific transcription-factor activity be inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJxyqucgv0aLVg90H21EOLACvtfcHk0liU4C70-qTJ1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D&md5=239132f634b138965e3a11c63f7cab76</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc906%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520Factors%2520as%2520Targets%2520for%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D740%26epage%3D749%26doi%3D10.1038%2Fnrc906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G. A.</span></span> <span> </span><span class="NLM_article-title">GATA Transcription Factors and Cancer</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1178</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1177/1947601911404223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1177%2F1947601911404223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=21779441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVSls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1178-1188&author=R.+Zhengauthor=G.+A.+Blobel&title=GATA+Transcription+Factors+and+Cancer&doi=10.1177%2F1947601911404223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">GATA transcription factors and cancer</span></div><div class="casAuthors">Zheng, Rena; Blobel, Gerd A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1178-1188</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  It has been almost a quarter century since it was first appreciated that a class of oncogenes contained in rapidly transforming avian retroviruses encoded DNA-binding transcription factors.  As with other oncogenes, genetic recombination with the viral genome led to their overexpression or functional alteration.  In the years that followed, alterations of numerous transcription factors were shown to be causatively involved in various cancers in human patients and model organisms.  Depending on their normal cellular functions, these factors were subsequently categorized as proto-oncogenes or tumor suppressor genes.  This review focuses on the role of GATA transcription factors in carcinogenesis.  GATA factors are zinc finger DNA binding proteins that control the development of diverse tissues by activating or repressing transcription.  GATA factors thus coordinate cellular maturation with proliferation arrest and cell survival.  Therefore, a role of this family of genes in human cancers is not surprising.  Prominent examples include structural mutations in GATA1 that are found in almost all megakaryoblastic leukemias in patients with Down syndrome; loss of GATA3 expression in aggressive, dedifferentiated breast cancers; and silencing of GATA4 and GATA5 expression in colorectal and lung cancers.  Here, we discuss possible mechanisms of carcinogenesis vis-a-vis the normal functions of GATA factors as they pertain to human patients and mouse models of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP6TgBqSjsqrVg90H21EOLACvtfcHk0liU4C70-qTJ1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVSls7k%253D&md5=9a6d7e4cec5120df504a8e017911c442</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1947601911404223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911404223%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DGATA%2520Transcription%2520Factors%2520and%2520Cancer%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D1178%26epage%3D1188%26doi%3D10.1177%2F1947601911404223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannagi, R.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors c-Myc and CDX2 mediate E-selectin Ligand Expression in Colon Cancer Cells undergoing EGF/bFGF-induced epithelial-mesenchymal Transition</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">7776</span>– <span class="NLM_lpage">7781</span>, <span class="refDoi"> DOI: 10.1073/pnas.1111135109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1073%2Fpnas.1111135109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=22547830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1Kntro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=7776-7781&author=K.+Sakumaauthor=M.+Aokiauthor=R.+Kannagi&title=Transcription+Factors+c-Myc+and+CDX2+mediate+E-selectin+Ligand+Expression+in+Colon+Cancer+Cells+undergoing+EGF%2FbFGF-induced+epithelial-mesenchymal+Transition&doi=10.1073%2Fpnas.1111135109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors c-Myc and CDX2 mediate E-selectin ligand expression in colon cancer cells undergoing EGF/bFGF-induced epithelial-mesenchymal transition</span></div><div class="casAuthors">Sakuma, Keiichiro; Aoki, Masahiro; Kannagi, Reiji</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7776-7781, S7776/1-S7776/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Sialyl Lewis x (sLex) and sialyl Lewis a (sLea) glycans are expressed on highly metastatic colon cancer cells.  They promote extravasation of cancer cells and tumor angiogenesis via interacting with E-selectin on endothelial cells.  Recently, epithelial-mesenchymal transition (EMT) has been noted as a crit. phenotypic alteration in metastatic cancer cells.  To address the assocn. between sLex/a expression and EMT, we assessed whether sLex/a are highly expressed on colon cancer cells undergoing EMT.  Treatment of HT29 and DLD-1 cells with EGF and/or basic FGF (bFGF) induced EMT and significantly increased sLex/a expression resulting in enhanced E-selectin binding activity.  The transcript levels of the glycosyltransferase genes ST3GAL1/3/4 and FUT3 were significantly elevated and that of FUT2 was significantly suppressed by the treatment.  We provide evidence that ST3GAL1/3/4 and FUT3 are transcriptionally up-regulated by c-Myc with probable involvement of Ser62 phosphorylation, and that FUT2 is transcriptionally down-regulated through the attenuation of CDX2.  The contribution of c-Myc and CDX2 to the sLex/a induction was proved to be significant by knockdown or forced expression expts.  Interestingly, the cells undergoing EMT exhibited significantly increased VEGF secretion, which can promote tumor angiogenesis in cooperation with sLex/a.  Finally, immunohistol. study indicated high E-selectin ligand expression on cancer cells undergoing EMT in vivo, supporting their coexistence obsd. in vitro.  These results suggest a significant link between sLex/a expression and EMT in colon cancer cells and a pivotal role of c-Myc and CDX2 in regulating sLex/a expression during EMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ysc2wLI0SrVg90H21EOLACvtfcHk0likwhKEMIK1hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1Kntro%253D&md5=4fffe94db4c0ae1ebfe5c579d32cdfc6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111135109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111135109%26sid%3Dliteratum%253Aachs%26aulast%3DSakuma%26aufirst%3DK.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DKannagi%26aufirst%3DR.%26atitle%3DTranscription%2520Factors%2520c-Myc%2520and%2520CDX2%2520mediate%2520E-selectin%2520Ligand%2520Expression%2520in%2520Colon%2520Cancer%2520Cells%2520undergoing%2520EGF%252FbFGF-induced%2520epithelial-mesenchymal%2520Transition%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D7776%26epage%3D7781%26doi%3D10.1073%2Fpnas.1111135109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. A.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oncogene and Tumor-suppressor Gene Expression: a novel Strategy for Cancer Prevention and Treatment</span>. <i>Ann. Surg. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1007/BF02303545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1007%2FBF02303545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=7834433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADyaK2M7js1Shuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1994&pages=79-86&author=J.+A.+Roth&title=Modulation+of+Oncogene+and+Tumor-suppressor+Gene+Expression%3A+a+novel+Strategy+for+Cancer+Prevention+and+Treatment&doi=10.1007%2FBF02303545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oncogene and tumor-suppressor gene expression: a novel strategy for cancer prevention and treatment</span></div><div class="casAuthors">Roth J A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of surgical oncology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-86</span>
        ISSN:<span class="NLM_cas:issn">1068-9265</span>.
    </div><div class="casAbstract">BACKGROUND:  Advances in our understanding of the molecular biology of cancer have the potential for translation into clinical cancer prevention and treatment strategies.  Oncogenes and tumor suppressor genes have been implicated in the development of human cancers Mutations in gene families occur in both premalignant lesions and invasive tumors.  CONCLUSION:  Recombinant DNA constructs can be made that eliminate expression of a mutant oncogene protein or provide a normal copy of a tumor suppressor gene to the cancer cell.  Reversal of a single genetic lesion is sufficient to inhibit cancer cell proliferation and tumorigenicity.  Viral vectors are now being evaluated for direct delivery of these constructs to human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXDQuuyW-2sHWEEzDrymxmfW6udTcc2eaQYvS5f-jduLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M7js1Shuw%253D%253D&md5=ee88d4695942a97597f8085649e9e15c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2FBF02303545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02303545%26sid%3Dliteratum%253Aachs%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26atitle%3DModulation%2520of%2520Oncogene%2520and%2520Tumor-suppressor%2520Gene%2520Expression%253A%2520a%2520novel%2520Strategy%2520for%2520Cancer%2520Prevention%2520and%2520Treatment%26jtitle%3DAnn.%2520Surg.%2520Oncol.%26date%3D1994%26volume%3D1%26spage%3D79%26epage%3D86%26doi%3D10.1007%2FBF02303545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajasekaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y. K.</span></span> <span> </span><span class="NLM_article-title">Loss of Tumor Suppressor Gene Function in Human Cancer: An Overview</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2647</span>– <span class="NLM_lpage">2693</span>, <span class="refDoi"> DOI: 10.1159/000495956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1159%2F000495956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30562755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlGqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2018&pages=2647-2693&author=L.-H.+Wangauthor=C.-F.+Wuauthor=N.+Rajasekaranauthor=Y.+K.+Shin&title=Loss+of+Tumor+Suppressor+Gene+Function+in+Human+Cancer%3A+An+Overview&doi=10.1159%2F000495956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of Tumor Suppressor Gene Function in Human Cancer: An Overview</span></div><div class="casAuthors">Wang, Li-Hui; Wu, Chun-Fu; Rajasekaran, Nirmal; Shin, Young Kee</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2647-2693</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">A review.  Cancer is a disease caused by the accumulation of genetic and epigenetic changes in two types of genes: tumor suppressor genes (TSGs) and proto-oncogenes.  Extensive research has been conducted over the last few decades to elucidate the role of TSGs in cancer development.  In cancer, loss of TSG function occurs via the deletion or inactivation of two alleles, according to Knudson's two-hit model hypothesis.  It has become clear that mutations in TSGs are recessive at the level of an individual cell; therefore, a single mutation in a TSG is not sufficient to cause carcinogenesis.  However, many studies have identified candidate TSGs that do not conform with this std. definition, including genes inactivated by epigenetic silencing rather than by deletion.  In addn., proteasomal degrdn. by ubiquitination, abnormal cellular localization, and transcriptional regulation are also involved in the inactivation of TSGs.  This review incorporates these novel addnl. mechanisms of TSG inactivation into the existing two-hit model and proposes a revised multiple-hit model that will enable the identification of novel TSGs that can be used as prognostic and predictive biomarkers of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1uM-lJOCirVg90H21EOLACvtfcHk0likwhKEMIK1hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlGqsw%253D%253D&md5=9a0933a0b0e8de97b77ca8a2e32ac6b6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1159%2F000495956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000495956%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.-H.%26aulast%3DWu%26aufirst%3DC.-F.%26aulast%3DRajasekaran%26aufirst%3DN.%26aulast%3DShin%26aufirst%3DY.%2BK.%26atitle%3DLoss%2520of%2520Tumor%2520Suppressor%2520Gene%2520Function%2520in%2520Human%2520Cancer%253A%2520An%2520Overview%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2018%26volume%3D51%26spage%3D2647%26epage%3D2693%26doi%3D10.1159%2F000495956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlich, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Methylation of Histone H3 Lys 4 in coding Regions of active Genes</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">8695</span>– <span class="NLM_lpage">8700</span>, <span class="refDoi"> DOI: 10.1073/pnas.082249499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1073%2Fpnas.082249499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=12060701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD38XltF2hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=8695-8700&author=B.+E.+Bernsteinauthor=E.+L.+Humphreyauthor=R.+L.+Erlichauthor=R.+Schneiderauthor=P.+Boumanauthor=J.+S.+Liuauthor=T.+Kouzaridesauthor=S.+L.+Schreiber&title=Methylation+of+Histone+H3+Lys+4+in+coding+Regions+of+active+Genes&doi=10.1073%2Fpnas.082249499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3 Lys 4 in coding regions of active genes</span></div><div class="casAuthors">Bernstein, Bradley E.; Humphrey, Emily L.; Erlich, Rachel L.; Schneider, Robert; Bouman, Peter; Liu, Jun S.; Kouzarides, Tony; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8695-8700</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone tails regulate chromatin structure and transcription.  Here we present global analyses of histone acetylation and histone H3 Lys 4 methylation patterns in yeast.  We observe a significant correlation between acetylation of histones H3 and H4 in promoter regions and transcriptional activity.  In contrast, we find that dimethylation of histone H3 Lys 4 in coding regions correlates with transcriptional activity.  The histone methyltransferase Set1 is required to maintain expression of these active, promoter-acetylated, and coding region-methylated genes.  Global comparisons reveal that genomic regions deacetylated by the yeast enzymes Rpd3 and Hda1 overlap extensively with Lys 4 hypo- but not hypermethylated regions.  In the context of recent studies showing that Lys 4 methylation precludes histone deacetylase recruitment, we conclude that Set1 facilitates transcription, in part, by protecting active coding regions from deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQbFpuyMjGbbVg90H21EOLACvtfcHk0ljs1HGb5kNU7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltF2hsb8%253D&md5=d468bafc67a92d127a18545866e39484</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.082249499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.082249499%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DHumphrey%26aufirst%3DE.%2BL.%26aulast%3DErlich%26aufirst%3DR.%2BL.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBouman%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DJ.%2BS.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DMethylation%2520of%2520Histone%2520H3%2520Lys%25204%2520in%2520coding%2520Regions%2520of%2520active%2520Genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D8695%26epage%3D8700%26doi%3D10.1073%2Fpnas.082249499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span> <span> </span><span class="NLM_article-title">Histone Lysine Methylation Dynamics: Establishment, Regulation, and biological Impact</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+van+Rechemauthor=J.+R.+Whetstine&title=Histone+Lysine+Methylation+Dynamics%3A+Establishment%2C+Regulation%2C+and+biological+Impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0ljs1HGb5kNU7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3Dvan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520Lysine%2520Methylation%2520Dynamics%253A%2520Establishment%252C%2520Regulation%252C%2520and%2520biological%2520Impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buratowski, S.</span></span> <span> </span><span class="NLM_article-title">Determinants of Histone H3K4 Methylation Patterns</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">785.e6</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2017.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=29129639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2017&pages=773-785.e6&author=L.+M.+Soaresauthor=P.+C.+Heauthor=Y.+Chunauthor=H.+Suhauthor=T.+Kimauthor=S.+Buratowski&title=Determinants+of+Histone+H3K4+Methylation+Patterns&doi=10.1016%2Fj.molcel.2017.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of Histone H3K4 Methylation Patterns</span></div><div class="casAuthors">Soares, Luis M.; He, P. Cody; Chun, Yujin; Suh, Hyunsuk; Kim, Tae Soo; Buratowski, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-785.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Various factors differentially recognize trimethylated histone H3 lysine 4 (H3K4me3) near promoters, H3K4me2 just downstream, and promoter-distal H3K4me1 to modulate gene expression.  This methylation "gradient" is thought to result from preferential binding of the H3K4 methyltransferase Set1/complex assocd. with Set1 (COMPASS) to promoter-proximal RNA polymerase II.  However, other studies have suggested that location-specific cues allosterically activate Set1.  Chromatin immunopptn. sequencing (ChIP-seq) expts. show that H3K4 methylation patterns on active genes are not universal or fixed and change in response to both transcription elongation rate and frequency as well as reduced COMPASS activity.  Fusing Set1 to RNA polymerase II results in H3K4me2 throughout transcribed regions and similarly extended H3K4me3 on highly transcribed genes.  Tethered Set1 still requires histone H2B ubiquitylation for activity.  These results show that higher-level methylations reflect not only Set1/COMPASS recruitment but also multiple rounds of transcription.  This model provides a simple explanation for non-canonical methylation patterns at some loci or in certain COMPASS mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX-6-jGE28KbVg90H21EOLACvtfcHk0ljs1HGb5kNU7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrrK&md5=250ecdd352958b2bcd57b38d5ce5e806</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DL.%2BM.%26aulast%3DHe%26aufirst%3DP.%2BC.%26aulast%3DChun%26aufirst%3DY.%26aulast%3DSuh%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DBuratowski%26aufirst%3DS.%26atitle%3DDeterminants%2520of%2520Histone%2520H3K4%2520Methylation%2520Patterns%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D68%26spage%3D773%26epage%3D785.e6%26doi%3D10.1016%2Fj.molcel.2017.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Hijacked in Cancer: the KMT2 (MLL) Family of Methyltransferases</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1038/nrc3929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc3929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=25998713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WrurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=334-346&author=R.+C.+Raoauthor=Y.+Dou&title=Hijacked+in+Cancer%3A+the+KMT2+%28MLL%29+Family+of+Methyltransferases&doi=10.1038%2Fnrc3929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacked in cancer: the KMT2 (MLL) family of methyltransferases</span></div><div class="casAuthors">Rao, Rajesh C.; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">334-346</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone-lysine N-methyltransferase 2 (KMT2) family proteins methylate lysine 4 on the histone H3 tail at important regulatory regions in the genome and thereby impart crucial functions through modulating chromatin structures and DNA accessibility.  Although the human KMT2 family was initially named the mixed-lineage leukemia (MLL) family, owing to the role of the first-found member KMT2A in this disease, recent exome-sequencing studies revealed KMT2 genes to be among the most frequently mutated genes in many types of human cancers.  Efforts to integrate the mol. mechanisms of KMT2 with its roles in tumorigenesis have led to the development of first-generation inhibitors of KMT2 function, which could become novel cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdmrdvWshyxrVg90H21EOLACvtfcHk0lhi24iHUVJMdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WrurvP&md5=c7a7405d0f951a4ae238ef0c9e183b06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc3929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3929%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DR.%2BC.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DHijacked%2520in%2520Cancer%253A%2520the%2520KMT2%2520%2528MLL%2529%2520Family%2520of%2520Methyltransferases%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D334%26epage%3D346%26doi%3D10.1038%2Fnrc3929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opravil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin Structure by site-specific Histone H3 Methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1038/35020506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+Chromatin+Structure+by+site-specific+Histone+H3+Methyltransferases&doi=10.1038%2F35020506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lhi24iHUVJMdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Chromatin%2520Structure%2520by%2520site-specific%2520Histone%2520H3%2520Methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599%26doi%3D10.1038%2F35020506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, U.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Insights on the Regulation of the MLL/SET1 Family Histone Methyltransferases</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1863</i></span>, <span class="NLM_elocation-id">194561</span> <span class="refDoi"> DOI: 10.1016/j.bbagrm.2020.194561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.bbagrm.2020.194561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32304759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2020&author=L.+Shaauthor=A.+Ayoubauthor=U.-S.+Choauthor=Y.+Dou&title=Insights+on+the+Regulation+of+the+MLL%2FSET1+Family+Histone+Methyltransferases&doi=10.1016%2Fj.bbagrm.2020.194561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Insights on the regulation of the MLL/SET1 family histone methyltransferases</span></div><div class="casAuthors">Sha, Liang; Ayoub, Alex; Cho, Uhn-Soo; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">194561</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In eukaryotes, histone H3K4 methylation by the MLL/SET1 family histone methyltransferases is enriched at transcription regulatory elements including gene promoters and enhancers.  The level of H3K4 methylation is highly correlated with transcription activation and is one of the most frequently used histone post-translational modifications to predict transcriptional outcome.  Recently, it has been shown that rearrangement of the cellular landscape of H3K4 mono-methylation at distal enhancers precedes cell fate transition and is used for identification of novel regulatory elements for development and disease progression.  Similarly, broad H3K4 tri-methylation regions have also been used to predict intrinsic tumor suppression properties of regulator regions in a variety of cellular models.  Understanding the regulation for how H3K4 methylation is deposited and regulated is of paramount importance.  In this review, we will discuss new findings on how the MLL/SET1 family enzymes are regulated on chromatin and their potential functional and regulatory implications.  This article is part of a Special Issue entitled: The MLL family of proteins in normal development and disease edited by Thomas A Milne.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyIc180oHEh7Vg90H21EOLACvtfcHk0lhi24iHUVJMdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVels7s%253D&md5=b9bea9c6381e2471ee860c9bb936dc97</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2020.194561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2020.194561%26sid%3Dliteratum%253Aachs%26aulast%3DSha%26aufirst%3DL.%26aulast%3DAyoub%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DU.-S.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DInsights%2520on%2520the%2520Regulation%2520of%2520the%2520MLL%252FSET1%2520Family%2520Histone%2520Methyltransferases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2020%26volume%3D1863%26doi%3D10.1016%2Fj.bbagrm.2020.194561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 Methyltransferase Activity by its Core Components</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">719</span>, <span class="refDoi"> DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+Methyltransferase+Activity+by+its+Core+Components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0lidSjlTPRqh0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520Methyltransferase%2520Activity%2520by%2520its%2520Core%2520Components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chacón
Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlund, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of WD Repeat-Containing Protein 5 (WDR5)-MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4315</span>– <span class="NLM_lpage">4333</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00224</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00224" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4315-4333&author=S.+Chac%C3%B3n%0ASimonauthor=F.+Wangauthor=L.+R.+Thomasauthor=J.+Phanauthor=B.+Zhaoauthor=E.+T.+Olejniczakauthor=J.+D.+Macdonaldauthor=J.+G.+Shawauthor=C.+Schlundauthor=W.+Payneauthor=J.+Creightonauthor=S.+R.+Staufferauthor=A.+G.+Watersonauthor=W.+P.+Tanseyauthor=S.+W.+Fesik&title=Discovery+of+WD+Repeat-Containing+Protein+5+%28WDR5%29-MYC+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.0c00224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of WD Repeat-Containing Protein 5 (WDR5)-MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Chacon Simon, Selena; Wang, Feng; Thomas, Lance R.; Phan, Jason; Zhao, Bin; Olejniczak, Edward T.; MacDonald, Jonathan D.; Shaw, J. Grace; Schlund, Caden; Payne, William; Creighton, Joy; Stauffer, Shaun R.; Waterson, Alex G.; Tansey, William P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4315-4333</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The frequent deregulation of MYC and its elevated expression via multiple mechanisms drives cells to a tumorigenic state.  Indeed, MYC is overexpressed in up to ~ 50% of human cancers and is considered a highly validated anticancer target.  Recently, we discovered that WD repeat-contg. protein 5 (WDR5) binds to MYC and is a crit. cofactor required for the recruitment of MYC to its target genes and reported the first small mol. inhibitors of the WDR5-MYC interaction using structure-based design.  These compds. display high binding affinity, but have poor physicochem. properties and are hence not suitable for in vivo studies.  Herein, we conducted an NMR-based fragment screening to identify addnl. chem. matter and, using a structure-based approach, we merged a fragment hit with the previously reported sulfonamide series.  Compds. in this series can disrupt the WDR5-MYC interaction in cells, and as a consequence, we obsd. a redn. of MYC localization to chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPe6Kwe2jWL7Vg90H21EOLACvtfcHk0lidSjlTPRqh0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCht7g%253D&md5=167a092b144cbeff004737328a71c069</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00224%26sid%3Dliteratum%253Aachs%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSchlund%26aufirst%3DC.%26aulast%3DPayne%26aufirst%3DW.%26aulast%3DCreighton%26aufirst%3DJ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529-MYC%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4315%26epage%3D4333%26doi%3D10.1021%2Facs.jmedchem.0c00224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velázquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolshan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-2496&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Vel%C3%A1zquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-Awar&title=Structure-Based+Optimization+of+a+Small+Molecule+Antagonist+of+the+Interaction+Between+WD+Repeat-Containing+Protein+5+%28WDR5%29+and+Mixed-Lineage+Leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0lidSjlTPRqh0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Vel%25C3%25A1zquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-Awar%26aufirst%3DR.%26atitle%3DStructure-Based%2520Optimization%2520of%2520a%2520Small%2520Molecule%2520Antagonist%2520of%2520the%2520Interaction%2520Between%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529%2520and%2520Mixed-Lineage%2520Leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D2496%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giambruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avellino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of the Wdr5-MLL Interaction in C/EBPα N-terminal Leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-Awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+Targeting+of+the+Wdr5-MLL+Interaction+in+C%2FEBP%CE%B1+N-terminal+Leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0ljFapTfyzWrPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520Targeting%2520of%2520the%2520Wdr5-MLL%2520Interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-terminal%2520Leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent 2-Aryl-6,7-dihydro-5 H-pyrrolo1,2- aimidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5623</span>– <span class="NLM_lpage">5642</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5623-5642&author=F.+Wangauthor=K.+O.+Jeonauthor=J.+M.+Salovichauthor=J.+D.+Macdonaldauthor=J.+Alvaradoauthor=R.+D.+Gogliottiauthor=J.+Phanauthor=E.+T.+Olejniczakauthor=Q.+Sunauthor=S.+Wangauthor=D.+Camperauthor=J.+P.+Yuhauthor=J.+G.+Shawauthor=J.+Saiauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+of+Potent+2-Aryl-6%2C7-dihydro-5+H-pyrrolo1%2C2-+aimidazoles+as+WDR5-WIN-Site+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.8b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Wang, Feng; Jeon, Kyu Ok; Salovich, James M.; Macdonald, Jonathan D.; Alvarado, Joseph; Gogliotti, Rocco D.; Phan, Jason; Olejniczak, Edward T.; Sun, Qi; Wang, Shidong; Camper, DeMarco; Yuh, Joannes P.; Shaw, J. Grace; Sai, Jiqing; Rossanese, Olivia W.; Tansey, William P.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5623-5642</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted pyrroloimidazoles such as I [R = 3,4-Cl2C6H3CH2, 3-MeO-4-MeC6H3, 2-Me-4-pyridinyl, 3,5-(MeO)2-4-MeC6H2] were prepd. as inhibitors of the chromatin-regulatory scaffold protein WDR5 WDR5-interaction (WIN) site as lead compds. for development of antitumor agents.  NMR-based screening of a large fragment library identified several chem. distinct hit series that bind to the WIN site within WDR5.  Using a structure-based design approach, pyrroloimidazoles were found with dissocn. consts. < 10 nM for WDR5 and micromolar cellular activity against an AML-leukemia cell line.  The structures of I (R = 3-MeO-4-MeC6H3) and of four other compds. bound to a fragment of WDR5 were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSZaiiuG6OrVg90H21EOLACvtfcHk0ljFapTfyzWrPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN&md5=2322c6458b2a060005bf5250a46f5cbd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJeon%26aufirst%3DK.%2BO.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Potent%25202-Aryl-6%252C7-dihydro-5%2520H-pyrrolo1%252C2-%2520aimidazoles%2520as%2520WDR5-WIN-Site%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5623%26epage%3D5642%26doi%3D10.1021%2Facs.jmedchem.8b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiebert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2916</span>– <span class="NLM_lpage">2928.e13</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2019.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.celrep.2019.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30865883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=2916-2928.e13&author=E.+R.+Ahoauthor=J.+Wangauthor=R.+D.+Gogliottiauthor=G.+C.+Howardauthor=J.+Phanauthor=P.+Acharyaauthor=J.+D.+Macdonaldauthor=K.+Chengauthor=S.+L.+Loreyauthor=B.+Luauthor=S.+Wenzelauthor=A.+M.+Foshageauthor=J.+Alvaradoauthor=F.+Wangauthor=J.+G.+Shawauthor=B.+Zhaoauthor=A.+M.+Weissmillerauthor=L.+R.+Thomasauthor=C.+R.+Vakocauthor=M.+D.+Hallauthor=S.+W.+Hiebertauthor=Q.+Liuauthor=S.+R.+Staufferauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Displacement+of+WDR5+from+Chromatin+by+a+WIN+Site+Inhibitor+with+Picomolar+Affinity&doi=10.1016%2Fj.celrep.2019.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span></div><div class="casAuthors">Aho, Erin R.; Wang, Jing; Gogliotti, Rocco D.; Howard, Gregory C.; Phan, Jason; Acharya, Pankaj; Macdonald, Jonathan D.; Cheng, Ken; Lorey, Shelly L.; Lu, Bin; Wenzel, Sabine; Foshage, Audra M.; Alvarado, Joseph; Wang, Feng; Shaw, J. Grace; Zhao, Bin; Weissmiller, April M.; Thomas, Lance R.; Vakoc, Christopher R.; Hall, Matthew D.; Hiebert, Scott W.; Liu, Qi; Stauffer, Shaun R.; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2916-2928.e13</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The chromatin-assocd. protein WDR5 is a promising target for pharmacol. inhibition in cancer.  Drug discovery efforts center on the blockade of the "WIN site" of WDR5, a well-defined pocket that is amenable to small mol. inhibition.  Various cancer contexts have been proposed to be targets for WIN site inhibitors, but a lack of understanding of WDR5 target genes and of the primary effects of WIN site inhibitors hampers their utility.  Here, by the discovery of potent WIN site inhibitors, we demonstrate that the WIN site links WDR5 to chromatin at a small cohort of loci, including a specific subset of ribosome protein genes.  WIN site inhibitors rapidly displace WDR5 from chromatin and decrease the expression of assocd. genes, causing translational inhibition, nucleolar stress, and p53 induction.  Our studies define a mode by which WDR5 engages chromatin and forecast that WIN site blockade could have utility against multiple cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprdgAaHB9WfLVg90H21EOLACvtfcHk0lh63UlGcqzcbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D&md5=56e64b66f0d04ff5c36e104e54594bbd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2019.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2019.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DWenzel%26aufirst%3DS.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DHiebert%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DDisplacement%2520of%2520WDR5%2520from%2520Chromatin%2520by%2520a%2520WIN%2520Site%2520Inhibitor%2520with%2520Picomolar%2520Affinity%26jtitle%3DCell%2520Rep.%26date%3D2019%26volume%3D26%26spage%3D2916%26epage%3D2928.e13%26doi%3D10.1016%2Fj.celrep.2019.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-P.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design and Synthesis of Small Molecular Inhibitors disturbing the Interaction of MLL1-WDR5</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.ejmech.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27116709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=1-8&author=D.-D.+Liauthor=W.-L.+Chenauthor=X.-L.+Xuauthor=F.+Jiangauthor=L.+Wangauthor=Y.-Y.+Xieauthor=X.-J.+Zhangauthor=X.-K.+Guoauthor=Q.-D.+Youauthor=H.-P.+Sun&title=Structure-based+Design+and+Synthesis+of+Small+Molecular+Inhibitors+disturbing+the+Interaction+of+MLL1-WDR5&doi=10.1016%2Fj.ejmech.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Xu, Xiao-Li; Jiang, Fen; Wang, Lei; Xie, Yi-Yue; Zhang, Xiao-Jin; Guo, Xiao-Ke; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1 complex catalyzes the methylation of H3K4, and plays important roles in the development of acute leukemia harboring MLL fusion proteins.  Targeting MLL1-WDR5 protein-protein interaction (PPI) to inhibit the activity of histone methyltransferase of MLL1 complex is a novel strategy for treating of acute leukemia.  I (IC50 = 0.3 μM) was defined as a potent small mol. to disturb the interaction of MLL1-WDR5.  Here, the authors described structure-based design and synthesis of small mol. inhibitors to block MLL1-WDR5 PPI.  Esp., compd. II (IC50 = 104 nM) was the most potent small mol., and about 3-times more potent than I.  The authors also discussed the SAR of these series of compds. with docking study, which may stimulate more potent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsAbY0it5GLVg90H21EOLACvtfcHk0lh63UlGcqzcbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D&md5=b18d482f34731dd1558e9b78b770c2c1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DGuo%26aufirst%3DX.-K.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DSun%26aufirst%3DH.-P.%26atitle%3DStructure-based%2520Design%2520and%2520Synthesis%2520of%2520Small%2520Molecular%2520Inhibitors%2520disturbing%2520the%2520Interaction%2520of%2520MLL1-WDR5%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ejmech.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span> <span> </span><span class="NLM_article-title">High-affinity Small Molecular Blockers of Mixed Lineage Leukemia 1 (MLL1)-WDR5 Interaction inhibit MLL1 Complex H3K4 Methyltransferase Activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.-D.+Liauthor=W.-L.+Chenauthor=Z.-H.+Wangauthor=Y.-Y.+Xieauthor=X.-L.+Xuauthor=Z.-Y.+Jiangauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=X.-K.+Guo&title=High-affinity+Small+Molecular+Blockers+of+Mixed+Lineage+Leukemia+1+%28MLL1%29-WDR5+Interaction+inhibit+MLL1+Complex+H3K4+Methyltransferase+Activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0lh63UlGcqzcbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DZ.-H.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DGuo%26aufirst%3DX.-K.%26atitle%3DHigh-affinity%2520Small%2520Molecular%2520Blockers%2520of%2520Mixed%2520Lineage%2520Leukemia%25201%2520%2528MLL1%2529-WDR5%2520Interaction%2520inhibit%2520MLL1%2520Complex%2520H3K4%2520Methyltransferase%2520Activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freie, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathsyaraja, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R. N.</span></span> <span> </span><span class="NLM_article-title">The MYC Transcription Factor Network: balancing Metabolism, Proliferation and Oncogenesis</span>. <i>Front. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">425</span>, <span class="refDoi"> DOI: 10.1007/s11684-018-0650-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1007%2Fs11684-018-0650-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30054853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ivVeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=412-425&author=P.+A.+Carrollauthor=B.+W.+Freieauthor=H.+Mathsyarajaauthor=R.+N.+Eisenman&title=The+MYC+Transcription+Factor+Network%3A+balancing+Metabolism%2C+Proliferation+and+Oncogenesis&doi=10.1007%2Fs11684-018-0650-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The MYC transcription factor network: balancing metabolism, proliferation and oncogenesis</span></div><div class="casAuthors">Carroll Patrick A; Freie Brian W; Mathsyaraja Haritha; Eisenman Robert N</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers of medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">412-425</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Transcription factor networks have evolved in order to control, coordinate, and separate, the functions of distinct network modules spatially and temporally.  In this review we focus on the MYC network (also known as the MAX-MLX Network), a highly conserved super-family of related basic-helix-loop-helix-zipper (bHLHZ) proteins that functions to integrate extracellular and intracellular signals and modulate global gene expression.  Importantly the MYC network has been shown to be deeply involved in a broad spectrum of human and other animal cancers.  Here we summarize molecular and biological properties of the network modules with emphasis on functional interactions among network members.  We suggest that these network interactions serve to modulate growth and metabolism at the transcriptional level in order to balance nutrient demand with supply, to maintain growth homeostasis, and to influence cell fate.  Moreover, oncogenic activation of MYC and/or loss of a MYC antagonist, results in an imbalance in the activity of the network as a whole, leading to tumor initiation, progression and maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRZan0K73QGlZoq0fPSOTvfW6udTcc2eb1VrnASAhYjLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ivVeksw%253D%253D&md5=40b49032a138dfbf686fc4fb19e0ca55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs11684-018-0650-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11684-018-0650-z%26sid%3Dliteratum%253Aachs%26aulast%3DCarroll%26aufirst%3DP.%2BA.%26aulast%3DFreie%26aufirst%3DB.%2BW.%26aulast%3DMathsyaraja%26aufirst%3DH.%26aulast%3DEisenman%26aufirst%3DR.%2BN.%26atitle%3DThe%2520MYC%2520Transcription%2520Factor%2520Network%253A%2520balancing%2520Metabolism%252C%2520Proliferation%2520and%2520Oncogenesis%26jtitle%3DFront.%2520Med.%26date%3D2018%26volume%3D12%26spage%3D412%26epage%3D425%26doi%3D10.1007%2Fs11684-018-0650-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC through WDR5</span>. <i>Mol. Cell. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">1709388</span> <span class="refDoi"> DOI: 10.1080/23723556.2019.1709388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1080%2F23723556.2019.1709388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32158922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADC%252BB383itFaltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2020&author=L.+R.+Thomasauthor=C.+M.+Adamsauthor=S.+W.+Fesikauthor=C.+M.+Eischenauthor=W.+P.+Tansey&title=Targeting+MYC+through+WDR5&doi=10.1080%2F23723556.2019.1709388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MYC through WDR5</span></div><div class="casAuthors">Thomas Lance R; Tansey William P; Adams Clare M; Eischen Christine M; Fesik Stephen W; Tansey William P</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & cellular oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1709388</span>
        ISSN:<span class="NLM_cas:issn">2372-3556</span>.
    </div><div class="casAbstract">The oncoprotein transcription factor MYC is overexpressed in most cancers and is responsible for hundreds of thousands of cancer deaths worldwide every year.  MYC is also a highly validated - but currently undruggable - anti-cancer target.  We recently showed that breaking the interaction of MYC with its chromatin co-factor WD repeat-containing protein 5 (WDR5) promotes tumor regression in mouse xenografts, laying the foundation for a new strategy to inhibit MYC in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJ85Bz5vYqRC86Js-BCLiifW6udTcc2eb1VrnASAhYjLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383itFaltg%253D%253D&md5=8fde22bb0fa8adcc112eb4467c1b6db4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F23723556.2019.1709388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F23723556.2019.1709388%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DAdams%26aufirst%3DC.%2BM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DTargeting%2520MYC%2520through%2520WDR5%26jtitle%3DMol.%2520Cell.%2520Oncol.%26date%3D2020%26volume%3D7%26doi%3D10.1080%2F23723556.2019.1709388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ‘undruggable’ Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%98undruggable%E2%80%99+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0li_AIJzaLU3MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodir, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnezis, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swigart, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evan, G. I.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Myc Family Proteins eradicates KRas-driven Lung Cancer in Mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1101/gad.205542.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1101%2Fgad.205542.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23475959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVKkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=504-513&author=L.+Soucekauthor=J.+R.+Whitfieldauthor=N.+M.+Sodirauthor=D.+Mass%C3%B3-Vall%C3%A9sauthor=E.+Serranoauthor=A.+N.+Karnezisauthor=L.+B.+Swigartauthor=G.+I.+Evan&title=Inhibition+of+Myc+Family+Proteins+eradicates+KRas-driven+Lung+Cancer+in+Mice&doi=10.1101%2Fgad.205542.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Myc family proteins eradicates KRas-driven lung cancer in mice</span></div><div class="casAuthors">Soucek, Laura; Whitfield, Jonathan R.; Sodir, Nicole M.; Masso-Valles, Daniel; Serrano, Erika; Karnezis, Anthony N.; Swigart, Lamorna Brown; Evan, Gerard I.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The principal reason for failure of targeted cancer therapies is the emergence of resistant clones that regenerate the tumor.  Therapeutic efficacy therefore depends on not only how effectively a drug inhibits its target, but also the innate or adaptive functional redundancy of that target and its attendant pathway.  In this regard, the Myc transcription factors are intriguing therapeutic targets because they serve the unique and irreplaceable role of coordinating expression of the many diverse genes that, together, are required for somatic cell proliferation.  Furthermore, Myc expression is deregulated in most-perhaps all-cancers, underscoring its irreplaceable role in proliferation.  We previously showed in a preclin. mouse model of non-small-cell lung cancer that systemic Myc inhibition using the dominant-neg. Myc mutant Omomyc exerts a dramatic therapeutic impact, triggering rapid regression of tumors with only mild and fully reversible side effects.  Using protracted episodic expression of Omomyc, we now demonstrate that metronomic Myc inhibition not only contains Ras-driven lung tumors indefinitely, but also leads to their progressive eradication.  Hence, Myc does indeed serve a unique and nondegenerate role in lung tumor maintenance that cannot be complemented by any adaptive mechanism, even in the most aggressive p53-deficient tumors.  These data endorse Myc as a compelling cancer drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfpivJKOSErbVg90H21EOLACvtfcHk0lj8GILjCS4rZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVKkurY%253D&md5=0fc5aa1a0223f4b09da3e7bf114bdc56</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1101%2Fgad.205542.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.205542.112%26sid%3Dliteratum%253Aachs%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DWhitfield%26aufirst%3DJ.%2BR.%26aulast%3DSodir%26aufirst%3DN.%2BM.%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DSerrano%26aufirst%3DE.%26aulast%3DKarnezis%26aufirst%3DA.%2BN.%26aulast%3DSwigart%26aufirst%3DL.%2BB.%26aulast%3DEvan%26aufirst%3DG.%2BI.%26atitle%3DInhibition%2520of%2520Myc%2520Family%2520Proteins%2520eradicates%2520KRas-driven%2520Lung%2520Cancer%2520in%2520Mice%26jtitle%3DGenes%2520Dev.%26date%3D2013%26volume%3D27%26spage%3D504%26epage%3D513%26doi%3D10.1101%2Fgad.205542.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC Proteins for Tumor Therapy</span>. <i>Annu. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-030518-055826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1146%2Fannurev-cancerbio-030518-055826" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=61-75&author=E.+Wolfauthor=M.+Eilers&title=Targeting+MYC+Proteins+for+Tumor+Therapy&doi=10.1146%2Fannurev-cancerbio-030518-055826"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-030518-055826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-030518-055826%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DE.%26aulast%3DEilers%26aufirst%3DM.%26atitle%3DTargeting%2520MYC%2520Proteins%2520for%2520Tumor%2520Therapy%26jtitle%3DAnnu.%2520Rev.%2520Cancer%2520Biol.%26date%3D2020%26volume%3D4%26spage%3D61%26epage%3D75%26doi%3D10.1146%2Fannurev-cancerbio-030518-055826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieb, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Interaction with WDR5 promotes Target Gene Recognition and Tumorigenesis by MYC</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">440</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=25818646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=440-452&author=L.+R.+Thomasauthor=Q.+Wangauthor=B.+C.+Griebauthor=J.+Phanauthor=A.+M.+Foshageauthor=Q.+Sunauthor=E.+T.+Olejniczakauthor=T.+Clarkauthor=S.+Deyauthor=S.+Loreyauthor=B.+Alicieauthor=G.+C.+Howardauthor=B.+Cawthonauthor=K.+C.+Essauthor=C.+M.+Eischenauthor=Z.+Zhaoauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Interaction+with+WDR5+promotes+Target+Gene+Recognition+and+Tumorigenesis+by+MYC&doi=10.1016%2Fj.molcel.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span></div><div class="casAuthors">Thomas, Lance R.; Wang, Qingguo; Grieb, Brian C.; Phan, Jason; Foshage, Audra M.; Sun, Qi; Olejniczak, Edward T.; Clark, Travis; Dey, Soumyadeep; Lorey, Shelly; Alicie, Bethany; Howard, Gregory C.; Cawthon, Bryan; Ess, Kevin C.; Eischen, Christine M.; Zhao, Zhongming; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">440-452</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC is an oncoprotein transcription factor that is overexpressed in the majority of malignancies.  The oncogenic potential of MYC stems from its ability to bind regulatory sequences in thousands of target genes, which depends on interaction of MYC with its obligate partner, MAX.  Here, we show that broad assocn. of MYC with chromatin also depends on interaction with the WD40-repeat protein WDR5.  MYC binds WDR5 via an evolutionarily conserved "MYC box IIIb" motif that engages a shallow, hydrophobic cleft on the surface of WDR5.  Structure-guided mutations in MYC that disrupt interaction with WDR5 attenuate binding of MYC at ∼80% of its chromosomal locations and disable its ability to promote induced pluripotent stem cell formation and drive tumorigenesis.  Our data reveal WDR5 as a key determinant for MYC recruitment to chromatin and uncover a tractable target for the discovery of anticancer therapies against MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfifOnQkKDHrVg90H21EOLACvtfcHk0lj8GILjCS4rZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D&md5=329876ab3db1a16fb58bd5de7f1de7b9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DGrieb%26aufirst%3DB.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DLorey%26aufirst%3DS.%26aulast%3DAlicie%26aufirst%3DB.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DCawthon%26aufirst%3DB.%26aulast%3DEss%26aufirst%3DK.%2BC.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DInteraction%2520with%2520WDR5%2520promotes%2520Target%2520Gene%2520Recognition%2520and%2520Tumorigenesis%2520by%2520MYC%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D440%26epage%3D452%26doi%3D10.1016%2Fj.molcel.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnaccia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rellinger, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matlock, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">WDR5 is a conserved Regulator of Protein Synthesis Gene Expression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2924</span>– <span class="NLM_lpage">2941</span>, <span class="refDoi"> DOI: 10.1093/nar/gkaa051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fnar%2Fgkaa051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=31996893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVemsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=2924-2941&author=A.+F.+Bryanauthor=J.+Wangauthor=G.+C.+Howardauthor=A.+D.+Guarnacciaauthor=C.+M.+Woodleyauthor=E.+R.+Ahoauthor=E.+J.+Rellingerauthor=B.+K.+Matlockauthor=D.+K.+Flahertyauthor=S.+L.+Loreyauthor=D.+H.+Chungauthor=S.+W.+Fesikauthor=Q.+Liuauthor=A.+M.+Weissmillerauthor=W.+P.+Tansey&title=WDR5+is+a+conserved+Regulator+of+Protein+Synthesis+Gene+Expression&doi=10.1093%2Fnar%2Fgkaa051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 is a conserved regulator of protein synthesis gene expression</span></div><div class="casAuthors">Bryan, Audra F.; Wang, Jing; Howard, Gregory C.; Guarnaccia, Alissa D.; Woodley, Chase M.; Aho, Erin R.; Rellinger, Eric J.; Matlock, Brittany K.; Flaherty, David K.; Lorey, Shelly L.; Chung, Dai H.; Fesik, Stephen W.; Liu, Qi; Weissmiller, April M.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2924-2941</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">WDR5 is a highly-conserved nuclear protein that performs multiple scaffolding functions in the context of chromatin.  WDR5 is also a promising target for pharmacol. inhibition in cancer, with small mol. inhibitors of an arginine-binding pocket of WDR5 (the 'WIN' site) showing efficacy against a range of cancer cell lines in vitro.  Efforts to understand WDR5, or establish the mechanism of action of WIN site inhibitors, however, are stymied by its many functions in the nucleus, and a lack of knowledge of the conserved gene networks--if any--that are under its control.  Here, we have performed comparative genomic analyses to identify the conserved sites of WDR5 binding to chromatin, and the conserved genes regulated by WDR5, across a diverse panel of cancer cell lines.  We show that a specific cohort of protein synthesis genes (PSGs) are invariantly bound by WDR5, demonstrate that the WIN site anchors WDR5 to chromatin at these sites, and establish that PSGs are bona fide, acute, and persistent targets of WIN site blockade.  Together, these data reveal that WDR5 plays a predominant transcriptional role in biomass accumulation and provide further evidence that WIN site inhibitors act to repress gene networks linked to protein synthesis homeostasis.  In Saccharomyces cerevisiae, ribosomal protein gene (RPG) promoters display binding sites for either Rap1 or Abf1 transcription factors.  Unlike Rap1-assocd. promoters, the small cohort of Abf1-dependent RPGs (Abf1-RPGs) has not been extensively investigated.  We show that RPL3, RPL4B, RPP1A, RPS22B and RPS28A/B share a common promoter architecture, with an Abf1 site upstream of a conserved element matching the sequence recognized by Fhl1, a transcription factor which together with Ifh1 orchestrates Rap1-assocd. RPG regulation.  Abf1 and Fhl1 promoter assocn. was confirmed by ChIP and/or gel retardation assays.  Mutational anal. revealed a more severe requirement of Abf1 than Fhl1 binding sites for RPG transcription.  In the case of RPS22B an unusual Tbf1 binding site promoted both RPS22B and intron-hosted SNR44 expression.  Abf1-RPG down-regulation upon TOR pathway inhibition was much attenuated at defective mutant promoters unable to bind Abf1.  TORC1 inactivation caused the expected redn. of Ifh1 occupancy at RPS22B and RPL3 promoters, but unexpectedly it entailed largely increased Abf1 assocn. with Abf1-RPG promoters.  We present evidence that Abf1 recruitment upon nutritional stress, also obsd. for representative ribosome biogenesis genes, favors RPG transcriptional rescue upon nutrient replenishment, thus pointing to nutrient-regulated Abf1 dynamics at promoters as a novel mechanism in ribosome biogenesis control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNV7Dv7YF5lLVg90H21EOLACvtfcHk0lgwExYUy1CGhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVemsrrF&md5=52e65a8c20ac0c1f8db15ee2fa7d0225</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkaa051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkaa051%26sid%3Dliteratum%253Aachs%26aulast%3DBryan%26aufirst%3DA.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DGuarnaccia%26aufirst%3DA.%2BD.%26aulast%3DWoodley%26aufirst%3DC.%2BM.%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DRellinger%26aufirst%3DE.%2BJ.%26aulast%3DMatlock%26aufirst%3DB.%2BK.%26aulast%3DFlaherty%26aufirst%3DD.%2BK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DChung%26aufirst%3DD.%2BH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DWDR5%2520is%2520a%2520conserved%2520Regulator%2520of%2520Protein%2520Synthesis%2520Gene%2520Expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26spage%3D2924%26epage%3D2941%26doi%3D10.1093%2Fnar%2Fgkaa051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacón Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5-MYC Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11232</span>– <span class="NLM_lpage">11259</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11232-11259&author=J.+D.+Macdonaldauthor=S.+Chac%C3%B3n+Simonauthor=C.+Hanauthor=F.+Wangauthor=J.+G.+Shawauthor=J.+E.+Howesauthor=J.+Saiauthor=J.+P.+Yuhauthor=D.+Camperauthor=B.+M.+Alicieauthor=J.+Alvaradoauthor=S.+Nikharauthor=W.+Payneauthor=E.+R.+Ahoauthor=J.+A.+Bauerauthor=B.+Zhaoauthor=J.+Phanauthor=L.+R.+Thomasauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=A.+G.+Watersonauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+and+Optimization+of+Salicylic+Acid-Derived+Sulfonamide+Inhibitors+of+the+WD+Repeat-Containing+Protein+5-MYC+Protein-Protein+Interaction&doi=10.1021%2Facs.jmedchem.9b01411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5-MYC Protein-Protein Interaction</span></div><div class="casAuthors">Macdonald, Jonathan D.; Chacon Simon, Selena; Han, Changho; Wang, Feng; Shaw, J. Grace; Howes, Jennifer E.; Sai, Jiqing; Yuh, Joannes P.; Camper, Demarco; Alicie, Bethany M.; Alvarado, Joseph; Nikhar, Sameer; Payne, William; Aho, Erin R.; Bauer, Joshua A.; Zhao, Bin; Phan, Jason; Thomas, Lance R.; Rossanese, Olivia W.; Tansey, William P.; Waterson, Alex G.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11232-11259</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery.  Here, we present a new strategy toward the inhibition of MYC via the disruption of the protein-protein interaction between MYC and its chromatin cofactor WD Repeat-Contg. Protein 5.  Blocking the assocn. of these proteins is hypothesized to disrupt the localization of MYC to chromatin, thus disrupting the ability of MYC to sustain tumorigenesis.  Utilizing a high-throughput screening campaign and subsequent structure-guided design, we identify small-mol. inhibitors of this interaction with potent in vitro binding affinity and report structurally related neg. controls that can be used to study the effect of this disruption.  Our work suggests that disruption of this protein-protein interaction may provide a path toward an effective approach for the treatment of multiple tumors and anticipate that the mols. disclosed can be used as starting points for future efforts toward compds. with improved drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHEVMRoQUUdbVg90H21EOLACvtfcHk0lgwExYUy1CGhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLfI&md5=e7bb66ca44a1d04f2240fc9c8309b362</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01411%26sid%3Dliteratum%253Aachs%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DHowes%26aufirst%3DJ.%2BE.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DAlicie%26aufirst%3DB.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DNikhar%26aufirst%3DS.%26aulast%3DPayne%26aufirst%3DW.%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Salicylic%2520Acid-Derived%2520Sulfonamide%2520Inhibitors%2520of%2520the%2520WD%2520Repeat-Containing%2520Protein%25205-MYC%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11232%26epage%3D11259%26doi%3D10.1021%2Facs.jmedchem.9b01411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gherardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmadibrata, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trahair, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shohet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüttelmaier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braithwaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span> <span> </span><span class="NLM_article-title">WDR5 supports an N-Myc Transcriptional Complex that drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">5143</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1158%2F0008-5472.CAN-15-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26471359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5143-5154&author=Y.+Sunauthor=J.+L.+Bellauthor=D.+Carterauthor=S.+Gherardiauthor=R.+C.+Poulosauthor=G.+Milazzoauthor=J.+W.+H.+Wongauthor=R.+Al-Awarauthor=A.+E.+Teeauthor=P.+Y.+Liuauthor=B.+Liuauthor=B.+Atmadibrataauthor=M.+Wongauthor=T.+Trahairauthor=Q.+Zhaoauthor=J.+M.+Shohetauthor=Y.+Hauptauthor=J.+H.+Schulteauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=K.+L.+MacKenzieauthor=S.+H%C3%BCttelmaierauthor=G.+Periniauthor=G.+M.+Marshallauthor=A.+Braithwaiteauthor=T.+Liu&title=WDR5+supports+an+N-Myc+Transcriptional+Complex+that+drives+a+Protumorigenic+Gene+Expression+Signature+in+Neuroblastoma&doi=10.1158%2F0008-5472.CAN-15-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Supports an N-Myc Transcriptional Complex That Drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span></div><div class="casAuthors">Sun, Yuting; Bell, Jessica L.; Carter, Daniel; Gherardi, Samuele; Poulos, Rebecca C.; Milazzo, Giorgio; Wong, Jason W. H.; Al-Awar, Rima; Tee, Andrew E.; Liu, Pei Y.; Liu, Bing; Atmadibrata, Bernard; Wong, Matthew; Trahair, Toby; Zhao, Quan; Shohet, Jason M.; Haupt, Ygal; Schulte, Johannes H.; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; MacKenzie, Karen L.; Huettelmaier, Stefan; Perini, Giovanni; Marshall, Glenn M.; Braithwaite, Antony; Liu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5143-5154</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MYCN gene amplification in neuroblastoma drives a gene expression program that correlates strongly with aggressive disease.  Mechanistically, trimethylation of histone H3 lysine 4 (H3K4) at target gene promoters is a strict prerequisite for this transcriptional program to be enacted.  WDR5 is a histone H3K4 presenter that has been found to have an essential role in H3K4 trimethylation.  For this reason, in this study, we investigated the relationship between WDR5-mediated H3K4 trimethylation and N-Myc transcriptional programs in neuroblastoma cells.  N-Myc upregulated WDR5 expression in neuroblastoma cells.  Gene expression anal. revealed that WDR5 target genes included those with MYC-binding elements at promoters such as MDM2.  We showed that WDR5 could form a protein complex at the MDM2 promoter with N-Myc, but not p53, leading to histone H3K4 trimethylation and activation of MDM2 transcription.  RNAi-mediated attenuation of WDR5 upregulated expression of wild-type but not mutant p53, an effect assocd. with growth inhibition and apoptosis.  Similarly, a small-mol. antagonist of WDR5 reduced N-Myc/WDR5 complex formation, N-Myc target gene expression, and cell growth in neuroblastoma cells.  In MYCN-transgenic mice, WDR5 was overexpressed in precancerous ganglion and neuroblastoma cells compared with normal ganglion cells.  Clin., elevated levels of WDR5 in neuroblastoma specimens were an independent predictor of poor overall survival.  Overall, our results identify WDR5 as a key cofactor for N-Myc-regulated transcriptional activation and tumorigenesis and as a novel therapeutic target for MYCN-amplified neuroblastomas.  Cancer Res; 75(23); 5143-54. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdt2xhQOC6ibVg90H21EOLACvtfcHk0lgwExYUy1CGhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI&md5=26513b24ba2ba6b82cc691534f5939cd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0423%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DCarter%26aufirst%3DD.%26aulast%3DGherardi%26aufirst%3DS.%26aulast%3DPoulos%26aufirst%3DR.%2BC.%26aulast%3DMilazzo%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTee%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DP.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DAtmadibrata%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DTrahair%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DShohet%26aufirst%3DJ.%2BM.%26aulast%3DHaupt%26aufirst%3DY.%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DH%25C3%25BCttelmaier%26aufirst%3DS.%26aulast%3DPerini%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DG.%2BM.%26aulast%3DBraithwaite%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DT.%26atitle%3DWDR5%2520supports%2520an%2520N-Myc%2520Transcriptional%2520Complex%2520that%2520drives%2520a%2520Protumorigenic%2520Gene%2520Expression%2520Signature%2520in%2520Neuroblastoma%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5143%26epage%3D5154%26doi%3D10.1158%2F0008-5472.CAN-15-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carugo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genovese, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nezi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicalese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettazzoni, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akdemir, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bristow, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estecio, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellino, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Francesco, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alesio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelicci, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanfrancone, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G. F.</span></span> <span> </span><span class="NLM_article-title">In Vivo Functional Platform Targeting Patient-derived Xenografts identifies WDR5-Myc Association as a critical Determinant of Pancreatic Cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.celrep.2016.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27320920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVWltrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=133-147&author=A.+Carugoauthor=G.+Genoveseauthor=S.+Sethauthor=L.+Neziauthor=J.+L.+Roseauthor=D.+Bossiauthor=A.+Cicaleseauthor=P.+K.+Shahauthor=A.+Vialeauthor=P.+F.+Pettazzoniauthor=K.+C.+Akdemirauthor=C.+A.+Bristowauthor=F.+S.+Robinsonauthor=J.+Tepperauthor=N.+Sanchezauthor=S.+Guptaauthor=M.+R.+Estecioauthor=V.+Giulianiauthor=G.+I.+Dellinoauthor=L.+Rivaauthor=W.+Yaoauthor=M.+E.+Di+Francescoauthor=T.+Greenauthor=C.+D%E2%80%99Alesioauthor=D.+Cortiauthor=Y.+Kangauthor=P.+Jonesauthor=H.+Wangauthor=J.+B.+Flemingauthor=A.+Maitraauthor=P.+G.+Pelicciauthor=L.+Chinauthor=R.+A.+DePinhoauthor=L.+Lanfranconeauthor=T.+P.+Heffernanauthor=G.+F.+Draetta&title=In+Vivo+Functional+Platform+Targeting+Patient-derived+Xenografts+identifies+WDR5-Myc+Association+as+a+critical+Determinant+of+Pancreatic+Cancer&doi=10.1016%2Fj.celrep.2016.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Functional Platform Targeting Patient-Derived Xenografts Identifies WDR5-Myc Association as a Critical Determinant of Pancreatic Cancer</span></div><div class="casAuthors">Carugo, Alessandro; Genovese, Giannicola; Seth, Sahil; Nezi, Luigi; Rose, Johnathon Lynn; Bossi, Daniela; Cicalese, Angelo; Shah, Parantu Krushnakant; Viale, Andrea; Pettazzoni, Piergiorgio Francesco; Akdemir, Kadir Caner; Bristow, Christopher Aaron; Robinson, Frederick Scott; Tepper, James; Sanchez, Nora; Gupta, Sonal; Estecio, Marcos Roberto; Giuliani, Virginia; Dellino, Gaetano Ivan; Riva, Laura; Yao, Wantong; Di Francesco, Maria Emilia; Green, Tessa; D'Alesio, Carolina; Corti, Denise; Kang, Ya-an; Jones, Philip; Wang, Huamin; Fleming, Jason Bates; Maitra, Anirban; Pelicci, Pier Giuseppe; Chin, Lynda; DePinho, Ronald Anthony; Lanfrancone, Luisa; Heffernan, Timothy Paul; Draetta, Giulio Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-147</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Current treatment regimens for pancreatic ductal adenocarcinoma (PDAC) yield poor 5-yr survival, emphasizing the crit. need to identify druggable targets essential for PDAC maintenance.  We developed an unbiased and in vivo target discovery approach to identify mol. vulnerabilities in low-passage and patient-derived PDAC xenografts or genetically engineered mouse model-derived allografts.  Focusing on epigenetic regulators, we identified WDR5, a core member of the COMPASS histone H3 Lys4 (H3K4) MLL (1-4) methyltransferase complex, as a top tumor maintenance hit required across multiple human and mouse tumors.  Mechanistically, WDR5 functions to sustain proper execution of DNA replication in PDAC cells, as previously suggested by replication stress studies involving MLL1, and c-Myc, also found to interact with WDR5.  We indeed demonstrate that interaction with c-Myc is crit. for this function.  By showing that ATR inhibition mimicked the effects of WDR5 suppression, these data provide rationale to test ATR and WDR5 inhibitors for activity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgPl1yxAd8rVg90H21EOLACvtfcHk0lgvbGcGpsGeQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVWltrnK&md5=7b9a7dea457be616aa4f103740b2be02</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCarugo%26aufirst%3DA.%26aulast%3DGenovese%26aufirst%3DG.%26aulast%3DSeth%26aufirst%3DS.%26aulast%3DNezi%26aufirst%3DL.%26aulast%3DRose%26aufirst%3DJ.%2BL.%26aulast%3DBossi%26aufirst%3DD.%26aulast%3DCicalese%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DP.%2BK.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DPettazzoni%26aufirst%3DP.%2BF.%26aulast%3DAkdemir%26aufirst%3DK.%2BC.%26aulast%3DBristow%26aufirst%3DC.%2BA.%26aulast%3DRobinson%26aufirst%3DF.%2BS.%26aulast%3DTepper%26aufirst%3DJ.%26aulast%3DSanchez%26aufirst%3DN.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DEstecio%26aufirst%3DM.%2BR.%26aulast%3DGiuliani%26aufirst%3DV.%26aulast%3DDellino%26aufirst%3DG.%2BI.%26aulast%3DRiva%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DDi%2BFrancesco%26aufirst%3DM.%2BE.%26aulast%3DGreen%26aufirst%3DT.%26aulast%3DD%25E2%2580%2599Alesio%26aufirst%3DC.%26aulast%3DCorti%26aufirst%3DD.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFleming%26aufirst%3DJ.%2BB.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DPelicci%26aufirst%3DP.%2BG.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DLanfrancone%26aufirst%3DL.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DDraetta%26aufirst%3DG.%2BF.%26atitle%3DIn%2520Vivo%2520Functional%2520Platform%2520Targeting%2520Patient-derived%2520Xenografts%2520identifies%2520WDR5-Myc%2520Association%2520as%2520a%2520critical%2520Determinant%2520of%2520Pancreatic%2520Cancer%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D133%26epage%3D147%26doi%3D10.1016%2Fj.celrep.2016.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suganuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J. L.</span></span> <span> </span><span class="NLM_article-title">Diverse Functions of WD40 Repeat Proteins in Histone Recognition</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1268</span>, <span class="refDoi"> DOI: 10.1101/gad.1676208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1101%2Fgad.1676208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18483215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVWmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1265-1268&author=T.+Suganumaauthor=S.+G.+Pattendenauthor=J.+L.+Workman&title=Diverse+Functions+of+WD40+Repeat+Proteins+in+Histone+Recognition&doi=10.1101%2Fgad.1676208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse functions of WD40 repeat proteins in histone recognition</span></div><div class="casAuthors">Suganuma, Tamaki; Pattenden, Samantha G.; Workman, Jerry L.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1265-1268</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  WD40 repeat proteins have been shown to bind the histone H3 tail at the center of their β-propeller structure.  In contrast, in this issue of Genes & Development, J. J. Song and colleagues (ibid., 1313-1318) demonstrate that the WD40 repeat protein p55 binds a structured region of H4 through a novel binding pocket on the side of β-propeller, illustrating a diversity of histone recognition by WD40 repeat proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomI1QwxcjGi7Vg90H21EOLACvtfcHk0lgvbGcGpsGeQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVWmsb0%253D&md5=9bf2255697ab857bb71ec76e6516405c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.1676208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1676208%26sid%3Dliteratum%253Aachs%26aulast%3DSuganuma%26aufirst%3DT.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DWorkman%26aufirst%3DJ.%2BL.%26atitle%3DDiverse%2520Functions%2520of%2520WD40%2520Repeat%2520Proteins%2520in%2520Histone%2520Recognition%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D1265%26epage%3D1268%26doi%3D10.1101%2Fgad.1676208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">A conserved Arginine-containing Motif crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32162</span>– <span class="NLM_lpage">32175</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+Arginine-containing+Motif+crucial+for+the+Assembly+and+Enzymatic+Activity+of+the+Mixed+Lineage+Leukemia+Protein-1+Core+Complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0ljOz4l9xM2_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520Arginine-containing%2520Motif%2520crucial%2520for%2520the%2520Assembly%2520and%2520Enzymatic%2520Activity%2520of%2520the%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the Von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules to disrupt the VHL/HIF-1α Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+Von+Hippel-Lindau+E3+Ubiquitin+Ligase+Using+Small+Molecules+to+disrupt+the+VHL%2FHIF-1%CE%B1+Interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0ljOz4l9xM2_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3Dvan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520Von%2520Hippel-Lindau%2520E3%2520Ubiquitin%2520Ligase%2520Using%2520Small%2520Molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%25CE%25B1%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozilovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diebold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudvarski Stankovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baluapuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönemann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhandare, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotriffer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E.</span></span> <span> </span><span class="NLM_article-title">PROTAC-mediated Degradation reveals a non-catalytic Function of AURORA-A Kinase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1179</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-00652-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fs41589-020-00652-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32989298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFegurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1179-1188&author=B.+Adhikariauthor=J.+Bozilovicauthor=M.+Dieboldauthor=J.+D.+Schwarzauthor=J.+Hofstetterauthor=M.+Schr%C3%B6derauthor=M.+Waniorauthor=A.+Narainauthor=M.+Vogtauthor=N.+Dudvarski+Stankovicauthor=A.+Baluapuriauthor=L.+Sch%C3%B6nemannauthor=L.+Eingauthor=P.+Bhandareauthor=B.+Kusterauthor=A.+Schlosserauthor=S.+Heinzlmeirauthor=C.+Sotrifferauthor=S.+Knappauthor=E.+Wolf&title=PROTAC-mediated+Degradation+reveals+a+non-catalytic+Function+of+AURORA-A+Kinase&doi=10.1038%2Fs41589-020-00652-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-mediated degradation reveals a non-catalytic function of AURORA-A kinase</span></div><div class="casAuthors">Adhikari, Bikash; Bozilovic, Jelena; Diebold, Mathias; Schwarz, Jessica Denise; Hofstetter, Julia; Schroeder, Martin; Wanior, Marek; Narain, Ashwin; Vogt, Markus; Dudvarski Stankovic, Nevenka; Baluapuri, Apoorva; Schoenemann, Lars; Eing, Lorenz; Bhandare, Pranjali; Kuster, Bernhard; Schlosser, Andreas; Heinzlmeir, Stephanie; Sotriffer, Christoph; Knapp, Stefan; Wolf, Elmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1179-1188</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The mitotic kinase AURORA-A is essential for cell cycle progression and is considered a priority cancer target.  Although the catalytic activity of AURORA-A is essential for its mitotic function, recent reports indicate an addnl. non-catalytic function, which is difficult to target by conventional small mols.  We therefore developed a series of chem. degraders (PROTACs) by connecting a clin. kinase inhibitor of AURORA-A to E3 ligase-binding mols. (for example, thalidomide).  One degrader induced rapid, durable and highly specific degrdn. of AURORA-A.  In addn., we found that the degrader complex was stabilized by cooperative binding between AURORA-A and CEREBLON.  Degrader-mediated AURORA-A depletion caused an S-phase defect, which is not the cell cycle effect obsd. upon kinase inhibition, supporting an important non-catalytic function of AURORA-A during DNA replication.  AURORA-A degrdn. induced rampant apoptosis in cancer cell lines and thus represents a versatile starting point for developing new therapeutics to counter AURORA-A function in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSD0j67urh7Vg90H21EOLACvtfcHk0ljOz4l9xM2_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFegurvM&md5=3cdb2fa0d7c7e800cd3936765218b5d7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00652-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00652-y%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DB.%26aulast%3DBozilovic%26aufirst%3DJ.%26aulast%3DDiebold%26aufirst%3DM.%26aulast%3DSchwarz%26aufirst%3DJ.%2BD.%26aulast%3DHofstetter%26aufirst%3DJ.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DNarain%26aufirst%3DA.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DDudvarski%2BStankovic%26aufirst%3DN.%26aulast%3DBaluapuri%26aufirst%3DA.%26aulast%3DSch%25C3%25B6nemann%26aufirst%3DL.%26aulast%3DEing%26aufirst%3DL.%26aulast%3DBhandare%26aufirst%3DP.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DSchlosser%26aufirst%3DA.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DSotriffer%26aufirst%3DC.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DE.%26atitle%3DPROTAC-mediated%2520Degradation%2520reveals%2520a%2520non-catalytic%2520Function%2520of%2520AURORA-A%2520Kinase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1179%26epage%3D1188%26doi%3D10.1038%2Fs41589-020-00652-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided Design and Optimization of Small Molecules targeting the Protein-Protein Interaction between the Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in vitro Nanomolar Affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+Design+and+Optimization+of+Small+Molecules+targeting+the+Protein-Protein+Interaction+between+the+Von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase+and+the+Hypoxia+Inducible+Factor+%28HIF%29+Alpha+Subunit+with+in+vitro+Nanomolar+Affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lh8HSAs6kxnYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3Dvan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520Design%2520and%2520Optimization%2520of%2520Small%2520Molecules%2520targeting%2520the%2520Protein-Protein%2520Interaction%2520between%2520the%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%2520and%2520the%2520Hypoxia%2520Inducible%2520Factor%2520%2528HIF%2529%2520Alpha%2520Subunit%2520with%2520in%2520vitro%2520Nanomolar%2520Affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The Use of Differential Scanning Fluorimetry to detect Ligand Interactions that promote Protein Stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+Use+of+Differential+Scanning+Fluorimetry+to+detect+Ligand+Interactions+that+promote+Protein+Stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lh8HSAs6kxnYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520Use%2520of%2520Differential%2520Scanning%2520Fluorimetry%2520to%2520detect%2520Ligand%2520Interactions%2520that%2520promote%2520Protein%2520Stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid Measurement of Binding Constants and Heats of Binding using a new Titration Calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+Measurement+of+Binding+Constants+and+Heats+of+Binding+using+a+new+Titration+Calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lh8HSAs6kxnYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520Measurement%2520of%2520Binding%2520Constants%2520and%2520Heats%2520of%2520Binding%2520using%2520a%2520new%2520Titration%2520Calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214.e11</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214.e11&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+Wide-Spectrum+Kinase+Profiling+in+Live+Cells+for+Assessing+the+Effect+of+Cellular+ATP+on+Target+Engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0li0IEAt8e6Sfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520Wide-Spectrum%2520Kinase%2520Profiling%2520in%2520Live%2520Cells%2520for%2520Assessing%2520the%2520Effect%2520of%2520Cellular%2520ATP%2520on%2520Target%2520Engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214.e11%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target Engagement and Drug Residence Time can be observed in living Cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">10091</span> <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+Engagement+and+Drug+Residence+Time+can+be+observed+in+living+Cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0li0IEAt8e6Sfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520Engagement%2520and%2520Drug%2520Residence%2520Time%2520can%2520be%2520observed%2520in%2520living%2520Cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoefer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boychenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">Robust Microflow LC-MS/MS for Proteome Analysis: 38 000 Runs and Counting</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">3686</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.1c00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.1c00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktFemtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2021&pages=3686-3690&author=Y.+Bianauthor=F.+P.+Bayerauthor=Y.-C.+Changauthor=C.+Mengauthor=S.+Hoeferauthor=N.+Dengauthor=R.+Zhengauthor=O.+Boychenkoauthor=B.+Kuster&title=Robust+Microflow+LC-MS%2FMS+for+Proteome+Analysis%3A+38+000+Runs+and+Counting&doi=10.1021%2Facs.analchem.1c00257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Robust Microflow LC-MS/MS for Proteome Analysis: 38 000 Runs and Counting</span></div><div class="casAuthors">Bian, Yangyang; Bayer, Florian P.; Chang, Yun-Chien; Meng, Chen; Hoefer, Stefanie; Deng, Nan; Zheng, Runsheng; Boychenko, Oleksandr; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3686-3690</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Microflow liq. chromatog. tandem mass spectrometry (μLC-MS/MS) is becoming a viable alternative to nanoflow LC-MS/MS for the anal. of proteomes.  We have recently demonstrated the potential of such a system operating with a 1 mm i.d. x 150 mm column and at a flow rate of 50 μL/min for high-throughput applications.  On the basis of the anal. of ~ 38 000 samples measured on two instruments over the past two years, we now show that the approach is extremely robust.  Up to 1500 analyses were performed within one month, and >14 000 samples could be analyzed on a single column without loss of chromatog. performance.  Samples included proteomes of cell lines, tissues, and human body fluids, which were analyzed with or without prior peptide fractionation or stable isotope labeling.  We show that the μLC-MS/MS system is capable of measuring 2600 proteins from undepleted human plasma and ~ 5000 proteins from crude human urine in 1 day, demonstrating its potential for in-depth as well as high-throughput clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8VCCnqHc2BbVg90H21EOLACvtfcHk0li0IEAt8e6Sfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXktFemtbk%253D&md5=5e3e071c1e167a43b0a2a2a2930e4671</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.1c00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.1c00257%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DY.%26aulast%3DBayer%26aufirst%3DF.%2BP.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DHoefer%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBoychenko%26aufirst%3DO.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DRobust%2520Microflow%2520LC-MS%252FMS%2520for%2520Proteome%2520Analysis%253A%252038%2520000%2520Runs%2520and%2520Counting%26jtitle%3DAnal.%2520Chem.%26date%3D2021%26volume%3D93%26spage%3D3686%26epage%3D3690%26doi%3D10.1021%2Facs.analchem.1c00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baynham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boychenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">Robust, Reproducible and Quantitative Analysis of Thousands of Proteomes by Micro-Flow LC-MS/MS</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">157</span> <span class="refDoi"> DOI: 10.1038/s41467-019-13973-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fs41467-019-13973-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=31919466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1ynsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=Y.+Bianauthor=R.+Zhengauthor=F.+P.+Bayerauthor=C.+Wongauthor=Y.-C.+Changauthor=C.+Mengauthor=D.+P.+Zolgauthor=M.+Reineckeauthor=J.+Zechaauthor=S.+Wiechmannauthor=S.+Heinzlmeirauthor=J.+Scherrauthor=B.+Hemmerauthor=M.+Baynhamauthor=A.-C.+Gingrasauthor=O.+Boychenkoauthor=B.+Kuster&title=Robust%2C+Reproducible+and+Quantitative+Analysis+of+Thousands+of+Proteomes+by+Micro-Flow+LC-MS%2FMS&doi=10.1038%2Fs41467-019-13973-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Robust, reproducible and quantitative analysis of thousands of proteomes by micro-flow LC-MS/MS</span></div><div class="casAuthors">Bian, Yangyang; Zheng, Runsheng; Bayer, Florian P.; Wong, Cassandra; Chang, Yun-Chien; Meng, Chen; Zolg, Daniel P.; Reinecke, Maria; Zecha, Jana; Wiechmann, Svenja; Heinzlmeir, Stephanie; Scherr, Johannes; Hemmer, Bernhard; Baynham, Mike; Gingras, Anne-Claude; Boychenko, Oleksandr; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nano-flow liq. chromatog. tandem mass spectrometry (nano-flow LC-MS/MS) is the mainstay in proteome research because of its excellent sensitivity but often comes at the expense of robustness.  Here we show that micro-flow LC-MS/MS using a 1 x 150 mm column shows excellent reproducibility of chromatog. retention time (<0.3% coeff. of variation, CV) and protein quantification (<7.5% CV) using data from >2000 samples of human cell lines, tissues and body fluids.  Deep proteome anal. identifies >9000 proteins and >120,000 peptides in 16 h and sample multiplexing using tandem mass tags increases throughput to 11 proteomes in 16 h.  The system identifies >30,000 phosphopeptides in 12 h and protein-protein or protein-drug interaction expts. can be analyzed in 20 min per sample.  We show that the same column can be used to analyze >7500 samples without apparent loss of performance.  This study demonstrates that micro-flow LC-MS/MS is suitable for a broad range of proteomic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOFIaSfJ3g17Vg90H21EOLACvtfcHk0liP2PQpXqQMtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1ynsA%253D%253D&md5=70f125faf86cdc8ff71b86241636b881</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-13973-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-13973-x%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBayer%26aufirst%3DF.%2BP.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DWiechmann%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DScherr%26aufirst%3DJ.%26aulast%3DHemmer%26aufirst%3DB.%26aulast%3DBaynham%26aufirst%3DM.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DBoychenko%26aufirst%3DO.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DRobust%252C%2520Reproducible%2520and%2520Quantitative%2520Analysis%2520of%2520Thousands%2520of%2520Proteomes%2520by%2520Micro-Flow%2520LC-MS%252FMS%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-019-13973-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Médard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemeer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">DMSO enhances Electrospray Response, boosting Sensitivity of Proteomic Experiments</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">991</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnmeth.2610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23975139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWkurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=989-991&author=H.+Hahneauthor=F.+Pachlauthor=B.+Ruprechtauthor=S.+K.+Maierauthor=S.+Klaegerauthor=D.+Helmauthor=G.+M%C3%A9dardauthor=M.+Wilmauthor=S.+Lemeerauthor=B.+Kuster&title=DMSO+enhances+Electrospray+Response%2C+boosting+Sensitivity+of+Proteomic+Experiments&doi=10.1038%2Fnmeth.2610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DMSO enhances electrospray response, boosting sensitivity of proteomic experiments</span></div><div class="casAuthors">Hahne, Hannes; Pachl, Fiona; Ruprecht, Benjamin; Maier, Stefan K.; Klaeger, Susan; Helm, Dominic; Medard, Guillaume; Wilm, Matthias; Lemeer, Simone; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">989-991</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report that low percentages of dimethylsulfoxide (DMSO) in liq. chromatog. solvents lead to a strong enhancement of electrospray ionization of peptides, improving the sensitivity of protein identification in bottom-up proteomics by up to tenfold.  The method can be easily implemented on any LC-MS/MS system without modification to hardware or software and at no addnl. cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYxQsbxyeCubVg90H21EOLACvtfcHk0liP2PQpXqQMtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWkurjJ&md5=dd957dd264736e8bc48b66df2355363e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2610%26sid%3Dliteratum%253Aachs%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DMaier%26aufirst%3DS.%2BK.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DM%25C3%25A9dard%26aufirst%3DG.%26aulast%3DWilm%26aufirst%3DM.%26aulast%3DLemeer%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DDMSO%2520enhances%2520Electrospray%2520Response%252C%2520boosting%2520Sensitivity%2520of%2520Proteomic%2520Experiments%26jtitle%3DNat.%2520Methods%26date%3D2013%26volume%3D10%26spage%3D989%26epage%3D991%26doi%3D10.1038%2Fnmeth.2610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">MaxQuant enables high Peptide Identification Rates, Individualized p.p.b.-Range Mass Accuracies and Proteome-wide Protein Quantification</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1372</span>, <span class="refDoi"> DOI: 10.1038/nbt.1511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnbt.1511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=19029910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1367-1372&author=J.+Coxauthor=M.+Mann&title=MaxQuant+enables+high+Peptide+Identification+Rates%2C+Individualized+p.p.b.-Range+Mass+Accuracies+and+Proteome-wide+Protein+Quantification&doi=10.1038%2Fnbt.1511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</span></div><div class="casAuthors">Cox, Juergen; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1367-1372</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Efficient anal. of very large amts. of raw data for peptide identification and protein quantification is a principal challenge in mass spectrometry (MS)-based proteomics.  Here we describe MaxQuant, an integrated suite of algorithms specifically developed for high-resoln., quant. MS data.  Using correlation anal. and graph theory, MaxQuant detects peaks, isotope clusters and stable amino acid isotope-labeled (SILAC) peptide pairs as three-dimensional objects in m/z, elution time and signal intensity space.  By integrating multiple mass measurements and correcting for linear and nonlinear mass offsets, we achieve mass accuracy in the p.p.b. range, a sixfold increase over std. techniques.  We increase the proportion of identified fragmentation spectra to 73% for SILAC peptide pairs via unambiguous assignment of isotope and missed-cleavage state and individual mass precision.  MaxQuant automatically quantifies several hundred thousand peptides per SILAC-proteome expt. and allows statistically robust identification and quantification of >4000 proteins in mammalian cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzVwiKHxD8XbVg90H21EOLACvtfcHk0liP2PQpXqQMtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWjtLzJ&md5=675d31ca84e3a7e4fb9bdd601d8075ea</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1511%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DMaxQuant%2520enables%2520high%2520Peptide%2520Identification%2520Rates%252C%2520Individualized%2520p.p.b.-Range%2520Mass%2520Accuracies%2520and%2520Proteome-wide%2520Protein%2520Quantification%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D1367%26epage%3D1372%26doi%3D10.1038%2Fnbt.1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhauser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheltema, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">Andromeda: a Peptide Search Engine integrated into the MaxQuant Environment</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1794</span>– <span class="NLM_lpage">1805</span>, <span class="refDoi"> DOI: 10.1021/pr101065j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr101065j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gis74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=1794-1805&author=J.+Coxauthor=N.+Neuhauserauthor=A.+Michalskiauthor=R.+A.+Scheltemaauthor=J.+V.+Olsenauthor=M.+Mann&title=Andromeda%3A+a+Peptide+Search+Engine+integrated+into+the+MaxQuant+Environment&doi=10.1021%2Fpr101065j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Andromeda: A Peptide Search Engine Integrated into the MaxQuant Environment</span></div><div class="casAuthors">Cox, Juergen; Neuhauser, Nadin; Michalski, Annette; Scheltema, Richard A.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1794-1805</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key step in mass spectrometry (MS)-based proteomics is the identification of peptides in sequence databases by their fragmentation spectra.  Here the authors describe Andromeda, a novel peptide search engine using a probabilistic scoring model.  On proteome data, Andromeda performs as well as Mascot, a widely used com. search engine, as judged by sensitivity and specificity anal. based on target decoy searches.  Furthermore, it can handle data with arbitrarily high fragment mass accuracy, is able to assign and score complex patterns of post-translational modifications, such as highly phosphorylated peptides, and accommodates extremely large databases.  The algorithms of Andromeda are provided.  Andromeda can function independently or as an integrated search engine of the widely used MaxQuant computational proteomics platform and both are freely available at www.maxquant.org.  The combination enables anal. of large data sets in a simple anal. workflow on a desktop computer.  For searching individual spectra Andromeda is also accessible via a web server.  The authors demonstrate the flexibility of the system by implementing the capability to identify cofragmented peptides, significantly improving the total no. of identified peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4zTprqNti7Vg90H21EOLACvtfcHk0ljF5vvF52ejrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gis74%253D&md5=7587da6364fe0ff020e3dbf1d80bb22f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fpr101065j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr101065j%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DNeuhauser%26aufirst%3DN.%26aulast%3DMichalski%26aufirst%3DA.%26aulast%3DScheltema%26aufirst%3DR.%2BA.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DAndromeda%253A%2520a%2520Peptide%2520Search%2520Engine%2520integrated%2520into%2520the%2520MaxQuant%2520Environment%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D1794%26epage%3D1805%26doi%3D10.1021%2Fpr101065j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyanova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinitcyn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span> <span> </span><span class="NLM_article-title">The Perseus Computational Platform for comprehensive Analysis of (Prote)omics Data</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">740</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnmeth.3901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27348712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKntbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=731-740&author=S.+Tyanovaauthor=T.+Temuauthor=P.+Sinitcynauthor=A.+Carlsonauthor=M.+Y.+Heinauthor=T.+Geigerauthor=M.+Mannauthor=J.+Cox&title=The+Perseus+Computational+Platform+for+comprehensive+Analysis+of+%28Prote%29omics+Data&doi=10.1038%2Fnmeth.3901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The Perseus computational platform for comprehensive analysis of (prote)omics data</span></div><div class="casAuthors">Tyanova, Stefka; Temu, Tikira; Sinitcyn, Pavel; Carlson, Arthur; Hein, Marco Y.; Geiger, Tamar; Mann, Matthias; Cox, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">731-740</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A main bottleneck in proteomics is the downstream biol. anal. of highly multivariate quant. protein abundance data generated using mass-spectrometry-based anal.  We developed the Perseus software platform (http://www.perseus-framework.org) to support biol. and biomedical researchers in interpreting protein quantification, interaction and post-translational modification data.  Perseus contains a comprehensive portfolio of statistical tools for high-dimensional omics data anal. covering normalization, pattern recognition, time-series anal., cross-omics comparisons and multiple-hypothesis testing.  A machine learning module supports the classification and validation of patient groups for diagnosis and prognosis, and it also detects predictive protein signatures.  Central to Perseus is a user-friendly, interactive workflow environment that provides complete documentation of computational methods used in a publication.  All activities in Perseus are realized as plugins, and users can extend the software by programming their own, which can be shared through a plugin store.  We anticipate that Perseus's arsenal of algorithms and its intuitive usability will empower interdisciplinary anal. of complex large data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVXoplfOxdPLVg90H21EOLACvtfcHk0ljF5vvF52ejrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKntbnN&md5=f8c3e2876e4d724518054bb1a2d1e6ee</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3901%26sid%3Dliteratum%253Aachs%26aulast%3DTyanova%26aufirst%3DS.%26aulast%3DTemu%26aufirst%3DT.%26aulast%3DSinitcyn%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DA.%26aulast%3DHein%26aufirst%3DM.%2BY.%26aulast%3DGeiger%26aufirst%3DT.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DJ.%26atitle%3DThe%2520Perseus%2520Computational%2520Platform%2520for%2520comprehensive%2520Analysis%2520of%2520%2528Prote%2529omics%2520Data%26jtitle%3DNat.%2520Methods%26date%3D2016%26volume%3D13%26spage%3D731%26epage%3D740%26doi%3D10.1038%2Fnmeth.3901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vizcaíno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csordas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del-Toro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dianes, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavidas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Riverol, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ternent, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span> <span> </span><span class="NLM_article-title">2016 Update of the PRIDE Database and its related Tools</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D447</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fnar%2Fgkv1145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26527722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D447-56&author=J.+A.+Vizca%C3%ADnoauthor=A.+Csordasauthor=N.+del-Toroauthor=J.+A.+Dianesauthor=J.+Grissauthor=I.+Lavidasauthor=G.+Mayerauthor=Y.+Perez-Riverolauthor=F.+Reisingerauthor=T.+Ternentauthor=Q.-W.+Xuauthor=R.+Wangauthor=H.+Hermjakob&title=2016+Update+of+the+PRIDE+Database+and+its+related+Tools&doi=10.1093%2Fnar%2Fgkv1145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">2016 update of the PRIDE database and its related tools</span></div><div class="casAuthors">Vizcaino, Juan Antonio; Csordas, Attila; del-Toro, Noemi; Dianes, Jose A.; Griss, Johannes; Lavidas, Ilias; Mayer, Gerhard; Perez-Riverol, Yasset; Reisinger, Florian; Ternent, Tobias; Xu, Qing-Wei; Wang, Rui; Hermjakob, Henning</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D447-D456</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The PRoteomics IDEntifications (PRIDE) database is one of the world-leading data repositories of mass spectrometry (MS)-based proteomics data.  Since the beginning of 2014, PRIDE Archive (http://www.ebi.ac.uk/pride/archive/) is the new PRIDE archival system, replacing the original PRIDE database.  Here we summarize the developments in PRIDE resources and related tools since the previous update manuscript in the Database Issue in 2013.  PRIDE Archive constitutes a complete redevelopment of the original PRIDE, comprising a new storage backend, data submission system and web interface, among other components.  PRIDE Archive supports the most widely used PSI (Proteomics Stds. Initiative) data std. formats (mzML and mzIdentML) and implements the data requirements and guidelines of the ProteomeXchange Consortium.  The wide adoption of ProteomeXchange within the community has triggered an unprecedented increase in the no. of submitted data sets (around 150 data sets per mo).  We outline some statistics on the current PRIDE Archive data contents.  We also report on the status of the PRIDE related stand-alone tools: PRIDE Inspector, PRIDE Converter 2 and the ProteomeXchange submission tool.  Finally, we will give a brief update on the resources under development 'PRIDE Cluster' and 'PRIDE Proteomes', which provide a complementary view and quality-scored information of the peptide and protein identification data available in PRIDE Archive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_BbPYtr63s7Vg90H21EOLACvtfcHk0lh8de5E-Hex2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7vF&md5=946a9ff49868659b6de6fb24d6a282ef</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1145%26sid%3Dliteratum%253Aachs%26aulast%3DVizca%25C3%25ADno%26aufirst%3DJ.%2BA.%26aulast%3DCsordas%26aufirst%3DA.%26aulast%3Ddel-Toro%26aufirst%3DN.%26aulast%3DDianes%26aufirst%3DJ.%2BA.%26aulast%3DGriss%26aufirst%3DJ.%26aulast%3DLavidas%26aufirst%3DI.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DPerez-Riverol%26aufirst%3DY.%26aulast%3DReisinger%26aufirst%3DF.%26aulast%3DTernent%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DQ.-W.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DHermjakob%26aufirst%3DH.%26atitle%3D2016%2520Update%2520of%2520the%2520PRIDE%2520Database%2520and%2520its%2520related%2520Tools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD447%26epage%3D56%26doi%3D10.1093%2Fnar%2Fgkv1145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span> <i>Molecular Operating Environment (MOE)</i>; <span class="NLM_publisher-name">Chemical
Computing Group ULC</span>: <span class="NLM_publisher-loc">1010 Sherbooke
St West, Suite #910, Montreal, OC, Canada, H3A 2R7</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%3B+Chemical%0AComputing+Group+ULC%3A+1010+Sherbooke%0ASt+West%2C+Suite+%23910%2C+Montreal%2C+OC%2C+Canada%2C+H3A+2R7%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%250AComputing%2520Group%2520ULC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Protonate3D: Assignment of Ionization States and Hydrogen Coordinates to Macromolecular Structures</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+Assignment+of+Ionization+States+and+Hydrogen+Coordinates+to+Macromolecular+Structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0lh8de5E-Hex2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520Assignment%2520of%2520Ionization%2520States%2520and%2520Hydrogen%2520Coordinates%2520to%2520Macromolecular%2520Structures%26jtitle%3DProteins%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lh8de5E-Hex2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">DSX: A Knowledge-based Scoring Function for the Assessment of Protein-Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2731</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/ci200274q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200274q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2731-2745&author=G.+Neudertauthor=G.+Klebe&title=DSX%3A+A+Knowledge-based+Scoring+Function+for+the+Assessment+of+Protein-Ligand+Complexes&doi=10.1021%2Fci200274q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein-Ligand Complexes</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2731-2745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the new knowledge-based scoring function DSX that consists of distance-dependent pair potentials, novel torsion angle potentials, and newly defined solvent accessible surface-dependent potentials.  DSX pair potentials are based on the statistical formalism of DrugScore, extended by a much more specialized set of atom types.  The original DrugScore-like ref. state is rather unstable with respect to modifications in the used atom types.  Therefore, an important method to overcome this problem and to allow for robust results when deriving pair potentials for arbitrary sets of atom types is presented.  A validation based on a carefully prepd. test set is shown, enabling direct comparison to the majority of other popular scoring functions.  Here, DSX features superior performance with respect to docking- and ranking power and runtime requirements.  Furthermore, the beneficial combination with torsion angle-dependent and desolvation-dependent potentials is demonstrated.  DSX is robust, flexible, and capable of working together with special features of popular docking engines, e.g., flexible protein residues in AutoDock or GOLD.  The program is freely available to the scientific community and can be downloaded from our Web site www.agklebe.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvQyrP7qPi5bVg90H21EOLACvtfcHk0lh8de5E-Hex2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K&md5=c90af0aa9af9ae15d458d9ba759163bb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci200274q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200274q%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDSX%253A%2520A%2520Knowledge-based%2520Scoring%2520Function%2520for%2520the%2520Assessment%2520of%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2731%26epage%3D2745%26doi%3D10.1021%2Fci200274q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">fconv: Format Conversion, Manipulation and Feature Computation of Molecular Data</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fbioinformatics%2Fbtr055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=21335612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1yqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1021-1022&author=G.+Neudertauthor=G.+Klebe&title=fconv%3A+Format+Conversion%2C+Manipulation+and+Feature+Computation+of+Molecular+Data&doi=10.1093%2Fbioinformatics%2Fbtr055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">fconv: format conversion, manipulation and feature computation of molecular data</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1021-1022</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: fconv is a program intended for parsing and manipulating multiple aspects and properties of mol. data.  Up to now, it has been developed and extensively tested for 3 years.  It has become a very robust and comprehensive tool involved in a broad range of computational workflows that are currently applied in our drug design environment.  Typical tasks are as follows: conversion and error correction of formats such as PDB(QT), MOL2, SDF, DLG and CIF; extg. ligands from PDB as MOL2; automatic or ligand-based cavity detection; rmsd calcn. and clustering; substructure searches; alignment and structural superposition; building of crystal packings; adding hydrogens; calcn. of various properties like the no. of rotatable bonds; mol. wts. or vdW vols.  The atom type classification is based on a consistent assignment of internal atom types, which are by far more differentiated compared with e.g.  Sybyl atom types.  Apart from the predefined mapping of these types onto Sybyl types, the user is able to assign own mappings by providing modified template files, thus allowing for tailor-made atom type sets.  Availability: fconv is free software available under GNU General Public License.  C++ sources and precompiled executables for LINUX/UNIX, Mac OS and Windows, as well as tutorials are available on http://www.agklebe.de.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="274c4b4245426754534641410952494e0a4a46554552554009434209">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmzwJ0Mgz9SLVg90H21EOLACvtfcHk0lj6A8fkLU__5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1yqsL4%253D&md5=3869574573503ffd1c89d23132c4dfb9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr055%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3Dfconv%253A%2520Format%2520Conversion%252C%2520Manipulation%2520and%2520Feature%2520Computation%2520of%2520Molecular%2520Data%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1021%26epage%3D1022%26doi%3D10.1093%2Fbioinformatics%2Fbtr055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan R.  Whitfield</span>, <span class="hlFld-ContribAuthor ">Laura  Soucek</span>. </span><span class="cited-content_cbyCitation_article-title">The long journey to bring a Myc inhibitor to the clinic. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cell Biology</span><span> <strong>2021,</strong> <em>220 </em>
                                    (8)
                                     <a href="https://doi.org/10.1083/jcb.202103090" title="DOI URL">https://doi.org/10.1083/jcb.202103090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1083/jcb.202103090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1083%2Fjcb.202103090%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cell%2520Biology%26atitle%3DThe%252Blong%252Bjourney%252Bto%252Bbring%252Ba%252BMyc%252Binhibitor%252Bto%252Bthe%252Bclinic%26aulast%3DWhitfield%26aufirst%3DJonathan%2BR.%26date%3D2021%26date%3D2021%26volume%3D220%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Synthesis design of WDR5 degraders based on available crystal structures. (a) Crystal structures of WDR5 in complex with the small-molecule antagonist OICR-9429 (pink; the chemical structure of OICR-9429 is displayed above) and (b) a small molecule published by Wang and co-workers (blue; chemical structure of the chemical probe is displayed above). The red-colored spheres indicate the solvent-exposed site and represent the attachment point for linkers. pdb entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DAK">6DAK</a>. (c) (Upper panel) Schematic illustration of a heterobifunctional degrader molecule (also called PROTAC) consisting of a ligand for the target protein (WDR5, herein colored in pink/blue) and a ligand binding an E3 ligase (orange). (Lower panel) Chemical structures of two developed degrader series targeting WDR5. The molecules contain either the OICR-9429 derivate (pink) or the pyrroloimidazole derivate (blue) and are connected via different linkers (red) to different E3 ligase ligands (orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Route of WDR5 Degraders <b>7a–e</b> Addressing the E3 Ligase Cereblon (CRBN)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) <i>N</i>,<i>N-</i>Diisopropylethylamine (DIEA), EtOH, 80 °C, 16 h; (b) Zn, NH<sub>4</sub>Cl, dioxane/water (3/1), rt, 30 min; (c) (4-(<i>tert</i>-butoxycarbonyl)phenyl)boronic acid, XPhos PdG3, NaHCO<sub>3</sub>, dioxane/water (3/1), 85 °C, 16 h; (d) 1. carboxylic acid, SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/ACN (1/1), 50 °C, 3 h; 2. pyridine, CH<sub>2</sub>Cl<sub>2</sub>/ACN (1/1), 50 °C, 16 h; (e) 1. Trifluoroacetic acid (TFA)/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, <i>N</i>,<i>N</i>-dimethylformamide (DMF), CRBN ligase ligand linker <b>L1–L5</b>, rt, 3–5 h. The type and nature of the linker are indicated in red.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Chemical Structures of WDR5 Degraders <b>8a–j</b> Addressing the E3 Ligase Von–Hippel–Lindau (VHL)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The synthesis steps starting from intermediate <b>6</b> are shown: (a) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, linker <b>L6</b>–<b>L14</b>, rt, 3–5 h. The type and nature of the linker are indicated in red.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis Route of WDR5 Degraders <b>9a–c</b> Addressing the E3 Ligase MDM2<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The synthesis steps starting from intermediate <b>6</b> are shown: (a) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, linker, rt, 3–5 h; (b) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1 h; 2. HATU, DIEA, DMF, idasanutlin, rt, 12 h. The type and nature of the linker are indicated in red.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of WDR5 degraders <b>17a–g</b> addressing the E3 ligase VHL<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) POBr<sub>3</sub>, MeCN, MW, 70 °C, 2 h; (b) (3-cyano-4-methoxyphenyl)boronic acid, XPhos PdG3, NaOH, THF/H<sub>2</sub>O, 80 °C, 21 h; (c) LiAlH<sub>4</sub>, THF, 60 °C–rt, 21 h; (d) 2-(3,4-dichlorophenyl)acetic acid, EDC, HOBt, DIEA, DMF, rt, 16 h; (e) 1. BBr<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C to rt, 21 h; 2. NaOH/H<sub>2</sub>O; (f) linker-OTs, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C, 16–22.5 h; (g) 1. TFA/CH<sub>2</sub>Cl<sub>2</sub> (1/1), rt, 1.5 h; 2. HATU, DIEA, DMF, VHL hydrochloride linker, rt, 3–18 h. The type and nature of the linker are indicated in red.</p></p></figure><figure data-id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cellular permeability and target engagement studies were performed with the BRET assay. (a) Tracer titration of all three synthesized tracer molecules and either C-terminal or N-terminal Nanoluc-tagged WDR5 (WDR5-C or WDR5-N). (b) Chemical structure of the most suitable tracer molecule <b>19c</b>. (c) and (d) NanoBRET dose–response curves of the degrader <b>8g</b>, ligand <b>6</b>, and WDR5 antagonist OICR-9429 in live cells (c) as well as in lysed cells (d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cellular degradation studies on WDR5. (a) Immunoblot of WDR5. WDR5 was fused to the fragment of luciferase (HiBiT) and stably expressed in MV4-11 cells. Naive (-) MV4-11 cells and HiBiT-WDR5 (+) MV4-11 cells. Vinculin was used as a loading control. (b) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>8g</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (c) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of degraders <b>8a</b>, <b>8e</b>, <b>8f</b>, <b>8h</b>, and <b>8i</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (d) Quantification of WDR5 <i>D</i><sub>max</sub> (maximal degradation) from the HiBiT assay for degraders with different aliphatic linkers. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of degraders for 6 h or 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity of linkers comprising ethyl: <b>8e</b>, propyl: <b>8f</b>, butyl: <b>8g</b>, pentyl: <b>8h</b>, and hexyl: <b>8i</b>. (e) and (f) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>14</b>, <b>21</b>, and <b>17b</b> (e) and <b>6</b>, <b>20</b>, and <b>8g</b> (f) for 24 h. Cells were then lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Degrader-induced depletion of WDR5 depends on the ubiquitin system. (a) Immunoblot of WDR5. MV4-11<sup>WDR5-HiBiT</sup> cells were treated with different concentrations of <b>8g</b> for 24 h and compared with dimethyl sulfoxide (DMSO)-treated or naive MV4-11 cells. Vinculin was used as a loading control (as in all further immunoblot experiments). Quantification is based on both protein bands (endogenous/HiBiT-tagged WDR5). (b) Immunoblot of WDR5. Naive MV4-11 cells were treated with different concentrations of <b>8g</b> for 72 h and compared with DMSO-treated cells. (c) Immunoblot and quantification of WDR5 levels. WDR5 protein stability was evaluated by treating 1 μM <b>8g</b> or DMSO-incubated MV4-11 cells for 0, 1, 2, 6, and 12 h with cycloheximide (CHX). The data is mean ± s.d from <i>n</i> = 2 biological replicates. (d) Quantitative reverse transcriptase polymerase chain reaction (RT-PCR) analysis of WDR5 mRNA levels. RNA was extracted from MV4-11 cells incubated with 1 μM <b>8g</b>, 1 μM <b>6</b>, 3 μM <b>17b</b>, and 3 μM <b>14</b> for 24 h. WDR5 expression levels were normalized with a reference gene (<i>B</i>2<i>M</i>). Bars represent mean ± s.d. of <i>n</i> = 3 technical replicates. (e) Immunoblot of WDR5. MV4-11 cells were treated for 6 h with 1 μM <b>8g</b>, 5 μM <b>6</b>, 10 μM VHL ligand, or a combination of them. (f) Immunoblot of WDR5. MV4-11 cells were treated for 6 h with 3 μM <b>17b</b>, 5 μM MLN4924, 10 μM MG132, or a combination of them. (g) Immunoblot of WDR5. HL-60 cells were treated with different concentrations of <b>8g</b> for 24 h. (h) and (i) Volcano plot exhibiting global proteomics change. MV4-11 cells were treated with 1 μM <b>8g</b> (h) or 5 μM <b>17b</b> (i) for 9 h, and lysates were analyzed by quantitative proteomics. WDR5 (blue) and other SET1/MLL complex core subunits: KMT2A, KTM2B, KTM2C, KTM2D, SETD1A, RBBP5, ASH2L, and DPY30 (orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. VHL overexpression increases PROTACs/degrader efficiency and cellular responses. (a) Cumulative growth curve in MV4-11 cells. MV4-11 cells were treated with 10, 5, and 3 μM of <b>17b</b>, <b>21</b>, and <b>14</b> and counted at indicated time points. To prevent the overgrowth, cells were reseeded to the original density every third day in fresh media with compounds and treatment was continued for 15 days. Data represent mean ± s.d. of <i>n</i> = 2 biological replicates. Asterisks indicate <i>P</i>-value calculated from the 15<sup>th</sup> day cumulative cell number (two-tailed unpaired <i>t</i>-test assuming equal variance against DMSO treatment). * <i>P</i> ≤ 0.05. (b) WDR5 levels based on luciferase measurements. MV4-11<sup>WDR5-HiBiT</sup> cells (Ctr) and MV4-11<sup>WDR5-HiBiT/VHL</sup> cells (VHL OE) were treated with different concentrations of <b>17b</b> for 24 h, lysed, complemented with the second luciferase fragment (largeBiT), and measured for luciferase activity. (c) Immunoblot of WDR5 and VHL. MV4-11 cells (Ctr) and MV4-11<sup>VHL</sup> (VHL OE) were treated for 24 h with various concentrations of <b>17b</b>. †, overexpressed VHL; ††, endogenous VHL. (d) Cumulative growth curve in MV4-11<sup>VHL</sup> cells. MV4-11<sup>VHL</sup> cells were treated with 10, 5, and 3 μM of <b>17b</b>, <b>21</b>, and <b>14</b> and counted at indicated time points. To prevent the overgrowth, cells were reseeded to the original density every third day in fresh media with compounds and treatment was continued for 15 days. Data represent mean ± s.d. of <i>n</i> = 2 biological replicates. Asterisks indicate <i>P</i>-values calculated from the 15<sup>th</sup> day cumulative cell number (two-tailed unpaired <i>t</i>-test assuming equal variance against DMSO treatment). * <i>P</i> ≤ 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/medium/jm1c00146_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Computational docking studies on WDR5 and VHL. (a) Structures of the selected protein/protein docking solutions compatible with a short linker distance. VHL (blue) and VH032 (orange) form a complex with WDR5 (gray) and a modified OICR-9429 ligand (pink), with the attachment points for linkers in close proximity (<4 Å). (b–f) DSX top-ranked docking pose of degrader <b>8g</b> docked to each of the selected protein/protein complexes. The functional parts of molecule <b>8g</b> are colored in orange for the interaction with VHL, in pink for the interaction with WDR5, and in red for the linker that connects both moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00146/20210728/images/large/jm1c00146_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00146&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, B. F.</span></span> <span> </span><span class="NLM_article-title">Understanding Nucleosome Dynamics and their Links to Gene Expression and DNA Replication</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrm.2017.47" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=28537572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVahurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=548-562&author=W.+K.+M.+Laiauthor=B.+F.+Pugh&title=Understanding+Nucleosome+Dynamics+and+their+Links+to+Gene+Expression+and+DNA+Replication&doi=10.1038%2Fnrm.2017.47"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding nucleosome dynamics and their links to gene expression and DNA replication</span></div><div class="casAuthors">Lai, William K. M.; Pugh, B. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">548-562</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Advances in genomics technol. have provided the means to probe myriad chromatin interactions at unprecedented spatial and temporal resoln.  This has led to a profound understanding of nucleosome organization within the genome, revealing that nucleosomes are highly dynamic.  Nucleosome dynamics are governed by a complex interplay of histone compn., histone post-translational modifications, nucleosome occupancy and positioning within chromatin, which are influenced by numerous regulatory factors, including general regulatory factors, chromatin remodellers, chaperones and polymerases.  It is now known that these dynamics regulate diverse cellular processes ranging from gene transcription to DNA replication and repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqp-i8SEjuIrVg90H21EOLACvtfcHk0lhMsg6hZRSBhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVahurY%253D&md5=185a8ed8b369f77075d3dc602852d3b0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.47%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DW.%2BK.%2BM.%26aulast%3DPugh%26aufirst%3DB.%2BF.%26atitle%3DUnderstanding%2520Nucleosome%2520Dynamics%2520and%2520their%2520Links%2520to%2520Gene%2520Expression%2520and%2520DNA%2520Replication%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D548%26epage%3D562%26doi%3D10.1038%2Fnrm.2017.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbruzzese, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. A. G.</span></span> <span> </span><span class="NLM_article-title">The PI 3-kinase/Akt Signaling Pathway is activated due to aberrant Pten Expression and Targets Transcription Factors NF-kappaB and c-Myc in Pancreatic Cancer Cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">8571</span>– <span class="NLM_lpage">8580</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fsj.onc.1207902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=15467756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsF2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=8571-8580&author=T.+Asanoauthor=Y.+Yaoauthor=J.+Zhuauthor=D.+Liauthor=J.+L.+Abbruzzeseauthor=S.+A.+G.+Reddy&title=The+PI+3-kinase%2FAkt+Signaling+Pathway+is+activated+due+to+aberrant+Pten+Expression+and+Targets+Transcription+Factors+NF-kappaB+and+c-Myc+in+Pancreatic+Cancer+Cells&doi=10.1038%2Fsj.onc.1207902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The PI 3-kinase/Akt signaling pathway is activated due to aberrant Pten expression and targets transcription factors NF-κB and c-Myc in pancreatic cancer cells</span></div><div class="casAuthors">Asano, Takayuki; Yao, Yixin; Zhu, Jijiang; Li, Donghui; Abbruzzese, James L.; Reddy, Shrikanth A. G.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">53</span>),
    <span class="NLM_cas:pages">8571-8580</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The persistent activation of signaling cascades results in dramatic consequences that include loss of cellular growth control and neoplastic transformation.  We show here that phosphoinositide 3-kinase (PI 3-kinase) and its mediator Akt were constitutively activated in pancreatic cancer and that this might be due to the aberrant expression of their natural antagonist MMAC/PTEN.  Indeed, our results show that MMAC/PTEN expression was either lost or significantly reduced in five of eight cell lines and in twelve of seventeen tumor specimens examd.  That the poor expression of MMAC/PTEN in pancreatic cancer cells could be due to promoter methylation was indicated by methylation-specific PCR anal.  Our studies also indicated that PI 3-kinase targeted two important transcription factors in pancreatic cancer cells.  The ability of constitutively activated NF-κB to induce gene expression and the stabilization of c-MYC protein by decreased phosphorylation of Thr58 were both dependent on PI 3-kinase activity.  When pancreatic cancer cells were treated with a peptide antagonist of NF-κB nuclear translocation, or stably transfected with a dominant-neg. mutant of MYC, their proliferation was markedly inhibited. Taken together, these data indicate that the aberrant expression of MMAC/PTEN contributes to the activation of the PI 3-kinase/Akt pathway and its transcription factor mediators in pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNKTz7-bIWcbVg90H21EOLACvtfcHk0lhGejWgWhqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsF2msrk%253D&md5=49f2070b7000c93a96f1354327de9be5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207902%26sid%3Dliteratum%253Aachs%26aulast%3DAsano%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAbbruzzese%26aufirst%3DJ.%2BL.%26aulast%3DReddy%26aufirst%3DS.%2BA.%2BG.%26atitle%3DThe%2520PI%25203-kinase%252FAkt%2520Signaling%2520Pathway%2520is%2520activated%2520due%2520to%2520aberrant%2520Pten%2520Expression%2520and%2520Targets%2520Transcription%2520Factors%2520NF-kappaB%2520and%2520c-Myc%2520in%2520Pancreatic%2520Cancer%2520Cells%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D8571%26epage%3D8580%26doi%3D10.1038%2Fsj.onc.1207902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors as Targets for Cancer Therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1038/nrc906</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc906" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=12360277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=740-749&author=J.+E.+Darnell&title=Transcription+Factors+as+Targets+for+Cancer+Therapy&doi=10.1038%2Fnrc906"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors as targets for cancer therapy</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">740-749</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A limited list of transcription factors are overactive in most human cancer cells, which makes them targets for the development of anticancer drugs.  That they are the most direct and hopeful targets for treating cancer is proposed, and this is supported by the fact that there are many more human oncogenes in signaling pathways than there are oncogenic transcription factors.  But how could specific transcription-factor activity be inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJxyqucgv0aLVg90H21EOLACvtfcHk0lhGejWgWhqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D&md5=239132f634b138965e3a11c63f7cab76</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc906%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520Factors%2520as%2520Targets%2520for%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D740%26epage%3D749%26doi%3D10.1038%2Fnrc906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G. A.</span></span> <span> </span><span class="NLM_article-title">GATA Transcription Factors and Cancer</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1178</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1177/1947601911404223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1177%2F1947601911404223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=21779441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVSls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1178-1188&author=R.+Zhengauthor=G.+A.+Blobel&title=GATA+Transcription+Factors+and+Cancer&doi=10.1177%2F1947601911404223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">GATA transcription factors and cancer</span></div><div class="casAuthors">Zheng, Rena; Blobel, Gerd A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1178-1188</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  It has been almost a quarter century since it was first appreciated that a class of oncogenes contained in rapidly transforming avian retroviruses encoded DNA-binding transcription factors.  As with other oncogenes, genetic recombination with the viral genome led to their overexpression or functional alteration.  In the years that followed, alterations of numerous transcription factors were shown to be causatively involved in various cancers in human patients and model organisms.  Depending on their normal cellular functions, these factors were subsequently categorized as proto-oncogenes or tumor suppressor genes.  This review focuses on the role of GATA transcription factors in carcinogenesis.  GATA factors are zinc finger DNA binding proteins that control the development of diverse tissues by activating or repressing transcription.  GATA factors thus coordinate cellular maturation with proliferation arrest and cell survival.  Therefore, a role of this family of genes in human cancers is not surprising.  Prominent examples include structural mutations in GATA1 that are found in almost all megakaryoblastic leukemias in patients with Down syndrome; loss of GATA3 expression in aggressive, dedifferentiated breast cancers; and silencing of GATA4 and GATA5 expression in colorectal and lung cancers.  Here, we discuss possible mechanisms of carcinogenesis vis-a-vis the normal functions of GATA factors as they pertain to human patients and mouse models of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP6TgBqSjsqrVg90H21EOLACvtfcHk0lhNRviefAmgaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVSls7k%253D&md5=9a6d7e4cec5120df504a8e017911c442</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1947601911404223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911404223%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DGATA%2520Transcription%2520Factors%2520and%2520Cancer%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D1178%26epage%3D1188%26doi%3D10.1177%2F1947601911404223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannagi, R.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors c-Myc and CDX2 mediate E-selectin Ligand Expression in Colon Cancer Cells undergoing EGF/bFGF-induced epithelial-mesenchymal Transition</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">7776</span>– <span class="NLM_lpage">7781</span>, <span class="refDoi"> DOI: 10.1073/pnas.1111135109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1073%2Fpnas.1111135109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=22547830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1Kntro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=7776-7781&author=K.+Sakumaauthor=M.+Aokiauthor=R.+Kannagi&title=Transcription+Factors+c-Myc+and+CDX2+mediate+E-selectin+Ligand+Expression+in+Colon+Cancer+Cells+undergoing+EGF%2FbFGF-induced+epithelial-mesenchymal+Transition&doi=10.1073%2Fpnas.1111135109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors c-Myc and CDX2 mediate E-selectin ligand expression in colon cancer cells undergoing EGF/bFGF-induced epithelial-mesenchymal transition</span></div><div class="casAuthors">Sakuma, Keiichiro; Aoki, Masahiro; Kannagi, Reiji</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7776-7781, S7776/1-S7776/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Sialyl Lewis x (sLex) and sialyl Lewis a (sLea) glycans are expressed on highly metastatic colon cancer cells.  They promote extravasation of cancer cells and tumor angiogenesis via interacting with E-selectin on endothelial cells.  Recently, epithelial-mesenchymal transition (EMT) has been noted as a crit. phenotypic alteration in metastatic cancer cells.  To address the assocn. between sLex/a expression and EMT, we assessed whether sLex/a are highly expressed on colon cancer cells undergoing EMT.  Treatment of HT29 and DLD-1 cells with EGF and/or basic FGF (bFGF) induced EMT and significantly increased sLex/a expression resulting in enhanced E-selectin binding activity.  The transcript levels of the glycosyltransferase genes ST3GAL1/3/4 and FUT3 were significantly elevated and that of FUT2 was significantly suppressed by the treatment.  We provide evidence that ST3GAL1/3/4 and FUT3 are transcriptionally up-regulated by c-Myc with probable involvement of Ser62 phosphorylation, and that FUT2 is transcriptionally down-regulated through the attenuation of CDX2.  The contribution of c-Myc and CDX2 to the sLex/a induction was proved to be significant by knockdown or forced expression expts.  Interestingly, the cells undergoing EMT exhibited significantly increased VEGF secretion, which can promote tumor angiogenesis in cooperation with sLex/a.  Finally, immunohistol. study indicated high E-selectin ligand expression on cancer cells undergoing EMT in vivo, supporting their coexistence obsd. in vitro.  These results suggest a significant link between sLex/a expression and EMT in colon cancer cells and a pivotal role of c-Myc and CDX2 in regulating sLex/a expression during EMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ysc2wLI0SrVg90H21EOLACvtfcHk0lhNRviefAmgaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1Kntro%253D&md5=4fffe94db4c0ae1ebfe5c579d32cdfc6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111135109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111135109%26sid%3Dliteratum%253Aachs%26aulast%3DSakuma%26aufirst%3DK.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DKannagi%26aufirst%3DR.%26atitle%3DTranscription%2520Factors%2520c-Myc%2520and%2520CDX2%2520mediate%2520E-selectin%2520Ligand%2520Expression%2520in%2520Colon%2520Cancer%2520Cells%2520undergoing%2520EGF%252FbFGF-induced%2520epithelial-mesenchymal%2520Transition%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D7776%26epage%3D7781%26doi%3D10.1073%2Fpnas.1111135109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. A.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oncogene and Tumor-suppressor Gene Expression: a novel Strategy for Cancer Prevention and Treatment</span>. <i>Ann. Surg. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1007/BF02303545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1007%2FBF02303545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=7834433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADyaK2M7js1Shuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1994&pages=79-86&author=J.+A.+Roth&title=Modulation+of+Oncogene+and+Tumor-suppressor+Gene+Expression%3A+a+novel+Strategy+for+Cancer+Prevention+and+Treatment&doi=10.1007%2FBF02303545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oncogene and tumor-suppressor gene expression: a novel strategy for cancer prevention and treatment</span></div><div class="casAuthors">Roth J A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of surgical oncology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-86</span>
        ISSN:<span class="NLM_cas:issn">1068-9265</span>.
    </div><div class="casAbstract">BACKGROUND:  Advances in our understanding of the molecular biology of cancer have the potential for translation into clinical cancer prevention and treatment strategies.  Oncogenes and tumor suppressor genes have been implicated in the development of human cancers Mutations in gene families occur in both premalignant lesions and invasive tumors.  CONCLUSION:  Recombinant DNA constructs can be made that eliminate expression of a mutant oncogene protein or provide a normal copy of a tumor suppressor gene to the cancer cell.  Reversal of a single genetic lesion is sufficient to inhibit cancer cell proliferation and tumorigenicity.  Viral vectors are now being evaluated for direct delivery of these constructs to human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXDQuuyW-2sHWEEzDrymxmfW6udTcc2eb2VaSb2phKnLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M7js1Shuw%253D%253D&md5=ee88d4695942a97597f8085649e9e15c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2FBF02303545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02303545%26sid%3Dliteratum%253Aachs%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26atitle%3DModulation%2520of%2520Oncogene%2520and%2520Tumor-suppressor%2520Gene%2520Expression%253A%2520a%2520novel%2520Strategy%2520for%2520Cancer%2520Prevention%2520and%2520Treatment%26jtitle%3DAnn.%2520Surg.%2520Oncol.%26date%3D1994%26volume%3D1%26spage%3D79%26epage%3D86%26doi%3D10.1007%2FBF02303545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajasekaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y. K.</span></span> <span> </span><span class="NLM_article-title">Loss of Tumor Suppressor Gene Function in Human Cancer: An Overview</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2647</span>– <span class="NLM_lpage">2693</span>, <span class="refDoi"> DOI: 10.1159/000495956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1159%2F000495956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30562755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlGqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2018&pages=2647-2693&author=L.-H.+Wangauthor=C.-F.+Wuauthor=N.+Rajasekaranauthor=Y.+K.+Shin&title=Loss+of+Tumor+Suppressor+Gene+Function+in+Human+Cancer%3A+An+Overview&doi=10.1159%2F000495956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of Tumor Suppressor Gene Function in Human Cancer: An Overview</span></div><div class="casAuthors">Wang, Li-Hui; Wu, Chun-Fu; Rajasekaran, Nirmal; Shin, Young Kee</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2647-2693</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">A review.  Cancer is a disease caused by the accumulation of genetic and epigenetic changes in two types of genes: tumor suppressor genes (TSGs) and proto-oncogenes.  Extensive research has been conducted over the last few decades to elucidate the role of TSGs in cancer development.  In cancer, loss of TSG function occurs via the deletion or inactivation of two alleles, according to Knudson's two-hit model hypothesis.  It has become clear that mutations in TSGs are recessive at the level of an individual cell; therefore, a single mutation in a TSG is not sufficient to cause carcinogenesis.  However, many studies have identified candidate TSGs that do not conform with this std. definition, including genes inactivated by epigenetic silencing rather than by deletion.  In addn., proteasomal degrdn. by ubiquitination, abnormal cellular localization, and transcriptional regulation are also involved in the inactivation of TSGs.  This review incorporates these novel addnl. mechanisms of TSG inactivation into the existing two-hit model and proposes a revised multiple-hit model that will enable the identification of novel TSGs that can be used as prognostic and predictive biomarkers of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1uM-lJOCirVg90H21EOLACvtfcHk0lhuscz8j4w4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlGqsw%253D%253D&md5=9a0933a0b0e8de97b77ca8a2e32ac6b6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1159%2F000495956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000495956%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.-H.%26aulast%3DWu%26aufirst%3DC.-F.%26aulast%3DRajasekaran%26aufirst%3DN.%26aulast%3DShin%26aufirst%3DY.%2BK.%26atitle%3DLoss%2520of%2520Tumor%2520Suppressor%2520Gene%2520Function%2520in%2520Human%2520Cancer%253A%2520An%2520Overview%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2018%26volume%3D51%26spage%3D2647%26epage%3D2693%26doi%3D10.1159%2F000495956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlich, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Methylation of Histone H3 Lys 4 in coding Regions of active Genes</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">8695</span>– <span class="NLM_lpage">8700</span>, <span class="refDoi"> DOI: 10.1073/pnas.082249499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1073%2Fpnas.082249499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=12060701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD38XltF2hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=8695-8700&author=B.+E.+Bernsteinauthor=E.+L.+Humphreyauthor=R.+L.+Erlichauthor=R.+Schneiderauthor=P.+Boumanauthor=J.+S.+Liuauthor=T.+Kouzaridesauthor=S.+L.+Schreiber&title=Methylation+of+Histone+H3+Lys+4+in+coding+Regions+of+active+Genes&doi=10.1073%2Fpnas.082249499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3 Lys 4 in coding regions of active genes</span></div><div class="casAuthors">Bernstein, Bradley E.; Humphrey, Emily L.; Erlich, Rachel L.; Schneider, Robert; Bouman, Peter; Liu, Jun S.; Kouzarides, Tony; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8695-8700</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone tails regulate chromatin structure and transcription.  Here we present global analyses of histone acetylation and histone H3 Lys 4 methylation patterns in yeast.  We observe a significant correlation between acetylation of histones H3 and H4 in promoter regions and transcriptional activity.  In contrast, we find that dimethylation of histone H3 Lys 4 in coding regions correlates with transcriptional activity.  The histone methyltransferase Set1 is required to maintain expression of these active, promoter-acetylated, and coding region-methylated genes.  Global comparisons reveal that genomic regions deacetylated by the yeast enzymes Rpd3 and Hda1 overlap extensively with Lys 4 hypo- but not hypermethylated regions.  In the context of recent studies showing that Lys 4 methylation precludes histone deacetylase recruitment, we conclude that Set1 facilitates transcription, in part, by protecting active coding regions from deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQbFpuyMjGbbVg90H21EOLACvtfcHk0lgjgRSny_wgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltF2hsb8%253D&md5=d468bafc67a92d127a18545866e39484</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.082249499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.082249499%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DHumphrey%26aufirst%3DE.%2BL.%26aulast%3DErlich%26aufirst%3DR.%2BL.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBouman%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DJ.%2BS.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DMethylation%2520of%2520Histone%2520H3%2520Lys%25204%2520in%2520coding%2520Regions%2520of%2520active%2520Genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D8695%26epage%3D8700%26doi%3D10.1073%2Fpnas.082249499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span> <span> </span><span class="NLM_article-title">Histone Lysine Methylation Dynamics: Establishment, Regulation, and biological Impact</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+van+Rechemauthor=J.+R.+Whetstine&title=Histone+Lysine+Methylation+Dynamics%3A+Establishment%2C+Regulation%2C+and+biological+Impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0lgjgRSny_wgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3Dvan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520Lysine%2520Methylation%2520Dynamics%253A%2520Establishment%252C%2520Regulation%252C%2520and%2520biological%2520Impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buratowski, S.</span></span> <span> </span><span class="NLM_article-title">Determinants of Histone H3K4 Methylation Patterns</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">785.e6</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2017.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=29129639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2017&pages=773-785.e6&author=L.+M.+Soaresauthor=P.+C.+Heauthor=Y.+Chunauthor=H.+Suhauthor=T.+Kimauthor=S.+Buratowski&title=Determinants+of+Histone+H3K4+Methylation+Patterns&doi=10.1016%2Fj.molcel.2017.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of Histone H3K4 Methylation Patterns</span></div><div class="casAuthors">Soares, Luis M.; He, P. Cody; Chun, Yujin; Suh, Hyunsuk; Kim, Tae Soo; Buratowski, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-785.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Various factors differentially recognize trimethylated histone H3 lysine 4 (H3K4me3) near promoters, H3K4me2 just downstream, and promoter-distal H3K4me1 to modulate gene expression.  This methylation "gradient" is thought to result from preferential binding of the H3K4 methyltransferase Set1/complex assocd. with Set1 (COMPASS) to promoter-proximal RNA polymerase II.  However, other studies have suggested that location-specific cues allosterically activate Set1.  Chromatin immunopptn. sequencing (ChIP-seq) expts. show that H3K4 methylation patterns on active genes are not universal or fixed and change in response to both transcription elongation rate and frequency as well as reduced COMPASS activity.  Fusing Set1 to RNA polymerase II results in H3K4me2 throughout transcribed regions and similarly extended H3K4me3 on highly transcribed genes.  Tethered Set1 still requires histone H2B ubiquitylation for activity.  These results show that higher-level methylations reflect not only Set1/COMPASS recruitment but also multiple rounds of transcription.  This model provides a simple explanation for non-canonical methylation patterns at some loci or in certain COMPASS mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX-6-jGE28KbVg90H21EOLACvtfcHk0lgjgRSny_wgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrrK&md5=250ecdd352958b2bcd57b38d5ce5e806</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DL.%2BM.%26aulast%3DHe%26aufirst%3DP.%2BC.%26aulast%3DChun%26aufirst%3DY.%26aulast%3DSuh%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DBuratowski%26aufirst%3DS.%26atitle%3DDeterminants%2520of%2520Histone%2520H3K4%2520Methylation%2520Patterns%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D68%26spage%3D773%26epage%3D785.e6%26doi%3D10.1016%2Fj.molcel.2017.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Hijacked in Cancer: the KMT2 (MLL) Family of Methyltransferases</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1038/nrc3929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc3929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=25998713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WrurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=334-346&author=R.+C.+Raoauthor=Y.+Dou&title=Hijacked+in+Cancer%3A+the+KMT2+%28MLL%29+Family+of+Methyltransferases&doi=10.1038%2Fnrc3929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacked in cancer: the KMT2 (MLL) family of methyltransferases</span></div><div class="casAuthors">Rao, Rajesh C.; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">334-346</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone-lysine N-methyltransferase 2 (KMT2) family proteins methylate lysine 4 on the histone H3 tail at important regulatory regions in the genome and thereby impart crucial functions through modulating chromatin structures and DNA accessibility.  Although the human KMT2 family was initially named the mixed-lineage leukemia (MLL) family, owing to the role of the first-found member KMT2A in this disease, recent exome-sequencing studies revealed KMT2 genes to be among the most frequently mutated genes in many types of human cancers.  Efforts to integrate the mol. mechanisms of KMT2 with its roles in tumorigenesis have led to the development of first-generation inhibitors of KMT2 function, which could become novel cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdmrdvWshyxrVg90H21EOLACvtfcHk0lg0VcjKab-Fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WrurvP&md5=c7a7405d0f951a4ae238ef0c9e183b06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc3929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3929%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DR.%2BC.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DHijacked%2520in%2520Cancer%253A%2520the%2520KMT2%2520%2528MLL%2529%2520Family%2520of%2520Methyltransferases%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D334%26epage%3D346%26doi%3D10.1038%2Fnrc3929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opravil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin Structure by site-specific Histone H3 Methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1038/35020506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+Chromatin+Structure+by+site-specific+Histone+H3+Methyltransferases&doi=10.1038%2F35020506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lg0VcjKab-Fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Chromatin%2520Structure%2520by%2520site-specific%2520Histone%2520H3%2520Methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599%26doi%3D10.1038%2F35020506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, U.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Insights on the Regulation of the MLL/SET1 Family Histone Methyltransferases</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1863</i></span>, <span class="NLM_elocation-id">194561</span> <span class="refDoi"> DOI: 10.1016/j.bbagrm.2020.194561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.bbagrm.2020.194561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32304759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2020&author=L.+Shaauthor=A.+Ayoubauthor=U.-S.+Choauthor=Y.+Dou&title=Insights+on+the+Regulation+of+the+MLL%2FSET1+Family+Histone+Methyltransferases&doi=10.1016%2Fj.bbagrm.2020.194561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Insights on the regulation of the MLL/SET1 family histone methyltransferases</span></div><div class="casAuthors">Sha, Liang; Ayoub, Alex; Cho, Uhn-Soo; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">194561</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In eukaryotes, histone H3K4 methylation by the MLL/SET1 family histone methyltransferases is enriched at transcription regulatory elements including gene promoters and enhancers.  The level of H3K4 methylation is highly correlated with transcription activation and is one of the most frequently used histone post-translational modifications to predict transcriptional outcome.  Recently, it has been shown that rearrangement of the cellular landscape of H3K4 mono-methylation at distal enhancers precedes cell fate transition and is used for identification of novel regulatory elements for development and disease progression.  Similarly, broad H3K4 tri-methylation regions have also been used to predict intrinsic tumor suppression properties of regulator regions in a variety of cellular models.  Understanding the regulation for how H3K4 methylation is deposited and regulated is of paramount importance.  In this review, we will discuss new findings on how the MLL/SET1 family enzymes are regulated on chromatin and their potential functional and regulatory implications.  This article is part of a Special Issue entitled: The MLL family of proteins in normal development and disease edited by Thomas A Milne.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyIc180oHEh7Vg90H21EOLACvtfcHk0lg5FeoD9Vnfpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVels7s%253D&md5=b9bea9c6381e2471ee860c9bb936dc97</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2020.194561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2020.194561%26sid%3Dliteratum%253Aachs%26aulast%3DSha%26aufirst%3DL.%26aulast%3DAyoub%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DU.-S.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DInsights%2520on%2520the%2520Regulation%2520of%2520the%2520MLL%252FSET1%2520Family%2520Histone%2520Methyltransferases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2020%26volume%3D1863%26doi%3D10.1016%2Fj.bbagrm.2020.194561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 Methyltransferase Activity by its Core Components</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">719</span>, <span class="refDoi"> DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+Methyltransferase+Activity+by+its+Core+Components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0lg5FeoD9Vnfpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520Methyltransferase%2520Activity%2520by%2520its%2520Core%2520Components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chacón
Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlund, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of WD Repeat-Containing Protein 5 (WDR5)-MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4315</span>– <span class="NLM_lpage">4333</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00224</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00224" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4315-4333&author=S.+Chac%C3%B3n%0ASimonauthor=F.+Wangauthor=L.+R.+Thomasauthor=J.+Phanauthor=B.+Zhaoauthor=E.+T.+Olejniczakauthor=J.+D.+Macdonaldauthor=J.+G.+Shawauthor=C.+Schlundauthor=W.+Payneauthor=J.+Creightonauthor=S.+R.+Staufferauthor=A.+G.+Watersonauthor=W.+P.+Tanseyauthor=S.+W.+Fesik&title=Discovery+of+WD+Repeat-Containing+Protein+5+%28WDR5%29-MYC+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.0c00224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of WD Repeat-Containing Protein 5 (WDR5)-MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Chacon Simon, Selena; Wang, Feng; Thomas, Lance R.; Phan, Jason; Zhao, Bin; Olejniczak, Edward T.; MacDonald, Jonathan D.; Shaw, J. Grace; Schlund, Caden; Payne, William; Creighton, Joy; Stauffer, Shaun R.; Waterson, Alex G.; Tansey, William P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4315-4333</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The frequent deregulation of MYC and its elevated expression via multiple mechanisms drives cells to a tumorigenic state.  Indeed, MYC is overexpressed in up to ~ 50% of human cancers and is considered a highly validated anticancer target.  Recently, we discovered that WD repeat-contg. protein 5 (WDR5) binds to MYC and is a crit. cofactor required for the recruitment of MYC to its target genes and reported the first small mol. inhibitors of the WDR5-MYC interaction using structure-based design.  These compds. display high binding affinity, but have poor physicochem. properties and are hence not suitable for in vivo studies.  Herein, we conducted an NMR-based fragment screening to identify addnl. chem. matter and, using a structure-based approach, we merged a fragment hit with the previously reported sulfonamide series.  Compds. in this series can disrupt the WDR5-MYC interaction in cells, and as a consequence, we obsd. a redn. of MYC localization to chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPe6Kwe2jWL7Vg90H21EOLACvtfcHk0liyrE13v_QwFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCht7g%253D&md5=167a092b144cbeff004737328a71c069</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00224%26sid%3Dliteratum%253Aachs%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSchlund%26aufirst%3DC.%26aulast%3DPayne%26aufirst%3DW.%26aulast%3DCreighton%26aufirst%3DJ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529-MYC%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4315%26epage%3D4333%26doi%3D10.1021%2Facs.jmedchem.0c00224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velázquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolshan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-2496&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Vel%C3%A1zquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-Awar&title=Structure-Based+Optimization+of+a+Small+Molecule+Antagonist+of+the+Interaction+Between+WD+Repeat-Containing+Protein+5+%28WDR5%29+and+Mixed-Lineage+Leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0liyrE13v_QwFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Vel%25C3%25A1zquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-Awar%26aufirst%3DR.%26atitle%3DStructure-Based%2520Optimization%2520of%2520a%2520Small%2520Molecule%2520Antagonist%2520of%2520the%2520Interaction%2520Between%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529%2520and%2520Mixed-Lineage%2520Leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D2496%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giambruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avellino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of the Wdr5-MLL Interaction in C/EBPα N-terminal Leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-Awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+Targeting+of+the+Wdr5-MLL+Interaction+in+C%2FEBP%CE%B1+N-terminal+Leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0lhNjX2N1fgkxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520Targeting%2520of%2520the%2520Wdr5-MLL%2520Interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-terminal%2520Leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent 2-Aryl-6,7-dihydro-5 H-pyrrolo1,2- aimidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5623</span>– <span class="NLM_lpage">5642</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5623-5642&author=F.+Wangauthor=K.+O.+Jeonauthor=J.+M.+Salovichauthor=J.+D.+Macdonaldauthor=J.+Alvaradoauthor=R.+D.+Gogliottiauthor=J.+Phanauthor=E.+T.+Olejniczakauthor=Q.+Sunauthor=S.+Wangauthor=D.+Camperauthor=J.+P.+Yuhauthor=J.+G.+Shawauthor=J.+Saiauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+of+Potent+2-Aryl-6%2C7-dihydro-5+H-pyrrolo1%2C2-+aimidazoles+as+WDR5-WIN-Site+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.8b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Wang, Feng; Jeon, Kyu Ok; Salovich, James M.; Macdonald, Jonathan D.; Alvarado, Joseph; Gogliotti, Rocco D.; Phan, Jason; Olejniczak, Edward T.; Sun, Qi; Wang, Shidong; Camper, DeMarco; Yuh, Joannes P.; Shaw, J. Grace; Sai, Jiqing; Rossanese, Olivia W.; Tansey, William P.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5623-5642</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted pyrroloimidazoles such as I [R = 3,4-Cl2C6H3CH2, 3-MeO-4-MeC6H3, 2-Me-4-pyridinyl, 3,5-(MeO)2-4-MeC6H2] were prepd. as inhibitors of the chromatin-regulatory scaffold protein WDR5 WDR5-interaction (WIN) site as lead compds. for development of antitumor agents.  NMR-based screening of a large fragment library identified several chem. distinct hit series that bind to the WIN site within WDR5.  Using a structure-based design approach, pyrroloimidazoles were found with dissocn. consts. < 10 nM for WDR5 and micromolar cellular activity against an AML-leukemia cell line.  The structures of I (R = 3-MeO-4-MeC6H3) and of four other compds. bound to a fragment of WDR5 were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSZaiiuG6OrVg90H21EOLACvtfcHk0lgFsShGEWmpKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN&md5=2322c6458b2a060005bf5250a46f5cbd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJeon%26aufirst%3DK.%2BO.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Potent%25202-Aryl-6%252C7-dihydro-5%2520H-pyrrolo1%252C2-%2520aimidazoles%2520as%2520WDR5-WIN-Site%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5623%26epage%3D5642%26doi%3D10.1021%2Facs.jmedchem.8b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiebert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2916</span>– <span class="NLM_lpage">2928.e13</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2019.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.celrep.2019.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30865883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=2916-2928.e13&author=E.+R.+Ahoauthor=J.+Wangauthor=R.+D.+Gogliottiauthor=G.+C.+Howardauthor=J.+Phanauthor=P.+Acharyaauthor=J.+D.+Macdonaldauthor=K.+Chengauthor=S.+L.+Loreyauthor=B.+Luauthor=S.+Wenzelauthor=A.+M.+Foshageauthor=J.+Alvaradoauthor=F.+Wangauthor=J.+G.+Shawauthor=B.+Zhaoauthor=A.+M.+Weissmillerauthor=L.+R.+Thomasauthor=C.+R.+Vakocauthor=M.+D.+Hallauthor=S.+W.+Hiebertauthor=Q.+Liuauthor=S.+R.+Staufferauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Displacement+of+WDR5+from+Chromatin+by+a+WIN+Site+Inhibitor+with+Picomolar+Affinity&doi=10.1016%2Fj.celrep.2019.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span></div><div class="casAuthors">Aho, Erin R.; Wang, Jing; Gogliotti, Rocco D.; Howard, Gregory C.; Phan, Jason; Acharya, Pankaj; Macdonald, Jonathan D.; Cheng, Ken; Lorey, Shelly L.; Lu, Bin; Wenzel, Sabine; Foshage, Audra M.; Alvarado, Joseph; Wang, Feng; Shaw, J. Grace; Zhao, Bin; Weissmiller, April M.; Thomas, Lance R.; Vakoc, Christopher R.; Hall, Matthew D.; Hiebert, Scott W.; Liu, Qi; Stauffer, Shaun R.; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2916-2928.e13</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The chromatin-assocd. protein WDR5 is a promising target for pharmacol. inhibition in cancer.  Drug discovery efforts center on the blockade of the "WIN site" of WDR5, a well-defined pocket that is amenable to small mol. inhibition.  Various cancer contexts have been proposed to be targets for WIN site inhibitors, but a lack of understanding of WDR5 target genes and of the primary effects of WIN site inhibitors hampers their utility.  Here, by the discovery of potent WIN site inhibitors, we demonstrate that the WIN site links WDR5 to chromatin at a small cohort of loci, including a specific subset of ribosome protein genes.  WIN site inhibitors rapidly displace WDR5 from chromatin and decrease the expression of assocd. genes, causing translational inhibition, nucleolar stress, and p53 induction.  Our studies define a mode by which WDR5 engages chromatin and forecast that WIN site blockade could have utility against multiple cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprdgAaHB9WfLVg90H21EOLACvtfcHk0lgFsShGEWmpKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D&md5=56e64b66f0d04ff5c36e104e54594bbd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2019.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2019.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DWenzel%26aufirst%3DS.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DHiebert%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DDisplacement%2520of%2520WDR5%2520from%2520Chromatin%2520by%2520a%2520WIN%2520Site%2520Inhibitor%2520with%2520Picomolar%2520Affinity%26jtitle%3DCell%2520Rep.%26date%3D2019%26volume%3D26%26spage%3D2916%26epage%3D2928.e13%26doi%3D10.1016%2Fj.celrep.2019.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-P.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design and Synthesis of Small Molecular Inhibitors disturbing the Interaction of MLL1-WDR5</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.ejmech.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27116709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=1-8&author=D.-D.+Liauthor=W.-L.+Chenauthor=X.-L.+Xuauthor=F.+Jiangauthor=L.+Wangauthor=Y.-Y.+Xieauthor=X.-J.+Zhangauthor=X.-K.+Guoauthor=Q.-D.+Youauthor=H.-P.+Sun&title=Structure-based+Design+and+Synthesis+of+Small+Molecular+Inhibitors+disturbing+the+Interaction+of+MLL1-WDR5&doi=10.1016%2Fj.ejmech.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Xu, Xiao-Li; Jiang, Fen; Wang, Lei; Xie, Yi-Yue; Zhang, Xiao-Jin; Guo, Xiao-Ke; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1 complex catalyzes the methylation of H3K4, and plays important roles in the development of acute leukemia harboring MLL fusion proteins.  Targeting MLL1-WDR5 protein-protein interaction (PPI) to inhibit the activity of histone methyltransferase of MLL1 complex is a novel strategy for treating of acute leukemia.  I (IC50 = 0.3 μM) was defined as a potent small mol. to disturb the interaction of MLL1-WDR5.  Here, the authors described structure-based design and synthesis of small mol. inhibitors to block MLL1-WDR5 PPI.  Esp., compd. II (IC50 = 104 nM) was the most potent small mol., and about 3-times more potent than I.  The authors also discussed the SAR of these series of compds. with docking study, which may stimulate more potent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsAbY0it5GLVg90H21EOLACvtfcHk0ljV6i6gokgQRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D&md5=b18d482f34731dd1558e9b78b770c2c1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DGuo%26aufirst%3DX.-K.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DSun%26aufirst%3DH.-P.%26atitle%3DStructure-based%2520Design%2520and%2520Synthesis%2520of%2520Small%2520Molecular%2520Inhibitors%2520disturbing%2520the%2520Interaction%2520of%2520MLL1-WDR5%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ejmech.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span> <span> </span><span class="NLM_article-title">High-affinity Small Molecular Blockers of Mixed Lineage Leukemia 1 (MLL1)-WDR5 Interaction inhibit MLL1 Complex H3K4 Methyltransferase Activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.-D.+Liauthor=W.-L.+Chenauthor=Z.-H.+Wangauthor=Y.-Y.+Xieauthor=X.-L.+Xuauthor=Z.-Y.+Jiangauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=X.-K.+Guo&title=High-affinity+Small+Molecular+Blockers+of+Mixed+Lineage+Leukemia+1+%28MLL1%29-WDR5+Interaction+inhibit+MLL1+Complex+H3K4+Methyltransferase+Activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0ljV6i6gokgQRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DZ.-H.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DGuo%26aufirst%3DX.-K.%26atitle%3DHigh-affinity%2520Small%2520Molecular%2520Blockers%2520of%2520Mixed%2520Lineage%2520Leukemia%25201%2520%2528MLL1%2529-WDR5%2520Interaction%2520inhibit%2520MLL1%2520Complex%2520H3K4%2520Methyltransferase%2520Activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freie, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathsyaraja, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R. N.</span></span> <span> </span><span class="NLM_article-title">The MYC Transcription Factor Network: balancing Metabolism, Proliferation and Oncogenesis</span>. <i>Front. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">425</span>, <span class="refDoi"> DOI: 10.1007/s11684-018-0650-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1007%2Fs11684-018-0650-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=30054853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ivVeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=412-425&author=P.+A.+Carrollauthor=B.+W.+Freieauthor=H.+Mathsyarajaauthor=R.+N.+Eisenman&title=The+MYC+Transcription+Factor+Network%3A+balancing+Metabolism%2C+Proliferation+and+Oncogenesis&doi=10.1007%2Fs11684-018-0650-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The MYC transcription factor network: balancing metabolism, proliferation and oncogenesis</span></div><div class="casAuthors">Carroll Patrick A; Freie Brian W; Mathsyaraja Haritha; Eisenman Robert N</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers of medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">412-425</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Transcription factor networks have evolved in order to control, coordinate, and separate, the functions of distinct network modules spatially and temporally.  In this review we focus on the MYC network (also known as the MAX-MLX Network), a highly conserved super-family of related basic-helix-loop-helix-zipper (bHLHZ) proteins that functions to integrate extracellular and intracellular signals and modulate global gene expression.  Importantly the MYC network has been shown to be deeply involved in a broad spectrum of human and other animal cancers.  Here we summarize molecular and biological properties of the network modules with emphasis on functional interactions among network members.  We suggest that these network interactions serve to modulate growth and metabolism at the transcriptional level in order to balance nutrient demand with supply, to maintain growth homeostasis, and to influence cell fate.  Moreover, oncogenic activation of MYC and/or loss of a MYC antagonist, results in an imbalance in the activity of the network as a whole, leading to tumor initiation, progression and maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRZan0K73QGlZoq0fPSOTvfW6udTcc2eZqKYXQwEpQ4Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ivVeksw%253D%253D&md5=40b49032a138dfbf686fc4fb19e0ca55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs11684-018-0650-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11684-018-0650-z%26sid%3Dliteratum%253Aachs%26aulast%3DCarroll%26aufirst%3DP.%2BA.%26aulast%3DFreie%26aufirst%3DB.%2BW.%26aulast%3DMathsyaraja%26aufirst%3DH.%26aulast%3DEisenman%26aufirst%3DR.%2BN.%26atitle%3DThe%2520MYC%2520Transcription%2520Factor%2520Network%253A%2520balancing%2520Metabolism%252C%2520Proliferation%2520and%2520Oncogenesis%26jtitle%3DFront.%2520Med.%26date%3D2018%26volume%3D12%26spage%3D412%26epage%3D425%26doi%3D10.1007%2Fs11684-018-0650-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC through WDR5</span>. <i>Mol. Cell. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">1709388</span> <span class="refDoi"> DOI: 10.1080/23723556.2019.1709388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1080%2F23723556.2019.1709388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32158922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A280%3ADC%252BB383itFaltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2020&author=L.+R.+Thomasauthor=C.+M.+Adamsauthor=S.+W.+Fesikauthor=C.+M.+Eischenauthor=W.+P.+Tansey&title=Targeting+MYC+through+WDR5&doi=10.1080%2F23723556.2019.1709388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MYC through WDR5</span></div><div class="casAuthors">Thomas Lance R; Tansey William P; Adams Clare M; Eischen Christine M; Fesik Stephen W; Tansey William P</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & cellular oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1709388</span>
        ISSN:<span class="NLM_cas:issn">2372-3556</span>.
    </div><div class="casAbstract">The oncoprotein transcription factor MYC is overexpressed in most cancers and is responsible for hundreds of thousands of cancer deaths worldwide every year.  MYC is also a highly validated - but currently undruggable - anti-cancer target.  We recently showed that breaking the interaction of MYC with its chromatin co-factor WD repeat-containing protein 5 (WDR5) promotes tumor regression in mouse xenografts, laying the foundation for a new strategy to inhibit MYC in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJ85Bz5vYqRC86Js-BCLiifW6udTcc2eZqKYXQwEpQ4Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383itFaltg%253D%253D&md5=8fde22bb0fa8adcc112eb4467c1b6db4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F23723556.2019.1709388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F23723556.2019.1709388%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DAdams%26aufirst%3DC.%2BM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DTargeting%2520MYC%2520through%2520WDR5%26jtitle%3DMol.%2520Cell.%2520Oncol.%26date%3D2020%26volume%3D7%26doi%3D10.1080%2F23723556.2019.1709388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ‘undruggable’ Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%98undruggable%E2%80%99+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0lgRfxAdlZInDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodir, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnezis, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swigart, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evan, G. I.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Myc Family Proteins eradicates KRas-driven Lung Cancer in Mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1101/gad.205542.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1101%2Fgad.205542.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23475959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVKkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=504-513&author=L.+Soucekauthor=J.+R.+Whitfieldauthor=N.+M.+Sodirauthor=D.+Mass%C3%B3-Vall%C3%A9sauthor=E.+Serranoauthor=A.+N.+Karnezisauthor=L.+B.+Swigartauthor=G.+I.+Evan&title=Inhibition+of+Myc+Family+Proteins+eradicates+KRas-driven+Lung+Cancer+in+Mice&doi=10.1101%2Fgad.205542.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Myc family proteins eradicates KRas-driven lung cancer in mice</span></div><div class="casAuthors">Soucek, Laura; Whitfield, Jonathan R.; Sodir, Nicole M.; Masso-Valles, Daniel; Serrano, Erika; Karnezis, Anthony N.; Swigart, Lamorna Brown; Evan, Gerard I.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The principal reason for failure of targeted cancer therapies is the emergence of resistant clones that regenerate the tumor.  Therapeutic efficacy therefore depends on not only how effectively a drug inhibits its target, but also the innate or adaptive functional redundancy of that target and its attendant pathway.  In this regard, the Myc transcription factors are intriguing therapeutic targets because they serve the unique and irreplaceable role of coordinating expression of the many diverse genes that, together, are required for somatic cell proliferation.  Furthermore, Myc expression is deregulated in most-perhaps all-cancers, underscoring its irreplaceable role in proliferation.  We previously showed in a preclin. mouse model of non-small-cell lung cancer that systemic Myc inhibition using the dominant-neg. Myc mutant Omomyc exerts a dramatic therapeutic impact, triggering rapid regression of tumors with only mild and fully reversible side effects.  Using protracted episodic expression of Omomyc, we now demonstrate that metronomic Myc inhibition not only contains Ras-driven lung tumors indefinitely, but also leads to their progressive eradication.  Hence, Myc does indeed serve a unique and nondegenerate role in lung tumor maintenance that cannot be complemented by any adaptive mechanism, even in the most aggressive p53-deficient tumors.  These data endorse Myc as a compelling cancer drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfpivJKOSErbVg90H21EOLACvtfcHk0lgRfxAdlZInDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVKkurY%253D&md5=0fc5aa1a0223f4b09da3e7bf114bdc56</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1101%2Fgad.205542.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.205542.112%26sid%3Dliteratum%253Aachs%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DWhitfield%26aufirst%3DJ.%2BR.%26aulast%3DSodir%26aufirst%3DN.%2BM.%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DSerrano%26aufirst%3DE.%26aulast%3DKarnezis%26aufirst%3DA.%2BN.%26aulast%3DSwigart%26aufirst%3DL.%2BB.%26aulast%3DEvan%26aufirst%3DG.%2BI.%26atitle%3DInhibition%2520of%2520Myc%2520Family%2520Proteins%2520eradicates%2520KRas-driven%2520Lung%2520Cancer%2520in%2520Mice%26jtitle%3DGenes%2520Dev.%26date%3D2013%26volume%3D27%26spage%3D504%26epage%3D513%26doi%3D10.1101%2Fgad.205542.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC Proteins for Tumor Therapy</span>. <i>Annu. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-030518-055826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1146%2Fannurev-cancerbio-030518-055826" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=61-75&author=E.+Wolfauthor=M.+Eilers&title=Targeting+MYC+Proteins+for+Tumor+Therapy&doi=10.1146%2Fannurev-cancerbio-030518-055826"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-030518-055826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-030518-055826%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DE.%26aulast%3DEilers%26aufirst%3DM.%26atitle%3DTargeting%2520MYC%2520Proteins%2520for%2520Tumor%2520Therapy%26jtitle%3DAnnu.%2520Rev.%2520Cancer%2520Biol.%26date%3D2020%26volume%3D4%26spage%3D61%26epage%3D75%26doi%3D10.1146%2Fannurev-cancerbio-030518-055826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieb, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Interaction with WDR5 promotes Target Gene Recognition and Tumorigenesis by MYC</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">440</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.molcel.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=25818646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=440-452&author=L.+R.+Thomasauthor=Q.+Wangauthor=B.+C.+Griebauthor=J.+Phanauthor=A.+M.+Foshageauthor=Q.+Sunauthor=E.+T.+Olejniczakauthor=T.+Clarkauthor=S.+Deyauthor=S.+Loreyauthor=B.+Alicieauthor=G.+C.+Howardauthor=B.+Cawthonauthor=K.+C.+Essauthor=C.+M.+Eischenauthor=Z.+Zhaoauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Interaction+with+WDR5+promotes+Target+Gene+Recognition+and+Tumorigenesis+by+MYC&doi=10.1016%2Fj.molcel.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span></div><div class="casAuthors">Thomas, Lance R.; Wang, Qingguo; Grieb, Brian C.; Phan, Jason; Foshage, Audra M.; Sun, Qi; Olejniczak, Edward T.; Clark, Travis; Dey, Soumyadeep; Lorey, Shelly; Alicie, Bethany; Howard, Gregory C.; Cawthon, Bryan; Ess, Kevin C.; Eischen, Christine M.; Zhao, Zhongming; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">440-452</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC is an oncoprotein transcription factor that is overexpressed in the majority of malignancies.  The oncogenic potential of MYC stems from its ability to bind regulatory sequences in thousands of target genes, which depends on interaction of MYC with its obligate partner, MAX.  Here, we show that broad assocn. of MYC with chromatin also depends on interaction with the WD40-repeat protein WDR5.  MYC binds WDR5 via an evolutionarily conserved "MYC box IIIb" motif that engages a shallow, hydrophobic cleft on the surface of WDR5.  Structure-guided mutations in MYC that disrupt interaction with WDR5 attenuate binding of MYC at ∼80% of its chromosomal locations and disable its ability to promote induced pluripotent stem cell formation and drive tumorigenesis.  Our data reveal WDR5 as a key determinant for MYC recruitment to chromatin and uncover a tractable target for the discovery of anticancer therapies against MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfifOnQkKDHrVg90H21EOLACvtfcHk0lj38VXzlh2bXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D&md5=329876ab3db1a16fb58bd5de7f1de7b9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DGrieb%26aufirst%3DB.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DLorey%26aufirst%3DS.%26aulast%3DAlicie%26aufirst%3DB.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DCawthon%26aufirst%3DB.%26aulast%3DEss%26aufirst%3DK.%2BC.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DInteraction%2520with%2520WDR5%2520promotes%2520Target%2520Gene%2520Recognition%2520and%2520Tumorigenesis%2520by%2520MYC%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D440%26epage%3D452%26doi%3D10.1016%2Fj.molcel.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnaccia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rellinger, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matlock, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">WDR5 is a conserved Regulator of Protein Synthesis Gene Expression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2924</span>– <span class="NLM_lpage">2941</span>, <span class="refDoi"> DOI: 10.1093/nar/gkaa051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fnar%2Fgkaa051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=31996893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVemsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=2924-2941&author=A.+F.+Bryanauthor=J.+Wangauthor=G.+C.+Howardauthor=A.+D.+Guarnacciaauthor=C.+M.+Woodleyauthor=E.+R.+Ahoauthor=E.+J.+Rellingerauthor=B.+K.+Matlockauthor=D.+K.+Flahertyauthor=S.+L.+Loreyauthor=D.+H.+Chungauthor=S.+W.+Fesikauthor=Q.+Liuauthor=A.+M.+Weissmillerauthor=W.+P.+Tansey&title=WDR5+is+a+conserved+Regulator+of+Protein+Synthesis+Gene+Expression&doi=10.1093%2Fnar%2Fgkaa051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 is a conserved regulator of protein synthesis gene expression</span></div><div class="casAuthors">Bryan, Audra F.; Wang, Jing; Howard, Gregory C.; Guarnaccia, Alissa D.; Woodley, Chase M.; Aho, Erin R.; Rellinger, Eric J.; Matlock, Brittany K.; Flaherty, David K.; Lorey, Shelly L.; Chung, Dai H.; Fesik, Stephen W.; Liu, Qi; Weissmiller, April M.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2924-2941</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">WDR5 is a highly-conserved nuclear protein that performs multiple scaffolding functions in the context of chromatin.  WDR5 is also a promising target for pharmacol. inhibition in cancer, with small mol. inhibitors of an arginine-binding pocket of WDR5 (the 'WIN' site) showing efficacy against a range of cancer cell lines in vitro.  Efforts to understand WDR5, or establish the mechanism of action of WIN site inhibitors, however, are stymied by its many functions in the nucleus, and a lack of knowledge of the conserved gene networks--if any--that are under its control.  Here, we have performed comparative genomic analyses to identify the conserved sites of WDR5 binding to chromatin, and the conserved genes regulated by WDR5, across a diverse panel of cancer cell lines.  We show that a specific cohort of protein synthesis genes (PSGs) are invariantly bound by WDR5, demonstrate that the WIN site anchors WDR5 to chromatin at these sites, and establish that PSGs are bona fide, acute, and persistent targets of WIN site blockade.  Together, these data reveal that WDR5 plays a predominant transcriptional role in biomass accumulation and provide further evidence that WIN site inhibitors act to repress gene networks linked to protein synthesis homeostasis.  In Saccharomyces cerevisiae, ribosomal protein gene (RPG) promoters display binding sites for either Rap1 or Abf1 transcription factors.  Unlike Rap1-assocd. promoters, the small cohort of Abf1-dependent RPGs (Abf1-RPGs) has not been extensively investigated.  We show that RPL3, RPL4B, RPP1A, RPS22B and RPS28A/B share a common promoter architecture, with an Abf1 site upstream of a conserved element matching the sequence recognized by Fhl1, a transcription factor which together with Ifh1 orchestrates Rap1-assocd. RPG regulation.  Abf1 and Fhl1 promoter assocn. was confirmed by ChIP and/or gel retardation assays.  Mutational anal. revealed a more severe requirement of Abf1 than Fhl1 binding sites for RPG transcription.  In the case of RPS22B an unusual Tbf1 binding site promoted both RPS22B and intron-hosted SNR44 expression.  Abf1-RPG down-regulation upon TOR pathway inhibition was much attenuated at defective mutant promoters unable to bind Abf1.  TORC1 inactivation caused the expected redn. of Ifh1 occupancy at RPS22B and RPL3 promoters, but unexpectedly it entailed largely increased Abf1 assocn. with Abf1-RPG promoters.  We present evidence that Abf1 recruitment upon nutritional stress, also obsd. for representative ribosome biogenesis genes, favors RPG transcriptional rescue upon nutrient replenishment, thus pointing to nutrient-regulated Abf1 dynamics at promoters as a novel mechanism in ribosome biogenesis control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNV7Dv7YF5lLVg90H21EOLACvtfcHk0lj38VXzlh2bXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVemsrrF&md5=52e65a8c20ac0c1f8db15ee2fa7d0225</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkaa051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkaa051%26sid%3Dliteratum%253Aachs%26aulast%3DBryan%26aufirst%3DA.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DGuarnaccia%26aufirst%3DA.%2BD.%26aulast%3DWoodley%26aufirst%3DC.%2BM.%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DRellinger%26aufirst%3DE.%2BJ.%26aulast%3DMatlock%26aufirst%3DB.%2BK.%26aulast%3DFlaherty%26aufirst%3DD.%2BK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DChung%26aufirst%3DD.%2BH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DWDR5%2520is%2520a%2520conserved%2520Regulator%2520of%2520Protein%2520Synthesis%2520Gene%2520Expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26spage%3D2924%26epage%3D2941%26doi%3D10.1093%2Fnar%2Fgkaa051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacón Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5-MYC Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11232</span>– <span class="NLM_lpage">11259</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11232-11259&author=J.+D.+Macdonaldauthor=S.+Chac%C3%B3n+Simonauthor=C.+Hanauthor=F.+Wangauthor=J.+G.+Shawauthor=J.+E.+Howesauthor=J.+Saiauthor=J.+P.+Yuhauthor=D.+Camperauthor=B.+M.+Alicieauthor=J.+Alvaradoauthor=S.+Nikharauthor=W.+Payneauthor=E.+R.+Ahoauthor=J.+A.+Bauerauthor=B.+Zhaoauthor=J.+Phanauthor=L.+R.+Thomasauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=A.+G.+Watersonauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+and+Optimization+of+Salicylic+Acid-Derived+Sulfonamide+Inhibitors+of+the+WD+Repeat-Containing+Protein+5-MYC+Protein-Protein+Interaction&doi=10.1021%2Facs.jmedchem.9b01411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5-MYC Protein-Protein Interaction</span></div><div class="casAuthors">Macdonald, Jonathan D.; Chacon Simon, Selena; Han, Changho; Wang, Feng; Shaw, J. Grace; Howes, Jennifer E.; Sai, Jiqing; Yuh, Joannes P.; Camper, Demarco; Alicie, Bethany M.; Alvarado, Joseph; Nikhar, Sameer; Payne, William; Aho, Erin R.; Bauer, Joshua A.; Zhao, Bin; Phan, Jason; Thomas, Lance R.; Rossanese, Olivia W.; Tansey, William P.; Waterson, Alex G.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11232-11259</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery.  Here, we present a new strategy toward the inhibition of MYC via the disruption of the protein-protein interaction between MYC and its chromatin cofactor WD Repeat-Contg. Protein 5.  Blocking the assocn. of these proteins is hypothesized to disrupt the localization of MYC to chromatin, thus disrupting the ability of MYC to sustain tumorigenesis.  Utilizing a high-throughput screening campaign and subsequent structure-guided design, we identify small-mol. inhibitors of this interaction with potent in vitro binding affinity and report structurally related neg. controls that can be used to study the effect of this disruption.  Our work suggests that disruption of this protein-protein interaction may provide a path toward an effective approach for the treatment of multiple tumors and anticipate that the mols. disclosed can be used as starting points for future efforts toward compds. with improved drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHEVMRoQUUdbVg90H21EOLACvtfcHk0lglRgdME2FFdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLfI&md5=e7bb66ca44a1d04f2240fc9c8309b362</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01411%26sid%3Dliteratum%253Aachs%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DHowes%26aufirst%3DJ.%2BE.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DAlicie%26aufirst%3DB.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DNikhar%26aufirst%3DS.%26aulast%3DPayne%26aufirst%3DW.%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Salicylic%2520Acid-Derived%2520Sulfonamide%2520Inhibitors%2520of%2520the%2520WD%2520Repeat-Containing%2520Protein%25205-MYC%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11232%26epage%3D11259%26doi%3D10.1021%2Facs.jmedchem.9b01411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gherardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmadibrata, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trahair, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shohet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüttelmaier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braithwaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span> <span> </span><span class="NLM_article-title">WDR5 supports an N-Myc Transcriptional Complex that drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">5143</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1158%2F0008-5472.CAN-15-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26471359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5143-5154&author=Y.+Sunauthor=J.+L.+Bellauthor=D.+Carterauthor=S.+Gherardiauthor=R.+C.+Poulosauthor=G.+Milazzoauthor=J.+W.+H.+Wongauthor=R.+Al-Awarauthor=A.+E.+Teeauthor=P.+Y.+Liuauthor=B.+Liuauthor=B.+Atmadibrataauthor=M.+Wongauthor=T.+Trahairauthor=Q.+Zhaoauthor=J.+M.+Shohetauthor=Y.+Hauptauthor=J.+H.+Schulteauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=K.+L.+MacKenzieauthor=S.+H%C3%BCttelmaierauthor=G.+Periniauthor=G.+M.+Marshallauthor=A.+Braithwaiteauthor=T.+Liu&title=WDR5+supports+an+N-Myc+Transcriptional+Complex+that+drives+a+Protumorigenic+Gene+Expression+Signature+in+Neuroblastoma&doi=10.1158%2F0008-5472.CAN-15-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Supports an N-Myc Transcriptional Complex That Drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span></div><div class="casAuthors">Sun, Yuting; Bell, Jessica L.; Carter, Daniel; Gherardi, Samuele; Poulos, Rebecca C.; Milazzo, Giorgio; Wong, Jason W. H.; Al-Awar, Rima; Tee, Andrew E.; Liu, Pei Y.; Liu, Bing; Atmadibrata, Bernard; Wong, Matthew; Trahair, Toby; Zhao, Quan; Shohet, Jason M.; Haupt, Ygal; Schulte, Johannes H.; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; MacKenzie, Karen L.; Huettelmaier, Stefan; Perini, Giovanni; Marshall, Glenn M.; Braithwaite, Antony; Liu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5143-5154</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MYCN gene amplification in neuroblastoma drives a gene expression program that correlates strongly with aggressive disease.  Mechanistically, trimethylation of histone H3 lysine 4 (H3K4) at target gene promoters is a strict prerequisite for this transcriptional program to be enacted.  WDR5 is a histone H3K4 presenter that has been found to have an essential role in H3K4 trimethylation.  For this reason, in this study, we investigated the relationship between WDR5-mediated H3K4 trimethylation and N-Myc transcriptional programs in neuroblastoma cells.  N-Myc upregulated WDR5 expression in neuroblastoma cells.  Gene expression anal. revealed that WDR5 target genes included those with MYC-binding elements at promoters such as MDM2.  We showed that WDR5 could form a protein complex at the MDM2 promoter with N-Myc, but not p53, leading to histone H3K4 trimethylation and activation of MDM2 transcription.  RNAi-mediated attenuation of WDR5 upregulated expression of wild-type but not mutant p53, an effect assocd. with growth inhibition and apoptosis.  Similarly, a small-mol. antagonist of WDR5 reduced N-Myc/WDR5 complex formation, N-Myc target gene expression, and cell growth in neuroblastoma cells.  In MYCN-transgenic mice, WDR5 was overexpressed in precancerous ganglion and neuroblastoma cells compared with normal ganglion cells.  Clin., elevated levels of WDR5 in neuroblastoma specimens were an independent predictor of poor overall survival.  Overall, our results identify WDR5 as a key cofactor for N-Myc-regulated transcriptional activation and tumorigenesis and as a novel therapeutic target for MYCN-amplified neuroblastomas.  Cancer Res; 75(23); 5143-54. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdt2xhQOC6ibVg90H21EOLACvtfcHk0lglRgdME2FFdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI&md5=26513b24ba2ba6b82cc691534f5939cd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0423%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DCarter%26aufirst%3DD.%26aulast%3DGherardi%26aufirst%3DS.%26aulast%3DPoulos%26aufirst%3DR.%2BC.%26aulast%3DMilazzo%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTee%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DP.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DAtmadibrata%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DTrahair%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DShohet%26aufirst%3DJ.%2BM.%26aulast%3DHaupt%26aufirst%3DY.%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DH%25C3%25BCttelmaier%26aufirst%3DS.%26aulast%3DPerini%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DG.%2BM.%26aulast%3DBraithwaite%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DT.%26atitle%3DWDR5%2520supports%2520an%2520N-Myc%2520Transcriptional%2520Complex%2520that%2520drives%2520a%2520Protumorigenic%2520Gene%2520Expression%2520Signature%2520in%2520Neuroblastoma%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5143%26epage%3D5154%26doi%3D10.1158%2F0008-5472.CAN-15-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carugo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genovese, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nezi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicalese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettazzoni, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akdemir, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bristow, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estecio, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellino, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Francesco, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alesio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelicci, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanfrancone, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G. F.</span></span> <span> </span><span class="NLM_article-title">In Vivo Functional Platform Targeting Patient-derived Xenografts identifies WDR5-Myc Association as a critical Determinant of Pancreatic Cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.celrep.2016.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27320920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVWltrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=133-147&author=A.+Carugoauthor=G.+Genoveseauthor=S.+Sethauthor=L.+Neziauthor=J.+L.+Roseauthor=D.+Bossiauthor=A.+Cicaleseauthor=P.+K.+Shahauthor=A.+Vialeauthor=P.+F.+Pettazzoniauthor=K.+C.+Akdemirauthor=C.+A.+Bristowauthor=F.+S.+Robinsonauthor=J.+Tepperauthor=N.+Sanchezauthor=S.+Guptaauthor=M.+R.+Estecioauthor=V.+Giulianiauthor=G.+I.+Dellinoauthor=L.+Rivaauthor=W.+Yaoauthor=M.+E.+Di+Francescoauthor=T.+Greenauthor=C.+D%E2%80%99Alesioauthor=D.+Cortiauthor=Y.+Kangauthor=P.+Jonesauthor=H.+Wangauthor=J.+B.+Flemingauthor=A.+Maitraauthor=P.+G.+Pelicciauthor=L.+Chinauthor=R.+A.+DePinhoauthor=L.+Lanfranconeauthor=T.+P.+Heffernanauthor=G.+F.+Draetta&title=In+Vivo+Functional+Platform+Targeting+Patient-derived+Xenografts+identifies+WDR5-Myc+Association+as+a+critical+Determinant+of+Pancreatic+Cancer&doi=10.1016%2Fj.celrep.2016.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Functional Platform Targeting Patient-Derived Xenografts Identifies WDR5-Myc Association as a Critical Determinant of Pancreatic Cancer</span></div><div class="casAuthors">Carugo, Alessandro; Genovese, Giannicola; Seth, Sahil; Nezi, Luigi; Rose, Johnathon Lynn; Bossi, Daniela; Cicalese, Angelo; Shah, Parantu Krushnakant; Viale, Andrea; Pettazzoni, Piergiorgio Francesco; Akdemir, Kadir Caner; Bristow, Christopher Aaron; Robinson, Frederick Scott; Tepper, James; Sanchez, Nora; Gupta, Sonal; Estecio, Marcos Roberto; Giuliani, Virginia; Dellino, Gaetano Ivan; Riva, Laura; Yao, Wantong; Di Francesco, Maria Emilia; Green, Tessa; D'Alesio, Carolina; Corti, Denise; Kang, Ya-an; Jones, Philip; Wang, Huamin; Fleming, Jason Bates; Maitra, Anirban; Pelicci, Pier Giuseppe; Chin, Lynda; DePinho, Ronald Anthony; Lanfrancone, Luisa; Heffernan, Timothy Paul; Draetta, Giulio Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-147</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Current treatment regimens for pancreatic ductal adenocarcinoma (PDAC) yield poor 5-yr survival, emphasizing the crit. need to identify druggable targets essential for PDAC maintenance.  We developed an unbiased and in vivo target discovery approach to identify mol. vulnerabilities in low-passage and patient-derived PDAC xenografts or genetically engineered mouse model-derived allografts.  Focusing on epigenetic regulators, we identified WDR5, a core member of the COMPASS histone H3 Lys4 (H3K4) MLL (1-4) methyltransferase complex, as a top tumor maintenance hit required across multiple human and mouse tumors.  Mechanistically, WDR5 functions to sustain proper execution of DNA replication in PDAC cells, as previously suggested by replication stress studies involving MLL1, and c-Myc, also found to interact with WDR5.  We indeed demonstrate that interaction with c-Myc is crit. for this function.  By showing that ATR inhibition mimicked the effects of WDR5 suppression, these data provide rationale to test ATR and WDR5 inhibitors for activity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgPl1yxAd8rVg90H21EOLACvtfcHk0ljc_xybMqCXVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVWltrnK&md5=7b9a7dea457be616aa4f103740b2be02</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCarugo%26aufirst%3DA.%26aulast%3DGenovese%26aufirst%3DG.%26aulast%3DSeth%26aufirst%3DS.%26aulast%3DNezi%26aufirst%3DL.%26aulast%3DRose%26aufirst%3DJ.%2BL.%26aulast%3DBossi%26aufirst%3DD.%26aulast%3DCicalese%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DP.%2BK.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DPettazzoni%26aufirst%3DP.%2BF.%26aulast%3DAkdemir%26aufirst%3DK.%2BC.%26aulast%3DBristow%26aufirst%3DC.%2BA.%26aulast%3DRobinson%26aufirst%3DF.%2BS.%26aulast%3DTepper%26aufirst%3DJ.%26aulast%3DSanchez%26aufirst%3DN.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DEstecio%26aufirst%3DM.%2BR.%26aulast%3DGiuliani%26aufirst%3DV.%26aulast%3DDellino%26aufirst%3DG.%2BI.%26aulast%3DRiva%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DDi%2BFrancesco%26aufirst%3DM.%2BE.%26aulast%3DGreen%26aufirst%3DT.%26aulast%3DD%25E2%2580%2599Alesio%26aufirst%3DC.%26aulast%3DCorti%26aufirst%3DD.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFleming%26aufirst%3DJ.%2BB.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DPelicci%26aufirst%3DP.%2BG.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DLanfrancone%26aufirst%3DL.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DDraetta%26aufirst%3DG.%2BF.%26atitle%3DIn%2520Vivo%2520Functional%2520Platform%2520Targeting%2520Patient-derived%2520Xenografts%2520identifies%2520WDR5-Myc%2520Association%2520as%2520a%2520critical%2520Determinant%2520of%2520Pancreatic%2520Cancer%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D133%26epage%3D147%26doi%3D10.1016%2Fj.celrep.2016.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suganuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J. L.</span></span> <span> </span><span class="NLM_article-title">Diverse Functions of WD40 Repeat Proteins in Histone Recognition</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1268</span>, <span class="refDoi"> DOI: 10.1101/gad.1676208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1101%2Fgad.1676208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18483215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVWmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1265-1268&author=T.+Suganumaauthor=S.+G.+Pattendenauthor=J.+L.+Workman&title=Diverse+Functions+of+WD40+Repeat+Proteins+in+Histone+Recognition&doi=10.1101%2Fgad.1676208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse functions of WD40 repeat proteins in histone recognition</span></div><div class="casAuthors">Suganuma, Tamaki; Pattenden, Samantha G.; Workman, Jerry L.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1265-1268</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  WD40 repeat proteins have been shown to bind the histone H3 tail at the center of their β-propeller structure.  In contrast, in this issue of Genes & Development, J. J. Song and colleagues (ibid., 1313-1318) demonstrate that the WD40 repeat protein p55 binds a structured region of H4 through a novel binding pocket on the side of β-propeller, illustrating a diversity of histone recognition by WD40 repeat proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomI1QwxcjGi7Vg90H21EOLACvtfcHk0lh_eUPEt7V0HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVWmsb0%253D&md5=9bf2255697ab857bb71ec76e6516405c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.1676208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1676208%26sid%3Dliteratum%253Aachs%26aulast%3DSuganuma%26aufirst%3DT.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DWorkman%26aufirst%3DJ.%2BL.%26atitle%3DDiverse%2520Functions%2520of%2520WD40%2520Repeat%2520Proteins%2520in%2520Histone%2520Recognition%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D1265%26epage%3D1268%26doi%3D10.1101%2Fgad.1676208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">A conserved Arginine-containing Motif crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32162</span>– <span class="NLM_lpage">32175</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+Arginine-containing+Motif+crucial+for+the+Assembly+and+Enzymatic+Activity+of+the+Mixed+Lineage+Leukemia+Protein-1+Core+Complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0lh_eUPEt7V0HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520Arginine-containing%2520Motif%2520crucial%2520for%2520the%2520Assembly%2520and%2520Enzymatic%2520Activity%2520of%2520the%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the Von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules to disrupt the VHL/HIF-1α Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+Von+Hippel-Lindau+E3+Ubiquitin+Ligase+Using+Small+Molecules+to+disrupt+the+VHL%2FHIF-1%CE%B1+Interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0lj3-n3Oquj0yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3Dvan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520Von%2520Hippel-Lindau%2520E3%2520Ubiquitin%2520Ligase%2520Using%2520Small%2520Molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%25CE%25B1%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozilovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diebold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudvarski Stankovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baluapuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönemann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhandare, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotriffer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E.</span></span> <span> </span><span class="NLM_article-title">PROTAC-mediated Degradation reveals a non-catalytic Function of AURORA-A Kinase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1179</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-00652-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fs41589-020-00652-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=32989298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFegurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1179-1188&author=B.+Adhikariauthor=J.+Bozilovicauthor=M.+Dieboldauthor=J.+D.+Schwarzauthor=J.+Hofstetterauthor=M.+Schr%C3%B6derauthor=M.+Waniorauthor=A.+Narainauthor=M.+Vogtauthor=N.+Dudvarski+Stankovicauthor=A.+Baluapuriauthor=L.+Sch%C3%B6nemannauthor=L.+Eingauthor=P.+Bhandareauthor=B.+Kusterauthor=A.+Schlosserauthor=S.+Heinzlmeirauthor=C.+Sotrifferauthor=S.+Knappauthor=E.+Wolf&title=PROTAC-mediated+Degradation+reveals+a+non-catalytic+Function+of+AURORA-A+Kinase&doi=10.1038%2Fs41589-020-00652-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-mediated degradation reveals a non-catalytic function of AURORA-A kinase</span></div><div class="casAuthors">Adhikari, Bikash; Bozilovic, Jelena; Diebold, Mathias; Schwarz, Jessica Denise; Hofstetter, Julia; Schroeder, Martin; Wanior, Marek; Narain, Ashwin; Vogt, Markus; Dudvarski Stankovic, Nevenka; Baluapuri, Apoorva; Schoenemann, Lars; Eing, Lorenz; Bhandare, Pranjali; Kuster, Bernhard; Schlosser, Andreas; Heinzlmeir, Stephanie; Sotriffer, Christoph; Knapp, Stefan; Wolf, Elmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1179-1188</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The mitotic kinase AURORA-A is essential for cell cycle progression and is considered a priority cancer target.  Although the catalytic activity of AURORA-A is essential for its mitotic function, recent reports indicate an addnl. non-catalytic function, which is difficult to target by conventional small mols.  We therefore developed a series of chem. degraders (PROTACs) by connecting a clin. kinase inhibitor of AURORA-A to E3 ligase-binding mols. (for example, thalidomide).  One degrader induced rapid, durable and highly specific degrdn. of AURORA-A.  In addn., we found that the degrader complex was stabilized by cooperative binding between AURORA-A and CEREBLON.  Degrader-mediated AURORA-A depletion caused an S-phase defect, which is not the cell cycle effect obsd. upon kinase inhibition, supporting an important non-catalytic function of AURORA-A during DNA replication.  AURORA-A degrdn. induced rampant apoptosis in cancer cell lines and thus represents a versatile starting point for developing new therapeutics to counter AURORA-A function in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSD0j67urh7Vg90H21EOLACvtfcHk0lj3-n3Oquj0yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFegurvM&md5=3cdb2fa0d7c7e800cd3936765218b5d7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00652-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00652-y%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DB.%26aulast%3DBozilovic%26aufirst%3DJ.%26aulast%3DDiebold%26aufirst%3DM.%26aulast%3DSchwarz%26aufirst%3DJ.%2BD.%26aulast%3DHofstetter%26aufirst%3DJ.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DNarain%26aufirst%3DA.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DDudvarski%2BStankovic%26aufirst%3DN.%26aulast%3DBaluapuri%26aufirst%3DA.%26aulast%3DSch%25C3%25B6nemann%26aufirst%3DL.%26aulast%3DEing%26aufirst%3DL.%26aulast%3DBhandare%26aufirst%3DP.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DSchlosser%26aufirst%3DA.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DSotriffer%26aufirst%3DC.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DE.%26atitle%3DPROTAC-mediated%2520Degradation%2520reveals%2520a%2520non-catalytic%2520Function%2520of%2520AURORA-A%2520Kinase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1179%26epage%3D1188%26doi%3D10.1038%2Fs41589-020-00652-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided Design and Optimization of Small Molecules targeting the Protein-Protein Interaction between the Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in vitro Nanomolar Affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+Design+and+Optimization+of+Small+Molecules+targeting+the+Protein-Protein+Interaction+between+the+Von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase+and+the+Hypoxia+Inducible+Factor+%28HIF%29+Alpha+Subunit+with+in+vitro+Nanomolar+Affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhreNR6yeqpSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3Dvan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520Design%2520and%2520Optimization%2520of%2520Small%2520Molecules%2520targeting%2520the%2520Protein-Protein%2520Interaction%2520between%2520the%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%2520and%2520the%2520Hypoxia%2520Inducible%2520Factor%2520%2528HIF%2529%2520Alpha%2520Subunit%2520with%2520in%2520vitro%2520Nanomolar%2520Affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The Use of Differential Scanning Fluorimetry to detect Ligand Interactions that promote Protein Stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+Use+of+Differential+Scanning+Fluorimetry+to+detect+Ligand+Interactions+that+promote+Protein+Stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lhreNR6yeqpSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520Use%2520of%2520Differential%2520Scanning%2520Fluorimetry%2520to%2520detect%2520Ligand%2520Interactions%2520that%2520promote%2520Protein%2520Stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid Measurement of Binding Constants and Heats of Binding using a new Titration Calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+Measurement+of+Binding+Constants+and+Heats+of+Binding+using+a+new+Titration+Calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0li4u_EcMjPnzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520Measurement%2520of%2520Binding%2520Constants%2520and%2520Heats%2520of%2520Binding%2520using%2520a%2520new%2520Titration%2520Calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214.e11</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214.e11&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+Wide-Spectrum+Kinase+Profiling+in+Live+Cells+for+Assessing+the+Effect+of+Cellular+ATP+on+Target+Engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0li4u_EcMjPnzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520Wide-Spectrum%2520Kinase%2520Profiling%2520in%2520Live%2520Cells%2520for%2520Assessing%2520the%2520Effect%2520of%2520Cellular%2520ATP%2520on%2520Target%2520Engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214.e11%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target Engagement and Drug Residence Time can be observed in living Cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">10091</span> <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+Engagement+and+Drug+Residence+Time+can+be+observed+in+living+Cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0lhyVmdrAa1BKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520Engagement%2520and%2520Drug%2520Residence%2520Time%2520can%2520be%2520observed%2520in%2520living%2520Cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoefer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boychenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">Robust Microflow LC-MS/MS for Proteome Analysis: 38 000 Runs and Counting</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">3686</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.1c00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.1c00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktFemtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2021&pages=3686-3690&author=Y.+Bianauthor=F.+P.+Bayerauthor=Y.-C.+Changauthor=C.+Mengauthor=S.+Hoeferauthor=N.+Dengauthor=R.+Zhengauthor=O.+Boychenkoauthor=B.+Kuster&title=Robust+Microflow+LC-MS%2FMS+for+Proteome+Analysis%3A+38+000+Runs+and+Counting&doi=10.1021%2Facs.analchem.1c00257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Robust Microflow LC-MS/MS for Proteome Analysis: 38 000 Runs and Counting</span></div><div class="casAuthors">Bian, Yangyang; Bayer, Florian P.; Chang, Yun-Chien; Meng, Chen; Hoefer, Stefanie; Deng, Nan; Zheng, Runsheng; Boychenko, Oleksandr; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3686-3690</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Microflow liq. chromatog. tandem mass spectrometry (μLC-MS/MS) is becoming a viable alternative to nanoflow LC-MS/MS for the anal. of proteomes.  We have recently demonstrated the potential of such a system operating with a 1 mm i.d. x 150 mm column and at a flow rate of 50 μL/min for high-throughput applications.  On the basis of the anal. of ~ 38 000 samples measured on two instruments over the past two years, we now show that the approach is extremely robust.  Up to 1500 analyses were performed within one month, and >14 000 samples could be analyzed on a single column without loss of chromatog. performance.  Samples included proteomes of cell lines, tissues, and human body fluids, which were analyzed with or without prior peptide fractionation or stable isotope labeling.  We show that the μLC-MS/MS system is capable of measuring 2600 proteins from undepleted human plasma and ~ 5000 proteins from crude human urine in 1 day, demonstrating its potential for in-depth as well as high-throughput clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8VCCnqHc2BbVg90H21EOLACvtfcHk0lhL-XQ95artmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXktFemtbk%253D&md5=5e3e071c1e167a43b0a2a2a2930e4671</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.1c00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.1c00257%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DY.%26aulast%3DBayer%26aufirst%3DF.%2BP.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DHoefer%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBoychenko%26aufirst%3DO.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DRobust%2520Microflow%2520LC-MS%252FMS%2520for%2520Proteome%2520Analysis%253A%252038%2520000%2520Runs%2520and%2520Counting%26jtitle%3DAnal.%2520Chem.%26date%3D2021%26volume%3D93%26spage%3D3686%26epage%3D3690%26doi%3D10.1021%2Facs.analchem.1c00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baynham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boychenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">Robust, Reproducible and Quantitative Analysis of Thousands of Proteomes by Micro-Flow LC-MS/MS</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">157</span> <span class="refDoi"> DOI: 10.1038/s41467-019-13973-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fs41467-019-13973-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=31919466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1ynsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=Y.+Bianauthor=R.+Zhengauthor=F.+P.+Bayerauthor=C.+Wongauthor=Y.-C.+Changauthor=C.+Mengauthor=D.+P.+Zolgauthor=M.+Reineckeauthor=J.+Zechaauthor=S.+Wiechmannauthor=S.+Heinzlmeirauthor=J.+Scherrauthor=B.+Hemmerauthor=M.+Baynhamauthor=A.-C.+Gingrasauthor=O.+Boychenkoauthor=B.+Kuster&title=Robust%2C+Reproducible+and+Quantitative+Analysis+of+Thousands+of+Proteomes+by+Micro-Flow+LC-MS%2FMS&doi=10.1038%2Fs41467-019-13973-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Robust, reproducible and quantitative analysis of thousands of proteomes by micro-flow LC-MS/MS</span></div><div class="casAuthors">Bian, Yangyang; Zheng, Runsheng; Bayer, Florian P.; Wong, Cassandra; Chang, Yun-Chien; Meng, Chen; Zolg, Daniel P.; Reinecke, Maria; Zecha, Jana; Wiechmann, Svenja; Heinzlmeir, Stephanie; Scherr, Johannes; Hemmer, Bernhard; Baynham, Mike; Gingras, Anne-Claude; Boychenko, Oleksandr; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nano-flow liq. chromatog. tandem mass spectrometry (nano-flow LC-MS/MS) is the mainstay in proteome research because of its excellent sensitivity but often comes at the expense of robustness.  Here we show that micro-flow LC-MS/MS using a 1 x 150 mm column shows excellent reproducibility of chromatog. retention time (<0.3% coeff. of variation, CV) and protein quantification (<7.5% CV) using data from >2000 samples of human cell lines, tissues and body fluids.  Deep proteome anal. identifies >9000 proteins and >120,000 peptides in 16 h and sample multiplexing using tandem mass tags increases throughput to 11 proteomes in 16 h.  The system identifies >30,000 phosphopeptides in 12 h and protein-protein or protein-drug interaction expts. can be analyzed in 20 min per sample.  We show that the same column can be used to analyze >7500 samples without apparent loss of performance.  This study demonstrates that micro-flow LC-MS/MS is suitable for a broad range of proteomic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOFIaSfJ3g17Vg90H21EOLACvtfcHk0lhL-XQ95artmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1ynsA%253D%253D&md5=70f125faf86cdc8ff71b86241636b881</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-13973-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-13973-x%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DBayer%26aufirst%3DF.%2BP.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DWiechmann%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DScherr%26aufirst%3DJ.%26aulast%3DHemmer%26aufirst%3DB.%26aulast%3DBaynham%26aufirst%3DM.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DBoychenko%26aufirst%3DO.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DRobust%252C%2520Reproducible%2520and%2520Quantitative%2520Analysis%2520of%2520Thousands%2520of%2520Proteomes%2520by%2520Micro-Flow%2520LC-MS%252FMS%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-019-13973-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Médard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemeer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">DMSO enhances Electrospray Response, boosting Sensitivity of Proteomic Experiments</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">991</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnmeth.2610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=23975139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWkurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=989-991&author=H.+Hahneauthor=F.+Pachlauthor=B.+Ruprechtauthor=S.+K.+Maierauthor=S.+Klaegerauthor=D.+Helmauthor=G.+M%C3%A9dardauthor=M.+Wilmauthor=S.+Lemeerauthor=B.+Kuster&title=DMSO+enhances+Electrospray+Response%2C+boosting+Sensitivity+of+Proteomic+Experiments&doi=10.1038%2Fnmeth.2610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DMSO enhances electrospray response, boosting sensitivity of proteomic experiments</span></div><div class="casAuthors">Hahne, Hannes; Pachl, Fiona; Ruprecht, Benjamin; Maier, Stefan K.; Klaeger, Susan; Helm, Dominic; Medard, Guillaume; Wilm, Matthias; Lemeer, Simone; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">989-991</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report that low percentages of dimethylsulfoxide (DMSO) in liq. chromatog. solvents lead to a strong enhancement of electrospray ionization of peptides, improving the sensitivity of protein identification in bottom-up proteomics by up to tenfold.  The method can be easily implemented on any LC-MS/MS system without modification to hardware or software and at no addnl. cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYxQsbxyeCubVg90H21EOLACvtfcHk0ljvd-bniQKIAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWkurjJ&md5=dd957dd264736e8bc48b66df2355363e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2610%26sid%3Dliteratum%253Aachs%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DMaier%26aufirst%3DS.%2BK.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DM%25C3%25A9dard%26aufirst%3DG.%26aulast%3DWilm%26aufirst%3DM.%26aulast%3DLemeer%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DDMSO%2520enhances%2520Electrospray%2520Response%252C%2520boosting%2520Sensitivity%2520of%2520Proteomic%2520Experiments%26jtitle%3DNat.%2520Methods%26date%3D2013%26volume%3D10%26spage%3D989%26epage%3D991%26doi%3D10.1038%2Fnmeth.2610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">MaxQuant enables high Peptide Identification Rates, Individualized p.p.b.-Range Mass Accuracies and Proteome-wide Protein Quantification</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1372</span>, <span class="refDoi"> DOI: 10.1038/nbt.1511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnbt.1511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=19029910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1367-1372&author=J.+Coxauthor=M.+Mann&title=MaxQuant+enables+high+Peptide+Identification+Rates%2C+Individualized+p.p.b.-Range+Mass+Accuracies+and+Proteome-wide+Protein+Quantification&doi=10.1038%2Fnbt.1511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</span></div><div class="casAuthors">Cox, Juergen; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1367-1372</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Efficient anal. of very large amts. of raw data for peptide identification and protein quantification is a principal challenge in mass spectrometry (MS)-based proteomics.  Here we describe MaxQuant, an integrated suite of algorithms specifically developed for high-resoln., quant. MS data.  Using correlation anal. and graph theory, MaxQuant detects peaks, isotope clusters and stable amino acid isotope-labeled (SILAC) peptide pairs as three-dimensional objects in m/z, elution time and signal intensity space.  By integrating multiple mass measurements and correcting for linear and nonlinear mass offsets, we achieve mass accuracy in the p.p.b. range, a sixfold increase over std. techniques.  We increase the proportion of identified fragmentation spectra to 73% for SILAC peptide pairs via unambiguous assignment of isotope and missed-cleavage state and individual mass precision.  MaxQuant automatically quantifies several hundred thousand peptides per SILAC-proteome expt. and allows statistically robust identification and quantification of >4000 proteins in mammalian cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzVwiKHxD8XbVg90H21EOLACvtfcHk0ljvd-bniQKIAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWjtLzJ&md5=675d31ca84e3a7e4fb9bdd601d8075ea</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1511%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DMaxQuant%2520enables%2520high%2520Peptide%2520Identification%2520Rates%252C%2520Individualized%2520p.p.b.-Range%2520Mass%2520Accuracies%2520and%2520Proteome-wide%2520Protein%2520Quantification%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D1367%26epage%3D1372%26doi%3D10.1038%2Fnbt.1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhauser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheltema, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">Andromeda: a Peptide Search Engine integrated into the MaxQuant Environment</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1794</span>– <span class="NLM_lpage">1805</span>, <span class="refDoi"> DOI: 10.1021/pr101065j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr101065j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gis74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=1794-1805&author=J.+Coxauthor=N.+Neuhauserauthor=A.+Michalskiauthor=R.+A.+Scheltemaauthor=J.+V.+Olsenauthor=M.+Mann&title=Andromeda%3A+a+Peptide+Search+Engine+integrated+into+the+MaxQuant+Environment&doi=10.1021%2Fpr101065j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Andromeda: A Peptide Search Engine Integrated into the MaxQuant Environment</span></div><div class="casAuthors">Cox, Juergen; Neuhauser, Nadin; Michalski, Annette; Scheltema, Richard A.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1794-1805</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key step in mass spectrometry (MS)-based proteomics is the identification of peptides in sequence databases by their fragmentation spectra.  Here the authors describe Andromeda, a novel peptide search engine using a probabilistic scoring model.  On proteome data, Andromeda performs as well as Mascot, a widely used com. search engine, as judged by sensitivity and specificity anal. based on target decoy searches.  Furthermore, it can handle data with arbitrarily high fragment mass accuracy, is able to assign and score complex patterns of post-translational modifications, such as highly phosphorylated peptides, and accommodates extremely large databases.  The algorithms of Andromeda are provided.  Andromeda can function independently or as an integrated search engine of the widely used MaxQuant computational proteomics platform and both are freely available at www.maxquant.org.  The combination enables anal. of large data sets in a simple anal. workflow on a desktop computer.  For searching individual spectra Andromeda is also accessible via a web server.  The authors demonstrate the flexibility of the system by implementing the capability to identify cofragmented peptides, significantly improving the total no. of identified peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4zTprqNti7Vg90H21EOLACvtfcHk0ljvd-bniQKIAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gis74%253D&md5=7587da6364fe0ff020e3dbf1d80bb22f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fpr101065j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr101065j%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DNeuhauser%26aufirst%3DN.%26aulast%3DMichalski%26aufirst%3DA.%26aulast%3DScheltema%26aufirst%3DR.%2BA.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DAndromeda%253A%2520a%2520Peptide%2520Search%2520Engine%2520integrated%2520into%2520the%2520MaxQuant%2520Environment%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D1794%26epage%3D1805%26doi%3D10.1021%2Fpr101065j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyanova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinitcyn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span> <span> </span><span class="NLM_article-title">The Perseus Computational Platform for comprehensive Analysis of (Prote)omics Data</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">740</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1038%2Fnmeth.3901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=27348712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKntbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=731-740&author=S.+Tyanovaauthor=T.+Temuauthor=P.+Sinitcynauthor=A.+Carlsonauthor=M.+Y.+Heinauthor=T.+Geigerauthor=M.+Mannauthor=J.+Cox&title=The+Perseus+Computational+Platform+for+comprehensive+Analysis+of+%28Prote%29omics+Data&doi=10.1038%2Fnmeth.3901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The Perseus computational platform for comprehensive analysis of (prote)omics data</span></div><div class="casAuthors">Tyanova, Stefka; Temu, Tikira; Sinitcyn, Pavel; Carlson, Arthur; Hein, Marco Y.; Geiger, Tamar; Mann, Matthias; Cox, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">731-740</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A main bottleneck in proteomics is the downstream biol. anal. of highly multivariate quant. protein abundance data generated using mass-spectrometry-based anal.  We developed the Perseus software platform (http://www.perseus-framework.org) to support biol. and biomedical researchers in interpreting protein quantification, interaction and post-translational modification data.  Perseus contains a comprehensive portfolio of statistical tools for high-dimensional omics data anal. covering normalization, pattern recognition, time-series anal., cross-omics comparisons and multiple-hypothesis testing.  A machine learning module supports the classification and validation of patient groups for diagnosis and prognosis, and it also detects predictive protein signatures.  Central to Perseus is a user-friendly, interactive workflow environment that provides complete documentation of computational methods used in a publication.  All activities in Perseus are realized as plugins, and users can extend the software by programming their own, which can be shared through a plugin store.  We anticipate that Perseus's arsenal of algorithms and its intuitive usability will empower interdisciplinary anal. of complex large data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVXoplfOxdPLVg90H21EOLACvtfcHk0lhDGa3hJ2ovFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKntbnN&md5=f8c3e2876e4d724518054bb1a2d1e6ee</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3901%26sid%3Dliteratum%253Aachs%26aulast%3DTyanova%26aufirst%3DS.%26aulast%3DTemu%26aufirst%3DT.%26aulast%3DSinitcyn%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DA.%26aulast%3DHein%26aufirst%3DM.%2BY.%26aulast%3DGeiger%26aufirst%3DT.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DJ.%26atitle%3DThe%2520Perseus%2520Computational%2520Platform%2520for%2520comprehensive%2520Analysis%2520of%2520%2528Prote%2529omics%2520Data%26jtitle%3DNat.%2520Methods%26date%3D2016%26volume%3D13%26spage%3D731%26epage%3D740%26doi%3D10.1038%2Fnmeth.3901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vizcaíno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csordas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del-Toro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dianes, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavidas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Riverol, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ternent, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span> <span> </span><span class="NLM_article-title">2016 Update of the PRIDE Database and its related Tools</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D447</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fnar%2Fgkv1145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=26527722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D447-56&author=J.+A.+Vizca%C3%ADnoauthor=A.+Csordasauthor=N.+del-Toroauthor=J.+A.+Dianesauthor=J.+Grissauthor=I.+Lavidasauthor=G.+Mayerauthor=Y.+Perez-Riverolauthor=F.+Reisingerauthor=T.+Ternentauthor=Q.-W.+Xuauthor=R.+Wangauthor=H.+Hermjakob&title=2016+Update+of+the+PRIDE+Database+and+its+related+Tools&doi=10.1093%2Fnar%2Fgkv1145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">2016 update of the PRIDE database and its related tools</span></div><div class="casAuthors">Vizcaino, Juan Antonio; Csordas, Attila; del-Toro, Noemi; Dianes, Jose A.; Griss, Johannes; Lavidas, Ilias; Mayer, Gerhard; Perez-Riverol, Yasset; Reisinger, Florian; Ternent, Tobias; Xu, Qing-Wei; Wang, Rui; Hermjakob, Henning</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D447-D456</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The PRoteomics IDEntifications (PRIDE) database is one of the world-leading data repositories of mass spectrometry (MS)-based proteomics data.  Since the beginning of 2014, PRIDE Archive (http://www.ebi.ac.uk/pride/archive/) is the new PRIDE archival system, replacing the original PRIDE database.  Here we summarize the developments in PRIDE resources and related tools since the previous update manuscript in the Database Issue in 2013.  PRIDE Archive constitutes a complete redevelopment of the original PRIDE, comprising a new storage backend, data submission system and web interface, among other components.  PRIDE Archive supports the most widely used PSI (Proteomics Stds. Initiative) data std. formats (mzML and mzIdentML) and implements the data requirements and guidelines of the ProteomeXchange Consortium.  The wide adoption of ProteomeXchange within the community has triggered an unprecedented increase in the no. of submitted data sets (around 150 data sets per mo).  We outline some statistics on the current PRIDE Archive data contents.  We also report on the status of the PRIDE related stand-alone tools: PRIDE Inspector, PRIDE Converter 2 and the ProteomeXchange submission tool.  Finally, we will give a brief update on the resources under development 'PRIDE Cluster' and 'PRIDE Proteomes', which provide a complementary view and quality-scored information of the peptide and protein identification data available in PRIDE Archive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_BbPYtr63s7Vg90H21EOLACvtfcHk0lhDGa3hJ2ovFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7vF&md5=946a9ff49868659b6de6fb24d6a282ef</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1145%26sid%3Dliteratum%253Aachs%26aulast%3DVizca%25C3%25ADno%26aufirst%3DJ.%2BA.%26aulast%3DCsordas%26aufirst%3DA.%26aulast%3Ddel-Toro%26aufirst%3DN.%26aulast%3DDianes%26aufirst%3DJ.%2BA.%26aulast%3DGriss%26aufirst%3DJ.%26aulast%3DLavidas%26aufirst%3DI.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DPerez-Riverol%26aufirst%3DY.%26aulast%3DReisinger%26aufirst%3DF.%26aulast%3DTernent%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DQ.-W.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DHermjakob%26aufirst%3DH.%26atitle%3D2016%2520Update%2520of%2520the%2520PRIDE%2520Database%2520and%2520its%2520related%2520Tools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD447%26epage%3D56%26doi%3D10.1093%2Fnar%2Fgkv1145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span> <i>Molecular Operating Environment (MOE)</i>; <span class="NLM_publisher-name">Chemical
Computing Group ULC</span>: <span class="NLM_publisher-loc">1010 Sherbooke
St West, Suite #910, Montreal, OC, Canada, H3A 2R7</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%3B+Chemical%0AComputing+Group+ULC%3A+1010+Sherbooke%0ASt+West%2C+Suite+%23910%2C+Montreal%2C+OC%2C+Canada%2C+H3A+2R7%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%250AComputing%2520Group%2520ULC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Protonate3D: Assignment of Ionization States and Hydrogen Coordinates to Macromolecular Structures</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+Assignment+of+Ionization+States+and+Hydrogen+Coordinates+to+Macromolecular+Structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0lhDGa3hJ2ovFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520Assignment%2520of%2520Ionization%2520States%2520and%2520Hydrogen%2520Coordinates%2520to%2520Macromolecular%2520Structures%26jtitle%3DProteins%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lhDGa3hJ2ovFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">DSX: A Knowledge-based Scoring Function for the Assessment of Protein-Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2731</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/ci200274q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200274q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2731-2745&author=G.+Neudertauthor=G.+Klebe&title=DSX%3A+A+Knowledge-based+Scoring+Function+for+the+Assessment+of+Protein-Ligand+Complexes&doi=10.1021%2Fci200274q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein-Ligand Complexes</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2731-2745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the new knowledge-based scoring function DSX that consists of distance-dependent pair potentials, novel torsion angle potentials, and newly defined solvent accessible surface-dependent potentials.  DSX pair potentials are based on the statistical formalism of DrugScore, extended by a much more specialized set of atom types.  The original DrugScore-like ref. state is rather unstable with respect to modifications in the used atom types.  Therefore, an important method to overcome this problem and to allow for robust results when deriving pair potentials for arbitrary sets of atom types is presented.  A validation based on a carefully prepd. test set is shown, enabling direct comparison to the majority of other popular scoring functions.  Here, DSX features superior performance with respect to docking- and ranking power and runtime requirements.  Furthermore, the beneficial combination with torsion angle-dependent and desolvation-dependent potentials is demonstrated.  DSX is robust, flexible, and capable of working together with special features of popular docking engines, e.g., flexible protein residues in AutoDock or GOLD.  The program is freely available to the scientific community and can be downloaded from our Web site www.agklebe.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvQyrP7qPi5bVg90H21EOLACvtfcHk0lj0mp0UsaqUSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K&md5=c90af0aa9af9ae15d458d9ba759163bb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci200274q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200274q%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDSX%253A%2520A%2520Knowledge-based%2520Scoring%2520Function%2520for%2520the%2520Assessment%2520of%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2731%26epage%3D2745%26doi%3D10.1021%2Fci200274q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">fconv: Format Conversion, Manipulation and Feature Computation of Molecular Data</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=10.1093%2Fbioinformatics%2Fbtr055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=21335612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1yqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1021-1022&author=G.+Neudertauthor=G.+Klebe&title=fconv%3A+Format+Conversion%2C+Manipulation+and+Feature+Computation+of+Molecular+Data&doi=10.1093%2Fbioinformatics%2Fbtr055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">fconv: format conversion, manipulation and feature computation of molecular data</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1021-1022</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: fconv is a program intended for parsing and manipulating multiple aspects and properties of mol. data.  Up to now, it has been developed and extensively tested for 3 years.  It has become a very robust and comprehensive tool involved in a broad range of computational workflows that are currently applied in our drug design environment.  Typical tasks are as follows: conversion and error correction of formats such as PDB(QT), MOL2, SDF, DLG and CIF; extg. ligands from PDB as MOL2; automatic or ligand-based cavity detection; rmsd calcn. and clustering; substructure searches; alignment and structural superposition; building of crystal packings; adding hydrogens; calcn. of various properties like the no. of rotatable bonds; mol. wts. or vdW vols.  The atom type classification is based on a consistent assignment of internal atom types, which are by far more differentiated compared with e.g.  Sybyl atom types.  Apart from the predefined mapping of these types onto Sybyl types, the user is able to assign own mappings by providing modified template files, thus allowing for tailor-made atom type sets.  Availability: fconv is free software available under GNU General Public License.  C++ sources and precompiled executables for LINUX/UNIX, Mac OS and Windows, as well as tutorials are available on http://www.agklebe.de.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="ff94939a9d9abf8c8b9e9999d18a9196d2929e8d9d8a8d98d19b9ad1">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmzwJ0Mgz9SLVg90H21EOLACvtfcHk0lj0mp0UsaqUSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1yqsL4%253D&md5=3869574573503ffd1c89d23132c4dfb9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr055%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3Dfconv%253A%2520Format%2520Conversion%252C%2520Manipulation%2520and%2520Feature%2520Computation%2520of%2520Molecular%2520Data%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1021%26epage%3D1022%26doi%3D10.1093%2Fbioinformatics%2Fbtr055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1','PDB','4QL1'); return false;">PDB: 4QL1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DAK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DAK','PDB','6DAK'); return false;">PDB: 6DAK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H','PDB','4W9H'); return false;">PDB: 4W9H</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i88"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00146">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_40786"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00146?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00146</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Thermal shift data; ITC data; BRET assay data; HiBiT data; immunoblotting data; proteomics data; computational data; experimental details for synthesis of all degraders, intermediates and negative controls; appendix with spectra for all degraders, intermediates, and negative controls (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Quantitative proteomics table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_002.csv">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_001.pdf">jm1c00146_si_001.pdf (31.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_002.csv">jm1c00146_si_002.csv (3.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00146/suppl_file/jm1c00146_si_003.csv">jm1c00146_si_003.csv (5.3 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00146&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00146%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00146" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799dc720f103d6c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
