
<html lang="en"     class="pb-page"  data-request-id="f184c3b2-65bd-4f62-8543-e51a6d4df7c9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b01529;issue:issue:10.1021/jmcmar.2019.62.issue-4;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Macrocyclic Inhibitors of Apurinic/Apyrimidinic Endonuclease 1" /></meta><meta name="dc.Creator" content="Richard  Trilles" /></meta><meta name="dc.Creator" content="Dmitri  Beglov" /></meta><meta name="dc.Creator" content="Qiujia  Chen" /></meta><meta name="dc.Creator" content="Hongzhen  He" /></meta><meta name="dc.Creator" content="Randall  Wireman" /></meta><meta name="dc.Creator" content="April  Reed" /></meta><meta name="dc.Creator" content="Spandan  Chennamadhavuni" /></meta><meta name="dc.Creator" content="James S.  Panek" /></meta><meta name="dc.Creator" content="Lauren E.  Brown" /></meta><meta name="dc.Creator" content="Sandor  Vajda" /></meta><meta name="dc.Creator" content="John A.  Porco, Jr." /></meta><meta name="dc.Creator" content="Mark R.  Kelley" /></meta><meta name="dc.Creator" content="Millie M.  Georgiadis" /></meta><meta name="dc.Description" content="Apurinic/apyrimidinic endonuclease 1 (APE1) is an essential base excision repair enzyme that is upregulated in a number of cancers, contributes to resistance of tumors treated with DNA-alkylating o..." /></meta><meta name="Description" content="Apurinic/apyrimidinic endonuclease 1 (APE1) is an essential base excision repair enzyme that is upregulated in a number of cancers, contributes to resistance of tumors treated with DNA-alkylating o..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 17, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01529" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01529" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01529" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01529" /></link>
        
    
    

<title>Discovery of Macrocyclic Inhibitors of Apurinic/Apyrimidinic Endonuclease 1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01529" /></meta><meta property="og:title" content="Discovery of Macrocyclic Inhibitors of Apurinic/Apyrimidinic Endonuclease 1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0007.jpeg" /></meta><meta property="og:description" content="Apurinic/apyrimidinic endonuclease 1 (APE1) is an essential base excision repair enzyme that is upregulated in a number of cancers, contributes to resistance of tumors treated with DNA-alkylating or -oxidizing agents, and has recently been identified as an important therapeutic target. In this work, we identified hot spots for binding of small organic molecules experimentally in high resolution crystal structures of APE1 and computationally through the use of FTMAP analysis (http://ftmap.bu.edu/). Guided by these hot spots, a library of drug-like macrocycles was docked and then screened for inhibition of APE1 endonuclease activity. In an iterative process, hot-spot-guided docking, characterization of inhibition of APE1 endonuclease, and cytotoxicity of cancer cells were used to design next generation macrocycles. To assess target selectivity in cells, selected macrocycles were analyzed for modulation of DNA damage. Taken together, our studies suggest that macrocycles represent a promising class of compounds for inhibition of APE1 in cancer cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01529"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01529">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01529&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01529&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01529&amp;href=/doi/10.1021/acs.jmedchem.8b01529" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1971-1988</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01508" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01541" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Macrocyclic Inhibitors of Apurinic/Apyrimidinic Endonuclease 1</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Richard Trilles</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Trilles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Trilles">Richard Trilles</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dmitri Beglov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dmitri Beglov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dmitri++Beglov">Dmitri Beglov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiujia Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiujia Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiujia++Chen">Qiujia Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongzhen He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongzhen He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongzhen++He">Hongzhen He</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Randall Wireman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Randall Wireman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pediatrics and Herman B. Wells Center for Pediatric Research, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Randall++Wireman">Randall Wireman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">April Reed</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">April Reed</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pediatrics and Herman B. Wells Center for Pediatric Research, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=April++Reed">April Reed</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Spandan Chennamadhavuni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Spandan Chennamadhavuni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Spandan++Chennamadhavuni">Spandan Chennamadhavuni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James S. Panek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James S. Panek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+S.++Panek">James S. Panek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lauren E. Brown</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lauren E. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lauren+E.++Brown">Lauren E. Brown</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9489-484X" title="Orcid link">http://orcid.org/0000-0001-9489-484X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sandor Vajda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sandor Vajda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sandor++Vajda">Sandor Vajda</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John A. Porco Jr.</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John A. Porco, Jr.</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+A.++Porco%2C++Jr.">John A. Porco, Jr.</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2991-5680" title="Orcid link">http://orcid.org/0000-0002-2991-5680</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark R. Kelley</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark R. Kelley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pediatrics and Herman B. Wells Center for Pediatric Research, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><span class="conrtib-corresp"><strong>*</strong>M.R.K.: e-mail, <a href="/cdn-cgi/l/email-protection#e38e88868f8f869aa38a96cd868796"><span class="__cf_email__" data-cfemail="0f62646a63636a764f667a216a6b7a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+R.++Kelley">Mark R. Kelley</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Millie M. Georgiadis</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Millie M. Georgiadis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Purdue School of Science, Indiana University-Purdue University Indianapolis, Indianapolis, Indiana 46202, United States</div></div><span class="conrtib-corresp"><strong>*</strong>M.M.G.: e-mail, <a href="/cdn-cgi/l/email-protection#224f45474d50454b43624b570c474657"><span class="__cf_email__" data-cfemail="aec3c9cbc1dcc9c7cfeec7db80cbcadb">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Millie+M.++Georgiadis">Millie M. Georgiadis</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2976-4576" title="Orcid link">http://orcid.org/0000-0003-2976-4576</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01529&amp;href=/doi/10.1021%2Facs.jmedchem.8b01529" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1971–1988</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 17, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 October 2018</li><li><span class="item_label"><b>Published</b> online</span>17 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 February 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01529" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01529</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1971%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRichard%2BTrilles%252C%2BDmitri%2BBeglov%252C%2BQiujia%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D4%26contentID%3Dacs.jmedchem.8b01529%26title%3DDiscovery%2Bof%2BMacrocyclic%2BInhibitors%2Bof%2BApurinic%252FApyrimidinic%2BEndonuclease%2B1%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1988%26publicationDate%3DFebruary%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01529"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1368</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01529" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Macrocyclic Inhibitors of Apurinic/Apyrimidinic Endonuclease 1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Trilles&quot;},{&quot;first_name&quot;:&quot;Dmitri&quot;,&quot;last_name&quot;:&quot;Beglov&quot;},{&quot;first_name&quot;:&quot;Qiujia&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hongzhen&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Randall&quot;,&quot;last_name&quot;:&quot;Wireman&quot;},{&quot;first_name&quot;:&quot;April&quot;,&quot;last_name&quot;:&quot;Reed&quot;},{&quot;first_name&quot;:&quot;Spandan&quot;,&quot;last_name&quot;:&quot;Chennamadhavuni&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;S. Panek&quot;},{&quot;first_name&quot;:&quot;Lauren&quot;,&quot;last_name&quot;:&quot;E. Brown&quot;},{&quot;first_name&quot;:&quot;Sandor&quot;,&quot;last_name&quot;:&quot;Vajda&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;A. Porco&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;R. Kelley&quot;},{&quot;first_name&quot;:&quot;Millie&quot;,&quot;last_name&quot;:&quot;M. Georgiadis&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;1971-1988&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01529&quot;},&quot;abstract&quot;:&quot;Apurinic/apyrimidinic endonuclease 1 (APE1) is an essential base excision repair enzyme that is upregulated in a number of cancers, contributes to resistance of tumors treated with DNA-alkylating or -oxidizing agents, and has recently been identified as an important therapeutic target. In this work, we identified hot spots for binding of small organic molecules experimentally in high resolution crystal structures of APE1 and computationally through the use of FTMAP analysis (http://ftmap.bu.edu/). Guided by these hot spots, a library of drug-like macrocycles was docked and then screened for inhibition of APE1 endonuclease activity. In an iterative process, hot-spot-guided docking, characterization of inhibition of APE1 endonuclease, and cytotoxicity of cancer cells were used to design next generation macrocycles. To assess target selectivity in cells, selected macrocycles were analyzed for modulation of DNA damage. Taken together, our studies suggest that macrocycles represent a promising class of compounds f&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01529&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01529" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01529&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01529" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01529&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01529" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01529&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01529&amp;href=/doi/10.1021/acs.jmedchem.8b01529" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01529" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01529" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01529%26sid%3Dliteratum%253Aachs%26pmid%3D30653918%26genre%3Darticle%26aulast%3DTrilles%26date%3D2019%26atitle%3DDiscovery%2Bof%2BMacrocyclic%2BInhibitors%2Bof%2BApurinic%252FApyrimidinic%2BEndonuclease%2B1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D4%26spage%3D1971%26epage%3D1988%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291650" title="Alcohols">Alcohols</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/jmcmar.2019.62.issue-4/20190228/jmcmar.2019.62.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Apurinic/apyrimidinic endonuclease 1 (APE1) is an essential base excision repair enzyme that is upregulated in a number of cancers, contributes to resistance of tumors treated with DNA-alkylating or -oxidizing agents, and has recently been identified as an important therapeutic target. In this work, we identified hot spots for binding of small organic molecules experimentally in high resolution crystal structures of APE1 and computationally through the use of FTMAP analysis (<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>). Guided by these hot spots, a library of drug-like macrocycles was docked and then screened for inhibition of APE1 endonuclease activity. In an iterative process, hot-spot-guided docking, characterization of inhibition of APE1 endonuclease, and cytotoxicity of cancer cells were used to design next generation macrocycles. To assess target selectivity in cells, selected macrocycles were analyzed for modulation of DNA damage. Taken together, our studies suggest that macrocycles represent a promising class of compounds for inhibition of APE1 in cancer cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13528" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13528" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Targeting of DNA repair proteins for cancer therapeutic development represents a recent area of interest in drug discovery (reviewed in ref <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a>). It has long been known that DNA repair proteins including apurinic/apyrimidinic endonuclease 1 (APE1) are upregulated in cancer and can mediate resistance to a number of chemotherapeutic agents including those that target DNA directly through alkylation or indirectly by mechanisms such as the creation of reactive oxygen species that react with DNA.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2−5)</a> In its essential role in base excision repair (BER), APE1 catalyzes the Mg<sup>2+</sup>-dependent cleavage of the phosphodiester backbone 5′ of abasic sites that result from removal of damaged bases by glycosylases (reviewed in ref <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>).</div><div class="NLM_p">To date, a number of experimental and in silico high-throughput screens (HTS) to identify selective APE1 endonuclease inhibitors have been reported.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13">(7−13)</a> These efforts have largely focused on the screening of commercially available libraries of small molecules that would be predicted to bind directly to APE1. In an alternative approach, macrocycles, unrelated to those reported here, have been identified that bind directly to an abasic site in duplex DNA preventing APE1 from binding its substrate.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> While a number of APE1-targeting compounds exhibit low micromolar activity, demonstrating selectivity has been challenging.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Many of the existing inhibitors are negatively charged and disrupt other protein–DNA interactions as well as APE1–DNA interactions. Others, such as antimony-containing compounds, are not cell permeable.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Reactive blue 2 dye and myricetin, which inhibit APE1, are also known to bind several cellular targets and are problematic in terms of chemical optimization. Most recently, a novel class of heterocyclic APE1 inhibitors with low micromolar IC<sub>50</sub> values resulting from a focused medicinal chemistry effort was reported.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Existing ligands provide important information about the chemical structure and composition of APE1 endonuclease inhibitors. However, a limitation in the rational design and development of selective APE1 inhibitors remains due to a lack of structural information for APE1-inhibitor complexes. In this study, we used X-ray crystallography and computational solvent mapping to identify hot spots for binding of small organic molecules to APE1. Docking based on consideration of hot spot positioning suggested that macrocycles could bind to the active site of APE1.</div><div class="NLM_p">Among new chemical entities (NCEs) approved as drugs during the period 1981–2006, 60% are natural products and their derivatives.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Evolutionarily driven biosynthesis differs from laboratory organic synthesis, leading to a difference in properties between natural and synthetic compounds.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Natural products often violate the molecular weight limit of less than 500 Da set by Lipinski’s rule of five,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> while remaining pharmacologically active. Many of these violators are macrocycles; it has been observed that macrocycles have an advantage over similarly sized acyclic compounds in terms of pharmacokinetics, solubility, cell permeability, and potency.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> These advantages have been attributed to features such as a diminished entropic penalty on binding, as well as the potential for the dynamic, environmentally driven alteration of physiochemical properties (e.g. intramolecular hydrogen bond-mediated burial of solubilizing polar groups allowing for the traversal of nonpolar membrane environments).<a onclick="showRef(event, 'ref21 ref22 ref23 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref25">(21−23,25)</a> The main appeal of macrocycles as scaffolds for APE1 ligands is in their ability to provide a semiflexible, soluble scaffold linking the structural elements able to interact with the distant binding hot spots on the DNA-binding protein surface. Advances in the synthesis of non-natural macrocycles and their extensive testing in drug discovery<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> contributed to the creation of macrocyclic libraries, which are available both academically and commercially.</div><div class="NLM_p last">In a novel approach, in silico modeling, guided by our solvent bound APE1 X-ray crystal structures as well as computationally docked solvents that defined “hot spots” for binding of organic molecules, suggested that our macrocyclic libraries could serve as starting points for the structure-based design of APE1 ligands. Accordingly, a library of macrocycles was tested for inhibition of APE1 endonuclease activity, and four novel macrocycles with IC<sub>50</sub> values in the low micromolar range were identified. Building on these hits, additional macrocycles were synthesized to establish initial structure–activity relationship (SAR) contributing to the APE1 inhibitory properties of this newly identified chemotype.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55410" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55410" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Crystallographic Solvent Mapping for APE1</h3><div class="NLM_p">With a large number of inhibitors reported for APE1, we sought to provide a structural framework for understanding the nature of the interactions. While inhibitor complex structures have proven elusive, the solvent used to solubilize the inhibitors, DMSO, was bound to the active site of APE1 in numerous structures. We therefore prepared DMSO soaks and obtained a crystal structure of a DMSO-APE1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>) complex determined at 1.48 Å (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In this structure, DMSO binds to two distinct sites within APE1. We refer to the first location within the active site created through interactions with W280, F266, L282, and N174 as the “abasic site or AP (apurinic/apyrimidinic) site” (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B). The sulfoxide O atom of DMSO is hydrogen bonded to the side chain NH of N174 (N–O distance is 2.7 Å) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). N174 is positioned through hydrogen-bonding of its side chain carbonyl to the main chain amide nitrogen of G176 in the repair active site. DMSO bound in the abasic site superimposes directly on the baseless sugar, which is flipped out into a shallow pocket in the structure of the substrate APE1–DNA complex (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The second DMSO binding site is located 35 Å away from the abasic site and involves interactions with D163, F162, L140, and Q137. In this site, the sulfoxide O atom is located 2.7 Å from the main chain N in D163 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf" class="ext-link">Figure S1 in Supporting Information</a>). This second binding site is an isolated small binding pocket on the surface of the protein.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Crystallographic Data<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">DMSO</th><th class="colsep0 rowsep0" align="center">DMSO/Mg<sup>2+</sup></th><th class="colsep0 rowsep0" align="center">DMSO/Tris</th><th class="colsep0 rowsep0" align="center">GLC</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDB code</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i> (Å)</td><td class="colsep0 rowsep0" align="left">46.583</td><td class="colsep0 rowsep0" align="left">46.697</td><td class="colsep0 rowsep0" align="left">46.552</td><td class="colsep0 rowsep0" align="left">46.492</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>b</i> (Å)</td><td class="colsep0 rowsep0" align="left">137.789</td><td class="colsep0 rowsep0" align="left">141.487</td><td class="colsep0 rowsep0" align="left">136.601</td><td class="colsep0 rowsep0" align="left">137.815</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">45.243</td><td class="colsep0 rowsep0" align="left">45.334</td><td class="colsep0 rowsep0" align="left">45.084</td><td class="colsep0 rowsep0" align="left">45.193</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution limit (Å)</td><td class="colsep0 rowsep0" align="left">1.48</td><td class="colsep0 rowsep0" align="left">1.44</td><td class="colsep0 rowsep0" align="left">1.67</td><td class="colsep0 rowsep0" align="left">2.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">97.6 (97.2)</td><td class="colsep0 rowsep0" align="left">98.4 (74.3)</td><td class="colsep0 rowsep0" align="left">95.3(79.4)</td><td class="colsep0 rowsep0" align="left">99.0 (97.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub> (%)</td><td class="colsep0 rowsep0" align="left">6.0 (41.0)</td><td class="colsep0 rowsep0" align="left">4.7 (34.0)</td><td class="colsep0 rowsep0" align="left">7.7 (63.9)</td><td class="colsep0 rowsep0" align="left">7.7 (21.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ</td><td class="colsep0 rowsep0" align="left">22.8(3.0)</td><td class="colsep0 rowsep0" align="left">31.3 (3.4)</td><td class="colsep0 rowsep0" align="left">16.5 (2.1)</td><td class="colsep0 rowsep0" align="left">30.4 (4.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">refinement</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i> (%)</td><td class="colsep0 rowsep0" align="left">19.1</td><td class="colsep0 rowsep0" align="left">18.4</td><td class="colsep0 rowsep0" align="left">20.5</td><td class="colsep0 rowsep0" align="left">17.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">21.9</td><td class="colsep0 rowsep0" align="left">21.2</td><td class="colsep0 rowsep0" align="left">25.0</td><td class="colsep0 rowsep0" align="left">22.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD bonds (Å)</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.007</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD angles (deg)</td><td class="colsep0 rowsep0" align="left">1.03</td><td class="colsep0 rowsep0" align="left">1.177</td><td class="colsep0 rowsep0" align="left">0.934</td><td class="colsep0 rowsep0" align="left">0.810</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i>-factor</td><td class="colsep0 rowsep0" align="left">20.2</td><td class="colsep0 rowsep0" align="left">18.4</td><td class="colsep0 rowsep0" align="left">24.1</td><td class="colsep0 rowsep0" align="left">21.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Highest resolution shells in order of appearance in table: 1.51–1.48, 1.46–1.44, 1.7–1.67, 2.12–2.09.</p></div></div></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystallographic solvent mapping. (A) A molecular surface rendering of APE1, the DMSO bound structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>), with C, light gray, O, red, N, blue, and S, yellow. Superimposed on the molecular surface of APE1 is DNA (from 4IEM) in a stick rendering, C, cyan, P, orange, O, red, and N, blue with the abasic or AP residue in yellow. (B) DMSO bound to the abasic site of APE1. The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 3σ is shown for the crystal structure with DMSO bound to the AP site. A hydrogen bond between the O of DMSO and NH of N174 is indicated by a black dashed line. APE1 is shown as a semitransparent molecular surface rendering. Residues W280, F266, L282 (not labeled, directly underneath the bound DMSO), and N174 line the shallow AP site pocket. (C) DMSO (C, green, S, yellow, O, red) is shown superimposed with the abasic substrate (C, cyan) in stick renderings. (D) DMSO and Mg<sup>2+</sup> bound to the active site of APE1. The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map of the DMSO/Mg<sup>2+</sup> crystal structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>) is shown for DMSO and Mg<sup>2+</sup> contoured at 3σ. Coordinating ligands for Mg<sup>2+</sup> are shown in dashed black lines including D70, E96, and four water molecules. The DMSO is this structure and is not hydrogen bonded to N174. (E) DMSO and DMSO/Mg<sup>2+</sup> are shown superimposed in the abasic site pocket on a surface rendering of APE1. In the presence of Mg<sup>2+</sup>, the DMSO (C, blue) position is slightly shifted relative to that in the structure with just DMSO (C, green) bound. (F) Tris and DMSO bound to the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>). The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for Tris and DMSO is shown contoured at 3σ. Tris is hydrogen bonded to D70 and E96, the Mg<sup>2+</sup> coordinating ligands. (G) Tris/DMSO is shown in stick renderings superimposed on the DMSO bound structure. Mg<sup>2+</sup> from the DMSO/Mg<sup>2+</sup> structure is shown for reference. Tris binds to the Mg<sup>2+</sup> site. DMSO binds very similarly in the two structures. (H) Glycerol bound to the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>). The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map is shown contoured at 3σ. Hydrogen bonds between OH group in glycerol and NH in N174 and N212 are shown in dashed black lines. (I) Glycerol (C, blue) is shown superimposed with DMSO and Mg<sup>2+</sup> from the DMSO/Mg<sup>2+</sup> structure for reference. Glycerol binds deeper in the abasic site pocket than DMSO.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Additional complexes with DMSO include a DMSO/Mg<sup>2+</sup> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>) and a DMSO/Tris complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>). These complexes were obtained by screening for APE1-small molecule complexes, the first with 4-(hydroxymethyl)phenylacetic acid, a compound selected for its potential to bind to the abasic site and to Mg<sup>2+</sup> in the active site, and the second with an arylstibonic acid derivative, the APE1 inhibitor 13755,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> soaked in Tris pH 8.0 (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Methods">Experimental Methods</a>). Neither 4-(hydroxymethyl)phenylacetic acid nor APE1 inhibitor 13755 were evident in the electron density maps for these crystals. However, these structures, determined at 1.44 and 1.67 Å, respectively, provided additional insights on DMSO binding in the presence of other bound ligands in the active site, as Mg<sup>2+</sup> and Tris in these structures are bound to the same site. Mg<sup>2+</sup> exhibits an octahedral coordination geometry with oxygen atoms from D70 and E96, each 2.1 Å from the Mg<sup>2+</sup> atom, serving as ligands as well as four water molecules (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). DMSO is bound deeper within the abasic site pocket and is not hydrogen bonded to N174 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E); its sulfoxide oxygen now faces the backbone carbonyl of A230. The NH<sub>3</sub><sup>+</sup> group of Tris is hydrogen bonded to the O of D70 (2.7 Å) and to O of E96 (3.2 Å) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F). DMSO binds slightly differently in the presence of bound Mg<sup>2+</sup> than in the presence of Tris. In the DMSO/Tris structure and DMSO only structures, DMSO is bound identically in the abasic site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G). The second DMSO site is also identically occupied in these two structures, whereas in the DMSO/Mg<sup>2+</sup> structure, the second DMSO site is not occupied. This finding suggests that the presence of Mg<sup>2+</sup> modulates positioning and availability of small molecule binding sites in APE1.</div><div class="NLM_p last">Additional solvents that bind to the abasic sugar binding site within the repair active site of APE1 include glycerol (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>) and ethylene glycol. The basis for solvent binding within the repair active site of APE1 appears to rely on an interaction with the relatively small pocket designed to bind to the flipped out abasic sugar of substrate DNA. Binding of glycerol to the abasic site involves a hydrogen bonding interaction between a hydroxyl group in glycerol and the side chain carbonyl oxygen of N212 (2.79 Å) and a water-mediated hydrogen-bonding interaction between a second glycerol hydroxyl and the amide side chain group of N174 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H). The more polar glycerol binds a little deeper in the pocket than DMSO alone (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>I). The second glycerol binding site is located between two molecules in the lattice and forms a hydrogen bond between one of its hydroxyl groups and the main chain carbonyl oxygen of E242 (3.08 Å) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf" class="ext-link">Figure S1</a>). Ethylene glycol was used as the cryoprotectant in all structures except for that of the glycerol structure. Hydrogen bonding occurs between the carboxylate oxygen atom of E216 and a hydroxyl of ethylene glycol bound to APE1 in all of the structures in which ethylene glycol was used as a cryoprotectant (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf" class="ext-link">Figure S1</a>). Three additional ethylene glycol binding sites were identified in the DMSO structure and two additional sites in the previously reported Apo structure,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> including one bound to the abasic site (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf" class="ext-link">Figure S1</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Computational Solvent Mapping</h3><div class="NLM_p">The crystallographic solvent mapping results provided a relatively small number of preferred binding sites within APE1. Therefore, it was of interest to use computational methods to further investigate possible binding sites. Results of computational solvent mapping for the magnesium-bound APE1 structure are shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. The primary binding region (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) is composed of 7 consensus sites with the total population of 50 probe clusters. The most populated site, site 1 (17 probe clusters) is next to the magnesium ion position (shown in green). Site 2 (15 clusters) is the abasic DNA sugar binding site. We have experimentally validated binding of Tris near the Mg<sup>2+</sup> site in the Tris/Mg<sup>2+</sup> structure and binding to the abasic site by solvent probes such as DMSO, ethylene glycol, and glycerol as determined by multiple solvent crystal structures. The computational mapping also revealed a binding region consisting of a small hydrophobic cavity on the opposite side of the protein, formed by Leu 62, Ile 91, Phe 162, Phe 165, and Leu 318 residues. Two clusters with a total of 21 probes are located in this region (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The crystallographic experiments showed an ethylene glycol bound at this secondary region. Collectively, our crystallographic and computational solvent mapping provided the locations of preferred binding sites for small organic molecules in the active site of APE1 in the presence and absence of the APE1 cofactor, Mg<sup>2+</sup>, laying the foundation for a novel structure-based approach for the identification of APE1 inhibitors.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Computational solvent mapping with FTMap ((<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>). (A) Computational docking of small organic molecules reveals clustering of 50 probes (thin lines, C, cyan, O, red, N blue) into seven consensus sites in or near the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>) shown as a molecular surface rendering (C, light gray, O, red, N, blue, and S, yellow). DMSO (stick model C, green, O, red, S, yellow) from the crystal structure is shown bound in consensus site 2. Probes at consensus site 1, the most populated site, are in close proximity to the Mg<sup>2+</sup> ion, shown as a surface rendering in green. (B) A secondary site occupied by ethylene glycol in several crystal structures of APE1 is shown with a cluster of 21 probes shown in thin lines. Ethylene glycols bound in this site and in the vicinity are shown as stick renderings (C, magenta, O, red). This is a small pocket on the surface of the molecule. Thus, the major consensus sites for computational docking of probes in the active of APE1 and in another pocket have been validated in our crystallographic analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Identification of Macrocycles That Inhibit APE1 Endonuclease Activity</h3><div class="NLM_p">Focusing on sites 1 and 2 as key binding regions for potential APE1 inhibitors, we deduced that sites 3–7 show directions of possible ligand extensions. The distance between clusters 5 and 6 spanning the binding pocket is approximately 18 Å, suggesting that an optimal ligand for this binding region may be large in size. This observation of a large pocket size led us to consider the in silico evaluation of a small library of 15- and 16-membered macrocycles (105 compounds) from the BU-CMD screening collection as potential inhibitors (<b>MC001</b>–<b>MC105</b>; see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> for general synthetic route). These macrocycles possess molecular weights ranging from 350 to 800 kDa and maximal projection radii ranging from 6 to 10 Å. The library was originally synthesized according to the protocol outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Specifically, enantioenriched homoallylic amines <b>5</b> were first prepared via asymmetric crotylation of imine carbamate substrates with enantioenriched crotylsilane <b>2</b>. These acylimines were generated in situ by condensation of allyl carbamate <b>1</b> with a variety of aldehydes <b>3a</b>–<b>e</b> in a modified variant of our previously reported procedure.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> While <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> depicts the stereochemical outcome achieved from the use of (<i>S</i>)-<b>2</b>, the screening library was also populated with macrocycles derived from the enantiomeric crotylsilane (<i>R</i>)-<b>2</b>, leading to macrocycles, which are epimeric at the crotylation-derived stereocenters.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Subsequent palladium-mediated allyl carbamate deprotection of <b>4</b> afforded homallylic amines <b>5</b>, which were then further diversified via EDCI-mediated coupling to various commercial, Boc-protected amino acids <b>6a</b>–<b>d</b> to afford <b>7</b>. Following deprotection of <b>7</b>, macrocycles <b>MC001</b>–<b>MC105</b> were obtained via a tin-mediated tandem, one-pot aminolysis/macrolactonization of amino esters <b>8</b> with various γ-butyro or δ-valero lactone derivatives <b>9a</b>–<b>s</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0010.jpeg" id="GRAPHIC-d7e1126-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Design and Synthetic Route for Macrocyclic Lactone Library<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BF<sub>3</sub>–THF, −78 °C to −20 °C; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,3-dimethylbarbituric acid, THF; (c) EDCI, <i>i</i>-Pr<sub>2</sub>EtNH, DMF or CH<sub>2</sub>Cl<sub>2</sub>; (d) HCl, dioxane; (e) Otera’s catalyst, PhCF<sub>3</sub>. Depicted stereochemistry corresponds to use of (<i>S</i>)-<b>2</b> in step a. Intermediates <b>4</b>, <b>5</b>, <b>7</b>, <b>9</b> and macrocycles <b>MC</b> derived from crotylsilane (<i>R</i>)-<b>2</b> are epimeric at the methyl- and R-substituted positions.</p></p></figure><div class="NLM_p">We computationally docked this macrocyclic library, guided by hot spots identified by our crystallographic solvent mapping and FTMap analysis as described in <a class="ref internalNav" href="#sec4" aria-label="Experimental Methods">Experimental Methods</a>. Docking yielded a series of high quality poses, suggesting that select BU-CMD macrocycles generally match the topological features of the APE1 binding site. The docked molecules were scored on the basis of the average ensemble energy for their lower energy poses as described in <a class="ref internalNav" href="#sec4" aria-label="Experimental Methods">Experimental Methods</a>. The top 25 docked molecules and their average ensemble energies are shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Notably, macrocycles derived from (<i>S</i>)-<b>2</b> and tryptophan (<b>6c</b>) predominated among the top scored macrocycles.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Average Autodock Energy for the Top 25 Macrocycles</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" valign="middle"> </th><th class="colsep0 rowsep0" align="center" valign="middle"> </th><th class="colsep0 rowsep0" align="center">aldehyde</th><th class="colsep0 rowsep0" align="center">amino acid</th><th class="colsep0 rowsep0" align="center">lactone</th><th class="colsep0 rowsep0" align="center" valign="middle"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">code</th><th class="colsep0 rowsep0" align="center">crotylsilane</th><th class="colsep0 rowsep0" align="center"><b>3</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>9</b></th><th class="colsep0 rowsep0" align="center">energy (kcal/mol)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC042</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9c</b></td><td class="colsep0 rowsep0" align="left">–9.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC093</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–9.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC048</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9p</b></td><td class="colsep0 rowsep0" align="left">–9.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC044</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9f</b></td><td class="colsep0 rowsep0" align="left">–9.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC042</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9c</b></td><td class="colsep0 rowsep0" align="left">–9.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC047</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9k</b></td><td class="colsep0 rowsep0" align="left">–8.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC043</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–8.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC046</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9j</b></td><td class="colsep0 rowsep0" align="left">–8.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC030</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9l</b></td><td class="colsep0 rowsep0" align="left">–8.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC022</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9h</b></td><td class="colsep0 rowsep0" align="left">–8.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC047</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9k</b></td><td class="colsep0 rowsep0" align="left">–8.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC022</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(S)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9h</b></td><td class="colsep0 rowsep0" align="left">–8.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC045</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9i</b></td><td class="colsep0 rowsep0" align="left">–8.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC093</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–8.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC032</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9n</b></td><td class="colsep0 rowsep0" align="left">–8.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC043</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–8.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC030</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9l</b></td><td class="colsep0 rowsep0" align="left">–8.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC019</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9e</b></td><td class="colsep0 rowsep0" align="left">–8.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC061</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9m</b></td><td class="colsep0 rowsep0" align="left">–8.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC096</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left"><b>9m</b></td><td class="colsep0 rowsep0" align="left">–8.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC033</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9o</b></td><td class="colsep0 rowsep0" align="left">–8.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC036</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9s</b></td><td class="colsep0 rowsep0" align="left">–8.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC019</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9e</b></td><td class="colsep0 rowsep0" align="left">–8.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC058</b> (α-<i>R</i>)</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–8.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MC058</b> (α-<i>S</i>)</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>2</b></td><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>9d</b></td><td class="colsep0 rowsep0" align="left">–8.44</td></tr></tbody></table></div></div><div class="NLM_p">On the basis of favorable preliminary docking results, macrocycles available at the time of screening were then assayed as potential inhibitors of APE1 activity. Of the 66 compounds assayed, four (see below for codes) inhibited APE1 endonuclease activity by at least 45% and were further characterized for concentration dependent inhibition. By use of this approach, 6.1% of the compounds screened were found to inhibit APE1, whereas in our previous HTS efforts, the success rate was approximately 0.1%.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> All four of the compounds, <b>MC043</b>, <b>MC047</b>, <b>MC042</b>, and <b>MC019</b> exhibited concentration-dependent inhibition with IC<sub>50</sub> values ranging from 1.2 μM to 5.4 μM for the diastereomeric mixtures (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Initial hits from high-throughput screen of the macrocycle library. The chemical structures of the four hits obtained from screening of 66 macrocycles in an in vitro endonuclease activity are shown along with the IC<sub>50</sub> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From the primary screen, nascent SAR emerged suggesting that APE1 endonuclease inhibition activity was confined to library members originating from crotylation of <i>N</i>-tosylpyrrole 2-carboxaldehyde (<b>3b</b>) or benzaldehyde (<b>3a</b>) with crotylsilane (<i>S</i>)-<b>2</b>, and Boc-<span class="smallcaps smallerCapital">l</span>-tryptophan (<b>6c</b>) as the amino acid building block (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). These results were highly consistent with the outcome of our docking studies, in which these modalities appeared frequently among the top 25 structures. In addition, both diastereomers of all active hits are represented in the top 25 structures (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">With active macrocycles in hand, we next revisited our docking results to identify consensus poses for the actives that might inform further docking studies and analog designs. In general, two distinct docking modes with comparable energies were observed for the active macrocycles. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> depicts low-energy poses representing the two alternative binding modes of active macrocycle <b>MC047</b> as an illustrative example. <b>MC047</b>, which was screened as a mixture of diastereomers, was docked separately as the α-<i>S</i> (green) and α-<i>R</i> (cyan) diastereomers, which refers to stereochemistry at the position α to the macrolactone carbonyl. The docking poses of mode 1, shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, occupy 6 out of 7 hot spots in the main binding site. The difference in binding for the two diastereomers is mainly due to a slight change in the positioning of the tryptophan indole as depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A. Specifically, this heterocycle is parallel to the plane of the Met270 hydrophobic atoms in the pose for the α-<i>S</i> isomer, while the pose for the α-<i>R</i> isomer projects the indole N–H toward this plane. It is interesting to observe that bound DNA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DE9">1DE9</a>) shares a bicyclic heterocycle positioned below Met270, similar to the mode 1 docking pose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B hot spot region 4). The main feature of binding mode 2, in contrast, is an interaction of the sulfonamide group with the magnesium ion, as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C. This pose, which was only obtained for the <i>N</i>-tosylpyrrole-containing macrocycles, occupies 5 out of 7 hot spots in the main binding site. For <b>MC047</b>, the trifluoromethyl group of the α-<i>S</i> isomer extends toward an additional hot spot (number 5 in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), while in the α-<i>R</i> pose this group interacts with the terminal carbon of Met270. In comparing this mode 2 pose to the binding mode of DNA, the lactone oxygens are situated proximal to where the DNA phosphate group binds the Mg<sup>2+</sup> ion (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D, hot spot region 2).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking of macrocycle hit <b>MC047</b>. (A) The two diastereomers of <b>MC047</b> are shown as stick models with α-<i>S</i> (C, green) and α-<i>R</i> (C, cyan) for mode 1 of the possible docking poses. APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>) is shown as a molecular surface rendering. In docking mode 1, <b>MC047</b> diastereomers occupy 6 out of 7 hot spots in the main binding site. The difference in binding for the two diastereomers is mainly due to a slight change in positioning of the tryptophan indole. The orientation is similar to that shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> as can be seen by the relative positions of M270 and R177. (B) In binding mode 1, the position of the bicyclic heterocycle of <b>MC047</b> (C, green) coincides with that of a guanine nucleotide from DNA (thin lines, C, cyan) bound to APE1 (1DE9) superimposed on the apo APE1 structure. APE1 is shown as a molecular surface rendering. (C) The two diastereomers of <b>MC04</b>7 are shown in mode 2 as similar renderings to those in (A), α-<i>S</i> (C, green), and α-<i>R</i> (C, cyan). In this binding mode, the sulfonamide group is in close proximity to the Mg<sup>2+</sup> ion (green on the molecular surface rendering). The mode 2 pose was only obtained for the <i>N</i>-tosylpyrrole-containing macrocycles and occupies 5 out of 7 hot spots in the main binding site. For <b>MC047</b>, the trifluoromethyl group of the α-<i>S</i> isomer extends toward an additional hot spot (number 5 in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), while in the α-<i>R</i> pose this group interacts with the terminal carbon of Met270. (D) <b>MC047</b> α-<i>S</i> (C, green) is shown with superimposed DNA (1DE9). In this mode, the lactone oxygens are situated proximal to where the DNA phosphate group binds the Mg<sup>2+</sup> ion near hot spot region 2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Synthesis and in Vitro Assessment of Follow-On Analogs</h3><div class="NLM_p">As part of our initial validation of the chemotype, a small set of simplified analogs <b>10</b>–<b>13</b> were synthesized (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). APE1 endonuclease activities for these lactones are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Among this set, the <i>N</i>-tosylpyrrole-substituted analogs <b>12</b> and <b>13</b> displayed superior APE1 inhibition potencies for the macrocycles in comparison to their phenyl-substituted counterparts <b>10</b> and <b>11</b>. Interestingly, the most potent macrolactone of this set, compound <b>13</b>, also falls in the top 25 docked macrocycles from the virtual library (<b>MC045</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) These observations led us to focus on binding mode 2 (cf. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), which exhibits a key interaction between the sulfonamide and Mg<sup>2+</sup> ion, for subsequent docking experiments. Following identification of the initial screening hits and simplified analogs, our subsequent optimization efforts focused on exploring further modifications of the linker group connecting the <span class="smallcaps smallerCapital">l</span>-tryptophan/homoallylic amine core, with the parallel goals of improving both potency and drug-like properties of the chemotype. A liability of lactone- and ester-containing drugs (as exemplified in macrocyclic lactone drugs with bacterial resistance mechanisms toward macrolide antibiotics) is the potential for drug inactivation due to esterase-mediated hydrolysis.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> To address this anticipated liability, we next targeted macrolactam analogs (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Substrates <b>7a</b> and <b>7b</b> were deprotected and coupled to various achiral unsubstituted (<b>14a</b>–<b>c</b>) and chiral, nonracemic (<b>14d</b>–<b>k</b>) Boc-protected β-, γ-, and δ-amino acids (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Following coupling, tandem Boc and methyl ester deprotections were followed by HATU-mediated macrolactamization to afford macrocyclic lactams in modest yields for the three-step sequence in most cases (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0011.jpeg" id="GRAPHIC-d7e2433-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Simplified Follow-On Macrolactone Analogs</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. APE1 Endonuclease Activity of Macrolactones <b>10</b>–<b>13</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">APE1 IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">PaO2c EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">63.5</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">10.6</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">9.3</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div></div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0012.jpeg" id="GRAPHIC-d7e2539-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthetic Route toward Second-Generation Macrocyclic Lactams<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCl/dioxane; (b) HATU, <i>i</i>-Pr<sub>2</sub>EtNH, CH<sub>2</sub>Cl<sub>2</sub>; (c) 2.5 M NaOH, THF; (d) HCl, dioxane; (e) HATU, <i>i</i>-Pr<sub>2</sub>EtNH, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Synthetic Yields and APE1 Endonuclease Activity of 14–16 Membered Macrolactams</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0013.gif" alt="" id="GRAPHIC-d7e2577-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Trp-coupled precursor</th><th class="colsep0 rowsep0" align="center">cyclization precursor (yield from <b>7</b> (%))</th><th class="colsep0 rowsep0" align="center">cyclization yield (3 steps) (%)</th><th class="colsep0 rowsep0" align="center">APE1 IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">PaO2c EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16a</b> (81)</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">29.2</td><td class="colsep0 rowsep0" align="left">35.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16b</b> (74)</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16c</b> (93)</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">33.2</td><td class="colsep0 rowsep0" align="left">58.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16d</b> (81)</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16e</b> (82)</td><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16f</b> (77)</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">52.6</td><td class="colsep0 rowsep0" align="left">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16g</b> (70)</td><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16h</b> (81)</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16i</b> (79)</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16j</b> (84)</td><td class="colsep0 rowsep0" align="left">43</td><td class="colsep0 rowsep0" align="left">12.1</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left"><b>16k</b> (80)</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left"><b>15g</b> (75)</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">26.5</td><td class="colsep0 rowsep0" align="left">58.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left"><b>15i</b> (77)</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">55.1</td><td class="colsep0 rowsep0" align="left">36.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left"><b>15j</b> (78)</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">22</td></tr></tbody></table></div></div><div class="NLM_p">The structures and APE1 endonuclease activities of macrocycles <b>17</b>–<b>30</b> are shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Initially, the direct O- to N-replacement was probed using compounds <b>17</b> and <b>18</b>, the direct lactam analogs of lactones <b>12</b> and <b>13</b>, respectively, as well as the 14-membered macrolactam <b>19</b> derived from β-alanine. A drop in activity was observed for all unsubstituted compounds <b>17</b>, <b>18</b>, and <b>19</b>. For follow-on substituted lactam analogs, a cohort of chiral amino acids <b>14d</b>–<b>k</b> were selected based on additional docking experiments. Specifically, a large virtual library comprising lactams derived from commercially available amino acid building blocks was docked and evaluated, and the macrocyclic lactams that produced the lowest energy poses in binding modes 1 and 2 were selected for synthesis and evaluation. In addition to the tosylpyrrole-substituted macrolactams <b>17</b>–<b>27</b>, phenyl-substituted variants of select analogs (<b>28</b>–<b>30</b>) were also synthesized for comparison.</div><div class="NLM_p">Similar to the macrolactone series, there appear to be key substitution patterns on the tethering β-amino acid of the 14-membered macrolactam scaffolds that govern endonuclease activity. For example, substitution α to the lactam carbonyl appears to impart enhanced potency in most cases, while the comparison of lactams <b>20</b> and <b>21</b> suggests that this position may not be sensitive to stereochemical inversion. This result is generally consistent with our docking, wherein both epimers generally produced favorable docking poses. Much like the lactone series, the <i>N</i>-tosylpyrrole-substituted lactams <b>23</b>, <b>25</b>, and <b>26</b> also all exhibited superior potencies to their phenyl-substituted counterparts <b>28</b>, <b>29</b>, and <b>30</b>, respectively. Interestingly, the <i>meta</i>-(trifluoromethyl)phenyl macrolactam <b>22</b> exhibited a significant reduction in activity in comparison to its <i>para</i>-substituted phenyl and benzyl analogs <b>23</b>–<b>27</b>.</div><div class="NLM_p">In summary, the observed SAR is generally consistent with the consensus binding mode 2 observed in our docking studies. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> depicts an overlay of the consistent low energy mode 2 binding poses of molecules <b>MC047</b>, <b>11</b>, <b>26</b>, and <b>27</b>. Molecules <b>MC047</b>, <b>26</b>, and <b>27</b> all demonstrate projection of halogen moieties toward the hot spot region 5, as well as the interaction of sulfonamide groups with the magnesium ion. Attempts to validate the computationally obtained poses through soaking into preformed crystals or cocrystallization experiments employing the most potent APE1 inhibitors have thus far yielded only solvent bound structures as we obtained for other classes of APE1 inhibitors. Notably, the first glycerol-APE1 structure shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> was serendipitously obtained through soaking efforts with macrocycle <b>20</b>.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay of “mode 2” docking poses obtained for stick renderings of <b>MC047</b> (C, light gray), <b>11</b> (C, teal), <b>25</b> (C, dark green), and <b>26</b> (C, pink) in the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>). Compounds <b>MC047</b>, <b>25</b>, and <b>26</b> all display an interaction between the sulfonamide groups and the magnesium ion (light green sphere), as well as a projection of their halogenated aryl rings toward hot spot region 5, left-hand side.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Effect of Macrocycles on Viability and Repair of DNA Damage in Cancer Cells</h3><div class="NLM_p">With a series of validated biochemical APE1 inhibitors in hand, we next sought to understand the effects of these inhibitors in a cellular context. Knock-down of APE1 arrests the growth of ovarian cancer cells in a xenograft mouse model;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> thus, compounds that specifically inhibit APE1 might be expected to have low cytotoxicity in cancer cell culture models. In contrast, compounds that exhibit high cytotoxicity may have off-target effects. Most of the macrocycles tested exhibit low cytotoxicity with the exception of <b>17</b>, <b>19</b>, <b>22</b>, <b>28</b>, <b>29</b>, and <b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). However, low cytotoxicity cannot be distinguished from low cell permeability in cell viability assays.</div><div class="NLM_p">To determine whether the compounds inhibit APE1 endonuclease activity in the cell, DNA damage in cells treated with selected macrocycles was assessed by comet assays<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> done in the presence and absence of the DNA-damaging agent, methyl methanesulfonate, MMS, which alkylates DNA. The comet assay was selected as a cell-based assay that reliably reports single-strand and double-strand breaks in DNA. In this assay, following exposure to MMS, alkylated bases are removed by DNA glycosylases creating abasic sites. APE1 then cleaves the DNA 5′ of the abasic sites leading to single strand breaks (or potentially double-strand breaks if single-strand breaks are produced proximally in both strands). The breaks produce a long tail in the alkaline comet assay analyzed 1 h after adding MMS; this time point is short enough to ensure that the first two steps of base excision repair, removal of the damaged base by a DNA glycosylase and processing by APE1, have occurred. It is, however, insufficient time for the cell to have completed repair of the DNA through the base excision repair pathway. If the macrocycle inhibits APE1 preventing cleavage of the DNA backbone, then the length of the DNA tail in the comet assay is expected to be shorter than that observed for treatment with MMS alone. An inherent assumption for the comet assay is that only compounds that exhibit some level of cell permeability have the potential to affect the tail length of the DNA.</div><div class="NLM_p">Macrocycles chosen for the alkaline comet assay include noncytotoxic APE1 inhibitors of varying potency (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>): <b>13</b> (IC<sub>50</sub> = 9.3 μM, EC<sub>50</sub> > 100 μM), <b>21</b> (IC<sub>50</sub> = 2.7 μM, EC<sub>50</sub> > 100 μM), <b>23</b> (IC<sub>50</sub> = 1.3 μM, EC<sub>50</sub> > 100 μM), and <b>24</b> (IC<sub>50</sub> = 2.5 μM, EC<sub>50</sub> > 100 μM), as well as low-potency APE1 inhibitors with moderate cytotoxicities <b>19</b> (IC<sub>50</sub> = 16.9 μM, EC<sub>50</sub> = 58.4 μM) and <b>28</b> (IC<sub>50</sub> = 26.5 μM, EC<sub>50</sub> = 58.5 μM). Malignant peripheral nerve sheath tumor ST8814 cells were treated with the macrocycles in the presence and absence of MMS at either 100 μM or, in the case of cytotoxic compounds, the EC<sub>30</sub> value and then assessed for comet tail length (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The concentrations were selected to deliver the highest possible concentration of macrocycle to the cells without causing significant cell death. Alkaline comet assay results for <b>19</b> and <b>23</b> are similar to those obtained for MMS alone suggesting that these macrocycles do not effectively inhibit APE1 endonuclease activity in the cell. Macrocycle <b>28</b> had a very modest effect on tail length consistent with its modest IC<sub>50</sub> value for APE1 inhibition. Macrocycles <b>21</b> and <b>24</b> appear to reduce comet tail length, while macrocycle <b>13</b> had the largest impact on tail length. These results suggest that despite having EC<sub>50</sub> values for cell killing of greater than 100 μM, these compounds appear to permeate cells.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Macrocycle treatment alone and in combination with DNA damaging agent methyl methanesulfonate (MMS) in malignant peripheral nerve sheath tumor cell line ST8814 in alkaline CometAssay. APE1 repair inhibitor candidates (macrocycles) and APE1 repair inhibitor control, APE1 repair inhibitor III (ARiIII; Calbiochem), were tested in ST8814 at a single dose (100 μM or EC<sub>30</sub>) and in combination with a single dose of MMS (1.0 mM) for a 1.0 h treatment. Cells were seeded in 6-well tissue culture plates at 130 000 cells/well in DMEM + 10% FBS and grown overnight at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with Opti-MEM (Gibco) medium containing macrocycle alone or spiked with 1.0 mM MMS. Cells were then incubated for 1.0 h at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with PBS, and cells were treated with 0.25% trypsin (HyClone), collected, and washed with PBS. Cells were then counted by hemacytometer. DNA damage was evaluated by CometAssay (Trevigen) performed under alkaline conditions. Cells were resuspended in PBS at 1 × 10<sup>5</sup>/mL and then added to premelted and cooled (37 °C) agarose at a 1:10 ratio. Cells were gently mixed, and then 50 μL of the agarose cell mix was transferred to prewarmed comet slides (37 °C, CometSlide). After solidifying at 4 °C, slides were placed in lysis solution for 60 min and then placed in freshly prepared alkaline unwinding solution (NaOH, pH 13) for 20 min. Slides were then subjected to electrophoresis under alkaline conditions (NaOH, pH 13) at 1 V/cm (300 mA) for 30 min. Slides were washed in H<sub>2</sub>O, placed in 70% ethanol, and allowed to dry at 37 °C for 30 min. Slides were stained with 100 μL of 1:10 000 diluted SYBR Gold (Invitrogen) in TE pH 7.5 and incubated for 30 min. Slides were then rinsed in H<sub>2</sub>O briefly, and comets were captured by fluorescent microscope (Leica DMIL) and quantified by CometScore Pro (TriTec Corp). DNA damage measured by percent Tail DNA for controls and selected macrocycles is shown in (A) in two independent experiments. Representative images of comet slide DNA damage are shown in (B) for controls and selected macrocycles. There was an average of 20 comet readings/compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08057" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have employed a novel computational and structure-based approach to predict and identify new macrocyclic inhibitors of the DNA repair protein APE1. From screening of a library of 66 macrocyclic scaffolds, four compounds were identified with micromolar APE1 inhibition values. The chemotype was further explored, and preliminary structure–activity relationships were determined. Our preliminary characterization of the effects of the compounds on cellular activities indicates that at least three of the macrocycles (<b>13</b>, <b>21</b>, and <b>24</b>) are effective in a cellular context (with <b>13</b> being the most effective) as assessed by comet analysis following treatment with MMS while retaining reasonable potency for inhibition of endonuclease activity in vitro. These results are promising and support further development of these chemotypes as APE1 inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48621" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48621" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Preparation of APE1 Protein</h3><div class="NLM_p last">For crystallization, a single amino-acid substitution (C138A) was introduced within the N-terminally truncated protein lacking the first 40 amino acids (Δ40APE1). DNA encoding residues 40–318 of APE1 was inserted within the PET28A vector by using the NheI and XhoI restriction sites. Site-directed mutagenesis was then used to introduce a C138A mutation and confirmed by DNA sequencing. This protein was expressed as an N-terminal hexa-His tagged protein and purified as previously described for the C65A and wild-type Δ40APE1 proteins.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In brief, the cells were lysed by using a French press, and the crude extract was subjected to purification by Ni-NTA and SP-Sepharose ion-exchange chromatography. The affinity tag was then removed by treatment with thrombin, and the protein was further purified by SP-Sepharose ion-exchange chromatography.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Crystallization and Data Collection</h3><div class="NLM_p">Crystals were obtained by mixing equal parts (1 μL each) of microseeds with a precipitant solution containing 100 mM MES, pH 6.0, 200 mM NaCl and 18–21% PEG4000 and Δ40 hAPE1 C138A (10 mg/mL) buffered in 10 mM HEPES, pH 7.5. Self-nucleated crystals of APE1 were obtained under similar conditions and used to produce microseeds.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Solvent-Bound Complexes</h4><div class="NLM_p last">Solvents including dimethylsulfoxide (DMSO), ethylene glycol (EG), or glycerol (GLC) were trapped in bound complexes with APE1. The DMSO complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>) was obtained by soaking preformed crystals in 5% DMSO overnight and then cryocooled in 100 mM MES, pH 6.0, 200 mM NaCl, 22% PEG4000, 20–22% ethylene glycol, and 5% DMSO. The GLC complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>) was obtained by soaking preformed crystals overnight in 1 mM macrocycle <b>20</b> and then cryocooling in the solvent above with 20% glycerol in place of ethylene glycol. All of the crystals cryocooled in ethylene glycol had multiple binding sites for this cryosolvent.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Other Complexes</h4><div class="NLM_p last">Two additional complexes were obtained from soaks. One including Tris and DMSO (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>) was obtained from a soak that also included a reported arylstibonic acid inhibitor, compound 13755, from the NCI Diversity Set. In this case, the crystals were grown as described above and then soaked in 100 mM Tris, pH 7.5, 200 mM NaCl, 1 mM 13755 (dissolved in DMSO resulting in a final concentration of 4% DMSO), and 20% PEG 3350. Crystals were cryocooled in 100 mM Tris-Cl, pH 7.5, 200 mM NaCl, 20% PEG 3350, 1 mM 13755, 4% DMSO, and 20% ethylene glycol. The second complex included Mg<sup>2+</sup> and DMSO (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>), obtained from a soak including 20 mM 4-hydroxymethylphenylacetic acid (Sigma-Aldrich) along with 1 mM MgCl<sub>2</sub>. Crystals were cryocooled in 100 mM MES, pH 6.0, 200 mM NaCl, 22% PEG4000, 20–22% ethylene glycol, 20 mM 4-(hydroxymethyl)phenylacetic acid, and 4% DMSO. In each of these complexes, the inhibitor was not evident in the difference electron density map but DMSO was bound to the abasic sugar site with either Mg<sup>2+</sup> or Tris in the metal binding site.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Data Collection and Processing</h4><div class="NLM_p last">Data for all complexes except the GLC with C138A APE1 were collected at the GM/CA 23ID-D and SBC 19-BM beamlines at the Advanced Photon Source, Argonne National Laboratory, and processed with HKL2000.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The GLC data set was collected on our Bruker X8 Prospector (Bruker Corporation, Billerica, MA) with Cu Kα radiation (1.5418 Å) at 100 K by using an Oxford Cryosystem. Data were integrated using SAINT<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and scaled with SADABS.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> XPREP<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> was used to determine the space group and analyze the data. All of the crystals belong to <i>P</i>2<sub>1</sub>2<sub>1</sub>2 space group with cell dimensions as shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Apo, Mg<sup>2+</sup>, and Mn<sup>2+</sup> bound structures have been previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Structure Determination and Refinement</h4><div class="NLM_p last">Phasing for the structures was obtained by molecular replacement (MOLREP) using the coordinates of C138A Δ40 hAPE1 as the search model. Several rounds of initial refinement were carried out using REFMAC5, accompanied by iterative model building using COOT. Following placement of well-ordered water molecules, excluding those in the repair active site of the enzyme, the bound solvent or metal was identified as a strong <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> difference peak. Final rounds of refinement for all of the structures were carried out by using PHENIX with isotropic temperature factor refinement. Statistics for refinements are compiled in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Computational Methods</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Computational Mapping and Binding Site Identification</h4><div class="NLM_p last">Computational solvent mapping was performed using the FTMap algorithm through its online server (<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>) using magnesium bound (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>) and apo (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHD">4QHD</a>) crystal structures of APE1, solved recently.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In brief, FTMAP simulates the interaction of the input protein structures with a library of 16 small organic probe molecules, with varying aromaticity, hydrophobicity, and hydrogen bonding properties. For each probe in the library, the algorithm identifies six clusters, with lowest mean energy. Those clusters from different probes are further clustered into consensus sites (CS). Consensus sites are ranked by the number of its probe clusters. It was observed experimentally that probes bind to the major binding sites of the protein. A detailed description of the method is provided in refs <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a>. It was observed experimentally that probes bind to the major binding sites of the protein. We have also reported the FTSite approach, which uses FTMap results for binding site identification.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The method calculates a percentage contact score for each consensus cluster. The method calculates the percentage contact score for each consensus cluster. The cluster having the highest percentage contact score is considered core of the main site. The binding site is identified as all the consensus clusters within 12 Å from the core consensus site.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Docking and Scoring of the Compounds</h4><div class="NLM_p last">Molecular docking of 105 macrocyclic compounds selected from the BU-CMD small molecule screening collection (<a href="http://cmd.bu.edu/" class="extLink">http://cmd.bu.edu/</a>) was performed on both the apo and Mg<sup>2+</sup> bound structures using the Autodock program.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Initial conformations of macrocycles were generated with the Marvin program of ChemAxon.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The Marvin Conformer plugin was used for generating 3D conformers of the macrocycles, and the Charge plugin was used for calculating partial charges prior to docking. The protein was held rigid during docking experiments. The docking box was selected to cover all mapping hot spots present in the site around largest consensus cluster. Initial conformations of macrocycles were generated with the Marvin program of ChemAxon. From the set of 100 conformations generated for each isomer, 10 lowest energy conformers were chosen. Preference was given to <i>trans</i> conformers of the peptide bond in the cycle. <i>Cis</i> conformers were only chosen if a set of 100 has less than 10 trans conformers. Conformer generation led to 1440 Autodock runs. Autodock was set to generate up to 10 clusters of docking poses. We chose the B site from two alternative positions of magnesium present in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a> PDB structures. Ranking of the compounds was based on the average ensemble energy of the 50 lowest Autodock energy poses.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Apurinic/Apyrimidinic Endonuclease 1 Assay</h3><div class="NLM_p last">Macrocycles were tested for APE1 inhibition using a well characterized APE1 DNA repair activity assay performed in our laboratory and others.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The APE1 repair activity assay was performed in a 96-well plate assay using purified full-length APE1 enzyme and an AP site mimic consisting of two annealed oligonucleotides (5′-6-FAM-GCCCCC*GGGGACGTACGATATCCCGCTCC-3′ and 3′-Q-CGGGGGCCCCCTGCATGCTATAGGGCGAGG-5′) custom synthesized by Eurogentec Ltd. (Belgium). The oligonucleotides contained a quencher on one strand and a fluorescent 6-FAM label with an AP site mimic, tetrahydrofuran (*), on the complementary strand. The AP site mimic is a direct target of APE1’s endonuclease function. Cleavage of the oligo at this site results in the release of the 6-FAM portion of the oligo from the complementary strand with the quencher. The amount of fluorescence due to this cleavage is directly proportional to APE1’s endonuclease activity. Each macrocycle was tested in triplicate by a 1:2 serially diluted, 10-point dose scheme with 100 μM maximum concentrations in 200 μL final volume. A master mix was used providing a final amount of 50 nM annealed oligo, 50 mM Tris, 1 mM MgCl<sub>2</sub>, and 50 mM NaCl, pH 7.5. Due to the rapid enzymatic activity, 0.25–1.25 nM full-length APE1 was added to the wells at once and then immediately assayed. The fluorescence was read kinetically at five, one-minute intervals using a Bio-Tek Synergy H4 (Herman B Wells Center, Indiana University School of Medicine). The rate of the reaction was used to determine the change in APE1 repair activity as compared to the vehicle control (DMSO).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cell Cytotoxicity Assays</h3><div class="NLM_p last">Low passage patient-derived pancreatic cells (Pa02c)<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> maintained in 10% FBS DMEM growth medium were plated at 2000 cells per well in poly-<span class="smallcaps smallerCapital">d</span>-lysine treated 96-well clear bottom black plates and grown overnight in 5% CO<sub>2</sub> at 37 °C. Cells were then treated with 100 μL of macrocycle in doses made up in 5% FBS DMEM medium at 100 μM concentration and serially diluted 1:2 in a 5-point dose scheme. Wells dedicated to drug background and vehicle control (DMSO) were also included. Cells were grown for 48 h in 5% CO<sub>2</sub> at 37 °C. Fresh 5% FBS DMEM medium was exchanged, and a fluorescent metabolic indicator, Alamar Blue was added to each well at 10% final. After a 4 h incubation in 5% CO<sub>2</sub> at 37 °C, plates were read on a Synergy H4 (Bio-Tek). For each drug dose, background was subtracted and then further normalized to media alone. Data presented are percent normalized to media with mean standard error and include vehicle (DMSO) response.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Alkaline Comet Assays</h3><div class="NLM_p last">APE1 repair inhibitor candidates and APE1 repair inhibitor control, APE1 repair inhibitor III (ARiIII; Calbiochem), were tested in ST8814 at a single dose (100 μM or EC<sub>30</sub>) and in combination with a single dose of MMS (1.0 mM) for a 1.0 h treatment. Cells were seeded in 6-well tissue culture plates at 130 000 cells/well in DMEM + 10% FBS and grown overnight at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with Opti-MEM (Gibco) medium containing macrocycle alone or spiked with 1.0 mM MMS. Cells were then incubated for 1.0 h at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with PBS, and cells were treated with 0.25% trypsin (HyClone), collected, and washed with PBS. Cells were then counted by hemacytometer. DNA damage was evaluated by CometAssay (Trevigen) performed under alkaline conditions. Cells were resuspended in PBS at 1 × 10<sup>5</sup>/mL and then added to premelted and cooled (37 °C) agarose at a 1:10 ratio. Cells were gently mixed, and then 50 μL of the agarose cell mix was transferred to prewarmed comet slides (37 °C; CometSlide). After solidifying at 4 °C, slides were placed in lysis solution for 60 min and then placed in freshly prepared alkaline unwinding solution (NaOH; pH 13) for 20 min. Slides were then subjected to electrophoresis under alkaline conditions (NaOH; pH 13) at 1 V/cm (300 mA) for 30 min and then washed in H<sub>2</sub>O, placed in 70% ethanol, and allowed to dry at 37 °C for 30 min. Slides were stained with 100 μL of 1:10 000 diluted SYBR Gold (Invitrogen) in TE pH 7.5 and incubated for 30 min. Slides were then rinsed in H<sub>2</sub>O briefly, and comets were captured by fluorescent microscope (Leica DMIL) and quantified by CometScore Pro (TriTec Corp). DNA damage was measured by percent Tail DNA, and average values were obtained for 20 comet readings/macrocycle.</div></div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Chemistry. General Methods</h3><div class="NLM_p last"><sup>1</sup>H NMR spectra were recorded at 400 or 500 MHz at ambient temperature with CD<sub>3</sub>COCD<sub>3</sub> as the solvent unless otherwise stated. <sup>13</sup>C NMR spectra were recorded at 100 or 125 MHz at ambient temperature with CD<sub>3</sub>COCD<sub>3</sub> as the solvent unless otherwise stated. Chemical shifts are reported in parts per million. Data for <sup>1</sup>H NMR are reported as follows: chemical shift, multiplicity (app = apparent, br = broad, s = singlet, d = doublet, t = triplet, q = quartet, sxt = sextet, m = multiplet, ovrlp = overlap), coupling constants, and integration. All <sup>13</sup>C NMR spectra were recorded with complete proton decoupling. High resolution mass spectra were obtained at the Boston University Chemical Instrumentation Center using a Waters Q-TOF mass spectrometer. Optical rotations were measured on all compounds for which sufficient material was available using a Rudolph Autopol II polarimeter. Analytical thin layer chromatography was performed using 0.25 mm silica gel 60-F plates. Flash chromatography was performed using 200–400 mesh silica gel (Sorbent Technologies, Inc.) or prepack column (SI-HC, puriFlash) by Interchim puriFlash450 or Yamazen Smart Flash EPCLC W-Prep2XY system. Isolated yields refer to chromatographically and spectroscopically pure compounds, unless otherwise stated. All reactions were carried out in oven-dried glassware under an argon atmosphere unless otherwise noted. Analytical LC–MS experiments were performed using a Waters Acquity UPLC (ultraperformance liquid chromatography) with a binary solvent manager, SQ mass spectrometer, Waters 2996 PDA (photodiode array) detector, and evaporative light scattering detector (ELSD). All microwave experiments were performed on a CEM Discover microwave reactor, using a sealed 10 or 35 mL vessel with temperatures monitored by an external sensor. All compounds tested in biological assays were determined to be >95% pure by UPLC–MS-ELSD analysis.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Tosyl-1<i>H</i>-pyrrole-2-carbaldehyde (<b>3b</b>)</h3><div class="NLM_p last">In a flame-dried, 250 mL round-bottomed flask equipped with an addition funnel under N<sub>2</sub>, a suspension of sodium hydride (1.89 g, 47.33 mmol, 60% purity) was stirred in THF (24 mL). The reaction was cooled to 0 °C in an ice bath. A solution of 1<i>H</i>-pyrrole-2-carbaldehyde (3.00 g, 31.55 mmol) in THF (8 mL) was added steadily, dropwise, via addition funnel. An additional 4 mL of THF was used to rinse the aldehyde solution vial and addition funnel. The ice bath was removed, and the reaction was allowed to stir at room for 60 min. The tan suspension was then recooled to 0 °C, and a solution of 4-methylbenzenesulfonyl chloride (7.22 g, 37.86 mmol) in THF (12 mL) was added steadily, dropwise, via the addition funnel, which was then rinsed with an additional 12 mL of THF. The resulting reddish-tan suspension was stirred at room temperature overnight. The reaction was quenched with water and poured into a separatory funnel containing ethyl acetate and water. The layers were separated, and the organic layer was washed first with water, then brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified via silica plug, eluting with 15% ethyl acetate in hexanes (300 mL) and 30% ethyl acetate in hexanes (300 mL). The eluent was condensed and the resulting residue was recrystallized from ethyl acetate/hexanes to give off-white solid (3.69 g). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.97 (s, 1H), 7.80 (d, <i>J</i> = 8.2 Hz, 2H), 7.62 (dd, <i>J</i> = 2.9, 1.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 2H), 7.16–7.14 (m, 1H), 7.16 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 6.40 (t, <i>J</i> = 3.2 Hz, 1H), 2.42 (s, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 145.9, 130.1,129.4, 127.4, 124.4, 112.4, 21.6; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>12</sub>NO<sub>3</sub>S [M + H]<sup>+</sup>, 250.0538; found, 250.0537.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> General Procedure for the Synthesis of Enantioenriched Alloc-Protected Homoallylic Amines <b>4</b></h3><div class="NLM_p last">To a flame-dried, nitrogen-purged flask equipped with a stir bar was added allyl carbamate (1.0 equiv). Anhydrous dichloromethane was added, followed by the requisite aldehydes <b>14</b>–<b>18</b>. The reaction mixture was cooled to −78 °C and BF<sub>3</sub>–diethyl etherate was slowly added via syringe. The reaction was stirred at −78 °C for 30 min. Crotylsilane <b>13</b> was added, and the reaction vessel was transferred to a chiller set to −35 °C. The reaction was stirred at this temperature for 48–72 h, tracking by TLC. Upon completion, the reaction mixture was carefully quenched with the addition of saturated aqueous sodium bicarbonate and extracted three times with dichloromethane. The combined organic fractions were washed with brine, dried over sodium sulfate, and condensed to give a crude product that was then purified by flash column chromatography.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Methyl (5<i>S</i>,6<i>R</i>,<i>E</i>)-6-(((Allyloxy)carbonyl)amino)-5-methyl-6-phenylhex-3-enoate (<b>4a</b>)</h3><div class="NLM_p last">Obtained as a clear oil (90% yield, 10:1 dr) from aldehyde <b>3a</b> utilizing the general procedure. [α]<sup>25</sup><sub>D</sub> +38 (<i>c</i> 0.39, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 7.24–7.16 (m, 2H), 7.16–7.10 (m, 3H), 5.81 (s, 2H), 5.46 (dt, <i>J</i> = 15.4, 6.6 Hz, 1H), 5.27 (dd, <i>J</i> = 15.4, 8.0 Hz, 1H), 5.19 (d, <i>J</i> = 16.8 Hz, 1H), 5.08 (d, <i>J</i> = 10.6 Hz, 1H), 4.58 (t, <i>J</i> = 7.4 Hz, 1H), 4.48 (dd, <i>J</i> = 13.3, 5.9 Hz, 1H), 4.41 (dd, <i>J</i> = 13.3, 4.3 Hz, 1H), 3.57 (s, 3H), 2.91 (d, <i>J</i> = 6.6 Hz, 2H), 2.56 (ddt, <i>J</i> = 8.0, 7.4, 7.0 Hz, 1H), 0.92 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ = 171.7, 155.2, 140.0, 135.2, 127.7, 126.9, 126.7, 123.0, 117.0, 65.0, 58.9, 51.4, 41.6, 37.3, 16.2; HRMS (<i>m</i>/<i>z</i>): [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>23</sub>NO<sub>4</sub>Na, 340.1525; found, 340.1528.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Methyl (5<i>S</i>,6<i>R</i>,<i>E</i>)-6-(((Allyloxy)carbonyl)amino)-5-methyl-6-(1-tosyl-1<i>H</i>-pyrrol-2-yl)hex-3-enoate (<b>4b</b>)</h3><div class="NLM_p last">Obtained as a clear oil (42% yield, >20:1 dr) from aldehyde <b>3b</b> utilizing the general procedure. [α]<sup>25</sup><sub>D</sub> +41 (<i>c</i> 0.31, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.74 (d, <i>J</i> = 7.6 Hz, 2H), 7.25 (d, <i>J</i> = 7.6 Hz, 2H), 7.18 (br s, 1H), 6.17 (t, <i>J</i> = 2.7 Hz, 1H), 6.15–6.13 (m, 1H), 5.89 (ddd, <i>J</i> = 17.2, 11.0, 5.9, 5.5 Hz, 1H), 5.45 (dt, <i>J</i> = 15.6, 6.6 Hz, 1H), 5.37 (dd, <i>J</i> = 15.6, 6.6 Hz, 1H), 5.26 (d, <i>J</i> = 17.2 Hz, 1H), 5.22–5.10 (m, 3H), 4.51 (dd, <i>J</i> = 13.5, 5.5 Hz, 1H), 4.46 (dd, <i>J</i> = 13.5, 5.9 Hz, 1H), 3.65 (s, 3H), 2.94 (d, <i>J</i> = 6.6 Hz, 2H), 2.80 (sxt, <i>J</i> = 6.5 Hz, 1H), 2.37 (s, 3H), 0.95 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 171.8, 154.9, 144.5, 135.8, 135.3, 134.5, 132.4, 132.2, 129.5, 126.5, 123.0, 122.5, 117.2, 113.6, 111.5, 76.4, 65.2, 51.9, 51.4, 40.8, 37.5, 21.2, 14.8. HRMS (<i>m</i>/<i>z</i>): [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>28</sub>N<sub>2</sub>O<sub>6</sub>SNa, 483.1566; found, 483.1579.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Methyl (5<i>S</i>,6<i>R</i>,<i>E</i>)-6-((<i>S</i>)-2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(1<i>H</i>-Indol-3-yl)propanamido)-5-methyl-6-phenylhex-3-enoate (<b>7a</b>)</h3><div class="NLM_p">In a 50 mL round-bottomed flask under N<sub>2</sub> was stirred <b>4a</b> (394.00 mg, 1.24 mmol, 1.0 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (4.5 mL) and methanol (1.5 mL). Dimethylbarbituric acid (242 mg, 1.55 mmol, 1.25 equiv) was then added, followed by tetrakis(triphenylphosphine)palladium(0) (85.97 mg, 74.40 μmol, 6 mol %). The reaction was stirred at room temperature for 4 h, forming a dark orange-reddish solution. MP-TsOH (65) resin (Biotage, Inc.) was added (1.6 g; 4.96 mmol; 4 equiv; loading, 3.11 mmol/g) in one portion. The reaction was diluted with CH<sub>2</sub>Cl<sub>2</sub> (3 mL) to allow for better mixing of resin. The resulting suspension was stirred at room temperature for 2 h. The resin was then filtered off and rinsed with CH<sub>2</sub>Cl<sub>2</sub> (5 × 10 mL). The resin was then transferred to a round-bottomed flask and suspended in CH<sub>2</sub>Cl<sub>2</sub> (20 mL). To the suspension was added Et<sub>3</sub>N (5.5 mL). The mixture was stirred at room temperature for 30 min. The resin was then filtered off and rinsed with CH<sub>2</sub>Cl<sub>2</sub> (4 × 5 mL), retaining the mother liquor. The resin was retreated with 20 mL of CH<sub>2</sub>Cl<sub>2</sub> and 5 mL of Et<sub>3</sub>N for 20 min and filtered again, rinsing with 3 × 5 mL of CH<sub>2</sub>Cl<sub>2</sub>. The combined mother liquors from both treatments were concentrated to afford the deprotected amine <b>5a</b> that was carried forward to the next step without further purification.</div><div class="NLM_p last">Compound <b>5a</b> and Boc-<span class="smallcaps smallerCapital">L</span>-tryptophan (434.60 mg, 1.43 mmol, 1.5 equiv) were combined in CH<sub>2</sub>Cl<sub>2</sub> (6 mL). HATU (542.97 mg, 1.43 mmol, 1.5 equiv) was then added, followed by Hunig’s base (384.49 mg, 2.97 mmol, 2.4 equiv). The reaction was stirred at room temperature for 3.5 h. The solvent was removed in vacuo, and the crude residue was purified by flash chromatography (SiO<sub>2</sub>, gradient elution 35–45% ethyl acetate in hexanes) to give product <b>7a</b> (451 mg; 72.9% yield over two steps). [α]<sup>25</sup><sub>D</sub> +29 (<i>c</i> 0.20, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.05 (br s, 1H), 7.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.35 (br d, <i>J</i> = 7.8 Hz, 2H), 7.27–7.20 (m, 4H), 7.19–7.14 (m, 1H), 7.13–7.04 (m, 2H), 6.99 (br t, <i>J</i> = 7.2 Hz, 1H), 5.98 (br d, <i>J</i> = 7.0 Hz, 1H), 5.24 (br d, <i>J</i> = 2.3 Hz, 2H), 4.82 (br t, <i>J</i> = 7.6 Hz, 1H), 4.50–4.37 (m, <i>J</i> = 14.3, 7.0 Hz, 1H), 3.56 (s, 3H), 3.21 (dd, <i>J</i> = 14.3, 6.6 Hz, 1H), 3.09 (dd, <i>J</i> = 14.3, 7.0 Hz, 1H), 2.86 (br s, 2H), 2.47–2.35 (m, <i>J</i> = 6.6 Hz, 1H), 1.33 (br s, 9H), 0.82 (br d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 206.4, 172.4, 171.8, 156.3, 141.8, 137.6, 136.4, 128.8, 128.6, 128.4, 127.6, 124.4, 124.2, 122.1, 119.5, 119.5, 112.2, 112.2, 111.5, 79.4, 58.0, 56.4, 51.9, 42.4, 38.0, 29.0, 28.6, 16.7; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>37</sub>N<sub>3</sub>O<sub>5</sub>Na [M + Na]<sup>+</sup>, 542.2631; found, 542.2630.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Methyl (5<i>S</i>,6<i>R</i>,<i>E</i>)-6-((<i>S</i>)-2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(1<i>H</i>-indol-3-yl)propanamido)-5-methyl-6-(1-tosyl-1<i>H</i>-pyrrol-2-yl)hex-3-enoate (<b>7b</b>)</h3><div class="NLM_p">In a 100 mL round-bottomed flask under N<sub>2</sub>, <b>4b</b> (1.10 g, 2.39 mmol, 1.0 equiv) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (9 mL) and methanol (4 mL). Dimethyl barbituric acid (447.53 mg, 2.87 mmol, 1.2 equiv) was then added, followed by tetrakis(triphenylphosphine)palladium(0) (165.60 mg, 143.31 μmol, 6 mol %). The reaction was stirred at room temperature for 4 h. The dark orange reaction was then diluted with CH<sub>2</sub>Cl<sub>2</sub> (5 mL), and MP-TsOH (65) resin (Biotage, Inc.) (3.1 g; 9.56 mmol; 4 equiv; loading, 3.11 mmol/g) was added. The reaction was stirred at room temperature for 2 h. The resin was filtered off and rinsed with CH<sub>2</sub>Cl<sub>2</sub> (4 × 5 mL), 10% MeOH/CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL), CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL).</div><div class="NLM_p">The resin was then then returned to the round-bottom flask and dissolved in CH<sub>2</sub>Cl<sub>2</sub> (20 mL). Et<sub>3</sub>N (5 mL) was added, and the suspension was stirred at room temperature for 1.25 h. The resin was filtered off and rinsed with CH<sub>2</sub>Cl<sub>2</sub> (4 × 10 mL), retaining the mother liquor. The resin was then retreated with CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and Et<sub>3</sub>N (5 mL) for 45 min. The resin was again filtered off, rinsing with CH<sub>2</sub>Cl<sub>2</sub> (3 × 5 mL). The combined mother liquors from both treatments were concentrated to afford the deprotected amine <b>5b</b>, which was carried forward to the next step without further purification.</div><div class="NLM_p last">In a 100 mL round bottomed flask, amine <b>5b</b> was stirred with Boc-<span class="smallcaps smallerCapital">L</span>-tryptophan (836.33 mg, 2.75 mmol, 1.15 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (11 mL). HATU (957.80 mg, 2.52 mmol, 1.05 equiv) was then added, followed by Hunig’s base (739.9 mg, 5.73 mmol, 999.86 μL, 2.4 equiv). The reaction was stirred at room temperature overnight. The reaction was then diluted with CH<sub>2</sub>Cl<sub>2</sub> and washed with water. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude residue was purified by flash chromatography (SiO<sub>2</sub>; gradient elution 38–48% ethyl acetate in hexanes) to give product <b>7b</b> (1.09 g, 71.8% yield over two steps). [α]<sup>25</sup><sub>D</sub> +34 (<i>c</i> 0.22, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.07 (br s, 1H), 7.95 (br d, <i>J</i> = 8.2 Hz, 2H), 7.62 (d, <i>J</i> = 7.8 Hz, 1H), 7.38 (br d, <i>J</i> = 8.2 Hz, 3H), 7.29 (br d, <i>J</i> = 9.0 Hz, 1H), 7.15 (br d, <i>J</i> = 1.6 Hz, 2H), 7.09 (t, <i>J</i> = 7.4 Hz, 1H), 7.00 (t, <i>J</i> = 7.4 Hz, 1H), 6.26–6.15 (m, 2H), 5.99 (br d, <i>J</i> = 7.2 Hz, 1H), 5.86–5.76 (m, 1H), 5.50–5.26 (m, 2H), 4.49–4.38 (m, 1H), 3.60 (s, 3H), 3.25 (dd, <i>J</i> = 14.5, 6.3 Hz, 1H), 3.08 (br dd, <i>J</i> = 14.3, 7.2 Hz, 1H), 2.93 (br s, 2H), 2.63 (br d, <i>J</i> = 5.5 Hz, 1H), 2.39 (s, 3H), 1.35 (br s, 9H), 0.87 (br d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.8, 145.2, 136.5, 136.0, 130.1, 127.3, 123.4, 122.5, 119.9, 119.2, 114.2, 112.2, 111.5, 52.2, 37.9, 28.6, 21.9; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>35</sub>H<sub>42</sub>N<sub>4</sub>O<sub>7</sub>SNa [M + Na]<sup>+</sup>, 685.2672; found, 685.2673.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Macrocyclic Library Synthesis</h3><div class="NLM_p last">Macrocycles employed in the primary screen were synthesized according to the general procedure for macrolactone synthesis outlined below. Crude reaction mixtures were directly purified using mass-guided, preparative HPLC on a Waters FractionLynx system. Purified compounds meeting a >90% purity threshold (as determined by UPLC–MS-ELSD) were stored as 20 mM DMSO stocks at −30 °C prior to aliquotting for screening.</div></div><div id="sec4_7_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> General Procedure for Macrolactone Library Synthesis (<b>MC001</b>–<b>MC105</b>)</h3><div class="NLM_p last">In a 25 mL round-bottomed flask was stirred Boc-protected amine <b>7</b> (0.57 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (6 mL). A solution of 4.0 M HCl in dioxane (0.86 mL, 6.0 equiv) was added. The reaction was stirred at room temperature for 3.25 h. The solvent was evaporated, and the residue was azeotroped four times with CH<sub>2</sub>Cl<sub>2</sub> (4 mL portions). The resultant gum was then taken up in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). MP-carbonate resin (Biotage, Inc.) was added (0.31 g; 0.89 mmol; 3.6 equiv; 2.9 mmol/g loading). The suspension was stirred at room temperature for 1.5 h. The resin was filtered off and rinsed with CH<sub>2</sub>Cl<sub>2</sub> (3 × 4 mL). The mother liquor was evaporated, and the residue was transferred to a 10 mL microwave vessel using trifluorotoluene (1.2 mL). Lactone <b>9</b> (6.0 equiv) was added, followed by Otera’s catalyst (15 mol %). The vessel was capped and heated in the microwave at 175 °C for 45 min. For the original macrocycle library, the solvent was removed using centrifugal evaporation and the resultant residues were dissolved in DMSO and purified via mass-guided preparative HPLC. For follow-on syntheses and characterizations, the reaction suspension was directly loaded onto a silica/Celite column (prewet with 20% acetone/H) using CH<sub>2</sub>Cl<sub>2</sub>. Elution with a gradient of 20% acetone/hexanes, (40 mL), 30% acetone/hexanes (20 mL), 40% acetone/hexanes (20 mL), 50% acetone/hexanes (20 mL), 60% acetone/hexanes (20 mL) afforded the analytically pure macrolactone.</div></div><div id="sec4_7_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Synthetic Yields and Full Characterization Data for Macrolactones <b>10</b>–<b>13</b>: (7<i>S</i>,10<i>R</i>,11<i>S</i>,<i>E</i>)-7-((1<i>H</i>-Indol-3-yl)methyl)-11-methyl-10-phenyl-1-oxa-6,9-diazacyclopentadec-12-ene-5,8,15-trione (<b>10</b>)</h3><div class="NLM_p last">Macrolactone <b>10</b> was obtained from Boc-amine <b>7a</b> and lactone <b>9a</b> according to the general procedure (29.8% yield over two steps). [α]<sup>25</sup><sub>D</sub> +7 (<i>c</i> 0.13, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.01 (br s, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.37–7.20 (m, 8H), 7.17–7.02 (m, 2H), 6.98 (t, <i>J</i> = 7.8 Hz, 1H), 5.60 (ddd, <i>J</i> = 15.2, 8.6, 5.9 Hz, 1H), 5.14 (dd, <i>J</i> = 15.2, 9.2 Hz, 1H), 4.94 (dd, <i>J</i> = 7.8, 3.1 Hz, 1H), 4.82–4.72 (m, 1H), 4.33–4.24 (m, 1H), 3.36 (d, <i>J</i> = 15.0, 3.1 Hz, 1H), 3.10 (d, <i>J</i> = 15.0, 8.5 Hz, 1H), 3.03 (dd, <i>J</i> = 16.0, 5.9 Hz, 1H), 2.96 (dd, <i>J</i> = 16.0, 8.6 Hz, 1H), 2.69–2.59 (m, 1H), 2.42–2.32 (m, 1H), 2.31–2.20 (m, 2H), 1.76–1.65 (m, 1H), 0.87 (d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.9, 172.5, 171.8, 140.2, 138.2, 136.4, 129.3, 129.3, 129.6, 129.1, 128.5, 128.3, 126.2, 125.2, 124.7, 124.5, 122.7, 120.2, 120.1, 119.8, 112.7, 112.5, 67.0, 58.7, 56.0, 42.1, 40.0, 35.0, 28.3, 24.2, 18.2; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>3</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 496.2212; found, 496.2223</div></div><div id="sec4_7_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (8<i>S</i>,11<i>R</i>,12<i>S</i>,<i>E</i>)-8-((1<i>H</i>-Indol-3-yl)methyl)-12-methyl-11-phenyl-1-oxa-7,10-diazacyclohexadec-13-ene-6,9,16-trione (<b>11</b>)</h3><div class="NLM_p last">Macrolactone <b>11</b> was obtained from Boc-amine <b>7a</b> and lactone <b>9i</b> according to the general procedure (32% yield over two steps). [α]<sup>25</sup><sub>D</sub> −24 (<i>c</i> 0.17, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.99 (br s, 1H), 7.49 (d, <i>J</i> = 8.2 Hz, 1H), 7.45 (d, <i>J</i> = 8.2 Hz, 1H), 7.37–7.22 (m, 7H), 7.07 (t, <i>J</i> = 7.6 Hz, 1H), 6.96 (t, <i>J</i> = 7.0 Hz, 1H), 6.87 (br s, 1h), 5.71 (ddd, <i>J</i> = 15.5, 10.0, 4.3, 1H), 5.26 (dd, <i>J</i> = 15.5, 9.0 Hz, 1H), 4.93 (dd, <i>J</i> = 8.2, 3.5 Hz, 1H), 4.78 (td, <i>J</i> = 8.0, 4.5 Hz, 1H), 4.24 (td, <i>J</i> = 10.5, 3.3 Hz, 1H), 4.01 (dt, <i>J</i> = 10.5, 4.7 Hz, 1H), 3.19 (dd, <i>J</i> = 15.0, 4.5 Hz, 1H), 3.11 (dd, <i>J</i> = 15.0, 8.0 Hz, 1H), 3.09 (dd, <i>J</i> = 15.0, 6.6 Hz), 2.9–2.7 (m, 2H, obsc.), 2.65–2.56 (m, 1H), 2.23 (td, <i>J</i> = 9.4, 4.7 Hz, 1H), 1.95–1.85 (m, 1H), 1.80–1.65 (m, 2H), 1.65–1.57 (m, 1H), 0.92 (t, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.0, 170.8, 170.7, 139.8, 136.8, 134.2, 128.6, 128.4, 127.8, 127.2, 124.9, 123.8, 121.5, 119.0, 118.6, 111.5, 62.4, 57.5, 54.4, 41.9, 40.8, 39.2, 34.1, 27.4, 27.2, 21.8, 18.1; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>33</sub>N<sub>3</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 510.2369; found, 510.2354.</div></div><div id="sec4_7_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (7<i>S</i>,10<i>R</i>,11<i>S</i>,<i>E</i>)-7-((1<i>H</i>-Indol-3-yl)methyl)-11-methyl-10-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1-oxa-6,9-diazacyclopentadec-12-ene-5,8,15-trione (<b>12</b>)</h3><div class="NLM_p last">Macrolactone <b>12</b> was obtained from Boc-amine <b>7b</b> and lactone <b>9a</b> according to the general procedure (36.9% yield over two steps). [α]<sup>25</sup><sub>D</sub> +5 (<i>c</i> 0.526, CHCl<sub>3</sub>); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.02 (br s, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 2H), 7.60 (d, <i>J</i> = 7.8 Hz, 1H), 7.42 (d, <i>J</i> = 8.2 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 1H), 7.30 (br s, 1H), 7.13–7.04 (m, 3H), 7.00 (t, <i>J</i> = 7.8 Hz, 1H), 6.32 (br s, 1H), 6.28 (t, <i>J</i> = 3.1 Hz, 1H), 5.94 (dd, <i>J</i> = 8.8, 4.1 Hz, 1H), 5.52 (dt, <i>J</i> = 15.2, 7.4, 7.4 Hz, 1H), 5.38 (dd, <i>J</i> = 15.2, 7.8 Hz, 1H), 4.68 (ddd, <i>J</i> = 10.6, 7.8, 3.5 Hz, 1H), 4.23–4.15 (m, 1H), 4.06 (t, <i>J</i> = 9.4 Hz, 1H), 3.44 (dd, <i>J</i> = 15.0, 3.5 Hz, 1H), 3.03–2.95 (m, 2H), 2.89 (dd, J = 15.0, 10.6 Hz, 1H), 2.74–2.63 (m, 1H), 2.39 (s, 3H), 2.34–2.20 (m, 1H), 2.20–2.07 (m, 2H), 1.66–1.53 (m, 1H), 0.90 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.9, 173.0, 171.8, 146.8, 138.2, 137.8, 136.7, 134.9, 131.6, 128.8, 126.3, 124.6, 124.1, 122.8, 120.2, 119.8, 116.0, 113.5, 112.8, 112.7, 66.9, 56.4, 50.1, 41.6, 39.9, 35.1, 29.0, 24.3, 22.1, 17.1; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>36</sub>N<sub>4</sub>O<sub>6</sub>SNa [M + Na]<sup>+</sup>, 639.2253; found, 639.2248.</div></div><div id="sec4_7_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (8<i>S</i>,11<i>R</i>,12<i>S</i>,<i>E</i>)-8-((1<i>H</i>-Indol-3-yl)methyl)-12-methyl-11-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1-oxa-7,10-diazacyclohexadec-13-ene-6,9,16-trione (<b>13</b>)</h3><div class="NLM_p last">Macrolactone <b>13</b> was obtained from Boc-amine <b>7b</b> and lactone <b>9i</b> according to the general procedure (30.5% yield over two steps). [α]<sup>25</sup><sub>D</sub> −29 (<i>c</i> 0.32, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.99 (br s, 1H), 8.06 (d, <i>J</i> = 8.0 Hz, 2H), 7.50 (d, <i>J</i> = 7.8 Hz, 1H), 7.82 (d, <i>J</i> = 8.0 Hz, 2H), 7.42–7.37 (m, 1H), 7.35 (d, <i>J</i> = 8.2 Hz, 1H), 7.23 (dd, <i>J</i> = 3.1, 2.0 Hz, 1H), 7.17 (br d, <i>J</i> = 8.2 Hz, 1H), 7.06 (t, <i>J</i> = 7.6 Hz, 1H), 6.95 (br s, 1H), 6.93 (t, <i>J</i> = 7.0 Hz, 1H), 6.25–6.20 (m, 2H), 5.94 (dd, <i>J</i> = 8.2, 3.5 Hz, 1H), 5.63 (ddd, <i>J</i> = 16.0, 6.0, 5.0 Hz, 1H), 5.54 (dd, <i>J</i> = 16.0, 6.3 Hz, 1H), 4.70 (td, <i>J</i> = 9.0, 4.0 Hz, 1H), 4.18 (ddd, <i>J</i> = 10.9, 9.0, 4.3 Hz, 1H), 4.12 (dd, <i>J</i> = 10.9, 5.1 Hz, 1H), 3.20 (dd, <i>J</i> = 15.4, 4.0 Hz, 1H), 3.09 (dd, <i>J</i> = 15.4, 9.0 Hz, 1H), 3.02 (d, <i>J</i> = 5.9 Hz, 1H), 3.00 (d, <i>J</i> = 6.3 Hz, 1H), 2.77–2.70 (m, 1H), 2.37 (s, 3H), 2.21 (dt, <i>J</i> = 14.3, 7.4 Hz, 1H), 1.98 (dt, <i>J</i> = 14.3, 7.8 Hz, 1H), 1.90–1.80 (m, 1H), 1.70–1.60 (m, 2H), 1.60–1.50 (m, 1H), 0.93 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.1, 171.4, 171.2, 145.5, 137.0, 136.8, 134.9, 130.3, 128.2, 127.8, 124.2, 123.7, 123.3, 121.6, 119.1, 118.7, 114.9, 112.6, 111.6, 110.9, 62.7, 55.4, 49.7, 41.0, 38.7, 34.4, 27.9, 27.7, 21.9, 21.0, 15.3; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>39</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup>, 631.2590; found, 631.2586.</div></div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Synthetic Procedures and Full Characterization Data for Macrolactams <b>17</b>–<b>32</b>. Part A: General Procedure for Amino Acid Coupling</h3><div class="NLM_p">In a 25 mL round-bottomed flask, compound <b>7a</b>/<b>7b</b> (0.576 mmol, 1.0 equiv) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (6 mL). A solution of 4.0 M HCl in dioxane (0.86 mL, 6.0 equiv) was then added. The reaction was stirred at room temperature for 3.25 h, at which time the solvent was removed in vacuo. The resultant residue was azeotroped four times with 4 mL portions of CH<sub>2</sub>Cl<sub>2</sub> to afford <b>8a</b>/<b>8b</b> as a foamy gum. This crude gum was then dissolved in CH<sub>2</sub>Cl<sub>2</sub> (1 mL) in a 25 mL round-bottomed flask. The appropriate Boc-amino acid <b>14a</b>–<b>k</b> (1.1 equiv) was added, followed by HATU (1.1 equiv) and Hunig’s base (2.0 equiv). After stirring at room temperature for 4 h, the reaction mixture was concentrated in vacuo and the crude residue was directly purified by flash column chromatography using a gradient elution of 25–45% acetone in hexanes to afford the Boc-protected intermediate <b>15/16</b>.</div><div class="NLM_p last">Several of the Boc-protected intermediates exhibited significant peak splitting, possibly due to the presence of Boc rotamers. In addition, several Boc intermediates were not sufficiently pure for characterization and were carried onto the next step (part B) without further attempts to purify. Only the Boc-intermediates of sufficient purity for characterization are described below.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Part B: General Procedure for Three-Step Deprotection/Cyclization Sequence</h3><div class="NLM_p last">Intermediate <b>15/16</b> (0.105 mmol) was dissolved in methanol (1 mL). A solution of 2.5 M aqueous NaOH (1.5 equiv) was then added, and the reaction was stirred at room temperature overnight. The solvent was removed in vacuo, and the residue was dissolved in THF (1 mL). The solution was acidified via addition of 1 N HCl (0.13 mL). The mixture was stirred for 2 min, and the solvent was removed in vacuo. The residue was then dissolved in CH<sub>2</sub>Cl<sub>2</sub> and the solution was passed through an Isolute HM-N cartridge, eluting with additional CH<sub>2</sub>Cl<sub>2</sub>. The solution was concentrated in a 25 mL round-bottom flask and 1 mL of CH<sub>2</sub>Cl<sub>2</sub> was added, followed by a solution of 4.0 N HCl in dioxane (5.0 equiv). The reaction was stirred at room temperature for 3 h and then concentrated in vacuo. The resulting residue was azeotroped four times with CH<sub>2</sub>Cl<sub>2</sub>, and transferred to a two dram vial in DMF (1.2 mL). To this solution was added HATU (1.75 equiv), followed by Hunig’s base (4.0 equiv). The reaction was stirred at room temperature overnight. The solvent was removed in vacuo and the crude residue was purified by flash column chromatography (SiO2; gradient elution 20–60% acetone in hexanes) to afford the macrolactam product.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Methyl (6<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-6-(4-fluorophenyl)-2,2,14-trimethyl-4,8,11-trioxo-13-phenyl-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>15i</b>)</h3><div class="NLM_p last">77% yield from <b>7a</b>, after deprotection and coupling to <b>14i</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +16 (<i>c</i> 0.146, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.11 (br s, 1H), 7.66 (br d, <i>J</i> = 7.8 Hz, 1H), 7.47 (br d, <i>J</i> = 7.4 Hz, 1H), 7.39 (br d, <i>J</i> = 7.8 Hz, 3H), 7.34–7.08 (m, 9H), 7.04 (br t, <i>J</i> = 7.4 Hz, 1H), 6.95 (br t, <i>J</i> = 8.6 Hz,1H), 6.83 (br d, <i>J</i> = 6.3 Hz, 1H), 5.18–4.97 (m, 3H), 4.83–4.69 (m, 2H), 3.59 (s, 3H), 3.17–3.03 (m, 2H), 2.81 (br d, <i>J</i> = 6.3 Hz, 2H), 2.62–2.49 (m, 2H), 2.28–2.16 (m, 1H), 1.36 (br s, 9H), 0.73 (br d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.5, 171.1, 170.7, 162.6 (d, <i>J</i><sub>C–F</sub> = 242.8 Hz, 1 C), 155.8, 141.8, 137.7, 136.2, 129.1, 129.0, 128.8, 128.7 (2 C), 128.5 (2 C), 127.7, 124.5 (2 C), 124.4 (2 C), 122.3, 119.7, 119.7, 115.7 (d, <i>J</i><sub>C–F</sub> = 21.4 Hz, 2 C), 112.3, 111.6, 79.1, 58.2, 55.3, 52.4, 52.0, 43.1, 42.4, 38.0, 28.7, 17.0; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>46</sub>FN<sub>4</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 685.3401; found, 685.3389.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Methyl (6<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-6-(4-chlorophenyl)-2,2,14-trimethyl-4,8,11-trioxo-13-phenyl-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>15j</b>)</h3><div class="NLM_p last">78% yield from <b>7a</b>, after deprotection and coupling to <b>14j</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +13 (<i>c</i> 0.173, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.06 (br s, 1H), 7.64 (d, <i>J</i> = 7.8 Hz, 1H), 7.39 (br d, <i>J</i> = 7.8 Hz, 1H), 7.32–7.14 (m, 11H), 7.14–7.08 (m, 2H), 7.04 (t, <i>J</i> = 7.1 Hz, 1H), 6.82 (br s, 1H), 5.21–4.94 (m, 3H), 4.79–4.63 (m, 2H), 3.60 (s, 3H), 3.06 (br d, <i>J</i> = 7.0 Hz, 2H), 2.73 (s, 3H), 2.62 (br d, <i>J</i> = 9.8 Hz, 1H), 2.30–2.13 (m, 1H), 1.36 (br s, 9H), 0.74 (br d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 172.2, 169.8, 169.7, 154.9, 138.6, 135.9, 134.6, 132.5, 128.2, 127.6, 127.1, 127.0, 126.8, 123.4, 122.8, 121.9, 119.3, 118.4, 111.1, 110.3, 79.3, 56.8, 54.0, 51.6, 41.6, 41.0, 38.3, 36.9, 29.3, 28.5, 28.0, 16.3; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>45</sub>ClN<sub>4</sub>O<sub>6</sub>Na [M + Na]<sup>+</sup>, 723.2925; found, 723.2924.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Methyl (11<i>S</i>,14<i>R</i>,15<i>S</i>,<i>E</i>)-11-((1<i>H</i>-Indol-3-yl)methyl)-2,2,15-trimethyl-4,9,12-trioxo-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-3-oxa-5,10,13-triazanonadec-16-en-19-oate (<b>16a</b>)</h3><div class="NLM_p last">81% yield from <b>7b</b>, after deprotection and coupling to <b>14a</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +17 (<i>c</i> 0.14, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.04 (br s, 1H), 7.91 (br d, <i>J</i> = 8.2 Hz, 2H), 7.60 (d, <i>J</i> = 7.8 Hz, 1H), 7.38 (br d, <i>J</i> = 9.0 Hz, 1H), 7.36–7.29 (m, 4H), 7.14 (br s, 2H), 7.09–7.02 (m, 1H), 7.01–6.92 (m, 1H), 6.16 (br s, 1H), 6.13 (t, <i>J</i> = 3.3 Hz, 1H), 5.94 (br s, 1H), 5.77 (dd, <i>J</i> = 8.8, 5.7 Hz, 1H), 5.41–5.22 (m, 2H), 4.73 (q, <i>J</i> = 7.4 Hz, 1H), 3.56 (s, 3H), 3.24 (dd, <i>J</i> = 14.9, 6.3 6 Hz, 1H), 3.05 (dd, <i>J</i> = 14.5, 7.4 Hz, 1H), 2.97–2.83 (m, 4H), 2.63–2.52 (m, 1H), 2.33 (s, 3H), 2.15–2.08 (m, 2H), 1.71–1.56 (m, 1H), 1.70–1.55 (m, 1H), 1.72–1.55 (m, 2H), 1.35 (s, 9H), 0.83 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.5, 172.9, 172.3, 146.5, 138.0, 137.7, 136.9, 136.7, 131.3, 129.2, 128.5, 124.8, 124.4, 124.1, 122.6, 120.0, 119.9, 115.3, 113.5, 112.6, 112.0, 79.0, 55.2, 52.3, 51.2, 43.1, 41.0, 38.6, 34.3, 31.1, 30.9, 30.7, 30.5, 30.1, 30.0, 29.8, 30.3, 29.2, 27.4, 22.0, 15.8; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>49</sub>N<sub>5</sub>O<sub>8</sub>SNa [M + Na]<sup>+</sup>, 770.3200; found, 770.3185.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Methyl (12<i>S</i>,15<i>R</i>,16<i>S</i>,<i>E</i>)-12-((1<i>H</i>-Indol-3-yl)methyl)-2,2,16-trimethyl-4,10,13-trioxo-15-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-3-oxa-5,11,14-triazaicos-17-en-20-oate (<b>16b</b>)</h3><div class="NLM_p last">74% yield from <b>7b</b>, after deprotection and coupling to <b>14b</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +19 (<i>c</i> 0.14, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.05 (br s, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 2H), 7.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.40–7.29 (m, 4H), 7.23 (br d, <i>J</i> = 7.8 Hz, 1H), 7.15 (d, <i>J</i> = 1.6 Hz, 2H), 7.06 (t, <i>J</i> = 1.0 Hz, 1H), 6.98 (t, <i>J</i> = 1.0 Hz, 1H), 6.19–6.10 (m, 2H), 5.82 (br s, 1H), 5.76 (dd, <i>J</i> = 9.0, 5.5 Hz, 1H), 5.36–5.30 (m, 2H), 4.73 (q, <i>J</i> = 7.6 Hz, 1H), 3.56 (s, 3H), 3.24 (dd, <i>J</i> = 14.5, 6.6 Hz, 1H), 3.04 (dd, <i>J</i> = 14.5, 7.8 Hz, 1H), 2.97–2.84 (m, 4H), 2.64–2.50 (m, 1H), 2.33 (s, 3H), 2.13–2.07 (m, 2H), 1.55–1.41 (m, 2H), 1.36 (s, 9H), 1.33–1.22 (m, 2H), 0.82 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.3, 172.5, 171.9, 156.7, 146.1, 137.6, 137.2, 136.4, 136.3, 130.9, 128.8, 128.1, 124.4, 124.0, 123.7, 122.2, 119.6, 119.5, 114.9, 113.1, 112.2, 112.2, 111.6, 78.4, 54.7, 51.9, 50.7, 42.5, 40.7, 38.2, 36.2, 30.7, 28.8, 23.5, 21.6, 15.3; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>40</sub>H<sub>51</sub>N<sub>5</sub>O<sub>8</sub>SNa [M + Na]<sup>+</sup>, 784.3356; found, 784.3343.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Methyl (10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-2,2,14-trimethyl-4,8,11-trioxo-13-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>16c</b>)</h3><div class="NLM_p last">93% yield from <b>7b</b>, after deprotection and coupling to <b>14c</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +19 (<i>c</i> 0.27, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.06 (br s, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 2H), 7.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.42–7.29 (m, 5H), 7.15 (br s, 2H), 7.10–7.02 (m, 1H), 6.92–7.02 (m, 1H), 6.19–6.12 (m, 2H), 5.85 (br s, 1HN), 5.76 (br dd, <i>J</i> = 9.0, 5.5 Hz, 1H), 5.35–5.28 (m, 2H), 4.72 (q, <i>J</i> = 7.2 Hz, 1H), 3.56 (s, 3H), 3.30–3.15 (m, 3H), 3.13–3.00 (m, 1H), 2.88 (br d, <i>J</i> = 5.5 Hz, 2H), 2.64–2.50 (m, 1H), 2.33 (s, 3H), 2.36–2.23 (m, 2H), 1.34 (s, 9H), 0.82 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.9, 172.4, 172.1, 172.1, 157.0, 146.5, 138.0, 137.7, 136.7, 131.3, 129.2, 128.6, 124.9, 124.8, 124.4, 124.1, 124.1, 122.7, 120.0, 120.0, 115.3, 113.6, 112.7, 112.6, 112.0, 79.2, 55.2, 52.3, 51.2, 43.0, 39.2, 38.6, 38.2, 37.1, 29.3, 29.1, 22.0, 15.8; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>38</sub>H<sub>47</sub>N<sub>5</sub>O<sub>8</sub>SNa [M + Na]<sup>+</sup>, 756.3043; found, 756.3033.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Methyl (6<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-2,2,14-trimethyl-4,8,11-trioxo-13-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-6-(4-(trifluoromethyl)phenyl)-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>16g</b>)</h3><div class="NLM_p last">70% yield from <b>7b</b>, after deprotection and coupling to <b>14g</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +3 (<i>c</i> 0.36, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.07 (br s, 1H), 7.88 (br d, <i>J</i> = 8.6 Hz, 2H), 7.59 (br d, <i>J</i> = 7.8 Hz, 1H), 7.51 (br d, <i>J</i> = 7.8 Hz, 2H), 7.45 (d, <i>J</i> = 7.8 Hz, 3H), 7.37–7.26 (m, 4H), 7.17–7.04 (m, 3H), 6.98 (t, <i>J</i> = 7.4 Hz, 1H), 6.90 (br d, <i>J</i> = 7.4 Hz, 1H), 6.16–6.07 (m, 2H), 5.72 (dd, <i>J</i> = 8.6, 5.9 Hz, 1H), 5.21 (br s, 2H), 5.11 (br s, 1H), 4.69 (q, <i>J</i> = 7.0 Hz, 1H), 3.56 (s, 3H), 3.13 (t, <i>J</i> = 14.5 Hz, 1H), 3.00 (t, <i>J</i> = 14.5 Hz, 1H), 2.92–2.74 (m, 4H), 2.52–2.44 (m, 1H), 2.30 (s, 3H), 1.33 (br s, 9H), 0.75 (br d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.5, 172.5, 171.4, 170.6, 170.6, 165.9, 155.8, 148.8, 146.0, 137.6, 137.3, 136.2, 130.9, 129.5, 128.7, 128.1, 128.0, 126.8, 126.0, 124.4, 128.0, 124.0, 123.6, 122.2, 119.6, 119.6, 114.9, 114.7, 113.2, 113.1, 113.1, 113.1, 112.2, 112.2, 112.2, 111.4, 79.3, 54.8, 52.7, 52.6, 51.9, 51.9, 50.7, 42.7, 42.5, 38.8, 38.1, 28.6, 29.5, 21.6, 15.4; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>45</sub>H<sub>51</sub>F<sub>3</sub>N<sub>5</sub>O<sub>8</sub>S [M + H]<sup>+</sup>, 878.3410; found, 878.3417.</div></div><div id="sec4_8_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Methyl (6<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-6-(4-fluorophenyl)-2,2,14-trimethyl-4,8,11-trioxo-13-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>16i</b>)</h3><div class="NLM_p last">79% yield from <b>7b</b>, after deprotection and coupling to <b>14i</b>, according to the general procedure, part A. [α]<sup>25</sup><sub>D</sub> +5 (<i>c</i> 0.28, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.06 (br s, 1H), 7.92 (d, <i>J</i> = 8.6 Hz, 2H), 7.62 (d, <i>J</i> = 7.8 Hz, 1H), 7.47–7.26 (m, 8H), 7.17 (dd, <i>J</i> = 3.1, 2.0 Hz, 1H), 7.14–7.08 (m, 2H), 7.01 (t, <i>J</i> = 7.2 Hz, 1H), 6.95 (br t, <i>J</i> = 8.8 Hz, 1H), 6.89–6.76 (m, 1H), 6.16 (br t, <i>J</i> = 3.3 Hz, 1H), 6.13 (br s, 1H), 5.75 (dd, <i>J</i> = 8.8, 5.7 Hz, 1H), 5.27–5.21 (m, 2H), 5.06 (br s, 1H), 4.71 (dd, <i>J</i> = 14.5, 7.3 Hz, 1H), 3.60 (s, 3H), 3.16 (dd, <i>J</i> = 14.5, 7.0 Hz, 1H), 3.02 (dd, J = 14.0, 7.0 Hz, 1H), 2.86 (br d, <i>J</i> = 2.3 Hz, 2H), 2.82–2.72 (m, 1H), 2.68–2.59 (m, 1H), 2.51 (br dd, J = 6.50, 11.00 Hz, 1H), 2.35 (s, 3H), 1.36 (br s, 9H), 0.79 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.5, 171.4, 170.8, 163.9, 161.4, 155.8, 146.1, 137.6, 137.3, 136.3, 136.2, 130.9, 129.2, 129.1, 128.7, 128.1, 124.5, 124.4, 124.0, 123.6, 122.3, 119.7, 119.6, 115.8, 115.6, 115.2, 114.9, 113.1, 112.3, 111.4, 79.2, 79.1, 54.8, 52.3, 51.9, 51.9, 50.7, 50.7, 43.1, 42.6, 38.2, 29.1, 28.7, 21.6, 15.4; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>44</sub>H<sub>51</sub>FN<sub>5</sub>O<sub>8</sub>S [M + H]<sup>+</sup>, 828.3442; found, 828.3446.</div></div><div id="sec4_8_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Methyl (6<i>R</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,<i>E</i>)-10-((1<i>H</i>-Indol-3-yl)methyl)-6-(4-chlorobenzyl)-2,2,14-trimethyl-4,8,11-trioxo-13-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-3-oxa-5,9,12-triazaoctadec-15-en-18-oate (<b>16k</b>)</h3><div class="NLM_p last">80% yield from <b>7b</b>, after deprotection and coupling to <b>14k</b>, according to the general procedure, part A. <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.06 (br s, 1H), 7.93 (d, <i>J</i> = 8.3 Hz, 2H), 7.67 (d, <i>J</i> = 7.83 Hz, 1H), 7.39–7.35 (m, 4H), 7.22 (s, 1H), 7.19–7.14 (m, 3H), 7.12–7.08 (m, 1H), 7.00–7.05 (m, 2H), 7.05–7.03 (m, 1H), 6.24 (dd, <i>J</i> = 3.4, 1.5 Hz, 1H), 6.17 (t, <i>J</i> = 3.4 Hz, 1H), 6.13 (br d, <i>J</i> = 8.8 Hz, 1H), 5.79 (dd, <i>J</i> = 8.8, 5.9 Hz, 1H), 5.37–5.31 (m, 2H), 4.83 (q, <i>J</i> = 7.5 Hz, 1H), 4.07–3.98 (m, 1H), 3.60 (s, 3H), 3.28 (dd, <i>J</i> = 14.9, 6.4 Hz, 1H), 3.11 (dd, <i>J</i> = 14.9, 7.6 Hz, 1H), 2.89 (br d, <i>J</i> = 4.4 Hz, 2H), 2.82 (s, 1H), 2.67–2.59 (m, 3H), 2.37 (dd, <i>J</i> = 6.3, 3.3 Hz, 2H), 2.35 (s, 3H), 1.33 (s, 9H), 0.85 (d, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (126 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.5, 171.7, 171.5, 156.0, 146.0, 138.8, 137.7, 137.3, 136.4, 136.3, 132.2, 132.0 (2C), 130.9 (2C), 129.0 (2C), 128.8, 128.8, 128.1 (2C), 124.4, 124.0, 123.7, 122.3, 119.7, 119.6, 114.9, 113.1, 112.3, 111.6, 78.8, 54.6, 51.9, 51.9, 50.8, 42.7, 40.5, 40.2, 38.2, 29.0, 28.7, 21.6, 15.5; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>45</sub>H<sub>52</sub>ClN<sub>5</sub>O<sub>8</sub>SNa [M + Na]<sup>+</sup>, 880.3123; found, 880.3133.</div></div><div id="sec4_8_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (3<i>S</i>,14<i>S</i>,15<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-14-methyl-15-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,9-triazacyclopentadec-12-ene-2,5,10-trione (<b>17</b>)</h3><div class="NLM_p last">29% yield from <b>16a</b> according to the general procedure, part B. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.02 (br s, 1H), 7.92 (d, <i>J</i> = 8.2 Hz, 2H), 7.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.40 (d, <i>J</i> = 8.2 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 1H), 7.30 (q, <i>J</i> = 1.5 Hz, 1H), 7.12 (s, 1H), 7.11–6.98 (m, 5H), 6.62 (q, <i>J</i> = 1.5 Hz, 1H), 6.28 (t, <i>J</i> = 3.3 Hz, 1H), 5.96 (dd, <i>J</i> = 9.0, 4.3 Hz, 1H), 5.48–5.30 (m, 2H), 4.63 (ddd, <i>J</i> = 10.9, 7.7, 3.2 Hz, 1H), 3.48 (dd, <i>J</i> = 14.9, 3.1 Hz, 1H), 3.44–3.32 (m, 1H), 2.97–2.87 (m, 1H), 2.83 (br d, <i>J</i> = 6.6 Hz, 2H), 2.75–2.62 (m, 1H), 2.72 (br d, <i>J</i> = 8.6 Hz, 1H), 2.38 (s, 3H), 2.34–2.23 (m, 1H), 2.13–2.06 (m, 1H), 1.87 (q, <i>J</i> = 12.5 Hz, 1H), 1.50–1.38 (m, 1H), 0.90 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 174.4, 171.8, 171.3, 146.2, 137.7, 137.3, 136.2, 134.2, 131.0 (2 C), 128.7, 128.2 (2 C), 127.1, 124.1, 123.5, 122.2, 119.7, 119.2, 116.2, 113.1, 112.3, 112.3, 56.0, 49.8, 41.5, 41.2, 41.1, 35.1, 28.7, 23.9, 21.6, 16.7; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>37</sub>N<sub>5</sub>O<sub>5</sub>SNa [M + Na]<sup>+</sup>, 638.2413; found, 638.2415.</div></div><div id="sec4_8_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (3<i>S</i>,15<i>S</i>,16<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-15-methyl-16-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,10-triazacyclohexadec-13-ene-2,5,11-trione (<b>18</b>)</h3><div class="NLM_p last">22% yield from <b>16b</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +49 (<i>c</i> 0.11, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.99 (br s, 1H), 8.02 (d, <i>J</i> = 8.2 Hz, 2H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.40 (d, <i>J</i> = 8.2 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.25 (br d, <i>J</i> = 1.6 Hz, 1H), 7.21 (br d, <i>J</i> = 7.8 Hz, 1H), 7.14–7.05 (m, 2H), 7.03 (s, 1H), 6.97 (t, <i>J</i> = 7.4 Hz, 1H), 6.44 (br s, 1H), 6.25 (br s, 1H), 6.04 (dd, <i>J</i> = 8.4, 3.3 Hz, 1H), 5.51 (ddd, <i>J</i> = 15.6, 7.8, 5.5 Hz, 1H), 5.41 (dd, <i>J</i> = 15.6, 7.8 Hz, 1H), 4.71–4.64 (m, 1H), 3.69–3.54 (m, 1H), 3.33–3.23 (m, 2H), 3.03–2.81 (m, 3H), 1.98–1.84 (m, 2H), 1.68–1.49 (m, 4H), 2.34 (s, 3H), 0.94 (d,, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 174.3, 171.9, 171.6, 146.1, 137.6, 137.4, 135.7, 135.1, 130.9 (2 C), 128.7, 128.4 (2 C), 126.7, 124.1, 123.8, 122.2, 119.6, 119.2, 116.0, 113.3, 112.3, 111.9, 55.8, 49.7, 41.9, 41.2, 37.1, 34.3, 30.7, 28.7, 28.6, 21.6, 16.7; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>39</sub>N<sub>5</sub>O<sub>5</sub>SNa [M + Na]<sup>+</sup>, 652.2570; found, 652.2565.</div></div><div id="sec4_8_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (3<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>19</b>)</h3><div class="NLM_p last">41% yield from <b>16c</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +5 (<i>c</i> 0.193, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.02 (br s, 1H), 8.03 (d, <i>J</i> = 8.2 Hz, 2H), 7.59–7.49 (m, 2H), 7.40 (d, <i>J</i> = 8.2 Hz, 2H), 7.35 (d, <i>J</i> = 8.2 Hz, 1H), 7.23 (d, <i>J</i> = 2.0 Hz, 1H), 7.08 (t, <i>J</i> = 7.1 Hz, 1H), 7.02 (s, 1H), 6.98 (t, <i>J</i> = 7.5 Hz, 1H), 6.89 (br d, <i>J</i> = 7.4 Hz, 1H), 6.26 (s, 2H), 5.86 (dd, <i>J</i> = 7.4, 3.9 Hz, 1H), 5.71–5.56 (m, 2H), 4.69 (t, <i>J</i> = 8.1 Hz, 1H), 3.51–3.40 (m, 1H), 3.23 (dd, <i>J</i> = 14.9, 3.9 Hz, 1H), 3.19–3.10 (m, 1H), 3.08–2.96 (m, 2H), 2.94–2.84 (m, 1H), 2.83–2.68 (m, 2H), 2.54 (tt, <i>J</i> = 11.4, 4.1 Hz, 1H), 2.36 (s, 3H), 0.90 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.0, 171.8, 171.5, 146.1, 137.6, 137.4, 135.8, 135.0, 130.9, 128.6, 128.2, 127.7, 124.1, 123.8, 122.2, 119.6, 119.1, 115.1, 113.4, 112.3, 111.6, 56.3, 50.2, 42.3, 41.2, 36.0, 35.5, 28.6, 21.6, 14.8; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>5</sub>SNa [M + Na]<sup>+</sup>, 624.2257; found, 624.2268.</div></div><div id="sec4_8_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-7-phenethyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>20</b>)</h3><div class="NLM_p last">70% overall yield from <b>7b/14d</b> according to the general procedures, parts A and B; [α]<sup>25</sup><sub>D</sub> +13 (<i>c</i> 0.11, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.00 (br s, 1H), 7.99 (d, <i>J</i> = 8.4 Hz, 2H), 7.60 (br d, <i>J</i> = 8.6 Hz, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 7.40 (br d, <i>J</i> = 8.4 Hz, 2H), 7.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.28–7.31 (m, 1H), 7.14–7.27 (m, 6H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 7.03 (s, 1H), 6.98 (t, <i>J</i> = 8.0 Hz, 1H), 6.86 (br d, <i>J</i> = 8.6 Hz, 1H), 6.31 (br s, 1H), 6.30–6.28 (m, 1H), 5.99 (dd, <i>J</i> = 8.6, 3.5 Hz, 1H), 5.59 (ddd, <i>J</i> = 15.8, 8.2, 5.0 Hz, 1H), 5.51 (dd, <i>J</i> = 15.8, 7.4 Hz, 1H), 4.74 (ddd, <i>J</i> = 10.2, 8.6, 3.0 Hz, 1H), 4.12–4.25 (m, 1H), 3.34 (dd, <i>J</i> = 14.8, 3.5 Hz, 1H), 2.84–2.92 (m, H<sub>2</sub>O overlap), 2.50–2.70 (m, 6H), 2.35 (s, 3H), 2.24 (dd, <i>J</i> = 15.2, 3.5 Hz, 1H), 1.81–1.93 (m, 1H), 1.65–1.77 (m, 1H), 0.85 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.4, 171.2, 170.6, 145.6, 142.4, 137.0, 136.6, 134.1, 133.5, 130.3, 128.7, 128.6, 128.0, 127.7, 126.0, 123.4, 123.2, 121.6, 119.1, 118.5, 55.0, 48.5, 47.3, 42.2, 41.5, 41.0, 36.8, 32.7, 28.0, 21.0, 15.6; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>40</sub>H<sub>44</sub>N<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 706.3063; found, 706.3052.</div></div><div id="sec4_8_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> (3<i>S</i>,7<i>R</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-7-phenethyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>21</b>)</h3><div class="NLM_p last">39% overall yield from <b>7b/14e</b> according to the general procedures; [α]<sup>25</sup><sub>D</sub> +57 (<i>c</i> 0.10, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.00 (br s, 1H), 7.98 (d, <i>J</i> = 8.2 Hz, 2H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.34–7.41 (m, 3H), 7.22–7.32 (m, 4H), 7.14–7.20 (m, 3H), 7.09 (br t, <i>J</i> = 8.0 Hz, 2 H), 6.97–7.04 (m, 2H), 6.87 (br d, <i>J</i> = 8.6 Hz, 1H), 6.50 (br s, 1H), 6.31 (t, <i>J</i> = 3.5 Hz, 1H), 5.99 (dd, <i>J</i> = 8.6, 3.5 Hz, 1H), 5.60 (ddd, <i>J</i> = 14.9, 7.8, 6.3 Hz, 1H), 5.49 (dd, <i>J</i> = 14.9, 7.4 Hz, 1H), 4.56 (ddd, <i>J</i> = 11.0, 8.2, 3.5 Hz, 1H), 4.07–4.18 (m, 1H), 3.23 (dd, <i>J</i> = 14.8, 3.5 Hz, 1H), 3.05 (dd, <i>J</i> = 12.1, 5.9 Hz, 1H), 2.69–2.77 (m, 2H), 2.52–2.68 (m, 4H), 2.31 (s, 3H), 2.14 (dd, <i>J</i> = 14.8, 7.8 Hz, 1H), 1.93–2.02 (m, 1H), 1.72–1.83 (m, 1H), 0.93 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.3, 171.1, 170.9, 145.5, 142.4, 137.0, 136.7, 134.7, 134.2, 130.3, 128.7, 128.6, 128.0, 127.6, 127.1, 126.0, 123.5, 123.1, 121.6, 119.0, 118.5, 115.5, 112.8, 111.7, 111.1, 56.4, 49.1, 46.6, 41.6, 41.2, 40.7, 36.9, 32.8, 28.0, 20.9, 15.5; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>40</sub>H<sub>44</sub>N<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 706.3063; found, 706.3029.</div></div><div id="sec4_8_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-7-(3-(trifluoromethyl)phenyl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>22</b>)</h3><div class="NLM_p last">28% overall yield from <b>7b/14f</b> according to the general procedures, parts A and B. [α]<sup>25</sup><sub>D</sub> +27 (<i>c</i> 0.19, CHCl<sub>3</sub>); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.01 (br s, 1H), 7.94 (d, <i>J</i> = 8.4 Hz, 2H), 7.82 (d, <i>J</i> = 8.6 Hz, 1H), 7.72 (s, 1H), 7.67 (d, <i>J</i> = 7.4 Hz, 1H), 7.61–7.54 (m, 3H), 7.49 (d, <i>J</i> = 7.0 Hz, 1H), 7.39 (d, <i>J</i> = 8.4 Hz, 2H), 7.36 (d, <i>J</i> = 7.8 Hz, 1H), 7.33 (dd, <i>J</i> = 3.1, 1.6 Hz, 1H), 7.13–6.99 (m, 4H), 6.54 (dd, <i>J</i> = 3.1, 1.6 Hz, 1H), 6.34 (t, <i>J</i> = 3.1 Hz, 1H), 6.03 (dd, <i>J</i> = 8.6, 4.3 Hz, 1H), 5.70–5.47 (m, 3H), 4.45 (ddd, <i>J</i> = 10.9, 7.4, 5.3 Hz, 1H), 3.26 (dd, <i>J</i> = 14.9, 3.5 Hz, 1H), 3.10 (dd, <i>J</i> = 11.9, 6.0 Hz, 1H), 2.99 (dd, <i>J</i> = 14.9, 10.9 Hz, 1H), 2.77–2.70 (m, 2H), 2.65–2.60 (m, 1H), 2.52 (dd, <i>J</i> = 14.5, 10.9, 1H), 2.29 (s, 3H), 0.95 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.1, 171.0, 170.6, 145.6, 144.4, 137.1, 136.7, 135.3, 133.7, 130.7, 130.3, 130.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 70.0 Hz), 129.7, 127.9, 127.6, 126.9 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 262.3 Hz), 126.5, 124.1 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.9 Hz), 123.6, 123.4 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.9 Hz), 123.1, 121.7, 119.1, 118.5, 115.8, 112.8, 111.8, 111.2, 57.5, 50.1, 49.2, 43.4, 41.4, 41.2, 27.8, 20.9, 16.2; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>39</sub>F<sub>3</sub>N<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 746.2624; found, 746.2642.</div></div><div id="sec4_8_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-7-(4-(trifluoromethyl)phenyl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>23</b>)</h3><div class="NLM_p last">46% yield from <b>16g</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +38 (<i>c</i> 0.36, CHCl<sub>3</sub>); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.99 (br s, 1H), 7.94 (d, <i>J</i> = 8.4 Hz, 2H), 7.75 (d, <i>J</i> = 8.5 Hz, 1H), 7.64 (d, <i>J</i> = 7.6 Hz, 2H), 7.60–7.56 (m, 3H), 7.49 (br d, <i>J</i> = 7.3 Hz, 1H), 7.39 (d, <i>J</i> = 7.9 Hz, 2H), 7.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.32 (dd, <i>J</i> = 3.4, 1.7 Hz, 1H), 7.10 (ddd, <i>J</i> = 8.0, 6.9, 1.2 Hz, 1H), 7.07 (s, 1H), 7.03 (dt, <i>J</i> = 1.1, 7.4 Hz, 2H), 6.53 (dd, <i>J</i> = 3.4, 1.5 Hz, 1H), 6.34 (t, <i>J</i> = 3.4 Hz, 1H), 6.02 (dd, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.64 (ddd, <i>J</i> = 15.4, 8.8, 6.0 Hz, 1H), 5.48–5.57 (m, 2H), 4.44 (ddd, <i>J</i> = 11.0, 7.6, 3.7 Hz, 1H), 3.25 (dd, <i>J</i> = 15.0, 3.6 Hz, 1H), 3.11 (dd, <i>J</i> = 11.9, 6.1 Hz, 1H), 2.96 (dd, <i>J</i> = 14.5, 10.7 Hz, 1H), 2.81–2.79 (m, 1H), 2.77–2.71 (m, 5H), 2.66–2.58 (m, 1H), 2.51 (dd, <i>J</i> = 14.5, 10.8 Hz, 1H), 0.95 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (126 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.7, 171.6, 171.2, 148.1, 146.2, 137.7, 137.3, 135.8, 134.3, 130.9 (2 C), 129.5 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.2 Hz, C), 128.5, 128.2 (2 C), 128.0 (2 C), 127.1, 126.2 (q, <sub>3<i>J</i>C–F</sub> = 3.6 Hz, 2 C), 127.5 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 259 Hz, CF<sub>3</sub>), 124.2, 123.7, 122.4, 119.7, 119.1, 116.4, 113.3, 112.4, 111.9, 58.1, 50.6, 49.8, 43.8, 41.9, 41.7, 28.3, 21.5, 16.7; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>38</sub>F<sub>3</sub>N<sub>5</sub>O<sub>5</sub>SNa [M + Na]<sup>+</sup>, 768.2443; found, 768.2479.</div></div><div id="sec4_8_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-7-(<i>p</i>-tolyl)-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>24</b>)</h3><div class="NLM_p last">26% overall yield from <b>7b/14h</b> according to the general procedures, parts A and B. [α]<sup>25</sup><sub>D</sub> +40 (<i>c</i> 0.166, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.01 (br s, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 2H), 7.70 (d, <i>J</i> = 9.0 Hz, 1H), 7.59 (d, <i>J</i> = 8.2 Hz, 1H), 7.42 (d, <i>J</i> = 7.4 Hz, 1H), 7.40–7.34 (m, 3H), 7.33 (d, <i>J</i> = 3.5, 1.6 Hz, 1H), 7.22 (d, <i>J</i> = 8.2 Hz, 2H), 7.12–6.97 (m, 6H), 6.63 (dd, <i>J</i> = 3.1, 1.6 Hz, 1H), 6.34 (dd, <i>J</i> = 3.1, 3.5 Hz, 1H), 6.07 (dd, <i>J</i> = 9.0, 4.3 Hz, 1H), 5.60 (ddd, <i>J</i> = 15.4, 9.4, 5.3 Hz, 1H), 5.47 (dd, <i>J</i> = 15.4, 8.6 Hz, 1H), 5.49–5.40 (m, 1H), 4.45 (ddd, <i>J</i> = 10.8, 7.5, 3.4 Hz, 1H), 3.26 (dd, <i>J</i> = 14.7, 3.4 Hz, 1H), 3.06 (dd, <i>J</i> = 11.7, 5.3 Hz, 1H), 2.92 (dd, <i>J</i> = 14.7, 10.8 Hz, 1H), 2.69 (dd, <i>J</i> = 11.7, 9.4 Hz, 1H), 2.63 (dd, <i>J</i> = 14.5, 4.5 Hz, 1H), 2.60–2.50 (m, 1H), 2.41 (dd, <i>J</i> = 14.3, 11.5 Hz, 1H), 2.28 (s, 3H), 2.26 (s, 3H), 0.96 (d, <i>J</i> = 6.6 Hz, 3H); <sup>1</sup>H NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.0, 170.5, 170.2, 145.2, 139.5, 136.7, 136.3 (2C), 134.7, 133.1, 130.0, 128.9, 127.6, 127.2, 126.4, 126.1, 123.3, 122.7, 121.3, 118.7, 118.1, 115.7, 112.4, 111.4, 110.0, 57.1, 49.5, 48.6, 43.8, 41.1, 40.9, 27.5, 20.6, 20.1, 16.1; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>42</sub>N<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 692.2907; found, 692.2915.</div></div><div id="sec4_8_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-7-(4-fluorophenyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>25</b>)</h3><div class="NLM_p last">42% yield from <b>16i</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +17 (<i>c</i> 0.146, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.01 (br s, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 2H), 7.71 (br d, <i>J</i> = 8.6 Hz, 1H), 7.58 (d, <i>J</i> = 7.8 Hz, 1H), 7.44 (br d, <i>J</i> = 7.0 Hz, 1H), 7.42–7.34 (m, 5H), 7.33 (dd, <i>J</i> = 3.1, 1.6 Hz, 1H), 7.13–6.97 (m, 6H), 6.58 (d, <i>J</i> = 1.6 Hz, 1H), 6.34 (t, <i>J</i> = 3.3 Hz, 1H), 6.04 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 5.66–5.56 (m, 1H), 5.53–5.42 (m, 2H), 4.44 (s, 1H), 3.26 (dd, <i>J</i> = 14.5, 3.1 Hz, 1H), 3.06 (dd, <i>J</i> = 11.7, 5.5 Hz, 1H), 2.96–2.88 (m, 1H), 2.74–2.64 (m, 2H), 2.62–2.55 (m, 1H), 2.45 (dd, <i>J</i> = 14.1, 11.3 Hz, 1H), 2.29 (s, 3H), 0.95 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.8, 171.5, 171.1, 162.7 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 258.3 Hz, 2C), 146.1, 139.7, 137.7, 137.3, 135.7, 134.2, 130.9 (2C), 129.1, 129.2, 128.5, 128.2 (2C), 127.2, 124.2, 123.7, 122.3, 119.7, 119.1, 116.6, 115.8 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.7 Hz, 2C), 113.3, 112.4, 112.3, 111.9, 58.0, 50.2, 49.7, 44.3, 42.0, 41.8, 30.9, 30.7, 28.4, 21.5, 16.9; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>38</sub>H<sub>38</sub>FN<sub>5</sub>O<sub>5</sub>SNa [M + Na]<sup>+</sup>, 718.2475; found, 718.2488.</div></div><div id="sec4_8_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-7-(4-chlorophenyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>26</b>)</h3><div class="NLM_p last">36% overall yield from <b>7b/14j</b>, according to the general procedures, parts A and B. [α]<sup>25</sup><sub>D</sub> +33 (<i>c</i> 0.14, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.03 (br s, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 2H), 7.75 (d, <i>J</i> = 8.6 Hz, 1H), 7.58 (d, <i>J</i> = 7.8 Hz, 1H), 7.49 (d, <i>J</i> = 7.4 Hz, 1H), 7.43–7.30 (m, 8H), 7.14–7.00 (m, 4H), 6.56 (br s, 1H), 6.33 (t, <i>J</i> = 3.1 Hz, 1H), 6.03 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 5.62 (ddd, <i>J</i> = 14.9, 8.2, 6.1 Hz, 1H), 5.50 (dd, <i>J</i> = 14.9, 8.6 Hz, 1H), 5.50–5.40 (m, 1H), 4.44 (ddd, <i>J</i> = 10.8, 7.8, 2.9 Hz, 1H), 3.25 (d, <i>J</i> = 14.9, 2.9 Hz, 1H), 3.08 (dd, <i>J</i> = 11.9, 6.1 Hz, 1H), 2.94 (dd, <i>J</i> = 14.9, 10.8 Hz, 1H), 2.71 (dd, <i>J</i> = 11.9, 8.2 Hz, 1H), 2.68 (dd, <i>J</i> = 14.1, 4.5 Hz, 1H), 2.63–2.56 (m, 1H), 2.46 (dd, <i>J</i> = 14.1, 11.3 Hz, 1H), 2.29 (s, 3H), 0.95 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, acetone-<i>d</i><sub>6</sub>) δ 170.8, 170.6, 170.2, 145.2, 141.5, 136.7, 136.3, 134.8, 133.2, 132.1, 130.0, 128.3, 128.1, 127.5, 127.2, 126.2, 123.3, 122.7, 121.4, 118.7, 118.1, 115.5, 112.4, 111.4, 110.9, 57.1, 49.3, 48.8, 43.1, 41.0, 40.8, 27.4, 20.6, 15.9; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>38</sub>H<sub>39</sub>ClN<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 712.2360; found, 712.2380.</div></div><div id="sec4_8_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (3<i>S</i>,7<i>R</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-7-(4-chlorobenzyl)-13-methyl-14-(1-tosyl-1<i>H</i>-pyrrol-2-yl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>27</b>)</h3><div class="NLM_p last">42% yield from <b>16k</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +57 (<i>c</i> 0.22, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.98 (br s, 1H), 7.98 (d, <i>J</i> = 8.3 Hz, 2H), 7.53 (d, <i>J</i> = 7.8 Hz, 1H), 7.41–7.36 (m, 3H), 7.33 (br d, <i>J</i> = 7.8 Hz, 1H), 7.28–7.24 (m, 3H), 7.20 (d, <i>J</i> = 8.8 Hz, 2H), 7.09 (t, <i>J</i> = 7.6 Hz, 2H), 7.00–6.95 (m, 2H), 6.87 (br d, <i>J</i> = 7.8 Hz, 1H), 6.42 (dd, <i>J</i> = 3.2, 1.7 Hz, 1H), 6.29 (t, <i>J</i> = 3.2 Hz, 1H), 5.96 (dd, <i>J</i> = 8.6, 4.2 Hz, 1H), 5.59 (ddd, <i>J</i> = 15.2, 8.8, 6.7 Hz, 1H), 5.49 (dd, <i>J</i> = 15.2, 8.3 Hz, 1H), 4.59 (ddd, <i>J</i> = 10.4, 8.0, 3.7 Hz, 1H), 4.29 (dddd, <i>J</i> = 15.2, 8.5, 7.3, 4.9 Hz, 1H), 3.23 (dd, <i>J</i> = 14.9, 23.7 Hz, 1H), 3.01–2.87 (m, 5H), 2.64 (dd, <i>J</i> = 12.2, 8.5 Hz, 1H), 2.51 (dd, <i>J</i> = 15.0, 4.9 Hz, 1H), 2.32 (s, 3H), 2.12 (dd, <i>J</i> = 15.0, 7.3 Hz, 1H), 0.91 (d, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.3, 172.0, 172.0, 146.6, 139.4, 138.2, 137.9, 135.9, 135.4, 133.0, 132.5, 131.4, 129.7, 129.7, 129.6, 129.1, 12.7, 128.1, 124.6, 124.3, 122.8, 120.2, 119.6, 116.5, 113.9, 112.8, 112.8, 112.3, 57.5, 50.3, 49.4, 42.6, 42.3, 40.9, 40.5, 29.1, 22.0, 16.6; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>39</sub>H<sub>43</sub>ClN<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>, 726.2517; found, 726.2508.</div></div><div id="sec4_8_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-13-methyl-14-phenyl-7-(4-(trifluoromethyl)phenyl)-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>28</b>)</h3><div class="NLM_p last">19% overall yield from <b>7a/14g</b>, according to the general procedures parts A and B. [α]<sup>25</sup><sub>D</sub> +10 (<i>c</i> 0.35, CHCl<sub>3</sub>); <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.99 (br s, 1H), 7.92 (s, 1H), 7.80 (br d, <i>J</i> = 8.2 Hz, 1H), 7.67–7.55 (m, 5H), 7.52 (d, <i>J</i> = 7.8 Hz, 1H), 7.34–7.27 (m, 5H), 7.27–7.20 (m, 1H), 7.07 (s, 1H), 7.04 (t, <i>J</i> = 7.4 Hz, 1H), 6.95 (t, <i>J</i> = 7.4 Hz, 1H), 5.75 (ddd, <i>J</i> = 15.3, 9.1, 6.5 Hz, 1H), 5.59–5.49 (m, 1H), 5.29 (dd, <i>J</i> = 15.3, 9.4 Hz, 1H), 4.95 (dd, <i>J</i> = 8.4, 4.1 Hz, 1H), 4.53–4.44 (m, 1H), 3.33 (dd, <i>J</i> = 15.0, 4.4 Hz, 1H), 3.14 (dd, <i>J</i> = 15.0, 10.6 Hz, 1H), 3.10–3.05 (m, 1H), 2.80–2.75 (m, 1H), 2.68 (dd, <i>J</i> = 11.9, 9.6 Hz, 1H), 2.62–2.49 (m, 2H), 0.85 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.7, 171.5, 171.1, 148.2, 139.8, 137.7, 135.7, 129.4 (2 C), 129.5 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.3 Hz, 1 C), 128.7 (2 C), 128.6, 128.0 (2 C), 127.9, 127.2, 126.2 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.8 Hz, 2C), 124.3, 122.2, 125.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 271.7 Hz, 1 C), 119.7, 119.2, 112.3, 112.3, 112.0, 58.5, 57.8, 50.6, 43.7, 42.4, 41.6, 27.6, 17.6; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>44</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 603.2583; found, 603.2582.</div></div><div id="sec4_8_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-7-(4-fluorophenyl)-13-methyl-14-phenyl-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>29</b>)</h3><div class="NLM_p last">33% yield from <b>15i</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +15 (<i>c</i> 0.133, CHCl<sub>3</sub>); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.97 (br s, 1H), 7.69 (br d, <i>J</i> = 8.0 Hz, 1H), 7.55–7.54 (m, 1H), 7.56 (br d, <i>J</i> = 7.8 Hz, 1H), 7.43 (d, <i>J</i> = 5.3 Hz, 1H), 7.41 (d, <i>J</i> = 5.3 Hz, 1H), 7.36–7.31 (m, 6H), 7.31–7.25 (m, 2H), 7.11–7.09 (m, 2H), 7.09 (d, <i>J</i> = 5.3 Hz, 1H), 7.07 (s, 1H), 7.06–7.04 (m, 1H), 6.99 (t, <i>J</i> = 8.0 Hz, 1H), 5.76 (ddd, <i>J</i> = 15.2, 9.8, 6.1 Hz, 1H), 5.49 (dddd, <i>J</i> = 12.2, 8.1, 3.7, 0.4 Hz, 1H), 5.30 (dd, <i>J</i> = 15.2, 9.8 Hz, 1H), 4.98 (dd, <i>J</i> = 8.3, 3.9 Hz, 1H), 4.52 (ddd, <i>J</i> = 10.2, 7.5, 3.9 Hz, 1H), 3.36 (dd, <i>J</i> = 14.9, 3.9 Hz, 1H), 3.15 (dd, <i>J</i> = 14.9, 10.2 Hz, 1H), 3.08 (dd, <i>J</i> = 12.2, 5.9 Hz, 1H), 2.76 (d, <i>J</i> = 4.4 Hz, 1H), 2.74 (dd, <i>J</i> = 14.9, 3.9 Hz, 1H), 2.77–2.72 (m, 1H), 2.74 (dd, <i>J</i> = 12.2, 5.9 Hz, 1H), 2.69 (dd, <i>J</i> = 12.2, 9.5 Hz, 1H), 2.58 (dd, <i>J</i> = 14.9, 3.7 Hz, 2H), 2.56–2.54 (m, 1H), 1.50–1.39 (m, 3H), 0.90 (d, <i>J</i> = 6.9 Hz, 3H); <sup>13</sup>C NMR (126 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.9, 171.3, 171.0, 162.7 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 243.4 Hz, 1 C), 139.8, 137.7, 135.6, 129.5 (2 C), 129.3, 129.2, 128.6 (2 C), 127.9 (2 C), 127.3, 124.3, 124.2, 122.2, 119.7, 119.2, 115.9 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.1 Hz, 1C), 112.3, 112.0, 58.5, 57.7, 50.2, 44.3, 42.5, 41.7, 27.7, 17.8; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>34</sub>FN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 553.2615; found, 553.2634.</div></div><div id="sec4_8_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> (3<i>S</i>,7<i>S</i>,13<i>S</i>,14<i>R</i>,<i>E</i>)-3-((1<i>H</i>-Indol-3-yl)methyl)-7-(4-chlorophenyl)-13-methyl-14-phenyl-1,4,8-triazacyclotetradec-11-ene-2,5,9-trione (<b>30</b>)</h3><div class="NLM_p last">13% yield from <b>15j</b> according to the general procedure, part B. [α]<sup>25</sup><sub>D</sub> +16 (<i>c</i> 0.146, CHCl<sub>3</sub>); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.97 (br s, 1H), 7.69 (br d, <i>J</i> = 8.8 Hz, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 2H), 7.41 (d, <i>J</i> = 8.3 Hz, 2H), 7.36–7.30 (m, 7H), 7.32–7.25 (m, 3H), 7.09 (d, <i>J</i> = 2.0 Hz, 1H), 7.07 (t, <i>J</i> = 7.8 Hz, 1H), 6.99 (t, <i>J</i> = 6.9 Hz, 1H), 5.76 (ddd, <i>J</i> = 15.3, 9.3, 5.9 Hz, 1H), 5.48 (ddd, <i>J</i> = 10.8, 8.8, 3.9 Hz, 1H), 5.31 (dd, <i>J</i> = 15.3, 9.6 Hz, 1H), 4.98 (dd, <i>J</i> = 8.3, 3.9 Hz, 1H), 4.52 (ddd, <i>J</i> = 10.2, 7.5, 3.9 Hz, 1H), 3.36 (dd, <i>J</i> = 14.9, 3.7 Hz, 1H), 3.15 (dd, <i>J</i> = 14.9, 10.0 Hz, 1H), 3.09 (dd, <i>J</i> = 11.9, 5.9 Hz, 1H), 2.76 (dd, <i>J</i> = 14.7, 3.9 Hz, 1H), 2.70 (dd, <i>J</i> = 11.9, 9.3 Hz, 1H), 2.57 (dd, <i>J</i> = 14.7, 10.2 Hz, 1H), 2.60–2.52 (m, 1H), 0.90 (d, <i>J</i> = 6.9 Hz, 3H); <sup>13</sup>C NMR (126 MHz, acetone-<i>d</i><sub>6</sub>) δ 170.8, 170.3, 170.0, 141.6, 138.9, 136.7, 134.6, 132.1, 128.5, 128.3, 128.1, 127.7, 127.6, 126.9, 126.2, 123.3, 121.3, 118.7, 118.2, 111.3, 111.0, 57.5, 56.7, 49.3, 43.1, 41.5, 40.7, 26.7, 16.8; HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 591.2139; found, 591.2147.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01529" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16763" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16763" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01529" class="ext-link">10.1021/acs.jmedchem.8b01529</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figure S1 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinate files used to generate <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinate files used to generate <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_004.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf">jm8b01529_si_001.pdf (159.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_002.csv">jm8b01529_si_002.csv (2.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_003.pdb">jm8b01529_si_003.pdb (295.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_004.pdb">jm8b01529_si_004.pdb (254.79 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates have been deposited with the PDB with the following accession numbers: APE1 bound to DMSO (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>), DMSO/Mg<sup>2+</sup> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>), DMSO/Tris (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>), and GLC (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01529" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12006" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12006" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark R. Kelley</span> - <span class="hlFld-Affiliation affiliation">Department of
Pediatrics and Herman
B. Wells Center for Pediatric Research, and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fa97919f96969f83ba938fd49f9e8f"><span class="__cf_email__" data-cfemail="5c313739303039251c352972393829">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Millie M. Georgiadis</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Molecular Biology, 
    , and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, Purdue School of Science, Indiana University-Purdue University Indianapolis, Indianapolis, Indiana 46202, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2976-4576" title="Orcid link">http://orcid.org/0000-0003-2976-4576</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f59892909a87929c94b59c80db909180"><span class="__cf_email__" data-cfemail="19747e7c766b7e707859706c377c7d6c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Trilles</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dmitri Beglov</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiujia Chen</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Molecular Biology, and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongzhen He</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Molecular Biology, and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Randall Wireman</span> - <span class="hlFld-Affiliation affiliation">Department of
Pediatrics and Herman
B. Wells Center for Pediatric Research, and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">April Reed</span> - <span class="hlFld-Affiliation affiliation">Department of
Pediatrics and Herman
B. Wells Center for Pediatric Research, and , Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Spandan Chennamadhavuni</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span>Present Address:
                        S.C.: Department of Chemistry, Emory University, Atlanta, GA 30322</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James S. Panek</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lauren E. Brown</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9489-484X" title="Orcid link">http://orcid.org/0000-0001-9489-484X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandor Vajda</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John A. Porco</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2991-5680" title="Orcid link">http://orcid.org/0000-0002-2991-5680</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15855" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15855" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from the National Institutes of Health (Grant R01CA205166 to M.R.K. and M.M.G. and Grant R01CA167291 to M.R.K.) and by the Earl and Betty Herr Professor in Pediatric Oncology Research, Jeff Gordon Children’s Foundation, and the Riley Children’s Foundation (M.R.K.). Work at the BU-CMD (J.A.P., L.E.B., R.T.) is supported by the National Institutes of Health, Grant R24 GM111625. D.B. and S.V. were supported by the National Institutes of Health, Grant R35 GM118078.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate, hexafluorophosphate azabenzotriazole tetramethyluronium</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messmann, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehrenbacher, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Exploiting the Ref-1-APE1 node in cancer signaling and other diseases: from bench to clinic</span>. <i>NPJ Precis. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41698-017-0023-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fs41698-017-0023-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=29872691" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=1-19&author=F.+Shahauthor=D.+Logsdonauthor=R.+A.+Messmannauthor=J.+C.+Fehrenbacherauthor=M.+L.+Fishelauthor=M.+R.+Kelley&title=Exploiting+the+Ref-1-APE1+node+in+cancer+signaling+and+other+diseases%3A+from+bench+to+clinic&doi=10.1038%2Fs41698-017-0023-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41698-017-0023-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41698-017-0023-0%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DF.%26aulast%3DLogsdon%26aufirst%3DD.%26aulast%3DMessmann%26aufirst%3DR.%2BA.%26aulast%3DFehrenbacher%26aufirst%3DJ.%2BC.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DExploiting%2520the%2520Ref-1-APE1%2520node%2520in%2520cancer%2520signaling%2520and%2520other%2520diseases%253A%2520from%2520bench%2520to%2520clinic%26jtitle%3DNPJ%2520Precis.%2520Oncol.%26date%3D2017%26volume%3D1%26spage%3D1%26epage%3D19%26doi%3D10.1038%2Fs41698-017-0023-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, R.</span></span> <span> </span><span class="NLM_article-title">APE1 overexpression is associated with poor survival in patients with solid tumors: a meta-analysis</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">59720</span>– <span class="NLM_lpage">59728</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.18632%2Foncotarget.19814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=59720-59728&author=C.+L.+Yuanauthor=F.+Heauthor=J.+Z.+Yeauthor=H.+N.+Wuauthor=J.+Y.+Zhangauthor=Z.+H.+Liuauthor=Y.+Q.+Liauthor=X.+L.+Luoauthor=Y.+Linauthor=R.+Liang&title=APE1+overexpression+is+associated+with+poor+survival+in+patients+with+solid+tumors%3A+a+meta-analysis&doi=10.18632%2Foncotarget.19814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19814%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DC.%2BL.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DYe%26aufirst%3DJ.%2BZ.%26aulast%3DWu%26aufirst%3DH.%2BN.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DY.%2BQ.%26aulast%3DLuo%26aufirst%3DX.%2BL.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DR.%26atitle%3DAPE1%2520overexpression%2520is%2520associated%2520with%2520poor%2520survival%2520in%2520patients%2520with%2520solid%2520tumors%253A%2520a%2520meta-analysis%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D59720%26epage%3D59728%26doi%3D10.18632%2Foncotarget.19814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juhnke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heumann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirico, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoflmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hube-Magg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moller-Koop, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thederan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlomm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luebke, A. M.</span></span> <span> </span><span class="NLM_article-title">Apurinic/apyrimidinic endonuclease 1 (APE1/Ref-1) overexpression is an independent prognostic marker in prostate cancer without TMPRSS2:ERG fusion</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2135</span>– <span class="NLM_lpage">2145</span>, <span class="refDoi"> DOI: 10.1002/mc.22670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2Fmc.22670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=28467610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=2135-2145&author=M.+Juhnkeauthor=A.+Heumannauthor=V.+Chiricoauthor=D.+Hoflmayerauthor=A.+Menzauthor=A.+Hinschauthor=C.+Hube-Maggauthor=M.+Kluthauthor=D.+S.+Langauthor=C.+Moller-Koopauthor=G.+Sauterauthor=R.+Simonauthor=B.+Beyerauthor=R.+Pompeauthor=I.+Thederanauthor=T.+Schlommauthor=A.+M.+Luebke&title=Apurinic%2Fapyrimidinic+endonuclease+1+%28APE1%2FRef-1%29+overexpression+is+an+independent+prognostic+marker+in+prostate+cancer+without+TMPRSS2%3AERG+fusion&doi=10.1002%2Fmc.22670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Apurinic/apyrimidinic endonuclease 1 (APE1/Ref-1) overexpression is an independent prognostic marker in prostate cancer without TMPRSS2:ERG fusion</span></div><div class="casAuthors">Juhnke, Manuela; Heumann, Asmus; Chirico, Viktoria; Hoeflmayer, Doris; Menz, Anne; Hinsch, Andrea; Hube-Magg, Claudia; Kluth, Martina; Lang, Dagmar S.; Moeller-Koop, Christina; Sauter, Guido; Simon, Ronald; Beyer, Burkhard; Pompe, Raisa; Thederan, Imke; Schlomm, Thorsten; Luebke, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2135-2145</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Polymorphisms of the base excision repair gene APE1 may be assocd. with an increased risk for developing prostate cancer.  In other cancer types, altered APE1 protein expression is a candidate prognostic marker.  Using immunohistochem., we thus analyzed APE1 expression in 9763 prostate cancers in a tissue microarray (TMA) with attached clin. and mol. data.  The comparison with normal prostate tissue revealed an upregulation of APE1 in cancer samples.  APE1 immunostaining was considered weak in 20.2%, moderate in 36.7%, and strong in 33.4% of cancers.  Strong APE1 expression was markedly more frequent in prostate cancers harboring the TMPRSS2:ERG fusion (52.9%) than in ERG-neg. cancers (19.1%, P < 0.0001).  Significant assocns. with Gleason grade, tumor stage, tumor grade, and early biochem. recurrence (P < 0.0001 each) were largely limited to ERG-neg. tumors.  Multivariable anal. revealed that the prognostic value of APE1 upregulation in ERG-neg. prostate cancers was independent from established histopathol. and clin. parameters.  In conclusion, the results of our study demonstrate that APE1 overexpression is an independent prognostic marker, but exclusively in ERG-neg. prostate cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp18rsnuvOO_rVg90H21EOLACvtfcHk0ljDIEFAYW94fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKns7Y%253D&md5=be3eb0f6a4a6360f4d6ac90b7b016c65</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fmc.22670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.22670%26sid%3Dliteratum%253Aachs%26aulast%3DJuhnke%26aufirst%3DM.%26aulast%3DHeumann%26aufirst%3DA.%26aulast%3DChirico%26aufirst%3DV.%26aulast%3DHoflmayer%26aufirst%3DD.%26aulast%3DMenz%26aufirst%3DA.%26aulast%3DHinsch%26aufirst%3DA.%26aulast%3DHube-Magg%26aufirst%3DC.%26aulast%3DKluth%26aufirst%3DM.%26aulast%3DLang%26aufirst%3DD.%2BS.%26aulast%3DMoller-Koop%26aufirst%3DC.%26aulast%3DSauter%26aufirst%3DG.%26aulast%3DSimon%26aufirst%3DR.%26aulast%3DBeyer%26aufirst%3DB.%26aulast%3DPompe%26aufirst%3DR.%26aulast%3DThederan%26aufirst%3DI.%26aulast%3DSchlomm%26aufirst%3DT.%26aulast%3DLuebke%26aufirst%3DA.%2BM.%26atitle%3DApurinic%252Fapyrimidinic%2520endonuclease%25201%2520%2528APE1%252FRef-1%2529%2520overexpression%2520is%2520an%2520independent%2520prognostic%2520marker%2520in%2520prostate%2520cancer%2520without%2520TMPRSS2%253AERG%2520fusion%26jtitle%3DMol.%2520Carcinog.%26date%3D2017%26volume%3D56%26spage%3D2135%26epage%3D2145%26doi%3D10.1002%2Fmc.22670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">The expression of APE1 in triple-negative breast cancer and its effect on drug sensitivity of olaparib</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1010428317713390</span>, <span class="refDoi"> DOI: 10.1177/1010428317713390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1177%2F1010428317713390" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=1010428317713390&author=T.+Chenauthor=C.+Liuauthor=H.+Luauthor=M.+Yinauthor=C.+Shaoauthor=X.+Huauthor=J.+Wuauthor=Y.+Wang&title=The+expression+of+APE1+in+triple-negative+breast+cancer+and+its+effect+on+drug+sensitivity+of+olaparib&doi=10.1177%2F1010428317713390"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1010428317713390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1010428317713390%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DThe%2520expression%2520of%2520APE1%2520in%2520triple-negative%2520breast%2520cancer%2520and%2520its%2520effect%2520on%2520drug%2520sensitivity%2520of%2520olaparib%26jtitle%3DTumor%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D1010428317713390%26doi%3D10.1177%2F1010428317713390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The expression profile and prognostic value of APE/Ref-1 and NPM1 in high-grade serous ovarian adenocarcinoma</span>. <i>APMIS</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1111/apm.12733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1111%2Fapm.12733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=28766835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=857-862&author=X.+Fanauthor=L.+Wenauthor=Y.+Liauthor=L.+Louauthor=W.+Liuauthor=J.+Zhang&title=The+expression+profile+and+prognostic+value+of+APE%2FRef-1+and+NPM1+in+high-grade+serous+ovarian+adenocarcinoma&doi=10.1111%2Fapm.12733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The expression profile and prognostic value of APE/Ref-1 and NPM1 in high-grade serous ovarian adenocarcinoma</span></div><div class="casAuthors">Fan, Xiaomei; Wen, Lixuan; Li, Yuehong; Lou, Lei; Liu, Weina; Zhang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">857-862</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">1600-0463</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To analyze the expression trends and clin. significance of Apurinic/Apyrimidinic Endodeoxyribonuclease 1 (APE1/Ref-1) and Nucleophosmin (NPM1) proteins in high-grade serous ovarian adenocarcinoma (HGSC).  The expressions of APE1/Ref-1 and NPM1 proteins in 94 patients with HGSC were detd. using the immunohistochem. (IHC) method, and their relationships with clinicopathol. features were analyzed by the χ2 test or Fisher's exact test.  The follow-up data, Cox proportional hazards univariate and multivariate survival analyses were integrated to evaluate the prognostic factors affecting patients with HGSC.  In the normal fallopian tubes, APE1/Ref-1 and NPM1 protein were mainly distributed in the nuclear.  The HGSC experienced changes in the cellular localization of APE1/Ref-1 and NPM1 protein expressions, which were abnormally expressed in the cytoplasm.  The rates of abnormal cytoplasmic expression of APE1/Ref-1 and NPM1 proteins in 94 patients with HGSC were 69.1% and 73.4%, resp., which were significantly higher than the normal fallopian tube tissues (p < 0.05).  The abnormal cytoplasmic APE1/Ref-1 and NPM1 are significantly correlated with the lymph node metastasis, chemosensitivity, FIGO staging, and prognosis.  The COX multivariate survival anal. showed that the abnormal expression of APE1/Ref-1 protein, FIGO staging, and lymph node metastasis are independent prognostic factors.  Collectively, the abnormal cytoplasmic APE1/Ref-1 and NPM1 proteins are assocd. with the oncogenic progression and chemoresistance of HGSC, and predict a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8XAgdJUSZrVg90H21EOLACvtfcHk0ljQE_Tvw9Dzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtr3E&md5=2656395d77f2ce015e6ab4ca2754de54</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fapm.12733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fapm.12733%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520expression%2520profile%2520and%2520prognostic%2520value%2520of%2520APE%252FRef-1%2520and%2520NPM1%2520in%2520high-grade%2520serous%2520ovarian%2520adenocarcinoma%26jtitle%3DAPMIS%26date%3D2017%26volume%3D125%26spage%3D857%26epage%3D862%26doi%3D10.1111%2Fapm.12733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Human apurinic/apyrimidinic endonuclease 1</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1089/ars.2013.5492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1089%2Fars.2013.5492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=23834463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsFynuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=678-707&author=M.+Liauthor=D.+M.+Wilson&title=Human+apurinic%2Fapyrimidinic+endonuclease+1&doi=10.1089%2Fars.2013.5492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Human Apurinic/Apyrimidinic Endonuclease 1</span></div><div class="casAuthors">Li, Mengxia; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">678-707</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Significance: Human apurinic/apyrimidinic endonuclease 1 (APE1, also known as REF-1) was isolated based on its ability to cleave at AP sites in DNA or activate the DNA binding activity of certain transcription factors.  We review herein topics related to this multi-functional DNA repair and stress-response protein.  Recent Advances: APE1 displays homol. to Escherichia coli exonuclease III and is a member of the divalent metal-dependent α/β fold-contg. phosphoesterase superfamily of enzymes.  APE1 has acquired distinct active site and loop elements that dictate substrate selectivity, and a unique N-terminus which at min. imparts nuclear targeting and interaction specificity.  Addnl. activities ascribed to APE1 include 3'-5' exonuclease, 3'-repair diesterase, nucleotide incision repair, damaged or site-specific RNA cleavage, and multiple transcription regulatory roles.  Crit. Issues: APE1 is essential for mouse embryogenesis and contributes to cell viability in a genetic background-dependent manner.  Haploinsufficient APE1+/- mice exhibit reduced survival, increased cancer formation, and cellular/tissue hyper-sensitivity to oxidative stress, supporting the notion that impaired APE1 function assocs. with disease susceptibility.  Although abnormal APE1 expression/localization has been seen in cancer and neuropathologies, and impaired-function variants have been described, a causal link between an APE1 defect and human disease remains elusive.  Future Directions: Ongoing efforts aim at delineating the biol. role(s) of the different APE1 activities, as well as the regulatory mechanisms for its intra-cellular distribution and participation in diverse mol. pathways.  The detn. of whether APE1 defects contribute to human disease, particularly pathologies that involve oxidative stress, and whether APE1 small-mol. regulators have clin. utility, is central to future investigations.  Antioxid. Redox Signal. 20, 678-707.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAm6RzdhioobVg90H21EOLACvtfcHk0lhcv67BHux9Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsFynuw%253D%253D&md5=ded413cf1790fd14e73629fefce7913a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2013.5492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2013.5492%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DHuman%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2014%26volume%3D20%26spage%3D678%26epage%3D707%26doi%3D10.1089%2Fars.2013.5492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimpton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houlbrook, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammonds, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freemont, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dianov, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">Isolation of a small molecule inhibitor of DNA base excision repair</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">4711</span>– <span class="NLM_lpage">4724</span>, <span class="refDoi"> DOI: 10.1093/nar/gki781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fnar%2Fgki781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=16113242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=4711-4724&author=S.+Madhusudanauthor=F.+Smartauthor=P.+Shrimptonauthor=J.+L.+Parsonsauthor=L.+Gardinerauthor=S.+Houlbrookauthor=D.+C.+Talbotauthor=T.+Hammondsauthor=P.+A.+Freemontauthor=M.+J.+Sternbergauthor=G.+L.+Dianovauthor=I.+D.+Hickson&title=Isolation+of+a+small+molecule+inhibitor+of+DNA+base+excision+repair&doi=10.1093%2Fnar%2Fgki781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a small molecule inhibitor of DNA base excision repair</span></div><div class="casAuthors">Madhusudan, Srinivasan; Smart, Fiona; Shrimpton, Paul; Parsons, Jason L.; Gardiner, Laurence; Houlbrook, Sue; Talbot, Denis C.; Hammonds, Timothy; Freemont, Paul A.; Sternberg, Michael J. E.; Dianov, Grigory L.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4711-4724</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The base excision repair (BER) pathway is essential for the removal of DNA bases damaged by alkylation or oxidn.  A key step in BER is the processing of an apurinic/apyrimidinic (AP) site intermediate by an AP endonuclease.  The major AP endonuclease in human cells (APE1, also termed HAP1 and Ref-1) accounts for >95% of the total AP endonuclease activity, and is essential for the protection of cells against the toxic effects of several classes of DNA damaging agents.  Moreover, APE1 overexpression has been linked to radio- and chemo-resistance in human tumors.  Using a newly developed high-throughput screen, several chem. inhibitors of APE1 have been isolated.  Amongst these, CRT0044876 was identified as a potent and selective APE1 inhibitor.  CRT0044876 inhibits the AP endonuclease, 3'-phosphodiesterase and 3'-phosphatase activities of APE1 at low micromolar concns., and is a specific inhibitor of the exonuclease III family of enzymes to which APE1 belongs.  At non-cytotoxic concns., CRT0044876 potentiates the cytotoxicity of several DNA base-targeting compds.  This enhancement of cytotoxicity is assocd. with an accumulation of unrepaired AP sites.  In silico modeling studies suggest that CRT0044876 binds to the active site of APE1.  These studies provide both a novel reagent for probing APE1 function in human cells, and a rational basis for the development of APE1-targeting drugs for antitumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo35mTihPN15bVg90H21EOLACvtfcHk0lhcv67BHux9Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D&md5=01a434b054d60d670d683dc0fa38e649</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki781%26sid%3Dliteratum%253Aachs%26aulast%3DMadhusudan%26aufirst%3DS.%26aulast%3DSmart%26aufirst%3DF.%26aulast%3DShrimpton%26aufirst%3DP.%26aulast%3DParsons%26aufirst%3DJ.%2BL.%26aulast%3DGardiner%26aufirst%3DL.%26aulast%3DHoulbrook%26aufirst%3DS.%26aulast%3DTalbot%26aufirst%3DD.%2BC.%26aulast%3DHammonds%26aufirst%3DT.%26aulast%3DFreemont%26aufirst%3DP.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DDianov%26aufirst%3DG.%2BL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DIsolation%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520DNA%2520base%2520excision%2520repair%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D4711%26epage%3D4724%26doi%3D10.1093%2Fnar%2Fgki781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiple, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardellina, J. H.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span> <span> </span><span class="NLM_article-title">Potent inhibition of human apurinic/apyrimidinic endonuclease 1 by arylstibonic acids</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1124/mol.107.042622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fmol.107.042622" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2007&pages=669-677&author=L.+A.+Seipleauthor=J.+H.+Cardellinaauthor=R.+Akeeauthor=J.+T.+Stivers&title=Potent+inhibition+of+human+apurinic%2Fapyrimidinic+endonuclease+1+by+arylstibonic+acids&doi=10.1124%2Fmol.107.042622"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.042622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.042622%26sid%3Dliteratum%253Aachs%26aulast%3DSeiple%26aufirst%3DL.%2BA.%26aulast%3DCardellina%26aufirst%3DJ.%2BH.%26aulast%3DAkee%26aufirst%3DR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26atitle%3DPotent%2520inhibition%2520of%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520by%2520arylstibonic%2520acids%26jtitle%3DMol.%2520Pharmacol.%26date%3D2007%26volume%3D73%26spage%3D669%26epage%3D677%26doi%3D10.1124%2Fmol.107.042622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zawahir, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Pharmacophore guided discovery of small-molecule human apurinic/apyrimidinic endonuclease 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1021/jm800739m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800739m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFSjtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=20-32&author=Z.+Zawahirauthor=R.+Dayamauthor=J.+Dengauthor=C.+Pereiraauthor=N.+Neamati&title=Pharmacophore+guided+discovery+of+small-molecule+human+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm800739m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore Guided Discovery of Small-Molecule Human Apurinic/Apyrimidinic Endonuclease 1 Inhibitors</span></div><div class="casAuthors">Zawahir, Zahrah; Dayam, Raveendra; Deng, Jinxia; Pereira, Cherelene; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-32</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human apurinic/apyrimidinic endonuclease 1 (APE1) is an important enzyme in the base excision repair (BER) pathway that is essential for the repair of abasic sites in the genome.  Evidence for APE1 as an attractive therapeutic target in anticancer drug development has been demonstrated by studies that link overexpression of APE1 in many cancers to resistance of tumor cells to radio- and chemotherapy.  APE1 also shows a protective effect in several cancer cell models to a variety of DNA damaging agents.  This study represents the first rational design of selective small-mol. APE1 inhibitors utilizing a three-dimensional interaction-based pharmacophore perception.  All of our most potent mols. show inhibitory activity below 10 μM and are selective for APE1 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHAjB_Hy-QWbVg90H21EOLACvtfcHk0lhnKPVo6MwOxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFSjtbvM&md5=9f5209c466ce2203eafa7195a772bb7f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm800739m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800739m%26sid%3Dliteratum%253Aachs%26aulast%3DZawahir%26aufirst%3DZ.%26aulast%3DDayam%26aufirst%3DR.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DC.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DPharmacophore%2520guided%2520discovery%2520of%2520small-molecule%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D20%26epage%3D32%26doi%3D10.1021%2Fjm800739m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bapat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Novel small molecule inhibitor of Ape1 endonuclease blocks proliferation and reduces viability of glioblastoma cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">988</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1124/jpet.110.169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fjpet.110.169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20504914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=988-998&author=A.+Bapatauthor=L.+S.+Glassauthor=M.+Luoauthor=M.+L.+Fishelauthor=E.+C.+Longauthor=M.+M.+Georgiadisauthor=M.+R.+Kelley&title=Novel+small+molecule+inhibitor+of+Ape1+endonuclease+blocks+proliferation+and+reduces+viability+of+glioblastoma+cells&doi=10.1124%2Fjpet.110.169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span></div><div class="casAuthors">Bapat, Aditi; Glass, LaTeca S.; Luo, Meihua; Fishel, Melissa L.; Long, Eric C.; Georgiadis, Millie M.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic (AP) endonuclease 1 (Ape1) is an essential DNA repair protein that plays a crit. role in repair of AP sites via base excision repair.  Ape1 has received attention as a druggable oncotherapeutic target, esp. for treating intractable cancers such as glioblastoma.  The goal of this study was to identify small-mol. inhibitors of Ape1 AP endonuclease.  For this purpose, a fluorescence-based high-throughput assay was used to screen a library of 60,000 small-mol. compds. for ability to inhibit Ape1 AP endonuclease activity.  Four compds. with IC50 values less than 10 μM were identified, validated, and characterized.  One of the most promising compds., designated Ape1 repair inhibitor 03 (2,4,9-trimethylbenzo[b][1,8]-naphthyridin-5-amine; AR03), inhibited cleavage of AP sites in vivo in SF767 glioblastoma cells and in vitro in whole cell exts. and inhibited purified human Ape1 in vitro.  AR03 has low affinity for double-stranded DNA and weakly inhibits the Escherichia coli endonuclease IV, requiring a 20-fold higher concn. than for inhibition of Ape1.  AR03 also potentiates the cytotoxicity of Me methanesulfonate and temozolomide in SF767 cells.  AR03 is chem. distinct from the previously reported small-mol. inhibitors of Ape1.  AR03 is a novel small-mol. inhibitor of Ape1, which may have potential as an oncotherapeutic drug for treating glioblastoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaq7PkWFe6bVg90H21EOLACvtfcHk0lgAXntsI3cjfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J&md5=9fefb66d80fb0087db3b67755cd028c1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1124%2Fjpet.110.169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.110.169128%26sid%3Dliteratum%253Aachs%26aulast%3DBapat%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DL.%2BS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DLong%26aufirst%3DE.%2BC.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DNovel%2520small%2520molecule%2520inhibitor%2520of%2520Ape1%2520endonuclease%2520blocks%2520proliferation%2520and%2520reduces%2520viability%2520of%2520glioblastoma%2520cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D988%26epage%3D998%26doi%3D10.1124%2Fjpet.110.169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorjsuren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of inhibitors of human apurinic/apyrimidinic endonuclease APE1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e5740</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0005740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1371%2Fjournal.pone.0005740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19484131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A280%3ADC%252BD1MzovVentw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=A.+Simeonovauthor=A.+Kulkarniauthor=D.+Dorjsurenauthor=A.+Jadhavauthor=M.+Shenauthor=D.+R.+McNeillauthor=C.+P.+Austinauthor=D.+M.+Wilson&title=Identification+and+characterization+of+inhibitors+of+human+apurinic%2Fapyrimidinic+endonuclease+APE1&doi=10.1371%2Fjournal.pone.0005740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of inhibitors of human apurinic/apyrimidinic endonuclease APE1</span></div><div class="casAuthors">Simeonov Anton; Kulkarni Avanti; Dorjsuren Dorjbal; Jadhav Ajit; Shen Min; McNeill Daniel R; Austin Christopher P; Wilson David M 3rd</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e5740</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">APE1 is the major nuclease for excising abasic (AP) sites and particular 3'-obstructive termini from DNA, and is an integral participant in the base excision repair (BER) pathway.  BER capacity plays a prominent role in dictating responsiveness to agents that generate oxidative or alkylation DNA damage, as well as certain chain-terminating nucleoside analogs and 5-fluorouracil.  We describe within the development of a robust, 1536-well automated screening assay that employs a deoxyoligonucleotide substrate operating in the red-shifted fluorescence spectral region to identify APE1 endonuclease inhibitors.  This AP site incision assay was used in a titration-based high-throughput screen of the Library of Pharmacologically Active Compounds (LOPAC(1280)), a collection of well-characterized, drug-like molecules representing all major target classes.  Prioritized hits were authenticated and characterized via two high-throughput screening assays -- a Thiazole Orange fluorophore-DNA displacement test and an E. coli endonuclease IV counterscreen -- and a conventional, gel-based radiotracer incision assay.  The top, validated compounds, i.e. 6-hydroxy-DL-DOPA, Reactive Blue 2 and myricetin, were shown to inhibit AP site cleavage activity of whole cell protein extracts from HEK 293T and HeLa cell lines, and to enhance the cytotoxic and genotoxic potency of the alkylating agent methylmethane sulfonate.  The studies herein report on the identification of novel, small molecule APE1-targeted bioactive inhibitor probes, which represent initial chemotypes towards the development of potential pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSLiX2uWPYz9wpJitSxOdIlfW6udTcc2eZb8KSZ8pbMCrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzovVentw%253D%253D&md5=5590848b09580b34791ea17864903b93</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0005740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0005740%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DKulkarni%26aufirst%3DA.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DMcNeill%26aufirst%3DD.%2BR.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520inhibitors%2520of%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%2520APE1%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0005740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyjayanti, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbotts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of human AP endonuclease inhibitors in melanoma and glioma cell lines</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6606058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fsj.bjc.6606058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=21266972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2011&pages=653-663&author=M.+Z.+Mohammedauthor=V.+N.+Vyjayantiauthor=C.+A.+Laughtonauthor=L.+V.+Dekkerauthor=P.+M.+Fischerauthor=D.+M.+Wilsonauthor=R.+Abbottsauthor=S.+Shahauthor=P.+M.+Patelauthor=I.+D.+Hicksonauthor=S.+Madhusudan&title=Development+and+evaluation+of+human+AP+endonuclease+inhibitors+in+melanoma+and+glioma+cell+lines&doi=10.1038%2Fsj.bjc.6606058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of human AP endonuclease inhibitors in melanoma and glioma cell lines</span></div><div class="casAuthors">Mohammed, M. Z.; Vyjayanti, V. N.; Laughton, C. A.; Dekker, L. V.; Fischer, P. M.; Wilson, D. M., III; Abbotts, R.; Shah, S.; Patel, P. M.; Hickson, I. D.; Madhusudan, S.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">653-663</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aims: Modulation of DNA base excision repair (BER) has the potential to enhance response to chemotherapy and improve outcomes in tumors such as melanoma and glioma.  APE1, a crit. protein in BER that processes potentially cytotoxic abasic sites (AP sites), is a promising new target in cancer.  In the current study, we aimed to develop small mol. inhibitors of APE1 for cancer therapy.  Methods: An industry-std. high throughput virtual screening strategy was adopted.  The Sybyl8.0 (Tripos, St Louis, MO, USA) mol. modeling software suite was used to build inhibitor templates.  Similarity searching strategies were then applied using ROCS 2.3 (Open Eye Scientific, Santa Fe, NM, USA) to ext. pharmacophorically related subsets of compds. from a chem. diverse database of 2.6 million compds.  The compds. in these subsets were subjected to docking against the active site of the APE1 model, using the genetic algorithm-based program GOLD2.7 (CCDC, Cambridge, UK).  Predicted ligand poses were ranked on the basis of several scoring functions.  The top virtual hits with promising pharmaceutical properties underwent detailed in vitro analyses using fluorescence-based APE1 cleavage assays and counter screened using endonuclease IV cleavage assays, fluorescence quenching assays and radiolabeled oligonucleotide assays.  Biochem. APE1 inhibitors were then subjected to detailed cytotoxicity analyses.  Results: Several specific APE1 inhibitors were isolated by this approach.  The IC50 for APE1 inhibition ranged between 30 n and 50 μ.  We demonstrated that APE1 inhibitors lead to accumulation of AP sites in genomic DNA and potentiated the cytotoxicity of alkylating agents in melanoma and glioma cell lines.  Conclusions: Our study provides evidence that APE1 is an emerging drug target and could have therapeutic application in patients with melanoma and glioma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRvuWYUOMcELVg90H21EOLACvtfcHk0lhmydWYz6l9kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKhsrg%253D&md5=c29b7758524de944df907d9e267359de</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6606058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6606058%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DM.%2BZ.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DDekker%26aufirst%3DL.%2BV.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DAbbotts%26aufirst%3DR.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DP.%2BM.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520human%2520AP%2520endonuclease%2520inhibitors%2520in%2520melanoma%2520and%2520glioma%2520cell%2520lines%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2011%26volume%3D104%26spage%3D653%26epage%3D663%26doi%3D10.1038%2Fsj.bjc.6606058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholia, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span> <span> </span><span class="NLM_article-title">DNA repair and redox activities and inhibitors of apurinic/apyrimidinic endonuclease 1/redox effector factor 1 (APE1/Ref-1): a comparative analysis and their scope and limitations toward anticancer drug development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10256</span>, <span class="refDoi"> DOI: 10.1021/jm500865u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500865u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10241-10256&author=G.+Kaurauthor=R.+P.+Choliaauthor=A.+K.+Manthaauthor=R.+Kumar&title=DNA+repair+and+redox+activities+and+inhibitors+of+apurinic%2Fapyrimidinic+endonuclease+1%2Fredox+effector+factor+1+%28APE1%2FRef-1%29%3A+a+comparative+analysis+and+their+scope+and+limitations+toward+anticancer+drug+development&doi=10.1021%2Fjm500865u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm500865u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500865u%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DCholia%26aufirst%3DR.%2BP.%26aulast%3DMantha%26aufirst%3DA.%2BK.%26aulast%3DKumar%26aufirst%3DR.%26atitle%3DDNA%2520repair%2520and%2520redox%2520activities%2520and%2520inhibitors%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%252Fredox%2520effector%2520factor%25201%2520%2528APE1%252FRef-1%2529%253A%2520a%2520comparative%2520analysis%2520and%2520their%2520scope%2520and%2520limitations%2520toward%2520anticancer%2520drug%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10241%26epage%3D10256%26doi%3D10.1021%2Fjm500865u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M. P.</span></span> <span> </span><span class="NLM_article-title">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span>. <i>Chem. Commun. (Cambridge, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15948</span>– <span class="NLM_lpage">15951</span>, <span class="refDoi"> DOI: 10.1039/C5CC06084B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1039%2FC5CC06084B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=26377038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFajt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15948-15951&author=N.+Koteraauthor=F.+Poyerauthor=A.+Granzhanauthor=M.+P.+Teulade-Fichou&title=Efficient+inhibition+of+human+AP+endonuclease+1+%28APE1%29+via+substrate+masking+by+abasic+site-binding+macrocyclic+ligands&doi=10.1039%2FC5CC06084B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span></div><div class="casAuthors">Kotera, Naoko; Poyer, Florent; Granzhan, Anton; Teulade-Fichou, Marie-Paule</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">88</span>),
    <span class="NLM_cas:pages">15948-15951</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bis-naphthalene macrocycles, which bind with high affinity and selectivity to abasic sites in DNA, efficiently inhibit their cleavage by APE1 (IC50 = 55-60 nM in the kinetic assay with a model THF substrate).  These results demonstrate that substrate masking by non-covalent abasic-site ligands is an efficient strategy for inhibition of APE1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps5EUcFXG2UrVg90H21EOLACvtfcHk0lhmydWYz6l9kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFajt7%252FI&md5=1ac3974b315995557ac5c74ec421e2f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC5CC06084B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06084B%26sid%3Dliteratum%253Aachs%26aulast%3DKotera%26aufirst%3DN.%26aulast%3DPoyer%26aufirst%3DF.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26atitle%3DEfficient%2520inhibition%2520of%2520human%2520AP%2520endonuclease%25201%2520%2528APE1%2529%2520via%2520substrate%2520masking%2520by%2520abasic%2520site-binding%2520macrocyclic%2520ligands%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D15948%26epage%3D15951%26doi%3D10.1039%2FC5CC06084B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of DNA repair nuclease activities of APE1</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">3621</span>– <span class="NLM_lpage">3631</span>, <span class="refDoi"> DOI: 10.1007/s00018-010-0488-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs00018-010-0488-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20809131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=3621-3631&author=D.+M.+Wilsonauthor=A.+Simeonov&title=Small+molecule+inhibitors+of+DNA+repair+nuclease+activities+of+APE1&doi=10.1007%2Fs00018-010-0488-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of DNA repair nuclease activities of APE1</span></div><div class="casAuthors">Wilson, David M., III; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3621-3631</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  APE1 is a multifunctional protein that possesses several nuclease activities, including the ability to incise at apurinic/apyrimidinic (AP) sites in DNA or RNA, to excise 3'-blocking termini from DNA ends, and to cleave at certain oxidized base lesions in DNA.  Pre-clin. and clin. data indicate a role for APE1 in the pathogenesis of cancer and in resistance to DNA-interactive drugs, particularly monofunctional alkylators and antimetabolites.  In an effort to improve the efficacy of therapeutic compds., such as temozolomide, groups have begun to develop high-throughput screening assays and to identify small mol. inhibitors against APE1 repair nuclease activities.  It is envisioned that such inhibitors will be used in combinatorial treatment paradigms to enhance the efficacy of DNA-interactive drugs that introduce relevant cytotoxic DNA lesions.  In this review, we summarize the current state of the efforts to design potent and selective inhibitors against APE1 AP site incision activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN4mUcnbEDarVg90H21EOLACvtfcHk0liBuKqAp52fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN&md5=7406c275670396d1652dfda2cf68e2af</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs00018-010-0488-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-010-0488-2%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520DNA%2520repair%2520nuclease%2520activities%2520of%2520APE1%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D3621%26epage%3D3631%26doi%3D10.1007%2Fs00018-010-0488-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyjayanti, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorjsuren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3101</span>– <span class="NLM_lpage">3112</span>, <span class="refDoi"> DOI: 10.1021/jm201537d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201537d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3101-3112&author=G.+Raiauthor=V.+N.+Vyjayantiauthor=D.+Dorjsurenauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+M.+Wilsonauthor=D.+J.+Maloney&title=Synthesis%2C+biological+evaluation%2C+and+structure-activity+relationships+of+a+novel+class+of+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm201537d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span></div><div class="casAuthors">Rai, Ganesha; Vyjayanti, Vaddadi N.; Dorjsuren, Dorjbal; Simeonov, Anton; Jadhav, Ajit; Wilson, David M.; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3101-3112</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">APE1 is an essential protein that operates in the base excision repair (BER) pathway and is responsible for ≥95% of the total apurinic/apyrimidinic (AP) endonuclease activity in human cells.  BER is a major pathway that copes with DNA damage induced by several anticancer agents, including ionizing radiation and temozolomide.  Overexpression of APE1 and enhanced AP endonuclease activity have been linked to increased resistance of tumor cells to treatment with monofunctional alkylators, implicating inhibition of APE1 as a valid strategy for cancer therapy.  We report herein the results of a focused medicinal chem. effort around a novel APE1 inhibitor, N-(3-(benzo[d]thiazol-2-yl)-6-isopropyl-4,5,6,7-tetrahydrothieno[2,3-c]pyridin-2-yl)acetamide (I).  Compd. I and related analogs exhibit single-digit micromolar activity against the purified APE1 enzyme and comparable activity in HeLa whole cell ext. assays and potentiate the cytotoxicity of the alkylating agents methylmethane sulfonate and temozolomide.  Moreover, this class of compds. possesses a generally favorable in vitro ADME profile, along with good exposure levels in plasma and brain following i.p. dosing (30 mg/kg body wt.) in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25R0Ebe53srVg90H21EOLACvtfcHk0lgnEkigzcN6WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D&md5=1dac481ed0ec1cc67d59ad3091520ccc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm201537d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201537d%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DSynthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520novel%2520class%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3101%26epage%3D3112%26doi%3D10.1021%2Fjm201537d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogbourne, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, P. G.</span></span> <span> </span><span class="NLM_article-title">The value of nature’s natural product library for the discovery of New Chemical Entities: the discovery of ingenol mebutate</span>. <i>Fitoterapia</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1016/j.fitote.2014.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.fitote.2014.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25016953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlantLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2014&pages=36-44&author=S.+M.+Ogbourneauthor=P.+G.+Parsons&title=The+value+of+nature%E2%80%99s+natural+product+library+for+the+discovery+of+New+Chemical+Entities%3A+the+discovery+of+ingenol+mebutate&doi=10.1016%2Fj.fitote.2014.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The value of nature's natural product library for the discovery of New Chemical Entities: The discovery of ingenol mebutate</span></div><div class="casAuthors">Ogbourne, Steven M.; Parsons, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Fitoterapia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">FTRPAE</span>;
        ISSN:<span class="NLM_cas:issn">0367-326X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent decades, 'Big Pharma' has invested billions of dollars into ingenious and innovative strategies designed to develop drugs using high throughput screening of small mol. libraries generated on the lab. bench.  Within the same time frame, screening of natural products by pharmaceutical companies has suffered an equally significant redn.  This is despite the fact that the complexity, functional diversity and druggability of nature's natural product library are considered by many to be superior to any library any team of scientists can prep.  It is therefore no coincidence that the no. of New Chem. Entities reaching the market has also suffered a substantial decrease, leading to a productivity crisis within the pharmaceutical sector.  In fact, the current dearth of New Chem. Entities reaching the market in recent decades might be a direct consequence of the strategic decision to move away from screening of natural products.  Nearly 700 novel drugs derived from natural product New Chem. Entities were approved between 1981 and 2010; more than 60% of all approved drugs over the same time.  In this review, we use the example of ingenol mebutate, a natural product identified from Euphorbia peplus and later approved as a therapy for actinic keratosis, as why nature's natural product library remains the most valuable library for discovery of New Chem. Entities and of novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMRLEFPnv8KrVg90H21EOLACvtfcHk0lgnEkigzcN6WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlantLjO&md5=018f18eff0c2d50123a68a7c344974c9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.fitote.2014.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fitote.2014.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DOgbourne%26aufirst%3DS.%2BM.%26aulast%3DParsons%26aufirst%3DP.%2BG.%26atitle%3DThe%2520value%2520of%2520nature%25E2%2580%2599s%2520natural%2520product%2520library%2520for%2520the%2520discovery%2520of%2520New%2520Chemical%2520Entities%253A%2520the%2520discovery%2520of%2520ingenol%2520mebutate%26jtitle%3DFitoterapia%26date%3D2014%26volume%3D98%26spage%3D36%26epage%3D44%26doi%3D10.1016%2Fj.fitote.2014.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wessjohann, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruijter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Rivera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, W.</span></span> <span> </span><span class="NLM_article-title">What can a chemist learn from nature’s macrocycles?--a brief, conceptual view</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1007/s11030-005-1314-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs11030-005-1314-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=15789564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=171-186&author=L.+A.+Wessjohannauthor=E.+Ruijterauthor=D.+Garcia-Riveraauthor=W.+Brandt&title=What+can+a+chemist+learn+from+nature%E2%80%99s+macrocycles%3F%2D%2Da+brief%2C+conceptual+view&doi=10.1007%2Fs11030-005-1314-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">What can a chemist learn from nature's macrocycles? - A brief, conceptual view</span></div><div class="casAuthors">Wessjohann, Ludger A.; Ruijter, Eelco; Garcia-Rivera, Daniel; Brandt, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">171-186</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Macrocyclic natural products often display remarkable biol. activities, and many of these compds. (or their derivs.) are used as drugs.  The chem. diversity of these compds. is immense and may provide inspiration for innovative drug design.  Therefore, a database of naturally occurring macrocycles was analyzed for ring size, mol. wt. distribution, and the frequency of some common substructural motifs.  The underlying principles of the chem. diversity are reviewed in terms of biosynthetic origin and nature's strategies for diversity and complexity generation in relation to the structural diversity and similarities found in the macrocycle database.  Finally, it is suggested that synthetic chemists should use not only nature's mols., but also nature's strategies as a source of inspiration.  To illustrate this, the biosynthesis of macrocycles by non-ribosomal peptide synthetases and terpene and polyketide cyclases, as well as recent advances of these strategies in an integrated synthesis/biotechnol. approach are briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVB4g1QplcWLVg90H21EOLACvtfcHk0li5CI8ovj7Yrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D&md5=88df3ca095b446e3781369708ccddfd2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11030-005-1314-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-005-1314-x%26sid%3Dliteratum%253Aachs%26aulast%3DWessjohann%26aufirst%3DL.%2BA.%26aulast%3DRuijter%26aufirst%3DE.%26aulast%3DGarcia-Rivera%26aufirst%3DD.%26aulast%3DBrandt%26aufirst%3DW.%26atitle%3DWhat%2520can%2520a%2520chemist%2520learn%2520from%2520nature%25E2%2580%2599s%2520macrocycles%253F--a%2520brief%252C%2520conceptual%2520view%26jtitle%3DMol.%2520Diversity%26date%3D2005%26volume%3D9%26spage%3D171%26epage%3D186%26doi%3D10.1007%2Fs11030-005-1314-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0li5CI8ovj7Yrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+five+in+2015+and+beyond%3A+Target+and+ligand+structural+limitations%2C+ligand+chemistry+structure+and+drug+discovery+project+decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lhVRVNXKlPI_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520ligand%2520structural%2520limitations%252C%2520ligand%2520chemistry%2520structure%2520and%2520drug%2520discovery%2520project%2520decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lhVRVNXKlPI_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lhVRVNXKlPI_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lgVIik8SegP6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and synthetic methodologies toward macrocycles</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6230</span>– <span class="NLM_lpage">6268</span>, <span class="refDoi"> DOI: 10.3390/molecules18066230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.3390%2Fmolecules18066230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=23708234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFemurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=6230-6268&author=X.+Yuauthor=D.+Sun&title=Macrocyclic+drugs+and+synthetic+methodologies+toward+macrocycles&doi=10.3390%2Fmolecules18066230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic drugs and synthetic methodologies toward macrocycles</span></div><div class="casAuthors">Yu, Xufen; Sun, Dianqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6230-6268</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Macrocyclic scaffolds are commonly found in bioactive natural products and pharmaceutical mols.  So far, a large no. of macrocyclic natural products were isolated and synthesized.  The construction of macrocycles is generally considered as a crucial and challenging step in the synthesis of macrocyclic natural products.  Over the last several decades, numerous efforts were undertaken toward the synthesis of complex naturally occurring macrocycles and great progresses were made to advance the field of total synthesis.  The commonly used synthetic methodologies toward macrocyclization include macrolactonization, macrolactamization, transition metal-catalyzed cross coupling, ring-closing metathesis, and click reaction, among others.  Selected recent examples of macrocyclic synthesis of natural products and drug-like macrocycles with significant biol. relevance are highlighted in each class.  The primary goal of this review was to summarize currently used macrocyclic drugs, highlight the therapeutic potential of this under-explored drug class and outline the general synthetic methodologies for the synthesis of macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVX-oHAYozLrVg90H21EOLACvtfcHk0lgVIik8SegP6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFemurw%253D&md5=b5756f10331fd590049422000f67b900</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18066230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18066230%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DMacrocyclic%2520drugs%2520and%2520synthetic%2520methodologies%2520toward%2520macrocycles%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D6230%26epage%3D6268%26doi%3D10.3390%2Fmolecules18066230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Drug discovery: tools and rules for macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">696</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnchembio.1605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25038789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=696-698&author=C.+Heinis&title=Drug+discovery%3A+tools+and+rules+for+macrocycles&doi=10.1038%2Fnchembio.1605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery. Tools and rules for macrocycles</span></div><div class="casAuthors">Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">696-698</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The application of macrocycles to previously undruggable targets has aroused a great deal of interest in this structural class.  Recent studies advance our understanding of the way macrocycles bind protein targets and provide new strategies and tools to generate peptide-based macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqeFUEIoJytLVg90H21EOLACvtfcHk0lhY9O-QXUMBlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nM&md5=c9063baf5e54835719af3ff6e3cabe8f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1605%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDrug%2520discovery%253A%2520tools%2520and%2520rules%2520for%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D696%26epage%3D698%26doi%3D10.1038%2Fnchembio.1605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lhY9O-QXUMBlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">DNA-bound structure and mutants reveal abasic DNA binding by APE1 DNA repair and coordination</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1038/35000249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2F35000249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10667800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD3cXpslaitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=451-456&author=C.+D.+Molauthor=T.+Izumiauthor=S.+Mitraauthor=J.+A.+Tainer&title=DNA-bound+structure+and+mutants+reveal+abasic+DNA+binding+by+APE1+DNA+repair+and+coordination&doi=10.1038%2F35000249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-bound structures and mutants reveal abasic DNA binding by APE1 DNA repair and coordination</span></div><div class="casAuthors">Mol, Clifford D.; Izumi, Tadahide; Mitra, Sankar; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6768</span>),
    <span class="NLM_cas:pages">451-456</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Non-coding apurinic/apyrimidinic (AP) sites in DNA are continually created in cells both spontaneously and by damage- specific DNA glycosylases.  The biol. crit. human base excision repair enzyme APE1 cleaves the DNA sugar-phosphate backbone at a position 5' of AP sites to prime DNA repair synthesis.  Here we report three co-crystal structures of human APE1 bound to abasic DNA which show that APEI uses a rigid, pre-formed, pos. charged surface to kink the DNA helix and engulf the AP-DNA strand.  APE1 inserts loops into both the DNA major and minor grooves and binds a flipped-out AP site in a pocket that excludes DNA bases and racemized β-anomer AP sites.  Both the APE1 active-site geometry and a complex with cleaved AP-DNA and Mn2+ support a testable structure-based catalytic mechanism.  Alanine substitutions of the residues that penetrate the DNA helix unexpectedly show that human APE1 is structurally optimized to retain the cleaved DNA product.  These structural and mutational results show how APE1 probably displaces bound glycosylases and retains the nicked DNA product, suggesting that APEI acts in vivo to coordinate the orderly transfer of unstable DNA damage intermediates between the excision and synthesis steps of DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTgcGoYjDzMrVg90H21EOLACvtfcHk0lioMfka20d4NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXpslaitg%253D%253D&md5=59d3d4ba71c516f9bf4693ae7199aa37</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F35000249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35000249%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DIzumi%26aufirst%3DT.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DDNA-bound%2520structure%2520and%2520mutants%2520reveal%2520abasic%2520DNA%2520binding%2520by%2520APE1%2520DNA%2520repair%2520and%2520coordination%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D451%26epage%3D456%26doi%3D10.1038%2F35000249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span> <span> </span><span class="NLM_article-title">High-resolution crystal structures reveal plasticity in the metal binding site of apurinic/apyrimidinic endonuclease I</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6529</span>, <span class="refDoi"> DOI: 10.1021/bi500676p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500676p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Sju7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6520-6529&author=H.+Heauthor=Q.+Chenauthor=M.+M.+Georgiadis&title=High-resolution+crystal+structures+reveal+plasticity+in+the+metal+binding+site+of+apurinic%2Fapyrimidinic+endonuclease+I&doi=10.1021%2Fbi500676p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structures Reveal Plasticity in the Metal Binding Site of Apurinic/Apyrimidinic Endonuclease I</span></div><div class="casAuthors">He, Hongzhen; Chen, Qiujia; Georgiadis, Millie M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6520-6529</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic endonuclease I (APE1) is an essential base excision repair enzyme that catalyzes a Mg2+-dependent reaction in which the phosphodiester backbone is cleaved 5' of an abasic site in duplex DNA.  This reaction has been proposed to involve either one or two metal ions bound to the active site.  In the present study, we report crystal structures of Mg2+, Mn2+, and apo-APE1 detd. at 1.4, 2.2, and 1.65 Å, resp., representing two of the highest resoln. structures yet reported for APE1.  In our structures, a single well-ordered Mn2+ ion was obsd. coordinated by D70 and E96; the Mg2+ site exhibited disorder modeled as two closely positioned sites coordinated by D70 and E96 or E96 alone.  Direct metal binding anal. of wild-type, D70A, and E96A APE1, as assessed by differential scanning fluorimetry, indicated a role for D70 and E96 in binding of Mg2+ or Mn2+ to APE1.  Consistent with the disorder exhibited by Mg2+ bound to the active site, two different conformations of E96 were obsd. coordinated to Mg2+.  A third conformation for E96 in the apo structure is similar to that obsd. in the APE1-DNA-Mg2+ complex structure.  Thus, binding of Mg2+ in three different positions within the active site of APE1 in these crystal structures corresponds directly with three different conformations of E96.  Taken together, our results are consistent with the initial capture of metal by D70 and E96 and repositioning of Mg2+ facilitated by the structural plasticity of E96 in the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwsb3wP6G4ILVg90H21EOLACvtfcHk0lioMfka20d4NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Sju7fK&md5=337d74f58a6c1d6907dd54f4debb5956</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fbi500676p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500676p%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26atitle%3DHigh-resolution%2520crystal%2520structures%2520reveal%2520plasticity%2520in%2520the%2520metal%2520binding%2520site%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%2520I%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D6520%26epage%3D6529%26doi%3D10.1021%2Fbi500676p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span> <span> </span><span class="NLM_article-title">Bifunctional homoallylic carbamates from chiral silane additions to in situ generated N-acyl iminium ions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3624</span>– <span class="NLM_lpage">3627</span>, <span class="refDoi"> DOI: 10.1021/ol301428y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol301428y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=3624-3627&author=J.+Wuauthor=K.+C.+Zhuauthor=P.+W.+Yuanauthor=J.+S.+Panek&title=Bifunctional+homoallylic+carbamates+from+chiral+silane+additions+to+in+situ+generated+N-acyl+iminium+ions&doi=10.1021%2Fol301428y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bifunctional homoallylic carbamates from chiral silane additions to in situ generated N-acyl iminium ions</span></div><div class="casAuthors">Wu, Jie; Zhu, Kai-Cheng; Yuan, Ping-Wei; Panek, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3624-3627</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Homoallylic carbamates bearing an α,β-unsatd. ester or an allylic carbonate were generated resp. utilizing novel chiral silanes through Lewis acid promoted asym. aminocrotylation.  Those bifunctional building blocks could further undergo Michael addn. or cyclization to selectively form functionalized lactams, azetidines, and tetrahydropyrimidinones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Rwub53UlwrVg90H21EOLACvtfcHk0lioMfka20d4NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cgtr8%253D&md5=673a276eb136329485b8cb5ea02b04aa</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fol301428y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol301428y%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%2BC.%26aulast%3DYuan%26aufirst%3DP.%2BW.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26atitle%3DBifunctional%2520homoallylic%2520carbamates%2520from%2520chiral%2520silane%2520additions%2520to%2520in%2520situ%2520generated%2520N-acyl%2520iminium%2520ions%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D3624%26epage%3D3627%26doi%3D10.1021%2Fol301428y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beresis, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of chiral (E)-Cryotylsilanes: [3R- and 3S-]-(4E)-Methyl 3-(Dimethylphenylsilyl)-4-Hexenoate</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi"> DOI: 10.1002/0471264180.os075.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2F0471264180.os075.10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1998&pages=78&author=R.+T.+Beresisauthor=J.+S.+Solomonauthor=M.+G.+Yangauthor=N.+F.+Jainauthor=J.+S.+Panek&title=Synthesis+of+chiral+%28E%29-Cryotylsilanes%3A+%5B3R-+and+3S-%5D-%284E%29-Methyl+3-%28Dimethylphenylsilyl%29-4-Hexenoate&doi=10.1002%2F0471264180.os075.10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F0471264180.os075.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264180.os075.10%26sid%3Dliteratum%253Aachs%26aulast%3DBeresis%26aufirst%3DR.%2BT.%26aulast%3DSolomon%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DM.%2BG.%26aulast%3DJain%26aufirst%3DN.%2BF.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520chiral%2520%2528E%2529-Cryotylsilanes%253A%2520%255B3R-%2520and%25203S-%255D-%25284E%2529-Methyl%25203-%2528Dimethylphenylsilyl%2529-4-Hexenoate%26jtitle%3DOrg.%2520Synth.%26date%3D1998%26volume%3D75%26spage%3D78%26doi%3D10.1002%2F0471264180.os075.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beeler, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquilano, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Synthesis of a library of complex macrodiolides employing cyclodimerization of hydroxy esters</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1021/cc050064b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc050064b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVChtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=673-681&author=A.+B.+Beelerauthor=D.+E.+Acquilanoauthor=Q.+Suauthor=F.+Yanauthor=B.+L.+Rothauthor=J.+S.+Panekauthor=J.+A.+Porco&title=Synthesis+of+a+library+of+complex+macrodiolides+employing+cyclodimerization+of+hydroxy+esters&doi=10.1021%2Fcc050064b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of a Library of Complex Macrodiolides Employing Cyclodimerization of Hydroxy Esters</span></div><div class="casAuthors">Beeler, Aaron B.; Acquilano, Dayle E.; Su, Qibin; Yan, Feng; Roth, Bryan L.; Panek, James S.; Porco, John A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">673-681</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of complex macrodiolides involving microwave-accelerated transesterification of chiral, nonracemic, hydroxy esters is described.  Thus, reacting hydroxy esters I and II using [[F3C(CF2)5CH2CH2]2SnO]n as catalyst gave macrodiolide III in 99% yield.  Methodol. development studies indicate that both microwave power and reaction temp. play an important role in the efficiency of cyclodimerizations.  Hydroxy ester monomer pairs were evaluated using an anal. rehearsal leading to the prepn. of a 127-member library of highly diverse and stereochem. well-defined macrodiolides.  Preliminary assays identified a novel macrodiolide antagonist, IV (Ts = SO2C6H4Me-4), of the κ opioid receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplBVRsXsFxErVg90H21EOLACvtfcHk0liu2QvlNgZG3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVChtbY%253D&md5=42c552bdccdd3d4c43ca9dbb92c9ed57</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcc050064b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc050064b%26sid%3Dliteratum%253Aachs%26aulast%3DBeeler%26aufirst%3DA.%2BB.%26aulast%3DAcquilano%26aufirst%3DD.%2BE.%26aulast%3DSu%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26atitle%3DSynthesis%2520of%2520a%2520library%2520of%2520complex%2520macrodiolides%2520employing%2520cyclodimerization%2520of%2520hydroxy%2520esters%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2005%26volume%3D7%26spage%3D673%26epage%3D681%26doi%3D10.1021%2Fcc050064b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, Y.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of bacterial resistance to macrolide antibiotics</span>. <i>J. Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1007/s101560050011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs101560050011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=11810493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADyaK1MXktlCgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=61-74&author=Y.+Nakajima&title=Mechanisms+of+bacterial+resistance+to+macrolide+antibiotics&doi=10.1007%2Fs101560050011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of bacterial resistance to macrolide antibiotics</span></div><div class="casAuthors">Nakajima, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infection and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">61-74</span>CODEN:
                <span class="NLM_cas:coden">JICHFN</span>;
        ISSN:<span class="NLM_cas:issn">1341-321X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Tokyo</span>)
        </div><div class="casAbstract">A review with 125 refs.  On the basis of information obtained from extensive refs. and studies of resistance mechanisms to macrolide antibiotics, the mode of action of the drugs, as effectors, and a hypothetical explanation of the regulation of the mechanism with regard to induction of macrolide resistance are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyNVI7zYQrLrVg90H21EOLACvtfcHk0lgBDKavAhFBVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlCgs7s%253D&md5=a56376c70e47e9ee4b546c1d3e2dc750</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2Fs101560050011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs101560050011%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DY.%26atitle%3DMechanisms%2520of%2520bacterial%2520resistance%2520to%2520macrolide%2520antibiotics%26jtitle%3DJ.%2520Infect.%2520Chemother.%26date%3D1999%26volume%3D5%26spage%3D61%26epage%3D74%26doi%3D10.1007%2Fs101560050011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pengelly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koteva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Mechanism and diversity of the erythromycin esterase family of enzymes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1740</span>– <span class="NLM_lpage">1751</span>, <span class="refDoi"> DOI: 10.1021/bi201790u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi201790u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1OnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1740-1751&author=M.+Morarauthor=K.+Pengellyauthor=K.+Kotevaauthor=G.+D.+Wright&title=Mechanism+and+diversity+of+the+erythromycin+esterase+family+of+enzymes&doi=10.1021%2Fbi201790u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and Diversity of the Erythromycin Esterase Family of Enzymes</span></div><div class="casAuthors">Morar, Mariya; Pengelly, Kate; Koteva, Kalinka; Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1740-1751</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrolide antibiotics such as azithromycin and erythromycin are mainstays of modern antibacterial chemotherapy, and like all antibiotics, they are vulnerable to resistance.  One mechanism of macrolide resistance is via drug inactivation: enzymic hydrolysis of the macrolactone ring catalyzed by erythromycin esterases, EreA and EreB.  A genomic enzymol. approach was taken to gain insight into the catalytic mechanisms and origins of Ere enzymes.  Our anal. reveals that erythromycin esterases comprise a sep. group in the hydrolase superfamily, which includes homologues of uncharacterized function found on the chromosome of Bacillus cereus, Bcr135 and Bcr136, whose three-dimensional structures have been detd.  Biochem. characterization of Bcr136 confirms that it is an esterase; however, it is unable to inactivate macrolides.  Using steady-state kinetics, homol.-based structure modeling, site-directed mutagenesis, solvent isotope effect studies, pH, and inhibitor profiling performed in various combinations for EreA, EreB, and Bcr136 enzymes, we identified the active site and gained insight into some catalytic features of this novel enzyme superfamily.  We rule out the possibility of a Ser/Thr nucleophile and show that one histidine, H46 (EreB numbering), is essential for catalytic function.  This residue is proposed to serve as a general base in activation of a water mol. as the reaction nucleophile.  Furthermore, we show that EreA, EreB, and Bcr136 are distinct, with only EreA inhibited by chelating agents and hypothesized to contain a noncatalytic metal.  Detailed characterization of these esterases allows for a direct comparison of the resistance determinants, EreA and EreB, with their prototype, Bcr136, and for the discussion of their potential connections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEzsk8G72di7Vg90H21EOLACvtfcHk0lgBDKavAhFBVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1OnsbY%253D&md5=f8a48c116f66a4a5cdad00880328ab81</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fbi201790u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi201790u%26sid%3Dliteratum%253Aachs%26aulast%3DMorar%26aufirst%3DM.%26aulast%3DPengelly%26aufirst%3DK.%26aulast%3DKoteva%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DMechanism%2520and%2520diversity%2520of%2520the%2520erythromycin%2520esterase%2520family%2520of%2520enzymes%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D1740%26epage%3D1751%26doi%3D10.1021%2Fbi201790u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin-Sinex, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Knockdown of the DNA repair and redox signaling protein Ape1/Ref-1 blocks ovarian cancer cell and tumor growth</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.09.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.dnarep.2007.09.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=17974506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisl2gtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=177-186&author=M.%0AL.+Fishelauthor=Y.+Heauthor=A.+M.+Reedauthor=H.+Chin-Sinexauthor=G.+D.+Hutchinsauthor=M.+S.+Mendoncaauthor=M.+R.+Kelley&title=Knockdown+of+the+DNA+repair+and+redox+signaling+protein+Ape1%2FRef-1+blocks+ovarian+cancer+cell+and+tumor+growth&doi=10.1016%2Fj.dnarep.2007.09.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Knockdown of the DNA repair and redox signaling protein Ape1/Ref-1 blocks ovarian cancer cell and tumor growth</span></div><div class="casAuthors">Fishel, Melissa L.; He, Ying; Reed, April M.; Chin-Sinex, Helen; Hutchins, Gary D.; Mendonca, Marc S.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-186</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apurinic endonuclease 1/redox effector factor-1 (Ape1/Ref-1 or Ape1) is an essential protein with two distinct functions.  It is a DNA repair enzyme in the base excision repair (BER) pathway and a redn.-oxidn. (redox) signaling factor maintaining transcription factors in an active reduced state.  The authors' lab. previously demonstrated that Ape1 is overexpressed in ovarian cancer and potentially contributes to resistance.  Therefore, the authors utilized siRNA technol. to knockdown protein levels of Ape1 in ovarian cancer cell line, SKOV-3x.  Knocking Ape1 down had dramatic effects on cell growth in vitro but was not due to an increase in apoptosis and at least partially due to an extension in transit time through S-phase.  Similarly, human ovarian tumor xenografts with reduced levels of Ape1 protein demonstrated a dramatic redn. in tumor vol. and also statistically significant differences in 18F-fluorodeoxyglucose (FDG) uptake indicating reduced glucose metab. and cellular proliferation.  Ape1's role in DNA repair and redox signaling is important to the basic understanding of ovarian cancer cell growth and these findings strongly support Ape1 as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVlAid1yFC8bVg90H21EOLACvtfcHk0lgvXcT0gmhNNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisl2gtQ%253D%253D&md5=5aa8979ccf9449727259b0bb5e51da97</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.09.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.09.008%26sid%3Dliteratum%253Aachs%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DReed%26aufirst%3DA.%2BM.%26aulast%3DChin-Sinex%26aufirst%3DH.%26aulast%3DHutchins%26aufirst%3DG.%2BD.%26aulast%3DMendonca%26aufirst%3DM.%2BS.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DKnockdown%2520of%2520the%2520DNA%2520repair%2520and%2520redox%2520signaling%2520protein%2520Ape1%252FRef-1%2520blocks%2520ovarian%2520cancer%2520cell%2520and%2520tumor%2520growth%26jtitle%3DDNA%2520Repair%26date%3D2008%26volume%3D7%26spage%3D177%26epage%3D186%26doi%3D10.1016%2Fj.dnarep.2007.09.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A. R.</span></span> <span> </span><span class="NLM_article-title">The comet assay for DNA damage and repair: principles, applications, and limitations</span>. <i>Mol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1385/MB:26:3:249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1385%2FMB%3A26%3A3%3A249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=15004294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivF2gu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=249-261&author=A.+R.+Collins&title=The+comet+assay+for+DNA+damage+and+repair%3A+principles%2C+applications%2C+and+limitations&doi=10.1385%2FMB%3A26%3A3%3A249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The comet assay for DNA damage and repair: principles, applications, and limitations</span></div><div class="casAuthors">Collins, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-261</span>CODEN:
                <span class="NLM_cas:coden">MLBOEO</span>;
        ISSN:<span class="NLM_cas:issn">1073-6085</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  The comet assay (single-cell gel electrophoresis) is a simple method for measuring DNA (DNA) strand breaks in eukaryotic cells.  Cells embedded in agarose on a microscope slide are lysed with detergent and high salt to form nucleoids contg. supercoiled loops of DNA linked to the nuclear matrix.  Electrophoresis at high pH results in structures resembling comets, obsd. by fluorescence microscopy; the intensity of the comet tail relative to the head reflects the no. of DNA breaks.  The likely basis for this is that loops contg. a break lose their supercoiling and become free to extend toward the anode.  The assay has applications in testing novel chems. for genotoxicity, monitoring environmental contamination with genotoxins, human biomonitoring and mol. epidemiol., and fundamental research in DNA damage and repair.  The sensitivity and specificity of the assay are greatly enhanced if the nucleoids are incubated with bacterial repair endonucleases that recognize specific kinds of damage in the DNA and convert lesions to DNA breaks, increasing the amt. of DNA in the comet tail.  DNA repair can be monitored by incubating cells after treatment with damaging agent and measuring the damage remaining at intervals.  Alternatively, the repair activity in a cell ext. can be measured by incubating it with nucleoids contg. specific damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8T2ekl_rxn7Vg90H21EOLACvtfcHk0lgvXcT0gmhNNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivF2gu7c%253D&md5=159fc301847b54325077c45f10795f13</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1385%2FMB%3A26%3A3%3A249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1385%252FMB%253A26%253A3%253A249%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DA.%2BR.%26atitle%3DThe%2520comet%2520assay%2520for%2520DNA%2520damage%2520and%2520repair%253A%2520principles%252C%2520applications%252C%2520and%2520limitations%26jtitle%3DMol.%2520Biotechnol.%26date%3D2004%26volume%3D26%26spage%3D249%26epage%3D261%26doi%3D10.1385%2FMB%3A26%3A3%3A249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agurell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beevers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendler-Schwaab, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlinson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thybaud, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, R. R.</span></span> <span> </span><span class="NLM_article-title">th International Comet Assay, W. Recommendations for conducting the in vivo alkaline Comet assay. 4th International Comet Assay Workshop</span>. <i>Mutagenesis</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1093/mutage/18.1.45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fmutage%2F18.1.45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=12473734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsFChu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=45-51&author=A.+Hartmannauthor=E.+Agurellauthor=C.+Beeversauthor=S.+Brendler-Schwaabauthor=B.+Burlinsonauthor=P.+Clayauthor=A.+Collinsauthor=A.+Smithauthor=G.+Speitauthor=V.+Thybaudauthor=R.+R.+Tice&title=th+International+Comet+Assay%2C+W.+Recommendations+for+conducting+the+in+vivo+alkaline+Comet+assay.+4th+International+Comet+Assay+Workshop&doi=10.1093%2Fmutage%2F18.1.45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendations for conducting the in vivo alkaline Comet assay</span></div><div class="casAuthors">Hartmann, A.; Agurell, E.; Beevers, C.; Brendler-Schwaab, S.; Burlinson, B.; Clay, P.; Collins, A.; Smith, A.; Speit, G.; Thybaud, V.; Tice, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Mutagenesis</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">MUTAEX</span>;
        ISSN:<span class="NLM_cas:issn">0267-8357</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The in vivo alk. single cell gel electrophoresis assay, hereafter the Comet assay, can be used to investigate the genotoxicity of industrial chems., biocides, agrochems. and pharmaceuticals.  The major advantages of this assay include the relative ease of application to any tissue of interest, the detection of multiple classes of DNA damage and the generation of data at the level of the single cell.  These features give the Comet assay potential advantages over other in vivo test methods, which are limited largely to proliferating cells and/or a single tissue.  The Comet assay has demonstrated its reliability in many testing circumstances and is, in general, considered to be acceptable for regulatory purposes.  However, despite the considerable data published on the in vivo Comet assay and the general agreement within the international scientific community over many protocol-related issues, it was felt that a document giving detailed practical guidance on the protocol required for regulatory acceptance of the assay was required.  In a recent meeting held in conjunction with the 4th International Comet Assay Workshop (Ulm, Germany, 22-25 July 2001) an expert panel reviewed existing data and recent developments of the Comet assay with a view to developing such a document.  This paper is intended to act as an update to the more general guidelines which were published as a result of the International Workshop on Genotoxicity Test Procedures.  The recommendations are also seen as a major step towards gaining more formal regulatory acceptance of the Comet assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAzJMHaGXCLLVg90H21EOLACvtfcHk0lgWG4Ny27GN9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsFChu7o%253D&md5=c3ad3b2907b23d0471de6f90b8dad721</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fmutage%2F18.1.45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmutage%252F18.1.45%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DA.%26aulast%3DAgurell%26aufirst%3DE.%26aulast%3DBeevers%26aufirst%3DC.%26aulast%3DBrendler-Schwaab%26aufirst%3DS.%26aulast%3DBurlinson%26aufirst%3DB.%26aulast%3DClay%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DSpeit%26aufirst%3DG.%26aulast%3DThybaud%26aufirst%3DV.%26aulast%3DTice%26aufirst%3DR.%2BR.%26atitle%3Dth%2520International%2520Comet%2520Assay%252C%2520W.%2520Recommendations%2520for%2520conducting%2520the%2520in%2520vivo%2520alkaline%2520Comet%2520assay.%25204th%2520International%2520Comet%2520Assay%2520Workshop%26jtitle%3DMutagenesis%26date%3D2003%26volume%3D18%26spage%3D45%26epage%3D51%26doi%3D10.1093%2Fmutage%2F18.1.45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaplane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M.</span></span> <span> </span><span class="NLM_article-title">Evolution of the redox function in mammalian apurinic/apyrimidinic endonuclease</span>. <i>Mutat. Res., Fundam. Mol. Mech. Mutagen.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>643</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.mrfmmm.2008.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.mrfmmm.2008.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=18579163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=643&publication_year=2008&pages=54-63&author=M.+Georgiadisauthor=M.+Luoauthor=R.+Gaurauthor=S.+Delaplaneauthor=X.+Liauthor=M.+Kelley&title=Evolution+of+the+redox+function+in+mammalian+apurinic%2Fapyrimidinic+endonuclease&doi=10.1016%2Fj.mrfmmm.2008.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of the redox function in mammalian apurinic/apyrimidinic endonuclease</span></div><div class="casAuthors">Georgiadis, M. M.; Luo, M.; Gaur, R. K.; Delaplane, S.; Li, X.; Kelley, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, Fundamental and Molecular Mechanisms of Mutagenesis</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">643</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">54-63</span>CODEN:
                <span class="NLM_cas:coden">MUREAV</span>;
        ISSN:<span class="NLM_cas:issn">0027-5107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human apurinic/apyrimidinic endonuclease (hApe1) encodes two important functional activities: an essential base excision repair (BER) activity and a redox activity that regulates expression of a no. of genes through redn. of their transcription factors, AP-1, NFκB, HIF-1α, CREB, p53 and others.  The BER function is highly conserved from prokaryotes (E. coli exonuclease III) to humans (hApe1).  Here, we provide evidence supporting a redox function unique to mammalian Apes.  An evolutionary anal. of Ape sequences reveals that, of the 7 Cys residues, Cys-93, 99, 208, 296, and 310 are conserved in both mammalian and non-mammalian vertebrate Apes, while Cys-65 is unique to mammalian Apes.  In the zebrafish Ape (zApe), selected as the vertebrate sequence most distant from human, the residue equiv. to Cys-65 is Thr-58.  The wild-type zApe enzyme was tested for redox activity in both in vitro EMSA and transactivation assays and found to be inactive, similar to C65A hApe1.  Substitution of Thr-58 with Cys in zApe, however, resulted in a redox active enzyme, suggesting that a Cys residue in this position is indeed crit. for redox function.  To further probe differences between redox active and inactive enzymes, we have detd. the crystal structures of vertebrate redox inactive enzymes, the C65A human Ape1 enzyme and the zApe enzyme at 1.9 and 2.3 Å, resp.  Our results provide new insights on the redox function and highlight a dramatic gain-of-function activity for Ape1 in mammals not found in non-mammalian vertebrates or lower organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJCx6EpFifxLVg90H21EOLACvtfcHk0lgWG4Ny27GN9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWitrrK&md5=5d7f84bd92612a8822ed2e17631b0884</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.mrfmmm.2008.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mrfmmm.2008.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgiadis%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DGaur%26aufirst%3DR.%26aulast%3DDelaplane%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DKelley%26aufirst%3DM.%26atitle%3DEvolution%2520of%2520the%2520redox%2520function%2520in%2520mammalian%2520apurinic%252Fapyrimidinic%2520endonuclease%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D2008%26volume%3D643%26spage%3D54%26epage%3D63%26doi%3D10.1016%2Fj.mrfmmm.2008.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgWG4Ny27GN9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span> <i>SAINT</i>,  <i>SADABS</i>, and  <i>XPREP</i>; <span class="NLM_publisher-name">Bruker AXS Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, U.S.</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SAINT%2C+SADABS%2C+and+XPREP%3B+Bruker+AXS+Inc.%3A+Madison%2C+WI%2C+U.S.%2C+2007."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSAINT%26jtitle%3DSADABS%26jtitle%3DXPREP%26pub%3DBruker%2520AXS%2520Inc%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Fragment-based identification of druggable “hot spots” of proteins using Fourier domain correlation techniques</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fbioinformatics%2Fbtp036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19176554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=621-627&author=R.+Brenkeauthor=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=D.+Hallauthor=M.+R.+Landonauthor=C.+Mattosauthor=S.+Vajda&title=Fragment-based+identification+of+druggable+%E2%80%9Chot+spots%E2%80%9D+of+proteins+using+Fourier+domain+correlation+techniques&doi=10.1093%2Fbioinformatics%2Fbtp036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of druggable hot spots' of proteins using Fourier domain correlation techniques</span></div><div class="casAuthors">Brenke, Ryan; Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Hall, David; Landon, Melissa R.; Mattos, Carla; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">621-627</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The binding sites of proteins generally contain smaller regions that provide major contributions to the binding free energy and hence are the prime targets in drug design.  Screening libraries of fragment-sized compds. by NMR or X-ray crystallog. demonstrates that such hot spot' regions bind a large variety of small org. mols., and that a relatively high hit rate' is predictive of target sites that are likely to bind drug-like ligands with high affinity.  The authors' goal is to det. the hot spots' computationally rather than exptl.  The authors have developed the FTMAP algorithm that performs global search of the entire protein surface for regions that bind a no. of small org. probe mols.  The search is based on the extremely efficient fast Fourier transform (FFT) correlation approach which can sample billions of probe positions on dense translational and rotational grids, but can use only sums of correlation functions for scoring and hence is generally restricted to very simple energy expressions.  The novelty of FTMAP is that the authors were able to incorporate and represent on grids a detailed energy expression, resulting in a very accurate identification of low-energy probe clusters.  Overlapping clusters of different probes are defined as consensus sites (CSs).  The authors show that the largest CS is generally located at the most important subsite of the protein binding site, and the nearby smaller CSs identify other important subsites.  Mapping results are presented for elastase whose structure was solved in aq. solns. of 8 org. solvents, and they show that FTMAP provides very similar information.  The second application is to renin, a long-standing pharmaceutical target for the treatment of hypertension, and the authors show that the major CSs trace out the shape of the first approved renin inhibitor, aliskiren.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWSzH6Fg129bVg90H21EOLACvtfcHk0lhTF1v1WdWDOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D&md5=e5a466428f4dd47e89a4ae8361d3da41</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp036%26sid%3Dliteratum%253Aachs%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFragment-based%2520identification%2520of%2520druggable%2520%25E2%2580%259Chot%2520spots%25E2%2580%259D%2520of%2520proteins%2520using%2520Fourier%2520domain%2520correlation%2520techniques%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D621%26epage%3D627%26doi%3D10.1093%2Fbioinformatics%2Fbtp036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0liGYxji43WJ4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">FTSite: high accuracy detection of ligand binding sites on unbound protein structures</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fbioinformatics%2Fbtr651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=22113084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ans7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=286-7&author=C.+H.+Nganauthor=D.+R.+Hallauthor=B.+Zerbeauthor=L.+E.+Groveauthor=D.+Kozakovauthor=S.+Vajda&title=FTSite%3A+high+accuracy+detection+of+ligand+binding+sites+on+unbound+protein+structures&doi=10.1093%2Fbioinformatics%2Fbtr651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">FTSite: high accuracy detection of ligand binding sites on unbound protein structures</span></div><div class="casAuthors">Ngan, Chi-Ho; Hall, David R.; Zerbe, Brandon; Grove, Laurie E.; Kozakov, Dima; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-287</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Binding site identification is a classical problem that is important for a range of applications, including the structure-based prediction of function, the elucidation of functional relationships among proteins, protein engineering and drug design.  We describe an accurate method of binding site identification, namely FTSite.  This method is based on exptl. evidence that ligand binding sites also bind small org. mols. of various shapes and polarity.  The FTSite algorithm does not rely on any evolutionary or statistical information, but achieves near exptl. accuracy: it is capable of identifying the binding sites in over 94% of apo proteins from established test sets that have been used to evaluate many other binding site prediction methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri18NAnmz2g7Vg90H21EOLACvtfcHk0liGYxji43WJ4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ans7c%253D&md5=f30385a90b4e4c32905c9babc5827bff</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr651%26sid%3Dliteratum%253Aachs%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFTSite%253A%2520high%2520accuracy%2520detection%2520of%2520ligand%2520binding%2520sites%2520on%2520unbound%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D286%26epage%3D7%26doi%3D10.1093%2Fbioinformatics%2Fbtr651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0li4Srs3RZPVqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span> <i>Marvin</i>, version <span class="NLM_edition">6.1.0</span>; <span class="NLM_publisher-name">ChemAxon</span>, <span class="NLM_year">2013</span>; <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Marvin%2C+version+6.1.0%3B+ChemAxon%2C+2013%3B+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvin%26pub%3DChemAxon%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bapat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">988</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1124/jpet.110.169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fjpet.110.169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20504914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=988-998&author=A.+Bapatauthor=L.+S.+Glassauthor=M.+Luoauthor=M.+L.+Fishelauthor=E.+C.+Longauthor=M.+M.+Georgiadisauthor=M.+R.+Kelley&title=Novel+small-molecule+inhibitor+of+apurinic%2Fapyrimidinic+endonuclease+1+blocks+proliferation+and+reduces+viability+of+glioblastoma+cells&doi=10.1124%2Fjpet.110.169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span></div><div class="casAuthors">Bapat, Aditi; Glass, LaTeca S.; Luo, Meihua; Fishel, Melissa L.; Long, Eric C.; Georgiadis, Millie M.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic (AP) endonuclease 1 (Ape1) is an essential DNA repair protein that plays a crit. role in repair of AP sites via base excision repair.  Ape1 has received attention as a druggable oncotherapeutic target, esp. for treating intractable cancers such as glioblastoma.  The goal of this study was to identify small-mol. inhibitors of Ape1 AP endonuclease.  For this purpose, a fluorescence-based high-throughput assay was used to screen a library of 60,000 small-mol. compds. for ability to inhibit Ape1 AP endonuclease activity.  Four compds. with IC50 values less than 10 μM were identified, validated, and characterized.  One of the most promising compds., designated Ape1 repair inhibitor 03 (2,4,9-trimethylbenzo[b][1,8]-naphthyridin-5-amine; AR03), inhibited cleavage of AP sites in vivo in SF767 glioblastoma cells and in vitro in whole cell exts. and inhibited purified human Ape1 in vitro.  AR03 has low affinity for double-stranded DNA and weakly inhibits the Escherichia coli endonuclease IV, requiring a 20-fold higher concn. than for inhibition of Ape1.  AR03 also potentiates the cytotoxicity of Me methanesulfonate and temozolomide in SF767 cells.  AR03 is chem. distinct from the previously reported small-mol. inhibitors of Ape1.  AR03 is a novel small-mol. inhibitor of Ape1, which may have potential as an oncotherapeutic drug for treating glioblastoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaq7PkWFe6bVg90H21EOLACvtfcHk0li4Srs3RZPVqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J&md5=9fefb66d80fb0087db3b67755cd028c1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1124%2Fjpet.110.169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.110.169128%26sid%3Dliteratum%253Aachs%26aulast%3DBapat%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DL.%2BS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DLong%26aufirst%3DE.%2BC.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DNovel%2520small-molecule%2520inhibitor%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520blocks%2520proliferation%2520and%2520reduces%2520viability%2520of%2520glioblastoma%2520cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D988%26epage%3D998%26doi%3D10.1124%2Fjpet.110.169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zyromski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supuran, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of HIF1alpha under hypoxia by APE1/Ref-1 impacts CA9 expression: Dual targeting in patient-derived 3D pancreatic cancer models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2732</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1158%2F1535-7163.MCT-16-0253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27535970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2722-2732&author=D.+P.+Logsdonauthor=M.+Grimardauthor=M.+Luoauthor=S.+Shahdaauthor=Y.+Jiangauthor=Y.+Tongauthor=Z.+Yuauthor=N.+Zyromskiauthor=E.+Schipaniauthor=F.+Cartaauthor=C.+T.+Supuranauthor=M.+Korcauthor=M.+Ivanauthor=M.+R.+Kelleyauthor=M.+L.+Fishel&title=Regulation+of+HIF1alpha+under+hypoxia+by+APE1%2FRef-1+impacts+CA9+expression%3A+Dual+targeting+in+patient-derived+3D+pancreatic+cancer+models&doi=10.1158%2F1535-7163.MCT-16-0253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of HIF1α under hypoxia by APE1/Ref-1 impacts CA9 expression: dual targeting in patient-derived 3D pancreatic cancer models</span></div><div class="casAuthors">Logsdon, Derek P.; Grimard, Michelle; Luo, Meihua; Shahda, Safi; Jiang, Yanlin; Tong, Yan; Yu, Zhangsheng; Zyromski, Nicholas; Schipani, Ernestina; Carta, Fabrizio; Supuran, Claudiu T.; Korc, Murray; Ivan, Mircea; Kelley, Mark R.; Fishel, Melissa L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2722-2732</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer-related mortality in the United States.  Aggressive treatment regimens have not changed the disease course, and the median survival has just recently reached a year.  Several mechanisms are proposed to play a role in PDAC therapeutic resistance, including hypoxia, which creates a more aggressive phenotype with increased metastatic potential and impaired therapeutic efficacy.  AP Endonuclease-1/Redox Effector Factor 1 (APE1/Ref-1) is a multifunctional protein possessing a DNA repair function in base excision repair and the ability to reduce oxidized transcription factors, enabling them to bind to their DNA target sequences.  APE1/Ref-1 regulates several transcription factors involved in survival mechanisms, tumor growth, and hypoxia signaling.  Here, we explore the mechanisms underlying PDAC cell responses to hypoxia and modulation of APE1/Ref-1 redox signaling activity, which regulates the transcriptional activation of hypoxia-inducible factor 1 alpha (HIF1α).  Carbonic anhydrase IX (CA9) is regulated by HIF1α and functions as a part of the cellular response to hypoxia to regulate intracellular pH, thereby promoting cell survival.  We hypothesized that modulating APE1/Ref-1 function will block activation of downstream transcription factors, STAT3 and HIF1α, interfering with the hypoxia-induced gene expression.  We demonstrate APE1/Ref-1 inhibition in patient-derived and established PDAC cells results in decreased HIF1α-mediated induction of CA9.  Furthermore, an ex vivo three-dimensional tumor coculture model demonstrates dramatic enhancement of APE1/Ref-1-induced cell killing upon dual targeting of APE1/Ref-1 and CA9.  Both APE1/Ref-1 and CA9 are under clin. development; therefore, these studies have the potential to direct novel PDAC therapeutic treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGivvfAYcobbVg90H21EOLACvtfcHk0ljs2wq0QsPxSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsr4%253D&md5=741b7b96d5d93e9331a210590898112c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0253%26sid%3Dliteratum%253Aachs%26aulast%3DLogsdon%26aufirst%3DD.%2BP.%26aulast%3DGrimard%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DShahda%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DZyromski%26aufirst%3DN.%26aulast%3DSchipani%26aufirst%3DE.%26aulast%3DCarta%26aufirst%3DF.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DKorc%26aufirst%3DM.%26aulast%3DIvan%26aufirst%3DM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26atitle%3DRegulation%2520of%2520HIF1alpha%2520under%2520hypoxia%2520by%2520APE1%252FRef-1%2520impacts%2520CA9%2520expression%253A%2520Dual%2520targeting%2520in%2520patient-derived%25203D%2520pancreatic%2520cancer%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2722%26epage%3D2732%26doi%3D10.1158%2F1535-7163.MCT-16-0253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D. P.</span>; <span class="NLM_string-name">Cheng, H.</span>; <span class="NLM_string-name">Luo, M.</span>; <span class="NLM_string-name">Shahda, S.</span>; <span class="NLM_string-name">Ivan, M.</span>; <span class="NLM_string-name">Hao, Y.</span>; <span class="NLM_string-name">Tong, Y.</span>; <span class="NLM_string-name">Yu, Z.</span>; <span class="NLM_string-name">Zyromski, N.</span>; <span class="NLM_string-name">Schipani, E.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Supuran, C. T.</span>; <span class="NLM_string-name">Kelley, M. R.</span>; <span class="NLM_string-name">Fishel, M. L.</span></span> <span> </span><span class="NLM_article-title">Targeting APE1/Ref-1 results in inhibition of hypoxia signaling genes</span>. Presented at the AACR-NC-EORTC International Conference: Molecular Targets and Cancer Therapeutics, Boston, MA, December 2015;  <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">14</span>,  <span class="NLM_fpage">B158</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-B158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1158%2F1535-7163.TARG-15-B158" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=B158&author=D.+P.+Logsdon&author=H.+Cheng&author=M.+Luo&author=S.+Shahda&author=M.+Ivan&author=Y.+Hao&author=Y.+Tong&author=Z.+Yu&author=N.+Zyromski&author=E.+Schipani&author=Y.+Liu&author=C.+T.+Supuran&author=M.+R.+Kelley&author=M.+L.+Fishel&title=Targeting+APE1%2FRef-1+results+in+inhibition+of+hypoxia+signaling+genes&doi=10.1158%2F1535-7163.TARG-15-B158"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-B158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-B158%26sid%3Dliteratum%253Aachs%26aulast%3DLogsdon%26aufirst%3DD.%2BP.%26atitle%3DTargeting%2520APE1%252FRef-1%2520results%2520in%2520inhibition%2520of%2520hypoxia%2520signaling%2520genes%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DB158%26doi%3D10.1158%2F1535-7163.TARG-15-B158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David M.  Wilson</span>, <span class="hlFld-ContribAuthor ">Ashley M.  Deacon</span>, <span class="hlFld-ContribAuthor ">Matthew A.J.  Duncton</span>, <span class="hlFld-ContribAuthor ">Patricia  Pellicena</span>, <span class="hlFld-ContribAuthor ">Millie M.  Georgiadis</span>, <span class="hlFld-ContribAuthor ">Andrew P.  Yeh</span>, <span class="hlFld-ContribAuthor ">Andrew S.  Arvai</span>, <span class="hlFld-ContribAuthor ">Davide  Moiani</span>, <span class="hlFld-ContribAuthor ">John A.  Tainer</span>, <span class="hlFld-ContribAuthor ">Debanu  Das</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment- and structure-based drug discovery for developing therapeutic agents targeting the DNA Damage Response. </span><span class="cited-content_cbyCitation_journal-name">Progress in Biophysics and Molecular Biology</span><span> <strong>2021,</strong> <em>163 </em>, 130-142. <a href="https://doi.org/10.1016/j.pbiomolbio.2020.10.005" title="DOI URL">https://doi.org/10.1016/j.pbiomolbio.2020.10.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pbiomolbio.2020.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pbiomolbio.2020.10.005%26sid%3Dliteratum%253Aachs%26jtitle%3DProgress%2520in%2520Biophysics%2520and%2520Molecular%2520Biology%26atitle%3DFragment-%252Band%252Bstructure-based%252Bdrug%252Bdiscovery%252Bfor%252Bdeveloping%252Btherapeutic%252Bagents%252Btargeting%252Bthe%252BDNA%252BDamage%252BResponse%26aulast%3DWilson%26aufirst%3DDavid%2BM.%26date%3D2021%26volume%3D163%26spage%3D130%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel A.  Caston</span>, <span class="hlFld-ContribAuthor ">Silpa  Gampala</span>, <span class="hlFld-ContribAuthor ">Lee  Armstrong</span>, <span class="hlFld-ContribAuthor ">Richard A.  Messmann</span>, <span class="hlFld-ContribAuthor ">Melissa L.  Fishel</span>, <span class="hlFld-ContribAuthor ">Mark R.  Kelley</span>. </span><span class="cited-content_cbyCitation_article-title">The multifunctional APE1 DNA repair–redox signaling protein as a drug target in human disease. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>26 </em>
                                    (1)
                                     , 218-228. <a href="https://doi.org/10.1016/j.drudis.2020.10.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.10.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.10.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Bmultifunctional%252BAPE1%252BDNA%252Brepair%2525E2%252580%252593redox%252Bsignaling%252Bprotein%252Bas%252Ba%252Bdrug%252Btarget%252Bin%252Bhuman%252Bdisease%26aulast%3DCaston%26aufirst%3DRachel%2BA.%26date%3D2021%26volume%3D26%26issue%3D1%26spage%3D218%26epage%3D228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander V.  Popov</span>, <span class="hlFld-ContribAuthor ">Inga R.  Grin</span>, <span class="hlFld-ContribAuthor ">Antonina P.  Dvornikova</span>, <span class="hlFld-ContribAuthor ">Bakhyt T.  Matkarimov</span>, <span class="hlFld-ContribAuthor ">Regina  Groisman</span>, <span class="hlFld-ContribAuthor ">Murat  Saparbaev</span>, <span class="hlFld-ContribAuthor ">Dmitry O.  Zharkov</span>. </span><span class="cited-content_cbyCitation_article-title">Reading Targeted DNA Damage in the Active Demethylation Pathway: Role of Accessory Domains of Eukaryotic AP Endonucleases and Thymine-DNA Glycosylases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2020,</strong> <em>432 </em>
                                    (6)
                                     , 1747-1768. <a href="https://doi.org/10.1016/j.jmb.2019.12.020" title="DOI URL">https://doi.org/10.1016/j.jmb.2019.12.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2019.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2019.12.020%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DReading%252BTargeted%252BDNA%252BDamage%252Bin%252Bthe%252BActive%252BDemethylation%252BPathway%25253A%252BRole%252Bof%252BAccessory%252BDomains%252Bof%252BEukaryotic%252BAP%252BEndonucleases%252Band%252BThymine-DNA%252BGlycosylases%26aulast%3DPopov%26aufirst%3DAlexander%2BV.%26date%3D2020%26volume%3D432%26issue%3D6%26spage%3D1747%26epage%3D1768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystallographic solvent mapping. (A) A molecular surface rendering of APE1, the DMSO bound structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>), with C, light gray, O, red, N, blue, and S, yellow. Superimposed on the molecular surface of APE1 is DNA (from 4IEM) in a stick rendering, C, cyan, P, orange, O, red, and N, blue with the abasic or AP residue in yellow. (B) DMSO bound to the abasic site of APE1. The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 3σ is shown for the crystal structure with DMSO bound to the AP site. A hydrogen bond between the O of DMSO and NH of N174 is indicated by a black dashed line. APE1 is shown as a semitransparent molecular surface rendering. Residues W280, F266, L282 (not labeled, directly underneath the bound DMSO), and N174 line the shallow AP site pocket. (C) DMSO (C, green, S, yellow, O, red) is shown superimposed with the abasic substrate (C, cyan) in stick renderings. (D) DMSO and Mg<sup>2+</sup> bound to the active site of APE1. The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map of the DMSO/Mg<sup>2+</sup> crystal structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>) is shown for DMSO and Mg<sup>2+</sup> contoured at 3σ. Coordinating ligands for Mg<sup>2+</sup> are shown in dashed black lines including D70, E96, and four water molecules. The DMSO is this structure and is not hydrogen bonded to N174. (E) DMSO and DMSO/Mg<sup>2+</sup> are shown superimposed in the abasic site pocket on a surface rendering of APE1. In the presence of Mg<sup>2+</sup>, the DMSO (C, blue) position is slightly shifted relative to that in the structure with just DMSO (C, green) bound. (F) Tris and DMSO bound to the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>). The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for Tris and DMSO is shown contoured at 3σ. Tris is hydrogen bonded to D70 and E96, the Mg<sup>2+</sup> coordinating ligands. (G) Tris/DMSO is shown in stick renderings superimposed on the DMSO bound structure. Mg<sup>2+</sup> from the DMSO/Mg<sup>2+</sup> structure is shown for reference. Tris binds to the Mg<sup>2+</sup> site. DMSO binds very similarly in the two structures. (H) Glycerol bound to the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>). The initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map is shown contoured at 3σ. Hydrogen bonds between OH group in glycerol and NH in N174 and N212 are shown in dashed black lines. (I) Glycerol (C, blue) is shown superimposed with DMSO and Mg<sup>2+</sup> from the DMSO/Mg<sup>2+</sup> structure for reference. Glycerol binds deeper in the abasic site pocket than DMSO.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Computational solvent mapping with FTMap ((<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>). (A) Computational docking of small organic molecules reveals clustering of 50 probes (thin lines, C, cyan, O, red, N blue) into seven consensus sites in or near the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>) shown as a molecular surface rendering (C, light gray, O, red, N, blue, and S, yellow). DMSO (stick model C, green, O, red, S, yellow) from the crystal structure is shown bound in consensus site 2. Probes at consensus site 1, the most populated site, are in close proximity to the Mg<sup>2+</sup> ion, shown as a surface rendering in green. (B) A secondary site occupied by ethylene glycol in several crystal structures of APE1 is shown with a cluster of 21 probes shown in thin lines. Ethylene glycols bound in this site and in the vicinity are shown as stick renderings (C, magenta, O, red). This is a small pocket on the surface of the molecule. Thus, the major consensus sites for computational docking of probes in the active of APE1 and in another pocket have been validated in our crystallographic analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0010.jpeg" id="rightTab-GRAPHIC-d7e1126-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Design and Synthetic Route for Macrocyclic Lactone Library<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BF<sub>3</sub>–THF, −78 °C to −20 °C; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,3-dimethylbarbituric acid, THF; (c) EDCI, <i>i</i>-Pr<sub>2</sub>EtNH, DMF or CH<sub>2</sub>Cl<sub>2</sub>; (d) HCl, dioxane; (e) Otera’s catalyst, PhCF<sub>3</sub>. Depicted stereochemistry corresponds to use of (<i>S</i>)-<b>2</b> in step a. Intermediates <b>4</b>, <b>5</b>, <b>7</b>, <b>9</b> and macrocycles <b>MC</b> derived from crotylsilane (<i>R</i>)-<b>2</b> are epimeric at the methyl- and R-substituted positions.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Initial hits from high-throughput screen of the macrocycle library. The chemical structures of the four hits obtained from screening of 66 macrocycles in an in vitro endonuclease activity are shown along with the IC<sub>50</sub> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking of macrocycle hit <b>MC047</b>. (A) The two diastereomers of <b>MC047</b> are shown as stick models with α-<i>S</i> (C, green) and α-<i>R</i> (C, cyan) for mode 1 of the possible docking poses. APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>) is shown as a molecular surface rendering. In docking mode 1, <b>MC047</b> diastereomers occupy 6 out of 7 hot spots in the main binding site. The difference in binding for the two diastereomers is mainly due to a slight change in positioning of the tryptophan indole. The orientation is similar to that shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> as can be seen by the relative positions of M270 and R177. (B) In binding mode 1, the position of the bicyclic heterocycle of <b>MC047</b> (C, green) coincides with that of a guanine nucleotide from DNA (thin lines, C, cyan) bound to APE1 (1DE9) superimposed on the apo APE1 structure. APE1 is shown as a molecular surface rendering. (C) The two diastereomers of <b>MC04</b>7 are shown in mode 2 as similar renderings to those in (A), α-<i>S</i> (C, green), and α-<i>R</i> (C, cyan). In this binding mode, the sulfonamide group is in close proximity to the Mg<sup>2+</sup> ion (green on the molecular surface rendering). The mode 2 pose was only obtained for the <i>N</i>-tosylpyrrole-containing macrocycles and occupies 5 out of 7 hot spots in the main binding site. For <b>MC047</b>, the trifluoromethyl group of the α-<i>S</i> isomer extends toward an additional hot spot (number 5 in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), while in the α-<i>R</i> pose this group interacts with the terminal carbon of Met270. (D) <b>MC047</b> α-<i>S</i> (C, green) is shown with superimposed DNA (1DE9). In this mode, the lactone oxygens are situated proximal to where the DNA phosphate group binds the Mg<sup>2+</sup> ion near hot spot region 2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0011.jpeg" id="rightTab-GRAPHIC-d7e2433-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Simplified Follow-On Macrolactone Analogs</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0012.jpeg" id="rightTab-GRAPHIC-d7e2539-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthetic Route toward Second-Generation Macrocyclic Lactams<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCl/dioxane; (b) HATU, <i>i</i>-Pr<sub>2</sub>EtNH, CH<sub>2</sub>Cl<sub>2</sub>; (c) 2.5 M NaOH, THF; (d) HCl, dioxane; (e) HATU, <i>i</i>-Pr<sub>2</sub>EtNH, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay of “mode 2” docking poses obtained for stick renderings of <b>MC047</b> (C, light gray), <b>11</b> (C, teal), <b>25</b> (C, dark green), and <b>26</b> (C, pink) in the active site of APE1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE">4QHE</a>). Compounds <b>MC047</b>, <b>25</b>, and <b>26</b> all display an interaction between the sulfonamide groups and the magnesium ion (light green sphere), as well as a projection of their halogenated aryl rings toward hot spot region 5, left-hand side.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/medium/jm-2018-01529y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Macrocycle treatment alone and in combination with DNA damaging agent methyl methanesulfonate (MMS) in malignant peripheral nerve sheath tumor cell line ST8814 in alkaline CometAssay. APE1 repair inhibitor candidates (macrocycles) and APE1 repair inhibitor control, APE1 repair inhibitor III (ARiIII; Calbiochem), were tested in ST8814 at a single dose (100 μM or EC<sub>30</sub>) and in combination with a single dose of MMS (1.0 mM) for a 1.0 h treatment. Cells were seeded in 6-well tissue culture plates at 130 000 cells/well in DMEM + 10% FBS and grown overnight at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with Opti-MEM (Gibco) medium containing macrocycle alone or spiked with 1.0 mM MMS. Cells were then incubated for 1.0 h at 37 °C, 5% CO<sub>2</sub>. Medium was exchanged with PBS, and cells were treated with 0.25% trypsin (HyClone), collected, and washed with PBS. Cells were then counted by hemacytometer. DNA damage was evaluated by CometAssay (Trevigen) performed under alkaline conditions. Cells were resuspended in PBS at 1 × 10<sup>5</sup>/mL and then added to premelted and cooled (37 °C) agarose at a 1:10 ratio. Cells were gently mixed, and then 50 μL of the agarose cell mix was transferred to prewarmed comet slides (37 °C, CometSlide). After solidifying at 4 °C, slides were placed in lysis solution for 60 min and then placed in freshly prepared alkaline unwinding solution (NaOH, pH 13) for 20 min. Slides were then subjected to electrophoresis under alkaline conditions (NaOH, pH 13) at 1 V/cm (300 mA) for 30 min. Slides were washed in H<sub>2</sub>O, placed in 70% ethanol, and allowed to dry at 37 °C for 30 min. Slides were stained with 100 μL of 1:10 000 diluted SYBR Gold (Invitrogen) in TE pH 7.5 and incubated for 30 min. Slides were then rinsed in H<sub>2</sub>O briefly, and comets were captured by fluorescent microscope (Leica DMIL) and quantified by CometScore Pro (TriTec Corp). DNA damage measured by percent Tail DNA for controls and selected macrocycles is shown in (A) in two independent experiments. Representative images of comet slide DNA damage are shown in (B) for controls and selected macrocycles. There was an average of 20 comet readings/compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-4/acs.jmedchem.8b01529/20190221/images/large/jm-2018-01529y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01529&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messmann, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehrenbacher, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Exploiting the Ref-1-APE1 node in cancer signaling and other diseases: from bench to clinic</span>. <i>NPJ Precis. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41698-017-0023-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fs41698-017-0023-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=29872691" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=1-19&author=F.+Shahauthor=D.+Logsdonauthor=R.+A.+Messmannauthor=J.+C.+Fehrenbacherauthor=M.+L.+Fishelauthor=M.+R.+Kelley&title=Exploiting+the+Ref-1-APE1+node+in+cancer+signaling+and+other+diseases%3A+from+bench+to+clinic&doi=10.1038%2Fs41698-017-0023-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41698-017-0023-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41698-017-0023-0%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DF.%26aulast%3DLogsdon%26aufirst%3DD.%26aulast%3DMessmann%26aufirst%3DR.%2BA.%26aulast%3DFehrenbacher%26aufirst%3DJ.%2BC.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DExploiting%2520the%2520Ref-1-APE1%2520node%2520in%2520cancer%2520signaling%2520and%2520other%2520diseases%253A%2520from%2520bench%2520to%2520clinic%26jtitle%3DNPJ%2520Precis.%2520Oncol.%26date%3D2017%26volume%3D1%26spage%3D1%26epage%3D19%26doi%3D10.1038%2Fs41698-017-0023-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, R.</span></span> <span> </span><span class="NLM_article-title">APE1 overexpression is associated with poor survival in patients with solid tumors: a meta-analysis</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">59720</span>– <span class="NLM_lpage">59728</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.18632%2Foncotarget.19814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=59720-59728&author=C.+L.+Yuanauthor=F.+Heauthor=J.+Z.+Yeauthor=H.+N.+Wuauthor=J.+Y.+Zhangauthor=Z.+H.+Liuauthor=Y.+Q.+Liauthor=X.+L.+Luoauthor=Y.+Linauthor=R.+Liang&title=APE1+overexpression+is+associated+with+poor+survival+in+patients+with+solid+tumors%3A+a+meta-analysis&doi=10.18632%2Foncotarget.19814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19814%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DC.%2BL.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DYe%26aufirst%3DJ.%2BZ.%26aulast%3DWu%26aufirst%3DH.%2BN.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DY.%2BQ.%26aulast%3DLuo%26aufirst%3DX.%2BL.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DR.%26atitle%3DAPE1%2520overexpression%2520is%2520associated%2520with%2520poor%2520survival%2520in%2520patients%2520with%2520solid%2520tumors%253A%2520a%2520meta-analysis%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D59720%26epage%3D59728%26doi%3D10.18632%2Foncotarget.19814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juhnke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heumann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirico, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoflmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hube-Magg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moller-Koop, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thederan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlomm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luebke, A. M.</span></span> <span> </span><span class="NLM_article-title">Apurinic/apyrimidinic endonuclease 1 (APE1/Ref-1) overexpression is an independent prognostic marker in prostate cancer without TMPRSS2:ERG fusion</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2135</span>– <span class="NLM_lpage">2145</span>, <span class="refDoi"> DOI: 10.1002/mc.22670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2Fmc.22670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=28467610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=2135-2145&author=M.+Juhnkeauthor=A.+Heumannauthor=V.+Chiricoauthor=D.+Hoflmayerauthor=A.+Menzauthor=A.+Hinschauthor=C.+Hube-Maggauthor=M.+Kluthauthor=D.+S.+Langauthor=C.+Moller-Koopauthor=G.+Sauterauthor=R.+Simonauthor=B.+Beyerauthor=R.+Pompeauthor=I.+Thederanauthor=T.+Schlommauthor=A.+M.+Luebke&title=Apurinic%2Fapyrimidinic+endonuclease+1+%28APE1%2FRef-1%29+overexpression+is+an+independent+prognostic+marker+in+prostate+cancer+without+TMPRSS2%3AERG+fusion&doi=10.1002%2Fmc.22670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Apurinic/apyrimidinic endonuclease 1 (APE1/Ref-1) overexpression is an independent prognostic marker in prostate cancer without TMPRSS2:ERG fusion</span></div><div class="casAuthors">Juhnke, Manuela; Heumann, Asmus; Chirico, Viktoria; Hoeflmayer, Doris; Menz, Anne; Hinsch, Andrea; Hube-Magg, Claudia; Kluth, Martina; Lang, Dagmar S.; Moeller-Koop, Christina; Sauter, Guido; Simon, Ronald; Beyer, Burkhard; Pompe, Raisa; Thederan, Imke; Schlomm, Thorsten; Luebke, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2135-2145</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Polymorphisms of the base excision repair gene APE1 may be assocd. with an increased risk for developing prostate cancer.  In other cancer types, altered APE1 protein expression is a candidate prognostic marker.  Using immunohistochem., we thus analyzed APE1 expression in 9763 prostate cancers in a tissue microarray (TMA) with attached clin. and mol. data.  The comparison with normal prostate tissue revealed an upregulation of APE1 in cancer samples.  APE1 immunostaining was considered weak in 20.2%, moderate in 36.7%, and strong in 33.4% of cancers.  Strong APE1 expression was markedly more frequent in prostate cancers harboring the TMPRSS2:ERG fusion (52.9%) than in ERG-neg. cancers (19.1%, P < 0.0001).  Significant assocns. with Gleason grade, tumor stage, tumor grade, and early biochem. recurrence (P < 0.0001 each) were largely limited to ERG-neg. tumors.  Multivariable anal. revealed that the prognostic value of APE1 upregulation in ERG-neg. prostate cancers was independent from established histopathol. and clin. parameters.  In conclusion, the results of our study demonstrate that APE1 overexpression is an independent prognostic marker, but exclusively in ERG-neg. prostate cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp18rsnuvOO_rVg90H21EOLACvtfcHk0lgu9QAUFrCouQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKns7Y%253D&md5=be3eb0f6a4a6360f4d6ac90b7b016c65</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fmc.22670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.22670%26sid%3Dliteratum%253Aachs%26aulast%3DJuhnke%26aufirst%3DM.%26aulast%3DHeumann%26aufirst%3DA.%26aulast%3DChirico%26aufirst%3DV.%26aulast%3DHoflmayer%26aufirst%3DD.%26aulast%3DMenz%26aufirst%3DA.%26aulast%3DHinsch%26aufirst%3DA.%26aulast%3DHube-Magg%26aufirst%3DC.%26aulast%3DKluth%26aufirst%3DM.%26aulast%3DLang%26aufirst%3DD.%2BS.%26aulast%3DMoller-Koop%26aufirst%3DC.%26aulast%3DSauter%26aufirst%3DG.%26aulast%3DSimon%26aufirst%3DR.%26aulast%3DBeyer%26aufirst%3DB.%26aulast%3DPompe%26aufirst%3DR.%26aulast%3DThederan%26aufirst%3DI.%26aulast%3DSchlomm%26aufirst%3DT.%26aulast%3DLuebke%26aufirst%3DA.%2BM.%26atitle%3DApurinic%252Fapyrimidinic%2520endonuclease%25201%2520%2528APE1%252FRef-1%2529%2520overexpression%2520is%2520an%2520independent%2520prognostic%2520marker%2520in%2520prostate%2520cancer%2520without%2520TMPRSS2%253AERG%2520fusion%26jtitle%3DMol.%2520Carcinog.%26date%3D2017%26volume%3D56%26spage%3D2135%26epage%3D2145%26doi%3D10.1002%2Fmc.22670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">The expression of APE1 in triple-negative breast cancer and its effect on drug sensitivity of olaparib</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1010428317713390</span>, <span class="refDoi"> DOI: 10.1177/1010428317713390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1177%2F1010428317713390" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=1010428317713390&author=T.+Chenauthor=C.+Liuauthor=H.+Luauthor=M.+Yinauthor=C.+Shaoauthor=X.+Huauthor=J.+Wuauthor=Y.+Wang&title=The+expression+of+APE1+in+triple-negative+breast+cancer+and+its+effect+on+drug+sensitivity+of+olaparib&doi=10.1177%2F1010428317713390"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1010428317713390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1010428317713390%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DThe%2520expression%2520of%2520APE1%2520in%2520triple-negative%2520breast%2520cancer%2520and%2520its%2520effect%2520on%2520drug%2520sensitivity%2520of%2520olaparib%26jtitle%3DTumor%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D1010428317713390%26doi%3D10.1177%2F1010428317713390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The expression profile and prognostic value of APE/Ref-1 and NPM1 in high-grade serous ovarian adenocarcinoma</span>. <i>APMIS</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1111/apm.12733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1111%2Fapm.12733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=28766835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=857-862&author=X.+Fanauthor=L.+Wenauthor=Y.+Liauthor=L.+Louauthor=W.+Liuauthor=J.+Zhang&title=The+expression+profile+and+prognostic+value+of+APE%2FRef-1+and+NPM1+in+high-grade+serous+ovarian+adenocarcinoma&doi=10.1111%2Fapm.12733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The expression profile and prognostic value of APE/Ref-1 and NPM1 in high-grade serous ovarian adenocarcinoma</span></div><div class="casAuthors">Fan, Xiaomei; Wen, Lixuan; Li, Yuehong; Lou, Lei; Liu, Weina; Zhang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">857-862</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">1600-0463</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To analyze the expression trends and clin. significance of Apurinic/Apyrimidinic Endodeoxyribonuclease 1 (APE1/Ref-1) and Nucleophosmin (NPM1) proteins in high-grade serous ovarian adenocarcinoma (HGSC).  The expressions of APE1/Ref-1 and NPM1 proteins in 94 patients with HGSC were detd. using the immunohistochem. (IHC) method, and their relationships with clinicopathol. features were analyzed by the χ2 test or Fisher's exact test.  The follow-up data, Cox proportional hazards univariate and multivariate survival analyses were integrated to evaluate the prognostic factors affecting patients with HGSC.  In the normal fallopian tubes, APE1/Ref-1 and NPM1 protein were mainly distributed in the nuclear.  The HGSC experienced changes in the cellular localization of APE1/Ref-1 and NPM1 protein expressions, which were abnormally expressed in the cytoplasm.  The rates of abnormal cytoplasmic expression of APE1/Ref-1 and NPM1 proteins in 94 patients with HGSC were 69.1% and 73.4%, resp., which were significantly higher than the normal fallopian tube tissues (p < 0.05).  The abnormal cytoplasmic APE1/Ref-1 and NPM1 are significantly correlated with the lymph node metastasis, chemosensitivity, FIGO staging, and prognosis.  The COX multivariate survival anal. showed that the abnormal expression of APE1/Ref-1 protein, FIGO staging, and lymph node metastasis are independent prognostic factors.  Collectively, the abnormal cytoplasmic APE1/Ref-1 and NPM1 proteins are assocd. with the oncogenic progression and chemoresistance of HGSC, and predict a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8XAgdJUSZrVg90H21EOLACvtfcHk0lgR0IJRAcA1lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtr3E&md5=2656395d77f2ce015e6ab4ca2754de54</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fapm.12733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fapm.12733%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520expression%2520profile%2520and%2520prognostic%2520value%2520of%2520APE%252FRef-1%2520and%2520NPM1%2520in%2520high-grade%2520serous%2520ovarian%2520adenocarcinoma%26jtitle%3DAPMIS%26date%3D2017%26volume%3D125%26spage%3D857%26epage%3D862%26doi%3D10.1111%2Fapm.12733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Human apurinic/apyrimidinic endonuclease 1</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1089/ars.2013.5492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1089%2Fars.2013.5492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=23834463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsFynuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=678-707&author=M.+Liauthor=D.+M.+Wilson&title=Human+apurinic%2Fapyrimidinic+endonuclease+1&doi=10.1089%2Fars.2013.5492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Human Apurinic/Apyrimidinic Endonuclease 1</span></div><div class="casAuthors">Li, Mengxia; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">678-707</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Significance: Human apurinic/apyrimidinic endonuclease 1 (APE1, also known as REF-1) was isolated based on its ability to cleave at AP sites in DNA or activate the DNA binding activity of certain transcription factors.  We review herein topics related to this multi-functional DNA repair and stress-response protein.  Recent Advances: APE1 displays homol. to Escherichia coli exonuclease III and is a member of the divalent metal-dependent α/β fold-contg. phosphoesterase superfamily of enzymes.  APE1 has acquired distinct active site and loop elements that dictate substrate selectivity, and a unique N-terminus which at min. imparts nuclear targeting and interaction specificity.  Addnl. activities ascribed to APE1 include 3'-5' exonuclease, 3'-repair diesterase, nucleotide incision repair, damaged or site-specific RNA cleavage, and multiple transcription regulatory roles.  Crit. Issues: APE1 is essential for mouse embryogenesis and contributes to cell viability in a genetic background-dependent manner.  Haploinsufficient APE1+/- mice exhibit reduced survival, increased cancer formation, and cellular/tissue hyper-sensitivity to oxidative stress, supporting the notion that impaired APE1 function assocs. with disease susceptibility.  Although abnormal APE1 expression/localization has been seen in cancer and neuropathologies, and impaired-function variants have been described, a causal link between an APE1 defect and human disease remains elusive.  Future Directions: Ongoing efforts aim at delineating the biol. role(s) of the different APE1 activities, as well as the regulatory mechanisms for its intra-cellular distribution and participation in diverse mol. pathways.  The detn. of whether APE1 defects contribute to human disease, particularly pathologies that involve oxidative stress, and whether APE1 small-mol. regulators have clin. utility, is central to future investigations.  Antioxid. Redox Signal. 20, 678-707.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAm6RzdhioobVg90H21EOLACvtfcHk0lgR0IJRAcA1lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsFynuw%253D%253D&md5=ded413cf1790fd14e73629fefce7913a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2013.5492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2013.5492%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DHuman%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2014%26volume%3D20%26spage%3D678%26epage%3D707%26doi%3D10.1089%2Fars.2013.5492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimpton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houlbrook, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammonds, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freemont, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dianov, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">Isolation of a small molecule inhibitor of DNA base excision repair</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">4711</span>– <span class="NLM_lpage">4724</span>, <span class="refDoi"> DOI: 10.1093/nar/gki781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fnar%2Fgki781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=16113242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=4711-4724&author=S.+Madhusudanauthor=F.+Smartauthor=P.+Shrimptonauthor=J.+L.+Parsonsauthor=L.+Gardinerauthor=S.+Houlbrookauthor=D.+C.+Talbotauthor=T.+Hammondsauthor=P.+A.+Freemontauthor=M.+J.+Sternbergauthor=G.+L.+Dianovauthor=I.+D.+Hickson&title=Isolation+of+a+small+molecule+inhibitor+of+DNA+base+excision+repair&doi=10.1093%2Fnar%2Fgki781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a small molecule inhibitor of DNA base excision repair</span></div><div class="casAuthors">Madhusudan, Srinivasan; Smart, Fiona; Shrimpton, Paul; Parsons, Jason L.; Gardiner, Laurence; Houlbrook, Sue; Talbot, Denis C.; Hammonds, Timothy; Freemont, Paul A.; Sternberg, Michael J. E.; Dianov, Grigory L.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4711-4724</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The base excision repair (BER) pathway is essential for the removal of DNA bases damaged by alkylation or oxidn.  A key step in BER is the processing of an apurinic/apyrimidinic (AP) site intermediate by an AP endonuclease.  The major AP endonuclease in human cells (APE1, also termed HAP1 and Ref-1) accounts for >95% of the total AP endonuclease activity, and is essential for the protection of cells against the toxic effects of several classes of DNA damaging agents.  Moreover, APE1 overexpression has been linked to radio- and chemo-resistance in human tumors.  Using a newly developed high-throughput screen, several chem. inhibitors of APE1 have been isolated.  Amongst these, CRT0044876 was identified as a potent and selective APE1 inhibitor.  CRT0044876 inhibits the AP endonuclease, 3'-phosphodiesterase and 3'-phosphatase activities of APE1 at low micromolar concns., and is a specific inhibitor of the exonuclease III family of enzymes to which APE1 belongs.  At non-cytotoxic concns., CRT0044876 potentiates the cytotoxicity of several DNA base-targeting compds.  This enhancement of cytotoxicity is assocd. with an accumulation of unrepaired AP sites.  In silico modeling studies suggest that CRT0044876 binds to the active site of APE1.  These studies provide both a novel reagent for probing APE1 function in human cells, and a rational basis for the development of APE1-targeting drugs for antitumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo35mTihPN15bVg90H21EOLACvtfcHk0lgR0IJRAcA1lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D&md5=01a434b054d60d670d683dc0fa38e649</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki781%26sid%3Dliteratum%253Aachs%26aulast%3DMadhusudan%26aufirst%3DS.%26aulast%3DSmart%26aufirst%3DF.%26aulast%3DShrimpton%26aufirst%3DP.%26aulast%3DParsons%26aufirst%3DJ.%2BL.%26aulast%3DGardiner%26aufirst%3DL.%26aulast%3DHoulbrook%26aufirst%3DS.%26aulast%3DTalbot%26aufirst%3DD.%2BC.%26aulast%3DHammonds%26aufirst%3DT.%26aulast%3DFreemont%26aufirst%3DP.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DDianov%26aufirst%3DG.%2BL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DIsolation%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520DNA%2520base%2520excision%2520repair%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D4711%26epage%3D4724%26doi%3D10.1093%2Fnar%2Fgki781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiple, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardellina, J. H.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span> <span> </span><span class="NLM_article-title">Potent inhibition of human apurinic/apyrimidinic endonuclease 1 by arylstibonic acids</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1124/mol.107.042622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fmol.107.042622" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2007&pages=669-677&author=L.+A.+Seipleauthor=J.+H.+Cardellinaauthor=R.+Akeeauthor=J.+T.+Stivers&title=Potent+inhibition+of+human+apurinic%2Fapyrimidinic+endonuclease+1+by+arylstibonic+acids&doi=10.1124%2Fmol.107.042622"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.042622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.042622%26sid%3Dliteratum%253Aachs%26aulast%3DSeiple%26aufirst%3DL.%2BA.%26aulast%3DCardellina%26aufirst%3DJ.%2BH.%26aulast%3DAkee%26aufirst%3DR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26atitle%3DPotent%2520inhibition%2520of%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520by%2520arylstibonic%2520acids%26jtitle%3DMol.%2520Pharmacol.%26date%3D2007%26volume%3D73%26spage%3D669%26epage%3D677%26doi%3D10.1124%2Fmol.107.042622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zawahir, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Pharmacophore guided discovery of small-molecule human apurinic/apyrimidinic endonuclease 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1021/jm800739m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800739m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFSjtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=20-32&author=Z.+Zawahirauthor=R.+Dayamauthor=J.+Dengauthor=C.+Pereiraauthor=N.+Neamati&title=Pharmacophore+guided+discovery+of+small-molecule+human+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm800739m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore Guided Discovery of Small-Molecule Human Apurinic/Apyrimidinic Endonuclease 1 Inhibitors</span></div><div class="casAuthors">Zawahir, Zahrah; Dayam, Raveendra; Deng, Jinxia; Pereira, Cherelene; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-32</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human apurinic/apyrimidinic endonuclease 1 (APE1) is an important enzyme in the base excision repair (BER) pathway that is essential for the repair of abasic sites in the genome.  Evidence for APE1 as an attractive therapeutic target in anticancer drug development has been demonstrated by studies that link overexpression of APE1 in many cancers to resistance of tumor cells to radio- and chemotherapy.  APE1 also shows a protective effect in several cancer cell models to a variety of DNA damaging agents.  This study represents the first rational design of selective small-mol. APE1 inhibitors utilizing a three-dimensional interaction-based pharmacophore perception.  All of our most potent mols. show inhibitory activity below 10 μM and are selective for APE1 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHAjB_Hy-QWbVg90H21EOLACvtfcHk0lh5XILM7B7QOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFSjtbvM&md5=9f5209c466ce2203eafa7195a772bb7f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm800739m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800739m%26sid%3Dliteratum%253Aachs%26aulast%3DZawahir%26aufirst%3DZ.%26aulast%3DDayam%26aufirst%3DR.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DC.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DPharmacophore%2520guided%2520discovery%2520of%2520small-molecule%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D20%26epage%3D32%26doi%3D10.1021%2Fjm800739m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bapat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Novel small molecule inhibitor of Ape1 endonuclease blocks proliferation and reduces viability of glioblastoma cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">988</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1124/jpet.110.169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fjpet.110.169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20504914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=988-998&author=A.+Bapatauthor=L.+S.+Glassauthor=M.+Luoauthor=M.+L.+Fishelauthor=E.+C.+Longauthor=M.+M.+Georgiadisauthor=M.+R.+Kelley&title=Novel+small+molecule+inhibitor+of+Ape1+endonuclease+blocks+proliferation+and+reduces+viability+of+glioblastoma+cells&doi=10.1124%2Fjpet.110.169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span></div><div class="casAuthors">Bapat, Aditi; Glass, LaTeca S.; Luo, Meihua; Fishel, Melissa L.; Long, Eric C.; Georgiadis, Millie M.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic (AP) endonuclease 1 (Ape1) is an essential DNA repair protein that plays a crit. role in repair of AP sites via base excision repair.  Ape1 has received attention as a druggable oncotherapeutic target, esp. for treating intractable cancers such as glioblastoma.  The goal of this study was to identify small-mol. inhibitors of Ape1 AP endonuclease.  For this purpose, a fluorescence-based high-throughput assay was used to screen a library of 60,000 small-mol. compds. for ability to inhibit Ape1 AP endonuclease activity.  Four compds. with IC50 values less than 10 μM were identified, validated, and characterized.  One of the most promising compds., designated Ape1 repair inhibitor 03 (2,4,9-trimethylbenzo[b][1,8]-naphthyridin-5-amine; AR03), inhibited cleavage of AP sites in vivo in SF767 glioblastoma cells and in vitro in whole cell exts. and inhibited purified human Ape1 in vitro.  AR03 has low affinity for double-stranded DNA and weakly inhibits the Escherichia coli endonuclease IV, requiring a 20-fold higher concn. than for inhibition of Ape1.  AR03 also potentiates the cytotoxicity of Me methanesulfonate and temozolomide in SF767 cells.  AR03 is chem. distinct from the previously reported small-mol. inhibitors of Ape1.  AR03 is a novel small-mol. inhibitor of Ape1, which may have potential as an oncotherapeutic drug for treating glioblastoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaq7PkWFe6bVg90H21EOLACvtfcHk0lh5XILM7B7QOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J&md5=9fefb66d80fb0087db3b67755cd028c1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1124%2Fjpet.110.169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.110.169128%26sid%3Dliteratum%253Aachs%26aulast%3DBapat%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DL.%2BS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DLong%26aufirst%3DE.%2BC.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DNovel%2520small%2520molecule%2520inhibitor%2520of%2520Ape1%2520endonuclease%2520blocks%2520proliferation%2520and%2520reduces%2520viability%2520of%2520glioblastoma%2520cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D988%26epage%3D998%26doi%3D10.1124%2Fjpet.110.169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorjsuren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of inhibitors of human apurinic/apyrimidinic endonuclease APE1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e5740</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0005740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1371%2Fjournal.pone.0005740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19484131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A280%3ADC%252BD1MzovVentw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=A.+Simeonovauthor=A.+Kulkarniauthor=D.+Dorjsurenauthor=A.+Jadhavauthor=M.+Shenauthor=D.+R.+McNeillauthor=C.+P.+Austinauthor=D.+M.+Wilson&title=Identification+and+characterization+of+inhibitors+of+human+apurinic%2Fapyrimidinic+endonuclease+APE1&doi=10.1371%2Fjournal.pone.0005740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of inhibitors of human apurinic/apyrimidinic endonuclease APE1</span></div><div class="casAuthors">Simeonov Anton; Kulkarni Avanti; Dorjsuren Dorjbal; Jadhav Ajit; Shen Min; McNeill Daniel R; Austin Christopher P; Wilson David M 3rd</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e5740</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">APE1 is the major nuclease for excising abasic (AP) sites and particular 3'-obstructive termini from DNA, and is an integral participant in the base excision repair (BER) pathway.  BER capacity plays a prominent role in dictating responsiveness to agents that generate oxidative or alkylation DNA damage, as well as certain chain-terminating nucleoside analogs and 5-fluorouracil.  We describe within the development of a robust, 1536-well automated screening assay that employs a deoxyoligonucleotide substrate operating in the red-shifted fluorescence spectral region to identify APE1 endonuclease inhibitors.  This AP site incision assay was used in a titration-based high-throughput screen of the Library of Pharmacologically Active Compounds (LOPAC(1280)), a collection of well-characterized, drug-like molecules representing all major target classes.  Prioritized hits were authenticated and characterized via two high-throughput screening assays -- a Thiazole Orange fluorophore-DNA displacement test and an E. coli endonuclease IV counterscreen -- and a conventional, gel-based radiotracer incision assay.  The top, validated compounds, i.e. 6-hydroxy-DL-DOPA, Reactive Blue 2 and myricetin, were shown to inhibit AP site cleavage activity of whole cell protein extracts from HEK 293T and HeLa cell lines, and to enhance the cytotoxic and genotoxic potency of the alkylating agent methylmethane sulfonate.  The studies herein report on the identification of novel, small molecule APE1-targeted bioactive inhibitor probes, which represent initial chemotypes towards the development of potential pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSLiX2uWPYz9wpJitSxOdIlfW6udTcc2eYH5IlfrTOpcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzovVentw%253D%253D&md5=5590848b09580b34791ea17864903b93</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0005740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0005740%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DKulkarni%26aufirst%3DA.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DMcNeill%26aufirst%3DD.%2BR.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520inhibitors%2520of%2520human%2520apurinic%252Fapyrimidinic%2520endonuclease%2520APE1%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0005740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyjayanti, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbotts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of human AP endonuclease inhibitors in melanoma and glioma cell lines</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6606058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fsj.bjc.6606058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=21266972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2011&pages=653-663&author=M.+Z.+Mohammedauthor=V.+N.+Vyjayantiauthor=C.+A.+Laughtonauthor=L.+V.+Dekkerauthor=P.+M.+Fischerauthor=D.+M.+Wilsonauthor=R.+Abbottsauthor=S.+Shahauthor=P.+M.+Patelauthor=I.+D.+Hicksonauthor=S.+Madhusudan&title=Development+and+evaluation+of+human+AP+endonuclease+inhibitors+in+melanoma+and+glioma+cell+lines&doi=10.1038%2Fsj.bjc.6606058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of human AP endonuclease inhibitors in melanoma and glioma cell lines</span></div><div class="casAuthors">Mohammed, M. Z.; Vyjayanti, V. N.; Laughton, C. A.; Dekker, L. V.; Fischer, P. M.; Wilson, D. M., III; Abbotts, R.; Shah, S.; Patel, P. M.; Hickson, I. D.; Madhusudan, S.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">653-663</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aims: Modulation of DNA base excision repair (BER) has the potential to enhance response to chemotherapy and improve outcomes in tumors such as melanoma and glioma.  APE1, a crit. protein in BER that processes potentially cytotoxic abasic sites (AP sites), is a promising new target in cancer.  In the current study, we aimed to develop small mol. inhibitors of APE1 for cancer therapy.  Methods: An industry-std. high throughput virtual screening strategy was adopted.  The Sybyl8.0 (Tripos, St Louis, MO, USA) mol. modeling software suite was used to build inhibitor templates.  Similarity searching strategies were then applied using ROCS 2.3 (Open Eye Scientific, Santa Fe, NM, USA) to ext. pharmacophorically related subsets of compds. from a chem. diverse database of 2.6 million compds.  The compds. in these subsets were subjected to docking against the active site of the APE1 model, using the genetic algorithm-based program GOLD2.7 (CCDC, Cambridge, UK).  Predicted ligand poses were ranked on the basis of several scoring functions.  The top virtual hits with promising pharmaceutical properties underwent detailed in vitro analyses using fluorescence-based APE1 cleavage assays and counter screened using endonuclease IV cleavage assays, fluorescence quenching assays and radiolabeled oligonucleotide assays.  Biochem. APE1 inhibitors were then subjected to detailed cytotoxicity analyses.  Results: Several specific APE1 inhibitors were isolated by this approach.  The IC50 for APE1 inhibition ranged between 30 n and 50 μ.  We demonstrated that APE1 inhibitors lead to accumulation of AP sites in genomic DNA and potentiated the cytotoxicity of alkylating agents in melanoma and glioma cell lines.  Conclusions: Our study provides evidence that APE1 is an emerging drug target and could have therapeutic application in patients with melanoma and glioma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRvuWYUOMcELVg90H21EOLACvtfcHk0lhuXDtcG6E2FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKhsrg%253D&md5=c29b7758524de944df907d9e267359de</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6606058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6606058%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DM.%2BZ.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DDekker%26aufirst%3DL.%2BV.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DAbbotts%26aufirst%3DR.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DP.%2BM.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520human%2520AP%2520endonuclease%2520inhibitors%2520in%2520melanoma%2520and%2520glioma%2520cell%2520lines%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2011%26volume%3D104%26spage%3D653%26epage%3D663%26doi%3D10.1038%2Fsj.bjc.6606058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholia, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span> <span> </span><span class="NLM_article-title">DNA repair and redox activities and inhibitors of apurinic/apyrimidinic endonuclease 1/redox effector factor 1 (APE1/Ref-1): a comparative analysis and their scope and limitations toward anticancer drug development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10256</span>, <span class="refDoi"> DOI: 10.1021/jm500865u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500865u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10241-10256&author=G.+Kaurauthor=R.+P.+Choliaauthor=A.+K.+Manthaauthor=R.+Kumar&title=DNA+repair+and+redox+activities+and+inhibitors+of+apurinic%2Fapyrimidinic+endonuclease+1%2Fredox+effector+factor+1+%28APE1%2FRef-1%29%3A+a+comparative+analysis+and+their+scope+and+limitations+toward+anticancer+drug+development&doi=10.1021%2Fjm500865u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm500865u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500865u%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DCholia%26aufirst%3DR.%2BP.%26aulast%3DMantha%26aufirst%3DA.%2BK.%26aulast%3DKumar%26aufirst%3DR.%26atitle%3DDNA%2520repair%2520and%2520redox%2520activities%2520and%2520inhibitors%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%252Fredox%2520effector%2520factor%25201%2520%2528APE1%252FRef-1%2529%253A%2520a%2520comparative%2520analysis%2520and%2520their%2520scope%2520and%2520limitations%2520toward%2520anticancer%2520drug%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10241%26epage%3D10256%26doi%3D10.1021%2Fjm500865u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M. P.</span></span> <span> </span><span class="NLM_article-title">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span>. <i>Chem. Commun. (Cambridge, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15948</span>– <span class="NLM_lpage">15951</span>, <span class="refDoi"> DOI: 10.1039/C5CC06084B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1039%2FC5CC06084B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=26377038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFajt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15948-15951&author=N.+Koteraauthor=F.+Poyerauthor=A.+Granzhanauthor=M.+P.+Teulade-Fichou&title=Efficient+inhibition+of+human+AP+endonuclease+1+%28APE1%29+via+substrate+masking+by+abasic+site-binding+macrocyclic+ligands&doi=10.1039%2FC5CC06084B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span></div><div class="casAuthors">Kotera, Naoko; Poyer, Florent; Granzhan, Anton; Teulade-Fichou, Marie-Paule</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">88</span>),
    <span class="NLM_cas:pages">15948-15951</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bis-naphthalene macrocycles, which bind with high affinity and selectivity to abasic sites in DNA, efficiently inhibit their cleavage by APE1 (IC50 = 55-60 nM in the kinetic assay with a model THF substrate).  These results demonstrate that substrate masking by non-covalent abasic-site ligands is an efficient strategy for inhibition of APE1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps5EUcFXG2UrVg90H21EOLACvtfcHk0lhuXDtcG6E2FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFajt7%252FI&md5=1ac3974b315995557ac5c74ec421e2f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC5CC06084B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06084B%26sid%3Dliteratum%253Aachs%26aulast%3DKotera%26aufirst%3DN.%26aulast%3DPoyer%26aufirst%3DF.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26atitle%3DEfficient%2520inhibition%2520of%2520human%2520AP%2520endonuclease%25201%2520%2528APE1%2529%2520via%2520substrate%2520masking%2520by%2520abasic%2520site-binding%2520macrocyclic%2520ligands%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D15948%26epage%3D15951%26doi%3D10.1039%2FC5CC06084B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of DNA repair nuclease activities of APE1</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">3621</span>– <span class="NLM_lpage">3631</span>, <span class="refDoi"> DOI: 10.1007/s00018-010-0488-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs00018-010-0488-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20809131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=3621-3631&author=D.+M.+Wilsonauthor=A.+Simeonov&title=Small+molecule+inhibitors+of+DNA+repair+nuclease+activities+of+APE1&doi=10.1007%2Fs00018-010-0488-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of DNA repair nuclease activities of APE1</span></div><div class="casAuthors">Wilson, David M., III; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3621-3631</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  APE1 is a multifunctional protein that possesses several nuclease activities, including the ability to incise at apurinic/apyrimidinic (AP) sites in DNA or RNA, to excise 3'-blocking termini from DNA ends, and to cleave at certain oxidized base lesions in DNA.  Pre-clin. and clin. data indicate a role for APE1 in the pathogenesis of cancer and in resistance to DNA-interactive drugs, particularly monofunctional alkylators and antimetabolites.  In an effort to improve the efficacy of therapeutic compds., such as temozolomide, groups have begun to develop high-throughput screening assays and to identify small mol. inhibitors against APE1 repair nuclease activities.  It is envisioned that such inhibitors will be used in combinatorial treatment paradigms to enhance the efficacy of DNA-interactive drugs that introduce relevant cytotoxic DNA lesions.  In this review, we summarize the current state of the efforts to design potent and selective inhibitors against APE1 AP site incision activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN4mUcnbEDarVg90H21EOLACvtfcHk0lg6fW98fCsVvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN&md5=7406c275670396d1652dfda2cf68e2af</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs00018-010-0488-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-010-0488-2%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520DNA%2520repair%2520nuclease%2520activities%2520of%2520APE1%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D3621%26epage%3D3631%26doi%3D10.1007%2Fs00018-010-0488-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyjayanti, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorjsuren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3101</span>– <span class="NLM_lpage">3112</span>, <span class="refDoi"> DOI: 10.1021/jm201537d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201537d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3101-3112&author=G.+Raiauthor=V.+N.+Vyjayantiauthor=D.+Dorjsurenauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+M.+Wilsonauthor=D.+J.+Maloney&title=Synthesis%2C+biological+evaluation%2C+and+structure-activity+relationships+of+a+novel+class+of+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm201537d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span></div><div class="casAuthors">Rai, Ganesha; Vyjayanti, Vaddadi N.; Dorjsuren, Dorjbal; Simeonov, Anton; Jadhav, Ajit; Wilson, David M.; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3101-3112</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">APE1 is an essential protein that operates in the base excision repair (BER) pathway and is responsible for ≥95% of the total apurinic/apyrimidinic (AP) endonuclease activity in human cells.  BER is a major pathway that copes with DNA damage induced by several anticancer agents, including ionizing radiation and temozolomide.  Overexpression of APE1 and enhanced AP endonuclease activity have been linked to increased resistance of tumor cells to treatment with monofunctional alkylators, implicating inhibition of APE1 as a valid strategy for cancer therapy.  We report herein the results of a focused medicinal chem. effort around a novel APE1 inhibitor, N-(3-(benzo[d]thiazol-2-yl)-6-isopropyl-4,5,6,7-tetrahydrothieno[2,3-c]pyridin-2-yl)acetamide (I).  Compd. I and related analogs exhibit single-digit micromolar activity against the purified APE1 enzyme and comparable activity in HeLa whole cell ext. assays and potentiate the cytotoxicity of the alkylating agents methylmethane sulfonate and temozolomide.  Moreover, this class of compds. possesses a generally favorable in vitro ADME profile, along with good exposure levels in plasma and brain following i.p. dosing (30 mg/kg body wt.) in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25R0Ebe53srVg90H21EOLACvtfcHk0lg6fW98fCsVvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D&md5=1dac481ed0ec1cc67d59ad3091520ccc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm201537d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201537d%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DSynthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520novel%2520class%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3101%26epage%3D3112%26doi%3D10.1021%2Fjm201537d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogbourne, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, P. G.</span></span> <span> </span><span class="NLM_article-title">The value of nature’s natural product library for the discovery of New Chemical Entities: the discovery of ingenol mebutate</span>. <i>Fitoterapia</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1016/j.fitote.2014.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.fitote.2014.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25016953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlantLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2014&pages=36-44&author=S.+M.+Ogbourneauthor=P.+G.+Parsons&title=The+value+of+nature%E2%80%99s+natural+product+library+for+the+discovery+of+New+Chemical+Entities%3A+the+discovery+of+ingenol+mebutate&doi=10.1016%2Fj.fitote.2014.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The value of nature's natural product library for the discovery of New Chemical Entities: The discovery of ingenol mebutate</span></div><div class="casAuthors">Ogbourne, Steven M.; Parsons, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Fitoterapia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">FTRPAE</span>;
        ISSN:<span class="NLM_cas:issn">0367-326X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent decades, 'Big Pharma' has invested billions of dollars into ingenious and innovative strategies designed to develop drugs using high throughput screening of small mol. libraries generated on the lab. bench.  Within the same time frame, screening of natural products by pharmaceutical companies has suffered an equally significant redn.  This is despite the fact that the complexity, functional diversity and druggability of nature's natural product library are considered by many to be superior to any library any team of scientists can prep.  It is therefore no coincidence that the no. of New Chem. Entities reaching the market has also suffered a substantial decrease, leading to a productivity crisis within the pharmaceutical sector.  In fact, the current dearth of New Chem. Entities reaching the market in recent decades might be a direct consequence of the strategic decision to move away from screening of natural products.  Nearly 700 novel drugs derived from natural product New Chem. Entities were approved between 1981 and 2010; more than 60% of all approved drugs over the same time.  In this review, we use the example of ingenol mebutate, a natural product identified from Euphorbia peplus and later approved as a therapy for actinic keratosis, as why nature's natural product library remains the most valuable library for discovery of New Chem. Entities and of novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMRLEFPnv8KrVg90H21EOLACvtfcHk0lgGqJk6cBSNeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlantLjO&md5=018f18eff0c2d50123a68a7c344974c9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.fitote.2014.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fitote.2014.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DOgbourne%26aufirst%3DS.%2BM.%26aulast%3DParsons%26aufirst%3DP.%2BG.%26atitle%3DThe%2520value%2520of%2520nature%25E2%2580%2599s%2520natural%2520product%2520library%2520for%2520the%2520discovery%2520of%2520New%2520Chemical%2520Entities%253A%2520the%2520discovery%2520of%2520ingenol%2520mebutate%26jtitle%3DFitoterapia%26date%3D2014%26volume%3D98%26spage%3D36%26epage%3D44%26doi%3D10.1016%2Fj.fitote.2014.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wessjohann, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruijter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Rivera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, W.</span></span> <span> </span><span class="NLM_article-title">What can a chemist learn from nature’s macrocycles?--a brief, conceptual view</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1007/s11030-005-1314-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs11030-005-1314-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=15789564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=171-186&author=L.+A.+Wessjohannauthor=E.+Ruijterauthor=D.+Garcia-Riveraauthor=W.+Brandt&title=What+can+a+chemist+learn+from+nature%E2%80%99s+macrocycles%3F%2D%2Da+brief%2C+conceptual+view&doi=10.1007%2Fs11030-005-1314-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">What can a chemist learn from nature's macrocycles? - A brief, conceptual view</span></div><div class="casAuthors">Wessjohann, Ludger A.; Ruijter, Eelco; Garcia-Rivera, Daniel; Brandt, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">171-186</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Macrocyclic natural products often display remarkable biol. activities, and many of these compds. (or their derivs.) are used as drugs.  The chem. diversity of these compds. is immense and may provide inspiration for innovative drug design.  Therefore, a database of naturally occurring macrocycles was analyzed for ring size, mol. wt. distribution, and the frequency of some common substructural motifs.  The underlying principles of the chem. diversity are reviewed in terms of biosynthetic origin and nature's strategies for diversity and complexity generation in relation to the structural diversity and similarities found in the macrocycle database.  Finally, it is suggested that synthetic chemists should use not only nature's mols., but also nature's strategies as a source of inspiration.  To illustrate this, the biosynthesis of macrocycles by non-ribosomal peptide synthetases and terpene and polyketide cyclases, as well as recent advances of these strategies in an integrated synthesis/biotechnol. approach are briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVB4g1QplcWLVg90H21EOLACvtfcHk0lgfYVoN8OmGeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D&md5=88df3ca095b446e3781369708ccddfd2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11030-005-1314-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-005-1314-x%26sid%3Dliteratum%253Aachs%26aulast%3DWessjohann%26aufirst%3DL.%2BA.%26aulast%3DRuijter%26aufirst%3DE.%26aulast%3DGarcia-Rivera%26aufirst%3DD.%26aulast%3DBrandt%26aufirst%3DW.%26atitle%3DWhat%2520can%2520a%2520chemist%2520learn%2520from%2520nature%25E2%2580%2599s%2520macrocycles%253F--a%2520brief%252C%2520conceptual%2520view%26jtitle%3DMol.%2520Diversity%26date%3D2005%26volume%3D9%26spage%3D171%26epage%3D186%26doi%3D10.1007%2Fs11030-005-1314-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lgfYVoN8OmGeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+five+in+2015+and+beyond%3A+Target+and+ligand+structural+limitations%2C+ligand+chemistry+structure+and+drug+discovery+project+decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lgfYVoN8OmGeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520ligand%2520structural%2520limitations%252C%2520ligand%2520chemistry%2520structure%2520and%2520drug%2520discovery%2520project%2520decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lgfYVoN8OmGeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lgsRIuC5YyoxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lgsRIuC5YyoxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and synthetic methodologies toward macrocycles</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6230</span>– <span class="NLM_lpage">6268</span>, <span class="refDoi"> DOI: 10.3390/molecules18066230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.3390%2Fmolecules18066230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=23708234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFemurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=6230-6268&author=X.+Yuauthor=D.+Sun&title=Macrocyclic+drugs+and+synthetic+methodologies+toward+macrocycles&doi=10.3390%2Fmolecules18066230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic drugs and synthetic methodologies toward macrocycles</span></div><div class="casAuthors">Yu, Xufen; Sun, Dianqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6230-6268</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Macrocyclic scaffolds are commonly found in bioactive natural products and pharmaceutical mols.  So far, a large no. of macrocyclic natural products were isolated and synthesized.  The construction of macrocycles is generally considered as a crucial and challenging step in the synthesis of macrocyclic natural products.  Over the last several decades, numerous efforts were undertaken toward the synthesis of complex naturally occurring macrocycles and great progresses were made to advance the field of total synthesis.  The commonly used synthetic methodologies toward macrocyclization include macrolactonization, macrolactamization, transition metal-catalyzed cross coupling, ring-closing metathesis, and click reaction, among others.  Selected recent examples of macrocyclic synthesis of natural products and drug-like macrocycles with significant biol. relevance are highlighted in each class.  The primary goal of this review was to summarize currently used macrocyclic drugs, highlight the therapeutic potential of this under-explored drug class and outline the general synthetic methodologies for the synthesis of macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVX-oHAYozLrVg90H21EOLACvtfcHk0lgsRIuC5YyoxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFemurw%253D&md5=b5756f10331fd590049422000f67b900</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18066230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18066230%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DMacrocyclic%2520drugs%2520and%2520synthetic%2520methodologies%2520toward%2520macrocycles%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D6230%26epage%3D6268%26doi%3D10.3390%2Fmolecules18066230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Drug discovery: tools and rules for macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">696</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnchembio.1605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25038789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=696-698&author=C.+Heinis&title=Drug+discovery%3A+tools+and+rules+for+macrocycles&doi=10.1038%2Fnchembio.1605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery. Tools and rules for macrocycles</span></div><div class="casAuthors">Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">696-698</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The application of macrocycles to previously undruggable targets has aroused a great deal of interest in this structural class.  Recent studies advance our understanding of the way macrocycles bind protein targets and provide new strategies and tools to generate peptide-based macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqeFUEIoJytLVg90H21EOLACvtfcHk0lhGYYw2cvQRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nM&md5=c9063baf5e54835719af3ff6e3cabe8f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1605%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDrug%2520discovery%253A%2520tools%2520and%2520rules%2520for%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D696%26epage%3D698%26doi%3D10.1038%2Fnchembio.1605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lhGYYw2cvQRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">DNA-bound structure and mutants reveal abasic DNA binding by APE1 DNA repair and coordination</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1038/35000249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2F35000249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10667800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD3cXpslaitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=451-456&author=C.+D.+Molauthor=T.+Izumiauthor=S.+Mitraauthor=J.+A.+Tainer&title=DNA-bound+structure+and+mutants+reveal+abasic+DNA+binding+by+APE1+DNA+repair+and+coordination&doi=10.1038%2F35000249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-bound structures and mutants reveal abasic DNA binding by APE1 DNA repair and coordination</span></div><div class="casAuthors">Mol, Clifford D.; Izumi, Tadahide; Mitra, Sankar; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6768</span>),
    <span class="NLM_cas:pages">451-456</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Non-coding apurinic/apyrimidinic (AP) sites in DNA are continually created in cells both spontaneously and by damage- specific DNA glycosylases.  The biol. crit. human base excision repair enzyme APE1 cleaves the DNA sugar-phosphate backbone at a position 5' of AP sites to prime DNA repair synthesis.  Here we report three co-crystal structures of human APE1 bound to abasic DNA which show that APEI uses a rigid, pre-formed, pos. charged surface to kink the DNA helix and engulf the AP-DNA strand.  APE1 inserts loops into both the DNA major and minor grooves and binds a flipped-out AP site in a pocket that excludes DNA bases and racemized β-anomer AP sites.  Both the APE1 active-site geometry and a complex with cleaved AP-DNA and Mn2+ support a testable structure-based catalytic mechanism.  Alanine substitutions of the residues that penetrate the DNA helix unexpectedly show that human APE1 is structurally optimized to retain the cleaved DNA product.  These structural and mutational results show how APE1 probably displaces bound glycosylases and retains the nicked DNA product, suggesting that APEI acts in vivo to coordinate the orderly transfer of unstable DNA damage intermediates between the excision and synthesis steps of DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTgcGoYjDzMrVg90H21EOLACvtfcHk0lhGYYw2cvQRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXpslaitg%253D%253D&md5=59d3d4ba71c516f9bf4693ae7199aa37</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F35000249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35000249%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DIzumi%26aufirst%3DT.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DDNA-bound%2520structure%2520and%2520mutants%2520reveal%2520abasic%2520DNA%2520binding%2520by%2520APE1%2520DNA%2520repair%2520and%2520coordination%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D451%26epage%3D456%26doi%3D10.1038%2F35000249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span> <span> </span><span class="NLM_article-title">High-resolution crystal structures reveal plasticity in the metal binding site of apurinic/apyrimidinic endonuclease I</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6529</span>, <span class="refDoi"> DOI: 10.1021/bi500676p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500676p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Sju7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6520-6529&author=H.+Heauthor=Q.+Chenauthor=M.+M.+Georgiadis&title=High-resolution+crystal+structures+reveal+plasticity+in+the+metal+binding+site+of+apurinic%2Fapyrimidinic+endonuclease+I&doi=10.1021%2Fbi500676p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structures Reveal Plasticity in the Metal Binding Site of Apurinic/Apyrimidinic Endonuclease I</span></div><div class="casAuthors">He, Hongzhen; Chen, Qiujia; Georgiadis, Millie M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6520-6529</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic endonuclease I (APE1) is an essential base excision repair enzyme that catalyzes a Mg2+-dependent reaction in which the phosphodiester backbone is cleaved 5' of an abasic site in duplex DNA.  This reaction has been proposed to involve either one or two metal ions bound to the active site.  In the present study, we report crystal structures of Mg2+, Mn2+, and apo-APE1 detd. at 1.4, 2.2, and 1.65 Å, resp., representing two of the highest resoln. structures yet reported for APE1.  In our structures, a single well-ordered Mn2+ ion was obsd. coordinated by D70 and E96; the Mg2+ site exhibited disorder modeled as two closely positioned sites coordinated by D70 and E96 or E96 alone.  Direct metal binding anal. of wild-type, D70A, and E96A APE1, as assessed by differential scanning fluorimetry, indicated a role for D70 and E96 in binding of Mg2+ or Mn2+ to APE1.  Consistent with the disorder exhibited by Mg2+ bound to the active site, two different conformations of E96 were obsd. coordinated to Mg2+.  A third conformation for E96 in the apo structure is similar to that obsd. in the APE1-DNA-Mg2+ complex structure.  Thus, binding of Mg2+ in three different positions within the active site of APE1 in these crystal structures corresponds directly with three different conformations of E96.  Taken together, our results are consistent with the initial capture of metal by D70 and E96 and repositioning of Mg2+ facilitated by the structural plasticity of E96 in the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwsb3wP6G4ILVg90H21EOLACvtfcHk0licfGxrh3Gm1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Sju7fK&md5=337d74f58a6c1d6907dd54f4debb5956</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fbi500676p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500676p%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26atitle%3DHigh-resolution%2520crystal%2520structures%2520reveal%2520plasticity%2520in%2520the%2520metal%2520binding%2520site%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%2520I%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D6520%26epage%3D6529%26doi%3D10.1021%2Fbi500676p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span> <span> </span><span class="NLM_article-title">Bifunctional homoallylic carbamates from chiral silane additions to in situ generated N-acyl iminium ions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3624</span>– <span class="NLM_lpage">3627</span>, <span class="refDoi"> DOI: 10.1021/ol301428y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol301428y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=3624-3627&author=J.+Wuauthor=K.+C.+Zhuauthor=P.+W.+Yuanauthor=J.+S.+Panek&title=Bifunctional+homoallylic+carbamates+from+chiral+silane+additions+to+in+situ+generated+N-acyl+iminium+ions&doi=10.1021%2Fol301428y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bifunctional homoallylic carbamates from chiral silane additions to in situ generated N-acyl iminium ions</span></div><div class="casAuthors">Wu, Jie; Zhu, Kai-Cheng; Yuan, Ping-Wei; Panek, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3624-3627</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Homoallylic carbamates bearing an α,β-unsatd. ester or an allylic carbonate were generated resp. utilizing novel chiral silanes through Lewis acid promoted asym. aminocrotylation.  Those bifunctional building blocks could further undergo Michael addn. or cyclization to selectively form functionalized lactams, azetidines, and tetrahydropyrimidinones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Rwub53UlwrVg90H21EOLACvtfcHk0licfGxrh3Gm1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cgtr8%253D&md5=673a276eb136329485b8cb5ea02b04aa</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fol301428y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol301428y%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%2BC.%26aulast%3DYuan%26aufirst%3DP.%2BW.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26atitle%3DBifunctional%2520homoallylic%2520carbamates%2520from%2520chiral%2520silane%2520additions%2520to%2520in%2520situ%2520generated%2520N-acyl%2520iminium%2520ions%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D3624%26epage%3D3627%26doi%3D10.1021%2Fol301428y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beresis, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of chiral (E)-Cryotylsilanes: [3R- and 3S-]-(4E)-Methyl 3-(Dimethylphenylsilyl)-4-Hexenoate</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi"> DOI: 10.1002/0471264180.os075.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2F0471264180.os075.10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1998&pages=78&author=R.+T.+Beresisauthor=J.+S.+Solomonauthor=M.+G.+Yangauthor=N.+F.+Jainauthor=J.+S.+Panek&title=Synthesis+of+chiral+%28E%29-Cryotylsilanes%3A+%5B3R-+and+3S-%5D-%284E%29-Methyl+3-%28Dimethylphenylsilyl%29-4-Hexenoate&doi=10.1002%2F0471264180.os075.10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F0471264180.os075.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264180.os075.10%26sid%3Dliteratum%253Aachs%26aulast%3DBeresis%26aufirst%3DR.%2BT.%26aulast%3DSolomon%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DM.%2BG.%26aulast%3DJain%26aufirst%3DN.%2BF.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520chiral%2520%2528E%2529-Cryotylsilanes%253A%2520%255B3R-%2520and%25203S-%255D-%25284E%2529-Methyl%25203-%2528Dimethylphenylsilyl%2529-4-Hexenoate%26jtitle%3DOrg.%2520Synth.%26date%3D1998%26volume%3D75%26spage%3D78%26doi%3D10.1002%2F0471264180.os075.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beeler, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquilano, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Synthesis of a library of complex macrodiolides employing cyclodimerization of hydroxy esters</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1021/cc050064b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc050064b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVChtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=673-681&author=A.+B.+Beelerauthor=D.+E.+Acquilanoauthor=Q.+Suauthor=F.+Yanauthor=B.+L.+Rothauthor=J.+S.+Panekauthor=J.+A.+Porco&title=Synthesis+of+a+library+of+complex+macrodiolides+employing+cyclodimerization+of+hydroxy+esters&doi=10.1021%2Fcc050064b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of a Library of Complex Macrodiolides Employing Cyclodimerization of Hydroxy Esters</span></div><div class="casAuthors">Beeler, Aaron B.; Acquilano, Dayle E.; Su, Qibin; Yan, Feng; Roth, Bryan L.; Panek, James S.; Porco, John A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">673-681</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of complex macrodiolides involving microwave-accelerated transesterification of chiral, nonracemic, hydroxy esters is described.  Thus, reacting hydroxy esters I and II using [[F3C(CF2)5CH2CH2]2SnO]n as catalyst gave macrodiolide III in 99% yield.  Methodol. development studies indicate that both microwave power and reaction temp. play an important role in the efficiency of cyclodimerizations.  Hydroxy ester monomer pairs were evaluated using an anal. rehearsal leading to the prepn. of a 127-member library of highly diverse and stereochem. well-defined macrodiolides.  Preliminary assays identified a novel macrodiolide antagonist, IV (Ts = SO2C6H4Me-4), of the κ opioid receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplBVRsXsFxErVg90H21EOLACvtfcHk0licfGxrh3Gm1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVChtbY%253D&md5=42c552bdccdd3d4c43ca9dbb92c9ed57</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcc050064b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc050064b%26sid%3Dliteratum%253Aachs%26aulast%3DBeeler%26aufirst%3DA.%2BB.%26aulast%3DAcquilano%26aufirst%3DD.%2BE.%26aulast%3DSu%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DPanek%26aufirst%3DJ.%2BS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26atitle%3DSynthesis%2520of%2520a%2520library%2520of%2520complex%2520macrodiolides%2520employing%2520cyclodimerization%2520of%2520hydroxy%2520esters%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2005%26volume%3D7%26spage%3D673%26epage%3D681%26doi%3D10.1021%2Fcc050064b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, Y.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of bacterial resistance to macrolide antibiotics</span>. <i>J. Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1007/s101560050011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1007%2Fs101560050011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=11810493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADyaK1MXktlCgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=61-74&author=Y.+Nakajima&title=Mechanisms+of+bacterial+resistance+to+macrolide+antibiotics&doi=10.1007%2Fs101560050011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of bacterial resistance to macrolide antibiotics</span></div><div class="casAuthors">Nakajima, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infection and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">61-74</span>CODEN:
                <span class="NLM_cas:coden">JICHFN</span>;
        ISSN:<span class="NLM_cas:issn">1341-321X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Tokyo</span>)
        </div><div class="casAbstract">A review with 125 refs.  On the basis of information obtained from extensive refs. and studies of resistance mechanisms to macrolide antibiotics, the mode of action of the drugs, as effectors, and a hypothetical explanation of the regulation of the mechanism with regard to induction of macrolide resistance are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyNVI7zYQrLrVg90H21EOLACvtfcHk0li0QaaWfXoSTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlCgs7s%253D&md5=a56376c70e47e9ee4b546c1d3e2dc750</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2Fs101560050011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs101560050011%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DY.%26atitle%3DMechanisms%2520of%2520bacterial%2520resistance%2520to%2520macrolide%2520antibiotics%26jtitle%3DJ.%2520Infect.%2520Chemother.%26date%3D1999%26volume%3D5%26spage%3D61%26epage%3D74%26doi%3D10.1007%2Fs101560050011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pengelly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koteva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Mechanism and diversity of the erythromycin esterase family of enzymes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1740</span>– <span class="NLM_lpage">1751</span>, <span class="refDoi"> DOI: 10.1021/bi201790u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi201790u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1OnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1740-1751&author=M.+Morarauthor=K.+Pengellyauthor=K.+Kotevaauthor=G.+D.+Wright&title=Mechanism+and+diversity+of+the+erythromycin+esterase+family+of+enzymes&doi=10.1021%2Fbi201790u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and Diversity of the Erythromycin Esterase Family of Enzymes</span></div><div class="casAuthors">Morar, Mariya; Pengelly, Kate; Koteva, Kalinka; Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1740-1751</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrolide antibiotics such as azithromycin and erythromycin are mainstays of modern antibacterial chemotherapy, and like all antibiotics, they are vulnerable to resistance.  One mechanism of macrolide resistance is via drug inactivation: enzymic hydrolysis of the macrolactone ring catalyzed by erythromycin esterases, EreA and EreB.  A genomic enzymol. approach was taken to gain insight into the catalytic mechanisms and origins of Ere enzymes.  Our anal. reveals that erythromycin esterases comprise a sep. group in the hydrolase superfamily, which includes homologues of uncharacterized function found on the chromosome of Bacillus cereus, Bcr135 and Bcr136, whose three-dimensional structures have been detd.  Biochem. characterization of Bcr136 confirms that it is an esterase; however, it is unable to inactivate macrolides.  Using steady-state kinetics, homol.-based structure modeling, site-directed mutagenesis, solvent isotope effect studies, pH, and inhibitor profiling performed in various combinations for EreA, EreB, and Bcr136 enzymes, we identified the active site and gained insight into some catalytic features of this novel enzyme superfamily.  We rule out the possibility of a Ser/Thr nucleophile and show that one histidine, H46 (EreB numbering), is essential for catalytic function.  This residue is proposed to serve as a general base in activation of a water mol. as the reaction nucleophile.  Furthermore, we show that EreA, EreB, and Bcr136 are distinct, with only EreA inhibited by chelating agents and hypothesized to contain a noncatalytic metal.  Detailed characterization of these esterases allows for a direct comparison of the resistance determinants, EreA and EreB, with their prototype, Bcr136, and for the discussion of their potential connections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEzsk8G72di7Vg90H21EOLACvtfcHk0li0QaaWfXoSTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1OnsbY%253D&md5=f8a48c116f66a4a5cdad00880328ab81</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fbi201790u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi201790u%26sid%3Dliteratum%253Aachs%26aulast%3DMorar%26aufirst%3DM.%26aulast%3DPengelly%26aufirst%3DK.%26aulast%3DKoteva%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DMechanism%2520and%2520diversity%2520of%2520the%2520erythromycin%2520esterase%2520family%2520of%2520enzymes%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D1740%26epage%3D1751%26doi%3D10.1021%2Fbi201790u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin-Sinex, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Knockdown of the DNA repair and redox signaling protein Ape1/Ref-1 blocks ovarian cancer cell and tumor growth</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.09.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.dnarep.2007.09.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=17974506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisl2gtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=177-186&author=M.%0AL.+Fishelauthor=Y.+Heauthor=A.+M.+Reedauthor=H.+Chin-Sinexauthor=G.+D.+Hutchinsauthor=M.+S.+Mendoncaauthor=M.+R.+Kelley&title=Knockdown+of+the+DNA+repair+and+redox+signaling+protein+Ape1%2FRef-1+blocks+ovarian+cancer+cell+and+tumor+growth&doi=10.1016%2Fj.dnarep.2007.09.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Knockdown of the DNA repair and redox signaling protein Ape1/Ref-1 blocks ovarian cancer cell and tumor growth</span></div><div class="casAuthors">Fishel, Melissa L.; He, Ying; Reed, April M.; Chin-Sinex, Helen; Hutchins, Gary D.; Mendonca, Marc S.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-186</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apurinic endonuclease 1/redox effector factor-1 (Ape1/Ref-1 or Ape1) is an essential protein with two distinct functions.  It is a DNA repair enzyme in the base excision repair (BER) pathway and a redn.-oxidn. (redox) signaling factor maintaining transcription factors in an active reduced state.  The authors' lab. previously demonstrated that Ape1 is overexpressed in ovarian cancer and potentially contributes to resistance.  Therefore, the authors utilized siRNA technol. to knockdown protein levels of Ape1 in ovarian cancer cell line, SKOV-3x.  Knocking Ape1 down had dramatic effects on cell growth in vitro but was not due to an increase in apoptosis and at least partially due to an extension in transit time through S-phase.  Similarly, human ovarian tumor xenografts with reduced levels of Ape1 protein demonstrated a dramatic redn. in tumor vol. and also statistically significant differences in 18F-fluorodeoxyglucose (FDG) uptake indicating reduced glucose metab. and cellular proliferation.  Ape1's role in DNA repair and redox signaling is important to the basic understanding of ovarian cancer cell growth and these findings strongly support Ape1 as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVlAid1yFC8bVg90H21EOLACvtfcHk0li0QaaWfXoSTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisl2gtQ%253D%253D&md5=5aa8979ccf9449727259b0bb5e51da97</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.09.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.09.008%26sid%3Dliteratum%253Aachs%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DReed%26aufirst%3DA.%2BM.%26aulast%3DChin-Sinex%26aufirst%3DH.%26aulast%3DHutchins%26aufirst%3DG.%2BD.%26aulast%3DMendonca%26aufirst%3DM.%2BS.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DKnockdown%2520of%2520the%2520DNA%2520repair%2520and%2520redox%2520signaling%2520protein%2520Ape1%252FRef-1%2520blocks%2520ovarian%2520cancer%2520cell%2520and%2520tumor%2520growth%26jtitle%3DDNA%2520Repair%26date%3D2008%26volume%3D7%26spage%3D177%26epage%3D186%26doi%3D10.1016%2Fj.dnarep.2007.09.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A. R.</span></span> <span> </span><span class="NLM_article-title">The comet assay for DNA damage and repair: principles, applications, and limitations</span>. <i>Mol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1385/MB:26:3:249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1385%2FMB%3A26%3A3%3A249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=15004294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivF2gu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=249-261&author=A.+R.+Collins&title=The+comet+assay+for+DNA+damage+and+repair%3A+principles%2C+applications%2C+and+limitations&doi=10.1385%2FMB%3A26%3A3%3A249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The comet assay for DNA damage and repair: principles, applications, and limitations</span></div><div class="casAuthors">Collins, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-261</span>CODEN:
                <span class="NLM_cas:coden">MLBOEO</span>;
        ISSN:<span class="NLM_cas:issn">1073-6085</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  The comet assay (single-cell gel electrophoresis) is a simple method for measuring DNA (DNA) strand breaks in eukaryotic cells.  Cells embedded in agarose on a microscope slide are lysed with detergent and high salt to form nucleoids contg. supercoiled loops of DNA linked to the nuclear matrix.  Electrophoresis at high pH results in structures resembling comets, obsd. by fluorescence microscopy; the intensity of the comet tail relative to the head reflects the no. of DNA breaks.  The likely basis for this is that loops contg. a break lose their supercoiling and become free to extend toward the anode.  The assay has applications in testing novel chems. for genotoxicity, monitoring environmental contamination with genotoxins, human biomonitoring and mol. epidemiol., and fundamental research in DNA damage and repair.  The sensitivity and specificity of the assay are greatly enhanced if the nucleoids are incubated with bacterial repair endonucleases that recognize specific kinds of damage in the DNA and convert lesions to DNA breaks, increasing the amt. of DNA in the comet tail.  DNA repair can be monitored by incubating cells after treatment with damaging agent and measuring the damage remaining at intervals.  Alternatively, the repair activity in a cell ext. can be measured by incubating it with nucleoids contg. specific damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8T2ekl_rxn7Vg90H21EOLACvtfcHk0lhBJXbNnzA76Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivF2gu7c%253D&md5=159fc301847b54325077c45f10795f13</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1385%2FMB%3A26%3A3%3A249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1385%252FMB%253A26%253A3%253A249%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DA.%2BR.%26atitle%3DThe%2520comet%2520assay%2520for%2520DNA%2520damage%2520and%2520repair%253A%2520principles%252C%2520applications%252C%2520and%2520limitations%26jtitle%3DMol.%2520Biotechnol.%26date%3D2004%26volume%3D26%26spage%3D249%26epage%3D261%26doi%3D10.1385%2FMB%3A26%3A3%3A249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agurell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beevers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendler-Schwaab, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlinson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thybaud, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, R. R.</span></span> <span> </span><span class="NLM_article-title">th International Comet Assay, W. Recommendations for conducting the in vivo alkaline Comet assay. 4th International Comet Assay Workshop</span>. <i>Mutagenesis</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1093/mutage/18.1.45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fmutage%2F18.1.45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=12473734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsFChu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=45-51&author=A.+Hartmannauthor=E.+Agurellauthor=C.+Beeversauthor=S.+Brendler-Schwaabauthor=B.+Burlinsonauthor=P.+Clayauthor=A.+Collinsauthor=A.+Smithauthor=G.+Speitauthor=V.+Thybaudauthor=R.+R.+Tice&title=th+International+Comet+Assay%2C+W.+Recommendations+for+conducting+the+in+vivo+alkaline+Comet+assay.+4th+International+Comet+Assay+Workshop&doi=10.1093%2Fmutage%2F18.1.45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendations for conducting the in vivo alkaline Comet assay</span></div><div class="casAuthors">Hartmann, A.; Agurell, E.; Beevers, C.; Brendler-Schwaab, S.; Burlinson, B.; Clay, P.; Collins, A.; Smith, A.; Speit, G.; Thybaud, V.; Tice, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Mutagenesis</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">MUTAEX</span>;
        ISSN:<span class="NLM_cas:issn">0267-8357</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The in vivo alk. single cell gel electrophoresis assay, hereafter the Comet assay, can be used to investigate the genotoxicity of industrial chems., biocides, agrochems. and pharmaceuticals.  The major advantages of this assay include the relative ease of application to any tissue of interest, the detection of multiple classes of DNA damage and the generation of data at the level of the single cell.  These features give the Comet assay potential advantages over other in vivo test methods, which are limited largely to proliferating cells and/or a single tissue.  The Comet assay has demonstrated its reliability in many testing circumstances and is, in general, considered to be acceptable for regulatory purposes.  However, despite the considerable data published on the in vivo Comet assay and the general agreement within the international scientific community over many protocol-related issues, it was felt that a document giving detailed practical guidance on the protocol required for regulatory acceptance of the assay was required.  In a recent meeting held in conjunction with the 4th International Comet Assay Workshop (Ulm, Germany, 22-25 July 2001) an expert panel reviewed existing data and recent developments of the Comet assay with a view to developing such a document.  This paper is intended to act as an update to the more general guidelines which were published as a result of the International Workshop on Genotoxicity Test Procedures.  The recommendations are also seen as a major step towards gaining more formal regulatory acceptance of the Comet assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAzJMHaGXCLLVg90H21EOLACvtfcHk0lhBJXbNnzA76Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsFChu7o%253D&md5=c3ad3b2907b23d0471de6f90b8dad721</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fmutage%2F18.1.45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmutage%252F18.1.45%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DA.%26aulast%3DAgurell%26aufirst%3DE.%26aulast%3DBeevers%26aufirst%3DC.%26aulast%3DBrendler-Schwaab%26aufirst%3DS.%26aulast%3DBurlinson%26aufirst%3DB.%26aulast%3DClay%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DSpeit%26aufirst%3DG.%26aulast%3DThybaud%26aufirst%3DV.%26aulast%3DTice%26aufirst%3DR.%2BR.%26atitle%3Dth%2520International%2520Comet%2520Assay%252C%2520W.%2520Recommendations%2520for%2520conducting%2520the%2520in%2520vivo%2520alkaline%2520Comet%2520assay.%25204th%2520International%2520Comet%2520Assay%2520Workshop%26jtitle%3DMutagenesis%26date%3D2003%26volume%3D18%26spage%3D45%26epage%3D51%26doi%3D10.1093%2Fmutage%2F18.1.45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaplane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M.</span></span> <span> </span><span class="NLM_article-title">Evolution of the redox function in mammalian apurinic/apyrimidinic endonuclease</span>. <i>Mutat. Res., Fundam. Mol. Mech. Mutagen.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>643</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.mrfmmm.2008.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2Fj.mrfmmm.2008.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=18579163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=643&publication_year=2008&pages=54-63&author=M.+Georgiadisauthor=M.+Luoauthor=R.+Gaurauthor=S.+Delaplaneauthor=X.+Liauthor=M.+Kelley&title=Evolution+of+the+redox+function+in+mammalian+apurinic%2Fapyrimidinic+endonuclease&doi=10.1016%2Fj.mrfmmm.2008.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of the redox function in mammalian apurinic/apyrimidinic endonuclease</span></div><div class="casAuthors">Georgiadis, M. M.; Luo, M.; Gaur, R. K.; Delaplane, S.; Li, X.; Kelley, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, Fundamental and Molecular Mechanisms of Mutagenesis</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">643</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">54-63</span>CODEN:
                <span class="NLM_cas:coden">MUREAV</span>;
        ISSN:<span class="NLM_cas:issn">0027-5107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human apurinic/apyrimidinic endonuclease (hApe1) encodes two important functional activities: an essential base excision repair (BER) activity and a redox activity that regulates expression of a no. of genes through redn. of their transcription factors, AP-1, NFκB, HIF-1α, CREB, p53 and others.  The BER function is highly conserved from prokaryotes (E. coli exonuclease III) to humans (hApe1).  Here, we provide evidence supporting a redox function unique to mammalian Apes.  An evolutionary anal. of Ape sequences reveals that, of the 7 Cys residues, Cys-93, 99, 208, 296, and 310 are conserved in both mammalian and non-mammalian vertebrate Apes, while Cys-65 is unique to mammalian Apes.  In the zebrafish Ape (zApe), selected as the vertebrate sequence most distant from human, the residue equiv. to Cys-65 is Thr-58.  The wild-type zApe enzyme was tested for redox activity in both in vitro EMSA and transactivation assays and found to be inactive, similar to C65A hApe1.  Substitution of Thr-58 with Cys in zApe, however, resulted in a redox active enzyme, suggesting that a Cys residue in this position is indeed crit. for redox function.  To further probe differences between redox active and inactive enzymes, we have detd. the crystal structures of vertebrate redox inactive enzymes, the C65A human Ape1 enzyme and the zApe enzyme at 1.9 and 2.3 Å, resp.  Our results provide new insights on the redox function and highlight a dramatic gain-of-function activity for Ape1 in mammals not found in non-mammalian vertebrates or lower organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJCx6EpFifxLVg90H21EOLACvtfcHk0lhBJXbNnzA76Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWitrrK&md5=5d7f84bd92612a8822ed2e17631b0884</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.mrfmmm.2008.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mrfmmm.2008.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgiadis%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DGaur%26aufirst%3DR.%26aulast%3DDelaplane%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DKelley%26aufirst%3DM.%26atitle%3DEvolution%2520of%2520the%2520redox%2520function%2520in%2520mammalian%2520apurinic%252Fapyrimidinic%2520endonuclease%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D2008%26volume%3D643%26spage%3D54%26epage%3D63%26doi%3D10.1016%2Fj.mrfmmm.2008.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgDNtWX0KeIKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span> <i>SAINT</i>,  <i>SADABS</i>, and  <i>XPREP</i>; <span class="NLM_publisher-name">Bruker AXS Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, U.S.</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SAINT%2C+SADABS%2C+and+XPREP%3B+Bruker+AXS+Inc.%3A+Madison%2C+WI%2C+U.S.%2C+2007."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSAINT%26jtitle%3DSADABS%26jtitle%3DXPREP%26pub%3DBruker%2520AXS%2520Inc%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Fragment-based identification of druggable “hot spots” of proteins using Fourier domain correlation techniques</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fbioinformatics%2Fbtp036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19176554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=621-627&author=R.+Brenkeauthor=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=D.+Hallauthor=M.+R.+Landonauthor=C.+Mattosauthor=S.+Vajda&title=Fragment-based+identification+of+druggable+%E2%80%9Chot+spots%E2%80%9D+of+proteins+using+Fourier+domain+correlation+techniques&doi=10.1093%2Fbioinformatics%2Fbtp036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of druggable hot spots' of proteins using Fourier domain correlation techniques</span></div><div class="casAuthors">Brenke, Ryan; Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Hall, David; Landon, Melissa R.; Mattos, Carla; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">621-627</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The binding sites of proteins generally contain smaller regions that provide major contributions to the binding free energy and hence are the prime targets in drug design.  Screening libraries of fragment-sized compds. by NMR or X-ray crystallog. demonstrates that such hot spot' regions bind a large variety of small org. mols., and that a relatively high hit rate' is predictive of target sites that are likely to bind drug-like ligands with high affinity.  The authors' goal is to det. the hot spots' computationally rather than exptl.  The authors have developed the FTMAP algorithm that performs global search of the entire protein surface for regions that bind a no. of small org. probe mols.  The search is based on the extremely efficient fast Fourier transform (FFT) correlation approach which can sample billions of probe positions on dense translational and rotational grids, but can use only sums of correlation functions for scoring and hence is generally restricted to very simple energy expressions.  The novelty of FTMAP is that the authors were able to incorporate and represent on grids a detailed energy expression, resulting in a very accurate identification of low-energy probe clusters.  Overlapping clusters of different probes are defined as consensus sites (CSs).  The authors show that the largest CS is generally located at the most important subsite of the protein binding site, and the nearby smaller CSs identify other important subsites.  Mapping results are presented for elastase whose structure was solved in aq. solns. of 8 org. solvents, and they show that FTMAP provides very similar information.  The second application is to renin, a long-standing pharmaceutical target for the treatment of hypertension, and the authors show that the major CSs trace out the shape of the first approved renin inhibitor, aliskiren.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWSzH6Fg129bVg90H21EOLACvtfcHk0lgDNtWX0KeIKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D&md5=e5a466428f4dd47e89a4ae8361d3da41</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp036%26sid%3Dliteratum%253Aachs%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFragment-based%2520identification%2520of%2520druggable%2520%25E2%2580%259Chot%2520spots%25E2%2580%259D%2520of%2520proteins%2520using%2520Fourier%2520domain%2520correlation%2520techniques%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D621%26epage%3D627%26doi%3D10.1093%2Fbioinformatics%2Fbtp036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lgDNtWX0KeIKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">FTSite: high accuracy detection of ligand binding sites on unbound protein structures</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1093%2Fbioinformatics%2Fbtr651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=22113084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ans7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=286-7&author=C.+H.+Nganauthor=D.+R.+Hallauthor=B.+Zerbeauthor=L.+E.+Groveauthor=D.+Kozakovauthor=S.+Vajda&title=FTSite%3A+high+accuracy+detection+of+ligand+binding+sites+on+unbound+protein+structures&doi=10.1093%2Fbioinformatics%2Fbtr651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">FTSite: high accuracy detection of ligand binding sites on unbound protein structures</span></div><div class="casAuthors">Ngan, Chi-Ho; Hall, David R.; Zerbe, Brandon; Grove, Laurie E.; Kozakov, Dima; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-287</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Binding site identification is a classical problem that is important for a range of applications, including the structure-based prediction of function, the elucidation of functional relationships among proteins, protein engineering and drug design.  We describe an accurate method of binding site identification, namely FTSite.  This method is based on exptl. evidence that ligand binding sites also bind small org. mols. of various shapes and polarity.  The FTSite algorithm does not rely on any evolutionary or statistical information, but achieves near exptl. accuracy: it is capable of identifying the binding sites in over 94% of apo proteins from established test sets that have been used to evaluate many other binding site prediction methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri18NAnmz2g7Vg90H21EOLACvtfcHk0lh-QlTIg9T6jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ans7c%253D&md5=f30385a90b4e4c32905c9babc5827bff</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr651%26sid%3Dliteratum%253Aachs%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFTSite%253A%2520high%2520accuracy%2520detection%2520of%2520ligand%2520binding%2520sites%2520on%2520unbound%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D286%26epage%3D7%26doi%3D10.1093%2Fbioinformatics%2Fbtr651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lh-QlTIg9T6jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span> <i>Marvin</i>, version <span class="NLM_edition">6.1.0</span>; <span class="NLM_publisher-name">ChemAxon</span>, <span class="NLM_year">2013</span>; <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Marvin%2C+version+6.1.0%3B+ChemAxon%2C+2013%3B+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvin%26pub%3DChemAxon%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bapat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span> <span> </span><span class="NLM_article-title">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">988</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1124/jpet.110.169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1124%2Fjpet.110.169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=20504914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=988-998&author=A.+Bapatauthor=L.+S.+Glassauthor=M.+Luoauthor=M.+L.+Fishelauthor=E.+C.+Longauthor=M.+M.+Georgiadisauthor=M.+R.+Kelley&title=Novel+small-molecule+inhibitor+of+apurinic%2Fapyrimidinic+endonuclease+1+blocks+proliferation+and+reduces+viability+of+glioblastoma+cells&doi=10.1124%2Fjpet.110.169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel small-molecule inhibitor of apurinic/apyrimidinic endonuclease 1 blocks proliferation and reduces viability of glioblastoma cells</span></div><div class="casAuthors">Bapat, Aditi; Glass, LaTeca S.; Luo, Meihua; Fishel, Melissa L.; Long, Eric C.; Georgiadis, Millie M.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apurinic/apyrimidinic (AP) endonuclease 1 (Ape1) is an essential DNA repair protein that plays a crit. role in repair of AP sites via base excision repair.  Ape1 has received attention as a druggable oncotherapeutic target, esp. for treating intractable cancers such as glioblastoma.  The goal of this study was to identify small-mol. inhibitors of Ape1 AP endonuclease.  For this purpose, a fluorescence-based high-throughput assay was used to screen a library of 60,000 small-mol. compds. for ability to inhibit Ape1 AP endonuclease activity.  Four compds. with IC50 values less than 10 μM were identified, validated, and characterized.  One of the most promising compds., designated Ape1 repair inhibitor 03 (2,4,9-trimethylbenzo[b][1,8]-naphthyridin-5-amine; AR03), inhibited cleavage of AP sites in vivo in SF767 glioblastoma cells and in vitro in whole cell exts. and inhibited purified human Ape1 in vitro.  AR03 has low affinity for double-stranded DNA and weakly inhibits the Escherichia coli endonuclease IV, requiring a 20-fold higher concn. than for inhibition of Ape1.  AR03 also potentiates the cytotoxicity of Me methanesulfonate and temozolomide in SF767 cells.  AR03 is chem. distinct from the previously reported small-mol. inhibitors of Ape1.  AR03 is a novel small-mol. inhibitor of Ape1, which may have potential as an oncotherapeutic drug for treating glioblastoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaq7PkWFe6bVg90H21EOLACvtfcHk0lj6BdRxjHHUpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyksL3J&md5=9fefb66d80fb0087db3b67755cd028c1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1124%2Fjpet.110.169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.110.169128%26sid%3Dliteratum%253Aachs%26aulast%3DBapat%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DL.%2BS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DLong%26aufirst%3DE.%2BC.%26aulast%3DGeorgiadis%26aufirst%3DM.%2BM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DNovel%2520small-molecule%2520inhibitor%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520blocks%2520proliferation%2520and%2520reduces%2520viability%2520of%2520glioblastoma%2520cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D988%26epage%3D998%26doi%3D10.1124%2Fjpet.110.169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zyromski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supuran, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of HIF1alpha under hypoxia by APE1/Ref-1 impacts CA9 expression: Dual targeting in patient-derived 3D pancreatic cancer models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2732</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1158%2F1535-7163.MCT-16-0253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=27535970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2722-2732&author=D.+P.+Logsdonauthor=M.+Grimardauthor=M.+Luoauthor=S.+Shahdaauthor=Y.+Jiangauthor=Y.+Tongauthor=Z.+Yuauthor=N.+Zyromskiauthor=E.+Schipaniauthor=F.+Cartaauthor=C.+T.+Supuranauthor=M.+Korcauthor=M.+Ivanauthor=M.+R.+Kelleyauthor=M.+L.+Fishel&title=Regulation+of+HIF1alpha+under+hypoxia+by+APE1%2FRef-1+impacts+CA9+expression%3A+Dual+targeting+in+patient-derived+3D+pancreatic+cancer+models&doi=10.1158%2F1535-7163.MCT-16-0253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of HIF1α under hypoxia by APE1/Ref-1 impacts CA9 expression: dual targeting in patient-derived 3D pancreatic cancer models</span></div><div class="casAuthors">Logsdon, Derek P.; Grimard, Michelle; Luo, Meihua; Shahda, Safi; Jiang, Yanlin; Tong, Yan; Yu, Zhangsheng; Zyromski, Nicholas; Schipani, Ernestina; Carta, Fabrizio; Supuran, Claudiu T.; Korc, Murray; Ivan, Mircea; Kelley, Mark R.; Fishel, Melissa L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2722-2732</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer-related mortality in the United States.  Aggressive treatment regimens have not changed the disease course, and the median survival has just recently reached a year.  Several mechanisms are proposed to play a role in PDAC therapeutic resistance, including hypoxia, which creates a more aggressive phenotype with increased metastatic potential and impaired therapeutic efficacy.  AP Endonuclease-1/Redox Effector Factor 1 (APE1/Ref-1) is a multifunctional protein possessing a DNA repair function in base excision repair and the ability to reduce oxidized transcription factors, enabling them to bind to their DNA target sequences.  APE1/Ref-1 regulates several transcription factors involved in survival mechanisms, tumor growth, and hypoxia signaling.  Here, we explore the mechanisms underlying PDAC cell responses to hypoxia and modulation of APE1/Ref-1 redox signaling activity, which regulates the transcriptional activation of hypoxia-inducible factor 1 alpha (HIF1α).  Carbonic anhydrase IX (CA9) is regulated by HIF1α and functions as a part of the cellular response to hypoxia to regulate intracellular pH, thereby promoting cell survival.  We hypothesized that modulating APE1/Ref-1 function will block activation of downstream transcription factors, STAT3 and HIF1α, interfering with the hypoxia-induced gene expression.  We demonstrate APE1/Ref-1 inhibition in patient-derived and established PDAC cells results in decreased HIF1α-mediated induction of CA9.  Furthermore, an ex vivo three-dimensional tumor coculture model demonstrates dramatic enhancement of APE1/Ref-1-induced cell killing upon dual targeting of APE1/Ref-1 and CA9.  Both APE1/Ref-1 and CA9 are under clin. development; therefore, these studies have the potential to direct novel PDAC therapeutic treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGivvfAYcobbVg90H21EOLACvtfcHk0lj6BdRxjHHUpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsr4%253D&md5=741b7b96d5d93e9331a210590898112c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0253%26sid%3Dliteratum%253Aachs%26aulast%3DLogsdon%26aufirst%3DD.%2BP.%26aulast%3DGrimard%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DShahda%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DZyromski%26aufirst%3DN.%26aulast%3DSchipani%26aufirst%3DE.%26aulast%3DCarta%26aufirst%3DF.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DKorc%26aufirst%3DM.%26aulast%3DIvan%26aufirst%3DM.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26atitle%3DRegulation%2520of%2520HIF1alpha%2520under%2520hypoxia%2520by%2520APE1%252FRef-1%2520impacts%2520CA9%2520expression%253A%2520Dual%2520targeting%2520in%2520patient-derived%25203D%2520pancreatic%2520cancer%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2722%26epage%3D2732%26doi%3D10.1158%2F1535-7163.MCT-16-0253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Logsdon, D. P.</span>; <span class="NLM_string-name">Cheng, H.</span>; <span class="NLM_string-name">Luo, M.</span>; <span class="NLM_string-name">Shahda, S.</span>; <span class="NLM_string-name">Ivan, M.</span>; <span class="NLM_string-name">Hao, Y.</span>; <span class="NLM_string-name">Tong, Y.</span>; <span class="NLM_string-name">Yu, Z.</span>; <span class="NLM_string-name">Zyromski, N.</span>; <span class="NLM_string-name">Schipani, E.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Supuran, C. T.</span>; <span class="NLM_string-name">Kelley, M. R.</span>; <span class="NLM_string-name">Fishel, M. L.</span></span> <span> </span><span class="NLM_article-title">Targeting APE1/Ref-1 results in inhibition of hypoxia signaling genes</span>. Presented at the AACR-NC-EORTC International Conference: Molecular Targets and Cancer Therapeutics, Boston, MA, December 2015;  <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">14</span>,  <span class="NLM_fpage">B158</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-B158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;key=10.1158%2F1535-7163.TARG-15-B158" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=B158&author=D.+P.+Logsdon&author=H.+Cheng&author=M.+Luo&author=S.+Shahda&author=M.+Ivan&author=Y.+Hao&author=Y.+Tong&author=Z.+Yu&author=N.+Zyromski&author=E.+Schipani&author=Y.+Liu&author=C.+T.+Supuran&author=M.+R.+Kelley&author=M.+L.+Fishel&title=Targeting+APE1%2FRef-1+results+in+inhibition+of+hypoxia+signaling+genes&doi=10.1158%2F1535-7163.TARG-15-B158"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-B158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-B158%26sid%3Dliteratum%253Aachs%26aulast%3DLogsdon%26aufirst%3DD.%2BP.%26atitle%3DTargeting%2520APE1%252FRef-1%2520results%2520in%2520inhibition%2520of%2520hypoxia%2520signaling%2520genes%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DB158%26doi%3D10.1158%2F1535-7163.TARG-15-B158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3','PDB','6MK3'); return false;">PDB: 6MK3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK','PDB','6MKK'); return false;">PDB: 6MKK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM','PDB','6MKM'); return false;">PDB: 6MKM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO','PDB','6MKO'); return false;">PDB: 6MKO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHE','PDB','4QHE'); return false;">PDB: 4QHE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DE9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DE9','PDB','1DE9'); return false;">PDB: 1DE9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QHD','PDB','4QHD'); return false;">PDB: 4QHD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01529">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_70671"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01529">10.1021/acs.jmedchem.8b01529</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figure S1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_002.csv">CSV</a>)</p></li><li><p class="inline">Coordinate files used to generate <a id="rightTab-" class=" internalNav" href="#fig4">Figure <a id="rightTab-" class=" internalNav" href="#fig4">4</a></a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Coordinate files used to generate <a id="rightTab-" class=" internalNav" href="#fig5">Figure <a id="rightTab-" class=" internalNav" href="#fig5">5</a></a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_004.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_001.pdf">jm8b01529_si_001.pdf (159.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_002.csv">jm8b01529_si_002.csv (2.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_003.pdb">jm8b01529_si_003.pdb (295.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01529/suppl_file/jm8b01529_si_004.pdb">jm8b01529_si_004.pdb (254.79 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates have been deposited with the PDB with the following accession numbers: APE1 bound to DMSO (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MK3">6MK3</a>), DMSO/Mg<sup>2+</sup> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKK">6MKK</a>), DMSO/Tris (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKM">6MKM</a>), and GLC (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MKO">6MKO</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01529&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01529%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-4%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01529" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799cea37f4e3cd6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
