
<html lang="en"     class="pb-page"  data-request-id="8d45f128-35cd-4480-813d-2ab500f887f5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01372;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2020.63.issue-1;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Ligand-Based Design of Allosteric Retinoic Acid Receptor-Related Orphan Receptor γt (RORγt) Inverse Agonists" /></meta><meta name="dc.Creator" content="Femke A.  Meijer" /></meta><meta name="dc.Creator" content="Richard G.  Doveston" /></meta><meta name="dc.Creator" content="Rens M.J.M.  de Vries" /></meta><meta name="dc.Creator" content="Gaël M.  Vos" /></meta><meta name="dc.Creator" content="Alex A.A.  Vos" /></meta><meta name="dc.Creator" content="Seppe  Leysen" /></meta><meta name="dc.Creator" content="Marcel  Scheepstra" /></meta><meta name="dc.Creator" content="Christian  Ottmann" /></meta><meta name="dc.Creator" content="Lech-Gustav  Milroy" /></meta><meta name="dc.Creator" content="Luc  Brunsveld" /></meta><meta name="dc.Description" content="Retinoic acid receptor-related orphan receptor γt (RORγt) is a nuclear receptor associated with the pathogenesis of autoimmune diseases. Allosteric inhibition of RORγt is conceptually new, unique f..." /></meta><meta name="Description" content="Retinoic acid receptor-related orphan receptor γt (RORγt) is a nuclear receptor associated with the pathogenesis of autoimmune diseases. Allosteric inhibition of RORγt is conceptually new, unique f..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 10, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01372" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01372" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01372" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01372" /></link>
        
    
    

<title>Ligand-Based Design of Allosteric Retinoic Acid Receptor-Related Orphan Receptor γt (RORγt) Inverse Agonists | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01372" /></meta><meta property="og:title" content="Ligand-Based Design of Allosteric Retinoic Acid Receptor-Related Orphan Receptor γt (RORγt) Inverse Agonists" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0013.jpeg" /></meta><meta property="og:description" content="Retinoic acid receptor-related orphan receptor γt (RORγt) is a nuclear receptor associated with the pathogenesis of autoimmune diseases. Allosteric inhibition of RORγt is conceptually new, unique for this specific nuclear receptor, and offers advantages over traditional orthosteric inhibition. Here, we report a highly efficient in silico-guided approach that led to the discovery of novel allosteric RORγt inverse agonists with a distinct isoxazole chemotype. The the most potent compound, 25 (FM26), displayed submicromolar inhibition in a coactivator recruitment assay and effectively reduced IL-17a mRNA production in EL4 cells, a marker of RORγt activity. The projected allosteric mode of action of 25 was confirmed by biochemical experiments and cocrystallization with the RORγt ligand binding domain. The isoxazole compounds have promising pharmacokinetic properties comparable to other allosteric ligands but with a more diverse chemotype. The efficient ligand-based design approach adopted demonstrates its versatility in generating chemical diversity for allosteric targeting of RORγt." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01372"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01372">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01372&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01372&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01372&amp;href=/doi/10.1021/acs.jmedchem.9b01372" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 241-259</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01157" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01567" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Ligand-Based Design of Allosteric Retinoic Acid Receptor-Related Orphan Receptor γt (RORγt) Inverse Agonists</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Femke A. Meijer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Femke A. Meijer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Femke+A.++Meijer">Femke A. Meijer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4412-9968" title="Orcid link">http://orcid.org/0000-0003-4412-9968</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard G. Doveston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard G. Doveston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div><div class="loa-info-affiliations-info">Leicester Institute of Structural and Chemical Biology and Department of Chemistry, University of Leicester, University Road, Leicester LE1 7RH, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard+G.++Doveston">Richard G. Doveston</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rens M.J.M. de Vries</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rens M.J.M. de Vries</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rens+M.J.M.++de+Vries">Rens M.J.M. de Vries</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gaël M. Vos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gaël M. Vos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ga%C3%ABl+M.++Vos">Gaël M. Vos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex A.A. Vos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex A.A. Vos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex+A.A.++Vos">Alex A.A. Vos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Seppe Leysen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Seppe Leysen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Seppe++Leysen">Seppe Leysen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcel Scheepstra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcel Scheepstra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcel++Scheepstra">Marcel Scheepstra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Ottmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Ottmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Ottmann">Christian Ottmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7315-0315" title="Orcid link">http://orcid.org/0000-0001-7315-0315</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lech-Gustav Milroy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lech-Gustav Milroy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lech-Gustav++Milroy">Lech-Gustav Milroy</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Luc Brunsveld</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luc Brunsveld</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Technische Universiteit Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#224e0c4050574c5154474e46625657470c4c4e"><span class="__cf_email__" data-cfemail="4824662a3a3d263b3e2d242c083c3d2d662624">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luc++Brunsveld">Luc Brunsveld</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5675-511X" title="Orcid link">http://orcid.org/0000-0001-5675-511X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01372&amp;href=/doi/10.1021%2Facs.jmedchem.9b01372" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 241–259</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 10, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 August 2019</li><li><span class="item_label"><b>Published</b> online</span>10 December 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 January 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01372" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01372</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc text">with CC-BY-NC-ND<a href="/page/policy/authorchoice_ccbyncnd_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01372"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3437</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01372" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Ligand-Based Design of Allosteric Retinoic Acid Receptor-Related Orphan Receptor γt (RORγt) Inverse Agonists&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Femke&quot;,&quot;last_name&quot;:&quot;A. Meijer&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;G. Doveston&quot;},{&quot;first_name&quot;:&quot;Rens&quot;,&quot;last_name&quot;:&quot;M.J.M. de Vries&quot;},{&quot;first_name&quot;:&quot;Gaël&quot;,&quot;last_name&quot;:&quot;M. Vos&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;A.A. Vos&quot;},{&quot;first_name&quot;:&quot;Seppe&quot;,&quot;last_name&quot;:&quot;Leysen&quot;},{&quot;first_name&quot;:&quot;Marcel&quot;,&quot;last_name&quot;:&quot;Scheepstra&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Ottmann&quot;},{&quot;first_name&quot;:&quot;Lech-Gustav&quot;,&quot;last_name&quot;:&quot;Milroy&quot;},{&quot;first_name&quot;:&quot;Luc&quot;,&quot;last_name&quot;:&quot;Brunsveld&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;241-259&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01372&quot;},&quot;abstract&quot;:&quot;Retinoic acid receptor-related orphan receptor γt (RORγt) is a nuclear receptor associated with the pathogenesis of autoimmune diseases. Allosteric inhibition of RORγt is conceptually new, unique for this specific nuclear receptor, and offers advantages over traditional orthosteric inhibition. Here, we report a highly efficient in silico-guided approach that led to the discovery of novel allosteric RORγt inverse agonists with a distinct isoxazole chemotype. The the most potent compound, 25 (FM26), displayed submicromolar inhibition in a coactivator recruitment assay and effectively reduced IL-17a mRNA production in EL4 cells, a marker of RORγt activity. The projected allosteric mode of action of 25 was confirmed by biochemical experiments and cocrystallization with the RORγt ligand binding domain. The isoxazole compounds have promising pharmacokinetic properties comparable to other allosteric ligands but with a more diverse chemotype. The efficient ligand-based design approach adopted demonstrates its versati&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01372&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01372" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01372&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01372" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01372&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01372" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01372&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01372&amp;href=/doi/10.1021/acs.jmedchem.9b01372" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01372" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01372" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01372%26sid%3Dliteratum%253Aachs%26pmid%3D31821760%26genre%3Darticle%26aulast%3DMeijer%26date%3D2020%26atitle%3DLigand-Based%2BDesign%2Bof%2BAllosteric%2BRetinoic%2BAcid%2BReceptor-Related%2BOrphan%2BReceptor%2B%25CE%25B3t%2B%2528ROR%25CE%25B3t%2529%2BInverse%2BAgonists%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D1%26spage%3D241%26epage%3D259%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292589" title="Agonists">Agonists</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291274" title="Addition reactions">Addition reactions</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/jmcmar.2020.63.issue-1/20200109/jmcmar.2020.63.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Retinoic acid receptor-related orphan receptor γt (RORγt) is a nuclear receptor associated with the pathogenesis of autoimmune diseases. Allosteric inhibition of RORγt is conceptually new, unique for this specific nuclear receptor, and offers advantages over traditional orthosteric inhibition. Here, we report a highly efficient in silico-guided approach that led to the discovery of novel allosteric RORγt inverse agonists with a distinct isoxazole chemotype. The the most potent compound, <b>25</b> (<b>FM26</b>), displayed submicromolar inhibition in a coactivator recruitment assay and effectively reduced IL-17a mRNA production in EL4 cells, a marker of RORγt activity. The projected allosteric mode of action of <b>25</b> was confirmed by biochemical experiments and cocrystallization with the RORγt ligand binding domain. The isoxazole compounds have promising pharmacokinetic properties comparable to other allosteric ligands but with a more diverse chemotype. The efficient ligand-based design approach adopted demonstrates its versatility in generating chemical diversity for allosteric targeting of RORγt.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11806" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11806" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The nuclear receptor (NR) RORγt has emerged as an important therapeutic target in recent years because of its important role in both cancer and autoimmune disease. Inhibition of RORγt is a promising therapeutic strategy for the treatment of prostate cancer because it stimulates androgen receptor (AR) gene transcription.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> However, RORγt is most prominently targeted for inhibition because of its essential role in promoting T helper 17 (Th17) cell differentiation.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> Th17 cells produce the cytokine IL-17 which is strongly implicated in the pathogenesis of autoimmune diseases<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> such as psoriasis,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> multiple sclerosis,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and inflammatory bowel disease.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Disrupting the Th17/IL-17 pathway using IL-17 monoclonal antibodies (mAb) is a successful therapeutic strategy, with three mAbs approved for the treatment of plaque psoriasis: secukinumab (Cosentyx),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> brodalumab (Siliq),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and ixekizumab (Taltz).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Inhibition of RORγt with small molecules to disrupt the Th17/IL-17 pathway has been the focus of much research in recent years,<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20">(13−20)</a> with several compounds having progressed to clinical trials.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">RORγt contains a hydrophobic ligand binding pocket located within a ligand binding domain (LBD) that is highly conserved across the NR family.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, its transcriptional activity is not dependent on ligand binding because the apo protein retains the C-terminal helix 12 (H12) in a conformational state that allows for partial recruitment of coactivator proteins.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Although formally an orphan receptor with no proven endogenous ligands, RORγt is responsive to binding of naturally occurring cholesterol derivatives. Hydroxycholesterols have been shown to be effective agonists that stabilize H12 in such a way to further promote coactivator binding.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In contrast, digoxin (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) is an inverse agonist that stabilizes H12 in a conformation that is unsuitable for coactivator binding but promotes corepressor binding, thus leading to diminished gene transcription.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Numerous synthetic inverse agonists are also known, including T0901317 (<b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In all these cases, the ligands target the same orthosteric ligand binding pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Orthosteric and allosteric RORγt ligand binding sites are shown by overlay of the crystal structures of RORγt LBD in complex with orthosteric inverse agonist <b>2</b> (orange, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NB6">4NB6</a>) and allosteric inverse agonist <b>3</b> (blue, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>). The structures of the orthosteric inverse agonist <b>1</b> and allosteric inverse agonist <b>4</b> are also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">NR orthosteric ligand binding pockets are the target for numerous and highly effective drug molecules.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Nevertheless, the highly conserved nature of this pocket across the NR family has led to issues associated with selectivity and mutation-induced resistance. Furthermore, dosing levels must be appropriate to compete with endogenous ligands. Molecules that target allosteric binding sites on NRs could circumvent such problems, for example because of the chemical uniqueness of the pocket and the absence of a competitive endogenous ligand. Such allosteric compounds are therefore extremely valuable for both drug discovery and chemical biology applications.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> The discovery that the potent RORγt inverse agonists MRL-871 (<b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and later <b>4</b><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> target a previously unreported allosteric binding site within the RORγt LBD was therefore highly significant. These ligands were observed to directly interact with the activation function loop between H11 and H12 (AF-2 domain), thus forcing H12 to adopt an unusual conformation that prevents coactivator recruitment (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p last">Allosteric modulation of RORγt has enormous potential as a novel therapeutic strategy, but the examples of ligands that unambiguously target the allosteric pocket have been limited to compounds based on closely related chemotypes containing indazole or imidazopyridine cores.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> As an example, indazoles <b>3</b> and <b>4</b> displayed promising in vivo activity,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> but challenges remain, such as PPARγ cross-activity and pharmacokinetic (PK) profiles, for which novel chemotypes are needed.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In order to better exploit the strategy of allosteric modulation for therapeutic purposes, there is thus an urgent need to identify novel chemotypes targeting the allosteric site. In this study, we report the design, synthesis, and evaluation of a novel class of RORγt allosteric inverse agonists. The novel chemotype, discovered by in silico-guided pharmacophore screening and optimization, is based on a trisubstituted isoxazole core that, following efficient optimization of two substituents, led to the discovery of a submicromolar inverse agonist. Protein X-ray crystallography and biophysical data unambiguously proved the designed allosteric mode of action. The compounds effectively inhibit cellular IL-17a expression and thus constitute valuable leads in the development of treatments for autoimmune diseases. To the best of our knowledge, our highly efficient in silico-guided approach is the first example of a medicinal chemistry program to overtly identify and develop a novel chemotype that targets the RORγt allosteric site.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20188" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20188" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  In Silico Pharmacophore Screen</h3><div class="NLM_p">In order to identify novel chemotypes for chemical optimization, we used the crystal structure of RORγt LBD in complex with <b>3</b> as the basis for an in silico 3D pharmacophore screen against virtual compound libraries. An analogous scaffold hopping approach had been used previously to identify similar scaffolds to <b>3</b> such as the potent inverse agonist thienopyrrazole <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), although an allosteric mode of action was not proven.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> We created a 3D pharmacophore hypothesis based on the crystal structure of <b>3</b> bound to the allosteric pocket using Phase (Schrödinger 2017–2).<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> Six structural features of <b>3</b> known to be important for activity were incorporated in the hypothesis: the three six-membered aromatic rings, an anionic group, and two hydrophobic substituents (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This hypothesis was used to interrogate a virtual library of 289,174 compounds from the Asinex Gold–Platinum collection of drug-like molecules.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Compounds matching at least four out of the six pharmacophore features were deemed to be a good hit. These were ranked using the “Phase Screen Score” with higher scores indicating a better alignment with the hypothesis. The Phase Screen scores for <b>3</b> and <b>5</b> were used as contextual references. The four highest ranking hit structures were all found to be based around the same trisubstituted isoxazole scaffold with <b>6</b> returned as the best match (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This same scaffold was present in 13 of the top 30 hits. However, in each case we noted that only four out of six pharmacophore features were matched. Therefore, we designed two virtual ligands, <b>7</b> and <b>8</b>, that incorporated five and six of the features, respectively. As expected, this led to improved Phase Screen Scores (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), and these compounds were therefore selected as initial targets for experimental investigation.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. 3D Pharmacophore screening identifies a compound class with a novel isoxazole-based chemotype for experimental evaluation. The structural features of <b>3</b> incorporated into the pharmacophore hypothesis are indicated: orange = aromatic rings, green = hydrophobic groups, and red = anionic group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Exploratory Structure–Activity Relationship Study</h3><div class="NLM_p">Isoxazoles <b>7</b> and <b>8</b> were synthesized via [3 + 2] dipolar cycloaddition of a nitrile oxide (generated in situ from the oxime chloride <b>9a</b>) and a commercially available alkyne.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The regiochemistry of the resulting trisubstituted isoxazole esters <b>10</b> was confirmed by 2D-NMR experiments (key HMBC correlations are highlighted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Ester hydrolysis followed by amide coupling of <i>tert</i>-butyl-4-amino benzoate via the respective acid chloride, and finally deprotection of the <i>tert</i>-butyl ester furnished the target compounds in an efficient manner (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Trisubstituted Isoxazoles <b>7</b> and <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Key HMBC correlations used to confirm the regiochemistry of <b>10a</b> and <b>10b</b> are shown. The <sup>13</sup>C-NMR signals for the C-5 carbons are distinctively downfield at 175 and 173 ppm, respectively. Reagents and conditions: (a) NH<sub>2</sub>OH·HCl, NaOH (aq), EtOH, rt, 18 h, 83%; (b) NCS, DMF, 60 °C, 18 h, 86%; then (c) alkyne, NEt<sub>3</sub>, THF, 80 °C, 4 h, 69% (<b>10a</b>), 80% (<b>10b</b>); (d) LiOH, EtOH, H<sub>2</sub>O, 70 °C, 8 h, 84% (<b>11a</b>), 95% (<b>11b</b>); and (e) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) <i>tert</i>-butyl-4-amino benzoate, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, 6 h; and (iii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, 42% (<b>7</b>), 69% (<b>8</b>).</p></p></figure><div class="NLM_p">To determine if the compounds showed a functional response in terms of RORγt affinity for a coactivator, <b>7</b> and <b>8</b> were tested in a time-resolved FRET (TR-FRET) coactivator recruitment assay.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Remarkably, both compounds inhibited coactivator recruitment in a dose-dependent manner. The phenyl derivative <b>8</b> was found to be significantly more potent than the methyl derivative <b>7</b>: half-maximum inhibitory concentrations (IC<sub>50</sub>) of 53.5 ± 2.9 μM for <b>8</b> compared to >100 μM for <b>7</b>. In line with previous reports, <b>3</b> and <b>5</b> were determined to be significantly more potent with an IC<sub>50</sub> of 7.8 ± 0.5 nM and 425 ± 61 nM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure-Activity Relationships around the C-4 Isoxazole Position<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">Glide score</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">0.0078 ± 0.0005</td><td class="colsep0 rowsep0" align="left">– 14.576</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">0.425 ± 0.061</td><td class="colsep0 rowsep0" align="left">– 13.109</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">– 13.372</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">53.5 ± 2.9</td><td class="colsep0 rowsep0" align="left">– 14.184</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">– 10.130</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">– 13.724</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">73.9 ± 3.4</td><td class="colsep0 rowsep0" align="left">– 12.995</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">91.1 ± 4.6</td><td class="colsep0 rowsep0" align="left">– 14.308</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">8.76 ± 0.48</td><td class="colsep0 rowsep0" align="left">– 12.020</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">9.60 ± 0.60</td><td class="colsep0 rowsep0" align="left">– 14.012</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">– 13.550</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">30.9 ± 1.3</td><td class="colsep0 rowsep0" align="left">– 13.519</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">62.6 ± 4.4</td><td class="colsep0 rowsep0" align="left">– 13.003</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">TR-FRET IC<sub>50</sub> values (μM) and respective Glide docking scores are shown. TR-FRET data is recorded in triplicate; values are representative of >3 repeated experiments.</p></div></div></div><div class="NLM_p">In view of these highly promising TR-FRET results with the in silico derived compounds around the trisubstituted isoxazole scaffold already showing activity, phenyl isoxazole <b>8</b> was selected as the focus of a subsequent structure–activity relationship (SAR) study focusing on the isoxazole C-4 position. As such, a small library of 11 derivatives was synthesized using carboxylic acid <b>11b</b> as the cornerstone intermediate (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) and evaluated using the coactivator recruitment assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). While limited in size, this SAR study indicated that a benzoic acid-containing substituent at the C-4 position was essential for potency: examples bearing no C-4 substitution (<b>11b</b>), a <i>para</i>-benzoate (<b>14</b>), or a methylene carboxylic acid (<b>15</b>) showed much reduced potency compared to the initial hit. Moving the acid moiety to the meta-position (<b>16</b>) or adding a <i>meta</i>-fluoro substituent (<b>17</b>) somewhat lowered the activity. However, the insertion of a single methylene unit between the amide and benzoic acid moieties (<b>18</b>) led to a 6-fold increase in potency compared to the initial hit. The corresponding amine (<b>19</b>) displayed similar activity. Finally, reversing the relative positions of carbonyl and nitrogen components of the amide bond (<b>20</b>–<b>22</b>) did not result in a corresponding increase in potency.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of C-4 Isoxazole Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) NH<sub>2</sub>R, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, 6 h, 27–87%; (b) LiOH, MeOH, H<sub>2</sub>O, 70 °C, 8 h, 43–99%; (c) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) MeNH(OMe), NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 6 h; (iii) LiAlH<sub>4</sub>, THF, 0 °C, 30 min, 65%; (d) (i) ethyl-4-aminobenzoate, AcOH, MeOH, reflux, 24 h; (ii) NaCNBH<sub>3</sub>, MeOH, reflux, 12 h, 31%; (e) (i) DPPA, <i>t</i>-BuOH, 85 °C, 18 h; (ii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 8 h, 59%; (f) (i) monomethyl terephthalate, SOCl<sub>2</sub>, 50 °C, 2 h; (ii) <b>13</b>, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 76%; (g) (i) methyl-4-formyl benzoate, AcOH, MeOH, reflux, 24 h; (ii) NaCNBH<sub>3</sub>, MeOH, reflux, 18 h, 43%; and (h) methyl-4-(chlorosulfonyl)benzoate, pyridine, 60 °C, 24 h, 71%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.3.  In Silico Docking Directs Secondary SAR Study</h3><div class="NLM_p">In order to further improve the potency of our compounds, we next explored the SAR at the isoxazole C-5 position. For this, molecular docking (Glide, Schrodinger 2017–2)<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> was used to select, with an attention to synthetic resource, C-5 substituents that were optimal for allosteric binding and therefore activity. For the study, a single C-4 substituent, the amine of compound <b>19</b> was chosen based on its experimental activity and in silico docking score (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, vide infra). A virtual library of 84 C-5 analogues was enumerated using the open-source ChemT software.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This library was docked against the allosteric site of RORγt as defined by the X-ray crystal structure of <b>3</b> in complex with the RORγt LBD.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> A single docking pose was returned for each virtual ligand, and these were ranked using the “Glide Score”, an empirical measure of binding enthalpy.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> We contextualized these scores by comparison to those of compounds with known activity. The results (summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a> for full information) indicated that smaller heteroaromatic moieties at the C-5 position would improve allosteric binding of the isoxazole ligands relative to <b>19</b>, heteroatoms at the 2-postion were predicted to be optimal, for example, furan <b>23</b> and thiophene <b>24</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The introduction of a hydrogen-bond donor on the ring (specifically at the 3-position) was predicted to be even more beneficial: docking poses indicated that an additional hydrogen-bonding interaction with the backbone of helix 4 might be possible (e.g., pyrrole <b>25</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Bulkier substituents were predicted to be detrimental for binding (e.g., naphthyl <b>26</b>). To explore the predicted effect of a hydrogen-bond donating group further we interrogated a designed subset of ligands in the same docking experiment (see the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). None of these ligands showed an improved Glide score compared to pyrrole <b>25</b>. However, we noted that 3-hydroxyl substitution of the C-5 phenyl ring (<b>27</b>) was predicted to significantly enhance binding relative to <b>19</b>. To validate our findings experimentally, we selected a cross section of five derivatives for synthesis (i.e., <b>23</b>–<b>27</b>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. In silico modeled docking pose of <b>25</b> (green) overlaid with crystal structure of RORγt with <b>3</b> (orange) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>). For <b>25</b>, the potential additional hydrogen bond with the RORγt H4 backbone is indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure-Activity Relationships around the C-5 Isoxazole Position<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0011.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0012.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup><sup>a</sup></sup><p class="last">TR-FRET IC<sub>50</sub> values (μM) and respective Glide docking scores are shown. TR-FRET data is recorded in triplicate; values are representative of >3 repeated experiments.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">2.4.  Docking-Guided C-5 SAR Study</h3><div class="NLM_p">To expedite the synthesis of isoxazole analogues with various C-5 and C-4 substituents, we redesigned our synthetic approach. It was envisaged that 5-bromo-4-carboxy isoxazole intermediate <b>30</b> would enable later stage introduction of the desired C-5 substituents via palladium-mediated cross-coupling chemistry. Introduction of C-4 substituents by manipulation of a carbonyl functional group (as developed previously) would then be possible (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Retrosynthetic Analysis of Trisubstituted Isoxazole <b>28</b> Allowing for Late-Stage Diversification</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The intermediate <b>30</b> was prepared using analogous methodology to that used previously. In this case it was necessary to isolate nitrile oxide <b>33</b> prior to [3 + 2] cycloaddition with alkynyl bromide <b>32</b>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> An efficient cycloaddition reaction led to an essentially quantitative recovery of a 7:3 mixture of 5-bromoisoxazole <b>30a</b> and 4-bromoisoxazole <b>30b</b> as determined by <sup>1</sup>H NMR (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). This result was in close alignment with literature examples that indicated the 5-bromo isomer would predominate.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The mixture of regioisomers was purified by recrystallization from hot <i>n</i>-heptane resulting in the isolation of a 97:3 regiomeric mixture (43% recovery) that was employed in subsequent steps. Assignment of the 5-bromoisoxazole <b>30a</b> as the major regioisomer was confirmed by 2D-NMR analysis of downstream products and by synthesis via an independent route (see the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>).</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Isoxazole C-5 Analogues <b>23–27</b>.<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>“R” groups are defined in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Reagents and conditions: (a) NBS, AgNO<sub>3</sub>, Me<sub>2</sub>C(O), rt, 20 h, 80%; (b) (i) NCS, DMF, 60 °C, 18 h, (ii) NEt<sub>3</sub>, THF, rt, 30 min, 85%; (c) THF, 80 °C, 4 h, <b>30a</b> 43%; (d) RB(pin), Pd(dppf)Cl<sub>2</sub>, DME, 85 °C, 8 h, 39–58%; (e) (i) LiAlH<sub>4</sub>, THF, 0 °C → rt, 2 h, then (ii) DMP, CH<sub>2</sub>Cl<sub>2</sub>, rt, 8 h, 51–96%; (f) <i>tert</i>-butyl-4-amino benzoate, MeOH, AcOH, reflux, 24 h then (ii) NaBH<sub>4</sub>, EtOH, 85 °C, 2–6 h, 16–24%; (iii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, <b>23</b>, <b>24</b>, <b>26</b>, 48–73%; and (g) methyl-4-amino benzoate, MeOH, AcOH, reflux, 24 h then (ii) NaBH<sub>4</sub>, MeOH, reflux, 2–4 h, 16–19%; and (iii) LiOH, MeOH, H<sub>2</sub>O, 70 °C, 8 h, <b>25</b>, 57%, <b>27</b>, 99%.</p></p></figure><div class="NLM_p">The desired substituents were introduced at the C-5 position by way of a Suzuki cross-coupling with a pinacol boronate<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (to give intermediates <b>34</b>–<b>38</b>) before conversion of the C-4 ester to an aldehyde (<b>39</b>–<b>43</b>) and reductive amination (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The lability of the 5-bromo group under the conditions for ester reduction dictated the order in which the synthesis steps were performed. Hydrolysis of the benzoic methyl ester to the free acid yielded the desired compounds <b>23</b>–<b>27</b>.</div><div class="NLM_p">In order to explore the SAR around the isoxazole C-5 position, the five analogues prepared in this second synthesis campaign were evaluated using the HTRF coactivator recruitment assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). We were gratified to observe that furan <b>23</b> gave a 9-fold improvement in potency compared to phenyl <b>19</b>. By comparison, thiophene <b>24</b> was slightly less potent. Most significantly, pyrrole <b>25</b>, which also showed the most beneficial Glide score, was 36-fold more potent than <b>19</b> and with an IC<sub>50</sub> value lower than the putative allosteric modulator <b>5</b>. These results were in excellent agreement with the in silico Glide scores obtained (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and the improvements in potency are a notable step toward emulating the high potency of indazole <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). As predicted, the bulky naphthyl group of <b>26</b> was detrimental for activity such that no IC<sub>50</sub> curve could be fitted. The phenol derivative <b>27</b> showed a small improvement in potency compared to <b>19</b>. For this more bulky group at the C-5 position, compared to pyrrole <b>25</b>, the potential for additional hydrogen bonding, as indicated in the docking study, is thus not strongly expressed.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical RORγt assay data for <b>25</b>, <b>3</b>, and <b>5</b>. (A) Dose–response curves from the TR-FRET coactivator recruitment assay; (B–E) dose–response curves from the competitive TR-FRET coactivator recruitment assay with fixed concentrations of cholesterol (0, 0.25, and 1.0 μM); and (F) dose–response curves from the ligand displacement HTRF assay using <b>44</b> (G) as an allosteric probe.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">2.5.  Mode-of-Action Studies</h3><div class="NLM_p">The allosteric mode-of-action for the novel lead compound <b>25</b> was first explored using a competitive TR-FRET coactivator recruitment assay against fixed concentrations of cholesterol (an orthosteric agonist). If an allosteric ligand and cholesterol bind in a noncompetitive manner at different sites on the RORγt LBD then the IC<sub>50</sub> of the allosteric ligand should be independent of cholesterol concentration. By contrast, ligands competing for the same binding site should show a cholesterol-dependent activity profile whereby increasing cholesterol concentration should result in a corresponding increase in IC<sub>50</sub> of the competing ligand.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In our assay, increasing concentrations of <b>25</b> perturbed coactivator recruitment in the absence of cholesterol with an IC<sub>50</sub> value of 247.8 ± 17.7 nM. Interestingly, increasing concentrations of cholesterol indeed resulted not in an increase but in a further decrease in the IC<sub>50</sub> value for <b>25</b> with a concomitant sharpening of the Hill slope (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This result provides strong evidence not only for an allosteric mode-of-action but also for cooperative behavior between orthosteric and allosteric ligand binding. The same profile was observed for <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), providing the first evidence that this compound also modulates RORγt activity in an allosteric fashion. Indazole <b>3</b> also exhibited this behavior (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). By comparison, the IC<sub>50</sub> value for the orthosteric inverse agonist <b>1</b> increased as the concentration of cholesterol increased (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Collectively, our competitive assay data provided strong evidence that <b>25</b> functioned as an allosteric inverse agonist.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> and Hill Slope Values Observed in the Competitive TR-FRET Cofactor Recruitment Assay</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">0 μM cholesterol</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">0.25 μM cholesterol</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">1.0 μM cholesterol</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="±">Hill slope</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="±">Hill slope</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="±">Hill slope</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char="±">247.8 ± 17.7</td><td class="colsep0 rowsep0" align="char" char="±">–0.77 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">138.0 ± 5.9</td><td class="colsep0 rowsep0" align="char" char="±">–0.86 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">94.1 ± 3.3</td><td class="colsep0 rowsep0" align="char" char="±">–1.01 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char="±">547.3 ± 60.1</td><td class="colsep0 rowsep0" align="char" char="±">–0.74 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">299.5 ± 18.0</td><td class="colsep0 rowsep0" align="char" char="±">–0.87 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">268.9 ± 18.8</td><td class="colsep0 rowsep0" align="char" char="±">–0.90 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char="±">12.7 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">–0.97 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">9.4 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">–1.04 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">7.8 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">–1.20 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">7012 ± 588</td><td class="colsep0 rowsep0" align="char" char="±">–0.76 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">33620 ± 1649</td><td class="colsep0 rowsep0" align="char" char="±">–0.77 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">85400 ± 4276</td><td class="colsep0 rowsep0" align="char" char="±">–1.01 ± 0.06</td></tr></tbody></table></div></div><div class="NLM_p">To further confirm the allosteric mode-of-action for <b>25</b> on RORγt, we used an orthogonal assay to directly probe for allosteric ligand binding, as opposed to measuring indirect effects on coactivator recruitment. This assay used the previously described AlexaFluor647-labeled MRL-871 derivative <b>44</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>G), which upon binding to RORγt shows fluorescent emission as a result of FRET from an anti-His terbium cryptate antibody donor.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The results of this experiment indeed corroborated the data obtained from the competitive cofactor recruitment assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F): the isoxazole <b>25</b> displaced the allosteric probe <b>44</b> with an IC<sub>50</sub> = 117.5 ± 8.5 nM, which was lower than that of <b>5</b> (IC<sub>50</sub> = 180.0 ± 17.5 nM). As expected, indazole <b>3</b> was highly potent (IC<sub>50</sub> = 17.3 ± 1.4 nM).</div><div class="NLM_p">Indazole <b>3</b> had previously been shown to be selective for RORγt over other NRs (>100-fold), with only minor cross-activity on PPARγ.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> To give an indication of the cross-reactivity of the isoxazole series on PPARγ, an HTRF coactivator recruitment assay was performed with compounds <b>3</b>, <b>5</b>, and isoxazoles <b>19</b> and <b>23</b>–<b>27</b>. <b>3</b> and <b>5</b> show IC<sub>50</sub> values of 7.2 μM and 14.7 μM, respectively, for PPARγ (vs 7.8 nM and 425 nM for RORγt) (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), meaning that they show some cross-reactivity to PPARγ but still are 923- and 35-fold selective for RORγt. <b>25</b> and all other compounds of the isoxazole series result in only weak to no PPARγ inhibition (IC<sub>50</sub> values >50 μM), indicating that the isoxazole scaffold leads to favorably low PPARγ cross-reactivity. Thus, these data indicate that the novel class of allosteric isoxazole inverse agonists features potential as efficacious and selective RORγt inverse agonists.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IC<sub>50</sub> Values Observed in the Competitive TR-FRET Cofactor Recruitment Assay with PPARγ</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">7.2 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">14.7 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">78.6 ± 5.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">99.3 ± 6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">2.6.  Crystallography</h3><div class="NLM_p">Co-crystallization studies were performed for the most potent isoxazole <b>25</b> with the RORγt-LBD, to provide molecular insights in the ligand–receptor interaction. Crystals grew in a <i>P6</i><sub><i>1</i></sub><i>22</i> space group and diffracted to a resolution of 1.61 Å (<a class="ref internalNav" href="#notes-2" aria-label="Table S6">Table S6</a>). In the experimental electron density map, clear density for compound <b>25</b> is observed in the allosteric site, formed by helices 4, 5, 11, and 12 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <a class="ref internalNav" href="#notes-2" aria-label="Figure S2">Figure S2</a>). The compound binds to this allosteric site in a similar orientation as <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), as was predicted by our docking studies (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The 2,6-disubstituted phenyl ring common to both <b>3</b> and of <b>25</b> is located in the exact same part of the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Moreover, hydrogen-bonding interactions between the carboxylic acid group and the main-chain amide hydrogen atoms of A497 and F498, as well as with the side chain of residue Q329, are also evident in both structures. Unique to <b>25</b> is the pyrrole ring, which is oriented to allow a hydrogen bond interaction with the main-chain carbonyls of residues L353 and K354 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The isoxazole scaffold also allows a deeper penetration of this compound toward helix 4 of RORγt. In the case of isoxazole <b>25</b>, the AF-2 loop of the protein and the allosteric ligand are positioned slightly further apart as compared to <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). These structural data provide clear evidence for the allosteric binding of <b>25</b> to RORγt in an orientation that was predicted with remarkable accuracy in the docking study (<a class="ref internalNav" href="#notes-2" aria-label="Figure S3">Figure S3</a>) but with specific additional molecular effects resulting from the novel isoxazole scaffold and pyrrole based substition pattern.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Co-crystal structure of RORγt with compound <b>25</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAL">6SAL</a>). (A) The tertiary structure of RORγt bound to <b>25</b> (stick representation). The final 2Fo–Fc electron density map of <b>25</b> is shown as an isomesh contoured at 1σ; (B) overlay of the crystal structure of RORγt bound to <b>25</b> and RORγt bound to <b>3</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C4O">5C4O</a>); and (C) zoom-in on the allosteric pocket of RORγt showing the interactions between <b>25</b> and the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">2.7.  Isoxazole 25 Inhibits IL-17a Expression in EL4 Cells</h3><div class="NLM_p">EL4 is a murine lymphoblast cell line that constitutively expresses RORγt. Because RORγt promotes IL-17a production, an effective means to determine the cellular activity of RORγt inverse agonists is to measure the reduction in IL-17a mRNA expression levels by quantitative reverse transcriptase PCR (RT-PCR). To this end, EL4 cells were treated with 10 μM of <b>3, 25</b>, and <b>23</b> for 24 h before IL-17a mRNA levels were measured (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The most potent isoxazole in vitro, <b>25</b>, significantly reduced IL-17a mRNA expression 27-fold, while the weaker inverse agonist <b>23</b> showed a smaller reduction (3.6-fold) compared to the DMSO control. As expected, <b>3</b> led to the most significant decrease in IL-17a expression (48-fold) which was in line with previous reports. This result demonstrates that the allosteric modulation of RORγt by optimized trisubstituted isoxazoles leads to an effective cellular response, correlating with the biochemical protein binding data and which is known to be beneficial for the treatment of autoimmune disease.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a></div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. IL-17a mRNA expression in EL4 cells treated with ligand <b>3</b>, <b>25</b>, and <b>23</b> (10 μM, 24h) or DMSO. The level of IL-17a expression was normalized to that of the GAPDH expression. All data are expressed as the mean ± s.d. (<i>n</i> = 3). The relative gene expression was calculated by the 2<sup>–ΔΔCt</sup> (Livak) method using the DMSO control as calibrator.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">2.8.  Absorption, Distribution, Metabolism, and Excretion (ADME) Profile</h3><div class="NLM_p">To further assess the potential of <b>25</b> and isoxazole analogues such as <b>23</b> and <b>8</b>, we investigated the ADME profile of these compounds and compared them to indazole <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The isoxazole compounds showed favorable profiles compared to <b>3</b> in terms of chemical stability, solubility, and permeability through artificial plasma membranes (PAMPA). A metabolic stability study with human liver microsomes indicated that the 4-methylamino isoxazoles <b>23</b> and <b>25</b> were more liable to phase 1 metabolism compared to indazole <b>3</b>, which showed good stability. <b>23</b> and <b>25</b> showed promising phase 2 stability. In blood plasma, while inferior to <b>3</b>, the stability of <b>23</b> and <b>25</b> was acceptable, although all these compounds showed high levels of binding to plasma proteins. Pleasingly, the 5-phenyl-4-amido isoxazole <b>8</b> showed a good ADME profile, with comparable microsomal stability to <b>3</b> and reduced plasma protein binding. This likely indicates that further optimization of the C-4 and C-5 isoxazole substituents has the potential to produce candidate molecules with desirable in vivo efficacy.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. ADME Properties for Ligands <b>3</b>, <b>8</b>, <b>23</b>, and <b>25</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">microsomal stability</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">chemical stability (% remain)</th><th class="colsep0 rowsep0" align="center">solubility (μM)</th><th class="colsep0 rowsep0" align="center" char=".">PAMPA (% flux)</th><th class="colsep0 rowsep0" align="center" char=".">phase 1 (CL<sub>int</sub>, μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">phase 2 (% remain)</th><th class="colsep0 rowsep0" align="center" char=".">plasma stability (% remain)</th><th class="colsep0 rowsep0" align="center" char=".">plasma protein binding (% bound)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">81.0</td><td class="colsep0 rowsep0" align="left">390</td><td class="colsep0 rowsep0" align="char" char=".">23.7</td><td class="colsep0 rowsep0" align="char" char=".">–1.2</td><td class="colsep0 rowsep0" align="char" char=".">47.1</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">99.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">95.4</td><td class="colsep0 rowsep0" align="left">490</td><td class="colsep0 rowsep0" align="char" char=".">60.1</td><td class="colsep0 rowsep0" align="char" char=".">–0.1</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">99.9</td><td class="colsep0 rowsep0" align="char" char=".">97.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="left">392</td><td class="colsep0 rowsep0" align="char" char=".">50.4</td><td class="colsep0 rowsep0" align="char" char=".">43.2</td><td class="colsep0 rowsep0" align="char" char=".">92.8</td><td class="colsep0 rowsep0" align="char" char=".">86.5</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">95.3</td><td class="colsep0 rowsep0" align="left">411</td><td class="colsep0 rowsep0" align="char" char=".">33.6</td><td class="colsep0 rowsep0" align="char" char=".">20.7</td><td class="colsep0 rowsep0" align="char" char=".">69.8</td><td class="colsep0 rowsep0" align="char" char=".">85.9</td><td class="colsep0 rowsep0" align="char" char=".">99.9</td></tr></tbody></table></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To summarize, we report the design, synthesis, and early optimization of a novel class of RORγt allosteric inverse agonists. The chemotype of the central aromatic ring system differs significantly from all the other fused bicyclic ring systems reported thus far. To identify this novel, more diverse, molecular scaffold, we used the crystal structure of <b>3</b> bound to the RORγt allosteric site as the basis for a 3D pharmacophore screen against a virtual compound library. Rational design steps led to the discovery of the in silico designed hit <b>8</b>, which already featured a modest inhibition of transcriptional coactivator recruitment to the RORγt LBD and served as a starting point for further optimization in a SAR campaign. A second and highly efficient iteration of lead optimization was guided by in silico docking. Through the synthesis of just five derivatives (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), this process delivered <b>25</b> (<b>FM26</b>), a submicromolar allosteric inverse agonist. It is highly noteworthy that there was a strong correlation between the Glide dockings scores and the RORγt biochemical activity within this new class of isoxazole. Whereas screening approaches do not overtly identify allosteric ligands, our rational scaffold hopping approach is much more targeted, with less demand on experimental resource. Overall, the discovery workflow adopted, with a central role for structure-driven in silico screening and optimization, showed to be highly effective and might have wider application in expediting NR allosteric drug discovery.</div><div class="NLM_p last">Competitive coactivator recruitment and ligand binding assays were used to confirm the allosteric mode-of-action, with concomitant cooperative RORγt binding with an inverse agonist. This was also shown for thienopyrrazole <b>5</b>, having not previously been disclosed. The cocrystal structure of <b>25</b> in complex with the RORγt LBD unequivocally proved the allosteric binding mode, via a similar mechanism to <b>3</b> and was impressively similar to the initially docked structure of <b>25</b> in RORγt. The cocrystal of <b>25</b> with the RORγt LBD revealed a number of unique interactions and structural RORγt modifications that bring forward intriguing insights and new lines of exploration regarding RORγt allosteric ligand binding, selectivity, and affinity optimization, which are currently explored. Compound <b>25</b> was shown to significantly reduce IL-17a mRNA expression in EL4 cells and to have a promising ADME profile. These factors highlight the potential of this new isoxazole-based ligand class and overt targeting of the RORγt allosteric site to deliver effective treatments for autoimmune diseases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">4.1.  Pharmacophore Screening</h3><div class="NLM_p last">The receptor–ligand complex structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>) was prepared using the Protein Preparation Wizard within Maestro (version 2017–2, Schrödinger LLC, New York, NY, USA) (default parameters). A 3D pharmacophore model for <b>3</b> bound to the allosteric pocket of RORγt LBD was created using Phase (version 2017-2, Schrödinger LLC, default hypothesis settings). Energy minimized 3D ligand conformations for each molecule to be investigated were generated using the Ligand Preparation wizard within Maestro (default parameters). These were screened against the hypothesis whereby up to 50 ligand conformations were generated for each molecule. A hit was returned for compounds that matched 4 out of 6 pharmacophore features, and these were ranked using the Phase Screen Score. The structure and ranking for the top 30 hits identified from the Asinex Gold Platinum library can be found in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">4.2.  Molecular Docking Studies</h3><div class="NLM_p last">The receptor–ligand structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>) was prepared as described above. A receptor grid centered on the bound ligand was created using Glide (version 2017-2, Schrödinger LLC). All parameters were kept as the default. Ligand libraries were either enumerated in SMILES format using the open-access Chem-T software or generated manually (see the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). Ligands were prepared using the Ligand Preparation wizard as described above. Ligands were docked using Glide (version 2017-2, Schrödinger LLC) in standard precision mode with flexible ligand sampling. The predicted binding modes of all ligands were ranked according to their Glide Score (see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a> for selected examples).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25">4.3.  General Chemistry</h3><div class="NLM_p">All nonaqueous reactions were performed under an argon atmosphere unless otherwise stated. Water-sensitive reactions were performed in oven-dried glassware, cooled under argon before use. Solvents were removed in vacuo using a Büchi rotary evaporator and a diaphragm pump. THF and CH<sub>2</sub>Cl<sub>2</sub> were dried and purified by means of a MBRAUN Solvent Purification System (MB-SPS-800). Anhydrous DMF was obtained in SureSeal bottles from Sigma-Aldrich. All other solvents used were of chromatography or analytical grade and supplied by Biosolve or Sigma-Aldrich. Commercially available starting materials were obtained from Sigma-Aldrich, Fluka, Acros, Alfa-Aesar, or Fluorochem and were used without further purification unless stated. TLC was carried out on aluminum-backed silica (Merck silica gel 60 F254) plates supplied by Merck. Visualization of the plates was achieved using an ultraviolet lamp (λ<sub>max</sub> = 254 nm), KMnO<sub>4</sub>, anisaldehyde, or ninhydrin. Column chromatography was either performed manually using silica gel (60–63 um particle size) or using an automated Grace Reveleris X2 chromatograph with prepacked silica columns supplied by Buchi/Grace (40 μm particle size). LC–MS analysis was carried out with a system comprising a Thermo Fischer LCQ Fleet Ion Trap Mass Spectrometer and C18 Jupiter SuC4300A 150 × 2.0 mm column using a gradient of 5–100% MeCN in water (+ 0.1% HCOOH) over 15 min. The purity of the samples was assessed using a UV detector at 254 nm. Unless otherwise stated all final compounds were >95% pure as judged by HPLC. GCMS analysis was performed on a Phenomenex Zebron ZB-5MS 30 m × 0.25 mm × 0.25 mm column with a gradient of 80 °C for 1 min to 300 °C for 1 min with a rate of 30 °C/min in helium gas connected to a GCMS-QP2010 Plus Quadrupole Mass Spectrometer. High resolution mass spectra (HRMS) were recorded using a Waters ACQUITY UPLC I-Class LC system coupled to a Xevo G2 Quadrupole Time of Flight (Q-tof) mass spectrometer. Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectral data were collected on a 400 MHz Bruker Cryomagnet or 400 MHz Varian Gemini. Chemical shifts (δ) are quoted in parts per million (ppm) and referenced to the residual solvent peak. Coupling constants (<i>J</i>) are quoted in Hertz (Hz) and splitting patterns reported in an abbreviated manner: app. (apparent), s (singlet), d (doublet), t (triplet), q (quartet), and m (multiplet). Assignments were made with the aid of 2D COSY, HMQC, and HMBC experiments.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">4.3.1.  General Procedure for Ester Hydrolysis</h4><div class="NLM_p last">LiOH·H<sub>2</sub>O (5.0 equiv) was added to a suspension of ester (1.0 equiv) in a 4:1 mixture of MeOH-H<sub>2</sub>O (0.2 M). The reaction mixture was heated to 70 °C until TLC analysis indicated complete consumption of the starting material. MeOH was removed in vacuo, and the resulting aqueous mixture was acidified to pH 3 using 10% v/v aqueous HCl and extracted with a 9:1 mixture of CH<sub>2</sub>Cl<sub>2</sub>-MeOH (× 5). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish an ester which was purified as described.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">4.3.2.  General Procedure for Amide Coupling</h4><div class="NLM_p last">Carboxylic acids (1.0 equiv) were dissolved in SOCl<sub>2</sub> (50 equiv) and heated to 50 °C for 2 h. The excess SOCl<sub>2</sub> was removed in vacuo to furnish an acid chloride intermediate that was immediately dissolved in CH<sub>2</sub>Cl<sub>2</sub> (0.1 M). To this was added NEt<sub>3</sub> (3.0 equiv), the appropriate amine or aniline (1.5 equiv), and DMAP (0.1 equiv), and the reaction mixture was stirred at reflux for 18 h. The reaction mixture was diluted with saturated aqueous NH<sub>4</sub>Cl solution and extracted with ethyl acetate (3×). The combined organic phase was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography (SiO<sub>2</sub>) using the specified eluent.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">4.3.3.  General Procedure for <i>tert</i>-Butyl Ester Deprotection</h4><div class="NLM_p last">Esters (1.0 equiv) were treated with a 20% trifluoroacetic acid solution in CH<sub>2</sub>Cl<sub>2</sub> (0.2 M). The reaction mixture was stirred at the specified temperature for the specified amount of time and then concentrated in vacuo. The crude product was purified as indicated.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">4.3.4.  General Procedure for Suzuki Coupling</h4><div class="NLM_p last">Under an inert atmosphere, the pinacol boronate (2.0 equiv), Cs<sub>2</sub>CO<sub>3</sub> (2.0 equiv), and Pd(dppf)Cl<sub>2</sub> (0.1 equiv) were added to a solution of bromide <b>30a</b> (1.0 equiv) in degassed DME. The reaction mixture was heated at 85 °C for 8 h, cooled to room temperature, diluted with saturated aqueous NH<sub>4</sub>Cl, and extracted with EtOAc (3×). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography using the specified eluent.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">4.3.5.  General Procedure for Conversion of Esters to Aldehydes</h4><div class="NLM_p last">LiAlH<sub>4</sub> (1 M in THF, 1.0 equiv) was added dropwise to a solution of ester (1.0 equiv) in THF (0.2 M) at 0 °C. The reaction mixture was warmed to room temperature and stirred until TLC analysis indicated complete consumption of the starting material. The reaction mixture was cooled to 0 °C, quenched by the addition of saturated aqueous NH<sub>4</sub>Cl solution, and extracted with EtOAc (3×). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The resulting intermediate product was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (0.2 M). To this was added Dess-Martin Periodinane (1.5 equiv), and the reaction mixture was stirred at room temperature until TLC analysis indicated complete consumption of the intermediate. The reaction mixture was quenched by the addition of 10% aqueous Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> solution and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic phase was washed with saturated aqueous NaHCO<sub>3</sub> and H<sub>2</sub>O, dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish the title compound which was purified as described.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">4.3.6.  General Procedure for Reductive Amination</h4><div class="NLM_p last">The chosen amine or aniline (1.0 equiv) was added to a solution of the appropriate aldehyde (1.0 equiv) and AcOH (0.1 equiv) in MeOH or EtOH (0.25 M). The reaction mixture was heated at reflux for 24 h and then concentrated in vacuo. The intermediate imine was isolated by flash column chromatography using the specified eluent and then dissolved in MeOH or EtOH (0.2 M), cooled to 0 °C (ice), and treated with NaBH<sub>4</sub> (5.0 equiv). The reaction mixture was held at the specified temperature until TLC analysis indicated complete consumption of the imine. The solvent was removed in vacuo, and the mixture was dissolved in CH<sub>2</sub>Cl<sub>2</sub> and washed with water. The aqueous phase was further extracted with CH<sub>2</sub>Cl<sub>2</sub> (2×), dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography (SiO<sub>2</sub>) using the specified eluent.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">4.3.7.  4-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-methyl-1,2-oxazole-4-amido}benzoic Acid (<b>7</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11a</b> (60.0 mg, 0.199 mmol) was coupled with <i>tert</i>-butyl-4-amino benzoate. The crude product was purified by flash column chromatography, eluting with 20% EtOAc in cyclo-hexane, to furnish the amide (46.0 mg, 48%). The intermediate product (43.0 mg, 0.089 mmol) was subject to <i>tert</i>-butyl ester deprotection (see <a class="ref internalNav" href="#sec4_3_3" aria-label="General Procedure for tert-Butyl Ester Deprotection">General Procedure for tert-Butyl Ester Deprotection</a>) and purified by trituation with Et<sub>2</sub>O to furnish <b>7</b> (33.0 mg, 87%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.52 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, MeOD): δ (ppm) 7.97 (2 H, d, <i>J</i> = 8.8, benzoate H-2), 7.81 (2 H, app. d, <i>J</i> = 8.0, Ar H-3 and Ar H-5), 7.67 (1 H, t, <i>J</i> = 8.0, Ar H-4), 7.61 (2 H, d, <i>J</i> = 8.8, benzoate H-3), 2.76 (3 H, s, C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, MeOD): δ (ppm) 171.3 (C-5), 161.2 (<i>C</i>O<sub>2</sub>H), 159.5 (C-3), 143.8 (benzoate C-4), 137.4 (ArC-2), 134.4 (ArC-3), 132.7 (q, <i>J</i> = 30.9, ArC-6), 132.6 (ArC-4), 131.8 (benzoate C-2), 127.6 (ArC-1), 127.5 (benzoate C-1), 126.1 (q, <i>J</i> = 5.1, ArC-5), 120.6 (benzoate C-3), 116.1 (C-4), 12.8 (<i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>19</sub>H<sub>13</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 425.04, observed: 425.17, LC <i>R</i><sub>t</sub>: 6.22 min. HRMS (ESI): calc. for C<sub>19</sub>H<sub>13</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 425.0516, observed: 425.0511.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33">4.3.8.  4-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-amido}benzoic Acid (<b>8</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl benzoate <b>14</b> (30.0 mg, 0.055 mmol) was deprotected according to the General Procedure for <i>tert</i>-butyl ester deprotection. The crude product was purified by trituation with Et<sub>2</sub>O to furnish <b>8</b> (21.0 mg, 78%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.55 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 7.97 (1 H, d, <i>J</i> = 8.2, ArH-3 or ArH-5), 7.92 (1 H, d, <i>J</i> = 7.9, ArH-3 or ArH-5), 7.85–7.83 (4 H, m, benzoate H-2, PhH-ortho), 7.78 (1 H, app. t, <i>J</i> = 8.1, ArH-4), 7.58–7.54 (5 H, m, benzoate H-3, PhH-meta, PhH-para); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 167.0 (C-5), 166.8 (<i>C</i>O<sub>2</sub>H), 158.9 (<i>C</i>O<sub>2</sub>NH), 158.7 (C-3), 142.3 (benzoate C-4), 135.5 (ArC-2), 133.7 (ArC-3), 132.4 (PhC-quart.), 131.6 (ArC-4), 130.6 (ArC-6), 130.3 (benzoate C-2), 129.4 (PhC-ortho), 127.3 (PhC-meta), 126.0 (PhC-para), 125.9 (benzoate C-1), 125.3 (ArC-5 and ArC-1), 119.3 (benzoate C-3), 113.4 (C-4), quartet for <i>C</i>F<sub>3</sub> not observed; LC–MS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.06, observed: 487.17, LC <i>R</i><sub>t</sub>: 7.00 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.0672, observed: 487.0662.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">4.3.9.  (E)-N-{[2-Chloro-6-(trifluoromethyl)phenyl]methylidene}hydroxylamine (<b>9</b>)</h4><div class="NLM_p last">Hydroxylamine hydrochloride (3.95 g, 57.0 mmol, 1.2 equiv) was suspended in EtOH (20 mL) and 10% w/v aqueous solution of NaOH (20 mL) was added such that the final pH of the resulting solution was < pH 9. 2-Chloro-6-(trifluoromethyl)benzaldehyde (9.88 g, 47.4 mmol, 1.0 equiv) was then added as a solution in EtOH (20 mL) and the mixture was stirred at room temperature for 18 h. The reaction mixture was diluted with H<sub>2</sub>O (100 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 100 mL). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish <b>9</b> (8.77 g, 83%) as a colorless solid which was used without further purification. <i>R</i><sub><i>f</i></sub> = 0.45 (4:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.97 (1H, s, N═C<i>H</i>), 8.36 (1H, s, NO<i>H</i>), 7.67–7.64 (2 H, m, H-3, H-5), 7.45 (1 H, app. t, <i>J</i> = 8.0, H-4); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 145.3 (N═<i>C</i>H), 135.7 (C-2), 133.6 (C-3), 131.3 (q, <i>J</i> = 31.2, C-6), 130.2 (C-4), 129.0 (C-1), 125.1 (q, <i>J</i> = 5.5, C-5), 123.22 (q, <i>J</i> = 274.2, F<sub>3</sub><i>C</i>); LC–MS (ESI): calcd for C<sub>8</sub>H<sub>6</sub>ClF<sub>3</sub>NO [M + H]<sup>+</sup>: 224.00, observed: 224.00, LC <i>R</i><sub>t</sub>: 5.82 min.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">4.3.10.  (Z)-2-Chloro-<i>N</i>-hydroxy-6-(trifluoromethyl)benzene-1-carbonimidoyl Chloride (<b>9a</b>)</h4><div class="NLM_p last"><i>N</i>-Chlorosuccinamide (5.22 g, 39.1 mmol, 1.0 equiv) was added to a solution of hydroxylamine <b>9</b> (8.74 g, 39.1 mmol, 1.0 equiv) in DMF (80 mL). The reaction mixture was stirred at 60 °C for 18 h then diluted with H<sub>2</sub>O (150 mL) and extracted with Et<sub>2</sub>O (3 × 100 mL). The combined organic phase was washed with H<sub>2</sub>O (3 × 100 mL) and brine (100 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish <b>9a</b> (9.10 g, 95% purity, 86%) which was used immediately in the next step without further purification. <i>R</i><sub><i>f</i></sub> = 0.42 (4:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.50 (1H, s, NO<i>H</i>), 7.68–7.66 (2 H, m, H-3, H-5), 7.53 (1 H, app. t, <i>J</i> = 8.1, H-4).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">4.3.11.  Ethyl 3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-methyl-1,2-oxazole-4-carboxylate (<b>10a</b>)</h4><div class="NLM_p last">NEt<sub>3</sub> (0.570 mL, 4.07 mmol, 2.5 equiv) was added to a solution of imidoyl chloride <b>9a</b> (0.421 g, 1.63 mmol, 1.0 equiv) in THF (2.5 mL). A white precipitate formed immediately after which was added ethyl 2-butynoate (0.190 mL, 1.63 mmol, 1.0 equiv). The reaction mixture was heated to reflux for 4 h, filtered, and concentrated in vacuo. The crude material was purified by automated flash column chromatography, eluting with a gradient of 2–5% EtOAc in c-hexane, to furnish <b>10a</b> (0.375 g, 69%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.22 (9:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.69 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.66 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.53 (1 H, app. t, <i>J</i> = 8.0, ArH-4); 4.07 (2 H, 2× dq (overlapping), <i>J</i> = 12.7, 7.1, CO<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>3</sub>), 2.78 (3 H, s, C<i>H</i><sub>3</sub>), 0.98 (3 H, t, <i>J</i> = 7.1, OCH<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 175.4 (C-5), 161.1 (<i>C</i>O<sub>2</sub>Et), 158.1 (C-3), 136.3 (ArC-2), 132.7 (ArC-3), 131.6 (q, <i>J</i> = 31.6, ArC-6), 130.6 (ArC-4), 127.5 (ArC-1), 124.6 (q, <i>J</i> = 5.0, ArC-5), 123.0 (q, <i>J</i> = 274.5, F<sub>3</sub><i>C</i>), 110.2 (C-4), 60.6 (O<i>C</i>H<sub>2</sub>CH<sub>3</sub>), 13.66 (<i>C</i>H<sub>3</sub>), 13.3 (OCH<sub>2</sub><i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>14</sub>H<sub>12</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 334.04, observed: 334.08, LC <i>R</i><sub>t</sub>: 7.47 min.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">4.3.12.  Ethyl 3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-carboxylate (<b>10b</b>)</h4><div class="NLM_p last">NEt<sub>3</sub> (3.92 mL, 28.1 mmol, 2.5 equiv) was added to a solution of imidoyl chloride <b>9a</b> (2.90 g, 11.3 mmol, 1.0 equiv) in THF (17 mL). A white precipitate formed immediately after which was added ethyl-3-phenyl propionate (1.85 mL, 11.3 mmol, 1.0 equiv). The reaction mixture was heated to reflux for 6 h, filtered, and concentrated in vacuo. The crude material was purified by automated flash column chromatography, eluting with a gradient of 0–25% EtOAc in <i>n</i>-heptane, to furnish <b>10b</b> (3.59 g, 80%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.47 (4:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.16–8.13 (2 H, m, PhH-ortho), 7.74 (1 H, d, <i>J</i> = 7.9, ArH-3 or ArH-5), 7.71 (1 H, d, <i>J</i> = 8.2, H-3 or H-5), 7.59–7.51 (4 H, m, H-4, Ar–H), 4.06 (2 H, dq (2 overlapping), <i>J</i> = 11.8, 7.1, CO<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>3</sub>), 0.88 (3 H, t, <i>J</i> = 7.1, OCH<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 173.0 (C-5), 160.8 (<i>C</i>O<sub>2</sub>Et), 159.2 (C-3), 136.3 (ArC-2), 132.6 (ArC-3), 131.7 (PhC-quart.), 131.5 (q, <i>J</i> = 30.5, ArC-6), 130.5 (ArC-4), 129.2 (PhC-ortho), 128.5 (PhC-meta), 127.8 (q, <i>J</i> = 1.8, ArC-1), 126.3 (PhC-meta), 124.5 (q, <i>J</i> = 5.0, ArC-5), 122.9 (q, <i>J</i> = 274.3, F<sub>3</sub><i>C</i>), 109.3 (C-4), 60.8 (O<i>C</i>H<sub>2</sub>CH<sub>3</sub>), 13.3 (OCH<sub>2</sub><i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>19</sub>H<sub>14</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 396.06, observed: 396.17, LC <i>R</i><sub>t</sub>: 8.40 min.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38">4.3.13.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-methyl-1,2-oxazole-4-carboxylic acid (<b>11a</b>)</h4><div class="NLM_p last">According to the General Procedure for ester hydrolysis, ester <b>10a</b> (0.273 g, 0.820 mmol) was hydrolyzed in 8 h to furnish <b>11a</b> (0.210 g, 84%) as a colorless solid which required no further purification. <i>R</i><sub><i>f</i></sub> = 0.61 (EtOAc); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.09 (1 H, br. s, CO<sub>2</sub><i>H</i>), 7.96 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.90 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.77 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 2.76 (3 H, s, C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 175.3 (C-5), 161.9 (<i>C</i>O<sub>2</sub>H), 158.2 (C-3), 135.3 (ArC-2), 133.4 (ArC-3), 131.8 (ArC-4), 130.1 (q, <i>J</i> = 30.8, ArC-6), 126.8 (ArC-1), 125.1 (q, <i>J</i> = 5.0, ArC-5), 123.0 (q, <i>J</i> = 274.4, F<sub>3</sub><i>C</i>), 110.5 (C-4), 12.8 (<i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>12</sub>H<sub>6</sub>ClF<sub>3</sub>NO<sub>3</sub> [M-H]<sup>−</sup>: 304.01, observed: 304.17, LC <i>R</i><sub>t</sub>: 5.82 min.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">4.3.14.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-carboxylic acid (<b>11b</b>)</h4><div class="NLM_p last">According to the General Procedure for ester hydrolysis, ester <b>10b</b> (3.66 g, 9.26 mmol) was hydrolyzed in 8 h to furnish <b>11b</b> (3.25 g, 95%) as a colorless solid which required no further purification. <i>R</i><sub><i>f</i></sub> = 0.35 (1:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.05–8.03 (2 H, m, PhH-ortho), 7.71(1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.68 (1 H, d, <i>J</i> = 8.1, H-3 or H-5), 7.60–7.50 (4 H, m, H-4, Ar–H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 171.8 (C-5), 161.5 (<i>C</i>O<sub>2</sub>H), 159.3 (C-3), 135.2 (ArC-2), 133.4 (ArC-3), 132.0 (PhC-quart.), 131.8 (ArC-4), 130.0 (q, <i>J</i> = 30.5, ArC-6), 129.0 (PhC-ortho), 128.7 (PhC-meta), 126.7 (ArC-1), 125.7 (PhC-meta), 125.2 (q, <i>J</i> = 5.0, ArC-5), 123.0 (q, <i>J</i> = 274.4, F<sub>3</sub><i>C</i>), 99.4 (C-4); LC–MS (ESI): calcd for C<sub>17</sub>H<sub>10</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 368.02, observed: 368.08, LC retention time: 7.03 min. HRMS (ESI): calcd for C<sub>17</sub>H<sub>10</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 368.0301, observed: 368.0299.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40">4.3.15.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-carbaldehyde (<b>12</b>)</h4><div class="NLM_p last">Carboxylic acid <b>11b</b> (2.0 g, 5.44 mmol,1.0 equiv) was dissolved in SOCl<sub>2</sub> (10.0 mL, 138 mmol, 25 equiv) and heated to 60 °C for 2 h. Excess SOCl<sub>2</sub> was removed in vacuo, and the intermediate acid chloride was immediately dissolved in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and cooled to 0 °C. To this was added NEt<sub>3</sub> (2.27 mL, 16.3 mmol, 3.0 equiv) and <i>N</i>,<i>O</i>-dimethyl hydroxylamine hydrochloride (0.580 g, 5.98 mmol, 1.1 equiv). The reaction mixture was allowed to warm to room temperature with stirring over 16 h before being quenched by the addition of saturated aqueous NH<sub>4</sub>Cl solution and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The resulting Weinreb amide was dissolved in THF (20 mL) and cooled to 0 °C. To this was added LiAlH<sub>4</sub> (1 M in THF, 2.72 mL, 2.72 mmol, 0.5 equiv), and the reaction mixture was stirred at 0 °C for 1 h before being quenched by the addition of saturated aqueous NH<sub>4</sub>Cl solution (20 mL) and extracted with EtOAc (3 × 20 mL). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish a crude product which was purified by automated flash column chromatography, eluting with a gradient of 10–30% EtOAc in <i>n</i>-heptane, to furnish <b>12</b> (1.25 g, 65%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.65 (1:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 9.93 (1 H, s, C<i>H</i>O), 8.06–8.04 (2 H, m, PhH-ortho), 7.77 (1 H, d, <i>J</i> = 7.9, ArH-3 or ArH-5), 7.74 (1 H, d, <i>J</i> = 7.8, ArH-3 or ArH-5), 7.68–7.59 (4 H, m, H-4, PhH); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 182.8 (<i>C</i>HO), 174.6 (C-5), 160.8 (<i>C</i>O<sub>2</sub>Et), 158.6 (C-3), 136.3 (ArC-2), 133.2 (ArC-3), 132.7 (PhC-quart.), 132.0 (q, <i>J</i> = 30.9, ArC-6), 131.3 (ArC-4), 129.5 (PhC-ortho), 128.9 (PhC-meta), 125.9 (ArC-1), 125.7 (PhC-para), 125.0 (ArC-5), 123.0 (q, <i>J</i> = 274.9, F<sub>3</sub><i>C</i>), 116.2 (C-4); LC–MS (ESI): calcd for C<sub>17</sub>H<sub>10</sub>ClF<sub>3</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: 352.03, observed: 352.08, LC <i>R</i><sub>t</sub>: 7.28 min.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41">4.3.16.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-amine (<b>13</b>)</h4><div class="NLM_p last">NEt<sub>3</sub> (0.760 mL, 5.44 mmol, 1.1 equiv) and diphenylphosphoryl azide (1.06 mL, 4.95 mmol, 1.0 equiv) were added to a prewarmed solution of acid <b>11b</b> (1.82 g, 4.95 mmol, 1.0 equiv) in <i>t</i>-BuOH (18 mL) at 50 °C. The reaction mixture was then heated to 85 °C for 18 h after which it was cooled to room temperature and diluted with 1 M aqueous HCl (50 mL) and extracted with EtOAc (3 × 30 mL). The combined organic phase was washed with saturated aqueous NaHCO<sub>3</sub> (100 mL) and brine (100 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by automated flash column chromatography, eluting with a gradient of 0–20% EtOAc in <i>n</i>-heptane, to furnish a carbamate (1.41 g, 65%) as a colorless solid. Trifluoroacetic acid (3.0 mL) was added to a solution of the carbamate (1.13 g, 2.58 mmol, 1.0 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (9.0 mL). The reaction mixture was heated at reflux for 4 h then cooled to room temperature and concentrated in vacuo. The crude product was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) and washed with saturated aqueous NaHCO<sub>3</sub> (100 mL) and water (100 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish <b>13</b> (0.798 g, 91%) as a pale yellow solid that was used without further purification. <i>R</i><sub><i>f</i></sub> = 0.56 (1:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.86 (1 H, d, <i>J</i> = 7.0, ArH-3 or ArH-5), 7.79–7.76 (2 H, m, PhH-ortho), 7.61 (1 H, app. t, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.54–7.49 (3 H, m, PhH), 7.41 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 2.97 (2 H, s, N<i>H</i><sub>2</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 164.5 (C-5), 154.7 (C-3), 137.0 (ArC-2), 133.4 (ArC-3), 131.4 (PhC-quart.), 129.1 (PhC-ortho), 128.9 (9ArC-4), 128.3 (PhC-para), 126.5 (ArC-5), 126.0 (q, <i>J</i> = 4.0, ArC-1), 125.2 (PhC-meta), 122.9 (q, <i>J</i> = 274.4, <i>C</i>F<sub>3</sub>), 110.4 (C-4); LC–MS (ESI): calcd for C<sub>16</sub>H<sub>11</sub>ClF<sub>3</sub>N<sub>2</sub>O [M + H]<sup>+</sup>: 339.04, observed: 339.08, LC <i>R</i><sub>t</sub>: 7.12 min.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42">4.3.17.  <i>tert</i>-Butyl 4-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-amido}benzoate (<b>14</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11b</b> (0.200 g, 0.540 mmol) was coupled with <i>tert</i>-butyl-4-amino benzoate. The crude product was purified by flash column chromatography, eluting with 15% EtOAc in c-hexane, to furnish amide <b>14</b> (0.260 g, 88%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.55 (3:2 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.96 (2 H, dd, <i>J</i> = 7.9, 1.7, PhH-ortho), 7.86 (2 H, d, <i>J</i> = 8.7, benzoate H-2), 7.77 (2 H, app. t, <i>J</i> = 8.1, ArH-3 and ArH-5), 7.63–7.56 (4 H, m, PhH-meta, PhH-para, ArH-4), 7.37 (1 H, br. s, C(O)N<i>H</i>), 7.29 (2 H, d, <i>J</i> = 8.7, benzoate H-3), 1.55 (9 H, s, CO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 169.4 (C-5), 165.1 (<i>C</i>O<sub>2</sub>H), 158.4 (C-3 and <i>C</i>O<sub>2</sub>NH), 140.8 (benzoate C-4), 136.6 (ArC-2), 133.3 (ArC-3), 132.1 (PhC-quart.), 131.5 (ArC-4), 130.7 (ArC-6), 129.5 (PhC-ortho and benzoate C-2), 128.7 (PhC-meta), 128.2 (benzoate C-1), 126.2 (ArC-1), 126.0 (PhC-para), 125.1 (ArC-5), 118.7 (benzoate C-3), 113.0 (C-4), 81.1 (CO<sub>2</sub><i>C</i>(CH<sub>3</sub>)<sub>3</sub>), 28.3 (CO<sub>2</sub>C(<i>C</i>H<sub>3</sub>)<sub>3</sub>), quartet for <i>C</i>F<sub>3</sub> not observed; LC–MS (ESI): calcd for C<sub>28</sub>H<sub>23</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 543.12, observed: 543.08, LC <i>R</i><sub>t</sub>: 8.88 min. HRMS (ESI): calcd for C<sub>28</sub>H<sub>23</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 543.1298, observed: 543.1292.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43">4.3.18.  2-(4-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-amido}phenyl) acetic acid (<b>15</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11b</b> (0.200 g, 0.540 mmol) was coupled with methyl-(4-aminophenyl) acetate. The crude product was purified by flash column chromatography, eluting with a gradient of 15–45% EtOAc in <i>n</i>-heptane, to furnish the amide (0.173 g, 62%). The intermediate product (0.117 g, 0.230 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>15</b> (0.099 g, 87%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.55 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.32 (1 H, br. s, CO<sub>2</sub><i>H</i>), 10.49 (1 H, s, CON<i>H</i>), 7.98 (1 H, d, <i>J</i> = 8.2, ArH-3 or ArH-5), 7.93 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.88–7.87 (2 H, m, PhH-ortho), 7.79 (1 H, app. t, <i>J</i> = 8.1, ArH-4), 7.61–7.59 (3 H, m, PhH-meta, PhH-para), 7.40 (2 H, d, <i>J</i> = 8.2, benzoate H-2), 7.15 (2 H, d, <i>J</i> = 8.2, benzoate H-3), 3.49 (2 H, br. s, benzylic C<i>H</i><sub>2</sub>); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 172.7 (<i>C</i>O<sub>2</sub>H), 166.6 (C-5), 158.7 (<i>C</i>O<sub>2</sub>NH), 158.4 (C-3), 136.9 (benzoate C-4), 135.5 (ArC-2), 133.6 (ArC-3), 132.3 (PhC-quart.), 131.4 (ArC-4), 130.9 (benzoate C-1), 130.5 (q, <i>J</i> = 30.9, ArC-6), 129.7 (benzoate C-2), 129.4 (PhC-ortho), 127.2 (PhC-meta), 126.0 (PhC-para), 125.5 (ArC-5), 125.3 (ArC-1), 122.9 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 119.9 (benzoate C-3), 113.8 (C-4), 30.7 (benzylic <i>C</i>H<sub>2</sub>); LC–MS (ESI): calcd for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 501.08, observed: 501.17, LC <i>R</i><sub>t</sub>: 6.67 min. HRMS (ESI): calcd for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 501.0829, observed: 501.0829.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44">4.3.19.  3-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-amido}benzoic Acid (<b>16</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11b</b> (0.200 g, 0.540 mmol) was coupled with methyl-3-amino benzoate. The crude product was purified by flash column chromatography, eluting with 15% EtOAc in c-hexane, to furnish the amide (0.212 g, 78%). The intermediate product (0.100 g, 0.200 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>16</b> (0.070 g, 72%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.51 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.01 (1 H, br. s, CO<sub>2</sub><i>H</i>), 10.66 (1 H, s, CON<i>H</i>), 8.11 (1 H, s, benzoate H-2), 7.98 (1 H, d, <i>J</i> = 8.1, ArH-3 or ArH-5), 7.91 (1 H, d, <i>J</i> = 7.7, ArH-3 or ArH-5), 7.87 (2 H, m, PhH-ortho), 7.80 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.68–7.64 (2 H, m, benzoate H-4 and H-6), 7.63–7.59 (3 H, m, PhH-meta, PhH-para), 7.40 (1 H, app. t, <i>J</i> = 7.8, benzoate H-5); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 166.9 (C-5), 158.7 (<i>C</i>O<sub>2</sub>H), 158.6 (<i>C</i>O<sub>2</sub>NH), 154.5 (C-3), 138.5 (benzoate C-3), 135.4 (ArC-2), 133.6 (ArC-3), 132.4 (PhC-quart.), 131.5 (ArC-4), 129.4 (PhC-ortho), 129.0 (benzoate C-5), 127.3 (PhC-meta), 125.9 (PhC-para and benzoate C-1), 125.4 (ArC-5 and ArC-1), 124.9 (benzoate C-6), 124.0 (benzoate C-4), 120.6 (benzoate C-2), 113.5 (C-4), (quartet for <i>C</i>F<sub>3</sub> not observed); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.06, observed: 487.25, LC <i>R</i><sub>t</sub>: 7.10 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.0672, observed: 487.0667.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45">4.3.20.  4-{3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazole-4-amido}-2-fluorobenzoic Acid (<b>17</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11b</b> (0.200 g, 0.540 mmol) was coupled with methyl-4-amino-2-fluoro benzoate. The crude product was purified by flash column chromatography, eluting with a gradient of 20–25% EtOAc in <i>n</i>-heptane, to furnish the amide (0.075 g, 27%). The intermediate product (0.063 g, 0.120 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>17</b> (0.053 g, 87%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.27 (1:1 n-heptate-EtOAc); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.05 (1 H, br. s, CO<sub>2</sub><i>H</i>), 10.94 (1 H, s, CON<i>H</i>), 7.99 (1 H, d, <i>J</i> = 8.2, ArH-3 or ArH-5), 7.94 (1 H, d, <i>J</i> = 7.9, ArH-3 or ArH-5), 7.87–7.78 (4 H, m, PhH-ortho, ArH-4, benzoate H-6), 7.62–7.59 (3 H, m, PhH-meta, PhH-para), 7.51 (1 H, d, <i>J</i> 13.1, benzoate H-3), 7.28 (1 H, d, <i>J</i> 8.7, benzoate H-5); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 167.3 (C-5), 164.5 (<i>C</i>O<sub>2</sub>H), 161.5 (d, <i>J</i> = 256.0, benzoate C-2), 159.1 (<i>C</i>O<sub>2</sub>NH), 158.7 (C-3), 143.6 (d, <i>J</i> = 11.4, benzoate C-4), 135.4 (ArC-2), 133.7 (ArC-3), 132.8 (benzoate C-6), 132.4 (PhC-quart.), 131.7 (ArC-4), 130.4 (q, <i>J</i> = 30.6, ArC-6), 129.4 (PhC-ortho), 127.4 (PhC-meta), 125.7 (PhC-para), 125.4 (ArC-5), 125.1 (ArC-1), 122.9 (q, <i>J</i> = 274.6, <i>C</i>F<sub>3</sub>), 115.0 (benzoate C-5), 114.2 (d, <i>J</i> = 10.1, benzoate C-1), 113.1 (C-4), 107.2 (d, <i>J</i> = 27.5, benzoate C-3); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>14</sub>ClF<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 505.05, observed: 505.25, LC <i>R</i><sub>t</sub>: 7.10 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>14</sub>ClF<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 505.0578, observed: 505.0569.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46">4.3.21.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-yl}formamido)methyl]benzoic Acid (<b>18</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, carboxylic acid <b>11b</b> (0.200 g, 0.540 mmol) was coupled with methyl-4-aminomethyl benzoate. The crude product was purified by flash column chromatography, eluting with 15% EtOAc in cyclo-hexane, to furnish the amide (0.173 g, 62%). The intermediate product (0.106 g, 0.200 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>18</b> (0.096 g, 96%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.51 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, MeOD): δ (ppm) 7.91 (2 H, d, <i>J</i> = 8.3, benzoate H-2), 7.84 (1 H, d, <i>J</i> = 7.7, ArH-3 or ArH-5), 7.83 (1 H, d, <i>J</i> = 8.3, ArH-3 or ArH-5), 7.78–7.76 (2 H, m, PhH-ortho), 7.72 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.57–7.53 (1 H, m, PhH-para), 7.49–7.45 (2 H, m, PhH-meta), 7.24 (2 H, d, <i>J</i> = 8.3, benzoate H-3), 4.42 (1 H, d, <i>J</i> = 15.2, benzylic C<i>H</i>a), 4.40 (1 H, d, <i>J</i> = 15.2, benzylic C<i>H</i>b); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 167.2 (C-5), 166.6 (<i>C</i>O<sub>2</sub>H), 160.0 (<i>C</i>O<sub>2</sub>NH), 158.4 (C-3), 143.6 (benzoate C-4), 135.6 (ArC-2), 133.6 (ArC-3), 132.3 (PhC-quart.), 131.3 (ArC-4), 130.6 (q, <i>J</i> = 31.6, ArC-6), 129.5 (benzoate C-1), 129.2 (benzoate C-2 and PhC-ortho), 127.5 (PhC-meta), 127.4 (PhC-para), 125.9 (ArC-5), 125.4 (ArC-1), 122.9 (q, <i>J</i> = 274.1, <i>C</i>F<sub>3</sub>), 118.8 (benzoate C-3), 113.6 (C-4), 42.4 (benzylic <i>C</i>H<sub>2</sub>); LC–MS (ESI): calcd for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 501.08, observed: 501.25, LC <i>R</i><sub>t</sub>: 6.67 min. HRMS (ESI): calcd for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 501.0829, observed: 501.0818.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47">4.3.22.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-yl}methyl)amino] benzoic Acid (<b>19</b>)</h4><div class="NLM_p last">Ethyl-4-aminobenzoate (86 mg, 0.52 mmol, 1.0 equiv)) and AcOH (0.5 mL) were added to a solution of aldehyde <b>12</b> (0.183 g, 0.520 mmol, 1.0 equiv) in EtOH (10.0 mL). The reaction mixture was heated at reflux for 4 h after which time it was cooled to room temperature and NaCNBH<sub>3</sub> (65.3 mg, 1.04 mmol, 2.0 equiv) was added. The reaction mixture was then heated at reflux for a further 12 h then concentrated in vacuo, diluted with EtOAc (10 mL) and washed with saturated aqueous NaHCO<sub>3</sub> (20 mL), water (20 mL), and brine (20 mL). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography, eluting with 17% EtOAc in <i>n</i>-heptane, to furnish the ester (81.4 mg, 31%). The intermediate product (0.049 g, 0.098 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>19</b> (0.046 g, 99%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = baseline (4:1 cyclo-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.03 (1 H, br. s, CO<sub>2</sub><i>H</i>), 7.92–7.85 (4 H, m, ArH-3, ArH-5, PhH-ortho), 7.73 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.64–7.58 (3 H, m, PhH-ortho and PhH-meta), 7.53 (2 H, d, <i>J</i> = 8.8, benzoate C-2), 6.48 (1 H, t, <i>J</i> = 5.3 CH<sub>2</sub>N<i>H</i>), 6.37 (2 H, d, <i>J</i> = 8.8, benzoate H-3), 4.21 (1 H, dd, <i>J</i> = 14.7, 5.3, C<i>H</i><sub>a</sub>NH), 4.12 (1 H, dd, <i>J</i> = 14.7, 5.3, C<i>H</i><sub>b</sub>NH); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 167.4 (C-5), 166.4 (<i>C</i>O<sub>2</sub>H), 159.6 (C-3), 151.6 (benzoate C-4), 135.5 (ArC-2), 133.7 (ArC-3), 132.2 (PhC-quart.), 130.8 (benzoate C-2), 130.7 (ArC-4), 130.6 (q, <i>J</i> = 30.4, ArC-6), 129.3 (PhC-ortho),127.2 (PhC-meta), 126.6 (PhC-para), 125.6 125.4 (ArC-5), 125.4 (q, <i>J</i> = 5.0, ArC-1), 122.9 (q, <i>J</i> = 274.6, <i>C</i>F<sub>3</sub>), 117.6 (benzoate C-1), 113.1 (C-4), 110.7 (benzoate C-3), 35.7 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 473.08, observed 473.00, LC <i>R</i><sub>t</sub>: 7.35 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 473.0880, observed: 473.0862.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48">4.3.23.  4-({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-yl}carbamoyl) Benzoic Acid (<b>20</b>)</h4><div class="NLM_p last">According to the General Procedure for amide coupling, monomethyl terephthalate (23.0 mg, 0.120 mmol) was coupled with amine <b>13</b> (43.0 mg, 0.120 mmol). The crude product was purified by flash column chromatography, eluting with 25% EtOAc in c-hexane, to furnish the amide (46.0 mg, 76%). The intermediate product (35.0 mg, 0.070 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>20</b> (22.6 mg, 66%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.56 (9:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.21 (1 H, br. s, CO<sub>2</sub><i>H</i>), 10.39 (1 H, s, N<i>H</i>CO), 8.01 (2 H, d, <i>J</i> = 8.4, benzoate C-3), 7.94–7.86 (6 H, m, benzoate C-2, ArH-3 and ArH-5, phenyl H-ortho), 7.75 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.59–7.51 (3 H, m, phenyl H-meta and phenyl H-para); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 166.7 (C-5), 165.2 (<i>C</i>O<sub>2</sub>H), 161.7 (<i>C</i>O<sub>2</sub>NH), 157.8 (C-3), 137.1 (benzoate C-4), 135.3 (PhC-quart), 133.8 (benzoate C-1), 133.6 (ArC-3), 132.2 (ArC-2), 130.9 (q, <i>J</i> = 30.9, ArC-6), 130.6 (benzoate C-4), 129.3 (benzoate C-2), 129.2 (PhC-ortho), 127.9 (PhC-meta), 126.4 (PhC-para), 125.9 (benzoate C-3), 125.7 (q, <i>J</i> = 5.0, ArC-5), 125.1 (ArC-1), 122.9 (q, <i>J</i> = 274.7, <i>C</i>F<sub>3</sub>), 114.5 (C-4); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.06, observed 487.17, LC <i>R</i><sub>t</sub>: 6.73 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 487.0672, observed: 487.0677.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49">4.3.24.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-yl}amino) methyl] Benzoic Acid (<b>21</b>)</h4><div class="NLM_p last">Methly-4-formyl benzoate (93.0 mg, 0.570 mmol, 1.0 equiv) and AcOH (0.5 mL) were added to a solution of amine <b>13</b> (0.201 g, 0.590 mmol, 1.05 equiv) in MeOH (10.0 mL). The reaction mixture was heated at reflux for 18 h after which time it was cooled to room temperature, and NaCNBH<sub>3</sub> (74.0 mg, 1.18 mmol, 2.0 equiv) was added. The reaction mixture was heated at reflux for a further 18 h then concentrated in vacuo, diluted with EtOAc (10 mL), and washed with saturated aqueous NaHCO<sub>3</sub> (20 mL), water (20 mL), and brine (20 mL). The combined organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography, eluting with 17% EtOAc in <i>n</i>-heptane, to furnish the ester (0.120 g, 43%). The intermediate product (0.106 g, 0.220 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>21</b> (0.094 g, 98%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = baseline (1:1 cyclo-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.82 (1 H, br. s, CO<sub>2</sub><i>H</i>), 7.95 (1 H, d, <i>J</i> = 8.1, ArH-3), 7.88 (1 H, d, <i>J</i> = 8.1, ArH-5), 7.84 (2 H, d, <i>J</i> = 7.3, PhH-ortho), 7.80–7.78 (1 H, m, ArH-4), 7.75 (2 H, d, <i>J</i> = 8.1, benzoate H-2), 7.53–7.50 (2 H, m, PhH-meta), 7.47–7.43 (1 H, m, PhH-para), 7.15 (2 H, d, <i>J</i> = 8.1, benzoate H-3), 5.11 (1 H, app. t, <i>J</i> = 6.0, N<i>H</i>CH<sub>2</sub>), 4.00 (1 H, dd, <i>J</i> = 15.5, 6.0, NHC<i>H</i><sub>a</sub>), 3.94 (1 H, dd, <i>J</i> = 15.5, 6.0, NHC<i>H</i><sub>b</sub>); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 167.2 (C-5), 155.3 (<i>C</i>O<sub>2</sub>H), 154.3 (C-3), 145.0 (benzoate C-4), 136.0 (ArC-2), 133.6 (ArC-3), 132.2 (PhC-quart.), 131.1 (q, <i>J</i> = 30.5, ArC-6), 129.22 (benzoate C-1), 129.17 (ArC-4), 129.0 (PhC-ortho), 128.9 (benzoate C-2), 127.6 (PhC-para), 127.1 (PhC-meta), 126.0 (ArC-5), 125.9 (C-4), 125.7 (benzoate C-3), 125.4 (q, <i>J</i> = 5.0, ArC-1), 122.9 (q, <i>J</i> = 274.6, <i>C</i>F<sub>3</sub>), 49.8 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 473.08, observed 473.17, LC <i>R</i><sub>t</sub>: 7.48 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 473.0880, observed: 473.0883.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50">4.3.25.  4-({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-phenyl-1,2-oxazol-4-yl}sulfamoyl) Benzoic Acid (<b>22</b>)</h4><div class="NLM_p last">Methyl-4-(chlorosulfonyl)benzoate (0.464 g, 1.70 mmol, 3.0 equiv) was added to a solution of amine <b>13</b> (0.192 g, 0.570 mmol, 1.0 equiv) in pyridine (10.0 mL). The reaction mixture was stirred at 60 °C for 24 h then cooled to room temperature and concentrated in vacuo. The mixture was suspended in EtOAc (20 mL) and washed with 1 M aqueous HCl (20 mL), saturated aqueous NaHCO<sub>3</sub> (20 mL), water (20 mL), and brine (20 mL). The organic phase was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to furnish the ester (0.260 g, 71%) that was used without further purification. The intermediate product (0.188 g, 0.350 mmol) was subject to ester hydrolysis according to the General Procedure for ester hydrolysis and purified by trituation with Et<sub>2</sub>O to furnish <b>22</b> (0.079 g, 43%) as a yellow solid. <i>R</i><sub><i>f</i></sub> = baseline (1:1 cyclo-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.30 (1 H, br. s, CO<sub>2</sub><i>H</i>), 10.49 (1 H, br. s, N<i>H</i>SO<sub>2</sub>), 7.87 (1 H, d, <i>J</i> = 8.0, ArC-3), 7.79 (1 H, d, <i>J</i> = 8.0, ArC-5), 7.75–7.68 (5 H, m, ArH-4, PhH-ortho, benzoate H-3), 7.51 (2 H, d, <i>J</i> = 8.2, benzoate H-2), 7.46–7.37 (3 H, m, PhH-meta and PhH-para); <sup>13</sup>C NMR (100 MHz, DMSO-d6): δ (ppm) 165.9 (C-5), 164.8 (<i>C</i>O<sub>2</sub>H), 159.2 (C-3), 143.6 (benzoate C-4), 135.4 (ArC-2), 134.2 (ArC-3), 133.5 (benzoate C-1), 132.3 (PhC-quart.), 130.9 (q, <i>J</i> = 30.9, ArC-6), 130.8 (ArC-4), 129.7 (benzoate C-2), 128.8 (PhC-ortho), 126.2 (benzoate C-3), 126.1 (PhC-meta), 125.5 (q, <i>J</i> = 5.0, ArC-1), 125.2 (PhC-para), 124.4 (ArC-5), 122.8 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 112.4 (C-4); LC–MS (ESI): calcd for C<sub>23</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>5</sub>S [M + H]<sup>+</sup>: 523.03, observed: 523.00, LC <i>R</i><sub>t</sub>: 6.70 min. HRMS (ESI): calcd for C<sub>23</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>5</sub>S [M + H]<sup>+</sup>: 523.0342, observed: 523.0333.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51">4.3.26.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(furan-2-yl)-1,2-oxazol-4-yl}methyl)amino]benzoic Acid (<b>23</b>)</h4><div class="NLM_p last">According to the General Procedure for reductive amination, aldehyde <b>39</b> (0.180 g, 0.520 mmol) was reacted with <i>tert</i>-butyl-4-amino benzoate (0.102 g, 0.520 mmol) in MeOH. The crude product was purified by column chromatography, eluting with a gradient of 5–20% EtOAc in <i>n</i>-heptane, to furnish the intermediate imine (0.136 g) that was immediately subjected to the reduction step performed in EtOH at 85 °C. The crude product was purified by column chromatography, eluting with a gradient of 10–20% EtOAc in <i>n</i>-heptane, to furnish the intermediate amine (0.065 g, 24%). This product was subject to <i>tert</i>-butyl ester deprotection according to the General Procedure for <i>tert</i>-butyl ester deprotection. The crude product was purified by column chromatography, eluting with 1% MeOH in CH<sub>2</sub>Cl<sub>2</sub>, to furnish the carboxylic acid <b>23</b> (0.040 g, 73%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.13 (99:1 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, MeOD): δ (ppm) 7.84 (1 H, d, <i>J</i> = 1.8, furanyl H-5), 7.75 (1 H, d, <i>J</i> = 8.1, Ar H-3 or ArH-5), 7.74 (1 H, d, <i>J</i> = 8.1, ArH-3 or ArH-5), 7.61 (1 H, app. t, <i>J</i> = 8.1, ArH-4), 7.61 (2 H, d, <i>J</i> = 8.8, benzoate H-2), 7.12 (1 H, d, <i>J</i> = 3.5, furanyl H-3), 7.62 (1 H, dd, <i>J</i> = 3.5, 1.8, furanyl H-4), 6.34 (2 H, d, <i>J</i> = 8.8, benzoate H-3), 4.46 (1 H, d, <i>J</i> = 15.3, C<i>H</i><sub>a</sub>NH), 4.36 (1 H, d, <i>J</i> = 15.3, C<i>H</i><sub>b</sub>NH); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 170.5 (C-5), 160.6 (<i>C</i>O<sub>2</sub>H), 159.9 (C-3), 153.4 (furanyl C-2), 146.5 (furanyl C-5), 144.1 (benzoate C-4), 137.6 (ArC-2), 134.6 (ArC-3), 132.9 (q, <i>J</i> = 32.5, ArC-6), 132.7 (ArC-4), 132.4 (benzoate C-2), 127.3 (ArC-1), 126.3 (q, <i>J</i> = 5.1, ArC-5), 124.4 (q, <i>J</i> = 273.8, <i>C</i>F<sub>3</sub>), 118.7 (benzoate C-1), 114.6 (C-4), 113.2 (furanyl C-4), 113.1 (furanyl C-3), 111.8 (benzoate C-3), 36.4 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 463.06, observed: 462.92, LC <i>R</i><sub>t</sub>: 6.67 min; HRMS (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 463.0672, observed: 463.0661.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52">4.3.27.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(thiophen-2-yl)-1,2-oxazol-4-yl}methyl)amino]benzoic Acid (<b>24</b>)</h4><div class="NLM_p last">According to the General Procedure for reductive amination, aldehyde <b>40</b> (0.096 g, 0.270 mmol) was reacted with <i>tert</i>-butyl-4-amino benzoate (0.052 g, 0.27 mmol) in MeOH. The crude product was purified by column chromatography, eluting with a gradient of 5–10% EtOAc in <i>n</i>-heptane, to furnish the intermediate imine (0.125 g) that was immediately subjected to the reduction step performed in EtOH at 85 °C for 5 h. The crude product was purified by column chromatography, eluting with a gradient of 3–10% EtOAc in <i>n</i>-heptane, to furnish the intermediate amine (24.0 mg, 17%). This product was subject to <i>tert</i>-butyl ester deprotection according to the General Procedure for <i>tert</i>-butyl ester deprotection. The crude product was purified by column chromatography, eluting with 3% MeOH in CH<sub>2</sub>Cl<sub>2</sub>, to furnish the carboxylic acid <b>24</b> (12.0 mg, 56%) as a pale yellow solid. <i>R</i><sub><i>f</i></sub> = 0.13 (98:2 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.02 (1 H, br. s, CO<sub>2</sub><i>H</i>), 7.96 (1 H, dd, <i>J</i> = 5.1, 1.1, thiophenyl H-5), 7.93 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.87 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.77–7.73 (2 H, m, ArH-4, thiophenyl H-3), 7.55 (2 H, d, <i>J</i> = 8.8, benzoate H-2), 7.33 (1 H, dd, <i>J</i> = 5.1, 3.7, thiophenyl H-4), 6.48 (1 H, app. t, <i>J</i> = 5.0, CH<sub>2</sub>N<i>H),</i> 6.42 (2 H, d, <i>J</i> = 8.8, benzoate H-3), 4.20 (1 H, dd, <i>J</i> = 14.8, 5.0, C<i>H</i><sub>a</sub>NH), 4.13 (1 H, dd, <i>J</i> = 14.8, 5.0, C<i>H</i><sub>b</sub>NH); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 167.4 (C-5), 162.0 (CO<sub>2</sub><i>H</i>), 159.5 (C-3), 151.6 (benzoate C-4), 135.5 (ArC-2), 133.8 (ArC-3), 132.3 (ArC-4), 130.8 (benzoate C-2), 130.6 (q, <i>J</i> = 30.7, ArC-6), 130.6 (thiophenyl C-5), 128.8 (thiophenyl C-3), 128.7 (thiophenyl C-4), 127.2 (thiophenyl C-2), 125.5 (q, <i>J</i> = 4.9, ArC-5), 125.3 (ArC-1), 122.8 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 117.6 (benzoate C-1), 112.0 (C-4), 110.8 (benzoate (C-3), 35.6 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>S [M + H]<sup>+</sup>: 479.04, observed: 479.00, LC <i>R</i><sub>t</sub>: 7.23 min. HRMS (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>S [M + H]<sup>+</sup>: 479.0444, observed: 479.0429.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53">4.3.28.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(1H-pyrrol-3-yl)-1,2-oxazol-4-yl}methyl)amino]benzoic Acid (<b>25</b>)</h4><div class="NLM_p last">According to the General Procedure for reductive amination, aldehyde <b>41</b> (0.060 g, 0.176 mmol) was reacted with methyl-4-amino benzoate (0.032 g, 0.211 mmol) in MeOH. The crude product was purified by column chromatography, eluting with 25% EtOAc in <i>n</i>-heptane, to furnish the intermediate imine (0.034 mg) that was immediately subjected to the reduction step performed in MeOH at reflux for 2 h. The crude product was purified by column chromatography, eluting with a gradient of 20% EtOAc in <i>n</i>-heptane, to furnish the intermediate amine (16.3 mg, 19%). This product was subject to ester hydrolysis according to the General Procedure for <i>tert</i>-Butyl ester deprotection. The crude product was purified by column chromatography, eluting with 1.5% MeOH in CH<sub>2</sub>Cl<sub>2</sub>, to furnish the carboxylic acid <b>25</b> (5.40 mg, 57%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.10 (96:4 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.99 (1 H, br. s, CO<sub>2</sub><i>H</i>), 11.52 (1 H, s, pyrrole-N<i>H</i>), 7.91 (1 H, d, <i>J</i> = 8.1, ArH-3 or ArH-5), 7.85 (1 H, d, <i>J</i> = 7.9, ArH-3 or ArH-5), 7.72 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.54 (2 H, d, <i>J</i> = 8.4, benzoate C-2), 7.39 (1 H, m, pyrrole H-2), 6.98 (1 H, app. q, <i>J</i> = 2.4, pyrrole H-5), 6.55 (1 H, d, <i>J</i> = 2.4, pyrrole H-4), 6.42 (2 H, d, <i>J</i> = 8.4, benzoate H-3), 6.34 (1 H, app. t, <i>J</i> = 5.0, CH<sub>2</sub>N<i>H</i>), 4.08 (1 H, dd, <i>J</i> = 14.5, 5.0, C<i>H</i><sub>a</sub>NH), 4.00 (1 H, dd, <i>J</i> = 14.5, 5.0, C<i>H</i><sub>a</sub>NH); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 167.9 (C-5), 165.7 (<i>C</i>O<sub>2</sub>H), 159.2 (C-3), 152.3 (benzoate C-4), 136.0 (ArC-2), 134.1 (ArC-3), 132.4 (ArC-4), 131.3 (benzoate C-2), 130.9 (q, <i>J</i> = 30.4, ArC-6), 126.8 (ArC-1), 125.8 (q, <i>J</i> = 5.0, ArC-5), 122.0 (q, <i>J</i> = 274.3, <i>C</i>F<sub>3</sub>), 120.5 (pyrrole C-5), 119.2 (pyrrole C-2), 117.8 (benzoate C-1), 111.2 (benzoate C-3), 110.5 (C-4), 109.2 (pyrrole C-3), 106.7 (pyrrole C-4), 36.2 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>22</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 462.08, observed: 462.00, LC <i>R</i><sub>t</sub>: 6.20 min. HRMS (ESI): calcd for C<sub>22</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 462.0832, observed: 462.0834.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54">4.3.29.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(naphthalen-1-yl)-1,2-oxazol-4-yl}methyl)amino]benzoic Acid (<b>26</b>)</h4><div class="NLM_p last">According to the General Procedure for reductive amination, aldehyde <b>42</b> (0.034 g, 0.0850 mmol) was reacted with <i>tert</i>-butyl-4-amino benzoate (0.016 g, 0.085 mmol) in MeOH. The crude product was purified by column chromatography, eluting with a gradient of 2–10% EtOAc in <i>n</i>-heptane, to furnish the intermediate imine (0.010 g) that was immediately subjected to the reduction step performed in EtOH at 85 °C for 5 h. The crude product was purified by column chromatography, eluting with 20% EtOAc in <i>n</i>-heptane, to furnish the intermediate amine (8.00 mg, 16%). This product was subjected to <i>tert</i>-butyl ester deprotection according to the General Procedure for <i>tert</i>-Butyl ester deprotection. The crude product was purified by column chromatography, eluting with 3% MeOH in CH<sub>2</sub>Cl<sub>2</sub>, to furnish the carboxylic acid <b>26</b> (3.00 mg, 48%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.16 (96:4 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.93 (1 H, br. s, CO<sub>2</sub><i>H</i>), 8.23 (1 H, d, <i>J</i> = 8.3, naphthyl-H), 8.12 (1 H, d, <i>J</i> = 7.5, naphthyl-H), 7.94 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.88 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.84 (1 H, d, <i>J</i> = 7.0, naphthyl-H), 7.76–7.66 (5 H, m, ArH-4, naphthyl-H), 7.31 (2 H, d, <i>J</i> = 8.3, benzoate C-2), 6.36 (1 H, app. t, <i>J</i> = 5.8, CH<sub>2</sub>N<i>H</i>), 6.08 (2 H, d, <i>J</i> = 8.3, benzoate C-3), 4.08 (1 H, dd, <i>J</i> = 15.8, 5.8, C<i>H</i><sub>a</sub>NH), 4.01 (1 H, dd, <i>J</i> = 15.8, 5.8, C<i>H</i><sub>a</sub>NH); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 167.3 (C-5), 166.9 (CO<sub>2</sub><i>H</i>), 158.8 (C-3), 151.3 (benzoate C-4), 135.4 (ArC-2), 133.7 (ArC-3), 133.3 (naphthyl-C), 132.1 (ArC-4), 131.4 (naphthyl-C), 130.9 (naphthyl-C), 130.5 (q, <i>J</i> = 31.1, ArC-6), 130.5 (benzoate C-2), 129.3 (naphthyl-C), 128.7 (naphthyl-C), 127.8 (naphthyl-C), 126.8 (naphthyl-C), 125.8 (ArC-1), 125.5 (q, <i>J</i> = 4.0, ArC-5), 125.4 (naphthyl-C), 124.3 (naphthyl-C), 123.5 (naphthyl-C), 123.0 (q, <i>J</i> = 274.3, <i>C</i>F<sub>3</sub>), 117.2 (benzoate C-1), 116.3 (C-4), 110.4 (benzoate C-3), 35.3 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>28</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 523.10, observed: 522.92, LC <i>R</i><sub>t</sub>: 7.68 min. HRMS (ESI): calcd for C<sub>28</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 523.1036, observed: 523.1046.</div></div><div id="sec4_3_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55">4.3.30.  4-[({3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(3-hydroxyphenyl)-1,2-oxazol-4-yl}methyl)amino]benzoic Acid (<b>27</b>)</h4><div class="NLM_p last">According to the General Procedure for reductive amination, aldehyde <b>43</b> (0.123 g, 0.255 mmol) was reacted with methyl-4-amino benzoate (0.038 g, 0.255 mmol) in MeOH. The crude product was purified by column chromatography, eluting with a gradient of 2–12% EtOAc in <i>n</i>-heptane, to furnish the intermediate imine (0.053 g) that was immediately subjected to the reduction step performed in MeOH at reflux for 3.5 h. This step occurred with concomitant loss of the silyl protecting group. The crude product was purified by column chromatography, eluting with a gradient of 15–35% EtOAc in <i>n</i>-heptane, to furnish the intermediate amine (21.0 mg, 16%). This product was subject to ester hydrolysis according to the General Procedure for <i>tert</i>-butyl ester deprotection. The crude product was purified by column chromatography, eluting with 3% MeOH in CH<sub>2</sub>Cl<sub>2</sub>, to furnish the carboxylic acid <b>27</b> (17.4 mg, 99%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.10 (96:4 CH<sub>2</sub>Cl<sub>2</sub>-MeOH); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 7.90 (1 H, d, <i>J</i> = 8.1, ArH-3 or ArH-5), 7.85 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.72 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.52 (2 H, d, <i>J</i> = 8.8, benzoate H-2), 7.40 (1 H, app. t, <i>J</i> = 8.0, phenol H-5), 1.30–7.26 (2 H, m, phenol H-2 and phenol H-4), 6.99 (1 H, dd, <i>J</i> = 8.0, 1.5, phenol H-6), 6.42 (1 H, t, <i>J</i> = 5.0, CH<sub>2</sub>N<i>H</i>), 6.36 (2 H, d, <i>J</i> = 8.8, benzoate H-3), 4.17 (1 H, dd, <i>J</i> = 14.6, 5.0, C<i>H</i><sub>a</sub>NH), 4.09 (1 H, dd, <i>J</i> = 14.6, 5.0, C<i>H</i><sub>b</sub>NH); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 167.4 (C-5), 166.4 (<i>C</i>O<sub>2</sub>H), 159.6 (C-3), 157.9 (phenol C-3), 151.6 (benzoate C-4), 135.5 (ArC-2), 133.7 (ArC-3), 132.2 (ArC-4), 130.8 (benzoate C-2), 130.5 (q, <i>J</i> = 30.7, ArC-6), 130.5 (phenol C-1), 127.7 (phenol C-5), 125.7 (ArC-1), 125.4 (q, <i>J</i> = 4.0, ArC-5), 122.8 (q, <i>J</i> = 274.3, <i>C</i>F<sub>3</sub>), 117.8 (phenol C-6), 117.6 (benzoate C-1), 113.6 (phenol C-2), 112.9 (C-4), 110.7 (benzoate C-3), 35.7 (<i>C</i>H<sub>2</sub>NH); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 489.08, observed: 489.00, LC <i>R</i><sub>t</sub>: 6.30 min. HRMS (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 489.0829, observed: 489.0823.</div></div><div id="sec4_3_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56">4.3.31.  Methyl 5-Bromo-3-[2-chloro-6-(trifluoromethyl)phenyl]-1,2-oxazole-4-carboxylate (<b>30a</b>)</h4><div class="NLM_p last">Methyl 3-bromopropiolate (prepared according to ref <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a>, 2.04 g, 12.5 mmol, 1.0 equiv) was added to a solution of nitrile oxide <b>33</b> (2.75 g, 12.5 mmol, 1.0 equiv) in THF (25.0 mL), and the reaction mixture was heated at reflux for 4 h. The reaction mixture was concentrated in vacuo to give a crude product as a 7:3 mixture of regioisomers. Purification by recrystallization from hot <i>n</i>-heptane furnished bromide <b>30a</b> (2.05 g, 43%) as a colorless solid (97:3 mixture of regioisomers, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). <i>R</i><sub><i>f</i></sub> = 0.20 (7:3 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.73 (1 H, d, <i>J</i> = 8.6, ArH-3 or ArH-5), 7.71 (1 H, d, <i>J</i> = 8.9, ArH-3 or ArH-5), 7.58 (1 H, app. t. <i>J</i> = 8.0, ArH-3), 3.70 (3 H, s, CO<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 160.0 (C-5), 159.7 (<i>C</i>O<sub>2</sub>Me), 148.0 (C-3), 136.3 (ArC-2), 133.0 (ArC-3), 131.7 (q, <i>J</i> = 31.7, ArC-6), 131.3 (ArC-4), 126.0 (ArC-1), 124.8 (q, <i>J</i> = 5.0, ArC-5), 122.9 (q, <i>J</i> = 274.4, <i>C</i>F<sub>3</sub>), 113.2 (C-4), 52.3 (CO<sub>2</sub><i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>12</sub>H<sub>7</sub>BrClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 383.92, observed 386.00, LC <i>R</i><sub>t</sub>: 7.12 min.</div></div><div id="sec4_3_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57">4.3.32.  [2-Chloro-6-(trifluoromethyl)phenyl]formonitrile Oxide (<b>33</b>)</h4><div class="NLM_p last">NEt<sub>3</sub> (5.80 mL, 41.5 mmol, 1.2 equiv) was added dropwise to a solution of imidoyl chloride <b>9a</b> (8.90 g, 34.6 mmol, 1.0 equiv) in THF (110 mL). A white precipitate formed immediately. The resulting suspension was stirred vigorously at room temperature for 30 min and then filtered through a pad of SiO<sub>2</sub> that was subsequently washed with THF (250 mL). The solution was concentrated in vacuo to furnish <b>33</b> (8.25 g, 99%) as a colorless solid which was used immediately. <i>R</i><sub><i>f</i></sub> = 0.33 (9:1 c-hexane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.70 (1 H, d, <i>J</i> = 8.2, H-3 or H-5), 7.67 (1 H, d, <i>J</i> = 8.2, H-3 or H-5), 7.54 (1 H, app. t, <i>J</i> = 8.2, H-4).</div></div><div id="sec4_3_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58">4.3.33.  Methyl 3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(furan-2-yl)-1,2-oxazole-4-carboxylate (<b>34</b>)</h4><div class="NLM_p last">According to the General Procedure for Suzuki coupling, bromide <b>30a</b> (0.150 g, 0.390 mmol) was coupled to furan-2-boronic acid pinacol ester (0.114 g, 0.585 mmol). The crude product was purified by flash column chromatography, eluting with a gradient of 10–50% EtOAc in <i>n</i>-heptane, to furnish <b>34</b> (0.088 g, 58%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.19 (85:15 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.82 (1 H, dd, <i>J</i> = 3.6, 0.7, furanyl H-5), 7.73 (1 H, d, <i>J</i> = 8.0, Ar H-3 or ArH-5), 7.72 (1 H, dd, <i>J</i> = 1.8, 0.7, furanyl H-3), 7.70 (1 H, d, <i>J</i> = 8.0, Ar H-3 or ArH-5), 7.56 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 6.66 (1 H, dd, <i>J</i> = 3.6, 1.8, furanyl H-4), 3.62 (3 H, s, CO<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 163.4 (C-5), 160.9 (<i>C</i>O<sub>2</sub>CH<sub>3</sub>), 158.7 (C-3), 146.3 (furanyl C-3), 141.6 (furanyl C-2), 136.3 (ArC-2), 132.8 (ArC-3 and ArC-4), 131.6 (q, <i>J</i> = 31.3, ArC-6), 130.8 (ArC-4), 127.1 (ArC-1), 124.7 (q, <i>J</i> = 5.0, ArC-5), 123.0 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 118.3 (furanyl C-5), 112.6 (furanyl C-4), 107.3 (C-4), 51.9 (CO<sub>2</sub><i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>16</sub>H<sub>10</sub>ClF<sub>3</sub>NO<sub>4</sub> [M + H]<sup>+</sup>: 371.02, observed: 372.08, LC <i>R</i><sub>t</sub>: 7.77 min.</div></div><div id="sec4_3_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59">4.3.34.  Methyl 3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(thiophen-2-yl)-1,2-oxazole-4-carboxylate (<b>35</b>)</h4><div class="NLM_p last">According to the General Procedure for Suzuki coupling, bromide <b>30a</b> (0.250 g, 0.650 mmol) was coupled to thiophene-2-boronic acid pinacol ester (0.273 g, 1.30 mmol). The crude product was purified by flash column chromatography, eluting with a gradient of 5–20% EtOAc in <i>n</i>-heptane, to furnish <b>35</b> (0.136 g, 54%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.32 (4:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.29 (1 H, dd, <i>J</i> = 3.9, 1.2, thiophenyl H-5), 7.75–7.69 (3 H, m, ArH-3, ArH-5, thiophenyl H-3), 7.57 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.24 (1 H, dd, <i>J</i> = 5.1, 3.9, thiophenyl H-4), 3.64 (3 H, s, CO<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 167.9 (C-5), 161.4 (<i>C</i>O<sub>2</sub>CH<sub>3</sub>), 159.2 (C-3), 136.3 (ArC-2), 132.8 (thiophenyl C-5), 132.7 (ArC-4), 132.2 (ArC-3 and thiophenyl C-3), 131.7 (q, <i>J</i> = 31.4, ArC-6), 130.8 (ArH-4), 128.0 (thiophenyl C-4), 127.4 (thiophenyl C-2), 127.3 (ArC-1), 124.7 (q, <i>J</i> = 5.0, ArC-5), 123.0 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 107.2 (C-4), 51.9 (CO<sub>2</sub><i>C</i>H<sub>3</sub>); LC–MS (ESI): calcd for C<sub>16</sub>H<sub>10</sub>ClF<sub>3</sub>NO<sub>3</sub>S [M + H]<sup>+</sup>: 387.99, observed: 388.25, LC <i>R</i><sub>t</sub>: 7.62 min.</div></div><div id="sec4_3_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60">4.3.35.  Methyl 5-{1-[(<i>tert</i>-Butoxy)carbonyl]-1H-pyrrol-3-yl}-3-[2-chloro-6-(trifluoromethyl)phenyl]-1,2-oxazole-4-carboxylate (<b>36</b>)</h4><div class="NLM_p last">According to the General Procedure for Suzuki coupling, bromide <b>30a</b> (0.250 g, 0.650 mmol) was coupled to <i>tert</i>-butyl 3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1H-pyrrole-1-carboxylate (0.381 g, 1.30 mmol). The crude product was purified by flash column chromatography, eluting with a gradient of 5–20% EtOAc in <i>n</i>-heptane, to furnish <b>36</b> (0.119 g, 39%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.30 (4:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.45 (1 H, app. t, <i>J</i> = 2.0, pyrrole H-2), 7.72–7.67 (2 H, m, ArH-3 and ArH-5), 7.54 (1 H, app. t, <i>J</i> = 7.9, ArH-4), 7.35 (1 H, dd, <i>J</i> = 3.4, 2.0, pyrrole H-5), 6.98 (dd, <i>J</i> = 3.4, 2.0, pyrrole H-4), 3.61 (3 H, s, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 1.63 (9 H, s, C(C<i>H</i><sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 168.9 (C-5), 161.6 (<i>C</i>O<sub>2</sub>CH<sub>3</sub>), 159.0 (C-3), 148.2 (N<i>C</i>O<sub>2</sub>), 136.3 (ArC-2), 132.8 (ArC-3), 131.6 (q, <i>J</i> = 31.3, ArC-6), 130.6 (ArC-4), 127.7 (ArC-1), 124.7 (q, <i>J</i> = 5.0, ArC-5), 124.1 (pyrrole C-2), 123.0 (q, <i>J</i> = 274.4, <i>C</i>F<sub>3</sub>), 121.0 (pyrrole C-5), 113.8 (pyrrole C-3), 111.5 (pyrrole C-4), 107.2 (C-4), 85.2 (<i>C</i>(CH<sub>3</sub>)<sub>3</sub>), 51.7 (CO<sub>2</sub><i>C</i>H<sub>3</sub>), 28.0 (C(<i>C</i>H<sub>3</sub>)<sub>3</sub>); LC–MS (ESI): calcd for C<sub>21</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>: 471.09, observed: 471.17, LC <i>R</i><sub>t</sub>: 8.55 min.</div></div><div id="sec4_3_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61">4.3.36.  Methyl 3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(naphthalen-1-yl)-1,2-oxazole-4-carboxylate (<b>37</b>)</h4><div class="NLM_p last">According to the General Procedure for Suzuki coupling, bromide <b>30a</b> (0.150 g, 0.390 mmol) was coupled to naphthalene-1-boronic acid pinacol ester (0.198 g, 0.780 mmol). The crude product was purified by flash column chromatography, eluting with a gradient of 10– 20% EtOAc in <i>n</i>-heptane, to furnish <b>37</b> (0.059 g, 35%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.43 (4:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.07 (1 H, d, <i>J</i> = 8.3, naphthyl-H), 7.97–7.94 (1 H, m, naphthyl-H), 7.81–7.75 (3 H, m, naphthyl-H), 7.73–7.71 (1 H, m, ArH-3 or ArH-5), 7.65–7.60 (2 H, m, ArH-3 or ArH-5, naphthyl-H), 7.58–7.56 (2 H, m, ArH-3 or ArH-5, naphthyl-H), 3.41 (3 H, s, CO<sub>2</sub>C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 173.7 (C-5), 161.0 (<i>C</i>O<sub>2</sub>CH<sub>3</sub>), 158.9 (C-4), 136.4, 133.5, 132.9, 131.9, 131.8 (q, <i>J</i> = 31.4, ArC-6), 131.4, 130.9, 130.2, 129.4, 128.7, 127.6, 127.1 (ArC-1), 126.7, 124.9, 124.8, 124.2, 123.2 (q, <i>J</i> = 274.4, <i>C</i>F<sub>3</sub>), 112.2 (C-4), 51.8 (CO<sub>2</sub><i>C</i>H<sub>3</sub>), (not all peaks could be precisely assigned with certainty); LC–MS (ESI): calcd for C<sub>22</sub>H<sub>14</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 432.05, observed: 432.25, LC <i>R</i><sub>t</sub>: 8.14 min.</div></div><div id="sec4_3_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62">4.3.37.  Methyl 5-{3-[(<i>tert</i>-Butyldimethylsilyl)oxy]phenyl}-3-[2-chloro-6-(trifluoromethyl)phenyl]-1,2-oxazole-4-carboxylate (<b>38</b>)</h4><div class="NLM_p last">According to the General Procedure for Suzuki coupling, bromide <b>30a</b> (0.400 g, 1.04 mmol) was coupled to 3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenol (0.319 g, 1.45 mmol). The crude product was purified by flash column chromatography, eluting with a gradient of 10–20% EtOAc in <i>n</i>-heptane, to furnish the phenol (0.167 g, 40%) as a colorless solid. Imidazole (94.0 mg, 1.38 mmol, 3.0 equiv) and <i>tert</i>-butyldimethylsilyl chloride (0.104 g, 0.690 mmol, 1.5 equiv) were added to a solution of the phenol (0.183 g, 0.460 mmol, 1.0 equiv) in DMF (3.0 mL). The reaction mixture was stirred at room temperature for 3 h then diluted with saturated aqueous NH<sub>4</sub>Cl (20 mL) and extracted with EtOAc (2 × 20 mL). The combined organic phase was washed with water (2 × 50 mL) and brine (50 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo. The crude product was purified by column chromatography, eluting with a gradient of 0–20% EtOAc in <i>n</i>-heptane, to furnish silyl ether <b>38</b> (56.0 mg, 70%) as a colorless oil. <i>R</i><sub>f</sub> = 0.26 (9:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.73 (2 H, app. t, J = 8.6, ArH-3 and ArH-5), 7.70 (1 H, ddd, J = 8.1, 1.7, 1.0, phenol H-4), 7.59–7.55 (2 H, m, phenol H-2, ArH-4), 7.39 (1 H, app. t, J = 8.1, phenol H-5), 7.04 (1 H, ddd, J = 8.1, 2.4, 1.0, phenol H-6), 3.59 (3 H, s, CO<sub>2</sub>CH<sub>3</sub>), 1.02 (9 H, s, Si(CH<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>), 0.26 (6 H, s, Si(CH<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 172.7 (C-5), 161.4 (CO<sub>2</sub>CH<sub>3</sub>), 159.4 (C-3), 155.9 (phenol C-3), 136.4 9ArC-2), 132.9 (ArC-3), 131.7 (q, J = 31.3, ArC-6), 130.8 (ArC-4), 129.7 (phenol C-5), 127.6 (ArC-1), 127.5 (phenol C-1), 124.8 (q, J = 5.0, ArC-5), 123.7 (phenol C-6), 123.0 (q, J = 274.4, CF<sub>3</sub>), 122.4 (phenol C-4), 120.8 (phenol C-2), 109.3 (C-4), 52.0 (CO<sub>2</sub>CH<sub>3</sub>), 25.8 (Si(CH<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>), 18.3 (Si(CH<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>), −4.3 (Si(CH<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>); LC–MS (ESI): calcd for C<sub>24</sub>H<sub>26</sub>ClF<sub>3</sub>NO<sub>4</sub>Si [M + H]<sup>+</sup>: 512.12, observed: 512.25, LC <i>R</i><sub>t</sub>: 9.62 min.</div></div><div id="sec4_3_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63">4.3.38.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(furan-2-yl)-1,2-oxazole-4-carbaldehyde (<b>39</b>)</h4><div class="NLM_p last">Ester <b>34</b> (0.192 g, 0.516 mmol) was treated according to the General Procedure for conversion of esters to aldehydes, to furnish aldehyde <b>39</b> (0.169 g, 96%) as a colorless oil. The crude product was used without further purification. <i>R</i><sub><i>f</i></sub> = 0.51 (3:2 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.20 (1 H, s, C<i>H</i>O), 7.77 (1 H, dd, <i>J</i> = 1.8, 0.8, furanyl H-5), 7.76 (1 H, d, <i>J</i> = 8.0, Ar H-3 or ArH-5), 7.73 (1 H, d, <i>J</i> = 7.8, ArH-3 or ArH-5), 7.60 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.55 (1 H, dd, <i>J</i> = 3.6, 0.8, furanyl H-3), 6.72 (1 H, dd, <i>J</i> = 3.6, 1.8, furanyl H-4); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 182.8 (<i>C</i>HO), 164.1 (C-5), 154.7 (C-3), 147.2 (furanyl C-3), 142.2 (furanyl C-2), 136.2 (ArC-2), 133.1 (ArC-3), 131.8 (q, <i>J</i> = 31.5, ArC-6), 131.3 (ArC-4), 125.6 (ArC-1), 125.0 (q, <i>J</i> = 5.0, ArC-5), 122.9 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 116.9 (furanyl C-5), 115.2 (C-4), 112.9 (furanyl C-4); LC–MS (ESI): calcd for C<sub>15</sub>H<sub>8</sub>ClF<sub>3</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: 342.01, observed: 342.08, LC <i>R</i><sub>t</sub>: 6.93 min.</div></div><div id="sec4_3_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64">4.3.39.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(thiophen-2-yl)-1,2-oxazole-4-carbaldehyde (<b>40</b>)</h4><div class="NLM_p last">Ester <b>35</b> (0.082 g, 0.210 mmol) was treated according to the General Procedure for conversion of esters to aldehydes. The crude product was purified by flash column chromatography, eluting with a gradient of 2–10% EtOAc in <i>n</i>-heptane, to furnish aldehyde <b>40</b> (0.048 g, 64%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.28 (4:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 9.78 (1 H, s, C<i>H</i>O), 8.32 (1 H, dd, <i>J</i> = 3.9, 1.2, thiophenyl H-5), 7.80–7.75 (3 H, m, ArH-3, ArH-5, thiophenyl H-3), 7.64 (1 H, app. t, <i>J</i> = 8.1, ArH-4), 7.29 (1 H, dd, <i>J</i> = 5.1, 3.9, thiophenyl H-4); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 182.2 (<i>C</i>HO), 168.0 (C-5), 159.0 (C-3), 136.6 (ArC-2), 133.4 (thiophenyl C-5), 132.9 (ArC-3), 132.8 (ArC-4), 132.3 (q, <i>J</i> = 31.5, ArC-6), 131.6 (thiophenyl C-3), 128.7 thiophenyl C-4), 127.0 (thiophenyl C-2), 125.3 (ArC-1), 125.2 (q, <i>J</i> = 5.0, ArC-5), 122.9 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 114.4 (C-4); LC–MS (ESI): calcd for C<sub>15</sub>H<sub>8</sub>ClF<sub>3</sub>NO<sub>2</sub>S [M + H]<sup>+</sup>: 357.98, observed: 358.17, LC <i>R</i><sub>t</sub>: 7.27 min.</div></div><div id="sec4_3_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65">4.3.40.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(1H-pyrrol-3-yl)-1,2-oxazole-4-carbaldehyde (<b>41</b>)</h4><div class="NLM_p last">Ester <b>36</b> (0.228 g, 0.486 mmol) was treated according to the General Procedure for conversion of esters to aldehydes with a modification: the reduction step was performed with 3.0 equiv of reducing agent and without cooling; this step occurred with concomitant loss of the Boc protecting group. The crude product was purified by flash column chromatography, eluting with 30% EtOAc in <i>n</i>-heptane, to furnish aldehyde <b>41</b> (0.084 g, 51%) as a brown solid. <i>R</i><sub><i>f</i></sub> = 0.20 (7:3 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 9.62 (1 H, s, C<i>H</i>O), 8.77 (1 H, br. s, N<i>H</i>), 8.26 (1 H, app. dt, <i>J</i> = 3.3, 1.7, pyrrole H-2), 7.78 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.75 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.61 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.02–6.95 (1 H, m, pyrrole H-5), 6.95 (1 H, dd, <i>J</i> = 2.7, 1.7, pyrrole H-4); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 182.8 (<i>C</i>HO), 170.7 (C-5), 160.0 (C-3), 133.6 (ArC-2), 133.3 (ArC-3), 132.3 (q, <i>J</i> = 31.5, ArC-6), 131.4 (ArC-4), 125.7 (ArC-1), 125.1 (q, <i>J</i> = 5.0, ArC-5), 123.8 (pyrrole C-2), 122.9 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 120.3 (pyrrole C-5), 113.2 (pyrrole C-3), 110.7 (C-4), 108.6 (pyrrole C-4); LC–MS (ESI): calcd for C<sub>15</sub>H<sub>9</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 341.02, observed: 341.08, LC <i>R</i><sub>t</sub>: 6.33 min.</div></div><div id="sec4_3_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66">4.3.41.  3-[2-Chloro-6-(trifluoromethyl)phenyl]-5-(naphthalen-1-yl)-1,2-oxazole-4-carbaldehyde (<b>42</b>)</h4><div class="NLM_p last">Ester <b>37</b> (0.068 g, 0.160 mmol) was treated according to the General Procedure for conversion of esters to aldehydes. The crude product was purified by flash column chromatography, eluting with 17% EtOAc in <i>n</i>-heptane, to furnish aldehyde <b>42</b> (0.039 g, 61%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.28 (4:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 9.73 (1 H, s, C<i>H</i>O), 8.14 (1 H, d, <i>J</i> = 8.5, naphthyl-H), 8.02–7.98 (2 H, m, naphthyl-H), 7.83 (1 H, dd, <i>J</i> = 7.1, 1.2. naphthyl-H), 7.79 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.76 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.68–7.61 (4 H, m, ArH-4 and naphthy-H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 183.1 (<i>C</i>HO), 176.6 (C-5), 157.4 (C-4), 136.3, 133.8, 133.1, 132.9, 131.8 (q, <i>J</i> = 31.5, ArC-6), 131.4, 131.2, 130.1, 128.9, 128.5, 127.3, 126.1 (ArC-1), 125.1, 125.0 (q, <i>J</i> = 5.0, ArC-5), 124.7, 123.1 (q, <i>J</i> = 274.4, <i>C</i>F<sub>3</sub>), 122.5, 118.7 (C-4) (not all peaks could be precisely assigned with certainty); LC–MS (ESI): calcd for C<sub>21</sub>H<sub>12</sub>ClF<sub>3</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: 402.04, observed: 401.92, LC <i>R</i><sub>t</sub>: 8.15 min.</div></div><div id="sec4_3_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67">4.3.42.  5-{3-[(<i>tert</i>-Butyldimethylsilyl)oxy]phenyl}-3-[2-chloro-6-(trifluoromethyl)phenyl]-1,2-oxazole-4-carbaldehyde (<b>43</b>)</h4><div class="NLM_p last">Ester <b>38</b> (0.173 g, 0.340 mmol) was treated according to the General Procedure for conversion of esters to aldehydes. The crude product was purified by flash column chromatography, eluting with a gradient of 0–10% EtOAc in <i>n</i>-heptane, to furnish aldehyde <b>43</b> (0.123 g, 80%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.34 (9:1 <i>n</i>-heptane-EtOAc); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 9.94 (1 H, s, C<i>H</i>O), 7.77 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.74 (1 H, d, <i>J</i> = 8.0, ArH-3 or ArH-5), 7.63–7.59 (2 H, m, phenol H-4 and phenol H-5), 7.51 (1 H, app. t, <i>J</i> = 2.0, phenol H-2), 7.46 (1 H, app. t, <i>J</i> = 8.0, ArH-4), 7.11 (1 H, ddd, <i>J</i> = 8.2, 2.0, 1.0, phenol H-6), 1.02 (9 H, s, Si(CH<sub>3</sub>)<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>), 0.27 (6 H, s, Si(C<i>H</i><sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ (ppm) 182.8 (<i>C</i>HO), 174.4 (C-5), 158.5 (C-3), 156.6 (phenol C-3), 136.4 (ArC-2), 133.2 (ArC-3), 132.0 (q, <i>J</i> = 31.6, ArC-6), 131.3 (ArC-4), 130.6 (phenol C-5), 126.8 (ArC-1), 126.0 (phenol C-1), 125.0 (q, <i>J</i> = 5.1, ArC-5), 124.6 (phenol C-6), 123.0 (q, <i>J</i> = 274.5, <i>C</i>F<sub>3</sub>), 122.0 (phenol C-4), 120.3 (phenol C-2), 116.3 (C-4), 25.6 (Si(CH<sub>3</sub>)<sub>2</sub>C(<i>C</i>H<sub>3</sub>)<sub>3</sub>), 18.4 (Si(CH<sub>3</sub>)<sub>2</sub><i>C</i>(CH<sub>3</sub>)<sub>3</sub>), −4.2 (Si(<i>C</i>H<sub>3</sub>)<sub>2</sub>C(CH<sub>3</sub>)<sub>3</sub>); LC–MS (ESI): calcd for C<sub>23</sub>H<sub>24</sub>ClF<sub>3</sub>NO<sub>3</sub>Si [M + H]<sup>+</sup>: 482.11, observed: 482.17, LC <i>R</i><sub>t</sub>: 9.45 min.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68">4.4.  Biophysical Assays</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69">4.4.1.  RORγt-LBD Expression and Purification (Used for TR-FRET Assays)</h4><div class="NLM_p last">A pET15b expression vector encoding the human RORγt LBD (residues 265–518) with an <i>N</i>-terminal His<sub>6</sub>-tag was transformed by heat shock into BL21(DE3) <i>E. coli</i> cells. Single colonies were used to inoculate precultures of 8 mL LB-media containing 100 μg/mL ampicillin. After overnight incubation at 37 °C, each preculture was transferred to 1L TB media supplemented with ampicillin (100 μg/mL) and incubated at 37 °C until an OD<sub>600 nm</sub> = 1.0 was reached. Protein expression was then induced with 0.5 mM isopropyl-<i>b</i>-d-thiogalactoside (IPTG), and cultures were incubated for 16 h at 18°C. The cells were collected by centrifugation and suspended in lysis buffer (300 mM NaCl, 20 mM TrisHCl pH 8.0, 20 mM imidazole, 1 mM TCEP, 10% v/v glycerol, complete, EDTA-free Protease Inhibitor Cocktail tablets (1 tablet/50 mL lysate) and benzonase (0.1 μL/1 mL)). After lysis using a C3 Emulsiflex-C3 homogenizer (Avestin), the cell lysate was cleared by centrifugation at 4 °C and the protein was purified via Ni<sup>2+</sup> affinity column chromatography. Fractions containing the protein of interest were combined and dialyzed against 150 mM NaCl, 20 mM Tris HCl pH 8.0, 5 mM DTT, and 10% v/v glycerol.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70">4.4.2.  TR-FRET Coactivator Recruitment Assay</h4><div class="NLM_p last">Assays were conducted using 100 nM <i>N</i>-terminal biotinylated SRC-1 box2 peptide (Biotin-N-PSSHSSLTARHKILHRLLQEGSPSD-CONH<sub>2</sub>) and 20 nM His<sub>6</sub>-RORγt-LBD or 100 nM His<sub>6</sub>-PPARγ-LBD in buffer containing 10 mM HEPES, 150 mM NaCl, 5 mM DTT, 0.1% BSA (w/v), and 0.1 mM CHAPS, pH 7.5. A terbium labeled anti-His antibody (CisBio Bioassays, 61HISTLA) and D2-labeled streptavidin (CisBio Bioassays, 610SADLA) were used at the concentrations recommended by the supplier. In the case of PPARγ, the assay was performed in the presence of 1 μM rosiglitazone, in order to initially activate PPARγ. Compounds (dissolved in DMSO) were titrated using a 2× dilution series in Corning white low volume, low binding, 384-well plates at a final volume of 10 μL. The final DMSO concentration was 2% v/v throughout. The plate was incubated at room temperature for 30 min and centrifuged before reading (excitation = 340 nm; emission = 665 and 620 nm) on a Tecan infinite F500 plate reader using the parameters recommended by CisBio Bioassays. The data were analyzed with Origin Software. The dose–response curve was fitted represented by<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_m001.gif" alt="" id="_i71" /></img></span>where y is the FRET ratio, <i>A</i><sub>1</sub> is the bottom asymptote, <i>A</i><sub>2</sub> is the top asymptote, <i>p</i> is the Hills slope, and <i>x</i> is the ligand concentration. Where dose–response curves did not reach a bottom asymptote, this was fixed at the value of the negative control. (Data recorded in triplicate; error shown is standard deviation from the mean; curves are representative of >3 repeated experiments).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72">4.4.3.  Competition TR-FRET Coactivator Recruitment Assay</h4><div class="NLM_p last">Competition assays were performed in an analogous fashion to that described above only in the presence of fixed concentrations of cholesterol: 0 μM (DMSO), 0.25 μM, 1.0 μM, such that the final concentration of DMSO remained at 1.2% v/v.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73">4.4.4.  Ligand Binding TR-FRET Assay</h4><div class="NLM_p last">Assays were conducted using 100 nM Alexa647-labeled MRL-871 and 20 nM His<sub>6</sub>-RORγt-LBD in buffer as described above. A terbium-labeled anti-His antibody (CisBio Bioassays, 61HISTLA) was used at the concentrations recommended by the supplier. The assay was carried out in Corning black low volume, low binding, 384-well plates at a final volume of 10 μL in the same manner as described above.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74">4.5.  Protein X-ray Crystallography</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75">4.5.1.  RORγt-LBD Expression and Purification (Used for Crystallography)</h4><div class="NLM_p last">A pET15b expression vector was ordered from GenScript encoding for the RORγt LBD (residues 265–507) containing a C455H mutation (RORγtC455H) and a C-terminal His-tag. The plasmid was transformed by heat shock into BL21(DE3) <i>E. coli</i> cells. A single colony was used to start three precultures of 24 mL LB medium containing 100 μg/mL ampicillin. After overnight incubation at 37 °C, each preculture was transferred to 2 L of 2× YT medium supplied with 0.05% antifoam SE-15 (Sigma-Aldrich). These cultures were incubated until they reached an OD<sub>600</sub> = 0.6. Protein expression was induced by adding 0.25 mM IPTG. The temperature was decreased to 15 °C, and protein expression proceeded overnight. The cells were collected by centrifugation at 10.000 RCF for 10 min at 4 °C. The resulting 30 g of cell pellet was dissolved in lysis buffer (20 mM Tris, 500 mM NaCl, 2 mM TCEP, 0.1% Tween20, 10% glycerol, 10 cOmplete Protease Inhibitor Cocktail tablets (Roche), and 25 U/mL Bezonase Nuclease (Millipore), adjusted to pH = 8.0). After cell lysis using an Emulsiflex-C3 homogenizer (Avestin), the cell lysate was cleared by centrifugation at 40.000 RCF for 40 min at 4 °C, and the supernatant was loaded on a 5 mL Ni-NTA Superflow cartridge (QIAGEN) pre-equilibrated with buffer A (20 mM Tris, 500 mM NaCl, 2 mM TCEP, 0.1% Tween20, and 10% glycerol). The column was washed with 10 CVs of buffer A supplied with 20 mM and sequentially with 10 CVs of Buffer A supplied with 50 mM imidazole. The protein was eluted from the resin using an eight column volumes elution buffer (buffer A supplied with 200 mM imidazole). The purified protein was then dialyzed overnight to buffer A containing 1.2 U of restriction-grade thrombin (Millipore) per milligram of purified protein to remove the His-tag. Next, the protein mixture was concentrated using an Amicon Ultra centrifugal filter with a 10-kDa cutoff (Millipore) and loaded on a Superdex 75 pg 16/60 size-exclusion column (GE Life Sciences) using 20 mM Tris, 100 mM NaCl, and 5 mM DTT (adjusted to pH = 8.0) as a running buffer. The flow-through was collected as 3 mL fractions which were analyzed using Q-ToF LC–MS. The fractions containing RORγtC455H were combined and concentrated to a final concentration of 11.1 mg/mL. The concentrated protein sample was then aliquoted, flash-frozen, and stored at −80 °C.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76">4.5.2.  X-ray Crystallography</h4><div class="NLM_p last">The RORγtC455H solution (11.1 mg/mL) was mixed with 2 equiv of <b>25</b> and incubated on ice for 1 h. Next, the sample was centrifuged at 20.000 RCF for 20 min at 4 °C to remove precipitated proteins. MRC-2 well crystallization plates (Hampton Research, sitting drop) were prepared using a Mosquito pipetting robot (TTP Labtech). Well-diffracting crystals were obtained by mixing 0.9 ul of protein solution with 0.3 μL of 1.6–2.0 M ammonium sulfate and 0.1 M Tris (pH = 8.5). The well was filled with 80 μL precipitant solution, and plates were placed at 20 °C. Crystals could be observed after 1 h of incubation and grew to their final size overnight. The crystals were cryoprotected by transferring the crystals briefly to a solution containing 1.2 M AmSO4, 0.1 M Tris (pH = 8.5), and 25% glycerol before being flash cooled in liquid N<sub>2</sub>. Diffraction data were collected at 100 K at the P11 beamline of the PETRA III facility at DESY (Hamburg, Germany) and processed using the CCP4 suite (version 7.0.075).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> DIALS was used to integrate and scale the data.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The data was phased with PHASER using 5C4O as a molecular replacement model and ligand restraints of <b>25</b> were generated with AceDRG.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> Sequential model building and refinement were performed with COOT and REFMAC, respectively.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> PyMOL (version 2.2.3, Schrödinger) was used to make the figures.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The structure of RORγtC455 in complex with <b>25</b> was deposited in the protein data bank (PDB) under code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAL">6SAL</a>.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77">4.6.  Quantitative IL-17a mRNA RT-PCR Assay</h3><div class="NLM_p last">EL4 cells (Sigma-Aldrich) were grown in DMEM (Gibco) with 10% FBS. At 24 h after the cells were seeded onto a 12-well plate, the cells were incubated with 10 μM compound (from 10 mM stock in DMSO) or DMSO for 24 h and activated with phorbol 12-myristate 13-acetate (PMA, 50 ng/mL; Sigma-Aldrich) and ionomycin (1 μg/mL; Sigma-Aldrich) for 5 h. The cells were then collected, and RNA was isolated using a RNeasy Mini Kit (Qiagen) and reverse transcribed using the iScript Advanced cDNA Synthesis Kit (Bio-Rad). Quantitative RT-PCR was performed to analyze mRNA levels of mouse IL-17a levels (in triplicate) using SYBR green technology (Bio-Rad) on a CFX Real-Time System (Bio-Rad). The following primer assays were purchased from Bio-Rad: IL-17a (qMmuCID0026592) and Gapdh (qMmuCED0027497). The level of IL-17a mRNA expression was normalized to that of Gapdh expression. The relative gene expression was calculated by the 2<sup>–ΔΔCt</sup> (Livak) method using the DMSO control as calibrator. (Data recorded in triplicate; error shown is standard deviation from the mean; data are representative of >3 repeated experiments).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78">4.7.  Absorption, Distribution, Metabolism, and Excretion Experiments</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79">4.7.1.  Chemical Stability</h4><div class="NLM_p last">Chemical stability was determined by incubating test compounds at a final concentration of 2 μM in aqueous buffer at pH 7.4 for 1, 7, and 24 h, respectively. The percentage of remaining compound (% remain) in relation to the zero time point was calculated following LC–MS-based measurement of sample aliquots of each time point.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80">4.7.2.  Kinetic Solubility</h4><div class="NLM_p last">Aqueous solubility of compounds was determined by spectrophotometrical measurement of the kinetic solubility of a 500 μM compound solution in aqueous buffer pH 7.4 compared to a solution in the organic solvent acetonitrile after 90 min of vigorous shaking at room temperature.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81">4.7.3.  PAMPA</h4><div class="NLM_p last">Permeability through artificial membranes (PAMPA) was performed at an initial concentration of 500 μM of the compound in the donor compartment. After an incubation period of 20 h, absorption of the receiver wells was measured by spectrophotometry and permeation was calculated by normalization of the compound flux across a blank filter.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82">4.7.4.  Microsomal Stability Phase I</h4><div class="NLM_p last">Metabolic stability under oxidative conditions was measured in liver microsomes from different species by LC–MS-based measuring of depletion of compound at a concentration of 3 μM over time up to 50 min at 37 °C. On the basis of compound half-life <i>t</i><sub>1/2</sub>, in vitro intrinsic clearance CL<sub>int</sub> was calculated. <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_m002.gif" alt="" id="_i83" /></img></span></div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84">4.7.5.  Microsomal Stability Phase II</h4><div class="NLM_p last">Metabolic stability under conjugative conditions was measured in the glucuronidation assay by LC–MS-based determination of % remaining of selected compounds at a concentration of 5 μM in incubations with liver microsomes supplemented with UDPGA for 1 h at 37 °C.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85">4.7.6.  Plasma Stability</h4><div class="NLM_p last">Plasma stability was measured by LC–MS-based determination of % remaining of selected compounds at a concentration of 5 μM after incubation in 100% plasma obtained from different species for 1 h at 37 °C.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86">4.7.7.  Plasma Protein Binding</h4><div class="NLM_p last">Assessment of plasma protein binding was measured by equilibrium dialysis by incubating plasma with the compound of interest at a concentration of 5 μM for 6 h at 37 °C followed by LC–MS-based determination of final compound concentrations.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01372" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01372?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01372</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">S1.0. In silico experiments: phase pharmacophore screen scores (Table S1 and S2), Chem-T C-5 virtual library SMILES strings (Table S3), manually generated C-5 virtual library (Figure S1), and glide docking scores (Tables S4 and S5); S2.0. Chemistry Supporting Information and Spectra: (S2.1) Regiochemical assignment of <b>30a</b>, (S2.2) synthesis of <b>5</b> (Glenmark) and (S2.3) NMR spectra and LC-UV traces for assayed compounds; and S3.0. Crystallography: data collection and refinement statistics (molecular replacement) of <b>25</b> (Table S6), two-dimensional protein–ligand interaction plot for <b>25</b> (Figure S2), and overlay of crystal structure and docking pose for <b>25</b> (Figure S3) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Recommended compound characterization checklist (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_002.xls" class="ext-link">XLS</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_004.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_001.pdf">jm9b01372_si_001.pdf (4.97 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_002.xls">jm9b01372_si_002.xls (102.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_003.csv">jm9b01372_si_003.csv (1.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_004.xlsx">jm9b01372_si_004.xlsx (11.33 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the RORγt bound to compound <b>25</b> have been deposited in the Protein Data Bank under accession code 6SAL. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01372" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74120" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74120" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luc Brunsveld</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5675-511X" title="Orcid link">http://orcid.org/0000-0001-5675-511X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbd795d9c9ced5c8cdded7dffbcfcede95d5d7"><span class="__cf_email__" data-cfemail="016d2f6373746f7277646d65417574642f6f6d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Femke A. Meijer</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4412-9968" title="Orcid link">http://orcid.org/0000-0003-4412-9968</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard G. Doveston</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Leicester
Institute of Structural and Chemical Biology and Department of Chemistry, University of Leicester, University Road, Leicester LE1 7RH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rens M.J.M. de Vries</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gaël M. Vos</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex A.A. Vos</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Seppe Leysen</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcel Scheepstra</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Ottmann</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7315-0315" title="Orcid link">http://orcid.org/0000-0001-7315-0315</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lech-Gustav Milroy</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Chemical
Biology, Department of Biomedical Engineering and Institute for Complex
Molecular Systems, Technische Universiteit
Eindhoven, Den Dolech 2, 5612 AZ Eindhoven, The Netherlands</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.A.M. and R.G.D. contributed equally to this work. The manuscript was written through contributions of all authors. R.G.D. and S.L. performed in silico work; R.G.D., F.A.M., G.M.V., and A.A.A.V. performed synthesis and biochemical studies; F.A.M. performed RT-PCR experiments; R.M.J.M.V. performed protein expression and crystallization studies; R.G.D., F.A.M., R.M.J.M.V., S.L., M.S., C.O., L.G.M., and L.B. designed the studies. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by The Netherlands Organization for Scientific Research through Gravity program 024.001.035, VICI grant 016.150.366, ECHO grant 711.018.003, and the European Union through a MSCA Individual Fellowship (R.G.D., H2020-MSCA-IEF-2016, grant no. 705188).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7601-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i90">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51576" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Rowin J.P.M. de Visser for synthesis of the AlexaFluor647-labelled MRL-871 probe, Guido J.M. Oerlemans and Maxime C.M. van den Oetelaar for synthesis optimization, Iris A. Leijten-van de Gevel for expression of the RORγt protein (used for TR-FRET assays), and Joost L.J. van Dongen for performing HRMS measurements. We also thank the Lead Discovery Center in Dortmund (Matthias Baumann, Jan Eickhoff, and Bert Klebl) for performing ADME measurements. Finally, we thank the DLS-CCP4 Data Collection and Structure Solution Workshop 2017 at Diamond Light Source (Oxfordshire, UK).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">DME</td><td class="NLM_def"><p class="first last">1,2-dimethoxyethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMP</td><td class="NLM_def"><p class="first last">Dess-Martin periodinane</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphosphoryl azide</p></td></tr><tr><td class="NLM_term">H12</td><td class="NLM_def"><p class="first last">helix 12</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl-<i>b</i>-d-thiogalactoside</p></td></tr><tr><td class="NLM_term">mAb</td><td class="NLM_def"><p class="first last">monoclonal antibody</p></td></tr><tr><td class="NLM_term">NBS</td><td class="NLM_def"><p class="first last"><i>N</i>-bromosuccinimide</p></td></tr><tr><td class="NLM_term">NCS</td><td class="NLM_def"><p class="first last"><i>N-</i>chlorosuccinimide</p></td></tr><tr><td class="NLM_term">NR</td><td class="NLM_def"><p class="first last">nuclear receptor</p></td></tr><tr><td class="NLM_term">Q-tof</td><td class="NLM_def"><p class="first last">quadrupole time-of-flight</p></td></tr><tr><td class="NLM_term">RORγt</td><td class="NLM_def"><p class="first last">retinoic acid receptor-related orphan receptor γ t</p></td></tr><tr><td class="NLM_term">RT-PCR</td><td class="NLM_def"><p class="first last">real time PCR</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">Th17</td><td class="NLM_def"><p class="first last">T helper 17</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved FRET</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Yong</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span> <span> </span><span class="NLM_article-title">ROR-γ drives androgen receptor expression and represents a therapeutic target in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/nm.4070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnm.4070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27019329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFShurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=488-496&author=J.+Wangauthor=J.+X.+Zouauthor=X.+Xueauthor=D.+Caiauthor=Y.+Zhangauthor=Z.+Duanauthor=Q.+Xiangauthor=J.+C.+Yangauthor=M.+C.+Louieauthor=A.+D.+Borowskyauthor=A.+Gaoauthor=C.+P.+Evansauthor=K.+S.+Lamauthor=J.+Xuauthor=H.-J.+Kungauthor=R.+M.+Evansauthor=Yong+Xuauthor=H.-W.+Chen&title=ROR-%CE%B3+drives+androgen+receptor+expression+and+represents+a+therapeutic+target+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.4070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">ROR-γ drives androgen receptor expression and represents a therapeutic target in castration-resistant prostate cancer</span></div><div class="casAuthors">Wang, Junjian; Zou, June X.; Xue, Xiaoqian; Cai, Demin; Zhang, Yan; Duan, Zhijian; Xiang, Qiuping; Yang, Joy C.; Louie, Maggie C.; Borowsky, Alexander D.; Gao, Allen C.; Evans, Christopher P.; Lam, Kit S.; Xu, Jianzhen; Kung, Hsing-Jien; Evans, Ronald M.; Xu, Yong; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-496</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is overexpressed and hyperactivated in human castration-resistant prostate cancer (CRPC).  However, the determinants of AR overexpression in CRPC are poorly defined.  Here we show that retinoic acid receptor-related orphan receptor γ (ROR-γ) is overexpressed and amplified in metastatic CRPC tumors, and that ROR-γ drives AR expression in the tumors.  ROR-γ recruits nuclear receptor coactivator 1 and 3 (NCOA1 and NCOA3, also known as SRC-1 and SRC-3) to an AR-ROR response element (RORE) to stimulate AR gene transcription.  ROR-γ antagonists suppress the expression of both AR and its variant AR-V7 in prostate cancer (PCa) cell lines and tumors.  ROR-γ antagonists also markedly diminish genome-wide AR binding, H3K27ac abundance and expression of the AR target gene network.  Finally, ROR-γ antagonists suppressed tumor growth in multiple AR-expressing, but not AR-neg., xenograft PCa models, and they effectively sensitized CRPC tumors to enzalutamide, without overt toxicity, in mice.  Taken together, these results establish ROR-γ as a key player in CRPC by acting upstream of AR and as a potential therapeutic target for advanced PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX9ywlnlPbX7Vg90H21EOLACvtfcHk0lilYeKtSS99_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFShurY%253D&md5=3817be87b91c2f148987d35e2305365c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnm.4070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4070%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DLouie%26aufirst%3DM.%2BC.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DEvans%26aufirst%3DC.%2BP.%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKung%26aufirst%3DH.-J.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DXu%26aufirst%3DYong%26aulast%3DChen%26aufirst%3DH.-W.%26atitle%3DROR-%25CE%25B3%2520drives%2520androgen%2520receptor%2520expression%2520and%2520represents%2520a%2520therapeutic%2520target%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D488%26epage%3D496%26doi%3D10.1038%2Fnm.4070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loomes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of XY101, a potent, selective, and orally bioavailable RORγ inverse agonist for treatment of castration-resistant prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4716</span>– <span class="NLM_lpage">4730</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00327</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00327" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4716-4730&author=Y.+Zhangauthor=X.+Wuauthor=X.+Xueauthor=C.+Liauthor=J.+Wangauthor=R.+Wangauthor=C.+Zhangauthor=C.+Wangauthor=Y.+Shiauthor=L.+Zouauthor=Q.+Liauthor=Z.+Huangauthor=X.+Haoauthor=K.+Loomesauthor=D.+Wuauthor=H.-W.+Chenauthor=J.+Xuauthor=Y.+Xu&title=Discovery+and+characterization+of+XY101%2C+a+potent%2C+selective%2C+and+orally+bioavailable+ROR%CE%B3+inverse+agonist+for+treatment+of+castration-resistant+prostate+cancer&doi=10.1021%2Facs.jmedchem.9b00327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of XY101, a potent, selective, and orally bioavailable RORγ inverse agonist for treatment of castration-resistant prostate cancer</span></div><div class="casAuthors">Zhang, Yan; Wu, Xishan; Xue, Xiaoqian; Li, Chenchang; Wang, Junjian; Wang, Rui; Zhang, Cheng; Wang, Chao; Shi, Yudan; Zou, Lingjiao; Li, Qiu; Huang, Zenghong; Hao, Xiaojuan; Loomes, Kerry; Wu, Donghai; Chen, Hong-Wu; Xu, Jinxin; Xu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4716-4730</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the design, optimization, and biol. evaluation of nuclear receptor RORγ inverse agonists as therapeutic agents for prostate cancer treatment.  The most potent compd. 27(I) (designated as XY101) exhibited cellular activity with an IC50 value of 30 nM in a cell-based reporter gene assay with good selectivity against other nuclear receptor subtypes.  The cocrystal structure of I in complex with the RORγ ligand binding domain provided a solid structural basis for its antagonistic mechanism.  I potently inhibited cell growth, colony formation, and the expression of AR, AR-V7, and PSA.  I also exhibited good metabolic stability and a pharmacokinetic profile with oral bioavailability of 59% and a half-life of 7.3 h.  Notably, I demonstrated promising therapeutic effects with significant tumor growth inhibition in a prostate cancer xenograft model in mice.  The potent, selective, metabolically stable, and orally available RORγ inverse agonists represent a new class of compds. as potential therapeutics against prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUiAUjMO2c7Vg90H21EOLACvtfcHk0ljFLngLj6jh2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFSjtrw%253D&md5=5332cfeb5f08b7c695361af45ae68c60</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00327%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DLoomes%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DH.-W.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520XY101%252C%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520ROR%25CE%25B3%2520inverse%2520agonist%2520for%2520treatment%2520of%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4716%26epage%3D4730%26doi%3D10.1021%2Facs.jmedchem.9b00327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepelley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.cell.2006.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=16990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=1121-1133&author=I.+I.+Ivanovauthor=B.+S.+McKenzieauthor=L.+Zhouauthor=C.+E.+Tadokoroauthor=A.+Lepelleyauthor=J.+J.+Lafailleauthor=D.+J.+Cuaauthor=D.+R.+Littman&title=The+orphan+nuclear+receptor+ROR%CE%B3t+directs+the+differentiation+program+of+proinflammatory+IL-17%2B+T+helper+cells&doi=10.1016%2Fj.cell.2006.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span></div><div class="casAuthors">Ivanov, Ivaylo I.; McKenzie, Brent S.; Zhou, Liang; Tadokoro, Carlos E.; Lepelley, Alice; Lafaille, Juan J.; Cua, Daniel J.; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17-producing T lymphocytes have been recently shown to comprise a distinct lineage of proinflammatory T helper cells, termed Th17 cells, that are major contributors to autoimmune disease.  The authors show here that the orphan nuclear receptor RORγt is the key transcription factor that orchestrates the differentiation of this effector cell lineage.  RORγt induces transcription of the genes encoding IL-17 and the related cytokine IL-17F in naive CD4+ T helper cells and is required for their expression in response to IL-6 and TGF-β, the cytokines known to induce IL-17.  Th17 cells are constitutively present throughout the intestinal lamina propria, express RORγt, and are absent in mice deficient for RORγt or IL-6.  Mice with RORγt-deficient T cells have attenuated autoimmune disease and lack tissue-infiltrating Th17 cells.  Together, these studies suggest that RORγt is a key regulator of immune homeostasis and highlight its potential as a therapeutic target in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsJd6RyFqWrVg90H21EOLACvtfcHk0ljFLngLj6jh2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE&md5=4f5693603c30dca4f6a2d0994aeaf9b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DI.%2BI.%26aulast%3DMcKenzie%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTadokoro%26aufirst%3DC.%2BE.%26aulast%3DLepelley%26aufirst%3DA.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520orphan%2520nuclear%2520receptor%2520ROR%25CE%25B3t%2520directs%2520the%2520differentiation%2520program%2520of%2520proinflammatory%2520IL-17%252B%2520T%2520helper%2520cells%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D1121%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2006.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unutmaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The differentiation of human TH-17 cells requires transforming growth factor-β and induction of the nuclear receptor RORγt</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/ni.1610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fni.1610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=18454151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVWqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=641-649&author=N.+Manelauthor=D.+Unutmazauthor=D.+R.+Littman&title=The+differentiation+of+human+TH-17+cells+requires+transforming+growth+factor-%CE%B2+and+induction+of+the+nuclear+receptor+ROR%CE%B3t&doi=10.1038%2Fni.1610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The differentiation of human TH-17 cells requires transforming growth factor-β and induction of the nuclear receptor RORγt</span></div><div class="casAuthors">Manel, Nicolas; Unutmaz, Derya; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TH-17 cells are interleukin 17 (IL-17)-secreting CD4+ T helper cells involved in autoimmune disease and mucosal immunity.  In naive CD4+ T cells from mice, IL-17 is expressed in response to a combination of IL-6 or IL-21 and transforming growth factor-β (TGF-β) and requires induction of the nuclear receptor RORγt.  It has been suggested that the differentiation of human TH-17 cells is independent of TGF-β and thus differs fundamentally from that in mice.  The authors show here that TGF-β, IL-1β and IL-6, IL-21 or IL-23 in serum-free conditions were necessary and sufficient to induce IL-17 expression in naive human CD4+ T cells from cord blood.  TGF-β upregulated RORγt expression but simultaneously inhibited its ability to induce IL-17 expression.  Inflammatory cytokines relieved this inhibition and increased RORγt-directed IL-17 expression.  Other gene products detected in TH-17 cells after RORγt induction included the chemokine receptor CCR6, the IL-23 receptor, IL-17F and IL-26.  The authors' studies identify RORγt as having a central function in the differentiation of human TH-17 cells from naive CD4+ T cells and suggest that similar cytokine pathways are involved in this process in mice and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZBHk62Mj3R7Vg90H21EOLACvtfcHk0lgEeHzHjUt0dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVWqurg%253D&md5=9c1406dfd60902657b1bd5231618cc70</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fni.1610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1610%26sid%3Dliteratum%253Aachs%26aulast%3DManel%26aufirst%3DN.%26aulast%3DUnutmaz%26aufirst%3DD.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520differentiation%2520of%2520human%2520TH-17%2520cells%2520requires%2520transforming%2520growth%2520factor-%25CE%25B2%2520and%2520induction%2520of%2520the%2520nuclear%2520receptor%2520ROR%25CE%25B3t%26jtitle%3DNat.%2520Immunol.%26date%3D2008%26volume%3D9%26spage%3D641%26epage%3D649%26doi%3D10.1038%2Fni.1610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akimzhanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panopoulos, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluns, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2007.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.immuni.2007.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=18164222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=29-39&author=X.+O.+Yangauthor=B.+P.+Pappuauthor=R.+Nurievaauthor=A.+Akimzhanovauthor=H.+S.+Kangauthor=Y.+Chungauthor=L.+Maauthor=B.+Shahauthor=A.+D.+Panopoulosauthor=K.+S.+Schlunsauthor=S.+S.+Watowichauthor=Q.+Tianauthor=A.+M.+Jettenauthor=C.+Dong&title=T+helper+17+lineage+differentiation+is+programmed+by+orphan+nuclear+receptors+ROR%CE%B1+and+ROR%CE%B3&doi=10.1016%2Fj.immuni.2007.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span></div><div class="casAuthors">Yang, Xuexian O.; Pappu, Bhanu P.; Nurieva, Roza; Akimzhanov, Askar; Kang, Hong Soon; Chung, Yeonseok; Ma, Li; Shah, Bhavin; Panopoulos, Athanasia D.; Schluns, Kimberly S.; Watowich, Stephanie S.; Tian, Qiang; Jetten, Anton M.; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T cell functional differentiation is mediated by lineage-specific transcription factors.  T helper 17 (Th17) has been recently identified as a distinct Th lineage mediating tissue inflammation.  Retinoic acid receptor-related orphan receptor γ (RORγ) was shown to regulate Th17 differentiation; RORγ deficiency, however, did not completely abolish Th17 cytokine expression.  Here, we report Th17 cells highly expressed another related nuclear receptor, RORα, induced by transforming growth factor-β and interleukin-6 (IL-6), which is dependent on signal transducer and activator of transcription 3.  Overexpression of RORα promoted Th17 differentiation, possibly through the conserved noncoding sequence 2 in II17-II17f locus.  RORα deficiency resulted in reduced IL-17 expression in vitro and in vivo.  Furthermore, RORα and RORγ coexpression synergistically led to greater Th17 differentiation.  Double deficiencies in RORα and RORγ globally impaired Th17 generation and completely protected mice against exptl. autoimmune encephalomyelitis.  Therefore, Th17 differentiation is directed by two lineage-specific nuclear receptors, RORα and RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBQRyY1XQpJbVg90H21EOLACvtfcHk0lgEeHzHjUt0dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D&md5=436602649095b96d366c983addf526f7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2007.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2007.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DPappu%26aufirst%3DB.%2BP.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DAkimzhanov%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%2BS.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPanopoulos%26aufirst%3DA.%2BD.%26aulast%3DSchluns%26aufirst%3DK.%2BS.%26aulast%3DWatowich%26aufirst%3DS.%2BS.%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DJetten%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DT%2520helper%252017%2520lineage%2520differentiation%2520is%2520programmed%2520by%2520orphan%2520nuclear%2520receptors%2520ROR%25CE%25B1%2520and%2520ROR%25CE%25B3%26jtitle%3DImmunity%26date%3D2008%26volume%3D28%26spage%3D29%26epage%3D39%26doi%3D10.1016%2Fj.immuni.2007.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miossec, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolls, J. K.</span></span> <span> </span><span class="NLM_article-title">Targeting IL-17 and TH17 cells in chronic inflammation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">763</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1038/nrd3794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd3794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=23023676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqsb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=763-776&author=P.+Miossecauthor=J.+K.+Kolls&title=Targeting+IL-17+and+TH17+cells+in+chronic+inflammation&doi=10.1038%2Fnrd3794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting IL-17 and TH17 cells in chronic inflammation</span></div><div class="casAuthors">Miossec, Pierre; Kolls, Jay K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">763-776</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The key role of interleukin-17 (IL-17) and T helper 17 (TH17) cells in tissue inflammation, autoimmunity and host defense led to the exptl. targeting of these mols. in mouse models of diseases as well as in clin. settings.  Moreover, the demonstration that IL-17 and TH17 cells contribute to local and systemic aspects of disease pathogenesis, as well as the finding that the IL-17-TH17 cell pathway is regulated by IL-23, prompted the identification of inhibitors.  These inhibitors include biotechnol. products that target IL-23 as well as the leading member of the IL-17 family, IL-17A, and one of its receptors, IL-17 receptor A.  Several clin. trials of these inhibitors are underway, and pos. results have been obtained in psoriasis, rheumatoid arthritis and ankylosing spondylitis.  This Review focuses on the current knowledge of the IL-17-TH17 cell pathway to better understand the pos. as well as potential neg. consequences of targeting them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLAiqCF66JHrVg90H21EOLACvtfcHk0lgEeHzHjUt0dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqsb7L&md5=0627e4086ae6e50881cc3de52f1a2752</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd3794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3794%26sid%3Dliteratum%253Aachs%26aulast%3DMiossec%26aufirst%3DP.%26aulast%3DKolls%26aufirst%3DJ.%2BK.%26atitle%3DTargeting%2520IL-17%2520and%2520TH17%2520cells%2520in%2520chronic%2520inflammation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D763%26epage%3D776%26doi%3D10.1038%2Fnrd3794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burkett, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">IL-17 blockade in psoriasis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1669</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.cell.2016.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27984714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWlsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1669&author=P.+R.+Burkettauthor=V.+K.+Kuchroo&title=IL-17+blockade+in+psoriasis&doi=10.1016%2Fj.cell.2016.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17 Blockade in Psoriasis</span></div><div class="casAuthors">Burkett, Patrick R.; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1669</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17A both directly induces and synergizes with other cytokines to promote autoimmune tissue inflammation.  Secukinumab and ixekizumab are monoclonal antibodies (mAb) that inhibit interleukin-17A.  These two agents were recently approved for treatment of psoriasis, and secukinumab is also approved for treatment of two spondyloarthropathies, psoriatic arthritis and ankylosing spondylitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobNyijIUaJ-rVg90H21EOLACvtfcHk0liW67izsQDoBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWlsrrE&md5=e2d2918a30d5bf98cd5b405e51f171d4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DBurkett%26aufirst%3DP.%2BR.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DIL-17%2520blockade%2520in%2520psoriasis%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1669%26doi%3D10.1016%2Fj.cell.2016.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendolan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer-Gould, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strober, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannella, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klonowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oksenberg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raine, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinman, L.</span></span> <span> </span><span class="NLM_article-title">Gene-microarray analysis of multiple sclerosis lesions yields new targets validated in autoimmune encephalomyelitis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nm0502-500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnm0502-500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=11984595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1Kgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=500-508&author=C.+Lockauthor=G.+Hermansauthor=R.+Pedottiauthor=A.+Brendolanauthor=E.+Schadtauthor=H.+Garrenauthor=A.+Langer-Gouldauthor=S.+Stroberauthor=B.+Cannellaauthor=J.+Allardauthor=P.+Klonowskiauthor=A.+Austinauthor=N.+Ladauthor=N.+Kaminskiauthor=S.+J.+Galliauthor=J.+R.+Oksenbergauthor=C.+S.+Raineauthor=R.+Hellerauthor=L.+Steinman&title=Gene-microarray+analysis+of+multiple+sclerosis+lesions+yields+new+targets+validated+in+autoimmune+encephalomyelitis&doi=10.1038%2Fnm0502-500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Gene-microarray analysis of multiple sclerosis lesions yields new targets validated in autoimmune encephalomyelitis</span></div><div class="casAuthors">Lock, C.; Hermans, G.; Pedotti, R.; Brendolan, A.; Schadt, E.; Garren, H.; Langer-Gould, A.; Strober, S.; Cannella, B.; Allard, J.; Klonowski, P.; Austin, A.; Lad, N.; Kaminski, N.; Galli, S. J.; Oksenberg, J. R.; Raine, C. S.; Heller, R.; Steinman, L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">500-508</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Microarray anal. of multiple sclerosis (MS) lesions obtained at autopsy revealed increased transcripts of genes encoding inflammatory cytokines, particularly interleukin-6 and -17, interferon-γ and assocd. downstream pathways.  Comparison of two poles of MS pathol.-acute lesions with inflammation vs. 'silent' lesions without inflammation-revealed differentially transcribed genes.  Some products of these genes were chosen as targets for therapy of exptl. autoimmune encephalomyelitis (EAE) in mice.  Granulocyte colony-stimulating factor is upregulated in acute, but not in chronic, MS lesions, and the effect on ameliorating EAE is more pronounced in the acute phase, in contrast to knocking out the Ig Fc receptor common γ chain where the effect is greatest on chronic disease.  These results in EAE corroborate the microarray studies on MS lesions.  Large-scale anal. of transcripts in MS lesions elucidates new aspects of pathol. and opens possibilities for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPWj6FNx8JZ7Vg90H21EOLACvtfcHk0liW67izsQDoBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1Kgur8%253D&md5=1e50114f238ec03ff0d534966273f245</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnm0502-500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0502-500%26sid%3Dliteratum%253Aachs%26aulast%3DLock%26aufirst%3DC.%26aulast%3DHermans%26aufirst%3DG.%26aulast%3DPedotti%26aufirst%3DR.%26aulast%3DBrendolan%26aufirst%3DA.%26aulast%3DSchadt%26aufirst%3DE.%26aulast%3DGarren%26aufirst%3DH.%26aulast%3DLanger-Gould%26aufirst%3DA.%26aulast%3DStrober%26aufirst%3DS.%26aulast%3DCannella%26aufirst%3DB.%26aulast%3DAllard%26aufirst%3DJ.%26aulast%3DKlonowski%26aufirst%3DP.%26aulast%3DAustin%26aufirst%3DA.%26aulast%3DLad%26aufirst%3DN.%26aulast%3DKaminski%26aufirst%3DN.%26aulast%3DGalli%26aufirst%3DS.%2BJ.%26aulast%3DOksenberg%26aufirst%3DJ.%2BR.%26aulast%3DRaine%26aufirst%3DC.%2BS.%26aulast%3DHeller%26aufirst%3DR.%26aulast%3DSteinman%26aufirst%3DL.%26atitle%3DGene-microarray%2520analysis%2520of%2520multiple%2520sclerosis%2520lesions%2520yields%2520new%2520targets%2520validated%2520in%2520autoimmune%2520encephalomyelitis%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D500%26epage%3D508%26doi%3D10.1038%2Fnm0502-500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duerr, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brant, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverberg, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhart, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regueiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassopoulos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Targan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kistner, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumm, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregersen, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolae, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span> <span> </span><span class="NLM_article-title">A genome-wide association study identifies IL23R as an inflammatory bowel disease gene</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>314</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">3</span>, <span class="refDoi"> DOI: 10.1126/science.1135245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1126%2Fscience.1135245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17068223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CntrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=2006&pages=1461-3&author=R.+H.+Duerrauthor=K.+D.+Taylorauthor=S.+R.+Brantauthor=J.+D.+Riouxauthor=M.+S.+Silverbergauthor=M.+J.+Dalyauthor=A.+H.+Steinhartauthor=C.+Abrahamauthor=M.+Regueiroauthor=A.+Griffithsauthor=T.+Dassopoulosauthor=A.+Bittonauthor=H.+Yangauthor=S.+Targanauthor=L.+W.+Dattaauthor=E.+O.+Kistnerauthor=L.+P.+Schummauthor=A.+T.+Leeauthor=P.+K.+Gregersenauthor=M.+M.+Barmadaauthor=J.+I.+Rotterauthor=D.+L.+Nicolaeauthor=J.+H.+Cho&title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&doi=10.1126%2Fscience.1135245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Genome-Wide Association Study Identifies IL23R as an Inflammatory Bowel Disease Gene</span></div><div class="casAuthors">Duerr, Richard H.; Taylor, Kent D.; Brant, Steven R.; Rioux, John D.; Silverberg, Mark S.; Daly, Mark J.; Steinhart, A. Hillary; Abraham, Clara; Regueiro, Miguel; Griffiths, Anne; Dassopoulos, Themistocles; Bitton, Alain; Yang, Huiying; Targan, Stephan; Datta, Lisa Wu; Kistner, Emily O.; Schumm, L. Philip; Lee, Annette T.; Gregersen, Peter K.; Barmada, M. Michael; Rotter, Jerome I.; Nicolae, Dan L.; Cho, Judy H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">5804</span>),
    <span class="NLM_cas:pages">1461-1463</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inflammatory bowel diseases Crohn's disease and ulcerative colitis are common, chronic disorders that cause abdominal pain, diarrhea, and gastrointestinal bleeding.  To identify genetic factors that might contribute to these disorders, we performed a genome-wide assocn. study.  We found a highly significant assocn. between Crohn's disease and the IL23R gene on chromosome 1p31, which encodes a subunit of the receptor for the pro-inflammatory cytokine interleukin-23.  An uncommon coding variant (rs11209026, c.1142G>A, p.Arg381Gln) confers strong protection against Crohn's disease, and addnl. noncoding IL23R variants are independently assocd.  Replication studies confirmed IL23R assocns. in independent cohorts of patients with Crohn's disease or ulcerative colitis.  These results and previous studies on the pro-inflammatory role of IL-23 prioritize this signaling pathway as a therapeutic target in inflammatory bowel disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruSERfyA1l3rVg90H21EOLACvtfcHk0liW67izsQDoBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CntrjF&md5=3d95555a1772c6f89dc4171f12105242</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1135245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1135245%26sid%3Dliteratum%253Aachs%26aulast%3DDuerr%26aufirst%3DR.%2BH.%26aulast%3DTaylor%26aufirst%3DK.%2BD.%26aulast%3DBrant%26aufirst%3DS.%2BR.%26aulast%3DRioux%26aufirst%3DJ.%2BD.%26aulast%3DSilverberg%26aufirst%3DM.%2BS.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DSteinhart%26aufirst%3DA.%2BH.%26aulast%3DAbraham%26aufirst%3DC.%26aulast%3DRegueiro%26aufirst%3DM.%26aulast%3DGriffiths%26aufirst%3DA.%26aulast%3DDassopoulos%26aufirst%3DT.%26aulast%3DBitton%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DTargan%26aufirst%3DS.%26aulast%3DDatta%26aufirst%3DL.%2BW.%26aulast%3DKistner%26aufirst%3DE.%2BO.%26aulast%3DSchumm%26aufirst%3DL.%2BP.%26aulast%3DLee%26aufirst%3DA.%2BT.%26aulast%3DGregersen%26aufirst%3DP.%2BK.%26aulast%3DBarmada%26aufirst%3DM.%2BM.%26aulast%3DRotter%26aufirst%3DJ.%2BI.%26aulast%3DNicolae%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26atitle%3DA%2520genome-wide%2520association%2520study%2520identifies%2520IL23R%2520as%2520an%2520inflammatory%2520bowel%2520disease%2520gene%26jtitle%3DScience%26date%3D2006%26volume%3D314%26spage%3D1461%26epage%3D3%26doi%3D10.1126%2Fscience.1135245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hueber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koroleva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draelos, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M. H.</span></span>; <span class="NLM_contrib-group">Psoriasis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Reino, J. J.</span></span>; <span class="NLM_contrib-group">Rheumatoid Arthritis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falk, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. D.</span></span>; <span class="NLM_contrib-group">Uveitis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Padova, F.</span></span> <span> </span><span class="NLM_article-title">Effects of AIN457, a fully human antibody to interleukin-17A, on psoriasis, rheumatoid arthritis, and uveitis</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">52ra72</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.3001107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1126%2Fscitranslmed.3001107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=20926833" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=52ra72&author=W.+Hueberauthor=D.+D.+Patelauthor=T.+Dryjaauthor=A.+M.+Wrightauthor=I.+Korolevaauthor=G.+Bruinauthor=C.+Antoniauthor=Z.+Draelosauthor=M.+H.+Goldauthor=Psoriasis+Study+Groupauthor=P.+Durezauthor=P.+P.+Takauthor=J.+J.+Gomez-Reinoauthor=Rheumatoid+Arthritis+Study+Groupauthor=C.+S.+Fosterauthor=R.+Y.+Kimauthor=C.+M.+Samsonauthor=N.+S.+Falkauthor=D.+S.+Chuauthor=D.+Callananauthor=Q.+D.+Nguyenauthor=Uveitis+Study+Groupauthor=K.+Roseauthor=A.+Haiderauthor=F.+Di+Padova&title=Effects+of+AIN457%2C+a+fully+human+antibody+to+interleukin-17A%2C+on+psoriasis%2C+rheumatoid+arthritis%2C+and+uveitis&doi=10.1126%2Fscitranslmed.3001107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001107%26sid%3Dliteratum%253Aachs%26aulast%3DHueber%26aufirst%3DW.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DDryja%26aufirst%3DT.%26aulast%3DWright%26aufirst%3DA.%2BM.%26aulast%3DKoroleva%26aufirst%3DI.%26aulast%3DBruin%26aufirst%3DG.%26aulast%3DAntoni%26aufirst%3DC.%26aulast%3DDraelos%26aufirst%3DZ.%26aulast%3DGold%26aufirst%3DM.%2BH.%26aulast%3D%26aulast%3DDurez%26aufirst%3DP.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DGomez-Reino%26aufirst%3DJ.%2BJ.%26aulast%3D%26aulast%3DFoster%26aufirst%3DC.%2BS.%26aulast%3DKim%26aufirst%3DR.%2BY.%26aulast%3DSamson%26aufirst%3DC.%2BM.%26aulast%3DFalk%26aufirst%3DN.%2BS.%26aulast%3DChu%26aufirst%3DD.%2BS.%26aulast%3DCallanan%26aufirst%3DD.%26aulast%3DNguyen%26aufirst%3DQ.%2BD.%26aulast%3D%26aulast%3DRose%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DA.%26aulast%3DDi%2BPadova%26aufirst%3DF.%26atitle%3DEffects%2520of%2520AIN457%252C%2520a%2520fully%2520human%2520antibody%2520to%2520interleukin-17A%252C%2520on%2520psoriasis%252C%2520rheumatoid%2520arthritis%252C%2520and%2520uveitis%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2010%26volume%3D2%26spage%3D52ra72%26doi%3D10.1126%2Fscitranslmed.3001107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Attia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abushouk, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadelkarim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgebaly, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Daim, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negida, A.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy of brodalumab for moderate-to-severe plaque psoriasis: A systematic review and meta-analysis</span>. <i>Clin. Drug Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1007/s40261-017-0500-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1007%2Fs40261-017-0500-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28197901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVaqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=439-451&author=A.+Attiaauthor=A.+I.+Abushoukauthor=H.+Ahmedauthor=M.+Gadelkarimauthor=A.+Elgebalyauthor=Z.+Hassanauthor=M.+M.+Abdel-Daimauthor=A.+Negida&title=Safety+and+efficacy+of+brodalumab+for+moderate-to-severe+plaque+psoriasis%3A+A+systematic+review+and+meta-analysis&doi=10.1007%2Fs40261-017-0500-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Safety and Efficacy of Brodalumab for Moderate-to-Severe Plaque Psoriasis: A Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Attia, Attia; Abushouk, Abdelrahman Ibrahim; Ahmed, Hussien; Gadelkarim, Mohamed; Elgebaly, Ahmed; Hassan, Zeinab; Abdel-Daim, Mohamed M.; Negida, Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Drug Investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">439-451</span>CODEN:
                <span class="NLM_cas:coden">CDINFR</span>;
        ISSN:<span class="NLM_cas:issn">1173-2563</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Background: Psoriasis is an inflammatory skin disease that affects 2-3% of the worldwide population.  The interleukin-17 cytokine family has been proven to play a central role in the pathogenesis of psoriasis.  Brodalumab is a novel biol. agent that targets interleukin-17 mols. and their receptors.  We aimed to evaluate the safety and efficacy of brodalumab as a therapeutic agent for moderate-to-severe psoriasis in a meta-anal. framework.  Methods: A computer literature search of PubMed, OVID, Cochrane Central, EMBASE, EBSCO, Scopus, and Web of Science was conducted using relevant keywords.  Data were extd. from eligible trials and analyzed using RevMan (Version 5.3 for windows) and OpenMeta [Analyst] software.  Results: Six clin. trials (n = 4118 patients) were pooled in the final anal.  The pooled effect size favored brodalumab over placebo in terms of the Psoriasis Area and Severity Index (PASI) 75 [risk ratio (RR) = 12.61, 95% confidence interval (CI) 9.74-16.34], PASI 90 (RR = 28.72, 95% CI 18.34-44.98), and PASI 100 (RR = 61.23, 95% CI 25.48-147.17).  Anal. of secondary outcomes showed that brodalumab was superior to placebo in terms of static physician's global assessment (RR = 32.53, 95% CI 13.80-76.69) and psoriasis symptoms inventory scores (RR = 14.70, 95% CI 8.38-25.78).  Meta-regression anal. found a significant linear assocn. between the brodalumab dose and the effect size on PASI and psoriasis symptoms inventory scores.  The rate of overall adverse events was slightly higher in the brodalumab group (RR = 1.13, 95% CI 1.06-1.22); however, none of the individual adverse events were significantly higher in the brodalumab group, compared to the placebo group.  Conclusion: Brodalumab showed an acceptable safety profile and a robust efficacy in the treatment of moderate-to-severe plaque psoriasis.  However, the current evidence is insufficient to confirm maintenance of these results in the long term; therefore, larger studies with longer follow-up periods are required.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOSFFowbNs27Vg90H21EOLACvtfcHk0lhBvw8Fm8adSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVaqsbo%253D&md5=b9996588017aad4e3944b67299cc62df</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs40261-017-0500-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40261-017-0500-9%26sid%3Dliteratum%253Aachs%26aulast%3DAttia%26aufirst%3DA.%26aulast%3DAbushouk%26aufirst%3DA.%2BI.%26aulast%3DAhmed%26aufirst%3DH.%26aulast%3DGadelkarim%26aufirst%3DM.%26aulast%3DElgebaly%26aufirst%3DA.%26aulast%3DHassan%26aufirst%3DZ.%26aulast%3DAbdel-Daim%26aufirst%3DM.%2BM.%26aulast%3DNegida%26aufirst%3DA.%26atitle%3DSafety%2520and%2520efficacy%2520of%2520brodalumab%2520for%2520moderate-to-severe%2520plaque%2520psoriasis%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%26jtitle%3DClin.%2520Drug%2520Invest.%26date%3D2017%26volume%3D37%26spage%3D439%26epage%3D451%26doi%3D10.1007%2Fs40261-017-0500-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mease, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Heijde, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchlin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuchacovich, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuler, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladman, D. D.</span></span>; <span class="NLM_contrib-group">SPIRIT-P1 Study Group</span> <span> </span><span class="NLM_article-title">Ixekizumab, an interleukin-17A specific monoclonal antibody, for the treatment of biologic-naive patients with active psoriatic arthritis: results from the 24-week randomised, double-blind, placebo-controlled and active (adalimumab)-controlled period of the phase III trial SPIRIT-P1</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2016-209709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1136%2Fannrheumdis-2016-209709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27553214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2017&pages=79-87&author=P.+J.+Measeauthor=D.+van+der+Heijdeauthor=C.+T.+Ritchlinauthor=M.+Okadaauthor=R.+S.+Cuchacovichauthor=C.+L.+Shulerauthor=C.+Y.+Linauthor=D.+K.+Braunauthor=C.+H.+Leeauthor=D.+D.+Gladmanauthor=SPIRIT-P1+Study+Group&title=Ixekizumab%2C+an+interleukin-17A+specific+monoclonal+antibody%2C+for+the+treatment+of+biologic-naive+patients+with+active+psoriatic+arthritis%3A+results+from+the+24-week+randomised%2C+double-blind%2C+placebo-controlled+and+active+%28adalimumab%29-controlled+period+of+the+phase+III+trial+SPIRIT-P1&doi=10.1136%2Fannrheumdis-2016-209709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ixekizumab, an interleukin-17A specific monoclonal antibody, for the treatment of biologic-naive patients with active psoriatic arthritis: results from the 24-week randomised, double-blind, placebo controlled and active (adalimumab)-controlled period of the phase III trial SPIRIT-P1</span></div><div class="casAuthors">Mease, Philip J.; van der Heijde, Desiree; Ritchlin, Christopher T.; Okada, Masato; Cuchacovich, Raquel S.; Shuler, Catherine L.; Lin, Chen-Yen; Braun, Daniel K.; Lee, Chin H.; Gladman, Dafna D.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-87</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective: To assess the safety and efficacy of ixekizumab, a monoclonal antibody that inhibits interleukin-17A, in a double-blind phase III trial enrolling patients with active psoriatic arthritis (PsA).  Methods: Patients naive to biol. therapy with active PsA were randomised to s.c. injections of placebo (N = 106), adalimumab 40 mg once every 2 wk (active ref.; N = 101), ixekizumab 80 mg once every 2 wk (IXEQ2W) (N = 103), or ixekizumab 80 mg once every 4 wk (IXEQ4W) (N = 107).  Both ixekizumab regimens included a 160-mg starting dose.  The primary objective was to assess the superiority of IXEQ2W or IXEQ4W vs. placebo as measured by the proportion of patients achieving an American College of Rheumatol. 20 (ACR20) response at week 24.  Results: Significantly more patients treated with ixekizumab achieved an ACR20 response with IXEQ2W (62.1%) or IXEQ4W (57.9%) than placebo (30.2%) (p≤0.001; non-responder imputation method).  Disease activity and functional disability were significantly improved with both ixekizumab doses vs. placebo at weeks 12 and 24, and there was significantly less progression of structural damage at week 24 (p≤0.01).  Clearance of plaque psoriasis was greater with ixekizumab than placebo (p≤0.001).  Efficacy results with adalimumab, the active ref. arm, showed significant improvements vs. placebo.  Treatment-emergent adverse events were more frequent with ixekizumab (65.7-66.4%) and adalimumab (64.4%) than placebo (47.2%) (p<0.05).  Conclusions: In biol.-naive patients with active PsA, ixekizumab treatment resulted in improvements in disease activity and phys. function, as well as in the inhibition of structural damage progression.  Overall, adverse events were more frequent in all active groups compared with placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3hHplmarqLVg90H21EOLACvtfcHk0lhWKQJtE5wVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFahsr4%253D&md5=3beef8f867a6e8ada97080e4f3bc8cbe</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2016-209709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2016-209709%26sid%3Dliteratum%253Aachs%26aulast%3DMease%26aufirst%3DP.%2BJ.%26aulast%3Dvan%2Bder%2BHeijde%26aufirst%3DD.%26aulast%3DRitchlin%26aufirst%3DC.%2BT.%26aulast%3DOkada%26aufirst%3DM.%26aulast%3DCuchacovich%26aufirst%3DR.%2BS.%26aulast%3DShuler%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DBraun%26aufirst%3DD.%2BK.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DGladman%26aufirst%3DD.%2BD.%26aulast%3D%26atitle%3DIxekizumab%252C%2520an%2520interleukin-17A%2520specific%2520monoclonal%2520antibody%252C%2520for%2520the%2520treatment%2520of%2520biologic-naive%2520patients%2520with%2520active%2520psoriatic%2520arthritis%253A%2520results%2520from%2520the%252024-week%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520and%2520active%2520%2528adalimumab%2529-controlled%2520period%2520of%2520the%2520phase%2520III%2520trial%2520SPIRIT-P1%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2017%26volume%3D76%26spage%3D79%26epage%3D87%26doi%3D10.1136%2Fannrheumdis-2016-209709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Modulators of the nuclear receptor retinoic acid receptor-related orphan receptor-γ (RORγ or RORc)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5871</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/jm401901d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401901d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5871-92&author=B.+P.+Fauberauthor=S.+Magnuson&title=Modulators+of+the+nuclear+receptor+retinoic+acid+receptor-related+orphan+receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29&doi=10.1021%2Fjm401901d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Modulators of the Nuclear Receptor Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc)</span></div><div class="casAuthors">Fauber, Benjamin P.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5871-5892</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  As the biol. surrounding the nuclear receptor retinoic acid receptor-related orphan receptor-gamma (RORγ or RORc) continues to evolve, significant effort has been invested in discovering modulators of this potentially important target for the treatment of metabolic and immunol. diseases.  Several major pharmaceutical and biotechnol. companies have disclosed RORc inhibitors or partnered with other players in the field.  In this perspective, we discuss both the biol. and the underlying structural biol. of RORc, and summarize the RORc modulators disclosed in the scientific and patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhzxefrY4hLVg90H21EOLACvtfcHk0lhWKQJtE5wVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVartbo%253D&md5=42b350bff7776d4b5689275eab214d86</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm401901d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401901d%26sid%3Dliteratum%253Aachs%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DModulators%2520of%2520the%2520nuclear%2520receptor%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5871%26epage%3D92%26doi%3D10.1021%2Fjm401901d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbieg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span> <span> </span><span class="NLM_article-title">RORγ antagonists and inverse agonists: a patent review</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1236918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1080%2F13543776.2017.1236918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27629281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=101-112&author=S.+M.+Bronnerauthor=J.+R.+Zbiegauthor=J.+J.+Crawford&title=ROR%CE%B3+antagonists+and+inverse+agonists%3A+a+patent+review&doi=10.1080%2F13543776.2017.1236918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">RORγ antagonists and inverse agonists: a patent review</span></div><div class="casAuthors">Bronner, Sarah M.; Zbieg, Jason R.; Crawford, James J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-112</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The transcription factor RORγ plays a crit. role in the expression of pro-inflammatory cytokine interleukin IL-17 and is therefore an attractive target for the treatment of inflammatory diseases.  Interest in this mol. target has been heightened by the advancement of orally and topically administered RORγ modulators into clin. trials.  The present review seeks to summarize published patent applications from assignee companies that have disclosed Investigational New Drug (IND) filings for small mol. RORγ/RORγt antagonists and inverse agonists.  The field of RORγ research is extremely competitive, with the majority of companies targeting psoriasis as the primary disease indication.  Vitae Pharmaceuticals is currently the most advanced, with a potential first-in-class oral RORγ-modulator for the treatment of psoriasis.  Future efforts will likely expand into potential applications of RORγ-modulators in the lesser explored immune-related areas of rheumatoid arthritis, type 1 diabetes, lupus, and irritable bowel disorder, as well as cancer immunotherapy and castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj8lMzWsYVA7Vg90H21EOLACvtfcHk0lhWKQJtE5wVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nsbzP&md5=d80db6c4c948bc12c5d0ce4b41edd2fb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1236918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1236918%26sid%3Dliteratum%253Aachs%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DZbieg%26aufirst%3DJ.%2BR.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26atitle%3DROR%25CE%25B3%2520antagonists%2520and%2520inverse%2520agonists%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D101%26epage%3D112%26doi%3D10.1080%2F13543776.2017.1236918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, R. C.</span></span> <span> </span><span class="NLM_article-title">Combating autoimmune diseases with retinoic acid receptor-related orphan receptor-γ (RORγ or RORc) inhibitors: hits and misses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10976</span>– <span class="NLM_lpage">10995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00588</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00588" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10976-10995&author=V.+B.+Pandyaauthor=S.+Kumarauthor=+Sachchidanandauthor=R.+Sharmaauthor=R.+C.+Desai&title=Combating+autoimmune+diseases+with+retinoic+acid+receptor-related+orphan+receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29+inhibitors%3A+hits+and+misses&doi=10.1021%2Facs.jmedchem.8b00588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span></div><div class="casAuthors">Pandya, Vrajesh B.; Kumar, Sanjay; Sachchidanand; Sharma, Rajiv; Desai, Ranjit C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10976-10995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear receptor retinoic acid receptor-related orphan receptor gamma (RORγ or RORc) is a key transcription factor for the prodn. of pro-inflammatory cytokines implicated in the pathogenesis of autoimmune diseases.  Recently, small mol. inhibitors of RORc drew the enormous attention of the research community worldwide as a possible therapy for autoimmune diseases, mediated by the IL-17 cytokine.  With the clin. proof-of-concept inferred from a small mol. inhibitor VTP-43742 for psoriasis and recent inflow of several RORc inhibitors into the clinic for therapeutic interventions in autoimmune diseases, this field continues to evolve.  This review briefly summarizes the RORc inhibitors disclosed in the literature and discusses the progress made by these inhibitors in combating autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGqZl3bPbtLVg90H21EOLACvtfcHk0ljSq-g1_hitiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E&md5=4113d22f73ab2911535eef053801994d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00588%26sid%3Dliteratum%253Aachs%26aulast%3DPandya%26aufirst%3DV.%2BB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSachchidanand%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DDesai%26aufirst%3DR.%2BC.%26atitle%3DCombating%2520autoimmune%2520diseases%2520with%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%2520inhibitors%253A%2520hits%2520and%2520misses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10976%26epage%3D10995%26doi%3D10.1021%2Facs.jmedchem.8b00588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanis, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent Barbay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummer, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelen, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlueter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spurlino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, K.</span></span> <span> </span><span class="NLM_article-title">3-Substituted quinolines as RORγt inverse agonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2019.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=31010722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXot1Sru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1463-1470&author=V.+M.+Tanisauthor=H.+Venkatesanauthor=M.+D.+Cummingsauthor=M.+Albersauthor=J.+Kent+Barbayauthor=K.+Hermanauthor=D.+A.+Kummerauthor=C.+Milliganauthor=M.+I.+Nelenauthor=R.+Nishimuraauthor=T.+Schlueterauthor=B.+Scottauthor=J.+Spurlinoauthor=R.+Wolinauthor=C.+Woodsauthor=X.+Xueauthor=J.+P.+Edwardsauthor=A.+M.+Fourieauthor=K.+Leonard&title=3-Substituted+quinolines+as+ROR%CE%B3t+inverse+agonists&doi=10.1016%2Fj.bmcl.2019.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">3-Substituted Quinolines as RORγt Inverse Agonists</span></div><div class="casAuthors">Tanis, Virginia M.; Venkatesan, Hariharan; Cummings, Maxwell D.; Albers, Michael; Kent Barbay, J.; Herman, Krystal; Kummer, David A.; Milligan, Cynthia; Nelen, Marina I.; Nishimura, Rachel; Schlueter, Thomas; Scott, Brian; Spurlino, John; Wolin, Ronald; Woods, Craig; Xue, Xiaohua; Edwards, James P.; Fourie, Anne M.; Leonard, Kristi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1470</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have previously reported the syntheses of a series of 3,6-disubstituted quinolines as modulators of the retinoic acid receptor-related orphan receptor gamma t (RORγt).  These mols. are potent binders but are high mol. wt. and they exhibited poor soly. at pH 2 and pH 7.  This manuscript details our efforts at improving phys. chem. properties for this series of compds. by increasing the diversity at the 3-position (i.e. introducing heteroatoms and lowering the mol. wt.).  These efforts have led to mols. which are potent binders with improved soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhylTaPGx9U7Vg90H21EOLACvtfcHk0ljSq-g1_hitiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXot1Sru74%253D&md5=14b0728bc23fb5c8cafdab363276ebf7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DTanis%26aufirst%3DV.%2BM.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DAlbers%26aufirst%3DM.%26aulast%3DKent%2BBarbay%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DK.%26aulast%3DKummer%26aufirst%3DD.%2BA.%26aulast%3DMilligan%26aufirst%3DC.%26aulast%3DNelen%26aufirst%3DM.%2BI.%26aulast%3DNishimura%26aufirst%3DR.%26aulast%3DSchlueter%26aufirst%3DT.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DSpurlino%26aufirst%3DJ.%26aulast%3DWolin%26aufirst%3DR.%26aulast%3DWoods%26aufirst%3DC.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DFourie%26aufirst%3DA.%2BM.%26aulast%3DLeonard%26aufirst%3DK.%26atitle%3D3-Substituted%2520quinolines%2520as%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1463%26epage%3D1470%26doi%3D10.1016%2Fj.bmcl.2019.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shichijo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">A novel RORγt inhibitor is a potential therapeutic agent for the topical treatment of psoriasis with low risk of thymic aberrations</span>. <i>J. Dermatol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.jdermsci.2019.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.jdermsci.2019.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30905492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvV2ksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2019&pages=176-185&author=C.+Imuraauthor=A.+Ueyamaauthor=Y.+Sasakiauthor=M.+Shimizuauthor=Y.+Furueauthor=N.+Taiauthor=K.+Tsujiiauthor=K.+Katayamaauthor=T.+Okunoauthor=M.+Shichijoauthor=K.+Yasuiauthor=M.+Yamamoto&title=A+novel+ROR%CE%B3t+inhibitor+is+a+potential+therapeutic+agent+for+the+topical+treatment+of+psoriasis+with+low+risk+of+thymic+aberrations&doi=10.1016%2Fj.jdermsci.2019.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A novel RORγt inhibitor is a potential therapeutic agent for the topical treatment of psoriasis with low risk of thymic aberrations</span></div><div class="casAuthors">Imura, Chihiro; Ueyama, Azumi; Sasaki, Yoshikazu; Shimizu, Masaya; Furue, Yoko; Tai, Nobuyuki; Tsujii, Kenichiro; Katayama, Kazufumi; Okuno, Takayuki; Shichijo, Michitaka; Yasui, Kiyoshi; Yamamoto, Mina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dermatological Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">176-185</span>CODEN:
                <span class="NLM_cas:coden">JDSCEI</span>;
        ISSN:<span class="NLM_cas:issn">0923-1811</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptor gamma t (RORγt) has crit. roles in the development, maintenance and function of interleukin (IL)-17-producing cells and is a highly attractive target for the treatment of IL-17-mediated autoimmune disease, particularly psoriasis.  On the other hand, RORγt is also crit. for controlling apoptosis during thymopoiesis, and genetic RORγt ablation or systematic RORγt inhibition cause progressive thymic aberrations leading to T cell lymphomas.  We investigated whether topical administration of our novel RORγt inhibitor, S18-000003 has therapeutic potential for psoriasis with low risk of thymic aberrations.  We evaluated the effect of topical S18-000003 on psoriasis-like skin inflammation and influence on the thymus in a 12-O-tetradecanoylphorbol-13-acetate-induced K14.Stat3C mouse psoriasis model.  S18-000003 markedly inhibited the development of psoriatic skin inflammation via suppression of the IL-17 pathway.  In the skin, S18-000003 suppressed all subsets of IL-17-producing cells that we previously identified in this psoriasis model: Th17 cells, Tc17 cells, dermal γδ T cells, TCR- cells that probably included innate lymphoid cells, and CD4-CD8- double-neg. αβ T cells.  Notably, neither redn. of CD4+CD8+ double-pos. thymocytes nor dysregulation of cell cycling was obsd. in S18-000003-treated mice, even at a high dose.  Our topically administered RORγt inhibitor is a potential therapeutic agent for psoriasis with low risk of thymic lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuafyicGpTLVg90H21EOLACvtfcHk0ljSq-g1_hitiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvV2ksbY%253D&md5=e700b0f60f059581b93976032695f822</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jdermsci.2019.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jdermsci.2019.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DImura%26aufirst%3DC.%26aulast%3DUeyama%26aufirst%3DA.%26aulast%3DSasaki%26aufirst%3DY.%26aulast%3DShimizu%26aufirst%3DM.%26aulast%3DFurue%26aufirst%3DY.%26aulast%3DTai%26aufirst%3DN.%26aulast%3DTsujii%26aufirst%3DK.%26aulast%3DKatayama%26aufirst%3DK.%26aulast%3DOkuno%26aufirst%3DT.%26aulast%3DShichijo%26aufirst%3DM.%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DA%2520novel%2520ROR%25CE%25B3t%2520inhibitor%2520is%2520a%2520potential%2520therapeutic%2520agent%2520for%2520the%2520topical%2520treatment%2520of%2520psoriasis%2520with%2520low%2520risk%2520of%2520thymic%2520aberrations%26jtitle%3DJ.%2520Dermatol.%2520Sci.%26date%3D2019%26volume%3D93%26spage%3D176%26epage%3D185%26doi%3D10.1016%2Fj.jdermsci.2019.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-indanyl benzamides as potent RORγt inverse agonists</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.ejmech.2019.01.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30743096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFygtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=37-48&author=J.+Tianauthor=N.+Sunauthor=M.+Yuauthor=X.+Guauthor=Q.+Xieauthor=L.+Shaoauthor=J.+Liuauthor=L.+Liuauthor=Y.+Wang&title=Discovery+of+N-indanyl+benzamides+as+potent+ROR%CE%B3t+inverse+agonists&doi=10.1016%2Fj.ejmech.2019.01.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-indanyl benzamides as potent RORγt inverse agonists</span></div><div class="casAuthors">Tian, Jinlong; Sun, Nannan; Yu, Mingcheng; Gu, Xianfeng; Xie, Qiong; Shao, Liming; Liu, Jin; Liu, Li; Wang, Yonghui</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-48</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The retinoic acid receptor-related orphan receptor-gamma-t (RORγt) is a promising therapeutic target for treatment of Th17 cell-mediated autoimmune diseases.  Based on a scaffold hopping/conformational restriction strategy, a series of N-indanyl benzamides as novel RORγt inverse agonists was discovered.  Exploration of structure-activity relationship on the piperazine ring, benzoyl moiety and cyclopentyl moiety of N-indanyl benzamides 2a and 2d led to identification of potent RORγt inverse agonists.  Compd. 5c with (S)-enantiomer was found having an IC50 of 153.7 nM in Fluorescence Resonance Energy Transfer (FRET) assay, and an IC50 of 47.1 nM in mouse Th17 cell differentiation assay, which represents a promising starting point for developing potent small mol. RORγt inverse agonists.  Binding modes of the two enantiomers 5c and 5d in RORγt ligand binding domain were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtcwkL0EZAzrVg90H21EOLACvtfcHk0liQFFaAJNCXnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFygtbs%253D&md5=52135643f47a06c14abc425a69341667</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.082%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DShao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520N-indanyl%2520benzamides%2520as%2520potent%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D37%26epage%3D48%26doi%3D10.1016%2Fj.ejmech.2019.01.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotoku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujioka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suwa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikenogami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozaki, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of second generation RORγ inhibitors composed of an azole scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2842</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01567</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01567" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Sksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2837-2842&author=M.+Kotokuauthor=T.+Maebaauthor=S.+Fujiokaauthor=M.+Yokotaauthor=N.+Sekiauthor=K.+Itoauthor=Y.+Suwaauthor=T.+Ikenogamiauthor=K.+Hirataauthor=Y.+Haseauthor=Y.+Katsudaauthor=N.+Miyagawaauthor=K.+Aritaauthor=K.+Asahinaauthor=M.+Noguchiauthor=A.+Nomuraauthor=S.+Doiauthor=T.+Adachiauthor=P.+Croweauthor=H.+Taoauthor=S.+Thacherauthor=H.+Hashimotoauthor=T.+Suzukiauthor=M.+Shiozaki&title=Discovery+of+second+generation+ROR%CE%B3+inhibitors+composed+of+an+azole+scaffold&doi=10.1021%2Facs.jmedchem.8b01567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Second Generation RORγ Inhibitors Composed of an Azole Scaffold</span></div><div class="casAuthors">Kotoku, Masayuki; Maeba, Takaki; Fujioka, Shingo; Yokota, Masahiro; Seki, Noriyoshi; Ito, Keisuke; Suwa, Yoshihiro; Ikenogami, Taku; Hirata, Kazuyuki; Hase, Yasunori; Katsuda, Yoshiaki; Miyagawa, Naoki; Arita, Kojo; Asahina, Kota; Noguchi, Masato; Nomura, Akihiro; Doi, Satoki; Adachi, Tsuyoshi; Crowe, Paul; Tao, Haiyan; Thacher, Scott; Hashimoto, Hiromasa; Suzuki, Takayoshi; Shiozaki, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2837-2842</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Starting from a previously reported retinoid-related orphan receptor γ (RORγ) inhibitor, successive efforts to improve in vivo potency were continued.  Introduction of metabolically beneficial motifs in conjunction with scaffold hopping was examd., resulting in discovery of the second generation RORγ inhibitor composed of a 4-(isoxazol-3-yl)butanoic acid scaffold (I).  Compd. I achieved a 10-fold improvement in in vivo potency in a mouse CD3 challenge model along with significant anti-inflammatory effects in a mouse dermatitis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIwG22VwIKKbVg90H21EOLACvtfcHk0liQFFaAJNCXnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Sksr4%253D&md5=b62cac378cd152f9785731453c2d6ad0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01567%26sid%3Dliteratum%253Aachs%26aulast%3DKotoku%26aufirst%3DM.%26aulast%3DMaeba%26aufirst%3DT.%26aulast%3DFujioka%26aufirst%3DS.%26aulast%3DYokota%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSuwa%26aufirst%3DY.%26aulast%3DIkenogami%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DHase%26aufirst%3DY.%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DMiyagawa%26aufirst%3DN.%26aulast%3DArita%26aufirst%3DK.%26aulast%3DAsahina%26aufirst%3DK.%26aulast%3DNoguchi%26aufirst%3DM.%26aulast%3DNomura%26aufirst%3DA.%26aulast%3DDoi%26aufirst%3DS.%26aulast%3DAdachi%26aufirst%3DT.%26aulast%3DCrowe%26aufirst%3DP.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DThacher%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DShiozaki%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520second%2520generation%2520ROR%25CE%25B3%2520inhibitors%2520composed%2520of%2520an%2520azole%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2837%26epage%3D2842%26doi%3D10.1021%2Facs.jmedchem.8b01567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruzanov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galella, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanteru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salter-Cid, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, T. G. M.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of phenyl (3-phenylpyrrolidin-3-yl)sulfones as selective, orally active RORγt inverse agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVyltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=367-373&author=J.+J.+Duanauthor=Z.+Luauthor=B.+Jiangauthor=S.+Stachuraauthor=C.+A.+Weigeltauthor=J.+S.+Sackauthor=J.+Khanauthor=M.+Ruzanovauthor=M.+A.+Galellaauthor=D.+R.+Wuauthor=M.+Yardeauthor=D.+R.+Shenauthor=D.+J.+Shusterauthor=V.+Borowskiauthor=J.+H.+Xieauthor=L.+Zhangauthor=S.+Vanteruauthor=A.+K.+Guptaauthor=A.+Mathurauthor=Q.+Zhaoauthor=W.+Fosterauthor=L.+M.+Salter-Cidauthor=P.+H.+Carterauthor=T.+G.+M.+Dhar&title=Structure-based+discovery+of+phenyl+%283-phenylpyrrolidin-3-yl%29sulfones+as+selective%2C+orally+active+ROR%CE%B3t+inverse+agonists&doi=10.1021%2Facsmedchemlett.9b00010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Discovery of Phenyl (3-Phenylpyrrolidin-3-yl)sulfones as Selective, Orally Active RORγt Inverse Agonists</span></div><div class="casAuthors">Duan, James J.-W.; Lu, Zhonghui; Jiang, Bin; Stachura, Sylwia; Weigelt, Carolyn A.; Sack, John S.; Khan, Javed; Ruzanov, Max; Galella, Michael A.; Wu, Dauh-Rurng; Yarde, Melissa; Shen, Ding-Ren; Shuster, David J.; Borowski, Virna; Xie, Jenny H.; Zhang, Lisa; Vanteru, Sridhar; Gupta, Arun Kumar; Mathur, Arvind; Zhao, Qihong; Foster, William; Salter-Cid, Luisa M.; Carter, Percy H.; Dhar, T. G. Murali</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">367-373</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new Ph (3-phenylpyrrolidin-3-yl)sulfone series of RORγt inverse agonists was discovered utilizing the binding conformation of previously reported bicyclic sulfonamide 1.  Through a combination of structure-based design and structure-activity relationship studies, a polar set of amides at N1-position of the pyrrolidine ring and perfluoroisopropyl group at para-position of the 3-Ph group were identified as crit. structural elements to achieve high selectivity against PXR, LXRα, and LXRβ.  Further optimization led to the discovery of (1R,4r)-4-((R)-3-((4-fluorophenyl)sulfonyl)-3-(4-(perfluoropropan-2-yl)phenyl)pyrrolidine-1-carbonyl)cyclohexane-1-carboxylic acid (26), which displayed excellent selectivity, desirable liability and pharmacokinetic properties in vitro, and a good pharmacokinetic profile in mouse.  Oral administration of 26 demonstrated dose-dependent inhibition of IL-17 prodn. in a mouse IL-2/IL-23-induced pharmacodynamic model and biol.-like efficacy in an IL-23-induced mouse acanthosis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZBJUkp-y6zbVg90H21EOLACvtfcHk0li3yHHxscKbMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVyltLw%253D&md5=16114ce5cec852904e038f8d33be67a1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00010%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%2BJ.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DStachura%26aufirst%3DS.%26aulast%3DWeigelt%26aufirst%3DC.%2BA.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DRuzanov%26aufirst%3DM.%26aulast%3DGalella%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DD.%2BR.%26aulast%3DYarde%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DShuster%26aufirst%3DD.%2BJ.%26aulast%3DBorowski%26aufirst%3DV.%26aulast%3DXie%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVanteru%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DA.%2BK.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DFoster%26aufirst%3DW.%26aulast%3DSalter-Cid%26aufirst%3DL.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DDhar%26aufirst%3DT.%2BG.%2BM.%26atitle%3DStructure-based%2520discovery%2520of%2520phenyl%2520%25283-phenylpyrrolidin-3-yl%2529sulfones%2520as%2520selective%252C%2520orally%2520active%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D367%26epage%3D373%26doi%3D10.1021%2Facsmedchemlett.9b00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, V.</span></span> <span> </span><span class="NLM_article-title">Principles for modulation of the nuclear receptor superfamily</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">950</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1038/nrd1551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd1551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=15520817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=950-964&author=H.+Gronemeyerauthor=J.-%C3%85.+Gustafssonauthor=V.+Laudet&title=Principles+for+modulation+of+the+nuclear+receptor+superfamily&doi=10.1038%2Fnrd1551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for modulation of the nuclear receptor superfamily</span></div><div class="casAuthors">Gronemeyer, Hinrich; Gustafsson, Jan-A.; Laudet, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">950-964</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nuclear receptors are major targets for drug discovery and have key roles in development and homeostasis, as well as in many diseases such as obesity, diabetes and cancer.  This review provides a general overview of the mechanism of action of nuclear receptors and explores the various factors that are instrumental in modulating their pharmacol.  In most cases, the response of a given receptor to a particular ligand in a specific tissue will be dictated by the set of proteins with which the receptor is able to interact.  One of the most promising aspects of nuclear receptor pharmacol. is that it is now possible to develop ligands with a large spectrum of full, partial or inverse agonist or antagonist activities, but also compds., called selective nuclear receptor modulators, that activate only a subset of the functions induced by the cognate ligand or that act in a cell-type-selective manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWL9yf1CceLVg90H21EOLACvtfcHk0li3yHHxscKbMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrY%253D&md5=aedb60785c235bf44cb5f0ea34236090</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd1551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1551%26sid%3Dliteratum%253Aachs%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DLaudet%26aufirst%3DV.%26atitle%3DPrinciples%2520for%2520modulation%2520of%2520the%2520nuclear%2520receptor%2520superfamily%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D950%26epage%3D964%26doi%3D10.1038%2Fnrd1551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muegge, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kugler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terenzio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadia, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrow, N. A.</span></span> <span> </span><span class="NLM_article-title">Structural studies unravel the active conformation of apo RORγt nuclear receptor and a common inverse agonism of two diverse classes of RORγt inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">11618</span>– <span class="NLM_lpage">11630</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.789024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1074%2Fjbc.M117.789024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28546429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOjtbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=11618-11630&author=X.+Liauthor=M.+Andersonauthor=D.+Collinauthor=I.+Mueggeauthor=J.+Wanauthor=D.+Brennanauthor=S.+Kuglerauthor=D.+Terenzioauthor=C.+Kennedyauthor=S.+Linauthor=M.+E.+Labadiaauthor=B.+Cookauthor=R.+Hughesauthor=N.+A.+Farrow&title=Structural+studies+unravel+the+active+conformation+of+apo+ROR%CE%B3t+nuclear+receptor+and+a+common+inverse+agonism+of+two+diverse+classes+of+ROR%CE%B3t+inhibitors&doi=10.1074%2Fjbc.M117.789024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural studies unravel the active conformation of apo RORγt nuclear receptor and a common inverse agonism of two diverse classes of RORγt inhibitors</span></div><div class="casAuthors">Li, Xiang; Anderson, Marie; Collin, Delphine; Muegge, Ingo; Wan, John; Brennan, Debra; Kugler, Stanley; Terenzio, Donna; Kennedy, Charles; Lin, Siqi; Labadia, Mark E.; Cook, Brian; Hughes, Robert; Farrow, Neil A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11618-11630</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The nuclear receptor retinoid acid receptor-related orphan receptor γt (RORγt) is a master regulator of the Th17/IL-17 pathway that plays crucial roles in the pathogenesis of autoimmunity.  RORγt has recently emerged as a highly promising target for treatment of a no. of autoimmune diseases.  Through high-throughput screening, we previously identified several classes of inverse agonists for RORγt.  Here, we report the crystal structures for the ligand-binding domain of RORγt in both apo and ligand-bound states.  We show that apo RORγt adopts an active conformation capable of recruiting coactivator peptides and present a detailed anal. of the structural determinants that stabilize helix 12 (H12) of RORγt in the active state in the absence of a ligand.  The structures of ligand-bound RORγt revealed that binding of the inverse agonists disrupted crit. interactions that stabilized H12.  This destabilizing effect was supported by ab initio calcns. and exptl. by a normalized crystallog. B-factor anal.  Of note, the H12 destabilization in the active state shifted the conformational equil. of RORγt toward an inactive state, which underlies the mol. mechanism of action for the inverse agonists reported here.  These findings highlight that nuclear receptor structure and function are dictated by a dynamic conformational equil. and that subtle changes in ligand structures can shift this equil. in opposite directions, leading to a functional switch from agonists to inverse agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzicB5efbjf7Vg90H21EOLACvtfcHk0li3yHHxscKbMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOjtbjJ&md5=8224bb328772787a2e1df0aec43566d6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.789024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.789024%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DD.%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DWan%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DD.%26aulast%3DKugler%26aufirst%3DS.%26aulast%3DTerenzio%26aufirst%3DD.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLabadia%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DB.%26aulast%3DHughes%26aufirst%3DR.%26aulast%3DFarrow%26aufirst%3DN.%2BA.%26atitle%3DStructural%2520studies%2520unravel%2520the%2520active%2520conformation%2520of%2520apo%2520ROR%25CE%25B3t%2520nuclear%2520receptor%2520and%2520a%2520common%2520inverse%2520agonism%2520of%2520two%2520diverse%2520classes%2520of%2520ROR%25CE%25B3t%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D11618%26epage%3D11630%26doi%3D10.1074%2Fjbc.M117.789024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muscat, G. E. O.</span></span> <span> </span><span class="NLM_article-title">Characterization of the retinoid orphan-related receptor-alpha coactivator binding interface: a structural basis for ligand-independent transcription</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1012</span>, <span class="refDoi"> DOI: 10.1210/mend.16.5.0837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1210%2Fmend.16.5.0837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=11981035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFGhsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=998-1012&author=J.+M.+Harrisauthor=P.+Lauauthor=S.+L.+Chenauthor=G.+E.+O.+Muscat&title=Characterization+of+the+retinoid+orphan-related+receptor-alpha+coactivator+binding+interface%3A+a+structural+basis+for+ligand-independent+transcription&doi=10.1210%2Fmend.16.5.0837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the retinoid orphan-related receptor-α coactivator binding interface: A structural basis for ligand-independent transcription</span></div><div class="casAuthors">Harris, Jonathan M.; Lau, Patrick; Chen, Shen Liang; Muscat, George E. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">998-1012</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">The retinoid orphan-related receptor-α (RORα) is a member of the ROR subfamily of orphan receptors and acts as a constitutive activator of transcription in the absence of exogenous ligands.  To understand the basis of this activity, we constructed a homol. model of RORα using the closely related TRβ as a template.  Mol. modeling suggested that bulky hydrophobic side chains occupy the RORα ligand cavity leaving a small but distinct cavity that may be involved in receptor stabilization.  This model was subject to docking simulation with a receptor-interacting peptide from the steroid receptor coactivator, GR-interacting protein-1, which delineated a coactivator binding surface consisting of the signature motif spanning helixes 3-5 and helix 12 [activation function 2 (AF2)].  Probing this surface with scanning alanine mutagenesis showed structural and functional equivalence between homologous residues of RORα and TRβ.  This was surprising (given that RORα is a ligand-independent activator, whereas TRβ has an abs. requirement for ligand) and prompted us to use mol. modeling to identify differences between RORα and TRβ in the way that the AF2 helix interacts with the rest of the receptor.  Modeling highlighted a nonconserved amino acid in helix 11 of RORα (Phe491) and a short-length of 3.10 helix at the N terminus of AF2 which we suggest (1) ensures that AF2 is locked permanently in the holoconformation described for other liganded receptors and thus (2) enables ligand-independent recruitment of coactivators.  Consistent with this, mutation of RORα Phe491 to either methionine or alanine (methionine is the homologous residue in TRβ), reduced and ablated transcriptional activation and recruitment of coactivators, resp.  Furthermore, we were able to reconstitute transcriptional activity for both a deletion mutant of RORα lacking AF2, and Phe491Met, by overexpression of a GAL-AF2 fusion protein, demonstrating ligand-independent recruitment of AF2 and a role for Phe491 in recruiting AF2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5veDP0-3KL7Vg90H21EOLACvtfcHk0ljO1ig8xqUzwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFGhsLw%253D&md5=cffc5abe6423fc40e711443bc6231686</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1210%2Fmend.16.5.0837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fmend.16.5.0837%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DLau%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DS.%2BL.%26aulast%3DMuscat%26aufirst%3DG.%2BE.%2BO.%26atitle%3DCharacterization%2520of%2520the%2520retinoid%2520orphan-related%2520receptor-alpha%2520coactivator%2520binding%2520interface%253A%2520a%2520structural%2520basis%2520for%2520ligand-independent%2520transcription%26jtitle%3DMol.%2520Endocrinol.%26date%3D2002%26volume%3D16%26spage%3D998%26epage%3D1012%26doi%3D10.1210%2Fmend.16.5.0837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesch, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendling, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aicher, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, G. D.</span></span> <span> </span><span class="NLM_article-title">Sterol metabolism controls TH17 differentiation by generating endogenous RORγ agonists</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnchembio.1714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=25558972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=141-147&author=X.+Huauthor=Y.+Wangauthor=L.+Y.+Haoauthor=X.+Liuauthor=C.+A.+Leschauthor=B.+M.+Sanchezauthor=J.+M.+Wendlingauthor=R.+W.+Morganauthor=T.+D.+Aicherauthor=L.+L.+Carterauthor=P.+L.+Toogoodauthor=G.+D.+Glick&title=Sterol+metabolism+controls+TH17+differentiation+by+generating+endogenous+ROR%CE%B3+agonists&doi=10.1038%2Fnchembio.1714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sterol metabolism controls TH17 differentiation by generating endogenous RORγ agonists</span></div><div class="casAuthors">Hu, Xiao; Wang, Yahong; Hao, Ling-Yang; Liu, Xikui; Lesch, Chuck A.; Sanchez, Brian M.; Wendling, Jay M.; Morgan, Rodney W.; Aicher, Tom D.; Carter, Laura L.; Toogood, Peter L.; Glick, Gary D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptor γ (RORγt) controls the differentiation of naive CD4+ T cells into the TH17 lineage, which are crit. cells in the pathogenesis of autoimmune diseases.  Here the authors report that during TH17 differentiation, cholesterol biosynthesis and uptake programs are induced, whereas their metab. and efflux programs are suppressed.  These changes result in the accumulation of the cholesterol precursor, desmosterol, which functions as a potent endogenous RORγ agonist.  Generation of cholesterol precursors is essential for TH17 differentiation as blocking cholesterol synthesis with chem. inhibitors at steps before the formation of active precursors reduces differentiation.  Upon activation, metabolic changes also lead to prodn. of specific sterol-sulfate conjugates that favor activation of RORγ over the TH17-inhibiting sterol receptor LXR.  Thus, TH17 differentiation is orchestrated by coordinated sterol synthesis, mobilization and metab. to selectively activate RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKonRBn05nmbVg90H21EOLACvtfcHk0ljO1ig8xqUzwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislOltQ%253D%253D&md5=e82af9d561a1e4683a86b815cb734971</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1714%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DL.%2BY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLesch%26aufirst%3DC.%2BA.%26aulast%3DSanchez%26aufirst%3DB.%2BM.%26aulast%3DWendling%26aufirst%3DJ.%2BM.%26aulast%3DMorgan%26aufirst%3DR.%2BW.%26aulast%3DAicher%26aufirst%3DT.%2BD.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DGlick%26aufirst%3DG.%2BD.%26atitle%3DSterol%2520metabolism%2520controls%2520TH17%2520differentiation%2520by%2520generating%2520endogenous%2520ROR%25CE%25B3%2520agonists%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D141%26epage%3D147%26doi%3D10.1038%2Fnchembio.1714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huh, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krout, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malapaka, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciofani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santori, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gin, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastinejad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">Digoxin and its derivatives suppress TH17 cell differentiation by antagonizing RORγt activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>472</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1038/nature09978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnature09978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21441909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=472&publication_year=2011&pages=486-490&author=J.+R.+Huhauthor=M.+W.+Leungauthor=P.+Huangauthor=D.+A.+Ryanauthor=M.+R.+Kroutauthor=R.+R.+Malapakaauthor=J.+Chowauthor=N.+Manelauthor=M.+Ciofaniauthor=S.+V.+Kimauthor=A.+Cuestaauthor=F.+R.+Santoriauthor=J.+J.+Lafailleauthor=H.+E.+Xuauthor=D.+Y.+Ginauthor=F.+Rastinejadauthor=D.+R.+Littman&title=Digoxin+and+its+derivatives+suppress+TH17+cell+differentiation+by+antagonizing+ROR%CE%B3t+activity&doi=10.1038%2Fnature09978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Digoxin and its derivatives suppress TH17 cell differentiation by antagonizing RORγt activity</span></div><div class="casAuthors">Huh, Jun R.; Leung, Monica W. L.; Huang, Pengxiang; Ryan, Daniel A.; Krout, Michael R.; Malapaka, Raghu R. V.; Chow, Jonathan; Manel, Nicolas; Ciofani, Maria; Kim, Sangwon V.; Cuesta, Adolfo; Santori, Fabio R.; Lafaille, Juan J.; Xu, H. Eric; Gin, David Y.; Rastinejad, Fraydoon; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">472</span>
        (<span class="NLM_cas:issue">7344</span>),
    <span class="NLM_cas:pages">486-490</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD4+ T helper lymphocytes that express interleukin-17 (TH17 cells) have crit. roles in mouse models of autoimmunity, and there is mounting evidence that they also influence inflammatory processes in humans.  Genome-wide assocn. studies in humans have linked genes involved in TH17 cell differentiation and function with susceptibility to Crohn's disease, rheumatoid arthritis and psoriasis.  Thus, the pathway towards differentiation of TH17 cells and, perhaps, of related innate lymphoid cells with similar effector functions, is an attractive target for therapeutic applications.  Mouse and human TH17 cells are distinguished by expression of the retinoic acid receptor-related orphan nuclear receptor RORγt, which is required for induction of IL-17 transcription and for the manifestation of TH17-dependent autoimmune disease in mice.  By performing a chem. screen with an insect cell-based reporter system, we identified the cardiac glycoside digoxin as a specific inhibitor of RORγt transcriptional activity.  Digoxin inhibited murine TH17 cell differentiation without affecting differentiation of other T cell lineages and was effective in delaying the onset and reducing the severity of autoimmune disease in mice.  At high concns., digoxin is toxic for human cells, but non-toxic synthetic derivs. 20,22-dihydrodigoxin-21,23-diol and digoxin-21-salicylidene specifically inhibited induction of IL-17 in human CD4+ T cells.  Using these small-mol. compds., we demonstrate that RORγt is important for the maintenance of IL-17 expression in mouse and human effector T cells.  These data indicate that derivs. of digoxin can be used as chem. templates for the development of RORγt-targeted therapeutic agents that attenuate inflammatory lymphocyte function and autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZeBtm9HrHi7Vg90H21EOLACvtfcHk0ljO1ig8xqUzwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvVOisr8%253D&md5=d25efc57bc2849ed23456c95c59cc006</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature09978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09978%26sid%3Dliteratum%253Aachs%26aulast%3DHuh%26aufirst%3DJ.%2BR.%26aulast%3DLeung%26aufirst%3DM.%2BW.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DRyan%26aufirst%3DD.%2BA.%26aulast%3DKrout%26aufirst%3DM.%2BR.%26aulast%3DMalapaka%26aufirst%3DR.%2BR.%26aulast%3DChow%26aufirst%3DJ.%26aulast%3DManel%26aufirst%3DN.%26aulast%3DCiofani%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%2BV.%26aulast%3DCuesta%26aufirst%3DA.%26aulast%3DSantori%26aufirst%3DF.%2BR.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DH.%2BE.%26aulast%3DGin%26aufirst%3DD.%2BY.%26aulast%3DRastinejad%26aufirst%3DF.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DDigoxin%2520and%2520its%2520derivatives%2520suppress%2520TH17%2520cell%2520differentiation%2520by%2520antagonizing%2520ROR%25CE%25B3t%2520activity%26jtitle%3DNature%26date%3D2011%26volume%3D472%26spage%3D486%26epage%3D490%26doi%3D10.1038%2Fnature09978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solt, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conkright, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Istrate, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Ordonez, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span> <span> </span><span class="NLM_article-title">The benzenesulfoamide T0901317 [N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide] is a novel retinoic acid receptor-related orphan receptor-alpha/gamma inverse agonist</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1124/mol.109.060905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1124%2Fmol.109.060905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=19887649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFeitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2010&pages=228-236&author=N.+Kumarauthor=L.+A.+Soltauthor=J.+J.+Conkrightauthor=Y.+Wangauthor=M.+A.+Istrateauthor=S.+A.+Busbyauthor=R.+D.+Garcia-Ordonezauthor=T.+P.+Burrisauthor=P.+R.+Griffin&title=The+benzenesulfoamide+T0901317+%5BN-%282%2C2%2C2-trifluoroethyl%29-N-%5B4-%5B2%2C2%2C2-trifluoro-1-hydroxy-1-%28trifluoromethyl%29ethyl%5Dphenyl%5D-benzenesulfonamide%5D+is+a+novel+retinoic+acid+receptor-related+orphan+receptor-alpha%2Fgamma+inverse+agonist&doi=10.1124%2Fmol.109.060905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The benzenesulfoamide T0901317 [N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide] is a novel retinoic acid receptor-related orphan receptor-α/γ inverse agonist</span></div><div class="casAuthors">Kumar, Naresh; Solt, Laura A.; Conkright, Juliana J.; Wang, Yongjun; Istrate, Monica A.; Busby, Scott A.; Garcia-Ordonez, Ruben D.; Burris, Thomas P.; Griffin, Patrick R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">228-236</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptors (RORs) regulate a variety of physiol. processes including hepatic gluconeogenesis, lipid metab., circadian rhythm, and immune function.  Here we present the first high-affinity synthetic ligand for both RORα and RORγ.  In a screen against all 48 human nuclear receptors, the benzenesulfonamide liver X receptor (LXR) agonist N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide (T0901317) inhibited transactivation activity of RORα and RORγ but not RORβ.  T0901317 was found to directly bind to RORα and RORγ with high affinity (Ki = 132 and 51 nM, resp.), resulting in the modulation of the receptor's ability to interact with transcriptional cofactor proteins.  T0901317 repressed RORα/γ-dependent transactivation of ROR-responsive reporter genes and in HepG2 cells reduced recruitment of steroid receptor coactivator-2 by RORα at an endogenous ROR target gene (G6Pase).  Using small interference RNA, we demonstrate that repression of the gluconeogenic enzyme glucose-6-phosphatase in HepG2 cells by T0901317 is ROR-dependent and is not due to the compd.'s LXR activity.  In summary, T0901317 represents a novel chem. probe to examine RORα/γ function and an excellent starting point for the development of ROR selective modulators.  More importantly, our results demonstrate that small mols. can be used to target the RORs for therapeutic intervention in metabolic and immune disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1FxGyT5l2qbVg90H21EOLACvtfcHk0ljsLYvWwjivsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFeitLk%253D&md5=04aaf2bdef4b9d64340e221f85838665</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.060905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.060905%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DSolt%26aufirst%3DL.%2BA.%26aulast%3DConkright%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DIstrate%26aufirst%3DM.%2BA.%26aulast%3DBusby%26aufirst%3DS.%2BA.%26aulast%3DGarcia-Ordonez%26aufirst%3DR.%2BD.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26atitle%3DThe%2520benzenesulfoamide%2520T0901317%2520%255BN-%25282%252C2%252C2-trifluoroethyl%2529-N-%255B4-%255B2%252C2%252C2-trifluoro-1-hydroxy-1-%2528trifluoromethyl%2529ethyl%255Dphenyl%255D-benzenesulfonamide%255D%2520is%2520a%2520novel%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-alpha%252Fgamma%2520inverse%2520agonist%26jtitle%3DMol.%2520Pharmacol.%26date%3D2010%26volume%3D77%26spage%3D228%26epage%3D236%26doi%3D10.1124%2Fmol.109.060905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">How many drug targets are there?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=How+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0ljsLYvWwjivsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DHow%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leijten-van de Gevel, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, R. M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Allosteric small molecule modulators of nuclear receptors</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>485</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2019.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.mce.2019.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30703487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVWhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2019&pages=20-34&author=F.+A.+Meijerauthor=I.+A.+Leijten-van+de+Gevelauthor=R.+M.+J.+M.+de+Vriesauthor=L.+Brunsveld&title=Allosteric+small+molecule+modulators+of+nuclear+receptors&doi=10.1016%2Fj.mce.2019.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric small molecule modulators of nuclear receptors</span></div><div class="casAuthors">Meijer, Femke A.; Leijten-van de Gevel, Iris A.; de Vries, Rens M. J. M.; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-34</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear Receptors (NRs) are multi-domain proteins, whose natural regulation occurs via ligands for a classical, orthosteric, binding pocket and via intra- and inter-domain allosteric mechanisms.  Allosteric modulation of NRs via synthetic small mols. has recently emerged as an interesting entry to address the need for small mols. targeting NRs in pathol., via novel modes of action and with beneficial profiles.  In this review the general concept of allosteric modulation in drug discovery is first discussed, serving as a background and inspiration for NRs.  Subsequently, the review focuses on examples of small mols. that allosterically modulate NRs, with a strong focus on structural information and the ligand binding domain.  Recently discovered nanomolar potent allosteric site NR modulators are catapulting allosteric targeting of NRs to the center of attention.  The obtained insights serve as a basis for recommendations for the next steps to take in allosteric small mol. targeting of NRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsuqvFC3-gY7Vg90H21EOLACvtfcHk0ljsLYvWwjivsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVWhs7k%253D&md5=a76ceb455cdf901bb7d7dae77686afc9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2019.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2019.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DMeijer%26aufirst%3DF.%2BA.%26aulast%3DLeijten-van%2Bde%2BGevel%26aufirst%3DI.%2BA.%26aulast%3Dde%2BVries%26aufirst%3DR.%2BM.%2BJ.%2BM.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DAllosteric%2520small%2520molecule%2520modulators%2520of%2520nuclear%2520receptors%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2019%26volume%3D485%26spage%3D20%26epage%3D34%26doi%3D10.1016%2Fj.mce.2019.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-J.</span></span> <span> </span><span class="NLM_article-title">Non-canonical modulators of nuclear receptors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4157</span>– <span class="NLM_lpage">4164</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.07.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2016.07.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27503683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlaju7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4157-4164&author=C.+M.+Ticeauthor=Y.-J.+Zheng&title=Non-canonical+modulators+of+nuclear+receptors&doi=10.1016%2Fj.bmcl.2016.07.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Non-canonical modulators of nuclear receptors</span></div><div class="casAuthors">Tice, Colin M.; Zheng, Ya-Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4157-4164</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Like G protein-coupled receptors (GPCRs) and protein kinases, nuclear receptors (NRs) are a rich source of pharmaceutical targets.  Over 80 NR-targeting drugs have been approved for 18 NRs.  The focus of drug discovery in NRs has hitherto been on identifying ligands that bind to the canonical ligand binding pockets of the C-terminal ligand binding domains (LBDs).  Due to the development of drug resistance and selectivity concerns, there has been considerable interest in exploring other, non-canonical ligand binding sites.  Unfortunately, the potencies of compds. binding at other sites have generally not been sufficient for clin. development.  However, the situation has changed dramatically over the last 3 years, as compds. with sufficient potency have been reported for several NR targets.  Here we review recent developments in this area from a medicinal chem. point of view in the hope of stimulating further interest in this area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopu9N4QDGsXrVg90H21EOLACvtfcHk0li7DKrPylWlyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlaju7bN&md5=5677a03f0e607260d148c48e24d49110</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.07.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.07.067%26sid%3Dliteratum%253Aachs%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DZheng%26aufirst%3DY.-J.%26atitle%3DNon-canonical%2520modulators%2520of%2520nuclear%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4157%26epage%3D4164%26doi%3D10.1016%2Fj.bmcl.2016.07.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayne, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J.</span></span> <span> </span><span class="NLM_article-title">Not picking pockets: nuclear receptor alternate-site modulators (NRAMs)</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">683</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1210/me.2009-0362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1210%2Fme.2009-0362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=19933380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVOhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=683-95&author=T.+W.+Mooreauthor=C.+G.+Mayneauthor=J.+Katzenellenbogen&title=Not+picking+pockets%3A+nuclear+receptor+alternate-site+modulators+%28NRAMs%29&doi=10.1210%2Fme.2009-0362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Not picking pockets: nuclear receptor alternate-site modulators (NRAMs)</span></div><div class="casAuthors">Moore, Terry W.; Mayne, Christopher G.; Katzenellenbogen, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-695</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Because of their central importance in gene regulation and mediating the actions of many hormones, the nuclear receptors (NRs) have long been recognized as very important biol. and pharmaceutical targets.  Of all the surfaces available on a given NR, the singular site for regulation of receptor activity has almost invariably been the ligand-binding pocket of the receptor, the site where agonists, antagonists, and selective NR modulators interact.  With our increasing understanding of the multiple mol. components involved in NR action, researchers have recently begun to look to addnl. interaction sites on NRs for regulating their activities by novel mechanisms.  The alternate NR-assocd. interaction sites that have been targeted include the coactivator-binding groove and allosteric sites in the ligand-binding domain, the zinc fingers of the DNA-binding domain, and the NR response element in DNA.  The studies thus far have been performed with the estrogen receptors, the androgen receptor (AR), the thyroid hormone receptors, and the pregnane X receptor.  Phenotypic and conformation-based screens have also identified small mol. modulators that are believed to function through the NRs but have, as yet, unknown sites and mechanisms of action.  The rewards from investigation of these NR alternate-site modulators should be the discovery of new therapeutic approaches and novel agents for regulating the activities of these important NR proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVtxERmnC5SLVg90H21EOLACvtfcHk0li7DKrPylWlyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVOhsb4%253D&md5=cd35a2ad9714316347764e692455cf65</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fme.2009-0362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2009-0362%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DT.%2BW.%26aulast%3DMayne%26aufirst%3DC.%2BG.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%26atitle%3DNot%2520picking%2520pockets%253A%2520nuclear%2520receptor%2520alternate-site%2520modulators%2520%2528NRAMs%2529%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D683%26epage%3D95%26doi%3D10.1210%2Fme.2009-0362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheepstra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Almen, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piesvaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutilek, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Eenennaam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagpal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correll, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oubrie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Identification of an allosteric binding site for RORγt inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8833</span>, <span class="refDoi"> DOI: 10.1038/ncomms9833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fncomms9833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=26640126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKit7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8833&author=M.+Scheepstraauthor=S.+Leysenauthor=G.+C.+van+Almenauthor=J.+R.+Millerauthor=J.+Piesvauxauthor=V.+Kutilekauthor=H.+van+Eenennaamauthor=H.+Zhangauthor=K.+Barrauthor=S.+Nagpalauthor=S.+M.+Soissonauthor=M.+Kornienkoauthor=K.+Wileyauthor=N.+Elsenauthor=S.+Sharmaauthor=C.+C.+Correllauthor=B.+W.+Trotterauthor=M.+van+der+Steltauthor=A.+Oubrieauthor=C.+Ottmannauthor=G.+Parthasarathyauthor=L.+Brunsveld&title=Identification+of+an+allosteric+binding+site+for+ROR%CE%B3t+inhibition&doi=10.1038%2Fncomms9833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an allosteric binding site for RORγt inhibition</span></div><div class="casAuthors">Scheepstra, Marcel; Leysen, Seppe; van Almen, Geert C.; Miller, J. Richard; Piesvaux, Jennifer; Kutilek, Victoria; van Eenennaam, Hans; Zhang, Hongjun; Barr, Kenneth; Nagpal, Sunil; Soisson, Stephen M.; Kornienko, Maria; Wiley, Kristen; Elsen, Nathaniel; Sharma, Sujata; Correll, Craig C.; Trotter, B. Wesley; van der Stelt, Mario; Oubrie, Arthur; Ottmann, Christian; Parthasarathy, Gopal; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8833/1-8833/10</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RORγt is crit. for the differentiation and proliferation of Th17 cells assocd. with several chronic autoimmune diseases.  We report the discovery of a novel allosteric binding site on the nuclear receptor RORγt.  Co-crystn. of the ligand binding domain (LBD) of RORγt with a series of small-mol. antagonists demonstrates occupancy of a previously unreported allosteric binding pocket.  Binding at this non-canonical site induces an unprecedented conformational reorientation of helix 12 in the RORγt LBD, which blocks cofactor binding.  The functional consequence of this allosteric ligand-mediated conformation is inhibition of function as evidenced by both biochem. and cellular studies.  RORγt function is thus antagonized in a manner molecularly distinct from that of previously described orthosteric RORγt ligands.  This brings forward an approach to target RORγt for the treatment of Th17-mediated autoimmune diseases.  The elucidation of an unprecedented modality of pharmacol. antagonism establishes a mechanism for modulation of nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcktcUUFl77Vg90H21EOLACvtfcHk0li7DKrPylWlyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKit7zL&md5=409b08232ecdd3f3e82fa660585c989d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fncomms9833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9833%26sid%3Dliteratum%253Aachs%26aulast%3DScheepstra%26aufirst%3DM.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3Dvan%2BAlmen%26aufirst%3DG.%2BC.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DPiesvaux%26aufirst%3DJ.%26aulast%3DKutilek%26aufirst%3DV.%26aulast%3Dvan%2BEenennaam%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBarr%26aufirst%3DK.%26aulast%3DNagpal%26aufirst%3DS.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DWiley%26aufirst%3DK.%26aulast%3DElsen%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DCorrell%26aufirst%3DC.%2BC.%26aulast%3DTrotter%26aufirst%3DB.%2BW.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26aulast%3DOubrie%26aufirst%3DA.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520an%2520allosteric%2520binding%2520site%2520for%2520ROR%25CE%25B3t%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8833%26doi%3D10.1038%2Fncomms9833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouvry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouix-Peter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciesielski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chantalat, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feret, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luzy, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musicki, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orfila, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parnet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taquet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoreau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennequin, L. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of phenoxyindazoles and phenylthioindazoles as RORγ inverse agonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5802</span>– <span class="NLM_lpage">5808</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2016.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27815118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslynsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5802-5808&author=G.+Ouvryauthor=C.+Bouix-Peterauthor=F.+Ciesielskiauthor=L.+Chantalatauthor=O.+Christinauthor=C.+Cominoauthor=D.+Duvertauthor=C.+Feretauthor=C.+S.+Harrisauthor=L.+Lamyauthor=A.+P.+Luzyauthor=B.+Musickiauthor=D.+Orfilaauthor=J.+Pascauauthor=V.+Parnetauthor=A.+Perrinauthor=R.+Pierreauthor=G.+Polgeauthor=C.+Raffinauthor=Y.+Rivalauthor=N.+Taquetauthor=E.+Thoreauauthor=L.+F.+Hennequin&title=Discovery+of+phenoxyindazoles+and+phenylthioindazoles+as+ROR%CE%B3+inverse+agonists&doi=10.1016%2Fj.bmcl.2016.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of phenoxyindazoles and phenylthioindazoles as RORγ inverse agonists</span></div><div class="casAuthors">Ouvry, Gilles; Bouix-Peter, Claire; Ciesielski, Fabrice; Chantalat, Laurent; Christin, Olivier; Comino, Catherine; Duvert, Denis; Feret, Christophe; Harris, Craig S.; Lamy, Laurent; Luzy, Anne-Pascale; Musicki, Branislav; Orfila, Danielle; Pascau, Jonathan; Parnet, Veronique; Perrin, Agnes; Pierre, Romain; Polge, Gaelle; Raffin, Catherine; Rival, Yves; Taquet, Nathalie; Thoreau, Etienne; Hennequin, Laurent F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5802-5808</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeting the IL17 pathway and more specifically the nuclear receptor RORγ is thought to be beneficial in multiple skin disorders.  The Letter describes the discovery of phenoxyindazoles and thiophenoxy indazoles as potent RORγ inverse agonists.  Optimization of the potency and efforts to mitigate the phototoxic liability of the series are presented.  Finally, crystn. of the lead compd. revealed that the series bound to an allosteric site of the nuclear receptor.  Such compds. could be useful as tool compds. for understanding the impact of topical treatment on skin disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX7Dguvi_h8LVg90H21EOLACvtfcHk0lhroVwAwfoKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslynsr%252FM&md5=cd3aa2c246cfa35c98306096d823dccc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DOuvry%26aufirst%3DG.%26aulast%3DBouix-Peter%26aufirst%3DC.%26aulast%3DCiesielski%26aufirst%3DF.%26aulast%3DChantalat%26aufirst%3DL.%26aulast%3DChristin%26aufirst%3DO.%26aulast%3DComino%26aufirst%3DC.%26aulast%3DDuvert%26aufirst%3DD.%26aulast%3DFeret%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DC.%2BS.%26aulast%3DLamy%26aufirst%3DL.%26aulast%3DLuzy%26aufirst%3DA.%2BP.%26aulast%3DMusicki%26aufirst%3DB.%26aulast%3DOrfila%26aufirst%3DD.%26aulast%3DPascau%26aufirst%3DJ.%26aulast%3DParnet%26aufirst%3DV.%26aulast%3DPerrin%26aufirst%3DA.%26aulast%3DPierre%26aufirst%3DR.%26aulast%3DPolge%26aufirst%3DG.%26aulast%3DRaffin%26aufirst%3DC.%26aulast%3DRival%26aufirst%3DY.%26aulast%3DTaquet%26aufirst%3DN.%26aulast%3DThoreau%26aufirst%3DE.%26aulast%3DHennequin%26aufirst%3DL.%2BF.%26atitle%3DDiscovery%2520of%2520phenoxyindazoles%2520and%2520phenylthioindazoles%2520as%2520ROR%25CE%25B3%2520inverse%2520agonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5802%26epage%3D5808%26doi%3D10.1016%2Fj.bmcl.2016.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIsaac, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce-Shaikh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristescu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudgett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piers, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltus, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehmet, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aicher, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawwari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correll, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of RORγT skews TCRα gene rearrangement and limits T cell repertoire diversity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3206</span>– <span class="NLM_lpage">3218</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.11.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.celrep.2016.11.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28009290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnsrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=3206-3218&author=Y.+Guoauthor=K.+D.+MacIsaacauthor=Y.+Chenauthor=R.+J.+Millerauthor=R.+Jainauthor=B.+Joyce-Shaikhauthor=H.+Fergusonauthor=I.+M.+Wangauthor=R.+Cristescuauthor=J.+Mudgettauthor=L.+Engstromauthor=K.+J.+Piersauthor=G.+A.+Baltusauthor=K.+Barrauthor=H.+Zhangauthor=H.+Mehmetauthor=L.+G.+Hegdeauthor=X.+Huauthor=L.+L.+Carterauthor=T.+D.+Aicherauthor=G.+Glickauthor=D.+Zallerauthor=A.+Hawwariauthor=C.+C.+Correllauthor=D.+C.+Jonesauthor=D.+J.+Cua&title=Inhibition+of+ROR%CE%B3T+skews+TCR%CE%B1+gene+rearrangement+and+limits+T+cell+repertoire+diversity&doi=10.1016%2Fj.celrep.2016.11.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RORγT Skews TCRα Gene Rearrangement and Limits T Cell Repertoire Diversity</span></div><div class="casAuthors">Guo, Yanxia; MacIsaac, Kenzie D.; Chen, Yi; Miller, Richard J.; Jain, Renu; Joyce-Shaikh, Barbara; Ferguson, Heidi; Wang, I-Ming; Cristescu, Razvan; Mudgett, John; Engstrom, Laura; Piers, Kyle J.; Baltus, Gretchen A.; Barr, Kenneth; Zhang, Hongjun; Mehmet, Huseyin; Hegde, Laxminarayan G.; Hu, Xiao; Carter, Laura L.; Aicher, Thomas D.; Glick, Gary; Zaller, Dennis; Hawwari, Abbas; Correll, Craig C.; Jones, Dallas C.; Cua, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3206-3218</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent studies have elucidated the mol. mechanism of RORγT transcriptional regulation of Th17 differentiation and function.  RORγT was initially identified as a transcription factor required for thymopoiesis by maintaining survival of CD4+CD8+ (DP) thymocytes.  While RORγ antagonists are currently being developed to treat autoimmunity, it remains unclear how RORγT inhibition may impact thymocyte development.  In this study, we show that in addn. to regulating DP thymocytes survival, RORγT also controls genes that regulate thymocyte migration, proliferation, and T cell receptor (TCR)α selection.  Strikingly, pharmacol. inhibition of RORγ skews TCRα gene rearrangement, limits T cell repertoire diversity, and inhibits development of autoimmune encephalomyelitis.  Thus, targeting RORγT not only inhibits Th17 cell development and function but also fundamentally alters thymic-emigrant recognition of self and foreign antigens.  The anal. of RORγ inhibitors has allowed us to gain a broader perspective of the diverse function of RORγT and its impact on T cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS2YbVQ9dNS7Vg90H21EOLACvtfcHk0lhroVwAwfoKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnsrjP&md5=6b3766d0673b02fbe6a6b0cc1a10f4a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.11.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.11.073%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DMacIsaac%26aufirst%3DK.%2BD.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMiller%26aufirst%3DR.%2BJ.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DJoyce-Shaikh%26aufirst%3DB.%26aulast%3DFerguson%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DI.%2BM.%26aulast%3DCristescu%26aufirst%3DR.%26aulast%3DMudgett%26aufirst%3DJ.%26aulast%3DEngstrom%26aufirst%3DL.%26aulast%3DPiers%26aufirst%3DK.%2BJ.%26aulast%3DBaltus%26aufirst%3DG.%2BA.%26aulast%3DBarr%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMehmet%26aufirst%3DH.%26aulast%3DHegde%26aufirst%3DL.%2BG.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DAicher%26aufirst%3DT.%2BD.%26aulast%3DGlick%26aufirst%3DG.%26aulast%3DZaller%26aufirst%3DD.%26aulast%3DHawwari%26aufirst%3DA.%26aulast%3DCorrell%26aufirst%3DC.%2BC.%26aulast%3DJones%26aufirst%3DD.%2BC.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520ROR%25CE%25B3T%2520skews%2520TCR%25CE%25B1%2520gene%2520rearrangement%2520and%2520limits%2520T%2520cell%2520repertoire%2520diversity%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D17%26spage%3D3206%26epage%3D3218%26doi%3D10.1016%2Fj.celrep.2016.11.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule RORγt antagonists: one stone kills two birds</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.it.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.it.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28258823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFWns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=229-231&author=C.+Zhongauthor=J.+Zhu&title=Small-molecule+ROR%CE%B3t+antagonists%3A+one+stone+kills+two+birds&doi=10.1016%2Fj.it.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule RORγt Antagonists: One Stone Kills Two Birds</span></div><div class="casAuthors">Zhong, Chao; Zhu, Jinfang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">229-231</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Although small-mol. thymus-specific isoform of retinoic acid receptor-related orphan nuclear receptor γ (RORγt) antagonists suppressing interleukin (IL)-17-producing T helper (Th17) cells are widely reported, the effect of these mols. on other RORγt-expressing cells is unknown.  However, a new study reports that RORγt inhibition in CD4+CD8+ thymocytes resulted in skewed T cell repertoire, contributing to a redn. in the frequency of self-reactive T cells and resistance to autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ57AU79_MU7Vg90H21EOLACvtfcHk0lik8cbbDozyLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFWns7o%253D&md5=205d340b7c65ebb90d7a6a356e242927</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DSmall-molecule%2520ROR%25CE%25B3t%2520antagonists%253A%2520one%2520stone%2520kills%2520two%2520birds%26jtitle%3DTrends%2520Immunol.%26date%3D2017%26volume%3D38%26spage%3D229%26epage%3D231%26doi%3D10.1016%2Fj.it.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gege, C.</span></span> <span> </span><span class="NLM_article-title">Retinoid-related orphan receptor γ t modulators: comparison of Glenmark’s me-too patent application (WO2015008234) with the originator application from Merck Sharp and Dohme (WO2012106995)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1215</span>– <span class="NLM_lpage">1221</span>, <span class="refDoi"> DOI: 10.1517/13543776.2015.1065816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1517%2F13543776.2015.1065816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=26156084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWgsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1215-1221&author=C.+Gege&title=Retinoid-related+orphan+receptor+%CE%B3+t+modulators%3A+comparison+of+Glenmark%E2%80%99s+me-too+patent+application+%28WO2015008234%29+with+the+originator+application+from+Merck+Sharp+and+Dohme+%28WO2012106995%29&doi=10.1517%2F13543776.2015.1065816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Retinoid-related orphan receptor γ t modulators: comparison of Glenmark's me-too patent application (WO2015008234) with the originator application from Merck Sharp and Dohme (WO2012106995)</span></div><div class="casAuthors">Gege, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1215-1221</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Retinoic acid receptor-related orphan nuclear receptor γ t (RORγt) is a key transcription factor for the development of TH17 cells.  Inhibiting RORγt activity is thought to be beneficial in targeting a variety of inflammatory and autoimmune disorders.  In their patent application WO2015008234, Glenmark applied a scaffold-hop approach regarding to the originator application from Merck Sharp & Dohme (WO2012106995): a 6-membered annelated arom. moiety is replaced by an annelated 5-membered heteroaryl (exemplified and claimed is however only thiophene), resulting in potent RORγt inverse agonists with a thieno[3,2-b]pyrrole or thieno[3,2-c]pyrazole core.  Based on the patent disclosure, the novelty and utility of these me-too compds. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0QHWy40oFn7Vg90H21EOLACvtfcHk0lik8cbbDozyLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWgsrjE&md5=e2515513e802fe86494d4f2e0e315eb5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F13543776.2015.1065816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2015.1065816%26sid%3Dliteratum%253Aachs%26aulast%3DGege%26aufirst%3DC.%26atitle%3DRetinoid-related%2520orphan%2520receptor%2520%25CE%25B3%2520t%2520modulators%253A%2520comparison%2520of%2520Glenmark%25E2%2580%2599s%2520me-too%2520patent%2520application%2520%2528WO2015008234%2529%2520with%2520the%2520originator%2520application%2520from%2520Merck%2520Sharp%2520and%2520Dohme%2520%2528WO2012106995%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2015%26volume%3D25%26spage%3D1215%26epage%3D1221%26doi%3D10.1517%2F13543776.2015.1065816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smondyrev, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">PHASE: A new engine for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1007/s10822-006-9087-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1007%2Fs10822-006-9087-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17124629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCksbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=647-671&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=E.+H.+Knollauthor=S.+N.+Raoauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=PHASE%3A+A+new+engine+for+pharmacophore+perception%2C+3D+QSAR+model+development%2C+and+3D+database+screening%3A+1.+Methodology+and+preliminary+results&doi=10.1007%2Fs10822-006-9087-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PHASE: A new design for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Knoll, Eric H.; Rao, Shashidhar N.; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10/11</span>),
    <span class="NLM_cas:pages">647-671</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We introduce PHASE, a highly flexible system for common pharmacophore identification and assessment, 3D QSAR model development, and 3D database creation and searching.  The primary workflows and tasks supported by PHASE are described, and details of the underlying scientific methodologies are provided. Using results from previously published investigations, PHASE is compared directly to other ligand-based software for its ability to identify target pharmacophores, rationalize structure-activity data, and predict activities of external compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof5WS3oFk3MLVg90H21EOLACvtfcHk0lik8cbbDozyLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCksbvE&md5=512f55e6e3c3dfbae3fc94886482b69a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9087-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9087-6%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DRao%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DPHASE%253A%2520A%2520new%2520engine%2520for%2520pharmacophore%2520perception%252C%25203D%2520QSAR%2520model%2520development%252C%2520and%25203D%2520database%2520screening%253A%25201.%2520Methodology%2520and%2520preliminary%2520results%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D647%26epage%3D671%26doi%3D10.1007%2Fs10822-006-9087-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smondyrev, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S. N.</span></span> <span> </span><span class="NLM_article-title">PHASE: A novel approach to pharmacophore modeling and 3D database searching</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2006.00384.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1111%2Fj.1747-0285.2006.00384.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=16784462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=370-372&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=S.+N.+Rao&title=PHASE%3A+A+novel+approach+to+pharmacophore+modeling+and+3D+database+searching&doi=10.1111%2Fj.1747-0285.2006.00384.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel approach to pharmacophore modeling and 3D database searching</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Rao, Shashidhar N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">370-372</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmacophore modeling and 3D database searching are now recognized as integral components of lead discovery and lead optimization, and the continuing need for improved pharmacophore-based tools has driven the development of PHASE.  PHASE pharmacophore and 3D QSAR models provide a means for rationalizing structure-activity data, predicting the activity of new compds., and efficiently extg. actives from a prototypical database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo50KpRHTGgAbVg90H21EOLACvtfcHk0lihfVpRfxsOpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D&md5=30d06fe99776e203eed35b0f774bbe64</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00384.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00384.x%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DRao%26aufirst%3DS.%2BN.%26atitle%3DPHASE%253A%2520A%2520novel%2520approach%2520to%2520pharmacophore%2520modeling%2520and%25203D%2520database%2520searching%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D370%26epage%3D372%26doi%3D10.1111%2Fj.1747-0285.2006.00384.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Asinex Corp.</span> Gold and platinum collections,
release: 1994–2004, update: 2016–10, <a href="http://www.asinex.com/libraries-html/libraries_gold_platinum-html" class="extLink">www.asinex.com/libraries-html/libraries_gold_platinum-html</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Asinex+Corp.+Gold+and+platinum+collections%2C%0Arelease%3A+1994%E2%80%932004%2C+update%3A+2016%E2%80%9310%2C+www.asinex.com%2Flibraries-html%2Flibraries_gold_platinum-html."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudrel-Grosay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Chiral mercaptoacetamides display enantioselective inhibition of histone deacetylase 6 and exhibit neuroprotection in cortical neuron models of oxidative stress</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201100522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1002%2Fcmdc.201100522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=22234885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=425-39&author=J.+H.+Kalinauthor=H.+Zhangauthor=S.+Gaudrel-Grosayauthor=G.+Vistoliauthor=A.+P.+Kozikowski&title=Chiral+mercaptoacetamides+display+enantioselective+inhibition+of+histone+deacetylase+6+and+exhibit+neuroprotection+in+cortical+neuron+models+of+oxidative+stress&doi=10.1002%2Fcmdc.201100522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral Mercaptoacetamides Display Enantioselective Inhibition of Histone Deacetylase 6 and Exhibit Neuroprotection in Cortical Neuron Models of Oxidative Stress</span></div><div class="casAuthors">Kalin, Jay H.; Zhang, Hankun; Gaudrel-Grosay, Sophie; Vistoli, Giulio; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-439</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mercaptoacetamide-based ligands have been designed as a new class of histone deacetylase (HDAC) inhibitors for possible use in the treatment of neurodegenerative diseases.  The thiol group of these compds. provides a key binding element for interaction with the catalytic zinc ion, and thus differs from the more typically employed hydroxamic acid based zinc binding groups.  Herein we disclose the chem. and biol. of some substituted mercaptoacetamides with the intention of increasing HDAC6 isoform selectivity while maintaining potency similar to their hydroxamic acid analogs.  The introduction of a stereocenter α to the thiol group was found to have a considerable impact on HDAC inhibitor potency.  These new compds. were also profiled for their therapeutic potential in an in vitro model of stress-induced neuronal injury and were found to act as nontoxic neuroprotective agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBURxV9LVkJbVg90H21EOLACvtfcHk0lihfVpRfxsOpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKitA%253D%253D&md5=245d434560a99b41c32f6532fd233e0b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100522%26sid%3Dliteratum%253Aachs%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGaudrel-Grosay%26aufirst%3DS.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DChiral%2520mercaptoacetamides%2520display%2520enantioselective%2520inhibition%2520of%2520histone%2520deacetylase%25206%2520and%2520exhibit%2520neuroprotection%2520in%2520cortical%2520neuron%2520models%2520of%2520oxidative%2520stress%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D425%26epage%3D39%26doi%3D10.1002%2Fcmdc.201100522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lihfVpRfxsOpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhfOcVN1ydPNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abreu, R. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froufe, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, P. O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queiroz, M. J. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, I. C. F. R.</span></span> <span> </span><span class="NLM_article-title">ChemT, an open-source software for building template-based chemical libraries</span>. <i>SAR QSAR Environ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1080/1062936X.2011.604097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1080%2F1062936X.2011.604097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21846264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVCrt7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=603-610&author=R.+M.+V.+Abreuauthor=H.+J.+C.+Froufeauthor=P.+O.+M.+Danielauthor=M.+J.+R.+P.+Queirozauthor=I.+C.+F.+R.+Ferreira&title=ChemT%2C+an+open-source+software+for+building+template-based+chemical+libraries&doi=10.1080%2F1062936X.2011.604097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">ChemT, an open-source software for building template-based chemical libraries</span></div><div class="casAuthors">Abreu, R. M. V.; Froufe, H. J. C.; Daniel, P. O. M.; Queiroz, M. J. R. P.; Ferreira, I. C. F. R.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">603-610</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1026-776X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In computational chem., vast quantities of compds. are generated, and there is a need for cheminformatic tools to efficiently build chem. compd. libraries.  Several software tools for drawing and editing compd. structures are available, but they lack options for automatic generation of chem. libraries.  We have implemented ChemT, an easy-to-use open-source software tool that automates the process of prepg. custom-made template-based chem. libraries.  ChemT automatically generates three-dimensional chem. libraries by inputting a chem. template and the functional groups of interest.  The graphical user interface of ChemT is self-explanatory, and a complete tutorial is provided.  Several file formats are accepted by ChemT, and it is possible to filter the generated compds. according to different physicochem. properties.  The compds. can be subject to force field minimization, and the resulting three-dimensional structures recorded on commonly used file formats.  ChemT may be a valuable tool for investigators interested in using in silico virtual screening tools, such as quant. structure-activity relationship modeling or mol. docking, in order to prioritize compds. for further chem. synthesis.  To demonstrate the usefulness of ChemT, we describe an example based on a thieno[3,2-b]pyridine template.  ChemT is available free of charge from our website at .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcglJxM_ucLVg90H21EOLACvtfcHk0lhfOcVN1ydPNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVCrt7zJ&md5=cfde2f5c3c4e4bdd570a5fe2be633cb7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1080%2F1062936X.2011.604097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1062936X.2011.604097%26sid%3Dliteratum%253Aachs%26aulast%3DAbreu%26aufirst%3DR.%2BM.%2BV.%26aulast%3DFroufe%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DDaniel%26aufirst%3DP.%2BO.%2BM.%26aulast%3DQueiroz%26aufirst%3DM.%2BJ.%2BR.%2BP.%26aulast%3DFerreira%26aufirst%3DI.%2BC.%2BF.%2BR.%26atitle%3DChemT%252C%2520an%2520open-source%2520software%2520for%2520building%2520template-based%2520chemical%2520libraries%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2011%26volume%3D22%26spage%3D603%26epage%3D610%26doi%3D10.1080%2F1062936X.2011.604097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, J.</span></span> <span> </span><span class="NLM_article-title">Preparation of 3-bromopropiolic esters: methyl and tert-butyl 3-bromopropiolates</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">212</span>, <span class="refDoi"> DOI: 10.15227/orgsyn.074.0212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.15227%2Forgsyn.074.0212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXjtFyqtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1997&pages=212&author=J.+Leroy&title=Preparation+of+3-bromopropiolic+esters%3A+methyl+and+tert-butyl+3-bromopropiolates&doi=10.15227%2Forgsyn.074.0212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of 3-bromopropiolic esters: methyl and tert-butyl 3-bromopropiolates</span></div><div class="casAuthors">Leroy, J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Syntheses</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">212-216</span>CODEN:
                <span class="NLM_cas:coden">ORSYAT</span>;
        ISSN:<span class="NLM_cas:issn">0078-6209</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">Brominating HC≡CCO2R (R = Me, Me3C) with NBS in Me2CO contg. AgNO3 gave 97% BrC≡CCO2R (same R).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojmQPJhHGjqrVg90H21EOLACvtfcHk0lhfOcVN1ydPNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtFyqtL8%253D&md5=e67ff62d7de1e80d2cb6660fa7a0d5cd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.15227%2Forgsyn.074.0212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15227%252Forgsyn.074.0212%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DJ.%26atitle%3DPreparation%2520of%25203-bromopropiolic%2520esters%253A%2520methyl%2520and%2520tert-butyl%25203-bromopropiolates%26jtitle%3DOrg.%2520Synth.%26date%3D1997%26volume%3D74%26spage%3D212%26doi%3D10.15227%2Forgsyn.074.0212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oakdale, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sit, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Ruthenium-catalyzed cycloadditions of 1-haloalkynes with nitrile oxides and organic azides: synthesis of 4-haloisoxazoles and 5-halotriazoles</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">11101</span>– <span class="NLM_lpage">11110</span>, <span class="refDoi"> DOI: 10.1002/chem.201402559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1002%2Fchem.201402559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=25059647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1WksrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=11101-11110&author=J.+S.+Oakdaleauthor=R.+K.+Sitauthor=V.+V.+Fokin&title=Ruthenium-catalyzed+cycloadditions+of+1-haloalkynes+with+nitrile+oxides+and+organic+azides%3A+synthesis+of+4-haloisoxazoles+and+5-halotriazoles&doi=10.1002%2Fchem.201402559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ruthenium-Catalyzed Cycloadditions of 1-Haloalkynes with Nitrile Oxides and Organic Azides: Synthesis of 4-Haloisoxazoles and 5-Halotriazoles</span></div><div class="casAuthors">Oakdale, James S.; Sit, Rakesh K.; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">11101-11110</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">(Cyclopentadienyl)(cyclooctadiene) ruthenium(II) chloride [CpRuCl(cod)] catalyzes the reaction between nitrile oxides and electronically deficient 1-choro-, 1-bromo-, and 1-iodoalkynes leading to 4-haloisoxazoles.  Org. azides are also suitable 1,3-dipoles, resulting in 5-halo-1,2,3-triazoles.  These air-tolerant reactions can be performed at room temp. with 1.25 equiv of the resp. 1,3-dipole relative to the alkyne component.  Reactive 1-haloalkynes include propiolic amides, esters, ketones, and phosphonates.  Post-functionalization of the halogenated azole products can be accomplished by using palladium-catalyzed cross-coupling reactions and by manipulation of reactive amide groups.  The lack of catalysis obsd. with [Cp*RuCl(cod)] (Cp*=pentamethylcyclopentadienyl) is attributed to steric demands of the Cp* (η5-C5Me5) ligand in comparison to the parent Cp (η5-C5H5).  This hypothesis is supported by the poor reactivity of [(η5-C5Me4CF3)RuCl(cod)], which serves as a an isosteric mimic of Cp* and as an isoelectronic analog of Cp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLjNQBLY8HrLVg90H21EOLACvtfcHk0li4gKKw1RlX_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1WksrfO&md5=a469ce8399cb624a3cad71869b869485</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fchem.201402559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201402559%26sid%3Dliteratum%253Aachs%26aulast%3DOakdale%26aufirst%3DJ.%2BS.%26aulast%3DSit%26aufirst%3DR.%2BK.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DRuthenium-catalyzed%2520cycloadditions%2520of%25201-haloalkynes%2520with%2520nitrile%2520oxides%2520and%2520organic%2520azides%253A%2520synthesis%2520of%25204-haloisoxazoles%2520and%25205-halotriazoles%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D11101%26epage%3D11110%26doi%3D10.1002%2Fchem.201402559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckman, D. O.</span>; <span class="NLM_string-name">Nicholas, J. B.</span>; <span class="NLM_string-name">Emayan, K.</span>; <span class="NLM_string-name">Seiwert, S. D.</span>; <span class="NLM_string-name">Yuan, S.</span></span> <span> </span><span class="NLM_article-title">Lysophosphatidic acid receptor antagonists</span>. International patent <span class="NLM_patent">WO/2014/113485</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+O.+Buckman&author=J.+B.+Nicholas&author=K.+Emayan&author=S.+D.+Seiwert&author=S.+Yuan&title=Lysophosphatidic+acid+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBuckman%26aufirst%3DD.%2BO.%26atitle%3DLysophosphatidic%2520acid%2520receptor%2520antagonists%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkenburg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojdyr, M.</span></span> <span> </span><span class="NLM_article-title">CCP412: the new graphical user interface to the CCP4 program suite</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1107/S2059798317016035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798317016035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=29533233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFartLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=68-84&author=L.+Pottertonauthor=J.+Agirreauthor=C.+Ballardauthor=K.+Cowtanauthor=E.+Dodsonauthor=P.+R.+Evansauthor=H.+T.+Jenkinsauthor=R.+Keeganauthor=E.+Krissinelauthor=K.+Stevensonauthor=A.+Lebedevauthor=S.+J.+McNicholasauthor=R.+A.+Nichollsauthor=M.+Nobleauthor=N.+S.+Pannuauthor=C.+Rothauthor=G.+Sheldrickauthor=P.+Skubakauthor=J.+Turkenburgauthor=V.+Uskiauthor=F.+von+Delftauthor=D.+Watermanauthor=K.+Wilsonauthor=M.+Winnauthor=M.+Wojdyr&title=CCP412%3A+the+new+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS2059798317016035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">CCP4i2: the new graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Liz; Agirre, Jon; Ballard, Charles; Cowtan, Kevin; Dodson, Eleanor; Evans, Phil R.; Jenkins, Huw T.; Keegan, Ronan; Krissinel, Eugene; Stevenson, Kyle; Lebedev, Andrey; McNicholas, Stuart J.; Nicholls, Robert A.; Noble, Martin; Pannu, Navraj S.; Roth, Christian; Sheldrick, George; Skubak, Pavol; Turkenburg, Johan; Uski, Ville; von Delft, Frank; Waterman, David; Wilson, Keith; Winn, Martyn; Wojdyr, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">68-84</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, No. 4) software suite for macromol. structure detn. by X-ray crystallog. groups brings together many programs and libraries that, by means of well established conventions, interoperate effectively without adhering to strict design guidelines.  Because of this inherent flexibility, users are often presented with diverse, even divergent, choices for solving every type of problem.  Recently, CCP4 introduced CCP4i2, a modern graphical interface designed to help structural biologists to navigate the process of structure detn., with an emphasis on pipelining and the streamlined presentation of results.  In addn., CCP4i2 provides a framework for writing structure-soln. scripts that can be built up incrementally to create increasingly automatic procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ2hzpmyiYpLVg90H21EOLACvtfcHk0li4gKKw1RlX_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFartLY%253D&md5=b9d1efe5ebbe0236027a07dbe2556892</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS2059798317016035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317016035%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DAgirre%26aufirst%3DJ.%26aulast%3DBallard%26aufirst%3DC.%26aulast%3DCowtan%26aufirst%3DK.%26aulast%3DDodson%26aufirst%3DE.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DJenkins%26aufirst%3DH.%2BT.%26aulast%3DKeegan%26aufirst%3DR.%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DStevenson%26aufirst%3DK.%26aulast%3DLebedev%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DNoble%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRoth%26aufirst%3DC.%26aulast%3DSheldrick%26aufirst%3DG.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DJ.%26aulast%3DUski%26aufirst%3DV.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DWaterman%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DK.%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DWojdyr%26aufirst%3DM.%26atitle%3DCCP412%253A%2520the%2520new%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D68%26epage%3D84%26doi%3D10.1107%2FS2059798317016035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clabbers, M. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruene, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrahams, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span> <span> </span><span class="NLM_article-title">Electron diffraction data processing with DIALS</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1107/S2059798318007726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798318007726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=29872002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=506-518&author=M.+T.+B.+Clabbersauthor=T.+Grueneauthor=J.+M.+Parkhurstauthor=J.+P.+Abrahamsauthor=D.+G.+Waterman&title=Electron+diffraction+data+processing+with+DIALS&doi=10.1107%2FS2059798318007726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Electron diffraction data processing with DIALS</span></div><div class="casAuthors">Clabbers, Max T. B.; Gruene, Tim; Parkhurst, James M.; Abrahams, Jan Pieter; Waterman, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-518</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Electron diffraction is a relatively novel alternative to X-ray crystallog. for the structure detn. of macromols. from three-dimensional nanometer-sized crystals.  The continuous-rotation method of data collection has been adapted for the electron microscope.  However, there are important differences in geometry that must be considered for successful data integration.  The wavelength of electrons in a TEM is typically around 40 times shorter than that of X-rays, implying a nearly flat Ewald sphere, and consequently low diffraction angles and a high effective sample-to-detector distance.  Nevertheless, the DIALS software package can, with specific adaptations, successfully process continuous-rotation electron diffraction data.  Pathologies encountered specifically in electron diffraction make data integration more challenging.  Errors can arise from instrumentation, such as beam drift or distorted diffraction patterns from lens imperfections.  The diffraction geometry brings addnl. challenges such as strong correlation between lattice parameters and detector distance.  These issues are compounded if calibration is incomplete, leading to uncertainty in exptl. geometry, such as the effective detector distance and the rotation rate or direction.  Dynamic scattering, absorption, radiation damage and incomplete wedges of data are addnl. factors that complicate data processing.  Here, recent features of DIALS as adapted to electron diffraction processing are shown, including diagnostics for problematic diffraction geometry refinement, refinement of a smoothly varying beam model and corrections for distorted diffraction images.  These novel features, combined with the existing tools in DIALS, make data integration and refinement feasible for electron crystallog., even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ2xvp5nytBbVg90H21EOLACvtfcHk0li4gKKw1RlX_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM&md5=0ef3d54b8435ed67f31774547a35d263</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS2059798318007726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798318007726%26sid%3Dliteratum%253Aachs%26aulast%3DClabbers%26aufirst%3DM.%2BT.%2BB.%26aulast%3DGruene%26aufirst%3DT.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DAbrahams%26aufirst%3DJ.%2BP.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26atitle%3DElectron%2520diffraction%2520data%2520processing%2520with%2520DIALS%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D506%26epage%3D518%26doi%3D10.1107%2FS2059798318007726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span> <span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lhgZaE7g0pJQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title"><i>AceDRG:</i> a stereochemical description generator for ligands</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/S2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28177307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Grazulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+a+stereochemical+description+generator+for+ligands&doi=10.1107%2FS2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lhgZaE7g0pJQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGrazulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520a%2520stereochemical%2520description%2520generator%2520for%2520ligands%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhgZaE7g0pJQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title"><i>REFMAC</i>5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lhEdahOJsMQ0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span> <i>The PyMOL Molecular
Graphics System, version 2.2.3</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>, <span class="NLM_year">2000</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+version+2.2.3%3B+Schr%C3%B6dinger%2C%0ALLC%2C+2000."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%252C%2520version%25202.2.3%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Nannan Sun, Qiong Xie, Yongjun Dang, <span class="NLM_string-name hlFld-ContribAuthor">Yonghui Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Agonist Lock Touched and Untouched Retinoic Acid Receptor-Related Orphan Receptor-γt (RORγt) Inverse Agonists: Classification Based on the Molecular Mechanisms of Action. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02178%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAgonist%252BLock%252BTouched%252Band%252BUntouched%252BRetinoic%252BAcid%252BReceptor-Related%252BOrphan%252BReceptor-%2525CE%2525B3t%252B%252528ROR%2525CE%2525B3t%252529%252BInverse%252BAgonists%25253A%252BClassification%252BBased%252Bon%252Bthe%252BMolecular%252BMechanisms%252Bof%252BAction%26aulast%3DSun%26aufirst%3DNannan%26date%3D2021%26date%3D2020%26date%3D16122020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Femke A. Meijer, Annet O.W.M. Saris, Richard G. Doveston, Guido J.M. Oerlemans, Rens M.J.M. de Vries, Bente A. Somsen, Anke Unger, Bert Klebl, Christian Ottmann, Peter J. Cossar, <span class="NLM_string-name hlFld-ContribAuthor">Luc Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationship Studies of Trisubstituted Isoxazoles as Selective Allosteric Ligands for the Retinoic-Acid-Receptor-Related Orphan Receptor γt. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9238-9258. <a href="https://doi.org/10.1021/acs.jmedchem.1c00475" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationship%252BStudies%252Bof%252BTrisubstituted%252BIsoxazoles%252Bas%252BSelective%252BAllosteric%252BLigands%252Bfor%252Bthe%252BRetinoic-Acid-Receptor-Related%252BOrphan%252BReceptor%252B%2525CE%2525B3t%26aulast%3DMeijer%26aufirst%3DFemke%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16032021%26date%3D19052021%26volume%3D64%26issue%3D13%26spage%3D9238%26epage%3D9258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Femke A. Meijer, Maxime C. M. van den Oetelaar, Richard G. Doveston, Ella N. R. Sampers, <span class="NLM_string-name hlFld-ContribAuthor">Luc Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Occlusion of the RORγt Ligand Binding Pocket Allows Unambiguous Targeting of an Allosteric Site. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 631-639. <a href="https://doi.org/10.1021/acsmedchemlett.1c00029" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00029%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCovalent%252BOcclusion%252Bof%252Bthe%252BROR%2525CE%2525B3t%252BLigand%252BBinding%252BPocket%252BAllows%252BUnambiguous%252BTargeting%252Bof%252Ban%252BAllosteric%252BSite%26aulast%3DMeijer%26aufirst%3DFemke%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D17012021%26date%3D03032021%26date%3D08032021%26volume%3D12%26issue%3D4%26spage%3D631%26epage%3D639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Femke A. Meijer, Guido J.M. Oerlemans, <span class="NLM_string-name hlFld-ContribAuthor">Luc Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Orthosteric and Allosteric Dual Targeting of the Nuclear Receptor RORγt with a Bitopic Ligand. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (3)
                                     , 510-519. <a href="https://doi.org/10.1021/acschembio.0c00941" title="DOI URL">https://doi.org/10.1021/acschembio.0c00941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00941%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DOrthosteric%252Band%252BAllosteric%252BDual%252BTargeting%252Bof%252Bthe%252BNuclear%252BReceptor%252BROR%2525CE%2525B3t%252Bwith%252Ba%252BBitopic%252BLigand%26aulast%3DMeijer%26aufirst%3DFemke%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D06122020%26date%3D08022021%26date%3D17022021%26volume%3D16%26issue%3D3%26spage%3D510%26epage%3D519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masato  Tsuda</span>, <span class="hlFld-ContribAuthor ">Taiki  Morita</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Nakamura</span>. </span><span class="cited-content_cbyCitation_article-title">Suzuki-Miyaura cross-coupling of 3,4-disubstituted 5-bromoisoxazoles: An efficient access to trisubstituted isoxazoles. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2021,</strong> <em>75 </em>, 153185. <a href="https://doi.org/10.1016/j.tetlet.2021.153185" title="DOI URL">https://doi.org/10.1016/j.tetlet.2021.153185</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2021.153185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2021.153185%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DSuzuki-Miyaura%252Bcross-coupling%252Bof%252B3%25252C4-disubstituted%252B5-bromoisoxazoles%25253A%252BAn%252Befficient%252Baccess%252Bto%252Btrisubstituted%252Bisoxazoles%26aulast%3DTsuda%26aufirst%3DMasato%26date%3D2021%26volume%3D75%26spage%3D153185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Gege</span>. </span><span class="cited-content_cbyCitation_article-title">Retinoic acid-related orphan receptor gamma (RORγ) inverse agonists/antagonists for the treatment of inflammatory diseases – where are we presently?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em> </em><a href="https://doi.org/10.1080/17460441.2021.1948833" title="DOI URL">https://doi.org/10.1080/17460441.2021.1948833</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1948833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1948833%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DRetinoic%252Bacid-related%252Borphan%252Breceptor%252Bgamma%252B%252528ROR%2525CE%2525B3%252529%252Binverse%252Bagonists%25252Fantagonists%252Bfor%252Bthe%252Btreatment%252Bof%252Binflammatory%252Bdiseases%252B%2525E2%252580%252593%252Bwhere%252Bare%252Bwe%252Bpresently%25253F%26aulast%3DGege%26aufirst%3DChristian%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rens M. J. M.  de Vries</span>, <span class="hlFld-ContribAuthor ">Femke A.  Meijer</span>, <span class="hlFld-ContribAuthor ">Richard G.  Doveston</span>, <span class="hlFld-ContribAuthor ">Iris A.  Leijten-van de Gevel</span>, <span class="hlFld-ContribAuthor ">Luc  Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Cooperativity between the orthosteric and allosteric ligand binding sites of RORγt. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2021,</strong> <em>118 </em>
                                    (6)
                                     , e2021287118. <a href="https://doi.org/10.1073/pnas.2021287118" title="DOI URL">https://doi.org/10.1073/pnas.2021287118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2021287118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2021287118%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DCooperativity%252Bbetween%252Bthe%252Borthosteric%252Band%252Ballosteric%252Bligand%252Bbinding%252Bsites%252Bof%252BROR%2525CE%2525B3t%26aulast%3Dde%2BVries%26aufirst%3DRens%2BM.%2BJ.%2BM.%26date%3D2021%26date%3D2021%26volume%3D118%26issue%3D6%26spage%3De2021287118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mian  Huang</span>, <span class="hlFld-ContribAuthor ">Shelby  Bolin</span>, <span class="hlFld-ContribAuthor ">Hannah  Miller</span>, <span class="hlFld-ContribAuthor ">Ho Leung  Ng</span>. </span><span class="cited-content_cbyCitation_article-title">RORγ Structural Plasticity and Druggability. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (15)
                                     , 5329. <a href="https://doi.org/10.3390/ijms21155329" title="DOI URL">https://doi.org/10.3390/ijms21155329</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21155329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21155329%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DROR%2525CE%2525B3%252BStructural%252BPlasticity%252Band%252BDruggability%26aulast%3DHuang%26aufirst%3DMian%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D15%26spage%3D5329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iris A.  Leijten-van de Gevel</span>, <span class="hlFld-ContribAuthor ">Luc  Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Delineation of the molecular determinants of the unique allosteric binding site of the orphan nuclear receptor RORγt. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (27)
                                     , 9183-9191. <a href="https://doi.org/10.1074/jbc.RA120.013581" title="DOI URL">https://doi.org/10.1074/jbc.RA120.013581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA120.013581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA120.013581%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DDelineation%252Bof%252Bthe%252Bmolecular%252Bdeterminants%252Bof%252Bthe%252Bunique%252Ballosteric%252Bbinding%252Bsite%252Bof%252Bthe%252Borphan%252Bnuclear%252Breceptor%252BROR%2525CE%2525B3t%26aulast%3DLeijten-van%2Bde%2BGevel%26aufirst%3DIris%2BA.%26date%3D2020%26volume%3D295%26issue%3D27%26spage%3D9183%26epage%3D9191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rens M. J. M.  Vries</span>, <span class="hlFld-ContribAuthor ">Richard G.  Doveston</span>, <span class="hlFld-ContribAuthor ">Femke A.  Meijer</span>, <span class="hlFld-ContribAuthor ">Luc  Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidation of an Allosteric Mode of Action for a Thienopyrazole RORγt Inverse Agonist. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (7)
                                     , 561-565. <a href="https://doi.org/10.1002/cmdc.202000044" title="DOI URL">https://doi.org/10.1002/cmdc.202000044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000044%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DElucidation%252Bof%252Ban%252BAllosteric%252BMode%252Bof%252BAction%252Bfor%252Ba%252BThienopyrazole%252BROR%2525CE%2525B3t%252BInverse%252BAgonist%26aulast%3DVries%26aufirst%3DRens%2BM.%2BJ.%2BM.%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D7%26spage%3D561%26epage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Orthosteric and allosteric RORγt ligand binding sites are shown by overlay of the crystal structures of RORγt LBD in complex with orthosteric inverse agonist <b>2</b> (orange, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NB6">4NB6</a>) and allosteric inverse agonist <b>3</b> (blue, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>). The structures of the orthosteric inverse agonist <b>1</b> and allosteric inverse agonist <b>4</b> are also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. 3D Pharmacophore screening identifies a compound class with a novel isoxazole-based chemotype for experimental evaluation. The structural features of <b>3</b> incorporated into the pharmacophore hypothesis are indicated: orange = aromatic rings, green = hydrophobic groups, and red = anionic group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Trisubstituted Isoxazoles <b>7</b> and <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Key HMBC correlations used to confirm the regiochemistry of <b>10a</b> and <b>10b</b> are shown. The <sup>13</sup>C-NMR signals for the C-5 carbons are distinctively downfield at 175 and 173 ppm, respectively. Reagents and conditions: (a) NH<sub>2</sub>OH·HCl, NaOH (aq), EtOH, rt, 18 h, 83%; (b) NCS, DMF, 60 °C, 18 h, 86%; then (c) alkyne, NEt<sub>3</sub>, THF, 80 °C, 4 h, 69% (<b>10a</b>), 80% (<b>10b</b>); (d) LiOH, EtOH, H<sub>2</sub>O, 70 °C, 8 h, 84% (<b>11a</b>), 95% (<b>11b</b>); and (e) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) <i>tert</i>-butyl-4-amino benzoate, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, 6 h; and (iii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, 42% (<b>7</b>), 69% (<b>8</b>).</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of C-4 Isoxazole Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) NH<sub>2</sub>R, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, 6 h, 27–87%; (b) LiOH, MeOH, H<sub>2</sub>O, 70 °C, 8 h, 43–99%; (c) (i) SOCl<sub>2</sub>, 50 °C, 2 h; (ii) MeNH(OMe), NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 6 h; (iii) LiAlH<sub>4</sub>, THF, 0 °C, 30 min, 65%; (d) (i) ethyl-4-aminobenzoate, AcOH, MeOH, reflux, 24 h; (ii) NaCNBH<sub>3</sub>, MeOH, reflux, 12 h, 31%; (e) (i) DPPA, <i>t</i>-BuOH, 85 °C, 18 h; (ii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 8 h, 59%; (f) (i) monomethyl terephthalate, SOCl<sub>2</sub>, 50 °C, 2 h; (ii) <b>13</b>, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 76%; (g) (i) methyl-4-formyl benzoate, AcOH, MeOH, reflux, 24 h; (ii) NaCNBH<sub>3</sub>, MeOH, reflux, 18 h, 43%; and (h) methyl-4-(chlorosulfonyl)benzoate, pyridine, 60 °C, 24 h, 71%.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. In silico modeled docking pose of <b>25</b> (green) overlaid with crystal structure of RORγt with <b>3</b> (orange) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ">4YPQ</a>). For <b>25</b>, the potential additional hydrogen bond with the RORγt H4 backbone is indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Retrosynthetic Analysis of Trisubstituted Isoxazole <b>28</b> Allowing for Late-Stage Diversification</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Isoxazole C-5 Analogues <b>23–27</b>.<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>“R” groups are defined in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Reagents and conditions: (a) NBS, AgNO<sub>3</sub>, Me<sub>2</sub>C(O), rt, 20 h, 80%; (b) (i) NCS, DMF, 60 °C, 18 h, (ii) NEt<sub>3</sub>, THF, rt, 30 min, 85%; (c) THF, 80 °C, 4 h, <b>30a</b> 43%; (d) RB(pin), Pd(dppf)Cl<sub>2</sub>, DME, 85 °C, 8 h, 39–58%; (e) (i) LiAlH<sub>4</sub>, THF, 0 °C → rt, 2 h, then (ii) DMP, CH<sub>2</sub>Cl<sub>2</sub>, rt, 8 h, 51–96%; (f) <i>tert</i>-butyl-4-amino benzoate, MeOH, AcOH, reflux, 24 h then (ii) NaBH<sub>4</sub>, EtOH, 85 °C, 2–6 h, 16–24%; (iii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, <b>23</b>, <b>24</b>, <b>26</b>, 48–73%; and (g) methyl-4-amino benzoate, MeOH, AcOH, reflux, 24 h then (ii) NaBH<sub>4</sub>, MeOH, reflux, 2–4 h, 16–19%; and (iii) LiOH, MeOH, H<sub>2</sub>O, 70 °C, 8 h, <b>25</b>, 57%, <b>27</b>, 99%.</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical RORγt assay data for <b>25</b>, <b>3</b>, and <b>5</b>. (A) Dose–response curves from the TR-FRET coactivator recruitment assay; (B–E) dose–response curves from the competitive TR-FRET coactivator recruitment assay with fixed concentrations of cholesterol (0, 0.25, and 1.0 μM); and (F) dose–response curves from the ligand displacement HTRF assay using <b>44</b> (G) as an allosteric probe.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Co-crystal structure of RORγt with compound <b>25</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAL">6SAL</a>). (A) The tertiary structure of RORγt bound to <b>25</b> (stick representation). The final 2Fo–Fc electron density map of <b>25</b> is shown as an isomesh contoured at 1σ; (B) overlay of the crystal structure of RORγt bound to <b>25</b> and RORγt bound to <b>3</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C4O">5C4O</a>); and (C) zoom-in on the allosteric pocket of RORγt showing the interactions between <b>25</b> and the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/medium/jm9b01372_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. IL-17a mRNA expression in EL4 cells treated with ligand <b>3</b>, <b>25</b>, and <b>23</b> (10 μM, 24h) or DMSO. The level of IL-17a expression was normalized to that of the GAPDH expression. All data are expressed as the mean ± s.d. (<i>n</i> = 3). The relative gene expression was calculated by the 2<sup>–ΔΔCt</sup> (Livak) method using the DMSO control as calibrator.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01372/20200106/images/large/jm9b01372_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01372&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Yong</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span> <span> </span><span class="NLM_article-title">ROR-γ drives androgen receptor expression and represents a therapeutic target in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/nm.4070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnm.4070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27019329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFShurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=488-496&author=J.+Wangauthor=J.+X.+Zouauthor=X.+Xueauthor=D.+Caiauthor=Y.+Zhangauthor=Z.+Duanauthor=Q.+Xiangauthor=J.+C.+Yangauthor=M.+C.+Louieauthor=A.+D.+Borowskyauthor=A.+Gaoauthor=C.+P.+Evansauthor=K.+S.+Lamauthor=J.+Xuauthor=H.-J.+Kungauthor=R.+M.+Evansauthor=Yong+Xuauthor=H.-W.+Chen&title=ROR-%CE%B3+drives+androgen+receptor+expression+and+represents+a+therapeutic+target+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.4070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">ROR-γ drives androgen receptor expression and represents a therapeutic target in castration-resistant prostate cancer</span></div><div class="casAuthors">Wang, Junjian; Zou, June X.; Xue, Xiaoqian; Cai, Demin; Zhang, Yan; Duan, Zhijian; Xiang, Qiuping; Yang, Joy C.; Louie, Maggie C.; Borowsky, Alexander D.; Gao, Allen C.; Evans, Christopher P.; Lam, Kit S.; Xu, Jianzhen; Kung, Hsing-Jien; Evans, Ronald M.; Xu, Yong; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-496</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is overexpressed and hyperactivated in human castration-resistant prostate cancer (CRPC).  However, the determinants of AR overexpression in CRPC are poorly defined.  Here we show that retinoic acid receptor-related orphan receptor γ (ROR-γ) is overexpressed and amplified in metastatic CRPC tumors, and that ROR-γ drives AR expression in the tumors.  ROR-γ recruits nuclear receptor coactivator 1 and 3 (NCOA1 and NCOA3, also known as SRC-1 and SRC-3) to an AR-ROR response element (RORE) to stimulate AR gene transcription.  ROR-γ antagonists suppress the expression of both AR and its variant AR-V7 in prostate cancer (PCa) cell lines and tumors.  ROR-γ antagonists also markedly diminish genome-wide AR binding, H3K27ac abundance and expression of the AR target gene network.  Finally, ROR-γ antagonists suppressed tumor growth in multiple AR-expressing, but not AR-neg., xenograft PCa models, and they effectively sensitized CRPC tumors to enzalutamide, without overt toxicity, in mice.  Taken together, these results establish ROR-γ as a key player in CRPC by acting upstream of AR and as a potential therapeutic target for advanced PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX9ywlnlPbX7Vg90H21EOLACvtfcHk0lgbDAv4GdPunw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFShurY%253D&md5=3817be87b91c2f148987d35e2305365c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnm.4070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4070%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DLouie%26aufirst%3DM.%2BC.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DEvans%26aufirst%3DC.%2BP.%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKung%26aufirst%3DH.-J.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DXu%26aufirst%3DYong%26aulast%3DChen%26aufirst%3DH.-W.%26atitle%3DROR-%25CE%25B3%2520drives%2520androgen%2520receptor%2520expression%2520and%2520represents%2520a%2520therapeutic%2520target%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D488%26epage%3D496%26doi%3D10.1038%2Fnm.4070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loomes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of XY101, a potent, selective, and orally bioavailable RORγ inverse agonist for treatment of castration-resistant prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4716</span>– <span class="NLM_lpage">4730</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00327</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00327" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4716-4730&author=Y.+Zhangauthor=X.+Wuauthor=X.+Xueauthor=C.+Liauthor=J.+Wangauthor=R.+Wangauthor=C.+Zhangauthor=C.+Wangauthor=Y.+Shiauthor=L.+Zouauthor=Q.+Liauthor=Z.+Huangauthor=X.+Haoauthor=K.+Loomesauthor=D.+Wuauthor=H.-W.+Chenauthor=J.+Xuauthor=Y.+Xu&title=Discovery+and+characterization+of+XY101%2C+a+potent%2C+selective%2C+and+orally+bioavailable+ROR%CE%B3+inverse+agonist+for+treatment+of+castration-resistant+prostate+cancer&doi=10.1021%2Facs.jmedchem.9b00327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of XY101, a potent, selective, and orally bioavailable RORγ inverse agonist for treatment of castration-resistant prostate cancer</span></div><div class="casAuthors">Zhang, Yan; Wu, Xishan; Xue, Xiaoqian; Li, Chenchang; Wang, Junjian; Wang, Rui; Zhang, Cheng; Wang, Chao; Shi, Yudan; Zou, Lingjiao; Li, Qiu; Huang, Zenghong; Hao, Xiaojuan; Loomes, Kerry; Wu, Donghai; Chen, Hong-Wu; Xu, Jinxin; Xu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4716-4730</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the design, optimization, and biol. evaluation of nuclear receptor RORγ inverse agonists as therapeutic agents for prostate cancer treatment.  The most potent compd. 27(I) (designated as XY101) exhibited cellular activity with an IC50 value of 30 nM in a cell-based reporter gene assay with good selectivity against other nuclear receptor subtypes.  The cocrystal structure of I in complex with the RORγ ligand binding domain provided a solid structural basis for its antagonistic mechanism.  I potently inhibited cell growth, colony formation, and the expression of AR, AR-V7, and PSA.  I also exhibited good metabolic stability and a pharmacokinetic profile with oral bioavailability of 59% and a half-life of 7.3 h.  Notably, I demonstrated promising therapeutic effects with significant tumor growth inhibition in a prostate cancer xenograft model in mice.  The potent, selective, metabolically stable, and orally available RORγ inverse agonists represent a new class of compds. as potential therapeutics against prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUiAUjMO2c7Vg90H21EOLACvtfcHk0lh8t0unzwEXjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFSjtrw%253D&md5=5332cfeb5f08b7c695361af45ae68c60</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00327%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DLoomes%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DH.-W.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520XY101%252C%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520ROR%25CE%25B3%2520inverse%2520agonist%2520for%2520treatment%2520of%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4716%26epage%3D4730%26doi%3D10.1021%2Facs.jmedchem.9b00327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepelley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.cell.2006.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=16990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=1121-1133&author=I.+I.+Ivanovauthor=B.+S.+McKenzieauthor=L.+Zhouauthor=C.+E.+Tadokoroauthor=A.+Lepelleyauthor=J.+J.+Lafailleauthor=D.+J.+Cuaauthor=D.+R.+Littman&title=The+orphan+nuclear+receptor+ROR%CE%B3t+directs+the+differentiation+program+of+proinflammatory+IL-17%2B+T+helper+cells&doi=10.1016%2Fj.cell.2006.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span></div><div class="casAuthors">Ivanov, Ivaylo I.; McKenzie, Brent S.; Zhou, Liang; Tadokoro, Carlos E.; Lepelley, Alice; Lafaille, Juan J.; Cua, Daniel J.; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17-producing T lymphocytes have been recently shown to comprise a distinct lineage of proinflammatory T helper cells, termed Th17 cells, that are major contributors to autoimmune disease.  The authors show here that the orphan nuclear receptor RORγt is the key transcription factor that orchestrates the differentiation of this effector cell lineage.  RORγt induces transcription of the genes encoding IL-17 and the related cytokine IL-17F in naive CD4+ T helper cells and is required for their expression in response to IL-6 and TGF-β, the cytokines known to induce IL-17.  Th17 cells are constitutively present throughout the intestinal lamina propria, express RORγt, and are absent in mice deficient for RORγt or IL-6.  Mice with RORγt-deficient T cells have attenuated autoimmune disease and lack tissue-infiltrating Th17 cells.  Together, these studies suggest that RORγt is a key regulator of immune homeostasis and highlight its potential as a therapeutic target in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsJd6RyFqWrVg90H21EOLACvtfcHk0lh8t0unzwEXjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE&md5=4f5693603c30dca4f6a2d0994aeaf9b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DI.%2BI.%26aulast%3DMcKenzie%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTadokoro%26aufirst%3DC.%2BE.%26aulast%3DLepelley%26aufirst%3DA.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520orphan%2520nuclear%2520receptor%2520ROR%25CE%25B3t%2520directs%2520the%2520differentiation%2520program%2520of%2520proinflammatory%2520IL-17%252B%2520T%2520helper%2520cells%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D1121%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2006.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unutmaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The differentiation of human TH-17 cells requires transforming growth factor-β and induction of the nuclear receptor RORγt</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/ni.1610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fni.1610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=18454151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVWqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=641-649&author=N.+Manelauthor=D.+Unutmazauthor=D.+R.+Littman&title=The+differentiation+of+human+TH-17+cells+requires+transforming+growth+factor-%CE%B2+and+induction+of+the+nuclear+receptor+ROR%CE%B3t&doi=10.1038%2Fni.1610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The differentiation of human TH-17 cells requires transforming growth factor-β and induction of the nuclear receptor RORγt</span></div><div class="casAuthors">Manel, Nicolas; Unutmaz, Derya; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TH-17 cells are interleukin 17 (IL-17)-secreting CD4+ T helper cells involved in autoimmune disease and mucosal immunity.  In naive CD4+ T cells from mice, IL-17 is expressed in response to a combination of IL-6 or IL-21 and transforming growth factor-β (TGF-β) and requires induction of the nuclear receptor RORγt.  It has been suggested that the differentiation of human TH-17 cells is independent of TGF-β and thus differs fundamentally from that in mice.  The authors show here that TGF-β, IL-1β and IL-6, IL-21 or IL-23 in serum-free conditions were necessary and sufficient to induce IL-17 expression in naive human CD4+ T cells from cord blood.  TGF-β upregulated RORγt expression but simultaneously inhibited its ability to induce IL-17 expression.  Inflammatory cytokines relieved this inhibition and increased RORγt-directed IL-17 expression.  Other gene products detected in TH-17 cells after RORγt induction included the chemokine receptor CCR6, the IL-23 receptor, IL-17F and IL-26.  The authors' studies identify RORγt as having a central function in the differentiation of human TH-17 cells from naive CD4+ T cells and suggest that similar cytokine pathways are involved in this process in mice and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZBHk62Mj3R7Vg90H21EOLACvtfcHk0lipDwLQcVLa4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVWqurg%253D&md5=9c1406dfd60902657b1bd5231618cc70</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fni.1610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1610%26sid%3Dliteratum%253Aachs%26aulast%3DManel%26aufirst%3DN.%26aulast%3DUnutmaz%26aufirst%3DD.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520differentiation%2520of%2520human%2520TH-17%2520cells%2520requires%2520transforming%2520growth%2520factor-%25CE%25B2%2520and%2520induction%2520of%2520the%2520nuclear%2520receptor%2520ROR%25CE%25B3t%26jtitle%3DNat.%2520Immunol.%26date%3D2008%26volume%3D9%26spage%3D641%26epage%3D649%26doi%3D10.1038%2Fni.1610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akimzhanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panopoulos, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluns, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2007.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.immuni.2007.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=18164222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=29-39&author=X.+O.+Yangauthor=B.+P.+Pappuauthor=R.+Nurievaauthor=A.+Akimzhanovauthor=H.+S.+Kangauthor=Y.+Chungauthor=L.+Maauthor=B.+Shahauthor=A.+D.+Panopoulosauthor=K.+S.+Schlunsauthor=S.+S.+Watowichauthor=Q.+Tianauthor=A.+M.+Jettenauthor=C.+Dong&title=T+helper+17+lineage+differentiation+is+programmed+by+orphan+nuclear+receptors+ROR%CE%B1+and+ROR%CE%B3&doi=10.1016%2Fj.immuni.2007.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span></div><div class="casAuthors">Yang, Xuexian O.; Pappu, Bhanu P.; Nurieva, Roza; Akimzhanov, Askar; Kang, Hong Soon; Chung, Yeonseok; Ma, Li; Shah, Bhavin; Panopoulos, Athanasia D.; Schluns, Kimberly S.; Watowich, Stephanie S.; Tian, Qiang; Jetten, Anton M.; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T cell functional differentiation is mediated by lineage-specific transcription factors.  T helper 17 (Th17) has been recently identified as a distinct Th lineage mediating tissue inflammation.  Retinoic acid receptor-related orphan receptor γ (RORγ) was shown to regulate Th17 differentiation; RORγ deficiency, however, did not completely abolish Th17 cytokine expression.  Here, we report Th17 cells highly expressed another related nuclear receptor, RORα, induced by transforming growth factor-β and interleukin-6 (IL-6), which is dependent on signal transducer and activator of transcription 3.  Overexpression of RORα promoted Th17 differentiation, possibly through the conserved noncoding sequence 2 in II17-II17f locus.  RORα deficiency resulted in reduced IL-17 expression in vitro and in vivo.  Furthermore, RORα and RORγ coexpression synergistically led to greater Th17 differentiation.  Double deficiencies in RORα and RORγ globally impaired Th17 generation and completely protected mice against exptl. autoimmune encephalomyelitis.  Therefore, Th17 differentiation is directed by two lineage-specific nuclear receptors, RORα and RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBQRyY1XQpJbVg90H21EOLACvtfcHk0lipDwLQcVLa4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D&md5=436602649095b96d366c983addf526f7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2007.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2007.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DPappu%26aufirst%3DB.%2BP.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DAkimzhanov%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%2BS.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPanopoulos%26aufirst%3DA.%2BD.%26aulast%3DSchluns%26aufirst%3DK.%2BS.%26aulast%3DWatowich%26aufirst%3DS.%2BS.%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DJetten%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DT%2520helper%252017%2520lineage%2520differentiation%2520is%2520programmed%2520by%2520orphan%2520nuclear%2520receptors%2520ROR%25CE%25B1%2520and%2520ROR%25CE%25B3%26jtitle%3DImmunity%26date%3D2008%26volume%3D28%26spage%3D29%26epage%3D39%26doi%3D10.1016%2Fj.immuni.2007.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miossec, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolls, J. K.</span></span> <span> </span><span class="NLM_article-title">Targeting IL-17 and TH17 cells in chronic inflammation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">763</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1038/nrd3794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd3794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=23023676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqsb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=763-776&author=P.+Miossecauthor=J.+K.+Kolls&title=Targeting+IL-17+and+TH17+cells+in+chronic+inflammation&doi=10.1038%2Fnrd3794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting IL-17 and TH17 cells in chronic inflammation</span></div><div class="casAuthors">Miossec, Pierre; Kolls, Jay K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">763-776</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The key role of interleukin-17 (IL-17) and T helper 17 (TH17) cells in tissue inflammation, autoimmunity and host defense led to the exptl. targeting of these mols. in mouse models of diseases as well as in clin. settings.  Moreover, the demonstration that IL-17 and TH17 cells contribute to local and systemic aspects of disease pathogenesis, as well as the finding that the IL-17-TH17 cell pathway is regulated by IL-23, prompted the identification of inhibitors.  These inhibitors include biotechnol. products that target IL-23 as well as the leading member of the IL-17 family, IL-17A, and one of its receptors, IL-17 receptor A.  Several clin. trials of these inhibitors are underway, and pos. results have been obtained in psoriasis, rheumatoid arthritis and ankylosing spondylitis.  This Review focuses on the current knowledge of the IL-17-TH17 cell pathway to better understand the pos. as well as potential neg. consequences of targeting them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLAiqCF66JHrVg90H21EOLACvtfcHk0lipDwLQcVLa4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqsb7L&md5=0627e4086ae6e50881cc3de52f1a2752</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd3794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3794%26sid%3Dliteratum%253Aachs%26aulast%3DMiossec%26aufirst%3DP.%26aulast%3DKolls%26aufirst%3DJ.%2BK.%26atitle%3DTargeting%2520IL-17%2520and%2520TH17%2520cells%2520in%2520chronic%2520inflammation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D763%26epage%3D776%26doi%3D10.1038%2Fnrd3794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burkett, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">IL-17 blockade in psoriasis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1669</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.cell.2016.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27984714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWlsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1669&author=P.+R.+Burkettauthor=V.+K.+Kuchroo&title=IL-17+blockade+in+psoriasis&doi=10.1016%2Fj.cell.2016.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17 Blockade in Psoriasis</span></div><div class="casAuthors">Burkett, Patrick R.; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1669</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17A both directly induces and synergizes with other cytokines to promote autoimmune tissue inflammation.  Secukinumab and ixekizumab are monoclonal antibodies (mAb) that inhibit interleukin-17A.  These two agents were recently approved for treatment of psoriasis, and secukinumab is also approved for treatment of two spondyloarthropathies, psoriatic arthritis and ankylosing spondylitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobNyijIUaJ-rVg90H21EOLACvtfcHk0lgZPRoeI_aDpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWlsrrE&md5=e2d2918a30d5bf98cd5b405e51f171d4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DBurkett%26aufirst%3DP.%2BR.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DIL-17%2520blockade%2520in%2520psoriasis%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1669%26doi%3D10.1016%2Fj.cell.2016.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendolan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer-Gould, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strober, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannella, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klonowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oksenberg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raine, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinman, L.</span></span> <span> </span><span class="NLM_article-title">Gene-microarray analysis of multiple sclerosis lesions yields new targets validated in autoimmune encephalomyelitis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nm0502-500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnm0502-500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=11984595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1Kgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=500-508&author=C.+Lockauthor=G.+Hermansauthor=R.+Pedottiauthor=A.+Brendolanauthor=E.+Schadtauthor=H.+Garrenauthor=A.+Langer-Gouldauthor=S.+Stroberauthor=B.+Cannellaauthor=J.+Allardauthor=P.+Klonowskiauthor=A.+Austinauthor=N.+Ladauthor=N.+Kaminskiauthor=S.+J.+Galliauthor=J.+R.+Oksenbergauthor=C.+S.+Raineauthor=R.+Hellerauthor=L.+Steinman&title=Gene-microarray+analysis+of+multiple+sclerosis+lesions+yields+new+targets+validated+in+autoimmune+encephalomyelitis&doi=10.1038%2Fnm0502-500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Gene-microarray analysis of multiple sclerosis lesions yields new targets validated in autoimmune encephalomyelitis</span></div><div class="casAuthors">Lock, C.; Hermans, G.; Pedotti, R.; Brendolan, A.; Schadt, E.; Garren, H.; Langer-Gould, A.; Strober, S.; Cannella, B.; Allard, J.; Klonowski, P.; Austin, A.; Lad, N.; Kaminski, N.; Galli, S. J.; Oksenberg, J. R.; Raine, C. S.; Heller, R.; Steinman, L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">500-508</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Microarray anal. of multiple sclerosis (MS) lesions obtained at autopsy revealed increased transcripts of genes encoding inflammatory cytokines, particularly interleukin-6 and -17, interferon-γ and assocd. downstream pathways.  Comparison of two poles of MS pathol.-acute lesions with inflammation vs. 'silent' lesions without inflammation-revealed differentially transcribed genes.  Some products of these genes were chosen as targets for therapy of exptl. autoimmune encephalomyelitis (EAE) in mice.  Granulocyte colony-stimulating factor is upregulated in acute, but not in chronic, MS lesions, and the effect on ameliorating EAE is more pronounced in the acute phase, in contrast to knocking out the Ig Fc receptor common γ chain where the effect is greatest on chronic disease.  These results in EAE corroborate the microarray studies on MS lesions.  Large-scale anal. of transcripts in MS lesions elucidates new aspects of pathol. and opens possibilities for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPWj6FNx8JZ7Vg90H21EOLACvtfcHk0lgZPRoeI_aDpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1Kgur8%253D&md5=1e50114f238ec03ff0d534966273f245</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnm0502-500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0502-500%26sid%3Dliteratum%253Aachs%26aulast%3DLock%26aufirst%3DC.%26aulast%3DHermans%26aufirst%3DG.%26aulast%3DPedotti%26aufirst%3DR.%26aulast%3DBrendolan%26aufirst%3DA.%26aulast%3DSchadt%26aufirst%3DE.%26aulast%3DGarren%26aufirst%3DH.%26aulast%3DLanger-Gould%26aufirst%3DA.%26aulast%3DStrober%26aufirst%3DS.%26aulast%3DCannella%26aufirst%3DB.%26aulast%3DAllard%26aufirst%3DJ.%26aulast%3DKlonowski%26aufirst%3DP.%26aulast%3DAustin%26aufirst%3DA.%26aulast%3DLad%26aufirst%3DN.%26aulast%3DKaminski%26aufirst%3DN.%26aulast%3DGalli%26aufirst%3DS.%2BJ.%26aulast%3DOksenberg%26aufirst%3DJ.%2BR.%26aulast%3DRaine%26aufirst%3DC.%2BS.%26aulast%3DHeller%26aufirst%3DR.%26aulast%3DSteinman%26aufirst%3DL.%26atitle%3DGene-microarray%2520analysis%2520of%2520multiple%2520sclerosis%2520lesions%2520yields%2520new%2520targets%2520validated%2520in%2520autoimmune%2520encephalomyelitis%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D500%26epage%3D508%26doi%3D10.1038%2Fnm0502-500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duerr, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brant, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverberg, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhart, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regueiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassopoulos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Targan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kistner, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumm, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregersen, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolae, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span> <span> </span><span class="NLM_article-title">A genome-wide association study identifies IL23R as an inflammatory bowel disease gene</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>314</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">3</span>, <span class="refDoi"> DOI: 10.1126/science.1135245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1126%2Fscience.1135245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17068223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CntrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=2006&pages=1461-3&author=R.+H.+Duerrauthor=K.+D.+Taylorauthor=S.+R.+Brantauthor=J.+D.+Riouxauthor=M.+S.+Silverbergauthor=M.+J.+Dalyauthor=A.+H.+Steinhartauthor=C.+Abrahamauthor=M.+Regueiroauthor=A.+Griffithsauthor=T.+Dassopoulosauthor=A.+Bittonauthor=H.+Yangauthor=S.+Targanauthor=L.+W.+Dattaauthor=E.+O.+Kistnerauthor=L.+P.+Schummauthor=A.+T.+Leeauthor=P.+K.+Gregersenauthor=M.+M.+Barmadaauthor=J.+I.+Rotterauthor=D.+L.+Nicolaeauthor=J.+H.+Cho&title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&doi=10.1126%2Fscience.1135245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Genome-Wide Association Study Identifies IL23R as an Inflammatory Bowel Disease Gene</span></div><div class="casAuthors">Duerr, Richard H.; Taylor, Kent D.; Brant, Steven R.; Rioux, John D.; Silverberg, Mark S.; Daly, Mark J.; Steinhart, A. Hillary; Abraham, Clara; Regueiro, Miguel; Griffiths, Anne; Dassopoulos, Themistocles; Bitton, Alain; Yang, Huiying; Targan, Stephan; Datta, Lisa Wu; Kistner, Emily O.; Schumm, L. Philip; Lee, Annette T.; Gregersen, Peter K.; Barmada, M. Michael; Rotter, Jerome I.; Nicolae, Dan L.; Cho, Judy H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">5804</span>),
    <span class="NLM_cas:pages">1461-1463</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inflammatory bowel diseases Crohn's disease and ulcerative colitis are common, chronic disorders that cause abdominal pain, diarrhea, and gastrointestinal bleeding.  To identify genetic factors that might contribute to these disorders, we performed a genome-wide assocn. study.  We found a highly significant assocn. between Crohn's disease and the IL23R gene on chromosome 1p31, which encodes a subunit of the receptor for the pro-inflammatory cytokine interleukin-23.  An uncommon coding variant (rs11209026, c.1142G>A, p.Arg381Gln) confers strong protection against Crohn's disease, and addnl. noncoding IL23R variants are independently assocd.  Replication studies confirmed IL23R assocns. in independent cohorts of patients with Crohn's disease or ulcerative colitis.  These results and previous studies on the pro-inflammatory role of IL-23 prioritize this signaling pathway as a therapeutic target in inflammatory bowel disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruSERfyA1l3rVg90H21EOLACvtfcHk0lgZPRoeI_aDpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CntrjF&md5=3d95555a1772c6f89dc4171f12105242</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1135245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1135245%26sid%3Dliteratum%253Aachs%26aulast%3DDuerr%26aufirst%3DR.%2BH.%26aulast%3DTaylor%26aufirst%3DK.%2BD.%26aulast%3DBrant%26aufirst%3DS.%2BR.%26aulast%3DRioux%26aufirst%3DJ.%2BD.%26aulast%3DSilverberg%26aufirst%3DM.%2BS.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DSteinhart%26aufirst%3DA.%2BH.%26aulast%3DAbraham%26aufirst%3DC.%26aulast%3DRegueiro%26aufirst%3DM.%26aulast%3DGriffiths%26aufirst%3DA.%26aulast%3DDassopoulos%26aufirst%3DT.%26aulast%3DBitton%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DTargan%26aufirst%3DS.%26aulast%3DDatta%26aufirst%3DL.%2BW.%26aulast%3DKistner%26aufirst%3DE.%2BO.%26aulast%3DSchumm%26aufirst%3DL.%2BP.%26aulast%3DLee%26aufirst%3DA.%2BT.%26aulast%3DGregersen%26aufirst%3DP.%2BK.%26aulast%3DBarmada%26aufirst%3DM.%2BM.%26aulast%3DRotter%26aufirst%3DJ.%2BI.%26aulast%3DNicolae%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26atitle%3DA%2520genome-wide%2520association%2520study%2520identifies%2520IL23R%2520as%2520an%2520inflammatory%2520bowel%2520disease%2520gene%26jtitle%3DScience%26date%3D2006%26volume%3D314%26spage%3D1461%26epage%3D3%26doi%3D10.1126%2Fscience.1135245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hueber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koroleva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draelos, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M. H.</span></span>; <span class="NLM_contrib-group">Psoriasis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Reino, J. J.</span></span>; <span class="NLM_contrib-group">Rheumatoid Arthritis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falk, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. D.</span></span>; <span class="NLM_contrib-group">Uveitis Study Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Padova, F.</span></span> <span> </span><span class="NLM_article-title">Effects of AIN457, a fully human antibody to interleukin-17A, on psoriasis, rheumatoid arthritis, and uveitis</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">52ra72</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.3001107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1126%2Fscitranslmed.3001107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=20926833" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=52ra72&author=W.+Hueberauthor=D.+D.+Patelauthor=T.+Dryjaauthor=A.+M.+Wrightauthor=I.+Korolevaauthor=G.+Bruinauthor=C.+Antoniauthor=Z.+Draelosauthor=M.+H.+Goldauthor=Psoriasis+Study+Groupauthor=P.+Durezauthor=P.+P.+Takauthor=J.+J.+Gomez-Reinoauthor=Rheumatoid+Arthritis+Study+Groupauthor=C.+S.+Fosterauthor=R.+Y.+Kimauthor=C.+M.+Samsonauthor=N.+S.+Falkauthor=D.+S.+Chuauthor=D.+Callananauthor=Q.+D.+Nguyenauthor=Uveitis+Study+Groupauthor=K.+Roseauthor=A.+Haiderauthor=F.+Di+Padova&title=Effects+of+AIN457%2C+a+fully+human+antibody+to+interleukin-17A%2C+on+psoriasis%2C+rheumatoid+arthritis%2C+and+uveitis&doi=10.1126%2Fscitranslmed.3001107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001107%26sid%3Dliteratum%253Aachs%26aulast%3DHueber%26aufirst%3DW.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DDryja%26aufirst%3DT.%26aulast%3DWright%26aufirst%3DA.%2BM.%26aulast%3DKoroleva%26aufirst%3DI.%26aulast%3DBruin%26aufirst%3DG.%26aulast%3DAntoni%26aufirst%3DC.%26aulast%3DDraelos%26aufirst%3DZ.%26aulast%3DGold%26aufirst%3DM.%2BH.%26aulast%3D%26aulast%3DDurez%26aufirst%3DP.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DGomez-Reino%26aufirst%3DJ.%2BJ.%26aulast%3D%26aulast%3DFoster%26aufirst%3DC.%2BS.%26aulast%3DKim%26aufirst%3DR.%2BY.%26aulast%3DSamson%26aufirst%3DC.%2BM.%26aulast%3DFalk%26aufirst%3DN.%2BS.%26aulast%3DChu%26aufirst%3DD.%2BS.%26aulast%3DCallanan%26aufirst%3DD.%26aulast%3DNguyen%26aufirst%3DQ.%2BD.%26aulast%3D%26aulast%3DRose%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DA.%26aulast%3DDi%2BPadova%26aufirst%3DF.%26atitle%3DEffects%2520of%2520AIN457%252C%2520a%2520fully%2520human%2520antibody%2520to%2520interleukin-17A%252C%2520on%2520psoriasis%252C%2520rheumatoid%2520arthritis%252C%2520and%2520uveitis%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2010%26volume%3D2%26spage%3D52ra72%26doi%3D10.1126%2Fscitranslmed.3001107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Attia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abushouk, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadelkarim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgebaly, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Daim, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negida, A.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy of brodalumab for moderate-to-severe plaque psoriasis: A systematic review and meta-analysis</span>. <i>Clin. Drug Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1007/s40261-017-0500-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1007%2Fs40261-017-0500-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28197901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVaqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=439-451&author=A.+Attiaauthor=A.+I.+Abushoukauthor=H.+Ahmedauthor=M.+Gadelkarimauthor=A.+Elgebalyauthor=Z.+Hassanauthor=M.+M.+Abdel-Daimauthor=A.+Negida&title=Safety+and+efficacy+of+brodalumab+for+moderate-to-severe+plaque+psoriasis%3A+A+systematic+review+and+meta-analysis&doi=10.1007%2Fs40261-017-0500-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Safety and Efficacy of Brodalumab for Moderate-to-Severe Plaque Psoriasis: A Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Attia, Attia; Abushouk, Abdelrahman Ibrahim; Ahmed, Hussien; Gadelkarim, Mohamed; Elgebaly, Ahmed; Hassan, Zeinab; Abdel-Daim, Mohamed M.; Negida, Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Drug Investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">439-451</span>CODEN:
                <span class="NLM_cas:coden">CDINFR</span>;
        ISSN:<span class="NLM_cas:issn">1173-2563</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Background: Psoriasis is an inflammatory skin disease that affects 2-3% of the worldwide population.  The interleukin-17 cytokine family has been proven to play a central role in the pathogenesis of psoriasis.  Brodalumab is a novel biol. agent that targets interleukin-17 mols. and their receptors.  We aimed to evaluate the safety and efficacy of brodalumab as a therapeutic agent for moderate-to-severe psoriasis in a meta-anal. framework.  Methods: A computer literature search of PubMed, OVID, Cochrane Central, EMBASE, EBSCO, Scopus, and Web of Science was conducted using relevant keywords.  Data were extd. from eligible trials and analyzed using RevMan (Version 5.3 for windows) and OpenMeta [Analyst] software.  Results: Six clin. trials (n = 4118 patients) were pooled in the final anal.  The pooled effect size favored brodalumab over placebo in terms of the Psoriasis Area and Severity Index (PASI) 75 [risk ratio (RR) = 12.61, 95% confidence interval (CI) 9.74-16.34], PASI 90 (RR = 28.72, 95% CI 18.34-44.98), and PASI 100 (RR = 61.23, 95% CI 25.48-147.17).  Anal. of secondary outcomes showed that brodalumab was superior to placebo in terms of static physician's global assessment (RR = 32.53, 95% CI 13.80-76.69) and psoriasis symptoms inventory scores (RR = 14.70, 95% CI 8.38-25.78).  Meta-regression anal. found a significant linear assocn. between the brodalumab dose and the effect size on PASI and psoriasis symptoms inventory scores.  The rate of overall adverse events was slightly higher in the brodalumab group (RR = 1.13, 95% CI 1.06-1.22); however, none of the individual adverse events were significantly higher in the brodalumab group, compared to the placebo group.  Conclusion: Brodalumab showed an acceptable safety profile and a robust efficacy in the treatment of moderate-to-severe plaque psoriasis.  However, the current evidence is insufficient to confirm maintenance of these results in the long term; therefore, larger studies with longer follow-up periods are required.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOSFFowbNs27Vg90H21EOLACvtfcHk0lg_xvTzclyfMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVaqsbo%253D&md5=b9996588017aad4e3944b67299cc62df</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs40261-017-0500-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40261-017-0500-9%26sid%3Dliteratum%253Aachs%26aulast%3DAttia%26aufirst%3DA.%26aulast%3DAbushouk%26aufirst%3DA.%2BI.%26aulast%3DAhmed%26aufirst%3DH.%26aulast%3DGadelkarim%26aufirst%3DM.%26aulast%3DElgebaly%26aufirst%3DA.%26aulast%3DHassan%26aufirst%3DZ.%26aulast%3DAbdel-Daim%26aufirst%3DM.%2BM.%26aulast%3DNegida%26aufirst%3DA.%26atitle%3DSafety%2520and%2520efficacy%2520of%2520brodalumab%2520for%2520moderate-to-severe%2520plaque%2520psoriasis%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%26jtitle%3DClin.%2520Drug%2520Invest.%26date%3D2017%26volume%3D37%26spage%3D439%26epage%3D451%26doi%3D10.1007%2Fs40261-017-0500-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mease, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Heijde, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchlin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuchacovich, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuler, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladman, D. D.</span></span>; <span class="NLM_contrib-group">SPIRIT-P1 Study Group</span> <span> </span><span class="NLM_article-title">Ixekizumab, an interleukin-17A specific monoclonal antibody, for the treatment of biologic-naive patients with active psoriatic arthritis: results from the 24-week randomised, double-blind, placebo-controlled and active (adalimumab)-controlled period of the phase III trial SPIRIT-P1</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2016-209709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1136%2Fannrheumdis-2016-209709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27553214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2017&pages=79-87&author=P.+J.+Measeauthor=D.+van+der+Heijdeauthor=C.+T.+Ritchlinauthor=M.+Okadaauthor=R.+S.+Cuchacovichauthor=C.+L.+Shulerauthor=C.+Y.+Linauthor=D.+K.+Braunauthor=C.+H.+Leeauthor=D.+D.+Gladmanauthor=SPIRIT-P1+Study+Group&title=Ixekizumab%2C+an+interleukin-17A+specific+monoclonal+antibody%2C+for+the+treatment+of+biologic-naive+patients+with+active+psoriatic+arthritis%3A+results+from+the+24-week+randomised%2C+double-blind%2C+placebo-controlled+and+active+%28adalimumab%29-controlled+period+of+the+phase+III+trial+SPIRIT-P1&doi=10.1136%2Fannrheumdis-2016-209709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ixekizumab, an interleukin-17A specific monoclonal antibody, for the treatment of biologic-naive patients with active psoriatic arthritis: results from the 24-week randomised, double-blind, placebo controlled and active (adalimumab)-controlled period of the phase III trial SPIRIT-P1</span></div><div class="casAuthors">Mease, Philip J.; van der Heijde, Desiree; Ritchlin, Christopher T.; Okada, Masato; Cuchacovich, Raquel S.; Shuler, Catherine L.; Lin, Chen-Yen; Braun, Daniel K.; Lee, Chin H.; Gladman, Dafna D.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-87</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective: To assess the safety and efficacy of ixekizumab, a monoclonal antibody that inhibits interleukin-17A, in a double-blind phase III trial enrolling patients with active psoriatic arthritis (PsA).  Methods: Patients naive to biol. therapy with active PsA were randomised to s.c. injections of placebo (N = 106), adalimumab 40 mg once every 2 wk (active ref.; N = 101), ixekizumab 80 mg once every 2 wk (IXEQ2W) (N = 103), or ixekizumab 80 mg once every 4 wk (IXEQ4W) (N = 107).  Both ixekizumab regimens included a 160-mg starting dose.  The primary objective was to assess the superiority of IXEQ2W or IXEQ4W vs. placebo as measured by the proportion of patients achieving an American College of Rheumatol. 20 (ACR20) response at week 24.  Results: Significantly more patients treated with ixekizumab achieved an ACR20 response with IXEQ2W (62.1%) or IXEQ4W (57.9%) than placebo (30.2%) (p≤0.001; non-responder imputation method).  Disease activity and functional disability were significantly improved with both ixekizumab doses vs. placebo at weeks 12 and 24, and there was significantly less progression of structural damage at week 24 (p≤0.01).  Clearance of plaque psoriasis was greater with ixekizumab than placebo (p≤0.001).  Efficacy results with adalimumab, the active ref. arm, showed significant improvements vs. placebo.  Treatment-emergent adverse events were more frequent with ixekizumab (65.7-66.4%) and adalimumab (64.4%) than placebo (47.2%) (p<0.05).  Conclusions: In biol.-naive patients with active PsA, ixekizumab treatment resulted in improvements in disease activity and phys. function, as well as in the inhibition of structural damage progression.  Overall, adverse events were more frequent in all active groups compared with placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3hHplmarqLVg90H21EOLACvtfcHk0lg_xvTzclyfMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFahsr4%253D&md5=3beef8f867a6e8ada97080e4f3bc8cbe</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2016-209709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2016-209709%26sid%3Dliteratum%253Aachs%26aulast%3DMease%26aufirst%3DP.%2BJ.%26aulast%3Dvan%2Bder%2BHeijde%26aufirst%3DD.%26aulast%3DRitchlin%26aufirst%3DC.%2BT.%26aulast%3DOkada%26aufirst%3DM.%26aulast%3DCuchacovich%26aufirst%3DR.%2BS.%26aulast%3DShuler%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DBraun%26aufirst%3DD.%2BK.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DGladman%26aufirst%3DD.%2BD.%26aulast%3D%26atitle%3DIxekizumab%252C%2520an%2520interleukin-17A%2520specific%2520monoclonal%2520antibody%252C%2520for%2520the%2520treatment%2520of%2520biologic-naive%2520patients%2520with%2520active%2520psoriatic%2520arthritis%253A%2520results%2520from%2520the%252024-week%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520and%2520active%2520%2528adalimumab%2529-controlled%2520period%2520of%2520the%2520phase%2520III%2520trial%2520SPIRIT-P1%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2017%26volume%3D76%26spage%3D79%26epage%3D87%26doi%3D10.1136%2Fannrheumdis-2016-209709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Modulators of the nuclear receptor retinoic acid receptor-related orphan receptor-γ (RORγ or RORc)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5871</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/jm401901d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401901d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5871-92&author=B.+P.+Fauberauthor=S.+Magnuson&title=Modulators+of+the+nuclear+receptor+retinoic+acid+receptor-related+orphan+receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29&doi=10.1021%2Fjm401901d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Modulators of the Nuclear Receptor Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc)</span></div><div class="casAuthors">Fauber, Benjamin P.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5871-5892</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  As the biol. surrounding the nuclear receptor retinoic acid receptor-related orphan receptor-gamma (RORγ or RORc) continues to evolve, significant effort has been invested in discovering modulators of this potentially important target for the treatment of metabolic and immunol. diseases.  Several major pharmaceutical and biotechnol. companies have disclosed RORc inhibitors or partnered with other players in the field.  In this perspective, we discuss both the biol. and the underlying structural biol. of RORc, and summarize the RORc modulators disclosed in the scientific and patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhzxefrY4hLVg90H21EOLACvtfcHk0ljjDrZRVTWT5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVartbo%253D&md5=42b350bff7776d4b5689275eab214d86</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm401901d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401901d%26sid%3Dliteratum%253Aachs%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DModulators%2520of%2520the%2520nuclear%2520receptor%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5871%26epage%3D92%26doi%3D10.1021%2Fjm401901d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbieg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span> <span> </span><span class="NLM_article-title">RORγ antagonists and inverse agonists: a patent review</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1236918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1080%2F13543776.2017.1236918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27629281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=101-112&author=S.+M.+Bronnerauthor=J.+R.+Zbiegauthor=J.+J.+Crawford&title=ROR%CE%B3+antagonists+and+inverse+agonists%3A+a+patent+review&doi=10.1080%2F13543776.2017.1236918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">RORγ antagonists and inverse agonists: a patent review</span></div><div class="casAuthors">Bronner, Sarah M.; Zbieg, Jason R.; Crawford, James J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-112</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The transcription factor RORγ plays a crit. role in the expression of pro-inflammatory cytokine interleukin IL-17 and is therefore an attractive target for the treatment of inflammatory diseases.  Interest in this mol. target has been heightened by the advancement of orally and topically administered RORγ modulators into clin. trials.  The present review seeks to summarize published patent applications from assignee companies that have disclosed Investigational New Drug (IND) filings for small mol. RORγ/RORγt antagonists and inverse agonists.  The field of RORγ research is extremely competitive, with the majority of companies targeting psoriasis as the primary disease indication.  Vitae Pharmaceuticals is currently the most advanced, with a potential first-in-class oral RORγ-modulator for the treatment of psoriasis.  Future efforts will likely expand into potential applications of RORγ-modulators in the lesser explored immune-related areas of rheumatoid arthritis, type 1 diabetes, lupus, and irritable bowel disorder, as well as cancer immunotherapy and castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj8lMzWsYVA7Vg90H21EOLACvtfcHk0ljjDrZRVTWT5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nsbzP&md5=d80db6c4c948bc12c5d0ce4b41edd2fb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1236918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1236918%26sid%3Dliteratum%253Aachs%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DZbieg%26aufirst%3DJ.%2BR.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26atitle%3DROR%25CE%25B3%2520antagonists%2520and%2520inverse%2520agonists%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D101%26epage%3D112%26doi%3D10.1080%2F13543776.2017.1236918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, R. C.</span></span> <span> </span><span class="NLM_article-title">Combating autoimmune diseases with retinoic acid receptor-related orphan receptor-γ (RORγ or RORc) inhibitors: hits and misses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10976</span>– <span class="NLM_lpage">10995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00588</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00588" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10976-10995&author=V.+B.+Pandyaauthor=S.+Kumarauthor=+Sachchidanandauthor=R.+Sharmaauthor=R.+C.+Desai&title=Combating+autoimmune+diseases+with+retinoic+acid+receptor-related+orphan+receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29+inhibitors%3A+hits+and+misses&doi=10.1021%2Facs.jmedchem.8b00588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span></div><div class="casAuthors">Pandya, Vrajesh B.; Kumar, Sanjay; Sachchidanand; Sharma, Rajiv; Desai, Ranjit C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10976-10995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear receptor retinoic acid receptor-related orphan receptor gamma (RORγ or RORc) is a key transcription factor for the prodn. of pro-inflammatory cytokines implicated in the pathogenesis of autoimmune diseases.  Recently, small mol. inhibitors of RORc drew the enormous attention of the research community worldwide as a possible therapy for autoimmune diseases, mediated by the IL-17 cytokine.  With the clin. proof-of-concept inferred from a small mol. inhibitor VTP-43742 for psoriasis and recent inflow of several RORc inhibitors into the clinic for therapeutic interventions in autoimmune diseases, this field continues to evolve.  This review briefly summarizes the RORc inhibitors disclosed in the literature and discusses the progress made by these inhibitors in combating autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGqZl3bPbtLVg90H21EOLACvtfcHk0ljjDrZRVTWT5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E&md5=4113d22f73ab2911535eef053801994d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00588%26sid%3Dliteratum%253Aachs%26aulast%3DPandya%26aufirst%3DV.%2BB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSachchidanand%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DDesai%26aufirst%3DR.%2BC.%26atitle%3DCombating%2520autoimmune%2520diseases%2520with%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%2520inhibitors%253A%2520hits%2520and%2520misses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10976%26epage%3D10995%26doi%3D10.1021%2Facs.jmedchem.8b00588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanis, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent Barbay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummer, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelen, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlueter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spurlino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, K.</span></span> <span> </span><span class="NLM_article-title">3-Substituted quinolines as RORγt inverse agonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2019.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=31010722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXot1Sru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1463-1470&author=V.+M.+Tanisauthor=H.+Venkatesanauthor=M.+D.+Cummingsauthor=M.+Albersauthor=J.+Kent+Barbayauthor=K.+Hermanauthor=D.+A.+Kummerauthor=C.+Milliganauthor=M.+I.+Nelenauthor=R.+Nishimuraauthor=T.+Schlueterauthor=B.+Scottauthor=J.+Spurlinoauthor=R.+Wolinauthor=C.+Woodsauthor=X.+Xueauthor=J.+P.+Edwardsauthor=A.+M.+Fourieauthor=K.+Leonard&title=3-Substituted+quinolines+as+ROR%CE%B3t+inverse+agonists&doi=10.1016%2Fj.bmcl.2019.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">3-Substituted Quinolines as RORγt Inverse Agonists</span></div><div class="casAuthors">Tanis, Virginia M.; Venkatesan, Hariharan; Cummings, Maxwell D.; Albers, Michael; Kent Barbay, J.; Herman, Krystal; Kummer, David A.; Milligan, Cynthia; Nelen, Marina I.; Nishimura, Rachel; Schlueter, Thomas; Scott, Brian; Spurlino, John; Wolin, Ronald; Woods, Craig; Xue, Xiaohua; Edwards, James P.; Fourie, Anne M.; Leonard, Kristi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1470</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have previously reported the syntheses of a series of 3,6-disubstituted quinolines as modulators of the retinoic acid receptor-related orphan receptor gamma t (RORγt).  These mols. are potent binders but are high mol. wt. and they exhibited poor soly. at pH 2 and pH 7.  This manuscript details our efforts at improving phys. chem. properties for this series of compds. by increasing the diversity at the 3-position (i.e. introducing heteroatoms and lowering the mol. wt.).  These efforts have led to mols. which are potent binders with improved soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhylTaPGx9U7Vg90H21EOLACvtfcHk0lj9rFxQMajsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXot1Sru74%253D&md5=14b0728bc23fb5c8cafdab363276ebf7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DTanis%26aufirst%3DV.%2BM.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DAlbers%26aufirst%3DM.%26aulast%3DKent%2BBarbay%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DK.%26aulast%3DKummer%26aufirst%3DD.%2BA.%26aulast%3DMilligan%26aufirst%3DC.%26aulast%3DNelen%26aufirst%3DM.%2BI.%26aulast%3DNishimura%26aufirst%3DR.%26aulast%3DSchlueter%26aufirst%3DT.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DSpurlino%26aufirst%3DJ.%26aulast%3DWolin%26aufirst%3DR.%26aulast%3DWoods%26aufirst%3DC.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DFourie%26aufirst%3DA.%2BM.%26aulast%3DLeonard%26aufirst%3DK.%26atitle%3D3-Substituted%2520quinolines%2520as%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1463%26epage%3D1470%26doi%3D10.1016%2Fj.bmcl.2019.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shichijo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">A novel RORγt inhibitor is a potential therapeutic agent for the topical treatment of psoriasis with low risk of thymic aberrations</span>. <i>J. Dermatol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.jdermsci.2019.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.jdermsci.2019.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30905492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvV2ksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2019&pages=176-185&author=C.+Imuraauthor=A.+Ueyamaauthor=Y.+Sasakiauthor=M.+Shimizuauthor=Y.+Furueauthor=N.+Taiauthor=K.+Tsujiiauthor=K.+Katayamaauthor=T.+Okunoauthor=M.+Shichijoauthor=K.+Yasuiauthor=M.+Yamamoto&title=A+novel+ROR%CE%B3t+inhibitor+is+a+potential+therapeutic+agent+for+the+topical+treatment+of+psoriasis+with+low+risk+of+thymic+aberrations&doi=10.1016%2Fj.jdermsci.2019.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A novel RORγt inhibitor is a potential therapeutic agent for the topical treatment of psoriasis with low risk of thymic aberrations</span></div><div class="casAuthors">Imura, Chihiro; Ueyama, Azumi; Sasaki, Yoshikazu; Shimizu, Masaya; Furue, Yoko; Tai, Nobuyuki; Tsujii, Kenichiro; Katayama, Kazufumi; Okuno, Takayuki; Shichijo, Michitaka; Yasui, Kiyoshi; Yamamoto, Mina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dermatological Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">176-185</span>CODEN:
                <span class="NLM_cas:coden">JDSCEI</span>;
        ISSN:<span class="NLM_cas:issn">0923-1811</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptor gamma t (RORγt) has crit. roles in the development, maintenance and function of interleukin (IL)-17-producing cells and is a highly attractive target for the treatment of IL-17-mediated autoimmune disease, particularly psoriasis.  On the other hand, RORγt is also crit. for controlling apoptosis during thymopoiesis, and genetic RORγt ablation or systematic RORγt inhibition cause progressive thymic aberrations leading to T cell lymphomas.  We investigated whether topical administration of our novel RORγt inhibitor, S18-000003 has therapeutic potential for psoriasis with low risk of thymic aberrations.  We evaluated the effect of topical S18-000003 on psoriasis-like skin inflammation and influence on the thymus in a 12-O-tetradecanoylphorbol-13-acetate-induced K14.Stat3C mouse psoriasis model.  S18-000003 markedly inhibited the development of psoriatic skin inflammation via suppression of the IL-17 pathway.  In the skin, S18-000003 suppressed all subsets of IL-17-producing cells that we previously identified in this psoriasis model: Th17 cells, Tc17 cells, dermal γδ T cells, TCR- cells that probably included innate lymphoid cells, and CD4-CD8- double-neg. αβ T cells.  Notably, neither redn. of CD4+CD8+ double-pos. thymocytes nor dysregulation of cell cycling was obsd. in S18-000003-treated mice, even at a high dose.  Our topically administered RORγt inhibitor is a potential therapeutic agent for psoriasis with low risk of thymic lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuafyicGpTLVg90H21EOLACvtfcHk0lj9rFxQMajsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvV2ksbY%253D&md5=e700b0f60f059581b93976032695f822</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jdermsci.2019.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jdermsci.2019.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DImura%26aufirst%3DC.%26aulast%3DUeyama%26aufirst%3DA.%26aulast%3DSasaki%26aufirst%3DY.%26aulast%3DShimizu%26aufirst%3DM.%26aulast%3DFurue%26aufirst%3DY.%26aulast%3DTai%26aufirst%3DN.%26aulast%3DTsujii%26aufirst%3DK.%26aulast%3DKatayama%26aufirst%3DK.%26aulast%3DOkuno%26aufirst%3DT.%26aulast%3DShichijo%26aufirst%3DM.%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DA%2520novel%2520ROR%25CE%25B3t%2520inhibitor%2520is%2520a%2520potential%2520therapeutic%2520agent%2520for%2520the%2520topical%2520treatment%2520of%2520psoriasis%2520with%2520low%2520risk%2520of%2520thymic%2520aberrations%26jtitle%3DJ.%2520Dermatol.%2520Sci.%26date%3D2019%26volume%3D93%26spage%3D176%26epage%3D185%26doi%3D10.1016%2Fj.jdermsci.2019.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-indanyl benzamides as potent RORγt inverse agonists</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.ejmech.2019.01.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30743096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFygtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=37-48&author=J.+Tianauthor=N.+Sunauthor=M.+Yuauthor=X.+Guauthor=Q.+Xieauthor=L.+Shaoauthor=J.+Liuauthor=L.+Liuauthor=Y.+Wang&title=Discovery+of+N-indanyl+benzamides+as+potent+ROR%CE%B3t+inverse+agonists&doi=10.1016%2Fj.ejmech.2019.01.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-indanyl benzamides as potent RORγt inverse agonists</span></div><div class="casAuthors">Tian, Jinlong; Sun, Nannan; Yu, Mingcheng; Gu, Xianfeng; Xie, Qiong; Shao, Liming; Liu, Jin; Liu, Li; Wang, Yonghui</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-48</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The retinoic acid receptor-related orphan receptor-gamma-t (RORγt) is a promising therapeutic target for treatment of Th17 cell-mediated autoimmune diseases.  Based on a scaffold hopping/conformational restriction strategy, a series of N-indanyl benzamides as novel RORγt inverse agonists was discovered.  Exploration of structure-activity relationship on the piperazine ring, benzoyl moiety and cyclopentyl moiety of N-indanyl benzamides 2a and 2d led to identification of potent RORγt inverse agonists.  Compd. 5c with (S)-enantiomer was found having an IC50 of 153.7 nM in Fluorescence Resonance Energy Transfer (FRET) assay, and an IC50 of 47.1 nM in mouse Th17 cell differentiation assay, which represents a promising starting point for developing potent small mol. RORγt inverse agonists.  Binding modes of the two enantiomers 5c and 5d in RORγt ligand binding domain were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtcwkL0EZAzrVg90H21EOLACvtfcHk0lj9rFxQMajsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFygtbs%253D&md5=52135643f47a06c14abc425a69341667</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.082%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DShao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520N-indanyl%2520benzamides%2520as%2520potent%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D37%26epage%3D48%26doi%3D10.1016%2Fj.ejmech.2019.01.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotoku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujioka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suwa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikenogami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozaki, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of second generation RORγ inhibitors composed of an azole scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2842</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01567</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01567" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Sksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2837-2842&author=M.+Kotokuauthor=T.+Maebaauthor=S.+Fujiokaauthor=M.+Yokotaauthor=N.+Sekiauthor=K.+Itoauthor=Y.+Suwaauthor=T.+Ikenogamiauthor=K.+Hirataauthor=Y.+Haseauthor=Y.+Katsudaauthor=N.+Miyagawaauthor=K.+Aritaauthor=K.+Asahinaauthor=M.+Noguchiauthor=A.+Nomuraauthor=S.+Doiauthor=T.+Adachiauthor=P.+Croweauthor=H.+Taoauthor=S.+Thacherauthor=H.+Hashimotoauthor=T.+Suzukiauthor=M.+Shiozaki&title=Discovery+of+second+generation+ROR%CE%B3+inhibitors+composed+of+an+azole+scaffold&doi=10.1021%2Facs.jmedchem.8b01567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Second Generation RORγ Inhibitors Composed of an Azole Scaffold</span></div><div class="casAuthors">Kotoku, Masayuki; Maeba, Takaki; Fujioka, Shingo; Yokota, Masahiro; Seki, Noriyoshi; Ito, Keisuke; Suwa, Yoshihiro; Ikenogami, Taku; Hirata, Kazuyuki; Hase, Yasunori; Katsuda, Yoshiaki; Miyagawa, Naoki; Arita, Kojo; Asahina, Kota; Noguchi, Masato; Nomura, Akihiro; Doi, Satoki; Adachi, Tsuyoshi; Crowe, Paul; Tao, Haiyan; Thacher, Scott; Hashimoto, Hiromasa; Suzuki, Takayoshi; Shiozaki, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2837-2842</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Starting from a previously reported retinoid-related orphan receptor γ (RORγ) inhibitor, successive efforts to improve in vivo potency were continued.  Introduction of metabolically beneficial motifs in conjunction with scaffold hopping was examd., resulting in discovery of the second generation RORγ inhibitor composed of a 4-(isoxazol-3-yl)butanoic acid scaffold (I).  Compd. I achieved a 10-fold improvement in in vivo potency in a mouse CD3 challenge model along with significant anti-inflammatory effects in a mouse dermatitis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIwG22VwIKKbVg90H21EOLACvtfcHk0li7e8MtQIT2vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Sksr4%253D&md5=b62cac378cd152f9785731453c2d6ad0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01567%26sid%3Dliteratum%253Aachs%26aulast%3DKotoku%26aufirst%3DM.%26aulast%3DMaeba%26aufirst%3DT.%26aulast%3DFujioka%26aufirst%3DS.%26aulast%3DYokota%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSuwa%26aufirst%3DY.%26aulast%3DIkenogami%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DHase%26aufirst%3DY.%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DMiyagawa%26aufirst%3DN.%26aulast%3DArita%26aufirst%3DK.%26aulast%3DAsahina%26aufirst%3DK.%26aulast%3DNoguchi%26aufirst%3DM.%26aulast%3DNomura%26aufirst%3DA.%26aulast%3DDoi%26aufirst%3DS.%26aulast%3DAdachi%26aufirst%3DT.%26aulast%3DCrowe%26aufirst%3DP.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DThacher%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DShiozaki%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520second%2520generation%2520ROR%25CE%25B3%2520inhibitors%2520composed%2520of%2520an%2520azole%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2837%26epage%3D2842%26doi%3D10.1021%2Facs.jmedchem.8b01567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruzanov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galella, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanteru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salter-Cid, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, T. G. M.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of phenyl (3-phenylpyrrolidin-3-yl)sulfones as selective, orally active RORγt inverse agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVyltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=367-373&author=J.+J.+Duanauthor=Z.+Luauthor=B.+Jiangauthor=S.+Stachuraauthor=C.+A.+Weigeltauthor=J.+S.+Sackauthor=J.+Khanauthor=M.+Ruzanovauthor=M.+A.+Galellaauthor=D.+R.+Wuauthor=M.+Yardeauthor=D.+R.+Shenauthor=D.+J.+Shusterauthor=V.+Borowskiauthor=J.+H.+Xieauthor=L.+Zhangauthor=S.+Vanteruauthor=A.+K.+Guptaauthor=A.+Mathurauthor=Q.+Zhaoauthor=W.+Fosterauthor=L.+M.+Salter-Cidauthor=P.+H.+Carterauthor=T.+G.+M.+Dhar&title=Structure-based+discovery+of+phenyl+%283-phenylpyrrolidin-3-yl%29sulfones+as+selective%2C+orally+active+ROR%CE%B3t+inverse+agonists&doi=10.1021%2Facsmedchemlett.9b00010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Discovery of Phenyl (3-Phenylpyrrolidin-3-yl)sulfones as Selective, Orally Active RORγt Inverse Agonists</span></div><div class="casAuthors">Duan, James J.-W.; Lu, Zhonghui; Jiang, Bin; Stachura, Sylwia; Weigelt, Carolyn A.; Sack, John S.; Khan, Javed; Ruzanov, Max; Galella, Michael A.; Wu, Dauh-Rurng; Yarde, Melissa; Shen, Ding-Ren; Shuster, David J.; Borowski, Virna; Xie, Jenny H.; Zhang, Lisa; Vanteru, Sridhar; Gupta, Arun Kumar; Mathur, Arvind; Zhao, Qihong; Foster, William; Salter-Cid, Luisa M.; Carter, Percy H.; Dhar, T. G. Murali</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">367-373</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new Ph (3-phenylpyrrolidin-3-yl)sulfone series of RORγt inverse agonists was discovered utilizing the binding conformation of previously reported bicyclic sulfonamide 1.  Through a combination of structure-based design and structure-activity relationship studies, a polar set of amides at N1-position of the pyrrolidine ring and perfluoroisopropyl group at para-position of the 3-Ph group were identified as crit. structural elements to achieve high selectivity against PXR, LXRα, and LXRβ.  Further optimization led to the discovery of (1R,4r)-4-((R)-3-((4-fluorophenyl)sulfonyl)-3-(4-(perfluoropropan-2-yl)phenyl)pyrrolidine-1-carbonyl)cyclohexane-1-carboxylic acid (26), which displayed excellent selectivity, desirable liability and pharmacokinetic properties in vitro, and a good pharmacokinetic profile in mouse.  Oral administration of 26 demonstrated dose-dependent inhibition of IL-17 prodn. in a mouse IL-2/IL-23-induced pharmacodynamic model and biol.-like efficacy in an IL-23-induced mouse acanthosis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZBJUkp-y6zbVg90H21EOLACvtfcHk0li7e8MtQIT2vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVyltLw%253D&md5=16114ce5cec852904e038f8d33be67a1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00010%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%2BJ.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DStachura%26aufirst%3DS.%26aulast%3DWeigelt%26aufirst%3DC.%2BA.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DRuzanov%26aufirst%3DM.%26aulast%3DGalella%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DD.%2BR.%26aulast%3DYarde%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DShuster%26aufirst%3DD.%2BJ.%26aulast%3DBorowski%26aufirst%3DV.%26aulast%3DXie%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVanteru%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DA.%2BK.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DFoster%26aufirst%3DW.%26aulast%3DSalter-Cid%26aufirst%3DL.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DDhar%26aufirst%3DT.%2BG.%2BM.%26atitle%3DStructure-based%2520discovery%2520of%2520phenyl%2520%25283-phenylpyrrolidin-3-yl%2529sulfones%2520as%2520selective%252C%2520orally%2520active%2520ROR%25CE%25B3t%2520inverse%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D367%26epage%3D373%26doi%3D10.1021%2Facsmedchemlett.9b00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, V.</span></span> <span> </span><span class="NLM_article-title">Principles for modulation of the nuclear receptor superfamily</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">950</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1038/nrd1551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd1551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=15520817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=950-964&author=H.+Gronemeyerauthor=J.-%C3%85.+Gustafssonauthor=V.+Laudet&title=Principles+for+modulation+of+the+nuclear+receptor+superfamily&doi=10.1038%2Fnrd1551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for modulation of the nuclear receptor superfamily</span></div><div class="casAuthors">Gronemeyer, Hinrich; Gustafsson, Jan-A.; Laudet, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">950-964</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nuclear receptors are major targets for drug discovery and have key roles in development and homeostasis, as well as in many diseases such as obesity, diabetes and cancer.  This review provides a general overview of the mechanism of action of nuclear receptors and explores the various factors that are instrumental in modulating their pharmacol.  In most cases, the response of a given receptor to a particular ligand in a specific tissue will be dictated by the set of proteins with which the receptor is able to interact.  One of the most promising aspects of nuclear receptor pharmacol. is that it is now possible to develop ligands with a large spectrum of full, partial or inverse agonist or antagonist activities, but also compds., called selective nuclear receptor modulators, that activate only a subset of the functions induced by the cognate ligand or that act in a cell-type-selective manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWL9yf1CceLVg90H21EOLACvtfcHk0liegv2Vlh0nKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrY%253D&md5=aedb60785c235bf44cb5f0ea34236090</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd1551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1551%26sid%3Dliteratum%253Aachs%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DLaudet%26aufirst%3DV.%26atitle%3DPrinciples%2520for%2520modulation%2520of%2520the%2520nuclear%2520receptor%2520superfamily%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D950%26epage%3D964%26doi%3D10.1038%2Fnrd1551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muegge, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kugler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terenzio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadia, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrow, N. A.</span></span> <span> </span><span class="NLM_article-title">Structural studies unravel the active conformation of apo RORγt nuclear receptor and a common inverse agonism of two diverse classes of RORγt inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">11618</span>– <span class="NLM_lpage">11630</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.789024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1074%2Fjbc.M117.789024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28546429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOjtbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=11618-11630&author=X.+Liauthor=M.+Andersonauthor=D.+Collinauthor=I.+Mueggeauthor=J.+Wanauthor=D.+Brennanauthor=S.+Kuglerauthor=D.+Terenzioauthor=C.+Kennedyauthor=S.+Linauthor=M.+E.+Labadiaauthor=B.+Cookauthor=R.+Hughesauthor=N.+A.+Farrow&title=Structural+studies+unravel+the+active+conformation+of+apo+ROR%CE%B3t+nuclear+receptor+and+a+common+inverse+agonism+of+two+diverse+classes+of+ROR%CE%B3t+inhibitors&doi=10.1074%2Fjbc.M117.789024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural studies unravel the active conformation of apo RORγt nuclear receptor and a common inverse agonism of two diverse classes of RORγt inhibitors</span></div><div class="casAuthors">Li, Xiang; Anderson, Marie; Collin, Delphine; Muegge, Ingo; Wan, John; Brennan, Debra; Kugler, Stanley; Terenzio, Donna; Kennedy, Charles; Lin, Siqi; Labadia, Mark E.; Cook, Brian; Hughes, Robert; Farrow, Neil A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11618-11630</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The nuclear receptor retinoid acid receptor-related orphan receptor γt (RORγt) is a master regulator of the Th17/IL-17 pathway that plays crucial roles in the pathogenesis of autoimmunity.  RORγt has recently emerged as a highly promising target for treatment of a no. of autoimmune diseases.  Through high-throughput screening, we previously identified several classes of inverse agonists for RORγt.  Here, we report the crystal structures for the ligand-binding domain of RORγt in both apo and ligand-bound states.  We show that apo RORγt adopts an active conformation capable of recruiting coactivator peptides and present a detailed anal. of the structural determinants that stabilize helix 12 (H12) of RORγt in the active state in the absence of a ligand.  The structures of ligand-bound RORγt revealed that binding of the inverse agonists disrupted crit. interactions that stabilized H12.  This destabilizing effect was supported by ab initio calcns. and exptl. by a normalized crystallog. B-factor anal.  Of note, the H12 destabilization in the active state shifted the conformational equil. of RORγt toward an inactive state, which underlies the mol. mechanism of action for the inverse agonists reported here.  These findings highlight that nuclear receptor structure and function are dictated by a dynamic conformational equil. and that subtle changes in ligand structures can shift this equil. in opposite directions, leading to a functional switch from agonists to inverse agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzicB5efbjf7Vg90H21EOLACvtfcHk0liegv2Vlh0nKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOjtbjJ&md5=8224bb328772787a2e1df0aec43566d6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.789024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.789024%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DD.%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DWan%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DD.%26aulast%3DKugler%26aufirst%3DS.%26aulast%3DTerenzio%26aufirst%3DD.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLabadia%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DB.%26aulast%3DHughes%26aufirst%3DR.%26aulast%3DFarrow%26aufirst%3DN.%2BA.%26atitle%3DStructural%2520studies%2520unravel%2520the%2520active%2520conformation%2520of%2520apo%2520ROR%25CE%25B3t%2520nuclear%2520receptor%2520and%2520a%2520common%2520inverse%2520agonism%2520of%2520two%2520diverse%2520classes%2520of%2520ROR%25CE%25B3t%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D11618%26epage%3D11630%26doi%3D10.1074%2Fjbc.M117.789024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muscat, G. E. O.</span></span> <span> </span><span class="NLM_article-title">Characterization of the retinoid orphan-related receptor-alpha coactivator binding interface: a structural basis for ligand-independent transcription</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1012</span>, <span class="refDoi"> DOI: 10.1210/mend.16.5.0837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1210%2Fmend.16.5.0837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=11981035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFGhsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=998-1012&author=J.+M.+Harrisauthor=P.+Lauauthor=S.+L.+Chenauthor=G.+E.+O.+Muscat&title=Characterization+of+the+retinoid+orphan-related+receptor-alpha+coactivator+binding+interface%3A+a+structural+basis+for+ligand-independent+transcription&doi=10.1210%2Fmend.16.5.0837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the retinoid orphan-related receptor-α coactivator binding interface: A structural basis for ligand-independent transcription</span></div><div class="casAuthors">Harris, Jonathan M.; Lau, Patrick; Chen, Shen Liang; Muscat, George E. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">998-1012</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">The retinoid orphan-related receptor-α (RORα) is a member of the ROR subfamily of orphan receptors and acts as a constitutive activator of transcription in the absence of exogenous ligands.  To understand the basis of this activity, we constructed a homol. model of RORα using the closely related TRβ as a template.  Mol. modeling suggested that bulky hydrophobic side chains occupy the RORα ligand cavity leaving a small but distinct cavity that may be involved in receptor stabilization.  This model was subject to docking simulation with a receptor-interacting peptide from the steroid receptor coactivator, GR-interacting protein-1, which delineated a coactivator binding surface consisting of the signature motif spanning helixes 3-5 and helix 12 [activation function 2 (AF2)].  Probing this surface with scanning alanine mutagenesis showed structural and functional equivalence between homologous residues of RORα and TRβ.  This was surprising (given that RORα is a ligand-independent activator, whereas TRβ has an abs. requirement for ligand) and prompted us to use mol. modeling to identify differences between RORα and TRβ in the way that the AF2 helix interacts with the rest of the receptor.  Modeling highlighted a nonconserved amino acid in helix 11 of RORα (Phe491) and a short-length of 3.10 helix at the N terminus of AF2 which we suggest (1) ensures that AF2 is locked permanently in the holoconformation described for other liganded receptors and thus (2) enables ligand-independent recruitment of coactivators.  Consistent with this, mutation of RORα Phe491 to either methionine or alanine (methionine is the homologous residue in TRβ), reduced and ablated transcriptional activation and recruitment of coactivators, resp.  Furthermore, we were able to reconstitute transcriptional activity for both a deletion mutant of RORα lacking AF2, and Phe491Met, by overexpression of a GAL-AF2 fusion protein, demonstrating ligand-independent recruitment of AF2 and a role for Phe491 in recruiting AF2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5veDP0-3KL7Vg90H21EOLACvtfcHk0ljz5nMKzJgshw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFGhsLw%253D&md5=cffc5abe6423fc40e711443bc6231686</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1210%2Fmend.16.5.0837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fmend.16.5.0837%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DLau%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DS.%2BL.%26aulast%3DMuscat%26aufirst%3DG.%2BE.%2BO.%26atitle%3DCharacterization%2520of%2520the%2520retinoid%2520orphan-related%2520receptor-alpha%2520coactivator%2520binding%2520interface%253A%2520a%2520structural%2520basis%2520for%2520ligand-independent%2520transcription%26jtitle%3DMol.%2520Endocrinol.%26date%3D2002%26volume%3D16%26spage%3D998%26epage%3D1012%26doi%3D10.1210%2Fmend.16.5.0837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesch, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendling, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aicher, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, G. D.</span></span> <span> </span><span class="NLM_article-title">Sterol metabolism controls TH17 differentiation by generating endogenous RORγ agonists</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnchembio.1714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=25558972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=141-147&author=X.+Huauthor=Y.+Wangauthor=L.+Y.+Haoauthor=X.+Liuauthor=C.+A.+Leschauthor=B.+M.+Sanchezauthor=J.+M.+Wendlingauthor=R.+W.+Morganauthor=T.+D.+Aicherauthor=L.+L.+Carterauthor=P.+L.+Toogoodauthor=G.+D.+Glick&title=Sterol+metabolism+controls+TH17+differentiation+by+generating+endogenous+ROR%CE%B3+agonists&doi=10.1038%2Fnchembio.1714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sterol metabolism controls TH17 differentiation by generating endogenous RORγ agonists</span></div><div class="casAuthors">Hu, Xiao; Wang, Yahong; Hao, Ling-Yang; Liu, Xikui; Lesch, Chuck A.; Sanchez, Brian M.; Wendling, Jay M.; Morgan, Rodney W.; Aicher, Tom D.; Carter, Laura L.; Toogood, Peter L.; Glick, Gary D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptor γ (RORγt) controls the differentiation of naive CD4+ T cells into the TH17 lineage, which are crit. cells in the pathogenesis of autoimmune diseases.  Here the authors report that during TH17 differentiation, cholesterol biosynthesis and uptake programs are induced, whereas their metab. and efflux programs are suppressed.  These changes result in the accumulation of the cholesterol precursor, desmosterol, which functions as a potent endogenous RORγ agonist.  Generation of cholesterol precursors is essential for TH17 differentiation as blocking cholesterol synthesis with chem. inhibitors at steps before the formation of active precursors reduces differentiation.  Upon activation, metabolic changes also lead to prodn. of specific sterol-sulfate conjugates that favor activation of RORγ over the TH17-inhibiting sterol receptor LXR.  Thus, TH17 differentiation is orchestrated by coordinated sterol synthesis, mobilization and metab. to selectively activate RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKonRBn05nmbVg90H21EOLACvtfcHk0ljz5nMKzJgshw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislOltQ%253D%253D&md5=e82af9d561a1e4683a86b815cb734971</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1714%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DL.%2BY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLesch%26aufirst%3DC.%2BA.%26aulast%3DSanchez%26aufirst%3DB.%2BM.%26aulast%3DWendling%26aufirst%3DJ.%2BM.%26aulast%3DMorgan%26aufirst%3DR.%2BW.%26aulast%3DAicher%26aufirst%3DT.%2BD.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DGlick%26aufirst%3DG.%2BD.%26atitle%3DSterol%2520metabolism%2520controls%2520TH17%2520differentiation%2520by%2520generating%2520endogenous%2520ROR%25CE%25B3%2520agonists%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D141%26epage%3D147%26doi%3D10.1038%2Fnchembio.1714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huh, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krout, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malapaka, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciofani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santori, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gin, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastinejad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">Digoxin and its derivatives suppress TH17 cell differentiation by antagonizing RORγt activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>472</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1038/nature09978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnature09978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21441909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=472&publication_year=2011&pages=486-490&author=J.+R.+Huhauthor=M.+W.+Leungauthor=P.+Huangauthor=D.+A.+Ryanauthor=M.+R.+Kroutauthor=R.+R.+Malapakaauthor=J.+Chowauthor=N.+Manelauthor=M.+Ciofaniauthor=S.+V.+Kimauthor=A.+Cuestaauthor=F.+R.+Santoriauthor=J.+J.+Lafailleauthor=H.+E.+Xuauthor=D.+Y.+Ginauthor=F.+Rastinejadauthor=D.+R.+Littman&title=Digoxin+and+its+derivatives+suppress+TH17+cell+differentiation+by+antagonizing+ROR%CE%B3t+activity&doi=10.1038%2Fnature09978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Digoxin and its derivatives suppress TH17 cell differentiation by antagonizing RORγt activity</span></div><div class="casAuthors">Huh, Jun R.; Leung, Monica W. L.; Huang, Pengxiang; Ryan, Daniel A.; Krout, Michael R.; Malapaka, Raghu R. V.; Chow, Jonathan; Manel, Nicolas; Ciofani, Maria; Kim, Sangwon V.; Cuesta, Adolfo; Santori, Fabio R.; Lafaille, Juan J.; Xu, H. Eric; Gin, David Y.; Rastinejad, Fraydoon; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">472</span>
        (<span class="NLM_cas:issue">7344</span>),
    <span class="NLM_cas:pages">486-490</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD4+ T helper lymphocytes that express interleukin-17 (TH17 cells) have crit. roles in mouse models of autoimmunity, and there is mounting evidence that they also influence inflammatory processes in humans.  Genome-wide assocn. studies in humans have linked genes involved in TH17 cell differentiation and function with susceptibility to Crohn's disease, rheumatoid arthritis and psoriasis.  Thus, the pathway towards differentiation of TH17 cells and, perhaps, of related innate lymphoid cells with similar effector functions, is an attractive target for therapeutic applications.  Mouse and human TH17 cells are distinguished by expression of the retinoic acid receptor-related orphan nuclear receptor RORγt, which is required for induction of IL-17 transcription and for the manifestation of TH17-dependent autoimmune disease in mice.  By performing a chem. screen with an insect cell-based reporter system, we identified the cardiac glycoside digoxin as a specific inhibitor of RORγt transcriptional activity.  Digoxin inhibited murine TH17 cell differentiation without affecting differentiation of other T cell lineages and was effective in delaying the onset and reducing the severity of autoimmune disease in mice.  At high concns., digoxin is toxic for human cells, but non-toxic synthetic derivs. 20,22-dihydrodigoxin-21,23-diol and digoxin-21-salicylidene specifically inhibited induction of IL-17 in human CD4+ T cells.  Using these small-mol. compds., we demonstrate that RORγt is important for the maintenance of IL-17 expression in mouse and human effector T cells.  These data indicate that derivs. of digoxin can be used as chem. templates for the development of RORγt-targeted therapeutic agents that attenuate inflammatory lymphocyte function and autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZeBtm9HrHi7Vg90H21EOLACvtfcHk0ljz5nMKzJgshw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvVOisr8%253D&md5=d25efc57bc2849ed23456c95c59cc006</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature09978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09978%26sid%3Dliteratum%253Aachs%26aulast%3DHuh%26aufirst%3DJ.%2BR.%26aulast%3DLeung%26aufirst%3DM.%2BW.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DRyan%26aufirst%3DD.%2BA.%26aulast%3DKrout%26aufirst%3DM.%2BR.%26aulast%3DMalapaka%26aufirst%3DR.%2BR.%26aulast%3DChow%26aufirst%3DJ.%26aulast%3DManel%26aufirst%3DN.%26aulast%3DCiofani%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%2BV.%26aulast%3DCuesta%26aufirst%3DA.%26aulast%3DSantori%26aufirst%3DF.%2BR.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DH.%2BE.%26aulast%3DGin%26aufirst%3DD.%2BY.%26aulast%3DRastinejad%26aufirst%3DF.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DDigoxin%2520and%2520its%2520derivatives%2520suppress%2520TH17%2520cell%2520differentiation%2520by%2520antagonizing%2520ROR%25CE%25B3t%2520activity%26jtitle%3DNature%26date%3D2011%26volume%3D472%26spage%3D486%26epage%3D490%26doi%3D10.1038%2Fnature09978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solt, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conkright, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Istrate, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Ordonez, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span> <span> </span><span class="NLM_article-title">The benzenesulfoamide T0901317 [N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide] is a novel retinoic acid receptor-related orphan receptor-alpha/gamma inverse agonist</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1124/mol.109.060905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1124%2Fmol.109.060905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=19887649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFeitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2010&pages=228-236&author=N.+Kumarauthor=L.+A.+Soltauthor=J.+J.+Conkrightauthor=Y.+Wangauthor=M.+A.+Istrateauthor=S.+A.+Busbyauthor=R.+D.+Garcia-Ordonezauthor=T.+P.+Burrisauthor=P.+R.+Griffin&title=The+benzenesulfoamide+T0901317+%5BN-%282%2C2%2C2-trifluoroethyl%29-N-%5B4-%5B2%2C2%2C2-trifluoro-1-hydroxy-1-%28trifluoromethyl%29ethyl%5Dphenyl%5D-benzenesulfonamide%5D+is+a+novel+retinoic+acid+receptor-related+orphan+receptor-alpha%2Fgamma+inverse+agonist&doi=10.1124%2Fmol.109.060905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The benzenesulfoamide T0901317 [N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide] is a novel retinoic acid receptor-related orphan receptor-α/γ inverse agonist</span></div><div class="casAuthors">Kumar, Naresh; Solt, Laura A.; Conkright, Juliana J.; Wang, Yongjun; Istrate, Monica A.; Busby, Scott A.; Garcia-Ordonez, Ruben D.; Burris, Thomas P.; Griffin, Patrick R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">228-236</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related orphan receptors (RORs) regulate a variety of physiol. processes including hepatic gluconeogenesis, lipid metab., circadian rhythm, and immune function.  Here we present the first high-affinity synthetic ligand for both RORα and RORγ.  In a screen against all 48 human nuclear receptors, the benzenesulfonamide liver X receptor (LXR) agonist N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide (T0901317) inhibited transactivation activity of RORα and RORγ but not RORβ.  T0901317 was found to directly bind to RORα and RORγ with high affinity (Ki = 132 and 51 nM, resp.), resulting in the modulation of the receptor's ability to interact with transcriptional cofactor proteins.  T0901317 repressed RORα/γ-dependent transactivation of ROR-responsive reporter genes and in HepG2 cells reduced recruitment of steroid receptor coactivator-2 by RORα at an endogenous ROR target gene (G6Pase).  Using small interference RNA, we demonstrate that repression of the gluconeogenic enzyme glucose-6-phosphatase in HepG2 cells by T0901317 is ROR-dependent and is not due to the compd.'s LXR activity.  In summary, T0901317 represents a novel chem. probe to examine RORα/γ function and an excellent starting point for the development of ROR selective modulators.  More importantly, our results demonstrate that small mols. can be used to target the RORs for therapeutic intervention in metabolic and immune disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1FxGyT5l2qbVg90H21EOLACvtfcHk0ljil5f4Sp32KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFeitLk%253D&md5=04aaf2bdef4b9d64340e221f85838665</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.060905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.060905%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DSolt%26aufirst%3DL.%2BA.%26aulast%3DConkright%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DIstrate%26aufirst%3DM.%2BA.%26aulast%3DBusby%26aufirst%3DS.%2BA.%26aulast%3DGarcia-Ordonez%26aufirst%3DR.%2BD.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26atitle%3DThe%2520benzenesulfoamide%2520T0901317%2520%255BN-%25282%252C2%252C2-trifluoroethyl%2529-N-%255B4-%255B2%252C2%252C2-trifluoro-1-hydroxy-1-%2528trifluoromethyl%2529ethyl%255Dphenyl%255D-benzenesulfonamide%255D%2520is%2520a%2520novel%2520retinoic%2520acid%2520receptor-related%2520orphan%2520receptor-alpha%252Fgamma%2520inverse%2520agonist%26jtitle%3DMol.%2520Pharmacol.%26date%3D2010%26volume%3D77%26spage%3D228%26epage%3D236%26doi%3D10.1124%2Fmol.109.060905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">How many drug targets are there?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=How+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0ljil5f4Sp32KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DHow%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leijten-van de Gevel, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, R. M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Allosteric small molecule modulators of nuclear receptors</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>485</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2019.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.mce.2019.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=30703487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVWhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2019&pages=20-34&author=F.+A.+Meijerauthor=I.+A.+Leijten-van+de+Gevelauthor=R.+M.+J.+M.+de+Vriesauthor=L.+Brunsveld&title=Allosteric+small+molecule+modulators+of+nuclear+receptors&doi=10.1016%2Fj.mce.2019.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric small molecule modulators of nuclear receptors</span></div><div class="casAuthors">Meijer, Femke A.; Leijten-van de Gevel, Iris A.; de Vries, Rens M. J. M.; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-34</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear Receptors (NRs) are multi-domain proteins, whose natural regulation occurs via ligands for a classical, orthosteric, binding pocket and via intra- and inter-domain allosteric mechanisms.  Allosteric modulation of NRs via synthetic small mols. has recently emerged as an interesting entry to address the need for small mols. targeting NRs in pathol., via novel modes of action and with beneficial profiles.  In this review the general concept of allosteric modulation in drug discovery is first discussed, serving as a background and inspiration for NRs.  Subsequently, the review focuses on examples of small mols. that allosterically modulate NRs, with a strong focus on structural information and the ligand binding domain.  Recently discovered nanomolar potent allosteric site NR modulators are catapulting allosteric targeting of NRs to the center of attention.  The obtained insights serve as a basis for recommendations for the next steps to take in allosteric small mol. targeting of NRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsuqvFC3-gY7Vg90H21EOLACvtfcHk0ljil5f4Sp32KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVWhs7k%253D&md5=a76ceb455cdf901bb7d7dae77686afc9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2019.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2019.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DMeijer%26aufirst%3DF.%2BA.%26aulast%3DLeijten-van%2Bde%2BGevel%26aufirst%3DI.%2BA.%26aulast%3Dde%2BVries%26aufirst%3DR.%2BM.%2BJ.%2BM.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DAllosteric%2520small%2520molecule%2520modulators%2520of%2520nuclear%2520receptors%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2019%26volume%3D485%26spage%3D20%26epage%3D34%26doi%3D10.1016%2Fj.mce.2019.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-J.</span></span> <span> </span><span class="NLM_article-title">Non-canonical modulators of nuclear receptors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4157</span>– <span class="NLM_lpage">4164</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.07.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2016.07.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27503683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlaju7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4157-4164&author=C.+M.+Ticeauthor=Y.-J.+Zheng&title=Non-canonical+modulators+of+nuclear+receptors&doi=10.1016%2Fj.bmcl.2016.07.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Non-canonical modulators of nuclear receptors</span></div><div class="casAuthors">Tice, Colin M.; Zheng, Ya-Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4157-4164</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Like G protein-coupled receptors (GPCRs) and protein kinases, nuclear receptors (NRs) are a rich source of pharmaceutical targets.  Over 80 NR-targeting drugs have been approved for 18 NRs.  The focus of drug discovery in NRs has hitherto been on identifying ligands that bind to the canonical ligand binding pockets of the C-terminal ligand binding domains (LBDs).  Due to the development of drug resistance and selectivity concerns, there has been considerable interest in exploring other, non-canonical ligand binding sites.  Unfortunately, the potencies of compds. binding at other sites have generally not been sufficient for clin. development.  However, the situation has changed dramatically over the last 3 years, as compds. with sufficient potency have been reported for several NR targets.  Here we review recent developments in this area from a medicinal chem. point of view in the hope of stimulating further interest in this area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopu9N4QDGsXrVg90H21EOLACvtfcHk0ljil5f4Sp32KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlaju7bN&md5=5677a03f0e607260d148c48e24d49110</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.07.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.07.067%26sid%3Dliteratum%253Aachs%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DZheng%26aufirst%3DY.-J.%26atitle%3DNon-canonical%2520modulators%2520of%2520nuclear%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4157%26epage%3D4164%26doi%3D10.1016%2Fj.bmcl.2016.07.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayne, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J.</span></span> <span> </span><span class="NLM_article-title">Not picking pockets: nuclear receptor alternate-site modulators (NRAMs)</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">683</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1210/me.2009-0362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1210%2Fme.2009-0362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=19933380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVOhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=683-95&author=T.+W.+Mooreauthor=C.+G.+Mayneauthor=J.+Katzenellenbogen&title=Not+picking+pockets%3A+nuclear+receptor+alternate-site+modulators+%28NRAMs%29&doi=10.1210%2Fme.2009-0362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Not picking pockets: nuclear receptor alternate-site modulators (NRAMs)</span></div><div class="casAuthors">Moore, Terry W.; Mayne, Christopher G.; Katzenellenbogen, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-695</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Because of their central importance in gene regulation and mediating the actions of many hormones, the nuclear receptors (NRs) have long been recognized as very important biol. and pharmaceutical targets.  Of all the surfaces available on a given NR, the singular site for regulation of receptor activity has almost invariably been the ligand-binding pocket of the receptor, the site where agonists, antagonists, and selective NR modulators interact.  With our increasing understanding of the multiple mol. components involved in NR action, researchers have recently begun to look to addnl. interaction sites on NRs for regulating their activities by novel mechanisms.  The alternate NR-assocd. interaction sites that have been targeted include the coactivator-binding groove and allosteric sites in the ligand-binding domain, the zinc fingers of the DNA-binding domain, and the NR response element in DNA.  The studies thus far have been performed with the estrogen receptors, the androgen receptor (AR), the thyroid hormone receptors, and the pregnane X receptor.  Phenotypic and conformation-based screens have also identified small mol. modulators that are believed to function through the NRs but have, as yet, unknown sites and mechanisms of action.  The rewards from investigation of these NR alternate-site modulators should be the discovery of new therapeutic approaches and novel agents for regulating the activities of these important NR proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVtxERmnC5SLVg90H21EOLACvtfcHk0lh1uqTVSk7-RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVOhsb4%253D&md5=cd35a2ad9714316347764e692455cf65</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fme.2009-0362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2009-0362%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DT.%2BW.%26aulast%3DMayne%26aufirst%3DC.%2BG.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%26atitle%3DNot%2520picking%2520pockets%253A%2520nuclear%2520receptor%2520alternate-site%2520modulators%2520%2528NRAMs%2529%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D683%26epage%3D95%26doi%3D10.1210%2Fme.2009-0362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheepstra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Almen, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piesvaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutilek, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Eenennaam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagpal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correll, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oubrie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Identification of an allosteric binding site for RORγt inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8833</span>, <span class="refDoi"> DOI: 10.1038/ncomms9833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1038%2Fncomms9833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=26640126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKit7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8833&author=M.+Scheepstraauthor=S.+Leysenauthor=G.+C.+van+Almenauthor=J.+R.+Millerauthor=J.+Piesvauxauthor=V.+Kutilekauthor=H.+van+Eenennaamauthor=H.+Zhangauthor=K.+Barrauthor=S.+Nagpalauthor=S.+M.+Soissonauthor=M.+Kornienkoauthor=K.+Wileyauthor=N.+Elsenauthor=S.+Sharmaauthor=C.+C.+Correllauthor=B.+W.+Trotterauthor=M.+van+der+Steltauthor=A.+Oubrieauthor=C.+Ottmannauthor=G.+Parthasarathyauthor=L.+Brunsveld&title=Identification+of+an+allosteric+binding+site+for+ROR%CE%B3t+inhibition&doi=10.1038%2Fncomms9833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an allosteric binding site for RORγt inhibition</span></div><div class="casAuthors">Scheepstra, Marcel; Leysen, Seppe; van Almen, Geert C.; Miller, J. Richard; Piesvaux, Jennifer; Kutilek, Victoria; van Eenennaam, Hans; Zhang, Hongjun; Barr, Kenneth; Nagpal, Sunil; Soisson, Stephen M.; Kornienko, Maria; Wiley, Kristen; Elsen, Nathaniel; Sharma, Sujata; Correll, Craig C.; Trotter, B. Wesley; van der Stelt, Mario; Oubrie, Arthur; Ottmann, Christian; Parthasarathy, Gopal; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8833/1-8833/10</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RORγt is crit. for the differentiation and proliferation of Th17 cells assocd. with several chronic autoimmune diseases.  We report the discovery of a novel allosteric binding site on the nuclear receptor RORγt.  Co-crystn. of the ligand binding domain (LBD) of RORγt with a series of small-mol. antagonists demonstrates occupancy of a previously unreported allosteric binding pocket.  Binding at this non-canonical site induces an unprecedented conformational reorientation of helix 12 in the RORγt LBD, which blocks cofactor binding.  The functional consequence of this allosteric ligand-mediated conformation is inhibition of function as evidenced by both biochem. and cellular studies.  RORγt function is thus antagonized in a manner molecularly distinct from that of previously described orthosteric RORγt ligands.  This brings forward an approach to target RORγt for the treatment of Th17-mediated autoimmune diseases.  The elucidation of an unprecedented modality of pharmacol. antagonism establishes a mechanism for modulation of nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcktcUUFl77Vg90H21EOLACvtfcHk0lh1uqTVSk7-RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKit7zL&md5=409b08232ecdd3f3e82fa660585c989d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fncomms9833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9833%26sid%3Dliteratum%253Aachs%26aulast%3DScheepstra%26aufirst%3DM.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3Dvan%2BAlmen%26aufirst%3DG.%2BC.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DPiesvaux%26aufirst%3DJ.%26aulast%3DKutilek%26aufirst%3DV.%26aulast%3Dvan%2BEenennaam%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBarr%26aufirst%3DK.%26aulast%3DNagpal%26aufirst%3DS.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DWiley%26aufirst%3DK.%26aulast%3DElsen%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DCorrell%26aufirst%3DC.%2BC.%26aulast%3DTrotter%26aufirst%3DB.%2BW.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26aulast%3DOubrie%26aufirst%3DA.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520an%2520allosteric%2520binding%2520site%2520for%2520ROR%25CE%25B3t%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8833%26doi%3D10.1038%2Fncomms9833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouvry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouix-Peter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciesielski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chantalat, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feret, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luzy, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musicki, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orfila, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parnet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taquet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoreau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennequin, L. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of phenoxyindazoles and phenylthioindazoles as RORγ inverse agonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5802</span>– <span class="NLM_lpage">5808</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.bmcl.2016.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=27815118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslynsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5802-5808&author=G.+Ouvryauthor=C.+Bouix-Peterauthor=F.+Ciesielskiauthor=L.+Chantalatauthor=O.+Christinauthor=C.+Cominoauthor=D.+Duvertauthor=C.+Feretauthor=C.+S.+Harrisauthor=L.+Lamyauthor=A.+P.+Luzyauthor=B.+Musickiauthor=D.+Orfilaauthor=J.+Pascauauthor=V.+Parnetauthor=A.+Perrinauthor=R.+Pierreauthor=G.+Polgeauthor=C.+Raffinauthor=Y.+Rivalauthor=N.+Taquetauthor=E.+Thoreauauthor=L.+F.+Hennequin&title=Discovery+of+phenoxyindazoles+and+phenylthioindazoles+as+ROR%CE%B3+inverse+agonists&doi=10.1016%2Fj.bmcl.2016.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of phenoxyindazoles and phenylthioindazoles as RORγ inverse agonists</span></div><div class="casAuthors">Ouvry, Gilles; Bouix-Peter, Claire; Ciesielski, Fabrice; Chantalat, Laurent; Christin, Olivier; Comino, Catherine; Duvert, Denis; Feret, Christophe; Harris, Craig S.; Lamy, Laurent; Luzy, Anne-Pascale; Musicki, Branislav; Orfila, Danielle; Pascau, Jonathan; Parnet, Veronique; Perrin, Agnes; Pierre, Romain; Polge, Gaelle; Raffin, Catherine; Rival, Yves; Taquet, Nathalie; Thoreau, Etienne; Hennequin, Laurent F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5802-5808</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeting the IL17 pathway and more specifically the nuclear receptor RORγ is thought to be beneficial in multiple skin disorders.  The Letter describes the discovery of phenoxyindazoles and thiophenoxy indazoles as potent RORγ inverse agonists.  Optimization of the potency and efforts to mitigate the phototoxic liability of the series are presented.  Finally, crystn. of the lead compd. revealed that the series bound to an allosteric site of the nuclear receptor.  Such compds. could be useful as tool compds. for understanding the impact of topical treatment on skin disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX7Dguvi_h8LVg90H21EOLACvtfcHk0lg51NxqVEhCrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslynsr%252FM&md5=cd3aa2c246cfa35c98306096d823dccc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DOuvry%26aufirst%3DG.%26aulast%3DBouix-Peter%26aufirst%3DC.%26aulast%3DCiesielski%26aufirst%3DF.%26aulast%3DChantalat%26aufirst%3DL.%26aulast%3DChristin%26aufirst%3DO.%26aulast%3DComino%26aufirst%3DC.%26aulast%3DDuvert%26aufirst%3DD.%26aulast%3DFeret%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DC.%2BS.%26aulast%3DLamy%26aufirst%3DL.%26aulast%3DLuzy%26aufirst%3DA.%2BP.%26aulast%3DMusicki%26aufirst%3DB.%26aulast%3DOrfila%26aufirst%3DD.%26aulast%3DPascau%26aufirst%3DJ.%26aulast%3DParnet%26aufirst%3DV.%26aulast%3DPerrin%26aufirst%3DA.%26aulast%3DPierre%26aufirst%3DR.%26aulast%3DPolge%26aufirst%3DG.%26aulast%3DRaffin%26aufirst%3DC.%26aulast%3DRival%26aufirst%3DY.%26aulast%3DTaquet%26aufirst%3DN.%26aulast%3DThoreau%26aufirst%3DE.%26aulast%3DHennequin%26aufirst%3DL.%2BF.%26atitle%3DDiscovery%2520of%2520phenoxyindazoles%2520and%2520phenylthioindazoles%2520as%2520ROR%25CE%25B3%2520inverse%2520agonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5802%26epage%3D5808%26doi%3D10.1016%2Fj.bmcl.2016.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIsaac, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce-Shaikh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristescu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudgett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piers, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltus, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehmet, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aicher, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawwari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correll, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of RORγT skews TCRα gene rearrangement and limits T cell repertoire diversity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3206</span>– <span class="NLM_lpage">3218</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.11.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.celrep.2016.11.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28009290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnsrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=3206-3218&author=Y.+Guoauthor=K.+D.+MacIsaacauthor=Y.+Chenauthor=R.+J.+Millerauthor=R.+Jainauthor=B.+Joyce-Shaikhauthor=H.+Fergusonauthor=I.+M.+Wangauthor=R.+Cristescuauthor=J.+Mudgettauthor=L.+Engstromauthor=K.+J.+Piersauthor=G.+A.+Baltusauthor=K.+Barrauthor=H.+Zhangauthor=H.+Mehmetauthor=L.+G.+Hegdeauthor=X.+Huauthor=L.+L.+Carterauthor=T.+D.+Aicherauthor=G.+Glickauthor=D.+Zallerauthor=A.+Hawwariauthor=C.+C.+Correllauthor=D.+C.+Jonesauthor=D.+J.+Cua&title=Inhibition+of+ROR%CE%B3T+skews+TCR%CE%B1+gene+rearrangement+and+limits+T+cell+repertoire+diversity&doi=10.1016%2Fj.celrep.2016.11.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RORγT Skews TCRα Gene Rearrangement and Limits T Cell Repertoire Diversity</span></div><div class="casAuthors">Guo, Yanxia; MacIsaac, Kenzie D.; Chen, Yi; Miller, Richard J.; Jain, Renu; Joyce-Shaikh, Barbara; Ferguson, Heidi; Wang, I-Ming; Cristescu, Razvan; Mudgett, John; Engstrom, Laura; Piers, Kyle J.; Baltus, Gretchen A.; Barr, Kenneth; Zhang, Hongjun; Mehmet, Huseyin; Hegde, Laxminarayan G.; Hu, Xiao; Carter, Laura L.; Aicher, Thomas D.; Glick, Gary; Zaller, Dennis; Hawwari, Abbas; Correll, Craig C.; Jones, Dallas C.; Cua, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3206-3218</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent studies have elucidated the mol. mechanism of RORγT transcriptional regulation of Th17 differentiation and function.  RORγT was initially identified as a transcription factor required for thymopoiesis by maintaining survival of CD4+CD8+ (DP) thymocytes.  While RORγ antagonists are currently being developed to treat autoimmunity, it remains unclear how RORγT inhibition may impact thymocyte development.  In this study, we show that in addn. to regulating DP thymocytes survival, RORγT also controls genes that regulate thymocyte migration, proliferation, and T cell receptor (TCR)α selection.  Strikingly, pharmacol. inhibition of RORγ skews TCRα gene rearrangement, limits T cell repertoire diversity, and inhibits development of autoimmune encephalomyelitis.  Thus, targeting RORγT not only inhibits Th17 cell development and function but also fundamentally alters thymic-emigrant recognition of self and foreign antigens.  The anal. of RORγ inhibitors has allowed us to gain a broader perspective of the diverse function of RORγT and its impact on T cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS2YbVQ9dNS7Vg90H21EOLACvtfcHk0lg51NxqVEhCrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnsrjP&md5=6b3766d0673b02fbe6a6b0cc1a10f4a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.11.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.11.073%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DMacIsaac%26aufirst%3DK.%2BD.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMiller%26aufirst%3DR.%2BJ.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DJoyce-Shaikh%26aufirst%3DB.%26aulast%3DFerguson%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DI.%2BM.%26aulast%3DCristescu%26aufirst%3DR.%26aulast%3DMudgett%26aufirst%3DJ.%26aulast%3DEngstrom%26aufirst%3DL.%26aulast%3DPiers%26aufirst%3DK.%2BJ.%26aulast%3DBaltus%26aufirst%3DG.%2BA.%26aulast%3DBarr%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMehmet%26aufirst%3DH.%26aulast%3DHegde%26aufirst%3DL.%2BG.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DAicher%26aufirst%3DT.%2BD.%26aulast%3DGlick%26aufirst%3DG.%26aulast%3DZaller%26aufirst%3DD.%26aulast%3DHawwari%26aufirst%3DA.%26aulast%3DCorrell%26aufirst%3DC.%2BC.%26aulast%3DJones%26aufirst%3DD.%2BC.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520ROR%25CE%25B3T%2520skews%2520TCR%25CE%25B1%2520gene%2520rearrangement%2520and%2520limits%2520T%2520cell%2520repertoire%2520diversity%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D17%26spage%3D3206%26epage%3D3218%26doi%3D10.1016%2Fj.celrep.2016.11.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule RORγt antagonists: one stone kills two birds</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.it.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1016%2Fj.it.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28258823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFWns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=229-231&author=C.+Zhongauthor=J.+Zhu&title=Small-molecule+ROR%CE%B3t+antagonists%3A+one+stone+kills+two+birds&doi=10.1016%2Fj.it.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule RORγt Antagonists: One Stone Kills Two Birds</span></div><div class="casAuthors">Zhong, Chao; Zhu, Jinfang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">229-231</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Although small-mol. thymus-specific isoform of retinoic acid receptor-related orphan nuclear receptor γ (RORγt) antagonists suppressing interleukin (IL)-17-producing T helper (Th17) cells are widely reported, the effect of these mols. on other RORγt-expressing cells is unknown.  However, a new study reports that RORγt inhibition in CD4+CD8+ thymocytes resulted in skewed T cell repertoire, contributing to a redn. in the frequency of self-reactive T cells and resistance to autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ57AU79_MU7Vg90H21EOLACvtfcHk0lirhBAFJ2neMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFWns7o%253D&md5=205d340b7c65ebb90d7a6a356e242927</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DSmall-molecule%2520ROR%25CE%25B3t%2520antagonists%253A%2520one%2520stone%2520kills%2520two%2520birds%26jtitle%3DTrends%2520Immunol.%26date%3D2017%26volume%3D38%26spage%3D229%26epage%3D231%26doi%3D10.1016%2Fj.it.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gege, C.</span></span> <span> </span><span class="NLM_article-title">Retinoid-related orphan receptor γ t modulators: comparison of Glenmark’s me-too patent application (WO2015008234) with the originator application from Merck Sharp and Dohme (WO2012106995)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1215</span>– <span class="NLM_lpage">1221</span>, <span class="refDoi"> DOI: 10.1517/13543776.2015.1065816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1517%2F13543776.2015.1065816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=26156084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWgsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1215-1221&author=C.+Gege&title=Retinoid-related+orphan+receptor+%CE%B3+t+modulators%3A+comparison+of+Glenmark%E2%80%99s+me-too+patent+application+%28WO2015008234%29+with+the+originator+application+from+Merck+Sharp+and+Dohme+%28WO2012106995%29&doi=10.1517%2F13543776.2015.1065816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Retinoid-related orphan receptor γ t modulators: comparison of Glenmark's me-too patent application (WO2015008234) with the originator application from Merck Sharp and Dohme (WO2012106995)</span></div><div class="casAuthors">Gege, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1215-1221</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Retinoic acid receptor-related orphan nuclear receptor γ t (RORγt) is a key transcription factor for the development of TH17 cells.  Inhibiting RORγt activity is thought to be beneficial in targeting a variety of inflammatory and autoimmune disorders.  In their patent application WO2015008234, Glenmark applied a scaffold-hop approach regarding to the originator application from Merck Sharp & Dohme (WO2012106995): a 6-membered annelated arom. moiety is replaced by an annelated 5-membered heteroaryl (exemplified and claimed is however only thiophene), resulting in potent RORγt inverse agonists with a thieno[3,2-b]pyrrole or thieno[3,2-c]pyrazole core.  Based on the patent disclosure, the novelty and utility of these me-too compds. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0QHWy40oFn7Vg90H21EOLACvtfcHk0lirhBAFJ2neMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWgsrjE&md5=e2515513e802fe86494d4f2e0e315eb5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F13543776.2015.1065816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2015.1065816%26sid%3Dliteratum%253Aachs%26aulast%3DGege%26aufirst%3DC.%26atitle%3DRetinoid-related%2520orphan%2520receptor%2520%25CE%25B3%2520t%2520modulators%253A%2520comparison%2520of%2520Glenmark%25E2%2580%2599s%2520me-too%2520patent%2520application%2520%2528WO2015008234%2529%2520with%2520the%2520originator%2520application%2520from%2520Merck%2520Sharp%2520and%2520Dohme%2520%2528WO2012106995%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2015%26volume%3D25%26spage%3D1215%26epage%3D1221%26doi%3D10.1517%2F13543776.2015.1065816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smondyrev, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">PHASE: A new engine for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1007/s10822-006-9087-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1007%2Fs10822-006-9087-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17124629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCksbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=647-671&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=E.+H.+Knollauthor=S.+N.+Raoauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=PHASE%3A+A+new+engine+for+pharmacophore+perception%2C+3D+QSAR+model+development%2C+and+3D+database+screening%3A+1.+Methodology+and+preliminary+results&doi=10.1007%2Fs10822-006-9087-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PHASE: A new design for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Knoll, Eric H.; Rao, Shashidhar N.; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10/11</span>),
    <span class="NLM_cas:pages">647-671</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We introduce PHASE, a highly flexible system for common pharmacophore identification and assessment, 3D QSAR model development, and 3D database creation and searching.  The primary workflows and tasks supported by PHASE are described, and details of the underlying scientific methodologies are provided. Using results from previously published investigations, PHASE is compared directly to other ligand-based software for its ability to identify target pharmacophores, rationalize structure-activity data, and predict activities of external compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof5WS3oFk3MLVg90H21EOLACvtfcHk0lhxJCZq-BoTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCksbvE&md5=512f55e6e3c3dfbae3fc94886482b69a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9087-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9087-6%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DRao%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DPHASE%253A%2520A%2520new%2520engine%2520for%2520pharmacophore%2520perception%252C%25203D%2520QSAR%2520model%2520development%252C%2520and%25203D%2520database%2520screening%253A%25201.%2520Methodology%2520and%2520preliminary%2520results%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D647%26epage%3D671%26doi%3D10.1007%2Fs10822-006-9087-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smondyrev, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S. N.</span></span> <span> </span><span class="NLM_article-title">PHASE: A novel approach to pharmacophore modeling and 3D database searching</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2006.00384.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1111%2Fj.1747-0285.2006.00384.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=16784462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=370-372&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=S.+N.+Rao&title=PHASE%3A+A+novel+approach+to+pharmacophore+modeling+and+3D+database+searching&doi=10.1111%2Fj.1747-0285.2006.00384.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel approach to pharmacophore modeling and 3D database searching</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Rao, Shashidhar N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">370-372</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmacophore modeling and 3D database searching are now recognized as integral components of lead discovery and lead optimization, and the continuing need for improved pharmacophore-based tools has driven the development of PHASE.  PHASE pharmacophore and 3D QSAR models provide a means for rationalizing structure-activity data, predicting the activity of new compds., and efficiently extg. actives from a prototypical database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo50KpRHTGgAbVg90H21EOLACvtfcHk0lhxJCZq-BoTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D&md5=30d06fe99776e203eed35b0f774bbe64</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00384.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00384.x%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DRao%26aufirst%3DS.%2BN.%26atitle%3DPHASE%253A%2520A%2520novel%2520approach%2520to%2520pharmacophore%2520modeling%2520and%25203D%2520database%2520searching%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D370%26epage%3D372%26doi%3D10.1111%2Fj.1747-0285.2006.00384.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Asinex Corp.</span> Gold and platinum collections,
release: 1994–2004, update: 2016–10, <a href="http://www.asinex.com/libraries-html/libraries_gold_platinum-html" class="extLink">www.asinex.com/libraries-html/libraries_gold_platinum-html</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Asinex+Corp.+Gold+and+platinum+collections%2C%0Arelease%3A+1994%E2%80%932004%2C+update%3A+2016%E2%80%9310%2C+www.asinex.com%2Flibraries-html%2Flibraries_gold_platinum-html."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudrel-Grosay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Chiral mercaptoacetamides display enantioselective inhibition of histone deacetylase 6 and exhibit neuroprotection in cortical neuron models of oxidative stress</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201100522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1002%2Fcmdc.201100522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=22234885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=425-39&author=J.+H.+Kalinauthor=H.+Zhangauthor=S.+Gaudrel-Grosayauthor=G.+Vistoliauthor=A.+P.+Kozikowski&title=Chiral+mercaptoacetamides+display+enantioselective+inhibition+of+histone+deacetylase+6+and+exhibit+neuroprotection+in+cortical+neuron+models+of+oxidative+stress&doi=10.1002%2Fcmdc.201100522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral Mercaptoacetamides Display Enantioselective Inhibition of Histone Deacetylase 6 and Exhibit Neuroprotection in Cortical Neuron Models of Oxidative Stress</span></div><div class="casAuthors">Kalin, Jay H.; Zhang, Hankun; Gaudrel-Grosay, Sophie; Vistoli, Giulio; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-439</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mercaptoacetamide-based ligands have been designed as a new class of histone deacetylase (HDAC) inhibitors for possible use in the treatment of neurodegenerative diseases.  The thiol group of these compds. provides a key binding element for interaction with the catalytic zinc ion, and thus differs from the more typically employed hydroxamic acid based zinc binding groups.  Herein we disclose the chem. and biol. of some substituted mercaptoacetamides with the intention of increasing HDAC6 isoform selectivity while maintaining potency similar to their hydroxamic acid analogs.  The introduction of a stereocenter α to the thiol group was found to have a considerable impact on HDAC inhibitor potency.  These new compds. were also profiled for their therapeutic potential in an in vitro model of stress-induced neuronal injury and were found to act as nontoxic neuroprotective agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBURxV9LVkJbVg90H21EOLACvtfcHk0lhxJCZq-BoTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKitA%253D%253D&md5=245d434560a99b41c32f6532fd233e0b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100522%26sid%3Dliteratum%253Aachs%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGaudrel-Grosay%26aufirst%3DS.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DChiral%2520mercaptoacetamides%2520display%2520enantioselective%2520inhibition%2520of%2520histone%2520deacetylase%25206%2520and%2520exhibit%2520neuroprotection%2520in%2520cortical%2520neuron%2520models%2520of%2520oxidative%2520stress%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D425%26epage%3D39%26doi%3D10.1002%2Fcmdc.201100522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0li5xG-s-c_enw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0li5xG-s-c_enw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abreu, R. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froufe, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, P. O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queiroz, M. J. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, I. C. F. R.</span></span> <span> </span><span class="NLM_article-title">ChemT, an open-source software for building template-based chemical libraries</span>. <i>SAR QSAR Environ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1080/1062936X.2011.604097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1080%2F1062936X.2011.604097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21846264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVCrt7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=603-610&author=R.+M.+V.+Abreuauthor=H.+J.+C.+Froufeauthor=P.+O.+M.+Danielauthor=M.+J.+R.+P.+Queirozauthor=I.+C.+F.+R.+Ferreira&title=ChemT%2C+an+open-source+software+for+building+template-based+chemical+libraries&doi=10.1080%2F1062936X.2011.604097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">ChemT, an open-source software for building template-based chemical libraries</span></div><div class="casAuthors">Abreu, R. M. V.; Froufe, H. J. C.; Daniel, P. O. M.; Queiroz, M. J. R. P.; Ferreira, I. C. F. R.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">603-610</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1026-776X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In computational chem., vast quantities of compds. are generated, and there is a need for cheminformatic tools to efficiently build chem. compd. libraries.  Several software tools for drawing and editing compd. structures are available, but they lack options for automatic generation of chem. libraries.  We have implemented ChemT, an easy-to-use open-source software tool that automates the process of prepg. custom-made template-based chem. libraries.  ChemT automatically generates three-dimensional chem. libraries by inputting a chem. template and the functional groups of interest.  The graphical user interface of ChemT is self-explanatory, and a complete tutorial is provided.  Several file formats are accepted by ChemT, and it is possible to filter the generated compds. according to different physicochem. properties.  The compds. can be subject to force field minimization, and the resulting three-dimensional structures recorded on commonly used file formats.  ChemT may be a valuable tool for investigators interested in using in silico virtual screening tools, such as quant. structure-activity relationship modeling or mol. docking, in order to prioritize compds. for further chem. synthesis.  To demonstrate the usefulness of ChemT, we describe an example based on a thieno[3,2-b]pyridine template.  ChemT is available free of charge from our website at .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcglJxM_ucLVg90H21EOLACvtfcHk0ljfqE901L70vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVCrt7zJ&md5=cfde2f5c3c4e4bdd570a5fe2be633cb7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1080%2F1062936X.2011.604097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1062936X.2011.604097%26sid%3Dliteratum%253Aachs%26aulast%3DAbreu%26aufirst%3DR.%2BM.%2BV.%26aulast%3DFroufe%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DDaniel%26aufirst%3DP.%2BO.%2BM.%26aulast%3DQueiroz%26aufirst%3DM.%2BJ.%2BR.%2BP.%26aulast%3DFerreira%26aufirst%3DI.%2BC.%2BF.%2BR.%26atitle%3DChemT%252C%2520an%2520open-source%2520software%2520for%2520building%2520template-based%2520chemical%2520libraries%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2011%26volume%3D22%26spage%3D603%26epage%3D610%26doi%3D10.1080%2F1062936X.2011.604097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, J.</span></span> <span> </span><span class="NLM_article-title">Preparation of 3-bromopropiolic esters: methyl and tert-butyl 3-bromopropiolates</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">212</span>, <span class="refDoi"> DOI: 10.15227/orgsyn.074.0212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.15227%2Forgsyn.074.0212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXjtFyqtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1997&pages=212&author=J.+Leroy&title=Preparation+of+3-bromopropiolic+esters%3A+methyl+and+tert-butyl+3-bromopropiolates&doi=10.15227%2Forgsyn.074.0212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of 3-bromopropiolic esters: methyl and tert-butyl 3-bromopropiolates</span></div><div class="casAuthors">Leroy, J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Syntheses</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">212-216</span>CODEN:
                <span class="NLM_cas:coden">ORSYAT</span>;
        ISSN:<span class="NLM_cas:issn">0078-6209</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">Brominating HC≡CCO2R (R = Me, Me3C) with NBS in Me2CO contg. AgNO3 gave 97% BrC≡CCO2R (same R).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojmQPJhHGjqrVg90H21EOLACvtfcHk0ljfqE901L70vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtFyqtL8%253D&md5=e67ff62d7de1e80d2cb6660fa7a0d5cd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.15227%2Forgsyn.074.0212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15227%252Forgsyn.074.0212%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DJ.%26atitle%3DPreparation%2520of%25203-bromopropiolic%2520esters%253A%2520methyl%2520and%2520tert-butyl%25203-bromopropiolates%26jtitle%3DOrg.%2520Synth.%26date%3D1997%26volume%3D74%26spage%3D212%26doi%3D10.15227%2Forgsyn.074.0212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oakdale, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sit, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Ruthenium-catalyzed cycloadditions of 1-haloalkynes with nitrile oxides and organic azides: synthesis of 4-haloisoxazoles and 5-halotriazoles</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">11101</span>– <span class="NLM_lpage">11110</span>, <span class="refDoi"> DOI: 10.1002/chem.201402559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1002%2Fchem.201402559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=25059647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1WksrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=11101-11110&author=J.+S.+Oakdaleauthor=R.+K.+Sitauthor=V.+V.+Fokin&title=Ruthenium-catalyzed+cycloadditions+of+1-haloalkynes+with+nitrile+oxides+and+organic+azides%3A+synthesis+of+4-haloisoxazoles+and+5-halotriazoles&doi=10.1002%2Fchem.201402559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ruthenium-Catalyzed Cycloadditions of 1-Haloalkynes with Nitrile Oxides and Organic Azides: Synthesis of 4-Haloisoxazoles and 5-Halotriazoles</span></div><div class="casAuthors">Oakdale, James S.; Sit, Rakesh K.; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">11101-11110</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">(Cyclopentadienyl)(cyclooctadiene) ruthenium(II) chloride [CpRuCl(cod)] catalyzes the reaction between nitrile oxides and electronically deficient 1-choro-, 1-bromo-, and 1-iodoalkynes leading to 4-haloisoxazoles.  Org. azides are also suitable 1,3-dipoles, resulting in 5-halo-1,2,3-triazoles.  These air-tolerant reactions can be performed at room temp. with 1.25 equiv of the resp. 1,3-dipole relative to the alkyne component.  Reactive 1-haloalkynes include propiolic amides, esters, ketones, and phosphonates.  Post-functionalization of the halogenated azole products can be accomplished by using palladium-catalyzed cross-coupling reactions and by manipulation of reactive amide groups.  The lack of catalysis obsd. with [Cp*RuCl(cod)] (Cp*=pentamethylcyclopentadienyl) is attributed to steric demands of the Cp* (η5-C5Me5) ligand in comparison to the parent Cp (η5-C5H5).  This hypothesis is supported by the poor reactivity of [(η5-C5Me4CF3)RuCl(cod)], which serves as a an isosteric mimic of Cp* and as an isoelectronic analog of Cp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLjNQBLY8HrLVg90H21EOLACvtfcHk0ljfqE901L70vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1WksrfO&md5=a469ce8399cb624a3cad71869b869485</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fchem.201402559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201402559%26sid%3Dliteratum%253Aachs%26aulast%3DOakdale%26aufirst%3DJ.%2BS.%26aulast%3DSit%26aufirst%3DR.%2BK.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DRuthenium-catalyzed%2520cycloadditions%2520of%25201-haloalkynes%2520with%2520nitrile%2520oxides%2520and%2520organic%2520azides%253A%2520synthesis%2520of%25204-haloisoxazoles%2520and%25205-halotriazoles%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D11101%26epage%3D11110%26doi%3D10.1002%2Fchem.201402559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckman, D. O.</span>; <span class="NLM_string-name">Nicholas, J. B.</span>; <span class="NLM_string-name">Emayan, K.</span>; <span class="NLM_string-name">Seiwert, S. D.</span>; <span class="NLM_string-name">Yuan, S.</span></span> <span> </span><span class="NLM_article-title">Lysophosphatidic acid receptor antagonists</span>. International patent <span class="NLM_patent">WO/2014/113485</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+O.+Buckman&author=J.+B.+Nicholas&author=K.+Emayan&author=S.+D.+Seiwert&author=S.+Yuan&title=Lysophosphatidic+acid+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBuckman%26aufirst%3DD.%2BO.%26atitle%3DLysophosphatidic%2520acid%2520receptor%2520antagonists%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkenburg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojdyr, M.</span></span> <span> </span><span class="NLM_article-title">CCP412: the new graphical user interface to the CCP4 program suite</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1107/S2059798317016035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798317016035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=29533233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFartLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=68-84&author=L.+Pottertonauthor=J.+Agirreauthor=C.+Ballardauthor=K.+Cowtanauthor=E.+Dodsonauthor=P.+R.+Evansauthor=H.+T.+Jenkinsauthor=R.+Keeganauthor=E.+Krissinelauthor=K.+Stevensonauthor=A.+Lebedevauthor=S.+J.+McNicholasauthor=R.+A.+Nichollsauthor=M.+Nobleauthor=N.+S.+Pannuauthor=C.+Rothauthor=G.+Sheldrickauthor=P.+Skubakauthor=J.+Turkenburgauthor=V.+Uskiauthor=F.+von+Delftauthor=D.+Watermanauthor=K.+Wilsonauthor=M.+Winnauthor=M.+Wojdyr&title=CCP412%3A+the+new+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS2059798317016035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">CCP4i2: the new graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Liz; Agirre, Jon; Ballard, Charles; Cowtan, Kevin; Dodson, Eleanor; Evans, Phil R.; Jenkins, Huw T.; Keegan, Ronan; Krissinel, Eugene; Stevenson, Kyle; Lebedev, Andrey; McNicholas, Stuart J.; Nicholls, Robert A.; Noble, Martin; Pannu, Navraj S.; Roth, Christian; Sheldrick, George; Skubak, Pavol; Turkenburg, Johan; Uski, Ville; von Delft, Frank; Waterman, David; Wilson, Keith; Winn, Martyn; Wojdyr, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">68-84</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, No. 4) software suite for macromol. structure detn. by X-ray crystallog. groups brings together many programs and libraries that, by means of well established conventions, interoperate effectively without adhering to strict design guidelines.  Because of this inherent flexibility, users are often presented with diverse, even divergent, choices for solving every type of problem.  Recently, CCP4 introduced CCP4i2, a modern graphical interface designed to help structural biologists to navigate the process of structure detn., with an emphasis on pipelining and the streamlined presentation of results.  In addn., CCP4i2 provides a framework for writing structure-soln. scripts that can be built up incrementally to create increasingly automatic procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ2hzpmyiYpLVg90H21EOLACvtfcHk0li7eGpjbMNDDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFartLY%253D&md5=b9d1efe5ebbe0236027a07dbe2556892</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS2059798317016035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317016035%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DAgirre%26aufirst%3DJ.%26aulast%3DBallard%26aufirst%3DC.%26aulast%3DCowtan%26aufirst%3DK.%26aulast%3DDodson%26aufirst%3DE.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DJenkins%26aufirst%3DH.%2BT.%26aulast%3DKeegan%26aufirst%3DR.%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DStevenson%26aufirst%3DK.%26aulast%3DLebedev%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DNoble%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRoth%26aufirst%3DC.%26aulast%3DSheldrick%26aufirst%3DG.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DJ.%26aulast%3DUski%26aufirst%3DV.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DWaterman%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DK.%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DWojdyr%26aufirst%3DM.%26atitle%3DCCP412%253A%2520the%2520new%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D68%26epage%3D84%26doi%3D10.1107%2FS2059798317016035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clabbers, M. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruene, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrahams, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span> <span> </span><span class="NLM_article-title">Electron diffraction data processing with DIALS</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1107/S2059798318007726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798318007726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=29872002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=506-518&author=M.+T.+B.+Clabbersauthor=T.+Grueneauthor=J.+M.+Parkhurstauthor=J.+P.+Abrahamsauthor=D.+G.+Waterman&title=Electron+diffraction+data+processing+with+DIALS&doi=10.1107%2FS2059798318007726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Electron diffraction data processing with DIALS</span></div><div class="casAuthors">Clabbers, Max T. B.; Gruene, Tim; Parkhurst, James M.; Abrahams, Jan Pieter; Waterman, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-518</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Electron diffraction is a relatively novel alternative to X-ray crystallog. for the structure detn. of macromols. from three-dimensional nanometer-sized crystals.  The continuous-rotation method of data collection has been adapted for the electron microscope.  However, there are important differences in geometry that must be considered for successful data integration.  The wavelength of electrons in a TEM is typically around 40 times shorter than that of X-rays, implying a nearly flat Ewald sphere, and consequently low diffraction angles and a high effective sample-to-detector distance.  Nevertheless, the DIALS software package can, with specific adaptations, successfully process continuous-rotation electron diffraction data.  Pathologies encountered specifically in electron diffraction make data integration more challenging.  Errors can arise from instrumentation, such as beam drift or distorted diffraction patterns from lens imperfections.  The diffraction geometry brings addnl. challenges such as strong correlation between lattice parameters and detector distance.  These issues are compounded if calibration is incomplete, leading to uncertainty in exptl. geometry, such as the effective detector distance and the rotation rate or direction.  Dynamic scattering, absorption, radiation damage and incomplete wedges of data are addnl. factors that complicate data processing.  Here, recent features of DIALS as adapted to electron diffraction processing are shown, including diagnostics for problematic diffraction geometry refinement, refinement of a smoothly varying beam model and corrections for distorted diffraction images.  These novel features, combined with the existing tools in DIALS, make data integration and refinement feasible for electron crystallog., even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ2xvp5nytBbVg90H21EOLACvtfcHk0li7eGpjbMNDDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM&md5=0ef3d54b8435ed67f31774547a35d263</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS2059798318007726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798318007726%26sid%3Dliteratum%253Aachs%26aulast%3DClabbers%26aufirst%3DM.%2BT.%2BB.%26aulast%3DGruene%26aufirst%3DT.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DAbrahams%26aufirst%3DJ.%2BP.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26atitle%3DElectron%2520diffraction%2520data%2520processing%2520with%2520DIALS%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D506%26epage%3D518%26doi%3D10.1107%2FS2059798318007726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span> <span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lh25op9Atzoug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title"><i>AceDRG:</i> a stereochemical description generator for ligands</span>. <i>Acta Crystallogr. D. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/S2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=28177307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Grazulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+a+stereochemical+description+generator+for+ligands&doi=10.1107%2FS2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lh25op9Atzoug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGrazulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520a%2520stereochemical%2520description%2520generator%2520for%2520ligands%26jtitle%3DActa%2520Crystallogr.%2520D.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh25op9Atzoug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title"><i>REFMAC</i>5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lhLqex1cl7RHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span> <i>The PyMOL Molecular
Graphics System, version 2.2.3</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>, <span class="NLM_year">2000</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+version+2.2.3%3B+Schr%C3%B6dinger%2C%0ALLC%2C+2000."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%252C%2520version%25202.2.3%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NB6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NB6','PDB','4NB6'); return false;">PDB: 4NB6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YPQ','PDB','4YPQ'); return false;">PDB: 4YPQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAL','PDB','6SAL'); return false;">PDB: 6SAL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C4O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C4O','PDB','5C4O'); return false;">PDB: 5C4O</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i87"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01372">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_16202"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01372?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01372</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">S1.0. In silico experiments: phase pharmacophore screen scores (Table S1 and S2), Chem-T C-5 virtual library SMILES strings (Table S3), manually generated C-5 virtual library (Figure S1), and glide docking scores (Tables S4 and S5); S2.0. Chemistry Supporting Information and Spectra: (S2.1) Regiochemical assignment of <b>30a</b>, (S2.2) synthesis of <b>5</b> (Glenmark) and (S2.3) NMR spectra and LC-UV traces for assayed compounds; and S3.0. Crystallography: data collection and refinement statistics (molecular replacement) of <b>25</b> (Table S6), two-dimensional protein–ligand interaction plot for <b>25</b> (Figure S2), and overlay of crystal structure and docking pose for <b>25</b> (Figure S3) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Recommended compound characterization checklist (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_002.xls">XLS</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_003.csv">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_004.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_001.pdf">jm9b01372_si_001.pdf (4.97 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_002.xls">jm9b01372_si_002.xls (102.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_003.csv">jm9b01372_si_003.csv (1.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01372/suppl_file/jm9b01372_si_004.xlsx">jm9b01372_si_004.xlsx (11.33 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the RORγt bound to compound <b>25</b> have been deposited in the Protein Data Bank under accession code 6SAL. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01372&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01372%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-1%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01372" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679921dbd82022d8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
