
<html lang="en"     class="pb-page"  data-request-id="ed6b46ea-2b75-4398-965d-ab53d60f5335"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-19;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00964;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor" /></meta><meta name="dc.Creator" content="Jakub  Gunera" /></meta><meta name="dc.Creator" content="Jillian G.  Baker" /></meta><meta name="dc.Creator" content="Niek  van Hilten" /></meta><meta name="dc.Creator" content="Daniel M.  Rosenbaum" /></meta><meta name="dc.Creator" content="Peter  Kolb" /></meta><meta name="dc.Description" content="The orexin receptors are peptide-sensing G protein-coupled receptors that are intimately linked with regulation of the sleep/wake cycle. We used a recently solved X-ray structure of the orexin rece..." /></meta><meta name="Description" content="The orexin receptors are peptide-sensing G protein-coupled receptors that are intimately linked with regulation of the sleep/wake cycle. We used a recently solved X-ray structure of the orexin rece..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 26, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00964" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00964" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00964" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00964" /></link>
        
    
    

<title>Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00964" /></meta><meta property="og:title" content="Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0009.jpeg" /></meta><meta property="og:description" content="The orexin receptors are peptide-sensing G protein-coupled receptors that are intimately linked with regulation of the sleep/wake cycle. We used a recently solved X-ray structure of the orexin receptor subtype 2 in computational docking calculations with the aim to identify additional ligands with unprecedented chemotypes. We found validated ligands with a high hit rate of 29% out of those tested, none of them showing selectivity with respect to the orexin receptor subtype 1. Furthermore, of the higher-affinity compounds examined, none showed any agonist activity. While novel chemical structures can thus be found, selectivity is a challenge owing to the largely identical binding pockets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00964"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00964">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00964&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00964&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00964&amp;href=/doi/10.1021/acs.jmedchem.0c00964" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 11045-11053</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00956" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00996" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jakub Gunera</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jakub Gunera</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jakub++Gunera">Jakub Gunera</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jillian G. Baker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jillian G. Baker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cell Signalling, School of Life Sciences, Queen’s Medical Centre, University of Nottingham, Nottingham NG7 2UH, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jillian+G.++Baker">Jillian G. Baker</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2371-8202" title="Orcid link">http://orcid.org/0000-0003-2371-8202</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Niek van Hilten</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Niek van Hilten</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Niek++van+Hilten">Niek van Hilten</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel M. Rosenbaum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel M. Rosenbaum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biophysics and Biochemistry, UT Southwestern Medical Center, Dallas, Texas 75390-8816, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+M.++Rosenbaum">Daniel M. Rosenbaum</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Peter Kolb</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Kolb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f28297869780dc999d9e90b2879c9bdf9f938090878095dc9697"><span class="__cf_email__" data-cfemail="671702130215490c080b052712090e4a0a061505121500490302">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Kolb">Peter Kolb</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4089-614X" title="Orcid link">http://orcid.org/0000-0003-4089-614X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00964&amp;href=/doi/10.1021%2Facs.jmedchem.0c00964" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 11045–11053</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 25, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 June 2020</li><li><span class="item_label"><b>Published</b> online</span>26 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00964" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00964</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11045%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJakub%2BGunera%252C%2BJillian%2BG.%2BBaker%252C%2BNiek%2Bvan%2BHilten%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D19%26contentID%3Dacs.jmedchem.0c00964%26title%3DStructure-Based%2BDiscovery%2Bof%2BNovel%2BLigands%2Bfor%2Bthe%2BOrexin%2B2%2BReceptor%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11053%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00964"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1535</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00964" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jakub&quot;,&quot;last_name&quot;:&quot;Gunera&quot;},{&quot;first_name&quot;:&quot;Jillian&quot;,&quot;last_name&quot;:&quot;G. Baker&quot;},{&quot;first_name&quot;:&quot;Niek&quot;,&quot;last_name&quot;:&quot;van Hilten&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;M. Rosenbaum&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Kolb&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;11045-11053&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00964&quot;},&quot;abstract&quot;:&quot;The orexin receptors are peptide-sensing G protein-coupled receptors that are intimately linked with regulation of the sleep/wake cycle. We used a recently solved X-ray structure of the orexin receptor subtype 2 in computational docking calculations with the aim to identify additional ligands with unprecedented chemotypes. We found validated ligands with a high hit rate of 29% out of those tested, none of them showing selectivity with respect to the orexin receptor subtype 1. Furthermore, of the higher-affinity compounds examined, none showed any agonist activity. While novel chemical structures can thus be found, selectivity is a challenge owing to the largely identical binding pockets.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00964&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00964" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00964&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00964" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00964&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00964" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00964&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00964&amp;href=/doi/10.1021/acs.jmedchem.0c00964" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00964" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00964" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00964%26sid%3Dliteratum%253Aachs%26pmid%3D32977721%26genre%3Darticle%26aulast%3DGunera%26date%3D2020%26atitle%3DStructure-Based%2BDiscovery%2Bof%2BNovel%2BLigands%2Bfor%2Bthe%2BOrexin%2B2%2BReceptor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D19%26spage%3D11045%26epage%3D11053%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/jmcmar.2020.63.issue-19/20201008/jmcmar.2020.63.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The orexin receptors are peptide-sensing G protein-coupled receptors that are intimately linked with regulation of the sleep/wake cycle. We used a recently solved X-ray structure of the orexin receptor subtype 2 in computational docking calculations with the aim to identify additional ligands with unprecedented chemotypes. We found validated ligands with a high hit rate of 29% out of those tested, none of them showing selectivity with respect to the orexin receptor subtype 1. Furthermore, of the higher-affinity compounds examined, none showed any agonist activity. While novel chemical structures can thus be found, selectivity is a challenge owing to the largely identical binding pockets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The sleep/wake cycle is one of the fundamental features of organisms with a central nervous system. Especially in humans, it guides our everyday lives and is one of the key activities that keep us healthy and sane. Both sleep deprivation and the inability to sleep (insomnia) are therefore unpleasant for an individual, have been shown to have harmful physiological effects,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and constitute a huge burden on society as a whole.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In order to ameliorate insomnia, two ways can be envisioned: one is to make the brain artificially sleepy and the other is to block the signals that mediate wakefulness. While more traditional insomnia medications have tried to do the former, the latter seems like a strategy that should lead to fewer side effects.</div><div class="NLM_p">In humans, one of the main pathways to transmit wakefulness signals is the orexin peptide/orexin receptor system.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> It consists of two peptides, orexin A and orexin B, and two G protein-coupled receptors (GPCRs), orexin receptor subtype 1 (OX<sub>1</sub>R) and subtype 2 (OX<sub>2</sub>R). The two peptides are 33 and 28 amino acids in length, respectively, and bind with nanomolar affinities to both receptors. At the same time, the two receptors are highly homologous, with 63% sequence identity, and, most importantly, differ by only two amino acid substitutions in the binding pocket. Despite this high sequence similarity, orexin A achieves a certain amount of selectivity for OX<sub>1</sub>R over OX<sub>2</sub>R.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">In 2014, the first antagonist of the OX<sub>2</sub>R, suvorexant, obtained regulatory approval as an insomnia medication in the US. One year later, the crystal structure of this molecule in complex with OX<sub>2</sub>R was published.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> It showed that suvorexant was binding in a “horseshoe shape”, which is consistent with earlier NMR investigations showing that suvorexant was able to adopt such a conformation in solution.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The receptor was confirmed to be a typical GPCR with seven membrane-spanning helices. Moreover, the crystallographic data demonstrated that suvorexant was interacting with the receptor mainly through hydrophobic interactions. In fact, there are only two polar interactions, one a direct hydrogen bond to Asn324<sup>6.55</sup>, the other a water-mediated hydrogen bond with His350<sup>7.39</sup> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two-dimensional representation of the binding mode of suvorexant (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>) generated with the MOE suite. Two interactions of suvorexant with OX<sub>2</sub>R are indicated as dashed green lines: an H-bond between amide carbonyl and Asn324<sup>6.55</sup> and a π–π interaction between 5-methyl-2-triazol-phenyl and His350<sup>7.39</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Aiming to exploit this structure in an unbiased computational screen, we have asked three questions. First, could an unbiased docking screen and the analysis of the predicted binding modes suggest other polar interaction possibilities for small molecules in the binding pocket? Second, could an apolar binding pocket as this one be as suitable for docking (manifested by high hit rates) as the more polar pockets of the aminergic receptors investigated earlier?<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11">(7−11)</a> Lastly, could comparison of our hit molecules with the already known chemical space for the orexin receptors identify novel chemotypes that might present scaffolds worthwhile to elaborate on further with medicinal chemistry?</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Selection of Parent Hits: Primary Docking Screen</h3><div class="NLM_p">Docking of 7.337 million compounds from the DrugsNow subset of the ZINC database into the OX<sub>2</sub>R (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details) and subsequent postprocessing of its outcome resulted in 6500 unique poses, split up in 6000 from the top ranks of the lists sorted according to HYBRID, DSX, or SZYBKI scores and an additional 500 from the calculations with DOCK. The visual inspection of these poses resulted in 85 virtual hit candidates (HYBRID: 14; DSX: 25; SZYBKI: 14, and DOCK: 32 poses (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>)). Forty-three compounds were available from their vendors and analyzed pharmacologically. These compounds will be referred to as the “parent” compounds <b>P1</b>–<b>P43</b> in the following and are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Table S2</a>, with selected compounds also displayed in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic of the primary docking screen workflow. (A) ZINC DrugsNow subset of ∼7.4 million compounds was subjected to conformer generation used for docking. In addition, ∼4 million ZINC LeadLike compounds were docked as is. (B) Three-fold postprocessing of the top 1% slice (6500 poses, 2000 each from HYBRID, DSX, and SZYBKI and 500 from DOCK) concluded by visual inspection, selection of compounds, and their experimental validation. (C) Schematic of the secondary similarity screen workflow. Eight experimentally validated hits were used as queries for a fingerprint-based similarity screen of three 2000 slices from the primary screen. The three sets of 30 nearest neighbors each were optimized with SZYBKI and subjected to visual inspection. Selected virtual hits were validated experimentally.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Validation of Cell Lines and Examination of Primary Docking Screen</h3><div class="NLM_p">Pharmacological studies were conducted in CHO cells stably expressing the human orexin 1 receptor (CHO-OX<sub>1</sub>) or human orexin 2 receptor (CHO-OX<sub>2</sub>). Suvorexant, a known dual OX<sub>1</sub>/OX<sub>2</sub> receptor antagonist (DORA), inhibited the specific binding of both <sup>3</sup>H-SB674042 to the human OX<sub>1</sub>R and <sup>3</sup>H-EMPA to the human OX<sub>2</sub>R with high affinity (pK<sub>i</sub> 8.51 and 8.53, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>)). The known OX<sub>1</sub>R antagonists SB334867 and SB674042 were found to have much higher affinity for the CHO-OX<sub>1</sub> cells than for the CHO-OX<sub>2</sub> cells, whereas the known OX<sub>2</sub>R antagonists JNJ10397049 and EMPA had much higher affinity for the CHO-OX<sub>2</sub> cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These are consistent with previous values and demonstrate the existence of each of the receptors in the two respective cell lines.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> Conversely, in cells not transfected with either receptor, no binding of <sup>3</sup>H-SB674042 or <sup>3</sup>H-EMPA, respectively, could be observed.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. pK<sub>i</sub> Values Obtained for Five Known Orexin Ligands and the Initial 43 Compounds Identified from the Primary Docking Screen from Whole Cell Binding Studies as Determined from Using <sup>3</sup>H-SB674042 (CHO-OX<sub>1</sub> Cells) and <sup>3</sup>H-EMPA (CHO-OX<sub>2</sub> Cells)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>2</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">suvorexant</td><td class="colsep0 rowsep0" align="left">8.51 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="left">8.53 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SB334867</td><td class="colsep0 rowsep0" align="left">7.38 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">5.43 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SB674042</td><td class="colsep0 rowsep0" align="left">8.41 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">6.70 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JNJ10397049</td><td class="colsep0 rowsep0" align="left">5.87 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">8.18 ± 0.14</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EMPA</td><td class="colsep0 rowsep0" align="left">5.35 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="left">8.17 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P3</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P4</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P6</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P7</b></td><td class="colsep0 rowsep0" align="left">4.48 ± 0.08<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.57 ± 0.10<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P8</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P9</b></td><td class="colsep0 rowsep0" align="left">4.85 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">5.40 ± 0.06<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P10</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P11</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P12</b></td><td class="colsep0 rowsep0" align="left">4.75 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.21 ± 0.02<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P13</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.22 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P14</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P15</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.17 ± 0.01<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P16</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P17</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P18</b></td><td class="colsep0 rowsep0" align="left">4.67 ± 0.11<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P19</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P20</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P21</b></td><td class="colsep0 rowsep0" align="left">4.36 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.23 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P22</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.49 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P23</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P24</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P25</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P26</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P27</b></td><td class="colsep0 rowsep0" align="left">4.81 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">5.07 ± 0.03<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P28</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P29</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P30</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P31</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P32</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P33</b></td><td class="colsep0 rowsep0" align="left">4.93 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="left">5.55 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P34</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.69 ± 0.02<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P35</b></td><td class="colsep0 rowsep0" align="left">5.27 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">5.27 ± 0.06<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P36</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P37</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P38</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P39</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P40</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P41</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P42</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P43</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values are mean ± sem from <i>n</i> separate experiments. app = apparent K<sub>i</sub> value. Here, the maximal achievable concentration of competing ligand was not able to fully inhibit the radioligand specific binding. In cases where greater than 50% specific binding was inhibited, an apparent K<sub>i</sub> value is given, assuming that if a greater concentration of ligand was possible, full inhibition of specific binding would have occurred. IC<sub>50</sub> > 100 μM. Here, inhibition of specific binding by the competing ligand was less than 50% so an IC<sub>50</sub> value, and therefore K<sub>i</sub> value, could not be calculated. The IC<sub>50</sub> must therefore have been greater than the maximum concentration of competing ligand. ep = early plateau. Some ligands did not fully inhibit radioligand binding but appeared to reach an early plateau. Here, increasing concentrations of competing ligand reached a maximum inhibition that was less than that achieved by suvorexant (similar to that seen in ref. <a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>). In these cases, an apparent K<sub>i</sub> value is given for the partial inhibition of specific binding. The percentages of specific binding inhibited were 79.2 ± 3.3% for <b>P27</b> and 63.7 ± 2.1% for <b>P35</b> in the CHO-OX<sub>2</sub> cells.</p></div></div></div><div class="NLM_p">Of the 43 initial compounds identified from the primary docking screen, 11 had some degree of measurable affinity for either the OX<sub>1</sub> or OX<sub>2</sub> receptor, giving an initial hit rate of 26%. Of these, <b>P33</b> had the highest affinity with a pK<sub>i</sub> value of 5.54 at the OX<sub>2</sub>R (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, for space reasons, all other compounds that are not among the 10 with the highest affinity are depicted in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Table S2</a>)). Almost all measured compounds did not show selectivity. Five compounds showed a very minor degree of selectivity: <b>P34</b> had a 5-fold OX<sub>2</sub>R selectivity (ΔpK<sub>i</sub> of −0.69), <b>P9</b> and <b>P33</b> had a 4-fold OX<sub>2</sub>R selectivity (ΔpK<sub>i</sub> of −0.62 and –0.59, respectively) while <b>P12</b> and <b>P18</b> had at least a 3-fold OX<sub>1</sub>R selectivity.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition of (A) <sup>3</sup>H-SB67404 binding to CHO-OX<sub>1</sub> cells and (B) <sup>3</sup>H-EMPA binding to CHO-OX<sub>2</sub> cells in response to <b>P33</b>, <b>F33.3</b>, and suvorexant. Bars represent total and nonspecific binding (as determined by 10 μM suvorexant) and data points are mean ± sem of triplicate determinations. These experiments are representative of four separate experiments. The concentration of radioligand in these experiments was (A) 1.14 and (B) 0.60 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ten compounds with the highest measured affinity in the <sup>3</sup>H-EMPA binding assay to CHO-OX<sub>2</sub> cells identified in this study. <b>P33</b> and <b>P35</b> compound families are emphasized with a box.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Examination of Daughter Compounds</h3><div class="NLM_p">For nine parent compounds, daughter derivatives (<b>F</b>) were obtained: We selected between 1 (<b>P22</b> and <b>P31</b>) and 15 (<b>P35</b>) daughter molecules from each parent—in total 54 compounds—based on the availability of similar compounds and their favorable docking scores and poses. The identifiers of the daughter compounds in this text are based on the ID of the parent compound and the order number of each of them, separated by a point. For example, compound <b>F33.1</b> is the first daughter compound of parent <b>P33</b>. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> shows their affinities measured against OX<sub>1</sub>R and OX<sub>2</sub>R. Compound <b>P33</b> can be considered the most fruitful parent compound as all but one of the seven derivatives measured also showed measurable affinity against the OX<sub>2</sub>R. Moreover, its daughter <b>F33.3</b> bound with the highest affinity of all ligands, at a pK<sub>i</sub> of 6.18 (K<sub>i</sub> of 660 nM, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In total, 16 of the 54 derivatives tested bound to the receptor. This brings the hit rate of daughter compounds to 30%. Finally, to further exclude any non-orexin receptor mediated effects, the most potent ligands were examined for their ability to bind to an unrelated receptor, the human β<sub>1</sub>-adrenergic receptor. Although the β<sub>1</sub>-antagonist CGP20712A inhibited specific binding with high affinity, none of the ligands with orexin receptor affinity had any detectable binding for the β<sub>1</sub>-adrenergic receptor (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. pK<sub>i</sub> Values Obtained for Parent and Daughter Compounds from Whole Cell Binding Studies as Determined from Using <sup>3</sup>H-SB674042 (CHO-OX<sub>1</sub> Cells) and <sup>3</sup>H-EMPA (CHO-OX<sub>2</sub> Cells)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>2</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P7</b></td><td class="colsep0 rowsep0" align="left">4.48 ± 0.08<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.57 ± 0.10<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.1</b></td><td class="colsep0 rowsep0" align="left">4.68 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.48 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.3</b></td><td class="colsep0 rowsep0" align="left">5.90 ± 0.10<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">5.83 ± 0.05<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.4</b></td><td class="colsep0 rowsep0" align="left">4.83 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.89 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.6</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.7</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.34 ± 0.06<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7.8</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P9</b></td><td class="colsep0 rowsep0" align="left">4.85 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">5.40 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.1</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.3</b></td><td class="colsep0 rowsep0" align="left">4.49 ± 0.04<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.4</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.6</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.7</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.8</b></td><td class="colsep0 rowsep0" align="left">5.11 ± 0.10<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.97 ± 0.06<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.9</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > −5</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > −5</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.10</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.11</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9.12</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P21</b></td><td class="colsep0 rowsep0" align="left">4.36 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.23 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F21.1</b><sup>a</sup></td><td class="colsep0 rowsep0" align="left">4.83 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.44 ± 0.01<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F21.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F21.3</b><sup>b</sup></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F21.4</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P22</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.49 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F22.1</b><sup>b</sup></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P27</b></td><td class="colsep0 rowsep0" align="left">4.81 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">5.07 ± 0.03<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.2</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.39 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.3</b></td><td class="colsep0 rowsep0" align="left">4.82 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.49 ± 0.01<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.4</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.6</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.23 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.7</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F27.8</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P31</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F31.1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P32</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F32.1</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.22 ± 0.05<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F32.2</b></td><td class="colsep0 rowsep0" align="left">5.10 ± 0.01<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.98 ± 0.06<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P33</b></td><td class="colsep0 rowsep0" align="left">4.93 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="left">5.55 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.1</b><sup>b</sup></td><td class="colsep0 rowsep0" align="left">4.83 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">4.44 ± 0.01<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.2</b></td><td class="colsep0 rowsep0" align="left">4.77 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">5.53 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.3</b></td><td class="colsep0 rowsep0" align="left">5.49 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">6.18 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.4</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.27 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.5</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.6</b></td><td class="colsep0 rowsep0" align="left">5.05 ± 0.12</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">5.50 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F33.7</b></td><td class="colsep0 rowsep0" align="left">5.45 ± 0.13</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">5.50 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>P35</b></td><td class="colsep0 rowsep0" align="left">5.27 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">5.27 ± 0.06<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.1</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.3</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.4</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.6</b></td><td class="colsep0 rowsep0" align="left">4.61 ± 0.03<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.38 ± 0.08<sup>app</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.7</b></td><td class="colsep0 rowsep0" align="left">5.29 ± 0.05<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">5.19 ± 0.07<sup>ep</sup></td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.8</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.9</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.10</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.11</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.12</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.13</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.14</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.15</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F35.16</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The compound corresponding to F21.1 and F33.1 was found twice in the similarity searches for the parent compounds P21 and P33, respectively. Hence, we report it in both family trees but count it only once for the hit rate.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The compound corresponding to F21.3 and F22.1 was found twice in the similarity searches for the parent compounds P21 and P22, respectively. Hence, we report it in both family trees.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Values are mean ± sem from <i>n</i> separate experiments. app = apparent K<sub>i</sub> value (as above, greater than 50% specific binding was inhibited, so an apparent K<sub>i</sub> value is given, assuming that if a greater concentration of ligand was possible, full inhibition of specific binding would have occurred). Where inhibition of specific binding was less than 50%, IC<sub>50</sub> > concentration of maximum concentration of competing ligand used is given. ep = early plateau (as above, competing ligand appeared to reach a plateau whereby increasing competing ligand concentration could not fully inhibit the radioligand. An apparent K<sub>i</sub> value is given for the partial inhibition of specific binding. The percentages of specific binding inhibited were 57.2 ± 2.2% for <b>F7.3</b>, 52.2 ± 2.2% for <b>F9.8</b>, 64.6 ± 3.5% for <b>F32.2</b>, and 58.7 ± 4.7% for <b>F35.7</b> in the CHO-OX<sub>1</sub>. In the CHO-OX<sub>2</sub> cells, the percentages of specific binding inhibited were 61.7 ± 4.6% for <b>F7.3</b>, 49.9 ± 4.5% for <b>F9.8</b>, 39.9 ± 3.5% for <b>F32.2</b>, and 67.1 ± 4.6% for <b>F35.7</b>.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Examination of Functional Responses</h3><div class="NLM_p">Orexin 1 and 2 receptors are both G<sub>q</sub>-coupled GPCRs and therefore stimulate an increase in intracellular calcium release. To examine whether the compounds had an agonist activity, the ability of some of the higher affinity compounds to stimulate intracellular calcium release was measured. Orexin A stimulated a potent agonist response in both the CHO-OX<sub>1</sub> cells (pEC<sub>50</sub> 9.67 ± 0.13, 19.4 ± 2.4 fold over basal, 58.8 ± 2.4% that of 10 μM ionomycin, <i>n</i> = 5) and the CHO-OX<sub>2</sub> cells (pEC<sub>50</sub> 10.31 ± 0.07, 32.8 ± 4.2 fold over basal, 71.8 ± 1.8% that of 10 μM ionomycin, <i>n</i> = 6; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). No agonist response was seen in response to any of the other ligands examined. Thus, no agonist response was seen to parent compounds <b>P9</b>, <b>P33</b>, or <b>P35</b> or daughter compounds <b>F7.3</b>, <b>F33.2</b>, <b>F33.3</b>, <b>F33.6</b>, <b>F33.7</b>, or <b>S2.7</b> (<i>n</i> = 3 to 9 for each ligand at each receptor; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). In addition, no response was seen to parent compounds <b>P7</b>, <b>P21</b>, <b>P22</b>, <b>P27</b>, <b>P32</b>, and <b>P34</b> or daughter compounds <b>F7.1</b>, <b>F21.1</b>, <b>F27.2</b>, and <b>F27.3</b> at the OX<sub>2</sub>R (<i>n</i> = 5). Therefore, these compounds, including the highest affinity parent compound (<b>P33</b>) and derivative (<b>F33.3</b>) had no agonist activity and thus are antagonists at the human OX<sub>1</sub> and OX<sub>2</sub> receptors.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Intracellular calcium release in (A) CHO-OX<sub>1</sub> cells and (B) CHO-OX<sub>2</sub> cells in response to orexin A, <b>P33</b>, and <b>F33.3</b>. Bars represent basal intracellular calcium release and that in response to 10 μM ionomycin alone. Data points are mean ± sem of triplicate determinations and these single experiments are representative of four separate experiments in each case.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structure–Activity Relationship</h3><div class="NLM_p">The lack of polar features in the binding pocket is echoed in the ligands. Therefore, it is challenging to develop a clear-cut SAR for them. There are many chemical solutions for a molecule to form favorable interactions with this receptor. In particular, apolar interactions are relatively tolerant toward subtle geometric changes. This tolerance can be seen here as several bulky substituents had very little effect on affinity.</div><div class="NLM_p">Looking at <b>P33</b> and its derivatives (<b>F33.1</b>, <b>F33.2</b>, <b>F33.3</b>, (<b>F33.4</b>: no affinity), <b>F33.5</b>, <b>F33.6</b>, and <b>F33.7</b>), all these points are confirmed. This miniseries consists of the most potent parent (<b>P33</b>) and daughter molecules (<b>F33.3</b>) with the highest number of active daughter compounds (six). The only polar interaction these molecules form is with Asn324<sup>6.55</sup>, yet the overall binding mode of <b>P33</b> is not suvorexant-like (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, right). The daughters tell us that the position or existence of the halogens on the benzene rings is not important (<b>F33.2</b> and <b>F33.3</b> bind), but that more bulk in this region is not favorable (<b>F33.1</b> is worse). There seems to be more space around the methyl-furane, however, as it can be replaced with bulkier groups (<b>F33.6</b> and <b>F33.7</b>). For these latter molecules, a more suvorexant-like binding mode is conceivable. Interestingly, all the most active compounds show a double acceptor feature, which is absent in the weaker ligands. The most potent compound of this series, <b>F33.3</b>, might also be the only one truly capable of adapting a suvorexant-like conformation according to docking (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, left).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Depictions of (left) <b>F33.3</b> and (right) <b>P33</b> showing poses of the most potent and the most OX<sub>2</sub>R-selective compound, respectively. Compounds are shown in green sticks. Asn324<sup>6.55</sup> and His350<sup>7.39</sup> are shown in orange sticks (bold and thin, respectively). Helix 7 was removed for clarity. Color code: C, green; N, blue; O, red; S, yellow; Cl, dark green; F, cyan. PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For <b>P7</b> and similar (<b>F7.1</b>, (<b>F7.2</b>: no affinity), <b>F7.3</b>, <b>F7.4</b>, (<b>F7.5</b> and <b>F7.6</b>: no affinity), and <b>F7.7</b>), a slightly different picture emerges. Compound <b>P7</b> interacts with residues Glu212<sup>45.52</sup> and Arg328<sup>6.59</sup>, both located at the entrance of the binding site and forming a lid, shielding it from bulk solvent. The derivatives are only bound to Asn324<sup>6.55</sup>. Of note, <b>F7.7</b> binds in an orientation reminiscent of a reverse suvorexant binding mode (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Despite a certain difference in the overall layout, all of the compounds with affinity feature a pyrimidinone ring and a relatively apolar double ring system.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Depictions of poses of (left) <b>P7</b> and (right) <b>F7.7</b>. Compounds are shown in green sticks. Asn324<sup>6.55</sup> and His350<sup>7.39</sup> are shown in orange sticks (bold and thin, respectively). Glu212<sup>45.52</sup> and Arg328<sup>6.59</sup> are shown in cyan sticks (bold and thin, respectively). Helix 7 was removed for clarity. Color code: C, green; N, blue; O, red; S, yellow; Cl, dark green; F, cyan. The protein is PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, as a negative example, we turn to parent compound <b>P9</b>. Although <b>P9</b> has affinity for orexin receptors, none of the 12 derivatives showed much binding. This series also shows the perils of similarity searches as several of the ligands would not be considered similar by a chemist. Yet, these seemed to interact favorably with the binding pocket in docking. <b>F9.8</b> is the only compound with a similar “double lactam” cyclohexene ring and the only to show some effect.</div><div class="NLM_p last">Overall, our docking screen showed that even for relatively featureless apolar binding pockets, ligands can be found in large databases based on shape complementarity (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> for the 10 most affine compounds found in this study). Interestingly, the scarcity of strong polar or even charge–charge interaction possibilities did not hamper docking’s ability to find ligands, evident from the overall hit rate of 29%, which compares favorably with other studies on class A GPCRs. Several novel chemotypes were identified and might serve as seeds for further development.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our unbiased docking screen (using the OX<sub>2</sub>R crystal structure) answers the three questions posed in the introduction. A metric that is often used as the figure of merit in a docking screen is the hit rate, defined as the percentage of all tested molecules that are found to interact with the receptor. In this study, taking the hits from the parent and daughter screen into account, a hit rate of 29% was achieved in the CHO-OX<sub>2</sub> cells, based on binding affinity. Despite the challenging nature of the binding pocket, with only few polar interactions sprinkled throughout, this puts the present screen in the same league as previous studies with polar or even charged features within the binding pockets. Docking itself thus does not seem to be hampered by this comparative scarcity of directional interactions in the OX<sub>2</sub>R. The hit rates of the daughter generation are almost the same as the parents’ rates, however. This might be because the similarity calculations were based on global similarities. A stricter similarity search, retaining the scaffolds of the parent compounds, might have produced higher hit rates.</div><div class="NLM_p">Second, we took a closer look at the predicted binding modes and compared them with the ones of suvorexant and SB-674042. The majority of molecules interact with very similar residues, predominantly Asn324<sup>6.55</sup> and His350<sup>7.39</sup>. Interestingly, however, a few of them also formed interactions with Thr111<sup>2.61</sup>, a residue that differs in OX<sub>1</sub>R, where it is a serine (Ser103<sup>2.61</sup>). Despite this being a small deviation, such compounds might be starting points for selective orexin receptor antagonists. On top of this, compound <b>P22</b> forms a hydrogen bond with the backbone carbonyl of Pro131<sup>3.29</sup>, which has so far not been observed in any of the crystal structures.</div><div class="NLM_p">Lastly, this docking strategy has managed to identify several scaffolds that can be considered novel by common chemoinformatic criteria. Among them is compound <b>P27</b>, whose ECFP4 Tanimoto similarity to its closest match among any known ligands of OX<sub>2</sub>R is only 0.237 (median similarity of 0.115). Two additional compounds, <b>F33.2</b> and <b>F7.4</b>, show also distant closest matches and low maximum similarities of 0.241 (median of 0.124) and 0.254 (median of 0.121), respectively. In total, 52 compounds that show some affinity in our assay had an ECFP4 Tanimoto similarity to their closest neighbors of 0.45 or less and 22 with a similarity value of 0.30 and less, commonly regarded as a threshold indicating dissimilarity (ECFP4 Tanimoto similarity values and ROCS TanimotoCombo scores can be found in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Tables S3 and S5</a>)).</div><div class="NLM_p">A key question for future investigations is whether these scaffolds can easily be derivatized. To answer this question, the composition was analyzed with the PINGUI<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> toolbox, which we have developed earlier. Each of the compounds shows a facile synthetic breaking point, yielding fragments of a size that is frequently occurring in building block databases. The number of applicable reactions to fragments of our hits after their retrosynthetical decomposition ranges from two (<b>F21.3</b>) to nine (<b>P18</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Table S6</a>). It is likely that also the derivatives will show affinity as we have already exhausted existing chemical space through SAR by catalog. In particular, compound <b>P33</b> seems very promising as five of its daughter compounds (<b>F33.1</b>–<b>3</b>, <b>F33.6</b>, and <b>F33.7</b>) also show binding to the receptor. Our analysis found 103 retrosynthetical disconnections. Four unique reactions can be applied to the resulting fragment set. It has to be noted that <b>P33</b> shares some moieties with the known ligands EMPA and SB-649868. Based on our SAR considerations and our experience from similar projects,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> we are convinced that there is room for diversity around the common rings, however.</div><div class="NLM_p">By way of better characterization, we also tested our compounds against the OX<b><sub>1</sub></b>R, despite the fact that we never made a prediction about selectivity. Not unexpectedly, most of the compounds behave as DORAs. In fact, our OX<sub>2</sub>R-focused docking produced five compounds with a measurable affinity selectivity between OX<sub>2</sub>R and OX<sub>1</sub>R, but only up to a maximum selectivity of 5-fold (a ΔpK<sub>i</sub> of 0.6). Although measurable, these values are low and at the detection limit of the assay. At the same time, OX<sub>2</sub>R-selective compounds are generally regarded as preferable for the treatment of insomnia as they carry lower risks of OX<sub>1</sub>R-mediated side effects, such as compulsive behavior and substance misuse disorders.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">Would this be different if we had docked to both subtypes? In order to answer this question, we did a dual docking, as described before.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Briefly, the same small molecule dataset was docked to both the OX<sub>1</sub>R and the OX<sub>2</sub>R. For the OX<sub>1</sub>R, the X-ray structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJ8">4ZJ8</a><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> was used and prepared in an identical fashion as its OX<sub>2</sub>R counterpart. After docking, molecules were reranked in order to favor molecules that would display the desired selectivity.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The molecules populating the top 500 ranks of this docking were all distinct from the ones of the original docking against the OX<sub>2</sub>R, and therefore, there was also no overlap between the sets of assayed molecules. However, even with this differential docking, we did not find ligands with more than a ΔpK<sub>i</sub> of 0.5. The data for these molecules is presented in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. It thus stands to reason that this featureless binding pocket presents a case where standard docking protocols run in high-throughput mode do not discriminate well enough in order to predict selectivity between the two orexin receptor subtypes. As we have shown recently, this is different for dedicated more precise docking calculations used to optimize ligand selectivity, driving it toward higher OX<sub>1</sub>R affinity.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. pK<sub>i</sub> Values Obtained for Compounds Originating from Differential Docking from Whole Cell Binding Studies as Determined from Using <sup>3</sup>H-SB674042 (CHO-OX<sub>1</sub> Cells) and <sup>3</sup>H-EMPA (CHO-OX<sub>2</sub> Cells)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">CHO-OX<sub>2</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">pK<sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.3</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.4</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.5</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.6</b></td><td class="colsep0 rowsep0" align="left">4.83 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.64 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S1.7</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.2</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.28 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.3</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.4</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.5</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.6</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.7</b></td><td class="colsep0 rowsep0" align="left">5.38 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">5.80 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.8</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2.9</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.1</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.2</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.3</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.4</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.40 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.5</b></td><td class="colsep0 rowsep0" align="left">4.47 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.56 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.6</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.7</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 100 μM</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.8</b></td><td class="colsep0 rowsep0" align="left">4.61 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">4.82 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S3.9</b></td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">no binding</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Values are mean ± sem from <i>n</i> separate experiments.</p></div></div></div><div class="NLM_p last">In summary, our <i>in silico</i> screen yielded several compounds with potential for further development as orexin receptor ligands and shed light on possible alternative interaction patterns that can be exploited in future screening and optimization work.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Receptor Preparation</h3><div class="NLM_p last">The crystal structure of the OX<sub>2</sub>R liganded by suvorexant (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>)<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> was prepared using the MOE software suite.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Briefly, (i) all water molecules, metal ions, precipitants, and lipids were removed. (ii) The fusion protein glycogen synthase from <i>Pyrococcus abyssi</i> was cleaved from the receptor at amino acids Lys294<sup>6.25</sup> and Gln254<sup>5.69</sup>. (iii) Loop breaks and fusion protein cleavage sites were capped with methylamine and acetyl at C- and N-termini, respectively. (iv) Hydrogen atoms were added, and the protonation states of titratable amino acids were assigned using MOE’s Protein Preparation subroutines (pH value, 7.4; temperature, 300 K). Protonation states of histidines and rotamers of head groups of glutamines and asparagines were visually inspected and adjusted according to their protein environment.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Database Generation</h3><div class="NLM_p last">The DrugsNow subset of the ZINC database<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was downloaded as SMILES strings (7.377 million entries). The subset was split into batches of 10,000 molecules (738 batches). Each batch was submitted to OpenEye’s OMEGA program for conformer generation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> For each molecular entity, a maximum number of 400 conformers was generated. The energy window parameter was set to 10.0 and the root mean square deviation of conformer coordinate parameter (rms) was set to 0.5. The “strict” flag was set to “true” ensuring that molecules without fully specified stereochemistry were discarded. A total of 1.726 billion conformers were generated. For docking with DOCK, the ZINC LeadLike subset was downloaded in a flexibase format (3.987 million entries) and used as is (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Docking</h3><div class="NLM_p">Docking was performed using OpenEye’s HYBRID program.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> The ensemble of conformations of each molecule was overlaid with the cocrystallized ligand (suvorexant) in order to determine the best suited conformer for the following exhaustive docking. The method for overlaying conformers is built directly into the HYBRID engine and is based on the same methodology as implemented in the OEChem API and the ROCS application.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For the actual docking step—translational and rotational optimization of a compound conformer within the binding site of the protein—HYBRID scores for a given protein–ligand complex were calculated based on the shape and electrostatic complementarity of the ligand and the protein’s binding site. Shape and electrostatic features are represented by Gaussian potentials. During optimization, the overlap between ligand and protein features is maximized. After docking, the 100 best-scored poses were extracted from each batch and aggregated into one sorted scoring list (73,800 entries, corresponds to best-scored percentile of the entire docking run).</div><div class="NLM_p">The ZINC LeadLike subset was docked with DOCK.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Briefly, molecules were placed using guiding points inside the pocket that had been derived from suvorexant bound to the OX<sub>2</sub>R in the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p last">All docking scores can be found in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Table S4</a>). Molecules were purchased from various vendors, as listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">Table S2</a>. Purity of all ligands is ≥95%, as determined by LC/MS and different methods employed by the vendors.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Postprocessing</h3><div class="NLM_p">The scoring list of ∼74,000 poses was treated in three different ways: (i) left unchanged (sorted by HYBRID score), (ii) rescored by DSX,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and (iii) submitted to a rigid body optimization procedure using OpenEye’s SZYBKI program.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> During this optimization process, the atoms of the receptor were kept fixed at their crystallographic positions, while the rotational, translational, and torsional degrees of freedom of each pose were optimized using a Poisson–Boltzmann solvation model. The resulting poses were sorted by the SZYBKI ligand–protein interaction energy.</div><div class="NLM_p last">From each of the scoring lists (HYBRID, DSX, and SZYBKI), a slice of 2000 poses was extracted and subjected to visual inspection in order to remove those that form improbable interactions that are not sufficiently penalized by present-day scoring functions. In addition, the 500 best-scored poses from the docking run with DOCK were added to the visual inspection. Selected compounds were acquired from their respective vendors and analyzed pharmacologically. Parent generation of compounds is denoted with a capital <b>P</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Similarity Screen</h3><div class="NLM_p last">Several experimentally validated compounds from the parent generation (both with and without affinity against the receptor) were used as queries for the retrieval of close analogues by a fingerprint-based similarity screen: For each of the nine hits (<b>P7</b>, <b>P9</b>, <b>P21</b>, <b>P22</b>, <b>P27</b>, <b>P31</b>, <b>P32</b>, <b>P33</b>, and <b>P35</b>), the 30 nearest neighbors were retrieved from the three 2000 entries-long slices (HYBRID, DSX, and SZYBKI (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B)). The retrieved poses were all subjected to optimization with SZYBKI. The resulting geometries were visually inspected and the selected compounds were acquired from their respective vendors and analyzed pharmacologically (daughter generation of compounds is denoted with a capital <b>F</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Experimental Validation</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Materials</h4><div class="NLM_p last"><sup>3</sup>H-SB674042 was from Metis Laboratories (New York, USA) and <sup>3</sup>H-EMPA was from Novandi Chemistry (Södertälje, Sweden). Fluo-4AM and pluronic F-127 were from Invitrogen (Oregon, USA). Orexin A, SB334867, SB674042, JNJ10397049, and EMPA were from Tocris Life Sciences (Avonmouth, UK). Suvorexant was from Selleckchem (Houston, USA). Gibco fetal bovine serum was from Fischer Scientific (Loughborough, UK). All other reagents were from Sigma-Aldrich (Poole, UK).</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Cell Culture</h4><div class="NLM_p last">CHO cells stably expressing either the human OX<sub>1</sub> or OX<sub>2</sub> receptor (originally a gift from Heptares, UK) were secondarily transfected with an SRE-luciferase reporter gene and stable clones selected by dilution cloning to create CHO-OX<sub>1</sub> and CHO-OX<sub>2</sub> stable cell lines. Cells were grown in Dulbecco’s modified Eagle’s medium nutrient mix F12 (DMEM/F12) containing 10% fetal calf serum and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine in a 37 °C humidified 5% CO<sub>2</sub>:95% air atmosphere.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Whole Cell <sup>3</sup>H-Radioligand Binding</h4><div class="NLM_p">The affinity of compounds for the OX<sub>1</sub> and OX<sub>2</sub> receptors were determined from whole cell binding studies. Experiments were conducted on confluent cells in white-sided 96-well view plates, as previously described (2 h 37 °C incubation with radioligand and competing ligand in 200 μL of serum-free media, Baker 2005). A seven-point concentration response curve (each point in triplicate) was examined for each ligand in each experiment. Suvorexant (10 μM) was used to define nonspecific binding. The affinity of the radioligands has previously been determined as 4.65 nM for <sup>3</sup>H-SB674042 in the CHO-OX<sub>1</sub> cells and 7.86 nM for <sup>3</sup>H-EMPA in the CHO-OX<sub>2</sub> cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Cells were inspected under a light microscope to ensure that they were still present after the 2 h incubation, both before and after the wash. In a few cases, high concentrations (100 μM) of competing ligand caused the cells to round up and be washed off the plates. These concentrations were excluded from the analysis.</div><div class="NLM_p">The IC<sub>50</sub> value of competing ligands was determined from a sigmoidal response curve (plotted in Prism 7) where the IC<sub>50</sub> is the concentration required to inhibit 50% of the specific binding of the <sup>3</sup>H-radioligand, [<i>A</i>] is the concentration of the competing ligand.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_m001.gif" alt="" id="_i23" /></img></span></div><div class="NLM_p last">The K<sub>i</sub> values for the competing ligands were then determined from the IC<sub>50</sub> values using Cheng–Prusoff equation where [<i>L</i>] is the concentration of radioligand in that experiment and <i>L</i><sub>KD</sub> is the K<sub>D</sub> value of the radioligand (as determined from saturation binding<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>):<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_m002.gif" alt="" id="_i24" /></img></span></div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Intracellular Calcium Release</h4><div class="NLM_p">Calcium measurements were made using a Flexstation 3 at 37 °C. Cells, seeded into black 96-well view plates, were loaded with Fluo-4AM/pluronic-F127 in serum-free media containing probenecid (25 mM) for 45 min at 37 °C. Cells were washed with 200 μL of HEPES buffered saline (HBS, containing 2 μM CaCl<sub>2</sub>) per well before a further 80 μL of HBS was then added to each well. Ligands were diluted in HBS to 5 times the final concentration in round-bottom 96-well compound plates. During the experiment, the Flexstation robotics added 20 μL of ligand from the compound plate into the existing 80 μL of HBS in the cell plate (1:5 dilution in well). Ionomycin (10 μM) and orexin A (1 μM) were used as positive controls on each plate in each experiment. Calcium mobilization was followed for 120 s and data were plotted as the maximum value obtained for calcium mobilization over the basal value obtained for that well before the addition of ligand.</div><div class="NLM_p last">A sigmoidal concentration response curve was fitted to the data (using Prism 7) where <i>E</i><sub>max</sub> is the maximum response, [<i>A</i>] is the agonist concentration, and EC<sub>50</sub> is the concentration of agonist that produces 50% of the maximal response:<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_m003.gif" alt="" id="_i26" /></img></span></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00964" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58737" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58737" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00964?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00964</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tabular data containing compound structures, SMILES strings, and supplier information; Tanimoto fingerprint similarities, ROCS TanimotoCombo scores, docking scores, and retrosynthetical analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Dose–response curves (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Orexin receptor data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Predicted complex structures for all molecules (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_004.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf">jm0c00964_si_001.pdf (1.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_002.pdf">jm0c00964_si_002.pdf (382.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_003.csv">jm0c00964_si_003.csv (11.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_004.zip">jm0c00964_si_004.zip (13.59 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00964" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Kolb</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4089-614X" title="Orcid link">http://orcid.org/0000-0003-4089-614X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ccbca9b8a9bee2a7a3a0ae8cb9a2a5e1a1adbeaeb9beabe2a8a9"><span class="__cf_email__" data-cfemail="3d4d5849584f135652515f7d48535410505c4f5f484f5a135958">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jakub Gunera</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jillian G. Baker</span> - <span class="hlFld-Affiliation affiliation">Cell
Signalling, School of Life Sciences, Queen’s Medical Centre, University of Nottingham, Nottingham NG7 2UH, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2371-8202" title="Orcid link">http://orcid.org/0000-0003-2371-8202</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niek van Hilten</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Philipps-University, Marburg, Hesse 35032, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel M. Rosenbaum</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biophysics and Biochemistry, UT Southwestern
Medical Center, Dallas, Texas 75390-8816, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.G., D.M.R., and P.K. designed the research. J.G. and N.H. docking calculations and selected compounds together with P.K. J.G.B. performed pharmacological assays and analysis. J.G., J.G.B., and P.K. wrote the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>J.G.’s Ph.D. thesis was funded through the LOEWE Centre Synmikro. J.G.B. was funded by an MRC grant MR/M0032X-1. D.M.R. thanks the Welch Foundation (I-1770). P.K. thanks the German research foundation DFG for Heisenberg professorship KO4095/4-1.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Richard Proudman for technical assistance with the pharmacological assays and Matthäus Drabek for his input during compound selection.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bentivoglio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassi-Zucconi, G.</span></span> <span> </span><span class="NLM_article-title">The pioneering experimental studies on sleep deprivation</span>. <i>Sleep</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1093/sleep/20.7.570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1093%2Fsleep%2F20.7.570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=9322273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A280%3ADyaK2svmtlCqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1997&pages=570-576&author=M.+Bentivoglioauthor=G.+Grassi-Zucconi&title=The+pioneering+experimental+studies+on+sleep+deprivation&doi=10.1093%2Fsleep%2F20.7.570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The pioneering experimental studies on sleep deprivation</span></div><div class="casAuthors">Bentivoglio M; Grassi-Zucconi G</div><div class="citationInfo"><span class="NLM_cas:title">Sleep</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">570-6</span>
        ISSN:<span class="NLM_cas:issn">0161-8105</span>.
    </div><div class="casAbstract">The experimental studies on sleep deprivation were initiated by the Russian physician and scientist, Marie de Manaceine, who studied sleep-deprived puppies kept in constant activity.  She reported in 1894 that the complete absence of sleep was fatal in a few days, pointing out that the most severe lesions occurred in the brain.  In 1898, the Italian physiologists Lamberto Daddi and Giulio Tarozzi also kept dogs awake by walking them; the animals died after 9-17 days, and their survival was unrelated to food consumption.  In the histological study performed by Daddi, degenerative alterations, mainly represented by chromatolytic changes, were observed in neurons of the spinal ganglia, Purkinje cells of the cerebellum, and neurons of the frontal cortex.  Daddi ascribed these changes to a state of autointoxication of the brain during insomnia.  In 1898, the psychiatrist Cesare Agostini, interested in the psychic phenomena caused by prolonged insomnia in humans, sleep deprived dogs by keeping them in a metallic cage in order to avoid fatigue.  The dogs survived about 2 weeks, and degenerative changes were observed in their brains.  In these experimental paradigms, the effect of sleep loss was confounded by motor exhaustion and/or intense sensory stimulation.  In spite of the absence of adequate controls, the pioneering studies performed at the end of the 19th century represented the first experimental attempts to relate sleep with neural centers and suggested that sleep is a vital function and that the brain may be affected by insomnia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT1M6J7tIwvuVw2ofyr07jXfW6udTcc2eaIoZft5kQ_rrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svmtlCqsQ%253D%253D&md5=88a763de83f5c54ab666be9e9d156866</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fsleep%2F20.7.570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fsleep%252F20.7.570%26sid%3Dliteratum%253Aachs%26aulast%3DBentivoglio%26aufirst%3DM.%26aulast%3DGrassi-Zucconi%26aufirst%3DG.%26atitle%3DThe%2520pioneering%2520experimental%2520studies%2520on%2520sleep%2520deprivation%26jtitle%3DSleep%26date%3D1997%26volume%3D20%26spage%3D570%26epage%3D576%26doi%3D10.1093%2Fsleep%2F20.7.570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czeisler, C. A.</span></span> <span> </span><span class="NLM_article-title">Impact of sleepiness and sleep deficiency on public health--utility of biomarkers</span>. <i>J. Clin. Sleep Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">S6</span>– <span class="NLM_lpage">S8</span>, <span class="refDoi"> DOI: 10.5664/JCSM.1340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.5664%2FJCSM.1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=22003334" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=S6-S8&author=C.+A.+Czeisler&title=Impact+of+sleepiness+and+sleep+deficiency+on+public+health%2D%2Dutility+of+biomarkers&doi=10.5664%2FJCSM.1340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.5664%2FJCSM.1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5664%252FJCSM.1340%26sid%3Dliteratum%253Aachs%26aulast%3DCzeisler%26aufirst%3DC.%2BA.%26atitle%3DImpact%2520of%2520sleepiness%2520and%2520sleep%2520deficiency%2520on%2520public%2520health--utility%2520of%2520biomarkers%26jtitle%3DJ.%2520Clin.%2520Sleep%2520Med.%26date%3D2011%26volume%3D7%26spage%3DS6%26epage%3DS8%26doi%3D10.5664%2FJCSM.1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisden, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundlach, A. L.</span></span> <span> </span><span class="NLM_article-title">Dual-transmitter systems regulating arousal, attention, learning and memory</span>. <i>Neurosci. Biobehav. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.neubiorev.2017.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.neubiorev.2017.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=28757457" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2017&pages=21-33&author=S.+Maauthor=B.+Hangyaauthor=C.+S.+Leonardauthor=W.+Wisdenauthor=A.+L.+Gundlach&title=Dual-transmitter+systems+regulating+arousal%2C+attention%2C+learning+and+memory&doi=10.1016%2Fj.neubiorev.2017.07.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.neubiorev.2017.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neubiorev.2017.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DS.%26aulast%3DHangya%26aufirst%3DB.%26aulast%3DLeonard%26aufirst%3DC.%2BS.%26aulast%3DWisden%26aufirst%3DW.%26aulast%3DGundlach%26aufirst%3DA.%2BL.%26atitle%3DDual-transmitter%2520systems%2520regulating%2520arousal%252C%2520attention%252C%2520learning%2520and%2520memory%26jtitle%3DNeurosci.%2520Biobehav.%2520Rev.%26date%3D2017%26volume%3D85%26spage%3D21%26epage%3D33%26doi%3D10.1016%2Fj.neubiorev.2017.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amemiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuzaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemelli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arch, J. R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckingham, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Car, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcnulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elshourbagy, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsma, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanagisawa, M.</span></span> <span> </span><span class="NLM_article-title">Orexins and orexin receptors: a family of hypothalamic neuropeptides and G protein-coupled receptors that regulate feeding behavior</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80949-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2FS0092-8674%2800%2980949-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=9491897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADyaK1cXhsVKntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1998&pages=573-585&author=T.+Sakuraiauthor=A.+Amemiyaauthor=M.+Ishiiauthor=I.+Matsuzakiauthor=R.+M.+Chemelliauthor=H.+Tanakaauthor=S.+C.+Williamsauthor=J.+A.+Richardsonauthor=G.+P.+Kozlowskiauthor=S.+Wilsonauthor=J.+R.+S.+Archauthor=R.+E.+Buckinghamauthor=A.+C.+Haynesauthor=S.+A.+Carauthor=R.+S.+Annanauthor=D.+E.+Mcnultyauthor=W.-S.+Liuauthor=J.+A.+Terrettauthor=N.+A.+Elshourbagyauthor=D.+J.+Bergsmaauthor=M.+Yanagisawa&title=Orexins+and+orexin+receptors%3A+a+family+of+hypothalamic+neuropeptides+and+G+protein-coupled+receptors+that+regulate+feeding+behavior&doi=10.1016%2FS0092-8674%2800%2980949-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Orexins and orexin receptors: a family of hypothalamic neuropeptides and G protein-coupled receptors that regulate feeding behavior</span></div><div class="casAuthors">Sakurai, Takeshi; Amemiya, Akira; Ishii, Makoto; Matsuzaki, Ichiyo; Chemelli, Richard M.; Tanaka, Hirokazu; Williams, S. Clay; Richardson, James A.; Kozlowski, Gerald P.; Wilson, Shelagh; Arch, Jonathan R. S.; Buckingham, Robin E.; Haynes, Andrea C.; Carr, Steven A.; Annan, Roland S.; Mcnulty, Dean E.; Liu, Wu-Schyong; Terrett, Jonathan A.; Elshourbagy, Nabil A.; Bergsma, Derk J.; Yanagisawa, Masashi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-585</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The hypothalamus plays a central role in the integrated control of feeding and energy homeostasis.  The authors have identified two novel neuropeptides, both derived from the same precursor by proteolytic processing, that bind and activate two closely related (previously) orphan G protein-coupled receptors.  These peptides, termed orexin-A and -B, have no significant structural similarities to known families of regulatory peptides.  Prepro-orexin mRNA and immunoreactive orexin-A are localized in neurons within and around the lateral and posterior hypothalamus in the adult rat brain.  When administered centrally to rats, these peptides stimulate food consumption.  Prepro-orexin mRNA level is up-regulated upon fasting, suggesting a physiol. role for the peptides as mediators in the central feedback mechanism that regulates feeding behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA5y2uicTVlLVg90H21EOLACvtfcHk0lhkeOFUWvQfDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhsVKntr4%253D&md5=c0d02e69cd976521abfc5f88e8be7dda</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980949-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980949-6%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DT.%26aulast%3DAmemiya%26aufirst%3DA.%26aulast%3DIshii%26aufirst%3DM.%26aulast%3DMatsuzaki%26aufirst%3DI.%26aulast%3DChemelli%26aufirst%3DR.%2BM.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DWilliams%26aufirst%3DS.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BA.%26aulast%3DKozlowski%26aufirst%3DG.%2BP.%26aulast%3DWilson%26aufirst%3DS.%26aulast%3DArch%26aufirst%3DJ.%2BR.%2BS.%26aulast%3DBuckingham%26aufirst%3DR.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCar%26aufirst%3DS.%2BA.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DMcnulty%26aufirst%3DD.%2BE.%26aulast%3DLiu%26aufirst%3DW.-S.%26aulast%3DTerrett%26aufirst%3DJ.%2BA.%26aulast%3DElshourbagy%26aufirst%3DN.%2BA.%26aulast%3DBergsma%26aufirst%3DD.%2BJ.%26aulast%3DYanagisawa%26aufirst%3DM.%26atitle%3DOrexins%2520and%2520orexin%2520receptors%253A%2520a%2520family%2520of%2520hypothalamic%2520neuropeptides%2520and%2520G%2520protein-coupled%2520receptors%2520that%2520regulate%2520feeding%2520behavior%26jtitle%3DCell%26date%3D1998%26volume%3D92%26spage%3D573%26epage%3D585%26doi%3D10.1016%2FS0092-8674%2800%2980949-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobarec, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1038/nature14035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnature14035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=25533960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=247-250&author=J.+Yinauthor=J.+C.+Mobarecauthor=P.+Kolbauthor=D.+M.+Rosenbaum&title=Crystal+structure+of+the+human+OX2+orexin+receptor+bound+to+the+insomnia+drug+suvorexant&doi=10.1038%2Fnature14035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant</span></div><div class="casAuthors">Yin, Jie; Mobarec, Juan Carlos; Kolb, Peter; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">247-250</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) respond to orexin neuropeptides in the central nervous system to regulate sleep and other behavioral functions in humans.  Defects in orexin signalling are responsible for the human diseases of narcolepsy and cataplexy; inhibition of orexin receptors is an effective therapy for insomnia.  The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compds. including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant.  Here, using lipid-mediated crystn. and protein engineering with a novel fusion chimaera, we solved the structure of the human OX2R bound to suvorexant at 2.5 Å resoln.  The structure reveals how suvorexant adopts a π-stacked horseshoe-like conformation and binds to the receptor deep in the orthosteric pocket, stabilizing a network of extracellular salt bridges and blocking transmembrane (TM) helix motions necessary for activation.  Computational docking suggests how other classes of synthetic antagonists may interact with the receptor at a similar position in an analogous π-stacked fashion.  Elucidation of the mol. architecture of the human OX2R expands our understanding of peptidergic GPCR ligand recognition and will aid further efforts to modulate orexin signaling for therapeutic ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWKzbWNxv4W7Vg90H21EOLACvtfcHk0lhkeOFUWvQfDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntb0%253D&md5=461a6633274dbcfd9edce1a2e13b9c4f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature14035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14035%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DMobarec%26aufirst%3DJ.%2BC.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520OX2%2520orexin%2520receptor%2520bound%2520to%2520the%2520insomnia%2520drug%2520suvorexant%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D247%26epage%3D250%26doi%3D10.1038%2Fnature14035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogusky, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitman, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of N,N-disubstituted-1,4-diazepane orexin receptor antagonists and implications for receptor binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2997</span>– <span class="NLM_lpage">3001</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2009.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19406641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVeku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2997-3001&author=C.+D.+Coxauthor=G.+B.+McGaugheyauthor=M.+J.+Boguskyauthor=D.+B.+Whitmanauthor=R.+G.+Ballauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Coleman&title=Conformational+analysis+of+N%2CN-disubstituted-1%2C4-diazepane+orexin+receptor+antagonists+and+implications+for+receptor+binding&doi=10.1016%2Fj.bmcl.2009.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of N,N-disubstituted-1,4-diazepane orexin receptor antagonists and implications for receptor binding</span></div><div class="casAuthors">Cox, Christopher D.; McGaughey, Georgia B.; Bogusky, Michael J.; Whitman, David B.; Ball, Richard G.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2997-3001</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">NMR spectroscopy, X-ray crystallog., and mol. modeling studies indicate that N,N-disubstituted-1,4-diazepane orexin receptor antagonists exist in an unexpected low-energy conformation that is characterized by an intramol. π-stacking interaction and a twist-boat ring conformation.  Synthesis and evaluation of a macrocycle I that enforces a similar conformation suggest that this geometry mimics the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMJNXLU745TbVg90H21EOLACvtfcHk0lhkeOFUWvQfDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVeku7c%253D&md5=a3cec5d7ca8a09d9a14cd440d1cfd3b4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DC.%2BD.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DBogusky%26aufirst%3DM.%2BJ.%26aulast%3DWhitman%26aufirst%3DD.%2BB.%26aulast%3DBall%26aufirst%3DR.%2BG.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26atitle%3DConformational%2520analysis%2520of%2520N%252CN-disubstituted-1%252C4-diazepane%2520orexin%2520receptor%2520antagonists%2520and%2520implications%2520for%2520receptor%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2997%26epage%3D3001%26doi%3D10.1016%2Fj.bmcl.2009.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of β2-adrenergic receptor ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6848</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+%CE%B22-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0li-IUlrtlwI9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520%25CE%25B22-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0li-IUlrtlwI9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strachan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefkowitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Conformation Guides Molecular Efficacy in Docking Screens of Activated β-2 Adrenergic G Protein Coupled Receptor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1021/cb400103f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400103f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvVahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1018-1026&author=D.+R.+Weissauthor=S.+Ahnauthor=M.+F.+Sassanoauthor=A.+Kleistauthor=X.+Zhuauthor=R.+Strachanauthor=B.+L.+Rothauthor=R.+J.+Lefkowitzauthor=B.+K.+Shoichet&title=Conformation+Guides+Molecular+Efficacy+in+Docking+Screens+of+Activated+%CE%B2-2+Adrenergic+G+Protein+Coupled+Receptor&doi=10.1021%2Fcb400103f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Conformation Guides Molecular Efficacy in Docking Screens of Activated β-2 Adrenergic G Protein Coupled Receptor</span></div><div class="casAuthors">Weiss, Dahlia R.; Ahn, SeungKirl; Sassano, Maria F.; Kleist, Andrew; Zhu, Xiao; Strachan, Ryan; Roth, Bryan L.; Lefkowitz, Robert J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1018-1026</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A prospective, large library virtual screen against an activated β2-adrenergic receptor (β2AR) structure returned potent agonists to the exclusion of inverse-agonists, providing the first complement to the previous virtual screening campaigns against inverse-agonist-bound G protein coupled receptor (GPCR) structures, which predicted only inverse-agonists.  In addn., two hits recapitulated the signaling profile of the co-crystal ligand with respect to the G protein and arrestin mediated signaling.  This functional fidelity has important implications in drug design, as the ability to predict ligands with predefined signaling properties is highly desirable.  However, the agonist-bound state provides an uncertain template for modeling the activated conformation of other GPCRs, as a dopamine D2 receptor (DRD2) activated model templated on the activated β2AR structure returned few hits of only marginal potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5QKZOWGheuLVg90H21EOLACvtfcHk0lhtXMNJ3PLMMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvVahtbg%253D&md5=6394521533373197c59e49b242bf0c93</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcb400103f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400103f%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DKleist%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DStrachan%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DConformation%2520Guides%2520Molecular%2520Efficacy%2520in%2520Docking%2520Screens%2520of%2520Activated%2520%25CE%25B2-2%2520Adrenergic%2520G%2520Protein%2520Coupled%2520Receptor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1018%26epage%3D1026%26doi%3D10.1021%2Fcb400103f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topiol, S.</span></span> <span> </span><span class="NLM_article-title">Use of the X-ray structure of the beta2-adrenergic receptor for drug discovery. Part 2: Identification of active compounds</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5395</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.09.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2008.09.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=18829308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1CqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5391-5395&author=M.+Sabioauthor=K.+Jonesauthor=S.+Topiol&title=Use+of+the+X-ray+structure+of+the+beta2-adrenergic+receptor+for+drug+discovery.+Part+2%3A+Identification+of+active+compounds&doi=10.1016%2Fj.bmcl.2008.09.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the X-ray structure of the β2-adrenergic receptor for drug discovery. Part 2: Identification of active compounds</span></div><div class="casAuthors">Sabio, Michael; Jones, Kenneth; Topiol, Sid</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5391-5395</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The recently published X-ray structures of the β2-adrenergic receptor are the first examples of ligand-mediated GPCR crystal structures.  We have previously performed computational studies that examine the potential viability of these structures for use in drug design, exploiting known ligand activities.  Our previous study and a newly reported β2/Timolol X-ray complex provide validation of the computational approaches.  In the present work, we use the X-ray structures to ext., via in silico high-throughput docking, compds. from proprietary and com. databases and demonstrate the successful identification of active compds. by radioligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1AGaAYjN5WbVg90H21EOLACvtfcHk0lhtXMNJ3PLMMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1CqsrfP&md5=e91ea267e0851bc0e08b310b3f8966ee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.046%26sid%3Dliteratum%253Aachs%26aulast%3DSabio%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DTopiol%26aufirst%3DS.%26atitle%3DUse%2520of%2520the%2520X-ray%2520structure%2520of%2520the%2520beta2-adrenergic%2520receptor%2520for%2520drug%2520discovery.%2520Part%25202%253A%2520Identification%2520of%2520active%2520compounds%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5391%26epage%3D5395%26doi%3D10.1016%2Fj.bmcl.2008.09.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brox, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity using Multi-Target Docking</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1021/cb500577j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500577j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=715-724&author=D.+Schmidtauthor=V.+Bernatauthor=R.+Broxauthor=N.+Tschammerauthor=P.+Kolb&title=Identifying+Modulators+of+CXC+Receptors+3+and+4+with+Tailored+Selectivity+using+Multi-Target+Docking&doi=10.1021%2Fcb500577j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity Using Multi-Target Docking</span></div><div class="casAuthors">Schmidt, Denis; Bernat, Viachaslau; Brox, Regine; Tschammer, Nuska; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">715-724</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G protein-coupled receptors of the C-X-C subfamily form a group among the chemokine receptors whose endogenous ligands are peptides with a common Cys-X-Cys motif.  The CXC chemokine receptors 3 and 4 (CXCR3, CXCR4), which are investigated in this study, are linked to severe diseases such as cancer, multiple sclerosis, and HIV infections.  Of particular interest, this receptor pair potentially forms a target for a polypharmacol. drug treatment.  Considering known ligands from public databases, such dual binders have not been identified yet.  We therefore applied large-scale docking to the structure of CXCR4 and a homol. model of CXCR3 with the goal to predict such dual binders, as well as compds. selective for either one of the receptors.  Using signaling and biochem. assays, we showed that more than 50% of these predictions were correct in each category, yielding ligands with excellent binding efficiencies.  These results highlight that docking is a suitable tool for the identification of ligands with tailored binding profiles to GPCRs, even when using homol. models.  More importantly, we present novel CXCR3-CXCR4 dual modulators that might pave the road to understanding the mechanisms of polypharmacol. inhibition of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Px6d1pbV3rVg90H21EOLACvtfcHk0lhtXMNJ3PLMMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM&md5=9259d56e4c28e68360f3ac214ea62ea4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb500577j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500577j%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DBrox%26aufirst%3DR.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DIdentifying%2520Modulators%2520of%2520CXC%2520Receptors%25203%2520and%25204%2520with%2520Tailored%2520Selectivity%2520using%2520Multi-Target%2520Docking%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D715%26epage%3D724%26doi%3D10.1021%2Fcb500577j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerman, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdon, H. J.</span></span> <span> </span><span class="NLM_article-title">Characterisation of the binding of [3H]-SB-674042, a novel nonpeptide antagonist, to the human orexin-1 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fsj.bjp.0705610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=14691055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2004&pages=340-346&author=C.+J.+Langmeadauthor=J.+C.+Jermanauthor=S.+J.+Broughauthor=C.+Scottauthor=R.+A.+Porterauthor=H.+J.+Herdon&title=Characterisation+of+the+binding+of+%5B3H%5D-SB-674042%2C+a+novel+nonpeptide+antagonist%2C+to+the+human+orexin-1+receptor&doi=10.1038%2Fsj.bjp.0705610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of the binding of [3H]-SB-674042, a novel nonpeptide antagonist, to the human orexin-1 receptor</span></div><div class="casAuthors">Langmead, Christopher J.; Jerman, Jeffrey C.; Brough, Stephen J.; Scott, Claire; Porter, Rod A.; Herdon, Hugh J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">340-346</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This study characterizes the binding of a novel nonpeptide antagonist radioligand, [3H]SB-674042 (1-(5-(2-fluoro-phenyl)-2-methyl-thiazol-4-yl)-1-((S)-2-(5-phenyl-(1,3,4)oxadiazol-2-ylmethyl)-pyrrolidin-l-yl)-methanone), to the human orexin-1 (OX1) receptor stably expressed in Chinese hamster ovary (CHO) cells in both a whole cell assay and in a cell membrane-based scintillation proximity assay (SPA) format.  Specific binding of [3H]SB-674042 was saturable in both whole cell and membrane formats.  Analyses suggested a single high-affinity site, with Kd values of 3.76 ± 0.45 and 5.03 ± 0.31 nM, and corresponding Bmax values of 30.8±1.8 and 34.4±2.0 pmol mg protein-1, in whole cell and membrane formats, resp.  Kinetic studies yielded similar Kd values.  Competition studies in whole cells revealed that the native orexin peptides display a low affinity for the OX1 receptor, with orexin-A displaying a ∼five-fold higher affinity than orexin-B (Ki values of 318±158 and 1516±597 nM, resp.).  SB-334867, SB-408124 (1-(6,8-difluoro-2-methyl-quinolin-4-yl)-3-(4-dimethylamino-phenyl)-urea) and SB-410220 (1-(5,8-difluoro-quinolin-4-yl)-3-(4-dimethylamino-phenyl)-urea) all displayed high affinity for the OX1 receptor in both whole cell (Ki values 99±18, 57±8.3 and 19±4.5 nM, resp.) and membrane (Ki values 38±3.6, 27±4.1 and 4.5±0.2 nM, resp.) formats.  Calcium mobilization studies showed that SB-334867, SB-408124 and SB-410220 are all functional antagonists of the OX1 receptor, with potencies in line with their affinities, as measured in the radioligand binding assays, and with approx. 50-fold selectivity over the orexin-2 receptor.  These studies indicate that [3H]SB-674042 is a specific, high-affinity radioligand for the OX1 receptor.  The availability of this radioligand will be a valuable tool with which to investigate the physiol. functions of OX1 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGKhe4L8-w0bVg90H21EOLACvtfcHk0lhXal8C3sVlHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOisLk%253D&md5=9fe91c966ce2fd2b631efe0b29f75e15</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705610%26sid%3Dliteratum%253Aachs%26aulast%3DLangmead%26aufirst%3DC.%2BJ.%26aulast%3DJerman%26aufirst%3DJ.%2BC.%26aulast%3DBrough%26aufirst%3DS.%2BJ.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DHerdon%26aufirst%3DH.%2BJ.%26atitle%3DCharacterisation%2520of%2520the%2520binding%2520of%2520%255B3H%255D-SB-674042%252C%2520a%2520novel%2520nonpeptide%2520antagonist%252C%2520to%2520the%2520human%2520orexin-1%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2004%26volume%3D141%26spage%3D340%26epage%3D346%26doi%3D10.1038%2Fsj.bjp.0705610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malherbe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knust, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettekoven, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers-Evans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettstein, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, J. L.</span></span> <span> </span><span class="NLM_article-title">Biochemical and behavioural characterization of EMPA, a novel high-affinity, selective antagonist for the OX(2) receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1341</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1111%2Fj.1476-5381.2009.00127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19751316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2009&pages=1326-1341&author=P.+Malherbeauthor=E.+Borroniauthor=L.+Gobbiauthor=H.+Knustauthor=M.+Nettekovenauthor=E.+Pinardauthor=O.+Rocheauthor=M.+Rogers-Evansauthor=J.+G.+Wettsteinauthor=J.+L.+Moreau&title=Biochemical+and+behavioural+characterization+of+EMPA%2C+a+novel+high-affinity%2C+selective+antagonist+for+the+OX%282%29+receptor&doi=10.1111%2Fj.1476-5381.2009.00127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and behavioural characterization of EMPA, a novel high-affinity, selective antagonist for the OX2 receptor</span></div><div class="casAuthors">Malherbe, P.; Borroni, E.; Gobbi, L.; Knust, H.; Nettekoven, M.; Pinard, E.; Roche, O.; Rogers-Evans, M.; Wettstein, J. G.; Moreau, J.-L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1326-1341</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The OX2 receptor is a G-protein-coupled receptor that is abundantly found in the tuberomammillary nucleus, an important site for the regulation of the sleep-wake state.  Herein, we describe the in vitro and in vivo properties of a selective OX2 receptor antagonist, N-ethyl-2-[(6-methoxy-pyridin-3-yl)-(toluene-2-sulfonyl)-amino]-N-pyridin-3-ylmethyl-acetamide (EMPA).  The affinity of [3H]EMPA was assessed in membranes from HEK293-hOX2-cells using satn. and binding kinetics.  The antagonist properties of EMPA were detd. by Schild anal. using the orexin-A- or orexin-B-induced accumulation of [3H]inositol phosphates (IP).  Quant. autoradiog. was used to det. the distribution and abundance of OX2 receptors in rat brain.  The in vivo activity of EMPA was assessed by reversal of [Ala11,D-Leu15]orexin-B-induced hyperlocomotion during the resting phase in mice and the redn. of spontaneous locomotor activity (LMA) during the active phase in rats.  [3H]EMPA bound to human and rat OX2-HEK293 membranes with KD values of 1.1 and 1.4 nmol/L-1 resp.  EMPA competitively antagonized orexin-A- and orexin-B-evoked accumulation of [3H]IP at hOX2 receptors with pA2 values of 8.6 and 8.8 resp.  Autoradiog. of rat brain confirmed the selectivity of [3H]EMPA for OX2 receptors.  EMPA significantly reversed [Ala11,D-Leu15]orexin-B-induced hyperlocomotion dose-dependently during the resting phase in mice.  EMPA, injected i.p. in rats during the active phase, reduced LMA dose-dependently.  EMPA did not impair performance of rats in the rotarod procedure.  EMPA is a high-affinity, reversible and selective OX2 receptor antagonist, active in vivo, which should prove useful for anal. of OX2 receptor function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2LdWKzYFpjbVg90H21EOLACvtfcHk0lhXal8C3sVlHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVWmurc%253D&md5=88d91d8772c3f282498ed1a2e9cd5f6b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00127.x%26sid%3Dliteratum%253Aachs%26aulast%3DMalherbe%26aufirst%3DP.%26aulast%3DBorroni%26aufirst%3DE.%26aulast%3DGobbi%26aufirst%3DL.%26aulast%3DKnust%26aufirst%3DH.%26aulast%3DNettekoven%26aufirst%3DM.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DRoche%26aufirst%3DO.%26aulast%3DRogers-Evans%26aufirst%3DM.%26aulast%3DWettstein%26aufirst%3DJ.%2BG.%26aulast%3DMoreau%26aufirst%3DJ.%2BL.%26atitle%3DBiochemical%2520and%2520behavioural%2520characterization%2520of%2520EMPA%252C%2520a%2520novel%2520high-affinity%252C%2520selective%2520antagonist%2520for%2520the%2520OX%25282%2529%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D156%26spage%3D1326%26epage%3D1341%26doi%3D10.1111%2Fj.1476-5381.2009.00127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proudman, R. G.</span>; <span class="NLM_string-name">Baker, J. G.</span></span> <i>The affinity and selectivity of orexin A, orexin B and orexin antagonists for the human orexin 1 and orexin 2 receptors</i>, <span class="NLM_year">2018</span>. . <a href="http://www.pa2online.org/abstract/abstract.jsp?abid=33656" class="extLink">http://www.pa2online.org/abstract/abstract.jsp?abid=33656</a> (accessed August 30, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+G.+Proudman&author=J.+G.+Baker&title=The+affinity+and+selectivity+of+orexin+A%2C+orexin+B+and+orexin+antagonists+for+the+human+orexin+1+and+orexin+2+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DProudman%26aufirst%3DR.%2BG.%26jtitle%3DThe%2520affinity%2520and%2520selectivity%2520of%2520orexin%2520A%252C%2520orexin%2520B%2520and%2520orexin%2520antagonists%2520for%2520the%2520human%2520orexin%25201%2520and%2520orexin%25202%2520receptors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Hilten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Binding-Site Compatible Fragment Growing Applied to the Design of β<sub>2</sub>-Adrenergic Receptor Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1129</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVKnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1118-1129&author=F.+Chevillardauthor=H.+Rimmerauthor=C.+Bettiauthor=E.+Pardonauthor=S.+Balletauthor=N.+van+Hiltenauthor=J.+Steyaertauthor=W.+E.+Diederichauthor=P.+Kolb&title=Binding-Site+Compatible+Fragment+Growing+Applied+to+the+Design+of+%CE%B22-Adrenergic+Receptor+Ligands&doi=10.1021%2Facs.jmedchem.7b01558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands</span></div><div class="casAuthors">Chevillard, Florent; Rimmer, Helena; Betti, Cecilia; Pardon, Els; Ballet, Steven; van Hilten, Niek; Steyaert, Jan; Diederich, Wibke E.; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1118-1129</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design.  Although computers allow for the facile generation of millions of suggestions, synthetic feasibility is however often neglected.  In this study the authors computationally extended, chem. synthesized, and exptl. assayed new ligands for the β2-adrenergic receptor (β2AR) by growing fragment-sized ligands.  To address the synthetic tractability issue, the authors' in silico work-flow aims at derivatized products based on robust org. reactions.  The study started from the predicted binding modes of five fragments.  The authors suggested a total of eight diverse extensions that were easily synthesized, and further assays showed that four products had an improved affinity (up to 40-fold) compared to their resp. initial fragment.  The described work-flow, which the authors call "growing via merging" and for which the key tools are available online, can improve early fragment-based drug discovery projects, making it a useful creative tool for medicinal chemists during structure-activity relationship (SAR) studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGNVNIASFV5LVg90H21EOLACvtfcHk0lhXal8C3sVlHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVKnu7s%253D&md5=f915fcba6096ec506f49274a2b628b97</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01558%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DRimmer%26aufirst%3DH.%26aulast%3DBetti%26aufirst%3DC.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DBallet%26aufirst%3DS.%26aulast%3Dvan%2BHilten%26aufirst%3DN.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DBinding-Site%2520Compatible%2520Fragment%2520Growing%2520Applied%2520to%2520the%2520Design%2520of%2520%25CE%25B22-Adrenergic%2520Receptor%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1118%26epage%3D1129%26doi%3D10.1021%2Facs.jmedchem.7b01558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stotani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Interrogating dense ligand chemical space with a forward-synthetic library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">11496</span>– <span class="NLM_lpage">11501</span>, <span class="refDoi"> DOI: 10.1073/pnas.1818718116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.1818718116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=31113876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2itL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=11496-11501&author=F.+Chevillardauthor=S.+Stotaniauthor=A.+Karawajczykauthor=S.+Hristevaauthor=E.+Pardonauthor=j.+Steyaertauthor=D.+Tzalisauthor=P.+Kolb&title=Interrogating+dense+ligand+chemical+space+with+a+forward-synthetic+library&doi=10.1073%2Fpnas.1818718116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interrogating dense ligand chemical space with a forward-synthetic library</span></div><div class="casAuthors">Chevillard, Florent; Stotani, Silvia; Karawajczyk, Anna; Hristeva, Stanimira; Pardon, Els; Steyaert, Jan; Tzalis, Dimitrios; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11496-11501</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Forward-synthetic databases are an efficient way to enumerate chem. space.  The authors explored here whether these databases are good sources of novel protein ligands and how many mols. are obtainable and in which time frame.  Based on docking calcns., series of mols. were selected to gain insights into the ligand structur-activity relation.  To evaluate the novelty of compds. in a challenging way, the authors chose the β2-adrenergic receptor, for which a large no. of ligands is already known.  Finding dissimilar ligands is thus the exception rather than the rule.  Here the authors report on the results, the successful synthesis of 127/240 mols. in just 2 wk, the discovery of previously unreported dissimilar ligands of the β2-adrenergic receptor, and the optimization of one series to a KD of 519 nM in only one round.  Moreover, the finding that only 3 of 240 mols. had ever been synthesized before indicates that large parts of chem. space are unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf1x0UkWHsv7Vg90H21EOLACvtfcHk0lgICgdNncvhbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2itL%252FO&md5=22fa702bc6a87b1c1a62228538775a8f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1818718116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1818718116%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DStotani%26aufirst%3DS.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DSteyaert%26aufirst%3Dj.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DInterrogating%2520dense%2520ligand%2520chemical%2520space%2520with%2520a%2520forward-synthetic%2520library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D11496%26epage%3D11501%26doi%3D10.1073%2Fpnas.1818718116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roch, C.</span></span> <span> </span><span class="NLM_article-title">Recent trends in orexin research—2010 to 2015</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2887</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2015.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=26045032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2MXos1elu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2875-2887&author=C.+Bossauthor=C.+Roch&title=Recent+trends+in+orexin+research%E2%80%942010+to+2015&doi=10.1016%2Fj.bmcl.2015.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent trends in orexin research-2010 to 2015</span></div><div class="casAuthors">Boss, Christoph; Roch, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2875-2887</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Specific neurons in the lateral hypothalamus produce the orexin neuropeptides (orexin-A and orexin-B).  The orexin-peptides are transported to areas of the brain regulating sleep-wake cycles, controlling food intake or modulating emotional states such as panic or anxiety.  The orexin system, consisting of the two orexin-neuropeptides and two G-protein-coupled receptors (the orexin-1 and the orexin-2 receptor) is as well involved in reward and addictive behaviors.  The review reflects on the most recent activities in the field of orexin research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWboUNL_DMdrVg90H21EOLACvtfcHk0lgICgdNncvhbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXos1elu7o%253D&md5=ad2a998773925487a7ad5a2745c02c8c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DRoch%26aufirst%3DC.%26atitle%3DRecent%2520trends%2520in%2520orexin%2520research%25E2%2580%25942010%2520to%25202015%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2875%26epage%3D2887%26doi%3D10.1016%2Fj.bmcl.2015.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnsmb.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=26950369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=293-299&author=J.+Yinauthor=K.+Babaogluauthor=C.+A.+Brautigamauthor=L.+Clarkauthor=Z.+Shaoauthor=T.+H.+Scheuermannauthor=C.+M.+Harrellauthor=A.+L.+Gotterauthor=A.+J.+Roeckerauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Colemanauthor=D.+M.+Rosenbaum&title=Structure+and+ligand-binding+mechanism+of+the+human+OX1+and+OX2+orexin+receptors&doi=10.1038%2Fnsmb.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span></div><div class="casAuthors">Yin, Jie; Babaoglu, Kerim; Brautigam, Chad A.; Clark, Lindsay; Shao, Zhenhua; Scheuermann, Thomas H.; Harrell, Charles M.; Gotter, Anthony L.; Roecker, Anthony J.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) regulate sleep and other behavioral functions in mammals, and are therapeutic targets for sleep and wake disorders.  The human receptors hOX1R and hOX2R, which are 64% identical in sequence, have overlapping but distinct physiol. functions and potential therapeutic profiles.  We detd. structures of hOX1R bound to the OX1R-selective antagonist SB-674042 and the dual antagonist suvorexant at 2.8-Å and 2.75-Å resoln., resp., and used mol. modeling to illuminate mechanisms of antagonist subtype selectivity between hOX1R and hOX2R.  The hOX1R structures also reveal a conserved amphipathic α-helix, in the extracellular N-terminal region, that interacts with orexin-A and is essential for high-potency neuropeptide activation at both receptors.  The orexin-receptor crystal structures are valuable tools for the design and development of selective orexin-receptor antagonists and agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAwovW0NYj7Vg90H21EOLACvtfcHk0ljxWs7-896_ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D&md5=611dec443eca3b38f3cd4f2ccbee86a1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3183%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DHarrell%26aufirst%3DC.%2BM.%26aulast%3DGotter%26aufirst%3DA.%2BL.%26aulast%3DRoecker%26aufirst%3DA.%2BJ.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DStructure%2520and%2520ligand-binding%2520mechanism%2520of%2520the%2520human%2520OX1%2520and%2520OX2%2520orexin%2520receptors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26spage%3D293%26epage%3D299%26doi%3D10.1038%2Fnsmb.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hellmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pröll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weikert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-based development of a subtype-selective orexin 1 receptor antagonist</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">18059</span>– <span class="NLM_lpage">18067</span>, <span class="refDoi"> DOI: 10.1073/pnas.2002704117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.2002704117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=32669442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsF2ltbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=18059-18067&author=J.+Hellmannauthor=M.+Drabekauthor=J.+Yinauthor=J.+Guneraauthor=T.+Pr%C3%B6llauthor=F.+Krausauthor=C.+J.+Langmeadauthor=H.+H%C3%BCbnerauthor=D.+Weikertauthor=P.+Kolbauthor=D.+M.+Rosenbaumauthor=P.+Gmeiner&title=Structure-based+development+of+a+subtype-selective+orexin+1+receptor+antagonist&doi=10.1073%2Fpnas.2002704117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based development of a subtype-selective orexin 1 receptor antagonist</span></div><div class="casAuthors">Hellmann, Jan; Drabek, Matthaeus; Yin, Jie; Gunera, Jakub; Proell, Theresa; Kraus, Frank; Langmead, Christopher J.; Huebner, Harald; Weikert, Dorothee; Kolb, Peter; Rosenbaum, Daniel M.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">18059-18067</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Orexins are neuropeptides that activate the rhodopsin-like G protein-coupled receptors OX1R and OX2R.  The orexin system plays an important role in the regulation of the sleep-wake cycle and the regulation of feeding and emotions.  The nonselective orexin receptor antagonist suvorexant has been the first drug on the market targeting the orexin system and is prescribed for the treatment of insomnia.  Subtype-selective OX1R antagonists are valuable tools to further investigate the functions and physiol. role of the OX1R in vivo and promising lead compds. for the treatment of drug addiction, anxiety, pain or obesity.  Starting from the OX1R and OX2R crystal structures bound to suvorexant, we exploited a single amino acid difference in the orthosteric binding site by using mol. docking and structure-based drug design to optimize ligand interactions with the OX1R while introducing repulsive interactions with the OX2R.  A newly established enantiospecific synthesis provided ligands showing up to 75-fold selectivity for the OX1R over the OX2R subtype.  The structure of a new OX1R antagonist with subnanomolar affinity (JH112) was detd. by crystallog. in complex with the OX1R and corresponded closely to the docking-predicted geometry.  JH112 exhibits high selectivity over a panel of different GPCRs, is able to cross the blood-brain barrier and acts as slowly diffusing and insurmountable antagonist for Gq protein activation and in particular β-arrestin-2 recruitment at OX1R.  This study demonstrates the potential of structure-based drug design to develop more subtype-selective GPCR ligands with potentially reduced side effects and provides an attractive probe mol. and lead compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnsy53cU4frVg90H21EOLACvtfcHk0ljxWs7-896_ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsF2ltbbF&md5=447067d8b0aaaeb94934536100da41f2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2002704117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2002704117%26sid%3Dliteratum%253Aachs%26aulast%3DHellmann%26aufirst%3DJ.%26aulast%3DDrabek%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DPr%25C3%25B6ll%26aufirst%3DT.%26aulast%3DKraus%26aufirst%3DF.%26aulast%3DLangmead%26aufirst%3DC.%2BJ.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DWeikert%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-based%2520development%2520of%2520a%2520subtype-selective%2520orexin%25201%2520receptor%2520antagonist%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D18059%26epage%3D18067%26doi%3D10.1073%2Fpnas.2002704117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span>Chemical
Computing Group,
Inc. Molecular Operating Environment (MOE) 2015.10. <i>1010 Sheerbrooke
St. West, Suite #910, Montreal, QC, Canada, H3A 2R7,</i> <b>2015</b>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chemical%0AComputing+Group%2C%0AInc.+Molecular+Operating+Environment+%28MOE%29+2015.10.+1010+Sheerbrooke%0ASt.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">ZINC - a free database of commercially available compounds for virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+-+a+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2ebB9Wb2KSPCh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520-%2520a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and the Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+the+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0ljTcLcMSIQUhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520the%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED Pose Prediction and Virtual Screening Accuracy</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1021/ci100436p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100436p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFGqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=578-596&author=M.+McGann&title=FRED+Pose+Prediction+and+Virtual+Screening+Accuracy&doi=10.1021%2Fci100436p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">FRED Pose Prediction and Virtual Screening Accuracy</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-596</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Results of a previous docking study are reanalyzed and extended to include results from the docking program FRED and a detailed statistical anal. of both structure reprodn. and virtual screening results.  FRED is run both in a traditional docking mode and in a hybrid mode that makes use of 16 the structure of a bound ligand in addn. to the protein structure to screen mols.  This anal. shows that most docking programs are effective overall but highly inconsistent, tending to do well on one system and poorly on the next.  Comparing methods, the difference in mean performance on DUD is found to be statistically significant (95% confidence) 61% of the time when using a global enrichment metric (AUC).  Early enrichment metrics are found to have relatively poor statistical power, with 0.5% early enrichment only able to distinguish methods to 95% confidence 14% of the time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVRPdIDmubh7Vg90H21EOLACvtfcHk0ljSCrnYIrgM2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFGqsbk%253D&md5=8f3ae5972583d0a89d50e35a4b01212e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci100436p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100436p%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520Pose%2520Prediction%2520and%2520Virtual%2520Screening%2520Accuracy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D578%26epage%3D596%26doi%3D10.1021%2Fci100436p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED and HYBRID docking performance on standardized datasets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9584-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1007%2Fs10822-012-9584-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=22669221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=897-906&author=M.+McGann&title=FRED+and+HYBRID+docking+performance+on+standardized+datasets&doi=10.1007%2Fs10822-012-9584-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">FRED and HYBRID docking performance on standardized datasets</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">897-906</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The docking performance of the FRED and HYBRID programs are evaluated on two standardized datasets from the Docking and Scoring Symposium of the ACS Spring 2011 national meeting.  The evaluation includes cognate docking and virtual screening performance.  FRED docks 70 % of the structures to within 2 Å in the cognate docking test.  In the virtual screening test, FRED is found to have a mean AUC of 0.75.  The HYBRID program uses a modified version of FRED's algorithm that uses both ligand- and structure-based information to dock mols., which increases its mean AUC to 0.78.  HYBRID can also implicitly account for protein flexibility by making use of multiple crystal structures.  Using multiple crystal structures improves HYBRID's performance (mean AUC 0.80) with a negligible increase in docking time (∼15 %).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDNF1kmgp-WLVg90H21EOLACvtfcHk0ljSCrnYIrgM2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO&md5=299f2e795ca922412f57df9434b52661</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9584-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9584-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520and%2520HYBRID%2520docking%2520performance%2520on%2520standardized%2520datasets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D897%26epage%3D906%26doi%3D10.1007%2Fs10822-012-9584-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+Shape-Matching+and+Docking+as+Virtual+Screening+Tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0ljSCrnYIrgM2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Shape-Matching%2520and%2520Docking%2520as%2520Virtual%2520Screening%2520Tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span> <span> </span><span class="NLM_article-title">Ligand solvation in molecular docking</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19990101)34:1<4::AID-PROT2>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10336382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFartQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=4-16&author=B.+K.+Shoichetauthor=A.+R.+Leachauthor=I.+D.+Kuntz&title=Ligand+solvation+in+molecular+docking&doi=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand solvation in molecular docking</span></div><div class="casAuthors">Shoichet, Brian K.; Leach, Andrew R.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-16</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Solvation plays an important role in ligand-protein assocn. and has a strong impact on comparisons of binding energies for dissimilar mols.  When databases of such mols. are screened for complementarity to receptors of known structure, as often occurs in structure-based inhibitor discovery, failure to consider ligand solvation often leads to putative ligands that are too highly charged or too large.  To correct for the different charge states and sizes of the ligands, we calcd. electrostatic and non-polar solvation free energies for mols. in a widely used mol. database, the Available Chems. Directory (ACD).  A modified Born equation treatment was used to calc. the electrostatic component of ligand solvation.  The non-polar component of ligand solvation was calcd. based on the surface area of the ligand and parameters derived from the hydration energies of apolar ligands.  These solvation energies were subtracted from the ligand-receptor interaction energies.  We tested the usefulness of these corrections by screening the ACD for mols. that complemented three proteins of known structure, using a mol. docking program.  Correcting for ligand solvation improved the rankings of known ligands and discriminated against mols. with inappropriate charge states and sizes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonWLewHwEkBLVg90H21EOLACvtfcHk0lhPQM414yBOhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFartQ%253D%253D&md5=781a144809eeb00b2796fc21ef7fef74</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819990101%252934%253A1%253C4%253A%253AAID-PROT2%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1999%26volume%3D34%26spage%3D4%26epage%3D16%26doi%3D10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein–Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2731</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/ci200274q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200274q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2731-2745&author=G.+Neudertauthor=G.+Klebe&title=DSX%3A+A+Knowledge-Based+Scoring+Function+for+the+Assessment+of+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fci200274q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein-Ligand Complexes</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2731-2745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the new knowledge-based scoring function DSX that consists of distance-dependent pair potentials, novel torsion angle potentials, and newly defined solvent accessible surface-dependent potentials.  DSX pair potentials are based on the statistical formalism of DrugScore, extended by a much more specialized set of atom types.  The original DrugScore-like ref. state is rather unstable with respect to modifications in the used atom types.  Therefore, an important method to overcome this problem and to allow for robust results when deriving pair potentials for arbitrary sets of atom types is presented.  A validation based on a carefully prepd. test set is shown, enabling direct comparison to the majority of other popular scoring functions.  Here, DSX features superior performance with respect to docking- and ranking power and runtime requirements.  Furthermore, the beneficial combination with torsion angle-dependent and desolvation-dependent potentials is demonstrated.  DSX is robust, flexible, and capable of working together with special features of popular docking engines, e.g., flexible protein residues in AutoDock or GOLD.  The program is freely available to the scientific community and can be downloaded from our Web site www.agklebe.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvQyrP7qPi5bVg90H21EOLACvtfcHk0lhOZCXeCTqwRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K&md5=c90af0aa9af9ae15d458d9ba759163bb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci200274q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200274q%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDSX%253A%2520A%2520Knowledge-Based%2520Scoring%2520Function%2520for%2520the%2520Assessment%2520of%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2731%26epage%3D2745%26doi%3D10.1021%2Fci200274q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span> <i>OpenEye</i> <span class="NLM_publisher-name">Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Sante Fe, NM, USA.</span> .</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OpenEye+Scientific+Software%2C+Inc.%3A+Sante+Fe%2C+NM%2C+USA.+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DOpenEye%26pub%3DScientific%2520Software%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Vigneshwaran Namasivayam, Katja Silbermann, Michael Wiese, Jens Pahnke, <span class="NLM_string-name hlFld-ContribAuthor">Sven Marcel Stefan</span>. </span><span class="cited-content_cbyCitation_article-title"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="91d2d1c1d0">[email protected]</a>: Computer-Aided Pattern Analysis to Predict Multitarget ABC Transporter Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3350-3366. <a href="https://doi.org/10.1021/acs.jmedchem.0c02199" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02199%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DC%252540PA%25253A%252BComputer-Aided%252BPattern%252BAnalysis%252Bto%252BPredict%252BMultitarget%252BABC%252BTransporter%252BInhibitors%26aulast%3DNamasivayam%26aufirst%3DVigneshwaran%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D20122020%26date%3D16032021%26volume%3D64%26issue%3D6%26spage%3D3350%26epage%3D3366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen  Gulezian</span>, <span class="hlFld-ContribAuthor ">Christina  Crivello</span>, <span class="hlFld-ContribAuthor ">Janna  Bednenko</span>, <span class="hlFld-ContribAuthor ">Claudia  Zafra</span>, <span class="hlFld-ContribAuthor ">Yihui  Zhang</span>, <span class="hlFld-ContribAuthor ">Paul  Colussi</span>, <span class="hlFld-ContribAuthor ">Sunyia  Hussain</span>. </span><span class="cited-content_cbyCitation_article-title">Membrane protein production and formulation for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (8)
                                     , 657-674. <a href="https://doi.org/10.1016/j.tips.2021.05.006" title="DOI URL">https://doi.org/10.1016/j.tips.2021.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2021.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2021.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DMembrane%252Bprotein%252Bproduction%252Band%252Bformulation%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DGulezian%26aufirst%3DEllen%26date%3D2021%26volume%3D42%26issue%3D8%26spage%3D657%26epage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two-dimensional representation of the binding mode of suvorexant (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>) generated with the MOE suite. Two interactions of suvorexant with OX<sub>2</sub>R are indicated as dashed green lines: an H-bond between amide carbonyl and Asn324<sup>6.55</sup> and a π–π interaction between 5-methyl-2-triazol-phenyl and His350<sup>7.39</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic of the primary docking screen workflow. (A) ZINC DrugsNow subset of ∼7.4 million compounds was subjected to conformer generation used for docking. In addition, ∼4 million ZINC LeadLike compounds were docked as is. (B) Three-fold postprocessing of the top 1% slice (6500 poses, 2000 each from HYBRID, DSX, and SZYBKI and 500 from DOCK) concluded by visual inspection, selection of compounds, and their experimental validation. (C) Schematic of the secondary similarity screen workflow. Eight experimentally validated hits were used as queries for a fingerprint-based similarity screen of three 2000 slices from the primary screen. The three sets of 30 nearest neighbors each were optimized with SZYBKI and subjected to visual inspection. Selected virtual hits were validated experimentally.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition of (A) <sup>3</sup>H-SB67404 binding to CHO-OX<sub>1</sub> cells and (B) <sup>3</sup>H-EMPA binding to CHO-OX<sub>2</sub> cells in response to <b>P33</b>, <b>F33.3</b>, and suvorexant. Bars represent total and nonspecific binding (as determined by 10 μM suvorexant) and data points are mean ± sem of triplicate determinations. These experiments are representative of four separate experiments. The concentration of radioligand in these experiments was (A) 1.14 and (B) 0.60 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ten compounds with the highest measured affinity in the <sup>3</sup>H-EMPA binding assay to CHO-OX<sub>2</sub> cells identified in this study. <b>P33</b> and <b>P35</b> compound families are emphasized with a box.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Intracellular calcium release in (A) CHO-OX<sub>1</sub> cells and (B) CHO-OX<sub>2</sub> cells in response to orexin A, <b>P33</b>, and <b>F33.3</b>. Bars represent basal intracellular calcium release and that in response to 10 μM ionomycin alone. Data points are mean ± sem of triplicate determinations and these single experiments are representative of four separate experiments in each case.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Depictions of (left) <b>F33.3</b> and (right) <b>P33</b> showing poses of the most potent and the most OX<sub>2</sub>R-selective compound, respectively. Compounds are shown in green sticks. Asn324<sup>6.55</sup> and His350<sup>7.39</sup> are shown in orange sticks (bold and thin, respectively). Helix 7 was removed for clarity. Color code: C, green; N, blue; O, red; S, yellow; Cl, dark green; F, cyan. PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/medium/jm0c00964_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Depictions of poses of (left) <b>P7</b> and (right) <b>F7.7</b>. Compounds are shown in green sticks. Asn324<sup>6.55</sup> and His350<sup>7.39</sup> are shown in orange sticks (bold and thin, respectively). Glu212<sup>45.52</sup> and Arg328<sup>6.59</sup> are shown in cyan sticks (bold and thin, respectively). Helix 7 was removed for clarity. Color code: C, green; N, blue; O, red; S, yellow; Cl, dark green; F, cyan. The protein is PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00964/20201001/images/large/jm0c00964_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00964&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bentivoglio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassi-Zucconi, G.</span></span> <span> </span><span class="NLM_article-title">The pioneering experimental studies on sleep deprivation</span>. <i>Sleep</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1093/sleep/20.7.570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1093%2Fsleep%2F20.7.570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=9322273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A280%3ADyaK2svmtlCqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1997&pages=570-576&author=M.+Bentivoglioauthor=G.+Grassi-Zucconi&title=The+pioneering+experimental+studies+on+sleep+deprivation&doi=10.1093%2Fsleep%2F20.7.570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The pioneering experimental studies on sleep deprivation</span></div><div class="casAuthors">Bentivoglio M; Grassi-Zucconi G</div><div class="citationInfo"><span class="NLM_cas:title">Sleep</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">570-6</span>
        ISSN:<span class="NLM_cas:issn">0161-8105</span>.
    </div><div class="casAbstract">The experimental studies on sleep deprivation were initiated by the Russian physician and scientist, Marie de Manaceine, who studied sleep-deprived puppies kept in constant activity.  She reported in 1894 that the complete absence of sleep was fatal in a few days, pointing out that the most severe lesions occurred in the brain.  In 1898, the Italian physiologists Lamberto Daddi and Giulio Tarozzi also kept dogs awake by walking them; the animals died after 9-17 days, and their survival was unrelated to food consumption.  In the histological study performed by Daddi, degenerative alterations, mainly represented by chromatolytic changes, were observed in neurons of the spinal ganglia, Purkinje cells of the cerebellum, and neurons of the frontal cortex.  Daddi ascribed these changes to a state of autointoxication of the brain during insomnia.  In 1898, the psychiatrist Cesare Agostini, interested in the psychic phenomena caused by prolonged insomnia in humans, sleep deprived dogs by keeping them in a metallic cage in order to avoid fatigue.  The dogs survived about 2 weeks, and degenerative changes were observed in their brains.  In these experimental paradigms, the effect of sleep loss was confounded by motor exhaustion and/or intense sensory stimulation.  In spite of the absence of adequate controls, the pioneering studies performed at the end of the 19th century represented the first experimental attempts to relate sleep with neural centers and suggested that sleep is a vital function and that the brain may be affected by insomnia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT1M6J7tIwvuVw2ofyr07jXfW6udTcc2eYMW_PxGF1oILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svmtlCqsQ%253D%253D&md5=88a763de83f5c54ab666be9e9d156866</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fsleep%2F20.7.570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fsleep%252F20.7.570%26sid%3Dliteratum%253Aachs%26aulast%3DBentivoglio%26aufirst%3DM.%26aulast%3DGrassi-Zucconi%26aufirst%3DG.%26atitle%3DThe%2520pioneering%2520experimental%2520studies%2520on%2520sleep%2520deprivation%26jtitle%3DSleep%26date%3D1997%26volume%3D20%26spage%3D570%26epage%3D576%26doi%3D10.1093%2Fsleep%2F20.7.570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czeisler, C. A.</span></span> <span> </span><span class="NLM_article-title">Impact of sleepiness and sleep deficiency on public health--utility of biomarkers</span>. <i>J. Clin. Sleep Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">S6</span>– <span class="NLM_lpage">S8</span>, <span class="refDoi"> DOI: 10.5664/JCSM.1340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.5664%2FJCSM.1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=22003334" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=S6-S8&author=C.+A.+Czeisler&title=Impact+of+sleepiness+and+sleep+deficiency+on+public+health%2D%2Dutility+of+biomarkers&doi=10.5664%2FJCSM.1340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.5664%2FJCSM.1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5664%252FJCSM.1340%26sid%3Dliteratum%253Aachs%26aulast%3DCzeisler%26aufirst%3DC.%2BA.%26atitle%3DImpact%2520of%2520sleepiness%2520and%2520sleep%2520deficiency%2520on%2520public%2520health--utility%2520of%2520biomarkers%26jtitle%3DJ.%2520Clin.%2520Sleep%2520Med.%26date%3D2011%26volume%3D7%26spage%3DS6%26epage%3DS8%26doi%3D10.5664%2FJCSM.1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisden, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundlach, A. L.</span></span> <span> </span><span class="NLM_article-title">Dual-transmitter systems regulating arousal, attention, learning and memory</span>. <i>Neurosci. Biobehav. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.neubiorev.2017.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.neubiorev.2017.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=28757457" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2017&pages=21-33&author=S.+Maauthor=B.+Hangyaauthor=C.+S.+Leonardauthor=W.+Wisdenauthor=A.+L.+Gundlach&title=Dual-transmitter+systems+regulating+arousal%2C+attention%2C+learning+and+memory&doi=10.1016%2Fj.neubiorev.2017.07.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.neubiorev.2017.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neubiorev.2017.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DS.%26aulast%3DHangya%26aufirst%3DB.%26aulast%3DLeonard%26aufirst%3DC.%2BS.%26aulast%3DWisden%26aufirst%3DW.%26aulast%3DGundlach%26aufirst%3DA.%2BL.%26atitle%3DDual-transmitter%2520systems%2520regulating%2520arousal%252C%2520attention%252C%2520learning%2520and%2520memory%26jtitle%3DNeurosci.%2520Biobehav.%2520Rev.%26date%3D2017%26volume%3D85%26spage%3D21%26epage%3D33%26doi%3D10.1016%2Fj.neubiorev.2017.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amemiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuzaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemelli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arch, J. R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckingham, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Car, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcnulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elshourbagy, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsma, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanagisawa, M.</span></span> <span> </span><span class="NLM_article-title">Orexins and orexin receptors: a family of hypothalamic neuropeptides and G protein-coupled receptors that regulate feeding behavior</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80949-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2FS0092-8674%2800%2980949-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=9491897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADyaK1cXhsVKntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1998&pages=573-585&author=T.+Sakuraiauthor=A.+Amemiyaauthor=M.+Ishiiauthor=I.+Matsuzakiauthor=R.+M.+Chemelliauthor=H.+Tanakaauthor=S.+C.+Williamsauthor=J.+A.+Richardsonauthor=G.+P.+Kozlowskiauthor=S.+Wilsonauthor=J.+R.+S.+Archauthor=R.+E.+Buckinghamauthor=A.+C.+Haynesauthor=S.+A.+Carauthor=R.+S.+Annanauthor=D.+E.+Mcnultyauthor=W.-S.+Liuauthor=J.+A.+Terrettauthor=N.+A.+Elshourbagyauthor=D.+J.+Bergsmaauthor=M.+Yanagisawa&title=Orexins+and+orexin+receptors%3A+a+family+of+hypothalamic+neuropeptides+and+G+protein-coupled+receptors+that+regulate+feeding+behavior&doi=10.1016%2FS0092-8674%2800%2980949-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Orexins and orexin receptors: a family of hypothalamic neuropeptides and G protein-coupled receptors that regulate feeding behavior</span></div><div class="casAuthors">Sakurai, Takeshi; Amemiya, Akira; Ishii, Makoto; Matsuzaki, Ichiyo; Chemelli, Richard M.; Tanaka, Hirokazu; Williams, S. Clay; Richardson, James A.; Kozlowski, Gerald P.; Wilson, Shelagh; Arch, Jonathan R. S.; Buckingham, Robin E.; Haynes, Andrea C.; Carr, Steven A.; Annan, Roland S.; Mcnulty, Dean E.; Liu, Wu-Schyong; Terrett, Jonathan A.; Elshourbagy, Nabil A.; Bergsma, Derk J.; Yanagisawa, Masashi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-585</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The hypothalamus plays a central role in the integrated control of feeding and energy homeostasis.  The authors have identified two novel neuropeptides, both derived from the same precursor by proteolytic processing, that bind and activate two closely related (previously) orphan G protein-coupled receptors.  These peptides, termed orexin-A and -B, have no significant structural similarities to known families of regulatory peptides.  Prepro-orexin mRNA and immunoreactive orexin-A are localized in neurons within and around the lateral and posterior hypothalamus in the adult rat brain.  When administered centrally to rats, these peptides stimulate food consumption.  Prepro-orexin mRNA level is up-regulated upon fasting, suggesting a physiol. role for the peptides as mediators in the central feedback mechanism that regulates feeding behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA5y2uicTVlLVg90H21EOLACvtfcHk0li8HnhhJXsOjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhsVKntr4%253D&md5=c0d02e69cd976521abfc5f88e8be7dda</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980949-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980949-6%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DT.%26aulast%3DAmemiya%26aufirst%3DA.%26aulast%3DIshii%26aufirst%3DM.%26aulast%3DMatsuzaki%26aufirst%3DI.%26aulast%3DChemelli%26aufirst%3DR.%2BM.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DWilliams%26aufirst%3DS.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BA.%26aulast%3DKozlowski%26aufirst%3DG.%2BP.%26aulast%3DWilson%26aufirst%3DS.%26aulast%3DArch%26aufirst%3DJ.%2BR.%2BS.%26aulast%3DBuckingham%26aufirst%3DR.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCar%26aufirst%3DS.%2BA.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DMcnulty%26aufirst%3DD.%2BE.%26aulast%3DLiu%26aufirst%3DW.-S.%26aulast%3DTerrett%26aufirst%3DJ.%2BA.%26aulast%3DElshourbagy%26aufirst%3DN.%2BA.%26aulast%3DBergsma%26aufirst%3DD.%2BJ.%26aulast%3DYanagisawa%26aufirst%3DM.%26atitle%3DOrexins%2520and%2520orexin%2520receptors%253A%2520a%2520family%2520of%2520hypothalamic%2520neuropeptides%2520and%2520G%2520protein-coupled%2520receptors%2520that%2520regulate%2520feeding%2520behavior%26jtitle%3DCell%26date%3D1998%26volume%3D92%26spage%3D573%26epage%3D585%26doi%3D10.1016%2FS0092-8674%2800%2980949-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobarec, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1038/nature14035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnature14035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=25533960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=247-250&author=J.+Yinauthor=J.+C.+Mobarecauthor=P.+Kolbauthor=D.+M.+Rosenbaum&title=Crystal+structure+of+the+human+OX2+orexin+receptor+bound+to+the+insomnia+drug+suvorexant&doi=10.1038%2Fnature14035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant</span></div><div class="casAuthors">Yin, Jie; Mobarec, Juan Carlos; Kolb, Peter; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">247-250</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) respond to orexin neuropeptides in the central nervous system to regulate sleep and other behavioral functions in humans.  Defects in orexin signalling are responsible for the human diseases of narcolepsy and cataplexy; inhibition of orexin receptors is an effective therapy for insomnia.  The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compds. including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant.  Here, using lipid-mediated crystn. and protein engineering with a novel fusion chimaera, we solved the structure of the human OX2R bound to suvorexant at 2.5 Å resoln.  The structure reveals how suvorexant adopts a π-stacked horseshoe-like conformation and binds to the receptor deep in the orthosteric pocket, stabilizing a network of extracellular salt bridges and blocking transmembrane (TM) helix motions necessary for activation.  Computational docking suggests how other classes of synthetic antagonists may interact with the receptor at a similar position in an analogous π-stacked fashion.  Elucidation of the mol. architecture of the human OX2R expands our understanding of peptidergic GPCR ligand recognition and will aid further efforts to modulate orexin signaling for therapeutic ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWKzbWNxv4W7Vg90H21EOLACvtfcHk0lj1aUw916K3Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntb0%253D&md5=461a6633274dbcfd9edce1a2e13b9c4f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature14035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14035%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DMobarec%26aufirst%3DJ.%2BC.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520OX2%2520orexin%2520receptor%2520bound%2520to%2520the%2520insomnia%2520drug%2520suvorexant%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D247%26epage%3D250%26doi%3D10.1038%2Fnature14035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogusky, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitman, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of N,N-disubstituted-1,4-diazepane orexin receptor antagonists and implications for receptor binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2997</span>– <span class="NLM_lpage">3001</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2009.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19406641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVeku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2997-3001&author=C.+D.+Coxauthor=G.+B.+McGaugheyauthor=M.+J.+Boguskyauthor=D.+B.+Whitmanauthor=R.+G.+Ballauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Coleman&title=Conformational+analysis+of+N%2CN-disubstituted-1%2C4-diazepane+orexin+receptor+antagonists+and+implications+for+receptor+binding&doi=10.1016%2Fj.bmcl.2009.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of N,N-disubstituted-1,4-diazepane orexin receptor antagonists and implications for receptor binding</span></div><div class="casAuthors">Cox, Christopher D.; McGaughey, Georgia B.; Bogusky, Michael J.; Whitman, David B.; Ball, Richard G.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2997-3001</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">NMR spectroscopy, X-ray crystallog., and mol. modeling studies indicate that N,N-disubstituted-1,4-diazepane orexin receptor antagonists exist in an unexpected low-energy conformation that is characterized by an intramol. π-stacking interaction and a twist-boat ring conformation.  Synthesis and evaluation of a macrocycle I that enforces a similar conformation suggest that this geometry mimics the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMJNXLU745TbVg90H21EOLACvtfcHk0lj1aUw916K3Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVeku7c%253D&md5=a3cec5d7ca8a09d9a14cd440d1cfd3b4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DC.%2BD.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DBogusky%26aufirst%3DM.%2BJ.%26aulast%3DWhitman%26aufirst%3DD.%2BB.%26aulast%3DBall%26aufirst%3DR.%2BG.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26atitle%3DConformational%2520analysis%2520of%2520N%252CN-disubstituted-1%252C4-diazepane%2520orexin%2520receptor%2520antagonists%2520and%2520implications%2520for%2520receptor%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2997%26epage%3D3001%26doi%3D10.1016%2Fj.bmcl.2009.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of β2-adrenergic receptor ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6848</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+%CE%B22-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0lj1aUw916K3Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520%25CE%25B22-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lg-jZlokBzmHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strachan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefkowitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Conformation Guides Molecular Efficacy in Docking Screens of Activated β-2 Adrenergic G Protein Coupled Receptor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1021/cb400103f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400103f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvVahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1018-1026&author=D.+R.+Weissauthor=S.+Ahnauthor=M.+F.+Sassanoauthor=A.+Kleistauthor=X.+Zhuauthor=R.+Strachanauthor=B.+L.+Rothauthor=R.+J.+Lefkowitzauthor=B.+K.+Shoichet&title=Conformation+Guides+Molecular+Efficacy+in+Docking+Screens+of+Activated+%CE%B2-2+Adrenergic+G+Protein+Coupled+Receptor&doi=10.1021%2Fcb400103f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Conformation Guides Molecular Efficacy in Docking Screens of Activated β-2 Adrenergic G Protein Coupled Receptor</span></div><div class="casAuthors">Weiss, Dahlia R.; Ahn, SeungKirl; Sassano, Maria F.; Kleist, Andrew; Zhu, Xiao; Strachan, Ryan; Roth, Bryan L.; Lefkowitz, Robert J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1018-1026</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A prospective, large library virtual screen against an activated β2-adrenergic receptor (β2AR) structure returned potent agonists to the exclusion of inverse-agonists, providing the first complement to the previous virtual screening campaigns against inverse-agonist-bound G protein coupled receptor (GPCR) structures, which predicted only inverse-agonists.  In addn., two hits recapitulated the signaling profile of the co-crystal ligand with respect to the G protein and arrestin mediated signaling.  This functional fidelity has important implications in drug design, as the ability to predict ligands with predefined signaling properties is highly desirable.  However, the agonist-bound state provides an uncertain template for modeling the activated conformation of other GPCRs, as a dopamine D2 receptor (DRD2) activated model templated on the activated β2AR structure returned few hits of only marginal potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5QKZOWGheuLVg90H21EOLACvtfcHk0lg-jZlokBzmHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvVahtbg%253D&md5=6394521533373197c59e49b242bf0c93</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcb400103f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400103f%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DKleist%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DStrachan%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DConformation%2520Guides%2520Molecular%2520Efficacy%2520in%2520Docking%2520Screens%2520of%2520Activated%2520%25CE%25B2-2%2520Adrenergic%2520G%2520Protein%2520Coupled%2520Receptor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1018%26epage%3D1026%26doi%3D10.1021%2Fcb400103f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topiol, S.</span></span> <span> </span><span class="NLM_article-title">Use of the X-ray structure of the beta2-adrenergic receptor for drug discovery. Part 2: Identification of active compounds</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5395</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.09.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2008.09.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=18829308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1CqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5391-5395&author=M.+Sabioauthor=K.+Jonesauthor=S.+Topiol&title=Use+of+the+X-ray+structure+of+the+beta2-adrenergic+receptor+for+drug+discovery.+Part+2%3A+Identification+of+active+compounds&doi=10.1016%2Fj.bmcl.2008.09.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the X-ray structure of the β2-adrenergic receptor for drug discovery. Part 2: Identification of active compounds</span></div><div class="casAuthors">Sabio, Michael; Jones, Kenneth; Topiol, Sid</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5391-5395</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The recently published X-ray structures of the β2-adrenergic receptor are the first examples of ligand-mediated GPCR crystal structures.  We have previously performed computational studies that examine the potential viability of these structures for use in drug design, exploiting known ligand activities.  Our previous study and a newly reported β2/Timolol X-ray complex provide validation of the computational approaches.  In the present work, we use the X-ray structures to ext., via in silico high-throughput docking, compds. from proprietary and com. databases and demonstrate the successful identification of active compds. by radioligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1AGaAYjN5WbVg90H21EOLACvtfcHk0lg-jZlokBzmHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1CqsrfP&md5=e91ea267e0851bc0e08b310b3f8966ee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.046%26sid%3Dliteratum%253Aachs%26aulast%3DSabio%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DTopiol%26aufirst%3DS.%26atitle%3DUse%2520of%2520the%2520X-ray%2520structure%2520of%2520the%2520beta2-adrenergic%2520receptor%2520for%2520drug%2520discovery.%2520Part%25202%253A%2520Identification%2520of%2520active%2520compounds%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5391%26epage%3D5395%26doi%3D10.1016%2Fj.bmcl.2008.09.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brox, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity using Multi-Target Docking</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1021/cb500577j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500577j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=715-724&author=D.+Schmidtauthor=V.+Bernatauthor=R.+Broxauthor=N.+Tschammerauthor=P.+Kolb&title=Identifying+Modulators+of+CXC+Receptors+3+and+4+with+Tailored+Selectivity+using+Multi-Target+Docking&doi=10.1021%2Fcb500577j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity Using Multi-Target Docking</span></div><div class="casAuthors">Schmidt, Denis; Bernat, Viachaslau; Brox, Regine; Tschammer, Nuska; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">715-724</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G protein-coupled receptors of the C-X-C subfamily form a group among the chemokine receptors whose endogenous ligands are peptides with a common Cys-X-Cys motif.  The CXC chemokine receptors 3 and 4 (CXCR3, CXCR4), which are investigated in this study, are linked to severe diseases such as cancer, multiple sclerosis, and HIV infections.  Of particular interest, this receptor pair potentially forms a target for a polypharmacol. drug treatment.  Considering known ligands from public databases, such dual binders have not been identified yet.  We therefore applied large-scale docking to the structure of CXCR4 and a homol. model of CXCR3 with the goal to predict such dual binders, as well as compds. selective for either one of the receptors.  Using signaling and biochem. assays, we showed that more than 50% of these predictions were correct in each category, yielding ligands with excellent binding efficiencies.  These results highlight that docking is a suitable tool for the identification of ligands with tailored binding profiles to GPCRs, even when using homol. models.  More importantly, we present novel CXCR3-CXCR4 dual modulators that might pave the road to understanding the mechanisms of polypharmacol. inhibition of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Px6d1pbV3rVg90H21EOLACvtfcHk0lhDUyOedFM3zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM&md5=9259d56e4c28e68360f3ac214ea62ea4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb500577j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500577j%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DBrox%26aufirst%3DR.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DIdentifying%2520Modulators%2520of%2520CXC%2520Receptors%25203%2520and%25204%2520with%2520Tailored%2520Selectivity%2520using%2520Multi-Target%2520Docking%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D715%26epage%3D724%26doi%3D10.1021%2Fcb500577j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerman, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdon, H. J.</span></span> <span> </span><span class="NLM_article-title">Characterisation of the binding of [3H]-SB-674042, a novel nonpeptide antagonist, to the human orexin-1 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fsj.bjp.0705610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=14691055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2004&pages=340-346&author=C.+J.+Langmeadauthor=J.+C.+Jermanauthor=S.+J.+Broughauthor=C.+Scottauthor=R.+A.+Porterauthor=H.+J.+Herdon&title=Characterisation+of+the+binding+of+%5B3H%5D-SB-674042%2C+a+novel+nonpeptide+antagonist%2C+to+the+human+orexin-1+receptor&doi=10.1038%2Fsj.bjp.0705610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of the binding of [3H]-SB-674042, a novel nonpeptide antagonist, to the human orexin-1 receptor</span></div><div class="casAuthors">Langmead, Christopher J.; Jerman, Jeffrey C.; Brough, Stephen J.; Scott, Claire; Porter, Rod A.; Herdon, Hugh J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">340-346</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This study characterizes the binding of a novel nonpeptide antagonist radioligand, [3H]SB-674042 (1-(5-(2-fluoro-phenyl)-2-methyl-thiazol-4-yl)-1-((S)-2-(5-phenyl-(1,3,4)oxadiazol-2-ylmethyl)-pyrrolidin-l-yl)-methanone), to the human orexin-1 (OX1) receptor stably expressed in Chinese hamster ovary (CHO) cells in both a whole cell assay and in a cell membrane-based scintillation proximity assay (SPA) format.  Specific binding of [3H]SB-674042 was saturable in both whole cell and membrane formats.  Analyses suggested a single high-affinity site, with Kd values of 3.76 ± 0.45 and 5.03 ± 0.31 nM, and corresponding Bmax values of 30.8±1.8 and 34.4±2.0 pmol mg protein-1, in whole cell and membrane formats, resp.  Kinetic studies yielded similar Kd values.  Competition studies in whole cells revealed that the native orexin peptides display a low affinity for the OX1 receptor, with orexin-A displaying a ∼five-fold higher affinity than orexin-B (Ki values of 318±158 and 1516±597 nM, resp.).  SB-334867, SB-408124 (1-(6,8-difluoro-2-methyl-quinolin-4-yl)-3-(4-dimethylamino-phenyl)-urea) and SB-410220 (1-(5,8-difluoro-quinolin-4-yl)-3-(4-dimethylamino-phenyl)-urea) all displayed high affinity for the OX1 receptor in both whole cell (Ki values 99±18, 57±8.3 and 19±4.5 nM, resp.) and membrane (Ki values 38±3.6, 27±4.1 and 4.5±0.2 nM, resp.) formats.  Calcium mobilization studies showed that SB-334867, SB-408124 and SB-410220 are all functional antagonists of the OX1 receptor, with potencies in line with their affinities, as measured in the radioligand binding assays, and with approx. 50-fold selectivity over the orexin-2 receptor.  These studies indicate that [3H]SB-674042 is a specific, high-affinity radioligand for the OX1 receptor.  The availability of this radioligand will be a valuable tool with which to investigate the physiol. functions of OX1 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGKhe4L8-w0bVg90H21EOLACvtfcHk0lhDUyOedFM3zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOisLk%253D&md5=9fe91c966ce2fd2b631efe0b29f75e15</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705610%26sid%3Dliteratum%253Aachs%26aulast%3DLangmead%26aufirst%3DC.%2BJ.%26aulast%3DJerman%26aufirst%3DJ.%2BC.%26aulast%3DBrough%26aufirst%3DS.%2BJ.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DHerdon%26aufirst%3DH.%2BJ.%26atitle%3DCharacterisation%2520of%2520the%2520binding%2520of%2520%255B3H%255D-SB-674042%252C%2520a%2520novel%2520nonpeptide%2520antagonist%252C%2520to%2520the%2520human%2520orexin-1%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2004%26volume%3D141%26spage%3D340%26epage%3D346%26doi%3D10.1038%2Fsj.bjp.0705610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malherbe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knust, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettekoven, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers-Evans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettstein, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, J. L.</span></span> <span> </span><span class="NLM_article-title">Biochemical and behavioural characterization of EMPA, a novel high-affinity, selective antagonist for the OX(2) receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1341</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1111%2Fj.1476-5381.2009.00127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=19751316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2009&pages=1326-1341&author=P.+Malherbeauthor=E.+Borroniauthor=L.+Gobbiauthor=H.+Knustauthor=M.+Nettekovenauthor=E.+Pinardauthor=O.+Rocheauthor=M.+Rogers-Evansauthor=J.+G.+Wettsteinauthor=J.+L.+Moreau&title=Biochemical+and+behavioural+characterization+of+EMPA%2C+a+novel+high-affinity%2C+selective+antagonist+for+the+OX%282%29+receptor&doi=10.1111%2Fj.1476-5381.2009.00127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and behavioural characterization of EMPA, a novel high-affinity, selective antagonist for the OX2 receptor</span></div><div class="casAuthors">Malherbe, P.; Borroni, E.; Gobbi, L.; Knust, H.; Nettekoven, M.; Pinard, E.; Roche, O.; Rogers-Evans, M.; Wettstein, J. G.; Moreau, J.-L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1326-1341</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The OX2 receptor is a G-protein-coupled receptor that is abundantly found in the tuberomammillary nucleus, an important site for the regulation of the sleep-wake state.  Herein, we describe the in vitro and in vivo properties of a selective OX2 receptor antagonist, N-ethyl-2-[(6-methoxy-pyridin-3-yl)-(toluene-2-sulfonyl)-amino]-N-pyridin-3-ylmethyl-acetamide (EMPA).  The affinity of [3H]EMPA was assessed in membranes from HEK293-hOX2-cells using satn. and binding kinetics.  The antagonist properties of EMPA were detd. by Schild anal. using the orexin-A- or orexin-B-induced accumulation of [3H]inositol phosphates (IP).  Quant. autoradiog. was used to det. the distribution and abundance of OX2 receptors in rat brain.  The in vivo activity of EMPA was assessed by reversal of [Ala11,D-Leu15]orexin-B-induced hyperlocomotion during the resting phase in mice and the redn. of spontaneous locomotor activity (LMA) during the active phase in rats.  [3H]EMPA bound to human and rat OX2-HEK293 membranes with KD values of 1.1 and 1.4 nmol/L-1 resp.  EMPA competitively antagonized orexin-A- and orexin-B-evoked accumulation of [3H]IP at hOX2 receptors with pA2 values of 8.6 and 8.8 resp.  Autoradiog. of rat brain confirmed the selectivity of [3H]EMPA for OX2 receptors.  EMPA significantly reversed [Ala11,D-Leu15]orexin-B-induced hyperlocomotion dose-dependently during the resting phase in mice.  EMPA, injected i.p. in rats during the active phase, reduced LMA dose-dependently.  EMPA did not impair performance of rats in the rotarod procedure.  EMPA is a high-affinity, reversible and selective OX2 receptor antagonist, active in vivo, which should prove useful for anal. of OX2 receptor function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2LdWKzYFpjbVg90H21EOLACvtfcHk0lhDUyOedFM3zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVWmurc%253D&md5=88d91d8772c3f282498ed1a2e9cd5f6b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00127.x%26sid%3Dliteratum%253Aachs%26aulast%3DMalherbe%26aufirst%3DP.%26aulast%3DBorroni%26aufirst%3DE.%26aulast%3DGobbi%26aufirst%3DL.%26aulast%3DKnust%26aufirst%3DH.%26aulast%3DNettekoven%26aufirst%3DM.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DRoche%26aufirst%3DO.%26aulast%3DRogers-Evans%26aufirst%3DM.%26aulast%3DWettstein%26aufirst%3DJ.%2BG.%26aulast%3DMoreau%26aufirst%3DJ.%2BL.%26atitle%3DBiochemical%2520and%2520behavioural%2520characterization%2520of%2520EMPA%252C%2520a%2520novel%2520high-affinity%252C%2520selective%2520antagonist%2520for%2520the%2520OX%25282%2529%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D156%26spage%3D1326%26epage%3D1341%26doi%3D10.1111%2Fj.1476-5381.2009.00127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proudman, R. G.</span>; <span class="NLM_string-name">Baker, J. G.</span></span> <i>The affinity and selectivity of orexin A, orexin B and orexin antagonists for the human orexin 1 and orexin 2 receptors</i>, <span class="NLM_year">2018</span>. . <a href="http://www.pa2online.org/abstract/abstract.jsp?abid=33656" class="extLink">http://www.pa2online.org/abstract/abstract.jsp?abid=33656</a> (accessed August 30, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+G.+Proudman&author=J.+G.+Baker&title=The+affinity+and+selectivity+of+orexin+A%2C+orexin+B+and+orexin+antagonists+for+the+human+orexin+1+and+orexin+2+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DProudman%26aufirst%3DR.%2BG.%26jtitle%3DThe%2520affinity%2520and%2520selectivity%2520of%2520orexin%2520A%252C%2520orexin%2520B%2520and%2520orexin%2520antagonists%2520for%2520the%2520human%2520orexin%25201%2520and%2520orexin%25202%2520receptors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Hilten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Binding-Site Compatible Fragment Growing Applied to the Design of β<sub>2</sub>-Adrenergic Receptor Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1129</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVKnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1118-1129&author=F.+Chevillardauthor=H.+Rimmerauthor=C.+Bettiauthor=E.+Pardonauthor=S.+Balletauthor=N.+van+Hiltenauthor=J.+Steyaertauthor=W.+E.+Diederichauthor=P.+Kolb&title=Binding-Site+Compatible+Fragment+Growing+Applied+to+the+Design+of+%CE%B22-Adrenergic+Receptor+Ligands&doi=10.1021%2Facs.jmedchem.7b01558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands</span></div><div class="casAuthors">Chevillard, Florent; Rimmer, Helena; Betti, Cecilia; Pardon, Els; Ballet, Steven; van Hilten, Niek; Steyaert, Jan; Diederich, Wibke E.; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1118-1129</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design.  Although computers allow for the facile generation of millions of suggestions, synthetic feasibility is however often neglected.  In this study the authors computationally extended, chem. synthesized, and exptl. assayed new ligands for the β2-adrenergic receptor (β2AR) by growing fragment-sized ligands.  To address the synthetic tractability issue, the authors' in silico work-flow aims at derivatized products based on robust org. reactions.  The study started from the predicted binding modes of five fragments.  The authors suggested a total of eight diverse extensions that were easily synthesized, and further assays showed that four products had an improved affinity (up to 40-fold) compared to their resp. initial fragment.  The described work-flow, which the authors call "growing via merging" and for which the key tools are available online, can improve early fragment-based drug discovery projects, making it a useful creative tool for medicinal chemists during structure-activity relationship (SAR) studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGNVNIASFV5LVg90H21EOLACvtfcHk0ljfw-LD4-ORlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVKnu7s%253D&md5=f915fcba6096ec506f49274a2b628b97</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01558%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DRimmer%26aufirst%3DH.%26aulast%3DBetti%26aufirst%3DC.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DBallet%26aufirst%3DS.%26aulast%3Dvan%2BHilten%26aufirst%3DN.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DBinding-Site%2520Compatible%2520Fragment%2520Growing%2520Applied%2520to%2520the%2520Design%2520of%2520%25CE%25B22-Adrenergic%2520Receptor%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1118%26epage%3D1129%26doi%3D10.1021%2Facs.jmedchem.7b01558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stotani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span> <span> </span><span class="NLM_article-title">Interrogating dense ligand chemical space with a forward-synthetic library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">11496</span>– <span class="NLM_lpage">11501</span>, <span class="refDoi"> DOI: 10.1073/pnas.1818718116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.1818718116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=31113876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2itL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=11496-11501&author=F.+Chevillardauthor=S.+Stotaniauthor=A.+Karawajczykauthor=S.+Hristevaauthor=E.+Pardonauthor=j.+Steyaertauthor=D.+Tzalisauthor=P.+Kolb&title=Interrogating+dense+ligand+chemical+space+with+a+forward-synthetic+library&doi=10.1073%2Fpnas.1818718116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interrogating dense ligand chemical space with a forward-synthetic library</span></div><div class="casAuthors">Chevillard, Florent; Stotani, Silvia; Karawajczyk, Anna; Hristeva, Stanimira; Pardon, Els; Steyaert, Jan; Tzalis, Dimitrios; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11496-11501</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Forward-synthetic databases are an efficient way to enumerate chem. space.  The authors explored here whether these databases are good sources of novel protein ligands and how many mols. are obtainable and in which time frame.  Based on docking calcns., series of mols. were selected to gain insights into the ligand structur-activity relation.  To evaluate the novelty of compds. in a challenging way, the authors chose the β2-adrenergic receptor, for which a large no. of ligands is already known.  Finding dissimilar ligands is thus the exception rather than the rule.  Here the authors report on the results, the successful synthesis of 127/240 mols. in just 2 wk, the discovery of previously unreported dissimilar ligands of the β2-adrenergic receptor, and the optimization of one series to a KD of 519 nM in only one round.  Moreover, the finding that only 3 of 240 mols. had ever been synthesized before indicates that large parts of chem. space are unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf1x0UkWHsv7Vg90H21EOLACvtfcHk0ljfw-LD4-ORlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2itL%252FO&md5=22fa702bc6a87b1c1a62228538775a8f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1818718116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1818718116%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DStotani%26aufirst%3DS.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DSteyaert%26aufirst%3Dj.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DInterrogating%2520dense%2520ligand%2520chemical%2520space%2520with%2520a%2520forward-synthetic%2520library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D11496%26epage%3D11501%26doi%3D10.1073%2Fpnas.1818718116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roch, C.</span></span> <span> </span><span class="NLM_article-title">Recent trends in orexin research—2010 to 2015</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2887</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1016%2Fj.bmcl.2015.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=26045032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC2MXos1elu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2875-2887&author=C.+Bossauthor=C.+Roch&title=Recent+trends+in+orexin+research%E2%80%942010+to+2015&doi=10.1016%2Fj.bmcl.2015.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent trends in orexin research-2010 to 2015</span></div><div class="casAuthors">Boss, Christoph; Roch, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2875-2887</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Specific neurons in the lateral hypothalamus produce the orexin neuropeptides (orexin-A and orexin-B).  The orexin-peptides are transported to areas of the brain regulating sleep-wake cycles, controlling food intake or modulating emotional states such as panic or anxiety.  The orexin system, consisting of the two orexin-neuropeptides and two G-protein-coupled receptors (the orexin-1 and the orexin-2 receptor) is as well involved in reward and addictive behaviors.  The review reflects on the most recent activities in the field of orexin research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWboUNL_DMdrVg90H21EOLACvtfcHk0ljfw-LD4-ORlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXos1elu7o%253D&md5=ad2a998773925487a7ad5a2745c02c8c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DRoch%26aufirst%3DC.%26atitle%3DRecent%2520trends%2520in%2520orexin%2520research%25E2%2580%25942010%2520to%25202015%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2875%26epage%3D2887%26doi%3D10.1016%2Fj.bmcl.2015.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1038%2Fnsmb.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=26950369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=293-299&author=J.+Yinauthor=K.+Babaogluauthor=C.+A.+Brautigamauthor=L.+Clarkauthor=Z.+Shaoauthor=T.+H.+Scheuermannauthor=C.+M.+Harrellauthor=A.+L.+Gotterauthor=A.+J.+Roeckerauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Colemanauthor=D.+M.+Rosenbaum&title=Structure+and+ligand-binding+mechanism+of+the+human+OX1+and+OX2+orexin+receptors&doi=10.1038%2Fnsmb.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span></div><div class="casAuthors">Yin, Jie; Babaoglu, Kerim; Brautigam, Chad A.; Clark, Lindsay; Shao, Zhenhua; Scheuermann, Thomas H.; Harrell, Charles M.; Gotter, Anthony L.; Roecker, Anthony J.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) regulate sleep and other behavioral functions in mammals, and are therapeutic targets for sleep and wake disorders.  The human receptors hOX1R and hOX2R, which are 64% identical in sequence, have overlapping but distinct physiol. functions and potential therapeutic profiles.  We detd. structures of hOX1R bound to the OX1R-selective antagonist SB-674042 and the dual antagonist suvorexant at 2.8-Å and 2.75-Å resoln., resp., and used mol. modeling to illuminate mechanisms of antagonist subtype selectivity between hOX1R and hOX2R.  The hOX1R structures also reveal a conserved amphipathic α-helix, in the extracellular N-terminal region, that interacts with orexin-A and is essential for high-potency neuropeptide activation at both receptors.  The orexin-receptor crystal structures are valuable tools for the design and development of selective orexin-receptor antagonists and agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAwovW0NYj7Vg90H21EOLACvtfcHk0ljfw-LD4-ORlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D&md5=611dec443eca3b38f3cd4f2ccbee86a1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3183%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DHarrell%26aufirst%3DC.%2BM.%26aulast%3DGotter%26aufirst%3DA.%2BL.%26aulast%3DRoecker%26aufirst%3DA.%2BJ.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DStructure%2520and%2520ligand-binding%2520mechanism%2520of%2520the%2520human%2520OX1%2520and%2520OX2%2520orexin%2520receptors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26spage%3D293%26epage%3D299%26doi%3D10.1038%2Fnsmb.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hellmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pröll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weikert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-based development of a subtype-selective orexin 1 receptor antagonist</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">18059</span>– <span class="NLM_lpage">18067</span>, <span class="refDoi"> DOI: 10.1073/pnas.2002704117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1073%2Fpnas.2002704117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=32669442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsF2ltbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=18059-18067&author=J.+Hellmannauthor=M.+Drabekauthor=J.+Yinauthor=J.+Guneraauthor=T.+Pr%C3%B6llauthor=F.+Krausauthor=C.+J.+Langmeadauthor=H.+H%C3%BCbnerauthor=D.+Weikertauthor=P.+Kolbauthor=D.+M.+Rosenbaumauthor=P.+Gmeiner&title=Structure-based+development+of+a+subtype-selective+orexin+1+receptor+antagonist&doi=10.1073%2Fpnas.2002704117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based development of a subtype-selective orexin 1 receptor antagonist</span></div><div class="casAuthors">Hellmann, Jan; Drabek, Matthaeus; Yin, Jie; Gunera, Jakub; Proell, Theresa; Kraus, Frank; Langmead, Christopher J.; Huebner, Harald; Weikert, Dorothee; Kolb, Peter; Rosenbaum, Daniel M.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">18059-18067</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Orexins are neuropeptides that activate the rhodopsin-like G protein-coupled receptors OX1R and OX2R.  The orexin system plays an important role in the regulation of the sleep-wake cycle and the regulation of feeding and emotions.  The nonselective orexin receptor antagonist suvorexant has been the first drug on the market targeting the orexin system and is prescribed for the treatment of insomnia.  Subtype-selective OX1R antagonists are valuable tools to further investigate the functions and physiol. role of the OX1R in vivo and promising lead compds. for the treatment of drug addiction, anxiety, pain or obesity.  Starting from the OX1R and OX2R crystal structures bound to suvorexant, we exploited a single amino acid difference in the orthosteric binding site by using mol. docking and structure-based drug design to optimize ligand interactions with the OX1R while introducing repulsive interactions with the OX2R.  A newly established enantiospecific synthesis provided ligands showing up to 75-fold selectivity for the OX1R over the OX2R subtype.  The structure of a new OX1R antagonist with subnanomolar affinity (JH112) was detd. by crystallog. in complex with the OX1R and corresponded closely to the docking-predicted geometry.  JH112 exhibits high selectivity over a panel of different GPCRs, is able to cross the blood-brain barrier and acts as slowly diffusing and insurmountable antagonist for Gq protein activation and in particular β-arrestin-2 recruitment at OX1R.  This study demonstrates the potential of structure-based drug design to develop more subtype-selective GPCR ligands with potentially reduced side effects and provides an attractive probe mol. and lead compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnsy53cU4frVg90H21EOLACvtfcHk0lg-CDpanY9lvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsF2ltbbF&md5=447067d8b0aaaeb94934536100da41f2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2002704117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2002704117%26sid%3Dliteratum%253Aachs%26aulast%3DHellmann%26aufirst%3DJ.%26aulast%3DDrabek%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DPr%25C3%25B6ll%26aufirst%3DT.%26aulast%3DKraus%26aufirst%3DF.%26aulast%3DLangmead%26aufirst%3DC.%2BJ.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DWeikert%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-based%2520development%2520of%2520a%2520subtype-selective%2520orexin%25201%2520receptor%2520antagonist%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D18059%26epage%3D18067%26doi%3D10.1073%2Fpnas.2002704117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span>Chemical
Computing Group,
Inc. Molecular Operating Environment (MOE) 2015.10. <i>1010 Sheerbrooke
St. West, Suite #910, Montreal, QC, Canada, H3A 2R7,</i> <b>2015</b>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chemical%0AComputing+Group%2C%0AInc.+Molecular+Operating+Environment+%28MOE%29+2015.10.+1010+Sheerbrooke%0ASt.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">ZINC - a free database of commercially available compounds for virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+-+a+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2ebkVzrhXRL6R7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520-%2520a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and the Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+the+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lg-CDpanY9lvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520the%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED Pose Prediction and Virtual Screening Accuracy</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1021/ci100436p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100436p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFGqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=578-596&author=M.+McGann&title=FRED+Pose+Prediction+and+Virtual+Screening+Accuracy&doi=10.1021%2Fci100436p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">FRED Pose Prediction and Virtual Screening Accuracy</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-596</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Results of a previous docking study are reanalyzed and extended to include results from the docking program FRED and a detailed statistical anal. of both structure reprodn. and virtual screening results.  FRED is run both in a traditional docking mode and in a hybrid mode that makes use of 16 the structure of a bound ligand in addn. to the protein structure to screen mols.  This anal. shows that most docking programs are effective overall but highly inconsistent, tending to do well on one system and poorly on the next.  Comparing methods, the difference in mean performance on DUD is found to be statistically significant (95% confidence) 61% of the time when using a global enrichment metric (AUC).  Early enrichment metrics are found to have relatively poor statistical power, with 0.5% early enrichment only able to distinguish methods to 95% confidence 14% of the time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVRPdIDmubh7Vg90H21EOLACvtfcHk0ljhm3zws_aBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFGqsbk%253D&md5=8f3ae5972583d0a89d50e35a4b01212e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci100436p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100436p%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520Pose%2520Prediction%2520and%2520Virtual%2520Screening%2520Accuracy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D578%26epage%3D596%26doi%3D10.1021%2Fci100436p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED and HYBRID docking performance on standardized datasets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9584-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1007%2Fs10822-012-9584-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=22669221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=897-906&author=M.+McGann&title=FRED+and+HYBRID+docking+performance+on+standardized+datasets&doi=10.1007%2Fs10822-012-9584-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">FRED and HYBRID docking performance on standardized datasets</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">897-906</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The docking performance of the FRED and HYBRID programs are evaluated on two standardized datasets from the Docking and Scoring Symposium of the ACS Spring 2011 national meeting.  The evaluation includes cognate docking and virtual screening performance.  FRED docks 70 % of the structures to within 2 Å in the cognate docking test.  In the virtual screening test, FRED is found to have a mean AUC of 0.75.  The HYBRID program uses a modified version of FRED's algorithm that uses both ligand- and structure-based information to dock mols., which increases its mean AUC to 0.78.  HYBRID can also implicitly account for protein flexibility by making use of multiple crystal structures.  Using multiple crystal structures improves HYBRID's performance (mean AUC 0.80) with a negligible increase in docking time (∼15 %).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDNF1kmgp-WLVg90H21EOLACvtfcHk0ljhm3zws_aBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO&md5=299f2e795ca922412f57df9434b52661</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9584-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9584-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520and%2520HYBRID%2520docking%2520performance%2520on%2520standardized%2520datasets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D897%26epage%3D906%26doi%3D10.1007%2Fs10822-012-9584-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+Shape-Matching+and+Docking+as+Virtual+Screening+Tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0ljhm3zws_aBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Shape-Matching%2520and%2520Docking%2520as%2520Virtual%2520Screening%2520Tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span> <span> </span><span class="NLM_article-title">Ligand solvation in molecular docking</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19990101)34:1<4::AID-PROT2>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=10336382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFartQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=4-16&author=B.+K.+Shoichetauthor=A.+R.+Leachauthor=I.+D.+Kuntz&title=Ligand+solvation+in+molecular+docking&doi=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand solvation in molecular docking</span></div><div class="casAuthors">Shoichet, Brian K.; Leach, Andrew R.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-16</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Solvation plays an important role in ligand-protein assocn. and has a strong impact on comparisons of binding energies for dissimilar mols.  When databases of such mols. are screened for complementarity to receptors of known structure, as often occurs in structure-based inhibitor discovery, failure to consider ligand solvation often leads to putative ligands that are too highly charged or too large.  To correct for the different charge states and sizes of the ligands, we calcd. electrostatic and non-polar solvation free energies for mols. in a widely used mol. database, the Available Chems. Directory (ACD).  A modified Born equation treatment was used to calc. the electrostatic component of ligand solvation.  The non-polar component of ligand solvation was calcd. based on the surface area of the ligand and parameters derived from the hydration energies of apolar ligands.  These solvation energies were subtracted from the ligand-receptor interaction energies.  We tested the usefulness of these corrections by screening the ACD for mols. that complemented three proteins of known structure, using a mol. docking program.  Correcting for ligand solvation improved the rankings of known ligands and discriminated against mols. with inappropriate charge states and sizes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonWLewHwEkBLVg90H21EOLACvtfcHk0ljeThE4cAG4CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFartQ%253D%253D&md5=781a144809eeb00b2796fc21ef7fef74</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819990101%252934%253A1%253C4%253A%253AAID-PROT2%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1999%26volume%3D34%26spage%3D4%26epage%3D16%26doi%3D10.1002%2F%28SICI%291097-0134%2819990101%2934%3A1%3C4%3A%3AAID-PROT2%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neudert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein–Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2731</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/ci200274q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200274q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2731-2745&author=G.+Neudertauthor=G.+Klebe&title=DSX%3A+A+Knowledge-Based+Scoring+Function+for+the+Assessment+of+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fci200274q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">DSX: A Knowledge-Based Scoring Function for the Assessment of Protein-Ligand Complexes</span></div><div class="casAuthors">Neudert, Gerd; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2731-2745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the new knowledge-based scoring function DSX that consists of distance-dependent pair potentials, novel torsion angle potentials, and newly defined solvent accessible surface-dependent potentials.  DSX pair potentials are based on the statistical formalism of DrugScore, extended by a much more specialized set of atom types.  The original DrugScore-like ref. state is rather unstable with respect to modifications in the used atom types.  Therefore, an important method to overcome this problem and to allow for robust results when deriving pair potentials for arbitrary sets of atom types is presented.  A validation based on a carefully prepd. test set is shown, enabling direct comparison to the majority of other popular scoring functions.  Here, DSX features superior performance with respect to docking- and ranking power and runtime requirements.  Furthermore, the beneficial combination with torsion angle-dependent and desolvation-dependent potentials is demonstrated.  DSX is robust, flexible, and capable of working together with special features of popular docking engines, e.g., flexible protein residues in AutoDock or GOLD.  The program is freely available to the scientific community and can be downloaded from our Web site www.agklebe.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvQyrP7qPi5bVg90H21EOLACvtfcHk0ljeThE4cAG4CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Ggtb3K&md5=c90af0aa9af9ae15d458d9ba759163bb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci200274q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200274q%26sid%3Dliteratum%253Aachs%26aulast%3DNeudert%26aufirst%3DG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDSX%253A%2520A%2520Knowledge-Based%2520Scoring%2520Function%2520for%2520the%2520Assessment%2520of%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2731%26epage%3D2745%26doi%3D10.1021%2Fci200274q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span> <i>OpenEye</i> <span class="NLM_publisher-name">Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Sante Fe, NM, USA.</span> .</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OpenEye+Scientific+Software%2C+Inc.%3A+Sante+Fe%2C+NM%2C+USA.+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DOpenEye%26pub%3DScientific%2520Software%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V','PDB','4S0V'); return false;">PDB: 4S0V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJ8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJ8','PDB','4ZJ8'); return false;">PDB: 4ZJ8</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00964">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_37266"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00964?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00964</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tabular data containing compound structures, SMILES strings, and supplier information; Tanimoto fingerprint similarities, ROCS TanimotoCombo scores, docking scores, and retrosynthetical analysis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Dose–response curves (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Orexin receptor data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_003.csv">CSV</a>)</p></li><li><p class="inline">Predicted complex structures for all molecules (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_004.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_001.pdf">jm0c00964_si_001.pdf (1.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_002.pdf">jm0c00964_si_002.pdf (382.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_003.csv">jm0c00964_si_003.csv (11.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00964/suppl_file/jm0c00964_si_004.zip">jm0c00964_si_004.zip (13.59 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00964&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-19%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00964%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00964" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997732ed203cfa","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
