
<html lang="en"     class="pb-page"  data-request-id="b011f8ce-ab35-4cc1-91fa-f404b6adfcd0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.0c00111;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rapid Elaboration of Fragments into Leads by X-ray Crystallographic Screening of Parallel Chemical Libraries (REFiLX)" /></meta><meta name="dc.Creator" content="Matthew R.  Bentley" /></meta><meta name="dc.Creator" content="Olga V.  Ilyichova" /></meta><meta name="dc.Creator" content="Geqing  Wang" /></meta><meta name="dc.Creator" content="Martin L.  Williams" /></meta><meta name="dc.Creator" content="Gaurav  Sharma" /></meta><meta name="dc.Creator" content="Wesam S.  Alwan" /></meta><meta name="dc.Creator" content="Rebecca L.  Whitehouse" /></meta><meta name="dc.Creator" content="Biswaranjan  Mohanty" /></meta><meta name="dc.Creator" content="Peter J.  Scammells" /></meta><meta name="dc.Creator" content="Begoña  Heras" /></meta><meta name="dc.Creator" content="Jennifer L.  Martin" /></meta><meta name="dc.Creator" content="Makrina  Totsika" /></meta><meta name="dc.Creator" content="Ben  Capuano" /></meta><meta name="dc.Creator" content="Bradley C.  Doak" /></meta><meta name="dc.Creator" content="Martin J.  Scanlon" /></meta><meta name="dc.Description" content="A bottleneck in fragment-based lead development is the lack of systematic approaches to elaborate the initial fragment hits, which usually bind with low affinity to their target. Herein, we describ..." /></meta><meta name="Description" content="A bottleneck in fragment-based lead development is the lack of systematic approaches to elaborate the initial fragment hits, which usually bind with low affinity to their target. Herein, we describ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 12, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00111" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00111" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00111" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00111" /></link>
        
    
    

<title>Rapid Elaboration of Fragments into Leads by X-ray Crystallographic Screening of Parallel Chemical Libraries (REFiLX) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00111" /></meta><meta property="og:title" content="Rapid Elaboration of Fragments into Leads by X-ray Crystallographic Screening of Parallel Chemical Libraries (REFiLX)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0007.jpeg" /></meta><meta property="og:description" content="A bottleneck in fragment-based lead development is the lack of systematic approaches to elaborate the initial fragment hits, which usually bind with low affinity to their target. Herein, we describe an analysis using X-ray crystallography of a diverse library of compounds prepared using microscale parallel synthesis. This approach yielded an 8-fold increase in affinity and detailed structural information for the resulting complex, providing an efficient and broadly applicable approach to early fragment development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00111"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00111">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00111&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00111&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00111&amp;href=/doi/10.1021/acs.jmedchem.0c00111" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 6863-6875</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00225" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00233" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rapid Elaboration of Fragments into Leads by X-ray Crystallographic Screening of Parallel Chemical Libraries (REFiL<sub>X</sub>)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Matthew R. Bentley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew R. Bentley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+R.++Bentley">Matthew R. Bentley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olga V. Ilyichova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olga V. Ilyichova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olga+V.++Ilyichova">Olga V. Ilyichova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Geqing Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Geqing Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Genetics, La Trobe Institute for Molecular Science, La Trobe University, Melbourne, VIC 3086, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Geqing++Wang">Geqing Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin L. Williams</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin L. Williams</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin+L.++Williams">Martin L. Williams</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gaurav Sharma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gaurav Sharma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gaurav++Sharma">Gaurav Sharma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wesam S. Alwan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wesam S. Alwan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wesam+S.++Alwan">Wesam S. Alwan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebecca L. Whitehouse</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebecca L. Whitehouse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca+L.++Whitehouse">Rebecca L. Whitehouse</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Biswaranjan Mohanty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biswaranjan Mohanty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">ARC Training Centre for Fragment Based Design, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biswaranjan++Mohanty">Biswaranjan Mohanty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. Scammells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Scammells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Scammells">Peter J. Scammells</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2930-895X" title="Orcid link">http://orcid.org/0000-0003-2930-895X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Begoña Heras</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Begoña Heras</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Genetics, La Trobe Institute for Molecular Science, La Trobe University, Melbourne, VIC 3086, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bego%C3%B1a++Heras">Begoña Heras</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer L. Martin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer L. Martin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Griffith Institute for Drug Discovery, Building N75, Brisbane Innovation Park, Don Young Road, Nathan, QLD 4111, Australia</div><div class="loa-info-affiliations-info">Vice-Chancellor’s Unit, University of Wollongong, Northfields Avenue, Wollongong, NSW 2522, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+L.++Martin">Jennifer L. Martin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Makrina Totsika</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Makrina Totsika</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Biomedical Sciences, Institute of Health and Biomedical Innovation, Queensland University of Technology, Brisbane, QLD 4059, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Makrina++Totsika">Makrina Totsika</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ben Capuano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ben Capuano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">ARC Training Centre for Fragment Based Design, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ben++Capuano">Ben Capuano</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5434-0180" title="Orcid link">http://orcid.org/0000-0001-5434-0180</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bradley C. Doak</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bradley C. Doak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">ARC Training Centre for Fragment Based Design, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">Monash Fragment Platform, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6e0c1c0f0a020b17400a010f052e0301000f1d06400b0a1b"><span class="__cf_email__" data-cfemail="6b09190a0f070e12450f040a002b0604050a1803450e0f1e">[email protected]</span></a>. Phone: +61 3 9903 9117.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bradley+C.++Doak">Bradley C. Doak</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Martin J. Scanlon</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin J. Scanlon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">ARC Training Centre for Fragment Based Design, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div><div class="loa-info-affiliations-info">Monash Fragment Platform, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#82efe3f0f6ebecacf1e1e3eceeedecc2efedece3f1eaace7e6f7"><span class="__cf_email__" data-cfemail="cea3afbcbaa7a0e0bdadafa0a2a1a08ea3a1a0afbda6e0abaabb">[email protected]</span></a>. Phone: +61 3 9903 9540.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin+J.++Scanlon">Martin J. Scanlon</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9230-7506" title="Orcid link">http://orcid.org/0000-0002-9230-7506</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00111&amp;href=/doi/10.1021%2Facs.jmedchem.0c00111" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 6863–6875</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 12, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 February 2020</li><li><span class="item_label"><b>Published</b> online</span>12 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00111</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6863%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMatthew%2BR.%2BBentley%252C%2BOlga%2BV.%2BIlyichova%252C%2BGeqing%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D13%26contentID%3Dacs.jmedchem.0c00111%26title%3DRapid%2BElaboration%2Bof%2BFragments%2Binto%2BLeads%2Bby%2BX-ray%2BCrystallographic%2BScreening%2Bof%2BParallel%2BChemical%2BLibraries%2B%2528REFiLX%2529%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6875%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00111"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1378</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00111" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rapid Elaboration of Fragments into Leads by X-ray Crystallographic Screening of Parallel Chemical Libraries (REFiLX)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;R. Bentley&quot;},{&quot;first_name&quot;:&quot;Olga&quot;,&quot;last_name&quot;:&quot;V. Ilyichova&quot;},{&quot;first_name&quot;:&quot;Geqing&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;L. Williams&quot;},{&quot;first_name&quot;:&quot;Gaurav&quot;,&quot;last_name&quot;:&quot;Sharma&quot;},{&quot;first_name&quot;:&quot;Wesam&quot;,&quot;last_name&quot;:&quot;S. Alwan&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;L. Whitehouse&quot;},{&quot;first_name&quot;:&quot;Biswaranjan&quot;,&quot;last_name&quot;:&quot;Mohanty&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Scammells&quot;},{&quot;first_name&quot;:&quot;Begoña&quot;,&quot;last_name&quot;:&quot;Heras&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;L. Martin&quot;},{&quot;first_name&quot;:&quot;Makrina&quot;,&quot;last_name&quot;:&quot;Totsika&quot;},{&quot;first_name&quot;:&quot;Ben&quot;,&quot;last_name&quot;:&quot;Capuano&quot;},{&quot;first_name&quot;:&quot;Bradley&quot;,&quot;last_name&quot;:&quot;C. Doak&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;J. Scanlon&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6863-6875&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00111&quot;},&quot;abstract&quot;:&quot;A bottleneck in fragment-based lead development is the lack of systematic approaches to elaborate the initial fragment hits, which usually bind with low affinity to their target. Herein, we describe an analysis using X-ray crystallography of a diverse library of compounds prepared using microscale parallel synthesis. This approach yielded an 8-fold increase in affinity and detailed structural information for the resulting complex, providing an efficient and broadly applicable approach to early fragment development.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00111&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00111" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00111&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00111" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00111&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00111" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00111&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00111&amp;href=/doi/10.1021/acs.jmedchem.0c00111" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00111" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00111" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00111%26sid%3Dliteratum%253Aachs%26pmid%3D32529824%26genre%3Darticle%26aulast%3DBentley%26date%3D2020%26atitle%3DRapid%2BElaboration%2Bof%2BFragments%2Binto%2BLeads%2Bby%2BX-ray%2BCrystallographic%2BScreening%2Bof%2BParallel%2BChemical%2BLibraries%2B%2528REFiLX%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D6863%26epage%3D6875%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292272" title="Chemical reactions">Chemical reactions</a>,</li><li><a href="/action/doSearch?ConceptID=291100" title="Electron density">Electron density</a>,</li><li><a href="/action/doSearch?ConceptID=292382" title="X ray crystallography">X ray crystallography</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A bottleneck in fragment-based lead development is the lack of systematic approaches to elaborate the initial fragment hits, which usually bind with low affinity to their target. Herein, we describe an analysis using X-ray crystallography of a diverse library of compounds prepared using microscale parallel synthesis. This approach yielded an 8-fold increase in affinity and detailed structural information for the resulting complex, providing an efficient and broadly applicable approach to early fragment development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28003" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28003" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Over the past 20 years, fragment-based screening has become established as a robust method to find starting points for drug development.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Due to their small size, fragment hits typically form a small number of interactions with their target protein, which results in a low binding affinity. As a consequence, significant medicinal chemistry effort is required to produce higher affinity lead compounds.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> This design–synthesis–evaluation cycle required to elaborate fragment hits can become a bottleneck to development. Therefore, methods that enable a more rapid and systematic approach to fragment elaboration are needed.</div><div class="NLM_p">One such method, termed off-rate screening (ORS) by surface plasmon resonance (SPR), was described by Murray et al.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In this approach, compounds with higher affinity are detected by virtue of their slower dissociation kinetics observed in SPR sensorgrams. The dissociation rate is independent of free analyte concentration, which means that this method does not require the concentration of a compound to be determined before it is tested. Moreover, since weakly binding compounds dissociate rapidly, the dissociation phase of the sensorgram for a mixture is dominated by the most slowly dissociating component, even when it is a minor component of the mixture.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In addition, SPR requires only small quantities of materials, allowing moderately sized libraries of compounds to be synthesized on a small scale for testing by ORS without extensive purification. However, this approach requires that the dissociation rate constant for the test compounds be slow enough to be resolved by SPR. Typically, this requires that the dissociation rate constant <i>k</i><sub>off</sub> < 1 s<sup>–1</sup>. Assuming an association rate constant in the range of <i>k</i><sub>on</sub> = 1 × 10<sup>5</sup>–1 × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup>, this produces a threshold whereby the equilibrium dissociation constant for the products must be <i>K</i><sub>D</sub> <1–10 μM before ORS can be employed. As noted above, it is common for the initial fragment hits, particularly for difficult targets, to bind with weak affinity, i.e., often <i>K</i><sub>D</sub> > 500 μM. Therefore, in order for ORS to be successful, a significant improvement (>50-fold) would be required to meet the affinity threshold where dissociation kinetics are likely to be observed. In an alternative approach, X-ray crystallography has been used previously in fragment development in combination with dynamic combinatorial chemistry (DCC) to identify more potent compounds in situ.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, DCC is limited in scope by the small number of reactions suitable for in situ chemistry. Therefore, we sought to develop a method applicable to a broader range of chemistries<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (harsher conditions and organic solvents), as well as be applicable to the development of low-affinity fragments that bind too weakly to be amenable to ORS by SPR.</div><div class="NLM_p">Advantages of using X-ray crystallography include that it can be used to characterize binding across a broad range of affinities, including weakly binding compounds, and that it provides important structural information for subsequent chemical elaboration. Furthermore, recent advances in X-ray crystallography data collection and analysis, such as automation of crystallization experiments, development of single-photon-counting detectors, pipeline processing software, and multi-data-set analysis methods for identification of ligand binding, have enabled significantly higher throughput and more routine screening of libraries.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> We sought to exploit these developments and to utilize X-ray crystallography to evaluate the binding of libraries of compounds prepared in a similar manner to that reported for ORS by SPR, thereby taking advantage of the ability to synthesize compounds on a small scale without extensive purification. The drawbacks of X-ray crystallography screening may include false positives/negatives, ambiguity in the binding mode due to crystal contacts, and errors in interpretation of electron density. Despite these challenges, we hypothesized that screening unpurified reaction products by X-ray crystallography could identify compounds of higher affinity and provide key structural information to enable the rapid elaboration and evaluation of fragments that bind to targets for which ORS by SPR and other techniques are not applicable.</div><div class="NLM_p last">Herein, we report an initial proof-of-principle study that we performed on an <i>Escherichia coli</i> disulfide bond forming protein A (<i>Ec</i>DsbA). We have previously undertaken a fragment screen against <i>Ec</i>DsbA and demonstrated that fragment binding is amenable to analysis by X-ray crystallography.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a><i>Ec</i>DsbA is a disulfide oxidoreductase enzyme that is responsible for catalyzing the formation of disulfide bonds in a wide range of substrate proteins, including secreted proteins and virulence factors.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Therefore, inhibitors of <i>Ec</i>DsbA have been suggested as antivirulence agents.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a><i>Ec</i>DsbA can be considered a “difficult” target for drug development since its binding site is a shallow, extended hydrophobic groove that accommodates a diverse array of substrates. In addition, due to its hydrophobic nature, there are relatively few opportunities within this binding groove for specific polar interactions that enable the generation of high-affinity small-molecule binders. The aim of the current study was to improve the affinity of a previously validated hit fragment containing a diaryl ether core,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> using this method, which we termed rapid elaboration of fragments into leads using X-ray crystallography (REFiL<sub>X</sub>).</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Diaryl ether <b>1</b> was identified from a fragment screen and demonstrated weak binding to <i>Ec</i>DsbA (<i>K</i><sub>D</sub> > 1000 μM; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Structure-based approaches to elaborate fragment hit <b>1</b> by expanding the phenol to an acetic acid to optimize interactions with His32 together with further substitution of the benzonitrile ring to fill the hydrophobic pocket led to the synthesis of phenylacetic acid <b>2</b>, which bound to <i>Ec</i>DsbA with an improved affinity of <i>K</i><sub>D</sub> = 490 ± 20 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d). Analysis of the X-ray crystal structure of <b>2</b> bound to <i>Ec</i>DsbA suggests that the acetate moiety forms a charge reinforced hydrogen-bonding interaction with the imidazole NH of His32 in the active site of <i>Ec</i>DsbA, while the cyano group participates in a hydrogen bond with the side-chain amide NH of Gln35. The diaryl ether core fills a portion of the hydrophobic groove adjacent to the active site of <i>Ec</i>DsbA and contributes to π-stacking interactions with the side chains of His32 and Phe174 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of phenylacetic acid <b>2</b> binding to <i>Ec</i>DsbA. (a) Binding mode of phenylacetic acid <b>2</b> to oxidized <i>Ec</i>DsbA from the X-ray crystal structure (protein databank (PDB) code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD">6WHD</a>). Carbon atoms of phenylacetic acid <b>2</b> are shown as gray sticks, oxygen atoms are shown in red, and nitrogens in blue. The binding site on <i>Ec</i>DsbA is shown as a blue-gray surface with the carbon atoms of key residues shown as green sticks and active site cysteines (Cys30 and Cys33) as yellow spheres. Red dashed lines represent hydrogen-bond and polar interactions, and black dashed lines show π-stacking interactions. (b) Structure of diaryl ether hit <b>1</b> and phenylacetic acid <b>2</b>. Affinities of <b>1</b> and <b>2</b>, given as <i>K</i><sub>D</sub> ± error of fit, were calculated from NMR data using a single-site binding model with ligand depletion. (c) <sup>1</sup>H–<sup>15</sup>N heteronuclear single quantum coherence (HSQC) spectra of oxidized <sup>15</sup>N <i>Ec</i>DsbA (100 μM) in the absence (blue) and presence (red) of phenylacetic acid <b>2</b> (1.09 mM). The inset shows a titration with phenylacetic acid <b>2</b> at concentrations of 0, 136, 273, 545, 818, and 1090 μM with contours colored blue, green, orange, cyan, purple, and red, respectively. (d) Analysis of the <sup>1</sup>H–<sup>15</sup>N HSQC titration data, plotting the concentration of <b>2</b> (μM) vs chemical shift perturbation (CSP) in ppm, observed for residues that showed CSP > 0.04 ppm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To explore the development of analogues expanding from the carboxylic acid of <b>2</b>, a library of 93 diverse amine reagents was used. The library was designed to maximize the coverage of the three-dimensional (3D) pharmacophore space around the expansion vector while at the same time balancing physicochemical properties (Clog <i>P</i>, total polar surface area (tPSA), flexibility, complexity, etc.) and synthetic tractability. This was achieved using a custom in-house fingerprint and selection algorithm. A full list of the diverse amine library as well as desired products is given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The library was constrained to add 5–12 heavy atoms to the parent fragment, which, assuming a group efficiency of 0.3 kcal/(mol HAC), would potentially yield ∼10–100-fold improvement in affinity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Protecting groups were used where the reagents contained functional groups that were likely to interfere with the amide formation chemistry to be employed, which required deprotection prior to screening. Due to the wide range of reactivities expected for the amine reagents in the library, we first optimized the reaction conditions to identify those that gave the best yields across a small test set of representative amines of varying reactivities, comprising one aromatic, one primary, and one secondary amine. A series of 45 reactions were conducted varying the coupling agent, equivalents of amine, temperature, and solvent (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S2</a>). Initial exploration of solvent and equivalents of amine suggested that acetonitrile (MeCN) solvent with 1.5 equiv of amine was preferred, giving all three products in purity of 14–43%. Further optimization reactions in MeCN then indicated that the 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU) coupling agent at 25 °C with 1.5 equiv of amine was preferred, showing a product for the aromatic, primary, and secondary amines with increasing purity ranging between 20 and 85% and in line with the expected reactivity. Simultaneously, crystals of <i>Ec</i>DsbA were evaluated for their sensitivity to possible reaction conditions (reagents, solvents, and byproducts) and soaking times for the different conditions (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). These sensitivity experiments revealed that crystals soaked with (1-cyano-2-ethoxy-2-oxoethylidenaminooxy)dimethylamino-morpholino-carbenium hexafluorophosphate (COMU) diffracted at significantly lower resolution (2.4 ± 0.2 Å) compared to those soaked with 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC) and HATU (1.8 ± 0.1 and 1.9 ± 0.1 Å, respectively). The final reaction conditions chosen for the library synthesis were the HATU coupling agent in MeCN with 1.5 equiv of amine at 25 °C. These conditions gave consistent purity and produced high-resolution data by X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. REFiL<sub>X</sub> workflow diagram. Optimization: reaction conditions were optimized on a microscale in conjunction with optimizing X-ray soaking conditions and assessing the crystal diffraction quality. Screening: microscale reactions with a library of diverse reagents were performed in a 96-well plate; protecting groups were then cleaved and dimethyl sulfoxide (DMSO) stocks prepared. <i>Ec</i>DsbA crystals were soaked with unpurified reaction products in DMSO stocks; diffraction data were collected and processed automatically at the Australian Synchrotron. Analysis: sample purities were estimated by liquid chromatography–mass spectrometry (LCMS). Hits were identified where electron density maps additional density at the binding site that was inconsistent with the structure of the parent phenylacetic acid <b>2</b>. Confirmation: hits were resynthesized as pure compounds for full characterization and affinity determination. Pure compounds were cocrystallized with <i>Ec</i>DsbA and diffraction data measured to obtain high-resolution structures of the complexes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Solubility, Affinity, and PDB Codes for Crude Samples and Purified Analogues <b>2</b> and <b>6</b>–<b>12</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0005.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0006.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Presented as the mean ± standard error of the mean (SEM) from three independent experiments. For the other titrations, the affinity is reported as the fitted <i>K</i><sub>D</sub> value ± the error in the fit.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">The ligand complex for X-ray structure determination was generated by soaking crystals of <i>Ec</i>DsbA in a solution of phenylacetic acid <b>2</b>. All other complexes for pure compounds were prepared by cocrystallization.</p></div></div><div></div></div><div class="NLM_p">Reactions were performed on a microscale (2 μmol) in parallel in 96-well plates where each well contained a single amine reagent. Upon completion of the reactions, deprotections were performed as required (trifluoroacetic acid (TFA) treatment for acid labile and NaOH treatment for base labile protecting groups), solvents were removed by evaporation, and the reactions were dissolved in DMSO to generate a stock. The stocks were prepared without further purification at a notional concentration of 100 mM assuming 100% conversion to product in each reaction. The reactions were analyzed via LCMS to identify the presence of the desired product and to estimate purity. For 65 of the 93 reactions, the product was unambiguously identified by both UV–vis and mass spectrometries (MS). Of the remaining 28 reactions, only 3 identified the desired product mass in MS, while 15 showed only an additional peak in the UV–vis trace, consistent with the expected product. For all reactions where a UV–vis peak was observed for the product, purities were calculated by measuring the area of the new peak divided by the total area of all peaks, excluding the HATU peak. The remaining 10 reactions showed no evidence of product by either MS or UV–vis. The purities ranged from 0 (i.e., no product observed) to 90%, with an average of 25% across all 93 reactions. Significant amounts of phenylacetic acid starting material <b>2</b> were observed in the majority of low-purity samples. The importance of purity/conversion in the subsequent X-ray crystallography screening was investigated further, as described below. Each DMSO stock was diluted 10-fold and soaked into <i>Ec</i>DsbA crystals using previously described soaking and cryoprotection conditions.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Data were collected at MX1 and MX2 beamlines at the Australian Synchrotron and processed with the automated data-processing pipeline implemented at the beamline.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> Briefly, each data set was indexed, integrated, and scaled with xdsme and Aimless.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> The output statistical description of the data was manually inspected, and data sets with poor statistics were rejected at this stage (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). In total, 90 of the 93 samples had data sets with acceptable collection statistics and gave high-resolution data within the range of 1.8–2.5 Å (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figures S1 and S2</a>). For 9 of these 90 samples, lower sample concentration and shorter soaking times were required. The remaining three samples did not produce data sets of sufficient quality after multiple soaking and collection attempts and were not pursued further. All data sets were phased by molecular replacement (MR) and refined automatically using the Auto-Rickshaw molecular replacement multi-data-set pipeline,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> and the output was inspected manually. In addition, the data sets were analyzed using Pan-Dataset Density Analysis (PanDDA)<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> to automatically identify bound ligands. Eighty-two of the 90 samples that were analyzed showed electron density consistent with the parent phenylacetic acid <b>2</b>.</div><div class="NLM_p">Four samples were identified as hits due to the presence of additional electron density in the <i>Ec</i>DsbA maps, which was inconsistent with the parent phenylacetic acid <b>2</b> and consistent with the expected positioning of the corresponding amide expansion (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>; compounds <b>6</b>–<b>9</b>). These four hits were resynthesized on a batch scale and purified to obtain full chemical characterization and to enable the measurement of <i>K</i><sub>D</sub>. For the chiral morpholine carboxylic acid (compound <b>7</b>), synthesis of the racemic mixture was conducted. The binding modes of the purified compounds were validated by X-ray crystallography.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0004.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Hit Compounds and Control Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>,<i>N</i>-dimethylglycine hydrochloride, copper(I) iodide, Cs<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 100 °C, 18 h, 54%. (b) NaOH, MeOH, H<sub>2</sub>O, 25 °C, 3 h, 90%. (c) HATU, Et<sub>3</sub>N, amine, DMF, MeCN, 25 °C, 18 h, 30%. (d) (COCl)<sub>2</sub>, DMF cat., CDCl<sub>3</sub>, 0 °C, 5 h, then amine, Et<sub>3</sub>N, CDCl<sub>3</sub>, 25 °C, 24 h, 4–64%. Purities of library synthesized samples estimated by LCMS for hits <b>6</b> = 74%, <b>7</b> = 55%, <b>8</b> = 83%, and <b>9</b> = 62% as well as control analogues <b>10</b> = 86%, <b>11</b> = 6%, and <b>12</b> = 0%.</p></p></figure><div class="NLM_p">The solubility of resynthesized analogues was determined by recording a quantitative one-dimensional (1D) <sup>1</sup>H NMR of each compound in aqueous (aq) buffer at multiple concentrations as previously described (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Binding to <i>Ec</i>DsbA was assessed by recording <sup>1</sup>H–<sup>15</sup>N heteronuclear single quantum coherence (HSQC) spectra of oxidized <sup>15</sup>N-labeled <i>Ec</i>DsbA with increasing amounts of ligand. Titrations were continued until the ligand concentration reached either 1 mM or, where the ligand was not soluble at 1 mM, the maximum concentration at which they were deemed soluble. Affinities were determined by measuring chemical shift perturbations (CSPs) of backbone amide resonances as a function of ligand concentration and fitting the data to a single-site binding model, which accounts for the ligand depletion expected in the data. All of the hits demonstrated similar or better affinity for <i>Ec</i>DsbA compared to the starting phenylacetic acid <b>2</b>, with the highest affinity analogue pyrazole <b>9</b> showing an 8-fold improvement in affinity (<i>K</i><sub>D</sub> = 62 ± 9 μM, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The binding affinity for this fragment was confirmed by recording additional HSQC data at a protein concentration below the measured <i>K</i><sub>D</sub> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S3</a>). Purified compounds were analyzed by cocrystallization, which confirmed the binding mode to <i>Ec</i>DsbA and produced more complete electron density maps compared with those obtained from the crude products. This allowed us to model ligands into the electron density maps unambiguously (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S3</a>). Analysis of the crystal structure of pyrazole <b>9</b> bound to <i>Ec</i>DsbA identified polar and π-stacking interactions similar to those observed for the parent <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a) with additional water-mediated interactions of the amide carbonyl oxygen of pyrazole <b>9</b> with the backbone NH of His32. The pyrazole side chain of <b>9</b> projects out from the shallow groove of <i>Ec</i>DsbA and folds back toward the active site, where it forms an additional π-stacking interaction with the imidazole ring of His32. The additional interactions observed in this unexpected binding mode most likely underpin the increase in affinity of pyrazole <b>9</b> over the parent <b>2</b> in binding to <i>Ec</i>DsbA.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Binding mode of pyrazole <b>9</b> to oxidized <i>Ec</i>DsbA from the X-ray crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>). Carbon atoms of pyrazole <b>9</b> are shown as gray sticks, oxygen atoms are shown in red, and nitrogens in blue. The binding site on <i>Ec</i>DsbA is shown as a blue-gray surface with the carbon atoms of key residues shown as green sticks and active site cysteines (Cys30 and Cys33) as yellow spheres. Red dashed lines represent hydrogen bond and polar interactions, and black dashed lines show π-stacking interactions. (b) Electron density map of pyrazole <b>9</b> (cocrystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>) showing the 2mFo–DFc map contoured at 1σ (blue) and the mFo–DFc maps shown at ±3σ (positive green and negative red). (c) Structure of pyrazole <b>9</b> and its affinity for <i>Ec</i>DsbA estimated from <sup>1</sup>H–<sup>15</sup>N HSQC titrations. (d) <sup>1</sup>H–<sup>15</sup>N HSQC spectra of oxidized <sup>15</sup>N <i>Ec</i>DsbA (100 μM) in the absence (blue) and presence (red) of pyrazole <b>9</b> (195 μM). The inset shows a titration with pyrazole <b>9</b> at concentrations of 0, 78, 117, 156, and 195 μM with contours colored blue, green, orange, cyan, and red, respectively. (e) Analysis of the <sup>1</sup>H–<sup>15</sup>N HSQC titration data, plotting the concentration of <b>9</b> (μM) vs chemical shift perturbation (CSP) in ppm, observed for residues that showed CSP > 0.04 ppm. Data from three replicate titration experiments are plotted as mean ± SEM, fitted to a single-site binding model accounting for ligand depletion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Analysis of the LCMS data for the four hits as described above revealed that they had >50% purity in the crude library synthesis. To assess the effects of reaction conversion and purity on the outcome of the REFiL<sub>X</sub> screening strategy, several additional compounds were resynthesized and purified (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> compounds <b>10</b>–<b>12</b>). The library analogue oxadiazole <b>11</b> is structurally similar to the highest affinity hit pyrazole <b>9</b> and gave only low reaction purity in the library (6%) as estimated by LCMS; hence, it was resynthesized to test for false negatives in the screen. Screening by <sup>1</sup>H–<sup>15</sup>N HQSC determined its <i>K</i><sub>D</sub> to be 230 ± 20 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S4</a>), indicating that oxadiazole <b>11</b> did indeed bind with a 2-fold improvement in <i>K</i><sub>D</sub> over the parent <b>2</b>. Additional high conversion products <b>10</b> and low conversion product <b>12</b> that were not identified as hits were also resynthesized and confirmed as having similar or weaker affinity for <i>Ec</i>DsbA compared to parent <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). A structure was subsequently determined for oxadiazole <b>11</b> following cocrystallization of the pure compound with <i>Ec</i>DsbA (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S4</a>). The resulting electron density map shows partial electron density in a region of the structure that is consistent with a potentially plausible binding position for the amide extension of oxadiazole <b>11</b>. This additional density is only partial, and it does not provide clear evidence of the binding mode for the oxadiazole extension in the complex. However, had this density been observed in the initial screen, it would have led to <b>11</b> being identified as a possible hit. Oxadiazole <b>11</b> was not identified as a hit by REFiL<sub>X</sub> despite its improved affinity compared to parent <b>2</b>, which may have been due to its low purity in the library synthesis. This variability in conversion to product represents one of the potential limitations of screening unpurified reaction products by X-ray crystallography.</div><div class="NLM_p last">These limitations were further investigated via a series of soaking experiments containing both parent <b>2</b> and higher affinity hit pyrazole <b>9</b>, which were prepared from the purified compounds as mixtures at relative concentrations corresponding to conversions of 6, 13, and 28%. These samples were made in the absence or presence of a faux background amidation reaction mixture (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Density of the pyrazole extension was observed at all conversions in the absence of the background faux amidation reaction mixture; however, the electron density was observed to be less clear for the pyrazole expansion in the presence of faux background reactions and this was sensitive to conversion (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S5</a>). This suggests that X-ray screening can produce false negatives from crude reactions at low conversions (<28%) and is consistent with the outcome observed with oxadiazole <b>11</b>. Nonetheless, in the current data set, 39 samples in the library achieved conversion/purity of ≥28% by LCMS.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have implemented a fragment development approach that uses X-ray crystallography to screen chemical libraries synthesized in parallel on a microscale without extensive purification to identify compounds that bind to a protein target. Using this approach, termed REFiL<sub>X</sub>, we synthesized a library of 93 amides, which were each soaked into crystals of <i>Ec</i>DsbA. We observed additional electron density in the X-ray data for four products, which were deemed to represent “hits” in the screen (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S3</a>). The amide expansion with the highest affinity hit was pyrazole <b>9</b> with an affinity of <i>K</i><sub>D</sub> = 62 ± 9 μM. This approach is complementary to the strategy of ORS by SPR,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> with the distinction that REFiL<sub>X</sub> uses X-ray crystallography as the readout to identify higher affinity binders. An advantage of REFiL<sub>X</sub> is that it can operate over a wide affinity range. For example, although pyrazole <b>9</b> provided an 8-fold affinity gain over the parent compound, the dissociation kinetics for pyrazole <b>9</b> would be expected to be too fast to be measured by SPR. Current limitations of REFiL<sub>X</sub> include the availability of a suitably robust crystal system that is amenable to soaking and stable in the presence of organic cosolvents and the requirement for reasonable conversion in the parallel synthesis (≥∼28% in the current study). REFiL<sub>X</sub> can provide structural information on the most promising compounds in a library to guide further design of high-affinity ligands. We therefore expect that REFiL<sub>X</sub> will provide a useful tool in the armamentarium for early-stage fragment elaboration, particularly against challenging targets such as <i>Ec</i>DsbA, where the initial fragment hits bind with very low affinity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> General Chemistry</h3><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR spectra were acquired at 401 and 101 MHz, respectively, on a Bruker Avance III Nanobay spectrometer coupled to a BACS 60 automatic sample changer. Data were processed using MestReNova v6.0.2 software. <sup>1</sup>H and <sup>13</sup>C spectra were referenced with the residual proteo-solvent peak as follows: <sup>1</sup>H NMR shifts: 7.26 ppm CDCl<sub>3</sub>, 2.50 ppm (CD<sub>3</sub>)<sub>2</sub>SO, 3.31 ppm CD<sub>3</sub>OD, and 0.030 ppm for 4,4-dimethyl-4-silapentane-1-sulfonic acid (DSS) in D<sub>2</sub>O. <sup>13</sup>C NMR shifts: 77.16 ppm CDCl<sub>3</sub>, 39.52 ppm (CD<sub>3</sub>)<sub>2</sub>SO, and 49.00 CD<sub>3</sub>OD.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Overlapped carbon resonance frequencies were confirmed by two-dimensional (2D) <sup>1</sup>H–<sup>13</sup>C HSQC or heteronuclear multiple bond correlation (HMBC) experiments where possible.</div><div class="NLM_p">High-resolution mass spectrometry (HRMS) data were acquired on an Agilent 6224 time-of-flight (TOF) LC/MS mass spectrometer coupled to an Agilent 1290 Infinity coupled to a 2795 Alliance separations module. Data were mass-corrected via a dual-spray electrospray ionization (ESI) source with the following parameters: electrospray ionization, drying gas-flow: 11 L/min; nebulizer: 45 psi; drying gas temperature: 325 °C; capillary voltage (<i>V</i><sub>cap</sub>): 4000 V; fragmentor: 160 V; skimmer: 65 V; OCT RFV: 750 V; scan range acquired: 100–1500 <i>m</i>/<i>z</i>. Internal reference ions: positive ion mode = <i>m</i>/<i>z</i> = 121.050873 and 922.009798. Acquisition and analysis were performed using Agilent Mass Hunter Data Acquisition software vB.05.00 build 5.0.5042.2 and Mass Hunter Qualitative Analysis vB.05.00 build 5.0.519.13, respectively.</div><div class="NLM_p">Liquid chromatography–mass spectroscopy (LCMS) data were acquired using one of two methods, method 1 or method 2 as specified in individual experiments. Method 1 used a Shimadzu 2020 MS system (equipped with a NexeraX2 SIL-3AC autosampler, CTO-20A oven, NexeraX2 LC-30AD binary pump system, DGU-20A degasser, SPD-M30A photodiode array detector, CBM-20A communication BUS manager), and the data were processed using LabSolutions v.5.80 software. The system was equipped with a reverse-phase Merck C18 Chromolith FastGradient RP-18 endcapped (50 mm × 2.0 mm monolithic, 100 Å, 2 μm) column maintained at 35 °C. Standard gradient: 0–0.25 min isocratic 5% buffer B, 0.25–3.5 min gradient to 100% buffer B, 3.5–4 min isocratic 100% buffer B, 4.0–4.01 min gradient to 5% buffer B, 4.01–5.0 min isocratic 5% buffer B at 35 °C. Buffer A: 99.9% H<sub>2</sub>O, 0.1% AcOH; buffer B: 99.9% MeCN, 0.1% AcOH. Method 2 used an Agilent 6100 series single Quad LC/MS (equipped with a 1200 series G13111A quaternary pump, a G1329A thermostatted autosampler, and a G1314B variable wavelength detector), and the data were processed using LC/MSD Chemstation rev.B.04.01 SP1 coupled with easy access software. The system was equipped with a reverse-phase luna C8(2) (5 μm, 50 × 4.6 mm<sup>2</sup>, 100 Å) column maintained at 30 °C. All data were acquired and reference mass-corrected via a dual-spray electrospray ionization (ESI) source. Each scan or data point on the total ion chromatogram (TIC) is the average of 13 700 transients, producing one spectrum per second. Mass spectra were created by averaging the scans across each peak and background-subtracted against the first 10 s of the TIC. Data acquisition was carried out using Agilent Mass Hunter Data Acquisition Software version B.05.00 build 5.0.5042, and analysis was performed using Mass Hunter Qualitative Analysis version B.05.00 build 5.0.519.13. Elution in the LCMS standard method was performed on a gradient of 5% buffer B to 100% buffer B over 6 min at 0.5 mL/min. Buffer A = 99.9% H<sub>2</sub>O, 0.1% TFA. Buffer B = 99.9% MeCN, 0.1% TFA.</div><div class="NLM_p">The purity and retention time of final products were determined on an Agilent 1260 Infinity analytical reverse-phase high-performance liquid chromatography (HPLC) system fitted with an Agilent Zorbex SB-C18 rapid-resolution high-throughput 2.1 mm × 50 mm, 1.8 μm column using the specified gradient of buffer A: 99.9% H<sub>2</sub>O, 0.1% TFA; and buffer B: 99.9% MeCN, 0.1% TFA. Standard method 95% A and 5% B over 0.1 min, a gradient of 5–100% buffer B gradient over 9 min, and 100% buffer B over 2 min at a flow rate of 1 mL/min monitored at both 214 and 254 nm using a Waters 996 photodiode array detector. All screening compounds were >95% pure as determined by HPLC analysis.</div><div class="NLM_p last">Thin-layer chromatography (TLC) was carried out on silica gel 60 F<sub>254</sub> precoated plates (0.25 mm, Merck). TLC plates were visualized under UV illumination at 254 nm. Column chromatography was conducted using Carl Roth silica gel LC60 (40–63 μm, 230–400 mesh). Microscale derivatizations were performed on Corning 96-well round-bottom polypropylene plates, and solvents were removed using a Genevac EZ-2 elite personal evaporator at 80 °C. Batch-scale reaction solvents CDCl<sub>3</sub>, MeCN, tetrahydrofuran (THF), EtOH, dichloromethane (DCM), and DMF were stored over Alfa Aesar 4 Å molecular sieves and NMR solvents over Acros 3 Å, 8–12-mesh molecular sieves. Solvents were degassed by bubbling high-purity N<sub>2</sub> gas through the solvent for 5 min.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Procedure 1 for the Synthesis of Compounds <b>6</b>, <b>7</b>, and <b>9</b>–<b>11</b></h3><div class="NLM_p last">Oxalyl chloride (8–16 equiv) was dissolved in CDCl<sub>3</sub> (0.3 M) under N<sub>2</sub>. DMF (cat. two to three drops) was added at 0 °C, and the reaction mixture was stirred for 30 min. After this time, phenylacetic acid <b>2</b> (1.00 equiv) was added and the reaction stirred for a further 4–6 h at 0 °C. The reaction mixture was evaporated to dryness, and the residue was dissolved in dry DMF or CDCl<sub>3</sub> (0.3 M). Amine (1.2–2 equiv) and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) or Et<sub>3</sub>N (3–5 equiv) were added, and the reaction mixture was stirred at room temperature for a further 18–24 h. The reaction mixture was diluted in Et<sub>2</sub>O (50 mL), washed with 1 M HCl (aq, 2 × 50 mL), sat. NaHCO<sub>3</sub> (aq, 2 × 50 mL), and brine (3 × 30 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> 2-(4-(4-Cyano-3-methylphenoxy)phenyl)acetic Acid (<b>2</b>)</h3><div class="NLM_p last">Methyl 2-(4-(4-cyano-3-methylphenoxy)phenyl)acetate (4.56 g, 16.2 mmol) was dissolved in MeOH (50 mL) and 2 M NaOH (aq, 30 mL, 60.0 mmol) and stirred for 3 h at room temperature. After this time, the MeOH was removed under vacuum, and the aqueous layer was acidified with 2 M HCl (aq, 50 mL) and diluted in EtOAc (150 mL). The layers were separated, and the aqueous layer was further extracted with EtOAc (2 × 100 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness to give <b>2</b> as a white solid (3.89 g, 90%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 7.72 (d, <i>J</i> = 8.6 Hz, 1H), 7.39–7.30 (m, 2H), 7.09–7.03 (m, 2H), 7.01 (d, <i>J</i> = 2.4 Hz, 1H), 6.86 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 3.60 (s, 2H), 2.42 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.69, 160.98, 153.28, 144.37, 134.73, 131.86, 131.38, 119.98, 118.78, 117.97, 115.36, 105.75, 39.93, 20.03. LCMS (<i>m</i>/<i>z</i>): 265.9 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.31 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>16</sub>H<sub>13</sub>NO<sub>3</sub> requires 268.0968 [M + H]<sup>+</sup>; found 268.0965. HPLC: <i>t</i><sub>R</sub> = 5.74 min, >99%, standard method.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Methyl 2-(4-(4-Cyano-3-methylphenoxy)phenyl)acetate (<b>5</b>)</h3><div class="NLM_p last">4-Bromo-2-methylbenzonitrile (10.0 g, 51 mmol), methyl 2-(4-hydroxyphenyl)acetate (17.0 g, 102 mmol), Cs<sub>2</sub>CO<sub>3</sub> (33.2 g, 102 mmol), <i>N</i>,<i>N</i>-dimethylglycine hydrochloride (2.85 g, 20.4 mmol), and copper(I) iodide (1.94 g, 10.2 mmol) were sealed in a 250 mL round-bottom flask and vacuum-purged with N<sub>2</sub> (three times). Degassed dry DMF (60 mL) was added, and the reaction mixture was heated at 100 °C for 18 h. The reaction was then cooled to room temperature and diluted with H<sub>2</sub>O (200 mL) and Et<sub>2</sub>O (300 mL). The layers were separated, and the aqueous layer was further extracted with Et<sub>2</sub>O (2 × 100 mL). The combined organic layers were washed with H<sub>2</sub>O (2 × 200 mL) and brine (2 × 125 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash column chromatography using a gradient of 100:0 toluene/acetone to 90:10 toluene/acetone to give methyl 2-(4-(4-cyano-3-methylphenoxy)phenyl)acetate as a colorless oil (7.74 g, 54%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.73 (d, <i>J</i> = 8.6 Hz, 1H), 7.39–7.31 (m, 2H), 7.11–7.03 (m, 2H), 7.02 (d, <i>J</i> = 2.2 Hz, 1H), 6.87 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 3.71 (s, 2H), 3.63 (s, 3H), 2.43 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.51, 160.81, 153.41, 144.31, 134.68, 131.29, 131.06, 119.96, 118.79, 117.87, 115.37, 105.74, 51.69, 39.26, 19.94. LCMS (<i>m</i>/<i>z</i>): 281.9 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.59 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>17</sub>H<sub>15</sub>NO<sub>3</sub> requires 282.1125 [M + H]<sup>+</sup>; found 282.1124. HPLC: <i>t</i><sub>R</sub> = 6.74 min, >99%, standard method.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> 2-Methyl-4-(4-(2-(4-methyl-3-oxopiperazin-1-yl)-2-oxoethyl)phenoxy)benzonitrile (<b>6</b>)</h3><div class="NLM_p last">It was prepared according to general procedure 1 with oxalyl chloride (2.00 mL, 7.80 equiv, 23.3 mmol), dry CDCl<sub>3</sub> (12 mL), phenylacetic acid <b>2</b> (800 mg, 1.00 equiv, 3.00 mmol), 1-methylpiperazin-2-one (513 mg, 1.5 equiv, 4.49 mmol), and DIPEA (2.00 mL, 3.75 equiv, 11.2 mmol). The reaction mixture was diluted in Et<sub>2</sub>O (150 mL), washed with 1 M HCl (aq, 2 × 150 mL), sat. NaHCO<sub>3</sub> (aq, 2 × 150 mL), and brine (3 × 50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash column chromatography using a gradient of 10:90:0 pet spirits/EtOAc/MeOH to 0:90:10 pet spirits/EtOAc/MeOH to give the product as an orange oil (57 mg, 5.2%). <sup>1</sup>H NMR (401 MHz, CDCl<sub>3</sub>) δ 7.51 (d, <i>J</i> = 8.5 Hz, 1H), 7.29–7.22 (m, 2H), 7.02–6.96 (m, 2H), 6.85 (d, <i>J</i> = 2.1 Hz, 1H), 6.80 (dd, <i>J</i> = 8.5, 1.9 Hz, 1H), 4.25 (s, 0.6H), 4.14 (s, 1.4H), 3.86 (app t, <i>J</i> = 5.4 Hz, 1.4H), 3.77–3.67 (m, 2.6H), 3.35 (app t, <i>J</i> = 5.4 Hz, 1.4H), 3.30–3.23 (m, 0.6H), 2.98 (m, 3.00–2.96, 3H), 2.47 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.38, 168.95, 165.48, 164.36, 161.31, 161.22, 154.15, 154.13, 144.55, 134.41, 130.96, 130.76, 130.69, 130.64, 120.68, 119.01, 118.98, 118.24, 118.19, 115.50, 115.48, 106.70, 106.59, 49.37, 48.31, 47.89, 46.41, 42.95, 39.91, 39.86, 38.90, 34.61, 34.40, 20.68. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were consistent with rotamers in a 2:1 ratio. LCMS method 2 (<i>m</i>/<i>z</i>): 364.0 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.24 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>21</sub>H<sub>21</sub>N<sub>3</sub>O<sub>3</sub> requires 364.1656 [M + H]<sup>+</sup>; found 364.1666. HPLC: <i>t</i><sub>R</sub> = 5.72 min, >99%, standard method.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> 4-(2-(4-(4-Cyano-3-methylphenoxy)phenyl)acetyl)morpholine-3-carboxylic Acid (<b>7</b>)</h3><div class="NLM_p last">It was prepared and worked up according to general procedure 1 with oxalyl chloride (1.00 mL, 15.6 equiv, 11.7 mmol), dry CDCl<sub>3</sub> (2.5 mL), phenylacetic acid <b>2</b> (200 mg, 1.00 equiv, 0.75 mmol), dry DMF (3 mL), morpholine-3-carboxylic acid (118 mg, 1.2 equiv, 0.90 mmol), and DIPEA (666 μL, 5 equiv, 3.75 mmol). The residue was purified by flash column chromatography using a gradient of 75:25:0 pet spirits/EtOAc/AcOH to 74.5:25:0.5 pet spirits/EtOAc/AcOH to give the product as a colorless oil (10 mg, 3.51%). <sup>1</sup>H NMR (401 MHz, CD<sub>3</sub>OD) δ 7.50 (d, <i>J</i> = 8.6 Hz, 1H), 7.28–7.22 (m, 1.4H), 7.22–7.17 (m, 0.6H), 6.97–6.89 (m, 2H), 6.85 (d, <i>J</i> = 2.0 Hz, 1H), 6.77 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 4.86 (s, 0.7H), 4.57 (s, 0.3H), 4.32 (s, 0.4H), 4.29 (s, 0.6H), 4.10 (d, <i>J</i> = 12.7 Hz, 0.4H), 3.81 (dd, <i>J</i> = 11.6, 3.4 Hz, 0.4H), 3.76 (s, 1.5H), 3.75–3.64 (m, 1.5H), 3.62–3.50 (m, 1.4H), 3.48–3.30 (m, 1.8H), 2.36 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 173.93, 163.19, 163.16, 155.26, 155.19, 145.77, 135.57, 133.04, 132.97, 132.30, 132.14, 121.54, 119.86, 119.84, 118.96, 118.94, 116.46, 116.44, 107.23, 107.19, 68.80, 68.67, 67.62, 67.36 (two peaks confirmed by HSQC), 67.29, 44.84, 40.83, 40.10, 39.83, 20.51. <sup>1</sup>H NMR and<sup>13</sup>C NMR spectra were consistent with rotamers in a 2:1 ratio. LCMS method 2 (<i>m</i>/<i>z</i>): 381.0 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.82 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>21</sub>H<sub>20</sub>N<sub>2</sub>O<sub>5</sub> requires 381.1445 [M + H]<sup>+</sup>; found 381.1458. HPLC: <i>t</i><sub>R</sub> = 5.69 min,>99%, standard method.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> 2-Methyl-4-(4-(2-morpholino-2-oxoethyl)phenoxy)benzonitrile (<b>8</b>)</h3><div class="NLM_p last">To a solution of phenylacetic acid <b>2</b> (100 mg, 1.00 equiv, 0.37 mmol) and HATU (213 mg, 1.5 equiv, 0.56 mmol) dissolved in dry 1:1 MeCN/DMF (4 mL) under N<sub>2</sub> was added Et<sub>3</sub>N (0.27 mL, 5.3 equiv, 1.98 mmol). The reaction mixture was stirred for 15 min at room temperature. After this time, morpholine (65 μL, 2.0 equiv, 0.75 mmol) was added and the reaction stirred for a further 24 h at room temperature. The reaction was then diluted in Et<sub>2</sub>O (40 mL) and H<sub>2</sub>O (50 mL). The layers were separated, and the organic layer was washed with 0.1 M HCl (aq, 50 mL) and brine (2 × 50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash column chromatography using a gradient of 25:75 pet spirits/EtOAc to 0:100 pet spirits/EtOAc to give the product as a white solid (38 mg, 30%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.76 (d, <i>J</i> = 8.6 Hz, 1H), 7.33–7.27 (m, 2H), 7.09–7.04 (m, 2H), 7.02 (d, <i>J</i> = 2.4 Hz, 1H), 6.87 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 3.74 (s, 2H), 3.57–3.50 (m, 6H), 3.48–3.44 (m, 2H), 2.44 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.50, 161.34, 153.95, 144.54, 134.41, 131.56, 130.63, 120.62, 118.95, 118.22, 115.44, 106.58, 66.88, 66.56, 46.54, 42.28, 39.76, 20.69. LCMS method 1 (<i>m</i>/<i>z</i>): 337.15 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.11 min. HRMS (<i>m</i>/<i>z</i>): C<sub>20</sub>H<sub>20</sub>N<sub>2</sub>O<sub>3</sub> requires 337.1547 [M + H]<sup>+</sup>; found 337.1555. HPLC: <i>t</i><sub>R</sub> = 5.93 min, >99%, standard method.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <i>N</i>-(2-(1<i>H</i>-Pyrazol-1-yl)ethyl)-2-(4-(4-cyano-3-methylphenoxy)phenyl)-<i>N</i>-ethylacetamide (<b>9</b>)</h3><div class="NLM_p last">It was prepared and worked up according to general procedure 1 with oxalyl chloride (1.00 mL, 15.6 equiv, 11.7 mmol), dry CDCl<sub>3</sub> (2.5 mL), phenylacetic acid <b>2</b> (200 mg, 1.00 equiv, 0.75 mmol), dry DMF (3 mL), <i>N</i>-ethyl-2-pyrazol-1-yl-ethanamine (125 mg, 1.2 equiv, 0.90 mmol), and DIPEA (400 μL, 3.0 equiv, 2.24 mmol). The residue was purified by flash column chromatography using 75:25 pet spirits/EtOAc to give the product as an orange oil (74 mg, 25%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.77 (d, <i>J</i> = 6.7 Hz, 0.5H), 7.75 (d, <i>J</i> = 6.7 Hz, 0.5H), 7.73 (d, <i>J</i> = 1.7 Hz, 0.5H), 7.60 (d, <i>J</i> = 1.7 Hz, 0.5H), 7.54 (d, <i>J</i> = 1.2 Hz, 0.5H), 7.45 (d, <i>J</i> = 1.2 Hz, 0.5H), 7.35–7.29 (m, 1H), 7.15–7.10 (m, 1H), 7.10–7.06 (m, 1H), 7.06–7.00 (m, 2H), 6.90–6.84 (m, 1H), 6.30 (t, <i>J</i> = 2.0 Hz, 0.5H), 6.20 (t, <i>J</i> = 2.0 Hz, 0.5H), 4.32 (t, <i>J</i> = 5.8 Hz, 1H), 4.25 (t, <i>J</i> = 6.2 Hz, 1H), 3.74–3.68 (m, 2H), 3.61 (t, <i>J</i> = 6.1 Hz, 1H), 3.24 (q, <i>J</i> = 7.2 Hz, 1H), 3.20 (s, 1H), 3.06 (q, <i>J</i> = 7.2 Hz, 1H), 2.45 (s, 1.5H), 2.44 (s, 1.5H), 1.01 (t, <i>J</i> = 7.1 Hz, 1.5H), 0.91 (t, <i>J</i> = 7.1 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 173.54, 173.32, 163.12, 163.12, 155.26, 155.16, 145.75, 145.73, 141.15, 140.64, 135.57, 135.54, 133.44, 133.41, 132.47, 132.10, 132.06, 131.96, 121.61, 121.47, 119.82, 119.81, 118.94, 118.93, 116.44, 116.43, 107.26, 107.20, 107.16, 106.73, 50.97, 50.35, 48.83, 47.58, 44.89, 41.64, 40.33, 39.46, 20.54, 20.51, 13.88, 12.66. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were consistent with rotamers in 1:1 and 3:2 ratios, respectively. LCMS method 2 (<i>m</i>/<i>z</i>): 389.0 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.79 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>23</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub> requires 411.1791 [M + Na]<sup>+</sup>; found 411.1793. HPLC: <i>t</i><sub>R</sub> = 6.35 min, >99%, standard method.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 4-(4-(2-(4-Acetylpiperazin-1-yl)-2-oxoethyl)phenoxy)-2-methylbenzonitrile (<b>10</b>)</h3><div class="NLM_p last">It was prepared according to general procedure 1 with oxalyl chloride (1.0 mL, 7.8 equiv, 11.6 mmol), dry CDCl<sub>3</sub> (5 mL), phenylacetic acid <b>2</b> (400 mg, 1.0 equiv, 1.50 mmol), dry CDCl<sub>3</sub> (5 mL), 1-piperazin-1-ylethanone (384 mg, 2.0 equiv, 3.00 mmol), and Et<sub>3</sub>N (626 μL, 3 equiv, 4.49 mmol). The reaction mixture was diluted in CHCl<sub>3</sub> (50 mL), washed with H<sub>2</sub>O (2 × 50 mL), 1 M HCl (aq, 2 × 50 mL), and brine (30 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash column chromatography using 90:10 EtOAc/MeOH to give the product as a white solid (359 mg, 64%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.76 (d, <i>J</i> = 8.6 Hz, 1H), 7.34–7.29 (m, 2H), 7.11–7.05 (m, 2H), 7.02 (s, 1H), 6.88 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 3.78 (d, <i>J</i> = 4.1 Hz, 2H), 3.57–3.47 (m, 3H), 3.47–3.37 (m, 5H), 2.44 (s, 3H), 2.02 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.74, 169.45, 169.42, 169.09, 161.29, 154.14, 144.61, 134.46, 131.32, 130.65, 130.58, 120.67, 119.08, 118.24, 115.53, 106.71, 46.17, 46.09, 45.99, 45.83, 41.92, 41.70, 41.34, 41.21, 40.20, 40.02, 21.47, 21.45, 20.74. LCMS method 2 (<i>m</i>/<i>z</i>): 378.0 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.32 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>22</sub>H<sub>23</sub>N<sub>3</sub>O<sub>3</sub> requires 378.1812 [M + H]<sup>+</sup>; found 378.1820. HPLC: <i>t</i><sub>R</sub> = 5.80 min, >99%, standard method.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 2-(4-(4-Cyano-3-methylphenoxy)phenyl)-<i>N</i>-methyl-<i>N</i>-(2-(5-methyl-1,2,4-oxadiazol-3-yl)ethyl)acetamide (<b>11</b>)</h3><div class="NLM_p last">It was prepared and worked up according to general procedure 1 with oxalyl chloride (1.0 mL, 7.8 equiv, 11.6 mmol), dry CDCl<sub>3</sub> (2.5 mL), phenylacetic acid <b>2</b> (200 mg, 1.0 equiv, 0.75 mmol), <i>N</i>-methyl-2-(5-methyl-1,2,4-oxadiazol-3-yl)ethanamine hydrochloride (160 mg, 1.2 equiv, 0.90 mmol), and Et<sub>3</sub>N (521 μL, 5 equiv, 3.74 mmol). The residue was purified by flash column chromatography using a gradient of 25:75 pet spirits/EtOAc to 0:100 pet spirits/EtOAc to give the product as a colorless oil (50 mg, 17%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.75 (app dd, <i>J</i> = 8.6, 2.1 Hz, 1H), 7.29–7.22 (m, 2H), 7.08–7.03 (m, 2H), 7.02 (app t, <i>J</i> = 2.7 Hz, 1H), 6.88 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 3.75 (t, <i>J</i> = 7.2 Hz, 0.8H), 3.70 (app d, <i>J</i> = 3.7 Hz, 2H), 3.62 (t, <i>J</i> = 7.2 Hz, 1.2H), 3.01 (s, 1.8H), 2.93 (t, <i>J</i> = 7.1 Hz, 0.8H), 2.86 (t, <i>J</i> = 7.2 Hz, 1.2H), 2.83 (s, 1.2H), 2.57 (s, 1.2H), 2.54 (s, 1.8H), 2.44 (app d, <i>J</i> = 2.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 176.92, 176.48, 170.93, 170.81, 168.54, 167.60, 161.47, 161.46, 153.85, 153.80, 144.52, 134.39, 131.88, 131.66, 130.82, 130.81, 120.60, 120.53, 118.88, 118.87, 118.27, 115.41, 115.41 (confirmed via HMBC), 106.47, 47.46, 46.12, 40.21, 39.68, 36.58, 33.34, 25.33, 24.43, 20.68, 12.39, 12.37. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were consistent with rotamers in a 3:2 ratio. LCMS method 2 (<i>m</i>/<i>z</i>): 391.0 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.41 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>22</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub> requires 391.1765 [M + H]<sup>+</sup>; found 391.1774. HPLC: <i>t</i><sub>R</sub> = 6.12 min, >99%, standard method.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2-(4-(4-Cyano-3-methylphenoxy)phenyl)-<i>N</i>-(2-(4-methyl-1,4-diazepan-1-yl)ethyl)acetamide (<b>12</b>)</h3><div class="NLM_p last">Oxalyl chloride (1.00 mL, 7.8 equiv, 11.7 mmol) was dissolved in dry CDCl<sub>3</sub> (5 mL). DMF (cat. two to three drops) was added at 0 °C, and the reaction mixture was stirred for 30 min under N<sub>2</sub>. After this time, phenylacetic acid <b>2</b> (200 mg, 1.00 equiv, 0.94 mmol) was added, and the reaction was stirred for a further 4 h at 0 °C. 2-(4-Methyl-1,4-diazepan-1-yl)ethanamine (282 mg, 1.2 equiv, 1.8 mmol) and Et<sub>3</sub>N (1000 μL, 4.8 equiv, 7.2 mmol) were added, and the reaction was stirred at room temperature for a further 24 h. After this time, the reaction mixture was diluted in CHCl<sub>3</sub> (40 mL) and washed with sat. NaHCO<sub>3</sub> (aq, 2 × 20 mL). The layers were separated, and the organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash column chromatography using a gradient of 100:0 EtOAc/MeOH to 0:100 EtOAc/MeOH. The silica was resuspended in 2:98 Et<sub>3</sub>N/MeOH, filtered, and evaporated to dryness to give the product as a colorless oil (65 mg, 11%). <sup>1</sup>H NMR (401 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.92 (t, <i>J</i> = 5.3 Hz, 1H), 7.75 (d, <i>J</i> = 8.6 Hz, 1H), 7.38–7.31 (m, 2H), 7.09–7.04 (m, 2H), 7.02 (d, <i>J</i> = 2.2 Hz, 1H), 6.88 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 3.44 (s, 2H), 3.13 (dd, <i>J</i> = 12.5, 6.5 Hz, 2H), 2.68–2.59 (m, 4H), 2.55–2.46 (m, 6H), 2.44 (s, 3H), 2.23 (s, 3H), 1.67 (dt, <i>J</i> = 11.8, 6.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.82, 161.02, 152.97, 144.30, 134.70, 133.31, 130.87, 119.95, 118.64, 117.91, 115.28, 105.59, 57.41, 56.59, 56.17, 53.96, 53.67, 46.46, 41.64, 37.03, 26.86, 19.98. LCMS method 2 (<i>m</i>/<i>z</i>): 407.1 [M + H]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.25 min, standard method. HRMS (<i>m</i>/<i>z</i>): C<sub>24</sub>H<sub>30</sub>N<sub>4</sub>O<sub>2</sub> requires 407.2442 [M + H]<sup>+</sup>; found 407.2454. HPLC: <i>t</i><sub>R</sub> = 4.726 min, >99%, standard method.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Optimization of Microscale Reaction Conditions</h3><div class="NLM_p last">Forty-five microscale amidation reactions were conducted to optimize the reaction solvent, coupling reagent, temperature, and amine equivalents. In a polypropylene 96-well plate, phenylacetic acid <b>2</b> (0.2 M in 15% v/v dry Et<sub>3</sub>N/solvent, 10 μL, 1.0 equiv, 2 μmol) and coupling reagent (1 M in dry DMF for HATU and COMU and 1 M in H<sub>2</sub>O for EDC, 4 μL, 2.0 equiv, 4 μmol) were added to each well. The plate was sealed and left at room temperature for 15 min without agitation. After this time, a 0.5 M solution of 4-morpholinobutan-1-amine, 4-phenoxypiperidine, or ethyl 2-amino-1<i>H</i>-imidazole-5-carboxylate (1–3 equiv) was added in dry DMF, MeCN, or THF, and the plate was left at room temperature for a further 18 h without agitation. The solvent was removed under vacuum, and 20 μL of DMSO was added to each well. A 1 μL aliquot was taken from each well and diluted in MeCN (49 μL) and analyzed by LCMS (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S2</a>).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Microscale Reaction Plate Setup</h3><div class="NLM_p last">In a 96-well plate, phenylacetic acid <b>2</b> dissolved in 15% v/v dry Et<sub>3</sub>N/MeCN (0.2 M, 10 μL 1 equiv, 2 μmol) and HATU in dry DMF (1 M, 4 μL, 2.0 equiv, 4 μmol) were added to each well. The plate was sealed and left at room temperature for 15 min without agitation. After this time, a single amine reagent from the library dissolved in DMF (0.5 M, 6 μL, 1.5 equiv, 3 μmol) was added to each well (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S1</a>), and the plate was left at room temperature for a further 18 h without agitation. For wells containing amine reagents that required BOC deprotection or ester hydrolysis, this was achieved using 1:1 DCM/TFA (20 μL) or THF (10 μL) and NaOH (aq, 0.6 M, 10 μL, 3.0 equiv, 6 μmol), respectively. For the deprotection reactions, the plate was covered and left at room temperature for a further 3 h without agitation. The solvent was then removed under vacuum, and 20 μL of DMSO was added to each well, which would generate a stock solution at a concentration of 100 mM if 100% conversion to product were achieved in the reaction. A 1 μL aliquot was taken from each well, dispensed into a new 96-well plate, diluted in MeCN (49 μL) to a notional concentration of 2 mM, and analyzed by LCMS using method 1. The remaining DMSO samples were stored at −78 °C until crystallographic soaking.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Faux Background Reaction Mixture</h3><div class="NLM_p last">Isonicotinic acid in 15% v/v dry Et<sub>3</sub>N/MeCN (0.2 M, 125 μL 1 equiv, 25 μmol) and a solution of HATU in dry DMF (1 M, 50 μL, 2.0 equiv, 50 μmol) were combined, and the reaction was left at room temperature for 15 min without agitation. After this time, a solution of <i>N</i>-methylpiperazine in DMF (0.5 M, 75 μL, 1.5 equiv, 38 μmol) was added, and the reaction was left at room temperature for a further 18 h without agitation. Aliquots of this background reaction mixture were then used without further purification for soaking experiments (see below).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Protein Expression and Purification</h3><div class="NLM_p">Unlabeled and uniformly <sup>15</sup>N-labeled <i>Ec</i>DsbA were expressed using autoinduction as described previously.<a onclick="showRef(event, 'ref9 ref23'); return false;" href="javascript:void(0);" class="ref ref9 ref23">(9,23)</a> All media components were sterilized prior to use by autoclave or 0.22 μm filtration. Pre-expression cultures were prepared by inoculating 10 mL of Luria–Bertani broth supplemented with 50 μg/mL kanamycin from frozen glycerol stocks of <i>E. coli</i> BL21(DE3) carrying the plasmid B0013-(5644bb) coding for <i>Ec</i>DsbA, as described previously,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and incubating for 16 h at 37 °C with agitation at 220 rpm. ZYM-5052 or N-5052 media supplemented with 50 μg/mL kanamycin in baffled conical flasks were inoculated with 1% v/v preculture and incubated for 24–30 h at 37 °C with agitation at 170 rpm, after which the cells were harvested by centrifugation at 3200<i>g</i> at 4 °C for 20 min. The supernatant was discarded, and the pellets were stored at −20 °C for subsequent protein extraction and purification.</div><div class="NLM_p">Each frozen bacterial pellet was thawed, resuspended in an equal volume of osmotic shock buffer (20 mM Tris (pH 9.0), 10 mM ethylenediaminetetraacetic acid (EDTA), 50% w/v sucrose), and maintained at 4 °C with gentle stirring for 60 min, after which the cell suspension was rapidly diluted to 10 times its volume with deionized H<sub>2</sub>O at 4 °C. The suspension was maintained at 4 °C and stirred for a further 90 min before the cell pellet and periplasmic extract were separated by centrifugation at 50 000<i>g</i> at 4 °C for 30 min. The lysate was carefully decanted from the pellet and stored at −20 °C. The cell pellet was resuspended in 10 times its volume of lysis buffer (20 mM Tris (pH 8.0), 25 mM NaCl, 4 mg/mL colistin sulfate) and maintained at room temperature with gentle stirring for 18–24 h. This suspension was then centrifuged at 50 000<i>g</i> at 4 °C for 30 min, and the lysate was decanted and added to the thawed osmotic shock lysate. The pellet was discarded, and the target protein was purified from the combined lysate. (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> was added to the combined lysate with gentle stirring to a concentration of 0.8 M, and the solution was syringe-filtered (0.22 μm, Millipore) for loading onto a HiLoad 1610 Phenyl Sepharose HP hydrophobic interaction chromatography column (GE Healthcare) using 20 mM Tris (pH 8.0), 50 mM NaCl, and 1 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>. The bound proteins were eluted on a gradient from 1 to 0 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>. Fractions were analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), and those containing proteins running at a mass consistent with <i>Ec</i>DsbA were pooled and concentrated to a volume of 10 mL using an Amicon centrifugal diafiltration unit (10 000 molecular weight cutoff (MWCO), Millipore). This sample was buffer-exchanged to 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES) (pH 6.8) and 50 mM NaCl using a HiPrep 2610 desalting column (GE Healthcare). Following the buffer exchange, the fractions were analyzed by SDS-PAGE as above, and those containing <i>Ec</i>DsbA were pooled and loaded onto a MonoQ HR 10/10 anion-exchange column (GE Healthcare). In these buffer conditions, purified <i>Ec</i>DsbA was collected in the flow-through fraction. Protein purity was confirmed by SDS-PAGE.</div><div class="NLM_p">Following the anion-exchange chromatography step, the <i>Ec</i>DsbA solution was concentrated to a volume of approximately 8–9 mL (Amicon 3000 MWCO, Millipore) and treated with 0.1 volume of freshly prepared 15 mM copper–phenanthroline solution to oxidize the active site disulfide. After 1 h at 4 °C, copper–phenanthroline was removed from the protein solution by buffer exchange (50 mM HEPES pH 6.8, 50 mM NaCl) using a HiPrep 2610 desalting column.</div><div class="NLM_p last">Protein samples were adjusted to their required final concentrations using Amicon centrifugal diafiltration units (3000 MWCO, Millipore) or dilution using a storage/NMR buffer (50 mM HEPES (pH 6.8), 50 mM NaCl). For NMR experiments, the buffer was exchanged to 50 mM sodium phosphate, 25 mM NaCl, pH 6.8, and D<sub>2</sub>O was added to a final concentration of 10% (v/v); see below. Final protein concentrations were quantified using a NanoVue UV spectrophotometer (GE Healthcare) based on a calculated extinction coefficient for oxidized <i>Ec</i>DsbA of ε = 28545 M<sup>–1</sup> cm<sup>–1</sup>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Crystallization of <i>apo Ec</i>DsbA</h3><div class="NLM_p last">The previously published crystallization conditions were used as a guide to obtain <i>apo-</i>EcDsbA crystals.<a onclick="showRef(event, 'ref9 ref25'); return false;" href="javascript:void(0);" class="ref ref9 ref25">(9,25)</a> Protein was crystallized using hanging drop vapor diffusion. Then, 1 μL of 30 mg/mL <i>Ec</i>DsbA was mixed with an equal volume of crystallization buffer (11–13% poly(ethylene glycol) (PEG) 8000, 5–7.5% glycerol, 1 mM CuCl<sub>2</sub>, 100 mM sodium cacodylate, pH 6.1) and equilibrated against 0.5 mL of reservoir buffer at 18 °C. Crystals appeared within one to three days and grew to average dimensions of 0.6 mm × 0.4 mm × 0.2 mm.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Soaking of <i>Ec</i>DsbA Crystals with Reagent Mixtures</h3><div class="NLM_p">Reagent mixtures for crystal soaking trials were prepared for three coupling reagents: EDC (200 mM in Milli-Q H<sub>2</sub>O), COMU (200 mM in DMF), and HATU (200 mM in DMF). <i>Ec</i>DsbA crystals were soaked with each coupling agent at a final concentration of 20 mM for 3 h and then flash-cooled in liquid nitrogen and stored under liquid nitrogen in AS-type pucks in a Taylor Wharton CX100 Dewar, which had been precooled with liquid nitrogen.</div><div class="NLM_p last">The final soaking protocol for evaluating binding of the crude reaction product library involved transferring <i>Ec</i>DsbA crystals into 2 μL drops, which had been dispensed onto coverslips and comprised 24% PEG 8000, 22% glycerol, and 100 mM sodium cacodylate at pH 6.1 plus the crude reaction product in 10% v/v DMSO. The nominal concentration of the crude reaction products in the drop was 10 mM, which was calculated assuming 100% conversion in the reaction. Subsequently, coverslips were placed over the wells of crystallization plates containing 500 μL of the reservoir buffer and sealed. Crystals were left in this solution for 2–3 h and then flash-cooled in liquid nitrogen stored under liquid nitrogen in AS-type pucks in a Taylor Wharton CX100 Dewar precooled with liquid nitrogen prior to measuring diffraction data.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Estimating the Minimum Reactant-to-Product Conversion Required to Observe Electron Density in the Crystallographic Data</h3><div class="NLM_p last">A 10 μL aliquot of the faux background reaction mixture (see above) was dispensed into each well of a 96-well plate. Phenylacetic acid <b>2</b> and pyrazole <b>9</b> were added to each well as per <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S4</a>, and the solution was mixed. Crystals of <i>Ec</i>DsbA were soaked in each solution for 3 h as described above, looped out of the solution, flash-cooled in liquid nitrogen, and stored in loops under liquid nitrogen in AS-type pucks in a precooled Taylor Wharton CX100 Dewar.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cocrystallization and Cryoprotection of Purified Compound/<i>Ec</i>DsbA Complexes</h3><div class="NLM_p last">Purified resynthesized compounds were cocrystallized with <i>Ec</i>DsbA. For cocrystallization experiments, compounds were solubilized in DMSO to a final concentration of 100 mM. Prior to the crystallization experiment, 95 μL of 30 mg/mL <i>Ec</i>DsbA was mixed with 5 μL of the compound to give a final compound concentration of 5 mM in 5% v/v DMSO and incubated at 4 °C overnight. The conditions used to obtain the cocrystals (100–300 mM KBr, 25–35% PEG 2000 monomethyl ether (MME)) were based on previous reports.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Drops comprising 200/200 nL of protein–ligand solution/crystallization buffer were set up in sitting-drop vapor diffusion plates using an automated robotic facility in the CSIRO Collaborative Crystallization Centre (<a href="http://www.csiro.au/C3" class="extLink">www.csiro.au/C3</a>), Melbourne, Australia. Plates were incubated at 18 °C, and crystals appeared after 1–5 days. Prior to data collection, crystals were harvested with Hampton Research CryoLoops, briefly swiped through reservoir buffer, and flash-cooled in liquid nitrogen. Vitrified crystals were stored in AS-type pucks and shipped to the Australian synchrotron in a Taylor Wharton CX100 dry shipping dewar, which had been precooled with liquid nitrogen.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Automated Data Processing, Data Collection Statistics Analysis, and Refinement of Crystals Soaked with Reagent Mixtures</h3><div class="NLM_p">Library data were processed using the automated data-processing pipeline implemented at the MX1 beamline, where data were indexed, integrated, and scaled with xdsme and AIMLESS.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> The resulting data collection statistics for each data set were reviewed. For all data sets, the resolution cutoff used was based on the following criteria in the highest resolution range: CC<sub>1/2</sub> was at least 0.6, ⟨<i>I</i>/σ(<i>I</i>)⟩ was greater than 1.0, and completeness was greater than 90%.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In cases where data collection statistics did not fulfill these quality requirements or crystals diffracted to lower than 2.5 Å resolution, the crystallographic experiment was repeated. From the initial data set, nine crude products required repeat soaks. For the repeated soaks, the concentration of the crude reaction component was reduced initially to a nominal 5 mM and further to 2.5 mM if required. For compound <b>7</b>, crystals were observed to dissolve in the 2.5 mM solution, and in this case, the concentration of the crude component was reduced to a nominal 1 mM and the soaking time was cut down to 20 min. Overall 110 data sets were collected for the crude reaction products from the library. The data sets were phased by MR using chain A of the <i>apo</i>-<i>Ec</i>DsbA structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVK">1FVK</a>) as a search model.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> MR and automated refinement steps were carried out using the automated MR pipeline implemented in Auto-Rickshaw.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Briefly, MR was performed using the MOLREP<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> automated refinement protocol. This involved initial refinement in CNS<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> followed by refinement in REFMAC5.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In the CNS rigid-body refinement, <i>B</i>-factor (<i>B</i><sub>INDIVIDUAL</sub> refinement) and positional refinement were undertaken. The resulting model was used in REFMAC5 for maximum likelihood refinement including individual <i>B</i>-factor refinement. The final round of refinement was carried out in PHENIX<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> using rigid body, individual sites, individual sites in real space, and individual atomic displacement parameter (ADP) refinement with the addition of water molecules. TLS refinement was not used at this stage. The range of <i>R</i><sub>work</sub> and <i>R</i><sub>free</sub> values after automated refinement of the 90 library-soaked diffraction data sets is reported in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S2</a>. At this stage, the models were inspected manually using Coot.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In parallel, these models were used as input to PanDDA.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> To enable the PanDDA analysis, <i>apo</i>-<i>Ec</i>DsbA crystals were soaked with DMSO to generate 35 ground-state models. For each “event,” the PanDDA maps were inspected.</div><div class="NLM_p last">The structural models of four data sets that were identified as hits (compounds <b>6</b>–<b>9</b>) were manually refined as described below. The resulting molecular models were deposited to the protein databank (PDB). Statistics for the structures are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S5</a>. All automatically refined data sets and structure factors have been deposited to the PDB (accession codes are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). X-ray data collection and refinement statistics for the crystals soaked with the 90 unpurified reaction products from the library are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Figure S1</a>.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Manual Data Processing, Phasing, and Refinement</h3><div class="NLM_p last">The data sets collected for cocrystals of <i>Ec</i>DsbA in complex with purified compounds were processed with XDS<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> or IMOSFLM<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and AIMLESS.<a onclick="showRef(event, 'ref17 ref35'); return false;" href="javascript:void(0);" class="ref ref17 ref35">(17,35)</a> Phasing was performed by MR with Phaser<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> using chain A of <i>Ec</i>DsbA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVK">1FVK</a>)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> as a search model. Iterative cycles of manual model building and refinement were performed using Coot<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and phenix.refine.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> The <i>B</i>-factor model consisted of the isotropic <i>B</i>-factor for each non-hydrogen atom plus TLS refinement. TLS groups were chosen with the help of the TLSMD server.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Ligand restraints were generated in phenix.eLBOW and MOGUL.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> Model validation was carried out using MolProbity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Refinement statistics are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">Table S5</a>, prepared with the help of “Generate “<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>” for journal” utility within PHENIX.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Coordinates and structure factors have been deposited in the Protein Data Bank under the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD">6WHD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG1">6PG1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PC9">6PC9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PGJ">6PGJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PD7">6PD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG2">6PG2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PLI">6PLI</a>.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Assessing Solubility of Synthesized Analogues by qNMR</h3><div class="NLM_p last">Purified compounds were evaluated for their solubility and propensity to aggregate in aqueous solutions by recording a series of 1D <sup>1</sup>H qNMR experiments at increasing concentrations. Compounds were accurately weighed and dissolved in DMSO-<i>d</i><sub>6</sub> at a notional concentration of 100 mM. The NMR buffer for these experiments was prepared with 50 mM sodium phosphate and 25 mM NaCl at pH 7.4 in 100% D<sub>2</sub>O containing 100 μM 4,4-dimethyl-4-silapentane-1-sulfonic acid (DSS) as an internal standard. A serial dilution of the DMSO-<i>d</i><sub>6</sub> stock was performed to prepare a four-point twofold dilution series with final notional concentrations of 125, 250, 500,and 1000 μM containing 2% DMSO-<i>d</i><sub>6</sub> in the NMR buffer in a total volume of 600 μL. NMR spectra were recorded on a Bruker 400 MHz spectrometer with TD = 32 K, D1 = 5 s, 64 scans, and water suppression at 4.7 ppm. NMR spectra were processed in MNOVA and referenced to the DSS peak at 0.030 ppm. Compounds were deemed to have failed the solubility measure at a given concentration if they did not show a doubling in intensity concordant with doubling concentration or if any resonances in the spectrum showed chemical shift changes >0.004 ppm (1.6 Hz) from the lowest recorded concentration. Ligand concentrations were calculated relative to the DSS integral in the 1D <sup>1</sup>H qNMR spectra and used to calculate the correct stock concentration, and this corrected concentration was used in subsequent experiments for <i>K</i><sub>D</sub> determination.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Validation of Hits and Affinity Determination by <sup>1</sup>H–<sup>15</sup>N HSQC</h3><div class="NLM_p last">Hits from the REFiL<sub>X</sub> screen were further validated by measuring chemical shift perturbations (CSPs) in <sup>1</sup>H–<sup>15</sup>N HSQC NMR. A reference <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of uniformly labeled <sup>15</sup>N oxidized <i>Ec</i>DsbA (100 μM protein, 2% DMSO-<i>d</i><sub>6</sub>, 50 mM sodium phosphate, 25 mM NaCl, pH 6.8 in 10% D<sub>2</sub>O, 88% H<sub>2</sub>O) was acquired. A series of <sup>1</sup>H–<sup>15</sup>N HSQC spectra was acquired under the same conditions in the presence of a sample fragment, and the concentration of any fragment in the HSQC titration series was kept below the maximum solubility measured as described above. CSPs induced by fragment binding were calculated using the following equation:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_m001.gif" alt="" id="_i31" /></img></span>where Δ<i>H</i> and Δ<i>N</i> are the measured change in chemical shift of the backbone amide protons in the absence (<i>apo</i>) and presence of the fragment. Backbone amide assignments were taken from the biological magnetic resonance data bank with a BMRB code of 19838. All assigned peaks showing CSP > 0.040 ppm in the <sup>1</sup>H–<sup>15</sup>N HSQC spectrum recorded at the highest fragment concentration tested were used for affinity determination. CSPs below 0.023 ppm, peak assignments that were ambiguous due to overlap or merging through the titration, and residues with less than four data points were excluded. Equilibrium dissociation constants (<i>K</i><sub>D</sub>) were determined by fitting the measured CSP to a single-site binding model accounting for ligand depletion using the following equation where [P] is the total protein concentration and [L] is the total ligand concentration.<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_m002.gif" alt="" id="_i32" /></img></span></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00111" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00111?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00111</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full list of the structures in the synthesized library, details of microscale reaction optimizations, X-ray crystallography data collection, processing, and refinement statistics including control experiments, simulated annealing OMIT maps obtained, NMR solubility, binding analysis, and compound characterization data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">List of SMILES and the binding affinity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf">jm0c00111_si_001.pdf (5.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_002.csv">jm0c00111_si_002.csv (0.86 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are listed below for the atomic coordinates and experimental data for the cocrystal structures of <i>Ec</i>DsbA bound to <b>2</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD">6WHD</a>), <b>6</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG1">6PG1</a>), <b>6</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PC9">6PC9</a>), <b>7</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PGJ">6PGJ</a>), <b>7</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PD7">6PD7</a>), <b>8</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG2">6PG2</a>), <b>8</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PBI">6PBI</a>), <b>9</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PIQ">6PIQ</a>), <b>9</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>), and <b>11</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PLI">6PLI</a>).</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00111" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bradley C. Doak</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">ARC Training
Centre for Fragment Based Design, Monash
Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Monash
Fragment Platform, Monash Institute of Pharmaceutical
Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#afcdddcecbc3cad681cbc0cec4efc2c0c1cedcc781cacbda"><span class="__cf_email__" data-cfemail="d1b3a3b0b5bdb4a8ffb5beb0ba91bcbebfb0a2b9ffb4b5a4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin J. Scanlon</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">ARC Training
Centre for Fragment Based Design, Monash
Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Monash
Fragment Platform, Monash Institute of Pharmaceutical
Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9230-7506" title="Orcid link">http://orcid.org/0000-0002-9230-7506</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d4b9b5a6a0bdbafaa7b7b5bab8bbba94b9bbbab5a7bcfab1b0a1"><span class="__cf_email__" data-cfemail="dab7bba8aeb3b4f4a9b9bbb4b6b5b49ab7b5b4bba9b2f4bfbeaf">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew R. Bentley</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olga V. Ilyichova</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Geqing Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Genetics, La Trobe Institute for Molecular Science, La Trobe University, Melbourne, VIC 3086, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin L. Williams</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gaurav Sharma</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wesam S. Alwan</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca L. Whitehouse</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biswaranjan Mohanty</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">ARC Training
Centre for Fragment Based Design, Monash
Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Scammells</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2930-895X" title="Orcid link">http://orcid.org/0000-0003-2930-895X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Begoña Heras</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Genetics, La Trobe Institute for Molecular Science, La Trobe University, Melbourne, VIC 3086, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer L. Martin</span> - <span class="hlFld-Affiliation affiliation">Griffith
Institute for Drug Discovery, Building N75, Brisbane Innovation Park, Don Young Road, Nathan, QLD 4111, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Vice-Chancellor’s
Unit, University of Wollongong, Northfields Avenue, Wollongong, NSW 2522, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Makrina Totsika</span> - <span class="hlFld-Affiliation affiliation">School
of Biomedical Sciences, Institute of Health and Biomedical Innovation, Queensland University of Technology, Brisbane, QLD 4059, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ben Capuano</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">ARC Training
Centre for Fragment Based Design, Monash
Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, VIC 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5434-0180" title="Orcid link">http://orcid.org/0000-0001-5434-0180</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.R.B. and O.V.I. share first authorship.</p></li><li><span class="author-information-subsection-header">Funding</span><p>We thank Monash University and NHMRC (Project Grant ID GRT1099151) for funding. M.T. was supported by a Queensland University of Technology Vice-Chancellor’s Research Fellowship.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3096-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was undertaken in part using the facilities of the Monash Fragment Platform. X-ray crystallography was undertaken on the MX1 and MX2 beamlines at the Australian Synchrotron, part of ANSTO, and made use of the Australian Cancer Research Foundation (ACRF) detector. We acknowledge the CSIRO Collaborative Crystallisation Centre (<a href="http://www.csiro.au/C3" class="extLink">www.csiro.au/C3</a>).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">HSQC</td><td class="NLM_def"><p class="first last">heteronuclear single quantum correlation</p></td></tr><tr><td class="NLM_term">ORS</td><td class="NLM_def"><p class="first last">off-rate screening</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">DSB</td><td class="NLM_def"><p class="first last">disulfide bond</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug discovery</p></td></tr><tr><td class="NLM_term">REFiL<sub>X</sub></td><td class="NLM_def"><p class="first last">rapid elaboration of fragments into leads by X-ray crystallography</p></td></tr><tr><td class="NLM_term">MME</td><td class="NLM_def"><p class="first last">monomethyl ethers</p></td></tr><tr><td class="NLM_term">Ec</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term">DCC</td><td class="NLM_def"><p class="first last">dynamic combinatorial chemistry</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethyl amine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">COMU</td><td class="NLM_def"><p class="first last">(1-cyano-2-ethoxy-2-oxoethylidenaminooxy)dimethylamino-morpholino-carbenium hexafluorophosphate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0ljVPAihXf_HxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span> <span> </span><span class="NLM_article-title">Design principles for fragment libraries: maximizing the value of learnings from pharma fragment-based drug discovery (FBDD) programs for use in academia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8189</span>– <span class="NLM_lpage">8206</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00197</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00197" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFSit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8189-8206&author=G.+M.+Keser%C5%B1author=D.+A.+Erlansonauthor=G.+G.+Ferenczyauthor=M.+M.+Hannauthor=C.+W.+Murrayauthor=S.+D.+Pickett&title=Design+principles+for+fragment+libraries%3A+maximizing+the+value+of+learnings+from+pharma+fragment-based+drug+discovery+%28FBDD%29+programs+for+use+in+academia&doi=10.1021%2Facs.jmedchem.6b00197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Design Principles for Fragment Libraries: Maximizing the Value of Learnings from Pharma Fragment-Based Drug Discovery (FBDD) Programs for Use in Academia</span></div><div class="casAuthors">Keseru, Gyorgy M.; Erlanson, Daniel A.; Ferenczy, Gyorgy G.; Hann, Michael M.; Murray, Christopher W.; Pickett, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8189-8206</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is well suited for discovering both drug leads and chem. probes of protein function; it can cover broad swaths of chem. space and allows the use of creative chem.  FBDD is widely implemented for lead discovery in industry but is sometimes used less systematically in academia.  Design principles and implementation approaches for fragment libraries are continually evolving, and the lack of up-to-date guidance may prevent more effective application of FBDD in academia.  This Perspective explores many of the theor., practical, and strategic considerations that occur within FBDD programs, including the optimal size, complexity, physicochem. profile, and shape profile of fragments in FBDD libraries, as well as compd. storage, evaluation, and screening technologies.  This compilation of industry experience in FBDD will hopefully be useful for those pursuing FBDD in academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkrMUH3yVkGbVg90H21EOLACvtfcHk0lgEqNCTZtFrgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFSit7s%253D&md5=888fe2cf003c5564c2ec391cf54ea2ac</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00197%26sid%3Dliteratum%253Aachs%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26atitle%3DDesign%2520principles%2520for%2520fragment%2520libraries%253A%2520maximizing%2520the%2520value%2520of%2520learnings%2520from%2520pharma%2520fragment-based%2520drug%2520discovery%2520%2528FBDD%2529%2520programs%2520for%2520use%2520in%2520academia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8189%26epage%3D8206%26doi%3D10.1021%2Facs.jmedchem.6b00197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, P. A.</span></span> <span> </span><span class="NLM_article-title">Off-rate screening (ORS) by surface plasmon resonance. An efficient method to kinetically sample hit to lead chemical space from unpurified reaction products</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2845</span>– <span class="NLM_lpage">2850</span>, <span class="refDoi"> DOI: 10.1021/jm401848a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401848a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2845-2850&author=J.+B.+Murrayauthor=S.+D.+Roughleyauthor=N.+Matassovaauthor=P.+A.+Brough&title=Off-rate+screening+%28ORS%29+by+surface+plasmon+resonance.+An+efficient+method+to+kinetically+sample+hit+to+lead+chemical+space+from+unpurified+reaction+products&doi=10.1021%2Fjm401848a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Off-Rate Screening (ORS) By Surface Plasmon Resonance. An Efficient Method to Kinetically Sample Hit to Lead Chemical Space from Unpurified Reaction Products</span></div><div class="casAuthors">Murray, James B.; Roughley, Stephen D.; Matassova, Natalia; Brough, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2845-2850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dissocn. rate const. kd (off-rate) is the component of ligand-protein binding with the most significant potential to enhance compd. potency.  Here we provide theor. and empirical data to show that this parameter can be detd. accurately from unpurified reaction products contg. designed test compds.  This screening protocol is amenable to parallel chem., provides efficiencies of time and materials, and complements existing methodologies for the hit-to-lead phase in fragment-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounOUxQkuyZLVg90H21EOLACvtfcHk0lgEqNCTZtFrgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVWguro%253D&md5=d3744f00df767e349024641e876e8100</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm401848a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401848a%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DBrough%26aufirst%3DP.%2BA.%26atitle%3DOff-rate%2520screening%2520%2528ORS%2529%2520by%2520surface%2520plasmon%2520resonance.%2520An%2520efficient%2520method%2520to%2520kinetically%2520sample%2520hit%2520to%2520lead%2520chemical%2520space%2520from%2520unpurified%2520reaction%2520products%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2845%26epage%3D2850%26doi%3D10.1021%2Fjm401848a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devine, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granata, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O′Reilly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Detection of ligands from a dynamic combinatorial library by X-ray crystallography</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4617</span>– <span class="NLM_lpage">4620</span>, <span class="refDoi"> DOI: 10.1002/ange.200351951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fange.200351951" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2003&pages=4617-4620&author=M.+S.+Congreveauthor=D.+J.+Davisauthor=L.+Devineauthor=C.+Granataauthor=M.+O%E2%80%B2Reillyauthor=P.+G.+Wyattauthor=H.+Jhoti&title=Detection+of+ligands+from+a+dynamic+combinatorial+library+by+X-ray+crystallography&doi=10.1002%2Fange.200351951"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fange.200351951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.200351951%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%2BS.%26aulast%3DDavis%26aufirst%3DD.%2BJ.%26aulast%3DDevine%26aufirst%3DL.%26aulast%3DGranata%26aufirst%3DC.%26aulast%3DO%25E2%2580%25B2Reilly%26aufirst%3DM.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520ligands%2520from%2520a%2520dynamic%2520combinatorial%2520library%2520by%2520X-ray%2520crystallography%26jtitle%3DAngew.%2520Chem.%26date%3D2003%26volume%3D115%26spage%3D4617%26epage%3D4620%26doi%3D10.1002%2Fange.200351951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span> <span> </span><span class="NLM_article-title">The medicinal chemist’s toolbox: an analysis of reactions used in the pursuit of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3451</span>– <span class="NLM_lpage">3479</span>, <span class="refDoi"> DOI: 10.1021/jm200187y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200187y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3451-3479&author=S.+D.+Roughleyauthor=A.+M.+Jordan&title=The+medicinal+chemist%E2%80%99s+toolbox%3A+an+analysis+of+reactions+used+in+the+pursuit+of+drug+candidates&doi=10.1021%2Fjm200187y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemist's Toolbox: An Analysis of Reactions Used in the Pursuit of Drug Candidates</span></div><div class="casAuthors">Roughley, Stephen D.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3451-3479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A survey of selected medicinal chem. literature from 2008 was undertaken in order to det. the most common chem. reactions used in the prepn. of drug candidates.  In addn. to an examn. of the reactions, the frequency of occurrence of various functional groups in the drug targets was analyzed.  The Lipinski parameters and chirality of the surveyed compds. were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTUrcluTkPBLVg90H21EOLACvtfcHk0lgxl7S-5_fRVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D&md5=a94df26d82bbd58b269ed20aa3253f56</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm200187y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200187y%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520medicinal%2520chemist%25E2%2580%2599s%2520toolbox%253A%2520an%2520analysis%2520of%2520reactions%2520used%2520in%2520the%2520pursuit%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3451%26epage%3D3479%26doi%3D10.1021%2Fjm200187y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deane, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span> <span> </span><span class="NLM_article-title">A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">15123</span> <span class="refDoi"> DOI: 10.1038/ncomms15123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fncomms15123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=28436492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A280%3ADC%252BC1crgsFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=N.+M.+Pearceauthor=T.+Krojerauthor=A.+R.+Bradleyauthor=P.+Collinsauthor=R.+P.+Nowakauthor=R.+Talonauthor=B.+D.+Marsdenauthor=S.+Kelmauthor=J.+Shiauthor=C.+M.+Deaneauthor=F.+Von+Delft&title=A+multi-crystal+method+for+extracting+obscured+crystallographic+states+from+conventionally+uninterpretable+electron+density&doi=10.1038%2Fncomms15123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density</span></div><div class="casAuthors">Pearce Nicholas M; Krojer Tobias; Bradley Anthony R; Nowak Radoslaw P; Talon Romain; Marsden Brian D; von Delft Frank; Collins Patrick; von Delft Frank; Marsden Brian D; Kelm Sebastian; Shi Jiye; Deane Charlotte M; von Delft Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15123</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In macromolecular crystallography, the rigorous detection of changed states (for example, ligand binding) is difficult unless signal is strong.  Ambiguous ('weak' or 'noisy') density is experimentally common, since molecular states are generally only fractionally present in the crystal.  Existing methodologies focus on generating maximally accurate maps whereby minor states become discernible; in practice, such map interpretation is disappointingly subjective, time-consuming and methodologically unsound.  Here we report the PanDDA method, which automatically reveals clear electron density for the changed state-even from inaccurate maps-by subtracting a proportion of the confounding 'ground state'; changed states are objectively identified from statistical analysis of density distributions.  The method is completely general, implying new best practice for all changed-state studies, including the routine collection of multiple ground-state crystals.  More generally, these results demonstrate: the incompleteness of atomic models; that single data sets contain insufficient information to model them fully; and that accuracy requires further map-deconvolution approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZZnh02fVEczAyhZbj3lKdfW6udTcc2eZbA4mwUwavjrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crgsFWksA%253D%253D&md5=34019a4cb9081f77d9473fb0bf65a250</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncomms15123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15123%26sid%3Dliteratum%253Aachs%26aulast%3DPearce%26aufirst%3DN.%2BM.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DBradley%26aufirst%3DA.%2BR.%26aulast%3DCollins%26aufirst%3DP.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DTalon%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DKelm%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DDeane%26aufirst%3DC.%2BM.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26atitle%3DA%2520multi-crystal%2520method%2520for%2520extracting%2520obscured%2520crystallographic%2520states%2520from%2520conventionally%2520uninterpretable%2520electron%2520density%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fncomms15123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broennimann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinapoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eikenberry, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozzanica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlepütz, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmott, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, B.</span></span> <span> </span><span class="NLM_article-title">Performance of single-photon-counting PILATUS detector modules</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1107/S0909049509009911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0909049509009911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19395800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFCit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=368-375&author=P.+Kraftauthor=A.+Bergamaschiauthor=C.+Broennimannauthor=R.+Dinapoliauthor=E.+F.+Eikenberryauthor=B.+Henrichauthor=I.+Johnsonauthor=A.+Mozzanicaauthor=C.+M.+Schlep%C3%BCtzauthor=P.+R.+Willmottauthor=B.+Schmitt&title=Performance+of+single-photon-counting+PILATUS+detector+modules&doi=10.1107%2FS0909049509009911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Performance of single-photon-counting PILATUS detector modules</span></div><div class="casAuthors">Kraft, P.; Bergamaschi, A.; Broennimann, Ch.; Dinapoli, R.; Eikenberry, E. F.; Henrich, B.; Johnson, I.; Mozzanica, A.; Schlepuetz, C. M.; Willmott, P. R.; Schmitt, B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">368-375</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">PILATUS is a silicon hybrid pixel detector system, operating in single-photon-counting mode, that has been developed at the Paul Scherrer Institut for the needs of macromol. crystallog. at the Swiss Light Source (SLS).  A calibrated PILATUS module has been characterized with monochromatic synchrotron radiation.  The influence of charge sharing on the count rate and the overall energy resoln. of the detector were investigated.  The dead-time of the system was detd. using the attenuated direct synchrotron beam.  A single module detector was also tested in surface diffraction expts. at the SLS, whereby its performance regarding fluorescence suppression and satn. tolerance were evaluated, and have shown to greatly improve the sensitivity, reliability and speed of surface diffraction data acquisition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5mI0pKAfFi7Vg90H21EOLACvtfcHk0lgxl7S-5_fRVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFCit7w%253D&md5=256106f0fb728f37eb3702623e613d66</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1107%2FS0909049509009911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049509009911%26sid%3Dliteratum%253Aachs%26aulast%3DKraft%26aufirst%3DP.%26aulast%3DBergamaschi%26aufirst%3DA.%26aulast%3DBroennimann%26aufirst%3DC.%26aulast%3DDinapoli%26aufirst%3DR.%26aulast%3DEikenberry%26aufirst%3DE.%2BF.%26aulast%3DHenrich%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DMozzanica%26aufirst%3DA.%26aulast%3DSchlep%25C3%25BCtz%26aufirst%3DC.%2BM.%26aulast%3DWillmott%26aufirst%3DP.%2BR.%26aulast%3DSchmitt%26aufirst%3DB.%26atitle%3DPerformance%2520of%2520single-photon-counting%2520PILATUS%2520detector%2520modules%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2009%26volume%3D16%26spage%3D368%26epage%3D375%26doi%3D10.1107%2FS0909049509009911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinapoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horisberger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozzanica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theidel, G.</span></span> <span> </span><span class="NLM_article-title">EIGER: Next generation single photon counting detector for X-ray applications</span>. <i>Nucl. Instrum. Methods Phys. Res., Sect. A</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>650</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.nima.2010.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1016%2Fj.nima.2010.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2itLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=650&publication_year=2011&pages=79-83&author=R.+Dinapoliauthor=A.+Bergamaschiauthor=B.+Henrichauthor=R.+Horisbergerauthor=I.+Johnsonauthor=A.+Mozzanicaauthor=E.+Schmidauthor=B.+Schmittauthor=A.+Schreiberauthor=X.+Shiauthor=G.+Theidel&title=EIGER%3A+Next+generation+single+photon+counting+detector+for+X-ray+applications&doi=10.1016%2Fj.nima.2010.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">EIGER: Next generation single photon counting detector for X-ray applications</span></div><div class="casAuthors">Dinapoli, Roberto; Bergamaschi, Anna; Henrich, Beat; Horisberger, Roland; Johnson, Ian; Mozzanica, Aldo; Schmid, Elmar; Schmitt, Bernd; Schreiber, Akos; Shi, Xin-Tian; Theidel, Gerd</div><div class="citationInfo"><span class="NLM_cas:title">Nuclear Instruments & Methods in Physics Research, Section A: Accelerators, Spectrometers, Detectors, and Associated Equipment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">650</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-83</span>CODEN:
                <span class="NLM_cas:coden">NIMAER</span>;
        ISSN:<span class="NLM_cas:issn">0168-9002</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">EIGER is an advanced family of single photon counting hybrid pixel detectors, primarily aimed at diffraction expts. at synchrotrons.  Optimization of maximal functionality and minimal pixel size (using a 0.25μm process and conserving the radiation tolerant design) has resulted in 75×75μm2 pixels.  Every pixel comprises a preamplifier, shaper, discriminator (with a 6 bit DAC for threshold trimming), a configurable 4/8/12 bit counter with double buffering, as well as readout, control and test circuitry.  A novel feature of this chip is its double buffered counter, meaning a next frame can be acquired while the previous one is being readout.  An array of 256×256 pixels fits on a ∼2×2cm2 chip and a sensor of ∼8×4cm2 will be equipped with eight readout chips to form a module contg. 0.5 Mpixel.  Several modules can then be tiled to form larger area detectors.  Detectors up to 4×8 modules (16 Mpixel) are planned.  To achieve frame rates of up to 24 kHz the readout architecture is highly parallel, and the chip readout happens in parallel on 32 readout lines with a 100 MHz Double Data Rate clock.  Several chips and singles (i.e. a single chip bump-bonded to a single chip silicon sensor) were tested both with a lab X-ray source and at Swiss Light Source (SLS) beamlines.  These tests demonstrate the full functionality of the chip and provide a first assessment of its performance.  High resoln. X-ray images and "high speed movies" were produced, even without threshold trimming, at the target system frame rates (up to ∼24kHz in 4 bit mode).  In parallel, dedicated hardware, firmware and software had to be developed to comply with the enormous data rate the chip is capable of delivering.  Details of the chip design and tests will be given, as well as highlights of both test and final readout systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFJqLxbmJ1bVg90H21EOLACvtfcHk0liOLOU3HVADBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2itLvE&md5=a8051a572662777dd0bff189315d2dc8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.nima.2010.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nima.2010.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DDinapoli%26aufirst%3DR.%26aulast%3DBergamaschi%26aufirst%3DA.%26aulast%3DHenrich%26aufirst%3DB.%26aulast%3DHorisberger%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DMozzanica%26aufirst%3DA.%26aulast%3DSchmid%26aufirst%3DE.%26aulast%3DSchmitt%26aufirst%3DB.%26aulast%3DSchreiber%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DTheidel%26aufirst%3DG.%26atitle%3DEIGER%253A%2520Next%2520generation%2520single%2520photon%2520counting%2520detector%2520for%2520X-ray%2520applications%26jtitle%3DNucl.%2520Instrum.%2520Methods%2520Phys.%2520Res.%252C%2520Sect.%2520A%26date%3D2011%26volume%3D650%26spage%3D79%26epage%3D83%26doi%3D10.1016%2Fj.nima.2010.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohanty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilyichova, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcair, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totsika, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caria, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouldice, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headey, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Application of fragment-based screening to the design of inhibitors of <i>Escherichia coli</i> DsbA</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2179</span>– <span class="NLM_lpage">2184</span>, <span class="refDoi"> DOI: 10.1002/anie.201410341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fanie.201410341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFM%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=2179-2184&author=L.+A.+Adamsauthor=P.+Sharmaauthor=B.+Mohantyauthor=O.+V.+Ilyichovaauthor=M.+D.+Mulcairauthor=M.+L.+Williamsauthor=E.+C.+Gleesonauthor=M.+Totsikaauthor=B.+C.+Doakauthor=S.+Cariaauthor=K.+Rimmerauthor=J.+Horneauthor=S.+R.+Shouldiceauthor=M.+Vaziraniauthor=S.+J.+Headeyauthor=B.+R.+Plumbauthor=J.+L.+Martinauthor=B.+Herasauthor=J.+S.+Simpsonauthor=M.+J.+Scanlon&title=Application+of+fragment-based+screening+to+the+design+of+inhibitors+of+Escherichia+coli+DsbA&doi=10.1002%2Fanie.201410341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment-Based Screening to the Design of Inhibitors of Escherichia coli DsbA</span></div><div class="casAuthors">Adams, Luke A.; Sharma, Pooja; Mohanty, Biswaranjan; Ilyichova, Olga V.; Mulcair, Mark D.; Williams, Martin L.; Gleeson, Ellen C.; Totsika, Makrina; Doak, Bradley C.; Caria, Sofia; Rimmer, Kieran; Horne, James; Shouldice, Stephen R.; Vazirani, Mansha; Headey, Stephen J.; Plumb, Brent R.; Martin, Jennifer L.; Heras, Begona; Simpson, Jamie S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2179-2184</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The thiol-disulfide oxidoreductase enzyme DsbA catalyzes the formation of disulfide bonds in the periplasm of Gram-neg. bacteria.  DsbA substrates include proteins involved in bacterial virulence.  In the absence of DsbA, many of these proteins do not fold correctly, which renders the bacteria avirulent.  Thus DsbA is a crit. mediator of virulence and inhibitors may act as antivirulence agents.  Biophys. screening has been employed to identify fragments that bind to DsbA from Escherichia coli.  Elaboration of one of these fragments produced compds. that inhibit DsbA activity in vitro.  In cell-based assays, the compds. inhibit bacterial motility, but have no effect on growth in liq. culture, which is consistent with selective inhibition of DsbA.  Crystal structures of inhibitors bound to DsbA indicate that they bind adjacent to the active site.  Together, the data suggest that DsbA may be amenable to the development of novel antibacterial compds. that act by inhibiting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv5mTk2z8YLbVg90H21EOLACvtfcHk0liOLOU3HVADBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFM%253D&md5=dd5b947950fdacecf6a4dff69c59fc24</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fanie.201410341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201410341%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DL.%2BA.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DMohanty%26aufirst%3DB.%26aulast%3DIlyichova%26aufirst%3DO.%2BV.%26aulast%3DMulcair%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DM.%2BL.%26aulast%3DGleeson%26aufirst%3DE.%2BC.%26aulast%3DTotsika%26aufirst%3DM.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DCaria%26aufirst%3DS.%26aulast%3DRimmer%26aufirst%3DK.%26aulast%3DHorne%26aufirst%3DJ.%26aulast%3DShouldice%26aufirst%3DS.%2BR.%26aulast%3DVazirani%26aufirst%3DM.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DPlumb%26aufirst%3DB.%2BR.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DApplication%2520of%2520fragment-based%2520screening%2520to%2520the%2520design%2520of%2520inhibitors%2520of%2520Escherichia%2520coli%2520DsbA%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D2179%26epage%3D2184%26doi%3D10.1002%2Fanie.201410341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouldice, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totsika, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schembri, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">DSB proteins and bacterial pathogenicity</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrmicro2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19198617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFeqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=215-225&author=B.+Herasauthor=S.+R.+Shouldiceauthor=M.+Totsikaauthor=M.+J.+Scanlonauthor=M.+A.+Schembriauthor=J.+L.+Martin&title=DSB+proteins+and+bacterial+pathogenicity&doi=10.1038%2Fnrmicro2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DSB proteins and bacterial pathogenicity</span></div><div class="casAuthors">Heras, Begona; Shouldice, Stephen R.; Totsika, Makrina; Scanlon, Martin J.; Schembri, Mark A.; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-225</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  If DNA is the information of life, then proteins are the machines of life, but they must be assembled and correctly folded to function.  A key step in the protein-folding pathway is the introduction of disulfide bonds between Cys residues in a process called oxidative protein folding.  Many bacteria use an oxidative protein folding machinery to assemble disulfide bond-contg. proteins (DSB proteins) that are essential for cell integrity and to produce virulence factors.  Although current knowledge of this machinery stems largely from Escherichia coli K-12, this view must now be adjusted to encompass the wider range of disulfide catalytic systems present in bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp-m_RBloHOLVg90H21EOLACvtfcHk0liOLOU3HVADBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFeqsbc%253D&md5=d75810468658d9f5c00da4f54a6e3e24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2087%26sid%3Dliteratum%253Aachs%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DShouldice%26aufirst%3DS.%2BR.%26aulast%3DTotsika%26aufirst%3DM.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DSchembri%26aufirst%3DM.%2BA.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DDSB%2520proteins%2520and%2520bacterial%2520pathogenicity%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D215%26epage%3D225%26doi%3D10.1038%2Fnrmicro2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Targeting virulence not viability in the search for future antibacterials</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1111/bcp.12356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1111%2Fbcp.12356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=24552512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2015&pages=208-215&author=B.+Herasauthor=M.+J.+Scanlonauthor=J.+L.+Martin&title=Targeting+virulence+not+viability+in+the+search+for+future+antibacterials&doi=10.1111%2Fbcp.12356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting virulence not viability in the search for future antibacterials</span></div><div class="casAuthors">Heras, Begona; Scanlon, Martin J.; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-215</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">1365-2125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  New antibacterials need new approaches to overcome the problem of rapid antibiotic resistance.  Here we review the development of potential new antibacterial drugs that do not kill bacteria or inhibit their growth, but combat disease instead by targeting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR8H_GpurAKLVg90H21EOLACvtfcHk0lg6BQYVRmDDxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCrtbo%253D&md5=60b729ec4aad3b5b22d3a0f150a498bb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12356%26sid%3Dliteratum%253Aachs%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DTargeting%2520virulence%2520not%2520viability%2520in%2520the%2520search%2520for%2520future%2520antibacterials%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2015%26volume%3D79%26spage%3D208%26epage%3D215%26doi%3D10.1111%2Fbcp.12356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lg6BQYVRmDDxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phizackerley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0lg6BQYVRmDDxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aragao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrop, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T.</span></span> <span> </span><span class="NLM_article-title">MX1: a bending-magnet crystallography beamline serving both chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/S1600577514021717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS1600577514021717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=25537608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFeitLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=187-190&author=N.+P.+Cowiesonauthor=D.+Aragaoauthor=M.+Cliftauthor=D.+J.+Ericssonauthor=C.+Geeauthor=S.+J.+Harropauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Williamsonauthor=T.+Caradoc-Davies&title=MX1%3A+a+bending-magnet+crystallography+beamline+serving+both+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2FS1600577514021717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MX1: a bending-magnet crystallography beamline serving both chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Cowieson, Nathan Philip; Aragao, David; Clift, Mark; Ericsson, Daniel J.; Gee, Christine; Harrop, Stephen J.; Mudie, Nathan; Panjikar, Santosh; Price, Jason R.; Riboldi-Tunnicliffe, Alan; Williamson, Rachel; Caradoc-Davies, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-190</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX1 is a bending-magnet crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range from 8 to 18 keV to a focal spot at the sample position of 120 μm FWHM.  The beamline end station and ancillary equipment facilitate local and remote access for both chem. and biol. macromol. crystallog.  Here, the design of the beamline and end station are discussed.  The beamline has enjoyed a full user program for the last seven years and scientific highlights from the user program are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonncGct23S27Vg90H21EOLACvtfcHk0liNh0q3-zDFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFeitLrO&md5=2f00508e17ab0c8584519b30c3f0505b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS1600577514021717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577514021717%26sid%3Dliteratum%253Aachs%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DAragao%26aufirst%3DD.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%26aulast%3DHarrop%26aufirst%3DS.%2BJ.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%26atitle%3DMX1%253A%2520a%2520bending-magnet%2520crystallography%2520beamline%2520serving%2520both%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2015%26volume%3D22%26spage%3D187%26epage%3D190%26doi%3D10.1107%2FS1600577514021717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aragão, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukuvada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span> <span> </span><span class="NLM_article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1107/S1600577518003120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS1600577518003120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=29714201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=885-891&author=D.+Arag%C3%A3oauthor=J.+Aishimaauthor=H.+Cherukuvadaauthor=R.+Clarkenauthor=M.+Cliftauthor=N.+P.+Cowiesonauthor=D.+J.+Ericssonauthor=C.+L.+Geeauthor=S.+Macedoauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Rostanauthor=R.+Williamsonauthor=T.+T.+Caradoc-Davies&title=MX2%3A+a+high-flux+undulator+microfocus+beamline+serving+both+the+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2FS1600577518003120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Aragao, David; Aishima, Jun; Cherukuvada, Hima; Clarken, Robert; Clift, Mark; Cowieson, Nathan Philip; Ericsson, Daniel Jesper; Gee, Christine L.; Macedo, Sofia; Mudie, Nathan; Panjikar, Santosh; Price, Jason Roy; Riboldi-Tunnicliffe, Alan; Rostan, Robert; Williamson, Rachel; Caradoc-Davies, Thomas Tudor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">885-891</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX2 is an in-vacuum undulator-based crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range 4.8-21 keV to a focal spot of 22 × 12 μm FWHM (H × V).  At 13 keV the flux at the sample is 3.4 × 1012 photons s-1.  The beamline endstation allows robotic handling of cryogenic samples via an updated SSRL SAM robot.  This beamline is ideal for weakly diffracting hard-to-crystallize proteins, virus particles, protein assemblies and nucleic acids as well as smaller mols. such as inorg. catalysts and org. drug mols.  The beamline is now mature and has enjoyed a full user program for the last nine years.  This paper describes the beamline status, plans for its future and some recent scientific highlights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgtd7iUcXqY7Vg90H21EOLACvtfcHk0liNh0q3-zDFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D&md5=8a52a2db9f5d01958df27ae74e2496d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1107%2FS1600577518003120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577518003120%26sid%3Dliteratum%253Aachs%26aulast%3DArag%25C3%25A3o%26aufirst%3DD.%26aulast%3DAishima%26aufirst%3DJ.%26aulast%3DCherukuvada%26aufirst%3DH.%26aulast%3DClarken%26aufirst%3DR.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BL.%26aulast%3DMacedo%26aufirst%3DS.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DRostan%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26atitle%3DMX2%253A%2520a%2520high-flux%2520undulator%2520microfocus%2520beamline%2520serving%2520both%2520the%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2018%26volume%3D25%26spage%3D885%26epage%3D891%26doi%3D10.1107%2FS1600577518003120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0liNh0q3-zDFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lipzfgAyedVZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Legrand, P.</span></span> <i>XDSME: XDS Made Easier</i>. GitHub Repository, <span class="NLM_year">2017</span>, <a href="https://github.com/legrandp/xdsme" class="extLink">https://github.com/legrandp/xdsme</a>. <span class="refDoi"> DOI: 10.5281/zenodo.837885</span>  (accessed April 18, 2018).</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.5281%2Fzenodo.837885" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+Legrand&title=XDSME%3A+XDS+Made+Easier"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5281%2Fzenodo.837885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.5281%252Fzenodo.837885%26sid%3Dliteratum%253Aachs%26aulast%3DLegrand%26aufirst%3DP.%26jtitle%3DXDSME%253A%2520XDS%2520Made%2520Easier%26date%3D2017%26doi%3D10.5281%2Fzenodo.837885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">Auto-rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1107/S0907444905001307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444905001307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15805600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=449-457&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=Auto-rickshaw%3A+an+automated+crystal+structure+determination+platform+as+an+efficient+tool+for+the+validation+of+an+X-ray+diffraction+experiment&doi=10.1107%2FS0907444905001307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-457</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The EMBL-Hamburg Automated Crystal Structure Detn. Platform is a system that combines a no. of existing macromol. crystallog. computer programs and several decision-makers into a software pipeline for automated and efficient crystal structure detn.  The pipeline can be invoked as soon as x-ray data from derivatized protein crystals have been collected and processed.  It is controlled by a web-based graphical user interface for data and parameter input, and for monitoring the progress of structure detn.  A large no. of possible structure-soln. paths are encoded in the system and the optimal path is selected by the decision-makers as the structure soln. evolves.  The processes have been optimized for speed so that the pipeline can be used effectively for validating the x-ray expt. at a synchrotron beamline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZbIxeGX0aqrVg90H21EOLACvtfcHk0lipzfgAyedVZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D&md5=98b93bf2e456104466b2e7b07e3ac197</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001307%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DAuto-rickshaw%253A%2520an%2520automated%2520crystal%2520structure%2520determination%2520platform%2520as%2520an%2520efficient%2520tool%2520for%2520the%2520validation%2520of%2520an%2520X-ray%2520diffraction%2520experiment%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D449%26epage%3D457%26doi%3D10.1107%2FS0907444905001307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909029643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19770506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1089-1097&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=On+the+combination+of+molecular+replacement+and+single-wavelength+anomalous+diffraction+phasing+for+automated+structure+determination&doi=10.1107%2FS0907444909029643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1097</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A combination of mol. replacement and single-wavelength anomalous diffraction phasing has been incorporated into the automated structure-detn. platform Auto-Rickshaw.  The complete MRSAD procedure includes mol. replacement, model refinement, exptl. phasing, phase improvement and automated model building.  The improvement over the std. SAD or MR approaches is illustrated by ten test cases taken from the JCSG diffraction data-set database.  Poor MR or SAD phases with phase errors larger than 70° can be improved using the described procedure and a large fraction of the model can be detd. in a purely automatic manner from X-ray data extending to better than 2.6 Å resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz21EsxIkqJLVg90H21EOLACvtfcHk0lipzfgAyedVZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP&md5=e3629b21fee4dc43450e853417e6b598</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029643%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DOn%2520the%2520combination%2520of%2520molecular%2520replacement%2520and%2520single-wavelength%2520anomalous%2520diffraction%2520phasing%2520for%2520automated%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1089%26epage%3D1097%26doi%3D10.1107%2FS0907444909029643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulombe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordeleau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, P. L.</span></span> <span> </span><span class="NLM_article-title">Compound aggregation in drug discovery: implementing a practical NMR assay for medicinal chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5142</span>– <span class="NLM_lpage">5150</span>, <span class="refDoi"> DOI: 10.1021/jm400535b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400535b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslSqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5142-5150&author=S.+R.+LaPlanteauthor=R.+Carsonauthor=J.+Gillardauthor=N.+Aubryauthor=R.+Coulombeauthor=S.+Bordeleauauthor=P.+Bonneauauthor=M.+Littleauthor=J.+O%E2%80%99Mearaauthor=P.+L.+Beaulieu&title=Compound+aggregation+in+drug+discovery%3A+implementing+a+practical+NMR+assay+for+medicinal+chemists&doi=10.1021%2Fjm400535b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Compound Aggregation in Drug Discovery: Implementing a Practical NMR Assay for Medicinal Chemists</span></div><div class="casAuthors">LaPlante, Steven R.; Carson, Rebekah; Gillard, James; Aubry, Norman; Coulombe, Rene; Bordeleau, Sylvain; Bonneau, Pierre; Little, Michael; O'Meara, Jeff; Beaulieu, Pierre L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5142-5150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmaceutical industry has recognized that many drug-like mols. can self-aggregate in aq. media and have physicochem. properties that skew exptl. results and decisions.  Herein, we introduce the use of a simple NMR strategy for detecting the formation of aggregates using diln. expts. that can be performed on equipment prevalent in most synthetic chem. departments.  We show that 1H NMR resonances are sensitive to large mol.-size entities and to smaller multimers and mixts. of species.  Practical details are provided for sample prepn. and for detg. the concns. of single mol., aggregate entities, and ppt.  The crit. concns. above which aggregation begins can be found and were corroborated by comparisons with light scattering techniques.  Disaggregation can also be monitored using detergents.  This NMR assay should serve as a practical and readily available tool for medicinal chemists to better characterize how their compds. behave in aq. media and influence drug design decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAjZBdJ970nrVg90H21EOLACvtfcHk0lj3LZr7m6p48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslSqt7w%253D&md5=ba5e7894d6080752dc4c1bfb2ed38f73</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm400535b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400535b%26sid%3Dliteratum%253Aachs%26aulast%3DLaPlante%26aufirst%3DS.%2BR.%26aulast%3DCarson%26aufirst%3DR.%26aulast%3DGillard%26aufirst%3DJ.%26aulast%3DAubry%26aufirst%3DN.%26aulast%3DCoulombe%26aufirst%3DR.%26aulast%3DBordeleau%26aufirst%3DS.%26aulast%3DBonneau%26aufirst%3DP.%26aulast%3DLittle%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DJ.%26aulast%3DBeaulieu%26aufirst%3DP.%2BL.%26atitle%3DCompound%2520aggregation%2520in%2520drug%2520discovery%253A%2520implementing%2520a%2520practical%2520NMR%2520assay%2520for%2520medicinal%2520chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5142%26epage%3D5150%26doi%3D10.1021%2Fjm400535b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudelman, A.</span></span> <span> </span><span class="NLM_article-title">NMR chemical shifts of common laboratory solvents as trace impurities</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7512</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jo971176v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+chemical+shifts+of+common+laboratory+solvents+as+trace+impurities&doi=10.1021%2Fjo971176v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0lj3LZr7m6p48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520chemical%2520shifts%2520of%2520common%2520laboratory%2520solvents%2520as%2520trace%2520impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515%26doi%3D10.1021%2Fjo971176v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Studier, F. W.</span></span> <span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span>. <i>Protein Expr. Purif.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0lj3LZr7m6p48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expr.%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paxman, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borg, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahler, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakellaris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossjohn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">The structure of the bacterial oxidoreductase enzyme DsbA in complex with a peptide reveals a basis for substrate specificity in the catalytic cycle of DsbA enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">17835</span>– <span class="NLM_lpage">17845</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.011502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1074%2Fjbc.M109.011502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19389711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17835-17845&author=J.+J.+Paxmanauthor=N.+A.+Borgauthor=J.+Horneauthor=P.+E.+Thompsonauthor=Y.+Chinauthor=P.+Sharmaauthor=J.+S.+Simpsonauthor=J.+Wielensauthor=S.+Piekauthor=C.+M.+Kahlerauthor=H.+Sakellarisauthor=M.+Pearceauthor=S.+P.+Bottomleyauthor=J.+Rossjohnauthor=M.+J.+Scanlon&title=The+structure+of+the+bacterial+oxidoreductase+enzyme+DsbA+in+complex+with+a+peptide+reveals+a+basis+for+substrate+specificity+in+the+catalytic+cycle+of+DsbA+enzymes&doi=10.1074%2Fjbc.M109.011502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Structure of the Bacterial Oxidoreductase Enzyme DsbA in Complex with a Peptide Reveals a Basis for Substrate Specificity in the Catalytic Cycle of DsbA Enzymes</span></div><div class="casAuthors">Paxman, Jason J.; Borg, Natalie A.; Horne, James; Thompson, Philip E.; Chin, Yanni; Sharma, Pooja; Simpson, Jamie S.; Wielens, Jerome; Piek, Susannah; Kahler, Charlene M.; Sakellaris, Harry; Pearce, Mary; Bottomley, Stephen P.; Rossjohn, Jamie; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17835-17845</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Oxidative protein folding in Gram-neg. bacteria results in the formation of disulfide bonds between pairs of cysteine residues.  This is a multistep process in which the dithiol-disulfide oxidoreductase enzyme, DsbA, plays a central role.  The structure of DsbA comprises an all helical domain of unknown function and a thioredoxin domain, where active site cysteines shuttle between an oxidized, substrate-bound, reduced form and a DsbB-bound form, where DsbB is a membrane protein that reoxidizes DsbA.  Most DsbA enzymes interact with a wide variety of reduced substrates and show little specificity.  However, a no. of DsbA enzymes have now been identified that have narrow substrate repertoires and appear to interact specifically with a smaller no. of substrates.  The transient nature of the DsbA-substrate complex has hampered our understanding of the factors that govern the interaction of DsbA enzymes with their substrates.  Here we report the crystal structure of a complex between Escherichia coli DsbA and a peptide with a sequence derived from a substrate.  The binding site identified in the DsbA-peptide complex was distinct from that obsd. for DsbB in the DsbA-DsbB complex.  The structure revealed details of the DsbA-peptide interaction and suggested a mechanism by which DsbA can simultaneously show broad specificity for substrates yet exhibit specificity for DsbB.  This mode of binding was supported by soln. NMR data as well as functional data, which demonstrated that the substrate specificity of DsbA could be modified via changes at the binding interface identified in the structure of the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLyWFqwOudmrVg90H21EOLACvtfcHk0ljEzyjuckCTbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksbk%253D&md5=5de5d0259663de920a057589fc980ddb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.011502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.011502%26sid%3Dliteratum%253Aachs%26aulast%3DPaxman%26aufirst%3DJ.%2BJ.%26aulast%3DBorg%26aufirst%3DN.%2BA.%26aulast%3DHorne%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DP.%2BE.%26aulast%3DChin%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DPiek%26aufirst%3DS.%26aulast%3DKahler%26aufirst%3DC.%2BM.%26aulast%3DSakellaris%26aufirst%3DH.%26aulast%3DPearce%26aufirst%3DM.%26aulast%3DBottomley%26aufirst%3DS.%2BP.%26aulast%3DRossjohn%26aufirst%3DJ.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520structure%2520of%2520the%2520bacterial%2520oxidoreductase%2520enzyme%2520DsbA%2520in%2520complex%2520with%2520a%2520peptide%2520reveals%2520a%2520basis%2520for%2520substrate%2520specificity%2520in%2520the%2520catalytic%2520cycle%2520of%2520DsbA%2520enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17835%26epage%3D17845%26doi%3D10.1074%2Fjbc.M109.011502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waksman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, J. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Crystallization of DsbA, an <i>Escherichia coli</i> protein required for disulphide bond formation in vivo</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>230</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1006%2Fjmbi.1993.1226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=8478925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1eisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=230&publication_year=1993&pages=1097-1100&author=J.+L.+Martinauthor=G.+Waksmanauthor=J.+C.+A.+Bardwellauthor=J.+Beckwithauthor=J.+Kuriyan&title=Crystallization+of+DsbA%2C+an+Escherichia+coli+protein+required+for+disulphide+bond+formation+in+vivo&doi=10.1006%2Fjmbi.1993.1226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization of DsbA, an Escherichia coli protein required for disulfide bond formation in vivo</span></div><div class="casAuthors">Martin, Jennifer L.; Waksman, Gabriel; Bardwell, James C. A.; Beckwith, Jon; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">230</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1097-100</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">DsbA is a 21 kDa protein that facilitates disulfide bond formation and is required for the correct folding and stability of a no. of exported proteins in E. coli.  Crystals of oxidized DsbA have been obtained from polyethylene glycol 8000 (20 to 25%), 0.1 M-cacodylate buffer (pH 6.5) and 1% 2-methyl-2,4-pentanediol.  Oxidn. of the protein is crit. for reproducibly obtaining high quality crystals.  The resulting crystals diffract to 2 Å and belong to the monoclinic space group C2 with cell dimensions a = 117.5 Å, b = 65.0 Å, c = 76.3 Å, β = 126.3° with two mols. in the asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC5XXZuGR-e7Vg90H21EOLACvtfcHk0ljEzyjuckCTbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1eisrk%253D&md5=c011dbb251518ed1d404518b1e301ca0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1226%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DWaksman%26aufirst%3DG.%26aulast%3DBardwell%26aufirst%3DJ.%2BC.%2BA.%26aulast%3DBeckwith%26aufirst%3DJ.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystallization%2520of%2520DsbA%252C%2520an%2520Escherichia%2520coli%2520protein%2520required%2520for%2520disulphide%2520bond%2520formation%2520in%2520vivo%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D230%26spage%3D1097%26epage%3D1100%26doi%3D10.1006%2Fjmbi.1993.1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span> <span> </span><span class="NLM_article-title">Linking crystallographic model and data quality</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1218231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1126%2Fscience.1218231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=22628654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1030-1033&author=P.+A.+Karplusauthor=K.+Diederichs&title=Linking+crystallographic+model+and+data+quality&doi=10.1126%2Fscience.1218231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Linking Crystallographic Model and Data Quality</span></div><div class="casAuthors">Karplus, P. Andrew; Diederichs, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">1030-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In macromol. x-ray crystallog., refinement R values measure the agreement between obsd. and calcd. data.  Analogously, Rmerge values reporting on the agreement between multiple measurements of a given reflection are used to assess data quality.  Here, we show that despite their widespread use, Rmerge values are poorly suited for detg. the high-resoln. limit and that current std. protocols discard much useful data.  We introduce a statistic that ests. the correlation of an obsd. data set with the underlying (not measurable) true signal; this quantity, CC*, provides a single statistically valid guide for deciding which data are useful.  CC* also can be used to assess model and data quality on the same scale, and this reveals when data quality is limiting model improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3BGdcdUzbVg90H21EOLACvtfcHk0ljEzyjuckCTbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D&md5=b8ee6edacea44364cc66c44dd3a505fd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1218231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1218231%26sid%3Dliteratum%253Aachs%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26aulast%3DDiederichs%26aufirst%3DK.%26atitle%3DLinking%2520crystallographic%2520model%2520and%2520data%2520quality%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1030%26epage%3D1033%26doi%3D10.1126%2Fscience.1218231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glockshuber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber-Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of three His32 mutants of DsbA: support for an electrostatic role of His32 in DsbA stability</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1893</span>– <span class="NLM_lpage">1900</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fpro.5560060910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=9300489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFert7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=1893-1900&author=L.+W.+Guddatauthor=J.+C.+Bardwellauthor=R.+Glockshuberauthor=M.+Huber-Wunderlichauthor=T.+Zanderauthor=J.+L.+Martin&title=Structural+analysis+of+three+His32+mutants+of+DsbA%3A+support+for+an+electrostatic+role+of+His32+in+DsbA+stability&doi=10.1002%2Fpro.5560060910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of three His32 mutants of DsbA: support for an electrostatic role of His32 in DsbA stability</span></div><div class="casAuthors">Guddat, Luke W.; Bardwell, James C. A.; Glockshuber, Rudi; Huber-Wunderlich, Martina; Zander, Thomas; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1893-1900</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">DsbA, a 21-kDa protein from Escherichia coli, is a potent oxidizing disulfide catalyst required for disulfide bond formation in secreted proteins.  The active site of DsbA is similar to that of mammalian protein disulfide isomerases, and includes a reversible disulfide bond formed from cysteines sepd. by two residues (Cys30-Pro31-His32-Cys33).  Unlike most protein disulfides, the active-site disulfide of DsbA is highly reactive and the oxidized form of DsbA is much less stable than the reduced form at physiol. pH.  His32, one of the two residues between the active-site cysteines, is crit. to the oxidizing power of DsbA and to the relative instability of the protein in the oxidized form.  Mutation of this single residue to tyrosine, serine, or leucine results in a significant increase in stability (of ∼5-7 kcal/mol) of the oxidized His32 variants relative to the oxidized wild-type protein.  Despite the dramatic changes in stability, the structures of all three oxidized DsbA His32 variants are very similar to the wild-type oxidized structure, including conservation of solvent atoms near the active-site residue, Cys30.  These results show that the His32 residue does not exert a conformational effect on the structure of DsbA.  The destabilizing effect of His32 on oxidized DsbA is therefore most likely electrostatic in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooscLQFcNL7bVg90H21EOLACvtfcHk0lj6TJKT6zbiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFert7o%253D&md5=2a48236ec9e165e1bcd06b961b0ada46</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060910%26sid%3Dliteratum%253Aachs%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DBardwell%26aufirst%3DJ.%2BC.%26aulast%3DGlockshuber%26aufirst%3DR.%26aulast%3DHuber-Wunderlich%26aufirst%3DM.%26aulast%3DZander%26aufirst%3DT.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DStructural%2520analysis%2520of%2520three%2520His32%2520mutants%2520of%2520DsbA%253A%2520support%2520for%2520an%2520electrostatic%2520role%2520of%2520His32%2520in%2520DsbA%2520stability%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D1893%26epage%3D1900%26doi%3D10.1002%2Fpro.5560060910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lj6TJKT6zbiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR system: A new software suite for macromolecular structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2eaY60AnJYAuarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520system%253A%2520A%2520new%2520software%2520suite%2520for%2520macromolecular%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lg3r4y2FKcmxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammito, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">861</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1107/S2059798319011471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS2059798319011471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2019&pages=861-877&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=M.+L.+Bakerauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=T.+I.+Crollauthor=B.+Hintzeauthor=L.+W.+Hungauthor=S.+Jainauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=B.+K.+Poonauthor=M.+G.+Prisantauthor=R.+J.+Readauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=M.+D.+Sammitoauthor=O.+V.+Sobolevauthor=D.+H.+Stockwellauthor=T.+C.+Terwilligerauthor=A.+G.+Urzhumtsevauthor=L.+L.+Videauauthor=C.+J.+Williamsauthor=P.+D.+Adams&title=Macromolecular+structure+determination+using+X-rays%2C+neutrons+and+electrons%3A+recent+developments+in+Phenix&doi=10.1107%2FS2059798319011471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Baker, Matthew L.; Bunkoczi, Gabor; Chen, Vincent B.; Croll, Tristan I.; Hintze, Bradley; Hung, Li-Wei; Jain, Swati; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Poon, Billy K.; Prisant, Michael G.; Read, Randy J.; Richardson, Jane S.; Richardson, David C.; Sammito, Massimo D.; Sobolev, Oleg V.; Stockwell, Duncan H.; Terwilliger, Thomas C.; Urzhumtsev, Alexandre G.; Videau, Lizbeth L.; Williams, Christopher J.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">861-877</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Diffraction (X-ray, neutron and electron) and electron cryo-microscopy are powerful methods to det. three-dimensional macromol. structures, which are required to understand biol. processes and to develop new therapeutics against diseases.  The overall structure-soln. workflow is similar for these techniques, but nuances exist because the properties of the reduced exptl. data are different.  Software tools for structure detn. should therefore be tailored for each method.  Phenix is a comprehensive software package for macromol. structure detn. that handles data from any of these techniques.  Tasks performed with Phenix include data-quality assessment, map improvement, model building, the validation/rebuilding/refinement cycle and deposition.  Each tool caters to the type of exptl. data.  The design of Phenix emphasizes the automation of procedures, where possible, to minimize repetitive and time-consuming manual tasks, while default parameters are chosen to encourage best practice.  A graphical user interface provides access to many command-line features of Phenix and streamlines the transition between programs, project tracking and re-running of previous tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolHnU8ECe2CrVg90H21EOLACvtfcHk0lg3r4y2FKcmxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO&md5=e60be65abece48923abf937a4c2640a6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS2059798319011471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798319011471%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBaker%26aufirst%3DM.%2BL.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DCroll%26aufirst%3DT.%2BI.%26aulast%3DHintze%26aufirst%3DB.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DSammito%26aufirst%3DM.%2BD.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DStockwell%26aufirst%3DD.%2BH.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DMacromolecular%2520structure%2520determination%2520using%2520X-rays%252C%2520neutrons%2520and%2520electrons%253A%2520recent%2520developments%2520in%2520Phenix%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2019%26volume%3D75%26spage%3D861%26epage%3D877%26doi%3D10.1107%2FS2059798319011471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liXyORH_0we4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liXyORH_0we4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0liXyORH_0we4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0liLS1rLjCQ9DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liLS1rLjCQ9DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD web server for the generation of multi-group TLS models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+web+server+for+the+generation+of+multi-group+TLS+models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0liLS1rLjCQ9DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520web%2520server%2520for%2520the%2520generation%2520of%2520multi-group%2520TLS%2520models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkis, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orpen, A. G.</span></span> <span> </span><span class="NLM_article-title">Retrieval of crystallographically-derived molecular geometry information</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2133</span>– <span class="NLM_lpage">2144</span>, <span class="refDoi"> DOI: 10.1021/ci049780b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049780b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2133-2144&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=M.+Kesslerauthor=J.+Luoauthor=W.+D.+S.+Motherwellauthor=L.+H.+Purkisauthor=B.+R.+Smithauthor=R.+Taylorauthor=R.+I.+Cooperauthor=S.+E.+Harrisauthor=A.+G.+Orpen&title=Retrieval+of+crystallographically-derived+molecular+geometry+information&doi=10.1021%2Fci049780b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Retrieval of Crystallographically-Derived Molecular Geometry Information</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Kessler, Magnus; Luo, Jie; Motherwell, W. D. Sam; Purkis, Lucy H.; Smith, Barry R.; Taylor, Robin; Cooper, Richard I.; Harris, Stephanie E.; Orpen, A. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2133-2144</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crystallog. detd. bond length, valence angle, and torsion angle information in the Cambridge Structural Database (CSD) has many uses.  However, accessing it by conventional substructure searching requires nontrivial user intervention.  In consequence, these valuable data were underused and were not directly accessible to client applications.  The situation was remedied by development of a new program (Mogul) for automated retrieval of mol. geometry data from the CSD.  The program uses a system of keys to encode the chem. environments of fragments (bonds, valence angles, and acyclic torsions) from CSD structures.  Fragments with identical keys are deemed to be chem. identical and are grouped together, and the distribution of the appropriate geometrical parameter (bond length, valence angle, or torsion angle) is computed and stored.  Use of a search tree indexed on key values, together with a novel similarity calcn., then enables the distribution matching any given query fragment (or the distributions most closely matching, if an adequate exact match is unavailable) to be found easily and with no user intervention.  Validation expts. indicate that, with rare exceptions, search results afford precise and unbiased ests. of mol. geometrical preferences.  Such ests. may be used, for example, to validate the geometries of libraries of modeled mols. or of newly detd. crystal structures or to assist structure soln. from low-resoln. (e.g. powder diffraction) x-ray data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8AZn6w-T817Vg90H21EOLACvtfcHk0ljScrk73us6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlalsr8%253D&md5=a7909c257045351d5bff7b78c7fba2d5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci049780b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049780b%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DKessler%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DMotherwell%26aufirst%3DW.%2BD.%2BS.%26aulast%3DPurkis%26aufirst%3DL.%2BH.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DCooper%26aufirst%3DR.%2BI.%26aulast%3DHarris%26aufirst%3DS.%2BE.%26aulast%3DOrpen%26aufirst%3DA.%2BG.%26atitle%3DRetrieval%2520of%2520crystallographically-derived%2520molecular%2520geometry%2520information%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2133%26epage%3D2144%26doi%3D10.1021%2Fci049780b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cottrell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebeschuetz, J. W.</span></span> <span> </span><span class="NLM_article-title">Validating and understanding ring conformations using small molecule crystallographic data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1021/ci200439d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200439d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVWitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=956-962&author=S.+J.+Cottrellauthor=T.+S.+G.+Olssonauthor=R.+Taylorauthor=J.+C.+Coleauthor=J.+W.+Liebeschuetz&title=Validating+and+understanding+ring+conformations+using+small+molecule+crystallographic+data&doi=10.1021%2Fci200439d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Validating and Understanding Ring Conformations Using Small Molecule Crystallographic Data</span></div><div class="casAuthors">Cottrell, Simon J.; Olsson, Tjelvar S. G.; Taylor, Robin; Cole, Jason C.; Liebeschuetz, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">956-962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the conformational preferences of ring structures is fundamental to structure-based drug design.  Although the Cambridge Structural Database (CSD) provides information on the preferred conformations of small mols., analyzing this data can be very time-consuming.  In order to overcome this hurdle, tools have been developed for quickly extg. geometrical preferences from the CSD.  Here we describe how the program Mogul has been extended to analyze and compare ring conformations, using a library derived from over 900 000 ring fragments in the CSD.  We illustrate how these can be used to understand the conformational preferences of mols. in a crystal lattice and bound to proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8DAjmnZg2x7Vg90H21EOLACvtfcHk0ljScrk73us6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVWitLg%253D&md5=fb061c588b417493fc2f580f46595329</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci200439d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200439d%26sid%3Dliteratum%253Aachs%26aulast%3DCottrell%26aufirst%3DS.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DLiebeschuetz%26aufirst%3DJ.%2BW.%26atitle%3DValidating%2520and%2520understanding%2520ring%2520conformations%2520using%2520small%2520molecule%2520crystallographic%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D956%26epage%3D962%26doi%3D10.1021%2Fci200439d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Molprobity: All-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Molprobity%3A+All-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljScrk73us6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolprobity%253A%2520All-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fandi  Sutanto</span>, <span class="hlFld-ContribAuthor ">Shabnam  Shaabani</span>, <span class="hlFld-ContribAuthor ">Rick  Oerlemans</span>, <span class="hlFld-ContribAuthor ">Deniz  Eris</span>, <span class="hlFld-ContribAuthor ">Pravin  Patil</span>, <span class="hlFld-ContribAuthor ">Mojgan  Hadian</span>, <span class="hlFld-ContribAuthor ">Meitian  Wang</span>, <span class="hlFld-ContribAuthor ">May Elizabeth  Sharpe</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Groves</span>, <span class="hlFld-ContribAuthor ">Alexander  Dömling</span>. </span><span class="cited-content_cbyCitation_article-title">Combining High‐Throughput Synthesis and High‐Throughput Protein Crystallography for Accelerated Hit Identification. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (33)
                                     , 18231-18239. <a href="https://doi.org/10.1002/anie.202105584" title="DOI URL">https://doi.org/10.1002/anie.202105584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202105584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202105584%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DCombining%252BHigh%2525E2%252580%252590Throughput%252BSynthesis%252Band%252BHigh%2525E2%252580%252590Throughput%252BProtein%252BCrystallography%252Bfor%252BAccelerated%252BHit%252BIdentification%26aulast%3DSutanto%26aufirst%3DFandi%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D33%26spage%3D18231%26epage%3D18239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fandi  Sutanto</span>, <span class="hlFld-ContribAuthor ">Shabnam  Shaabani</span>, <span class="hlFld-ContribAuthor ">Rick  Oerlemans</span>, <span class="hlFld-ContribAuthor ">Deniz  Eris</span>, <span class="hlFld-ContribAuthor ">Pravin  Patil</span>, <span class="hlFld-ContribAuthor ">Mojgan  Hadian</span>, <span class="hlFld-ContribAuthor ">Meitian  Wang</span>, <span class="hlFld-ContribAuthor ">May Elizabeth  Sharpe</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Groves</span>, <span class="hlFld-ContribAuthor ">Alexander  Dömling</span>. </span><span class="cited-content_cbyCitation_article-title">Combining High‐Throughput Synthesis and High‐Throughput Protein Crystallography for Accelerated Hit Identification. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (33)
                                     , 18379-18387. <a href="https://doi.org/10.1002/ange.202105584" title="DOI URL">https://doi.org/10.1002/ange.202105584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202105584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202105584%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DCombining%252BHigh%2525E2%252580%252590Throughput%252BSynthesis%252Band%252BHigh%2525E2%252580%252590Throughput%252BProtein%252BCrystallography%252Bfor%252BAccelerated%252BHit%252BIdentification%26aulast%3DSutanto%26aufirst%3DFandi%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D33%26spage%3D18379%26epage%3D18387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luke F.  Duncan</span>, <span class="hlFld-ContribAuthor ">Geqing  Wang</span>, <span class="hlFld-ContribAuthor ">Olga V.  Ilyichova</span>, <span class="hlFld-ContribAuthor ">Rabeb  Dhouib</span>, <span class="hlFld-ContribAuthor ">Makrina  Totsika</span>, <span class="hlFld-ContribAuthor ">Martin J.  Scanlon</span>, <span class="hlFld-ContribAuthor ">Begoña  Heras</span>, <span class="hlFld-ContribAuthor ">Belinda M.  Abbott</span>. </span><span class="cited-content_cbyCitation_article-title">Elaboration of a Benzofuran Scaffold and Evaluation of Binding Affinity and Inhibition of Escherichia coli DsbA: A Fragment-Based Drug Design Approach to Novel Antivirulence Compounds. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>9 </em>, 116315. <a href="https://doi.org/10.1016/j.bmc.2021.116315" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116315</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116315%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DElaboration%252Bof%252Ba%252BBenzofuran%252BScaffold%252Band%252BEvaluation%252Bof%252BBinding%252BAffinity%252Band%252BInhibition%252Bof%252BEscherichia%252Bcoli%252BDsbA%25253A%252BA%252BFragment-Based%252BDrug%252BDesign%252BApproach%252Bto%252BNovel%252BAntivirulence%252BCompounds%26aulast%3DDuncan%26aufirst%3DLuke%2BF.%26date%3D2021%26volume%3D9%26spage%3D116315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew D.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Maksims  Yevglevskis</span>, <span class="hlFld-ContribAuthor ">Amit  Nathubhai</span>, <span class="hlFld-ContribAuthor ">Tony D.  James</span>, <span class="hlFld-ContribAuthor ">Michael D.  Threadgill</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Woodman</span>. </span><span class="cited-content_cbyCitation_article-title">Racemases and epimerases operating through a 1,1-proton transfer mechanism: reactivity, mechanism and inhibition. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>50 </em>
                                    (10)
                                     , 5952-5984. <a href="https://doi.org/10.1039/D0CS00540A" title="DOI URL">https://doi.org/10.1039/D0CS00540A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00540A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00540A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DRacemases%252Band%252Bepimerases%252Boperating%252Bthrough%252Ba%252B1%25252C1-proton%252Btransfer%252Bmechanism%25253A%252Breactivity%25252C%252Bmechanism%252Band%252Binhibition%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2021%26date%3D2021%26volume%3D50%26issue%3D10%26spage%3D5952%26epage%3D5984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lisa M.  Baker</span>, <span class="hlFld-ContribAuthor ">Anthony  Aimon</span>, <span class="hlFld-ContribAuthor ">James B.  Murray</span>, <span class="hlFld-ContribAuthor ">Allan E.  Surgenor</span>, <span class="hlFld-ContribAuthor ">Natalia  Matassova</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Roughley</span>, <span class="hlFld-ContribAuthor ">Patrick M.  Collins</span>, <span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Roderick E.  Hubbard</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid optimisation of fragments and hits to lead compounds from screening of crude reaction mixtures. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2020,</strong> <em>3 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-020-00367-0" title="DOI URL">https://doi.org/10.1038/s42004-020-00367-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-020-00367-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-020-00367-0%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DRapid%252Boptimisation%252Bof%252Bfragments%252Band%252Bhits%252Bto%252Blead%252Bcompounds%252Bfrom%252Bscreening%252Bof%252Bcrude%252Breaction%252Bmixtures%26aulast%3DBaker%26aufirst%3DLisa%2BM.%26date%3D2020%26date%3D2020%26volume%3D3%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">James S  Fraser</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of allosteric binding sites by crystallographic fragment screening. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>65 </em>, 209-216. <a href="https://doi.org/10.1016/j.sbi.2020.08.004" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.08.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.08.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DDiscovery%252Bof%252Ballosteric%252Bbinding%252Bsites%252Bby%252Bcrystallographic%252Bfragment%252Bscreening%26aulast%3DKrojer%26aufirst%3DTobias%26date%3D2020%26volume%3D65%26spage%3D209%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of phenylacetic acid <b>2</b> binding to <i>Ec</i>DsbA. (a) Binding mode of phenylacetic acid <b>2</b> to oxidized <i>Ec</i>DsbA from the X-ray crystal structure (protein databank (PDB) code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD">6WHD</a>). Carbon atoms of phenylacetic acid <b>2</b> are shown as gray sticks, oxygen atoms are shown in red, and nitrogens in blue. The binding site on <i>Ec</i>DsbA is shown as a blue-gray surface with the carbon atoms of key residues shown as green sticks and active site cysteines (Cys30 and Cys33) as yellow spheres. Red dashed lines represent hydrogen-bond and polar interactions, and black dashed lines show π-stacking interactions. (b) Structure of diaryl ether hit <b>1</b> and phenylacetic acid <b>2</b>. Affinities of <b>1</b> and <b>2</b>, given as <i>K</i><sub>D</sub> ± error of fit, were calculated from NMR data using a single-site binding model with ligand depletion. (c) <sup>1</sup>H–<sup>15</sup>N heteronuclear single quantum coherence (HSQC) spectra of oxidized <sup>15</sup>N <i>Ec</i>DsbA (100 μM) in the absence (blue) and presence (red) of phenylacetic acid <b>2</b> (1.09 mM). The inset shows a titration with phenylacetic acid <b>2</b> at concentrations of 0, 136, 273, 545, 818, and 1090 μM with contours colored blue, green, orange, cyan, purple, and red, respectively. (d) Analysis of the <sup>1</sup>H–<sup>15</sup>N HSQC titration data, plotting the concentration of <b>2</b> (μM) vs chemical shift perturbation (CSP) in ppm, observed for residues that showed CSP > 0.04 ppm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. REFiL<sub>X</sub> workflow diagram. Optimization: reaction conditions were optimized on a microscale in conjunction with optimizing X-ray soaking conditions and assessing the crystal diffraction quality. Screening: microscale reactions with a library of diverse reagents were performed in a 96-well plate; protecting groups were then cleaved and dimethyl sulfoxide (DMSO) stocks prepared. <i>Ec</i>DsbA crystals were soaked with unpurified reaction products in DMSO stocks; diffraction data were collected and processed automatically at the Australian Synchrotron. Analysis: sample purities were estimated by liquid chromatography–mass spectrometry (LCMS). Hits were identified where electron density maps additional density at the binding site that was inconsistent with the structure of the parent phenylacetic acid <b>2</b>. Confirmation: hits were resynthesized as pure compounds for full characterization and affinity determination. Pure compounds were cocrystallized with <i>Ec</i>DsbA and diffraction data measured to obtain high-resolution structures of the complexes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0004.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Hit Compounds and Control Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>,<i>N</i>-dimethylglycine hydrochloride, copper(I) iodide, Cs<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 100 °C, 18 h, 54%. (b) NaOH, MeOH, H<sub>2</sub>O, 25 °C, 3 h, 90%. (c) HATU, Et<sub>3</sub>N, amine, DMF, MeCN, 25 °C, 18 h, 30%. (d) (COCl)<sub>2</sub>, DMF cat., CDCl<sub>3</sub>, 0 °C, 5 h, then amine, Et<sub>3</sub>N, CDCl<sub>3</sub>, 25 °C, 24 h, 4–64%. Purities of library synthesized samples estimated by LCMS for hits <b>6</b> = 74%, <b>7</b> = 55%, <b>8</b> = 83%, and <b>9</b> = 62% as well as control analogues <b>10</b> = 86%, <b>11</b> = 6%, and <b>12</b> = 0%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/medium/jm0c00111_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Binding mode of pyrazole <b>9</b> to oxidized <i>Ec</i>DsbA from the X-ray crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>). Carbon atoms of pyrazole <b>9</b> are shown as gray sticks, oxygen atoms are shown in red, and nitrogens in blue. The binding site on <i>Ec</i>DsbA is shown as a blue-gray surface with the carbon atoms of key residues shown as green sticks and active site cysteines (Cys30 and Cys33) as yellow spheres. Red dashed lines represent hydrogen bond and polar interactions, and black dashed lines show π-stacking interactions. (b) Electron density map of pyrazole <b>9</b> (cocrystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>) showing the 2mFo–DFc map contoured at 1σ (blue) and the mFo–DFc maps shown at ±3σ (positive green and negative red). (c) Structure of pyrazole <b>9</b> and its affinity for <i>Ec</i>DsbA estimated from <sup>1</sup>H–<sup>15</sup>N HSQC titrations. (d) <sup>1</sup>H–<sup>15</sup>N HSQC spectra of oxidized <sup>15</sup>N <i>Ec</i>DsbA (100 μM) in the absence (blue) and presence (red) of pyrazole <b>9</b> (195 μM). The inset shows a titration with pyrazole <b>9</b> at concentrations of 0, 78, 117, 156, and 195 μM with contours colored blue, green, orange, cyan, and red, respectively. (e) Analysis of the <sup>1</sup>H–<sup>15</sup>N HSQC titration data, plotting the concentration of <b>9</b> (μM) vs chemical shift perturbation (CSP) in ppm, observed for residues that showed CSP > 0.04 ppm. Data from three replicate titration experiments are plotted as mean ± SEM, fitted to a single-site binding model accounting for ligand depletion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00111/20200701/images/large/jm0c00111_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00111&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lic5m08Hb1jtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span> <span> </span><span class="NLM_article-title">Design principles for fragment libraries: maximizing the value of learnings from pharma fragment-based drug discovery (FBDD) programs for use in academia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8189</span>– <span class="NLM_lpage">8206</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00197</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00197" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFSit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8189-8206&author=G.+M.+Keser%C5%B1author=D.+A.+Erlansonauthor=G.+G.+Ferenczyauthor=M.+M.+Hannauthor=C.+W.+Murrayauthor=S.+D.+Pickett&title=Design+principles+for+fragment+libraries%3A+maximizing+the+value+of+learnings+from+pharma+fragment-based+drug+discovery+%28FBDD%29+programs+for+use+in+academia&doi=10.1021%2Facs.jmedchem.6b00197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Design Principles for Fragment Libraries: Maximizing the Value of Learnings from Pharma Fragment-Based Drug Discovery (FBDD) Programs for Use in Academia</span></div><div class="casAuthors">Keseru, Gyorgy M.; Erlanson, Daniel A.; Ferenczy, Gyorgy G.; Hann, Michael M.; Murray, Christopher W.; Pickett, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8189-8206</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is well suited for discovering both drug leads and chem. probes of protein function; it can cover broad swaths of chem. space and allows the use of creative chem.  FBDD is widely implemented for lead discovery in industry but is sometimes used less systematically in academia.  Design principles and implementation approaches for fragment libraries are continually evolving, and the lack of up-to-date guidance may prevent more effective application of FBDD in academia.  This Perspective explores many of the theor., practical, and strategic considerations that occur within FBDD programs, including the optimal size, complexity, physicochem. profile, and shape profile of fragments in FBDD libraries, as well as compd. storage, evaluation, and screening technologies.  This compilation of industry experience in FBDD will hopefully be useful for those pursuing FBDD in academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkrMUH3yVkGbVg90H21EOLACvtfcHk0lhXNn_NlOjy2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFSit7s%253D&md5=888fe2cf003c5564c2ec391cf54ea2ac</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00197%26sid%3Dliteratum%253Aachs%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26atitle%3DDesign%2520principles%2520for%2520fragment%2520libraries%253A%2520maximizing%2520the%2520value%2520of%2520learnings%2520from%2520pharma%2520fragment-based%2520drug%2520discovery%2520%2528FBDD%2529%2520programs%2520for%2520use%2520in%2520academia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8189%26epage%3D8206%26doi%3D10.1021%2Facs.jmedchem.6b00197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, P. A.</span></span> <span> </span><span class="NLM_article-title">Off-rate screening (ORS) by surface plasmon resonance. An efficient method to kinetically sample hit to lead chemical space from unpurified reaction products</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2845</span>– <span class="NLM_lpage">2850</span>, <span class="refDoi"> DOI: 10.1021/jm401848a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401848a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2845-2850&author=J.+B.+Murrayauthor=S.+D.+Roughleyauthor=N.+Matassovaauthor=P.+A.+Brough&title=Off-rate+screening+%28ORS%29+by+surface+plasmon+resonance.+An+efficient+method+to+kinetically+sample+hit+to+lead+chemical+space+from+unpurified+reaction+products&doi=10.1021%2Fjm401848a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Off-Rate Screening (ORS) By Surface Plasmon Resonance. An Efficient Method to Kinetically Sample Hit to Lead Chemical Space from Unpurified Reaction Products</span></div><div class="casAuthors">Murray, James B.; Roughley, Stephen D.; Matassova, Natalia; Brough, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2845-2850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dissocn. rate const. kd (off-rate) is the component of ligand-protein binding with the most significant potential to enhance compd. potency.  Here we provide theor. and empirical data to show that this parameter can be detd. accurately from unpurified reaction products contg. designed test compds.  This screening protocol is amenable to parallel chem., provides efficiencies of time and materials, and complements existing methodologies for the hit-to-lead phase in fragment-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounOUxQkuyZLVg90H21EOLACvtfcHk0lhXNn_NlOjy2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVWguro%253D&md5=d3744f00df767e349024641e876e8100</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm401848a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401848a%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DBrough%26aufirst%3DP.%2BA.%26atitle%3DOff-rate%2520screening%2520%2528ORS%2529%2520by%2520surface%2520plasmon%2520resonance.%2520An%2520efficient%2520method%2520to%2520kinetically%2520sample%2520hit%2520to%2520lead%2520chemical%2520space%2520from%2520unpurified%2520reaction%2520products%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2845%26epage%3D2850%26doi%3D10.1021%2Fjm401848a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devine, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granata, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O′Reilly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Detection of ligands from a dynamic combinatorial library by X-ray crystallography</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4617</span>– <span class="NLM_lpage">4620</span>, <span class="refDoi"> DOI: 10.1002/ange.200351951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fange.200351951" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2003&pages=4617-4620&author=M.+S.+Congreveauthor=D.+J.+Davisauthor=L.+Devineauthor=C.+Granataauthor=M.+O%E2%80%B2Reillyauthor=P.+G.+Wyattauthor=H.+Jhoti&title=Detection+of+ligands+from+a+dynamic+combinatorial+library+by+X-ray+crystallography&doi=10.1002%2Fange.200351951"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fange.200351951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.200351951%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%2BS.%26aulast%3DDavis%26aufirst%3DD.%2BJ.%26aulast%3DDevine%26aufirst%3DL.%26aulast%3DGranata%26aufirst%3DC.%26aulast%3DO%25E2%2580%25B2Reilly%26aufirst%3DM.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520ligands%2520from%2520a%2520dynamic%2520combinatorial%2520library%2520by%2520X-ray%2520crystallography%26jtitle%3DAngew.%2520Chem.%26date%3D2003%26volume%3D115%26spage%3D4617%26epage%3D4620%26doi%3D10.1002%2Fange.200351951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span> <span> </span><span class="NLM_article-title">The medicinal chemist’s toolbox: an analysis of reactions used in the pursuit of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3451</span>– <span class="NLM_lpage">3479</span>, <span class="refDoi"> DOI: 10.1021/jm200187y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200187y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3451-3479&author=S.+D.+Roughleyauthor=A.+M.+Jordan&title=The+medicinal+chemist%E2%80%99s+toolbox%3A+an+analysis+of+reactions+used+in+the+pursuit+of+drug+candidates&doi=10.1021%2Fjm200187y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemist's Toolbox: An Analysis of Reactions Used in the Pursuit of Drug Candidates</span></div><div class="casAuthors">Roughley, Stephen D.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3451-3479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A survey of selected medicinal chem. literature from 2008 was undertaken in order to det. the most common chem. reactions used in the prepn. of drug candidates.  In addn. to an examn. of the reactions, the frequency of occurrence of various functional groups in the drug targets was analyzed.  The Lipinski parameters and chirality of the surveyed compds. were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTUrcluTkPBLVg90H21EOLACvtfcHk0lhXNn_NlOjy2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D&md5=a94df26d82bbd58b269ed20aa3253f56</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm200187y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200187y%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520medicinal%2520chemist%25E2%2580%2599s%2520toolbox%253A%2520an%2520analysis%2520of%2520reactions%2520used%2520in%2520the%2520pursuit%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3451%26epage%3D3479%26doi%3D10.1021%2Fjm200187y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deane, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span> <span> </span><span class="NLM_article-title">A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">15123</span> <span class="refDoi"> DOI: 10.1038/ncomms15123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fncomms15123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=28436492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A280%3ADC%252BC1crgsFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=N.+M.+Pearceauthor=T.+Krojerauthor=A.+R.+Bradleyauthor=P.+Collinsauthor=R.+P.+Nowakauthor=R.+Talonauthor=B.+D.+Marsdenauthor=S.+Kelmauthor=J.+Shiauthor=C.+M.+Deaneauthor=F.+Von+Delft&title=A+multi-crystal+method+for+extracting+obscured+crystallographic+states+from+conventionally+uninterpretable+electron+density&doi=10.1038%2Fncomms15123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density</span></div><div class="casAuthors">Pearce Nicholas M; Krojer Tobias; Bradley Anthony R; Nowak Radoslaw P; Talon Romain; Marsden Brian D; von Delft Frank; Collins Patrick; von Delft Frank; Marsden Brian D; Kelm Sebastian; Shi Jiye; Deane Charlotte M; von Delft Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15123</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In macromolecular crystallography, the rigorous detection of changed states (for example, ligand binding) is difficult unless signal is strong.  Ambiguous ('weak' or 'noisy') density is experimentally common, since molecular states are generally only fractionally present in the crystal.  Existing methodologies focus on generating maximally accurate maps whereby minor states become discernible; in practice, such map interpretation is disappointingly subjective, time-consuming and methodologically unsound.  Here we report the PanDDA method, which automatically reveals clear electron density for the changed state-even from inaccurate maps-by subtracting a proportion of the confounding 'ground state'; changed states are objectively identified from statistical analysis of density distributions.  The method is completely general, implying new best practice for all changed-state studies, including the routine collection of multiple ground-state crystals.  More generally, these results demonstrate: the incompleteness of atomic models; that single data sets contain insufficient information to model them fully; and that accuracy requires further map-deconvolution approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZZnh02fVEczAyhZbj3lKdfW6udTcc2ebJ8GcA_j2gt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crgsFWksA%253D%253D&md5=34019a4cb9081f77d9473fb0bf65a250</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncomms15123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15123%26sid%3Dliteratum%253Aachs%26aulast%3DPearce%26aufirst%3DN.%2BM.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DBradley%26aufirst%3DA.%2BR.%26aulast%3DCollins%26aufirst%3DP.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DTalon%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DKelm%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DDeane%26aufirst%3DC.%2BM.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26atitle%3DA%2520multi-crystal%2520method%2520for%2520extracting%2520obscured%2520crystallographic%2520states%2520from%2520conventionally%2520uninterpretable%2520electron%2520density%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fncomms15123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broennimann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinapoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eikenberry, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozzanica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlepütz, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmott, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, B.</span></span> <span> </span><span class="NLM_article-title">Performance of single-photon-counting PILATUS detector modules</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1107/S0909049509009911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0909049509009911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19395800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFCit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=368-375&author=P.+Kraftauthor=A.+Bergamaschiauthor=C.+Broennimannauthor=R.+Dinapoliauthor=E.+F.+Eikenberryauthor=B.+Henrichauthor=I.+Johnsonauthor=A.+Mozzanicaauthor=C.+M.+Schlep%C3%BCtzauthor=P.+R.+Willmottauthor=B.+Schmitt&title=Performance+of+single-photon-counting+PILATUS+detector+modules&doi=10.1107%2FS0909049509009911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Performance of single-photon-counting PILATUS detector modules</span></div><div class="casAuthors">Kraft, P.; Bergamaschi, A.; Broennimann, Ch.; Dinapoli, R.; Eikenberry, E. F.; Henrich, B.; Johnson, I.; Mozzanica, A.; Schlepuetz, C. M.; Willmott, P. R.; Schmitt, B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">368-375</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">PILATUS is a silicon hybrid pixel detector system, operating in single-photon-counting mode, that has been developed at the Paul Scherrer Institut for the needs of macromol. crystallog. at the Swiss Light Source (SLS).  A calibrated PILATUS module has been characterized with monochromatic synchrotron radiation.  The influence of charge sharing on the count rate and the overall energy resoln. of the detector were investigated.  The dead-time of the system was detd. using the attenuated direct synchrotron beam.  A single module detector was also tested in surface diffraction expts. at the SLS, whereby its performance regarding fluorescence suppression and satn. tolerance were evaluated, and have shown to greatly improve the sensitivity, reliability and speed of surface diffraction data acquisition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5mI0pKAfFi7Vg90H21EOLACvtfcHk0li1G6sfEADxIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFCit7w%253D&md5=256106f0fb728f37eb3702623e613d66</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1107%2FS0909049509009911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049509009911%26sid%3Dliteratum%253Aachs%26aulast%3DKraft%26aufirst%3DP.%26aulast%3DBergamaschi%26aufirst%3DA.%26aulast%3DBroennimann%26aufirst%3DC.%26aulast%3DDinapoli%26aufirst%3DR.%26aulast%3DEikenberry%26aufirst%3DE.%2BF.%26aulast%3DHenrich%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DMozzanica%26aufirst%3DA.%26aulast%3DSchlep%25C3%25BCtz%26aufirst%3DC.%2BM.%26aulast%3DWillmott%26aufirst%3DP.%2BR.%26aulast%3DSchmitt%26aufirst%3DB.%26atitle%3DPerformance%2520of%2520single-photon-counting%2520PILATUS%2520detector%2520modules%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2009%26volume%3D16%26spage%3D368%26epage%3D375%26doi%3D10.1107%2FS0909049509009911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinapoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horisberger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozzanica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theidel, G.</span></span> <span> </span><span class="NLM_article-title">EIGER: Next generation single photon counting detector for X-ray applications</span>. <i>Nucl. Instrum. Methods Phys. Res., Sect. A</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>650</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.nima.2010.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1016%2Fj.nima.2010.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2itLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=650&publication_year=2011&pages=79-83&author=R.+Dinapoliauthor=A.+Bergamaschiauthor=B.+Henrichauthor=R.+Horisbergerauthor=I.+Johnsonauthor=A.+Mozzanicaauthor=E.+Schmidauthor=B.+Schmittauthor=A.+Schreiberauthor=X.+Shiauthor=G.+Theidel&title=EIGER%3A+Next+generation+single+photon+counting+detector+for+X-ray+applications&doi=10.1016%2Fj.nima.2010.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">EIGER: Next generation single photon counting detector for X-ray applications</span></div><div class="casAuthors">Dinapoli, Roberto; Bergamaschi, Anna; Henrich, Beat; Horisberger, Roland; Johnson, Ian; Mozzanica, Aldo; Schmid, Elmar; Schmitt, Bernd; Schreiber, Akos; Shi, Xin-Tian; Theidel, Gerd</div><div class="citationInfo"><span class="NLM_cas:title">Nuclear Instruments & Methods in Physics Research, Section A: Accelerators, Spectrometers, Detectors, and Associated Equipment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">650</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-83</span>CODEN:
                <span class="NLM_cas:coden">NIMAER</span>;
        ISSN:<span class="NLM_cas:issn">0168-9002</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">EIGER is an advanced family of single photon counting hybrid pixel detectors, primarily aimed at diffraction expts. at synchrotrons.  Optimization of maximal functionality and minimal pixel size (using a 0.25μm process and conserving the radiation tolerant design) has resulted in 75×75μm2 pixels.  Every pixel comprises a preamplifier, shaper, discriminator (with a 6 bit DAC for threshold trimming), a configurable 4/8/12 bit counter with double buffering, as well as readout, control and test circuitry.  A novel feature of this chip is its double buffered counter, meaning a next frame can be acquired while the previous one is being readout.  An array of 256×256 pixels fits on a ∼2×2cm2 chip and a sensor of ∼8×4cm2 will be equipped with eight readout chips to form a module contg. 0.5 Mpixel.  Several modules can then be tiled to form larger area detectors.  Detectors up to 4×8 modules (16 Mpixel) are planned.  To achieve frame rates of up to 24 kHz the readout architecture is highly parallel, and the chip readout happens in parallel on 32 readout lines with a 100 MHz Double Data Rate clock.  Several chips and singles (i.e. a single chip bump-bonded to a single chip silicon sensor) were tested both with a lab X-ray source and at Swiss Light Source (SLS) beamlines.  These tests demonstrate the full functionality of the chip and provide a first assessment of its performance.  High resoln. X-ray images and "high speed movies" were produced, even without threshold trimming, at the target system frame rates (up to ∼24kHz in 4 bit mode).  In parallel, dedicated hardware, firmware and software had to be developed to comply with the enormous data rate the chip is capable of delivering.  Details of the chip design and tests will be given, as well as highlights of both test and final readout systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFJqLxbmJ1bVg90H21EOLACvtfcHk0li1G6sfEADxIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2itLvE&md5=a8051a572662777dd0bff189315d2dc8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.nima.2010.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nima.2010.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DDinapoli%26aufirst%3DR.%26aulast%3DBergamaschi%26aufirst%3DA.%26aulast%3DHenrich%26aufirst%3DB.%26aulast%3DHorisberger%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DMozzanica%26aufirst%3DA.%26aulast%3DSchmid%26aufirst%3DE.%26aulast%3DSchmitt%26aufirst%3DB.%26aulast%3DSchreiber%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DTheidel%26aufirst%3DG.%26atitle%3DEIGER%253A%2520Next%2520generation%2520single%2520photon%2520counting%2520detector%2520for%2520X-ray%2520applications%26jtitle%3DNucl.%2520Instrum.%2520Methods%2520Phys.%2520Res.%252C%2520Sect.%2520A%26date%3D2011%26volume%3D650%26spage%3D79%26epage%3D83%26doi%3D10.1016%2Fj.nima.2010.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohanty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilyichova, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcair, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totsika, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caria, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouldice, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headey, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Application of fragment-based screening to the design of inhibitors of <i>Escherichia coli</i> DsbA</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2179</span>– <span class="NLM_lpage">2184</span>, <span class="refDoi"> DOI: 10.1002/anie.201410341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fanie.201410341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFM%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=2179-2184&author=L.+A.+Adamsauthor=P.+Sharmaauthor=B.+Mohantyauthor=O.+V.+Ilyichovaauthor=M.+D.+Mulcairauthor=M.+L.+Williamsauthor=E.+C.+Gleesonauthor=M.+Totsikaauthor=B.+C.+Doakauthor=S.+Cariaauthor=K.+Rimmerauthor=J.+Horneauthor=S.+R.+Shouldiceauthor=M.+Vaziraniauthor=S.+J.+Headeyauthor=B.+R.+Plumbauthor=J.+L.+Martinauthor=B.+Herasauthor=J.+S.+Simpsonauthor=M.+J.+Scanlon&title=Application+of+fragment-based+screening+to+the+design+of+inhibitors+of+Escherichia+coli+DsbA&doi=10.1002%2Fanie.201410341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment-Based Screening to the Design of Inhibitors of Escherichia coli DsbA</span></div><div class="casAuthors">Adams, Luke A.; Sharma, Pooja; Mohanty, Biswaranjan; Ilyichova, Olga V.; Mulcair, Mark D.; Williams, Martin L.; Gleeson, Ellen C.; Totsika, Makrina; Doak, Bradley C.; Caria, Sofia; Rimmer, Kieran; Horne, James; Shouldice, Stephen R.; Vazirani, Mansha; Headey, Stephen J.; Plumb, Brent R.; Martin, Jennifer L.; Heras, Begona; Simpson, Jamie S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2179-2184</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The thiol-disulfide oxidoreductase enzyme DsbA catalyzes the formation of disulfide bonds in the periplasm of Gram-neg. bacteria.  DsbA substrates include proteins involved in bacterial virulence.  In the absence of DsbA, many of these proteins do not fold correctly, which renders the bacteria avirulent.  Thus DsbA is a crit. mediator of virulence and inhibitors may act as antivirulence agents.  Biophys. screening has been employed to identify fragments that bind to DsbA from Escherichia coli.  Elaboration of one of these fragments produced compds. that inhibit DsbA activity in vitro.  In cell-based assays, the compds. inhibit bacterial motility, but have no effect on growth in liq. culture, which is consistent with selective inhibition of DsbA.  Crystal structures of inhibitors bound to DsbA indicate that they bind adjacent to the active site.  Together, the data suggest that DsbA may be amenable to the development of novel antibacterial compds. that act by inhibiting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv5mTk2z8YLbVg90H21EOLACvtfcHk0lh2VJkRwxlGOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFM%253D&md5=dd5b947950fdacecf6a4dff69c59fc24</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fanie.201410341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201410341%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DL.%2BA.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DMohanty%26aufirst%3DB.%26aulast%3DIlyichova%26aufirst%3DO.%2BV.%26aulast%3DMulcair%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DM.%2BL.%26aulast%3DGleeson%26aufirst%3DE.%2BC.%26aulast%3DTotsika%26aufirst%3DM.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DCaria%26aufirst%3DS.%26aulast%3DRimmer%26aufirst%3DK.%26aulast%3DHorne%26aufirst%3DJ.%26aulast%3DShouldice%26aufirst%3DS.%2BR.%26aulast%3DVazirani%26aufirst%3DM.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DPlumb%26aufirst%3DB.%2BR.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DApplication%2520of%2520fragment-based%2520screening%2520to%2520the%2520design%2520of%2520inhibitors%2520of%2520Escherichia%2520coli%2520DsbA%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D2179%26epage%3D2184%26doi%3D10.1002%2Fanie.201410341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouldice, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totsika, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schembri, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">DSB proteins and bacterial pathogenicity</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrmicro2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19198617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFeqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=215-225&author=B.+Herasauthor=S.+R.+Shouldiceauthor=M.+Totsikaauthor=M.+J.+Scanlonauthor=M.+A.+Schembriauthor=J.+L.+Martin&title=DSB+proteins+and+bacterial+pathogenicity&doi=10.1038%2Fnrmicro2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DSB proteins and bacterial pathogenicity</span></div><div class="casAuthors">Heras, Begona; Shouldice, Stephen R.; Totsika, Makrina; Scanlon, Martin J.; Schembri, Mark A.; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-225</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  If DNA is the information of life, then proteins are the machines of life, but they must be assembled and correctly folded to function.  A key step in the protein-folding pathway is the introduction of disulfide bonds between Cys residues in a process called oxidative protein folding.  Many bacteria use an oxidative protein folding machinery to assemble disulfide bond-contg. proteins (DSB proteins) that are essential for cell integrity and to produce virulence factors.  Although current knowledge of this machinery stems largely from Escherichia coli K-12, this view must now be adjusted to encompass the wider range of disulfide catalytic systems present in bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp-m_RBloHOLVg90H21EOLACvtfcHk0lh2VJkRwxlGOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFeqsbc%253D&md5=d75810468658d9f5c00da4f54a6e3e24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2087%26sid%3Dliteratum%253Aachs%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DShouldice%26aufirst%3DS.%2BR.%26aulast%3DTotsika%26aufirst%3DM.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DSchembri%26aufirst%3DM.%2BA.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DDSB%2520proteins%2520and%2520bacterial%2520pathogenicity%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D215%26epage%3D225%26doi%3D10.1038%2Fnrmicro2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Targeting virulence not viability in the search for future antibacterials</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1111/bcp.12356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1111%2Fbcp.12356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=24552512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2015&pages=208-215&author=B.+Herasauthor=M.+J.+Scanlonauthor=J.+L.+Martin&title=Targeting+virulence+not+viability+in+the+search+for+future+antibacterials&doi=10.1111%2Fbcp.12356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting virulence not viability in the search for future antibacterials</span></div><div class="casAuthors">Heras, Begona; Scanlon, Martin J.; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-215</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">1365-2125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  New antibacterials need new approaches to overcome the problem of rapid antibiotic resistance.  Here we review the development of potential new antibacterial drugs that do not kill bacteria or inhibit their growth, but combat disease instead by targeting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR8H_GpurAKLVg90H21EOLACvtfcHk0lh2VJkRwxlGOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCrtbo%253D&md5=60b729ec4aad3b5b22d3a0f150a498bb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12356%26sid%3Dliteratum%253Aachs%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DTargeting%2520virulence%2520not%2520viability%2520in%2520the%2520search%2520for%2520future%2520antibacterials%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2015%26volume%3D79%26spage%3D208%26epage%3D215%26doi%3D10.1111%2Fbcp.12356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0ljeLvbMZzL2Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phizackerley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0ljeLvbMZzL2Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aragao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrop, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T.</span></span> <span> </span><span class="NLM_article-title">MX1: a bending-magnet crystallography beamline serving both chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/S1600577514021717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS1600577514021717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=25537608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFeitLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=187-190&author=N.+P.+Cowiesonauthor=D.+Aragaoauthor=M.+Cliftauthor=D.+J.+Ericssonauthor=C.+Geeauthor=S.+J.+Harropauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Williamsonauthor=T.+Caradoc-Davies&title=MX1%3A+a+bending-magnet+crystallography+beamline+serving+both+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2FS1600577514021717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MX1: a bending-magnet crystallography beamline serving both chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Cowieson, Nathan Philip; Aragao, David; Clift, Mark; Ericsson, Daniel J.; Gee, Christine; Harrop, Stephen J.; Mudie, Nathan; Panjikar, Santosh; Price, Jason R.; Riboldi-Tunnicliffe, Alan; Williamson, Rachel; Caradoc-Davies, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-190</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX1 is a bending-magnet crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range from 8 to 18 keV to a focal spot at the sample position of 120 μm FWHM.  The beamline end station and ancillary equipment facilitate local and remote access for both chem. and biol. macromol. crystallog.  Here, the design of the beamline and end station are discussed.  The beamline has enjoyed a full user program for the last seven years and scientific highlights from the user program are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonncGct23S27Vg90H21EOLACvtfcHk0ljeLvbMZzL2Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFeitLrO&md5=2f00508e17ab0c8584519b30c3f0505b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS1600577514021717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577514021717%26sid%3Dliteratum%253Aachs%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DAragao%26aufirst%3DD.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%26aulast%3DHarrop%26aufirst%3DS.%2BJ.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%26atitle%3DMX1%253A%2520a%2520bending-magnet%2520crystallography%2520beamline%2520serving%2520both%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2015%26volume%3D22%26spage%3D187%26epage%3D190%26doi%3D10.1107%2FS1600577514021717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aragão, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukuvada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span> <span> </span><span class="NLM_article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1107/S1600577518003120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS1600577518003120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=29714201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=885-891&author=D.+Arag%C3%A3oauthor=J.+Aishimaauthor=H.+Cherukuvadaauthor=R.+Clarkenauthor=M.+Cliftauthor=N.+P.+Cowiesonauthor=D.+J.+Ericssonauthor=C.+L.+Geeauthor=S.+Macedoauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Rostanauthor=R.+Williamsonauthor=T.+T.+Caradoc-Davies&title=MX2%3A+a+high-flux+undulator+microfocus+beamline+serving+both+the+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2FS1600577518003120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Aragao, David; Aishima, Jun; Cherukuvada, Hima; Clarken, Robert; Clift, Mark; Cowieson, Nathan Philip; Ericsson, Daniel Jesper; Gee, Christine L.; Macedo, Sofia; Mudie, Nathan; Panjikar, Santosh; Price, Jason Roy; Riboldi-Tunnicliffe, Alan; Rostan, Robert; Williamson, Rachel; Caradoc-Davies, Thomas Tudor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">885-891</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX2 is an in-vacuum undulator-based crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range 4.8-21 keV to a focal spot of 22 × 12 μm FWHM (H × V).  At 13 keV the flux at the sample is 3.4 × 1012 photons s-1.  The beamline endstation allows robotic handling of cryogenic samples via an updated SSRL SAM robot.  This beamline is ideal for weakly diffracting hard-to-crystallize proteins, virus particles, protein assemblies and nucleic acids as well as smaller mols. such as inorg. catalysts and org. drug mols.  The beamline is now mature and has enjoyed a full user program for the last nine years.  This paper describes the beamline status, plans for its future and some recent scientific highlights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgtd7iUcXqY7Vg90H21EOLACvtfcHk0lg1wxhMmTa4cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D&md5=8a52a2db9f5d01958df27ae74e2496d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1107%2FS1600577518003120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577518003120%26sid%3Dliteratum%253Aachs%26aulast%3DArag%25C3%25A3o%26aufirst%3DD.%26aulast%3DAishima%26aufirst%3DJ.%26aulast%3DCherukuvada%26aufirst%3DH.%26aulast%3DClarken%26aufirst%3DR.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BL.%26aulast%3DMacedo%26aufirst%3DS.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DRostan%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26atitle%3DMX2%253A%2520a%2520high-flux%2520undulator%2520microfocus%2520beamline%2520serving%2520both%2520the%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2018%26volume%3D25%26spage%3D885%26epage%3D891%26doi%3D10.1107%2FS1600577518003120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lg1wxhMmTa4cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lg1wxhMmTa4cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Legrand, P.</span></span> <i>XDSME: XDS Made Easier</i>. GitHub Repository, <span class="NLM_year">2017</span>, <a href="https://github.com/legrandp/xdsme" class="extLink">https://github.com/legrandp/xdsme</a>. <span class="refDoi"> DOI: 10.5281/zenodo.837885</span>  (accessed April 18, 2018).</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.5281%2Fzenodo.837885" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+Legrand&title=XDSME%3A+XDS+Made+Easier"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5281%2Fzenodo.837885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.5281%252Fzenodo.837885%26sid%3Dliteratum%253Aachs%26aulast%3DLegrand%26aufirst%3DP.%26jtitle%3DXDSME%253A%2520XDS%2520Made%2520Easier%26date%3D2017%26doi%3D10.5281%2Fzenodo.837885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">Auto-rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1107/S0907444905001307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444905001307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15805600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=449-457&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=Auto-rickshaw%3A+an+automated+crystal+structure+determination+platform+as+an+efficient+tool+for+the+validation+of+an+X-ray+diffraction+experiment&doi=10.1107%2FS0907444905001307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-457</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The EMBL-Hamburg Automated Crystal Structure Detn. Platform is a system that combines a no. of existing macromol. crystallog. computer programs and several decision-makers into a software pipeline for automated and efficient crystal structure detn.  The pipeline can be invoked as soon as x-ray data from derivatized protein crystals have been collected and processed.  It is controlled by a web-based graphical user interface for data and parameter input, and for monitoring the progress of structure detn.  A large no. of possible structure-soln. paths are encoded in the system and the optimal path is selected by the decision-makers as the structure soln. evolves.  The processes have been optimized for speed so that the pipeline can be used effectively for validating the x-ray expt. at a synchrotron beamline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZbIxeGX0aqrVg90H21EOLACvtfcHk0lgEqUtBgCqWpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D&md5=98b93bf2e456104466b2e7b07e3ac197</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001307%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DAuto-rickshaw%253A%2520an%2520automated%2520crystal%2520structure%2520determination%2520platform%2520as%2520an%2520efficient%2520tool%2520for%2520the%2520validation%2520of%2520an%2520X-ray%2520diffraction%2520experiment%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D449%26epage%3D457%26doi%3D10.1107%2FS0907444905001307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909029643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19770506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1089-1097&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=On+the+combination+of+molecular+replacement+and+single-wavelength+anomalous+diffraction+phasing+for+automated+structure+determination&doi=10.1107%2FS0907444909029643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1097</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A combination of mol. replacement and single-wavelength anomalous diffraction phasing has been incorporated into the automated structure-detn. platform Auto-Rickshaw.  The complete MRSAD procedure includes mol. replacement, model refinement, exptl. phasing, phase improvement and automated model building.  The improvement over the std. SAD or MR approaches is illustrated by ten test cases taken from the JCSG diffraction data-set database.  Poor MR or SAD phases with phase errors larger than 70° can be improved using the described procedure and a large fraction of the model can be detd. in a purely automatic manner from X-ray data extending to better than 2.6 Å resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz21EsxIkqJLVg90H21EOLACvtfcHk0lgEqUtBgCqWpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP&md5=e3629b21fee4dc43450e853417e6b598</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029643%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DOn%2520the%2520combination%2520of%2520molecular%2520replacement%2520and%2520single-wavelength%2520anomalous%2520diffraction%2520phasing%2520for%2520automated%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1089%26epage%3D1097%26doi%3D10.1107%2FS0907444909029643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulombe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordeleau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, P. L.</span></span> <span> </span><span class="NLM_article-title">Compound aggregation in drug discovery: implementing a practical NMR assay for medicinal chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5142</span>– <span class="NLM_lpage">5150</span>, <span class="refDoi"> DOI: 10.1021/jm400535b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400535b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslSqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5142-5150&author=S.+R.+LaPlanteauthor=R.+Carsonauthor=J.+Gillardauthor=N.+Aubryauthor=R.+Coulombeauthor=S.+Bordeleauauthor=P.+Bonneauauthor=M.+Littleauthor=J.+O%E2%80%99Mearaauthor=P.+L.+Beaulieu&title=Compound+aggregation+in+drug+discovery%3A+implementing+a+practical+NMR+assay+for+medicinal+chemists&doi=10.1021%2Fjm400535b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Compound Aggregation in Drug Discovery: Implementing a Practical NMR Assay for Medicinal Chemists</span></div><div class="casAuthors">LaPlante, Steven R.; Carson, Rebekah; Gillard, James; Aubry, Norman; Coulombe, Rene; Bordeleau, Sylvain; Bonneau, Pierre; Little, Michael; O'Meara, Jeff; Beaulieu, Pierre L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5142-5150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmaceutical industry has recognized that many drug-like mols. can self-aggregate in aq. media and have physicochem. properties that skew exptl. results and decisions.  Herein, we introduce the use of a simple NMR strategy for detecting the formation of aggregates using diln. expts. that can be performed on equipment prevalent in most synthetic chem. departments.  We show that 1H NMR resonances are sensitive to large mol.-size entities and to smaller multimers and mixts. of species.  Practical details are provided for sample prepn. and for detg. the concns. of single mol., aggregate entities, and ppt.  The crit. concns. above which aggregation begins can be found and were corroborated by comparisons with light scattering techniques.  Disaggregation can also be monitored using detergents.  This NMR assay should serve as a practical and readily available tool for medicinal chemists to better characterize how their compds. behave in aq. media and influence drug design decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAjZBdJ970nrVg90H21EOLACvtfcHk0lgEqUtBgCqWpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslSqt7w%253D&md5=ba5e7894d6080752dc4c1bfb2ed38f73</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm400535b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400535b%26sid%3Dliteratum%253Aachs%26aulast%3DLaPlante%26aufirst%3DS.%2BR.%26aulast%3DCarson%26aufirst%3DR.%26aulast%3DGillard%26aufirst%3DJ.%26aulast%3DAubry%26aufirst%3DN.%26aulast%3DCoulombe%26aufirst%3DR.%26aulast%3DBordeleau%26aufirst%3DS.%26aulast%3DBonneau%26aufirst%3DP.%26aulast%3DLittle%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DJ.%26aulast%3DBeaulieu%26aufirst%3DP.%2BL.%26atitle%3DCompound%2520aggregation%2520in%2520drug%2520discovery%253A%2520implementing%2520a%2520practical%2520NMR%2520assay%2520for%2520medicinal%2520chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5142%26epage%3D5150%26doi%3D10.1021%2Fjm400535b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudelman, A.</span></span> <span> </span><span class="NLM_article-title">NMR chemical shifts of common laboratory solvents as trace impurities</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7512</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jo971176v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+chemical+shifts+of+common+laboratory+solvents+as+trace+impurities&doi=10.1021%2Fjo971176v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0lgEqUtBgCqWpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520chemical%2520shifts%2520of%2520common%2520laboratory%2520solvents%2520as%2520trace%2520impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515%26doi%3D10.1021%2Fjo971176v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Studier, F. W.</span></span> <span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span>. <i>Protein Expr. Purif.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0lgUf-K4Ce5u6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expr.%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paxman, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borg, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahler, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakellaris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossjohn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">The structure of the bacterial oxidoreductase enzyme DsbA in complex with a peptide reveals a basis for substrate specificity in the catalytic cycle of DsbA enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">17835</span>– <span class="NLM_lpage">17845</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.011502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1074%2Fjbc.M109.011502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19389711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17835-17845&author=J.+J.+Paxmanauthor=N.+A.+Borgauthor=J.+Horneauthor=P.+E.+Thompsonauthor=Y.+Chinauthor=P.+Sharmaauthor=J.+S.+Simpsonauthor=J.+Wielensauthor=S.+Piekauthor=C.+M.+Kahlerauthor=H.+Sakellarisauthor=M.+Pearceauthor=S.+P.+Bottomleyauthor=J.+Rossjohnauthor=M.+J.+Scanlon&title=The+structure+of+the+bacterial+oxidoreductase+enzyme+DsbA+in+complex+with+a+peptide+reveals+a+basis+for+substrate+specificity+in+the+catalytic+cycle+of+DsbA+enzymes&doi=10.1074%2Fjbc.M109.011502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Structure of the Bacterial Oxidoreductase Enzyme DsbA in Complex with a Peptide Reveals a Basis for Substrate Specificity in the Catalytic Cycle of DsbA Enzymes</span></div><div class="casAuthors">Paxman, Jason J.; Borg, Natalie A.; Horne, James; Thompson, Philip E.; Chin, Yanni; Sharma, Pooja; Simpson, Jamie S.; Wielens, Jerome; Piek, Susannah; Kahler, Charlene M.; Sakellaris, Harry; Pearce, Mary; Bottomley, Stephen P.; Rossjohn, Jamie; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17835-17845</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Oxidative protein folding in Gram-neg. bacteria results in the formation of disulfide bonds between pairs of cysteine residues.  This is a multistep process in which the dithiol-disulfide oxidoreductase enzyme, DsbA, plays a central role.  The structure of DsbA comprises an all helical domain of unknown function and a thioredoxin domain, where active site cysteines shuttle between an oxidized, substrate-bound, reduced form and a DsbB-bound form, where DsbB is a membrane protein that reoxidizes DsbA.  Most DsbA enzymes interact with a wide variety of reduced substrates and show little specificity.  However, a no. of DsbA enzymes have now been identified that have narrow substrate repertoires and appear to interact specifically with a smaller no. of substrates.  The transient nature of the DsbA-substrate complex has hampered our understanding of the factors that govern the interaction of DsbA enzymes with their substrates.  Here we report the crystal structure of a complex between Escherichia coli DsbA and a peptide with a sequence derived from a substrate.  The binding site identified in the DsbA-peptide complex was distinct from that obsd. for DsbB in the DsbA-DsbB complex.  The structure revealed details of the DsbA-peptide interaction and suggested a mechanism by which DsbA can simultaneously show broad specificity for substrates yet exhibit specificity for DsbB.  This mode of binding was supported by soln. NMR data as well as functional data, which demonstrated that the substrate specificity of DsbA could be modified via changes at the binding interface identified in the structure of the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLyWFqwOudmrVg90H21EOLACvtfcHk0lgUf-K4Ce5u6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksbk%253D&md5=5de5d0259663de920a057589fc980ddb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.011502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.011502%26sid%3Dliteratum%253Aachs%26aulast%3DPaxman%26aufirst%3DJ.%2BJ.%26aulast%3DBorg%26aufirst%3DN.%2BA.%26aulast%3DHorne%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DP.%2BE.%26aulast%3DChin%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DPiek%26aufirst%3DS.%26aulast%3DKahler%26aufirst%3DC.%2BM.%26aulast%3DSakellaris%26aufirst%3DH.%26aulast%3DPearce%26aufirst%3DM.%26aulast%3DBottomley%26aufirst%3DS.%2BP.%26aulast%3DRossjohn%26aufirst%3DJ.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520structure%2520of%2520the%2520bacterial%2520oxidoreductase%2520enzyme%2520DsbA%2520in%2520complex%2520with%2520a%2520peptide%2520reveals%2520a%2520basis%2520for%2520substrate%2520specificity%2520in%2520the%2520catalytic%2520cycle%2520of%2520DsbA%2520enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17835%26epage%3D17845%26doi%3D10.1074%2Fjbc.M109.011502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waksman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, J. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Crystallization of DsbA, an <i>Escherichia coli</i> protein required for disulphide bond formation in vivo</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>230</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1006%2Fjmbi.1993.1226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=8478925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1eisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=230&publication_year=1993&pages=1097-1100&author=J.+L.+Martinauthor=G.+Waksmanauthor=J.+C.+A.+Bardwellauthor=J.+Beckwithauthor=J.+Kuriyan&title=Crystallization+of+DsbA%2C+an+Escherichia+coli+protein+required+for+disulphide+bond+formation+in+vivo&doi=10.1006%2Fjmbi.1993.1226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization of DsbA, an Escherichia coli protein required for disulfide bond formation in vivo</span></div><div class="casAuthors">Martin, Jennifer L.; Waksman, Gabriel; Bardwell, James C. A.; Beckwith, Jon; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">230</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1097-100</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">DsbA is a 21 kDa protein that facilitates disulfide bond formation and is required for the correct folding and stability of a no. of exported proteins in E. coli.  Crystals of oxidized DsbA have been obtained from polyethylene glycol 8000 (20 to 25%), 0.1 M-cacodylate buffer (pH 6.5) and 1% 2-methyl-2,4-pentanediol.  Oxidn. of the protein is crit. for reproducibly obtaining high quality crystals.  The resulting crystals diffract to 2 Å and belong to the monoclinic space group C2 with cell dimensions a = 117.5 Å, b = 65.0 Å, c = 76.3 Å, β = 126.3° with two mols. in the asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC5XXZuGR-e7Vg90H21EOLACvtfcHk0lgUf-K4Ce5u6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1eisrk%253D&md5=c011dbb251518ed1d404518b1e301ca0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1226%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DWaksman%26aufirst%3DG.%26aulast%3DBardwell%26aufirst%3DJ.%2BC.%2BA.%26aulast%3DBeckwith%26aufirst%3DJ.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystallization%2520of%2520DsbA%252C%2520an%2520Escherichia%2520coli%2520protein%2520required%2520for%2520disulphide%2520bond%2520formation%2520in%2520vivo%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D230%26spage%3D1097%26epage%3D1100%26doi%3D10.1006%2Fjmbi.1993.1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span> <span> </span><span class="NLM_article-title">Linking crystallographic model and data quality</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1218231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1126%2Fscience.1218231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=22628654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1030-1033&author=P.+A.+Karplusauthor=K.+Diederichs&title=Linking+crystallographic+model+and+data+quality&doi=10.1126%2Fscience.1218231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Linking Crystallographic Model and Data Quality</span></div><div class="casAuthors">Karplus, P. Andrew; Diederichs, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">1030-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In macromol. x-ray crystallog., refinement R values measure the agreement between obsd. and calcd. data.  Analogously, Rmerge values reporting on the agreement between multiple measurements of a given reflection are used to assess data quality.  Here, we show that despite their widespread use, Rmerge values are poorly suited for detg. the high-resoln. limit and that current std. protocols discard much useful data.  We introduce a statistic that ests. the correlation of an obsd. data set with the underlying (not measurable) true signal; this quantity, CC*, provides a single statistically valid guide for deciding which data are useful.  CC* also can be used to assess model and data quality on the same scale, and this reveals when data quality is limiting model improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3BGdcdUzbVg90H21EOLACvtfcHk0lhUMZ68enNgUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D&md5=b8ee6edacea44364cc66c44dd3a505fd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1218231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1218231%26sid%3Dliteratum%253Aachs%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26aulast%3DDiederichs%26aufirst%3DK.%26atitle%3DLinking%2520crystallographic%2520model%2520and%2520data%2520quality%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1030%26epage%3D1033%26doi%3D10.1126%2Fscience.1218231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glockshuber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber-Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of three His32 mutants of DsbA: support for an electrostatic role of His32 in DsbA stability</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1893</span>– <span class="NLM_lpage">1900</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1002%2Fpro.5560060910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=9300489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFert7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=1893-1900&author=L.+W.+Guddatauthor=J.+C.+Bardwellauthor=R.+Glockshuberauthor=M.+Huber-Wunderlichauthor=T.+Zanderauthor=J.+L.+Martin&title=Structural+analysis+of+three+His32+mutants+of+DsbA%3A+support+for+an+electrostatic+role+of+His32+in+DsbA+stability&doi=10.1002%2Fpro.5560060910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of three His32 mutants of DsbA: support for an electrostatic role of His32 in DsbA stability</span></div><div class="casAuthors">Guddat, Luke W.; Bardwell, James C. A.; Glockshuber, Rudi; Huber-Wunderlich, Martina; Zander, Thomas; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1893-1900</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">DsbA, a 21-kDa protein from Escherichia coli, is a potent oxidizing disulfide catalyst required for disulfide bond formation in secreted proteins.  The active site of DsbA is similar to that of mammalian protein disulfide isomerases, and includes a reversible disulfide bond formed from cysteines sepd. by two residues (Cys30-Pro31-His32-Cys33).  Unlike most protein disulfides, the active-site disulfide of DsbA is highly reactive and the oxidized form of DsbA is much less stable than the reduced form at physiol. pH.  His32, one of the two residues between the active-site cysteines, is crit. to the oxidizing power of DsbA and to the relative instability of the protein in the oxidized form.  Mutation of this single residue to tyrosine, serine, or leucine results in a significant increase in stability (of ∼5-7 kcal/mol) of the oxidized His32 variants relative to the oxidized wild-type protein.  Despite the dramatic changes in stability, the structures of all three oxidized DsbA His32 variants are very similar to the wild-type oxidized structure, including conservation of solvent atoms near the active-site residue, Cys30.  These results show that the His32 residue does not exert a conformational effect on the structure of DsbA.  The destabilizing effect of His32 on oxidized DsbA is therefore most likely electrostatic in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooscLQFcNL7bVg90H21EOLACvtfcHk0lhUMZ68enNgUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFert7o%253D&md5=2a48236ec9e165e1bcd06b961b0ada46</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060910%26sid%3Dliteratum%253Aachs%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DBardwell%26aufirst%3DJ.%2BC.%26aulast%3DGlockshuber%26aufirst%3DR.%26aulast%3DHuber-Wunderlich%26aufirst%3DM.%26aulast%3DZander%26aufirst%3DT.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DStructural%2520analysis%2520of%2520three%2520His32%2520mutants%2520of%2520DsbA%253A%2520support%2520for%2520an%2520electrostatic%2520role%2520of%2520His32%2520in%2520DsbA%2520stability%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D1893%26epage%3D1900%26doi%3D10.1002%2Fpro.5560060910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lhUMZ68enNgUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR system: A new software suite for macromolecular structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2earb8ErX53L2bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520system%253A%2520A%2520new%2520software%2520suite%2520for%2520macromolecular%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgkdffgBVV1fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammito, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">861</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1107/S2059798319011471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS2059798319011471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2019&pages=861-877&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=M.+L.+Bakerauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=T.+I.+Crollauthor=B.+Hintzeauthor=L.+W.+Hungauthor=S.+Jainauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=B.+K.+Poonauthor=M.+G.+Prisantauthor=R.+J.+Readauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=M.+D.+Sammitoauthor=O.+V.+Sobolevauthor=D.+H.+Stockwellauthor=T.+C.+Terwilligerauthor=A.+G.+Urzhumtsevauthor=L.+L.+Videauauthor=C.+J.+Williamsauthor=P.+D.+Adams&title=Macromolecular+structure+determination+using+X-rays%2C+neutrons+and+electrons%3A+recent+developments+in+Phenix&doi=10.1107%2FS2059798319011471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Baker, Matthew L.; Bunkoczi, Gabor; Chen, Vincent B.; Croll, Tristan I.; Hintze, Bradley; Hung, Li-Wei; Jain, Swati; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Poon, Billy K.; Prisant, Michael G.; Read, Randy J.; Richardson, Jane S.; Richardson, David C.; Sammito, Massimo D.; Sobolev, Oleg V.; Stockwell, Duncan H.; Terwilliger, Thomas C.; Urzhumtsev, Alexandre G.; Videau, Lizbeth L.; Williams, Christopher J.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">861-877</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Diffraction (X-ray, neutron and electron) and electron cryo-microscopy are powerful methods to det. three-dimensional macromol. structures, which are required to understand biol. processes and to develop new therapeutics against diseases.  The overall structure-soln. workflow is similar for these techniques, but nuances exist because the properties of the reduced exptl. data are different.  Software tools for structure detn. should therefore be tailored for each method.  Phenix is a comprehensive software package for macromol. structure detn. that handles data from any of these techniques.  Tasks performed with Phenix include data-quality assessment, map improvement, model building, the validation/rebuilding/refinement cycle and deposition.  Each tool caters to the type of exptl. data.  The design of Phenix emphasizes the automation of procedures, where possible, to minimize repetitive and time-consuming manual tasks, while default parameters are chosen to encourage best practice.  A graphical user interface provides access to many command-line features of Phenix and streamlines the transition between programs, project tracking and re-running of previous tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolHnU8ECe2CrVg90H21EOLACvtfcHk0lgkdffgBVV1fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO&md5=e60be65abece48923abf937a4c2640a6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS2059798319011471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798319011471%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBaker%26aufirst%3DM.%2BL.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DCroll%26aufirst%3DT.%2BI.%26aulast%3DHintze%26aufirst%3DB.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DSammito%26aufirst%3DM.%2BD.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DStockwell%26aufirst%3DD.%2BH.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DMacromolecular%2520structure%2520determination%2520using%2520X-rays%252C%2520neutrons%2520and%2520electrons%253A%2520recent%2520developments%2520in%2520Phenix%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2019%26volume%3D75%26spage%3D861%26epage%3D877%26doi%3D10.1107%2FS2059798319011471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lh66ILFsCVhDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh553odqUlhCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0lh553odqUlhCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0lh553odqUlhCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh553odqUlhCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD web server for the generation of multi-group TLS models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+web+server+for+the+generation+of+multi-group+TLS+models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0ljkdWVxVbpeFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520web%2520server%2520for%2520the%2520generation%2520of%2520multi-group%2520TLS%2520models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkis, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orpen, A. G.</span></span> <span> </span><span class="NLM_article-title">Retrieval of crystallographically-derived molecular geometry information</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2133</span>– <span class="NLM_lpage">2144</span>, <span class="refDoi"> DOI: 10.1021/ci049780b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049780b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2133-2144&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=M.+Kesslerauthor=J.+Luoauthor=W.+D.+S.+Motherwellauthor=L.+H.+Purkisauthor=B.+R.+Smithauthor=R.+Taylorauthor=R.+I.+Cooperauthor=S.+E.+Harrisauthor=A.+G.+Orpen&title=Retrieval+of+crystallographically-derived+molecular+geometry+information&doi=10.1021%2Fci049780b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Retrieval of Crystallographically-Derived Molecular Geometry Information</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Kessler, Magnus; Luo, Jie; Motherwell, W. D. Sam; Purkis, Lucy H.; Smith, Barry R.; Taylor, Robin; Cooper, Richard I.; Harris, Stephanie E.; Orpen, A. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2133-2144</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crystallog. detd. bond length, valence angle, and torsion angle information in the Cambridge Structural Database (CSD) has many uses.  However, accessing it by conventional substructure searching requires nontrivial user intervention.  In consequence, these valuable data were underused and were not directly accessible to client applications.  The situation was remedied by development of a new program (Mogul) for automated retrieval of mol. geometry data from the CSD.  The program uses a system of keys to encode the chem. environments of fragments (bonds, valence angles, and acyclic torsions) from CSD structures.  Fragments with identical keys are deemed to be chem. identical and are grouped together, and the distribution of the appropriate geometrical parameter (bond length, valence angle, or torsion angle) is computed and stored.  Use of a search tree indexed on key values, together with a novel similarity calcn., then enables the distribution matching any given query fragment (or the distributions most closely matching, if an adequate exact match is unavailable) to be found easily and with no user intervention.  Validation expts. indicate that, with rare exceptions, search results afford precise and unbiased ests. of mol. geometrical preferences.  Such ests. may be used, for example, to validate the geometries of libraries of modeled mols. or of newly detd. crystal structures or to assist structure soln. from low-resoln. (e.g. powder diffraction) x-ray data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8AZn6w-T817Vg90H21EOLACvtfcHk0ljkdWVxVbpeFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlalsr8%253D&md5=a7909c257045351d5bff7b78c7fba2d5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci049780b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049780b%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DKessler%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DMotherwell%26aufirst%3DW.%2BD.%2BS.%26aulast%3DPurkis%26aufirst%3DL.%2BH.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DCooper%26aufirst%3DR.%2BI.%26aulast%3DHarris%26aufirst%3DS.%2BE.%26aulast%3DOrpen%26aufirst%3DA.%2BG.%26atitle%3DRetrieval%2520of%2520crystallographically-derived%2520molecular%2520geometry%2520information%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2133%26epage%3D2144%26doi%3D10.1021%2Fci049780b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cottrell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebeschuetz, J. W.</span></span> <span> </span><span class="NLM_article-title">Validating and understanding ring conformations using small molecule crystallographic data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1021/ci200439d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200439d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVWitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=956-962&author=S.+J.+Cottrellauthor=T.+S.+G.+Olssonauthor=R.+Taylorauthor=J.+C.+Coleauthor=J.+W.+Liebeschuetz&title=Validating+and+understanding+ring+conformations+using+small+molecule+crystallographic+data&doi=10.1021%2Fci200439d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Validating and Understanding Ring Conformations Using Small Molecule Crystallographic Data</span></div><div class="casAuthors">Cottrell, Simon J.; Olsson, Tjelvar S. G.; Taylor, Robin; Cole, Jason C.; Liebeschuetz, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">956-962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the conformational preferences of ring structures is fundamental to structure-based drug design.  Although the Cambridge Structural Database (CSD) provides information on the preferred conformations of small mols., analyzing this data can be very time-consuming.  In order to overcome this hurdle, tools have been developed for quickly extg. geometrical preferences from the CSD.  Here we describe how the program Mogul has been extended to analyze and compare ring conformations, using a library derived from over 900 000 ring fragments in the CSD.  We illustrate how these can be used to understand the conformational preferences of mols. in a crystal lattice and bound to proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8DAjmnZg2x7Vg90H21EOLACvtfcHk0ljkdWVxVbpeFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVWitLg%253D&md5=fb061c588b417493fc2f580f46595329</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci200439d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200439d%26sid%3Dliteratum%253Aachs%26aulast%3DCottrell%26aufirst%3DS.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DLiebeschuetz%26aufirst%3DJ.%2BW.%26atitle%3DValidating%2520and%2520understanding%2520ring%2520conformations%2520using%2520small%2520molecule%2520crystallographic%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D956%26epage%3D962%26doi%3D10.1021%2Fci200439d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Molprobity: All-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Molprobity%3A+All-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0liTloYULFO8nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolprobity%253A%2520All-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD','PDB','6WHD'); return false;">PDB: 6WHD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH','PDB','6PDH'); return false;">PDB: 6PDH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVK','PDB','1FVK'); return false;">PDB: 1FVK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG1','PDB','6PG1'); return false;">PDB: 6PG1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PC9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PC9','PDB','6PC9'); return false;">PDB: 6PC9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PGJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PGJ','PDB','6PGJ'); return false;">PDB: 6PGJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PD7','PDB','6PD7'); return false;">PDB: 6PD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG2','PDB','6PG2'); return false;">PDB: 6PG2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PLI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PLI','PDB','6PLI'); return false;">PDB: 6PLI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PBI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PBI','PDB','6PBI'); return false;">PDB: 6PBI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PIQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PIQ','PDB','6PIQ'); return false;">PDB: 6PIQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i33"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00111">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_63570"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00111?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00111</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full list of the structures in the synthesized library, details of microscale reaction optimizations, X-ray crystallography data collection, processing, and refinement statistics including control experiments, simulated annealing OMIT maps obtained, NMR solubility, binding analysis, and compound characterization data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf">PDF</a>)</p></li><li><p class="inline">List of SMILES and the binding affinity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_001.pdf">jm0c00111_si_001.pdf (5.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00111/suppl_file/jm0c00111_si_002.csv">jm0c00111_si_002.csv (0.86 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are listed below for the atomic coordinates and experimental data for the cocrystal structures of <i>Ec</i>DsbA bound to <b>2</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WHD">6WHD</a>), <b>6</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG1">6PG1</a>), <b>6</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PC9">6PC9</a>), <b>7</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PGJ">6PGJ</a>), <b>7</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PD7">6PD7</a>), <b>8</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PG2">6PG2</a>), <b>8</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PBI">6PBI</a>), <b>9</b> crude (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PIQ">6PIQ</a>), <b>9</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PDH">6PDH</a>), and <b>11</b> pure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PLI">6PLI</a>).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00111&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00111%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00111" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799533048a22483","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
