
<html lang="en"     class="pb-page"  data-request-id="5b465f26-7099-4ab2-8f75-e81de17b4c32"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01286;issue:issue:10.1021/jmcmar.2016.59.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets" /></meta><meta name="dc.Creator" content="Bradley C.  Doak" /></meta><meta name="dc.Creator" content="Jie  Zheng" /></meta><meta name="dc.Creator" content="Doreen  Dobritzsch" /></meta><meta name="dc.Creator" content="Jan  Kihlberg" /></meta><meta name="dc.Description" content="To improve discovery of drugs for difficult targets, the opportunities of chemical space beyond the rule of 5 (bRo5) were examined by retrospective analysis of a comprehensive set of structures for..." /></meta><meta name="Description" content="To improve discovery of drugs for difficult targets, the opportunities of chemical space beyond the rule of 5 (bRo5) were examined by retrospective analysis of a comprehensive set of structures for..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 30, 2015" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01286" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01286" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01286" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01286" /></link>
        
    
    

<title>How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01286" /></meta><meta property="og:title" content="How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0013.jpeg" /></meta><meta property="og:description" content="To improve discovery of drugs for difficult targets, the opportunities of chemical space beyond the rule of 5 (bRo5) were examined by retrospective analysis of a comprehensive set of structures for complexes between drugs and clinical candidates and their targets. The analysis illustrates the potential of compounds far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites. However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required. Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility. However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites. These insights should contribute to altering perceptions of what targets are considered “druggable” and provide support for drug design in beyond rule of 5 space." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01286"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01286">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01286&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01286&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01286&amp;href=/doi/10.1021/acs.jmedchem.5b01286" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2312-2327</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01157" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.5b01716" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bradley+C.++Doak">Bradley C. Doak</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Zheng">Jie Zheng</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Doreen++Dobritzsch">Doreen Dobritzsch</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Kihlberg">Jan Kihlberg</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Chemistry—BMC, Uppsala University, Box 576, SE-751 23 Uppsala, Sweden</span></div><div class="corresp-info"><strong>*</strong>Phone: +46 (0)18 4713801. E-mail: <a href="/cdn-cgi/l/email-protection#2b414a450540424347494e594c6b404e4642055e5e05584e"><span class="__cf_email__" data-cfemail="650f040b4b0e0c0d0907001702250e00080c4b10104b1600">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01286&amp;href=/doi/10.1021%2Facs.jmedchem.5b01286" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2312–2327</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 12, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 August 2015</li><li><span class="item_label"><b>Published</b> online</span>30 October 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01286" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01286</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2312%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBradley%2BC.%2BDoak%252C%2BJie%2BZheng%252C%2BDoreen%2BDobritzsch%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D6%26contentID%3Dacs.jmedchem.5b01286%26title%3DHow%2BBeyond%2BRule%2Bof%2B5%2BDrugs%2Band%2BClinical%2BCandidates%2BBind%2Bto%2BTheir%2BTargets%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2327%26publicationDate%3DMarch%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01286"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">12244</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">166</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01286" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Bradley&quot;,&quot;last_name&quot;:&quot;C. Doak&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Doreen&quot;,&quot;last_name&quot;:&quot;Dobritzsch&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Kihlberg&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;2312-2327&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01286&quot;},&quot;abstract&quot;:&quot;To improve discovery of drugs for difficult targets, the opportunities of chemical space beyond the rule of 5 (bRo5) were examined by retrospective analysis of a comprehensive set of structures for complexes between drugs and clinical candidates and their targets. The analysis illustrates the potential of compounds far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites. However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required. Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility. However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind wel&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01286&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01286" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01286&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01286" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01286&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01286" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01286&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01286&amp;href=/doi/10.1021/acs.jmedchem.5b01286" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01286" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01286" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26pmid%3D26457449%26genre%3Darticle%26aulast%3DDoak%26date%3D2016%26atitle%3DHow%2BBeyond%2BRule%2Bof%2B5%2BDrugs%2Band%2BClinical%2BCandidates%2BBind%2Bto%2BTheir%2BTargets%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D6%26spage%3D2312%26epage%3D2327%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290817" title="Interfaces">Interfaces</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/jmcmar.2016.59.issue-6/20160324/jmcmar.2016.59.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">To improve discovery of drugs for difficult targets, the opportunities of chemical space beyond the rule of 5 (bRo5) were examined by retrospective analysis of a comprehensive set of structures for complexes between drugs and clinical candidates and their targets. The analysis illustrates the potential of compounds far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites. However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required. Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility. However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites. These insights should contribute to altering perceptions of what targets are considered “druggable” and provide support for drug design in beyond rule of 5 space.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88118" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88118" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Drug discovery is at a crossroads where ground-breaking advances in our understanding of how diseases develop are now made at an unprecedented pace. However, efficiency of drug discovery has continued to decline as the number of new drugs approved each year has essentially been constant during the past 30 years, while the costs of pharmaceutical development have increased dramatically.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> This decline has been attributed to a few fundamental issues, including a need to deliver first-in-class treatments for complex diseases while at the same time meeting increased demands for safety and efficacy.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> As a result there is high attrition in phase II and III clinical trials, mainly due to lack of efficacy and safety issues.<a onclick="showRef(event, 'ref2 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref2 ref5 ref6">(2, 5, 6)</a> Therefore, it has been emphasized that improved selection of targets that are associated with diseases is the single most important factor required to increase efficacy and deliver innovative medicines.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">During the past two decades the human genome and various other genomes have been mapped,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and significant progress has been made toward mapping the human proteome.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> These rapid advances have made an increased number of <i>potential</i> drug targets accessible that belong to both established and novel target classes. Despite the advances in target identification, less than a quarter of recently approved drugs are directed against novel targets, and the majority of these drugs target established classes of G protein-coupled receptors (GPCRs), transporters, or enzymes.<a onclick="showRef(event, 'ref1 ref10'); return false;" href="javascript:void(0);" class="ref ref1 ref10">(1, 10)</a> A limiting factor may be that approximately 3000 of the genes in the human genome have been estimated to be related to disease. Out of these only 600–1500 have been considered amenable for manipulation with “traditional” small molecule drugs,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> i.e., drugs that comply with the rule of 5 (Ro5) guidelines and are highly likely to be cell permeable and orally bioavailable. Still, it has been pointed out that large portions of well-established target classes such as ion channels, GPCRs, and nuclear receptors remain unexplored.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, an even larger number of targets from less explored and novel classes which are “difficult-to-drug” using Ro5 compliant compounds could provide significant, additional opportunities for drug discovery. For example, the human proteome<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> is estimated to have 100 000–1 000 000 binary protein–protein interactions (PPIs)<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> and may constitute one of the most important sources of novel targets for drug discovery. However, the proportion of the proteome and its massive number of PPIs that are involved in pathogenic mechanisms remains to be established. Even with that caveat the recent and rapid developments in target identification urgently need to be matched by innovative approaches for modulating nontraditional target classes, such as PPIs.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a></div><div class="NLM_p">Targets currently classified as “difficult-to-drug” with Ro5 compliant ligands characteristically have binding sites that are large, highly lipophilic, or highly polar, flexible, flat, or featureless (i.e., contain few opportunities for molecular interactions such as hydrogen bond donors and acceptors).<a onclick="showRef(event, 'ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19">(16-19)</a> In addition, the perceived lack of oral bioavailability outside Ro5 space has led many to abandon these targets and classify them as “undruggable”. Thus, what initially appears as vast opportunities of novel targets emerging from advances in genomics and proteomics will, to a large extent, require small molecule drug discovery to move outside Ro5 space into what has been termed beyond Ro5 (bRo5) or “middle space”.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Interestingly, recent analysis of drugs and clinical candidates that fall outside Ro5 space has shown that this space offers significant possibilities for discovery of orally bioavailable and cell permeable compounds, possibly more than previously thought.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> It can therefore be argued that a too strict implementation of the Ro5 may have hampered the pharmaceutical industry from seizing opportunities involving novel but more difficult targets.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21-25)</a></div><div class="NLM_p last">We and others have hypothesized the benefits of using bRo5 drugs for difficult targets,<a onclick="showRef(event, 'ref14 ref15 ref21 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref21 ref26 ref27">(14, 15, 21, 26, 27)</a> and examples and case studies have been reported in the literature. Here we present a comprehensive analysis of bRo5 drugs and clinical candidates that highlight their ability to modulate difficult targets, thereby expanding the number of targets for which we can design oral and parenteral drugs. First, we assessed what target classes current drugs and clinical candidates outside Ro5 space are directed toward in comparison to Ro5 compliant drugs. Analysis then focused on how drugs and clinical candidates outside Ro5 space bind to their targets based on crystal structures of 130 clinically relevant complexes, which were compared to drug–target complexes in Ro5 space. This allowed us to define to what extent binding site and ligand characteristics such as size, shape, molecular interactions, affinity, and ligand efficiencies differ between different drug spaces. The influence of conformational flexibility of the ligand and its shape was also investigated for compounds in beyond Ro5 space. The results are then discussed to provide guidance for design of bioactive small molecule drugs outside Ro5 space for difficult targets.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Drugs and Clinical Candidates Data Sets</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63081" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63081" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To facilitate this in-depth analysis of how drugs and clinical candidates that do not comply with the Ro5 bind to their targets, a comprehensive data set of 475 drugs and clinical candidates with MW > 500 Da was classified by the compounds’ calculated physicochemical properties. They were then divided into two data sets where intuitive and natural divisions in the ligand property distributions appeared as previously reported,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> each representing different chemical spaces (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>a). Two data sets of Ro5 compliant drugs were also compiled from ChEMBL<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and the recent literature<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> for comparison during analysis (<a class="ref internalNav" href="#fig1" aria-label="Figures 1">Figures 1</a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). In this analysis compounds in <i>rule of 5 space</i> adhere to all of Lipinski’s guidelines, whereas compounds that break one Ro5 guideline (MW = 500–700 Da) and also have other properties which may extend a short distance outside strict Ro5 space were classified as being in <i>extended Ro5 space</i> (eRo5).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Finally, compounds in <i>beyond Ro5 space</i> (bRo5) all have MW > 500 Da and in addition have one or more properties outside the eRo5 ranges. They are thus far beyond Ro5 space but with an upper MW limit of 3000 Da set to exclude biologics such as insulin. The classification into eRo5 and bRo5 space is useful to completely separate compounds in Ro5 space from those that reside far away in bRo5 space and do not conform to its trends. Thus, eRo5 space may be thought of as a buffer zone between Ro5 and bRo5 space, representing the natural tail of the distribution of Ro5 drugs, in line with the original report of Lipinski,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and the beginning of bRo5 space. The rationale for the eRo5 and bRo5 classification is further highlighted by the two data sets having mean quantitative estimates of druglikeness (QED) scores of 0.31 and 0.16, respectively, providing a single measure of the distance from traditional rule of 5 space. Both of these QED scores are significantly below 0.67 and 0.49, which are the mean values identified by medicinal chemists for compounds being “attractive” and “unattractive” for drug development, respectively.<a onclick="showRef(event, 'ref21 ref30'); return false;" href="javascript:void(0);" class="ref ref21 ref30">(21, 30)</a> Our data set of drugs and clinical candidates was obtained by searching different databases for compounds with MW ranging from 500 to 3000 Da followed by filtering to remove contrast agents, veterinarian products, etc.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Therefore, some drugs and clinical candidates outside our strict definition of Ro5 space, i.e., those with MW < 500 Da and one of HBD > 5, HBA > 10, ClogP > 5, or ClogP < 0, have not been included in the analysis. We also highlight that some calculated properties are highly correlated to each other (e.g., HBA and PSA, <i>r</i><sub>s</sub> = 0.89–0.94), as illustrated in the correlation tables and principle component analysis, which can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information (Figures S1 and S2)</a>. It is nonetheless useful to base the classification on these seemingly redundant properties to aid filtering and analysis in different situations, ranging from computer assisted to practical, “back of the envelope” calculations. The three data sets were then analyzed and compared extensively across different ligand–target interaction properties. Differences are described as being “significant” where statistically significant different mean values were found (unpaired <i>t</i> test, with a <i>p</i>-value of <0.05); full details and <i>p</i>-values can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a> but are also denoted in the figures.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Classification of 475 drugs and clinical candidates that have MW > 500 Da into extended rule of 5 (blue) and beyond rule of 5 (green) chemical space based on calculated physicochemical properties. (b) Development pipeline by chemical space and chemical class showing peptides/peptidomimetics (green), natural products and derivatives (blue), and de novo designed drugs and clinical candidates (red) by phase. Orals are in dark and parenterals in light colors, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The data set of 475 drugs and clinical candidates in eRo5 and bRo5 space that make up the current data set was previously curated and used to investigate oral bioavailability in eRo5 and bRo5 space.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> It was also classified with regards to chemical class, route of administration, and phase of development.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This allows discussion of trends in drug development and demonstrates that de novo designed compounds are in majority (43%), with equal numbers of natural products and peptides/peptiodomimetics (26% each) across the full data set.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The majority of de novo designed compounds are oral (64%), whereas natural products and in particular peptides/peptidomimetics are mainly parenteral (59% and 80%, respectively). Analyzing the data set by phase of development, chemical space, and chemical class demonstrates that de novo designed compounds dominate strongly in all clinical phases in eRo5 space and that the majority of them are intended for oral administration (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b). In bRo5 space peptides constitute the largest group across clinical candidates with proportions of de novo designed compounds and natural products being only somewhat lower. In phase II, phase III, and approved, bRo5 natural products and peptides are mainly for parenteral administration, whereas the proportion of orals is >45% for de novo designed compounds. In summary, the drug discovery industry is focusing on development of de novo designed drug candidates for oral administration in eRo5 space while relying on all three chemical classes and more on parenteral delivery in bRo5 space. It is noteworthy that a significant number of compounds (161 in total) from all three chemical classes in bRo5 space are in clinical development, indicating a willingness to venture outside the Ro5. This is further supported by the emergence of a number of biotechnology companies that focus on this chemical space,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> often in partnership with larger pharmaceutical companies. In the current analysis we aim to globally assess bRo5 ligand–target interactions to define if moving far from traditional Ro5 space is warranted in efforts to conquer difficult targets and for what target classes and types of binding sites compounds in bRo5 space provide advantages.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">3 Target Class Modulation by Chemical Space</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To analyze the target class preferences of drugs and clinical candidates in eRo5 and bRo5 space, the data set of 475 drugs and clinical candidates with MW > 500 Da<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was first classified using a similar taxonomy as employed to dissect trends and innovations in drug development for a large data set of approved drugs (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> A Ro5 target class reference data set was obtained by filtering this large literature data set of approved drugs<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> by all of the Ro5 guidelines and selection of only one target per drug. Drugs in eRo5 and bRo5 space showed interesting differences in their target class preferences compared to Ro5 compliant drugs (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). For instance, an increased proportion of eRo5 and bRo5 drugs and clinical candidates modulate protease and kinase targets, which have been increasingly explored only in the past decade.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Similar to the approved Ro5 compliant kinase inhibitors in our data set, tyrosine kinases were the largest subgroup targeted by eRo5 and bRo5 drugs and clinical candidates (42%). Kinase inhibitors currently in clinical trials originate from Ro5, as well as from eRo5 and bRo5 space, and no significant trends linking subgroups of kinases to a particular chemical space were apparent. Nevertheless it is clear that medicinal chemists are drawing from compounds outside traditional Ro5 space to target an expanding number of kinases. Structural and adhesion targets such as tubulin as well as transferases and isomerases are also more prevalent for eRo5 and/or bRo5 drugs and clinical candidates. In our analysis there is also a higher prevalence of bRo5 and eRo5 drugs and clinical candidates at “other” targets, a class consisting of antioxidants, vitamin and hormone replacements, orphan drugs, and other unclassifiable molecular targets. Moreover, as compared to Ro5 drugs, a smaller proportion of bRo5 drugs and clinical candidates bind to well-established target classes such as ion channels and nuclear receptors. Similar trends in target class preference are also observed for the oral only and approved only subsets of eRo5 and bRo5 drugs and clinical candidates (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>). Importantly, a number of classes that are more frequently targeted by eRo5 and bRo5 drugs and clinical candidates, such as proteases, kinases, and transferases, are among those recently concluded to be underexplored in drug discovery.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">As different targets have been found to have different preferred ligand chemical spaces,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> we conclude that increased exploration of eRo5 and bRo5 space should be beneficial for future development of drugs for underexplored target classes. The discovery of protein kinase inhibitors, now commonly used in oncology, is an important example of how exploration of novel chemical space can expand what targets are considered “druggable”.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> It should also be noted that eRo5 and bRo5 compounds also appear to be among the most suitable for modulating the increasing number of protein–protein interactions (PPI) that are emerging as therapeutic targets.<a onclick="showRef(event, 'ref14 ref34'); return false;" href="javascript:void(0);" class="ref ref14 ref34">(14, 34)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Creation of target class preference data sets. The data set compiled by Rask-Andersen et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> was filtered by all the rule of 5 (Ro5) guidelines, and the primary target reported in the literature was selected for each drug to give the Ro5 data set. The primary targets reported in the literature for the full set of extended Ro5 (eRo5) and beyond Ro5 (bRo5) drugs and clinical candidates were classified using the same taxonomy. (b) Proportion of Ro5 drugs (red), eRo5 (blue), and bRo5 (green) drugs and clinical candidates modulating the indicated target classes. The proportion of compounds in each of the three chemical space data sets that modulate a specific target class (the number of compounds that modulate that target class divided by the total number in the data set, <i>N</i>) is shown by the vertical bars. Proportions were calculated to show differences in target preferences as the number of compounds differ significantly between the three data sets. Compounds that are in phase (P) I, II, or III or approved (App.) are shown by increasingly darker color shadings of the data sets. Targets are arranged by Ro5 target class preference from highest (left) to lowest (right). Alternative plots that show only approved, clinical, or orally bioavailable dugs and clinical candidates in eRo5 and bRo5 space, as well as the exact number of compounds in each category, are included in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">4 Characterization of Drug–Target Complexes by Chemical Space</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12537" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12537" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">4.1 Generating Drug–Target Structure Data Sets</h3><div class="NLM_p">To probe how drugs outside Ro5 space bind to their targets, structural data for all available complexes of eRo5 and bRo5 drugs and clinical candidates with their targets were extracted by cross-referencing the 475 drugs and clinical candidates with the Protein Data Bank (PDB). In total, 93 drugs had crystal structures of relevant drug–target complexes that fulfilled the chosen quality requirements (20% of the data set, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). A reference set of 37 crystal structures of relevant Ro5 drug–target complexes was also obtained after clustering a Ro5 filtered ChEMBL drugs data set according to their physicochemical properties (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). In order to remove bias toward highly explored targets or drug classes and ensure that conclusions reflect the true variation between chemical spaces, redundant complexes between a target and other members of the same drug class were excluded. For example, the erythromycin A–ribosome complex <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JZY">1JZY</a> was used as representative of all erythronolide–ribosome complexes. This produced three nonredundant data sets of crystal structures of Ro5 drugs (<i>N</i> = 29), eRo5 (<i>N</i> = 26), and bRo5 (<i>N</i> = 22) drugs and clinical candidates bound to their targets (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). An extensive set of additional annotated figures for the all structures and the nonredundant data sets can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a> along with statistical analysis.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cross-referencing the extended Ro5, beyond Ro5 data sets, and a representative set of ChEMBL<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Ro5 drugs with the Protein Data Bank (PDB) and filtering by quality constraints gave three data sets of relevant Ro5, eRo5, and bRo5 drug–target structures. To avoid bias toward highly explored drug classes, further filtering to remove redundant drug class–target structures gave unbiased, nonredundant data sets which contain only one compound per drug–target class (e.g., one erythronolide, one azole anti-infective, etc.). The number of compounds approved (App.) and the number in phases (P) III, II, and I development are shown. Results of the analysis of the complete data sets can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a> for comparison, while the nonredundant data sets are analyzed within the paper as well as in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Analyzed Nonredundant Drug–Target Complexes in Extended and Beyond Ro5 Chemical Space</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0011.gif" alt="" id="GRAPHIC-d1862e431-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Route of administration used in the indicated phase of development.</p></div></div><div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">4.2 Shape and Size of Binding Sites</h3><div class="NLM_p">The binding site shape of drug–target complexes can be assessed manually or with the aid of descriptors calculated from binding sites identified by automated methods. Two such methods, the recently described Difference of Gaussian Site (DoGSite)<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> and MetaPocket 2.0,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> the latter of which is based on several algorithms, were used to analyze the three data sets of drug–target complexes. The threshold for successful identification of binding sites was set as >20% of the volume of the bound drug being covered by the calculated binding site. With this lenient cutoff only 54% and 43% of all bRo5 drug–target bindings sites were successfully identified by DoGSite and MetaPocket, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S4a</a>). In addition, the successfully calculated bRo5 binding sites covered a significantly lower proportion of the bound drug than Ro5 drug binding sites (mean 54% vs 94%, respectively; <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S4</a>). Hence, bindings site shape was also assessed manually by visual inspection and classification as being flat, groove, tunnel, pocket, or internal. These correspond to the drug interacting with its target by a single face for a flat site, two or three faces for a groove, and four faces with two noninteracting opposing faces for a tunnel-shaped site. Interactions of the drug through four or five faces, leaving one noninteractive face, characterizes a pocket, and for an internal binding site the drug is completely buried inside the target (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S5</a>). For binding sites that were successfully calculated with DoGSite, descriptors such as enclosure and depth corresponded well to the shape classifications and thereby support the manual classification (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S6</a>). However, volume and sphericity descriptors failed to accurately describe the size and shape of flat- and groove-shaped binding sites, as the calculated sites poorly covered the actual drug or clinical candidate binding site. The manual classification is also supported by the increase in the mean proportion of drug surface areas (SA) that become buried upon binding to the target from flat to internal binding sites (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S8</a>). Due to the low success rate in binding site calculation and the inability of descriptors to characterize those sites that were successfully calculated, the manual classification of binding site shapes was used throughout the current analysis.</div><div class="NLM_p">The distribution of binding site shapes showed striking differences between the three sets of drug–target complexes (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a), with higher proportions of bRo5 drugs and clinical candidates binding to the “difficult” open, flat, and groove binding sites compared to Ro5 drugs. In contrast, Ro5 drugs display a preference for pocket and internal binding sites, which conforms well with the view of such sites as being highly “druggable” with Ro5 compliant compounds. Binding site shapes observed for eRo5 drugs and clinical candidates are more evenly distributed between groove, tunnel, pocket, and internal sites, revealing an ability of compounds residing just outside Ro5 chemical space to target a wide range of sites, with the exception of flat binding sites. It should be noted that compounds in eRo5 space also bind to pocket-shaped and internal sites, which are then larger than those that have Ro5 compliant ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S8</a>). Too few ligands in bRo5 space bind to pockets to draw any statistically significant conclusions regarding binding site size. The shapes of the ligands, in their target bound conformations, were also assessed using normalized principle moment of inertia (nPMI) plots, which characterize ligands by their similarity to rod, disk, and sphere shapes (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). In agreement with previous analyses based on calculated 3D conformations,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Ro5 compliant drugs were predominantly rodlike, while those in eRo5 and particularly in bRo5 space were more disk- and spherelike. Flat and groove binding sites were also found to have ligands that were significantly more disk- and spherelike compared to ligands for pocket and internal binding site shapes (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figures S9 and S10</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Distribution of binding site shapes, (b) ligand normalized principle moment of inertia (nPMI) shape plot, (c) buried ligand surface area (SA, Å<sup>2</sup>) as a function of total ligand SA (Å<sup>2</sup>), (d) box plot of buried ligand SA (Å<sup>2</sup>), and (e) box plot of proportion of buried ligand SA. Each panel presents data for rule of 5 drugs (Ro5, red), extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates in complex with their respective targets. Protein–protein interaction (PPI) interface SA data were extracted from Luo et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and reanalyzed. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition to binding site shape, the ligand surface area (SA) that is buried upon binding and its proportion to the total ligand SA also provide useful information about the nature of the binding sites. While the buried SA indicates the size of the binding site, the proportion of buried ligand SA can indicate how open or exposed the binding site is. The plot of buried ligand SA against total ligand SA shows that eRo5 and bRo5 drugs have larger SAs buried in complexes with their targets than Ro5 drugs (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c,d). The proportion of buried ligand SA is, however, lower for drugs in eRo5 and bRo5 space compared to Ro5 space (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c,e, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S8</a>), which is consistent with the preference of eRo5 and bRo5 drugs for flat and groove binding sites. Although the buried SA of eRo5 and bRo5 drugs is larger than that of Ro5 drugs, it is still slightly smaller than interface areas in weak protein–protein interactions (PPIs with <i>K</i><sub>d</sub> values of >1 μM) and significantly smaller than those in strong protein–protein interactions (PPIs with <i>K</i><sub>d</sub> values of <1 μM, <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> However, most of the affinity of protein–protein interactions arises from smaller hotspot areas within the interface<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> and drugs in bRo5 space may reach the critical size required to bind such hotspots and gain sufficient affinity toward PPI interfaces.<a onclick="showRef(event, 'ref14 ref25'); return false;" href="javascript:void(0);" class="ref ref14 ref25">(14, 25)</a> In conclusion, drugs in bRo5 space are less rodlike in ligand shape and bind to larger binding sites than Ro5 drugs but with a lower proportion of their total SA buried in the complex. This agrees well with the observation that eRo5 and particularly bRo5 drugs bind to difficult, larger, and more open binding sites that approach the size of PPI interfaces. In addition, compounds in bRo5 space are likely large enough to interact effectively with hotspots of PPI interfaces.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">4.3 Molecular Interactions at Binding Sites</h3><div class="NLM_p">The overall polarity of the drug and the target at the binding site and the number of intermolecular interactions, such as hydrogen bonds, are important descriptors for binding sites. The proportion of nonpolar heavy atoms at the binding site interface, i.e., the number of carbon and sulfur atoms divided by the total number of heavy atoms at the interface, provides a measure of the overall polarity of the ligand and target interfaces. No significant differences in the mean values of this measure of interface polarity from Ro5 to bRo5 space were found for the ligand or target interface data sets (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S11</a>). The mean proportion of nonpolar atoms at the targets interfaces is similar to previous estimates<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and is also consistently lower than that of the ligands interfaces. Since bRo5 drugs have a smaller proportion of their total surface area buried upon binding to the target compared to Ro5 drugs, they might enrich polar or lipophilic atoms at the binding site interface to improve interactions. However, the polarity of the ligand binding site interface is similar to the overall polarity of the ligand for all three chemical spaces, indicating that neither polarity nor lipophilicity is enriched at the interface with the target (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Proportion of nonpolar heavy atoms (carbon and sulfur) to the total number of heavy atoms for the ligand binding site interface, the overall ligand, and the target binding site interface. The distributions of the number of (b) hydrogen bond acceptor (HBA) atom interactions and (c) hydrogen bond donor (HBD) atom interactions of ligands with their targets. Each panel presents data for rule of 5 (Ro5, red) drugs, extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates in complexes with their respective targets. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Similar to interface polarity, the number of atoms forming hydrogen bonding interactions between the ligand and target does not significantly differ between the three sets of drugs. Ligands in bRo5 space display a slightly higher mean number of HBA atom interactions but a statistically similar number of mean HBD atom interactions compared to Ro5 ligands (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b,c, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figures S13 and S14</a>). At first glance it may appear that bRo5 drugs show a much wider distribution, particularly a higher maximum number of HBD interactions. However, it should be taken into account that a criterion for including drugs in the Ro5 and eRo5 data sets is ≤5 HBD atoms, limiting their possible HBD interactions. In addition, the eRo5 and bRo5 data sets encompass both oral and parenteral drugs and clinical candidates, with two parenterally delivered polysaccharides, amikacin and β-acarbose, being the only two drugs with >5 HBD interactions in the data set. The number of π–π interactions in drug–target complexes is also similar for all three data sets (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S15</a>). Other types of interactions, such as ionic, cation−π interactions, and halogen bonding, were less frequently observed in all three data sets; hence, no confident conclusions could be drawn about these types of interactions. Therefore, despite the increase in size and the difference in shape observed for binding sites of bRo5 drugs, the overall polarity as well as type and number of molecular interactions in the binding site remains similar to those of Ro5 drugs. These findings indicate that the same approaches for lead optimization may be applied to design of bRo5 ligands as for Ro5 ligands, with recent illustrative examples being the discovery of hepatitis C virus NS3/4a protease inhibitors.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">4.4 Affinities and Ligand Efficiencies</h3><div class="NLM_p">Affinities, measured as equilibrium dissociation constants, inhibition constants, or concentrations giving 50% inhibition of target activity (<i>K</i><sub>d</sub>, <i>K</i><sub>i</sub>, IC<sub>50</sub>), were extracted from the literature for drug–target complexes in the three data sets where available. Affinity data were consistent with those previously reported for a large data set of drugs,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and drugs in Ro5, eRo5, and bRo5 space had similar mean values and distributions of affinities (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a). This also holds true for 102 of the 177 approved drugs in the original eRo5 and bRo5 data sets<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> for which affinities were available (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S16</a>) and their analysis leads to at least two important conclusions. First, drugs outside Ro5 space do not require higher affinities for their targets compared to Ro5 compliant drugs to compensate for any perceived or actual unfavorable pharmacokinetics. Second, despite being perceived as “difficult”, binding sites that are larger and more open can be modulated by drugs with similar affinities as drugs directed to sites traditionally considered highly “druggable”. This correlates well with the observations that similar numbers and types of molecular interactions are formed between large, open, and smaller, enclosed binding sites and their respective ligands (cf. <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S11–S15</a>).</div><div class="NLM_p">Ligand efficiency metrics have found widespread use;<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> however, they also have some limitations associated with their application, particularly outside traditional Ro5 drug space.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> We nonetheless believe it is useful to characterize the ligand efficiency (LE) and lipophilic ligand efficiency (LLE) distributions observed in eRo5 and bRo5 space to provide guides for those who wish to use them in drug development. As the drugs in eRo5 and bRo5 space are significantly bigger than Ro5 drugs, i.e., they have higher molecular weights and more heavy atoms, their LE is significantly lower (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S17</a>). While LE is known to not be completely independent of the size of compounds,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> we also find that it is correlated to the proportion of buried SA and, hence, also the shape of the binding site (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>c). It should be noted that this correlation is not lost for size corrected ligand efficiencies (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figures S18 and S19</a> for full details). Hence, ligand efficiencies are significantly lower for flat and groove binding sites than for pocket and internal sites, with mean values increasing from 0.19 to 0.42 kcal/(mol·HAC) from flat to internal sites. LLE,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> a measure of the affinity of a compound with respect to its lipophilicity, is similar for all three data sets and across all binding site shapes (mean values ± standard deviations: 4.0 ± 1.4 to 6.6 ± 3.7, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S17</a>), indicative of Ro5, eRo5, and bRo5 drugs having similar lipophilicities and affinities. Therefore, efforts to develop drugs in eRo5 and bRo5 space should aim to generate compounds with similar affinities to those in Ro5 space but with altered guidelines for LE, particularly for difficult binding sites. Typical bRo5 drugs have a LE of 0.11–0.30 kcal/(mol·HAC), with both increased compound size and open, flat binding sites contributing to their reduced values. It should be emphasized that the ligand efficiencies discussed herein reflect the historical development of drugs and that a recent review of inhibitors of PPIs indicates that LE values in the top half of our data set, 0.20 and upward, are possible for compounds targeting difficult binding sites.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Affinities and (b) ligand efficiencies (LE, kcal/(mol·HAC)) for rule of 5 (Ro5, red), extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates. (c) Relationship between LE and binding site shape. As a rule of thumb, Ro5 drug candidates are optimized to have 10 nM affinities, corresponding to LEs of 0.30 for a compound with a molecular weight of 500 Da (heavy atom count, HAC ≈ 36). This “guideline” for optimization is marked with a gray line in (b) and (c). Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">5 Examining beyond Rule of 5 Drugs and Clinical Candidates</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44602" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44602" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A key result of this analysis is that drugs and clinical candidates in bRo5 space are of particular interest because of their greater ability to modulate difficult more open, flat, and groove-shaped binding sites. Two recent investigations provide additional support for this observation. The first found that a representative set of macrocyclic natural products with high MW bind either face-on to flat binding sites or edge-on to groove-shaped binding sites.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The second highlighted that de novo designed inhibitors of protein–protein interactions that entered clinical trials during the past decade are predominantly nonmacrocyclic and bind to groove- or pocket-shaped binding sites in rodlike conformations, which likely reflects the higher druggability of groove- and pocket-shaped binding sites.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Our nonredundant data set of bRo5 ligand–target structures contains three drug classes that bind to flat binding sites, all of which are used to treat infectious diseases (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>) and 11 that target groove-shaped binding sites, five of which are used in oncology. Investigating if the chemical class of the ligand in bRo5 space (i.e., de novo designed, natural product, or peptide/peptidomimetic) affected the properties of the complex with the target showed no significant differences for properties discussed above. This is most likely due to the small number of complexes in each subset of the bRo5 data set.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Chemical structures of drugs and clinical candidates in bRo5 space that bind to flat and groove-shaped binding sites on targets. Compounds are grouped by therapeutic indication and their status, route of administration and clinically relevant targets are given below each structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, more than half of the nonredundant data set of bRo5 drugs and clinical candidates that bind to flat and groove-shaped binding sites are macrocycles (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Examination of the full data set of 475 drugs and clinical candidates that have a MW > 500 Da also reveals that orally bioavailable, as well as parenteral, macrocycles are significantly enriched in bRo5 space compared to nonmacrocycles (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a), a finding that has been highlighted before.<a onclick="showRef(event, 'ref21 ref25 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref25 ref27">(21, 25, 27)</a> Though the reason for this enrichment has not been conclusively identified, it is known that conformational constraints imposed by the macrocyclic structure can convey improved pharmacokinetics but also higher potency and better selectivity as compared to related acyclic analogues at similar binding sites.<a onclick="showRef(event, 'ref26 ref27 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref48 ref49">(26, 27, 48, 49)</a> Conformational restriction has therefore been postulated as a general principle leading to macrocycle enrichment in bRo5 space<a onclick="showRef(event, 'ref26 ref27 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref48 ref49">(26, 27, 48, 49)</a> and is consistent with studies showing that increasing the number of rotatable bonds has a negative effect on the oral bioavailability of drugs, independent of their chemical class.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50, 51)</a> Generally increasing the conformational flexibility of drug candidates is expected to reduce both the affinity and selectivity of target binding,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> although the correlation between flexibility and promiscuity has been questioned.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52, 53)</a> As the influence of conformational restriction through macrocyclization on target binding still remains unclear, at least in some cases,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> we also investigated flexibility and shape of macrocycles and nonmacrocycles for our data set of bRo5 drugs and clinical candidates to obtain an overview of flexibility in bRo5 space.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">5.1 Flexibility of Macrocycles and Nonmacrocycles</h3><div class="NLM_p">We first investigated whether macrocyclic drugs and clinical candidates are more rigid than nonmacrocyclic ones in bRo5 chemical space. Due to the well-known difficulty in predicting the conformations of bRo5 compounds using computational methods,<a onclick="showRef(event, 'ref49 ref55'); return false;" href="javascript:void(0);" class="ref ref49 ref55">(49, 55)</a> we analyzed all available experimental conformers from crystal structures in the Protein Databank (PDB) and Cambridge Structural Database (CSD) for our data set of bRo5 drugs and clinical candidates. Arguably such an analysis is limited by the size of the data set and the data that are available for each compound and possibly biased by crystal packing effects and poorly refined geometries,<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> but it has the advantage of being based on experimental data. We found a total of 24 drugs and clinical candidates in bRo5 space, with 12 nonredundant drug classes that displayed multiple conformations in the crystalline state. In spite of its somewhat limited size, this data set consists of the largest number of bRo5 drugs for which experimentally determined conformations have been analyzed and allowed us to reach some interesting but potentially preliminary conclusions.</div><div class="NLM_p">All conformers observed for a given drug or clinical candidate were clustered to identify a set of representative conformers that showed >1 Å root-mean-square deviation (rmsd) of all heavy atoms between different conformers. This ensured that the subsequent analysis was not biased by multiple crystal structures of the same conformation. The representative conformers were then compared using average rmsd, where a high value indicates that the compound is flexible and can adopt one or more significantly different conformations. In addition to the rmsd values for all atoms, rmsd values for the macrocyclic core atoms subset and core plus single heavy atoms attached directly to the core (peripheral atoms) subset were also calculated for macrocyclic drugs and clinical candidates.</div><div class="NLM_p">We found that rmsd values of the macrocyclic core atoms are similar to those of the core plus peripheral atom subset, but that rmsd values are significantly higher when all atoms in the macrocycle drugs are taken into account (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S20</a>). This indicates that the side chains are commonly the most dynamic regions of macrocyclic drugs. Furthermore, the all atom rmsd of macrocycles is similar to that of nonmacrocyclic drugs and clinical candidates in bRo5 space, suggesting that both chemical classes have a similar degree of overall flexibility. Although rmsd values are useful for comparison of conformations, they give little insight into the source of conformational flexibility. Therefore, an analysis of the location of bonds that rotate to give the different conformations was conducted. For macrocyclic drugs in bRo5 space, bonds from all regions, i.e., the macrocyclic core, the side chains, as well as bonds linking the two, were rotated to form the different conformers (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>c, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S21</a>). However, most bonds throughout all regions of the macrocycle display only modest or limited rotational freedom. Nonmacrocyclic drugs in bRo5 space also demonstrate a similar distribution of rotational freedom about bonds (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>d, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S21</a>). Conformationally constrained regions of nonmacrocyclic drugs arise from aromatic rings, other π-systems such as amides, and substituted aliphatic rings that occur in higher proportions than for macrocyclic drugs (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S20e,f</a>). The flexibility of nonmacrocycles mainly originates from rotation around bonds that connect these rigid elements. Overall, this analysis of conformational flexibility and its origins indicates that macrocycles are <i>not more rigid</i> than nonmacrocyclic drugs in bRo5 space across different drug classes. Instead, macrocyclisation could be considered as a complementary strategy to other, more traditional approaches for introducing rigidity into drugs. Hence, decreased flexibility may not be the primary reason for the enrichment of oral macrocycles in bRo5 space.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Distribution of parenterals (triangles) and orals (circles) for macrocycles (right) in beyond Ro5 (bRo5 MC, green) and extended Ro5 space (eRo5 MC, orange), as well as nonmacrocycles (left) in the two chemical spaces (bRo5 non-MC, red, and eRo5 non-MC, blue). The dashed black box shows the eRo5 space limits for ClogP and MW. (b) Comparison of average root-mean-square deviation (rmsd, Å) values of the representative conformers of bRo5 macrocyclic drugs (<i>N</i> = 8, mean number of representative conformers of 3.5) and bRo5 nonmacrocyclic drugs (<i>N</i> = 4, mean number of representative conformers of 2.5). For the macrocycles, rmsd values for core atoms (atoms in the macrocycle ring), core plus periphery atoms (the core atoms plus all single heavy atoms attached to the core), as well as all atoms are compared. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes, and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values. (c, d) Chemical structure, all observed crystal structures, and the most representative structure colored by the circular standard deviation of each bond of erythromycin A and ritonavir, respectively. For erythromycin A core atoms are in purple, periphery atoms in orange, and side chain atoms in black. The superimpositions show all experimentally determined structures from the Cambridge Structural Database and Protein Databank for the two drugs with heavy atoms colored by chemical element (green for carbon, blue for nitrogen, red for oxygen, and yellow for sulfur). Flexible bonds giving rise to different conformers of each drug were identified from the circular standard deviation of the dihedral angles and are color-coded from white (0) to red (1) representing rigid to flexible bonds, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">5.2 Ligand Efficiency and Shapes of Ligands and Binding Sites</h3><div class="NLM_p">To further investigate the differences and similarities of macrocyclic and nonmacrocyclic drugs and clinical candidates in bRo5 space the affinity, ligand efficiency (LE), binding site shape, and ligand shape were analyzed. Affinities did not differ significantly between macrocyclic and nonmacrocyclic drugs and clinical candidates in bRo5 space, but macrocycles were found to have a slightly lower LE compared to nonmacrocycles (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a,b, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S22</a>). This is consistent with the observations that drugs and clinical candidates binding to flat binding sites have lower LE and that only macrocyclic drugs in bRo5 space bind to flat binding sites for the nonredundant data sets (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>c). In contrast, nonmacrocyclic drugs display a slight preference for groove- and pocket-shaped binding sites. Although the number of nonredundant drugs and clinical candidates in these two categories is low, the complete bRo5 drug–target structure data set also retains a higher proportion of flat binding sites for macrocycles (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S23</a>). In line with these findings, the nPMI shape of bRo5 drugs indicates that macrocycles have a trend to be more disk- and spherelike than nonmacrocycles (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S23</a>). As already discussed above, recent investigations of macrocyclic natural products<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and of inhibitors of protein–protein interactions<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> provide additional support for these trends. Thus, one may conclude that the shape of macrocycles in combination with suitable rigidity and conformational preferences make them well suited for binding to difficult flat binding sites with sufficient potency and selectivity. The more linear and often aromatic nonmacrocycles appear to be somewhat better adapted for groove- and pocket-shaped binding sites, although macrocycles also bind to these sites.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Affinities, (b) ligand efficiencies (LE), (c) binding site shapes, and (d) normalized principle moment of inertia (nPMI) plot of bioactive conformations of drugs for beyond Ro5 macrocyclic (bRo5 MC, green) and beyond Ro5 nonmacrocyclic (bRo5 non-MC, red) drugs and targets. As a rule of thumb, Ro5 drug candidates are optimized to have 10 nM affinities, corresponding to a LE of 0.30 for a compound with a molecular weight of 500 Da (heavy atom count, HAC ≈ 36). This “guideline” for optimization is marked with a gray line in (b). Box plots show minimum and maximum values as whiskers; the 25th, , and 75th percentiles as boxes; and mean values as crosses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">5.3 Advantages of Macrocycles in Beyond Rule of 5 Space</h3><div class="NLM_p last">As highlighted in the literature<a onclick="showRef(event, 'ref26 ref27 ref48'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref48">(26, 27, 48)</a> and discussed above, macrocyclization of a linear compound can improve both oral bioavailability and affinity at the binding site of a specific target. As macrocyclization decreases flexibility, it has been suggested that decreased flexibility is the main reason for enrichment of macrocycles in bRo5 space.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> However, in the current data set we examine drugs and clinical candidates acting at different targets giving a global picture of flexibility in bRo5 space. This indicates that both macrocyclic and nonmacrocyclic drugs in bRo5 space have similar flexibilities and affinities across different targets and binding sites. Interestingly, we also find that disk and spherelike macrocycles bind more commonly to flat binding sites than rodlike nonmacrocycles, which more frequently target groove-shaped binding sites. Hence, we conclude that the unique ability of macrocycles to adopt disk- and spherelike shapes that are better suited for binding to flat and groove-shaped sites is an important reason for enrichment of macrocycles in bRo5 space. Improved permeability across membranes, which translates into higher oral bioavailability, has been demonstrated for a number of macrocycles and constitutes another reason.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> It should also be remembered that most macrocycles in bRo5 space are natural products that were discovered prior to target-based drug discovery and high throughput screening. Thus, their current enrichment as oral drugs in beyond Ro5 space also reflects the history of drug discovery for difficult targets while at the same time providing inspiration and insight for future discovery of oral drugs for difficult targets.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">6 Perspective</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">6.1 Designing Drugs beyond the Rule of 5</h3><div class="NLM_p">A key conclusion from this analysis is that drugs and clinical candidates in bRo5 space are better suited to modulate “difficult” and emerging target classes compared to Ro5 compliant drugs, in particular when binding sites are large and flat or groove-shaped (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). In previous analyses we demonstrated that 93% of the current oral drugs and clinical candidates in eRo5 and bRo5 space fall within an “outer limit” of physicochemical space where there still remains <i>a reasonable chance</i> to design orally bioavailable drugs.<a onclick="showRef(event, 'ref21 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref27">(21, 27)</a> This space was delineated by MW ≤ 1000 Da, −2 ≤ ClogP ≤ 10, HBD ≤ 6, HBA ≤ 15, PSA ≤ 250 Å<sup>2</sup>, and NRotB ≤ 20. The current target binding analysis and the previous oral bioavailability analyses therefore provide further guidance for design of bRo5 ligands for “difficult” targets, in particular with regard to the shape of the binding site, the flexibility of the ligand, and the LE that may be attained for binding sites of different shapes.</div><div class="NLM_p">When designing orals in bRo5 space, molecular weight may be increased up to 1000 Da which allows high-affinity binding to “difficult” extended, open, flat, and groove-shaped binding sites that bury up to 800–900 Å<sup>2</sup> of ligand surface area (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S25</a>). In contrast, drugs that adhere strictly to the Ro5 bind to pockets and internal sites and bury up to 600 Å<sup>2</sup> of ligand surface area. Overall polarity at the ligand interface is similar for bRo5, eRo5, and Ro5 drugs and clinical candidates which is mirrored in their similar lipophilicity, i.e., ClogP values centered around 4. Hence, an increase in 2D PSA up to 250 Å<sup>2</sup> is often found at higher molecular weights in bRo5 space to maintain this balance of polarity. The majority of this increased PSA should originate from an increased number of HBA, as HBD must be strictly controlled at ≤6 to avoid reducing oral bioavailability. This corresponds well to the observed small increase in the number of ligand HBA atom interactions between the ligand and its target in bRo5 space as compared to eRo5 and Ro5 space, while ligand HBD atoms interactions are similar. Similar overall ligand–target affinities were observed between Ro5, eRo5, and bRo5 space, which resulted in reduced ligand efficiency in bRo5 space. Additional, detailed comparisons of the properties of ligand–target interfaces in each chemical space revealed that trends are similar between orals and parenterals for drugs and clinical candidates (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figures S24–28</a>). However, parenterals in bRo5 space have reduced LE and increased LLE as compared to orals in bRo5 space (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S27</a>). These differences are caused by drugs in parenteral bRo5 space which have high molecular weight and high polarity; they are often peptidic in nature and thereby difficult to administer orally. In conclusion, this analysis indicates that the same approaches for optimization of oral drugs can be applied in bRo5 space as in Ro5 space, provided that physicochemical properties are kept within the recently reported “outer limits”.<a onclick="showRef(event, 'ref21 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref27">(21, 27)</a></div><div class="NLM_p">Compounds in bRo5 space require an overall appropriate balance between flexibility and rigidity that allows them to bind with high affinity to the target without paying an unnecessary entropic penalty. Conformational flexibility may be reduced through macrocyclization and use of aromatic and substituted aliphatic rings and other π-systems, such as amides, with equal success but at different types of binding sites. Interestingly, macrocyclization appears to favor disk- and spherelike conformations that facilitate binding of the ligand to the very difficult flat binding sites. Incorporation of other rigidifying structural elements in nonmacrocycles is more prone to yield rodlike conformations that bind to groove-shaped binding sites. During optimization of beyond Ro5 drugs it is also important to adjust goals for LE. Compounds and series with a LE value below 0.30 kcal/(mol·HAC) are often not prioritized in drug discovery. However, during optimization of bRo5 drugs for difficult targets, leads with LE < 0.30 kcal/(mol·HAC) should not be discarded, as this analysis indicates that they retain a reasonable chance of having sufficient affinity for their target while still being developed into a “possible to be oral” drug space. Instead, the size and flexibility of the ligand and the shape of the target binding site should be taken into account, allowing progression of compounds that may give candidate drugs with ligand efficiencies of ≥0.12 kcal/(mol·HAC), a guideline that captures 90% of current oral drugs and clinical candidates in bRo5 space. Finally, intramolecular hydrogen bonding, saturation of efflux transporters with high doses, use of improved formulations, and capitalizing on selective transporter mediated distribution to target organs can also play a role in producing a cell permeable and orally bioavailable drug in bRo5 space.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of Interactions between Drugs and Clinical Candidates and Their Targets by Chemical Space<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0012.gif" alt="" id="GRAPHIC-d1862e953-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Property values shown are the 10th to 90th percentile of the all structures encompassing both orals and parenterals. Orals and parenterals show very similar property values (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figures S24–S28</a>).</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Boxes indicate differences between bRo5 space and small molecule drug space.</p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">LE values should be adjusted based on the size of the ligand and shape of binding site during optimization with the final aim of being in this range and as high as possible.</p></div></div><div></div></div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">6.2 Conclusion and Future Challenges</h3><div class="NLM_p">This analysis suggests that many of the currently recognized “difficult” targets, and the multitude of novel targets that are emerging from genomics and proteomics, are out of reach for drugs in Ro5 space but that they may be suited to manipulation by drugs in bRo5 space. Disk- and sphere-shaped macrocycles and rodlike nonmacrocycles in bRo5 space are particularly well suited to targets that have large, flat-, or groove-shaped binding sites, respectively. Orally bioavailable and cell permeable drugs for difficult targets can be designed within the current “outer limits” of physicochemical property space beyond which pharmacokinetics currently present an almost insurmountable challenge.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In order to capitalize on opportunities in bRo5 space, changes in the perception of target properties that are considered “druggable” and which ligand properties allow orally bioavailability are required. Realizing that physicochemical property space can be expanded and introduction of reduced LE goals, which should be tailored to binding site shape and size, will enable development of drugs in bRo5 space. It may also be important to focus optimization so that candidate drugs have an appropriate balance between rigidity and flexibility in order to obtain satisfactory potency and ADMET properties.</div><div class="NLM_p last">The increased interest in drug discovery for difficult targets is becoming apparent from the increasing number of relevant publications from industry and academia, the emergence of several companies specializing in this chemical space, and the growing number of approved drugs outside Ro5 chemical space.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Thus, it is important to consider the discoveries and breakthroughs that will be required to enhance drug discovery in bRo5 space. Improved lead generation for novel classes of targets with difficult binding sites will be key to success. Extrapolation of current trends indicates that natural products and peptides will continue to constitute valuable starting points in this chemical space.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Fragment-based lead generation is also beginning to prove its worth in delivering drugs and clinical candidates for difficult targets<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and may become of increasing value for bRo5 lead discovery. Compounds in bRo5 space that bind to difficult targets will be bigger and more complex than Ro5 drugs. Although significant progress has been made in preparation of complex structures, e.g., macrocycles,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> synthesis will likely remain difficult, time-consuming, and costly for the immediate future. Therefore, development of improved predictive methods would also have a major impact on bRo5 drug discovery by allowing only the most highly prioritized compounds to be selected for synthesis. Access to efficient and reliable methods for 3D conformer generation would allow more accurate predictions of flexibility, lipophilicities, and polar surface areas and facilitate modeling of cell permeability and target binding. Predictive models for cellular efflux and metabolism would also be extremely valuable, even though it may not be realistic to expect them to be developed in the near future. In summary, we believe that developments in lead generation, synthetic methodology, and predictive methods, in combination with insights into how targets are engaged and improved understanding of ADMET properties, will allow more effective drug discovery in bRo5 space in the near future. It is our hope that this will ultimately contribute to an enhanced delivery of innovative medicines and increased pharmaceutical R&D efficiency.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01286">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72125" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72125" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01286" class="ext-link">10.1021/acs.jmedchem.5b01286</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full methods for generation of the data set and its anlysis; figures for all data discussed in the manuscript as well as figures for the full structure data set along with statistical anlysis. (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf">jm5b01286_si_001.pdf (3.37 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01286" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36152" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36152" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Kihlberg</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry—BMC, Uppsala University, Box
576, SE-751 23 Uppsala, Sweden</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7319121d5d181a1b1f111601143318161e1a5d06065d0016"><span class="__cf_email__" data-cfemail="3c565d5212575554505e594e5b7c57595155124949124f59">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bradley C. Doak</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry—BMC, Uppsala University, Box
576, SE-751 23 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Zheng</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry—BMC, Uppsala University, Box
576, SE-751 23 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Doreen Dobritzsch</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry—BMC, Uppsala University, Box
576, SE-751 23 Uppsala, Sweden</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors contributed to generation of data, to its analysis, and to the writing of the manuscript. They have approved the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Bradley C. Doak</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=BIO-d1862e997-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Bradley C. Doak</b> is a postdoctoral researcher in the Department of Organic Chemistry, Uppsala University, Sweden. Brad obtained a Bachelor of Medicinal Chemistry at the Department of Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Australia. He then completed a Ph.D. in Synthetic Medicinal Chemistry at Monash University in 2012, before taking his current position in Sweden. His research interests include beyond rule of 5 drug design, specifically macrocycles, as well as fragment-based drug design and difficult targets.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Jie Zheng</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=BIO-d1862e1002-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jie Zheng</b> holds a Masters of Organic Chemistry at the Department of Organic Chemistry, Uppsala University, Sweden. She obtained a Bachelor of Environmental Engineering from Shandong University of Technology, China, and then went on to study organic chemistry at Umeå University and Uppsala University, Sweden. Following research in environmental and technical chemistry, her research interests now include computational chemistry with an emphasis on how compounds bind to their targets and applications in molecular design.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Doreen Dobritzsch</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=BIO-d1862e1007-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Doreen Dobritzsch</b> is a senior lecturer in Biochemistry at Uppsala University, Sweden, since 2013. Previously she held an Assistant Professorship at the Karolinska Institute, where she also performed her postdoctoral studies after obtaining a Ph.D. in Biochemistry from the Martin-Luther-Universität Halle-Wittenberg, Germany, in 1999. Her research in the field of structural biochemistry is focused on structure–function relationships in enzymes, primarily those involved in pyrimidine degradation, and interactions occurring between immune molecules and self-antigens in rheumatoid arthritis.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Jan Kihlberg</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=BIO-d1862e1012-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jan Kihlberg</b> holds a Chair in Organic Chemistry at Uppsala University, Sweden, since 2013. During the previous 10 years at AstraZeneca R&D Mölndal he held positions as Director of Medicinal Chemistry, then as Director of Competitive Intelligence and Business Foresight Analysis. He became Professor in Organic Chemistry at Umeå University in 1996 after having established his research group at Lund Institute of Technology in 1991. He holds a Ph.D. in Organic Chemistry from Lund Institute of Technology. His key research interests are to understand what properties convey cell permeability and target binding to compounds outside the rule of 5 as well as studies of the chemical biology of glycopeptides, peptides, and their mimetics.</p></figure></div><div class="ack" id="ACK-d1862e1036-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">B.C.D. was supported by a postdoctoral fellowship funded by Uppsala University. The authors thank Prof. Helena Danielson, Uppsala University, Sweden, for providing valuable comments on the manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i25" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i25"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i26" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i26"> Abbreviations Used</h2><tr><td class="NLM_term">bRo5</td><td class="NLM_def"><p class="first last">beyond rule of 5</p></td></tr><tr><td class="NLM_term">eRo5</td><td class="NLM_def"><p class="first last">extended rule of 5</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">nPMI</td><td class="NLM_def"><p class="first last">normalized principle moment of inertia</p></td></tr><tr><td class="NLM_term">QED</td><td class="NLM_def"><p class="first last">quantitative estimate of druglikeness</p></td></tr><tr><td class="NLM_term">Ro5</td><td class="NLM_def"><p class="first last">rule of 5</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82140" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82140" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Trends in the exploitation of novel drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">579</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span><span class="refDoi"> DOI: 10.1038/nrd3478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21804595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=579-590&author=M.+Rask-Andersenauthor=M.+S.+Almenauthor=H.+B.+Schioth&title=Trends+in+the+exploitation+of+novel+drug+targets&doi=10.1038%2Fnrd3478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in the exploitation of novel drug targets</span></div><div class="casAuthors">Rask-Andersen, Mathias; Almen, Markus Saellman; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery and exploitation of new drug targets is a key focus for both the pharmaceutical industry and academic biomedical research.  To provide an insight into trends in the exploitation of new drug targets, we have analyzed the drugs that were approved by the US Food and Drug Administration during the past three decades and examd. the interactions of these drugs with therapeutic targets that are encoded by the human genome, using the DrugBank database and extensive manual curation.  We have identified 435 effect-mediating drug targets in the human genome, which are modulated by 989 unique drugs, through 2,242 drug-target interactions.  We also analyze trends in the introduction of drugs that modulate previously unexploited targets, and discuss the network pharmacol. of the drugs in our data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo19FXXFB-PsLVg90H21EOLACvtfcHk0lje2FLV8Nm49A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D&md5=9255431a28f2c07759b0f36975711f84</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3478%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DAlmen%26aufirst%3DM.%2BS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DTrends%2520in%2520the%2520exploitation%2520of%2520novel%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D579%26epage%3D590%26doi%3D10.1038%2Fnrd3478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bunnage, M. E.</span><span> </span><span class="NLM_article-title">Getting pharmaceutical R&D back on target</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/nchembio.581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchembio.581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21587251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtF2lur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=335-339&author=M.+E.+Bunnage&title=Getting+pharmaceutical+R%26D+back+on+target&doi=10.1038%2Fnchembio.581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Getting pharmaceutical R&D back on target</span></div><div class="casAuthors">Bunnage, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">335-339</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is in a period of crisis due to the low no. of new drug approvals relative to the high levels of R&D investment.  It is argued here that improving the quality of target selection is the single most important factor to transform industry productivity and bring innovative new medicines to patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyfBkdRkjR8LVg90H21EOLACvtfcHk0lgDgn3hXm3M_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtF2lur4%253D&md5=458abeefd10325afc74b5ced717d3b40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.581%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26atitle%3DGetting%2520pharmaceutical%2520R%2526D%2520back%2520on%2520target%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D335%26epage%3D339%26doi%3D10.1038%2Fnchembio.581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kinch, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoyer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patridge, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plummer, M.</span><span> </span><span class="NLM_article-title">Target selection for FDA-approved medicines</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">784</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.drudis.2014.11.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=784-789&author=M.+S.+Kinchauthor=D.+Hoyerauthor=E.+Patridgeauthor=M.+Plummer&title=Target+selection+for+FDA-approved+medicines&doi=10.1016%2Fj.drudis.2014.11.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DKinch%26aufirst%3DM.%2BS.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DPatridge%26aufirst%3DE.%26aulast%3DPlummer%26aufirst%3DM.%26atitle%3DTarget%2520selection%2520for%2520FDA-approved%2520medicines%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D784%26epage%3D789%26doi%3D10.1016%2Fj.drudis.2014.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Scannell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanckley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boldon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warrington, B.</span><span> </span><span class="NLM_article-title">Diagnosing the decline in pharmaceutical R&D efficiency</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1038/nrd3681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22378269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=191-200&author=J.+W.+Scannellauthor=A.+Blanckleyauthor=H.+Boldonauthor=B.+Warrington&title=Diagnosing+the+decline+in+pharmaceutical+R%26D+efficiency&doi=10.1038%2Fnrd3681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Diagnosing the decline in pharmaceutical R&D efficiency</span></div><div class="casAuthors">Scannell, Jack W.; Blanckley, Alex; Boldon, Helen; Warrington, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-200</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 60 years have seen huge advances in many of the scientific, technol. and managerial factors that should tend to raise the efficiency of com. drug research and development (RD).  Yet the no. of new drugs approved per billion US dollars spent on RD has halved roughly every 9 years since 1950, falling around 80-fold in inflation-adjusted terms.  There have been many proposed solns. to the problem of declining RD efficiency.  However, their apparent lack of impact so far and the contrast between improving inputs and declining output in terms of the no. of new drugs make it sensible to ask whether the underlying problems have been correctly diagnosed.  Here, we discuss four factors that we consider to be primary causes, which we call the 'better than the Beatles' problem; the 'cautious regulator' problem; the 'throw money at it' tendency; and the 'basic research-brute force' bias.  Our aim is to provoke a more systematic anal. of the causes of the decline in RD efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5e_81DC6r8LVg90H21EOLACvtfcHk0lgDgn3hXm3M_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D&md5=db3e4548260e55f47a9964e8d1b216fd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3681%26sid%3Dliteratum%253Aachs%26aulast%3DScannell%26aufirst%3DJ.%2BW.%26aulast%3DBlanckley%26aufirst%3DA.%26aulast%3DBoldon%26aufirst%3DH.%26aulast%3DWarrington%26aufirst%3DB.%26atitle%3DDiagnosing%2520the%2520decline%2520in%2520pharmaceutical%2520R%2526D%2520efficiency%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D191%26epage%3D200%26doi%3D10.1038%2Fnrd3681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Hay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craighead, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Economides, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenthal, J.</span><span> </span><span class="NLM_article-title">Clinical development success rates for investigational drugs</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1038/nbt.2786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnbt.2786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24406927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvFyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=40-51&author=M.+Hayauthor=D.+W.+Thomasauthor=J.+L.+Craigheadauthor=C.+Economidesauthor=J.+Rosenthal&title=Clinical+development+success+rates+for+investigational+drugs&doi=10.1038%2Fnbt.2786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical development success rates for investigational drugs</span></div><div class="casAuthors">Hay, Michael; Thomas, David W.; Craighead, John L.; Economides, Celia; Rosenthal, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-51</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The most comprehensive survey of clin. success rates across the drug industry to date shows productivity may be even lower than previous ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdqyRgsoFYDbVg90H21EOLACvtfcHk0lgDgn3hXm3M_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvFyjsA%253D%253D&md5=87a63ec8d8bb213b3d42519395725038</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2786%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DD.%2BW.%26aulast%3DCraighead%26aufirst%3DJ.%2BL.%26aulast%3DEconomides%26aufirst%3DC.%26aulast%3DRosenthal%26aufirst%3DJ.%26atitle%3DClinical%2520development%2520success%2520rates%2520for%2520investigational%2520drugs%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D40%26epage%3D51%26doi%3D10.1038%2Fnbt.2786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Cook, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexander, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">March, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satterthwaite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pangalos, M. N.</span><span> </span><span class="NLM_article-title">Lessons learned from the fate of AstraZeneca’s drug pipeline: a five-dimensional framework</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1038/nrd4309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd4309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24833294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=419-431&author=D.+Cookauthor=D.+Brownauthor=R.+Alexanderauthor=R.+Marchauthor=P.+Morganauthor=G.+Satterthwaiteauthor=M.+N.+Pangalos&title=Lessons+learned+from+the+fate+of+AstraZeneca%E2%80%99s+drug+pipeline%3A+a+five-dimensional+framework&doi=10.1038%2Fnrd4309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the fate of AstraZeneca's drug pipeline: a five-dimensional framework</span></div><div class="casAuthors">Cook, David; Brown, Dearg; Alexander, Robert; March, Ruth; Morgan, Paul; Satterthwaite, Gemma; Pangalos, Menelas N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">419-431</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Maintaining research and development (R&D) productivity at a sustainable level is one of the main challenges currently facing the pharmaceutical industry.  In this article, we discuss the results of a comprehensive longitudinal review of AstraZeneca's small-mol. drug projects from 2005 to 2010.  The anal. allowed us to establish a framework based on the five most important tech. determinants of project success and pipeline quality, which we describe as the five 'R's: the right target, the right patient, the right tissue, the right safety and the right com. potential.  A sixth factor - the right culture - is also crucial in encouraging effective decision-making based on these tech. determinants.  AstraZeneca is currently applying this framework to guide its R&D teams, and although it is too early to demonstrate whether this has improved the company's R&D productivity, we present our data and anal. here in the hope that it may assist the industry overall in addressing this key challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4yhPFZOUNO7Vg90H21EOLACvtfcHk0ljquBQZsG9uJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyqtrw%253D&md5=746a86c7223f170f7e6c4f9817760e5b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4309%26sid%3Dliteratum%253Aachs%26aulast%3DCook%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DMarch%26aufirst%3DR.%26aulast%3DMorgan%26aufirst%3DP.%26aulast%3DSatterthwaite%26aufirst%3DG.%26aulast%3DPangalos%26aufirst%3DM.%2BN.%26atitle%3DLessons%2520learned%2520from%2520the%2520fate%2520of%2520AstraZeneca%25E2%2580%2599s%2520drug%2520pipeline%253A%2520a%2520five-dimensional%2520framework%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D419%26epage%3D431%26doi%3D10.1038%2Fnrd4309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">International Human Genome Sequencing Consortium.</span><span> </span><span class="NLM_article-title">Finishing the euchromatic sequence of the human genome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">431</span><span class="NLM_x">, </span> <span class="NLM_fpage">931</span><span class="NLM_x">–</span> <span class="NLM_lpage">945</span><span class="refDoi"> DOI: 10.1038/nature03001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature03001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=15496913" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=431&publication_year=2004&pages=931-945&author=International+Human+Genome+Sequencing+Consortium.&title=Finishing+the+euchromatic+sequence+of+the+human+genome&doi=10.1038%2Fnature03001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature03001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03001%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DFinishing%2520the%2520euchromatic%2520sequence%2520of%2520the%2520human%2520genome%26jtitle%3DNature%26date%3D2004%26volume%3D431%26spage%3D931%26epage%3D945%26doi%3D10.1038%2Fnature03001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nirujogi, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manda, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaerkady, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madugundu, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelkar, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isserlin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthusamy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leal-Rojas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasrabuddhe, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Advani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renuse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selvan, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nanjappa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radhakrishnan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbannayya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raju, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreenivasamurthy, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathe, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Datta, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbannayya, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yelamanchi, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopalan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murthy, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syed, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mudgal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freed, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahari, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahadevan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satishchandra, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirdeshmukh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maitra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drake, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halushka, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacobuzio-Donahue, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span> </span><span class="NLM_article-title">A draft map of the human proteome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span><span class="refDoi"> DOI: 10.1038/nature13302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature13302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24870542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=575-581&author=M.+S.+Kimauthor=S.+M.+Pintoauthor=D.+Getnetauthor=R.+S.+Nirujogiauthor=S.+S.+Mandaauthor=R.+Chaerkadyauthor=A.+K.+Madugunduauthor=D.+S.+Kelkarauthor=R.+Isserlinauthor=S.+Jainauthor=J.+K.+Thomasauthor=B.+Muthusamyauthor=P.+Leal-Rojasauthor=P.+Kumarauthor=N.+A.+Sahasrabuddheauthor=L.+Balakrishnanauthor=J.+Advaniauthor=B.+Georgeauthor=S.+Renuseauthor=L.+D.+Selvanauthor=A.+H.+Patilauthor=V.+Nanjappaauthor=A.+Radhakrishnanauthor=S.+Prasadauthor=T.+Subbannayyaauthor=R.+Rajuauthor=M.+Kumarauthor=S.+K.+Sreenivasamurthyauthor=A.+Marimuthuauthor=G.+J.+Satheauthor=S.+Chavanauthor=K.+K.+Dattaauthor=Y.+Subbannayyaauthor=A.+Sahuauthor=S.+D.+Yelamanchiauthor=S.+Jayaramauthor=P.+Rajagopalanauthor=J.+Sharmaauthor=K.+R.+Murthyauthor=N.+Syedauthor=R.+Goelauthor=A.+A.+Khanauthor=S.+Ahmadauthor=G.+Deyauthor=K.+Mudgalauthor=A.+Chatterjeeauthor=T.+C.+Huangauthor=J.+Zhongauthor=X.+Wuauthor=P.+G.+Shawauthor=D.+Freedauthor=M.+S.+Zahariauthor=K.+K.+Mukherjeeauthor=S.+Shankarauthor=A.+Mahadevanauthor=H.+Lamauthor=C.+J.+Mitchellauthor=S.+K.+Shankarauthor=P.+Satishchandraauthor=J.+T.+Schroederauthor=R.+Sirdeshmukhauthor=A.+Maitraauthor=S.+D.+Leachauthor=C.+G.+Drakeauthor=M.+K.+Halushkaauthor=T.+S.+Prasadauthor=R.+H.+Hrubanauthor=C.+L.+Kerrauthor=G.+D.+Baderauthor=C.+A.+Iacobuzio-Donahueauthor=H.+Gowdaauthor=A.+Pandey&title=A+draft+map+of+the+human+proteome&doi=10.1038%2Fnature13302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A draft map of the human proteome</span></div><div class="casAuthors">Kim, Min-Sik; Pinto, Sneha M.; Getnet, Derese; Nirujogi, Raja Sekhar; Manda, Srikanth S.; Chaerkady, Raghothama; Madugundu, Anil K.; Kelkar, Dhanashree S.; Isserlin, Ruth; Jain, Shobhit; Thomas, Joji K.; Muthusamy, Babylakshmi; Leal-Rojas, Pamela; Kumar, Praveen; Sahasrabuddhe, Nandini A.; Balakrishnan, Lavanya; Advani, Jayshree; George, Bijesh; Renuse, Santosh; Selvan, Lakshmi Dhevi N.; Patil, Arun H.; Nanjappa, Vishalakshi; Radhakrishnan, Aneesha; Prasad, Samarjeet; Subbannayya, Tejaswini; Raju, Rajesh; Kumar, Manish; Sreenivasamurthy, Sreelakshmi K.; Marimuthu, Arivusudar; Sathe, Gajanan J.; Chavan, Sandip; Datta, Keshava K.; Subbannayya, Yashwanth; Sahu, Apeksha; Yelamanchi, Soujanya D.; Jayaram, Savita; Rajagopalan, Pavithra; Sharma, Jyoti; Murthy, Krishna R.; Syed, Nazia; Goel, Renu; Khan, Aafaque A.; Ahmad, Sartaj; Dey, Gourav; Mudgal, Keshav; Chatterjee, Aditi; Huang, Tai-Chung; Zhong, Jun; Wu, Xinyan; Shaw, Patrick G.; Freed, Donald; Zahari, Muhammad S.; Mukherjee, Kanchan K.; Shankar, Subramanian; Mahadevan, Anita; Lam, Henry; Mitchell, Christopher J.; Shankar, Susarla Krishna; Satishchandra, Parthasarathy; Schroeder, John T.; Sirdeshmukh, Ravi; Maitra, Anirban; Leach, Steven D.; Drake, Charles G.; Halushka, Marc K.; Prasad, T. S. Keshava; Hruban, Ralph H.; Kerr, Candace L.; Bader, Gary D.; Iacobuzio-Donahue, Christine A.; Gowda, Harsha; Pandey, Akhilesh</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7502</span>),
    <span class="NLM_cas:pages">575-581</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of human genome sequence has transformed biomedical research over the past decade.  However, an equiv. map for the human proteome with direct measurements of proteins and peptides does not exist yet.  Here we present a draft map of the human proteome using high-resoln. Fourier-transform mass spectrometry.  In-depth proteomic profiling of 30 histol. normal human samples, including 17 adult tissues, 7 fetal tissues and 6 purified primary hematopoietic cells, resulted in identification of proteins encoded by 17,294 genes accounting for approx. 84% of the total annotated protein-coding genes in humans.  A unique and comprehensive strategy for proteogenomic anal. enabled us to discover a no. of novel protein-coding regions, which includes translated pseudogenes, non-coding RNAs and upstream open reading frames.  This large human proteome catalog (available as an interactive web-based resource at http://www.humanproteomemap.org) will complement available human genome and transcriptome data to accelerate biomedical research in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNW2lqYhs8ULVg90H21EOLACvtfcHk0ljquBQZsG9uJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCrtrc%253D&md5=facf459fc3d9c8258fba1c1e94a378c6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature13302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13302%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DPinto%26aufirst%3DS.%2BM.%26aulast%3DGetnet%26aufirst%3DD.%26aulast%3DNirujogi%26aufirst%3DR.%2BS.%26aulast%3DManda%26aufirst%3DS.%2BS.%26aulast%3DChaerkady%26aufirst%3DR.%26aulast%3DMadugundu%26aufirst%3DA.%2BK.%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DIsserlin%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DJ.%2BK.%26aulast%3DMuthusamy%26aufirst%3DB.%26aulast%3DLeal-Rojas%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DSahasrabuddhe%26aufirst%3DN.%2BA.%26aulast%3DBalakrishnan%26aufirst%3DL.%26aulast%3DAdvani%26aufirst%3DJ.%26aulast%3DGeorge%26aufirst%3DB.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DSelvan%26aufirst%3DL.%2BD.%26aulast%3DPatil%26aufirst%3DA.%2BH.%26aulast%3DNanjappa%26aufirst%3DV.%26aulast%3DRadhakrishnan%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DSubbannayya%26aufirst%3DT.%26aulast%3DRaju%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DM.%26aulast%3DSreenivasamurthy%26aufirst%3DS.%2BK.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DSathe%26aufirst%3DG.%2BJ.%26aulast%3DChavan%26aufirst%3DS.%26aulast%3DDatta%26aufirst%3DK.%2BK.%26aulast%3DSubbannayya%26aufirst%3DY.%26aulast%3DSahu%26aufirst%3DA.%26aulast%3DYelamanchi%26aufirst%3DS.%2BD.%26aulast%3DJayaram%26aufirst%3DS.%26aulast%3DRajagopalan%26aufirst%3DP.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DMurthy%26aufirst%3DK.%2BR.%26aulast%3DSyed%26aufirst%3DN.%26aulast%3DGoel%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BA.%26aulast%3DAhmad%26aufirst%3DS.%26aulast%3DDey%26aufirst%3DG.%26aulast%3DMudgal%26aufirst%3DK.%26aulast%3DChatterjee%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DT.%2BC.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DShaw%26aufirst%3DP.%2BG.%26aulast%3DFreed%26aufirst%3DD.%26aulast%3DZahari%26aufirst%3DM.%2BS.%26aulast%3DMukherjee%26aufirst%3DK.%2BK.%26aulast%3DShankar%26aufirst%3DS.%26aulast%3DMahadevan%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DH.%26aulast%3DMitchell%26aufirst%3DC.%2BJ.%26aulast%3DShankar%26aufirst%3DS.%2BK.%26aulast%3DSatishchandra%26aufirst%3DP.%26aulast%3DSchroeder%26aufirst%3DJ.%2BT.%26aulast%3DSirdeshmukh%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DS.%2BD.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DHalushka%26aufirst%3DM.%2BK.%26aulast%3DPrasad%26aufirst%3DT.%2BS.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DKerr%26aufirst%3DC.%2BL.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DIacobuzio-Donahue%26aufirst%3DC.%2BA.%26aulast%3DGowda%26aufirst%3DH.%26aulast%3DPandey%26aufirst%3DA.%26atitle%3DA%2520draft%2520map%2520of%2520the%2520human%2520proteome%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D575%26epage%3D581%26doi%3D10.1038%2Fnature13302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Wilhelm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahne, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghaddas Gholami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberenz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziegler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butzmann, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gessulat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marx, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemeer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnatbaum, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wenschuh, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mollenhauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slotta-Huspenina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boese, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstmair, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faerber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span> </span><span class="NLM_article-title">Mass-spectrometry-based draft of the human proteome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">582</span><span class="NLM_x">–</span> <span class="NLM_lpage">587</span><span class="refDoi"> DOI: 10.1038/nature13319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature13319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24870543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=582-587&author=M.+Wilhelmauthor=J.+Schleglauthor=H.+Hahneauthor=A.+Moghaddas+Gholamiauthor=M.+Lieberenzauthor=M.+M.+Savitskiauthor=E.+Zieglerauthor=L.+Butzmannauthor=S.+Gessulatauthor=H.+Marxauthor=T.+Mathiesonauthor=S.+Lemeerauthor=K.+Schnatbaumauthor=U.+Reimerauthor=H.+Wenschuhauthor=M.+Mollenhauerauthor=J.+Slotta-Huspeninaauthor=J.+H.+Boeseauthor=M.+Bantscheffauthor=A.+Gerstmairauthor=F.+Faerberauthor=B.+Kuster&title=Mass-spectrometry-based+draft+of+the+human+proteome&doi=10.1038%2Fnature13319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mass-spectrometry-based draft of the human proteome</span></div><div class="casAuthors">Wilhelm, Mathias; Schlegl, Judith; Hahne, Hannes; Gholami, Amin Moghaddas; Lieberenz, Marcus; Savitski, Mikhail M.; Ziegler, Emanuel; Butzmann, Lars; Gessulat, Siegfried; Marx, Harald; Mathieson, Toby; Lemeer, Simone; Schnatbaum, Karsten; Reimer, Ulf; Wenschuh, Holger; Mollenhauer, Martin; Slotta-Huspenina, Julia; Boese, Joos-Hendrik; Bantscheff, Marcus; Gerstmair, Anja; Faerber, Franz; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7502</span>),
    <span class="NLM_cas:pages">582-587</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteomes are characterized by large protein-abundance differences, cell-type- and time-dependent expression patterns and post-translational modifications, all of which carry biol. information that is not accessible by genomics or transcriptomics.  Here we present a mass-spectrometry-based draft of the human proteome and a public, high-performance, in-memory database for real-time anal. of terabytes of big data, called ProteomicsDB.  The information assembled from human tissues, cell lines and body fluids enabled estn. of the size of the protein-coding genome, and identified organ-specific proteins and a large no. of translated lincRNAs (long intergenic non-coding RNAs).  Anal. of mRNA and protein-expression profiles of human tissues revealed conserved control of protein abundance, and integration of drug-sensitivity data enabled the identification of proteins predicting resistance or sensitivity.  The proteome profiles also hold considerable promise for analyzing the compn. and stoichiometry of protein complexes.  ProteomicsDB thus enables navigation of proteomes, provides biol. insight and fosters the development of proteomic technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK5rb2mey2G7Vg90H21EOLACvtfcHk0liCSRyblhUSNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCrt7k%253D&md5=b9e4b32b349b92160996c860de293191</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature13319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13319%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DMoghaddas%2BGholami%26aufirst%3DA.%26aulast%3DLieberenz%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DZiegler%26aufirst%3DE.%26aulast%3DButzmann%26aufirst%3DL.%26aulast%3DGessulat%26aufirst%3DS.%26aulast%3DMarx%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DLemeer%26aufirst%3DS.%26aulast%3DSchnatbaum%26aufirst%3DK.%26aulast%3DReimer%26aufirst%3DU.%26aulast%3DWenschuh%26aufirst%3DH.%26aulast%3DMollenhauer%26aufirst%3DM.%26aulast%3DSlotta-Huspenina%26aufirst%3DJ.%26aulast%3DBoese%26aufirst%3DJ.%2BH.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGerstmair%26aufirst%3DA.%26aulast%3DFaerber%26aufirst%3DF.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DMass-spectrometry-based%2520draft%2520of%2520the%2520human%2520proteome%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D582%26epage%3D587%26doi%3D10.1038%2Fnature13319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">The druggable genome: evaluation of drug targets in clinical trials suggests major shifts in molecular class and indication</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-011613-135943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1146%2Fannurev-pharmtox-011613-135943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24016212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=9-26&author=M.+Rask-Andersenauthor=S.+Masuramauthor=H.+B.+Schioth&title=The+druggable+genome%3A+evaluation+of+drug+targets+in+clinical+trials+suggests+major+shifts+in+molecular+class+and+indication&doi=10.1146%2Fannurev-pharmtox-011613-135943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The druggable genome: evaluation of drug targets in clinical trials suggests major shifts in molecular class and indication</span></div><div class="casAuthors">Rask-Andersen, Mathias; Masuram, Surendar; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-26</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The largest innovations within pharmaceutical development come through new compds. that have unique and novel modes of action.  These innovations commonly involve expanding the protein space targeted by pharmaceutical agents.  At present, information about drugs and drug targets is available online via public databases such as DrugBank and the Therapeutic Targets Database.  However, this information is biased, understandably so, toward established drugs and drug-target interactions.  To gain a better overview of the drug-targeted portion of the human proteome and the directions of current drug development, we developed a data set of clin. trial drug-target interactions based on CenterWatch's Drugs in Clin. Trials Database, one of the largest databases of its kind.  Our curation identified 475 potentially novel clin. trial drug targets.  This review aims to identify trends in drug development based on the potentially novel targets currently being explored in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrACyvG8vYfDLVg90H21EOLACvtfcHk0liCSRyblhUSNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVSqsbk%253D&md5=2a31db63098fe4a01c16f6179f3b88e3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-011613-135943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-011613-135943%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DMasuram%26aufirst%3DS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DThe%2520druggable%2520genome%253A%2520evaluation%2520of%2520drug%2520targets%2520in%2520clinical%2520trials%2520suggests%2520major%2520shifts%2520in%2520molecular%2520class%2520and%2520indication%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2014%26volume%3D54%26spage%3D9%26epage%3D26%26doi%3D10.1146%2Fannurev-pharmtox-011613-135943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lhDvvllaC-sYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Venkatesan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rual, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vazquez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stelzl, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmens, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirozane-Kishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zenkner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goh, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yildirim, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonis, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinzmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebreab, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahalie, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cevik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Smet, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolyar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinayagam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szeto, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borick, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dricot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klitgord, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lalowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cusick, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavernier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wanker, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barabasi, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidal, M.</span><span> </span><span class="NLM_article-title">An empirical framework for binary interactome mapping</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1038/nmeth.1280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnmeth.1280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=19060904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=83-90&author=K.+Venkatesanauthor=J.+F.+Rualauthor=A.+Vazquezauthor=U.+Stelzlauthor=I.+Lemmensauthor=T.+Hirozane-Kishikawaauthor=T.+Haoauthor=M.+Zenknerauthor=X.+Xinauthor=K.+I.+Gohauthor=M.+A.+Yildirimauthor=N.+Simonisauthor=K.+Heinzmannauthor=F.+Gebreabauthor=J.+M.+Sahalieauthor=S.+Cevikauthor=C.+Simonauthor=A.+S.+de+Smetauthor=E.+Dannauthor=A.+Smolyarauthor=A.+Vinayagamauthor=H.+Yuauthor=D.+Szetoauthor=H.+Borickauthor=A.+Dricotauthor=N.+Klitgordauthor=R.+R.+Murrayauthor=C.+Linauthor=M.+Lalowskiauthor=J.+Timmauthor=K.+Rauauthor=C.+Booneauthor=P.+Braunauthor=M.+E.+Cusickauthor=F.+P.+Rothauthor=D.+E.+Hillauthor=J.+Tavernierauthor=E.+E.+Wankerauthor=A.+L.+Barabasiauthor=M.+Vidal&title=An+empirical+framework+for+binary+interactome+mapping&doi=10.1038%2Fnmeth.1280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An empirical framework for binary interactome mapping</span></div><div class="casAuthors">Venkatesan, Kavitha; Rual, Jean-Francois; Vazquez, Alexei; Stelzl, Ulrich; Lemmens, Irma; Hirozane-Kishikawa, Tomoko; Hao, Tong; Zenkner, Martina; Xin, Xiaofeng; Goh, Kwang-Il; Yildirim, Muhammed A.; Simonis, Nicolas; Heinzmann, Kathrin; Gebreab, Fana; Sahalie, Julie M.; Cevik, Sebiha; Simon, Christophe; de Smet, Anne-Sophie; Dann, Elizabeth; Smolyar, Alex; Vinayagam, Arunachalam; Yu, Haiyuan; Szeto, David; Borick, Heather; Dricot, Amelie; Klitgord, Niels; Murray, Ryan R.; Lin, Chenwei; Lalowski, Maciej; Timm, Jan; Rau, Kirstin; Boone, Charles; Braun, Pascal; Cusick, Michael E.; Roth, Frederick P.; Hill, David E.; Tavernier, Jan; Wanker, Erich E.; Barabasi, Albert-Laszlo; Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-90</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several attempts have been made to systematically map protein-protein interaction, or 'interactome', networks.  However, it remains difficult to assess the quality and coverage of existing data sets.  Here the authors describe a framework that uses an empirically-based approach to rigorously dissect quality parameters of currently available human interactome maps.  The authors' results indicate that high-throughput yeast two-hybrid (HT-Y2H) interactions for human proteins are more precise than literature-curated interactions supported by a single publication, suggesting that HT-Y2H is suitable to map a significant portion of the human interactome.  The authors est. that the human interactome contains ∼130,000 binary interactions, most of which remain to be mapped.  Similar to ests. of DNA sequence data quality and genome size early in the Human Genome Project, ests. of protein interaction data quality and interactome size are crucial to establish the magnitude of the task of comprehensive human interactome mapping and to elucidate a path toward this goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9l4eKnzTSQ7Vg90H21EOLACvtfcHk0lhDvvllaC-sYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7%252FE&md5=7d796aad73a5a62b02880b5d14a9eaa2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1280%26sid%3Dliteratum%253Aachs%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DRual%26aufirst%3DJ.%2BF.%26aulast%3DVazquez%26aufirst%3DA.%26aulast%3DStelzl%26aufirst%3DU.%26aulast%3DLemmens%26aufirst%3DI.%26aulast%3DHirozane-Kishikawa%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DT.%26aulast%3DZenkner%26aufirst%3DM.%26aulast%3DXin%26aufirst%3DX.%26aulast%3DGoh%26aufirst%3DK.%2BI.%26aulast%3DYildirim%26aufirst%3DM.%2BA.%26aulast%3DSimonis%26aufirst%3DN.%26aulast%3DHeinzmann%26aufirst%3DK.%26aulast%3DGebreab%26aufirst%3DF.%26aulast%3DSahalie%26aufirst%3DJ.%2BM.%26aulast%3DCevik%26aufirst%3DS.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3Dde%2BSmet%26aufirst%3DA.%2BS.%26aulast%3DDann%26aufirst%3DE.%26aulast%3DSmolyar%26aufirst%3DA.%26aulast%3DVinayagam%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSzeto%26aufirst%3DD.%26aulast%3DBorick%26aufirst%3DH.%26aulast%3DDricot%26aufirst%3DA.%26aulast%3DKlitgord%26aufirst%3DN.%26aulast%3DMurray%26aufirst%3DR.%2BR.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DLalowski%26aufirst%3DM.%26aulast%3DTimm%26aufirst%3DJ.%26aulast%3DRau%26aufirst%3DK.%26aulast%3DBoone%26aufirst%3DC.%26aulast%3DBraun%26aufirst%3DP.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DRoth%26aufirst%3DF.%2BP.%26aulast%3DHill%26aufirst%3DD.%2BE.%26aulast%3DTavernier%26aufirst%3DJ.%26aulast%3DWanker%26aufirst%3DE.%2BE.%26aulast%3DBarabasi%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DAn%2520empirical%2520framework%2520for%2520binary%2520interactome%2520mapping%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D83%26epage%3D90%26doi%3D10.1038%2Fnmeth.1280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Zhang, Q. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thu, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisikirska, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefebvre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Accili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maniatis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Califano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Structure-based prediction of protein-protein interactions on a genome-wide scale</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">490</span><span class="NLM_x">, </span> <span class="NLM_fpage">556</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1038/nature11503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature11503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23023127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=556-560&author=Q.+C.+Zhangauthor=D.+Petreyauthor=L.+Dengauthor=L.+Qiangauthor=Y.+Shiauthor=C.+A.+Thuauthor=B.+Bisikirskaauthor=C.+Lefebvreauthor=D.+Acciliauthor=T.+Hunterauthor=T.+Maniatisauthor=A.+Califanoauthor=B.+Honig&title=Structure-based+prediction+of+protein-protein+interactions+on+a+genome-wide+scale&doi=10.1038%2Fnature11503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of protein-protein interactions on a genome-wide scale</span></div><div class="casAuthors">Zhang, Qiangfeng Cliff; Petrey, Donald; Deng, Lei; Qiang, Li; Shi, Yu; Thu, Chan Aye; Bisikirska, Brygida; Lefebvre, Celine; Accili, Domenico; Hunter, Tony; Maniatis, Tom; Califano, Andrea; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">556-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The genome-wide identification of pairs of interacting proteins is an important step in the elucidation of cell regulatory mechanisms.  Much of our present knowledge derives from high-throughput techniques such as the yeast two-hybrid assay and affinity purifn., as well as from manual curation of expts. on individual systems.  A variety of computational approaches based, for example, on sequence homol., gene co-expression and phylogenetic profiles, have also been developed for the genome-wide inference of protein-protein interactions (PPIs).  Yet comparative studies suggest that the development of accurate and complete repertoires of PPIs is still in its early stages.  Here we show that three-dimensional structural information can be used to predict PPIs with an accuracy and coverage that are superior to predictions based on non-structural evidence.  Moreover, an algorithm, termed PrePPI, which combines structural information with other functional clues, is comparable in accuracy to high-throughput expts., yielding over 30,000 high-confidence interactions for yeast and over 300,000 for human.  Exptl. tests of a no. of predictions demonstrate the ability of the PrePPI algorithm to identify unexpected PPIs of considerable biol. interest.  The surprising effectiveness of three-dimensional structural information can be attributed to the use of homol. models combined with the exploitation of both close and remote geometric relationships between proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIaJJWiy7hKLVg90H21EOLACvtfcHk0lhDvvllaC-sYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtb7P&md5=bb27a4b275f5d9c727859b172c6b6d0f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature11503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11503%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%2BC.%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DQiang%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DThu%26aufirst%3DC.%2BA.%26aulast%3DBisikirska%26aufirst%3DB.%26aulast%3DLefebvre%26aufirst%3DC.%26aulast%3DAccili%26aufirst%3DD.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DManiatis%26aufirst%3DT.%26aulast%3DCalifano%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DStructure-based%2520prediction%2520of%2520protein-protein%2520interactions%2520on%2520a%2520genome-wide%2520scale%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D556%26epage%3D560%26doi%3D10.1038%2Fnature11503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: progressing toward the reality</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1102</span><span class="NLM_x">–</span> <span class="NLM_lpage">1114</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lgoANLrQqMyAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0lgoANLrQqMyAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0lgoANLrQqMyAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">DrugPred: A structure-based approach to predict protein druggability developed using an extensive nonredundant data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">2842</span><span class="refDoi"> DOI: 10.1021/ci200266d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200266d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2829-2842&author=A.+Krasowskiauthor=D.+Muthasauthor=A.+Sarkarauthor=S.+Schmittauthor=R.+Brenk&title=DrugPred%3A+A+structure-based+approach+to+predict+protein+druggability+developed+using+an+extensive+nonredundant+data+set&doi=10.1021%2Fci200266d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DrugPred: A Structure-Based Approach To Predict Protein Druggability Developed Using an Extensive Nonredundant Data Set</span></div><div class="casAuthors">Krasowski, Agata; Muthas, Daniel; Sarkar, Aurijit; Schmitt, Stefan; Brenk, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2842</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Judging if a protein is able to bind orally available mols. with high affinity, i.e. if a protein is druggable, is an important step in target assessment.  In order to derive a structure-based method to predict protein druggability, a comprehensive, nonredundant data set contg. crystal structures of 71 druggable and 44 less druggable proteins was compiled by literature search and data mining.  This data set was subsequently used to train a structure-based druggability predictor (DrugPred) using partial least-squares projection to latent structures discriminant anal. (PLS-DA).  DrugPred performed well in discriminating druggable from less druggable binding sites for both internal and external predictions.  The method is robust against conformational changes in the binding site and outperforms previously published methods.  The superior performance of DrugPred is likely due to the size and compn. of the training set which, in contrast to most previously developed methods, only contains cavities that have evolved to bind a natural ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLISVIZ5UnXrVg90H21EOLACvtfcHk0liZYpjY9SBHgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN&md5=a352186705196d6c58b4a590e2848896</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci200266d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200266d%26sid%3Dliteratum%253Aachs%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DDrugPred%253A%2520A%2520structure-based%2520approach%2520to%2520predict%2520protein%2520druggability%2520developed%2520using%2520an%2520extensive%2520nonredundant%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2829%26epage%3D2842%26doi%3D10.1021%2Fci200266d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span> </span><span class="NLM_article-title">Development of a rule-based method for the assessment of protein druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">1038</span><span class="refDoi"> DOI: 10.1021/ci200613b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200613b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1027-1038&author=E.+Perolaauthor=L.+Hermanauthor=J.+Weiss&title=Development+of+a+rule-based+method+for+the+assessment+of+protein+druggability&doi=10.1021%2Fci200613b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Rule-Based Method for the Assessment of Protein Druggability</span></div><div class="casAuthors">Perola, Emanuele; Herman, Lee; Weiss, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1027-1038</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target selection is a crit. step in the majority of modern drug discovery programs.  The viability of a drug target depends on two components: biol. relevance and chem. tractability.  The concept of druggability was introduced to describe the second component, and it is defined as the ability of a target to bind a drug-like mol. with a therapeutically useful level of affinity.  To investigate the rules that govern druggability, we developed an algorithm to isolate and characterize the binding pockets of protein targets.  Using this algorithm, we performed a comparative anal. between the relevant pockets of 60 targets of approved drugs and a diverse set of 440 ligand-binding pockets.  As a result, we defined a preferred property space for druggable pockets based on five key properties (vol., depth, enclosure, percentage of charged residues and hydrophobicity), and we represented it with a set of simple rules.  These rules may be applicable in the future to evaluate the chem. tractability of prospective targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRGoeAUMXUbVg90H21EOLACvtfcHk0liZYpjY9SBHgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D&md5=49a4c07150749f6d09a062365512b8c2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci200613b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200613b%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DHerman%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520a%2520rule-based%2520method%2520for%2520the%2520assessment%2520of%2520protein%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1027%26epage%3D1038%26doi%3D10.1021%2Fci200613b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0liZYpjY9SBHgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Terrett, N.</span><span> </span><span class="NLM_article-title">Drugs in middle space</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">474</span><span class="NLM_x">–</span> <span class="NLM_lpage">475</span><span class="refDoi"> DOI: 10.1039/c2md90062a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1039%2Fc2md90062a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFShtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=474-475&author=N.+Terrett&title=Drugs+in+middle+space&doi=10.1039%2Fc2md90062a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs in middle space</span></div><div class="casAuthors">Terrett, Nick</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-475</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The paper by Lipinski et al. in 1997 first outlining the 'Rule of 5' suggested that poor absorption or permeability would result if compds. exceeded clearly defined hydrogen-bond counts, mol. wt. (MW) or lipophilicity (log D).  These simple and useful guidelines for drug design were rapidly applied throughout the pharmaceutical industry.  One particular consequence was a reluctance to contemplate investigation of any drug mol. with a mol. wt. exceeding 500 Daltons.  Many researchers now focus on the region of chem. space that exists between small mols. and biologics, based on the expectation that larger mols. (with MW between 500 and 1500) will more readily provide high affinity interactions with featureless protein surfaces, and yet can be optimized to achieve oral absorption and cell permeability.  The challenge for middle space research is to design similarly active compds. but in novel synthetically accessible forms, and to fine-tune phys. properties so that irresp. of mol. wt., the compds. are orally bioavailable and can penetrate cell membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorWAjFt2huMrVg90H21EOLACvtfcHk0ligIFWc9nbH7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFShtLs%253D&md5=2537549322060be5598b031d2cccb9ac</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2Fc2md90062a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md90062a%26sid%3Dliteratum%253Aachs%26aulast%3DTerrett%26aufirst%3DN.%26atitle%3DDrugs%2520in%2520middle%2520space%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D474%26epage%3D475%26doi%3D10.1039%2Fc2md90062a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Doak, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Over, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: insights from drugs and clinical candidates</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1115</span><span class="NLM_x">–</span> <span class="NLM_lpage">1142</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0ligIFWc9nbH7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span> </span><span class="NLM_article-title">A sorcerer’s apprentice and the rule of five: from rule-of-thumb to commandment and beyond</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">995</span><span class="NLM_x">–</span> <span class="NLM_lpage">997</span><span class="refDoi"> DOI: 10.1016/j.drudis.2007.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.drudis.2007.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18061876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A280%3ADC%252BD2sjivVGrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=995-997&author=C.+Abad-Zapatero&title=A+sorcerer%E2%80%99s+apprentice+and+the+rule+of+five%3A+from+rule-of-thumb+to+commandment+and+beyond&doi=10.1016%2Fj.drudis.2007.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A sorcerer's apprentice and The Rule of Five: from rule-of-thumb to commandment and beyond</span></div><div class="casAuthors">Abad-Zapatero Cele</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">995-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4k8xMsSJHYGLFM_lbR1VmfW6udTcc2ea36kimm8-oCrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjivVGrtQ%253D%253D&md5=d09c56d696d2977f7a4e5f16992631a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26atitle%3DA%2520sorcerer%25E2%2580%2599s%2520apprentice%2520and%2520the%2520rule%2520of%2520five%253A%2520from%2520rule-of-thumb%2520to%2520commandment%2520and%2520beyond%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D995%26epage%3D997%26doi%3D10.1016%2Fj.drudis.2007.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Zhang, M.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, B.</span><span> </span><span class="NLM_article-title">Drug discovery beyond the ‘rule-of-five’</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span><span class="refDoi"> DOI: 10.1016/j.copbio.2007.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.copbio.2007.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18035532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=478-488&author=M.-Q.+Zhangauthor=B.+Wilkinson&title=Drug+discovery+beyond+the+%E2%80%98rule-of-five%E2%80%99&doi=10.1016%2Fj.copbio.2007.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the 'rule-of-five'</span></div><div class="casAuthors">Zhang, Ming-Qiang; Wilkinson, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">478-488</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although a very useful guideline for orally bioavailable small-mol. drug design, the 'rule-of-five' (also known as 'Lipinski's rule of drug-likeness') has to some extent been overemphasized.  Firstly, only 51% of all FDA-approved small-mol. drugs are both used orally and comply with the 'rule-of-five'.  This does not even include the increasing no. of biologicals of which several have reached blockbuster' status.  Secondly, it does not cover natural product and semisynthetic natural product drugs, which constitute over one-third of all marketed small-mol. drugs.  A more balanced and programmatic approach to drug discovery should be more productive than to rely on an overemphasis of 'rule-of-five' compliance.  Rather it should consider proactively the development of parenteral drugs in parallel to oral drugs and to consider the development of therapeutic antibodies in parallel to small-mol. drugs.  These are particularly relevant for efforts against 'first-in-class' and/or particularly challenging targets such as proteases and those involving protein-protein interactions.  In addn., more effort should be invested in natural product research.  Emerging novel technologies such as synthetic biol. (genetic engineering of living organisms to produce small-mol. therapeutics) may address several challenging issues of natural product-based drug discovery including synthetic feasibility and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollnOuY9kGrbVg90H21EOLACvtfcHk0ligIFWc9nbH7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D&md5=4121024557295c4f241255e43b43ea44</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.-Q.%26aulast%3DWilkinson%26aufirst%3DB.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520%25E2%2580%2598rule-of-five%25E2%2580%2599%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D478%26epage%3D488%26doi%3D10.1016%2Fj.copbio.2007.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Walters, W. P.</span><span> </span><span class="NLM_article-title">Going further than Lipinski’s rule in drug design</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1517/17460441.2012.648612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1517%2F17460441.2012.648612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=99-107&author=W.+P.+Walters&title=Going+further+than+Lipinski%E2%80%99s+rule+in+drug+design&doi=10.1517%2F17460441.2012.648612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.648612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.648612%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26atitle%3DGoing%2520further%2520than%2520Lipinski%25E2%2580%2599s%2520rule%2520in%2520drug%2520design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D99%26epage%3D107%26doi%3D10.1517%2F17460441.2012.648612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chennamadhavuni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">How proteins bind macrocycles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">723</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span><span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0li9GYDiWbHVXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Driggers, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery - an underexploited structural class</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery+-+an+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0li9GYDiWbHVXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery%2520-%2520an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">295</span><span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0li9GYDiWbHVXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lhJqkmUBiZRbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhJqkmUBiZRbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muresan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Quantifying the chemical beauty of drugs</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">98</span><span class="refDoi"> DOI: 10.1038/nchem.1243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchem.1243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22270643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=90-98&author=G.+R.+Bickertonauthor=G.+V.+Paoliniauthor=J.+Besnardauthor=S.+Muresanauthor=A.+L.+Hopkins&title=Quantifying+the+chemical+beauty+of+drugs&doi=10.1038%2Fnchem.1243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chemical beauty of drugs</span></div><div class="casAuthors">Bickerton, G. Richard; Paolini, Gaia V.; Besnard, Jeremy; Muresan, Sorel; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">90-98</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-likeness is a key consideration when selecting compds. during the early stages of drug discovery.  However, evaluation of drug-likeness in abs. terms does not reflect adequately the whole spectrum of compd. quality.  More worryingly, widely used rules may inadvertently foster undesirable mol. property inflation as they permit the encroachment of rule-compliant compds. towards their boundaries.  We propose a measure of drug-likeness based on the concept of desirability called the quant. est. of drug-likeness (QED).  The empirical rationale of QED reflects the underlying distribution of mol. properties.  QED is intuitive, transparent, straightforward to implement in many practical settings and allows compds. to be ranked by their relative merit.  We extended the utility of QED by applying it to the problem of mol. target druggability assessment by prioritizing a large set of published bioactive compds.  The measure may also capture the abstr. notion of aesthetics in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCb7vrfeUodrVg90H21EOLACvtfcHk0lhJqkmUBiZRbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D&md5=7ab125d4ab381924f965d072695f7432</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1243%26sid%3Dliteratum%253Aachs%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DQuantifying%2520the%2520chemical%2520beauty%2520of%2520drugs%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D90%26epage%3D98%26doi%3D10.1038%2Fnchem.1243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span> </span><span class="NLM_article-title">Dependence of molecular properties on proteomic family for marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3451</span><span class="NLM_x">–</span> <span class="NLM_lpage">3453</span><span class="refDoi"> DOI: 10.1021/jm0603825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3451-3453&author=M.+Viethauthor=J.+J.+Sutherland&title=Dependence+of+molecular+properties+on+proteomic+family+for+marketed+oral+drugs&doi=10.1021%2Fjm0603825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span></div><div class="casAuthors">Vieth, Michal; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3451-3453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assocn. of drugs with their proteomic family reveals that mol. properties of drugs targeting proteases, lipid and peptide G-protein-coupled receptors (GPCRs), and nuclear hormone receptors significantly exceed limits for some properties in the "rule of five", while drugs targeting cytochrome P450s, biogenic amine GPCRs, and transporters have significantly lower values for certain properties.  Also, the variation in drug targets appears to be a factor explaining increasing mol. wt. over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC7vpvPxrV7Vg90H21EOLACvtfcHk0li8ZfXCv232cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D&md5=9272583ace87f6080bd0e1b01ad7c657</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0603825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603825%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DDependence%2520of%2520molecular%2520properties%2520on%2520proteomic%2520family%2520for%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3451%26epage%3D3453%26doi%3D10.1021%2Fjm0603825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">The influence of target family and functional activity on the physicochemical properties of pre-clinical compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2969</span><span class="NLM_x">–</span> <span class="NLM_lpage">2978</span><span class="refDoi"> DOI: 10.1021/jm0512185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0512185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtlOit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2969-2978&author=R.+Morphy&title=The+influence+of+target+family+and+functional+activity+on+the+physicochemical+properties+of+pre-clinical+compounds&doi=10.1021%2Fjm0512185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Influence of Target Family and Functional Activity on the Physicochemical Properties of Pre-Clinical Compounds</span></div><div class="casAuthors">Morphy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2969-2978</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The target families of greatest interest in drug discovery can be differentiated on the basis of the physicochem. properties of their preclin. ligands.  The ligands for peptidergic targets, such as peptide G protein-coupled receptors (GPCRs) and integrin receptors, possess significantly higher median property values than those for aminergic targets, such as monoamine transporters and GPCRs.  The ligands for peptide GPCRs were found to be less efficient, in terms of their binding energy per unit of mol. wt. or lipophilicity, than ligands for monoamine GPCRs.  The changes in the property values during the optimization process were found to vary only slightly across the target families, with the main determinant of the drug-likeness of the optimized compds. being the profile of the starting compds.  Agonists for monoamine GPCRs, opioid receptors and ion channels were typically smaller and less lipophilic than the antagonists, but there was no difference between the agonists and the antagonists for peptide GPCRs and nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0EegclZUHNbVg90H21EOLACvtfcHk0li8ZfXCv232cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtlOit7g%253D&md5=fcf63bff570d58d56492b46ddecb5eb1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm0512185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0512185%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DThe%2520influence%2520of%2520target%2520family%2520and%2520functional%2520activity%2520on%2520the%2520physicochemical%2520properties%2520of%2520pre-clinical%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2969%26epage%3D2978%26doi%3D10.1021%2Fjm0512185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0li8ZfXCv232cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Nero, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morton, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holien, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wielens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, M. W.</span><span> </span><span class="NLM_article-title">Oncogenic protein interfaces: small molecules, big challenges</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nrc3690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrc3690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24622521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=248-262&author=T.+L.+Neroauthor=C.+J.+Mortonauthor=J.+K.+Holienauthor=J.+Wielensauthor=M.+W.+Parker&title=Oncogenic+protein+interfaces%3A+small+molecules%2C+big+challenges&doi=10.1038%2Fnrc3690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic protein interfaces: small molecules, big challenges</span></div><div class="casAuthors">Nero, Tracy L.; Morton, Craig J.; Holien, Jessica K.; Wielens, Jerome; Parker, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-262</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Historically, targeting protein-protein interactions with small mols. was not thought possible because the corresponding interfaces were considered mostly flat and featureless and therefore 'undruggable'.  Instead, such interactions were targeted with larger mols., such as peptides and antibodies.  However, the past decade has seen encouraging breakthroughs through the refinement of existing techniques and the development of new ones, together with the identification and exploitation of unexpected aspects of protein-protein interaction surfaces.  In this Review, we describe some of the latest techniques to discover modulators of protein-protein interactions and how current drug discovery approaches have been adapted to successfully target these interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk_D7e8hVV7Vg90H21EOLACvtfcHk0li8ZfXCv232cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D&md5=7aad54df0c10c6cdb9b1a7ae0f15bfe9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrc3690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3690%26sid%3Dliteratum%253Aachs%26aulast%3DNero%26aufirst%3DT.%2BL.%26aulast%3DMorton%26aufirst%3DC.%2BJ.%26aulast%3DHolien%26aufirst%3DJ.%2BK.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26atitle%3DOncogenic%2520protein%2520interfaces%253A%2520small%2520molecules%252C%2520big%2520challenges%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D248%26epage%3D262%26doi%3D10.1038%2Fnrc3690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Wirth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoete, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michielin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, W. H.</span><span> </span><span class="NLM_article-title">Protein pocket and ligand shape comparison and its application in virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span><span class="refDoi"> DOI: 10.1007/s10822-013-9659-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-013-9659-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=511-524&author=M.+Wirthauthor=A.+Volkamerauthor=V.+Zoeteauthor=F.+Rippmannauthor=O.+Michielinauthor=M.+Rareyauthor=W.+H.+Sauer&title=Protein+pocket+and+ligand+shape+comparison+and+its+application+in+virtual+screening&doi=10.1007%2Fs10822-013-9659-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9659-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9659-1%26sid%3Dliteratum%253Aachs%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DZoete%26aufirst%3DV.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DSauer%26aufirst%3DW.%2BH.%26atitle%3DProtein%2520pocket%2520and%2520ligand%2520shape%2520comparison%2520and%2520its%2520application%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D511%26epage%3D524%26doi%3D10.1007%2Fs10822-013-9659-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grombacher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Combining global and local measures for structure-based druggability predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1021/ci200454v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200454v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=360-372&author=A.+Volkamerauthor=D.+Kuhnauthor=T.+Grombacherauthor=F.+Rippmannauthor=M.+Rarey&title=Combining+global+and+local+measures+for+structure-based+druggability+predictions&doi=10.1021%2Fci200454v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Global and Local Measures for Structure-Based Druggability Predictions</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Grombacher, Thomas; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting druggability and prioritizing certain disease modifying targets for the drug development process is of high practical relevance in pharmaceutical research.  DoGSiteScorer is a fully automatic algorithm for pocket and druggability prediction.  Besides consideration of global properties of the pocket, also local similarities shared between pockets are reflected.  Druggability scores are predicted by a support vector machine (SVM), trained, and tested on the druggability data set (DD) and its nonredundant version (NRDD).  The DD consists of 1069 targets with assigned druggable, difficult, and undruggable classes.  In 90% of the NRDD, the SVM model based on global descriptors correctly classifies a target as either druggable or undruggable.  Nevertheless, global properties suffer from binding site changes due to ligand binding and from the pocket boundary definition.  Therefore, local pocket properties are addnl. investigated in terms of a nearest neighbor search.  Local similarities are described by distance dependent histograms between atom pairs.  In 88% of the DD pocket set, the nearest neighbor and the structure itself conform with their druggability type.  A discriminant feature between druggable and undruggable pockets is having less short-range hydrophilic-hydrophilic pairs and more short-range lipophilic-lipophilic pairs.  Our findings for global pocket descriptors coincide with previously published methods affirming that size, shape, and hydrophobicity are important global pocket descriptors for automatic druggability prediction.  Nevertheless, the variety of pocket shapes and their flexibility upon ligand binding limit the automatic projection of druggable features onto descriptors.  Incorporating local pocket properties is another step toward a reliable descriptor-based druggability prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRnTVZG_4qNLVg90H21EOLACvtfcHk0liE5S7taP-1kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP&md5=ec276048e78e1297610e3d4b756ab6e8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci200454v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200454v%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DGrombacher%26aufirst%3DT.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DCombining%2520global%2520and%2520local%2520measures%2520for%2520structure-based%2520druggability%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D360%26epage%3D372%26doi%3D10.1021%2Fci200454v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">MetaPocket: a meta approach to improve protein ligand binding site prediction</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.1089/omi.2009.0045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1089%2Fomi.2009.0045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=325-330&author=B.+Huang&title=MetaPocket%3A+a+meta+approach+to+improve+protein+ligand+binding+site+prediction&doi=10.1089%2Fomi.2009.0045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1089%2Fomi.2009.0045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fomi.2009.0045%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DMetaPocket%253A%2520a%2520meta%2520approach%2520to%2520improve%2520protein%2520ligand%2520binding%2520site%2520prediction%26jtitle%3DOMICS%26date%3D2009%26volume%3D13%26spage%3D325%26epage%3D330%26doi%3D10.1089%2Fomi.2009.0045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Ruddigkeit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Deursen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2864</span><span class="NLM_x">–</span> <span class="NLM_lpage">2875</span><span class="refDoi"> DOI: 10.1021/ci300415d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300415d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2864-2875&author=L.+Ruddigkeitauthor=R.+van+Deursenauthor=L.+C.+Blumauthor=J.+L.+Reymond&title=Enumeration+of+166+billion+organic+small+molecules+in+the+chemical+universe+database+GDB-17&doi=10.1021%2Fci300415d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Enumeration of 166 Billion Organic Small Molecules in the Chemical Universe Database GDB-17</span></div><div class="casAuthors">Ruddigkeit, Lars; van Deursen, Ruud; Blum, Lorenz C.; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2864-2875</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug mols. consist of a few tens of atoms connected by covalent bonds.  How many such mols. are possible in total and what is their structure.  This question is of pressing interest in medicinal chem. to help solve the problems of drug potency, selectivity, and toxicity and reduce attrition rates by pointing to new mol. series.  To better define the unknown chem. space, we have enumerated 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens forming the chem. universe database GDB-17, covering a size range contg. many drugs and typical for lead compds.  GDB-17 contains millions of isomers of known drugs, including analogs with high shape similarity to the parent drug.  Compared to known mols. in PubChem, GDB-17 mols. are much richer in nonarom. heterocycles, quaternary centers, and stereoisomers, densely populate the third dimension in shape space, and represent many more scaffold types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1XVggP7C05rVg90H21EOLACvtfcHk0liE5S7taP-1kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J&md5=d0bf9a29f3e9ae1e57bb1c953a562ced</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci300415d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300415d%26sid%3Dliteratum%253Aachs%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DEnumeration%2520of%2520166%2520billion%2520organic%2520small%2520molecules%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2864%26epage%3D2875%26doi%3D10.1021%2Fci300415d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span> </span><span class="NLM_article-title">A functional feature analysis on diverse protein-protein interactions: application for the prediction of binding affinity</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1007/s10822-014-9746-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-014-9746-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=619-629&author=J.+Luoauthor=Y.+Guoauthor=Y.+Zhongauthor=D.+Maauthor=W.+Liauthor=M.+Li&title=A+functional+feature+analysis+on+diverse+protein-protein+interactions%3A+application+for+the+prediction+of+binding+affinity&doi=10.1007%2Fs10822-014-9746-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9746-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9746-y%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DM.%26atitle%3DA%2520functional%2520feature%2520analysis%2520on%2520diverse%2520protein-protein%2520interactions%253A%2520application%2520for%2520the%2520prediction%2520of%2520binding%2520affinity%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D619%26epage%3D629%26doi%3D10.1007%2Fs10822-014-9746-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">A hot spot of binding energy in a hormone-receptor interface</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1126/science.7529940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1126%2Fscience.7529940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=7529940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=383-386&author=T.+Clacksonauthor=J.+A.+Wells&title=A+hot+spot+of+binding+energy+in+a+hormone-receptor+interface&doi=10.1126%2Fscience.7529940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A hot spot of binding energy in a hormone-receptor interface</span></div><div class="casAuthors">Clackson, Tim; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5196</span>),
    <span class="NLM_cas:pages">383-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The x-ray crystal structure of the complex between human growth hormone (hGH) and the extracellular domain of its first bound receptor (hGHbp) shows that about 30 side chains from each protein make contact.  Individual replacement of contact residues in the hGHbp with alanine showed that a central hydrophobic region, dominated by two tryptophan residues, accounts for more than three-quarters of the binding free energy.  This "functional epitope" is surrounded by less important contact residues that are generally hydrophilic and partially hydrated, so that the interface resembles a cross section through a globular protein.  The functionally important residues on the hGHbp directly contact those on hGH.  Thus, only a small and complementary set of contact residues maintains binding affinity, a property that may be general to protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4AKI2EsvEQLVg90H21EOLACvtfcHk0ljg6XFcL01zPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D&md5=62b214455d644fd07afb2817d1c482fe</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.7529940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7529940%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520hot%2520spot%2520of%2520binding%2520energy%2520in%2520a%2520hormone-receptor%2520interface%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D383%26epage%3D386%26doi%3D10.1126%2Fscience.7529940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2eYeRuvxRlvbk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Schmidtke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Understanding and predicting druggability. A high-throughput method for detection of drug binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5858</span><span class="NLM_x">–</span> <span class="NLM_lpage">5867</span><span class="refDoi"> DOI: 10.1021/jm100574m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100574m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5858-5867&author=P.+Schmidtkeauthor=X.+Barril&title=Understanding+and+predicting+druggability.+A+high-throughput+method+for+detection+of+drug+binding+sites&doi=10.1021%2Fjm100574m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding and Predicting Druggability. A High-Throughput Method for Detection of Drug Binding Sites</span></div><div class="casAuthors">Schmidtke, Peter; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5858-5867</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability predictions are important to avoid intractable targets and to focus drug discovery efforts on sites offering better prospects.  However, few druggability prediction tools have been released and none has been extensively tested.  Here, a set of druggable and nondruggable cavities has been compiled in a collaborative platform (http://fpocket.sourceforge.net/dcd) that can be used, contributed, and curated by the community.  Druggable binding sites are often oversimplified as closed, hydrophobic cavities, but data set anal. reveals that polar groups in druggable binding sites have properties that enable them to play a decisive role in ligand recognition.  Finally, the data set has been used in conjunction with the open source fpocket suite to train and validate a logistic model.  State of the art performance was achieved for predicting druggability on known binding sites and on virtual screening expts. where druggable pockets are retrieved from a pool of decoys.  The algorithm is free, extremely fast, and can effectively be used to automatically sieve through massive collections of structures (http://fpocket.sourceforge.net).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos1ixAU42dJrVg90H21EOLACvtfcHk0ljg6XFcL01zPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D&md5=d9fc5c4d52a6e0f032b8d5c021a6cd89</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm100574m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100574m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DUnderstanding%2520and%2520predicting%2520druggability.%2520A%2520high-throughput%2520method%2520for%2520detection%2520of%2520drug%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5858%26epage%3D5867%26doi%3D10.1021%2Fjm100574m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Rosenquist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raboisson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vendeville, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Kock, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalmeijer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Rosa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beumont-Mauviel, M.</span><span> </span><span class="NLM_article-title">Discovery and development of simeprevir (TMC435), a HCV NS3/4A protease inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1673</span><span class="NLM_x">–</span> <span class="NLM_lpage">1693</span><span class="refDoi"> DOI: 10.1021/jm401507s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401507s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1673-1693&author=A.+Rosenquistauthor=B.+Samuelssonauthor=P.+O.+Johanssonauthor=M.+D.+Cummingsauthor=O.+Lenzauthor=P.+Raboissonauthor=K.+Simmenauthor=S.+Vendevilleauthor=H.+de+Kockauthor=M.+Nilssonauthor=A.+Horvathauthor=R.+Kalmeijerauthor=G.+de+la+Rosaauthor=M.+Beumont-Mauviel&title=Discovery+and+development+of+simeprevir+%28TMC435%29%2C+a+HCV+NS3%2F4A+protease+inhibitor&doi=10.1021%2Fjm401507s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Simeprevir (TMC435), a HCV NS3/4A Protease Inhibitor</span></div><div class="casAuthors">Rosenquist, Aasa; Samuelsson, Bertil; Johansson, Per-Ola; Cummings, Maxwell D.; Lenz, Oliver; Raboisson, Pierre; Simmen, Kenny; Vendeville, Sandrine; de Kock, Herman; Nilsson, Magnus; Horvath, Andras; Kalmeijer, Ronald; de la Rosa, Guy; Beumont-Mauviel, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1673-1693</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Hepatitis C virus is a blood-borne infection and the leading cause of chronic liver disease (including cirrhosis and cancer) and liver transplantation.  Since the identification of HCV in 1989, there has been an extensive effort to identify and improve treatment options.  An important milestone was reached in 2011 with the approval of the first-generation HCV NS3/4A protease inhibitors.  However, new therapies are needed to improve cure rates, shorten treatment duration, and improve tolerability.  Here we summarize the extensive medicinal chem. effort to develop novel P2 cyclopentane macrocyclic inhibitors guided by HCV NS3 protease assays, the cellular replicon system, structure-based design, and a panel of DMPK assays.  The selection of compd. 29 (simeprevir, TMC435) as clin. candidate was based on its excellent biol., PK, and safety pharmacol. profile.  Compd. 29 has recently been approved for treatment of chronic HCV infection in combination with pegylated interferon-α and ribavirin in Japan, Canada, and USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryCRg3NRRjzrVg90H21EOLACvtfcHk0ljg6XFcL01zPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslekuw%253D%253D&md5=fd9c1df095e47a111d9680af4c40ccc6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm401507s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401507s%26sid%3Dliteratum%253Aachs%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DJohansson%26aufirst%3DP.%2BO.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DLenz%26aufirst%3DO.%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DSimmen%26aufirst%3DK.%26aulast%3DVendeville%26aufirst%3DS.%26aulast%3Dde%2BKock%26aufirst%3DH.%26aulast%3DNilsson%26aufirst%3DM.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DKalmeijer%26aufirst%3DR.%26aulast%3Dde%2Bla%2BRosa%26aufirst%3DG.%26aulast%3DBeumont-Mauviel%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520development%2520of%2520simeprevir%2520%2528TMC435%2529%252C%2520a%2520HCV%2520NS3%252F4A%2520protease%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1673%26epage%3D1693%26doi%3D10.1021%2Fjm401507s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">How many drug targets are there?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">996</span><span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=How+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0lgvbhkQAPExHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DHow%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lgvbhkQAPExHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, C. A.</span><span> </span><span class="NLM_article-title">Ligand efficiency metrics considered harmful</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1007/s10822-014-9757-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-014-9757-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24899110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2ksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=699-710&author=P.+W.+Kennyauthor=A.+Leitaoauthor=C.+A.+Montanari&title=Ligand+efficiency+metrics+considered+harmful&doi=10.1007%2Fs10822-014-9757-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency metrics considered harmful</span></div><div class="casAuthors">Kenny, Peter W.; Leitao, Andrei; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">699-710</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Ligand efficiency metrics are used in drug discovery to normalize biol. activity or affinity with respect to physicochem. properties such as lipophilicity and mol. size.  This Perspective provides an overview of ligand efficiency metrics and summarizes thermodn. of protein-ligand binding.  Different classes of ligand efficiency metric are critically examd. and the study concludes with suggestions for alternative ways to account for physicochem. properties when prioritizing and optimizing leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWFJmrjl6YXrVg90H21EOLACvtfcHk0lgvbhkQAPExHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2ksLw%253D&md5=48885bcd2fcc646577ef78a5e7037494</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9757-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9757-8%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DLeitao%26aufirst%3DA.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DLigand%2520efficiency%2520metrics%2520considered%2520harmful%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D699%26epage%3D710%26doi%3D10.1007%2Fs10822-014-9757-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Reynolds, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bembenek, S. D.</span><span> </span><span class="NLM_article-title">Ligand binding efficiency: trends, physical basis, and implications</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2432</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span><span class="refDoi"> DOI: 10.1021/jm701255b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701255b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFSgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2432-2438&author=C.+H.+Reynoldsauthor=B.+A.+Toungeauthor=S.+D.+Bembenek&title=Ligand+binding+efficiency%3A+trends%2C+physical+basis%2C+and+implications&doi=10.1021%2Fjm701255b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand binding efficiency: trends, physical basis, and implications</span></div><div class="casAuthors">Reynolds, Charles H.; Tounge, Brett A.; Bembenek, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2432-2438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand efficiency (i.e., potency/size) has emerged as an important metric in drug discovery.  In general, smaller, more efficient ligands are believed to have improved prospects for good drug properties (e.g., bioavailability).  Our anal. of thousands of ligands across a variety of targets shows that ligand efficiency is dependent on ligand size with smaller ligands having greater efficiencies, on av., than larger ligands.  We propose two primary causes for this size dependence: the inevitable redn. in the quality of fit between ligand and receptor as the ligand becomes larger and more complex and the redn. in accessible ligand surface area on a per atom basis as size increases.  These results have far-ranging implications for anal. of high-throughput screening hits, fragment-based approaches to drug discovery, and even computational models of potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkrPWFEZZEMrVg90H21EOLACvtfcHk0lixqAfvtGtO_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFSgsbw%253D&md5=a9a41b44dbca7f34562c40b56c7ae683</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm701255b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701255b%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26atitle%3DLigand%2520binding%2520efficiency%253A%2520trends%252C%2520physical%2520basis%252C%2520and%2520implications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2432%26epage%3D2438%26doi%3D10.1021%2Fjm701255b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1438</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lixqAfvtGtO_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Chen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foloppe, N.</span><span> </span><span class="NLM_article-title">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">–</span> <span class="NLM_lpage">7920</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.+J.+Chenauthor=N.+Foloppe&title=Tackling+the+conformational+sampling+of+larger+flexible+compounds+and+macrocycles+in+pharmacology+and+drug+discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0lixqAfvtGtO_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520conformational%2520sampling%2520of%2520larger%2520flexible%2520compounds%2520and%2520macrocycles%2520in%2520pharmacology%2520and%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lixqAfvtGtO_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Varma, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troutman, M. D.</span><span> </span><span class="NLM_article-title">Physicochemical space for optimum oral bioavailability: contribution of human intestinal absorption and first-pass elimination</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1098</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1021/jm901371v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901371v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFGisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1098-1108&author=M.+V.+Varmaauthor=R.+S.+Obachauthor=C.+Rotterauthor=H.+R.+Millerauthor=G.+Changauthor=S.+J.+Steynauthor=A.+El-Kattanauthor=M.+D.+Troutman&title=Physicochemical+space+for+optimum+oral+bioavailability%3A+contribution+of+human+intestinal+absorption+and+first-pass+elimination&doi=10.1021%2Fjm901371v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical Space for Optimum Oral Bioavailability: Contribution of Human Intestinal Absorption and First-Pass Elimination</span></div><div class="casAuthors">Varma, Manthena V. S.; Obach, R. Scott; Rotter, Charles; Miller, Howard R.; Chang, George; Steyn, Stefanus J.; El-Kattan, Ayman; Troutman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1098-1108</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability (F) is a product of fraction absorbed (Fa), fraction escaping gut-wall elimination (Fg), and fraction escaping hepatic elimination (Fh).  In this study, using a database comprised of Fa, Fg, Fh, and F values for 309 drugs in humans, an anal. of the interrelation of physicochem. properties and the individual parameters was carried out in order to define the physicochem. space for optimum human oral bioavailability.  Trend anal. clearly indicated mol. wt. (MW), ionization state, lipophilicity, polar descriptors, and free rotatable bonds (RB) influence bioavailability.  These trends were due to a combination of effects of the properties on Fa and first-pass elimination (Fg and Fh).  Higher MW significantly impacted Fa, while Fg and Fh decreased with increasing lipophilicity.  Parabolic trends were obsd. for bioavailability with polar descriptors.  Interestingly, RB has a neg. effect on all three parameters, leading to its pronounced effect on bioavailability.  In conclusion, physicochem. properties influence bioavailability with typically opposing effects on Fa and first-pass elimination.  This anal. may provide a rational judgment on the physicochem. space to optimize oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCkIt1i03orVg90H21EOLACvtfcHk0ljWm1Cvzlt6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFGisg%253D%253D&md5=9a9e5f4ff74f3d8d9a35c3b2d7e37c78</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm901371v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901371v%26sid%3Dliteratum%253Aachs%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DRotter%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DH.%2BR.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26atitle%3DPhysicochemical%2520space%2520for%2520optimum%2520oral%2520bioavailability%253A%2520contribution%2520of%2520human%2520intestinal%2520absorption%2520and%2520first-pass%2520elimination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1098%26epage%3D1108%26doi%3D10.1021%2Fjm901371v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">He, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, Z. K.</span><span> </span><span class="NLM_article-title">Promiscuity and the conformational rearrangement of drug-like molecules: insight from the Protein Data Bank</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">238</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span><span class="refDoi"> DOI: 10.1002/cmdc.201402389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1002%2Fcmdc.201402389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25491400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCks7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=238-244&author=M.+W.+Heauthor=P.+S.+Leeauthor=Z.+K.+Sweeney&title=Promiscuity+and+the+conformational+rearrangement+of+drug-like+molecules%3A+insight+from+the+Protein+Data+Bank&doi=10.1002%2Fcmdc.201402389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuity and the Conformational Rearrangement of Drug-Like Molecules: Insight from the Protein Data Bank</span></div><div class="casAuthors">He, Michael W.; Lee, Patrick S.; Sweeney, Zachary K.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">238-244</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Selectivity is a central aspect of lead optimization in the drug discovery process.  Medicinal chemists often try to decrease mol. flexibility to improve selectivity, given the common belief that the two are interdependent.  To investigate the relationship between polypharmacol. and conformational flexibility, we mined the Protein Data Bank and constructed a dataset of pharmaceutically relevant ligands that crystd. in more than one protein target while binding to each co-crystd. receptor with similar in vitro affinities.  After analyzing the mol. conformations of these 100 ligands, we found that 59 ligands bound to different protein targets without significantly changing conformation, suggesting that there is no distinct correlation between conformational flexibility and polypharmacol. within our dataset.  Ligands crystd. in similar proteins and highly ligand-efficient compds. with five or fewer rotatable bonds were less likely to adjust conformation when binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd8jc_c6ldDbVg90H21EOLACvtfcHk0ljWm1Cvzlt6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCks7jM&md5=e3e9f6af0a7a15a53e836d9155701d84</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402389%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%2BW.%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DSweeney%26aufirst%3DZ.%2BK.%26atitle%3DPromiscuity%2520and%2520the%2520conformational%2520rearrangement%2520of%2520drug-like%2520molecules%253A%2520insight%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D238%26epage%3D244%26doi%3D10.1002%2Fcmdc.201402389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Drug promiscuity in PDB: protein binding site similarity is key</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e65894</span><span class="refDoi"> DOI: 10.1371/journal.pone.0065894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1371%2Fjournal.pone.0065894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23805191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e65894&author=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Drug+promiscuity+in+PDB%3A+protein+binding+site+similarity+is+key&doi=10.1371%2Fjournal.pone.0065894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Drug promiscuity in PDB: protein binding site similarity is key</span></div><div class="casAuthors">Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65894</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug repositioning applies established drugs to new disease indications with increasing success.  A pre-requisite for drug repurposing is drug promiscuity (polypharmacol.) - a drug's ability to bind to several targets.  There is a long standing debate on the reasons for drug promiscuity.  Based on large compd. screens, hydrophobicity and mol. wt. have been suggested as key reasons.  However, the results are sometimes contradictory and leave space for further anal.  Protein structures offer a structural dimension to explain promiscuity: Can a drug bind multiple targets because the drug is flexible or because the targets are structurally similar or even share similar binding sites.  We present a systematic study of drug promiscuity based on structural data of PDB target proteins with a set of 164 promiscuous drugs.  We show that there is no correlation between the degree of promiscuity and ligand properties such as hydrophobicity or mol. wt. but a weak correlation to conformational flexibility.  However, we do find a correlation between promiscuity and structural similarity as well as binding site similarity of protein targets.  In particular, 71% of the drugs have at least two targets with similar binding sites.  In order to overcome issues in detection of remotely similar binding sites, we employed a score for binding site similarity: Ligand RMSD measures the similarity of the aligned ligands and uncovers remote local similarities in proteins.  It can be applied to arbitrary structural binding site alignments.  Three representative examples, namely the anti-cancer drug methotrexate, the natural product quercetin and the anti-diabetic drug acarbose are discussed in detail.  Our findings suggest that global structural and binding site similarity play a more important role to explain the obsd. drug promiscuity in the PDB than physicochem. drug properties like hydrophobicity or mol. wt.  Addnl., we find ligand flexibility to have a minor influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXd_1yYo5lLVg90H21EOLACvtfcHk0ljWm1Cvzlt6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP&md5=99574519028e547ebec5a4c2ed27f6a5</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065894%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DDrug%2520promiscuity%2520in%2520PDB%253A%2520protein%2520binding%2520site%2520similarity%2520is%2520key%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De65894%26doi%3D10.1371%2Fjournal.pone.0065894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">DeLorbe, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clements, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiddon, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. F.</span><span> </span><span class="NLM_article-title">Thermodynamic and structural effects of macrocyclic constraints in protein–ligand interactions</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span><span class="refDoi"> DOI: 10.1021/ml100142y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100142y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=448-452&author=J.+E.+DeLorbeauthor=J.+H.+Clementsauthor=B.+B.+Whiddonauthor=S.+F.+Martin&title=Thermodynamic+and+structural+effects+of+macrocyclic+constraints+in+protein%E2%80%93ligand+interactions&doi=10.1021%2Fml100142y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fml100142y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100142y%26sid%3Dliteratum%253Aachs%26aulast%3DDeLorbe%26aufirst%3DJ.%2BE.%26aulast%3DClements%26aufirst%3DJ.%2BH.%26aulast%3DWhiddon%26aufirst%3DB.%2BB.%26aulast%3DMartin%26aufirst%3DS.%2BF.%26atitle%3DThermodynamic%2520and%2520structural%2520effects%2520of%2520macrocyclic%2520constraints%2520in%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D448%26epage%3D452%26doi%3D10.1021%2Fml100142y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Watts, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tebben, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheney, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J. C.</span><span> </span><span class="NLM_article-title">Macrocycle conformational sampling with MacroModel</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2680</span><span class="NLM_x">–</span> <span class="NLM_lpage">2696</span><span class="refDoi"> DOI: 10.1021/ci5001696</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001696" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2680-2696&author=K.+S.+Wattsauthor=P.+Dalalauthor=A.+J.+Tebbenauthor=D.+L.+Cheneyauthor=J.+C.+Shelley&title=Macrocycle+conformational+sampling+with+MacroModel&doi=10.1021%2Fci5001696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycle Conformational Sampling with MacroModel</span></div><div class="casAuthors">Watts, K. Shawn; Dalal, Pranav; Tebben, Andrew J.; Cheney, Daniel L.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2680-2696</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling low energy conformations of macrocycles is challenging due to the large size of many of these mols. and the constraints imposed by the macrocycle.  We present a new conformational search method (implemented in MacroModel) that uses brief MD simulations followed by minimization and normal-mode search steps.  The method was parametrized using a set of 100 macrocycles from the PDB and CSD.  It was then tested on a publicly available data set for which there are published results using alternative methods; we found that when the same force field is used (in this case MMFFs in vacuum), our method tended to identify conformations with lower energies than what the other methods identified.  The performance on a new set of 50 macrocycles from the PDB and CSD was also quite good; the mean and median RMSD values for just the ring atoms were 0.60 and 0.33 Å, resp.  However, the RMSD values for macrocycles with more than 30 ring-atoms were quite a bit larger compared to the smaller macrocycles.  Possible origins for this and ideas for improving the performance on very large macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBKXPDzoVcLVg90H21EOLACvtfcHk0lgsuuueZAvorg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN&md5=612c949e25807bbf6f850c2a2c37b821</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci5001696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001696%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DK.%2BS.%26aulast%3DDalal%26aufirst%3DP.%26aulast%3DTebben%26aufirst%3DA.%2BJ.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DMacrocycle%2520conformational%2520sampling%2520with%2520MacroModel%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2680%26epage%3D2696%26doi%3D10.1021%2Fci5001696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Sondergaard, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carstensen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J. E.</span><span> </span><span class="NLM_article-title">Structural artifacts in protein-ligand X-ray structures: implications for the development of docking scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5673</span><span class="NLM_x">–</span> <span class="NLM_lpage">5684</span><span class="refDoi"> DOI: 10.1021/jm8016464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8016464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5673-5684&author=C.+R.+Sondergaardauthor=A.+E.+Garrettauthor=T.+Carstensenauthor=G.+Pollastriauthor=J.+E.+Nielsen&title=Structural+artifacts+in+protein-ligand+X-ray+structures%3A+implications+for+the+development+of+docking+scoring+functions&doi=10.1021%2Fjm8016464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Artifacts in Protein-Ligand X-ray Structures: Implications for the Development of Docking Scoring Functions</span></div><div class="casAuthors">Sondergaard, Chresten R.; Garrett, Alison Elizabeth; Carstensen, Tommy; Pollastri, Gianluca; Nielsen, Jens Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5673-5684</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of docking scoring functions requires high-resoln. 3D structures of protein-ligand complexes for which the binding affinity of the ligand has been measured exptl.  Protein-ligand binding affinities are measured in soln. expts., and high resoln. protein-ligand structures can be detd. only by X-ray crystallog.  Protein-ligand scoring functions must therefore reproduce soln. binding energies using analyses of proteins in a crystal environment.  We present an anal. of the prevalence of crystal-induced artifacts and water-mediated contacts in protein-ligand complexes and demonstrate the effect that these can have on the performance of protein-ligand scoring functions.  We find 36% of ligands in the PDBBind 2007 refined data set to be influenced by crystal contacts and find the performance of a scoring function to be affected by these.  A Web server for detecting crystal contacts in protein-ligand complexes is available at http://enzyme.ucd.ie/LIGCRYST.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAMRmv0ZVao7Vg90H21EOLACvtfcHk0lgsuuueZAvorg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu73P&md5=c5db24f33a237968934499f0f2192597</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm8016464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8016464%26sid%3Dliteratum%253Aachs%26aulast%3DSondergaard%26aufirst%3DC.%2BR.%26aulast%3DGarrett%26aufirst%3DA.%2BE.%26aulast%3DCarstensen%26aufirst%3DT.%26aulast%3DPollastri%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26atitle%3DStructural%2520artifacts%2520in%2520protein-ligand%2520X-ray%2520structures%253A%2520implications%2520for%2520the%2520development%2520of%2520docking%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5673%26epage%3D5684%26doi%3D10.1021%2Fjm8016464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Liebeschuetz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennemann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1007/s10822-011-9538-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-011-9538-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22246295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=169-183&author=J.+Liebeschuetzauthor=J.+Hennemannauthor=T.+Olssonauthor=C.+R.+Groom&title=The+good%2C+the+bad+and+the+twisted%3A+a+survey+of+ligand+geometry+in+protein+crystal+structures&doi=10.1007%2Fs10822-011-9538-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span></div><div class="casAuthors">Liebeschuetz, John; Hennemann, Jana; Olsson, Tjelvar; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The protein databank now contains the structures of over 11,000 ligands bound to proteins.  These structures are invaluable in applied areas such as structure-based drug design, but are also the substrate for understanding the energetics of intermol. interactions with proteins.  Despite their obvious importance, the careful anal. of ligands bound to protein structures lags behind the anal. of the protein structures themselves.  We present an anal. of the geometry of ligands bound to proteins and highlight the role of small mol. crystal structures in enabling mol. modellers to critically evaluate a ligand model's quality and investigate protein-induced strain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpceotnqHSI-rVg90H21EOLACvtfcHk0lgsuuueZAvorg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D&md5=dd85bf6e63910bc85db95509b97c018a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9538-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9538-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%26aulast%3DHennemann%26aufirst%3DJ.%26aulast%3DOlsson%26aufirst%3DT.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520good%252C%2520the%2520bad%2520and%2520the%2520twisted%253A%2520a%2520survey%2520of%2520ligand%2520geometry%2520in%2520protein%2520crystal%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D169%26epage%3D183%26doi%3D10.1007%2Fs10822-011-9538-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lidLIHQCd4tIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lidLIHQCd4tIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 166 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wen Shuai, Guan Wang, Yiwen Zhang, Faqian Bu, Sicheng Zhang, Duane D. Miller, Wei Li, Liang Ouyang, <span class="NLM_string-name hlFld-ContribAuthor">Yuxi Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Progress on Tubulin Inhibitors with Dual Targeting Capabilities for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7963-7990. <a href="https://doi.org/10.1021/acs.jmedchem.1c00100" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00100%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BProgress%252Bon%252BTubulin%252BInhibitors%252Bwith%252BDual%252BTargeting%252BCapabilities%252Bfor%252BCancer%252BTherapy%26aulast%3DShuai%26aufirst%3DWen%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D08062021%26volume%3D64%26issue%3D12%26spage%3D7963%26epage%3D7990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongjae Lee, Sungjin Lee, Jieun Choi, Yoo-Kyung Song, Min Ju Kim, Dae-Seop Shin, Myung Ae Bae, Yong-Chul Kim, Chin-Ju Park, Kyeong-Ryoon Lee, Jun-Ho Choi, <span class="NLM_string-name hlFld-ContribAuthor">Jiwon Seo</span>. </span><span class="cited-content_cbyCitation_article-title">Interplay among Conformation, Intramolecular Hydrogen Bonds, and Chameleonicity in the Membrane Permeability and Cyclophilin A Binding of Macrocyclic Peptide Cyclosporin O Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8272-8286. <a href="https://doi.org/10.1021/acs.jmedchem.1c00211" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00211%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInterplay%252Bamong%252BConformation%25252C%252BIntramolecular%252BHydrogen%252BBonds%25252C%252Band%252BChameleonicity%252Bin%252Bthe%252BMembrane%252BPermeability%252Band%252BCyclophilin%252BA%252BBinding%252Bof%252BMacrocyclic%252BPeptide%252BCyclosporin%252BO%252BDerivatives%26aulast%3DLee%26aufirst%3DDongjae%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D04022021%26date%3D07062021%26volume%3D64%26issue%3D12%26spage%3D8272%26epage%3D8286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul D. Leeson, A. Patricia Bento, Anna Gaulton, Anne Hersey, Emma J. Manners, Chris J. Radoux, <span class="NLM_string-name hlFld-ContribAuthor">Andrew R. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Target-Based Evaluation of “Drug-Like” Properties and Ligand Efficiencies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7210-7230. <a href="https://doi.org/10.1021/acs.jmedchem.1c00416" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00416%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget-Based%252BEvaluation%252Bof%252B%2525E2%252580%25259CDrug-Like%2525E2%252580%25259D%252BProperties%252Band%252BLigand%252BEfficiencies%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D13052021%26volume%3D64%26issue%3D11%26spage%3D7210%26epage%3D7230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Yoseph Atilaw, Mate Erdelyi, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Cell Permeability of Isomeric Macrocycles: Predictions and NMR Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 983-990. <a href="https://doi.org/10.1021/acsmedchemlett.1c00126" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00126%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCell%252BPermeability%252Bof%252BIsomeric%252BMacrocycles%25253A%252BPredictions%252Band%252BNMR%252BStudies%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D02032021%26date%3D13052021%26date%3D18052021%26volume%3D12%26issue%3D6%26spage%3D983%26epage%3D990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yunhui Ge, Si Zhang, Mate Erdelyi, <span class="NLM_string-name hlFld-ContribAuthor">Vincent A. Voelz</span>. </span><span class="cited-content_cbyCitation_article-title">Solution-State Preorganization of Cyclic β-Hairpin Ligands Determines Binding Mechanism and Affinities for MDM2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2353-2367. <a href="https://doi.org/10.1021/acs.jcim.1c00029" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00029%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSolution-State%252BPreorganization%252Bof%252BCyclic%252B%2525CE%2525B2-Hairpin%252BLigands%252BDetermines%252BBinding%252BMechanism%252Band%252BAffinities%252Bfor%252BMDM2%26aulast%3DGe%26aufirst%3DYunhui%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D11012021%26date%3D27042021%26volume%3D61%26issue%3D5%26spage%3D2353%26epage%3D2367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael Bauder, Christian Meyners, Patrick L. Purder, Stephanie Merz, Wisely Oki Sugiarto, Andreas M. Voll, Tim Heymann, <span class="NLM_string-name hlFld-ContribAuthor">Felix Hausch</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3320-3349. <a href="https://doi.org/10.1021/acs.jmedchem.0c02195" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02195%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252BHigh-Affinity%252BMacrocyclic%252BFKBP51%252BInhibitors%26aulast%3DBauder%26aufirst%3DMichael%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D19122020%26date%3D05032021%26volume%3D64%26issue%3D6%26spage%3D3320%26epage%3D3349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alexander C. Brueckner, Qiaolin Deng, Ann E. Cleves, Charles A. Lesburg, Juan C. Alvarez, Mikhail Y. Reibarkh, Edward C. Sherer, <span class="NLM_string-name hlFld-ContribAuthor">Ajay N. Jain</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3282-3298. <a href="https://doi.org/10.1021/acs.jmedchem.0c02159" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02159%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DConformational%252BStrain%252Bof%252BMacrocyclic%252BPeptides%252Bin%252BLigand%2525E2%252580%252593Receptor%252BComplexes%252BBased%252Bon%252BAdvanced%252BRefinement%252Bof%252BBound-State%252BConformers%26aulast%3DBrueckner%26aufirst%3DAlexander%2BC.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D16032021%26volume%3D64%26issue%3D6%26spage%3D3282%26epage%3D3298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ashley N. Matthew, Florian Leidner, Gordon J. Lockbaum, Mina Henes, Jacqueto Zephyr, Shurong Hou, Desaboini Nageswara Rao, Jennifer Timm, Linah N. Rusere, Debra A. Ragland, Janet L. Paulsen, Kristina Prachanronarong, Djade I. Soumana, Ellen A. Nalivaika, Nese Kurt Yilmaz, Akbar Ali, <span class="NLM_string-name hlFld-ContribAuthor">Celia A. Schiffer</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design Strategies to Avoid Resistance in Direct-Acting Antivirals and Beyond. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2021,</strong> <em>121 </em>
                                    (6)
                                     , 3238-3270. <a href="https://doi.org/10.1021/acs.chemrev.0c00648" title="DOI URL">https://doi.org/10.1021/acs.chemrev.0c00648</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.0c00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.0c00648%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DDrug%252BDesign%252BStrategies%252Bto%252BAvoid%252BResistance%252Bin%252BDirect-Acting%252BAntivirals%252Band%252BBeyond%26aulast%3DMatthew%26aufirst%3DAshley%2BN.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22062020%26date%3D07012021%26volume%3D121%26issue%3D6%26spage%3D3238%26epage%3D3270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Richard S. Hong, Alessandra Mattei, Ahmad Y. Sheikh, Rajni Miglani Bhardwaj, Michael A. Bellucci, Keith F. McDaniel, M. Olivia Pierce, Guangxu Sun, Sizhu Li, Lingle Wang, Sayan Mondal, Jianguo Ji, <span class="NLM_string-name hlFld-ContribAuthor">Thomas B. Borchardt</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Physics-Based Ensemble Modeling Approach That Utilizes 3D Molecular Conformation and Packing to Access Aqueous Thermodynamic Solubility: A Case Study of Orally Available Bromodomain and Extraterminal Domain Inhibitor Lead Optimization Series. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (3)
                                     , 1412-1426. <a href="https://doi.org/10.1021/acs.jcim.0c01410" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01410%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNovel%252BPhysics-Based%252BEnsemble%252BModeling%252BApproach%252BThat%252BUtilizes%252B3D%252BMolecular%252BConformation%252Band%252BPacking%252Bto%252BAccess%252BAqueous%252BThermodynamic%252BSolubility%25253A%252BA%252BCase%252BStudy%252Bof%252BOrally%252BAvailable%252BBromodomain%252Band%252BExtraterminal%252BDomain%252BInhibitor%252BLead%252BOptimization%252BSeries%26aulast%3DHong%26aufirst%3DRichard%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D07122020%26date%3D04032021%26volume%3D61%26issue%3D3%26spage%3D1412%26epage%3D1426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paula C. Ortet, Samantha N. Muellers, Lauren A. Viarengo-Baker, Kristina Streu, Blair R. Szymczyna, Aaron B. Beeler, Karen N. Allen, <span class="NLM_string-name hlFld-ContribAuthor">Adrian Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Recapitulating the Binding Affinity of Nrf2 for KEAP1 in a Cyclic Heptapeptide, Guided by NMR, X-ray Crystallography, and Machine Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (10)
                                     , 3779-3793. <a href="https://doi.org/10.1021/jacs.0c09799" title="DOI URL">https://doi.org/10.1021/jacs.0c09799</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c09799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c09799%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DRecapitulating%252Bthe%252BBinding%252BAffinity%252Bof%252BNrf2%252Bfor%252BKEAP1%252Bin%252Ba%252BCyclic%252BHeptapeptide%25252C%252BGuided%252Bby%252BNMR%25252C%252BX-ray%252BCrystallography%25252C%252Band%252BMachine%252BLearning%26aulast%3DOrtet%26aufirst%3DPaula%2BC.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13092020%26date%3D08032021%26volume%3D143%26issue%3D10%26spage%3D3779%26epage%3D3793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dean G. Brown, <span class="NLM_string-name hlFld-ContribAuthor">Heike J. Wobst</span>. </span><span class="cited-content_cbyCitation_article-title">A Decade of FDA-Approved Drugs (2010–2019): Trends and Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2312-2338. <a href="https://doi.org/10.1021/acs.jmedchem.0c01516" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01516</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01516%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BDecade%252Bof%252BFDA-Approved%252BDrugs%252B%2525282010%2525E2%252580%2525932019%252529%25253A%252BTrends%252Band%252BFuture%252BDirections%26aulast%3DBrown%26aufirst%3DDean%2BG.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D30082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2312%26epage%3D2338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amber L. H. Gray, Carlos A. Steren, Isaac W. Haynes, Guillermo A. Bermejo, Filippo Favretto, Markus Zweckstetter, <span class="NLM_string-name hlFld-ContribAuthor">Thanh D. Do</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Flexibility of Cyclosporine A Is Mediated by Amide Cis–Trans Isomerization and the Chameleonic Roles of Calcium. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (5)
                                     , 1378-1391. <a href="https://doi.org/10.1021/acs.jpcb.0c11152" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c11152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c11152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c11152%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DStructural%252BFlexibility%252Bof%252BCyclosporine%252BA%252BIs%252BMediated%252Bby%252BAmide%252BCis%2525E2%252580%252593Trans%252BIsomerization%252Band%252Bthe%252BChameleonic%252BRoles%252Bof%252BCalcium%26aulast%3DGray%26aufirst%3DAmber%2BL.%2BH.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D01022021%26volume%3D125%26issue%3D5%26spage%3D1378%26epage%3D1391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Björn Over, Marie Castaldo, Stefan Geschwindner, Patrik Johansson, Mohit Tyagi, Christian Tyrchan, Lisa Wissler, Peter Sjö, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Mining Natural Products for Macrocycles to Drug Difficult Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 1054-1072. <a href="https://doi.org/10.1021/acs.jmedchem.0c01569" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01569%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMining%252BNatural%252BProducts%252Bfor%252BMacrocycles%252Bto%252BDrug%252BDifficult%252BTargets%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D08092020%26date%3D18122020%26volume%3D64%26issue%3D2%26spage%3D1054%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Colin N. Kelly, Chad E. Townsend, Ajay N. Jain, Matthew R. Naylor, Cameron R. Pye, Joshua Schwochert, <span class="NLM_string-name hlFld-ContribAuthor">R. Scott Lokey</span>. </span><span class="cited-content_cbyCitation_article-title">Geometrically Diverse Lariat Peptide Scaffolds Reveal an Untapped Chemical Space of High Membrane Permeability. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (2)
                                     , 705-714. <a href="https://doi.org/10.1021/jacs.0c06115" title="DOI URL">https://doi.org/10.1021/jacs.0c06115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c06115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c06115%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DGeometrically%252BDiverse%252BLariat%252BPeptide%252BScaffolds%252BReveal%252Ban%252BUntapped%252BChemical%252BSpace%252Bof%252BHigh%252BMembrane%252BPermeability%26aulast%3DKelly%26aufirst%3DColin%2BN.%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D31122020%26volume%3D143%26issue%3D2%26spage%3D705%26epage%3D714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Giulia Caron, Jan Kihlberg, Gilles Goetz, Ekaterina Ratkova, Vasanthanathan Poongavanam, <span class="NLM_string-name hlFld-ContribAuthor">Giuseppe Ermondi</span>. </span><span class="cited-content_cbyCitation_article-title">Steering New Drug Discovery Campaigns: Permeability, Solubility, and Physicochemical Properties in the bRo5 Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 13-23. <a href="https://doi.org/10.1021/acsmedchemlett.0c00581" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00581%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSteering%252BNew%252BDrug%252BDiscovery%252BCampaigns%25253A%252BPermeability%25252C%252BSolubility%25252C%252Band%252BPhysicochemical%252BProperties%252Bin%252Bthe%252BbRo5%252BChemical%252BSpace%26aulast%3DCaron%26aufirst%3DGiulia%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D04112020%26date%3D15122020%26date%3D05012021%26volume%3D12%26issue%3D1%26spage%3D13%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">John T. Randolph, Eric A. Voight, Stephen N. Greszler, Brice E. Uno, James N. Newton, Kenneth M. Gleason, DeAnne Stolarik, Cecilia Van Handel, Daniel A. J. Bow, <span class="NLM_string-name hlFld-ContribAuthor">David A. DeGoey</span>. </span><span class="cited-content_cbyCitation_article-title">Prodrug Strategies to Improve the Solubility of the HCV NS5A Inhibitor Pibrentasvir (ABT-530). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 11034-11044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00956" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00956</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00956%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProdrug%252BStrategies%252Bto%252BImprove%252Bthe%252BSolubility%252Bof%252Bthe%252BHCV%252BNS5A%252BInhibitor%252BPibrentasvir%252B%252528ABT-530%252529%26aulast%3DRandolph%26aufirst%3DJohn%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D04062020%26date%3D17092020%26date%3D03092020%26volume%3D63%26issue%3D19%26spage%3D11034%26epage%3D11044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher P. Tinworth, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Young</span>. </span><span class="cited-content_cbyCitation_article-title">Facts, Patterns, and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10091-10108. <a href="https://doi.org/10.1021/acs.jmedchem.9b01596" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01596</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01596%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFacts%25252C%252BPatterns%25252C%252Band%252BPrinciples%252Bin%252BDrug%252BDiscovery%25253A%252BAppraising%252Bthe%252BRule%252Bof%252B5%252Bwith%252BMeasured%252BPhysicochemical%252BData%26aulast%3DTinworth%26aufirst%3DChristopher%2BP.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D25092019%26date%3D11052020%26date%3D23042020%26volume%3D63%26issue%3D18%26spage%3D10091%26epage%3D10108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">K. H. Brian Lam, J. C. Yves Le Blanc, <span class="NLM_string-name hlFld-ContribAuthor">J. Larry Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">Separating Isomers, Conformers, and Analogues of Cyclosporin using Differential Mobility Spectroscopy, Mass Spectrometry, and Hydrogen–Deuterium Exchange. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2020,</strong> <em>92 </em>
                                    (16)
                                     , 11053-11061. <a href="https://doi.org/10.1021/acs.analchem.0c00191" title="DOI URL">https://doi.org/10.1021/acs.analchem.0c00191</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.0c00191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.0c00191%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DSeparating%252BIsomers%25252C%252BConformers%25252C%252Band%252BAnalogues%252Bof%252BCyclosporin%252Busing%252BDifferential%252BMobility%252BSpectroscopy%25252C%252BMass%252BSpectrometry%25252C%252Band%252BHydrogen%2525E2%252580%252593Deuterium%252BExchange%26aulast%3DLam%26aufirst%3DK.%2BH.%2BBrian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15012020%26date%3D22072020%26date%3D06082020%26date%3D22072020%26volume%3D92%26issue%3D16%26spage%3D11053%26epage%3D11061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anna S. Kamenik, Johannes Kraml, Florian Hofer, Franz Waibl, Patrick K. Quoika, Ursula Kahler, Michael Schauperl, <span class="NLM_string-name hlFld-ContribAuthor">Klaus R. Liedl</span>. </span><span class="cited-content_cbyCitation_article-title">Macrocycle Cell Permeability Measured by Solvation Free Energies in Polar and Apolar Environments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (7)
                                     , 3508-3517. <a href="https://doi.org/10.1021/acs.jcim.0c00280" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00280%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMacrocycle%252BCell%252BPermeability%252BMeasured%252Bby%252BSolvation%252BFree%252BEnergies%252Bin%252BPolar%252Band%252BApolar%252BEnvironments%26aulast%3DKamenik%26aufirst%3DAnna%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19032020%26date%3D29062020%26date%3D18062020%26volume%3D60%26issue%3D7%26spage%3D3508%26epage%3D3517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zi-Yi Yang, Jun-Hong He, Ai-Ping Lu, Ting-Jun Hou, <span class="NLM_string-name hlFld-ContribAuthor">Dong-Sheng Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Negative Design To Design a More Desirable Virtual Screening Library. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4411-4429. <a href="https://doi.org/10.1021/acs.jmedchem.9b01476" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01476</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01476%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DApplication%252Bof%252BNegative%252BDesign%252BTo%252BDesign%252Ba%252BMore%252BDesirable%252BVirtual%252BScreening%252BLibrary%26aulast%3DYang%26aufirst%3DZi-Yi%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D05092019%26date%3D27012020%26date%3D13012020%26volume%3D63%26issue%3D9%26spage%3D4411%26epage%3D4429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuzhe Wang, Jagna Witek, Gregory A. Landrum, <span class="NLM_string-name hlFld-ContribAuthor">Sereina Riniker</span>. </span><span class="cited-content_cbyCitation_article-title">Improving Conformer Generation for Small Rings and Macrocycles Based on Distance Geometry and Experimental Torsional-Angle Preferences. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2044-2058. <a href="https://doi.org/10.1021/acs.jcim.0c00025" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BConformer%252BGeneration%252Bfor%252BSmall%252BRings%252Band%252BMacrocycles%252BBased%252Bon%252BDistance%252BGeometry%252Band%252BExperimental%252BTorsional-Angle%252BPreferences%26aulast%3DWang%26aufirst%3DShuzhe%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09012020%26date%3D20032020%26date%3D10032020%26volume%3D60%26issue%3D4%26spage%3D2044%26epage%3D2058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jie Yang, Vladimir O. Talibov, Stefan Peintner, Claire Rhee, Vasanthanathan Poongavanam, Matthis Geitmann, Matteo Rossi Sebastiano, Bernd Simon, Janosch Hennig, Doreen Dobritzsch, U. Helena Danielson, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Macrocyclic Peptides Uncover a Novel Binding Mode for Reversible Inhibitors of LSD1. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (8)
                                     , 3979-3995. <a href="https://doi.org/10.1021/acsomega.9b03493" title="DOI URL">https://doi.org/10.1021/acsomega.9b03493</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b03493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b03493%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMacrocyclic%252BPeptides%252BUncover%252Ba%252BNovel%252BBinding%252BMode%252Bfor%252BReversible%252BInhibitors%252Bof%252BLSD1%26aulast%3DYang%26aufirst%3DJie%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D19102019%26date%3D30012020%26date%3D17022020%26volume%3D5%26issue%3D8%26spage%3D3979%26epage%3D3995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Cheng Peng, Yoseph Atilaw, Jinan Wang, Zhijian Xu, Vasanthanathan Poongavanam, Jiye Shi, Jan Kihlberg, Weiliang Zhu, <span class="NLM_string-name hlFld-ContribAuthor">Máté Erdélyi</span>. </span><span class="cited-content_cbyCitation_article-title">Conformation of the Macrocyclic Drug Lorlatinib in Polar and Nonpolar Environments: A MD Simulation and NMR Study. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (26)
                                     , 22245-22250. <a href="https://doi.org/10.1021/acsomega.9b03797" title="DOI URL">https://doi.org/10.1021/acsomega.9b03797</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b03797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b03797%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DConformation%252Bof%252Bthe%252BMacrocyclic%252BDrug%252BLorlatinib%252Bin%252BPolar%252Band%252BNonpolar%252BEnvironments%25253A%252BA%252BMD%252BSimulation%252Band%252BNMR%252BStudy%26aulast%3DPeng%26aufirst%3DCheng%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07112019%26date%3D26112019%26date%3D16122019%26volume%3D4%26issue%3D26%26spage%3D22245%26epage%3D22250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Megan Egbert, Adrian Whitty, György M. Keserű, <span class="NLM_string-name hlFld-ContribAuthor">Sandor Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Why Some Targets Benefit from beyond Rule of Five Drugs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (22)
                                     , 10005-10025. <a href="https://doi.org/10.1021/acs.jmedchem.8b01732" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01732%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhy%252BSome%252BTargets%252BBenefit%252Bfrom%252Bbeyond%252BRule%252Bof%252BFive%252BDrugs%26aulast%3DEgbert%26aufirst%3DMegan%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13112018%26date%3D26062019%26date%3D12062019%26volume%3D62%26issue%3D22%26spage%3D10005%26epage%3D10025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Isaac
W. Haynes, Guangcheng Wu, Md. Ashraful Haque, Hao Li, <span class="NLM_string-name hlFld-ContribAuthor">Thanh D. Do</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Preference of Macrocycles Investigated by Ion-Mobility Mass Spectrometry and Distance Geometry Modeling. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2019,</strong> <em>91 </em>
                                    (21)
                                     , 13439-13447. <a href="https://doi.org/10.1021/acs.analchem.9b02100" title="DOI URL">https://doi.org/10.1021/acs.analchem.9b02100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.9b02100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.9b02100%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DConformational%252BPreference%252Bof%252BMacrocycles%252BInvestigated%252Bby%252BIon-Mobility%252BMass%252BSpectrometry%252Band%252BDistance%252BGeometry%252BModeling%26aulast%3DHaynes%26aufirst%3DIsaac%2BW.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04052019%26date%3D10102019%26date%3D24102019%26date%3D10102019%26volume%3D91%26issue%3D21%26spage%3D13439%26epage%3D13447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Junting Zhou, Xueyang Jiang, Siyu He, Hongli Jiang, Feng Feng, Wenyuan Liu, Wei Qu, <span class="NLM_string-name hlFld-ContribAuthor">Haopeng Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design of Multitarget-Directed Ligands: Strategies and Emerging Paradigms. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 8881-8914. <a href="https://doi.org/10.1021/acs.jmedchem.9b00017" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00017%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Bof%252BMultitarget-Directed%252BLigands%25253A%252BStrategies%252Band%252BEmerging%252BParadigms%26aulast%3DZhou%26aufirst%3DJunting%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04012019%26date%3D28052019%26date%3D13052019%26volume%3D62%26issue%3D20%26spage%3D8881%26epage%3D8914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yichao Huang, Mareike Margarete Wiedmann, <span class="NLM_string-name hlFld-ContribAuthor">Hiroaki Suga</span>. </span><span class="cited-content_cbyCitation_article-title">RNA Display Methods for the Discovery of Bioactive Macrocycles. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (17)
                                     , 10360-10391. <a href="https://doi.org/10.1021/acs.chemrev.8b00430" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00430</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00430%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DRNA%252BDisplay%252BMethods%252Bfor%252Bthe%252BDiscovery%252Bof%252BBioactive%252BMacrocycles%26aulast%3DHuang%26aufirst%3DYichao%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D08072018%26date%3D05112018%26volume%3D119%26issue%3D17%26spage%3D10360%26epage%3D10391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kim T. Mortensen, Thomas J. Osberger, Thomas A. King, Hannah F. Sore, <span class="NLM_string-name hlFld-ContribAuthor">David R. Spring</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies for the Diversity-Oriented Synthesis of Macrocycles. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (17)
                                     , 10288-10317. <a href="https://doi.org/10.1021/acs.chemrev.9b00084" title="DOI URL">https://doi.org/10.1021/acs.chemrev.9b00084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.9b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.9b00084%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DStrategies%252Bfor%252Bthe%252BDiversity-Oriented%252BSynthesis%252Bof%252BMacrocycles%26aulast%3DMortensen%26aufirst%3DKim%2BT.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04022019%26date%3D20062019%26volume%3D119%26issue%3D17%26spage%3D10288%26epage%3D10317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maxwell D. Cummings, <span class="NLM_string-name hlFld-ContribAuthor">Sivakumar Sekharan</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (15)
                                     , 6843-6853. <a href="https://doi.org/10.1021/acs.jmedchem.8b01985" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01985</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BMacrocycle%252BDesign%252Bin%252BSmall-Molecule%252BDrug%252BDiscovery%252Band%252BSimple%252BMetrics%252BTo%252BIdentify%252BOpportunities%252Bfor%252BMacrocyclization%252Bof%252BSmall-Molecule%252BLigands%26aulast%3DCummings%26aufirst%3DMaxwell%2BD.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18122018%26date%3D22032019%26date%3D12032019%26volume%3D62%26issue%3D15%26spage%3D6843%26epage%3D6853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Michael D. Shultz</span>. </span><span class="cited-content_cbyCitation_article-title">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (4)
                                     , 1701-1714. <a href="https://doi.org/10.1021/acs.jmedchem.8b00686" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTwo%252BDecades%252Bunder%252Bthe%252BInfluence%252Bof%252Bthe%252BRule%252Bof%252BFive%252Band%252Bthe%252BChanging%252BProperties%252Bof%252BApproved%252BOral%252BDrugs%26aulast%3DShultz%26aufirst%3DMichael%2BD.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D30042018%26date%3D27092018%26date%3D13092018%26volume%3D62%26issue%3D4%26spage%3D1701%26epage%3D1714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jagna Witek, Shuzhe Wang, Benjamin Schroeder, Robin Lingwood, Andreas Dounas, Hans-Jörg Roth, Marianne Fouché, Markus Blatter, Oliver Lemke, Bettina Keller, <span class="NLM_string-name hlFld-ContribAuthor">Sereina Riniker</span>. </span><span class="cited-content_cbyCitation_article-title">Rationalization of the Membrane Permeability Differences in a Series of Analogue Cyclic Decapeptides. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 294-308. <a href="https://doi.org/10.1021/acs.jcim.8b00485" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00485</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00485%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRationalization%252Bof%252Bthe%252BMembrane%252BPermeability%252BDifferences%252Bin%252Ba%252BSeries%252Bof%252BAnalogue%252BCyclic%252BDecapeptides%26aulast%3DWitek%26aufirst%3DJagna%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21072018%26date%3D04122018%26date%3D20112018%26volume%3D59%26issue%3D1%26spage%3D294%26epage%3D308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bradley
C. Doak, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclophilin Succumbs to a Macrocyclic Chameleon. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (21)
                                     , 9469-9472. <a href="https://doi.org/10.1021/acs.jmedchem.8b01555" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01555</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01555%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCyclophilin%252BSuccumbs%252Bto%252Ba%252BMacrocyclic%252BChameleon%26aulast%3DDoak%26aufirst%3DBradley%2BC.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D05102018%26date%3D25102018%26volume%3D61%26issue%3D21%26spage%3D9469%26epage%3D9472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vasanthanathan Poongavanam, Emma Danelius, Stefan Peintner, Lilian Alcaraz, Giulia Caron, Maxwell D. Cummings, Stanislaw Wlodek, Mate Erdelyi, Paul C. D. Hawkins, Giuseppe Ermondi, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (9)
                                     , 11742-11757. <a href="https://doi.org/10.1021/acsomega.8b01379" title="DOI URL">https://doi.org/10.1021/acsomega.8b01379</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DConformational%252BSampling%252Bof%252BMacrocyclic%252BDrugs%252Bin%252BDifferent%252BEnvironments%25253A%252BCan%252BWe%252BFind%252Bthe%252BRelevant%252BConformations%25253F%26aulast%3DPoongavanam%26aufirst%3DVasanthanathan%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D19062018%26date%3D04092018%26volume%3D3%26issue%3D9%26spage%3D11742%26epage%3D11757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mohit Tyagi, Vasanthanathan Poongavanam, Marika Lindhagen, Anna Pettersen, Peter Sjö, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Toward the Design of Molecular Chameleons: Flexible Shielding of an Amide Bond Enhances Macrocycle Cell Permeability. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2018,</strong> <em>20 </em>
                                    (18)
                                     , 5737-5742. <a href="https://doi.org/10.1021/acs.orglett.8b02447" title="DOI URL">https://doi.org/10.1021/acs.orglett.8b02447</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.8b02447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.8b02447%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DToward%252Bthe%252BDesign%252Bof%252BMolecular%252BChameleons%25253A%252BFlexible%252BShielding%252Bof%252Ban%252BAmide%252BBond%252BEnhances%252BMacrocycle%252BCell%252BPermeability%26aulast%3DTyagi%26aufirst%3DMohit%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01082018%26date%3D12092018%26volume%3D20%26issue%3D18%26spage%3D5737%26epage%3D5742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Young, <span class="NLM_string-name hlFld-ContribAuthor">Paul D. Leeson</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping the Efficiency and Physicochemical Trajectories of Successful Optimizations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6421-6467. <a href="https://doi.org/10.1021/acs.jmedchem.8b00180" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00180%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMapping%252Bthe%252BEfficiency%252Band%252BPhysicochemical%252BTrajectories%252Bof%252BSuccessful%252BOptimizations%26aulast%3DYoung%26aufirst%3DRobert%2BJ.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D03022018%26date%3D25042018%26date%3D05042018%26volume%3D61%26issue%3D15%26spage%3D6421%26epage%3D6467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matteo Rossi Sebastiano, Bradley C. Doak, Maria Backlund, Vasanthanathan Poongavanam, Björn Over, Giuseppe Ermondi, Giulia Caron, Pär Matsson, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Dynamically Exposed Polarity on Permeability and Solubility of Chameleonic Drugs Beyond the Rule of 5. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (9)
                                     , 4189-4202. <a href="https://doi.org/10.1021/acs.jmedchem.8b00347" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00347</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00347%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImpact%252Bof%252BDynamically%252BExposed%252BPolarity%252Bon%252BPermeability%252Band%252BSolubility%252Bof%252BChameleonic%252BDrugs%252BBeyond%252Bthe%252BRule%252Bof%252B5%26aulast%3DRossi%2BSebastiano%26aufirst%3DMatteo%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D04032018%26date%3D16042018%26date%3D02042018%26volume%3D61%26issue%3D9%26spage%3D4189%26epage%3D4202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David A. DeGoey, Hui-Ju Chen, Philip B. Cox, <span class="NLM_string-name hlFld-ContribAuthor">Michael D. Wendt</span>. </span><span class="cited-content_cbyCitation_article-title">Beyond the Rule of 5: Lessons Learned from AbbVie’s Drugs and Compound Collection. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (7)
                                     , 2636-2651. <a href="https://doi.org/10.1021/acs.jmedchem.7b00717" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00717</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBeyond%252Bthe%252BRule%252Bof%252B5%25253A%252BLessons%252BLearned%252Bfrom%252BAbbVie%2525E2%252580%252599s%252BDrugs%252Band%252BCompound%252BCollection%26aulast%3DDeGoey%26aufirst%3DDavid%2BA.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D15052017%26date%3D27092017%26date%3D19092017%26volume%3D61%26issue%3D7%26spage%3D2636%26epage%3D2651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hannes Ludewig, Clarissa M. Czekster, Emilia Oueis, Elizabeth S. Munday, Mohammed Arshad, Silvia A. Synowsky, Andrew F. Bent, <span class="NLM_string-name hlFld-ContribAuthor">James H. Naismith</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of the Fast and Promiscuous Macrocyclase from Plant PCY1 Enables the Use of Simple Substrates. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (3)
                                     , 801-811. <a href="https://doi.org/10.1021/acschembio.8b00050" title="DOI URL">https://doi.org/10.1021/acschembio.8b00050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00050%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCharacterization%252Bof%252Bthe%252BFast%252Band%252BPromiscuous%252BMacrocyclase%252Bfrom%252BPlant%252BPCY1%252BEnables%252Bthe%252BUse%252Bof%252BSimple%252BSubstrates%26aulast%3DLudewig%26aufirst%3DHannes%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16012018%26date%3D29012018%26date%3D12022018%26date%3D29012018%26volume%3D13%26issue%3D3%26spage%3D801%26epage%3D811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew P.  Crew</span>, <span class="hlFld-ContribAuthor ">Kanak  Raina</span>, <span class="hlFld-ContribAuthor ">Hanqing  Dong</span>, <span class="hlFld-ContribAuthor ">Yimin  Qian</span>, <span class="hlFld-ContribAuthor ">Jing  Wang</span>, <span class="hlFld-ContribAuthor ">Dominico  Vigil</span>, <span class="hlFld-ContribAuthor ">Yevgeniy V.  Serebrenik</span>, <span class="hlFld-ContribAuthor ">Brian D.  Hamman</span>, <span class="hlFld-ContribAuthor ">Alicia  Morgan</span>, <span class="hlFld-ContribAuthor ">Caterina  Ferraro</span>, <span class="hlFld-ContribAuthor ">Kam  Siu</span>, <span class="hlFld-ContribAuthor ">Taavi K.  Neklesa</span>, <span class="hlFld-ContribAuthor ">James D.  Winkler</span>, <span class="hlFld-ContribAuthor ">Kevin G.  Coleman</span>, and <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (2)
                                     , 583-598. <a href="https://doi.org/10.1021/acs.jmedchem.7b00635" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00635</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Band%252BCharacterization%252Bof%252BVon%252BHippel-Lindau-Recruiting%252BProteolysis%252BTargeting%252BChimeras%252B%252528PROTACs%252529%252Bof%252BTANK-Binding%252BKinase%252B1%26aulast%3DCrew%26aufirst%3DAndrew%2BP.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D04052017%26date%3D10072017%26date%3D25012018%26volume%3D61%26issue%3D2%26spage%3D583%26epage%3D598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhengrong  Zhu</span>, <span class="hlFld-ContribAuthor ">Alex  Shaginian</span>, <span class="hlFld-ContribAuthor ">LaShadric C.  Grady</span>, <span class="hlFld-ContribAuthor ">Thomas  O’Keeffe</span>, <span class="hlFld-ContribAuthor ">Xiangguo E.  Shi</span>, <span class="hlFld-ContribAuthor ">Christopher P.  Davie</span>, <span class="hlFld-ContribAuthor ">Graham L.  Simpson</span>, <span class="hlFld-ContribAuthor ">Jeffrey A.  Messer</span>, <span class="hlFld-ContribAuthor ">Ghotas  Evindar</span>, <span class="hlFld-ContribAuthor ">Robert N.  Bream</span>, <span class="hlFld-ContribAuthor ">Praew P.  Thansandote</span>, <span class="hlFld-ContribAuthor ">Naomi R.  Prentice</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Mason</span>, and <span class="hlFld-ContribAuthor ">Sandeep  Pal</span>  . </span><span class="cited-content_cbyCitation_article-title">Design and Application of a DNA-Encoded Macrocyclic Peptide Library. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (1)
                                     , 53-59. <a href="https://doi.org/10.1021/acschembio.7b00852" title="DOI URL">https://doi.org/10.1021/acschembio.7b00852</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00852&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00852%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDesign%252Band%252BApplication%252Bof%252Ba%252BDNA-Encoded%252BMacrocyclic%252BPeptide%252BLibrary%26aulast%3DZhu%26aufirst%3DZhengrong%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28092017%26date%3D29112017%26date%3D04122017%26date%3D19012018%26date%3D29112017%26volume%3D13%26issue%3D1%26spage%3D53%26epage%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eman M. M.  Abdelraheem</span>, <span class="hlFld-ContribAuthor ">Michel P.  de Haan</span>, <span class="hlFld-ContribAuthor ">Pravin  Patil</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kurpiewska</span>, <span class="hlFld-ContribAuthor ">Justyna  Kalinowska-Tłuścik</span>, <span class="hlFld-ContribAuthor ">Shabnam  Shaabani</span>, and <span class="hlFld-ContribAuthor ">Alexander  Dömling</span>  . </span><span class="cited-content_cbyCitation_article-title">Concise Synthesis of Tetrazole Macrocycle. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2017,</strong> <em>19 </em>
                                    (19)
                                     , 5078-5081. <a href="https://doi.org/10.1021/acs.orglett.7b02319" title="DOI URL">https://doi.org/10.1021/acs.orglett.7b02319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.7b02319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.7b02319%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DConcise%252BSynthesis%252Bof%252BTetrazole%252BMacrocycle%26aulast%3DAbdelraheem%26aufirst%3DEman%2BM.%2BM.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D27072017%26date%3D13092017%26date%3D06102017%26volume%3D19%26issue%3D19%26spage%3D5078%26epage%3D5081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dan  Sindhikara</span>, <span class="hlFld-ContribAuthor ">Steven A.  Spronk</span>, <span class="hlFld-ContribAuthor ">Tyler  Day</span>, <span class="hlFld-ContribAuthor ">Ken  Borrelli</span>, <span class="hlFld-ContribAuthor ">Daniel L.  Cheney</span>, and <span class="hlFld-ContribAuthor ">Shana L.  Posy</span>  . </span><span class="cited-content_cbyCitation_article-title">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 1881-1894. <a href="https://doi.org/10.1021/acs.jcim.7b00052" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00052%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BAccuracy%25252C%252BDiversity%25252C%252Band%252BSpeed%252Bwith%252BPrime%252BMacrocycle%252BConformational%252BSampling%26aulast%3DSindhikara%26aufirst%3DDan%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D27012017%26date%3D08082017%26date%3D28082017%26date%3D20072017%26volume%3D57%26issue%3D8%26spage%3D1881%26epage%3D1894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcian E.  Van Dort</span>, <span class="hlFld-ContribAuthor ">Stefanie  Galbán</span>, <span class="hlFld-ContribAuthor ">Charles A.  Nino</span>, <span class="hlFld-ContribAuthor ">Hao  Hong</span>, <span class="hlFld-ContribAuthor ">April A.  Apfelbaum</span>, <span class="hlFld-ContribAuthor ">Gary D.  Luker</span>, <span class="hlFld-ContribAuthor ">Greg M.  Thurber</span>, <span class="hlFld-ContribAuthor ">Lydia  Atangcho</span>, <span class="hlFld-ContribAuthor ">Cagri G.  Besirli</span>, and <span class="hlFld-ContribAuthor ">Brian D.  Ross</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Design and Initial Studies of a Bifunctional MEK/PI3K Inhibitor (ST-168). </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (8)
                                     , 808-813. <a href="https://doi.org/10.1021/acsmedchemlett.7b00111" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.7b00111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.7b00111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.7b00111%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure-Guided%252BDesign%252Band%252BInitial%252BStudies%252Bof%252Ba%252BBifunctional%252BMEK%25252FPI3K%252BInhibitor%252B%252528ST-168%252529%26aulast%3DVan%2BDort%26aufirst%3DMarcian%2BE.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D15032017%26date%3D24072017%26date%3D25072017%26date%3D10082017%26date%3D24072017%26volume%3D8%26issue%3D8%26spage%3D808%26epage%3D813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon J.  Burkhart</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schwalen</span>, <span class="hlFld-ContribAuthor ">Greg  Mann</span>, <span class="hlFld-ContribAuthor ">James H.  Naismith</span>, and <span class="hlFld-ContribAuthor ">Douglas A.  Mitchell</span>  . </span><span class="cited-content_cbyCitation_article-title">YcaO-Dependent Posttranslational Amide Activation: Biosynthesis, Structure, and Function. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2017,</strong> <em>117 </em>
                                    (8)
                                     , 5389-5456. <a href="https://doi.org/10.1021/acs.chemrev.6b00623" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00623%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DYcaO-Dependent%252BPosttranslational%252BAmide%252BActivation%25253A%252BBiosynthesis%25252C%252BStructure%25252C%252Band%252BFunction%26aulast%3DBurkhart%26aufirst%3DBrandon%2BJ.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D12092016%26date%3D03032017%26date%3D26042017%26volume%3D117%26issue%3D8%26spage%3D5389%26epage%3D5456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joris W.  De Schutter</span>, <span class="hlFld-ContribAuthor ">James P.  Morrison</span>, <span class="hlFld-ContribAuthor ">Michael J.  Morrison</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>, and <span class="hlFld-ContribAuthor ">Barbara  Imperiali</span>  . </span><span class="cited-content_cbyCitation_article-title">Targeting Bacillosamine Biosynthesis in Bacterial Pathogens: Development of Inhibitors to a Bacterial Amino-Sugar Acetyltransferase from Campylobacter jejuni. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 2099-2118. <a href="https://doi.org/10.1021/acs.jmedchem.6b01869" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01869</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01869%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBacillosamine%252BBiosynthesis%252Bin%252BBacterial%252BPathogens%25253A%252BDevelopment%252Bof%252BInhibitors%252Bto%252Ba%252BBacterial%252BAmino-Sugar%252BAcetyltransferase%252Bfrom%252BCampylobacter%252Bjejuni%26aulast%3DDe%2BSchutter%26aufirst%3DJoris%2BW.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D21122016%26date%3D22022017%26date%3D09032017%26date%3D09022017%26volume%3D60%26issue%3D5%26spage%3D2099%26epage%3D2118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pär  Matsson</span> and <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>  . </span><span class="cited-content_cbyCitation_article-title">How Big Is Too Big for Cell Permeability?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1662-1664. <a href="https://doi.org/10.1021/acs.jmedchem.7b00237" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00237%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHow%252BBig%252BIs%252BToo%252BBig%252Bfor%252BCell%252BPermeability%25253F%26aulast%3DMatsson%26aufirst%3DP%25C3%25A4r%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D12022017%26date%3D24022017%26date%3D09032017%26volume%3D60%26issue%3D5%26spage%3D1662%26epage%3D1664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hiba  Alogheli</span>, <span class="hlFld-ContribAuthor ">Gustav  Olanders</span>, <span class="hlFld-ContribAuthor ">Wesley  Schaal</span>, <span class="hlFld-ContribAuthor ">Peter  Brandt</span>, and <span class="hlFld-ContribAuthor ">Anders  Karlén</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking of Macrocycles: Comparing Rigid and Flexible Docking in Glide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (2)
                                     , 190-202. <a href="https://doi.org/10.1021/acs.jcim.6b00443" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00443%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDocking%252Bof%252BMacrocycles%25253A%252BComparing%252BRigid%252Band%252BFlexible%252BDocking%252Bin%252BGlide%26aulast%3DAlogheli%26aufirst%3DHiba%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D01082016%26date%3D02022017%26date%3D27022017%26date%3D12012017%26volume%3D57%26issue%3D2%26spage%3D190%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Allyn T.  Londregan</span>, <span class="hlFld-ContribAuthor ">David W.  Piotrowski</span>, and <span class="hlFld-ContribAuthor ">Liuqing  Wei</span>  . </span><span class="cited-content_cbyCitation_article-title">Synthesis of Pyridazine-Based α-Helix Mimetics. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2016,</strong> <em>18 </em>
                                    (10)
                                     , 651-654. <a href="https://doi.org/10.1021/acscombsci.6b00111" title="DOI URL">https://doi.org/10.1021/acscombsci.6b00111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.6b00111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.6b00111%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DSynthesis%252Bof%252BPyridazine-Based%252B%2525CE%2525B1-Helix%252BMimetics%26aulast%3DLondregan%26aufirst%3DAllyn%2BT.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D25072016%26date%3D24082016%26date%3D14092016%26date%3D10102016%26date%3D29082016%26volume%3D18%26issue%3D10%26spage%3D651%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jagna  Witek</span>, <span class="hlFld-ContribAuthor ">Bettina G.  Keller</span>, <span class="hlFld-ContribAuthor ">Markus  Blatter</span>, <span class="hlFld-ContribAuthor ">Axel  Meissner</span>, <span class="hlFld-ContribAuthor ">Trixie  Wagner</span>, and <span class="hlFld-ContribAuthor ">Sereina  Riniker</span>  . </span><span class="cited-content_cbyCitation_article-title">Kinetic Models of Cyclosporin A in Polar and Apolar Environments Reveal Multiple Congruent Conformational States. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (8)
                                     , 1547-1562. <a href="https://doi.org/10.1021/acs.jcim.6b00251" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00251</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00251%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DKinetic%252BModels%252Bof%252BCyclosporin%252BA%252Bin%252BPolar%252Band%252BApolar%252BEnvironments%252BReveal%252BMultiple%252BCongruent%252BConformational%252BStates%26aulast%3DWitek%26aufirst%3DJagna%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D02052016%26date%3D19072016%26date%3D22082016%26date%3D07072016%26volume%3D56%26issue%3D8%26spage%3D1547%26epage%3D1562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Ugolini</span>, <span class="hlFld-ContribAuthor ">Mireille  Kenigsberg</span>, <span class="hlFld-ContribAuthor ">Alexey  Rak</span>, <span class="hlFld-ContribAuthor ">Francois  Vallée</span>, <span class="hlFld-ContribAuthor ">Jacques  Houtmann</span>, <span class="hlFld-ContribAuthor ">Maryse  Lowinski</span>, <span class="hlFld-ContribAuthor ">Cécile  Capdevila</span>, <span class="hlFld-ContribAuthor ">Jean  Khider</span>, <span class="hlFld-ContribAuthor ">Eva  Albert</span>, <span class="hlFld-ContribAuthor ">Nathalie  Martinet</span>, <span class="hlFld-ContribAuthor ">Conception  Nemecek</span>, <span class="hlFld-ContribAuthor ">Sandrine  Grapinet</span>, <span class="hlFld-ContribAuthor ">Eric  Bacqué</span>, <span class="hlFld-ContribAuthor ">Manfred  Roesner</span>, <span class="hlFld-ContribAuthor ">Christine  Delaisi</span>, <span class="hlFld-ContribAuthor ">Loreley  Calvet</span>, <span class="hlFld-ContribAuthor ">Fabrice  Bonche</span>, <span class="hlFld-ContribAuthor ">Dorothée  Semiond</span>, <span class="hlFld-ContribAuthor ">Coumaran  Egile</span>, <span class="hlFld-ContribAuthor ">Hélène  Goulaouic</span>, and <span class="hlFld-ContribAuthor ">Laurent  Schio</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Pharmacokinetic and Pharmacological Properties of the Potent and Selective MET Kinase Inhibitor 1-{6-[6-(4-Fluorophenyl)-[1,2,4]triazolo[4,3-b]pyridazin-3-ylsulfanyl]benzothiazol-2-yl}-3-(2-morpholin-4-ylethyl)urea (SAR125844). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (15)
                                     , 7066-7074. <a href="https://doi.org/10.1021/acs.jmedchem.6b00280" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00280%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BPharmacokinetic%252Band%252BPharmacological%252BProperties%252Bof%252Bthe%252BPotent%252Band%252BSelective%252BMET%252BKinase%252BInhibitor%252B1-%25257B6-%25255B6-%2525284-Fluorophenyl%252529-%25255B1%25252C2%25252C4%25255Dtriazolo%25255B4%25252C3-b%25255Dpyridazin-3-ylsulfanyl%25255Dbenzothiazol-2-yl%25257D-3-%2525282-morpholin-4-ylethyl%252529urea%252B%252528SAR125844%252529%26aulast%3DUgolini%26aufirst%3DAntonio%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D22022016%26date%3D13072016%26date%3D11082016%26date%3D29062016%26volume%3D59%26issue%3D15%26spage%3D7066%26epage%3D7074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua  Schwochert</span>, <span class="hlFld-ContribAuthor ">Yongtong  Lao</span>, <span class="hlFld-ContribAuthor ">Cameron R.  Pye</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Naylor</span>, <span class="hlFld-ContribAuthor ">Prashant V.  Desai</span>, <span class="hlFld-ContribAuthor ">Isabel C.  Gonzalez Valcarcel</span>, <span class="hlFld-ContribAuthor ">Jaclyn A.  Barrett</span>, <span class="hlFld-ContribAuthor ">Geri  Sawada</span>, <span class="hlFld-ContribAuthor ">Maria-Jesus  Blanco</span>, and <span class="hlFld-ContribAuthor ">R. Scott  Lokey</span>  . </span><span class="cited-content_cbyCitation_article-title">Stereochemistry Balances Cell Permeability and Solubility in the Naturally Derived Phepropeptin Cyclic Peptides. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (8)
                                     , 757-761. <a href="https://doi.org/10.1021/acsmedchemlett.6b00100" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00100%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStereochemistry%252BBalances%252BCell%252BPermeability%252Band%252BSolubility%252Bin%252Bthe%252BNaturally%252BDerived%252BPhepropeptin%252BCyclic%252BPeptides%26aulast%3DSchwochert%26aufirst%3DJoshua%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D08032016%26date%3D06062016%26date%3D10062016%26date%3D11082016%26date%3D06062016%26volume%3D7%26issue%3D8%26spage%3D757%26epage%3D761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raphael M.  Franzini</span> and <span class="hlFld-ContribAuthor ">Cassie  Randolph</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Space of DNA-Encoded Libraries. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (14)
                                     , 6629-6644. <a href="https://doi.org/10.1021/acs.jmedchem.5b01874" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01874</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01874%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChemical%252BSpace%252Bof%252BDNA-Encoded%252BLibraries%26aulast%3DFranzini%26aufirst%3DRaphael%2BM.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D04122015%26date%3D25022016%26date%3D28072016%26volume%3D59%26issue%3D14%26spage%3D6629%26epage%3D6644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicholas A.  Meanwell</span>  . </span><span class="cited-content_cbyCitation_article-title">Improving Drug Design: An Update on Recent Applications of Efficiency Metrics, Strategies for Replacing Problematic Elements, and Compounds in Nontraditional Drug Space. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Toxicology</span><span> <strong>2016,</strong> <em>29 </em>
                                    (4)
                                     , 564-616. <a href="https://doi.org/10.1021/acs.chemrestox.6b00043" title="DOI URL">https://doi.org/10.1021/acs.chemrestox.6b00043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrestox.6b00043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrestox.6b00043%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Toxicology%26atitle%3DImproving%252BDrug%252BDesign%25253A%252BAn%252BUpdate%252Bon%252BRecent%252BApplications%252Bof%252BEfficiency%252BMetrics%25252C%252BStrategies%252Bfor%252BReplacing%252BProblematic%252BElements%25252C%252Band%252BCompounds%252Bin%252BNontraditional%252BDrug%252BSpace%26aulast%3DMeanwell%26aufirst%3DNicholas%2BA.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D03022016%26date%3D14032016%26date%3D18042016%26volume%3D29%26issue%3D4%26spage%3D564%26epage%3D616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiamin  Zheng</span>, <span class="hlFld-ContribAuthor ">Dongdong  Chen</span>, <span class="hlFld-ContribAuthor ">Jie  Xu</span>, <span class="hlFld-ContribAuthor ">Xiao  Ding</span>, <span class="hlFld-ContribAuthor ">Yao  Wu</span>, <span class="hlFld-ContribAuthor ">Hong C.  Shen</span>, <span class="hlFld-ContribAuthor ">Xuefei  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule approaches to treat autoimmune and inflammatory diseases (Part III): Targeting cytokines and cytokine receptor complexes. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>48 </em>, 128229. <a href="https://doi.org/10.1016/j.bmcl.2021.128229" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128229%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSmall%252Bmolecule%252Bapproaches%252Bto%252Btreat%252Bautoimmune%252Band%252Binflammatory%252Bdiseases%252B%252528Part%252BIII%252529%25253A%252BTargeting%252Bcytokines%252Band%252Bcytokine%252Breceptor%252Bcomplexes%26aulast%3DZheng%26aufirst%3DJiamin%26date%3D2021%26volume%3D48%26spage%3D128229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuteng  Wu</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Bertran</span>, <span class="hlFld-ContribAuthor ">James  Rowley</span>, <span class="hlFld-ContribAuthor ">Ewen D. D.  Calder</span>, <span class="hlFld-ContribAuthor ">Dhira  Joshi</span>, <span class="hlFld-ContribAuthor ">Louise J.  Walport</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorescent Amino Acid Initiated
              de novo
              Cyclic Peptides for the Label‐Free Assessment of Cell Permeability**. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>48 </em><a href="https://doi.org/10.1002/cmdc.202100315" title="DOI URL">https://doi.org/10.1002/cmdc.202100315</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100315%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DFluorescent%252BAmino%252BAcid%252BInitiated%252Bde%252Bnovo%252BCyclic%252BPeptides%252Bfor%252Bthe%252BLabel%2525E2%252580%252590Free%252BAssessment%252Bof%252BCell%252BPermeability%252A%252A%26aulast%3DWu%26aufirst%3DYuteng%26date%3D2021%26date%3D2021%26volume%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piotr  Raubo</span>, <span class="hlFld-ContribAuthor ">Rodrigo J.  Carbajo</span>, <span class="hlFld-ContribAuthor ">William  McCoull</span>, <span class="hlFld-ContribAuthor ">Joanna  Raubo</span>, <span class="hlFld-ContribAuthor ">Morgan  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Diversity-orientated synthesis of macrocyclic heterocycles using a double S
              N
              Ar approach. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (28)
                                     , 6274-6290. <a href="https://doi.org/10.1039/D1OB00612F" title="DOI URL">https://doi.org/10.1039/D1OB00612F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00612F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00612F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDiversity-orientated%252Bsynthesis%252Bof%252Bmacrocyclic%252Bheterocycles%252Busing%252Ba%252Bdouble%252BS%252BN%252BAr%252Bapproach%26aulast%3DRaubo%26aufirst%3DPiotr%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D28%26spage%3D6274%26epage%3D6290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dirk  Menche</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Simplified Polyketide Analogs: New Modalities beyond the Rule of 5. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (13)
                                     , 2068-2074. <a href="https://doi.org/10.1002/cmdc.202100150" title="DOI URL">https://doi.org/10.1002/cmdc.202100150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100150%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BSimplified%252BPolyketide%252BAnalogs%25253A%252BNew%252BModalities%252Bbeyond%252Bthe%252BRule%252Bof%252B5%26aulast%3DMenche%26aufirst%3DDirk%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D13%26spage%3D2068%26epage%3D2074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hepeng  Li</span>, <span class="hlFld-ContribAuthor ">Feiying  Zhang</span>, <span class="hlFld-ContribAuthor ">Qunying  Jin</span>, <span class="hlFld-ContribAuthor ">Tangjun  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Preparative separation and purification of Cyclosporin D from fungus Hypoxylon Spp. by improved closed-loop recycling counter-current chromatography. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chromatography A</span><span> <strong>2021,</strong> <em>1649 </em>, 462221. <a href="https://doi.org/10.1016/j.chroma.2021.462221" title="DOI URL">https://doi.org/10.1016/j.chroma.2021.462221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chroma.2021.462221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chroma.2021.462221%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chromatography%2520A%26atitle%3DPreparative%252Bseparation%252Band%252Bpurification%252Bof%252BCyclosporin%252BD%252Bfrom%252Bfungus%252BHypoxylon%252BSpp.%252Bby%252Bimproved%252Bclosed-loop%252Brecycling%252Bcounter-current%252Bchromatography%26aulast%3DLi%26aufirst%3DHepeng%26date%3D2021%26volume%3D1649%26spage%3D462221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanyuan  Jiao</span>, <span class="hlFld-ContribAuthor ">Jia  Xu</span>, <span class="hlFld-ContribAuthor ">Hong  Chen</span>, <span class="hlFld-ContribAuthor ">Qiuyan  Guo</span>, <span class="hlFld-ContribAuthor ">Xiaofang  Deng</span>, <span class="hlFld-ContribAuthor ">Tong  Zhang</span>, <span class="hlFld-ContribAuthor ">Jingbo  Zhang</span>, <span class="hlFld-ContribAuthor ">Chenjing  Shi</span>, <span class="hlFld-ContribAuthor ">Ping  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of the Main Active Components and Mechanism of Wang Bi Tablet in Treating Rheumatoid Arthritis Based on Integrative Pharmacology. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.669551" title="DOI URL">https://doi.org/10.3389/fphar.2021.669551</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.669551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.669551%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DIdentification%252Bof%252Bthe%252BMain%252BActive%252BComponents%252Band%252BMechanism%252Bof%252BWang%252BBi%252BTablet%252Bin%252BTreating%252BRheumatoid%252BArthritis%252BBased%252Bon%252BIntegrative%252BPharmacology%26aulast%3DJiao%26aufirst%3DYuanyuan%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zi-Yi  Yang</span>, <span class="hlFld-ContribAuthor ">Zhi-Jiang  Yang</span>, <span class="hlFld-ContribAuthor ">Ai-Ping  Lu</span>, <span class="hlFld-ContribAuthor ">Ting-Jun  Hou</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Scopy: an integrated negative design python library for desirable HTS/VS database design. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (3)
                                     <a href="https://doi.org/10.1093/bib/bbaa194" title="DOI URL">https://doi.org/10.1093/bib/bbaa194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbaa194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbaa194%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DScopy%25253A%252Ban%252Bintegrated%252Bnegative%252Bdesign%252Bpython%252Blibrary%252Bfor%252Bdesirable%252BHTS%25252FVS%252Bdatabase%252Bdesign%26aulast%3DYang%26aufirst%3DZi-Yi%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Henni-Karoliina  Ropponen</span>, <span class="hlFld-ContribAuthor ">Robert  Richter</span>, <span class="hlFld-ContribAuthor ">Anna K.H.  Hirsch</span>, <span class="hlFld-ContribAuthor ">Claus-Michael  Lehr</span>. </span><span class="cited-content_cbyCitation_article-title">Mastering the Gram-negative bacterial barrier – Chemical approaches to increase bacterial bioavailability of antibiotics. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2021,</strong> <em>172 </em>, 339-360. <a href="https://doi.org/10.1016/j.addr.2021.02.014" title="DOI URL">https://doi.org/10.1016/j.addr.2021.02.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2021.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2021.02.014%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DMastering%252Bthe%252BGram-negative%252Bbacterial%252Bbarrier%252B%2525E2%252580%252593%252BChemical%252Bapproaches%252Bto%252Bincrease%252Bbacterial%252Bbioavailability%252Bof%252Bantibiotics%26aulast%3DRopponen%26aufirst%3DHenni-Karoliina%26date%3D2021%26volume%3D172%26spage%3D339%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lester T.  Sigauke</span>, <span class="hlFld-ContribAuthor ">Özlem  Taştan Bishop</span>, <span class="hlFld-ContribAuthor ">Kevin A.  Lobb</span>. </span><span class="cited-content_cbyCitation_article-title">Introducing DerivatizeME and its Application in the Augmentation of a Natural Product Library. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Biophysics and Chemistry</span><span> <strong>2021,</strong> <em>20 </em>
                                    (03)
                                     , 233-250. <a href="https://doi.org/10.1142/S2737416521500101" title="DOI URL">https://doi.org/10.1142/S2737416521500101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1142/S2737416521500101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1142%2FS2737416521500101%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Biophysics%2520and%2520Chemistry%26atitle%3DIntroducing%252BDerivatizeME%252Band%252Bits%252BApplication%252Bin%252Bthe%252BAugmentation%252Bof%252Ba%252BNatural%252BProduct%252BLibrary%26aulast%3DSigauke%26aufirst%3DLester%2BT.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D03%26spage%3D233%26epage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David A.  DeGoey</span>, <span class="hlFld-ContribAuthor ">Philip B.  Cox</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Beyond the Rule of Five. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-35. <a href="https://doi.org/10.1002/0471266949.bmc258" title="DOI URL">https://doi.org/10.1002/0471266949.bmc258</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc258%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDiscovery%252BBeyond%252Bthe%252BRule%252Bof%252BFive%26aulast%3DDeGoey%26aufirst%3DDavid%2BA.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D35%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlotta  Cecchini</span>, <span class="hlFld-ContribAuthor ">Sara  Pannilunghi</span>, <span class="hlFld-ContribAuthor ">Sébastien  Tardy</span>, <span class="hlFld-ContribAuthor ">Leonardo  Scapozza</span>. </span><span class="cited-content_cbyCitation_article-title">From Conception to Development: Investigating PROTACs Features for Improved Cell Permeability and Successful Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.672267" title="DOI URL">https://doi.org/10.3389/fchem.2021.672267</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.672267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.672267%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFrom%252BConception%252Bto%252BDevelopment%25253A%252BInvestigating%252BPROTACs%252BFeatures%252Bfor%252BImproved%252BCell%252BPermeability%252Band%252BSuccessful%252BProtein%252BDegradation%26aulast%3DCecchini%26aufirst%3DCarlotta%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clinton G. L.  Veale</span>. </span><span class="cited-content_cbyCitation_article-title">Into the Fray! A Beginner's Guide to Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (8)
                                     , 1199-1225. <a href="https://doi.org/10.1002/cmdc.202000929" title="DOI URL">https://doi.org/10.1002/cmdc.202000929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000929%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DInto%252Bthe%252BFray%252521%252BA%252BBeginner%252527s%252BGuide%252Bto%252BMedicinal%252BChemistry%26aulast%3DVeale%26aufirst%3DClinton%2BG.%2BL.%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D8%26spage%3D1199%26epage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aranhikkal  Shamsiya</span>, <span class="hlFld-ContribAuthor ">Damodaran  Bahulayan</span>. </span><span class="cited-content_cbyCitation_article-title">Chromene–triazole-pyrimidine based chemosensor therapeutics for the
              in vivo
              and
              in vitro
              detection of Fe
              3+
              ions. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2021,</strong> <em>45 </em>
                                    (15)
                                     , 6760-6767. <a href="https://doi.org/10.1039/D0NJ05697A" title="DOI URL">https://doi.org/10.1039/D0NJ05697A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NJ05697A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NJ05697A%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DChromene%2525E2%252580%252593triazole-pyrimidine%252Bbased%252Bchemosensor%252Btherapeutics%252Bfor%252Bthe%252Bin%252Bvivo%252Band%252Bin%252Bvitro%252Bdetection%252Bof%252BFe%252B3%25252B%252Bions%26aulast%3DShamsiya%26aufirst%3DAranhikkal%26date%3D2021%26date%3D2021%26volume%3D45%26issue%3D15%26spage%3D6760%26epage%3D6767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hiroyuki  Miyachi</span>, <span class="hlFld-ContribAuthor ">Kayoko  Kanamitsu</span>, <span class="hlFld-ContribAuthor ">Mayumi  Ishii</span>, <span class="hlFld-ContribAuthor ">Eri  Watanabe</span>, <span class="hlFld-ContribAuthor ">Akira  Katsuyama</span>, <span class="hlFld-ContribAuthor ">Satoko  Otsuguro</span>, <span class="hlFld-ContribAuthor ">Fumika  Yakushiji</span>, <span class="hlFld-ContribAuthor ">Mizuki  Watanabe</span>, <span class="hlFld-ContribAuthor ">Kouhei  Matsui</span>, <span class="hlFld-ContribAuthor ">Yukina  Sato</span>, <span class="hlFld-ContribAuthor ">Satoshi  Shuto</span>, <span class="hlFld-ContribAuthor ">Takashi  Tadokoro</span>, <span class="hlFld-ContribAuthor ">Shunsuke  Kita</span>, <span class="hlFld-ContribAuthor ">Takanori  Matsumaru</span>, <span class="hlFld-ContribAuthor ">Akira  Matsuda</span>, <span class="hlFld-ContribAuthor ">Tomoyasu  Hirose</span>, <span class="hlFld-ContribAuthor ">Masato  Iwatsuki</span>, <span class="hlFld-ContribAuthor ">Yasuteru  Shigeta</span>, <span class="hlFld-ContribAuthor ">Tetsuo  Nagano</span>, <span class="hlFld-ContribAuthor ">Hirotatsu  Kojima</span>, <span class="hlFld-ContribAuthor ">Satoshi  Ichikawa</span>, <span class="hlFld-ContribAuthor ">Toshiaki  Sunazuka</span>, <span class="hlFld-ContribAuthor ">Katsumi  Maenaka</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, solubility, and permeability relationships in a diverse middle molecule library. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>37 </em>, 127847. <a href="https://doi.org/10.1016/j.bmcl.2021.127847" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127847</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127847%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure%25252C%252Bsolubility%25252C%252Band%252Bpermeability%252Brelationships%252Bin%252Ba%252Bdiverse%252Bmiddle%252Bmolecule%252Blibrary%26aulast%3DMiyachi%26aufirst%3DHiroyuki%26date%3D2021%26volume%3D37%26spage%3D127847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel J.  Ruiz-Moreno</span>, <span class="hlFld-ContribAuthor ">Atilio  Reyes-Romero</span>, <span class="hlFld-ContribAuthor ">Alexander  Dömling</span>, <span class="hlFld-ContribAuthor ">Marco A.  Velasco-Velázquez</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Design and Selection of New Tetrahydroisoquinoline-Based CD44 Antagonist Candidates. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (7)
                                     , 1877. <a href="https://doi.org/10.3390/molecules26071877" title="DOI URL">https://doi.org/10.3390/molecules26071877</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26071877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26071877%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIn%252BSilico%252BDesign%252Band%252BSelection%252Bof%252BNew%252BTetrahydroisoquinoline-Based%252BCD44%252BAntagonist%252BCandidates%26aulast%3DRuiz-Moreno%26aufirst%3DAngel%2BJ.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D7%26spage%3D1877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren A.  Viarengo-Baker</span>, <span class="hlFld-ContribAuthor ">Lauren E.  Brown</span>, <span class="hlFld-ContribAuthor ">Anna A.  Rzepiela</span>, <span class="hlFld-ContribAuthor ">Adrian  Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Defining and navigating macrocycle chemical space. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (12)
                                     , 4309-4328. <a href="https://doi.org/10.1039/D0SC05788F" title="DOI URL">https://doi.org/10.1039/D0SC05788F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05788F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05788F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DDefining%252Band%252Bnavigating%252Bmacrocycle%252Bchemical%252Bspace%26aulast%3DViarengo-Baker%26aufirst%3DLauren%2BA.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D12%26spage%3D4309%26epage%3D4328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ikumi  Kuriwaki</span>, <span class="hlFld-ContribAuthor ">Minoru  Kameda</span>, <span class="hlFld-ContribAuthor ">Kazuhiko  Iikubo</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Hisamichi</span>, <span class="hlFld-ContribAuthor ">Yuichiro  Kawamoto</span>, <span class="hlFld-ContribAuthor ">Shigetoshi  Kikuchi</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Moritomo</span>, <span class="hlFld-ContribAuthor ">Yutaka  Kondoh</span>, <span class="hlFld-ContribAuthor ">Tadashi  Terasaka</span>, <span class="hlFld-ContribAuthor ">Yasushi  Amano</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Tateishi</span>, <span class="hlFld-ContribAuthor ">Yuka  Echizen</span>, <span class="hlFld-ContribAuthor ">Yoshinori  Iwai</span>, <span class="hlFld-ContribAuthor ">Atsushi  Noda</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Tomiyama</span>, <span class="hlFld-ContribAuthor ">Taisuke  Nakazawa</span>, <span class="hlFld-ContribAuthor ">Masaaki  Hirano</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and structure-activity relationships of pyrimidine derivatives as potent and orally active FGFR3 inhibitors with both increased systemic exposure and enhanced in vitro potency. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>33 </em>, 116019. <a href="https://doi.org/10.1016/j.bmc.2021.116019" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116019%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bstructure-activity%252Brelationships%252Bof%252Bpyrimidine%252Bderivatives%252Bas%252Bpotent%252Band%252Borally%252Bactive%252BFGFR3%252Binhibitors%252Bwith%252Bboth%252Bincreased%252Bsystemic%252Bexposure%252Band%252Benhanced%252Bin%252Bvitro%252Bpotency%26aulast%3DKuriwaki%26aufirst%3DIkumi%26date%3D2021%26volume%3D33%26spage%3D116019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan  Bogaerts</span>, <span class="hlFld-ContribAuthor ">Yoseph  Atilaw</span>, <span class="hlFld-ContribAuthor ">Stefan  Peintner</span>, <span class="hlFld-ContribAuthor ">Roy  Aerts</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>, <span class="hlFld-ContribAuthor ">Christian  Johannessen</span>, <span class="hlFld-ContribAuthor ">Máté  Erdélyi</span>. </span><span class="cited-content_cbyCitation_article-title">Employing complementary spectroscopies to study the conformations of an epimeric pair of side-chain stapled peptides in aqueous solution. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (7)
                                     , 4200-4208. <a href="https://doi.org/10.1039/D0RA10167B" title="DOI URL">https://doi.org/10.1039/D0RA10167B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA10167B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA10167B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DEmploying%252Bcomplementary%252Bspectroscopies%252Bto%252Bstudy%252Bthe%252Bconformations%252Bof%252Ban%252Bepimeric%252Bpair%252Bof%252Bside-chain%252Bstapled%252Bpeptides%252Bin%252Baqueous%252Bsolution%26aulast%3DBogaerts%26aufirst%3DJonathan%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D7%26spage%3D4200%26epage%3D4208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazuto  Mochizuki</span>, <span class="hlFld-ContribAuthor ">Lisa  Matsukura</span>, <span class="hlFld-ContribAuthor ">Yuji  Ito</span>, <span class="hlFld-ContribAuthor ">Naoyuki  Miyashita</span>, <span class="hlFld-ContribAuthor ">Masumi  Taki</span>. </span><span class="cited-content_cbyCitation_article-title">A medium-firm drug-candidate library of cryptand-like structures on T7 phage: design and selection of a strong binder for Hsp90. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (1)
                                     , 146-150. <a href="https://doi.org/10.1039/D0OB01855D" title="DOI URL">https://doi.org/10.1039/D0OB01855D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB01855D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB01855D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DA%252Bmedium-firm%252Bdrug-candidate%252Blibrary%252Bof%252Bcryptand-like%252Bstructures%252Bon%252BT7%252Bphage%25253A%252Bdesign%252Band%252Bselection%252Bof%252Ba%252Bstrong%252Bbinder%252Bfor%252BHsp90%26aulast%3DMochizuki%26aufirst%3DKazuto%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D1%26spage%3D146%26epage%3D150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Taiwo Olayemi  Elufioye</span>, <span class="hlFld-ContribAuthor ">Adeboye  Adejare</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmaceutical profiling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 155-167. <a href="https://doi.org/10.1016/B978-0-12-820007-0.00008-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-820007-0.00008-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820007-0.00008-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820007-0.00008-8%26sid%3Dliteratum%253Aachs%26atitle%3DPharmaceutical%252Bprofiling%26aulast%3DElufioye%26aufirst%3DTaiwo%2BOlayemi%26date%3D2021%26spage%3D155%26epage%3D167%26pub%3DElsevier%26atitle%3DRemington%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeevan  Patra</span>, <span class="hlFld-ContribAuthor ">Deepanmol  Singh</span>, <span class="hlFld-ContribAuthor ">Sapna  Jain</span>, <span class="hlFld-ContribAuthor ">Neeraj  Mahindroo</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Docking for Lead Optimization. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 271-294. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00012-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00012-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00012-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00012-6%26sid%3Dliteratum%253Aachs%26atitle%3DApplication%252Bof%252BDocking%252Bfor%252BLead%252BOptimization%26aulast%3DPatra%26aufirst%3DJeevan%26date%3D2021%26spage%3D271%26epage%3D294%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Yoseph  Atilaw</span>, <span class="hlFld-ContribAuthor ">Sofie  Ye</span>, <span class="hlFld-ContribAuthor ">Lianne H.E.  Wieske</span>, <span class="hlFld-ContribAuthor ">Mate  Erdelyi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ermondi</span>, <span class="hlFld-ContribAuthor ">Giulia  Caron</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting the Permeability of Macrocycles from Conformational Sampling – Limitations of Molecular Flexibility. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical Sciences</span><span> <strong>2021,</strong> <em>110 </em>
                                    (1)
                                     , 301-313. <a href="https://doi.org/10.1016/j.xphs.2020.10.052" title="DOI URL">https://doi.org/10.1016/j.xphs.2020.10.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.xphs.2020.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.xphs.2020.10.052%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DPredicting%252Bthe%252BPermeability%252Bof%252BMacrocycles%252Bfrom%252BConformational%252BSampling%252B%2525E2%252580%252593%252BLimitations%252Bof%252BMolecular%252BFlexibility%26aulast%3DPoongavanam%26aufirst%3DVasanthanathan%26date%3D2021%26volume%3D110%26issue%3D1%26spage%3D301%26epage%3D313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renaud  Prudent</span>, <span class="hlFld-ContribAuthor ">D. Allen  Annis</span>, <span class="hlFld-ContribAuthor ">Peter J.  Dandliker</span>, <span class="hlFld-ContribAuthor ">Jean-Yves  Ortholand</span>, <span class="hlFld-ContribAuthor ">Didier  Roche</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring new targets and chemical space with affinity selection-mass spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Chemistry</span><span> <strong>2021,</strong> <em>5 </em>
                                    (1)
                                     , 62-71. <a href="https://doi.org/10.1038/s41570-020-00229-2" title="DOI URL">https://doi.org/10.1038/s41570-020-00229-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41570-020-00229-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41570-020-00229-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Chemistry%26atitle%3DExploring%252Bnew%252Btargets%252Band%252Bchemical%252Bspace%252Bwith%252Baffinity%252Bselection-mass%252Bspectrometry%26aulast%3DPrudent%26aufirst%3DRenaud%26date%3D2021%26date%3D2020%26volume%3D5%26issue%3D1%26spage%3D62%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosario  González-Muñiz</span>, <span class="hlFld-ContribAuthor ">María Ángeles  Bonache</span>, <span class="hlFld-ContribAuthor ">María Jesús  Pérez de Vega</span>. </span><span class="cited-content_cbyCitation_article-title">Modulating Protein–Protein Interactions by Cyclic and Macrocyclic Peptides. Prominent Strategies and Examples. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (2)
                                     , 445. <a href="https://doi.org/10.3390/molecules26020445" title="DOI URL">https://doi.org/10.3390/molecules26020445</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26020445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26020445%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DModulating%252BProtein%2525E2%252580%252593Protein%252BInteractions%252Bby%252BCyclic%252Band%252BMacrocyclic%252BPeptides.%252BProminent%252BStrategies%252Band%252BExamples%26aulast%3DGonz%25C3%25A1lez-Mu%25C3%25B1iz%26aufirst%3DRosario%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D2%26spage%3D445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samantha  Stone</span>, <span class="hlFld-ContribAuthor ">David J.  Newman</span>, <span class="hlFld-ContribAuthor ">Steven L.  Colletti</span>, <span class="hlFld-ContribAuthor ">Derek S.  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Cheminformatic analysis of natural product-based drugs and chemical probes. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>83 </em><a href="https://doi.org/10.1039/D1NP00039J" title="DOI URL">https://doi.org/10.1039/D1NP00039J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1NP00039J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1NP00039J%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DCheminformatic%252Banalysis%252Bof%252Bnatural%252Bproduct-based%252Bdrugs%252Band%252Bchemical%252Bprobes%26aulast%3DStone%26aufirst%3DSamantha%26date%3D2021%26date%3D2021%26volume%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas M.  Leissing</span>, <span class="hlFld-ContribAuthor ">Laura M.  Luh</span>, <span class="hlFld-ContribAuthor ">Philipp M.  Cromm</span>. </span><span class="cited-content_cbyCitation_article-title">Structure driven compound optimization in targeted protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2020,</strong> <em>61 </em><a href="https://doi.org/10.1016/j.ddtec.2020.11.005" title="DOI URL">https://doi.org/10.1016/j.ddtec.2020.11.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2020.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2020.11.005%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DStructure%252Bdriven%252Bcompound%252Boptimization%252Bin%252Btargeted%252Bprotein%252Bdegradation%26aulast%3DLeissing%26aufirst%3DThomas%2BM.%26date%3D2020%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112750. <a href="https://doi.org/10.1016/j.ejmech.2020.112750" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bepigenetic%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528Epi-PROTACs%252529%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26volume%3D207%26spage%3D112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Josephine  Eshon</span>, <span class="hlFld-ContribAuthor ">Kate A.  Nicastri</span>, <span class="hlFld-ContribAuthor ">Steven C.  Schmid</span>, <span class="hlFld-ContribAuthor ">William T.  Raskopf</span>, <span class="hlFld-ContribAuthor ">Ilia A.  Guzei</span>, <span class="hlFld-ContribAuthor ">Israel  Fernández</span>, <span class="hlFld-ContribAuthor ">Jennifer M.  Schomaker</span>. </span><span class="cited-content_cbyCitation_article-title">Intermolecular [3+3] ring expansion of aziridines to dehydropiperi-dines through the intermediacy of aziridinium ylides. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-15134-x" title="DOI URL">https://doi.org/10.1038/s41467-020-15134-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-15134-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-15134-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DIntermolecular%252B%25255B3%25252B3%25255D%252Bring%252Bexpansion%252Bof%252Baziridines%252Bto%252Bdehydropiperi-dines%252Bthrough%252Bthe%252Bintermediacy%252Bof%252Baziridinium%252Bylides%26aulast%3DEshon%26aufirst%3DJosephine%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akihiro  Furukawa</span>, <span class="hlFld-ContribAuthor ">Joshua  Schwochert</span>, <span class="hlFld-ContribAuthor ">Cameron R.  Pye</span>, <span class="hlFld-ContribAuthor ">Daigo  Asano</span>, <span class="hlFld-ContribAuthor ">Quinn D.  Edmondson</span>, <span class="hlFld-ContribAuthor ">Alexandra C.  Turmon</span>, <span class="hlFld-ContribAuthor ">Victoria G.  Klein</span>, <span class="hlFld-ContribAuthor ">Satoshi  Ono</span>, <span class="hlFld-ContribAuthor ">Okimasa  Okada</span>, <span class="hlFld-ContribAuthor ">R. Scott  Lokey</span>. </span><span class="cited-content_cbyCitation_article-title">Drug‐Like Properties in Macrocycles above MW 1000: Backbone Rigidity versus Side‐Chain Lipophilicity. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (48)
                                     , 21571-21577. <a href="https://doi.org/10.1002/anie.202004550" title="DOI URL">https://doi.org/10.1002/anie.202004550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202004550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202004550%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DDrug%2525E2%252580%252590Like%252BProperties%252Bin%252BMacrocycles%252Babove%252BMW%252B1000%25253A%252BBackbone%252BRigidity%252Bversus%252BSide%2525E2%252580%252590Chain%252BLipophilicity%26aulast%3DFurukawa%26aufirst%3DAkihiro%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D48%26spage%3D21571%26epage%3D21577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akihiro  Furukawa</span>, <span class="hlFld-ContribAuthor ">Joshua  Schwochert</span>, <span class="hlFld-ContribAuthor ">Cameron R.  Pye</span>, <span class="hlFld-ContribAuthor ">Daigo  Asano</span>, <span class="hlFld-ContribAuthor ">Quinn D.  Edmondson</span>, <span class="hlFld-ContribAuthor ">Alexandra C.  Turmon</span>, <span class="hlFld-ContribAuthor ">Victoria G.  Klein</span>, <span class="hlFld-ContribAuthor ">Satoshi  Ono</span>, <span class="hlFld-ContribAuthor ">Okimasa  Okada</span>, <span class="hlFld-ContribAuthor ">R. Scott  Lokey</span>. </span><span class="cited-content_cbyCitation_article-title">Drug‐Like Properties in Macrocycles above MW 1000: Backbone Rigidity versus Side‐Chain Lipophilicity. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (48)
                                     , 21755-21761. <a href="https://doi.org/10.1002/ange.202004550" title="DOI URL">https://doi.org/10.1002/ange.202004550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202004550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202004550%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDrug%2525E2%252580%252590Like%252BProperties%252Bin%252BMacrocycles%252Babove%252BMW%252B1000%25253A%252BBackbone%252BRigidity%252Bversus%252BSide%2525E2%252580%252590Chain%252BLipophilicity%26aulast%3DFurukawa%26aufirst%3DAkihiro%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D48%26spage%3D21755%26epage%3D21761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>, <span class="hlFld-ContribAuthor ">Guoshun  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting estrogen receptor α for degradation with PROTACs: A promising approach to overcome endocrine resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112689. <a href="https://doi.org/10.1016/j.ejmech.2020.112689" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112689%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bestrogen%252Breceptor%252B%2525CE%2525B1%252Bfor%252Bdegradation%252Bwith%252BPROTACs%25253A%252BA%252Bpromising%252Bapproach%252Bto%252Bovercome%252Bendocrine%252Bresistance%26aulast%3DLin%26aufirst%3DXin%26date%3D2020%26volume%3D206%26spage%3D112689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert I.  Troup</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>, <span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>. </span><span class="cited-content_cbyCitation_article-title">Current strategies for the design of PROTAC linkers: a critical review. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (5)
                                     <a href="https://doi.org/10.37349/etat.2020.00018" title="DOI URL">https://doi.org/10.37349/etat.2020.00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00018%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DCurrent%252Bstrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BPROTAC%252Blinkers%25253A%252Ba%252Bcritical%252Breview%26aulast%3DTroup%26aufirst%3DRobert%2BI.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Di</span>, <span class="hlFld-ContribAuthor ">Per  Artursson</span>, <span class="hlFld-ContribAuthor ">Alex  Avdeef</span>, <span class="hlFld-ContribAuthor ">Leslie Z.  Benet</span>, <span class="hlFld-ContribAuthor ">J. Brian  Houston</span>, <span class="hlFld-ContribAuthor ">Manfred  Kansy</span>, <span class="hlFld-ContribAuthor ">Edward H.  Kerns</span>, <span class="hlFld-ContribAuthor ">Hans  Lennernäs</span>, <span class="hlFld-ContribAuthor ">Dennis A.  Smith</span>, <span class="hlFld-ContribAuthor ">Kiyohiko  Sugano</span>. </span><span class="cited-content_cbyCitation_article-title">The Critical Role of Passive Permeability in Designing Successful Drugs. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (20)
                                     , 1862-1874. <a href="https://doi.org/10.1002/cmdc.202000419" title="DOI URL">https://doi.org/10.1002/cmdc.202000419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000419%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DThe%252BCritical%252BRole%252Bof%252BPassive%252BPermeability%252Bin%252BDesigning%252BSuccessful%252BDrugs%26aulast%3DDi%26aufirst%3DLi%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D20%26spage%3D1862%26epage%3D1874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rugmini  Ammal P</span>, <span class="hlFld-ContribAuthor ">Anupama R.  Prasad</span>, <span class="hlFld-ContribAuthor ">Abraham  Joseph</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, characterization, in silico, and in vitro biological screening of coordination compounds with 1,2,4-triazine based biocompatible ligands and selected 3d-metal ions. </span><span class="cited-content_cbyCitation_journal-name">Heliyon</span><span> <strong>2020,</strong> <em>6 </em>
                                    (10)
                                     , e05144. <a href="https://doi.org/10.1016/j.heliyon.2020.e05144" title="DOI URL">https://doi.org/10.1016/j.heliyon.2020.e05144</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.heliyon.2020.e05144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.heliyon.2020.e05144%26sid%3Dliteratum%253Aachs%26jtitle%3DHeliyon%26atitle%3DSynthesis%25252C%252Bcharacterization%25252C%252Bin%252Bsilico%25252C%252Band%252Bin%252Bvitro%252Bbiological%252Bscreening%252Bof%252Bcoordination%252Bcompounds%252Bwith%252B1%25252C2%25252C4-triazine%252Bbased%252Bbiocompatible%252Bligands%252Band%252Bselected%252B3d-metal%252Bions%26aulast%3DAmmal%2BP%26aufirst%3DRugmini%26date%3D2020%26volume%3D6%26issue%3D10%26spage%3De05144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sérgio  Albuquerque</span>, <span class="hlFld-ContribAuthor ">Lorenzo  Cianni</span>, <span class="hlFld-ContribAuthor ">Daniela  Vita</span>, <span class="hlFld-ContribAuthor ">Carla  Duque</span>, <span class="hlFld-ContribAuthor ">Ana S. M.  Gomes</span>, <span class="hlFld-ContribAuthor ">Paula  Gomes</span>, <span class="hlFld-ContribAuthor ">Charles  Laughton</span>, <span class="hlFld-ContribAuthor ">Andrei  Leitão</span>, <span class="hlFld-ContribAuthor ">Carlos A.  Montanari</span>, <span class="hlFld-ContribAuthor ">Raphael  Montanari</span>, <span class="hlFld-ContribAuthor ">Jean F. R.  Ribeiro</span>, <span class="hlFld-ContribAuthor ">João Santana  Silva</span>, <span class="hlFld-ContribAuthor ">Cátia  Teixeira</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design aided by random forests and synthesis of potent trypanocidal agents as cruzain inhibitors for Chagas disease treatment. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2020,</strong> <em>96 </em>
                                    (3)
                                     , 948-960. <a href="https://doi.org/10.1111/cbdd.13663" title="DOI URL">https://doi.org/10.1111/cbdd.13663</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13663%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DMolecular%252Bdesign%252Baided%252Bby%252Brandom%252Bforests%252Band%252Bsynthesis%252Bof%252Bpotent%252Btrypanocidal%252Bagents%252Bas%252Bcruzain%252Binhibitors%252Bfor%252BChagas%252Bdisease%252Btreatment%26aulast%3DAlbuquerque%26aufirst%3DS%25C3%25A9rgio%26date%3D2020%26date%3D2020%26volume%3D96%26issue%3D3%26spage%3D948%26epage%3D960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wiktor  Beker</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Wołos</span>, <span class="hlFld-ContribAuthor ">Sara  Szymkuć</span>, <span class="hlFld-ContribAuthor ">Bartosz A.  Grzybowski</span>. </span><span class="cited-content_cbyCitation_article-title">Minimal-uncertainty prediction of general drug-likeness based on Bayesian neural networks. </span><span class="cited-content_cbyCitation_journal-name">Nature Machine Intelligence</span><span> <strong>2020,</strong> <em>2 </em>
                                    (8)
                                     , 457-465. <a href="https://doi.org/10.1038/s42256-020-0209-y" title="DOI URL">https://doi.org/10.1038/s42256-020-0209-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42256-020-0209-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42256-020-0209-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Machine%2520Intelligence%26atitle%3DMinimal-uncertainty%252Bprediction%252Bof%252Bgeneral%252Bdrug-likeness%252Bbased%252Bon%252BBayesian%252Bneural%252Bnetworks%26aulast%3DBeker%26aufirst%3DWiktor%26date%3D2020%26date%3D2020%26volume%3D2%26issue%3D8%26spage%3D457%26epage%3D465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander V.  Strizhak</span>, <span class="hlFld-ContribAuthor ">Oleg  Babii</span>, <span class="hlFld-ContribAuthor ">Sergii  Afonin</span>, <span class="hlFld-ContribAuthor ">Iuliia  Bakanovich</span>, <span class="hlFld-ContribAuthor ">Teodors  Pantelejevs</span>, <span class="hlFld-ContribAuthor ">Wenshu  Xu</span>, <span class="hlFld-ContribAuthor ">Elaine  Fowler</span>, <span class="hlFld-ContribAuthor ">Rohan  Eapen</span>, <span class="hlFld-ContribAuthor ">Krishna  Sharma</span>, <span class="hlFld-ContribAuthor ">Maxim O.  Platonov</span>, <span class="hlFld-ContribAuthor ">Vasyl V.  Hurmach</span>, <span class="hlFld-ContribAuthor ">Laura  Itzhaki</span>, <span class="hlFld-ContribAuthor ">Marko  Hyvönen</span>, <span class="hlFld-ContribAuthor ">Anne S.  Ulrich</span>, <span class="hlFld-ContribAuthor ">David R.  Spring</span>, <span class="hlFld-ContribAuthor ">Igor V.  Komarov</span>. </span><span class="cited-content_cbyCitation_article-title">Diarylethene moiety as an enthalpy-entropy switch: photoisomerizable stapled peptides for modulating p53/MDM2 interaction. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (28)
                                     , 5359-5369. <a href="https://doi.org/10.1039/D0OB00831A" title="DOI URL">https://doi.org/10.1039/D0OB00831A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB00831A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB00831A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDiarylethene%252Bmoiety%252Bas%252Ban%252Benthalpy-entropy%252Bswitch%25253A%252Bphotoisomerizable%252Bstapled%252Bpeptides%252Bfor%252Bmodulating%252Bp53%25252FMDM2%252Binteraction%26aulast%3DStrizhak%26aufirst%3DAlexander%2BV.%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D28%26spage%3D5359%26epage%3D5369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jintong  Du</span>, <span class="hlFld-ContribAuthor ">Lulu  Liu</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Xuben  Hou</span>, <span class="hlFld-ContribAuthor ">Jinming  Yu</span>, <span class="hlFld-ContribAuthor ">Hao  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based virtual screening, biological evaluation and biophysical study of novel Mcl-1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1293-1304. <a href="https://doi.org/10.4155/fmc-2020-0114" title="DOI URL">https://doi.org/10.4155/fmc-2020-0114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0114%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bvirtual%252Bscreening%25252C%252Bbiological%252Bevaluation%252Band%252Bbiophysical%252Bstudy%252Bof%252Bnovel%252BMcl-1%252Binhibitors%26aulast%3DDu%26aufirst%3DJintong%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1293%26epage%3D1304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Igor V.  Komarov</span>, <span class="hlFld-ContribAuthor ">Iuliia V.  Bakanovych</span>. </span><span class="cited-content_cbyCitation_article-title">Photoregulated macrocyclic cell-penetrating peptides (microreview). </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2020,</strong> <em>56 </em>
                                    (6)
                                     , 719-721. <a href="https://doi.org/10.1007/s10593-020-02721-5" title="DOI URL">https://doi.org/10.1007/s10593-020-02721-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-020-02721-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-020-02721-5%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DPhotoregulated%252Bmacrocyclic%252Bcell-penetrating%252Bpeptides%252B%252528microreview%252529%26aulast%3DKomarov%26aufirst%3DIgor%2BV.%26date%3D2020%26date%3D2020%26volume%3D56%26issue%3D6%26spage%3D719%26epage%3D721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emma  Danelius</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Stefan  Peintner</span>, <span class="hlFld-ContribAuthor ">Lianne H. E.  Wieske</span>, <span class="hlFld-ContribAuthor ">Máté  Erdélyi</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Solution Conformations Explain the Chameleonic Behaviour of Macrocyclic Drugs. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (23)
                                     , 5231-5244. <a href="https://doi.org/10.1002/chem.201905599" title="DOI URL">https://doi.org/10.1002/chem.201905599</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201905599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201905599%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DSolution%252BConformations%252BExplain%252Bthe%252BChameleonic%252BBehaviour%252Bof%252BMacrocyclic%252BDrugs%26aulast%3DDanelius%26aufirst%3DEmma%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D23%26spage%3D5231%26epage%3D5244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Ermondi</span>, <span class="hlFld-ContribAuthor ">Maura  Vallaro</span>, <span class="hlFld-ContribAuthor ">Gilles  Goetz</span>, <span class="hlFld-ContribAuthor ">Marina  Shalaeva</span>, <span class="hlFld-ContribAuthor ">Giulia  Caron</span>. </span><span class="cited-content_cbyCitation_article-title">Updating the portfolio of physicochemical descriptors related to permeability in the beyond the rule of 5 chemical space. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2020,</strong> <em>146 </em>, 105274. <a href="https://doi.org/10.1016/j.ejps.2020.105274" title="DOI URL">https://doi.org/10.1016/j.ejps.2020.105274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2020.105274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2020.105274%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DUpdating%252Bthe%252Bportfolio%252Bof%252Bphysicochemical%252Bdescriptors%252Brelated%252Bto%252Bpermeability%252Bin%252Bthe%252Bbeyond%252Bthe%252Brule%252Bof%252B5%252Bchemical%252Bspace%26aulast%3DErmondi%26aufirst%3DGiuseppe%26date%3D2020%26volume%3D146%26spage%3D105274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saumitra  Sengupta</span>, <span class="hlFld-ContribAuthor ">Goverdhan  Mehta</span>. </span><span class="cited-content_cbyCitation_article-title">Macrocyclization
              via
              C–H functionalization: a new paradigm in macrocycle synthesis. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (10)
                                     , 1851-1876. <a href="https://doi.org/10.1039/C9OB02765C" title="DOI URL">https://doi.org/10.1039/C9OB02765C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB02765C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB02765C%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DMacrocyclization%252Bvia%252BC%2525E2%252580%252593H%252Bfunctionalization%25253A%252Ba%252Bnew%252Bparadigm%252Bin%252Bmacrocycle%252Bsynthesis%26aulast%3DSengupta%26aufirst%3DSaumitra%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D10%26spage%3D1851%26epage%3D1876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gustav  Olanders</span>, <span class="hlFld-ContribAuthor ">Hiba  Alogheli</span>, <span class="hlFld-ContribAuthor ">Peter  Brandt</span>, <span class="hlFld-ContribAuthor ">Anders  Karlén</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational analysis of macrocycles: comparing general and specialized methods. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (3)
                                     , 231-252. <a href="https://doi.org/10.1007/s10822-020-00277-2" title="DOI URL">https://doi.org/10.1007/s10822-020-00277-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00277-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00277-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DConformational%252Banalysis%252Bof%252Bmacrocycles%25253A%252Bcomparing%252Bgeneral%252Band%252Bspecialized%252Bmethods%26aulast%3DOlanders%26aufirst%3DGustav%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D3%26spage%3D231%26epage%3D252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenzhan  Yang</span>, <span class="hlFld-ContribAuthor ">Prajakta  Gadgil</span>, <span class="hlFld-ContribAuthor ">Venkata R.  Krishnamurthy</span>, <span class="hlFld-ContribAuthor ">Margaret  Landis</span>, <span class="hlFld-ContribAuthor ">Pankajini  Mallick</span>, <span class="hlFld-ContribAuthor ">Dipal  Patel</span>, <span class="hlFld-ContribAuthor ">Phenil J.  Patel</span>, <span class="hlFld-ContribAuthor ">Darren L.  Reid</span>, <span class="hlFld-ContribAuthor ">Manuel  Sanchez-Felix</span>. </span><span class="cited-content_cbyCitation_article-title">The Evolving Druggability and Developability Space: Chemically Modified New Modalities and Emerging Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">The AAPS Journal</span><span> <strong>2020,</strong> <em>22 </em>
                                    (2)
                                     <a href="https://doi.org/10.1208/s12248-019-0402-2" title="DOI URL">https://doi.org/10.1208/s12248-019-0402-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1208/s12248-019-0402-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1208%2Fs12248-019-0402-2%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520AAPS%2520Journal%26atitle%3DThe%252BEvolving%252BDruggability%252Band%252BDevelopability%252BSpace%25253A%252BChemically%252BModified%252BNew%252BModalities%252Band%252BEmerging%252BSmall%252BMolecules%26aulast%3DYang%26aufirst%3DWenzhan%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohit  Tyagi</span>, <span class="hlFld-ContribAuthor ">Fabio  Begnini</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Bradley C.  Doak</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Syntheses Beyond the Rule of 5. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (1)
                                     , 49-88. <a href="https://doi.org/10.1002/chem.201902716" title="DOI URL">https://doi.org/10.1002/chem.201902716</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201902716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201902716%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DDrug%252BSyntheses%252BBeyond%252Bthe%252BRule%252Bof%252B5%26aulast%3DTyagi%26aufirst%3DMohit%26date%3D2020%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D49%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Facs.jmedchem.5b01286&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01286%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=166&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Classification of 475 drugs and clinical candidates that have MW > 500 Da into extended rule of 5 (blue) and beyond rule of 5 (green) chemical space based on calculated physicochemical properties. (b) Development pipeline by chemical space and chemical class showing peptides/peptidomimetics (green), natural products and derivatives (blue), and de novo designed drugs and clinical candidates (red) by phase. Orals are in dark and parenterals in light colors, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Creation of target class preference data sets. The data set compiled by Rask-Andersen et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> was filtered by all the rule of 5 (Ro5) guidelines, and the primary target reported in the literature was selected for each drug to give the Ro5 data set. The primary targets reported in the literature for the full set of extended Ro5 (eRo5) and beyond Ro5 (bRo5) drugs and clinical candidates were classified using the same taxonomy. (b) Proportion of Ro5 drugs (red), eRo5 (blue), and bRo5 (green) drugs and clinical candidates modulating the indicated target classes. The proportion of compounds in each of the three chemical space data sets that modulate a specific target class (the number of compounds that modulate that target class divided by the total number in the data set, <i>N</i>) is shown by the vertical bars. Proportions were calculated to show differences in target preferences as the number of compounds differ significantly between the three data sets. Compounds that are in phase (P) I, II, or III or approved (App.) are shown by increasingly darker color shadings of the data sets. Targets are arranged by Ro5 target class preference from highest (left) to lowest (right). Alternative plots that show only approved, clinical, or orally bioavailable dugs and clinical candidates in eRo5 and bRo5 space, as well as the exact number of compounds in each category, are included in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cross-referencing the extended Ro5, beyond Ro5 data sets, and a representative set of ChEMBL<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Ro5 drugs with the Protein Data Bank (PDB) and filtering by quality constraints gave three data sets of relevant Ro5, eRo5, and bRo5 drug–target structures. To avoid bias toward highly explored drug classes, further filtering to remove redundant drug class–target structures gave unbiased, nonredundant data sets which contain only one compound per drug–target class (e.g., one erythronolide, one azole anti-infective, etc.). The number of compounds approved (App.) and the number in phases (P) III, II, and I development are shown. Results of the analysis of the complete data sets can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a> for comparison, while the nonredundant data sets are analyzed within the paper as well as in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Distribution of binding site shapes, (b) ligand normalized principle moment of inertia (nPMI) shape plot, (c) buried ligand surface area (SA, Å<sup>2</sup>) as a function of total ligand SA (Å<sup>2</sup>), (d) box plot of buried ligand SA (Å<sup>2</sup>), and (e) box plot of proportion of buried ligand SA. Each panel presents data for rule of 5 drugs (Ro5, red), extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates in complex with their respective targets. Protein–protein interaction (PPI) interface SA data were extracted from Luo et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and reanalyzed. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Proportion of nonpolar heavy atoms (carbon and sulfur) to the total number of heavy atoms for the ligand binding site interface, the overall ligand, and the target binding site interface. The distributions of the number of (b) hydrogen bond acceptor (HBA) atom interactions and (c) hydrogen bond donor (HBD) atom interactions of ligands with their targets. Each panel presents data for rule of 5 (Ro5, red) drugs, extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates in complexes with their respective targets. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Affinities and (b) ligand efficiencies (LE, kcal/(mol·HAC)) for rule of 5 (Ro5, red), extended Ro5 (eRo5, blue), and beyond Ro5 (bRo5, green) drugs and clinical candidates. (c) Relationship between LE and binding site shape. As a rule of thumb, Ro5 drug candidates are optimized to have 10 nM affinities, corresponding to LEs of 0.30 for a compound with a molecular weight of 500 Da (heavy atom count, HAC ≈ 36). This “guideline” for optimization is marked with a gray line in (b) and (c). Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes; and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Chemical structures of drugs and clinical candidates in bRo5 space that bind to flat and groove-shaped binding sites on targets. Compounds are grouped by therapeutic indication and their status, route of administration and clinically relevant targets are given below each structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Distribution of parenterals (triangles) and orals (circles) for macrocycles (right) in beyond Ro5 (bRo5 MC, green) and extended Ro5 space (eRo5 MC, orange), as well as nonmacrocycles (left) in the two chemical spaces (bRo5 non-MC, red, and eRo5 non-MC, blue). The dashed black box shows the eRo5 space limits for ClogP and MW. (b) Comparison of average root-mean-square deviation (rmsd, Å) values of the representative conformers of bRo5 macrocyclic drugs (<i>N</i> = 8, mean number of representative conformers of 3.5) and bRo5 nonmacrocyclic drugs (<i>N</i> = 4, mean number of representative conformers of 2.5). For the macrocycles, rmsd values for core atoms (atoms in the macrocycle ring), core plus periphery atoms (the core atoms plus all single heavy atoms attached to the core), as well as all atoms are compared. Box plots show minimum and maximum values as whiskers; the 25th, 50th, and 75th percentiles as boxes, and mean values as crosses. Horizontal lines indicate unpaired and unequal variance <i>t</i> test of compared data sets with their <i>p</i>-values. (c, d) Chemical structure, all observed crystal structures, and the most representative structure colored by the circular standard deviation of each bond of erythromycin A and ritonavir, respectively. For erythromycin A core atoms are in purple, periphery atoms in orange, and side chain atoms in black. The superimpositions show all experimentally determined structures from the Cambridge Structural Database and Protein Databank for the two drugs with heavy atoms colored by chemical element (green for carbon, blue for nitrogen, red for oxygen, and yellow for sulfur). Flexible bonds giving rise to different conformers of each drug were identified from the circular standard deviation of the dihedral angles and are color-coded from white (0) to red (1) representing rigid to flexible bonds, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/medium/jm-2015-01286d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Affinities, (b) ligand efficiencies (LE), (c) binding site shapes, and (d) normalized principle moment of inertia (nPMI) plot of bioactive conformations of drugs for beyond Ro5 macrocyclic (bRo5 MC, green) and beyond Ro5 nonmacrocyclic (bRo5 non-MC, red) drugs and targets. As a rule of thumb, Ro5 drug candidates are optimized to have 10 nM affinities, corresponding to a LE of 0.30 for a compound with a molecular weight of 500 Da (heavy atom count, HAC ≈ 36). This “guideline” for optimization is marked with a gray line in (b). Box plots show minimum and maximum values as whiskers; the 25th, , and 75th percentiles as boxes; and mean values as crosses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01286/20160318/images/large/jm-2015-01286d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01286&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Trends in the exploitation of novel drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">579</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span><span class="refDoi"> DOI: 10.1038/nrd3478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21804595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=579-590&author=M.+Rask-Andersenauthor=M.+S.+Almenauthor=H.+B.+Schioth&title=Trends+in+the+exploitation+of+novel+drug+targets&doi=10.1038%2Fnrd3478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in the exploitation of novel drug targets</span></div><div class="casAuthors">Rask-Andersen, Mathias; Almen, Markus Saellman; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery and exploitation of new drug targets is a key focus for both the pharmaceutical industry and academic biomedical research.  To provide an insight into trends in the exploitation of new drug targets, we have analyzed the drugs that were approved by the US Food and Drug Administration during the past three decades and examd. the interactions of these drugs with therapeutic targets that are encoded by the human genome, using the DrugBank database and extensive manual curation.  We have identified 435 effect-mediating drug targets in the human genome, which are modulated by 989 unique drugs, through 2,242 drug-target interactions.  We also analyze trends in the introduction of drugs that modulate previously unexploited targets, and discuss the network pharmacol. of the drugs in our data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo19FXXFB-PsLVg90H21EOLACvtfcHk0lhtxO7jJoP7ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D&md5=9255431a28f2c07759b0f36975711f84</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3478%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DAlmen%26aufirst%3DM.%2BS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DTrends%2520in%2520the%2520exploitation%2520of%2520novel%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D579%26epage%3D590%26doi%3D10.1038%2Fnrd3478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bunnage, M. E.</span><span> </span><span class="NLM_article-title">Getting pharmaceutical R&D back on target</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/nchembio.581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchembio.581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21587251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtF2lur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=335-339&author=M.+E.+Bunnage&title=Getting+pharmaceutical+R%26D+back+on+target&doi=10.1038%2Fnchembio.581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Getting pharmaceutical R&D back on target</span></div><div class="casAuthors">Bunnage, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">335-339</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is in a period of crisis due to the low no. of new drug approvals relative to the high levels of R&D investment.  It is argued here that improving the quality of target selection is the single most important factor to transform industry productivity and bring innovative new medicines to patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyfBkdRkjR8LVg90H21EOLACvtfcHk0lhtxO7jJoP7ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtF2lur4%253D&md5=458abeefd10325afc74b5ced717d3b40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.581%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26atitle%3DGetting%2520pharmaceutical%2520R%2526D%2520back%2520on%2520target%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D335%26epage%3D339%26doi%3D10.1038%2Fnchembio.581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kinch, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoyer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patridge, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plummer, M.</span><span> </span><span class="NLM_article-title">Target selection for FDA-approved medicines</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">784</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.drudis.2014.11.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=784-789&author=M.+S.+Kinchauthor=D.+Hoyerauthor=E.+Patridgeauthor=M.+Plummer&title=Target+selection+for+FDA-approved+medicines&doi=10.1016%2Fj.drudis.2014.11.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DKinch%26aufirst%3DM.%2BS.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DPatridge%26aufirst%3DE.%26aulast%3DPlummer%26aufirst%3DM.%26atitle%3DTarget%2520selection%2520for%2520FDA-approved%2520medicines%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D784%26epage%3D789%26doi%3D10.1016%2Fj.drudis.2014.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Scannell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanckley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boldon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warrington, B.</span><span> </span><span class="NLM_article-title">Diagnosing the decline in pharmaceutical R&D efficiency</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1038/nrd3681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22378269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=191-200&author=J.+W.+Scannellauthor=A.+Blanckleyauthor=H.+Boldonauthor=B.+Warrington&title=Diagnosing+the+decline+in+pharmaceutical+R%26D+efficiency&doi=10.1038%2Fnrd3681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Diagnosing the decline in pharmaceutical R&D efficiency</span></div><div class="casAuthors">Scannell, Jack W.; Blanckley, Alex; Boldon, Helen; Warrington, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-200</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 60 years have seen huge advances in many of the scientific, technol. and managerial factors that should tend to raise the efficiency of com. drug research and development (RD).  Yet the no. of new drugs approved per billion US dollars spent on RD has halved roughly every 9 years since 1950, falling around 80-fold in inflation-adjusted terms.  There have been many proposed solns. to the problem of declining RD efficiency.  However, their apparent lack of impact so far and the contrast between improving inputs and declining output in terms of the no. of new drugs make it sensible to ask whether the underlying problems have been correctly diagnosed.  Here, we discuss four factors that we consider to be primary causes, which we call the 'better than the Beatles' problem; the 'cautious regulator' problem; the 'throw money at it' tendency; and the 'basic research-brute force' bias.  Our aim is to provoke a more systematic anal. of the causes of the decline in RD efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5e_81DC6r8LVg90H21EOLACvtfcHk0ljLhX8_r1E6NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D&md5=db3e4548260e55f47a9964e8d1b216fd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3681%26sid%3Dliteratum%253Aachs%26aulast%3DScannell%26aufirst%3DJ.%2BW.%26aulast%3DBlanckley%26aufirst%3DA.%26aulast%3DBoldon%26aufirst%3DH.%26aulast%3DWarrington%26aufirst%3DB.%26atitle%3DDiagnosing%2520the%2520decline%2520in%2520pharmaceutical%2520R%2526D%2520efficiency%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D191%26epage%3D200%26doi%3D10.1038%2Fnrd3681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Hay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craighead, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Economides, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenthal, J.</span><span> </span><span class="NLM_article-title">Clinical development success rates for investigational drugs</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1038/nbt.2786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnbt.2786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24406927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvFyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=40-51&author=M.+Hayauthor=D.+W.+Thomasauthor=J.+L.+Craigheadauthor=C.+Economidesauthor=J.+Rosenthal&title=Clinical+development+success+rates+for+investigational+drugs&doi=10.1038%2Fnbt.2786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical development success rates for investigational drugs</span></div><div class="casAuthors">Hay, Michael; Thomas, David W.; Craighead, John L.; Economides, Celia; Rosenthal, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-51</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The most comprehensive survey of clin. success rates across the drug industry to date shows productivity may be even lower than previous ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdqyRgsoFYDbVg90H21EOLACvtfcHk0ljLhX8_r1E6NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvFyjsA%253D%253D&md5=87a63ec8d8bb213b3d42519395725038</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2786%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DD.%2BW.%26aulast%3DCraighead%26aufirst%3DJ.%2BL.%26aulast%3DEconomides%26aufirst%3DC.%26aulast%3DRosenthal%26aufirst%3DJ.%26atitle%3DClinical%2520development%2520success%2520rates%2520for%2520investigational%2520drugs%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D40%26epage%3D51%26doi%3D10.1038%2Fnbt.2786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Cook, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexander, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">March, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satterthwaite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pangalos, M. N.</span><span> </span><span class="NLM_article-title">Lessons learned from the fate of AstraZeneca’s drug pipeline: a five-dimensional framework</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1038/nrd4309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd4309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24833294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=419-431&author=D.+Cookauthor=D.+Brownauthor=R.+Alexanderauthor=R.+Marchauthor=P.+Morganauthor=G.+Satterthwaiteauthor=M.+N.+Pangalos&title=Lessons+learned+from+the+fate+of+AstraZeneca%E2%80%99s+drug+pipeline%3A+a+five-dimensional+framework&doi=10.1038%2Fnrd4309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the fate of AstraZeneca's drug pipeline: a five-dimensional framework</span></div><div class="casAuthors">Cook, David; Brown, Dearg; Alexander, Robert; March, Ruth; Morgan, Paul; Satterthwaite, Gemma; Pangalos, Menelas N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">419-431</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Maintaining research and development (R&D) productivity at a sustainable level is one of the main challenges currently facing the pharmaceutical industry.  In this article, we discuss the results of a comprehensive longitudinal review of AstraZeneca's small-mol. drug projects from 2005 to 2010.  The anal. allowed us to establish a framework based on the five most important tech. determinants of project success and pipeline quality, which we describe as the five 'R's: the right target, the right patient, the right tissue, the right safety and the right com. potential.  A sixth factor - the right culture - is also crucial in encouraging effective decision-making based on these tech. determinants.  AstraZeneca is currently applying this framework to guide its R&D teams, and although it is too early to demonstrate whether this has improved the company's R&D productivity, we present our data and anal. here in the hope that it may assist the industry overall in addressing this key challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4yhPFZOUNO7Vg90H21EOLACvtfcHk0ljLhX8_r1E6NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyqtrw%253D&md5=746a86c7223f170f7e6c4f9817760e5b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4309%26sid%3Dliteratum%253Aachs%26aulast%3DCook%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DMarch%26aufirst%3DR.%26aulast%3DMorgan%26aufirst%3DP.%26aulast%3DSatterthwaite%26aufirst%3DG.%26aulast%3DPangalos%26aufirst%3DM.%2BN.%26atitle%3DLessons%2520learned%2520from%2520the%2520fate%2520of%2520AstraZeneca%25E2%2580%2599s%2520drug%2520pipeline%253A%2520a%2520five-dimensional%2520framework%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D419%26epage%3D431%26doi%3D10.1038%2Fnrd4309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">International Human Genome Sequencing Consortium.</span><span> </span><span class="NLM_article-title">Finishing the euchromatic sequence of the human genome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">431</span><span class="NLM_x">, </span> <span class="NLM_fpage">931</span><span class="NLM_x">–</span> <span class="NLM_lpage">945</span><span class="refDoi"> DOI: 10.1038/nature03001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature03001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=15496913" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=431&publication_year=2004&pages=931-945&author=International+Human+Genome+Sequencing+Consortium.&title=Finishing+the+euchromatic+sequence+of+the+human+genome&doi=10.1038%2Fnature03001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature03001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03001%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DFinishing%2520the%2520euchromatic%2520sequence%2520of%2520the%2520human%2520genome%26jtitle%3DNature%26date%3D2004%26volume%3D431%26spage%3D931%26epage%3D945%26doi%3D10.1038%2Fnature03001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nirujogi, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manda, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaerkady, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madugundu, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelkar, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isserlin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthusamy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leal-Rojas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasrabuddhe, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Advani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renuse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selvan, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nanjappa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radhakrishnan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbannayya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raju, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreenivasamurthy, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathe, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Datta, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbannayya, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yelamanchi, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopalan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murthy, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syed, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mudgal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freed, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahari, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahadevan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satishchandra, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirdeshmukh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maitra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drake, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halushka, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacobuzio-Donahue, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span> </span><span class="NLM_article-title">A draft map of the human proteome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span><span class="refDoi"> DOI: 10.1038/nature13302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature13302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24870542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=575-581&author=M.+S.+Kimauthor=S.+M.+Pintoauthor=D.+Getnetauthor=R.+S.+Nirujogiauthor=S.+S.+Mandaauthor=R.+Chaerkadyauthor=A.+K.+Madugunduauthor=D.+S.+Kelkarauthor=R.+Isserlinauthor=S.+Jainauthor=J.+K.+Thomasauthor=B.+Muthusamyauthor=P.+Leal-Rojasauthor=P.+Kumarauthor=N.+A.+Sahasrabuddheauthor=L.+Balakrishnanauthor=J.+Advaniauthor=B.+Georgeauthor=S.+Renuseauthor=L.+D.+Selvanauthor=A.+H.+Patilauthor=V.+Nanjappaauthor=A.+Radhakrishnanauthor=S.+Prasadauthor=T.+Subbannayyaauthor=R.+Rajuauthor=M.+Kumarauthor=S.+K.+Sreenivasamurthyauthor=A.+Marimuthuauthor=G.+J.+Satheauthor=S.+Chavanauthor=K.+K.+Dattaauthor=Y.+Subbannayyaauthor=A.+Sahuauthor=S.+D.+Yelamanchiauthor=S.+Jayaramauthor=P.+Rajagopalanauthor=J.+Sharmaauthor=K.+R.+Murthyauthor=N.+Syedauthor=R.+Goelauthor=A.+A.+Khanauthor=S.+Ahmadauthor=G.+Deyauthor=K.+Mudgalauthor=A.+Chatterjeeauthor=T.+C.+Huangauthor=J.+Zhongauthor=X.+Wuauthor=P.+G.+Shawauthor=D.+Freedauthor=M.+S.+Zahariauthor=K.+K.+Mukherjeeauthor=S.+Shankarauthor=A.+Mahadevanauthor=H.+Lamauthor=C.+J.+Mitchellauthor=S.+K.+Shankarauthor=P.+Satishchandraauthor=J.+T.+Schroederauthor=R.+Sirdeshmukhauthor=A.+Maitraauthor=S.+D.+Leachauthor=C.+G.+Drakeauthor=M.+K.+Halushkaauthor=T.+S.+Prasadauthor=R.+H.+Hrubanauthor=C.+L.+Kerrauthor=G.+D.+Baderauthor=C.+A.+Iacobuzio-Donahueauthor=H.+Gowdaauthor=A.+Pandey&title=A+draft+map+of+the+human+proteome&doi=10.1038%2Fnature13302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A draft map of the human proteome</span></div><div class="casAuthors">Kim, Min-Sik; Pinto, Sneha M.; Getnet, Derese; Nirujogi, Raja Sekhar; Manda, Srikanth S.; Chaerkady, Raghothama; Madugundu, Anil K.; Kelkar, Dhanashree S.; Isserlin, Ruth; Jain, Shobhit; Thomas, Joji K.; Muthusamy, Babylakshmi; Leal-Rojas, Pamela; Kumar, Praveen; Sahasrabuddhe, Nandini A.; Balakrishnan, Lavanya; Advani, Jayshree; George, Bijesh; Renuse, Santosh; Selvan, Lakshmi Dhevi N.; Patil, Arun H.; Nanjappa, Vishalakshi; Radhakrishnan, Aneesha; Prasad, Samarjeet; Subbannayya, Tejaswini; Raju, Rajesh; Kumar, Manish; Sreenivasamurthy, Sreelakshmi K.; Marimuthu, Arivusudar; Sathe, Gajanan J.; Chavan, Sandip; Datta, Keshava K.; Subbannayya, Yashwanth; Sahu, Apeksha; Yelamanchi, Soujanya D.; Jayaram, Savita; Rajagopalan, Pavithra; Sharma, Jyoti; Murthy, Krishna R.; Syed, Nazia; Goel, Renu; Khan, Aafaque A.; Ahmad, Sartaj; Dey, Gourav; Mudgal, Keshav; Chatterjee, Aditi; Huang, Tai-Chung; Zhong, Jun; Wu, Xinyan; Shaw, Patrick G.; Freed, Donald; Zahari, Muhammad S.; Mukherjee, Kanchan K.; Shankar, Subramanian; Mahadevan, Anita; Lam, Henry; Mitchell, Christopher J.; Shankar, Susarla Krishna; Satishchandra, Parthasarathy; Schroeder, John T.; Sirdeshmukh, Ravi; Maitra, Anirban; Leach, Steven D.; Drake, Charles G.; Halushka, Marc K.; Prasad, T. S. Keshava; Hruban, Ralph H.; Kerr, Candace L.; Bader, Gary D.; Iacobuzio-Donahue, Christine A.; Gowda, Harsha; Pandey, Akhilesh</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7502</span>),
    <span class="NLM_cas:pages">575-581</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of human genome sequence has transformed biomedical research over the past decade.  However, an equiv. map for the human proteome with direct measurements of proteins and peptides does not exist yet.  Here we present a draft map of the human proteome using high-resoln. Fourier-transform mass spectrometry.  In-depth proteomic profiling of 30 histol. normal human samples, including 17 adult tissues, 7 fetal tissues and 6 purified primary hematopoietic cells, resulted in identification of proteins encoded by 17,294 genes accounting for approx. 84% of the total annotated protein-coding genes in humans.  A unique and comprehensive strategy for proteogenomic anal. enabled us to discover a no. of novel protein-coding regions, which includes translated pseudogenes, non-coding RNAs and upstream open reading frames.  This large human proteome catalog (available as an interactive web-based resource at http://www.humanproteomemap.org) will complement available human genome and transcriptome data to accelerate biomedical research in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNW2lqYhs8ULVg90H21EOLACvtfcHk0liyFvC0jyw-Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCrtrc%253D&md5=facf459fc3d9c8258fba1c1e94a378c6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature13302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13302%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DPinto%26aufirst%3DS.%2BM.%26aulast%3DGetnet%26aufirst%3DD.%26aulast%3DNirujogi%26aufirst%3DR.%2BS.%26aulast%3DManda%26aufirst%3DS.%2BS.%26aulast%3DChaerkady%26aufirst%3DR.%26aulast%3DMadugundu%26aufirst%3DA.%2BK.%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DIsserlin%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DJ.%2BK.%26aulast%3DMuthusamy%26aufirst%3DB.%26aulast%3DLeal-Rojas%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DSahasrabuddhe%26aufirst%3DN.%2BA.%26aulast%3DBalakrishnan%26aufirst%3DL.%26aulast%3DAdvani%26aufirst%3DJ.%26aulast%3DGeorge%26aufirst%3DB.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DSelvan%26aufirst%3DL.%2BD.%26aulast%3DPatil%26aufirst%3DA.%2BH.%26aulast%3DNanjappa%26aufirst%3DV.%26aulast%3DRadhakrishnan%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DSubbannayya%26aufirst%3DT.%26aulast%3DRaju%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DM.%26aulast%3DSreenivasamurthy%26aufirst%3DS.%2BK.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DSathe%26aufirst%3DG.%2BJ.%26aulast%3DChavan%26aufirst%3DS.%26aulast%3DDatta%26aufirst%3DK.%2BK.%26aulast%3DSubbannayya%26aufirst%3DY.%26aulast%3DSahu%26aufirst%3DA.%26aulast%3DYelamanchi%26aufirst%3DS.%2BD.%26aulast%3DJayaram%26aufirst%3DS.%26aulast%3DRajagopalan%26aufirst%3DP.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DMurthy%26aufirst%3DK.%2BR.%26aulast%3DSyed%26aufirst%3DN.%26aulast%3DGoel%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BA.%26aulast%3DAhmad%26aufirst%3DS.%26aulast%3DDey%26aufirst%3DG.%26aulast%3DMudgal%26aufirst%3DK.%26aulast%3DChatterjee%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DT.%2BC.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DShaw%26aufirst%3DP.%2BG.%26aulast%3DFreed%26aufirst%3DD.%26aulast%3DZahari%26aufirst%3DM.%2BS.%26aulast%3DMukherjee%26aufirst%3DK.%2BK.%26aulast%3DShankar%26aufirst%3DS.%26aulast%3DMahadevan%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DH.%26aulast%3DMitchell%26aufirst%3DC.%2BJ.%26aulast%3DShankar%26aufirst%3DS.%2BK.%26aulast%3DSatishchandra%26aufirst%3DP.%26aulast%3DSchroeder%26aufirst%3DJ.%2BT.%26aulast%3DSirdeshmukh%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DS.%2BD.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DHalushka%26aufirst%3DM.%2BK.%26aulast%3DPrasad%26aufirst%3DT.%2BS.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DKerr%26aufirst%3DC.%2BL.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DIacobuzio-Donahue%26aufirst%3DC.%2BA.%26aulast%3DGowda%26aufirst%3DH.%26aulast%3DPandey%26aufirst%3DA.%26atitle%3DA%2520draft%2520map%2520of%2520the%2520human%2520proteome%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D575%26epage%3D581%26doi%3D10.1038%2Fnature13302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Wilhelm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahne, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghaddas Gholami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberenz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziegler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butzmann, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gessulat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marx, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemeer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnatbaum, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wenschuh, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mollenhauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slotta-Huspenina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boese, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstmair, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faerber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span> </span><span class="NLM_article-title">Mass-spectrometry-based draft of the human proteome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">582</span><span class="NLM_x">–</span> <span class="NLM_lpage">587</span><span class="refDoi"> DOI: 10.1038/nature13319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature13319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24870543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=582-587&author=M.+Wilhelmauthor=J.+Schleglauthor=H.+Hahneauthor=A.+Moghaddas+Gholamiauthor=M.+Lieberenzauthor=M.+M.+Savitskiauthor=E.+Zieglerauthor=L.+Butzmannauthor=S.+Gessulatauthor=H.+Marxauthor=T.+Mathiesonauthor=S.+Lemeerauthor=K.+Schnatbaumauthor=U.+Reimerauthor=H.+Wenschuhauthor=M.+Mollenhauerauthor=J.+Slotta-Huspeninaauthor=J.+H.+Boeseauthor=M.+Bantscheffauthor=A.+Gerstmairauthor=F.+Faerberauthor=B.+Kuster&title=Mass-spectrometry-based+draft+of+the+human+proteome&doi=10.1038%2Fnature13319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mass-spectrometry-based draft of the human proteome</span></div><div class="casAuthors">Wilhelm, Mathias; Schlegl, Judith; Hahne, Hannes; Gholami, Amin Moghaddas; Lieberenz, Marcus; Savitski, Mikhail M.; Ziegler, Emanuel; Butzmann, Lars; Gessulat, Siegfried; Marx, Harald; Mathieson, Toby; Lemeer, Simone; Schnatbaum, Karsten; Reimer, Ulf; Wenschuh, Holger; Mollenhauer, Martin; Slotta-Huspenina, Julia; Boese, Joos-Hendrik; Bantscheff, Marcus; Gerstmair, Anja; Faerber, Franz; Kuster, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7502</span>),
    <span class="NLM_cas:pages">582-587</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteomes are characterized by large protein-abundance differences, cell-type- and time-dependent expression patterns and post-translational modifications, all of which carry biol. information that is not accessible by genomics or transcriptomics.  Here we present a mass-spectrometry-based draft of the human proteome and a public, high-performance, in-memory database for real-time anal. of terabytes of big data, called ProteomicsDB.  The information assembled from human tissues, cell lines and body fluids enabled estn. of the size of the protein-coding genome, and identified organ-specific proteins and a large no. of translated lincRNAs (long intergenic non-coding RNAs).  Anal. of mRNA and protein-expression profiles of human tissues revealed conserved control of protein abundance, and integration of drug-sensitivity data enabled the identification of proteins predicting resistance or sensitivity.  The proteome profiles also hold considerable promise for analyzing the compn. and stoichiometry of protein complexes.  ProteomicsDB thus enables navigation of proteomes, provides biol. insight and fosters the development of proteomic technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK5rb2mey2G7Vg90H21EOLACvtfcHk0liyFvC0jyw-Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCrt7k%253D&md5=b9e4b32b349b92160996c860de293191</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature13319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13319%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DMoghaddas%2BGholami%26aufirst%3DA.%26aulast%3DLieberenz%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DZiegler%26aufirst%3DE.%26aulast%3DButzmann%26aufirst%3DL.%26aulast%3DGessulat%26aufirst%3DS.%26aulast%3DMarx%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DLemeer%26aufirst%3DS.%26aulast%3DSchnatbaum%26aufirst%3DK.%26aulast%3DReimer%26aufirst%3DU.%26aulast%3DWenschuh%26aufirst%3DH.%26aulast%3DMollenhauer%26aufirst%3DM.%26aulast%3DSlotta-Huspenina%26aufirst%3DJ.%26aulast%3DBoese%26aufirst%3DJ.%2BH.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGerstmair%26aufirst%3DA.%26aulast%3DFaerber%26aufirst%3DF.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DMass-spectrometry-based%2520draft%2520of%2520the%2520human%2520proteome%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D582%26epage%3D587%26doi%3D10.1038%2Fnature13319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">The druggable genome: evaluation of drug targets in clinical trials suggests major shifts in molecular class and indication</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-011613-135943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1146%2Fannurev-pharmtox-011613-135943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24016212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=9-26&author=M.+Rask-Andersenauthor=S.+Masuramauthor=H.+B.+Schioth&title=The+druggable+genome%3A+evaluation+of+drug+targets+in+clinical+trials+suggests+major+shifts+in+molecular+class+and+indication&doi=10.1146%2Fannurev-pharmtox-011613-135943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The druggable genome: evaluation of drug targets in clinical trials suggests major shifts in molecular class and indication</span></div><div class="casAuthors">Rask-Andersen, Mathias; Masuram, Surendar; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-26</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The largest innovations within pharmaceutical development come through new compds. that have unique and novel modes of action.  These innovations commonly involve expanding the protein space targeted by pharmaceutical agents.  At present, information about drugs and drug targets is available online via public databases such as DrugBank and the Therapeutic Targets Database.  However, this information is biased, understandably so, toward established drugs and drug-target interactions.  To gain a better overview of the drug-targeted portion of the human proteome and the directions of current drug development, we developed a data set of clin. trial drug-target interactions based on CenterWatch's Drugs in Clin. Trials Database, one of the largest databases of its kind.  Our curation identified 475 potentially novel clin. trial drug targets.  This review aims to identify trends in drug development based on the potentially novel targets currently being explored in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrACyvG8vYfDLVg90H21EOLACvtfcHk0ljVh2niiBsSQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVSqsbk%253D&md5=2a31db63098fe4a01c16f6179f3b88e3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-011613-135943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-011613-135943%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DMasuram%26aufirst%3DS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DThe%2520druggable%2520genome%253A%2520evaluation%2520of%2520drug%2520targets%2520in%2520clinical%2520trials%2520suggests%2520major%2520shifts%2520in%2520molecular%2520class%2520and%2520indication%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2014%26volume%3D54%26spage%3D9%26epage%3D26%26doi%3D10.1146%2Fannurev-pharmtox-011613-135943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0ljVh2niiBsSQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Venkatesan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rual, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vazquez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stelzl, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmens, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirozane-Kishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zenkner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goh, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yildirim, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonis, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinzmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebreab, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahalie, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cevik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Smet, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolyar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinayagam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szeto, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borick, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dricot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klitgord, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lalowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cusick, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavernier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wanker, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barabasi, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidal, M.</span><span> </span><span class="NLM_article-title">An empirical framework for binary interactome mapping</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1038/nmeth.1280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnmeth.1280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=19060904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=83-90&author=K.+Venkatesanauthor=J.+F.+Rualauthor=A.+Vazquezauthor=U.+Stelzlauthor=I.+Lemmensauthor=T.+Hirozane-Kishikawaauthor=T.+Haoauthor=M.+Zenknerauthor=X.+Xinauthor=K.+I.+Gohauthor=M.+A.+Yildirimauthor=N.+Simonisauthor=K.+Heinzmannauthor=F.+Gebreabauthor=J.+M.+Sahalieauthor=S.+Cevikauthor=C.+Simonauthor=A.+S.+de+Smetauthor=E.+Dannauthor=A.+Smolyarauthor=A.+Vinayagamauthor=H.+Yuauthor=D.+Szetoauthor=H.+Borickauthor=A.+Dricotauthor=N.+Klitgordauthor=R.+R.+Murrayauthor=C.+Linauthor=M.+Lalowskiauthor=J.+Timmauthor=K.+Rauauthor=C.+Booneauthor=P.+Braunauthor=M.+E.+Cusickauthor=F.+P.+Rothauthor=D.+E.+Hillauthor=J.+Tavernierauthor=E.+E.+Wankerauthor=A.+L.+Barabasiauthor=M.+Vidal&title=An+empirical+framework+for+binary+interactome+mapping&doi=10.1038%2Fnmeth.1280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An empirical framework for binary interactome mapping</span></div><div class="casAuthors">Venkatesan, Kavitha; Rual, Jean-Francois; Vazquez, Alexei; Stelzl, Ulrich; Lemmens, Irma; Hirozane-Kishikawa, Tomoko; Hao, Tong; Zenkner, Martina; Xin, Xiaofeng; Goh, Kwang-Il; Yildirim, Muhammed A.; Simonis, Nicolas; Heinzmann, Kathrin; Gebreab, Fana; Sahalie, Julie M.; Cevik, Sebiha; Simon, Christophe; de Smet, Anne-Sophie; Dann, Elizabeth; Smolyar, Alex; Vinayagam, Arunachalam; Yu, Haiyuan; Szeto, David; Borick, Heather; Dricot, Amelie; Klitgord, Niels; Murray, Ryan R.; Lin, Chenwei; Lalowski, Maciej; Timm, Jan; Rau, Kirstin; Boone, Charles; Braun, Pascal; Cusick, Michael E.; Roth, Frederick P.; Hill, David E.; Tavernier, Jan; Wanker, Erich E.; Barabasi, Albert-Laszlo; Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-90</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several attempts have been made to systematically map protein-protein interaction, or 'interactome', networks.  However, it remains difficult to assess the quality and coverage of existing data sets.  Here the authors describe a framework that uses an empirically-based approach to rigorously dissect quality parameters of currently available human interactome maps.  The authors' results indicate that high-throughput yeast two-hybrid (HT-Y2H) interactions for human proteins are more precise than literature-curated interactions supported by a single publication, suggesting that HT-Y2H is suitable to map a significant portion of the human interactome.  The authors est. that the human interactome contains ∼130,000 binary interactions, most of which remain to be mapped.  Similar to ests. of DNA sequence data quality and genome size early in the Human Genome Project, ests. of protein interaction data quality and interactome size are crucial to establish the magnitude of the task of comprehensive human interactome mapping and to elucidate a path toward this goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9l4eKnzTSQ7Vg90H21EOLACvtfcHk0ljVh2niiBsSQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7%252FE&md5=7d796aad73a5a62b02880b5d14a9eaa2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1280%26sid%3Dliteratum%253Aachs%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DRual%26aufirst%3DJ.%2BF.%26aulast%3DVazquez%26aufirst%3DA.%26aulast%3DStelzl%26aufirst%3DU.%26aulast%3DLemmens%26aufirst%3DI.%26aulast%3DHirozane-Kishikawa%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DT.%26aulast%3DZenkner%26aufirst%3DM.%26aulast%3DXin%26aufirst%3DX.%26aulast%3DGoh%26aufirst%3DK.%2BI.%26aulast%3DYildirim%26aufirst%3DM.%2BA.%26aulast%3DSimonis%26aufirst%3DN.%26aulast%3DHeinzmann%26aufirst%3DK.%26aulast%3DGebreab%26aufirst%3DF.%26aulast%3DSahalie%26aufirst%3DJ.%2BM.%26aulast%3DCevik%26aufirst%3DS.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3Dde%2BSmet%26aufirst%3DA.%2BS.%26aulast%3DDann%26aufirst%3DE.%26aulast%3DSmolyar%26aufirst%3DA.%26aulast%3DVinayagam%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSzeto%26aufirst%3DD.%26aulast%3DBorick%26aufirst%3DH.%26aulast%3DDricot%26aufirst%3DA.%26aulast%3DKlitgord%26aufirst%3DN.%26aulast%3DMurray%26aufirst%3DR.%2BR.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DLalowski%26aufirst%3DM.%26aulast%3DTimm%26aufirst%3DJ.%26aulast%3DRau%26aufirst%3DK.%26aulast%3DBoone%26aufirst%3DC.%26aulast%3DBraun%26aufirst%3DP.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DRoth%26aufirst%3DF.%2BP.%26aulast%3DHill%26aufirst%3DD.%2BE.%26aulast%3DTavernier%26aufirst%3DJ.%26aulast%3DWanker%26aufirst%3DE.%2BE.%26aulast%3DBarabasi%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DAn%2520empirical%2520framework%2520for%2520binary%2520interactome%2520mapping%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D83%26epage%3D90%26doi%3D10.1038%2Fnmeth.1280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Zhang, Q. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thu, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisikirska, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefebvre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Accili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maniatis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Califano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Structure-based prediction of protein-protein interactions on a genome-wide scale</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">490</span><span class="NLM_x">, </span> <span class="NLM_fpage">556</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1038/nature11503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnature11503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23023127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=556-560&author=Q.+C.+Zhangauthor=D.+Petreyauthor=L.+Dengauthor=L.+Qiangauthor=Y.+Shiauthor=C.+A.+Thuauthor=B.+Bisikirskaauthor=C.+Lefebvreauthor=D.+Acciliauthor=T.+Hunterauthor=T.+Maniatisauthor=A.+Califanoauthor=B.+Honig&title=Structure-based+prediction+of+protein-protein+interactions+on+a+genome-wide+scale&doi=10.1038%2Fnature11503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of protein-protein interactions on a genome-wide scale</span></div><div class="casAuthors">Zhang, Qiangfeng Cliff; Petrey, Donald; Deng, Lei; Qiang, Li; Shi, Yu; Thu, Chan Aye; Bisikirska, Brygida; Lefebvre, Celine; Accili, Domenico; Hunter, Tony; Maniatis, Tom; Califano, Andrea; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">556-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The genome-wide identification of pairs of interacting proteins is an important step in the elucidation of cell regulatory mechanisms.  Much of our present knowledge derives from high-throughput techniques such as the yeast two-hybrid assay and affinity purifn., as well as from manual curation of expts. on individual systems.  A variety of computational approaches based, for example, on sequence homol., gene co-expression and phylogenetic profiles, have also been developed for the genome-wide inference of protein-protein interactions (PPIs).  Yet comparative studies suggest that the development of accurate and complete repertoires of PPIs is still in its early stages.  Here we show that three-dimensional structural information can be used to predict PPIs with an accuracy and coverage that are superior to predictions based on non-structural evidence.  Moreover, an algorithm, termed PrePPI, which combines structural information with other functional clues, is comparable in accuracy to high-throughput expts., yielding over 30,000 high-confidence interactions for yeast and over 300,000 for human.  Exptl. tests of a no. of predictions demonstrate the ability of the PrePPI algorithm to identify unexpected PPIs of considerable biol. interest.  The surprising effectiveness of three-dimensional structural information can be attributed to the use of homol. models combined with the exploitation of both close and remote geometric relationships between proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIaJJWiy7hKLVg90H21EOLACvtfcHk0lhxjNJAGPmopA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtb7P&md5=bb27a4b275f5d9c727859b172c6b6d0f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature11503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11503%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%2BC.%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DQiang%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DThu%26aufirst%3DC.%2BA.%26aulast%3DBisikirska%26aufirst%3DB.%26aulast%3DLefebvre%26aufirst%3DC.%26aulast%3DAccili%26aufirst%3DD.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DManiatis%26aufirst%3DT.%26aulast%3DCalifano%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DStructure-based%2520prediction%2520of%2520protein-protein%2520interactions%2520on%2520a%2520genome-wide%2520scale%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D556%26epage%3D560%26doi%3D10.1038%2Fnature11503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: progressing toward the reality</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1102</span><span class="NLM_x">–</span> <span class="NLM_lpage">1114</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lhxjNJAGPmopA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0lhxjNJAGPmopA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0lj-1ePWXdHPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">DrugPred: A structure-based approach to predict protein druggability developed using an extensive nonredundant data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">2842</span><span class="refDoi"> DOI: 10.1021/ci200266d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200266d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2829-2842&author=A.+Krasowskiauthor=D.+Muthasauthor=A.+Sarkarauthor=S.+Schmittauthor=R.+Brenk&title=DrugPred%3A+A+structure-based+approach+to+predict+protein+druggability+developed+using+an+extensive+nonredundant+data+set&doi=10.1021%2Fci200266d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DrugPred: A Structure-Based Approach To Predict Protein Druggability Developed Using an Extensive Nonredundant Data Set</span></div><div class="casAuthors">Krasowski, Agata; Muthas, Daniel; Sarkar, Aurijit; Schmitt, Stefan; Brenk, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2842</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Judging if a protein is able to bind orally available mols. with high affinity, i.e. if a protein is druggable, is an important step in target assessment.  In order to derive a structure-based method to predict protein druggability, a comprehensive, nonredundant data set contg. crystal structures of 71 druggable and 44 less druggable proteins was compiled by literature search and data mining.  This data set was subsequently used to train a structure-based druggability predictor (DrugPred) using partial least-squares projection to latent structures discriminant anal. (PLS-DA).  DrugPred performed well in discriminating druggable from less druggable binding sites for both internal and external predictions.  The method is robust against conformational changes in the binding site and outperforms previously published methods.  The superior performance of DrugPred is likely due to the size and compn. of the training set which, in contrast to most previously developed methods, only contains cavities that have evolved to bind a natural ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLISVIZ5UnXrVg90H21EOLACvtfcHk0lj-1ePWXdHPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN&md5=a352186705196d6c58b4a590e2848896</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci200266d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200266d%26sid%3Dliteratum%253Aachs%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DDrugPred%253A%2520A%2520structure-based%2520approach%2520to%2520predict%2520protein%2520druggability%2520developed%2520using%2520an%2520extensive%2520nonredundant%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2829%26epage%3D2842%26doi%3D10.1021%2Fci200266d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span> </span><span class="NLM_article-title">Development of a rule-based method for the assessment of protein druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">1038</span><span class="refDoi"> DOI: 10.1021/ci200613b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200613b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1027-1038&author=E.+Perolaauthor=L.+Hermanauthor=J.+Weiss&title=Development+of+a+rule-based+method+for+the+assessment+of+protein+druggability&doi=10.1021%2Fci200613b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Rule-Based Method for the Assessment of Protein Druggability</span></div><div class="casAuthors">Perola, Emanuele; Herman, Lee; Weiss, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1027-1038</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target selection is a crit. step in the majority of modern drug discovery programs.  The viability of a drug target depends on two components: biol. relevance and chem. tractability.  The concept of druggability was introduced to describe the second component, and it is defined as the ability of a target to bind a drug-like mol. with a therapeutically useful level of affinity.  To investigate the rules that govern druggability, we developed an algorithm to isolate and characterize the binding pockets of protein targets.  Using this algorithm, we performed a comparative anal. between the relevant pockets of 60 targets of approved drugs and a diverse set of 440 ligand-binding pockets.  As a result, we defined a preferred property space for druggable pockets based on five key properties (vol., depth, enclosure, percentage of charged residues and hydrophobicity), and we represented it with a set of simple rules.  These rules may be applicable in the future to evaluate the chem. tractability of prospective targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRGoeAUMXUbVg90H21EOLACvtfcHk0lj-1ePWXdHPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D&md5=49a4c07150749f6d09a062365512b8c2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci200613b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200613b%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DHerman%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520a%2520rule-based%2520method%2520for%2520the%2520assessment%2520of%2520protein%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1027%26epage%3D1038%26doi%3D10.1021%2Fci200613b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0ljXRLNsvN2e-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Terrett, N.</span><span> </span><span class="NLM_article-title">Drugs in middle space</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">474</span><span class="NLM_x">–</span> <span class="NLM_lpage">475</span><span class="refDoi"> DOI: 10.1039/c2md90062a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1039%2Fc2md90062a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFShtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=474-475&author=N.+Terrett&title=Drugs+in+middle+space&doi=10.1039%2Fc2md90062a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs in middle space</span></div><div class="casAuthors">Terrett, Nick</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-475</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The paper by Lipinski et al. in 1997 first outlining the 'Rule of 5' suggested that poor absorption or permeability would result if compds. exceeded clearly defined hydrogen-bond counts, mol. wt. (MW) or lipophilicity (log D).  These simple and useful guidelines for drug design were rapidly applied throughout the pharmaceutical industry.  One particular consequence was a reluctance to contemplate investigation of any drug mol. with a mol. wt. exceeding 500 Daltons.  Many researchers now focus on the region of chem. space that exists between small mols. and biologics, based on the expectation that larger mols. (with MW between 500 and 1500) will more readily provide high affinity interactions with featureless protein surfaces, and yet can be optimized to achieve oral absorption and cell permeability.  The challenge for middle space research is to design similarly active compds. but in novel synthetically accessible forms, and to fine-tune phys. properties so that irresp. of mol. wt., the compds. are orally bioavailable and can penetrate cell membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorWAjFt2huMrVg90H21EOLACvtfcHk0ljXRLNsvN2e-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFShtLs%253D&md5=2537549322060be5598b031d2cccb9ac</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2Fc2md90062a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md90062a%26sid%3Dliteratum%253Aachs%26aulast%3DTerrett%26aufirst%3DN.%26atitle%3DDrugs%2520in%2520middle%2520space%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D474%26epage%3D475%26doi%3D10.1039%2Fc2md90062a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Doak, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Over, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: insights from drugs and clinical candidates</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1115</span><span class="NLM_x">–</span> <span class="NLM_lpage">1142</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0ljXRLNsvN2e-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span> </span><span class="NLM_article-title">A sorcerer’s apprentice and the rule of five: from rule-of-thumb to commandment and beyond</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">995</span><span class="NLM_x">–</span> <span class="NLM_lpage">997</span><span class="refDoi"> DOI: 10.1016/j.drudis.2007.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.drudis.2007.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18061876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A280%3ADC%252BD2sjivVGrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=995-997&author=C.+Abad-Zapatero&title=A+sorcerer%E2%80%99s+apprentice+and+the+rule+of+five%3A+from+rule-of-thumb+to+commandment+and+beyond&doi=10.1016%2Fj.drudis.2007.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A sorcerer's apprentice and The Rule of Five: from rule-of-thumb to commandment and beyond</span></div><div class="casAuthors">Abad-Zapatero Cele</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">995-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4k8xMsSJHYGLFM_lbR1VmfW6udTcc2eZAG-tr6EODCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjivVGrtQ%253D%253D&md5=d09c56d696d2977f7a4e5f16992631a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26atitle%3DA%2520sorcerer%25E2%2580%2599s%2520apprentice%2520and%2520the%2520rule%2520of%2520five%253A%2520from%2520rule-of-thumb%2520to%2520commandment%2520and%2520beyond%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D995%26epage%3D997%26doi%3D10.1016%2Fj.drudis.2007.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Zhang, M.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, B.</span><span> </span><span class="NLM_article-title">Drug discovery beyond the ‘rule-of-five’</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span><span class="refDoi"> DOI: 10.1016/j.copbio.2007.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.copbio.2007.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18035532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=478-488&author=M.-Q.+Zhangauthor=B.+Wilkinson&title=Drug+discovery+beyond+the+%E2%80%98rule-of-five%E2%80%99&doi=10.1016%2Fj.copbio.2007.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the 'rule-of-five'</span></div><div class="casAuthors">Zhang, Ming-Qiang; Wilkinson, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">478-488</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although a very useful guideline for orally bioavailable small-mol. drug design, the 'rule-of-five' (also known as 'Lipinski's rule of drug-likeness') has to some extent been overemphasized.  Firstly, only 51% of all FDA-approved small-mol. drugs are both used orally and comply with the 'rule-of-five'.  This does not even include the increasing no. of biologicals of which several have reached blockbuster' status.  Secondly, it does not cover natural product and semisynthetic natural product drugs, which constitute over one-third of all marketed small-mol. drugs.  A more balanced and programmatic approach to drug discovery should be more productive than to rely on an overemphasis of 'rule-of-five' compliance.  Rather it should consider proactively the development of parenteral drugs in parallel to oral drugs and to consider the development of therapeutic antibodies in parallel to small-mol. drugs.  These are particularly relevant for efforts against 'first-in-class' and/or particularly challenging targets such as proteases and those involving protein-protein interactions.  In addn., more effort should be invested in natural product research.  Emerging novel technologies such as synthetic biol. (genetic engineering of living organisms to produce small-mol. therapeutics) may address several challenging issues of natural product-based drug discovery including synthetic feasibility and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollnOuY9kGrbVg90H21EOLACvtfcHk0ljDnM519LawVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D&md5=4121024557295c4f241255e43b43ea44</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.-Q.%26aulast%3DWilkinson%26aufirst%3DB.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520%25E2%2580%2598rule-of-five%25E2%2580%2599%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D478%26epage%3D488%26doi%3D10.1016%2Fj.copbio.2007.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Walters, W. P.</span><span> </span><span class="NLM_article-title">Going further than Lipinski’s rule in drug design</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1517/17460441.2012.648612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1517%2F17460441.2012.648612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=99-107&author=W.+P.+Walters&title=Going+further+than+Lipinski%E2%80%99s+rule+in+drug+design&doi=10.1517%2F17460441.2012.648612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.648612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.648612%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26atitle%3DGoing%2520further%2520than%2520Lipinski%25E2%2580%2599s%2520rule%2520in%2520drug%2520design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D99%26epage%3D107%26doi%3D10.1517%2F17460441.2012.648612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chennamadhavuni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">How proteins bind macrocycles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">723</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span><span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljDnM519LawVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Driggers, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery - an underexploited structural class</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery+-+an+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0ljDnM519LawVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery%2520-%2520an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">295</span><span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0ljDnM519LawVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0ljjWulvGykHAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0ljjWulvGykHAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muresan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Quantifying the chemical beauty of drugs</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">98</span><span class="refDoi"> DOI: 10.1038/nchem.1243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnchem.1243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22270643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=90-98&author=G.+R.+Bickertonauthor=G.+V.+Paoliniauthor=J.+Besnardauthor=S.+Muresanauthor=A.+L.+Hopkins&title=Quantifying+the+chemical+beauty+of+drugs&doi=10.1038%2Fnchem.1243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chemical beauty of drugs</span></div><div class="casAuthors">Bickerton, G. Richard; Paolini, Gaia V.; Besnard, Jeremy; Muresan, Sorel; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">90-98</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-likeness is a key consideration when selecting compds. during the early stages of drug discovery.  However, evaluation of drug-likeness in abs. terms does not reflect adequately the whole spectrum of compd. quality.  More worryingly, widely used rules may inadvertently foster undesirable mol. property inflation as they permit the encroachment of rule-compliant compds. towards their boundaries.  We propose a measure of drug-likeness based on the concept of desirability called the quant. est. of drug-likeness (QED).  The empirical rationale of QED reflects the underlying distribution of mol. properties.  QED is intuitive, transparent, straightforward to implement in many practical settings and allows compds. to be ranked by their relative merit.  We extended the utility of QED by applying it to the problem of mol. target druggability assessment by prioritizing a large set of published bioactive compds.  The measure may also capture the abstr. notion of aesthetics in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCb7vrfeUodrVg90H21EOLACvtfcHk0ljjWulvGykHAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D&md5=7ab125d4ab381924f965d072695f7432</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1243%26sid%3Dliteratum%253Aachs%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DQuantifying%2520the%2520chemical%2520beauty%2520of%2520drugs%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D90%26epage%3D98%26doi%3D10.1038%2Fnchem.1243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span> </span><span class="NLM_article-title">Dependence of molecular properties on proteomic family for marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3451</span><span class="NLM_x">–</span> <span class="NLM_lpage">3453</span><span class="refDoi"> DOI: 10.1021/jm0603825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3451-3453&author=M.+Viethauthor=J.+J.+Sutherland&title=Dependence+of+molecular+properties+on+proteomic+family+for+marketed+oral+drugs&doi=10.1021%2Fjm0603825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span></div><div class="casAuthors">Vieth, Michal; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3451-3453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assocn. of drugs with their proteomic family reveals that mol. properties of drugs targeting proteases, lipid and peptide G-protein-coupled receptors (GPCRs), and nuclear hormone receptors significantly exceed limits for some properties in the "rule of five", while drugs targeting cytochrome P450s, biogenic amine GPCRs, and transporters have significantly lower values for certain properties.  Also, the variation in drug targets appears to be a factor explaining increasing mol. wt. over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC7vpvPxrV7Vg90H21EOLACvtfcHk0lgHymh7i_SZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D&md5=9272583ace87f6080bd0e1b01ad7c657</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0603825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603825%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DDependence%2520of%2520molecular%2520properties%2520on%2520proteomic%2520family%2520for%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3451%26epage%3D3453%26doi%3D10.1021%2Fjm0603825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">The influence of target family and functional activity on the physicochemical properties of pre-clinical compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2969</span><span class="NLM_x">–</span> <span class="NLM_lpage">2978</span><span class="refDoi"> DOI: 10.1021/jm0512185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0512185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtlOit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2969-2978&author=R.+Morphy&title=The+influence+of+target+family+and+functional+activity+on+the+physicochemical+properties+of+pre-clinical+compounds&doi=10.1021%2Fjm0512185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Influence of Target Family and Functional Activity on the Physicochemical Properties of Pre-Clinical Compounds</span></div><div class="casAuthors">Morphy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2969-2978</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The target families of greatest interest in drug discovery can be differentiated on the basis of the physicochem. properties of their preclin. ligands.  The ligands for peptidergic targets, such as peptide G protein-coupled receptors (GPCRs) and integrin receptors, possess significantly higher median property values than those for aminergic targets, such as monoamine transporters and GPCRs.  The ligands for peptide GPCRs were found to be less efficient, in terms of their binding energy per unit of mol. wt. or lipophilicity, than ligands for monoamine GPCRs.  The changes in the property values during the optimization process were found to vary only slightly across the target families, with the main determinant of the drug-likeness of the optimized compds. being the profile of the starting compds.  Agonists for monoamine GPCRs, opioid receptors and ion channels were typically smaller and less lipophilic than the antagonists, but there was no difference between the agonists and the antagonists for peptide GPCRs and nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0EegclZUHNbVg90H21EOLACvtfcHk0lgHymh7i_SZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtlOit7g%253D&md5=fcf63bff570d58d56492b46ddecb5eb1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm0512185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0512185%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DThe%2520influence%2520of%2520target%2520family%2520and%2520functional%2520activity%2520on%2520the%2520physicochemical%2520properties%2520of%2520pre-clinical%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2969%26epage%3D2978%26doi%3D10.1021%2Fjm0512185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lgHymh7i_SZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Nero, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morton, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holien, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wielens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, M. W.</span><span> </span><span class="NLM_article-title">Oncogenic protein interfaces: small molecules, big challenges</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nrc3690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrc3690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24622521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=248-262&author=T.+L.+Neroauthor=C.+J.+Mortonauthor=J.+K.+Holienauthor=J.+Wielensauthor=M.+W.+Parker&title=Oncogenic+protein+interfaces%3A+small+molecules%2C+big+challenges&doi=10.1038%2Fnrc3690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic protein interfaces: small molecules, big challenges</span></div><div class="casAuthors">Nero, Tracy L.; Morton, Craig J.; Holien, Jessica K.; Wielens, Jerome; Parker, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-262</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Historically, targeting protein-protein interactions with small mols. was not thought possible because the corresponding interfaces were considered mostly flat and featureless and therefore 'undruggable'.  Instead, such interactions were targeted with larger mols., such as peptides and antibodies.  However, the past decade has seen encouraging breakthroughs through the refinement of existing techniques and the development of new ones, together with the identification and exploitation of unexpected aspects of protein-protein interaction surfaces.  In this Review, we describe some of the latest techniques to discover modulators of protein-protein interactions and how current drug discovery approaches have been adapted to successfully target these interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk_D7e8hVV7Vg90H21EOLACvtfcHk0lgHymh7i_SZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D&md5=7aad54df0c10c6cdb9b1a7ae0f15bfe9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrc3690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3690%26sid%3Dliteratum%253Aachs%26aulast%3DNero%26aufirst%3DT.%2BL.%26aulast%3DMorton%26aufirst%3DC.%2BJ.%26aulast%3DHolien%26aufirst%3DJ.%2BK.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26atitle%3DOncogenic%2520protein%2520interfaces%253A%2520small%2520molecules%252C%2520big%2520challenges%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D248%26epage%3D262%26doi%3D10.1038%2Fnrc3690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Wirth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoete, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michielin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, W. H.</span><span> </span><span class="NLM_article-title">Protein pocket and ligand shape comparison and its application in virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span><span class="refDoi"> DOI: 10.1007/s10822-013-9659-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-013-9659-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=511-524&author=M.+Wirthauthor=A.+Volkamerauthor=V.+Zoeteauthor=F.+Rippmannauthor=O.+Michielinauthor=M.+Rareyauthor=W.+H.+Sauer&title=Protein+pocket+and+ligand+shape+comparison+and+its+application+in+virtual+screening&doi=10.1007%2Fs10822-013-9659-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9659-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9659-1%26sid%3Dliteratum%253Aachs%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DZoete%26aufirst%3DV.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DSauer%26aufirst%3DW.%2BH.%26atitle%3DProtein%2520pocket%2520and%2520ligand%2520shape%2520comparison%2520and%2520its%2520application%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D511%26epage%3D524%26doi%3D10.1007%2Fs10822-013-9659-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grombacher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Combining global and local measures for structure-based druggability predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1021/ci200454v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200454v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=360-372&author=A.+Volkamerauthor=D.+Kuhnauthor=T.+Grombacherauthor=F.+Rippmannauthor=M.+Rarey&title=Combining+global+and+local+measures+for+structure-based+druggability+predictions&doi=10.1021%2Fci200454v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Global and Local Measures for Structure-Based Druggability Predictions</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Grombacher, Thomas; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting druggability and prioritizing certain disease modifying targets for the drug development process is of high practical relevance in pharmaceutical research.  DoGSiteScorer is a fully automatic algorithm for pocket and druggability prediction.  Besides consideration of global properties of the pocket, also local similarities shared between pockets are reflected.  Druggability scores are predicted by a support vector machine (SVM), trained, and tested on the druggability data set (DD) and its nonredundant version (NRDD).  The DD consists of 1069 targets with assigned druggable, difficult, and undruggable classes.  In 90% of the NRDD, the SVM model based on global descriptors correctly classifies a target as either druggable or undruggable.  Nevertheless, global properties suffer from binding site changes due to ligand binding and from the pocket boundary definition.  Therefore, local pocket properties are addnl. investigated in terms of a nearest neighbor search.  Local similarities are described by distance dependent histograms between atom pairs.  In 88% of the DD pocket set, the nearest neighbor and the structure itself conform with their druggability type.  A discriminant feature between druggable and undruggable pockets is having less short-range hydrophilic-hydrophilic pairs and more short-range lipophilic-lipophilic pairs.  Our findings for global pocket descriptors coincide with previously published methods affirming that size, shape, and hydrophobicity are important global pocket descriptors for automatic druggability prediction.  Nevertheless, the variety of pocket shapes and their flexibility upon ligand binding limit the automatic projection of druggable features onto descriptors.  Incorporating local pocket properties is another step toward a reliable descriptor-based druggability prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRnTVZG_4qNLVg90H21EOLACvtfcHk0lgW8hLqG8POQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP&md5=ec276048e78e1297610e3d4b756ab6e8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci200454v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200454v%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DGrombacher%26aufirst%3DT.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DCombining%2520global%2520and%2520local%2520measures%2520for%2520structure-based%2520druggability%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D360%26epage%3D372%26doi%3D10.1021%2Fci200454v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">MetaPocket: a meta approach to improve protein ligand binding site prediction</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.1089/omi.2009.0045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1089%2Fomi.2009.0045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=325-330&author=B.+Huang&title=MetaPocket%3A+a+meta+approach+to+improve+protein+ligand+binding+site+prediction&doi=10.1089%2Fomi.2009.0045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1089%2Fomi.2009.0045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fomi.2009.0045%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DMetaPocket%253A%2520a%2520meta%2520approach%2520to%2520improve%2520protein%2520ligand%2520binding%2520site%2520prediction%26jtitle%3DOMICS%26date%3D2009%26volume%3D13%26spage%3D325%26epage%3D330%26doi%3D10.1089%2Fomi.2009.0045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Ruddigkeit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Deursen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2864</span><span class="NLM_x">–</span> <span class="NLM_lpage">2875</span><span class="refDoi"> DOI: 10.1021/ci300415d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300415d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2864-2875&author=L.+Ruddigkeitauthor=R.+van+Deursenauthor=L.+C.+Blumauthor=J.+L.+Reymond&title=Enumeration+of+166+billion+organic+small+molecules+in+the+chemical+universe+database+GDB-17&doi=10.1021%2Fci300415d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Enumeration of 166 Billion Organic Small Molecules in the Chemical Universe Database GDB-17</span></div><div class="casAuthors">Ruddigkeit, Lars; van Deursen, Ruud; Blum, Lorenz C.; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2864-2875</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug mols. consist of a few tens of atoms connected by covalent bonds.  How many such mols. are possible in total and what is their structure.  This question is of pressing interest in medicinal chem. to help solve the problems of drug potency, selectivity, and toxicity and reduce attrition rates by pointing to new mol. series.  To better define the unknown chem. space, we have enumerated 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens forming the chem. universe database GDB-17, covering a size range contg. many drugs and typical for lead compds.  GDB-17 contains millions of isomers of known drugs, including analogs with high shape similarity to the parent drug.  Compared to known mols. in PubChem, GDB-17 mols. are much richer in nonarom. heterocycles, quaternary centers, and stereoisomers, densely populate the third dimension in shape space, and represent many more scaffold types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1XVggP7C05rVg90H21EOLACvtfcHk0lgW8hLqG8POQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J&md5=d0bf9a29f3e9ae1e57bb1c953a562ced</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci300415d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300415d%26sid%3Dliteratum%253Aachs%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DEnumeration%2520of%2520166%2520billion%2520organic%2520small%2520molecules%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2864%26epage%3D2875%26doi%3D10.1021%2Fci300415d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span> </span><span class="NLM_article-title">A functional feature analysis on diverse protein-protein interactions: application for the prediction of binding affinity</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1007/s10822-014-9746-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-014-9746-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=619-629&author=J.+Luoauthor=Y.+Guoauthor=Y.+Zhongauthor=D.+Maauthor=W.+Liauthor=M.+Li&title=A+functional+feature+analysis+on+diverse+protein-protein+interactions%3A+application+for+the+prediction+of+binding+affinity&doi=10.1007%2Fs10822-014-9746-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9746-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9746-y%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DM.%26atitle%3DA%2520functional%2520feature%2520analysis%2520on%2520diverse%2520protein-protein%2520interactions%253A%2520application%2520for%2520the%2520prediction%2520of%2520binding%2520affinity%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D619%26epage%3D629%26doi%3D10.1007%2Fs10822-014-9746-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">A hot spot of binding energy in a hormone-receptor interface</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1126/science.7529940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1126%2Fscience.7529940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=7529940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=383-386&author=T.+Clacksonauthor=J.+A.+Wells&title=A+hot+spot+of+binding+energy+in+a+hormone-receptor+interface&doi=10.1126%2Fscience.7529940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A hot spot of binding energy in a hormone-receptor interface</span></div><div class="casAuthors">Clackson, Tim; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5196</span>),
    <span class="NLM_cas:pages">383-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The x-ray crystal structure of the complex between human growth hormone (hGH) and the extracellular domain of its first bound receptor (hGHbp) shows that about 30 side chains from each protein make contact.  Individual replacement of contact residues in the hGHbp with alanine showed that a central hydrophobic region, dominated by two tryptophan residues, accounts for more than three-quarters of the binding free energy.  This "functional epitope" is surrounded by less important contact residues that are generally hydrophilic and partially hydrated, so that the interface resembles a cross section through a globular protein.  The functionally important residues on the hGHbp directly contact those on hGH.  Thus, only a small and complementary set of contact residues maintains binding affinity, a property that may be general to protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4AKI2EsvEQLVg90H21EOLACvtfcHk0lhLTryiEEOjgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D&md5=62b214455d644fd07afb2817d1c482fe</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.7529940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7529940%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520hot%2520spot%2520of%2520binding%2520energy%2520in%2520a%2520hormone-receptor%2520interface%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D383%26epage%3D386%26doi%3D10.1126%2Fscience.7529940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2eZtTQdBDanHmrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Schmidtke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Understanding and predicting druggability. A high-throughput method for detection of drug binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5858</span><span class="NLM_x">–</span> <span class="NLM_lpage">5867</span><span class="refDoi"> DOI: 10.1021/jm100574m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100574m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5858-5867&author=P.+Schmidtkeauthor=X.+Barril&title=Understanding+and+predicting+druggability.+A+high-throughput+method+for+detection+of+drug+binding+sites&doi=10.1021%2Fjm100574m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding and Predicting Druggability. A High-Throughput Method for Detection of Drug Binding Sites</span></div><div class="casAuthors">Schmidtke, Peter; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5858-5867</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability predictions are important to avoid intractable targets and to focus drug discovery efforts on sites offering better prospects.  However, few druggability prediction tools have been released and none has been extensively tested.  Here, a set of druggable and nondruggable cavities has been compiled in a collaborative platform (http://fpocket.sourceforge.net/dcd) that can be used, contributed, and curated by the community.  Druggable binding sites are often oversimplified as closed, hydrophobic cavities, but data set anal. reveals that polar groups in druggable binding sites have properties that enable them to play a decisive role in ligand recognition.  Finally, the data set has been used in conjunction with the open source fpocket suite to train and validate a logistic model.  State of the art performance was achieved for predicting druggability on known binding sites and on virtual screening expts. where druggable pockets are retrieved from a pool of decoys.  The algorithm is free, extremely fast, and can effectively be used to automatically sieve through massive collections of structures (http://fpocket.sourceforge.net).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos1ixAU42dJrVg90H21EOLACvtfcHk0lhLTryiEEOjgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D&md5=d9fc5c4d52a6e0f032b8d5c021a6cd89</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm100574m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100574m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DUnderstanding%2520and%2520predicting%2520druggability.%2520A%2520high-throughput%2520method%2520for%2520detection%2520of%2520drug%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5858%26epage%3D5867%26doi%3D10.1021%2Fjm100574m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Rosenquist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raboisson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vendeville, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Kock, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalmeijer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Rosa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beumont-Mauviel, M.</span><span> </span><span class="NLM_article-title">Discovery and development of simeprevir (TMC435), a HCV NS3/4A protease inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1673</span><span class="NLM_x">–</span> <span class="NLM_lpage">1693</span><span class="refDoi"> DOI: 10.1021/jm401507s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401507s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1673-1693&author=A.+Rosenquistauthor=B.+Samuelssonauthor=P.+O.+Johanssonauthor=M.+D.+Cummingsauthor=O.+Lenzauthor=P.+Raboissonauthor=K.+Simmenauthor=S.+Vendevilleauthor=H.+de+Kockauthor=M.+Nilssonauthor=A.+Horvathauthor=R.+Kalmeijerauthor=G.+de+la+Rosaauthor=M.+Beumont-Mauviel&title=Discovery+and+development+of+simeprevir+%28TMC435%29%2C+a+HCV+NS3%2F4A+protease+inhibitor&doi=10.1021%2Fjm401507s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Simeprevir (TMC435), a HCV NS3/4A Protease Inhibitor</span></div><div class="casAuthors">Rosenquist, Aasa; Samuelsson, Bertil; Johansson, Per-Ola; Cummings, Maxwell D.; Lenz, Oliver; Raboisson, Pierre; Simmen, Kenny; Vendeville, Sandrine; de Kock, Herman; Nilsson, Magnus; Horvath, Andras; Kalmeijer, Ronald; de la Rosa, Guy; Beumont-Mauviel, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1673-1693</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Hepatitis C virus is a blood-borne infection and the leading cause of chronic liver disease (including cirrhosis and cancer) and liver transplantation.  Since the identification of HCV in 1989, there has been an extensive effort to identify and improve treatment options.  An important milestone was reached in 2011 with the approval of the first-generation HCV NS3/4A protease inhibitors.  However, new therapies are needed to improve cure rates, shorten treatment duration, and improve tolerability.  Here we summarize the extensive medicinal chem. effort to develop novel P2 cyclopentane macrocyclic inhibitors guided by HCV NS3 protease assays, the cellular replicon system, structure-based design, and a panel of DMPK assays.  The selection of compd. 29 (simeprevir, TMC435) as clin. candidate was based on its excellent biol., PK, and safety pharmacol. profile.  Compd. 29 has recently been approved for treatment of chronic HCV infection in combination with pegylated interferon-α and ribavirin in Japan, Canada, and USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryCRg3NRRjzrVg90H21EOLACvtfcHk0lhLTryiEEOjgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslekuw%253D%253D&md5=fd9c1df095e47a111d9680af4c40ccc6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm401507s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401507s%26sid%3Dliteratum%253Aachs%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DJohansson%26aufirst%3DP.%2BO.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DLenz%26aufirst%3DO.%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DSimmen%26aufirst%3DK.%26aulast%3DVendeville%26aufirst%3DS.%26aulast%3Dde%2BKock%26aufirst%3DH.%26aulast%3DNilsson%26aufirst%3DM.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DKalmeijer%26aufirst%3DR.%26aulast%3Dde%2Bla%2BRosa%26aufirst%3DG.%26aulast%3DBeumont-Mauviel%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520development%2520of%2520simeprevir%2520%2528TMC435%2529%252C%2520a%2520HCV%2520NS3%252F4A%2520protease%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1673%26epage%3D1693%26doi%3D10.1021%2Fjm401507s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">How many drug targets are there?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">996</span><span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=How+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0lhlUfIxsh3TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DHow%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lhlUfIxsh3TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, C. A.</span><span> </span><span class="NLM_article-title">Ligand efficiency metrics considered harmful</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1007/s10822-014-9757-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-014-9757-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24899110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2ksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=699-710&author=P.+W.+Kennyauthor=A.+Leitaoauthor=C.+A.+Montanari&title=Ligand+efficiency+metrics+considered+harmful&doi=10.1007%2Fs10822-014-9757-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency metrics considered harmful</span></div><div class="casAuthors">Kenny, Peter W.; Leitao, Andrei; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">699-710</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Ligand efficiency metrics are used in drug discovery to normalize biol. activity or affinity with respect to physicochem. properties such as lipophilicity and mol. size.  This Perspective provides an overview of ligand efficiency metrics and summarizes thermodn. of protein-ligand binding.  Different classes of ligand efficiency metric are critically examd. and the study concludes with suggestions for alternative ways to account for physicochem. properties when prioritizing and optimizing leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWFJmrjl6YXrVg90H21EOLACvtfcHk0lhlUfIxsh3TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2ksLw%253D&md5=48885bcd2fcc646577ef78a5e7037494</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9757-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9757-8%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DLeitao%26aufirst%3DA.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DLigand%2520efficiency%2520metrics%2520considered%2520harmful%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D699%26epage%3D710%26doi%3D10.1007%2Fs10822-014-9757-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Reynolds, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bembenek, S. D.</span><span> </span><span class="NLM_article-title">Ligand binding efficiency: trends, physical basis, and implications</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2432</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span><span class="refDoi"> DOI: 10.1021/jm701255b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701255b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFSgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2432-2438&author=C.+H.+Reynoldsauthor=B.+A.+Toungeauthor=S.+D.+Bembenek&title=Ligand+binding+efficiency%3A+trends%2C+physical+basis%2C+and+implications&doi=10.1021%2Fjm701255b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand binding efficiency: trends, physical basis, and implications</span></div><div class="casAuthors">Reynolds, Charles H.; Tounge, Brett A.; Bembenek, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2432-2438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand efficiency (i.e., potency/size) has emerged as an important metric in drug discovery.  In general, smaller, more efficient ligands are believed to have improved prospects for good drug properties (e.g., bioavailability).  Our anal. of thousands of ligands across a variety of targets shows that ligand efficiency is dependent on ligand size with smaller ligands having greater efficiencies, on av., than larger ligands.  We propose two primary causes for this size dependence: the inevitable redn. in the quality of fit between ligand and receptor as the ligand becomes larger and more complex and the redn. in accessible ligand surface area on a per atom basis as size increases.  These results have far-ranging implications for anal. of high-throughput screening hits, fragment-based approaches to drug discovery, and even computational models of potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkrPWFEZZEMrVg90H21EOLACvtfcHk0li-cfQsT4qnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFSgsbw%253D&md5=a9a41b44dbca7f34562c40b56c7ae683</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm701255b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701255b%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26atitle%3DLigand%2520binding%2520efficiency%253A%2520trends%252C%2520physical%2520basis%252C%2520and%2520implications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2432%26epage%3D2438%26doi%3D10.1021%2Fjm701255b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1438</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0li-cfQsT4qnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Chen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foloppe, N.</span><span> </span><span class="NLM_article-title">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">–</span> <span class="NLM_lpage">7920</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.+J.+Chenauthor=N.+Foloppe&title=Tackling+the+conformational+sampling+of+larger+flexible+compounds+and+macrocycles+in+pharmacology+and+drug+discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0li-cfQsT4qnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520conformational%2520sampling%2520of%2520larger%2520flexible%2520compounds%2520and%2520macrocycles%2520in%2520pharmacology%2520and%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0li-cfQsT4qnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Varma, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troutman, M. D.</span><span> </span><span class="NLM_article-title">Physicochemical space for optimum oral bioavailability: contribution of human intestinal absorption and first-pass elimination</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1098</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1021/jm901371v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901371v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFGisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1098-1108&author=M.+V.+Varmaauthor=R.+S.+Obachauthor=C.+Rotterauthor=H.+R.+Millerauthor=G.+Changauthor=S.+J.+Steynauthor=A.+El-Kattanauthor=M.+D.+Troutman&title=Physicochemical+space+for+optimum+oral+bioavailability%3A+contribution+of+human+intestinal+absorption+and+first-pass+elimination&doi=10.1021%2Fjm901371v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical Space for Optimum Oral Bioavailability: Contribution of Human Intestinal Absorption and First-Pass Elimination</span></div><div class="casAuthors">Varma, Manthena V. S.; Obach, R. Scott; Rotter, Charles; Miller, Howard R.; Chang, George; Steyn, Stefanus J.; El-Kattan, Ayman; Troutman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1098-1108</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability (F) is a product of fraction absorbed (Fa), fraction escaping gut-wall elimination (Fg), and fraction escaping hepatic elimination (Fh).  In this study, using a database comprised of Fa, Fg, Fh, and F values for 309 drugs in humans, an anal. of the interrelation of physicochem. properties and the individual parameters was carried out in order to define the physicochem. space for optimum human oral bioavailability.  Trend anal. clearly indicated mol. wt. (MW), ionization state, lipophilicity, polar descriptors, and free rotatable bonds (RB) influence bioavailability.  These trends were due to a combination of effects of the properties on Fa and first-pass elimination (Fg and Fh).  Higher MW significantly impacted Fa, while Fg and Fh decreased with increasing lipophilicity.  Parabolic trends were obsd. for bioavailability with polar descriptors.  Interestingly, RB has a neg. effect on all three parameters, leading to its pronounced effect on bioavailability.  In conclusion, physicochem. properties influence bioavailability with typically opposing effects on Fa and first-pass elimination.  This anal. may provide a rational judgment on the physicochem. space to optimize oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCkIt1i03orVg90H21EOLACvtfcHk0lgwVIayU9tqgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFGisg%253D%253D&md5=9a9e5f4ff74f3d8d9a35c3b2d7e37c78</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm901371v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901371v%26sid%3Dliteratum%253Aachs%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DRotter%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DH.%2BR.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26atitle%3DPhysicochemical%2520space%2520for%2520optimum%2520oral%2520bioavailability%253A%2520contribution%2520of%2520human%2520intestinal%2520absorption%2520and%2520first-pass%2520elimination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1098%26epage%3D1108%26doi%3D10.1021%2Fjm901371v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">He, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, Z. K.</span><span> </span><span class="NLM_article-title">Promiscuity and the conformational rearrangement of drug-like molecules: insight from the Protein Data Bank</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">238</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span><span class="refDoi"> DOI: 10.1002/cmdc.201402389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1002%2Fcmdc.201402389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=25491400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCks7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=238-244&author=M.+W.+Heauthor=P.+S.+Leeauthor=Z.+K.+Sweeney&title=Promiscuity+and+the+conformational+rearrangement+of+drug-like+molecules%3A+insight+from+the+Protein+Data+Bank&doi=10.1002%2Fcmdc.201402389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuity and the Conformational Rearrangement of Drug-Like Molecules: Insight from the Protein Data Bank</span></div><div class="casAuthors">He, Michael W.; Lee, Patrick S.; Sweeney, Zachary K.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">238-244</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Selectivity is a central aspect of lead optimization in the drug discovery process.  Medicinal chemists often try to decrease mol. flexibility to improve selectivity, given the common belief that the two are interdependent.  To investigate the relationship between polypharmacol. and conformational flexibility, we mined the Protein Data Bank and constructed a dataset of pharmaceutically relevant ligands that crystd. in more than one protein target while binding to each co-crystd. receptor with similar in vitro affinities.  After analyzing the mol. conformations of these 100 ligands, we found that 59 ligands bound to different protein targets without significantly changing conformation, suggesting that there is no distinct correlation between conformational flexibility and polypharmacol. within our dataset.  Ligands crystd. in similar proteins and highly ligand-efficient compds. with five or fewer rotatable bonds were less likely to adjust conformation when binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd8jc_c6ldDbVg90H21EOLACvtfcHk0lgwVIayU9tqgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCks7jM&md5=e3e9f6af0a7a15a53e836d9155701d84</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402389%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%2BW.%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DSweeney%26aufirst%3DZ.%2BK.%26atitle%3DPromiscuity%2520and%2520the%2520conformational%2520rearrangement%2520of%2520drug-like%2520molecules%253A%2520insight%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D238%26epage%3D244%26doi%3D10.1002%2Fcmdc.201402389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Drug promiscuity in PDB: protein binding site similarity is key</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e65894</span><span class="refDoi"> DOI: 10.1371/journal.pone.0065894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1371%2Fjournal.pone.0065894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23805191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e65894&author=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Drug+promiscuity+in+PDB%3A+protein+binding+site+similarity+is+key&doi=10.1371%2Fjournal.pone.0065894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Drug promiscuity in PDB: protein binding site similarity is key</span></div><div class="casAuthors">Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65894</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug repositioning applies established drugs to new disease indications with increasing success.  A pre-requisite for drug repurposing is drug promiscuity (polypharmacol.) - a drug's ability to bind to several targets.  There is a long standing debate on the reasons for drug promiscuity.  Based on large compd. screens, hydrophobicity and mol. wt. have been suggested as key reasons.  However, the results are sometimes contradictory and leave space for further anal.  Protein structures offer a structural dimension to explain promiscuity: Can a drug bind multiple targets because the drug is flexible or because the targets are structurally similar or even share similar binding sites.  We present a systematic study of drug promiscuity based on structural data of PDB target proteins with a set of 164 promiscuous drugs.  We show that there is no correlation between the degree of promiscuity and ligand properties such as hydrophobicity or mol. wt. but a weak correlation to conformational flexibility.  However, we do find a correlation between promiscuity and structural similarity as well as binding site similarity of protein targets.  In particular, 71% of the drugs have at least two targets with similar binding sites.  In order to overcome issues in detection of remotely similar binding sites, we employed a score for binding site similarity: Ligand RMSD measures the similarity of the aligned ligands and uncovers remote local similarities in proteins.  It can be applied to arbitrary structural binding site alignments.  Three representative examples, namely the anti-cancer drug methotrexate, the natural product quercetin and the anti-diabetic drug acarbose are discussed in detail.  Our findings suggest that global structural and binding site similarity play a more important role to explain the obsd. drug promiscuity in the PDB than physicochem. drug properties like hydrophobicity or mol. wt.  Addnl., we find ligand flexibility to have a minor influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXd_1yYo5lLVg90H21EOLACvtfcHk0lgwVIayU9tqgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP&md5=99574519028e547ebec5a4c2ed27f6a5</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065894%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DDrug%2520promiscuity%2520in%2520PDB%253A%2520protein%2520binding%2520site%2520similarity%2520is%2520key%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De65894%26doi%3D10.1371%2Fjournal.pone.0065894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">DeLorbe, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clements, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiddon, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. F.</span><span> </span><span class="NLM_article-title">Thermodynamic and structural effects of macrocyclic constraints in protein–ligand interactions</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span><span class="refDoi"> DOI: 10.1021/ml100142y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100142y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=448-452&author=J.+E.+DeLorbeauthor=J.+H.+Clementsauthor=B.+B.+Whiddonauthor=S.+F.+Martin&title=Thermodynamic+and+structural+effects+of+macrocyclic+constraints+in+protein%E2%80%93ligand+interactions&doi=10.1021%2Fml100142y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fml100142y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100142y%26sid%3Dliteratum%253Aachs%26aulast%3DDeLorbe%26aufirst%3DJ.%2BE.%26aulast%3DClements%26aufirst%3DJ.%2BH.%26aulast%3DWhiddon%26aufirst%3DB.%2BB.%26aulast%3DMartin%26aufirst%3DS.%2BF.%26atitle%3DThermodynamic%2520and%2520structural%2520effects%2520of%2520macrocyclic%2520constraints%2520in%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D448%26epage%3D452%26doi%3D10.1021%2Fml100142y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Watts, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tebben, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheney, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J. C.</span><span> </span><span class="NLM_article-title">Macrocycle conformational sampling with MacroModel</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2680</span><span class="NLM_x">–</span> <span class="NLM_lpage">2696</span><span class="refDoi"> DOI: 10.1021/ci5001696</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001696" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2680-2696&author=K.+S.+Wattsauthor=P.+Dalalauthor=A.+J.+Tebbenauthor=D.+L.+Cheneyauthor=J.+C.+Shelley&title=Macrocycle+conformational+sampling+with+MacroModel&doi=10.1021%2Fci5001696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycle Conformational Sampling with MacroModel</span></div><div class="casAuthors">Watts, K. Shawn; Dalal, Pranav; Tebben, Andrew J.; Cheney, Daniel L.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2680-2696</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling low energy conformations of macrocycles is challenging due to the large size of many of these mols. and the constraints imposed by the macrocycle.  We present a new conformational search method (implemented in MacroModel) that uses brief MD simulations followed by minimization and normal-mode search steps.  The method was parametrized using a set of 100 macrocycles from the PDB and CSD.  It was then tested on a publicly available data set for which there are published results using alternative methods; we found that when the same force field is used (in this case MMFFs in vacuum), our method tended to identify conformations with lower energies than what the other methods identified.  The performance on a new set of 50 macrocycles from the PDB and CSD was also quite good; the mean and median RMSD values for just the ring atoms were 0.60 and 0.33 Å, resp.  However, the RMSD values for macrocycles with more than 30 ring-atoms were quite a bit larger compared to the smaller macrocycles.  Possible origins for this and ideas for improving the performance on very large macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBKXPDzoVcLVg90H21EOLACvtfcHk0ljgidpsHimwyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN&md5=612c949e25807bbf6f850c2a2c37b821</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci5001696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001696%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DK.%2BS.%26aulast%3DDalal%26aufirst%3DP.%26aulast%3DTebben%26aufirst%3DA.%2BJ.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DMacrocycle%2520conformational%2520sampling%2520with%2520MacroModel%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2680%26epage%3D2696%26doi%3D10.1021%2Fci5001696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Sondergaard, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carstensen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J. E.</span><span> </span><span class="NLM_article-title">Structural artifacts in protein-ligand X-ray structures: implications for the development of docking scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5673</span><span class="NLM_x">–</span> <span class="NLM_lpage">5684</span><span class="refDoi"> DOI: 10.1021/jm8016464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8016464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5673-5684&author=C.+R.+Sondergaardauthor=A.+E.+Garrettauthor=T.+Carstensenauthor=G.+Pollastriauthor=J.+E.+Nielsen&title=Structural+artifacts+in+protein-ligand+X-ray+structures%3A+implications+for+the+development+of+docking+scoring+functions&doi=10.1021%2Fjm8016464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Artifacts in Protein-Ligand X-ray Structures: Implications for the Development of Docking Scoring Functions</span></div><div class="casAuthors">Sondergaard, Chresten R.; Garrett, Alison Elizabeth; Carstensen, Tommy; Pollastri, Gianluca; Nielsen, Jens Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5673-5684</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of docking scoring functions requires high-resoln. 3D structures of protein-ligand complexes for which the binding affinity of the ligand has been measured exptl.  Protein-ligand binding affinities are measured in soln. expts., and high resoln. protein-ligand structures can be detd. only by X-ray crystallog.  Protein-ligand scoring functions must therefore reproduce soln. binding energies using analyses of proteins in a crystal environment.  We present an anal. of the prevalence of crystal-induced artifacts and water-mediated contacts in protein-ligand complexes and demonstrate the effect that these can have on the performance of protein-ligand scoring functions.  We find 36% of ligands in the PDBBind 2007 refined data set to be influenced by crystal contacts and find the performance of a scoring function to be affected by these.  A Web server for detecting crystal contacts in protein-ligand complexes is available at http://enzyme.ucd.ie/LIGCRYST.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAMRmv0ZVao7Vg90H21EOLACvtfcHk0ljgidpsHimwyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu73P&md5=c5db24f33a237968934499f0f2192597</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm8016464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8016464%26sid%3Dliteratum%253Aachs%26aulast%3DSondergaard%26aufirst%3DC.%2BR.%26aulast%3DGarrett%26aufirst%3DA.%2BE.%26aulast%3DCarstensen%26aufirst%3DT.%26aulast%3DPollastri%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26atitle%3DStructural%2520artifacts%2520in%2520protein-ligand%2520X-ray%2520structures%253A%2520implications%2520for%2520the%2520development%2520of%2520docking%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5673%26epage%3D5684%26doi%3D10.1021%2Fjm8016464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Liebeschuetz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennemann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1007/s10822-011-9538-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1007%2Fs10822-011-9538-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=22246295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=169-183&author=J.+Liebeschuetzauthor=J.+Hennemannauthor=T.+Olssonauthor=C.+R.+Groom&title=The+good%2C+the+bad+and+the+twisted%3A+a+survey+of+ligand+geometry+in+protein+crystal+structures&doi=10.1007%2Fs10822-011-9538-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span></div><div class="casAuthors">Liebeschuetz, John; Hennemann, Jana; Olsson, Tjelvar; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The protein databank now contains the structures of over 11,000 ligands bound to proteins.  These structures are invaluable in applied areas such as structure-based drug design, but are also the substrate for understanding the energetics of intermol. interactions with proteins.  Despite their obvious importance, the careful anal. of ligands bound to protein structures lags behind the anal. of the protein structures themselves.  We present an anal. of the geometry of ligands bound to proteins and highlight the role of small mol. crystal structures in enabling mol. modellers to critically evaluate a ligand model's quality and investigate protein-induced strain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpceotnqHSI-rVg90H21EOLACvtfcHk0ljgidpsHimwyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D&md5=dd85bf6e63910bc85db95509b97c018a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9538-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9538-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%26aulast%3DHennemann%26aufirst%3DJ.%26aulast%3DOlsson%26aufirst%3DT.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520good%252C%2520the%2520bad%2520and%2520the%2520twisted%253A%2520a%2520survey%2520of%2520ligand%2520geometry%2520in%2520protein%2520crystal%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D169%26epage%3D183%26doi%3D10.1007%2Fs10822-011-9538-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0ljmpfHulQF1mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0ljmpfHulQF1mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JZY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JZY','PDB','1JZY'); return false;">PDB: 1JZY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i18"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01286">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_47778"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01286">10.1021/acs.jmedchem.5b01286</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full methods for generation of the data set and its anlysis; figures for all data discussed in the manuscript as well as figures for the full structure data set along with statistical anlysis. (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01286/suppl_file/jm5b01286_si_001.pdf">jm5b01286_si_001.pdf (3.37 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01286%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01286" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991a451d1c3dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
