
<html lang="en"     class="pb-page"  data-request-id="96fffb20-5329-48e7-9c3b-a23e9acfea41"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-10;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.8b00330"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Design, Synthesis, and Characterization of Next-Generation Meprin β Inhibitors" /></meta><meta name="dc.Creator" content="Daniel  Ramsbeck" /></meta><meta name="dc.Creator" content="Antje  Hamann" /></meta><meta name="dc.Creator" content="Georg  Richter" /></meta><meta name="dc.Creator" content="Dagmar  Schlenzig" /></meta><meta name="dc.Creator" content="Stefanie  Geissler" /></meta><meta name="dc.Creator" content="Vera  Nykiel" /></meta><meta name="dc.Creator" content="Holger  Cynis" /></meta><meta name="dc.Creator" content="Stephan  Schilling" /></meta><meta name="dc.Creator" content="Mirko  Buchholz" /></meta><meta name="dc.Description" content="The metalloproteinase meprin β emerged as a current drug target for the treatment of a number of disorders, among those fibrosis, inflammatory bowel disease and Morbus Alzheimer. A major obstacle i..." /></meta><meta name="Description" content="The metalloproteinase meprin β emerged as a current drug target for the treatment of a number of disorders, among those fibrosis, inflammatory bowel disease and Morbus Alzheimer. A major obstacle i..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 25, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00330" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00330" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00330" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00330" /></link>
        
    
    

<title>Structure-Guided Design, Synthesis, and Characterization of Next-Generation Meprin β Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00330" /></meta><meta property="og:title" content="Structure-Guided Design, Synthesis, and Characterization of Next-Generation Meprin β Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0010.jpeg" /></meta><meta property="og:description" content="The metalloproteinase meprin β emerged as a current drug target for the treatment of a number of disorders, among those fibrosis, inflammatory bowel disease and Morbus Alzheimer. A major obstacle in the development of metalloprotease inhibitors is target selectivity to avoid side effects by blocking related enzymes with physiological functions. Here, we describe the structure-guided design of a novel series of compounds, based on previously reported highly active meprin β inhibitors. The bioisosteric replacement of the sulfonamide scaffold gave rise to a next generation of meprin inhibitors. Selected compounds based on this novel amine scaffold exhibit high activity against meprin β and also remarkable selectivity over related metalloproteases, i.e., matrix metalloproteases and A disintegrin and metalloproteinases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00330"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00330">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00330&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00330&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00330&amp;href=/doi/10.1021/acs.jmedchem.8b00330" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 4578-4592</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00298" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00389" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Design, Synthesis, and Characterization of Next-Generation Meprin β Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Daniel Ramsbeck</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Ramsbeck</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#eb8f8a85828e87c5998a8698898e8880ab829182c58d998a9e8583848d8e99c58f8e"><span class="__cf_email__" data-cfemail="5c383d32353930722e3d312f3e393f371c352635723a2e3d293234333a392e723839">[email protected]</span></a>. Phone: +49 345 1314 2826.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Ramsbeck">Daniel Ramsbeck</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4966-142X" title="Orcid link">http://orcid.org/0000-0002-4966-142X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antje Hamann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antje Hamann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antje++Hamann">Antje Hamann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Georg Richter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Georg Richter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Georg++Richter">Georg Richter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dagmar Schlenzig</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dagmar Schlenzig</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dagmar++Schlenzig">Dagmar Schlenzig</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefanie Geissler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefanie Geissler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefanie++Geissler">Stefanie Geissler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vera Nykiel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vera Nykiel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vera++Nykiel">Vera Nykiel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Holger Cynis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Holger Cynis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Holger++Cynis">Holger Cynis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephan Schilling</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephan Schilling</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephan++Schilling">Stephan Schilling</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Mirko Buchholz</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mirko Buchholz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Cell Therapy and Immunology IZI, Department of Drug Design and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#e78a8e958c88c98592848f8f888b9da78e9d8ec981958692898f88818295c98382"><span class="__cf_email__" data-cfemail="a5c8ccd7ceca8bc7d0c6cdcdcac9dfe5ccdfcc8bc3d7c4d0cbcdcac3c0d78bc1c0">[email protected]</span></a>. Phone: +49 345 1314 2825.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mirko++Buchholz">Mirko Buchholz</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00330&amp;href=/doi/10.1021%2Facs.jmedchem.8b00330" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 4578–4592</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 25, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 February 2018</li><li><span class="item_label"><b>Published</b> online</span>25 April 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 May 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00330" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00330</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4578%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDaniel%2BRamsbeck%252C%2BAntje%2BHamann%252C%2BGeorg%2BRichter%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D10%26contentID%3Dacs.jmedchem.8b00330%26title%3DStructure-Guided%2BDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2BNext-Generation%2BMeprin%2B%25CE%25B2%2BInhibitors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4592%26publicationDate%3DMay%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00330"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1507</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00330" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Design, Synthesis, and Characterization of Next-Generation Meprin β Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Ramsbeck&quot;},{&quot;first_name&quot;:&quot;Antje&quot;,&quot;last_name&quot;:&quot;Hamann&quot;},{&quot;first_name&quot;:&quot;Georg&quot;,&quot;last_name&quot;:&quot;Richter&quot;},{&quot;first_name&quot;:&quot;Dagmar&quot;,&quot;last_name&quot;:&quot;Schlenzig&quot;},{&quot;first_name&quot;:&quot;Stefanie&quot;,&quot;last_name&quot;:&quot;Geissler&quot;},{&quot;first_name&quot;:&quot;Vera&quot;,&quot;last_name&quot;:&quot;Nykiel&quot;},{&quot;first_name&quot;:&quot;Holger&quot;,&quot;last_name&quot;:&quot;Cynis&quot;},{&quot;first_name&quot;:&quot;Stephan&quot;,&quot;last_name&quot;:&quot;Schilling&quot;},{&quot;first_name&quot;:&quot;Mirko&quot;,&quot;last_name&quot;:&quot;Buchholz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;4578-4592&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00330&quot;},&quot;abstract&quot;:&quot;The metalloproteinase meprin β emerged as a current drug target for the treatment of a number of disorders, among those fibrosis, inflammatory bowel disease and Morbus Alzheimer. A major obstacle in the development of metalloprotease inhibitors is target selectivity to avoid side effects by blocking related enzymes with physiological functions. Here, we describe the structure-guided design of a novel series of compounds, based on previously reported highly active meprin β inhibitors. The bioisosteric replacement of the sulfonamide scaffold gave rise to a next generation of meprin inhibitors. Selected compounds based on this novel amine scaffold exhibit high activity against meprin β and also remarkable selectivity over related metalloproteases, i.e., matrix metalloproteases and A disintegrin and metalloproteinases.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00330&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00330" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00330&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00330" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00330&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00330" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00330&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00330&amp;href=/doi/10.1021/acs.jmedchem.8b00330" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00330" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00330" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00330%26sid%3Dliteratum%253Aachs%26pmid%3D29694039%26genre%3Darticle%26aulast%3DRamsbeck%26date%3D2018%26atitle%3DStructure-Guided%2BDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2BNext-Generation%2BMeprin%2B%25CE%25B2%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D10%26spage%3D4578%26epage%3D4592%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/jmcmar.2018.61.issue-10/20180524/jmcmar.2018.61.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The metalloproteinase meprin β emerged as a current drug target for the treatment of a number of disorders, among those fibrosis, inflammatory bowel disease and Morbus Alzheimer. A major obstacle in the development of metalloprotease inhibitors is target selectivity to avoid side effects by blocking related enzymes with physiological functions. Here, we describe the structure-guided design of a novel series of compounds, based on previously reported highly active meprin β inhibitors. The bioisosteric replacement of the sulfonamide scaffold gave rise to a next generation of meprin inhibitors. Selected compounds based on this novel amine scaffold exhibit high activity against meprin β and also remarkable selectivity over related metalloproteases, i.e., matrix metalloproteases and A disintegrin and metalloproteinases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26037" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26037" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Proteases represent the largest family of enzymes in vertebrates. In human, they are involved in the regulation of manifold physiological as well as pathophysiological processes. Thus, their inhibition has always been a valuable source for the development of novel therapeutic concepts.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> A subfamily of proteases that draw notable attention in the past are metalloproteases. In particular, the family of metzincins containing, among others, matrix metalloproteases (MMPs) and A disintegrin and metalloproteinases (ADAMs) served as targets for drug development. However, most drug discovery programs failed caused by low selectivity of the inhibitors, leading to severe side effects. Nevertheless, a better understanding of protease biology and new opportunities in inhibitor design, e.g., nonhydroxamates or nonzinc-binding inhibitors, offer novel perspectives for the discovery of drugs in this field.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><div class="NLM_p">A family of zinc-dependent proteases closely related to the MMPs and ADAMs are the astacins. In mammals, this family includes meprin α and β, ovastacin, and BMP-1 (together with alternative splice variants and tolloid-like proteases).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> The latter has already been in focus for the development of novel treatments for fibrotic disorders in the past.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p">In particular, meprin β emerged as a potential drug target in recent years. Several substrates have been identified that link meprin β activity to pathophysiological conditions and to the onset of different diseases.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Both meprins act as procollagenases and are the only known proteases that cleave the protein at the N- as well as the C-terminus.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This is a crucial requirement for the initiation of fibrillation and collagen formation. Hence, they are involved in the formation of connective tissue but also in the development of fibrotic disorders, i.e., lung fibrosis, scarring, and keloids, and are valuable drug targets for treating these diseases.<a onclick="showRef(event, 'ref9 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref11">(9,11)</a> Furthermore, meprin β is potentially involved in the pathogenesis of inflammatory bowel disease via the processing of proinflammatory cytokines, while meprin α exerts a rather protective role.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Membrane bound meprin β, but not meprin α, has also been shown to cleave the amyloid-precursor protein (APP) independently from BACE1.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> Thus, the release of neurotoxic Aβ-peptides renders meprin β as a new player in the amyloid cascade, potentially involved in the onset of Alzheimer’s disease.</div><div class="NLM_p">Nevertheless, little is known about inhibitors of meprin β. The naturally occurring actinonin and the sulfonamide <i>N</i>-isobutyl-<i>N</i>-(4-methoxyphenylsulfonyl)-glycyl hydroxamic acid (NNGH) are the first reported inhibitors with activities in the micromolar range (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known inhibitors of meprin β.<a onclick="showRef(event, 'ref16 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref20">(16,20)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">More recently, a high-throughput screen revealed NF449 and PPNDS as very potent meprin β inhibitors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Albeit these compounds are the most potent meprin inhibitors reported to date, they are rather unselective and are also inhibitors of other metalloproteases.</div><div class="NLM_p">Meprin β exhibits a unique substrate specificity, i.e., a high preference for acidic amino acids adjacent to the scissile bond.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The structural basis for this specificity is the occurrence of three arginine residues (Arg146, Arg184, Arg238) within the active site cleft, shaping the S1, S1′, and S2′ subpockets<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This creates a basic environment, suitable for the interaction with acidic amino acids of the respective substrates and inhibitors.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Active site view of meprin β (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a>). The arginine residues shaping the S1, S1′, and S2′ subpockets are colored in blue. The tetrahedral coordinated Zn<sup>2+</sup> is colored in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the basis of this unique architecture, we reported the first systematic study of the structure–activity relationship of meprin β-inhibitors recently.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Starting from NNGH, compounds <b>1</b> and <b>2</b> were identified as potent inhibitors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The SAR of this compound series clearly revealed that acidic moieties are crucial for inhibiting meprin β, most likely by forming ionic interactions with the above-mentioned arginines in the active site. Although compound <b>1</b> was also identified as a potent inhibitor of MMPs and ADAMs, compound <b>2</b> already exhibited a favorable selectivity profile. This could be attributed to the polar carboxylic acids that are unable to address the lipophilic S1′-pocket of MMPs. In contrast, the methoxyaryl moiety of <b>1</b> would fit within the S1′ pockets of MMPs and ADAMs. Thus, it is also found in other MMP inhibitors, e.g., NNGH. On the basis of these initial observations, we here describe the further evolutionary optimization of these sulfonamide based inhibitors with regard to potency and selectivity. A new series of compounds was thereby designed and evaluated, giving rise to a next generation of selective meprin β inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23285" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23285" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Compound Design</h3><div class="NLM_p">The majority of inhibitors targeting metzincin-metalloproteases are derived from either succinyl hydroxamic acids, e.g., actinonin, Batimastat, and TAPI, or sulfonamide hydroxamic acids, e.g., NNGH.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Important for a high binding affinity of the respective compounds is the coordination of the catalytic metal ion via the hydroxamic acid moiety. Additionally, inhibitors of both scaffolds also form important hydrogen bonds to the protein backbone.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In case of the sulfonamide type inhibitors, this is achieved via the sulfonamide-oxygens.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> These residues form hydrogen bonds to a loop adjacent to the active site, the so-called S-shaped loop, which connects two beta-sheets of the respective protease. The connection of the strands β3 and β4 forming the northern wall of the active site cleft creates a bulge that is involved in inhibitor and substrate binding.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Two exemplary structures are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The inhibitors of ADAM17 (A and D; pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DDF">2DDF</a>) and MMP13 (B and E; pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZXH">3ZXH</a>) are bound to the active site, forming a hydrogen bond between a carbonyl- (ADAM17) or sulfonamide-oxygen (MMP13) and the loop connecting β3 and β4, colored in yellow.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural comparison of ADAM17 (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DDF">2DDF</a>, A), MMP13 (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZXH">3ZXH</a>, B), and meprin β (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a>, C). The S-shaped loop is colored in yellow, forming a bulge adjacent to the active site. The bound inhibitors of ADAM17 (D) and MMP13 (E) interact with the protein backbone via hydrogen bonds, formed with the S-shaped loop. The respective small loop in meprin β (F) rules out these protein–ligand interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The ability of compound <b>1</b> to address these interactions, as well, provides a rationale for the high activity of this inhibitor toward several metalloproteases<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). However, in proteases of the astacin family, this S-loop is missing or only built of very few amino acids.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> As depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,F, the crystal structure of meprin β (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a>) shows a sharp turn of this small loop. This creates an opened active site cleft in this area compared to other related enzymes of the metzincin family. However, this affects predominantly the S1′ and S2′ subsites that are essentially contributing to the substrate specificity and selectivity of the proteases. Hence, it rules out the formation of a hydrogen bond between the loop-backbone and the respective inhibitor.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Schematic representation of possible hydrogen bond interactions of compound <b>1</b> with the S-shaped loop of MMPs or ADAMs (A) and missing hydrogen bond of <b>1</b> bound to meprin β, lacking the S-shaped loop (B). The SO<sub>2</sub> → CH<sub>2</sub> switch could disrupt the respective protein–ligand interaction in MMPs and ADAMs (C) while directing the benzyl residue correctly into the S1′ pocket in meprin β without the need of hydrogen bond formation (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It was already stated earlier that, due to the lacking S-shaped loop, the binding of inhibitors to meprin β is restricted to side chain interactions, rather than the formation of hydrogen bonds involving the actual inhibitor scaffold.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Due to this missing interaction motif on the protein side, we assumed that inhibitors of meprin β might not necessarily feature the respective hydrogen bond acceptor on the inhibitor side. Thus, for meprin β inhibitors of the sulfonamide scaffold like <b>1</b> and <b>2</b>, the SO<sub>2</sub> moiety would just serve as a linker, directing the aryl-substituent into the S<sub>1</sub>′ pocket (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In order to remove this substructure but maintain the tetrahedral geometry of the sulfonyl function, it was replaced by a methylene group. The resulting tertiary amines are likely to resemble the bioactive conformation of the inhibitors, but rule out the formation of any hydrogen bonds involving the scaffold and the β3−β4 connecting loop (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Hence, one important interaction between the inhibitors and other metzincin proteases, i.e., MMPs and ADAMs, would be prohibited, leading to an improved selectivity of the compounds. Although, the remaining interactions of the inhibitors with the S1′ pocket and adjacent binding sites should enable binding to meprin β with sufficient affinity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthesis of the compounds was accomplished starting from Fmoc-protected amino acids <b>3a</b>–<b>o</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In the first step, the protected hydroxamic acid was introduced by coupling the amino acid derivatives to tritylhydroxylamine. Fmoc deprotection by means of DBU in THF yielded the respective primary amines <b>4a</b>–<b>o</b>. For the synthesis of the asymmetrically substituted tertiary amines (<b>6a</b>–<b>o</b>), the first benzyl residue was introduced via reductive amination using either <i>tert</i>-butyl 3-formylbenzoate or another suitable aldehyde and sodium borohydride, yielding intermediates <b>5a</b>–<b>i</b>. The second benzyl residue was introduced via <i>N</i>-alkylation using benzyl halides in the presence of triethylamine under microwave irradiation or at room temperature. Final concurrent removal of all protecting groups using trifluoroacetic acid in dichloromethane yielded the respective tertiary amines <b>6a</b>–<b>o</b>. However, the introduction of the second benzyl residue was not possible in case of any compound derived from C<sub>α</sub>-branched amino acids. Thus, the respective secondary amines obtained by the reaction of <b>4d</b>–<b>o</b> with <i>tert</i>-butyl 3-formylbenzoate were directly subjected to the removal of protecting groups, yielding the secondary amines <b>7b</b>–<b>m</b>. For the synthesis of the symmetric substituted tertiary amines <b>8a</b>–<b>o</b>, <b>4a</b> was alkylated with excess benzyl halide in the presence of triethylamine in DMF, either at room temperature or under microwave irradiation, followed by subsequent cleavage of the protecting groups. For compounds <b>8g</b>–<b>l</b>, the final deprotection was carried out using boron tribromide. This enabled the simultaneous cleavage of the methoxy-protecting groups of the respective phenols and the trityl-protecting group of the hydroxamate.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reaction Scheme for the Synthesis of Secondary and Tertiary Amine Based Hydroxamic Acids<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) i: Trt-O-NH<sub>2</sub>, TBTU, DIPEA, DMF, rt; ii: DBU, THF, rt; (b) R-CHO, NaBH<sub>4</sub>, MeOH, rt; (c) <i>tert</i>-butyl 3-(chloromethyl)benzoate, TEA, DMF, rt or MW 120 °C (for <b>6a,b,c,d,f,g,h,n</b>); (d) TFA/DCM (1:1 v/v), TIS, rt; (e) <i>tert</i>-butyl 3-formylbenzoate, NaBH<sub>4</sub>, MeOH, rt; (f) 1.1 equiv of R-Hal, TEA, DMF, rt or MW 120 °C (for <b>6e,i,j,k,l,m</b>); (g) 2.2 equiv of R-Hal, TEA, DMF, rt or MW 140 °C; (h) BBr<sub>3</sub>, DCM, 0° to rt (for <b>8g–l</b>).</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationships</h3><div class="NLM_p">To probe the hypothesis whether amines could serve as a bioisosteric replacement of the sulfonamide moiety and exhibit activity toward meprin β rather than binding to other metalloproteases, the activity and selectivity of the respective analogue of compound <b>1</b> was assessed in a preliminary experiment. Compound <b>6a</b> proved to be a potent inhibitor of meprin β with similar activity compared to compound <b>1</b>, unaffected by the SO<sub>2</sub> → CH<sub>2</sub> switch (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Hence, binding to meprin β was possible without the formation of a hydrogen bond involving the sulfonamide moiety, underpinning the assumed binding mode of the inhibitors within the active site of meprin β, ruling out an interaction with the S-shaped loop as found for MMP inhibitors to their targets. Furthermore, no inhibition of MMP2, 9, 13 and ADAM10 and 17 by <b>6a</b> could be observed (vide infra). Albeit compound <b>2</b> was a selective inhibitor of meprin β per se,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> the respective amine analogue <b>8a</b> also exhibited similar inhibition of meprin β without affecting the other investigated proteases (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Thus, the replacement of the sulfonamide by a simple tertiary amine proved to be a successful modification, giving rise to a novel class of selective meprin inhibitors by taking advantage of the special structural features of the astacin proteases.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibition of Meprin β by Asymmetric Tertiary Amine Hydroxamates</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0007.gif" alt="" id="GRAPHIC-d309e596-autogenerated" /></img><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibition of Meprin β by Symmetric Tertiary Amine Hydroxamates</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0008.gif" alt="" id="GRAPHIC-d309e603-autogenerated" /></img><div></div></div><div class="NLM_p">To deepen the understanding of the structure–activity relationships of this inhibitor class, further amine derivatives were evaluated regarding their meprin β inhibition. First, the influence of the spacer length between the hydroxamic acid and the amine was investigated. The introduction of an ethylene or propylene spacer, creating a β-alanine- (<b>6b</b>) or GABA-derivative (<b>6c</b>), respectively, led to a decreased activity by a factor ∼10, compared to <b>6a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Thus, all subsequent modifications were just investigated on the basis of the glycyl hydroxamic acid scaffold, represented by <b>6a</b>. As reported earlier, the SAR of the sulfonamides revealed the necessity of acidic substituents and a preference of a <i>meta</i>-substituted carboxybenzyl side chain. Thus, this moiety was kept constant again, and different additional benzyl residues were introduced. The fact that the amine successfully could replace the sulfonamide without losing activity against meprin β was further corroborated by <b>6e</b>, <b>g</b>, <b>n</b>, and <b>o</b>. These compounds exhibit virtually the same activity like their sulfonamide analogues included in the previous study.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the <i>para</i>-methoxy moiety of <b>6a</b> could only be replaced by chlorine (<b>6k</b>) without losing the activity. All other modifications led to a decreased inhibition of meprin β, either by creating potential steric hindrance (<b>6d</b>, <b>f</b>, and <b>n</b>) or by altered, obviously unfavorable, electrostatic properties of the arylmoiety (<b>6g</b>, <b>h</b>, <b>j</b>, <b>l</b>, <b>m</b>).</div><div class="NLM_p">As already mentioned above, <b>8a</b> exhibited similar potency, compared to <b>2</b>. The SAR results derived from this symmetric substituted amine are listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Again, as already found for the sulfonamide series, a clear preference of the <i>meta</i>-substitution of both benzyl residues was found. The <i>para</i>-carboxyl substituted compound (<b>8b</b>) exhibited a decreased potency by a factor ∼10. The activity of the combined <i>meta</i>- and <i>para</i>-carboxyl substituted asymmetric amine <b>6o</b> was decreased by a factor ∼5 compared to <b>8a</b>, respectively. However, this contradicts the recent proposals that predicted a favorable activity for <i>para</i>-carboxyl substituted compounds as well.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The importance of an acidic substitution is corroborated by compounds <b>8c</b>, <b>d</b>, <b>e</b>, and <b>f</b>. The esterification and amidation of the carboxylic acids or the introduction of a cyano-group led to a significant drop or virtually complete loss of activity. Just the introduction of a dioxolane (<b>8f</b>) maintained some activity in the single digit micromolar range. However, the activity of <b>8e</b> and <b>f</b> was slightly shifted to an increased inhibition of meprin α (<b>8e</b>, IC<sub>50</sub> = 1.47 ± 0.12 μM; <b>8f</b>, IC<sub>50</sub> = 0.83 ± 0.05 μM<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>).</div><div class="NLM_p">Because carboxylic acid derivatives often exhibit poor pharmacokinetic properties, this moiety was replaced by bioisosteric halophenols.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The introduction of a difluorophenol (<b>8h</b>) and a chlorofluorophenol (<b>8i</b>) mimicking the <i>meta</i>-carboxylic acid resulted in inhibitors with similar potency compared to <b>8a</b>. The same was observed by replacing the <i>para</i>-carboxylic acid (<b>8b</b>) by the respective difluorophenol (<b>8g</b>), even though exhibiting the same decreased activity compared to <b>8a</b>. Albeit the halophenols were able to replace the carboxylic acid, their reduced acidity also led to a decreased selectivity for meprin β, by significantly increasing the activity against meprin α compared to <b>8a</b> (vide infra). The S1 and S2′ subpockets within the active site of meprin β are formed by arginines Arg184 and Arg146, respectively. In meprin α, both residues are replaced by tyrosines.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> For <b>8a</b>, this may lead to repulsive interactions, reflected by poor meprin α inhibition and high selectivity. These repulsive interactions might be diminished for the halophenols <b>8h</b> and <b>i</b>, leading to increased potency against meprin α and less selectivity. Nevertheless, sufficient acidity of the phenols is important for inhibition of meprin β. Further decreasing the p<i>K</i><sub>a</sub> of the phenol by deleting one fluorine leads to a decreased activity by a factor ∼50 (<b>8k</b>) or ∼100 (<b>8l</b>), respectively. The potency is also strongly decreased by moving one fluorine to the <i>ortho</i> position (<b>8j</b>), most likely caused by introducing some steric hindrance. As expected, the methylation of the phenols (<b>8m</b> and <b>n</b>) led to a drop in potency by ruling out an interaction of the acidic group with the active site arginines. More recently, −OCHF<sub>2</sub> was reported as a potential bioisosteric replacement of a hydroxyl group, acting as a “lipophilic H-bond donor”.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, the introduction of a −OCHF<sub>2</sub> moiety in <i>meta</i> position of the benzyl residues (<b>8o</b>) resulted in a complete loss of inhibitory activity. This corroborates the importance of “true“ acidic functional groups, which cannot be replaced just by a hydrogen bond donor.</div><div class="NLM_p">As already mentioned in the chemistry section, we were only successful in the synthesis of tertiary amines derived from glycine, not from any C<sub>α</sub>-branched amino acid. Hence, it was only possible to elaborate the influence of C<sub>α</sub>-substitution on the activity of the compounds on basis of the secondary amines <b>7a</b>–<b>m</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, the introduction of any substituent, in both <i>R</i> and <i>S</i> configurations, resulted in a significant drop of potency compared to the glycine derivative <b>7a</b>, most likely caused by steric hindrance within the narrow active site cleft of meprin β, created by the bulky side chain of Arg184 forming the S1 pocket. This effect was less pronounced for the aspartate or glutamate derivatives <b>7k</b>, <b>l</b>, and <b>m</b>. These acidic side chains match the substrate specificity of meprin β, and stronger ionic interactions, most likely with Arg184, might partially override the steric hindrance of the side chains.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibition of Meprin β by Secondary Amine Hydroxamates</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0009.gif" alt="" id="GRAPHIC-d309e881-autogenerated" /></img><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Docking and Putative Binding Mode</h3><div class="NLM_p">To gain an insight into a putative binding mode of the inhibitors, compounds <b>6a</b>, <b>8a</b>, and <b>8i</b> were docked into the active site of meprin β using GOLD.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The docking solutions were visually inspected with regard to a reasonable complexation of the zinc atom. Two plausible docking poses of <b>6a</b> and the top ranked poses of <b>8a</b> and <b>i</b> are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. In general, the found docking poses suggest a binding of all three compounds to the S1 and S1′ pockets, formed by Arg184 and Arg238, respectively. However, a binding mode involving an interaction with the S2′ pocket was not predicted. Due to the high flexibility of the dibenzylamine moiety, which allows a flipping of the two benzyl residues within the active site, two different binding modes are assumed for <b>6a</b>. The first involves an interaction of the <i>meta</i>-carboxylic acid with Arg238, i.e., the S1′ subsite, and a potential interaction of the <i>para</i>-methoxyaryl moiety with Arg184 (S1 subsite; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The second, flipped binding mode suggests an interaction of the carboxylate with Arg184, while the <i>para</i>-methoxyphenyl is directed toward the S1′ pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). In both cases, ionic interactions or charged hydrogen bonds of the carboxylic acid and the respective arginine could be assumed.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding modes of <b>6a</b>, <b>8a</b>, and <b>8i</b> within the active site of meprin β. (A) Compound <b>6a</b> solution ranked 3 of 20 and (B) 7 of 20. (C) Compound <b>8a</b> solution ranked 1 of 20. (D) Compound <b>8i</b> solution ranked 1 of 20.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The <i>para</i>-methoxyphenyl residue and the arginine on the other side could form a cation−π interaction or hydrogen bond, as well. However, reliable information based on the docking results could not be gained. Due to the flexibility of the compounds, the orientation of the asymmetric substituted derivatives within the active site remains elusive. For compound <b>8a</b>, the predicted binding mode involves charged interactions targeting both arginines, the S1 (Arg184) and S1′ (Arg238) subsite (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The charged interaction with Arg184 could explain the high selectivity of <b>8a</b> for meprin β (<a class="ref internalNav" href="#notes1" aria-label="Supplementary Figure 1A">Supplementary Figure 1A</a>). The respective residue in meprin α is a tyrosine (Tyr187), while the arginine forming the S1′ pocket is conserved.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Thus, direction of one carboxylic acid to the S1 pocket would cause repulsive forces (<a class="ref internalNav" href="#notes1" aria-label="Supplementary Figure 1B">Supplementary Figure 1B</a>). However, for <b>6a</b>, a flipping of the benzyl moieties would enable a binding mode within the active site of meprin α with the carboxylic acid targeting the S1′ and the <i>para</i>-methoxyaryl moiety addressing the S1 pocket via hydrogen bonding (<a class="ref internalNav" href="#notes1" aria-label="Supplementary Figure 1C">Supplementary Figure 1C</a>). This may be reflected by the increased activity toward meprin α of compound <b>6a</b> compared to <b>8a</b>. The same ionic interactions like those predicted for <b>8a</b> are also addressed by <b>8i</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D), although by weaker acidic phenols that do not completely rule out an interaction with the tyrosine in meprin α. In addition to the predicted charged interactions of the carboxylates and phenolates with Arg184 and Arg238, the aryl moieties of <b>8a</b> and <b>i</b> addressing the S1′ pocket could be fixed via a hydrogen bond or OH–Cl interaction within this subsite involving Ser212.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Selectivity and Preliminary ADME Properties</h3><div class="NLM_p">The aim of the present study was to optimize the recently reported sulfonamide derived meprin β inhibitors regarding their selectivity toward other related metalloproteases. Compound <b>1</b> was found to be not only a potent meprin β inhibitor<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> but also exhibited high activity on selected MMPs and ADAMs. However, the structure-guided design of the novel tertiary amine scaffold revealed a remarkable improvement of compound selectivity. Compound <b>6a</b>, the direct analogue of <b>1</b>, does not exhibit significant inhibition of the studied proteases. Even at a compound concentration of 200 μM, the inhibition of MMP2, 9, 13 and ADAM10 and 17 is only modest, and no reduction to half-maximal enzyme activity was observed, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Thus, compared to <b>1</b>, compound <b>6a</b> was more selective. As mentioned above, the inhibition of meprin β is not affected by the replacement of the sulfonamide by the methylene group. Furthermore, the selectivity over meprin α is similar to <b>1</b> with a factor ∼40. The same selectivity profile within the tested protease panel was observed for the remaining evaluated compounds <b>6k</b>, <b>8a</b>, <b>8h</b>, and <b>8i</b>. None of the other investigated proteases were significantly inhibited (for comparison: the meprin α IC<sub>50</sub> reported for <b>8a</b> corresponds to a residual enzyme activity of <10% @ 200 μM inhibitor). Furthermore, only moderate inhibition of cytochromes 3A4 or 2C9 was observed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Hence, the tertiary amine hydroxamic acid scaffold reported here seems to be a valuable basis for the discovery of novel inhibitors of meprin or astacin metalloproteases in general.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selectivity Profiling of Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>1</b></th><th class="colsep0 rowsep0" align="center"><b>6a</b></th><th class="colsep0 rowsep0" align="center"><b>6k</b></th><th class="colsep0 rowsep0" align="center"><b>8a</b></th><th class="colsep0 rowsep0" align="center"><b>8h</b></th><th class="colsep0 rowsep0" align="center"><b>8i</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">meprin β IC<sub>50</sub> [nM]</td><td class="colsep0 rowsep0" align="left">57 ± 1</td><td class="colsep0 rowsep0" align="left">77 ± 10</td><td class="colsep0 rowsep0" align="left">76 ± 4</td><td class="colsep0 rowsep0" align="left">49 ± 11</td><td class="colsep0 rowsep0" align="left">23 ± 1</td><td class="colsep0 rowsep0" align="left">24 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">meprin α IC<sub>50</sub> [nM]</td><td class="colsep0 rowsep0" align="left">3293 ± 405</td><td class="colsep0 rowsep0" align="left">3420 ± 325</td><td class="colsep0 rowsep0" align="left">1795 ± 205</td><td class="colsep0 rowsep0" align="left">16050 ± 212</td><td class="colsep0 rowsep0" align="left">626 ± 1</td><td class="colsep0 rowsep0" align="left">368 ± 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BMP-1 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MMP2 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left">68</td><td class="colsep0 rowsep0" align="left">69</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MMP9 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MMP13 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">102</td><td class="colsep0 rowsep0" align="left">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADAM10 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADAM17 RA [%]<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">76</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP 3A4 RA [%]<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP 2C9RA [%]<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">102</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SY-5Y [%]<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">87*</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">96*</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HepG2 [%]<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">100*</td><td class="colsep0 rowsep0" align="left">104</td><td class="colsep0 rowsep0" align="left">97*</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">101</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Remaining enzyme activity @ 200 μM inhibitor.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Remaining activity @ 25 μM inhibitor.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Cell viability @ 30 μM inhibitor (*100 μM inhibitor). n.t. = not tested.</p></div></div></div><div class="NLM_p">However, as already observed for the sulfonamides, the highest selectivity for meprin β over meprin α is achieved by the introduction of two <i>meta</i>-carboxylic acids (<b>8a</b>), leading to a selectivity factor of ∼300 between both isoforms. The deletion of one acid (<b>6a</b> and <b>6k</b>) or the introduction of the less acidic halophenols (<b>8h</b> and <b>8i</b>) increases the potency toward meprin α. Due to the acceptable potency of <b>8h</b> and <b>8i</b> for inhibition of both, meprin α and β, they may serve as pan-meprin inhibitors and thus enable the investigation of the inhibition of both proteases in diverse disease models, without affecting other metalloproteases.</div><div class="NLM_p">Most interesting is the lack of activity of <b>8a</b>, <b>h</b>, and <b>i</b> against bone morphogenetic protein 1 (BMP-1), another astacin protease present in humans. Although the structure of BMP-1 has the same overall fold as meprin β, subtle differences in the active site allow a discrimination of both proteases by inhibitors.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The arginine residue Arg184 shaping the S1-pocket of meprin β is replaced by glutamine in BMP-1. Hence, the loss of favorable interactions between the inhibitor’s acidic groups and the protein’s basic residues could explain the observed selectivity.</div><div class="NLM_p">Since no cytotoxicity was observed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), the three most active compounds (<b>8a</b>, <b>h</b>, and <b>i</b>) were selected for oral and intraperitoneal rat screening experiments to assess their bioavailability and first insights into their pharmacokinetic properties (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). However, the plasma concentrations after oral administration are rather low. This might be due to the polar properties of the compounds, combining a hydroxamic acid together with acidic functional groups hampering the absorption within the gastrointestinal tract or degradation during their passage. However, the bioavailability and the plasma concentrations of <b>8a</b> and <b>8i</b> after intraperitoneal application are in an acceptable range. Albeit a general metabolic liability of hydroxamic acids has been reported,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> the compounds exhibit a reasonable stability and plasma clearance, reflected by their acceptable half-life (<i>t</i><sub>1/2</sub>) after intravenous application and their good bioavailability after i.p. application, which also includes a first-pass metabolism of the compounds.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Results of Bioavailability Screening<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>8a</b></th><th class="colsep0 rowsep0" align="center"><b>8h</b></th><th class="colsep0 rowsep0" align="center"><b>8i</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">p.o.</td><td class="colsep0 rowsep0" align="left"><i>F</i><sub>abs</sub></td><td class="colsep0 rowsep0" align="left">13%</td><td class="colsep0 rowsep0" align="left">5%</td><td class="colsep0 rowsep0" align="left">9%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">0.34 mg/L</td><td class="colsep0 rowsep0" align="left">1.08 mg/L</td><td class="colsep0 rowsep0" align="left">0.81 mg/L</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">0.17 h</td><td class="colsep0 rowsep0" align="left">0.13 h</td><td class="colsep0 rowsep0" align="left">0.25 h</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>d</sub></td><td class="colsep0 rowsep0" align="left">0.073 L</td><td class="colsep0 rowsep0" align="left">0.045 L</td><td class="colsep0 rowsep0" align="left">0.058 L</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">i.p.</td><td class="colsep0 rowsep0" align="left"><i>F</i><sub>abs</sub></td><td class="colsep0 rowsep0" align="left">84%</td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">84%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">6.46 mg/L</td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">10.88 mg/L</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">0.12 h</td><td class="colsep0 rowsep0" align="left">n.t.</td><td class="colsep0 rowsep0" align="left">0.17 h</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">i.v.</td><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub></td><td class="colsep0 rowsep0" align="left">2.5 h</td><td class="colsep0 rowsep0" align="left">1.1 h</td><td class="colsep0 rowsep0" align="left">1.9 h</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL</td><td class="colsep0 rowsep0" align="left">1.78 L/h/kg</td><td class="colsep0 rowsep0" align="left">1.31 L/h/kg</td><td class="colsep0 rowsep0" align="left">1.26 L/h/kg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>dss</sub></td><td class="colsep0 rowsep0" align="left">0.42 L/kg</td><td class="colsep0 rowsep0" align="left">0.41 L/kg</td><td class="colsep0 rowsep0" align="left">0.33 L/kg</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">n.t. = not tested.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Compelling data renders the metalloproteinase meprin β as a promising drug target. To study the role of the enzyme in various diseases, potent and selective probes are necessary. However, the majority of known meprin β inhibitors are lacking a sufficient selectivity. Thus, it is hard to discriminate and link the effects caused by inhibitors, e.g., actinonin, to meprin β activity rather than to other metalloproteases, e.g., MMPs. Recently, we already reported the first selective inhibitors of meprin β, based on an established sulfonamide scaffold. The aim of the present study was to further optimize the selectivity of the inhibitors against other members of the metzincin family. Using a structure-based approach, the successful replacement of the sulfonamide by a tertiary amine led to inhibitors with high inhibitory potency against meprin β and a remarkable selectivity against other metalloproteinases. Hence, the inhibitors reported herein could be used as potent probes to investigate meprin β or the concomitant inhibition of both meprins in complex systems. Although their oral bioavailability is comparatively low, they exhibit reasonable absorption after intraperitoneal application. This makes them suitable as tool compounds for the utilization in animal models to further study the role of meprin β or meprin β/α in various disease models. Furthermore, the compounds could also serve as leads for further optimizations. Even their low bioavailability may be beneficial for particular indications, i.e., the local application for their utilization in inflammatory bowel disease or fibrosis.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p last">Starting materials and solvents were purchased from Aldrich, Activate Scientific, Alfa Aesar, Iris Biotech, and Merck Millipore. The purity of the compounds was assessed by HPLC and confirmed to be ≥95%. The analytical HPLC system consisted of a Merck–Hitachi device (model LaChrom) utilizing a Phenomenex Luna 5 μM C18(2) column (125 mm × 4.0 mm) with λ = 214 nm as the reporting wavelength. The compounds were analyzed using a gradient at a flow rate of 1 mL/min, whereby eluent (A) was acetonitrile and eluent (B) was water, both containing 0.04% (v/v) trifluoroacetic acid applying the following gradient: gradient 1: 0–5 min, 5% (A); 5–15 min, 5–60% (A); 15–20 min, 60–95% (A); 20–30 min, 95% (A); gradient 2: 0–15 min, 5–50% (A); 15–20 min, 50–95% (A); 20–23 min, 95% (A). The purities of all reported compounds were determined by the percentage of the peak area at 214 nm. ESI-Mass spectra were obtained with an Expression CMS spectrometer (Advion). The high resolution positive ion ESI mass spectra were obtained from a LTQ Orbitrap XL (Thermo Fisher Scientific). The <sup>1</sup>H NMR spectra were recorded at a DDR2 400 spectrometer (Agilent). DMSO-<i>d</i><sub>6</sub> was used as solvent unless otherwise specified. Chemical shifts are expressed as parts per million (ppm). The solvent was used as internal standard. Splitting patterns have been designated as follows: s (singlet), d (doublet), dd (doublet of doublet), t (triplet), m (multiplet), and br (broad signal). Semipreparative HPLC was performed on a Prepstar device (Varian) equipped with a Phenomenex Luna 10 μM C18(2) column (250 mm × 21 mm). The compounds were eluted using the same solvent system as described above, applying a flow rate of 21 mL/min.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Method for the Synthesis of the Trityl Protected Hydroxamic Acids <b>4a</b>–<b>o</b></h3><div class="NLM_p last">The respective Fmoc-amino acid (1 equiv) was dissolved in DMF (1 mL/mmol). Tritylhydroxylamine (1 equiv), TBTU (1 equiv), and DIPEA (2 equiv) were added, and the mixture was stirred at room temperature for 1.5 h. The reaction was quenched with water. The resulting precipitate was collected by filtration and washed with diethyl ether and a small amount of saturated aqueous NaHCO<sub>3</sub>. The residue was redissolved in THF (2 mL/mmol) and treated with DBU (1.5 equiv). The mixture was stirred at room temperature until completion (TLC monitoring, usually about 30 min). The solvent was evaporated, and the residue was purified by flash chromatography (silica, CHCl<sub>3</sub>/MeOH gradient).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Method for the Synthesis of the Secondary Amines <b>5a</b>–<b>i</b></h3><div class="NLM_p last">The respective trityl protected hydroxamic acid (1 equiv) was dissolved in MeOH (10 mL/mmol). <i>tert</i>-Butyl 3-formylbenzoate (1.1 equiv), or another suitable aldehyde, was added, and the mixture was stirred at room temperature. After 3 h, sodium borohydride (1–1.2 equiv) was added carefully in small portions. The reaction was stirred at the same temperature for further 30 min. The mixture was quenched with water and extracted with EtOAc (3 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated, and the residue was purified by flash chromatography (silica, CHCl<sub>3</sub>/MeOH gradient), if necessary, or used without further purification.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Method for the Synthesis of the Asymmetric Secondary Amines <b>6a</b>–<b>o</b></h3><div class="NLM_p last">The respective trityl protected secondary amines (1 equiv) obtained as described above were dissolved in DMF (5 mL/mmol). Triethylamine (1.1 equiv) and the respective benzyl halide (1.1 equiv) were added, and the mixture was stirred at room temperature overnight. Alternatively, the reaction was heated to 120° for 20 min under microwave irradiation (Biotage Initiator<sup>+</sup>). The reaction was diluted with water and extracted with EtOAc (3 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated. The residue was treated with CH<sub>2</sub>Cl<sub>2</sub>/TFA (1:1 v/v, 5 mL) and triisopropysilane (1.5 equiv). After stirring at room temperature for 2 h, the volatiles were evaporated. The residue was purified by semipreparative HPLC or by flash chromatography (silica, CHCl<sub>3</sub>/MeOH gradient).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Method for the Synthesis of the Secondary Amines <b>7b</b>–<b>m</b></h3><div class="NLM_p last">The respective trityl protected hydroxamic acid (1 equiv) was dissolved in MeOH (10 mL/mmol). <i>tert</i>-Butyl 3-formylbenzoate (1.1 equiv) was added, and the mixture was stirred at room temperature. After 3 h, sodium borohydride (1 equiv) was added carefully in small portions. The reaction was stirred at the same temperature for further 30 min. The mixture was quenched with water and extracted with EtOAc (3 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated, and the residue was directly subjected to deprotection by means of CH<sub>2</sub>Cl<sub>2</sub>/TFA (1:1 v/v, 5 mL) and triisopropylsilane (1.5 equiv) without further purification. After stirring at room temperature for 2 h, the volatiles were evaporated. The residue was purified by semipreparative HPLC.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Method for the Synthesis of the Symmetric Secondary Amines <b>8a</b>–<b>f</b>,<b>m</b>–<b>o</b></h3><div class="NLM_p last">2-Amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 1 equiv) was dissolved in DMF (3–5 mL/mmol). Triethylamine (2.2–4 equiv) and the respective benzyl halide (2.2 equiv) were added, and the mixture was stirred at room temperature overnight. Alternatively, the reaction was heated in a microwave to 140 °C for 20 min. The reaction was quenched with water and extracted with EtOAc (3 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated. The residue was treated with CH<sub>2</sub>Cl<sub>2</sub>/TFA (1:1 v/v, 5 mL) and triisopropylsilane (1.5 equiv) and stirred at room temperature for 2 h. The volatiles were evaporated, and the residue was purified by semipreparative HPLC.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Method for the Synthesis of the Symmetric Secondary Amines <b>8g</b>–<b>l</b></h3><div class="NLM_p">2-Amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 1 equiv) was dissolved in DMF (3–5 mL/mmol). Triethylamine (2.2–4 equiv) and the respective benzyl halide (2.2 equiv) were added, and the mixture was stirred at room temperature overnight. Alternatively, the reaction was heated in a microwave to 140 °C for 20 min. The reaction was quenched with water and extracted with EtOAc (3 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated. The residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and cooled on an ice bath. Boron tribromide (1 M in CH<sub>2</sub>Cl<sub>2</sub>, 6 equiv) was added dropwise. After complete addition, the reaction was allowed to warm to room temperature and stirred overnight. Excess water was added, and stirring was continued for further 10 min. The volatiles were evaporated, and the remaining aqueous solution was lyophilized. The residue was purified by semipreparative HPLC.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]-[(4-methoxyphenyl)methyl]amino]methyl]benzoic Acid (<b>6a</b>)</h4><div class="NLM_p last">The compound was synthesized starting from 2-[(4-methoxyphenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5a</b>, 226 mg, 0.5 mmol, 1 equiv), <i>tert</i>-butyl 3-(chloromethyl)benzoate (125 mg, 0.55 mmol, 1.1 equiv), and TEA (77 μL, 0.55 mmol, 1.1 equiv) as described above and purified by semipreparative HPLC. Yield: 84 mg (36.7%). ESI-MS: <i>m</i>/<i>z</i> 345.5 [M + H]<sup>+</sup>. HPLC (gradient 2): rt 8.72 min (98.2%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.30–3.43 (m, 2H), 3.78 (s, 3H), 4.07–4.49 (m, 4H), 6.96–7.01 (m, 2H), 7.39–7.47 (m, 1H), 7.52–7.59 (m, 2H), 7.71–7.78 (m, 1H), 7.94–8.00 (m, 1H), 8.09–8.19 (m, 1H), 9.02–9.43 (m, 1H), 10.62–10.75 (m, 1H), 13.12 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 345.1447 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>21</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 345.1445.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 3-[[[3-(Hydroxyamino)-3-oxo-propyl]-[(4-methoxyphenyl)methyl]amino]-methyl]benzoic Acid (<b>6b</b>)</h4><div class="NLM_p last">The compound was synthesized from 3-[(4-methoxyphenyl)methylamino]-<i>N</i>-trityloxy-propanamide (<b>5b</b>, 178 mg, 0.38 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (95 mg, 0.42 mmol). Yield: 28 mg (19.5%). ESI-MS: <i>m</i>/<i>z</i> 359.4 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.21 min (95.4%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.00–3.31 (m, 3H), 3.74–3.87 (m, 4H), 4.08–4.57 (m, 4H), 6.93–7.09 (m, 2H), 7.3–7.5 (m, 2H), 7.5–7.67 (m, 1H), 7.67–7.84 (m, 1H), 7.87–8.22 (m, 1H), 8,56–9.4 (m, 1H), 9.67–10.02 (m, 1H), 10.58–10.81 (m, 1H), 13.18 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 359.1598 [M + H]<sup>+</sup>; calcd for C<sub>19</sub>H<sub>23</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 359.1602.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 3-[[[4-(Hydroxyamino)-4-oxo-butyl]-[(4-methoxyphenyl)methyl]amino]-methyl]benzoic Acid (<b>6c</b>)</h4><div class="NLM_p last">The compound was synthesized from 4-[(4-methoxyphenyl)methylamino]-<i>N</i>-trityloxy-butanamide (<b>5c</b>, 690 mg, 1.44 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (358 mg, 1.58 mmol). Yield: 37.4 mg (6.8%). ESI-MS: <i>m</i>/<i>z</i> 373.4 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.40 min (97.9%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.80–2.05 (m, 4H), 2.90–3.05 (m, 2H), 3.75–3.85 (m, 3H), 4.20–4.6 (m, 4H), 6.90–7.10 (m, 2H), 7.35–7.50 (m, 2H), 7.55–7.65 (m, 1H), 7.70–7.80 (m, 1H), 9.95–8.20 (m, 2H), 8,70–9.00 (m, 1H), 9.95–10.10 (m, 1H), 10,52 (s, 1H), 13.20 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 373.1755 [M + H]<sup>+</sup>; calcd for C<sub>20</sub>H<sub>25</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 373.1758.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]-[(3-methoxyphenyl)methyl]amino]-methyl]benzoic acid (<b>6d</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-[(3-methoxyphenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5d</b>, 530 mg, 1.17 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (292 mg, 1.29 mmol). Yield: 150 mg (36.7%). ESI-MS: <i>m</i>/<i>z</i> 345.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.21 min (98.6%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.27 (s, 2H), 3.76–4.47 (m, 7H), 6,90–6.93 (m, 1H), 7.00–7.11 (m, 2H), 7.28–7.37 (m, 1H), 7.50–7.58 (m, 1H), 7.69–7.76 (m, 1H), 7.91–7.98 (m, 1H), 8.07–8.17 (m, 1H), 10,68 (s, 1H), 13.07 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 345.1447 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>21</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 345.1445.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 3-[[1,3-Benzodioxol-5-ylmethyl-[2-(hydroxy-amino)-2-oxo-ethyl]amino]-methyl]benzoic Acid (<b>6e</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)-ethyl]amino]methyl]benzoate (<b>5i</b>, 400 mg, 0.77 mmol) and 5-(bromomethyl)-1,3-benzodioxole (181 mg, 0.84 mmol). Yield: 210 mg (76.6%). ESI-MS: <i>m</i>/<i>z</i> 359.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.95 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.85–4.43 (m, 6H), 6.02–6.06 (m, 2H), 6.89–7.00 (m, 2H), 7.06–7.10 (m, 1H), 7.51–7.58 (m, 1H), 7.70–7.76 (m, 1H), 7.91–7.99 (m, 1H), 8.05–8.16 (m, 1H), 9.07 (br s, 1H), 10.70 (s, 1H), 13.07 (m, 1H). HRMS: <i>m</i>/<i>z</i> 359.1238 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 359.1238.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]-[(4-propoxyphenyl)methyl]amino]-methyl]benzoic Acid (<b>6f</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-[(4-propoxyphenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5e</b>, 263 mg, 0.52 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (130 mg, 0.57 mmol). Yield: 110 mg (55.3%). ESI-MS: <i>m</i>/<i>z</i> 373.4 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 10.29 min (97.3%); <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 0.98 (t, 3H, <sup>3</sup><i>J</i> = 7.5 Hz), 1.73 (q, 2H, <sup>3</sup><i>J</i> = 7.5 Hz), 3.84–4.45 (m, 8H), 6.94–6.99 (m, 2H), 7.36–7.45 (m, 2H), 7.52–7.59 (m, 1H), 7.71–7.76 (m, 1H), 7.93–8.00 (m, 1H), 8.08–8.19 (m, 1H), 9.08 (br s, 1H), 10.74 (s, 1H), 13.10 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 373.1758 [M + H]<sup>+</sup>; calcd for C<sub>20</sub>H<sub>25</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 373.1758.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 3-[[(3-Fluoro-4-methoxy-phenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]-benzoic Acid (<b>6g</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-[(3-fluoro-4-methoxy-phenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5f</b>, 350 mg, 0.74 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (186 mg, 0.82 mmol). Yield: 95 mg (34.3%). ESI-MS: <i>m</i>/<i>z</i> 363.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.40 min (97.8%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.84–4.06 (m, 9H), 7.14–7.40 (m, 3H), 7.50–7.58 (m, 1H), 7.69–7.75 (m, 1H), 7.90–8.14 (m, 2H), 10.68 (s, 1H), 13.06 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 363.1350 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>20</sub>FN<sub>2</sub>O<sub>5</sub><sup>+</sup>, 363.1351.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 3-[[(2,6-Difluoro-4-methoxy-phenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]-benzoic Acid (<b>6h</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-[(2,6-difluoro-4-methoxy-phenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5g</b>, 320 mg, 0.6 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (151 mg, 0.67 mmol). Yield: 40 mg (17.0%). ESI-MS: <i>m</i>/<i>z</i> 380.9 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 9.36 min (97.7%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.77–3.83 (m, 5H), 3.92–3.93 (m, 4H), 6.72–6.84 (m, 2H), 7.44–7.48 (m, 1H), 7.62–7.64 (m, 1H), 7.84–7.87 (m, 1H), 7.95–7.97 (m, 1H), 10,47 (s, 1H), 13,00 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 381.1259 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>F<sub>2</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup>, 381.1257.</div></div><div id="sec4_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 3-[[Benzyl-[2-(hydroxyamino)-2-oxo-ethyl]-amino]methyl]benzoic Acid (<b>6i</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 345 mg, 0.67 mmol) and benzyl bromide (87 μL, 0.73 mmol). Yield: 160 mg (76.4%). ESI-MS: <i>m</i>/<i>z</i> 315.0 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.63 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 4.01–4.39 (m, 6H), 7.34–7.76 (m, 7H), 7.91–8.18 (m, 2H), 8.98 (br s, 1H), 10.65 (s, 1H), 13.08 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 315.1342 [M + H]<sup>+</sup>; calcd for C<sub>17</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 315.1339.</div></div><div id="sec4_7_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]-(<i>p</i>-tolyl-methyl)amino]methyl]benzoic Acid (<b>6j</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 345 mg, 0.67 mmol) and 4-methylbenzyl bromide (136 mg, 0.73 mmol). Yield: 150 mg (68.6%). ESI-MS: <i>m</i>/<i>z</i> 329.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.72 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 2.31 (s, 3H), 3.70–4.44 (m, 6H), 7.18–7.40 (m, 4H), 7.49–7.57 (m, 1H), 7.69–7.76 (m, 1H), 7.89–8.15 (m, 2H), 8.98 (br s, 1H), 10,61 (s, 1H), 13.05 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 329.1497 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>21</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 329.1496.</div></div><div id="sec4_7_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 3-[[(4-Chlorophenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>6k</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 345 mg, 0.67 mmol) and 4-chlorobenzyl chloride (118 mg, 0.73 mmol). Yield: 170 mg (71.0%). ESI-MS: <i>m</i>/<i>z</i> 349.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 9.55 min (97.0%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.18 (s, 2H), 3.93–3.99 (m, 4H), 7.35–7.60 (m, 5H), 7.64–7.74 (m, 1H), 7.87–8.14 (m, 2H), 10.61 (s, 1H), 13.03 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 349.0950 [M + H]<sup>+</sup>; calcd for C<sub>17</sub>H<sub>18</sub>ClN<sub>2</sub>O<sub>4</sub><sup>+</sup>, 349.0950.</div></div><div id="sec4_7_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-[[(4-Fluorophenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>6l</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 400 mg, 0.77 mmol) and 4-fluorobenzyl bromide (159 mg, 0.84 mmol). Yield: 150 mg (59.0%). ESI-MS: <i>m</i>/<i>z</i> 333.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.21 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.98–4.04 (m, 4H), 4.34–4.40 (m, 2H), 7.19–7.28 (m, 2H), 7.47–7.57 (m, 3H), 7.69–7.75 (m, 1H), 7.90–8.15 (m, 2H), 8.99 (br s, 1H), 10.65 (s, 1H), 13.05 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 333.1249 [M + H]<sup>+</sup>; calcd for C<sub>17</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>4</sub><sup>+</sup>, 333.1245.</div></div><div id="sec4_7_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 3-[[(4-Cyanophenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>6m</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 345 mg, 0.67 mmol) and 4-(bromomethyl)benzonitrile (144 mg, 0.73 mmol). Yield: 200 mg (88.5%). ESI-MS: <i>m</i>/<i>z</i> 340.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 8.88 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.10 (s, 2H), 3.88–3.92 (m, 4H), 7.44–7.53 (m, 1H), 7.61–7.71 (m, 3H), 7.81–8.06 (m, 4H), 10.54 (s, 1H), 12.98 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 340.1296 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub><sup>+</sup>, 340.1292.</div></div><div id="sec4_7_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]-[(4-biphenyl)methyl]amino]-methyl]benzoic Acid (<b>6n</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-[(4-biphenyl)methylamino]-<i>N</i>-trityloxy-acetamide (<b>5h</b>, 190 mg, 0.38 mmol) and <i>tert</i>-butyl 3-(chloromethyl)benzoate (95 mg, 0.42 mmol). Yield: 60 mg (40.3%). ESI-MS: <i>m</i>/<i>z</i> 391.9 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 11.57 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.89–4.50 (m, 6H), 7.39–7.42 (m, 1H), 7.46–7.63 (m, 5H),7.68–7.79 (m, 5H), 7.92–7.99 (m, 1H), 8.09–8.20 (m, 1H), 9.33 (br s, 1H), 10.71 (s, 1H), 13.05 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 391.1654 [M + H]<sup>+</sup>; calcd for C<sub>23</sub>H<sub>23</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 391.1652.</div></div><div id="sec4_7_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 3-[[(4-Carboxyphenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>6o</b>)</h4><div class="NLM_p last">The compound was synthesized from <i>tert</i>-butyl 3-[[[2-oxo-2-(trityloxyamino)ethyl]amino]-methyl]benzoate (<b>5i</b>, 410 mg, 0.78 mmol) and 4-bromomethylbenzoic acid <i>tert</i>-butyl ester (234 mg, 0.86 mmol). Yield: 35 mg (12.5%). ESI-MS: <i>m</i>/<i>z</i> 359.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.01 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.18 (m, 2H), 4.00 (m, 4H), 7.49–7.63 (m, 3H), 7.69–7.75 (m, 1H), 7.89–7.99 (m, 3H), 8.02–8.14 (m, 1H), 9.35–9.37 (m, 1H), 10.60 (br s, 1H), 13.04 (br s, 2H). HRMS: <i>m</i>/<i>z</i> 359.1239 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 359.1238.</div></div><div id="sec4_7_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 3-[[[2-(Hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7a</b>)</h4><div class="NLM_p last">Isolated as side product of <b>8a</b>. Yield: 24 mg (10.7%). ESI-MS: <i>m</i>/<i>z</i> 224.9 [M + H]<sup>+</sup>. HPLC (gradient 2): rt 3.84 min (99.1%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.59 (s, 1.5H), 3.91 (s, 0.5H), 4.24–4.28 (m, 2H), 7.55–7.59 (m, 1H), 7.70–7.74 (m, 1H), 7.95–7.99 (m, 1H), 8.13–8.15 (m, 1H), 9.21–9.58 (m, 3H), 10.69 (s, 0.2H), 10.94 (s, 0.8H), 13.19 (br s, 1H).</div></div><div id="sec4_7_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 3-[[[(1<i>S</i>)-2-(Hydroxyamino)-1-methyl-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7b</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>S</i>)-2-amino-<i>N</i>-trityloxy-propanamide (<b>4d</b>, 361 mg, 1 mmol, 1 equiv), with <i>tert</i>-butyl 3-formylbenzoate (237 mg, 1.1 mmol, 1.1 equiv) and sodium borohydride (38 mg, 1 mmol, 1 equiv) as described above. Yield: 52 mg (20.9%). ESI-MS: <i>m</i>/<i>z</i> 239.0 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.52 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.36–1.51 (m, 3H), 3.67–3.78 (m, 1H), 4.06–4.26 (m, 2H), 7.54–7.62 (m, 1H), 7.67–7.75 (m, 1H), 7.96–8.03 (m, 1H), 8.11–8.18 (m, 1H), 9.14–9.59 (m, 3H), 11.14 (s, 1H), 13.21 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 239.1024 [M + H]<sup>+</sup>; calcd for C<sub>11</sub>H<sub>15</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 239.1027.</div></div><div id="sec4_7_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 3-[[[(1<i>R</i>)-2-(Hydroxyamino)-1-methyl-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7c</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>R</i>)-2-amino-<i>N</i>-trityloxy-propanamide (<b>4e</b>, 321 mg, 0.9 mmol) and <i>tert</i>-butyl 3-formylbenzoate (210 mg, 1.02 mmol). Yield: 53 mg (24.0%). ESI-MS: <i>m</i>/<i>z</i> 239.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.47 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.38–1.52 (m, 3H), 3.65–3.74 (m, 1H), 4.05–4.31 (m, 2H), 7.55–7.62 (m, 1H), 7.67–7.75 (m, 1H), 7,99 (d, 1H, <sup>3</sup><i>J</i> = 7.77 Hz), 8.1–8.17 (m, 1H), 9.05–9.57 (m, 2H), 11.10 (s, 1H), 13.13 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 239.1026 [M + H]<sup>+</sup>; calcd for C<sub>11</sub>H<sub>15</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 239.1027.</div></div><div id="sec4_7_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 3-[[[(1<i>S</i>)-1-(Hydroxycarbamoyl)-2-methyl-propyl]amino]methyl]benzoic acid (<b>7d</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>S</i>)-2-amino-3-methyl-<i>N</i>-trityloxy-butanamide (<b>4f</b>, 445 mg, 1.2 mmol) and <i>tert</i>-butyl 3-formylbenzoate (270 mg, 1.3 mmol). Yield: 25 mg (7.9%). ESI-MS: <i>m</i>/<i>z</i> 267.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 4.19 min (100%); <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 0.91–1.01 (m, 6H), 2.09–2.2 (m, 1H), 4.03–4.20 (m, 2H), 7.53–7.60 (m, 1H), 7.2 (d, 1H, <sup>3</sup><i>J</i> = 7.69 Hz), 7.99 (d, 1H, <sup>3</sup><i>J</i> = 7.77 Hz), 8.09–8.16 (m, 1H), 8.83–9.57 (br s, 3H), 11.11 (s, 1H), 13.08 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 267.1338 [M + H]<sup>+</sup>; calcd for C<sub>13</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 267.1340.</div></div><div id="sec4_7_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 3-[[[(1<i>R</i>)-1-(Hydroxycarbamoyl)-2-methyl-propyl]amino]methyl]benzoic Acid (<b>7e</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>R</i>)-2-amino-3-methyl-<i>N</i>-trityloxy-butanamide (<b>4g</b>, 535 mg, 1.4 mmol) and <i>tert</i>-butyl 3-formylbenzoate (324 mg, 1.6 mmol). Yield: 10 mg (2.6%). ESI-MS: <i>m</i>/<i>z</i> 267.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 4.29 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 0.83–1.05 (m, 6H), 2.09–2.22 (m,1H), 4.00–4.21 (m, 2H), 7.53–7.61 (m, 1H), 7.7 (d 1H, <sup>3</sup><i>J</i> = 7.57), 7.99 (d, 1H, <sup>3</sup><i>J</i> = 7.69), 8.09–8.17 (m, 1H), 8.96–9.55 (m, 2H), 11.11 (s, 1H), 13.09 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 267.1338 [M + H]<sup>+</sup>; calcd for C<sub>13</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 267.1340.</div></div><div id="sec4_7_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 3-[[[(1<i>S</i>)-1-Benzyl-2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7f</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>S</i>)-2-amino-3-phenyl-<i>N</i>-trityloxy-propanamide (<b>4h</b>, 367 mg, 0.9 mmol) and <i>tert</i>-butyl 3-formylbenzoate (197 mg, 0.96 mmol). Yield: 75 mg (27.4%). ESI-MS: <i>m</i>/<i>z</i> 315.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 6.77 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.00–3.09 (m, 1H), 3.15–3.23 (m, 1H), 3.72–3.82 (m, 1H), 4.10–4.23 (m, 2H), 7.16–7.22 (m, 2H), 7.24–7.34 (m, 3H), 7.53–7.64 (m, 1H), 7.66–7.73 (m, 1H), 7.95–8.03 (m, 1H) 8.10–8.16 (m, 1H), 9.26–9.78 (m, 3H), 10,99 (s, 1H), 13.11 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 315.1336 [M + H]<sup>+</sup>; calcd for C<sub>17</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 315.1340.</div></div><div id="sec4_7_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-[[[(1<i>R</i>)-1-Benzyl-2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7g</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>R</i>)-2-amino-3-phenyl-<i>N</i>-trityloxy-propanamide (<b>4i</b>, 441 mg, 1.05 mmol) and <i>tert</i>-butyl 3-formylbenzoate (237 mg, 1.15 mmol). Yield: 70 mg (21.3%). ESI-MS: <i>m</i>/<i>z</i> 315.2 [M + H]<sup>+</sup>; HPLC (gradient 1): rt 6.75 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 2.97–3.27 (m, 2H), 3.71–3.81 (m, 1H), 4.04–4.24 (m, 2H), 7.15–7.22 (m, 2H), 7.24–7.38 (m, 3H), 7.53–7.62 (m, 1H), 7.65–7.74 (m, 1H), 7.96–8.02 (m, 1H), 8.11–8.16 (m, 1H), 9.16–9.98 (m, 3H), 10.99 (s, 1H), 13.11 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 315.1337 [M + H]<sup>+</sup>; calcd for C<sub>17</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 315.1340.</div></div><div id="sec4_7_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 3-[[[(1<i>S</i>)-2-(Hydroxyamino)-2-oxo-1-phenyl-ethyl]amino]methyl]benzoic Acid (<b>7h</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>S</i>)-2-amino-2-phenyl-<i>N</i>-trityloxy-acetamide (<b>4j</b>, 545 mg, 1.33 mmol) and <i>tert</i>-butyl 3-formylbenzoate (303 mg, 1.47 mmol). Yield: 26 mg (6.5%). ESI-MS: <i>m</i>/<i>z</i> 301.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 6.80 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 4.00–4–20 (m, 2H), 4.74 (s, 1H), 7.45–7.59 (m, 6H), 7.62–7.68 (m, 1H), 7.96–8.01 (m, 1H), 8.08–8.15 (m, 1H), 9.20–9.50 (m, 1H), 9.70–10.20 (m, 2H), 11.28 (s, 1H), 13.16 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 301.1181 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>17</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 301.1183.</div></div><div id="sec4_7_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 3-[[[(1<i>R</i>)-2-(Hydroxyamino)-2-oxo-1-phenyl-ethyl]amino]methyl]benzoic Acid (<b>7i</b>)</h4><div class="NLM_p last">The compound was synthesized starting from (2<i>R</i>)-2-amino-2-phenyl-<i>N</i>-trityloxy-acetamide (<b>4k</b>, 404 mg, 1 mmol) and <i>tert</i>-butyl 3-formylbenzoate (224 mg, 1.1 mmol). Yield: 4 mg (1.3%). ESI-MS: <i>m</i>/<i>z</i> 301.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 6.88 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 4.00–4–20 (m, 2H), 4.73 (s, 1H), 7.44–7.59 (m, 6H), 7.61–7.68 (m, 1H), 7.96–8.01 (m, 1H), 8.08–8.13 (m, 1H), 9.28–9.41 (m, 1H), 9.81–10.12 (m, 2H), 11.26 (s, 1H), 13.16 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 301.1180 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>17</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 301.1183.</div></div><div id="sec4_7_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 3-[[[(1<i>S</i>)-1-(Carboxymethyl)-2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7j</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <i>tert</i>-butyl (3<i>S</i>)-3-amino-4-oxo-4-(trityloxyamino)-butanoate (<b>4l</b>, 750 mg, 1.68 mmol) and <i>tert</i>-butyl 3-formylbenzoate (381 mg, 1.8 mmol). Yield: 15 mg (3.1%). ESI-MS: <i>m</i>/<i>z</i> 283.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.36 min (95.1%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 2.69–2.92 (m, 2H), 3.72–3.91 (m, 1H), 3.97–4.31 (m, 2H), 7.52–7.61 (m, 1H), 7.63–7.75 (m, 1H), 7.91–8.02 (m, 1H), 8.03–8.15 (m, 1H), 8.58–9.90 (m, 2H), 11.01–11.29 (m, 1H), 11.34–14.42 (m, 2H). HRMS: <i>m</i>/<i>z</i> 283.0919 [M + H]<sup>+</sup>; calcd for C<sub>12</sub>H<sub>15</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 283.0925.</div></div><div id="sec4_7_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 3-[[[(1<i>R</i>)-1-(Carboxymethyl)-2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>7k</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <i>tert</i>-butyl (3<i>R</i>)-3-amino-4-oxo-4-(trityloxyamino)-butanoate (<b>4m</b>, 700 mg, 1.57 mmol) and <i>tert</i>-butyl 3-formylbenzoate (356 mg, 1.72 mmol). Yield: 3 mg (0.7%). ESI-MS: <i>m</i>/<i>z</i> 283.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.44 min (93.8%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 2.73–2.90 (m, 2H), 3.78–3.88 (m, 1H), 4.01–4.20 (m, 2H), 7.53–7.61 (m, 1H), 7.64–7.73 (m, 1H), 7.93–8.01 (m, 1H), 8.06–8.14 (m, 1H), 9.05–9.88 (m, 2H), 11.15 (s, 1H), 11.61–14.38 (m, 2H). HRMS: <i>m</i>/<i>z</i> 283.0920 [M + H]<sup>+</sup>; calcd for C<sub>12</sub>H<sub>15</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 283.0925.</div></div><div id="sec4_7_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 3-[[[(1<i>S</i>)-3-Carboxy-1-(hydroxylcarbamoyl)-propyl]amino]methyl]benzoic Acid (<b>7l</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <i>tert</i>-butyl (4<i>S</i>)-4-amino-5-oxo-5-(trityloxyamino)-pentanoate (<b>4n</b>, 538 mg, 1.17 mmol) and <i>tert</i>-butyl 3-formylbenzoate (265 mg, 1.29 mmol). Yield: 11 mg (3.1%). ESI-MS: <i>m</i>/<i>z</i> 297.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.63 min (96.5%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.88–2.16 (m, 2H), 2.17–2.4 (m, 2H), 3.54–3.68 (m, 1H), 4.01–4.26 (m, 2H), 7.54–7.62 (m, 1H), 7.66–7.75 (m, 1H), 7.94–8.04 (m, 1H), 8.09–8.18 (m, 1H), 9.14–9.67 (m, 2H), 11.09–11.36 (m, 1H), 12.05–12.74 (br s, 1H), 12.76–13.53 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 297.1078 [M + H]<sup>+</sup>; calcd for C<sub>13</sub>H<sub>17</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 297.1081.</div></div><div id="sec4_7_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 3-[[[(1<i>R</i>)-3-Carboxy-1-(hydroxycarbamoyl)-propyl]amino]methyl]benzoic Acid (<b>7m</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <i>tert</i>-butyl (4<i>R</i>)-4-amino-5-oxo-5-(trityloxy-amino)pentanoate (<b>4o</b>, 378 mg, 0.82 mmol) and <i>tert</i>-butyl 3-formylbenzoate (185 mg, 0.89 mmol). Yield: 22 mg (8.7%). ESI-MS: <i>m</i>/<i>z</i> 297.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 3.63 min (95.4%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.89–2.17 (m, 2H), 2.19–2.41 (m, 2H), 3.59–3.68 (m, 1H), 3.98–4.29 (m, 2H), 7.54–7.62 (m, 1H), 7.67–7.74 (m, 1H), 7.97–8.02 (m, 1H), 8.10–8.16 (m, 1H), 9.03–9.86 (m, 3H), 11.25 (s, 1H), 11.89–13–56 (m, 2H). HRMS: <i>m</i>/<i>z</i> 297.1078 [M + H]<sup>+</sup>; calcd for C<sub>13</sub>H<sub>17</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 297.1081.</div></div><div id="sec4_7_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 3-[[(3-Carboxyphenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic Acid (<b>8a</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <b>4a</b> (332 mg, 1 mmol, 1 equiv), <i>tert</i>-butyl 3-(chloromethyl)benzoate (499 mg, 2.2 mmol, 2.2 equiv), and triethylamine (305 μL, 2.2 mmol, 2.2 equiv) as described above. Yield: 43 mg (12.0%). ESI-MS: <i>m</i>/<i>z</i> 359.1 [M + H]<sup>+</sup>. HPLC (gradient 2): rt 7.73 min (97.4%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.24 (br s, 1H), 4.07 (br s, 3.1 H), 4.43 (br s, 0.9 H), 7.49–7.58 (m, 2H), 7.68–7.74 (m, 2H), 7.89–7.97 (m, 2H), 8.03 (br s, 1.6H), 8.14 (br s, 0.4H), 10.64 (s, 1H), 13.06 (br s, 2H). HRMS: <i>m</i>/<i>z</i> 359.1239 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 359.1238.</div></div><div id="sec4_7_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 4-[[(4-Carboxyphenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzoic acid (<b>8b</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol) and 4-bromomethylbenzoic acid <i>tert</i>-butyl ester (596 mg, 2.2 mmol). Yield: 12 mg (3.3%). ESI-MS: <i>m</i>/<i>z</i> 359.3 [M + H]<sup>+</sup>. HPLC (gradient 2): rt 7.39 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.10 (s, 2H), 3.90 (s, 4H), 7.54–7.56 (m, 4H), 7.91–7.93 (m, 4H), 10.52 (br s, 1H), 12.94 (br s, 2H). HRMS: <i>m</i>/<i>z</i> 359.1241 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 359.1238</div></div><div id="sec4_7_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Ethyl 3-[[(3-Ethoxycarbonylphenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]-benzoate (<b>8c</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 499 mg, 1.5 mmol) and ethyl 3-(chloromethyl)benzoate (656 mg, 3.3 mmol). Yield: 70 mg (11.2%). ESI-MS: <i>m</i>/<i>z</i> 415.0 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 12.93 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 1.32–1.35 (m, 6H), 3.18–3.40 (m, 2H), 3.90–4.08 (m, 4H), 4.30–4.36 (m, 4H), 7.48–7.56 (m, 2H), 7.64–7.75 (m, 2H), 7.86–8.10 (m, 4H), 10.59 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 415.1860 [M + H]<sup>+</sup>; calcd for C<sub>22</sub>H<sub>27</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 415.1864.</div></div><div id="sec4_7_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 3-[[(3-Carbamoylphenyl)methyl-[2-(hydroxyamino)-2-oxo-ethyl]amino]methyl]benzamide (<b>8d</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 499 mg, 1.5 mmol) and methyl 3-(bromomethyl)benzoate (687 mg, 3 mmol). Aminolysis of the methyl ester by means of NH<sub>3</sub>/MeOH was performed prior to final deprotection. Yield: 44 mg (8.2%). ESI-MS: <i>m</i>/<i>z</i> 357.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 4.93 min (98.7%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.95–4.12 (m, 4H), 4.24–4.47 (m, 2H), 7.42–7.68 (m, 6H), 7.78–8.18 (m, 6H), 9.24–9.42 (m, 1H), 10.56–10.71 (m, 1H). HRMS: <i>m</i>/<i>z</i> 379.1377 [M + Na]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>20</sub>N<sub>4</sub>NaO<sub>4</sub><sup>+</sup>, 379.1377.</div></div><div id="sec4_7_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-[bis[(3-Cyanophenyl)methyl]amino]-ethanehydroxamic Acid (<b>8e</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 996 mg, 3 mmol) and 3-(bromomethyl)benzonitrile (1176 mg, 6 mmol). Yield: 52 mg (5.7%). ESI-MS: <i>m</i>/<i>z</i> 321.2 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 11.63 min (98.2%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.09 (s, 3H), 3.72–3.89 (m, 4H), 7.53–7.62 (m, 2H), 7.68–7.83 (m, 4H), 7.90–7.98 (m, 2H), 10.62 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 321.1344 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>, 321.1346.</div></div><div id="sec4_7_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-[bis(1,3-Benzodioxol-5-ylmethyl)amino]-ethanehydroxamic Acid (<b>8f</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol) and 5-(bromomethyl)-1,3-benzodioxole (473 mg, 2.2 mmol). Yield: 56 mg (11.9%). ESI-MS: <i>m</i>/<i>z</i> 359.1 [M + H]<sup>+</sup>. HPLC (gradient 2): rt 10.03 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.40 (s, 2H), 4.12 (br s, 3H), 4.33 (br s, 1H), 6.06–6.07 (m, 4H), 6.96–7.03 (m, 4H), 7.09–7.13 (m, 2H), 9.51 (br s, 1H), 10.67–10.85 (m, 1H). HRMS: <i>m</i>/<i>z</i> 359.1239 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>6</sub><sup>+</sup>, 359.1238.</div></div><div id="sec4_7_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-[bis[(3,5-Difluoro-4-hydroxy-phenyl)methyl]-amino]ethanehydroxamic Acid (<b>8g</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol), 3,5-difluoro-4-methoxybenzyl bromide (522 mg, 2.2 mmol), and triethylamine (836 μL, 6 mmol, 4 equiv) as described above. Yield: 105 mg (27.7%). ESI-MS: <i>m</i>/<i>z</i> 374.9 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.47 min (98.6%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.15–3.23 (m, 2H), 3.78–3.99 (m, 4H), 7.12–7.23 (m, 4H), 10.30 (br s, 2H), 10.68 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 375.0957 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 375.0962.</div></div><div id="sec4_7_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-[bis[(2,4-Difluoro-3-hydroxy-phenyl)methyl]amino]ethanehydroxamic Acid (<b>8h</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <b>4a</b> (525 mg, 1.5 mmol, 1 equiv) and 2,4-difluoro-3-methoxybenzyl bromide (782 mg, 3.3 mmol, 2.2 equiv). Yield: 102 mg (18.2%). ESI-MS: <i>m</i>/<i>z</i> 374.9 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.47 min (98.6%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.15–3.23 (m, 2H), 3.78–3.99 (m, 4H), 7.12–7.23 (m, 4H), 10.30 (br s, 2H), 10.68 (br s,1H) HRMS: <i>m</i>/<i>z</i> 375.0959 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 375.0962.</div></div><div id="sec4_7_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-[bis[(4-Chloro-2-fluoro-3-hydroxy-phenyl)methyl]amino]ethanehydroxamic Acid (<b>8i</b>)</h4><div class="NLM_p last">The compound was synthesized starting from <b>4a</b> (280 mg, 0.8 mmol, 1 equiv), 4-chloro-2-fluoro-3-methoxybenzyl bromide (446 mg, 1.8 mmol, 2.2 equiv), and triethylamine (446 μL, 3.2 mmol, 4 equiv) as described above. Yield: 110 mg (33.8%). ESI-MS: <i>m</i>/<i>z</i> 407.3 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 11.33 min (97.8%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.11 (s, 2H), 3.86 (s, 4H), 6.94–6.99 (m, 2H), 7.16–7.19 (m, 2H), 10.30–10.44 (m, 3H). HRMS: <i>m</i>/<i>z</i> 407.0368 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>15</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 407.0371.</div></div><div id="sec4_7_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-[bis[(2,6-Difluoro-3-hydroxy-phenyl)methyl]-amino]ethanehydroxamic Acid (<b>8j</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol) and 2,6-difluoro-3-methoxybenzyl bromide (522 mg, 2.2 mmol). Yield: 73 mg (19.6%). ESI-MS: <i>m</i>/<i>z</i> 375.4 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 9.52 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.08 (s, 2H), 3.85–3.92 (m, 4H), 6.83–6.93 (m, 4H), 9.88 (br s, 2H), 10.05 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 375.0957 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 375.0962.</div></div><div id="sec4_7_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-[bis[(4-Fluoro-3-hydroxy-phenyl)methyl]-amino]ethanehydroxamic Acid (<b>8k</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol) and 4-bromomethyl-1-fluoro-2-methoxybenzene (482 mg, 2.2 mmol). Yield: 59 mg (17.5%). ESI-MS: <i>m</i>/<i>z</i> 339.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 7.36 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.28 (s, 2H), 3.72–4.11 (m, 4H), 6.87–6.98 (m, 2H), 7.04–7.24 (m, 4H), 8.70–9.58 (m, 1H), 9.80–10.27 (m, 2H), 10.53–10.84 (m, 1H). HRMS: <i>m</i>/<i>z</i> 339.1146 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>17</sub>F<sub>2</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 339.1151.</div></div><div id="sec4_7_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 2-[bis[(2-Fluoro-3-hydroxy-phenyl)methyl]-amino]ethanehydroxamic Acid (<b>8l</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 262 mg, 0.78 mmol) and 2-fluoro-3-methoxybenzyl bromide (379 mg, 1.7 mmol). Yield: 63 mg (24.3%). ESI-MS: <i>m</i>/<i>z</i> 339.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 6.83 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.20 (s, 2H), 3.97 (s, 4H), 6.86–7.08 (m, 6H), 9.20–9.42 (m, 1H), 9.71–10.14 (m, 2H), 10.51 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 339.1147 [M + H]<sup>+</sup>; calcd for C<sub>16</sub>H<sub>17</sub>F<sub>2</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 339.1151.</div></div><div id="sec4_7_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-[bis[(2,4-Difluoro-3-methoxy-phenyl)methyl]-amino]ethanehydroxamic Acid (<b>8m</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 332 mg, 1 mmol) and 2,4-difluoro-3-methoxybenzyl bromide (522 mg, 2.2 mmol). Yield: 26 mg (6.4%). ESI-MS: <i>m</i>/<i>z</i> 403.0 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 14.45 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.18 (s, 2H), 3.82 (s, 4H), 3.89 (s, 6H), 7.08–7.13 (m, 2H), 7.20–7.25 (m, 2H), 10.43 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 403.1277 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>18</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 403.1275.</div></div><div id="sec4_7_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-[bis[(4-Chloro-2-fluoro-3-methoxy-phenyl)-methyl]amino]ethanehydroxamic acid (<b>8n</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 298 mg, 0.9 mmol) and 4-chloro-2-fluoro-3-methoxybenzyl bromide (448 mg, 1.8 mmol). Yield: 30 mg (9.3%). ESI-MS: <i>m</i>/<i>z</i> 321.0 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 12.83 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.05 (s, 2H), 3.84 (s, 4H), 7.61–7.63 (m, 4H), 7.80–7.83 (m, 4H), 10.51 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 435.0680 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 435.0684.</div></div><div id="sec4_7_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-[bis[[3-(Difluoromethoxy)phenyl]methyl]-amino]ethanehydroxamic Acid (<b>8o</b>)</h4><div class="NLM_p last">The compound was synthesized from 2-amino-<i>N</i>-trityloxy-acetamide (<b>4a</b>, 498 mg, 1.5 mmol) and 3-(difluoromethoxy)benzyl bromide (756 mg, 3.2 mmol). Yield: 58 mg (10%). ESI-MS: <i>m</i>/<i>z</i> 403.1 [M + H]<sup>+</sup>. HPLC (gradient 1): rt 13.68 min (100%). <sup>1</sup>H NMR, 400 MHz, DMSO-<i>d</i><sub>6</sub>: δ 3.16 (s, 2H), 3.78–3.95 (m, 4H), 7.04–7.47 (m, 9H), 10.61 (br s, 1H). HRMS: <i>m</i>/<i>z</i> 403.1269 [M + H]<sup>+</sup>; calcd for C<sub>18</sub>H<sub>19</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub><sup>+</sup>, 403.1275.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Docking</h3><div class="NLM_p last">For the molecular docking of compounds <b>6a</b>, <b>8a</b>, and <b>8i</b>, the pdb file <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a> (<a href="http://www.rcsb.org" class="ext-link">www.rcsb.org</a>) was used as receptor, with a manual replacement of Cd<sup>2+</sup> by Zn<sup>2+</sup>. The docking procedure was set up in GOLD (v5.5). The active site was defined by the Zn<sup>2+</sup> within a radius of 10 Å. The ligands were utilized in their deprotonated forms. For each compound, 20 docking runs were performed, with a set search efficacy of 100%, and ChemScore was used as the scoring function. The resulting solutions for each ligand were imported and visualized in MOE (ChemicalComputingGroup, version 2016.08). For <b>6a</b>, two solutions with a plausible Zn-binding were manually selected, which represented the two reasonable possible binding modes of the compound. For <b>8a</b> and <b>8i</b>, the top ranked solutions were selected, showing a reasonable binding mode, respectively.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Enzymatic Assays</h3><div class="NLM_p">Recombinant human meprin β was expressed in yeast and characterized as previously described.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Recombinant human meprin α was expressed and purified from insect cells (S2) and characterized analogously. MMP2, 9, 13, and ADAM10 and 17 were purchased from a commercial vendor (R&D systems). MMPs were activated prior to measurement by APMA (<i>p</i>-aminophenylmercuric acetate) treatment according to the manufacturer’s instructions. The protease domain of BMP1 was expressed in <i>E. coli</i> as inclusion bodies and refolded according to Mac Sweeney et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p last">The determination of enzymatic activity was based on the cleavage of internally quenched peptide substrates (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_001.pdf" class="ext-link">Supporting Information</a>). A typical assay of 250 μL total volume measured in black 96-well plates consisted of 100 μL of buffer, 50 μL of enzyme at a final concentration of 5 × 10<sup>–9</sup> to 5 × 10<sup>–10</sup> M (one concentration in the mentioned range, depending on the activity of the respective enzyme batch; in case of BMP-1, high enzyme concentrations had to be applied, 7 × 10<sup>–7</sup> M), 50 μLof substrate (in buffer, 0.5% DMSO), and 50 μL of inhibitor solution (in buffer, 1% DMSO). In the case of 125 μL assay volume (black 96 half area well plates), all volumes were cut in half. Enzymatic activity of ADAMs was measured in 384-well plates with 60 μL total assay volume consisting of 20 μL of inhibitor, 20 μL of buffer, 10 μL of enzyme, and 10 μL of substrate. To ensure reproducibility, the parameters were determined at least in duplicates independently on separate days. For IC<sub>50</sub> values, the influence of 12 inhibitor concentrations ranging from 0 to 5 × 10<sup>–5</sup> M on the enzymatic activity was investigated in the presence of one standard substrate concentration (10 μM). Initial velocities were determined and converted into concentration units applying a standard curve obtained after complete conversion of different substrate concentrations under assay conditions. All measurements were performed using a fluorescence plate reader (FLUOstar OPTIMA, BMG Labtech) at 30 °C. The excitation/emission wavelength was 340/410 nm. The kinetic data was evaluated using GraphPad Prism (version 5.04, GraphPad Software). The inhibition of Cyctochromes 3A4 and 2A9 was analyzed using commercial assays (Cyp3A4 Blue and Cyp2C9 Blue, Life Technologies) according to the manufacturer’s protocols and guidelines.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Cell Viability Assay</h3><div class="NLM_p last">Cell viability was assessed in human hepatocellular carcinoma cell line Hep-G2 and in human neuroblastoma cell line SH-SY5Y. Hep-G2 cells were cultivated in RPMI1640 (ThermoFisher) supplemented with 10% FBS, and SH-SY5Y cells were cultivated in DMEM (high-glucose, pyruvate) (ThermoFisher) also supplemented with 10% FBS in a humidified atmosphere of 37 °C and 5% CO<sub>2</sub> according to standard cell culture procedures. For the assay, cells were plated in 96-well microtiter plates (Greiner bio-one) at densities of 50,000 cells/well (Hep-G2) and 60,000 cells/well (SH-SY5Y), respectively. After 24 h, compounds dissolved in DMSO are added to fresh medium at concentrations of 30 and 100 μM (final concentration of DMSO: 1% (v/v)) and applied to the cells for another 24 h. On the next day, cellular viability is determined using the CytoTox-ONE kit (Promega) based on the viability in control wells incubated with culture medium and 1% DMSO. The CytoTox-ONE assay kit was used according to the manufacturer’s recommendations.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Determination of Pharmacokinetic Parameters</h3><div class="NLM_p last">Pharmacokinetic parameters were determined in catheterized Sprague–Dawley rats after approval by the animal welfare committee of the local state of Saxony-Anhalt (approval number: 203.m-42502-2-1361). Briefly, male RjHan:SD rats were purchased from Janvier Laboratories and housed in pairs for a minimum of 2 weeks prior to surgery. For the catheter placement, rats were anesthetized using Isofluran, and a 2 cm incision was made at the ventral part of the neck. Catheters were placed in the right <i>V. jugularis</i> and left <i>A. carotis communis</i> and tunneled to the dorsal neck. Surgical incisions were closed by sutures. Analgesia was done by subcutaneously injecting Meloxicam 1 mg/kg during and for 2 days after surgery. After a recovery period of several days, test compounds were applied orally (p.o.; <b>8a</b>, 5 mg/kg; <b>8h,i</b>, 10 mg/kg), intravenously (i.v., 6 mg/kg), or intraperitoneally (i.p., 10 mg/kg). In general, the catheter placed in the <i>V. jugularis</i> was used for i.v. applications. Blood (50–70 μL per time point) was collected from the catheter placed in the <i>A. carotis communis</i>. Full pharmacokinetic profiles were obtained from single animals over a time course of 8 h (12 sampling points). Compound levels were determined in Heparin-plasma using LC–MS. From compound concentration/time curves, parameters <i>C</i><sub>max</sub> and <i>t</i><sub>max</sub> were derived. Volume of distribution <i>V</i><sub>d</sub> (p.o.) was calculated according to following formula: <i>V</i><sub>d</sub> = <i>D</i>/<i>C</i><sub>p0</sub>, where <i>D</i> is the applied dose in mg and <i>C</i><sub>p</sub><sub>0</sub> is the plasma concentration at time point 0. Terminal half-life was calculated from the terminal phase of the compound profile by calculating ln 2/<i>k</i><sub>e</sub>, where <i>k</i><sub>e</sub> represents the elimination constant calculated according to the following formula: <i>k</i><sub>e</sub> = (ln <i>C</i><sub>p1</sub> – ln <i>C</i><sub>p2</sub>)/(<i>t</i><sub>2</sub> – <i>t</i><sub>1</sub>). In this regard, <i>C</i><sub>p1</sub> and <i>C</i><sub>p2</sub> are measured compound concentrations at given time points <i>t</i><sub>1</sub> and <i>t</i><sub>2</sub>. Bioavailability was calculated by the following formula: <i>F</i>(%) = ((<i>D</i><sub>i.v.</sub> × AUC<sub>n.i.v.</sub>)/(<i>D</i><sub>n.i.v.</sub> × AUC<sub>i.v.</sub>)) × 100, where <i>D</i> is the applied dose in micrograms, AUC is the area under the curve in μg/mL·h, i.v. is intravenous application, and n.i.v. is the nonintravenous application. AUCs were calculated using the AUC function in GraphPad Prism 5 (GraphPad Software Inc.). Total body clearance (CL) was calculated according to the following formula: CL = <i>D</i>/AUC<sub>0-infinity</sub>. The volume of distribution in steady-state (<i>V</i><sub>dss</sub>) was calculated according to the following formula: <i>V</i><sub>dss</sub>=(<i>D</i> × AUMC<sub>0-inf</sub>)/(AUC<sub>0-inf</sub>)<sup>2</sup>, where <i>D</i> is the applied dose (i.v.) and AUMC is the area under the moment curve.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00330" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00330" class="ext-link">10.1021/acs.jmedchem.8b00330</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Analytical data of intermediates, supplementary figure, and substrates used for enzymatic assays (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_001.pdf">jm8b00330_si_001.pdf (594.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_002.csv">jm8b00330_si_002.csv (2.33 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00330" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Ramsbeck</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4966-142X" title="Orcid link">http://orcid.org/0000-0002-4966-142X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5c383d32353930722e3d312f3e393f371c352635723a2e3d293234333a392e723839"><span class="__cf_email__" data-cfemail="3a5e5b54535f5614485b5749585f59517a534053145c485b4f5452555c5f48145e5f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mirko Buchholz</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#83eeeaf1e8ecade1f6e0ebebeceff9c3eaf9eaade5f1e2f6edebece5e6f1ade7e6"><span class="__cf_email__" data-cfemail="45282c372e2a6b2730262d2d2a293f052c3f2c6b233724302b2d2a2320376b2120">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antje Hamann</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Georg Richter</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dagmar Schlenzig</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefanie Geissler</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vera Nykiel</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Holger Cynis</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephan Schilling</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer Institute for
Cell Therapy
and Immunology IZI, Department of Drug Design
and Target Validation MWT, Biocenter, Weinbergweg 22, 06120 Halle (Saale), Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Mercedes Scharfe (IZI-MWT), Dr. Andrea Porzel (Leibniz Institute of Plant Biochemistry, Halle), and Dr. Christian Ihling (Martin-Luther University, Halle-Wittenberg) for their excellent technical support. We gratefully acknowledge the funding of this project by the federal state of Saxony-Anhalt, Germany.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVATIONS USED</td></tr><tr><td class="NLM_term">ADAM</td><td class="NLM_def"><p class="first last">A disintegrin and metalloprotease</p></td></tr><tr><td class="NLM_term">APP</td><td class="NLM_def"><p class="first last">amyloid precursor protein</p></td></tr><tr><td class="NLM_term">BACE1</td><td class="NLM_def"><p class="first last">beta-site amyloid precursor protein cleaving enzyme 1</p></td></tr><tr><td class="NLM_term">BMP-1</td><td class="NLM_def"><p class="first last">bone morphogenetic protein 1</p></td></tr><tr><td class="NLM_term">DBU</td><td class="NLM_def"><p class="first last">1,8-diazabicyclo[5.4.0]undec-7-en</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">GABA</td><td class="NLM_def"><p class="first last">gamma-amino butyric acid</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matrix metalloprotease</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NNGH</td><td class="NLM_def"><p class="first last"><i>N</i>-isobutyl-<i>N</i>-(4-methoxyphenylsulfonyl)glycyl hydroxamic acid</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">TBTU</td><td class="NLM_def"><p class="first last">2-(1<i>H</i>-benzotriazole-1-yl)-1,1,3,3-tetramethylaminium tetrafluoroborate</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">trimethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TIS</td><td class="NLM_def"><p class="first last">triisopropylsilane</p></td></tr><tr><td class="NLM_term">i.v.</td><td class="NLM_def"><p class="first last">intravenous</p></td></tr><tr><td class="NLM_term">i.p.</td><td class="NLM_def"><p class="first last">intraperitoneal</p></td></tr><tr><td class="NLM_term">p.o.</td><td class="NLM_def"><p class="first last">peroral</p></td></tr><tr><td class="NLM_term">n.t.</td><td class="NLM_def"><p class="first last">not tested</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">Emerging principles in protease-based drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">701</span>, <span class="refDoi"> DOI: 10.1038/nrd3053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fnrd3053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=20811381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSisbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=690-701&author=M.+Dragauthor=G.+S.+Salvesen&title=Emerging+principles+in+protease-based+drug+discovery&doi=10.1038%2Fnrd3053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging principles in protease-based drug discovery</span></div><div class="casAuthors">Drag, Marcin; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">690-701</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteases have an important role in many signalling pathways, and represent potential drug targets for diseases ranging from cardiovascular disorders to cancer, as well as for combating many parasites and viruses.  Although inhibitors of well-established protease targets such as angiotensin-converting enzyme and HIV protease have shown substantial therapeutic success, developing drugs for new protease targets has proved challenging in recent years.  This in part could be due to issues such as the difficulty of achieving selectivity when targeting protease active sites.  This Perspective discusses the general principles in protease-based drug discovery, highlighting the lessons learned and the emerging strategies, such as targeting allosteric sites, which could help harness the therapeutic potential of new protease targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz3PQDA1pnT7Vg90H21EOLACvtfcHk0liqL-5pYSimEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSisbzP&md5=28a9571473c3f753027665b77d3257d0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3053%26sid%3Dliteratum%253Aachs%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DEmerging%2520principles%2520in%2520protease-based%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D690%26epage%3D701%26doi%3D10.1038%2Fnrd3053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleifeld, O.</span></span> <span> </span><span class="NLM_article-title">Towards third generation matrix metalloproteinase inhibitors for cancer therapy</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">946</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6603043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fsj.bjc.6603043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=16538215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFGqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2006&pages=941-946&author=C.+M.+Overallauthor=O.+Kleifeld&title=Towards+third+generation+matrix+metalloproteinase+inhibitors+for+cancer+therapy&doi=10.1038%2Fsj.bjc.6603043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Towards third generation matrix metalloproteinase inhibitors for cancer therapy</span></div><div class="casAuthors">Overall, C. M.; Kleifeld, O.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-946</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The failure of matrix metalloproteinase (MMP) inhibitor drug clin. trials in cancer was partly due to the inadvertent inhibition of MMP antitargets that counterbalanced the benefits of MMP target inhibition.  We explore how MMP inhibitor drugs might be developed to achieve potent selectivity for validated MMP targets yet therapeutically spare MMP antitargets that are crit. in host protection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWaFApPMIKRLVg90H21EOLACvtfcHk0liqL-5pYSimEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFGqt7g%253D&md5=5e672a6e9495ec2e283856edc6e61a60</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6603043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6603043%26sid%3Dliteratum%253Aachs%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DKleifeld%26aufirst%3DO.%26atitle%3DTowards%2520third%2520generation%2520matrix%2520metalloproteinase%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2006%26volume%3D94%26spage%3D941%26epage%3D946%26doi%3D10.1038%2Fsj.bjc.6603043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenbroucke, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span> <span> </span><span class="NLM_article-title">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd4390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fnrd4390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=25376097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=904-927&author=R.+E.+Vandenbrouckeauthor=C.+Libert&title=Is+there+new+hope+for+therapeutic+matrix+metalloproteinase+inhibition%3F&doi=10.1038%2Fnrd4390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span></div><div class="casAuthors">Vandenbroucke, Roosmarijn E.; Libert, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">904-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent endopeptidases that form a family of 24 members in mammals.  Evidence of the pathol. roles of MMPs in various diseases, combined with their druggability, has made them attractive therapeutic targets.  Initial drug discovery efforts focused on the roles of MMPs in cancer progression, and more than 50 MMP inhibitors have been investigated in clin. trials in various cancers.  However, all of these trials failed.  Reasons for failure include the lack of inhibitor specificity and insufficient knowledge about the complexity of the disease biol.  MMPs are also known to be involved in several inflammatory processes, and there are new therapeutic opportunities for MMP inhibitors to treat such diseases.  In this Review, we discuss the recent advances made in understanding the role of MMPs in inflammatory diseases and the therapeutic potential of MMP inhibition in those conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpua2GBt8P5b7Vg90H21EOLACvtfcHk0liqL-5pYSimEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I&md5=032265fe59e9f7567e4d3c7d072b3cde</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4390%26sid%3Dliteratum%253Aachs%26aulast%3DVandenbroucke%26aufirst%3DR.%2BE.%26aulast%3DLibert%26aufirst%3DC.%26atitle%3DIs%2520there%2520new%2520hope%2520for%2520therapeutic%2520matrix%2520metalloproteinase%2520inhibition%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D904%26epage%3D927%26doi%3D10.1038%2Fnrd4390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenspan, D. S.</span></span> <span> </span><span class="NLM_article-title">The bone morphogenetic protein 1/tolloid-like metalloproteinases</span>. <i>Matrix Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">523</span>, <span class="refDoi"> DOI: 10.1016/j.matbio.2007.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.matbio.2007.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=17560775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=508-523&author=D.+R.+Hopkinsauthor=S.+Kelesauthor=D.+S.+Greenspan&title=The+bone+morphogenetic+protein+1%2Ftolloid-like+metalloproteinases&doi=10.1016%2Fj.matbio.2007.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The bone morphogenetic protein 1/Tolloid-like metalloproteinases</span></div><div class="casAuthors">Hopkins, Delana R.; Keles, Sunduz; Greenspan, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Matrix Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">508-523</span>CODEN:
                <span class="NLM_cas:coden">MTBOEC</span>;
        ISSN:<span class="NLM_cas:issn">0945-053X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A decade ago, bone morphogenetic protein 1 (BMP1) was shown to provide the activity necessary for proteolytic removal of the C-propeptides of procollagens I-III, precursors of the major fibrillar collagens.  Subsequent studies have shown BMP1 to be the prototype of a small group of extracellular metalloproteinases that play manifold roles in regulating the formation of the extracellular matrix (ECM).  Soon after the initial cloning of BMP1, genetic studies showed the related Drosophila proteinase, Tolloid (TLD), to be necessary for the formation of the dorsal-ventral axis in early embryogenesis.  It is now clear that the BMP1/TLD-like proteinases, conserved in species ranging from Drosophila to humans, act in dorsal-ventral patterning via activation of transforming growth factor β (TGFβ)-like proteins BMP2, BMP4 (vertebrates), and decapentaplegic (arthropods).  More recently, it has become apparent that the BMP1/TLD-like proteinases are activators of a broader subset of the TGFβ superfamily of proteins, with implications that these proteinases may be key in orchestrating the formation of ECM with growth factor activation and BMP signaling in morphogenetic processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxFod_zt9QlLVg90H21EOLACvtfcHk0liqL-5pYSimEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qs7vM&md5=b6de6f0fcaddbfe9b5633e2d5e6cdcd7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.matbio.2007.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.matbio.2007.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DD.%2BR.%26aulast%3DKeles%26aufirst%3DS.%26aulast%3DGreenspan%26aufirst%3DD.%2BS.%26atitle%3DThe%2520bone%2520morphogenetic%2520protein%25201%252Ftolloid-like%2520metalloproteinases%26jtitle%3DMatrix%2520Biol.%26date%3D2007%26volume%3D26%26spage%3D508%26epage%3D523%26doi%3D10.1016%2Fj.matbio.2007.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Meprins, membrane-bound and secreted astacin metalloproteinases</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.mam.2008.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=18783725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=309-328&author=E.+E.+Sterchiauthor=W.+St%C3%B6ckerauthor=J.+S.+Bond&title=Meprins%2C+membrane-bound+and+secreted+astacin+metalloproteinases&doi=10.1016%2Fj.mam.2008.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Meprins, membrane-bound and secreted astacin metalloproteinases</span></div><div class="casAuthors">Sterchi, Erwin E.; Stocker, Walter; Bond, Judith S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-328</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The astacins are a subfamily of the metzincin superfamily of metalloproteinases.  The first to be characterized was the crayfish enzyme astacin.  To date more than 200 members of this family have been identified in species ranging from bacteria to humans.  Astacins are involved in developmental morphogenesis, matrix assembly, tissue differentiation and digestion.  Family members include the procollagen C-proteinase (BMP1, bone morphogenetic protein 1), tolloid and mammalian tolloid-like, HMP (Hydra vulgaris metalloproteinase), sea urchin BP10 (blastula protein) and SPAN (Strongylocentrotus purpuratus astacin), the hatching' subfamily comprising alveolin, ovastacin, LCE, HCE ( low' and high' choriolytic enzymes), nephrosin (from carp head kidney), UVS.2 from frog, and the meprins.  In the human and mouse genomes, there are six astacin family genes (two meprins, three BMP1/tolloid-like, one ovastacin), but in Caenorhabditis elegans there are 40.  Meprins are the only astacin proteinases that function on the membrane and extracellularly by virtue of the fact that they can be membrane-bound or secreted.  They are unique in their domain structure and covalent subunit dimerization, oligomerization propensities, and expression patterns.  They are normally highly regulated at the transcriptional and post-translational levels, localize to specific membranes or extracellular spaces, and can hydrolyze biol. active peptides, cytokines, extracellular matrix (ECM) proteins and cell-surface proteins.  The in vivo substrates of meprins are unknown, but the abundant expression of these proteinases in the epithelial cells of the intestine, kidney and skin provide clues to their functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKeid0CPGYrLVg90H21EOLACvtfcHk0lis-pLCEQSElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnO&md5=ce85a8d627e13b8149e899de77ebbe36</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DMeprins%252C%2520membrane-bound%2520and%2520secreted%2520astacin%2520metalloproteinases%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2008%26volume%3D29%26spage%3D309%26epage%3D328%26doi%3D10.1016%2Fj.mam.2008.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turtle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W.-B.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of procollagen C-proteinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">7397</span>– <span class="NLM_lpage">7401</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.bmcl.2012.10.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=7397-7401&author=E.+Turtleauthor=N.+Chowauthor=C.+Yangauthor=S.+Sosaauthor=U.+Bauerauthor=M.+Brennerauthor=D.+Solow-Corderoauthor=W.-B.+Ho&title=Design+and+synthesis+of+procollagen+C-proteinase+inhibitors&doi=10.1016%2Fj.bmcl.2012.10.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DTurtle%26aufirst%3DE.%26aulast%3DChow%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DSosa%26aufirst%3DS.%26aulast%3DBauer%26aufirst%3DU.%26aulast%3DBrenner%26aufirst%3DM.%26aulast%3DSolow-Cordero%26aufirst%3DD.%26aulast%3DHo%26aufirst%3DW.-B.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520procollagen%2520C-proteinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D7397%26epage%3D7401%26doi%3D10.1016%2Fj.bmcl.2012.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turtle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W.-B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of procollagen C-proteinase: fibrosis and beyond</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1517/13543776.14.8.1185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1517%2F13543776.14.8.1185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=1185-1197&author=E.+D.+Turtleauthor=W.-B.+Ho&title=Inhibition+of+procollagen+C-proteinase%3A+fibrosis+and+beyond&doi=10.1517%2F13543776.14.8.1185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of procollagen C-proteinase: fibrosis and beyond</span></div><div class="casAuthors">Turtle, Eric D.; Ho, Wen-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1185-1197</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Procollagen C-proteinase (PCP) plays a key role in fibrosis.  The activity of PCP is necessary to convert sol. procollagens into fully formed collagen fibrils.  Fibrosis is a pathol. condition in which the normal wound healing process goes awry, culminating in the excessive prodn. and deposition of collagen, a major component of scar tissue.  The persistent formation of scar tissue hinders proper tissue function and can lead to organ failure in a wide range of fibrotic diseases.  PCP is recognized as an attractive target for antifibrotic therapies.  This review provides a discussion of the therapeutic potential of PCP inhibition and an examn. of relevant patent literature, primarily focused on the identification of small mol. inhibitors of PCP for the prevention of dermal scarring and fibrotic disease.  The emerging roles of PCP in biol. processes other than collagen biosynthesis are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy4W1KYeLHOLVg90H21EOLACvtfcHk0lis-pLCEQSElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSjurk%253D&md5=460843d020a76e59297dd89120feb656</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F13543776.14.8.1185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.14.8.1185%26sid%3Dliteratum%253Aachs%26aulast%3DTurtle%26aufirst%3DE.%2BD.%26aulast%3DHo%26aufirst%3DW.-B.%26atitle%3DInhibition%2520of%2520procollagen%2520C-proteinase%253A%2520fibrosis%2520and%2520beyond%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2004%26volume%3D14%26spage%3D1185%26epage%3D1197%26doi%3D10.1517%2F13543776.14.8.1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">The metalloproteases meprin α and meprin β: unique enzymes in inflammation, neurodegeneration, cancer and fibrosis</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1042/BJ20121751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1042%2FBJ20121751" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2013&pages=253-264&author=C.+Broderauthor=C.+Becker-Pauly&title=The+metalloproteases+meprin+%CE%B1+and+meprin+%CE%B2%3A+unique+enzymes+in+inflammation%2C+neurodegeneration%2C+cancer+and+fibrosis&doi=10.1042%2FBJ20121751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20121751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121751%26sid%3Dliteratum%253Aachs%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DThe%2520metalloproteases%2520meprin%2520%25CE%25B1%2520and%2520meprin%2520%25CE%25B2%253A%2520unique%2520enzymes%2520in%2520inflammation%252C%2520neurodegeneration%252C%2520cancer%2520and%2520fibrosis%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D450%26spage%3D253%26epage%3D264%26doi%3D10.1042%2FBJ20121751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">Meprin α and meprin β: Procollagen proteinases in health and disease</span>. <i>Matrix Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>44–46</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.matbio.2015.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.matbio.2015.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=25617491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFChsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44%E2%80%9346&publication_year=2015&pages=7-13&author=J.+Proxauthor=P.+Arnoldauthor=C.+Becker-Pauly&title=Meprin+%CE%B1+and+meprin+%CE%B2%3A+Procollagen+proteinases+in+health+and+disease&doi=10.1016%2Fj.matbio.2015.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Meprin α and meprin β: Procollagen proteinases in health and disease</span></div><div class="casAuthors">Prox, Johannes; Arnold, Philipp; Becker-Pauly, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Matrix Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44-46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-13</span>CODEN:
                <span class="NLM_cas:coden">MTBOEC</span>;
        ISSN:<span class="NLM_cas:issn">0945-053X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Metalloproteases meprin α and meprin β were recently discovered as procollagen proteinases, capable of cleaving off the globular C- and N-terminal prodomains of fibrillar collagen type I and type III.  This proteolytic process is indeed sufficient to induce collagen fibril assembly as visualized by transmission electron microscopy.  The biol. relevance was demonstrated with the help of meprin α and meprin β knock-out mice, which exhibit decreased collagen deposition in skin resulting in impaired tensile strength.  On the other hand, overexpression of meprin metalloproteases was found under fibrotic conditions in the skin (keloids) and the lung (pulmonary hypertension).  Thus, regulation of meprin activity by specific inhibition to reduce collagen maturation might be a suitable approach for the treatment of certain pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlUbO2xzhcbVg90H21EOLACvtfcHk0lis-pLCEQSElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFChsrg%253D&md5=85291d65f92aceac7c6e8e0aa1e224f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.matbio.2015.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.matbio.2015.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DProx%26aufirst%3DJ.%26aulast%3DArnold%26aufirst%3DP.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DMeprin%2520%25CE%25B1%2520and%2520meprin%2520%25CE%25B2%253A%2520Procollagen%2520proteinases%2520in%2520health%2520and%2520disease%26jtitle%3DMatrix%2520Biol.%26date%3D2015%26volume%3D44%25E2%2580%259346%26spage%3D7%26epage%3D13%26doi%3D10.1016%2Fj.matbio.2015.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadon-Le Goff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konerding, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koudelka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tholey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulmes, D. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moali, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">Metalloproteases meprin and meprin are C- and N-procollagen proteinases important for collagen assembly and tensile strength</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">14219</span>– <span class="NLM_lpage">14224</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305464110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1073%2Fpnas.1305464110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=23940311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=14219-14224&author=C.+Broderauthor=P.+Arnoldauthor=S.+Vadon-Le+Goffauthor=M.+A.+Konerdingauthor=K.+Bahrauthor=S.+Mullerauthor=C.+M.+Overallauthor=J.+S.+Bondauthor=T.+Koudelkaauthor=A.+Tholeyauthor=D.+J.+S.+Hulmesauthor=C.+Moaliauthor=C.+Becker-Pauly&title=Metalloproteases+meprin+and+meprin+are+C-+and+N-procollagen+proteinases+important+for+collagen+assembly+and+tensile+strength&doi=10.1073%2Fpnas.1305464110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metalloproteases meprin α and meprin β are C- and N-procollagen proteinases important for collagen assembly and tensile strength</span></div><div class="casAuthors">Broder, Claudia; Arnold, Philipp; Vadon-Le Goff, Sandrine; Konerding, Moritz A.; Bahr, Kerstin; Mueller, Stefan; Overall, Christopher M.; Bond, Judith S.; Koudelka, Tomas; Tholey, Andreas; Hulmes, David J. S.; Moali, Catherine; Becker-Pauly, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">14219-14224,S14219/1-S14219/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Type I fibrillar collagen is the most abundant protein in the human body, crucial for the formation and strength of bones, skin, and tendon.  Proteolytic enzymes are essential for initiation of the assembly of collagen fibrils by cleaving off the propeptides.  We report that Mep1a-/- and Mep1b-/- mice revealed lower amts. of mature collagen I compared with WT mice and exhibited significantly reduced collagen deposition in skin, along with markedly decreased tissue tensile strength.  While exploring the mechanism of this phenotype, we found that cleavage of full-length human procollagen I heterotrimers by either meprin α or meprin β led to the generation of mature collagen mols. that spontaneously assembled into collagen fibrils.  Thus, meprin α and meprin β are unique in their ability to process and release both C- and N-propeptides from type I procollagen in vitro and in vivo and contribute to the integrity of connective tissue in skin, with consequent implications for inherited connective tissue disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplkClBQ8SJtrVg90H21EOLACvtfcHk0lic8d3Wesiibw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtrnP&md5=a326ff562fe4570a3625c58faf1aadfd</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305464110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305464110%26sid%3Dliteratum%253Aachs%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DArnold%26aufirst%3DP.%26aulast%3DVadon-Le%2BGoff%26aufirst%3DS.%26aulast%3DKonerding%26aufirst%3DM.%2BA.%26aulast%3DBahr%26aufirst%3DK.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26aulast%3DKoudelka%26aufirst%3DT.%26aulast%3DTholey%26aufirst%3DA.%26aulast%3DHulmes%26aufirst%3DD.%2BJ.%2BS.%26aulast%3DMoali%26aufirst%3DC.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DMetalloproteases%2520meprin%2520and%2520meprin%2520are%2520C-%2520and%2520N-procollagen%2520proteinases%2520important%2520for%2520collagen%2520assembly%2520and%2520tensile%2520strength%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D14219%26epage%3D14224%26doi%3D10.1073%2Fpnas.1305464110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biasin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wygrecka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brcic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klepetko, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olschewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwapiszewska, G.</span></span> <span> </span><span class="NLM_article-title">Meprin β contributes to collagen deposition in lung fibrosis</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">39969</span>, <span class="refDoi"> DOI: 10.1038/srep39969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fsrep39969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=28059112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=39969&author=V.+Biasinauthor=M.+Wygreckaauthor=L.+M.+Marshauthor=C.+Becker-Paulyauthor=L.+Brcicauthor=B.+Ghanimauthor=W.+Klepetkoauthor=A.+Olschewskiauthor=G.+Kwapiszewska&title=Meprin+%CE%B2+contributes+to+collagen+deposition+in+lung+fibrosis&doi=10.1038%2Fsrep39969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Meprin β contributes to collagen deposition in lung fibrosis</span></div><div class="casAuthors">Biasin, V.; Wygrecka, M.; Marsh, L. M.; Becker-Pauly, C.; Brcic, L.; Ghanim, B.; Klepetko, W.; Olschewski, A.; Kwapiszewska, G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39969</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lung fibrosis is a severe disease characterized by epithelial cell injury, inflammation and collagen deposition.  The metalloproteases meprinα and meprinβ have been shown to enhance collagen maturation and inflammatory cell infiltration via cleavage of cell-cell contact mols.; therefore we hypothesized that meprins could play a role in lung fibrosis.  An exhaustive characterization of bleomycin-treated meprinα, meprinβ and the double meprinsαβ knock-out (KO) with resp. wt-littermates was performed by using several different methods.  We obsd. no difference in lung function parameters and no change in inflammatory cells infiltrating the lung between wt and all meprins KO mice after 14 days bleomycin.  No difference in epithelial integrity as assessed by e-cadherin protein level was detected in bleomycin-treated lungs.  However, morphol. anal. in the bleomycin-treated mice revealed decrease collagen deposition and tissue d. in meprinβ KO, but not in meprinα and meprinαβ KO mice.  This finding was accompanied by localization of meprinβ to epithelial cells in regions with immature collagen in mice.  Similarly, in human IPF lungs meprinβ was mostly localized in epithelium.  These findings suggest that local environment triggers meprinβ expression to support collagen maturation.  In conclusion, our data demonstrate the in vivo relevance of meprinβ in collagen deposition in lung fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAeLWcsnM5_rVg90H21EOLACvtfcHk0lic8d3Wesiibw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlensQ%253D%253D&md5=7de6854a178a98d2da31810315a9a229</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsrep39969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39969%26sid%3Dliteratum%253Aachs%26aulast%3DBiasin%26aufirst%3DV.%26aulast%3DWygrecka%26aufirst%3DM.%26aulast%3DMarsh%26aufirst%3DL.%2BM.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DBrcic%26aufirst%3DL.%26aulast%3DGhanim%26aufirst%3DB.%26aulast%3DKlepetko%26aufirst%3DW.%26aulast%3DOlschewski%26aufirst%3DA.%26aulast%3DKwapiszewska%26aufirst%3DG.%26atitle%3DMeprin%2520%25CE%25B2%2520contributes%2520to%2520collagen%2520deposition%2520in%2520lung%2520fibrosis%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D39969%26doi%3D10.1038%2Fsrep39969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matters, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gailey, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Balance of meprin A and B in mice affects the progression of experimental inflammatory bowel disease</span>. <i>Am. J. Physiol. Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">G273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00504.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1152%2Fajpgi.00504.2009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2011&pages=G273-282&author=S.+Banerjeeauthor=G.+Jinauthor=S.+G.+Bradleyauthor=G.+L.+Mattersauthor=R.+D.+Gaileyauthor=J.+M.+Crismanauthor=J.+S.+Bond&title=Balance+of+meprin+A+and+B+in+mice+affects+the+progression+of+experimental+inflammatory+bowel+disease&doi=10.1152%2Fajpgi.00504.2009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00504.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00504.2009%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DBradley%26aufirst%3DS.%2BG.%26aulast%3DMatters%26aufirst%3DG.%2BL.%26aulast%3DGailey%26aufirst%3DR.%2BD.%26aulast%3DCrisman%26aufirst%3DJ.%2BM.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DBalance%2520of%2520meprin%2520A%2520and%2520B%2520in%2520mice%2520affects%2520the%2520progression%2520of%2520experimental%2520inflammatory%2520bowel%2520disease%26jtitle%3DAm.%2520J.%2520Physiol.%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2011%26volume%3D300%26spage%3DG273%26epage%3D282%26doi%3D10.1152%2Fajpgi.00504.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">The metalloprotease meprin β is an alternative β-secretase of APP</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">146</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2016.00159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.3389%2Ffnmol.2016.00159" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=146&author=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=The+metalloprotease+meprin+%CE%B2+is+an+alternative+%CE%B2-secretase+of+APP&doi=10.3389%2Ffnmol.2016.00159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2016.00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2016.00159%26sid%3Dliteratum%253Aachs%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DThe%2520metalloprotease%2520meprin%2520%25CE%25B2%2520is%2520an%2520alternative%2520%25CE%25B2-secretase%2520of%2520APP%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2017%26volume%3D9%26spage%3D146%26doi%3D10.3389%2Ffnmol.2016.00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Causevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jumpertz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Multhaup, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">The metalloprotease meprin generates amino terminal-truncated amyloid peptide species</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">33304</span>– <span class="NLM_lpage">33313</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.395608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fjbc.M112.395608" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=33304-33313&author=J.+Bienauthor=T.+Jeffersonauthor=M.+Causevicauthor=T.+Jumpertzauthor=L.+Munterauthor=G.+Multhaupauthor=S.+Weggenauthor=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=The+metalloprotease+meprin+generates+amino+terminal-truncated+amyloid+peptide+species&doi=10.1074%2Fjbc.M112.395608"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.395608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.395608%26sid%3Dliteratum%253Aachs%26aulast%3DBien%26aufirst%3DJ.%26aulast%3DJefferson%26aufirst%3DT.%26aulast%3DCausevic%26aufirst%3DM.%26aulast%3DJumpertz%26aufirst%3DT.%26aulast%3DMunter%26aufirst%3DL.%26aulast%3DMulthaup%26aufirst%3DG.%26aulast%3DWeggen%26aufirst%3DS.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DThe%2520metalloprotease%2520meprin%2520generates%2520amino%2520terminal-truncated%2520amyloid%2520peptide%2520species%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D33304%26epage%3D33313%26doi%3D10.1074%2Fjbc.M112.395608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenthaler, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">Generation of aggregation prone N-terminally truncated amyloid β peptides by meprin β depends on the sequence specificity at the cleavage site</span>. <i>Mol. Neurodegener.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1186/s13024-016-0084-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1186%2Fs13024-016-0084-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=26758977" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=4&author=C.+Sch%C3%B6nherrauthor=J.+Bienauthor=S.+Isbertauthor=R.+Wichertauthor=J.+Proxauthor=H.+Altmeppenauthor=S.+Kumarauthor=J.+Walterauthor=S.+F.+Lichtenthalerauthor=S.+Weggenauthor=M.+Glatzelauthor=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=Generation+of+aggregation+prone+N-terminally+truncated+amyloid+%CE%B2+peptides+by+meprin+%CE%B2+depends+on+the+sequence+specificity+at+the+cleavage+site&doi=10.1186%2Fs13024-016-0084-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2Fs13024-016-0084-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13024-016-0084-5%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DC.%26aulast%3DBien%26aufirst%3DJ.%26aulast%3DIsbert%26aufirst%3DS.%26aulast%3DWichert%26aufirst%3DR.%26aulast%3DProx%26aufirst%3DJ.%26aulast%3DAltmeppen%26aufirst%3DH.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DWalter%26aufirst%3DJ.%26aulast%3DLichtenthaler%26aufirst%3DS.%2BF.%26aulast%3DWeggen%26aufirst%3DS.%26aulast%3DGlatzel%26aufirst%3DM.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DGeneration%2520of%2520aggregation%2520prone%2520N-terminally%2520truncated%2520amyloid%2520%25CE%25B2%2520peptides%2520by%2520meprin%2520%25CE%25B2%2520depends%2520on%2520the%2520sequence%2520specificity%2520at%2520the%2520cleavage%2520site%26jtitle%3DMol.%2520Neurodegener.%26date%3D2016%26volume%3D11%26spage%3D4%26doi%3D10.1186%2Fs13024-016-0084-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lottaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yiallouros, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span> <span> </span><span class="NLM_article-title">Human meprin alpha and beta homo-oligomers: cleavage of basement membrane proteins and sensitivity to metalloprotease inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1042/bj20031163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1042%2Fbj20031163" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2004&pages=383-389&author=M.-N.+Kruseauthor=C.+Beckerauthor=D.+Lottazauthor=D.+K%C3%B6hlerauthor=I.+Yiallourosauthor=H.-W.+Krellauthor=E.+E.+Sterchiauthor=W.+St%C3%B6cker&title=Human+meprin+alpha+and+beta+homo-oligomers%3A+cleavage+of+basement+membrane+proteins+and+sensitivity+to+metalloprotease+inhibitors&doi=10.1042%2Fbj20031163"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2Fbj20031163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20031163%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DM.-N.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DLottaz%26aufirst%3DD.%26aulast%3DK%25C3%25B6hler%26aufirst%3DD.%26aulast%3DYiallouros%26aufirst%3DI.%26aulast%3DKrell%26aufirst%3DH.-W.%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26atitle%3DHuman%2520meprin%2520alpha%2520and%2520beta%2520homo-oligomers%253A%2520cleavage%2520of%2520basement%2520membrane%2520proteins%2520and%2520sensitivity%2520to%2520metalloprotease%2520inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2004%26volume%3D378%26spage%3D383%26epage%3D389%26doi%3D10.1042%2Fbj20031163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madoux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tredup, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scampavia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minond, D.</span></span> <span> </span><span class="NLM_article-title">Development of high throughput screening assays and pilot screen for inhibitors of metalloproteases meprin α and β</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1002/bip.22527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1002%2Fbip.22527" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2014&pages=396-406&author=F.+Madouxauthor=C.+Tredupauthor=T.+P.+Spicerauthor=L.+Scampaviaauthor=P.+S.+Chaseauthor=P.+S.+Hodderauthor=G.+B.+Fieldsauthor=C.+Becker-Paulyauthor=D.+Minond&title=Development+of+high+throughput+screening+assays+and+pilot+screen+for+inhibitors+of+metalloproteases+meprin+%CE%B1+and+%CE%B2&doi=10.1002%2Fbip.22527"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fbip.22527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22527%26sid%3Dliteratum%253Aachs%26aulast%3DMadoux%26aufirst%3DF.%26aulast%3DTredup%26aufirst%3DC.%26aulast%3DSpicer%26aufirst%3DT.%2BP.%26aulast%3DScampavia%26aufirst%3DL.%26aulast%3DChase%26aufirst%3DP.%2BS.%26aulast%3DHodder%26aufirst%3DP.%2BS.%26aulast%3DFields%26aufirst%3DG.%2BB.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DMinond%26aufirst%3DD.%26atitle%3DDevelopment%2520of%2520high%2520throughput%2520screening%2520assays%2520and%2520pilot%2520screen%2520for%2520inhibitors%2520of%2520metalloproteases%2520meprin%2520%25CE%25B1%2520and%2520%25CE%25B2%26jtitle%3DBiopolymers%26date%3D2014%26volume%3D102%26spage%3D396%26epage%3D406%26doi%3D10.1002%2Fbip.22527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barre, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">auf dem Keller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappelhoff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteomic analyses reveal an acidic prime side specificity for the astacin metalloprotease family reflected by physiological substrates</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">M111.009233</span>, <span class="refDoi"> DOI: 10.1074/mcp.M111.009233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fmcp.M111.009233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=21693781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Gjtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M111.009233&author=C.+Becker-Paulyauthor=O.+Barreauthor=O.+Schillingauthor=U.+auf+dem+Kellerauthor=A.+Ohlerauthor=C.+Broderauthor=A.+Schutteauthor=R.+Kappelhoffauthor=W.+St%C3%B6ckerauthor=C.+M.+Overall&title=Proteomic+analyses+reveal+an+acidic+prime+side+specificity+for+the+astacin+metalloprotease+family+reflected+by+physiological+substrates&doi=10.1074%2Fmcp.M111.009233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analyses reveal an acidic prime side specificity for the astacin metalloprotease family reflected by physiological substrates</span></div><div class="casAuthors">Becker-Pauly, Christoph; Barre, Olivier; Schilling, Oliver; auf dem Keller, Ulrich; Ohler, Anke; Broder, Claudia; Schuette, Andre; Kappelhoff, Reinhild; Stoecker, Walter; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">M111.009233, 19 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Astacins are secreted and membrane-bound metalloproteases with clear assocns. to many important pathol. and physiol. processes.  Yet with only a few substrates described their biol. roles are enigmatic.  Moreover, the lack of knowledge of astacin cleavage site specificities hampers assay and drug development.  Using PICS (proteomic identification of protease cleavage site specificity) and TAILS (terminal amine isotopic labeling of substrates) degradomics approaches >3000 cleavage sites were proteomically identified for five different astacins.  Such broad coverage enables family-wide detn. of specificities N- and C-terminal to the scissile peptide bond.  Remarkably, meprin α, meprin β, and LAST_MAM proteases exhibit a strong preference for aspartate in the peptide (P)1' position because of a conserved pos. charged residue in the active cleft subsite (S)1'.  This unparalleled specificity has not been found for other families of extracellular proteases.  Interestingly, cleavage specificity is also strongly influenced by proline in P2' or P3' leading to a rare example of subsite cooperativity.  This specificity characterizes the astacins as unique contributors to extracellular proteolysis that is corroborated by known cleavage sites in procollagen I+III, VEGF (vascular endothelial growth factor)-A, IL (interleukin)-1β, and pro-kallikrein 7.  Indeed, cleavage sites in VEGF-A and pro-kallikrein 7 identified by terminal amine isotopic labeling of substrates matched those reported by Edman degrdn.  Moreover, the novel substrate FGF-19 was validated biochem. and shown to exhibit altered biol. activity after meprin processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptuiOk0MgRobVg90H21EOLACvtfcHk0liN9LiE_rE-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Gjtb%252FE&md5=6645b26c29b358d1c38b77c4101d2809</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M111.009233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M111.009233%26sid%3Dliteratum%253Aachs%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DBarre%26aufirst%3DO.%26aulast%3DSchilling%26aufirst%3DO.%26aulast%3Dauf%2Bdem%2BKeller%26aufirst%3DU.%26aulast%3DOhler%26aufirst%3DA.%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DSchutte%26aufirst%3DA.%26aulast%3DKappelhoff%26aufirst%3DR.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DProteomic%2520analyses%2520reveal%2520an%2520acidic%2520prime%2520side%2520specificity%2520for%2520the%2520astacin%2520metalloprotease%2520family%2520reflected%2520by%2520physiological%2520substrates%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM111.009233%26doi%3D10.1074%2Fmcp.M111.009233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arolas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guevara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Ruth, F. X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the sheddase function of human meprin metalloproteinase at the plasma membrane</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16131</span>– <span class="NLM_lpage">16136</span>, <span class="refDoi"> DOI: 10.1073/pnas.1211076109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1073%2Fpnas.1211076109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16131-16136&author=J.+L.+Arolasauthor=C.+Broderauthor=T.+Jeffersonauthor=T.+Guevaraauthor=E.+E.+Sterchiauthor=W.+Bodeauthor=W.+St%C3%B6ckerauthor=C.+Becker-Paulyauthor=F.+X.+Gomis-Ruth&title=Structural+basis+for+the+sheddase+function+of+human+meprin+metalloproteinase+at+the+plasma+membrane&doi=10.1073%2Fpnas.1211076109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1211076109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1211076109%26sid%3Dliteratum%253Aachs%26aulast%3DArolas%26aufirst%3DJ.%2BL.%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DJefferson%26aufirst%3DT.%26aulast%3DGuevara%26aufirst%3DT.%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DGomis-Ruth%26aufirst%3DF.%2BX.%26atitle%3DStructural%2520basis%2520for%2520the%2520sheddase%2520function%2520of%2520human%2520meprin%2520metalloproteinase%2520at%2520the%2520plasma%2520membrane%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16131%26epage%3D16136%26doi%3D10.1073%2Fpnas.1211076109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsbeck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenzig, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchholz, M.</span></span> <span> </span><span class="NLM_article-title">First insight into structure-activity relationships of selective meprin β inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2428</span>– <span class="NLM_lpage">2431</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.bmcl.2017.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2428-2431&author=D.+Ramsbeckauthor=A.+Hamannauthor=D.+Schlenzigauthor=S.+Schillingauthor=M.+Buchholz&title=First+insight+into+structure-activity+relationships+of+selective+meprin+%CE%B2+inhibitors&doi=10.1016%2Fj.bmcl.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DRamsbeck%26aufirst%3DD.%26aulast%3DHamann%26aufirst%3DA.%26aulast%3DSchlenzig%26aufirst%3DD.%26aulast%3DSchilling%26aufirst%3DS.%26aulast%3DBuchholz%26aufirst%3DM.%26atitle%3DFirst%2520insight%2520into%2520structure-activity%2520relationships%2520of%2520selective%2520meprin%2520%25CE%25B2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2428%26epage%3D2431%26doi%3D10.1016%2Fj.bmcl.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Recent advances in MMP inhibitor design</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1007/s10555-006-7894-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1007%2Fs10555-006-7894-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=16680577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD28Xktlaksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=115-136&author=J.+F.+Fisherauthor=S.+Mobashery&title=Recent+advances+in+MMP+inhibitor+design&doi=10.1007%2Fs10555-006-7894-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in MMP inhibitor design</span></div><div class="casAuthors">Fisher, Jed F.; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-136</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The search for an MMP inhibitor with anticancer efficacy is a nearly three-decade endeavor.  This inhibitor is yet to be found.  The reasons for this failure include shortcomings in the chem. of these compds. (including broad MMP sub-type selectivity, metabolic lability, and toxicity) as well as the emerging, and arguably extraordinary, complexity of MMP cell (and cancer) biol.  Together these suggest that the successful anticancer inhibitor must possess MMP selectivity against the MMP subtype whose involvement is crit., yet highly temporally (with respect to metastatic progression) and mechanistically (with respect to matrix degrdn.) regulated.  This review summarizes the progression of chem. structure and mechanistic thinking toward these objectives, with emphasis on the disappointment, the perseverance, and the resilient optimism that such an inhibitor is there to be discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqgy2dldRRXLVg90H21EOLACvtfcHk0lg7y6zvgHBy4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xktlaksrs%253D&md5=ceed8438ad533dac294e22102e5ba8bf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10555-006-7894-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-006-7894-9%26sid%3Dliteratum%253Aachs%26aulast%3DFisher%26aufirst%3DJ.%2BF.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520MMP%2520inhibitor%2520design%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2006%26volume%3D25%26spage%3D115%26epage%3D136%26doi%3D10.1007%2Fs10555-006-7894-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yiotakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dive, V.</span></span> <span> </span><span class="NLM_article-title">Synthetic active site-directed inhibitors of metzincins: achievement and perspectives</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.mam.2008.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=18657570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=329-338&author=A.+Yiotakisauthor=V.+Dive&title=Synthetic+active+site-directed+inhibitors+of+metzincins%3A+achievement+and+perspectives&doi=10.1016%2Fj.mam.2008.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic active site-directed inhibitors of metzincins: Achievement and perspectives</span></div><div class="casAuthors">Yiotakis, Athanasios; Dive, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-338</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The involvement of many zinc metalloproteinases belonging to the metzincin family with a variety of pathol. states raises the possibility of therapeutic intervention using synthetic inhibitors with appropriate selectivity.  Knowledge of the catalytic domain 3D-structures for various members of the metzincin family has been successfully exploited by chemists to develop potent synthetic inhibitors.  However, despite intense efforts, very few highly selective inhibitors of metzincins have been discovered up to now.  A survey of the literature suggests that the over-exploitation of the hydroxamate function as a zinc-binding group to develop inhibitors might be responsible for this situation.  The use of alternative zinc-binding groups has led to more selective inhibitors, but the most encouraging results have been obtained for MMP-13 with compds. that do not incorporate zinc-binding groups in their structure.  This new family of inhibitors exploits the presence of a deep S 1 ' cavity in the protease active site, a specific trait shared by many members of the metzincin family.  However, to be successfully transposed to the metzincin members, this strategy will not only be able to exploit the structural detail of these S 1 ' cavities, but probably also subtle difference in their dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGory6HknHN-obVg90H21EOLACvtfcHk0lg7y6zvgHBy4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnP&md5=44bb1c612fad4769f3bdea1de9ba4792</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYiotakis%26aufirst%3DA.%26aulast%3DDive%26aufirst%3DV.%26atitle%3DSynthetic%2520active%2520site-directed%2520inhibitors%2520of%2520metzincins%253A%2520achievement%2520and%2520perspectives%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2008%26volume%3D29%26spage%3D329%26epage%3D338%26doi%3D10.1016%2Fj.mam.2008.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. P.</span></span> <span> </span><span class="NLM_article-title">QSAR studies on hydroxamic acids: a fascinating family of chemicals with a wide spectrum of activities</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">6427</span>– <span class="NLM_lpage">6490</span>, <span class="refDoi"> DOI: 10.1021/cr500483r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500483r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFaju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=6427-6490&author=S.+P.+Gupta&title=QSAR+studies+on+hydroxamic+acids%3A+a+fascinating+family+of+chemicals+with+a+wide+spectrum+of+activities&doi=10.1021%2Fcr500483r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Studies on Hydroxamic Acids: A Fascinating Family of Chemicals with a Wide Spectrum of Activities</span></div><div class="casAuthors">Gupta, Satya P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6427-6490</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lCpf2GPOi7Vg90H21EOLACvtfcHk0lgV1AbV9K7LSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFaju7o%253D&md5=2adbf316cd257ce2a4c2fd7da1d5090f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcr500483r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500483r%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DS.%2BP.%26atitle%3DQSAR%2520studies%2520on%2520hydroxamic%2520acids%253A%2520a%2520fascinating%2520family%2520of%2520chemicals%2520with%2520a%2520wide%2520spectrum%2520of%2520activities%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D6427%26epage%3D6490%26doi%3D10.1021%2Fcr500483r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saravanan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span> <span> </span><span class="NLM_article-title">Sulphonamides: deserving class as MMP inhibitors?</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.ejmech.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=23287054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVeiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2013&pages=89-100&author=P.+Jainauthor=C.+Saravananauthor=S.+K.+Singh&title=Sulphonamides%3A+deserving+class+as+MMP+inhibitors%3F&doi=10.1016%2Fj.ejmech.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sulphonamides: Deserving class as MMP inhibitors?</span></div><div class="casAuthors">Jain, Pranjali; Saravanan, Chinnadurai; Singh, Sushil Kumar</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The importance of sulfonamide moiety in medicinal chem. cannot be ignored as it constitutes an important class of extensively used drugs.  Recently, sulfonamides have also been reported for their matrix metalloproteinase (MMP) inhibitory activity.  MMPs are calcium- and zinc-dependent endopeptidases, involved in both inter- and intra-cellular activity.  This review documents the emergence of sulfonamides as matrix metalloproteinase inhibitors (MMPIs) from the first generation to the recent third generation MMPIs, their mode of action - how sulfonamides act on MMPs as well as the structure activity relationship along with their therapeutic uses in chronic obstructive pulmonary disease (COPD), ulcer, asthma, arthritis and cancer.  From this review, readers can get answer for the question- is sulfonamides a potential class of MMPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1FEujtONsfbVg90H21EOLACvtfcHk0lgV1AbV9K7LSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVeiu7o%253D&md5=10dc47ad65684dad2cd3622b82378d2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DP.%26aulast%3DSaravanan%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26atitle%3DSulphonamides%253A%2520deserving%2520class%2520as%2520MMP%2520inhibitors%253F%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D60%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.ejmech.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grams, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinemer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Rüth, F.
X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span> <span> </span><span class="NLM_article-title">The metzincins-topological and sequential relations between the astacins, adamalysins, serralysins, and matrixins (collagenases) define a superfamily of zinc-peptidases</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1002/pro.5560040502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1002%2Fpro.5560040502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=7663339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A280%3ADyaK2Mzpt1ertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=823-840&author=W.+St%C3%B6ckerauthor=F.+Gramsauthor=U.+Baumannauthor=P.+Reinemerauthor=F.%0AX.+Gomis-R%C3%BCthauthor=D.+B.+McKayauthor=W.+Bode&title=The+metzincins-topological+and+sequential+relations+between+the+astacins%2C+adamalysins%2C+serralysins%2C+and+matrixins+%28collagenases%29+define+a+superfamily+of+zinc-peptidases&doi=10.1002%2Fpro.5560040502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The metzincins--topological and sequential relations between the astacins, adamalysins, serralysins, and matrixins (collagenases) define a superfamily of zinc-peptidases</span></div><div class="casAuthors">Stocker W; Grams F; Baumann U; Reinemer P; Gomis-Ruth F X; McKay D B; Bode W</div><div class="citationInfo"><span class="NLM_cas:title">Protein science : a publication of the Protein Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">823-40</span>
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    </div><div class="casAbstract">The three-dimensional structures of the zinc endopeptidases human neutrophil collagenase, adamalysin II from rattle snake venom, alkaline proteinase from Pseudomonas aeruginosa, and astacin from crayfish are topologically similar, with respect to a five-stranded beta-sheet and three alpha-helices arranged in typical sequential order.  The four proteins exhibit the characteristic consensus motif HEXXHXXGXXH, whose three histidine residues are involved in binding of the catalytically essential zinc ion.  Moreover, they all share a conserved methionine residue beneath the active site metal as part of a superimposable "Met-turn." This structural relationship is supported by a sequence alignment performed on the basis of topological equivalence showing faint but distinct sequential similarity.  The alkaline proteinase is about equally distant (26% sequence identity) to both human neutrophil collagenase and astacin and a little further away from adamalysin II (17% identity).  The pairs astacin/adamalysin II, astacin/human neutrophil collagenase, and adamalysin II/human neutrophil collagenase exhibit sequence identities of 16%, 14%, and 13%, respectively.  Therefore, the corresponding four distinct families of zinc peptidases, the astacins, the matrix metalloproteinases (matrixins, collagenases), the adamalysins/reprolysins (snake venom proteinases/reproductive tract proteins), and the serralysins (large bacterial proteases from Serratia, Erwinia, and Pseudomonas) appear to have originated by divergent evolution from a common ancestor and form a superfamily of proteolytic enzymes for which the designation "metzincins" has been proposed.  There is also a faint but significant structural relationship of the metzincins to the thermolysin-like enzymes, which share the truncated zinc-binding motif HEXXH and, moreover, similar topologies in their N-terminal domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqG27Pg4uBlQaK-_ZudimnfW6udTcc2ebscs95lAE4yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2Mzpt1ertw%253D%253D&md5=f79a467f5e62c72238bf01f10ebfed26</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560040502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560040502%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DGrams%26aufirst%3DF.%26aulast%3DBaumann%26aufirst%3DU.%26aulast%3DReinemer%26aufirst%3DP.%26aulast%3DGomis-R%25C3%25BCth%26aufirst%3DF.%2BX.%26aulast%3DMcKay%26aufirst%3DD.%2BB.%26aulast%3DBode%26aufirst%3DW.%26atitle%3DThe%2520metzincins-topological%2520and%2520sequential%2520relations%2520between%2520the%2520astacins%252C%2520adamalysins%252C%2520serralysins%252C%2520and%2520matrixins%2520%2528collagenases%2529%2520define%2520a%2520superfamily%2520of%2520zinc-peptidases%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D823%26epage%3D840%26doi%3D10.1002%2Fpro.5560040502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, T.</span></span> <span> </span><span class="NLM_article-title">Shedding light on designing potential meprin β inhibitors through ligand-based robust validated computational approaches. A proposal to chemists!</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1080/07391102.2017.1374210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1080%2F07391102.2017.1374210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=28862075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A280%3ADC%252BC1cbjslequg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1-20&author=N.+Adhikariauthor=S.+A.+Aminauthor=B.+Ghoshauthor=T.+Jha&title=Shedding+light+on+designing+potential+meprin+%CE%B2+inhibitors+through+ligand-based+robust+validated+computational+approaches.+A+proposal+to+chemists%21&doi=10.1080%2F07391102.2017.1374210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Shedding light on designing potential meprin β inhibitors through ligand-based robust validated computational approaches: A proposal to chemists!</span></div><div class="casAuthors">Adhikari Nilanjan; Amin Sk Abdul; Jha Tarun; Ghosh Balaram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human meprin (EC 3.4.24.18) is a member of the metzincin superfamily.  It correlates with matrix metalloproteinases and ADAMs (a disintegrin and metalloproteinase).  Overexpression of meprin β is implicated in fibrosis, inflammatory diseases and cancers.  However, selective meprin β inhibition is crucial to reduce cancer metastasis and adverse effects in inflammation.  It also plays critical roles in modulating several interleukins and growth factors.  Moreover, meprin β cleaves amyloid precursor protein, thought to be involved in the progression of Alzheimer's disease.  Therefore, meprin β inhibitors are considered to be emerging therapeutics with paramount importance in the treatment of kidney failure, fibrosis, inflammatory bowel diseases and cancer.  Despite its crucial implication in several diseases, no meprin β inhibitors are available as drug candidates till date.  Therefore, it is an urgent need to identify new potential meprin β inhibitors as prospective therapeutics.  In this article, a series of meprin β inhibitors has been analysed through multiple molecular modelling studies as the first initiative to get an idea about their structural, physicochemical and pharmacophoric requirements for higher activity.  All in silico approaches performed here are statistically validated and subsequently adjudicated each other.  Compounds with p-carboxylic acid substituted arylsulphonamide moiety attached with m-carboxylic acid substituted benzyl group along with a methylene hydroxamate function may be crucial for imparting potential meprin β inhibition.  Depending on the results obtained, 14 molecules have been proposed by QSAR model that predicted a minimum of 4-fold higher activity compared to these compounds of the current study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQciZA6A96nPiM8MEzWGjS9fW6udTcc2eYLWnWiRq28jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbjslequg%253D%253D&md5=58341e01d051248190a26412772c0a38</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1080%2F07391102.2017.1374210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2017.1374210%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DAmin%26aufirst%3DS.%2BA.%26aulast%3DGhosh%26aufirst%3DB.%26aulast%3DJha%26aufirst%3DT.%26atitle%3DShedding%2520light%2520on%2520designing%2520potential%2520meprin%2520%25CE%25B2%2520inhibitors%2520through%2520ligand-based%2520robust%2520validated%2520computational%2520approaches.%2520A%2520proposal%2520to%2520chemists%2521%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2017%26spage%3D1%26epage%3D20%26doi%3D10.1080%2F07391102.2017.1374210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, K.</span>; <span class="NLM_string-name">Jäger, C.</span>; <span class="NLM_string-name">Schlenzig, D.</span>; <span class="NLM_string-name">Schilling, S.</span>; <span class="NLM_string-name">Buchholz, M.</span>; <span class="NLM_string-name">Ramsbeck, D.</span></span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Tan%2C+K.%3B+J%C3%A4ger%2C+C.%3B+Schlenzig%2C+D.%3B+Schilling%2C+S.%3B+Buchholz%2C+M.%3B+Ramsbeck%2C+D.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DK." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lassalas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasfargeas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayendran, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span> <span> </span><span class="NLM_article-title">Structure property relationships of carboxylic acid isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3203</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3183-3203&author=P.+Lassalasauthor=B.+Gayauthor=C.+Lasfargeasauthor=M.+J.+Jamesauthor=V.+Tranauthor=K.+G.+Vijayendranauthor=K.+R.+Brundenauthor=M.+C.+Kozlowskiauthor=C.+J.+Thomasauthor=A.+B.+Smithauthor=D.+M.+Hurynauthor=C.+Ballatore&title=Structure+property+relationships+of+carboxylic+acid+isosteres&doi=10.1021%2Facs.jmedchem.5b01963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Property Relationships of Carboxylic Acid Isosteres</span></div><div class="casAuthors">Lassalas, Pierrik; Gay, Bryant; Lasfargeas, Caroline; James, Michael J.; Tran, Van; Vijayendran, Krishna G.; Brunden, Kurt R.; Kozlowski, Marisa C.; Thomas, Craig J.; Smith, Amos B.; Huryn, Donna M.; Ballatore, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3183-3203</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The replacement of a carboxylic acid with a surrogate structure, or (bio)-isostere, is a classical strategy in medicinal chem.  The general underlying principle is that by maintaining the features of the carboxylic acid crit. for biol. activity, but appropriately modifying the physicochem. properties, improved analogs may result.  In this context, a systematic assessment of the physicochem. properties of carboxylic acid isosteres would be desirable to enable more informed decisions of potential replacements to be used for analog design.  Herein we report the structure-property relationships (SPR) of 35 phenylpropionic acid derivs., in which the carboxylic acid moiety is replaced with a series of known isosteres.  The data set generated provides an assessment of the relative impact on the physicochem. properties that these replacements may have compared to the carboxylic acid analog.  As such, this study presents a framework for how to rationally apply isosteric replacements of the carboxylic acid functional group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIF98i2ITNrLVg90H21EOLACvtfcHk0liNaGH5093qNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D&md5=34490fc72fdb321d8c9751bef1bf79c5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01963%26sid%3Dliteratum%253Aachs%26aulast%3DLassalas%26aufirst%3DP.%26aulast%3DGay%26aufirst%3DB.%26aulast%3DLasfargeas%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DVijayendran%26aufirst%3DK.%2BG.%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DKozlowski%26aufirst%3DM.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26aulast%3DBallatore%26aufirst%3DC.%26atitle%3DStructure%2520property%2520relationships%2520of%2520carboxylic%2520acid%2520isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3183%26epage%3D3203%26doi%3D10.1021%2Facs.jmedchem.5b01963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Beirn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">2,6-Difluorophenol as a bioisostere of a carboxylic acid: bioisosteric analogues of γ-aminobutyric acid</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1021/jm980435l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980435l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=329-332&author=J.+Qiuauthor=S.+H.+Stevensonauthor=M.+J.+O%E2%80%99Beirnauthor=R.+B.+Silverman&title=2%2C6-Difluorophenol+as+a+bioisostere+of+a+carboxylic+acid%3A+bioisosteric+analogues+of+%CE%B3-aminobutyric+acid&doi=10.1021%2Fjm980435l"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm980435l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980435l%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DStevenson%26aufirst%3DS.%2BH.%26aulast%3DO%25E2%2580%2599Beirn%26aufirst%3DM.%2BJ.%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3D2%252C6-Difluorophenol%2520as%2520a%2520bioisostere%2520of%2520a%2520carboxylic%2520acid%253A%2520bioisosteric%2520analogues%2520of%2520%25CE%25B3-aminobutyric%2520acid%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D329%26epage%3D332%26doi%3D10.1021%2Fjm980435l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villa, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertenshaw, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Critical amino acids in the active site of meprin metalloproteinases for substrate and peptide bond specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">42545</span>– <span class="NLM_lpage">42550</span>, <span class="refDoi"> DOI: 10.1074/jbc.M303718200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fjbc.M303718200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=42545-42550&author=J.+P.+Villaauthor=G.+P.+Bertenshawauthor=J.+S.+Bond&title=Critical+amino+acids+in+the+active+site+of+meprin+metalloproteinases+for+substrate+and+peptide+bond+specificity&doi=10.1074%2Fjbc.M303718200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M303718200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M303718200%26sid%3Dliteratum%253Aachs%26aulast%3DVilla%26aufirst%3DJ.%2BP.%26aulast%3DBertenshaw%26aufirst%3DG.%2BP.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DCritical%2520amino%2520acids%2520in%2520the%2520active%2520site%2520of%2520meprin%2520metalloproteinases%2520for%2520substrate%2520and%2520peptide%2520bond%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D42545%26epage%3D42550%26doi%3D10.1074%2Fjbc.M303718200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zafrani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeffet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sod-Moriah, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berliner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershonov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saphier, S.</span></span> <span> </span><span class="NLM_article-title">Difluoromethyl bioisostere. Examining the ″lipophilic hydrogen bond donor″ concept</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01691</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01691" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=797-804&author=Y.+Zafraniauthor=D.+Yeffetauthor=G.+Sod-Moriahauthor=A.+Berlinerauthor=D.+Amirauthor=D.+Marcianoauthor=E.+Gershonovauthor=S.+Saphier&title=Difluoromethyl+bioisostere.+Examining+the+%E2%80%B3lipophilic+hydrogen+bond+donor%E2%80%B3+concept&doi=10.1021%2Facs.jmedchem.6b01691"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01691%26sid%3Dliteratum%253Aachs%26aulast%3DZafrani%26aufirst%3DY.%26aulast%3DYeffet%26aufirst%3DD.%26aulast%3DSod-Moriah%26aufirst%3DG.%26aulast%3DBerliner%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DD.%26aulast%3DMarciano%26aufirst%3DD.%26aulast%3DGershonov%26aufirst%3DE.%26aulast%3DSaphier%26aufirst%3DS.%26atitle%3DDifluoromethyl%2520bioisostere.%2520Examining%2520the%2520%25E2%2580%25B3lipophilic%2520hydrogen%2520bond%2520donor%25E2%2580%25B3%2520concept%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D797%26epage%3D804%26doi%3D10.1021%2Facs.jmedchem.6b01691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lh6TVf_HNDlHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Rüth, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trillo-Muyo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span> <span> </span><span class="NLM_article-title">Functional and structural insights into astacin metallopeptidases</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>393</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1041</span>, <span class="refDoi"> DOI: 10.1515/hsz-2012-0149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1515%2Fhsz-2012-0149" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=2012&pages=1027-1041&author=F.+X.+Gomis-R%C3%BCthauthor=S.+Trillo-Muyoauthor=W.+St%C3%B6cker&title=Functional+and+structural+insights+into+astacin+metallopeptidases&doi=10.1515%2Fhsz-2012-0149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1515%2Fhsz-2012-0149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhsz-2012-0149%26sid%3Dliteratum%253Aachs%26aulast%3DGomis-R%25C3%25BCth%26aufirst%3DF.%2BX.%26aulast%3DTrillo-Muyo%26aufirst%3DS.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26atitle%3DFunctional%2520and%2520structural%2520insights%2520into%2520astacin%2520metallopeptidases%26jtitle%3DBiol.%2520Chem.%26date%3D2012%26volume%3D393%26spage%3D1027%26epage%3D1041%26doi%3D10.1515%2Fhsz-2012-0149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flipo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Hydroxamates: Relationships between structure and plasma stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6790</span>– <span class="NLM_lpage">6802</span>, <span class="refDoi"> DOI: 10.1021/jm900648x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900648x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6790-6802&author=M.+Flipoauthor=J.+Chartonauthor=A.+Hocineauthor=S.+Dassonnevilleauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Hydroxamates%3A+Relationships+between+structure+and+plasma+stability&doi=10.1021%2Fjm900648x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamates: Relationships between Structure and Plasma Stability</span></div><div class="casAuthors">Flipo, Marion; Charton, Julie; Hocine, Akila; Dassonneville, Sandrine; Deprez, Benoit; Deprez-Poulain, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6790-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydroxamates are valuable tools for chem. biol. as well as interesting leads for medicinal chem.  Although many hydroxamates display nanomolar activities against metalloproteases, only three hydroxamates have reached the market, among which is the HDAC inhibitor vorinostat.  Failures in development are generally attributed to lack of selectivity, toxicity, or poor stability.  To help medicinal chemists with respect to plasma stability, the authors have performed the first and preliminary study on structure-plasma stability for hydroxamates.  The authors define some structural rules to predict or improve the plasma stability in the preclin. stage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk66ooWU1NhbVg90H21EOLACvtfcHk0lh6TVf_HNDlHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ&md5=069a87988fbeb68a248d006c7f6307a0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm900648x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900648x%26sid%3Dliteratum%253Aachs%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DCharton%26aufirst%3DJ.%26aulast%3DHocine%26aufirst%3DA.%26aulast%3DDassonneville%26aufirst%3DS.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DHydroxamates%253A%2520Relationships%2520between%2520structure%2520and%2520plasma%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6790%26epage%3D6802%26doi%3D10.1021%2Fjm900648x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piveteau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gealageas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roignant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolojanahary, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renard, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemdani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourotte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herledan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Controlling plasma stability of hydroxamic acids. A MedChem toolbox</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9067</span>– <span class="NLM_lpage">9089</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01444</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01444" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9067-9089&author=P.+Hermantauthor=D.+Boscauthor=C.+Piveteauauthor=R.+Gealageasauthor=B.+Lamauthor=C.+Roncoauthor=M.+Roignantauthor=H.+Tolojanaharyauthor=L.+Jeanauthor=P.-Y.+Renardauthor=M.+Lemdaniauthor=M.+Bourotteauthor=A.+Herledanauthor=C.+Bedartauthor=A.+Bielaauthor=F.+Lerouxauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Controlling+plasma+stability+of+hydroxamic+acids.+A+MedChem+toolbox&doi=10.1021%2Facs.jmedchem.7b01444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01444%26sid%3Dliteratum%253Aachs%26aulast%3DHermant%26aufirst%3DP.%26aulast%3DBosc%26aufirst%3DD.%26aulast%3DPiveteau%26aufirst%3DC.%26aulast%3DGealageas%26aufirst%3DR.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DRonco%26aufirst%3DC.%26aulast%3DRoignant%26aufirst%3DM.%26aulast%3DTolojanahary%26aufirst%3DH.%26aulast%3DJean%26aufirst%3DL.%26aulast%3DRenard%26aufirst%3DP.-Y.%26aulast%3DLemdani%26aufirst%3DM.%26aulast%3DBourotte%26aufirst%3DM.%26aulast%3DHerledan%26aufirst%3DA.%26aulast%3DBedart%26aufirst%3DC.%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DLeroux%26aufirst%3DF.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DControlling%2520plasma%2520stability%2520of%2520hydroxamic%2520acids.%2520A%2520MedChem%2520toolbox%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9067%26epage%3D9089%26doi%3D10.1021%2Facs.jmedchem.7b01444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlenzig, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsbeck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moenke-Wedler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, S.</span></span> <span> </span><span class="NLM_article-title">Expression, purification and initial characterization of human meprin β from Pichia pastoris</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.pep.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=26256061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCis7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2015&pages=75-81&author=D.+Schlenzigauthor=M.+Wermannauthor=D.+Ramsbeckauthor=T.+Moenke-Wedlerauthor=S.+Schilling&title=Expression%2C+purification+and+initial+characterization+of+human+meprin+%CE%B2+from+Pichia+pastoris&doi=10.1016%2Fj.pep.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification and initial characterization of human meprin β from Pichia pastoris</span></div><div class="casAuthors">Schlenzig, D.; Wermann, M.; Ramsbeck, D.; Moenke-Wedler, T.; Schilling, S.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-81</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Human meprin β (h-meprin β), a single-zinc metalloendoprotease of the astacin family, is potentially involved in disorders such as fibrosis and Alzheimer's disease.  Here, the authors describe the expression of the enzyme in the yeast Pichia pastoris.  The N-terminal signal sequence was replaced by the α-leader of Saccharomyces, enabling efficient secretion of the mature enzyme, harboring either an N-terminal or C-terminal His-tag.  The purifn. by affinity and hydrophobic interaction chromatog. gave 58.4 mg/l of homogeneous human pro-meprin β from fermn. broth.  The activated enzyme isolated from yeast (yh-meprin β) displayed virtually identical enzymic activity as h-meprin from a mammalian cell line.  Furthermore, the yh-meprin β was N-glycosylated and secreted as a dimer with a mol. mass of 148 kDa.  Endoglycosidase H treatment generated a protein with a mol. mass of 133 kDa, but essentially unchanged kinetic parameters.  Thus, the authors' data suggest that human meprin β expressed in P. pastoris displays virtually identical parameters as meprin from other sources.  The high yield of protein expression, the ease of purifn. and the deglycosylation in its native state appear to favor further studies aiming at inhibitor screening and structure-based inhibitor refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMPvtA3KzZBbVg90H21EOLACvtfcHk0lgLc_2Aaf-gYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCis7jL&md5=fc8e221669da77c9f437ed9d71ba7abd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchlenzig%26aufirst%3DD.%26aulast%3DWermann%26aufirst%3DM.%26aulast%3DRamsbeck%26aufirst%3DD.%26aulast%3DMoenke-Wedler%26aufirst%3DT.%26aulast%3DSchilling%26aufirst%3DS.%26atitle%3DExpression%252C%2520purification%2520and%2520initial%2520characterization%2520of%2520human%2520meprin%2520%25CE%25B2%2520from%2520Pichia%2520pastoris%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2015%26volume%3D116%26spage%3D75%26epage%3D81%26doi%3D10.1016%2Fj.pep.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mac
Sweeney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Parrado, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinzenz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodendorf, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhartz, B.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the substrate specificity of bone morphogenetic protein 1/tolloid-like metalloproteases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.jmb.2008.09.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=228-239&author=A.+Mac%0ASweeneyauthor=S.+Gil-Parradoauthor=D.+Vinzenzauthor=A.+Bernardiauthor=A.+Heinauthor=U.+Bodendorfauthor=P.+Erbelauthor=C.+Logelauthor=B.+Gerhartz&title=Structural+basis+for+the+substrate+specificity+of+bone+morphogenetic+protein+1%2Ftolloid-like+metalloproteases&doi=10.1016%2Fj.jmb.2008.09.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DMac%2BSweeney%26aufirst%3DA.%26aulast%3DGil-Parrado%26aufirst%3DS.%26aulast%3DVinzenz%26aufirst%3DD.%26aulast%3DBernardi%26aufirst%3DA.%26aulast%3DHein%26aufirst%3DA.%26aulast%3DBodendorf%26aufirst%3DU.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DLogel%26aufirst%3DC.%26aulast%3DGerhartz%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520for%2520the%2520substrate%2520specificity%2520of%2520bone%2520morphogenetic%2520protein%25201%252Ftolloid-like%2520metalloproteases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26spage%3D228%26epage%3D239%26doi%3D10.1016%2Fj.jmb.2008.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miriam  Linnert</span>, <span class="hlFld-ContribAuthor ">Claudia  Fritz</span>, <span class="hlFld-ContribAuthor ">Christian  Jäger</span>, <span class="hlFld-ContribAuthor ">Dagmar  Schlenzig</span>, <span class="hlFld-ContribAuthor ">Daniel  Ramsbeck</span>, <span class="hlFld-ContribAuthor ">Martin  Kleinschmidt</span>, <span class="hlFld-ContribAuthor ">Michael  Wermann</span>, <span class="hlFld-ContribAuthor ">Hans-Ulrich  Demuth</span>, <span class="hlFld-ContribAuthor ">Christoph  Parthier</span>, <span class="hlFld-ContribAuthor ">Stephan  Schilling</span>. </span><span class="cited-content_cbyCitation_article-title">Structure and Dynamics of Meprin β in Complex with a Hydroxamate-Based Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5651. <a href="https://doi.org/10.3390/ijms22115651" title="DOI URL">https://doi.org/10.3390/ijms22115651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115651%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DStructure%252Band%252BDynamics%252Bof%252BMeprin%252B%2525CE%2525B2%252Bin%252BComplex%252Bwith%252Ba%252BHydroxamate-Based%252BInhibitor%26aulast%3DLinnert%26aufirst%3DMiriam%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathrin  Tan</span>, <span class="hlFld-ContribAuthor ">Christian  Jäger</span>, <span class="hlFld-ContribAuthor ">Hagen  Körschgen</span>, <span class="hlFld-ContribAuthor ">Stefanie  Geissler</span>, <span class="hlFld-ContribAuthor ">Dagmar  Schlenzig</span>, <span class="hlFld-ContribAuthor ">Mirko  Buchholz</span>, <span class="hlFld-ContribAuthor ">Walter  Stöcker</span>, <span class="hlFld-ContribAuthor ">Daniel  Ramsbeck</span>. </span><span class="cited-content_cbyCitation_article-title">Heteroaromatic Inhibitors of the Astacin Proteinases Meprin α, Meprin β and Ovastacin Discovered by a Scaffold‐Hopping Approach. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (6)
                                     , 976-988. <a href="https://doi.org/10.1002/cmdc.202000822" title="DOI URL">https://doi.org/10.1002/cmdc.202000822</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000822%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DHeteroaromatic%252BInhibitors%252Bof%252Bthe%252BAstacin%252BProteinases%252BMeprin%252B%2525CE%2525B1%25252C%252BMeprin%252B%2525CE%2525B2%252Band%252BOvastacin%252BDiscovered%252Bby%252Ba%252BScaffold%2525E2%252580%252590Hopping%252BApproach%26aulast%3DTan%26aufirst%3DKathrin%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D6%26spage%3D976%26epage%3D988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Juan  Diez</span>, <span class="hlFld-ContribAuthor ">Hajeung  Park</span>, <span class="hlFld-ContribAuthor ">Christoph  Becker-Pauly</span>, <span class="hlFld-ContribAuthor ">Gregg B.  Fields</span>, <span class="hlFld-ContribAuthor ">Timothy P.  Spicer</span>, <span class="hlFld-ContribAuthor ">Louis D.  Scampavia</span>, <span class="hlFld-ContribAuthor ">Dmitriy  Minond</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Bannister</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Selective Inhibitors of Meprin α (Part II). </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (3)
                                     , 197. <a href="https://doi.org/10.3390/ph14030197" title="DOI URL">https://doi.org/10.3390/ph14030197</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14030197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14030197%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BSelective%252BInhibitors%252Bof%252BMeprin%252B%2525CE%2525B1%252B%252528Part%252BII%252529%26aulast%3DWang%26aufirst%3DChao%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D3%26spage%3D197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shurong  Hou</span>, <span class="hlFld-ContribAuthor ">Juan  Diez</span>, <span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Christoph  Becker-Pauly</span>, <span class="hlFld-ContribAuthor ">Gregg B.  Fields</span>, <span class="hlFld-ContribAuthor ">Thomas  Bannister</span>, <span class="hlFld-ContribAuthor ">Timothy P.  Spicer</span>, <span class="hlFld-ContribAuthor ">Louis D.  Scampavia</span>, <span class="hlFld-ContribAuthor ">Dmitriy  Minond</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Selective Inhibitors of Meprin α (Part I). </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (3)
                                     , 203. <a href="https://doi.org/10.3390/ph14030203" title="DOI URL">https://doi.org/10.3390/ph14030203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14030203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14030203%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BSelective%252BInhibitors%252Bof%252BMeprin%252B%2525CE%2525B1%252B%252528Part%252BI%252529%26aulast%3DHou%26aufirst%3DShurong%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D3%26spage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hagen  Körschgen</span>, <span class="hlFld-ContribAuthor ">Christian  Jäger</span>, <span class="hlFld-ContribAuthor ">Kathrin  Tan</span>, <span class="hlFld-ContribAuthor ">Mirko  Buchholz</span>, <span class="hlFld-ContribAuthor ">Walter  Stöcker</span>, <span class="hlFld-ContribAuthor ">Daniel  Ramsbeck</span>. </span><span class="cited-content_cbyCitation_article-title">A Primary Evaluation of Potential Small‐Molecule Inhibitors of the Astacin Metalloproteinase Ovastacin, a Novel Drug Target in Female Infertility Treatment. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (16)
                                     , 1499-1504. <a href="https://doi.org/10.1002/cmdc.202000397" title="DOI URL">https://doi.org/10.1002/cmdc.202000397</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000397%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BPrimary%252BEvaluation%252Bof%252BPotential%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252Bthe%252BAstacin%252BMetalloproteinase%252BOvastacin%25252C%252Ba%252BNovel%252BDrug%252BTarget%252Bin%252BFemale%252BInfertility%252BTreatment%26aulast%3DK%25C3%25B6rschgen%26aufirst%3DHagen%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D16%26spage%3D1499%26epage%3D1504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuthika  Dholey</span>, <span class="hlFld-ContribAuthor ">Ankur  Chaudhuri</span>, <span class="hlFld-ContribAuthor ">Sibani  Sen Chakraborty</span>. </span><span class="cited-content_cbyCitation_article-title">An integrated
              in silico
              approach to understand protein–protein interactions: human meprin-β with fetuin-A. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (7)
                                     , 2080-2092. <a href="https://doi.org/10.1080/07391102.2019.1626284" title="DOI URL">https://doi.org/10.1080/07391102.2019.1626284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1626284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1626284%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DAn%252Bintegrated%252Bin%252Bsilico%252Bapproach%252Bto%252Bunderstand%252Bprotein%2525E2%252580%252593protein%252Binteractions%25253A%252Bhuman%252Bmeprin-%2525CE%2525B2%252Bwith%252Bfetuin-A%26aulast%3DDholey%26aufirst%3DYuthika%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D7%26spage%3D2080%26epage%3D2092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ankur  Chaudhuri</span>, <span class="hlFld-ContribAuthor ">Nandagopal  Hudait</span>, <span class="hlFld-ContribAuthor ">Sibani Sen  Chakraborty</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore modeling coupled with molecular dynamic simulation approach to identify new leads for meprin-β metalloprotease. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2019,</strong> <em>80 </em>, 292-306. <a href="https://doi.org/10.1016/j.compbiolchem.2019.04.014" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2019.04.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2019.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2019.04.014%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DPharmacophore%252Bmodeling%252Bcoupled%252Bwith%252Bmolecular%252Bdynamic%252Bsimulation%252Bapproach%252Bto%252Bidentify%252Bnew%252Bleads%252Bfor%252Bmeprin-%2525CE%2525B2%252Bmetalloprotease%26aulast%3DChaudhuri%26aufirst%3DAnkur%26date%3D2019%26volume%3D80%26spage%3D292%26epage%3D306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazunori  Hirayama</span>, <span class="hlFld-ContribAuthor ">Yuuki  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Tohru  Terada</span>, <span class="hlFld-ContribAuthor ">Kentaro  Shimizu</span>, <span class="hlFld-ContribAuthor ">Keiji  Wada</span>, <span class="hlFld-ContribAuthor ">Tomohiro  Kabuta</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening identification of novel chemical inhibitors for aberrant interactions between pathogenic mutant SOD1 and tubulin. </span><span class="cited-content_cbyCitation_journal-name">Neurochemistry International</span><span> <strong>2019,</strong> <em>126 </em>, 19-26. <a href="https://doi.org/10.1016/j.neuint.2019.02.020" title="DOI URL">https://doi.org/10.1016/j.neuint.2019.02.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neuint.2019.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neuint.2019.02.020%26sid%3Dliteratum%253Aachs%26jtitle%3DNeurochemistry%2520International%26atitle%3DVirtual%252Bscreening%252Bidentification%252Bof%252Bnovel%252Bchemical%252Binhibitors%252Bfor%252Baberrant%252Binteractions%252Bbetween%252Bpathogenic%252Bmutant%252BSOD1%252Band%252Btubulin%26aulast%3DHirayama%26aufirst%3DKazunori%26date%3D2019%26volume%3D126%26spage%3D19%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathrin  Tan</span>, <span class="hlFld-ContribAuthor ">Christian  Jäger</span>, <span class="hlFld-ContribAuthor ">Dagmar  Schlenzig</span>, <span class="hlFld-ContribAuthor ">Stephan  Schilling</span>, <span class="hlFld-ContribAuthor ">Mirko  Buchholz</span>, <span class="hlFld-ContribAuthor ">Daniel  Ramsbeck</span>. </span><span class="cited-content_cbyCitation_article-title">Tertiary-Amine-Based Inhibitors of the Astacin Protease Meprin α. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (16)
                                     , 1619-1624. <a href="https://doi.org/10.1002/cmdc.201800300" title="DOI URL">https://doi.org/10.1002/cmdc.201800300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800300%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DTertiary-Amine-Based%252BInhibitors%252Bof%252Bthe%252BAstacin%252BProtease%252BMeprin%2525E2%252580%252585%2525CE%2525B1%26aulast%3DTan%26aufirst%3DKathrin%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D16%26spage%3D1619%26epage%3D1624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known inhibitors of meprin β.<a onclick="showRef(event, 'ref16 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref20">(16,20)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Active site view of meprin β (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a>). The arginine residues shaping the S1, S1′, and S2′ subpockets are colored in blue. The tetrahedral coordinated Zn<sup>2+</sup> is colored in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural comparison of ADAM17 (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DDF">2DDF</a>, A), MMP13 (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZXH">3ZXH</a>, B), and meprin β (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN">4GWN</a>, C). The S-shaped loop is colored in yellow, forming a bulge adjacent to the active site. The bound inhibitors of ADAM17 (D) and MMP13 (E) interact with the protein backbone via hydrogen bonds, formed with the S-shaped loop. The respective small loop in meprin β (F) rules out these protein–ligand interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Schematic representation of possible hydrogen bond interactions of compound <b>1</b> with the S-shaped loop of MMPs or ADAMs (A) and missing hydrogen bond of <b>1</b> bound to meprin β, lacking the S-shaped loop (B). The SO<sub>2</sub> → CH<sub>2</sub> switch could disrupt the respective protein–ligand interaction in MMPs and ADAMs (C) while directing the benzyl residue correctly into the S1′ pocket in meprin β without the need of hydrogen bond formation (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reaction Scheme for the Synthesis of Secondary and Tertiary Amine Based Hydroxamic Acids<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) i: Trt-O-NH<sub>2</sub>, TBTU, DIPEA, DMF, rt; ii: DBU, THF, rt; (b) R-CHO, NaBH<sub>4</sub>, MeOH, rt; (c) <i>tert</i>-butyl 3-(chloromethyl)benzoate, TEA, DMF, rt or MW 120 °C (for <b>6a,b,c,d,f,g,h,n</b>); (d) TFA/DCM (1:1 v/v), TIS, rt; (e) <i>tert</i>-butyl 3-formylbenzoate, NaBH<sub>4</sub>, MeOH, rt; (f) 1.1 equiv of R-Hal, TEA, DMF, rt or MW 120 °C (for <b>6e,i,j,k,l,m</b>); (g) 2.2 equiv of R-Hal, TEA, DMF, rt or MW 140 °C; (h) BBr<sub>3</sub>, DCM, 0° to rt (for <b>8g–l</b>).</p></p></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/medium/jm-2018-003309_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding modes of <b>6a</b>, <b>8a</b>, and <b>8i</b> within the active site of meprin β. (A) Compound <b>6a</b> solution ranked 3 of 20 and (B) 7 of 20. (C) Compound <b>8a</b> solution ranked 1 of 20. (D) Compound <b>8i</b> solution ranked 1 of 20.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00330/20180517/images/large/jm-2018-003309_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00330&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75716" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75716" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">Emerging principles in protease-based drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">701</span>, <span class="refDoi"> DOI: 10.1038/nrd3053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fnrd3053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=20811381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSisbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=690-701&author=M.+Dragauthor=G.+S.+Salvesen&title=Emerging+principles+in+protease-based+drug+discovery&doi=10.1038%2Fnrd3053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging principles in protease-based drug discovery</span></div><div class="casAuthors">Drag, Marcin; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">690-701</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteases have an important role in many signalling pathways, and represent potential drug targets for diseases ranging from cardiovascular disorders to cancer, as well as for combating many parasites and viruses.  Although inhibitors of well-established protease targets such as angiotensin-converting enzyme and HIV protease have shown substantial therapeutic success, developing drugs for new protease targets has proved challenging in recent years.  This in part could be due to issues such as the difficulty of achieving selectivity when targeting protease active sites.  This Perspective discusses the general principles in protease-based drug discovery, highlighting the lessons learned and the emerging strategies, such as targeting allosteric sites, which could help harness the therapeutic potential of new protease targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz3PQDA1pnT7Vg90H21EOLACvtfcHk0liEv_-8-Zfljw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSisbzP&md5=28a9571473c3f753027665b77d3257d0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3053%26sid%3Dliteratum%253Aachs%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DEmerging%2520principles%2520in%2520protease-based%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D690%26epage%3D701%26doi%3D10.1038%2Fnrd3053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleifeld, O.</span></span> <span> </span><span class="NLM_article-title">Towards third generation matrix metalloproteinase inhibitors for cancer therapy</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">946</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6603043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fsj.bjc.6603043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=16538215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFGqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2006&pages=941-946&author=C.+M.+Overallauthor=O.+Kleifeld&title=Towards+third+generation+matrix+metalloproteinase+inhibitors+for+cancer+therapy&doi=10.1038%2Fsj.bjc.6603043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Towards third generation matrix metalloproteinase inhibitors for cancer therapy</span></div><div class="casAuthors">Overall, C. M.; Kleifeld, O.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-946</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The failure of matrix metalloproteinase (MMP) inhibitor drug clin. trials in cancer was partly due to the inadvertent inhibition of MMP antitargets that counterbalanced the benefits of MMP target inhibition.  We explore how MMP inhibitor drugs might be developed to achieve potent selectivity for validated MMP targets yet therapeutically spare MMP antitargets that are crit. in host protection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWaFApPMIKRLVg90H21EOLACvtfcHk0lgh14Oo6unl7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFGqt7g%253D&md5=5e672a6e9495ec2e283856edc6e61a60</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6603043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6603043%26sid%3Dliteratum%253Aachs%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DKleifeld%26aufirst%3DO.%26atitle%3DTowards%2520third%2520generation%2520matrix%2520metalloproteinase%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2006%26volume%3D94%26spage%3D941%26epage%3D946%26doi%3D10.1038%2Fsj.bjc.6603043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenbroucke, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span> <span> </span><span class="NLM_article-title">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd4390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fnrd4390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=25376097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=904-927&author=R.+E.+Vandenbrouckeauthor=C.+Libert&title=Is+there+new+hope+for+therapeutic+matrix+metalloproteinase+inhibition%3F&doi=10.1038%2Fnrd4390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span></div><div class="casAuthors">Vandenbroucke, Roosmarijn E.; Libert, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">904-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent endopeptidases that form a family of 24 members in mammals.  Evidence of the pathol. roles of MMPs in various diseases, combined with their druggability, has made them attractive therapeutic targets.  Initial drug discovery efforts focused on the roles of MMPs in cancer progression, and more than 50 MMP inhibitors have been investigated in clin. trials in various cancers.  However, all of these trials failed.  Reasons for failure include the lack of inhibitor specificity and insufficient knowledge about the complexity of the disease biol.  MMPs are also known to be involved in several inflammatory processes, and there are new therapeutic opportunities for MMP inhibitors to treat such diseases.  In this Review, we discuss the recent advances made in understanding the role of MMPs in inflammatory diseases and the therapeutic potential of MMP inhibition in those conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpua2GBt8P5b7Vg90H21EOLACvtfcHk0lgh14Oo6unl7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I&md5=032265fe59e9f7567e4d3c7d072b3cde</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4390%26sid%3Dliteratum%253Aachs%26aulast%3DVandenbroucke%26aufirst%3DR.%2BE.%26aulast%3DLibert%26aufirst%3DC.%26atitle%3DIs%2520there%2520new%2520hope%2520for%2520therapeutic%2520matrix%2520metalloproteinase%2520inhibition%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D904%26epage%3D927%26doi%3D10.1038%2Fnrd4390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenspan, D. S.</span></span> <span> </span><span class="NLM_article-title">The bone morphogenetic protein 1/tolloid-like metalloproteinases</span>. <i>Matrix Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">523</span>, <span class="refDoi"> DOI: 10.1016/j.matbio.2007.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.matbio.2007.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=17560775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=508-523&author=D.+R.+Hopkinsauthor=S.+Kelesauthor=D.+S.+Greenspan&title=The+bone+morphogenetic+protein+1%2Ftolloid-like+metalloproteinases&doi=10.1016%2Fj.matbio.2007.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The bone morphogenetic protein 1/Tolloid-like metalloproteinases</span></div><div class="casAuthors">Hopkins, Delana R.; Keles, Sunduz; Greenspan, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Matrix Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">508-523</span>CODEN:
                <span class="NLM_cas:coden">MTBOEC</span>;
        ISSN:<span class="NLM_cas:issn">0945-053X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A decade ago, bone morphogenetic protein 1 (BMP1) was shown to provide the activity necessary for proteolytic removal of the C-propeptides of procollagens I-III, precursors of the major fibrillar collagens.  Subsequent studies have shown BMP1 to be the prototype of a small group of extracellular metalloproteinases that play manifold roles in regulating the formation of the extracellular matrix (ECM).  Soon after the initial cloning of BMP1, genetic studies showed the related Drosophila proteinase, Tolloid (TLD), to be necessary for the formation of the dorsal-ventral axis in early embryogenesis.  It is now clear that the BMP1/TLD-like proteinases, conserved in species ranging from Drosophila to humans, act in dorsal-ventral patterning via activation of transforming growth factor β (TGFβ)-like proteins BMP2, BMP4 (vertebrates), and decapentaplegic (arthropods).  More recently, it has become apparent that the BMP1/TLD-like proteinases are activators of a broader subset of the TGFβ superfamily of proteins, with implications that these proteinases may be key in orchestrating the formation of ECM with growth factor activation and BMP signaling in morphogenetic processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxFod_zt9QlLVg90H21EOLACvtfcHk0lizknMzitI_8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qs7vM&md5=b6de6f0fcaddbfe9b5633e2d5e6cdcd7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.matbio.2007.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.matbio.2007.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DD.%2BR.%26aulast%3DKeles%26aufirst%3DS.%26aulast%3DGreenspan%26aufirst%3DD.%2BS.%26atitle%3DThe%2520bone%2520morphogenetic%2520protein%25201%252Ftolloid-like%2520metalloproteinases%26jtitle%3DMatrix%2520Biol.%26date%3D2007%26volume%3D26%26spage%3D508%26epage%3D523%26doi%3D10.1016%2Fj.matbio.2007.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Meprins, membrane-bound and secreted astacin metalloproteinases</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.mam.2008.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=18783725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=309-328&author=E.+E.+Sterchiauthor=W.+St%C3%B6ckerauthor=J.+S.+Bond&title=Meprins%2C+membrane-bound+and+secreted+astacin+metalloproteinases&doi=10.1016%2Fj.mam.2008.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Meprins, membrane-bound and secreted astacin metalloproteinases</span></div><div class="casAuthors">Sterchi, Erwin E.; Stocker, Walter; Bond, Judith S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-328</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The astacins are a subfamily of the metzincin superfamily of metalloproteinases.  The first to be characterized was the crayfish enzyme astacin.  To date more than 200 members of this family have been identified in species ranging from bacteria to humans.  Astacins are involved in developmental morphogenesis, matrix assembly, tissue differentiation and digestion.  Family members include the procollagen C-proteinase (BMP1, bone morphogenetic protein 1), tolloid and mammalian tolloid-like, HMP (Hydra vulgaris metalloproteinase), sea urchin BP10 (blastula protein) and SPAN (Strongylocentrotus purpuratus astacin), the hatching' subfamily comprising alveolin, ovastacin, LCE, HCE ( low' and high' choriolytic enzymes), nephrosin (from carp head kidney), UVS.2 from frog, and the meprins.  In the human and mouse genomes, there are six astacin family genes (two meprins, three BMP1/tolloid-like, one ovastacin), but in Caenorhabditis elegans there are 40.  Meprins are the only astacin proteinases that function on the membrane and extracellularly by virtue of the fact that they can be membrane-bound or secreted.  They are unique in their domain structure and covalent subunit dimerization, oligomerization propensities, and expression patterns.  They are normally highly regulated at the transcriptional and post-translational levels, localize to specific membranes or extracellular spaces, and can hydrolyze biol. active peptides, cytokines, extracellular matrix (ECM) proteins and cell-surface proteins.  The in vivo substrates of meprins are unknown, but the abundant expression of these proteinases in the epithelial cells of the intestine, kidney and skin provide clues to their functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKeid0CPGYrLVg90H21EOLACvtfcHk0lizknMzitI_8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnO&md5=ce85a8d627e13b8149e899de77ebbe36</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DMeprins%252C%2520membrane-bound%2520and%2520secreted%2520astacin%2520metalloproteinases%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2008%26volume%3D29%26spage%3D309%26epage%3D328%26doi%3D10.1016%2Fj.mam.2008.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turtle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W.-B.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of procollagen C-proteinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">7397</span>– <span class="NLM_lpage">7401</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.bmcl.2012.10.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=7397-7401&author=E.+Turtleauthor=N.+Chowauthor=C.+Yangauthor=S.+Sosaauthor=U.+Bauerauthor=M.+Brennerauthor=D.+Solow-Corderoauthor=W.-B.+Ho&title=Design+and+synthesis+of+procollagen+C-proteinase+inhibitors&doi=10.1016%2Fj.bmcl.2012.10.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DTurtle%26aufirst%3DE.%26aulast%3DChow%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DSosa%26aufirst%3DS.%26aulast%3DBauer%26aufirst%3DU.%26aulast%3DBrenner%26aufirst%3DM.%26aulast%3DSolow-Cordero%26aufirst%3DD.%26aulast%3DHo%26aufirst%3DW.-B.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520procollagen%2520C-proteinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D7397%26epage%3D7401%26doi%3D10.1016%2Fj.bmcl.2012.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turtle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W.-B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of procollagen C-proteinase: fibrosis and beyond</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1517/13543776.14.8.1185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1517%2F13543776.14.8.1185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=1185-1197&author=E.+D.+Turtleauthor=W.-B.+Ho&title=Inhibition+of+procollagen+C-proteinase%3A+fibrosis+and+beyond&doi=10.1517%2F13543776.14.8.1185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of procollagen C-proteinase: fibrosis and beyond</span></div><div class="casAuthors">Turtle, Eric D.; Ho, Wen-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1185-1197</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Procollagen C-proteinase (PCP) plays a key role in fibrosis.  The activity of PCP is necessary to convert sol. procollagens into fully formed collagen fibrils.  Fibrosis is a pathol. condition in which the normal wound healing process goes awry, culminating in the excessive prodn. and deposition of collagen, a major component of scar tissue.  The persistent formation of scar tissue hinders proper tissue function and can lead to organ failure in a wide range of fibrotic diseases.  PCP is recognized as an attractive target for antifibrotic therapies.  This review provides a discussion of the therapeutic potential of PCP inhibition and an examn. of relevant patent literature, primarily focused on the identification of small mol. inhibitors of PCP for the prevention of dermal scarring and fibrotic disease.  The emerging roles of PCP in biol. processes other than collagen biosynthesis are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy4W1KYeLHOLVg90H21EOLACvtfcHk0lizknMzitI_8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSjurk%253D&md5=460843d020a76e59297dd89120feb656</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F13543776.14.8.1185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.14.8.1185%26sid%3Dliteratum%253Aachs%26aulast%3DTurtle%26aufirst%3DE.%2BD.%26aulast%3DHo%26aufirst%3DW.-B.%26atitle%3DInhibition%2520of%2520procollagen%2520C-proteinase%253A%2520fibrosis%2520and%2520beyond%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2004%26volume%3D14%26spage%3D1185%26epage%3D1197%26doi%3D10.1517%2F13543776.14.8.1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">The metalloproteases meprin α and meprin β: unique enzymes in inflammation, neurodegeneration, cancer and fibrosis</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1042/BJ20121751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1042%2FBJ20121751" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2013&pages=253-264&author=C.+Broderauthor=C.+Becker-Pauly&title=The+metalloproteases+meprin+%CE%B1+and+meprin+%CE%B2%3A+unique+enzymes+in+inflammation%2C+neurodegeneration%2C+cancer+and+fibrosis&doi=10.1042%2FBJ20121751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20121751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121751%26sid%3Dliteratum%253Aachs%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DThe%2520metalloproteases%2520meprin%2520%25CE%25B1%2520and%2520meprin%2520%25CE%25B2%253A%2520unique%2520enzymes%2520in%2520inflammation%252C%2520neurodegeneration%252C%2520cancer%2520and%2520fibrosis%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D450%26spage%3D253%26epage%3D264%26doi%3D10.1042%2FBJ20121751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">Meprin α and meprin β: Procollagen proteinases in health and disease</span>. <i>Matrix Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>44–46</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.matbio.2015.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.matbio.2015.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=25617491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFChsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44%E2%80%9346&publication_year=2015&pages=7-13&author=J.+Proxauthor=P.+Arnoldauthor=C.+Becker-Pauly&title=Meprin+%CE%B1+and+meprin+%CE%B2%3A+Procollagen+proteinases+in+health+and+disease&doi=10.1016%2Fj.matbio.2015.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Meprin α and meprin β: Procollagen proteinases in health and disease</span></div><div class="casAuthors">Prox, Johannes; Arnold, Philipp; Becker-Pauly, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Matrix Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44-46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-13</span>CODEN:
                <span class="NLM_cas:coden">MTBOEC</span>;
        ISSN:<span class="NLM_cas:issn">0945-053X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Metalloproteases meprin α and meprin β were recently discovered as procollagen proteinases, capable of cleaving off the globular C- and N-terminal prodomains of fibrillar collagen type I and type III.  This proteolytic process is indeed sufficient to induce collagen fibril assembly as visualized by transmission electron microscopy.  The biol. relevance was demonstrated with the help of meprin α and meprin β knock-out mice, which exhibit decreased collagen deposition in skin resulting in impaired tensile strength.  On the other hand, overexpression of meprin metalloproteases was found under fibrotic conditions in the skin (keloids) and the lung (pulmonary hypertension).  Thus, regulation of meprin activity by specific inhibition to reduce collagen maturation might be a suitable approach for the treatment of certain pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlUbO2xzhcbVg90H21EOLACvtfcHk0lhYTONxYfZCIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFChsrg%253D&md5=85291d65f92aceac7c6e8e0aa1e224f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.matbio.2015.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.matbio.2015.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DProx%26aufirst%3DJ.%26aulast%3DArnold%26aufirst%3DP.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DMeprin%2520%25CE%25B1%2520and%2520meprin%2520%25CE%25B2%253A%2520Procollagen%2520proteinases%2520in%2520health%2520and%2520disease%26jtitle%3DMatrix%2520Biol.%26date%3D2015%26volume%3D44%25E2%2580%259346%26spage%3D7%26epage%3D13%26doi%3D10.1016%2Fj.matbio.2015.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadon-Le Goff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konerding, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koudelka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tholey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulmes, D. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moali, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span> <span> </span><span class="NLM_article-title">Metalloproteases meprin and meprin are C- and N-procollagen proteinases important for collagen assembly and tensile strength</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">14219</span>– <span class="NLM_lpage">14224</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305464110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1073%2Fpnas.1305464110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=23940311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=14219-14224&author=C.+Broderauthor=P.+Arnoldauthor=S.+Vadon-Le+Goffauthor=M.+A.+Konerdingauthor=K.+Bahrauthor=S.+Mullerauthor=C.+M.+Overallauthor=J.+S.+Bondauthor=T.+Koudelkaauthor=A.+Tholeyauthor=D.+J.+S.+Hulmesauthor=C.+Moaliauthor=C.+Becker-Pauly&title=Metalloproteases+meprin+and+meprin+are+C-+and+N-procollagen+proteinases+important+for+collagen+assembly+and+tensile+strength&doi=10.1073%2Fpnas.1305464110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metalloproteases meprin α and meprin β are C- and N-procollagen proteinases important for collagen assembly and tensile strength</span></div><div class="casAuthors">Broder, Claudia; Arnold, Philipp; Vadon-Le Goff, Sandrine; Konerding, Moritz A.; Bahr, Kerstin; Mueller, Stefan; Overall, Christopher M.; Bond, Judith S.; Koudelka, Tomas; Tholey, Andreas; Hulmes, David J. S.; Moali, Catherine; Becker-Pauly, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">14219-14224,S14219/1-S14219/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Type I fibrillar collagen is the most abundant protein in the human body, crucial for the formation and strength of bones, skin, and tendon.  Proteolytic enzymes are essential for initiation of the assembly of collagen fibrils by cleaving off the propeptides.  We report that Mep1a-/- and Mep1b-/- mice revealed lower amts. of mature collagen I compared with WT mice and exhibited significantly reduced collagen deposition in skin, along with markedly decreased tissue tensile strength.  While exploring the mechanism of this phenotype, we found that cleavage of full-length human procollagen I heterotrimers by either meprin α or meprin β led to the generation of mature collagen mols. that spontaneously assembled into collagen fibrils.  Thus, meprin α and meprin β are unique in their ability to process and release both C- and N-propeptides from type I procollagen in vitro and in vivo and contribute to the integrity of connective tissue in skin, with consequent implications for inherited connective tissue disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplkClBQ8SJtrVg90H21EOLACvtfcHk0lhYTONxYfZCIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtrnP&md5=a326ff562fe4570a3625c58faf1aadfd</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305464110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305464110%26sid%3Dliteratum%253Aachs%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DArnold%26aufirst%3DP.%26aulast%3DVadon-Le%2BGoff%26aufirst%3DS.%26aulast%3DKonerding%26aufirst%3DM.%2BA.%26aulast%3DBahr%26aufirst%3DK.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26aulast%3DKoudelka%26aufirst%3DT.%26aulast%3DTholey%26aufirst%3DA.%26aulast%3DHulmes%26aufirst%3DD.%2BJ.%2BS.%26aulast%3DMoali%26aufirst%3DC.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26atitle%3DMetalloproteases%2520meprin%2520and%2520meprin%2520are%2520C-%2520and%2520N-procollagen%2520proteinases%2520important%2520for%2520collagen%2520assembly%2520and%2520tensile%2520strength%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D14219%26epage%3D14224%26doi%3D10.1073%2Fpnas.1305464110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biasin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wygrecka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brcic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klepetko, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olschewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwapiszewska, G.</span></span> <span> </span><span class="NLM_article-title">Meprin β contributes to collagen deposition in lung fibrosis</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">39969</span>, <span class="refDoi"> DOI: 10.1038/srep39969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1038%2Fsrep39969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=28059112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=39969&author=V.+Biasinauthor=M.+Wygreckaauthor=L.+M.+Marshauthor=C.+Becker-Paulyauthor=L.+Brcicauthor=B.+Ghanimauthor=W.+Klepetkoauthor=A.+Olschewskiauthor=G.+Kwapiszewska&title=Meprin+%CE%B2+contributes+to+collagen+deposition+in+lung+fibrosis&doi=10.1038%2Fsrep39969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Meprin β contributes to collagen deposition in lung fibrosis</span></div><div class="casAuthors">Biasin, V.; Wygrecka, M.; Marsh, L. M.; Becker-Pauly, C.; Brcic, L.; Ghanim, B.; Klepetko, W.; Olschewski, A.; Kwapiszewska, G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39969</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lung fibrosis is a severe disease characterized by epithelial cell injury, inflammation and collagen deposition.  The metalloproteases meprinα and meprinβ have been shown to enhance collagen maturation and inflammatory cell infiltration via cleavage of cell-cell contact mols.; therefore we hypothesized that meprins could play a role in lung fibrosis.  An exhaustive characterization of bleomycin-treated meprinα, meprinβ and the double meprinsαβ knock-out (KO) with resp. wt-littermates was performed by using several different methods.  We obsd. no difference in lung function parameters and no change in inflammatory cells infiltrating the lung between wt and all meprins KO mice after 14 days bleomycin.  No difference in epithelial integrity as assessed by e-cadherin protein level was detected in bleomycin-treated lungs.  However, morphol. anal. in the bleomycin-treated mice revealed decrease collagen deposition and tissue d. in meprinβ KO, but not in meprinα and meprinαβ KO mice.  This finding was accompanied by localization of meprinβ to epithelial cells in regions with immature collagen in mice.  Similarly, in human IPF lungs meprinβ was mostly localized in epithelium.  These findings suggest that local environment triggers meprinβ expression to support collagen maturation.  In conclusion, our data demonstrate the in vivo relevance of meprinβ in collagen deposition in lung fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAeLWcsnM5_rVg90H21EOLACvtfcHk0lhYTONxYfZCIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlensQ%253D%253D&md5=7de6854a178a98d2da31810315a9a229</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsrep39969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39969%26sid%3Dliteratum%253Aachs%26aulast%3DBiasin%26aufirst%3DV.%26aulast%3DWygrecka%26aufirst%3DM.%26aulast%3DMarsh%26aufirst%3DL.%2BM.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DBrcic%26aufirst%3DL.%26aulast%3DGhanim%26aufirst%3DB.%26aulast%3DKlepetko%26aufirst%3DW.%26aulast%3DOlschewski%26aufirst%3DA.%26aulast%3DKwapiszewska%26aufirst%3DG.%26atitle%3DMeprin%2520%25CE%25B2%2520contributes%2520to%2520collagen%2520deposition%2520in%2520lung%2520fibrosis%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D39969%26doi%3D10.1038%2Fsrep39969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matters, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gailey, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Balance of meprin A and B in mice affects the progression of experimental inflammatory bowel disease</span>. <i>Am. J. Physiol. Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">G273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00504.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1152%2Fajpgi.00504.2009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2011&pages=G273-282&author=S.+Banerjeeauthor=G.+Jinauthor=S.+G.+Bradleyauthor=G.+L.+Mattersauthor=R.+D.+Gaileyauthor=J.+M.+Crismanauthor=J.+S.+Bond&title=Balance+of+meprin+A+and+B+in+mice+affects+the+progression+of+experimental+inflammatory+bowel+disease&doi=10.1152%2Fajpgi.00504.2009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00504.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00504.2009%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DBradley%26aufirst%3DS.%2BG.%26aulast%3DMatters%26aufirst%3DG.%2BL.%26aulast%3DGailey%26aufirst%3DR.%2BD.%26aulast%3DCrisman%26aufirst%3DJ.%2BM.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DBalance%2520of%2520meprin%2520A%2520and%2520B%2520in%2520mice%2520affects%2520the%2520progression%2520of%2520experimental%2520inflammatory%2520bowel%2520disease%26jtitle%3DAm.%2520J.%2520Physiol.%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2011%26volume%3D300%26spage%3DG273%26epage%3D282%26doi%3D10.1152%2Fajpgi.00504.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">The metalloprotease meprin β is an alternative β-secretase of APP</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">146</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2016.00159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.3389%2Ffnmol.2016.00159" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=146&author=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=The+metalloprotease+meprin+%CE%B2+is+an+alternative+%CE%B2-secretase+of+APP&doi=10.3389%2Ffnmol.2016.00159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2016.00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2016.00159%26sid%3Dliteratum%253Aachs%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DThe%2520metalloprotease%2520meprin%2520%25CE%25B2%2520is%2520an%2520alternative%2520%25CE%25B2-secretase%2520of%2520APP%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2017%26volume%3D9%26spage%3D146%26doi%3D10.3389%2Ffnmol.2016.00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Causevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jumpertz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Multhaup, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">The metalloprotease meprin generates amino terminal-truncated amyloid peptide species</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">33304</span>– <span class="NLM_lpage">33313</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.395608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fjbc.M112.395608" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=33304-33313&author=J.+Bienauthor=T.+Jeffersonauthor=M.+Causevicauthor=T.+Jumpertzauthor=L.+Munterauthor=G.+Multhaupauthor=S.+Weggenauthor=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=The+metalloprotease+meprin+generates+amino+terminal-truncated+amyloid+peptide+species&doi=10.1074%2Fjbc.M112.395608"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.395608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.395608%26sid%3Dliteratum%253Aachs%26aulast%3DBien%26aufirst%3DJ.%26aulast%3DJefferson%26aufirst%3DT.%26aulast%3DCausevic%26aufirst%3DM.%26aulast%3DJumpertz%26aufirst%3DT.%26aulast%3DMunter%26aufirst%3DL.%26aulast%3DMulthaup%26aufirst%3DG.%26aulast%3DWeggen%26aufirst%3DS.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DThe%2520metalloprotease%2520meprin%2520generates%2520amino%2520terminal-truncated%2520amyloid%2520peptide%2520species%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D33304%26epage%3D33313%26doi%3D10.1074%2Fjbc.M112.395608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenthaler, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrzik, C. U.</span></span> <span> </span><span class="NLM_article-title">Generation of aggregation prone N-terminally truncated amyloid β peptides by meprin β depends on the sequence specificity at the cleavage site</span>. <i>Mol. Neurodegener.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1186/s13024-016-0084-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1186%2Fs13024-016-0084-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=26758977" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=4&author=C.+Sch%C3%B6nherrauthor=J.+Bienauthor=S.+Isbertauthor=R.+Wichertauthor=J.+Proxauthor=H.+Altmeppenauthor=S.+Kumarauthor=J.+Walterauthor=S.+F.+Lichtenthalerauthor=S.+Weggenauthor=M.+Glatzelauthor=C.+Becker-Paulyauthor=C.+U.+Pietrzik&title=Generation+of+aggregation+prone+N-terminally+truncated+amyloid+%CE%B2+peptides+by+meprin+%CE%B2+depends+on+the+sequence+specificity+at+the+cleavage+site&doi=10.1186%2Fs13024-016-0084-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2Fs13024-016-0084-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13024-016-0084-5%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DC.%26aulast%3DBien%26aufirst%3DJ.%26aulast%3DIsbert%26aufirst%3DS.%26aulast%3DWichert%26aufirst%3DR.%26aulast%3DProx%26aufirst%3DJ.%26aulast%3DAltmeppen%26aufirst%3DH.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DWalter%26aufirst%3DJ.%26aulast%3DLichtenthaler%26aufirst%3DS.%2BF.%26aulast%3DWeggen%26aufirst%3DS.%26aulast%3DGlatzel%26aufirst%3DM.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DPietrzik%26aufirst%3DC.%2BU.%26atitle%3DGeneration%2520of%2520aggregation%2520prone%2520N-terminally%2520truncated%2520amyloid%2520%25CE%25B2%2520peptides%2520by%2520meprin%2520%25CE%25B2%2520depends%2520on%2520the%2520sequence%2520specificity%2520at%2520the%2520cleavage%2520site%26jtitle%3DMol.%2520Neurodegener.%26date%3D2016%26volume%3D11%26spage%3D4%26doi%3D10.1186%2Fs13024-016-0084-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lottaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yiallouros, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span> <span> </span><span class="NLM_article-title">Human meprin alpha and beta homo-oligomers: cleavage of basement membrane proteins and sensitivity to metalloprotease inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1042/bj20031163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1042%2Fbj20031163" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2004&pages=383-389&author=M.-N.+Kruseauthor=C.+Beckerauthor=D.+Lottazauthor=D.+K%C3%B6hlerauthor=I.+Yiallourosauthor=H.-W.+Krellauthor=E.+E.+Sterchiauthor=W.+St%C3%B6cker&title=Human+meprin+alpha+and+beta+homo-oligomers%3A+cleavage+of+basement+membrane+proteins+and+sensitivity+to+metalloprotease+inhibitors&doi=10.1042%2Fbj20031163"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2Fbj20031163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20031163%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DM.-N.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DLottaz%26aufirst%3DD.%26aulast%3DK%25C3%25B6hler%26aufirst%3DD.%26aulast%3DYiallouros%26aufirst%3DI.%26aulast%3DKrell%26aufirst%3DH.-W.%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26atitle%3DHuman%2520meprin%2520alpha%2520and%2520beta%2520homo-oligomers%253A%2520cleavage%2520of%2520basement%2520membrane%2520proteins%2520and%2520sensitivity%2520to%2520metalloprotease%2520inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2004%26volume%3D378%26spage%3D383%26epage%3D389%26doi%3D10.1042%2Fbj20031163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madoux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tredup, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scampavia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minond, D.</span></span> <span> </span><span class="NLM_article-title">Development of high throughput screening assays and pilot screen for inhibitors of metalloproteases meprin α and β</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1002/bip.22527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1002%2Fbip.22527" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2014&pages=396-406&author=F.+Madouxauthor=C.+Tredupauthor=T.+P.+Spicerauthor=L.+Scampaviaauthor=P.+S.+Chaseauthor=P.+S.+Hodderauthor=G.+B.+Fieldsauthor=C.+Becker-Paulyauthor=D.+Minond&title=Development+of+high+throughput+screening+assays+and+pilot+screen+for+inhibitors+of+metalloproteases+meprin+%CE%B1+and+%CE%B2&doi=10.1002%2Fbip.22527"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fbip.22527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22527%26sid%3Dliteratum%253Aachs%26aulast%3DMadoux%26aufirst%3DF.%26aulast%3DTredup%26aufirst%3DC.%26aulast%3DSpicer%26aufirst%3DT.%2BP.%26aulast%3DScampavia%26aufirst%3DL.%26aulast%3DChase%26aufirst%3DP.%2BS.%26aulast%3DHodder%26aufirst%3DP.%2BS.%26aulast%3DFields%26aufirst%3DG.%2BB.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DMinond%26aufirst%3DD.%26atitle%3DDevelopment%2520of%2520high%2520throughput%2520screening%2520assays%2520and%2520pilot%2520screen%2520for%2520inhibitors%2520of%2520metalloproteases%2520meprin%2520%25CE%25B1%2520and%2520%25CE%25B2%26jtitle%3DBiopolymers%26date%3D2014%26volume%3D102%26spage%3D396%26epage%3D406%26doi%3D10.1002%2Fbip.22527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barre, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">auf dem Keller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappelhoff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteomic analyses reveal an acidic prime side specificity for the astacin metalloprotease family reflected by physiological substrates</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">M111.009233</span>, <span class="refDoi"> DOI: 10.1074/mcp.M111.009233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fmcp.M111.009233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=21693781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Gjtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M111.009233&author=C.+Becker-Paulyauthor=O.+Barreauthor=O.+Schillingauthor=U.+auf+dem+Kellerauthor=A.+Ohlerauthor=C.+Broderauthor=A.+Schutteauthor=R.+Kappelhoffauthor=W.+St%C3%B6ckerauthor=C.+M.+Overall&title=Proteomic+analyses+reveal+an+acidic+prime+side+specificity+for+the+astacin+metalloprotease+family+reflected+by+physiological+substrates&doi=10.1074%2Fmcp.M111.009233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analyses reveal an acidic prime side specificity for the astacin metalloprotease family reflected by physiological substrates</span></div><div class="casAuthors">Becker-Pauly, Christoph; Barre, Olivier; Schilling, Oliver; auf dem Keller, Ulrich; Ohler, Anke; Broder, Claudia; Schuette, Andre; Kappelhoff, Reinhild; Stoecker, Walter; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">M111.009233, 19 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Astacins are secreted and membrane-bound metalloproteases with clear assocns. to many important pathol. and physiol. processes.  Yet with only a few substrates described their biol. roles are enigmatic.  Moreover, the lack of knowledge of astacin cleavage site specificities hampers assay and drug development.  Using PICS (proteomic identification of protease cleavage site specificity) and TAILS (terminal amine isotopic labeling of substrates) degradomics approaches >3000 cleavage sites were proteomically identified for five different astacins.  Such broad coverage enables family-wide detn. of specificities N- and C-terminal to the scissile peptide bond.  Remarkably, meprin α, meprin β, and LAST_MAM proteases exhibit a strong preference for aspartate in the peptide (P)1' position because of a conserved pos. charged residue in the active cleft subsite (S)1'.  This unparalleled specificity has not been found for other families of extracellular proteases.  Interestingly, cleavage specificity is also strongly influenced by proline in P2' or P3' leading to a rare example of subsite cooperativity.  This specificity characterizes the astacins as unique contributors to extracellular proteolysis that is corroborated by known cleavage sites in procollagen I+III, VEGF (vascular endothelial growth factor)-A, IL (interleukin)-1β, and pro-kallikrein 7.  Indeed, cleavage sites in VEGF-A and pro-kallikrein 7 identified by terminal amine isotopic labeling of substrates matched those reported by Edman degrdn.  Moreover, the novel substrate FGF-19 was validated biochem. and shown to exhibit altered biol. activity after meprin processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptuiOk0MgRobVg90H21EOLACvtfcHk0liKYAKzFu93vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Gjtb%252FE&md5=6645b26c29b358d1c38b77c4101d2809</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M111.009233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M111.009233%26sid%3Dliteratum%253Aachs%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DBarre%26aufirst%3DO.%26aulast%3DSchilling%26aufirst%3DO.%26aulast%3Dauf%2Bdem%2BKeller%26aufirst%3DU.%26aulast%3DOhler%26aufirst%3DA.%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DSchutte%26aufirst%3DA.%26aulast%3DKappelhoff%26aufirst%3DR.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DProteomic%2520analyses%2520reveal%2520an%2520acidic%2520prime%2520side%2520specificity%2520for%2520the%2520astacin%2520metalloprotease%2520family%2520reflected%2520by%2520physiological%2520substrates%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM111.009233%26doi%3D10.1074%2Fmcp.M111.009233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arolas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guevara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterchi, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Ruth, F. X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the sheddase function of human meprin metalloproteinase at the plasma membrane</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16131</span>– <span class="NLM_lpage">16136</span>, <span class="refDoi"> DOI: 10.1073/pnas.1211076109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1073%2Fpnas.1211076109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16131-16136&author=J.+L.+Arolasauthor=C.+Broderauthor=T.+Jeffersonauthor=T.+Guevaraauthor=E.+E.+Sterchiauthor=W.+Bodeauthor=W.+St%C3%B6ckerauthor=C.+Becker-Paulyauthor=F.+X.+Gomis-Ruth&title=Structural+basis+for+the+sheddase+function+of+human+meprin+metalloproteinase+at+the+plasma+membrane&doi=10.1073%2Fpnas.1211076109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1211076109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1211076109%26sid%3Dliteratum%253Aachs%26aulast%3DArolas%26aufirst%3DJ.%2BL.%26aulast%3DBroder%26aufirst%3DC.%26aulast%3DJefferson%26aufirst%3DT.%26aulast%3DGuevara%26aufirst%3DT.%26aulast%3DSterchi%26aufirst%3DE.%2BE.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DGomis-Ruth%26aufirst%3DF.%2BX.%26atitle%3DStructural%2520basis%2520for%2520the%2520sheddase%2520function%2520of%2520human%2520meprin%2520metalloproteinase%2520at%2520the%2520plasma%2520membrane%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16131%26epage%3D16136%26doi%3D10.1073%2Fpnas.1211076109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsbeck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenzig, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchholz, M.</span></span> <span> </span><span class="NLM_article-title">First insight into structure-activity relationships of selective meprin β inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2428</span>– <span class="NLM_lpage">2431</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.bmcl.2017.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2428-2431&author=D.+Ramsbeckauthor=A.+Hamannauthor=D.+Schlenzigauthor=S.+Schillingauthor=M.+Buchholz&title=First+insight+into+structure-activity+relationships+of+selective+meprin+%CE%B2+inhibitors&doi=10.1016%2Fj.bmcl.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DRamsbeck%26aufirst%3DD.%26aulast%3DHamann%26aufirst%3DA.%26aulast%3DSchlenzig%26aufirst%3DD.%26aulast%3DSchilling%26aufirst%3DS.%26aulast%3DBuchholz%26aufirst%3DM.%26atitle%3DFirst%2520insight%2520into%2520structure-activity%2520relationships%2520of%2520selective%2520meprin%2520%25CE%25B2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2428%26epage%3D2431%26doi%3D10.1016%2Fj.bmcl.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Recent advances in MMP inhibitor design</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1007/s10555-006-7894-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1007%2Fs10555-006-7894-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=16680577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD28Xktlaksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=115-136&author=J.+F.+Fisherauthor=S.+Mobashery&title=Recent+advances+in+MMP+inhibitor+design&doi=10.1007%2Fs10555-006-7894-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in MMP inhibitor design</span></div><div class="casAuthors">Fisher, Jed F.; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-136</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The search for an MMP inhibitor with anticancer efficacy is a nearly three-decade endeavor.  This inhibitor is yet to be found.  The reasons for this failure include shortcomings in the chem. of these compds. (including broad MMP sub-type selectivity, metabolic lability, and toxicity) as well as the emerging, and arguably extraordinary, complexity of MMP cell (and cancer) biol.  Together these suggest that the successful anticancer inhibitor must possess MMP selectivity against the MMP subtype whose involvement is crit., yet highly temporally (with respect to metastatic progression) and mechanistically (with respect to matrix degrdn.) regulated.  This review summarizes the progression of chem. structure and mechanistic thinking toward these objectives, with emphasis on the disappointment, the perseverance, and the resilient optimism that such an inhibitor is there to be discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqgy2dldRRXLVg90H21EOLACvtfcHk0liKYAKzFu93vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xktlaksrs%253D&md5=ceed8438ad533dac294e22102e5ba8bf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10555-006-7894-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-006-7894-9%26sid%3Dliteratum%253Aachs%26aulast%3DFisher%26aufirst%3DJ.%2BF.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520MMP%2520inhibitor%2520design%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2006%26volume%3D25%26spage%3D115%26epage%3D136%26doi%3D10.1007%2Fs10555-006-7894-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yiotakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dive, V.</span></span> <span> </span><span class="NLM_article-title">Synthetic active site-directed inhibitors of metzincins: achievement and perspectives</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.mam.2008.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=18657570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=329-338&author=A.+Yiotakisauthor=V.+Dive&title=Synthetic+active+site-directed+inhibitors+of+metzincins%3A+achievement+and+perspectives&doi=10.1016%2Fj.mam.2008.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic active site-directed inhibitors of metzincins: Achievement and perspectives</span></div><div class="casAuthors">Yiotakis, Athanasios; Dive, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-338</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The involvement of many zinc metalloproteinases belonging to the metzincin family with a variety of pathol. states raises the possibility of therapeutic intervention using synthetic inhibitors with appropriate selectivity.  Knowledge of the catalytic domain 3D-structures for various members of the metzincin family has been successfully exploited by chemists to develop potent synthetic inhibitors.  However, despite intense efforts, very few highly selective inhibitors of metzincins have been discovered up to now.  A survey of the literature suggests that the over-exploitation of the hydroxamate function as a zinc-binding group to develop inhibitors might be responsible for this situation.  The use of alternative zinc-binding groups has led to more selective inhibitors, but the most encouraging results have been obtained for MMP-13 with compds. that do not incorporate zinc-binding groups in their structure.  This new family of inhibitors exploits the presence of a deep S 1 ' cavity in the protease active site, a specific trait shared by many members of the metzincin family.  However, to be successfully transposed to the metzincin members, this strategy will not only be able to exploit the structural detail of these S 1 ' cavities, but probably also subtle difference in their dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGory6HknHN-obVg90H21EOLACvtfcHk0liKYAKzFu93vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ShsLnP&md5=44bb1c612fad4769f3bdea1de9ba4792</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYiotakis%26aufirst%3DA.%26aulast%3DDive%26aufirst%3DV.%26atitle%3DSynthetic%2520active%2520site-directed%2520inhibitors%2520of%2520metzincins%253A%2520achievement%2520and%2520perspectives%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2008%26volume%3D29%26spage%3D329%26epage%3D338%26doi%3D10.1016%2Fj.mam.2008.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. P.</span></span> <span> </span><span class="NLM_article-title">QSAR studies on hydroxamic acids: a fascinating family of chemicals with a wide spectrum of activities</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">6427</span>– <span class="NLM_lpage">6490</span>, <span class="refDoi"> DOI: 10.1021/cr500483r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500483r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFaju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=6427-6490&author=S.+P.+Gupta&title=QSAR+studies+on+hydroxamic+acids%3A+a+fascinating+family+of+chemicals+with+a+wide+spectrum+of+activities&doi=10.1021%2Fcr500483r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Studies on Hydroxamic Acids: A Fascinating Family of Chemicals with a Wide Spectrum of Activities</span></div><div class="casAuthors">Gupta, Satya P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6427-6490</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lCpf2GPOi7Vg90H21EOLACvtfcHk0liVZHxowl73dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFaju7o%253D&md5=2adbf316cd257ce2a4c2fd7da1d5090f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcr500483r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500483r%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DS.%2BP.%26atitle%3DQSAR%2520studies%2520on%2520hydroxamic%2520acids%253A%2520a%2520fascinating%2520family%2520of%2520chemicals%2520with%2520a%2520wide%2520spectrum%2520of%2520activities%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D6427%26epage%3D6490%26doi%3D10.1021%2Fcr500483r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saravanan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span> <span> </span><span class="NLM_article-title">Sulphonamides: deserving class as MMP inhibitors?</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.ejmech.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=23287054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVeiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2013&pages=89-100&author=P.+Jainauthor=C.+Saravananauthor=S.+K.+Singh&title=Sulphonamides%3A+deserving+class+as+MMP+inhibitors%3F&doi=10.1016%2Fj.ejmech.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sulphonamides: Deserving class as MMP inhibitors?</span></div><div class="casAuthors">Jain, Pranjali; Saravanan, Chinnadurai; Singh, Sushil Kumar</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The importance of sulfonamide moiety in medicinal chem. cannot be ignored as it constitutes an important class of extensively used drugs.  Recently, sulfonamides have also been reported for their matrix metalloproteinase (MMP) inhibitory activity.  MMPs are calcium- and zinc-dependent endopeptidases, involved in both inter- and intra-cellular activity.  This review documents the emergence of sulfonamides as matrix metalloproteinase inhibitors (MMPIs) from the first generation to the recent third generation MMPIs, their mode of action - how sulfonamides act on MMPs as well as the structure activity relationship along with their therapeutic uses in chronic obstructive pulmonary disease (COPD), ulcer, asthma, arthritis and cancer.  From this review, readers can get answer for the question- is sulfonamides a potential class of MMPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1FEujtONsfbVg90H21EOLACvtfcHk0liVZHxowl73dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVeiu7o%253D&md5=10dc47ad65684dad2cd3622b82378d2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DP.%26aulast%3DSaravanan%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26atitle%3DSulphonamides%253A%2520deserving%2520class%2520as%2520MMP%2520inhibitors%253F%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D60%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.ejmech.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grams, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinemer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Rüth, F.
X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span> <span> </span><span class="NLM_article-title">The metzincins-topological and sequential relations between the astacins, adamalysins, serralysins, and matrixins (collagenases) define a superfamily of zinc-peptidases</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1002/pro.5560040502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1002%2Fpro.5560040502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=7663339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A280%3ADyaK2Mzpt1ertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=823-840&author=W.+St%C3%B6ckerauthor=F.+Gramsauthor=U.+Baumannauthor=P.+Reinemerauthor=F.%0AX.+Gomis-R%C3%BCthauthor=D.+B.+McKayauthor=W.+Bode&title=The+metzincins-topological+and+sequential+relations+between+the+astacins%2C+adamalysins%2C+serralysins%2C+and+matrixins+%28collagenases%29+define+a+superfamily+of+zinc-peptidases&doi=10.1002%2Fpro.5560040502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The metzincins--topological and sequential relations between the astacins, adamalysins, serralysins, and matrixins (collagenases) define a superfamily of zinc-peptidases</span></div><div class="casAuthors">Stocker W; Grams F; Baumann U; Reinemer P; Gomis-Ruth F X; McKay D B; Bode W</div><div class="citationInfo"><span class="NLM_cas:title">Protein science : a publication of the Protein Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">823-40</span>
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    </div><div class="casAbstract">The three-dimensional structures of the zinc endopeptidases human neutrophil collagenase, adamalysin II from rattle snake venom, alkaline proteinase from Pseudomonas aeruginosa, and astacin from crayfish are topologically similar, with respect to a five-stranded beta-sheet and three alpha-helices arranged in typical sequential order.  The four proteins exhibit the characteristic consensus motif HEXXHXXGXXH, whose three histidine residues are involved in binding of the catalytically essential zinc ion.  Moreover, they all share a conserved methionine residue beneath the active site metal as part of a superimposable "Met-turn." This structural relationship is supported by a sequence alignment performed on the basis of topological equivalence showing faint but distinct sequential similarity.  The alkaline proteinase is about equally distant (26% sequence identity) to both human neutrophil collagenase and astacin and a little further away from adamalysin II (17% identity).  The pairs astacin/adamalysin II, astacin/human neutrophil collagenase, and adamalysin II/human neutrophil collagenase exhibit sequence identities of 16%, 14%, and 13%, respectively.  Therefore, the corresponding four distinct families of zinc peptidases, the astacins, the matrix metalloproteinases (matrixins, collagenases), the adamalysins/reprolysins (snake venom proteinases/reproductive tract proteins), and the serralysins (large bacterial proteases from Serratia, Erwinia, and Pseudomonas) appear to have originated by divergent evolution from a common ancestor and form a superfamily of proteolytic enzymes for which the designation "metzincins" has been proposed.  There is also a faint but significant structural relationship of the metzincins to the thermolysin-like enzymes, which share the truncated zinc-binding motif HEXXH and, moreover, similar topologies in their N-terminal domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqG27Pg4uBlQaK-_ZudimnfW6udTcc2eazTz-TcJlnX7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2Mzpt1ertw%253D%253D&md5=f79a467f5e62c72238bf01f10ebfed26</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560040502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560040502%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26aulast%3DGrams%26aufirst%3DF.%26aulast%3DBaumann%26aufirst%3DU.%26aulast%3DReinemer%26aufirst%3DP.%26aulast%3DGomis-R%25C3%25BCth%26aufirst%3DF.%2BX.%26aulast%3DMcKay%26aufirst%3DD.%2BB.%26aulast%3DBode%26aufirst%3DW.%26atitle%3DThe%2520metzincins-topological%2520and%2520sequential%2520relations%2520between%2520the%2520astacins%252C%2520adamalysins%252C%2520serralysins%252C%2520and%2520matrixins%2520%2528collagenases%2529%2520define%2520a%2520superfamily%2520of%2520zinc-peptidases%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D823%26epage%3D840%26doi%3D10.1002%2Fpro.5560040502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, T.</span></span> <span> </span><span class="NLM_article-title">Shedding light on designing potential meprin β inhibitors through ligand-based robust validated computational approaches. A proposal to chemists!</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1080/07391102.2017.1374210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1080%2F07391102.2017.1374210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=28862075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A280%3ADC%252BC1cbjslequg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1-20&author=N.+Adhikariauthor=S.+A.+Aminauthor=B.+Ghoshauthor=T.+Jha&title=Shedding+light+on+designing+potential+meprin+%CE%B2+inhibitors+through+ligand-based+robust+validated+computational+approaches.+A+proposal+to+chemists%21&doi=10.1080%2F07391102.2017.1374210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Shedding light on designing potential meprin β inhibitors through ligand-based robust validated computational approaches: A proposal to chemists!</span></div><div class="casAuthors">Adhikari Nilanjan; Amin Sk Abdul; Jha Tarun; Ghosh Balaram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human meprin (EC 3.4.24.18) is a member of the metzincin superfamily.  It correlates with matrix metalloproteinases and ADAMs (a disintegrin and metalloproteinase).  Overexpression of meprin β is implicated in fibrosis, inflammatory diseases and cancers.  However, selective meprin β inhibition is crucial to reduce cancer metastasis and adverse effects in inflammation.  It also plays critical roles in modulating several interleukins and growth factors.  Moreover, meprin β cleaves amyloid precursor protein, thought to be involved in the progression of Alzheimer's disease.  Therefore, meprin β inhibitors are considered to be emerging therapeutics with paramount importance in the treatment of kidney failure, fibrosis, inflammatory bowel diseases and cancer.  Despite its crucial implication in several diseases, no meprin β inhibitors are available as drug candidates till date.  Therefore, it is an urgent need to identify new potential meprin β inhibitors as prospective therapeutics.  In this article, a series of meprin β inhibitors has been analysed through multiple molecular modelling studies as the first initiative to get an idea about their structural, physicochemical and pharmacophoric requirements for higher activity.  All in silico approaches performed here are statistically validated and subsequently adjudicated each other.  Compounds with p-carboxylic acid substituted arylsulphonamide moiety attached with m-carboxylic acid substituted benzyl group along with a methylene hydroxamate function may be crucial for imparting potential meprin β inhibition.  Depending on the results obtained, 14 molecules have been proposed by QSAR model that predicted a minimum of 4-fold higher activity compared to these compounds of the current study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQciZA6A96nPiM8MEzWGjS9fW6udTcc2eYanFxjMP-yTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbjslequg%253D%253D&md5=58341e01d051248190a26412772c0a38</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1080%2F07391102.2017.1374210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2017.1374210%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DAmin%26aufirst%3DS.%2BA.%26aulast%3DGhosh%26aufirst%3DB.%26aulast%3DJha%26aufirst%3DT.%26atitle%3DShedding%2520light%2520on%2520designing%2520potential%2520meprin%2520%25CE%25B2%2520inhibitors%2520through%2520ligand-based%2520robust%2520validated%2520computational%2520approaches.%2520A%2520proposal%2520to%2520chemists%2521%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2017%26spage%3D1%26epage%3D20%26doi%3D10.1080%2F07391102.2017.1374210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, K.</span>; <span class="NLM_string-name">Jäger, C.</span>; <span class="NLM_string-name">Schlenzig, D.</span>; <span class="NLM_string-name">Schilling, S.</span>; <span class="NLM_string-name">Buchholz, M.</span>; <span class="NLM_string-name">Ramsbeck, D.</span></span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Tan%2C+K.%3B+J%C3%A4ger%2C+C.%3B+Schlenzig%2C+D.%3B+Schilling%2C+S.%3B+Buchholz%2C+M.%3B+Ramsbeck%2C+D.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DK." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lassalas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasfargeas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayendran, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span> <span> </span><span class="NLM_article-title">Structure property relationships of carboxylic acid isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3203</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3183-3203&author=P.+Lassalasauthor=B.+Gayauthor=C.+Lasfargeasauthor=M.+J.+Jamesauthor=V.+Tranauthor=K.+G.+Vijayendranauthor=K.+R.+Brundenauthor=M.+C.+Kozlowskiauthor=C.+J.+Thomasauthor=A.+B.+Smithauthor=D.+M.+Hurynauthor=C.+Ballatore&title=Structure+property+relationships+of+carboxylic+acid+isosteres&doi=10.1021%2Facs.jmedchem.5b01963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Property Relationships of Carboxylic Acid Isosteres</span></div><div class="casAuthors">Lassalas, Pierrik; Gay, Bryant; Lasfargeas, Caroline; James, Michael J.; Tran, Van; Vijayendran, Krishna G.; Brunden, Kurt R.; Kozlowski, Marisa C.; Thomas, Craig J.; Smith, Amos B.; Huryn, Donna M.; Ballatore, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3183-3203</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The replacement of a carboxylic acid with a surrogate structure, or (bio)-isostere, is a classical strategy in medicinal chem.  The general underlying principle is that by maintaining the features of the carboxylic acid crit. for biol. activity, but appropriately modifying the physicochem. properties, improved analogs may result.  In this context, a systematic assessment of the physicochem. properties of carboxylic acid isosteres would be desirable to enable more informed decisions of potential replacements to be used for analog design.  Herein we report the structure-property relationships (SPR) of 35 phenylpropionic acid derivs., in which the carboxylic acid moiety is replaced with a series of known isosteres.  The data set generated provides an assessment of the relative impact on the physicochem. properties that these replacements may have compared to the carboxylic acid analog.  As such, this study presents a framework for how to rationally apply isosteric replacements of the carboxylic acid functional group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIF98i2ITNrLVg90H21EOLACvtfcHk0lhWo0v40tNcEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D&md5=34490fc72fdb321d8c9751bef1bf79c5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01963%26sid%3Dliteratum%253Aachs%26aulast%3DLassalas%26aufirst%3DP.%26aulast%3DGay%26aufirst%3DB.%26aulast%3DLasfargeas%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DVijayendran%26aufirst%3DK.%2BG.%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DKozlowski%26aufirst%3DM.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26aulast%3DBallatore%26aufirst%3DC.%26atitle%3DStructure%2520property%2520relationships%2520of%2520carboxylic%2520acid%2520isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3183%26epage%3D3203%26doi%3D10.1021%2Facs.jmedchem.5b01963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Beirn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">2,6-Difluorophenol as a bioisostere of a carboxylic acid: bioisosteric analogues of γ-aminobutyric acid</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1021/jm980435l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980435l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=329-332&author=J.+Qiuauthor=S.+H.+Stevensonauthor=M.+J.+O%E2%80%99Beirnauthor=R.+B.+Silverman&title=2%2C6-Difluorophenol+as+a+bioisostere+of+a+carboxylic+acid%3A+bioisosteric+analogues+of+%CE%B3-aminobutyric+acid&doi=10.1021%2Fjm980435l"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm980435l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980435l%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DStevenson%26aufirst%3DS.%2BH.%26aulast%3DO%25E2%2580%2599Beirn%26aufirst%3DM.%2BJ.%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3D2%252C6-Difluorophenol%2520as%2520a%2520bioisostere%2520of%2520a%2520carboxylic%2520acid%253A%2520bioisosteric%2520analogues%2520of%2520%25CE%25B3-aminobutyric%2520acid%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D329%26epage%3D332%26doi%3D10.1021%2Fjm980435l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villa, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertenshaw, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, J. S.</span></span> <span> </span><span class="NLM_article-title">Critical amino acids in the active site of meprin metalloproteinases for substrate and peptide bond specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">42545</span>– <span class="NLM_lpage">42550</span>, <span class="refDoi"> DOI: 10.1074/jbc.M303718200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1074%2Fjbc.M303718200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=42545-42550&author=J.+P.+Villaauthor=G.+P.+Bertenshawauthor=J.+S.+Bond&title=Critical+amino+acids+in+the+active+site+of+meprin+metalloproteinases+for+substrate+and+peptide+bond+specificity&doi=10.1074%2Fjbc.M303718200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M303718200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M303718200%26sid%3Dliteratum%253Aachs%26aulast%3DVilla%26aufirst%3DJ.%2BP.%26aulast%3DBertenshaw%26aufirst%3DG.%2BP.%26aulast%3DBond%26aufirst%3DJ.%2BS.%26atitle%3DCritical%2520amino%2520acids%2520in%2520the%2520active%2520site%2520of%2520meprin%2520metalloproteinases%2520for%2520substrate%2520and%2520peptide%2520bond%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D42545%26epage%3D42550%26doi%3D10.1074%2Fjbc.M303718200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zafrani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeffet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sod-Moriah, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berliner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershonov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saphier, S.</span></span> <span> </span><span class="NLM_article-title">Difluoromethyl bioisostere. Examining the ″lipophilic hydrogen bond donor″ concept</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01691</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01691" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=797-804&author=Y.+Zafraniauthor=D.+Yeffetauthor=G.+Sod-Moriahauthor=A.+Berlinerauthor=D.+Amirauthor=D.+Marcianoauthor=E.+Gershonovauthor=S.+Saphier&title=Difluoromethyl+bioisostere.+Examining+the+%E2%80%B3lipophilic+hydrogen+bond+donor%E2%80%B3+concept&doi=10.1021%2Facs.jmedchem.6b01691"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01691%26sid%3Dliteratum%253Aachs%26aulast%3DZafrani%26aufirst%3DY.%26aulast%3DYeffet%26aufirst%3DD.%26aulast%3DSod-Moriah%26aufirst%3DG.%26aulast%3DBerliner%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DD.%26aulast%3DMarciano%26aufirst%3DD.%26aulast%3DGershonov%26aufirst%3DE.%26aulast%3DSaphier%26aufirst%3DS.%26atitle%3DDifluoromethyl%2520bioisostere.%2520Examining%2520the%2520%25E2%2580%25B3lipophilic%2520hydrogen%2520bond%2520donor%25E2%2580%25B3%2520concept%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D797%26epage%3D804%26doi%3D10.1021%2Facs.jmedchem.6b01691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0li0k0oiuVHdwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomis-Rüth, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trillo-Muyo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöcker, W.</span></span> <span> </span><span class="NLM_article-title">Functional and structural insights into astacin metallopeptidases</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>393</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1041</span>, <span class="refDoi"> DOI: 10.1515/hsz-2012-0149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1515%2Fhsz-2012-0149" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=2012&pages=1027-1041&author=F.+X.+Gomis-R%C3%BCthauthor=S.+Trillo-Muyoauthor=W.+St%C3%B6cker&title=Functional+and+structural+insights+into+astacin+metallopeptidases&doi=10.1515%2Fhsz-2012-0149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1515%2Fhsz-2012-0149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhsz-2012-0149%26sid%3Dliteratum%253Aachs%26aulast%3DGomis-R%25C3%25BCth%26aufirst%3DF.%2BX.%26aulast%3DTrillo-Muyo%26aufirst%3DS.%26aulast%3DSt%25C3%25B6cker%26aufirst%3DW.%26atitle%3DFunctional%2520and%2520structural%2520insights%2520into%2520astacin%2520metallopeptidases%26jtitle%3DBiol.%2520Chem.%26date%3D2012%26volume%3D393%26spage%3D1027%26epage%3D1041%26doi%3D10.1515%2Fhsz-2012-0149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flipo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Hydroxamates: Relationships between structure and plasma stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6790</span>– <span class="NLM_lpage">6802</span>, <span class="refDoi"> DOI: 10.1021/jm900648x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900648x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6790-6802&author=M.+Flipoauthor=J.+Chartonauthor=A.+Hocineauthor=S.+Dassonnevilleauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Hydroxamates%3A+Relationships+between+structure+and+plasma+stability&doi=10.1021%2Fjm900648x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamates: Relationships between Structure and Plasma Stability</span></div><div class="casAuthors">Flipo, Marion; Charton, Julie; Hocine, Akila; Dassonneville, Sandrine; Deprez, Benoit; Deprez-Poulain, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6790-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydroxamates are valuable tools for chem. biol. as well as interesting leads for medicinal chem.  Although many hydroxamates display nanomolar activities against metalloproteases, only three hydroxamates have reached the market, among which is the HDAC inhibitor vorinostat.  Failures in development are generally attributed to lack of selectivity, toxicity, or poor stability.  To help medicinal chemists with respect to plasma stability, the authors have performed the first and preliminary study on structure-plasma stability for hydroxamates.  The authors define some structural rules to predict or improve the plasma stability in the preclin. stage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk66ooWU1NhbVg90H21EOLACvtfcHk0li0k0oiuVHdwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ&md5=069a87988fbeb68a248d006c7f6307a0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm900648x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900648x%26sid%3Dliteratum%253Aachs%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DCharton%26aufirst%3DJ.%26aulast%3DHocine%26aufirst%3DA.%26aulast%3DDassonneville%26aufirst%3DS.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DHydroxamates%253A%2520Relationships%2520between%2520structure%2520and%2520plasma%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6790%26epage%3D6802%26doi%3D10.1021%2Fjm900648x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piveteau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gealageas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roignant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolojanahary, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renard, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemdani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourotte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herledan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Controlling plasma stability of hydroxamic acids. A MedChem toolbox</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9067</span>– <span class="NLM_lpage">9089</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01444</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01444" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9067-9089&author=P.+Hermantauthor=D.+Boscauthor=C.+Piveteauauthor=R.+Gealageasauthor=B.+Lamauthor=C.+Roncoauthor=M.+Roignantauthor=H.+Tolojanaharyauthor=L.+Jeanauthor=P.-Y.+Renardauthor=M.+Lemdaniauthor=M.+Bourotteauthor=A.+Herledanauthor=C.+Bedartauthor=A.+Bielaauthor=F.+Lerouxauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Controlling+plasma+stability+of+hydroxamic+acids.+A+MedChem+toolbox&doi=10.1021%2Facs.jmedchem.7b01444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01444%26sid%3Dliteratum%253Aachs%26aulast%3DHermant%26aufirst%3DP.%26aulast%3DBosc%26aufirst%3DD.%26aulast%3DPiveteau%26aufirst%3DC.%26aulast%3DGealageas%26aufirst%3DR.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DRonco%26aufirst%3DC.%26aulast%3DRoignant%26aufirst%3DM.%26aulast%3DTolojanahary%26aufirst%3DH.%26aulast%3DJean%26aufirst%3DL.%26aulast%3DRenard%26aufirst%3DP.-Y.%26aulast%3DLemdani%26aufirst%3DM.%26aulast%3DBourotte%26aufirst%3DM.%26aulast%3DHerledan%26aufirst%3DA.%26aulast%3DBedart%26aufirst%3DC.%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DLeroux%26aufirst%3DF.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DControlling%2520plasma%2520stability%2520of%2520hydroxamic%2520acids.%2520A%2520MedChem%2520toolbox%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9067%26epage%3D9089%26doi%3D10.1021%2Facs.jmedchem.7b01444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlenzig, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsbeck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moenke-Wedler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, S.</span></span> <span> </span><span class="NLM_article-title">Expression, purification and initial characterization of human meprin β from Pichia pastoris</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.pep.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=26256061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCis7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2015&pages=75-81&author=D.+Schlenzigauthor=M.+Wermannauthor=D.+Ramsbeckauthor=T.+Moenke-Wedlerauthor=S.+Schilling&title=Expression%2C+purification+and+initial+characterization+of+human+meprin+%CE%B2+from+Pichia+pastoris&doi=10.1016%2Fj.pep.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification and initial characterization of human meprin β from Pichia pastoris</span></div><div class="casAuthors">Schlenzig, D.; Wermann, M.; Ramsbeck, D.; Moenke-Wedler, T.; Schilling, S.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-81</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Human meprin β (h-meprin β), a single-zinc metalloendoprotease of the astacin family, is potentially involved in disorders such as fibrosis and Alzheimer's disease.  Here, the authors describe the expression of the enzyme in the yeast Pichia pastoris.  The N-terminal signal sequence was replaced by the α-leader of Saccharomyces, enabling efficient secretion of the mature enzyme, harboring either an N-terminal or C-terminal His-tag.  The purifn. by affinity and hydrophobic interaction chromatog. gave 58.4 mg/l of homogeneous human pro-meprin β from fermn. broth.  The activated enzyme isolated from yeast (yh-meprin β) displayed virtually identical enzymic activity as h-meprin from a mammalian cell line.  Furthermore, the yh-meprin β was N-glycosylated and secreted as a dimer with a mol. mass of 148 kDa.  Endoglycosidase H treatment generated a protein with a mol. mass of 133 kDa, but essentially unchanged kinetic parameters.  Thus, the authors' data suggest that human meprin β expressed in P. pastoris displays virtually identical parameters as meprin from other sources.  The high yield of protein expression, the ease of purifn. and the deglycosylation in its native state appear to favor further studies aiming at inhibitor screening and structure-based inhibitor refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMPvtA3KzZBbVg90H21EOLACvtfcHk0lgvbKH7Qufldw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCis7jL&md5=fc8e221669da77c9f437ed9d71ba7abd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchlenzig%26aufirst%3DD.%26aulast%3DWermann%26aufirst%3DM.%26aulast%3DRamsbeck%26aufirst%3DD.%26aulast%3DMoenke-Wedler%26aufirst%3DT.%26aulast%3DSchilling%26aufirst%3DS.%26atitle%3DExpression%252C%2520purification%2520and%2520initial%2520characterization%2520of%2520human%2520meprin%2520%25CE%25B2%2520from%2520Pichia%2520pastoris%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2015%26volume%3D116%26spage%3D75%26epage%3D81%26doi%3D10.1016%2Fj.pep.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mac
Sweeney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Parrado, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinzenz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodendorf, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhartz, B.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the substrate specificity of bone morphogenetic protein 1/tolloid-like metalloproteases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;key=10.1016%2Fj.jmb.2008.09.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=228-239&author=A.+Mac%0ASweeneyauthor=S.+Gil-Parradoauthor=D.+Vinzenzauthor=A.+Bernardiauthor=A.+Heinauthor=U.+Bodendorfauthor=P.+Erbelauthor=C.+Logelauthor=B.+Gerhartz&title=Structural+basis+for+the+substrate+specificity+of+bone+morphogenetic+protein+1%2Ftolloid-like+metalloproteases&doi=10.1016%2Fj.jmb.2008.09.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DMac%2BSweeney%26aufirst%3DA.%26aulast%3DGil-Parrado%26aufirst%3DS.%26aulast%3DVinzenz%26aufirst%3DD.%26aulast%3DBernardi%26aufirst%3DA.%26aulast%3DHein%26aufirst%3DA.%26aulast%3DBodendorf%26aufirst%3DU.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DLogel%26aufirst%3DC.%26aulast%3DGerhartz%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520for%2520the%2520substrate%2520specificity%2520of%2520bone%2520morphogenetic%2520protein%25201%252Ftolloid-like%2520metalloproteases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26spage%3D228%26epage%3D239%26doi%3D10.1016%2Fj.jmb.2008.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GWN','PDB','4GWN'); return false;">PDB: 4GWN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DDF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DDF','PDB','2DDF'); return false;">PDB: 2DDF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZXH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZXH','PDB','3ZXH'); return false;">PDB: 3ZXH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00330">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78120"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00330">10.1021/acs.jmedchem.8b00330</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Analytical data of intermediates, supplementary figure, and substrates used for enzymatic assays (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_001.pdf">jm8b00330_si_001.pdf (594.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00330/suppl_file/jm8b00330_si_002.csv">jm8b00330_si_002.csv (2.33 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00330&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-10%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00330" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00330" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bac93d6221a6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
