
<html lang="en"     class="pb-page"  data-request-id="0000f47e-b758-49a7-9b71-0fc4b43b55b4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b00007;issue:issue:10.1021/jmcmar.2016.59.issue-7;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors" /></meta><meta name="dc.Creator" content="Madeline E.  Kavanagh" /></meta><meta name="dc.Creator" content="Anthony G.  Coyne" /></meta><meta name="dc.Creator" content="Kirsty J.  McLean" /></meta><meta name="dc.Creator" content="Guy G.  James" /></meta><meta name="dc.Creator" content="Colin W.  Levy" /></meta><meta name="dc.Creator" content="Leonardo B.  Marino" /></meta><meta name="dc.Creator" content="Luiz Pedro S.  de Carvalho" /></meta><meta name="dc.Creator" content="Daniel S. H.  Chan" /></meta><meta name="dc.Creator" content="Sean A.  Hudson" /></meta><meta name="dc.Creator" content="Sachin  Surade" /></meta><meta name="dc.Creator" content="David  Leys" /></meta><meta name="dc.Creator" content="Andrew W.  Munro" /></meta><meta name="dc.Creator" content="Chris  Abell" /></meta><meta name="dc.Description" content="The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis. A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 us..." /></meta><meta name="Description" content="The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis. A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 us..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 22, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00007" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00007" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00007" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00007" /></link>
        
    
    

<title>Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00007" /></meta><meta property="og:title" content="Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0018.jpeg" /></meta><meta property="og:description" content="The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis. A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 using a cascade of biophysical assays. Synthetic merging and optimization of 1 produced a 100-fold improvement in binding affinity, yielding lead compound 2 (KD = 15 μM). Deconstruction of 2 into its component retrofragments allowed the group efficiency of structural motifs to be assessed, the identification of more LE scaffolds for optimization and highlighted binding affinity hotspots. Structure-guided addition of a metal-binding pharmacophore onto LE retrofragment scaffolds produced low nanomolar (KD = 15 nM) CYP121 ligands. Elaboration of these compounds to target binding hotspots in the distal active site afforded compounds with excellent selectivity against human drug-metabolizing P450s. Analysis of the factors governing ligand potency and selectivity using X-ray crystallography, UV–vis spectroscopy, and native mass spectrometry provides insight for subsequent drug development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00007"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00007">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00007&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00007&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00007&amp;href=/doi/10.1021/acs.jmedchem.6b00007" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3272-3302</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b02029" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00019" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Based Approaches to the Development of <i>Mycobacterium tuberculosis</i> CYP121 Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Madeline+E.++Kavanagh">Madeline E. Kavanagh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+G.++Coyne">Anthony G. Coyne</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kirsty+J.++McLean">Kirsty J. McLean</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guy+G.++James">Guy G. James</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Colin+W.++Levy">Colin W. Levy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Leonardo+B.++Marino">Leonardo B. Marino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luiz+Pedro+S.++de+Carvalho">Luiz Pedro S. de Carvalho</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+S.+H.++Chan">Daniel S. H. Chan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sean+A.++Hudson">Sean A. Hudson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sachin++Surade">Sachin Surade</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Leys">David Leys</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+W.++Munro">Andrew W. Munro</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chris++Abell">Chris Abell</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Centre for Synthetic Biology of Fine and Specialty Chemicals (SYNBIOCHEM), Manchester Institute of Biotechnology, Faculty of Life Sciences, University of Manchester, 131 Princess Street, Manchester M1 7DN, U.K.</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Laboratory of Mycobacterial Metabolism and Antibiotic Research, Francis Crick Institute, The Mill Hill Laboratory, London NW7 1AA, U.K.</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">School of Pharmaceutical Sciences, São Paulo State University (UNESP), 4801-902 Araraquara, SP, Brazil</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biochemistry, University of Cambridge, 80 Tennis Court Road, Cambridge CB2 1GA U.K.</span></div><div class="corresp-info"><strong>*</strong>Phone: +44 (0) 1223 336405. Fax: +44 (0) 1223 336362. E-mail: <a href="/cdn-cgi/l/email-protection#11727023275172707c3f70723f647a"><span class="__cf_email__" data-cfemail="86e5e7b4b0c6e5e7eba8e7e5a8f3ed">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00007&amp;href=/doi/10.1021%2Facs.jmedchem.6b00007" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3272–3302</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 22, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 January 2016</li><li><span class="item_label"><b>Published</b> online</span>22 March 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 April 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00007" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00007</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00007"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4068</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">34</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00007" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Madeline&quot;,&quot;last_name&quot;:&quot;E. Kavanagh&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;G. Coyne&quot;},{&quot;first_name&quot;:&quot;Kirsty&quot;,&quot;last_name&quot;:&quot;J. McLean&quot;},{&quot;first_name&quot;:&quot;Guy&quot;,&quot;last_name&quot;:&quot;G. James&quot;},{&quot;first_name&quot;:&quot;Colin&quot;,&quot;last_name&quot;:&quot;W. Levy&quot;},{&quot;first_name&quot;:&quot;Leonardo&quot;,&quot;last_name&quot;:&quot;B. Marino&quot;},{&quot;first_name&quot;:&quot;Luiz&quot;,&quot;last_name&quot;:&quot;Pedro S. de Carvalho&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;S. H. Chan&quot;},{&quot;first_name&quot;:&quot;Sean&quot;,&quot;last_name&quot;:&quot;A. Hudson&quot;},{&quot;first_name&quot;:&quot;Sachin&quot;,&quot;last_name&quot;:&quot;Surade&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Leys&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;W. Munro&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;Abell&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;3272-3302&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00007&quot;},&quot;abstract&quot;:&quot;The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis. A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 using a cascade of biophysical assays. Synthetic merging and optimization of 1 produced a 100-fold improvement in binding affinity, yielding lead compound 2 (KD = 15 μM). Deconstruction of 2 into its component retrofragments allowed the group efficiency of structural motifs to be assessed, the identification of more LE scaffolds for optimization and highlighted binding affinity hotspots. Structure-guided addition of a metal-binding pharmacophore onto LE retrofragment scaffolds produced low nanomolar (KD = 15 nM) CYP121 ligands. Elaboration of these compounds to target binding hotspots in the distal active site afforded compounds with excellent selectivity against human drug-metabolizing P450s. Analysis of the factors governing ligand potency and selectivity using X-ray crystallography, UV–vis spectroscopy, and native &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00007&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00007" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00007&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00007" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00007&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00007" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00007&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00007&amp;href=/doi/10.1021/acs.jmedchem.6b00007" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00007" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00007" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00007%26sid%3Dliteratum%253Aachs%26pmid%3D27002486%26genre%3Darticle%26aulast%3DKavanagh%26date%3D2016%26atitle%3DFragment-Based%2BApproaches%2Bto%2Bthe%2BDevelopment%2Bof%2BMycobacterium%2Btuberculosis%2BCYP121%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D7%26spage%3D3272%26epage%3D3302%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292460" title="Bioinorganic chemistry">Bioinorganic chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/jmcmar.2016.59.issue-7/20160414-01/jmcmar.2016.59.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of <i>Mycobacterium tuberculosis.</i> A triazol-1-yl phenol fragment <b>1</b> was identified to bind to CYP121 using a cascade of biophysical assays. Synthetic merging and optimization of <b>1</b> produced a 100-fold improvement in binding affinity, yielding lead compound <b>2</b> (<i>K</i><sub>D</sub> = 15 μM). Deconstruction of <b>2</b> into its component retrofragments allowed the group efficiency of structural motifs to be assessed, the identification of more LE scaffolds for optimization and highlighted binding affinity hotspots. Structure-guided addition of a metal-binding pharmacophore onto LE retrofragment scaffolds produced low nanomolar (<i>K</i><sub>D</sub> = 15 nM) CYP121 ligands. Elaboration of these compounds to target binding hotspots in the distal active site afforded compounds with excellent selectivity against human drug-metabolizing P450s. Analysis of the factors governing ligand potency and selectivity using X-ray crystallography, UV–vis spectroscopy, and native mass spectrometry provides insight for subsequent drug development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06675" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Tuberculosis (TB) is responsible for the death of 1.5 million people annually.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The development of antibiotic resistant strains of the causal pathogen <i>Mycobacterium tuberculosis</i> (<i>Mtb</i>) and comorbidity with immunosuppressive conditions such as HIV and diabetes are major contributors to the current crisis and there is an urgent need for the development of new drugs with novel mechanisms of action.</div><div class="NLM_p">Cytochrome P450 enzymes (P450s or CYPs) are a currently unexploited class of protein targets for TB therapeutics. The <i>Mtb</i> genome encodes genes for 20 P450 isoforms, which is more than 200 times as many as humans for the relative size of the genome.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The large proportion of genetic material dedicated to P450s suggests that these enzymes might have important functions for the survival and virulence of the pathogen.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Biochemical and genetic studies over the past 15 years have provided evidence of essential roles for <i>Mtb</i> P450s in bacterial viability, the catabolism of host cholesterol, establishment of infection, persistence in the granuloma, cellular stress response mechanisms, and drug susceptibility.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14">(4-14)</a> A number of azole-containing antifungal compounds that have potent antimycotic activity against <i>Mtb</i> H37Rv both in vitro and in murine models of TB, have tight binding affinity (<i>K</i><sub>D</sub>) for a number of the <i>Mtb</i> P450s, which provides further support for these enzymes as potential drug targets.<a onclick="showRef(event, 'ref12 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref15 ref16">(12, 15, 16)</a> There is correlation between the minimum inhibitory concentration (MIC) values of the most potent azole compounds clotrimazole, econazole, and miconazole, which have MICs of 11, 8, and 8 μg/mL, respectively, against <i>Mtb</i> H37Rv,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and their <i>K</i><sub>D</sub>’s for the most strongly inhibited P450 CYP121 (<i>K</i><sub>D</sub> = 73–136 nM).<a onclick="showRef(event, 'ref12 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref17">(12, 17)</a> In addition, evidence that the gene encoding CYP121 (<i>rv2276</i>) is essential for <i>Mtb</i> H37Rv viability<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> implies that this P450 is the major target responsible for the efficacy of these compounds. While azole antifungal drugs have promise as treatments for TB, the specific compounds that have been shown to have efficacy against <i>Mtb</i> suffer from either low oral bioavailability or cause significant toxic side effects because of their broad spectrum activity against human P450s or steroidogenic inhibition.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> These properties make the azole drugs incompatible with long-term dosing regimens required for TB treatment. Consequently, there is interest in developing more potent and selective inhibitors of <i>Mtb</i> P450s.</div><div class="NLM_p">CYP121 is a soluble, monomeric enzyme, which has limited similarity (≤34% protein sequence identity) to other <i>Mtb</i> P450s.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The enzyme catalyzes the cyclization of the dipeptide cyclo-(<span class="smallcaps smallerCapital">l</span>-Tyr-<span class="smallcaps smallerCapital">l</span>-Tyr) (cYY) to form the diketopiperazine mycocyclosin.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> While the function of mycocyclosin has not been determined, diketopiperazine secondary metabolites often have antimicrobial or cytotoxic activity, properties which might be of importance for <i>Mtb</i> virulence.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This combination of gene essentiality, low sequence similarity to other P450s, and tight azole binding affinity has made the development of CYP121 inhibitors an area of interest in the search for new TB drugs and the focus of the research presented here.</div><div class="NLM_p">Fragment-based drug discovery (FBDD) is an established technique in both academia and industry that has been applied to a wide range of molecular targets.<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24 ref25">(20-25)</a> The availability of high resolution crystal structures of CYP121, in both the substrate-free and substrate/ligand-bound forms, make it amenable to a fragment-based approach.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We have previously reported the development of low micromolar affinity CYP121 inhibitors, developed from fragments that were identified in a biophysical screen of our fragment library.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> Six fragment hits were crystallized with CYP121, one of which was 4-(1<i>H</i>-1,2,4-triazol-1-yl)phenol <b>1</b> (<i>K</i><sub>D</sub> = 1.7 mM) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Fragment <b>1</b> bound in two different overlapping orientations in the CYP121 active site, suggesting an approach based on fragment merging. Synthesis of a range of analogues, guided by in silico screening to optimize the binding interactions of the heterocyclic fragment core, resulted in the identification of the triphenol pyrazole-amine <b>2</b> (<i>K</i><sub>D</sub> = 15 μM). An X-ray crystal structure of <b>2</b> bound to CYP121 revealed that <b>2</b> maintained the binding mode of the original fragment <b>1</b>. The optimized hydrogen bonding interactions between the 5-aminopyrazole core of <b>2</b> and distal pocket residues Gln385, Ala167, and Thr77 represented a privileged binding mode for an azole-containing compound, which generally bind directly to the heme cofactor of P450s using a heterocyclic nitrogen atom. The combination of the low micromolar affinity of <b>2</b> and its unique binding mode, which was conserved with fragment <b>1</b>, led us to pursue further optimization of this compound series.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment merging and in silico directed optimization of fragment <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G47">4G47</a>) led to the discovery of low micromolar CYP121 inhibitor <b>2</b> (salmon, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a>). Fragment <b>1</b> and lead <b>2</b> retained a conserved binding mode distal to the heme cofactor (magenta). Hydrogen bonding interactions between the 5-aminopyrazole ring and CYP121 residues Gln385 and Ala167, and the phenol group with Asn85 and active site water molecules are shown as yellow dashed lines. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite a 100-fold improvement in binding affinity between fragment <b>1</b> and lead <b>2</b>, the low ligand efficiency (LE)<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> of <b>2</b> (LE = 0.23) led us to undertake a deconstruction–reconstruction approach to identify more efficient scaffolds for subsequent optimization. Retrofragmentation of <b>2</b> to its component “fragments” was used to assess the binding contribution, or group efficiency (GE),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> of the three aromatic rings that decorate the amino-pyrazole core, hereafter referred to as Ar1, Ar2, and Ar3, as defined in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>.</div><div class="NLM_p last">GE analysis provided an assessment of the change in binding affinity per heavy atom that could be attributed to each of the Ar1, Ar2, and Ar3 rings of lead <b>2</b>, as the compound was sequentially rebuilt from a series of retrofragments <b>3</b>–<b>6</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> X-ray crystal structures of CYP121 in complex with retrofragments <b>3</b>–<b>6</b> were used to validate our GE analysis, enabled the identification of conserved binding hotspots within the enzyme active site and was used to guide our subsequent optimization of <b>2</b> using fragment growing and merging strategies. The significant improvements in binding affinity and LE that can be achieved by the structure-guided incorporation of a metal binding pharmacophore into the newly identified minimal scaffolds is illustrated. In contrast, fragment growing toward pockets in the distal active site produced only modest improvements in binding affinity and resulted in the development of a second series of compounds with a binding mode that was distinct from the metal-coordinating ligands. Investigation of the SAR governing these interactions and knowledge of binding hot-spots were subsequently exploited to improve the isoform selectivity of inhibitors against human P450 enzymes, and also identified regions of the CYP121 active site that were tolerant to structural modification and which could be exploited during the future optimization of these compounds.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28368" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28368" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Retrofragmentation</h3><div class="NLM_p">Retrosynthetic fragmentation of lead compound <b>2</b> indicated that the relative binding contributions of Ar1, Ar2, and Ar3 could be determined from a combination of monophenol and biphenol fragments <b>3</b>, <b>4</b>, <b>5</b>, and <b>6</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The sequential reconstruction of <b>2</b> from these fragments enabled the GE of each aromatic ring to be calculated from the difference in the free energy of binding per heavy atom added at each stage of reconstruction (GE = −ΔΔ<i>G</i>/Δ<i>HA</i>).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Retrofragments <b>3</b> and <b>5</b> were synthesized according to the previously reported literature procedure,<a onclick="showRef(event, 'ref28 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref33">(28, 33)</a> which was also used to access fragment <b>4</b>. Biphenol fragment <b>6</b> was synthesized according to the general route for accessing 3,5-disubstituted aminopyrazoles described by Johnson et al.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> X-ray crystal structures of fragments <b>3</b>–<b>6</b><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> in complex with CYP121 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a–c) were obtained in order to aid the interpretation of binding affinity and GE trends. Retrofragments <b>3</b>–<b>6</b> each reproduced the privileged non-heme binding mode of lead compound <b>2</b>, which is unusual for P450 ligands containing azole motifs. Analysis of the conserved and variable binding interactions made by the retrofragments highlighted structural features likely to be important for driving the binding affinity of <b>2</b>.<a onclick="showRef(event, 'ref32 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref35">(32, 35)</a> Monophenol <b>4</b> bound in an orientation that overlapped with that of Ar2 of the lead compound <b>2</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a). The water-bridged hydrogen bonding network between Asn85, Thr229, and the heme propionate group with the 4-hydroxy substituent of <b>4</b> was conserved, while the 5-aminopyrazole ring was shifted toward the I-helix (highlighted yellow in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) to enable hydrogen bonding interactions with Val228 and aromatic interactions with Phe168 and Trp182, mimicking the function originally fulfilled by Ar1 of lead compound <b>2</b>. The small shift in position of the aminopyrazole ring of <b>4</b> indicated that interactions made at the Ar1 site might contribute more significantly to the affinity of lead compound <b>2</b> than do the hydrogen-bonding interactions made by <b>2</b> with Gln385.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray crystal structures of lead compound <b>2</b> and component retrofragments <b>4</b>–<b>6</b> in complex with CYP121. The I- (yellow), and F-, and G- (blue) helices are shown in cartoon representation. (a) Overlaid structures of retrofragments <b>4</b> (purple) (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTJ">4KTJ</a>), <b>5</b> (yellow) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF">4KTF</a>), and <b>6</b> (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>) with lead compound <b>2</b> (salmon) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a>), illustrating the conserved binding mode distal to the heme cofactor (magenta).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (b) Structures, binding affinity, and ligand efficiency (LE) of compound <b>2</b> and retrofragments <b>4</b>, <b>5</b>, and <b>6</b>. (c–e) X-ray crystal structures of CYP121 in complex with retrofragments <b>4</b>, <b>5</b>, and <b>6</b>, respectively, indicating polar interactions (yellow dashes) with active site residues (blue) and solvent (red spheres). Co-crystallized sulfate (yellow lines) is also indicated in (d). The associated omit <i>F</i><sub>o</sub><i>F</i><sub>c</sub> electron density map of fragment <b>6</b> contoured to 3σ has been provided in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A 1.65 Å X-ray crystal structure of retrofragment <b>3</b> in complex with CYP121 was obtained, however incomplete ligand occupancy prevented adequate refinement of the structure. The poorly resolved density of <b>3</b> suggested the fragment likely acquired multiple different binding orientations, as had been previously observed for the original hit fragment <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Analysis of the ligand density suggested that <b>3</b> bound CYP121 in an orientation that overlapped with Ar1 of lead <b>2</b>. However, the 5-aminopyrazole ring was rotated 180°, further indicating that hydrogen bonding to Gln385 probably contributed less to binding affinity than other conserved interactions (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S1</a>, Supporting Information). The fused Ar1–Ar2 fragment <b>5</b> accurately recapitulated the binding mode of <b>2</b>, satisfying the offset and edge-face π-stacking interactions with Phe168 and Trp182 that were identified as important in the region occupied by Ar1, as well as the hydrogen bonding networks formed by the Ar2 phenol and between the aminopyrazole ring and Gln385 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). The conserved binding mode of <b>5</b> in the absence of Ar3 suggested a minimal contribution to binding from this motif and was consistent with observations that Ar3 resided in a large, water-filled pocket in the X-ray structure of <b>2</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). This lack of binding contribution was reflected in the low GE calculated for Ar3 on reforming <b>2</b> from <b>5</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).</div><div class="NLM_p">The X-ray crystal structure of the Ar1–Ar3 fused fragment <b>6</b> showed that the fragment bound CYP121 in an orientation that was flipped 180° from that expected in lead <b>2</b>. The 5-amino-phenol ring that distinguished <b>6</b> from retrofragment <b>3</b> was oriented toward the Ar1 site, instead of the Ar3 pocket, and the original phenol of <b>3</b> was shifted into the Ar2 site (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>c). In this orientation, the pyrazole core formed new hydrogen bonding interactions with I-helix residue Thr229, similar to those observed for fragment <b>4</b>. This change in binding mode accounts for the good GE values calculated when elaborating fragment <b>3</b> into <b>6</b> (vide infra). The 100-fold improvement in binding affinity obtained for <b>6</b> was thus attributed to optimizing interactions at the Ar1 and Ar2 sites, demonstrating the importance of obtaining structural data for the accurate interpretation of GE analysis. From these data, it was concluded that Ar1 and Ar2 formed highly conserved interactions that could be exploited in the design of subsequent analogues. In contrast, Ar3 contributed minimally to binding and would likely be tolerant to structural modifications. As such, investigating the SAR of Ar3 was a priority in order to identify more GE replacements for the original phenol ring of lead compound <b>2</b>.</div><div class="NLM_p">The binding affinity and LE of the retrofragments for CYP121 was determined using ITC (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Fragments <b>3</b> (<i>K</i><sub>D</sub> = 1.3 mM, LE = 0.30) and <b>4</b> (<i>K</i><sub>D</sub> = 1.9 mM, LE = 0.29) were found to have comparable binding affinities and LE to that of the original fragment hit 4-(1<i>H</i>-1,2,4-triazol-1-yl)phenol <b>1</b>, indicating that substitution of the original triazole core of <b>1</b> for a 5-aminopyrazole ring resulted in no significant difference in binding affinity. The combination of any two phenol rings to satisfy binding interactions at the Ar1 and Ar2 sites, as exemplified by biphenol retrofragment <b>5</b> (<i>K</i><sub>D</sub> = 40 μM) or <b>6</b> (<i>K</i><sub>D</sub> = 10.6 μM), resulted in an almost additive improvement in binding affinity. The small difference in the binding affinity of <b>5</b> or <b>6</b> (Δ<i>G</i> ≈ −0.9–1.7 kJ mol<sup>–1</sup>) compared to that calculated for the theoretical sum of <b>3</b> and <b>4</b> (Δ<i>G</i><sub><b>3+4</b></sub> (theor) ≈ −7.7 kJ mol<sup>–1</sup>) can be attributed in part to binding contributions from the 5-aminopyrazole motif that is common to the structures of the monophenol fragments. The additive binding affinity was reflected in the good GE values (GE = 0.29–0.41) calculated for the addition of an aromatic ring to a monophenol scaffold <b>3</b> or <b>4</b>. Reduced torsional strain between the phenol rings was suggested to account for the slightly lower <i>K</i><sub>D</sub> value of retrofragment <b>6</b> versus <b>5</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The addition of a third aromatic ring to fragment <b>5</b> or <b>6</b> to give lead <b>2</b> had a negligible effect on binding affinity, as reflected in the small GE values of −0.03–0.08 calculated for occupation of the Ar3 pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Deconstruction of lead compound <b>2</b> into retrosynthetic fragments <b>3</b>–<b>6</b> was used to assess the binding affinity, ligand efficiency (LE), and group efficiency (GE) contribution of the individual components. The three aromatic rings decorating the aminopyrazole core of lead compound <b>2</b> have been labeled Ar1, Ar2, and Ar3 as illustrated, which refers to the orientation the rings occupy in the X-ray crystal structure of <b>2</b> in complex with CYP121 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Retrofragments <b>3</b> and <b>4</b> represent Ar1 and Ar2, respectively. The lower panel tabulates the binding affinity (<i>K</i><sub>D</sub>) and LE and GE properties of compounds from the sequential reconstruction of lead <b>2</b> from component monoaryl (<b>3</b> and <b>4</b>) and biaryl (<b>5</b> and <b>6</b>) retrofragments. <i>K</i><sub>D</sub> values were determined using ITC. LE was calculated from the Gibbs free energy of binding divided by the number of heavy atoms (Δ<i>G</i>/HA).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> GE contributions of the three aromatic rings Ar1, Ar2, and Ar3 were calculated from the change in Gibbs free energy per added heavy atom (−ΔΔ<i>G</i>/Δ<i>H</i>A) as the retrofragments were progressively recombined to form lead compound <b>2</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><i>K</i><sub>D</sub> previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The LE of retrofragments (LE ≈ 0.3) was comparable to that of the original fragment hit <b>1</b> (LE = 0.32), indicating that both mono- and biphenol scaffolds represented better starting points for the optimization of CYP121 inhibitors than did lead compound <b>2</b> (LE = 0.24). Retrofragments <b>4</b> and <b>5</b> were selected as the most favorable scaffolds for further optimization because of the consistency of their binding modes relative to lead compound <b>2</b> and because of the opportunities these scaffolds provided for elaboration toward the heme iron from Ar2 and from the exocyclic pyrazole-5-amino group to explore the Ar3 pocket. The change in the binding mode of retrofragment <b>6</b> relative to lead <b>2</b> led us to deprioritize further optimization based on this scaffold despite its slightly better binding affinity compared to <b>5</b>. This combination of GE analysis, assessment of LE, and interrogation of X-ray crystal structures facilitated a rational strategy for the independent optimization of each aromatic ring Ar1, Ar2, and Ar3 of lead <b>2</b>, guided by insights into the key binding interactions and activity hotspots.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Analogue Design Rationale</h3><div class="NLM_p">Our approach to developing potent CYP121 inhibitors focused on three independent series of analogues, each optimizing the interactions of the Ar1, Ar2, and Ar3 rings of lead compound <b>2</b>, respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a–c). Analogues were designed as derivatives of either the biaryl scaffold <b>5</b>, which had been identified as the most LE retrofragment that also retained the original binding mode of <b>2</b>, or the monoaryl <b>4</b> scaffold, which had been identified to participate in a conserved hydrogen-bonding network and bound to CYP121 in close proximity to the heme cofactor.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Proposed optimization strategies for Ar1, Ar2, and Ar3 of lead CYP121 inhibitor <b>2</b>. (a) Overlaid X-ray crystal structures of biaryl fragment <b>5</b> (yellow) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF">4KTF</a>) and benzamide fragment <b>7</b> (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>) bound to CYP121. The associated omit <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> electron density map of fragment <b>7</b> contoured to 3σ can be found in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. (b) Overlaid X-ray crystal structures of CYP121 in complex with retrofragment <b>5</b> (yellow) and with heme binding fragments <b>8</b> (copper) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G44">4G44</a>) and <b>9</b> (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G45">4G45</a>),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> indicating the proposed <i>meta</i>-attachment point for substituents and distance (6.5 Å) to the heme coordinating position (yellow dashed line). (c) Docking of Ar3 analogues of lead <b>2</b>, elaborating from the 5-aminopyrazole group of retrofragment <b>5</b> (gray) with carbonyl (orange) or methylene (green) linkages to enhance binding interactions with the distal active site pocket (Asn74, Thr65, gray sticks) of CYP121. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Ar1 Analogues</h4><div class="NLM_p last">Analogues in the Ar1 series were designed to probe the effect of electronics and hydrogen bonding on the conserved π-stacking interactions formed between this motif and Phe168 and Trp182 and the potential to form polar interactions with Thr77 and Asp185, residues residing on the substrate selective F/G-helices.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The crystal structure of a substituted benzamide fragment <b>7</b> (<i>K</i><sub>D</sub> = 3.4 mM), which had been identified during our initial fragment screen on CYP121 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> indicated that <i>meta</i>-substitution or elaboration from the <i>para</i>-position would be most favorable (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). Ligand docking simulations<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> were used to support our design and indicated that changing the position of the aromatic substituent was unlikely to cause conformational strain or perturb the global geometry of the ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information).</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Ar2 Analogues</h4><div class="NLM_p">The proximity and orientation of Ar2 in relation to the heme cofactor, and the conserved hydrogen bonding interactions made by the 4-hydroxy substituent with Asn85 and the water-bridged heme propionate group, indicated that Ar2 could provide an ideal anchor for the incorporation of a metal-binding functional group. Compounds which interact with both active site residues and the heme cofactor are generally considered to be more effective than inhibitors that only utilize one form of interaction and it was proposed that exploiting the high enthalpic contribution of metal coordination could significantly improve the potency of CYP121 ligands.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Comparison of the X-ray crystal structures of <b>2</b>, <b>4</b>, and <b>5</b> with those of heme binding fragments <b>8</b> and <b>9</b>, which had been identified during our original fragment screen on CYP121,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> indicated that extending from the <i>meta</i>-position of Ar2 with substituents 5.5–6.5 Å in length would provide the appropriate proximity for metal binding (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). A range of substituents containing aromatic or basic nitrogen atoms were selected as functional groups commonly occurring in heme binding P450 inhibitors. Primary anilines were prioritized because of the prevalence of this chemotype in fragment hits identified during our original fragment screen on CYP121 and also from our subsequent optimization attempts which had yielded several co-crystal structures of CYP121 with aniline ligands.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p last">Analogues derived from both monoaryl <b>4</b> and biaryl <b>5</b> fragment scaffolds were proposed to maximize LE and to allow the importance of Ar1 on the binding mode, affinity, and selectivity of the compounds to be investigated (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). A range of analogues in which the 4-hydroxy group of Ar2 was removed were also designed in order to determine whether torsional strain resulting from interactions between the 4-hydroxy group and metal-binding substituents introduced <i>ortho</i> to this might perturb the geometry required for metal coordination. It was hypothesized that the loss of enthalpic contributions from hydrogen bonds made by the 4-hydroxy group of Ar2 was likely to be small in comparison to that gained by metal-coordination.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Ar3 Analogues</h4><div class="NLM_p last">A fragment growing approach was devised to explore the SAR of Ar3 and to improve the GE of this motif by potentially introducing binding interactions with residues in the back of the active site. Interrogation of the X-ray crystal structure of lead <b>2</b> and ligand docking studies<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> were used to select an appropriate linker-length for growing, with methylene or carbonyl linkers being prioritized. A range of five- and six-membered (hetero)aromatic groups were selected to incorporate onto these linkers, including phenols as direct mimics of lead <b>2</b> and the natural cYY substrate, phenol bioisosteres, and substituents that were predicted by docking simulations to form favorable cation−π interactions with a nearby Arg386 residue, or polar interactions with amide groups of the protein backbone (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Synthetic Chemistry</h3><div class="NLM_p">Analogues varying at Ar1 were synthesized according the published procedure for biaryl retrofragment <b>5</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref28 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref33">(28, 33)</a> In brief, the Claisen condensation of 4-methoxyphenylacetonitrile <b>11</b> with the appropriately substituted benzoate ester <b>10a</b>–<b>d</b> afforded the β-ketonitrile intermediates <b>12a</b>–<b>d</b>. Reaction of <b>12a</b>–<b>d</b> with hydrazine hydrate and concentrated hydrochloric acid gave the 5-aminopyrazoles <b>13a</b>–<b>d</b> in excellent yields (91–97%). Subsequent deprotection of the aryl-methyl ether(s) with BBr<sub>3</sub> (1.0 M in DCM) gave the desired phenol analogues <b>5</b> and <b>14b</b>–<b>d</b> in near quantitative yields.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Biaryl-aminopyrazole Analogues (Ar1)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaH, THF, 60 °C, 20–48 h; (b) HCl (37%), NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 90 °C, 14 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 6 h. Synthesis and characterization of <b>14d</b> previously described.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></p></figure><div class="NLM_p">The syntheses of analogues incorporating a metal binding functional group at the <i>meta</i> position of Ar2 are summarized in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> (i) and (ii) for analogues based on the monoaryl <b>4</b> or biaryl <b>5</b> retrofragment scaffolds, respectively. Analogues retaining the 4-hydroxy substituent of Ar2 were synthesized from the appropriately protected 4-benzyloxy <b>15b</b> or 4-methoxy <b>15c</b> substituted 3-bromophenylacetonitrile starting materials, which were either obtained directly from commercial sources or synthesized in two steps by deprotection of the aryl methyl ether using BBr<sub>3</sub> followed by benzyl protection under Finkelstein conditions. Synthesis of the common 5-aminopyrazole ring followed a similar route to that employed to access Ar1 analogues (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Claisen condensation of phenylacetonitriles <b>15a</b>–<b>c</b> with either ethyl formate or methyl 4-methoxybenzoate afforded the 3-oxopropanenitrile <b>16a</b>–<b>c</b> or β-ketonitrile <b>21a</b>–<b>b</b> intermediates, respectively. The crude products were treated with hydrazine hydrate at acidic pH and heated under reflux to give the 5-aminopyrazole compounds <b>17a</b>–<b>c</b> and <b>22a</b>–<b>b</b> in yields of 48–67% and 77–95%, respectively, over two reaction steps. A microwave-assisted Suzuki–Miyaura cross-coupling reaction was used to introduce a range of nitrogen-containing heterocycles <b>18e</b>–<b>i</b> or nitro-substituted aromatics <b>18a</b>–<b>d</b>, <b>23a</b>–<b>b</b> from their boronic acid precursors. Nitroarenes were subsequently reduced using tin(II) chloride to reveal the heme binding anilines <b>19a</b>–<b>d</b>, <b>24a</b>–<b>b</b> in yields up to 77%. Removal of the methyl or benzyl protecting groups using BBr<sub>3</sub> afforded the 4-hydroxyphenyl Ar2 analogues <b>20a</b> and <b>25a</b>–<b>b</b>, which were purified by flash chromatography to give 46–50% yields. However, the presence of inorganic contaminants in <b>20a</b> prevented the testing of this compound.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Ar2 Analogues Based on the Core Scaffold of (i) Monoaryl Fragment <b>4</b> or (ii) Biaryl Fragment <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) ethyl formate, NaOEt, EtOH, 0–90 °C, 22 h; (b) AcOH, NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 75 °C, 3 h; (c) aryl-boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane:H<sub>2</sub>O (1:1 v/v), μw, 180 min; (d) SnCl<sub>2</sub>·2H<sub>2</sub>O, EtOH, 75 °C, 15 h; (e) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 4 h; (g) methyl 4-methoxybenzoate, NaH, THF, 60 °C, 48 h; (h) HCl (37%), NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 90 °C, 16 h.</p></p></figure><div class="NLM_p">Ar3 analogues containing a methylamine-linked substituent <b>26a</b>–<b>f</b> were synthesized from the reductive alkylation of biaryl retrofragment <b>5</b> with the respective aldehyde (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Reaction conditions, including solvent polarity, acid catalyst, and use of either sodium triacetoxyborohydride or sodium cyanoborohydride reducing reagents were optimized for individual substrates.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Analogues <b>26g</b> and <b>26h</b> were synthesized in good yields (78–95%) under similar conditions by reductive alkylation of the dimethoxy protected precursor <b>13a</b>. Subsequent deprotection of intermediates <b>27a</b>–<b>b</b> with BBr<sub>3</sub> revealed the diphenols <b>26g</b>–<b>h</b> in 61–68% yields.</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Benzyl-amine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aldehyde (0.9–1.1 equiv), AcOH*, Na(OAc)<sub>3</sub>BH, DMF, DCE, rt, 24–48 h (b) aldehyde (1 equiv), AcOH, NaCNBH<sub>3</sub>, MeOH, rt, 15–24 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 2–24 h. *No AcOH with <b>26d</b>.</p></p></figure><div class="NLM_p">A series of 5-amidopyrazole analogues <b>29a</b>–<b>c</b> containing a carbonyl linker were synthesized from 5-aminopyrazole <b>13a</b> and the respective benzoyl chloride to give <b>28a</b>–<b>c</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Benzoyl chloride electrophiles were either purchased or synthesized in situ from the corresponding benzoic acid precursor using oxalyl chloride and catalytic DMF. Yields of the desired 5-amido regioisomers were low (27–34%), but significant polarity differences enabled easy separation from endocyclic 1N-substituted byproducts using column chromatography. Subsequent deprotection of the aryl-ethers with BBr<sub>3</sub> afforded the analogues <b>29a</b>–<b>c</b>.</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Benzamide Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (4-OMe)-benzoyl chloride, Et<sub>3</sub>N, DCM, 0 °C–rt, 3.5 h; (b) nicotinic acid, oxalyl chloride, DMF, DCM, 0 °C–rt, 2 h, then Et<sub>3</sub>N, DCM, 0 °C–rt, 14 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, −78 °C–rt, 14 h.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Inhibitor Binding Mode and Selection of Heme Binding Functional Groups for Ar2 Analogues</h3><div class="NLM_p">The binding mode of analogues was initially characterized using UV–vis spectroscopy to allow the identification of ligands that interacted with the ferric iron, or with the axial water ligand of the heme cofactor, from changes in the optical spectrum of CYP121. P450 inhibitors which displace the axial water ligand and directly coordinate to the ferric iron of the heme cofactor, typically by the lone pair electrons of a nitrogen atom, stabilize an inactive state of the enzyme by sterically blocking the active site and usually lowering the reduction potential of the enzyme.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> This binding mode (type II) produces a red-shift in the Soret band of the P450 optical spectrum, the magnitude (Δλ<sub>max</sub>, nm) of which is dependent on both the affinity (<i>K</i><sub>D</sub>) and concentrations of the ligand and P450 under specific assay conditions.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> In contrast, ligands which interact with the heme cofactor via water-bridged hydrogen bonding to the axial water ligand, or which displace the axial water but do not directly coordinate to the ferric iron, can produce reverse type I and type I spectra, respectively.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p">A single shot UV–vis absorbance assay was used to rapidly identify Ar2 substituents with the appropriate properties and coordination geometry for heme binding interactions (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). A preliminary series of Ar2 analogues, substituted at the <i>meta</i>-position with aniline or heteroaromatic groups, were synthesized based on the monoaryl scaffold <b>4</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> (i)). Only primary 3- or 4-aminophenyl analogues <b>19a</b> and <b>19d</b> were found to cause a significant red-shift (Δλ<sub>max</sub> > 1 nm) in the Soret band of CYP121, which occurs at 416.5 nm in the water-ligated resting state.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Indole <b>18g</b>,<b>h</b>, pyrazole <b>18i</b>, and 3-pyridyl <b>18e</b> substituents showed no heme binding interactions (Δλ<sub>max</sub> = 0 nm), and only minor perturbation of the Soret band was observed for the 4-pyridyl isomer <b>18f</b> (Δλ<sub>max</sub> = 1 nm).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Identification of Heme Binding Functional Groups Using UV–Vis Spectroscopy<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0014.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0015.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">Analogues were prepared as 0.1–1 mM stocks in DMSO-<i>d</i><sub>6</sub>. The change in the wavelength of the Soret band (Δλ<sub>max</sub>, nm) of CYP121 (5 μM) was calculated as the difference of that found in the presence of analogues compared to 1% v/v DMSO-<i>d</i><sub>6</sub> alone (λ<sub>max</sub> = 416.5 nm). Binding affinities (<i>K</i><sub>D</sub>) for selected compounds were calculated from the titration of compounds (0.2 μL aliquots of 10–100 mM stocks) into CYP121 (5 μM). Full details are provided in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</p></div></div><div></div></div><div class="NLM_p">The binding affinities of the 3-aminophenyl <b>19a</b> (<i>K</i><sub>D</sub> = 14 μM) and 4-aminophenyl <b>19d</b> (<i>K</i><sub>D</sub> = 230 μM) substituted analogues were determined by optical titration according to the previously described procedure.<a onclick="showRef(event, 'ref17 ref41'); return false;" href="javascript:void(0);" class="ref ref17 ref41">(17, 41)</a> The 16-fold difference in affinity between the two regioisomers clearly indicated that the 3-aminophenyl substituent provided a better geometry for heme ligation, and consequently this functional group was incorporated into all subsequent analogues in the Ar2 series.</div><div class="NLM_p">All subsequent Ar2 analogues containing a 3-aminophenyl substituent (<b>19b</b>,<b>c</b>, <b>24a</b>, <b>25a</b>,<b>b</b>) caused type II shifts in the CYP121 optical spectrum, allowing their binding affinity to be determined by optical titration in addition to ITC (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> and <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). The magnitude of the type II shift decreased with increasing steric bulk <i>ortho</i> to the 3-aminophenyl substituent. These results supported the hypothesis that torsional strain in the biphenyl system, or the steric bulk of the benzyl or methyl hydroxy protecting groups, in analogues <b>19b</b> and <b>19c</b>, respectively, prevented the ligands approaching the cofactor for heme coordination.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinity and Ligand Efficiency Values of CYP121 Inhibitors<a class="ref internalNav" href="#tbl2-fn1" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0016.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Binding affinity of <b>14d</b> previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Poor solubility prevented <i>K</i><sub>D</sub> being determined for <b>24a</b> using ITC.</p></div><div class="footnote" id="tbl2-fn1"><sup>Table c</sup><p class="last">Binding affinity (<i>K</i><sub>D</sub>, μM) values for all analogues were measured by ITC. The <i>K</i><sub>D</sub> values for the Ar2 series of heme binding compounds were also determined by UV–vis optical titration as described previously.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Ligand efficiency (LE) was calculated from the Gibbs free energy of binding divided by number of heavy atoms (Δ<i>G</i>/HA).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> Full details are provided in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</p></div></div><div></div></div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optical titration difference spectra (top panel) and concentration-dependent changes in CYP121 absorbance (bottom panel) for representative heme binding ligands (a) <b>24a</b>, (b) <b>25b</b>, and (c) <b>19a</b>. Absorption spectra of CYP121 (5 μM) were collected between 250 and 800 nm after sequential additions of 0.2 μL aliquots of DMSO-<i>d</i><sub>6</sub> stock solutions (0.15–83 mM) of ligands. Difference spectra were obtained from the subtraction of the ligand-free CYP121 spectra from those obtained over the course of the titration. The maximum change in absorbance from each difference spectrum was plotted against ligand concentration and data were fitted using a modified version of the Morrison equation for tight binding ligands <b>24a</b> and <b>25b</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> or a hyperbolic function for <b>19a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The only analogue in the Ar1 or Ar3 series of compounds to perturb the Soret band was the 3(4)-imidazole-methylamine substituted compound <b>26h</b>. The type II shift observed was small (Δλ<sub>max</sub><b>26h</b> (100 μM) = +1.5 nm) compared to that observed for the 3-aminophenyl substituted Ar2 analogues (Δλ<sub>max</sub><b>25a</b> [100 μM] = +5.0 nm), which was consistent with results observed for heteroaromatic nitrogen substituents during our initial selection of a heme binding functional group for Ar2 analogues (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). A 2.0 Å X-ray crystal structure of <b>26h</b> in complex with CYP121 indicated that the compound must adopt multiple different binding modes, as the 3(4)-imidazole substituent of the crystallized compound resided in the Ar3 pocket, distal to the heme cofactor (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b). These competing binding modes might account for the weak affinity of <b>26h</b> (<i>K</i><sub>D</sub> = 274 μM) compared to the structurally similar 3-thiophenyl <b>26c</b> or 2-furanyl <b>26d</b> methylamine analogues, which did not perturb the optical spectrum of CYP121 (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Owing to a lack of spectral perturbation, the binding affinity of all other Ar1 and Ar3 analogues was assumed to result from binding interactions made with residues in the distal active site pocket and provided support that these analogues retained a similar binding mode to lead <b>2</b>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Binding Affinity and X-ray Crystallography</h3><div class="NLM_p">The binding affinity (<i>K</i><sub>D</sub>) of analogues for CYP121 was determined by ITC (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a–c). The <i>K</i><sub>D</sub>s of heme binding Ar2 analogues were also determined by UV–vis optical titration (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43-45)</a> Comparison of the <i>K</i><sub>D</sub>s obtained for the Ar2 series by ITC and optical titration enabled robust comparisons to be made with the affinity of the non-heme binding Ar1 and Ar3 series of analogues, as well as with previously reported type II inhibitors in the literature.<a onclick="showRef(event, 'ref12 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref12 ref27 ref28">(12, 27, 28)</a> In general, <i>K</i><sub>D</sub> values obtained by optical titration were 10–100-fold lower than those measured in ITC, consistent with previous reports in the literature.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Variations in affinity data generated using different biophysical techniques have been previously attributed to the kinetically significant but spectrally silent, multistep binding interactions of P450s with ligands, the potential presence of heterogeneous enzyme populations in solution, requirement for conformational change, and/or enzyme cooperativity.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45, 46)</a> High resolution (1.6–2.2 Å) X-ray crystal structures of CYP121 in complex with a number of analogues were obtained, allowing inhibitor binding modes to be confirmed and enabling robust comparison of the SAR contributing to binding affinity (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>,<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>).</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Representative ITC binding isotherms and integrated enthalpy plots for (a) fused Ar1–Ar3 fragment <b>6</b> (500 μM); (b) non-heme binding Ar3 analogue <b>26a</b> (1 mM); (c) heme binding analogue <b>25a</b> (500 μM). The isotherm of each compound titrated into buffer alone was subtracted from that containing CYP121 (50 μM). The integrated enthalpy change for each injection plotted as a function of molar ratio of ligand-to-CYP121 was fitted using a one-site equilibrium binding model to calculate binding affinity (<i>K</i><sub>D</sub>). The binding stoichiometry (N) was allowed to vary for analogues <b>6</b>, <b>26a</b>, and <b>25a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Ar1 Analogues</h4><div class="NLM_p last">Changing the position of the original 4-hydroxy group to the 3-position <b>14b</b> resulted in a moderate reduction in binding affinity, while substitution with a 3-fluoro substituent increased the <i>K</i><sub>D</sub> of <b>14c</b> 5-fold compared to <b>5</b>. These results suggest that aromatic stacking interactions made by Ar1 contribute to binding affinity in this region. The change in π-electron density from the introduction of <i>m</i>-hydroxy or <i>m</i>-fluoro substituents would likely weaken both offset face-to-face aromatic interactions with Phe168 and edge-to-face interactions with Trp182, in addition to disrupting hydrogen bonding interactions with two conserved water molecules shown to solvate the <i>p</i>-hydroxy substituent of fragments <b>1</b>, <b>5</b>, and lead compound <b>2</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>d).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Disubstitution of Ar1 with hydroxy groups in the 2- and 4-positions, <b>14d</b>, also lowered the binding affinity (<i>K</i><sub>D</sub> = 180 μM) despite introducing polar interactions with residues Thr77 and Ala167 as shown in the X-ray crystal structure of <b>14d</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Intramolecular hydrogen bonding between the 2-hydroxy group and the 1-N<i>H</i>-pyrazole heterocycle might account for this result by favoring the 3-aminopyrazole tautomer and thus weaken bonding interactions to Gln385. No overall change in the conformation of the ligands was observed due to changes made to Ar1 substituents, consistent with our initial docking results.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The sensitivity of Ar1 to even minor structural changes confirmed our previous predictions from retrofragmentation and GE analysis that binding interactions in this region represented a key affinity hotspot for CYP121 that might be of use during later stages of compound development. We thus directed our attention toward investigating the Ar2 and Ar3 aromatic rings.</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Ar2 Analogues</h4><div class="NLM_p">Incorporation of a 3-aminophenyl substituent onto the Ar2 ring of the monoaryl scaffold <b>4</b> or the biaryl scaffold <b>5</b> resulted in a more than 100-fold improvement in binding affinity. Analogue <b>19a</b> had an equivalent <i>K</i><sub>D</sub> of 14 μM (LE <b> = </b> 0.35) by optical titration to that of lead compound <b>2</b> and was made significantly more LE by removal of the Ar3 motif. The reincorporation of the Ar1 phenol ring to give compound <b>25b</b> (<i>K</i><sub>D</sub> = 0.29 μM) resulted in a further 50-fold improvement in binding affinity, as had been previously observed during our lead-deconstruction analysis when the Ar1 and Ar2 fragments <b>3</b> and <b>4</b> were fused to give biaryl retrofragment <b>5</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Removal of the Ar2 4-hydroxy group of <b>25b</b> to relieve torsional strain in the heme binding biphenyl-amine system improved CYP121 affinity a further 20-fold to produce the low nanomolar, high LE compounds <b>24a</b> and <b>25a</b>. While the 4-hydroxy analogue of <b>19a</b> was not analyzed, the effect of <i>ortho</i>-substitution on weakening heme binding interactions was made apparent through assessment of the 4-benzyloxy and 4-methoxy substituted analogues <b>19b</b> and <b>19c</b>. While both <b>19b</b> and <b>19c</b> caused type II shifts in the CYP121 Soret band at high concentrations (100–1000 μM), the bulky protecting groups clearly prevented ligands obtaining the optimal geometry for heme ligation and a <i>K</i><sub>D</sub> could not be obtained for these compounds by optical titration within solubility limits. The relatively good binding affinity of <b>19b</b> (<i>K</i><sub>D</sub> = 28 μM) determined by ITC suggested that this compound might instead adopt an alternative “heme-silent” binding mode in the distal active site.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_p">The improved LE of <b>19a</b>, <b>24a</b>, and <b>25a</b>–<b>b</b> compared to lead <b>2</b> (LE = 0.24) was reflected in the good GE value of 0.42 calculated for the 3-aminophenyl substituent based on the introduction of seven heavy atoms to retrofragment <b>5</b>.<a onclick="showRef(event, 'ref31 ref48'); return false;" href="javascript:void(0);" class="ref ref31 ref48">(31, 48)</a> Comparison with the low GE of 0.08 calculated previously for the Ar3 phenol (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), which had the same number of heavy atoms, demonstrated the utility of our lead deconstruction-elaboration approach in allowing the identification of maximally efficient scaffolds and in targeting binding hotspots.</div><div class="NLM_p">The 2.1 Å crystal structure of <b>25b</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a), the structurally most similar compound to retrofragment <b>5</b>, in complex with CYP121 revealed that the analogue accurately replicated the conserved binding mode of the lead <b>2</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). All intermolecular interactions that had been identified as binding hotspots, such as aromatic stacking of Ar1 with Phe168 and Trp182, interaction of the Ar2 4-hydroxy group with a water-bridged heme propionate network, and hydrogen bonding between the aminopyrazole ring and Gln385 were satisfied. The 3-aminophenyl substituent was located 2.5 Å from the heme-iron and at an angle of 36°, consistent with that required for tetrahedral NH<sub>2</sub>–Fe(III) coordination.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Hydrogen bonding interactions were also identified between the 3-amino group of <b>25b</b> and Ser237, which is thought to be the conserved alcohol residue of CYP121 required for the catalysis of proton transfer to the heme iron and for organization of the axial water ligand.<a onclick="showRef(event, 'ref12 ref36'); return false;" href="javascript:void(0);" class="ref ref12 ref36">(12, 36)</a></div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray crystal structures of heme binding Ar2 analogues <b>25b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>), <b>25a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE">5IBE</a>), and <b>19a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF">5IBF</a>) in complex with CYP121. (a) <b>25b</b> (cyan) closely recapitulated the binding of lead <b>2</b>, forming polar contacts (yellow dashes) with amino acids (blue sticks) in binding hotspots as well as additional binding contacts to Ser237, Val228, and the heme iron. (b) <b>25a</b> (orange) was rotated away from the I-helix (yellow cartoon), placing the 5-aminopyrazole ring in the Ar3 pocket. Polar contacts to the heme iron and Ser237 are indicated (yellow dashes). (c) Two molecules of <b>19a</b> bound per active site, each ligand fulfilling a subset of the interactions identified as binding hotspots. Inset: Heme binding <b>19a</b> recapitulated the binding mode of <b>25a</b> while the non-heme binding <b>19a</b> ligand satisfied aromatic interactions in the Ar1 pocket, as well as the hydrogen bonding interactions of fragment <b>7</b>. (d) Extracted structures of <b>25b</b>, <b>25a</b>, and <b>19a</b> annotated to show metal coordination (yellow dashes) distance, angle of approach to the porphyrin ring (yellow angle), and dihedral torsion in the biphenyl-aniline motif. Removal of the Ar2 4-hydroxy group reduced the dihedral angle of the biphenyl aniline system in <b>19a</b> and <b>25a</b> by 30° compared to <b>25b</b> and allowed the analogues to approach the plane of the porphyrin at a sharper angle (31–33°). The omit <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps of ligands <b>19a</b>, <b>25a</b>, and <b>25b</b> contoured to 3σ have been provided in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 1.6 Å crystal structure of <b>25a</b> confirmed that removal of the Ar2 4-hydroxy group resulted in a 30° reduction in the dihedral angle of the biphenylamine motif (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b,d). As for analogue <b>25b</b>, the 3-aminophenyl substituent of <b>25a</b> bound in close proximity (2.3 Å) and at an appropriate orientation (34°) for heme coordination. However, reduced rotation in the biphenyl system allowed a slightly sharper angle of approach to the heme ring. Close overlap between the aniline ring of <b>25a</b> and the sterically unhindered aniline-containing fragments <b>8</b> and <b>9</b> suggested that aromatic stacking interactions were optimized by the removal of the 4-hydroxy substituent from <b>25b</b>. These improved interactions might account for the 9-fold improvement in potency between the two analogues and illustrate the utility of fragments as tools for elucidating preferred binding motifs (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b, and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). The combination of improved heme interactions and removal of functional groups identified to interact with binding hotspots, such as the Ar2 4-hydroxy group, resulted in a change in orientation of <b>25a</b> compared to the conserved position of <b>25b</b> and lead <b>2</b>. Analogue <b>25a</b> was tilted sideways to place the exocyclic 5-amino group of the pyrazole ring into the pocket normally occupied by Ar3, resulting in a loss of interactions with Gln385. The fact that <b>24a</b> retained good affinity despite this loss of interactions with Gln385 was consistent with observations made about the binding of retrofragments during lead deconstruction (vide supra). The equipotent (<i>K</i><sub>D</sub> = 33 nM) 4-methoxy protected precursor <b>24a</b> was found to bind in an identical orientation to <b>25a</b>, demonstrating some tolerance of the Ar1 group to substituent variation (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S4</a>, Supporting Information).</div><div class="NLM_p last">Two molecules of analogue <b>19a</b>, which can be considered a fragment of <b>25a</b> (lacking only the Ar1 ring), were found to bind within each active site of the CYP121 crystallographic dimer (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>c). The orientation of the two overlapping molecules of <b>19a</b> illustrated both the importance of the Ar1 region as a binding hotspot and also demonstrated optimization strategies which might further improve selectivity. One molecule of <b>19a</b> ligated the heme iron as observed for <b>25a</b>, with a coordination distance of 2.4 Å and a 32° angle of approach to the heme ring. The 5-aminopyrazole group similarly occupied the water-ligated Ar3 site. The second molecule of <b>19a</b> satisfied the requirements of the remaining binding hotspots, including hydrogen-bonding interactions with Gln385 and aromatic stacking interactions in the Ar1 site. Extension of the 3-aminobiphenyl system into the Ar1 pocket demonstrated a route to achieving the additional hydrogen bonding interactions with the F/G-helix residues Thr77, Ala178, and Asp185 that had earlier been suggested by the identification of hit fragment <b>7</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>c, inset and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a).</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Ar3 Analogues</h4><div class="NLM_p">The binding affinity of Ar3 analogues was largely unaffected by the structural changes made to the 5-aminopyrazole substituent. Analogues substituted with a range of aromatic or heteroaromatic groups maintained low micromolar binding affinities (<i>K</i><sub>D</sub> < 100 μM), and a number of compounds (<b>26b</b>–<b>d</b>) had comparable potency to lead <b>2</b> (<i>K</i><sub>D</sub> = 15 μM) (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Efforts to grow further out into the active site pocket occupied by Ar3 by the addition of a methylene linker resulted in a small improvement in binding affinity with a <i>K</i><sub>D</sub> of 6.3 μM determined for the 4-hydroxybenzyl analogue <b>26a</b>, making it the most potent non-heme binding CYP121 ligand described to date. The poorly defined electron density of the 4-hydroxybenzyl ring in the X-ray crystal structure of <b>26a</b> suggested that it was conformationally flexible and only interactions with active site solvent molecules were observed (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a). All Ar3 analogues containing methylamine-linked aromatic substituents were more potent than their carbonyl-linked amido-pyrazole counterparts <b>29a</b>–<b>c</b>. For example, comparison of 4-hydroxy-benzyl <b>26a</b> and benzamide <b>29b</b> analogues or 4-pyridinyl-methylamine <b>26g</b> and isonicotinamide <b>29c</b> analogues revealed a 2.5–16-fold loss in affinity, respectively, for the carbonyl-linked analogues. Increased conformational rigidity enforced by the 5-amido group or changes to the hydrogen bond donor–acceptor interactions of the 5-amino-NH with Gln385 was used to rationalize these SAR. A preference for 4-hydroxy substituted aromatics was observed in both the methylamine (<b>26</b>) and amide-linked (<b>29</b>) analogues, which was unsurprising considering the structure of the natural cYY CYP121 substrate.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> The addition of bromine <i>ortho</i> to the 4-hydroxy group in analogue <b>26b</b> had little effect on binding affinity and indicated that larger substituents could be tolerated in this region of the active site. Indole <b>26e</b> and indazole <b>26f</b> bioisosteres of the 4-hydroxyphenyl group resulted in a 3–4-fold increase in <i>K</i><sub>D</sub>, possibly indicating that the greater acidity or hydrogen bonding ability of the hydroxyl group was favorable for binding. Further supporting this assumption, analogues containing basic aromatic nitrogens, such as the pyridinyl <b>26g</b> or 3(4)-imidazole <b>26h</b> methylamine-linked analogues, significantly weakened CYP121 binding affinity.</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structures of Ar3 analogues (a) <b>26a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI">5IBI</a>) and (b) <b>26h</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH">5IBH</a>) in complex with CYP121. Both compounds closely recapitulated the binding mode of lead <b>2</b>. Ar3 substituents remained flexible, interacting only with active site water molecules for <b>26a</b> or forming novel interactions with active site residues Ala75 and Asn74 for <b>26h</b>. The omit <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps of ligands <b>26a</b> and <b>26h</b> contoured to 3σ have been provided in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As previously highlighted, a 2.0 Å X-ray crystal structure of <b>26h</b> in complex with CYP121 was obtained that showed the imidazole ring binding distal to the heme cofactor in the Ar3 pocket (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b). Hydrogen bonding interactions with Asn74 and Ala75 had not been previously observed in other CYP121–ligand structures and could provide a new route for compound optimization. Despite these novel interactions the binding affinity of <b>26h</b> was weak (<i>K</i><sub>D</sub> = 274 μM), which could have been the result of the mixed binding modes suggested by UV–vis spectral analysis (vide supra). The consistency of the binding mode of <b>26a</b> and <b>26h</b> with lead <b>2</b> demonstrated in X-ray crystallography provided confidence when interpreting the SAR of the remainder of the Ar3 analogues. The binding affinity of thiophenyl–<b>26c</b> or furanyl–<b>26d</b> methylamine-linked analogues was comparable to that of lead <b>2</b> and tolerance to these smaller, hydrophobic heterocycles might enable the pharmacokinetic properties of subsequent analogues to be modulated during later phases of drug development. The insensitivity of the Ar3 pocket to structural modifications could indicate a functional adaption of the enzyme to enable the accommodation of different substrates. Recent evidence that CYP121 binds a number of other cyclic dipeptides, including cYF and cYW, supports this proposal.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Inhibitor Affinity and Stoichiometry</h3><div class="NLM_p">The relative affinity of lead compounds, their binding stoichiometry to CYP121 and their effect on the oligomerization state of the enzyme were investigated using native mass spectrometry. Most bacterial P450s are monomeric and soluble cytosolic proteins. However, some form higher homomeric oligomerization states, heterodimers with their cognate reductase proteins, or are expressed as fusion products covalently linked to a reductase domain.<a onclick="showRef(event, 'ref36 ref51'); return false;" href="javascript:void(0);" class="ref ref36 ref51">(36, 51)</a> We have previously reported that CYP121 exists predominantly as a dimer in solution, with a small percentage of monomeric protein present.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Dimeric CYP121 forms complexes with two ligand molecules simultaneously and the dimer interface does not appear to occlude the substrate entry channel. The stability of the dimer is reduced by the binding of azole antifungal compounds. In contrast, other small molecule CYP121 inhibitors and the CYP121 substrate (cYY) form stable complexes with the protein and do not affect the oligomerization state.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p">Native mass spectra were collected for CYP121 with the most potent Ar3 analogue <b>26a</b>, Ar2 analogues <b>19a</b>, <b>24a</b>, <b>25a</b>, and <b>25b</b> and the biaryl retrofragment <b>6</b> (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a–c). All analogues formed stable complexes with the CYP121 monomer and dimer. The proportion of ligand-bound CYP121 increased in a concentration-dependent manner as the ratio of ligand-to-protein was increased from sub- (0.5:1) to superstoichiometric (25:1) values. The relative occupancy of monomeric and dimeric CYP121 and binding stoichiometry observed at three different ligand concentrations allowed analogues to be ranked in terms of their relative binding affinities (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S1</a>, Supporting Information). The rank order of affinities correlated well with <i>K</i><sub>D</sub> values calculated previously by ITC and UV–vis optical titration (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Both the CYP121 monomer and dimer were entirely in the bound state at stoichiometric ligand-to-protein concentrations of Ar2 analogues <b>25a</b> and <b>25b</b>, consistent with the low nanomolar affinities of these compounds. In contrast, complete occupancy of the CYP121 dimer by the more weakly binding analogues <b>19a</b> and <b>26a</b> was not achieved until ligands were present in 25-fold excess ligand-to-protein concentrations. Despite the comparable <i>K</i><sub>D</sub> of retrofragment <b>6</b> to Ar2 analogue <b>19a</b>, the CYP121 dimer remained singly bound even at 25-fold excess concentration of <b>6</b>. This apparent difference in binding strength between <b>6</b> and <b>19a</b> observed by native mass spectrometry might reflect the different binding mode of the two analogues. Heme binding interactions made by <b>19a</b> would likely be strengthened in the gas phase of the mass spectrometer, while the closer proximity of <b>6</b> to the entrance of the active site could allow the ligand to be displaced more easily during ionization. The double-occupancy of the CYP121 monomer observed for analogue <b>19a</b> in X-ray crystallography was not detected within the concentration ranges tested in native mass spectrometry experiments (2.5–218 μM ligand).</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Representative native mass spectra of CYP121 and selected compounds <b>26a</b>, <b>25a</b>, and <b>19a</b> demonstrated that compounds formed stable complexes with both monomeric (<i>m</i>/<i>z</i> 5250–6750) and dimeric (<i>m</i>/<i>z</i> 5250–6750) CYP121. The number of bound ligands (orange spheres) increased in a concentration-dependent manner and was proportionate to the binding affinity (<i>K</i><sub>D</sub>) of the compounds. Spectra were collected for CYP121 (8.7 or 5 μM) at three different ligand-to-protein ratios (0.5:1, 1:1, and 25:1) and in the presence of DMSO-<i>d</i><sub>6</sub> (2.2% or 5% v/v) alone. The concentration of protein or DMSO-<i>d</i><sub>6</sub> did not affect the quality of the spectra or ligand binding interactions, but higher concentrations of DMSO-<i>d</i><sub>6</sub> reduced the charge state of the protein. Ligand binding stoichiometry was calculated from the difference in mass of ligand-bound and unbound (orange dashed lines) protein peaks divided by the molecular weight of ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S1</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">No significant change in the oligomerization state of CYP121 was observed for any of the analogues compared to that previously noted with the azole antifungals (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S2</a>, Supporting Information).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The different effects of Ar2 analogues and azole antifungal compounds on the oligomerization state might reflect the nature of the metal binding functional group in each series. Greater conformational reorganization of the CYP121 active site might be expected in order to accommodate axial coordination of an sp<sup>2</sup>-hybridized imidazole ring (∼75°),<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> while the primary aniline present in the current analogues approaches the porphyrin at a sharper angle (∼31–36°, <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>d).<a onclick="showRef(event, 'ref46 ref49'); return false;" href="javascript:void(0);" class="ref ref46 ref49">(46, 49)</a></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Selectivity against <i>Mtb</i> P450s</h3><div class="NLM_p">Selectivity screening against off-target microbial P450s is far less common during pharmaceutical development than screening against mammalian P450s and consequently the functions and promiscuity of these enzymes remain largely uncharacterized. All of the <i>Mtb</i> P450s that have been studied to date bind with varying affinity to compounds from the azole series of antifungal drugs, but there have been few reports detailing their interactions with non-imidazole/triazole ligands.<a onclick="showRef(event, 'ref4 ref7 ref9 ref12 ref49 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref4 ref7 ref9 ref12 ref49 ref53 ref54 ref55">(4, 7, 9, 12, 49, 53-55)</a> Compounds from our development of CYP121 inhibitors were screened against a panel of <i>Mtb</i> P450s using UV–vis spectroscopy (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) to both assess their selectivity and also to aid the development of chemical tools which could be used to help characterize these P450 systems in <i>Mtb.</i></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding of Select Retrofragments and CYP121 Ligands to <i>Mtb</i> P450 Isoforms<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0017.gif" alt="" id="GRAPHIC-d7e2930-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>Table a</sup><p class="last">The binding mode of compounds is indicated as type II heme binding (II, orange), binding to the distal pocket (N, green), or nonbinding (X, unshaded). Binding was determined using a single shot UV–vis assay in which analogues (100 μM or 1 mM) were added to a solution of each P450 enzyme (2.5–5 μM) and the absorbance spectrum was collected between 800 and 250 nm. Compounds were prepared as DMSO-<i>d</i><sub>6</sub> stocks and added to give a maximum concentration of 1% v/v DMSO-<i>d</i><sub>6</sub>. The change in the maximum wavelength of the Soret band (Δλ<sub>max</sub>, nm) compared to that measured for each P450 in buffer containing 1% v/v DMSO-<i>d</i><sub>6</sub> was used to assess the binding mode and the relative strength of binding. Compounds that did not change the Soret λ<sub>max</sub> were tested in competition with known type II inhibitors (50 μM), with compounds that decreased the Δλ<sub>max</sub> compared to that observed in the presence of the type II inhibitor alone being designated as “non-heme binding” (N) competitive ligands. The binding affinity of analogue <b>25a</b> was calculated by direct or competitive optical titration, as described previously, depending on the mode of binding to each P450 respectively.<a onclick="showRef(event, 'ref17 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref17 ref58 ref59">(17, 58, 59)</a> *<i>K</i><sub>D</sub> calculated by ITC for analogue <b>25b</b>.</p></div></div><div></div></div><div class="NLM_p">A qualitative assessment of the binding promiscuity of the CYP121 ligands developed in this study was obtained by measuring the change in the absorbance maximum (Δλ<sub>max</sub>, nm) of the characteristic Soret band of each P450 isoform when incubated with the ligands (100 μM to 1 mM) compared to a DMSO (1% v/v) control. Binding interactions made by the compounds with the distal part of the P450 active site, which do not directly perturb the heme environment or λ<sub>max</sub> of the Soret band, were characterized by measuring the decrease in the Δλ<sub>max</sub> caused by known type II ligands when <i>Mtb</i> P450s were coincubated with mixtures of the type II ligand and the novel CYP121 inhibitors. The binding affinity of the most potent CYP121 compound <b>25a</b> has been provided to give a quantitative assessment of ligand selectivity across the different P450 isoforms. Experiments were performed as direct titrations for <i>Mtb</i> P450s CYP121, CYP126, and CYP142, or in competition with a known type II inhibitor where <b>25a</b> exhibited a binding mode that did not influence the λ<sub>max</sub> of the heme absorbance spectrum, as in the case of <i>Mtb</i> P450 isoforms CYP124 and CYP125. Owing to the weak binding affinity of <b>25a</b> for the <i>Mtb</i> P450 CYP144, a <i>K</i><sub>D</sub> was obtained instead for the related compound <b>25b</b> by ITC.</div><div class="NLM_p last">Subsequent to commencing work on these compounds, the original 4-(1<i>H</i>-1,2,4-triazol-1-yl)phenol fragment hit <b>1</b> for CYP121 had been identified to bind to all of the P450s reported here in ligand-observed NMR experiments (unpublished data). The biaryl retrofragment <b>5</b>, which consisted of the fused Ar1 and Ar2 motifs of lead <b>2</b> and could be seen to represent a “merged” analogue of hit fragment <b>1</b>, was less promiscuous, only binding to five of the seven P450s screened. Elaboration of <b>5</b> to include an Ar3 substituent, as seen in the 4-hydroxybenzyl substituted analogue <b>26a</b>, further enhanced CYP121 selectivity. In contrast, the incorporation of a heme binding 3-aminophenyl substituent onto the scaffold of <b>5</b>, as in <b>25a</b> and <b>25b</b>, reduced the selectivity for inhibitors for CYP121, as might be expected from the introduction of a binding group targeting a conserved active site feature. While type II spectral changes were identified for 4 of the 7 P450s studied, <b>25a</b> and <b>25b</b> bound in positions that did not perturb the P450 heme spectrum in 3 of the 7 isoforms tested, illustrating the preference of different P450s for certain metal-binding pharmacophores and the impact of protein microenvironment on metal chelation.<a onclick="showRef(event, 'ref49 ref56'); return false;" href="javascript:void(0);" class="ref ref49 ref56">(49, 56)</a> The <i>K</i><sub>D</sub> of <b>25a</b> varied between 0.44 and 27 μM across the different P450 isoforms, which gave a 30–1800-fold selectivity ratio (SR = off-target binding affinity/on-target binding affinity) for CYP121.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The low solubility of Ar2 analogue <b>19a</b> limited testing to the lower concentration of 100 μM which, in combination with the small size of the compound, possibly accounted for the apparently greater selectivity observed in the single-shot UV–vis assay. These results demonstrated how optimizing ligand affinity by targeting binding interactions made in the distal pocket, as through the introduction of Ar1 or Ar3 motifs, can enhance drug selectivity between P450s.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Selectivity against Human P450s</h3><div class="NLM_p">Drug–drug interactions resulting from the inhibition of human P450s, or toxicities resulting from the metabolic activation of drugs by human P450s, are leading causes of attrition during pharmaceutical development. Thus, understanding the SAR influencing P450 promiscuity and isoform selectivity is a topic of significant interest. The five human P450s CYP1A (CYP1A1 and CYP1A2 isoforms), CYP2C9, CYP2C19, CYP2D6, and CYP3A4 are responsible for the phase I metabolism of more than 80% of marketed drugs.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Compounds <b>19a</b>, <b>25a</b>, and <b>26a</b> were screened for inhibitory activity against these five human P450s and were found to have good selectivity profiles, showing weak or no inhibition (IC<sub>50</sub> > 10 μM) against all five human P450 isoforms (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The exception was compound <b>19a</b>, which was a moderately potent (1 μM < IC<sub>50</sub> < 10 μM) inhibitor of CYP1A (IC<sub>50</sub> = 1.22 μM), an isoform which has a documented preference for aromatic and heterocyclic amine substrates.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The reincorporation of Ar1 into analogue <b>19a</b> to give potent analogue <b>25a</b> resulted in a 20-fold increase in IC<sub>50</sub> against CYP1A (IC<sub>50</sub><b>25a</b> = 21.4 μM). When the corresponding improvement in CYP121 binding affinity for the two compounds was also taken into account, this amounted to an approximate 15000-fold better apparent selectivity ratio (app SR = off-target potency/on-target potency).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The 4-hydroxybenzyl substituted Ar3 analogue <b>26a</b> did not inhibit any of the human P450s by 50% within the tested concentration range. Taken together, the improved selectivity from reintroducing Ar1 to the heme binding analogue <b>19a</b> to give <b>25a</b>, and the high selectivity of the non-heme binding analogue <b>26a</b>, supported our hypothesis that targeting variable residues and hotspots in the distal pocket of the CYP121 active site could be used to achieve P450 isoform selectivity. Aromatic amines, such as those present in the potent CYP121 ligands <b>25a</b>, <b>24a</b>, <b>25b</b>, and <b>19a</b>, are commonly identified as potential toxicophores. However, considering 13% of marketed drugs contain aniline functional groups, it is premature to exclude all compounds with this functional group from drug-like chemical space.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The toxicity of aniline structures requires metabolic activation, which is primarily mediated by human P450s.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The good selectivity of CYP121 inhibitors, particularly compound <b>25a</b>, demonstrated here against human P450s, is promising for the development of this series of CYP121 inhibitors, however further assessment against other P450, monooxygenase, and peroxidase enzymes is warranted. The selectivity of compounds derived from lead <b>2</b> against human P450s was good compared to that of other classes of P450 inhibitors that have been reported recently<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63, 64)</a> and suggests that the careful optimization of metal binding functional groups and targeting active site residues are methods which could reduce P450 liability during drug development.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selectivity of CYP121 Ligands against Drug-Metabolizing Human P450 Isoforms<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" align="center" char="."><i>K</i><sub>D</sub> (μM)</th><th class="rowsep1 colsep0" align="center" char=".">app SR</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">CYP1A</th><th class="colsep0 rowsep0" align="center" char=".">CYP2C9</th><th class="colsep0 rowsep0" align="center" char=".">CYP2C19</th><th class="colsep0 rowsep0" align="center" char=".">CYP2D6</th><th class="colsep0 rowsep0" align="center" char=".">CYP3A4</th><th class="colsep0 rowsep0" align="center" char=".">CYP121</th><th class="colsep0 rowsep0" align="center" char=".">(IC<sub>50</sub>/<i>K</i><sub>D</sub>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="char" char=".">1.22</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">14.0</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25a</b></td><td class="colsep0 rowsep0" align="char" char=".">21.4</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">11.3</td><td class="colsep0 rowsep0" align="char" char=".">0.015</td><td class="colsep0 rowsep0" align="char" char=".">1427</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26a</b></td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">>25</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char=".">>4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup>a</sup><p class="last">The concentration of analogues <b>19a</b>, <b>25a</b>, and <b>26a</b> required to inhibit the activity of human P450s by 50% (IC<sub>50</sub>, μM) was calculated from a seven-point dose–response curve. Compounds that did not achieve 50% inhibition over the tested concentration range 0–25 μM were considered non-inhibitors.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Studies were performed in human liver microsomes. Turnover of human P450 substrates (CYP1A, ethoxyresorufin; CYP2C9, tolbutamide; CYP2C19, mephenytoin; CYP2D6, dextromethorphan; CYP3A4, midazolam) was detected by LC-MS/MS or fluorescence for CYP1A. Reported CYP1A activity represents the combined activities of both the CYP1A1 and CYP1A2 isoforms. The apparent selectivity ratio (app SR = off-target potency (IC<sub>50</sub>)/on-target potency (<i>K</i><sub>D</sub>)) was calculated as a ratio of the IC<sub>50</sub> (μM) of each compound for CYP1A versus their respective CYP121 <i>K</i><sub>D</sub> (μM).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The IC<sub>50</sub> values of known inhibitors were determined as positive controls and are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S3</a> of the Supporting Information. Experiments were conducted by Cyprotex (Cheshire, UK).</p></div></div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cellular Potency</h3><div class="NLM_p last">The antimycobacterial activity of the most potent analogues <b>19a</b>, <b>24a</b>, <b>25a</b>, <b>25b</b>, <b>26a</b>, and biaryl retrofragment <b>5</b> was assessed in vitro against <i>Mtb</i> H37Rv. No significant inhibition of bacterial growth was detected for any of the lead compounds (MIC<sub>90</sub> ≥ 50 μM) over the tested concentration range (0–100 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S4</a>, Supporting Information). Achieving adequate intracellular drug exposure is a particular challenge for TB therapeutics because of the unique cell-wall structure of the bacterium and the expression of numerous multisubstrate efflux transporters.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65, 66)</a> Efforts to determine whether low membrane permeability or efflux might account for the poor cellular activity of the potent CYP121 ligands described here are currently in progress.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Using a fragment-based approach, a novel series of potent and isoform selective inhibitors of the essential <i>Mtb</i> enzyme CYP121 have been developed. An initial fragment screen and synthetic optimization strategy yielded the previously reported non-heme binding lead <b>2</b>, which had micromolar affinity for CYP121 but low LE. The deconstruction of lead compound <b>2</b> and synthesis of component retrofragments <b>3</b>–<b>6</b> allowed the GE of individual structural motifs Ar1, Ar2, and Ar3 to be assessed. Analysis of the binding mode of retrofragments enabled the identification of activity hotspots and the selection of more LE scaffolds for synthetic optimization. The rational incorporation of a metal-binding functional group onto Ar2, guided by X-ray crystallography and UV–vis spectroscopy, was used to achieve a 1000-fold improvement in binding affinity over lead compound <b>2</b>, while the incorporation of structural groups that projected into the Ar1 or Ar3 regions of the distal active site was shown to modulate the isoform selectivity of inhibitors against human P450s. Extensive characterization of analogues by X-ray crystallography, UV–vis spectroscopy, and native mass spectrometry validated our design strategy, demonstrating that analogues maintained binding interactions in identified activity hotspots and formed stable complexes with the native CYP121 dimer. Analogues with low nanomolar potency (<i>K</i><sub>D</sub> = 15 nM) for CYP121 are reported, and these molecules have good selectivity against human drug metabolizing P450s and a 30-fold selectivity ratio for CYP121 over other <i>Mtb</i> P450 isoforms. Analysis of selectivity data from this series provides insight into factors governing the selectivity of P450s, including the effect of the protein microenvironment and preference for specific metal-binding pharmacophores,<a onclick="showRef(event, 'ref56 ref67'); return false;" href="javascript:void(0);" class="ref ref56 ref67">(56, 67)</a> which has relevance more broadly for drug development. Further structural optimization to improve the cellular activity of these analogues is ongoing with the aim of developing novel classes of anti-TB agents.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Chemistry</h3><div class="NLM_p">All reagents were purchased from Sigma-Aldrich Company Ltd. (Poole, UK) or Alfa Aesar (Heysham, UK) unless otherwise specified. Unless otherwise stated, reactions were conducted under positive pressure of a dry nitrogen atmosphere. Temperatures of 0 °C, −10 °C, and −78 °C were obtained by cooling the reaction vessel in a bath of ice, salt, and ice or acetone and dry ice (CO<sub>2</sub>(s)), respectively. Anhydrous DCM was prepared by distillation over calcium hydride. THF was freshly distilled over calcium hydride and lithium aluminum hydride, using triphenylmethane as an indicator. Other anhydrous solvents were purchased directly from commercial sources and used without further purification.</div><div class="NLM_p">Analytical thin layer chromatography (TLC) was performed using Merck glass-backed silica (Kieselgel 60 F254 0.25 mm) plates. Compounds were visualized using short wave (254 nm) or long wave (365 nm) ultraviolet light or potassium permanganate stain. Retention factors (<i>R</i><sub>f</sub>) are quoted with respect to the solvent system used to develop the plate. Flash column chromatography was performed using an Isolera Spektra One/Four purification system and the appropriately sized Biotage SNAP column containing KP-silica gel (50 μm). Solvents are reported as volume/volume eluent mixture where applicable. Reactions were monitored by TLC and LCMS to determine consumption of starting materials.</div><div class="NLM_p">Infrared absorption spectra were recorded on a Bruker ALPHA FT-IR or Spectrum One FT-IR (PerkinElmer) spectrometer by attenuated total reflectance (ATR) using a diamond crystal. Data are reported as vibrational frequency (cm<sup>–1</sup>) and intensity (strong, medium, weak, or broad) for the assigned functional group.</div><div class="NLM_p">Nuclear magnetic resonance (NMR) spectra were recorded at 300 K unless otherwise stated, using either a Bruker 400 MHz AVANCE III HD Smart Probe, 400 MHz QNP cryoprobe, or 500 MHz DCH cryoprobe spectrometer. Spectra acquired on the 500 MHz DCH cryoprobe have been converted to analogue and back to reduce baseline roll. All spectra were recorded in the deuterated solvent indicated. Data are reported as chemical shift in parts per million (δ ppm) relative to the residual protonated solvent resonance peak. The relative integral, multiplicity, coupling constants (<i>J</i> Hz), and assignment of peaks has been provided where possible. Assignment of <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra was assisted by DEPT, homonuclear (COSY), and heteronuclear (edited <sup>1</sup>H–<sup>13</sup>C-HSQC and <sup>1</sup>H–<sup>13</sup>C HMBC) 2D-NMR experiments. <sup>13</sup>C NMR spectra were processed with line broadening (LB) of 1–12 Hz for individual compounds to enable resolution of signals from slowly relaxing quaternary pyrazole carbons C3, C4, and C5 and resolution of closely overlapping aromatic signals. LB has been noted for relevant spectra and <sup>13</sup>C signals requiring LB for resolution have been noted as broad (br). <sup>13</sup>C signals that were obscured or were only identified through proton relaxation in 2D-NMR experiments are also noted.</div><div class="NLM_p">Liquid chromatography mass spectrometry (LCMS) was carried out using an AQUITY UPLC H-class system (Waters, Manchester UK). Samples were run using a gradient of water (1–5%) (+0.1% formic acid) in acetonitrile over a period of 4 min. High resolution mass spectra (HRMS) were recorded using a Micromass Quadrapole-Time-of-Flight (Q-Tof) spectrophotometer or an Orbitrap LCMS spectrometer attached to a Dionex Ultimate 3000 HPLC. The mass to charge ratio (<i>m</i>/<i>z</i>) of the molecular ion and difference from calculated mass (δ ppm) have been quoted.</div><div class="NLM_p">All final compounds had a purity greater than 98% by LCMS analysis, except for compounds <b>14b</b>, <b>18g</b>, <b>18f</b>, <b>19c</b>, and <b>26b</b>, which were 92–98% pure by LCMS analysis.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Synthesis of Retrofragment (<b>3</b>)</h4><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 4-(5-Amino-1<i>H</i>-pyrazol-3-yl)phenol</h5><div class="NLM_p"><i>Step 1: Synthesis of 3-(4-methoxyphenyl)-3-oxopropanenitrile</i>. Methyl 4-methoxybenzoate (831 mg, 5 mmol) and acetonitrile (1 mL) were added to a stirred suspension of NaH (60% dispersion in mineral oil, 490 mg, 12.25 mmol) in anhydrous THF (25 mL). The reaction was stirred at 60 °C for 40 h, and then the volatiles were removed under reduced pressure. The crude residue was diluted with H<sub>2</sub>O (15 mL) and acidified with 1 M HCl, and then the product was extracted into DCM (3 × 20 mL). The combined organic fractions were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (40–100% v/v DCM in petroleum ether) to yield 3-(4-methoxyphenyl)-3-oxopropanenitrile as a white solid (578 mg, 3.30 mmol, 66%), <i>R</i><sub>f</sub> 0.30 (1:2 v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 174.1 [M – H]<sup>−</sup>, retention time 1.68 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>10</sub>H<sub>10</sub>NO<sub>2</sub> [M + H]<sup>+</sup> = 176.0712), obsd 176.0703 (δ ppm = 5.1). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.90 (d, <i>J</i> = 9.0 Hz, 2H), 6.98 (d, <i>J</i> = 9.0 Hz, 2H), 4.02 (s, 2H), 3.90 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 185.5, 164.9, 131.1, 127.4, 114.5, 114.2, 55.8, 29.2 ppm. IR (thin film) <i>v</i><sub>max</sub> 3083, 3020, 2942, 2844 (w, C–H), 2259 (w, nitrile C–N), 1685 (s, C═O), 1623 (w, C═C), 1596 (s), 1514 (s), 1465, 1450 (s, CH<sub>3</sub>), 1439 (m, C═C), 1326 (s, CH<sub>3</sub>), 1268 (s, C–O, CH<sub>2</sub>), 1224, 1172 (s, C–O), 1011 (s, C–O, CH<sub>2</sub>), 930, 828, 814 (s, C–H) cm<sup>–1</sup>. </div><div class="NLM_p"><i>Step 2: Synthesis of 3-(4-methoxyphenyl)-1H-pyrazol-5-amine</i>. Concentrated HCl (37% v/v, 2.4 mL) was added slowly to a stirred solution of 3-(4-methoxyphenyl)-3-oxopropanenitrile (552 mg, 3.15 mmol) in absolute EtOH (20 mL). The reaction was heated to reflux, and then hydrazine hydrate (750 μL, 15.5 mmol) was added dropwise over 10 min. The reaction was heated under reflux for 5 h and then concentrated under reduced pressure (5 mL), diluted with H<sub>2</sub>O (10 mL), and brought to pH 8.0 using saturated NaHCO<sub>3</sub>. The product was extracted into EtOAc (3 × 20 mL), and the combined organic fractions were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield 3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine as a white solid (512 mg, 2.71 mmol, 86%), <i>R</i><sub>f</sub> 0.18 (5% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 190.2 [M + H]<sup>+</sup>, retention time 1.27 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>10</sub>H<sub>12</sub>N<sub>3</sub>O [M + H]<sup>+</sup> = 190.0975), obsd 190.0970 (δ ppm = 2.4). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.50 (br, s, 1H), 7.38 (d, <i>J</i> = 8.8 Hz, 2H), 6.93 (d, <i>J</i> = 8.8 Hz, 2H), 3.79 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 159.4, 152.7, 128.9 (2 × C), 127.0, 115.2, 109.6, 55.7 ppm. IR (thin film) <i>v</i><sub>max</sub> 3397, 3148 (m, br, N–H), 2962, 2839 (m, C–H), 1615 (m, N–H), 1505, 1479, 1462 (s, N–H, C–H, C═C), 1439 (m, C═C), 1284 (m, C–N, C–O), 1244 (s, C–N, C–O), 1177, 1111 (m, C–H), 1028 (s, C–O), 831, 817, 769 (s, C–H), 710 (m, C–H) cm<sup>–1</sup>. </div><div class="NLM_p last"><i>Step 3: Synthesis of 4-(5-Amino-1H-pyrazol-3-yl)phenol (<b>3</b>).</i> A solution of 1 M BBr<sub>3</sub> in DCM (12.5 mL, 12.5 mmol) was added dropwise over a period of 10 min to a stirred solution of 3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (473 mg, 2.5 mmol) in anhydrous DCM (22 mL) at 0 °C. The reaction was stirred at 0 °C and allowed to come to room temperature over 14 h. When complete, the reaction was quenched with saturated NaHCO<sub>3</sub> and extracted into EtOAc (3 × 30 mL). The organic phase was combined, washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure to yield 4-(5-amino-1<i>H</i>-pyrazol-3-yl)phenol <b>3</b> as an off-white solid (342 mg, 1.95 mmol, 75%), <i>R</i><sub>f</sub> 0.35 (10% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 176.2 [M + H]<sup>+</sup>, retention time 0.81 min (100%). HRMS (+ ESI): <i>m</i>/<i>z</i> (calcd C<sub>9</sub>H<sub>10</sub>N<sub>3</sub>O [M + H]<sup>+</sup> = 176.0824), obsd 176.0823 (δ ppm = 0.6). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.49 (s, 1H), 7.44 (d, <i>J</i> = 9.0 Hz, 2H), 6.76 (d, <i>J</i> = 8.5 Hz, 2H), 5.63 (s, 1H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.9, 153.6, 145.2, 126.1, 122.7, 115.4, 86.9 ppm. IR (thin film) <i>v</i><sub>max</sub> 3305, 3141 (w, br, N–H, O–H), 1615 (m, N–H, C═C), 1519 (s, N–H), 1489 (s, N–H), 1377 (w, O–H), 1279, 1244 (m, C–N, C–O), 1179 (m, C–O), 1111, 1014 (m, C–H), 834 (s, C–H), 765 (s, C–H) cm<sup>–1</sup></div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Synthesis of Retrofragment (<b>6</b>)</h4><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 4-((3-(4-Hydroxyphenyl)-1<i>H</i>-pyrazol-5-yl)amino)phenol</h5><div class="NLM_p"><i>Step 1: Synthesis of methyl 3-(4-methoxyphenyl)-3-oxopropanoate</i>. Dimethyl carbonate (1.26 mL, 15 mmol) was added to a stirred suspension of NaH (60% dispersion in mineral oil, 600 mg, 15 mmol) in anhydrous toluene (2 mL). The reaction was brought to reflux, and a solution of 4-methoxyacetophenone (0.90 g, 6 mmol) in anhydrous toluene (3 mL) was added dropwise. The reaction was heated at reflux for a further 15 min and then allowed to cool to room temperature. The reaction was quenched with a solution of AcOH (0.5 mL) in water (2 mL). The organic layer was separated, and the aqueous fraction was extracted with Et<sub>2</sub>O (2 × 5 mL). The organics were combined, washed with a solution of saturated K<sub>2</sub>CO<sub>3</sub>, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (10–50% v/v EtOAc in petroleum ether) to yield methyl 3-(4-methoxyphenyl)-3-oxopropanoate as a yellow oil (1.05 g). The product was 90% pure by <sup>1</sup>H NMR integration and was taken forward without further purification; <i>R</i><sub>f</sub> 0.18 (20% v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 207.2 [M – H]<sup>−</sup>, retention time 1.75 min (96%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>11</sub>H<sub>13</sub>O<sub>4</sub> [M + H]<sup>+</sup> = 209.0808), obsd 209.0803 (δ ppm = 2.4). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 9.0 Hz, 2H), 6.94 (d, <i>J</i> = 9.0 Hz, 2H), 3.96 (s, 2H), 3.87 (s, 3H), 3.74 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 190.9, 168.3, 164.2, 130.0, 129.2, 114.1, 55.7, 52.6, 45.7 ppm. IR (thin film) <i>v</i><sub>max</sub> 2954, 2842 (w, C–H), 1739 (s, C═O), 1675 (s, C═O), 1599 (s, C═C), 1575, 1511 (m, C═C), 1437, 1421 (m, CH<sub>2</sub>, CH<sub>3</sub>), 1323 (m, C–O, C–H), 1258 (s, C–O), 1215 (m), 1147, 1170 (s, C–O, C–H), 1025 (m, C–O), 986 (m), 840 (m, C–H) cm<sup>–1</sup>. </div><div class="NLM_p"><i>Step 2: Synthesis of N,3-Bis(4-methoxyphenyl)-3-oxopropanamide</i>. A solution of 4-methoxyaniline (406 mg, 3.3 mmol) in <i>m</i>-xylene (6 mL) was added to a stirred solution of methyl 3-(4-methoxyphenyl)-3-oxopropanoate (623 mg, 3 mmol) in <i>m</i>-xylene (6 mL), and the reaction was heated to 150 °C for 20 h. The product precipitated on cooling to room temperature and was purified by trituration at 0 °C with ice-cold EtOAc:hexane (1:5 v/v). The precipitate was collected by vacuum filtration, washed with 1:5 EtOAc:hexane followed by 1:3 EtOAc:hexane, and dried under reduced pressure to give <i>N</i>,3-bis(4-methoxyphenyl)-3-oxopropanamide as a brown powder (567 mg, 1.9 mmol, 63%), <i>R</i><sub>f</sub> 0.06 (1:2 EtOAc:petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 322.0 [M + Na]<sup>+</sup>, retention time 1.87 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>18</sub>NO<sub>4</sub> [M + H]<sup>+</sup> = 300.1236), obsd 300.1251 (δ ppm = 5.0). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.26 (s, 1H), 8.02 (d, <i>J</i> = 8.9 Hz, 2H), 7.48 (d, <i>J</i> = 9.0 Hz, 2H), 6.97 (d, <i>J</i> = 8.9 Hz, 2H), 6.86 (d, <i>J</i> = 9.0 Hz, 2H), 4.04 (s, 2H), 3.89 (s, 3H), 3.79 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 195.1, 164.7, 164.0, 156.6, 131.2, 130.9, 129.2, 122.0, 114.3, 114.2, 55.8, 55.6, 45.1 ppm. IR (thin film) <i>v</i><sub>max</sub> 3279 (w, N–H), 3144, 2963, 2840 (w, C–H), 1685 (m, C═O), 1598 (s, C═C), 1556 (m, C═C), 1509 (s, N–H), 1465, 1419 (w, CH<sub>2</sub>, CH<sub>3</sub>), 1323, 1263, 1246, 1217, 1170 (m, C═C, C–O, C–N), 1028, 1013 (C–O), 826 (s, C–H) cm<sup>–1</sup>. </div><div class="NLM_p"><i>Step 3: Synthesis of N,3-Bis(4-methoxyphenyl)-1H-pyrazol-5-amine</i>. Lawesson’s reagent (419 mg, 1.04 mmol) was added to a stirred solution of <i>N</i>,3-bis(4-methoxyphenyl)-3-oxopropanamide (120 mg, 0.4 mmol) in dry THF (20 mL). The reaction was heated to reflux for 4 h and then concentrated under reduced pressure to give a red oil. The sulphonamide intermediate was redissolved in absolute EtOH (5 mL), and glacial AcOH (69 μL, 1.3 mmol) was added. The flask was evacuated, flushed with N<sub>2</sub> gas, and then hydrazine monohydrate (66 μL, 1.3 mmol) added. The reaction was heated to reflux for 3 h and then allowed to stir at room temperature for 24 h. When complete, the reaction was concentrated under reduced pressure. The crude orange oil was diluted with DCM (20 mL) and made basic with a satd NaHCO<sub>3</sub> solution (5 mL). The phases were separated, and the aqueous layer was extracted with DCM (2 × 2 mL). The organic fractions were combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the crude product dried directly onto silica gel. Flash chromatography (25–80% v/v EtOAc in petroleum ether) yielded an off-white solid that was further purified by trituration with Et<sub>2</sub>O to give <i>N</i>,3-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine as a white powder (41 mg, 0.14 mmol, 35%), <i>R</i><sub>f</sub> 0.11 (2:1 v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 296.3 [M + H]<sup>+</sup>, retention time 1.95 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 296.1399), obsd 296.1393 (δ ppm = 2.0). <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.42 (s, 1H), 7.67 (d, <i>J</i> = 8.7 Hz, 2H), 7.35 (m, 2H), 6.99 (d, <i>J</i> = 8.9 Hz, 2H), 6.81 (d, <i>J</i> = 9.0 Hz, 2H), 6.13 (d, <i>J</i> = 1.2 Hz, 1H), 3.83 (s, 3H), 3.73 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>) δ 160.9, 154.2, 144.7, 139.2, 127.7, 124.6, 118.0, 117.97, 115.42, 115.40, 91.0, 56.1, 56.0 ppm. <sup>13</sup>C peak at 144.7 identified from <sup>1</sup>H–<sup>13</sup>C-HMBC spectrum. IR (thin film) <i>v</i><sub>max</sub> 3402 (m, N–H), 3270 (br, w, N–H), 2968, 2840 (w, C–H), 1612, 1598, 1587 (m, C═C), 1554, 1505, 1464, 1427 (s, C–H, C═N), 1294 (m, C–N), 1239 (s, C–O), 1182 (s, C–N), 1111 (m), 1026 (s, C–O), 959 (m), 823, 762 (s, C–H) cm<sup>–1</sup>. </div><div class="NLM_p last"><i>Step 4: 4-((3-(4-Hydroxyphenyl)-1H-pyrazol-5-yl)amino)phenol (<b>6</b>)</i>. A 1 M solution of BBr<sub>3</sub> in DCM (1.4 mL, 1.4 mmol) was added dropwise over a period of 15 min to a stirred suspension of <i>N</i>-3-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (104 mg, 0.35 mmol) in dry DCM (7.5 mL) at 0 °C. The reaction was allowed to come slowly to room temperature and stirred for 8 h, then quenched with ice water (40 mL). EtOAc (20 mL) was added, and the aqueous phase was made slightly basic with satd NaHCO<sub>3</sub>. The phases were separated, and the aqueous fraction was extracted with EtOAc (3 × 15 mL). The organic fractions were combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (5–10% v/v MeOH in DCM, followed by 5% MeOH in EtOAc) to yield 4-((3-(4-hydroxyphenyl)-1<i>H</i>-pyrazol-5-yl)amino)phenol (<b>6</b>) as an off-white solid (43 mg, 0.16 mmol, 46%), <i>R</i><sub>f</sub> 0.52 (10% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 268.2 [M + H]<sup>+</sup>, retention time 1.37 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 268.1081), obsd 296.1070 (δ ppm = 4.0). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.49 (d, <i>J</i> = 8.5 Hz, 2H), 7.05 (d, <i>J</i> = 8.0 Hz, 2H), 6.82 (d, <i>J</i> = <i>J</i> = 8.7 Hz, 2H), 6.70 (d, <i>J</i> = 8.9 Hz, 2H), 6.03 (s, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.9, 155.5 (br), 152.0, 145.5 (br), 137.9, 127.8, 122.8, 119.2, 116.6, 116.6, 90.7 (br) ppm. IR (thin film) <i>v</i><sub>max</sub> 3417 (s, br, O–H), 3292 (s, br, N–H), 1607, 1615 (w, C═C), 1574 (s, C═C, N–H), 1522 (s, C═C, N–H), 1383 (w, C–O), 1254 (m, br, C–O, C–N), 1107 (w, C–H), 830 (s, C–H), 759 (m, O–H) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> General Procedure A</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Synthesis of β-Ketonitriles <b>12a</b>–<b>c</b> and <b>21a</b>–<b>b</b></h4><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 2,3-Bis(4-methoxyphenyl)-3-oxopropanenitrile (<b>12a</b>)</h5><div class="NLM_p last">2-(4-Methoxyphenyl)acetonitrile <b>11</b> (2.32 mL, 17.1 mmol) and methyl 4-methoxybenzoate <b>10a</b> (2.58 g, 15.5 mmol) were added to a stirred suspension of NaH (60% in mineral oil, 1.34 g, 33.6 mmol) in anhydrous THF (30 mL). The reaction was heated to 60 °C for 20 h then cooled to 0 °C and quenched with H<sub>2</sub>O (10 mL). The suspension was concentrated under reduced pressure then diluted with dichloromethane (50 mL) and acidified with 3 M HCl to pH 2–3. The organic layer was separated off, and the aqueous phase was extracted with dichloromethane (3 × 50 mL). The organic fractions were combined, washed with brine (10 mL), dried over anhydrous MgSO<sub>4</sub>, and the solvent removed under reduced pressure to yield 2,3-bis(4-methoxyphenyl)-3-oxopropanenitrile <b>12a</b> as a yellow gum (3.84 g, 13.7 mmol, 88%). The crude oxopropanenitriles <b>12a</b> was used in subsequent reactions without further purification, although other analogues were purified by flash chromatography eluting with the solvent system specified. LCMS (+ ESI): <i>m</i>/<i>z</i> 280.2 [M + Na]<sup>+</sup>; retention time 2.11 min (93%). HRMS (+ ESI): <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>16</sub>NO<sub>3</sub> [M + H]<sup>+</sup> = 282.1130), obsd 282.1135 (δ ppm =1.8). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 9.0 Hz, 2H), 7.34 (d, <i>J</i> = 8.8 Hz, 2H), 6.92 (d, <i>J</i> = 7.8 Hz, 2H), 6.90 (d, <i>J</i> = 7.8 Hz, 2H), 5.50 (s, 1H), 3.85 (s, 3H,), 3.78 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 187.6, 164.5, 160.1, 131.8, 129.5, 126.5, 122.7, 117.2, 115.1, 114.3, 55.7, 55.5, 45.8 ppm. IR (thin film) <i>v</i><sub>max</sub> 3008, 2936, 2839 (w, C–H), 2248 (w, nitrile CN), 1677 (m, C═O), 1597, 1510 (s, C═C), 1462, 1422 (w), 1305 (m, C–O), 1254, 1237, 1173 (s, C–O), 1027 (m), 939, 822, 780 (m) cm<sup>–1</sup>.</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 3-(3-Methoxyphenyl)-2-(4-methoxyphenyl)-3-oxopropanenitrile (<b>12b</b>)</h5><div class="NLM_p last">Prepared as an orange oil (0.703 g, 2.50 mmol, 83%) according to general procedure A. The crude product was purified by flash chromatography (0–50% v/v EtOAc in petroleum ether), <i>R</i><sub>f</sub> 0.55 (50% v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 294.2 [M + H]<sup>+</sup>, retention time 2.13 min. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.49 (d, <i>J</i> = 7.6 Hz, 1H), 7.45 (app t, <i>J</i> = 2.4, 1.6 Hz, 1H), 7.34 (m, 3H), 7.11 (d, <i>J</i> = 8.4, 2.0 Hz, 1H), 6.90 (d, <i>J</i> = 8.8 Hz, 2H), 5.53 (s, 1H), 3.82 (s, 3H), 3.78 (s, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 189.1, 160.2, 160.1, 135.1, 130.0, 129.6, 122.3, 121.8, 121.0, 116.9, 115.2, 113.7, 55.6, 55.5, 46.2 ppm.</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 3-(3-Fluorophenyl)-2-(4-methoxyphenyl)-3-oxopropanenitrile (<b>12c</b>)</h5><div class="NLM_p last">Prepared as an orange solid (229 mg, 0.852 mmol, 85%) according to general procedure A. The crude product was purified by flash chromatography (10–50% v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 270.2 [M + H]<sup>+</sup>, retention time 2.08 min (89%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>13</sub>FNO<sub>2</sub> [M + H]<sup>+</sup> = 270.0930), obsd 270.0933 (δ ppm = 1.1). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.71 (d, <i>J</i> = 8.0 Hz, 1H), 7.61 (d, <i>J</i> = 9.0 Hz, 1H), 7.43 (m, 1H), 7.33 (d, <i>J</i> = 8.8 Hz, 2H), 7.29 (m, 1H), 6.91 (d, <i>J</i> = 8.9 Hz, 2H), 5.49 (s, 1H), 3.79 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 188.1, 162.8 (d, <i>J</i><sub>C–F</sub> = 248 Hz), 160.4, 135.6 (<i>J</i><sub>C–F</sub> =  6.4 Hz), 130.8 (d, <i>J</i><sub>C–F</sub> =  8 Hz), 129.5, 125.1 (d, <i>J</i><sub>C–F</sub> =  2 Hz), 121.6 (d, <i>J</i><sub>C–F</sub> = 10 Hz), 121.5, 116.5, 116.0 (d, <i>J</i><sub>C–F</sub> =  23 Hz), 115.3, 55.5, 46.4 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3076, 2959, 2936, 2840 (w, C–H), 2248, 2207 (w, nitrile CN), 1694 (m, C═O), 1608, 1588 (m, C═C), 1510 (s), 1439, 1304 (m), 1252 (s), 1149, 1030 (m, C–O), 885 (w), 834, 803, 788 (m) cm<sup>–1</sup>.</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 2-(3-Bromophenyl)-3-(4-methoxyphenyl)-3-oxopropanenitrile (<b>21a</b>)</h5><div class="NLM_p last">Prepared as a yellow semisolid (2.18 g) according to general procedure A and used in subsequent reactions without further purification. The crude product contained 15% unreacted starting material by <sup>1</sup>H NMR integration; <i>R</i><sub>f</sub> 0.18 (50% v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 328.1 [M – H]<sup>−</sup>, retention time 2.23 min (59%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (d, <i>J</i> = 8.9 Hz, 2H), 7.62 (s, 1H), 7.49 (d, <i>J</i> = 8.0 Hz, 1H), 7.8 (d, <i>J</i> = 7.8 Hz, 1H), 7.27 (m, 1H), 6.95 (d, <i>J</i> = 8.9 Hz, 2H), 5.53 (s, 1H), 3.89 (s, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 186.7, 164.9, 132.9, 132.4, 131.9, 131.3, 127.0, 126.3, 123.6, 121.7, 116.4, 114.6, 55.8, 45.7 ppm. IR (thin film) <i>v</i><sub>max</sub> 2937, 2842 (w, C–H), 2252 (w, CN), 1679 (m, C═O), 1597 (s), 1572, 1512, 1424, 1323 (m, C═C, C–H), 1264, 1238, 1172 (s, C–N, C–O), 1026 (m), 941, 833, 775, 692 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 2-(4-(Benzyloxy)-3-bromophenyl)-3-(4-methoxyphenyl)-3-oxopropanenitrile (<b>21b</b>)</h5><div class="NLM_p last">Prepared as an orange semisolid (0.845 g, 1.56 mmol, quant.) according to general procedure A and either used directly in subsequent reactions or further purified by flash chromatography (0–10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 436.1 [M + H]<sup>+</sup>, retention time 2.44 min (94%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>23</sub>H<sub>19</sub>BrNO<sub>3</sub> [M + H]<sup>+</sup> = 436.0548), obsd 436.0561 (δ ppm = 3.0). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 8.9 Hz, 2H), 7.63 (d, <i>J</i> = 2.4 Hz, 1H), 7.44 (d, <i>J</i> = 7.4 Hz, 2H), 7.38 (appt, <i>J</i> = 7.6, 7.2 Hz, 2H), 7.34–7.30 (m, 2H), 6.93 (m, 3H), 5.45 (s, 1H), 5.13 (s, 2H), 3.87 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 186.8, 164.6, 155.6, 136.0, 133.1, 131.8, 128.7, 128.3, 128.1, 126.9, 126.2, 124.2, 116.7, 114.5, 114.3, 113.5, 70.9, 55.7, 44.9 ppm. IR (thin film) <i>v</i><sub>max</sub> 3064, 3033, 2935, 2840 (w, C–H), 2249 (w, nitrile C–N), 1678 (m, C═O), 1597 (s), 1572, 1510 (m, C═C), 1492 (s), 1454 (m), 1382 (w, C–N), 1261, 1220, 1173 (s, C–N, C–O), 1052, 1024 (m, C–O), 808, 735 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> General Procedure B</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis of Bisphenyl-1<i>H</i>-pyrazol-5-amines <b>13a</b>–<b>c</b> and <b>22a</b>–<b>b</b></h4><div id="sec4_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 3,4-Bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>13a</b>)<a onclick="showRef(event, 'ref28 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref33">(28, 33)</a></h5><div class="NLM_p last">Concentrated HCl (37% v/v, 6.25 mL) was added slowly to a stirred solution of crude 2,3-bis(4-methoxyphenyl)-3-oxopropanenitrile <b>12a</b> (4.16g, ∼11.84 mmol, 1 equiv) in absolute EtOH (63 mL). The reaction was then heated to reflux, and hydrazine hydrate (3.6 mL, 74 mmol, 5 equiv) was added dropwise over 10 min. After heating at reflux until all starting material had been consumed (typically 5–15 h), the reaction was cooled to room temperature and concentrated under reduced pressure (20 mL). The reaction was diluted in water (15 mL) and brought to pH 8.0 using saturated NaHCO<sub>3</sub>. The aqueous emulsion was extracted with EtOAc (3 × 20 mL). The organic fractions were combined and washed with brine (10 mL), dried over anhydrous MgSO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (40% v/v EtOAc in petroleum ether, then 0–10% v/v MeOH in EtOAc) to yield 3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine <b>13a</b> as a yellow solid (3.171 g, 10.7 mmol, 91%), <i>R</i><sub>f</sub> 0.29 (10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 296.1 [M + H]<sup>+</sup>; retention time 2.01 min (95%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 296.1399), obsd 296.1392 (δ ppm = 2.4). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.77 (s, br, 1H), 7.22 (d, <i>J</i> = 8.8 Hz, 2H), 7.10 (d, <i>J</i> = 8.5 Hz, 2H), 6.89 (d, <i>J</i> = 8.8 Hz, 2H), 6.86 (d, <i>J</i> = 8.8 Hz, 2H), 4.39 (s, br, 2H), 3.75 (s, 3H), 3.74 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.6, 157.4, 152.6, 139.4, 130.5, 128.3, 125.9, 123.8, 114.0, 113.8, 103.0, 55.6, 55.0 ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (solid) <i>v</i><sub>max</sub> 3432, 3349 (w, N–H), 3210 (w, br, N–H), 3932, 2833 (w, C–H), 1622 (w, C═C), 1513, 1489 (m, N–H, C–H), 1303, 1284 (w), 1241 (s, C–O, C–N), 1179 (s, C–O), 1111 (w), 1020 (s, C–O), 964 (w, C–H), 834 (s, C–H), 798, 735 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_3_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 3-(3-Methoxyphenyl)-4-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>13b</b>)</h5><div class="NLM_p last">Prepared as an orange amorphous solid (680 mg, 2.30 mmol, 92%) according to general procedure B. The crude material was carried forward without further purification. LCMS (ESI+) <i>m</i>/<i>z</i> 296.2 [M + H]<sup>+</sup>, retention time 1.80 min (93%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 296.1394), obsd 296.1381 (δ ppm = 4.4). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.19–7.12 (m, 3H), 6.90 (m, 3H), 6.85 (dd, <i>J</i> = 2.6, 1.5 Hz, 1H), 6.80 (ddd, <i>J</i> = 8.3, 2.6, 1.0 Hz, 1H), 3.77 (s, 3H), 3.62 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 161.0, 160.0, 153.1 (br), 143.3 (br), 133.4 (br), 132.2, 130.6, 126.4, 120.8, 115.3, 115.0, 113.8, 106.3 (br), 55.7, 55.5 ppm. <sup>13</sup>C NMR spectrum processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3422, 3336, 3182 (m, br, N–H), 2999, 2938, 2834 (m, C–H), 1607, 1578, 1525, 1488, 1464, 1285 (m, C═C), 1243 (s, C–N), 1177, 1040, 1022 (m, C–O), 834, 788, 749, 694 (m) cm<sup>–1</sup>.</div></div><div id="sec4_3_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 3-(3-Fluorophenyl)-4-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>13c</b>)</h5><div class="NLM_p last">Prepared as an orange solid (142 mg, 0.50 mmol, 97%) according to general procedure B. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 284.2 [M + H]<sup>+</sup>, retention time 1.82 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>15</sub>FN<sub>3</sub>O [M + H]<sup>+</sup> = 284.1199), obsd 284.1188 (δ ppm = 3.9). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.24 (m, 1H), 7.19 (d, <i>J</i> = 8.7 Hz, 2H), 7.12 (dt, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.03 (ddd, <i>J</i> = 9.8, 2.6, 1.6 Hz, 1H), 6.97 (app tdd, <i>J</i> = 8.4, 2.6, 1.0 Hz, 1H), 6.91 (d, <i>J</i> = 8.7 Hz, 2H), 3.83 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 162.9 (d, <i>J</i><sub>C–F</sub> = 249 Hz), 158.7, 152.8, 140.2, 132.7 (d, <i>J</i><sub>C–F</sub> =  8 Hz), 130.9, 130.4 (d, <i>J</i><sub>C–F</sub>= 8 Hz), 124.4, 123.0 (d, <i>J</i><sub>C–F</sub> = 3 Hz), 115.2 (d, <i>J</i><sub>C–F</sub> = 21 Hz), 114.6, 114.3 (d, <i>J</i><sub>C–F</sub> = 23 Hz), 106.3, 55.4 ppm. <sup>19</sup>F-NMR (376 MHz, CDCl<sub>3</sub>) δ −112.4 ppm. IR (thin film) <i>v</i><sub>max</sub> 3310, 3190 (m, br, N–H), 3000, 2934, 2836 (m, C–H), 1613 (m, C═C), 1579, 1524 (m, N–H), 1488 (m, C═C), 1286 (m), 1245 (s, C–N, C–O), 1176, 1021 (m), 864, 835 (m, C–H), 788, 751 (m, C–H), 690, 605 (w) cm<sup>–1</sup>.</div></div><div id="sec4_3_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 4-(3-Bromophenyl)-3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>22a</b>)</h5><div class="NLM_p last">Prepared as a brown amorphous solid (1.86 g, 5.42 mmol, 77% over two reaction steps from 3-bromophenylacetonitrile). The crude product was purified by flash chromatography (50–100% v/v EtOAc in petroleum ether), <i>R</i><sub>f</sub> 0.08 (50% v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 346.1 [M (<sup>81</sup>Br) + H]<sup>+</sup>, retention time 1.96 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>15</sub><sup>79</sup>BrN<sub>3</sub>O [M + H]<sup>+</sup> = 344.0393), obsd 344.0385 (δ ppm = 2.5). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.37 (m, 2H), 7.24–7.19 (m, 4H), 6.88 (d, <i>J</i> = 7.2 Hz, 2H), 3.78 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 161.3, 152.6 (br), 144.2 (br), 137.1, 133.4, 131.4, 130.2 (2C), 129.5, 124.2 (br), 123.6, 115.1, 104.6 (br), 55.7 ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3063, 2936, 2841 (w, C–H), 2250 (w, nitrile C–N), 1677 (m, C═O), 1596 (s), 1574, 1510, 1474, 1423 (m, C═C), 1322 (m), 1264, 1239 (s, C–O), 1173 (s), 1027 (m, C–O), 941 (m), 832, 772, 693 (m) cm<sup>–1</sup>.</div></div><div id="sec4_3_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 4-(4-(Benzyloxy)-3-bromophenyl)-3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>22b</b>)</h5><div class="NLM_p last">Prepared as a yellow foam (881 mg, 1.96 mmol, 95%) according to general procedure B. No further purification by flash chromatography was required. LCMS (ESI+) <i>m</i>/<i>z</i> 450.2, 452.2 [M + H]<sup>+</sup>, retention time 2.17 min (98%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>23</sub>H<sub>21</sub>BrN<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 450.0817), obsd 450.0811 (δ ppm = 1.3). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.52 (d, <i>J</i> = 2.1 Hz, 1H), 7.49 (d, <i>J</i> = 7.3 Hz, 2H), 7.40 (app t, <i>J</i> = 7.5 Hz, 2H), 7.33 (t, <i>J</i> = 7.3 Hz, 1H), 7.24 (d, <i>J</i> = 8.7 Hz, 2H), 7.11 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.91 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (d, <i>J</i> = 8.8 Hz, 2H), 5.16 (s, 2H), 3.80 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 159.9, 153.9, 152.8, 141.4, 136.6, 134.3, 129.9, 128.8, 128.7, 128.1, 127.2, 126.9, 122.3, 114.4, 114.2, 112.9, 104.1, 71.0, 55.4 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3419, 3197 (w, br, N–H), 2933, 2836 (w, C–H), 1613 (m, C═C), 1523 (s, N–H), 1454 (m), 1381 (w), 1280, 1249 (s, C–N), 1178, 1031 (m, C–O), 834, 737, 697 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> General Procedure C</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Synthesis of Oxopropanenitriles <b>16a</b>–<b>c</b></h4><div id="sec4_4_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-(3-Bromophenyl)-3-oxopropanenitrile (<b>16a</b>)</h5><div class="NLM_p last">2-(3-Bromophenyl)acetonitrile <b>15a</b> (1.09 g, 5.5 mmol) was added to a freshly prepared solution of sodium metal (139 mg, 6.05 mmol) in EtOH (7.5 mL) at 0 °C. After the reaction was stirred on ice for 5 min, ethyl formate (1.11 mL, 13.8 mmol) was added and the reaction was heated to reflux for 15–24 h. When no starting material remained, the reaction was cooled and concentrated under reduced pressure. The residue was diluted with H<sub>2</sub>O (5 mL) and acidified to pH 4 using 1 M HCl. The aqueous phase was extracted with Et<sub>2</sub>O or EtOAc (3 × 10 mL), and the combined organic fractions were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent evaporated to yield an off-white solid (1.18 g, 5.26 mmol, 96%). The crude product was carried forward into subsequent reaction steps as a mixture of enantiomers and some acetonitrile starting material; <i>R</i><sub>f</sub> 0.08 (50% v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 222.0 [M – H]<sup>−</sup>, retention time 2.01 min (97%). HRMS (ESI−): <i>m</i>/<i>z</i> (calcd C<sub>9</sub>H<sub>6</sub>BrNO [M – H]<sup>−</sup> = 221.9560), obsd 221.9556 (δ ppm = 1.7). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.94 (s, 1H), 7.90 (s, 1H), 7.66 (d, <i>J</i> = 7.8 Hz, 1H), 7.59 (m, 1H), 7.57 (s, 1H), 7.54 (m, 1H), 7.51 (d, <i>J</i> = 7.9 Hz, 1H), 7.44–7.33 (m, 4H), 7.33–7.20 (m, 3H), 4.76 (d, <i>J</i> = 7.4 Hz, 1H) ppm. IR (thin film) <i>v</i><sub>max</sub> 3103, 2977 (m, C–H), 2725 (w, aldehyde C–H), 2223 (m, nitrile C–N), 1644 (m, C═O), 1594, 1558 (m, C═C), 1473 (m), 1378 (s, aldehyde C–H), 1253 (s), 1194, 1073 (m), 874 (w), 771 (s), 680 (m) cm<sup>–1</sup>.</div></div><div id="sec4_4_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Synthesis of 2-(4-(Benzyloxy)-3-bromophenyl)acetonitrile (<b>15b</b>)</h5><div class="NLM_p last"><i>Step 1: Synthesis of 2-(3-bromo-4-hydroxyphenyl)acetonitrile</i>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> A solution of BBr<sub>3</sub> in DCM (1 M, 6.64 mL) was added dropwise to a cooled (−78 °C) solution of 2-(3-bromo-4-methoxyphenyl)acetonitrile (500 mg, 2.2 mmol) in DCM (2.2 mL). The reaction was allowed to warm to room temperature and stirred for 14 h or until all starting material had been consumed. The reaction was quenched by slowly pouring into ice–water (15 mL). The aqueous phase was extracted with DCM (3 × 10 mL). The organic fractions were combined, washed with water (10 mL) and brine (10 mL), and then dried over anhydrous MgSO<sub>4</sub>. The solvent was evaporated under reduced pressure to yield 2-(3-bromo-4-hydroxyphenyl)acetonitrile as white crystals (0.460 g, 2.17 mmol, 98%), <i>R</i><sub>f</sub> 0.23 (1:2 EtOAc:hexane). LCMS (ESI−) <i>m</i>/<i>z</i> 210.0 [M – H]<sup>−</sup>; retention time 1.76 min (95%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.45 (d, <i>J</i> = 1.8 Hz, 1H), 7.18 (dd, <i>J</i> = 8.2 Hz, 1.9 Hz, 1H), 7.02 (d, <i>J</i> = 8.4 Hz, 1H), 5.57 (s, 1H), 3.67 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 152.4, 131.5, 129.0, 123.4, 117.6, 116.8, 110.7, 22.6 ppm. IR (thin film) <i>v</i><sub>max</sub> 3305 (s, br, O–H), 2924, 2904 (w, C–H), 2272 (w, nitrile CN), 1613 (w), 1511, 1423, 1410 (s, C═C), 1356 (m, C–N), 1279 (s), 1219 (m, C–O), 802 (s), 658 (m) cm<sup>–1</sup>.</div></div><div id="sec4_4_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2-(4-(Benzyloxy)-3-bromophenyl)acetonitrile (<b>15b</b>)</h5><div class="NLM_p last">Potassium carbonate (1.94 g, 14.0 mmol), potassium iodide (1.28 g, 7.70 mmol), and benzyl bromide (1.67 mL, 14.0 mmol) were added to a stirred solution of 2-(3-bromo-4-hydroxyphenyl)acetonitrile (1.48 g, 7.00 mmol) in acetone (65 mL). The suspension was heated to reflux for 15 h, at which point all of the starting material had been consumed. The reaction was concentrated in vacuo, and the resulting residue was diluted with EtOAc (120 mL). The organic fraction was washed with water (3 × 50 mL) and brine (50 mL), then dried over anhydrous MgSO<sub>4</sub>. The solvent was evaporated under reduced pressure to yield a brown oil. The crude product was purified by flash chromatography (0–30% v/v EtOAc in petroleum ether) to yield 2-(4-(benzyloxy)-3-bromophenyl)acetonitrile <b>15b</b> as a pale-yellow oil (1.960 g, 6.49 mmol, 93%), <i>R</i><sub>f</sub> 0.44 (1:2 EtOAc:hexane). LCMS (ESI+) <i>m</i>/<i>z</i> 306, 308 [M + H, reduced nitrile]<sup>+</sup>; retention time 2.60 min (98%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>13</sub>BrNO [M + H]<sup>+</sup> = 302.0181), obsd 302.0166 (δ ppm = 5.0). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.53 (d, <i>J</i> = 2.2 Hz, 1H), 7.45 (m, 2H), 7.39 (app t, <i>J</i> = 7.4 Hz, 6 Hz, 2H), 7.34 (t, <i>J</i> = 7.3 Hz, 1H), 7.20 (dd, <i>J</i> = 8.5 Hz, 2.2 Hz, 1H), 6.92 (d, <i>J</i> = 8.4 Hz, 1H), 5.18 (s, 2H), 3.60 (s, 2H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 154.9, 136.2, 133.0, 128.8, 128.2, 128.1, 127.1, 123.6, 117.7, 114.2, 113.2, 71.0, 22.6 ppm. IR (thin film) <i>v</i><sub>max</sub> 3089, 3064, 2915, 2869 (w, C–H), 2253 (w, nitrile CN), 1604 (m), 1493 (s), 1454, 1413, 1382 (m, C–N), 1282, 1254 (s), 1157 (w), 1051 (s, C–O), 1006 (m), 806 (m), 737, 696 (s, C–Br) cm<sup>–1</sup>.</div></div><div id="sec4_4_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2-(4-(Benzyloxy)-3-bromophenyl)-3-oxopropanenitrile (<b>16b</b>)</h5><div class="NLM_p last">Prepared as an orange residue (334 mg, 1.01 mmol, 51%) according to general procedure C. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to give a mixture of isomers; <i>R</i><sub>f</sub> 0.07 (1:2 v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 330.0 [M – H]<sup>−</sup>, retention time 2.28 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>13</sub>BrNO<sub>2</sub> [M + H]<sup>+</sup> = 330.0130), obsd 330.0140 (δ ppm = 3.0). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.99 (d, <i>J</i> = 2.2 Hz, 1H), 7.54 (m, 2H), 7.46 (m, 5H), 7.43–7.36 (m, 5H), 7.33 (m, 2H), 7.21 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 6.94 (d, <i>J</i> = 9.0 Hz, 1H), 6.91 (d, <i>J</i> = 8.4 Hz, 1H), 5.17 (m, 4H), 3.52 (s, 1H), 3.45 (d, <i>J</i> = 3.2 Hz, 1H) ppm. IR (thin film) <i>v</i><sub>max</sub> 3142 (br), 3034 (w, C–H), 2722 (w, C–H aldehyde), 2216 (w, nitrile, C–N), 1644 (m, C═O), 1599 (w, C═C), 1494 (s), 1379 (w, C–H aldehyde), 1285 (s), 1079, 1003, 808 (m), 731 (s), 694 (s, C–Br) cm<sup>–1</sup>.</div></div><div id="sec4_4_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2-(3-Bromo-4-methoxyphenyl)-3-oxopropanenitrile (<b>16c</b>)</h5><div class="NLM_p last">Prepared as a cream-colored solid (508 mg) according to general procedure C. The crude product was taken forward without further purification; <i>R</i><sub>f</sub> 0.11 (1/3 v/v EtOAc in petroleum ether). LCMS (ESI−) <i>m</i>/<i>z</i> 253.1 [M – H]<sup>−</sup>, retention time 1.96 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>10</sub>H<sub>9</sub>BrNO<sub>2</sub> [M + H]<sup>+</sup> = 253.9817), obsd 253.9829 (δ ppm = 4.7). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.98 (d, <i>J</i> = 2.2 Hz, 1H), 7.77 (s, 1H), 7.66 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.60 (dd, <i>J</i> = 6.3, 2.2 Hz, 1H), 7.56 (s, 1H), 7.49 (s, 1H), 7.38 (dt, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.34 (dt, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.06 (d, <i>J</i> = 8.8 Hz, 3H), 4.73 (d, <i>J</i> = 6.6 Hz, 1H), 3.95–3.80 (m, 10H) ppm. IR (solid) <i>v</i><sub>max</sub> 3243 (m, br), 3063, 2968, 2838 (w, C–H), 2216 (m, CN), 1845 (w), 1654 (m, C═O), 1602 (m), 1499 (s, C–N, C–H), 1438, 1375, 1349 (w, C═C, N–H), 1289 (m), 1256 (s, C–O), 1208 (m), 1056, 1013 (s, C–O), 870 (m), 803 (s, C–H), 686 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> General Procedure D</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Synthesis of Phenyl-1<i>H</i>-pyrazol-5-amines <b>17a</b>–<b>c</b></h4><div id="sec4_5_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 4-(3-Bromophenyl)-1<i>H</i>-pyrazol-5-amine (<b>17a</b>)</h5><div class="NLM_p last">Glacial AcOH (679 μL, 12.9 mmol) followed by hydrazine hydrate (500 μL, 10.3 mmol) was added to a stirred solution of the crude 2-(3-bromophenyl)-3-oxopropanenitrile <b>16a</b> (1.15 g, 5.14 mmol) in EtOH (7 mL) at room temperature. The reaction was heated to 85 °C for 3–5 h, and then the solvent was removed under reduced pressure. The residue was diluted with water (10 mL) and Et<sub>2</sub>O or EtOAc (10 mL), and brought to pH 8.0 with a satd NaHCO<sub>3</sub> solution. The phases were separated, and the aqueous phase was extracted with Et<sub>2</sub>O or EtOAc (3 × 10 mL). The combined organic fractions were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The product was purified by trituration with petroleum ether and DCM to yield 4-(3-bromophenyl)-1<i>H</i>-pyrazol-5-amine <b>17a</b> (865 mg, 3.70 mmol, 67% over two reactions steps from 3-bromophenylacetonitrile) as a pink solid; <i>R</i><sub>f</sub> 0.09 (2:1 v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 240.1 [M (<sup>81</sup>Br) + H]<sup>+</sup>, retention time 1.61 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>9</sub>H<sub>9</sub>BrN<sub>3</sub> [M + H]<sup>+</sup> = 237.9980), obsd 237.9984 (δ ppm = 1.7). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.67 (s, 1H), 7.62 (s, br, 1H), 7.46 (d, <i>J</i> = 7.4 Hz, 1H), 7.32 (d, <i>J</i> = 7.8 Hz, 1H), 7.26 (app t., <i>J</i> = 7.8 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 153.2 (br), 139.6 (br), 137.2, 131.5, 130.1, 129.4, 126.0 (br), 123.8, 108.4 (br) ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3392, 3227 (m, br, N–H), 2900 (w, C–H), 1600, 1567, 1496 (m, C═C, C–N), 1409 (w), 1079 (w), 872 (m), 779 (s), 687 (m) cm<sup>–1</sup>.</div></div><div id="sec4_5_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 4-(4-(Benzyloxy)-3-bromophenyl)-1<i>H</i>-pyrazol-5-amine (<b>17b</b>)</h5><div class="NLM_p last">Prepared as a cream solid (648 mg, 1.88 mmol, 94%) according to general procedure D. The crude product was used without further purification; <i>R</i><sub>f</sub> 0.23 (10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 346.1 [M + H]<sup>+</sup>, retention time 1.94 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>15</sub>N<sub>3</sub>OBr [M + H]<sup>+</sup> = 344.0398), obsd 344.0387 (δ ppm = −3.2). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.69 (d, <i>J</i> = 2.2 Hz, 1H), 7.54 (s, 1H), 7.49 (d, <i>J</i> = 7.5 Hz, 2H), 7.36 (m, 3H), 7.30 (t, <i>J</i> = 7.4 Hz, 1H), 7.09 (d, <i>J</i> = 8.5 Hz, 1H), 5.17 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 154.4, 138.3, 132.3, 129.5, 129.1, 128.9, 128.3, 127.7, 115.6, 113.6, 107.3, 71.9 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3400, 3318 (m, N–H), 3165 (m, br, C–H), 3061, 3036 (m, C–H), 2889 (m, C–H), 1600, 1565, 1500 (m, C═C), 1486 (s, N–H), 1396, 1381, 1367 (w), 1292, 1257 (s, C–N), 1047 (s, C–O), 1013, 872 (m), 802 (s), 726, 704, 641 (m) cm<sup>–1</sup>.</div></div><div id="sec4_5_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 4-(3-Bromo-4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>17c</b>)</h5><div class="NLM_p last">Prepared as a white solid (339 mg, 1.26 mmol, 66% over two reaction steps) according to general procedure D. The crude material was purified by flash chromatography (2–10% v/v MeOH in EtOAc), <i>R</i><sub>f</sub> 0.05 (50% v/v EtOAc in petroleum ether). LCMS (ESI+) <i>m</i>/<i>z</i> 268.1 [M + H]<sup>+</sup>, retention time 1.59 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>10</sub>H<sub>11</sub>BrN<sub>3</sub>O [M + H]<sup>+</sup> = 268.0080), obsd 268.0071 (δ ppm = 3.4) <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.05 (br, s), 7.72 (d, <i>J</i> = 2.1 Hz, 1H), 7.63 (s, 1H), 7.49 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.07 (d, <i>J</i> = 8.5 Hz, 1H), 4.38 (br, s, 2H), 3.88 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, Acetone-<i>d</i><sub>6</sub>) δ 153.6, 150.0 (br), 130.6, 128.4, 126.3, 124.3, 112.6, 112.5, 111.4, 105.5 (br), 55.7 ppm. One <sup>13</sup>C signal unresolved. <sup>13</sup>C NMR processed with LB = 3.0 Hz. IR (solid) <i>v</i><sub>max</sub> 3407, 3333, 3225 (m, br, N–H), 2937, 2841 (w, C–H), 1619 (w, C═C), 1574 (m, N–H), 1516 (m), 1482 (s, C═C, C–H), 1383 (w, CH<sub>3</sub>), 1286 (m, C–N/C-O), 1250 (s, C–N, C–O), 1184 (w), 1045, 1015 (m, C–O, C–H), 744 (m, C–H), 680 (s, C–Br) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> General Procedure E</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Suzuki–Miyaura Coupling to Synthesize Biphenyl-1<i>H</i>-pyrazol-5-amines <b>18a</b>,<b>b</b>, <b>18d</b>–<b>i</b> and <b>23a</b>–<b>b</b></h4><div id="sec4_6_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 4-(3′-Nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>18a</b>)</h5><div class="NLM_p last">4-(3-Bromophenyl)-1<i>H</i>-pyrazol-5-amine <b>17a</b> (238 mg, 1 mmol), 3-nitrophenyl boronic acid (150 mg, 1.5 mmol) (1–2 equiv), Na<sub>2</sub>CO<sub>3</sub> (318 mg, 3 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (116 mg, 0.1 mmol) were combined in a microwave reaction vessel in a 1:1 v/v mixture of 1,4-dioxane:water (12 mL). The reaction was heated at 120 °C in a microwave reactor for 1.5–3 h. The reaction was then allowed to cool to room temperature and diluted with DCM (20 mL) and water (5 mL). The phases were separated, and the aqueous phase was extracted with DCM (3 × 3 mL). The organic fractions were combined, washed with brine (3 mL), dried over anhydrous MgSO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield 4-(3′-nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine <b>18a</b> as a yellow amorphous solid (152 mg, 0.50 mmol, 54%), <i>R</i><sub>f</sub> 0.28 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 281.2 [M + H]<sup>+</sup>, retention time 1.82 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>13</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 281.1033), obsd 281.1023 (δ ppm = 3.6). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.46 (app t, <i>J</i> = 2.0, 2.0 Hz, 1H), 8.21 (ddd, <i>J</i> = 8.3, 2.0, 2.0 Hz, 1H), 7.93 (ddd, <i>J</i> = 7.3, 1.5, 1.0 Hz, 1H), 7.70 (s, 1H), 7.62 (app t, <i>J</i> = 8.0, 7.5 Hz, 1H), 7.59 (s, 1H), 7.53–7.48 (m, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 151.2 (br), 148.8, 142.8, 139.6, 134.1, 133.2, 129.9, 129.9, 128.8 (br), 126.9, 125.7, 125.1, 122.3, 122.1, 108.5 (br) ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3182 (m, br, N–H), 2952 (m, C–H), 1606 (m, N–H), 1524 (s, NO<sub>2</sub>), 1470, 1438 (w, C═C), 1345 (s, NO<sub>2</sub>), 1290, 1265 (w, C–N), 1087, 1055, 1026 (w, C–H), 793, 738, 685 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4-(6-(Benzyloxy)-3′-nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>18b</b>)</h5><div class="NLM_p last">Prepared as a yellow–brown foam (76 mg, 0.20 mmol, 70%) according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 387.0 [M + H]<sup>+</sup>, retention time 2.07 min (94%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 387.1457), obsd 387.1456 (δ ppm = 0.3). <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.51 (app t, <i>J</i> = 2.0, 2.0 Hz, 1H), 8.18 (ddd, <i>J</i> = 8.3, 2.0, 1.0 Hz, 1H), 7.97 (ddd, <i>J</i> = 8.0, 1.5, 1.0 Hz, 1H), 7.63 (app t, <i>J</i> = 8.0, 7.0 Hz, 1H, 16-CH), 7.52–7.48 (m, 2H), 7.35 (m, 2H), 7.30–7.26 (m, 2H), 7.24 (d, <i>J</i> = 8.4 Hz, 1H), 5.15 (s, 2H) ppm. One aromatic <sup>1</sup>H unresolved. <sup>13</sup>C NMR (125 MHz, MeOD) δ 155.0, 152.8, 149.3, 141.6, 138.4, 136.9, 130.3, 130.2, 129.9, 129.5 (2C), 129.1, 128.48, 128.4, 128.3, 125.6, 122.6, 115.1, 71.8 ppm. IR (thin film <i>v</i><sub>max</sub> 3415, 3325, 3193 (br, m, N–H), 3033, 2941, 2880 (w, C–H), 1613, 1578 (w, C═C), 1525 (s, NO<sub>2</sub>), 1525 (m), 1380 (s, NO<sub>2</sub>), 1287, 1260 (m, C–N, C–O), 1055, 1021 (w, C–O), 890 (w), 811, 735, 688 (m) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 4-(6-Methoxy-3′-nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>18c</b>)</h5><div class="NLM_p last">Prepared as a yellow amorphous solid (165 mg, 0.53 mmol, 53%) according to general procedure E, except that the reactions was carried out at 120 °C using conventional heating over 14 h. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM), <i>R</i><sub>f</sub> 0.21 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 311.2 [M + H]<sup>+</sup>, retention time 1.83 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>15</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 311.1144), obsd 311.1139 (δ ppm = 1.6). <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.41 (d, <i>J</i> = 2.2 Hz, 1H), 8.17 (d, <i>J</i> = 8.2 Hz, 1H), 7.93 (d, <i>J</i> = 7.9 Hz, 1H), 7.68–7.55 (m, 2H), 7.51 (d, <i>J</i> = 8.6 Hz, 1H), 7.47 (s, 1H), 7.14 (d, <i>J</i> = 8.5 Hz, 1H), 3.84 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 156.0, 152.8, 149.4, 141.7, 136.9, 130.2, 129.9, 129.7, 129.2, 128.8, 127.7, 125.3, 122.6, 113.2, 109.2, 56.2 ppm. <sup>13</sup>C NMR processed with LB = 3.0 Hz. IR (thin film) <i>v</i><sub>max</sub> 3421, 3182 (m, br, N–H), 2952 (m, C–H), 1613, 1577 (w, N–H, C═C), 1525 (s, NO<sub>2</sub>), 1348 (s, NO<sub>2</sub>), 1260 (m, C–O, C–N), 1181 (w, C–H), 1022 (m, C–O), 814, 747, 688 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 4-(4′-Nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>18d</b>)</h5><div class="NLM_p last">Prepared according to general procedure E and crudely purified by elution through a short silica plug (0–5% v/v MeOH in DCM) to remove inorganic byproducts. The product mixture was then carried forward with further purification and characterization performed after reduction of the aromatic nitro group. LCMS (ESI+): <i>m</i>/<i>z</i> 281.4 [M + H]<sup>+</sup>, retention time 3.89 min (52%).</div></div><div id="sec4_6_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 4-(3-(Pyridin-3-yl)phenyl)-1<i>H</i>-pyrazol-5-amine (<b>18e</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>18e</b> (0.030 g, 0.13 mmol, 61%). LCMS (ESI+): <i>m</i>/<i>z</i> 237.3 [M + H]<sup>+</sup>, retention time 1.12 min (100%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.84 (dd, <i>J</i> = 2.4, 0.8 Hz, 1H), 8.58 (dd, <i>J</i> = 4.9, 1.4 Hz, 1H), 7.86 (ddd, <i>J</i> = 7.9, 2.3, 1.7 Hz, 1H), 7.64 (m, 1H), 7.54 (s, 1H), 7.46 (m, 2H), 7.42 (m, 1H), 7.35 (ddd, <i>J</i> = 8.0, 4.8, 0.8 Hz, 1H), 5.65 (br s, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 150.8, 148.5, 148.2, 138.5, 136.6, 134.6, 134.0, 129.7, 129.1, 126.4, 125.5, 124.8, 123.7, 108.2 ppm. IR (neat) <i>v</i><sub>max</sub> 3365 (w, N–H), 3149 (w, br, N–H), 2932 (w, br, C–H), 1608 (m), 1506 (w), 1463 (m), 1400, 1380 (w), 1188, 1123, 1059, 1044, 1027 (w), 843 (m), 774, 708, 695 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 4-(3-(Pyridin-4-yl)phenyl)-1<i>H</i>-pyrazol-5-amine (<b>18f</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>18f</b> (0.014 g, 0.060 mmol, 29%). LCMS (ESI+): <i>m</i>/<i>z</i> 237.2 [M + H]<sup>+</sup> (calcd for C<sub>14</sub>H<sub>12</sub>N<sub>4</sub>, 237.1), retention time 1.04 min (96%). <sup>1</sup>H NMR (500 MHz, MeOD): δ 8.58 (d, <i>J</i> = 1.6 Hz, 1H), 8.57 (d, <i>J</i> = 1.6 Hz, 1H), 7.87 (t, <i>J</i> = 1.6 Hz, 1H), 7.77 (d, <i>J</i> = 1.6 Hz, 1H), 7.76 (d, <i>J</i> = 1.6 Hz, 1H), 7.70 (s, 1H), 7.56–7.61 (m, 2H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 151.3, 151.2, 150.3, 139.3, 135.9, 132.0, 128.5, 126.0, 125.3, 123.4, 108.2 ppm. one unresolved quaternary <sup>13</sup>C. IR (thin film) <i>v</i><sub>max</sub> 3458, 3159 (m, br, N–H), 291 (m, C–H), 1618 (m, C═C), 1594 (s, N–H, C–H)), 1561 (m), 1508, 1402 (s, C–N), 1334, 1223, 1054, 1023 (m), 789, 701 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 4-(3-(1<i>H</i>-Indol-5-yl)phenyl)-1<i>H</i>-pyrazol-5-amine (<b>18g</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>18g</b> (0.029 g, 0.11 mmol, 50%). LCMS (ESI+): <i>m</i>/<i>z</i> 275.3 [M + H]<sup>+</sup>, retention time 1.74 min (94%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd for C<sub>17</sub>H<sub>15</sub>N<sub>4</sub> [M + H]<sup>+</sup> = 275.1297), obsd 275.1297 (δ ppm = 0). <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.82 (app s, 1H), 7.75 (app t, 1H), 7.65 (s, 1H), 7.37–7.49 (m, 5H), 7.25 (d, <i>J</i> = 3.1 Hz, 1H), 6.49 (d, <i>J</i> = 3.2 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 151.2, 144.8, 137.3, 134.8, 133.7, 131.5, 130.1, 130.0, 126.4, 126.3, 125.6, 125.3, 122.0, 119.7, 112.4, 109.2, 102.8 ppm. IR (thin film) <i>v</i><sub>max</sub> 3405, 3181 (m, br, N–H), 2936 (m, C–H), 1602 (s), 1504, 1462, 1415, 1307 (m, C═C, N–H), 1016 (s), 879 (m), 787, 763, 728, 667 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 4-(3-(1<i>H</i>-Indol-6-yl)phenyl)-1<i>H</i>-pyrazol-5-amine (<b>18h</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>18h</b> (0.026 g, 0.095 mmol, 45%). LCMS (ESI+): <i>m</i>/<i>z</i> 275.3 [M + H]<sup>+</sup>, retention time 1.82 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>17</sub>H<sub>14</sub>N<sub>4</sub> [M + H]<sup>+</sup> = 275.1297), obsd 275.1288 (δ ppm = 3.3). <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.76 (app t, <i>J</i> = 1.8 Hz, 1H), 7.65–7.66 (m, 2H), 7.60 (dd, <i>J</i> = 8.3, 0.6 Hz, 1H), 7.49–7.33 (dt, <i>J</i> = 7.0, 1.8 Hz; 1H), 7.44 (m, 2H), 7.33 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 7.25 (d, <i>J</i> = 3.2 Hz, 1H), 6.45 (dd, <i>J</i> = 3.2, 0.9 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD): δ 151.1, 144.5, 138.3, 135.9, 134.8, 131.6, 130.2, 128.9, 126.4, 126.3, 125.7, 125.6, 121.4, 119.8, 110.6, 109.2, 102.2 ppm. IR (thin film) <i>v</i><sub>max</sub> 3384 (s, N–H), 3175 (m, br, N–H), 1602 (s, N–H, C═C), 1567, 1455 (s, C═C), 1350, 1310 (m, C–N), 1227, 1198 (w), 1096, 1055, 1015 (m, C–N), 863 (m), 785, 722 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 4-(3-(1<i>H</i>-Pyrazol-4-yl)phenyl)-1<i>H</i>-pyrazol-5-amine (<b>18i</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>18i</b> (0.012 g, 0.05 mmol, 37%). LCMS (ESI+): <i>m</i>/<i>z</i> 226.2 [M + H]<sup>+</sup>, retention time 1.30 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd for C<sub>12</sub>H<sub>12</sub>N<sub>5</sub> [M + H]<sup>+</sup> = 226.1093), obsd 226.1104 (δ ppm = 4.9). <sup>1</sup>H NMR (400 MHz, MeOD): δ 7.99 (s, 2H), 7.69 (s, 1H), 7.65 (s, 1H), 7.42 (dt, <i>J</i> = 7.0, 1.9 Hz, 1H), 7.39–7.30 (m, 2H) ppm. <sup>13</sup>C NMR (100 MHz, MeOD): δ 151.1, 135.1, 134.5, 131.6, 130.3, 125.5, 124.6, 123.9, 123.7, 108.9 ppm. One <sup>13</sup>C signal unresolved. IR (neat) <i>v</i><sub>max</sub> 3131 (w, br, N–H), 2917 (w, C–H), 2547, 2358, 2247 (m, br), 1607, 1588 (m, N–H, C═C), 1503 (s), 1437, 1373, 1299 (w, C–N), 1034, 903 (m), 860, 787, 694, 667 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 3-(4-Methoxyphenyl)-4-(3′-nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>23a</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) and then triturated with petroleum ether and DCM to yield <b>23a</b> as a cream-colored solid (434 mg, 1.12 mmol, 37%). LCMS (ESI+) <i>m</i>/<i>z</i> 387.3 [M + H]<sup>+</sup>, retention time 2.08 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 387.1452), obsd 387.1443 (δ ppm = 2.2). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.30 (app t, <i>J</i> = 2.0, 2.0 Hz, 1H), 8.17 (ddd, <i>J</i> = 8.2, 1.3, 1.0 Hz, 1H), 7.81 (ddd, <i>J</i> = 7.7, 1.6, 1.1 Hz, 1H), 7.57 (app t, <i>J</i> = 8.0, 8.0 Hz, 1H), 7.49 (m, 3H), 7.38 (app dt, <i>J</i> = 6.9, 2.0, 1.7 Hz, 1H), 7.30 (d, <i>J</i> = 8.9 Hz, 2H), 6.89 (d, <i>J</i> = 8.9 Hz, 2H), 3.82 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 160.0, 153.0, 148.7, 142.6, 141.4, 139.2, 133.9, 133.0, 129.7, 129.3, 128.9, 128.2, 125.2, 122.3, 122.1, 122.0, 114.4, 105.3, 55.3 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3500–3194 (w, br, N–H), 3056, 2970, 2835 (w, C–H), 1605 (m, N–H), 1524, 1506 (s, NO<sub>2</sub>), 1436 (m, C═C), 1347 (s, NO<sub>2</sub>, C–N), 1246 (s, C–N, C–O), 1176 (s), 1119 (m), 1029 (m, C–O), 835, 795 (m C–H), 735, 721 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_6_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 4-(6-(Benzyloxy)-3′-nitro-[1,1′-biphenyl]-3-yl)-3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>23b</b>)</h5><div class="NLM_p last">Prepared according to general procedure E. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>23b</b> as a brown amorphous solid (459 mg, 0.93 mmol, 46%). LCMS (ESI+) <i>m</i>/<i>z</i> 493.3 [M + H]<sup>+</sup>, retention time 2.28 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>29</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> = 493.1876), obsd 493.1890 (δ ppm = 2.8). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.29 (app t, <i>J</i> = 2.0, 1.9 Hz, 1H), 8.05 (ddd, <i>J</i> = 8.3, 2.3, 1.0 Hz, 1H), 7.70 (dt, <i>J</i> = 7.8, 1.5, 1.2 Hz, 1H), 7.41 (app t, <i>J</i> = 8.0, 8.0 Hz, 1H), 7.35 (m, 4H), 7.33–7.29 (m, 4H), 7.28 (m, 1H), 7.23 (m, 1H), 7.21 (m, 3H), 7.19 (s, 1H), 7.00 (d, <i>J</i> = 8.5 Hz, 1H), 6.81 (d, <i>J</i> = 8.8 Hz, 1H), 5.13 (s, 2H), 3.83 (s, 3H) ppm. <sup>13</sup>C NMR (125 Hz, CDCl<sub>3</sub>) δ 159.9, 154.2, 153.2, 148.0, 141.1, 139.7, 136.5, 135.6, 131.8, 130.7, 128.9, 128.8, 128.7, 128.6, 128.0, 127.1, 126.0, 124.7, 122.4, 121.8, 114.3, 113.5, 104.9, 70.6, 55.3 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3422, 3337, 3202 (w, br, N–H), 2955, 2932, 2853 (w, C–H), 1614 (m), 1523 (s, NO<sub>2</sub>), 1348 (s, NO<sub>2</sub>), 1250 (s), 1178, 1028 (m, C–O), 835, 734, 690, 629 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> General Procedure F</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Reduction of Nitrophenyl Analogues <b>19a</b>–<b>d</b>, <b>24a</b>,<b>b</b></h4><div id="sec4_7_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 4-(3′-Amino-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>19a</b>)</h5><div class="NLM_p last">Tin(II) chloride dihydrate (564 mg, 2.5 mmol) was added to a solution of 4-(3′-nitro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine <b>18a</b> (140 mg, 0.5 mmol) in absolute EtOH (50 mL). The reaction was heated to 85 °C for 1.5–16 h until all starting material had been consumed. The reaction was then concentrated under reduced pressure (∼10 mL) and then diluted with EtOAc (50 mL), brine (50 mL), and saturated sodium potassium tartrate solution (40 mL). The product was extracted into EtOAc (2 × 20 mL), the combined organic fractions were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude material was purified by flash chromatography (0–10% v/v MeOH in DCM) to give 4-(3′-amino-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine <b>19a</b> as a yellow amorphous solid (63 mg, 0.25 mmol, 52%), <i>R</i><sub>f</sub> 021 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 251.2 [M + H]<sup>+</sup>, retention time 1.21 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>15</sub>N<sub>4</sub> [M + H]<sup>+</sup> = 251.1291), obsd 251.1284 (δ ppm = 2.9). <sup>1</sup>H NMR (500 MHz, MeOD<sub>3</sub>) δ 7.68 (d, <i>J</i> = 1.5 Hz, 1H), 7.63 (s, 1H), 7.42 (m, 3H), 7.17 (app t, <i>J</i> = 8.0, 7.5 Hz, 1H), 7.02 (app t, <i>J</i> = 2.0, 2.0 Hz, 1H), 6.98 (appdt, <i>J</i> = 7.5, 1.5, 1.0 Hz, 1H), 6.72 (ddd, <i>J</i> = 8.0, 2.3, 1.0 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 151.3 (br), 149.0, 143.5, 143.4, 135.0, 131.5 (br), 130.5, 130.1, 126.3, 126.1, 125.3, 118.2, 115.7, 115.2, 108.9 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3341, 3206 (m, br, NH<sub>2</sub>), 2972, 2940 (w, C–H), 1602 (s, N–H), 1506, 1475 (m, C═C, N–H), 1314 (w, C–N), 1213 (w, C–N, C–H), 1165, 1052 (w, C–H), 782, 700 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_7_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 4-(3′-Amino-6-(benzyloxy)-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>19b</b>)</h5><div class="NLM_p last">Prepared according to general procedure F. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield <b>19b</b> as a cream-colored solid (99 mg, 0.28 mmol, 58%), <i>R</i><sub>f</sub> 0.33 (10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 357.2 [M + H]<sup>+</sup>, retention time 1.64 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>21</sub>N<sub>4</sub>O [M + H]<sup>+</sup> = 357.1715), obsd 357.1724 (δ ppm = 2.5). <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.99 (br, s, 2H), 7.62 (s, 1H), 7.47 (d, <i>J</i> = 2.4 Hz, 1H), 7.45–7.39 (m, 3H), 7.33 (app t, <i>J</i> = 7.6 Hz, 2H), 7.27 (t, <i>J</i> = 7.3 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 7.08 (app t, <i>J</i> = 7.8 Hz, 1H), 6.95 (app t, <i>J</i> = 2.0 Hz, 1H), 6.86 (d, <i>J</i> = 7.6 Hz, 1H), 6.63 (dd, <i>J</i> = 7.7, 2.2 Hz, 1H), 5.12 (s, 2H), 4.58 (br, s, 2H), 4.32 (br, s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, <i>d</i><sub>6</sub>-acetone) δ 154.7, 151.3 (br), 149.2, 140.8, 139.1, 133.5, 129.8, 129.6, 129.5, 128.7, 128.6, 128.3, 127.2, 125.4 (br), 119.6, 117.1, 115.2, 114.2, 108.0 (br), 71.5 ppm. <sup>13</sup>C NMR spectrum processed with LB = 1–10 Hz. IR (thin film) <i>v</i><sub>max</sub> 3334, 3196 (m, br, N–H), 3033, 2944 (m, C–H), 1602, 1584 (s, C═C), 1498, 1481, 1452 (m, C═C), 1379 (s, C–N), 1280 (s, C–N), 1145 (w), 1021 (m, C–O), 871 (w), 787, 734 (m), 696, 655 (s) cm<sup>–1</sup></div></div><div id="sec4_7_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 4-(3′-Amino-6-methoxy-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>19c</b>)</h5><div class="NLM_p last">Prepared according to general procedure F. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield <b>19b</b> as a white amorphous solid (103 mg, 0.38 mmol, 77%), <i>R</i><sub>f</sub> 0.34 (10% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 282.3 [M + H]<sup>+</sup>, retention time 1.30 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>16</sub>H<sub>17</sub>N<sub>4</sub>O [M + H]<sup>+</sup> = 281.1397), obsd 281.1388 (δ ppm = 3.3). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.53 (s, 1H), 7.39 (dd, <i>J</i> = 8.4, 2.3 Hz, 1H), 7.36 (d, <i>J</i> = 2.4 Hz, 1H), 7.12 (app t, <i>J</i> = 7.8 Hz, 1H), 7.06 (d, <i>J</i> = 8.5 Hz, 1H), 6.91 (app t, <i>J</i> = 2.0 Hz, 1H), 6.86 (d appt, <i>J</i> = 7.6, 1.3 Hz, 1H), 6.69 (ddd, <i>J</i> = 7.9, 2.4, 1.0 Hz, 1H), 3.78 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 156.2, 152.7, 148.1, 140.8, 132.9, 130.1, 129.5, 128.6 (br), 127.7, 127.2, 121.0, 118.1, 115.4, 113.2, 109.4, 56.2 ppm. IR (thin film) <i>v</i><sub>max</sub> 3340, 3210 (m, br, N–H), 2953 (m, C–H), 1603, 1584 (s, N–H), 1500 (m, N–H, C═C), 1482 (s, N–H), 1454 (m, C═C), 1282 (m, C–N), 1246 (s, C–N, C–O), 1380 (w, CH<sub>3</sub>), 1055, 1021 (m, C–O), 873 (w, C–H), 814, 798, 700 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_7_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 4-(4′-Amino-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-5-amine (<b>19d</b>)</h5><div class="NLM_p last">Prepared according to general procedure F. The crude product was purified by flash chromatography (0–8% v/v MeOH in DCM) to yield <b>19d</b> (0.005 g, 0.020 mmol, 10%, over two steps). LCMS (ESI+): <i>m</i>/<i>z</i> 251.2 [M + H]<sup>+</sup>; retention time 1.30 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>15</sub>N<sub>4</sub> [M + H]<sup>+</sup> = 251.1297), obsd 251.1300 (δ ppm = 1.2). <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.66–7.59 (m, 2H), 7.42 (d, <i>J</i> 8.6 Hz, 1H), 7.39–7.32 (m, 3H), 6.79 (d, <i>J</i> = 8.5 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD): δ 148.4, 143.2, 134.9, 132.2, 130.1, 128.7, 125.31, 125.28, 124.6, 116.8, 109.1 (br) ppm. Two <sup>13</sup>C signals unresolved, <sup>13</sup>C NMR processed with LB = 1–12 Hz.</div></div><div id="sec4_7_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 4-(3′-Amino-[1,1′-biphenyl]-3-yl)-3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>24a</b>)</h5><div class="NLM_p last">Prepared according to general procedure F. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>24a</b> as a brown amorphous solid (65 mg, 0.18 mmol, 22%). Impure fractions (157 mg), which contained approximately 90% product, were also retained for use in subsequent synthetic steps, with easier chromatographic purification achieved after deprotection of the aryl methyl ether; <i>R</i><sub>f</sub> 0.32 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 357.2 [M + H]<sup>+</sup>, retention time 1.60 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>21</sub>N<sub>4</sub>O [M + H]<sup>+</sup> = 357.1710), obsd 357.1698 (δ ppm = 3.5). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.43 (m, 2H), 7.37 (dd, <i>J</i> = 7.8, 7.5 Hz, 1H), 7.29 (d, <i>J</i> = 8.8 Hz, 2H), 7.20 (d, <i>J</i> = 7.6 Hz, 1H), 7.10 (app t, <i>J</i> = 7.8 Hz, 1H), 6.87 (m, 3H), 6.81 (d, <i>J</i> = 7.4 Hz, 1H), 6.68 (ddd, <i>J</i> = 7.9, 2.3, 1.0 Hz, 1H), 3.77 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 161.2, 154.4 (br), 149.0, 143.3, 143.2, 142.1 (br), 134.8, 130.5, 130.2, 130.1, 129.4, 129.3, 126.0, 123.8 (br), 118.1, 115.7, 115.1 (2C), 106.7 (br), 55.8 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3341 (br, m, N–H), 3207 (br, m, N–H), 2833 (w, C–H), 1601 (s, N–H, C═C), 1520 (s, C═C, N–H), 1506 (s, C═C, N–H), 1436 (m, C═C), 1290 (w, C–N), 1247 (s, C–O), 1178 (m, C–H), 1113 (m, C–H), 1029 (m, C–O), 972 (m), 834 (m, C–H), 783 (m, C–H), 701 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_7_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 4-(3′-Amino-6-(benzyloxy)-[1,1′-biphenyl]-3-yl)-3-(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>24b</b>)</h5><div class="NLM_p last">Prepared according to general procedure F. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>24b</b> as a brown amorphous solid (370 mg, 0.80 mmol, 65%), <i>R</i><sub>f</sub> 0.16 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 463.3 [M + H]<sup>+</sup>, retention time 1.95 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>29</sub>H<sub>27</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 463.2134), obsd 463.2141 (δ ppm = 1.5). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.33–7.27 (m, 6H), 7.23 (app t, <i>J</i> = 7.1 Hz, 1H), 7.13 (m, 2H), 7.08–7.04 (m, 2H), 6.89–6.82 (m, 3H), 6.80 (dt, <i>J</i> = 7.7, 1.2 Hz, 1H), 6.66 (ddd, <i>J</i> = 7.9, 2.3, 1.0 Hz, 1H), 5.03 (s, 2H), 3.75 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 161.1, 155.6, 154.4, 148.0, 141.8, 140.6, 138.8, 133.4, 133.2, 130.6, 130.1, 129.5, 129.4, 128.6, 128.2, 127.2, 123.9, 121.1, 118.1, 115.4, 115.3, 115.0, 106.3, 71.7, 55.8 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3349, 3211 (br, w, N–H), 2929, 2833 (br, w, C–H), 1603 (s), 1585, 1507 (s, C═C, C–H), 1453 (s), 1244, 1177 (s, C–N, C–O), 1024 (s, C–O), 833, 735, 696 (s, C–H) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> General Procedure G</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Reductive Amination Procedure (1) for the Synthesis of Ar3 Analogues <b>26a</b>–<b>e</b></h4><div id="sec4_8_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 4,4′-(5-((4-Hydroxybenzyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26a</b>)</h5><div class="NLM_p last">4-Hydroxybenzaldehyde (83 mg, 0.68 mmol) and glacial acetic acid (72 μL, 1.4 mmol) were added to a stirred solution of 4,4′-(5-amino-1<i>H</i>-pyrazole-3,4-diyl)diphenol <b>5</b> (200 mg, 0.75 mmol) in DMF (0.5 mL) and DCE (7 mL). The reaction was stirred at room temperature for 20–60 min then cooled to 0 °C, and Na(OAc)<sub>3</sub>BH (360 mg, 1.7 mmol) was added as a single portion. The reaction was allowed to warm to room temperature and stirred for 20 h. For some substrates, an additional equivalent of Na(OAc)<sub>3</sub>BH was then added and the reaction was stirred for a further 7 h at room temperature until all aldehyde had been consumed. The reaction was then diluted with EtOAc (50 mL) and water (10 mL) and brought to pH 7–8 using saturated NaHCO<sub>3</sub> solution. The phases were separated, and the aqueous fraction was extracted with EtOAc (2 × 5 mL). The combined organic fractions were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed under reduced pressure. The crude material was purified by flash chromatography (50–100% EtOAc in petroleum ether) to yield 4,4′-(5-((4-hydroxybenzyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol <b>26a</b> as a cream-colored powder (184 mg, 0.49 mmol, 72%), <i>R</i><sub>f</sub> 0.13 (10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 374.2 [M + H]<sup>+</sup>, retention time 1.25 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>20</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 374.1505), obsd 374.1495 (δ ppm = 2.7). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.19–7.11 (m, 4H), 7.01 (d, <i>J</i> = 8.5 Hz, 2H), 6.76 (d, <i>J</i> = 8.5 Hz, 2H), 6.70 (m, 4H), 4.24 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.6, 157.5, 157.3, 156.1 (br), 142.3 (br), 132.3 (2 × CH), 129.9, 125.3, 123.1 (br), 116.7, 116.2, 116.1, 105.7 (br), 49.9 ppm. One quaternary <sup>13</sup>C signal obscured. <sup>13</sup>C NMR processed with LB = 1–9 Hz. IR (thin film) <i>v</i><sub>max</sub> 3270 (s, br, O–H, N–H), 2940 (w, C–H), 1612, 1596 (m, C═C), 1511 (s, N–H), 1441 (m, C═C), 1370 (m), 1234 (s, C–O, C–N), 1171, 1102, 1011 (m, C–O), 834 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_8_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 4,4′-(5-((3-Bromo-4-hydroxybenzyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26b</b>)</h5><div class="NLM_p last">Prepared according to general procedure G. The crude product was purified by flash chromatography (50–100% v/v EtOAc in petroleum ether) to yield <b>26b</b> as a yellow solid (229 mg, 0.51 mmol, 74%), <i>R</i><sub>f</sub> 0.11 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 454.2 [M + H]<sup>+</sup>, retention time 1.59 min (92%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>19</sub>BrN<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 452.0610), obsd 452.0603 (δ ppm = 1.5). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.41 (d, <i>J</i> = 2.1 Hz, 1H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.10 (dd, <i>J</i> = 1.9, 8.4 Hz, 1H), 7.03 (d, <i>J</i> = 8.7 Hz, 2H), 6.80 (d, <i>J</i> = 8.4 Hz, 1H), 6.78 (d, <i>J</i> = 8.6 Hz, 2H, 6.69 (d, <i>J</i> = 8.7 Hz, 2H), 4.24 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.6, 157.3, 155.5, 154.1, 142.6, 134.5, 133.2, 132.3, 129.9, 128.9, 125.2, 123.1, 117.0, 116.7, 116.2, 110.6, 105.8, 48.3 ppm. <sup>13</sup>C NMR processed with LB = 12.0 Hz. IR (thin film) <i>v</i><sub>max</sub>) 3230 (br, s, N–H, O–H), 2928 (m, C–H), 1651, 1612, 1590 (m, C═C), 1527, 1494, 1432 (s, N–H), 1383, 1343 (m), 1255, 1233 (s, C–N, C–N), 1171 (m), 1105, 1010 (m, C–O), 835 (s, C–H), 666, 527 (m, C–Br) cm<sup>–1</sup>.</div></div><div id="sec4_8_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 4,4′-(5-((Thiophen-3-ylmethyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26c</b>)</h5><div class="NLM_p last">Prepared according to general procedure G. The crude product was purified by flash chromatography (50–90% v/v EtOAc in petroleum ether) to yield <b>26c</b> as a white solid (156 mg, 0.43 mmol, 63%), <i>R</i><sub>f</sub> 0.33 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 364.2 [M + H]<sup>+</sup>, retention time 1.71 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>20</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> = 364.1120), obsd 364.1107 (δ ppm = 3.6). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>DMSO-<i>d</i><sub>6</sub>) δ 11.71 (s, br, 1H, 1-NH), 9.50 (s, br, 1H), 9.30 (s, br, 1H), 7.45 (dd, <i>J</i> = 4.8, 2.8 Hz, 1H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.10 (m, 3H), 6.98 (d, <i>J</i> = 8.4 Hz, 2H), 6.72 (d, <i>J</i> = 8.4 Hz, 2H), 6.68 (d, <i>J</i> = 8.4 Hz, 2H), 4.49 (s, br, 1H), 4.27 (d, <i>J</i> = 4.27 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ157.0, 155.6, 154.1, 142.7, 139.1, 130.7, 128.3, 128.0, 125.5, 123.9, 121.3, 115.6, 115.2, 103.5, 42.8 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3306 (s, br, O–H, N–H), 3101, 2928, 2855 (w, C–H), 1613, 1591 (m, C═C), 1529, 1512 (s, C–N, C–O), 1436 (C═C),1375 (m), 1262, 1239 (s, C–N, C–O), 1172 (m, C–O), 836 (s, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_8_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> 4,4′-(5-((Furan-2-ylmethyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26d</b>)</h5><div class="NLM_p last">Prepared according to general procedure G, however no AcOH was added to the reaction and no NaHCO<sub>3</sub> was used during reaction workup. The crude product was purified by flash chromatography (50–90% v/v EtOAc in petroleum ether) to yield <b>26d</b> as a yellow amorphous solid (82 mg, 0.24 mmol, 38%), <i>R</i><sub>f</sub> 0.54 (10% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 348.2 [M + H]<sup>+</sup>, retention time 1.54 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>20</sub>H<sub>18</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 348.1348), obsd 348.1356 (δ ppm =2.3). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.38 (d, <i>J</i> = 1.8 Hz, 1H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 7.01 (d, <i>J</i> = 8.6 Hz, 2H), 6.78 (d, <i>J</i> = 8.7 Hz, 2H), 6.70 (d, <i>J</i> = 8.8 Hz, 2H), 6.30 (m, 1H), 6.18 (d, <i>J</i> = 3.2 Hz, 1H), 4.33 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.6, 157.3, 155.6 (br), 154.9, 142.8, 142.2 (br), 132.3, 129.9, 125.1, 122.9 (br), 116.7, 116.2, 111.2, 107.6, 106.0 (br), 42.5 ppm, <sup>13</sup>C NMR processed with LB = 6 Hz. IR (thin film) <i>v</i><sub>max</sub> 3290 (s, br, N–H, O–H), 2935, 2816 (w, C–H), 1614 (m, C═C), 1529, 1500 (s, N–H), 1434 (m), 1378, 1343 (w), 1261, 1238 (s, C–N, C–O), 1172 (m, C–O), 1012 (w), 836 (s, C–H), 739 (m, furan, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_8_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 4,4′-(5-(((1<i>H</i>-Indol-5-yl)methyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26e</b>)</h5><div class="NLM_p last">Prepared according to general procedure G. The product was purified by two sequential flash chromatography columns, eluting with 0–5% v/v MeOH in DCM, followed by 50–100% v/v EtOAc in petroleum ether, to yield <b>26e</b> as a brown solid (52 mg, 0.13 mmol, 52%), <i>R</i><sub>f</sub> 0.14 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 397.3 [M + H]<sup>+</sup>, retention time 1.53 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>24</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 397.1665), obsd 397.1653 (δ ppm = 3.0). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.50 (d, <i>J</i> = 0.8 Hz, 1H), 7.31 (d, <i>J</i> = 8.3 Hz, 1H), 7.19 (d, <i>J</i> = 3.2 Hz, 1H), 7.17 (d, <i>J</i> = 8.7 Hz, 2H), 7.09 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 7.01 (d, <i>J</i> = 8.5 Hz, 2H), 6.75 (d, <i>J</i> = 8.6 Hz, 2H), 6.70 (d, <i>J</i> = 8.7 Hz, 2H), 4.41 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.6, 157.3, 156.4 (br), 142.1 (br), 137.0, 132.3, 131.7, 130.0, 129.5, 125.9, 125.3, 123.0 (br), 122.6, 120.4, 116.7, 116.2, 112.1, 105.7 (br), 102.3, 50.5 ppm. <sup>13</sup>C NMR processed with LB = 6 Hz. IR (thin film) <i>v</i><sub>max</sub> 3406 (s, br, N–H, O–H), 2927 (w, C–H), 1613, 1594 (m, C═C), 1529, 1511 (s) 1451, 1435, 1349 (m), 1262, 1240 (s, C–O, C–N), 1172 (m, C–O), 837 (s, C–H), 731 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> General Procedure H</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Reduction Amination Procedure (2) For the Synthesis of Ar3 Analogues <b>27a</b>–<b>b</b> and <b>26f</b></h4><div id="sec4_9_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> 3,4-Bis(4-methoxyphenyl)-<i>N</i>-(pyridin-4-ylmethyl)-1<i>H</i>-pyrazol-5-amine (<b>27a</b>)</h5><div class="NLM_p last">Glacial AcOH (1 mL) was added to a suspension of 3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine <b>13a</b> (222 mg, 0.75 mmol) and pyridine-4-carbaldehyde (71 uL, 0.75 mmol) in dry MeOH (7.5 mL). The reaction was stirred at room temperature for 30–90 min, and then NaCNBH<sub>3</sub> (47 mg, 0.75 mmol) was added as a single portion. The reaction was stirred at room temperature for 14–24 h and then concentrated under reduced pressure. The crude product was redissolved in EtOAc (40 mL) and washed with water (10 mL) that had been made basic with saturated NaHCO<sub>3</sub> solution. The aqueous phase was then extracted with EtOAc (2 × 10 mL), and the combined organic fractions were washed with brine (5 mL). The solvent was removed under reduced pressure to dry the product directly onto silica gel. Flash chromatography (0–5% v/v MeOH in EtOAc) yielded 3,4-bis(4-methoxyphenyl)-<i>N</i>-(pyridin-4-ylmethyl)-1<i>H</i>-pyrazol-5-amine <b>27a</b> as a yellow amorphous solid (226 mg, 0.59 mmol, 78%), <i>R</i><sub>f</sub> 0.13 (5% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 386.6 [M + H]<sup>+</sup>, retention time 1.49 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>23</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 387.1821), obsd 387.1819 (δppm = 0.5). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.53 (d, <i>J</i> = 5.5 Hz, 2H), 7.30 (d, <i>J</i> = 5.7 Hz, 2H), 7.22 (m, 4H), 6.91 (d, <i>J</i> = 8.9 Hz, 2H), 6.83 (d, <i>J</i> = 8.9 Hz, 2H), 4.49 (s, 2H), 3.82 (s, 3H), 3.78 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 159.8, 158.6, 154.8, 149.9, 140.8, 131.0, 128.7, 124.7, 122.5, 122.4, 114.6, 114.3, 104.7, 55.4 (2 ×C), 47.1 ppm. One quaternary <sup>13</sup>C signal unresolved. <sup>13</sup>C NMR processed with LB = 1 Hz. IR (thin film) <i>v</i><sub>max</sub> 3414 (br, w, N–H), 3001, 2934, 2835 (w, C–H), 1604 (w), 1525, 1511 (s), 1461, 1361 (m), 1285 (s, C–N), 1245, 1032 (m, C–O), 834 (s), 801 (m) cm<sup>–1</sup>.</div></div><div id="sec4_9_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>N</i>-((1<i>H</i>-Imidazol-4-yl)methyl)-3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine (<b>27b</b>)</h5><div class="NLM_p last">Prepared according to general procedure H. The crude product was purified by flash chromatography (0–20% v/v MeOH in DCM) to yield <b>27b</b> as a white amorphous solid (484 mg, 1.29 mmol, 95%). LCMS (ESI+) <i>m</i>/<i>z</i> 376.3 [M + H]<sup>+</sup>, retention time 1.61 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>21</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 376.1773), obsd 376.1766 (δ ppm = 2.1). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.75 (br, d, <i>J</i> = 1.5 Hz, 1H), 7.24 (d, <i>J</i> = 8.8 Hz, 2H), 7.12 (d, <i>J</i> = 8.8 Hz, 2H), 7.00 (br, d, <i>J</i> = 0.9 Hz, 1H), 6.89 (d, <i>J</i> = 8.7 Hz, 2H), 6.84 (d, <i>J</i> = 8.8 Hz, 2H), 4.34 (s, 2H), 3.78 (s, 3H), 3.77 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 161.0, 159.9, 155.2, 142.8, 137.2, 136.2, 132.2, 130.0, 126.3, 124.2, 118.2, 115.3, 114.9, 105.6, 55.7, 55.7, 42.0 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3145 (w, br, N–H), 2902, 2835 (w, C–H), 1612, 1590 (w, C═C), 1526, 1511 (s, N–H), 1462 (m, C═C), 1246 (s, C–O, C–N), 1177, 1031 (m, C–O), 834 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_9_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> 4,4′-(5-(((1H-Indazol-4-yl)methyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26f</b>)</h5><div class="NLM_p last">Prepared according to general procedure H except that 4,4′-(5-amino-1<i>H</i>-pyrazole-3,4-diyl)diphenol <b>5</b> was used as the amine component. The crude product was purified by flash chromatography (80–100% v/v EtOAc in petroleum ether, followed by 0–5% v/v MeOH in EtOAc) to yield <b>26f</b> as a white solid (93 mg, 0.24 mmol, 47%), <i>R</i><sub>f</sub> 0.45 (10% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 398.1 [M + H]<sup>+</sup>, retention time 1.48 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>23</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 398.1617), obsd 398.1612 (δ ppm = 1.3). <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.11 (d, <i>J</i> = 1.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.4 Hz, 1H), 7.30 (dd, <i>J</i> = 8.4, 6.9 Hz, 1H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.09 (dd, <i>J</i> = 6.9, 1.0 Hz, 1H), 7.01 (d, <i>J</i> = 8.7 Hz, 2H), 6.75 (d, <i>J</i> = 8.6 Hz, 2H), 6.70 (d, <i>J</i> = 8.7 Hz, 2H), 4.72 (s, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.7, 157.3, 155.7 (br), 142.8 (br), 141.9, 134.9, 133.7, 132.3, 129.9, 127.9, 125.2, 123.1, 123.0 (br), 116.7, 116.2, 109.9, 105.7, 47.6 (br) ppm. <sup>13</sup>C NMR processed with LB = 1–12 Hz. IR (solid) <i>v</i><sub>max</sub> 3432, 3443 (m, N–H, O–H), 2986, 2797, m, C–H), 1612, 1589 (m, C═C), 1528, 1509 (s, C═C, N–H), 1450, 1383, 1262, 1240 (s, C–O, C–N), 1173, 1086, 1003 (m, C–O, C–N), 949 (m), 821 (s, C–H), 777, 739 (m, C–H) cm<sup>–1</sup>.</div></div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Procedures for the Synthesis of Pyrazol-5-yl-benzamide Analogues <b>28a</b>–<b>c</b></h4><div id="sec4_9_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> <i>N</i>-(3,4-Bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-yl)benzamide (<b>28a</b>)</h5><div class="NLM_p last">Benzoyl chloride (31 μL, 0.263 mmol) and triethylamine (42 μL, 0.3 mmol) were added sequentially to a stirred solution of 3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine <b>13a</b> in anhydrous DCM (1.25 mL) at 0 °C. The reaction was stirred at 0 °C for 5 min and then allowed to warm to room temperature and stirred for 3.5 h. The reaction was then diluted with DCM (10 mL), washed with water (3 mL) and brine (3 mL), and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (0–5% v/v MeOH in DCM) to yield <b>28a</b> as a white solid (31 mg, 0.08 mmol, 31%), <i>R</i><sub>f</sub> 0.16 (5% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 400.2 [M + H]<sup>+</sup>, retention time 2.03 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>24</sub>H<sub>22</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 400.1661), obsd 400.1662 (δ ppm = 0.2). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.00 (s, 1H), 9.95 (s, 1H), 7.88 (d, <i>J</i> = 6.3 Hz, 2H), 7.56 (app t, <i>J</i> = 7.5, 6.9 Hz, 1H), 7.47 (appt, <i>J</i> = 7.5, 6.9 Hz, 2H), 7.29 (d, <i>J</i> = 8.4 Hz, 2H), 7.14 (d, <i>J</i> = 8.4 Hz, 2H), 6.96 (d, <i>J</i> = 8.4 Hz, 2H), 6.84 (d, <i>J</i> = 8.4 Hz, 2H), 3.76 (s, 3H), 3.69 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.8, 159.1, 157.9, 144.2, 139.3, 134.0, 131.6, 130.3, 128.6, 128.4, 127.5, 124.6, 122.1, 114.2, 113.8, 113.7 (partially obscured), 55.2, 54.9 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3216 (w, br, N–H), 3004, 2932, 2836 (w, C–H), 1657 (m, C═O), 1602, 1579 (m, C═C), 1509 (s, N–H), 1463 (m, C═C), 1247 (s, C–N), 1178, 1027 (m, C–O), 835 (m, C–H), 7–9 (m) cm<sup>–1</sup>.</div></div><div id="sec4_9_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> <i>N</i>-(3,4-Bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-yl)-4-methoxybenzamide (<b>28b</b>)</h5><div class="NLM_p last">4-Methoxybenzoyl chloride (90 mg, 0.525 mmol) and triethylamine (84 μL, 0.6 mmol) were added sequentially to a stirred solution of 3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine <b>13a</b> in anhydrous DCM (2.5 mL) at 0 °C. The reaction was stirred at 0 °C for 5 min and then allowed to warm to room temperature and stirred for 5 h. The reaction was then diluted with DCM (10 mL), washed with water (5 mL) and brine (5 mL), dried over anhydrous MgSO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (80–100% v/v EtOAc in petroleum ether) to yield <b>28b</b> as an orange solid (58 mg, 0.13 mmol, 27%), <i>R</i><sub>f</sub> 0.49 (5% v/v MeOH in EtOAc). LCMS (ESI+) <i>m</i>/<i>z</i> 430.3 [M + H]<sup>+</sup>, retention time 2.02 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>25</sub>H<sub>24</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> = 430.1767), obsd 430.1778 (δ ppm = 2.6). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.98 (s, 1H), 9.80 (s, 1H), 7.88 (d, <i>J</i> = 8.3 Hz, 2H), 7.29 (d, <i>J</i> = 8.6 Hz, 2H), 7.13 (d, <i>J</i> = 8.8 Hz, 2H), 7.01 (d, <i>J</i> = 8.4 Hz, 2H), 6.95 (d, <i>J</i> = 7.1 Hz, 2H), 6.83 (d, <i>J</i> = 8.2 Hz, 2H), 3.81 (s, 3H), 3.76 (s, 3H), 3.69 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 161.9, 159.1, 157.9, 144.5, 139.3, 130.4, 129.6, 128.7, 126.2, 124.7, 122.2, 114.2, 113.8, 113.7, 55.5, 55.2, 55.0 ppm. One quaternary <sup>13</sup>C peak obscured. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (solid) <i>v</i><sub>max</sub> 3294, 3154 (w, br, N–H), 2938, 2837 (w, C–H), 1738, 1669, 1644, 1605 (w, C═C), 1501 (m, N–H, C–H), 1241, 1177 (s, C–N, C–O), 1019 (m), 961 (w), 834 (s, C–H), 759, 693 (w, C–H) cm<sup>–1</sup></div></div><div id="sec4_9_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> <i>N</i>-(3,4-Bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-yl)isonicotinamide (<b>28c</b>)</h5><div class="NLM_p last">Isonicotinic acid (38 mg, 0.31 mmol) was suspended in anhydrous DCM (2 mL). The solution was cooled to 0 °C, and oxalyl chloride (29 μL, 0.34 mmol) was added along with a catalytic amount of DMF. The reaction was allowed to come to room temperature and stirred for 2 h. The volatiles were removed under reduced pressure, and the resulting residue was resuspended in anhydrous DCM (1.25 mL). The solution was cooled to 0 °C and the aminopyrazole <b>5</b> (74 mg, 0.25 mmol) was added, followed by Et<sub>3</sub>N (122 μL, 0.875 mmol). The reaction was stirred at 0 °C for 1 h, then allowed to warm to room temperature and stirred overnight or until all of the aminopyrazole had been consumed. The reaction was then diluted with DCM (20 mL), washed with water (5 mL), then brine (3 mL), dried over anhydrous MgSO<sub>4</sub>, and the solvent evaporated under reduced pressure. The crude product was purified by silica gel chromatography (0–10% MeOH in DCM) to yield <b>30</b> as a white solid (34 mg, 0.086 mmol, 34%), <i>R</i><sub>f</sub> 0.20 (10% v/v MeOH in DCM). LCMS (ESI+): <i>m</i>/<i>z</i> 401.2 [M + H]<sup>+</sup>, retention time 1.76 min (98%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>23</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 401.1614), obsd 401.1617 (δ ppm = 0.7). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.76 (d, <i>J</i> = 4.8 Hz, 2H), 8.23 (s, 1H), 7.59 (d, <i>J</i> = 4.4 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.20 (d, <i>J</i> = 8.1 Hz, 2H), 6.95 (d, <i>J</i> = 8.2 Hz, 2H), 6.83 (d, <i>J</i> = 8.8 Hz, 2H), 3.84 (s, 3H), 3.79 (s, 3H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 163.7, 159.7, 159.2, 151.0, 145.9 (br), 140.2, 138.6 (br), 131.0, 129.1, 124.2 (br), 123.7, 120.9, 115.0, 114.1, 108.0 (br), 55.4, 55.4 ppm. <sup>13</sup>C NMR processed with LB = 6 Hz. IR (thin film) <i>v</i><sub>max</sub> 3219 (w, br, N–H), 2967, 2836, 2371 (w, C–H) 1663 (m), 1613 (w), 1525, 1508, 1297 (m, C═C), 1248 (s, C–N), 1025 (w, C–O), 835 (m) cm<sup>–1</sup>.</div></div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> General Procedure I</h3><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> Deprotection of Aryl-methyl Ethers to Synthesize Analogues <b>5</b>, <b>14b</b>–<b>c</b>, <b>25a</b>–<b>b</b>, <b>26g</b>–<b>h</b>, and <b>29a</b>–<b>c</b></h4><div id="sec4_10_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> 4,4′-(5-Amino-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>5</b>)</h5><div class="NLM_p last">A solution of 1 M BBr<sub>3</sub> in DCM (21 mL, 21 mmol) was added dropwise over a period of 30 min to a stirred solution of 3,4-bis(4-methoxyphenyl)-1<i>H</i>-pyrazol-5-amine <b>13a</b> (886 mg, 3.0 mmol) in anhydrous DCM (75 mL) at 0 °C. The reaction was stirred at 0 °C for 3.5–24 h, or until all starting material had been consumed. The reaction was quenched with saturated NaHCO<sub>3</sub> at 0 °C, and the resulting precipitate was extracted into EtOAc (50 mL). The organic phase was collected, and the aqueous phase was re-extracted with EtOAc (3 × 25 mL). The organic fractions were combined, washed with brine (5 mL), dried over anhydrous MgSO<sub>4</sub> or Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed under reduced pressure. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield 4,4′-(5-amino-1<i>H</i>-pyrazole-3,4-diyl)diphenol <b>5</b> as a pale-brown solid (795 mg, 2.97 mmol, 99%), <i>R</i><sub>f</sub> 0.27 (10% v/v MeOH in EtOAc). LCMS (ESI+): <i>m</i>/<i>z</i> 269.2 [M + H]<sup>+</sup>; retention time 1.19 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 268.1086), obsd 268.1075 (δ ppm = 4.1). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.64 (br, s, 1H), 9.52 (br, s, 1H), 9.30 (s, 1H), 7.10 (d, <i>J</i> = 8.6 Hz, 2H), 6.97 (d, <i>J</i> = 8.2 Hz, 2H), 6.71 (d, <i>J</i> = 8.4 Hz, 2H), 6.67 (d, <i>J</i> = 8.4 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.8, 155.4, 153.1, 139.0, 130.5, 128.3, 124.2, 121.9, 115.5, 115.2, 103.8 ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3500–2700 (br, O–H, N–H), 2797, 2670, 2591 (w), 1613 (m), 1513 (s), 1488, 1435 (m), 1374 (w), 1261, 1234 (s), 1172 (m), 1101 (w), 1028, 969 (w), 834 (s), 756 (w) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 3-(5-Amino-4-(4-hydroxyphenyl)-1<i>H</i>-pyrazol-3-yl)phenol (<b>14b</b>)</h5><div class="NLM_p last">Prepared according to general procedure I as a pale-brown foam in quantitative yield (157 mg, 0.5 mmol) and used without further purification. LCMS (ESI+) <i>m</i>/<i>z</i> 268.2 [M + H]<sup>+</sup>, retention time 1.25 min (95%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 268.1086), obsd 268.1075 (δ ppm = 4.1). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.11 (app t, <i>J</i> = 7.9, 7.9 Hz, 1H), 7.07 (d, <i>J</i> = 8.6 Hz, 2H), 6.82 (ddd, <i>J</i> = 8.7, 1.2, 1.0 Hz, 1H), 6.79 (m, 3H), 6.71 (ddd, <i>J</i> = 8.2, 2.6, 1.0 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.5, 157.3, 153.0, 143.3, 133.3, 132.0, 130.5, 125.0, 119.9, 116.7, 116.1, 115.5, 106.6 ppm. <sup>13</sup>C NMR spectrum processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3396–3290 (w, br, O–H, N–H), 2923, 2853 (s, C–H), 1714, 1606, 1579 (w), 1455 (m), 1377 (w), 1260 (m, C–N), 1097, 1030 (m, C–O), 874 (w), 802 (m) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> 4-(5-Amino-3-(3-fluorophenyl)-1<i>H</i>-pyrazol-4-yl)phenol (<b>14c</b>)</h5><div class="NLM_p last">Prepared according to general procedure I, except that only 3.5 equiv of 1.0 M BBr<sub>3</sub> in DCM were used. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield <b>14c</b> as a cream-colored amorphous solid (255 mg, 0.95 mmol, 95%), <i>R</i><sub>f</sub> 0.22 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 270.2 [M + H]<sup>+</sup>, retention time 1.56 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>15</sub>H<sub>13</sub>FN<sub>3</sub>O [M + H]<sup>+</sup> = 270.1037), obsd 270.1024 (δ ppm = 1.9). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.28 (m, 1H), 7.15 (d, <i>J</i> = 7.7 Hz, 1H), 7.06 (m, 3H), 6.98 (app t, <i>J</i> = 10.0 Hz, 1H), 6.81 (d, <i>J</i> = 8.6 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 164.1 (<i>J</i><sub>C–F</sub> =  242.8 Hz), 157.6, 154.8 (br), 140.1 (br), 133.8, 132.5, 131.3, 124.6, 124.3, 116.8, 115.6, 115.0 (<i>J</i><sub>C–F</sub> = 22.5 Hz), 107.7 ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3300 (br, m, O–H, N–H), 2962, 2936, 2856, 2794 (w, C–H), 1614, 1578, 1524 (s, C═C), 1488, 1445 (m), 1265, 1236, 1215 (s, C–N), 1171 (m, C–O)), 867, 838, 788, 686 (m) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> 4-(5-Amino-4-(3′-amino-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-3-yl)phenol (<b>25a</b>)</h5><div class="NLM_p last">Prepared according to general procedure I, except that only 3.5 equiv of 1.0 M BBr<sub>3</sub> in DCM were used. The crude product was purified by flash chromatography (2–10% v/v MeOH in DCM) to yield <b>25a</b> as a brown amorphous solid (30 mg, 0.089 mmol, 50%), <i>R</i><sub>f</sub> 0.05 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 343.3 [M + H]<sup>+</sup>, retention time 1.37 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>21</sub>H<sub>19</sub>N<sub>4</sub>O [M + H]<sup>+</sup> = 343.1553), obsd 343.1541 (δ ppm = 3.6). <sup>1</sup>H NMR (500 MHz, MeOD) 7.46–7.40 (m, 2H), 7.38 (app t, <i>J</i> = 7.6 Hz, 1H), 7.22 (d, <i>J</i> = 8.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.6 Hz, 2H), 7.11 (app t, <i>J</i> = 7.7 Hz, 1H), 6.86 (s, 1H), 6.83 (d, <i>J</i> = 7.7 Hz, 1H), 6.75 (d, <i>J</i> = 8.6 Hz, 2H), 6.68 (dd, <i>J</i> = 8.0, 2.2 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.9, 154.2 (br), 149.0, 143.2, 142.4 (br), 134.9, 130.4, 130.3, 130.1, 129.8, 129.43, 129.35, 125.8, 122.7 (br), 118.1, 116.4, 115.7, 115.2, 106.4 ppm. <sup>13</sup>C NMR processed with LB = 6 Hz. IR (thin film) <i>v</i><sub>max</sub> 3341, 3206 (br, m, N–H, O–H), 2984 (m, C–H), 1604 (s, N–H, C═C), 1522, 1505 (C═C), 1470 (m, C═C), 1266, 1242 (C–O), 1173 (w, C–H), 837 (m, C–H), 786 (m, C–H), 701 (m, C–H) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 3′-Amino-5-(5-amino-3-(4-hydroxyphenyl)-1<i>H</i>-pyrazol-4-yl)-[1,1′-biphenyl]-2-ol (<b>25b</b>)</h5><div class="NLM_p last">Prepared according to general procedure I. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield <b>25b</b> as a white solid (25 mg, 0.07 mmol, 46%), <i>R</i><sub>f</sub> 0.07 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 359.2 [M + H]<sup>+</sup>, retention time 1.21 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>21</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 359.1508), obsd 359.1505 (δ ppm = 0.8). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.21 (d, <i>J</i> = 8.7 Hz, 2H), 7.12–7.06 (m, 2H), 7.04 (dd, <i>J</i> = 8.2, 2.2 Hz, 1H), 6.91–6.85 (m, 2H), 6.82 (d. app t, <i>J</i> = 7.6, 1.4 Hz, 1H), 6.75 (d, <i>J</i> = 8.7 Hz, 2H), 6.66 (ddd, <i>J</i> = 7.9, 2.3, 1.0 Hz, 1H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.7, 154.3 (br), 154.0, 148.1, 142.1 (br), 140.8, 133.2, 130.6, 130.1, 129.7, 125.7, 123.6 (br), 120.7, 117.9, 117.4, 116.3, 115.3, 106.0 (br) ppm. One quaternary <sup>13</sup>C signal obscured by peaks 130–129 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3374–3200 (br, s, O–H, N–H), 1612, 1510 (s, C═C), 1442, 1399 (m, C–N), 1268 (s, C–N), 1174 (m, C–O), 1111, 1032 (w, C–O), 836 (s), 700 (w) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 4,4′-(5-((Pyridin-4-ylmethyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26g</b>)</h5><div class="NLM_p last">Prepared according to general procedure I, except that the reaction was allowed to warm to room temperature and stirred for 2 h before being quenched with saturated NaHCO<sub>3</sub> solution. The crude product was purified by flash chromatography (0–10% v/v MeOH in DCM) to yield <b>26g</b> as a white solid (51 mg, 0.14 mmol, 61%), <i>R</i><sub>f</sub> 0.10 (10% v/v MeOH in DCM). LCMS (ESI+) <i>m</i>/<i>z</i> 359.2 [M + H]<sup>+</sup>, retention time 1.65 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>21</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 359.1508), obsd 359.1508 (δppm = 0.0). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.63 (s, 1H), 9.54 (s, br, 1H), 9.32 (s, br, 1H), 8.44 (d, <i>J</i> = 6.0 Hz, 2H), 7.32 (d, <i>J</i> = 5.6 Hz, 2H), 7.08 (d, <i>J</i> = 8.4 Hz, 2H), 7.03 (d, <i>J</i> = 8.4 Hz, 2H), 6.75 (d, <i>J</i> = 8.4 Hz, 2H), 6.67 (d, <i>J</i> = 8.4 Hz, 2H), 5.21 (t, <i>J</i> = 5.6 Hz, 1H), 4.31 (d, <i>J</i> = 6.0 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.7, 155.8, 153.9, 151.3, 149.2, 139.2, 130.8, 128.4, 123.8, 122.5, 121.4, 115.6, 115.3, 103.4, 46.1 ppm. <sup>13</sup>C NMR processed with LB = 2.0 Hz. IR (solid) <i>v</i><sub>max</sub> 3629, 2414, 3210 (w, N–H, O–H), 2905, 2585 (w, C–H), 1602, 1575 (m, C═C), 1537, 1513, 1479 (s, N–H, C–H), 1437, 1418 (m), 1385 (w, C–O), 1246 (s, C–O, C–N), 1157 (m, C–O), 1026, 1006 (m), 835 (s, C–H) 799, 661 (m) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 4,4′-(5-(((1<i>H</i>-Imidazol-4-yl)methyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol (<b>26h</b>)</h5><div class="NLM_p last">Prepared according to general procedure I. The crude product was purified by flash chromatography (1–20% v/v MeOH in DCM) to yield <b>26h</b> as a white solid (118 mg, 0.34 mmol, 68%). LCMS (ESI+) <i>m</i>/<i>z</i> 348.2 [M + H]<sup>+</sup>, retention time 1.26 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>19</sub>H<sub>18</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 348.1461), obsd 348.1468 (δ ppm = 2.0). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.60 (d, <i>J</i> = 1.2 Hz, 1H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.02 (d, <i>J</i> = 8.6 Hz, 2H), 6.94 (d, <i>J</i> = 1.2 Hz, 1H), 6.76 (d, <i>J</i> = 8.6 Hz, 2H), 6.70 (d, <i>J</i> = 8.7 Hz, 2H), 4.31 (s, 2H, 14-CH<sub>2</sub>) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 158.7, 157.3, 155.3 (br), 142.8 (br), 137.3, 136.2, 132.2, 129.9, 125.1, 123.1, 118.2 (br), 116.7, 116.2, 105.8, 41.9 ppm. <sup>13</sup>C NMR processed with LB = 3 Hz. IR (thin film) <i>v</i><sub>max</sub> 3228 (s, br, O–H, N–H), 2929 (m, br, C–H), 2471 (m), 1613, 1589 (m, C═C), 1527, 1511 (s), 1445, 1379 (m, br, C–N), 1260 (s, br, C–N), 1171, 1104 (m, C–O), 974 (m), 836 (s), 739 (w) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> <i>N</i>-(3,4-Bis(4-hydroxyphenyl)-1<i>H</i>-pyrazol-5-yl)benzamide (<b>29a</b>)</h5><div class="NLM_p last">Prepared as a tan powder (43 mg, 0.105 mmol, quant) according to general procedure I. No chromatographic purification was required. LCMS (ESI+) <i>m</i>/<i>z</i> 372.2 [M + H]<sup>+</sup>, retention time 1.58 min (100%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>18</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 372.1348), obsd 372.1344 (δ ppm = 1.1). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.87 (d, <i>J</i> = 7.7 Hz, 2H), 7.56 (t, <i>J</i> = 7.4 Hz, 1H), 7.47 (app t, <i>J</i> = 7.6 Hz, 2H), 7.24 (d, <i>J</i> = 8.6 Hz, 2H), 7.10 (d, <i>J</i> = 8.1 Hz, 2H), 6.76 (d, <i>J</i> = 8.7 Hz, 2H), 6.71 (d, <i>J</i> = 8.6 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 170.8, 159.0, 157.5, 145.1 (br), 142.4 (br), 135.2, 133.1, 131.9, 130.2, 129.6, 128.7, 124.5, 122.5 (br), 116.4, 116.3, 115.7 (br) ppm. <sup>13</sup>C NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3254 (s, br, O–H, N–H), 2973, 2927 (w, C–H), 1656 (m, C═O), 1614 (m, C═C), 1510 (s, br, N–H), 1439 (m, C═C), 1371 (m), 1263 (s, br, C–O, C–N), 1173 (m, C–O), 836 (m, C–H), 710 (w) cm<sup>–1</sup>.</div></div><div id="sec4_10_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> <i>N</i>-(3,4-Bis(4-hydroxyphenyl)-1<i>H</i>-pyrazol-5-yl)-4-hydroxybenzamide (<b>29b</b>)</h5><div class="NLM_p last">Prepared according to general procedure I. The crude product was purified by flash chromatography (1–10% v/v MeOH in DCM) to yield <b>29b</b> as a cream-colored amorphous solid (29 mg, 0.074 mmol, 81%). LCMS (ESI+) <i>m</i>/<i>z</i> 372.2 [M + H]<sup>+</sup>, retention time 1.59 min (100%). HRMS (ESI+) <i>m</i>/<i>z</i> (calcd C<sub>22</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> = 388.1297), obsd 388.1286 (δ ppm = 2.8). <sup>1</sup>H NMR (500 MHz, MeOD) δ 7.76 (d, <i>J</i> = 8.4 Hz, 2H), 7.23 (d, <i>J</i> = 8.8 Hz, 2H), 7.09 (d, <i>J</i> = 8.8 Hz, 2H), 6.83 (d, <i>J</i> = 8.4 Hz, 2H), 6.75 (d, <i>J</i> = 8.4 Hz, 2H), 6.71 (d, <i>J</i> = 8.8 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 170.4, 162.5, 158.9, 157.48, 144.8 (br), 143.0 (br), 131.9, 130.9, 130.2, 125.9, 124.6, 122.8 (br), 116.4, 116.3, 116.1, 115.5 (br) ppm. <sup>13</sup>C-NMR processed with LB = 12 Hz. IR (thin film) <i>v</i><sub>max</sub> 3204 (br, m, O–H, N–H), 1648, 1607, 1587 (m, C═C), 1529 (s), 1441 (m), 1261, 1236, 1172 (s, C–N, C–O), 1022, 1010 (w, C–O), 837 (m), 764 (w) cm<sup>–1</sup></div></div><div id="sec4_10_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> <i>N</i>-(3,4-Bis(4-hydroxyphenyl)-1<i>H</i>-pyrazol-5-yl)isonicotinamide (<b>29c</b>)</h5><div class="NLM_p last">Prepared according to general procedure I, except that 1.0 M BBr<sub>3</sub> in DCM was added at −78 °C and then the reaction was allowed to warm slowly to 0 °C and then stirred overnight at room temperature. The crude product was purified by flash chromatography (1–10% v/v MeOH in EtOAc) to yield <b>29c</b> as an off-white solid (42.3 mg, 0.114 mmol, 87%). LCMS (ESI+) <i>m</i>/<i>z</i> 373.2 [M + H]<sup>+</sup>, retention time 1.30 min (98%). HRMS (ESI+): <i>m</i>/<i>z</i> (calcd C<sub>21</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 373.1301), obsd 373.1306 (δ ppm = 1.3). <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.72 (s, br, 2H), 7.82 (d, <i>J</i> = 4.5 Hz, 2H), 7.23 (d, <i>J</i> = 8.5 Hz, 2H), 7.08 (d, <i>J</i> = 8.5 Hz, 2H), 6.76 (d, <i>J</i> = 8.5 Hz, 2H), 6.72 (d, <i>J</i> = 8.5 Hz, 2H) ppm. <sup>13</sup>C NMR (125 MHz, MeOD) δ 168.3, 159.1, 157.7, 150.9, 143.6 (br), 143.0 (br), 131.9, 130.2, 124.3, 123.2, 122.4 (br), 116.5, 116.4, 115.6 (br) ppm. One quaternary <sup>13</sup>C signal unresolved. <sup>13</sup>C NMR processed with LB = 6 Hz. IR (thin film) <i>v</i><sub>max</sub> 3440–3230 (br, s, O–H, N–H), 1666, 1614 (m), 1526, 1508 (C═C), 1415 (m), 1264 (s, C–N), 1175 (m, C–O), 837 (s), 751, 685 (w) cm<sup>–1</sup></div></div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> Molecular Biology, Biophysical, and Biochemical Methods</h3><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Expression and Purification of CYP121</h4><div class="NLM_p last">The <i>CYP121</i> (<i>Rv2276</i>) gene was expressed using a <i>pHAT2/CYP121</i> expression clone in <i>Escherichia coli</i> C41(DE3) cells as the His<sub>6</sub>-tagged construct, following previously described conditions,<a onclick="showRef(event, 'ref7 ref27'); return false;" href="javascript:void(0);" class="ref ref7 ref27">(7, 27)</a> with minor modifications. Expression was typically performed on a 10 × 1 L scale. C41 (DE3) <i>E. coli</i> cells were transformed with <i>pHAT/CYP121</i> and cultured overnight in LB medium containing ampicillin (100 mg/L). Expression cultures of terrific broth (TB) medium containing ampicillin (100 mg/L) were inoculated using an overnight culture of the transformant cells (0.4% v/v) and grown at 37 °C, 200 rpm until an OD<sub>600</sub> = 0.5–0.6, at which point the temperature was reduced to 23 °C. <i>CYP121</i> gene expression was induced with IPTG (0.15 mM), and heme synthesis was supported with δ-aminolevulinic acid (0.1 mM). Cultures were incubated for 24 h, and then cells were harvested by centrifugation (9000<i>g</i>, 4 °C, 20 min) and stored at −20 °C until purification. Pellets were defrosted and resuspended in chilled 50 mM potassium phosphate buffer (pH 8.0), containing 50 mM KCl, 10%v/v glycerol, and commercial protease inhibitors (cOmplete EDTA-free protease inhibitor cocktail tablets, Roche (1 tablet/50 mL), PMSF (1 mM), and benzamidine hydrochloride (1 mM)) (buffer A). Cells were lysed by sonication at 0 °C (30 s on/60 s off, 12 cycles) using a Sonics Vibra-Cell 505 sonicator (Sonics and Materials Inc., CT, USA). Cell debris was pelleted by centrifugation (40000<i>g</i>, 30 min, 4 °C), and the supernatant was immediately loaded onto a freshly recharged HisTrap FF affinity column (GE Healthcare, Little Chalfont UK). Affinity chromatography was performed using an ÄKTA purifier (GE Healthcare) at 18 °C. Protein and P450 heme absorbance (A<sub>280</sub> and A<sub>417</sub>, respectively) were monitored, and CYP121 was eluted with a linear gradient of imidazole (0–200 mM) in buffer A. The purity of fractions collected was assessed by SDS-PAGE. Pure fractions were combined and dialyzed into 50 mM Tris-HCl (pH 7.2), containing 500 mM KCl and 1 mM EDTA (buffer B) over 24–48 h using a Slide-A-Lyser (ThermoScientific, Hemel Hempstead UK) dialysis cassette. Dialyzed protein was concentrated using a Vivaspin (10000 MWCO) ultrafiltration column at 6000<i>g</i>, 4 °C, and then stored at −80 °C until further use. Impure protein fractions were pooled, dialyzed into buffer A to remove imidazole, and repurified by a second HisTrap FF affinity column, as described above. Protein concentration and purity was determined by SDS-PAGE and amino acid analysis (Protein and Nucleic Acid Chemistry Facility [PNAC], Department of Biochemistry, University of Cambridge). Repeated affinity chromatography steps meant that the previously cited anion exchange and gel filtration chromatography steps were not required to obtain sufficiently pure protein.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> Expression and Purification of Mtb CYP125, CYP126, CYP142, CYP143, and CYP144</h4><div class="NLM_p last">Plasmid vectors encoding N-terminal His<sub>6</sub>-tagged CYP125 (pET15b/<i>Rv3545c</i>) and CYP126 (pET15b/<i>Rv0778</i>) were provided by Dr. Kirsty McLean (Manchester Institute of Biotechnology, University of Manchester). CYP125<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and CYP126<a onclick="showRef(event, 'ref27 ref68'); return false;" href="javascript:void(0);" class="ref ref27 ref68">(27, 68)</a> proteins were expressed and purified in <i>E. coli</i> C41(DE3) cells according to the literature. CYP142,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> CYP143, and CYP144-TRV<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> proteins were prepared according to the literature and provided by Dr. Kirsty McLean (Manchester Institute of Biotechnology, University of Manchester).</div></div><div id="sec4_11_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> Ligand Screening by UV–Visible Spectroscopy</h4><div class="NLM_p">UV–vis screening was performed using a CARY400 UV–vis spectrophotometer (Varian, CA, USA). Ligands were prepared as DMSO-<i>d</i><sub>6</sub> stock solutions (1–100 mM) and added to protein (5 μM) solutions (198 μL) or to buffer alone to give a final DMSO-<i>d</i><sub>6</sub> concentration of 1% v/v. CYP121 was prepared in 50 mM Tris-HCl (pH 7.2), 1 mM EDTA buffer. CYP125, CYP126, CYP142, CYP143, and CYP144 were prepared in 50 mM Tris-HCl (pH 7.5), 100 mM KCl buffer. CYP124 was prepared in 100 mM potassium phosphate (pH 7.0), 100 mM KCl buffer. Spectra were recorded between 800 and 250 nm at 25 °C. Any spectral interference from the inherent absorbance of added ligands/solvent was removed by subtracting spectra collected for protein-free samples of the same ligands/solvent in buffer. The difference in Soret wavelength maximum (Δλ<sub>max</sub>) of the respective P450s Soret band (λ<sub>max</sub>) in the presence of ligands compared to that obtained for a 1% v/v DMSO-<i>d</i><sub>6</sub> control was used to identify type I or type II heme-binding interactions. Changes in the Soret less than ±1 nm were considered within experimental error.</div><div class="NLM_p last">Competition assays were performed to detect the binding interactions of ligands that did not perturb that heme cofactor. A known type II inhibitor, typically clotrimazole (5–50 μM) or econazole (5–50 μM), was added to assay solutions to achieve a final DMSO-<i>d</i><sub>6</sub> concentration of 1% v/v. The difference in the red-shift of the Soret band between competition assays and that found for the known type II ligand in the absence of competing ligands was used to determine competitive binding. Buffer control spectra were collected to account for any inherent absorbance of added ligands/solvent and all solutions were inspected and found to be free of precipitate. All spectra were generated using Origin software (OriginLab, Northampton, MA) and processed using Microsoft Excel (Microsoft Office, 2010) and GraphPad Prism 5.01 (GraphPad Software, San Diego, USA).</div></div><div id="sec4_11_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Optical Titrations</h4><div class="NLM_p">Optical titrations to determine <i>K</i><sub>D</sub> values were carried out a Varian Cary 400 UV–vis spectrophotometer (Varian, CA, USA) according to a previously described procedure.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Assays were performed in reduced volume (200 μL) quartz cuvettes with a path length of 1 cm (Starna, Essex, UK). Ligands were prepared as DMSO-<i>d</i><sub>6</sub> stock solutions, and proteins (5 μM final concentration) were diluted in the corresponding buffer as described in the ligand screening by UV–vis spectroscopy protocol above. Aliquots (0.2–1.0 μL) of ligand stock solutions were added directly to cuvettes containing either protein solutions, or buffer alone, to achieve final concentrations of 100 nM to 1 mM. The final DMSO-<i>d</i><sub>6</sub> concentration did not exceed 1% v/v of the assay solution. Spectra were recorded between 800 and 250 nm at 25 °C after the addition of each aliquot of ligand. Buffer control spectra were subtracted manually from protein spectra during processing or saved as a baseline for automatic subtraction from protein spectra. Difference spectra were generated by subtracting the initial ligand-free protein spectrum from each successive titration spectrum. The maximum change in absorbance for each difference spectrum was then plotted against ligand concentration and fitted using either a one-site binding model hyperbolic eq (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>), or a modified version of the Morrison eq (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>) for tight binding inhibitors.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> All spectral analysis was done using Origin software (OriginLab, Northampton, MA), and data were processed using Microsoft Excel (Microsoft Office, 2010). Plots and curve fitting were performed using GraphPad Prism 5.01 (GraphPad Software, San Diego, USA).<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/jm-2016-00007f_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span><span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/jm-2016-00007f_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p last">In <a class="ref internalNav" href="#eq1" aria-label="eqs 1">eqs 1</a> and <a class="ref internalNav" href="#eq2" aria-label="2">2</a>, <i>A</i><sub>obs</sub> is the observed change in absorbance, <i>A</i><sub>max</sub> is the maximum absorbance change at saturation, <i>E</i> is the total enzyme concentration, and <i>L</i> is the concentration of ligand.</div></div><div id="sec4_11_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> Competition Optical Titrations for Non-heme Binding Ligands</h4><div class="NLM_p">The <i>K</i><sub>D</sub> values of non-heme binding ligands were calculated from the change in the apparent <i>K</i><sub>D</sub> of a known type II ligand, typically 4,4′-(5-(((1<i>H</i>-imidazol-4-yl)methyl)amino)-1<i>H</i>-pyrazole-3,4-diyl)diphenol <b>26h</b>, when it was titrated into proteins (5 μM) in the presence of the non-heme binding ligand (50 μM). Optical titrations were performed and processed as described above, except that the non-heme binding ligand was added initially to the ligand-free protein and buffer control cuvettes. Data were fitted using either <a class="ref internalNav" href="#eq1" aria-label="eqs 1">eqs 1</a> or <a class="ref internalNav" href="#eq2" aria-label="2">2</a> to determine the <i>K</i><sub>D</sub> (apparent) of <b>26h</b>, and then the <i>K</i><sub>D</sub> of competitive ligand was calculated according to <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a><span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/jm-2016-00007f_m003.gif" alt="" /></img><span class="labelSpan">(3)</span></span></div><div class="NLM_p last">In <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>, <i>K</i><sub>D</sub><sup>A</sup>(app) is the apparent binding affinity calculated from the competition titration, <i>K</i><sub>D</sub><sup>A</sup> is the binding affinity of the known inhibitor, <i>K</i><sub>D</sub><sup>B</sup> is the binding affinity of the non-heme binding ligand, and [B] is the concentration of the non-heme binding ligand.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Isothermal Titration Calorimetry</h4><div class="NLM_p last">ITC experiments were performed on a MicroCal iTC200 (Malvern Instruments, Malvern UK) instrument at 25 °C. Ligands were prepared as stock solutions in DMSO-<i>d</i><sub>6</sub>, diluted to 0.5–2.5 mM depending on solubility, and titrated into CYP121 (50 μM). Both ligands and CYP121 were prepared in the same 50 mM Tris-HCl (pH 7.5) buffer, containing 1 mM EDTA and a final concentration of 10% v/v DMSO-<i>d</i><sub>6</sub>. Titrations were comprised of 19 × 2.0 μL injections of ligand solution at 60–90 s intervals. An initial 0.2 μL injection was made and the data discarded during analysis. Control titrations were also performed (ligand into buffer) to measure any heats of dilution or buffer mismatch. Control titrations were subtracted from ligand titrations during data processing. Binding isotherms were integrated to give the enthalpy change of each injection and plotted against the molar ratio of ligand added to the sample cell. Titrations were fitted using a one-site binding model using Origin Analysis Software, setting the stoichiometry to <i>N</i> = 1 for weak binding compounds, or allowing N to vary for potent compounds which showed high goodness of fit.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Native Mass Spectrometry</h4><div class="NLM_p last">The following experimental protocol was followed for ligands <b>6</b>, <b>19a</b>, and <b>25a</b>: Protein stock solutions (10 μM) were prepared by dilution of purified proteins (300–500 μM) in 200 mM ammonium acetate buffer (pH 7.0). Samples were buffer exchanged by size exclusion chromatography using Micro Biospin 6 columns, molecular weight cutoff 6 kDa (BioRad, Hemel Hempstead, UK). Ligands were prepared as stock solutions in DMSO-<i>d</i><sub>6</sub> at 50, 100, and 2500 μM concentrations. Ligand–protein samples were prepared by diluting protein stocks (10 μL) and ligand stocks (1 μL) with ammonium acetate buffer (9 μL) to give final concentrations of 5 μM CYP121, 2.5–125 μM ligand, and 5% v/v DMSO-<i>d</i><sub>6</sub>. An identical protocol was followed for ligands <b>24a</b>, <b>25b</b>, and <b>26a</b>, except that concentrations of CYP121 (8.7 μM) and DMSO-<i>d</i><sub>6</sub> (2.175% v/v) were altered. The concentrations of ligands was adjusted to maintain the same ligand-to-protein ratios. Mass spectra were recorded on a Synapt HDMS instrument (Waters UK Ltd., Manchester, UK). Capillaries for nanoESI– were purchased from ThermoFisher (Hemel Hempstead, UK). Capillary tips were cut under a stereomicroscope to give inner diameters of 1–5 μm and then loaded with 2.5 μL of sample solutions. Given below are the general instrumental conditions used to acquire the reported spectra. However, conditions were optimized over the course of each experiment to preserve the noncovalent protein–ligand complex and minimize nonspecific interactions. All measurements were carried out in a positive ion mode with ion source temperature of 20 °C. A capillary voltage of 1.50–1.75 kV, cone voltage of 60–80 V, and extraction cone voltage of 2.3–4.1 V were applied to perform nanoESI. All reported spectra were collected with a trap collision energy 12–18 V, transfer collision energy 12 V, IMS pressure 5.02 × 10<sup>–1</sup> mbar, and TOF analyzer pressure 1.17 × 10<sup>–6</sup> mbar. External calibration of the spectra was achieved using cesium iodide at 100 mg mL<sup>–1</sup> in water. Data acquisition and processing were performed using Micromass MassLynx v4.1. Mass differences resulting from ligand binding were calculated relative to the relevant 5% v/v DMSO-<i>d</i><sub>6</sub> spectra of CYP121 (5 μM). Mass differences were divided by the molecular weight of the ligand to calculate binding stoichiometry. The percentage of monomeric CYP121 was calculated from the sum of the intensities of peaks corresponding to monomeric CYP121 (<i>m</i>/<i>z</i> 3500–4750) divided by the sum of peak intensities corresponding to both monomeric and dimeric (<i>m</i>/<i>z</i> 5250–6750) CYP121.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> X-ray Crystallography</h4><div class="NLM_p last">Untagged CYP121 protein and crystals were prepared as described previously with the following adaptations.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Crystals were prepared by vapor diffusion in 1.5–2.1 M ammonium sulfate and 0.1 M sodium MES or Cacodylate from pH 5.5–6.15. Additional crystallization was also conducted on the nanolitre scale using a Mosquito pipetting robot (Molecular Dimensions, Newmarket, UK), and crystals were produced in the Morpheus crystallization screen (Molecular Dimensions) in 800 nL drops with protein-to-mother liquor at a ratio of 1:1. Ligand soaks were carried out either by directly dissolving solid ligand at saturation or by the addition of 2–5 mM ligand solution in DMSO to the mother liquor and soaking was carried out for 24 h. Crystals were cryoprotected and frozen as described previously except from Morpheus conditions that were frozen directly. Data were collected on beamline I24 (wavelength 0.9173/1.0118 Å) and I02 (wavelength 0.9795 Å) at the Diamond Light Source Facility (Oxfordshire, UK). Data were processed and refined as previously described. Data tables and statistics are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information. CYP121–ligand cocrystal structures are deposited in the Protein Data Bank (<a href="http://www.rcsb.org/pdb/" class="extLink">http://www.rcsb.org/pdb/</a>) under the accession codes: <b>6</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>, <b>7</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>, <b>19a</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF">5IBF</a>, <b>24a</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBD">5IBD</a>, <b>25a</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE">5IBE</a>, <b>25b</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBG">5IBG</a>, <b>26a</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI">5IBI</a>, and <b>26h</b>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH">5IBH</a>. Images for presentation were rendered using an academic version of the PyMOL Molecular Graphics System, version 1.3, 2010, Schrödinger, LLC.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Human P450 Inhibition</h4><div class="NLM_p last">Experimental determination of IC<sub>50</sub> values for the inhibition of human P450s by <b>19a</b>, <b>25a</b>, and <b>26a</b> were performed by Cyprotex (Cheshire, UK). Inhibition assays of CYP1A, CYP2C9, CYP2C19, CYP2D6, and CYP3A4 were performed in human liver microsomes in quadruplicate, over a 7-point ligand concentration range (0–25 μM). The formation of metabolites from the turnover of isoform-specific substrates (CYP2C9, tolbutamide; CYP2C19, mephenytoin; CYP2D6, dextromethorphan; and CYP3A4, midazolam) was monitored by LC/MS. Turnover of the CYP1A substrate ethoxyresorufin was monitored by fluorescence. Activities reported for CYP1A refer to the combined inhibition of CYP1A1 and CYP1A2 isoforms. IC<sub>50</sub> values were calculated relative to the vehicle controls. IC<sub>50</sub> values for known inhibitors (α-naphthoflavone, sulfaphenazole, tranylcypromine, quinidine, ketoconazole) were determined as a positive control and were found to be within the expected range (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S3</a>, Supporting Information).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Antimycobacterial Activity Assay</h4><div class="NLM_p last">Three <i>M. tuberculosis</i> (Mtb) H37Rv cultures were prepared in 7H9 liquid medium supplemented with 10% OADC enrichment (Sigma-Aldrich, Poole UK). Cultures were incubated at 37 °C until the OD<sub>600</sub> reached 1.00. After this period, the bacterial suspensions were washed twice with fresh 7H9 medium and then used in the antimycobacterial activity assays. The minimum inhibitory concentration (MIC<sub>90</sub>) of the compounds was measure in triplicate in a 24 well plate-based format. Plates were prepared with 2 mL of 7H10 solid medium supplemented with 10% OADC enrichment (Sigma-Aldrich) per well. Test compounds were prepared as stock solutions in DMSO and diluted into the 7H10 medium to achieve final concentrations of 100, 50, 25, 12.5, and 6.25 μM of CYP121 inhibitors and a final 1% v/v DMSO concentration. Rifampicin was used as a positive control and tested at 3.00, 1.50, 0.75, 0.37, and 0.18 μM concentrations. Prepared <i>Mtb</i> suspensions (OD<sub>600</sub> = 1.0, approximately 3.00 × 105 CFU) were aliquoted (10 μL) onto the compound-infused solid medium and incubated at 37 °C for 2 weeks. The MIC<sub>90</sub> was determined as the lowest concentration of compound that resulted in 90% inhibition of bacterial growth compared to the negative control wells, which were not treated with compounds and contained either 0% or 1% v/v DMSO.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Molecular Modeling and Docking</h4><div class="NLM_p last">Lead compound <b>2</b>, retrofragments <b>5</b> and <b>6</b>, and all Ar1, Ar2, and Ar3 analogues were prepared using the LigPrep v3.2 and Epik v3.0 functions of Schrödinger suite software (Schrödinger LLC, NY), selecting to include metal binding states when generating ligand ionization states. Ligands were docking into the crystal structure of CYP121 bound to lead <b>2</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a>) or the previously reported ligand 4-(1<i>H</i>-1,2,4-triazol-1-yl)quinolin-6-amine (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G1X">4G1X</a>). Proteins were prepared using the internal Protein Preparation function of the Schrödinger suite software. Ionization states were generated to be compatible with metal-binding interactions, and the heme-iron was manually adjusted to the ferric (+3) oxidation state. All water molecules were removed from the structure PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G1X">4G1X</a>. Duplicate energy minimized (OPLS 2005) structures of PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a> were prepared either with all water molecules removed or retaining the axial heme–water ligand only. Ar1 and Ar3 analogues were docked using core constraints to replicate the position of the aminopyrazole ring of lead <b>2</b>. Ar2 analogues were docked using core constraints to replicate the heme binding interactions of the 4-(1<i>H</i>-1,2,4-triazol-1-yl)quinolin-6-amine ligand in PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G1X">4G1X</a>. Images were generated using the PyMOL Molecular Graphics System, version 1.3, 2010 (Schrödinger, LLC).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i120"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00007">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00007" class="ext-link">10.1021/acs.jmedchem.6b00007</a>. Additional data related to this publication is available at the University of Cambridge data repository: <a href="https://www.repository.cam.ac.uk/handle/1810/254048" class="extLink">https://www.repository.cam.ac.uk/handle/1810/254048</a><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">X-ray crystallographic data tables, ligand density maps, human P450 inhibition data, and additional native mass spectrometry data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf">jm6b00007_si_001.pdf (2.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_002.csv">jm6b00007_si_002.csv (7.64 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>6</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>; <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>, <b>19a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF">5IBF</a>; <b>24a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBD">5IBD</a>; <b>25a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE">5IBE</a>; <b>25b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBG">5IBG</a>; <b>26a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI">5IBI</a>; and <b>26h</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH">5IBH</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b00007" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38457" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38457" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris Abell</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#680b095a5e280b090546090b461d03"><span class="__cf_email__" data-cfemail="3c5f5d0e0a7c5f5d51125d5f124957">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Madeline E. Kavanagh</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony G. Coyne</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kirsty J. McLean</span> - <span class="hlFld-Affiliation affiliation">Centre
for Synthetic Biology of Fine and Specialty Chemicals (SYNBIOCHEM),
Manchester Institute of Biotechnology, Faculty of Life Sciences, University of Manchester, 131 Princess Street, Manchester M1 7DN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guy G. James</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Colin W. Levy</span> - <span class="hlFld-Affiliation affiliation">Centre
for Synthetic Biology of Fine and Specialty Chemicals (SYNBIOCHEM),
Manchester Institute of Biotechnology, Faculty of Life Sciences, University of Manchester, 131 Princess Street, Manchester M1 7DN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leonardo B. Marino</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Mycobacterial Metabolism and Antibiotic Research, Francis Crick
Institute, The Mill Hill Laboratory, London NW7 1AA, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, São Paulo
State University (UNESP), 4801-902 Araraquara, SP, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luiz Pedro S. de Carvalho</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Mycobacterial Metabolism and Antibiotic Research, Francis Crick
Institute, The Mill Hill Laboratory, London NW7 1AA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel S. H. Chan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sean A. Hudson</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sachin Surade</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Cambridge, 80 Tennis Court Road, Cambridge CB2 1GA U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Leys</span> - <span class="hlFld-Affiliation affiliation">Centre
for Synthetic Biology of Fine and Specialty Chemicals (SYNBIOCHEM),
Manchester Institute of Biotechnology, Faculty of Life Sciences, University of Manchester, 131 Princess Street, Manchester M1 7DN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew W. Munro</span> - <span class="hlFld-Affiliation affiliation">Centre
for Synthetic Biology of Fine and Specialty Chemicals (SYNBIOCHEM),
Manchester Institute of Biotechnology, Faculty of Life Sciences, University of Manchester, 131 Princess Street, Manchester M1 7DN, U.K.</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e10021-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i123">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge Dr. Dijana Matak-Vinkovic for assistance with the native mass spectrometry experiments and the Diamond Light Source for access to beamlines i02, i04, and i24 (Proposal MX8997) that contributed to the results presented here. M.E.K. was supported by a Commonwealth (University of Cambridge) Scholarship awarded in conjunction with the Cambridge Commonwealth Trust and Cambridge Overseas Trust. A.G.C. and K.J.M. were supported by grants from the BBSRC (grant no. BB/I019669/1 and BB/I019227/1). G.G.J. received funding from the Ogden Trust and the Isaac Newton Trust administered through the University of Cambridge Bursary Scheme. D.S.C.H. was supported by a Croucher Cambridge International Scholarship awarded in conjunction between the Croucher Foundation and the Cambridge Overseas Trust. S.A.H. was supported by an Oliphant Cambridge Australia Scholarship (app no. 10132070) awarded by the Cambridge Commonwealth Trust. The contributions of L.B.M. and L.P.S.C. were supported by funds from the Francis Crick Institute, which receives its core funding principally from Wellcome Trust, Cancer Research UK, and the UK Medical Research Council (to L.P.S.C., MC_UP_A253_1111) and funds from FAPESP, CNPq, and CAPES-PDSE (to L.B.M., 2011/21232-1, 140079/2013-0, 99999.003125/2014-09).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i124" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i124"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i125" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i125"> Abbreviations Used</h2><tr><td class="NLM_term">cYY</td><td class="NLM_def"><p class="first last">cyclo-dityrosine (cyclo-<span class="smallcaps smallerCapital">l</span>-Tyr-<span class="smallcaps smallerCapital">l</span>-Tyr)</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">binding affinity</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">GE</td><td class="NLM_def"><p class="first last">group efficiency</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term"><i>Mtb</i></td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i></p></td></tr><tr><td class="NLM_term">P450</td><td class="NLM_def"><p class="first last">cytochrome P450 enzyme</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">TB</td><td class="NLM_def"><p class="first last">tuberculosis</p></td></tr><tr><td class="NLM_term">UV–vis</td><td class="NLM_def"><p class="first last">UV–visible spectrophotometry</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84181" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84181" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span>Global Tuberculosis Report; <span class="NLM_publisher-name">World Health Organisation</span>: <span class="NLM_publisher-loc">Geneva</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Global+Tuberculosis+Report%3B+World+Health+Organisation%3A+Geneva%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DGlobal%2520Tuberculosis%2520Report%26pub%3DWorld%2520Health%2520Organisation%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Cole, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eiglmeier, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekaia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badcock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chillingworth, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devlin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feltwell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentles, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holroyd, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornsby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krogh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moule, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quail, M. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajandream, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squares, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squares, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulston, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, B. G.</span><span> </span><span class="NLM_article-title">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">393</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">544</span><span class="refDoi"> DOI: 10.1038/31159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2F31159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=9634230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1cXjvV2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=1998&pages=537-544&author=S.+T.+Coleauthor=R.+Broschauthor=J.+Parkhillauthor=T.+Garnierauthor=C.+Churcherauthor=D.+Harrisauthor=S.+V.+Gordonauthor=K.+Eiglmeierauthor=S.+Gasauthor=C.+E.+Barryauthor=F.+Tekaiaauthor=K.+Badcockauthor=D.+Bashamauthor=D.+Brownauthor=T.+Chillingworthauthor=R.+Connorauthor=R.+Daviesauthor=K.+Devlinauthor=T.+Feltwellauthor=S.+Gentlesauthor=N.+Hamlinauthor=S.+Holroydauthor=T.+Hornsbyauthor=K.+Jagelsauthor=A.+Kroghauthor=J.+McLeanauthor=S.+Mouleauthor=L.+Murphyauthor=K.+Oliverauthor=J.+Osborneauthor=M.+a+Quailauthor=M.+A.+Rajandreamauthor=J.+Rogersauthor=S.+Rutterauthor=K.+Seegerauthor=J.+Skeltonauthor=R.+Squaresauthor=S.+Squaresauthor=J.+E.+Sulstonauthor=K.+Taylorauthor=S.+Whiteheadauthor=B.+G.+Barrell&title=Deciphering+the+biology+of+Mycobacterium+tuberculosis+from+the+complete+genome+sequence&doi=10.1038%2F31159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence</span></div><div class="casAuthors">Cole, S. T.; Brosch, R.; Parkhill, J.; Garnier, T.; Churcher, C.; Harris, D.; Gordon, S. V.; Eiglmeier, K.; Gas, S.; Barry, C. E., III; Tekaia, F.; Badcock, K.; Basham, D.; Brown, D.; Chillingworth, T.; Connor, R.; Davies, R.; Devlin, K.; Feltwell, T.; Gentles, S.; Hamlin, N.; Holroyd, S.; Hornsby, T.; Jagels, K.; Krogh, A.; McLean, J.; Moule, S.; Murphy, L.; Oliver, K.; Osborne, J.; Quail, M. A.; Rajandream, M.-A.; Rogers, J.; Rutter, S.; Seeger, K.; Skelton, J.; Squares, R.; Squares, S.; Sulston, J. E.; Taylor, K.; Whitehead, S.; Barrell, B. G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">6685</span>),
    <span class="NLM_cas:pages">537-544</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Countless millions of people have died from tuberculosis, a chronic infectious disease caused by the tubercle bacillus.  The complete genome sequence of the best-characterized strain of Mycobacterium tuberculosis, H37Rv, was detd. and analyzed in order to improve our understanding of the biol. of this slow-growing pathogen and to help the conception of new prophylactic and therapeutic interventions.  The genome comprises 4,411,529 base pairs, contains around 4000 genes, and has a very high G+C content that is reflected in the biased amino acid content of the proteins.  M. tuberculosis differs radically from other bacteria in that a very large portion of its coding capacity is devoted to the prodn. of enzymes involved in lipogenesis and lipolysis, and to 2 new families of glycine-rich proteins with a repetitive structure that may represent a source of antigenic variation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9Q-iByIunurVg90H21EOLACvtfcHk0ljhucViyDMdMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvV2ksrY%253D&md5=44d23e7674bfeb1a3e30b55abf5268d2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F31159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F31159%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DBrosch%26aufirst%3DR.%26aulast%3DParkhill%26aufirst%3DJ.%26aulast%3DGarnier%26aufirst%3DT.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DS.%2BV.%26aulast%3DEiglmeier%26aufirst%3DK.%26aulast%3DGas%26aufirst%3DS.%26aulast%3DBarry%26aufirst%3DC.%2BE.%26aulast%3DTekaia%26aufirst%3DF.%26aulast%3DBadcock%26aufirst%3DK.%26aulast%3DBasham%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DChillingworth%26aufirst%3DT.%26aulast%3DConnor%26aufirst%3DR.%26aulast%3DDavies%26aufirst%3DR.%26aulast%3DDevlin%26aufirst%3DK.%26aulast%3DFeltwell%26aufirst%3DT.%26aulast%3DGentles%26aufirst%3DS.%26aulast%3DHamlin%26aufirst%3DN.%26aulast%3DHolroyd%26aufirst%3DS.%26aulast%3DHornsby%26aufirst%3DT.%26aulast%3DJagels%26aufirst%3DK.%26aulast%3DKrogh%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DJ.%26aulast%3DMoule%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DK.%26aulast%3DOsborne%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DM.%2Ba%26aulast%3DRajandream%26aufirst%3DM.%2BA.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DRutter%26aufirst%3DS.%26aulast%3DSeeger%26aufirst%3DK.%26aulast%3DSkelton%26aufirst%3DJ.%26aulast%3DSquares%26aufirst%3DR.%26aulast%3DSquares%26aufirst%3DS.%26aulast%3DSulston%26aufirst%3DJ.%2BE.%26aulast%3DTaylor%26aufirst%3DK.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26atitle%3DDeciphering%2520the%2520biology%2520of%2520Mycobacterium%2520tuberculosis%2520from%2520the%2520complete%2520genome%2520sequence%26jtitle%3DNature%26date%3D1998%26volume%3D393%26spage%3D537%26epage%3D544%26doi%3D10.1038%2F31159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clift, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balding, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The preponderance of P450s in the Mycobacterium tuberculosis genome</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">220</span><span class="NLM_x">–</span> <span class="NLM_lpage">228</span><span class="refDoi"> DOI: 10.1016/j.tim.2006.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.tim.2006.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=220-228&author=K.+J.+McLeanauthor=D.+Cliftauthor=D.+G.+Lewisauthor=M.+Sabriauthor=P.+R.+Baldingauthor=M.+J.+Sutcliffeauthor=D.+Leysauthor=A.+W.+Munro&title=The+preponderance+of+P450s+in+the+Mycobacterium+tuberculosis+genome&doi=10.1016%2Fj.tim.2006.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2006.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2006.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DClift%26aufirst%3DD.%26aulast%3DLewis%26aufirst%3DD.%2BG.%26aulast%3DSabri%26aufirst%3DM.%26aulast%3DBalding%26aufirst%3DP.%2BR.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520preponderance%2520of%2520P450s%2520in%2520the%2520Mycobacterium%2520tuberculosis%2520genome%26jtitle%3DTrends%2520Microbiol.%26date%3D2006%26volume%3D14%26spage%3D220%26epage%3D228%26doi%3D10.1016%2Fj.tim.2006.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jowitt, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parish, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Expression and characterization of Mycobacterium tuberculosis CYP144: Common themes and lessons learned in the M. tuberculosis P450 enzyme family</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Proteins Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1814</span><span class="NLM_x">, </span> <span class="NLM_fpage">76</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1016/j.bbapap.2010.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.bbapap.2010.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20621636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1814&publication_year=2011&pages=76-87&author=M.+D.+Driscollauthor=K.+J.+McLeanauthor=M.+R.+Cheesmanauthor=T.+A.+Jowittauthor=M.+Howardauthor=P.+Carrollauthor=T.+Parishauthor=A.+W.+Munro&title=Expression+and+characterization+of+Mycobacterium+tuberculosis+CYP144%3A+Common+themes+and+lessons+learned+in+the+M.+tuberculosis+P450+enzyme+family&doi=10.1016%2Fj.bbapap.2010.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and characterization of Mycobacterium tuberculosis CYP144: Common themes and lessons learned in the M. tuberculosis P450 enzyme family</span></div><div class="casAuthors">Driscoll, Max D.; McLean, Kirsty J.; Cheesman, Myles R.; Jowitt, Thomas A.; Howard, Marjorie; Carroll, Paul; Parish, Tanya; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1814</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-87</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">CYP144 from Mycobacterium tuberculosis was expressed and purified.  CYP144 demonstrates heme thiolate coordination in its ferric form, but the cysteinate is protonated to thiol in both the carbon monoxide-bound and ligand-free ferrous forms (forming P420 in the former).  Tight binding of various azole drugs was shown, with affinity for miconazole (Kd = 0.98 μM), clotrimazole (0.37 μM) and econazole (0.78 μM) being highest.  These azoles are also the trio with the highest affinity for the essential CYP121 and for the cholesterol oxidase CYP125 (essential for host infection), and have high potency as anti-mycobacterial drugs.  Construction of a Mtb gene knockout strain demonstrated that CYP144 is not essential for growth in vitro.  However the deletion strain was more sensitive to azole inhibition in culture suggesting an important role for CYP144 in cell physiol. and/or in mediating azole resistance.  The biophys. and genetic features of CYP144 are compared to those of other characterized Mtb P 450s, identifying both commonality in properties (including thiolate protonation in ferrous P 450s) and intriguing differences in thermodn. and spectroscopic features.  Our developing knowledge of the Mtb P 450s has revealed unusual biochem. and gene essentiality, highlighting their potential as drug targets in this human pathogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizrKdwo_jGLVg90H21EOLACvtfcHk0ljtoaDWRyLApg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygs73I&md5=11d03eff159437d2df871c7ad343ecec</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2010.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2010.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DJowitt%26aufirst%3DT.%2BA.%26aulast%3DHoward%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DP.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DExpression%2520and%2520characterization%2520of%2520Mycobacterium%2520tuberculosis%2520CYP144%253A%2520Common%2520themes%2520and%2520lessons%2520learned%2520in%2520the%2520M.%2520tuberculosis%2520P450%2520enzyme%2520family%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2011%26volume%3D1814%26spage%3D76%26epage%3D87%26doi%3D10.1016%2Fj.bbapap.2010.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Tailleux, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelizzola, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortellaro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Withers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castagnoli, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gicquel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoker, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butcher, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyrolles, O.</span><span> </span><span class="NLM_article-title">Probing host pathogen cross-talk by transcriptional profiling of both Mycobacterium tuberculosis and infected human dendritic cells and macrophages</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1403</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1371%2Fjournal.pone.0001403" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1403&author=L.+Tailleuxauthor=S.+J.+Waddellauthor=M.+Pelizzolaauthor=A.+Mortellaroauthor=M.+Withersauthor=A.+Tanneauthor=P.+R.+Castagnoliauthor=B.+Gicquelauthor=N.+G.+Stokerauthor=P.+D.+Butcherauthor=M.+Fotiauthor=O.+Neyrolles&title=Probing+host+pathogen+cross-talk+by+transcriptional+profiling+of+both+Mycobacterium+tuberculosis+and+infected+human+dendritic+cells+and+macrophages&doi=10.1371%2Fjournal.pone.0001403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001403%26sid%3Dliteratum%253Aachs%26aulast%3DTailleux%26aufirst%3DL.%26aulast%3DWaddell%26aufirst%3DS.%2BJ.%26aulast%3DPelizzola%26aufirst%3DM.%26aulast%3DMortellaro%26aufirst%3DA.%26aulast%3DWithers%26aufirst%3DM.%26aulast%3DTanne%26aufirst%3DA.%26aulast%3DCastagnoli%26aufirst%3DP.%2BR.%26aulast%3DGicquel%26aufirst%3DB.%26aulast%3DStoker%26aufirst%3DN.%2BG.%26aulast%3DButcher%26aufirst%3DP.%2BD.%26aulast%3DFoti%26aufirst%3DM.%26aulast%3DNeyrolles%26aufirst%3DO.%26atitle%3DProbing%2520host%2520pathogen%2520cross-talk%2520by%2520transcriptional%2520profiling%2520of%2520both%2520Mycobacterium%2520tuberculosis%2520and%2520infected%2520human%2520dendritic%2520cells%2520and%2520macrophages%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1403%26doi%3D10.1371%2Fjournal.pone.0001403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Holsclaw, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogi, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilmore, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schelle, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leavell, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertozzi, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leary, J. A.</span><span> </span><span class="NLM_article-title">Structural characterization of a novel sulfated menaquinone produced by stf3 from Mycobacterium tuberculosis</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1021/cb800145r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb800145r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1GjsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=619-624&author=C.+M.+Holsclawauthor=K.+M.+Sogiauthor=S.+A.+Gilmoreauthor=M.+W.+Schelleauthor=M.+D.+Leavellauthor=C.+R.+Bertozziauthor=J.+A.+Leary&title=Structural+characterization+of+a+novel+sulfated+menaquinone+produced+by+stf3+from+Mycobacterium+tuberculosis&doi=10.1021%2Fcb800145r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of a novel sulfated menaquinone produced by stf3 from Mycobacterium tuberculosis</span></div><div class="casAuthors">Holsclaw, Cynthia M.; Sogi, Kimberly M.; Gilmore, Sarah A.; Schelle, Michael W.; Leavell, Michael D.; Bertozzi, Carolyn R.; Leary, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">619-624</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">M. tuberculosis, the causative agent of tuberculosis, produces unique sulfated metabolites assocd. with virulence.  One such metabolite from M. tuberculosis lipid exts., S881, has been shown to neg. regulate the virulence of M. tuberculosis in mouse infection studies, and its cell-surface localization suggests a role in modulating host-pathogen interactions.  However, a detailed structural anal. of S881 has remained elusive.  Here we use high-resoln., high-mass-accuracy, and tandem mass spectrometry to characterize the structure of S881.  Exact mass measurements showed that S881 is highly unsatd., tandem mass spectrometry indicated a polyisoprene-derived structure, and characterization of synthetic structural analogs confirmed that S881 is a previously undescribed sulfated deriv. of dihydromenaquinone-9, the primary quinol electron carrier in M. tuberculosis.  To our knowledge, this is the 1st example of a sulfated menaquinone produced in any prokaryote.  Together with previous studies, these findings suggest that this redox cofactor may play a role in mycobacterial pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTkFafyS9f7Vg90H21EOLACvtfcHk0ljtoaDWRyLApg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1GjsbvJ&md5=cf89de5bfdf4d7c68d0b9d119395c1f6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb800145r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb800145r%26sid%3Dliteratum%253Aachs%26aulast%3DHolsclaw%26aufirst%3DC.%2BM.%26aulast%3DSogi%26aufirst%3DK.%2BM.%26aulast%3DGilmore%26aufirst%3DS.%2BA.%26aulast%3DSchelle%26aufirst%3DM.%2BW.%26aulast%3DLeavell%26aufirst%3DM.%2BD.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26aulast%3DLeary%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520characterization%2520of%2520a%2520novel%2520sulfated%2520menaquinone%2520produced%2520by%2520stf3%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2008%26volume%3D3%26spage%3D619%26epage%3D624%26doi%3D10.1021%2Fcb800145r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafite, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pikuleva, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The structure of Mycobacterium tuberculosis CYP125: Molecular basis for cholesterol binding in a P450 needed for host infection</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">35524</span><span class="NLM_x">–</span> <span class="NLM_lpage">35533</span><span class="refDoi"> DOI: 10.1074/jbc.M109.032706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.032706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19846552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35524-35533&author=K.+J.+McLeanauthor=P.+Lafiteauthor=C.+Levyauthor=M.+R.+Cheesmanauthor=N.+Mastauthor=I.+A.+Pikulevaauthor=D.+Leysauthor=A.+W.+Munro&title=The+structure+of+Mycobacterium+tuberculosis+CYP125%3A+Molecular+basis+for+cholesterol+binding+in+a+P450+needed+for+host+infection&doi=10.1074%2Fjbc.M109.032706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Structure of Mycobacterium tuberculosis CYP125: Molecular Basis for Cholesterol Binding in a P450 Needed for Host Infection</span></div><div class="casAuthors">McLean, Kirsty J.; Lafite, Pierre; Levy, Colin; Cheesman, Myles R.; Mast, Natalia; Pikuleva, Irina A.; Leys, David; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">35524-35533</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We report characterization and the crystal structure of the Mycobacterium tuberculosis cytochrome P 450 CYP125, a P 450 implicated in metab. of host cholesterol and essential for establishing infection in mice.  CYP125 is purified in a high spin form and undergoes both type I and II spectral shifts with various azole drugs.  The 1.4-Å structure of ligand-free CYP125 reveals a "letterbox" active site cavity of dimensions appropriate for entry of a polycyclic sterol.  A mixt. of hexa-coordinate and penta-coordinate states could be discerned, with water binding as the 6th heme-ligand linked to conformation of the I-helix Val267 residue.  Structures in complex with androstenedione and the antitubercular drug econazole reveal that binding of hydrophobic ligands occurs within the active site cavity.  Due to the funnel shape of the active site near the heme, neither approaches the heme iron.  A model of the cholesterol-CYP125 complex shows that the alkyl side chain extends toward the heme iron, predicting hydroxylation of cholesterol C27.  The alkyl chain is in close contact to Val267, suggesting a substrate binding-induced low-spin to high-spin transition coupled to reorientation of the latter residue.  Reconstitution of CYP125 activity with a redox partner system revealed exclusively cholesterol 27-hydroxylation, consistent with structure and modeling.  This activity may enable catabolism of host cholesterol or generation of immunomodulatory compds. that enable persistence in the host.  This study reveals structural and catalytic properties of a potential M. tuberculosis drug target enzyme, and the likely mode by which the host-derived substrate is bound and hydroxylated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobJ0KfPDTC-rVg90H21EOLACvtfcHk0lj0pnSTQcVTBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnK&md5=86dd9f8bcf0072432b6e47ee937e3edd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.032706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.032706%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DLevy%26aufirst%3DC.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DMast%26aufirst%3DN.%26aulast%3DPikuleva%26aufirst%3DI.%2BA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520structure%2520of%2520Mycobacterium%2520tuberculosis%2520CYP125%253A%2520Molecular%2520basis%2520for%2520cholesterol%2520binding%2520in%2520a%2520P450%2520needed%2520for%2520host%2520infection%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35524%26epage%3D35533%26doi%3D10.1074%2Fjbc.M109.032706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Capyk, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalscheuer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eltis, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohn, W. W.</span><span> </span><span class="NLM_article-title">Mycobacterial cytochrome P450 125 (Cyp125) catalyzes the terminal hydroxylation of C27 steroids</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">35534</span><span class="NLM_x">–</span> <span class="NLM_lpage">35542</span><span class="refDoi"> DOI: 10.1074/jbc.M109.072132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.072132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19846551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35534-35542&author=J.+K.+Capykauthor=R.+Kalscheuerauthor=G.+R.+Stewartauthor=J.+Liuauthor=H.+Kwonauthor=R.+Zhaoauthor=S.+Okamotoauthor=W.+R.+Jacobsauthor=L.+D.+Eltisauthor=W.+W.+Mohn&title=Mycobacterial+cytochrome+P450+125+%28Cyp125%29+catalyzes+the+terminal+hydroxylation+of+C27+steroids&doi=10.1074%2Fjbc.M109.072132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mycobacterial cytochrome P450 125 (Cyp125) catalyzes the terminal hydroxylation of C27 steroids</span></div><div class="casAuthors">Capyk, Jenna K.; Kalscheuer, Rainer; Stewart, Gordon R.; Liu, Jie; Kwon, Hyukin; Zhao, Rafael; Okamoto, Sachi; Jacobs, William R., Jr.; Eltis, Lindsay D.; Mohn, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">35534-35542</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cyp125 (Rv3545c), a cytochrome P 450, is encoded as part of the cholesterol degrdn. gene cluster conserved among members of the Mycobacterium tuberculosis complex.  This enzyme has been implicated in mycobacterial pathogenesis, and a homolog initiates cholesterol catabolism in the soil actinomycete Rhodococcus jostii RHA1.  In Mycobacterium bovis BCG, cyp125 was up-regulated 7.1-fold with growth on cholesterol.  A cyp125 deletion mutant of BCG did not grow on cholesterol and accumulated 4-cholesten-3-one when incubated in the presence of cholesterol.  Wild-type BCG grew on this metabolite.  By contrast, a parallel cyp125 deletion mutation of M. tuberculosis H37Rv did not affect growth on cholesterol.  Purified Cyp125 from M. tuberculosis, heterologously produced in R. jostii RHA1, bound cholesterol and 4-cholesten-3-one with apparent dissocn. consts. of 0.20 ± 0.02 μM and 0.27 ± 0.05 μM, resp.  When reconstituted with KshB, the cognate reductase of the ketosteroid 9α-hydroxylase, Cyp125 catalyzed the hydroxylation of these steroids.  MS and NMR analyses revealed that hydroxylation occurred at carbon 26 of the steroid side chain, allowing unambiguous classification of Cyp125 as a steroid C26-hydroxylase.  This study establishes the catalytic function of Cyp125 and, in identifying an important difference in the catabolic potential of M. bovis and M. tuberculosis, suggests that Cyp125 may have an addnl. function in pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqInd-ojV5-urVg90H21EOLACvtfcHk0lj0pnSTQcVTBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnL&md5=3fb8869a235726445347ac42fe981722</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.072132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.072132%26sid%3Dliteratum%253Aachs%26aulast%3DCapyk%26aufirst%3DJ.%2BK.%26aulast%3DKalscheuer%26aufirst%3DR.%26aulast%3DStewart%26aufirst%3DG.%2BR.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DJacobs%26aufirst%3DW.%2BR.%26aulast%3DEltis%26aufirst%3DL.%2BD.%26aulast%3DMohn%26aufirst%3DW.%2BW.%26atitle%3DMycobacterial%2520cytochrome%2520P450%2520125%2520%2528Cyp125%2529%2520catalyzes%2520the%2520terminal%2520hydroxylation%2520of%2520C27%2520steroids%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35534%26epage%3D35542%26doi%3D10.1074%2Fjbc.M109.072132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Johnston, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kells, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Biochemical and structural characterization of CYP124: a methyl-branched lipid omega-hydroxylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">20687</span><span class="NLM_x">–</span> <span class="NLM_lpage">20692</span><span class="refDoi"> DOI: 10.1073/pnas.0907398106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.0907398106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19933331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFOntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=20687-20692&author=J.+B.+Johnstonauthor=P.+M.+Kellsauthor=L.+M.+Podustauthor=P.+R.+Ortiz+de+Montellano&title=Biochemical+and+structural+characterization+of+CYP124%3A+a+methyl-branched+lipid+omega-hydroxylase+from+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.0907398106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and structural characterization of CYP124: a methyl-branched lipid ω-hydroxylase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Johnston, Jonathan B.; Kells, Petrea M.; Podust, Larissa M.; Ortiz de Montellano, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">20687-20692, S20687/1-S20687/29</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mycobacterium tuberculosis (Mtb) produces a variety of methyl-branched lipids that serve important functions, including modulating the immune response during pathogenesis and contributing to a robust cell wall that is impermeable to many chem. agents.  Here, we report characterization of Mtb CYP124 (Rv2266) that includes demonstration of preferential oxidn. of methyl-branched lipids.  Spectrophotometric titrns. and anal. of reaction products indicate that CYP124 tightly binds and hydroxylates these substrates at the chem. disfavored ω-position.  We also report x-ray crystal structures of the ligand-free and phytanic acid-bound protein at a resoln. of 1.5 Å and 2.1 Å, resp., which provide structural insights into a cytochrome P 450 with predominant ω-hydroxylase activity.  The structures of ligand-free and substrate-bound CYP124 reveal several differences induced by substrate binding, including reorganization of the I helix and closure of the active site by elements of the F, G, and D helixes that bind the substrate and exclude solvent from the hydrophobic active site cavity.  The obsd. regiospecific catalytic activity suggests roles of CYP124 in the physiol. oxidn. of relevant Mtb methyl-branched lipids.  The enzymic specificity and structures reported here provide a scaffold for the design and testing of specific inhibitors of CYP124.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmsqzu2qXOKLVg90H21EOLACvtfcHk0lgtW27G0RYZCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFOntA%253D%253D&md5=74dd7e0f4dff778c6af163b4961a99cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0907398106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0907398106%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DJ.%2BB.%26aulast%3DKells%26aufirst%3DP.%2BM.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DBiochemical%2520and%2520structural%2520characterization%2520of%2520CYP124%253A%2520a%2520methyl-branched%2520lipid%2520omega-hydroxylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D20687%26epage%3D20692%26doi%3D10.1073%2Fpnas.0907398106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Gondry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauguet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amouroux, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuphile, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courçon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lautru, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernodet, J.-L.</span><span> </span><span class="NLM_article-title">Cyclodipeptide synthases are a family of tRNA-dependent peptide bond-forming enzymes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1038/nchembio.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnchembio.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19430487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1Cmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=414-420&author=M.+Gondryauthor=L.+Sauguetauthor=P.+Belinauthor=R.+Thaiauthor=R.+Amourouxauthor=C.+Tellierauthor=K.+Tuphileauthor=M.+Jacquetauthor=S.+Braudauthor=M.+Cour%C3%A7onauthor=C.+Massonauthor=S.+Duboisauthor=S.+Lautruauthor=A.+Lecoqauthor=S.+Hashimotoauthor=R.+Genetauthor=J.-L.+Pernodet&title=Cyclodipeptide+synthases+are+a+family+of+tRNA-dependent+peptide+bond-forming+enzymes&doi=10.1038%2Fnchembio.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclodipeptide synthases are a family of tRNA-dependent peptide bond-forming enzymes</span></div><div class="casAuthors">Gondry, Muriel; Sauguet, Ludovic; Belin, Pascal; Thai, Robert; Amouroux, Rachel; Tellier, Carine; Tuphile, Karine; Jacquet, Mickael; Braud, Sandrine; Courcon, Marie; Masson, Cedric; Dubois, Steven; Lautru, Sylvie; Lecoq, Alain; Hashimoto, Shin-ichi; Genet, Roger; Pernodet, Jean-Luc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">414-420</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclodipeptides and their derivs. belong to the diketopiperazine (DKP) family, which is comprised of a broad array of natural products that exhibit useful biol. properties.  In the few known DKP biosynthetic pathways, nonribosomal peptide synthetases (NRPSs) are involved in the synthesis of cyclodipeptides that constitute the DKP scaffold, except in the albonoursin (1) pathway.  Albonoursin, or cyclo(α,β-dehydroPhe-α,β-dehydroLeu), is an antibacterial DKP produced by Streptomyces noursei.  In this pathway, the formation of the cyclo(Phe-Leu) (2) intermediate is catalyzed by AlbC, a small protein unrelated to NRPSs.  We demonstrated that AlbC uses aminoacyl-tRNAs as substrates to catalyze the formation of the DKP peptide bonds.  Moreover, several other bacterial proteins, presenting moderate similarity to AlbC, also use aminoacyl-tRNAs to synthesize various cyclodipeptides.  Therefore, AlbC and these related proteins belong to a newly defined family of enzymes that we have named cyclodipeptide synthases (CDPSs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqok2iBeYtA37Vg90H21EOLACvtfcHk0lgtW27G0RYZCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1Cmtrs%253D&md5=760e57619b188f6232c4c5be8cef8422</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.175%26sid%3Dliteratum%253Aachs%26aulast%3DGondry%26aufirst%3DM.%26aulast%3DSauguet%26aufirst%3DL.%26aulast%3DBelin%26aufirst%3DP.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DAmouroux%26aufirst%3DR.%26aulast%3DTellier%26aufirst%3DC.%26aulast%3DTuphile%26aufirst%3DK.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DBraud%26aufirst%3DS.%26aulast%3DCour%25C3%25A7on%26aufirst%3DM.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DDubois%26aufirst%3DS.%26aulast%3DLautru%26aufirst%3DS.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DHashimoto%26aufirst%3DS.%26aulast%3DGenet%26aufirst%3DR.%26aulast%3DPernodet%26aufirst%3DJ.-L.%26atitle%3DCyclodipeptide%2520synthases%2520are%2520a%2520family%2520of%2520tRNA-dependent%2520peptide%2520bond-forming%2520enzymes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D414%26epage%3D420%26doi%3D10.1038%2Fnchembio.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Belin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Du, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fielding, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lequin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charbonnier, J.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courçon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugave, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernodet, J.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gondry, M.</span><span> </span><span class="NLM_article-title">Identification and structural basis of the reaction catalyzed by CYP121, an essential cytochrome P450 in Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7426</span><span class="NLM_x">–</span> <span class="NLM_lpage">7431</span><span class="refDoi"> DOI: 10.1073/pnas.0812191106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.0812191106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19416919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD1MzitlGitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7426-7431&author=P.+Belinauthor=M.+H.+Le+Duauthor=A.+Fieldingauthor=O.+Lequinauthor=M.+Jacquetauthor=J.-B.+Charbonnierauthor=A.+Lecoqauthor=R.+Thaiauthor=M.+Cour%C3%A7onauthor=C.+Massonauthor=C.+Dugaveauthor=R.+Genetauthor=J.-L.+Pernodetauthor=M.+Gondry&title=Identification+and+structural+basis+of+the+reaction+catalyzed+by+CYP121%2C+an+essential+cytochrome+P450+in+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.0812191106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and structural basis of the reaction catalyzed by CYP121, an essential cytochrome P450 in Mycobacterium tuberculosis</span></div><div class="casAuthors">Belin Pascal; Le Du Marie Helene; Fielding Alistair; Lequin Olivier; Jacquet Mickael; Charbonnier Jean-Baptiste; Lecoq Alain; Thai Robert; Courcon Marie; Masson Cedric; Dugave Christophe; Genet Roger; Pernodet Jean-Luc; Gondry Muriel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7426-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The gene encoding the cytochrome P450 CYP121 is essential for Mycobacterium tuberculosis.  However, the CYP121 catalytic activity remains unknown.  Here, we show that the cyclodipeptide cyclo(l-Tyr-l-Tyr) (cYY) binds to CYP121, and is efficiently converted into a single major product in a CYP121 activity assay containing spinach ferredoxin and ferredoxin reductase.  NMR spectroscopy analysis of the reaction product shows that CYP121 catalyzes the formation of an intramolecular C-C bond between 2 tyrosyl carbon atoms of cYY resulting in a novel chemical entity.  The X-ray structure of cYY-bound CYP121, solved at high resolution (1.4 A), reveals one cYY molecule with full occupancy in the large active site cavity.  One cYY tyrosyl approaches the heme and establishes a specific H-bonding network with Ser-237, Gln-385, Arg-386, and 3 water molecules, including the sixth iron ligand.  These observations are consistent with low temperature EPR spectra of cYY-bound CYP121 showing a change in the heme environment with the persistence of the sixth heme iron ligand.  As the carbon atoms involved in the final C-C coupling are located 5.4 A apart according to the CYP121-cYY complex crystal structure, we propose that C-C coupling is concomitant with substrate tyrosyl movements.  This study provides insight into the catalytic activity, mechanism, and biological function of CYP121.  Also, it provides clues for rational design of putative CYP121 substrate-based antimycobacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD1RfRZi76rY2y4bB4mnvYfW6udTcc2ebCpEV7ubhWzLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzitlGitA%253D%253D&md5=9ea72ef7c9e8ff65a47d050300c36808</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812191106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812191106%26sid%3Dliteratum%253Aachs%26aulast%3DBelin%26aufirst%3DP.%26aulast%3DLe%2BDu%26aufirst%3DM.%2BH.%26aulast%3DFielding%26aufirst%3DA.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DCharbonnier%26aufirst%3DJ.-B.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DCour%25C3%25A7on%26aufirst%3DM.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DDugave%26aufirst%3DC.%26aulast%3DGenet%26aufirst%3DR.%26aulast%3DPernodet%26aufirst%3DJ.-L.%26aulast%3DGondry%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520structural%2520basis%2520of%2520the%2520reaction%2520catalyzed%2520by%2520CYP121%252C%2520an%2520essential%2520cytochrome%2520P450%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7426%26epage%3D7431%26doi%3D10.1073%2Fpnas.0812191106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seward, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neeli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsollier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenkrands, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parish, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Characterization of active site structure in CYP121: A cytochrome P450 essential for viability of Mycobacterium tuberculosis H37Rv</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">33406</span><span class="NLM_x">–</span> <span class="NLM_lpage">33416</span><span class="refDoi"> DOI: 10.1074/jbc.M802115200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M802115200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=33406-33416&author=K.+J.+McLeanauthor=P.+Carrollauthor=D.+G.+Lewisauthor=A.+J.+Dunfordauthor=H.+E.+Sewardauthor=R.+Neeliauthor=M.+R.+Cheesmanauthor=L.+Marsollierauthor=P.+Douglasauthor=W.+E.+Smithauthor=I.+Rosenkrandsauthor=S.+T.+Coleauthor=D.+Leysauthor=T.+Parishauthor=A.+W.+Munro&title=Characterization+of+active+site+structure+in+CYP121%3A+A+cytochrome+P450+essential+for+viability+of+Mycobacterium+tuberculosis+H37Rv&doi=10.1074%2Fjbc.M802115200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802115200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802115200%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCarroll%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DD.%2BG.%26aulast%3DDunford%26aufirst%3DA.%2BJ.%26aulast%3DSeward%26aufirst%3DH.%2BE.%26aulast%3DNeeli%26aufirst%3DR.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DMarsollier%26aufirst%3DL.%26aulast%3DDouglas%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DW.%2BE.%26aulast%3DRosenkrands%26aufirst%3DI.%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DCharacterization%2520of%2520active%2520site%2520structure%2520in%2520CYP121%253A%2520A%2520cytochrome%2520P450%2520essential%2520for%2520viability%2520of%2520Mycobacterium%2520tuberculosis%2520H37Rv%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D33406%26epage%3D33416%26doi%3D10.1074%2Fjbc.M802115200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neeli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Structure, function and drug targeting in Mycobacterium tuberculosis cytochrome P450 systems</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1016/j.abb.2007.03.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.abb.2007.03.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2007&pages=228-240&author=K.+J.+McLeanauthor=A.+J.+Dunfordauthor=R.+Neeliauthor=M.+D.+Driscollauthor=A.+W.+Munro&title=Structure%2C+function+and+drug+targeting+in+Mycobacterium+tuberculosis+cytochrome+P450+systems&doi=10.1016%2Fj.abb.2007.03.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2007.03.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2007.03.026%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DDunford%26aufirst%3DA.%2BJ.%26aulast%3DNeeli%26aufirst%3DR.%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520function%2520and%2520drug%2520targeting%2520in%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520systems%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2007%26volume%3D464%26spage%3D228%26epage%3D240%26doi%3D10.1016%2Fj.abb.2007.03.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belcher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golovanova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The Mycobacterium tuberculosis cytochromes P450</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1339</span><span class="NLM_x">–</span> <span class="NLM_lpage">1353</span><span class="refDoi"> DOI: 10.4155/fmc.10.216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.4155%2Ffmc.10.216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=21426022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGru7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1339-1353&author=K.+J.+McLeanauthor=J.+Belcherauthor=M.+D.+Driscollauthor=C.+C.+Fernandezauthor=D.+L.+Vanauthor=S.+Buiauthor=M.+Golovanovaauthor=A.+W.+Munro&title=The+Mycobacterium+tuberculosis+cytochromes+P450&doi=10.4155%2Ffmc.10.216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Mycobacterium tuberculosis cytochromes P450: physiology, biochemistry & molecular intervention</span></div><div class="casAuthors">McLean, Kirsty J.; Belcher, James; Driscoll, Max D.; Fernandez, Christine C.; Le Van, Duyet; Bui, Soi; Golovanova, Marina; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1339-1353</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The human pathogen Mycobacterium tuberculosis (Mtb) encodes 20 cytochrome P 450 enzymes.  Gene essentiality for viability or host infection was demonstrated for Mtb P450s CYP128, CYP121, and CYP125.  Structure/function studies on Mtb P450s revealed key roles contributing to bacterial virulence and persistence in the host.  Various azole-class drugs bind with high affinity to the Mtb P 450 heme and are potent Mtb antibiotics.  This paper reviews the current understanding of the biochem. of Mtb P450s, their interactions with azoles and their potential as novel Mtb drug targets.  Mtb multidrug resistance is widespread and novel therapeutics are desperately needed.  Simultaneous drug targeting of several Mtb P450s crucial to bacterial viability/persistence could offer a new route to effective antibiotics and minimize the development of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAbxeIc99WbVg90H21EOLACvtfcHk0ljUDrJn9rXdLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGru7vJ&md5=436069b67f128151ffb82db04a6e1a4a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.216%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DBelcher%26aufirst%3DJ.%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DFernandez%26aufirst%3DC.%2BC.%26aulast%3DVan%26aufirst%3DD.%2BL.%26aulast%3DBui%26aufirst%3DS.%26aulast%3DGolovanova%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520Mycobacterium%2520tuberculosis%2520cytochromes%2520P450%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1339%26epage%3D1353%26doi%3D10.4155%2Ffmc.10.216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">Azole antifungals as novel chemotherapeutic agents against murine tuberculosis</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1111/j.1574-6968.2006.00350.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1111%2Fj.1574-6968.2006.00350.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2006&pages=181-186&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=Azole+antifungals+as+novel+chemotherapeutic+agents+against+murine+tuberculosis&doi=10.1111%2Fj.1574-6968.2006.00350.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6968.2006.00350.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6968.2006.00350.x%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DAzole%2520antifungals%2520as%2520novel%2520chemotherapeutic%2520agents%2520against%2520murine%2520tuberculosis%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2006%26volume%3D261%26spage%3D181%26epage%3D186%26doi%3D10.1111%2Fj.1574-6968.2006.00350.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">In vitro and ex vivo antimycobacterial potential of azole drugs against Mycobacterium tuberculosis H37Rv</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">251</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.femsle.2005.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.femsle.2005.07.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=2005&pages=19-22&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=In+vitro+and+ex+vivo+antimycobacterial+potential+of+azole+drugs+against+Mycobacterium+tuberculosis+H37Rv&doi=10.1016%2Fj.femsle.2005.07.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.femsle.2005.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.femsle.2005.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DIn%2520vitro%2520and%2520ex%2520vivo%2520antimycobacterial%2520potential%2520of%2520azole%2520drugs%2520against%2520Mycobacterium%2520tuberculosis%2520H37Rv%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2005%26volume%3D251%26spage%3D19%26epage%3D22%26doi%3D10.1016%2Fj.femsle.2005.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivers, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Expression, purification and spectroscopic characterization of the cytochrome P450 CYP121 from Mycobacterium tuberculosis</span> <span class="citation_source-journal">J. Inorg. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1016/S0162-0134(02)00479-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0162-0134%2802%2900479-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2002&pages=527-541&author=K.+J.+McLeanauthor=M.+R.+Cheesmanauthor=S.+L.+Riversauthor=A.+Richmondauthor=D.+Leysauthor=S.+K.+Chapmanauthor=G.+A.+Reidauthor=N.+C.+Priceauthor=S.+M.+Kellyauthor=J.+Clarksonauthor=W.+E.+Smithauthor=A.+W.+Munro&title=Expression%2C+purification+and+spectroscopic+characterization+of+the+cytochrome+P450+CYP121+from+Mycobacterium+tuberculosis&doi=10.1016%2FS0162-0134%2802%2900479-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0162-0134%2802%2900479-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0162-0134%252802%252900479-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DRivers%26aufirst%3DS.%2BL.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DReid%26aufirst%3DG.%2BA.%26aulast%3DPrice%26aufirst%3DN.%2BC.%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DClarkson%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DW.%2BE.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DExpression%252C%2520purification%2520and%2520spectroscopic%2520characterization%2520of%2520the%2520cytochrome%2520P450%2520CYP121%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Inorg.%2520Biochem.%26date%3D2002%26volume%3D91%26spage%3D527%26epage%3D541%26doi%3D10.1016%2FS0162-0134%2802%2900479-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">Chemotherapeutic evaluation of alginate nanoparticle-encapsulated azole antifungal and antitubercular drugs against murine tuberculosis</span> <span class="citation_source-journal">Nanomedicine</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1016/j.nano.2007.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.nano.2007.05.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=239-243&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=Chemotherapeutic+evaluation+of+alginate+nanoparticle-encapsulated+azole+antifungal+and+antitubercular+drugs+against+murine+tuberculosis&doi=10.1016%2Fj.nano.2007.05.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.nano.2007.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nano.2007.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DChemotherapeutic%2520evaluation%2520of%2520alginate%2520nanoparticle-encapsulated%2520azole%2520antifungal%2520and%2520antitubercular%2520drugs%2520against%2520murine%2520tuberculosis%26jtitle%3DNanomedicine%26date%3D2007%26volume%3D3%26spage%3D239%26epage%3D243%26doi%3D10.1016%2Fj.nano.2007.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Como, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dismukes, W. E.</span><span> </span><span class="NLM_article-title">Oral azole drugs as systemic antifungal therapy</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">330</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1056/NEJM199401273300407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1056%2FNEJM199401273300407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=8272088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADyaK2c%252FpsleksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=1994&pages=263-272&author=J.+A.+Comoauthor=W.+E.+Dismukes&title=Oral+azole+drugs+as+systemic+antifungal+therapy&doi=10.1056%2FNEJM199401273300407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Oral azole drugs as systemic antifungal therapy</span></div><div class="casAuthors">Como J A; Dismukes W E</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-72</span>
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">The oral azole drugs--ketoconazole, fluconazole, and itraconazole--represent a major advance in systemic antifungal therapy.  Among the three, fluconazole has the most attractive pharmacologic profile, including the capacity to produce high concentrations of active drug in cerebrospinal fluid and urine.  Ketoconazole, the first oral azole to be introduced, is less well tolerated than either fluconazole or itraconazole and is associated with more clinically important toxic effects, including hepatitis and inhibition of steroid hormone synthesis.  However, ketoconazole is less expensive than fluconazole and itraconazole--an especially important consideration for patients receiving long-term therapy.  All three drugs are effective alternatives to amphotericin B and flucytosine as therapy for selected systemic mycoses.  Ketoconazole and itraconazole are effective in patients with the chronic, indolent forms of the endemic mycoses, including blastomycosis, coccidioidomycosis, and histoplasmosis; itraconazole is also effective in patients with sporotrichosis.  Fluconazole is useful in the common forms of fungal meningitis--namely, coccidioidal and cryptococcal meningitis.  In addition, fluconazole is effective for selected patients with serious candida syndromes such as candidemia, and itraconazole is the most effective of the azoles for the treatment of aspergillosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJRByzIex4HNenUz0lpNqnfW6udTcc2ea5C5BWd7Pdxrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c%252FpsleksQ%253D%253D&md5=7d975fcad941723d2bdfb53882050521</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2FNEJM199401273300407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM199401273300407%26sid%3Dliteratum%253Aachs%26aulast%3DComo%26aufirst%3DJ.%2BA.%26aulast%3DDismukes%26aufirst%3DW.%2BE.%26atitle%3DOral%2520azole%2520drugs%2520as%2520systemic%2520antifungal%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1994%26volume%3D330%26spage%3D263%26epage%3D272%26doi%3D10.1056%2FNEJM199401273300407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent developments in fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">3680</span><span class="refDoi"> DOI: 10.1021/jm8000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-3680&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+developments+in+fragment-based+drug+discovery&doi=10.1021%2Fjm8000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments in fragment-based drug discovery</span></div><div class="casAuthors">Congreve, Miles; Chessari, Gianni; Tisi, Dominic; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3661-3680</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The field of fragment-based drug discovery (FBDD) has developed significantly over the past 10 years and is now recognized as a tangible alternative to more traditional methods of hit identification, such as high throughput screening (HTS).  The authors discuss trends and developments of FBDD over the past 4 years.  Addnl., this review looks in some detail at representative protein-ligand complexes obsd. between fragment-sized mols. and their protein targets from the Protein Data Bank (PDB).  More than the underlying technol. itself, the legacy of FBDD will be conceptual and that "less is more"; breaking down the problem of finding a drug mol. into its smallest component parts opens up new opportunities to solve challenging problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_YtOS7_glD7Vg90H21EOLACvtfcHk0lgQr9Uo1ysjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D&md5=41dbcf884305a190bffc683ae5ed9e45</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520developments%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D3680%26doi%3D10.1021%2Fjm8000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lgQr9Uo1ysjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">5003</span><span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lgQr9Uo1ysjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Hall, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Efficient exploration of chemical space by fragment-based screening</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">82</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.pbiomolbio.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=25268064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ymu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=82-91&author=R.+J.+Hallauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Efficient+exploration+of+chemical+space+by+fragment-based+screening&doi=10.1016%2Fj.pbiomolbio.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient exploration of chemical space by fragment-based screening</span></div><div class="casAuthors">Hall, Richard J.; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">82-91</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Screening methods seek to sample a vast chem. space in order to identify starting points for further chem. optimization.  Fragment based drug discovery exploits the superior sampling of chem. space that can be achieved when the mol. wt. is restricted.  Here we show that com. available fragment space is still relatively poorly sampled and argue for highly sensitive screening methods to allow the detection of smaller fragments.  We analyze the properties of our fragment library vs. the properties of X-ray hits derived from the library.  We particularly consider properties related to the degree of planarity of the fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdghyX93HMZLVg90H21EOLACvtfcHk0ljmZBBKkoxlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ymu7zO&md5=a63e451443207393b781d3d68b10bcd1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DEfficient%2520exploration%2520of%2520chemical%2520space%2520by%2520fragment-based%2520screening%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26spage%3D82%26epage%3D91%26doi%3D10.1016%2Fj.pbiomolbio.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery and design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1021/ci400731w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400731w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=693-704&author=D.+Joseph-McCarthyauthor=A.+J.+Campbellauthor=G.+Kernauthor=D.+Moustakas&title=Fragment-based+lead+discovery+and+design&doi=10.1021%2Fci400731w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery and Design</span></div><div class="casAuthors">Joseph-McCarthy, Diane; Campbell, Arthur J.; Kern, Gunther; Moustakas, Demetri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">693-704</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery and design has and continues to show increasing promise in drug discovery.  In this article, the current state of the art in terms of hot-spot characterization, fragment screening techniques, and fragment-based design is discussed.  Three overall fragment-based lead generation strategies are explored and involve the chem. biol. characterization of biol. targets via fragment screening, fragment screening as a complementary approach to high-throughput screening of drug-like compds., and direct fragment-based drug discovery, resp.  The evolution and development of fragment libraries is described.  With an emphasis on computational approaches and the strategies applied at AstraZeneca, the review illustrates how integration of data from one regime can inform the design of expts. in the other, ultimately leading to the discovery of high quality chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCa5krM_w737Vg90H21EOLACvtfcHk0ljmZBBKkoxlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D&md5=31b72219dfd8cec7e3612de1b61f8150</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci400731w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400731w%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DKern%26aufirst%3DG.%26aulast%3DMoustakas%26aufirst%3DD.%26atitle%3DFragment-based%2520lead%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D693%26epage%3D704%26doi%3D10.1021%2Fci400731w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span> </span><span class="NLM_article-title">Vemurafenib: the first drug approved for BRAF-mutant cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+the+first+drug+approved+for+BRAF-mutant+cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0ljmZBBKkoxlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520the%2520first%2520drug%2520approved%2520for%2520BRAF-mutant%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowat, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkinshaw, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Atomic structure of Mycobacterium tuberculosis CYP121 to 1.06 Å reveals novel features of cytochrome P450</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">5141</span><span class="NLM_x">–</span> <span class="NLM_lpage">5147</span><span class="refDoi"> DOI: 10.1074/jbc.M209928200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M209928200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=5141-5147&author=D.+Leysauthor=C.+G.+Mowatauthor=K.+J.+McLeanauthor=A.+Richmondauthor=S.+K.+Chapmanauthor=M.+D.+Walkinshawauthor=A.+W.+Munro&title=Atomic+structure+of+Mycobacterium+tuberculosis+CYP121+to+1.06+%C3%85+reveals+novel+features+of+cytochrome+P450&doi=10.1074%2Fjbc.M209928200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M209928200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M209928200%26sid%3Dliteratum%253Aachs%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMowat%26aufirst%3DC.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DAtomic%2520structure%2520of%2520Mycobacterium%2520tuberculosis%2520CYP121%2520to%25201.06%2520%25C3%2585%2520reveals%2520novel%2520features%2520of%2520cytochrome%2520P450%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D5141%26epage%3D5147%26doi%3D10.1074%2Fjbc.M209928200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Application of fragment screening and merging to the discovery of inhibitors of the Mycobacterium tuberculosis cytochrome P450 CYP121</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">9311</span><span class="NLM_x">–</span> <span class="NLM_lpage">9316</span><span class="refDoi"> DOI: 10.1002/anie.201202544</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fanie.201202544" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOmt7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=9311-9316&author=S.+A.+Hudsonauthor=K.+J.+McLeanauthor=S.+Suradeauthor=Y.+Q.+Yangauthor=D.+Leysauthor=A.+Ciulliauthor=A.+W.+Munroauthor=C.+Abell&title=Application+of+fragment+screening+and+merging+to+the+discovery+of+inhibitors+of+the+Mycobacterium+tuberculosis+cytochrome+P450+CYP121&doi=10.1002%2Fanie.201202544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment Screening and Merging to the Discovery of Inhibitors of the Mycobacterium tuberculosis Cytochrome P450 CYP121</span></div><div class="casAuthors">Hudson, Sean A.; McLean, Kirsty J.; Surade, Sachin; Yang, Yong-Qing; Leys, David; Ciulli, Alessio; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9311-9316, S9311/1-S9311/27</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This study represents the first successful application of fragment-based approaches to a cytochrome P 450.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZpopos9EebVg90H21EOLACvtfcHk0ljIvqJAXN53nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOmt7bO&md5=e90a7694c1aa055db088d855705413af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201202544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201202544%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%2BQ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520and%2520merging%2520to%2520the%2520discovery%2520of%2520inhibitors%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520CYP121%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D9311%26epage%3D9316%26doi%3D10.1002%2Fanie.201202544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mclean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Overcoming the limitations of fragment merging: Rescuing a strained merged fragment series targeting mycobacterium tuberculosis CYP121</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1456</span><span class="refDoi"> DOI: 10.1002/cmdc.201300219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcmdc.201300219" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1451-1456&author=S.+A.+Hudsonauthor=S.+Suradeauthor=A.+G.+Coyneauthor=K.+J.+Mcleanauthor=D.+Leysauthor=A.+W.+Munroauthor=C.+Abell&title=Overcoming+the+limitations+of+fragment+merging%3A+Rescuing+a+strained+merged+fragment+series+targeting+mycobacterium+tuberculosis+CYP121&doi=10.1002%2Fcmdc.201300219"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300219%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DMclean%26aufirst%3DK.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DOvercoming%2520the%2520limitations%2520of%2520fragment%2520merging%253A%2520Rescuing%2520a%2520strained%2520merged%2520fragment%2520series%2520targeting%2520mycobacterium%2520tuberculosis%2520CYP121%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1451%26epage%3D1456%26doi%3D10.1002%2Fcmdc.201300219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">The maximal affinity of ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">9997</span><span class="NLM_x">–</span> <span class="NLM_lpage">10002</span><span class="refDoi"> DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of ≈-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0ljIvqJAXN53nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ea-Ay0J1oJswLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Group efficiency: A guideline for hits-to-leads chemistry</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1179</span><span class="NLM_x">–</span> <span class="NLM_lpage">1180</span><span class="refDoi"> DOI: 10.1002/cmdc.200800132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcmdc.200800132" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1179-1180&author=M.+L.+Verdonkauthor=D.+C.+Rees&title=Group+efficiency%3A+A+guideline+for+hits-to-leads+chemistry&doi=10.1002%2Fcmdc.200800132"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800132%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DGroup%2520efficiency%253A%2520A%2520guideline%2520for%2520hits-to-leads%2520chemistry%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1179%26epage%3D1180%26doi%3D10.1002%2Fcmdc.200800132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">The additivity of substrate fragments in enzyme-ligand binding</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00086-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0969-2126%2898%2900086-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=9687366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1cXltVSjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=839-848&author=T.+J.+Stoutauthor=C.+R.+Sageauthor=R.+M.+Stroud&title=The+additivity+of+substrate+fragments+in+enzyme-ligand+binding&doi=10.1016%2FS0969-2126%2898%2900086-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The additivity of substrate fragments in enzyme-ligand binding</span></div><div class="casAuthors">Stout, Thomas J.; Sage, Carleton R.; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">839-848</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">Enzymes have evolved to recognize their target substrates with exquisite selectivity and specificity.  Whether fragments of the substrate - perhaps never available to the evolving enzyme - are bound in the same manner as the parent substrate addresses the fundamental basis of specificity.  An understanding of the relative contributions of individual portions of ligand mols. to the enzyme-binding interaction may offer considerable insight into the principles of substrate recognition.  We report 12 crystal structures of Escherichia coli thymidylate synthase in complexes with available fragments of the substrate (dUMP), both with and without the presence of a cofactor analog.  The structures display considerable fidelity of binding mode and interactions.  These complexes reveal several interesting features: the cofactor analog enhances the localization of substrate and substrate fragments near the reactive thiol; the ribose moiety reduces local disorder through addnl. specific enzyme-ligand interactions; the pyrimidine has multiple roles, ranging from stereospecificity to mechanistic competence; and the glycosidic linkage has an important role in the formation of a covalent attachment between substrate and enzyme.  The requirements of ligand-protein binding can be understood in terms of the binding of sep. fragments of the ligand.  Fragments which are subsystems of the natural substrate for the enzyme confer specific contributions to the binding affinity, orientation or electrostatics of the enzymic mechanism.  This ligand-binding anal. provides a complementary method to the more prevalent approaches utilizing site-directed mutagenesis.  In addn., these observations suggest a modular approach for rational drug design utilizing chem. fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3xLbwAs8Qb7Vg90H21EOLACvtfcHk0lgoOa3Qg_yfgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltVSjs78%253D&md5=ecac0bbb008bcdea46c38403e3fe0c6f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900086-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900086-0%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DSage%26aufirst%3DC.%2BR.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DThe%2520additivity%2520of%2520substrate%2520fragments%2520in%2520enzyme-ligand%2520binding%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D839%26epage%3D848%26doi%3D10.1016%2FS0969-2126%2898%2900086-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Compton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebacz, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katzenellenbogen, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katzenellenbogen, J. A.</span><span> </span><span class="NLM_article-title">Pyrazolo[1,5-a]pyrimidines: estrogen receptor ligands possessing estrogen receptor beta antagonist activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5872</span><span class="NLM_x">–</span> <span class="NLM_lpage">5893</span><span class="refDoi"> DOI: 10.1021/jm049631k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049631k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5872-5893&author=D.+R.+Comptonauthor=S.+Shengauthor=K.+E.+Carlsonauthor=N.+A.+Rebaczauthor=I.+Y.+Leeauthor=B.+S.+Katzenellenbogenauthor=J.+A.+Katzenellenbogen&title=Pyrazolo%5B1%2C5-a%5Dpyrimidines%3A+estrogen+receptor+ligands+possessing+estrogen+receptor+beta+antagonist+activity&doi=10.1021%2Fjm049631k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm049631k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049631k%26sid%3Dliteratum%253Aachs%26aulast%3DCompton%26aufirst%3DD.%2BR.%26aulast%3DSheng%26aufirst%3DS.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DRebacz%26aufirst%3DN.%2BA.%26aulast%3DLee%26aufirst%3DI.%2BY.%26aulast%3DKatzenellenbogen%26aufirst%3DB.%2BS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26atitle%3DPyrazolo%255B1%252C5-a%255Dpyrimidines%253A%2520estrogen%2520receptor%2520ligands%2520possessing%2520estrogen%2520receptor%2520beta%2520antagonist%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5872%26epage%3D5893%26doi%3D10.1021%2Fjm049631k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Johnson, M. D.; Teng, M.; Zhu, J.</span><span> </span><span class="NLM_article-title">Aminopyrazole compounds and use as CHK1 inhibitors</span>. WO2005/00945 A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=M.+D.+Johnson&author=M.+Teng&author=J.+Zhu&title=Aminopyrazole+compounds+and+use+as+CHK1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26atitle%3DAminopyrazole%2520compounds%2520and%2520use%2520as%2520CHK1%2520inhibitors%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0lgTn6QNiV7RwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span> <span class="citation_source-book">Cytochrome P450: Structure, Mechanism and Biochemistry</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span>, Ed.; <span class="NLM_publisher-name">Kluwer Academic/Plenum Publishers</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1007%2Fb139087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=P.+R.+Ortiz+de+Montellano&title=Cytochrome+P450%3A+Structure%2C+Mechanism+and+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fb139087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb139087%26sid%3Dliteratum%253Aachs%26btitle%3DCytochrome%2520P450%253A%2520Structure%252C%2520Mechanism%2520and%2520Biochemistry%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26pub%3DKluwer%2520Academic%252FPlenum%2520Publishers%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span>Glide, <span class="NLM_edition">version 6.5</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide%2C+version+6.5%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Smith, A. J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Beyond picomolar affinities: Quantitative aspects of noncovalent and covalent binding of drugs to proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1021/jm800498e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800498e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1OltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=225-233&author=A.+J.+T.+Smithauthor=X.+Zhangauthor=A.+G.+Leachauthor=K.+N.+Houk&title=Beyond+picomolar+affinities%3A+Quantitative+aspects+of+noncovalent+and+covalent+binding+of+drugs+to+proteins&doi=10.1021%2Fjm800498e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond picomolar affinities: quantitative aspects of noncovalent and covalent binding of drugs to proteins</span></div><div class="casAuthors">Smith, Adam J. T.; Zhang, Xiyun; Leach, Andrew G.; Houk, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review of 104 refs. summarized quant. aspects of noncovalent and covalent binding of drugs to proteins with subtopics including (1) introduction; (2) factors involved in ligand-recepto binding; (3) distribution of exptl. binding consts.; (3) covalent interactions with transition states as the origins of enzyme proficiency; (4) potential quant. advantages of covalent enzyme inhibitors and colent drugs; (5) selected examples of covalent inhibitors and drugs; (6) prospects for the design of covalent drugs; (7) conclusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhUbRxt1fu7Vg90H21EOLACvtfcHk0lgTn6QNiV7RwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1OltLw%253D&md5=529d4181046100b5e3c37a8fef571218</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm800498e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800498e%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DA.%2BJ.%2BT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DBeyond%2520picomolar%2520affinities%253A%2520Quantitative%2520aspects%2520of%2520noncovalent%2520and%2520covalent%2520binding%2520of%2520drugs%2520to%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D225%26epage%3D233%26doi%3D10.1021%2Fjm800498e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Abdel-Magid, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maryanoff, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, R. D.</span><span> </span><span class="NLM_article-title">Reductive amination of aldehydes and ketones with sodium triacetoxyborohydride. Studies on direct and indirect reductive amination procedures</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">3849</span><span class="NLM_x">–</span> <span class="NLM_lpage">3862</span><span class="refDoi"> DOI: 10.1021/jo960057x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo960057x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK28XivFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=3849-3862&author=A.+F.+Abdel-Magidauthor=K.+G.+Carsonauthor=B.+D.+Harrisauthor=C.+A.+Maryanoffauthor=R.+D.+Shah&title=Reductive+amination+of+aldehydes+and+ketones+with+sodium+triacetoxyborohydride.+Studies+on+direct+and+indirect+reductive+amination+procedures&doi=10.1021%2Fjo960057x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Reductive Amination of Aldehydes and Ketones with Sodium Triacetoxyborohydride. Studies on Direct and Indirect Reductive Amination Procedures</span></div><div class="casAuthors">Abdel-Magid, Ahmed F.; Carson, Kenneth G.; Harris, Bruce D.; Maryanoff, Cynthia A.; Shah, Rekha D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3849-3862</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sodium triacetoxyborohydride is presented as a general reducing agent for the reductive amination of aldehydes and ketones.  Procedures for using this mild and selective reagent have been developed for a wide variety of substrates.  The scope of the reaction includes aliph. acyclic and cyclic ketones, aliph. and arom. aldehydes, and primary and secondary amines including a variety of weakly basic and nonbasic amines.  Limitations include reactions with arom. and unsatd. ketones and some sterically hindered ketones and amines.  1,2-Dichloroethane (DCE) is the preferred reaction solvent, but reactions can also be carried out in THF and occasionally in acetonitrile.  Acetic acid may be used as catalyst with ketone reactions, but it is generally not needed with aldehydes.  The procedure is carried out effectively in the presence of acid sensitive functional groups such as acetals and ketals; it can also be carried out in the presence of reducible functional groups such as C-C multiple bonds and cyano and nitro groups.  Reactions are generally faster in DCE than in THF, and in both solvents, reactions are faster in the presence of AcOH.  In comparison with other reductive amination procedures such as NaBH3CN/MeOH, borane-pyridine, and catalytic hydrogenation, NaBH(OAc)3 gave consistently higher yields and fewer side products.  In the reductive amination of some aldehydes with primary amines where dialkylation is a problem we adopted a stepwise procedure involving imine formation in MeOH followed by redn. with NaBH4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1pj6c2VgVU7Vg90H21EOLACvtfcHk0lhjudEne8g2pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XivFShu7o%253D&md5=9cc14c9adbbb916bd3326cb3e83a9b40</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjo960057x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo960057x%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26aulast%3DCarson%26aufirst%3DK.%2BG.%26aulast%3DHarris%26aufirst%3DB.%2BD.%26aulast%3DMaryanoff%26aufirst%3DC.%2BA.%26aulast%3DShah%26aufirst%3DR.%2BD.%26atitle%3DReductive%2520amination%2520of%2520aldehydes%2520and%2520ketones%2520with%2520sodium%2520triacetoxyborohydride.%2520Studies%2520on%2520direct%2520and%2520indirect%2520reductive%2520amination%2520procedures%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D3849%26epage%3D3862%26doi%3D10.1021%2Fjo960057x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Vasudevan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaskovich, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkanda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebti, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, A. D.</span><span> </span><span class="NLM_article-title">Potent, highly selective, and non-thiol inhibitors of protein geranylgeranyltransferase-I</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1333</span><span class="NLM_x">–</span> <span class="NLM_lpage">1340</span><span class="refDoi"> DOI: 10.1021/jm9900873</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9900873" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1333-1340&author=A.+Vasudevanauthor=Y.+Qianauthor=A.+Vogtauthor=M.+A.+Blaskovichauthor=J.+Ohkandaauthor=S.+M.+Sebtiauthor=A.+D.+Hamilton&title=Potent%2C+highly+selective%2C+and+non-thiol+inhibitors+of+protein+geranylgeranyltransferase-I&doi=10.1021%2Fjm9900873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm9900873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9900873%26sid%3Dliteratum%253Aachs%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DA.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3DPotent%252C%2520highly%2520selective%252C%2520and%2520non-thiol%2520inhibitors%2520of%2520protein%2520geranylgeranyltransferase-I%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1333%26epage%3D1340%26doi%3D10.1021%2Fjm9900873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Schenkman, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remmer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estabrook, R. W.</span><span> </span><span class="NLM_article-title">Spectral studies of drug interaction with hepatic microsomal cytochrome</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1967</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=4382749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaF2sXosVCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1967&pages=113-123&author=J.+B.+Schenkmanauthor=H.+Remmerauthor=R.+W.+Estabrook&title=Spectral+studies+of+drug+interaction+with+hepatic+microsomal+cytochrome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Spectral studies of drug interaction with hepatic microsomal cytochrome</span></div><div class="casAuthors">Schenkman, John B.; Remmer, Herbert; Estabrook, Ronald W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1967</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">113-23</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Two types of spectral changes are described as resulting from substrate interaction with a hepatic microsomal cytochrome; the magnitude of these spectral changes is dependent on protein concn. and substrate concn. as well as the substrate employed.  In 2 of the 3 types of substrates examd. the concn. of substrate necessary to evoke half-max. enzyme activity was similar to the concn. of the same substrate necessary for half-max. spectral changes.  The addn. of NADPH, a corequirement for the microsomal mixed function oxidase, causes a modification of both types of spectral changes, without altering substrate affinity.  Three possible hypotheses are advanced, based upon the exptl. observations, to explain the two types of spectral changes observed.  24 references.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoInUTNOrGCNbVg90H21EOLACvtfcHk0lhjudEne8g2pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2sXosVCjtQ%253D%253D&md5=8f1a923fb13a3c6bbeed43c6d0e80e5e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchenkman%26aufirst%3DJ.%2BB.%26aulast%3DRemmer%26aufirst%3DH.%26aulast%3DEstabrook%26aufirst%3DR.%2BW.%26atitle%3DSpectral%2520studies%2520of%2520drug%2520interaction%2520with%2520hepatic%2520microsomal%2520cytochrome%26jtitle%3DMol.%2520Pharmacol.%26date%3D1967%26volume%3D3%26spage%3D113%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kells, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span> </span><span class="NLM_article-title">Reverse type I inhibitor of Mycobacterium tuberculosis CYP125A1</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">332</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.bmcl.2010.11.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=332-337&author=H.+Ouelletauthor=P.+M.+Kellsauthor=P.+R.+Ortiz+de+Montellanoauthor=L.+M.+Podust&title=Reverse+type+I+inhibitor+of+Mycobacterium+tuberculosis+CYP125A1&doi=10.1016%2Fj.bmcl.2010.11.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3DKells%26aufirst%3DP.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26atitle%3DReverse%2520type%2520I%2520inhibitor%2520of%2520Mycobacterium%2520tuberculosis%2520CYP125A1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D332%26epage%3D337%26doi%3D10.1016%2Fj.bmcl.2010.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fowler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieser, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurcha, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besra, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Azole antifungals are potent inhibitors of cytochrome P450 mono-oxygenases and bacterial growth in mycobacteria and streptomycetes</span> <span class="citation_source-journal">Microbiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">148</span><span class="NLM_x">, </span> <span class="NLM_fpage">2937</span><span class="NLM_x">–</span> <span class="NLM_lpage">2949</span><span class="refDoi"> DOI: 10.1099/00221287-148-10-2937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1099%2F00221287-148-10-2937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12368427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD38XotFymtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2002&pages=2937-2949&author=K.+J.+McLeanauthor=K.+R.+Marshallauthor=A.+Richmondauthor=I.+S.+Hunterauthor=K.+Fowlerauthor=T.+Kieserauthor=S.+S.+Gurchaauthor=G.+S.+Besraauthor=A.+W.+Munro&title=Azole+antifungals+are+potent+inhibitors+of+cytochrome+P450+mono-oxygenases+and+bacterial+growth+in+mycobacteria+and+streptomycetes&doi=10.1099%2F00221287-148-10-2937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Azole antifungals are potent inhibitors of cytochrome P450 monooxygenases and bacterial growth in mycobacteria and streptomycetes</span></div><div class="casAuthors">McLean, Kirsty J.; Marshall, Ker R.; Richmond, Alison; Hunter, Iain S.; Fowler, Kay; Kieser, Tobias; Gurcha, Sudagar S.; Besra, Gurydal S.; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology (Reading, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2937-2949</span>CODEN:
                <span class="NLM_cas:coden">MROBEO</span>;
        ISSN:<span class="NLM_cas:issn">1350-0872</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">The genome sequence of Mycobacterium tuberculosis has revealed the presence of 20 different cytochrome P 450 monooxygenases (P450s) within this organism, and subsequent genome sequences of other mycobacteria and of Streptomyces coelicolor have indicated that these actinomycetes also have large complements of P450s, pointing to important physiol. roles for these enzymes.  The actinomycete P450s include homologues of 14α-sterol demethylases, the targets for the azole class of drugs in yeast and fungi.  Previously, this type of P 450 was considered to be absent from bacteria.  When present at low concns. in growth medium, azole antifungal drugs were shown to be potent inhibitors of the growth of Mycobacterium smegmatis and of Streptomyces strains, indicating that one or more of the P450s in these bacteria were viable drug targets.  The drugs econazole and clotrimazole were most effective against M. smegmatis (MIC values of <0·2 and 0·3 μM, resp.) and were superior inhibitors of mycobacterial growth compared to rifampicin and isoniazid (which had MIC values of 1·2 and 36·5 μM, resp.).  In contrast to their effects on the actinomycetes, the azoles showed minimal effects on the growth of Escherichia coli, which is devoid of P450s.  Azole drugs coordinated tightly to the heme iron in M. tuberculosis H37Rv P450s encoded by genes Rv0764c (the sterol demethylase CYP51) and Rv2276 (CYP121).  However, the azoles had a higher affinity for M. tuberculosis CYP121, with Kd values broadly in line with the MIC values for M. smegmatis.  This suggested that CYP121 may be a more realistic target enzyme for the azole drugs than CYP51, particularly in light of the fact that an S. coelicolor ΔCYP51 strain was viable and showed little difference in its sensitivity to azole drugs compared to the wild-type.  If the azole drugs prove to inhibit a no. of important P450s in M. smegmatis and S. coelicolor, then the likelihood of drug resistance developing in these species should be minimal.  This suggests that azole drug therapy may provide a novel antibiotic strategy against strains of M. tuberculosis that have already developed resistance to isoniazid and other front-line drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkLDhhvksLVg90H21EOLACvtfcHk0ljjxx-E1E45bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotFymtrg%253D&md5=249eccb4ffedf2acee629b132d8799b7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1099%2F00221287-148-10-2937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F00221287-148-10-2937%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DHunter%26aufirst%3DI.%2BS.%26aulast%3DFowler%26aufirst%3DK.%26aulast%3DKieser%26aufirst%3DT.%26aulast%3DGurcha%26aufirst%3DS.%2BS.%26aulast%3DBesra%26aufirst%3DG.%2BS.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DAzole%2520antifungals%2520are%2520potent%2520inhibitors%2520of%2520cytochrome%2520P450%2520mono-oxygenases%2520and%2520bacterial%2520growth%2520in%2520mycobacteria%2520and%2520streptomycetes%26jtitle%3DMicrobiology%26date%3D2002%26volume%3D148%26spage%3D2937%26epage%3D2949%26doi%3D10.1099%2F00221287-148-10-2937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Yoshida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, Y.</span><span> </span><span class="NLM_article-title">Interaction of azole antifungal agents with cytochrome P-45014DM purified from Saccharomyces cerevisiae microsomes</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">229</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1016/0006-2952(87)90694-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0006-2952%2887%2990694-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1987&pages=229-235&author=Y.+Yoshidaauthor=Y.+Aoyama&title=Interaction+of+azole+antifungal+agents+with+cytochrome+P-45014DM+purified+from+Saccharomyces+cerevisiae+microsomes&doi=10.1016%2F0006-2952%2887%2990694-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2887%2990694-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252887%252990694-0%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DY.%26aulast%3DAoyama%26aufirst%3DY.%26atitle%3DInteraction%2520of%2520azole%2520antifungal%2520agents%2520with%2520cytochrome%2520P-45014DM%2520purified%2520from%2520Saccharomyces%2520cerevisiae%2520microsomes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1987%26volume%3D36%26spage%3D229%26epage%3D235%26doi%3D10.1016%2F0006-2952%2887%2990694-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Conner, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, W. M.</span><span> </span><span class="NLM_article-title">Interactions of cytochrome P450s with their ligands</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.abb.2010.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.abb.2010.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20939998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFegtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2011&pages=56-65&author=K.+P.+Connerauthor=C.+Woodsauthor=W.+M.+Atkins&title=Interactions+of+cytochrome+P450s+with+their+ligands&doi=10.1016%2Fj.abb.2010.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions of cytochrome P 450s with their ligands</span></div><div class="casAuthors">Conner, Kip P.; Woods, Caleb M.; Atkins, William M.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-65</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P 450s (CYPs) are heme-contg. monooxygenases that contribute to an enormous range of enzymic function including biosynthetic and detoxification roles.  Here, the authors summarize recent studies concerning interactions of CYPs with ligands including substrates, inhibitors, and diat. heme-ligating mols.  These studies highlight the complexity in the relation between the heme spin state and active site occupancy, the roles of water in directing protein-ligand and ligand-heme interactions, and the details of interactions between heme and gaseous diat. CYP ligands.  Both kinetic and thermodn. aspects of ligand binding are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDC5lsgk8Hj7Vg90H21EOLACvtfcHk0ljjxx-E1E45bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFegtLk%253D&md5=8bf56701a4f12b52dbf3d569ef6731cf</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2010.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2010.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DConner%26aufirst%3DK.%2BP.%26aulast%3DWoods%26aufirst%3DC.%26aulast%3DAtkins%26aufirst%3DW.%2BM.%26atitle%3DInteractions%2520of%2520cytochrome%2520P450s%2520with%2520their%2520ligands%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2011%26volume%3D507%26spage%3D56%26epage%3D65%26doi%3D10.1016%2Fj.abb.2010.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Mycobacterium tuberculosis CYP130: Crystal structure, biophysical characterization, and interactions with antifungal azole drugs</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">5069</span><span class="NLM_x">–</span> <span class="NLM_lpage">5080</span><span class="refDoi"> DOI: 10.1074/jbc.M708734200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M708734200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=5069-5080&author=H.+Ouelletauthor=L.+M.+Podustauthor=P.+R.+Ortiz+de+Montellano&title=Mycobacterium+tuberculosis+CYP130%3A+Crystal+structure%2C+biophysical+characterization%2C+and+interactions+with+antifungal+azole+drugs&doi=10.1074%2Fjbc.M708734200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M708734200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M708734200%26sid%3Dliteratum%253Aachs%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DMycobacterium%2520tuberculosis%2520CYP130%253A%2520Crystal%2520structure%252C%2520biophysical%2520characterization%252C%2520and%2520interactions%2520with%2520antifungal%2520azole%2520drugs%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D5069%26epage%3D5080%26doi%3D10.1074%2Fjbc.M708734200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Wheeler, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Origin of substituent effects in edge-to-face aryl–aryl interactions</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">749</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1080/00268970802537614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1080%2F00268970802537614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20046948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotlWltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2009&pages=749-760&author=S.+E.+Wheelerauthor=K.+N.+Houk&title=Origin+of+substituent+effects+in+edge-to-face+aryl%E2%80%93aryl+interactions&doi=10.1080%2F00268970802537614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Origin of substituent effects in edge-to-face aryl-aryl interactions</span></div><div class="casAuthors">Wheeler, Steven E.; Houk, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">8-12</span>),
    <span class="NLM_cas:pages">749-760</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Substituent effects in the edge-to-face configuration of the benzene dimer have been studied using modern d. functional theory.  An accurate interaction potential energy curve has been computed for the unsubstituted dimer using ab initio methods with large basis sets.  The recommended binding energy for the edge-to-face benzene dimer is 2.31 kcal mol-1, estd. at the counterpoise-cor. CCSD(T)/aug-cc-pVTZ level of theory.  For both edge-ring and face-ring-substituted dimers, interaction energies correlate with σm for the substituents, indicating that substituent effects can be understood qual. in terms of simple electrostatic effects, although in the latter case dispersion results in some scatter in the data.  In contrast to prevailing models of substituent effects in benzene dimers, polarization of the π-system of the substituted ring does not induce substituent effects.  For edge-ring-substituted dimers, substituent effects arise from differential electrostatic interactions between the hydrogens on the substituted ring and the π-cloud of the face ring and direct interactions of the substituents with the unsubstituted ring.  For face-ring-substituted dimers, substituent effects arise from direct electrostatic and dispersion interactions of the substituents with the edge ring.  Substituents with σm > 0.12 favor edge ring substitution while for σm < 0.12 substitution on the face ring is preferred.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo86TqFLnFfV7Vg90H21EOLACvtfcHk0ljjxx-E1E45bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotlWltb8%253D&md5=07465d5ed7b46f562c4b746c839f0bb9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1080%2F00268970802537614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268970802537614%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DS.%2BE.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DOrigin%2520of%2520substituent%2520effects%2520in%2520edge-to-face%2520aryl%25E2%2580%2593aryl%2520interactions%26jtitle%3DMol.%2520Phys.%26date%3D2009%26volume%3D107%26spage%3D749%26epage%3D760%26doi%3D10.1080%2F00268970802537614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserü, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lh8cpVzK3zesQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Kries, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Interaction of Mycobacterium tuberculosis CYP130 with heterocyclic arylamines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">25211</span><span class="NLM_x">–</span> <span class="NLM_lpage">25219</span><span class="refDoi"> DOI: 10.1074/jbc.M109.017632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.017632" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=25211-25219&author=L.+M.+Podustauthor=H.+Ouelletauthor=J.+P.+von+Kriesauthor=P.+R.+Ortiz+de+Montellano&title=Interaction+of+Mycobacterium+tuberculosis+CYP130+with+heterocyclic+arylamines&doi=10.1074%2Fjbc.M109.017632"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.017632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.017632%26sid%3Dliteratum%253Aachs%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3Dvon%2BKries%26aufirst%3DJ.%2BP.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DInteraction%2520of%2520Mycobacterium%2520tuberculosis%2520CYP130%2520with%2520heterocyclic%2520arylamines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D25211%26epage%3D25219%26doi%3D10.1074%2Fjbc.M109.017632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Fonvielle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Du, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lequin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grach, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gondry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belin, P.</span><span> </span><span class="NLM_article-title">Substrate and reaction specificity of Mycobacterium tuberculosis cytochrome P450 CYP121: Insights from biochemical studies and crystal structures</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">17347</span><span class="NLM_x">–</span> <span class="NLM_lpage">17359</span><span class="refDoi"> DOI: 10.1074/jbc.M112.443853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M112.443853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=23620594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFamtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=17347-17359&author=M.+Fonvielleauthor=M.+H.+Le+Duauthor=O.+Lequinauthor=A.+Lecoqauthor=M.+Jacquetauthor=R.+Thaiauthor=S.+Duboisauthor=G.+Grachauthor=M.+Gondryauthor=P.+Belin&title=Substrate+and+reaction+specificity+of+Mycobacterium+tuberculosis+cytochrome+P450+CYP121%3A+Insights+from+biochemical+studies+and+crystal+structures&doi=10.1074%2Fjbc.M112.443853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate and Reaction Specificity of Mycobacterium tuberculosis Cytochrome P450 CYP121</span></div><div class="casAuthors">Fonvielle, Matthieu; Marie-Helene Le, D. u; Lequin, Olivier; Lecoq, Alain; Jacquet, Mickael; Thai, Robert; Dubois, Steven; Grach, Guillaume; Gondry, Muriel; Belin, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">17347-17359</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cytochrome P 450 CYP121 is essential for the viability of Mycobacterium tuberculosis.  Studies in vitro show that it can use the cyclodipeptide cyclo(L-Tyr-L-Tyr) (cYY) as a substrate.  We report an investigation of the substrate and reaction specificities of CYP121 involving anal. of the interaction between CYP121 and 14 cYY analogs with various modifications of the side chains or the diketopiperazine (DKP) ring.  Spectral titrn. expts. show that CYP121 significantly bound only cyclodipeptides with a conserved DKP ring carrying two aryl side chains in L-configuration.  CYP121 did not efficiently or selectively transform any of the cYY analogs tested, indicating a high specificity for cYY.  The mol. determinants of this specificity were inferred from both crystal structures of CYP121-analog complexes solved at high resoln. and soln. NMR spectroscopy of the analogs.  Bound cYY or its analogs all displayed a similar set of contacts with CYP121 residues Asn85, Phe168, and Trp182.  The propensity of the cYY tyrosyl to point toward Arg386 was dependent on the presence of the DKP ring that limits the conformational freedom of the ligand.  The correct positioning of the hydroxyl of this tyrosyl was essential for conversion of cYY.  Thus, the specificity of CYP121 results from both a restricted binding specificity and a fine-tuned P 450 substrate relationship.  These results document the catalytic mechanism of CYP121 and improve our understanding of its function in vivo.  This work contributes to progress toward the design of inhibitors of this essential protein of M. tuberculosis that could be used for antituberculosis therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAjF8TY2rD_bVg90H21EOLACvtfcHk0lh8cpVzK3zesQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFamtbY%253D&md5=f27bb300ba1f5a1e66ff196fe1be113e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.443853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.443853%26sid%3Dliteratum%253Aachs%26aulast%3DFonvielle%26aufirst%3DM.%26aulast%3DLe%2BDu%26aufirst%3DM.%2BH.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DS.%26aulast%3DGrach%26aufirst%3DG.%26aulast%3DGondry%26aufirst%3DM.%26aulast%3DBelin%26aufirst%3DP.%26atitle%3DSubstrate%2520and%2520reaction%2520specificity%2520of%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520CYP121%253A%2520Insights%2520from%2520biochemical%2520studies%2520and%2520crystal%2520structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D17347%26epage%3D17359%26doi%3D10.1074%2Fjbc.M112.443853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ost, T. W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miles, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lysek, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moser, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutton, P. L.</span><span> </span><span class="NLM_article-title">P450 BM3: the very model of a modern flavocytochrome</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">250</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span><span class="refDoi"> DOI: 10.1016/S0968-0004(02)02086-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0968-0004%2802%2902086-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12076537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD38XktlSmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2002&pages=250-257&author=A.+W.+Munroauthor=D.+G.+Leysauthor=K.+J.+McLeanauthor=K.+R.+Marshallauthor=T.+W.+B.+Ostauthor=S.+Daffauthor=C.+S.+Milesauthor=S.+K.+Chapmanauthor=D.+A.+Lysekauthor=C.+C.+Moserauthor=C.+C.+Pageauthor=P.+L.+Dutton&title=P450+BM3%3A+the+very+model+of+a+modern+flavocytochrome&doi=10.1016%2FS0968-0004%2802%2902086-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">P 450 BM3: the very model of a modern flavocytochrome</span></div><div class="casAuthors">Munro, Andrew W.; Leys, David G.; McLean, Kirsty J.; Marshall, Ker R.; Ost, Tobias W. B.; Daff, Simon; Miles, Caroline S.; Chapman, Stephen K.; Lysek, Dominikus A.; Moser, Christopher C.; Page, Christopher C.; Dutton, P. Leslie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">250-257</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  Flavocytochrome P 450 BM3 is a bacterial P 450 system in which a fatty acid hydroxylase/monooxygenase P 450 is fused to a mammalian-like diflavin NADPH-cytochrome P 450 reductase in a single polypeptide.  The enzyme is sol. (unlike mammalian P 450 redox systems) and its fusion arrangement affords it the highest catalytic activity of any P 450 monooxygenase.  This review discusses the fundamental properties of P 450 BM3 and how progress with this model P 450 has affected the comprehension of P 450 systems in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT9WMdY1BB8bVg90H21EOLACvtfcHk0lg0acJ2YEOd8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktlSmsLw%253D&md5=f56df51e02f516fd26407169cbe8b3e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2802%2902086-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252802%252902086-8%26sid%3Dliteratum%253Aachs%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DLeys%26aufirst%3DD.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DOst%26aufirst%3DT.%2BW.%2BB.%26aulast%3DDaff%26aufirst%3DS.%26aulast%3DMiles%26aufirst%3DC.%2BS.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DLysek%26aufirst%3DD.%2BA.%26aulast%3DMoser%26aufirst%3DC.%2BC.%26aulast%3DPage%26aufirst%3DC.%2BC.%26aulast%3DDutton%26aufirst%3DP.%2BL.%26atitle%3DP450%2520BM3%253A%2520the%2520very%2520model%2520of%2520a%2520modern%2520flavocytochrome%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2002%26volume%3D27%26spage%3D250%26epage%3D257%26doi%3D10.1016%2FS0968-0004%2802%2902086-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Duffell, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matak-Vinković, D.</span><span> </span><span class="NLM_article-title">Nanoelectrospray ionization mass spectrometric study of Mycobacterium tuberculosis CYP121-ligand interactions</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">5707</span><span class="NLM_x">–</span> <span class="NLM_lpage">5714</span><span class="refDoi"> DOI: 10.1021/ac400236z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac400236z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2013&pages=5707-5714&author=K.+M.+Duffellauthor=S.+A.+Hudsonauthor=K.+J.+McLeanauthor=A.+W.+Munroauthor=C.+Abellauthor=D.+Matak-Vinkovi%C4%87&title=Nanoelectrospray+ionization+mass+spectrometric+study+of+Mycobacterium+tuberculosis+CYP121-ligand+interactions&doi=10.1021%2Fac400236z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fac400236z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac400236z%26sid%3Dliteratum%253Aachs%26aulast%3DDuffell%26aufirst%3DK.%2BM.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DMatak-Vinkovi%25C4%2587%26aufirst%3DD.%26atitle%3DNanoelectrospray%2520ionization%2520mass%2520spectrometric%2520study%2520of%2520Mycobacterium%2520tuberculosis%2520CYP121-ligand%2520interactions%26jtitle%3DAnal.%2520Chem.%26date%3D2013%26volume%3D85%26spage%3D5707%26epage%3D5714%26doi%3D10.1021%2Fac400236z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pikuleva, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafite, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigby, S. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Structural and biochemical characterization of Mycobacterium tuberculosis CYP142: evidence for multiple cholesterol 27-hydroxylase activities in a human pathogen</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">38270</span><span class="NLM_x">–</span> <span class="NLM_lpage">38282</span><span class="refDoi"> DOI: 10.1074/jbc.M110.164293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M110.164293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20889498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOku73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=38270-38282&author=M.+D.+Driscollauthor=K.+J.+McLeanauthor=C.+Levyauthor=N.+Mastauthor=I.+A.+Pikulevaauthor=P.+Lafiteauthor=S.+E.+J.+Rigbyauthor=D.+Leysauthor=A.+W.+Munro&title=Structural+and+biochemical+characterization+of+Mycobacterium+tuberculosis+CYP142%3A+evidence+for+multiple+cholesterol+27-hydroxylase+activities+in+a+human+pathogen&doi=10.1074%2Fjbc.M110.164293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Biochemical Characterization of Mycobacterium tuberculosis CYP142: Evidence for Multiple Cholesterol 27-Hydroxylase Activities in a Human Pathogen</span></div><div class="casAuthors">Driscoll, Max D.; McLean, Kirsty J.; Levy, Colin; Mast, Natalia; Pikuleva, Irina A.; Lafite, Pierre; Rigby, Stephen E. J.; Leys, David; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">38270-38282</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The Mycobacterium tuberculosis cytochrome P 450 enzyme CYP142 is encoded in a large gene cluster involved in metab. of host cholesterol.  CYP142 was expressed and purified as a sol., low spin P 450 hemoprotein.  CYP142 binds tightly to cholesterol and its oxidized deriv. cholest-4-en-3-one, with extensive shift of the heme iron to the high spin state.  High affinity for azole antibiotics was demonstrated, highlighting their therapeutic potential.  CYP142 catalyzes either 27-hydroxylation of cholesterol/cholest-4-en-3-one or generates 5-cholestenoic acid/cholest-4-en-3-one-27-oic acid from these substrates by successive sterol oxidns., with the catalytic outcome dependent on the redox partner system used.  The CYP142 crystal structure was solved to 1.6 Å, revealing a similar active site organization to the cholesterol-metabolizing M. tuberculosis CYP125, but having a near-identical organization of distal pocket residues to the branched fatty acid oxidizing M. tuberculosis CYP124.  The cholesterol oxidizing activity of CYP142 provides an explanation for previous findings that ΔCYP125 strains of Mycobacterium bovis and M. bovis BCG cannot grow on cholesterol, because these strains have a defective CYP142 gene.  CYP142 is revealed as a cholesterol 27-oxidase with likely roles in host response modulation and cholesterol metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsGGdzyIjIULVg90H21EOLACvtfcHk0lg0acJ2YEOd8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOku73O&md5=7c64404f457b85b229042e39c73347ad</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.164293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.164293%26sid%3Dliteratum%253Aachs%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DLevy%26aufirst%3DC.%26aulast%3DMast%26aufirst%3DN.%26aulast%3DPikuleva%26aufirst%3DI.%2BA.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DRigby%26aufirst%3DS.%2BE.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DStructural%2520and%2520biochemical%2520characterization%2520of%2520Mycobacterium%2520tuberculosis%2520CYP142%253A%2520evidence%2520for%2520multiple%2520cholesterol%252027-hydroxylase%2520activities%2520in%2520a%2520human%2520pathogen%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D38270%26epage%3D38282%26doi%3D10.1074%2Fjbc.M110.164293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Bellamine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangla, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nes, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterman, M. R.</span><span> </span><span class="NLM_article-title">Characterization and catalytic properties of the sterol 14alpha-demethylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">8937</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1073/pnas.96.16.8937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.96.16.8937" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=8937-8942&author=A.+Bellamineauthor=A.+T.+Manglaauthor=W.+D.+Nesauthor=M.+R.+Waterman&title=Characterization+and+catalytic+properties+of+the+sterol+14alpha-demethylase+from+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.96.16.8937"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.16.8937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.16.8937%26sid%3Dliteratum%253Aachs%26aulast%3DBellamine%26aufirst%3DA.%26aulast%3DMangla%26aufirst%3DA.%2BT.%26aulast%3DNes%26aufirst%3DW.%2BD.%26aulast%3DWaterman%26aufirst%3DM.%2BR.%26atitle%3DCharacterization%2520and%2520catalytic%2520properties%2520of%2520the%2520sterol%252014alpha-demethylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D8937%26epage%3D8942%26doi%3D10.1073%2Fpnas.96.16.8937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warman, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seward, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girvan, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Biophysical characterization of the sterol demethylase P450 from Mycobacterium tuberculosis, its cognate ferredoxin, and their interactions</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">8427</span><span class="NLM_x">–</span> <span class="NLM_lpage">8443</span><span class="refDoi"> DOI: 10.1021/bi0601609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0601609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsFOqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=8427-8443&author=K.+J.+McLeanauthor=A.+J.+Warmanauthor=H.+E.+Sewardauthor=K.+R.+Marshallauthor=H.+M.+Girvanauthor=M.+R.+Cheesmanauthor=M.+R.+Watermanauthor=A.+W.+Munro&title=Biophysical+characterization+of+the+sterol+demethylase+P450+from+Mycobacterium+tuberculosis%2C+its+cognate+ferredoxin%2C+and+their+interactions&doi=10.1021%2Fbi0601609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical Characterization of the Sterol Demethylase P450 from Mycobacterium tuberculosis, Its Cognate Ferredoxin, and Their Interactions</span></div><div class="casAuthors">McLean, Kirsty J.; Warman, Ashley J.; Seward, Harriet E.; Marshall, Ker R.; Girvan, Hazel M.; Cheesman, Myles R.; Waterman, Michael R.; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8427-8443</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mycobacterium tuberculosis encodes a P 450 of the sterol demethylase family (CYP51) chromosomally located adjacent to a ferredoxin (Fdx).  CYP51 and Fdx were purified to homogeneity and characterized.  Spectroscopic analyses were consistent with cysteinate- and aqua-ligated heme iron in CYP51.  An ε419 of 134 mM-1 cm-1 was detd. for oxidized CYP51.  Anal. of interactions of 1-, 2-, and 4-phenylimidazoles with CYP51 showed that the 1- and 4-forms were heme iron-coordinating inhibitors, while 2-phenylimidazole induced a substrate-like optical shift.  The 2-phenyimidazole-bound CYP51 demonstrated unusual decreases in high-spin heme iron content at elevated temps. and an almost complete absence of high-spin heme iron by low-temp. EPR.  These data suggest thermally induced alterations in CYP51 active site structure and/or binding modes for the small ligand.  Redn. of CYP51 in the presence of carbon monoxide leads to formation of an Fe(II)-CO complex with a Soret absorption max. at 448.5 nm, which collapses (at 0.246 min-1 at pH 7.0) forming a species with a Soret max. at 421.5 nm (the inactive P420 form).  The rate of P420 formation is accelerated at lower pH, consistent with protonation of the cysteinate (Cys 394) to a thiol underlying the P 450-P420 transition.  The P 450 form is stabilized by estriol, which induces a type I spectral shift on binding CYP51 (Kd = 21.7 μM).  Nonstandard spectral changes occur on CYP51 redn. (using either dithionite or natural redox partners), including a blue-shifted Soret band and development of a strong feature at ∼558.5 nm, suggestive of cysteine thiol ligation.  Thus, ligand-free ferrous CYP51 is prone to thiolate ligand protonation even in the absence of carbon monoxide.  Anal. of reoxidized CYP51 demonstrates that the enzyme re-forms P 450, indicating that Cys 394 thiol is readily deprotonated to thiolate in the ferric form.  Spectroscopic anal. of Fdx by EPR (resonance at g = 2.03) and magnetic CD (intensity for oxidized and reduced forms and signal intensity dependence on field strength and temp.) demonstrated that Fdx binds a [3Fe-4S] iron-sulfur cluster.  Potentiometric studies show that the midpoint potential for ligand-free CYP51 is -375 mV, increasing to -225 mV in the estriol-bound form.  The Fdx potential is -31 mV.  Fdx forms a productive electron transfer complex with CYP51 and reduces it at a rate of 3.0 min-1 in the ligand-free form and 4.3 min-1 in the estriol-bound form, despite a thermodn. barrier.  Steady-state anal. of a M. tuberculosis class I redox system comprising flavoprotein reductase A (FprA), Fdx, and estriol-bound CYP51 indicates heme iron redn. as a rate-limiting step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNrTneY87rx7Vg90H21EOLACvtfcHk0lgTlXwp6czgMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsFOqs74%253D&md5=cdf20fe98cb285065fce7b5777108d14</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi0601609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0601609%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DWarman%26aufirst%3DA.%2BJ.%26aulast%3DSeward%26aufirst%3DH.%2BE.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DGirvan%26aufirst%3DH.%2BM.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DWaterman%26aufirst%3DM.%2BR.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DBiophysical%2520characterization%2520of%2520the%2520sterol%2520demethylase%2520P450%2520from%2520Mycobacterium%2520tuberculosis%252C%2520its%2520cognate%2520ferredoxin%252C%2520and%2520their%2520interactions%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D8427%26epage%3D8443%26doi%3D10.1021%2Fbi0601609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Martin, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blachly, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, S. M.</span><span> </span><span class="NLM_article-title">Exploring the influence of the protein environment on metal-binding pharmacophores</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7126</span><span class="NLM_x">–</span> <span class="NLM_lpage">7135</span><span class="refDoi"> DOI: 10.1021/jm500984b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500984b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7126-7135&author=D.+P.+Martinauthor=P.+G.+Blachlyauthor=J.+A.+McCammonauthor=S.+M.+Cohen&title=Exploring+the+influence+of+the+protein+environment+on+metal-binding+pharmacophores&doi=10.1021%2Fjm500984b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm500984b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500984b%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BP.%26aulast%3DBlachly%26aufirst%3DP.%2BG.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DExploring%2520the%2520influence%2520of%2520the%2520protein%2520environment%2520on%2520metal-binding%2520pharmacophores%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7126%26epage%3D7135%26doi%3D10.1021%2Fjm500984b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Lead optimization and structure–activity relationships for reversible inhibitors</span>. In  <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2F9781118540398.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DLead%2520optimization%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520for%2520reversible%2520inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhbitors</span> <span class="citation_source-journal">Biochem. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span><span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhbitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lgTlXwp6czgMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhbitors%26jtitle%3DBiochem.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Hu, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eftink, M. R.</span><span> </span><span class="NLM_article-title">Thermodynamic studies of the interaction of trp aporepressor with tryptophan analogs</span> <span class="citation_source-journal">Biophys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/0301-4622(93)E0073-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0301-4622%2893%29E0073-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=8018820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFWmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1994&pages=233-239&author=D.+D.+Huauthor=M.+R.+Eftink&title=Thermodynamic+studies+of+the+interaction+of+trp+aporepressor+with+tryptophan+analogs&doi=10.1016%2F0301-4622%2893%29E0073-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic studies of the interaction of trp aporepressor with tryptophan analogs</span></div><div class="casAuthors">Hu, Dana D.; Eftink, Maurice R.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-9</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    </div><div class="casAbstract">The assocn. of L-tryptophan and some of its analogs, including 3 conformationally restricted analogs, with trp aporepressor (apo trpR) was studied by isothermal titrn. microcalorimetry.  Contributions of the functional groups of a ligand to the free energy change, ΔG°', and enthalpy change, ΔH°', of the interaction were evaluated on a mol. basis.  Analogs without the α-amino group (i.e. desamino analogs) bind with a slightly higher affinity to the protein.  On the other hand, descarboxy analogs show weaker binding to the apo trpR.  In addn., there exists enthalpy-entropy compensation for the assocn. of the congener series of ligands with the protein.  The entropy change, ΔS°', appears to play a more important role in the binding of the conformationally restricted analogs than in the binding of L-tryptophan and the unlocked ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6OPuEqrHTdbVg90H21EOLACvtfcHk0lgTlXwp6czgMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFWmsL4%253D&md5=4a6db76073796876d92fb77dcfc8dcae</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0301-4622%2893%29E0073-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0301-4622%252893%2529E0073-E%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DD.%2BD.%26aulast%3DEftink%26aufirst%3DM.%2BR.%26atitle%3DThermodynamic%2520studies%2520of%2520the%2520interaction%2520of%2520trp%2520aporepressor%2520with%2520tryptophan%2520analogs%26jtitle%3DBiophys.%2520Chem.%26date%3D1994%26volume%3D49%26spage%3D233%26epage%3D239%26doi%3D10.1016%2F0301-4622%2893%29E0073-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span>Inhibitor screening against human cytochrome P450s was outsourced to Cyprotex Discovery Ltd., Macclesfield, Cheshire, UK.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Inhibitor+screening+against+human+cytochrome+P450s+was+outsourced+to+Cyprotex+Discovery+Ltd.%2C+Macclesfield%2C+Cheshire%2C+UK."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Mirza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynisson, J.</span><span> </span><span class="NLM_article-title">Known drug space as a metric in exploring the boundaries of drug-like chemical space</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">5006</span><span class="NLM_x">–</span> <span class="NLM_lpage">5011</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2009.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.ejmech.2009.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19782440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCksLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2009&pages=5006-5011&author=A.+Mirzaauthor=R.+Desaiauthor=J.+Reynisson&title=Known+drug+space+as+a+metric+in+exploring+the+boundaries+of+drug-like+chemical+space&doi=10.1016%2Fj.ejmech.2009.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Known drug space as a metric in exploring the boundaries of drug-like chemical space</span></div><div class="casAuthors">Mirza, Amin; Desai, Radha; Reynisson, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5006-5011</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Marketed drug compds. (or known drug space) were used as a metric to test the principles of eliminating parent structures of the nitrenium ion (aryl-amine/nitro compds.) as well as sulfur and halogen contg. mols. from screening compd. collections.  Mols. contg. such moieties and/or atoms have biol. and physiochem. properties, which possibly make them less attractive as leads in drug development.  Precursors to the nitrenium ion were relatively abundant in known drug space at 14%.  Thus, their simple elimination from drug-like chem. space is not advisable.  Interestingly, the mutagenic potential of the nitrenium ions is linked to their stability and quantum mech. calcns. can be used to est. it.  Furthermore, 24% of drugs investigated contained sulfur atoms and around 28% were halogenated.  As some sulfur contg. moieties were abundant while others were scarce, it was deduced that it would be more effective to eliminate specific mol. scaffolds rather than all sulfur contg. mols.  In conclusion, it has been shown that by statistically analyzing known drug space a better understanding of the boundaries of drug-like chem. space was established which can help medicinal chemists in finding rewarding regions of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodBhMTaj39JbVg90H21EOLACvtfcHk0lh89W8ZimTZeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCksLnJ&md5=30408326c9c7521db094f43d8c285a5e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2009.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2009.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DDesai%26aufirst%3DR.%26aulast%3DReynisson%26aufirst%3DJ.%26atitle%3DKnown%2520drug%2520space%2520as%2520a%2520metric%2520in%2520exploring%2520the%2520boundaries%2520of%2520drug-like%2520chemical%2520space%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2009%26volume%3D44%26spage%3D5006%26epage%3D5011%26doi%3D10.1016%2Fj.ejmech.2009.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guengerich, F. P.</span><span> </span><span class="NLM_article-title">Cytochrome P450 activation of arylamines and heterocyclic amines</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1146/annurev.pharmtox.45.120403.100010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1146%2Fannurev.pharmtox.45.120403.100010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=15822170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=27-49&author=D.+Kimauthor=F.+P.+Guengerich&title=Cytochrome+P450+activation+of+arylamines+and+heterocyclic+amines&doi=10.1146%2Fannurev.pharmtox.45.120403.100010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 activation of arylamines and heterocyclic amines</span></div><div class="casAuthors">Kim, Donghak; Guengerich, F. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-49</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Arylamines and heterocyclic arylamines (HAAs) are of particular interest because of demonstrated carcinogenicity in animals and humans and the broad exposure to many of these compds.  The activation of these, and also some arylamine drugs, involves N-hydroxylation, usually by cytochrome P 450 (P 450).  P 450 1A2 plays a prominent role in these reactions.  However, P 450 1A1 and 1B1 and other P450s are also important in humans as well as exptl. animals.  Some arylamines (including drugs) are N-hydroxylated predominantly by P450s other than those in Family 1.  Other oxygenases can also have roles.  An important issue is extrapolation between species in predicting cancer risks, as shown by the low rates of HAA activation by rat P 450 1A2 and low levels of P 450 1A2 expression in some nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6INgRLGF4_7Vg90H21EOLACvtfcHk0ljz9ePcbJnPbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrg%253D&md5=1fc485c6e77e6244b02fd9421f3fa2ed</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.45.120403.100010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.45.120403.100010%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DCytochrome%2520P450%2520activation%2520of%2520arylamines%2520and%2520heterocyclic%2520amines%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D45%26spage%3D27%26epage%3D49%26doi%3D10.1146%2Fannurev.pharmtox.45.120403.100010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvet, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieira, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunatilleke, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">R-configuration of 4-aminopyridyl-based inhibitors of CYP51 confers superior efficacy against Trypanosoma cruzi</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">434</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1021/ml500010m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500010m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=434-439&author=J.+Y.+Choiauthor=C.+M.+Calvetauthor=D.+F.+Vieiraauthor=S.+S.+Gunatillekeauthor=M.+D.+Cameronauthor=J.+H.+McKerrowauthor=L.+M.+Podustauthor=W.+R.+Roush&title=R-configuration+of+4-aminopyridyl-based+inhibitors+of+CYP51+confers+superior+efficacy+against+Trypanosoma+cruzi&doi=10.1021%2Fml500010m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fml500010m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500010m%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DCalvet%26aufirst%3DC.%2BM.%26aulast%3DVieira%26aufirst%3DD.%2BF.%26aulast%3DGunatilleke%26aufirst%3DS.%2BS.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DR-configuration%2520of%25204-aminopyridyl-based%2520inhibitors%2520of%2520CYP51%2520confers%2520superior%2520efficacy%2520against%2520Trypanosoma%2520cruzi%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D434%26epage%3D439%26doi%3D10.1021%2Fml500010m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvet, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunatilleke, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">Rational development of 4-aminopyridyl-based inhibitors targeting Trypanosoma cruzi CYP51 as anti-chagas agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7651</span><span class="NLM_x">–</span> <span class="NLM_lpage">7668</span><span class="refDoi"> DOI: 10.1021/jm401067s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401067s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7651-7668&author=J.+Y.+Choiauthor=C.+M.+Calvetauthor=S.+S.+Gunatillekeauthor=C.+Ruizauthor=M.+D.+Cameronauthor=J.+H.+McKerrowauthor=L.+M.+Podustauthor=W.+R.+Roush&title=Rational+development+of+4-aminopyridyl-based+inhibitors+targeting+Trypanosoma+cruzi+CYP51+as+anti-chagas+agents&doi=10.1021%2Fjm401067s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm401067s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401067s%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DCalvet%26aufirst%3DC.%2BM.%26aulast%3DGunatilleke%26aufirst%3DS.%2BS.%26aulast%3DRuiz%26aufirst%3DC.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DRational%2520development%2520of%25204-aminopyridyl-based%2520inhibitors%2520targeting%2520Trypanosoma%2520cruzi%2520CYP51%2520as%2520anti-chagas%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7651%26epage%3D7668%26doi%3D10.1021%2Fjm401067s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Brennan, P. J.</span><span> </span><span class="NLM_article-title">Structure, function, and biogenesis of the cell wall of Mycobacterium tuberculosis</span> <span class="citation_source-journal">Tuberculosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span><span class="refDoi"> DOI: 10.1016/S1472-9792(02)00089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS1472-9792%2802%2900089-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12758196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD3s3js1Kgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2003&pages=91-97&author=P.+J.+Brennan&title=Structure%2C+function%2C+and+biogenesis+of+the+cell+wall+of+Mycobacterium+tuberculosis&doi=10.1016%2FS1472-9792%2802%2900089-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function, and biogenesis of the cell wall of Mycobacterium tuberculosis</span></div><div class="casAuthors">Brennan P J</div><div class="citationInfo"><span class="NLM_cas:title">Tuberculosis (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">91-7</span>
        ISSN:<span class="NLM_cas:issn">1472-9792</span>.
    </div><div class="casAbstract">Much of the early structural definition of the cell wall of Mycobacterium spp. was initiated in the 1960s and 1970s.  There was a long period of inactivity, but more recent developments in NMR and mass spectral analysis and definition of the M. tuberculosis genome have resulted in a thorough understanding, not only of the structure of the mycobacterial cell wall and its lipids but also the basic genetics and biosynthesis.  Our understanding nowadays of cell-wall architecture amounts to a massive "core" comprised of peptidoglycan covalently attached via a linker unit (L-Rha-D-GlcNAc-P) to a linear galactofuran, in turn attached to several strands of a highly branched arabinofuran, in turn attached to mycolic acids.  The mycolic acids are oriented perpendicular to the plane of the membrane and provide a truly special lipid barrier responsible for many of the physiological and disease-inducing aspects of M. tuberculosis.  Intercalated within this lipid environment are the lipids that have intrigued researchers for over five decades: the phthiocerol dimycocerosate, cord factor/dimycolyltrehalose, the sulfolipids, the phosphatidylinositol mannosides, etc.  Knowledge of their roles in "signaling" events, in pathogenesis, and in the immune response is now emerging, sometimes piecemeal and sometimes in an organized fashion.  Some of the more intriguing observations are those demonstrating that mycolic acids are recognized by CD1-restricted T-cells, that antigen 85, one of the most powerful protective antigens of M. tuberculosis, is a mycolyltransferase, and that lipoarabinomannan (LAM), when "capped" with short mannose oligosaccharides, is involved in phagocytosis of M. tuberculosis.  Definition of the genome of M. tuberculosis has greatly aided efforts to define the biosynthetic pathways for all of these exotic molecules: the mycolic acids, the mycocerosates, phthiocerol, LAM, and the polyprenyl phosphates.  For example, we know that synthesis of the entire core is initiated on a decaprenyl-P with synthesis of the linker unit, and then there is concomitant extension of the galactan and arabinan chains while this intermediate is transported through the cytoplasmic membrane.  The final steps in these events, the attachment of mycolic acids and ligation to peptidoglycan, await definition and will prove to be excellent targets for a new generation of anti-tuberculosis drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmjBRWYNdwmJPlX7E30qdzfW6udTcc2eZlHybA5b47-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3js1Kgsw%253D%253D&md5=7c8428a77e3aef5bce2db8c9e54ba0a9</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2FS1472-9792%2802%2900089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1472-9792%252802%252900089-6%26sid%3Dliteratum%253Aachs%26aulast%3DBrennan%26aufirst%3DP.%2BJ.%26atitle%3DStructure%252C%2520function%252C%2520and%2520biogenesis%2520of%2520the%2520cell%2520wall%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DTuberculosis%26date%3D2003%26volume%3D83%26spage%3D91%26epage%3D97%26doi%3D10.1016%2FS1472-9792%2802%2900089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">De Rossi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aínsa, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riccardi, G.</span><span> </span><span class="NLM_article-title">Role of mycobacterial efflux transporters in drug resistance: An unresolved question</span> <span class="citation_source-journal">FEMS Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">36</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1111/j.1574-6976.2005.00002.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1111%2Fj.1574-6976.2005.00002.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=16438679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=36-52&author=E.+De+Rossiauthor=J.+A.+A%C3%ADnsaauthor=G.+Riccardi&title=Role+of+mycobacterial+efflux+transporters+in+drug+resistance%3A+An+unresolved+question&doi=10.1111%2Fj.1574-6976.2005.00002.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Role of mycobacterial efflux transporters in drug resistance: an unresolved question</span></div><div class="casAuthors">De Rossi, Edda; Ainsa, Jose A.; Riccardi, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-52</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Two mechanisms are thought to be involved in the natural drug resistance of mycobacteria: the mycobacterial cell wall permeability barrier and active multidrug efflux pumps.  Genes encoding drug efflux transporters have been isolated from several mycobacterial species.  These proteins transport tetracycline, fluoroquinolones, aminoglycosides and other compds.  Recent reports have suggested that efflux pumps may also be involved in transporting isoniazid, one of the main drugs used to treat tuberculosis.  This review highlights recent advances in our understanding of efflux-mediated drug resistance in mycobacteria, including the distribution of efflux systems in these organisms, their substrate profiles and their contribution to drug resistance.  The balance between the drug transport into the cell and drug efflux is not yet clearly understood, and further studies are required in mycobacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwjdJiKjbO7Vg90H21EOLACvtfcHk0lhOnZLQF5GN0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D&md5=41af6cedab317d501c489400a6979920</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2005.00002.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2005.00002.x%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BRossi%26aufirst%3DE.%26aulast%3DA%25C3%25ADnsa%26aufirst%3DJ.%2BA.%26aulast%3DRiccardi%26aufirst%3DG.%26atitle%3DRole%2520of%2520mycobacterial%2520efflux%2520transporters%2520in%2520drug%2520resistance%253A%2520An%2520unresolved%2520question%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D30%26spage%3D36%26epage%3D52%26doi%3D10.1111%2Fj.1574-6976.2005.00002.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Martin, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blachly, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marts, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, C. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tierney, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, S. M.</span><span> </span><span class="NLM_article-title">Unconventional” coordination chemistry by metal chelating fragments in a metalloprotein active site</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">5400</span><span class="NLM_x">–</span> <span class="NLM_lpage">5406</span><span class="refDoi"> DOI: 10.1021/ja500616m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja500616m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=5400-5406&author=D.+P.+Martinauthor=P.+G.+Blachlyauthor=A.+R.+Martsauthor=T.+M.+Woodruffauthor=C.+A.+F.+de+Oliveiraauthor=J.+A.+McCammonauthor=D.+L.+Tierneyauthor=S.+M.+Cohen&title=Unconventional%E2%80%9D+coordination+chemistry+by+metal+chelating+fragments+in+a+metalloprotein+active+site&doi=10.1021%2Fja500616m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fja500616m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja500616m%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BP.%26aulast%3DBlachly%26aufirst%3DP.%2BG.%26aulast%3DMarts%26aufirst%3DA.%2BR.%26aulast%3DWoodruff%26aufirst%3DT.%2BM.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%2BA.%2BF.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DTierney%26aufirst%3DD.%2BL.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DUnconventional%25E2%2580%259D%2520coordination%2520chemistry%2520by%2520metal%2520chelating%2520fragments%2520in%2520a%2520metalloprotein%2520active%2520site%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D5400%26epage%3D5406%26doi%3D10.1021%2Fja500616m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashalidis, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Biofragments: An approach towards predicting protein function using biologically related fragments and its application to mycobacterium tuberculosis CYP126</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">555</span><span class="refDoi"> DOI: 10.1002/cbic.201300697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcbic.201300697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=24677424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFaiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=549-555&author=S.+A.+Hudsonauthor=E.+H.+Mashalidisauthor=A.+Benderauthor=K.+J.+McLeanauthor=A.+W.+Munroauthor=C.+Abell&title=Biofragments%3A+An+approach+towards+predicting+protein+function+using+biologically+related+fragments+and+its+application+to+mycobacterium+tuberculosis+CYP126&doi=10.1002%2Fcbic.201300697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biofragments: An Approach towards Predicting Protein Function Using Biologically Related Fragments and its Application to Mycobacterium tuberculosis CYP126</span></div><div class="casAuthors">Hudson, Sean A.; Mashalidis, Ellene H.; Bender, Andreas; McLean, Kirsty J.; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We present a novel fragment-based approach that tackles some of the challenges for chem. biol. of predicting protein function.  The general approach, which we have termed biofragments, comprises two key stages.  First, a biol. relevant fragment library (biofragment library) can be designed and constructed from known sets of substrate-like ligands for a protein class of interest.  Second, the library can be screened for binding to a novel putative ligand-binding protein from the same or similar class, and the characterization of hits provides insight into the basis of ligand recognition, selectivity, and function at the substrate level.  As a proof-of-concept, we applied the biofragments approach to the functionally uncharacterized Mycobacterium tuberculosis (Mtb) cytochrome P 450 isoform, CYP126.  This led to the development of a tailored CYP biofragment library with notable 3D characteristics and a significantly higher screening hit rate (14 %) than std. drug-like fragment libraries screened previously against Mtb CYP121 and 125 (4 % and 1 %, resp.).  Biofragment hits were identified that make both substrate-like type-I and inhibitor-like type-II interactions with CYP126.  A chem.-fingerprint-based substrate model was built from the hits and used to search a virtual TB metabolome, which led to the discovery that CYP126 has a strong preference for the recognition of aroms. and substrate-like type-I binding of chlorophenol moieties within the active site near the heme.  Future catalytic analyses will be focused on assessing CYP126 for potential substrate oxidative dehalogenation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3kRhqwkn5LVg90H21EOLACvtfcHk0lhOnZLQF5GN0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFaiur0%253D&md5=4307140caec78e91c17088e0354fb2fe</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300697%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DBiofragments%253A%2520An%2520approach%2520towards%2520predicting%2520protein%2520function%2520using%2520biologically%2520related%2520fragments%2520and%2520its%2520application%2520to%2520mycobacterium%2520tuberculosis%2520CYP126%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D549%26epage%3D555%26doi%3D10.1002%2Fcbic.201300697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 34 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jakob S. Pallesen, Dilip Narayanan, Kim T. Tran, Sara M. Ø. Solbak, Giuseppe Marseglia, Louis M. E. Sørensen, Lars J. Høj, Federico Munafò, Rosa M. C. Carmona, Anthony D. Garcia, Haritha L. Desu, Roberta Brambilla, Tommy N. Johansen, Grzegorz M. Popowicz, Michael Sattler, Michael Gajhede, <span class="NLM_string-name hlFld-ContribAuthor">Anders Bach</span>. </span><span class="cited-content_cbyCitation_article-title">Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4623-4661. <a href="https://doi.org/10.1021/acs.jmedchem.0c02094" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDeconstructing%252BNoncovalent%252BKelch-like%252BECH-Associated%252BProtein%252B1%252B%252528Keap1%252529%252BInhibitors%252Binto%252BFragments%252Bto%252BReconstruct%252BNew%252BPotent%252BCompounds%26aulast%3DPallesen%26aufirst%3DJakob%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03122020%26date%3D05042021%26volume%3D64%26issue%3D8%26spage%3D4623%26epage%3D4661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ruiheng Zhu, Ramesh Reddy, Man Ding, Ming Xu, Chaoyi Deng, Sam Tadayon, Hui Li, Kristopher Depew, <span class="NLM_string-name hlFld-ContribAuthor">Benjamin Lane</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Scale-Up of a Continuous Manufacturing Process for a Hydrazine Condensation Reaction. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research & Development</span><span> <strong>2021,</strong> <em>25 </em>
                                    (2)
                                     , 199-205. <a href="https://doi.org/10.1021/acs.oprd.0c00424" title="DOI URL">https://doi.org/10.1021/acs.oprd.0c00424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.oprd.0c00424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.oprd.0c00424%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DDevelopment%252Band%252BScale-Up%252Bof%252Ba%252BContinuous%252BManufacturing%252BProcess%252Bfor%252Ba%252BHydrazine%252BCondensation%252BReaction%26aulast%3DZhu%26aufirst%3DRuiheng%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24092020%26date%3D11012021%26volume%3D25%26issue%3D2%26spage%3D199%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sunnia Rajput, Kirsty J. McLean, Harshwardhan Poddar, Irwin R. Selvam, Gayathri Nagalingam, James A. Triccas, Colin W. Levy, Andrew W. Munro, <span class="NLM_string-name hlFld-ContribAuthor">Craig A. Hutton</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships of cyclo(l-Tyrosyl-l-tyrosine) Derivatives Binding to Mycobacterium tuberculosis CYP121: Iodinated Analogues Promote Shift to High-Spin Adduct. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9792-9805. <a href="https://doi.org/10.1021/acs.jmedchem.9b01199" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01199%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252Bof%252Bcyclo%252528l-Tyrosyl-l-tyrosine%252529%252BDerivatives%252BBinding%252Bto%252BMycobacterium%252Btuberculosis%252BCYP121%25253A%252BIodinated%252BAnalogues%252BPromote%252BShift%252Bto%252BHigh-Spin%252BAdduct%26aulast%3DRajput%26aufirst%3DSunnia%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D24072019%26date%3D31102019%26date%3D16102019%26volume%3D62%26issue%3D21%26spage%3D9792%26epage%3D9805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher N. Johnson, Daniel A. Erlanson, Wolfgang Jahnke, Paul N. Mortenson, <span class="NLM_string-name hlFld-ContribAuthor">David C. Rees</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2016. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (5)
                                     , 1774-1784. <a href="https://doi.org/10.1021/acs.jmedchem.7b01298" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01298%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2016%26aulast%3DJohnson%26aufirst%3DChristopher%2BN.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D31082017%26date%3D31102017%26volume%3D61%26issue%3D5%26spage%3D1774%26epage%3D1784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew J.  Fielding</span>, <span class="hlFld-ContribAuthor ">Kednerlin  Dornevil</span>, <span class="hlFld-ContribAuthor ">Li  Ma</span>, <span class="hlFld-ContribAuthor ">Ian  Davis</span>, and <span class="hlFld-ContribAuthor ">Aimin  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Probing Ligand Exchange in the P450 Enzyme CYP121 from Mycobacterium tuberculosis: Dynamic Equilibrium of the Distal Heme Ligand as a Function of pH and Temperature. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2017,</strong> <em>139 </em>
                                    (48)
                                     , 17484-17499. <a href="https://doi.org/10.1021/jacs.7b08911" title="DOI URL">https://doi.org/10.1021/jacs.7b08911</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.7b08911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.7b08911%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DProbing%252BLigand%252BExchange%252Bin%252Bthe%252BP450%252BEnzyme%252BCYP121%252Bfrom%252BMycobacterium%252Btuberculosis%25253A%252BDynamic%252BEquilibrium%252Bof%252Bthe%252BDistal%252BHeme%252BLigand%252Bas%252Ba%252BFunction%252Bof%252BpH%252Band%252BTemperature%26aulast%3DFielding%26aufirst%3DAndrew%2BJ.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24082017%26date%3D20112017%26date%3D06122017%26date%3D01112017%26volume%3D139%26issue%3D48%26spage%3D17484%26epage%3D17499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Madeline E.  Kavanagh</span>, <span class="hlFld-ContribAuthor ">Jude  Chenge</span>, <span class="hlFld-ContribAuthor ">Azedine  Zoufir</span>, <span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>, and <span class="hlFld-ContribAuthor ">Chris  Abell</span>  . </span><span class="cited-content_cbyCitation_article-title">Fragment Profiling Approach to Inhibitors of the Orphan M. tuberculosis P450 CYP144A1. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2017,</strong> <em>56 </em>
                                    (11)
                                     , 1559-1572. <a href="https://doi.org/10.1021/acs.biochem.6b00954" title="DOI URL">https://doi.org/10.1021/acs.biochem.6b00954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.6b00954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.6b00954%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DFragment%252BProfiling%252BApproach%252Bto%252BInhibitors%252Bof%252Bthe%252BOrphan%252BM.%252Btuberculosis%252BP450%252BCYP144A1%26aulast%3DKavanagh%26aufirst%3DMadeline%2BE.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D2017%26date%3D20092016%26date%3D29012017%26date%3D06032017%26date%3D21032017%26date%3D07022017%26volume%3D56%26issue%3D11%26spage%3D1559%26epage%3D1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amit  Kumar</span>, <span class="hlFld-ContribAuthor ">Christopher S.  Campomizzi</span>, <span class="hlFld-ContribAuthor ">Natalie  Jay</span>, <span class="hlFld-ContribAuthor ">Shaun  Ferguson</span>, <span class="hlFld-ContribAuthor ">Emelie-Jo  Scheffler</span>, <span class="hlFld-ContribAuthor ">James  Lioi</span>, <span class="hlFld-ContribAuthor ">Chengjian  Tu</span>, <span class="hlFld-ContribAuthor ">Jun  Qu</span>, <span class="hlFld-ContribAuthor ">Claire  Simons</span>, <span class="hlFld-ContribAuthor ">D. Fernando  Estrada</span>. </span><span class="cited-content_cbyCitation_article-title">Surface hydrophobics mediate functional dimerization of CYP121A1 of Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-79545-y" title="DOI URL">https://doi.org/10.1038/s41598-020-79545-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-79545-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-79545-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DSurface%252Bhydrophobics%252Bmediate%252Bfunctional%252Bdimerization%252Bof%252BCYP121A1%252Bof%252BMycobacterium%252Btuberculosis%26aulast%3DKumar%26aufirst%3DAmit%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Runlai  Liu</span>, <span class="hlFld-ContribAuthor ">Shuai  Zhang</span>, <span class="hlFld-ContribAuthor ">Mingxin  Huang</span>, <span class="hlFld-ContribAuthor ">Zhenpeng  Guo</span>, <span class="hlFld-ContribAuthor ">Long  Li</span>, <span class="hlFld-ContribAuthor ">Mi  Li</span>, <span class="hlFld-ContribAuthor ">Lan  Wu</span>, <span class="hlFld-ContribAuthor ">Qi  Guan</span>, <span class="hlFld-ContribAuthor ">Weige  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and bioevaluation of 2,7-diaryl-pyrazolo[1,5-a]pyrimidines as tubulin polymerization inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>115 </em>, 105220. <a href="https://doi.org/10.1016/j.bioorg.2021.105220" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105220</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105220%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbioevaluation%252Bof%252B2%25252C7-diaryl-pyrazolo%25255B1%25252C5-a%25255Dpyrimidines%252Bas%252Btubulin%252Bpolymerization%252Binhibitors%26aulast%3DLiu%26aufirst%3DRunlai%26date%3D2021%26volume%3D115%26spage%3D105220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabell  Walter</span>, <span class="hlFld-ContribAuthor ">Sebastian  Adam</span>, <span class="hlFld-ContribAuthor ">Maria Virginia  Gentilini</span>, <span class="hlFld-ContribAuthor ">Andreas M.  Kany</span>, <span class="hlFld-ContribAuthor ">Christian  Brengel</span>, <span class="hlFld-ContribAuthor ">Andreas  Thomann</span>, <span class="hlFld-ContribAuthor ">Tim  Sparwasser</span>, <span class="hlFld-ContribAuthor ">Jesko  Köhnke</span>, <span class="hlFld-ContribAuthor ">Rolf W.  Hartmann</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Activity Relationship and Mode‐of‐Action Studies Highlight 1‐(4‐Biphenylylmethyl)‐1
              H
              ‐imidazole‐Derived Small Molecules as Potent CYP121 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>13 </em><a href="https://doi.org/10.1002/cmdc.202100283" title="DOI URL">https://doi.org/10.1002/cmdc.202100283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100283%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStructure%2525E2%252580%252590Activity%252BRelationship%252Band%252BMode%2525E2%252580%252590of%2525E2%252580%252590Action%252BStudies%252BHighlight%252B1%2525E2%252580%252590%2525284%2525E2%252580%252590Biphenylylmethyl%252529%2525E2%252580%2525901%252BH%252B%2525E2%252580%252590imidazole%2525E2%252580%252590Derived%252BSmall%252BMolecules%252Bas%252BPotent%252BCYP121%252BInhibitors%26aulast%3DWalter%26aufirst%3DIsabell%26date%3D2021%26date%3D2021%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian C.  Monk</span>, <span class="hlFld-ContribAuthor ">Mikhail V.  Keniya</span>. </span><span class="cited-content_cbyCitation_article-title">Roles for Structural Biology in the Discovery of Drugs and Agrochemicals Targeting Sterol 14α-Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Fungi</span><span> <strong>2021,</strong> <em>7 </em>
                                    (2)
                                     , 67. <a href="https://doi.org/10.3390/jof7020067" title="DOI URL">https://doi.org/10.3390/jof7020067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jof7020067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjof7020067%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Fungi%26atitle%3DRoles%252Bfor%252BStructural%252BBiology%252Bin%252Bthe%252BDiscovery%252Bof%252BDrugs%252Band%252BAgrochemicals%252BTargeting%252BSterol%252B14%2525CE%2525B1-Demethylases%26aulast%3DMonk%26aufirst%3DBrian%2BC.%26date%3D2021%26date%3D2021%26volume%3D7%26issue%3D2%26spage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luiz G. F.  Lopes</span>, <span class="hlFld-ContribAuthor ">Edinilton Muniz  Carvalho</span>, <span class="hlFld-ContribAuthor ">Eduardo H. S.  Sousa</span>. </span><span class="cited-content_cbyCitation_article-title">A bioinorganic chemistry perspective on the roles of metals as drugs and targets against
              Mycobacterium tuberculosis
              – a journey of opportunities. </span><span class="cited-content_cbyCitation_journal-name">Dalton Transactions</span><span> <strong>2020,</strong> <em>49 </em>
                                    (45)
                                     , 15988-16003. <a href="https://doi.org/10.1039/D0DT01365J" title="DOI URL">https://doi.org/10.1039/D0DT01365J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0DT01365J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0DT01365J%26sid%3Dliteratum%253Aachs%26jtitle%3DDalton%2520Transactions%26atitle%3DA%252Bbioinorganic%252Bchemistry%252Bperspective%252Bon%252Bthe%252Broles%252Bof%252Bmetals%252Bas%252Bdrugs%252Band%252Btargets%252Bagainst%252BMycobacterium%252Btuberculosis%252B%2525E2%252580%252593%252Ba%252Bjourney%252Bof%252Bopportunities%26aulast%3DLopes%26aufirst%3DLuiz%2BG.%2BF.%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D45%26spage%3D15988%26epage%3D16003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuchao  Ma</span>, <span class="hlFld-ContribAuthor ">Ben  Ouyang</span>, <span class="hlFld-ContribAuthor ">Linan  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Biological Evaluation of 3-Aryl-4-alkylpyrazol-5-amines Based on the Target Fishing. </span><span class="cited-content_cbyCitation_journal-name">Current Computer-Aided Drug Design</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 564-570. <a href="https://doi.org/10.2174/1573409915666191003123900" title="DOI URL">https://doi.org/10.2174/1573409915666191003123900</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573409915666191003123900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573409915666191003123900%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Computer-Aided%2520Drug%2520Design%26atitle%3DDesign%252Band%252BBiological%252BEvaluation%252Bof%252B3-Aryl-4-alkylpyrazol-5-amines%252BBased%252Bon%252Bthe%252BTarget%252BFishing%26aulast%3DMa%26aufirst%3DShuchao%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D564%26epage%3D570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandra  Ortega Ugalde</span>, <span class="hlFld-ContribAuthor ">Kerstin  Wallraven</span>, <span class="hlFld-ContribAuthor ">Alexander  Speer</span>, <span class="hlFld-ContribAuthor ">Wilbert  Bitter</span>, <span class="hlFld-ContribAuthor ">Tom N.  Grossmann</span>, <span class="hlFld-ContribAuthor ">Jan N.M.  Commandeur</span>. </span><span class="cited-content_cbyCitation_article-title">Acetylene containing cyclo(L-Tyr-L-Tyr)-analogs as mechanism-based inhibitors of CYP121A1 from Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>177 </em>, 113938. <a href="https://doi.org/10.1016/j.bcp.2020.113938" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.113938</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.113938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.113938%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DAcetylene%252Bcontaining%252Bcyclo%252528L-Tyr-L-Tyr%252529-analogs%252Bas%252Bmechanism-based%252Binhibitors%252Bof%252BCYP121A1%252Bfrom%252BMycobacterium%252Btuberculosis%26aulast%3DOrtega%2BUgalde%26aufirst%3DSandra%26date%3D2020%26volume%3D177%26spage%3D113938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Molly M.  Lockart</span>, <span class="hlFld-ContribAuthor ">Joseph T.  Butler</span>, <span class="hlFld-ContribAuthor ">Carson J.  Mize</span>, <span class="hlFld-ContribAuthor ">Morgan N.  Fair</span>, <span class="hlFld-ContribAuthor ">Alex A.  Cruce</span>, <span class="hlFld-ContribAuthor ">Kip P.  Conner</span>, <span class="hlFld-ContribAuthor ">William M.  Atkins</span>, <span class="hlFld-ContribAuthor ">Michael K.  Bowman</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple drug binding modes in Mycobacterium tuberculosis CYP51B1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Inorganic Biochemistry</span><span> <strong>2020,</strong> <em>205 </em>, 110994. <a href="https://doi.org/10.1016/j.jinorgbio.2020.110994" title="DOI URL">https://doi.org/10.1016/j.jinorgbio.2020.110994</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jinorgbio.2020.110994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jinorgbio.2020.110994%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Inorganic%2520Biochemistry%26atitle%3DMultiple%252Bdrug%252Bbinding%252Bmodes%252Bin%252BMycobacterium%252Btuberculosis%252BCYP51B1%26aulast%3DLockart%26aufirst%3DMolly%2BM.%26date%3D2020%26volume%3D205%26spage%3D110994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandra  Ortega Ugalde</span>, <span class="hlFld-ContribAuthor ">Maikel  Boot</span>, <span class="hlFld-ContribAuthor ">Jan N. M.  Commandeur</span>, <span class="hlFld-ContribAuthor ">Paul  Jennings</span>, <span class="hlFld-ContribAuthor ">Wilbert  Bitter</span>, <span class="hlFld-ContribAuthor ">J. Chris  Vos</span>. </span><span class="cited-content_cbyCitation_article-title">Function, essentiality, and expression of cytochrome P450 enzymes and their cognate redox partners in Mycobacterium tuberculosis: are they drug targets?. </span><span class="cited-content_cbyCitation_journal-name">Applied Microbiology and Biotechnology</span><span> <strong>2019,</strong> <em>103 </em>
                                    (9)
                                     , 3597-3614. <a href="https://doi.org/10.1007/s00253-019-09697-z" title="DOI URL">https://doi.org/10.1007/s00253-019-09697-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00253-019-09697-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00253-019-09697-z%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Microbiology%2520and%2520Biotechnology%26atitle%3DFunction%25252C%252Bessentiality%25252C%252Band%252Bexpression%252Bof%252Bcytochrome%252BP450%252Benzymes%252Band%252Btheir%252Bcognate%252Bredox%252Bpartners%252Bin%252BMycobacterium%252Btuberculosis%25253A%252Bare%252Bthey%252Bdrug%252Btargets%25253F%26aulast%3DOrtega%2BUgalde%26aufirst%3DSandra%26date%3D2019%26date%3D2019%26volume%3D103%26issue%3D9%26spage%3D3597%26epage%3D3614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Min'an  Zhai</span>, <span class="hlFld-ContribAuthor ">Shiyuan  Liu</span>, <span class="hlFld-ContribAuthor ">Meiqi  Gao</span>, <span class="hlFld-ContribAuthor ">Long  Wang</span>, <span class="hlFld-ContribAuthor ">Jun  Sun</span>, <span class="hlFld-ContribAuthor ">Jianan  Du</span>, <span class="hlFld-ContribAuthor ">Qi  Guan</span>, <span class="hlFld-ContribAuthor ">Kai  Bao</span>, <span class="hlFld-ContribAuthor ">Daiying  Zuo</span>, <span class="hlFld-ContribAuthor ">Yingliang  Wu</span>, <span class="hlFld-ContribAuthor ">Weige  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">3,5-Diaryl-1H-pyrazolo[3,4-b]pyridines as potent tubulin polymerization inhibitors: Rational design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>168 </em>, 426-435. <a href="https://doi.org/10.1016/j.ejmech.2018.12.053" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.12.053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.12.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.12.053%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D3%25252C5-Diaryl-1H-pyrazolo%25255B3%25252C4-b%25255Dpyridines%252Bas%252Bpotent%252Btubulin%252Bpolymerization%252Binhibitors%25253A%252BRational%252Bdesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DZhai%26aufirst%3DMin%2527an%26date%3D2019%26volume%3D168%26spage%3D426%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Borgman</span>, <span class="hlFld-ContribAuthor ">Ryan D.  Lopez</span>, <span class="hlFld-ContribAuthor ">Amy L.  Lane</span>. </span><span class="cited-content_cbyCitation_article-title">The expanding spectrum of diketopiperazine natural product biosynthetic pathways containing cyclodipeptide synthases. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (9)
                                     , 2305-2314. <a href="https://doi.org/10.1039/C8OB03063D" title="DOI URL">https://doi.org/10.1039/C8OB03063D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8OB03063D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8OB03063D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DThe%252Bexpanding%252Bspectrum%252Bof%252Bdiketopiperazine%252Bnatural%252Bproduct%252Bbiosynthetic%252Bpathways%252Bcontaining%252Bcyclodipeptide%252Bsynthases%26aulast%3DBorgman%26aufirst%3DPaul%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D9%26spage%3D2305%26epage%3D2314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vaishali P.  Waman</span>, <span class="hlFld-ContribAuthor ">Sundeep Chaitanya  Vedithi</span>, <span class="hlFld-ContribAuthor ">Sherine E.  Thomas</span>, <span class="hlFld-ContribAuthor ">Bridget P.  Bannerman</span>, <span class="hlFld-ContribAuthor ">Asma  Munir</span>, <span class="hlFld-ContribAuthor ">Marcin J.  Skwark</span>, <span class="hlFld-ContribAuthor ">Sony  Malhotra</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>. </span><span class="cited-content_cbyCitation_article-title">Mycobacterial genomics and structural bioinformatics: opportunities and challenges in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Emerging Microbes & Infections</span><span> <strong>2019,</strong> <em>8 </em>
                                    (1)
                                     , 109-118. <a href="https://doi.org/10.1080/22221751.2018.1561158" title="DOI URL">https://doi.org/10.1080/22221751.2018.1561158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/22221751.2018.1561158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F22221751.2018.1561158%26sid%3Dliteratum%253Aachs%26jtitle%3DEmerging%2520Microbes%2520%2526%2520Infections%26atitle%3DMycobacterial%252Bgenomics%252Band%252Bstructural%252Bbioinformatics%25253A%252Bopportunities%252Band%252Bchallenges%252Bin%252Bdrug%252Bdiscovery%26aulast%3DWaman%26aufirst%3DVaishali%2BP.%26date%3D2019%26date%3D2019%26volume%3D8%26issue%3D1%26spage%3D109%26epage%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">CHIARA  MARCHETTI</span>, <span class="hlFld-ContribAuthor ">DANIEL S. H.  CHAN</span>, <span class="hlFld-ContribAuthor ">ANTHONY G.  COYNE</span>, <span class="hlFld-ContribAuthor ">CHRIS  ABELL</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based approaches to TB drugs. </span><span class="cited-content_cbyCitation_journal-name">Parasitology</span><span> <strong>2018,</strong> <em>145 </em>
                                    (2)
                                     , 184-195. <a href="https://doi.org/10.1017/S0031182016001876" title="DOI URL">https://doi.org/10.1017/S0031182016001876</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1017/S0031182016001876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1017%2FS0031182016001876%26sid%3Dliteratum%253Aachs%26jtitle%3DParasitology%26atitle%3DFragment-based%252Bapproaches%252Bto%252BTB%252Bdrugs%26aulast%3DMARCHETTI%26aufirst%3DCHIARA%26date%3D2018%26date%3D2016%26volume%3D145%26issue%3D2%26spage%3D184%26epage%3D195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajnish  Kumar,</span>, <span class="hlFld-ContribAuthor ">Ramandeep  Kaur,</span>, <span class="hlFld-ContribAuthor ">Lalita  Dahiya,</span>, <span class="hlFld-ContribAuthor ">Manoj  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Vicinal Diaryl Pyrazole: A Therapeutically Potential Molecular Scaffold. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 161-189. <a href="https://doi.org/10.1016/B978-0-08-102237-5.00005-5" title="DOI URL">https://doi.org/10.1016/B978-0-08-102237-5.00005-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-102237-5.00005-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-102237-5.00005-5%26sid%3Dliteratum%253Aachs%26atitle%3DVicinal%252BDiaryl%252BPyrazole%25253A%252BA%252BTherapeutically%252BPotential%252BMolecular%252BScaffold%26aulast%3DKumar%252C%26aufirst%3DRajnish%26date%3D2018%26spage%3D161%26epage%3D189%26pub%3DElsevier%26atitle%3DVicinal%252BDiaryl%252BSubstituted%252BHeterocycles%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura N.  Jeffreys</span>, <span class="hlFld-ContribAuthor ">Hazel M.  Girvan</span>, <span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of Cytochrome P450 Enzymes and Their Applications in Synthetic Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 189-261. <a href="https://doi.org/10.1016/bs.mie.2018.06.013" title="DOI URL">https://doi.org/10.1016/bs.mie.2018.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.mie.2018.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.mie.2018.06.013%26sid%3Dliteratum%253Aachs%26atitle%3DCharacterization%252Bof%252BCytochrome%252BP450%252BEnzymes%252Band%252BTheir%252BApplications%252Bin%252BSynthetic%252BBiology%26aulast%3DJeffreys%26aufirst%3DLaura%2BN.%26date%3D2018%26spage%3D189%26epage%3D261%26pub%3DElsevier%26atitle%3DEnzymes%252Bin%252BSynthetic%252BBiology%26date%3D2018%26volume%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wagdy M.  Eldehna</span>, <span class="hlFld-ContribAuthor ">Dina H.  EL-Naggar</span>, <span class="hlFld-ContribAuthor ">Ahmed R.  Hamed</span>, <span class="hlFld-ContribAuthor ">Hany S.  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Hazem A.  Ghabbour</span>, <span class="hlFld-ContribAuthor ">Hatem A.  Abdel-Aziz</span>. </span><span class="cited-content_cbyCitation_article-title">One-pot three-component synthesis of novel spirooxindoles with potential cytotoxic activity against triple-negative breast cancer MDA-MB-231 cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2018,</strong> <em>33 </em>
                                    (1)
                                     , 309-318. <a href="https://doi.org/10.1080/14756366.2017.1417276" title="DOI URL">https://doi.org/10.1080/14756366.2017.1417276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2017.1417276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2017.1417276%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DOne-pot%252Bthree-component%252Bsynthesis%252Bof%252Bnovel%252Bspirooxindoles%252Bwith%252Bpotential%252Bcytotoxic%252Bactivity%252Bagainst%252Btriple-negative%252Bbreast%252Bcancer%252BMDA-MB-231%252Bcells%26aulast%3DEldehna%26aufirst%3DWagdy%2BM.%26date%3D2018%26date%3D2017%26volume%3D33%26issue%3D1%26spage%3D309%26epage%3D318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Khalid  Karrouchi</span>, <span class="hlFld-ContribAuthor ">Smaail  Radi</span>, <span class="hlFld-ContribAuthor ">Youssef  Ramli</span>, <span class="hlFld-ContribAuthor ">Jamal  Taoufik</span>, <span class="hlFld-ContribAuthor ">Yahia  Mabkhot</span>, <span class="hlFld-ContribAuthor ">Faiz  Al-aizari</span>, <span class="hlFld-ContribAuthor ">M’hammed  Ansar</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Pharmacological Activities of Pyrazole Derivatives: A Review. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (1)
                                     , 134. <a href="https://doi.org/10.3390/molecules23010134" title="DOI URL">https://doi.org/10.3390/molecules23010134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23010134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23010134%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%252Band%252BPharmacological%252BActivities%252Bof%252BPyrazole%252BDerivatives%25253A%252BA%252BReview%26aulast%3DKarrouchi%26aufirst%3DKhalid%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D1%26spage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Brengel</span>, <span class="hlFld-ContribAuthor ">Andreas  Thomann</span>, <span class="hlFld-ContribAuthor ">Alexander  Schifrin</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Allegretta</span>, <span class="hlFld-ContribAuthor ">Ahmed A. M.  Kamal</span>, <span class="hlFld-ContribAuthor ">Jörg  Haupenthal</span>, <span class="hlFld-ContribAuthor ">Isabell  Schnorr</span>, <span class="hlFld-ContribAuthor ">Sang Hyun  Cho</span>, <span class="hlFld-ContribAuthor ">Scott G.  Franzblau</span>, <span class="hlFld-ContribAuthor ">Martin  Empting</span>, <span class="hlFld-ContribAuthor ">Jens  Eberhard</span>, <span class="hlFld-ContribAuthor ">Rolf W.  Hartmann</span>. </span><span class="cited-content_cbyCitation_article-title">Biophysical Screening of a Focused Library for the Discovery of CYP121 Inhibitors as Novel Antimycobacterials. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (19)
                                     , 1616-1626. <a href="https://doi.org/10.1002/cmdc.201700363" title="DOI URL">https://doi.org/10.1002/cmdc.201700363</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700363%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DBiophysical%252BScreening%252Bof%252Ba%252BFocused%252BLibrary%252Bfor%252Bthe%252BDiscovery%252Bof%252BCYP121%252BInhibitors%252Bas%252BNovel%252BAntimycobacterials%26aulast%3DBrengel%26aufirst%3DChristian%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D19%26spage%3D1616%26epage%3D1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi  Xu</span>, <span class="hlFld-ContribAuthor ">Chuan  Gao</span>, <span class="hlFld-ContribAuthor ">Qing-Cheng  Ren</span>, <span class="hlFld-ContribAuthor ">Xu-Feng  Song</span>, <span class="hlFld-ContribAuthor ">Lian-Shun  Feng</span>, <span class="hlFld-ContribAuthor ">Zao-Sheng  Lv</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances of pyrazole-containing derivatives as anti-tubercular agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>139 </em>, 429-440. <a href="https://doi.org/10.1016/j.ejmech.2017.07.059" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.07.059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.07.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.07.059%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bof%252Bpyrazole-containing%252Bderivatives%252Bas%252Banti-tubercular%252Bagents%26aulast%3DXu%26aufirst%3DZhi%26date%3D2017%26volume%3D139%26spage%3D429%26epage%3D440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sherine E.  Thomas</span>, <span class="hlFld-ContribAuthor ">Vitor  Mendes</span>, <span class="hlFld-ContribAuthor ">So Yeon  Kim</span>, <span class="hlFld-ContribAuthor ">Sony  Malhotra</span>, <span class="hlFld-ContribAuthor ">Bernardo  Ochoa-Montaño</span>, <span class="hlFld-ContribAuthor ">Michal  Blaszczyk</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Biology and the Design of New Therapeutics: From HIV and Cancer to Mycobacterial Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2017,</strong> <em>429 </em>
                                    (17)
                                     , 2677-2693. <a href="https://doi.org/10.1016/j.jmb.2017.06.014" title="DOI URL">https://doi.org/10.1016/j.jmb.2017.06.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2017.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2017.06.014%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DStructural%252BBiology%252Band%252Bthe%252BDesign%252Bof%252BNew%252BTherapeutics%25253A%252BFrom%252BHIV%252Band%252BCancer%252Bto%252BMycobacterial%252BInfections%26aulast%3DThomas%26aufirst%3DSherine%2BE.%26date%3D2017%26volume%3D429%26issue%3D17%26spage%3D2677%26epage%3D2693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kednerlin  Dornevil</span>, <span class="hlFld-ContribAuthor ">Ian  Davis</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Fielding</span>, <span class="hlFld-ContribAuthor ">James R.  Terrell</span>, <span class="hlFld-ContribAuthor ">Li  Ma</span>, <span class="hlFld-ContribAuthor ">Aimin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Cross-linking of dicyclotyrosine by the cytochrome P450 enzyme CYP121 from Mycobacterium tuberculosis proceeds through a catalytic shunt pathway. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2017,</strong> <em>292 </em>
                                    (33)
                                     , 13645-13657. <a href="https://doi.org/10.1074/jbc.M117.794099" title="DOI URL">https://doi.org/10.1074/jbc.M117.794099</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M117.794099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M117.794099%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DCross-linking%252Bof%252Bdicyclotyrosine%252Bby%252Bthe%252Bcytochrome%252BP450%252Benzyme%252BCYP121%252Bfrom%252BMycobacterium%252Btuberculosis%252Bproceeds%252Bthrough%252Ba%252Bcatalytic%252Bshunt%252Bpathway%26aulast%3DDornevil%26aufirst%3DKednerlin%26date%3D2017%26volume%3D292%26issue%3D33%26spage%3D13645%26epage%3D13657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter J.  Lindsay-Scott</span>, <span class="hlFld-ContribAuthor ">Peter T.  Gallagher</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of heterocycles from arylacetonitriles: Powerful tools for medicinal chemists. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (27)
                                     , 2629-2635. <a href="https://doi.org/10.1016/j.tetlet.2017.05.089" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.05.089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.05.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.05.089%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DSynthesis%252Bof%252Bheterocycles%252Bfrom%252Barylacetonitriles%25253A%252BPowerful%252Btools%252Bfor%252Bmedicinal%252Bchemists%26aulast%3DLindsay-Scott%26aufirst%3DPeter%2BJ.%26date%3D2017%26volume%3D58%26issue%3D27%26spage%3D2629%26epage%3D2635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>. </span><span class="cited-content_cbyCitation_article-title">Drug targeting of heme proteins in Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2017,</strong> <em>22 </em>
                                    (3)
                                     , 566-575. <a href="https://doi.org/10.1016/j.drudis.2016.11.004" title="DOI URL">https://doi.org/10.1016/j.drudis.2016.11.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2016.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2016.11.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DDrug%252Btargeting%252Bof%252Bheme%252Bproteins%252Bin%252BMycobacterium%252Btuberculosis%26aulast%3DMcLean%26aufirst%3DKirsty%2BJ.%26date%3D2017%26volume%3D22%26issue%3D3%26spage%3D566%26epage%3D575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vitor  Mendes</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting tuberculosis using structure-guided fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2017,</strong> <em>22 </em>
                                    (3)
                                     , 546-554. <a href="https://doi.org/10.1016/j.drudis.2016.10.003" title="DOI URL">https://doi.org/10.1016/j.drudis.2016.10.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2016.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2016.10.003%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DTargeting%252Btuberculosis%252Busing%252Bstructure-guided%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DMendes%26aufirst%3DVitor%26date%3D2017%26volume%3D22%26issue%3D3%26spage%3D546%26epage%3D554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jude T.  Chenge</span>, <span class="hlFld-ContribAuthor ">Le  Van Duyet</span>, <span class="hlFld-ContribAuthor ">Shalini  Swami</span>, <span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Madeline E.  Kavanagh</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>, <span class="hlFld-ContribAuthor ">Stephen E.J.  Rigby</span>, <span class="hlFld-ContribAuthor ">Myles R.  Cheesman</span>, <span class="hlFld-ContribAuthor ">Hazel M.  Girvan</span>, <span class="hlFld-ContribAuthor ">Colin W.  Levy</span>, <span class="hlFld-ContribAuthor ">Bernd  Rupp</span>, <span class="hlFld-ContribAuthor ">Jens P.  von Kries</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>, <span class="hlFld-ContribAuthor ">David  Leys</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Characterization and Ligand/Inhibitor Identification Provide Functional Insights into the Mycobacterium tuberculosis Cytochrome P450 CYP126A1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2017,</strong> <em>292 </em>
                                    (4)
                                     , 1310-1329. <a href="https://doi.org/10.1074/jbc.M116.748822" title="DOI URL">https://doi.org/10.1074/jbc.M116.748822</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M116.748822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M116.748822%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DStructural%252BCharacterization%252Band%252BLigand%25252FInhibitor%252BIdentification%252BProvide%252BFunctional%252BInsights%252Binto%252Bthe%252BMycobacterium%252Btuberculosis%252BCytochrome%252BP450%252BCYP126A1%26aulast%3DChenge%26aufirst%3DJude%2BT.%26date%3D2017%26volume%3D292%26issue%3D4%26spage%3D1310%26epage%3D1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Madeline E.  Kavanagh</span>, <span class="hlFld-ContribAuthor ">Janine L.  Gray</span>, <span class="hlFld-ContribAuthor ">Sophie H.  Gilbert</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>, <span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Holly J.  Davis</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>. </span><span class="cited-content_cbyCitation_article-title">Substrate Fragmentation for the Design of
              M. tuberculosis
              CYP121 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (17)
                                     , 1924-1935. <a href="https://doi.org/10.1002/cmdc.201600248" title="DOI URL">https://doi.org/10.1002/cmdc.201600248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600248%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSubstrate%252BFragmentation%252Bfor%252Bthe%252BDesign%252Bof%252BM.%2525E2%252580%252585tuberculosis%252BCYP121%252BInhibitors%26aulast%3DKavanagh%26aufirst%3DMadeline%2BE.%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D17%26spage%3D1924%26epage%3D1935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mary  Snow Setzer</span>, <span class="hlFld-ContribAuthor ">Javad  Sharifi-Rad</span>, <span class="hlFld-ContribAuthor ">William  Setzer</span>. </span><span class="cited-content_cbyCitation_article-title">The Search for Herbal Antibiotics: An In-Silico Investigation of Antibacterial Phytochemicals. </span><span class="cited-content_cbyCitation_journal-name">Antibiotics</span><span> <strong>2016,</strong> <em>5 </em>
                                    (3)
                                     , 30. <a href="https://doi.org/10.3390/antibiotics5030030" title="DOI URL">https://doi.org/10.3390/antibiotics5030030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/antibiotics5030030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fantibiotics5030030%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotics%26atitle%3DThe%252BSearch%252Bfor%252BHerbal%252BAntibiotics%25253A%252BAn%252BIn-Silico%252BInvestigation%252Bof%252BAntibacterial%252BPhytochemicals%26aulast%3DSnow%2BSetzer%26aufirst%3DMary%26date%3D2016%26date%3D2016%26volume%3D5%26issue%3D3%26spage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Holly J.  Davis</span>, <span class="hlFld-ContribAuthor ">Madeline E.  Kavanagh</span>, <span class="hlFld-ContribAuthor ">Tudor  Balan</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>. </span><span class="cited-content_cbyCitation_article-title">Spirooxindoles as novel 3D-fragment scaffolds: Synthesis and screening against CYP121 from M. tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (15)
                                     , 3735-3740. <a href="https://doi.org/10.1016/j.bmcl.2016.05.073" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.05.073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.05.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.05.073%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSpirooxindoles%252Bas%252Bnovel%252B3D-fragment%252Bscaffolds%25253A%252BSynthesis%252Band%252Bscreening%252Bagainst%252BCYP121%252Bfrom%252BM.%252Btuberculosis%26aulast%3DDavis%26aufirst%3DHolly%2BJ.%26date%3D2016%26volume%3D26%26issue%3D15%26spage%3D3735%26epage%3D3740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment merging and in silico directed optimization of fragment <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G47">4G47</a>) led to the discovery of low micromolar CYP121 inhibitor <b>2</b> (salmon, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a>). Fragment <b>1</b> and lead <b>2</b> retained a conserved binding mode distal to the heme cofactor (magenta). Hydrogen bonding interactions between the 5-aminopyrazole ring and CYP121 residues Gln385 and Ala167, and the phenol group with Asn85 and active site water molecules are shown as yellow dashed lines. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray crystal structures of lead compound <b>2</b> and component retrofragments <b>4</b>–<b>6</b> in complex with CYP121. The I- (yellow), and F-, and G- (blue) helices are shown in cartoon representation. (a) Overlaid structures of retrofragments <b>4</b> (purple) (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTJ">4KTJ</a>), <b>5</b> (yellow) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF">4KTF</a>), and <b>6</b> (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>) with lead compound <b>2</b> (salmon) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL">4KTL</a>), illustrating the conserved binding mode distal to the heme cofactor (magenta).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (b) Structures, binding affinity, and ligand efficiency (LE) of compound <b>2</b> and retrofragments <b>4</b>, <b>5</b>, and <b>6</b>. (c–e) X-ray crystal structures of CYP121 in complex with retrofragments <b>4</b>, <b>5</b>, and <b>6</b>, respectively, indicating polar interactions (yellow dashes) with active site residues (blue) and solvent (red spheres). Co-crystallized sulfate (yellow lines) is also indicated in (d). The associated omit <i>F</i><sub>o</sub><i>F</i><sub>c</sub> electron density map of fragment <b>6</b> contoured to 3σ has been provided in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Deconstruction of lead compound <b>2</b> into retrosynthetic fragments <b>3</b>–<b>6</b> was used to assess the binding affinity, ligand efficiency (LE), and group efficiency (GE) contribution of the individual components. The three aromatic rings decorating the aminopyrazole core of lead compound <b>2</b> have been labeled Ar1, Ar2, and Ar3 as illustrated, which refers to the orientation the rings occupy in the X-ray crystal structure of <b>2</b> in complex with CYP121 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Retrofragments <b>3</b> and <b>4</b> represent Ar1 and Ar2, respectively. The lower panel tabulates the binding affinity (<i>K</i><sub>D</sub>) and LE and GE properties of compounds from the sequential reconstruction of lead <b>2</b> from component monoaryl (<b>3</b> and <b>4</b>) and biaryl (<b>5</b> and <b>6</b>) retrofragments. <i>K</i><sub>D</sub> values were determined using ITC. LE was calculated from the Gibbs free energy of binding divided by the number of heavy atoms (Δ<i>G</i>/HA).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> GE contributions of the three aromatic rings Ar1, Ar2, and Ar3 were calculated from the change in Gibbs free energy per added heavy atom (−ΔΔ<i>G</i>/Δ<i>H</i>A) as the retrofragments were progressively recombined to form lead compound <b>2</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><i>K</i><sub>D</sub> previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Proposed optimization strategies for Ar1, Ar2, and Ar3 of lead CYP121 inhibitor <b>2</b>. (a) Overlaid X-ray crystal structures of biaryl fragment <b>5</b> (yellow) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF">4KTF</a>) and benzamide fragment <b>7</b> (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>) bound to CYP121. The associated omit <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> electron density map of fragment <b>7</b> contoured to 3σ can be found in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. (b) Overlaid X-ray crystal structures of CYP121 in complex with retrofragment <b>5</b> (yellow) and with heme binding fragments <b>8</b> (copper) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G44">4G44</a>) and <b>9</b> (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G45">4G45</a>),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> indicating the proposed <i>meta</i>-attachment point for substituents and distance (6.5 Å) to the heme coordinating position (yellow dashed line). (c) Docking of Ar3 analogues of lead <b>2</b>, elaborating from the 5-aminopyrazole group of retrofragment <b>5</b> (gray) with carbonyl (orange) or methylene (green) linkages to enhance binding interactions with the distal active site pocket (Asn74, Thr65, gray sticks) of CYP121. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Biaryl-aminopyrazole Analogues (Ar1)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaH, THF, 60 °C, 20–48 h; (b) HCl (37%), NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 90 °C, 14 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 6 h. Synthesis and characterization of <b>14d</b> previously described.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Ar2 Analogues Based on the Core Scaffold of (i) Monoaryl Fragment <b>4</b> or (ii) Biaryl Fragment <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) ethyl formate, NaOEt, EtOH, 0–90 °C, 22 h; (b) AcOH, NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 75 °C, 3 h; (c) aryl-boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane:H<sub>2</sub>O (1:1 v/v), μw, 180 min; (d) SnCl<sub>2</sub>·2H<sub>2</sub>O, EtOH, 75 °C, 15 h; (e) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 4 h; (g) methyl 4-methoxybenzoate, NaH, THF, 60 °C, 48 h; (h) HCl (37%), NH<sub>2</sub>NH<sub>2</sub>·xH<sub>2</sub>O, EtOH, 90 °C, 16 h.</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Benzyl-amine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aldehyde (0.9–1.1 equiv), AcOH*, Na(OAc)<sub>3</sub>BH, DMF, DCE, rt, 24–48 h (b) aldehyde (1 equiv), AcOH, NaCNBH<sub>3</sub>, MeOH, rt, 15–24 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, 0 °C, 2–24 h. *No AcOH with <b>26d</b>.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Benzamide Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (4-OMe)-benzoyl chloride, Et<sub>3</sub>N, DCM, 0 °C–rt, 3.5 h; (b) nicotinic acid, oxalyl chloride, DMF, DCM, 0 °C–rt, 2 h, then Et<sub>3</sub>N, DCM, 0 °C–rt, 14 h; (c) BBr<sub>3</sub> (1.0 M in DCM), DCM, −78 °C–rt, 14 h.</p></p></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optical titration difference spectra (top panel) and concentration-dependent changes in CYP121 absorbance (bottom panel) for representative heme binding ligands (a) <b>24a</b>, (b) <b>25b</b>, and (c) <b>19a</b>. Absorption spectra of CYP121 (5 μM) were collected between 250 and 800 nm after sequential additions of 0.2 μL aliquots of DMSO-<i>d</i><sub>6</sub> stock solutions (0.15–83 mM) of ligands. Difference spectra were obtained from the subtraction of the ligand-free CYP121 spectra from those obtained over the course of the titration. The maximum change in absorbance from each difference spectrum was plotted against ligand concentration and data were fitted using a modified version of the Morrison equation for tight binding ligands <b>24a</b> and <b>25b</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> or a hyperbolic function for <b>19a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Representative ITC binding isotherms and integrated enthalpy plots for (a) fused Ar1–Ar3 fragment <b>6</b> (500 μM); (b) non-heme binding Ar3 analogue <b>26a</b> (1 mM); (c) heme binding analogue <b>25a</b> (500 μM). The isotherm of each compound titrated into buffer alone was subtracted from that containing CYP121 (50 μM). The integrated enthalpy change for each injection plotted as a function of molar ratio of ligand-to-CYP121 was fitted using a one-site equilibrium binding model to calculate binding affinity (<i>K</i><sub>D</sub>). The binding stoichiometry (N) was allowed to vary for analogues <b>6</b>, <b>26a</b>, and <b>25a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray crystal structures of heme binding Ar2 analogues <b>25b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>), <b>25a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE">5IBE</a>), and <b>19a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF">5IBF</a>) in complex with CYP121. (a) <b>25b</b> (cyan) closely recapitulated the binding of lead <b>2</b>, forming polar contacts (yellow dashes) with amino acids (blue sticks) in binding hotspots as well as additional binding contacts to Ser237, Val228, and the heme iron. (b) <b>25a</b> (orange) was rotated away from the I-helix (yellow cartoon), placing the 5-aminopyrazole ring in the Ar3 pocket. Polar contacts to the heme iron and Ser237 are indicated (yellow dashes). (c) Two molecules of <b>19a</b> bound per active site, each ligand fulfilling a subset of the interactions identified as binding hotspots. Inset: Heme binding <b>19a</b> recapitulated the binding mode of <b>25a</b> while the non-heme binding <b>19a</b> ligand satisfied aromatic interactions in the Ar1 pocket, as well as the hydrogen bonding interactions of fragment <b>7</b>. (d) Extracted structures of <b>25b</b>, <b>25a</b>, and <b>19a</b> annotated to show metal coordination (yellow dashes) distance, angle of approach to the porphyrin ring (yellow angle), and dihedral torsion in the biphenyl-aniline motif. Removal of the Ar2 4-hydroxy group reduced the dihedral angle of the biphenyl aniline system in <b>19a</b> and <b>25a</b> by 30° compared to <b>25b</b> and allowed the analogues to approach the plane of the porphyrin at a sharper angle (31–33°). The omit <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps of ligands <b>19a</b>, <b>25a</b>, and <b>25b</b> contoured to 3σ have been provided in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structures of Ar3 analogues (a) <b>26a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI">5IBI</a>) and (b) <b>26h</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH">5IBH</a>) in complex with CYP121. Both compounds closely recapitulated the binding mode of lead <b>2</b>. Ar3 substituents remained flexible, interacting only with active site water molecules for <b>26a</b> or forming novel interactions with active site residues Ala75 and Asn74 for <b>26h</b>. The omit <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps of ligands <b>26a</b> and <b>26h</b> contoured to 3σ have been provided in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Figure S5</a>. Figures prepared using PyMOL v1.7.4 (Schrödinger, LLC).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/medium/jm-2016-00007f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Representative native mass spectra of CYP121 and selected compounds <b>26a</b>, <b>25a</b>, and <b>19a</b> demonstrated that compounds formed stable complexes with both monomeric (<i>m</i>/<i>z</i> 5250–6750) and dimeric (<i>m</i>/<i>z</i> 5250–6750) CYP121. The number of bound ligands (orange spheres) increased in a concentration-dependent manner and was proportionate to the binding affinity (<i>K</i><sub>D</sub>) of the compounds. Spectra were collected for CYP121 (8.7 or 5 μM) at three different ligand-to-protein ratios (0.5:1, 1:1, and 25:1) and in the presence of DMSO-<i>d</i><sub>6</sub> (2.2% or 5% v/v) alone. The concentration of protein or DMSO-<i>d</i><sub>6</sub> did not affect the quality of the spectra or ligand binding interactions, but higher concentrations of DMSO-<i>d</i><sub>6</sub> reduced the charge state of the protein. Ligand binding stoichiometry was calculated from the difference in mass of ligand-bound and unbound (orange dashed lines) protein peaks divided by the molecular weight of ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf" class="ext-link">Table S1</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-7/acs.jmedchem.6b00007/20160415/images/large/jm-2016-00007f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00007&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span>Global Tuberculosis Report; <span class="NLM_publisher-name">World Health Organisation</span>: <span class="NLM_publisher-loc">Geneva</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Global+Tuberculosis+Report%3B+World+Health+Organisation%3A+Geneva%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DGlobal%2520Tuberculosis%2520Report%26pub%3DWorld%2520Health%2520Organisation%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Cole, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eiglmeier, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekaia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badcock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chillingworth, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devlin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feltwell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentles, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holroyd, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornsby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krogh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moule, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quail, M. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajandream, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squares, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squares, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulston, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, B. G.</span><span> </span><span class="NLM_article-title">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">393</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">544</span><span class="refDoi"> DOI: 10.1038/31159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2F31159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=9634230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1cXjvV2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=1998&pages=537-544&author=S.+T.+Coleauthor=R.+Broschauthor=J.+Parkhillauthor=T.+Garnierauthor=C.+Churcherauthor=D.+Harrisauthor=S.+V.+Gordonauthor=K.+Eiglmeierauthor=S.+Gasauthor=C.+E.+Barryauthor=F.+Tekaiaauthor=K.+Badcockauthor=D.+Bashamauthor=D.+Brownauthor=T.+Chillingworthauthor=R.+Connorauthor=R.+Daviesauthor=K.+Devlinauthor=T.+Feltwellauthor=S.+Gentlesauthor=N.+Hamlinauthor=S.+Holroydauthor=T.+Hornsbyauthor=K.+Jagelsauthor=A.+Kroghauthor=J.+McLeanauthor=S.+Mouleauthor=L.+Murphyauthor=K.+Oliverauthor=J.+Osborneauthor=M.+a+Quailauthor=M.+A.+Rajandreamauthor=J.+Rogersauthor=S.+Rutterauthor=K.+Seegerauthor=J.+Skeltonauthor=R.+Squaresauthor=S.+Squaresauthor=J.+E.+Sulstonauthor=K.+Taylorauthor=S.+Whiteheadauthor=B.+G.+Barrell&title=Deciphering+the+biology+of+Mycobacterium+tuberculosis+from+the+complete+genome+sequence&doi=10.1038%2F31159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence</span></div><div class="casAuthors">Cole, S. T.; Brosch, R.; Parkhill, J.; Garnier, T.; Churcher, C.; Harris, D.; Gordon, S. V.; Eiglmeier, K.; Gas, S.; Barry, C. E., III; Tekaia, F.; Badcock, K.; Basham, D.; Brown, D.; Chillingworth, T.; Connor, R.; Davies, R.; Devlin, K.; Feltwell, T.; Gentles, S.; Hamlin, N.; Holroyd, S.; Hornsby, T.; Jagels, K.; Krogh, A.; McLean, J.; Moule, S.; Murphy, L.; Oliver, K.; Osborne, J.; Quail, M. A.; Rajandream, M.-A.; Rogers, J.; Rutter, S.; Seeger, K.; Skelton, J.; Squares, R.; Squares, S.; Sulston, J. E.; Taylor, K.; Whitehead, S.; Barrell, B. G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">6685</span>),
    <span class="NLM_cas:pages">537-544</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Countless millions of people have died from tuberculosis, a chronic infectious disease caused by the tubercle bacillus.  The complete genome sequence of the best-characterized strain of Mycobacterium tuberculosis, H37Rv, was detd. and analyzed in order to improve our understanding of the biol. of this slow-growing pathogen and to help the conception of new prophylactic and therapeutic interventions.  The genome comprises 4,411,529 base pairs, contains around 4000 genes, and has a very high G+C content that is reflected in the biased amino acid content of the proteins.  M. tuberculosis differs radically from other bacteria in that a very large portion of its coding capacity is devoted to the prodn. of enzymes involved in lipogenesis and lipolysis, and to 2 new families of glycine-rich proteins with a repetitive structure that may represent a source of antigenic variation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9Q-iByIunurVg90H21EOLACvtfcHk0ljlP7kYR6Np0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvV2ksrY%253D&md5=44d23e7674bfeb1a3e30b55abf5268d2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F31159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F31159%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DBrosch%26aufirst%3DR.%26aulast%3DParkhill%26aufirst%3DJ.%26aulast%3DGarnier%26aufirst%3DT.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DS.%2BV.%26aulast%3DEiglmeier%26aufirst%3DK.%26aulast%3DGas%26aufirst%3DS.%26aulast%3DBarry%26aufirst%3DC.%2BE.%26aulast%3DTekaia%26aufirst%3DF.%26aulast%3DBadcock%26aufirst%3DK.%26aulast%3DBasham%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DChillingworth%26aufirst%3DT.%26aulast%3DConnor%26aufirst%3DR.%26aulast%3DDavies%26aufirst%3DR.%26aulast%3DDevlin%26aufirst%3DK.%26aulast%3DFeltwell%26aufirst%3DT.%26aulast%3DGentles%26aufirst%3DS.%26aulast%3DHamlin%26aufirst%3DN.%26aulast%3DHolroyd%26aufirst%3DS.%26aulast%3DHornsby%26aufirst%3DT.%26aulast%3DJagels%26aufirst%3DK.%26aulast%3DKrogh%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DJ.%26aulast%3DMoule%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DK.%26aulast%3DOsborne%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DM.%2Ba%26aulast%3DRajandream%26aufirst%3DM.%2BA.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DRutter%26aufirst%3DS.%26aulast%3DSeeger%26aufirst%3DK.%26aulast%3DSkelton%26aufirst%3DJ.%26aulast%3DSquares%26aufirst%3DR.%26aulast%3DSquares%26aufirst%3DS.%26aulast%3DSulston%26aufirst%3DJ.%2BE.%26aulast%3DTaylor%26aufirst%3DK.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26atitle%3DDeciphering%2520the%2520biology%2520of%2520Mycobacterium%2520tuberculosis%2520from%2520the%2520complete%2520genome%2520sequence%26jtitle%3DNature%26date%3D1998%26volume%3D393%26spage%3D537%26epage%3D544%26doi%3D10.1038%2F31159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clift, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balding, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The preponderance of P450s in the Mycobacterium tuberculosis genome</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">220</span><span class="NLM_x">–</span> <span class="NLM_lpage">228</span><span class="refDoi"> DOI: 10.1016/j.tim.2006.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.tim.2006.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=220-228&author=K.+J.+McLeanauthor=D.+Cliftauthor=D.+G.+Lewisauthor=M.+Sabriauthor=P.+R.+Baldingauthor=M.+J.+Sutcliffeauthor=D.+Leysauthor=A.+W.+Munro&title=The+preponderance+of+P450s+in+the+Mycobacterium+tuberculosis+genome&doi=10.1016%2Fj.tim.2006.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2006.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2006.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DClift%26aufirst%3DD.%26aulast%3DLewis%26aufirst%3DD.%2BG.%26aulast%3DSabri%26aufirst%3DM.%26aulast%3DBalding%26aufirst%3DP.%2BR.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520preponderance%2520of%2520P450s%2520in%2520the%2520Mycobacterium%2520tuberculosis%2520genome%26jtitle%3DTrends%2520Microbiol.%26date%3D2006%26volume%3D14%26spage%3D220%26epage%3D228%26doi%3D10.1016%2Fj.tim.2006.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jowitt, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parish, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Expression and characterization of Mycobacterium tuberculosis CYP144: Common themes and lessons learned in the M. tuberculosis P450 enzyme family</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Proteins Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1814</span><span class="NLM_x">, </span> <span class="NLM_fpage">76</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1016/j.bbapap.2010.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.bbapap.2010.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20621636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1814&publication_year=2011&pages=76-87&author=M.+D.+Driscollauthor=K.+J.+McLeanauthor=M.+R.+Cheesmanauthor=T.+A.+Jowittauthor=M.+Howardauthor=P.+Carrollauthor=T.+Parishauthor=A.+W.+Munro&title=Expression+and+characterization+of+Mycobacterium+tuberculosis+CYP144%3A+Common+themes+and+lessons+learned+in+the+M.+tuberculosis+P450+enzyme+family&doi=10.1016%2Fj.bbapap.2010.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and characterization of Mycobacterium tuberculosis CYP144: Common themes and lessons learned in the M. tuberculosis P450 enzyme family</span></div><div class="casAuthors">Driscoll, Max D.; McLean, Kirsty J.; Cheesman, Myles R.; Jowitt, Thomas A.; Howard, Marjorie; Carroll, Paul; Parish, Tanya; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1814</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-87</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">CYP144 from Mycobacterium tuberculosis was expressed and purified.  CYP144 demonstrates heme thiolate coordination in its ferric form, but the cysteinate is protonated to thiol in both the carbon monoxide-bound and ligand-free ferrous forms (forming P420 in the former).  Tight binding of various azole drugs was shown, with affinity for miconazole (Kd = 0.98 μM), clotrimazole (0.37 μM) and econazole (0.78 μM) being highest.  These azoles are also the trio with the highest affinity for the essential CYP121 and for the cholesterol oxidase CYP125 (essential for host infection), and have high potency as anti-mycobacterial drugs.  Construction of a Mtb gene knockout strain demonstrated that CYP144 is not essential for growth in vitro.  However the deletion strain was more sensitive to azole inhibition in culture suggesting an important role for CYP144 in cell physiol. and/or in mediating azole resistance.  The biophys. and genetic features of CYP144 are compared to those of other characterized Mtb P 450s, identifying both commonality in properties (including thiolate protonation in ferrous P 450s) and intriguing differences in thermodn. and spectroscopic features.  Our developing knowledge of the Mtb P 450s has revealed unusual biochem. and gene essentiality, highlighting their potential as drug targets in this human pathogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizrKdwo_jGLVg90H21EOLACvtfcHk0ljlP7kYR6Np0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygs73I&md5=11d03eff159437d2df871c7ad343ecec</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2010.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2010.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DJowitt%26aufirst%3DT.%2BA.%26aulast%3DHoward%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DP.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DExpression%2520and%2520characterization%2520of%2520Mycobacterium%2520tuberculosis%2520CYP144%253A%2520Common%2520themes%2520and%2520lessons%2520learned%2520in%2520the%2520M.%2520tuberculosis%2520P450%2520enzyme%2520family%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2011%26volume%3D1814%26spage%3D76%26epage%3D87%26doi%3D10.1016%2Fj.bbapap.2010.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Tailleux, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelizzola, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortellaro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Withers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castagnoli, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gicquel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoker, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butcher, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyrolles, O.</span><span> </span><span class="NLM_article-title">Probing host pathogen cross-talk by transcriptional profiling of both Mycobacterium tuberculosis and infected human dendritic cells and macrophages</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1403</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1371%2Fjournal.pone.0001403" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1403&author=L.+Tailleuxauthor=S.+J.+Waddellauthor=M.+Pelizzolaauthor=A.+Mortellaroauthor=M.+Withersauthor=A.+Tanneauthor=P.+R.+Castagnoliauthor=B.+Gicquelauthor=N.+G.+Stokerauthor=P.+D.+Butcherauthor=M.+Fotiauthor=O.+Neyrolles&title=Probing+host+pathogen+cross-talk+by+transcriptional+profiling+of+both+Mycobacterium+tuberculosis+and+infected+human+dendritic+cells+and+macrophages&doi=10.1371%2Fjournal.pone.0001403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001403%26sid%3Dliteratum%253Aachs%26aulast%3DTailleux%26aufirst%3DL.%26aulast%3DWaddell%26aufirst%3DS.%2BJ.%26aulast%3DPelizzola%26aufirst%3DM.%26aulast%3DMortellaro%26aufirst%3DA.%26aulast%3DWithers%26aufirst%3DM.%26aulast%3DTanne%26aufirst%3DA.%26aulast%3DCastagnoli%26aufirst%3DP.%2BR.%26aulast%3DGicquel%26aufirst%3DB.%26aulast%3DStoker%26aufirst%3DN.%2BG.%26aulast%3DButcher%26aufirst%3DP.%2BD.%26aulast%3DFoti%26aufirst%3DM.%26aulast%3DNeyrolles%26aufirst%3DO.%26atitle%3DProbing%2520host%2520pathogen%2520cross-talk%2520by%2520transcriptional%2520profiling%2520of%2520both%2520Mycobacterium%2520tuberculosis%2520and%2520infected%2520human%2520dendritic%2520cells%2520and%2520macrophages%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1403%26doi%3D10.1371%2Fjournal.pone.0001403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Holsclaw, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogi, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilmore, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schelle, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leavell, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertozzi, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leary, J. A.</span><span> </span><span class="NLM_article-title">Structural characterization of a novel sulfated menaquinone produced by stf3 from Mycobacterium tuberculosis</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1021/cb800145r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb800145r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1GjsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=619-624&author=C.+M.+Holsclawauthor=K.+M.+Sogiauthor=S.+A.+Gilmoreauthor=M.+W.+Schelleauthor=M.+D.+Leavellauthor=C.+R.+Bertozziauthor=J.+A.+Leary&title=Structural+characterization+of+a+novel+sulfated+menaquinone+produced+by+stf3+from+Mycobacterium+tuberculosis&doi=10.1021%2Fcb800145r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of a novel sulfated menaquinone produced by stf3 from Mycobacterium tuberculosis</span></div><div class="casAuthors">Holsclaw, Cynthia M.; Sogi, Kimberly M.; Gilmore, Sarah A.; Schelle, Michael W.; Leavell, Michael D.; Bertozzi, Carolyn R.; Leary, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">619-624</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">M. tuberculosis, the causative agent of tuberculosis, produces unique sulfated metabolites assocd. with virulence.  One such metabolite from M. tuberculosis lipid exts., S881, has been shown to neg. regulate the virulence of M. tuberculosis in mouse infection studies, and its cell-surface localization suggests a role in modulating host-pathogen interactions.  However, a detailed structural anal. of S881 has remained elusive.  Here we use high-resoln., high-mass-accuracy, and tandem mass spectrometry to characterize the structure of S881.  Exact mass measurements showed that S881 is highly unsatd., tandem mass spectrometry indicated a polyisoprene-derived structure, and characterization of synthetic structural analogs confirmed that S881 is a previously undescribed sulfated deriv. of dihydromenaquinone-9, the primary quinol electron carrier in M. tuberculosis.  To our knowledge, this is the 1st example of a sulfated menaquinone produced in any prokaryote.  Together with previous studies, these findings suggest that this redox cofactor may play a role in mycobacterial pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTkFafyS9f7Vg90H21EOLACvtfcHk0liFERcILpY0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1GjsbvJ&md5=cf89de5bfdf4d7c68d0b9d119395c1f6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb800145r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb800145r%26sid%3Dliteratum%253Aachs%26aulast%3DHolsclaw%26aufirst%3DC.%2BM.%26aulast%3DSogi%26aufirst%3DK.%2BM.%26aulast%3DGilmore%26aufirst%3DS.%2BA.%26aulast%3DSchelle%26aufirst%3DM.%2BW.%26aulast%3DLeavell%26aufirst%3DM.%2BD.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26aulast%3DLeary%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520characterization%2520of%2520a%2520novel%2520sulfated%2520menaquinone%2520produced%2520by%2520stf3%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2008%26volume%3D3%26spage%3D619%26epage%3D624%26doi%3D10.1021%2Fcb800145r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafite, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pikuleva, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The structure of Mycobacterium tuberculosis CYP125: Molecular basis for cholesterol binding in a P450 needed for host infection</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">35524</span><span class="NLM_x">–</span> <span class="NLM_lpage">35533</span><span class="refDoi"> DOI: 10.1074/jbc.M109.032706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.032706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19846552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35524-35533&author=K.+J.+McLeanauthor=P.+Lafiteauthor=C.+Levyauthor=M.+R.+Cheesmanauthor=N.+Mastauthor=I.+A.+Pikulevaauthor=D.+Leysauthor=A.+W.+Munro&title=The+structure+of+Mycobacterium+tuberculosis+CYP125%3A+Molecular+basis+for+cholesterol+binding+in+a+P450+needed+for+host+infection&doi=10.1074%2Fjbc.M109.032706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Structure of Mycobacterium tuberculosis CYP125: Molecular Basis for Cholesterol Binding in a P450 Needed for Host Infection</span></div><div class="casAuthors">McLean, Kirsty J.; Lafite, Pierre; Levy, Colin; Cheesman, Myles R.; Mast, Natalia; Pikuleva, Irina A.; Leys, David; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">35524-35533</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We report characterization and the crystal structure of the Mycobacterium tuberculosis cytochrome P 450 CYP125, a P 450 implicated in metab. of host cholesterol and essential for establishing infection in mice.  CYP125 is purified in a high spin form and undergoes both type I and II spectral shifts with various azole drugs.  The 1.4-Å structure of ligand-free CYP125 reveals a "letterbox" active site cavity of dimensions appropriate for entry of a polycyclic sterol.  A mixt. of hexa-coordinate and penta-coordinate states could be discerned, with water binding as the 6th heme-ligand linked to conformation of the I-helix Val267 residue.  Structures in complex with androstenedione and the antitubercular drug econazole reveal that binding of hydrophobic ligands occurs within the active site cavity.  Due to the funnel shape of the active site near the heme, neither approaches the heme iron.  A model of the cholesterol-CYP125 complex shows that the alkyl side chain extends toward the heme iron, predicting hydroxylation of cholesterol C27.  The alkyl chain is in close contact to Val267, suggesting a substrate binding-induced low-spin to high-spin transition coupled to reorientation of the latter residue.  Reconstitution of CYP125 activity with a redox partner system revealed exclusively cholesterol 27-hydroxylation, consistent with structure and modeling.  This activity may enable catabolism of host cholesterol or generation of immunomodulatory compds. that enable persistence in the host.  This study reveals structural and catalytic properties of a potential M. tuberculosis drug target enzyme, and the likely mode by which the host-derived substrate is bound and hydroxylated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobJ0KfPDTC-rVg90H21EOLACvtfcHk0liFERcILpY0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnK&md5=86dd9f8bcf0072432b6e47ee937e3edd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.032706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.032706%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DLevy%26aufirst%3DC.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DMast%26aufirst%3DN.%26aulast%3DPikuleva%26aufirst%3DI.%2BA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520structure%2520of%2520Mycobacterium%2520tuberculosis%2520CYP125%253A%2520Molecular%2520basis%2520for%2520cholesterol%2520binding%2520in%2520a%2520P450%2520needed%2520for%2520host%2520infection%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35524%26epage%3D35533%26doi%3D10.1074%2Fjbc.M109.032706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Capyk, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalscheuer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eltis, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohn, W. W.</span><span> </span><span class="NLM_article-title">Mycobacterial cytochrome P450 125 (Cyp125) catalyzes the terminal hydroxylation of C27 steroids</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">35534</span><span class="NLM_x">–</span> <span class="NLM_lpage">35542</span><span class="refDoi"> DOI: 10.1074/jbc.M109.072132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.072132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19846551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35534-35542&author=J.+K.+Capykauthor=R.+Kalscheuerauthor=G.+R.+Stewartauthor=J.+Liuauthor=H.+Kwonauthor=R.+Zhaoauthor=S.+Okamotoauthor=W.+R.+Jacobsauthor=L.+D.+Eltisauthor=W.+W.+Mohn&title=Mycobacterial+cytochrome+P450+125+%28Cyp125%29+catalyzes+the+terminal+hydroxylation+of+C27+steroids&doi=10.1074%2Fjbc.M109.072132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mycobacterial cytochrome P450 125 (Cyp125) catalyzes the terminal hydroxylation of C27 steroids</span></div><div class="casAuthors">Capyk, Jenna K.; Kalscheuer, Rainer; Stewart, Gordon R.; Liu, Jie; Kwon, Hyukin; Zhao, Rafael; Okamoto, Sachi; Jacobs, William R., Jr.; Eltis, Lindsay D.; Mohn, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">35534-35542</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cyp125 (Rv3545c), a cytochrome P 450, is encoded as part of the cholesterol degrdn. gene cluster conserved among members of the Mycobacterium tuberculosis complex.  This enzyme has been implicated in mycobacterial pathogenesis, and a homolog initiates cholesterol catabolism in the soil actinomycete Rhodococcus jostii RHA1.  In Mycobacterium bovis BCG, cyp125 was up-regulated 7.1-fold with growth on cholesterol.  A cyp125 deletion mutant of BCG did not grow on cholesterol and accumulated 4-cholesten-3-one when incubated in the presence of cholesterol.  Wild-type BCG grew on this metabolite.  By contrast, a parallel cyp125 deletion mutation of M. tuberculosis H37Rv did not affect growth on cholesterol.  Purified Cyp125 from M. tuberculosis, heterologously produced in R. jostii RHA1, bound cholesterol and 4-cholesten-3-one with apparent dissocn. consts. of 0.20 ± 0.02 μM and 0.27 ± 0.05 μM, resp.  When reconstituted with KshB, the cognate reductase of the ketosteroid 9α-hydroxylase, Cyp125 catalyzed the hydroxylation of these steroids.  MS and NMR analyses revealed that hydroxylation occurred at carbon 26 of the steroid side chain, allowing unambiguous classification of Cyp125 as a steroid C26-hydroxylase.  This study establishes the catalytic function of Cyp125 and, in identifying an important difference in the catabolic potential of M. bovis and M. tuberculosis, suggests that Cyp125 may have an addnl. function in pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqInd-ojV5-urVg90H21EOLACvtfcHk0liFERcILpY0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCqurnL&md5=3fb8869a235726445347ac42fe981722</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.072132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.072132%26sid%3Dliteratum%253Aachs%26aulast%3DCapyk%26aufirst%3DJ.%2BK.%26aulast%3DKalscheuer%26aufirst%3DR.%26aulast%3DStewart%26aufirst%3DG.%2BR.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DJacobs%26aufirst%3DW.%2BR.%26aulast%3DEltis%26aufirst%3DL.%2BD.%26aulast%3DMohn%26aufirst%3DW.%2BW.%26atitle%3DMycobacterial%2520cytochrome%2520P450%2520125%2520%2528Cyp125%2529%2520catalyzes%2520the%2520terminal%2520hydroxylation%2520of%2520C27%2520steroids%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35534%26epage%3D35542%26doi%3D10.1074%2Fjbc.M109.072132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Johnston, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kells, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Biochemical and structural characterization of CYP124: a methyl-branched lipid omega-hydroxylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">20687</span><span class="NLM_x">–</span> <span class="NLM_lpage">20692</span><span class="refDoi"> DOI: 10.1073/pnas.0907398106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.0907398106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19933331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFOntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=20687-20692&author=J.+B.+Johnstonauthor=P.+M.+Kellsauthor=L.+M.+Podustauthor=P.+R.+Ortiz+de+Montellano&title=Biochemical+and+structural+characterization+of+CYP124%3A+a+methyl-branched+lipid+omega-hydroxylase+from+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.0907398106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and structural characterization of CYP124: a methyl-branched lipid ω-hydroxylase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Johnston, Jonathan B.; Kells, Petrea M.; Podust, Larissa M.; Ortiz de Montellano, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">20687-20692, S20687/1-S20687/29</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mycobacterium tuberculosis (Mtb) produces a variety of methyl-branched lipids that serve important functions, including modulating the immune response during pathogenesis and contributing to a robust cell wall that is impermeable to many chem. agents.  Here, we report characterization of Mtb CYP124 (Rv2266) that includes demonstration of preferential oxidn. of methyl-branched lipids.  Spectrophotometric titrns. and anal. of reaction products indicate that CYP124 tightly binds and hydroxylates these substrates at the chem. disfavored ω-position.  We also report x-ray crystal structures of the ligand-free and phytanic acid-bound protein at a resoln. of 1.5 Å and 2.1 Å, resp., which provide structural insights into a cytochrome P 450 with predominant ω-hydroxylase activity.  The structures of ligand-free and substrate-bound CYP124 reveal several differences induced by substrate binding, including reorganization of the I helix and closure of the active site by elements of the F, G, and D helixes that bind the substrate and exclude solvent from the hydrophobic active site cavity.  The obsd. regiospecific catalytic activity suggests roles of CYP124 in the physiol. oxidn. of relevant Mtb methyl-branched lipids.  The enzymic specificity and structures reported here provide a scaffold for the design and testing of specific inhibitors of CYP124.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmsqzu2qXOKLVg90H21EOLACvtfcHk0ljrpy-AdXqL1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFOntA%253D%253D&md5=74dd7e0f4dff778c6af163b4961a99cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0907398106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0907398106%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DJ.%2BB.%26aulast%3DKells%26aufirst%3DP.%2BM.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DBiochemical%2520and%2520structural%2520characterization%2520of%2520CYP124%253A%2520a%2520methyl-branched%2520lipid%2520omega-hydroxylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D20687%26epage%3D20692%26doi%3D10.1073%2Fpnas.0907398106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Gondry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauguet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amouroux, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuphile, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courçon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lautru, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernodet, J.-L.</span><span> </span><span class="NLM_article-title">Cyclodipeptide synthases are a family of tRNA-dependent peptide bond-forming enzymes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1038/nchembio.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnchembio.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19430487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1Cmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=414-420&author=M.+Gondryauthor=L.+Sauguetauthor=P.+Belinauthor=R.+Thaiauthor=R.+Amourouxauthor=C.+Tellierauthor=K.+Tuphileauthor=M.+Jacquetauthor=S.+Braudauthor=M.+Cour%C3%A7onauthor=C.+Massonauthor=S.+Duboisauthor=S.+Lautruauthor=A.+Lecoqauthor=S.+Hashimotoauthor=R.+Genetauthor=J.-L.+Pernodet&title=Cyclodipeptide+synthases+are+a+family+of+tRNA-dependent+peptide+bond-forming+enzymes&doi=10.1038%2Fnchembio.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclodipeptide synthases are a family of tRNA-dependent peptide bond-forming enzymes</span></div><div class="casAuthors">Gondry, Muriel; Sauguet, Ludovic; Belin, Pascal; Thai, Robert; Amouroux, Rachel; Tellier, Carine; Tuphile, Karine; Jacquet, Mickael; Braud, Sandrine; Courcon, Marie; Masson, Cedric; Dubois, Steven; Lautru, Sylvie; Lecoq, Alain; Hashimoto, Shin-ichi; Genet, Roger; Pernodet, Jean-Luc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">414-420</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclodipeptides and their derivs. belong to the diketopiperazine (DKP) family, which is comprised of a broad array of natural products that exhibit useful biol. properties.  In the few known DKP biosynthetic pathways, nonribosomal peptide synthetases (NRPSs) are involved in the synthesis of cyclodipeptides that constitute the DKP scaffold, except in the albonoursin (1) pathway.  Albonoursin, or cyclo(α,β-dehydroPhe-α,β-dehydroLeu), is an antibacterial DKP produced by Streptomyces noursei.  In this pathway, the formation of the cyclo(Phe-Leu) (2) intermediate is catalyzed by AlbC, a small protein unrelated to NRPSs.  We demonstrated that AlbC uses aminoacyl-tRNAs as substrates to catalyze the formation of the DKP peptide bonds.  Moreover, several other bacterial proteins, presenting moderate similarity to AlbC, also use aminoacyl-tRNAs to synthesize various cyclodipeptides.  Therefore, AlbC and these related proteins belong to a newly defined family of enzymes that we have named cyclodipeptide synthases (CDPSs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqok2iBeYtA37Vg90H21EOLACvtfcHk0ljrpy-AdXqL1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1Cmtrs%253D&md5=760e57619b188f6232c4c5be8cef8422</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.175%26sid%3Dliteratum%253Aachs%26aulast%3DGondry%26aufirst%3DM.%26aulast%3DSauguet%26aufirst%3DL.%26aulast%3DBelin%26aufirst%3DP.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DAmouroux%26aufirst%3DR.%26aulast%3DTellier%26aufirst%3DC.%26aulast%3DTuphile%26aufirst%3DK.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DBraud%26aufirst%3DS.%26aulast%3DCour%25C3%25A7on%26aufirst%3DM.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DDubois%26aufirst%3DS.%26aulast%3DLautru%26aufirst%3DS.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DHashimoto%26aufirst%3DS.%26aulast%3DGenet%26aufirst%3DR.%26aulast%3DPernodet%26aufirst%3DJ.-L.%26atitle%3DCyclodipeptide%2520synthases%2520are%2520a%2520family%2520of%2520tRNA-dependent%2520peptide%2520bond-forming%2520enzymes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D414%26epage%3D420%26doi%3D10.1038%2Fnchembio.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Belin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Du, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fielding, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lequin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charbonnier, J.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courçon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugave, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernodet, J.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gondry, M.</span><span> </span><span class="NLM_article-title">Identification and structural basis of the reaction catalyzed by CYP121, an essential cytochrome P450 in Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7426</span><span class="NLM_x">–</span> <span class="NLM_lpage">7431</span><span class="refDoi"> DOI: 10.1073/pnas.0812191106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.0812191106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19416919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD1MzitlGitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7426-7431&author=P.+Belinauthor=M.+H.+Le+Duauthor=A.+Fieldingauthor=O.+Lequinauthor=M.+Jacquetauthor=J.-B.+Charbonnierauthor=A.+Lecoqauthor=R.+Thaiauthor=M.+Cour%C3%A7onauthor=C.+Massonauthor=C.+Dugaveauthor=R.+Genetauthor=J.-L.+Pernodetauthor=M.+Gondry&title=Identification+and+structural+basis+of+the+reaction+catalyzed+by+CYP121%2C+an+essential+cytochrome+P450+in+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.0812191106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and structural basis of the reaction catalyzed by CYP121, an essential cytochrome P450 in Mycobacterium tuberculosis</span></div><div class="casAuthors">Belin Pascal; Le Du Marie Helene; Fielding Alistair; Lequin Olivier; Jacquet Mickael; Charbonnier Jean-Baptiste; Lecoq Alain; Thai Robert; Courcon Marie; Masson Cedric; Dugave Christophe; Genet Roger; Pernodet Jean-Luc; Gondry Muriel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7426-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The gene encoding the cytochrome P450 CYP121 is essential for Mycobacterium tuberculosis.  However, the CYP121 catalytic activity remains unknown.  Here, we show that the cyclodipeptide cyclo(l-Tyr-l-Tyr) (cYY) binds to CYP121, and is efficiently converted into a single major product in a CYP121 activity assay containing spinach ferredoxin and ferredoxin reductase.  NMR spectroscopy analysis of the reaction product shows that CYP121 catalyzes the formation of an intramolecular C-C bond between 2 tyrosyl carbon atoms of cYY resulting in a novel chemical entity.  The X-ray structure of cYY-bound CYP121, solved at high resolution (1.4 A), reveals one cYY molecule with full occupancy in the large active site cavity.  One cYY tyrosyl approaches the heme and establishes a specific H-bonding network with Ser-237, Gln-385, Arg-386, and 3 water molecules, including the sixth iron ligand.  These observations are consistent with low temperature EPR spectra of cYY-bound CYP121 showing a change in the heme environment with the persistence of the sixth heme iron ligand.  As the carbon atoms involved in the final C-C coupling are located 5.4 A apart according to the CYP121-cYY complex crystal structure, we propose that C-C coupling is concomitant with substrate tyrosyl movements.  This study provides insight into the catalytic activity, mechanism, and biological function of CYP121.  Also, it provides clues for rational design of putative CYP121 substrate-based antimycobacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD1RfRZi76rY2y4bB4mnvYfW6udTcc2eak1k0uyoTbELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzitlGitA%253D%253D&md5=9ea72ef7c9e8ff65a47d050300c36808</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812191106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812191106%26sid%3Dliteratum%253Aachs%26aulast%3DBelin%26aufirst%3DP.%26aulast%3DLe%2BDu%26aufirst%3DM.%2BH.%26aulast%3DFielding%26aufirst%3DA.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DCharbonnier%26aufirst%3DJ.-B.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DCour%25C3%25A7on%26aufirst%3DM.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DDugave%26aufirst%3DC.%26aulast%3DGenet%26aufirst%3DR.%26aulast%3DPernodet%26aufirst%3DJ.-L.%26aulast%3DGondry%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520structural%2520basis%2520of%2520the%2520reaction%2520catalyzed%2520by%2520CYP121%252C%2520an%2520essential%2520cytochrome%2520P450%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7426%26epage%3D7431%26doi%3D10.1073%2Fpnas.0812191106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seward, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neeli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsollier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenkrands, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parish, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Characterization of active site structure in CYP121: A cytochrome P450 essential for viability of Mycobacterium tuberculosis H37Rv</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">33406</span><span class="NLM_x">–</span> <span class="NLM_lpage">33416</span><span class="refDoi"> DOI: 10.1074/jbc.M802115200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M802115200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=33406-33416&author=K.+J.+McLeanauthor=P.+Carrollauthor=D.+G.+Lewisauthor=A.+J.+Dunfordauthor=H.+E.+Sewardauthor=R.+Neeliauthor=M.+R.+Cheesmanauthor=L.+Marsollierauthor=P.+Douglasauthor=W.+E.+Smithauthor=I.+Rosenkrandsauthor=S.+T.+Coleauthor=D.+Leysauthor=T.+Parishauthor=A.+W.+Munro&title=Characterization+of+active+site+structure+in+CYP121%3A+A+cytochrome+P450+essential+for+viability+of+Mycobacterium+tuberculosis+H37Rv&doi=10.1074%2Fjbc.M802115200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802115200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802115200%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCarroll%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DD.%2BG.%26aulast%3DDunford%26aufirst%3DA.%2BJ.%26aulast%3DSeward%26aufirst%3DH.%2BE.%26aulast%3DNeeli%26aufirst%3DR.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DMarsollier%26aufirst%3DL.%26aulast%3DDouglas%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DW.%2BE.%26aulast%3DRosenkrands%26aufirst%3DI.%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DCharacterization%2520of%2520active%2520site%2520structure%2520in%2520CYP121%253A%2520A%2520cytochrome%2520P450%2520essential%2520for%2520viability%2520of%2520Mycobacterium%2520tuberculosis%2520H37Rv%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D33406%26epage%3D33416%26doi%3D10.1074%2Fjbc.M802115200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neeli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Structure, function and drug targeting in Mycobacterium tuberculosis cytochrome P450 systems</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1016/j.abb.2007.03.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.abb.2007.03.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2007&pages=228-240&author=K.+J.+McLeanauthor=A.+J.+Dunfordauthor=R.+Neeliauthor=M.+D.+Driscollauthor=A.+W.+Munro&title=Structure%2C+function+and+drug+targeting+in+Mycobacterium+tuberculosis+cytochrome+P450+systems&doi=10.1016%2Fj.abb.2007.03.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2007.03.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2007.03.026%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DDunford%26aufirst%3DA.%2BJ.%26aulast%3DNeeli%26aufirst%3DR.%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520function%2520and%2520drug%2520targeting%2520in%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520systems%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2007%26volume%3D464%26spage%3D228%26epage%3D240%26doi%3D10.1016%2Fj.abb.2007.03.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belcher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golovanova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">The Mycobacterium tuberculosis cytochromes P450</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1339</span><span class="NLM_x">–</span> <span class="NLM_lpage">1353</span><span class="refDoi"> DOI: 10.4155/fmc.10.216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.4155%2Ffmc.10.216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=21426022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGru7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1339-1353&author=K.+J.+McLeanauthor=J.+Belcherauthor=M.+D.+Driscollauthor=C.+C.+Fernandezauthor=D.+L.+Vanauthor=S.+Buiauthor=M.+Golovanovaauthor=A.+W.+Munro&title=The+Mycobacterium+tuberculosis+cytochromes+P450&doi=10.4155%2Ffmc.10.216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Mycobacterium tuberculosis cytochromes P450: physiology, biochemistry & molecular intervention</span></div><div class="casAuthors">McLean, Kirsty J.; Belcher, James; Driscoll, Max D.; Fernandez, Christine C.; Le Van, Duyet; Bui, Soi; Golovanova, Marina; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1339-1353</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The human pathogen Mycobacterium tuberculosis (Mtb) encodes 20 cytochrome P 450 enzymes.  Gene essentiality for viability or host infection was demonstrated for Mtb P450s CYP128, CYP121, and CYP125.  Structure/function studies on Mtb P450s revealed key roles contributing to bacterial virulence and persistence in the host.  Various azole-class drugs bind with high affinity to the Mtb P 450 heme and are potent Mtb antibiotics.  This paper reviews the current understanding of the biochem. of Mtb P450s, their interactions with azoles and their potential as novel Mtb drug targets.  Mtb multidrug resistance is widespread and novel therapeutics are desperately needed.  Simultaneous drug targeting of several Mtb P450s crucial to bacterial viability/persistence could offer a new route to effective antibiotics and minimize the development of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAbxeIc99WbVg90H21EOLACvtfcHk0lg20k5xCJWNNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGru7vJ&md5=436069b67f128151ffb82db04a6e1a4a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.216%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DBelcher%26aufirst%3DJ.%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DFernandez%26aufirst%3DC.%2BC.%26aulast%3DVan%26aufirst%3DD.%2BL.%26aulast%3DBui%26aufirst%3DS.%26aulast%3DGolovanova%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DThe%2520Mycobacterium%2520tuberculosis%2520cytochromes%2520P450%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1339%26epage%3D1353%26doi%3D10.4155%2Ffmc.10.216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">Azole antifungals as novel chemotherapeutic agents against murine tuberculosis</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1111/j.1574-6968.2006.00350.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1111%2Fj.1574-6968.2006.00350.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2006&pages=181-186&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=Azole+antifungals+as+novel+chemotherapeutic+agents+against+murine+tuberculosis&doi=10.1111%2Fj.1574-6968.2006.00350.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6968.2006.00350.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6968.2006.00350.x%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DAzole%2520antifungals%2520as%2520novel%2520chemotherapeutic%2520agents%2520against%2520murine%2520tuberculosis%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2006%26volume%3D261%26spage%3D181%26epage%3D186%26doi%3D10.1111%2Fj.1574-6968.2006.00350.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">In vitro and ex vivo antimycobacterial potential of azole drugs against Mycobacterium tuberculosis H37Rv</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">251</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.femsle.2005.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.femsle.2005.07.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=2005&pages=19-22&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=In+vitro+and+ex+vivo+antimycobacterial+potential+of+azole+drugs+against+Mycobacterium+tuberculosis+H37Rv&doi=10.1016%2Fj.femsle.2005.07.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.femsle.2005.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.femsle.2005.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DIn%2520vitro%2520and%2520ex%2520vivo%2520antimycobacterial%2520potential%2520of%2520azole%2520drugs%2520against%2520Mycobacterium%2520tuberculosis%2520H37Rv%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2005%26volume%3D251%26spage%3D19%26epage%3D22%26doi%3D10.1016%2Fj.femsle.2005.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivers, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Expression, purification and spectroscopic characterization of the cytochrome P450 CYP121 from Mycobacterium tuberculosis</span> <span class="citation_source-journal">J. Inorg. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1016/S0162-0134(02)00479-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0162-0134%2802%2900479-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2002&pages=527-541&author=K.+J.+McLeanauthor=M.+R.+Cheesmanauthor=S.+L.+Riversauthor=A.+Richmondauthor=D.+Leysauthor=S.+K.+Chapmanauthor=G.+A.+Reidauthor=N.+C.+Priceauthor=S.+M.+Kellyauthor=J.+Clarksonauthor=W.+E.+Smithauthor=A.+W.+Munro&title=Expression%2C+purification+and+spectroscopic+characterization+of+the+cytochrome+P450+CYP121+from+Mycobacterium+tuberculosis&doi=10.1016%2FS0162-0134%2802%2900479-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0162-0134%2802%2900479-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0162-0134%252802%252900479-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DRivers%26aufirst%3DS.%2BL.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DReid%26aufirst%3DG.%2BA.%26aulast%3DPrice%26aufirst%3DN.%2BC.%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DClarkson%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DW.%2BE.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DExpression%252C%2520purification%2520and%2520spectroscopic%2520characterization%2520of%2520the%2520cytochrome%2520P450%2520CYP121%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Inorg.%2520Biochem.%26date%3D2002%26volume%3D91%26spage%3D527%26epage%3D541%26doi%3D10.1016%2FS0162-0134%2802%2900479-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ahmad, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuller, G. K.</span><span> </span><span class="NLM_article-title">Chemotherapeutic evaluation of alginate nanoparticle-encapsulated azole antifungal and antitubercular drugs against murine tuberculosis</span> <span class="citation_source-journal">Nanomedicine</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1016/j.nano.2007.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.nano.2007.05.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=239-243&author=Z.+Ahmadauthor=S.+Sharmaauthor=G.+K.+Khuller&title=Chemotherapeutic+evaluation+of+alginate+nanoparticle-encapsulated+azole+antifungal+and+antitubercular+drugs+against+murine+tuberculosis&doi=10.1016%2Fj.nano.2007.05.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.nano.2007.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nano.2007.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DZ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKhuller%26aufirst%3DG.%2BK.%26atitle%3DChemotherapeutic%2520evaluation%2520of%2520alginate%2520nanoparticle-encapsulated%2520azole%2520antifungal%2520and%2520antitubercular%2520drugs%2520against%2520murine%2520tuberculosis%26jtitle%3DNanomedicine%26date%3D2007%26volume%3D3%26spage%3D239%26epage%3D243%26doi%3D10.1016%2Fj.nano.2007.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Como, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dismukes, W. E.</span><span> </span><span class="NLM_article-title">Oral azole drugs as systemic antifungal therapy</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">330</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1056/NEJM199401273300407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1056%2FNEJM199401273300407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=8272088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADyaK2c%252FpsleksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=1994&pages=263-272&author=J.+A.+Comoauthor=W.+E.+Dismukes&title=Oral+azole+drugs+as+systemic+antifungal+therapy&doi=10.1056%2FNEJM199401273300407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Oral azole drugs as systemic antifungal therapy</span></div><div class="casAuthors">Como J A; Dismukes W E</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-72</span>
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">The oral azole drugs--ketoconazole, fluconazole, and itraconazole--represent a major advance in systemic antifungal therapy.  Among the three, fluconazole has the most attractive pharmacologic profile, including the capacity to produce high concentrations of active drug in cerebrospinal fluid and urine.  Ketoconazole, the first oral azole to be introduced, is less well tolerated than either fluconazole or itraconazole and is associated with more clinically important toxic effects, including hepatitis and inhibition of steroid hormone synthesis.  However, ketoconazole is less expensive than fluconazole and itraconazole--an especially important consideration for patients receiving long-term therapy.  All three drugs are effective alternatives to amphotericin B and flucytosine as therapy for selected systemic mycoses.  Ketoconazole and itraconazole are effective in patients with the chronic, indolent forms of the endemic mycoses, including blastomycosis, coccidioidomycosis, and histoplasmosis; itraconazole is also effective in patients with sporotrichosis.  Fluconazole is useful in the common forms of fungal meningitis--namely, coccidioidal and cryptococcal meningitis.  In addition, fluconazole is effective for selected patients with serious candida syndromes such as candidemia, and itraconazole is the most effective of the azoles for the treatment of aspergillosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJRByzIex4HNenUz0lpNqnfW6udTcc2eayyD3sTE7sKrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c%252FpsleksQ%253D%253D&md5=7d975fcad941723d2bdfb53882050521</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2FNEJM199401273300407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM199401273300407%26sid%3Dliteratum%253Aachs%26aulast%3DComo%26aufirst%3DJ.%2BA.%26aulast%3DDismukes%26aufirst%3DW.%2BE.%26atitle%3DOral%2520azole%2520drugs%2520as%2520systemic%2520antifungal%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1994%26volume%3D330%26spage%3D263%26epage%3D272%26doi%3D10.1056%2FNEJM199401273300407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent developments in fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">3680</span><span class="refDoi"> DOI: 10.1021/jm8000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-3680&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+developments+in+fragment-based+drug+discovery&doi=10.1021%2Fjm8000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments in fragment-based drug discovery</span></div><div class="casAuthors">Congreve, Miles; Chessari, Gianni; Tisi, Dominic; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3661-3680</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The field of fragment-based drug discovery (FBDD) has developed significantly over the past 10 years and is now recognized as a tangible alternative to more traditional methods of hit identification, such as high throughput screening (HTS).  The authors discuss trends and developments of FBDD over the past 4 years.  Addnl., this review looks in some detail at representative protein-ligand complexes obsd. between fragment-sized mols. and their protein targets from the Protein Data Bank (PDB).  More than the underlying technol. itself, the legacy of FBDD will be conceptual and that "less is more"; breaking down the problem of finding a drug mol. into its smallest component parts opens up new opportunities to solve challenging problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_YtOS7_glD7Vg90H21EOLACvtfcHk0lgok1B3r7Ec_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D&md5=41dbcf884305a190bffc683ae5ed9e45</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520developments%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D3680%26doi%3D10.1021%2Fjm8000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lgok1B3r7Ec_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">5003</span><span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lg5FSrFaPd81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Hall, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Efficient exploration of chemical space by fragment-based screening</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">82</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.pbiomolbio.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=25268064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ymu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=82-91&author=R.+J.+Hallauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Efficient+exploration+of+chemical+space+by+fragment-based+screening&doi=10.1016%2Fj.pbiomolbio.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient exploration of chemical space by fragment-based screening</span></div><div class="casAuthors">Hall, Richard J.; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">82-91</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Screening methods seek to sample a vast chem. space in order to identify starting points for further chem. optimization.  Fragment based drug discovery exploits the superior sampling of chem. space that can be achieved when the mol. wt. is restricted.  Here we show that com. available fragment space is still relatively poorly sampled and argue for highly sensitive screening methods to allow the detection of smaller fragments.  We analyze the properties of our fragment library vs. the properties of X-ray hits derived from the library.  We particularly consider properties related to the degree of planarity of the fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdghyX93HMZLVg90H21EOLACvtfcHk0lg5FSrFaPd81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ymu7zO&md5=a63e451443207393b781d3d68b10bcd1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DEfficient%2520exploration%2520of%2520chemical%2520space%2520by%2520fragment-based%2520screening%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26spage%3D82%26epage%3D91%26doi%3D10.1016%2Fj.pbiomolbio.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery and design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1021/ci400731w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400731w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=693-704&author=D.+Joseph-McCarthyauthor=A.+J.+Campbellauthor=G.+Kernauthor=D.+Moustakas&title=Fragment-based+lead+discovery+and+design&doi=10.1021%2Fci400731w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery and Design</span></div><div class="casAuthors">Joseph-McCarthy, Diane; Campbell, Arthur J.; Kern, Gunther; Moustakas, Demetri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">693-704</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery and design has and continues to show increasing promise in drug discovery.  In this article, the current state of the art in terms of hot-spot characterization, fragment screening techniques, and fragment-based design is discussed.  Three overall fragment-based lead generation strategies are explored and involve the chem. biol. characterization of biol. targets via fragment screening, fragment screening as a complementary approach to high-throughput screening of drug-like compds., and direct fragment-based drug discovery, resp.  The evolution and development of fragment libraries is described.  With an emphasis on computational approaches and the strategies applied at AstraZeneca, the review illustrates how integration of data from one regime can inform the design of expts. in the other, ultimately leading to the discovery of high quality chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCa5krM_w737Vg90H21EOLACvtfcHk0lg5FSrFaPd81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D&md5=31b72219dfd8cec7e3612de1b61f8150</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci400731w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400731w%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DKern%26aufirst%3DG.%26aulast%3DMoustakas%26aufirst%3DD.%26atitle%3DFragment-based%2520lead%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D693%26epage%3D704%26doi%3D10.1021%2Fci400731w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span> </span><span class="NLM_article-title">Vemurafenib: the first drug approved for BRAF-mutant cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+the+first+drug+approved+for+BRAF-mutant+cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0lhcz0QeGPyG-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520the%2520first%2520drug%2520approved%2520for%2520BRAF-mutant%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowat, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkinshaw, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Atomic structure of Mycobacterium tuberculosis CYP121 to 1.06 Å reveals novel features of cytochrome P450</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">5141</span><span class="NLM_x">–</span> <span class="NLM_lpage">5147</span><span class="refDoi"> DOI: 10.1074/jbc.M209928200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M209928200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=5141-5147&author=D.+Leysauthor=C.+G.+Mowatauthor=K.+J.+McLeanauthor=A.+Richmondauthor=S.+K.+Chapmanauthor=M.+D.+Walkinshawauthor=A.+W.+Munro&title=Atomic+structure+of+Mycobacterium+tuberculosis+CYP121+to+1.06+%C3%85+reveals+novel+features+of+cytochrome+P450&doi=10.1074%2Fjbc.M209928200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M209928200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M209928200%26sid%3Dliteratum%253Aachs%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMowat%26aufirst%3DC.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DAtomic%2520structure%2520of%2520Mycobacterium%2520tuberculosis%2520CYP121%2520to%25201.06%2520%25C3%2585%2520reveals%2520novel%2520features%2520of%2520cytochrome%2520P450%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D5141%26epage%3D5147%26doi%3D10.1074%2Fjbc.M209928200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Application of fragment screening and merging to the discovery of inhibitors of the Mycobacterium tuberculosis cytochrome P450 CYP121</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">9311</span><span class="NLM_x">–</span> <span class="NLM_lpage">9316</span><span class="refDoi"> DOI: 10.1002/anie.201202544</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fanie.201202544" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOmt7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=9311-9316&author=S.+A.+Hudsonauthor=K.+J.+McLeanauthor=S.+Suradeauthor=Y.+Q.+Yangauthor=D.+Leysauthor=A.+Ciulliauthor=A.+W.+Munroauthor=C.+Abell&title=Application+of+fragment+screening+and+merging+to+the+discovery+of+inhibitors+of+the+Mycobacterium+tuberculosis+cytochrome+P450+CYP121&doi=10.1002%2Fanie.201202544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment Screening and Merging to the Discovery of Inhibitors of the Mycobacterium tuberculosis Cytochrome P450 CYP121</span></div><div class="casAuthors">Hudson, Sean A.; McLean, Kirsty J.; Surade, Sachin; Yang, Yong-Qing; Leys, David; Ciulli, Alessio; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9311-9316, S9311/1-S9311/27</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This study represents the first successful application of fragment-based approaches to a cytochrome P 450.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZpopos9EebVg90H21EOLACvtfcHk0lhcz0QeGPyG-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOmt7bO&md5=e90a7694c1aa055db088d855705413af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201202544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201202544%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%2BQ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520and%2520merging%2520to%2520the%2520discovery%2520of%2520inhibitors%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520CYP121%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D9311%26epage%3D9316%26doi%3D10.1002%2Fanie.201202544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mclean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Overcoming the limitations of fragment merging: Rescuing a strained merged fragment series targeting mycobacterium tuberculosis CYP121</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1456</span><span class="refDoi"> DOI: 10.1002/cmdc.201300219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcmdc.201300219" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1451-1456&author=S.+A.+Hudsonauthor=S.+Suradeauthor=A.+G.+Coyneauthor=K.+J.+Mcleanauthor=D.+Leysauthor=A.+W.+Munroauthor=C.+Abell&title=Overcoming+the+limitations+of+fragment+merging%3A+Rescuing+a+strained+merged+fragment+series+targeting+mycobacterium+tuberculosis+CYP121&doi=10.1002%2Fcmdc.201300219"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300219%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DMclean%26aufirst%3DK.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DOvercoming%2520the%2520limitations%2520of%2520fragment%2520merging%253A%2520Rescuing%2520a%2520strained%2520merged%2520fragment%2520series%2520targeting%2520mycobacterium%2520tuberculosis%2520CYP121%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1451%26epage%3D1456%26doi%3D10.1002%2Fcmdc.201300219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">The maximal affinity of ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">9997</span><span class="NLM_x">–</span> <span class="NLM_lpage">10002</span><span class="refDoi"> DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of ≈-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0lio1U2bPMeFmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eazpfnDP0cxCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Group efficiency: A guideline for hits-to-leads chemistry</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1179</span><span class="NLM_x">–</span> <span class="NLM_lpage">1180</span><span class="refDoi"> DOI: 10.1002/cmdc.200800132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcmdc.200800132" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1179-1180&author=M.+L.+Verdonkauthor=D.+C.+Rees&title=Group+efficiency%3A+A+guideline+for+hits-to-leads+chemistry&doi=10.1002%2Fcmdc.200800132"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800132%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DGroup%2520efficiency%253A%2520A%2520guideline%2520for%2520hits-to-leads%2520chemistry%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1179%26epage%3D1180%26doi%3D10.1002%2Fcmdc.200800132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">The additivity of substrate fragments in enzyme-ligand binding</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00086-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0969-2126%2898%2900086-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=9687366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK1cXltVSjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=839-848&author=T.+J.+Stoutauthor=C.+R.+Sageauthor=R.+M.+Stroud&title=The+additivity+of+substrate+fragments+in+enzyme-ligand+binding&doi=10.1016%2FS0969-2126%2898%2900086-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The additivity of substrate fragments in enzyme-ligand binding</span></div><div class="casAuthors">Stout, Thomas J.; Sage, Carleton R.; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">839-848</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">Enzymes have evolved to recognize their target substrates with exquisite selectivity and specificity.  Whether fragments of the substrate - perhaps never available to the evolving enzyme - are bound in the same manner as the parent substrate addresses the fundamental basis of specificity.  An understanding of the relative contributions of individual portions of ligand mols. to the enzyme-binding interaction may offer considerable insight into the principles of substrate recognition.  We report 12 crystal structures of Escherichia coli thymidylate synthase in complexes with available fragments of the substrate (dUMP), both with and without the presence of a cofactor analog.  The structures display considerable fidelity of binding mode and interactions.  These complexes reveal several interesting features: the cofactor analog enhances the localization of substrate and substrate fragments near the reactive thiol; the ribose moiety reduces local disorder through addnl. specific enzyme-ligand interactions; the pyrimidine has multiple roles, ranging from stereospecificity to mechanistic competence; and the glycosidic linkage has an important role in the formation of a covalent attachment between substrate and enzyme.  The requirements of ligand-protein binding can be understood in terms of the binding of sep. fragments of the ligand.  Fragments which are subsystems of the natural substrate for the enzyme confer specific contributions to the binding affinity, orientation or electrostatics of the enzymic mechanism.  This ligand-binding anal. provides a complementary method to the more prevalent approaches utilizing site-directed mutagenesis.  In addn., these observations suggest a modular approach for rational drug design utilizing chem. fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3xLbwAs8Qb7Vg90H21EOLACvtfcHk0liNxOXx0H3iiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltVSjs78%253D&md5=ecac0bbb008bcdea46c38403e3fe0c6f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900086-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900086-0%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DSage%26aufirst%3DC.%2BR.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DThe%2520additivity%2520of%2520substrate%2520fragments%2520in%2520enzyme-ligand%2520binding%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D839%26epage%3D848%26doi%3D10.1016%2FS0969-2126%2898%2900086-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Compton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebacz, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katzenellenbogen, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katzenellenbogen, J. A.</span><span> </span><span class="NLM_article-title">Pyrazolo[1,5-a]pyrimidines: estrogen receptor ligands possessing estrogen receptor beta antagonist activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5872</span><span class="NLM_x">–</span> <span class="NLM_lpage">5893</span><span class="refDoi"> DOI: 10.1021/jm049631k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049631k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5872-5893&author=D.+R.+Comptonauthor=S.+Shengauthor=K.+E.+Carlsonauthor=N.+A.+Rebaczauthor=I.+Y.+Leeauthor=B.+S.+Katzenellenbogenauthor=J.+A.+Katzenellenbogen&title=Pyrazolo%5B1%2C5-a%5Dpyrimidines%3A+estrogen+receptor+ligands+possessing+estrogen+receptor+beta+antagonist+activity&doi=10.1021%2Fjm049631k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm049631k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049631k%26sid%3Dliteratum%253Aachs%26aulast%3DCompton%26aufirst%3DD.%2BR.%26aulast%3DSheng%26aufirst%3DS.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DRebacz%26aufirst%3DN.%2BA.%26aulast%3DLee%26aufirst%3DI.%2BY.%26aulast%3DKatzenellenbogen%26aufirst%3DB.%2BS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26atitle%3DPyrazolo%255B1%252C5-a%255Dpyrimidines%253A%2520estrogen%2520receptor%2520ligands%2520possessing%2520estrogen%2520receptor%2520beta%2520antagonist%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5872%26epage%3D5893%26doi%3D10.1021%2Fjm049631k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Johnson, M. D.; Teng, M.; Zhu, J.</span><span> </span><span class="NLM_article-title">Aminopyrazole compounds and use as CHK1 inhibitors</span>. WO2005/00945 A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=M.+D.+Johnson&author=M.+Teng&author=J.+Zhu&title=Aminopyrazole+compounds+and+use+as+CHK1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26atitle%3DAminopyrazole%2520compounds%2520and%2520use%2520as%2520CHK1%2520inhibitors%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0lg4d2RvSJCNNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span> <span class="citation_source-book">Cytochrome P450: Structure, Mechanism and Biochemistry</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span>, Ed.; <span class="NLM_publisher-name">Kluwer Academic/Plenum Publishers</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1007%2Fb139087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=P.+R.+Ortiz+de+Montellano&title=Cytochrome+P450%3A+Structure%2C+Mechanism+and+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fb139087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb139087%26sid%3Dliteratum%253Aachs%26btitle%3DCytochrome%2520P450%253A%2520Structure%252C%2520Mechanism%2520and%2520Biochemistry%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26pub%3DKluwer%2520Academic%252FPlenum%2520Publishers%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span>Glide, <span class="NLM_edition">version 6.5</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide%2C+version+6.5%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Smith, A. J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Beyond picomolar affinities: Quantitative aspects of noncovalent and covalent binding of drugs to proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1021/jm800498e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800498e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1OltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=225-233&author=A.+J.+T.+Smithauthor=X.+Zhangauthor=A.+G.+Leachauthor=K.+N.+Houk&title=Beyond+picomolar+affinities%3A+Quantitative+aspects+of+noncovalent+and+covalent+binding+of+drugs+to+proteins&doi=10.1021%2Fjm800498e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond picomolar affinities: quantitative aspects of noncovalent and covalent binding of drugs to proteins</span></div><div class="casAuthors">Smith, Adam J. T.; Zhang, Xiyun; Leach, Andrew G.; Houk, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review of 104 refs. summarized quant. aspects of noncovalent and covalent binding of drugs to proteins with subtopics including (1) introduction; (2) factors involved in ligand-recepto binding; (3) distribution of exptl. binding consts.; (3) covalent interactions with transition states as the origins of enzyme proficiency; (4) potential quant. advantages of covalent enzyme inhibitors and colent drugs; (5) selected examples of covalent inhibitors and drugs; (6) prospects for the design of covalent drugs; (7) conclusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhUbRxt1fu7Vg90H21EOLACvtfcHk0lhj-sfRZE0ZoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1OltLw%253D&md5=529d4181046100b5e3c37a8fef571218</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm800498e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800498e%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DA.%2BJ.%2BT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DBeyond%2520picomolar%2520affinities%253A%2520Quantitative%2520aspects%2520of%2520noncovalent%2520and%2520covalent%2520binding%2520of%2520drugs%2520to%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D225%26epage%3D233%26doi%3D10.1021%2Fjm800498e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Abdel-Magid, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maryanoff, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, R. D.</span><span> </span><span class="NLM_article-title">Reductive amination of aldehydes and ketones with sodium triacetoxyborohydride. Studies on direct and indirect reductive amination procedures</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">3849</span><span class="NLM_x">–</span> <span class="NLM_lpage">3862</span><span class="refDoi"> DOI: 10.1021/jo960057x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo960057x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK28XivFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=3849-3862&author=A.+F.+Abdel-Magidauthor=K.+G.+Carsonauthor=B.+D.+Harrisauthor=C.+A.+Maryanoffauthor=R.+D.+Shah&title=Reductive+amination+of+aldehydes+and+ketones+with+sodium+triacetoxyborohydride.+Studies+on+direct+and+indirect+reductive+amination+procedures&doi=10.1021%2Fjo960057x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Reductive Amination of Aldehydes and Ketones with Sodium Triacetoxyborohydride. Studies on Direct and Indirect Reductive Amination Procedures</span></div><div class="casAuthors">Abdel-Magid, Ahmed F.; Carson, Kenneth G.; Harris, Bruce D.; Maryanoff, Cynthia A.; Shah, Rekha D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3849-3862</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sodium triacetoxyborohydride is presented as a general reducing agent for the reductive amination of aldehydes and ketones.  Procedures for using this mild and selective reagent have been developed for a wide variety of substrates.  The scope of the reaction includes aliph. acyclic and cyclic ketones, aliph. and arom. aldehydes, and primary and secondary amines including a variety of weakly basic and nonbasic amines.  Limitations include reactions with arom. and unsatd. ketones and some sterically hindered ketones and amines.  1,2-Dichloroethane (DCE) is the preferred reaction solvent, but reactions can also be carried out in THF and occasionally in acetonitrile.  Acetic acid may be used as catalyst with ketone reactions, but it is generally not needed with aldehydes.  The procedure is carried out effectively in the presence of acid sensitive functional groups such as acetals and ketals; it can also be carried out in the presence of reducible functional groups such as C-C multiple bonds and cyano and nitro groups.  Reactions are generally faster in DCE than in THF, and in both solvents, reactions are faster in the presence of AcOH.  In comparison with other reductive amination procedures such as NaBH3CN/MeOH, borane-pyridine, and catalytic hydrogenation, NaBH(OAc)3 gave consistently higher yields and fewer side products.  In the reductive amination of some aldehydes with primary amines where dialkylation is a problem we adopted a stepwise procedure involving imine formation in MeOH followed by redn. with NaBH4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1pj6c2VgVU7Vg90H21EOLACvtfcHk0lhj-sfRZE0ZoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XivFShu7o%253D&md5=9cc14c9adbbb916bd3326cb3e83a9b40</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjo960057x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo960057x%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26aulast%3DCarson%26aufirst%3DK.%2BG.%26aulast%3DHarris%26aufirst%3DB.%2BD.%26aulast%3DMaryanoff%26aufirst%3DC.%2BA.%26aulast%3DShah%26aufirst%3DR.%2BD.%26atitle%3DReductive%2520amination%2520of%2520aldehydes%2520and%2520ketones%2520with%2520sodium%2520triacetoxyborohydride.%2520Studies%2520on%2520direct%2520and%2520indirect%2520reductive%2520amination%2520procedures%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D3849%26epage%3D3862%26doi%3D10.1021%2Fjo960057x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Vasudevan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaskovich, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkanda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebti, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, A. D.</span><span> </span><span class="NLM_article-title">Potent, highly selective, and non-thiol inhibitors of protein geranylgeranyltransferase-I</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1333</span><span class="NLM_x">–</span> <span class="NLM_lpage">1340</span><span class="refDoi"> DOI: 10.1021/jm9900873</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9900873" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1333-1340&author=A.+Vasudevanauthor=Y.+Qianauthor=A.+Vogtauthor=M.+A.+Blaskovichauthor=J.+Ohkandaauthor=S.+M.+Sebtiauthor=A.+D.+Hamilton&title=Potent%2C+highly+selective%2C+and+non-thiol+inhibitors+of+protein+geranylgeranyltransferase-I&doi=10.1021%2Fjm9900873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm9900873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9900873%26sid%3Dliteratum%253Aachs%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DA.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3DPotent%252C%2520highly%2520selective%252C%2520and%2520non-thiol%2520inhibitors%2520of%2520protein%2520geranylgeranyltransferase-I%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1333%26epage%3D1340%26doi%3D10.1021%2Fjm9900873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Schenkman, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remmer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estabrook, R. W.</span><span> </span><span class="NLM_article-title">Spectral studies of drug interaction with hepatic microsomal cytochrome</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1967</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=4382749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaF2sXosVCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1967&pages=113-123&author=J.+B.+Schenkmanauthor=H.+Remmerauthor=R.+W.+Estabrook&title=Spectral+studies+of+drug+interaction+with+hepatic+microsomal+cytochrome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Spectral studies of drug interaction with hepatic microsomal cytochrome</span></div><div class="casAuthors">Schenkman, John B.; Remmer, Herbert; Estabrook, Ronald W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1967</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">113-23</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Two types of spectral changes are described as resulting from substrate interaction with a hepatic microsomal cytochrome; the magnitude of these spectral changes is dependent on protein concn. and substrate concn. as well as the substrate employed.  In 2 of the 3 types of substrates examd. the concn. of substrate necessary to evoke half-max. enzyme activity was similar to the concn. of the same substrate necessary for half-max. spectral changes.  The addn. of NADPH, a corequirement for the microsomal mixed function oxidase, causes a modification of both types of spectral changes, without altering substrate affinity.  Three possible hypotheses are advanced, based upon the exptl. observations, to explain the two types of spectral changes observed.  24 references.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoInUTNOrGCNbVg90H21EOLACvtfcHk0lhj-sfRZE0ZoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2sXosVCjtQ%253D%253D&md5=8f1a923fb13a3c6bbeed43c6d0e80e5e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchenkman%26aufirst%3DJ.%2BB.%26aulast%3DRemmer%26aufirst%3DH.%26aulast%3DEstabrook%26aufirst%3DR.%2BW.%26atitle%3DSpectral%2520studies%2520of%2520drug%2520interaction%2520with%2520hepatic%2520microsomal%2520cytochrome%26jtitle%3DMol.%2520Pharmacol.%26date%3D1967%26volume%3D3%26spage%3D113%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kells, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span> </span><span class="NLM_article-title">Reverse type I inhibitor of Mycobacterium tuberculosis CYP125A1</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">332</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.bmcl.2010.11.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=332-337&author=H.+Ouelletauthor=P.+M.+Kellsauthor=P.+R.+Ortiz+de+Montellanoauthor=L.+M.+Podust&title=Reverse+type+I+inhibitor+of+Mycobacterium+tuberculosis+CYP125A1&doi=10.1016%2Fj.bmcl.2010.11.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3DKells%26aufirst%3DP.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26atitle%3DReverse%2520type%2520I%2520inhibitor%2520of%2520Mycobacterium%2520tuberculosis%2520CYP125A1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D332%26epage%3D337%26doi%3D10.1016%2Fj.bmcl.2010.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fowler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieser, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurcha, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besra, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Azole antifungals are potent inhibitors of cytochrome P450 mono-oxygenases and bacterial growth in mycobacteria and streptomycetes</span> <span class="citation_source-journal">Microbiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">148</span><span class="NLM_x">, </span> <span class="NLM_fpage">2937</span><span class="NLM_x">–</span> <span class="NLM_lpage">2949</span><span class="refDoi"> DOI: 10.1099/00221287-148-10-2937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1099%2F00221287-148-10-2937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12368427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD38XotFymtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2002&pages=2937-2949&author=K.+J.+McLeanauthor=K.+R.+Marshallauthor=A.+Richmondauthor=I.+S.+Hunterauthor=K.+Fowlerauthor=T.+Kieserauthor=S.+S.+Gurchaauthor=G.+S.+Besraauthor=A.+W.+Munro&title=Azole+antifungals+are+potent+inhibitors+of+cytochrome+P450+mono-oxygenases+and+bacterial+growth+in+mycobacteria+and+streptomycetes&doi=10.1099%2F00221287-148-10-2937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Azole antifungals are potent inhibitors of cytochrome P450 monooxygenases and bacterial growth in mycobacteria and streptomycetes</span></div><div class="casAuthors">McLean, Kirsty J.; Marshall, Ker R.; Richmond, Alison; Hunter, Iain S.; Fowler, Kay; Kieser, Tobias; Gurcha, Sudagar S.; Besra, Gurydal S.; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology (Reading, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2937-2949</span>CODEN:
                <span class="NLM_cas:coden">MROBEO</span>;
        ISSN:<span class="NLM_cas:issn">1350-0872</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">The genome sequence of Mycobacterium tuberculosis has revealed the presence of 20 different cytochrome P 450 monooxygenases (P450s) within this organism, and subsequent genome sequences of other mycobacteria and of Streptomyces coelicolor have indicated that these actinomycetes also have large complements of P450s, pointing to important physiol. roles for these enzymes.  The actinomycete P450s include homologues of 14α-sterol demethylases, the targets for the azole class of drugs in yeast and fungi.  Previously, this type of P 450 was considered to be absent from bacteria.  When present at low concns. in growth medium, azole antifungal drugs were shown to be potent inhibitors of the growth of Mycobacterium smegmatis and of Streptomyces strains, indicating that one or more of the P450s in these bacteria were viable drug targets.  The drugs econazole and clotrimazole were most effective against M. smegmatis (MIC values of <0·2 and 0·3 μM, resp.) and were superior inhibitors of mycobacterial growth compared to rifampicin and isoniazid (which had MIC values of 1·2 and 36·5 μM, resp.).  In contrast to their effects on the actinomycetes, the azoles showed minimal effects on the growth of Escherichia coli, which is devoid of P450s.  Azole drugs coordinated tightly to the heme iron in M. tuberculosis H37Rv P450s encoded by genes Rv0764c (the sterol demethylase CYP51) and Rv2276 (CYP121).  However, the azoles had a higher affinity for M. tuberculosis CYP121, with Kd values broadly in line with the MIC values for M. smegmatis.  This suggested that CYP121 may be a more realistic target enzyme for the azole drugs than CYP51, particularly in light of the fact that an S. coelicolor ΔCYP51 strain was viable and showed little difference in its sensitivity to azole drugs compared to the wild-type.  If the azole drugs prove to inhibit a no. of important P450s in M. smegmatis and S. coelicolor, then the likelihood of drug resistance developing in these species should be minimal.  This suggests that azole drug therapy may provide a novel antibiotic strategy against strains of M. tuberculosis that have already developed resistance to isoniazid and other front-line drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkLDhhvksLVg90H21EOLACvtfcHk0liQY1cx2JT_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotFymtrg%253D&md5=249eccb4ffedf2acee629b132d8799b7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1099%2F00221287-148-10-2937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F00221287-148-10-2937%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DRichmond%26aufirst%3DA.%26aulast%3DHunter%26aufirst%3DI.%2BS.%26aulast%3DFowler%26aufirst%3DK.%26aulast%3DKieser%26aufirst%3DT.%26aulast%3DGurcha%26aufirst%3DS.%2BS.%26aulast%3DBesra%26aufirst%3DG.%2BS.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DAzole%2520antifungals%2520are%2520potent%2520inhibitors%2520of%2520cytochrome%2520P450%2520mono-oxygenases%2520and%2520bacterial%2520growth%2520in%2520mycobacteria%2520and%2520streptomycetes%26jtitle%3DMicrobiology%26date%3D2002%26volume%3D148%26spage%3D2937%26epage%3D2949%26doi%3D10.1099%2F00221287-148-10-2937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Yoshida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, Y.</span><span> </span><span class="NLM_article-title">Interaction of azole antifungal agents with cytochrome P-45014DM purified from Saccharomyces cerevisiae microsomes</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">229</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1016/0006-2952(87)90694-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0006-2952%2887%2990694-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1987&pages=229-235&author=Y.+Yoshidaauthor=Y.+Aoyama&title=Interaction+of+azole+antifungal+agents+with+cytochrome+P-45014DM+purified+from+Saccharomyces+cerevisiae+microsomes&doi=10.1016%2F0006-2952%2887%2990694-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2887%2990694-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252887%252990694-0%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DY.%26aulast%3DAoyama%26aufirst%3DY.%26atitle%3DInteraction%2520of%2520azole%2520antifungal%2520agents%2520with%2520cytochrome%2520P-45014DM%2520purified%2520from%2520Saccharomyces%2520cerevisiae%2520microsomes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1987%26volume%3D36%26spage%3D229%26epage%3D235%26doi%3D10.1016%2F0006-2952%2887%2990694-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Conner, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, W. M.</span><span> </span><span class="NLM_article-title">Interactions of cytochrome P450s with their ligands</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.abb.2010.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.abb.2010.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20939998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFegtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2011&pages=56-65&author=K.+P.+Connerauthor=C.+Woodsauthor=W.+M.+Atkins&title=Interactions+of+cytochrome+P450s+with+their+ligands&doi=10.1016%2Fj.abb.2010.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions of cytochrome P 450s with their ligands</span></div><div class="casAuthors">Conner, Kip P.; Woods, Caleb M.; Atkins, William M.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-65</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P 450s (CYPs) are heme-contg. monooxygenases that contribute to an enormous range of enzymic function including biosynthetic and detoxification roles.  Here, the authors summarize recent studies concerning interactions of CYPs with ligands including substrates, inhibitors, and diat. heme-ligating mols.  These studies highlight the complexity in the relation between the heme spin state and active site occupancy, the roles of water in directing protein-ligand and ligand-heme interactions, and the details of interactions between heme and gaseous diat. CYP ligands.  Both kinetic and thermodn. aspects of ligand binding are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDC5lsgk8Hj7Vg90H21EOLACvtfcHk0liQY1cx2JT_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFegtLk%253D&md5=8bf56701a4f12b52dbf3d569ef6731cf</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2010.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2010.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DConner%26aufirst%3DK.%2BP.%26aulast%3DWoods%26aufirst%3DC.%26aulast%3DAtkins%26aufirst%3DW.%2BM.%26atitle%3DInteractions%2520of%2520cytochrome%2520P450s%2520with%2520their%2520ligands%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2011%26volume%3D507%26spage%3D56%26epage%3D65%26doi%3D10.1016%2Fj.abb.2010.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Mycobacterium tuberculosis CYP130: Crystal structure, biophysical characterization, and interactions with antifungal azole drugs</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">5069</span><span class="NLM_x">–</span> <span class="NLM_lpage">5080</span><span class="refDoi"> DOI: 10.1074/jbc.M708734200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M708734200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=5069-5080&author=H.+Ouelletauthor=L.+M.+Podustauthor=P.+R.+Ortiz+de+Montellano&title=Mycobacterium+tuberculosis+CYP130%3A+Crystal+structure%2C+biophysical+characterization%2C+and+interactions+with+antifungal+azole+drugs&doi=10.1074%2Fjbc.M708734200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M708734200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M708734200%26sid%3Dliteratum%253Aachs%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DMycobacterium%2520tuberculosis%2520CYP130%253A%2520Crystal%2520structure%252C%2520biophysical%2520characterization%252C%2520and%2520interactions%2520with%2520antifungal%2520azole%2520drugs%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D5069%26epage%3D5080%26doi%3D10.1074%2Fjbc.M708734200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Wheeler, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Origin of substituent effects in edge-to-face aryl–aryl interactions</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">749</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1080/00268970802537614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1080%2F00268970802537614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20046948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotlWltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2009&pages=749-760&author=S.+E.+Wheelerauthor=K.+N.+Houk&title=Origin+of+substituent+effects+in+edge-to-face+aryl%E2%80%93aryl+interactions&doi=10.1080%2F00268970802537614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Origin of substituent effects in edge-to-face aryl-aryl interactions</span></div><div class="casAuthors">Wheeler, Steven E.; Houk, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">8-12</span>),
    <span class="NLM_cas:pages">749-760</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Substituent effects in the edge-to-face configuration of the benzene dimer have been studied using modern d. functional theory.  An accurate interaction potential energy curve has been computed for the unsubstituted dimer using ab initio methods with large basis sets.  The recommended binding energy for the edge-to-face benzene dimer is 2.31 kcal mol-1, estd. at the counterpoise-cor. CCSD(T)/aug-cc-pVTZ level of theory.  For both edge-ring and face-ring-substituted dimers, interaction energies correlate with σm for the substituents, indicating that substituent effects can be understood qual. in terms of simple electrostatic effects, although in the latter case dispersion results in some scatter in the data.  In contrast to prevailing models of substituent effects in benzene dimers, polarization of the π-system of the substituted ring does not induce substituent effects.  For edge-ring-substituted dimers, substituent effects arise from differential electrostatic interactions between the hydrogens on the substituted ring and the π-cloud of the face ring and direct interactions of the substituents with the unsubstituted ring.  For face-ring-substituted dimers, substituent effects arise from direct electrostatic and dispersion interactions of the substituents with the edge ring.  Substituents with σm > 0.12 favor edge ring substitution while for σm < 0.12 substitution on the face ring is preferred.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo86TqFLnFfV7Vg90H21EOLACvtfcHk0lhYuCpp7eatlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotlWltb8%253D&md5=07465d5ed7b46f562c4b746c839f0bb9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1080%2F00268970802537614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268970802537614%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DS.%2BE.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DOrigin%2520of%2520substituent%2520effects%2520in%2520edge-to-face%2520aryl%25E2%2580%2593aryl%2520interactions%26jtitle%3DMol.%2520Phys.%26date%3D2009%26volume%3D107%26spage%3D749%26epage%3D760%26doi%3D10.1080%2F00268970802537614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserü, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lhYuCpp7eatlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouellet, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Kries, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz de Montellano, P. R.</span><span> </span><span class="NLM_article-title">Interaction of Mycobacterium tuberculosis CYP130 with heterocyclic arylamines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">25211</span><span class="NLM_x">–</span> <span class="NLM_lpage">25219</span><span class="refDoi"> DOI: 10.1074/jbc.M109.017632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M109.017632" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=25211-25219&author=L.+M.+Podustauthor=H.+Ouelletauthor=J.+P.+von+Kriesauthor=P.+R.+Ortiz+de+Montellano&title=Interaction+of+Mycobacterium+tuberculosis+CYP130+with+heterocyclic+arylamines&doi=10.1074%2Fjbc.M109.017632"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.017632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.017632%26sid%3Dliteratum%253Aachs%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DOuellet%26aufirst%3DH.%26aulast%3Dvon%2BKries%26aufirst%3DJ.%2BP.%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3DInteraction%2520of%2520Mycobacterium%2520tuberculosis%2520CYP130%2520with%2520heterocyclic%2520arylamines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D25211%26epage%3D25219%26doi%3D10.1074%2Fjbc.M109.017632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Fonvielle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Du, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lequin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecoq, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grach, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gondry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belin, P.</span><span> </span><span class="NLM_article-title">Substrate and reaction specificity of Mycobacterium tuberculosis cytochrome P450 CYP121: Insights from biochemical studies and crystal structures</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">17347</span><span class="NLM_x">–</span> <span class="NLM_lpage">17359</span><span class="refDoi"> DOI: 10.1074/jbc.M112.443853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M112.443853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=23620594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFamtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=17347-17359&author=M.+Fonvielleauthor=M.+H.+Le+Duauthor=O.+Lequinauthor=A.+Lecoqauthor=M.+Jacquetauthor=R.+Thaiauthor=S.+Duboisauthor=G.+Grachauthor=M.+Gondryauthor=P.+Belin&title=Substrate+and+reaction+specificity+of+Mycobacterium+tuberculosis+cytochrome+P450+CYP121%3A+Insights+from+biochemical+studies+and+crystal+structures&doi=10.1074%2Fjbc.M112.443853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate and Reaction Specificity of Mycobacterium tuberculosis Cytochrome P450 CYP121</span></div><div class="casAuthors">Fonvielle, Matthieu; Marie-Helene Le, D. u; Lequin, Olivier; Lecoq, Alain; Jacquet, Mickael; Thai, Robert; Dubois, Steven; Grach, Guillaume; Gondry, Muriel; Belin, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">17347-17359</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cytochrome P 450 CYP121 is essential for the viability of Mycobacterium tuberculosis.  Studies in vitro show that it can use the cyclodipeptide cyclo(L-Tyr-L-Tyr) (cYY) as a substrate.  We report an investigation of the substrate and reaction specificities of CYP121 involving anal. of the interaction between CYP121 and 14 cYY analogs with various modifications of the side chains or the diketopiperazine (DKP) ring.  Spectral titrn. expts. show that CYP121 significantly bound only cyclodipeptides with a conserved DKP ring carrying two aryl side chains in L-configuration.  CYP121 did not efficiently or selectively transform any of the cYY analogs tested, indicating a high specificity for cYY.  The mol. determinants of this specificity were inferred from both crystal structures of CYP121-analog complexes solved at high resoln. and soln. NMR spectroscopy of the analogs.  Bound cYY or its analogs all displayed a similar set of contacts with CYP121 residues Asn85, Phe168, and Trp182.  The propensity of the cYY tyrosyl to point toward Arg386 was dependent on the presence of the DKP ring that limits the conformational freedom of the ligand.  The correct positioning of the hydroxyl of this tyrosyl was essential for conversion of cYY.  Thus, the specificity of CYP121 results from both a restricted binding specificity and a fine-tuned P 450 substrate relationship.  These results document the catalytic mechanism of CYP121 and improve our understanding of its function in vivo.  This work contributes to progress toward the design of inhibitors of this essential protein of M. tuberculosis that could be used for antituberculosis therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAjF8TY2rD_bVg90H21EOLACvtfcHk0lg2g4VpioTPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFamtbY%253D&md5=f27bb300ba1f5a1e66ff196fe1be113e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.443853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.443853%26sid%3Dliteratum%253Aachs%26aulast%3DFonvielle%26aufirst%3DM.%26aulast%3DLe%2BDu%26aufirst%3DM.%2BH.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DJacquet%26aufirst%3DM.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DS.%26aulast%3DGrach%26aufirst%3DG.%26aulast%3DGondry%26aufirst%3DM.%26aulast%3DBelin%26aufirst%3DP.%26atitle%3DSubstrate%2520and%2520reaction%2520specificity%2520of%2520Mycobacterium%2520tuberculosis%2520cytochrome%2520P450%2520CYP121%253A%2520Insights%2520from%2520biochemical%2520studies%2520and%2520crystal%2520structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D17347%26epage%3D17359%26doi%3D10.1074%2Fjbc.M112.443853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ost, T. W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miles, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lysek, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moser, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutton, P. L.</span><span> </span><span class="NLM_article-title">P450 BM3: the very model of a modern flavocytochrome</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">250</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span><span class="refDoi"> DOI: 10.1016/S0968-0004(02)02086-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS0968-0004%2802%2902086-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12076537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD38XktlSmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2002&pages=250-257&author=A.+W.+Munroauthor=D.+G.+Leysauthor=K.+J.+McLeanauthor=K.+R.+Marshallauthor=T.+W.+B.+Ostauthor=S.+Daffauthor=C.+S.+Milesauthor=S.+K.+Chapmanauthor=D.+A.+Lysekauthor=C.+C.+Moserauthor=C.+C.+Pageauthor=P.+L.+Dutton&title=P450+BM3%3A+the+very+model+of+a+modern+flavocytochrome&doi=10.1016%2FS0968-0004%2802%2902086-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">P 450 BM3: the very model of a modern flavocytochrome</span></div><div class="casAuthors">Munro, Andrew W.; Leys, David G.; McLean, Kirsty J.; Marshall, Ker R.; Ost, Tobias W. B.; Daff, Simon; Miles, Caroline S.; Chapman, Stephen K.; Lysek, Dominikus A.; Moser, Christopher C.; Page, Christopher C.; Dutton, P. Leslie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">250-257</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  Flavocytochrome P 450 BM3 is a bacterial P 450 system in which a fatty acid hydroxylase/monooxygenase P 450 is fused to a mammalian-like diflavin NADPH-cytochrome P 450 reductase in a single polypeptide.  The enzyme is sol. (unlike mammalian P 450 redox systems) and its fusion arrangement affords it the highest catalytic activity of any P 450 monooxygenase.  This review discusses the fundamental properties of P 450 BM3 and how progress with this model P 450 has affected the comprehension of P 450 systems in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT9WMdY1BB8bVg90H21EOLACvtfcHk0lg2g4VpioTPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktlSmsLw%253D&md5=f56df51e02f516fd26407169cbe8b3e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2802%2902086-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252802%252902086-8%26sid%3Dliteratum%253Aachs%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DLeys%26aufirst%3DD.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DOst%26aufirst%3DT.%2BW.%2BB.%26aulast%3DDaff%26aufirst%3DS.%26aulast%3DMiles%26aufirst%3DC.%2BS.%26aulast%3DChapman%26aufirst%3DS.%2BK.%26aulast%3DLysek%26aufirst%3DD.%2BA.%26aulast%3DMoser%26aufirst%3DC.%2BC.%26aulast%3DPage%26aufirst%3DC.%2BC.%26aulast%3DDutton%26aufirst%3DP.%2BL.%26atitle%3DP450%2520BM3%253A%2520the%2520very%2520model%2520of%2520a%2520modern%2520flavocytochrome%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2002%26volume%3D27%26spage%3D250%26epage%3D257%26doi%3D10.1016%2FS0968-0004%2802%2902086-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Duffell, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matak-Vinković, D.</span><span> </span><span class="NLM_article-title">Nanoelectrospray ionization mass spectrometric study of Mycobacterium tuberculosis CYP121-ligand interactions</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">5707</span><span class="NLM_x">–</span> <span class="NLM_lpage">5714</span><span class="refDoi"> DOI: 10.1021/ac400236z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac400236z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2013&pages=5707-5714&author=K.+M.+Duffellauthor=S.+A.+Hudsonauthor=K.+J.+McLeanauthor=A.+W.+Munroauthor=C.+Abellauthor=D.+Matak-Vinkovi%C4%87&title=Nanoelectrospray+ionization+mass+spectrometric+study+of+Mycobacterium+tuberculosis+CYP121-ligand+interactions&doi=10.1021%2Fac400236z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fac400236z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac400236z%26sid%3Dliteratum%253Aachs%26aulast%3DDuffell%26aufirst%3DK.%2BM.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DMatak-Vinkovi%25C4%2587%26aufirst%3DD.%26atitle%3DNanoelectrospray%2520ionization%2520mass%2520spectrometric%2520study%2520of%2520Mycobacterium%2520tuberculosis%2520CYP121-ligand%2520interactions%26jtitle%3DAnal.%2520Chem.%26date%3D2013%26volume%3D85%26spage%3D5707%26epage%3D5714%26doi%3D10.1021%2Fac400236z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Driscoll, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pikuleva, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafite, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigby, S. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Structural and biochemical characterization of Mycobacterium tuberculosis CYP142: evidence for multiple cholesterol 27-hydroxylase activities in a human pathogen</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">38270</span><span class="NLM_x">–</span> <span class="NLM_lpage">38282</span><span class="refDoi"> DOI: 10.1074/jbc.M110.164293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1074%2Fjbc.M110.164293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=20889498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOku73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=38270-38282&author=M.+D.+Driscollauthor=K.+J.+McLeanauthor=C.+Levyauthor=N.+Mastauthor=I.+A.+Pikulevaauthor=P.+Lafiteauthor=S.+E.+J.+Rigbyauthor=D.+Leysauthor=A.+W.+Munro&title=Structural+and+biochemical+characterization+of+Mycobacterium+tuberculosis+CYP142%3A+evidence+for+multiple+cholesterol+27-hydroxylase+activities+in+a+human+pathogen&doi=10.1074%2Fjbc.M110.164293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Biochemical Characterization of Mycobacterium tuberculosis CYP142: Evidence for Multiple Cholesterol 27-Hydroxylase Activities in a Human Pathogen</span></div><div class="casAuthors">Driscoll, Max D.; McLean, Kirsty J.; Levy, Colin; Mast, Natalia; Pikuleva, Irina A.; Lafite, Pierre; Rigby, Stephen E. J.; Leys, David; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">38270-38282</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The Mycobacterium tuberculosis cytochrome P 450 enzyme CYP142 is encoded in a large gene cluster involved in metab. of host cholesterol.  CYP142 was expressed and purified as a sol., low spin P 450 hemoprotein.  CYP142 binds tightly to cholesterol and its oxidized deriv. cholest-4-en-3-one, with extensive shift of the heme iron to the high spin state.  High affinity for azole antibiotics was demonstrated, highlighting their therapeutic potential.  CYP142 catalyzes either 27-hydroxylation of cholesterol/cholest-4-en-3-one or generates 5-cholestenoic acid/cholest-4-en-3-one-27-oic acid from these substrates by successive sterol oxidns., with the catalytic outcome dependent on the redox partner system used.  The CYP142 crystal structure was solved to 1.6 Å, revealing a similar active site organization to the cholesterol-metabolizing M. tuberculosis CYP125, but having a near-identical organization of distal pocket residues to the branched fatty acid oxidizing M. tuberculosis CYP124.  The cholesterol oxidizing activity of CYP142 provides an explanation for previous findings that ΔCYP125 strains of Mycobacterium bovis and M. bovis BCG cannot grow on cholesterol, because these strains have a defective CYP142 gene.  CYP142 is revealed as a cholesterol 27-oxidase with likely roles in host response modulation and cholesterol metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsGGdzyIjIULVg90H21EOLACvtfcHk0lg2g4VpioTPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOku73O&md5=7c64404f457b85b229042e39c73347ad</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.164293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.164293%26sid%3Dliteratum%253Aachs%26aulast%3DDriscoll%26aufirst%3DM.%2BD.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DLevy%26aufirst%3DC.%26aulast%3DMast%26aufirst%3DN.%26aulast%3DPikuleva%26aufirst%3DI.%2BA.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DRigby%26aufirst%3DS.%2BE.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DStructural%2520and%2520biochemical%2520characterization%2520of%2520Mycobacterium%2520tuberculosis%2520CYP142%253A%2520evidence%2520for%2520multiple%2520cholesterol%252027-hydroxylase%2520activities%2520in%2520a%2520human%2520pathogen%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D38270%26epage%3D38282%26doi%3D10.1074%2Fjbc.M110.164293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Bellamine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangla, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nes, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterman, M. R.</span><span> </span><span class="NLM_article-title">Characterization and catalytic properties of the sterol 14alpha-demethylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">8937</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1073/pnas.96.16.8937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1073%2Fpnas.96.16.8937" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=8937-8942&author=A.+Bellamineauthor=A.+T.+Manglaauthor=W.+D.+Nesauthor=M.+R.+Waterman&title=Characterization+and+catalytic+properties+of+the+sterol+14alpha-demethylase+from+Mycobacterium+tuberculosis&doi=10.1073%2Fpnas.96.16.8937"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.16.8937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.16.8937%26sid%3Dliteratum%253Aachs%26aulast%3DBellamine%26aufirst%3DA.%26aulast%3DMangla%26aufirst%3DA.%2BT.%26aulast%3DNes%26aufirst%3DW.%2BD.%26aulast%3DWaterman%26aufirst%3DM.%2BR.%26atitle%3DCharacterization%2520and%2520catalytic%2520properties%2520of%2520the%2520sterol%252014alpha-demethylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D8937%26epage%3D8942%26doi%3D10.1073%2Fpnas.96.16.8937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warman, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seward, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girvan, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheesman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterman, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span> </span><span class="NLM_article-title">Biophysical characterization of the sterol demethylase P450 from Mycobacterium tuberculosis, its cognate ferredoxin, and their interactions</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">8427</span><span class="NLM_x">–</span> <span class="NLM_lpage">8443</span><span class="refDoi"> DOI: 10.1021/bi0601609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0601609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsFOqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=8427-8443&author=K.+J.+McLeanauthor=A.+J.+Warmanauthor=H.+E.+Sewardauthor=K.+R.+Marshallauthor=H.+M.+Girvanauthor=M.+R.+Cheesmanauthor=M.+R.+Watermanauthor=A.+W.+Munro&title=Biophysical+characterization+of+the+sterol+demethylase+P450+from+Mycobacterium+tuberculosis%2C+its+cognate+ferredoxin%2C+and+their+interactions&doi=10.1021%2Fbi0601609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical Characterization of the Sterol Demethylase P450 from Mycobacterium tuberculosis, Its Cognate Ferredoxin, and Their Interactions</span></div><div class="casAuthors">McLean, Kirsty J.; Warman, Ashley J.; Seward, Harriet E.; Marshall, Ker R.; Girvan, Hazel M.; Cheesman, Myles R.; Waterman, Michael R.; Munro, Andrew W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8427-8443</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mycobacterium tuberculosis encodes a P 450 of the sterol demethylase family (CYP51) chromosomally located adjacent to a ferredoxin (Fdx).  CYP51 and Fdx were purified to homogeneity and characterized.  Spectroscopic analyses were consistent with cysteinate- and aqua-ligated heme iron in CYP51.  An ε419 of 134 mM-1 cm-1 was detd. for oxidized CYP51.  Anal. of interactions of 1-, 2-, and 4-phenylimidazoles with CYP51 showed that the 1- and 4-forms were heme iron-coordinating inhibitors, while 2-phenylimidazole induced a substrate-like optical shift.  The 2-phenyimidazole-bound CYP51 demonstrated unusual decreases in high-spin heme iron content at elevated temps. and an almost complete absence of high-spin heme iron by low-temp. EPR.  These data suggest thermally induced alterations in CYP51 active site structure and/or binding modes for the small ligand.  Redn. of CYP51 in the presence of carbon monoxide leads to formation of an Fe(II)-CO complex with a Soret absorption max. at 448.5 nm, which collapses (at 0.246 min-1 at pH 7.0) forming a species with a Soret max. at 421.5 nm (the inactive P420 form).  The rate of P420 formation is accelerated at lower pH, consistent with protonation of the cysteinate (Cys 394) to a thiol underlying the P 450-P420 transition.  The P 450 form is stabilized by estriol, which induces a type I spectral shift on binding CYP51 (Kd = 21.7 μM).  Nonstandard spectral changes occur on CYP51 redn. (using either dithionite or natural redox partners), including a blue-shifted Soret band and development of a strong feature at ∼558.5 nm, suggestive of cysteine thiol ligation.  Thus, ligand-free ferrous CYP51 is prone to thiolate ligand protonation even in the absence of carbon monoxide.  Anal. of reoxidized CYP51 demonstrates that the enzyme re-forms P 450, indicating that Cys 394 thiol is readily deprotonated to thiolate in the ferric form.  Spectroscopic anal. of Fdx by EPR (resonance at g = 2.03) and magnetic CD (intensity for oxidized and reduced forms and signal intensity dependence on field strength and temp.) demonstrated that Fdx binds a [3Fe-4S] iron-sulfur cluster.  Potentiometric studies show that the midpoint potential for ligand-free CYP51 is -375 mV, increasing to -225 mV in the estriol-bound form.  The Fdx potential is -31 mV.  Fdx forms a productive electron transfer complex with CYP51 and reduces it at a rate of 3.0 min-1 in the ligand-free form and 4.3 min-1 in the estriol-bound form, despite a thermodn. barrier.  Steady-state anal. of a M. tuberculosis class I redox system comprising flavoprotein reductase A (FprA), Fdx, and estriol-bound CYP51 indicates heme iron redn. as a rate-limiting step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNrTneY87rx7Vg90H21EOLACvtfcHk0liujENWdeLodQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsFOqs74%253D&md5=cdf20fe98cb285065fce7b5777108d14</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi0601609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0601609%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DWarman%26aufirst%3DA.%2BJ.%26aulast%3DSeward%26aufirst%3DH.%2BE.%26aulast%3DMarshall%26aufirst%3DK.%2BR.%26aulast%3DGirvan%26aufirst%3DH.%2BM.%26aulast%3DCheesman%26aufirst%3DM.%2BR.%26aulast%3DWaterman%26aufirst%3DM.%2BR.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26atitle%3DBiophysical%2520characterization%2520of%2520the%2520sterol%2520demethylase%2520P450%2520from%2520Mycobacterium%2520tuberculosis%252C%2520its%2520cognate%2520ferredoxin%252C%2520and%2520their%2520interactions%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D8427%26epage%3D8443%26doi%3D10.1021%2Fbi0601609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Martin, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blachly, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, S. M.</span><span> </span><span class="NLM_article-title">Exploring the influence of the protein environment on metal-binding pharmacophores</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7126</span><span class="NLM_x">–</span> <span class="NLM_lpage">7135</span><span class="refDoi"> DOI: 10.1021/jm500984b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500984b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7126-7135&author=D.+P.+Martinauthor=P.+G.+Blachlyauthor=J.+A.+McCammonauthor=S.+M.+Cohen&title=Exploring+the+influence+of+the+protein+environment+on+metal-binding+pharmacophores&doi=10.1021%2Fjm500984b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm500984b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500984b%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BP.%26aulast%3DBlachly%26aufirst%3DP.%2BG.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DExploring%2520the%2520influence%2520of%2520the%2520protein%2520environment%2520on%2520metal-binding%2520pharmacophores%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7126%26epage%3D7135%26doi%3D10.1021%2Fjm500984b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Lead optimization and structure–activity relationships for reversible inhibitors</span>. In  <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2F9781118540398.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DLead%2520optimization%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520for%2520reversible%2520inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhbitors</span> <span class="citation_source-journal">Biochem. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span><span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhbitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0liujENWdeLodQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhbitors%26jtitle%3DBiochem.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Hu, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eftink, M. R.</span><span> </span><span class="NLM_article-title">Thermodynamic studies of the interaction of trp aporepressor with tryptophan analogs</span> <span class="citation_source-journal">Biophys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/0301-4622(93)E0073-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2F0301-4622%2893%29E0073-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=8018820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFWmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1994&pages=233-239&author=D.+D.+Huauthor=M.+R.+Eftink&title=Thermodynamic+studies+of+the+interaction+of+trp+aporepressor+with+tryptophan+analogs&doi=10.1016%2F0301-4622%2893%29E0073-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic studies of the interaction of trp aporepressor with tryptophan analogs</span></div><div class="casAuthors">Hu, Dana D.; Eftink, Maurice R.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-9</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    </div><div class="casAbstract">The assocn. of L-tryptophan and some of its analogs, including 3 conformationally restricted analogs, with trp aporepressor (apo trpR) was studied by isothermal titrn. microcalorimetry.  Contributions of the functional groups of a ligand to the free energy change, ΔG°', and enthalpy change, ΔH°', of the interaction were evaluated on a mol. basis.  Analogs without the α-amino group (i.e. desamino analogs) bind with a slightly higher affinity to the protein.  On the other hand, descarboxy analogs show weaker binding to the apo trpR.  In addn., there exists enthalpy-entropy compensation for the assocn. of the congener series of ligands with the protein.  The entropy change, ΔS°', appears to play a more important role in the binding of the conformationally restricted analogs than in the binding of L-tryptophan and the unlocked ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6OPuEqrHTdbVg90H21EOLACvtfcHk0liCH8nXdCms7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFWmsL4%253D&md5=4a6db76073796876d92fb77dcfc8dcae</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0301-4622%2893%29E0073-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0301-4622%252893%2529E0073-E%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DD.%2BD.%26aulast%3DEftink%26aufirst%3DM.%2BR.%26atitle%3DThermodynamic%2520studies%2520of%2520the%2520interaction%2520of%2520trp%2520aporepressor%2520with%2520tryptophan%2520analogs%26jtitle%3DBiophys.%2520Chem.%26date%3D1994%26volume%3D49%26spage%3D233%26epage%3D239%26doi%3D10.1016%2F0301-4622%2893%29E0073-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span>Inhibitor screening against human cytochrome P450s was outsourced to Cyprotex Discovery Ltd., Macclesfield, Cheshire, UK.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Inhibitor+screening+against+human+cytochrome+P450s+was+outsourced+to+Cyprotex+Discovery+Ltd.%2C+Macclesfield%2C+Cheshire%2C+UK."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Mirza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynisson, J.</span><span> </span><span class="NLM_article-title">Known drug space as a metric in exploring the boundaries of drug-like chemical space</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">5006</span><span class="NLM_x">–</span> <span class="NLM_lpage">5011</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2009.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2Fj.ejmech.2009.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=19782440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCksLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2009&pages=5006-5011&author=A.+Mirzaauthor=R.+Desaiauthor=J.+Reynisson&title=Known+drug+space+as+a+metric+in+exploring+the+boundaries+of+drug-like+chemical+space&doi=10.1016%2Fj.ejmech.2009.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Known drug space as a metric in exploring the boundaries of drug-like chemical space</span></div><div class="casAuthors">Mirza, Amin; Desai, Radha; Reynisson, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5006-5011</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Marketed drug compds. (or known drug space) were used as a metric to test the principles of eliminating parent structures of the nitrenium ion (aryl-amine/nitro compds.) as well as sulfur and halogen contg. mols. from screening compd. collections.  Mols. contg. such moieties and/or atoms have biol. and physiochem. properties, which possibly make them less attractive as leads in drug development.  Precursors to the nitrenium ion were relatively abundant in known drug space at 14%.  Thus, their simple elimination from drug-like chem. space is not advisable.  Interestingly, the mutagenic potential of the nitrenium ions is linked to their stability and quantum mech. calcns. can be used to est. it.  Furthermore, 24% of drugs investigated contained sulfur atoms and around 28% were halogenated.  As some sulfur contg. moieties were abundant while others were scarce, it was deduced that it would be more effective to eliminate specific mol. scaffolds rather than all sulfur contg. mols.  In conclusion, it has been shown that by statistically analyzing known drug space a better understanding of the boundaries of drug-like chem. space was established which can help medicinal chemists in finding rewarding regions of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodBhMTaj39JbVg90H21EOLACvtfcHk0liCH8nXdCms7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCksLnJ&md5=30408326c9c7521db094f43d8c285a5e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2009.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2009.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DDesai%26aufirst%3DR.%26aulast%3DReynisson%26aufirst%3DJ.%26atitle%3DKnown%2520drug%2520space%2520as%2520a%2520metric%2520in%2520exploring%2520the%2520boundaries%2520of%2520drug-like%2520chemical%2520space%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2009%26volume%3D44%26spage%3D5006%26epage%3D5011%26doi%3D10.1016%2Fj.ejmech.2009.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guengerich, F. P.</span><span> </span><span class="NLM_article-title">Cytochrome P450 activation of arylamines and heterocyclic amines</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1146/annurev.pharmtox.45.120403.100010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1146%2Fannurev.pharmtox.45.120403.100010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=15822170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=27-49&author=D.+Kimauthor=F.+P.+Guengerich&title=Cytochrome+P450+activation+of+arylamines+and+heterocyclic+amines&doi=10.1146%2Fannurev.pharmtox.45.120403.100010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 activation of arylamines and heterocyclic amines</span></div><div class="casAuthors">Kim, Donghak; Guengerich, F. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-49</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Arylamines and heterocyclic arylamines (HAAs) are of particular interest because of demonstrated carcinogenicity in animals and humans and the broad exposure to many of these compds.  The activation of these, and also some arylamine drugs, involves N-hydroxylation, usually by cytochrome P 450 (P 450).  P 450 1A2 plays a prominent role in these reactions.  However, P 450 1A1 and 1B1 and other P450s are also important in humans as well as exptl. animals.  Some arylamines (including drugs) are N-hydroxylated predominantly by P450s other than those in Family 1.  Other oxygenases can also have roles.  An important issue is extrapolation between species in predicting cancer risks, as shown by the low rates of HAA activation by rat P 450 1A2 and low levels of P 450 1A2 expression in some nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6INgRLGF4_7Vg90H21EOLACvtfcHk0liCH8nXdCms7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrg%253D&md5=1fc485c6e77e6244b02fd9421f3fa2ed</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.45.120403.100010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.45.120403.100010%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DCytochrome%2520P450%2520activation%2520of%2520arylamines%2520and%2520heterocyclic%2520amines%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D45%26spage%3D27%26epage%3D49%26doi%3D10.1146%2Fannurev.pharmtox.45.120403.100010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvet, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieira, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunatilleke, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">R-configuration of 4-aminopyridyl-based inhibitors of CYP51 confers superior efficacy against Trypanosoma cruzi</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">434</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1021/ml500010m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500010m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=434-439&author=J.+Y.+Choiauthor=C.+M.+Calvetauthor=D.+F.+Vieiraauthor=S.+S.+Gunatillekeauthor=M.+D.+Cameronauthor=J.+H.+McKerrowauthor=L.+M.+Podustauthor=W.+R.+Roush&title=R-configuration+of+4-aminopyridyl-based+inhibitors+of+CYP51+confers+superior+efficacy+against+Trypanosoma+cruzi&doi=10.1021%2Fml500010m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fml500010m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500010m%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DCalvet%26aufirst%3DC.%2BM.%26aulast%3DVieira%26aufirst%3DD.%2BF.%26aulast%3DGunatilleke%26aufirst%3DS.%2BS.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DR-configuration%2520of%25204-aminopyridyl-based%2520inhibitors%2520of%2520CYP51%2520confers%2520superior%2520efficacy%2520against%2520Trypanosoma%2520cruzi%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D434%26epage%3D439%26doi%3D10.1021%2Fml500010m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvet, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunatilleke, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podust, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">Rational development of 4-aminopyridyl-based inhibitors targeting Trypanosoma cruzi CYP51 as anti-chagas agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7651</span><span class="NLM_x">–</span> <span class="NLM_lpage">7668</span><span class="refDoi"> DOI: 10.1021/jm401067s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401067s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7651-7668&author=J.+Y.+Choiauthor=C.+M.+Calvetauthor=S.+S.+Gunatillekeauthor=C.+Ruizauthor=M.+D.+Cameronauthor=J.+H.+McKerrowauthor=L.+M.+Podustauthor=W.+R.+Roush&title=Rational+development+of+4-aminopyridyl-based+inhibitors+targeting+Trypanosoma+cruzi+CYP51+as+anti-chagas+agents&doi=10.1021%2Fjm401067s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm401067s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401067s%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DCalvet%26aufirst%3DC.%2BM.%26aulast%3DGunatilleke%26aufirst%3DS.%2BS.%26aulast%3DRuiz%26aufirst%3DC.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DPodust%26aufirst%3DL.%2BM.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DRational%2520development%2520of%25204-aminopyridyl-based%2520inhibitors%2520targeting%2520Trypanosoma%2520cruzi%2520CYP51%2520as%2520anti-chagas%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7651%26epage%3D7668%26doi%3D10.1021%2Fjm401067s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Brennan, P. J.</span><span> </span><span class="NLM_article-title">Structure, function, and biogenesis of the cell wall of Mycobacterium tuberculosis</span> <span class="citation_source-journal">Tuberculosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span><span class="refDoi"> DOI: 10.1016/S1472-9792(02)00089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1016%2FS1472-9792%2802%2900089-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=12758196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A280%3ADC%252BD3s3js1Kgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2003&pages=91-97&author=P.+J.+Brennan&title=Structure%2C+function%2C+and+biogenesis+of+the+cell+wall+of+Mycobacterium+tuberculosis&doi=10.1016%2FS1472-9792%2802%2900089-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function, and biogenesis of the cell wall of Mycobacterium tuberculosis</span></div><div class="casAuthors">Brennan P J</div><div class="citationInfo"><span class="NLM_cas:title">Tuberculosis (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">91-7</span>
        ISSN:<span class="NLM_cas:issn">1472-9792</span>.
    </div><div class="casAbstract">Much of the early structural definition of the cell wall of Mycobacterium spp. was initiated in the 1960s and 1970s.  There was a long period of inactivity, but more recent developments in NMR and mass spectral analysis and definition of the M. tuberculosis genome have resulted in a thorough understanding, not only of the structure of the mycobacterial cell wall and its lipids but also the basic genetics and biosynthesis.  Our understanding nowadays of cell-wall architecture amounts to a massive "core" comprised of peptidoglycan covalently attached via a linker unit (L-Rha-D-GlcNAc-P) to a linear galactofuran, in turn attached to several strands of a highly branched arabinofuran, in turn attached to mycolic acids.  The mycolic acids are oriented perpendicular to the plane of the membrane and provide a truly special lipid barrier responsible for many of the physiological and disease-inducing aspects of M. tuberculosis.  Intercalated within this lipid environment are the lipids that have intrigued researchers for over five decades: the phthiocerol dimycocerosate, cord factor/dimycolyltrehalose, the sulfolipids, the phosphatidylinositol mannosides, etc.  Knowledge of their roles in "signaling" events, in pathogenesis, and in the immune response is now emerging, sometimes piecemeal and sometimes in an organized fashion.  Some of the more intriguing observations are those demonstrating that mycolic acids are recognized by CD1-restricted T-cells, that antigen 85, one of the most powerful protective antigens of M. tuberculosis, is a mycolyltransferase, and that lipoarabinomannan (LAM), when "capped" with short mannose oligosaccharides, is involved in phagocytosis of M. tuberculosis.  Definition of the genome of M. tuberculosis has greatly aided efforts to define the biosynthetic pathways for all of these exotic molecules: the mycolic acids, the mycocerosates, phthiocerol, LAM, and the polyprenyl phosphates.  For example, we know that synthesis of the entire core is initiated on a decaprenyl-P with synthesis of the linker unit, and then there is concomitant extension of the galactan and arabinan chains while this intermediate is transported through the cytoplasmic membrane.  The final steps in these events, the attachment of mycolic acids and ligation to peptidoglycan, await definition and will prove to be excellent targets for a new generation of anti-tuberculosis drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmjBRWYNdwmJPlX7E30qdzfW6udTcc2ebWJ7VPwgeBWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3js1Kgsw%253D%253D&md5=7c8428a77e3aef5bce2db8c9e54ba0a9</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2FS1472-9792%2802%2900089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1472-9792%252802%252900089-6%26sid%3Dliteratum%253Aachs%26aulast%3DBrennan%26aufirst%3DP.%2BJ.%26atitle%3DStructure%252C%2520function%252C%2520and%2520biogenesis%2520of%2520the%2520cell%2520wall%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DTuberculosis%26date%3D2003%26volume%3D83%26spage%3D91%26epage%3D97%26doi%3D10.1016%2FS1472-9792%2802%2900089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">De Rossi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aínsa, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riccardi, G.</span><span> </span><span class="NLM_article-title">Role of mycobacterial efflux transporters in drug resistance: An unresolved question</span> <span class="citation_source-journal">FEMS Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">36</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1111/j.1574-6976.2005.00002.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1111%2Fj.1574-6976.2005.00002.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=16438679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=36-52&author=E.+De+Rossiauthor=J.+A.+A%C3%ADnsaauthor=G.+Riccardi&title=Role+of+mycobacterial+efflux+transporters+in+drug+resistance%3A+An+unresolved+question&doi=10.1111%2Fj.1574-6976.2005.00002.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Role of mycobacterial efflux transporters in drug resistance: an unresolved question</span></div><div class="casAuthors">De Rossi, Edda; Ainsa, Jose A.; Riccardi, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-52</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Two mechanisms are thought to be involved in the natural drug resistance of mycobacteria: the mycobacterial cell wall permeability barrier and active multidrug efflux pumps.  Genes encoding drug efflux transporters have been isolated from several mycobacterial species.  These proteins transport tetracycline, fluoroquinolones, aminoglycosides and other compds.  Recent reports have suggested that efflux pumps may also be involved in transporting isoniazid, one of the main drugs used to treat tuberculosis.  This review highlights recent advances in our understanding of efflux-mediated drug resistance in mycobacteria, including the distribution of efflux systems in these organisms, their substrate profiles and their contribution to drug resistance.  The balance between the drug transport into the cell and drug efflux is not yet clearly understood, and further studies are required in mycobacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwjdJiKjbO7Vg90H21EOLACvtfcHk0lhOWfn1lnNtRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D&md5=41af6cedab317d501c489400a6979920</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2005.00002.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2005.00002.x%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BRossi%26aufirst%3DE.%26aulast%3DA%25C3%25ADnsa%26aufirst%3DJ.%2BA.%26aulast%3DRiccardi%26aufirst%3DG.%26atitle%3DRole%2520of%2520mycobacterial%2520efflux%2520transporters%2520in%2520drug%2520resistance%253A%2520An%2520unresolved%2520question%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D30%26spage%3D36%26epage%3D52%26doi%3D10.1111%2Fj.1574-6976.2005.00002.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Martin, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blachly, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marts, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, C. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tierney, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, S. M.</span><span> </span><span class="NLM_article-title">Unconventional” coordination chemistry by metal chelating fragments in a metalloprotein active site</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">5400</span><span class="NLM_x">–</span> <span class="NLM_lpage">5406</span><span class="refDoi"> DOI: 10.1021/ja500616m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja500616m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=5400-5406&author=D.+P.+Martinauthor=P.+G.+Blachlyauthor=A.+R.+Martsauthor=T.+M.+Woodruffauthor=C.+A.+F.+de+Oliveiraauthor=J.+A.+McCammonauthor=D.+L.+Tierneyauthor=S.+M.+Cohen&title=Unconventional%E2%80%9D+coordination+chemistry+by+metal+chelating+fragments+in+a+metalloprotein+active+site&doi=10.1021%2Fja500616m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fja500616m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja500616m%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BP.%26aulast%3DBlachly%26aufirst%3DP.%2BG.%26aulast%3DMarts%26aufirst%3DA.%2BR.%26aulast%3DWoodruff%26aufirst%3DT.%2BM.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%2BA.%2BF.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DTierney%26aufirst%3DD.%2BL.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DUnconventional%25E2%2580%259D%2520coordination%2520chemistry%2520by%2520metal%2520chelating%2520fragments%2520in%2520a%2520metalloprotein%2520active%2520site%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D5400%26epage%3D5406%26doi%3D10.1021%2Fja500616m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashalidis, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Biofragments: An approach towards predicting protein function using biologically related fragments and its application to mycobacterium tuberculosis CYP126</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">555</span><span class="refDoi"> DOI: 10.1002/cbic.201300697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=10.1002%2Fcbic.201300697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=24677424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFaiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=549-555&author=S.+A.+Hudsonauthor=E.+H.+Mashalidisauthor=A.+Benderauthor=K.+J.+McLeanauthor=A.+W.+Munroauthor=C.+Abell&title=Biofragments%3A+An+approach+towards+predicting+protein+function+using+biologically+related+fragments+and+its+application+to+mycobacterium+tuberculosis+CYP126&doi=10.1002%2Fcbic.201300697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biofragments: An Approach towards Predicting Protein Function Using Biologically Related Fragments and its Application to Mycobacterium tuberculosis CYP126</span></div><div class="casAuthors">Hudson, Sean A.; Mashalidis, Ellene H.; Bender, Andreas; McLean, Kirsty J.; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We present a novel fragment-based approach that tackles some of the challenges for chem. biol. of predicting protein function.  The general approach, which we have termed biofragments, comprises two key stages.  First, a biol. relevant fragment library (biofragment library) can be designed and constructed from known sets of substrate-like ligands for a protein class of interest.  Second, the library can be screened for binding to a novel putative ligand-binding protein from the same or similar class, and the characterization of hits provides insight into the basis of ligand recognition, selectivity, and function at the substrate level.  As a proof-of-concept, we applied the biofragments approach to the functionally uncharacterized Mycobacterium tuberculosis (Mtb) cytochrome P 450 isoform, CYP126.  This led to the development of a tailored CYP biofragment library with notable 3D characteristics and a significantly higher screening hit rate (14 %) than std. drug-like fragment libraries screened previously against Mtb CYP121 and 125 (4 % and 1 %, resp.).  Biofragment hits were identified that make both substrate-like type-I and inhibitor-like type-II interactions with CYP126.  A chem.-fingerprint-based substrate model was built from the hits and used to search a virtual TB metabolome, which led to the discovery that CYP126 has a strong preference for the recognition of aroms. and substrate-like type-I binding of chlorophenol moieties within the active site near the heme.  Future catalytic analyses will be focused on assessing CYP126 for potential substrate oxidative dehalogenation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3kRhqwkn5LVg90H21EOLACvtfcHk0ljrp6xatpMXFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFaiur0%253D&md5=4307140caec78e91c17088e0354fb2fe</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300697%26sid%3Dliteratum%253Aachs%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DBiofragments%253A%2520An%2520approach%2520towards%2520predicting%2520protein%2520function%2520using%2520biologically%2520related%2520fragments%2520and%2520its%2520application%2520to%2520mycobacterium%2520tuberculosis%2520CYP126%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D549%26epage%3D555%26doi%3D10.1002%2Fcbic.201300697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G47" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G47','PDB','4G47'); return false;">PDB: 4G47</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTL','PDB','4KTL'); return false;">PDB: 4KTL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTJ','PDB','4KTJ'); return false;">PDB: 4KTJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTF','PDB','4KTF'); return false;">PDB: 4KTF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ','PDB','5IBJ'); return false;">PDB: 5IBJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT','PDB','5EDT'); return false;">PDB: 5EDT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G44" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G44','PDB','4G44'); return false;">PDB: 4G44</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G45','PDB','4G45'); return false;">PDB: 4G45</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE','PDB','5IBE'); return false;">PDB: 5IBE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF','PDB','5IBF'); return false;">PDB: 5IBF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI','PDB','5IBI'); return false;">PDB: 5IBI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH','PDB','5IBH'); return false;">PDB: 5IBH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBD','PDB','5IBD'); return false;">PDB: 5IBD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBG','PDB','5IBG'); return false;">PDB: 5IBG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G1X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G1X','PDB','4G1X'); return false;">PDB: 4G1X</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i120"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00007">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42470"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00007">10.1021/acs.jmedchem.6b00007</a>. Additional data related to this publication is available at the University of Cambridge data repository: <a class="extLink" href="https://www.repository.cam.ac.uk/handle/1810/254048">https://www.repository.cam.ac.uk/handle/1810/254048</a><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">X-ray crystallographic data tables, ligand density maps, human P450 inhibition data, and additional native mass spectrometry data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_001.pdf">jm6b00007_si_001.pdf (2.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00007/suppl_file/jm6b00007_si_002.csv">jm6b00007_si_002.csv (7.64 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>6</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBJ">5IBJ</a>; <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDT">5EDT</a>, <b>19a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBF">5IBF</a>; <b>24a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBD">5IBD</a>; <b>25a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBE">5IBE</a>; <b>25b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBG">5IBG</a>; <b>26a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBI">5IBI</a>; and <b>26h</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IBH">5IBH</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00007%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00007" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679915fc3e4c3dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
