
<html lang="en"     class="pb-page"  data-request-id="9b84b81d-cbd4-445e-891a-646af57e7983"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01917;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2016.59.issue-8;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase" /></meta><meta name="dc.Creator" content="John B.  Jordan" /></meta><meta name="dc.Creator" content="Douglas A.  Whittington" /></meta><meta name="dc.Creator" content="Michael D.  Bartberger" /></meta><meta name="dc.Creator" content="E. Allen  Sickmier" /></meta><meta name="dc.Creator" content="Kui  Chen" /></meta><meta name="dc.Creator" content="Yuan  Cheng" /></meta><meta name="dc.Creator" content="Ted  Judd" /></meta><meta name="dc.Description" content="Fragment-based drug discovery (FBDD) has become a widely used tool in small-molecule drug discovery efforts. One of the most commonly used biophysical methods in detecting weak binding of fragments..." /></meta><meta name="Description" content="Fragment-based drug discovery (FBDD) has become a widely used tool in small-molecule drug discovery efforts. One of the most commonly used biophysical methods in detecting weak binding of fragments..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 6, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01917" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01917" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01917" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01917" /></link>
        
    
    

<title>Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01917" /></meta><meta property="og:title" content="Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0014.jpeg" /></meta><meta property="og:description" content="Fragment-based drug discovery (FBDD) has become a widely used tool in small-molecule drug discovery efforts. One of the most commonly used biophysical methods in detecting weak binding of fragments is nuclear magnetic resonance (NMR) spectroscopy. In particular, FBDD performed with 19F NMR-based methods has been shown to provide several advantages over 1H NMR using traditional magnetization-transfer and/or two-dimensional methods. Here, we demonstrate the utility and power of 19F-based fragment screening by detailing the identification of a second-site fragment through 19F NMR screening that binds to a specific pocket of the aspartic acid protease, β-secretase (BACE-1). The identification of this second-site fragment allowed the undertaking of a fragment-linking approach, which ultimately yielded a molecule exhibiting a more than 360-fold increase in potency while maintaining reasonable ligand efficiency and gaining much improved selectivity over cathepsin-D (CatD). X-ray crystallographic studies of the molecules demonstrated that the linked fragments exhibited binding modes consistent with those predicted from the targeted screening approach, through-space NMR data, and molecular modeling." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01917"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01917">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01917&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01917&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01917&amp;href=/doi/10.1021/acs.jmedchem.5b01917" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 3732-3749</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01884" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01925" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.7b00411"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Linking Approach Using <sup>19</sup>F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+B.++Jordan">John B. Jordan</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+A.++Whittington">Douglas A. Whittington</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+D.++Bartberger">Michael D. Bartberger</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=E.+Allen++Sickmier">E. Allen Sickmier</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kui++Chen">Kui Chen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Cheng">Yuan Cheng</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ted++Judd">Ted Judd</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand Oaks, California 91320, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6a08180b0e440005180e0b042a0b070d0f0444090507"><span class="__cf_email__" data-cfemail="23415142470d494c5147424d63424e44464d0d404c4e">[email protected]</span></a>. Tel.: 805-313-5647. Fax: 805-480-1337.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01917&amp;href=/doi/10.1021%2Facs.jmedchem.5b01917" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 3732–3749</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 15, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 December 2015</li><li><span class="item_label"><b>Published</b> online</span>6 April 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 April 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01917" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01917</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3732%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BB.%2BJordan%252C%2BDouglas%2BA.%2BWhittington%252C%2BMichael%2BD.%2BBartberger%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D8%26contentID%3Dacs.jmedchem.5b01917%26title%3DFragment-Linking%2BApproach%2BUsing%2B19F%2BNMR%2BSpectroscopy%2BTo%2BObtain%2BHighly%2BPotent%2Band%2BSelective%2BInhibitors%2Bof%2B%25CE%25B2-Secretase%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3749%26publicationDate%3DApril%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01917"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2291</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">36</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01917" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;B. Jordan&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;A. Whittington&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;D. Bartberger&quot;},{&quot;first_name&quot;:&quot;E.&quot;,&quot;last_name&quot;:&quot;Allen Sickmier&quot;},{&quot;first_name&quot;:&quot;Kui&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Ted&quot;,&quot;last_name&quot;:&quot;Judd&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;3732-3749&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01917&quot;},&quot;abstract&quot;:&quot;Fragment-based drug discovery (FBDD) has become a widely used tool in small-molecule drug discovery efforts. One of the most commonly used biophysical methods in detecting weak binding of fragments is nuclear magnetic resonance (NMR) spectroscopy. In particular, FBDD performed with 19F NMR-based methods has been shown to provide several advantages over 1H NMR using traditional magnetization-transfer and/or two-dimensional methods. Here, we demonstrate the utility and power of 19F-based fragment screening by detailing the identification of a second-site fragment through 19F NMR screening that binds to a specific pocket of the aspartic acid protease, β-secretase (BACE-1). The identification of this second-site fragment allowed the undertaking of a fragment-linking approach, which ultimately yielded a molecule exhibiting a more than 360-fold increase in potency while maintaining reasonable ligand efficiency and gaining much improved selectivity over cathepsin-D (CatD). X-ray crystallographic studies of the molec&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01917&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01917" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01917&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01917" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01917&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01917" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01917&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01917&amp;href=/doi/10.1021/acs.jmedchem.5b01917" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01917" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01917" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01917%26sid%3Dliteratum%253Aachs%26pmid%3D26978477%26genre%3Darticle%26aulast%3DJordan%26date%3D2016%26atitle%3DFragment-Linking%2BApproach%2BUsing%2B19F%2BNMR%2BSpectroscopy%2BTo%2BObtain%2BHighly%2BPotent%2Band%2BSelective%2BInhibitors%2Bof%2B%25CE%25B2-Secretase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D8%26spage%3D3732%26epage%3D3749%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/jmcmar.2016.59.issue-8/20160428/jmcmar.2016.59.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Fragment-based drug discovery (FBDD) has become a widely used tool in small-molecule drug discovery efforts. One of the most commonly used biophysical methods in detecting weak binding of fragments is nuclear magnetic resonance (NMR) spectroscopy. In particular, FBDD performed with <sup>19</sup>F NMR-based methods has been shown to provide several advantages over <sup>1</sup>H NMR using traditional magnetization-transfer and/or two-dimensional methods. Here, we demonstrate the utility and power of <sup>19</sup>F-based fragment screening by detailing the identification of a second-site fragment through <sup>19</sup>F NMR screening that binds to a specific pocket of the aspartic acid protease, β-secretase (BACE-1). The identification of this second-site fragment allowed the undertaking of a fragment-linking approach, which ultimately yielded a molecule exhibiting a more than 360-fold increase in potency while maintaining reasonable ligand efficiency and gaining much improved selectivity over cathepsin-D (CatD). X-ray crystallographic studies of the molecules demonstrated that the linked fragments exhibited binding modes consistent with those predicted from the targeted screening approach, through-space NMR data, and molecular modeling.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52441" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52441" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Fragment-based drug discovery (FBDD) has rapidly gained momentum in the discovery and development of small-molecule therapeutics. Over the past 10 years, FBDD has become a common workstream in the path to find novel chemical matter, often for challenging targets, and a number of molecules in various phases of development have been derived from an initial fragment screening hit.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> With the advent of FBDD came the necessity for highly sensitive biophysical methods aimed at detecting the weak binding of small molecules normally observed in fragment-based approaches. As such, various solution-phase NMR techniques and adaptations to surface plasmon resonance (SPR) methods (as well as other methods) have been utilized, both of which now comprise the majority of the biophysical toolbox used in high-throughput fragment screening.</div><div class="NLM_p">The use of NMR in screening for fragment binding has historically utilized two main approaches: protein-detected and ligand-detected experiments. In protein-detected experiments, pioneered by Fesik and co-workers, the target of interest is typically labeled with the NMR-active <sup>15</sup>N nucleus, and two-dimensional heteronuclear single-quantum coherence (HSQC) experiments are used to monitor the protein signals for changes upon ligand binding.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Conversely, typical ligand-detected approaches use unlabeled protein at low concentrations and rely on magnetization transfer pathways from either the protein itself (saturation transfer difference, or STD) or from bulk water (Water-Ligand Observe via Gradient SpectroscopY, or Water-LOGSY) to observe binding of the small molecule to the target of interest.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> More recently, we and others have discussed the use of <sup>19</sup>F NMR spectroscopy in FBDD as a powerful tool in detecting weak binding of small molecules, and even as a potentially general strategy for fragment screening by NMR.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The utility of using the fluorine nucleus as a highly sensitive detection method of weak binding is not a new development.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7">(4-7)</a> However, here the use of <sup>19</sup>F-based fragment screening is extended to so-called “second-site” approaches, in which a known ligand is used to block one region of a target’s pocket while another region is probed for small-molecule binding using <sup>19</sup>F NMR. In this example, this approach is applied to β-secretase, a potentially high-value target implicated in the onset and progression of Alzheimer’s disease (AD).</div><div class="NLM_p">BACE-1 is an aspartic protease that functions in the cleavage process of amyloid precursor protein (APP), an amyloidogenic pathway that ultimately yields shorter fragments of the Aβ peptide, including the pathogenic species Aβ40 and Aβ42, which are believed to play a primary role in the etiology of AD.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8-11)</a> The identification of potent ligands for BACE-1 has been met with some success by fragment-based approaches.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12-14)</a> A number of papers have detailed results of fragment screening against BACE-1 by various methods, and many of those fragments have been optimized through structure-based design to yield potent, and even <i>in vivo</i> efficacious, inhibitors of BACE-1.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16">(12-16)</a> In addition, recent publications have detailed extensive structure–activity relationship (SAR) efforts to obtain suitable selectivity of BACE-1 inhibitors against cathepsin D (CatD), a related aspartic protease residing in lysosomes.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> It has been observed that the acidic environment of lysosomes results in the accumulation of traditionally charged BACE-1 inhibitors,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> resulting in much higher local concentrations of the molecules and thereby presenting problems with off-target inhibition of other lysosomal enzymes, such as CatD. Off-target inhibition of CatD is of great potential concern, having been proposed to be involved in ocular as well as neurodegenerative side-effects.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> For this reason, a very high degree of selectivity against CatD and other lysosomal enzymes is necessary, and thus, CatD inhibition is a commonly used selectivity filter for small-molecule inhibitors of BACE-1.<a onclick="showRef(event, 'ref17 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref19">(17, 19)</a> There are some distinct structural differences in the binding pockets between BACE-1 and CatD, the most prominent being the nature and composition of the S3 pocket in BACE-1. Utilization of the S3 pocket by appropriate chemical matter has been shown to enhance CatD selectivity.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We postulated that further extension of a lead molecule past the S3 pocket and down into the S3<sub>subpocket</sub> would afford additional selectivity. Thus, one of our primary aims in this work was to use FBDD to find a second-site ligand to the S3<sub>subpocket</sub> of BACE-1 that could subsequently be linked to the core compound that would afford both an increase in potency and, importantly, a higher degree of CatD selectivity.</div><div class="NLM_p">One of the founding principles of FBDD, as originally described using SAR by NMR,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> is that leads can be rapidly derived from weakly binding fragments due to the fact that the binding energies of multiples fragments can be additive.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> Thus, linking a second-site ligand with an affinity of ∼1 mM with a first-site ligand could theoretically yield a potency gain of 3 orders of magnitude.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This, of course, is neglecting any effects on affinity due to entropic factors or from the linker design itself. The fragment-linking approach has been well-documented over the past decade and has proven to be a viable approach to small-molecule drug discovery.<a onclick="showRef(event, 'ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23">(20-23)</a> However, these approaches are not trivial, and many times, several iterations of linkers are required to provide optimal conformations of the ligand. It is often more common to find that the observed fragment linking additivity is less than the theoretical additivity than it is to find that they meet or exceed the theoretical gains in affinity.<a onclick="showRef(event, 'ref2 ref23'); return false;" href="javascript:void(0);" class="ref ref2 ref23">(2, 23)</a></div><div class="NLM_p last">In attempts to apply fragment-based methods to the design of potent and selective small-molecule inhibitors of BACE-1, a molecule derived from a core fragment and resulting from early stage SAR was used to serve as blocking compound for a large portion of the BACE-1 pocket, thereby allowing a “second-site” fragment screen to be conducted. Using this method of fragment linking, <sup>19</sup>F-NMR based fragment screening has been applied to identify chemical matter that binds specifically to the S3 and S3<sub>subpocket</sub> of BACE-1. Upon identification of fragments binding the “blocked” conformation of BACE-1, interligand NOE NMR experiments and molecular modeling led to the chemical linkage of the two compounds (core + fragment) and yielded compounds with both increased on-target potency (∼350×) as well as improved selectivity (∼2000×) against CatD.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10674" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10674" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The use of <sup>19</sup>F NMR in screening campaigns, particularly in fragment-based approaches, has been previously detailed.<a onclick="showRef(event, 'ref4 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref6 ref7">(4, 6, 7)</a> In FBDD, the use of <sup>19</sup>F NMR has many benefits; the major one being the utilization of the fluorine nucleus for a very sensitive detection tool for weak binding. Other attributes of using <sup>19</sup>F as a detection tool have been reviewed extensively by Dalvit and others.<a onclick="showRef(event, 'ref4 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref7">(4, 7)</a> In the realm of sensitivity, the <sup>19</sup>F nucleus is almost as sensitive as that of a proton (with a gyromagnetic ratio about 0.83 that of <sup>1</sup>H), and the lack of a protonated background in <sup>19</sup>F NMR enables the use of solvents, buffers, or detergents that would normally interfere with a typical <sup>1</sup>H-based NMR screening campaign, thereby increasing the effective sensitivity. This high sensitivity allows the detection of small quantities of compounds (∼20 μM) in very short experiment times (3–4 min). Also, The <sup>19</sup>F nucleus is characterized by a large chemical shift dispersion (∼200 ppm) and very narrow line widths (Δ<sub>w1/2</sub> ∼ 1–2 Hz) that allow screening to be conducted on large pools of compounds without the concern of signal overlap.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Standard <sup>1</sup>H-based NMR fragment screening experiments such as STD or LOGSY are performed in pools of five compounds or less.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a><sup>19</sup>F NMR, however, allows the screen to be conducted on pools of compounds containing up to 20 compounds-all possessing unique chemical shifts. This effectively negates the need for any specific pooling strategy when designing libraries.</div><div class="NLM_p">In approaching the second-site fragment screen for BACE-1, a suitable blocking molecule was needed to facilitate the identification of a true S3 pocket binder. Previously, Cheng et al. presented data demonstrating the elaboration of a 2-aminoquinoline fragment (<i>K</i><sub>D</sub> ∼ 900 μM) to a molecule with an affinity of 11 pM.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In a semiparallel effort to the traditional structure-based design efforts in this study, a second-site fragment screen was attempted using our in-house <sup>19</sup>F fragment library in an attempt to find additional chemical matter that would allow the linking of a core molecule to a fragment that was targeted to occupy the S3 binding site of BACE-1. Compound 58 from Cheng et al. (compound <b>1</b> in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) was used in the fragment screens as a blocking compound to occupy the majority of the binding site of BACE-1 (<i>K</i><sub>D</sub> ∼ 16 nM) while leaving the S3 and S3<sub>subpocket</sub> regions accessible (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Next, the <sup>19</sup>F-NMR fragment screen was performed with saturating concentrations (∼5 μM) of compound <b>1</b> in the experimental buffer.The NMR signals of the <sup>19</sup>F nucleus are extremely sensitive to changes in the environment surrounding the fluorine.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> When the free and protein-bound fractions of the fluorine-containing compound experience significantly different environments, broadening of the NMR signals occurs with a concomitant decrease in signal intensity. In all, the screen yielded seven fragments that exhibited binding to the target in the presence of the blocking compound. As depicted in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, the fluorine NMR signals of the ligands that bind to BACE-1 can experience significant line broadening, resulting in an obvious decrease in signal intensity upon binding to the larger protein. Following the screen, the binder with highest affinity (as determined by the effect on the NMR line shape and by SPR) was used in a competition binding assay to determine if the fragment indeed bound to the desired pocket. To do this, a more potent inhibitor (Compound <b>2</b>, <i>K</i><sub>D</sub> ∼ 600 pM, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) from the aminooxazoline xanthene series<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> that occupied all of the binding pockets in question (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B) was used as a competitor with the blocking compound <b>1</b> and the newly identified fragment (compound <b>3</b>). As evidenced in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, the fragment alone failed to bind the protein but clearly did so in the presence of the blocking compound. Upon the addition of the competitor compound (compound <b>2</b>), the <sup>19</sup>F NMR signal from the fragment became sharp again, suggesting that the competitor molecule replaced a substantial fraction of the fragment in the binding site, resulting in a higher concentration of free compound (fragment) in solution and a sharpening of the NMR signal. In addition, the fragment failed to exhibit binding to CatD both in the presence and absence of the blocking compound (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). These data were highly suggestive that the <sup>19</sup>F fragment was indeed binding to the S3 and/or S3<sub>subpocket</sub>. It should be noted that the displacement of the fragment was not complete in the competition assay; this was likely due to the poor solubility of the competitor molecule. However, in all cases tested, addition of the more potent inhibitor that fully occupied the S3 site resulted in significant sharpening of the <sup>19</sup>F signals, indicating the increased presence of unbound ligand.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0016.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) X-ray crystal structure of BACE-1 in complex with the blocking compound <b>1</b> (PDB ID 5I3 V) used in the study. Notice that the binding site of the protein is largely occupied while leaving S3 and S3<sub>subpocket (sp)</sub> accessible to fragments. (B) X-ray crystal structure of BACE-1 in complex with competitor compound <b>2</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W">5I3W</a>) that occupies the S3<sub>sp</sub>.(C) X-ray crystal structure of BACE-1 in complex with compound <b>5</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1">5IE1</a>) to which the <sup>19</sup>F fragment (compound <b>3</b>) was linked. Notice that the toluyl methyl group in compound <b>5</b> points directly toward the S3 pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0017.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <sup>19</sup>F NMR data of the best hit (compound <b>3</b>) from the fragment screen. (A) NMR spectra of the free ligand (black) and the fragment in the presence of BACE-1(red) and in the presence of CatD (green). No binding to either protein is observed. (B) NMR spectra of the free fragment (black) and the fragment in the presence of BACE-1 and CatD along with saturating concentrations of compound <b>1</b> (red and green, respectively). Binding of the fragment to BACE-1 is exhibited by broadening of the NMR signal, demonstrating that the fragment only binds in the presence of the blocking compound. The signal exhibits no change with CatD, indicating a lack of binding. (C) NMR spectra of the free fragment (black) and the fragment in the presence of BACE-1 and CatD, the blocking compound, and 2 μM of the competitor ligand (compound <b>2</b>, red and green, respectively). The sharpening of the NMR signal indicates that the newly added ligand competes out the binding fragment, thus suggesting that the fragment likely binds to the S3 pocket of BACE-1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1">Table 1<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0011.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Ligand efficiency calculated from SPR <i>K</i><sub>D</sub> value</p></div></div><div></div></div><div class="NLM_p">In order to determine the affinity of each fragment to BACE-1, surface plasmon resonance (SPR) experiments were used in the presence of saturating concentrations of the blocking compound (compound <b>1</b>, 5 μM). Three molecules exhibited binding to the blocked form of BACE-1 with affinities in the 100–300 μM range, with four more compounds exhibiting much weaker binding (<a class="ref internalNav" href="#notes-1" aria-label="Supplementary Figure 1">Supplementary Figure 1</a>). In particular, compound <b>3</b> stood out in its affinity to BACE-1 in the presence of the blocking compound. Furthermore, this compound exhibited undetectable binding to BACE-1 in the absence of the blocking compound (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, in agreement with the NMR data (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), and suggesting that the presence of the blocking compound helped to define the binding pocket for the fragment.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SPR sensorgrams of the lead <sup>19</sup>F fragment hit (compound <b>3</b>) binding to BACE-1 in the (A) absence of blocking compound (no binding observed) and (B) in the presence of saturating concentrations of the blocking compound (compound <b>1</b>, 5 μM). <i>K</i><sub>D</sub> values were obtained using steady-state analysis and fitting to a global <i>R</i><sub>max</sub> determined using a control compound. Sensorgrams shown are the same experiments run in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Taken together, these results were highly suggestive that compound <b>3</b> was indeed binding specifically to the S3 and S3<sub>subpocket</sub> of BACE-1. Based on the theory of fragment additivity, two fragments can be linked together to ultimately form a more potent compound. The extent of potency increases, however, is dictated by factors such as linker design, rigid body entropy, and even synergistic binding. A simplistic view to approximate the benefits of linking two fragments is the simple multiplicative effect of the two affinities (K<sub>A</sub>*K<sub>B</sub>).<a onclick="showRef(event, 'ref20 ref21 ref23'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref23">(20, 21, 23)</a>However, its affinities commonly fail to reach the optimal potency due to imperfect linker design or other entropic effects. Due to the high relative affinity of compound <b>3</b> compared to the other fragments (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>), this molecule was selected as the lead fragment, and further experiments were conducted to determine if a fragment-linking strategy was viable.</div><div class="NLM_p">With the knowledge that compound <b>3</b> likely bound specifically to the S3 site, a strategy was designed that would allow the linkage of compound <b>3</b> to a core molecule enabling direct access to the S3 and S3<sub>subpocket</sub> regions. From historical structural data, it was known that replacement of the pyridine ring of compound <b>1</b> with a toluyl ring at the 6-position of the core aminoquinoline allowed the ortho-methyl group to flip and face directly toward the S3 pocket (compound <b>5</b>, and structure in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C). This particular trajectory provided a direct line into the S3 pocket with which the linked molecule could possibly gain additional potency and selectivity against CatD. In order to test this hypothesis, interligand NOE (ILNOE) NMR experiments<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> were used in an attempt to identify the proximity and orientation of the bound fragment relative to another compound of known binding mode (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). For these experiments, a weaker 2-aminoquinoline tool molecule with a toluyl substitution at the 6 position of 2-aminoquinoline (compound <b>4</b>) was used as a probe (<i>K</i><sub>D</sub> ∼ 20 μM) to allow for strong NOEs between the methyl group and any protons of the <sup>19</sup>F fragment. As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, NOEs were clearly observed between the ortho-methyl group of the tool compound (<b>4</b>) and the 1, 2, and 3 protons of the chloropyridine moiety of the fragment (<b>3</b>). These data, along with structural data from compound <b>5</b> suggested that the ortho-methyl group of the tool compound and chloropyridine moiety of the fragment were within 5–6 Å from each other and that the fluorophenyl group of the <sup>19</sup>F fragment likely extended down into the hydrophobic S3<sub>subpocket</sub> of BACE-1. These results served to guide modeling efforts in attempts to design adequate linkers of these two compounds.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Transfer NOE experiments using compound <b>3</b> and compound <b>4</b>. NOEs are present between the number 11 proton of compound <b>4</b> and the number 1 proton (red circle) as well as the number 2 and 3 protons of compound <b>3</b> (black circle). Peaks present in the mixture spectrum (black) that are not present in the other two spectra of individual compounds (red) are inter-ligand NOEs between compound <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Using the spatial and orientation data obtained from the ILNOE experiments, molecular modeling studies were used to propose potential linking approaches to attach compound <b>3</b> onto the already potent core 2-aminoquinoline (compound <b>5</b>). With the observed orientation of the toluyl substituent (providing a nearly direct trajectory into the S3 region) coupled with the likely binding orientation of a benzamide S3 moiety (i.e., engagement of the Gly230 carbonyl by the amide NH and occupancy of the S3<sub>subpocket</sub> by the para-substituted arene), various rigidified linking agents were computationally probed via docking of the lowest energy adduct conformers into the BACE-1 active site. In all, four hybrid molecules, possessing linkers which were predicted to dispose the resultant pendent benzamide substituent in a manner described above, were proposed for synthesis (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed fragment-linked compounds derived from molecular modeling and ILNOE NMR experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Upon completion of the syntheses of compounds <b>6</b>–<b>9</b>, the compounds were tested for binding to BACE-1 by SPR. All of the compounds exhibited a reasonable improvement in affinity (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). However, conjugation of the fragment to the core molecule did result in a modest decrease in ligand efficiency (LE) (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) with compound <b>9</b> staying close to the generally targeted LE of 0.3. To test for potency and selectivity both in vitro and in vivo, the compounds were tested in BACE-1 enzyme and cellular assays along with a CatD enzyme assay (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> and <a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>). All compounds exhibited reasonable improvements in IC<sub>50</sub> values compared to the parent compound both in vitro and in vivo. However, compound <b>9</b> exhibited a roughly 350-fold gain in its affinity for BACE and only a 16-fold cell shift (compared with a cell shift of 380 for the parent molecule).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Most importantly, however, this particular compound exhibited a ∼2000-fold increase in selectivity against CatD (in both enzyme and cell assays) with respect to the parent compound (see data in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> and <a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SPR binding experiments to BACE-1 of (A) compound <b>6</b>, (B) compound <b>7</b>, (C) compound <b>8</b>, and (D) compound <b>9</b>. Reported <i>K</i><sub>D</sub> values are the average of two experiments, and the values in parentheses denote the fold increase in affinity over the parent molecule (compound <b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2">Table 2<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0012.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0013.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">All values reported in nanomolar.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Ligand efficiency values calculated using <i>K</i><sub>D</sub> from SPR.</p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">cLogP values calculated using ChemBioDraw Ultra.</p></div></div><div></div></div><div class="NLM_p">To thoroughly investigate the binding mode of the linked molecules, the compounds were submitted for X-ray crystallography in order to determine if the fluorophenyl group of the <sup>19</sup>F fragment was indeed binding in the S3 site. High-quality X-ray structures were determined for two of the compounds, thereby demonstrating unequivocally that the binding mode of the final compounds was consistent with the experimental results. In the crystal structure of BACE-1 with compound <b>6</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A), the aminoquinoline portion of the inhibitor engages the catalytic aspartic acid residues as expected, and the flexible aliphatic chain extending off from the aminoquinoline bends around and occupies the S1′ and S2′ pockets and forms a hydrogen bond with Gly34, as observed in the crystal structure of compound <b>5</b>. As predicted, the alkynyl linkage is oriented such that the cyclopropyl moiety binds in the S1 pocket and the fluorophenyl ring occupies the S3 pocket and part of the S3<sub>subpocket</sub>. In this position, compound <b>6</b> forms a hydrogen bond between its amide NH and the backbone carbonyl oxygen atom of Gly230 and between its amide carbonyl oxygen atom and a water molecule that in turn interacts with the backbone of Phe108. In the crystal structure of BACE-1 with compound <b>9</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B), many similar features are observed including interactions with the protein made by the aminoquinoline portion of the inhibitor derived from the original blocking compound. In place of the alkyne linker of compound <b>6</b>, however, compound <b>9</b> contains two methylene groups that allow a nearly 90° torsion angle to be made, enabling the pyridine ring to sit on the surface of the protein in the S1 pocket. The edge of the pyridine ring also makes van der Waals interactions with the face of the aminoquinoline ring. The amide NH of the inhibitor forms a hydrogen bond to Gly230, and while the amide moiety largely fills the S3 pocket, the fluorophenyl ring pushes deeply into the S3<sub>subpocket</sub>. These extensive interactions between compound <b>9</b> and BACE-1 likely account for its excellent in vitro and in vivo potency, and the deeper penetration of the fluorophenyl ring of compound <b>9</b> into the S3<sub>subpocket</sub>, relative to compound <b>6</b>, likely results in its higher selectivity over CatD. The crystallographic results also demonstrate why certain compounds, such as compound <b>3</b>, have stronger binding interactions in the presence of the blocking compound than in its absence.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Co-crystal structures of small-molecule ligands in complex with BACE-1. (A) Compound <b>6</b> and (B) compound <b>9</b>. Notice that the fluorphenyl group of compound <b>9</b> extends deep into the S3<sub>subpocket</sub>, providing additional potency against BACE-1 and additional selectivity against CatD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have demonstrated here the utility of <sup>19</sup>F-NMR in a fragment-based approach to designing potent and selective small-molecule inhibitors of BACE-1 using a fragment-linking approach. The final compound presented here was derived from a 2-aminoquinoline fragment originally proposed by Astex,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and structure-based drug design was used to optimize this compound into a highly potent molecule.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Here, we demonstrate that <sup>19</sup>F-NMR based fragment approaches are a rapid means to identify additional chemical matter that can be used in fragment-linking approaches to allow a higher degree of molecular diversity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76461" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76461" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> NMR Spectroscopy</h3><div class="NLM_p">All NMR experiments were performed on a Bruker Avance III NMR spectrometers (Bruker Biospin, Billerica, MA) operating at a <sup>1</sup>H frequency of 500.13 or 800.12 MHz. <sup>19</sup>F experiments were conducted using a SEF cryogenic probe equipped for direct <sup>19</sup>F detection while proton experiments were acquired at 800 MHz on a TCI cryoprobe. One-dimensional <sup>19</sup>F spectra were acquired for each sample at 283 K using <sup>1</sup>H decoupling with a spectra width of 71 428 Hz, an acquisition time of 917 ms, and 128 scans with a relaxation delay of 1 s. This yielded experiment times on the order of 6 min each and allowed 2 min for initial temperature equilibration. This experimental setup allowed all reference and screen spectra to be acquired in less than 24 h. The <sup>19</sup>F fragment screen for BACE-1 was conducted in 50 mM sodium acetate, pH 5.0, 100 mM NaCl, and 5% D<sub>2</sub>O (for field frequency lock). Protein and compound concentrations for the fragment screen were 10 μM and 20 μM, respectively.</div><div class="NLM_p">Two-dimensional Interligand NOE (ILNOE) experiments<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> were conducted in a similar buffer (except using deuterated components) using a standard two-dimensional <sup>1</sup>H–<sup>1</sup>H NOESY pulse sequence on an 800 MHz Bruker Avance III NMR spectrometer using a 500 ms mixing time and spectral widths of 8169 Hz (in F1 and F2) and 2048 and 512 data points in the direct and indirect dimensions, respectively.</div><div class="NLM_p last">All data were processed using Topspin 3.2 (Bruker Biospin, Billerica, MA) and were then compared visually to the reference spectra using the spectral overlay feature. Hits were identified by signal intensity and/or chemical shift changes. Because each compound in a pool had a unique chemical shift, hit identification was straightforward, and hit compounds could be identified by simply matching the chemical shift of the hit compound to that found in the compound database.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Surface Plasmon Resonance Spectroscopy</h3><div class="NLM_p">Dissociation constant (<i>K</i><sub>D</sub>) measurements were performed on Biacore S51 and SensiQ Pioneer FE SPR instruments (GE Healthcare and SensiQ Technologies, respectively). BACE-1 protein and inhibitors for SPR measurements were generated in-house; all other reagents were purchased from GE Healthcare, SensiQ, or Sigma-Aldrich. Glycosylated BACE-1 was reacted with sodium periodate to oxidize <i>cis</i>-diol groups on sugar chains to aldehydes. The oxidized BACE-1 was immobilized at high density (10 000–12 000 RU) onto either a CM5 or COOH5 sensor chip (GE Healthcare and SensiQ Technologies, respectively) using aldehyde coupling chemistry and resulted in surface activities close to 100% based on reference inhibitor binding. The immobilization running buffer consisted of 10 mM HEPES pH 7.4 with 150 mM NaCl and immobilization steps consisted of a 3–4 min EDC/NHS activation step [200 mM 1-ethyl-3-(3-(dimethylamino)propyl) carbodiimide hydrochloride, 50 mM <i>N</i>-hydroxysuccinimide], 7 min 5 mM carbohydrizide in water, 7 min 1 M ethanolamine hydrochloride pH 8.5, 10 min 20 μg/mL oxidized BACE-1 in 10 mM sodium acetate pH 4.0, 7 min 100 mM sodium cyanoborohydride in 100 mM sodium acetate pH 4.0, 3 × 30 s 50 mM glycine pH 9.5, and 3 × 30 s 1 M NaCl in 100 mM NaHCO<sub>3</sub> pH 9.5. The reference spot for all SPR experiments consisted of the blank surface of the sensor chip treated in an identical manner as the other flow cell spot excluding the addition of protein.</div><div class="NLM_p">For <i>K</i><sub>D</sub> measurements, the buffer was replaced with 50 mM sodium acetate pH 5.0, 150 mM NaCl, 0.005 (v/v) Tween-20, and 2% (v/v) DMSO. Compound stocks prepared in DMSO (typically 20 mM) were serially diluted in running buffer and injected over the immobilized BACE-1. Association and dissociation time were typically set to 45 and 90 s, respectively. However, in the case of the more potent compounds, dissociation times were extended in some cased to 20 min. All SPR experiments were performed at 25 °C with a flow rate of 30 μL/min and data collection rate of 10 Hz.</div><div class="NLM_p last">The data were processed and analyzed using either Scrubber-2 analysis software (BioLogic Software, Campbell, Australia) or QDat (SensiQ Technologies, Oklahoma City, OK). The sample response observed on the reference spot was subtracted from the sample response with immobilized BACE-1 to correct for systematic noise and baseline drift. Data was solvent corrected and the response from blank injections was used to double-reference the binding data. The data were molecular weight normalized and <i>K</i><sub>D</sub> values established using either simple steady analysis or single-site kinetic fit using a mass transport term (<i>k</i><sub>m</sub>).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Molecular Modeling</h3><div class="NLM_p last">Low-energy conformers for a variety of potential adducts possessing various rigidifying elements (alkynyl, cyclopropyl, etc. linkages) were generated with the MMFF94 force field as implemented in the MOE program suite<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> while constraining the aminoquinoline warhead and S1′-S2′ atoms to their positions found in the crystal structure of <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). The resultant conformers were then subjected to MM-GBSA minimization in the BACE-1 active site (rigid protein model) using the AMBER94/GAFF force field as implemented in AMBER 9.0<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Adducts possessing the best predicted binding affinities and which best presented the pendent benzamide NH in an orientation favorable for hydrogen bonding to the Gly230 carbonyl oxygen were proposed for synthesis.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-ray Crystallography</h3><div class="NLM_p last">The catalytic domain of human BACE-1 was expressed as inclusion bodies, purified, and apo crystals were grown as described previously.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The cocrystal structures of BACE-1 with compounds <b>1</b>,<b>2</b>, and <b>5</b> bound were obtained by soaking apo crystals in modified mother liquor solution (25% PEG 5000 MME, 0.1 M sodium citrate, pH 6.6, 0.2 M ammonium iodide, 3% (v/v) dimethyl sulfoxide) containing 0.5 mM compound for 6 h at room temperature. The crystals were then transferred briefly to the same solution supplemented with 20% (v/v) glycerol prior to flash-cooling for data collection. In order to obtain cocrystal structures of BACE-1 with compounds <b>6</b> and <b>9</b>, concentrated protein at ∼6 mg/mL was mixed with 0.5 mM inhibitor and incubated on ice for 90 min. The protein–inhibitor complexes were crystallized by hanging drop vapor diffusion at room temperature by mixing 1 μL of protein with 1 μL of precipitant solution (1.4–1.5 M ammonium sulfate, 0.2 M lithium chloride and either 0.1 M bis-tris (pH 5.5) or 0.1 M MES (pH 6.0)); wells contained 0.5 mL of precipitant solution. Single crystals appeared within 1–2 weeks. Prior to data collection, crystals were transferred briefly into cryo solutions consisting of mother liquor supplemented with 20% (v/v) glycerol and flash-cooled in liquid nitrogen. Diffraction data for cocrystals with compounds <b>9</b> and <b>12</b> were collected at SER-CAT beamline 22-BM at the Advanced Photon Source using a MAR225 CCD detector and λ = 1.0000 Å. Apo BACE-1 crystals soaked with compounds <b>1</b>, <b>2</b>, and <b>5</b> had data collected at the Advanced Light Source, beamline 5.0.2, using an ADSC Q315 CCD detector and λ = 1.0000 Å. Images were processed with HKL2000,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and the structures were refined using REFMAC.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Model building was performed with COOT.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Data collection and refinement statistics appear in Table S1 of the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> BACE-1 Enzymatic Assay</h3><div class="NLM_p last">BACE-1 and CatD enzymatic activities were determined by the enhancement of fluorescence intensity upon enzymatic cleavage of the fluorescence resonance energy transfer substrate. The BACE-1 recognition and cleavage sequence of the substrate is derived from the reported literature,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and the fluorophore and quencher dyes were attached to side chain of Lys residues at the termini of the substrate peptide. The human recombinant BACE-1<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>assay was performed in 50 mM acetate, pH 4.5, 8% DMSO, 100 μM Genepol, and 0.002% Brij-35. In dose–response IC<sub>50</sub> assays, 10 point 1:3 serial dilutions of compound in DMSO were preincubated with the enzyme for 60 min at room temperature. Subsequently, the substrate was added to initiate the reaction. After 60 min at room temperature, the reaction was stopped by addition of 0.1 M Tris base to raise the pH above the enzyme active range, and the increase of fluorescence intensity was measured on Safire II microplate reader (Tecan, Männedorf, Switzerland).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Chemistry</h3><div class="NLM_p">Unless otherwise noted, all materials were obtained from commercial suppliers and were used without further purification. All final compounds were purified to ≥95% purity as determined by high-performance liquid chromatography (HPLC).</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Synthesis of <i>N</i>-(1-(3-(2-Bromophenyl)prop-2-yn-1-yl)cyclopropyl)-4-fluorobenzamide Used in the Synthesis of Compound <b>6</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h4><div id="sec4_6_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i12" class="anchor-spacer"></div><h5 class="article-section__title" id="_i12"> Step 1: Synthesis of 3-((<i>tert</i>-Butyldimethylsilyl)oxy)propanenitrile</h5><div class="NLM_p">To a solution of 3-hydroxypropanenitrile (10 g, 141 mmol, 1 equiv) in tetrahydrofuran (200 mL) at 0 °C, was added imidazole (23.94 g, 352 mmol, 2.5 equiv) followed by <i>tert</i>-butyldimethylsilyl chloride (TBDMS-Cl) (25.4 g, 169 mmol, 1.2 equiv). After the addition was completed, the reaction mixture was stirred at room temperature for 16 h. After completion of the reaction (monitored by TLC), the mixture was quenched with water (20 mL) and extracted with ethyl acetate (2 × 100 mL). The combined organic layers were washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to give the crude material as a light-yellow oil. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 10% ethyl acetate in hexane to provide 3-((<i>tert</i>-butyldimethylsilyl)oxy)propanenitrile (20 g, 108 mmol, 77% yield) as a colorless liquid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 3.86 (t, <i>J</i> = 6.4 Hz, 2H), 2.56 (t, <i>J</i> = 6.3 Hz, 2H), 0.93 (s, 9H), 0.11 (s, 6H).</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0006.jpeg" id="GRAPHIC-d717e1084-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_6_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i13" class="anchor-spacer"></div><h5 class="article-section__title" id="_i13"> Step 2: Synthesis of 1-(2-((<i>tert</i>-Butyldimethylsilyl)oxy)ethyl)cyclopropan-1-amine</h5><div class="NLM_p last">To a solution of 3-((<i>tert</i>-butyldimethylsilyl)oxy)propanenitrile (20 g, 108 mmol, 1.0 equiv) in diethyl ether (200 mL), was added titanium(IV) isopropoxide (53.8 mL, 183 mmol, 1.7 equiv) at 0 °C. Ethylmagnesium bromide (90 mL, 270 mmol, 2.5 equiv, 3.0 M in diethyl ether) was added slowly. After addition, the reaction mixture was stirred at room temperature for 1 h and again cooled to 0 °C. To this mixture, boron trifluoride etherate was added (54.7 mL, 216 mmol, 2.0 equiv, 50% solution) dropwise. After addition, the reaction mixture was stirred at room temperature for 1 h. After completion of the reaction (monitored by TLC), the mixture was quenched with 10% sodium hydroxide solution (150 mL) and extracted with ethyl acetate (2 × 150 mL). The combined organic layer was washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to give the crude material as brown liquid. The crude material was purified by flash chromatography column (silica gel, 230–400 mesh), eluting with a gradient of 0 to 10% ethyl acetate in hexane to provide 1-(2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)cyclopropan-1-amine (10 g, 46.4 mmol, 43% yield) as a brown liquid. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 3.89 (d, <i>J</i> = 5.9 Hz, 2H), 2.92 (s, 2H), 1.69 (d, <i>J</i> = 5.9 Hz, 2H), 0.91 (s, 9H), 0.77–0.74 (m, 2H), 0.54–0.51 (m, 2H), 0.12 (s, 6H).</div></div><div id="sec4_6_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i14" class="anchor-spacer"></div><h5 class="article-section__title" id="_i14"> Step 3: Synthesis of <i>N</i>-(1-(2-((<i>tert</i>-Butyldimethylsilyl)oxy)ethyl)cyclopropyl)-4-fluorobenzamide</h5><div class="NLM_p last">To a solution of 1-(2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)cyclopropan-1-amine (10 g, 46.4 mmol, 1.0 equiv) and triethylamine (9.71 mL, 69.6 mmol, 1.5 equiv) in dichloromethane (100 mL) cooled to 0 °C, was added 4-fluorobenzoyl chloride(8.83 g, 55.7 mmol, 1.2 equiv). After addition, the reaction mixture was stirred at room temperature for 3 h. After completion of the reaction (monitored by TLC), the mixture was quenched with water (100 mL) and extracted with dichloromethane (2 × 50 mL). The combined organic layer was washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to give the crude material (light-yellow oil). The crude material was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 10% ethyl acetate in hexane, to provide <i>N</i>-(1-(2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)cyclopropyl)-4-fluorobenzamide (6 g, 17.78 mmol, 38.3% yield) as light yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 338.2 (M + 1). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.76–7.72 (m, 2H), 7.12–7.07 (m, 2H), 6.54 (s, 1H), 3.82 (t, <i>J</i> = 5.9 Hz, 2H), 1.90 (t, <i>J</i> = 6.1 Hz, 2H), 0.96–0.94 (m, 2H), 0.87 (s, 9H), 0.84–0.80 (m, 2H), 0.04 (s, 6H).</div></div><div id="sec4_6_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i15" class="anchor-spacer"></div><h5 class="article-section__title" id="_i15"> Step 4: Synthesis of 4-Fluoro-<i>N</i>-(1-(2-hydroxyethyl)cyclopropyl)benzamide</h5><div class="NLM_p last">To a solution of <i>N</i>-(1-(2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl) cyclopropyl)-4-fluorobenzamide (6 g, 17.78 mmol, 1.0 equiv) in tetrahydrofuran (60 mL) was added tetra-<i>N</i>-butylammonium fluoride (35.6 mL, 35.56 mmol, 2.0 equiv, 1.0 M in tetrahydrofuran) at room temperature and stirred at room temperature for 3 h. After completion of the reaction (monitored by TLC), the reaction mixture was quenched with water (60 mL) and extracted with ethyl acetate (2 × 60 mL). The organic layer was washed with brine (30 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material (light-yellow solid), which was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 30% Ethyl acetate in hexane to provide 4-fluoro-<i>N</i>-(1-(2-hydroxyethyl)cyclopropyl)benzamide (2.8 g, 12.54 mmol, 70.6% yield) as an off-white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 224.2 (M + 1). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.79–7.75 (m, 2H), 7.15–7.10 (m, 2H), 6.54 (s, 1H), 3.76–3.73 (m, 2H), 1.79–1.76 (m, 2H), 1.00–0.90 (m, 4H).</div></div><div id="sec4_6_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> Step 5: Synthesis of 4-Fluoro-<i>N</i>-(1-(2-oxoethyl)cyclopropyl)benzamide</h5><div class="NLM_p">To a solution of 4-fluoro-<i>N</i>-(1-(2-hydroxyethyl)cyclopropyl)benzamide (1.0 g, 4.48 mmol, 1.0 equiv) in dichloromethane (30 mL), was added Dess–Martin periodinane (2.85 g, 6.72 mmol, 1.5 equiv) at 0 °C, and then the reaction mixture was stirred at room temperature for 3 h. After completion of the reaction (monitored by TLC), the reaction was quenched with 10% sodium bicarbonate (50 mL) and extracted with dichloromethane (2 × 50 mL). The organic layer was washed with brine (10 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material (light-yellow gum), which was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 20% ethyl acetate in hexane to provide 4-fluoro-<i>N</i>-(1-(2-oxoethyl)cyclopropyl)benzamide (0.5 g, 2.26 mmol, 50.5% yield), as an off-white solid.</div><div class="NLM_p last">MS (ESI positive ion) <i>m</i>/<i>z</i>: 222.2 (M+1). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.85 (s, 1H), 7.76–7.71 (m, 2H), 7.13–7.08 (m, 2H), 6.77 (s, 1H), 2.92 (s, 2H), 1.04–1.01 (m, 2H), 0.87–0.85 (m, 2H).</div></div><div id="sec4_6_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> Step 6: Synthesis of 4-Fluoro-<i>N</i>-(1-(prop-2-yn-1-yl)cyclopropyl)benzamide</h5><div class="NLM_p">To a solution of 4-fluoro-<i>N</i>-(1-(2-oxoethyl)cyclopropyl)benzamide (500 mg, 2.26 mmol, 1.0 equiv) and potassium carbonate (1.2 g, 9.04 mmol, 4.0 equiv) in methanol (10 mL), was added dimethyl (1-diazo-2-oxo-propyl) phosphonate (521 mg, 2.71 mmol, 1.2 equiv) at 0 °C. After addition, the reaction mixture was stirred at room temperature for 4 h. After completion of the reaction (monitored by TLC), the mixture was quenched with water (15 mL) and extracted with ethyl acetate (2 × 20 mL). The organic layer was washed with brine (5 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material (light-yellow gum), which was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 10% ethyl acetate in hexane to provide 4-fluoro-<i>N</i>-(1-(prop-2-yn-1-yl)cyclopropyl)benzamide (300 mg, 1.38 mmol, 61.1% yield), as a light yellow solid.</div><div class="NLM_p last">MS (ESI positive ion) <i>m</i>/<i>z</i>: 218.2 (M+1). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.78–7.74 (m, 2H), 7.14–7.08 (m, 2H), 6.55 (s, 1H), 2.70 (s, 2H), 2.00 (t, <i>J</i> = 2.6 Hz, 1H), 0.99–0.94 (m, 4H).</div></div><div id="sec4_6_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Step 7: Synthesis of <i>N</i>-(1-(3-(2-bromophenyl)prop-2-yn-1-yl)cyclopropyl)-4-fluorobenzamide</h5><div class="NLM_p">To a solution of 4-fluoro-<i>N</i>-(1-(prop-2-yn-1-yl) cyclopropyl)benzamide (300 mg, 1.381 mmol, 1.0 equiv) in tetrahydrofuran (10 mL), was added 1-iodo-2-bromo-benzene (391 mg, 1.381 mmol, 1.0 equiv), triethylamine (210 mg, 2.071 mmol, 1.5 equiv), triphenyl phosphine (18.11 mg, 0.069 mmol, 0.05 equiv), and copper(I) iodide (26.3 mg, 0.138 mmol, 0.1 equiv) at room temperature. The reaction mixture was degassed with nitrogen, bis(triphenylphosphine)palladium(II)chloride (48.5 mg, 0.069 mmol, 0.05 equiv) was added, and the reaction mixture stirred at room temperature for 4 h. After completion (monitored by TLC), the reaction was quenched with water (15 mL), and extracted with ethyl acetate (2 × 10 mL). The organic layer was washed with brine (3 mL), and the organic extract was dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material. The crude product was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 10% ethyl acetate in hexane to provide <i>N</i>-(1-(3-(2-bromophenyl) prop-2-yn-1-yl)cyclopropyl)-4-fluorobenzamide (250 mg, 0.672 mmol, 48.6% yield) as an off-white solid.</div><div class="NLM_p last">MS (ESI positive ion) <i>m</i>/<i>z</i>: 372.0 (M+). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.81–7.75 (m, 2H), 7.56 (d, <i>J</i> = 8.0 Hz, 1H), 7.43–7.40 (m, 1H), 7.24–7.07 (m, 4H), 6.70 (s, 1H), 2.95 (s, 2H), 1.06–0.96 (m, 4H).</div></div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of Compound <b>6</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a></h4><div id="sec4_6_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Step 1: Synthesis of 6-Bromo-2-((4-methoxybenzyl)amino)quinoline-3-carbaldehyde</h5><div class="NLM_p">To a solution of 6-bromo-2-chloroquinoline-3-carbaldehyde (10 g, 37.0 mmol, 1.0 equiv) in absolute ethanol (200 mL in 1 L sealed tube) was added 4-methoxy benzyl amine (10.56 mL, 81 mmol, 2.2 equiv) at room temperature, and the mixture was heated to 90 °C for 24 h. The reaction was not completed as indicated by thin-layer chromatography (TLC). This reaction mixture was cooled to room temperature, and 4-methoxy benzyl amine (4.8 mL, 37 mmol, 1.0 equiv) was added and then heated to 95 °C for an additional 24 h. Once the reaction was completed (monitored by TLC), the reaction mixture was cooled to room temperature, and aqueous 6 N HCl was added (50 mL) and then heated to 90 °C for 24 h. After completion of the reaction (monitored by LC-MS), the reaction mixture was cooled to room temperature, and ethanol was removed under vacuum. The resulting solid was washed with water (3 × 30 mL), methanol (2 × 5 mL), and dried to afford 6-bromo-2-((4-methoxybenzyl)amino)quinoline-3-carbaldehyde (<b>2</b>, 10 g, 72.9%) as a yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 371.0, 373.0 (M+, M + 2). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 10.05 (s, 1H), 8.49 (br s, 1H), 8.24 (s, 1H), 7.88–7.74 (m, 3H), 7.38 (d, <i>J</i> = 8.4 Hz, 2H), 6.90 (d, <i>J</i> = 8.4 Hz, 2H), 4.88 (s, 2H), 3.85 (s, 3H).</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0007.jpeg" id="GRAPHIC-d717e1350-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_6_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Step 2: Synthesis of Methyl 3-(6-Bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylate</h5><div class="NLM_p last">To a solution of 6-bromo-2-((4-methoxybenzyl)amino)quinoline-3-carbaldehyde (5 g, 13.47 mmol, 1.0 equiv) in anhydrous dicholormethane (100 mL) at 0 °C was added methyl 2-(triphenylphosphoranylidene)acetate (4.50 g, 13.47 mmol, 1.0 equiv) under a nitrogen atmosphere, and the reaction mixture was stirred at room temperature for 3 h. After completion of the reaction (monitored by TLC), the reaction mixture was concentrated under vacuum. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh) using 15 to 20% of ethyl acetate in petroleum ether as an eluent to afford methyl 3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylate (3.76 g, 65.3%) as a yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 427.0, 429.0 (M+, M + 2). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.85 (s, 1H), 7.75–7.61 (m, 4H), 7.35 (d, <i>J</i> = 8.76 Hz, 2H), 6.93–6.87 (m, 2H), 6.47 (d, <i>J</i> = 15.7 Hz, 1H), 5.01 (br s, 1H), 4.74 (d, <i>J</i> = 5.1 Hz, 2H), 3.82 (s, 6H).</div></div><div id="sec4_6_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Step 3: Synthesis of (<i>E</i>)-3-(6-Bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylic acid</h5><div class="NLM_p last">To a solution of compound methyl 3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylate (3.6 g, 8.43 mmol, 1.0 equiv) in tetrahydrofuran (40 mL) and water (40 mL) was added lithium hydroxide mono hydrate (0.404 g, 16.85 mmol, 2.0 equiv) at room temperature. After addition, the reaction mixture was stirred at room temperature for 4 h. After completion of the reaction (monitored by TLC), the reaction was quenched with water (40 mL) and extracted with ethyl acetate (2 × 60 mL). The combined organic extract was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to provide (<i>E</i>)-3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylic acid (3.4 g, 8.23 mmol, 98% yield) as a light yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 413 (M+). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.59 (s, 1H), 8.06 (s, 1H), 7.90–7.85 (m, 3H), 7.40 (d, <i>J</i> = 8.6 Hz, 2H), 6.91 (d, <i>J</i> = 8.6 Hz, 2H), 6.64 (d, <i>J</i> = 15.3 Hz, 1H), 4.85 (s, 2H), 3.72 (s, 3H).</div></div><div id="sec4_6_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step 4: Synthesis of (<i>E</i>)-3-(6-Bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)acrylamide</h5><div class="NLM_p last">To a solution of (<i>E</i>)-3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)acrylic acid (3.4 g, 8.23 mmol, 1.0 equiv) in dichloromethane (100 mL), was added triethylamine (5.73 mL, 41.1 mmol, 5.0 equiv) and 2,4,6-tripropyl-1,3,5,2,4,6-trioxatriphosphinane 2,4,6-trioxide (7.85 g, 12.34 mmol, 1.5 equiv) in 50% ethyl acetate/dichloromethane at room temperature. The mixture was then cooled to 0 °C, and 3,3-dimethylbutan-1-amine (0.999 g, 9.87 mmol, 1.2 equiv) was added. After addition, the reaction mixture was warmed up to room temperature and stirred for 16 h. After completion (monitored by TLC) of the reaction, the reaction was quenched with water (60 mL) and extracted with ethyl acetate (2 × 60 mL). The organic layer was washed with brine solution (30 mL), and the organic extract was dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material as a light yellow solid. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 30% ethyl acetate in hexane, to provide (<i>E</i>)-3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)acrylamide (2.5 g, 5.04 mmol, 61.2% yield), as a light yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 496.0, 498.0 (M+, M + 2). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.78–7.61 (m, 5H), 7.36 (d, <i>J</i> = 8.5 Hz, 2H), 6.89 (d, <i>J</i> = 8.5 Hz, 2H), 6.38 (d, <i>J</i> = 15.1 Hz, 1H), 5.60 (s, 1H), 5.10 (s, 1H), 4.74 (s, 2H), 3.84 (s, 3H), 3.44–3.38 (m, 2H), 1.52–1.48 (m, 2H), 0.98 (s, 9H).</div></div><div id="sec4_6_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step 5: Synthesis of 3-(6-Bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide</h5><div class="NLM_p last">To a solution of (<i>E</i>)-3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)acrylamide (1 g, 2.014 mmol, 1.0 equiv) in ethyl acetate (20 mL), was added 10% platinum on carbon (100 mg, 10 wt %) at room temperature. After addition, the reaction mixture was stirred under hydrogen pressure (5 psi) at room temperature for 4 h. After completion (monitored by LCMS), the reaction mixture was passed through a Celite bed and washed with ethyl acetate (100 mL). The filtrate was concentrated in vacuo to yield 3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (1 g, 2.006 mmol, 100% yield) as a light yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 498.0, 500.0 (M+, M + 2). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.68 (d, <i>J</i> = 2.1 Hz, 1H), 7.63–7.56 (m, 2H), 7.50 (s, 1H), 7.39 (d, <i>J</i> = 8.5 Hz, 2H), 6.89 (d, <i>J</i> = 8.6 Hz, 2H), 5.69 (s, 1H), 5.32 (s, 1H), 4.75 (d, <i>J</i> = 5.1 Hz, 2H), 3.81 (s, 3H), 3.24–3.18 (m, 2H), 2.93 (t, <i>J</i> = 7.1 Hz, 2H), 2.49 (t, <i>J</i> = 7.1 Hz, 2H), 1.33–1.28 (m, 2H), 0.90 (s, 9H).</div></div><div id="sec4_6_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Step 6: Synthesis of 3-(2-Amino-6-bromoquinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide</h5><div class="NLM_p last">To a solution of 3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (500 mg, 1.003 mmol, 1.0 equiv) in trifluoroacetic acid (10 mL), was added anisole (1096 μL, 10.03 mmol, 10 equiv) at room temperature. After addition, the reaction mixture was stirred at 50 °C for 3 h. After completion of the reaction (monitored by TLC), the reaction mixture was concentrated and then diluted with ethyl acetate (40 mL). The organic layer was washed with 10% NaHCO<sub>3</sub> solution (10 mL) and water (5 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and finally concentrated in vacuo to yield the crude 3-(2-amino-6-bromoquinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (300 mg, 0.793 mmol, 79% yield) as a light yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 378.0 (M+). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.83–7.80 (m, 2H), 7.79 (s, 1H), 7.52 (d, <i>J</i> = 8.8 Hz, 1H), 7.37 (d, <i>J</i> = 8.8 Hz, 1H), 6.49 (s, 2H), 3.06–3.00 (m, 2H), 2.80 (t, <i>J</i> = 7.2 Hz, 2H), 2.43 (t, <i>J</i> = 7.2 Hz, 2H), 1.27–1.22 (m, 2H), 0.87 (s, 9H).</div></div><div id="sec4_6_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Step 7: Synthesis of 3-(2-Amino-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide</h5><div class="NLM_p last">To a solution of 3-(2-amino-6-bromoquinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (300 mg, 0.793 mmol, 1.0 equiv) in 1,4-dioxane (6 mL), was added potassium acetate (233 mg, 2.379 mmol, 3.0 equiv) and bis(pinacolato)diboron (242 mg, 0.952 mmol, 1.2 equiv). The mixture was then degassed with nitrogen, and PdCl<sub>2</sub>(dppf)–CH<sub>2</sub>Cl<sub>2</sub> (32.4 mg, 0.040 mmol, 0.05 equiv) was added at room temperature. After addition, the reaction mixture was stirred at 85 °C for 4 h. After completion of the reaction (monitored by TLC), the mixture was quenched with water (6 mL) and extracted with ethyl acetate (2 × 15 mL). The organic extract was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give the crude material which was passed through a bed of silica gel (for removing of inorganics) to provide the crude 3-(2-amino-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (400 mg) as a brown solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 426.2 (M + 1).</div></div><div id="sec4_6_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Step 8: Synthesis of <i>N</i>-(1-(3-(2-(2-Amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenyl)prop-2-yn-1-yl)cyclopropyl)-4-fluorobenzamide (Compound <b>6</b>)</h5><div class="NLM_p last">To a solution of 3-(2-amino-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (65% purity, 175 mg, 0.267 mmol, 1.0 equiv) and <i>N</i>-(1-(3-(2-bromophenyl) prop-2-yn-1-yl)cyclopropyl)-4-fluorobenzamide (100 mg, 0.267 mmol, 1.0 equiv) taken in 1,4-dioxane (4 mL) and water (1 mL), was added sodium carbonate (56.7 mg, 0.535 mmol, 2.0 equiv). The mixture was then degassed with nitrogen. Bis(triphenylphosphine)palladium(II)chloride (9.38 mg, 0.013 mmol, 0.05 equiv) was added at room temperature, and the reaction mixture stirred at 85 °C for 4 h. After completion of the reaction (monitored by TLC), the reaction mixture was quenched with water (5 mL) and extracted with ethyl acetate (2 × 20 mL). The organic extract was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give the crude material as a brown gum. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh), eluting with a gradient of 0 to 30% methanol in chloroform, to provide 100 mg compound <b>6</b>, which was further purified by preparative HPLC [YMC C8 (20 × 250 mm, 5 μ); mobile phase: 10 mM NH<sub>4</sub>OAc in water and acetonitrile; flow rate: 18 mL/min]. After concentration, the residue was mixed with water and extracted with dichloromethane. The organic extract was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated under reduced pressure. The residue was washed with hexane and lyophilized to provide compound <b>6</b> (14 mg, 0.024 mmol, 8.86% yield), as an off-white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 591.2 (M+1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.70 (s, 1H), 7.84–7.81 (m, 3H), 7.72–7.71 (m, 2H), 7.65 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.51 (d, <i>J</i> = 7.6 Hz, 1H), 7.44–7.39 (m, 3H), 7.34–7.29 (m, 1H), 7.25–7.21 (m, 2H), 6.40 (s, 2H), 3.06–3.00 (m, 2H), 2.88–2.78 (m, 4H), 2.42 (t, <i>J</i> = 7.6 Hz, 2H), 1.28–1.26 (m, 2H), 0.83 (s, 9H), 0.70–0.67 (m, 4H); <sup>19</sup>F NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ −109.4.</div></div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Synthesis of Compound <b>7</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h4><div id="sec4_6_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step 1: Synthesis of Methyl 3-Aminocyclobutane-1-carboxylate</h5><div class="NLM_p">To a solution of methyl 3-((tert-butoxycarbonyl)amino)cyclobutanecarboxylate (2 g, 8.72 mmol, 1.0 equiv) in dichloromethane (3 mL) at 0 °C was added dropwise trifluoroacetic acid (6.64 mL, 87 mmol, 10 equiv) under nitrogen atmosphere, and the reaction mixture was stirred at room temperature for 2 h. After completion of the reaction (monitored by TLC), the excess TFA was removed by concentration under vacuum (kept bath temperature below 40 °C). To the resulting mixture was added methanol (5 mL) followed by solid sodium bicarbonate (5 g), and the reaction was stirred for 10 min. The reaction mixture was diluted with DCM (45 mL), filtered through a Celite pad, and washed with dichloromethane:methanol (9:1, 3 × 20 mL). The filtrate was concentrated under vacuum to yield methyl 3-aminocyclobutanecarboxylate (1.05 g, 93%) as a colorless liquid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 130.2 (M+1). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.96 (s, 2H), 3.64–3.51 (m, 4H), 3.01–2.89 (m, 1H), 2.44–2.37 (m, 2H), 2.27–2.18 (m, 2H).</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0008.jpeg" id="GRAPHIC-d717e1709-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_6_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Step 2: Synthesis of Methyl 3-(4-Fluorobenzamido)cyclobutane-1-carboxylate (<b>3</b>)</h5><div class="NLM_p">To the suspension of methyl 3-aminocyclobutanecarboxylate (1.0 g, 7.74 mmol, 1.0 equiv) in anhydrous DCM (10 mL) was added triethylamine (1.62 mL, 11.61 mmol, 1.5 equiv) at room temperature under nitrogen atmosphere, and the reaction mixture was cooled to 0 °C. 4-Fluorobenzoyl chloride (1.0 mL, 8.52 mmol, 1.1 equiv) was added dropwise to the above reaction mixture and stirred at room temperature for 2 h. After completion of the reaction (monitored by TLC), the reaction mixture was quenched with ice-cold water (10 mL) and separated into biphasic layers. The aqueous layer was extracted with DCM (10 mL), and the combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh) using 25 to 30% of ethyl acetate in petroleum ether as an eluent to afford methyl 3-(4-fluorobenzamido)cyclobutanecarboxylate (1.9 g, 98%) as a white solid.</div><div class="NLM_p">MS (ESI positive ion) <i>m</i>/<i>z</i>: 252.0 (M + 1).</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.82–7.78 (m, 2H), 7.13 (d, <i>J</i> = 8.6 Hz, 2H), 6.45 (d, <i>J</i> = 8.3 Hz, 1H), 4.68–4.58 (m, 1H), 3.75 (s, 3H), 2.98–2.89 (m, 1H), 2.80–2.73 (m, 2H), 2.28–2.20 (m, 2H).</div></div><div id="sec4_6_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Step 3: Synthesis of 4-Fluoro-<i>N</i>-(3-(hydroxymethyl)cyclobutyl)benzamide</h5><div class="NLM_p last">To a cooled solution of methyl 3-(4-fluorobenzamido)cyclobutanecarboxylate (1.45 g, 5.77 mmol, 1.0 equiv) in anhydrous THF (15 mL) and methanol (0.93 mL, 23.08 mmol, 4.0 equiv) at 0 °C was added sodium triacetoxyborohydride (0.122 g, 0.577 mmol, 0.1 equiv) followed by portion-wise addition of sodium borohydride (0.284 g, 7.50 mmol, 1.3 equiv) under nitrogen atmosphere. Then the reaction was allowed to stir at room temperature for 16 h. After completion of the reaction (monitored by TLC), the reaction mixture was cooled to 0 °C and quenched with aqueous 10% NaHCO<sub>3</sub> solution (15 mL) and stirred for 15 min at room temperature. The reaction mixture was then extracted with ethyl acetate (3 × 15 mL), and combined organic layers were washed with brine (15 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 60 to 80% of ethyl acetate in petroleum ether as an eluent to afford 4-fluoro-<i>N</i>-(3-hydroxymethyl)cyclobutyl)benzamide (1.2 g, 93%) as a white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 224.2 (M + 1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.57 (d, <i>J</i> = 7.5 Hz, 1H), 7.94–7.89 (m, 2H), 7.30–7.24 (m, 2H), 4.51 (t, <i>J</i> = 5.2 Hz, 1H), 4.31–4.20 (m, 1H), 3.36 (t, <i>J</i> = 5.4 Hz, 2H), 2.26–2.21 (m, 2H), 2.10–2.04 (m, 1H), 1.81–1.73 (m, 2H).</div></div><div id="sec4_6_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Step 4: Synthesis of 4-Fluoro-<i>N</i>-(3-formylcyclobutyl)benzamide</h5><div class="NLM_p last">To a 0 °C cooled suspension of 4-fluoro-<i>N</i>-(3-(hydroxymethyl)cyclobutyl)benzamide (1.2 g, 5.38 mmol, 1.0 equiv) in anhydrous DCM (30 mL) was added Dess–Martin periodinane (3.42 g, 8.06 mmol, 1.5 equiv) under nitrogen atmosphere, and the reaction mixture was stirred at room temperature for 4 h. After completion of the reaction (monitored by TLC), the reaction mixture was cooled to 0 °C and quenched with aqueous 10% NaHCO<sub>3</sub> solution (30 mL) and extracted with DCM:methanol (9:1, 3 × 60 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 35 to 50% of ethyl acetate in petroleum ether as an eluent to afford 4-fluoro-<i>N</i>-(3-formylcyclobutyl)benzamide (0.86 g, 72.3%) as a white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 222.0 (M+1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.64 (s, 1H), 8.69 (d, <i>J</i> = 7.6 Hz, 1H), 7.94–7.90 (m, 2H), 7.32–7.27 (m, 2H), 4.49–4.40 (m, 1H), 3.02–2.93 (m, 1H), 2.44–2.37 (m, 2H), 2.29–2.22 (m, 2H).</div></div><div id="sec4_6_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Step 5: Synthesis of <i>N</i>-(3-Ethynylcyclobutyl)-4-fluorobenzamide</h5><div class="NLM_p last">To the solution of 4-fluoro-<i>N</i>-(3-formylcyclobutyl)benzamide (0.86 g, 3.89 mmol, 1.0 equiv) in anhydrous methanol (17 mL) was added anhydrous potassium carbonate (2.149 g, 15.55 mmol, 4 equiv) at room temperature under nitrogen atmosphere followed by dimethyl (1-diazo-2-oxopropyl)phosphonate (0.896 g, 4.66 mmol, 1.2 equiv), and the mixture was stirred for 2 h. After completion of the reaction (indicated by TLC), the reaction mixture was quenched with ice-cold water (35 mL) and extracted with 1:1 mixture of petroleum ether: ethyl acetate (3 × 20 mL). The combined organic layers was washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 20 to 25% of ethyl acetate in petroleum ether as an eluent to afford <i>N</i>-(3-ethynylcyclobutyl)-4-fluorobenzamide (0.465 g, 55.1%) as a white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 218.0 (M + 1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.79–7.76 (m, 2H), 7.15–7.10 (m, 2H), 6.21 (s, 1H), 4.58–4.45 (m, 1H), 2.89–2.76 (m, 3H), 2.34 (tdd, <i>J</i> = 9.6, 7.6, 2.6 Hz, 1H), 2.11 (qd, <i>J</i> = 8.9, 2.3 Hz, 2H).</div></div><div id="sec4_6_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Step 6: Synthesis of <i>N</i>-(3-((2-Bromophenyl)ethynyl)cyclobutyl)-4-fluorobenzamide</h5><div class="NLM_p last">To the mixture of <i>N</i>-(3-ethynylcyclobutyl)-4-fluorobenzamide (0.435 g, 2.002 mmol, 1.0 equiv), 1-bromo-2-iodobenzene (0.566 g, 0.680 mmol, 1.2 equiv), triphenylphosphine (0.026 g, 0.100 mmol, 0.05 equiv), bis(triphenylphosphine)palladium(II) chloride (0.070 g, 0.100 mmol, 0.05 equiv), and triethylamine (0.419 mL, 3.00 mmol, 1.5 equiv) was added anhydrous tetrahydrofuran (10 mL, degassed with nitrogen gas for 30 min) followed by copper(I) iodide (0.038 g, 0.200 mmol, 0.1 equiv) at room temperature under nitrogen atmosphere, and the reaction mixture was stirred for 16 h. After completion of the reaction (monitored by TLC), water (10 mL) was added to the reaction mixture and extracted with ethyl acetate (3 × 5 mL). The combined organic layers were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 13 to 18% of ethyl acetate in petroleum ether as an eluent to afford <i>N</i>-(3-((2-bromophenyl)ethynyl)cyclobutyl)-4-fluorobenzamide (0.37 g, 49.6%) as a white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 372.0, 374.0 (M, M+2). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.84–7.74 (m, 2H), 7.62–7.56 (m, 1H), 7.43 (dd, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.23 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.18–7.08 (m, 3H), 6.28 (s, 1H), 4.59 (q, <i>J</i> = 8.2 Hz, 1H), 3.09–2.93 (m, 3H), 2.27–2.18 (m, 2H).</div></div><div id="sec4_6_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Step 7: Synthesis of <i>N</i>-(3-((2-(2-Amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenyl)ethynyl)cyclobutyl)-4-fluorobenzamide (Compound <b>7</b>)</h5><div class="NLM_p last">To a solution of 3-(2-amino-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (65% purity, 175 mg, 0.267 mmol, 1.0 equiv) and <i>N</i>-(3-((2-bromophenyl)ethynyl)cyclobutyl)-4-fluorobenzamide (Step 6, 100 mg, 0.267 mmol, 1.0 equiv) taken in 1,4-dioxane (4 mL) and water (1 mL), was added sodium carbonate (56.7 mg, 0.535 mmol, 2.0 equiv), then degassed with nitrogen. Bis(triphenylphosphine)palladium(II) chloride (9.38 mg, 0.013 mmol, 0.05 equiv) was added at room temperature, and then the reaction mixture was stirred at 85 °C for 4 h. After completion of the reaction (monitored by TLC), the reaction mixture was quenched with water (5 mL) and extracted with ethyl acetate (2 × 20 mL). The organic extract was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give the crude material. The crude material was purified by flash column chromatography (silica gel, 230–400 mesh) eluting with a gradient of 0% to 30% methanol in chloroform, to provide 100 mg of the title product which was further purified by preparative HPLC [YMC C8 (20 × 250 mm, 5 μ); mobile phase: 10 mM NH<sub>4</sub>OAc in water and acetonitrile; flow rate: 18 mL/min]. After concentration the residue was taken in water and extracted with dichloromethane, the organic extract was dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo and the residue was washed with hexane and then lyophilized, to provide compound <b>7</b> (11 mg, 0.019 mmol, 6.96% yield) as an off-white solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 591.2 (M+1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.71 (d, <i>J</i> = 7.6 Hz, 1H), 7.93–7.87 (m, 2H), 7.85–7.78 (m, 2H), 7.76 (s, 1H), 7.71 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 7.55–7.46 (m, 3H), 7.44 (dd, <i>J</i> = 7.4, 1.5 Hz, 1H), 7.35–7.23 (m, 3H), 6.39 (s, 2H), 4.31 (d, <i>J</i> = 7.9 Hz, 1H), 3.06–3.00 (m, 2H), 2.92–2.85 (m, 1H), 2.80 (t, <i>J</i> = 7.4 Hz, 2H), 2.55 (t, <i>J</i> = 1.9 Hz, 2H), 2.42 (d, <i>J</i> = 7.5 Hz, 2H), 2.13 (q, <i>J</i> = 10.3 Hz, 2H), 1.29–1.24 (m, 2H), 0.84 (s, 9H); <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ −109.6.</div></div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Synthesis of Compound <b>8</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div id="sec4_6_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step 1: Synthesis of <i>N</i>-(6-Bromopyridin-3-yl)-4-fluorobenzamide</h5><div class="NLM_p">To the solution of 6-bromopyridin-3-amine (3 g, 17.34 mmol, 1.0 equiv) in anhydrous DCM (30 mL) was added triethylamine (3.63 mL, 26.0 mmol, 1.5 equiv) at room temperature under nitrogen atmosphere and cooled to 0 °C. 4-Fluorobenzoyl chloride (2.45 mL, 20.81 mmol, 1.2 equiv) was added dropwise to the above reaction mixture and stirred at room temperature for 5 h. After completion of the reaction (monitored by TLC), the reaction mixture was quenched with ice-cold water (30 mL), and the biphasic layers were separated. The aqueous layer was extracted with DCM (30 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum to afford crude <i>N</i>-(6-bromopyridin-3-yl)-4-fluorobenzamide (5.25 g) as a brown solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 295.0, 297.0 (M, M + 2). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.50 (s, 1H), 8.22 (ddd, <i>J</i> = 8.7, 3.1, 1.4 Hz, 1H), 7.94–7.88 (m, 3H), 7.52 (d, <i>J</i> = 8.6 Hz, 1H), 7.24–7.18 (m, 2H).</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0009.jpeg" id="GRAPHIC-d717e2049-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_6_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Step 2: Synthesis of 4-Fluoro-<i>N</i>-(6-((trimethylsilyl)ethynyl)pyridin-3-yl)benzamide</h5><div class="NLM_p last">To the mixture of <i>N</i>-(6-bromopyridin-3-yl)-4-fluorobenzamide (5.9 g, 19.99 mmol, 1.0 equiv), triphenylphosphine (0.262 g, 1.00 mmol, 0.05 equiv), bis(triphenylphosphine)palladium(II) chloride (0.702 g, 1.00 mmol, 0.05 equiv) and triethylamine (4.46 mL, 32.0 mmol, 1.6 equiv) was added anhydrous THF (120 mL, degassed with nitrogen gas for 30 min) and cooled to 0 °C. Trimethylsilylacetylene (4.21 mL, 30.0 mmol, 1.5 equiv) was added dropwise to the reaction mixture under nitrogen atmosphere followed by portion wise addition of copper(I) iodide (0.381 g, 1.99 mmol, 0.1 equiv) and stirred at room temperature for 16 h. After completion of the reaction (monitored by TLC), water (120 mL) was added to the reaction mixture and extracted with ethyl acetate (3 × 60 mL). The combined organic layers were washed with brine (60 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum to afford crude <i>N</i>-(3-((2-bromophenyl)ethynyl)cyclobutyl)-4-fluorobenzamide (4.14 g) as a brown solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 313.2 (M + 1).</div></div><div id="sec4_6_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Step 3: Synthesis of <i>N</i>-(6-Ethynylpyridin-3-yl)-4-fluorobenzamide</h5><div class="NLM_p last">To a solution of crude 4-fluoro-<i>N</i>-(6-((trimethylsilyl)ethynyl)pyridin-3-yl)benzamide (4.14 g, 13.25 mmol, 1.0 equiv) in anhydrous THF (20 mL) at 0 °C was added dropwise tetra-<i>N</i>-butylammonium fluoride (19.88 mL, 19.88 mmol, 1.5 equiv, 1.0 M solution in tetrahydrofuran) under nitrogen atmosphere, and the reaction mixture was stirred at room temperature for 2 h. After completion of the reaction (monitored by TLC), reaction mixture was added ice-cold water (20 mL) and extracted with ethyl acetate (3 × 20 mL). The combined organic layers were washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 30 to 45% of ethyl acetate in petroleum ether as an eluent to afford <i>N</i>-(6-ethynylpyridin-3-yl)-4-fluorobenzamide (0.77 g, 18% over two steps) as a pale brown solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 241.2 (M + 1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 8.92 (d, <i>J</i> = 2.5 Hz, 1H), 8.24 (dd, <i>J</i> = 8.5, 2.4 Hz, 1H), 8.10–8.01 (m, 2H), 7.59 (d, <i>J</i> = 8.5 Hz, 1H), 7.41 (tt, <i>J</i> = 9.6, 2.5 Hz, 2H), 4.27 (s, 1H).</div></div><div id="sec4_6_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step 4: Synthesis of <i>N</i>-(6-((2-Bromophenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide</h5><div class="NLM_p last">To the mixture of <i>N</i>-(6-ethynylpyridin-3-yl)-4-fluorobenzamide (0.77 g, 3.21 mmol, 1.0 equiv), 1-bromo-2-iodobenzene (1.088 g, 3.85 mmo, 1.2 equiv), triphenylphosphine (0.042 g, 0.160 mmol, 0.05 equiv), bis(triphenylphosphine)palladium(II) chloride (0.112 g, 0.160 mmol, 0.05 equiv) and triethylamine (0.67 mL, 4.81 mmol, 1.5 equiv) was added anhydrous tetrahydrofuran (15 mL, degassed with nitrogen gas for 30 min) followed by copper(I) iodide (0.061 g, 0.321 mmol, 0.1 equiv) at room temperature under nitrogen atmosphere, and the reaction mixture was stirred for 4 h. After completion of the reaction (monitored by TLC), water (10 mL) was added to the reaction mixture and extracted with ethyl acetate (3 × 10 mL). The combined organic layers were washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 25 to 30% of ethyl acetate in petroleum ether as an eluent to afford <i>N</i>-(6-((2-bromophenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (0.56 g, 44.2%) as yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 395.0, 397.0 (M, M + 2). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.77 (d, <i>J</i> = 2.6 Hz, 1H), 8.45 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 8.32 (s, 1H), 7.97 (ddd, <i>J</i> = 8.7, 5.2, 2.5 Hz, 2H), 7.67–7.58 (m, 3H), 7.31 (dd, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.26–7.16 (m, 3H).</div></div><div id="sec4_6_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Step 4a: Synthesis of <i>N</i>-(3,3-Dimethylbutyl)-3-(2-((4-methoxybenzyl)amino)-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)propanamide</h5><div class="NLM_p last">To a solution of 3-(6-bromo-2-((4-methoxybenzyl)amino)quinolin-3-yl)-<i>N</i>-(3,3-dimethylbutyl)propanamide (1 g, 2.006 mmol, 1.0 equiv) in 1,4-dioxane (20 mL), was added potassium acetate (0.591 g, 6.02 mmol, 3.0 equiv) and bis(pinacolato)diboron (0.611 g, 2.407 mmol, 1.2 equiv); subsequently, the reaction was mixture was degassed with nitrogen, and then PdCl<sub>2</sub>(dppf)–CH<sub>2</sub>Cl<sub>2</sub> (0.082 g, 0.100 μmol, 0.05 equiv) was added at room temperature. The reaction mixture was stirred at 85 °C for 16 h. After completion of the reaction (monitored by TLC), the mixture was quenched with water (20 mL) and extracted with ethyl acetate (2 × 25 mL). The organic layer was washed with brine (15 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered and concentrated in vacuo to give the crude material which was passed through a bed of silica gel for removing of inorganics to provide <i>N</i>-(3,3-dimethylbutyl)-3-(2-((4-methoxybenzyl)amino)-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)propanamide (1 g, 1.833 mmol, 91% yield), as a brown solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 546.2 (M + 1).</div></div><div id="sec4_6_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Step 5: Synthesis of <i>N</i>-(6-((2-(3-(3-((3,3-Dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide</h5><div class="NLM_p last">To the solution of crude <i>N</i>-(3,3-dimethylbutyl)-3-(2-((4-methoxybenzyl)amino)-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)quinolin-3-yl)propanamide (0.350 g, 0.642 mmol, 1.0 equiv) in 1,4-dioxane (6 mL) and water (2 mL) was added <i>N</i>-(6-((2-bromophenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (0.254 g, 0.642 mmol, 1.0 equiv) followed by potassium phosphate tribasic (0.340 g, 1.604 mmol, 2.5 equiv) at room temperature, and the solution was purged with nitrogen gas for 30 min. Bis(triphenylphosphine)palladium(ii) chloride (0.023 g, 0.032 mmol, 0.05 equiv) was added to the reaction mixture, and the mixture was heated to 110 °C for 36 h. After completion of the reaction (monitored by TLC), the reaction mixture was cooled to room temperature, water was added (10 mL), and extracted with ethyl acetate (2 × 10 mL). The combined organic layers were washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 45 to 60% of ethyl acetate in petroleum ether as an eluent to afford <i>N</i>-(6-((2-(3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (0.20 g, 70% pure, 29.7%) as a brown solid. MS (ESI negative ion) <i>m</i>/<i>z</i>: 732.0 (M – 1). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.71 (s, 1H), 8.19 (d, <i>J</i> = 8.6 Hz, 1H), 8.03 (s, 1H), 7.97 (t, <i>J</i> = 6.8 Hz, 2H), 7.82 (d, <i>J</i> = 8.6 Hz, 1H), 7.75–7.60 (m, 4H), 7.57–7.39 (m, 6H), 7.34–7.31 (m, 1H), 7.12 (d, <i>J</i> = 8.6 Hz, 3H), 6.86 (d, <i>J</i> = 8.1 Hz, 2H), 4.79 (d, <i>J</i> = 5.1 Hz, 2H), 3.78 (d, <i>J</i> = 1.8 Hz, 3H), 3.25–3.16 (m, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 1.35–1.25 (m, 4H), 0.84 (d, <i>J</i> = 1.7 Hz, 9H).</div></div><div id="sec4_6_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Step 6: Synthesis of <i>N</i>-(6-((2-(2-Amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (Compound <b>8</b>)</h5><div class="NLM_p last">To a mixture of <i>N</i>-(6-((2-(3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (0.050 g, 0.068 mmol, 1.0 equiv) and anisole (0.074 g, 0.681 mmol, 10 equiv) at 0 °C was added TFA (2 mL) under nitrogen atmosphere, and the mixture was heated to 50 °C for 2 h. After completion of the reaction (monitored by TLC), the reaction mixture was cooled to room temperature and concentrated under vacuum. Aqueous 10% NaHCO<sub>3</sub> solution (5 mL) was added to the reaction mixture (up to ∼pH 7) followed by ethyl acetate (5 mL) and stirred for 15 min. The biphasic layers were separated and the aqueous layer was extracted with ethyl acetate (3 × 5 mL). The combined organic layers were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 5 to 6% of methanol in DCM as an eluent to afford <i>N</i>-(6-((2-(2-amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (compound <b>8</b>, 0.029 g, 69.4%) as a pale yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 614.0 (M+1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.90 (d, <i>J</i> = 2.5 Hz, 1H), 8.22 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 8.10–8.03 (m, 2H), 7.92 (d, <i>J</i> = 2.1 Hz, 1H), 7.89–7.75 (m, 3H), 7.73 (d, <i>J</i> = 7.7 Hz, 1H), 7.65–7.51 (m, 3H), 7.50–7.33 (m, 4H), 6.65 (s, 2H), 3.08–2.99 (m, 2H), 2.85 (t, <i>J</i> = 7.3 Hz, 2H), 2.44 (t, <i>J</i> = 7.3 Hz, 2H), 1.28–1.22 (m, 2H), 0.81 (s, 9H); <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ −109.4.</div></div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Synthesis of Compound <b>9</b> (Continued from Synthesis of Compound <b>8</b>) (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>)</h4><div id="sec4_6_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Step 7: Synthesis of <i>N</i>-(6-(2-(3-(3-((3,3-Dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenethyl)pyridin-3-yl)-4-fluorobenzamide</h5><div class="NLM_p">To the solution of <i>N</i>-(6-((2-(3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenyl)ethynyl)pyridin-3-yl)-4-fluorobenzamide (0.050 g, 0.068 mmol, 1.0 equiv) in anhydrous ethanol (5 mL) was added 10% palladium on carbon (10 mg, 20 wt %) at room temperature, and the mixture was stirred under hydrogen pressure (15 psi) for 48 h. After completion of the reaction (monitored by TLC and HPLC), the reaction mixture was filtered through a pad of Celite, washed with ethanol (4 × 2 mL), and concentrated under vacuum to yield <i>N</i>-(6-(2-(3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenethyl)pyridin-3-yl)-4-fluorobenzamide (0.04 g, 80%) as a yellow solid. MS (ESI positive ion) <i>m</i>/<i>z</i>: 738.2 (M + 1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H), 8.74–8.70 (m, 1H), 8.06–8.02 (m, 2H), 7.97 (dd, <i>J</i> = 8.4, 2.6 Hz, 1H), 7.83 (t, <i>J</i> = 5.4 Hz, 1H), 7.70 (s, 1H), 7.67–7.51 (m, 6H), 7.40–7.34 (m, 3H), 7.32–7.13 (m, 3H), 7.00 (d, <i>J</i> = 8.4 Hz, 1H), 6.89–6.84 (m, 2H), 4.66 (d, <i>J</i> = 5.7 Hz, 2H), 3.71 (s, 3H), 3.09–3.02 (m, 2H), 2.99–2.94 (m, 2H), 2.89–2.83 (m, 4H), 1.29–1.21 (m, 4H), 0.83 (s, 9H).</div><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0010.jpeg" id="GRAPHIC-d717e2405-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_6_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Step 8: Synthesis of <i>N</i>-(6-(2-(2-Amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenethyl)pyridin-3-yl)-4-fluorobenzamide TFA (Compound <b>9</b>)</h5><div class="NLM_p last">To a mixture of <i>N</i>-(6-(2-(3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)-2-((4-methoxybenzyl)amino)quinolin-6-yl)phenethyl)pyridin-3-yl)-4-fluorobenzamide (0.045 g, 0.061 mmol, 1.0 equiv) and anisole (0.066 g, 0.610 mmol, 10 equiv) at 0 °C was added TFA (2 mL) under nitrogen atmosphere, and the mixture was heated to 50 °C for 3 h. After completion of the reaction (monitored by TLC), the reaction mixture was cooled to room temperature and concentrated under vacuum. Aqueous 10% NaHCO<sub>3</sub> solution (5 mL) was added to the reaction mixture (up to ∼ pH-7) followed by ethyl acetate (5 mL), and the solution was stirred for 15 min. The biphasic layers were separated, and the aqueous layer was extracted with ethyl acetate (3 × 5 mL). The combined organic layers were washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude compound was purified by flash column chromatography (silica gel, 230–400 mesh) using 5 to 6% of methanol in DCM as an eluent to yield the desired crude product (0.034 g, 91%), which was further purified by preparative HPLC (Chromosil-C18; mobile phase: 10 mM NH<sub>4</sub>OAc in water and acetonitrile; flow rate: 18 mL/min; peak RT: 16.1 min] and after partial concentration, the material was extracted from the aqueous layer (∼10 mL) with DCM (3 × 5 mL). The combined organic layers was washed with brine (5 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum to yield <i>N</i>-(6-(2-(2-amino-3-(3-((3,3-dimethylbutyl)amino)-3-oxopropyl)quinolin-6-yl)phenethyl)pyridin-3-yl)-4-fluorobenzamide (compound 9) as a TFA salt (0.015 g, 39.8%, white solid). MS (ESI positive ion) <i>m</i>/<i>z</i>: 618.2 (M + 1). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H), 8.71 (d, <i>J</i> = 2.6 Hz, 1H), 8.07–8.01 (m, 2H), 7.97 (dd, <i>J</i> = 8.4, 2.6 Hz, 1H), 7.83 (t, <i>J</i> = 5.5 Hz, 1H), 7.74 (s, 1H), 7.54–7.46 (m, 2H), 7.43–7.35 (m, 4H), 7.33–7.19 (m, 3H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 6.45 (s, 2H), 3.08–2.93 (m, 4H), 2.91–2.76 (m, 4H), 2.43 (t, <i>J</i> = 7.4 Hz, 2H), 1.29–1.24 (m, 2H), 0.82 (s, 9H); <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ −73.4, −108.5.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01917">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73707" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73707" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01917" class="ext-link">10.1021/acs.jmedchem.5b01917</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table of crystallographic data, SPR sensorgrams, BACE-1, BACE-2, and CatD enzyme assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Data table (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_001.pdf">jm5b01917_si_001.pdf (526.18 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_002.csv">jm5b01917_si_002.csv (0.54 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">New protein/ligand coordinates have been deposited in the PDB with IDs of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3V">5I3V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W">5I3W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3X">5I3X</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3Y">5I3Y</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1">5IE1</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01917" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John B. Jordan</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#35574754511b5f5a4751545b75545852505b1b565a58"><span class="__cf_email__" data-cfemail="4f2d3d2e2b6125203d2b2e210f2e22282a21612c2022">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas A. Whittington</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael D. Bartberger</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">E. Allen Sickmier</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kui Chen</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Cheng</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ted Judd</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery, Amgen, Inc. One Amgen Center Drive, Thousand
Oaks, California 91320, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d717e2522-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37625" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37625" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Alexander M. Long for purification of BACE-1 protein. X-ray diffraction data were collected at the Advanced Light Source, which is supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences under Contract No. DE-AC02-05CH11231. Diffraction data were also collected at Southeast Regional Collaborative Access Team (SER-CAT) beamline 22-BM at the Advanced Photon Source of Argonne National Laboratory. Use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under Contract No. W-3-109-Eng-38.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i50" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i50"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i51" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i51"> Abbreviations Used</h2><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">Fragment Based Drug Discovery</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">Fragment Based Screening</p></td></tr><tr><td class="NLM_term">CatD</td><td class="NLM_def"><p class="first last">Cathepsin-D</p></td></tr><tr><td class="NLM_term">BACE-1</td><td class="NLM_def"><p class="first last">β-Site APP Cleaving Enzyme-1</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">Ligand Efficiency</p></td></tr><tr><td class="NLM_term">HA</td><td class="NLM_def"><p class="first last">Heavy Atoms</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">High Throughput Screening</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">Nuclear Magnetic Resonance</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">Surface Plasmon Resonance</p></td></tr><tr><td class="NLM_term">ILNOE</td><td class="NLM_def"><p class="first last">Interligand Nuclear Overhauser Enhancement</p></td></tr><tr><td class="NLM_term">NOESY</td><td class="NLM_def"><p class="first last">Nuclear Overhauser Enhancement Spectroscopy</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">Saturation Transfer Difference</p></td></tr><tr><td class="NLM_term">WaterLOGSY</td><td class="NLM_def"><p class="first last">Water-Ligand Observe Gradient SpectroscopY</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">Structure Activity Relationship</p></td></tr><tr><td class="NLM_term">TBDMS-Cl</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyldimethylsilyl chloride</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0ljjiuYJAauI7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0ljjiuYJAauI7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0ljjiuYJAauI7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagerness, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadden, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarver, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B. J.</span><span> </span><span class="NLM_article-title">Fluorine-NMR experiments for high-throughput screening: theoretical aspects, practical considerations, and range of applicability</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">7696</span><span class="NLM_x">–</span> <span class="NLM_lpage">7703</span><span class="refDoi"> DOI: 10.1021/ja034646d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034646d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7696-7703&author=C.+Dalvitauthor=P.+E.+Fagernessauthor=D.+T.+Haddenauthor=R.+W.+Sarverauthor=B.+J.+Stockman&title=Fluorine-NMR+experiments+for+high-throughput+screening%3A+theoretical+aspects%2C+practical+considerations%2C+and+range+of+applicability&doi=10.1021%2Fja034646d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine-NMR experiments for high-throughput screening: Theoretical aspects, practical considerations, and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fagerness, Paul E.; Hadden, Daneen T. A.; Sarver, Ronald W.; Stockman, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7696-7703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition ligand-based NMR screening expts. have recently been introduced to overcome most of the problems assocd. with traditional ligand-based NMR screening.  Mols. with marginal soly. and high affinity for a given target can be easily identified in a high-throughput manner by screening chem. mixts. against the target in the presence of a weak- to medium-affinity ligand of known binding const.  While the original competition-based approaches utilized 1H detection, significant advantages are obtained using 19F detection.  The absence of spectral overlap permits the screening of large chem. mixts. and allows for automated anal. of the spectra, even in the presence of protonated buffers, solvents, and detergents.  The large chem. shift anisotropy of fluorine and the significant exchange contribution allow for the selection of a weak-affinity spy mol., thus resulting in a lower binding affinity threshold for the identified NMR hits.  The method, labeled FAXS (fluorine chem. shift anisotropy and exchange for screening) is rapid and requires only a limited amt. of protein and, therefore, compares favorably with the other established non-NMR techniques used in high-throughput screening.  Herein the theor. aspects of this powerful 19F-based approach are presented and discussed in detail.  The exptl. conditions together with the detection limits and binding const. measurements are investigated using human serum albumin as the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrluTqHSItDBLVg90H21EOLACvtfcHk0ljBmeWrQKTbgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D&md5=58b7a2db41871c28e06873302d4a1ecb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fja034646d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034646d%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFagerness%26aufirst%3DP.%2BE.%26aulast%3DHadden%26aufirst%3DD.%2BT.%26aulast%3DSarver%26aufirst%3DR.%2BW.%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26atitle%3DFluorine-NMR%2520experiments%2520for%2520high-throughput%2520screening%253A%2520theoretical%2520aspects%252C%2520practical%2520considerations%252C%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7696%26epage%3D7703%26doi%3D10.1021%2Fja034646d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jordan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michelsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnier, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nixey, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, W.</span><span> </span><span class="NLM_article-title">Fragment based drug discovery: practical implementation based on (1)(9)F NMR spectroscopy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">687</span><span class="refDoi"> DOI: 10.1021/jm201441k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201441k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=678-687&author=J.+B.+Jordanauthor=L.+Poppeauthor=X.+Xiaauthor=A.+C.+Chengauthor=Y.+Sunauthor=K.+Michelsenauthor=H.+Eastwoodauthor=P.+D.+Schnierauthor=T.+Nixeyauthor=W.+Zhong&title=Fragment+based+drug+discovery%3A+practical+implementation+based+on+%281%29%289%29F+NMR+spectroscopy&doi=10.1021%2Fjm201441k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm201441k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201441k%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMichelsen%26aufirst%3DK.%26aulast%3DEastwood%26aufirst%3DH.%26aulast%3DSchnier%26aufirst%3DP.%2BD.%26aulast%3DNixey%26aufirst%3DT.%26aulast%3DZhong%26aufirst%3DW.%26atitle%3DFragment%2520based%2520drug%2520discovery%253A%2520practical%2520implementation%2520based%2520on%2520%25281%2529%25289%2529F%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D678%26epage%3D687%26doi%3D10.1021%2Fjm201441k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">Design and NMR-based screening of LEF, a library of chemical fragments with different local environment of fluorine</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">12949</span><span class="NLM_x">–</span> <span class="NLM_lpage">12959</span><span class="refDoi"> DOI: 10.1021/ja905207t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja905207t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVCgt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=12949-12959&author=A.+Vulpettiauthor=U.+Hommelauthor=G.+Landrumauthor=R.+Lewisauthor=C.+Dalvit&title=Design+and+NMR-based+screening+of+LEF%2C+a+library+of+chemical+fragments+with+different+local+environment+of+fluorine&doi=10.1021%2Fja905207t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Design and NMR-Based Screening of LEF, a Library of Chemical Fragments with Different Local Environment of Fluorine</span></div><div class="casAuthors">Vulpetti, Anna; Hommel, Ulrich; Landrum, Gregory; Lewis, Richard; Dalvit, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12949-12959</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel strategy for the design of a fluorinated fragment library that takes into account the local environment of fluorine is described.  The procedure, based on a fluorine fingerprints descriptor, and the criteria used in the design, selection, and construction of the library are presented.  The library, named LEF (Local Environment of Fluorine), combined with 19F NMR ligand-based screening expts. represents an efficient and sensitive approach for the initial fragment identification within a fragment-based drug discovery project and for probing the presence of fluorophilic protein environments.  Proper setup of the method, according to described theor. simulations, allows the detection of very weak-affinity ligands and the detection of multiple ligands present within the same tested mixt., thus capturing all the potential fragments interacting with the receptor.  These NMR hits are then used in the FAXS expts. for the fragment optimization process and for the follow-up screening aimed at identifying other chem. scaffolds relevant for the binding to the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMsRpsM-vNR7Vg90H21EOLACvtfcHk0ljBmeWrQKTbgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVCgt7fM&md5=f152fedb394ae0196fb2f113bcd895d1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fja905207t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja905207t%26sid%3Dliteratum%253Aachs%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DHommel%26aufirst%3DU.%26aulast%3DLandrum%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DR.%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DDesign%2520and%2520NMR-based%2520screening%2520of%2520LEF%252C%2520a%2520library%2520of%2520chemical%2520fragments%2520with%2520different%2520local%2520environment%2520of%2520fluorine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D12949%26epage%3D12959%26doi%3D10.1021%2Fja905207t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Poppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zondlo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegley, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuanmanee, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheetham, J.</span><span> </span><span class="NLM_article-title">Discovery of ligands for Nurr1 by combined use of NMR screening with different isotopic and spin-labeling strategies</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1177/1087057106299161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1177%2F1087057106299161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=17438066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltlWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=301-311&author=L.+Poppeauthor=T.+S.+Harveyauthor=C.+Mohrauthor=J.+Zondloauthor=C.+M.+Tegleyauthor=O.+Nuanmaneeauthor=J.+Cheetham&title=Discovery+of+ligands+for+Nurr1+by+combined+use+of+NMR+screening+with+different+isotopic+and+spin-labeling+strategies&doi=10.1177%2F1087057106299161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ligands for Nurr1 by combined use of NMR screening with different isotopic and spin-labeling strategies</span></div><div class="casAuthors">Poppe, Leszek; Harvey, Timothy S.; Mohr, Christopher; Zondlo, James; Tegley, Christopher M.; Nuanmanee, Opas; Cheetham, Janet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A comprehensive approach to target screening, hit validation, and binding site detn. by NMR (NMR) spectroscopy is presented.  NMR 19F signal perturbation was used to screen a small compd. library and identify candidate ligands to the target of interest.  Ligand dissocn. consts. were measured using a pegylated form of the protein, which resulted in a 2-fold increase in the strength of the satn. transfer difference signal.  The initial small-moln. hits were further optimized by combining a residue-specific labeling strategy, to identify the specific sites of interaction with the protein, with a second site screening approach based on relaxation enhancement using a paramagnetic probe.  The advantages of this combination strategy in the identification and optimization of weak binding chem. entities early in a program are illustrated with the discovery of a low micromolar ligand (Kd = 20 μM) for Nurr1 and identification of the binding site location through residue-specific 15N isotope labeling and derivatization of Cys residues with 2-mercaptoethanol-1-13C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTXqTtwwsKT7Vg90H21EOLACvtfcHk0ljBmeWrQKTbgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltlWitbs%253D&md5=74295fe8775657b17052af532a800d60</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057106299161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106299161%26sid%3Dliteratum%253Aachs%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DHarvey%26aufirst%3DT.%2BS.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DZondlo%26aufirst%3DJ.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DNuanmanee%26aufirst%3DO.%26aulast%3DCheetham%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520ligands%2520for%2520Nurr1%2520by%2520combined%2520use%2520of%2520NMR%2520screening%2520with%2520different%2520isotopic%2520and%2520spin-labeling%2520strategies%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2007%26volume%3D12%26spage%3D301%26epage%3D311%26doi%3D10.1177%2F1087057106299161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Querfurth, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFerla, F. M.</span><span> </span><span class="NLM_article-title">Alzheimer’s disease</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">362</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1056/NEJMra0909142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1056%2FNEJMra0909142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=20107219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1aqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=329-344&author=H.+W.+Querfurthauthor=F.+M.+LaFerla&title=Alzheimer%E2%80%99s+disease&doi=10.1056%2FNEJMra0909142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease</span></div><div class="casAuthors">Querfurth, Henry W.; LaFerla, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">329-344</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The role of β-amyloid peptide in spontaneous self-aggregation in brain in Alzheimer's disease is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPeniV_Jn6orVg90H21EOLACvtfcHk0lhIHYHEXqJalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1aqt7s%253D&md5=c116f6a6b8fed9af9f5282cc4b755c70</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra0909142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra0909142%26sid%3Dliteratum%253Aachs%26aulast%3DQuerfurth%26aufirst%3DH.%2BW.%26aulast%3DLaFerla%26aufirst%3DF.%2BM.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D329%26epage%3D344%26doi%3D10.1056%2FNEJMra0909142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selkoe, D. J.</span><span> </span><span class="NLM_article-title">The amyloid hypothesis of Alzheimer’s disease: progress and problems on the road to therapeutics</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">297</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1126/science.1072994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.1072994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=12130773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Cju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=353-356&author=J.+Hardyauthor=D.+J.+Selkoe&title=The+amyloid+hypothesis+of+Alzheimer%E2%80%99s+disease%3A+progress+and+problems+on+the+road+to+therapeutics&doi=10.1126%2Fscience.1072994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The amyloid hypothesis of Alzheimer's disease: progress and problems on the road to therapeutics</span></div><div class="casAuthors">Hardy, John; Selkoe, Dennis J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5580</span>),
    <span class="NLM_cas:pages">353-356</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  It has been more than 10 yr since it was first proposed that the neurodegeneration in Alzheimer's disease (AD) may be caused by deposition of amyloid β-peptide (Aβ) in plaques in brain tissue.  According to the amyloid hypothesis, accumulation of Aβ in the brain is the primary influence driving AD pathogenesis.  The rest of the disease process, including formation of neurofibrillary tangles contg. tau protein, is proposed to result from an imbalance between Aβ prodn. and Aβ clearance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9G5omTHR6cbVg90H21EOLACvtfcHk0lhIHYHEXqJalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Cju7s%253D&md5=f7b8db1bc6f13e85f887b73f3042e86e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072994%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DJ.%26aulast%3DSelkoe%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520amyloid%2520hypothesis%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520progress%2520and%2520problems%2520on%2520the%2520road%2520to%2520therapeutics%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D353%26epage%3D356%26doi%3D10.1126%2Fscience.1072994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">De Strooper, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golde, T.</span><span> </span><span class="NLM_article-title">The secretases: enzymes with therapeutic potential in Alzheimer disease</span> <span class="citation_source-journal">Nat. Rev. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1038/nrneurol.2009.218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1038%2Fnrneurol.2009.218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=20139999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=99-107&author=B.+De+Strooperauthor=R.+Vassarauthor=T.+Golde&title=The+secretases%3A+enzymes+with+therapeutic+potential+in+Alzheimer+disease&doi=10.1038%2Fnrneurol.2009.218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The secretases: enzymes with therapeutic potential in Alzheimer disease</span></div><div class="casAuthors">De Strooper, Bart; Vassar, Robert; Golde, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-107</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The three secretases that process amyloid precursor protein are central to the generation of amyloid-β, and the accumulation of this peptide in extracellular plaques is one of the hallmarks of Alzheimer disease.  In this Review, De Strooper et al. discuss the evidence that suggests that these enzymes are potential therapeutic targets for Alzheimer disease drug treatments.  The amyloid hypothesis has yielded a series of well-validated candidate drug targets with potential for the treatment of Alzheimer disease (AD).  Three proteases that are involved in the processing of amyloid precursor protein-α-secretase, β-secretase and γ-secretase-are of particular interest as they are central to the generation and modulation of amyloid-β peptide and can be targeted by small compds. in vitro and in vivo.  Given that these proteases also fulfill other important biol. roles, inhibiting their activity will clearly be inherently assocd. with mechanism-based toxicity.  Carefully detg. a suitable therapeutic window and optimizing the selectivity of the drug treatments towards amyloid precursor protein processing might be ways of overcoming this potential complication.  Secretase inhibitors are likely to be the first small-mol. therapies aimed at AD modification that will be fully tested in the clinic.  Success or failure of these first-generation AD therapies will have enormous consequences for further drug development efforts for AD and possibly other neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBldqS_TKy7Vg90H21EOLACvtfcHk0lhIHYHEXqJalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Ogt70%253D&md5=40135a0a59da6b70e4856a81ddfa6088</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2009.218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2009.218%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BStrooper%26aufirst%3DB.%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DGolde%26aufirst%3DT.%26atitle%3DThe%2520secretases%253A%2520enzymes%2520with%2520therapeutic%2520potential%2520in%2520Alzheimer%2520disease%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2010%26volume%3D6%26spage%3D99%26epage%3D107%26doi%3D10.1038%2Fnrneurol.2009.218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu-Khan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendiaz, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplow, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amarante, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeloff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edenson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lile, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarosinski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biere, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treanor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Citron, M.</span><span> </span><span class="NLM_article-title">Beta-secretase cleavage of Alzheimer’s amyloid precursor protein by the transmembrane aspartic protease BACE</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">741</span><span class="refDoi"> DOI: 10.1126/science.286.5440.735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.286.5440.735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10531052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXntFWit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1999&pages=735-741&author=R.+Vassarauthor=B.+D.+Bennettauthor=S.+Babu-Khanauthor=S.+Kahnauthor=E.+A.+Mendiazauthor=P.+Denisauthor=D.+B.+Teplowauthor=S.+Rossauthor=P.+Amaranteauthor=R.+Loeloffauthor=Y.+Luoauthor=S.+Fisherauthor=J.+Fullerauthor=S.+Edensonauthor=J.+Lileauthor=M.+A.+Jarosinskiauthor=A.+L.+Biereauthor=E.+Curranauthor=T.+Burgessauthor=J.+C.+Louisauthor=F.+Collinsauthor=J.+Treanorauthor=G.+Rogersauthor=M.+Citron&title=Beta-secretase+cleavage+of+Alzheimer%E2%80%99s+amyloid+precursor+protein+by+the+transmembrane+aspartic+protease+BACE&doi=10.1126%2Fscience.286.5440.735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">β-Secretase cleavage of Alzheimer's amyloid precursor protein by the transmembrane aspartic protease BACE</span></div><div class="casAuthors">Vassar, Robert; Bennett, Brian D.; Babu-Khan, Safura; Kahn, Steve; Mendiaz, Elizabeth A.; Denis, Paul; Teplow, David B.; Ross, Sandra; Amarante, Patricia; Loeloff, Richard; Luo, Yi; Fisher, Seth; Fuller, Janis; Edenson, Steven; Lile, Jackson; Jarosinski, Mark A.; Biere, Anja Leona; Curran, Eileen; Burgess, Teresa; Louis, Jean-Claude; Collins, Frank; Treanor, James; Rogers, Gary; Citron, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">5440</span>),
    <span class="NLM_cas:pages">735-741</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cerebral deposition of amyloid β peptide (Aβ) is an early and crit. feature of Alzheimer's disease.  Aβ generation depends on proteolytic cleavage of the amyloid precursor protein (APP) by two unknown proteases: β-secretase and γ-secretase.  These proteases are prime therapeutic targets.  A transmembrane aspartic protease with all the known characteristics of β-secretase was cloned and characterized.  Overexpression of this protease, termed BACE (for beta-site APP-cleaving enzyme) increased the amt. of β-secretase cleavage products, and these were cleaved exactly and only at known β-secretase positions.  Antisense inhibition of endogenous BACE mRNA decreased the amt. of β-secretase cleavage products, and purified BACE protein cleaved APP-derived substrates with the same sequence specificity as β-secretase.  Finally, the expression pattern and subcellular localization of BACE were consistent with that expected for β-secretase.  Future development of BACE inhibitors may prove beneficial for the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSx3s1NFyByrVg90H21EOLACvtfcHk0lj4GGLP2XqWuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFWit74%253D&md5=3f36e8e5b5e9441c7670ccaeb0d2dd4f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.286.5440.735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.286.5440.735%26sid%3Dliteratum%253Aachs%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DBabu-Khan%26aufirst%3DS.%26aulast%3DKahn%26aufirst%3DS.%26aulast%3DMendiaz%26aufirst%3DE.%2BA.%26aulast%3DDenis%26aufirst%3DP.%26aulast%3DTeplow%26aufirst%3DD.%2BB.%26aulast%3DRoss%26aufirst%3DS.%26aulast%3DAmarante%26aufirst%3DP.%26aulast%3DLoeloff%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DFisher%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DEdenson%26aufirst%3DS.%26aulast%3DLile%26aufirst%3DJ.%26aulast%3DJarosinski%26aufirst%3DM.%2BA.%26aulast%3DBiere%26aufirst%3DA.%2BL.%26aulast%3DCurran%26aufirst%3DE.%26aulast%3DBurgess%26aufirst%3DT.%26aulast%3DLouis%26aufirst%3DJ.%2BC.%26aulast%3DCollins%26aufirst%3DF.%26aulast%3DTreanor%26aufirst%3DJ.%26aulast%3DRogers%26aufirst%3DG.%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DBeta-secretase%2520cleavage%2520of%2520Alzheimer%25E2%2580%2599s%2520amyloid%2520precursor%2520protein%2520by%2520the%2520transmembrane%2520aspartic%2520protease%2520BACE%26jtitle%3DScience%26date%3D1999%26volume%3D286%26spage%3D735%26epage%3D741%26doi%3D10.1126%2Fscience.286.5440.735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, N.</span><span> </span><span class="NLM_article-title">Application of fragment screening by X-ray crystallography to beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1116</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1021/jm0611962</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0611962" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1116-1123&author=C.+W.+Murrayauthor=O.+Callaghanauthor=G.+Chessariauthor=A.+Cleasbyauthor=M.+Congreveauthor=M.+Fredericksonauthor=M.+J.+Hartshornauthor=R.+McMenaminauthor=S.+Patelauthor=N.+Wallis&title=Application+of+fragment+screening+by+X-ray+crystallography+to+beta-secretase&doi=10.1021%2Fjm0611962"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm0611962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0611962%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWallis%26aufirst%3DN.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520by%2520X-ray%2520crystallography%2520to%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1116%26epage%3D1123%26doi%3D10.1021%2Fjm0611962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fremeau, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louie, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michelsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nixey, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sickmier, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Jean, D. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span> </span><span class="NLM_article-title">From fragment screening to in vivo efficacy: optimization of a series of 2-aminoquinolines as potent inhibitors of beta-site amyloid precursor protein cleaving enzyme 1 (BACE1)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5836</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span><span class="refDoi"> DOI: 10.1021/jm200544q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200544q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5836-5857&author=Y.+Chengauthor=T.+C.+Juddauthor=M.+D.+Bartbergerauthor=J.+Brownauthor=K.+Chenauthor=R.+T.+Fremeauauthor=D.+Hickmanauthor=S.+A.+Hitchcockauthor=B.+Jordanauthor=V.+Liauthor=P.+Lopezauthor=S.+W.+Louieauthor=Y.+Luoauthor=K.+Michelsenauthor=T.+Nixeyauthor=T.+S.+Powersauthor=C.+Rattanauthor=E.+A.+Sickmierauthor=D.+J.+St.+Jeanauthor=R.+C.+Wahlauthor=P.+H.+Wenauthor=S.+Wood&title=From+fragment+screening+to+in+vivo+efficacy%3A+optimization+of+a+series+of+2-aminoquinolines+as+potent+inhibitors+of+beta-site+amyloid+precursor+protein+cleaving+enzyme+1+%28BACE1%29&doi=10.1021%2Fjm200544q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From Fragment Screening to In Vivo Efficacy: Optimization of a Series of 2-Aminoquinolines as Potent Inhibitors of Beta-Site Amyloid Precursor Protein Cleaving Enzyme 1 (BACE1)</span></div><div class="casAuthors">Cheng, Yuan; Judd, Ted C.; Bartberger, Michael D.; Brown, James; Chen, Kui; Fremeau, Robert T.; Hickman, Dean; Hitchcock, Stephen A.; Jordan, Brad; Li, Vivian; Lopez, Patricia; Louie, Steven W.; Luo, Yi; Michelsen, Klaus; Nixey, Thomas; Powers, Timothy S.; Rattan, Claire; Sickmier, E. Allen; St. Jean, David J.; Wahl, Robert C.; Wen, Paul H.; Wood, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5836-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening of a focused fragment library, 2-aminoquinoline 1 was identified as an initial hit for BACE1.  Further SAR development was supported by X-ray structures of BACE1 cocrystd. with various ligands and mol. modeling studies to expedite the discovery of potent compds.  These strategies enabled us to integrate the C-3 side chain on 2-aminoquinoline 1 extending deep into the P2' binding pocket of BACE1 and enhancing the ligand's potency.  We were able to improve the BACE1 potency to subnanomolar range, over 106-fold more potent than the initial hit (900 μM).  Further elaboration of the phys. properties of the lead compds. to those more consistent with good blood-brain barrier permeability led to inhibitors with greatly improved cellular activity and permeability.  Compd. 59 showed an IC50 value of 11 nM on BACE1 and cellular activity of 80 nM.  This compd. was advanced into rat pharmacokinetic and pharmacodynamic studies and demonstrated significant redn. of Aβ levels in cerebrospinal fluid (CSF).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLRGyUyxO1fLVg90H21EOLACvtfcHk0lj4GGLP2XqWuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSksrk%253D&md5=fa0f2b9b7524a1c3b75e162801f54d90</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm200544q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200544q%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJudd%26aufirst%3DT.%2BC.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DHickman%26aufirst%3DD.%26aulast%3DHitchcock%26aufirst%3DS.%2BA.%26aulast%3DJordan%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DV.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLouie%26aufirst%3DS.%2BW.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMichelsen%26aufirst%3DK.%26aulast%3DNixey%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DT.%2BS.%26aulast%3DRattan%26aufirst%3DC.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DSt.%2BJean%26aufirst%3DD.%2BJ.%26aulast%3DWahl%26aufirst%3DR.%2BC.%26aulast%3DWen%26aufirst%3DP.%2BH.%26aulast%3DWood%26aufirst%3DS.%26atitle%3DFrom%2520fragment%2520screening%2520to%2520in%2520vivo%2520efficacy%253A%2520optimization%2520of%2520a%2520series%2520of%25202-aminoquinolines%2520as%2520potent%2520inhibitors%2520of%2520beta-site%2520amyloid%2520precursor%2520protein%2520cleaving%2520enzyme%25201%2520%2528BACE1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5836%26epage%3D5857%26doi%3D10.1021%2Fjm200544q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aharony, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, N.</span><span> </span><span class="NLM_article-title">Application of fragment screening by X-ray crystallography to the discovery of aminopyridines as inhibitors of beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.1021/jm061197u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061197u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1124-1132&author=M.+Congreveauthor=D.+Aharonyauthor=J.+Albertauthor=O.+Callaghanauthor=J.+Campbellauthor=R.+A.+Carrauthor=G.+Chessariauthor=S.+Cowanauthor=P.+D.+Edwardsauthor=M.+Fredericksonauthor=R.+McMenaminauthor=C.+W.+Murrayauthor=S.+Patelauthor=N.+Wallis&title=Application+of+fragment+screening+by+X-ray+crystallography+to+the+discovery+of+aminopyridines+as+inhibitors+of+beta-secretase&doi=10.1021%2Fjm061197u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm061197u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061197u%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DAharony%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DJ.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DEdwards%26aufirst%3DP.%2BD.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWallis%26aufirst%3DN.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520by%2520X-ray%2520crystallography%2520to%2520the%2520discovery%2520of%2520aminopyridines%2520as%2520inhibitors%2520of%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1124%26epage%3D1132%26doi%3D10.1021%2Fjm061197u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Stamford, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C.</span><span> </span><span class="NLM_article-title">Inhibitors of BACE for treating Alzheimer’s disease: a fragment-based drug discovery story</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2013.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2Fj.cbpa.2013.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=23683349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslaku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=320-328&author=A.+Stamfordauthor=C.+Strickland&title=Inhibitors+of+BACE+for+treating+Alzheimer%E2%80%99s+disease%3A+a+fragment-based+drug+discovery+story&doi=10.1016%2Fj.cbpa.2013.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of BACE for treating Alzheimer's disease: a fragment-based drug discovery story</span></div><div class="casAuthors">Stamford, Andrew; Strickland, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">320-328</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Several fragment-based methods have been applied to the discovery of new lead sources for inhibitors of BACE1, an important therapeutic target for Alzheimer's disease.  Among the most common fragment hits were various amidine-contg. mols. in which the amidine engaged in discrete H-bond donor-acceptor interaction with the BACE1 catalytic dyad.  Structure and medicinal chem. knowledge-based optimization with emphasis on ligand efficiency resulted in identification of a key pharmacophore comprising a non-planar cyclic amidine scaffold directly attached to a Ph group projecting into S1.  This key pharmacophore is a common feature of known clin. candidates and has dominated the recent patent literature.  A structural comparison of the non-planar cyclic amidine motif with other BACE1 pharmacophores highlights its uniqueness and distinct advantages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfLbpbjuHoU7Vg90H21EOLACvtfcHk0lh4a6c9lBg2_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslaku7w%253D&md5=f39c9a0499fe096cf4c0719f4cb41e37</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DStamford%26aufirst%3DA.%26aulast%3DStrickland%26aufirst%3DC.%26atitle%3DInhibitors%2520of%2520BACE%2520for%2520treating%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520a%2520fragment-based%2520drug%2520discovery%2520story%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D320%26epage%3D328%26doi%3D10.1016%2Fj.cbpa.2013.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaton, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamford, A. W.</span><span> </span><span class="NLM_article-title">Combining NMR and X-ray crystallography in fragment-based drug discovery: discovery of highly potent and selective BACE-1 inhibitors</span> <span class="citation_source-journal">Top. Curr. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1007/128_2011_183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1007%2F128_2011_183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2011&pages=83-114&author=D.+F.+Wyssauthor=Y.+S.+Wangauthor=H.+L.+Eatonauthor=C.+Stricklandauthor=J.+H.+Voigtauthor=Z.+Zhuauthor=A.+W.+Stamford&title=Combining+NMR+and+X-ray+crystallography+in+fragment-based+drug+discovery%3A+discovery+of+highly+potent+and+selective+BACE-1+inhibitors&doi=10.1007%2F128_2011_183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F128_2011_183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_183%26sid%3Dliteratum%253Aachs%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DEaton%26aufirst%3DH.%2BL.%26aulast%3DStrickland%26aufirst%3DC.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DStamford%26aufirst%3DA.%2BW.%26atitle%3DCombining%2520NMR%2520and%2520X-ray%2520crystallography%2520in%2520fragment-based%2520drug%2520discovery%253A%2520discovery%2520of%2520highly%2520potent%2520and%2520selective%2520BACE-1%2520inhibitors%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2011%26volume%3D317%26spage%3D83%26epage%3D114%26doi%3D10.1007%2F128_2011_183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittington, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, V. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dineen, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epstein, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graceffa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louie, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fremeau, R. T.,  Jr.</span><span> </span><span class="NLM_article-title">Structure- and property-based design of aminooxazoline xanthenes as selective, orally efficacious, and CNS penetrable BACE inhibitors for the treatment of Alzheimer’s disease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9156</span><span class="NLM_x">–</span> <span class="NLM_lpage">9169</span><span class="refDoi"> DOI: 10.1021/jm300598e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300598e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9156-9169&author=H.+Huangauthor=D.+S.+Laauthor=A.+C.+Chengauthor=D.+A.+Whittingtonauthor=V.+F.+Patelauthor=K.+Chenauthor=T.+A.+Dineenauthor=O.+Epsteinauthor=R.+Graceffaauthor=D.+Hickmanauthor=Y.+H.+Kiangauthor=S.+Louieauthor=Y.+Luoauthor=R.+C.+Wahlauthor=P.+H.+Wenauthor=S.+Woodauthor=R.+T.+Fremeau&title=Structure-+and+property-based+design+of+aminooxazoline+xanthenes+as+selective%2C+orally+efficacious%2C+and+CNS+penetrable+BACE+inhibitors+for+the+treatment+of+Alzheimer%E2%80%99s+disease&doi=10.1021%2Fjm300598e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and property-based design of aminooxazoline xanthenes as selective, orally efficacious, and CNS penetrable BACE inhibitors for the treatment of Alzheimer's disease</span></div><div class="casAuthors">Huang, Hongbing; La, Daniel S.; Cheng, Alan C.; Whittington, Douglas A.; Patel, Vinod F.; Chen, Kui; Dineen, Thomas A.; Epstein, Oleg; Graceffa, Russell; Hickman, Dean; Kiang, Y.-H.; Louie, Steven; Luo, Yi; Wahl, Robert C.; Wen, Paul H.; Wood, Stephen; Fremeau, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9156-9169</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A structure- and property-based drug design approach was employed to identify aminooxazoline xanthenes as potent and selective human β-secretase inhibitors.  These compds. exhibited good isolated enzyme, cell potency, and selectivity against the structurally related aspartyl protease cathepsin D.  Our efforts resulted in the identification of a potent, orally bioavailable CNS penetrant compd. that exhibited in vivo efficacy.  A single oral dose of compd. (I) resulted in a significant redn. of CNS Aβ40 in naive rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDEYfU4moxMbVg90H21EOLACvtfcHk0lh4a6c9lBg2_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLjJ&md5=0ecd50ec1ea82a058e499bb712c45272</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm300598e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300598e%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DLa%26aufirst%3DD.%2BS.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DPatel%26aufirst%3DV.%2BF.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DDineen%26aufirst%3DT.%2BA.%26aulast%3DEpstein%26aufirst%3DO.%26aulast%3DGraceffa%26aufirst%3DR.%26aulast%3DHickman%26aufirst%3DD.%26aulast%3DKiang%26aufirst%3DY.%2BH.%26aulast%3DLouie%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DWahl%26aufirst%3DR.%2BC.%26aulast%3DWen%26aufirst%3DP.%2BH.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26atitle%3DStructure-%2520and%2520property-based%2520design%2520of%2520aminooxazoline%2520xanthenes%2520as%2520selective%252C%2520orally%2520efficacious%252C%2520and%2520CNS%2520penetrable%2520BACE%2520inhibitors%2520for%2520the%2520treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9156%26epage%3D9169%26doi%3D10.1021%2Fjm300598e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Steinfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhardt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillebrand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraetzner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruck, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saftig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gartner, J.</span><span> </span><span class="NLM_article-title">Cathepsin D deficiency is associated with a human neurodegenerative disorder</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">988</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span><span class="refDoi"> DOI: 10.1086/504159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1086%2F504159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16685649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVyks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2006&pages=988-998&author=R.+Steinfeldauthor=K.+Reinhardtauthor=K.+Schreiberauthor=M.+Hillebrandauthor=R.+Kraetznerauthor=W.+Bruckauthor=P.+Saftigauthor=J.+Gartner&title=Cathepsin+D+deficiency+is+associated+with+a+human+neurodegenerative+disorder&doi=10.1086%2F504159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D deficiency is associated with a human neurodegenerative disorder</span></div><div class="casAuthors">Steinfeld, Robert; Reinhardt, Konstanze; Schreiber, Kathrin; Hillebrand, Merle; Kraetzner, Ralph; Brueck, Wolfgang; Saftig, Paul; Gaertner, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Cathepsin D is a ubiquitously expressed lysosomal protease that is involved in proteolytic degrdn., cell invasion, and apoptosis.  In mice and sheep, cathepsin D deficiency is known to cause a fatal neurodegenerative disease.  Here, we report a novel disorder in a child with early blindness and progressive psychomotor disability.  Two missense mutations in the CTSD gene, F229I and W383C, were identified and were found to cause markedly reduced proteolytic activity and a diminished amt. of cathepsin D in patient fibroblasts.  Expression of cathepsin D mutants in cathepsin D-/- mouse fibroblasts revealed disturbed posttranslational processing and intracellular targeting for W383C and diminished maximal enzyme velocity for F229I.  The structural effects of cathepsin D mutants were estd. by computer modeling, which suggested larger structural alterations for W383C than for F229I.  Our studies broaden the group of human neurodegenerative disorders and add new insight into the cellular functions of human cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohf0ApK0eyVLVg90H21EOLACvtfcHk0li3-bVTzJobUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVyks78%253D&md5=96a9bf74f1823244552ed50391eb5278</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1086%2F504159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F504159%26sid%3Dliteratum%253Aachs%26aulast%3DSteinfeld%26aufirst%3DR.%26aulast%3DReinhardt%26aufirst%3DK.%26aulast%3DSchreiber%26aufirst%3DK.%26aulast%3DHillebrand%26aufirst%3DM.%26aulast%3DKraetzner%26aufirst%3DR.%26aulast%3DBruck%26aufirst%3DW.%26aulast%3DSaftig%26aufirst%3DP.%26aulast%3DGartner%26aufirst%3DJ.%26atitle%3DCathepsin%2520D%2520deficiency%2520is%2520associated%2520with%2520a%2520human%2520neurodegenerative%2520disorder%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2006%26volume%3D78%26spage%3D988%26epage%3D998%26doi%3D10.1086%2F504159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Bjorklund, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oscarson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benkestock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borkakoti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenquist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span> </span><span class="NLM_article-title">Design and synthesis of potent and selective BACE-1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1458</span><span class="NLM_x">–</span> <span class="NLM_lpage">1464</span><span class="refDoi"> DOI: 10.1021/jm901168f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901168f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1458-1464&author=C.+Bjorklundauthor=S.+Oscarsonauthor=K.+Benkestockauthor=N.+Borkakotiauthor=K.+Janssonauthor=J.+Lindbergauthor=L.+Vrangauthor=A.+Hallbergauthor=A.+Rosenquistauthor=B.+Samuelsson&title=Design+and+synthesis+of+potent+and+selective+BACE-1+inhibitors&doi=10.1021%2Fjm901168f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm901168f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901168f%26sid%3Dliteratum%253Aachs%26aulast%3DBjorklund%26aufirst%3DC.%26aulast%3DOscarson%26aufirst%3DS.%26aulast%3DBenkestock%26aufirst%3DK.%26aulast%3DBorkakoti%26aufirst%3DN.%26aulast%3DJansson%26aufirst%3DK.%26aulast%3DLindberg%26aufirst%3DJ.%26aulast%3DVrang%26aufirst%3DL.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520BACE-1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1458%26epage%3D1464%26doi%3D10.1021%2Fjm901168f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Page, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jencks, W. P.</span><span> </span><span class="NLM_article-title">Entropic contributions to rate accelerations in enzymic and intramolecular reactions and the chelate effect</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">1678</span><span class="NLM_x">–</span> <span class="NLM_lpage">1683</span><span class="refDoi"> DOI: 10.1073/pnas.68.8.1678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1073%2Fpnas.68.8.1678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=5288752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaE3MXltVGhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=1971&pages=1678-1683&author=M.+I.+Pageauthor=W.+P.+Jencks&title=Entropic+contributions+to+rate+accelerations+in+enzymic+and+intramolecular+reactions+and+the+chelate+effect&doi=10.1073%2Fpnas.68.8.1678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic contributions to rate accelerations in enzymic and intramolecular reactions and the chelate effect</span></div><div class="casAuthors">Page, Michael I.; Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1678-83</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Translational and (overall) rotational motions provide the important entropic driving force for enzymic and intramol. rate accelerations and the chelate effect; internal rotations and unusually severe orientational requirements are generally of secondary importance.  The loss of translational and (overall) rotational entropy for 2 → 1 reactions in soln. is ordinarily on the order of 45 entropy units (e.u.) (std. state M, 25°); the translational entropy is much larger than 8 e.u. (corresponding to 55 M).  Low-frequency motions in products and transition states, ∼17 e.u. for cyclopentadiene dimerization, partially compensate for this loss, but effective concns. on the order of 108 M may be accounted for without the introduction of new chem. concepts or terms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3iq7fY_SBwrVg90H21EOLACvtfcHk0li3-bVTzJobUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXltVGhtLw%253D&md5=f2190b0dde6c8b5ffdf43adec063265a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.68.8.1678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.68.8.1678%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BI.%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DEntropic%2520contributions%2520to%2520rate%2520accelerations%2520in%2520enzymic%2520and%2520intramolecular%2520reactions%2520and%2520the%2520chelate%2520effect%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1971%26volume%3D68%26spage%3D1678%26epage%3D1683%26doi%3D10.1073%2Fpnas.68.8.1678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Jencks, W. P.</span><span> </span><span class="NLM_article-title">On the attribution and additivity of binding energies</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">4046</span><span class="NLM_x">–</span> <span class="NLM_lpage">4050</span><span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+attribution+and+additivity+of+binding+energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0liVDhk-vVMKFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Hajduk, P. J.; Huth, J. R.; Sun, C.</span><span> </span><span class="NLM_article-title">SAR by NMR: An Analysis of Potency Gains Realized Through Fragment-linking and Fragment-elaboration Strategies for Lead Generation</span>. In  <span class="citation_source-book">Fragment-based Approaches in Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F3527608761.ch9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=181-192&author=P.+J.+Hajduk&author=J.+R.+Huth&author=C.+Sunauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-based+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2F3527608761.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527608761.ch9%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DSAR%2520by%2520NMR%253A%2520An%2520Analysis%2520of%2520Potency%2520Gains%2520Realized%2520Through%2520Fragment-linking%2520and%2520Fragment-elaboration%2520Strategies%2520for%2520Lead%2520Generation%26btitle%3DFragment-based%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D181%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1023/A:1022446720849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1023%2FA%3A1022446720849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=12650591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=741-753&author=C.+W.+Murrayauthor=M.+L.+Verdonk&title=The+consequences+of+translational+and+rotational+entropy+lost+by+small+molecules+on+binding+to+proteins&doi=10.1023%2FA%3A1022446720849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-753</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  When a small mol. binds to a protein, it loses a significant amt. of rigid body translational and rotational entropy.  Ests. of the assocd. energy barrier vary widely in the literature yet accurate ests. are important in the interpretation of results from fragment-based drug discovery techniques.  This paper describes an anal. that allows the estn. of the rigid body entropy barrier from the increase in binding affinities that results when two fragments of known affinity and known binding mode are joined together.  The paper reviews the relatively rare no. of examples where good quality data is available.  From the anal. of this data, we est. that the barrier to binding, due to the loss of rigid-body entropy, is 15-20 kJ/mol, i.e. around 3 orders of magnitude in affinity at 298 K.  This large barrier explains why it is comparatively rare to observe multiple fragments binding to non-overlapping adjacent sites in enzymes.  The barrier is also consistent with medicinal chem. experience where small changes in the crit. binding regions of ligands are often poorly tolerated by enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KId6AyrMCLVg90H21EOLACvtfcHk0liVDhk-vVMKFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D&md5=4ad6aef74de3cb86302fe16f7e61c1c6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1022446720849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1022446720849%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DThe%2520consequences%2520of%2520translational%2520and%2520rotational%2520entropy%2520lost%2520by%2520small%2520molecules%2520on%2520binding%2520to%2520proteins%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D741%26epage%3D753%26doi%3D10.1023%2FA%3A1022446720849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Davies, T. G.; Monfort, R. L. M.; Williams, G.; Jhoti, H.</span><span> </span><span class="NLM_article-title">Pyramid: An integrated platform for fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-based Approaches in Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F3527608761.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=T.+G.+Davies&author=R.+L.+M.+Monfort&author=G.+Williams&author=H.+Jhotiauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-based+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F3527608761.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527608761.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DPyramid%253A%2520An%2520integrated%2520platform%2520for%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-based%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DOxford%2520University%2520Press%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeRose, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">London, R. E.</span><span> </span><span class="NLM_article-title">The inter-ligand Overhauser effect: a powerful new NMR approach for mapping structural relationships of macromolecular ligands</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.1023/A:1008360208627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1023%2FA%3A1008360208627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10549135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=71-76&author=D.+Liauthor=E.+F.+DeRoseauthor=R.+E.+London&title=The+inter-ligand+Overhauser+effect%3A+a+powerful+new+NMR+approach+for+mapping+structural+relationships+of+macromolecular+ligands&doi=10.1023%2FA%3A1008360208627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The inter-ligand Overhauser effect: a powerful new NMR approach for mapping structural relationships of macromolecular ligands</span></div><div class="casAuthors">Li, Dawei; DeRose, Eugene F.; London, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-76</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">NMR expts. that transfer conformational information from the bound to the uncomplexed state via exchange have been utilized for many years.  It is demonstrated here that inter-ligand NOEs ("ILOEs"), which exist in ternary complexes with enzymes or other macromol. receptors, can be transferred via exchange to pairs of uncomplexed ligands.  This approach is illustrated by studies of glycolate + NAD+ in the presence of porcine heart lactate dehydrogenase, and by glucose-6-phosphate + NADPH in the presence of L. mesenteroides glucose-6-phosphate dehydrogenase.  This strategy opens up a general methodol. for exploring the active sites of enzymes and for the development of artificial ligands which can function as inhibitors, or more generally as modifiers of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprcPANaPftSLVg90H21EOLACvtfcHk0lgpt5KsfilFjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSgtr8%253D&md5=ff105603708ffb0e2a4e1501639707e8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008360208627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008360208627%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DDeRose%26aufirst%3DE.%2BF.%26aulast%3DLondon%26aufirst%3DR.%2BE.%26atitle%3DThe%2520inter-ligand%2520Overhauser%2520effect%253A%2520a%2520powerful%2520new%2520NMR%2520approach%2520for%2520mapping%2520structural%2520relationships%2520of%2520macromolecular%2520ligands%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1999%26volume%3D15%26spage%3D71%26epage%3D76%26doi%3D10.1023%2FA%3A1008360208627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span> <span class="citation_source-book">Molecular Operating Environment</span>; <span class="NLM_publisher-name">Chemical Computing Group</span>: <span class="NLM_publisher-loc">Montreal, Canada</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment%3B+Chemical+Computing+Group%3A+Montreal%2C+Canada%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%26pub%3DChemical%2520Computing%2520Group%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span><span class="refDoi"> DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lgpt5KsfilFjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuillard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yon, J.</span><span> </span><span class="NLM_article-title">Apo and inhibitor complex structures of BACE (beta-secretase)</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2Fj.jmb.2004.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15451669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2004&pages=407-416&author=S.+Patelauthor=L.+Vuillardauthor=A.+Cleasbyauthor=C.+W.+Murrayauthor=J.+Yon&title=Apo+and+inhibitor+complex+structures+of+BACE+%28beta-secretase%29&doi=10.1016%2Fj.jmb.2004.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Apo and Inhibitor Complex Structures of BACE (β-secretase)</span></div><div class="casAuthors">Patel, Sahil; Vuillard, Laurent; Cleasby, Anne; Murray, Christopher W.; Yon, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-416</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human BACE, also known as β-secretase, shows promise as a potential therapeutic target for Alzheimer's disease.  The authors detd. the apo structure of BACE to 1.75 Å, and a structure of a hydroxyethylamine inhibitor complex derived by soaking.  These show significant active-site movements compared to previously described BACE structures.  Addnl., the structures reveal two pockets that could be targeted by structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0afg2-ZxjLVg90H21EOLACvtfcHk0lgpt5KsfilFjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVyqt7w%253D&md5=b14e17f86a4729fb8ee66d84ee6c1fab</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVuillard%26aufirst%3DL.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DYon%26aufirst%3DJ.%26atitle%3DApo%2520and%2520inhibitor%2520complex%2520structures%2520of%2520BACE%2520%2528beta-secretase%2529%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D343%26spage%3D407%26epage%3D416%26doi%3D10.1016%2Fj.jmb.2004.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lg1J9UIrK8a6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lg1J9UIrK8a6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Turner, R. T.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koelsch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castenheira, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ermolieff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span> </span><span class="NLM_article-title">Subsite specificity of memapsin 2 (beta-secretase): implications for inhibitor design</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">10001</span><span class="NLM_x">–</span> <span class="NLM_lpage">10006</span><span class="refDoi"> DOI: 10.1021/bi015546s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi015546s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=10001-10006&author=R.+T.+Turnerauthor=G.+Koelschauthor=L.+Hongauthor=P.+Castenheiraauthor=J.+Ermolieffauthor=A.+K.+Ghoshauthor=J.+Tang&title=Subsite+specificity+of+memapsin+2+%28beta-secretase%29%3A+implications+for+inhibitor+design&doi=10.1021%2Fbi015546s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi015546s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi015546s%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DR.%2BT.%26aulast%3DKoelsch%26aufirst%3DG.%26aulast%3DHong%26aufirst%3DL.%26aulast%3DCastenheira%26aufirst%3DP.%26aulast%3DErmolieff%26aufirst%3DJ.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DTang%26aufirst%3DJ.%26atitle%3DSubsite%2520specificity%2520of%2520memapsin%25202%2520%2528beta-secretase%2529%253A%2520implications%2520for%2520inhibitor%2520design%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D10001%26epage%3D10006%26doi%3D10.1021%2Fbi015546s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Haniu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendiaz, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Citron, M.</span><span> </span><span class="NLM_article-title">Characterization of Alzheimer’s beta -secretase protein BACE. A pepsin family member with unusual properties</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">21099</span><span class="NLM_x">–</span> <span class="NLM_lpage">21106</span><span class="refDoi"> DOI: 10.1074/jbc.M002095200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1074%2Fjbc.M002095200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21099-21106&author=M.+Haniuauthor=P.+Denisauthor=Y.+Youngauthor=E.+A.+Mendiazauthor=J.+Fullerauthor=J.+O.+Huiauthor=B.+D.+Bennettauthor=S.+Kahnauthor=S.+Rossauthor=T.+Burgessauthor=V.+Kattaauthor=G.+Rogersauthor=R.+Vassarauthor=M.+Citron&title=Characterization+of+Alzheimer%E2%80%99s+beta+-secretase+protein+BACE.+A+pepsin+family+member+with+unusual+properties&doi=10.1074%2Fjbc.M002095200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M002095200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M002095200%26sid%3Dliteratum%253Aachs%26aulast%3DHaniu%26aufirst%3DM.%26aulast%3DDenis%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DY.%26aulast%3DMendiaz%26aufirst%3DE.%2BA.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DHui%26aufirst%3DJ.%2BO.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DKahn%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DS.%26aulast%3DBurgess%26aufirst%3DT.%26aulast%3DKatta%26aufirst%3DV.%26aulast%3DRogers%26aufirst%3DG.%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DCharacterization%2520of%2520Alzheimer%25E2%2580%2599s%2520beta%2520-secretase%2520protein%2520BACE.%2520A%2520pepsin%2520family%2520member%2520with%2520unusual%2520properties%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21099%26epage%3D21106%26doi%3D10.1074%2Fjbc.M002095200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 36 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yong Ai, Lucia Hwang, Alexander D. MacKerell Jr., Ari Melnick, <span class="NLM_string-name hlFld-ContribAuthor">Fengtian Xue</span>. </span><span class="cited-content_cbyCitation_article-title">Progress toward B-Cell Lymphoma 6 BTB Domain Inhibitors for the Treatment of Diffuse Large B-Cell Lymphoma and Beyond. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4333-4358. <a href="https://doi.org/10.1021/acs.jmedchem.0c01686" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01686%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Btoward%252BB-Cell%252BLymphoma%252B6%252BBTB%252BDomain%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BDiffuse%252BLarge%252BB-Cell%252BLymphoma%252Band%252BBeyond%26aulast%3DAi%26aufirst%3DYong%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25092020%26date%3D12042021%26volume%3D64%26issue%3D8%26spage%3D4333%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Haoyu S. Yu, Kalyan Modugula, Osamu Ichihara, Kimberly Kramschuster, Simon Keng, Robert Abel, <span class="NLM_string-name hlFld-ContribAuthor">Lingle Wang</span>. </span><span class="cited-content_cbyCitation_article-title">General Theory of Fragment Linking in Molecular Design: Why Fragment Linking Rarely Succeeds and How to Improve Outcomes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (1)
                                     , 450-462. <a href="https://doi.org/10.1021/acs.jctc.0c01004" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c01004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c01004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c01004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DGeneral%252BTheory%252Bof%252BFragment%252BLinking%252Bin%252BMolecular%252BDesign%25253A%252BWhy%252BFragment%252BLinking%252BRarely%252BSucceeds%252Band%252BHow%252Bto%252BImprove%252BOutcomes%26aulast%3DYu%26aufirst%3DHaoyu%2BS.%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D25092020%26date%3D29122020%26volume%3D17%26issue%3D1%26spage%3D450%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alexandre Bancet, Claire Raingeval, Thierry Lomberget, Marc Le Borgne, Jean-François Guichou, <span class="NLM_string-name hlFld-ContribAuthor">Isabelle Krimm</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Linking Strategies for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (20)
                                     , 11420-11435. <a href="https://doi.org/10.1021/acs.jmedchem.0c00242" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00242</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00242%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment%252BLinking%252BStrategies%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DBancet%26aufirst%3DAlexandre%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09042020%26date%3D23062020%26date%3D15062020%26volume%3D63%26issue%3D20%26spage%3D11420%26epage%3D11435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianzhong Chen, Baohua Yin, Wei Wang, <span class="NLM_string-name hlFld-ContribAuthor">Haibo Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Effects of Disulfide Bonds on Binding of Inhibitors to β-Amyloid Cleaving Enzyme 1 Decoded by Multiple Replica Accelerated Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2020,</strong> <em>11 </em>
                                    (12)
                                     , 1811-1826. <a href="https://doi.org/10.1021/acschemneuro.0c00234" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00234</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00234%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DEffects%252Bof%252BDisulfide%252BBonds%252Bon%252BBinding%252Bof%252BInhibitors%252Bto%252B%2525CE%2525B2-Amyloid%252BCleaving%252BEnzyme%252B1%252BDecoded%252Bby%252BMultiple%252BReplica%252BAccelerated%252BMolecular%252BDynamics%252BSimulations%26aulast%3DChen%26aufirst%3DJianzhong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21042020%26date%3D27052020%26date%3D08062020%26date%3D27052020%26volume%3D11%26issue%3D12%26spage%3D1811%26epage%3D1826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianzhong Chen, Jinan Wang, Baohua Yin, Laixue Pang, Wei Wang, <span class="NLM_string-name hlFld-ContribAuthor">Weiliang Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Mechanism of Binding Selectivity of Inhibitors toward BACE1 and BACE2 Revealed by Multiple Short Molecular Dynamics Simulations and Free-Energy Predictions. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 4303-4318. <a href="https://doi.org/10.1021/acschemneuro.9b00348" title="DOI URL">https://doi.org/10.1021/acschemneuro.9b00348</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.9b00348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.9b00348%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DMolecular%252BMechanism%252Bof%252BBinding%252BSelectivity%252Bof%252BInhibitors%252Btoward%252BBACE1%252Band%252BBACE2%252BRevealed%252Bby%252BMultiple%252BShort%252BMolecular%252BDynamics%252BSimulations%252Band%252BFree-Energy%252BPredictions%26aulast%3DChen%26aufirst%3DJianzhong%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D19062019%26date%3D23092019%26date%3D02102019%26date%3D23092019%26volume%3D10%26issue%3D10%26spage%3D4303%26epage%3D4318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Claudio Dalvit, <span class="NLM_string-name hlFld-ContribAuthor">Anna Vulpetti</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-Based Fluorine NMR Screening: Principles and Applications in Drug Discovery Projects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2218-2244. <a href="https://doi.org/10.1021/acs.jmedchem.8b01210" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01210%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLigand-Based%252BFluorine%252BNMR%252BScreening%25253A%252BPrinciples%252Band%252BApplications%252Bin%252BDrug%252BDiscovery%252BProjects%26aulast%3DDalvit%26aufirst%3DClaudio%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D01082018%26date%3D29102018%26date%3D08102018%26volume%3D62%26issue%3D5%26spage%3D2218%26epage%3D2244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hangchen Hu, Zhaoqiang Chen, Xiang Xu, <span class="NLM_string-name hlFld-ContribAuthor">Yechun Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Survey of the Binding Modes of BACE1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (2)
                                     , 880-889. <a href="https://doi.org/10.1021/acschemneuro.8b00420" title="DOI URL">https://doi.org/10.1021/acschemneuro.8b00420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.8b00420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.8b00420%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DStructure-Based%252BSurvey%252Bof%252Bthe%252BBinding%252BModes%252Bof%252BBACE1%252BInhibitors%26aulast%3DHu%26aufirst%3DHangchen%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D18082018%26date%3D12122018%26date%3D27122018%26date%3D12122018%26volume%3D10%26issue%3D2%26spage%3D880%26epage%3D889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher N. Johnson, Daniel A. Erlanson, Wolfgang Jahnke, Paul N. Mortenson, <span class="NLM_string-name hlFld-ContribAuthor">David C. Rees</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2016. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (5)
                                     , 1774-1784. <a href="https://doi.org/10.1021/acs.jmedchem.7b01298" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01298%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2016%26aulast%3DJohnson%26aufirst%3DChristopher%2BN.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D31082017%26date%3D31102017%26volume%3D61%26issue%3D5%26spage%3D1774%26epage%3D1784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  Prati</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span>, <span class="hlFld-ContribAuthor ">Maria Laura  Bolognesi</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">BACE-1 Inhibitors: From Recent Single-Target Molecules to Multitarget Compounds for Alzheimer’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 619-637. <a href="https://doi.org/10.1021/acs.jmedchem.7b00393" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00393</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00393%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBACE-1%252BInhibitors%25253A%252BFrom%252BRecent%252BSingle-Target%252BMolecules%252Bto%252BMultitarget%252BCompounds%252Bfor%252BAlzheimer%2525E2%252580%252599s%252BDisease%26aulast%3DPrati%26aufirst%3DFederica%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D13032017%26date%3D08082017%26date%3D08022018%26date%3D27072017%26volume%3D61%26issue%3D3%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusuke  Kamada</span>, <span class="hlFld-ContribAuthor ">Nozomu  Sakai</span>, <span class="hlFld-ContribAuthor ">Satoshi  Sogabe</span>, <span class="hlFld-ContribAuthor ">Koh  Ida</span>, <span class="hlFld-ContribAuthor ">Hideyuki  Oki</span>, <span class="hlFld-ContribAuthor ">Kotaro  Sakamoto</span>, <span class="hlFld-ContribAuthor ">Weston  Lane</span>, <span class="hlFld-ContribAuthor ">Gyorgy  Snell</span>, <span class="hlFld-ContribAuthor ">Motoo  Iida</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>, <span class="hlFld-ContribAuthor ">Junichi  Sakamoto</span>, and <span class="hlFld-ContribAuthor ">Junji  Matsui</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (10)
                                     , 4358-4368. <a href="https://doi.org/10.1021/acs.jmedchem.7b00313" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00313</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00313%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BB-Cell%252BLymphoma%252B6%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInhibitor%252Bby%252Ba%252BBiophysics-Driven%252BFragment-Based%252BApproach%26aulast%3DKamada%26aufirst%3DYusuke%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D27022017%26date%3D11052017%26date%3D25052017%26date%3D04052017%26volume%3D60%26issue%3D10%26spage%3D4358%26epage%3D4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew K.  Urick</span>, <span class="hlFld-ContribAuthor ">Luis Pablo  Calle</span>, <span class="hlFld-ContribAuthor ">Juan F.  Espinosa</span>, <span class="hlFld-ContribAuthor ">Haitao  Hu</span>, and <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>  . </span><span class="cited-content_cbyCitation_article-title">Protein-Observed Fluorine NMR Is a Complementary Ligand Discovery Method to 1H CPMG Ligand-Observed NMR. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (11)
                                     , 3154-3164. <a href="https://doi.org/10.1021/acschembio.6b00730" title="DOI URL">https://doi.org/10.1021/acschembio.6b00730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.6b00730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.6b00730%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DProtein-Observed%252BFluorine%252BNMR%252BIs%252Ba%252BComplementary%252BLigand%252BDiscovery%252BMethod%252Bto%252B1H%252BCPMG%252BLigand-Observed%252BNMR%26aulast%3DUrick%26aufirst%3DAndrew%2BK.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D22082016%26date%3D14092016%26date%3D05102016%26date%3D18112016%26date%3D14092016%26volume%3D11%26issue%3D11%26spage%3D3154%26epage%3D3164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vanesa  Nozal</span>, <span class="hlFld-ContribAuthor ">Alfonso  García‐Rubia</span>, <span class="hlFld-ContribAuthor ">Eva P.  Cuevas</span>, <span class="hlFld-ContribAuthor ">Concepción  Pérez</span>, <span class="hlFld-ContribAuthor ">Carlota  Tosat‐Bitrián</span>, <span class="hlFld-ContribAuthor ">Fernando  Bartolomé</span>, <span class="hlFld-ContribAuthor ">Eva  Carro</span>, <span class="hlFld-ContribAuthor ">David  Ramírez</span>, <span class="hlFld-ContribAuthor ">Valle  Palomo</span>, <span class="hlFld-ContribAuthor ">Ana  Martínez</span>. </span><span class="cited-content_cbyCitation_article-title">From Kinase Inhibitors to Multitarget Ligands as Powerful Drug Leads for Alzheimer's Disease using Protein‐Templated Synthesis. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>20 </em><a href="https://doi.org/10.1002/ange.202106295" title="DOI URL">https://doi.org/10.1002/ange.202106295</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202106295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202106295%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DFrom%252BKinase%252BInhibitors%252Bto%252BMultitarget%252BLigands%252Bas%252BPowerful%252BDrug%252BLeads%252Bfor%252BAlzheimer%252527s%252BDisease%252Busing%252BProtein%2525E2%252580%252590Templated%252BSynthesis%26aulast%3DNozal%26aufirst%3DVanesa%26date%3D2021%26date%3D2021%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vanesa  Nozal</span>, <span class="hlFld-ContribAuthor ">Alfonso  García‐Rubia</span>, <span class="hlFld-ContribAuthor ">Eva P.  Cuevas</span>, <span class="hlFld-ContribAuthor ">Concepción  Pérez</span>, <span class="hlFld-ContribAuthor ">Carlota  Tosat‐Bitrián</span>, <span class="hlFld-ContribAuthor ">Fernando  Bartolomé</span>, <span class="hlFld-ContribAuthor ">Eva  Carro</span>, <span class="hlFld-ContribAuthor ">David  Ramírez</span>, <span class="hlFld-ContribAuthor ">Valle  Palomo</span>, <span class="hlFld-ContribAuthor ">Ana  Martínez</span>. </span><span class="cited-content_cbyCitation_article-title">From Kinase Inhibitors to Multitarget Ligands as Powerful Drug Leads for Alzheimer's Disease using Protein‐Templated Synthesis. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>7 </em><a href="https://doi.org/10.1002/anie.202106295" title="DOI URL">https://doi.org/10.1002/anie.202106295</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202106295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202106295%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DFrom%252BKinase%252BInhibitors%252Bto%252BMultitarget%252BLigands%252Bas%252BPowerful%252BDrug%252BLeads%252Bfor%252BAlzheimer%252527s%252BDisease%252Busing%252BProtein%2525E2%252580%252590Templated%252BSynthesis%26aulast%3DNozal%26aufirst%3DVanesa%26date%3D2021%26date%3D2021%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ben J.  Davis</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Screening by NMR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 247-270. <a href="https://doi.org/10.1007/978-1-0716-1197-5_11" title="DOI URL">https://doi.org/10.1007/978-1-0716-1197-5_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-1197-5_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-1197-5_11%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%252BScreening%252Bby%252BNMR%26aulast%3DDavis%26aufirst%3DBen%2BJ.%26date%3D2021%26date%3D2021%26spage%3D247%26epage%3D270%26pub%3DSpringer%2520US%26atitle%3DProtein-Ligand%252BInteractions%26aulast%3DDaviter%26aufirst%3DTina%26date%3D2021%26volume%3D2263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline R.  Buchholz</span>, <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">19
              F NMR viewed through two different lenses: ligand-observed and protein-observed
              19
              F NMR applications for fragment-based drug discovery. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>68 </em><a href="https://doi.org/10.1039/D1CB00085C" title="DOI URL">https://doi.org/10.1039/D1CB00085C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00085C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00085C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3D19%252BF%252BNMR%252Bviewed%252Bthrough%252Btwo%252Bdifferent%252Blenses%25253A%252Bligand-observed%252Band%252Bprotein-observed%252B19%252BF%252BNMR%252Bapplications%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%26aulast%3DBuchholz%26aufirst%3DCaroline%2BR.%26date%3D2021%26date%3D2021%26volume%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nikolaj S.  Troelsen</span>, <span class="hlFld-ContribAuthor ">Mads H.  Clausen</span>. </span><span class="cited-content_cbyCitation_article-title">Library Design Strategies To Accelerate Fragment‐Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (50)
                                     , 11391-11403. <a href="https://doi.org/10.1002/chem.202000584" title="DOI URL">https://doi.org/10.1002/chem.202000584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202000584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202000584%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DLibrary%252BDesign%252BStrategies%252BTo%252BAccelerate%252BFragment%2525E2%252580%252590Based%252BDrug%252BDiscovery%26aulast%3DTroelsen%26aufirst%3DNikolaj%2BS.%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D50%26spage%3D11391%26epage%3D11403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muslum  Yildiz</span>. </span><span class="cited-content_cbyCitation_article-title">Docking studies suggest the important role of interactions among the catalytic dyad and inhibitors for designing Bace1 specific inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2020,</strong> <em>1208 </em>, 127893. <a href="https://doi.org/10.1016/j.molstruc.2020.127893" title="DOI URL">https://doi.org/10.1016/j.molstruc.2020.127893</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2020.127893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2020.127893%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DDocking%252Bstudies%252Bsuggest%252Bthe%252Bimportant%252Brole%252Bof%252Binteractions%252Bamong%252Bthe%252Bcatalytic%252Bdyad%252Band%252Binhibitors%252Bfor%252Bdesigning%252BBace1%252Bspecific%252Binhibitors%26aulast%3DYildiz%26aufirst%3DMuslum%26date%3D2020%26volume%3D1208%26spage%3D127893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria  Kadukova</span>, <span class="hlFld-ContribAuthor ">Vladimir  Chupin</span>, <span class="hlFld-ContribAuthor ">Sergei  Grudinin</span>. </span><span class="cited-content_cbyCitation_article-title">Docking rigid macrocycles using Convex-PL, AutoDock Vina, and RDKit in the D3R Grand Challenge 4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (2)
                                     , 191-200. <a href="https://doi.org/10.1007/s10822-019-00263-3" title="DOI URL">https://doi.org/10.1007/s10822-019-00263-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00263-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00263-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDocking%252Brigid%252Bmacrocycles%252Busing%252BConvex-PL%25252C%252BAutoDock%252BVina%25252C%252Band%252BRDKit%252Bin%252Bthe%252BD3R%252BGrand%252BChallenge%252B4%26aulast%3DKadukova%26aufirst%3DMaria%26date%3D2020%26date%3D2019%26volume%3D34%26issue%3D2%26spage%3D191%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayang  Xie</span>, <span class="hlFld-ContribAuthor ">Ruirui  Liang</span>, <span class="hlFld-ContribAuthor ">Yajiang  Wang</span>, <span class="hlFld-ContribAuthor ">Junyi  Huang</span>, <span class="hlFld-ContribAuthor ">Xin  Cao</span>, <span class="hlFld-ContribAuthor ">Bing  Niu</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in Target Drug Molecules for Alzheimer's Disease. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (1)
                                     , 4-36. <a href="https://doi.org/10.2174/1568026619666191203113745" title="DOI URL">https://doi.org/10.2174/1568026619666191203113745</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666191203113745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666191203113745%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Bin%252BTarget%252BDrug%252BMolecules%252Bfor%252BAlzheimer%252527s%252BDisease%26aulast%3DXie%26aufirst%3DJiayang%26date%3D2020%26volume%3D20%26issue%3D1%26spage%3D4%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  Blaikie</span>, <span class="hlFld-ContribAuthor ">Graeme  Kay</span>, <span class="hlFld-ContribAuthor ">Paul  Kong Thoo Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Current and emerging therapeutic targets of alzheimer's disease for the design of multi-target directed ligands. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (12)
                                     , 2052-2072. <a href="https://doi.org/10.1039/C9MD00337A" title="DOI URL">https://doi.org/10.1039/C9MD00337A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00337A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00337A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DCurrent%252Band%252Bemerging%252Btherapeutic%252Btargets%252Bof%252Balzheimer%252527s%252Bdisease%252Bfor%252Bthe%252Bdesign%252Bof%252Bmulti-target%252Bdirected%252Bligands%26aulast%3DBlaikie%26aufirst%3DLaura%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D12%26spage%3D2052%26epage%3D2072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (1)
                                     , 9-15. <a href="https://doi.org/10.1016/j.chembiol.2018.10.001" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DFragment-Based%252BDrug%252BDiscovery%25253A%252BAdvancing%252BFragments%252Bin%252Bthe%252BAbsence%252Bof%252BCrystal%252BStructures%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D9%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Judite R. M.  Coimbra</span>, <span class="hlFld-ContribAuthor ">Daniela F. F.  Marques</span>, <span class="hlFld-ContribAuthor ">Salete J.  Baptista</span>, <span class="hlFld-ContribAuthor ">Cláudia M. F.  Pereira</span>, <span class="hlFld-ContribAuthor ">Paula I.  Moreira</span>, <span class="hlFld-ContribAuthor ">Teresa C. P.  Dinis</span>, <span class="hlFld-ContribAuthor ">Armanda E.  Santos</span>, <span class="hlFld-ContribAuthor ">Jorge A. R.  Salvador</span>. </span><span class="cited-content_cbyCitation_article-title">Highlights in BACE1 Inhibitors for Alzheimer's Disease Treatment. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00178" title="DOI URL">https://doi.org/10.3389/fchem.2018.00178</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00178%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DHighlights%252Bin%252BBACE1%252BInhibitors%252Bfor%252BAlzheimer%252527s%252BDisease%252BTreatment%26aulast%3DCoimbra%26aufirst%3DJudite%2BR.%2BM.%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satoru  Nagatoishi</span>, <span class="hlFld-ContribAuthor ">Sou  Yamaguchi</span>, <span class="hlFld-ContribAuthor ">Etsuko  Katoh</span>, <span class="hlFld-ContribAuthor ">Keita  Kajita</span>, <span class="hlFld-ContribAuthor ">Takane  Yokotagawa</span>, <span class="hlFld-ContribAuthor ">Satoru  Kanai</span>, <span class="hlFld-ContribAuthor ">Toshio  Furuya</span>, <span class="hlFld-ContribAuthor ">Kouhei  Tsumoto</span>. </span><span class="cited-content_cbyCitation_article-title">A combination of 19F NMR and surface plasmon resonance for site-specific hit selection and validation of fragment molecules that bind to the ATP-binding site of a kinase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (8)
                                     , 1929-1938. <a href="https://doi.org/10.1016/j.bmc.2018.02.041" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.02.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.02.041%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DA%252Bcombination%252Bof%252B19F%252BNMR%252Band%252Bsurface%252Bplasmon%252Bresonance%252Bfor%252Bsite-specific%252Bhit%252Bselection%252Band%252Bvalidation%252Bof%252Bfragment%252Bmolecules%252Bthat%252Bbind%252Bto%252Bthe%252BATP-binding%252Bsite%252Bof%252Ba%252Bkinase%26aulast%3DNagatoishi%26aufirst%3DSatoru%26date%3D2018%26volume%3D26%26issue%3D8%26spage%3D1929%26epage%3D1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meenakshi  Singh</span>, <span class="hlFld-ContribAuthor ">Benjamin  Tam</span>, <span class="hlFld-ContribAuthor ">Barak  Akabayov</span>. </span><span class="cited-content_cbyCitation_article-title">NMR-Fragment Based Virtual Screening: A Brief Overview. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (2)
                                     , 233. <a href="https://doi.org/10.3390/molecules23020233" title="DOI URL">https://doi.org/10.3390/molecules23020233</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23020233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23020233%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNMR-Fragment%252BBased%252BVirtual%252BScreening%25253A%252BA%252BBrief%252BOverview%26aulast%3DSingh%26aufirst%3DMeenakshi%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D2%26spage%3D233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergi  Gómez-Ganau</span>, <span class="hlFld-ContribAuthor ">Jesús Vicente  de Julián-Ortiz</span>, <span class="hlFld-ContribAuthor ">Rafael  Gozalbes</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Computational Approaches for Designing Potential Anti-Alzheimer’s Agents. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 25-59. <a href="https://doi.org/10.1007/978-1-4939-7404-7_2" title="DOI URL">https://doi.org/10.1007/978-1-4939-7404-7_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-7404-7_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-7404-7_2%26sid%3Dliteratum%253Aachs%26atitle%3DRecent%252BAdvances%252Bin%252BComputational%252BApproaches%252Bfor%252BDesigning%252BPotential%252BAnti-Alzheimer%2525E2%252580%252599s%252BAgents%26aulast%3DG%25C3%25B3mez-Ganau%26aufirst%3DSergi%26date%3D2018%26date%3D2017%26spage%3D25%26epage%3D59%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BModeling%252Bof%252BDrugs%252BAgainst%252BAlzheimer%2525E2%252580%252599s%252BDisease%26aulast%3DRoy%26aufirst%3DKunal%26date%3D2018%26volume%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amol D.  Sonawane</span>, <span class="hlFld-ContribAuthor ">Dinesh R.  Garud</span>, <span class="hlFld-ContribAuthor ">Taro  Udagawa</span>, <span class="hlFld-ContribAuthor ">Mamoru  Koketsu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of thieno[2,3-
              b
              ]quinoline and selenopheno[2,3-
              b
              ]quinoline derivatives
              via
              iodocyclization reaction and a DFT mechanistic study. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2018,</strong> <em>16 </em>
                                    (2)
                                     , 245-255. <a href="https://doi.org/10.1039/C7OB02523H" title="DOI URL">https://doi.org/10.1039/C7OB02523H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB02523H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB02523H%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DSynthesis%252Bof%252Bthieno%25255B2%25252C3-%252Bb%252B%25255Dquinoline%252Band%252Bselenopheno%25255B2%25252C3-%252Bb%252B%25255Dquinoline%252Bderivatives%252Bvia%252Biodocyclization%252Breaction%252Band%252Ba%252BDFT%252Bmechanistic%252Bstudy%26aulast%3DSonawane%26aufirst%3DAmol%2BD.%26date%3D2018%26date%3D2018%26volume%3D16%26issue%3D2%26spage%3D245%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Toshihiko  Sugiki</span>, <span class="hlFld-ContribAuthor ">Kyoko  Furuita</span>, <span class="hlFld-ContribAuthor ">Toshimichi  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Chojiro  Kojima</span>. </span><span class="cited-content_cbyCitation_article-title">Current NMR Techniques for Structure-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (1)
                                     , 148. <a href="https://doi.org/10.3390/molecules23010148" title="DOI URL">https://doi.org/10.3390/molecules23010148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23010148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23010148%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCurrent%252BNMR%252BTechniques%252Bfor%252BStructure-Based%252BDrug%252BDiscovery%26aulast%3DSugiki%26aufirst%3DToshihiko%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D1%26spage%3D148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudio  Dalvit</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">19
              F NMR isotropic chemical shift for efficient screening of fluorinated fragments which are racemates and/or display multiple conformers. </span><span class="cited-content_cbyCitation_journal-name">Magnetic Resonance in Chemistry</span><span> <strong>2017,</strong> <em>55 </em>
                                    (12)
                                     , 1091-1095. <a href="https://doi.org/10.1002/mrc.4640" title="DOI URL">https://doi.org/10.1002/mrc.4640</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mrc.4640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmrc.4640%26sid%3Dliteratum%253Aachs%26jtitle%3DMagnetic%2520Resonance%2520in%2520Chemistry%26atitle%3D19%252BF%252BNMR%252Bisotropic%252Bchemical%252Bshift%252Bfor%252Befficient%252Bscreening%252Bof%252Bfluorinated%252Bfragments%252Bwhich%252Bare%252Bracemates%252Band%25252For%252Bdisplay%252Bmultiple%252Bconformers%26aulast%3DDalvit%26aufirst%3DClaudio%26date%3D2017%26date%3D2017%26volume%3D55%26issue%3D12%26spage%3D1091%26epage%3D1095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Bigatti</span>, <span class="hlFld-ContribAuthor ">Alberto  Dal Corso</span>, <span class="hlFld-ContribAuthor ">Sara  Vanetti</span>, <span class="hlFld-ContribAuthor ">Samuele  Cazzamalli</span>, <span class="hlFld-ContribAuthor ">Ulrike  Rieder</span>, <span class="hlFld-ContribAuthor ">Jörg  Scheuermann</span>, <span class="hlFld-ContribAuthor ">Dario  Neri</span>, <span class="hlFld-ContribAuthor ">Filippo  Sladojevich</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a Central Scaffold on the Binding Affinity of Fragment Pairs Isolated from DNA-Encoded Self-Assembling Chemical Libraries. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (21)
                                     , 1748-1752. <a href="https://doi.org/10.1002/cmdc.201700569" title="DOI URL">https://doi.org/10.1002/cmdc.201700569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700569%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DImpact%252Bof%252Ba%252BCentral%252BScaffold%252Bon%252Bthe%252BBinding%252BAffinity%252Bof%252BFragment%252BPairs%252BIsolated%252Bfrom%252BDNA-Encoded%252BSelf-Assembling%252BChemical%252BLibraries%26aulast%3DBigatti%26aufirst%3DMartina%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D21%26spage%3D1748%26epage%3D1752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">György M  Keserű </span>, <span class="hlFld-ContribAuthor ">Michael M  Hann</span>. </span><span class="cited-content_cbyCitation_article-title">What is the future for fragment-based drug discovery?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (13)
                                     , 1457-1460. <a href="https://doi.org/10.4155/fmc-2017-0106" title="DOI URL">https://doi.org/10.4155/fmc-2017-0106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0106%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DWhat%252Bis%252Bthe%252Bfuture%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%25253F%26aulast%3DKeser%25C5%25B1%25E2%2580%258A%26aufirst%3DGy%25C3%25B6rgy%2BM%26date%3D2017%26volume%3D9%26issue%3D13%26spage%3D1457%26epage%3D1460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel D.  Cohen</span>, <span class="hlFld-ContribAuthor ">Gary J.  Pielak</span>. </span><span class="cited-content_cbyCitation_article-title">A cell is more than the sum of its (dilute) parts: A brief history of quinary structure. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2017,</strong> <em>26 </em>
                                    (3)
                                     , 403-413. <a href="https://doi.org/10.1002/pro.3092" title="DOI URL">https://doi.org/10.1002/pro.3092</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3092%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DA%252Bcell%252Bis%252Bmore%252Bthan%252Bthe%252Bsum%252Bof%252Bits%252B%252528dilute%252529%252Bparts%25253A%252BA%252Bbrief%252Bhistory%252Bof%252Bquinary%252Bstructure%26aulast%3DCohen%26aufirst%3DRachel%2BD.%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D3%26spage%3D403%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudio  Dalvit</span>, <span class="hlFld-ContribAuthor ">Martial  Piotto</span>. </span><span class="cited-content_cbyCitation_article-title">19
              F NMR transverse and longitudinal relaxation filter experiments for screening: a theoretical and experimental analysis. </span><span class="cited-content_cbyCitation_journal-name">Magnetic Resonance in Chemistry</span><span> <strong>2017,</strong> <em>55 </em>
                                    (2)
                                     , 106-114. <a href="https://doi.org/10.1002/mrc.4500" title="DOI URL">https://doi.org/10.1002/mrc.4500</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mrc.4500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmrc.4500%26sid%3Dliteratum%253Aachs%26jtitle%3DMagnetic%2520Resonance%2520in%2520Chemistry%26atitle%3D19%252BF%252BNMR%252Btransverse%252Band%252Blongitudinal%252Brelaxation%252Bfilter%252Bexperiments%252Bfor%252Bscreening%25253A%252Ba%252Btheoretical%252Band%252Bexperimental%252Banalysis%26aulast%3DDalvit%26aufirst%3DClaudio%26date%3D2017%26date%3D2016%26volume%3D55%26issue%3D2%26spage%3D106%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Hall</span>, <span class="hlFld-ContribAuthor ">H.J.M.  Gijsen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting β-Secretase (BACE) for the Treatment of Alzheimer’s Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 326-383. <a href="https://doi.org/10.1016/B978-0-12-409547-2.13809-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.13809-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.13809-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.13809-0%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252B%2525CE%2525B2-Secretase%252B%252528BACE%252529%252Bfor%252Bthe%252BTreatment%252Bof%252BAlzheimer%2525E2%252580%252599s%252BDisease%26aulast%3DHall%26aufirst%3DA.%26date%3D2017%26spage%3D326%26epage%3D383%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017%26volume%3D349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Roughley</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 371-439. <a href="https://doi.org/10.1016/bs.armc.2017.07.002" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.07.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.07.002%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BLead%252BDiscovery%26aulast%3DDavis%26aufirst%3DBen%2BJ.%26date%3D2017%26spage%3D371%26epage%3D439%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan D.  Low</span>, <span class="hlFld-ContribAuthor ">Michael D.  Bartberger</span>, <span class="hlFld-ContribAuthor ">Kui  Chen</span>, <span class="hlFld-ContribAuthor ">Yuan  Cheng</span>, <span class="hlFld-ContribAuthor ">Mark R.  Fielden</span>, <span class="hlFld-ContribAuthor ">Vijay  Gore</span>, <span class="hlFld-ContribAuthor ">Dean  Hickman</span>, <span class="hlFld-ContribAuthor ">Qingyian  Liu</span>, <span class="hlFld-ContribAuthor ">E.  Allen Sickmier</span>, <span class="hlFld-ContribAuthor ">Hugo M.  Vargas</span>, <span class="hlFld-ContribAuthor ">Jonathan  Werner</span>, <span class="hlFld-ContribAuthor ">Ryan D.  White</span>, <span class="hlFld-ContribAuthor ">Douglas A.  Whittington</span>, <span class="hlFld-ContribAuthor ">Stephen  Wood</span>, <span class="hlFld-ContribAuthor ">Ana E.  Minatti</span>. </span><span class="cited-content_cbyCitation_article-title">Development of 2-aminooxazoline 3-azaxanthene β-amyloid cleaving enzyme (BACE) inhibitors with improved selectivity against Cathepsin D. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (6)
                                     , 1196-1206. <a href="https://doi.org/10.1039/C7MD00106A" title="DOI URL">https://doi.org/10.1039/C7MD00106A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00106A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00106A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDevelopment%252Bof%252B2-aminooxazoline%252B3-azaxanthene%252B%2525CE%2525B2-amyloid%252Bcleaving%252Benzyme%252B%252528BACE%252529%252Binhibitors%252Bwith%252Bimproved%252Bselectivity%252Bagainst%252BCathepsin%252BD%26aulast%3DLow%26aufirst%3DJonathan%2BD.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D6%26spage%3D1196%26epage%3D1206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raymond  Norton</span>, <span class="hlFld-ContribAuthor ">Eleanor  Leung</span>, <span class="hlFld-ContribAuthor ">Indu  Chandrashekaran</span>, <span class="hlFld-ContribAuthor ">Christopher  MacRaild</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of 19F-NMR in Fragment-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2016,</strong> <em>21 </em>
                                    (7)
                                     , 860. <a href="https://doi.org/10.3390/molecules21070860" title="DOI URL">https://doi.org/10.3390/molecules21070860</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules21070860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules21070860%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DApplications%252Bof%252B19F-NMR%252Bin%252BFragment-Based%252BDrug%252BDiscovery%26aulast%3DNorton%26aufirst%3DRaymond%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D7%26spage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0016.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) X-ray crystal structure of BACE-1 in complex with the blocking compound <b>1</b> (PDB ID 5I3 V) used in the study. Notice that the binding site of the protein is largely occupied while leaving S3 and S3<sub>subpocket (sp)</sub> accessible to fragments. (B) X-ray crystal structure of BACE-1 in complex with competitor compound <b>2</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W">5I3W</a>) that occupies the S3<sub>sp</sub>.(C) X-ray crystal structure of BACE-1 in complex with compound <b>5</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1">5IE1</a>) to which the <sup>19</sup>F fragment (compound <b>3</b>) was linked. Notice that the toluyl methyl group in compound <b>5</b> points directly toward the S3 pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0017.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <sup>19</sup>F NMR data of the best hit (compound <b>3</b>) from the fragment screen. (A) NMR spectra of the free ligand (black) and the fragment in the presence of BACE-1(red) and in the presence of CatD (green). No binding to either protein is observed. (B) NMR spectra of the free fragment (black) and the fragment in the presence of BACE-1 and CatD along with saturating concentrations of compound <b>1</b> (red and green, respectively). Binding of the fragment to BACE-1 is exhibited by broadening of the NMR signal, demonstrating that the fragment only binds in the presence of the blocking compound. The signal exhibits no change with CatD, indicating a lack of binding. (C) NMR spectra of the free fragment (black) and the fragment in the presence of BACE-1 and CatD, the blocking compound, and 2 μM of the competitor ligand (compound <b>2</b>, red and green, respectively). The sharpening of the NMR signal indicates that the newly added ligand competes out the binding fragment, thus suggesting that the fragment likely binds to the S3 pocket of BACE-1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SPR sensorgrams of the lead <sup>19</sup>F fragment hit (compound <b>3</b>) binding to BACE-1 in the (A) absence of blocking compound (no binding observed) and (B) in the presence of saturating concentrations of the blocking compound (compound <b>1</b>, 5 μM). <i>K</i><sub>D</sub> values were obtained using steady-state analysis and fitting to a global <i>R</i><sub>max</sub> determined using a control compound. Sensorgrams shown are the same experiments run in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Transfer NOE experiments using compound <b>3</b> and compound <b>4</b>. NOEs are present between the number 11 proton of compound <b>4</b> and the number 1 proton (red circle) as well as the number 2 and 3 protons of compound <b>3</b> (black circle). Peaks present in the mixture spectrum (black) that are not present in the other two spectra of individual compounds (red) are inter-ligand NOEs between compound <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed fragment-linked compounds derived from molecular modeling and ILNOE NMR experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SPR binding experiments to BACE-1 of (A) compound <b>6</b>, (B) compound <b>7</b>, (C) compound <b>8</b>, and (D) compound <b>9</b>. Reported <i>K</i><sub>D</sub> values are the average of two experiments, and the values in parentheses denote the fold increase in affinity over the parent molecule (compound <b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Co-crystal structures of small-molecule ligands in complex with BACE-1. (A) Compound <b>6</b> and (B) compound <b>9</b>. Notice that the fluorphenyl group of compound <b>9</b> extends deep into the S3<sub>subpocket</sub>, providing additional potency against BACE-1 and additional selectivity against CatD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0006.jpeg" id="rightTab-GRAPHIC-d717e1084-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0007.jpeg" id="rightTab-GRAPHIC-d717e1350-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0008.jpeg" id="rightTab-GRAPHIC-d717e1709-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0009.jpeg" id="rightTab-GRAPHIC-d717e2049-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/medium/jm-2015-019176_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0010.jpeg" id="rightTab-GRAPHIC-d717e2405-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-8/acs.jmedchem.5b01917/20160422/images/large/jm-2015-019176_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01917&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lg06Q4Zcbcaxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0lgurpObf4M8lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lgurpObf4M8lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagerness, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadden, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarver, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B. J.</span><span> </span><span class="NLM_article-title">Fluorine-NMR experiments for high-throughput screening: theoretical aspects, practical considerations, and range of applicability</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">7696</span><span class="NLM_x">–</span> <span class="NLM_lpage">7703</span><span class="refDoi"> DOI: 10.1021/ja034646d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034646d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7696-7703&author=C.+Dalvitauthor=P.+E.+Fagernessauthor=D.+T.+Haddenauthor=R.+W.+Sarverauthor=B.+J.+Stockman&title=Fluorine-NMR+experiments+for+high-throughput+screening%3A+theoretical+aspects%2C+practical+considerations%2C+and+range+of+applicability&doi=10.1021%2Fja034646d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine-NMR experiments for high-throughput screening: Theoretical aspects, practical considerations, and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fagerness, Paul E.; Hadden, Daneen T. A.; Sarver, Ronald W.; Stockman, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7696-7703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition ligand-based NMR screening expts. have recently been introduced to overcome most of the problems assocd. with traditional ligand-based NMR screening.  Mols. with marginal soly. and high affinity for a given target can be easily identified in a high-throughput manner by screening chem. mixts. against the target in the presence of a weak- to medium-affinity ligand of known binding const.  While the original competition-based approaches utilized 1H detection, significant advantages are obtained using 19F detection.  The absence of spectral overlap permits the screening of large chem. mixts. and allows for automated anal. of the spectra, even in the presence of protonated buffers, solvents, and detergents.  The large chem. shift anisotropy of fluorine and the significant exchange contribution allow for the selection of a weak-affinity spy mol., thus resulting in a lower binding affinity threshold for the identified NMR hits.  The method, labeled FAXS (fluorine chem. shift anisotropy and exchange for screening) is rapid and requires only a limited amt. of protein and, therefore, compares favorably with the other established non-NMR techniques used in high-throughput screening.  Herein the theor. aspects of this powerful 19F-based approach are presented and discussed in detail.  The exptl. conditions together with the detection limits and binding const. measurements are investigated using human serum albumin as the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrluTqHSItDBLVg90H21EOLACvtfcHk0lgurpObf4M8lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D&md5=58b7a2db41871c28e06873302d4a1ecb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fja034646d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034646d%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFagerness%26aufirst%3DP.%2BE.%26aulast%3DHadden%26aufirst%3DD.%2BT.%26aulast%3DSarver%26aufirst%3DR.%2BW.%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26atitle%3DFluorine-NMR%2520experiments%2520for%2520high-throughput%2520screening%253A%2520theoretical%2520aspects%252C%2520practical%2520considerations%252C%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7696%26epage%3D7703%26doi%3D10.1021%2Fja034646d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jordan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michelsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnier, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nixey, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, W.</span><span> </span><span class="NLM_article-title">Fragment based drug discovery: practical implementation based on (1)(9)F NMR spectroscopy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">687</span><span class="refDoi"> DOI: 10.1021/jm201441k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201441k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=678-687&author=J.+B.+Jordanauthor=L.+Poppeauthor=X.+Xiaauthor=A.+C.+Chengauthor=Y.+Sunauthor=K.+Michelsenauthor=H.+Eastwoodauthor=P.+D.+Schnierauthor=T.+Nixeyauthor=W.+Zhong&title=Fragment+based+drug+discovery%3A+practical+implementation+based+on+%281%29%289%29F+NMR+spectroscopy&doi=10.1021%2Fjm201441k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm201441k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201441k%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMichelsen%26aufirst%3DK.%26aulast%3DEastwood%26aufirst%3DH.%26aulast%3DSchnier%26aufirst%3DP.%2BD.%26aulast%3DNixey%26aufirst%3DT.%26aulast%3DZhong%26aufirst%3DW.%26atitle%3DFragment%2520based%2520drug%2520discovery%253A%2520practical%2520implementation%2520based%2520on%2520%25281%2529%25289%2529F%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D678%26epage%3D687%26doi%3D10.1021%2Fjm201441k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">Design and NMR-based screening of LEF, a library of chemical fragments with different local environment of fluorine</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">12949</span><span class="NLM_x">–</span> <span class="NLM_lpage">12959</span><span class="refDoi"> DOI: 10.1021/ja905207t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja905207t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVCgt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=12949-12959&author=A.+Vulpettiauthor=U.+Hommelauthor=G.+Landrumauthor=R.+Lewisauthor=C.+Dalvit&title=Design+and+NMR-based+screening+of+LEF%2C+a+library+of+chemical+fragments+with+different+local+environment+of+fluorine&doi=10.1021%2Fja905207t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Design and NMR-Based Screening of LEF, a Library of Chemical Fragments with Different Local Environment of Fluorine</span></div><div class="casAuthors">Vulpetti, Anna; Hommel, Ulrich; Landrum, Gregory; Lewis, Richard; Dalvit, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12949-12959</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel strategy for the design of a fluorinated fragment library that takes into account the local environment of fluorine is described.  The procedure, based on a fluorine fingerprints descriptor, and the criteria used in the design, selection, and construction of the library are presented.  The library, named LEF (Local Environment of Fluorine), combined with 19F NMR ligand-based screening expts. represents an efficient and sensitive approach for the initial fragment identification within a fragment-based drug discovery project and for probing the presence of fluorophilic protein environments.  Proper setup of the method, according to described theor. simulations, allows the detection of very weak-affinity ligands and the detection of multiple ligands present within the same tested mixt., thus capturing all the potential fragments interacting with the receptor.  These NMR hits are then used in the FAXS expts. for the fragment optimization process and for the follow-up screening aimed at identifying other chem. scaffolds relevant for the binding to the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMsRpsM-vNR7Vg90H21EOLACvtfcHk0lgQZZszKK7Ycw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVCgt7fM&md5=f152fedb394ae0196fb2f113bcd895d1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fja905207t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja905207t%26sid%3Dliteratum%253Aachs%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DHommel%26aufirst%3DU.%26aulast%3DLandrum%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DR.%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DDesign%2520and%2520NMR-based%2520screening%2520of%2520LEF%252C%2520a%2520library%2520of%2520chemical%2520fragments%2520with%2520different%2520local%2520environment%2520of%2520fluorine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D12949%26epage%3D12959%26doi%3D10.1021%2Fja905207t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Poppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zondlo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegley, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuanmanee, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheetham, J.</span><span> </span><span class="NLM_article-title">Discovery of ligands for Nurr1 by combined use of NMR screening with different isotopic and spin-labeling strategies</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1177/1087057106299161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1177%2F1087057106299161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=17438066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltlWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=301-311&author=L.+Poppeauthor=T.+S.+Harveyauthor=C.+Mohrauthor=J.+Zondloauthor=C.+M.+Tegleyauthor=O.+Nuanmaneeauthor=J.+Cheetham&title=Discovery+of+ligands+for+Nurr1+by+combined+use+of+NMR+screening+with+different+isotopic+and+spin-labeling+strategies&doi=10.1177%2F1087057106299161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ligands for Nurr1 by combined use of NMR screening with different isotopic and spin-labeling strategies</span></div><div class="casAuthors">Poppe, Leszek; Harvey, Timothy S.; Mohr, Christopher; Zondlo, James; Tegley, Christopher M.; Nuanmanee, Opas; Cheetham, Janet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A comprehensive approach to target screening, hit validation, and binding site detn. by NMR (NMR) spectroscopy is presented.  NMR 19F signal perturbation was used to screen a small compd. library and identify candidate ligands to the target of interest.  Ligand dissocn. consts. were measured using a pegylated form of the protein, which resulted in a 2-fold increase in the strength of the satn. transfer difference signal.  The initial small-moln. hits were further optimized by combining a residue-specific labeling strategy, to identify the specific sites of interaction with the protein, with a second site screening approach based on relaxation enhancement using a paramagnetic probe.  The advantages of this combination strategy in the identification and optimization of weak binding chem. entities early in a program are illustrated with the discovery of a low micromolar ligand (Kd = 20 μM) for Nurr1 and identification of the binding site location through residue-specific 15N isotope labeling and derivatization of Cys residues with 2-mercaptoethanol-1-13C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTXqTtwwsKT7Vg90H21EOLACvtfcHk0lgQZZszKK7Ycw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltlWitbs%253D&md5=74295fe8775657b17052af532a800d60</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057106299161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106299161%26sid%3Dliteratum%253Aachs%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DHarvey%26aufirst%3DT.%2BS.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DZondlo%26aufirst%3DJ.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DNuanmanee%26aufirst%3DO.%26aulast%3DCheetham%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520ligands%2520for%2520Nurr1%2520by%2520combined%2520use%2520of%2520NMR%2520screening%2520with%2520different%2520isotopic%2520and%2520spin-labeling%2520strategies%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2007%26volume%3D12%26spage%3D301%26epage%3D311%26doi%3D10.1177%2F1087057106299161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Querfurth, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFerla, F. M.</span><span> </span><span class="NLM_article-title">Alzheimer’s disease</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">362</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1056/NEJMra0909142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1056%2FNEJMra0909142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=20107219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1aqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=329-344&author=H.+W.+Querfurthauthor=F.+M.+LaFerla&title=Alzheimer%E2%80%99s+disease&doi=10.1056%2FNEJMra0909142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease</span></div><div class="casAuthors">Querfurth, Henry W.; LaFerla, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">329-344</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The role of β-amyloid peptide in spontaneous self-aggregation in brain in Alzheimer's disease is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPeniV_Jn6orVg90H21EOLACvtfcHk0lgQZZszKK7Ycw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1aqt7s%253D&md5=c116f6a6b8fed9af9f5282cc4b755c70</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra0909142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra0909142%26sid%3Dliteratum%253Aachs%26aulast%3DQuerfurth%26aufirst%3DH.%2BW.%26aulast%3DLaFerla%26aufirst%3DF.%2BM.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D329%26epage%3D344%26doi%3D10.1056%2FNEJMra0909142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selkoe, D. J.</span><span> </span><span class="NLM_article-title">The amyloid hypothesis of Alzheimer’s disease: progress and problems on the road to therapeutics</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">297</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1126/science.1072994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.1072994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=12130773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Cju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=353-356&author=J.+Hardyauthor=D.+J.+Selkoe&title=The+amyloid+hypothesis+of+Alzheimer%E2%80%99s+disease%3A+progress+and+problems+on+the+road+to+therapeutics&doi=10.1126%2Fscience.1072994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The amyloid hypothesis of Alzheimer's disease: progress and problems on the road to therapeutics</span></div><div class="casAuthors">Hardy, John; Selkoe, Dennis J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5580</span>),
    <span class="NLM_cas:pages">353-356</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  It has been more than 10 yr since it was first proposed that the neurodegeneration in Alzheimer's disease (AD) may be caused by deposition of amyloid β-peptide (Aβ) in plaques in brain tissue.  According to the amyloid hypothesis, accumulation of Aβ in the brain is the primary influence driving AD pathogenesis.  The rest of the disease process, including formation of neurofibrillary tangles contg. tau protein, is proposed to result from an imbalance between Aβ prodn. and Aβ clearance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9G5omTHR6cbVg90H21EOLACvtfcHk0lg1mDgLL8oJvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Cju7s%253D&md5=f7b8db1bc6f13e85f887b73f3042e86e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072994%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DJ.%26aulast%3DSelkoe%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520amyloid%2520hypothesis%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520progress%2520and%2520problems%2520on%2520the%2520road%2520to%2520therapeutics%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D353%26epage%3D356%26doi%3D10.1126%2Fscience.1072994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">De Strooper, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golde, T.</span><span> </span><span class="NLM_article-title">The secretases: enzymes with therapeutic potential in Alzheimer disease</span> <span class="citation_source-journal">Nat. Rev. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1038/nrneurol.2009.218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1038%2Fnrneurol.2009.218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=20139999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=99-107&author=B.+De+Strooperauthor=R.+Vassarauthor=T.+Golde&title=The+secretases%3A+enzymes+with+therapeutic+potential+in+Alzheimer+disease&doi=10.1038%2Fnrneurol.2009.218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The secretases: enzymes with therapeutic potential in Alzheimer disease</span></div><div class="casAuthors">De Strooper, Bart; Vassar, Robert; Golde, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-107</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The three secretases that process amyloid precursor protein are central to the generation of amyloid-β, and the accumulation of this peptide in extracellular plaques is one of the hallmarks of Alzheimer disease.  In this Review, De Strooper et al. discuss the evidence that suggests that these enzymes are potential therapeutic targets for Alzheimer disease drug treatments.  The amyloid hypothesis has yielded a series of well-validated candidate drug targets with potential for the treatment of Alzheimer disease (AD).  Three proteases that are involved in the processing of amyloid precursor protein-α-secretase, β-secretase and γ-secretase-are of particular interest as they are central to the generation and modulation of amyloid-β peptide and can be targeted by small compds. in vitro and in vivo.  Given that these proteases also fulfill other important biol. roles, inhibiting their activity will clearly be inherently assocd. with mechanism-based toxicity.  Carefully detg. a suitable therapeutic window and optimizing the selectivity of the drug treatments towards amyloid precursor protein processing might be ways of overcoming this potential complication.  Secretase inhibitors are likely to be the first small-mol. therapies aimed at AD modification that will be fully tested in the clinic.  Success or failure of these first-generation AD therapies will have enormous consequences for further drug development efforts for AD and possibly other neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBldqS_TKy7Vg90H21EOLACvtfcHk0lg1mDgLL8oJvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Ogt70%253D&md5=40135a0a59da6b70e4856a81ddfa6088</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2009.218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2009.218%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BStrooper%26aufirst%3DB.%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DGolde%26aufirst%3DT.%26atitle%3DThe%2520secretases%253A%2520enzymes%2520with%2520therapeutic%2520potential%2520in%2520Alzheimer%2520disease%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2010%26volume%3D6%26spage%3D99%26epage%3D107%26doi%3D10.1038%2Fnrneurol.2009.218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu-Khan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendiaz, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplow, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amarante, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeloff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edenson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lile, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarosinski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biere, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treanor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Citron, M.</span><span> </span><span class="NLM_article-title">Beta-secretase cleavage of Alzheimer’s amyloid precursor protein by the transmembrane aspartic protease BACE</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">741</span><span class="refDoi"> DOI: 10.1126/science.286.5440.735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1126%2Fscience.286.5440.735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10531052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXntFWit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1999&pages=735-741&author=R.+Vassarauthor=B.+D.+Bennettauthor=S.+Babu-Khanauthor=S.+Kahnauthor=E.+A.+Mendiazauthor=P.+Denisauthor=D.+B.+Teplowauthor=S.+Rossauthor=P.+Amaranteauthor=R.+Loeloffauthor=Y.+Luoauthor=S.+Fisherauthor=J.+Fullerauthor=S.+Edensonauthor=J.+Lileauthor=M.+A.+Jarosinskiauthor=A.+L.+Biereauthor=E.+Curranauthor=T.+Burgessauthor=J.+C.+Louisauthor=F.+Collinsauthor=J.+Treanorauthor=G.+Rogersauthor=M.+Citron&title=Beta-secretase+cleavage+of+Alzheimer%E2%80%99s+amyloid+precursor+protein+by+the+transmembrane+aspartic+protease+BACE&doi=10.1126%2Fscience.286.5440.735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">β-Secretase cleavage of Alzheimer's amyloid precursor protein by the transmembrane aspartic protease BACE</span></div><div class="casAuthors">Vassar, Robert; Bennett, Brian D.; Babu-Khan, Safura; Kahn, Steve; Mendiaz, Elizabeth A.; Denis, Paul; Teplow, David B.; Ross, Sandra; Amarante, Patricia; Loeloff, Richard; Luo, Yi; Fisher, Seth; Fuller, Janis; Edenson, Steven; Lile, Jackson; Jarosinski, Mark A.; Biere, Anja Leona; Curran, Eileen; Burgess, Teresa; Louis, Jean-Claude; Collins, Frank; Treanor, James; Rogers, Gary; Citron, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">5440</span>),
    <span class="NLM_cas:pages">735-741</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cerebral deposition of amyloid β peptide (Aβ) is an early and crit. feature of Alzheimer's disease.  Aβ generation depends on proteolytic cleavage of the amyloid precursor protein (APP) by two unknown proteases: β-secretase and γ-secretase.  These proteases are prime therapeutic targets.  A transmembrane aspartic protease with all the known characteristics of β-secretase was cloned and characterized.  Overexpression of this protease, termed BACE (for beta-site APP-cleaving enzyme) increased the amt. of β-secretase cleavage products, and these were cleaved exactly and only at known β-secretase positions.  Antisense inhibition of endogenous BACE mRNA decreased the amt. of β-secretase cleavage products, and purified BACE protein cleaved APP-derived substrates with the same sequence specificity as β-secretase.  Finally, the expression pattern and subcellular localization of BACE were consistent with that expected for β-secretase.  Future development of BACE inhibitors may prove beneficial for the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSx3s1NFyByrVg90H21EOLACvtfcHk0lg1mDgLL8oJvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFWit74%253D&md5=3f36e8e5b5e9441c7670ccaeb0d2dd4f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.286.5440.735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.286.5440.735%26sid%3Dliteratum%253Aachs%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DBabu-Khan%26aufirst%3DS.%26aulast%3DKahn%26aufirst%3DS.%26aulast%3DMendiaz%26aufirst%3DE.%2BA.%26aulast%3DDenis%26aufirst%3DP.%26aulast%3DTeplow%26aufirst%3DD.%2BB.%26aulast%3DRoss%26aufirst%3DS.%26aulast%3DAmarante%26aufirst%3DP.%26aulast%3DLoeloff%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DFisher%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DEdenson%26aufirst%3DS.%26aulast%3DLile%26aufirst%3DJ.%26aulast%3DJarosinski%26aufirst%3DM.%2BA.%26aulast%3DBiere%26aufirst%3DA.%2BL.%26aulast%3DCurran%26aufirst%3DE.%26aulast%3DBurgess%26aufirst%3DT.%26aulast%3DLouis%26aufirst%3DJ.%2BC.%26aulast%3DCollins%26aufirst%3DF.%26aulast%3DTreanor%26aufirst%3DJ.%26aulast%3DRogers%26aufirst%3DG.%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DBeta-secretase%2520cleavage%2520of%2520Alzheimer%25E2%2580%2599s%2520amyloid%2520precursor%2520protein%2520by%2520the%2520transmembrane%2520aspartic%2520protease%2520BACE%26jtitle%3DScience%26date%3D1999%26volume%3D286%26spage%3D735%26epage%3D741%26doi%3D10.1126%2Fscience.286.5440.735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, N.</span><span> </span><span class="NLM_article-title">Application of fragment screening by X-ray crystallography to beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1116</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1021/jm0611962</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0611962" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1116-1123&author=C.+W.+Murrayauthor=O.+Callaghanauthor=G.+Chessariauthor=A.+Cleasbyauthor=M.+Congreveauthor=M.+Fredericksonauthor=M.+J.+Hartshornauthor=R.+McMenaminauthor=S.+Patelauthor=N.+Wallis&title=Application+of+fragment+screening+by+X-ray+crystallography+to+beta-secretase&doi=10.1021%2Fjm0611962"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm0611962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0611962%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWallis%26aufirst%3DN.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520by%2520X-ray%2520crystallography%2520to%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1116%26epage%3D1123%26doi%3D10.1021%2Fjm0611962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fremeau, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louie, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michelsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nixey, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sickmier, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Jean, D. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span> </span><span class="NLM_article-title">From fragment screening to in vivo efficacy: optimization of a series of 2-aminoquinolines as potent inhibitors of beta-site amyloid precursor protein cleaving enzyme 1 (BACE1)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5836</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span><span class="refDoi"> DOI: 10.1021/jm200544q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200544q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5836-5857&author=Y.+Chengauthor=T.+C.+Juddauthor=M.+D.+Bartbergerauthor=J.+Brownauthor=K.+Chenauthor=R.+T.+Fremeauauthor=D.+Hickmanauthor=S.+A.+Hitchcockauthor=B.+Jordanauthor=V.+Liauthor=P.+Lopezauthor=S.+W.+Louieauthor=Y.+Luoauthor=K.+Michelsenauthor=T.+Nixeyauthor=T.+S.+Powersauthor=C.+Rattanauthor=E.+A.+Sickmierauthor=D.+J.+St.+Jeanauthor=R.+C.+Wahlauthor=P.+H.+Wenauthor=S.+Wood&title=From+fragment+screening+to+in+vivo+efficacy%3A+optimization+of+a+series+of+2-aminoquinolines+as+potent+inhibitors+of+beta-site+amyloid+precursor+protein+cleaving+enzyme+1+%28BACE1%29&doi=10.1021%2Fjm200544q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From Fragment Screening to In Vivo Efficacy: Optimization of a Series of 2-Aminoquinolines as Potent Inhibitors of Beta-Site Amyloid Precursor Protein Cleaving Enzyme 1 (BACE1)</span></div><div class="casAuthors">Cheng, Yuan; Judd, Ted C.; Bartberger, Michael D.; Brown, James; Chen, Kui; Fremeau, Robert T.; Hickman, Dean; Hitchcock, Stephen A.; Jordan, Brad; Li, Vivian; Lopez, Patricia; Louie, Steven W.; Luo, Yi; Michelsen, Klaus; Nixey, Thomas; Powers, Timothy S.; Rattan, Claire; Sickmier, E. Allen; St. Jean, David J.; Wahl, Robert C.; Wen, Paul H.; Wood, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5836-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening of a focused fragment library, 2-aminoquinoline 1 was identified as an initial hit for BACE1.  Further SAR development was supported by X-ray structures of BACE1 cocrystd. with various ligands and mol. modeling studies to expedite the discovery of potent compds.  These strategies enabled us to integrate the C-3 side chain on 2-aminoquinoline 1 extending deep into the P2' binding pocket of BACE1 and enhancing the ligand's potency.  We were able to improve the BACE1 potency to subnanomolar range, over 106-fold more potent than the initial hit (900 μM).  Further elaboration of the phys. properties of the lead compds. to those more consistent with good blood-brain barrier permeability led to inhibitors with greatly improved cellular activity and permeability.  Compd. 59 showed an IC50 value of 11 nM on BACE1 and cellular activity of 80 nM.  This compd. was advanced into rat pharmacokinetic and pharmacodynamic studies and demonstrated significant redn. of Aβ levels in cerebrospinal fluid (CSF).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLRGyUyxO1fLVg90H21EOLACvtfcHk0liEag5pTRIcfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSksrk%253D&md5=fa0f2b9b7524a1c3b75e162801f54d90</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm200544q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200544q%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJudd%26aufirst%3DT.%2BC.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DHickman%26aufirst%3DD.%26aulast%3DHitchcock%26aufirst%3DS.%2BA.%26aulast%3DJordan%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DV.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLouie%26aufirst%3DS.%2BW.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMichelsen%26aufirst%3DK.%26aulast%3DNixey%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DT.%2BS.%26aulast%3DRattan%26aufirst%3DC.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DSt.%2BJean%26aufirst%3DD.%2BJ.%26aulast%3DWahl%26aufirst%3DR.%2BC.%26aulast%3DWen%26aufirst%3DP.%2BH.%26aulast%3DWood%26aufirst%3DS.%26atitle%3DFrom%2520fragment%2520screening%2520to%2520in%2520vivo%2520efficacy%253A%2520optimization%2520of%2520a%2520series%2520of%25202-aminoquinolines%2520as%2520potent%2520inhibitors%2520of%2520beta-site%2520amyloid%2520precursor%2520protein%2520cleaving%2520enzyme%25201%2520%2528BACE1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5836%26epage%3D5857%26doi%3D10.1021%2Fjm200544q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aharony, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, N.</span><span> </span><span class="NLM_article-title">Application of fragment screening by X-ray crystallography to the discovery of aminopyridines as inhibitors of beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.1021/jm061197u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061197u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1124-1132&author=M.+Congreveauthor=D.+Aharonyauthor=J.+Albertauthor=O.+Callaghanauthor=J.+Campbellauthor=R.+A.+Carrauthor=G.+Chessariauthor=S.+Cowanauthor=P.+D.+Edwardsauthor=M.+Fredericksonauthor=R.+McMenaminauthor=C.+W.+Murrayauthor=S.+Patelauthor=N.+Wallis&title=Application+of+fragment+screening+by+X-ray+crystallography+to+the+discovery+of+aminopyridines+as+inhibitors+of+beta-secretase&doi=10.1021%2Fjm061197u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm061197u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061197u%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DAharony%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DJ.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DEdwards%26aufirst%3DP.%2BD.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWallis%26aufirst%3DN.%26atitle%3DApplication%2520of%2520fragment%2520screening%2520by%2520X-ray%2520crystallography%2520to%2520the%2520discovery%2520of%2520aminopyridines%2520as%2520inhibitors%2520of%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1124%26epage%3D1132%26doi%3D10.1021%2Fjm061197u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Stamford, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C.</span><span> </span><span class="NLM_article-title">Inhibitors of BACE for treating Alzheimer’s disease: a fragment-based drug discovery story</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2013.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2Fj.cbpa.2013.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=23683349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslaku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=320-328&author=A.+Stamfordauthor=C.+Strickland&title=Inhibitors+of+BACE+for+treating+Alzheimer%E2%80%99s+disease%3A+a+fragment-based+drug+discovery+story&doi=10.1016%2Fj.cbpa.2013.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of BACE for treating Alzheimer's disease: a fragment-based drug discovery story</span></div><div class="casAuthors">Stamford, Andrew; Strickland, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">320-328</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Several fragment-based methods have been applied to the discovery of new lead sources for inhibitors of BACE1, an important therapeutic target for Alzheimer's disease.  Among the most common fragment hits were various amidine-contg. mols. in which the amidine engaged in discrete H-bond donor-acceptor interaction with the BACE1 catalytic dyad.  Structure and medicinal chem. knowledge-based optimization with emphasis on ligand efficiency resulted in identification of a key pharmacophore comprising a non-planar cyclic amidine scaffold directly attached to a Ph group projecting into S1.  This key pharmacophore is a common feature of known clin. candidates and has dominated the recent patent literature.  A structural comparison of the non-planar cyclic amidine motif with other BACE1 pharmacophores highlights its uniqueness and distinct advantages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfLbpbjuHoU7Vg90H21EOLACvtfcHk0lgjR6ae6zzPng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslaku7w%253D&md5=f39c9a0499fe096cf4c0719f4cb41e37</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DStamford%26aufirst%3DA.%26aulast%3DStrickland%26aufirst%3DC.%26atitle%3DInhibitors%2520of%2520BACE%2520for%2520treating%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520a%2520fragment-based%2520drug%2520discovery%2520story%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D320%26epage%3D328%26doi%3D10.1016%2Fj.cbpa.2013.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaton, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamford, A. W.</span><span> </span><span class="NLM_article-title">Combining NMR and X-ray crystallography in fragment-based drug discovery: discovery of highly potent and selective BACE-1 inhibitors</span> <span class="citation_source-journal">Top. Curr. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1007/128_2011_183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1007%2F128_2011_183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2011&pages=83-114&author=D.+F.+Wyssauthor=Y.+S.+Wangauthor=H.+L.+Eatonauthor=C.+Stricklandauthor=J.+H.+Voigtauthor=Z.+Zhuauthor=A.+W.+Stamford&title=Combining+NMR+and+X-ray+crystallography+in+fragment-based+drug+discovery%3A+discovery+of+highly+potent+and+selective+BACE-1+inhibitors&doi=10.1007%2F128_2011_183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F128_2011_183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_183%26sid%3Dliteratum%253Aachs%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DEaton%26aufirst%3DH.%2BL.%26aulast%3DStrickland%26aufirst%3DC.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DStamford%26aufirst%3DA.%2BW.%26atitle%3DCombining%2520NMR%2520and%2520X-ray%2520crystallography%2520in%2520fragment-based%2520drug%2520discovery%253A%2520discovery%2520of%2520highly%2520potent%2520and%2520selective%2520BACE-1%2520inhibitors%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2011%26volume%3D317%26spage%3D83%26epage%3D114%26doi%3D10.1007%2F128_2011_183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittington, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, V. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dineen, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epstein, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graceffa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louie, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fremeau, R. T.,  Jr.</span><span> </span><span class="NLM_article-title">Structure- and property-based design of aminooxazoline xanthenes as selective, orally efficacious, and CNS penetrable BACE inhibitors for the treatment of Alzheimer’s disease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9156</span><span class="NLM_x">–</span> <span class="NLM_lpage">9169</span><span class="refDoi"> DOI: 10.1021/jm300598e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300598e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9156-9169&author=H.+Huangauthor=D.+S.+Laauthor=A.+C.+Chengauthor=D.+A.+Whittingtonauthor=V.+F.+Patelauthor=K.+Chenauthor=T.+A.+Dineenauthor=O.+Epsteinauthor=R.+Graceffaauthor=D.+Hickmanauthor=Y.+H.+Kiangauthor=S.+Louieauthor=Y.+Luoauthor=R.+C.+Wahlauthor=P.+H.+Wenauthor=S.+Woodauthor=R.+T.+Fremeau&title=Structure-+and+property-based+design+of+aminooxazoline+xanthenes+as+selective%2C+orally+efficacious%2C+and+CNS+penetrable+BACE+inhibitors+for+the+treatment+of+Alzheimer%E2%80%99s+disease&doi=10.1021%2Fjm300598e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and property-based design of aminooxazoline xanthenes as selective, orally efficacious, and CNS penetrable BACE inhibitors for the treatment of Alzheimer's disease</span></div><div class="casAuthors">Huang, Hongbing; La, Daniel S.; Cheng, Alan C.; Whittington, Douglas A.; Patel, Vinod F.; Chen, Kui; Dineen, Thomas A.; Epstein, Oleg; Graceffa, Russell; Hickman, Dean; Kiang, Y.-H.; Louie, Steven; Luo, Yi; Wahl, Robert C.; Wen, Paul H.; Wood, Stephen; Fremeau, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9156-9169</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A structure- and property-based drug design approach was employed to identify aminooxazoline xanthenes as potent and selective human β-secretase inhibitors.  These compds. exhibited good isolated enzyme, cell potency, and selectivity against the structurally related aspartyl protease cathepsin D.  Our efforts resulted in the identification of a potent, orally bioavailable CNS penetrant compd. that exhibited in vivo efficacy.  A single oral dose of compd. (I) resulted in a significant redn. of CNS Aβ40 in naive rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDEYfU4moxMbVg90H21EOLACvtfcHk0lgjR6ae6zzPng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLjJ&md5=0ecd50ec1ea82a058e499bb712c45272</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm300598e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300598e%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DLa%26aufirst%3DD.%2BS.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DPatel%26aufirst%3DV.%2BF.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DDineen%26aufirst%3DT.%2BA.%26aulast%3DEpstein%26aufirst%3DO.%26aulast%3DGraceffa%26aufirst%3DR.%26aulast%3DHickman%26aufirst%3DD.%26aulast%3DKiang%26aufirst%3DY.%2BH.%26aulast%3DLouie%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DWahl%26aufirst%3DR.%2BC.%26aulast%3DWen%26aufirst%3DP.%2BH.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26atitle%3DStructure-%2520and%2520property-based%2520design%2520of%2520aminooxazoline%2520xanthenes%2520as%2520selective%252C%2520orally%2520efficacious%252C%2520and%2520CNS%2520penetrable%2520BACE%2520inhibitors%2520for%2520the%2520treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9156%26epage%3D9169%26doi%3D10.1021%2Fjm300598e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Steinfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhardt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillebrand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraetzner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruck, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saftig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gartner, J.</span><span> </span><span class="NLM_article-title">Cathepsin D deficiency is associated with a human neurodegenerative disorder</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">988</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span><span class="refDoi"> DOI: 10.1086/504159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1086%2F504159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16685649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVyks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2006&pages=988-998&author=R.+Steinfeldauthor=K.+Reinhardtauthor=K.+Schreiberauthor=M.+Hillebrandauthor=R.+Kraetznerauthor=W.+Bruckauthor=P.+Saftigauthor=J.+Gartner&title=Cathepsin+D+deficiency+is+associated+with+a+human+neurodegenerative+disorder&doi=10.1086%2F504159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D deficiency is associated with a human neurodegenerative disorder</span></div><div class="casAuthors">Steinfeld, Robert; Reinhardt, Konstanze; Schreiber, Kathrin; Hillebrand, Merle; Kraetzner, Ralph; Brueck, Wolfgang; Saftig, Paul; Gaertner, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">988-998</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Cathepsin D is a ubiquitously expressed lysosomal protease that is involved in proteolytic degrdn., cell invasion, and apoptosis.  In mice and sheep, cathepsin D deficiency is known to cause a fatal neurodegenerative disease.  Here, we report a novel disorder in a child with early blindness and progressive psychomotor disability.  Two missense mutations in the CTSD gene, F229I and W383C, were identified and were found to cause markedly reduced proteolytic activity and a diminished amt. of cathepsin D in patient fibroblasts.  Expression of cathepsin D mutants in cathepsin D-/- mouse fibroblasts revealed disturbed posttranslational processing and intracellular targeting for W383C and diminished maximal enzyme velocity for F229I.  The structural effects of cathepsin D mutants were estd. by computer modeling, which suggested larger structural alterations for W383C than for F229I.  Our studies broaden the group of human neurodegenerative disorders and add new insight into the cellular functions of human cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohf0ApK0eyVLVg90H21EOLACvtfcHk0lgjR6ae6zzPng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVyks78%253D&md5=96a9bf74f1823244552ed50391eb5278</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1086%2F504159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F504159%26sid%3Dliteratum%253Aachs%26aulast%3DSteinfeld%26aufirst%3DR.%26aulast%3DReinhardt%26aufirst%3DK.%26aulast%3DSchreiber%26aufirst%3DK.%26aulast%3DHillebrand%26aufirst%3DM.%26aulast%3DKraetzner%26aufirst%3DR.%26aulast%3DBruck%26aufirst%3DW.%26aulast%3DSaftig%26aufirst%3DP.%26aulast%3DGartner%26aufirst%3DJ.%26atitle%3DCathepsin%2520D%2520deficiency%2520is%2520associated%2520with%2520a%2520human%2520neurodegenerative%2520disorder%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2006%26volume%3D78%26spage%3D988%26epage%3D998%26doi%3D10.1086%2F504159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Bjorklund, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oscarson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benkestock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borkakoti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenquist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span> </span><span class="NLM_article-title">Design and synthesis of potent and selective BACE-1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1458</span><span class="NLM_x">–</span> <span class="NLM_lpage">1464</span><span class="refDoi"> DOI: 10.1021/jm901168f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901168f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1458-1464&author=C.+Bjorklundauthor=S.+Oscarsonauthor=K.+Benkestockauthor=N.+Borkakotiauthor=K.+Janssonauthor=J.+Lindbergauthor=L.+Vrangauthor=A.+Hallbergauthor=A.+Rosenquistauthor=B.+Samuelsson&title=Design+and+synthesis+of+potent+and+selective+BACE-1+inhibitors&doi=10.1021%2Fjm901168f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm901168f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901168f%26sid%3Dliteratum%253Aachs%26aulast%3DBjorklund%26aufirst%3DC.%26aulast%3DOscarson%26aufirst%3DS.%26aulast%3DBenkestock%26aufirst%3DK.%26aulast%3DBorkakoti%26aufirst%3DN.%26aulast%3DJansson%26aufirst%3DK.%26aulast%3DLindberg%26aufirst%3DJ.%26aulast%3DVrang%26aufirst%3DL.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520BACE-1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1458%26epage%3D1464%26doi%3D10.1021%2Fjm901168f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Page, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jencks, W. P.</span><span> </span><span class="NLM_article-title">Entropic contributions to rate accelerations in enzymic and intramolecular reactions and the chelate effect</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">1678</span><span class="NLM_x">–</span> <span class="NLM_lpage">1683</span><span class="refDoi"> DOI: 10.1073/pnas.68.8.1678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1073%2Fpnas.68.8.1678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=5288752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaE3MXltVGhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=1971&pages=1678-1683&author=M.+I.+Pageauthor=W.+P.+Jencks&title=Entropic+contributions+to+rate+accelerations+in+enzymic+and+intramolecular+reactions+and+the+chelate+effect&doi=10.1073%2Fpnas.68.8.1678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic contributions to rate accelerations in enzymic and intramolecular reactions and the chelate effect</span></div><div class="casAuthors">Page, Michael I.; Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1678-83</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Translational and (overall) rotational motions provide the important entropic driving force for enzymic and intramol. rate accelerations and the chelate effect; internal rotations and unusually severe orientational requirements are generally of secondary importance.  The loss of translational and (overall) rotational entropy for 2 → 1 reactions in soln. is ordinarily on the order of 45 entropy units (e.u.) (std. state M, 25°); the translational entropy is much larger than 8 e.u. (corresponding to 55 M).  Low-frequency motions in products and transition states, ∼17 e.u. for cyclopentadiene dimerization, partially compensate for this loss, but effective concns. on the order of 108 M may be accounted for without the introduction of new chem. concepts or terms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3iq7fY_SBwrVg90H21EOLACvtfcHk0lhMcDtwwA54fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXltVGhtLw%253D&md5=f2190b0dde6c8b5ffdf43adec063265a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.68.8.1678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.68.8.1678%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BI.%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DEntropic%2520contributions%2520to%2520rate%2520accelerations%2520in%2520enzymic%2520and%2520intramolecular%2520reactions%2520and%2520the%2520chelate%2520effect%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1971%26volume%3D68%26spage%3D1678%26epage%3D1683%26doi%3D10.1073%2Fpnas.68.8.1678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Jencks, W. P.</span><span> </span><span class="NLM_article-title">On the attribution and additivity of binding energies</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">4046</span><span class="NLM_x">–</span> <span class="NLM_lpage">4050</span><span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+attribution+and+additivity+of+binding+energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0lhMcDtwwA54fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Hajduk, P. J.; Huth, J. R.; Sun, C.</span><span> </span><span class="NLM_article-title">SAR by NMR: An Analysis of Potency Gains Realized Through Fragment-linking and Fragment-elaboration Strategies for Lead Generation</span>. In  <span class="citation_source-book">Fragment-based Approaches in Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F3527608761.ch9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=181-192&author=P.+J.+Hajduk&author=J.+R.+Huth&author=C.+Sunauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-based+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2F3527608761.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527608761.ch9%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DSAR%2520by%2520NMR%253A%2520An%2520Analysis%2520of%2520Potency%2520Gains%2520Realized%2520Through%2520Fragment-linking%2520and%2520Fragment-elaboration%2520Strategies%2520for%2520Lead%2520Generation%26btitle%3DFragment-based%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D181%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1023/A:1022446720849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1023%2FA%3A1022446720849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=12650591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=741-753&author=C.+W.+Murrayauthor=M.+L.+Verdonk&title=The+consequences+of+translational+and+rotational+entropy+lost+by+small+molecules+on+binding+to+proteins&doi=10.1023%2FA%3A1022446720849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-753</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  When a small mol. binds to a protein, it loses a significant amt. of rigid body translational and rotational entropy.  Ests. of the assocd. energy barrier vary widely in the literature yet accurate ests. are important in the interpretation of results from fragment-based drug discovery techniques.  This paper describes an anal. that allows the estn. of the rigid body entropy barrier from the increase in binding affinities that results when two fragments of known affinity and known binding mode are joined together.  The paper reviews the relatively rare no. of examples where good quality data is available.  From the anal. of this data, we est. that the barrier to binding, due to the loss of rigid-body entropy, is 15-20 kJ/mol, i.e. around 3 orders of magnitude in affinity at 298 K.  This large barrier explains why it is comparatively rare to observe multiple fragments binding to non-overlapping adjacent sites in enzymes.  The barrier is also consistent with medicinal chem. experience where small changes in the crit. binding regions of ligands are often poorly tolerated by enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KId6AyrMCLVg90H21EOLACvtfcHk0lhMcDtwwA54fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D&md5=4ad6aef74de3cb86302fe16f7e61c1c6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1022446720849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1022446720849%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DThe%2520consequences%2520of%2520translational%2520and%2520rotational%2520entropy%2520lost%2520by%2520small%2520molecules%2520on%2520binding%2520to%2520proteins%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D741%26epage%3D753%26doi%3D10.1023%2FA%3A1022446720849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Davies, T. G.; Monfort, R. L. M.; Williams, G.; Jhoti, H.</span><span> </span><span class="NLM_article-title">Pyramid: An integrated platform for fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-based Approaches in Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2F3527608761.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=T.+G.+Davies&author=R.+L.+M.+Monfort&author=G.+Williams&author=H.+Jhotiauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-based+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F3527608761.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527608761.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DPyramid%253A%2520An%2520integrated%2520platform%2520for%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-based%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DOxford%2520University%2520Press%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeRose, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">London, R. E.</span><span> </span><span class="NLM_article-title">The inter-ligand Overhauser effect: a powerful new NMR approach for mapping structural relationships of macromolecular ligands</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.1023/A:1008360208627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1023%2FA%3A1008360208627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10549135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=71-76&author=D.+Liauthor=E.+F.+DeRoseauthor=R.+E.+London&title=The+inter-ligand+Overhauser+effect%3A+a+powerful+new+NMR+approach+for+mapping+structural+relationships+of+macromolecular+ligands&doi=10.1023%2FA%3A1008360208627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The inter-ligand Overhauser effect: a powerful new NMR approach for mapping structural relationships of macromolecular ligands</span></div><div class="casAuthors">Li, Dawei; DeRose, Eugene F.; London, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-76</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">NMR expts. that transfer conformational information from the bound to the uncomplexed state via exchange have been utilized for many years.  It is demonstrated here that inter-ligand NOEs ("ILOEs"), which exist in ternary complexes with enzymes or other macromol. receptors, can be transferred via exchange to pairs of uncomplexed ligands.  This approach is illustrated by studies of glycolate + NAD+ in the presence of porcine heart lactate dehydrogenase, and by glucose-6-phosphate + NADPH in the presence of L. mesenteroides glucose-6-phosphate dehydrogenase.  This strategy opens up a general methodol. for exploring the active sites of enzymes and for the development of artificial ligands which can function as inhibitors, or more generally as modifiers of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprcPANaPftSLVg90H21EOLACvtfcHk0linbQur_FkKmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSgtr8%253D&md5=ff105603708ffb0e2a4e1501639707e8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008360208627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008360208627%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DDeRose%26aufirst%3DE.%2BF.%26aulast%3DLondon%26aufirst%3DR.%2BE.%26atitle%3DThe%2520inter-ligand%2520Overhauser%2520effect%253A%2520a%2520powerful%2520new%2520NMR%2520approach%2520for%2520mapping%2520structural%2520relationships%2520of%2520macromolecular%2520ligands%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1999%26volume%3D15%26spage%3D71%26epage%3D76%26doi%3D10.1023%2FA%3A1008360208627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span> <span class="citation_source-book">Molecular Operating Environment</span>; <span class="NLM_publisher-name">Chemical Computing Group</span>: <span class="NLM_publisher-loc">Montreal, Canada</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment%3B+Chemical+Computing+Group%3A+Montreal%2C+Canada%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%26pub%3DChemical%2520Computing%2520Group%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span><span class="refDoi"> DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0linbQur_FkKmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuillard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yon, J.</span><span> </span><span class="NLM_article-title">Apo and inhibitor complex structures of BACE (beta-secretase)</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2Fj.jmb.2004.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15451669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2004&pages=407-416&author=S.+Patelauthor=L.+Vuillardauthor=A.+Cleasbyauthor=C.+W.+Murrayauthor=J.+Yon&title=Apo+and+inhibitor+complex+structures+of+BACE+%28beta-secretase%29&doi=10.1016%2Fj.jmb.2004.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Apo and Inhibitor Complex Structures of BACE (β-secretase)</span></div><div class="casAuthors">Patel, Sahil; Vuillard, Laurent; Cleasby, Anne; Murray, Christopher W.; Yon, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-416</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human BACE, also known as β-secretase, shows promise as a potential therapeutic target for Alzheimer's disease.  The authors detd. the apo structure of BACE to 1.75 Å, and a structure of a hydroxyethylamine inhibitor complex derived by soaking.  These show significant active-site movements compared to previously described BACE structures.  Addnl., the structures reveal two pockets that could be targeted by structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0afg2-ZxjLVg90H21EOLACvtfcHk0lh-2Sjm5hQIUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVyqt7w%253D&md5=b14e17f86a4729fb8ee66d84ee6c1fab</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVuillard%26aufirst%3DL.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DYon%26aufirst%3DJ.%26atitle%3DApo%2520and%2520inhibitor%2520complex%2520structures%2520of%2520BACE%2520%2528beta-secretase%2529%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D343%26spage%3D407%26epage%3D416%26doi%3D10.1016%2Fj.jmb.2004.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lh-2Sjm5hQIUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh-2Sjm5hQIUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Turner, R. T.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koelsch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castenheira, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ermolieff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span> </span><span class="NLM_article-title">Subsite specificity of memapsin 2 (beta-secretase): implications for inhibitor design</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">10001</span><span class="NLM_x">–</span> <span class="NLM_lpage">10006</span><span class="refDoi"> DOI: 10.1021/bi015546s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi015546s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=10001-10006&author=R.+T.+Turnerauthor=G.+Koelschauthor=L.+Hongauthor=P.+Castenheiraauthor=J.+Ermolieffauthor=A.+K.+Ghoshauthor=J.+Tang&title=Subsite+specificity+of+memapsin+2+%28beta-secretase%29%3A+implications+for+inhibitor+design&doi=10.1021%2Fbi015546s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi015546s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi015546s%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DR.%2BT.%26aulast%3DKoelsch%26aufirst%3DG.%26aulast%3DHong%26aufirst%3DL.%26aulast%3DCastenheira%26aufirst%3DP.%26aulast%3DErmolieff%26aufirst%3DJ.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DTang%26aufirst%3DJ.%26atitle%3DSubsite%2520specificity%2520of%2520memapsin%25202%2520%2528beta-secretase%2529%253A%2520implications%2520for%2520inhibitor%2520design%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D10001%26epage%3D10006%26doi%3D10.1021%2Fbi015546s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Haniu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendiaz, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Citron, M.</span><span> </span><span class="NLM_article-title">Characterization of Alzheimer’s beta -secretase protein BACE. A pepsin family member with unusual properties</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">21099</span><span class="NLM_x">–</span> <span class="NLM_lpage">21106</span><span class="refDoi"> DOI: 10.1074/jbc.M002095200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;key=10.1074%2Fjbc.M002095200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21099-21106&author=M.+Haniuauthor=P.+Denisauthor=Y.+Youngauthor=E.+A.+Mendiazauthor=J.+Fullerauthor=J.+O.+Huiauthor=B.+D.+Bennettauthor=S.+Kahnauthor=S.+Rossauthor=T.+Burgessauthor=V.+Kattaauthor=G.+Rogersauthor=R.+Vassarauthor=M.+Citron&title=Characterization+of+Alzheimer%E2%80%99s+beta+-secretase+protein+BACE.+A+pepsin+family+member+with+unusual+properties&doi=10.1074%2Fjbc.M002095200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M002095200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M002095200%26sid%3Dliteratum%253Aachs%26aulast%3DHaniu%26aufirst%3DM.%26aulast%3DDenis%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DY.%26aulast%3DMendiaz%26aufirst%3DE.%2BA.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DHui%26aufirst%3DJ.%2BO.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DKahn%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DS.%26aulast%3DBurgess%26aufirst%3DT.%26aulast%3DKatta%26aufirst%3DV.%26aulast%3DRogers%26aufirst%3DG.%26aulast%3DVassar%26aufirst%3DR.%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DCharacterization%2520of%2520Alzheimer%25E2%2580%2599s%2520beta%2520-secretase%2520protein%2520BACE.%2520A%2520pepsin%2520family%2520member%2520with%2520unusual%2520properties%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21099%26epage%3D21106%26doi%3D10.1074%2Fjbc.M002095200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W','PDB','5I3W'); return false;">PDB: 5I3W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1','PDB','5IE1'); return false;">PDB: 5IE1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3V','PDB','5I3V'); return false;">PDB: 5I3V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3X','PDB','5I3X'); return false;">PDB: 5I3X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3Y','PDB','5I3Y'); return false;">PDB: 5I3Y</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i47"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01917">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_22462"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01917">10.1021/acs.jmedchem.5b01917</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table of crystallographic data, SPR sensorgrams, BACE-1, BACE-2, and CatD enzyme assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Data table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_001.pdf">jm5b01917_si_001.pdf (526.18 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01917/suppl_file/jm5b01917_si_002.csv">jm5b01917_si_002.csv (0.54 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">New protein/ligand coordinates have been deposited in the PDB with IDs of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3V">5I3V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3W">5I3W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3X">5I3X</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I3Y">5I3Y</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IE1">5IE1</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01917%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-8%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01917" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bf43de9b3d5a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
