
<html lang="en"     class="pb-page"  data-request-id="dc336ac3-e078-4b59-ab0c-11b6ef7ef286"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-18;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c01125;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Proteolysis-Targeting Chimera Molecules that Selectively Degrade the IRAK3 Pseudokinase" /></meta><meta name="dc.Creator" content="Sébastien L.  Degorce" /></meta><meta name="dc.Creator" content="Omid  Tavana" /></meta><meta name="dc.Creator" content="Erica  Banks" /></meta><meta name="dc.Creator" content="Claire  Crafter" /></meta><meta name="dc.Creator" content="Lakshmaiah  Gingipalli" /></meta><meta name="dc.Creator" content="David  Kouvchinov" /></meta><meta name="dc.Creator" content="Yumeng  Mao" /></meta><meta name="dc.Creator" content="Fiona  Pachl" /></meta><meta name="dc.Creator" content="Anisha  Solanki" /></meta><meta name="dc.Creator" content="Viia  Valge-Archer" /></meta><meta name="dc.Creator" content="Bin  Yang" /></meta><meta name="dc.Creator" content="Scott D.  Edmondson" /></meta><meta name="dc.Description" content="We report the first disclosure of IRAK3 degraders in the scientific literature. Taking advantage of an opportune byproduct obtained during our efforts to identify IRAK4 inhibitors, we identified re..." /></meta><meta name="Description" content="We report the first disclosure of IRAK3 degraders in the scientific literature. Taking advantage of an opportune byproduct obtained during our efforts to identify IRAK4 inhibitors, we identified re..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 17, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01125" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01125" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01125" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01125" /></link>
        
    
    

<title>Discovery of Proteolysis-Targeting Chimera Molecules that Selectively Degrade the IRAK3 Pseudokinase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01125" /></meta><meta property="og:title" content="Discovery of Proteolysis-Targeting Chimera Molecules that Selectively Degrade the IRAK3 Pseudokinase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0016.jpeg" /></meta><meta property="og:description" content="We report the first disclosure of IRAK3 degraders in the scientific literature. Taking advantage of an opportune byproduct obtained during our efforts to identify IRAK4 inhibitors, we identified ready-to-use, selective IRAK3 ligands in our compound collection with the required properties for conversion into proteolysis-targeting chimera (PROTAC) degraders. This work culminated with the discovery of PROTAC 23, which we demonstrated to be a potent and selective degrader of IRAK3 after 16 h in THP1 cells. 23 induced proteasome-dependent degradation of IRAK3 and required both CRBN and IRAK3 binding for activity. We conclude that PROTAC 23 constitutes an excellent in vitro tool with which to interrogate the biology of IRAK3." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01125"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01125">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01125&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01125&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01125&amp;href=/doi/10.1021/acs.jmedchem.0c01125" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 10460-10473</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01086" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01183" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Proteolysis-Targeting Chimera Molecules that Selectively Degrade the IRAK3 Pseudokinase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sébastien L. Degorce</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sébastien L. Degorce</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#88fbedeae9fbfce1ede6a6ecedefe7faebedc8e9fbfcfae9f2ede6edebe9a6ebe7e5"><span class="__cf_email__" data-cfemail="3e4d5b5c5f4d4a575b50105a5b59514c5d5b7e5f4d4a4c5f445b505b5d5f105d5153">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=S%C3%A9bastien+L.++Degorce">Sébastien L. Degorce</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9478-5106" title="Orcid link">http://orcid.org/0000-0002-9478-5106</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Omid Tavana</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Omid Tavana</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#cea1a3a7aae0baafb8afa0af8eafbdbabcafb4aba0abadafe0ada1a3"><span class="__cf_email__" data-cfemail="305f5d59541e445146515e517051434442514a555e5553511e535f5d">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Omid++Tavana">Omid Tavana</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8401-4546" title="Orcid link">http://orcid.org/0000-0001-8401-4546</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Erica Banks</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Erica Banks</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Erica++Banks">Erica Banks</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Claire Crafter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claire Crafter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Cambridge Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claire++Crafter">Claire Crafter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lakshmaiah Gingipalli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lakshmaiah Gingipalli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lakshmaiah++Gingipalli">Lakshmaiah Gingipalli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Kouvchinov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Kouvchinov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Kouvchinov">David Kouvchinov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yumeng Mao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yumeng Mao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Cambridge Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yumeng++Mao">Yumeng Mao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fiona Pachl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fiona Pachl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fiona++Pachl">Fiona Pachl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anisha Solanki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anisha Solanki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Cambridge Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anisha++Solanki">Anisha Solanki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Viia Valge-Archer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Viia Valge-Archer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioscience, Research and Early Development, Oncology R&D, AstraZeneca, Cambridge Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Viia++Valge-Archer">Viia Valge-Archer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bin Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Yang">Bin Yang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Scott D. Edmondson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott D. Edmondson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+D.++Edmondson">Scott D. Edmondson</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01125&amp;href=/doi/10.1021%2Facs.jmedchem.0c01125" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 10460–10473</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 17, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 June 2020</li><li><span class="item_label"><b>Published</b> online</span>17 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01125" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01125</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10460%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DS%25C3%25A9bastien%2BL.%2BDegorce%252C%2BOmid%2BTavana%252C%2BErica%2BBanks%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D18%26contentID%3Dacs.jmedchem.0c01125%26title%3DDiscovery%2Bof%2BProteolysis-Targeting%2BChimera%2BMolecules%2Bthat%2BSelectively%2BDegrade%2Bthe%2BIRAK3%2BPseudokinase%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10473%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01125"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3233</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01125" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Proteolysis-Targeting Chimera Molecules that Selectively Degrade the IRAK3 Pseudokinase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sébastien&quot;,&quot;last_name&quot;:&quot;L. Degorce&quot;},{&quot;first_name&quot;:&quot;Omid&quot;,&quot;last_name&quot;:&quot;Tavana&quot;},{&quot;first_name&quot;:&quot;Erica&quot;,&quot;last_name&quot;:&quot;Banks&quot;},{&quot;first_name&quot;:&quot;Claire&quot;,&quot;last_name&quot;:&quot;Crafter&quot;},{&quot;first_name&quot;:&quot;Lakshmaiah&quot;,&quot;last_name&quot;:&quot;Gingipalli&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Kouvchinov&quot;},{&quot;first_name&quot;:&quot;Yumeng&quot;,&quot;last_name&quot;:&quot;Mao&quot;},{&quot;first_name&quot;:&quot;Fiona&quot;,&quot;last_name&quot;:&quot;Pachl&quot;},{&quot;first_name&quot;:&quot;Anisha&quot;,&quot;last_name&quot;:&quot;Solanki&quot;},{&quot;first_name&quot;:&quot;Viia&quot;,&quot;last_name&quot;:&quot;Valge-Archer&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;D. Edmondson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;10460-10473&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01125&quot;},&quot;abstract&quot;:&quot;We report the first disclosure of IRAK3 degraders in the scientific literature. Taking advantage of an opportune byproduct obtained during our efforts to identify IRAK4 inhibitors, we identified ready-to-use, selective IRAK3 ligands in our compound collection with the required properties for conversion into proteolysis-targeting chimera (PROTAC) degraders. This work culminated with the discovery of PROTAC 23, which we demonstrated to be a potent and selective degrader of IRAK3 after 16 h in THP1 cells. 23 induced proteasome-dependent degradation of IRAK3 and required both CRBN and IRAK3 binding for activity. We conclude that PROTAC 23 constitutes an excellent in vitro tool with which to interrogate the biology of IRAK3.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01125&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01125" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01125&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01125" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01125&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01125" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01125&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01125&amp;href=/doi/10.1021/acs.jmedchem.0c01125" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01125" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01125" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01125%26sid%3Dliteratum%253Aachs%26pmid%3D32803978%26genre%3Darticle%26aulast%3DDegorce%26date%3D2020%26atitle%3DDiscovery%2Bof%2BProteolysis-Targeting%2BChimera%2BMolecules%2Bthat%2BSelectively%2BDegrade%2Bthe%2BIRAK3%2BPseudokinase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D18%26spage%3D10460%26epage%3D10473%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/jmcmar.2020.63.issue-18/20200924/jmcmar.2020.63.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report the first disclosure of IRAK3 degraders in the scientific literature. Taking advantage of an opportune byproduct obtained during our efforts to identify IRAK4 inhibitors, we identified ready-to-use, selective IRAK3 ligands in our compound collection with the required properties for conversion into proteolysis-targeting chimera (PROTAC) degraders. This work culminated with the discovery of PROTAC <b>23</b>, which we demonstrated to be a potent and selective degrader of IRAK3 after 16 h in THP1 cells. <b>23</b> induced proteasome-dependent degradation of IRAK3 and required both CRBN and IRAK3 binding for activity. We conclude that PROTAC <b>23</b> constitutes an excellent <i>in vitro</i> tool with which to interrogate the biology of IRAK3.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">IRAK3 (interleukin-1 receptor-associated kinase 3, also known as IRAK-M) is a class-1 pseudokinase member of the IRAK family (together with IRAK1, IRAK2, and IRAK4).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> IRAK3 signals via a noncatalytic mechanism believed to involve a scaffolding function.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Expression of IRAK3 has been shown to be predominantly restricted to leukocytes, where it is reported to suppress proinflammatory signaling in innate leukocytes (monocytes, macrophages, and neutrophils).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Genetic knockout of IRAK3 in mice led to a reprogramming of myeloid cells toward immune activation, which promoted effector T-cell proliferation, and this in turn, helped overcome immunosuppression and augmented the host response to checkpoint inhibition.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> A specific signaling role has been suggested for IRAK3 in colorectal cancer where IRAK3-deficient mice demonstrated reduced tumor burden compared to wild-type mice in a carcinogen-induced model.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Because it is a pseudokinase, it is unclear whether molecules binding to IRAK3 might have a functional effect, and to address this question, we pursued an IRAK3 degradation approach using proteolysis-targeting chimeras (PROTACs). These are heterobifunctional molecules, which bind to both an E3 ligase and a target of interest. Their mode of action involves bringing an E3 ligase into the proximity of the protein-of-interest (POI) to induce protein degradation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> This promising pharmacological approach broadens the range of druggable targets beyond those achievable by simple inhibitors, as nonfunctional domains/proteins can now be liganded to induce degradation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Many pseudokinases such as IRAK3 contain a classic and potentially druggable “ATP”-binding site amenable for small molecule targeting, yet the functions of these domains are either noncatalytic or remain unknown.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Consequently, pseudokinases may be particularly suited to targeting by the PROTAC modality. Furthermore, there was early evidence in the patent literature by GSK researchers that IRAK3 could be degraded by nonselective PROTAC molecules, thus improving confidence in the ability to degrade this target.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a></div><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Synthesis</h3><div class="NLM_p">The syntheses of pyrrolopyrimidines <b>1</b> and <b>2</b> have been previously described,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> while <b>3</b> can be obtained in a comparable fashion to the <i>trans</i> stereoisomer previously reported,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> using 1-(4-(<i>cis</i>-4-aminocyclohexyl)piperazin-1-yl)ethan-1-one instead. Likewise, <i>cis</i>-pyrrolotriazine <b>5</b> can be obtained in a similar S<sub>N</sub>Ar reaction with commercially available 4-chloro-5-isopropylpyrrolo[2,1-f][1,2,4]triazine as for the reported synthesis of <b>4</b>,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> but using the <i>cis</i> side chain. An alternative synthesis consists in installing the isopropyl group via palladium-catalyzed cross-coupling using the mixture of <i>cis</i> and <i>trans</i> stereoisomers of the bromo intermediate <b>4a</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), followed by hydrogenation of the isopropenyl group in <b>4b</b>. This was found to lead to an easier separation of the <i>cis</i> and <i>trans</i> stereoisomers <b>4</b> and <b>5</b>, albeit in lower overall yields.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. <sup>a</sup>Synthesis of Pyrrolotriazines <b>4–5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane, PdCl<sub>2</sub>(DtBPF), K<sub>3</sub>PO<sub>4</sub>, dioxane/water, 80 °C, 2 h (99%); (b) H<sub>2</sub>, Pd/C, MeOH, RT, 16 h (<b>4</b>, 11%; <b>5</b>, 15%).</p></p></figure><div class="NLM_p">The syntheses of all PROTAC analogues was achieved using the key <i>cis</i> amine <b>6c</b>, itself obtained via a similar sequence involving the reductive amination of Cbz-protected 4-aminocyclohexanone and <i>N</i>-Boc piperazine, separation of the stereoisomers, removal of the Cbz protection, and benzotriazol-1-yl-oxytripyrrolidinophosphonium hexafluorophosphate (PyBOP) coupling to 5-bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-ol to yield the <i>cis</i> bromo intermediate <b>6a</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Introduction of the 5-<i>i</i>Pr group was performed as described above (<b>6b</b>), and Boc deprotection afforded <b>6c</b> in good overall yield (51% over three steps). PROTAC molecules <b>6–11</b> were made via standard hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU) couplings between <b>6c</b> and the corresponding carboxylic acids. The syntheses of <b>12–24</b> required the installation of a 4-amino-4-oxobutanoate spacer, which was obtained through coupling with dihydrofuran-2,5-dione to yield carboxylic acid <b>12a</b>. Standard amide couplings with the required amines afforded the desired molecules. Polymer-supported proteomics probe <b>25</b> was synthesized in a similar way by coupling <b>12a</b> with <i>tert</i>-butyl (14-amino-3,6,9,12-tetraoxatetradecyl)carbamate, subsequent Boc deprotection, and final coupling onto proprietary beads.<named-content content-type="anchor" rid="sch1" type="simple"></named-content><named-content content-type="anchor" rid="sch2" type="simple"></named-content></div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. <sup>a</sup>Synthesis of Pyrrolotriazine PROTACs <b>6–24</b> and Proteomics Probe <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaBH(OAc)<sub>3</sub>, AcOH, RT, 3 d (32%); (ii) H<sub>2</sub>, Pd/C, MeOH, RT, 1 h (92%); (b) 5-bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-ol, PyBOP, DIPEA, DMF, RT to 40 °C, 24 h (88%); (c) 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane, PdCl<sub>2</sub>(DtBPF), K<sub>2</sub>CO<sub>3</sub>, dioxane/water, 90 °C, 3 h (60%); (d) (i) H<sub>2</sub>, Pd/C, THF, RT, 3 d (86%); (ii) 4 M HCl, dioxane/DCM, RT, 2 h (99%); (e) carboxylic acid, HATU, DMF, RT, 3 h (6–53%); (f) dihydrofuran-2,5-dione, DIPEA, DCM, 50 °C 1 h (80%); (g) amine, HATU, DMF, RT, 3 h (24–47%); (h) (i) <i>tert</i>-butyl (14-amino-3,6,9,12-tetraoxatetradecyl)carbamate, HATU, DIPEA, DMF, RT, 18 h (82%); (ii) HCl, MeOH, RT, 1 h (29%); (i) beads, NEt<sub>3</sub>, DMSO, RT, 18 h.</p></p></figure></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In our search for selective ligands of IRAK3, we elected to mine the AstraZeneca compound collection in the hope to find suitable starting points. As we have been conducting a number of kinase projects over three decades, we have access to a wealth of kinase data, ranging from project-specific assays to counter screens run at external companies. Activity against IRAK3 was available in these data sets, and mining the data enabled us to identify a number of valid starting points. In selecting potential starting points, we paid particular attention to general kinome selectivity, together with selectivity within the IRAK family, especially against IRAK4. Activity against IRAK3 does not necessarily correlate with activity against IRAK4, and the number of our molecules tested that showed strong binding activity against IRAK3 with both weak activity against IRAK4 and low overall hit frequency was small. An example is given in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, using IRAK3 and IRAK4 single concentration data (displacement assay performed at Thermo Fisher) as part of routine kinome selectivity screening. Ironically, one particularly interesting IRAK3 selective hit came from a series that we previously optimized against IRAK4, but in the form of a byproduct isolated in the synthesis of advanced lead pyrrolopyrimidine <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In the course of our IRAK4 program, we were only interested in the <i>trans</i> stereoisomer <b>1</b>, but fortunately we also isolated and fully characterized the undesired <i>cis</i> stereoisomer <b>2</b>, resulting from traces of the <i>cis</i> product in the reductive amination step to make the C4 side chain. While <b>1</b> had a pan-IRAK profile,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> we were surprised to observe that <b>2</b> conserved most of its IRAK3 activity, while losing its activity against IRAK4 and IRAK1. This observation led us to mine this series further and generate additional IRAK3 single-concentration data. We subsequently identified existing <i>cis</i>-pyrrolopyrimidine <b>3</b> as having a slightly higher affinity for IRAK3.</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. IRAK3 vs IRAK4 single-concentration data from Thermo Fisher Scientific kinome selectivity panels. Each marker represents a molecule tested at 1 μM. Colors denote a measure of hitting frequency, as calculated by pBSF<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (red, pBSF > 3; green, pBSF ≤ 3; gray, data unavailable). Labels refer to compound numbers used in this manuscript.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IRAK3 Ligands<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0012.gif" alt="" id="GRAPHIC-d7e628-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">core</th><th class="colsep0 rowsep0" align="center">stereo-chemistry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">IRAK4 IC<sub>50</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">IRAK3 IC<sub>50</sub><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">IRAK1 IC<sub>50</sub><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">A2B <i>P</i><sub>app</sub><a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center" char=".">efflux ratio<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">trans</td><td class="colsep0 rowsep0" align="left">4-THP</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="char" char=".">0.027</td><td class="colsep0 rowsep0" align="char" char=".">0.063</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">cis</td><td class="colsep0 rowsep0" align="left">4-THP</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.57</td><td class="colsep0 rowsep0" align="char" char=".">0.010</td><td class="colsep0 rowsep0" align="char" char=".">0.91<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">cis</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left"><i>N</i>Ac</td><td class="colsep0 rowsep0" align="left">0.84</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td><td class="colsep0 rowsep0" align="char" char=".">1.4<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b><a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">trans</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left"><i>N</i>Ac</td><td class="colsep0 rowsep0" align="left">0.036</td><td class="colsep0 rowsep0" align="char" char=".">0.031</td><td class="colsep0 rowsep0" align="char" char=".">0.052</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">0.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">cis</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left"><i>N</i>Ac</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.006</td><td class="colsep0 rowsep0" align="char" char=".">1.3<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">0.51</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All IC<sub>50</sub> data are expressed in micromolar (μM) and are the means of <i>n</i> ≥ 3 independent measurements. Each has a SEM ± 0.2 log units.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Enzymatic IRAK4 ([ATP] = 5 mM to 10 × <i>K</i><sub>m</sub>), measured as previously reported by our group.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">LanthaScreen Eu IRAK3 binding displacement assay using an Alexa Fluor conjugate and measured at Thermo Fisher Scientific.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Enzymatic IRAK1 ([ATP] = 25 mM to 10 × <i>K</i><sub>m</sub>), measured as previously reported by our group.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Measured using shake-flask methodology with a buffer/octanol volume ratio of 100:1.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Intrinsic permeability (10<sup>–6</sup> cm/s) using Caco2 cells.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">Efflux ratio measured in Caco2 cells.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">In the context of transforming a suitable ligand into a PROTAC molecule, <b>3</b> presented the advantage of bearing an amide group that we hypothesized should be pointing toward the solvent. This hypothesis was based on X-ray crystallographic evidence of morpholines/substituted piperazines in <i>trans</i> analogues consistently showing highly conserved binding modes to IRAK4 because of a salt bridge interaction of the basic nitrogen atom with Asp 272 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>),<a onclick="showRef(event, 'ref11 ref13 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref13 ref15">(11,13,15)</a> combined with manual docking of <i>cis</i> analogues in an IRAK3 homology model. Despite the change in stereochemistry for <i>cis</i> analogues, we anticipated a similar binding mode of the core to the hinge, with a conserved salt bridge interaction as this residue is conserved in IRAK3 (Asp 251 in the human IRAK3 sequence Q9Y616). Additionally, we assumed some protein accommodation and/or a possible boat conformation of the 1,4-<i>cis</i>-cyclohexyl motif. The cause of the resulting loss of activity within the IRAK family except for IRAK3 is unknown. However, one can speculate that long-distance differences in the protein sequences might compromise binding of the <i>cis</i> isomer to other members of the IRAK family, while IRAK3 might be more flexible and adaptable. In an attempt to identify the best possible starting point, we found that a related series of pyrrolotriazines (<b>4–5</b>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> also exhibited the same behavior, with <i>cis</i> isomer <b>5</b> showing an improved selectivity profile against the IRAK family (>100-fold selectivity vs IRAK4 and IRAK1) and the rest of the kinome (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Compound <b>5</b> also had the additional benefit of bearing one fewer hydrogen bond donor (HBD) than <b>3</b>, and this was viewed as a significant benefit from a cell permeability perspective, especially as we considered turning these ligands into larger PROTAC molecules. Compound <b>5</b> showed excellent intrinsic permeability (Caco2 <i>P</i><sub>app</sub> = 29 × 10<sup>–6</sup> cm/s) and low efflux (ER = 0.5), and thus this pre-existing molecule was selected as an ideal POI ligand for our <i>in vitro</i> investigations.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray crystallographic structure of <i>trans</i>-pyrolotriazine <b>4</b> in complex with IRAK4 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F3I">6F3I</a>), showing the <i>N</i>-acetylpiperazine pointing toward the solvent. Residues are colored based on similarity to IRAK3 (blue: identical, red: most different).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome selectivity of <b>5</b>, tested at 0.1 μM in a panel of 400 kinases. Inset: Only two kinases other than IRAK3 were identified with >50% activity: GSG2 (55%, IC<sub>50</sub> = 150 nM) and CDK11 (51%, IC<sub>50</sub> = 70 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our first wave of IRAK3 PROTAC molecules consisted of simple polyethylene glycol (PEG) linkers of various lengths connecting the amide moiety of <b>5</b> and either VHL-type (W1, <b>6–8</b>) or CRBN-type (W2, <b>9–11</b>) E3 ligase warheads (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> It is important to note that the POI ligand <b>5</b> itself did not induce degradation on its own. All derivatives maintained binding to IRAK3, albeit with a small loss (4- to 10-fold), and retained selectivity against IRAK4. Neither the linker length (3 to 5 PEG units) nor E3 warhead type appeared to influence binding beyond experimental error, suggesting that these parts of the molecules largely occupy space in solvents without interfering with the IRAK3 pseudo-ATP site. When investigated in an IRAK3 degradation assay in THP1 cells (a monocytic cancer cell line, which expresses high levels of IRAK3), CRBN-based PROTACs <b>10–11</b> showed superior degradation, and among these, a longer linker provided superior degradation at a concentration of 1 μM (<b>11</b> > <b>10</b> > <b>9</b>). All three molecules showed similar IRAK3 binding and occupied a similar property space as reflected by their measured log <i>D</i><sub>7.4</sub> and exposed polar surface area (ePSA). We therefore hypothesized that differences in degradation were a result of differences in three-dimensional conformations, bringing the ternary complex between IRAK3, PROTACs, and E3 ligase together.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>PEG-Derived Linkers</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0013.gif" alt="" id="GRAPHIC-d7e1109-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IRAK3 % deg<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">linker</th><th class="colsep0 rowsep0" align="center"><i>n</i></th><th class="colsep0 rowsep0" align="center">warhead</th><th class="colsep0 rowsep0" align="center">IRAK4 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">IRAK3 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">@1 μM</th><th class="colsep0 rowsep0" align="center" char=".">@10 μM</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">ePSA<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.006</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.021<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.051<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.033</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">7.8</td><td class="colsep0 rowsep0" align="char" char=".">0.034</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.050</td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.033</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">96</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data as per <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> unless otherwise specified. All IC<sub>50</sub> data are expressed in micromolar (μM) and are the means of <i>n</i> ≥ 2 independent measurements. Each has a SEM ± 0.2 log units.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">% degradation at a concentration of 1 or 10 μM in THP1 cells, expressed as the means of <i>n</i> = 2 independent measurements.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">ePSA (Å<sup>2</sup>) measured by the supercritical fluid chromatographic method.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">Encouraged by these initial results, we elected to introduce a bis-amide prelinker (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) because this was both synthetically enabling (allowing us to use amine containing linkers) and had also been shown to be beneficial in a number of previous PROTAC programs.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Thus, molecules <b>12–21</b> allowed us to investigate linker length with two linker types (PEG-like, L1, and full-carbon, L2) and two E3 warhead ligases. Similar IRAK3 binding was observed with PEG-like linkers <b>12–14</b>, irrespective of linker length, but their association to a VHL-type warhead (W1) failed to deliver degradation. Full-carbon linkers in <b>15–16</b> were significantly more lipophilic but showed equivalent polarity (<b>12–16</b>, 99 < ePSA < 103 Å<sup>2</sup>), and notably <b>16</b> showed enhanced binding (6 nM) in this subseries. In line with previous observations, improved IRAK3 binding did not necessarily translate into degradation, and both molecules bearing the VHL-type warhead showed no degradation. CRBN-based PROTACs <b>17–19</b> explored a lower lipophilicity space (2.0 < log <i>D</i><sub>7.4</sub> < 2.3) but showed similar binding to IRAK3 (10–22 nM). In contrast to their VHL analogues, these did show evidence of degradation, especially <b>19</b> at 1 μM (85%), while the reduced degradation at 10 μM (55%) might be indicative of a hook effect.<a onclick="showRef(event, 'ref17 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref20">(17,20)</a> The full carbon-linked CRBN-based <b>20–21</b> were also more lipophilic (in relation to <b>17–19</b>) and despite showing similar binding, led to no detectable degradation, confirming the need for the right combination of linker type, length, and warhead.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>Bis-amide-Linked PROTACs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0014.gif" alt="" id="GRAPHIC-d7e1512-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IRAK3 % deg</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">linker</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">warhead</th><th class="colsep0 rowsep0" align="center">IRAK4 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">IRAK3 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">@1 μM</th><th class="colsep0 rowsep0" align="center" char=".">@10 μM</th><th class="colsep0 rowsep0" align="center">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">ePSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.024</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="char" char=".">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.025<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.019</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.015</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="left">W1</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.006</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">>3.0</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">4.9<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">0.010</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="char" char=".">105</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.021<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="char" char=".">106</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">L1</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.022</td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="char" char=".">114</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.013</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="char" char=".">108</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">L2</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="left">W2</td><td class="colsep0 rowsep0" align="left">2.6<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">0.021</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="char" char=".">108</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Data as per <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, unless otherwise specified.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">Recent patent literature has seen the emergence of certain combinations of linkers and warheads, notably those used by others to successfully degrade a variety of targets including kinases such as EGFR.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21−25)</a> Given that some of these seemed efficient against a range of target classes, we decided to assess compounds <b>22</b> and <b>23</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), both bearing a bis-amide prelinker and a CRBN-type warhead connected through what appeared to be a privileged piperazine–piperidine combination. PROTACs <b>22</b> and <b>23</b> only differ by a single methylene unit, and while both IRAK3 and CRBN binding seemed largely unaffected, degradation seemed optimal with <b>23</b>. We were able to measure the concentration giving rise to 50% degradation (DC<sub>50</sub>) of the IRAK3 pseudokinase, which confirmed that <b>23</b> was a potent degrader (DC<sub>50</sub> = 2 nM) with a maximum degradation measured of 98%. We later made the methylated warhead variant <b>24</b> to validate that the degradation was CRBN-mediated. As expected, this methylation abrogated CRBN binding, while also retaining IRAK3 binding, and pleasingly prevented degradation of IRAK3, thus confirming the PROTAC mode of action.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. <a class="ref internalNav" href="#t4fn1" aria-label="a">a</a>Optimized Linkers</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0015.gif" alt="" id="GRAPHIC-d7e1984-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">IRAK4 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">IRAK3 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">IRAK1 IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">CRBN IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">IRAK3 DC<sub>50</sub> (<i>D</i><sub>max</sub>)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">ePSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">5.6<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">0.006<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">5.6<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">0.052 (74%)</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">114</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left">0.46</td><td class="colsep0 rowsep0" align="left">0.002 (98%)</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">113</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">>1</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="left">>12</td><td class="colsep0 rowsep0" align="left">>10 (-)</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">97</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Data as per <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, unless otherwise specified.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Concentration causing 50% degradation of IRAK3/maximum observed degradation in THP1 cells, expressed in micromolar (μM) or % and are the means of <i>n</i> ≥ 2 independent measurements.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">These encouraging results prompted us to profile <b>23</b> further, and we observed that the overall kinome selectivity of ligand <b>5</b> was largely maintained. Only four kinases other than IRAK3 were inhibited by <b>23</b> with >50% (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), and this was confirmed by concentration responses: CDK11 (IC<sub>50</sub> = 16 nM), CDK8 (IC<sub>50</sub> = 36 nM), TRKC (IC<sub>50</sub> = 100 nM), and GSG2 (IC<sub>50</sub> = 110 nM).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinome selectivity of <b>23</b>, tested at 0.1 μM in a panel of 400 kinases. Inset: only four other kinases were identified with >50% activity: CDK11 (82%), CDK8 (69%), TRKC (60%), and GSG2 (57%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further assess the selectivity of PROTAC <b>23</b>, we conducted a chemoproteomic profiling. Polymer-supported probe <b>25</b> allowed us to estimate binding of <b>23</b> to its targets, including those highlighted in the kinome selectivity panel (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<named-content content-type="anchor" rid="fig6" type="simple"></named-content> All apparent binding constants (<i>K</i><sub>d,app</sub>) for targets highlighted in the kinome selectivity panel showed >20-fold selectivity relative to IRAK3 (<i>K</i><sub>d,app</sub> = 34 nM). Selectivity over the other IRAK members was thus orthogonally confirmed, and this increased our confidence in selecting <b>23</b> as a selective probe for additional profiling. Selective degradation was confirmed using global proteomic profiling in THP1 cells where only IRAK3 and DHCR24 were significantly decreased upon PROTAC treatment with fold changes of −1.8 and −1.5, respectively (<i>p</i> < 0.05), while changes for IRAK1 and IRAK4 were negligible (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Upon closer examination, binding to DHCR24 was found to be incomplete (<40% binding at the highest concentration, 30 μM, data available in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf" class="ext-link">Supporting Information</a>), resulting in a flat dose–response and was thus seen as potentially weaker than initially recorded. Interestingly, the previously identified IMiD-mediated neoantigen IKZF1 and GSPT1 were not degraded by <b>23</b>, as demonstrated in the global proteomics experiments (data available in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf" class="ext-link">Supporting Information</a>). Collectively, these data confirm that <b>23</b> is a potent and selective IRAK3 degrader.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proteomic profiling for <b>23</b>: (A) dose–response curves using <b>23</b> after chemoproteomics selectivity profiling showing competed proteins hits with apparent binding (<i>K</i><sub>d,app</sub>) derived from RB<sub>50</sub>; (B) volcano plot showing fold changes of protein abundance from global proteomics analysis of THP1 cells treated with 0.1 μM of <b>23</b> for 8 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further characterize IRAK3 degradation, we conducted the series of complementary degradation experiments in THP-1 cells shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. PROTAC <b>23</b> induced potent IRAK3 degradation after 24 h, with maximal degradation observed at 0.1 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). To evaluate the kinetics of degradation, we performed time course experiments using 0.1 μM of <b>23</b>. IRAK3 degradation was observed as early as 2 h and maintained up to 72 h without replenishing the cell culture medium or redosing with <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). To monitor the duration of effect on IRAK3 degradation, THP-1 cells were treated with <b>23</b> for 16 h then rinsed with phosphate-buffered saline (PBS; to remove excess compound) and replenished with media devoid of the compound. It was observed that the IRAK3 protein level fully recovered after 24 h (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). We next assessed the necessity of the ubiquitin proteasome machinery for <b>23</b> activity and target engagement. Pretreatment with the proteasome inhibitor MG132<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> rescued IRAK3 degradation using <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, lanes 2 vs 4). Additionally, MLN4924,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> the NEDD8-activating enzyme inhibitor which prevents Cullin-mediated degradation, prevented <b>23</b>-mediated IRAK3 degradation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, lanes 2 vs 6). Pretreatment with excess IRAK3 ligand <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B) or excess thalidomide (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C) prevented IRAK3 degradation, further corroborating that IRAK3-targeted degradation occurs though CRBN and IRAK3 binding, respectively. In line with the requirement for CRBN engagement, we profiled the inactive PROTAC derivative <b>24</b> as a tool for comparison, which lacks binding to CRBN but maintains potent IRAK3 binding. In contrast to <b>23</b>-induced IRAK3 degradation, the inactive PROTAC <b>24</b> was unable to degrade IRAK3 at all doses tested (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). Collectively, the data supports that <b>23</b> potently and selectively degrades IRAK3 through ubiquitin-mediated degradation, requiring binding to both IRAK3 and CRBN.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>23</b> induces potent and rapid degradation of IRAK3 in THP-1 cells: (A) cells treated with compound for 24 h at indicated doses and representative quantification of percent degradation relative to DMSO; (B) cells treated with 0.1 μM of compound for indicated times; (C) cells treated with 0.1 μM of compound for 16 h followed and harvested at indicated times after PROTAC washout.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PROTAC <b>23</b>-mediated degradation of IRAK3 in THP-1 cells is dependent on the ubiquitin proteasome pathway, IRAK3 and CRBN: (A) cells pretreated with DMSO, MG132 (10 μM),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or MLN4924 (5 μM)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (B) cells pretreated with <b>5</b> (10 μM) for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (C) cells pretreated with thalidomide (10 μM) for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (D) cells treated with <b>23</b> or <b>24</b> at indicated doses for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also assessed IRAK3 degradation on primary macrophages, derived from purifying and differentiating monocytes (CD14+) from human PBMCs. PROTAC <b>23</b> induced significant degradation of IRAK3, which was sustained for up to 32 h of treatment (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). In contrast, treatment with inactive PROTAC <b>24</b> did not show any changes in the IRAK3 expression (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A), corroborating results shown in THP1 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). In order to understand whether <b>23</b> could induce and maintain IRAK3 degradation following TLR stimulation in primary macrophages, we pretreated macrophages with either <b>23</b> or <b>24</b> and stimulated them with the TLR3/4 agonist LPS. We observed that the active PROTAC <b>23</b> but not the inactive PROTAC <b>24</b> maintained IRAK3 degradation, regardless of the duration of LPS (100 ng/mL) stimulation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B), suggesting that LPS stimulation did not affect PROTAC <b>23</b>-induced IRAK3 degradation. Taken together, these data show that PROTAC <b>23</b> can induce significant IRAK3 degradation in both TLR-stimulated and nonstimulated primary macrophages.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. PROTAC <b>23</b> degrades IRAK3 in primary macrophages independent of LPS stimulation. (A) Primary macrophages treated with DMSO, compound <b>23</b> (0.3 μM) or compound <b>24</b> (0.3 μM) for indicated times. (B) Primary macrophages identically treated as in (A), but with LPS (100 ng/mL) treatment for 8 h prior to compound treatment for a total of 8, 24, or 32 h, as indicated. Data are representative of <i>n</i> = 6 donors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06624" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06624" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We rapidly identified potent and selective IRAK3 degraders by taking advantage of an isolated byproduct in the synthesis of an IRAK4 lead. This led us to identify a pre-existing high quality IRAK3 ligand (excellent kinome selectivity, good permeability, ease of synthesis) that allowed us to rapidly generate PROTAC derivatives. A short investigation of linker sizes and types culminated with the discovery of <i>cis</i>-pyrrolotriazine <b>23</b>, with which we demonstrated CRBN-mediated degradation of IRAK3 (<i>D</i><sub>max</sub> = 98% and DC<sub>50</sub> = 2 nM, after 16 h) in THP1 cells and in primary macrophages. Further experiments are required to understand the functional consequence of rapid and potent IRAK3 degradation and elucidate any <i>in vivo</i> therapeutic advantages of IRAK3 degradation in relation to tumor burden, some of which will be published in due course. Nevertheless, to our knowledge, this represents the first disclosure of IRAK3 degraders in the scientific literature. Although not fully optimized, <b>23</b> constitutes an excellent <i>in vitro</i> tool with which to interrogate the biology of IRAK3 degradation.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16586" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16586" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Synthesis</h3><div class="NLM_p">All solvents and chemicals used were of the reagent grade. Anhydrous solvents tetrahydrofuran (THF), dichloromethane (DCM), and dimethylformamide (DMF) were purchased from Aldrich. Microwave reactions were run in a Biotage “Initiator Robot 60” (100–120/220–240 V, 50–60 Hz, 100 V A). Flash column chromatography was carried out using prepacked silica cartridges (from 4 g up to 330 g) from Grace, Redisep, or Silicycle and eluted using an Isco Companion system. Purity and characterization of compounds were established by a combination of liquid chromatography–mass spectroscopy (LC–MS) and NMR analytical techniques, and purity was >95% for all test compounds.</div><div class="NLM_p"><sup>1</sup>H NMR were recorded on a Varian INOVA (600 MHz), Varian Gemini 2000 (300 MHz), Bruker AVANCE 700 (700 MHz), Bruker AVANCE 500 (500 MHz), or Bruker AVANCE DPX400 (400 MHz) and were determined in CDCl<sub>3</sub>, dimethyl sulfoxide (DMSO)-<i>d</i><sub>6</sub>, or MeOH-<i>d</i><sub>4</sub>. Chemical shifts are reported in ppm relative to TMS (0.00 ppm) or solvent peaks as the internal reference. Splitting patterns are indicated as follows: s, singlet; d, doublet; t, triplet; q, quartet; quin, quintuplet; m, multiplet; br, broad peak. Coupling constants (<i>J</i> values) are reported in Hz.</div><div class="NLM_p">Analytical LC–MS was carried out using a suitable system, such as a Waters 2790/95 LC system with a 2996 PDA and a 2000 amu ZQ single quadrupole mass spectrometer, or a UPLC system utilizing a Waters Aquity Binary pump with sample manager, Aquity PDA, and SQD Mass spectrometer. Sample purity was assessed by LC–MS and was >95%, unless otherwise stated.</div><div class="NLM_p">Accurate mass and MSMS fragmentation data were obtained using a Thermo Scientific hybrid LTQ-FT mass spectrometer with an Agilent 1100 Quaternary pump with PDA and autosampler; 5 μL of the sample dissolved in 50:50 acetonitrile/water 0.1% formic acid was injected onto a Thermo Scientific Hypersil Gold 50 × 2.1 mm 5 μm particle LC Column and eluted with a gradient of 5 to 100% B over 17 min with 3 min re-equilibration time at 5% B. The flow rate was 0.5 mL/min, with A being 0.1% formic acid in water and B 0.1% formic acid in acetonitrile. The MS and MSMS spectra were obtained in the ESI +ve mode in both the ion trap and ion cyclotron resonance (ICR) cell using helium as the collision gas at a normalized collision energy of 35 eV. The ICR cell was run at resolution settings of 25,000 in the MS mode and 12,500 in the MSMS mode.</div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <i>cis</i>-1-(4-(4-((5-Isopropyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl)amino)cyclohexyl)piperazin-1-yl)ethanone (<b>3</b>)</h4><div class="NLM_p">The title compound can be obtained in a fashion identical to the <i>trans</i> stereoisomer already reported,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> using 1-(4-(<i>cis</i>-4-aminocyclohexyl)piperazin-1-yl)ethan-1-one instead.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.27 (br s, 1H), 8.0–8.2 (m, 1H), 6.85 (s, 1H), 5.44 (br d, 1H, <i>J</i> = 6.6), 4.29 (br s, 1H), 3.4–3.5 (m, 4H), 3.2–3.3 (m, 3H), 2.4–2.5 (m, 2H), 2.33 (br s, 1H), 1.98 (s, 5H), 1.62 (br d, 6H, <i>J</i> = 4.6), 1.30 (d, 6H, <i>J</i> = 6.7). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 168.4, 156.1, 151.6, 151.2, 121.6, 116.5, 101.8, 61.3, 49.6, 49.0, 46.6, 46.2, 41.7, 40.6, 40.4, 40.3, 28.9, 26.2, 24.5, 24.2, 21.6. HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>21</sub>H<sub>33</sub>N<sub>6</sub>O (M + H)<sup>+</sup>, 385.2710; found, 385.2699.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>cis</i>-1-(4-(4-((5-Isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl)piperazin-1-yl)ethanone (<b>5</b>)</h4><div class="NLM_p">The title compound can be obtained in a fashion identical to the <i>trans</i> stereoisomer already reported,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> using 1-(4-(<i>cis</i>-4-aminocyclohexyl)piperazin-1-yl)ethan-1-one instead.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 7.79 (s, 1H), 7.54 (d, 1H, <i>J</i> = 2.7), 6.5–6.7 (m, 1H), 6.06 (br d, 1H, <i>J</i> = 6.7), 4.30 (br d, 1H, <i>J</i> = 3.4), 3.4–3.6 (m, 5H), 2.49 (br d, 2H, <i>J</i> = 4.9), 2.4–2.5 (m, 2H), 2.32 (br s, 1H), 1.99 (s, 5H), 1.6–1.7 (m, 6H), 1.30 (d, 6H, <i>J</i> = 6.7). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 168.4, 154.5, 147.4, 124.6, 117.8, 110.8, 107.9, 60.7, 49.7, 49.1, 47.0, 46.5, 41.7, 40.6, 40.4, 40.3, 28.3, 26.5, 24.9, 24.7, 21.6. HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>21</sub>H<sub>32</sub>N<sub>6</sub>O (M + H)<sup>+</sup>, 385.2710; found, 385.2713.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>tert</i>-Butyl 4-(<i>cis</i>-4-((5-Bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl)piperazine-1-carboxylate (<b>6a</b>)</h4><div class="NLM_p">PyBOP (24.3 g, 47 mmol) was added to 5-bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-ol (5.0 g, 23 mmol), <i>tert</i>-butyl 4-(<i>cis</i>-4-aminocyclohexyl)piperazine-1-carboxylate (6.6 g, 23 mmol), and <i>N</i>, <i>N</i>-diisopropylethylamine (DIPEA) (16.3 mL, 93 mmol) in DMF (80 mL), and the resulting mixture was stirred at room temperature (RT) for 20 h and then heated to 40 °C overnight. The reaction mixture was then poured into water (500 mL) and extracted with EtOAc (3 × 300 mL). The combined organic layers were washed with brine (3 × 100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The brown residue was purified by flash silica chromatography, with the elution gradient being 0 to 5% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (9.8 g, 88%) as a yellow gum.</div><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 7.92 (d, 1H), 7.70 (d, 1H), 6.79 (d, 1H), 6.75 (d, 1H), 3.10–3.16 (m, 1H), 2.92–3.05 (m, 4H), 2.40–2.46 (m, 4H), 2.27–2.37 (m, 1H), 1.68–1.74 (m, 4H), 1.38 (s, 9H), 1.25 (t, 4H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 479.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>tert</i>-Butyl 4-[<i>cis</i>-4-((5-Isopropylpyrrolo[2,1-f][1,2,4]triazin-4-yl)amino)cyclohexyl]-1-piperazinecarboxylate (<b>6b</b>)</h4><div class="NLM_p">PdCl<sub>2</sub>(DtBPF) (660 mg, 1.01 mmol) was added to a solution of <i>tert</i>-butyl 4-(<i>cis</i>-4-((5-bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl)piperazine-1-carboxylate (9.7 g, 20 mmol), 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane (5.0 g, 30 mmol), and K<sub>2</sub>CO<sub>3</sub> (5.6 g, 40 mmol) in dioxane (160 mL)/water (40 mL) at RT under nitrogen. The resulting mixture was stirred at 90 °C for 3 h, after which the reaction mixture was quenched with water (200 mL), and extracted with EtOAc (3 × 200 mL). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness; the residue was purified by flash silica chromatography, with the elution gradient being 0 to 5% MeOH in DCM. Pure fractions were evaporated to dryness to afford <i>tert</i>-butyl 4-(<i>cis</i>-4-((5-(prop-1-en-2-yl)pyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl)piperazine-1-carboxylate (5.4 g, 60%) as a brown gum that was used without further purification. <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 441.</div><div class="NLM_p">A solution of <i>tert</i>-butyl 4-(<i>cis</i>-4-((5-(prop-1-en-2-yl)pyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl)piperazine-1-carboxylate (5.2 g, 12 mmol) and Pd–C (2.5 g, 2.4 mmol) in THF (10 mL) was stirred under an atmosphere of hydrogen at 1 atm and RT for 3 d. The mixture was then filtered through Celite and washed with MeOH (500 mL), and the solvent was removed under reduced pressure, and the residue was purified by flash silica chromatography, with the elution gradient being 0 to 4% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (6.8 g, 86%) as a yellow foam.</div><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, 300 K): 7.81 (s, 1H), 7.42 (d, 1H), 6.49 (d, 1H), 5.56 (d, 1H), 4.40–4.47 (m, 1H), 3.44 (t, 4H), 3.19 (p, 1H), 2.50 (t, 4H), 2.36 (t, 1H), 1.91–2.07 (m, 2H), 1.71–1.81 (m, 4H), 1.52–1.66 (m, 2H), 1.46 (s, 9H), 1.40 (d, 6H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 443.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 5-Isopropyl-<i>N</i>-(<i>cis</i>-4-(1-piperazinyl)cyclohexyl)pyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-amine (<b>6c</b>)</h4><div class="NLM_p">HCl (4 M in dioxane, 51 mL, 203 mmol) was added to a solution of <i>tert</i>-butyl 4-[<i>cis</i>-4-((5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino)cyclohexyl]-1-piperazinecarboxylate (6 g, 13.6 mmol) in DCM (40 mL) at RT. The resulting mixture was stirred at RT for 2 h, and the precipitate was collected by filtration, washed with DCM (50 mL), and dried under vacuum to afford the title amine (6.0 g, 99%) as a colorless solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.74 (br s, 1H), 10.0–10.4 (m, 2H), 8.00 (s, 1H), 7.8–7.9 (m, 1H), 7.21 (br s, 1H), 6.77 (br s, 1H), 4.56 (br d, 1H, <i>J</i> = 3.5), 3.4–3.9 (m, 10H), 2.51 (br s, 1H), 2.34 (br d, 2H, <i>J</i> = 6.4), 1.8–2.0 (m, 5H), 1.28 (br d, 6H, <i>J</i> = 6.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 148.4, 138.1, 133.4, 123.3, 110.7, 108.0, 66.8, 62.7, 48.4, 45.8, 26.5, 26.4, 24.2, 22.5. <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 343.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure for the Synthesis of Molecules <b>6–11</b></h3><div class="NLM_p">To a solution of 5-isopropyl-<i>N</i>-(<i>cis</i>-4-(1-piperazinyl)cyclohexyl)pyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-amine, 2 HCl (8.3 mg, 20 μmol), a carboxylic acid (20 μmol) and DIPEA (7.8 mg, 60 μmol) in DMF (0.5 mL) was added HATU (11.4 mg, 30 μmol) in DMF (0.2 mL). The resulting solution was stirred at RT for 3 h, after which the reaction mixture was purified by reverse-phase high-performance liquid chromatography (HPLC) (C18 column, 0.1% formic acid in H<sub>2</sub>O/MeCN, 13–50%, 11 min). Pure fractions were evaporated under reduced pressure to give desired final products.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[2-[2-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-2-oxo-ethoxy]ethoxy]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (8.3 mg, 40%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (s, 1H), 8.59 (t, 1H, <i>J</i> = 6.1), 8.15 (s, 1H), 7.79 (s, 1H), 7.54 (d, 1H, <i>J</i> = 1.3), 7.4–7.5 (m, 5H), 6.55 (d, 1H, <i>J</i> = 2.4), 6.0–6.1 (m, 1H), 4.57 (d, 1H, <i>J</i> = 9.5), 4.3–4.5 (m, 3H), 4.2–4.3 (m, 2H), 4.12 (s, 2H), 3.97 (s, 2H), 3.7–3.7 (m, 1H), 3.5–3.6 (m, 14H), 3.4–3.5 (m, 5H), 3.35–3.25 (m, 2H), 3.17 (s, 1H), 2.90 (s, 1H), 2.74 (s, 1H), 2.44 (s, 1H), 2.31 (br s, 1H), 2.0–2.1 (m, 1H), 1.9–2.0 (m, 3H), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.8), 0.94 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>51</sub>H<sub>75</sub>N<sub>10</sub>O<sub>9</sub>S (M + H)<sup>+</sup>, 1003.5434; found, 1003.5452.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[2-[2-[2-[4-[<i>cis</i>-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-2-oxo-ethoxy]ethoxy]ethoxy]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>7</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (1.2 mg, 6%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (s, 1H), 8.60 (t, 1H, <i>J</i> = 6.1), 8.38 (br s, 1H), 7.79 (s, 1H), 7.54 (s, 1H), 7.4–7.5 (m, 5H), 6.5–6.6 (m, 1H), 6.0–6.1 (m, 1H), 5.16 (br s, 1H), 4.57 (d, 1H, <i>J</i> = 9.5), 4.3–4.5 (m, 3H), 4.2–4.3 (m, 2H), 4.1–4.2 (m, 2H), 3.97 (s, 2H), 3.4–3.6 (m, 28H), 2.4–2.5 (m, 2H), 2.3–2.4 (m, 1H), 1.97 (br d, 4H, <i>J</i> = 3.3), 1.6–1.7 (m, 5H), 1.29 (br d, 6H, <i>J</i> = 6.6), 0.94 (s, 9H).</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[2-[2-[2-[2-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-2-oxo-ethoxy]ethoxy]ethoxy]ethoxy]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>8</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (11.4 mg, 52%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (s, 1H), 8.60 (t, 1H, <i>J</i> = 6.0), 7.79 (s, 1H), 7.54 (s, 1H), 7.4–7.5 (m, 5H), 6.55 (d, 1H, <i>J</i> = 2.6), 5.9–6.2 (m, 1H), 4.57 (d, 1H, <i>J</i> = 9.5), 4.2–4.5 (m, 5H), 4.13 (s, 2H), 3.97 (s, 2H), 3.4–3.7 (m, 31H), 2.40–2.61 (m, 4H), 2.31 (br s, 1H), 1.9–2.1 (m, 4H), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.6), 0.92 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>55</sub>H<sub>83</sub>N<sub>10</sub>O<sub>11</sub>S (M + H)<sup>+</sup>, 1091.5958; found, 1091.5998.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-[2,6-Dioxo-3-piperidyl]-4-[2-[2-[2-[3-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-3-oxo-propoxy]ethoxy]ethoxy]ethylamino]isoindoline-1,3-dione (<b>9</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (6.9 mg, 42%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.10 (s, 1H), 7.79 (s, 1H), 7.58 (t, 1H, <i>J</i> = 7.8), 7.54 (s, 1H), 7.14 (d, 1H, <i>J</i> = 8.4), 7.04 (d, 1H, <i>J</i> = 7.0), 6.5–6.7 (m, 2H), 5.9–6.2 (m, 1H), 5.06 (dd, 1H, <i>J</i> = 5.3, 12.8), 4.28 (br s, 1H), 3.4–3.7 (m, 18H), 3.17 (s, 1H), 2.8–3.0 (m, 2H),2.63–2.45 (m, 5H), 2.5–2.7 (m, 1H), 2.41 (br s, 1H), 2.30 (br s, 1H), 1.9–2.1 (m, 3H), 1.5–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.6). HRMS (ESI+): anal. calcd for C<sub>41</sub>H<sub>56</sub>N<sub>9</sub>O<sub>8</sub> (M + H)<sup>+</sup>, 802.4246; found, 802.4252.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2-[2,6-Dioxo-3-piperidyl]-4-[2-[2-[2-[2-[3-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-3-oxo-propoxy]ethoxy]ethoxy]ethoxy]ethylamino]isoindoline-1,3-dione (<b>10</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (9.2 mg, 53%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.10 (s, 1H), 7.79 (s, 1H), 7.58 (t, 1H, <i>J</i> = 7.7), 7.54 (s, 1H), 7.15 (d, 1H, <i>J</i> = 8.6), 7.04 (d, 1H, <i>J</i> = 7.0), 6.6–6.7 (m, 1H), 6.55 (d, 1H, <i>J</i> = 2.0), 6.04 (br d, 1H, <i>J</i> = 5.3), 4.9–5.2 (m, 1H), 4.29 (br s, 1H), 3.4–3.7 (m, 22H), 3.17 (s, 1H), 2.90 (s, 2H), 2.63–2.45 (m, 6H), 2.4–2.5 (m, 1H), 2.3–2.3 (m, 1H), 1.97 (br s, 3H), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.6). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>43</sub>H<sub>60</sub>N<sub>9</sub>O<sub>9</sub> (M + H)<sup>+</sup>, 846.4509; found, 846.4503.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2-[2,6-Dioxo-3-piperidyl]-4-[2-[2-[2-[2-[2-[3-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-3-oxo-propoxy]ethoxy]ethoxy]ethoxy]ethoxy]ethylamino]isoindoline-1,3-dione (<b>11</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (27 mg, 58%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.09 (s, 1H), 7.79 (s, 1H), 7.59 (dd, 1H, <i>J</i> = 7.2, 8.3), 7.54 (d, 1H, <i>J</i> = 2.6), 7.15 (d, 1H, <i>J</i> = 8.7), 7.04 (d, 1H, <i>J</i> = 7.0), 6.61 (t, 1H, <i>J</i> = 5.7), 6.55 (d, 1H, <i>J</i> = 2.7), 6.05 (br d, 1H, <i>J</i> = 6.9), 5.06 (dd, 1H, <i>J</i> = 5.4, 12.7), 4.2–4.4 (m, 1H), 3.4–3.7 (m, 27H), 2.8–3.0 (m, 2H), 2.6–2.7 (m, 3H), 2.3–2.5 (m, 5H), 1.97 (br d, 3H, <i>J</i> = 5.8), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.7). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>45</sub>H<sub>64</sub>N<sub>9</sub>O<sub>10</sub> (M + H)<sup>+</sup>, 890.4771; found, 890.4769.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure for the Synthesis of Molecules <b>12–23</b></h3><div class="NLM_p">To a solution of 4-(4-(<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl)-1-piperazinyl)-4-oxobutanoic acid (10.3 mg, 20 μmol), an amine (20 μmol), and DIPEA (10.3 mg, 80 μmol) in DMF (0.5 mL), was added HATU (11.4 mg, 30 μmol) in DMF (0.2 mL). The resulting solution was stirred at RT for 3 h, after which the reaction mixture was purified by reverse-phase HPLC (C18 column, 0.1% formic acid in H<sub>2</sub>O/MeCN, 10–50%, 11 min). Pure fractions were evaporated under reduced pressure to give desired final products.</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-(4-(<i>cis</i>-4-[(5-Isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl)-1-piperazinyl)-4-oxobutanoic Acid (<b>12a</b>)</h4><div class="NLM_p">A solution of 5-isopropyl-<i>N</i>-(<i>cis</i>-4-(1-piperazinyl)cyclohexyl)pyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-amine, 2 HCl (150 mg, 0.4 mmol), DIPEA (0.207 mL, 1.2 mmol) and dihydrofuran-2,5-dione (40 mg, 0.4 mmol) in DCM (8 mL) was heated at 50 °C for 1 h. The solvent was then removed under reduced pressure, and the crude residue was purified by reverse-phase HPLC (C18 column, 01% formic acid in H<sub>2</sub>O/MeCN, 25 min). Pure fractions were lyophilized to give the title compound (140 mg, 80%) as a colorless solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 7.79 (s, 1H), 7.54 (d, <i>J</i> = 2.7, 1H), 6.53–6.60 (m, 1H), 6.06 (br d, <i>J</i> = 6.7, 1H), 4.24–4.39 (m, 1H), 3.41–3.56 (m, 6H), 2.52 (br s, 3H), 2.38–2.46 (m, 5H), 2.33 (br d, <i>J</i> = 6.1, 1H), 1.94–2.05 (m, 2H), 1.57–1.71 (m, 6H), 1.30 (d, <i>J</i> = 6.7, 6H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 443.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[[4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>12</b>)</h4><div class="NLM_p">The title compound was obtained as a colorless solid (6.5 mg, 29%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.9–9.1 (m, 1H), 8.59 (t, 1H, <i>J</i> = 6.0), 7.89 (t, 1H, <i>J</i> = 5.6), 7.7–7.8 (m, 1H), 7.54 (s, 1H), 7.45 (br d, 1H, <i>J</i> = 9.5),7.39 (d, 4H, <i>J</i> = 1.7), 6.5–6.7 (m, 1H), 6.05 (br d, 1H, <i>J</i> = 5.7), 4.6–4.7 (m, 1H), 4.45 (br t, 1H, <i>J</i> = 7.7), 4.3–4.4 (m, 2H), 4.2–4.3 (m, 2H), 3.98 (s, 2H),3.46 (br s, 14H), 3.2–3.3 (m, 1H), 3.17 (s, 3H), 2.55–2.60 (m, 3H), 2.4–2.5 (m, 2H), 2.32 (br t, 3H, <i>J</i> = 7.0), 2.0–2.1 (m, 1H), 1.9–2.0 (m, 3H), 1.6–1.7 (m, 6H), 1.29 (br d, 6H, <i>J</i> = 6.6), 0.92–1.0 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>51</sub>H<sub>74</sub>N<sub>11</sub>O<sub>8</sub>S (M + H)<sup>+</sup>, 1000.5437; found,1000.5453.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[2-[[4-[4-[4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]ethoxy]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>13</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (6 mg, 26%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (d, 1H, <i>J</i> = 1.8), 8.6–8.6 (m, 1H), 7.87 (br t, 1H, <i>J</i> = 5.7), 7.78 (d, 1H, <i>J</i> = 1.7), 7.54 (s, 1H), 7.4–7.5 (m, 5H),6.55 (s, 1H), 6.05 (br d, 1H, <i>J</i> = 5.9), 4.57 (br d, 1H, <i>J</i> = 9.5), 4.3–4.5 (m, 3H), 4.2–4.3 (m, 2H), 3.9–4.0 (m, 2H), 3.5–3.7 (m, 22H), 3.1–3.2 (m, 3H), 2.62–2.49 (m, 2H), 2.4–2.5 (m, 2H), 2.31 (br t, 3H, <i>J</i> = 6.7), 2.0–2.1 (m, 1H), 1.9–2.0 (m, 3H), 1.6–1.7 (m, 6H), 1.29 (br d, 6H, <i>J</i> = 6.6), 0.91 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>53</sub>H<sub>78</sub>N<sub>11</sub>O<sub>9</sub>S (M + H)<sup>+</sup>, 1044.5699; found, 1044.5707.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[[2-[2-[2-[2-[2-[[4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]ethoxy]ethoxy]ethoxy]ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>14</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (6.5 mg, 28%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (s, 1H), 8.60 (t, 1H, <i>J</i> = 6.1), 7.87 (br t, 1H, <i>J</i> = 5.5), 7.79 (s, 1H), 7.54 (s, 1H), 7.4–7.5 (m, 5H), 6.55 (s, 1H), 6.05 (br d, 1H, <i>J</i> = 6.1), 5.5–5.6 (m, 1H), 4.57 (d, 1H, <i>J</i> = 9.5), 4.3–4.5 (m, 3H), 4.2–4.3 (m, 2H), 3.97 (s, 2H), 3.4–3.7 (m, 19H), 3.30–3.44 (m, 3H), 3.1–3.2 (m, 3H), 2.44–2.60 (m, 5H), 2.4–2.4 (m, 1H), 2.31 (br t, 3H, <i>J</i> = 6.9), 2.05 (br d, 1H, <i>J</i> = 7.5), 1.9–2.0 (m, 3H), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.6), 0.9–1.0 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>55</sub>H<sub>82</sub>N<sub>11</sub>O<sub>10</sub>S (M + H)<sup>+</sup>, 1088.5961; found, 1088.5958.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[4-[[4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]butanoylamino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>15</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (4.4 mg, 22%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (s, 1H), 8.56 (t, 1H, <i>J</i> = 6.1), 7.87 (d, 1H, <i>J</i> = 9.2), 7.8–7.8 (m, 2H), 7.54 (s, 1H), 7.3–7.5 (m, 4H), 6.5–6.7 (m, 1H), 6.05 (br d, 1H, <i>J</i> = 6.2), 4.2–4.7 (m, 6H), 3.6–3.8 (m, 3H), 3.30–3.50 (m, 6H), 3.17 (s, 1H), 2.9–3.1 (m, 2H), 2.50–2.64 (m, 6H), 2.4–2.5 (m, 1H), 2.2–2.3 (m, 4H), 2.14 (br d, 1H, <i>J</i> = 7.2), 1.9–2.1 (m, 4H), 1.6–1.7 (m, 8H), 1.29 (d, 6H, <i>J</i> = 6.6), 0.94 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>49</sub>H<sub>70</sub>N<sub>11</sub>O<sub>6</sub>S (M + H)<sup>+</sup>, 940.5226; found, 940.5233.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-[7-[[4-[4-[4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]heptanoylamino]-3,3-dimethyl-butanoyl]-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>16</b>)</h4><div class="NLM_p">The title compound was obtained as a yellow solid (5.2 mg, 25%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.98 (d, 1H, <i>J</i> = 1.8), 8.56 (t, 1H, <i>J</i> = 6.0), 7.84 (br d, 1H, <i>J</i> = 9.4), 7.7–7.8 (m, 2H), 7.54 (s, 1H), 7.4–7.4 (m, 2H), 7.4–7.4 (m, 2H), 6.55 (s, 1H), 6.05 (br d, 1H, <i>J</i> = 5.1), 4.54 (br d, 1H, <i>J</i> = 9.4), 4.42 (br t, 2H, <i>J</i> = 8.0), 4.35 (br s, 1H), 4.29 (br s, 1H), 4.22 (br dd, 1H, <i>J</i> = 5.3, 15.8), 3.65 (br d, 2H, <i>J</i> = 2.0), 3.30–3.51 (m, 6H), 3.17 (s, 2H), 3.0–3.1 (m, 3H), 2.50.2.65 (m, 6H), 2.4–2.4 (m, 1H), 2.2–2.3 (m, 3H), 2.1–2.2 (m, 1H), 1.9–2.1 (m, 4H), 1.6–1.7 (m, 6H), 1.4–1.5 (m, 2H), 1.35 (br d, 2H, <i>J</i> = 6.2), 1.29 (d, 6H, <i>J</i> = 6.6), 1.23 (br s, 4H), 0.93 (s, 9H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>52</sub>H<sub>76</sub>N<sub>11</sub>O<sub>6</sub>S (M + H)<sup>+</sup>, 982.5695; found, 982.5700.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-[2-[2-[2-[[2-(2,6-Dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl]amino]ethoxy]ethoxy]ethyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>17</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (5.3 mg, 31%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 10.9–11.4 (m, 1H), 7.84 (br t, 1H, <i>J</i> = 5.5), 7.78 (s, 1H), 7.59 (t, 1H, <i>J</i> = 7.9), 7.54 (d, 1H, <i>J</i> = 2.6), 7.15 (d, 1H, <i>J</i> = 8.6), 7.04 (d, 1H, <i>J</i> = 7.0), 6.61 (br t, 1H, <i>J</i> = 5.8), 6.55 (d, 1H, <i>J</i> = 2.4), 6.05 (br d, 1H, <i>J</i> = 6.8), 5.06 (dd, 1H, <i>J</i> = 5.4, 12.7), 4.29 (br s, 1H), 3.62 (t, 2H, <i>J</i> = 5.4), 3.5–3.6 (m, 2H), 3.5–3.5 (m, 2H), 3.49–3.41 (m, 9H), 3.4–3.5 (m, 1H), 3.18 (q, 2H, <i>J</i> = 5.9), 2.50–2.65 (m,6H), 2.8–2.9 (m, 1H), 2.41 (br s, 1H), 2.30 (br t, 3H, <i>J</i> = 7.0), 1.97 (br d, 3H, <i>J</i> = 5.3), 1.6–1.7 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.8). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>42</sub>H<sub>57</sub>N<sub>10</sub>O<sub>8</sub> (M + H)<sup>+</sup>, 829.4355; found, 829.4361.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-[2-[2-[2-[2-[[2-(2,6-Dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl]amino]ethoxy]ethoxy]ethoxy]ethyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>18</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (5.8 mg, 33%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.09 (s, 1H), 7.85 (br t, 1H, <i>J</i> = 5.5), 7.79 (s, 1H), 7.59 (t, 1H, <i>J</i> = 8.0), 7.54 (s, 1H), 7.15 (br d, 1H, <i>J</i> = 8.6), 7.04 (d, 1H, <i>J</i> = 7.0), 6.6–6.6 (m, 1H), 6.55 (s, 1H), 6.06 (br d, 1H, <i>J</i> = 6.2), 5.06 (dd, 1H, <i>J</i> = 5.2, 12.7), 4.29 (br s, 1H), 3.4–3.7 (m, 19H), 3.17 (br d, 2H, <i>J</i> = 5.7), 2.8–3.0 (m, 3H), 2.50–2.65 (m, 4H), 2.42 (br s, 1H), 2.2–2.4 (m, 3H), 1.98 (br s, 3H), 1.5–1.7 (m, 6H), 1.29 (br d, 6H, <i>J</i> = 6.6). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>44</sub>H<sub>61</sub>N<sub>10</sub>O<sub>9</sub> (M + H)<sup>+</sup>, 873.4618; found, 873.4620.</div></div><div id="sec4_8_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-[2-[2-[2-[2-[2-[[2-(2,6-Dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl]amino]ethoxy]ethoxy]ethoxy]ethoxy]ethyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>19</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (6.2 mg, 32%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.09 (s, 1H), 7.85 (br s, 1H), 7.79 (s, 1H), 7.59 (dd, 1H, <i>J</i> = 7.3, 8.4), 7.55 (br d, 1H, <i>J</i> = 2.1), 7.15 (d, 1H, <i>J</i> = 8.5), 7.05 (d, 1H, <i>J</i> = 7.0), 6.61 (t, 1H, <i>J</i> = 5.9), 6.56 (d, 1H, <i>J</i> = 2.4), 6.05 (br d, 1H, <i>J</i> = 1.4), 5.0–5.1 (m, 1H), 4.3–4.4 (m, 1H), 3.63 (t, 2H, <i>J</i> = 5.5), 3.6–3.6 (m, 2H), 3.4–3.6 (m, 17H), 3.38 (t, 2H, <i>J</i> = 6.0), 3.1–3.2 (m, 2H), 2.8–3.0 (m, 1H), 2.64 (s, 1H), 2.5–2.6 (m, 5H), 2.37 (s, 2H), 2.32 (br s, 3H), 1.9–2.1 (m, 3H), 1.63 (br d, 6H, <i>J</i> = 7.3), 1.30 (d, 6H, <i>J</i> = 6.6). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>46</sub>H<sub>65</sub>N<sub>10</sub>O<sub>10</sub> (M + H)<sup>+</sup>, 917.4880; found, 917.4885.</div></div><div id="sec4_8_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-[4-[[2-(2,6-Dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl]amino]butyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>20</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (5.7 mg, 35%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.10 (br s, 1H), 7.83 (t, 1H, <i>J</i> = 5.6), 7.78 (d, 1H, <i>J</i> = 1.1), 7.58 (t, 1H, <i>J</i> = 7.8), 7.54 (s, 1H), 6.5–6.6 (m, 2H), 6.05 (br d, 1H, <i>J</i> = 4.0), 5.05 (br dd, 1H, <i>J</i> = 5.2, 12.7), 4.29 (br d, 1H, <i>J</i> = 3.1), 3.48 (br s, 5H), 3.31–3.45 (m, 5H), 3.30 (q, 1H, <i>J</i> = 6.2), 3.17 (s, 1H), 3.07 (q, 2H, <i>J</i> = 6.4), 2.8–2.9 (m, 1H), 2.42–2.60 (m, 4H), 2.40 (br s, 1H), 2.3–2.3 (m, 3H), 1.9–2.1 (m, 3H), 1.5–1.7 (m, 8H), 1.4–1.5 (m, 2H), 1.29 (d, 6H, <i>J</i> = 6.6). HRMS (ESI<sup>+</sup>): anal. calcd for C40H53N10O6 (M + H)<sup>+</sup>, 769.4144; found, 769.4128.</div></div><div id="sec4_8_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-[6-[[2-(2,6-Dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl]amino]hexyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>21</b>)</h4><div class="NLM_p">The title compound was obtained as a gum (5.4 mg, 33%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.10 (s, 1H), 7.7–7.8 (m, 2H), 7.58 (t, 1H, <i>J</i> = 7.8), 7.54 (d, 1H, <i>J</i> = 2.2), 7.09 (d, 1H, <i>J</i> = 8.6), 7.02 (d, 1H, <i>J</i> = 7.2), 6.4–6.7 (m, 2H), 6.05 (br d, 1H, <i>J</i> = 6.8), 4.9–5.2 (m, 1H), 4.29 (br s, 1H), 3.36–3.46 (m, 6H), 3.29 (br d, 1H, <i>J</i> = 6.6), 3.17 (s, 1H), 3.02 (q, 2H, <i>J</i> = 6.5), 2.88 (br s, 1H), 2.42–2.56 (m, 6H) 2.40 (br s, 1H), 2.3–2.3 (m, 3H), 1.97 (br s, 3H), 1.5–1.7 (m, 8H), 1.3–1.4 (m, 12H). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>42</sub>H<sub>57</sub>N<sub>10</sub>O<sub>6</sub> (M + H)<sup>+</sup>,797.4457; found, 797.4454.</div></div><div id="sec4_8_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-(2,6-Dioxo-3-piperidyl)-5-[4-[[1-[4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]-4-piperidyl]methyl]piperazin-1-yl]isoindoline-1,3-dione (<b>22</b>)</h4><div class="NLM_p">The title compound was obtained as a colorless solid (9.3 mg, 47%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.10 (s, 1H), 7.83 (s, 1H), 7.78 (d, 1H, <i>J</i> = 8.3), 7.59 (d, 1H, <i>J</i> = 1.7), 7.5–7.5 (m, 1H), 7.37 (br d, 1H, <i>J</i> = 8.6), 7.2–7.2 (m, 1H), 6.59 (d, 1H, <i>J</i> = 2.2), 5.96 (br d, 1H, <i>J</i> = 5.1), 5.10 (dd, 1H, <i>J</i> = 5.3, 12.8), 4.4–4.6 (m, 1H), 4.34 (br s, 2H), 4.1–4.3 (m, 1H), 3.94 (br d, 1H, <i>J</i> = 13.0), 3.6–3.8 (m, 1H), 3.5–3.6 (m, 2H), 3.4–3.5 (m, 2H), 3.35–3.36 9 M, (10 m), 3.1–3.2 (m, 1H), 3.05 (br t, 2H, <i>J</i> = 11.6), 2.9–2.9 (m, 1H), 2.60 (br d, 2H, <i>J</i> = 16.9), 2.40–252 (m, 5H), 2.21 (br d, 2H, <i>J</i> = 11.7), 2.0–2.1 (m, 1H), 1.97 (br s, 2H), 1.6–1.8 (m, 6H), 1.32 (d, 7H, <i>J</i> = 6.6), 1.16 (br d, 1H, <i>J</i> = 10.8), 1.01 (br d, 1H, <i>J</i> = 10.1). HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>46</sub>H<sub>62</sub>N<sub>11</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 864.4879; found, 864.4871.</div></div><div id="sec4_8_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>tert</i>-Butyl 4-(2-(4-(2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)piperazin-1-yl)ethyl)piperidine-1-carboxylate (<b>23a</b>)</h4><div class="NLM_p">Sodium acetate (295 mg, 3.6 mmol) was added in one portion to a stirred solution of <i>tert</i>-butyl 4-(2-oxoethyl)piperidine-1-carboxylate (273 mg, 1.2 mmol) and 2-(2,6-dioxo-3-piperidyl)-5-piperazin-1-yl-isoindoline-1,3-dione·HCl (500 mg, 1.3 mmol) in MeOH (2 mL) and DCM (10 mL) at 20 °C under air. The resulting suspension was stirred for 15 min at RT, and sodium cyanotrihydroborate (226 mg, 3.6 mmol) was added and the reaction mixture stirred at RT for 18 h. The reaction mixture was partitioned between EtOAc (100 mL) and water (40 mL). The organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash silica chromatography, with the elution gradient being 0 to 100% EtOAc in heptane and then 10% EtOH in EtOAc. Pure fractions were evaporated to dryness to afford the title compound (350 mg, 53%) as a yellow dry film.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 300 K): 7.91 (s, 1H), 7.69 (d, <i>J</i> = 8.6, 1H), 7.28 (d, <i>J</i> = 2.1, 1H), 7.03–7.09 (m, 1H), 4.94 (dd, <i>J</i> = 12.3, 5.4, 1H), 4.12 (q, <i>J</i> = 7.2, 2H), 3.43 (s, 4H), 2.64–2.94 (m, 5H), 2.59 (s, 4H), 2.44 (s, 2H), 2.13 (d, <i>J</i> = 13.2, 1H), 1.67 (d, <i>J</i> = 12.4, 2H), 1.53 (s, 3H), 1.46 (s, 9H), 1.46 (s, 9H), 1.46 (s, 9H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 554.30.</div></div><div id="sec4_8_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-(2,6-Dioxopiperidin-3-yl)-5-(4-(2-(piperidin-4-yl)ethyl)piperazin-1-yl)isoindoline-1,3-dione (<b>23b</b>)</h4><div class="NLM_p">A solution of HCl in dioxane (1.8 mL, 7.2 mmol) was added to <i>tert</i>-butyl 4-(2-(4-(2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)piperazin-1-yl)ethyl)piperidine-1-carboxylate (100 mg, 0.2 mmol). The resulting suspension was stirred at RT for 2 h. MeOH (1.0 mL) was then added, and the solution was stirred at RT for 30 min. The solvent was then removed under vacuum to afford the title compound (97 mg, 95%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.09 (s, 1H), 10.58 (s, 1H), 8.75 (s, 1H), 8.53–8.46 (m, 1H), 7.77 (d, <i>J</i> = 8.5, 1H), 7.50 (d, <i>J</i> = 2.3, 1H), 7.37 (dd, <i>J</i> = 8.6, 2.3, 1H), 5.10 (dd, <i>J</i> = 12.9, 5.4, 1H), 4.23 (s, 2H), 3.34 (s, 5H), 3.17 (t, <i>J</i> = 8.0, 4H), 2.97–2.78 (m, 3.5H), 2.65–2.53 (m, 2.5H), 2.10–1.99 (m, 1H), 1.83 (d, <i>J</i> = 12.8, 2H), 1.69–1.56 (m, 3H), 1.34 (dd, <i>J</i> = 13.2, 9.9, 2H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 454.</div></div><div id="sec4_8_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-(2,6-Dioxo-3-piperidyl)-5-[4-[2-[1-[4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]-4-piperidyl]ethyl]piperazin-1-yl]isoindoline-1,3-dione (<b>23</b>)</h4><div class="NLM_p">HATU (112 mg, 0.29 mmol) was added to a stirred solution of 4-(4-(<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl)-1-piperazinyl)-4-oxobutanoic acid (130 mg, 0.3 mmol), 2-[2,6-dioxo-3-piperidyl]-5-[4-[2-(4-piperidyl)ethyl]piperazin-1-yl]isoindoline-1,3-dione·2HCl (155 mg, 0.3 mmol), and DIPEA (0.15 mL, 0.9 mmol) in DMF (3 mL). The reaction mixture was stirred at RT for 1 h and then directly purified by reverse-phase HPLC (0–50%, 0.1% formic acid in H<sub>2</sub>O/MeCN, 25 min). The pure fractions were lyophilized to afford the title compound (100 mg, 37%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 11.07 (s, 1H), 7.79 (s, 1H), 7.68 (d, 1H, <i>J</i> = 8.5), 7.54 (d, 1H, <i>J</i> = 2.7), 7.34 (d, 1H, <i>J</i> = 1.8), 7.26 (dd, 1H, <i>J</i> = 2.1, 8.6), 6.56 (d, 1H, <i>J</i> = 2.6), 6.06 (br d, 1H, <i>J</i> = 6.7), 4.9–5.2 (m, 1H), 4.2–4.5 (m, 2H), 3.89 (br d, 1H, <i>J</i> = 12.1), 3.4–3.6 (m, 9H), 2.8–3.0 (m, 2H), 2.5–2.7 (m, 8H), 2.3–2.5 (m, 8H), 1.9–2.1 (m, 3H), 1.5–1.8 (m, 10H), 1.3–1.5 (m, 2H), 1.30 (d, 6H, <i>J</i> = 6.7), 1.10 (br dd, 1H, <i>J</i> = 2.4, 12.1), 0.9–1.0 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 173.3, 170.5, 170.3, 169.9, 168.0, 167.5, 155.7, 154.5, 147.4, 134.3, 125.4, 124.6, 118.8, 118.2, 117.8, 110.8, 108.3, 107.9, 60.7, 55.6, 52.9, 49.7, 49.2, 49.2, 47.4, 47.0, 45.6, 45.4, 42.0, 41.9, 40.9, 40.8, 40.6, 40.4, 40.3, 40.1, 34.1, 33.3, 32.9, 32.2, 31.5, 28.3, 27.9, 26.5, 24.8, 24.7, 22.7. HRMS (ESI<sup>+</sup>): anal. calcd for C<sub>47</sub>H<sub>64</sub>N<sub>11</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 878.5036; found, 878.5011.</div></div><div id="sec4_8_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>tert</i>-Butyl 4-[2-[4-[2-(1-Methyl-2,6-dioxo-3-piperidyl)-1,3-dioxo-isoindolin-5-yl]piperazin-1-yl]ethyl]piperidine-1-carboxylate (<b>24a</b>)</h4><div class="NLM_p">Sodium hydride (13 mg, 0.3 mmol) was added to a stirred solution of <i>tert</i>-butyl 4-(2-(4-(2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)piperazin-1-yl)ethyl)piperidine-1-carboxylate (180 mg, 0.3 mmol) in DMF at 0 °C. The resulting solution was stirred at RT for 5 min, then, methyl iodide (0.020 mL, 0.33 mmol) was added, and the reaction was stirred at RT for an additional 1 h and then quenched with water (1.0 mL). The mixture was partitioned with EtOAc (50 mL), the organic layer was separated, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the title compound (110 mg, 60%) as a light yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 7.68 (d, 1H, <i>J</i> = 8.5), 7.34 (d, 1H, <i>J</i> = 2.1), 7.26 (dd, 1H, <i>J</i> = 8.7, 2.3), 5.14 (dd, 1H, <i>J</i> = 13.0, 5.4), 3.91 (br d, 2H, <i>J</i> = 11.1), 3.41–3.47 (m, 4H), 3.02 (s, 3H), 2.91–2.99 (m, 1H), 2.54–2.81 (m, 5H), 2.31–2.39 (m, 3H), 1.99–2.08 (m, 1H), 1.61–1.72 (m, 2H), 1.42–1.53 (m, 3H), 1.39 (s, 9H), 0.94–1.08 (m, 4H). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 568.</div></div><div id="sec4_8_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 2-(1-Methyl-2,6-dioxo-3-piperidyl)-5-[4-[2-(4-piperidyl)ethyl]piperazin-1-yl]isoindoline-1,3-dione (<b>24b</b>)</h4><div class="NLM_p last">HCl (4 M in dioxane, 3.4 mL, 14 mmol) was added to a stirred solution of <i>tert</i>-butyl 4-[2-[4-[2-(1-methyl-2,6-dioxo-3-piperidyl)-1,3-dioxo-isoindolin-5-yl]piperazin-1-yl]ethyl]piperidine-1-carboxylate (80 mg, 0.14 mmol) in DCM (5 mL). The resulting solution was stirred at RT for 3 h, after which the solution was concentrated to give the title compound (62 mg, 94%) as a light yellow solid. <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 468.</div></div><div id="sec4_8_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5-[4-[2-[1-[4-[4-[<i>cis</i>-4-[(5-Isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]-4-piperidyl]ethyl]piperazin-1-yl]-2-(1-methyl-2,6-dioxo-3-piperidyl)isoindoline-1,3-dione (<b>24</b>)</h4><div class="NLM_p">HATU (14 mg, 0.04 mmol) was added to a solution of 4-(4-(<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl)-1-piperazinyl)-4-oxobutanoic acid (15 mg, 0.03 mmol), 2-(1-methyl-2,6-dioxo-3-piperidyl)-5-[4-[2-(4-piperidyl)ethyl]piperazin-1-yl]isoindoline-1,3-dione (15 mg, 0.03 mmol), and DIPEA (16 μL, 0.09 mmol) in DMF (1 mL). The resulting solution was stirred at RT for 3 h, after which the reaction mixture was purified by reverse-phase HPLC (C18 column, 0.1% TFA in H<sub>2</sub>O/MeCN, 25 min). Pure fractions were lyophilized to give title compound (25 mg, 73%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 9.7–9.8 (m, 2H), 7.84 (s, 1H), 7.78 (d, 1H, <i>J</i> = 8.5), 7.60 (d, 1H, <i>J</i> = 2.6), 7.50 (d, 1H, <i>J</i> = 2.0), 7.3–7.4 (m, 1H), 6.60 (d, 1H, <i>J</i> = 2.6), 5.98 (br d, 1H, <i>J</i> = 5.3), 5.17 (dd, 1H, <i>J</i> = 5.4, 13.0), 4.4–4.5 (m, 2H), 4.36 (br s, 3H), 4.24 (br d, 4H, <i>J</i> = 13.3), 3.9–4.0 (m, 1H), 3.60 (br s, 2H), 3.3–3.5 (m, 5H), 3.1–3.3 (m, 7H), 2.9–3.0 (m, 7H), 2.6–2.6 (m, 2H), 2.22 (br d, 2H, <i>J</i> = 11.0), 2.0–2.1 (m, 1H), 1.98 (br d, 2H, <i>J</i> = 7.2), 1.5–1.8 (m, 9H), 1.32 (d, 6H, <i>J</i> = 6.7), 1.1–1.2 (m, 1H), 0.9–1.0 (m, 1H). HRMS (ESI+): anal. calcd for C<sub>48</sub>H<sub>66</sub>N<sub>11</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 892.5192; found, 892.5193.</div></div><div id="sec4_8_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>tert</i>-Butyl <i>N</i>-[2-[2-[2-[2-[2-[[4-[4-[4-[(5-Isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]ethoxy]ethoxy]ethoxy]ethoxy]ethyl]carbamate (<b>25a</b>)</h4><div class="NLM_p">HATU (83 mg, 0.2 mmol) was added to a solution of 4-(4-(<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl)-1-piperazinyl)-4-oxobutanoic acid·HCO<sub>2</sub>H (43 mg, 0.09 mmol), <i>tert</i>-butyl (14-amino-3,6,9,12-tetraoxatetradecyl)carbamate (6 mg, 0.2 mmol), and DIPEA (46 μL, 0.3 mmol) in DMF (240 μL). The resulting solution was stirred overnight at RT, after which the reaction mixture was purified by reverse-phase HPLC (C18 column, 2 mM NH<sub>3</sub> in H<sub>2</sub>O/MeCN, 18 min). Pure fractions were lyophilized to give the title compound (54 mg, 82%) as a clear oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 7.85 (t, 1H, <i>J</i> = 5.6), 7.78 (s, 1H), 7.53 (d, 1H, <i>J</i> = 2.8), 6.73 (br s, 1H), 6.55 (d, 1H, <i>J</i> = 2.6), 6.05 (br d, 1H, <i>J</i> = 6.6), 4.29 (br s, 1H), 3.35–3.53 (m, 25H), 3.27–3.29 (m, 1H), 3.17 (q, 2H, <i>J</i> = 5.9), 3.05 (q, 2H, <i>J</i> = 6.1), 2.75 (br s, 1H), 2.41 (br s, 2H), 2.28–2.35 (m, 3H), 1.97 (br d, 2H, <i>J</i> = 5.7), 1.56–1.69 (m, 6H), 1.37 (s, 9H) 1.29 (d, 5H, <i>J</i> = 6.7). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 761.5.</div></div><div id="sec4_8_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-[2-[2-[2-[2-(2-Aminoethoxy)ethoxy]ethoxy]ethoxy]ethyl]-4-[4-[<i>cis</i>-4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanamide (<b>25b</b>)</h4><div class="NLM_p">HCl (4 M in dioxane, 1.0 mL, 4 mmol) was added to a stirred solution of <i>tert</i>-butyl <i>N</i>-[2-[2-[2-[2-[2-[[4-[4-[4-[(5-isopropylpyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-yl)amino]cyclohexyl]piperazin-1-yl]-4-oxo-butanoyl]amino]ethoxy]ethoxy]ethoxy]ethoxy]ethyl]carbamate (52 mg, 0.07 mmol) in MeOH (1 mL). The resulting solution was stirred at RT for 2 h and concentrated <i>in vacuo</i> to afford the crude product as a yellow oil. The crude product was injected onto a SCX column and washed with 5 CV of MeOH, followed by elution with 1 N NH<sub>3</sub> in MeOH. The resulting oil was concentrated <i>in vacuo</i> and purified by reverse-phase chromatography (C18, 0.1% formic acid in H<sub>2</sub>O/MeOH, 15 min). The pure fractions were lyophilized to afford the title compound (15 mg, 29%) as a colorless film.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): 8.40 (br s, 2H), 7.90 (br t, 1H, <i>J</i> = 5.5), 7.53 (d, 1H, <i>J</i> = 2.6), 6.55 (d, 1H, <i>J</i> = 2.8), 6.05 (br d, 1H, <i>J</i> = 6.9), 4.29 (br s, 1H), 3.85 (br s, 11H), 3.52 (d, 9H, <i>J</i> = 7.2), 3.34–3.41 (m, 3H), 3.18 (q, 2H, <i>J</i> = 5.9), 2.82 (br s, 2H), 2.51–2.57 (m, 1H), 2.47 (br s, 2H), 2.41 (br s, 2H), 2.27–2.35 (m, 3H), 1.91–2.04 (m, 2H), 1.50–1.74 (m, 6H), 1.29 (d, 6H, <i>J</i> = 6.7). <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 661.5.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01125" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58488" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58488" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01125?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01125</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Assay protocols and statistical analyses (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Raw data for the proteomics pull-down experiments (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf">jm0c01125_si_001.pdf (640.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_002.xlsx">jm0c01125_si_002.xlsx (936.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_003.csv">jm0c01125_si_003.csv (3.29 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01125" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sébastien L. Degorce</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9478-5106" title="Orcid link">http://orcid.org/0000-0002-9478-5106</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3a495f585b494e535f54145e5f5d5548595f7a5b494e485b405f545f595b14595557"><span class="__cf_email__" data-cfemail="cab9afa8abb9bea3afa4e4aeafada5b8a9af8aabb9beb8abb0afa4afa9abe4a9a5a7">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Omid Tavana</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8401-4546" title="Orcid link">http://orcid.org/0000-0001-8401-4546</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5a3537333e742e3b2c3b343b1a3b292e283b203f343f393b74393537"><span class="__cf_email__" data-cfemail="4a2527232e643e2b3c2b242b0a2b393e382b302f242f292b64292527">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erica Banks</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claire Crafter</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Cambridge
Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lakshmaiah Gingipalli</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Kouvchinov</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, BioPharmaceuticals R&D, AstraZeneca,
Boston, 35 Gatehouse
Drive, Waltham, MA, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yumeng Mao</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Cambridge
Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fiona Pachl</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, BioPharmaceuticals R&D, AstraZeneca,
Boston, 35 Gatehouse
Drive, Waltham, MA, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anisha Solanki</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Cambridge
Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Viia Valge-Archer</span> - <span class="hlFld-Affiliation affiliation">Bioscience,
Research and Early Development, Oncology R&D, AstraZeneca, Cambridge
Science Park, Unit 310 Darwin Building, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Yang</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott D. Edmondson</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development, Oncology R&D, AstraZeneca, Boston, 35 Gatehouse Drive, Waltham, MA 02451, United
States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26337" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26337" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Sharon Tentarelli for her help with the characterization of previously unreported molecules and Jamie Scott for proof-reading this manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain 4</p></td></tr><tr><td class="NLM_term">BSF</td><td class="NLM_def"><p class="first last">binomial survivor function</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DHCR24</td><td class="NLM_def"><p class="first last">24-dehydrocholesterol reductase</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">GSG2</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase haspin</p></td></tr><tr><td class="NLM_term">HBD</td><td class="NLM_def"><p class="first last">hydrogen bond donor</p></td></tr><tr><td class="NLM_term">IKZF1</td><td class="NLM_def"><p class="first last">Ikaros zince finger protein 1</p></td></tr><tr><td class="NLM_term">IMiD</td><td class="NLM_def"><p class="first last">immune-mediated inflammatory diseases</p></td></tr><tr><td class="NLM_term">IRAK</td><td class="NLM_def"><p class="first last">interleukin-1 receptor-associated kinase</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr><tr><td class="NLM_term">NEDD8</td><td class="NLM_def"><p class="first last">neural precursor cell expressed, developmentally down-regulated 8</p></td></tr><tr><td class="NLM_term">PBMC</td><td class="NLM_def"><p class="first last">peripheral blood mononuclear cell</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethylene glycol</p></td></tr><tr><td class="NLM_term">POI</td><td class="NLM_def"><p class="first last">protein of interest</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimeras</p></td></tr><tr><td class="NLM_term">THP</td><td class="NLM_def"><p class="first last">tetrahydropyran</p></td></tr><tr><td class="NLM_term">TRK</td><td class="NLM_def"><p class="first last">tropomyosin receptor kinase</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel Lindau</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.-F.</span></span> <span> </span><span class="NLM_article-title">IRAK family in inflammatory autoimmune diseases</span>. <i>Autoimmun. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">102461</span>, <span class="refDoi"> DOI: 10.1016/j.autrev.2020.102461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.autrev.2020.102461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31917263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVeksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=102461&author=L.-C.+Suauthor=W.-D.+Xuauthor=A.-F.+Huang&title=IRAK+family+in+inflammatory+autoimmune+diseases&doi=10.1016%2Fj.autrev.2020.102461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK family in inflammatory autoimmune diseases</span></div><div class="casAuthors">Su, Lin-Chong; Xu, Wang-Dong; Huang, An-Fang</div><div class="citationInfo"><span class="NLM_cas:title">Autoimmunity Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102461</span>CODEN:
                <span class="NLM_cas:coden">ARUEBU</span>;
        ISSN:<span class="NLM_cas:issn">1568-9972</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Innate immune signaling plays an important role in inflammation, and dysregulation of signaling components within this pathway has been focused as a crit. mediator in initiation, progression of inflammatory autoimmune diseases.  Toll-like receptors (TLRs) are the most upstream pattern recognition receptors in the immune cells, detecting pathogen assocd. mol. patterns, initiating signal transduction, by which interleukin-1 receptor-assocd. kinase (IRAK) family mediates activating signal from TLRs and interleukin-1 receptor.  The family comprises of four members, IRAK1, IRAK2, IRAK-M, IRAK4.  The family members have a role in either pos. or neg. regulation of innate immunity, adaptive immunity and inflammation.  Accumulated evidence proves that IRAK performs significantly in the pathogenesis of inflammatory autoimmune disorders.  On the one hand, both patients and animal modes reported abnormal expression of the family members.  On the other hand, functional study in vivo and in vitro demonstrated that the members are implicated in the development of the diseases.  Interestingly, IRAK inhibition has potential therapeutic benefits.  In this review, we focus on the family, review the physiol. roles in different immune cells, and summarize emerging data for highlighting the importance of them in inflammatory autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUEbD7FX0f7Vg90H21EOLACvtfcHk0liSPhOUHf5o7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVeksL8%253D&md5=2202adad3c59dd0bf7350cc6d22e8c46</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.autrev.2020.102461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.autrev.2020.102461%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.-C.%26aulast%3DXu%26aufirst%3DW.-D.%26aulast%3DHuang%26aufirst%3DA.-F.%26atitle%3DIRAK%2520family%2520in%2520inflammatory%2520autoimmune%2520diseases%26jtitle%3DAutoimmun.%2520Rev.%26date%3D2020%26volume%3D19%26spage%3D102461%26doi%3D10.1016%2Fj.autrev.2020.102461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeway, C. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medzhitov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flavell, R. A.</span></span> <span> </span><span class="NLM_article-title">IRAK-M is a negative regulator of Toll-like receptor signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(02)00827-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2FS0092-8674%2802%2900827-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=12150927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvV2htrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=191-202&author=K.+Kobayashiauthor=L.+D.+Hernandezauthor=J.+E.+Gal%C3%A1nauthor=C.+A.+Janewayauthor=R.+Medzhitovauthor=R.+A.+Flavell&title=IRAK-M+is+a+negative+regulator+of+Toll-like+receptor+signaling&doi=10.1016%2Fs0092-8674%2802%2900827-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-M is a negative regulator of toll-like receptor signaling</span></div><div class="casAuthors">Kobayashi, Koichi; Hernandez, Lorraine D.; Galan, Jorge E.; Janeway, Charles A., Jr.; Medzhitov, Ruslan; Flavell, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">191-202</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) detect microorganisms and protect multicellular organisms from infection.  TLRs transduce their signals through MyD88 and the serine/threonine kinase IRAK.  The IRAK family consists of two active kinases, IRAK and IRAK-4, and two inactive kinases, IRAK-2 and IRAK-M.  IRAK-M expression is restricted to monocytes/macrophages, whereas other IRAKs are ubiquitous.  We show here that IRAK-M is induced upon TLR stimulation and neg. regulates TLR signaling.  IRAK-M prevented dissocn. of IRAK and IRAK-4 from MyD88 and formation of IRAK-TRAF6 complexes.  IRAK-M-/- cells exhibited increased cytokine prodn. upon TLR/IL-1 stimulation and bacterial challenge, and IRAK-M-/- mice showed increased inflammatory responses to bacterial infection.  Endotoxin tolerance, a protection mechanism against endotoxin shock, was significantly reduced in IRAK-M-/- cells.  Thus, IRAK-M regulates TLR signaling and innate immune homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKFscWDZfrLLVg90H21EOLACvtfcHk0lgCJAy07S_t4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvV2htrw%253D&md5=80dc28f4773127eac27aa2755e1b2195</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900827-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900827-9%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DHernandez%26aufirst%3DL.%2BD.%26aulast%3DGal%25C3%25A1n%26aufirst%3DJ.%2BE.%26aulast%3DJaneway%26aufirst%3DC.%2BA.%26aulast%3DMedzhitov%26aufirst%3DR.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26atitle%3DIRAK-M%2520is%2520a%2520negative%2520regulator%2520of%2520Toll-like%2520receptor%2520signaling%26jtitle%3DCell%26date%3D2002%26volume%3D110%26spage%3D191%26epage%3D202%26doi%3D10.1016%2Fs0092-8674%2802%2900827-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starczynowski, D. T.</span></span> <span> </span><span class="NLM_article-title">IRAK signalling in cancer</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1038/bjc.2014.513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fbjc.2014.513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=25290089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCis7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=232-237&author=G.+W.+Rhyasenauthor=D.+T.+Starczynowski&title=IRAK+signalling+in+cancer&doi=10.1038%2Fbjc.2014.513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK signalling in cancer</span></div><div class="casAuthors">Rhyasen, G. W.; Starczynowski, D. T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">232-237</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Innate immune signalling has an essential role in inflammation, and the dysregulation of signalling components of this pathway is increasingly being recognized as an important mediator in cancer initiation and progression.  In some malignancies, dysregulation of inflammatory toll-like receptor (TLR) and interleukin-1 receptor (IL1R) signalling is typified by increased NF-κB activity, and it occurs through somatic mutations, chromosomal deletions, and/or transcriptional deregulation.  Interleukin-1 receptor-assocd. kinase (IRAK) family members are mediators of TLR/IL1R superfamily signalling, and mounting evidence implicates these kinases as viable cancer targets.  Although there have been previous efforts aimed at the development of IRAK kinase inhibitors, this is currently an area of renewed interest for cancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGshpWhcF0K7Vg90H21EOLACvtfcHk0lgCJAy07S_t4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCis7zJ&md5=c88416c9bc7754d997d87a489b997fa3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2014.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2014.513%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DStarczynowski%26aufirst%3DD.%2BT.%26atitle%3DIRAK%2520signalling%2520in%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2015%26volume%3D112%26spage%3D232%26epage%3D237%26doi%3D10.1038%2Fbjc.2014.513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Neutrophils deficient in innate suppressor IRAK-M enhances anti-tumor immune responses</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2019.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ymthe.2019.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31607540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2ltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=89-99&author=Y.+Zhangauthor=N.+Diaoauthor=C.+K.+Leeauthor=H.+W.+Chuauthor=L.+Baiauthor=L.+Li&title=Neutrophils+deficient+in+innate+suppressor+IRAK-M+enhances+anti-tumor+immune+responses&doi=10.1016%2Fj.ymthe.2019.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Neutrophils Deficient in Innate Suppressor IRAK-M Enhances Anti-tumor Immune Responses</span></div><div class="casAuthors">Zhang, Yao; Diao, Na; Lee, Christina K.; Chu, Hong Wei; Bai, Lan; Li, Liwu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-99</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0024</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tumor-assocd. immune-suppressive neutrophils are prevalent in various cancers, including colorectal cancer.  However, mechanisms of immune-suppressive neutrophils are not well understood.  We report that a key innate suppressor, IRAK-M (interleukin-1 receptor-assocd. kinase M), is critically involved in the establishment of immune-suppressive neutrophils.  In contrast to the wild-type (WT) neutrophils exhibiting immune-suppressive signatures of CD11bhighPD-L1highCD80low, IRAK-M-deficient neutrophils are rewired with reduced levels of inhibitory mols. PD-L1 and CD11b, as well as enhanced expression of stimulatory mols. CD80 and CD40.  The reprogramming of IRAK-M-deficient neutrophils is mediated by reduced activation of STAT1/3 and enhanced activation of STAT5.  As a consequence, IRAK-M-deficient neutrophils demonstrate enhanced capability to promote, instead of suppress, the proliferation and activation of effector T cells both in vitro and in vivo.  Functionally, we obsd. that the transfusion of IRAK-M-/- neutrophils can potently render an enhanced anti-tumor immune response in the murine inflammation-induced colorectal cancer model.  Collectively, our study defines IRAK-M as an innate suppressor for neutrophil function and reveals IRAK-M as a promising target for rewiring neutrophils in anti-cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-UavWHJH07Vg90H21EOLACvtfcHk0lgCJAy07S_t4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2ltro%253D&md5=59b0d0cbb080b216de8413a16fb56a60</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2019.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2019.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DC.%2BK.%26aulast%3DChu%26aufirst%3DH.%2BW.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DNeutrophils%2520deficient%2520in%2520innate%2520suppressor%2520IRAK-M%2520enhances%2520anti-tumor%2520immune%2520responses%26jtitle%3DMol.%2520Ther.%26date%3D2020%26volume%3D28%26spage%3D89%26epage%3D99%26doi%3D10.1016%2Fj.ymthe.2019.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kesselring, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaesner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiergeist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naschberger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wege, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croner, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neurath, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlitt, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fichtner-Feigl, S.</span></span> <span> </span><span class="NLM_article-title">IRAK-M expression in tumor cells supports colorectal cancer progression through reduction of antimicrobial defense and stabilization of STAT3</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">684</span>– <span class="NLM_lpage">696</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ccell.2016.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=27150039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFCitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=684-696&author=R.+Kesselringauthor=J.+Glaesnerauthor=A.+Hiergeistauthor=E.+Naschbergerauthor=H.+Neumannauthor=S.+M.+Brunnerauthor=A.+K.+Wegeauthor=C.+Seebauerauthor=G.+K%C3%B6hlauthor=S.+Merklauthor=R.+S.+Cronerauthor=C.+Hacklauthor=M.+St%C3%BCrzlauthor=M.+F.+Neurathauthor=A.+Gessnerauthor=H.-J.+Schlittauthor=E.+K.+Geisslerauthor=S.+Fichtner-Feigl&title=IRAK-M+expression+in+tumor+cells+supports+colorectal+cancer+progression+through+reduction+of+antimicrobial+defense+and+stabilization+of+STAT3&doi=10.1016%2Fj.ccell.2016.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-M Expression in Tumor Cells Supports Colorectal Cancer Progression through Reduction of Antimicrobial Defense and Stabilization of STAT3</span></div><div class="casAuthors">Kesselring, Rebecca; Glaesner, Joachim; Hiergeist, Andreas; Naschberger, Elisabeth; Neumann, Helmut; Brunner, Stefan M.; Wege, Anja K.; Seebauer, Caroline; Koehl, Gudrun; Merkl, Susanne; Croner, Roland S.; Hackl, Christina; Stuerzl, Michael; Neurath, Markus F.; Gessner, Andre; Schlitt, Hans-Juergen; Geissler, Edward K.; Fichtner-Feigl, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">684-696</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Colorectal cancer (CRC) is assocd. with loss of epithelial barrier integrity, which facilitates the interaction of the immunol. microenvironment with the luminal microbiome, eliciting tumor-supportive inflammation.  An important regulator of intestinal inflammatory responses is IRAK-M, a neg. regulator of TLR signaling.  Here we investigate the compartment-specific impact of IRAK-M on colorectal carcinogenesis using a mouse model.  We demonstrate that IRAK-M is expressed in tumor cells due to combined TLR and Wnt activation.  Tumor cell-intrinsic IRAK-M is responsible for regulation of microbial colonization of tumors and STAT3 protein stability in tumor cells, leading to tumor cell proliferation.  IRAK-M expression in human CRCs is assocd. with poor prognosis.  These results suggest that IRAK-M may be a potential therapeutic target for CRC treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7FxdAcfVYGbVg90H21EOLACvtfcHk0li1MmPcwOaWjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFCitbs%253D&md5=89338ee725e5b6b48b1ba11d9fa30935</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DKesselring%26aufirst%3DR.%26aulast%3DGlaesner%26aufirst%3DJ.%26aulast%3DHiergeist%26aufirst%3DA.%26aulast%3DNaschberger%26aufirst%3DE.%26aulast%3DNeumann%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DS.%2BM.%26aulast%3DWege%26aufirst%3DA.%2BK.%26aulast%3DSeebauer%26aufirst%3DC.%26aulast%3DK%25C3%25B6hl%26aufirst%3DG.%26aulast%3DMerkl%26aufirst%3DS.%26aulast%3DCroner%26aufirst%3DR.%2BS.%26aulast%3DHackl%26aufirst%3DC.%26aulast%3DSt%25C3%25BCrzl%26aufirst%3DM.%26aulast%3DNeurath%26aufirst%3DM.%2BF.%26aulast%3DGessner%26aufirst%3DA.%26aulast%3DSchlitt%26aufirst%3DH.-J.%26aulast%3DGeissler%26aufirst%3DE.%2BK.%26aulast%3DFichtner-Feigl%26aufirst%3DS.%26atitle%3DIRAK-M%2520expression%2520in%2520tumor%2520cells%2520supports%2520colorectal%2520cancer%2520progression%2520through%2520reduction%2520of%2520antimicrobial%2520defense%2520and%2520stabilization%2520of%2520STAT3%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D684%26epage%3D696%26doi%3D10.1016%2Fj.ccell.2016.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras as therapeutics and tools for biological discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras+as+therapeutics+and+tools+for+biological+discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0li1MmPcwOaWjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%2520as%2520therapeutics%2520and%2520tools%2520for%2520biological%2520discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ’beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%99beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lgfuZOr_bEpsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span> <span> </span><span class="NLM_article-title">Prospects for pharmacological targeting of pseudokinases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0018-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fs41573-019-0018-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=30850748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=501-526&author=J.+E.+Kungauthor=N.+Jura&title=Prospects+for+pharmacological+targeting+of+pseudokinases&doi=10.1038%2Fs41573-019-0018-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for pharmacological targeting of pseudokinases</span></div><div class="casAuthors">Kung, Jennifer E.; Jura, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">501-526</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Pseudokinases are members of the protein kinase superfamily but signal primarily through noncatalytic mechanisms.  Many pseudokinases contribute to the pathologies of human diseases, yet they remain largely unexplored as drug targets owing to challenges assocd. with modulation of their biol. functions.  Our understanding of the structure and physiol. roles of pseudokinases has improved substantially over the past decade, revealing intriguing similarities between pseudokinases and their catalytically active counterparts.  Pseudokinases often adopt conformations that are analogous to those seen in catalytically active kinases and, in some cases, can also bind metal cations and/or nucleotides.  Several clin. approved kinase inhibitors have been shown to influence the noncatalytic functions of active kinases, providing hope that similar properties in pseudokinases could be pharmacol. regulated.  In this Review, we discuss known roles of pseudokinases in disease, their unique structural features and the progress that has been made towards developing pseudokinase-directed therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFxQnlHTnn7Vg90H21EOLACvtfcHk0lgfuZOr_bEpsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D&md5=f3414770154e8d6449b454f9bc3f19bd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0018-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0018-3%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BE.%26aulast%3DJura%26aufirst%3DN.%26atitle%3DProspects%2520for%2520pharmacological%2520targeting%2520of%2520pseudokinases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D501%26epage%3D526%26doi%3D10.1038%2Fs41573-019-0018-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span>; <span class="NLM_string-name">Tinworth, C.</span></span> <span> </span><span class="NLM_article-title">Novel compounds</span>. <span class="NLM_patent">WO 2017211924 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+D.+Harling&author=C.+Tinworth&title=Novel+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DNovel%2520compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span>; <span class="NLM_string-name">Tinworth, C.</span></span> <span> </span><span class="NLM_article-title">Novel compounds</span>. <span class="NLM_patent">WO 2018033556 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=J.+D.+Harling&author=C.+Tinworth&title=Novel+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DNovel%2520compounds%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culshaw, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groombridge, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marden, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhirter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Pyrrolopyrimidine Inhibitors of Interleukin-1 Receptor Associated Kinase 4 (IRAK4) for the Treatment of Mutant MYD88L265P Diffuse Large B-Cell Lymphoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10071</span>– <span class="NLM_lpage">10091</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10071-10091&author=J.+S.+Scottauthor=S.+L.+Degorceauthor=R.+Anjumauthor=J.+Culshawauthor=R.+D.+M.+Daviesauthor=N.+L.+Daviesauthor=K.+S.+Dillmanauthor=J.+E.+Dowlingauthor=L.+Drewauthor=A.+D.+Fergusonauthor=S.+D.+Groombridgeauthor=C.+T.+Halsallauthor=J.+A.+Hudsonauthor=S.+Lamontauthor=N.+A.+Lindsayauthor=S.+K.+Mardenauthor=M.+F.+Mayoauthor=J.+E.+Peaseauthor=D.+R.+Perkinsauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=A.+Rosenauthor=M.+Shenauthor=C.+McWhirterauthor=D.+Wu&title=Discovery+and+Optimization+of+Pyrrolopyrimidine+Inhibitors+of+Interleukin-1+Receptor+Associated+Kinase+4+%28IRAK4%29+for+the+Treatment+of+Mutant+MYD88L265P+Diffuse+Large+B-Cell+Lymphoma&doi=10.1021%2Facs.jmedchem.7b01290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Pyrrolopyrimidine Inhibitors of Interleukin-1 Receptor Associated Kinase 4 (IRAK4) for the Treatment of Mutant MYD88L265P Diffuse Large B-Cell Lymphoma</span></div><div class="casAuthors">Scott, James S.; Degorce, Sebastien L.; Anjum, Rana; Culshaw, Janet; Davies, Robert D. M.; Davies, Nichola L.; Dillman, Keith S.; Dowling, James E.; Drew, Lisa; Ferguson, Andrew D.; Groombridge, Sam D.; Halsall, Christopher T.; Hudson, Julian A.; Lamont, Scott; Lindsay, Nicola A.; Marden, Stacey K.; Mayo, Michele F.; Pease, J. Elizabeth; Perkins, David R.; Pink, Jennifer H.; Robb, Graeme R.; Rosen, Alan; Shen, Minhui; McWhirter, Claire; Wu, Dedong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10071-10091</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the optimization of a series of pyrrolopyrimidine inhibitors of interleukin-1 receptor assocd. kinase 4 (IRAK4) using x-ray crystal structures and structure based design to identify and optimize our scaffold.  Compd. 28 demonstrated a favorable physicochem. and kinase selectivity profile and was identified as a promising in vivo tool with which to explore the role of IRAK4 inhibition in the treatment of mutant MYD88L265P diffuse large B-cell lymphoma (DLBCL).  Compd. 28 was shown to be capable of demonstrating inhibition of NF-κB activation and growth of the ABC subtype of DLBCL cell lines in vitro at high concns. but showed greater effects in combination with a BTK inhibitor at lower concns.  In vivo, the combination of compd. 28 and ibrutinib led to tumor regression in an ABC-DLBCL mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBCn7WCzPrrrVg90H21EOLACvtfcHk0lgfuZOr_bEpsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrjO&md5=9760f425ec33b238a43e1de0ec6d24f1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01290%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DCulshaw%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DR.%2BD.%2BM.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGroombridge%26aufirst%3DS.%2BD.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DHudson%26aufirst%3DJ.%2BA.%26aulast%3DLamont%26aufirst%3DS.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMarden%26aufirst%3DS.%2BK.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3DPerkins%26aufirst%3DD.%2BR.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRosen%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Pyrrolopyrimidine%2520Inhibitors%2520of%2520Interleukin-1%2520Receptor%2520Associated%2520Kinase%25204%2520%2528IRAK4%2529%2520for%2520the%2520Treatment%2520of%2520Mutant%2520MYD88L265P%2520Diffuse%2520Large%2520B-Cell%2520Lymphoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10071%26epage%3D10091%26doi%3D10.1021%2Facs.jmedchem.7b01290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumming, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span> <span> </span><span class="NLM_article-title">Lowering lipophilicity by adding carbon: one-carbon bridges of morpholines and piperazines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8934</span>– <span class="NLM_lpage">8943</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8934-8943&author=S.+L.+Degorceauthor=M.+S.+Bodnarchukauthor=I.+A.+Cummingauthor=J.+S.+Scott&title=Lowering+lipophilicity+by+adding+carbon%3A+one-carbon+bridges+of+morpholines+and+piperazines&doi=10.1021%2Facs.jmedchem.8b01148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Lowering Lipophilicity by Adding Carbon: One-Carbon Bridges of Morpholines and Piperazines</span></div><div class="casAuthors">Degorce, Sebastien L.; Bodnarchuk, Michael S.; Cumming, Iain A.; Scott, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8934-8943</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, we report our investigation of a phenomenon by which bridging morpholines across the ring with one-carbon tethers leads to a counterintuitive redn. in lipophilicity.  This effect was also found to occur in piperazines and piperidines and lowered the measured log D7.4 of the bridged mols. by as much as -0.8 relative to their unbridged counterparts.  As lowering lipophilicity without introducing addnl. heteroatoms can be desirable, we believe this potentially provides a useful tactic to improve the drug-like properties of mols. contg. morpholine-, piperazine-, and piperidine-like motifs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhZJB3SYRKD7Vg90H21EOLACvtfcHk0lh7YejandkABQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK&md5=27f30b8220ea767d6ef1e492f3fa1aba</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01148%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DCumming%26aufirst%3DI.%2BA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26atitle%3DLowering%2520lipophilicity%2520by%2520adding%2520carbon%253A%2520one-carbon%2520bridges%2520of%2520morpholines%2520and%2520piperazines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8934%26epage%3D8943%26doi%3D10.1021%2Facs.jmedchem.8b01148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groombridge, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Optimization of permeability in a series of pyrrolotriazine inhibitors of IRAK4</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.bmc.2018.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=29398441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=913-924&author=S.+L.+Degorceauthor=R.+Anjumauthor=K.+S.+Dillmanauthor=L.+Drewauthor=S.+D.+Groombridgeauthor=C.+T.+Halsallauthor=E.+M.+Lenzauthor=N.+A.+Lindsayauthor=M.+F.+Mayoauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=J.+S.+Scottauthor=S.+Stokesauthor=Y.+Xue&title=Optimization+of+permeability+in+a+series+of+pyrrolotriazine+inhibitors+of+IRAK4&doi=10.1016%2Fj.bmc.2018.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of permeability in a series of pyrrolotriazine inhibitors of IRAK4</span></div><div class="casAuthors">Degorce, Sebastien L.; Anjum, Rana; Dillman, Keith S.; Drew, Lisa; Groombridge, Sam D.; Halsall, Christopher T.; Lenz, Eva M.; Lindsay, Nicola A.; Mayo, Michele F.; Pink, Jennifer H.; Robb, Graeme R.; Scott, James S.; Stokes, Stephen; Xue, Yafeng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">913-924</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors have developed a series of orally efficacious IRAK4 inhibitors, based on a scaffold hopping strategy and using rational structure based design.  Efforts to tackle low permeability and high efflux in the authors' previously reported pyrrolopyrimidine series (Scott et al., 2017) led to the identification of pyrrolotriazines which contained one less formal hydrogen bond donor and were intrinsically more lipophilic.  Further optimization of substituents on this pyrrolotriazine core culminated with the discovery of compd. 30 (trans-1-(4-(4-((5-isopropylpyrrolo[2,1-f]-[1,2,4]triazin-4-yl)amino)cyclohexyl)piperazin-1-yl)ethanone) as a promising in vivo probe to assess the potential of IRAK4 inhibition for the treatment of MyD88 mutant Diffuse Large B Cell Lymphoma (DLBCL) in combination with a BTK inhibitor.  When tested in an ABC-DLBCL model with a dual MyD88/CD79 mutation (OCI-LY10), 30 demonstrated tumor regressions in combination with ibrutinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao5i7H0tVFrVg90H21EOLACvtfcHk0lh7YejandkABQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSitLc%253D&md5=772fab4ca7c899bb85bcd9ea8a9d47f3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DGroombridge%26aufirst%3DS.%2BD.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DLenz%26aufirst%3DE.%2BM.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DOptimization%2520of%2520permeability%2520in%2520a%2520series%2520of%2520pyrrolotriazine%2520inhibitors%2520of%2520IRAK4%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D913%26epage%3D924%26doi%3D10.1016%2Fj.bmc.2018.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+frequent-hitter+behavior+based+on+historical+high-throughput+screening+data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0lj2MysTf9-ZBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520frequent-hitter%2520behavior%2520based%2520on%2520historical%2520high-throughput%2520screening%2520data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloecher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of 5-azaquinazolines as orally efficacious IRAK4 inhibitors targeting MyD88L265P mutant diffuse large B cell lymphoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9918</span>– <span class="NLM_lpage">9930</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyitLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9918-9930&author=S.+L.+Degorceauthor=R.+Anjumauthor=A.+Bloecherauthor=R.+J.+Carbajoauthor=K.+S.+Dillmanauthor=L.+Drewauthor=C.+T.+Halsallauthor=E.+M.+Lenzauthor=N.+A.+Lindsayauthor=M.+F.+Mayoauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=A.+Rosenauthor=J.+S.+Scottauthor=Y.+Xue&title=Discovery+of+a+series+of+5-azaquinazolines+as+orally+efficacious+IRAK4+inhibitors+targeting+MyD88L265P+mutant+diffuse+large+B+cell+lymphoma&doi=10.1021%2Facs.jmedchem.9b01346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Series of 5-Azaquinazolines as Orally Efficacious IRAK4 Inhibitors Targeting MyD88L265P Mutant Diffuse Large B Cell Lymphoma</span></div><div class="casAuthors">Degorce, Sebastien L.; Anjum, Rana; Bloecher, Andrew; Carbajo, Rodrigo J.; Dillman, Keith S.; Drew, Lisa; Halsall, Christopher T.; Lenz, Eva M.; Lindsay, Nicola A.; Mayo, Michele F.; Pink, Jennifer H.; Robb, Graeme R.; Rosen, Alan; Scott, James S.; Xue, Yafeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9918-9930</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, the authors report the discovery of a series of 5-azaquinazolines as selective IRAK4 inhibitors.  From modestly potent quinazoline 4, the authors introduced a 5-aza substitution to mask the 4-NH hydrogen bond donor (HBD).  This allowed us to substitute the core with a 2-aminopyrazole, which showed large gains in cellular potency despite the addnl. formal HBD.  Further optimization led to 6-cyanomethyl-5-azaquinazoline 13, a selective IRAK4 inhibitor which proved efficacious in combination with ibrutinib, while showing very little activity as a single agent up to 100 mg/kg.  This contrasted to previously reported IRAK4 inhibitors that exhibited efficacy in the same model as single agents, and was attributed to the enhanced specificity of 13 towards IRAK4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9B-WmMsvfDrVg90H21EOLACvtfcHk0lj2MysTf9-ZBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyitLzE&md5=5029cfe2e62ca9bc3e7e9a0b02ab8cc0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01346%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DBloecher%26aufirst%3DA.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DLenz%26aufirst%3DE.%2BM.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRosen%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%25205-azaquinazolines%2520as%2520orally%2520efficacious%2520IRAK4%2520inhibitors%2520targeting%2520MyD88L265P%2520mutant%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9918%26epage%3D9930%26doi%3D10.1021%2Facs.jmedchem.9b01346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camenisch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Waterbeemd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkers, G.</span></span> <span> </span><span class="NLM_article-title">Estimation of permeability by passive diffusion through Caco-2 cell monolayers using the drugs’ lipophilicity and molecular weight</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/s0928-0987(97)10019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2FS0928-0987%2897%2910019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=9795088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A280%3ADC%252BD2cngsFyrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=317-324&author=G.+Camenischauthor=J.+Alsenzauthor=H.+van+de+Waterbeemdauthor=G.+Folkers&title=Estimation+of+permeability+by+passive+diffusion+through+Caco-2+cell+monolayers+using+the+drugs%E2%80%99+lipophilicity+and+molecular+weight&doi=10.1016%2Fs0928-0987%2897%2910019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of permeability by passive diffusion through Caco-2 cell monolayers using the drugs' lipophilicity and molecular weight</span></div><div class="casAuthors">Camenisch G; Alsenz J; van de Waterbeemd H; Folkers G</div><div class="citationInfo"><span class="NLM_cas:title">European journal of pharmaceutical sciences : official journal of the European Federation for Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-24</span>
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    </div><div class="casAbstract">A recently developed, new theoretical absorption model for passive diffusion through biological membranes describing the dependency of membrane permeability on lipophilicity and molecular size, predicts different sigmoid-hyperbolic permeability-lipophilicity relationships for different molecular weight ranges.  This model has been tested with experimental in vitro cultured epithelial cell (Caco-2) permeability data for structurally diverse drugs differing in lipophilicity, ionization state and molecular size.  These data were pooled with literature values.  Using this simple physicochemical approach, the permeability of a compound through Caco-2 cells by passive diffusion can be predicted from the compounds' distribution coefficient in 1-octanol/water (log D(oct)) and its molecular weight (MW).  Deviations from this expected behaviour may point to the involvement of biological components in the transport process, which may require further investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSF9qdSVu_jQBfKb6ELK8t0fW6udTcc2ebbJnqy_Wh-I7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2cngsFyrsQ%253D%253D&md5=c1d933c4bc0b5356feed693bb0c45cbe</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2897%2910019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252897%252910019-7%26sid%3Dliteratum%253Aachs%26aulast%3DCamenisch%26aufirst%3DG.%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DFolkers%26aufirst%3DG.%26atitle%3DEstimation%2520of%2520permeability%2520by%2520passive%2520diffusion%2520through%2520Caco-2%2520cell%2520monolayers%2520using%2520the%2520drugs%25E2%2580%2599%2520lipophilicity%2520and%2520molecular%2520weight%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D1998%26volume%3D6%26spage%3D317%26epage%3D324%26doi%3D10.1016%2Fs0928-0987%2897%2910019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs) — Past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29+%E2%80%94+Past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2ebbJnqy_Wh-I7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%2520%25E2%2580%2594%2520Past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugations for the degradation of oncogenic PI3K</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ejmech.2018.03.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=29625382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntFOktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=237-247&author=W.+Liauthor=C.+Gaoauthor=L.+Zhaoauthor=Z.+Yuanauthor=Y.+Chenauthor=Y.+Jiang&title=Phthalimide+conjugations+for+the+degradation+of+oncogenic+PI3K&doi=10.1016%2Fj.ejmech.2018.03.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugations for the degradation of oncogenic PI3K</span></div><div class="casAuthors">Li, Wenlu; Gao, Chunmei; Zhao, Lei; Yuan, Zigao; Chen, Yuzong; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">237-247</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">PI3K/Akt/mTOR pathway is crucial for carcinogenesis and its inhibitors have made a great progress in cancer treatment.  However, there is still a great developing space for PI3K inhibitors as the acquired drug resistance hindered their application in clin.  Proteolysis-targeting chimeras (PROTACs) with the potential to handle the challenges faced in drug development could be an alternative therapeutic strategy.  Moreover, the past two years have witnessed remarkable advances in the development of phthalimide conjugation as a strategy for the degrdn. instead of inhibition of the targets, including BET family proteins, Sirtuin 2, CDK 9, Smad 3, and BCR-ABL proteins.  Here, we designed and synthesized a series of potential small mol. PROTACs for the degrdn. of PI3K.  Four compds. induced remarkable PI3K degrdn. and down-regulated the phosphorylation of Akt, S6K and GSK-3β in liver cancer cells HepG2.  Furthermore, the representative compd. D proved to inhibit tumor cells proliferation by the induction of autophagy instead of apoptosis or cell cycle arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0E7-HLm8E5bVg90H21EOLACvtfcHk0li_WD4dGse5sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntFOktbo%253D&md5=f82652f95e23fc9088d53e541ebe1ad4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.066%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DPhthalimide%2520conjugations%2520for%2520the%2520degradation%2520of%2520oncogenic%2520PI3K%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D237%26epage%3D247%26doi%3D10.1016%2Fj.ejmech.2018.03.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0lhT2vI74iqxlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newham, P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras in drug development: A safety perspective</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1111/bph.15014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1111%2Fbph.15014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=32022252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2020&pages=1709-1718&author=K.+Moreauauthor=M.+Coenauthor=A.+X.+Zhangauthor=F.+Pachlauthor=M.+P.+Castaldiauthor=G.+Dahlauthor=H.+Boydauthor=C.+Scottauthor=P.+Newham&title=Proteolysis-targeting+chimeras+in+drug+development%3A+A+safety+perspective&doi=10.1111%2Fbph.15014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimeras in drug development: A safety perspective</span></div><div class="casAuthors">Moreau, Kevin; Coen, Muireann; Zhang, Andrew X.; Pachl, Fiona; Castaldi, M. Paola; Dahl, Goran; Boyd, Helen; Scott, Clay; Newham, Pete</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1709-1718</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras are a new drug modality that exploits the endogenous ubiquitin proteasome system to degrade a protein of interest for therapeutic benefit.  As the first-generation of proteolysis-targeting chimeras have now entered clin. trials for oncol. indications, it is timely to consider the theor. safety risks inherent with this modality which include off-target degrdn., intracellular accumulation of natural substrates for the E3 ligases used in the ubiquitin proteasome system, proteasome satn. by ubiquitinated proteins, and liabilities assocd. with the "hook effect" of proteolysis-targeting chimeras This review describes in vitro and non-clin. in vivo data that provide mechanistic insight of these safety risks and approaches being used to mitigate these risks in the next generation of proteolysis-targeting chimera mols. to extend therapeutic applications beyond life-threatening diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreTK_TldLxHbVg90H21EOLACvtfcHk0lhT2vI74iqxlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D&md5=1e26bbe1d623f2f8f4218c2e3fad9dd6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1111%2Fbph.15014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.15014%26sid%3Dliteratum%253Aachs%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DA.%2BX.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DNewham%26aufirst%3DP.%26atitle%3DProteolysis-targeting%2520chimeras%2520in%2520drug%2520development%253A%2520A%2520safety%2520perspective%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2020%26volume%3D177%26spage%3D1709%26epage%3D1718%26doi%3D10.1111%2Fbph.15014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Snyder, L. B.</span></span> <span> </span><span class="NLM_article-title">Cereblon ligands and bifunctional compounds comprising the same</span>. U.S. Patent <span class="NLM_patent">20180228907 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=C.+M.+Crews&author=H.+Dong&author=K.+R.+Hornberger&author=J.+Wang&author=Y.+Qian&author=K.+Zimmermann&author=M.+Berlin&author=L.+B.+Snyder&title=Cereblon+ligands+and+bifunctional+compounds+comprising+the+same"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DCereblon%2520ligands%2520and%2520bifunctional%2520compounds%2520comprising%2520the%2520same%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Snyder, L. B.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span></span> <span> </span><span class="NLM_article-title">Compounds and methods for the targeted degradation of androgen receptor</span>. U.S. Patent <span class="NLM_patent">20180099940 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=K.+R.+Hornberger&author=L.+B.+Snyder&author=K.+Zimmermann&author=J.+Wang&author=M.+Berlin&author=C.+M.+Crews&author=H.+Dong&title=Compounds+and+methods+for+the+targeted+degradation+of+androgen+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DCompounds%2520and%2520methods%2520for%2520the%2520targeted%2520degradation%2520of%2520androgen%2520receptor%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Tetrahydronaphthalene and Tetrahydroisoquinoline Derivatives as Estrogen Receptor Degraders and their Preparation</span>. <span class="NLM_patent">WO 2018102725 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=Y.+Qian&author=H.+Dong&author=J.+Wang&author=K.+R.+Hornberger&author=C.+M.+Crews&title=Tetrahydronaphthalene+and+Tetrahydroisoquinoline+Derivatives+as+Estrogen+Receptor+Degraders+and+their+Preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DTetrahydronaphthalene%2520and%2520Tetrahydroisoquinoline%2520Derivatives%2520as%2520Estrogen%2520Receptor%2520Degraders%2520and%2520their%2520Preparation%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Ishchenko, A.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Jaime-Figueroa, S.</span>; <span class="NLM_string-name">Burslem, G.</span></span> <span> </span><span class="NLM_article-title">Egfr proteolysis targeting chimeric molecules and associated methods of use</span>. <span class="NLM_patent">WO 2018119441 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=K.+Zimmermann&author=J.+Wang&author=M.+Berlin&author=H.+Dong&author=A.+Ishchenko&author=Y.+Qian&author=C.+M.+Crews&author=S.+Jaime-Figueroa&author=G.+Burslem&title=Egfr+proteolysis+targeting+chimeric+molecules+and+associated+methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DEgfr%2520proteolysis%2520targeting%2520chimeric%2520molecules%2520and%2520associated%2520methods%2520of%2520use%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Modulators of Estrogen Receptor Proteolysis and Associated Methods of use</span>. <span class="NLM_patent">WO 2018140809 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+Qian&author=A.+P.+Crew&author=C.+M.+Crews&author=H.+Dong&author=K.+R.+Hornberger&author=J.+Wang&title=Modulators+of+Estrogen+Receptor+Proteolysis+and+Associated+Methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DY.%26atitle%3DModulators%2520of%2520Estrogen%2520Receptor%2520Proteolysis%2520and%2520Associated%2520Methods%2520of%2520use%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of the Proteasome-dependent and Vacuolar Pathways of Protein Degradation inSaccharomyces cerevisiae</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27280</span>– <span class="NLM_lpage">27284</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.44.27280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1074%2Fjbc.271.44.27280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=8910302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADyaK28XmvVWms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27280-27284&author=D.+H.+Leeauthor=A.+L.+Goldberg&title=Selective+Inhibitors+of+the+Proteasome-dependent+and+Vacuolar+Pathways+of+Protein+Degradation+inSaccharomyces+cerevisiae&doi=10.1074%2Fjbc.271.44.27280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of the proteasome-dependent and vacuolar pathways of protein degradation in Saccharomyces cerevisiae</span></div><div class="casAuthors">Lee, Do Hee; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">27280-27284</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have studied whether various agents that inhibit purified yeast and mammalian 26 S proteasome can suppress the breakdown of different classes of proteins in Saccharomyces cerevisiae.  The degrdn. of short-lived proteins was inhibited reversibly by peptide aldehyde inhibitors of proteasomes, carbobenzoxyl-leucinyl-leucinyl-leucinal (MG132) and carbobenzoxyl-leucinyl-leucinyl-norvalinal (MG115), in a yeast mutant with enhanced permeability, but not in wild-type strains.  Lactacystin, an irreversible proteasome inhibitor, had no effect, but the β-lactone deriv. of lactacystin, which directly reacts with proteasomes, inhibited the degrdn. of short-lived proteins.  These inhibitors also blocked the rapid ubiquitin-dependent breakdown of a β-galactosidase fusion protein and caused accumulation of enzymically active mols. in cells.  The degrdn. of the bulk of cell proteins, which are long-lived mols., was not blocked by proteasome inhibitors, but could be blocked by phenylmethylsulfonyl fluoride.  This agent, which inhibits multiple vacuolar proteases, did not affect the proteasome or breakdown of short-lived proteins.  These two classes of inhibitors can thus be used to distinguish the cytosolic and vacuolar proteolytic pathways and to increase the cellular content of short-lived proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Sxabgj574rVg90H21EOLACvtfcHk0lgSmHD64wa-7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvVWms78%253D&md5=95d1d47fcba959ddf297a94edc93f580</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.44.27280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.44.27280%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DSelective%2520Inhibitors%2520of%2520the%2520Proteasome-dependent%2520and%2520Vacuolar%2520Pathways%2520of%2520Protein%2520Degradation%2520inSaccharomyces%2520cerevisiae%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27280%26epage%3D27284%26doi%3D10.1074%2Fjbc.271.44.27280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0liLpAistYsynw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shaowen Xie, Yuan Sun, Yulin Liu, Xinnan Li, Xinuo Li, Wenyi Zhong, Feiyan Zhan, Jingjie Zhu, Hong Yao, Dong-Hua Yang, Zhe-Sheng Chen, Jinyi Xu, <span class="NLM_string-name hlFld-ContribAuthor">Shengtao Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Alectinib-Based PROTACs as Novel Potent Degraders of Anaplastic Lymphoma Kinase (ALK). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9120-9140. <a href="https://doi.org/10.1021/acs.jmedchem.1c00270" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00270</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00270%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BAlectinib-Based%252BPROTACs%252Bas%252BNovel%252BPotent%252BDegraders%252Bof%252BAnaplastic%252BLymphoma%252BKinase%252B%252528ALK%252529%26aulast%3DXie%26aufirst%3DShaowen%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D11022021%26date%3D28062021%26volume%3D64%26issue%3D13%26spage%3D9120%26epage%3D9140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Yujing  Zhang</span>, <span class="hlFld-ContribAuthor ">Dongming  Xing</span>, <span class="hlFld-ContribAuthor ">Renshuai  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs technology for targeting non-oncoproteins: advances and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>98 </em>, 105109. <a href="https://doi.org/10.1016/j.bioorg.2021.105109" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105109%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DPROTACs%252Btechnology%252Bfor%252Btargeting%252Bnon-oncoproteins%25253A%252Badvances%252Band%252Bperspectives%26aulast%3DWang%26aufirst%3DChao%26date%3D2021%26volume%3D98%26spage%3D105109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew J.  Henley</span>, <span class="hlFld-ContribAuthor ">Angela N.  Koehler</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in targeting ‘undruggable’ transcription factors with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>172 </em><a href="https://doi.org/10.1038/s41573-021-00199-0" title="DOI URL">https://doi.org/10.1038/s41573-021-00199-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00199-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00199-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DAdvances%252Bin%252Btargeting%252B%2525E2%252580%252598undruggable%2525E2%252580%252599%252Btranscription%252Bfactors%252Bwith%252Bsmall%252Bmolecules%26aulast%3DHenley%26aufirst%3DMatthew%2BJ.%26date%3D2021%26date%3D2021%26volume%3D172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan K.  Brownsey</span>, <span class="hlFld-ContribAuthor ">Ben C.  Rowley</span>, <span class="hlFld-ContribAuthor ">Evgueni  Gorobets</span>, <span class="hlFld-ContribAuthor ">Benjamin S.  Gelfand</span>, <span class="hlFld-ContribAuthor ">Darren J.  Derksen</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid synthesis of pomalidomide-conjugates for the development of protein degrader libraries. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (12)
                                     , 4519-4525. <a href="https://doi.org/10.1039/D0SC05442A" title="DOI URL">https://doi.org/10.1039/D0SC05442A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05442A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05442A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DRapid%252Bsynthesis%252Bof%252Bpomalidomide-conjugates%252Bfor%252Bthe%252Bdevelopment%252Bof%252Bprotein%252Bdegrader%252Blibraries%26aulast%3DBrownsey%26aufirst%3DDuncan%2BK.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D12%26spage%3D4519%26epage%3D4525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nayana  Devang</span>, <span class="hlFld-ContribAuthor ">Adyashree  Pani</span>, <span class="hlFld-ContribAuthor ">G.K.  Rajanikant</span>. </span><span class="cited-content_cbyCitation_article-title">Pseudokinases: Prospects for expanding the therapeutic targets armamentarium. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 121-185. <a href="https://doi.org/10.1016/bs.apcsb.2020.09.004" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.09.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.09.004%26sid%3Dliteratum%253Aachs%26atitle%3DPseudokinases%25253A%252BProspects%252Bfor%252Bexpanding%252Bthe%252Btherapeutic%252Btargets%252Barmamentarium%26aulast%3DDevang%26aufirst%3DNayana%26date%3D2021%26spage%3D121%26epage%3D185%26pub%3DElsevier%26atitle%3DProtein%252BKinases%252Bin%252BDrug%252BDiscovery%26date%3D2021%26volume%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. <sup>a</sup>Synthesis of Pyrrolotriazines <b>4–5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane, PdCl<sub>2</sub>(DtBPF), K<sub>3</sub>PO<sub>4</sub>, dioxane/water, 80 °C, 2 h (99%); (b) H<sub>2</sub>, Pd/C, MeOH, RT, 16 h (<b>4</b>, 11%; <b>5</b>, 15%).</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. <sup>a</sup>Synthesis of Pyrrolotriazine PROTACs <b>6–24</b> and Proteomics Probe <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaBH(OAc)<sub>3</sub>, AcOH, RT, 3 d (32%); (ii) H<sub>2</sub>, Pd/C, MeOH, RT, 1 h (92%); (b) 5-bromopyrrolo[2,1-<i>f</i>][1,2,4]triazin-4-ol, PyBOP, DIPEA, DMF, RT to 40 °C, 24 h (88%); (c) 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane, PdCl<sub>2</sub>(DtBPF), K<sub>2</sub>CO<sub>3</sub>, dioxane/water, 90 °C, 3 h (60%); (d) (i) H<sub>2</sub>, Pd/C, THF, RT, 3 d (86%); (ii) 4 M HCl, dioxane/DCM, RT, 2 h (99%); (e) carboxylic acid, HATU, DMF, RT, 3 h (6–53%); (f) dihydrofuran-2,5-dione, DIPEA, DCM, 50 °C 1 h (80%); (g) amine, HATU, DMF, RT, 3 h (24–47%); (h) (i) <i>tert</i>-butyl (14-amino-3,6,9,12-tetraoxatetradecyl)carbamate, HATU, DIPEA, DMF, RT, 18 h (82%); (ii) HCl, MeOH, RT, 1 h (29%); (i) beads, NEt<sub>3</sub>, DMSO, RT, 18 h.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. IRAK3 vs IRAK4 single-concentration data from Thermo Fisher Scientific kinome selectivity panels. Each marker represents a molecule tested at 1 μM. Colors denote a measure of hitting frequency, as calculated by pBSF<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (red, pBSF > 3; green, pBSF ≤ 3; gray, data unavailable). Labels refer to compound numbers used in this manuscript.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray crystallographic structure of <i>trans</i>-pyrolotriazine <b>4</b> in complex with IRAK4 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F3I">6F3I</a>), showing the <i>N</i>-acetylpiperazine pointing toward the solvent. Residues are colored based on similarity to IRAK3 (blue: identical, red: most different).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome selectivity of <b>5</b>, tested at 0.1 μM in a panel of 400 kinases. Inset: Only two kinases other than IRAK3 were identified with >50% activity: GSG2 (55%, IC<sub>50</sub> = 150 nM) and CDK11 (51%, IC<sub>50</sub> = 70 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinome selectivity of <b>23</b>, tested at 0.1 μM in a panel of 400 kinases. Inset: only four other kinases were identified with >50% activity: CDK11 (82%), CDK8 (69%), TRKC (60%), and GSG2 (57%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proteomic profiling for <b>23</b>: (A) dose–response curves using <b>23</b> after chemoproteomics selectivity profiling showing competed proteins hits with apparent binding (<i>K</i><sub>d,app</sub>) derived from RB<sub>50</sub>; (B) volcano plot showing fold changes of protein abundance from global proteomics analysis of THP1 cells treated with 0.1 μM of <b>23</b> for 8 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>23</b> induces potent and rapid degradation of IRAK3 in THP-1 cells: (A) cells treated with compound for 24 h at indicated doses and representative quantification of percent degradation relative to DMSO; (B) cells treated with 0.1 μM of compound for indicated times; (C) cells treated with 0.1 μM of compound for 16 h followed and harvested at indicated times after PROTAC washout.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PROTAC <b>23</b>-mediated degradation of IRAK3 in THP-1 cells is dependent on the ubiquitin proteasome pathway, IRAK3 and CRBN: (A) cells pretreated with DMSO, MG132 (10 μM),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or MLN4924 (5 μM)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (B) cells pretreated with <b>5</b> (10 μM) for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (C) cells pretreated with thalidomide (10 μM) for 2 h and then cotreated with <b>23</b> (0.1 μM), as indicated for 6 h; (D) cells treated with <b>23</b> or <b>24</b> at indicated doses for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/medium/jm0c01125_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. PROTAC <b>23</b> degrades IRAK3 in primary macrophages independent of LPS stimulation. (A) Primary macrophages treated with DMSO, compound <b>23</b> (0.3 μM) or compound <b>24</b> (0.3 μM) for indicated times. (B) Primary macrophages identically treated as in (A), but with LPS (100 ng/mL) treatment for 8 h prior to compound treatment for a total of 8, 24, or 32 h, as indicated. Data are representative of <i>n</i> = 6 donors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01125/20200918/images/large/jm0c01125_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01125&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.-F.</span></span> <span> </span><span class="NLM_article-title">IRAK family in inflammatory autoimmune diseases</span>. <i>Autoimmun. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">102461</span>, <span class="refDoi"> DOI: 10.1016/j.autrev.2020.102461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.autrev.2020.102461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31917263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVeksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=102461&author=L.-C.+Suauthor=W.-D.+Xuauthor=A.-F.+Huang&title=IRAK+family+in+inflammatory+autoimmune+diseases&doi=10.1016%2Fj.autrev.2020.102461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK family in inflammatory autoimmune diseases</span></div><div class="casAuthors">Su, Lin-Chong; Xu, Wang-Dong; Huang, An-Fang</div><div class="citationInfo"><span class="NLM_cas:title">Autoimmunity Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102461</span>CODEN:
                <span class="NLM_cas:coden">ARUEBU</span>;
        ISSN:<span class="NLM_cas:issn">1568-9972</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Innate immune signaling plays an important role in inflammation, and dysregulation of signaling components within this pathway has been focused as a crit. mediator in initiation, progression of inflammatory autoimmune diseases.  Toll-like receptors (TLRs) are the most upstream pattern recognition receptors in the immune cells, detecting pathogen assocd. mol. patterns, initiating signal transduction, by which interleukin-1 receptor-assocd. kinase (IRAK) family mediates activating signal from TLRs and interleukin-1 receptor.  The family comprises of four members, IRAK1, IRAK2, IRAK-M, IRAK4.  The family members have a role in either pos. or neg. regulation of innate immunity, adaptive immunity and inflammation.  Accumulated evidence proves that IRAK performs significantly in the pathogenesis of inflammatory autoimmune disorders.  On the one hand, both patients and animal modes reported abnormal expression of the family members.  On the other hand, functional study in vivo and in vitro demonstrated that the members are implicated in the development of the diseases.  Interestingly, IRAK inhibition has potential therapeutic benefits.  In this review, we focus on the family, review the physiol. roles in different immune cells, and summarize emerging data for highlighting the importance of them in inflammatory autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUEbD7FX0f7Vg90H21EOLACvtfcHk0lgn12nKSB1xIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVeksL8%253D&md5=2202adad3c59dd0bf7350cc6d22e8c46</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.autrev.2020.102461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.autrev.2020.102461%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.-C.%26aulast%3DXu%26aufirst%3DW.-D.%26aulast%3DHuang%26aufirst%3DA.-F.%26atitle%3DIRAK%2520family%2520in%2520inflammatory%2520autoimmune%2520diseases%26jtitle%3DAutoimmun.%2520Rev.%26date%3D2020%26volume%3D19%26spage%3D102461%26doi%3D10.1016%2Fj.autrev.2020.102461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeway, C. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medzhitov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flavell, R. A.</span></span> <span> </span><span class="NLM_article-title">IRAK-M is a negative regulator of Toll-like receptor signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(02)00827-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2FS0092-8674%2802%2900827-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=12150927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvV2htrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=191-202&author=K.+Kobayashiauthor=L.+D.+Hernandezauthor=J.+E.+Gal%C3%A1nauthor=C.+A.+Janewayauthor=R.+Medzhitovauthor=R.+A.+Flavell&title=IRAK-M+is+a+negative+regulator+of+Toll-like+receptor+signaling&doi=10.1016%2Fs0092-8674%2802%2900827-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-M is a negative regulator of toll-like receptor signaling</span></div><div class="casAuthors">Kobayashi, Koichi; Hernandez, Lorraine D.; Galan, Jorge E.; Janeway, Charles A., Jr.; Medzhitov, Ruslan; Flavell, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">191-202</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) detect microorganisms and protect multicellular organisms from infection.  TLRs transduce their signals through MyD88 and the serine/threonine kinase IRAK.  The IRAK family consists of two active kinases, IRAK and IRAK-4, and two inactive kinases, IRAK-2 and IRAK-M.  IRAK-M expression is restricted to monocytes/macrophages, whereas other IRAKs are ubiquitous.  We show here that IRAK-M is induced upon TLR stimulation and neg. regulates TLR signaling.  IRAK-M prevented dissocn. of IRAK and IRAK-4 from MyD88 and formation of IRAK-TRAF6 complexes.  IRAK-M-/- cells exhibited increased cytokine prodn. upon TLR/IL-1 stimulation and bacterial challenge, and IRAK-M-/- mice showed increased inflammatory responses to bacterial infection.  Endotoxin tolerance, a protection mechanism against endotoxin shock, was significantly reduced in IRAK-M-/- cells.  Thus, IRAK-M regulates TLR signaling and innate immune homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKFscWDZfrLLVg90H21EOLACvtfcHk0lgn12nKSB1xIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvV2htrw%253D&md5=80dc28f4773127eac27aa2755e1b2195</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900827-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900827-9%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DHernandez%26aufirst%3DL.%2BD.%26aulast%3DGal%25C3%25A1n%26aufirst%3DJ.%2BE.%26aulast%3DJaneway%26aufirst%3DC.%2BA.%26aulast%3DMedzhitov%26aufirst%3DR.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26atitle%3DIRAK-M%2520is%2520a%2520negative%2520regulator%2520of%2520Toll-like%2520receptor%2520signaling%26jtitle%3DCell%26date%3D2002%26volume%3D110%26spage%3D191%26epage%3D202%26doi%3D10.1016%2Fs0092-8674%2802%2900827-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starczynowski, D. T.</span></span> <span> </span><span class="NLM_article-title">IRAK signalling in cancer</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1038/bjc.2014.513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fbjc.2014.513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=25290089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCis7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=232-237&author=G.+W.+Rhyasenauthor=D.+T.+Starczynowski&title=IRAK+signalling+in+cancer&doi=10.1038%2Fbjc.2014.513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK signalling in cancer</span></div><div class="casAuthors">Rhyasen, G. W.; Starczynowski, D. T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">232-237</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Innate immune signalling has an essential role in inflammation, and the dysregulation of signalling components of this pathway is increasingly being recognized as an important mediator in cancer initiation and progression.  In some malignancies, dysregulation of inflammatory toll-like receptor (TLR) and interleukin-1 receptor (IL1R) signalling is typified by increased NF-κB activity, and it occurs through somatic mutations, chromosomal deletions, and/or transcriptional deregulation.  Interleukin-1 receptor-assocd. kinase (IRAK) family members are mediators of TLR/IL1R superfamily signalling, and mounting evidence implicates these kinases as viable cancer targets.  Although there have been previous efforts aimed at the development of IRAK kinase inhibitors, this is currently an area of renewed interest for cancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGshpWhcF0K7Vg90H21EOLACvtfcHk0lgn12nKSB1xIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCis7zJ&md5=c88416c9bc7754d997d87a489b997fa3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2014.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2014.513%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DStarczynowski%26aufirst%3DD.%2BT.%26atitle%3DIRAK%2520signalling%2520in%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2015%26volume%3D112%26spage%3D232%26epage%3D237%26doi%3D10.1038%2Fbjc.2014.513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Neutrophils deficient in innate suppressor IRAK-M enhances anti-tumor immune responses</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2019.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ymthe.2019.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31607540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2ltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=89-99&author=Y.+Zhangauthor=N.+Diaoauthor=C.+K.+Leeauthor=H.+W.+Chuauthor=L.+Baiauthor=L.+Li&title=Neutrophils+deficient+in+innate+suppressor+IRAK-M+enhances+anti-tumor+immune+responses&doi=10.1016%2Fj.ymthe.2019.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Neutrophils Deficient in Innate Suppressor IRAK-M Enhances Anti-tumor Immune Responses</span></div><div class="casAuthors">Zhang, Yao; Diao, Na; Lee, Christina K.; Chu, Hong Wei; Bai, Lan; Li, Liwu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-99</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0024</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tumor-assocd. immune-suppressive neutrophils are prevalent in various cancers, including colorectal cancer.  However, mechanisms of immune-suppressive neutrophils are not well understood.  We report that a key innate suppressor, IRAK-M (interleukin-1 receptor-assocd. kinase M), is critically involved in the establishment of immune-suppressive neutrophils.  In contrast to the wild-type (WT) neutrophils exhibiting immune-suppressive signatures of CD11bhighPD-L1highCD80low, IRAK-M-deficient neutrophils are rewired with reduced levels of inhibitory mols. PD-L1 and CD11b, as well as enhanced expression of stimulatory mols. CD80 and CD40.  The reprogramming of IRAK-M-deficient neutrophils is mediated by reduced activation of STAT1/3 and enhanced activation of STAT5.  As a consequence, IRAK-M-deficient neutrophils demonstrate enhanced capability to promote, instead of suppress, the proliferation and activation of effector T cells both in vitro and in vivo.  Functionally, we obsd. that the transfusion of IRAK-M-/- neutrophils can potently render an enhanced anti-tumor immune response in the murine inflammation-induced colorectal cancer model.  Collectively, our study defines IRAK-M as an innate suppressor for neutrophil function and reveals IRAK-M as a promising target for rewiring neutrophils in anti-cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-UavWHJH07Vg90H21EOLACvtfcHk0lgqaai7g68qoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2ltro%253D&md5=59b0d0cbb080b216de8413a16fb56a60</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2019.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2019.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DC.%2BK.%26aulast%3DChu%26aufirst%3DH.%2BW.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DNeutrophils%2520deficient%2520in%2520innate%2520suppressor%2520IRAK-M%2520enhances%2520anti-tumor%2520immune%2520responses%26jtitle%3DMol.%2520Ther.%26date%3D2020%26volume%3D28%26spage%3D89%26epage%3D99%26doi%3D10.1016%2Fj.ymthe.2019.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kesselring, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaesner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiergeist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naschberger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wege, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croner, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neurath, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlitt, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fichtner-Feigl, S.</span></span> <span> </span><span class="NLM_article-title">IRAK-M expression in tumor cells supports colorectal cancer progression through reduction of antimicrobial defense and stabilization of STAT3</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">684</span>– <span class="NLM_lpage">696</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ccell.2016.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=27150039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFCitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=684-696&author=R.+Kesselringauthor=J.+Glaesnerauthor=A.+Hiergeistauthor=E.+Naschbergerauthor=H.+Neumannauthor=S.+M.+Brunnerauthor=A.+K.+Wegeauthor=C.+Seebauerauthor=G.+K%C3%B6hlauthor=S.+Merklauthor=R.+S.+Cronerauthor=C.+Hacklauthor=M.+St%C3%BCrzlauthor=M.+F.+Neurathauthor=A.+Gessnerauthor=H.-J.+Schlittauthor=E.+K.+Geisslerauthor=S.+Fichtner-Feigl&title=IRAK-M+expression+in+tumor+cells+supports+colorectal+cancer+progression+through+reduction+of+antimicrobial+defense+and+stabilization+of+STAT3&doi=10.1016%2Fj.ccell.2016.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-M Expression in Tumor Cells Supports Colorectal Cancer Progression through Reduction of Antimicrobial Defense and Stabilization of STAT3</span></div><div class="casAuthors">Kesselring, Rebecca; Glaesner, Joachim; Hiergeist, Andreas; Naschberger, Elisabeth; Neumann, Helmut; Brunner, Stefan M.; Wege, Anja K.; Seebauer, Caroline; Koehl, Gudrun; Merkl, Susanne; Croner, Roland S.; Hackl, Christina; Stuerzl, Michael; Neurath, Markus F.; Gessner, Andre; Schlitt, Hans-Juergen; Geissler, Edward K.; Fichtner-Feigl, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">684-696</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Colorectal cancer (CRC) is assocd. with loss of epithelial barrier integrity, which facilitates the interaction of the immunol. microenvironment with the luminal microbiome, eliciting tumor-supportive inflammation.  An important regulator of intestinal inflammatory responses is IRAK-M, a neg. regulator of TLR signaling.  Here we investigate the compartment-specific impact of IRAK-M on colorectal carcinogenesis using a mouse model.  We demonstrate that IRAK-M is expressed in tumor cells due to combined TLR and Wnt activation.  Tumor cell-intrinsic IRAK-M is responsible for regulation of microbial colonization of tumors and STAT3 protein stability in tumor cells, leading to tumor cell proliferation.  IRAK-M expression in human CRCs is assocd. with poor prognosis.  These results suggest that IRAK-M may be a potential therapeutic target for CRC treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7FxdAcfVYGbVg90H21EOLACvtfcHk0lgqaai7g68qoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFCitbs%253D&md5=89338ee725e5b6b48b1ba11d9fa30935</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DKesselring%26aufirst%3DR.%26aulast%3DGlaesner%26aufirst%3DJ.%26aulast%3DHiergeist%26aufirst%3DA.%26aulast%3DNaschberger%26aufirst%3DE.%26aulast%3DNeumann%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DS.%2BM.%26aulast%3DWege%26aufirst%3DA.%2BK.%26aulast%3DSeebauer%26aufirst%3DC.%26aulast%3DK%25C3%25B6hl%26aufirst%3DG.%26aulast%3DMerkl%26aufirst%3DS.%26aulast%3DCroner%26aufirst%3DR.%2BS.%26aulast%3DHackl%26aufirst%3DC.%26aulast%3DSt%25C3%25BCrzl%26aufirst%3DM.%26aulast%3DNeurath%26aufirst%3DM.%2BF.%26aulast%3DGessner%26aufirst%3DA.%26aulast%3DSchlitt%26aufirst%3DH.-J.%26aulast%3DGeissler%26aufirst%3DE.%2BK.%26aulast%3DFichtner-Feigl%26aufirst%3DS.%26atitle%3DIRAK-M%2520expression%2520in%2520tumor%2520cells%2520supports%2520colorectal%2520cancer%2520progression%2520through%2520reduction%2520of%2520antimicrobial%2520defense%2520and%2520stabilization%2520of%2520STAT3%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D684%26epage%3D696%26doi%3D10.1016%2Fj.ccell.2016.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras as therapeutics and tools for biological discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras+as+therapeutics+and+tools+for+biological+discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0lgqaai7g68qoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%2520as%2520therapeutics%2520and%2520tools%2520for%2520biological%2520discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ’beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%99beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lgRZHbvvU52jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span> <span> </span><span class="NLM_article-title">Prospects for pharmacological targeting of pseudokinases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0018-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fs41573-019-0018-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=30850748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=501-526&author=J.+E.+Kungauthor=N.+Jura&title=Prospects+for+pharmacological+targeting+of+pseudokinases&doi=10.1038%2Fs41573-019-0018-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for pharmacological targeting of pseudokinases</span></div><div class="casAuthors">Kung, Jennifer E.; Jura, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">501-526</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Pseudokinases are members of the protein kinase superfamily but signal primarily through noncatalytic mechanisms.  Many pseudokinases contribute to the pathologies of human diseases, yet they remain largely unexplored as drug targets owing to challenges assocd. with modulation of their biol. functions.  Our understanding of the structure and physiol. roles of pseudokinases has improved substantially over the past decade, revealing intriguing similarities between pseudokinases and their catalytically active counterparts.  Pseudokinases often adopt conformations that are analogous to those seen in catalytically active kinases and, in some cases, can also bind metal cations and/or nucleotides.  Several clin. approved kinase inhibitors have been shown to influence the noncatalytic functions of active kinases, providing hope that similar properties in pseudokinases could be pharmacol. regulated.  In this Review, we discuss known roles of pseudokinases in disease, their unique structural features and the progress that has been made towards developing pseudokinase-directed therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFxQnlHTnn7Vg90H21EOLACvtfcHk0lgRZHbvvU52jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D&md5=f3414770154e8d6449b454f9bc3f19bd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0018-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0018-3%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BE.%26aulast%3DJura%26aufirst%3DN.%26atitle%3DProspects%2520for%2520pharmacological%2520targeting%2520of%2520pseudokinases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D501%26epage%3D526%26doi%3D10.1038%2Fs41573-019-0018-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span>; <span class="NLM_string-name">Tinworth, C.</span></span> <span> </span><span class="NLM_article-title">Novel compounds</span>. <span class="NLM_patent">WO 2017211924 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+D.+Harling&author=C.+Tinworth&title=Novel+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DNovel%2520compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span>; <span class="NLM_string-name">Tinworth, C.</span></span> <span> </span><span class="NLM_article-title">Novel compounds</span>. <span class="NLM_patent">WO 2018033556 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=J.+D.+Harling&author=C.+Tinworth&title=Novel+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DNovel%2520compounds%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culshaw, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groombridge, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marden, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhirter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Pyrrolopyrimidine Inhibitors of Interleukin-1 Receptor Associated Kinase 4 (IRAK4) for the Treatment of Mutant MYD88L265P Diffuse Large B-Cell Lymphoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10071</span>– <span class="NLM_lpage">10091</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10071-10091&author=J.+S.+Scottauthor=S.+L.+Degorceauthor=R.+Anjumauthor=J.+Culshawauthor=R.+D.+M.+Daviesauthor=N.+L.+Daviesauthor=K.+S.+Dillmanauthor=J.+E.+Dowlingauthor=L.+Drewauthor=A.+D.+Fergusonauthor=S.+D.+Groombridgeauthor=C.+T.+Halsallauthor=J.+A.+Hudsonauthor=S.+Lamontauthor=N.+A.+Lindsayauthor=S.+K.+Mardenauthor=M.+F.+Mayoauthor=J.+E.+Peaseauthor=D.+R.+Perkinsauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=A.+Rosenauthor=M.+Shenauthor=C.+McWhirterauthor=D.+Wu&title=Discovery+and+Optimization+of+Pyrrolopyrimidine+Inhibitors+of+Interleukin-1+Receptor+Associated+Kinase+4+%28IRAK4%29+for+the+Treatment+of+Mutant+MYD88L265P+Diffuse+Large+B-Cell+Lymphoma&doi=10.1021%2Facs.jmedchem.7b01290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Pyrrolopyrimidine Inhibitors of Interleukin-1 Receptor Associated Kinase 4 (IRAK4) for the Treatment of Mutant MYD88L265P Diffuse Large B-Cell Lymphoma</span></div><div class="casAuthors">Scott, James S.; Degorce, Sebastien L.; Anjum, Rana; Culshaw, Janet; Davies, Robert D. M.; Davies, Nichola L.; Dillman, Keith S.; Dowling, James E.; Drew, Lisa; Ferguson, Andrew D.; Groombridge, Sam D.; Halsall, Christopher T.; Hudson, Julian A.; Lamont, Scott; Lindsay, Nicola A.; Marden, Stacey K.; Mayo, Michele F.; Pease, J. Elizabeth; Perkins, David R.; Pink, Jennifer H.; Robb, Graeme R.; Rosen, Alan; Shen, Minhui; McWhirter, Claire; Wu, Dedong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10071-10091</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the optimization of a series of pyrrolopyrimidine inhibitors of interleukin-1 receptor assocd. kinase 4 (IRAK4) using x-ray crystal structures and structure based design to identify and optimize our scaffold.  Compd. 28 demonstrated a favorable physicochem. and kinase selectivity profile and was identified as a promising in vivo tool with which to explore the role of IRAK4 inhibition in the treatment of mutant MYD88L265P diffuse large B-cell lymphoma (DLBCL).  Compd. 28 was shown to be capable of demonstrating inhibition of NF-κB activation and growth of the ABC subtype of DLBCL cell lines in vitro at high concns. but showed greater effects in combination with a BTK inhibitor at lower concns.  In vivo, the combination of compd. 28 and ibrutinib led to tumor regression in an ABC-DLBCL mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBCn7WCzPrrrVg90H21EOLACvtfcHk0lhqde2Ri6eYoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrjO&md5=9760f425ec33b238a43e1de0ec6d24f1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01290%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DCulshaw%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DR.%2BD.%2BM.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGroombridge%26aufirst%3DS.%2BD.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DHudson%26aufirst%3DJ.%2BA.%26aulast%3DLamont%26aufirst%3DS.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMarden%26aufirst%3DS.%2BK.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3DPerkins%26aufirst%3DD.%2BR.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRosen%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Pyrrolopyrimidine%2520Inhibitors%2520of%2520Interleukin-1%2520Receptor%2520Associated%2520Kinase%25204%2520%2528IRAK4%2529%2520for%2520the%2520Treatment%2520of%2520Mutant%2520MYD88L265P%2520Diffuse%2520Large%2520B-Cell%2520Lymphoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10071%26epage%3D10091%26doi%3D10.1021%2Facs.jmedchem.7b01290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumming, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span> <span> </span><span class="NLM_article-title">Lowering lipophilicity by adding carbon: one-carbon bridges of morpholines and piperazines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8934</span>– <span class="NLM_lpage">8943</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8934-8943&author=S.+L.+Degorceauthor=M.+S.+Bodnarchukauthor=I.+A.+Cummingauthor=J.+S.+Scott&title=Lowering+lipophilicity+by+adding+carbon%3A+one-carbon+bridges+of+morpholines+and+piperazines&doi=10.1021%2Facs.jmedchem.8b01148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Lowering Lipophilicity by Adding Carbon: One-Carbon Bridges of Morpholines and Piperazines</span></div><div class="casAuthors">Degorce, Sebastien L.; Bodnarchuk, Michael S.; Cumming, Iain A.; Scott, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8934-8943</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, we report our investigation of a phenomenon by which bridging morpholines across the ring with one-carbon tethers leads to a counterintuitive redn. in lipophilicity.  This effect was also found to occur in piperazines and piperidines and lowered the measured log D7.4 of the bridged mols. by as much as -0.8 relative to their unbridged counterparts.  As lowering lipophilicity without introducing addnl. heteroatoms can be desirable, we believe this potentially provides a useful tactic to improve the drug-like properties of mols. contg. morpholine-, piperazine-, and piperidine-like motifs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhZJB3SYRKD7Vg90H21EOLACvtfcHk0lhqde2Ri6eYoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK&md5=27f30b8220ea767d6ef1e492f3fa1aba</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01148%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DCumming%26aufirst%3DI.%2BA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26atitle%3DLowering%2520lipophilicity%2520by%2520adding%2520carbon%253A%2520one-carbon%2520bridges%2520of%2520morpholines%2520and%2520piperazines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8934%26epage%3D8943%26doi%3D10.1021%2Facs.jmedchem.8b01148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groombridge, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Optimization of permeability in a series of pyrrolotriazine inhibitors of IRAK4</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.bmc.2018.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=29398441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=913-924&author=S.+L.+Degorceauthor=R.+Anjumauthor=K.+S.+Dillmanauthor=L.+Drewauthor=S.+D.+Groombridgeauthor=C.+T.+Halsallauthor=E.+M.+Lenzauthor=N.+A.+Lindsayauthor=M.+F.+Mayoauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=J.+S.+Scottauthor=S.+Stokesauthor=Y.+Xue&title=Optimization+of+permeability+in+a+series+of+pyrrolotriazine+inhibitors+of+IRAK4&doi=10.1016%2Fj.bmc.2018.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of permeability in a series of pyrrolotriazine inhibitors of IRAK4</span></div><div class="casAuthors">Degorce, Sebastien L.; Anjum, Rana; Dillman, Keith S.; Drew, Lisa; Groombridge, Sam D.; Halsall, Christopher T.; Lenz, Eva M.; Lindsay, Nicola A.; Mayo, Michele F.; Pink, Jennifer H.; Robb, Graeme R.; Scott, James S.; Stokes, Stephen; Xue, Yafeng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">913-924</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors have developed a series of orally efficacious IRAK4 inhibitors, based on a scaffold hopping strategy and using rational structure based design.  Efforts to tackle low permeability and high efflux in the authors' previously reported pyrrolopyrimidine series (Scott et al., 2017) led to the identification of pyrrolotriazines which contained one less formal hydrogen bond donor and were intrinsically more lipophilic.  Further optimization of substituents on this pyrrolotriazine core culminated with the discovery of compd. 30 (trans-1-(4-(4-((5-isopropylpyrrolo[2,1-f]-[1,2,4]triazin-4-yl)amino)cyclohexyl)piperazin-1-yl)ethanone) as a promising in vivo probe to assess the potential of IRAK4 inhibition for the treatment of MyD88 mutant Diffuse Large B Cell Lymphoma (DLBCL) in combination with a BTK inhibitor.  When tested in an ABC-DLBCL model with a dual MyD88/CD79 mutation (OCI-LY10), 30 demonstrated tumor regressions in combination with ibrutinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao5i7H0tVFrVg90H21EOLACvtfcHk0lgLcfZqJSXBwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSitLc%253D&md5=772fab4ca7c899bb85bcd9ea8a9d47f3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DGroombridge%26aufirst%3DS.%2BD.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DLenz%26aufirst%3DE.%2BM.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DOptimization%2520of%2520permeability%2520in%2520a%2520series%2520of%2520pyrrolotriazine%2520inhibitors%2520of%2520IRAK4%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D913%26epage%3D924%26doi%3D10.1016%2Fj.bmc.2018.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+frequent-hitter+behavior+based+on+historical+high-throughput+screening+data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0lgLcfZqJSXBwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520frequent-hitter%2520behavior%2520based%2520on%2520historical%2520high-throughput%2520screening%2520data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloecher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsall, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pink, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of 5-azaquinazolines as orally efficacious IRAK4 inhibitors targeting MyD88L265P mutant diffuse large B cell lymphoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9918</span>– <span class="NLM_lpage">9930</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyitLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9918-9930&author=S.+L.+Degorceauthor=R.+Anjumauthor=A.+Bloecherauthor=R.+J.+Carbajoauthor=K.+S.+Dillmanauthor=L.+Drewauthor=C.+T.+Halsallauthor=E.+M.+Lenzauthor=N.+A.+Lindsayauthor=M.+F.+Mayoauthor=J.+H.+Pinkauthor=G.+R.+Robbauthor=A.+Rosenauthor=J.+S.+Scottauthor=Y.+Xue&title=Discovery+of+a+series+of+5-azaquinazolines+as+orally+efficacious+IRAK4+inhibitors+targeting+MyD88L265P+mutant+diffuse+large+B+cell+lymphoma&doi=10.1021%2Facs.jmedchem.9b01346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Series of 5-Azaquinazolines as Orally Efficacious IRAK4 Inhibitors Targeting MyD88L265P Mutant Diffuse Large B Cell Lymphoma</span></div><div class="casAuthors">Degorce, Sebastien L.; Anjum, Rana; Bloecher, Andrew; Carbajo, Rodrigo J.; Dillman, Keith S.; Drew, Lisa; Halsall, Christopher T.; Lenz, Eva M.; Lindsay, Nicola A.; Mayo, Michele F.; Pink, Jennifer H.; Robb, Graeme R.; Rosen, Alan; Scott, James S.; Xue, Yafeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9918-9930</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, the authors report the discovery of a series of 5-azaquinazolines as selective IRAK4 inhibitors.  From modestly potent quinazoline 4, the authors introduced a 5-aza substitution to mask the 4-NH hydrogen bond donor (HBD).  This allowed us to substitute the core with a 2-aminopyrazole, which showed large gains in cellular potency despite the addnl. formal HBD.  Further optimization led to 6-cyanomethyl-5-azaquinazoline 13, a selective IRAK4 inhibitor which proved efficacious in combination with ibrutinib, while showing very little activity as a single agent up to 100 mg/kg.  This contrasted to previously reported IRAK4 inhibitors that exhibited efficacy in the same model as single agents, and was attributed to the enhanced specificity of 13 towards IRAK4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9B-WmMsvfDrVg90H21EOLACvtfcHk0lgksQt2r6-nnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyitLzE&md5=5029cfe2e62ca9bc3e7e9a0b02ab8cc0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01346%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DBloecher%26aufirst%3DA.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DHalsall%26aufirst%3DC.%2BT.%26aulast%3DLenz%26aufirst%3DE.%2BM.%26aulast%3DLindsay%26aufirst%3DN.%2BA.%26aulast%3DMayo%26aufirst%3DM.%2BF.%26aulast%3DPink%26aufirst%3DJ.%2BH.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRosen%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%25205-azaquinazolines%2520as%2520orally%2520efficacious%2520IRAK4%2520inhibitors%2520targeting%2520MyD88L265P%2520mutant%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9918%26epage%3D9930%26doi%3D10.1021%2Facs.jmedchem.9b01346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camenisch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Waterbeemd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkers, G.</span></span> <span> </span><span class="NLM_article-title">Estimation of permeability by passive diffusion through Caco-2 cell monolayers using the drugs’ lipophilicity and molecular weight</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/s0928-0987(97)10019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2FS0928-0987%2897%2910019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=9795088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A280%3ADC%252BD2cngsFyrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=317-324&author=G.+Camenischauthor=J.+Alsenzauthor=H.+van+de+Waterbeemdauthor=G.+Folkers&title=Estimation+of+permeability+by+passive+diffusion+through+Caco-2+cell+monolayers+using+the+drugs%E2%80%99+lipophilicity+and+molecular+weight&doi=10.1016%2Fs0928-0987%2897%2910019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of permeability by passive diffusion through Caco-2 cell monolayers using the drugs' lipophilicity and molecular weight</span></div><div class="casAuthors">Camenisch G; Alsenz J; van de Waterbeemd H; Folkers G</div><div class="citationInfo"><span class="NLM_cas:title">European journal of pharmaceutical sciences : official journal of the European Federation for Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-24</span>
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    </div><div class="casAbstract">A recently developed, new theoretical absorption model for passive diffusion through biological membranes describing the dependency of membrane permeability on lipophilicity and molecular size, predicts different sigmoid-hyperbolic permeability-lipophilicity relationships for different molecular weight ranges.  This model has been tested with experimental in vitro cultured epithelial cell (Caco-2) permeability data for structurally diverse drugs differing in lipophilicity, ionization state and molecular size.  These data were pooled with literature values.  Using this simple physicochemical approach, the permeability of a compound through Caco-2 cells by passive diffusion can be predicted from the compounds' distribution coefficient in 1-octanol/water (log D(oct)) and its molecular weight (MW).  Deviations from this expected behaviour may point to the involvement of biological components in the transport process, which may require further investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSF9qdSVu_jQBfKb6ELK8t0fW6udTcc2eYr20XKgkebzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2cngsFyrsQ%253D%253D&md5=c1d933c4bc0b5356feed693bb0c45cbe</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2897%2910019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252897%252910019-7%26sid%3Dliteratum%253Aachs%26aulast%3DCamenisch%26aufirst%3DG.%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DFolkers%26aufirst%3DG.%26atitle%3DEstimation%2520of%2520permeability%2520by%2520passive%2520diffusion%2520through%2520Caco-2%2520cell%2520monolayers%2520using%2520the%2520drugs%25E2%2580%2599%2520lipophilicity%2520and%2520molecular%2520weight%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D1998%26volume%3D6%26spage%3D317%26epage%3D324%26doi%3D10.1016%2Fs0928-0987%2897%2910019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs) — Past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29+%E2%80%94+Past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eYr20XKgkebzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%2520%25E2%2580%2594%2520Past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugations for the degradation of oncogenic PI3K</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1016%2Fj.ejmech.2018.03.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=29625382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntFOktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=237-247&author=W.+Liauthor=C.+Gaoauthor=L.+Zhaoauthor=Z.+Yuanauthor=Y.+Chenauthor=Y.+Jiang&title=Phthalimide+conjugations+for+the+degradation+of+oncogenic+PI3K&doi=10.1016%2Fj.ejmech.2018.03.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugations for the degradation of oncogenic PI3K</span></div><div class="casAuthors">Li, Wenlu; Gao, Chunmei; Zhao, Lei; Yuan, Zigao; Chen, Yuzong; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">237-247</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">PI3K/Akt/mTOR pathway is crucial for carcinogenesis and its inhibitors have made a great progress in cancer treatment.  However, there is still a great developing space for PI3K inhibitors as the acquired drug resistance hindered their application in clin.  Proteolysis-targeting chimeras (PROTACs) with the potential to handle the challenges faced in drug development could be an alternative therapeutic strategy.  Moreover, the past two years have witnessed remarkable advances in the development of phthalimide conjugation as a strategy for the degrdn. instead of inhibition of the targets, including BET family proteins, Sirtuin 2, CDK 9, Smad 3, and BCR-ABL proteins.  Here, we designed and synthesized a series of potential small mol. PROTACs for the degrdn. of PI3K.  Four compds. induced remarkable PI3K degrdn. and down-regulated the phosphorylation of Akt, S6K and GSK-3β in liver cancer cells HepG2.  Furthermore, the representative compd. D proved to inhibit tumor cells proliferation by the induction of autophagy instead of apoptosis or cell cycle arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0E7-HLm8E5bVg90H21EOLACvtfcHk0lhhr1A5GKSFow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntFOktbo%253D&md5=f82652f95e23fc9088d53e541ebe1ad4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.066%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DPhthalimide%2520conjugations%2520for%2520the%2520degradation%2520of%2520oncogenic%2520PI3K%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D237%26epage%3D247%26doi%3D10.1016%2Fj.ejmech.2018.03.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0lhhr1A5GKSFow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newham, P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras in drug development: A safety perspective</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1111/bph.15014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1111%2Fbph.15014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=32022252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2020&pages=1709-1718&author=K.+Moreauauthor=M.+Coenauthor=A.+X.+Zhangauthor=F.+Pachlauthor=M.+P.+Castaldiauthor=G.+Dahlauthor=H.+Boydauthor=C.+Scottauthor=P.+Newham&title=Proteolysis-targeting+chimeras+in+drug+development%3A+A+safety+perspective&doi=10.1111%2Fbph.15014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimeras in drug development: A safety perspective</span></div><div class="casAuthors">Moreau, Kevin; Coen, Muireann; Zhang, Andrew X.; Pachl, Fiona; Castaldi, M. Paola; Dahl, Goran; Boyd, Helen; Scott, Clay; Newham, Pete</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1709-1718</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras are a new drug modality that exploits the endogenous ubiquitin proteasome system to degrade a protein of interest for therapeutic benefit.  As the first-generation of proteolysis-targeting chimeras have now entered clin. trials for oncol. indications, it is timely to consider the theor. safety risks inherent with this modality which include off-target degrdn., intracellular accumulation of natural substrates for the E3 ligases used in the ubiquitin proteasome system, proteasome satn. by ubiquitinated proteins, and liabilities assocd. with the "hook effect" of proteolysis-targeting chimeras This review describes in vitro and non-clin. in vivo data that provide mechanistic insight of these safety risks and approaches being used to mitigate these risks in the next generation of proteolysis-targeting chimera mols. to extend therapeutic applications beyond life-threatening diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreTK_TldLxHbVg90H21EOLACvtfcHk0lhoPiy31cu0tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D&md5=1e26bbe1d623f2f8f4218c2e3fad9dd6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1111%2Fbph.15014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.15014%26sid%3Dliteratum%253Aachs%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DA.%2BX.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DNewham%26aufirst%3DP.%26atitle%3DProteolysis-targeting%2520chimeras%2520in%2520drug%2520development%253A%2520A%2520safety%2520perspective%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2020%26volume%3D177%26spage%3D1709%26epage%3D1718%26doi%3D10.1111%2Fbph.15014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Snyder, L. B.</span></span> <span> </span><span class="NLM_article-title">Cereblon ligands and bifunctional compounds comprising the same</span>. U.S. Patent <span class="NLM_patent">20180228907 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=C.+M.+Crews&author=H.+Dong&author=K.+R.+Hornberger&author=J.+Wang&author=Y.+Qian&author=K.+Zimmermann&author=M.+Berlin&author=L.+B.+Snyder&title=Cereblon+ligands+and+bifunctional+compounds+comprising+the+same"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DCereblon%2520ligands%2520and%2520bifunctional%2520compounds%2520comprising%2520the%2520same%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Snyder, L. B.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span></span> <span> </span><span class="NLM_article-title">Compounds and methods for the targeted degradation of androgen receptor</span>. U.S. Patent <span class="NLM_patent">20180099940 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=K.+R.+Hornberger&author=L.+B.+Snyder&author=K.+Zimmermann&author=J.+Wang&author=M.+Berlin&author=C.+M.+Crews&author=H.+Dong&title=Compounds+and+methods+for+the+targeted+degradation+of+androgen+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DCompounds%2520and%2520methods%2520for%2520the%2520targeted%2520degradation%2520of%2520androgen%2520receptor%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Tetrahydronaphthalene and Tetrahydroisoquinoline Derivatives as Estrogen Receptor Degraders and their Preparation</span>. <span class="NLM_patent">WO 2018102725 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=Y.+Qian&author=H.+Dong&author=J.+Wang&author=K.+R.+Hornberger&author=C.+M.+Crews&title=Tetrahydronaphthalene+and+Tetrahydroisoquinoline+Derivatives+as+Estrogen+Receptor+Degraders+and+their+Preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DTetrahydronaphthalene%2520and%2520Tetrahydroisoquinoline%2520Derivatives%2520as%2520Estrogen%2520Receptor%2520Degraders%2520and%2520their%2520Preparation%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Zimmermann, K.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Ishchenko, A.</span>; <span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Jaime-Figueroa, S.</span>; <span class="NLM_string-name">Burslem, G.</span></span> <span> </span><span class="NLM_article-title">Egfr proteolysis targeting chimeric molecules and associated methods of use</span>. <span class="NLM_patent">WO 2018119441 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+P.+Crew&author=K.+Zimmermann&author=J.+Wang&author=M.+Berlin&author=H.+Dong&author=A.+Ishchenko&author=Y.+Qian&author=C.+M.+Crews&author=S.+Jaime-Figueroa&author=G.+Burslem&title=Egfr+proteolysis+targeting+chimeric+molecules+and+associated+methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26atitle%3DEgfr%2520proteolysis%2520targeting%2520chimeric%2520molecules%2520and%2520associated%2520methods%2520of%2520use%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Hornberger, K. R.</span>; <span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Modulators of Estrogen Receptor Proteolysis and Associated Methods of use</span>. <span class="NLM_patent">WO 2018140809 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+Qian&author=A.+P.+Crew&author=C.+M.+Crews&author=H.+Dong&author=K.+R.+Hornberger&author=J.+Wang&title=Modulators+of+Estrogen+Receptor+Proteolysis+and+Associated+Methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DY.%26atitle%3DModulators%2520of%2520Estrogen%2520Receptor%2520Proteolysis%2520and%2520Associated%2520Methods%2520of%2520use%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of the Proteasome-dependent and Vacuolar Pathways of Protein Degradation inSaccharomyces cerevisiae</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27280</span>– <span class="NLM_lpage">27284</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.44.27280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1074%2Fjbc.271.44.27280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=8910302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADyaK28XmvVWms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27280-27284&author=D.+H.+Leeauthor=A.+L.+Goldberg&title=Selective+Inhibitors+of+the+Proteasome-dependent+and+Vacuolar+Pathways+of+Protein+Degradation+inSaccharomyces+cerevisiae&doi=10.1074%2Fjbc.271.44.27280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of the proteasome-dependent and vacuolar pathways of protein degradation in Saccharomyces cerevisiae</span></div><div class="casAuthors">Lee, Do Hee; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">27280-27284</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have studied whether various agents that inhibit purified yeast and mammalian 26 S proteasome can suppress the breakdown of different classes of proteins in Saccharomyces cerevisiae.  The degrdn. of short-lived proteins was inhibited reversibly by peptide aldehyde inhibitors of proteasomes, carbobenzoxyl-leucinyl-leucinyl-leucinal (MG132) and carbobenzoxyl-leucinyl-leucinyl-norvalinal (MG115), in a yeast mutant with enhanced permeability, but not in wild-type strains.  Lactacystin, an irreversible proteasome inhibitor, had no effect, but the β-lactone deriv. of lactacystin, which directly reacts with proteasomes, inhibited the degrdn. of short-lived proteins.  These inhibitors also blocked the rapid ubiquitin-dependent breakdown of a β-galactosidase fusion protein and caused accumulation of enzymically active mols. in cells.  The degrdn. of the bulk of cell proteins, which are long-lived mols., was not blocked by proteasome inhibitors, but could be blocked by phenylmethylsulfonyl fluoride.  This agent, which inhibits multiple vacuolar proteases, did not affect the proteasome or breakdown of short-lived proteins.  These two classes of inhibitors can thus be used to distinguish the cytosolic and vacuolar proteolytic pathways and to increase the cellular content of short-lived proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Sxabgj574rVg90H21EOLACvtfcHk0litgXsPXn2oZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvVWms78%253D&md5=95d1d47fcba959ddf297a94edc93f580</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.44.27280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.44.27280%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DSelective%2520Inhibitors%2520of%2520the%2520Proteasome-dependent%2520and%2520Vacuolar%2520Pathways%2520of%2520Protein%2520Degradation%2520inSaccharomyces%2520cerevisiae%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27280%26epage%3D27284%26doi%3D10.1074%2Fjbc.271.44.27280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0litgXsPXn2oZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F3I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F3I','PDB','6F3I'); return false;">PDB: 6F3I</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i47"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01125">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_07057"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01125?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01125</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Assay protocols and statistical analyses (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Raw data for the proteomics pull-down experiments (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_001.pdf">jm0c01125_si_001.pdf (640.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_002.xlsx">jm0c01125_si_002.xlsx (936.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01125/suppl_file/jm0c01125_si_003.csv">jm0c01125_si_003.csv (3.29 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01125&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01125%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01125" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998653bb7a2212","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
