
<html lang="en"     class="pb-page"  data-request-id="876f7aca-461a-4a5a-b513-bdf36edb2e69"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2002.45.issue-8;wgroup:string:ACHS website Group;article:article:10.1021/jm010398p;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Novel Approach for Linking Genotype to Phenotype in Vitro by Exploiting an Extremely Strong Interaction between RNA and Protein" /></meta><meta name="dc.Creator" content="Satoshi  Fujita,†,‡" /></meta><meta name="dc.Creator" content="Shinya Y.  Sawata,‡" /></meta><meta name="dc.Creator" content="Rika  Yamamoto-Fujita,†,‡" /></meta><meta name="dc.Creator" content="Yaeta  Endo,§" /></meta><meta name="dc.Creator" content="Hideo  Kise,‖" /></meta><meta name="dc.Creator" content="Masahiro  Iwakura,⊥ and" /></meta><meta name="dc.Creator" content="Kazunari  Taira*,†,‡" /></meta><meta name="dc.Description" content="We recently isolated an aptamer that binds to the Tat protein of HIV-1 with extremely high affinity and specificity (Yamamoto, R.; et al. Genes Cells 2000, 5, 371.). In the present study, we exploi..." /></meta><meta name="Description" content="We recently isolated an aptamer that binds to the Tat protein of HIV-1 with extremely high affinity and specificity (Yamamoto, R.; et al. Genes Cells 2000, 5, 371.). In the present study, we exploi..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="March 7, 2002" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm010398p" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-45-8-1598" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(01)00398-3" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2002 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm010398p" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm010398p" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm010398p" /></link>
        
    
    

<title>Novel Approach for Linking Genotype to Phenotype in Vitro by Exploiting an Extremely Strong Interaction between RNA and Protein | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm010398p" /></meta><meta property="og:title" content="Novel Approach for Linking Genotype to Phenotype in Vitro by Exploiting an Extremely Strong Interaction between RNA and Protein" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pn00001.jpeg" /></meta><meta property="og:description" content=" We recently isolated an aptamer that binds to the Tat protein of HIV-1 with extremely high affinity and specificity (Yamamoto, R.; et al. Genes Cells 2000, 5, 371.). In the present study, we exploited this strong binding to develop a novel coupling method that links genotype with phenotype. To strengthen the original RNA−protein interaction still further, we connected three units of the aptamer in tandem and three units of a peptide derived from Tat that interacted with the aptamer. The binding of the resultant RNA, which consisted of three units of the aptamer, to the resultant peptide, which consisted of three units of the peptide, was extremely strong. In fact, the RNA−protein interaction was one of the strongest ever reported, with an apparent Kd below 16 pM. This strong interaction was attempted for the selection of functional proteins, namely, dihydrofolate reductase (DHFR) or streptavidin, which we chose as an example, and we succeeded in the expected selection, although to a limited extent, of the target protein. The noncovalent but strong interaction described above should be useful as a novel tool for the future selection of functional proteins from pools of random sequences of amino acids.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm010398p"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm010398p">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm010398p&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm010398p&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm010398p&amp;href=/doi/10.1021/jm010398p" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2002</span><span class="cit-fg-volume">, 45</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 1598-1606</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/45/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm011039x" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm010469k" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Novel Approach for Linking Genotype to Phenotype in Vitro by Exploiting an Extremely Strong Interaction between RNA and Protein</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Satoshi++Fujita">Satoshi Fujita</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shinya+Y.++Sawata">Shinya Y. Sawata</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rika++Yamamoto-Fujita">Rika Yamamoto-Fujita</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yaeta++Endo">Yaeta Endo</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hideo++Kise">Hideo Kise</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masahiro++Iwakura">Masahiro Iwakura</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kazunari++Taira">Kazunari Taira</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm010398p&amp;href=/doi/10.1021%2Fjm010398p" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2002</span></span><span class="cit-volume">, 45</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 1598–1606</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 7, 2002</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 August 2001</li><li><span class="item_label"><b>Published</b> online</span>7 March 2002</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 April 2002</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm010398p" title="DOI URL">https://doi.org/10.1021/jm010398p</a></div><div class="article_header-article-copyright"><strong>Copyright © 2002 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1598%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSatoshi%2BFujita%252C%2BShinya%2BY.%2BSawata%252C%2BRika%2BYamamoto-Fujita%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D45%26issueNum%3D8%26contentID%3Djm010398p%26title%3DNovel%2BApproach%2Bfor%2BLinking%2BGenotype%2Bto%2BPhenotype%2Bin%2BVitro%2Bby%2BExploiting%2Ban%2BExtremely%2BStrong%2BInteraction%2Bbetween%2BRNA%2Band%2BProtein%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1606%26publicationDate%3DApril%2B2002">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm010398p"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">296</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm010398p" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Novel Approach for Linking Genotype to Phenotype in Vitro by Exploiting an Extremely Strong Interaction between RNA and Protein&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Satoshi&quot;,&quot;last_name&quot;:&quot;Fujita&quot;},{&quot;first_name&quot;:&quot;Shinya&quot;,&quot;last_name&quot;:&quot;Y. Sawata&quot;},{&quot;first_name&quot;:&quot;Rika&quot;,&quot;last_name&quot;:&quot;Yamamoto-Fujita&quot;},{&quot;first_name&quot;:&quot;Yaeta&quot;,&quot;last_name&quot;:&quot;Endo&quot;},{&quot;first_name&quot;:&quot;Hideo&quot;,&quot;last_name&quot;:&quot;Kise&quot;},{&quot;first_name&quot;:&quot;Masahiro&quot;,&quot;last_name&quot;:&quot;Iwakura&quot;},{&quot;first_name&quot;:&quot;Kazunari&quot;,&quot;last_name&quot;:&quot;Taira&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2002&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;45&quot;,&quot;pages&quot;:&quot;1598-1606&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm010398p&quot;},&quot;abstract&quot;:&quot;We recently isolated an aptamer that binds to the Tat protein of HIV-1 with extremely high affinity and specificity (Yamamoto, R.; et al. Genes Cells 2000, 5, 371.). In the present study, we exploited this strong binding to develop a novel coupling method that links genotype with phenotype. To strengthen the original RNA−protein interaction still further, we connected three units of the aptamer in tandem and three units of a peptide derived from Tat that interacted with the aptamer. The binding of the resultant RNA, which consisted of three units of the aptamer, to the resultant peptide, which consisted of three units of the peptide, was extremely strong. In fact, the RNA−protein interaction was one of the strongest ever reported, with an apparent Kd below 16 pM. This strong interaction was attempted for the selection of functional proteins, namely, dihydrofolate reductase (DHFR) or streptavidin, which we chose as an example, and we succeeded in the expected selection, although to a limited extent, of the tar&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm010398p&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm010398p" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm010398p&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm010398p" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm010398p&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm010398p" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm010398p&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm010398p&amp;href=/doi/10.1021/jm010398p" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm010398p" /></input><a href="/doi/pdf/10.1021/jm010398p" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (419 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm010398p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm010398p%26sid%3Dliteratum%253Aachs%26pmid%3D11931614%26genre%3Darticle%26aulast%3DFujita%26date%3D2002%26atitle%3DNovel%2BApproach%2Bfor%2BLinking%2BGenotype%2Bto%2BPhenotype%2Bin%2BVitro%2Bby%2BExploiting%2Ban%2BExtremely%2BStrong%2BInteraction%2Bbetween%2BRNA%2Band%2BProtein%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D45%26issue%3D8%26spage%3D1598%26epage%3D1606%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291266" title="Fundamental interactions">Fundamental interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=290975" title="Biopolymers">Biopolymers</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/45/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We recently isolated an aptamer that binds to the Tat protein of HIV-1 with extremely high affinity and specificity (Yamamoto, R.; et al. <i>Genes Cells </i><b>2000</b>, <i>5</i>, 371.). In the present study, we exploited this strong binding to develop a novel coupling method that links genotype with phenotype. To strengthen the original RNA−protein interaction still further, we connected three units of the aptamer in tandem and three units of a peptide derived from Tat that interacted with the aptamer. The binding of the resultant RNA, which consisted of three units of the aptamer, to the resultant peptide, which consisted of three units of the peptide, was extremely strong. In fact, the RNA−protein interaction was one of the strongest ever reported, with an apparent <i>K</i><sub>d</sub> below 16 pM. This strong interaction was attempted for the selection of functional proteins, namely, dihydrofolate reductase (DHFR) or streptavidin, which we chose as an example, and we succeeded in the expected selection, although to a limited extent, of the target protein. The noncovalent but strong interaction described above should be useful as a novel tool for the future selection of functional proteins from pools of random sequences of amino acids. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm010398pAF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  The University of Tokyo.</p></span></div><div class="fnGroupItem" id="jm010398pAF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  Gene Discovery and Research Center, National Institute of Advanced Industrial Science and Technology. </p></span></div><div class="fnGroupItem" id="jm010398pAF4"><span><h4><span class="NLM_fn-symbol">§</span></h4></span><span><p class="last">  Ehime University.</p></span></div><div class="fnGroupItem" id="jm010398pAF5"><span><h4><span class="NLM_fn-symbol">‖</span></h4></span><span><p class="last">  University of Tsukuba.</p></span></div><div class="fnGroupItem" id="jm010398pAF6"><span><h4><span class="NLM_fn-symbol">⊥</span></h4></span><span><p class="last">  Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology.</p></span></div><div class="fnGroupItem" id="jm010398pAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed. Address:  Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan. Phone:  81-3-5841-8828 or 81-298-61-3015. Fax:  81-298-61-3019. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="99edf8f0ebf8d9faf1fcf4fbf0f6b7edb7ecb4edf6f2e0f6b7f8fab7f3e9b7">[email protected]</a></p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Methods for the selection and identification of functional proteins from pools of random sequences of amino acids are of great current interest. Many successful screening systems for the selection of specific functional proteins, such as phage-display and two-hybrid systems, are based on living cells.<sup><a onclick="showRef(event, 'jm010398pb00001 jm010398pb00002 jm010398pb00003 jm010398pb00004 jm010398pb00005 jm010398pb00006'); return false;" href="javascript:void(0);" class="ref jm010398pb00001 jm010398pb00002 jm010398pb00003 jm010398pb00004 jm010398pb00005 jm010398pb00006">1-6</a></sup> In such selection systems, the sequence information (genotype) that encodes the selected protein (phenotype) can be determined from the DNA introduced into each cell that displays the appropriate phenotype. The coupling of genotype and phenotype is the single critical determinant in the selection of functional proteins from random pools. However, as long as such methods are dependent on living cells, the diversity of sequence libraries is limited because, for example, of the inefficiency of transfection processes, limits to the size of displayed proteins, and limits to the nature of the protein in question, since any protein that is detrimental to cells cannot be selected. </div><div class="NLM_p">To overcome these problems, several systems for selection for proteins in cell-free systems, including a ribosome-display system, mRNA-display (in vitro virus) system, and a micelle system, have recently been developed.<sup><a onclick="showRef(event, 'jm010398pb00007 jm010398pb00008 jm010398pb00009 jm010398pb00010 jm010398pb00011 jm010398pb00012 jm010398pb00013 jm010398pb00014 jm010398pb00015 jm010398pb00016 jm010398pb00017 jm010398pb00018 jm010398pb00019 jm010398pb00020 jm010398pb00021 jm010398pb00022 jm010398pb00023 jm010398pb00024 jm010398pb00025 jm010398pb00026 jm010398pb00027 jm010398pb00028'); return false;" href="javascript:void(0);" class="ref jm010398pb00007 jm010398pb00008 jm010398pb00009 jm010398pb00010 jm010398pb00011 jm010398pb00012 jm010398pb00013 jm010398pb00014 jm010398pb00015 jm010398pb00016 jm010398pb00017 jm010398pb00018 jm010398pb00019 jm010398pb00020 jm010398pb00021 jm010398pb00022 jm010398pb00023 jm010398pb00024 jm010398pb00025 jm010398pb00026 jm010398pb00027 jm010398pb00028">7-28</a></sup> Presently available cell-free systems, however, still include steps that reduce diversity, although some successful examples were recently reported.<sup>26</sup> In the ribosome-display system, the ribosome forms a stable complex with the translated protein and the mRNA that encodes the protein because the release of the protein from the complex is slowed by the removal of a termination codon and by the addition of Mg(OAc)<sub>2</sub> and anti-ssrA oligonucleotides. Under such conditions, the release factor cannot effectively induce the release of the protein from the ribosome complex. However, the yield of the isolated mRNAs after one round of ribosome display was not very high.<sup>10,28</sup> Thus, further stabilization of the mRNA−ribosome−protein complex will be required for wider applications. In mRNA-display (in vitro virus) systems, a translated protein binds covalently to the respective mRNA, which has been labeled with puromycin at the 3‘ end. In this system, chemical synthesis, for attachment of puromycin to mRNA, is always required, and this process might be expected to reduce the size of the available library. In the micelle system, each individual, chemically modified DNA molecule is packaged in one colloidal micelle and, upon transcription and translation, the modified DNA is linked to the encoded protein within the micelle compartment. However, the dilution of the reaction mixture that is designed to ensure that one DNA molecule is packaged per individual micelle might also be expected to reduce the diversity of the sequence library. Therefore, although cell-free systems might, in theory, have the advantage of allowing examination of <a name="jm004500801599" id="jm004500801599"></a> a larger number of sequences compared to systems that use living cells, a significant portion of a pool's diversity might not be fully exploited in the presently available cell-free systems. </div><div class="NLM_p last">In this study, we attempted to develop a simpler system for selection of proteins in vitro that would not include any steps that might potentially reduce the diversity of the pool of sequences. Our approach was based on the extremely strong interaction between an RNA and a protein. We recently selected an aptamer (designated Tat aptamer) that binds to the Tat (trans activation) protein of HIV-1 two orders of magnitude more strongly than the RNA that naturally interacts with Tat, which is known as TAR (trans-activating response region).<sup><a onclick="showRef(event, 'jm010398pb00029'); return false;" href="javascript:void(0);" class="ref jm010398pb00029">29</a></sup> The interaction between the Tat aptamer and a peptide derived from Tat (RE peptide, 38 amino acids; or CQ peptide, 36 amino acids) was extremely strong with a dissociation constant (<i>K</i><sub>d</sub>) in the subnanomolar range. We postulated that such a strong RNA−protein interaction might be useful for linking genotype and phenotype, namely, an mRNA and its encoded protein. Because the stabilization and maintenance of the RNA−protein complex are key determinants in the successful selection of a target protein, we tried to reduce the <i>K</i><sub>d</sub> of the complex still further by increasing the number of aptamer and peptide motifs.<sup><a onclick="showRef(event, 'jm010398pb00030 jm010398pb00031'); return false;" href="javascript:void(0);" class="ref jm010398pb00030 jm010398pb00031">30,31</a></sup> In our system, the RNA aptamer, linked to a specific mRNA, should bind automatically to the encoded protein, upon translation, that contains the RE or CQ peptide. Thus, with such a high-affinity aptamer, formation and long-term maintenance of the complex should be straightforward. As a result, mRNAs after one round of the selection will be collected with high yields. And most importantly, upon complete optimization, this kind of system, exploiting the extremely strong RNA−protein interaction, should make the best use of the diversity of a pool in a cell-free system. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Enhanced Interaction between Tandemly Connected Aptamers and Tat-Derived Peptides.</b> We chose, arbitrarily, dihydrofolate reductase (DHFR) as the model functional protein, since its properties have been well characterized.<sup><a onclick="showRef(event, 'jm010398pb00032 jm010398pb00033 jm010398pb00034 jm010398pb00035 jm010398pb00036 jm010398pb00037'); return false;" href="javascript:void(0);" class="ref jm010398pb00032 jm010398pb00033 jm010398pb00034 jm010398pb00035 jm010398pb00036 jm010398pb00037">32-37</a></sup> DHFR, which catalyzes reduction of dihydrofolate to tetrahydrofolate, plays an important role in supplying cofactors of one-carbon-transfer reactions catalyzed by thymidylate synthase. To examine the effects on binding affinity of several aptamers and Tat-derived peptides (RE), we prepared three kinds of protein [DHFR−(RE)<i><sub>n</sub></i>] (Figure 1<a data-tab="pane-pcw-Figures" href="#jm010398pf00001" id="" class=" internalNav"></a>) and three types of tandem RNA aptamer [(Apt)<i><sub>n</sub></i>] (Figure 2<a data-tab="pane-pcw-Figures" href="#jm010398pf00002" id="" class=" internalNav"></a>A), in which “<i>n</i>” ranged from 1 to 3 and corresponded to the number of copies of each motif. The bands of purified proteins after SDS−PAGE are shown in Figure 1, and we confirmed that each protein had the expected DHFR activity (data not shown), proving that attachment of RE peptide motifs had not interfered with the correct folding of the DHFR domain. To maintain the appropriate secondary structure of each aptamer, we linked the Tat aptamers to one another via six adenosine nucleotides in tandem to yield (Apt)<sub>2</sub> and (Apt)<sub>3</sub>. The purified RNAs after PAGE on a nondenaturing gel are shown in the control lanes (C) in Figure 2A. The gel in Figure 2A also demonstrates that, as we had expected, the Tat aptamers did not interact with DHFR. The gel shows that the mobility of the aptamers did not change after mixing with increasing concentrations of <a name="jm004500801600" id="jm004500801600"></a> wild-type DHFR (the concentration of DHFR was increased from lane C toward the right in each case, as indicated by the triangles at the top of the gel). <div id="_i5"><figure id="jm010398pf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Gel after SDS−PAGE showing purified DHFR−(RE)<sub>1</sub><sub>-</sub><sub>3</sub> fusion proteins and wild-type (wt) DHFR. Purified recombinant proteins (∼100 μg) were subjected to SDS−PAGE and visualized by straining with Coomassie blue. Lane 1, markers; lane 2, wild-type DHFR; lane 3, DHFR−(RE)<sub>1</sub>; lane 4, DHFR−(RE)<sub>2</sub>; lane 5, DHFR−(RE)<sub>3</sub>. Molecular masses of marker proteins are shown on the left. Polyglycine linkers were inserted between DHFR and the first RE peptide and also between the RE and the adjacent RE peptides to maintain flexibility. To facilitate purification, polyhistidines (His) were connected at the N terminal of DHFR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm010398pf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00002.jpeg" id="_i7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Measurements of dissociation constants (<i>K</i><sub>d</sub>). In each lane C, only the aptamer was loaded. The black arrowheads indicate protein−RNA complexes, and white arrowheads indicate free RNA. Relative amounts of complex and free RNA were calculated from the radioactivity of <sup>32</sup>P. Equilibrium dissociation constants were determined from theoretical curves of the concentration of free RNA or complex RNA versus the concentration of the peptide by the least-squares methods. Diamonds indicate relative amounts of free RNA, and squares indicate relative amounts of complex. In each panel, the elongated black triangle indicates increasing concentrations of protein. (A) Absence of any interactions between (Apt)<sub>1</sub><sub>-</sub><sub>3</sub> and wild-type DHFR. Each lane C was loaded with 1 nM (Apt)<sub>1</sub><sub>-</sub><sub>3</sub> and 20, 100, and 500 nM wild-type DHFR were added as indicated by the elongated triangles. (B) Binding of (Apt)<sub>1</sub> with DHFR−(RE)<sub>1</sub>. A total of 30 pM (Apt)<sub>1</sub> (lane C) was combined with 0.25, 0.5, 1, 2, 4, 8, 16, and 32 nM DHFR−(RE)<sub>1</sub> as indicated by the elongated triangles. (C, D) Binding of (Apt)<sub>2</sub><sub>-</sub><sub>3</sub> with DHFR−(RE)<sub>2</sub><sub>-</sub><sub>3</sub> proteins. A total of 30 pM (Apt)<sub>2</sub> or (Apt)<sub>3</sub> RNA (lane C) was combined with 0.03, 0.06, 0.12, 0.25, 0.5, 1, 2, 4, 8, and 16 nM protein as indicated by the elongated triangles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">As we anticipated, the mobility on gels of aptamers (Apt)<sub>1</sub><sub>-</sub><sub>3</sub> changed after they had been mixed with the fusion proteins DHFR−(RE)<sub>1</sub><sub>-</sub><sub>3</sub> (Figure 2B−D). In these experiments, we mixed <sup>32</sup>P-labeled aptamers (final concentration, 30 pM) with an excess of each fusion protein, and the amount of complex formed was quantified with an imaging analyzer. Apparent values of <i>K</i><sub>d</sub> were calculated to be 24 nM, 200 pM, and less than 16 pM for <i>n</i> = 1, <i>n</i> = 2, and <i>n</i> = 3, respectively. Clearly, among the combinations that we examined, the interaction between (Apt)<sub>3</sub> and DHFR−(RE)<sub>3</sub> was the strongest [<i>K</i><sub>d(app)</sub> ≤ 16 pM]. To the best of our knowledge, this is one of the strongest interactions between an RNA and a protein reported to date. Although this complex appeared to adopt multiple conformations, stronger interactions were clearly achieved with an increase in the number of both the aptamer and the RE peptide. We used the (Apt)<sub>3</sub> and DHFR−(RE)<sub>3</sub> motifs in subsequent experiments because the maintenance of the stability of the RNA−protein complex, in other words, an extremely strong interaction between the RNA and the protein, is the key factor in the coupling of genotype and phenotype. </div><div class="NLM_p"><b>Linkage of Genotype and Phenotype by the Interaction between (Apt)<sub>3</sub> and (RE)<sub>3</sub>.</b> To confirm the potential utility of (Apt)<sub>3</sub> and DHFR−(RE)<sub>3</sub>, we prepared two types of plasmid. The plasmids encoded a wild-type and a mutant DHFR−(RE)<sub>3</sub> fusion protein, and the encoded mRNAs and translated proteins are shown schematically in Figure 3<a data-tab="pane-pcw-Figures" href="#jm010398pf00003" id="" class=" internalNav"></a>A [intact, pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub>; mutant, pMuDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub>]. Each plasmid included the sequence encoding DHFR with three RE peptides [(RE)<sub>3</sub>] in tandem, as well as the sequence of three Tat aptamers (Apt)<sub>3</sub> in tandem. The mutant protein [MuDHFR−(RE)<sub>3</sub>] derived from pMuDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> differed from the intact protein [DHFR−(RE)<sub>3</sub>] derived from pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> at only two positions, with the amino acids substitutions Asp 27 → Ala 27 and Phe 31 → Ala 31 at the active site of DHFR. These mutations abolished the ability to bind methotrexate (MTX).<sup>32-37</sup> As a second control, we also prepared a plasmid that encoded the intact DHFR−(RE)<sub>3</sub> fusion protein but no aptamer sequences, namely, pDHFR−(RE)<sub>3</sub>, so that the translated product, DHFR−(RE)<sub>3</sub>, would be unable to interact with its mRNA (Figure 3A, right). <div id="_i8"><figure id="jm010398pf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00003.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Model of the selection of a functional protein in vitro using interaction between (Apt)<sub>3</sub> and (RE)<sub>3</sub>. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of SDS−PAGE showing the enrichment for DHFR−(RE)<sub>3</sub> protein and mRNA (intact) on MTX beads. Each sample was then subjected to SDS−PAGE on a 15% gel. Lanes 1−3, 1 μL of lysate was loaded after the respective translation reactions. Lanes 4−6, 10 μL of eluate containing mRNA and protein that had been bound to MTX beads. Lanes 1 and 4 correspond to the intact protein, lanes 2 and 5 correspond to the mutant protein, and lanes 3 and 6 were loaded with control samples. (C) Quantification of proteins and mRNAs shown in (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">If the (Apt)<sub>3</sub>−(RE)<sub>3</sub> interaction could really be used to couple genotype and phenotype, only the intact system (Figure 3A, left) should yield both the protein (DHFR) and its mRNA that would bind to MTX−agarose beads when the reaction mixture, after translation, is mixed with MTX−agarose beads. By contrast, the mutant system (Figure 3A, center) should yield neither the protein (DHFR) nor its mRNA for binding to MTX−agarose beads, since the mutant DHFR was designed to be unable to bind to MTX. Finally, the control system (Figure 3A, right) should yield only the protein (DHFR) and not the mRNA for binding to MTX−agarose beads, since DHFR−(RE)<sub>3</sub> would be unable to interact with its mRNA because the letter lacks the (Apt)<sub>3</sub> aptamers. </div><div class="NLM_p">To confirm our predictions, we prepared three kinds of <sup>32</sup>P-labeled capped-mRNA, using T7 polymerase, from the respective plasmids and used them as templates to translate the respective proteins in a rabbit reticulocyte lysate. During translation, each protein was labeled with <sup>35</sup>S-methionine. After translation, the appropriate binding buffer and MTX−agarose beads were added to each lysate, and binding reactions were allowed to proceed for 30 min at 4 °C (this low temperature reduced the rate of degradation of mRNA by RNases in the lysate). Supernatants of the binding solutions containing unbound mRNA and protein were removed, and the MTX−agarose beads were washed twice with the binding buffer. The products eluted from beads in an elution buffer at an elevated temperature (95 °C) were analyzed by SDS−PAGE. We confirmed, by checking each lysate after translation, that amounts of translated proteins and input mRNAs were identical in each case (Figure 3B, lanes 1−3). </div><div class="NLM_p">Analysis of the eluted products demonstrated that, as we expected, DHFR was detectable in the intact and control systems (Figure 3B, lanes 4 and 6) while <a name="jm004500801601" id="jm004500801601"></a> themutant protein was barely detectable (lane 5). The yields of the intact DHFR and the control DHFR, respectively, were 9- and 16-fold higher than that of the mutant DHFR (Figure 3C, left). Moreover, the mRNA was detected only in the intact system (Figure 3B, lane 4), even though nonspecific binding of mRNA to the MTX beads could not be completely avoided (lanes 5 and 6) in this experiment, since the MTX beads were somewhat positively charged after attachment of MTX. The yield of the intact mRNA was only 2.5-fold higher than that of the background, i.e., nonspecifically bound mutant and control mRNAs (Figure 3C, right). We are at present testing other attachment procedures to prevent formation of beads with positive charges. Nevertheless, quantification of bands demonstrated clearly that the amounts of the proteins and mRNA were well above the background level, and these results confirmed the validity of the interactions that are shown schematically in Figure 3A. </div><div class="NLM_p"><b>Attempts To Select mRNA That Is Attached to Its Phenotype (Streptavidin) in Vitro. </b>If (Apt)<sub>3</sub> and DHFR−(RE)<sub>3</sub> are to function properly in the selection of the target protein, the respective protein should end up bound specifically to its mRNA when two distinguishable mRNAs are translated together. Since the presently available MTX−agarose beads were positively charged and nonspecific binding of irrelevant mRNAs could not be avoided completely (parts B and C of Figure 3), we tested alternative biotin−agarose beads. For this purpose, as a positive control, we made a new plasmid, pStreptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub>, in that the gene for DHFR within the pDHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> plasmid had been replaced by the gene for streptavidin (recall that both RE and CQ are Tat-derived peptides with identical functions).<sup>29</sup> In this system (Figure 4<a data-tab="pane-pcw-Figures" href="#jm010398pf00004" id="" class=" internalNav"></a>), the translated streptavidin but not the DHFR should end up bound to the biotin−agarose beads when we start from a mixture of mRNAs derived from pStreptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> and pDHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub>.<sup><a onclick="showRef(event, 'jm010398pb00038'); return false;" href="javascript:void(0);" class="ref jm010398pb00038">38</a></sup><div id="_i10"><figure id="jm010398pf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00004.jpeg" id="_i11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Selection of mRNA-encoding streptavidin in vitro from a pool of mixtures of streptavidin and DHFR mRNAs. (A) Schematic representation of interactions among the transcribed RNA, its translated protein, and biotin-conjugated agarose beads (biotin ligand). (B) Enrichment for streptavidin mRNA by binding to biotin−agarose, compared to DHFR mRNA. Only the streptavidin-containing complex was bound specifically to biotin−agarose. (C) Analysis by 3% agarose gel electrophoresis of the results of amplification by RT-PCR. PCR reactions were performed for 15 cycles. Each sample was subjected to 3% Sea Kem GTG agarose gel (FMC, Riceland, ME). Markers, size markers; lanes 1−3, products of mRNAs before selection (without selection) as controls; lanes 4−6, products of mRNAs selected with biotin−agarose. Lanes 1 and 4, streptavidin mRNA; lanes 2 and 5, DHFR mRNA; lanes 3 and 6, a mixture of streptavidin and DHFR mRNAs at a ratio of 1:1 (mole/mole).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">We prepared two kinds of <sup>32</sup>P-labeled capped-mRNA from pStreptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> and pDHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub>, using T7 polymerase. The mRNAs encoding streptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> and DHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> were translated in lysate as a mixture in the ratio of 1:1 (green, Figure 4B) and also were translated individually (red and blue, Figure 4B) in order to confirm that the mRNAs were bound to the biotin−agarose through a complex between the Tat aptamer and CQ peptide. Translation using <sup>32</sup>P-labeled mRNAs was examined to determine the amount of bound mRNA at each step during the selection procedure. After translation for 15 min, the appropriate binding buffer and biotin−agarose were added to lysates and binding reactions were allowed to proceed for 10 min at 4 °C. Biotin−agarose had been blocked, before combination with a translation mixture, by exposure to sterilized milk (Block Ace; Dainippon Pharmaceutical Co., Japan), and each binding and washing step was performed in the presence of the milk, since milk decreases the nonspecific binding of ribosome complexes to agarose.<sup>14</sup> After centrifugation, supernatants that contained unbound mRNA and protein were discarded and respective pelleted ligands were washed twice with binding buffer. At each step during the selection procedure, the relative amount of mRNA that remained in the mixture was determined by monitoring radioactivity (Figure 4B). </div><div class="NLM_p">Results shown in Figure 4B are the averages from four independent sets of experiments. DHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> mRNA (dark-blue line in Figure 4B) was soon washed away, while streptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> mRNA (red line in Figure 4B) was firmly retained with more than 1.5% of the input mRNA remaining after a second wash (Figure 4B). And streptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> mRNA was enriched 17-fold higher than DHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> mRNA (Figure 4B). It is important to note that the amount of recovered mRNA, which remained at 1.5% after repeated washes, was significantly higher than the corresponding values reported previously. In ribosome display, the amount of recovered mRNA was as little <a name="jm004500801602" id="jm004500801602"></a> as 0.015% (or 0.2% under certain conditions) of the input RNA, although some mRNA was recovered at a higher yield (8%) because the selection did not require complete folding of a short peptide such as His-tag<sup>28</sup>. </div><div class="NLM_p">We then eluted bound products from ligands (biotin−agarose) with elution buffer containing 1 M NaCl. The eluted mRNAs were purified with an RNeasy minikit (QIAGEN, Germany) and amplified by reverse transcriptase polymerase chain reaction (RT-PCR) in order to confirm the enrichment of the mRNA of interest (Figure 4C). We confirmed previously by PCR that each sample of mRNA did not contain any contaminating DNA (data not shown). Furthermore, we confirmed that the amount of each input mRNA was constant (Figure 4C, lanes 1−3). Our analyses by RT-PCR clearly demonstrated that mRNA-encoding streptavidin was successfully recovered by biotin−agarose (Figure 4C, lane 4), whereas the nonspecific mRNA-encoding DHFR was not detected (Figure 4C, lane 5). Clearly, the nonspecific binding of mRNAs to the biotin−agarose is very low and mRNA-encoding streptavidin binds to biotin−agarose correctly and efficiently. </div><div class="NLM_p">When we mixed the mRNAs that encoded streptavidin and DHFR together at a ratio of 1:1 (mole/mole), mRNA-encoding streptavidin was correctly concentrated from the mixture via binding to biotin−agarose (Figure 4C, lane 6), although a small amount of unwanted DHFR mRNA was detected (Figure 4, lane 6). These data demonstrate that the interactions are primarily cis and not trans, although a small amount of RNA−protein networks could not be avoided (see below). In support of this interpretation, when the mixed mRNAs (1:1 mole/mole ratio) were subjected to biotin−agarose, only the expected yield of mRNA was obtained (green line in Figure 4B). In this case, the amounts of the input mRNAs were kept constant, and thus, when the mixture was subjected to biotin−agarose (green line), the absolute amount of streptavidin mRNA or DHFR mRNA was half of that of the individual experiment (red line or blue line). If trans interactions had been the dominant interactions, close to 1.5% recovery (similar to that of the red line) would have been achieved from the 1:1 mixture of mRNAs (specifically 0.7% streptavidin mRNA plus 0.7% in trans-acted nonspecific DHFR mRNA, leading to total ∼1.5% recovered mRNAs) rather than the recovery of half the amount (0.7%, green line). It should be noted that unwanted DHFR mRNAs containing the tandem aptamer sequence can bind only trans to the biotin−agarose bound streptavidin that contains the CQ peptide region. </div><div class="NLM_p">Importantly, the present results with a partial success suggest that the strong RNA−protein interaction might be useful for the selection of functional proteins if the mRNA−protein interaction in a trans configuration could be further minimized by the adjustment of the length of linkers and proper combination of the respective aptamers and peptides (see below). </div><div class="NLM_p"><b>Combination of the Strong Interaction between (RE)<sub>3</sub> and (Apt)<sub>3</sub> with the Ribosome-Display System in the Selection of Functional Proteins. </b>Since we could partially select the functional streptavidin by the use of the strong RNA−protein interaction (Figure 4C, lane 6), we wondered whether the situation might be improved by the combination with the ribosome-display system. In the ribosome-display system, the ribosome forms a relatively stable complex with the translated protein and its respective mRNA because the release of the protein from the complex is slowed by the removal of a termination codon and by the addition of Mg(OAc)<sub>2</sub> and anti-ssrA oligonucleotides.<sup>10,11</sup> To test the effects of removal of the termination codon in our system, we newly prepared a DHFR mRNA that lacked the termination codon (DHFR − Stop mRNA, right panel in Figure 5<a data-tab="pane-pcw-Figures" href="#jm010398pf00005" id="" class=" internalNav"></a>A). As a control, we used a DHFR mRNA with the termination codon (DHFR + Stop mRNA, central panel). <div id="_i12"><figure id="jm010398pf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00005.jpeg" id="_i13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Comparison of the strong interaction between (RE)<sub>3</sub>−(Apt)<sub>3</sub> and the ribosome-display system in the selection of functional DHFR in rabbit reticulocyte lysate. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of electrophoresis of RT-PCR products in 2% agarose showing the enrichment for DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop, DHFR − Stop, and DHFR + Stop mRNA on MTX beads. PCR reactions were performed for 20 cycles. Each sample was subjected to 2% Sea Kem GTG agarose gel (FMC). Analysis of the PCR products was carried out once, and thus, quantification at the level of mRNAs shown in Figure 5C is more reliable. (C) Quantification of enriched mRNAs. The relative amounts of selected mRNAs were detected by the count of <sup>32</sup>P-labeled mRNA. Respective numbers of lanes correspond to that of lanes in Figure 5B. The results were the averages from four independent sets of experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">We compared the amounts of mRNA bound to MTX beads, upon translation in lysate and after the selection procedure, for three types of mRNA, namely, DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop [this is same as the DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> used above], DHFR + Stop, and DHFR − Stop. <a name="jm004500801603" id="jm004500801603"></a> During the translation, the DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop mRNA with a termination codon binds to the translated DHFR protein by the interaction between (RE)<sub>3</sub> and (Apt)<sub>3</sub> and the ribosome does not take part in this protein−mRNA complex because the ribosome is released from mRNA by the presence of the termination codon. These three kinds of mRNAs were translated in a rabbit reticulocyte lysate and were subjected to MTX−agarose beads under the same conditions described in Figure 3. The relative amounts of selected mRNAs were detected by the count of <sup>32</sup>P-labeled RT-PCR products and the originally bound mRNAs (parts B and C of Figure 5). Results shown in Figure 5C are the averages from four independent sets of experiments (in one case, analysis of the PCR products was carried out and the results are shown in Figure 5B). On average, the selected DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop mRNA (lane 1, Figure 5C) was 2.5-fold higher in concentration than the background level of DHFR + Stop mRNA (lane 2, Figure 5C), whereas the corresponding value for the DHFR − Stop mRNA was 1.4-fold in the absence of Mg(OAc)<sub>2</sub> and anti-ssrA oligonucleotides (lane 3, Figure 5C). Therefore, in the present system (not optimized for the ribosome display), the (RE)<sub>3</sub>−(Apt)<sub>3</sub> interaction was slightly more effective than the removal of the termination codon in the stabilization of the complex between the respective mRNA and its product. </div><div class="NLM_p">The ribosome-display system worked extremely well when functional antibodies were selected against their respective antigens and the ribosome−mRNA−protein ternary complex was stable for 10 days in an appropriate binding buffer.<sup>20,24</sup> Our results (Figure 5C) indicate, however, that the ribosome−mRNA−protein complex that had formed by the removal of the termination codon in lysate might not be stable enough, at least under the conditions we used for the general translation purpose (which might not be optimized for the ribosome display). </div><div class="NLM_p">Therefore, it should be advantageous to combine the (RE)<sub>3</sub>−(Apt)<sub>3</sub> interaction and the ribosome-display system for the improved stabilization of the complex between the respective mRNA and its product. For this purpose, we prepared another plasmid in that the (Apt)<sub>3</sub>−(RE)<sub>3</sub> region was placed upstream of the gene for DHFR (Figure 6<a data-tab="pane-pcw-Figures" href="#jm010398pf00006" id="" class=" internalNav"></a>). Furthermore, to examine the effects of releasing factors in the stabilization of the complex by the removal of the termination codon, we prepared polyclonal IgG antibodies against releasing factors (anti-RFs) <i>E. coli</i> RF1, RF2, and RF3.<sup><a onclick="showRef(event, 'jm010398pb00039'); return false;" href="javascript:void(0);" class="ref jm010398pb00039">39</a></sup> Since we placed the (Apt)<sub>3</sub>−(RE)<sub>3</sub> region at the 5‘ end in order to avoid appearance of termination codons within this region at the 3‘ end, we carried out, in the present studies shown in Figure 6, the translation of the respective mRNAs in <i>E. coli</i> S30 extracts by placing the SD sequence between (Apt)<sub>3</sub> and (RE)<sub>3</sub>. <div id="_i14"><figure id="jm010398pf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00006.jpeg" id="_i15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Improvement in stabilization of the complex between the respective mRNA and its product by combination of the (RE)<sub>3</sub>−(Apt)<sub>3</sub> interaction and the ribosome-display system in <i>E. coli</i> S30 extracts. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of electrophoresis of RT-PCR products in 2% agarose gel showing the PCR products for DHFR − Stop mRNA (parts 1 and 2) and (Apt)<sub>3</sub>−(RE)<sub>3</sub>−DHFR − Stop mRNA (parts 3 and 4) in the presence (parts 1 and 3) or in the absence (parts 2 and 4) of the polyclonal IgG antibodies against the releasing factors (<i>E. coli</i> RF1, RF2, and RF3), enriched on the MTX−agarose beads. Selected mRNAs were subjected to MTX−agarose beads under the same conditions as described in Figure 5. (C) Quantification of the RT-PCR products shown in (B). The results were the averages from two independent sets of experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">We compared the amounts of mRNA bound to MTX beads for two types of mRNA, namely, DHFR − Stop mRNA (Figure 6, parts 1 and 2) and (Apt)<sub>3</sub>−(RE)<sub>3</sub>−DHFR − Stop mRNA (Figure 6, parts 3 and 4), in the presence (parts 1 and 3) or in the absence (parts 2 and 4) of the polyclonal IgG antibodies against the releasing factors (anti-RFs). Translated mRNAs were subjected to MTX−agarose-bead selection under the same conditions described in Figure 3 (not optimized for the ribosome display). The relative amounts of selected mRNAs were detected by RT-PCR (parts B and C of Figure 6). Results shown in Figure 6C are the averages from two independent sets of experiments. These results clearly demonstrate that the combination of the (RE)<sub>3</sub>−(Apt)<sub>3</sub> interaction and the ribosome-display system, especially (unexpectedly, since the termination codon had been removed from the respective mRNA) in the presence of the polyclonal IgG antibodies against the releasing factors (although the exact reasons need to be elucidated), is advantageous in stabilizing the mRNA−ribosome complex. </div><div class="NLM_p"><b>Possible Existence of RNA−Protein Networks. </b>Experimental results described above clearly demonstrated that the strong aptamer−peptide interaction could further stabilize the ribosome-display complex, and thus, this kind of interaction should be useful for the selection of functional proteins. However, the efficiency of selection of correct proteins at this moment is not as high as we anticipated from the expectations of in vitro binding assays (Figure 2). Assuming a <a name="jm004500801604" id="jm004500801604"></a> diffusion-limited association constant of 10<sup>7</sup> M<sup>-1</sup> s<sup>-1</sup>, the dissociation rate constant of the best aptamer−peptide complex will be 1.6 × 10<sup>4</sup> s<sup>-1</sup>. This corresponds to a half-life of 52 min, which should be sufficient for the selection of the majority of functional proteins. Although we carried out the selection of the functional DHFR or streptavidin within 1 h, we had limited success. </div><div class="NLM_p last">The discrepancy between the in vitro binding constants and the efficiency of the selection could possibly be reconciled by the formation of RNA−protein networks, as schematically shown in the top diagram of Figure 7<a data-tab="pane-pcw-Figures" href="#jm010398pf00007" id="" class=" internalNav"></a>A. It is possible that the “conformations” exhibiting slower gel mobility, at least in part (Figure 2), are actually RNA−protein networks. Such networks would complicate the selection of proteins from libraries. Therefore, we are at present trying to prepare other combinations of aptamers and peptides, as schematically shown in the bottom diagram of Figure 7A (in addition to the improvement of the beads). Then, we should be able to optimize the system by the selection of the best linker sequence for the optimal interaction between the aptamers and respective peptides. <div id="_i16"><figure id="jm010398pf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00007.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 (A) Possible RNA−protein networks. Potentially, tandemly connected three RE peptides could interact with tandemly connected three aptamers not only in cis but also in trans configurations. Trans interactions might create RNA−protein networks. The trans interactions should be minimized by the combination of different units of peptides and aptamers. (B) A schematic diagram of the proposed method for selection of a functional protein in vitro. Binding proteins are generated and concentrated by multiple cycles of transcription, translation, affinity selection, and PCR. In the translation step, the tandem aptamer (Apt)<sub>3</sub> binds to the tandem RE peptide (RE)<sub>3</sub> with very high affinity, as demonstrated in the present study. The binding couples the protein and its mRNA. In this study, we used DHFR and streptavidin as the target in the selection step, but any protein can be used in combination with the tandem aptamer (Apt)<sub>3</sub> and the tandem RE peptide (RE)<sub>3</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In our present system, we arbitrarily chose linkers for the Tat aptamers and Tat derived peptides (RE and CQ) without any attempt at optimization. Nevertheless, the possibility that a strong interaction between RNA and protein can be used to couple genotype and phenotype was clearly demonstrated. Exploiting such unnatural combinations, namely, interactions that do not exist in nature, we should be able to select or engineer even stronger RNA−protein interactions. Then complexes of protein and mRNA with even longer half-lives can be generated and selection should become very easy. Moreover, our approach can be combined with the ribosome-display system by simply eliminating the termination codon from the fusion gene. The result should be a significantly more stable protein−mRNA complex as a consequence of attachment at two sites, as shown in part 3 of Figure 6. Once our approach has been optimized, there should be no steps at which a decrease might occur in the diversity of a sequence library. Most importantly, our proposed selection procedure is very simple and straightforward. Therefore, the present noncovalent but strong interaction should be considered as a novel tool for selection of functional proteins (Figure 7B), in addition to already available systems.<sup>1-28,40</sup><a onclick="showRef(event, 'jm010398pb00040'); return false;" href="javascript:void(0);" class="ref jm010398pb00040"></a></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63476" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63476" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Preparation of Three Kinds of Protein [DHFR−(RE)</b><b><i><sub>n</sub></i></b><b>].</b> For preparation of DHFR−(RE)<sub>1</sub><sub>-</sub><sub>3</sub> fusion protein, the gene for DHFR with a Pst I site at its 3‘ end, derived from plasmid pTZDHFR20 (ref 36), was inserted into plasmid pET30a (Novagen, Madison, WI). Sense and antisense oligonucleotides encoding the RE peptide (38 amino acids; RKKRR QRRRP PQGSQ TBQVS LSKQP TSQSR GDPTG PKE) were synthesized, annealed, and polymerized by rTaq polymerase (Takara Shuzo Co., Kyoto, Japan). The resultant dsDNAs were cut at the Pst I and EcoT22I sites and inserted on the 3‘ side of the gene for DHFR at the Pst I site. Because the Pst I and Eco T22I restriction sites can be connected and the resulting site can no longer be cleaved by either Pst I or Eco T22I, we were able to connect the sequence that encoded the RE peptide to the Pst I site on the 3‘ side of the gene for DHFR to obtain continuously pDHFR−(RE)<sub>1</sub> through pDHFR−(RE)<sub>3</sub>. These plasmids were used to transfect <i>E. coli</i> NovaBlue (DE3) (Novagen), and cells were incubated in 500 mL of LB medium at 30 °C. Then 1 mM of IPTG (isopropyl-1-thio-β-<span class="smallcaps smallerCapital">d</span>-galactoside) was added at the logarithmic phase of growth, and incubation was continued at 30 °C for 4 h. The proteins were expressed as polyhistidine-tagged polypeptides. Proteins were purified according to the protocol supplied with the HisTrap chelating kit (Amersham Pharmacia Biotech, Uppsala, Sweden), and small molecules were excluded using a HiTrap desalting system (Amersham Pharmacia Biotech). Furthermore, to remove degraded proteins, we performed cation-exchange HPLC using HiTrap SP (Amersham Pharmacia Biotech). Purity of proteins was checked by SDS−PAGE (Figure 1). Concentrations of purified proteins were measured with a BCA assay kit (Pierce, Rockford, IL). </div><div class="NLM_p"><a name="jm004500801605" id="jm004500801605"></a><b>Preparation of Three Types of Tandem RNA Aptamer [(Apt)</b><b><i><sub>n</sub></i></b><b>].</b> For preparation of tandem Tat aptamers [(Apt)<sub>1</sub><sub>-</sub><sub>3</sub>], sense and antisense oligonucleotides (5‘-AAA AAA CGA AGC TTG ATC CCG TTT GCC GGT CGA TCG CTT CG-3‘ and 5‘-TTT TTT CGA AGC GAT CGA CCG GCA AAC GGG ATC AAG CTT C-3‘) were synthesized. These oligonucleotides were annealed and ligated to yield tandemly connected Tat aptamers [(Apt)<sub>1</sub><sub>-</sub><sub>3</sub>] with a DNA ligation kit version 2 (Takara Shuzo Co.). Ligated DNA sequences coding one to three Tat aptamers were isolated after cloning (TA cloning kit, Invitrogen, Carlsbad, CA). The DNAs coding from one to three aptamers were transcribed with a T7 Ampliscribe kit (Epicentre Technologies, Madison, WI). </div><div class="NLM_p"><b>Measurements of Dissociation Constants (</b><b><i>K</i></b><b><sub>d</sub>). </b>For gel mobility shift assays, each transcribed aptamer [(Apt)<sub>1</sub><sub>-</sub><sub>3</sub>] was isolated by PAGE on a 15% polyacrylamide denaturing gel and dephosphorylated at its 5‘ end by alkaline phosphatase (<i>E. coli</i> A19; Takara Shuzo Co.) at 37 °C for 1 h. Then aptamers were labeled with <sup>32</sup>P by T4 polynucleotide kinase at 37 °C for 1 h and purified by PAGE (15% polyacrylamide). A fixed concentration of 5‘ end-labeled aptamer plus 40 nM tRNA was denatured in Tat binding buffer [10 mM Tris-HCl (pH 8.0), 70 mM NaCl, 2 mM EDTA and 0.1% Nonidet P-40] at 95 °C for 3 min, kept at room temperature for renaturation, and then mixed with a range of excess concentrations of DHFR−(RE)<sub>1</sub><sub>-</sub><sub>3</sub> in Tat binding buffer in a total volume of 10 μL at 30 °C for more than 1 h. Free and protein-bound or peptide-bound RNAs were separated by electrophoresis on a 5−15% nondenaturing polyacrylamide gel in 0.5 × TBE (Tris-borate/EDTA) buffer and were analyzed. </div><div class="NLM_p"><b>Construction of pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (Intact), pMuDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (Mutant), and pDHFR−(RE)<sub>3</sub> (Control). </b>For site-directed mutagenesis of the core region of DHFR, we prepared four oligonucleotides (5‘-ACA ATT CCC CTC TAG AAA TA-3‘, 5‘-GTG GTG GTG CTC GAG AAT TC-3‘, 5‘-CCT GCC GCC CTC GCC TGG GCC AAA CGC AAC ACC TTA AAT AAA-3‘, and 5‘-GCG TTT GGC CCA GGC GAG GGC GGC AGG CAG GTT CCA CGG-3‘). pMuDHFR−(RE)<sub>3</sub> was constructed by the introduction of two mutations into the gene for DHFR using these oligonucleotides and the megaprimer PCR mutagenesis strategy. Next, three aptamer sequences in tandem (Apt)<sub>3</sub> were introduced into the Eco RI site downstream of the RE peptides in pDHFR−(RE)<sub>3</sub> and pMuDHFR−(RE)<sub>3</sub> to yield pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (intact) and pMuDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (mutant). </div><div class="NLM_p"><b>Linkage of Genotype and Phenotype by the Interaction between (Apt)<sub>3</sub> and (RE)<sub>3</sub>.</b> Linearized plasmids [pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (intact), pMuDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> (mutant), and pDHFR−(RE)<sub>3</sub> (control)] were transcribed to yield capped mRNAs that were labeled with <sup>32</sup>P (Cap structure analogue; New England Bio Labs, Beverly, MA) in vitro at 37 °C for 2 h using a T7 Ampliscribe kit (Epicenter Technologies). Labeled mRNAs were purified on NICK columns (Amersham Pharmacia Biotech). One microgram of capped-mRNA-encoding DHFR−(RE)<sub>3</sub> (intact) or MuDHFR−(RE)<sub>3</sub> (mutant) was translated in 20 μL of rabbit reticulocyte lysate for 20 min at 30 °C (rabbit reticulocyte lysate system; Promega, Madison, WI). As a control, mRNA without aptamers was also translated (control). Each protein was labeled with <sup>35</sup>S-methionine. After translation, 180 μL of binding buffer [10 mM sodium phosphate buffer (pH 7.5), 70 mM NaCl, 10 μM tRNA] and 20 μL of a suspension of MTX−agarose beads (Sigma, St. Louis, MO), which had been treated with succinic anhydride for neutralization of positive charges on MTX−agarose beads, were added to the lysate and binding was allowed to proceed for 30 min at 4 °C. A total of 200 μL of supernatant containing unbound mRNA and protein was removed from the mixture, and MTX beads were washed twice with 180 μL of binding buffer. Finally, bound mRNA and protein were isolated from the MTX beads by elution at an elevated temperature (95 °C) with SDS-containing solution. Each sample was then subjected to SDS−PAGE on a 15% gel. </div><div class="NLM_p"><b>Selection of mRNA Encoding Streptavidin in Vitro from a Pool of Mixtures of Streptavidin and DHFR mRNAs.</b> The construction of pDHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> was nearly identical with that of pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub>. In short, sense and antisense oligonucleotides encoding the CQ peptide (36 amino acids; CFTTK ALGIS YGRKK RRQRR RPPQG SQTBQ VSLSK Q) were synthesized and the resultant fragment containing the CQ peptide was introduced between the sequences of DHFR and aptamer. A template DNA encoding streptavidin was a kind gift from Professor Masahiko Sisido and Mr. Takahiro Hohsaka of Okayama University. The DNA fragment containing the streptavidin gene was inserted upstream of the CQ peptide sequence in place of the DHFR gene. Respective <sup>32</sup>P-labeled RNAs [streptavidin−(CQ)<sub>3</sub>−(Apt)<sub>3</sub> and DHFR−(CQ)<sub>3</sub>−(Apt)<sub>3</sub>] were prepared as described above. Three sets of mRNA (0.5 μg in each set) (a mixture of mRNAs in the ratio of 1:1 or each mRNA-encoding streptavidin or DHFR as a control) were translated without adding DTT and Mg(OAc)<sub>2</sub> using the Flexi rabbit reticulocyte lysate system (Promega) (50 μL) for 15 min at 30 °C. After translation, 1 mL of binding buffer [140 mM NaCl, 2.7 mM KCl, 10.1 mM Na<sub>2</sub>HPO<sub>4</sub>, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, 0.04% Tween20, 2% Block Ace (Dainippon Pharmaceutical Co., Japan), 5 mM MgCl<sub>2</sub> (pH 7.3)] and 10 μL of a suspension of biotin−agarose beads (Sigma) were added to the lysate and binding was allowed to proceed for 10 min on ice. Supernatant containing unbound mRNAs and protein were removed from the mixture, and MTX beads were washed twice with 200 μL of washing buffer [140 mM NaCl, 2.7 mM KCl, 10.1 mM Na<sub>2</sub>HPO<sub>4</sub>, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, 0.04% Tween20, 2% Block Ace (Dainippon Pharmaceutical Co.), 50 mM EDTA (pH 7.3)]. Bound mRNAs were shaken vigorously in elution buffer [1 M NaCl, 50 mM EDTA] at room temperature for 30 min in order to isolate the binding mRNA from biotin beads. Each sample was purified with an RNeasy kit (QIAGEN, Hilden, Germany) and was used for reverse transcription and PCR. </div><div class="NLM_p"><b>Reverse Transcription and PCR. </b>Reverse transcription was performed with AMV reverse transcriptase (Promega) according to the supplier's recommendation. PCR was performed using 15 cycles with Ex Taq polymerase (Takara Co.). Products of PCR were analyzed by electrophoresis on agarose gels. </div><div class="NLM_p"><b>Combination of the Strong Interaction between (RE)<sub>3</sub> and (Apt)<sub>3</sub> with the Ribosome-Display System in the Selection of Functional Proteins. </b>The pET30a that contains the DHFR gene (pDHFR) and pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> plasmid, described above, was used as a template for the preparation of respective <sup>32</sup>P-labeled mRNAs [DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop, DHFR + Stop, and DHFR − Stop mRNAs]. <sup>32</sup>P-labeled mRNAs were translated and were subjected to selection by MTX beads as described above. Bound mRNAs were isolated from the MTX beads by elution at an elevated temperature. The comparative amounts of selected mRNAs were detected by a count of <sup>32</sup>P-labeled mRNA and RT-PCR. </div><div class="NLM_p last">The p(Apt)<sub>3</sub>−(RE)<sub>3</sub>−DHFR was constructed in the same way as pDHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub>. The linear DNAs for in vitro transcription were prepared as PCR products from respective template plasmids [p(Apt)<sub>3</sub>−(RE)<sub>3</sub>−DHFR and pDHFR] that possessed T7 promoter sequence. The mRNAs were prepared as described above. Polyclonal IgG antibodies against <i>E. coli</i> RF1 to RF3 were prepared separately for each releasing factor (<i>E. coli</i> RF1 to RF3), which was a kind gift from Professor Takuya Ueda in the University of Tokyo. The total amounts of mRNA in start pool were 2 μg, and in vitro translation was carried out at 37 °C for 10 min using the <i>E. coli</i> extract system for linear templates (Promega). The translated products were put on ice for 1 min in order to stop the translation reaction and performed the in vitro selection using MTX beads, as described above. The recovered mRNA was purified by RNeasy mini kits (QIAGEN), and it was reverse-transcribed by M-MLV reverse transcriptase (Promega). This reverse-transcribed sample, i.e., cDNA, was amplified by PCR (ExTaq; Takara) and analyzed by 2.0% agarose electrophoresis. To perform the semiquantitative analysis, the agarose gel was stained by SYBR Green I (Molecular Probes, Eugene, OR) and analyzed by Fluoro Imager 595 (Molecular Dynamics, Sunnyvale, CA). The strength of the interaction between (RE)<sub>3</sub> and (Apt)<sub>3</sub> was <a name="jm004500801606" id="jm004500801606"></a> estimated on the basis of the relative amounts of recovered mRNAs as RT-PCR products. </div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70358" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70358" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satoshi Fujita</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shinya Y. Sawata</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rika Yamamoto-Fujita</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yaeta Endo</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hideo Kise</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masahiro Iwakura</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kazunari Taira</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and Biotechnology, School of Engineering, The University of Tokyo, Hongo, Tokyo 113-8656, Japan, Gene Discovery Research Center and Institute of Molecular and Cell Biology, National Institute of Advanced Industrial Science and Technology (AIST), 1-1-1 Higashi, Tsukuba Science City 305-8562, Japan, Department of Applied Chemistry, Faculty of Engineering, Ehime University, Matsuyama 790-8577, Japan, and Institute of Materials Science, University of Tsukuba, 1-1-1 Tennoudai, Tsukuba Science City 305-8572, Japan </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm010398p" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Prof. M. Sisido and Dr. T. Hosaka of the University of Okayama for the kind gift of plasmid encoding streptavidin and Dr. Tatsuya Sawasaki, Mr. Akihiko Ozawa, and Dr. Kairat Madin of Ehime University for helpful comments. This research was supported by grants from Promotion of Basic Research Activities for Innovative Biosciences (PROBRAIN) and by grants from the Ministry of Economy, Trade and Industry (METI) of Japan. S.F. and R.Y.F. are recipients of the JSPS research fellowship, and S.Y.S. is a recipient of the JST research fellowship. </p></div><div class="refs-header-label"></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="jm010398pb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm010398pb00001_1"><span><span class="NLM_string-ref">Fields, S.; Song, O. A novel genetic system to detect protein−protein interactions. <i>  Nature</i><b>1989</b>, <i>340</i>, 245−246.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=1989&pages=246&author=S.+Fields&author=O+Song&title=A+novel+genetic+system+to+detect+protein%E2%88%92protein+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFields%26aufirst%3DS.%26atitle%3DA%2520novel%2520genetic%2520system%2520to%2520detect%2520protein%25E2%2588%2592protein%2520interactions%26jtitle%3DNature%26date%3D1989%26volume%3D340%26spage%3D246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm010398pb00002_1"><span><span class="NLM_string-ref">Harada, K.; Martin, S. S.; Frankel, A. D. Selection of RNA-binding peptides in vivo. <i>  Nature</i><b>1996</b>, <i>380</i>, 175−179.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=179&author=K.+Harada&author=S.+S.+Martin&author=A.+D+Frankel&title=Selection+of+RNA-binding+peptides+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DK.%26atitle%3DSelection%2520of%2520RNA-binding%2520peptides%2520in%2520vivo%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm010398pb00003_1"><span><span class="NLM_string-ref">Moore, J. C.; Arnold, F. H. Directed evolution of a para-nitrobenzyl esterase for aqueous−organic solvents. <i>  Nat. Biotechnol.</i><b>1996</b>, <i>14</i>, 458−467.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=467&author=J.+C.+Moore&author=F.+H+Arnold&title=Directed+evolution+of+a+para-nitrobenzyl+esterase+for+aqueous%E2%88%92organic+solvents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BC.%26atitle%3DDirected%2520evolution%2520of%2520a%2520para-nitrobenzyl%2520esterase%2520for%2520aqueous%25E2%2588%2592organic%2520solvents%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm010398pb00004_1"><span><span class="NLM_string-ref">Schatz, P. J.; Cull, M. G.; Martin, E. L.; Gates, C. M. Screening of peptide libraries linked to lac repressor. <i>  Methods Enzymol.</i><b>1996</b>, <i>267</i>, 171−191.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1996&pages=191&author=P.+J.+Schatz&author=M.+G.+Cull&author=E.+L.+Martin&author=C.+M+Gates&title=Screening+of+peptide+libraries+linked+to+lac+repressor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchatz%26aufirst%3DP.%2BJ.%26atitle%3DScreening%2520of%2520peptide%2520libraries%2520linked%2520to%2520lac%2520repressor%26jtitle%3DMethods%2520Enzymol.%26date%3D1996%26volume%3D267%26spage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm010398pb00005_1"><span><span class="NLM_string-ref">Boder, E. T.; Wittrup, K. D. Yeast surface display for screening combinatorial polypeptide libraries. <i>  Nat. Biotechnol.</i><b>1997</b>, <i>15</i>, 553−557.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=557&author=E.+T.+Boder&author=K.+D+Wittrup&title=Yeast+surface+display+for+screening+combinatorial+polypeptide+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoder%26aufirst%3DE.%2BT.%26atitle%3DYeast%2520surface%2520display%2520for%2520screening%2520combinatorial%2520polypeptide%2520libraries%26jtitle%3DNat.%2520Biotechnol.%26date%3D1997%26volume%3D15%26spage%3D557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm010398pb00006_1"><span><span class="NLM_string-ref">Smith, G. P.; Petrenko, V. A. Phage display. <i>  Chem. Rev.</i><b>1997</b>, <i>97</i>, 391−410.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=410&author=G.+P.+Smith&author=V.+A+Petrenko&title=Phage+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DG.%2BP.%26atitle%3DPhage%2520display%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm010398pb00007_1"><span><span class="NLM_string-ref">Mattheakis, L. C.; Bhatt, R. R.; Dower, W. J. An in vitro polysome display system for identifying ligands from very large peptide libraries. <i>  Proc. Natl. </i><i>Acad. Sci. U.S.A.</i><b>1994</b>, <i>91</i>, 9022−9026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=9026&author=L.+C.+Mattheakis&author=R.+R.+Bhatt&author=W.+J+Dower&title=An+in+vitro+polysome+display+system+for+identifying+ligands+from+very+large+peptide+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMattheakis%26aufirst%3DL.%2BC.%26atitle%3DAn%2520in%2520vitro%2520polysome%2520display%2520system%2520for%2520identifying%2520ligands%2520from%2520very%2520large%2520peptide%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1994%26volume%3D91%26spage%3D9026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm010398pb00008_1"><span><span class="NLM_string-ref">Mattheakis, L. C.; Dias, J. M.; Dower, W. J. Cell-free synthesis of peptide libraries displayed on polysomes. <i>  Methods Enzymol.</i><b>1996</b>, <i>267</i>, 195−207.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1996&pages=207&author=L.+C.+Mattheakis&author=J.+M.+Dias&author=W.+J+Dower&title=Cell-free+synthesis+of+peptide+libraries+displayed+on+polysomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMattheakis%26aufirst%3DL.%2BC.%26atitle%3DCell-free%2520synthesis%2520of%2520peptide%2520libraries%2520displayed%2520on%2520polysomes%26jtitle%3DMethods%2520Enzymol.%26date%3D1996%26volume%3D267%26spage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm010398pb00009_1"><span><span class="NLM_string-ref">Gersuk, G. M.; Corey, M. J.; Corey, E.; Stray, J. E.; Kawasaki, G. H.; Vessella, R. L. High-affinity peptide ligands to prostate-specific antigen identified by polysome selection. <i>  Biochem. Biophys. Res. Commun. </i><b>1997</b>, <i>232</i>, 578−582.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=232&publication_year=1997&pages=582&author=G.+M.+Gersuk&author=M.+J.+Corey&author=E.+Corey&author=J.+E.+Stray&author=G.+H.+Kawasaki&author=R.+L+Vessella&title=High-affinity+peptide+ligands+to+prostate-specific+antigen+identified+by+polysome+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGersuk%26aufirst%3DG.%2BM.%26atitle%3DHigh-affinity%2520peptide%2520ligands%2520to%2520prostate-specific%2520antigen%2520identified%2520by%2520polysome%2520selection%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D232%26spage%3D582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00010"><div class="reference"><div class="NLM_citation" id="jm010398pb00010_1"><span><span class="NLM_string-ref">Hanes, J.; Plückthun, A. In vitro selection and evolution of functional proteins by using ribosome display. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 4937−4942.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=4942&author=J.+Hanes&author=A+Pl%C3%BCckthun&title=In+vitro+selection+and+evolution+of+functional+proteins+by+using+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520selection%2520and%2520evolution%2520of%2520functional%2520proteins%2520by%2520using%2520ribosome%2520display%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D4942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm010398pb00011_1"><span><span class="NLM_string-ref">He, M.; Taussig, M. J. Antibody−ribosome−mRNA (ARM) complexes as efficient selection particles for in vitro display and evolution of antibody combining sites. <i>  Nucleic Acids Res.</i><b>1997</b>, <i>25</i>, 5132−5134.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=5134&author=M.+He&author=M.+J.+Taussig&author=+Antibody%E2%88%92ribosome%E2%88%92m&title=complexes+as+efficient+selection+particles+for+in+vitro+display+and+evolution+of+antibody+combining+sites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26atitle%3Dcomplexes%2520as%2520efficient%2520selection%2520particles%2520for%2520in%2520vitro%2520display%2520and%2520evolution%2520of%2520antibody%2520combining%2520sites%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D5134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm010398pb00012_1"><span><span class="NLM_string-ref">Nemoto, N.; Miyamoto-Sato, E.; Husimi, Y.; Yanagawa, H. In vitro virus:  bonding of mRNA bearing puromycin at the 3‘-terminal end to the C-terminal end of its encoded protein on the ribosome in vitro. <i>  FEBS Lett.</i><b>1997</b>, <i>414</i>, 405−408.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=1997&pages=408&author=N.+Nemoto&author=E.+Miyamoto-Sato&author=Y.+Husimi&author=H+Yanagawa&title=In+vitro+virus%3A%E2%80%89+bonding+of+mRNA+bearing+puromycin+at+the+3%E2%80%98-terminal+end+to+the+C-terminal+end+of+its+encoded+protein+on+the+ribosome+in+vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNemoto%26aufirst%3DN.%26atitle%3DIn%2520vitro%2520virus%253A%25E2%2580%2589%2520bonding%2520of%2520mRNA%2520bearing%2520puromycin%2520at%2520the%25203%25E2%2580%2598-terminal%2520end%2520to%2520the%2520C-terminal%2520end%2520of%2520its%2520encoded%2520protein%2520on%2520the%2520ribosome%2520in%2520vitro%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D414%26spage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm010398pb00013_1"><span><span class="NLM_string-ref">Roberts, R. W.; Szostak, J. W. RNA−peptide fusions for the in vitro selection of peptides and proteins. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 12297−12302.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=12302&author=R.+W.+Roberts&author=J.+W+Szostak&title=RNA%E2%88%92peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DR.%2BW.%26atitle%3DRNA%25E2%2588%2592peptide%2520fusions%2520for%2520the%2520in%2520vitro%2520selection%2520of%2520peptides%2520and%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D12302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm010398pb00014_1"><span><span class="NLM_string-ref">Hanes, J.; Jermutus, L.; Weber-Bornhauser, S.; Bosshard, H. R.; Plückthun, A. Ribosome display efficiently selects and evolves high-affinity antibodies in vitro from immune libraries. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1998</b>, <i>95</i>, 14130−14135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=14135&author=J.+Hanes&author=L.+Jermutus&author=S.+Weber-Bornhauser&author=H.+R.+Bosshard&author=A+Pl%C3%BCckthun&title=Ribosome+display+efficiently+selects+and+evolves+high-affinity+antibodies+in+vitro+from+immune+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DRibosome%2520display%2520efficiently%2520selects%2520and%2520evolves%2520high-affinity%2520antibodies%2520in%2520vitro%2520from%2520immune%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D14135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm010398pb00015_1"><span><span class="NLM_string-ref">Tawfik, D. S.; Griffiths, A. D. Man-made cell-like compartments for molecular evolution. <i>  Nat. Biotechnol.</i><b>1998</b>, <i>16</i>, 652−656.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1998&pages=656&author=D.+S.+Tawfik&author=A.+D+Griffiths&title=Man-made+cell-like+compartments+for+molecular+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTawfik%26aufirst%3DD.%2BS.%26atitle%3DMan-made%2520cell-like%2520compartments%2520for%2520molecular%2520evolution%26jtitle%3DNat.%2520Biotechnol.%26date%3D1998%26volume%3D16%26spage%3D656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm010398pb00016_1"><span><span class="NLM_string-ref">Doi, N.; Yanagawa, H. STABLE:  protein−DNA fusion system for screening of combinatorial protein libraries in vitro. <i>  FEBS Lett.</i><b>1999</b>, <i>457</i>, 227−230.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=1999&pages=230&author=N.+Doi&author=H.+Yanagawa&title=protein%E2%88%92DNA+fusion+system+for+screening+of+combinatorial+protein+libraries+in+vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDoi%26aufirst%3DN.%26atitle%3Dprotein%25E2%2588%2592DNA%2520fusion%2520system%2520for%2520screening%2520of%2520combinatorial%2520protein%2520libraries%2520in%2520vitro%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D457%26spage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm010398pb00017_1"><span><span class="NLM_string-ref">Hanes, J.; Jermutus, L.; Schaffitzel, C.; Plückthun, A. Comparison of <i>Escherichia coli</i> and rabbit reticulocyte ribosome display systems. <i>  FEBS </i><i>Lett.</i><b>1999</b>, <i>450</i>, 105−110.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=1999&pages=110&author=J.+Hanes&author=L.+Jermutus&author=C.+Schaffitzel&author=A+Pl%C3%BCckthun&title=Comparison+of+Escherichia+coli+and+rabbit+reticulocyte+ribosome+display+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DComparison%2520of%2520Escherichia%2520coli%2520and%2520rabbit%2520reticulocyte%2520ribosome%2520display%2520systems%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D450%26spage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm010398pb00018_1"><span><span class="NLM_string-ref">Makeyev, E. V.; Kolb, V. A.; Spirin, A. S. Cell-free immunology:  construction and in vitro expression of a PCR-based library encoding a single-chain antibody repertoire. <i>  FEBS Lett.</i><b>1999</b>, <i>444</i>, 177−180.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=1999&pages=180&author=E.+V.+Makeyev&author=V.+A.+Kolb&author=A.+S.+Spirin&author=+Cell&title=construction+and+in+vitro+expression+of+a+PCR-based+library+encoding+a+single-chain+antibody+repertoire"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMakeyev%26aufirst%3DE.%2BV.%26atitle%3Dconstruction%2520and%2520in%2520vitro%2520expression%2520of%2520a%2520PCR-based%2520library%2520encoding%2520a%2520single-chain%2520antibody%2520repertoire%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D444%26spage%3D180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm010398pb00019_1"><span><span class="NLM_string-ref">He, M.; Menges, M.; Groves, M. A. T.; Corps, E.; Liu, H.; Brüggemann, M.; Taussig, M. J. Selection of a human anti-progesterone antibody fragment from a transgenic mouse library by ARM ribosome display. <i>  J. Immunol. </i><i>Methods</i><b>1999</b>, <i>231</i>, 105−117.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=1999&pages=117&author=M.+He&author=M.+Menges&author=M.+A.+T.+Grovesauthor=H.+Liu&author=M.+Br%C3%BCggemann&author=M.+J+Taussig&title=Selection+of+a+human+anti-progesterone+antibody+fragment+from+a+transgenic+mouse+library+by+ARM+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3D%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DSelection%2520of%2520a%2520human%2520anti-progesterone%2520antibody%2520fragment%2520from%2520a%2520transgenic%2520mouse%2520library%2520by%2520ARM%2520ribosome%2520display%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1999%26volume%3D231%26spage%3D117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00020"><div class="reference"><div class="NLM_citation" id="jm010398pb00020_1"><span><span class="NLM_string-ref">Schaffitzel, C.; Hanes, J.; Jermutus, L.; Plückthun, A. Ribosome display:  an in vitro method for selection and evolution of antibodies from libraries. <i>  J. </i><i>Immunol. Methods</i><b>1999</b>, <i>231</i>, 119−135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=1999&pages=135&author=C.+Schaffitzel&author=J.+Hanes&author=L.+Jermutus&author=A.+Pl%C3%BCckthun&author=+Ribosome&title=an+in+vitro+method+for+selection+and+evolution+of+antibodies+from+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffitzel%26aufirst%3DC.%26atitle%3Dan%2520in%2520vitro%2520method%2520for%2520selection%2520and%2520evolution%2520of%2520antibodies%2520from%2520libraries%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1999%26volume%3D231%26spage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm010398pb00021_1"><span><span class="NLM_string-ref">Bieberich, E.; Kapitonov, D.; Tencomnao, T.; Yu, R. K. Protein−ribosome−mRNA display:  affinity isolation of enzyme−ribosome−mRNA complexes and cDNA cloning in a single-tube reaction. <i>  Anal. Biochem.</i><b>2000</b>, <i>287</i>, 294−298.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=298&author=E.+Bieberich&author=D.+Kapitonov&author=T.+Tencomnao&author=R.+K.+Yu&author=+Protein%E2%88%92ribosome%E2%88%92m&title=affinity+isolation+of+enzyme%E2%88%92ribosome%E2%88%92mRNA+complexes+and+cDNA+cloning+in+a+single-tube+reaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBieberich%26aufirst%3DE.%26atitle%3Daffinity%2520isolation%2520of%2520enzyme%25E2%2588%2592ribosome%25E2%2588%2592mRNA%2520complexes%2520and%2520cDNA%2520cloning%2520in%2520a%2520single-tube%2520reaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D287%26spage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm010398pb00022_1"><span><span class="NLM_string-ref">Cho, G.; Keefe, A. D.; Liu, R.; Wilson, D. S.; Szostak, J. W. Constructing high complexity synthetic libraries of long ORFs using in vitro selection. <i>  J. Mol. </i><i>Biol.</i><b>2000</b>, <i>297</i>, 309−319.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2000&pages=319&author=G.+Cho&author=A.+D.+Keefe&author=R.+Liu&author=D.+S.+Wilson&author=J.+W+Szostak&title=Constructing+high+complexity+synthetic+libraries+of+long+ORFs+using+in+vitro+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DG.%26atitle%3DConstructing%2520high%2520complexity%2520synthetic%2520libraries%2520of%2520long%2520ORFs%2520using%2520in%2520vitro%2520selection%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D297%26spage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm010398pb00023_1"><span><span class="NLM_string-ref">Liu, R.; Barrick, J. E.; Szostak, J. W.; Roberts, R. W. Optimized synthesis of RNA−protein fusions for in vitro protein selection. <i>  Methods Enzymol.</i><b>2000</b>, <i>318</i>, 268−293.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2000&pages=293&author=R.+Liu&author=J.+E.+Barrick&author=J.+W.+Szostak&author=R.+W+Roberts&title=Optimized+synthesis+of+RNA%E2%88%92protein+fusions+for+in+vitro+protein+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DR.%26atitle%3DOptimized%2520synthesis%2520of%2520RNA%25E2%2588%2592protein%2520fusions%2520for%2520in%2520vitro%2520protein%2520selection%26jtitle%3DMethods%2520Enzymol.%26date%3D2000%26volume%3D318%26spage%3D293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm010398pb00024_1"><span><span class="NLM_string-ref">Hanes, J.; Schaffitzel, C.; Knappik, A.; Plückthun, A. Picomolar affinity antibodies from a fully synthetic naive library selected and evolved by ribosome display. <i>  Nat. Biotechnol.</i><b>2000</b>, <i>18</i>, 1287−1292.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=1292&author=J.+Hanes&author=C.+Schaffitzel&author=A.+Knappik&author=A+Pl%C3%BCckthun&title=Picomolar+affinity+antibodies+from+a+fully+synthetic+naive+library+selected+and+evolved+by+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DPicomolar%2520affinity%2520antibodies%2520from%2520a%2520fully%2520synthetic%2520naive%2520library%2520selected%2520and%2520evolved%2520by%2520ribosome%2520display%26jtitle%3DNat.%2520Biotechnol.%26date%3D2000%26volume%3D18%26spage%3D1292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm010398pb00025_1"><span><span class="NLM_string-ref">Jermutus, L.; Honegger, A.; Schwesinger, F.; Hanes, J.; Plückthun, A. Tailoring in vitro evolution for protein affinity or stability. <i>  Proc. Natl. Acad. Sci. </i><i>U.S.A.</i><b>2001</b>, <i>98</i>, 75−80.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=80&author=L.+Jermutus&author=A.+Honegger&author=F.+Schwesinger&author=J.+Hanes&author=A+Pl%C3%BCckthun&title=Tailoring+in+vitro+evolution+for+protein+affinity+or+stability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJermutus%26aufirst%3DL.%26atitle%3DTailoring%2520in%2520vitro%2520evolution%2520for%2520protein%2520affinity%2520or%2520stability%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm010398pb00026_1"><span><span class="NLM_string-ref">Keefe, A. D.; Szostak, J. W. Functional proteins from a random-sequence library. <i>  Nature</i><b>2001</b>, <i>410</i>, 715−718.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2001&pages=718&author=A.+D.+Keefe&author=J.+W+Szostak&title=Functional+proteins+from+a+random-sequence+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DFunctional%2520proteins%2520from%2520a%2520random-sequence%2520library%26jtitle%3DNature%26date%3D2001%26volume%3D410%26spage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm010398pb00027_1"><span><span class="NLM_string-ref">Wilson, D. S.; Keefe, A. D.; Szostak, J. W. The use of mRNA display to select high-affinity protein-binding peptides. <i>  Proc. Natl. Acad. Sci. U.S.A</i>. <b>2001</b>, <i>98</i>, 3750−3755.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3755&author=D.+S.+Wilson&author=A.+D.+Keefe&author=J.+W+Szostak&title=The+use+of+mRNA+display+to+select+high-affinity+protein-binding+peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BS.%26atitle%3DThe%2520use%2520of%2520mRNA%2520display%2520to%2520select%2520high-affinity%2520protein-binding%2520peptides%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm010398pb00028_1"><span><span class="NLM_string-ref">Lamla, T.; Erdmann, V. A. In vitro selection of other proteins than antibodies by means of ribosome display. <i>  FEBS Lett.</i><b>2001</b>, <i>502</i>, 35−40.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2001&pages=40&author=T.+Lamla&author=V.+A+Erdmann&title=In+vitro+selection+of+other+proteins+than+antibodies+by+means+of+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamla%26aufirst%3DT.%26atitle%3DIn%2520vitro%2520selection%2520of%2520other%2520proteins%2520than%2520antibodies%2520by%2520means%2520of%2520ribosome%2520display%26jtitle%3DFEBS%2520Lett.%26date%3D2001%26volume%3D502%26spage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm010398pb00029_1"><span><span class="NLM_string-ref">Yamamoto, R.; Katahira, M.; Nishikawa, S.; Taira, K.; Kumar, P. K. R. A novel RNA motif that binds efficiently and specifically to the Tat protein of HIV and inhibits the trans-activation by Tat of transcription in vitro and in vivo. <i>  Genes Cells</i><b>2000</b>, <i>5</i>, 371−388.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=388&author=R.+Yamamoto&author=M.+Katahira&author=S.+Nishikawa&author=K.+Taira&author=P.+K.+R+Kumar&title=A+novel+RNA+motif+that+binds+efficiently+and+specifically+to+the+Tat+protein+of+HIV+and+inhibits+the+trans-activation+by+Tat+of+transcription+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DR.%26atitle%3DA%2520novel%2520RNA%2520motif%2520that%2520binds%2520efficiently%2520and%2520specifically%2520to%2520the%2520Tat%2520protein%2520of%2520HIV%2520and%2520inhibits%2520the%2520trans-activation%2520by%2520Tat%2520of%2520transcription%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DGenes%2520Cells%26date%3D2000%26volume%3D5%26spage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00030"><div class="reference"><div class="NLM_citation" id="jm010398pb00030_1"><span><span class="NLM_string-ref">Jencks, W. P. On the attribution and additivity of binding energies. <i>  Proc. Natl. </i><i>Acad. Sci. U.S.A.</i><b>1981</b>, <i>78</i>, 4046−4050.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4050&author=W.+P+Jencks&title=On+the+attribution+and+additivity+of+binding+energies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1981%26volume%3D78%26spage%3D4050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm010398pb00031_1"><span><span class="NLM_string-ref">Shuker, S. B.; Hajduk, P. J.; Meadows, R. P.; Fesik, S. W. Discovering high-affinity ligands for proteins:  SAR by NMR. <i>  Science</i><b>1996</b>, <i>274</i>, 1531−1534.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1534&author=S.+B.+Shuker&author=P.+J.+Hajduk&author=R.+P.+Meadows&author=S.+W+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A%E2%80%89+SAR+by+NMR"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%25E2%2580%2589%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm010398pb00032_1"><span><span class="NLM_string-ref">Blakley, R. L. In <i>Folates and Pterins</i>; Blakley, R. L., Benkovic, S. J., Eds.; Wiley:  New York, 1985; pp 191−253.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1985&pages=253&author=R.+Blakley&title=Folates+and+Pterins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBlakley%26aufirst%3DR.%26atitle%3DL%26btitle%3DFolates%2520and%2520Pterins%26pub%3DWiley%26date%3D1985%26spage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm010398pb00033_1"><span><span class="NLM_string-ref">Taira, K.; Fierke, C. A.; Chen, J.-T.; Johnson, K. A.; Benkovic, S. J. On interpreting the inhibition of and catalysis by dihydrofolate-reductase. <i>  Trends Biochem. Sci.</i><b>1987</b>, <i>12</i>, 275−278.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1987&pages=278&author=K.+Taira&author=C.+A.+Fierke&author=J.-T.+Chen&author=K.+A.+Johnson&author=S.+J+Benkovic&title=On+interpreting+the+inhibition+of+and+catalysis+by+dihydrofolate-reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaira%26aufirst%3DK.%26atitle%3DOn%2520interpreting%2520the%2520inhibition%2520of%2520and%2520catalysis%2520by%2520dihydrofolate-reductase%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1987%26volume%3D12%26spage%3D278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm010398pb00034_1"><span><span class="NLM_string-ref">Taira, K.; Benkovic, S. J. Evaluation of the importance of hydrophobic interactions in drug binding to dihydrofolate reductase. <i>  J. Med. Chem.</i><b>1988</b>, <i>31</i>, 129−137.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=137&author=K.+Taira&author=S.+J+Benkovic&title=Evaluation+of+the+importance+of+hydrophobic+interactions+in+drug+binding+to+dihydrofolate+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaira%26aufirst%3DK.%26atitle%3DEvaluation%2520of%2520the%2520importance%2520of%2520hydrophobic%2520interactions%2520in%2520drug%2520binding%2520to%2520dihydrofolate%2520reductase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm010398pb00035_1"><span><span class="NLM_string-ref">Iwakura, M.; Jones, B. E.; Luo, J.; Matthews, C. R. A strategy for testing the suitability of cysteine replacements in dihydrofolate reductase from <i>Escherichia coli</i>. <i>  J. Biochem. (Tokyo)</i><b>1995</b>, <i>117</i>, 480−488.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=488&author=M.+Iwakura&author=B.+E.+Jones&author=J.+Luo&author=C.+R+Matthews&title=A+strategy+for+testing+the+suitability+of+cysteine+replacements+in+dihydrofolate+reductase+from+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIwakura%26aufirst%3DM.%26atitle%3DA%2520strategy%2520for%2520testing%2520the%2520suitability%2520of%2520cysteine%2520replacements%2520in%2520dihydrofolate%2520reductase%2520from%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biochem.%2520%2528Tokyo%2529%26date%3D1995%26volume%3D117%26spage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm010398pb00036_1"><span><span class="NLM_string-ref">Fujita, S.; Koguma, T.; Ohkawa, J.; Mori, K.; Kohda, T.; Kise, H.; Nishikawa, S.; Iwakura, M.; Taira, K. Discrimination of a single base change in a ribozyme using the gene for dihydrofolate reductase as a selective marker. in <i>Escherichia coli</i>. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 391−396.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=396&author=S.+Fujita&author=T.+Koguma&author=J.+Ohkawa&author=K.+Mori&author=T.+Kohda&author=H.+Kise&author=S.+Nishikawa&author=M.+Iwakura&author=K+Taira&title=Discrimination+of+a+single+base+change+in+a+ribozyme+using+the+gene+for+dihydrofolate+reductase+as+a+selective+marker.+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DS.%26atitle%3DDiscrimination%2520of%2520a%2520single%2520base%2520change%2520in%2520a%2520ribozyme%2520using%2520the%2520gene%2520for%2520dihydrofolate%2520reductase%2520as%2520a%2520selective%2520marker.%2520in%2520Escherichia%2520coli%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm010398pb00037_1"><span><span class="NLM_string-ref">Hamada, M.; Fujita, S.; Kise, H.; Jigami, Y.; Taira, K. Comparison of in vivo activities of 5‘-connected and 3‘-connected cis-acting ribozymes:  selection of intracellularly active ribozymes using the gene for dihydrofolate reductase (DHFR) as a selective marker in <i>Escherichia coli</i>. <i>  J. Biochem. </i><i>(Tokyo)</i><b>1998</b>, <i>123</i>, 684−692.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=1998&pages=692&author=M.+Hamada&author=S.+Fujita&author=H.+Kise&author=Y.+Jigami&author=K+Taira&title=Comparison+of+in+vivo+activities+of+5%E2%80%98-connected+and+3%E2%80%98-connected+cis-acting+ribozymes%3A%E2%80%89+selection+of+intracellularly+active+ribozymes+using+the+gene+for+dihydrofolate+reductase+%28DHFR%29+as+a+selective+marker+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DM.%26atitle%3DComparison%2520of%2520in%2520vivo%2520activities%2520of%25205%25E2%2580%2598-connected%2520and%25203%25E2%2580%2598-connected%2520cis-acting%2520ribozymes%253A%25E2%2580%2589%2520selection%2520of%2520intracellularly%2520active%2520ribozymes%2520using%2520the%2520gene%2520for%2520dihydrofolate%2520reductase%2520%2528DHFR%2529%2520as%2520a%2520selective%2520marker%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biochem.%2520%2528Tokyo%2529%26date%3D1998%26volume%3D123%26spage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm010398pb00038_1"><span><span class="NLM_string-ref">Green, N. M. Avidin. <i>  Adv. Protein Chem.</i><b>1975</b>, <i>29</i>, 85−133.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Green%2C+N.+M.+Avidin.+Adv.+Protein+Chem.+1975%2C+29%2C+85%E2%88%92133.+Green%2C+N.+M.+Avidin.+Adv.+Protein+Chem.1975%2C+29%2C+85%E2%88%92133."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DN.%2BM.%26jtitle%3DAdv.%2520Protein%2520Chem.%26date%3D1975%26volume%3D29%26spage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00039"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm010398pb00039'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="jm010398pb00039_1"><span><span class="NLM_string-ref">Moffat, J. G.; Timms, K. M.; Trotman, C. N. A.; Tate, W. P. Interaction of the release factors with the <i>Escherichia coli</i> ribosome:  structurally and functionally important domains.<i> Biochimie</i><b>1991</b>, <i>73</i>, 1113−1120.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=1991&pages=1120&author=J.+G.+Moffat&author=K.+M.+Timms&author=C.+N.+A.+Trotman&author=W.+P+Tate&title=Interaction+of+the+release+factors+with+the+Escherichia+coli+ribosome%3A%E2%80%89+structurally+and+functionally+important+domains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26atitle%3DInteraction%2520of%2520the%2520release%2520factors%2520with%2520the%2520Escherichia%2520coli%2520ribosome%253A%25E2%2580%2589%2520structurally%2520and%2520functionally%2520important%2520domains%26jtitle%3DBiochimie%26date%3D1991%26volume%3D73%26spage%3D1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm010398pb00040"><div class="reference"><div class="NLM_citation" id="jm010398pb00040_1"><span><span class="NLM_string-ref">Zhou, J.-M.; Fujita, S.; Warashina, M.; Baba, T.; Taira, K. <i>  J. Am. Chem. Soc.</i><b>2002</b>, <i>124</i>, 538−543.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Zhou%2C+J.-M.%3B+Fujita%2C+S.%3B+Warashina%2C+M.%3B+Baba%2C+T.%3B+Taira%2C+K.+J.+Am.+Chem.+Soc.+2002%2C+124%2C+538%E2%88%92543.+Zhou%2C+J.-M.%3B+Fujita%2C+S.%3B+Warashina%2C+M.%3B+Baba%2C+T.%3B+Taira%2C+K.+J.+Am.+Chem.+Soc.2002%2C+124%2C+538%E2%88%92543."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.-M.%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm010398pb00001':['jm010398pb00001_1'],'jm010398pb00002':['jm010398pb00002_1'],'jm010398pb00003':['jm010398pb00003_1'],'jm010398pb00004':['jm010398pb00004_1'],'jm010398pb00005':['jm010398pb00005_1'],'jm010398pb00006':['jm010398pb00006_1'],'jm010398pb00007':['jm010398pb00007_1'],'jm010398pb00008':['jm010398pb00008_1'],'jm010398pb00009':['jm010398pb00009_1'],'jm010398pb00010':['jm010398pb00010_1'],'jm010398pb00011':['jm010398pb00011_1'],'jm010398pb00012':['jm010398pb00012_1'],'jm010398pb00013':['jm010398pb00013_1'],'jm010398pb00014':['jm010398pb00014_1'],'jm010398pb00015':['jm010398pb00015_1'],'jm010398pb00016':['jm010398pb00016_1'],'jm010398pb00017':['jm010398pb00017_1'],'jm010398pb00018':['jm010398pb00018_1'],'jm010398pb00019':['jm010398pb00019_1'],'jm010398pb00020':['jm010398pb00020_1'],'jm010398pb00021':['jm010398pb00021_1'],'jm010398pb00022':['jm010398pb00022_1'],'jm010398pb00023':['jm010398pb00023_1'],'jm010398pb00024':['jm010398pb00024_1'],'jm010398pb00025':['jm010398pb00025_1'],'jm010398pb00026':['jm010398pb00026_1'],'jm010398pb00027':['jm010398pb00027_1'],'jm010398pb00028':['jm010398pb00028_1'],'jm010398pb00029':['jm010398pb00029_1'],'jm010398pb00030':['jm010398pb00030_1'],'jm010398pb00031':['jm010398pb00031_1'],'jm010398pb00032':['jm010398pb00032_1'],'jm010398pb00033':['jm010398pb00033_1'],'jm010398pb00034':['jm010398pb00034_1'],'jm010398pb00035':['jm010398pb00035_1'],'jm010398pb00036':['jm010398pb00036_1'],'jm010398pb00037':['jm010398pb00037_1'],'jm010398pb00038':['jm010398pb00038_1'],'jm010398pb00039':['jm010398pb00039_1'],'jm010398pb00040':['jm010398pb00040_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Karuso</span>. </span><span class="cited-content_cbyCitation_article-title">Modern Methods for the Isolation of Natural Product Receptors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 513-567. <a href="https://doi.org/10.1016/B978-008045382-8.00210-0" title="DOI URL">https://doi.org/10.1016/B978-008045382-8.00210-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-008045382-8.00210-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-008045382-8.00210-0%26sid%3Dliteratum%253Aachs%26atitle%3DModern%252BMethods%252Bfor%252Bthe%252BIsolation%252Bof%252BNatural%252BProduct%252BReceptors%26aulast%3DKaruso%26aufirst%3DPeter%26date%3D2010%26spage%3D513%26epage%3D567%26pub%3DElsevier%26atitle%3DComprehensive%252BNatural%252BProducts%252BII%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yutaka  Ikeda</span>, <span class="hlFld-ContribAuthor ">Kazunari  Taira</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-Targeted Delivery of Therapeutic siRNA. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Research</span><span> <strong>2006,</strong> <em>23 </em>
                                    (8)
                                     , 1631-1640. <a href="https://doi.org/10.1007/s11095-006-9001-x" title="DOI URL">https://doi.org/10.1007/s11095-006-9001-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11095-006-9001-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11095-006-9001-x%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Research%26atitle%3DLigand-Targeted%252BDelivery%252Bof%252BTherapeutic%252BsiRNA%26aulast%3DIkeda%26aufirst%3DYutaka%26date%3D2006%26date%3D2006%26volume%3D23%26issue%3D8%26spage%3D1631%26epage%3D1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atsushi  Inoue</span>, <span class="hlFld-ContribAuthor ">Shinya Y.  Sawata</span>, <span class="hlFld-ContribAuthor ">Kazunari  Taira</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design and delivery of siRNA. </span><span class="cited-content_cbyCitation_journal-name">Journal of Drug Targeting</span><span> <strong>2006,</strong> <em>14 </em>
                                    (7)
                                     , 448-455. <a href="https://doi.org/10.1080/10611860600845397" title="DOI URL">https://doi.org/10.1080/10611860600845397</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10611860600845397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10611860600845397%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Drug%2520Targeting%26atitle%3DMolecular%252Bdesign%252Band%252Bdelivery%252Bof%252BsiRNA%26aulast%3DInoue%26aufirst%3DAtsushi%26date%3D2006%26date%3D2008%26volume%3D14%26issue%3D7%26spage%3D448%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satoshi  Fujita</span>, <span class="hlFld-ContribAuthor ">Takashi  Taki</span>, <span class="hlFld-ContribAuthor ">Kazunari  Taira</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of an Active Enzyme by Phage Display on the Basis of the Enzyme's Catalytic Activity in vivo. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2005,</strong> <em>6 </em>
                                    (2)
                                     , 315-321. <a href="https://doi.org/10.1002/cbic.200400215" title="DOI URL">https://doi.org/10.1002/cbic.200400215</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.200400215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.200400215%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DSelection%252Bof%252Ban%252BActive%252BEnzyme%252Bby%252BPhage%252BDisplay%252Bon%252Bthe%252BBasis%252Bof%252Bthe%252BEnzyme%252527s%252BCatalytic%252BActivity%252Bin%252Bvivo%26aulast%3DFujita%26aufirst%3DSatoshi%26date%3D2005%26date%3D2005%26volume%3D6%26issue%3D2%26spage%3D315%26epage%3D321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atsushi  Inoue</span>, <span class="hlFld-ContribAuthor ">Shinya Y.  Sawata</span>, <span class="hlFld-ContribAuthor ">Satoshi  Fujita</span>, <span class="hlFld-ContribAuthor ">Kazunari  Taira</span>. </span><span class="cited-content_cbyCitation_article-title">Additional RNA–Protein Interactions Facilitate in vitro Selection by Ribosome Display. </span><span class="cited-content_cbyCitation_journal-name">Chemistry Letters</span><span> <strong>2005,</strong> <em>34 </em>
                                    (1)
                                     , 26-27. <a href="https://doi.org/10.1246/cl.2005.26" title="DOI URL">https://doi.org/10.1246/cl.2005.26</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1246/cl.2005.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1246%2Fcl.2005.26%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520Letters%26atitle%3DAdditional%252BRNA%2525E2%252580%252593Protein%252BInteractions%252BFacilitate%252Bin%252Bvitro%252BSelection%252Bby%252BRibosome%252BDisplay%26aulast%3DInoue%26aufirst%3DAtsushi%26date%3D2005%26volume%3D34%26issue%3D1%26spage%3D26%26epage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Fermér</span>, <span class="hlFld-ContribAuthor ">Peter  Nilsson</span>, <span class="hlFld-ContribAuthor ">Mats  Larhed</span>. </span><span class="cited-content_cbyCitation_article-title">Microwave-assisted high-speed PCR. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2003,</strong> <em>18 </em>
                                    (2)
                                     , 129-132. <a href="https://doi.org/10.1016/S0928-0987(02)00252-X" title="DOI URL">https://doi.org/10.1016/S0928-0987(02)00252-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S0928-0987(02)00252-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS0928-0987%2802%2900252-X%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DMicrowave-assisted%252Bhigh-speed%252BPCR%26aulast%3DFerm%25C3%25A9r%26aufirst%3DChristian%26date%3D2003%26volume%3D18%26issue%3D2%26spage%3D129%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon  Delagrave</span>, <span class="hlFld-ContribAuthor ">Dennis J.  Murphy</span>. </span><span class="cited-content_cbyCitation_article-title">In Vitro
              Evolution of Proteins for Drug Development. </span><span class="cited-content_cbyCitation_journal-name">ASSAY and Drug Development Technologies</span><span> <strong>2003,</strong> <em>1 </em>
                                    (supplement 2)
                                     , 187-198. <a href="https://doi.org/10.1089/154065803321537917" title="DOI URL">https://doi.org/10.1089/154065803321537917</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/154065803321537917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2F154065803321537917%26sid%3Dliteratum%253Aachs%26jtitle%3DASSAY%2520and%2520Drug%2520Development%2520Technologies%26atitle%3DIn%252BVitro%252BEvolution%252Bof%252BProteins%252Bfor%252BDrug%252BDevelopment%26aulast%3DDelagrave%26aufirst%3DSimon%26date%3D2003%26volume%3D1%26issue%3Dsupplement%25202%26spage%3D187%26epage%3D198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Baskerville</span>, <span class="hlFld-ContribAuthor ">D. P.  Bartel</span>. </span><span class="cited-content_cbyCitation_article-title">A ribozyme that ligates RNA to protein. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2002,</strong> <em>99 </em>
                                    (14)
                                     , 9154-9159. <a href="https://doi.org/10.1073/pnas.142153799" title="DOI URL">https://doi.org/10.1073/pnas.142153799</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.142153799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.142153799%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DA%252Bribozyme%252Bthat%252Bligates%252BRNA%252Bto%252Bprotein%26aulast%3DBaskerville%26aufirst%3DS.%26date%3D2002%26date%3D2002%26volume%3D99%26issue%3D14%26spage%3D9154%26epage%3D9159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Gel after SDS−PAGE showing purified DHFR−(RE)<sub>1</sub><sub>-</sub><sub>3</sub> fusion proteins and wild-type (wt) DHFR. Purified recombinant proteins (∼100 μg) were subjected to SDS−PAGE and visualized by straining with Coomassie blue. Lane 1, markers; lane 2, wild-type DHFR; lane 3, DHFR−(RE)<sub>1</sub>; lane 4, DHFR−(RE)<sub>2</sub>; lane 5, DHFR−(RE)<sub>3</sub>. Molecular masses of marker proteins are shown on the left. Polyglycine linkers were inserted between DHFR and the first RE peptide and also between the RE and the adjacent RE peptides to maintain flexibility. To facilitate purification, polyhistidines (His) were connected at the N terminal of DHFR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00002.jpeg" id="rightTab-_i7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Measurements of dissociation constants (<i>K</i><sub>d</sub>). In each lane C, only the aptamer was loaded. The black arrowheads indicate protein−RNA complexes, and white arrowheads indicate free RNA. Relative amounts of complex and free RNA were calculated from the radioactivity of <sup>32</sup>P. Equilibrium dissociation constants were determined from theoretical curves of the concentration of free RNA or complex RNA versus the concentration of the peptide by the least-squares methods. Diamonds indicate relative amounts of free RNA, and squares indicate relative amounts of complex. In each panel, the elongated black triangle indicates increasing concentrations of protein. (A) Absence of any interactions between (Apt)<sub>1</sub><sub>-</sub><sub>3</sub> and wild-type DHFR. Each lane C was loaded with 1 nM (Apt)<sub>1</sub><sub>-</sub><sub>3</sub> and 20, 100, and 500 nM wild-type DHFR were added as indicated by the elongated triangles. (B) Binding of (Apt)<sub>1</sub> with DHFR−(RE)<sub>1</sub>. A total of 30 pM (Apt)<sub>1</sub> (lane C) was combined with 0.25, 0.5, 1, 2, 4, 8, 16, and 32 nM DHFR−(RE)<sub>1</sub> as indicated by the elongated triangles. (C, D) Binding of (Apt)<sub>2</sub><sub>-</sub><sub>3</sub> with DHFR−(RE)<sub>2</sub><sub>-</sub><sub>3</sub> proteins. A total of 30 pM (Apt)<sub>2</sub> or (Apt)<sub>3</sub> RNA (lane C) was combined with 0.03, 0.06, 0.12, 0.25, 0.5, 1, 2, 4, 8, and 16 nM protein as indicated by the elongated triangles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00003.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Model of the selection of a functional protein in vitro using interaction between (Apt)<sub>3</sub> and (RE)<sub>3</sub>. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of SDS−PAGE showing the enrichment for DHFR−(RE)<sub>3</sub> protein and mRNA (intact) on MTX beads. Each sample was then subjected to SDS−PAGE on a 15% gel. Lanes 1−3, 1 μL of lysate was loaded after the respective translation reactions. Lanes 4−6, 10 μL of eluate containing mRNA and protein that had been bound to MTX beads. Lanes 1 and 4 correspond to the intact protein, lanes 2 and 5 correspond to the mutant protein, and lanes 3 and 6 were loaded with control samples. (C) Quantification of proteins and mRNAs shown in (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00004.jpeg" id="rightTab-_i11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Selection of mRNA-encoding streptavidin in vitro from a pool of mixtures of streptavidin and DHFR mRNAs. (A) Schematic representation of interactions among the transcribed RNA, its translated protein, and biotin-conjugated agarose beads (biotin ligand). (B) Enrichment for streptavidin mRNA by binding to biotin−agarose, compared to DHFR mRNA. Only the streptavidin-containing complex was bound specifically to biotin−agarose. (C) Analysis by 3% agarose gel electrophoresis of the results of amplification by RT-PCR. PCR reactions were performed for 15 cycles. Each sample was subjected to 3% Sea Kem GTG agarose gel (FMC, Riceland, ME). Markers, size markers; lanes 1−3, products of mRNAs before selection (without selection) as controls; lanes 4−6, products of mRNAs selected with biotin−agarose. Lanes 1 and 4, streptavidin mRNA; lanes 2 and 5, DHFR mRNA; lanes 3 and 6, a mixture of streptavidin and DHFR mRNAs at a ratio of 1:1 (mole/mole).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00005.jpeg" id="rightTab-_i13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Comparison of the strong interaction between (RE)<sub>3</sub>−(Apt)<sub>3</sub> and the ribosome-display system in the selection of functional DHFR in rabbit reticulocyte lysate. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of electrophoresis of RT-PCR products in 2% agarose showing the enrichment for DHFR−(RE)<sub>3</sub>−(Apt)<sub>3</sub> + Stop, DHFR − Stop, and DHFR + Stop mRNA on MTX beads. PCR reactions were performed for 20 cycles. Each sample was subjected to 2% Sea Kem GTG agarose gel (FMC). Analysis of the PCR products was carried out once, and thus, quantification at the level of mRNAs shown in Figure 5C is more reliable. (C) Quantification of enriched mRNAs. The relative amounts of selected mRNAs were detected by the count of <sup>32</sup>P-labeled mRNA. Respective numbers of lanes correspond to that of lanes in Figure 5B. The results were the averages from four independent sets of experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00006.jpeg" id="rightTab-_i15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Improvement in stabilization of the complex between the respective mRNA and its product by combination of the (RE)<sub>3</sub>−(Apt)<sub>3</sub> interaction and the ribosome-display system in <i>E. coli</i> S30 extracts. (A) Schematic representation of interactions among transcribed RNA, translated protein, and methotrexate-conjugated agarose beads (MTX ligand). (B) Results of electrophoresis of RT-PCR products in 2% agarose gel showing the PCR products for DHFR − Stop mRNA (parts 1 and 2) and (Apt)<sub>3</sub>−(RE)<sub>3</sub>−DHFR − Stop mRNA (parts 3 and 4) in the presence (parts 1 and 3) or in the absence (parts 2 and 4) of the polyclonal IgG antibodies against the releasing factors (<i>E. coli</i> RF1, RF2, and RF3), enriched on the MTX−agarose beads. Selected mRNAs were subjected to MTX−agarose beads under the same conditions as described in Figure 5. (C) Quantification of the RT-PCR products shown in (B). The results were the averages from two independent sets of experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm010398pf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/medium/jm010398pf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00007.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 (A) Possible RNA−protein networks. Potentially, tandemly connected three RE peptides could interact with tandemly connected three aptamers not only in cis but also in trans configurations. Trans interactions might create RNA−protein networks. The trans interactions should be minimized by the combination of different units of peptides and aptamers. (B) A schematic diagram of the proposed method for selection of a functional protein in vitro. Binding proteins are generated and concentrated by multiple cycles of transcription, translation, affinity selection, and PCR. In the translation step, the tandem aptamer (Apt)<sub>3</sub> binds to the tandem RE peptide (RE)<sub>3</sub> with very high affinity, as demonstrated in the present study. The binding couples the protein and its mRNA. In this study, we used DHFR and streptavidin as the target in the selection step, but any protein can be used in combination with the tandem aptamer (Apt)<sub>3</sub> and the tandem RE peptide (RE)<sub>3</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2002/jmcmar.2002.45.issue-8/jm010398p/production/images/large/jm010398pf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm010398p&amp;id=jm010398pf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm010398pb00001"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00001_1"><span><span class="NLM_string-ref">Fields, S.; Song, O. A novel genetic system to detect protein−protein interactions. <i>  Nature</i><b>1989</b>, <i>340</i>, 245−246.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=1989&pages=246&author=S.+Fields&author=O+Song&title=A+novel+genetic+system+to+detect+protein%E2%88%92protein+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFields%26aufirst%3DS.%26atitle%3DA%2520novel%2520genetic%2520system%2520to%2520detect%2520protein%25E2%2588%2592protein%2520interactions%26jtitle%3DNature%26date%3D1989%26volume%3D340%26spage%3D246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00002"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00002_1"><span><span class="NLM_string-ref">Harada, K.; Martin, S. S.; Frankel, A. D. Selection of RNA-binding peptides in vivo. <i>  Nature</i><b>1996</b>, <i>380</i>, 175−179.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=179&author=K.+Harada&author=S.+S.+Martin&author=A.+D+Frankel&title=Selection+of+RNA-binding+peptides+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DK.%26atitle%3DSelection%2520of%2520RNA-binding%2520peptides%2520in%2520vivo%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00003"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00003_1"><span><span class="NLM_string-ref">Moore, J. C.; Arnold, F. H. Directed evolution of a para-nitrobenzyl esterase for aqueous−organic solvents. <i>  Nat. Biotechnol.</i><b>1996</b>, <i>14</i>, 458−467.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=467&author=J.+C.+Moore&author=F.+H+Arnold&title=Directed+evolution+of+a+para-nitrobenzyl+esterase+for+aqueous%E2%88%92organic+solvents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BC.%26atitle%3DDirected%2520evolution%2520of%2520a%2520para-nitrobenzyl%2520esterase%2520for%2520aqueous%25E2%2588%2592organic%2520solvents%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00004"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00004_1"><span><span class="NLM_string-ref">Schatz, P. J.; Cull, M. G.; Martin, E. L.; Gates, C. M. Screening of peptide libraries linked to lac repressor. <i>  Methods Enzymol.</i><b>1996</b>, <i>267</i>, 171−191.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1996&pages=191&author=P.+J.+Schatz&author=M.+G.+Cull&author=E.+L.+Martin&author=C.+M+Gates&title=Screening+of+peptide+libraries+linked+to+lac+repressor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchatz%26aufirst%3DP.%2BJ.%26atitle%3DScreening%2520of%2520peptide%2520libraries%2520linked%2520to%2520lac%2520repressor%26jtitle%3DMethods%2520Enzymol.%26date%3D1996%26volume%3D267%26spage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00005"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00005_1"><span><span class="NLM_string-ref">Boder, E. T.; Wittrup, K. D. Yeast surface display for screening combinatorial polypeptide libraries. <i>  Nat. Biotechnol.</i><b>1997</b>, <i>15</i>, 553−557.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=557&author=E.+T.+Boder&author=K.+D+Wittrup&title=Yeast+surface+display+for+screening+combinatorial+polypeptide+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoder%26aufirst%3DE.%2BT.%26atitle%3DYeast%2520surface%2520display%2520for%2520screening%2520combinatorial%2520polypeptide%2520libraries%26jtitle%3DNat.%2520Biotechnol.%26date%3D1997%26volume%3D15%26spage%3D557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00006"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00006_1"><span><span class="NLM_string-ref">Smith, G. P.; Petrenko, V. A. Phage display. <i>  Chem. Rev.</i><b>1997</b>, <i>97</i>, 391−410.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=410&author=G.+P.+Smith&author=V.+A+Petrenko&title=Phage+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DG.%2BP.%26atitle%3DPhage%2520display%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00007"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00007_1"><span><span class="NLM_string-ref">Mattheakis, L. C.; Bhatt, R. R.; Dower, W. J. An in vitro polysome display system for identifying ligands from very large peptide libraries. <i>  Proc. Natl. </i><i>Acad. Sci. U.S.A.</i><b>1994</b>, <i>91</i>, 9022−9026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=9026&author=L.+C.+Mattheakis&author=R.+R.+Bhatt&author=W.+J+Dower&title=An+in+vitro+polysome+display+system+for+identifying+ligands+from+very+large+peptide+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMattheakis%26aufirst%3DL.%2BC.%26atitle%3DAn%2520in%2520vitro%2520polysome%2520display%2520system%2520for%2520identifying%2520ligands%2520from%2520very%2520large%2520peptide%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1994%26volume%3D91%26spage%3D9026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00008"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00008_1"><span><span class="NLM_string-ref">Mattheakis, L. C.; Dias, J. M.; Dower, W. J. Cell-free synthesis of peptide libraries displayed on polysomes. <i>  Methods Enzymol.</i><b>1996</b>, <i>267</i>, 195−207.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1996&pages=207&author=L.+C.+Mattheakis&author=J.+M.+Dias&author=W.+J+Dower&title=Cell-free+synthesis+of+peptide+libraries+displayed+on+polysomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMattheakis%26aufirst%3DL.%2BC.%26atitle%3DCell-free%2520synthesis%2520of%2520peptide%2520libraries%2520displayed%2520on%2520polysomes%26jtitle%3DMethods%2520Enzymol.%26date%3D1996%26volume%3D267%26spage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00009"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00009_1"><span><span class="NLM_string-ref">Gersuk, G. M.; Corey, M. J.; Corey, E.; Stray, J. E.; Kawasaki, G. H.; Vessella, R. L. High-affinity peptide ligands to prostate-specific antigen identified by polysome selection. <i>  Biochem. Biophys. Res. Commun. </i><b>1997</b>, <i>232</i>, 578−582.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=232&publication_year=1997&pages=582&author=G.+M.+Gersuk&author=M.+J.+Corey&author=E.+Corey&author=J.+E.+Stray&author=G.+H.+Kawasaki&author=R.+L+Vessella&title=High-affinity+peptide+ligands+to+prostate-specific+antigen+identified+by+polysome+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGersuk%26aufirst%3DG.%2BM.%26atitle%3DHigh-affinity%2520peptide%2520ligands%2520to%2520prostate-specific%2520antigen%2520identified%2520by%2520polysome%2520selection%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D232%26spage%3D582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm010398pb00010_1"><span><span class="NLM_string-ref">Hanes, J.; Plückthun, A. In vitro selection and evolution of functional proteins by using ribosome display. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 4937−4942.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=4942&author=J.+Hanes&author=A+Pl%C3%BCckthun&title=In+vitro+selection+and+evolution+of+functional+proteins+by+using+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520selection%2520and%2520evolution%2520of%2520functional%2520proteins%2520by%2520using%2520ribosome%2520display%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D4942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00011"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00011_1"><span><span class="NLM_string-ref">He, M.; Taussig, M. J. Antibody−ribosome−mRNA (ARM) complexes as efficient selection particles for in vitro display and evolution of antibody combining sites. <i>  Nucleic Acids Res.</i><b>1997</b>, <i>25</i>, 5132−5134.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=5134&author=M.+He&author=M.+J.+Taussig&author=+Antibody%E2%88%92ribosome%E2%88%92m&title=complexes+as+efficient+selection+particles+for+in+vitro+display+and+evolution+of+antibody+combining+sites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26atitle%3Dcomplexes%2520as%2520efficient%2520selection%2520particles%2520for%2520in%2520vitro%2520display%2520and%2520evolution%2520of%2520antibody%2520combining%2520sites%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D5134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00012"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00012_1"><span><span class="NLM_string-ref">Nemoto, N.; Miyamoto-Sato, E.; Husimi, Y.; Yanagawa, H. In vitro virus:  bonding of mRNA bearing puromycin at the 3‘-terminal end to the C-terminal end of its encoded protein on the ribosome in vitro. <i>  FEBS Lett.</i><b>1997</b>, <i>414</i>, 405−408.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=1997&pages=408&author=N.+Nemoto&author=E.+Miyamoto-Sato&author=Y.+Husimi&author=H+Yanagawa&title=In+vitro+virus%3A%E2%80%89+bonding+of+mRNA+bearing+puromycin+at+the+3%E2%80%98-terminal+end+to+the+C-terminal+end+of+its+encoded+protein+on+the+ribosome+in+vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNemoto%26aufirst%3DN.%26atitle%3DIn%2520vitro%2520virus%253A%25E2%2580%2589%2520bonding%2520of%2520mRNA%2520bearing%2520puromycin%2520at%2520the%25203%25E2%2580%2598-terminal%2520end%2520to%2520the%2520C-terminal%2520end%2520of%2520its%2520encoded%2520protein%2520on%2520the%2520ribosome%2520in%2520vitro%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D414%26spage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00013"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00013_1"><span><span class="NLM_string-ref">Roberts, R. W.; Szostak, J. W. RNA−peptide fusions for the in vitro selection of peptides and proteins. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 12297−12302.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=12302&author=R.+W.+Roberts&author=J.+W+Szostak&title=RNA%E2%88%92peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DR.%2BW.%26atitle%3DRNA%25E2%2588%2592peptide%2520fusions%2520for%2520the%2520in%2520vitro%2520selection%2520of%2520peptides%2520and%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D12302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00014"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00014_1"><span><span class="NLM_string-ref">Hanes, J.; Jermutus, L.; Weber-Bornhauser, S.; Bosshard, H. R.; Plückthun, A. Ribosome display efficiently selects and evolves high-affinity antibodies in vitro from immune libraries. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1998</b>, <i>95</i>, 14130−14135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=14135&author=J.+Hanes&author=L.+Jermutus&author=S.+Weber-Bornhauser&author=H.+R.+Bosshard&author=A+Pl%C3%BCckthun&title=Ribosome+display+efficiently+selects+and+evolves+high-affinity+antibodies+in+vitro+from+immune+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DRibosome%2520display%2520efficiently%2520selects%2520and%2520evolves%2520high-affinity%2520antibodies%2520in%2520vitro%2520from%2520immune%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D14135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00015"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00015_1"><span><span class="NLM_string-ref">Tawfik, D. S.; Griffiths, A. D. Man-made cell-like compartments for molecular evolution. <i>  Nat. Biotechnol.</i><b>1998</b>, <i>16</i>, 652−656.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1998&pages=656&author=D.+S.+Tawfik&author=A.+D+Griffiths&title=Man-made+cell-like+compartments+for+molecular+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTawfik%26aufirst%3DD.%2BS.%26atitle%3DMan-made%2520cell-like%2520compartments%2520for%2520molecular%2520evolution%26jtitle%3DNat.%2520Biotechnol.%26date%3D1998%26volume%3D16%26spage%3D656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00016"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00016_1"><span><span class="NLM_string-ref">Doi, N.; Yanagawa, H. STABLE:  protein−DNA fusion system for screening of combinatorial protein libraries in vitro. <i>  FEBS Lett.</i><b>1999</b>, <i>457</i>, 227−230.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=1999&pages=230&author=N.+Doi&author=H.+Yanagawa&title=protein%E2%88%92DNA+fusion+system+for+screening+of+combinatorial+protein+libraries+in+vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDoi%26aufirst%3DN.%26atitle%3Dprotein%25E2%2588%2592DNA%2520fusion%2520system%2520for%2520screening%2520of%2520combinatorial%2520protein%2520libraries%2520in%2520vitro%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D457%26spage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00017"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00017_1"><span><span class="NLM_string-ref">Hanes, J.; Jermutus, L.; Schaffitzel, C.; Plückthun, A. Comparison of <i>Escherichia coli</i> and rabbit reticulocyte ribosome display systems. <i>  FEBS </i><i>Lett.</i><b>1999</b>, <i>450</i>, 105−110.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=1999&pages=110&author=J.+Hanes&author=L.+Jermutus&author=C.+Schaffitzel&author=A+Pl%C3%BCckthun&title=Comparison+of+Escherichia+coli+and+rabbit+reticulocyte+ribosome+display+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DComparison%2520of%2520Escherichia%2520coli%2520and%2520rabbit%2520reticulocyte%2520ribosome%2520display%2520systems%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D450%26spage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00018"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00018_1"><span><span class="NLM_string-ref">Makeyev, E. V.; Kolb, V. A.; Spirin, A. S. Cell-free immunology:  construction and in vitro expression of a PCR-based library encoding a single-chain antibody repertoire. <i>  FEBS Lett.</i><b>1999</b>, <i>444</i>, 177−180.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=1999&pages=180&author=E.+V.+Makeyev&author=V.+A.+Kolb&author=A.+S.+Spirin&author=+Cell&title=construction+and+in+vitro+expression+of+a+PCR-based+library+encoding+a+single-chain+antibody+repertoire"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMakeyev%26aufirst%3DE.%2BV.%26atitle%3Dconstruction%2520and%2520in%2520vitro%2520expression%2520of%2520a%2520PCR-based%2520library%2520encoding%2520a%2520single-chain%2520antibody%2520repertoire%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D444%26spage%3D180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00019"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00019_1"><span><span class="NLM_string-ref">He, M.; Menges, M.; Groves, M. A. T.; Corps, E.; Liu, H.; Brüggemann, M.; Taussig, M. J. Selection of a human anti-progesterone antibody fragment from a transgenic mouse library by ARM ribosome display. <i>  J. Immunol. </i><i>Methods</i><b>1999</b>, <i>231</i>, 105−117.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=1999&pages=117&author=M.+He&author=M.+Menges&author=M.+A.+T.+Grovesauthor=H.+Liu&author=M.+Br%C3%BCggemann&author=M.+J+Taussig&title=Selection+of+a+human+anti-progesterone+antibody+fragment+from+a+transgenic+mouse+library+by+ARM+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3D%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DSelection%2520of%2520a%2520human%2520anti-progesterone%2520antibody%2520fragment%2520from%2520a%2520transgenic%2520mouse%2520library%2520by%2520ARM%2520ribosome%2520display%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1999%26volume%3D231%26spage%3D117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm010398pb00020_1"><span><span class="NLM_string-ref">Schaffitzel, C.; Hanes, J.; Jermutus, L.; Plückthun, A. Ribosome display:  an in vitro method for selection and evolution of antibodies from libraries. <i>  J. </i><i>Immunol. Methods</i><b>1999</b>, <i>231</i>, 119−135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=1999&pages=135&author=C.+Schaffitzel&author=J.+Hanes&author=L.+Jermutus&author=A.+Pl%C3%BCckthun&author=+Ribosome&title=an+in+vitro+method+for+selection+and+evolution+of+antibodies+from+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffitzel%26aufirst%3DC.%26atitle%3Dan%2520in%2520vitro%2520method%2520for%2520selection%2520and%2520evolution%2520of%2520antibodies%2520from%2520libraries%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1999%26volume%3D231%26spage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00021"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00021_1"><span><span class="NLM_string-ref">Bieberich, E.; Kapitonov, D.; Tencomnao, T.; Yu, R. K. Protein−ribosome−mRNA display:  affinity isolation of enzyme−ribosome−mRNA complexes and cDNA cloning in a single-tube reaction. <i>  Anal. Biochem.</i><b>2000</b>, <i>287</i>, 294−298.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=298&author=E.+Bieberich&author=D.+Kapitonov&author=T.+Tencomnao&author=R.+K.+Yu&author=+Protein%E2%88%92ribosome%E2%88%92m&title=affinity+isolation+of+enzyme%E2%88%92ribosome%E2%88%92mRNA+complexes+and+cDNA+cloning+in+a+single-tube+reaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBieberich%26aufirst%3DE.%26atitle%3Daffinity%2520isolation%2520of%2520enzyme%25E2%2588%2592ribosome%25E2%2588%2592mRNA%2520complexes%2520and%2520cDNA%2520cloning%2520in%2520a%2520single-tube%2520reaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D287%26spage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00022"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00022_1"><span><span class="NLM_string-ref">Cho, G.; Keefe, A. D.; Liu, R.; Wilson, D. S.; Szostak, J. W. Constructing high complexity synthetic libraries of long ORFs using in vitro selection. <i>  J. Mol. </i><i>Biol.</i><b>2000</b>, <i>297</i>, 309−319.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2000&pages=319&author=G.+Cho&author=A.+D.+Keefe&author=R.+Liu&author=D.+S.+Wilson&author=J.+W+Szostak&title=Constructing+high+complexity+synthetic+libraries+of+long+ORFs+using+in+vitro+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DG.%26atitle%3DConstructing%2520high%2520complexity%2520synthetic%2520libraries%2520of%2520long%2520ORFs%2520using%2520in%2520vitro%2520selection%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D297%26spage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00023"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00023_1"><span><span class="NLM_string-ref">Liu, R.; Barrick, J. E.; Szostak, J. W.; Roberts, R. W. Optimized synthesis of RNA−protein fusions for in vitro protein selection. <i>  Methods Enzymol.</i><b>2000</b>, <i>318</i>, 268−293.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2000&pages=293&author=R.+Liu&author=J.+E.+Barrick&author=J.+W.+Szostak&author=R.+W+Roberts&title=Optimized+synthesis+of+RNA%E2%88%92protein+fusions+for+in+vitro+protein+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DR.%26atitle%3DOptimized%2520synthesis%2520of%2520RNA%25E2%2588%2592protein%2520fusions%2520for%2520in%2520vitro%2520protein%2520selection%26jtitle%3DMethods%2520Enzymol.%26date%3D2000%26volume%3D318%26spage%3D293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00024"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00024_1"><span><span class="NLM_string-ref">Hanes, J.; Schaffitzel, C.; Knappik, A.; Plückthun, A. Picomolar affinity antibodies from a fully synthetic naive library selected and evolved by ribosome display. <i>  Nat. Biotechnol.</i><b>2000</b>, <i>18</i>, 1287−1292.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=1292&author=J.+Hanes&author=C.+Schaffitzel&author=A.+Knappik&author=A+Pl%C3%BCckthun&title=Picomolar+affinity+antibodies+from+a+fully+synthetic+naive+library+selected+and+evolved+by+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanes%26aufirst%3DJ.%26atitle%3DPicomolar%2520affinity%2520antibodies%2520from%2520a%2520fully%2520synthetic%2520naive%2520library%2520selected%2520and%2520evolved%2520by%2520ribosome%2520display%26jtitle%3DNat.%2520Biotechnol.%26date%3D2000%26volume%3D18%26spage%3D1292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00025"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00025_1"><span><span class="NLM_string-ref">Jermutus, L.; Honegger, A.; Schwesinger, F.; Hanes, J.; Plückthun, A. Tailoring in vitro evolution for protein affinity or stability. <i>  Proc. Natl. Acad. Sci. </i><i>U.S.A.</i><b>2001</b>, <i>98</i>, 75−80.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=80&author=L.+Jermutus&author=A.+Honegger&author=F.+Schwesinger&author=J.+Hanes&author=A+Pl%C3%BCckthun&title=Tailoring+in+vitro+evolution+for+protein+affinity+or+stability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJermutus%26aufirst%3DL.%26atitle%3DTailoring%2520in%2520vitro%2520evolution%2520for%2520protein%2520affinity%2520or%2520stability%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00026"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00026_1"><span><span class="NLM_string-ref">Keefe, A. D.; Szostak, J. W. Functional proteins from a random-sequence library. <i>  Nature</i><b>2001</b>, <i>410</i>, 715−718.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2001&pages=718&author=A.+D.+Keefe&author=J.+W+Szostak&title=Functional+proteins+from+a+random-sequence+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DFunctional%2520proteins%2520from%2520a%2520random-sequence%2520library%26jtitle%3DNature%26date%3D2001%26volume%3D410%26spage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00027"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00027_1"><span><span class="NLM_string-ref">Wilson, D. S.; Keefe, A. D.; Szostak, J. W. The use of mRNA display to select high-affinity protein-binding peptides. <i>  Proc. Natl. Acad. Sci. U.S.A</i>. <b>2001</b>, <i>98</i>, 3750−3755.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3755&author=D.+S.+Wilson&author=A.+D.+Keefe&author=J.+W+Szostak&title=The+use+of+mRNA+display+to+select+high-affinity+protein-binding+peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BS.%26atitle%3DThe%2520use%2520of%2520mRNA%2520display%2520to%2520select%2520high-affinity%2520protein-binding%2520peptides%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00028"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00028_1"><span><span class="NLM_string-ref">Lamla, T.; Erdmann, V. A. In vitro selection of other proteins than antibodies by means of ribosome display. <i>  FEBS Lett.</i><b>2001</b>, <i>502</i>, 35−40.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2001&pages=40&author=T.+Lamla&author=V.+A+Erdmann&title=In+vitro+selection+of+other+proteins+than+antibodies+by+means+of+ribosome+display"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamla%26aufirst%3DT.%26atitle%3DIn%2520vitro%2520selection%2520of%2520other%2520proteins%2520than%2520antibodies%2520by%2520means%2520of%2520ribosome%2520display%26jtitle%3DFEBS%2520Lett.%26date%3D2001%26volume%3D502%26spage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00029"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00029_1"><span><span class="NLM_string-ref">Yamamoto, R.; Katahira, M.; Nishikawa, S.; Taira, K.; Kumar, P. K. R. A novel RNA motif that binds efficiently and specifically to the Tat protein of HIV and inhibits the trans-activation by Tat of transcription in vitro and in vivo. <i>  Genes Cells</i><b>2000</b>, <i>5</i>, 371−388.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=388&author=R.+Yamamoto&author=M.+Katahira&author=S.+Nishikawa&author=K.+Taira&author=P.+K.+R+Kumar&title=A+novel+RNA+motif+that+binds+efficiently+and+specifically+to+the+Tat+protein+of+HIV+and+inhibits+the+trans-activation+by+Tat+of+transcription+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DR.%26atitle%3DA%2520novel%2520RNA%2520motif%2520that%2520binds%2520efficiently%2520and%2520specifically%2520to%2520the%2520Tat%2520protein%2520of%2520HIV%2520and%2520inhibits%2520the%2520trans-activation%2520by%2520Tat%2520of%2520transcription%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DGenes%2520Cells%26date%3D2000%26volume%3D5%26spage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm010398pb00030_1"><span><span class="NLM_string-ref">Jencks, W. P. On the attribution and additivity of binding energies. <i>  Proc. Natl. </i><i>Acad. Sci. U.S.A.</i><b>1981</b>, <i>78</i>, 4046−4050.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4050&author=W.+P+Jencks&title=On+the+attribution+and+additivity+of+binding+energies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1981%26volume%3D78%26spage%3D4050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00031"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00031_1"><span><span class="NLM_string-ref">Shuker, S. B.; Hajduk, P. J.; Meadows, R. P.; Fesik, S. W. Discovering high-affinity ligands for proteins:  SAR by NMR. <i>  Science</i><b>1996</b>, <i>274</i>, 1531−1534.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1534&author=S.+B.+Shuker&author=P.+J.+Hajduk&author=R.+P.+Meadows&author=S.+W+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A%E2%80%89+SAR+by+NMR"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%25E2%2580%2589%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00032"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00032_1"><span><span class="NLM_string-ref">Blakley, R. L. In <i>Folates and Pterins</i>; Blakley, R. L., Benkovic, S. J., Eds.; Wiley:  New York, 1985; pp 191−253.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1985&pages=253&author=R.+Blakley&title=Folates+and+Pterins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBlakley%26aufirst%3DR.%26atitle%3DL%26btitle%3DFolates%2520and%2520Pterins%26pub%3DWiley%26date%3D1985%26spage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00033"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00033_1"><span><span class="NLM_string-ref">Taira, K.; Fierke, C. A.; Chen, J.-T.; Johnson, K. A.; Benkovic, S. J. On interpreting the inhibition of and catalysis by dihydrofolate-reductase. <i>  Trends Biochem. Sci.</i><b>1987</b>, <i>12</i>, 275−278.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1987&pages=278&author=K.+Taira&author=C.+A.+Fierke&author=J.-T.+Chen&author=K.+A.+Johnson&author=S.+J+Benkovic&title=On+interpreting+the+inhibition+of+and+catalysis+by+dihydrofolate-reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaira%26aufirst%3DK.%26atitle%3DOn%2520interpreting%2520the%2520inhibition%2520of%2520and%2520catalysis%2520by%2520dihydrofolate-reductase%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1987%26volume%3D12%26spage%3D278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00034"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00034_1"><span><span class="NLM_string-ref">Taira, K.; Benkovic, S. J. Evaluation of the importance of hydrophobic interactions in drug binding to dihydrofolate reductase. <i>  J. Med. Chem.</i><b>1988</b>, <i>31</i>, 129−137.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=137&author=K.+Taira&author=S.+J+Benkovic&title=Evaluation+of+the+importance+of+hydrophobic+interactions+in+drug+binding+to+dihydrofolate+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaira%26aufirst%3DK.%26atitle%3DEvaluation%2520of%2520the%2520importance%2520of%2520hydrophobic%2520interactions%2520in%2520drug%2520binding%2520to%2520dihydrofolate%2520reductase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00035"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00035_1"><span><span class="NLM_string-ref">Iwakura, M.; Jones, B. E.; Luo, J.; Matthews, C. R. A strategy for testing the suitability of cysteine replacements in dihydrofolate reductase from <i>Escherichia coli</i>. <i>  J. Biochem. (Tokyo)</i><b>1995</b>, <i>117</i>, 480−488.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=488&author=M.+Iwakura&author=B.+E.+Jones&author=J.+Luo&author=C.+R+Matthews&title=A+strategy+for+testing+the+suitability+of+cysteine+replacements+in+dihydrofolate+reductase+from+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIwakura%26aufirst%3DM.%26atitle%3DA%2520strategy%2520for%2520testing%2520the%2520suitability%2520of%2520cysteine%2520replacements%2520in%2520dihydrofolate%2520reductase%2520from%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biochem.%2520%2528Tokyo%2529%26date%3D1995%26volume%3D117%26spage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00036"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00036_1"><span><span class="NLM_string-ref">Fujita, S.; Koguma, T.; Ohkawa, J.; Mori, K.; Kohda, T.; Kise, H.; Nishikawa, S.; Iwakura, M.; Taira, K. Discrimination of a single base change in a ribozyme using the gene for dihydrofolate reductase as a selective marker. in <i>Escherichia coli</i>. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1997</b>, <i>94</i>, 391−396.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=396&author=S.+Fujita&author=T.+Koguma&author=J.+Ohkawa&author=K.+Mori&author=T.+Kohda&author=H.+Kise&author=S.+Nishikawa&author=M.+Iwakura&author=K+Taira&title=Discrimination+of+a+single+base+change+in+a+ribozyme+using+the+gene+for+dihydrofolate+reductase+as+a+selective+marker.+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DS.%26atitle%3DDiscrimination%2520of%2520a%2520single%2520base%2520change%2520in%2520a%2520ribozyme%2520using%2520the%2520gene%2520for%2520dihydrofolate%2520reductase%2520as%2520a%2520selective%2520marker.%2520in%2520Escherichia%2520coli%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00037"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00037_1"><span><span class="NLM_string-ref">Hamada, M.; Fujita, S.; Kise, H.; Jigami, Y.; Taira, K. Comparison of in vivo activities of 5‘-connected and 3‘-connected cis-acting ribozymes:  selection of intracellularly active ribozymes using the gene for dihydrofolate reductase (DHFR) as a selective marker in <i>Escherichia coli</i>. <i>  J. Biochem. </i><i>(Tokyo)</i><b>1998</b>, <i>123</i>, 684−692.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=1998&pages=692&author=M.+Hamada&author=S.+Fujita&author=H.+Kise&author=Y.+Jigami&author=K+Taira&title=Comparison+of+in+vivo+activities+of+5%E2%80%98-connected+and+3%E2%80%98-connected+cis-acting+ribozymes%3A%E2%80%89+selection+of+intracellularly+active+ribozymes+using+the+gene+for+dihydrofolate+reductase+%28DHFR%29+as+a+selective+marker+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DM.%26atitle%3DComparison%2520of%2520in%2520vivo%2520activities%2520of%25205%25E2%2580%2598-connected%2520and%25203%25E2%2580%2598-connected%2520cis-acting%2520ribozymes%253A%25E2%2580%2589%2520selection%2520of%2520intracellularly%2520active%2520ribozymes%2520using%2520the%2520gene%2520for%2520dihydrofolate%2520reductase%2520%2528DHFR%2529%2520as%2520a%2520selective%2520marker%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biochem.%2520%2528Tokyo%2529%26date%3D1998%26volume%3D123%26spage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00038"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00038_1"><span><span class="NLM_string-ref">Green, N. M. Avidin. <i>  Adv. Protein Chem.</i><b>1975</b>, <i>29</i>, 85−133.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Green%2C+N.+M.+Avidin.+Adv.+Protein+Chem.+1975%2C+29%2C+85%E2%88%92133.+Green%2C+N.+M.+Avidin.+Adv.+Protein+Chem.1975%2C+29%2C+85%E2%88%92133."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DN.%2BM.%26jtitle%3DAdv.%2520Protein%2520Chem.%26date%3D1975%26volume%3D29%26spage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00039"><div class="reference"><strong class="refLabel"><a href="#jm010398pb00039" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-jm010398pb00039_1"><span><span class="NLM_string-ref">Moffat, J. G.; Timms, K. M.; Trotman, C. N. A.; Tate, W. P. Interaction of the release factors with the <i>Escherichia coli</i> ribosome:  structurally and functionally important domains.<i> Biochimie</i><b>1991</b>, <i>73</i>, 1113−1120.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=1991&pages=1120&author=J.+G.+Moffat&author=K.+M.+Timms&author=C.+N.+A.+Trotman&author=W.+P+Tate&title=Interaction+of+the+release+factors+with+the+Escherichia+coli+ribosome%3A%E2%80%89+structurally+and+functionally+important+domains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26atitle%3DInteraction%2520of%2520the%2520release%2520factors%2520with%2520the%2520Escherichia%2520coli%2520ribosome%253A%25E2%2580%2589%2520structurally%2520and%2520functionally%2520important%2520domains%26jtitle%3DBiochimie%26date%3D1991%26volume%3D73%26spage%3D1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm010398pb00040"><div class="reference"><div class="NLM_citation" id="rightTab-jm010398pb00040_1"><span><span class="NLM_string-ref">Zhou, J.-M.; Fujita, S.; Warashina, M.; Baba, T.; Taira, K. <i>  J. Am. Chem. Soc.</i><b>2002</b>, <i>124</i>, 538−543.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Zhou%2C+J.-M.%3B+Fujita%2C+S.%3B+Warashina%2C+M.%3B+Baba%2C+T.%3B+Taira%2C+K.+J.+Am.+Chem.+Soc.+2002%2C+124%2C+538%E2%88%92543.+Zhou%2C+J.-M.%3B+Fujita%2C+S.%3B+Warashina%2C+M.%3B+Baba%2C+T.%3B+Taira%2C+K.+J.+Am.+Chem.+Soc.2002%2C+124%2C+538%E2%88%92543."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm010398pb00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.-M.%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm010398p&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2002.45.issue-8%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm010398p%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm010398p" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                419KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67999eee4b00198b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
