
<html lang="en"     class="pb-page"  data-request-id="1f581942-028f-4144-8438-3b8cef5c1f55"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm2005018;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2011.54.issue-15;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Identification of the Binding Modes of N-Phenylphthalimides Inhibiting Bacterial Thymidylate Synthase through X-Ray Crystallography Screening" /></meta><meta name="dc.Creator" content="Stefano  Mangani" /></meta><meta name="dc.Creator" content="Laura  Cancian" /></meta><meta name="dc.Creator" content="Rosalida  Leone" /></meta><meta name="dc.Creator" content="Cecilia  Pozzi" /></meta><meta name="dc.Creator" content="Sandra  Lazzari" /></meta><meta name="dc.Creator" content="Rosaria  Luciani" /></meta><meta name="dc.Creator" content="Stefania  Ferrari" /></meta><meta name="dc.Creator" content="M. Paola  Costi" /></meta><meta name="dc.Description" content="To identify specific bacterial thymidylate synthase (TS) inhibitors, we exploited phenolphthalein (PTH), which inhibits both bacterial and human enzymes. The X-ray crystal structure of Lactobacillu..." /></meta><meta name="Description" content="To identify specific bacterial thymidylate synthase (TS) inhibitors, we exploited phenolphthalein (PTH), which inhibits both bacterial and human enzymes. The X-ray crystal structure of Lactobacillu..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 12, 2011" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm2005018" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2011 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm2005018" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm2005018" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm2005018" /></link>
        
    
    

<title>Identification of the Binding Modes of N-Phenylphthalimides Inhibiting Bacterial Thymidylate Synthase through X-Ray Crystallography Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm2005018" /></meta><meta property="og:title" content="Identification of the Binding Modes of N-Phenylphthalimides Inhibiting Bacterial Thymidylate Synthase through X-Ray Crystallography Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0010.jpeg" /></meta><meta property="og:description" content="To identify specific bacterial thymidylate synthase (TS) inhibitors, we exploited phenolphthalein (PTH), which inhibits both bacterial and human enzymes. The X-ray crystal structure of Lactobacillus casei TS (LcTS) that binds PTH showed multiple binding modes of the inhibitor, which prevented a classical structure-based drug design approach. To overcome this issue, we synthesized two phthalimidic libraries that were tested against TS enzymes and then we performed X-ray crystallographic screening of the active compounds. Compounds 6A, 8A, and 12A showed 40-fold higher affinity for bacterial TS than human TS. The X-ray crystallographic screening characterized the binding mode of six inhibitors in complexes with LcTS. Of these, 20A, 23A, and 24A showed a common unique binding mode, whereas 8A showed a different, unique binding mode. A comparative analysis of the LcTS X-ray complexes that were obtained with the pathogenic TS enabled the selection of compounds 8A and 23A as specific compounds and starting points to be exploited for the specific inhibition of pathogen enzymes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm2005018"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm2005018">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm2005018&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm2005018&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm2005018&amp;href=/doi/10.1021/jm2005018" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2011</span><span class="cit-fg-volume">, 54</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 5454-5467</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/54/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm200476p" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm200505e" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of the Binding Modes of <i>N</i>-Phenylphthalimides Inhibiting Bacterial Thymidylate Synthase through X-Ray Crystallography Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefano++Mangani">Stefano Mangani</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Cancian">Laura Cancian</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosalida++Leone">Rosalida Leone</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cecilia++Pozzi">Cecilia Pozzi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sandra++Lazzari">Sandra Lazzari</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosaria++Luciani">Rosaria Luciani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Ferrari">Stefania Ferrari</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=M.+Paola++Costi">M. Paola Costi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-text">Dipartimento di Chimica, Università degli Studi di Siena, Via Aldo Moro 2, 53100 Siena, Italy</span></div><div class="corresp-info">For M.P.C.: phone, 0039-059-205-5125; fax, 0039-059-205-5131; E-mail, <a href="/cdn-cgi/l/email-protection#d0bdb1a2b9b1a0b1bfbcb1feb3bfa3a4b990a5beb9bdbfa2b5feb9a4"><span class="__cf_email__" data-cfemail="650804170c0415040a09044b060a16110c25100b0c080a17004b0c11">[email protected]</span></a>. For S.M.: phone, 0039-0577-234255; fax, 0039-0577-234233; E-mail, <a href="/cdn-cgi/l/email-protection#076a66696066696e4772696e746e296e73"><span class="__cf_email__" data-cfemail="f19c909f96909f98b1849f988298df9885">[email protected]</span></a>. For S.F.: phone, 0039-059-205-5331; fax, 0039-059-205-5131; E-mail, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="c3b0b7a6a5a2adaaa2eda5a6b1b1a2b1aa83b6adaaaeacb1a6edaab7">[email protected]</a><a href="mailto:"></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm2005018&amp;href=/doi/10.1021%2Fjm2005018" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2011</span></span><span class="cit-volume">, 54</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 5454–5467</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 22, 2011</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 April 2011</li><li><span class="item_label"><b>Published</b> online</span>12 July 2011</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 August 2011</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm2005018" title="DOI URL">https://doi.org/10.1021/jm2005018</a></div><div class="article_header-article-copyright"><strong>Copyright © 2011 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5454%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DStefano%2BMangani%252C%2BLaura%2BCancian%252C%2BRosalida%2BLeone%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D54%26issueNum%3D15%26contentID%3Djm2005018%26title%3DIdentification%2Bof%2Bthe%2BBinding%2BModes%2Bof%2BN-Phenylphthalimides%2BInhibiting%2BBacterial%2BThymidylate%2BSynthase%2Bthrough%2BX-Ray%2BCrystallography%2BScreening%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5467%26publicationDate%3DAugust%2B2011">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm2005018"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1143</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm2005018" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of the Binding Modes of N-Phenylphthalimides Inhibiting Bacterial Thymidylate Synthase through X-Ray Crystallography Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Stefano&quot;,&quot;last_name&quot;:&quot;Mangani&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Cancian&quot;},{&quot;first_name&quot;:&quot;Rosalida&quot;,&quot;last_name&quot;:&quot;Leone&quot;},{&quot;first_name&quot;:&quot;Cecilia&quot;,&quot;last_name&quot;:&quot;Pozzi&quot;},{&quot;first_name&quot;:&quot;Sandra&quot;,&quot;last_name&quot;:&quot;Lazzari&quot;},{&quot;first_name&quot;:&quot;Rosaria&quot;,&quot;last_name&quot;:&quot;Luciani&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Ferrari&quot;},{&quot;first_name&quot;:&quot;M.&quot;,&quot;last_name&quot;:&quot;Paola Costi&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2011&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;54&quot;,&quot;pages&quot;:&quot;5454-5467&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm2005018&quot;},&quot;abstract&quot;:&quot;To identify specific bacterial thymidylate synthase (TS) inhibitors, we exploited phenolphthalein (PTH), which inhibits both bacterial and human enzymes. The X-ray crystal structure of Lactobacillus casei TS (LcTS) that binds PTH showed multiple binding modes of the inhibitor, which prevented a classical structure-based drug design approach. To overcome this issue, we synthesized two phthalimidic libraries that were tested against TS enzymes and then we performed X-ray crystallographic screening of the active compounds. Compounds 6A, 8A, and 12A showed 40-fold higher affinity for bacterial TS than human TS. The X-ray crystallographic screening characterized the binding mode of six inhibitors in complexes with LcTS. Of these, 20A, 23A, and 24A showed a common unique binding mode, whereas 8A showed a different, unique binding mode. A comparative analysis of the LcTS X-ray complexes that were obtained with the pathogenic TS enabled the selection of compounds 8A and 23A as specific compounds and starting points t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2005018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2005018" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2005018&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2005018" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2005018&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2005018" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm2005018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2005018&amp;href=/doi/10.1021/jm2005018" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm2005018" /></input><a href="/doi/pdf/10.1021/jm2005018" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm2005018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm2005018%26sid%3Dliteratum%253Aachs%26pmid%3D21696158%26genre%3Darticle%26aulast%3DMangani%26date%3D2011%26atitle%3DIdentification%2Bof%2Bthe%2BBinding%2BModes%2Bof%2BN-Phenylphthalimides%2BInhibiting%2BBacterial%2BThymidylate%2BSynthase%2Bthrough%2BX-Ray%2BCrystallography%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D54%26issue%3D15%26spage%3D5454%26epage%3D5467%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292199" title="Crystallography">Crystallography</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/54/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jmcmar.2011.54.issue-15/production/jmcmar.2011.54.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">To identify specific bacterial thymidylate synthase (TS) inhibitors, we exploited phenolphthalein (PTH), which inhibits both bacterial and human enzymes. The X-ray crystal structure of <i><i>Lactobacillus casei</i></i> TS (LcTS) that binds PTH showed multiple binding modes of the inhibitor, which prevented a classical structure-based drug design approach. To overcome this issue, we synthesized two phthalimidic libraries that were tested against TS enzymes and then we performed X-ray crystallographic screening of the active compounds. Compounds <b>6A</b>, <b>8A</b>, and <b>12A</b> showed 40-fold higher affinity for bacterial TS than human TS. The X-ray crystallographic screening characterized the binding mode of six inhibitors in complexes with LcTS. Of these, <b>20A</b>, <b>23A</b>, and <b>24A</b> showed a common unique binding mode, whereas <b>8A</b> showed a different, unique binding mode. A comparative analysis of the LcTS X-ray complexes that were obtained with the pathogenic TS enabled the selection of compounds <b>8A</b> and <b>23A</b> as specific compounds and starting points to be exploited for the specific inhibition of pathogen enzymes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>§</span> Author Present Address</h4><p class="last">Roche Pharma (Schweiz) AG, Schönmattstrasse 2, 4153 Reinach (BL), Switzerland.</p></div><div class="extra-info-sec articleNote withdrawn"><h4><span></span> </h4><p class="first last">The X-ray coordinates of compounds <b>6A</b>, <b>8A</b>, <b>12A</b>, <b>20A</b>, <b>23A</b>, and <b>24A</b> bound to LcTS-dUMP have been deposited in the Protein Data Bank with accession numbers <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a> (<b>6A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A">3C0A</a> (<b>6A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ">3BNZ</a> (<b>8A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a> (<b>12A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a> (<b>12A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK1">3IK1</a> (<b>20A</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IJZ">3IJZ</a> (<b>23A</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK0">3IK0</a> (<b>24A</b>).</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38800" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38800" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Thymidylate synthase (TS) (EC 2.1.1.45) catalyzes the reductive methylation of 2′-deoxyuridine-5′-monophosphate (dUMP) to form 2′-deoxythymidine-5′-monophosphate (dTMP), using the cofactor <i>N</i><sup>5</sup>,<i>N</i><sup>10</sup>-methylene tetrahydrofolate (MTHF). Because TS represents the only means of synthesizing dTMP in human cells and in many pathogens, it is typically a target in designing chemotherapeutic agents such as anticancer drugs and, more recently, antibacterial drugs.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10">(1-10)</a></div><div class="NLM_p">In previous work, we obtained three X-ray crystal structures of <i><i>Lactobacillus casei</i></i> TS (LcTS) in a complex with phenolphthalein (PTH)<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> or naphthalein derivatives (α156, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSL">1TSL</a>; MR20, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSM">1TSM</a>) and one X-ray crystal structure of <i>Escherichia coli</i> TS (EcTS) in complex with a naphthalein derivative (GA9, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A9W">2A9W</a>)<a onclick="showRef(event, 'ref5 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref9">(5, 9)</a> (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information, Figure S1">Supporting Information, Figure S1</a>). Those results showed that when ligands of this class of compound entered into the active site, they adopted multiple binding modes; therefore, it was not possible to identify the key interactions made by the compounds within the binding site. Consequently, structure-based drug design is difficult to perform and the results might be inconclusive.</div><div class="NLM_p last">To develop high-affinity inhibitors targeting bacterial TS, we needed to identify specific inhibitors that presented a unique binding mode. Therefore, starting from PTH, we designed and synthesized a new class of phthalimidic compounds using a ligand-based approach with two scaffolds. The first round of synthesis provided specific compounds with moderate affinity for bacterial TS. X-ray crystallographic screening of the compound library yielded four crystallographic LcTS–inhibitor complexes showing unique binding modes. Information gained through the crystallographic studies on LcTS suggests structural guidelines that would prospectively lead to more specific, higher-affinity inhibitors toward TS from pathogenic organisms.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Scaffold Identification and Validation</h3><div class="NLM_p">PTH has received little attention as a TS inhibitor, with the exception of a limited structure–activity relationship study that investigated phenol-substituted analogues.<a onclick="showRef(event, 'ref7 ref8 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref10">(7, 8, 10)</a> We further explored PTH through a retrosynthetic approach<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> to identify fragments that could be further elaborated (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Two fragments were identified: phenol (<b>1</b>) and 2-benzofuran-1,3-dione (<b>2</b>). The phenol was not considered useful because of its small size and its expected low binding specificity. By contrast, the 2-benzofuran-1,3-dione was more suitable for multiple derivatizations; therefore, we used this core structure to examine the ChemFinder database.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> A number of compounds were retrieved, and we identified eight potential scaffolds, compounds <b>2</b>–<b>9</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), for further synthetic elaboration. In a preliminary chemical modification study, we applied different synthetic strategies. Compounds <b>3</b> (4-methyl-2-benzofuran-1,3-dione) and <b>4</b> (5-methyl-2-benzofuran-1,3-dione) were the most promising molecules because, with an amine library, we could selectively derivatize either the phthalic oxygen and/or positions 4 or 5, with a preliminary bromination. With respect to the bromination of the methyl group, compound <b>3</b> was the most active and compound <b>4</b> did not react properly. Compound <b>3</b> was selected as scaffold 1 for the subsequent production of the substituted phthalimide libraries. A second scaffold was derived from compound <b>3</b>, where the anhydride functionality was aminomethylated and the methyl in position 4 was modified to a bromo-methylene group to obtain scaffold 2 (i.e., compound <b>11</b>) (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Retrosynthetic analysis of phenolphthalein (PTH).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds <b>2</b>–<b>9</b> were identified for further synthetic elaboration through ChemFinder database exploration based on 2-benzofuran-1,3-dione (<b>2</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) method 1: the opportune amine, toluene, EtOH, reflux, 15 h; method 2: the opportune amine, acetic acid, reflux, 5 h. (b) NH<sub>2</sub>CH<sub>3</sub>, toluene, EtOH, yield 47%; (c) NBS, benzoyl peroxide, CCl<sub>4</sub>, hν, yield 64%; (d) the opportune amine, DMF, 75 °C, 15 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design and Synthesis of Libraries A and B</h3><div class="NLM_p">A fragment virtual library based on molecular diversity was designed with ChemFinder software.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Of the 23130 available amines, we discarded molecules (fragments) with a molecular weight greater than 200 Da, those that were structurally complex, and those with cross-reacting groups. The remaining 303 molecules were further filtered. Their molecular properties were calculated (ACD Laboratories 6.0),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and the final number was reduced to 47 candidates according to Lipinsky’s rules,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> molecular diversity, and commercial availability.</div><div class="NLM_p">The fragments used to synthesize libraries A and B were chosen to ensure that all the expected compounds possessed satisfactory drug-like properties and high molecular diversity (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Charts S1 and S2, Table S1, Figure S2">Supporting Information Charts S1 and S2, Table S1, Figure S2</a>). The compounds of library A possessed molecular masses ranging from 238 to 451 Da, exhibited predicted water solubility between 3.6 × 10<sup>–7</sup> and 1.34 mol/L, and had computed octanol/water partition coefficients (clogD values) from −2.21 to 5.52 (pH 7.4). The compounds of library B had molecular masses ranging from 243 to 370 Da, had predicted water solubilities from 2.6 × 10<sup>–6</sup> to 0.14 mol/L, and displayed clogD values from −3.80 to 4.46 (pH 7.4).</div><div class="NLM_p last">Library A derivatives were prepared with two different methods. In method 1, a small excess of the appropriate amine, previously dissolved in ethanol, was added dropwise to a solution of scaffold 1 in toluene, and the reaction mixture was refluxed for 15 h. The solvent was removed under reduced pressure, and the residue was washed with ethanol to eliminate excess amine. In method 2, the opportune amine and scaffold 1 were mixed at an equimolar ratio in acetic acid, and then the reaction mixture was refluxed for 5 h. After precipitation with water, the product was isolated by filtration. Compounds <b>1A</b>, <b>4A</b>, <b>6A</b>, <b>11A</b>–<b>12A</b>, <b>15A</b>, <b>18A</b>–<b>19A</b>, and <b>21A</b>–<b>23A</b> were synthesized using method 1, and compounds <b>5A</b>, <b>8A</b>, <b>17A</b>, <b>20A</b>, and <b>24A</b> were synthesized using method 2. The synthesis of library B commenced with the bromination of 2,4-dimethyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>10</b>) with <i>N</i>-bromosuccinimide and benzoyl peroxide under UV light to produce a good yield of 4-(bromomethyl)-2-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>11</b>). Treatment of <b>11</b> with the suitable amine in DMF at 75 °C for 15 h produced compounds <b>3B</b>, <b>4B</b>, <b>7B</b>, <b>9B</b>, <b>11B</b>, <b>13B</b>, <b>15B</b>, <b>16B</b>, <b>22B</b>, and <b>23B</b>. A total of 26 compounds were obtained with at least 95% purity.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biological Evaluation of Libraries <b>A</b> and <b>B</b></h3><div class="NLM_p">The 26 synthesized compounds were tested against TS enzyme from humans, from the pathogenic bacteria <i>Escherichia coli</i> and <i><i>Entercoccus faecalis</i></i>, and from the model bacteria <i>Lactobacillus casei</i> with a rapid screening assay (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">1</a> and <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S3">Supporting Information Figure S3</a>). All active compounds showed competitive inhibition with respect to the MTHF cofactor.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biological Activity of Libraries A and B Derivatives<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">code</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> hTS<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> EcTS<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> EfTS<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> LcTS<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α156<a class="ref internalNav" href="#t2fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">30 ± 5</td><td class="colsep0 rowsep0" align="center">0.60 ± 0.12</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">0.70 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PTH<a class="ref internalNav" href="#t2fn3" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">1.2 ± 0.2</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">1.4 ± 0.3</td><td class="colsep0 rowsep0" align="center">4.7 ± 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b> (scaffold 1)</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">49 ± 10</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b> (scaffold 2)</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">95 ± 22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">20 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">44 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">18 ± 2</td><td class="colsep0 rowsep0" align="center">60 ± 12</td><td class="colsep0 rowsep0" align="center">14 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">111 ± 25</td><td class="colsep0 rowsep0" align="center">7.0 ± 1.4</td><td class="colsep0 rowsep0" align="center">23 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">31 ± 6</td><td class="colsep0 rowsep0" align="center">8.0 ± 1.6</td><td class="colsep0 rowsep0" align="center">22 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11A</b></td><td class="colsep0 rowsep0" align="center">206 ± 41</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">33 ± 7</td><td class="colsep0 rowsep0" align="center">34 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12A</b></td><td class="colsep0 rowsep0" align="center">81 ± 16</td><td class="colsep0 rowsep0" align="center">3.0 ± 0.6</td><td class="colsep0 rowsep0" align="center">2.0 ± 0.35</td><td class="colsep0 rowsep0" align="center">3.0 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">11 ± 2</td><td class="colsep0 rowsep0" align="center">ni</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17A</b></td><td class="colsep0 rowsep0" align="center">60 ± 12</td><td class="colsep0 rowsep0" align="center">42 ± 8</td><td class="colsep0 rowsep0" align="center">11 ± 2</td><td class="colsep0 rowsep0" align="center">ni</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">61 ± 17</td><td class="colsep0 rowsep0" align="center">93 ± 19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19A</b></td><td class="colsep0 rowsep0" align="center">30 ± 6</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">69 ± 14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20A</b></td><td class="colsep0 rowsep0" align="center">43 ± 9</td><td class="colsep0 rowsep0" align="center">36 ± 7</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">14 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">26 ± 5</td><td class="colsep0 rowsep0" align="center">25 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">33 ± 7</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">36 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">41 ± 8</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">6.0 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24A</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">9.0 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">27 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">109 ± 22</td><td class="colsep0 rowsep0" align="center">56 ± 11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7B</b></td><td class="colsep0 rowsep0" align="center">48 ± 10</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">183 ± 37</td><td class="colsep0 rowsep0" align="center">41 ± 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">61 ± 12</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11B</b></td><td class="colsep0 rowsep0" align="center">21 ± 4</td><td class="colsep0 rowsep0" align="center">42 ± 8</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">33 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">82 ± 16</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">14 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">51 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">92 ± 18</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">10 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">14 ± 3</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23B</b></td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">ni</td><td class="colsep0 rowsep0" align="center">38 ± 8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last"><i>K</i><sub>i</sub> for the enzymes tested are reported (<i>K</i><sub>i</sub> hTS, <i>K</i><sub>i</sub> EcTS, <i>K</i><sub>i</sub> LcTS, <i>K</i><sub>i</sub> LcTS).</p></div><div class="footnote" id="t2fn1"><sup>b</sup><p class="last">ni = no detectable inhibition at 50 μM.</p></div><div class="footnote" id="t2fn2"><sup>c</sup><p class="last">Already reported in refs <a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">1,2</a>, and <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">6</a>.</p></div><div class="footnote" id="t2fn3"><sup>d</sup><p class="last">Already reported in ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">5</a>.</p></div></div></div><div class="NLM_p">Compound <b>3</b> was tested against TS enzymes and showed moderate affinity for EcTS (20% inhibition at 50 μM) but did not show affinity for human TS (hTS) (no activity at 50 μM).</div><div class="NLM_p">Compounds <b>11A</b>, <b>17A</b>, <b>19A</b>, and <b>20A</b> that inhibited hTS were deemed less likely candidates than those such as <b>5A</b>, <b>6A</b>, <b>8A</b>, <b>15A</b>, and <b>21A</b> that had no activity against hTS but had <i>K</i><sub>i</sub> values below 30 μM for EcTS or <i>Entercoccus faecalis</i> TS (EfTS). Compound <b>12A</b> had activity against hTS but even greater activity against EcTS and EfTS, so the compound displayed selectivity.</div><div class="NLM_p">Within library B, only one compound (<b>22B</b>) inhibited EcTS with a <i>K</i><sub>i</sub> value lower than 30 μM; none of the compounds show <i>K</i><sub>i</sub> values lower than 30 μM against EfTS. Only two compounds, <b>7B</b> and <b>11B</b>, showed some inhibitory activity against hTS with <i>K</i><sub>i</sub> values of 48 and 21 μM, respectively. All the other candidates in library B showed no detectable inhibitory activity against hTS at 50 μM concentration.</div><div class="NLM_p last">All compounds were also tested against LcTS because it is often used in TS crystallographic studies, even though it is from a nonpathogenic bacteria. All but four of the compounds (<b>15A</b>, <b>17A</b>, <b>9B</b>, and <b>22B</b>) were active against LcTS, with <i>K</i><sub>i</sub> values ranging from 3 to 93 μM.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-Ray Crystallographic Screening</h3><div class="NLM_p">We performed X-ray crystallographic screening to gain the necessary structural information on the binding modes of these compounds. Compared to library B, library A had a higher number of and more active and specific compounds that were easier to synthesize. Nevertheless, our attempts to crystallize library A compounds in complex with TS from the different pathogens (EcTS and EfTS) failed; crystal structures were obtained only with a TS from a nonpathogenic bacteria, LcTS. Despite not being able to rule out that the same compounds could show different binding modes in other bacterial enzymes, we could use proven binding modes as a guide to design better derivatives taking into account the structural differences among the different enzymes.</div><div class="NLM_p">Eight crystals of LcTS bound to six different ligands (<b>6A</b>, <b>8A</b>, <b>12A</b>, <b>20A</b>, <b>23A</b>, and <b>24A</b>) were successfully obtained and analyzed (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">2</a>). The LcTS structure consisted of a homodimer with the two subunits related by a 2-fold crystallographic axis of the hexagonal <i>P</i>6<sub>1</sub>22 space group; all LcTS-complex crystals showed this symmetry. Each LcTS subunit had two domains. The larger domain (residues 1–69 and 141–316) had an α/β structure with six α-helices, one β-hairpin, a two-stranded β-sheet, and a five-stranded β-sheet. The second domain, termed the small domain (SD, residues 70–139), consisted of four α-helices and a long interconnecting loop.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> This region consistently showed low electron density as a result of structural flexibility, especially between residues 95 and 120, which could not be modeled in some of the structures. The active site cavity was embedded in the large domain, but one of its walls contained two Arg residues (Arg178′ and Arg179′) from the symmetry-related subunit; these Arg residues contributed to the binding site of the dUMP phosphate moiety. The active site residues involved in catalysis included the nucleophilic Cys198, Ser219, His259, and Asn229, which could interact with the dUMP ribose and uridine moieties, as well as Ile81, Trp82, Trp85, Leu224, Phe228, and Val314, which defined a large hydrophobic cavity capable of hosting the aromatic moiety of MTHF. After the determination and refinement of the crystal structure (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>), the <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> difference Fourier maps (Δ<i>F</i>) were calculated with phases from the refined LcTS model; these Δ<i>F</i> maps were inspected to identify the bound inhibitor molecule. The Δ<i>F</i> maps showed a large positive electron density (between 2σ and 7σ) in the active sites due to the presence of dUMP and the different ligands. Although the Δ<i>F</i> maps showed a low or incomplete electron density for the inhibitor molecule, we were able to confidently position the main features of the ligand molecules. The dUMP substrate occupied the usual dUMP site, which extended from the phosphate binding site (Arg23, Arg218, and Ser219 from one subunit and Arg178′ and Arg179′ from the symmetry-related subunit), located at the interface between the two subunits, to the pyrimidine binding site located deeper in the cavity between the backbone of Asp221 and the side chain of Asn229.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Data Collection and Refinement Statistics for X-Ray Structures of Compounds <b>6A</b>, <b>8A</b>, <b>12A</b>, <b>20A</b>, <b>23A</b>, and <b>24A</b> Bound to <i>Lactobacillus casei</i> Thymidylate Synthase (LcTS) and its Substrate, 2′-Deoxyuridine-5′-monophosphate (dUMP)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">LcTS-dUMP– <b>6A</b>_1 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP– <b>6A</b>_2 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A">3C0A</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP–<b>8A</b><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ">3BNZ</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP–<b>12A</b>_1 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP– <b>12A</b>_2 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP–<b>20A</b><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK1">3IK1</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP–<b>23A</b><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IJZ">3IJZ</a></th><th class="colsep0 rowsep0" align="center">LcTS-dUMP–<b>24A</b><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK0">3IK0</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">beamline</td><td class="colsep0 rowsep0" align="left">ESRF ID14–1</td><td class="colsep0 rowsep0" align="left">ESRF ID23–1</td><td class="colsep0 rowsep0" align="left">ESRF ID14–1</td><td class="colsep0 rowsep0" align="left">ESRF ID14–1</td><td class="colsep0 rowsep0" align="left">EMBL BW7B</td><td class="colsep0 rowsep0" align="left">ESRF ID23–1</td><td class="colsep0 rowsep0" align="left">ESRF ID23–1</td><td class="colsep0 rowsep0" align="left">ESRF ID23–1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data collection oscillation angle (deg)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.75</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength, (Å)</td><td class="colsep0 rowsep0" align="left">0.930</td><td class="colsep0 rowsep0" align="left">1.008</td><td class="colsep0 rowsep0" align="left">0.930</td><td class="colsep0 rowsep0" align="left">0.930</td><td class="colsep0 rowsep0" align="left">1.078</td><td class="colsep0 rowsep0" align="left">0.983</td><td class="colsep0 rowsep0" align="left">0.983</td><td class="colsep0 rowsep0" align="left">0.983</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td><td class="colsep0 rowsep0" align="left"><i>P</i>6<sub>1</sub>22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions (Å)</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 76.57 <br /></br><i>b</i> = 76.57 <br /></br><i>c</i> = 212.93</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 78.52 <br /></br><i>b</i> = 78.52 <br /></br><i>c</i> = 224.38</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 76.56 <br /></br><i>b</i> = 76.56 <br /></br><i>c</i> = 213.07</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 77.01 <br /></br><i>b</i> = 77.01 <br /></br><i>c</i> = 213.97</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 77.92 <br /></br><i>b</i> = 77.92 <br /></br><i>c</i> = 224.18</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 77.99 <br /></br><i>b</i> = 77.99 <br /></br><i>c</i> = 224.72</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 77.66 <br /></br><i>b</i> = 77.66 <br /></br><i>c</i> = 224.97</td><td class="colsep0 rowsep0" align="left"><i>a</i> = 78.20 <br /></br><i>b</i> = 78.20 <br /></br><i>c</i> = 226.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Z</i></td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">38.29–2.60</td><td class="colsep0 rowsep0" align="left">50.12–2.40</td><td class="colsep0 rowsep0" align="left">36.04–2.60</td><td class="colsep0 rowsep0" align="left">48.74–2.40</td><td class="colsep0 rowsep0" align="left">33.40–2.70</td><td class="colsep0 rowsep0" align="left">67.57–2.25</td><td class="colsep0 rowsep0" align="left">66.27–2.21</td><td class="colsep0 rowsep0" align="left">75.59–2.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total refln</td><td class="colsep0 rowsep0" align="left">95318</td><td class="colsep0 rowsep0" align="left">171356</td><td class="colsep0 rowsep0" align="left">49201</td><td class="colsep0 rowsep0" align="left">71236</td><td class="colsep0 rowsep0" align="left">41655</td><td class="colsep0 rowsep0" align="left">401629</td><td class="colsep0 rowsep0" align="left">341948</td><td class="colsep0 rowsep0" align="left">430226</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique refln</td><td class="colsep0 rowsep0" align="left">11312</td><td class="colsep0 rowsep0" align="left">16592</td><td class="colsep0 rowsep0" align="left">11909</td><td class="colsep0 rowsep0" align="left">13070</td><td class="colsep0 rowsep0" align="left">10443</td><td class="colsep0 rowsep0" align="left">20159</td><td class="colsep0 rowsep0" align="left">20713</td><td class="colsep0 rowsep0" align="left">24941</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">98.2</td><td class="colsep0 rowsep0" align="left">98.4</td><td class="colsep0 rowsep0" align="left">98.4</td><td class="colsep0 rowsep0" align="left">84.4</td><td class="colsep0 rowsep0" align="left">88.6</td><td class="colsep0 rowsep0" align="left">99.9</td><td class="colsep0 rowsep0" align="left">98.2</td><td class="colsep0 rowsep0" align="left">99.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>sym</sub> (%)</td><td class="colsep0 rowsep0" align="left">0.100</td><td class="colsep0 rowsep0" align="left">0.095</td><td class="colsep0 rowsep0" align="left">0.169</td><td class="colsep0 rowsep0" align="left">0.095</td><td class="colsep0 rowsep0" align="left">0.139</td><td class="colsep0 rowsep0" align="left">0.079</td><td class="colsep0 rowsep0" align="left">0.093</td><td class="colsep0 rowsep0" align="left">0.106</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multiplicity</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">10.3</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">19.9</td><td class="colsep0 rowsep0" align="left">16.5</td><td class="colsep0 rowsep0" align="left">17.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">5.6</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">7.0</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">37.7</td><td class="colsep0 rowsep0" align="left">29.6</td><td class="colsep0 rowsep0" align="left">27.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>cryst</sub> (%)</td><td class="colsep0 rowsep0" align="left">21.86</td><td class="colsep0 rowsep0" align="left">25.54</td><td class="colsep0 rowsep0" align="left">21.42</td><td class="colsep0 rowsep0" align="left">20.83</td><td class="colsep0 rowsep0" align="left">22.58</td><td class="colsep0 rowsep0" align="left">21.5</td><td class="colsep0 rowsep0" align="left">21.0</td><td class="colsep0 rowsep0" align="left">20.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">29.18</td><td class="colsep0 rowsep0" align="left">32.61</td><td class="colsep0 rowsep0" align="left">32.31</td><td class="colsep0 rowsep0" align="left">28.32</td><td class="colsep0 rowsep0" align="left">32.62</td><td class="colsep0 rowsep0" align="left">27.1</td><td class="colsep0 rowsep0" align="left">25.3</td><td class="colsep0 rowsep0" align="left">25.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein atoms</td><td class="colsep0 rowsep0" align="left">2507</td><td class="colsep0 rowsep0" align="left">2394</td><td class="colsep0 rowsep0" align="left">2566</td><td class="colsep0 rowsep0" align="left">2577</td><td class="colsep0 rowsep0" align="left">2521</td><td class="colsep0 rowsep0" align="left">2762</td><td class="colsep0 rowsep0" align="left">2754</td><td class="colsep0 rowsep0" align="left">2786</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">70</td><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">123</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">158</td><td class="colsep0 rowsep0" align="left">146</td><td class="colsep0 rowsep0" align="left">178</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot stats (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">most allowed</td><td class="colsep0 rowsep0" align="left">81.8</td><td class="colsep0 rowsep0" align="left">88.5</td><td class="colsep0 rowsep0" align="left">87.8</td><td class="colsep0 rowsep0" align="left">90.8</td><td class="colsep0 rowsep0" align="left">81.6</td><td class="colsep0 rowsep0" align="left">93.5</td><td class="colsep0 rowsep0" align="left">94.5</td><td class="colsep0 rowsep0" align="left">96.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed</td><td class="colsep0 rowsep0" align="left">18.2</td><td class="colsep0 rowsep0" align="left">11.5</td><td class="colsep0 rowsep0" align="left">12.2</td><td class="colsep0 rowsep0" align="left">9.2</td><td class="colsep0 rowsep0" align="left">18.0</td><td class="colsep0 rowsep0" align="left">6.5</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">3.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">gen allowed</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.01</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd bond angles (deg)</td><td class="colsep0 rowsep0" align="left">2.11</td><td class="colsep0 rowsep0" align="left">2.20</td><td class="colsep0 rowsep0" align="left">2.17</td><td class="colsep0 rowsep0" align="left">1.93</td><td class="colsep0 rowsep0" align="left">2.15</td><td class="colsep0 rowsep0" align="left">2.03</td><td class="colsep0 rowsep0" align="left">2.23</td><td class="colsep0 rowsep0" align="left">1.84</td></tr></tbody></table></div></div><div class="NLM_p">For two ligands (<b>6A</b> and <b>12A</b>; parts A and B of Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>, respectively), the X-ray crystallographic screen was used to identify different binding modes. For each ligand, two structures were obtained from independent crystallization and diffraction experiments that showed the molecule bound in distinct orientations to the active site of LcTS. In one crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=C03A">C03A</a>), the inhibitor <b>6A</b> formed stacking interactions with Phe228, Trp82, and Trp85, a hydrogen bond with Glu60, and contact interactions with Ile81, Glu84, Leu195, Leu224, and Gly225 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). In the other crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a>), <b>6A</b> formed stacking interactions with Trp82, Trp85, and the pyrimidine ring of dUMP as well as contact interactions with Ile81, Leu224, and Gly225 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). A comparison of the two structures showed that, in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A">3C0A</a>, the phthalimidic ring of <b>6A</b> was rotated by ∼90° with respect to its position in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). The observed binding modes of <b>6A</b> in the two complexes displayed three important features: (i) the <b>6A</b> pyridine ring was located close to Trp82 and Trp85, (ii) the chlorine (Cl) atom interacted with the indole Nε1 atom of Trp85, and (iii) the phthalimidic moiety of <b>6A</b> was involved in the aromatic stacking interactions with dUMP or with Phe228.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Superposition of the two crystal structures of LcTS-dUMP–<b>6A</b>. (B) Superimposition of the two crystal structures of LcTS-dUMP–<b>12A</b>. Compound <b>12A</b> was modeled in two possible bound conformations in one crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>; the two conformations are shown with C in pink in the picture). (C) Crystal structure of LcTS-dUMP–<b>8A</b>. The protein is represented as a ribbon; ligands are represented as sticks that are color-coded according to the atom (N in blue, O in red, C in pink, yellow, or orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding modes of compounds as observed in the crystal structures. (A) <b>6A</b> bound in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A">3C0A</a> or (B) in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a>. (C and D) Compound <b>12A</b> is shown in its (C) first binding mode and (D) second binding mode as modeled in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>. (E) Compound <b>12A</b> is shown in its binding mode in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a> crystal structure; (F) compound <b>8A</b> bound in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ">3BNZ</a> crystal structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the case of compound <b>12A</b>, one structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a>), even though it was viewed at a relatively low resolution (2.70 Å), allowed us to model the ligand at full occupancy. By contrast, the other structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>; resolution 2.40 Å) had a weaker electron density; thus, the ligand was modeled in two possible bound conformations, and partial occupancy was assigned to each conformation (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C, 4D). In the first binding mode observed for molecule <b>12A</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>, conformation A), no hydrogen bond was formed, but stacking interactions were observed with Phe228 and Trp82 and contacts were made with Ile81, Trp85, Leu224, Gly225, Asn229, and dUMP (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C). In the second binding mode observed for molecule <b>12A</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>, conformation B), a hydrogen bond was formed with Trp85, stacking interactions were made with Trp85 and dUMP, and contact interactions were made with Arg23, Ala194, Leu195, Asp221, Gly225, Val314, Ala315, and Arg179′ (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D). In the last binding mode observed for molecule <b>12A</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a>), a hydrogen bond was formed with Trp85, stacking interactions were made with Trp82, Trp85, Phe228, and dUMP, and contact interactions were made with Ile81, Glu84, Leu195, and Leu224 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>E).</div><div class="NLM_p">Compounds <b>6A</b> and <b>12A</b> show the best outcome from this study based on having the best affinity and specificity profiles; however, their multiple binding modes prevent the possibility of applying a structure-based approach for further structural development.</div><div class="NLM_p">In the case of the LcTS-dUMP–<b>8A</b> complex (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ">3BNZ</a>), the Δ<i>F</i> map showed positive electron density in the active site that was clearly attributable to dUMP. Less density was present in the hydrophobic pocket, where the inhibitor was expected to bind. Compound <b>8A</b> interacted with LcTS by forming stacking interactions with Phe228 and dUMP and by making contacts with Glu60, Ile81, Glu84, Leu224, Gly225, and Asn229 (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C). Thus compound <b>8A</b> with its single binding mode represents a good initial candidate for EfTS inhibition. However, the EfTS crystallographic structure is not available in the Protein Data Bank for a three-dimensional comparison of the different TS isoforms. Sequence alignment (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C) showed that all the residues interacting with compound <b>8A</b> are conserved between LcTS and EfTS. Comparison with hTS showed that of these residues, only Glu84 is not conserved (Ala in hTS); the whole sequence (Pro53-Glu88) surrounding the interacting residues forming the side wall of the <b>8A</b> binding site is more conserved between LcTS and EfTS on one side than between LcTS and hTS on the other side (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A, 6C). These differences could account for the observed specificity.</div><div class="NLM_p">Three ligands (<b>20A</b>, <b>23A</b>, and <b>24A</b>) showed a common binding pattern, where the phthalimidic core of these molecules formed (i) stacking interactions with Trp82, Phe228, and the pyrimidine ring of the substrate, dUMP, (ii) a hydrogen bond with Asp221 through a conserved water molecule, and (iii) contacts with Glu60, Ile81, Leu224, and Gly225 (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). In all cases, because of the substituent linked at the phthalimidic nitrogen, the phenyl ring on the substituent interacted with the uridine ring of dUMP, with Trp85 (stacking interactions) and with Leu195 (contact). Small differences were observed in the hydrogen-bonding network due to the different functional groups that were present on the phenyl ring. In the case of <b>20A</b>, the hydroxyl group established a hydrogen bond with the ribose ring of dUMP, and the carboxyl group formed hydrogen bonds mediated by 1 or 2 water molecules with Tyr146, Pro196, and Arg218 (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). In the case of compound <b>23A</b>, the acetyl group interacted with the ribose moiety of dUMP through a water molecule bridge, with the phosphate moiety of dUMP and with Arg23 (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). In compound <b>24A</b>, the amidine group formed only a hydrogen bond with the ribose moiety of dUMP (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Superposition of the crystal structures for LcTS-dUMP–<b>20A</b>, LcTS-dUMP–<b>23A</b>, and LcTS-dUMP–<b>24A</b>. (B) Detailed interactions of the phenyl moiety of ligand <b>20A</b> with LcTS. (C) Detailed interaction of the phenyl moiety of ligand <b>23A</b> with LcTS. (D) Detailed interaction of the phenyl moiety of ligand <b>24A</b> with LcTS. All structures are color-coordinated, with oxygen in red, nitrogen in blue, sulfur in yellow, and phosphate in cyan. Carbon atoms are sky blue, purple, and green in LcTS-dUMP–<b>20A</b>, LcTS-dUMP–<b>23A</b>, and LcTS-dUMP–<b>24A</b>, respectively. Hydrogen bonds are indicated with black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Initially, we obtained compounds that showed unique binding modes to LcTS, such as <b>8A</b>, <b>20A</b>, <b>23A</b>, and <b>24A</b>. This is the first time that inhibitors structurally unrelated to the folate cofactor crystallized with the TS protein in a unique binding mode. However, it was necessary to understand how to use the information obtained on a nonpathogenic TS, such as LcTS, to design effective antibacterial agents against EcTS and EfTS targets. With this aim, we critically analyzed each crystallographic complex and then selected the most suitable complexes for further studies.</div><div class="NLM_p">Compounds <b>20A</b>, <b>23A</b>, and <b>24A</b> showed different specificity profiles: <b>20A</b> is active toward LcTS, EcTS, and hTS, and thus, it is not specific; <b>23A</b> is active toward LcTS and EcTS, and thus, it is moderately specific; <b>24A</b> is active only toward LcTS, and thus, it is of low interest. Comparison of LcTS–<b>20A</b> and LcTS–<b>23A</b> complexes shows that among the residues interacting with the two ligands, there is a conformational change in residue Arg23. This residue moves toward the ligand in the case of <b>23A</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). Analyzing the whole protein trace reveals that conformational changes between the two structures involve the whole sequence (Pro21-Gly27) as well as the adjacent small domain sequence from Gly105 to Glu113 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). Previous computational studies<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> described how these two regions move in a coordinated manner during the flexibile motion of the protein highlighting major differences between bacterial TS and hTS. We suggest that the differences in the dynamics of LcTS vs hTS could be fundamental to the specific profile of compound <b>23A</b>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Details of the LcTS–<b>8A</b> complex. Colored in magenta is the Pro53-Lys92 sequence. (B) Structure comparison of LcTS–<b>20A</b> (in magenta) and LcTS–<b>23A</b> (in cyan) complexes. For clarity purposes, only Arg23 residue and Pro21-Gly27 and Pro105-Glu113 trace sequences of LcTS–<b>23A</b> complex are shown to highlight the conformational differences between these two complexes. (C) Protein sequence alignment of EfTS, hTS and LcTS; written in white on a black background are identical residues among the three sequences; written in white on a gray background are identical residues between two out of three sequences. The magenta rectangle highlights the sequence Pro53-Lys92 (LcTS sequences number) that is more conserved between the two bacterial TS with respect to hTS; the green circles indicate the residues interacting with compound <b>8A</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structure-Based–Activity Relationships</h3><div class="NLM_p">All eight structures showed that all the ligands bound in the active site of the enzyme where the MTHF cofactor and the classical folate analogue inhibitors usually bind. This accounted for the competitive inhibition observed in the enzyme inhibition assays, in which competition for binding with the folate substrate was observed. Although all these compounds (<b>6A</b>, <b>8A</b>, <b>12A</b>, <b>20A</b>, <b>23A</b>, and <b>24A</b>) could bind to the same site, they showed different binding modes depending on the substituent present on the phthalimidic nitrogen; the phthalimidic core contributed to the interaction with the enzyme, but this core alone was unable to recognize a specific binding site in the enzyme active site cavity (Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>).</div><figure id="cht1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Library A, Compounds <b>1A</b>–<b>24A</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Three pairs of compounds from the two libraries have the same or similar substituents R/R′ (R on position 2 of scaffold 1 and R′ on position 4 of scaffold 2): <b>4A</b>–<b>9B</b>, <b>8A</b>–<b>15B</b>, and <b>23A</b>–<b>4B</b>. The compounds within the first pair show different specificity profiles. Compound <b>4A</b> is active only toward LcTS (<i>K</i><sub>i</sub> = 44 μM), while <b>9B</b> is inactive against LcTS and active only toward EcTS (<i>K</i><sub>i</sub> = 61 μM); no structural data was obtained with these compounds. In the pair <b>8A</b>–<b>15B</b>, compound <b>8A</b> is active toward all the bacterial enzymes, while compound <b>15B</b> is active only against LcTS. The analysis of the crystallographic complex of LcTS and <b>8A</b> (<i>K</i><sub>i</sub> of 22 μM) suggested that the binding site of <b>8A</b> can be conserved also for <b>15B</b> (<i>K</i><sub>i</sub> of 51 μM); <b>15B</b> can bind with slight motion of the enzyme to accommodate the larger R′ group in position 4 of scaffold 2 (Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>). Considering the case in which the main stacking interaction with Phe228 and dUMP are conserved, the <b>15B</b> ligand could, by flipping around the pthalimidic plane with respect to <b>8A</b>, cause its larger substituent in position 4 to interact with Trp82 and Phe64. Both compounds <b>23A</b>–<b>4B</b>, in which the two substituents are quite similar, show that they are both inactive toward hTS and active toward LcTS, even if <b>4B</b> is approximately 10 times less active than <b>23A</b> (6 and 56 μM, respectively); however, they differ in the activity toward the other bacterial enzymes: <b>23A</b> is active toward EcTS, while <b>4B</b> is active toward EfTS. The analysis of the crystallographic complex of LcTS and <b>23A</b> suggested that the binding site of <b>23A</b> can be conserved also in the case of <b>4B</b>, with a slight rotation of the ligand to accommodate the bigger R′ group in position 4 of scaffold 2. In this way it is possible that the main stacking interaction with Trp82 and Phe228 are retained and that this larger R′ substituent could face the inner pocket of the active site (particularly toward Phe64 and Tyr146) or toward the outer pocket.</div><figure id="cht2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Library B, Compounds <b>1B</b>–<b>23B</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34455" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34455" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In searching for new specific inhibitors of bacterial TS enzymes, we combined a ligand-based design study and a X-ray crystallographic screening of the synthesized libraries to characterize the binding modes of the active compounds.</div><div class="NLM_p">From PTH, a lead inhibitor unsuitable for structure-based drug design, we derived two potential scaffolds that we then used to design and synthesize candidate compound libraries. Derivatization of scaffold 1 gave active compounds in library A, which could be further crystallized with LcTS. The best compounds from this new series had <i>K</i><sub>i</sub> values in the same range as that of PTH toward bacterial TS but they showed lower or no activity against human TS, therefore they are selective for the pathogenic enzymes. X-ray crystallographic screening was applied and provided eight structures of LcTS complex with six compounds out of 26 compounds analyzed. Only compounds <b>20A</b>, <b>23A</b>, and <b>24A</b>, revealed a unique binding mode and a common binding pattern as shown by the crystallographic complexes, while compound <b>8A</b> showed a different but unique mode of binding. Even if our study was limited by having crystal structures from a non pathogenic TS, i.e. LcTS, it demonstrates that the suggested approach can successfully characterize the binding mode of the library compounds and identify unique binding modes even in cases of such a large and hydrophobic active site as that of TS.</div><div class="NLM_p last">In conclusion, our library design and subsequent crystallographic screening represent a strategic tool for providing leads with unique binding modes which are thus suitable starting points for further medicinal chemistry elaboration. In our case, further studies will take advantage of information derived from crystallographic analysis on compounds <b>8A</b> and <b>23A</b> to address the structure-based optimization of the phthalimidic core to develop other derivatives that retain their binding profile but have improved activity and specificity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66547" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66547" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Synthesis and Characterization of Libraries A and B</h3><div class="NLM_p">Reactions were performed using Büchi’s Syncore parallel synthesizer, which allowed the simultaneous production of 24 reactions. Reaction progresses were monitored by TLC on precoated silica gel 60 F<sub>254</sub> plates (Fluka), and visualization was accomplished with UV light (254 nm). The purity of all synthesized compounds was determined by elemental analyses performed on a Perkin-Elmer 240C instrument, and the results were within ±0.4% of the theoretical values. All the compounds tested through a rapid screening assay show >95% purity. Yields of these reactions referred to purified products, and not optimized, were: compounds A method 1 in the range of 20–50%, method 2 in the range 35–55% (except <b>24A</b> and <b>8A</b>, whose yields were 13% and 73% respectively), compounds B method 3 in the range 20–45%. Melting points were determined on a Stuart SMP3 capillary melting point apparatus and are uncorrected. All compounds were characterized by <sup>1</sup>H NMR on a Bruker AC200 and Bruker MX400 WB instruments (CIGS, University of Modena e Reggio Emilia). Two dimensional NMR techniques (heteronuclear single quantum coherence and heteronuclear multiple bond correlation) were used to aid the assignment of signals in <sup>1</sup>H spectra. Compound numbering for <sup>1</sup>H NMR peaks assignment is reported in <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information (Table S2)">Supporting Information (Table S2)</a>. Spectra were recorded in DMSO-<i>d</i><sub>6</sub> or CDCl<sub>3</sub>. Chemicals shifts are reported in ppm from tetramethylsilane as an internal standard. When peak multiplicities are given, the following abbreviations are used: s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; br, broadened signal. Mass spectra were determined on a Finnigan MAT SSQ 710A mass spectrometer (CIGS, University of Modena and Reggio Emilia).</div><div class="NLM_p last">Of the 47 expected compounds of libraries A and B (24 and 23 molecules respectively), only 26 compounds were successfully obtained (16 and 10 molecules, respectively). The other selected fragments did not react well, affording complex mixture of undesired products.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 2,4-Dimethyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>10</b>)</h3><div class="NLM_p last">A solution of methylamine (35%) in H<sub>2</sub>O (1.9 mL, 22 mmol) dissolved in ethanol (20 mL) was added dropwise to a solution of 4-methyl-2-benzofuran-1,3-dione (<b>3</b>) (3 g, 18 mmol) in toluene (25 mL), and the mixture was heated at 80 °C for 4 h. After evaporation of the solvent, the oil residue was extracted four times with diethyl ether. Yield 1.52 g, 47%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.71 (s, 3H, H-4), 3.16 (s, 3H, H-5), 7.45 (d, 1H, <i>J</i> = 7.6 Hz, H-3), 7.56 (t, 1H, <i>J</i> = 7.6 Hz, H-2), 7.59 (d, 1H, <i>J</i> = 7.6 Hz, H-1). Anal. Calcd for C<sub>10</sub>H<sub>9</sub>NO<sub>2</sub>: C, 68.56; H, 5.18; N, 8.00. Found: C, 68.94; H, 5.21; N 7.77.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 4-(Bromomethyl)-2-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>11</b>)</h3><div class="NLM_p last"><i>N</i>-Bromosuccinimide (1.52 g, 8.6 mmol) was added to a solution of <b>10</b> (1.5 g, 8.6 mmol) in CCl<sub>4</sub> (20 mL). The mixture was refluxed and irradiated with an UV lamp, and then the catalyst benzoyl peroxide (0.208 g, 0.86 mmol) in 6 mL of CCl<sub>4</sub> was added. The mixture was cooled with ice, and the resulting precipitate was removed by filtration. The filtrate was concentrated in vacuum, and the oil residue was crystallized several times with CH<sub>2</sub>Cl<sub>2</sub>/<i>n</i>-hexane, affording the product as a white solid. Yield 1.39 g, 64%; mp 138–140 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 254. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.19 (s, 3H, H-5), 4.99 (s, 2H, H-4), 7.70 (d, 1H, <i>J</i> = 6.4 Hz, H-1), 7.75 (t, 1H, <i>J</i> = 6.4 Hz, H-2), 7.80 (d, 1H, <i>J</i> = 6.4 Hz, H-3). Anal. Calcd for C<sub>10</sub>H<sub>8</sub>BrNO<sub>2</sub>: C, 47.27; H, 3.17; N, 5.51. Found: C, 46.94; H, 3.20; N, 5.46.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Synthetic Method 1 for Library A Compounds <b>1A</b>, <b>4A</b>, <b>6A</b>, <b>11</b>–<b>12A</b>, <b>15A</b>, <b>18</b>–<b>19A</b>, <b>21</b>–<b>23A</b></h3><div class="NLM_p">The appropriate ammine (0.37 mmol) in ethanol (2 mL) was added dropwise to a solution of 4-methyl-2-benzofuran-1,3-dione (<b>3</b>) (0.05 g, 0.31 mmol) in toluene (2 mL), and the reaction mixture was refluxed for 15 h. The solvent was removed under reduced pressure, and the residue was washed with ethanol (3 × 2 mL).</div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 5-(4-Methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)benzene-1,3-dicarboxylic Acid (<b>1A</b>)</h4><div class="NLM_p last">Compound <b>1A</b> was synthesized in a similar manner to the method 1 from 5-amino-isophthalic acid, and it was isolated as a beige solid; mp 250–254 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.68 (s, 3H, H-4), 7.70 (m, 1H, H-2), 7.78 (m, 2H, H-1 and H-3), 8.27 (t, 1H, <i>J</i> = 1.6 Hz, H-6), 8.49 (d, 2H, <i>J</i> = 1.6 Hz, H-5 and H-7), 12.95 (br, 2H, H-8). Anal. Calcd for C<sub>17</sub>H<sub>11</sub>NO<sub>6</sub>: C, 62.77; H, 3.41; N, 4.31. Found: C, 62.90; H, 3.40; N, 4.29.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 4-Methyl-2-(2-thiophen-2-ylethyl)-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>4A</b>)</h4><div class="NLM_p last">Compound <b>4A</b> was synthesized in a similar manner to the method 1 from 2-thiophen-2-ethylamine. The reaction was heated at 90 °C for 5 h. After the evaporation of the solvent, the crude residue was crystallized from methanol/H<sub>2</sub>O, affording the white solid product; mp 58–60 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 272. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.70 (s, 3H, H-4), 3.23 (t, 2H, <i>J</i> = 5.8 Hz, H-6), 3.96 (t, 2H, <i>J</i> = 5.8 Hz, H-5), 6.91 (m, 2H, H-7 and H-8), 7.15 (m, 1H, H-9), 7.46 (d, 1H, <i>J</i> = 7.8 Hz, H-3), 7.58 (t, 1H, <i>J</i> = 7.8 Hz, H-2), 7.69 (d, 1H, <i>J</i> = 7.8 Hz, H-1). Anal. Calcd for C<sub>15</sub>H<sub>13</sub>NO<sub>2</sub>S: C, 66.40; H, 4.83; N, 5.16. Found: C, 66.28; H, 4.92; N, 4.99.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 2-(2-Chloropyridin-4-yl)-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>6A</b>)</h4><div class="NLM_p last">Compound <b>6A</b> was synthesized in a similar manner to the method 1 from 4-amino-2-chloropyridine. The reaction was heated at 80 °C for 16 h. After the evaporation of the solvent, the residue was treated with diethyl ether and cooled. The precipitate, the unreacted anhydride, was filtered off while the filtrate was concentrated and the residue was washed with diethyl ether, affording the beige solid product; mp 155 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 273. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.78 (s, 3H, H-4), 7.61 (dd, 1H, <i>J</i> = 5.4, 1.8 Hz, H-7), 7.67 (d, 1H, <i>J</i> = 6.6 Hz, H-3), 7.70 (t, 1H, <i>J</i> = 6.6 Hz, H-2), 7.75 (d, 1H, <i>J</i> = 1.8 Hz, H-5), 7.83 (d, 1H, <i>J</i> = 6.6 Hz, H-1), 8.51 (d, 1H, <i>J</i> = 5.4 Hz, H-6). Anal. Calcd for C<sub>14</sub>H<sub>9</sub>ClN<sub>2</sub>O<sub>2</sub>: C, 61.66; H, 3.33; N 10.27. Found: C, 61.27; H, 3.08; N, 10.14.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 2-[4-(1<i>H</i>-Imidazol-1-yl)phenyl]-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>11A</b>)</h4><div class="NLM_p last">Compound <b>11A</b> was synthesized in a similar manner to the method 1 from 4-(1<i>H</i>-imidazol-1-yl)aniline, and it was isolated as a beige solid. mp 279–280 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.68 (s, 3H, H-4), 7.13 (s, 1H, H-10), 7.58 (d, 2H, <i>J</i> = 9.0 Hz, H-6 and H-7), 7.70 (t, 1H, <i>J</i> = 6.3 Hz, H-2), 7.78 (m, 2H, H-1 and H-3), 7.79 (s, 1H, H-9), 7.80 (d, 2H, <i>J</i> = 9.0 Hz, H-5 and H-8), 8.30 (s, 1H, H-11). Anal. Calcd for C<sub>18</sub>H<sub>13</sub>N<sub>3</sub>O<sub>2</sub>: C, 71.28; H, 4.32; N, 13.85. Found: C, 71.15; H, 4.31; N, 13.82.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-(4-Hydroxybiphenyl-3-yl)-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>12A</b>)</h4><div class="NLM_p last">Compound <b>12A</b> was synthesized in a similar manner to the method 1 from 3-aminobiphenyl-4-ol dissolved in a solution ethanol/methanol 1:1. The reaction was heated at 80 °C for 2 h. The brown solid product was washed with diethyl ether; mp 160–165 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 330. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.78 (s, 3H, H-4), 5.92 (br, 1H, H-13), 7.18 (d, 1H, <i>J</i> = 8.2 Hz, H-5), 7.27 (s, 1H, H-7), 7.32–7.46 (m, 2H, H-6 and H-10), 7.55–7.59 (m, 4H, H-8, H-9, H-11 and H-12), 7.55 (d, 1H, <i>J</i> = 7.4 Hz, H-3), 7.68 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.82 (d, 1H, <i>J</i> = 7.4 Hz, H-1). Anal. Calcd for C<sub>21</sub>H<sub>15</sub>NO<sub>3</sub>: C, 76.58; H, 4.59; N, 4.25. Found: C, 76.60; H, 4.50; N, 4.00.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-Methyl-2-(1,3-thiazol-2-yl)-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>15A</b>)</h4><div class="NLM_p last">Compound <b>15A</b> was synthesized in a similar manner to the method 1 from 2-aminothiazol, and it was isolated as a beige solid; mp 96–99 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.78 (s, 3H, H-4), 7.32 (s, 1H, H-6), 7.68 (d, 1H, <i>J</i> = 7.4 Hz, H-3), 7.69 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.81 (s, 1H, H-5), 7.83 (d, 1H, <i>J</i> = 7.4 Hz, H-1). Anal. Calcd for C<sub>12</sub>H<sub>8</sub>N<sub>2</sub>O<sub>2</sub>S: C, 59.00; H, 3.30; N, 11.47. Found: C, 58.83; H, 3.31; N, 11.49.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2-(3-Hydroxyphenyl)-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>18A</b>)</h4><div class="NLM_p last">Compound <b>18A</b> was synthesized in a similar manner to the method 1 from 3-aminophenol, and it was isolated as a beige solid; mp 195–197 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.65 (s, 3H, H-4), 6.82 (m, 3H, H-5, H-6 and H8), 7.29 (m, 1H, H-7), 7.68 (t, 1H, <i>J</i> = 7.2 Hz, H-2), 7.74 (m, 2H, H-1 and H-3), 9.68 (br, 1H, H-9). Anal. Calcd for C<sub>15</sub>H<sub>11</sub>NO<sub>3</sub>: C, 71.14; H, 4.38; N, 5.53. Found: C, 70.90; H, 4.38; N, 5.51.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 3-(4-Methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)benzoic acid (<b>19A</b>)</h4><div class="NLM_p last">Compound <b>19A</b> was synthesized in a similar manner to the method 1 from 3-aminobenzoic acid. The product was isolated as an oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.67 (s, 3H, H-4), 7.65 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.76 (m, 2H, H-1 and H-3), 7.77 (m, 2H, H-6 and H-7), 7.99 (d, 1H, <i>J</i> = 7.5 Hz, H-8), 8.35 (s, 1H, H-5), 13.00 (br, 1H, H-9). Anal. Calcd for C<sub>16</sub>H<sub>11</sub>NO<sub>4</sub>: C, 68.32; H, 3.94; N, 4.98. Found: C, 68.45; H, 3.93; N, 4.96.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2-(1<i>H</i>-Indol-4-yl)-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>21A</b>)</h4><div class="NLM_p last">Compound <b>21A</b> was synthesized in a similar manner to the method 1 from 4-aminoindole. The product was isolated as a black solid; mp 266–270 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.67 (s, 3H, H-4), 6.25 (d, 1H, <i>J</i> = 5.4 Hz, H-8), 6.99 (d, 1H, <i>J</i> = 7.2 Hz, H-7), 7.19 (t, 1H, <i>J</i> = 7.2 Hz, H-6), 7.36 (d, 1H, <i>J</i> = 5.4 Hz, H-9), 7.50 (d, 1H, <i>J</i> = 7.2 Hz, H-5), 7.69 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.78 (m, 2H, H-1 and H-3), 11.31 (br, 1H, H-10). Anal. Calcd for C<sub>17</sub>H<sub>12</sub>N<sub>2</sub>O<sub>2</sub>: C, 73.90; H, 4.38; N, 10.14. Found: C, 74.11; H, 4.37; N, 10.13.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 2-[4-(5-Chloro-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)-3-methylphenyl]-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>22A</b>)</h4><div class="NLM_p last">Compound <b>22A</b> was synthesized in a similar manner to the method 1 from 2-(4-amino-2-methylphenyl)-5-chloro-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione. The reaction was heated at 90 °C. After 24 h, 2 mL of acetic acid were added and the mixture was heated at 110 °C for other 4 h. The product was isolated through filtration as a white solid; mp 274–276 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 431. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.27 (s, 3H, H-8), 2.78 (s, 3H, H-4), 7.33 (d, 1H, <i>J</i> = 8.4 Hz, H-7), 7.43 (d, 1H, <i>J</i> = 8.0 Hz, H-9), 7.51 (s, 1H, H-5), 7.53 (d, 1H, <i>J</i> = 7.6 Hz, H-3), 7.67 (t, 1H, <i>J</i> = 7.5 Hz, H-2), 7.78 (d, 1H, <i>J</i> = 8.4 Hz, H-6), 7.82 (d, 1H, <i>J</i> = 7.6 Hz, H-1), 7.90 (s, 1H, H-11), 7.94 (m, 1H, H-10). Anal. Calcd for C<sub>24</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>4</sub>: C, 66.91; H, 3.51; N, 6.50. Found: C, 66.69; H, 3.73; N, 6.43.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-(4-Acetylphenyl)-4-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>23A</b>)</h4><div class="NLM_p last">Compound <b>23A</b> was synthesized in a similar manner to the method 1 from 4′-aminoacetophenone, and it was isolated as a white solid; mp 201–203 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.65 (s, 3H, H-4), 2.77 (s, 3H, H-9), 7.56 (d, 1H, <i>J</i> = 7.6 Hz, H-3), 7.63 (d, 2H, <i>J</i> = 8.8 Hz, H-5 and H-8), 7.67 (t, 1H, <i>J</i> = 7.6 Hz, H-2), 7.81 (d, 1H, <i>J</i> = 7.6 Hz, H-1), 8.10 (d, 2H, <i>J</i> = 8.8 Hz, H-6 and H-7). Anal. Calcd for C<sub>17</sub>H<sub>13</sub>NO<sub>3</sub>: C, 73.11; H, 4.69; N, 5.02. Found: C, 73.23; H, 4.70; N, 5.00.</div></div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Synthetic Method 2 for Library A Compounds <b>5A</b>, <b>8A</b>, <b>17A</b>, <b>20A</b>, and <b>24A</b></h3><div class="NLM_p">A mixture of amine (6.2 mmol) and <b>3</b> (1 g, 6.2 mmol) in acetic acid (10 mL) was stirred and refluxed for 5 h; the product was precipitated by addition of water to the cooled solution. The solid was filtrated and washed with water.</div><div id="sec4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-Methyl-2-[4-(trifluoromethyl)phenyl]-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>5A</b>)</h4><div class="NLM_p last">Compound <b>5A</b> was synthesized in a similar manner to the method 2 from 4-(trifluoromethyl)aniline and it was isolated as a white solid; mp 151–153 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 306. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.77 (s, 3H, H-4), 7.57 (d, 1H, <i>J</i> = 7.6 Hz, H-3), 7.62 (t, 1H, <i>J</i> = 7.6 Hz, H-2), 7.67 (d, 1H, <i>J</i> = 7.6 Hz, H-1), 7.74 (d, 2H, <i>J</i> = 8.0 Hz, H-6 and H-7), 7.82 (d, 2H, <i>J</i> = 8.0 Hz, H-5 and H-8). Anal. Calcd for C<sub>16</sub>H<sub>10</sub>F<sub>3</sub>NO<sub>2</sub>: C, 62.96; H, 3.30; N, 4.59. Found: C, 63.00; H, 3.12; N 4.78.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-(4-Methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)benzonitrile (<b>8A</b>)</h4><div class="NLM_p last">Compound <b>8A</b> was synthesized in a similar manner to the method 2 from 4-aminobenzonitrile, and it was isolated as a white solid; mp 245–250 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 263. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 2.77 (s, 3H, H-4), 7.58 (d, 1H, <i>J</i> = 7.2 Hz, H-3), 7.69 (t, 1H, <i>J</i> = 7.2 Hz, H-2), 7.70 (d, 2H, <i>J</i> = 9.0 Hz, H-5 and H-8), 7.81 (d, 1H, <i>J</i> = 7.2 Hz, H-1), 7.82 (d, 2H, <i>J</i> = 9.0 Hz, H-6 and H-7). Anal. Calcd for C<sub>16</sub>H<sub>10</sub>N<sub>2</sub>O<sub>2</sub>: C, 73.27; H, 3.64; N, 10.68. Found: C, 73.22; H, 3.45; N, 10.79.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> ({[4-(4-Methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)phenyl]carbonyl}amino)acetic Acid (<b>17A</b>)</h4><div class="NLM_p last">Compound <b>17A</b> was synthesized in a similar manner to the method 2 from {[(4-aminophenyl)carbonyl]amino}acetic acid, and it was isolated as a beige solid; mp 211–213 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 339. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.67 (s, 3H, H-4), 3.96 (s, 2H, H-10), 7.57 (d, 2H, <i>J</i> = 8.6 Hz, H-5 and H-8), 7.70 (t, 1H, <i>J</i> = 7.6 Hz, H-2), 7.78 (m, 2H, H-1 and H-3), 7.99 (d, 2H, <i>J</i> = 8.6 Hz, H-6 and H-7), 8.91 (br, 1H, H-9), 12.42 (br, 1H, H-11). Anal. Calcd for C<sub>18</sub>H<sub>14</sub>N<sub>2</sub>O<sub>5</sub>: C, 63.90; H, 4.17; N, 8.28. Found: C, 64.21; H, 4.02; N, 8.58.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-Hydroxy-4-(4-methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)benzoic Acid (<b>20A</b>)</h4><div class="NLM_p last">Compound <b>20A</b> was synthesized in a similar manner to the method 2 from 4-amino-2-hydroxybenzoic acid, and it was isolated as a beige solid; mp 289–291 °C. MS (M + H<sup>+</sup>) <i>m</i>/<i>z</i> 298. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.66 (s, 3H, H-4), 7.37 (d, 1H, <i>J</i> = 8.2 Hz, H-6), 7.48 (dd, 1H, <i>J</i> = 8.2, 1.6 Hz, H-7), 7.55 (d, 1H, <i>J</i> = 1.6 Hz, H-5), 7.69 (t, 1H, <i>J</i> = 7.3 Hz, H-2), 7.77 (m, 2H, H-1 and H-3), 10.23 (br, 1H, H-8), 13.0 (br, 1H, H-9). Anal. Calcd for C<sub>16</sub>H<sub>11</sub>NO<sub>5</sub>: C, 64.65; H, 3.73; N, 4.71. Found: C, 64.59; H, 3.99; N, 4.87.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-(4-Methyl-1,3-dioxo-1,3-dihydro-2<i>H</i>-isoindol-2-yl)benzenecarboximidamide Hydrochloride (<b>24A</b>)</h4><div class="NLM_p last">Compound <b>24A</b> was synthesized in a similar manner to the method 2 from 4-aminobenzenecarboximidamide hydrochloride. The reaction was refluxed for 20 h. The mixture was cooled, and the formed precipitate was filtered off. The filtrate was evaporated, and the residue was triturated with acetone, affording the beige solid product; mp 234–235 °C. MS (M – Cl)<sup>+</sup><i>m</i>/<i>z</i> 280. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.68 (s, 3H, H-4), 7.71 (d, 2H, <i>J</i> = 8.8 Hz, H-5 and H-8), 7.73 (t, 1H, <i>J</i> = 7.6 Hz, H-2), 7.79 (m, 2H, H-1 and H-3), 7.96 (d, 2H, <i>J</i> = 8.8 Hz, H-6 and H-7), 9.30 (br, 3H, H-9). Anal. Calcd for C<sub>16</sub>H<sub>14</sub>ClN<sub>3</sub>O<sub>2</sub>: C, 60.86; H, 4.47; N, 15.49. Found: C, 60.74; H, 4.75; N, 15.13.</div></div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Synthetic Method 3 for Library B Compounds</h3><div class="NLM_p">A mixture of <b>11</b> (0.05 g, 0.19 mmol) in DMF (2 mL) and an opportune amine (0.3 mmol) in DMF (1 mL) was heated at 75 °C for 15 h. Then 5 mL of H<sub>2</sub>O were added to the cooled solution.</div><div id="sec4_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl)methyl]amino}pyridine-3-carboxylic Acid (<b>3B</b>)</h4><div class="NLM_p last">Compound <b>3B</b> was synthesized in a similar manner to the method 3 from 5-aminopyridine-2-carboxylic acid, and it was isolated as a beige solid; mp 240–245 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.09 (s, 3H, H-5), 5.96 (s, 2H, H-4), 7.16 (d, 1H, <i>J</i> = 9.2 Hz, H-8), 7.80 (m, 3H, H-1, H-2 and H-3), 8.28 (d, 1H, <i>J</i> = 9.2 Hz, H-7), 8.80 (s, 1H, H-6). Anal. Calcd for C<sub>15</sub>H<sub>11</sub>N<sub>3</sub>O<sub>4</sub>: C, 60.61; H, 3.73; N, 14.14. Found: C, 60.45; H, 3.72; N, 14.11.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Ethyl 4-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl)methyl]amino}benzoate (<b>4B</b>)</h4><div class="NLM_p last">Compound <b>4B</b> was synthesized in a similar manner to the method 3 from benzocaine, and it was isolated as a white solid; mp 198–201 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 1.38 (t, 3H, <i>J</i> = 7.1 Hz, H-11), 3.21 (s, 3H, H-5), 4.34 (q, 2H, <i>J</i> = 7.1 Hz, H-10), 4.96 (s, 2H, H-4), 6.66 (d, 2H, <i>J</i> = 9.0 Hz, H-6 and H-9), 7.66 (m, 2H, H-1 and H-3), 7.78 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.86 (d, 2H, <i>J</i> = 9.0 Hz, H-7 and H-8). Anal. Calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub>: C, 62.76; H, 4.65; N, 12.92. Found: C, 62.62; H, 4.64; N, 12.96.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-{[(3-Methoxypropyl)amino]methyl}-2-methyl-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>7B</b>)</h4><div class="NLM_p last">Compound <b>7B</b> was synthesized in a similar manner to the method 3 from 3-methoxypropylamine, and it was isolated as a beige solid; mp 170–175 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 1.62 (m, 2H, H-8), 2.55 (m, 2H, H-7), 3.19 (s, 3H, H-5), 3.22 (s, 3H, H-10), 3.41 (t, 2H, <i>J</i> = 7.1 Hz, H-9), 4.20 (s, 2H, H-4), 7.78 (m, 3H, H-1, H-2 and H-3). Anal. Calcd for C<sub>13</sub>H<sub>16</sub>N<sub>2</sub>O<sub>3</sub>: C, 62.89; H, 6.50; N, 11.28. Found: C, 63.01; H, 6.49; N, 11.27.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-Methyl-4-{[(2-thiophen-2-ylethyl)amino]methyl}-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>9B</b>)</h4><div class="NLM_p last">Compound <b>9B</b> was synthesized in a similar manner to the method 3 from 2-thiopheneethylamine, and it was isolated as a beige solid; mp 150–155 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.19 (s, 3H, H-5), 3.10 (t, 2H, <i>J</i> = 7.4 Hz, H-8), 3.34 (m, 2H, H-7), 4.61 (br, 2H, H-4), 6.91 (m, 2H, H-9 and H-10), 7.18 (m, 1H, H-11), 7.68 (m, 2H, H-1 and H-3), 7.80 (t, 1H, <i>J</i> = 7.6 Hz, H-2). Anal. Calcd for C<sub>15</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub>S: C, 62.92; H, 4.93; N, 9.78. Found: C, 62.74; H, 4.94; N, 9.75.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl)methyl]amino}benzenesulfonic Acid (<b>11B</b>)</h4><div class="NLM_p last">Compound <b>11B</b> was synthesized in a similar manner to the method 3 from sulfanilic acid, and it was isolated as a beige solid; mp 90–95 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.05 (s, 3H, H-5), 4.95 (s, 2H, H-4), 6.45 (d, 2H, <i>J</i> = 8.8 Hz, H-6 and H-7), 7.28 (d, 2H, <i>J</i> = 8.8 Hz, H-8 and H-9), 7.74 (m, 2H, H-1 and H-3), 7.78 (t, 1H, <i>J</i> = 7.6 Hz, H-2). Anal. Calcd for C<sub>15</sub>H<sub>12</sub>N<sub>2</sub>O<sub>5</sub>S: C, 54.21; H, 3.64; N, 8.43. Found: C, 54.33; H, 3.65; N, 8.41.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-Methyl-4-({[4-(phenylcarbonyl)phenyl]amino}methyl)-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>13B</b>)</h4><div class="NLM_p last">Compound <b>13B</b> was synthesized in a similar manner to the method 3 from 4-aminobenzophenone, and it was isolated as a beige solid; mp 109–115 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.20 (s, 3H, H-5), 4.89 (s, 2H, H-4), 6.75 (d, 2H, <i>J</i> = 8.8 Hz, H-6 and H-8), 7.31 (m, 3H, H-11, H-12 and H-13), 7.66 (m, 2H, H-1 and H-3), 7.50 (d, 2H, <i>J</i> = 8.8 Hz, H-7 and H-9), 7.70 (m, 2H, H-10 and H-14), 7.78 (t, 1H, <i>J</i> = 7.6 Hz, H-2). Anal. Calcd for C<sub>22</sub>H<sub>16</sub>N<sub>2</sub>O<sub>3</sub>: C, 74.15; H, 4.53; N, 7.86. Found: C, 74.29; H, 4.55; N, 7.84.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl)methyl]amino}benzonitrile (<b>15B</b>)</h4><div class="NLM_p last">Compound <b>15B</b> was synthesized in a similar manner to the method 3 from 4-aminobenzonitrile, and it was isolated as a white solid; mp 179–188 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.20 (s, 3H, H-5), 4.81 (s, 2H, H-4), 6.61 (d, 2H, <i>J</i> = 8.4 Hz, H-6 and H-9), 7.40 (d, 2H, <i>J</i> = 8.4 Hz, H-7 and H-8), 7.66 (m, 2H, H-1 and H-3), 7.78 (t, 1H, <i>J</i> = 7.3 Hz, H-2). Anal. Calcd for C<sub>16</sub>H<sub>11</sub>N<sub>3</sub>O<sub>2</sub>: C, 69.31; H, 4.00; N, 15.15. Found: C, 69.25; H, 3.99; N, 15.13.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-Methyl-4-{[(4-nitronaphthalen-1-yl)amino]methyl}-1<i>H</i>-isoindole-1,3(2<i>H</i>)-dione (<b>16B</b>)</h4><div class="NLM_p last">Compound <b>16B</b> was synthesized in a similar manner to the method 3 from 4-nitro-1-naphthylamine, and it was isolated as a brown solid; mp 160–163 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.19 (s, 3H, H-5), 5.03 (s, 2H, H-4), 6.70 (d, 1H, <i>J</i> = 8.4 Hz, H-6), 7.59 (m, 4H, H-8, H-9, H-10 and H-11), 7.74 (m, 2H, H-1 and H-3), 7.82 (t, 1H, <i>J</i> = 7.5 Hz, H-2), 8.39 (d, 1H, <i>J</i> = 8.4 Hz, H-7). Anal. Calcd for C<sub>19</sub>H<sub>13</sub>N<sub>3</sub>O<sub>4</sub>: C, 65.70; H, 3.77; N, 12.10. Found: C, 65.76; H, 3.76; N, 12.12.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl)methyl]amino}ethyl)phosphonic Acid (<b>22B</b>)</h4><div class="NLM_p last">Compound <b>22B</b> was synthesized in a similar manner to the method 3 from 2-aminoethylphosphonic acid, and it was isolated as a beige solid; mp 110–113 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 2.54 (m, 4H, H-7 and H-8), 3.04 (s, 3H, H-5), 4.95 (s, 2H, H-4), 7.71 (d, 1H, <i>J</i> = 7.4 Hz, H-3), 7.79 (t, 1H, <i>J</i> = 7.4 Hz, H-2), 7.90 (d, 1H, <i>J</i> = 7.4 Hz, H-1), 8.13 (br, 1H, H-6). Anal. Calcd for C<sub>11</sub>H<sub>13</sub>N<sub>2</sub>O<sub>5</sub>P: C, 46.49; H, 4.61; N, 9.86. Found: C, 46.43; H, 4.62; N, 9.89.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 3-{[(2-Methyl-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl) methyl]amino} benzenesulfonamide (<b>23B</b>)</h4><div class="NLM_p last">Compound <b>23B</b> was synthesized in a similar manner to the method 3 from 3-aminobenzenesulfonamide, and it was isolated as a white solid; mp 230–235 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.06 (s, 3H, H-5), 4.77 (s, 2H, H-4), 6.71 (d, 1H, <i>J</i> = 7.2 Hz, H-6), 6.81 (br, H-10), 6.99 (d, 1H, <i>J</i> = 7.2 Hz, H-8), 7.06 (s, 1H, H-9), 7.16 (br, 2H, H-11), 7.21 (t, 1H, <i>J</i> = 7.2 Hz, H-7), 7.71 (m, 3H, H-1, H-2 and H-3). Anal. Calcd for C<sub>15</sub>H<sub>13</sub>N<sub>3</sub>O<sub>4</sub>S: C, 54.37; H, 3.95; N, 12.68. Found: C, 54.34; H, 3.94; N, 12.65.</div></div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Enzymology</h3><div class="NLM_p">The compounds were screened for their activity and their species-specificity against EcTS, EfTS, and hTS. Because crystallographic studies were performed also against LcTS, the activity of these compounds toward LcTS enzyme was assayed as well. LcTS, EcTS, and hTS were purified as described.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19-21)</a> EfTS was purified following the procedure used for EcTS. The EfTS strain was a gift of Prof. Robert M. Stroud, University of California San Francisco (CA, USA). Enzyme kinetics experiments were conducted by chromogenic assay under standard conditions.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The <i>K</i><sub>m</sub> values, by curve-fitting to hyperbolic curves, of EcTS, EfTS, LcTS, and hTS for MTHF were, respectively, 5.8, 13.6, 5.7, and 6.9 μM. The <i>K</i><sub>i</sub> ± standard errors were determined from the nonlinear regression analysis of data from at least two independent experiments performed in triplicate.</div><div class="NLM_p last">The inhibition patterns for all the compounds were determined by steady-state kinetic analysis of the dependence of enzyme activity on MTHF concentration at varying inhibitor concentrations. All the compounds showed competitive inhibition with respect to MTHF. The apparent inhibition constant (<i>K</i><sub>i</sub> app, simply defined as <i>K</i><sub>i</sub> within the text) values were obtained from the linear least-squares fit of the residual activity as a function of inhibitor concentration, using suitable equations for competitive inhibition.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In the inhibition assays, the reaction mixture was the same as in the TS standard assay. Stock solutions of each inhibitor were freshly prepared in DMSO and stored at −80 °C. In the reaction mixture, the concentration of DMSO never exceeded 5%; the effect of increasing DMSO concentration in the TS assay mixture was studied, and it was observed that no change in TS activity was seen at concentrations up to 8% DMSO.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> To obtain <i>K</i><sub>i</sub> values, the inhibitors were tested at concentrations ranging from 5 to 500 μM depending on the solubility of the compound studied. When the inhibition of hTS was studied, the concentration of the inhibitor was tested up to 50 μM.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Crystallization of Thymidylate Synthase Complexes</h3><div class="NLM_p">Crystals of LcTS complexes were grown over a few days using the sitting/hanging drop vapor diffusion technique at 20 °C from a 4 mg/mL LcTS solution (buffer: 25 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub>, pH 6.8–7.4, 1 mM EDTA) containing 40 mM dUMP. A 2 mM solution of each inhibitor in DMSO was then added in a 1:10 volume ratio to the protein/dUMP solution.<a onclick="showRef(event, 'ref15 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref15 ref25 ref26">(15, 25, 26)</a></div><div class="NLM_p">In the case of complexes LcTS-dUMP–<b>20A</b> and LcTS-dUMP–<b>23A</b> complexes, drops were prepared by mixing 3 μL of the ternary complex solutions with 1 μL of precipitant solution made of 10% saturated ammonium sulfate and 1 mM DTT. The drops were then inverted over wells containing 20 mM K<sub>2</sub>HPO<sub>4</sub> (pH 6.8) and 1 mM DTT. In the case of all the other complexes, drops were made by mixing equal volumes (2–3 μL) of the LcTS ternary complex solution with a precipitant solution consisting of (NH<sub>4</sub>)H<sub>2</sub>PO<sub>4</sub>/(NH<sub>4</sub>)<sub>2</sub>HPO<sub>4</sub> 100 mM, 5% v/v PEG 400, and 20 mM Tris-HCl buffer at pH 7.4 and equilibrated against the same precipitant solution.</div><div class="NLM_p last">Crystals of the complexes appear in 5–6 days as hexagonal bipyramids and grow to the final dimensions of about 150 μm × 150 μm × 400 μm in about 2 weeks. Before the data collection, crystals were transferred to a cryoprotectant solution composed of the precipitant solution with added 30% PEG 400 and flash frozen in liquid nitrogen. The data collection and refinement statistics are reported in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">2</a>.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Data Collection and Structure Determination</h3><div class="NLM_p">Data sets related to LcTS-dUMP–<b>6A</b>_1, LcTS-dUMP–<b>8A</b>, LcTS-dUMP–<b>12A</b>_1 complexes were collected using synchrotron radiation from the European Synchrotron Radiation Facility, Grenoble, France (ESRF) ID14–1 with wavelength of 0.930 Å with the rotation technique (Δφ = 1°) and an ADSC Quantum 4 detector. The data set of the LcTS-dUMP–<b>6A</b>_2 complex was collected using synchrotron radiation from the European Synchrotron Radiation Facility, Grenoble, France (ESRF) ID23–1 with wavelength of 1.008 Å, the rotation technique (Δφ = 0.75°), and a MARresearch CCD 225 detector. The data set of the LcTS-dUMP–<b>12A</b>_2 complex was collected using synchrotron radiation of European Molecular Biology Laboratory (EMBL) PX beamline BW7B at the DORIS storage ring, DESY, Hamburg, Germany, with wavelength of 1.078 Å, the rotation technique (Δφ = 1°), and detector MARresearch CCD 225 detector. The data sets of the LcTS-dUMP–<b>20A</b>, LcTS-dUMP–<b>23A</b>, and LcTS-dUMP–<b>24A</b> complexes were collected using synchrotron radiation from the ESRF beamline ID23–1 with wavelength of 0.983 Å, the rotation technique (Δφ = 0.5°), and a ADSC Quantum Q315r detector.</div><div class="NLM_p">All the data sets were processed using MOSFLM 6.2<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> and scaled with SCALA.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">The structure solutions of all complexes were performed by the molecular replacement method using the MOLREP software,<a onclick="showRef(event, 'ref28 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref30">(28, 30)</a> where one subunit of LcTS (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LCA">1LCA</a>) was used as a model for the rotation and translation functions. All structures were refined and water molecules were added using REFMAC<a onclick="showRef(event, 'ref29 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref31">(29, 31)</a> and ARP/WARP,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> respectively.</div><div class="NLM_p">The program Xtalview/Xfit<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and Coot<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> were used for manual rebuilding of the molecule, modeling of the inhibitor molecules into the electron density map and visualization for all structures.</div><div class="NLM_p last">Figures were generated using CCP4MG from the CCP4 suite<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and Chimera<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> program.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51"><a href="/doi/suppl/10.1021/jm2005018">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32301" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32301" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Binding modes of known phthalein and naphthalein compounds (PTH; α156, MR20, GA9); designed libraries A and B; calculated molecular properties for libraries A and B; atom numbering in compounds characterized through NMR; plot of the enzyme kinetic inhibition profiles (<i>K</i><sub>i</sub>) for libraries A and B; contact distances between substrates and LcTS residues. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm2005018/suppl_file/jm2005018_si_001.pdf">jm2005018_si_001.pdf (1.17 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm2005018" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18613" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18613" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefano Mangani</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Chimica, Università degli Studi di Siena, Via Aldo Moro 2, 53100 Siena, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c414d5e454d5c4d43404d024f435f58456c59424541435e490245580c414d424b4d42456c5942455f450245580c"><span class="__cf_email__" data-cfemail="ed808c9f848c9d8c82818cc38e829e9984ad98838480829f88c38499">[email protected]</span> <span class="__cf_email__" data-cfemail="234e424d44424d4a63564d4a504a0d4a57">[email protected]</span> </a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Ferrari</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#96fbf7e4fff7e6f7f9faf7b8f5f9e5e2ffd6e3f8fffbf9e4f3b8ffe2b6fbf7f8f1f7f8ffd6e3f8ffe5ffb8ffe2b6"><span class="__cf_email__" data-cfemail="b6dbd7c4dfd7c6d7d9dad798d5d9c5c2dff6c3d8dfdbd9c4d398dfc2">[email protected]</span> <span class="__cf_email__" data-cfemail="761b17181117181f3603181f051f581f02">[email protected]</span> </a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">M. Paola Costi</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81ece0f3e8e0f1e0eeede0afe2eef2f5e8c1f4efe8eceef3e4afe8f5a1ece0efe6e0efe8c1f4efe8f2e8afe8f5a1"><span class="__cf_email__" data-cfemail="aec3cfdcc7cfdecfc1c2cf80cdc1dddac7eedbc0c7c3c1dccb80c7da">[email protected]</span> <span class="__cf_email__" data-cfemail="573a36393036393e1722393e243e793e23">[email protected]</span> </a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Cancian</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosalida Leone</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Chimica, Università degli Studi di Siena, Via Aldo Moro 2, 53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cecilia Pozzi</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Chimica, Università degli Studi di Siena, Via Aldo Moro 2, 53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandra Lazzari</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosaria Luciani</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Scienze Farmaceutiche, Università degli Studi di Modena e Reggio Emilia, Via Campi 183, 41126 Modena, Italy</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03570" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03570" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was financially supported by MIUR-PRIN 2006-2006030430_004 CSTMPL. We thank the Cassa di Risparmio di Modena (CDRM) Foundation. We are grateful to Merck Eprova AG, Switzerland, an affiliate of Merck KGaA, Darmstadt, Germany, for providing the (6<i>R</i>)-5,10-methylenetetrahydrofolic acid sodium salt. We also acknowledge ESRF and the EMBL Hamburg outstation for the use of their beamlines for data collection. We thank Professor B. K. Shoichet for providing the pdb file of the crystallographic complex LcTS-PTH, Professor Marcella Rinaldi for assistance with the synthetic work, and Fabrizia Soragni for her technical assistance.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i52" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i52"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i53" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i53"> Abbreviations Used</h2><tr><td class="NLM_term">CDCl<sub>3</sub></td><td class="NLM_def"><p class="first last">deuterated trichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">dTMP</td><td class="NLM_def"><p class="first last">2′-deoxythymidine-5′-monophosphate</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiotreitol</p></td></tr><tr><td class="NLM_term">dUMP</td><td class="NLM_def"><p class="first last">2′-deoxyuridine-5′-monophosphate</p></td></tr><tr><td class="NLM_term">EcTS</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i> thymidylate synthase</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">EfTS</td><td class="NLM_def"><p class="first last"><i>Enterococcus faecalis</i> thymidylate synthase</p></td></tr><tr><td class="NLM_term">hTS</td><td class="NLM_def"><p class="first last">human thymidylate synthase</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>i</sub></td><td class="NLM_def"><p class="first last">inhibition constant</p></td></tr><tr><td class="NLM_term">LcTS</td><td class="NLM_def"><p class="first last"><i>Lactobacillus casei</i> thymidylate synthase</p></td></tr><tr><td class="NLM_term">MTHF</td><td class="NLM_def"><p class="first last"><i>N</i><sup>5</sup>,<i>N</i><sup>10</sup>-methylene tetrahydrofolate</p></td></tr><tr><td class="NLM_term">PEG 400</td><td class="NLM_def"><p class="first last">polyethylene glycol, MW 400</p></td></tr><tr><td class="NLM_term">PTH</td><td class="NLM_def"><p class="first last">phenolphthalein</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">small domain</p></td></tr><tr><td class="NLM_term">TS</td><td class="NLM_def"><p class="first last">thymidylate synthase</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35228" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35228" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Carreras, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">The catalytic mechanism and structure of thymidylate synthase</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">762</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1146%2Fannurev.bi.64.070195.003445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=7574499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2MXmsl2qtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1995&pages=721-762&author=C.+W.+Carrerasauthor=D.+V.+Santi&title=The+catalytic+mechanism+and+structure+of+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The catalytic mechanism and structure of thymidylate synthase</span></div><div class="casAuthors">Carreras, Christopher W.; Santi, Daniel V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">721-62</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review, with 193 refs.  Thymidylate synthase (TS, EC 2.1.1.45) catalyzes the reductive methylation of dUMP by CH2H4folate to produce dTMP and H2folate.  Knowledge of the catalytic mechanism and structure of TS has increased substantially over recent years.  Major advances were derived from crystal structures of TS bound to various ligands, the ability to overexpress TS in heterologous hosts, and the numerous mutants that have been prepd. and analyzed.  These advances, coupled with previous knowledge, have culminated in an in-depth understanding of many important mol. details of the reaction.  The authors review aspects of TS catalysis that are most pertinent to understanding the current status of the structure and catalytic mechanism of the enzyme.  Included is a discussion of available sources and assays for TS, a description of the enzyme's chem. mechanism and crystal structure, and a summary of data obtained from mutagenesis expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO-xv6vKzvm7Vg90H21EOLACvtfcHk0ljZ-DBiHuhEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmsl2qtrg%253D&md5=3fa2b583e048196a12193b27f9533770</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.64.070195.003445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.64.070195.003445%26sid%3Dliteratum%253Aachs%26aulast%3DCarreras%26aufirst%3DC.%2BW.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DThe%2520catalytic%2520mechanism%2520and%2520structure%2520of%2520thymidylate%2520synthase%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1995%26volume%3D64%26spage%3D721%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Chu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callender, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrell, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitz, J. C.</span><span> </span><span class="NLM_article-title">Thymidylate synthase inhibitors as anticancer agents: from bench to bedside</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S80</span><span class="NLM_x">–</span> <span class="NLM_lpage">S89</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=S80-S89&issue=Suppl1&author=E.+Chuauthor=M.+A.+Callenderauthor=M.+P.+Farrellauthor=J.+C.+Schmitz&title=Thymidylate+synthase+inhibitors+as+anticancer+agents%3A+from+bench+to+bedside"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DE.%26aulast%3DCallender%26aufirst%3DM.%2BA.%26aulast%3DFarrell%26aufirst%3DM.%2BP.%26aulast%3DSchmitz%26aufirst%3DJ.%2BC.%26atitle%3DThymidylate%2520synthase%2520inhibitors%2520as%2520anticancer%2520agents%253A%2520from%2520bench%2520to%2520bedside%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2003%26volume%3D52%26issue%3DSuppl1%26spage%3DS80%26epage%3DS89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shoichet, B. K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span> </span><span class="NLM_article-title">Structure-based discovery of inhibitors of thymidylate synthase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">259</span><span class="NLM_x">, </span> <span class="NLM_fpage">1445</span><span class="NLM_x">–</span> <span class="NLM_lpage">1450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1126%2Fscience.8451640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=8451640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK3sXktF2qsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=259&publication_year=1993&pages=1445-1450&author=B.+K+Shoichetauthor=R.+M.+Stroudauthor=D.+V.+Santiauthor=I.+D.+Kuntzauthor=K.+M.+Perry&title=Structure-based+discovery+of+inhibitors+of+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of inhibitors of thymidylate synthase</span></div><div class="casAuthors">Shoichet, Brian K.; Stroud, Robert M.; Santi, Daniel V.; Kuntz, Irwin D.; Perry, Kathy M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">259</span>
        (<span class="NLM_cas:issue">5100</span>),
    <span class="NLM_cas:pages">1445-50</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A mol. docking computer program (DOCK) was used to screen the Fine Chem. Directory, a database of com. available compds., for mols. that are complementary to thymidylate synthase (TS), a chemotherapeutic target.  Besides retrieving the substrate and several known inhibitors, DOCK proposed putative inhibitors previously unknown to bind to the enzyme.  Three of these compds. inhibited Lactobacillus casei TS at submillimolar concns.  One of these inhibitors, sulisobenzone, crystd. with TS in two configurations that differed from the DOCK-favored geometry: a counterion was bound in the substrate site, which resulted in a 6 to 9 angstrom displacement of the inhibitor.  The structure of the complexes suggested another binding region in the active site that could be exploited.  This region was probed with mols. sterically similar to sulisobenzone, which led to the identification of a family of phenolphthalein analogs that inhibit TS in the 1 to 30 micromolar range.  These inhibitors do not resemble the substrates of the enzyme.  A crystal structure of phenolphthalein with TS shows that it binds in the target site in a configuration that resembles the one suggested by DOCK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcal-kXenW8bVg90H21EOLACvtfcHk0liy9_uxtvhEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktF2qsLc%253D&md5=a2e9380f78c3dd0f213443df91cefb75</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.8451640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8451640%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26atitle%3DStructure-based%2520discovery%2520of%2520inhibitors%2520of%2520thymidylate%2520synthase%26jtitle%3DScience%26date%3D1993%26volume%3D259%26spage%3D1445%26epage%3D1450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Finer-Moore, J. S.; Blaney, J.; Stroud, R. M.</span>,  <span class="citation_source-book">Computational and Structural Approaches to Drug Discovery</span>; <span class="NLM_contrib-group">Stroud, R. M.; Finer-Moore, J. S.</span>, Eds; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge, UK</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>. p  <span class="NLM_fpage">9</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=9&author=J.+S.+Finer-Moore&author=J.+Blaney&author=R.+M.+Stroudauthor=R.+M.+Stroud&author=J.+S.+Finer-Moore&title=Computational+and+Structural+Approaches+to+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26btitle%3DComputational%2520and%2520Structural%2520Approaches%2520to%2520Drug%2520Discovery%26aulast%3DStroud%26aufirst%3DR.%2BM.%26pub%3DRSC%26date%3D2008%26spage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Structure-based design of inhibitors specific for bacterial thymidylate synthase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1607</span><span class="NLM_x">–</span> <span class="NLM_lpage">1617</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9815896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1607-1617&author=T.+J.+Stoutauthor=D.+Tondiauthor=M.+Rinaldiauthor=D.+Barloccoauthor=P.+Pecorariauthor=D.+V.+Santiauthor=I.+D.+Kuntzauthor=R.+M.+Stroudauthor=B.+K.+Shoichetauthor=M.+P.+Costi&title=Structure-based+design+of+inhibitors+specific+for+bacterial+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fbi9815896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815896%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520inhibitors%2520specific%2520for%2520bacterial%2520thymidylate%2520synthase%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D1607%26epage%3D1617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musiu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marangiu, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congiu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loi, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Colla, P.</span><span> </span><span class="NLM_article-title">Phthalein derivatives as a new tool for selectivity in thymidylate synthase inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">2112</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9900016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2112-2124&author=M.+P.+Costiauthor=M.+Rinaldiauthor=D.+Tondiauthor=P.+Pecorariauthor=D.+Barloccoauthor=S.+Ghelliauthor=R.+M.+Stroudauthor=D.+V.+Santiauthor=T.+J.+Stoutauthor=C.+Musiuauthor=E.+M.+Marangiuauthor=A.+Paniauthor=D.+Congiuauthor=G.+A.+Loiauthor=P.+La+Colla&title=Phthalein+derivatives+as+a+new+tool+for+selectivity+in+thymidylate+synthase+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm9900016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9900016%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DMusiu%26aufirst%3DC.%26aulast%3DMarangiu%26aufirst%3DE.%2BM.%26aulast%3DPani%26aufirst%3DA.%26aulast%3DCongiu%26aufirst%3DD.%26aulast%3DLoi%26aufirst%3DG.%2BA.%26aulast%3DLa%2BColla%26aufirst%3DP.%26atitle%3DPhthalein%2520derivatives%2520as%2520a%2520new%2520tool%2520for%2520selectivity%2520in%2520thymidylate%2520synthase%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2112%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title"><i>ortho</i>-Halogen naphthaleins as specific inhibitors of <i>Lactobacillus casei</i> thymidylate synthase. Conformational properties and biological activity</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">951</span><span class="NLM_x">–</span> <span class="NLM_lpage">963</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=951-963&author=S.+Ghelliauthor=M.+Rinaldiauthor=D.+Barloccoauthor=A.+Gelainauthor=P.+Pecorariauthor=D.+Tondiauthor=G.+Rastelliauthor=M.+P.+Costi&title=ortho-Halogen+naphthaleins+as+specific+inhibitors+of+Lactobacillus+casei+thymidylate+synthase.+Conformational+properties+and+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGelain%26aufirst%3DA.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3Dortho-Halogen%2520naphthaleins%2520as%2520specific%2520inhibitors%2520of%2520Lactobacillus%2520casei%2520thymidylate%2520synthase.%2520Conformational%2520properties%2520and%2520biological%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D951%26epage%3D963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Calò, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venturelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">A step further in the discovery of phthalein derivatives as thymidylate synthase inhibitors</span> <span class="citation_source-journal">ARKIVOC</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">396</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=382-396&author=S.+Cal%C3%B2author=D.+Tondiauthor=A.+Venturelliauthor=S.+Ferrariauthor=P.+Pecorariauthor=M.+Rinaldiauthor=S.+Ghelliauthor=M.+P.+Costi&title=A+step+further+in+the+discovery+of+phthalein+derivatives+as+thymidylate+synthase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCal%25C3%25B2%26aufirst%3DS.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DVenturelli%26aufirst%3DA.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DA%2520step%2520further%2520in%2520the%2520discovery%2520of%2520phthalein%2520derivatives%2520as%2520thymidylate%2520synthase%2520inhibitors%26jtitle%3DARKIVOC%26date%3D2004%26spage%3D382%26epage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Neil, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krucinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R.</span><span> </span><span class="NLM_article-title">The structure of <i>Cryptococcus neoformans</i> thymidylate synthase suggests strategies for using target dynamics for species-specific inhibition</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">D61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1320</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D61&publication_year=2005&pages=1320-1334&author=J.+S.+Finer-Mooreauthor=A.+C.+Andersonauthor=R.+H.+O%E2%80%99Neilauthor=M.+P.+Costiauthor=S.+Ferrariauthor=J.+Krucinskiauthor=R.+Stroud&title=The+structure+of+Cryptococcus+neoformans+thymidylate+synthase+suggests+strategies+for+using+target+dynamics+for+species-specific+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DAnderson%26aufirst%3DA.%2BC.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DR.%2BH.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DKrucinski%26aufirst%3DJ.%26aulast%3DStroud%26aufirst%3DR.%26atitle%3DThe%2520structure%2520of%2520Cryptococcus%2520neoformans%2520thymidylate%2520synthase%2520suggests%2520strategies%2520for%2520using%2520target%2520dynamics%2520for%2520species-specific%2520inhibition%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3DD61%26spage%3D1320%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soragni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raniero, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruberto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavazzuti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casolari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span> </span><span class="NLM_article-title">Anti-bacterial agent discovery using thymidylate synthase bio-library screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5958</span><span class="NLM_x">–</span> <span class="NLM_lpage">5968</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051187d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5958-5968&author=M.+P.+Costiauthor=A.+Gelainauthor=D.+Barloccoauthor=S.+Ghelliauthor=F.+Soragniauthor=F.+Ranieroauthor=T.+Rossiauthor=A.+Rubertoauthor=C.+Guillouauthor=A.+Cavazzutiauthor=C.+Casolariauthor=S.+Ferrari&title=Anti-bacterial+agent+discovery+using+thymidylate+synthase+bio-library+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm051187d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051187d%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DGelain%26aufirst%3DA.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DSoragni%26aufirst%3DF.%26aulast%3DRaniero%26aufirst%3DF.%26aulast%3DRossi%26aufirst%3DT.%26aulast%3DRuberto%26aufirst%3DA.%26aulast%3DGuillou%26aufirst%3DC.%26aulast%3DCavazzuti%26aufirst%3DA.%26aulast%3DCasolari%26aufirst%3DC.%26aulast%3DFerrari%26aufirst%3DS.%26atitle%3DAnti-bacterial%2520agent%2520discovery%2520using%2520thymidylate%2520synthase%2520bio-library%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5958%26epage%3D5968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0lgLLO3No-xTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span>ChemFinder Ultra Database 7.0; <span class="NLM_publisher-name">Cambridge Soft Corporation</span>, <a href="htpp://www.cambridgesoft.com" class="extLink">htpp://www.cambridgesoft.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ChemFinder+Ultra+Database+7.0%3B+Cambridge+Soft+Corporation%2C+htpp%3A%2F%2Fwww.cambridgesoft.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DChemFinder%2520Ultra%2520Database%25207.0%26pub%3DCambridge%2520Soft%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span>ACDlab 6.0 Software; <span class="NLM_publisher-name">Cambridge Soft Corporation</span>, <a href="htpp://www.cambridgesoft.com" class="extLink">htpp://www.cambridgesoft.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ACDlab+6.0+Software%3B+Cambridge+Soft+Corporation%2C+htpp%3A%2F%2Fwww.cambridgesoft.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACDlab%25206.0%2520Software%26pub%3DCambridge%2520Soft%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lgLLO3No-xTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montfort, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, M. O</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Atomic structure of thymidylate synthase: target for rational drug design</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=448-455&author=L.+W.+Hardyauthor=J.+S.+Finer-Mooreauthor=W.+R.+Montfortauthor=M.+O+Jonesauthor=D.+V.+Santiauthor=R.+M.+Stroud&title=Atomic+structure+of+thymidylate+synthase%3A+target+for+rational+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DMontfort%26aufirst%3DW.%2BR.%26aulast%3DJones%26aufirst%3DM.%2BO%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DAtomic%2520structure%2520of%2520thymidylate%2520synthase%253A%2520target%2520for%2520rational%2520drug%2520design%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D448%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Finer-Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R.</span><span> </span><span class="NLM_article-title">Refined structures of substrate-bound and phosphate-bound thymidylate synthase from <i>Lactobacillus casei</i></span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">232</span><span class="NLM_x">, </span> <span class="NLM_fpage">1101</span><span class="NLM_x">–</span> <span class="NLM_lpage">1116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=232&publication_year=1993&pages=1101-1116&author=J.+Finer-Mooreauthor=E.+B.+Faumanauthor=P.+G.+Fosterauthor=K.+M.+Perryauthor=D.+V.+Santiauthor=R.+Stroud&title=Refined+structures+of+substrate-bound+and+phosphate-bound+thymidylate+synthase+from+Lactobacillus+casei"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DFoster%26aufirst%3DP.%2BG.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStroud%26aufirst%3DR.%26atitle%3DRefined%2520structures%2520of%2520substrate-bound%2520and%2520phosphate-bound%2520thymidylate%2520synthase%2520from%2520Lactobacillus%2520casei%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D232%26spage%3D1101%26epage%3D1116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losasso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Sequence-Based identification of specific drug target regions in the thymidylate synthase enzyme family</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">401</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=392-401&author=S.+Ferrariauthor=V.+Losassoauthor=M.+P.+Costi&title=Sequence-Based+identification+of+specific+drug+target+regions+in+the+thymidylate+synthase+enzyme+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DLosasso%26aufirst%3DV.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DSequence-Based%2520identification%2520of%2520specific%2520drug%2520target%2520regions%2520in%2520the%2520thymidylate%2520synthase%2520enzyme%2520family%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D392%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R.</span><span> </span><span class="NLM_article-title">Inhibitor specificity via protein dynamics: insights from the design of antibacterial agents targeted against thymidylate synthase</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1183</span><span class="NLM_x">–</span> <span class="NLM_lpage">1193</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2Fj.chembiol.2003.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=14700626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVWis7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=1183-1193&author=S.+Ferrariauthor=M.+P.+Costiauthor=R.+Wade&title=Inhibitor+specificity+via+protein+dynamics%3A+insights+from+the+design+of+antibacterial+agents+targeted+against+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Specificity via Protein Dynamics Insights from the Design of Antibacterial Agents Targeted Against Thymidylate Synthase</span></div><div class="casAuthors">Ferrari, Stefania; Costi, Paola M.; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1183-1193</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Structure-based drug design of species-specific inhibitors generally exploits structural differences in proteins from different organisms.  Here, we demonstrate how achieving specificity can be aided by targeting differences in the dynamics of proteins.  Thymidylate synthase (TS) is a good target for anticancer agents and a potential target for antibacterial agents.  Most inhibitors are folate-analogs that bind at the folate binding site and are not species specific.  In contrast, α156 is not a folate-analog and is specific for bacterial TS; it has been shown crystallog. to bind in a nonconserved binding site.  Docking calcns. and crystal structure-based estn. of the essential dynamics of TSs from five different species show that differences in the dynamics of TSs make the active site more accessible to α156 in the prokaryotic than in the eukaryotic TSs and thereby enhance the specificity of α156.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiW86dIS4vLVg90H21EOLACvtfcHk0liW7GZg7mrIqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVWis7zJ&md5=5bfe50faeb8b9c14664763bbc74fc670</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2003.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2003.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DWade%26aufirst%3DR.%26atitle%3DInhibitor%2520specificity%2520via%2520protein%2520dynamics%253A%2520insights%2520from%2520the%2520design%2520of%2520antibacterial%2520agents%2520targeted%2520against%2520thymidylate%2520synthase%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D1183%26epage%3D1193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kealey, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Purification methods for recombinant <i>Lactobacillus casei</i> thymidylate synthase and mutants: a general, automated procedure</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">380</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2FS1046-5928%2805%2980039-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1992&pages=380-385&author=J.+T.+Kealeyauthor=D.+V.+Santi&title=Purification+methods+for+recombinant+Lactobacillus+casei+thymidylate+synthase+and+mutants%3A+a+general%2C+automated+procedure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1046-5928%2805%2980039-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1046-5928%252805%252980039-7%26sid%3Dliteratum%253Aachs%26aulast%3DKealey%26aufirst%3DJ.%2BT.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DPurification%2520methods%2520for%2520recombinant%2520Lactobacillus%2520casei%2520thymidylate%2520synthase%2520and%2520mutants%253A%2520a%2520general%252C%2520automated%2520procedure%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D1992%26volume%3D4%26spage%3D380%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span> </span><span class="NLM_article-title">Properties of a defined mutant of <i>Escherichia coli</i> thymidylate synthase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">7620</span><span class="NLM_x">–</span> <span class="NLM_lpage">7627</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=7620-7627&author=G.+F.+Maleyauthor=F.+Maley&title=Properties+of+a+defined+mutant+of+Escherichia+coli+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DMaley%26aufirst%3DF.%26atitle%3DProperties%2520of%2520a%2520defined%2520mutant%2520of%2520Escherichia%2520coli%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D7620%26epage%3D7627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Pedersen-Lane, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span> </span><span class="NLM_article-title">High-level expression of human thymidylate synthase</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">256</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1006%2Fprep.1997.0750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=9226722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2sXksFajs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1997&pages=256-262&author=J.+Pedersen-Laneauthor=G.+F.+Maleyauthor=E.+Chuauthor=F.+Maley&title=High-level+expression+of+human+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">High-level expression of human thymidylate synthase</span></div><div class="casAuthors">Pedersen-Lane, Joan; Maley, Gladys F.; Chu, Edward; Maley, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">256-262</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A method is presented for expressing human thymidylate synthase (TS) to the extent of 25-30% of the protein in Escherichia coli.  By this procedure, 200-400 mg of pure enzyme can be obtained from a 2-L culture of cells.  The key to the level of expression appears to be related to the conversion of purine bases in the 3rd, 4th, and 5th codons of the TS cDNA to thymine, without altering the encoded protein product.  Conversion of the penultimate proline to a leucine did not diminish expression, but while the isolated native enzyme contained only proline on its amino-terminal end, the mutated enzyme contained methionine on its amino terminus.  By contrast, the expression of the unmodified TS cDNA represented only 0.1-0.2% of the total cellular protein.  Unlike recombinant rat and human TSs, the resp. enzymes purified to homogeneity from eukaryotic cells were blocked at the amino ends and possessed 2- to 4-fold lower specific activities.  To det. at what level the impairment of expression occurred, an in vitro transcription, translation system was employed and the results showed that while transcription was unaffected, the translation of native TS mRNA was reduced by ≥20-fold relative to modified TS mRNA using a rabbit reticulocyte translation system.  Thus, it appears that at least for the TS gene, expression is greatly influenced by the GC content of the 5' coding region of the gene in both prokaryote and eukaryote systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujd06OIYeabVg90H21EOLACvtfcHk0lg--GtecQUYtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFajs7w%253D&md5=834d2d0186feb013bda20139c896e1b2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1006%2Fprep.1997.0750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fprep.1997.0750%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen-Lane%26aufirst%3DJ.%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DChu%26aufirst%3DE.%26aulast%3DMaley%26aufirst%3DF.%26atitle%3DHigh-level%2520expression%2520of%2520human%2520thymidylate%2520synthase%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D1997%26volume%3D10%26spage%3D256%26epage%3D262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Pogolotti, A. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danenberg, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Kinetics and mechanism of interaction of 10-propargyl-5,8-dideazafolate with thymidylate synthase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">482</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00154a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=478-482&author=A.+L.+Pogolottiauthor=P.+V.+Danenbergauthor=D.+V.+Santi&title=Kinetics+and+mechanism+of+interaction+of+10-propargyl-5%2C8-dideazafolate+with+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm00154a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00154a009%26sid%3Dliteratum%253Aachs%26aulast%3DPogolotti%26aufirst%3DA.%2BL.%26aulast%3DDanenberg%26aufirst%3DP.%2BV.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DKinetics%2520and%2520mechanism%2520of%2520interaction%2520of%252010-propargyl-5%252C8-dideazafolate%2520with%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D478%26epage%3D482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Segel, I. H.</span><span> </span><span class="NLM_article-title">Enzyme Kinetics</span>.  <span class="citation_source-book">Behaviour and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</span>; <span class="NLM_publisher-name">John Wiley and Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1975</span><span class="NLM_x">, </span>p  <span class="NLM_fpage">105</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&pages=105&author=I.+H.+Segel&title=Behaviour+and+Analysis+of+Rapid+Equilibrium+and+Steady-State+Enzyme+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26atitle%3DEnzyme%2520Kinetics%26btitle%3DBehaviour%2520and%2520Analysis%2520of%2520Rapid%2520Equilibrium%2520and%2520Steady-State%2520Enzyme%2520Systems%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D1975%26spage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Cavazzuti, A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paglietti, G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamarro, F</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piras, S</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriga, M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alleca, S</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, P</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tolloch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Discovery of potent pteridine reductase inhibitors ot guide antiparasite drug development</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1448-1453&author=A+Cavazzutiauthor=G+Pagliettiauthor=W.+N.+Hunterauthor=F+Gamarroauthor=S+Pirasauthor=M+Lorigaauthor=S+Allecaauthor=P+Coronaauthor=K+McLuskeyauthor=L.+Tollochauthor=F.+Gibelliniauthor=S.+Ferrariauthor=M.+P.+Costi&title=Discovery+of+potent+pteridine+reductase+inhibitors+ot+guide+antiparasite+drug+development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCavazzuti%26aufirst%3DA%26aulast%3DPaglietti%26aufirst%3DG%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGamarro%26aufirst%3DF%26aulast%3DPiras%26aufirst%3DS%26aulast%3DLoriga%26aufirst%3DM%26aulast%3DAlleca%26aufirst%3DS%26aulast%3DCorona%26aufirst%3DP%26aulast%3DMcLuskey%26aufirst%3DK%26aulast%3DTolloch%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520potent%2520pteridine%2520reductase%2520inhibitors%2520ot%2520guide%2520antiparasite%2520drug%2520development%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Jaspard, E.</span><span> </span><span class="NLM_article-title">Role of protein–solvent interactions in refolding: effects of cosolvent additives on the renaturation of porcine pancreatic elastase at various pHs</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">220</span><span class="NLM_x">–</span> <span class="NLM_lpage">228</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2000&pages=220-228&author=E.+Jaspard&title=Role+of+protein%E2%80%93solvent+interactions+in+refolding%3A+effects+of+cosolvent+additives+on+the+renaturation+of+porcine+pancreatic+elastase+at+various+pHs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJaspard%26aufirst%3DE.%26atitle%3DRole%2520of%2520protein%25E2%2580%2593solvent%2520interactions%2520in%2520refolding%253A%2520effects%2520of%2520cosolvent%2520additives%2520on%2520the%2520renaturation%2520of%2520porcine%2520pancreatic%2520elastase%2520at%2520various%2520pHs%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D15%26spage%3D220%26epage%3D228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bhattacharjya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaram, P.</span><span> </span><span class="NLM_article-title">Effects of organic solvents on protein structures: observation of a structured helical core in hen egg-white lysozyme in aqueous dimethylsulfoxide</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">492</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1997&pages=492-507&author=S.+Bhattacharjyaauthor=P.+Balaram&title=Effects+of+organic+solvents+on+protein+structures%3A+observation+of+a+structured+helical+core+in+hen+egg-white+lysozyme+in+aqueous+dimethylsulfoxide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharjya%26aufirst%3DS.%26aulast%3DBalaram%26aufirst%3DP.%26atitle%3DEffects%2520of%2520organic%2520solvents%2520on%2520protein%2520structures%253A%2520observation%2520of%2520a%2520structured%2520helical%2520core%2520in%2520hen%2520egg-white%2520lysozyme%2520in%2520aqueous%2520dimethylsulfoxide%26jtitle%3DProteins%26date%3D1997%26volume%3D29%26spage%3D492%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Leslie, A. G. W.</span> <span class="citation_source-book">Crystallographic Computing V</span>; <span class="NLM_contrib-group">Moras, D.; Podjarny, A. D.; Thierry, J.-C.</span>, Eds.; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford</span>,<span class="NLM_x"> </span><span class="NLM_year">1991</span><span class="NLM_x">; </span>pp <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1991&pages=50-61&author=A.+G.+W.+Leslieauthor=D.+Moras&author=A.+D.+Podjarny&author=J.-C.+Thierry&title=Crystallographic+Computing+V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26btitle%3DCrystallographic%2520Computing%2520V%26aulast%3DMoras%26aufirst%3DD.%26pub%3DOxford%2520University%2520Press%26date%3D1991%26spage%3D50%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Collaborative Computational Project, No. 4</span><span class="NLM_x">. </span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project%2C+No.+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">SCALA, continuous scaling program</span>.  <span class="citation_source-journal">Joint CCP4 ESF-EACBM Newsletter</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1997&pages=22-24&author=P.+R.+Evans&title=SCALA%2C+continuous+scaling+program"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DSCALA%252C%2520continuous%2520scaling%2520program%26jtitle%3DJoint%2520CCP4%2520ESF-EACBM%2520Newsletter%26date%3D1997%26volume%3D33%26spage%3D22%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Vagin, A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgYWPJuan7JYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0li73IwwZn8LQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Perrakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R. J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamzin, V. S.</span><span> </span><span class="NLM_article-title">Automated protein model building combined with iterative structure refinement</span> <span class="citation_source-journal">Nature Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1038%2F8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=10331874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=458-463&author=A.+Perrakisauthor=R.+J.+H.+Morrisauthor=V.+S.+Lamzin&title=Automated+protein+model+building+combined+with+iterative+structure+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein model building combined with iterative structure refinement</span></div><div class="casAuthors">Perrakis, Anastassis; Morris, Richard; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-463</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">In protein crystallog., much time and effort are often required to trace an initial model from an interpretable electron d. map and to refine it until it best agrees with the crystallog. data.  Here, we present a method to build and refine a protein model automatically and without user intervention, starting from diffraction data extending to resoln. higher than 2.3 Å and reasonable ests. of crystallog. phases.  The method is based on an iterative procedure that describes the electron d. map as a set of unconnected atoms and then searches for protein-like patterns.  Automatic pattern recognition (model building) combined with refinement, allows a structural model to be obtained reliably within a few CPU hours.  We demonstrate the power of the method with examples of a few recently solved structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbcrkUoPTzsLVg90H21EOLACvtfcHk0li73IwwZn8LQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D&md5=d12fc80eb73e1faebee29e2f6635c86d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2F8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F8263%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%2BJ.%2BH.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DAutomated%2520protein%2520model%2520building%2520combined%2520with%2520iterative%2520structure%2520refinement%26jtitle%3DNature%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D458%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">McRee, D. E.</span><span> </span><span class="NLM_article-title">XtalView/Xfit—a versatile program for manipulating atomic coordinates and electron density</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">156</span><span class="NLM_x">–</span> <span class="NLM_lpage">165</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1999&pages=156-165&author=D.+E.+McRee&title=XtalView%2FXfit%E2%80%94a+versatile+program+for+manipulating+atomic+coordinates+and+electron+density"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcRee%26aufirst%3DD.%2BE.%26atitle%3DXtalView%252FXfit%25E2%2580%2594a%2520versatile%2520program%2520for%2520manipulating%2520atomic%2520coordinates%2520and%2520electron%2520density%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D125%26spage%3D156%26epage%3D165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span>UCSF Chimera, Production Version 1 Build 2038 2004/10/18,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UCSF+Chimera%2C+Production+Version+1+Build+2038+2004%2F10%2F18%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DUCSF%2520Chimera%252C%2520Production%2520Version%25201%2520Build%25202038%25202004%252F10%252F18%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pasquale Linciano, Cecilia Pozzi, Lucia dello Iacono, Flavio di Pisa, Giacomo Landi, Alessio Bonucci, Sheraz Gul, Maria Kuzikov, Bernhard Ellinger, Gesa Witt, Nuno Santarem, Catarina Baptista, Caio Franco, Carolina B. Moraes, Wolfgang Müller, Ulrike Wittig, Rosaria Luciani, Antony Sesenna, Antonio Quotadamo, Stefania Ferrari, Ina Pöhner, Anabela Cordeiro-da-Silva, Stefano Mangani, Luca Costantino, <span class="NLM_string-name hlFld-ContribAuthor">Maria Paola Costi</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancement of Benzothiazoles as Pteridine Reductase-1 Inhibitors for the Treatment of Trypanosomatidic Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3989-4012. <a href="https://doi.org/10.1021/acs.jmedchem.8b02021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b02021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b02021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b02021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEnhancement%252Bof%252BBenzothiazoles%252Bas%252BPteridine%252BReductase-1%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BTrypanosomatidic%252BInfections%26aulast%3DLinciano%26aufirst%3DPasquale%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D26122018%26date%3D09042019%26date%3D25032019%26volume%3D62%26issue%3D8%26spage%3D3989%26epage%3D4012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yu-Chao Yuan, Christian Bruneau, Vincent Dorcet, Thierry Roisnel, <span class="NLM_string-name hlFld-ContribAuthor">Rafael Gramage-Doria</span>. </span><span class="cited-content_cbyCitation_article-title">Ru-Catalyzed Selective C–H Bond Hydroxylation of Cyclic Imides. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (4)
                                     , 1898-1907. <a href="https://doi.org/10.1021/acs.joc.8b02899" title="DOI URL">https://doi.org/10.1021/acs.joc.8b02899</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b02899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b02899%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DRu-Catalyzed%252BSelective%252BC%2525E2%252580%252593H%252BBond%252BHydroxylation%252Bof%252BCyclic%252BImides%26aulast%3DYuan%26aufirst%3DYu-Chao%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13112018%26date%3D25012019%26date%3D10012019%26volume%3D84%26issue%3D4%26spage%3D1898%26epage%3D1907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Samuele  Calò</span>, <span class="hlFld-ContribAuthor ">Rosalida  Leone</span>, <span class="hlFld-ContribAuthor ">Rosaria  Luciani</span>, <span class="hlFld-ContribAuthor ">Luca  Costantino</span>, <span class="hlFld-ContribAuthor ">Susan  Sammak</span>, <span class="hlFld-ContribAuthor ">Flavio  Di Pisa</span>, <span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>, and <span class="hlFld-ContribAuthor ">M. Paola  Costi</span>  . </span><span class="cited-content_cbyCitation_article-title">2′-Deoxyuridine 5′-Monophosphate Substrate Displacement in Thymidylate Synthase through 6-Hydroxy-2H-naphtho[1,8-bc]furan-2-one Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (22)
                                     , 9356-9360. <a href="https://doi.org/10.1021/jm4014086" title="DOI URL">https://doi.org/10.1021/jm4014086</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm4014086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm4014086%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D2%2525E2%252580%2525B2-Deoxyuridine%252B5%2525E2%252580%2525B2-Monophosphate%252BSubstrate%252BDisplacement%252Bin%252BThymidylate%252BSynthase%252Bthrough%252B6-Hydroxy-2H-naphtho%25255B1%25252C8-bc%25255Dfuran-2-one%252BDerivatives%26aulast%3DFerrari%26aufirst%3DStefania%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D12092013%26date%3D13112013%26date%3D27112013%26date%3D22102013%26volume%3D56%26issue%3D22%26spage%3D9356%26epage%3D9360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Rosaria  Luciani</span>, <span class="hlFld-ContribAuthor ">Giusy  Tassone</span>, <span class="hlFld-ContribAuthor ">Maria  Costi</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Comparison of Enterococcus faecalis and Human Thymidylate Synthase Complexes with the Substrate dUMP and Its Analogue FdUMP Provides Hints about Enzyme Conformational Variabilities. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (7)
                                     , 1257. <a href="https://doi.org/10.3390/molecules24071257" title="DOI URL">https://doi.org/10.3390/molecules24071257</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24071257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24071257%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DStructural%252BComparison%252Bof%252BEnterococcus%252Bfaecalis%252Band%252BHuman%252BThymidylate%252BSynthase%252BComplexes%252Bwith%252Bthe%252BSubstrate%252BdUMP%252Band%252BIts%252BAnalogue%252BFdUMP%252BProvides%252BHints%252Babout%252BEnzyme%252BConformational%252BVariabilities%26aulast%3DPozzi%26aufirst%3DCecilia%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D7%26spage%3D1257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Ludovica  Lopresti</span>, <span class="hlFld-ContribAuthor ">Giusy  Tassone</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Methyltransferases in Human Pathogenic Bacteria: Insights into Thymidylate Synthase (TS) and Flavin-Dependent TS (FDTS). </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (8)
                                     , 1638. <a href="https://doi.org/10.3390/molecules24081638" title="DOI URL">https://doi.org/10.3390/molecules24081638</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24081638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24081638%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DTargeting%252BMethyltransferases%252Bin%252BHuman%252BPathogenic%252BBacteria%25253A%252BInsights%252Binto%252BThymidylate%252BSynthase%252B%252528TS%252529%252Band%252BFlavin-Dependent%252BTS%252B%252528FDTS%252529%26aulast%3DPozzi%26aufirst%3DCecilia%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D8%26spage%3D1638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hua  Cheng</span>, <span class="hlFld-ContribAuthor ">Yan  Fu</span>, <span class="hlFld-ContribAuthor ">Qing  Chang</span>, <span class="hlFld-ContribAuthor ">Ni  Zhang</span>, <span class="hlFld-ContribAuthor ">Mengwei  Bu</span>, <span class="hlFld-ContribAuthor ">Yan  Niu</span>, <span class="hlFld-ContribAuthor ">Qiongyou  Wu</span>, <span class="hlFld-ContribAuthor ">Cheng  Chen</span>, <span class="hlFld-ContribAuthor ">Francis  Verpoort</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, biochemical evaluation and computational simulations of new cytochrome bc1 complex inhibitors based on N-(4-aryloxyphenyl) phthalimides. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2018,</strong> <em>29 </em>
                                    (12)
                                     , 1897-1900. <a href="https://doi.org/10.1016/j.cclet.2018.10.008" title="DOI URL">https://doi.org/10.1016/j.cclet.2018.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2018.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2018.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DSynthesis%25252C%252Bbiochemical%252Bevaluation%252Band%252Bcomputational%252Bsimulations%252Bof%252Bnew%252Bcytochrome%252Bbc1%252Bcomplex%252Binhibitors%252Bbased%252Bon%252BN-%2525284-aryloxyphenyl%252529%252Bphthalimides%26aulast%3DCheng%26aufirst%3DHua%26date%3D2018%26volume%3D29%26issue%3D12%26spage%3D1897%26epage%3D1900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ram W.  Sabnis</span>. </span><span class="cited-content_cbyCitation_article-title">Developments in the chemistry and applications of phthalein dyes. Part 2: biological applications. </span><span class="cited-content_cbyCitation_journal-name">Coloration Technology</span><span> <strong>2018,</strong> <em>134 </em>
                                    (5)
                                     , 347-363. <a href="https://doi.org/10.1111/cote.12353" title="DOI URL">https://doi.org/10.1111/cote.12353</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cote.12353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcote.12353%26sid%3Dliteratum%253Aachs%26jtitle%3DColoration%2520Technology%26atitle%3DDevelopments%252Bin%252Bthe%252Bchemistry%252Band%252Bapplications%252Bof%252Bphthalein%252Bdyes.%252BPart%252B2%25253A%252Bbiological%252Bapplications%26aulast%3DSabnis%26aufirst%3DRam%2BW.%26date%3D2018%26date%3D2018%26volume%3D134%26issue%3D5%26spage%3D347%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renyi  Shi</span>, <span class="hlFld-ContribAuthor ">Fan  Liao</span>, <span class="hlFld-ContribAuthor ">Huiying  Niu</span>, <span class="hlFld-ContribAuthor ">Aiwen  Lei</span>. </span><span class="cited-content_cbyCitation_article-title">Selective formation of phthalimides from amines, aldehydes and CO by Pd-catalyzed oxidative C–H aminocarbonylation. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2018,</strong> <em>5 </em>
                                    (12)
                                     , 1957-1961. <a href="https://doi.org/10.1039/C8QO00282G" title="DOI URL">https://doi.org/10.1039/C8QO00282G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8QO00282G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8QO00282G%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DSelective%252Bformation%252Bof%252Bphthalimides%252Bfrom%252Bamines%25252C%252Baldehydes%252Band%252BCO%252Bby%252BPd-catalyzed%252Boxidative%252BC%2525E2%252580%252593H%252Baminocarbonylation%26aulast%3DShi%26aufirst%3DRenyi%26date%3D2018%26date%3D2018%26volume%3D5%26issue%3D12%26spage%3D1957%26epage%3D1961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Jarmuła</span>, <span class="hlFld-ContribAuthor ">Piotr  Wilk</span>, <span class="hlFld-ContribAuthor ">Piotr  Maj</span>, <span class="hlFld-ContribAuthor ">Jan  Ludwiczak</span>, <span class="hlFld-ContribAuthor ">Anna  Dowierciał</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Banaszak</span>, <span class="hlFld-ContribAuthor ">Wojciech  Rypniewski</span>, <span class="hlFld-ContribAuthor ">Joanna  Cieśla</span>, <span class="hlFld-ContribAuthor ">Magdalena  Dąbrowska</span>, <span class="hlFld-ContribAuthor ">Tomasz  Frączyk</span>, <span class="hlFld-ContribAuthor ">Agnieszka K.  Bronowska</span>, <span class="hlFld-ContribAuthor ">Jakub  Jakowiecki</span>, <span class="hlFld-ContribAuthor ">Sławomir  Filipek</span>, <span class="hlFld-ContribAuthor ">Wojciech  Rode</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structures of nematode (parasitic T. spiralis and free living C. elegans ), compared to mammalian, thymidylate synthases (TS). Molecular docking and molecular dynamics simulations in search for nematode-specific inhibitors of TS. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2017,</strong> <em>77 </em>, 33-50. <a href="https://doi.org/10.1016/j.jmgm.2017.08.008" title="DOI URL">https://doi.org/10.1016/j.jmgm.2017.08.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2017.08.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2017.08.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DCrystal%252Bstructures%252Bof%252Bnematode%252B%252528parasitic%252BT.%252Bspiralis%252Band%252Bfree%252Bliving%252BC.%252Belegans%252B%252529%25252C%252Bcompared%252Bto%252Bmammalian%25252C%252Bthymidylate%252Bsynthases%252B%252528TS%252529.%252BMolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bin%252Bsearch%252Bfor%252Bnematode-specific%252Binhibitors%252Bof%252BTS%26aulast%3DJarmu%25C5%2582a%26aufirst%3DAdam%26date%3D2017%26volume%3D77%26spage%3D33%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kinga  Nyíri</span>, <span class="hlFld-ContribAuthor ">Beáta G.  Vértessy</span>. </span><span class="cited-content_cbyCitation_article-title">Perturbation of genome integrity to fight pathogenic microorganisms. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2017,</strong> <em>1861 </em>
                                    (1)
                                     , 3593-3612. <a href="https://doi.org/10.1016/j.bbagen.2016.05.024" title="DOI URL">https://doi.org/10.1016/j.bbagen.2016.05.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2016.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2016.05.024%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DPerturbation%252Bof%252Bgenome%252Bintegrity%252Bto%252Bfight%252Bpathogenic%252Bmicroorganisms%26aulast%3DNy%25C3%25ADri%26aufirst%3DKinga%26date%3D2017%26volume%3D1861%26issue%3D1%26spage%3D3593%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessia  Catalano</span>, <span class="hlFld-ContribAuthor ">Rosaria  Luciani</span>, <span class="hlFld-ContribAuthor ">Alessia  Carocci</span>, <span class="hlFld-ContribAuthor ">Debora  Cortesi</span>, <span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Chiara  Borsari</span>, <span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">X-ray crystal structures of Enterococcus faecalis thymidylate synthase with folate binding site inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>123 </em>, 649-664. <a href="https://doi.org/10.1016/j.ejmech.2016.07.066" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.07.066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.07.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.07.066%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DX-ray%252Bcrystal%252Bstructures%252Bof%252BEnterococcus%252Bfaecalis%252Bthymidylate%252Bsynthase%252Bwith%252Bfolate%252Bbinding%252Bsite%252Binhibitors%26aulast%3DCatalano%26aufirst%3DAlessia%26date%3D2016%26volume%3D123%26spage%3D649%26epage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazuhiro  Kobayashi</span>, <span class="hlFld-ContribAuthor ">Kota  Matsumoto</span>, <span class="hlFld-ContribAuthor ">Yuu  Shirai</span>, <span class="hlFld-ContribAuthor ">Hiroaki  Ishikawa</span>, <span class="hlFld-ContribAuthor ">Hiroki  Inouchi</span>, <span class="hlFld-ContribAuthor ">Miyuki  Tanmatsu</span>. </span><span class="cited-content_cbyCitation_article-title">Generation of Aryl(2-lithiophenyl)methanone
              O
              -Methyl Oximes and Their Use for the Synthesis of
              N
              -(3-Alkyl-1-aryl- or 1,3-diaryl-1
              H
              -isoindol-1-yl)-
              O
              -methylhydroxylamines
              via
              the Reaction with Nitriles. </span><span class="cited-content_cbyCitation_journal-name">Helvetica Chimica Acta</span><span> <strong>2014,</strong> <em>97 </em>
                                    (7)
                                     , 915-922. <a href="https://doi.org/10.1002/hlca.201400067" title="DOI URL">https://doi.org/10.1002/hlca.201400067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/hlca.201400067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fhlca.201400067%26sid%3Dliteratum%253Aachs%26jtitle%3DHelvetica%2520Chimica%2520Acta%26atitle%3DGeneration%252Bof%252BAryl%2525282-lithiophenyl%252529methanone%252BO%252B-Methyl%252BOximes%252Band%252BTheir%252BUse%252Bfor%252Bthe%252BSynthesis%252Bof%252BN%252B-%2525283-Alkyl-1-aryl-%252Bor%252B1%25252C3-diaryl-1%252BH%252B-isoindol-1-yl%252529-%252BO%252B-methylhydroxylamines%252Bvia%252Bthe%252BReaction%252Bwith%252BNitriles%26aulast%3DKobayashi%26aufirst%3DKazuhiro%26date%3D2014%26date%3D2014%26volume%3D97%26issue%3D7%26spage%3D915%26epage%3D922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Debora  Cortesi</span>, <span class="hlFld-ContribAuthor ">Rosaria  Luciani</span>, <span class="hlFld-ContribAuthor ">Robert M.  Stroud</span>, <span class="hlFld-ContribAuthor ">Alessia  Catalano</span>, <span class="hlFld-ContribAuthor ">Maria Paola  Costi</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">The structure of
              Enterococcus faecalis
              thymidylate synthase provides clues about folate bacterial metabolism. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section D Biological Crystallography</span><span> <strong>2012,</strong> <em>68 </em>
                                    (9)
                                     , 1232-1241. <a href="https://doi.org/10.1107/S0907444912026236" title="DOI URL">https://doi.org/10.1107/S0907444912026236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S0907444912026236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS0907444912026236%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520D%2520Biological%2520Crystallography%26atitle%3DThe%252Bstructure%252Bof%252BEnterococcus%252Bfaecalis%252Bthymidylate%252Bsynthase%252Bprovides%252Bclues%252Babout%252Bfolate%252Bbacterial%252Bmetabolism%26aulast%3DPozzi%26aufirst%3DCecilia%26date%3D2012%26date%3D2012%26volume%3D68%26issue%3D9%26spage%3D1232%26epage%3D1241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Retrosynthetic analysis of phenolphthalein (PTH).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds <b>2</b>–<b>9</b> were identified for further synthetic elaboration through ChemFinder database exploration based on 2-benzofuran-1,3-dione (<b>2</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) method 1: the opportune amine, toluene, EtOH, reflux, 15 h; method 2: the opportune amine, acetic acid, reflux, 5 h. (b) NH<sub>2</sub>CH<sub>3</sub>, toluene, EtOH, yield 47%; (c) NBS, benzoyl peroxide, CCl<sub>4</sub>, hν, yield 64%; (d) the opportune amine, DMF, 75 °C, 15 h.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Superposition of the two crystal structures of LcTS-dUMP–<b>6A</b>. (B) Superimposition of the two crystal structures of LcTS-dUMP–<b>12A</b>. Compound <b>12A</b> was modeled in two possible bound conformations in one crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>; the two conformations are shown with C in pink in the picture). (C) Crystal structure of LcTS-dUMP–<b>8A</b>. The protein is represented as a ribbon; ligands are represented as sticks that are color-coded according to the atom (N in blue, O in red, C in pink, yellow, or orange).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding modes of compounds as observed in the crystal structures. (A) <b>6A</b> bound in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A">3C0A</a> or (B) in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06">3C06</a>. (C and D) Compound <b>12A</b> is shown in its (C) first binding mode and (D) second binding mode as modeled in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX">3BYX</a>. (E) Compound <b>12A</b> is shown in its binding mode in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0">3BZ0</a> crystal structure; (F) compound <b>8A</b> bound in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ">3BNZ</a> crystal structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Superposition of the crystal structures for LcTS-dUMP–<b>20A</b>, LcTS-dUMP–<b>23A</b>, and LcTS-dUMP–<b>24A</b>. (B) Detailed interactions of the phenyl moiety of ligand <b>20A</b> with LcTS. (C) Detailed interaction of the phenyl moiety of ligand <b>23A</b> with LcTS. (D) Detailed interaction of the phenyl moiety of ligand <b>24A</b> with LcTS. All structures are color-coordinated, with oxygen in red, nitrogen in blue, sulfur in yellow, and phosphate in cyan. Carbon atoms are sky blue, purple, and green in LcTS-dUMP–<b>20A</b>, LcTS-dUMP–<b>23A</b>, and LcTS-dUMP–<b>24A</b>, respectively. Hydrogen bonds are indicated with black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Details of the LcTS–<b>8A</b> complex. Colored in magenta is the Pro53-Lys92 sequence. (B) Structure comparison of LcTS–<b>20A</b> (in magenta) and LcTS–<b>23A</b> (in cyan) complexes. For clarity purposes, only Arg23 residue and Pro21-Gly27 and Pro105-Glu113 trace sequences of LcTS–<b>23A</b> complex are shown to highlight the conformational differences between these two complexes. (C) Protein sequence alignment of EfTS, hTS and LcTS; written in white on a black background are identical residues among the three sequences; written in white on a gray background are identical residues between two out of three sequences. The magenta rectangle highlights the sequence Pro53-Lys92 (LcTS sequences number) that is more conserved between the two bacterial TS with respect to hTS; the green circles indicate the residues interacting with compound <b>8A</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Library A, Compounds <b>1A</b>–<b>24A</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/medium/jm-2011-005018_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Library B, Compounds <b>1B</b>–<b>23B</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-15/jm2005018/production/images/large/jm-2011-005018_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2005018&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36081" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36081" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Carreras, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">The catalytic mechanism and structure of thymidylate synthase</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">762</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1146%2Fannurev.bi.64.070195.003445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=7574499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2MXmsl2qtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1995&pages=721-762&author=C.+W.+Carrerasauthor=D.+V.+Santi&title=The+catalytic+mechanism+and+structure+of+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The catalytic mechanism and structure of thymidylate synthase</span></div><div class="casAuthors">Carreras, Christopher W.; Santi, Daniel V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">721-62</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review, with 193 refs.  Thymidylate synthase (TS, EC 2.1.1.45) catalyzes the reductive methylation of dUMP by CH2H4folate to produce dTMP and H2folate.  Knowledge of the catalytic mechanism and structure of TS has increased substantially over recent years.  Major advances were derived from crystal structures of TS bound to various ligands, the ability to overexpress TS in heterologous hosts, and the numerous mutants that have been prepd. and analyzed.  These advances, coupled with previous knowledge, have culminated in an in-depth understanding of many important mol. details of the reaction.  The authors review aspects of TS catalysis that are most pertinent to understanding the current status of the structure and catalytic mechanism of the enzyme.  Included is a discussion of available sources and assays for TS, a description of the enzyme's chem. mechanism and crystal structure, and a summary of data obtained from mutagenesis expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO-xv6vKzvm7Vg90H21EOLACvtfcHk0lgz82aJAXzEsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmsl2qtrg%253D&md5=3fa2b583e048196a12193b27f9533770</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.64.070195.003445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.64.070195.003445%26sid%3Dliteratum%253Aachs%26aulast%3DCarreras%26aufirst%3DC.%2BW.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DThe%2520catalytic%2520mechanism%2520and%2520structure%2520of%2520thymidylate%2520synthase%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1995%26volume%3D64%26spage%3D721%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Chu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callender, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrell, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitz, J. C.</span><span> </span><span class="NLM_article-title">Thymidylate synthase inhibitors as anticancer agents: from bench to bedside</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S80</span><span class="NLM_x">–</span> <span class="NLM_lpage">S89</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=S80-S89&issue=Suppl1&author=E.+Chuauthor=M.+A.+Callenderauthor=M.+P.+Farrellauthor=J.+C.+Schmitz&title=Thymidylate+synthase+inhibitors+as+anticancer+agents%3A+from+bench+to+bedside"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DE.%26aulast%3DCallender%26aufirst%3DM.%2BA.%26aulast%3DFarrell%26aufirst%3DM.%2BP.%26aulast%3DSchmitz%26aufirst%3DJ.%2BC.%26atitle%3DThymidylate%2520synthase%2520inhibitors%2520as%2520anticancer%2520agents%253A%2520from%2520bench%2520to%2520bedside%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2003%26volume%3D52%26issue%3DSuppl1%26spage%3DS80%26epage%3DS89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shoichet, B. K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span> </span><span class="NLM_article-title">Structure-based discovery of inhibitors of thymidylate synthase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">259</span><span class="NLM_x">, </span> <span class="NLM_fpage">1445</span><span class="NLM_x">–</span> <span class="NLM_lpage">1450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1126%2Fscience.8451640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=8451640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK3sXktF2qsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=259&publication_year=1993&pages=1445-1450&author=B.+K+Shoichetauthor=R.+M.+Stroudauthor=D.+V.+Santiauthor=I.+D.+Kuntzauthor=K.+M.+Perry&title=Structure-based+discovery+of+inhibitors+of+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of inhibitors of thymidylate synthase</span></div><div class="casAuthors">Shoichet, Brian K.; Stroud, Robert M.; Santi, Daniel V.; Kuntz, Irwin D.; Perry, Kathy M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">259</span>
        (<span class="NLM_cas:issue">5100</span>),
    <span class="NLM_cas:pages">1445-50</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A mol. docking computer program (DOCK) was used to screen the Fine Chem. Directory, a database of com. available compds., for mols. that are complementary to thymidylate synthase (TS), a chemotherapeutic target.  Besides retrieving the substrate and several known inhibitors, DOCK proposed putative inhibitors previously unknown to bind to the enzyme.  Three of these compds. inhibited Lactobacillus casei TS at submillimolar concns.  One of these inhibitors, sulisobenzone, crystd. with TS in two configurations that differed from the DOCK-favored geometry: a counterion was bound in the substrate site, which resulted in a 6 to 9 angstrom displacement of the inhibitor.  The structure of the complexes suggested another binding region in the active site that could be exploited.  This region was probed with mols. sterically similar to sulisobenzone, which led to the identification of a family of phenolphthalein analogs that inhibit TS in the 1 to 30 micromolar range.  These inhibitors do not resemble the substrates of the enzyme.  A crystal structure of phenolphthalein with TS shows that it binds in the target site in a configuration that resembles the one suggested by DOCK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcal-kXenW8bVg90H21EOLACvtfcHk0lgz82aJAXzEsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktF2qsLc%253D&md5=a2e9380f78c3dd0f213443df91cefb75</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.8451640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8451640%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26atitle%3DStructure-based%2520discovery%2520of%2520inhibitors%2520of%2520thymidylate%2520synthase%26jtitle%3DScience%26date%3D1993%26volume%3D259%26spage%3D1445%26epage%3D1450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Finer-Moore, J. S.; Blaney, J.; Stroud, R. M.</span>,  <span class="citation_source-book">Computational and Structural Approaches to Drug Discovery</span>; <span class="NLM_contrib-group">Stroud, R. M.; Finer-Moore, J. S.</span>, Eds; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge, UK</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>. p  <span class="NLM_fpage">9</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=9&author=J.+S.+Finer-Moore&author=J.+Blaney&author=R.+M.+Stroudauthor=R.+M.+Stroud&author=J.+S.+Finer-Moore&title=Computational+and+Structural+Approaches+to+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26btitle%3DComputational%2520and%2520Structural%2520Approaches%2520to%2520Drug%2520Discovery%26aulast%3DStroud%26aufirst%3DR.%2BM.%26pub%3DRSC%26date%3D2008%26spage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Structure-based design of inhibitors specific for bacterial thymidylate synthase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1607</span><span class="NLM_x">–</span> <span class="NLM_lpage">1617</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9815896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1607-1617&author=T.+J.+Stoutauthor=D.+Tondiauthor=M.+Rinaldiauthor=D.+Barloccoauthor=P.+Pecorariauthor=D.+V.+Santiauthor=I.+D.+Kuntzauthor=R.+M.+Stroudauthor=B.+K.+Shoichetauthor=M.+P.+Costi&title=Structure-based+design+of+inhibitors+specific+for+bacterial+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fbi9815896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815896%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520inhibitors%2520specific%2520for%2520bacterial%2520thymidylate%2520synthase%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D1607%26epage%3D1617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musiu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marangiu, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congiu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loi, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Colla, P.</span><span> </span><span class="NLM_article-title">Phthalein derivatives as a new tool for selectivity in thymidylate synthase inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">2112</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9900016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2112-2124&author=M.+P.+Costiauthor=M.+Rinaldiauthor=D.+Tondiauthor=P.+Pecorariauthor=D.+Barloccoauthor=S.+Ghelliauthor=R.+M.+Stroudauthor=D.+V.+Santiauthor=T.+J.+Stoutauthor=C.+Musiuauthor=E.+M.+Marangiuauthor=A.+Paniauthor=D.+Congiuauthor=G.+A.+Loiauthor=P.+La+Colla&title=Phthalein+derivatives+as+a+new+tool+for+selectivity+in+thymidylate+synthase+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm9900016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9900016%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DMusiu%26aufirst%3DC.%26aulast%3DMarangiu%26aufirst%3DE.%2BM.%26aulast%3DPani%26aufirst%3DA.%26aulast%3DCongiu%26aufirst%3DD.%26aulast%3DLoi%26aufirst%3DG.%2BA.%26aulast%3DLa%2BColla%26aufirst%3DP.%26atitle%3DPhthalein%2520derivatives%2520as%2520a%2520new%2520tool%2520for%2520selectivity%2520in%2520thymidylate%2520synthase%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2112%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title"><i>ortho</i>-Halogen naphthaleins as specific inhibitors of <i>Lactobacillus casei</i> thymidylate synthase. Conformational properties and biological activity</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">951</span><span class="NLM_x">–</span> <span class="NLM_lpage">963</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=951-963&author=S.+Ghelliauthor=M.+Rinaldiauthor=D.+Barloccoauthor=A.+Gelainauthor=P.+Pecorariauthor=D.+Tondiauthor=G.+Rastelliauthor=M.+P.+Costi&title=ortho-Halogen+naphthaleins+as+specific+inhibitors+of+Lactobacillus+casei+thymidylate+synthase.+Conformational+properties+and+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGelain%26aufirst%3DA.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3Dortho-Halogen%2520naphthaleins%2520as%2520specific%2520inhibitors%2520of%2520Lactobacillus%2520casei%2520thymidylate%2520synthase.%2520Conformational%2520properties%2520and%2520biological%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D951%26epage%3D963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Calò, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venturelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecorari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rinaldi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">A step further in the discovery of phthalein derivatives as thymidylate synthase inhibitors</span> <span class="citation_source-journal">ARKIVOC</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">396</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=382-396&author=S.+Cal%C3%B2author=D.+Tondiauthor=A.+Venturelliauthor=S.+Ferrariauthor=P.+Pecorariauthor=M.+Rinaldiauthor=S.+Ghelliauthor=M.+P.+Costi&title=A+step+further+in+the+discovery+of+phthalein+derivatives+as+thymidylate+synthase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCal%25C3%25B2%26aufirst%3DS.%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DVenturelli%26aufirst%3DA.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DPecorari%26aufirst%3DP.%26aulast%3DRinaldi%26aufirst%3DM.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DA%2520step%2520further%2520in%2520the%2520discovery%2520of%2520phthalein%2520derivatives%2520as%2520thymidylate%2520synthase%2520inhibitors%26jtitle%3DARKIVOC%26date%3D2004%26spage%3D382%26epage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Neil, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krucinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R.</span><span> </span><span class="NLM_article-title">The structure of <i>Cryptococcus neoformans</i> thymidylate synthase suggests strategies for using target dynamics for species-specific inhibition</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">D61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1320</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D61&publication_year=2005&pages=1320-1334&author=J.+S.+Finer-Mooreauthor=A.+C.+Andersonauthor=R.+H.+O%E2%80%99Neilauthor=M.+P.+Costiauthor=S.+Ferrariauthor=J.+Krucinskiauthor=R.+Stroud&title=The+structure+of+Cryptococcus+neoformans+thymidylate+synthase+suggests+strategies+for+using+target+dynamics+for+species-specific+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DAnderson%26aufirst%3DA.%2BC.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DR.%2BH.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DKrucinski%26aufirst%3DJ.%26aulast%3DStroud%26aufirst%3DR.%26atitle%3DThe%2520structure%2520of%2520Cryptococcus%2520neoformans%2520thymidylate%2520synthase%2520suggests%2520strategies%2520for%2520using%2520target%2520dynamics%2520for%2520species-specific%2520inhibition%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3DD61%26spage%3D1320%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soragni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raniero, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruberto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavazzuti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casolari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span> </span><span class="NLM_article-title">Anti-bacterial agent discovery using thymidylate synthase bio-library screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5958</span><span class="NLM_x">–</span> <span class="NLM_lpage">5968</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051187d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5958-5968&author=M.+P.+Costiauthor=A.+Gelainauthor=D.+Barloccoauthor=S.+Ghelliauthor=F.+Soragniauthor=F.+Ranieroauthor=T.+Rossiauthor=A.+Rubertoauthor=C.+Guillouauthor=A.+Cavazzutiauthor=C.+Casolariauthor=S.+Ferrari&title=Anti-bacterial+agent+discovery+using+thymidylate+synthase+bio-library+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm051187d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051187d%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DGelain%26aufirst%3DA.%26aulast%3DBarlocco%26aufirst%3DD.%26aulast%3DGhelli%26aufirst%3DS.%26aulast%3DSoragni%26aufirst%3DF.%26aulast%3DRaniero%26aufirst%3DF.%26aulast%3DRossi%26aufirst%3DT.%26aulast%3DRuberto%26aufirst%3DA.%26aulast%3DGuillou%26aufirst%3DC.%26aulast%3DCavazzuti%26aufirst%3DA.%26aulast%3DCasolari%26aufirst%3DC.%26aulast%3DFerrari%26aufirst%3DS.%26atitle%3DAnti-bacterial%2520agent%2520discovery%2520using%2520thymidylate%2520synthase%2520bio-library%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5958%26epage%3D5968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0liyKevoOzraOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span>ChemFinder Ultra Database 7.0; <span class="NLM_publisher-name">Cambridge Soft Corporation</span>, <a href="htpp://www.cambridgesoft.com" class="extLink">htpp://www.cambridgesoft.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ChemFinder+Ultra+Database+7.0%3B+Cambridge+Soft+Corporation%2C+htpp%3A%2F%2Fwww.cambridgesoft.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DChemFinder%2520Ultra%2520Database%25207.0%26pub%3DCambridge%2520Soft%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span>ACDlab 6.0 Software; <span class="NLM_publisher-name">Cambridge Soft Corporation</span>, <a href="htpp://www.cambridgesoft.com" class="extLink">htpp://www.cambridgesoft.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ACDlab+6.0+Software%3B+Cambridge+Soft+Corporation%2C+htpp%3A%2F%2Fwww.cambridgesoft.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACDlab%25206.0%2520Software%26pub%3DCambridge%2520Soft%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0liyKevoOzraOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montfort, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, M. O</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Atomic structure of thymidylate synthase: target for rational drug design</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=448-455&author=L.+W.+Hardyauthor=J.+S.+Finer-Mooreauthor=W.+R.+Montfortauthor=M.+O+Jonesauthor=D.+V.+Santiauthor=R.+M.+Stroud&title=Atomic+structure+of+thymidylate+synthase%3A+target+for+rational+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DMontfort%26aufirst%3DW.%2BR.%26aulast%3DJones%26aufirst%3DM.%2BO%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DAtomic%2520structure%2520of%2520thymidylate%2520synthase%253A%2520target%2520for%2520rational%2520drug%2520design%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D448%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Finer-Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R.</span><span> </span><span class="NLM_article-title">Refined structures of substrate-bound and phosphate-bound thymidylate synthase from <i>Lactobacillus casei</i></span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">232</span><span class="NLM_x">, </span> <span class="NLM_fpage">1101</span><span class="NLM_x">–</span> <span class="NLM_lpage">1116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=232&publication_year=1993&pages=1101-1116&author=J.+Finer-Mooreauthor=E.+B.+Faumanauthor=P.+G.+Fosterauthor=K.+M.+Perryauthor=D.+V.+Santiauthor=R.+Stroud&title=Refined+structures+of+substrate-bound+and+phosphate-bound+thymidylate+synthase+from+Lactobacillus+casei"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFiner-Moore%26aufirst%3DJ.%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DFoster%26aufirst%3DP.%2BG.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DStroud%26aufirst%3DR.%26atitle%3DRefined%2520structures%2520of%2520substrate-bound%2520and%2520phosphate-bound%2520thymidylate%2520synthase%2520from%2520Lactobacillus%2520casei%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D232%26spage%3D1101%26epage%3D1116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losasso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Sequence-Based identification of specific drug target regions in the thymidylate synthase enzyme family</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">401</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=392-401&author=S.+Ferrariauthor=V.+Losassoauthor=M.+P.+Costi&title=Sequence-Based+identification+of+specific+drug+target+regions+in+the+thymidylate+synthase+enzyme+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DLosasso%26aufirst%3DV.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DSequence-Based%2520identification%2520of%2520specific%2520drug%2520target%2520regions%2520in%2520the%2520thymidylate%2520synthase%2520enzyme%2520family%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D392%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R.</span><span> </span><span class="NLM_article-title">Inhibitor specificity via protein dynamics: insights from the design of antibacterial agents targeted against thymidylate synthase</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1183</span><span class="NLM_x">–</span> <span class="NLM_lpage">1193</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2Fj.chembiol.2003.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=14700626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVWis7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=1183-1193&author=S.+Ferrariauthor=M.+P.+Costiauthor=R.+Wade&title=Inhibitor+specificity+via+protein+dynamics%3A+insights+from+the+design+of+antibacterial+agents+targeted+against+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Specificity via Protein Dynamics Insights from the Design of Antibacterial Agents Targeted Against Thymidylate Synthase</span></div><div class="casAuthors">Ferrari, Stefania; Costi, Paola M.; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1183-1193</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Structure-based drug design of species-specific inhibitors generally exploits structural differences in proteins from different organisms.  Here, we demonstrate how achieving specificity can be aided by targeting differences in the dynamics of proteins.  Thymidylate synthase (TS) is a good target for anticancer agents and a potential target for antibacterial agents.  Most inhibitors are folate-analogs that bind at the folate binding site and are not species specific.  In contrast, α156 is not a folate-analog and is specific for bacterial TS; it has been shown crystallog. to bind in a nonconserved binding site.  Docking calcns. and crystal structure-based estn. of the essential dynamics of TSs from five different species show that differences in the dynamics of TSs make the active site more accessible to α156 in the prokaryotic than in the eukaryotic TSs and thereby enhance the specificity of α156.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiW86dIS4vLVg90H21EOLACvtfcHk0lgHZPGvbaUb_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVWis7zJ&md5=5bfe50faeb8b9c14664763bbc74fc670</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2003.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2003.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DWade%26aufirst%3DR.%26atitle%3DInhibitor%2520specificity%2520via%2520protein%2520dynamics%253A%2520insights%2520from%2520the%2520design%2520of%2520antibacterial%2520agents%2520targeted%2520against%2520thymidylate%2520synthase%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D1183%26epage%3D1193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kealey, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Purification methods for recombinant <i>Lactobacillus casei</i> thymidylate synthase and mutants: a general, automated procedure</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">380</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1016%2FS1046-5928%2805%2980039-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1992&pages=380-385&author=J.+T.+Kealeyauthor=D.+V.+Santi&title=Purification+methods+for+recombinant+Lactobacillus+casei+thymidylate+synthase+and+mutants%3A+a+general%2C+automated+procedure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1046-5928%2805%2980039-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1046-5928%252805%252980039-7%26sid%3Dliteratum%253Aachs%26aulast%3DKealey%26aufirst%3DJ.%2BT.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DPurification%2520methods%2520for%2520recombinant%2520Lactobacillus%2520casei%2520thymidylate%2520synthase%2520and%2520mutants%253A%2520a%2520general%252C%2520automated%2520procedure%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D1992%26volume%3D4%26spage%3D380%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span> </span><span class="NLM_article-title">Properties of a defined mutant of <i>Escherichia coli</i> thymidylate synthase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">7620</span><span class="NLM_x">–</span> <span class="NLM_lpage">7627</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=7620-7627&author=G.+F.+Maleyauthor=F.+Maley&title=Properties+of+a+defined+mutant+of+Escherichia+coli+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DMaley%26aufirst%3DF.%26atitle%3DProperties%2520of%2520a%2520defined%2520mutant%2520of%2520Escherichia%2520coli%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D7620%26epage%3D7627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Pedersen-Lane, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span> </span><span class="NLM_article-title">High-level expression of human thymidylate synthase</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">256</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1006%2Fprep.1997.0750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=9226722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2sXksFajs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1997&pages=256-262&author=J.+Pedersen-Laneauthor=G.+F.+Maleyauthor=E.+Chuauthor=F.+Maley&title=High-level+expression+of+human+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">High-level expression of human thymidylate synthase</span></div><div class="casAuthors">Pedersen-Lane, Joan; Maley, Gladys F.; Chu, Edward; Maley, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">256-262</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A method is presented for expressing human thymidylate synthase (TS) to the extent of 25-30% of the protein in Escherichia coli.  By this procedure, 200-400 mg of pure enzyme can be obtained from a 2-L culture of cells.  The key to the level of expression appears to be related to the conversion of purine bases in the 3rd, 4th, and 5th codons of the TS cDNA to thymine, without altering the encoded protein product.  Conversion of the penultimate proline to a leucine did not diminish expression, but while the isolated native enzyme contained only proline on its amino-terminal end, the mutated enzyme contained methionine on its amino terminus.  By contrast, the expression of the unmodified TS cDNA represented only 0.1-0.2% of the total cellular protein.  Unlike recombinant rat and human TSs, the resp. enzymes purified to homogeneity from eukaryotic cells were blocked at the amino ends and possessed 2- to 4-fold lower specific activities.  To det. at what level the impairment of expression occurred, an in vitro transcription, translation system was employed and the results showed that while transcription was unaffected, the translation of native TS mRNA was reduced by ≥20-fold relative to modified TS mRNA using a rabbit reticulocyte translation system.  Thus, it appears that at least for the TS gene, expression is greatly influenced by the GC content of the 5' coding region of the gene in both prokaryote and eukaryote systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujd06OIYeabVg90H21EOLACvtfcHk0lgHZPGvbaUb_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFajs7w%253D&md5=834d2d0186feb013bda20139c896e1b2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1006%2Fprep.1997.0750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fprep.1997.0750%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen-Lane%26aufirst%3DJ.%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DChu%26aufirst%3DE.%26aulast%3DMaley%26aufirst%3DF.%26atitle%3DHigh-level%2520expression%2520of%2520human%2520thymidylate%2520synthase%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D1997%26volume%3D10%26spage%3D256%26epage%3D262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Pogolotti, A. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danenberg, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Kinetics and mechanism of interaction of 10-propargyl-5,8-dideazafolate with thymidylate synthase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">482</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00154a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=478-482&author=A.+L.+Pogolottiauthor=P.+V.+Danenbergauthor=D.+V.+Santi&title=Kinetics+and+mechanism+of+interaction+of+10-propargyl-5%2C8-dideazafolate+with+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm00154a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00154a009%26sid%3Dliteratum%253Aachs%26aulast%3DPogolotti%26aufirst%3DA.%2BL.%26aulast%3DDanenberg%26aufirst%3DP.%2BV.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DKinetics%2520and%2520mechanism%2520of%2520interaction%2520of%252010-propargyl-5%252C8-dideazafolate%2520with%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D478%26epage%3D482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Segel, I. H.</span><span> </span><span class="NLM_article-title">Enzyme Kinetics</span>.  <span class="citation_source-book">Behaviour and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</span>; <span class="NLM_publisher-name">John Wiley and Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1975</span><span class="NLM_x">, </span>p  <span class="NLM_fpage">105</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&pages=105&author=I.+H.+Segel&title=Behaviour+and+Analysis+of+Rapid+Equilibrium+and+Steady-State+Enzyme+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26atitle%3DEnzyme%2520Kinetics%26btitle%3DBehaviour%2520and%2520Analysis%2520of%2520Rapid%2520Equilibrium%2520and%2520Steady-State%2520Enzyme%2520Systems%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D1975%26spage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Cavazzuti, A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paglietti, G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamarro, F</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piras, S</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriga, M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alleca, S</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, P</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tolloch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Discovery of potent pteridine reductase inhibitors ot guide antiparasite drug development</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1448-1453&author=A+Cavazzutiauthor=G+Pagliettiauthor=W.+N.+Hunterauthor=F+Gamarroauthor=S+Pirasauthor=M+Lorigaauthor=S+Allecaauthor=P+Coronaauthor=K+McLuskeyauthor=L.+Tollochauthor=F.+Gibelliniauthor=S.+Ferrariauthor=M.+P.+Costi&title=Discovery+of+potent+pteridine+reductase+inhibitors+ot+guide+antiparasite+drug+development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCavazzuti%26aufirst%3DA%26aulast%3DPaglietti%26aufirst%3DG%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGamarro%26aufirst%3DF%26aulast%3DPiras%26aufirst%3DS%26aulast%3DLoriga%26aufirst%3DM%26aulast%3DAlleca%26aufirst%3DS%26aulast%3DCorona%26aufirst%3DP%26aulast%3DMcLuskey%26aufirst%3DK%26aulast%3DTolloch%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520potent%2520pteridine%2520reductase%2520inhibitors%2520ot%2520guide%2520antiparasite%2520drug%2520development%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Jaspard, E.</span><span> </span><span class="NLM_article-title">Role of protein–solvent interactions in refolding: effects of cosolvent additives on the renaturation of porcine pancreatic elastase at various pHs</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">220</span><span class="NLM_x">–</span> <span class="NLM_lpage">228</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2000&pages=220-228&author=E.+Jaspard&title=Role+of+protein%E2%80%93solvent+interactions+in+refolding%3A+effects+of+cosolvent+additives+on+the+renaturation+of+porcine+pancreatic+elastase+at+various+pHs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJaspard%26aufirst%3DE.%26atitle%3DRole%2520of%2520protein%25E2%2580%2593solvent%2520interactions%2520in%2520refolding%253A%2520effects%2520of%2520cosolvent%2520additives%2520on%2520the%2520renaturation%2520of%2520porcine%2520pancreatic%2520elastase%2520at%2520various%2520pHs%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D15%26spage%3D220%26epage%3D228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bhattacharjya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaram, P.</span><span> </span><span class="NLM_article-title">Effects of organic solvents on protein structures: observation of a structured helical core in hen egg-white lysozyme in aqueous dimethylsulfoxide</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">492</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1997&pages=492-507&author=S.+Bhattacharjyaauthor=P.+Balaram&title=Effects+of+organic+solvents+on+protein+structures%3A+observation+of+a+structured+helical+core+in+hen+egg-white+lysozyme+in+aqueous+dimethylsulfoxide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharjya%26aufirst%3DS.%26aulast%3DBalaram%26aufirst%3DP.%26atitle%3DEffects%2520of%2520organic%2520solvents%2520on%2520protein%2520structures%253A%2520observation%2520of%2520a%2520structured%2520helical%2520core%2520in%2520hen%2520egg-white%2520lysozyme%2520in%2520aqueous%2520dimethylsulfoxide%26jtitle%3DProteins%26date%3D1997%26volume%3D29%26spage%3D492%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Leslie, A. G. W.</span> <span class="citation_source-book">Crystallographic Computing V</span>; <span class="NLM_contrib-group">Moras, D.; Podjarny, A. D.; Thierry, J.-C.</span>, Eds.; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford</span>,<span class="NLM_x"> </span><span class="NLM_year">1991</span><span class="NLM_x">; </span>pp <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1991&pages=50-61&author=A.+G.+W.+Leslieauthor=D.+Moras&author=A.+D.+Podjarny&author=J.-C.+Thierry&title=Crystallographic+Computing+V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26btitle%3DCrystallographic%2520Computing%2520V%26aulast%3DMoras%26aufirst%3DD.%26pub%3DOxford%2520University%2520Press%26date%3D1991%26spage%3D50%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Collaborative Computational Project, No. 4</span><span class="NLM_x">. </span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project%2C+No.+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">SCALA, continuous scaling program</span>.  <span class="citation_source-journal">Joint CCP4 ESF-EACBM Newsletter</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1997&pages=22-24&author=P.+R.+Evans&title=SCALA%2C+continuous+scaling+program"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DSCALA%252C%2520continuous%2520scaling%2520program%26jtitle%3DJoint%2520CCP4%2520ESF-EACBM%2520Newsletter%26date%3D1997%26volume%3D33%26spage%3D22%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Vagin, A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lizdFELY15GKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhH-gt_vogXbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Perrakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R. J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamzin, V. S.</span><span> </span><span class="NLM_article-title">Automated protein model building combined with iterative structure refinement</span> <span class="citation_source-journal">Nature Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm2005018&amp;key=10.1038%2F8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm2005018&amp;key=10331874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm2005018&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=458-463&author=A.+Perrakisauthor=R.+J.+H.+Morrisauthor=V.+S.+Lamzin&title=Automated+protein+model+building+combined+with+iterative+structure+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein model building combined with iterative structure refinement</span></div><div class="casAuthors">Perrakis, Anastassis; Morris, Richard; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-463</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">In protein crystallog., much time and effort are often required to trace an initial model from an interpretable electron d. map and to refine it until it best agrees with the crystallog. data.  Here, we present a method to build and refine a protein model automatically and without user intervention, starting from diffraction data extending to resoln. higher than 2.3 Å and reasonable ests. of crystallog. phases.  The method is based on an iterative procedure that describes the electron d. map as a set of unconnected atoms and then searches for protein-like patterns.  Automatic pattern recognition (model building) combined with refinement, allows a structural model to be obtained reliably within a few CPU hours.  We demonstrate the power of the method with examples of a few recently solved structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbcrkUoPTzsLVg90H21EOLACvtfcHk0lhH-gt_vogXbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D&md5=d12fc80eb73e1faebee29e2f6635c86d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2F8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F8263%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%2BJ.%2BH.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DAutomated%2520protein%2520model%2520building%2520combined%2520with%2520iterative%2520structure%2520refinement%26jtitle%3DNature%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D458%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">McRee, D. E.</span><span> </span><span class="NLM_article-title">XtalView/Xfit—a versatile program for manipulating atomic coordinates and electron density</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">156</span><span class="NLM_x">–</span> <span class="NLM_lpage">165</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1999&pages=156-165&author=D.+E.+McRee&title=XtalView%2FXfit%E2%80%94a+versatile+program+for+manipulating+atomic+coordinates+and+electron+density"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcRee%26aufirst%3DD.%2BE.%26atitle%3DXtalView%252FXfit%25E2%2580%2594a%2520versatile%2520program%2520for%2520manipulating%2520atomic%2520coordinates%2520and%2520electron%2520density%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D125%26spage%3D156%26epage%3D165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span>UCSF Chimera, Production Version 1 Build 2038 2004/10/18,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UCSF+Chimera%2C+Production+Version+1+Build+2038+2004%2F10%2F18%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DUCSF%2520Chimera%252C%2520Production%2520Version%25201%2520Build%25202038%25202004%252F10%252F18%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C06','PDB','3C06'); return false;">PDB: 3C06</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0A','PDB','3C0A'); return false;">PDB: 3C0A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BNZ','PDB','3BNZ'); return false;">PDB: 3BNZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYX','PDB','3BYX'); return false;">PDB: 3BYX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BZ0','PDB','3BZ0'); return false;">PDB: 3BZ0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK1','PDB','3IK1'); return false;">PDB: 3IK1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IJZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IJZ','PDB','3IJZ'); return false;">PDB: 3IJZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IK0','PDB','3IK0'); return false;">PDB: 3IK0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSL','PDB','1TSL'); return false;">PDB: 1TSL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1TSM','PDB','1TSM'); return false;">PDB: 1TSM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A9W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A9W','PDB','2A9W'); return false;">PDB: 2A9W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=C03A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=C03A','PDB','C03A'); return false;">PDB: C03A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LCA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LCA','PDB','1LCA'); return false;">PDB: 1LCA</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i51"><a href="/doi/suppl/10.1021/jm2005018">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_48508"></div></div></div></div></div><hr /></hr><p class="last">Binding modes of known phthalein and naphthalein compounds (PTH; α156, MR20, GA9); designed libraries A and B; calculated molecular properties for libraries A and B; atom numbering in compounds characterized through NMR; plot of the enzyme kinetic inhibition profiles (<i>K</i><sub>i</sub>) for libraries A and B; contact distances between substrates and LcTS residues. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm2005018/suppl_file/jm2005018_si_001.pdf">jm2005018_si_001.pdf (1.17 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm2005018&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm2005018%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2011.54.issue-15%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm2005018" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b2263da93d00","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
