
<html lang="en"     class="pb-page"  data-request-id="2eef1ce4-bdac-4023-8b90-eb53ff8c882a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2003.46.issue-9;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/jm020397c"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Pharmacophore and Quantitative Structure−Activity Relationship Modeling:  Complementary Approaches for the Rationalization and Prediction of UDP-Glucuronosyltransferase 1A4 Substrate Selectivity" /></meta><meta name="dc.Creator" content="Paul A.  Smith,*,†" /></meta><meta name="dc.Creator" content="Michael J.  Sorich,‡" /></meta><meta name="dc.Creator" content="Ross A.  McKinnon,‡ and" /></meta><meta name="dc.Creator" content="John O.  Miners†" /></meta><meta name="dc.Description" content="Pharmacophore, two-dimensional (2D), and three-dimensional (3D) quantitative structure−activity relationship (QSAR) modeling techniques were used to develop and test models capable of rationalizing..." /></meta><meta name="Description" content="Pharmacophore, two-dimensional (2D), and three-dimensional (3D) quantitative structure−activity relationship (QSAR) modeling techniques were used to develop and test models capable of rationalizing..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="April 2, 2003" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm020397c" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-46-9-1617" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(02)00397-7" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2003 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm020397c" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm020397c" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm020397c" /></link>
        
    
    

<title>Pharmacophore and Quantitative Structure−Activity Relationship Modeling:  Complementary Approaches for the Rationalization and Prediction of UDP-Glucuronosyltransferase 1A4 Substrate Selectivity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm020397c" /></meta><meta property="og:title" content="Pharmacophore and Quantitative Structure−Activity Relationship Modeling:  Complementary Approaches for the Rationalization and Prediction of UDP-Glucuronosyltransferase 1A4 Substrate Selectivity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cn00001.jpeg" /></meta><meta property="og:description" content=" Pharmacophore, two-dimensional (2D), and three-dimensional (3D) quantitative structure−activity relationship (QSAR) modeling techniques were used to develop and test models capable of rationalizing and predicting human UDP-glucuronosyltransferase 1A4 (UGT1A4) substrate selectivity and binding affinity (as Km,app). The dataset included 24 structurally diverse UGT1A4 substrates, with 18 of these comprising the training set and 6 an external prediction set. A common features pharmacophore was generated with the program Catalyst after overlapping the sites of conjugation using a novel, user-defined “glucuronidation” feature. Pharmacophore-based 3D-QSAR (r2 = 0.88) and molecular-field-based 3D-QSAR (r2 = 0.73) models were developed using Catalyst and self-organizing molecular field analysis (SOMFA) software, respectively. In addition, a 2D-QSAR (r2 = 0.80, CV r2 = 0.73) was generated using partial least-squares (PLS) regression and variable selection using an unsupervised forward selection (UFS) algorithm. Both UGT1A4 pharmacophores included two hydrophobic features and the glucuronidation site. The 2D-QSAR showed the best overall predictivity and highlighted the importance of hydrophobicity (as log P) in substrate−enzyme binding.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm020397c"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm020397c">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm020397c&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm020397c&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm020397c&amp;href=/doi/10.1021/jm020397c" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2003</span><span class="cit-fg-volume">, 46</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 1617-1626</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/46/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm0210156" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm0204035" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Pharmacophore and Quantitative Structure−Activity Relationship Modeling:  Complementary Approaches for the Rationalization and Prediction of UDP-Glucuronosyltransferase 1A4 Substrate Selectivity</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+A.++Smith">Paul A. Smith</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+J.++Sorich">Michael J. Sorich</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ross+A.++McKinnon">Ross A. McKinnon</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+O.++Miners">John O. Miners</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Clinical Pharmacology, Flinders University and Flinders Medical Centre, Bedford Park, 5042, South Australia, and School of Pharmaceutical, Molecular and Biomedical Sciences, University of South Australia, Adelaide, 5001, South Australia </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm020397c&amp;href=/doi/10.1021%2Fjm020397c" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2003</span></span><span class="cit-volume">, 46</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 1617–1626</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 2, 2003</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 September 2002</li><li><span class="item_label"><b>Published</b> online</span>2 April 2003</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 April 2003</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm020397c" title="DOI URL">https://doi.org/10.1021/jm020397c</a></div><div class="article_header-article-copyright"><strong>Copyright © 2003 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1617%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaul%2BA.%2BSmith%252C%2BMichael%2BJ.%2BSorich%252C%2BRoss%2BA.%2BMcKinnon%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D46%26issueNum%3D9%26contentID%3Djm020397c%26title%3DPharmacophore%2Band%2BQuantitative%2BStructure%25E2%2588%2592Activity%2BRelationship%2BModeling%253A%25E2%2580%2589%2BComplementary%2BApproaches%2Bfor%2Bthe%2BRationalization%2Band%2BPrediction%2Bof%2BUDP-Glucuronosyltransferase%2B1A4%2BSubstrate%2BSelectivity%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1626%26publicationDate%3DApril%2B2003">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm020397c"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">649</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">50</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm020397c" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Pharmacophore and Quantitative Structure−Activity Relationship Modeling:  Complementary Approaches for the Rationalization and Prediction of UDP-Glucuronosyltransferase 1A4 Substrate Selectivity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;A. Smith&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;J. Sorich&quot;},{&quot;first_name&quot;:&quot;Ross&quot;,&quot;last_name&quot;:&quot;A. McKinnon&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;O. Miners&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2003&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;46&quot;,&quot;pages&quot;:&quot;1617-1626&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm020397c&quot;},&quot;abstract&quot;:&quot;Pharmacophore, two-dimensional (2D), and three-dimensional (3D) quantitative structure−activity relationship (QSAR) modeling techniques were used to develop and test models capable of rationalizing and predicting human UDP-glucuronosyltransferase 1A4 (UGT1A4) substrate selectivity and binding affinity (as Km,app). The dataset included 24 structurally diverse UGT1A4 substrates, with 18 of these comprising the training set and 6 an external prediction set. A common features pharmacophore was generated with the program Catalyst after overlapping the sites of conjugation using a novel, user-defined “glucuronidation” feature. Pharmacophore-based 3D-QSAR (r2 = 0.88) and molecular-field-based 3D-QSAR (r2 = 0.73) models were developed using Catalyst and self-organizing molecular field analysis (SOMFA) software, respectively. In addition, a 2D-QSAR (r2 = 0.80, CV r2 = 0.73) was generated using partial least-squares (PLS) regression and variable selection using an unsupervised forward selection (UFS) algorithm. Both UG&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm020397c&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm020397c" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm020397c&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm020397c" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm020397c&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm020397c" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm020397c&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm020397c&amp;href=/doi/10.1021/jm020397c" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm020397c" /></input><a href="/doi/pdf/10.1021/jm020397c" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (244 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm020397c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm020397c%26sid%3Dliteratum%253Aachs%26pmid%3D12699380%26genre%3Darticle%26aulast%3DSmith%26date%3D2003%26atitle%3DPharmacophore%2Band%2BQuantitative%2BStructure%25E2%2588%2592Activity%2BRelationship%2BModeling%253A%25E2%2580%2589%2BComplementary%2BApproaches%2Bfor%2Bthe%2BRationalization%2Band%2BPrediction%2Bof%2BUDP-Glucuronosyltransferase%2B1A4%2BSubstrate%2BSelectivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D46%26issue%3D9%26spage%3D1617%26epage%3D1626%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290965" title="Catalysts">Catalysts</a>,</li><li><a href="/action/doSearch?ConceptID=290663" title="Pharmacophores">Pharmacophores</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=290957" title="Molecular modeling">Molecular modeling</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/46/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Pharmacophore, two-dimensional (2D), and three-dimensional (3D) quantitative structure−activity relationship (QSAR) modeling techniques were used to develop and test models capable of rationalizing and predicting human UDP-glucuronosyltransferase 1A4 (UGT1A4) substrate selectivity and binding affinity (as <i>K</i><sub>m,app</sub>). The dataset included 24 structurally diverse UGT1A4 substrates, with 18 of these comprising the training set and 6 an external prediction set. A common features pharmacophore was generated with the program Catalyst after overlapping the sites of conjugation using a novel, user-defined “glucuronidation” feature. Pharmacophore-based 3D-QSAR (<i>r</i><sup>2</sup> = 0.88) and molecular-field-based 3D-QSAR (<i>r</i><i><sup>2</sup></i><sup></sup> = 0.73) models were developed using Catalyst and self-organizing molecular field analysis (SOMFA) software, respectively. In addition, a 2D-QSAR (<i>r</i><sup>2</sup> = 0.80, CV<i> r</i><sup>2</sup> = 0.73) was generated using partial least-squares (PLS) regression and variable selection using an unsupervised forward selection (UFS) algorithm. Both UGT1A4 pharmacophores included two hydrophobic features and the glucuronidation site. The 2D-QSAR showed the best overall predictivity and highlighted the importance of hydrophobicity (as log <i>P</i>) in substrate−enzyme binding. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm020397cAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed. Phone:  61 8 8204 3155. Fax:  61 8 8204 5114. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0d7d6c7861237e606479654d6b61646369687f7e23686978236c7823">[email protected]</a></p></span></div><div class="fnGroupItem" id="jm020397cAF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  Flinders University and Flinders Medical Centre.</p></span></div><div class="fnGroupItem" id="jm020397cAF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  University of South Australia.</p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78371" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78371" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Recognition by the pharmaceutical industry that undesirable absorption, distribution, metabolism, and excretion (ADME) properties of new drug candidates are the cause of many clinical phase drug development failures has led to the need to identify such problems in the drug discovery process.<sup><a onclick="showRef(event, 'jm020397cb00001'); return false;" href="javascript:void(0);" class="ref jm020397cb00001">1</a></sup> In this regard, the demand for the rapid evaluation of increasing numbers of compounds has driven the development of methodologies and software, which has resulted in the increased application of molecular modeling techniques to ADME.<sup><a onclick="showRef(event, 'jm020397cb00002'); return false;" href="javascript:void(0);" class="ref jm020397cb00002">2</a></sup> Indeed, computational (in silico) and in vitro ADME approaches are now widely utilized throughout the discovery process to optimize the selection of the most suitable drug candidates for development. Notably, in silico approaches potentially allow the ADME properties of a newly discovered compound to be predicted by modeling the structural and physicochemical characteristics of that compound without recourse to laboratory-based procedures. </div><div class="NLM_p">An integral component of ADME evaluation is the identification of the enzyme(s) responsible for the metabolism of a new chemical entity, together with an estimate of kinetic parameters. Cytochrome P450 (CYP) and UDP-glucuronosyltransferase (UGT) are the principal drug metabolizing enzyme systems.<sup><a onclick="showRef(event, 'jm020397cb00003 jm020397cb00004'); return false;" href="javascript:void(0);" class="ref jm020397cb00003 jm020397cb00004">3,4</a></sup> Apart from metabolizing drugs, however, CYP and UGT also catalyze the biotransformation of a multitude of structurally diverse nondrug xenobiotics (e.g., environmental pollutants, dietary chemicals) and endogenous compounds (e.g., steroid hormones). Consistent with their broad substrate profiles, CYP and UGT are known to exist as superfamilies of independently regulated enzyme forms (“isoforms”) that exhibit distinct, but often overlapping, substrate and inhibitor selectivities. Thus, prediction of variability in drug elimination and response in humans requires knowledge of CYP and UGT isoform substrate and inhibitor selectivities and the factors that regulate isoform catalytic activity, particularly drug−drug interactions and genetic polymorphisms. </div><div class="NLM_p">In recent years, considerable effort has been directed toward predicting the substrate and inhibitor selectivities of human CYP isoforms using molecular modeling techniques including pharmacophore, 2D, and 3D quantitative structure−activity relationships (QSAR) and homology modeling.<sup><a onclick="showRef(event, 'jm020397cb00005 jm020397cb00006'); return false;" href="javascript:void(0);" class="ref jm020397cb00005 jm020397cb00006">5,6</a></sup> For example, pharmacophore and 3D-QSAR models have been developed that can be used to infer the active site binding requirements of substrates and inhibitors for numerous CYP isoforms and that can predict apparent <i>K</i><sub>m</sub> (<i>K</i><sub>m,app</sub>) for substrates and apparent <i>K</i><sub>i</sub> (<i>K</i><sub>i,app</sub>) for inhibitors.<sup><a onclick="showRef(event, 'jm020397cb00007'); return false;" href="javascript:void(0);" class="ref jm020397cb00007">7</a></sup> In particular, Ekins and colleagues have used the molecular modeling software Catalyst (Accelrys, Inc.) to derive pharmacophore models for several CYP isoforms that predict <i>K</i><sub>m,app</sub> or <i>K</i><sub>i,app</sub> values within 1 log unit.<sup><a onclick="showRef(event, 'jm020397cb00008 jm020397cb00009 jm020397cb00010 jm020397cb00011 jm020397cb00012'); return false;" href="javascript:void(0);" class="ref jm020397cb00008 jm020397cb00009 jm020397cb00010 jm020397cb00011 jm020397cb00012">8-12</a></sup> However, the pharmacophores do not necessarily overlay the oxidation sites of the chemicals, which may be a significant drawback for substrate-derived pharmacophores because it precludes any chemically intuitive interpretation of the alignments. Other groups<sup><a onclick="showRef(event, 'jm020397cb00013 jm020397cb00014'); return false;" href="javascript:void(0);" class="ref jm020397cb00013 jm020397cb00014">13,14</a></sup> have combined pharmacophore models with homology models based on crystal structures of bacterial CYPs and, more recently, the mammalian enzyme CYP2C5. These combined models provide powerful tools that take into <a name="jm004600901618" id="jm004600901618"></a> account the site of oxidation and allow investigation of active site features and substrate interactions. </div><div class="NLM_p">The successful adoption of in silico approaches for study of the CYP family has progressed in parallel with the increasing availability of CYP isoform substrate and inhibitor selectivities, and the development of such models has also been aided as a result of access to homology models of human CYPs. However, the development of models for predicting metabolism and for characterizing structural features of substrates for UGT isoforms is less advanced relative to CYP. Only recently has the substrate profile of UGT isoforms begun to approach an interpretable level, and an X-ray crystal structure is not yet available. Nevertheless, it is envisaged that molecular modeling techniques, in particular 2D- and 3D-QSAR and pharmacophore modeling, will translate well for application to UGT. </div><div class="NLM_p">Recent studies have investigated the substrate profile of the human UGT isoform UGT1A4.<sup><a onclick="showRef(event, 'jm020397cb00015 jm020397cb00016'); return false;" href="javascript:void(0);" class="ref jm020397cb00015 jm020397cb00016">15,16</a></sup> It has been demonstrated that UGT1A4 has the ability to metabolize a structurally diverse group of compounds, which includes drugs, nondrug xenobiotics, and hydroxysteroids. Of particular interest is the capacity of UGT1A4 to catalyze the glucuronidation of primary (e.g., 4-aminobiphenyl), secondary (e.g., desmethylclozapine), and tertiary amine (e.g., clozapine) containing compounds. <i>K</i><sub>m,app</sub> values determined for the UGT1A4 substrates ranged from 7 to 11 600 μM. </div><div class="NLM_p last">The studies reported here aimed to investigate the structural and chemical properties that characterize substrates of UGT1A4 and to evaluate several pattern recognition techniques for the prediction of <i>K</i><sub>m,app</sub> for a set of UGT1A4 substrates. Catalyst was used to generate two pharmacophores for UGT1A4 substrates in which the sites of conjugation were overlaid. One pharmacophore (“common features”) can be used to align molecules sensibly, and the other allows quantitative prediction of the substrate <i>K</i><sub>m,app</sub>. The alignment from the common features model was used to develop a molecular-field-based (SOMFA) model. Furthermore, a 2D-QSAR model was developed using the software packages Cerius2 (Accelrys, Inc.) and Dragon (Milano Chemometrics and QSAR Research Group, Milan, Italy) based on a range of descriptors that capture information associated with ligand−receptor binding events. The predictivity of the three QSAR models was comparable to those reported previously in the drug metabolism field. Overall, these models provide a complementary package for inferring various features associated with substrate binding to UGT1A4. The development of multiple models has also allowed comparison of these different computational approaches for the challenge of modeling UGTs. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Dataset Selection.</b> The dataset comprised UGT1A4 substrates characterized by Green and colleagues.<sup>15,16</sup> These data were derived from a single laboratory, using a common assay procedure to quantitate aglycone glucuronidation by UGT1A4 stably expressed in HK293 cells. Hence, the kinetic parameter modeled, namely, <i>K</i><sub>m,app</sub>, may be considered to be internally consistent. Substrates modeled were structurally diverse and had <i>K</i><sub>m,app</sub> values that spanned over 3 orders of magnitude, both prerequisites for the Catalyst Hypogen algorithm. The dataset comprised 24 substrates (Figure 1<a data-tab="pane-pcw-Figures" href="#jm020397cf00001" id="" class=" internalNav"></a>) and their corresponding <i>K</i><sub>m,app</sub> values, which ranged from 7 to 11 600 μM. <div id="_i5"><figure id="jm020397cf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Structures and <i>K</i><sub>m,app</sub> values of the 24 UGT1A4 substrates in the training and test sets.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Model Generation.</b> The molecular modeling studies were performed using Silicon Graphics Octane2 (Silicon Graphics, Mountain View, CA) and x86 Intel workstations. To test the predictivity of the models constructed (described below), the 24 substrates investigated were divided into two groups:  one to generate the models (18 chemicals) and the second to test the models (6 chemicals). The test set of six chemicals was selected to span a wide range of <i>K</i><sub>m,app</sub> values (47−1570 μM) and for diversity of chemical structure in order to stringently test the models generated. The ability of the models to predict the <i>K</i><sub>m,app</sub> values of these six chemicals was used as the common method of validation. Currently, prediction within 1 log unit of the experimental <i>K</i><sub>m,app</sub> value is considered standard in the drug metabolism field.<sup>10</sup> Randomization tests were employed to assess statistical significance whenever supported by the software. This involved the generation of alternative training sets by randomization of the association between <i>K</i><sub>m,app</sub> and the substrate such that a chemical was not correctly associated with its <i>K</i><sub>m,app</sub> value. Repetition of the model generation procedure with these randomized data sets allowed calculation of the probability that the model was a result of a chance correlation. To achieve a 99% confidence level, 99 randomization tests were necessary while 19 randomization tests were required to achieve a 95% confidence level. </div><div class="NLM_p"><b>Pharmacophore Modeling with Catalyst.</b> Catalyst, version 4.6 (Accelrys, Inc.), was used for modeling. Substrates for UGT1A4 were drawn from the View Compound Workbench using the Visualizer module. Conformer generation was performed using the “Best” function to ensure the widest possible conformer coverage. Each substrate was allowed a maximum of 255 conformers within an energy range of 20 kcal/mol. The default method used by Catalyst for fitting a substrate to a pharmacophore is the “Fast Fit” method. “Fast Fit” finds the optimum fit of a conformer to a pharmacophore without performing an energy minimization on the conformer. The “Best Fit” procedure starts with “Fast Fit” and performs a more thorough search of the torsional and rotational space of the conformer. Common features pharmacophores were developed using the Catalyst HipHop module, and predictive 3D-QSAR pharmacophores were developed using the Catalyst HypoGen module. Catalyst refers to these hypothetical pharmacophores as “hypotheses”, and these two terms will subsequently be used interchangeably. Interfeature spacing for both modules of Catalyst was set to 1 Å because all molecules were relatively small and feature-poor. </div><div class="NLM_p">HipHop hypotheses were developed in an iterative manner by gradually allowing inclusion of a greater number and diversity of chemical features. The final “common features” hypotheses were then analyzed for uniformity and sensibility of mappings of all compounds. Here, the principal measure of “sensibility” is the mapping of the glucuronidation site. In most cases, the best hypothesis was also found to be the lowest “cost” hypothesis. The total energy cost of a generated pharmacophore can be calculated from the deviation between the estimated activity and the observed activity, combined with the complexity of the hypothesis. A null hypothesis can also be calculated on the basis of the presumption that the experimental activities are normally distributed about their mean and that there is no relationship between the fit and the independent data. </div><div class="NLM_p">Hypogen is a Catalyst module that searches for a 3D arrangement of abstract chemical features that explain trends and variations of activity with the chemical structure. It can be used to predict the binding ability of chemicals toward an enzyme or receptor. The fit of the chemicals to the putative pharmacophore (i.e., how closely the features of each chemical can match the features of the pharmacophore) is then linearly correlated to the activity of the chemicals.<sup><a onclick="showRef(event, 'jm020397cb00017'); return false;" href="javascript:void(0);" class="ref jm020397cb00017">17</a></sup></div><div class="NLM_p"><b>Generation of User-Defined Glucuronidation Features within Catalyst.</b> Early experiments in pharmacophore development using the UGT1A4 dataset highlighted some difficulties caused by using the default hydrogen bond acceptor (HBA) and hydrogen bond donor (HBD) features to recognize glucuronidation sites within the substrates. For example, a problem was discovered with the choice of the default Catalyst <a name="jm004600901619" id="jm004600901619"></a> HBA feature because Catalyst does not consider sp<sup>2</sup> nitrogen atoms adjacent to aromatic rings (e.g., aniline-type nitrogens) as possessing a lone pair of electrons. Such atomic features are automatically assigned within the Catalyst drawing module and are the default, and consequently, it was not possible to map aniline-type nitrogen atoms to an HBA-based feature. Thus, design of an HBD-based glucuronidation feature was undertaken. Such a feature maps a vector in the direction of the hydrogen. A difficulty in using an HBD-based glucuronidation feature is to ensure that the feature recognizes tertiary amines as glucuronidation sites because they do not have a hydrogen. This was overcome by allowing the vector to map in the direction of the lone pair in the specific case of a tertiary nitrogen atom. This compromise is necessary in order to achieve sufficiently general recognition of conjugation sites. Such a compromise needs to be borne in mind when considering the mechanistic interpretation of mappings of the “exception” substrates. It should also be noted that an analogous approach could be applied to the previous HBA-based feature; the aniline type hydrogens could be allowed to map in the direction of the lone pair vector in specific cases. </div><div class="NLM_p">A third alternative can be considered whereby the glucuronidation feature has no associated vector. Pharmacophores containing such a feature may sacrifice specificity (because <a name="jm004600901620" id="jm004600901620"></a> only one feature, not two, are used), but the feature solves the difficulties alluded to above regarding substrate site recognition. This “general” glucuronidation feature performs very well and is widely recognized. All new features were added to the Feature Dictionary of Catalyst. </div><div class="NLM_p"><b>HBA-Based Glucuronidation Vector (HBA-Glc. Vct.).</b> An “HBA lipid” chemical function was edited to recognize only glucuronidation sites using the “View Hypothesis” workbench. This involved removal of the capability to recognize features such as ethers, esters, aromatic heteroatoms, etc. However, this function cannot recognize sp<sup>2</sup> nitrogen atoms adjacent to aromatic rings (i.e., aniline-type nitrogens) because Catalyst automatically assigns such atoms as lacking a lone pair of electrons (<i>n </i>= 0). Hence, they are considered unable to act as HBAs, despite the well-known nucleophilic nature of such atoms. </div><div class="NLM_p"><b>HBD-Based Glucuronidation Vector (HBD-Glc. Vct.).</b> In addition, an HBD chemical function was edited to recognize tertiary amines and map the vector in the direction of the lone pair electrons from the “View Hypothesis” workbench. This allows the chemical function to recognize all glucuronidation sites in the dataset, although a mechanistic interpretation may be less valid. Editing was achieved by altering the atom specifications for an “HBA lipid” chemical function to recognize only a tertiary nitrogen, and then this function was included as an “Or” option within the original HBD definition. Furthermore, the original HBD chemical function was modified to exclude primary and secondary amides as HBDs because these are not glucuronidation sites. Validation of the feature was achieved by testing its ability to recognize known glucuronidation sites of UGT1A4 substrates. </div><div class="NLM_p"><b>Glucuronidation Sphere (Glc. Sph.).</b> An “HBD” chemical function was modified in order to recognize the heavy atom (O, S, N) of any potential glucuronidation site. By use of the “View Hypothesis” workbench of Catalyst, vector associations were removed from the original HBD feature. Atom specifications were altered to prevent recognition of aromatic heteroatoms (O, S, N) and imine and amide nitrogens because these chemical features are not sites of glucuronidation (with the rare exception of some aromatic N centers, which are not present in this data set). Again, testing its ability to recognize known glucuronidation sites of UGT1A4 substrates validated the feature. </div><div class="NLM_p"><b>Modeling with SOMFA.</b> Self-organizing molecular field analysis (SOMFA) is based on a 3D-QSAR algorithm, similar to CoMFA, that can detect the 3D positions in the electrostatic and shape fields that influence the activities of a set of aligned molecules.<sup><a onclick="showRef(event, 'jm020397cb00018'); return false;" href="javascript:void(0);" class="ref jm020397cb00018">18</a></sup> The algorithm calculates the steric and electrostatic fields for a set of aligned conformers and allows visualization of significant interaction points on a ligand. The common features (HipHop) pharmacophore alignment was used to align the molecules for SOMFA model generation. The aligned chemicals were then input to the program along with the −log <i>K</i><sub>m,app</sub>. The shape and electrostatic field were sampled at 1 Å intervals over a 3D grid. The output consisted of a set of points in 3D that represent the regions of space where the shape or electrostatics of the substrate influenced the <i>K</i><sub>m,app</sub>. </div><div class="NLM_p"><b>2D-QSAR </b><b>Model</b><b>ing.</b> A wide range of 2D descriptors was calculated from the structure of each chemical using the Dragon (Milano Chemometrics and QSAR Research Group, Milan, Italy) and Cerius2 (Accelrys, Inc.) programs. These included topological, 2D autocorrelation, constitutional, BCUT, thermodynamic, and electronic descriptors.<sup>34</sup> Before generation of the model, the least collinear and most relevant subsets of descriptors were selected and all descriptors without a statistically significant correlation to −log <i>K</i><sub>m,app</sub> were removed. The significantly correlated descriptors were input to a program implementing the Unsupervised Forward Selection (UFS) algorithm.<sup><a onclick="showRef(event, 'jm020397cb00019'); return false;" href="javascript:void(0);" class="ref jm020397cb00019">19</a></sup> This method selects the two least well-correlated descriptors and then selects additional variables on the basis of their multiple correlation with those already chosen, thereby selecting a subset of variables that are as close to orthogonal as possible. Subsets of descriptors were selected using this procedure with varying degrees of collinearity allowed (<i>r</i><sup>2</sup><sub>max</sub> = 0.1, 0.2, ..., 0.9, 0.99). </div><div class="NLM_p last">A regression model was built for each descriptor subset using principal component (PCR) regression. The variable subset giving the best leave-one-out (LOO) cross-validated <i>r</i><sup>2</sup>, a measure of predictive ability of the model, was chosen for further optimization. Any descriptor found to have a significant correlation with the residuals of the model was included if it improved the LOO cross-validated <i>r</i><sup>2</sup>. Descriptors were omitted from the model if the removal resulted in an increase in the LOO cross-validated <i>r</i><sup>2</sup>. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07432" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07432" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Pharmacophore Modeling of UGT1A4 Substrates.</b> Catalyst identifies the 3D spatial orientation and overall range of structural features associated with the ligands and uses them to define a pharmacophore. Subsequently, Catalyst can be used in two ways. Identification of the features common to all substrates generates a qualitative “common features” model that highlights the minimum necessary 3D arrangement of features that are necessary for binding. Alternatively, the “fit” of a ligand to a pharmacophore can be correlated with a measure of binding affinity (e.g., <i>K</i><sub>m,app</sub>) to generate a quantitative model potentially capable of predicting that measure of binding affinity. </div><div class="NLM_p"><b>“Common Features” Pharmacophore Model.</b> The Catalyst HipHop module was used to generate a “common features” model. The initial aim was to deduce the distance(s) between one glucuronidation site and one other common feature for all 24 substrates, since this represents the simplest pharmacophore that can be modified progressively. To achieve this, the vector-free glucuronidation feature (Glc. Sph., see Materials and Methods), which recognized all 24 substrates, was used. The occurrence of this feature in a pharmacophore was set to a minimum of 1 (i.e., each pharmacophore had to include one glucuronidation feature), and Catalyst was given the option of choosing only one other feature from the default hydrophobe, HBA “lipid” or HBD features. The two solutions consisted of a glucuronidation feature and a hydrophobic feature separated by either 3 or 6.2 Å. With the exception of 4-nitrophenol, all substrates could be mapped to the glucuronidation feature and the hydrophobe feature of each pharmacophore. Catalyst interpreted 4-nitrophenol as containing a glucuronidation feature but no hydrophobe; hence, the substrate was mapped only to the glucuronidation feature of each pharmacophore. </div><div class="NLM_p">No single vectorized glucuronidation function could be designed to recognize the glucuronidation site of every molecule. For example, the HBA-based glucuronidation feature could not recognize the sites in the biphenylamines; hence, the HBD-based feature must be used for these substrates even though tertiary amines are not recognized by the HBD-based feature. Hence, the dataset was arbitrarily classified into groups comprising biphenyls, tertiary amines, steroids, and small molecules (see Table 2<a class="ref showTableEvent internalNav" data-ID="jm020397ct00001" href="#jm020397ct00001"></a><a class="ref showTableEvent internalNav" data-ID="jm020397ct00002" href="#jm020397ct00002"></a>). This allowed combinations of groups to be investigated with vectorized glucuronidation features. <div class="NLM_table-wrap" id="jm020397ct00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  Test Set Predicted <i>K</i><sub>m,app</sub> Values and Logarithm of the Residuals for the Three Models</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">substrate</td><td class="colsep2 rowsep2">obsd <i>K</i><sub>m,app</sub></td><td class="colsep2 rowsep2">pharmacophore predicted <i>K</i><sub>m,app</sub> <i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">SOMFA predicted <i>K</i><sub>m,app</sub> <i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">2D-QSAR predicted <i>K</i><sub>m,app</sub> <i><sup>a</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">2-aminobiphenyl </td><td class="colsep2 rowsep2">47 </td><td class="colsep2 rowsep2">190 (0.6) </td><td class="colsep2 rowsep2">190 (0.6) </td><td class="colsep2 rowsep2">270 (0.7) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">clozapine </td><td class="colsep2 rowsep2">62 </td><td class="colsep2 rowsep2">80 (0.1) </td><td class="colsep2 rowsep2">160 (0.4) </td><td class="colsep2 rowsep2">100 (0.2) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">retigabine </td><td class="colsep2 rowsep2">322 </td><td class="colsep2 rowsep2">70 (0.6) </td><td class="colsep2 rowsep2">250 (0.1) </td><td class="colsep2 rowsep2">260 (0.1) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">menthol </td><td class="colsep2 rowsep2">470 </td><td class="colsep2 rowsep2">150 (0.5) </td><td class="colsep2 rowsep2">260 (0.4) </td><td class="colsep2 rowsep2">1200 (0.4) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">carvacrol </td><td class="colsep2 rowsep2">1500 </td><td class="colsep2 rowsep2">190 (0.9) </td><td class="colsep2 rowsep2">190 (0.9) </td><td class="colsep2 rowsep2">430 (0.5) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">eugenol </td><td class="colsep2 rowsep2">1570 </td><td class="colsep2 rowsep2">150 (1.0) </td><td class="colsep2 rowsep2">390 (0.6) </td><td class="colsep2 rowsep2">1100 (0.2)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> <i>K</i><sub>m,app</sub> values in units of μM. Values in parentheses represent the log of the residuals (i.e., log of observed value minus predicted value).</p></div></div><div class="NLM_table-wrap" id="jm020397ct00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Substrate Groups from Which the “Common Features” Pharmacophore Was Developed</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">biphenyls</td><td class="colsep2 rowsep2">tertiary amines</td><td class="colsep2 rowsep2">steroids</td><td class="colsep2 rowsep2">small molecules </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">2-aminobiphenyl </td><td class="colsep2 rowsep2">loxapine </td><td class="colsep2 rowsep2">5α-pregnane-3β,20α-diol </td><td class="colsep2 rowsep2">menthol </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">4-aminobiphenyl </td><td class="colsep2 rowsep2">desmethylclozapine </td><td class="colsep2 rowsep2">tigogenin </td><td class="colsep2 rowsep2">borneol </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">diphenylamine </td><td class="colsep2 rowsep2">amitriptyline </td><td class="colsep2 rowsep2">diosgenin </td><td class="colsep2 rowsep2">carvacrol </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">retigabine </td><td class="colsep2 rowsep2">clozapine </td><td class="colsep2 rowsep2">5α-androstane-3α,17β-diol </td><td class="colsep2 rowsep2">eugenol </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">daidzein </td><td class="colsep2 rowsep2">ketotifen </td><td class="colsep2 rowsep2">hecogenin </td><td class="colsep2 rowsep2">4-nitrophenol </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">benzidine </td><td class="colsep2 rowsep2">chlorpromazine </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">cyproheptadine </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">imipramine </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr></tbody></table></div></div></div><div class="NLM_p">Each group was searched for common features, using a vectorized glucuronidation feature, and then combinations of the groups were searched. However, the small molecules group, which ranged in <i>K</i><sub>m,app</sub> from 470 to 11 600 μM, was excluded from pharmacophore <a name="jm004600901621" id="jm004600901621"></a> generation procedures because of the molecules' small size and feature poorness. Analysis of the biphenyl group for the common spatial arrangement of features generated only one pharmacophore consisting of two hydrophobic regions and one HBD-based glucuronidation feature (HBD-Glc. Vct). The arrangement of features is a triangular combination of the two distances found to be common to all 24 substrates in the initial simplified model (i.e., 3 and 6.2 Å) and is shown in Figure 2<a data-tab="pane-pcw-Figures" href="#jm020397cf00002" id="" class=" internalNav"></a>a. <div id="_i8"><figure id="jm020397cf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00002.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 (a) The “common features” pharmacophore model developed using the HipHop module of Catalyst. Cyan spheres represent hydrophobic regions, and purple represents the “glucuronidation” feature. (b) Eugenol mapped to the Hypogen pharmacophore. (c) Clozapine mapped to the Hypogen pharmacophore.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The common spatial arrangement of features for the combined biphenyl and steroid group was then investigated. The highest ranked pharmacophore for this substrate group shared the same features in the same geometry as that shown in Figure 2a (i.e., two hydrophobes and a HBD-Glc. Vct.). Substrates in this group varied in <i>K</i><sub>m,app</sub> from 7 to 460 μM. Similarly, the common spatial arrangements of features for the combined biphenyl and tertiary amine groups were examined, and the highest ranked pharmacophore for this substrate group was also the same as that shown in Figure 2a. Substrates in this group varied in <i>K</i><sub>m,app</sub> between 27 and 460 μM. Parts b and c of Figure 2 illustrate two representative UGT1A4 substrates, eugenol (which fitted despite exclusion from the training set) and clozapine, mapped to the “common features” pharmacophore. Table 3<a class="ref showTableEvent internalNav" data-ID="jm020397ct00003" href="#jm020397ct00003"></a> shows the number of features of the common features pharmacophore mapped by a given substrate, conformer number and energy, reported default fit value, and maximium omitted features (MOF) setting used for the hypothesis generation. <div class="NLM_table-wrap" id="jm020397ct00003"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 3.  Number of Features of the Common Features Pharmacophore Mapped by a Given Substrate, Conformer Number and Energy, Reported Default Fit Value, and Maximium Omitted Features (MOF) Setting Used for the Hypothesis Generation</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">UGT1A4 substrate</td><td class="colsep2 rowsep2">features mapped</td><td class="colsep2 rowsep2">conformer number</td><td class="colsep2 rowsep2">conformer energy<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">fit value</td><td class="colsep2 rowsep2">MOF<i><sup>b</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">tigogenin </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">17.6 </td><td class="colsep2 rowsep2">1.9 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">pregnane </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">8 </td><td class="colsep2 rowsep2">19.7 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">diosgenin </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">18.3 </td><td class="colsep2 rowsep2">1.7 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">androstane </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">14.2 </td><td class="colsep2 rowsep2">1.7 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">hecogenin </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">13.2 </td><td class="colsep2 rowsep2">1.8 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">loxapine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">17 </td><td class="colsep2 rowsep2">1.4 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">2-aminobiphenyl </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">12.6 </td><td class="colsep2 rowsep2">2.2 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">desmethylclozapine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">19.3 </td><td class="colsep2 rowsep2">2.7 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">amitriptyline </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">25 </td><td class="colsep2 rowsep2">5.9 </td><td class="colsep2 rowsep2">0.4 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">clozapine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">3.7 </td><td class="colsep2 rowsep2">2.9 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">4-aminobiphenyl </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">1.6 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ketotifen </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">9.5 </td><td class="colsep2 rowsep2">0.5 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">chlorpromazine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">8 </td><td class="colsep2 rowsep2">18.7 </td><td class="colsep2 rowsep2">2.1 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">cyproheptadine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">4.7 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">diphenylamine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">6.8 </td><td class="colsep2 rowsep2">0.6 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">imipramine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">4.6 </td><td class="colsep2 rowsep2">1.8 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">retigibine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">52 </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">daidzein </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">0.4 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">benzidine </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">0.4 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(+)-menthol </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(−)-<i>endo</i>-borneol </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">−1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">carvacrol </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">0.2 </td><td class="colsep2 rowsep2">1.9 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">eugenol </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">1.3 </td><td class="colsep2 rowsep2">2.8 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">4-nitrophenol </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">−1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Conformer energy has units of kcal/mol.<i><sup>b</sup></i><sup></sup> Maximum omitted features (MOF) = −1 means that Catalyst considers all possible mappings and finds the best possible fit. MOF = 0 means that Catalyst ignores mappings that involve a subset of features.</p></div></div></div><div class="NLM_p"><b>Pharmacophore-Based 3D-QSAR Model.</b> The HypoGen module of Catalyst was used to develop a 3D-QSAR for UGT1A4 using published <i>K</i><sub>m,app</sub> values shown in Figure 1 for a training set of 18 structurally diverse substrates of UGT1A4.<sup>15,16</sup> For model generation, the HBD-based glucuronidation feature (HBD-Glc. Vct.) developed here and the default hydrophobic, hydrogen bond acceptor, hydrogen bond donor, and ring aromatic features were chosen as feature options. Of the 10 hypotheses returned, the lowest cost model was assessed as the best. This assessment was based not only on the Catalyst cost analysis but also on the possession of features that were representative of the remaining hypotheses, the highest correlation coefficient (<i>r</i>), and the best prediction of the test set. A high correlation (<i>r </i>= 0.94; <i>r</i><i><sup>2</sup></i><sup></sup> = 0.88) between the experimentally derived and predicted <i>K</i><sub>m,app</sub> values for the training set was obtained, suggestive of a high-quality model. Figure 3<a data-tab="pane-pcw-Figures" href="#jm020397cf00003" id="" class=" internalNav"></a>a demonstrates the structural features and geometry of the pharmacophore derived using Hypogen. Mappings of imipramine and benzidine to the hypogen pharmacophore are shown in parts b and c of Figure 3, respectively. <div id="_i10"><figure id="jm020397cf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00003.jpeg" id="_i11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 (a) Pharmacophore-based 3D-QSAR model developed using the Hypogen module of Catalyst. Cyan spheres represent hydrophobic regions, and purple represents the “glucuronidation” feature. (b) Imipramine mapped to the Hypogen pharmacophore. (c) Benzidine mapped to the Hypogen pharmacophore.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><a name="jm004600901622" id="jm004600901622"></a>Further validation of the predictive 3D-QSAR model was undertaken by permutation and test set prediction. The statistical significance of the retrieved hypothesis was verified by randomizing the assignment of a dependent variable to a structure (i.e., the <i>K</i><sub>m,app</sub> of each structure was randomly assigned to a different structure and the hypothesis generation procedure was repeated). This permutation process was repeated 19 times without retrieving a more “significant” (lower cost) hypothesis, indicating that the original model was unlikely to be a chance event (<i>p</i> < 0.05). A permuted hypothesis is considered more “significant” if the associated “hypothesis cost” is closer to the “fixed cost” than the original hypothesis cost (see Materials and Methods for description of “cost”). </div><div class="NLM_p"><i>K</i><sub>m,app</sub> values for four of the six compounds comprising the validation set were estimated within 0.5 log unit of the experimentally derived <i>K</i><sub>m,app</sub> (see Table 1), with eugenol having an error slightly greater than 1 log unit. </div><div class="NLM_p"><b>SOMFA Model.</b> A 3D-QSAR model was developed using the SOMFA methodology of Robinson and co-workers.<sup>18</sup> Using the HipHop derived pharmacophore alignment, the same training set molecules (Figure 1) were imported into the SOMFA software program, and the electrostatic and steric maps were calculated for each. The resulting model showed a good correlation (<i>r</i> = 0.85; <i>r</i><i><sup>2</sup></i><sup></sup> = 0.73) for the training set using the shape grid potentials. All six of the molecules in the test set were then predicted within 1 log unit using the model (See Table 1). </div><div class="NLM_p"><b>2D-QSAR Model.</b> A 2D QSAR model was constructed with Cerius2 software (Accelrys, Inc.) using a range of chemical descriptors and the principal component (PCR) regression methodology. Of the 319 descriptors calculated for each chemical using the Dragon and Cerius2 software, 208 were found to be significantly correlated to −log <i>K</i><sub>m,app</sub> at the 95% confidence level. The UFS with an <i>r</i><sup>2</sup><sub>max</sub> of 0.99 gave a 21-descriptor model with the best LOO cross-validated <i>r</i><sup>2</sup> of 0.68. Sixteen descriptors in the model were found to increase the LOO cross-validated <i>r</i><sup>2</sup> when removed from the model generation process. Thus, the optimum model was found to be a 5-descriptor, 1-component model with <i>r</i><sup>2</sup> = 0.80 (<i>r </i>= 0.89) and LOO cross-validated<i> r</i><sup>2</sup> = 0.73. </div><div class="NLM_p">The model using the nonstandardized descriptors (i.e., calculated directly) is given by<a class="ref internalNav" href="#jm020397ce00001" aria-label=""></a><span class="NLM_disp-formula" id="jm020397ce00001"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397ce00001.gif" alt="" id="_i12" /></img></span> Standardization of the descriptors for the particular training set involves mean centering and unit normalization (by subtracting the mean and dividing by the standard deviation). Using standardized descriptors gives an indication of the relative contribution of each descriptor to the equation<a class="ref internalNav" href="#jm020397ce00002" aria-label=""></a><span class="NLM_disp-formula" id="jm020397ce00002"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397ce00002.gif" alt="" id="_i13" /></img></span><a name="jm004600901623" id="jm004600901623"></a> where </div><div class="NLM_p">(1) AlogP98 is log of the octanol/water partition coefficient calculated by an atom-based method, </div><div class="NLM_p">(2) MATS3v is the Moran autocorrelation of path length 3 weighted by atomic van der Waals volumes, </div><div class="NLM_p">(3) BENV1 is the negative burden eigenvalue 1 weighted by atomic van der Waals volumes, </div><div class="NLM_p">(4) MATS5e is the Moran autocorrelation of path length 5 weighted by atomic Sanderson electronegativities, and </div><div class="NLM_p">(5) nRO5 is number of five-membered rings. </div><div class="NLM_p last">The p<i>K</i><sub>m,app</sub> was permuted 100 times, and models were generated for each permution. No model resulted in as good a fit (<i>r</i><sup>2</sup>) as the original, indicating that the model is highly unlikely (<i>p</i> < 0.01) to be a result of a chance correlation. Furthermore, the average “leave <i>N</i> out” cross-validated <i>r</i><sup>2</sup> of 100 trials with <i>N</i> substrates omitted (<i>N</i> = 2−7) was performed to check that the model is likely to generalize to novel chemicals, and the results are summarized in Table 4<a class="ref showTableEvent internalNav" data-ID="jm020397ct00004" href="#jm020397ct00004"></a>. The models made with these descriptors had good CV <i>r</i><sup>2</sup> values, between 0.79 (<i>n</i> = 2, 11% of the set) and 0.75 (<i>n </i>= 7, 39% of the set), and showed low variability indicating a reliable model suited to generalization.<sup>19</sup><div class="NLM_table-wrap" id="jm020397ct00004"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 4.  Average “Leave <i>N</i> Out” (L<i><sub>N</sub></i>O) Cross-Validated <i>r</i><sup>2</sup> of 100 Trials, with <i>N</i> Substrates Omitted (<i>N</i> = 2−7) from the 18 Training Set Chemicals, and the Associated Standard Deviations</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<i><sub>N</sub></i>O</td><td class="colsep2 rowsep2">% chemicals left out</td><td class="colsep2 rowsep2">cross-validated <i>r</i><sup>2</sup></td><td class="colsep2 rowsep2">standard deviation </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>1</sub>O </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">0.791 </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>2</sub>O </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">0.800 </td><td class="colsep2 rowsep2">0.015 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>3</sub>O </td><td class="colsep2 rowsep2">17 </td><td class="colsep2 rowsep2">0.795 </td><td class="colsep2 rowsep2">0.025 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>4</sub>O </td><td class="colsep2 rowsep2">22 </td><td class="colsep2 rowsep2">0.793 </td><td class="colsep2 rowsep2">0.026 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>5</sub>O </td><td class="colsep2 rowsep2">28 </td><td class="colsep2 rowsep2">0.784 </td><td class="colsep2 rowsep2">0.061 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>6</sub>O </td><td class="colsep2 rowsep2">33 </td><td class="colsep2 rowsep2">0.775 </td><td class="colsep2 rowsep2">0.05 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">L<sub>7</sub>O </td><td class="colsep2 rowsep2">39 </td><td class="colsep2 rowsep2">0.752 </td><td class="colsep2 rowsep2">0.08</td></tr></tbody></table></div></div></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86627" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86627" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Determination of a pharmacophore represents an early, but significant, step toward the understanding of a given receptor−ligand binding event. However, while satisfaction of the pharmacophore criteria by a molecule is necessary for a binding interaction to occur, it may not be sufficient because many of the steric and electronic influences on the binding event are ignored in such an approach. Therefore, a 3D-QSAR modeling approach that more specifically investigates such influences, such as SOMFA, could complement the pharmacophore approach well. Still, there are other molecular determinants left unexplored by such approaches; hence, a 2D-QSAR approach is potentially beneficial because it allows exploration of an even wider range of molecular determinants that may explain observed phenomena. Thus, the aim of the research reported here was to determine whether a complementary range of models could be developed that describe the features necessary for a compound to undergo glucuronidation by UGT1A4 and aid in the prediction of <i>K</i><sub>m,app</sub> for putative substrates. Previous 2D- and 3D-QSAR models of UGT substrates/inhibitors have not been generally applicable because they have used compounds of low structural diversity (usually with a common substructure), making extrapolation to structurally diverse datasets virtually impossible.<sup><a onclick="showRef(event, 'jm020397cb00020 jm020397cb00021 jm020397cb00022 jm020397cb00023 jm020397cb00024 jm020397cb00025 jm020397cb00026 jm020397cb00027 jm020397cb00028'); return false;" href="javascript:void(0);" class="ref jm020397cb00020 jm020397cb00021 jm020397cb00022 jm020397cb00023 jm020397cb00024 jm020397cb00025 jm020397cb00026 jm020397cb00027 jm020397cb00028">20-28</a></sup></div><div class="NLM_p">Substrates of phase I and II enzymes share a binding mode whereby the atom at which reaction occurs must be located at a specific position in the active site for catalysis to occur. For substrates of UGT, this site is adjacent to the bound cofactor UDP-glucuronic acid. Hence, the minimum requirement for the alignment (i.e., pharmacophore) is the mutual overlay of the reactive site for each substrate. Indeed, the routine incorporation of a “site of reactivity” feature into other enzyme-based pharmacophore investigations should generally aid in the alignment process. Such an approach decreases the complexity of the “pharmacophore space” to be searched, thereby increasing the chance of finding a chemically sensible alignment between substrates. Successful pharmacophore developments for CYP have included the site of oxidation as a pharmacophore feature. For example, de Groot et al anchored the substrate oxidation site 3 Å from the iron atom.<sup><a onclick="showRef(event, 'jm020397cb00029 jm020397cb00030'); return false;" href="javascript:void(0);" class="ref jm020397cb00029 jm020397cb00030">29,30</a></sup> However, the reported CYP pharmacophores developed using Catalyst<sup>7-12</sup> make no mention of whether the alignments from the pharmacophores result in the overlay of a common site of metabolism. This deficiency was addressed recently in the development of a pharmacophore model of CYP2B6, using Catalyst, which incorporated the site of oxidation as a pharmacophore alignment feature.<sup>14</sup> It should be noted, however, that this model cannot be used to search a database of potential substrates because the “site of reactivity” feature is defined only for the training set of compounds and will not recognize the site of reactivity in other chemicals. </div><div class="NLM_p">There is general acceptance that UGT-catalyzed conjugation proceeds by a second-order nucleophilic substitution (S<sub>N</sub>2) mechanism<sup><a onclick="showRef(event, 'jm020397cb00031'); return false;" href="javascript:void(0);" class="ref jm020397cb00031">31</a></sup> and hence requires a nucleophilic site within a molecule. Furthermore, Yin and co-workers propose that a basic residue within the active site may deprotonate aglycons with a p<i>K</i><sub>a</sub> high enough to require it (e.g., a phenol) prior to reaction.<sup>27</sup> Although sufficient kinetic data are not available, the concept of this mechanism can be expanded to substrates with lower p<i>K</i><sub>a</sub> that are sufficiently nucleophilic that they do not require initial deprotonation but do require deprotonation of the intermediate transition state. This hydrogen-bonding interaction potentially represents a significant contribution in binding affinity to the protein, thus justifying the inclusion of a “glucuronidation feature” based on a HBD site in the pharmacophore. Hence, for a site to undergo UGT-catalyzed conjugation, the <i>general</i> requirement is a nucleophilic heteroatom attached to a proton (e.g., SH, OH, NH). There are, however, exceptions that must be borne in mind when considering the mechanism by which UGTs function. Examples such as tertiary amine or carbon glucuronidation cause a paradox in relation to describing a general pharmacophore feature that represents the glucuronidation site. HBDs do not accurately represent tertiary amines, and HBAs do not accurately represent C-based glucuronidation sites. In general, however, a HBA feature (with its implicit lone pair) is probably the most representative. </div><div class="NLM_p">The “common features” pharmacophore allows the inference that the general features responsible for the binding of UGT1A4 substrates with a <i>K</i><sub>m,app</sub> between 5 and 500 μM are two hydrophobic regions and a region <a name="jm004600901624" id="jm004600901624"></a> occupied by the glucuronidation site, in the geometry highlighted in Figure 2a. Hence, the pharmacophore provides a useful tool for gaining insight into the UGT active site in the absence of a crystal structure. However, these general features do not necessarily explain all the interactions of higher affinity substrates. For example, it is likely that steroids benefit from a second hydrogen-bonding interaction at the ring most distal to the site of glucuronidation. It was noted that a number of higher affinity substrates shared a relatively common region of HBA groups near the 6.5 Å hydrophobe when aligned, although not within sufficient tolerance to allow Catalyst to assign a feature to the region. It is hypothesized that an HBA feature within 1−2 Å of this hydrophobic site may account for some of the selectivity of the higher affinity substrates. It has been recognized for some time that substrate hydrophobicity (lipophilicity) is essential for glucuronidation by hepatic UGTs.<sup>25-27,31</sup> Interestingly, the interfeature distances highlighted (3−4 Å and 6.2−6.5 Å) in the “common features” pharmacophore parallel those proposed for rat UGT1A6.<sup><a onclick="showRef(event, 'jm020397cb00032'); return false;" href="javascript:void(0);" class="ref jm020397cb00032">32</a></sup> For the first time, this study sheds light on the geometry of the hydrophobic regions within the substrates and hence provides the beginning of a deeper, mechanistic understanding of the human UGT isoforms. Pharmacophore models developed concurrently for UGT1A1 in this laboratory indicate that this isoform may share a similar set of common features.<sup><a onclick="showRef(event, 'jm020397cb00033'); return false;" href="javascript:void(0);" class="ref jm020397cb00033">33</a></sup></div><div class="NLM_p">Although not quantitative, the “common features” model has potential for qualitative predictions of the preferred site of conjugation in molecules with multiple glucuronidation sites. For example, clozapine is known to have two sites of conjugation:  one secondary site and one tertiary amine site.<sup>15</sup> It is interesting to note that this model correctly predicts the major conjugation site (secondary amine) (see Figure 2c) as evidenced by the default “fast fit” alignment on the “glucuronidation” feature of the pharmacophore. Several of the small molecules (for menthol, carvacrol, eugenol, 500 < <i>K</i><sub>m,app</sub> < 1600 μM) also fit the model very well (e.g., Figure 2b) despite being excluded from the training set used to develop the model. In addition to potentially accounting for multiple binding modes, this adds further credence to the model regarding the minimum features necessary for binding. </div><div class="NLM_p">Of the predictive models, the 2D-QSAR exhibited the best overall predictivity for the chosen test set. To further investigate the robustness of the 2D-QSAR model for prediction, it is useful to test with more than one test set. One method for doing this is to split the dataset into training and test sets in multiple ways and to determine the average prediction accuracy of the multiple test sets. Leave-<i>N</i>-out cross validation allows this and is, in essence, a validation with hundreds of test sets of differing composition. Hence, our 2D-QSAR model was further assessed by determining the average “leave-<i>N</i>-out” cross-validated <i>r</i><sup>2</sup> of 100 trials with <i>N</i> substrates omitted (<i>N</i> = 2−7). Table 4 highlights these data and the associated standard deviations. The CV <i>r</i><sup>2</sup> values were good, between 0.80 (<i>n </i>= 2, 11% of the set) and 0.75 (<i>n</i> = 7, 39% of the set), and showed low variability indicating good prediction independent of the particular test set chosen. These values better represent the expected outcome<sup>18</sup> when predicting the activities of other chemicals than does a leave-one-out cross-validated <i>r</i><sup>2</sup>. A direct comparison of these results, unlike the single test set results, cannot be made with the 3D-QSAR models because <i>n</i>-fold cross-validation is not implemented in the software. </div><div class="NLM_p">Qualitative interpretation of the descriptors in the 2D-QSAR model is not simple because many molecular descriptors represent a number extracted by a well-defined algorithm from a molecular representation of a complex system.<sup><a onclick="showRef(event, 'jm020397cb00034'); return false;" href="javascript:void(0);" class="ref jm020397cb00034">34</a></sup> However, log <i>P</i> (AlogP98) indicates that substrate hydrophobicity is important for activity. The importance of substrate hydrophobicity may be due to properties of the active site of the enzyme and/or the membrane environment of the enzyme. Since UGTs are believed to be located on the lumenal face of the microsomal membrane, substrates must traverse the endoplasmic reticulum to gain access to the active site.<sup>31</sup> Inclusion of the descriptor nRO5 (the number of five-membered rings) may be due to the high-activity steroids present in the dataset but may contain “hidden” information relating to hydrophobicity also. Physical interpretation of the BCUT descriptor (BENV1) and the two 2D autocorrelation descriptors (MATS3V, MATS5e) is more difficult. It is noteworthy, however, that BCUT descriptors have been shown previously to incorporate connectivity information and atomic properties relevant to receptor−ligand intermolecular interactions.<sup><a onclick="showRef(event, 'jm020397cb00035'); return false;" href="javascript:void(0);" class="ref jm020397cb00035">35</a></sup> Furthermore, the autocorrelation descriptors encapsulate volume (MATS3V) and electronegativity (MATS5e) information associated with atom pairs separated by three and five bonds, respectively.<sup>34</sup></div><div class="NLM_p">Two 3D-QSARs were generated using Catalyst and SOMFA. The same sets of pharmacophoric features (two hydrophobes and a glucuronidation feature) were elucidated using the Catalyst Hypogen module (3D-QSAR) as were elucidated using HipHop, although the geometry was different. Hence, the discrimination was achieved on the basis of the differing “fit” of molecules as opposed to differing features of the highly specific substrates vs the low-specificity substrates. The Hypogen model showed a high correlation (<i>r</i> = 0.94; <i>r</i><i><sup>2</sup></i><sup></sup> = 0.89) between predicted and experimental <i>K</i><sub>m,app</sub> and was able to fit the 18 molecules in the training set to within 0.3 log unit of the experimentally determined <i>K</i><sub>m,app</sub>. Table 1 shows the predicted <i>K</i><sub>m,app</sub> for the test set of six molecules using the Hypogen model, five of which were predicted to be within 1 log unit. This currently represents the level of predictivity being achieved by most literature reports within the drug metabolism field.<sup>10</sup> It should be noted that some of the mappings for the predictive pharmacophore-based 3D-QSAR do not represent catalytically “sensible” mappings (e.g., Figure 3c) because the site of conjugation is not mapped to the glucuronidation function of the pharmacophore. For example, by use of Catalyst's default “Fast Fit” method, imipramine maps to all three features whereas benzidine maps preferentially only to the two hydrophobes, omitting the glucuronidation site. The Hypogen algorithm searches for structural differences to explain differences in activity, thus causing it to ignore the common scaffold when assigning pharmacophoric features.<sup><a onclick="showRef(event, 'jm020397cb00036'); return false;" href="javascript:void(0);" class="ref jm020397cb00036">36</a></sup> Hence, a situation arises whereby in order to account for some feature-poor or small compounds that bind with moderate affinity, it is necessary for the <a name="jm004600901625" id="jm004600901625"></a> Hypogen algorithm to allow compounds to map with high precision but to only two of the three features. For example, using Catalyst's default “Fast Fit” method, imipramine (<i>K</i><sub>m,app</sub> = 310 μM) maps to all three features but poorly, whereas small, feature-poor benzidine (<i>K</i><sub>m,app</sub> = 460 μM) maps very precisely but only to the two hydrophobes, omitting the glucuronidation site. This results in a nonsense alignment if judged by the requirement of a pharmacophore to overlay the “site of reactivity” as outlined above. The challenges presented to Catalyst by feature-poor or small compounds with moderate to high affinity are discussed further in a review by Sprague.<sup><a onclick="showRef(event, 'jm020397cb00037'); return false;" href="javascript:void(0);" class="ref jm020397cb00037">37</a></sup> This is one of the primary reasons for developing a “common features” model that maps features in a “sensible”, chemically reasonable manner because it has been built up sequentially and logically. Both models contribute useful, yet differing, information. </div><div class="NLM_p">Alignment and conformer selection have been highlighted as the most important stage in molecular field-based 3D-QSAR methods;<sup><a onclick="showRef(event, 'jm020397cb00038'); return false;" href="javascript:void(0);" class="ref jm020397cb00038">38</a></sup> hence, the SOMFA model serves as a partial validation of the “common features” alignment, as well as a model in its own right. The shape-based SOMFA model (<i>r </i>= 0.85; <i>r</i><i><sup>2 </sup></i><sup></sup>= 0.73) implies that the “common features” pharmacophore may provide a useful alignment of the UGT1A4 substrates and additionally may provide useful insights regarding steric interactions that cannot be predicted using only a pharmacophore model. The predictive ability of the SOMFA model, based on the six test chemicals, was comparable with that of the Hypogen model, although both were less accurate in prediction than the 2D QSAR. </div><div class="NLM_p last">As noted above, models generated for CYP have generally achieved prediction similar to that reported here (i.e., 1 log unit). However, increased prediction accuracy is clearly desirable. Important factors with regard to increasing prediction accuracy include (1) a systematic search of existing and new modeling techniques to find those most suitable for UGT and (2) an increase in the availability of isoform substrate/inhibitor selectivity and regioselectivity data. In addressing the first factor, this study provides a comparison of the major types of substrate-based modeling techniques in current use and provides the basis for ongoing studies in the field. While the linear 2D- and 3D-QSAR methods reported here show usefulness, it is possible that nonlinear pattern recognition techniques that are capable of accounting for data from more complex relationships will prove to be more appropriate. Research is currently underway in these laboratories to explore this possibility in light of the understanding gained from this study. It is also anticipated that the accuracy of predictive UGT models will further improve as increasing numbers of chemicals are screened against these enzymes. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we have reported here a generalized, complementary approach to the challenge of modeling substrates of UGT isoforms. The work provided important insights into the structural and chemical properties that characterize substrates of UGT1A4 and has allowed an evaluation of several pharmacophore, 2D-, and 3D-QSAR pattern recognition techniques for the quantitative prediction of binding affinity (<i>K</i><sub>m,app</sub>). In particular, the common features pharmacophore provides a chemically intuitive representation of the spatial characteristics of known UGT1A4 substrates and affords a useful tool for gaining insight into the UGT active site in the absence of a crystal structure. The 2D-QSAR highlights the correlation with substrate hydrophobicity among other descriptors. The development of a novel, user-defined “glucuronidation” feature within Catalyst is proposed as a useful aid to the further development of pharmacophore-based UGT models. In terms of quantiatative predictivity, it would appear that 2D-QSAR is more effective than the 3D-QSAR techniques used in this study for prediction of binding affinity (<i>K</i><sub>m,app</sub>). The outcomes of this study will aid in directing further iterations of model development in the UGT substrate modeling field and the drug metabolizing field in general. </div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul A. Smith</span> - <span class="hlFld-Affiliation affiliation">Department of Clinical Pharmacology, Flinders University and Flinders Medical Centre, Bedford Park, 5042, South Australia, and School of Pharmaceutical, Molecular and Biomedical Sciences, University of South Australia, Adelaide, 5001, South Australia </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael J. Sorich</span> - <span class="hlFld-Affiliation affiliation">Department of Clinical Pharmacology, Flinders University and Flinders Medical Centre, Bedford Park, 5042, South Australia, and School of Pharmaceutical, Molecular and Biomedical Sciences, University of South Australia, Adelaide, 5001, South Australia </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ross A. McKinnon</span> - <span class="hlFld-Affiliation affiliation">Department of Clinical Pharmacology, Flinders University and Flinders Medical Centre, Bedford Park, 5042, South Australia, and School of Pharmaceutical, Molecular and Biomedical Sciences, University of South Australia, Adelaide, 5001, South Australia </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John O. Miners</span> - <span class="hlFld-Affiliation affiliation">Department of Clinical Pharmacology, Flinders University and Flinders Medical Centre, Bedford Park, 5042, South Australia, and School of Pharmaceutical, Molecular and Biomedical Sciences, University of South Australia, Adelaide, 5001, South Australia </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm020397c" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48857" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48857" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors gratefully acknowledge Daniel Robinson and the Computational Chemistry Research Group (Oxford University, U.K.) for the use of the SOMFA software and to Milano Chemometrics and QSAR Research Group for use of the Dragon software. </p></div><div class="refs-header-label"><a name="jm004600901626" id="jm004600901626"></a></div><p class="references-count">
            This article references 38 other publications.
        </p><ol id="references" class="useLabel"><li id="jm020397cb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm020397cb00001_1"><span><span class="NLM_string-ref">Roberts, S. A. High-throughput screening approaches for investigating drug metabolism and pharmacokinetics [Review]. <i>  Xenobiotica </i><b>2001</b><i>, 31</i>, 557−589.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2001&pages=589&author=S.+A+Roberts&title=High-throughput+screening+approaches+for+investigating+drug+metabolism+and+pharmacokinetics+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DS.%2BA%26atitle%3DHigh-throughput%2520screening%2520approaches%2520for%2520investigating%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520%255BReview%255D%26jtitle%3DXenobiotica%26date%3D2001%26volume%3D31%26spage%3D589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm020397cb00002_1"><span><span class="NLM_string-ref">Ekins, S.; Waller, C. L.; Swaan, P. W.; Cruciani, G.; Wrighton, S. A.; et al. Progress in predicting human ADME parameters in silico [Review]. <i>  J. </i><i>Pharmacol. Toxicol. Methods </i><b>2000</b><i>, 44</i>, 251−272.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2000&pages=272&author=S.+Ekins&author=C.+L.+Waller&author=P.+W.+Swaan&author=G.+Cruciani&author=S.+A.+Wrighton&title=Progress+in+predicting+human+ADME+parameters+in+silico+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DProgress%2520in%2520predicting%2520human%2520ADME%2520parameters%2520in%2520silico%2520%255BReview%255D%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2000%26volume%3D44%26spage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm020397cb00003_1"><span><span class="NLM_string-ref">Miners, J. O.; Mackenzie, P. I. Drug glucuronidation in humans. <i>  Pharmacol. </i><i>Ther. </i><b>1991</b><i>, 51</i>, 347−369.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=369&author=J.+O.+Miners&author=P.+I+Mackenzie&title=Drug+glucuronidation+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiners%26aufirst%3DJ.%2BO.%26atitle%3DDrug%2520glucuronidation%2520in%2520humans%26jtitle%3DPharmacol.%2520Ther.%26date%3D1991%26volume%3D51%26spage%3D369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm020397cb00004_1"><span><span class="NLM_string-ref">Wrighton, S. A.; Stevens, J. C. The human hepatic cytochromes P450 involved in drug metabolism. <i>  Crit. Rev. Toxicol. </i><b>1992</b><i>, 22</i>, 1−21.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1992&pages=21&author=S.+A.+Wrighton&author=J.+C+Stevens&title=The+human+hepatic+cytochromes+P450+involved+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWrighton%26aufirst%3DS.%2BA.%26atitle%3DThe%2520human%2520hepatic%2520cytochromes%2520P450%2520involved%2520in%2520drug%2520metabolism%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D1992%26volume%3D22%26spage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm020397cb00005_1"><span><span class="NLM_string-ref">Lewis, D. F.; Dickins, M.; Eddershaw, P. J.; Tarbit, M. H.; Goldfarb, P. S. Cytochrome P450 substrate specificities, substrate structural templates and enzyme active site geometries. <i>  Drug Metab. Drug Interact. </i><b>1999</b><i>, 15</i>, 1−49.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=49&author=D.+F.+Lewis&author=M.+Dickins&author=P.+J.+Eddershaw&author=M.+H.+Tarbit&author=P.+S+Goldfarb&title=Cytochrome+P450+substrate+specificities%2C+substrate+structural+templates+and+enzyme+active+site+geometries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DD.%2BF.%26atitle%3DCytochrome%2520P450%2520substrate%2520specificities%252C%2520substrate%2520structural%2520templates%2520and%2520enzyme%2520active%2520site%2520geometries%26jtitle%3DDrug%2520Metab.%2520Drug%2520Interact.%26date%3D1999%26volume%3D15%26spage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm020397cb00006_1"><span><span class="NLM_string-ref">Dai, R.; Pincus, M. R.; Friedman, F. K. Molecular modeling of mammalian cytochrome P450s [Review]. <i>  Cell. Mol. Life Sci. </i><b>2000</b><i>, 57</i>, 487−499.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=499&author=R.+Dai&author=M.+R.+Pincus&author=F.+K+Friedman&title=Molecular+modeling+of+mammalian+cytochrome+P450s+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DR.%26atitle%3DMolecular%2520modeling%2520of%2520mammalian%2520cytochrome%2520P450s%2520%255BReview%255D%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2000%26volume%3D57%26spage%3D499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm020397cb00007_1"><span><span class="NLM_string-ref">Ekins, S.; De Groot, M. J.; Jones, J. P. Pharmacophore and three-dimensional quantitative structure activity relationship methods for modeling cytochrome P450 active sites [Review]. <i>  Drug Metab. Dispos. </i><b>2001</b><i>, 29</i>, 936−944.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=944&author=S.+Ekins&author=M.+J.+De+Groot&author=J.+P+Jones&title=Pharmacophore+and+three-dimensional+quantitative+structure+activity+relationship+methods+for+modeling+cytochrome+P450+active+sites+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DPharmacophore%2520and%2520three-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520methods%2520for%2520modeling%2520cytochrome%2520P450%2520active%2520sites%2520%255BReview%255D%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2001%26volume%3D29%26spage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm020397cb00008_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three- and four-dimensional quantitative structure activity relationship analyses of cytochrome P-450 3A4 inhibitors. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 290</i>, 429−438.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=1999&pages=438&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three-+and+four-dimensional+quantitative+structure+activity+relationship+analyses+of+cytochrome+P-450+3A4+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-%2520and%2520four-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%2520of%2520cytochrome%2520P-450%25203A4%2520inhibitors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D290%26spage%3D438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm020397cb00009_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three and four dimensional-quantitative structure activity relationship (3D/4D-QSAR) analyses of CYP2D6 inhibitors. <i>  Pharmacogenetics </i><b>1999</b><i>, 9</i>, 477−489.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=489&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three+and+four+dimensional-quantitative+structure+activity+relationship+%283D%2F4D-QSAR%29+analyses+of+CYP2D6+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree%2520and%2520four%2520dimensional-quantitative%2520structure%2520activity%2520relationship%2520%25283D%252F4D-QSAR%2529%2520analyses%2520of%2520CYP2D6%2520inhibitors%26jtitle%3DPharmacogenetics%26date%3D1999%26volume%3D9%26spage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00010"><div class="reference"><div class="NLM_citation" id="jm020397cb00010_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three- and four-dimensional-quantitative structure activity relationship (3D/4D-QSAR) analyses of CYP2C9 inhibitors. <i>  Drug Metab. Dispos. </i><b>2000</b><i>, 28</i>, 994−1002.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1002&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three-+and+four-dimensional-quantitative+structure+activity+relationship+%283D%2F4D-QSAR%29+analyses+of+CYP2C9+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-%2520and%2520four-dimensional-quantitative%2520structure%2520activity%2520relationship%2520%25283D%252F4D-QSAR%2529%2520analyses%2520of%2520CYP2C9%2520inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm020397cb00011_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Ring, B. J.; Gillespie, T. A.; Gillespie, J. S.; et al. Three-dimensional quantitative structure activity relationship analyses of substrates for CYP2B6. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 288</i>, 21−29.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1999&pages=29&author=S.+Ekins&author=G.+Bravi&author=B.+J.+Ring&author=T.+A.+Gillespie&author=J.+S.+Gillespie&title=Three-dimensional+quantitative+structure+activity+relationship+analyses+of+substrates+for+CYP2B6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%2520of%2520substrates%2520for%2520CYP2B6%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D288%26spage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm020397cb00012_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Wikel, J. H.; Wrighton, S. A. Three-dimensional-quantitative structure activity relationship analysis of cytochrome P-450 3A4 substrates. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 291</i>, 424−433.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=1999&pages=433&author=S.+Ekins&author=G.+Bravi&author=J.+H.+Wikel&author=S.+A+Wrighton&title=Three-dimensional-quantitative+structure+activity+relationship+analysis+of+cytochrome+P-450+3A4+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional-quantitative%2520structure%2520activity%2520relationship%2520analysis%2520of%2520cytochrome%2520P-450%25203A4%2520substrates%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D291%26spage%3D433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm020397cb00013_1"><span><span class="NLM_string-ref">Afzelius, L.; Zamora, I.; Ridderstrom, M.; Andersson, T. B.; Karlen, A.; et al. Competitive CYP2C9 inhibitors:  Enzyme inhibition studies, protein homology modeling, and three-dimensional quantitative structure−activity relationship analysis. <i>  Mol. Pharmacol. </i><b>2001</b><i>, 59</i>, 909−919.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2001&pages=919&author=L.+Afzelius&author=I.+Zamora&author=M.+Ridderstrom&author=T.+B.+Andersson&author=A.+Karlen&author=+Competitive&title=Enzyme+inhibition+studies%2C+protein+homology+modeling%2C+and+three-dimensional+quantitative+structure%E2%88%92activity+relationship+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAfzelius%26aufirst%3DL.%26atitle%3DEnzyme%2520inhibition%2520studies%252C%2520protein%2520homology%2520modeling%252C%2520and%2520three-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520analysis%26jtitle%3DMol.%2520Pharmacol.%26date%3D2001%26volume%3D59%26spage%3D919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm020397cb00014_1"><span><span class="NLM_string-ref">Wang, Q.; Halpert, J. R. Combined three-dimensional quantitative structure−activity relationship analysis of cytochrome P450 2B6 substrates and protein homology modeling. <i>  Drug Metab. Dispos. </i><b>2002</b><i>, 30</i>, 86−95.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=95&author=Q.+Wang&author=J.+R+Halpert&title=Combined+three-dimensional+quantitative+structure%E2%88%92activity+relationship+analysis+of+cytochrome+P450+2B6+substrates+and+protein+homology+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26atitle%3DCombined%2520three-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520analysis%2520of%2520cytochrome%2520P450%25202B6%2520substrates%2520and%2520protein%2520homology%2520modeling%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm020397cb00015_1"><span><span class="NLM_string-ref">Green, M. D.; Tephly, T. R. Glucuronidation of Amines and Hydroxylated Xenobiotics and Endobiotics Catalyzed by Expressed Human Ugt1.4 Protein. <i>  Drug Metab. Dispos. </i><b>1996</b><i>, 24</i>, 356−363.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1996&pages=363&author=M.+D.+Green&author=T.+R+Tephly&title=Glucuronidation+of+Amines+and+Hydroxylated+Xenobiotics+and+Endobiotics+Catalyzed+by+Expressed+Human+Ugt1.4+Protein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BD.%26atitle%3DGlucuronidation%2520of%2520Amines%2520and%2520Hydroxylated%2520Xenobiotics%2520and%2520Endobiotics%2520Catalyzed%2520by%2520Expressed%2520Human%2520Ugt1.4%2520Protein%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1996%26volume%3D24%26spage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm020397cb00016_1"><span><span class="NLM_string-ref">Green, M. D.; Bishop, W. P.; Tephly, T. R. Expressed Human Ugt1.4 Protein Catalyzes the Formation of Quaternary Ammonium-Linked Glucuronides. <i>  Drug Metab. Dispos. </i><b>1995</b><i>, 23</i>, 299−302.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1995&pages=302&author=M.+D.+Green&author=W.+P.+Bishop&author=T.+R+Tephly&title=Expressed+Human+Ugt1.4+Protein+Catalyzes+the+Formation+of+Quaternary+Ammonium-Linked+Glucuronides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BD.%26atitle%3DExpressed%2520Human%2520Ugt1.4%2520Protein%2520Catalyzes%2520the%2520Formation%2520of%2520Quaternary%2520Ammonium-Linked%2520Glucuronides%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1995%26volume%3D23%26spage%3D302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm020397cb00017_1"><span><span class="NLM_string-ref">Sprague, P. W. Automated Chemical Hypothesis Generation and Database Searching with Catalyst [Review]. <i>  Perspect. Drug Discovery Des. </i><b>1995</b><i>, 3</i>, 1−20.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=20&author=P.+W+Sprague&title=Automated+Chemical+Hypothesis+Generation+and+Database+Searching+with+Catalyst+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSprague%26aufirst%3DP.%2BW%26atitle%3DAutomated%2520Chemical%2520Hypothesis%2520Generation%2520and%2520Database%2520Searching%2520with%2520Catalyst%2520%255BReview%255D%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D1995%26volume%3D3%26spage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm020397cb00018_1"><span><span class="NLM_string-ref">Robinson, D. D.; Winn, P. J.; Lyne, P. D.; Richards, W. G. Self-organizing molecular field analysis:  A tool for structure−activity studies. <i>  J. Med. </i><i>Chem. </i><b>1999</b><i>, 42</i>, 573−583.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=583&author=D.+D.+Robinson&author=P.+J.+Winn&author=P.+D.+Lyne&author=W.+G+Richards&title=Self-organizing+molecular+field+analysis%3A%E2%80%89+A+tool+for+structure%E2%88%92activity+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DD.%2BD.%26atitle%3DSelf-organizing%2520molecular%2520field%2520analysis%253A%25E2%2580%2589%2520A%2520tool%2520for%2520structure%25E2%2588%2592activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm020397cb00019_1"><span><span class="NLM_string-ref">Whitley, D. C.; Ford, M. G.; Livingstone, D. J. Unsupervised forward selection:  a method for eliminating redundant variables. <i>  J. Chem. Inf. Comput. Sci. </i><b>2000</b><i>, 40</i>, 1160−1168.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=1168&author=D.+C.+Whitley&author=M.+G.+Ford&author=D.+J+Livingstone&title=Unsupervised+forward+selection%3A%E2%80%89+a+method+for+eliminating+redundant+variables"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DUnsupervised%2520forward%2520selection%253A%25E2%2580%2589%2520a%2520method%2520for%2520eliminating%2520redundant%2520variables%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2000%26volume%3D40%26spage%3D1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00020"><div class="reference"><div class="NLM_citation" id="jm020397cb00020_1"><span><span class="NLM_string-ref">Said, M.; Ziegler, J. C.; Magdalou, J.; Elass, A.; Vergoten, G. Inhibition of Bilirubin Udp-GlucuronosyltransferaseA Comparative Molecular Field Analysis (Comfa). <i>  Quant. Struct.</i><i>−</i><i>Act. Relat. </i><b>1996</b><i>, 15</i>, 382−388.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=388&author=M.+Said&author=J.+C.+Ziegler&author=J.+Magdalou&author=A.+Elass&author=G+Vergoten&title=Inhibition+of+Bilirubin+Udp-GlucuronosyltransferaseA+Comparative+Molecular+Field+Analysis+%28Comfa%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaid%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520Bilirubin%2520Udp-GlucuronosyltransferaseA%2520Comparative%2520Molecular%2520Field%2520Analysis%2520%2528Comfa%2529%26jtitle%3DQuant.%2520Struct.%25E2%2588%2592Act.%2520Relat.%26date%3D1996%26volume%3D15%26spage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm020397cb00021_1"><span><span class="NLM_string-ref">Said, M.; Battaglia, E.; Elass, A.; Cano, V.; Ziegler, J. C.; et al. Mechanism of inhibition of rat liver bilirubin UDP-glucuronosyltransferase by triphenylalkyl derivatives. <i>  J. Biochem. Mol. Toxicol. </i><b>1998</b><i>, 12</i>, 19−27.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=27&author=M.+Said&author=E.+Battaglia&author=A.+Elass&author=V.+Cano&author=J.+C.+Ziegler&title=Mechanism+of+inhibition+of+rat+liver+bilirubin+UDP-glucuronosyltransferase+by+triphenylalkyl+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaid%26aufirst%3DM.%26atitle%3DMechanism%2520of%2520inhibition%2520of%2520rat%2520liver%2520bilirubin%2520UDP-glucuronosyltransferase%2520by%2520triphenylalkyl%2520derivatives%26jtitle%3DJ.%2520Biochem.%2520Mol.%2520Toxicol.%26date%3D1998%26volume%3D12%26spage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm020397cb00022_1"><span><span class="NLM_string-ref">Naydenova, Z. G.; Grancharov, K. C.; Alargov, D. K.; Golovinsky, E. V.; Stanoeva, I. M.; et al. Inhibition of Udp-Glucuronosyltransferase by 5‘-<i>O</i>-Amino Acid and Oligopeptide Derivatives of Uridine-Structure−Activity Relationships. <i>  Z. Naturforsch., C:  J. Biosci. </i><b>1998</b><i>, 53</i>, 173−181.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&author=Z.+G.+Naydenova&author=K.+C.+Grancharov&author=D.+K.+Alargov&author=E.+V.+Golovinsky&author=I.+M.+Stanoeva&title=C%3A%E2%80%89+J.+Biosci."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DNaydenova%26aufirst%3DZ.%2BG.%26atitle%3DInhibition%2520of%2520Udp-Glucuronosyltransferase%2520by%25205%25E2%2580%2598-O-Amino%2520Acid%2520and%2520Oligopeptide%2520Derivatives%2520of%2520Uridine-Structure%25E2%2588%2592Activity%2520Relationships.%2520Z.%2520Naturforsch%26btitle%3DC%253A%25E2%2580%2589%2520J.%2520Biosci.%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm020397cb00023_1"><span><span class="NLM_string-ref">Cupid, B. C.; Holmes, E.; Wilson, I. D.; Lindon, J. C.; Nicholson, J. K. Quantitative structure−metabolism relationships (QSMR) using computational chemistry:  pattern recognition analysis and statistical prediction of phase II conjugation reactions of substituted benzoic acids in the rat. <i>  Xenobiotica </i><b>1999</b><i>, 29</i>, 27−42.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1999&pages=42&author=B.+C.+Cupid&author=E.+Holmes&author=I.+D.+Wilson&author=J.+C.+Lindon&author=J.+K+Nicholson&title=Quantitative+structure%E2%88%92metabolism+relationships+%28QSMR%29+using+computational+chemistry%3A%E2%80%89+pattern+recognition+analysis+and+statistical+prediction+of+phase+II+conjugation+reactions+of+substituted+benzoic+acids+in+the+rat"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCupid%26aufirst%3DB.%2BC.%26atitle%3DQuantitative%2520structure%25E2%2588%2592metabolism%2520relationships%2520%2528QSMR%2529%2520using%2520computational%2520chemistry%253A%25E2%2580%2589%2520pattern%2520recognition%2520analysis%2520and%2520statistical%2520prediction%2520of%2520phase%2520II%2520conjugation%2520reactions%2520of%2520substituted%2520benzoic%2520acids%2520in%2520the%2520rat%26jtitle%3DXenobiotica%26date%3D1999%26volume%3D29%26spage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm020397cb00024_1"><span><span class="NLM_string-ref">Ghauri, F. Y.; Blackledge, C. A.; Glen, R. C.; Sweatman, B. C.; Lindon, J. C.; et al. Quantitative structure−metabolism relationships for substituted benzoic acids in the rat. Computational chemistry, NMR spectroscopy and pattern recognition studies. <i>  Biochem. Pharmacol. </i><b>1992</b><i>, 44</i>, 1935−1946.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1992&pages=1946&author=F.+Y.+Ghauri&author=C.+A.+Blackledge&author=R.+C.+Glen&author=B.+C.+Sweatman&author=J.+C.+Lindon&title=Quantitative+structure%E2%88%92metabolism+relationships+for+substituted+benzoic+acids+in+the+rat.+Computational+chemistry%2C+NMR+spectroscopy+and+pattern+recognition+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhauri%26aufirst%3DF.%2BY.%26atitle%3DQuantitative%2520structure%25E2%2588%2592metabolism%2520relationships%2520for%2520substituted%2520benzoic%2520acids%2520in%2520the%2520rat.%2520Computational%2520chemistry%252C%2520NMR%2520spectroscopy%2520and%2520pattern%2520recognition%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1992%26volume%3D44%26spage%3D1946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm020397cb00025_1"><span><span class="NLM_string-ref">Kim, K. H. Quantitative structure−activity relationships of the metabolism of drugs by uridine diphosphate glucuronosyltransferase. <i>  J. Pharm. Sci. </i><b>1991</b><i>, </i><i>80</i>, 966−970.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1991&pages=970&author=K.+H+Kim&title=Quantitative+structure%E2%88%92activity+relationships+of+the+metabolism+of+drugs+by+uridine+diphosphate+glucuronosyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH%26atitle%3DQuantitative%2520structure%25E2%2588%2592activity%2520relationships%2520of%2520the%2520metabolism%2520of%2520drugs%2520by%2520uridine%2520diphosphate%2520glucuronosyltransferase%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1991%26volume%3D80%26spage%3D970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm020397cb00026_1"><span><span class="NLM_string-ref">Resetar, A.; Minick, D.; Spector, T. Glucuronidation of 3‘-azido-3‘-deoxythymidine catalyzed by human liver UDP-glucuronosyltransferase. Significance of nucleoside hydrophobicity and inhibition by xenobiotics. <i>  Biochem. Pharmacol. </i><b>1991</b><i>, 42</i>, 559−568.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1991&pages=568&author=A.+Resetar&author=D.+Minick&author=T+Spector&title=Glucuronidation+of+3%E2%80%98-azido-3%E2%80%98-deoxythymidine+catalyzed+by+human+liver+UDP-glucuronosyltransferase.+Significance+of+nucleoside+hydrophobicity+and+inhibition+by+xenobiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DResetar%26aufirst%3DA.%26atitle%3DGlucuronidation%2520of%25203%25E2%2580%2598-azido-3%25E2%2580%2598-deoxythymidine%2520catalyzed%2520by%2520human%2520liver%2520UDP-glucuronosyltransferase.%2520Significance%2520of%2520nucleoside%2520hydrophobicity%2520and%2520inhibition%2520by%2520xenobiotics%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1991%26volume%3D42%26spage%3D568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm020397cb00027_1"><span><span class="NLM_string-ref">Yin, H. Q.; Bennett, G.; Jones, J. P. Mechanistic Studies of Uridine Diphosphate Glucuronosyltransferase. <i>  Chem.-Biol. Interact. </i><b>1994</b><i>, 90</i>, 47−58.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1994&pages=58&author=H.+Q.+Yin&author=G.+Bennett&author=J.+P+Jones&title=Mechanistic+Studies+of+Uridine+Diphosphate+Glucuronosyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DH.%2BQ.%26atitle%3DMechanistic%2520Studies%2520of%2520Uridine%2520Diphosphate%2520Glucuronosyltransferase%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D1994%26volume%3D90%26spage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm020397cb00028_1"><span><span class="NLM_string-ref">Antonio, L.; Grillasca, J.; Taskinen, J.; Elovaara, E.; Burchell, B.; et al. Characterization of catechol glucuronidation in rat liver. <i>  Drug Metab. </i><i>Dispos. </i><b>2002</b><i>, 30</i>, 199−207.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=207&author=L.+Antonio&author=J.+Grillasca&author=J.+Taskinen&author=E.+Elovaara&author=B.+Burchell&title=Characterization+of+catechol+glucuronidation+in+rat+liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntonio%26aufirst%3DL.%26atitle%3DCharacterization%2520of%2520catechol%2520glucuronidation%2520in%2520rat%2520liver%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm020397cb00029_1"><span><span class="NLM_string-ref">de Groot, M. J.; Ackland, M. J.; Horne, V. A.; Alex, A. A.; Jones, B. C. A novel approach to predicting P450 mediated drug metabolism. CYP2D6 catalyzed N-dealkylation reactions and qualitative metabolite predictions using a combined protein and pharmacophore model for CYP2D6. <i>  J. Med. </i><i>Chem. </i><b>1999</b><i>, 42</i>, 4062−4070.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4070&author=M.+J.+de+Groot&author=M.+J.+Ackland&author=V.+A.+Horne&author=A.+A.+Alex&author=B.+C+Jones&title=A+novel+approach+to+predicting+P450+mediated+drug+metabolism.+CYP2D6+catalyzed+N-dealkylation+reactions+and+qualitative+metabolite+predictions+using+a+combined+protein+and+pharmacophore+model+for+CYP2D6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DM.%2BJ.%26atitle%3DA%2520novel%2520approach%2520to%2520predicting%2520P450%2520mediated%2520drug%2520metabolism.%2520CYP2D6%2520catalyzed%2520N-dealkylation%2520reactions%2520and%2520qualitative%2520metabolite%2520predictions%2520using%2520a%2520combined%2520protein%2520and%2520pharmacophore%2520model%2520for%2520CYP2D6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00030"><div class="reference"><div class="NLM_citation" id="jm020397cb00030_1"><span><span class="NLM_string-ref">de Groot, M. J.; Ackland, M. J.; Horne, V. A.; Alex, A. A.; Jones, B. C. Novel approach to predicting P450-mediated drug metabolism:  Development of a combined protein and pharmacophore model for CYP2D6. <i>  J. Med. Chem. </i><b>1999</b><i>, 42</i>, 1515−1524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1524&author=M.+J.+de+Groot&author=M.+J.+Ackland&author=V.+A.+Horne&author=A.+A.+Alex&author=B.+C+Jones&title=Novel+approach+to+predicting+P450-mediated+drug+metabolism%3A%E2%80%89+Development+of+a+combined+protein+and+pharmacophore+model+for+CYP2D6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DM.%2BJ.%26atitle%3DNovel%2520approach%2520to%2520predicting%2520P450-mediated%2520drug%2520metabolism%253A%25E2%2580%2589%2520Development%2520of%2520a%2520combined%2520protein%2520and%2520pharmacophore%2520model%2520for%2520CYP2D6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm020397cb00031_1"><span><span class="NLM_string-ref">Radominska-Pandya, A.; Czernik, P. J.; Little, J. M.; Battaglia, E.; Mackenzie, P. I. Structural and functional studies of UDP-glucuronosyltransferases [Review]. <i>  Drug Metab. Rev. </i><b>1999</b><i>, 31</i>, 817−899.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1999&pages=899&author=A.+Radominska-Pandya&author=P.+J.+Czernik&author=J.+M.+Little&author=E.+Battaglia&author=P.+I+Mackenzie&title=Structural+and+functional+studies+of+UDP-glucuronosyltransferases+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRadominska-Pandya%26aufirst%3DA.%26atitle%3DStructural%2520and%2520functional%2520studies%2520of%2520UDP-glucuronosyltransferases%2520%255BReview%255D%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D1999%26volume%3D31%26spage%3D899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm020397cb00032_1"><span><span class="NLM_string-ref">Jackson, M. R.; Fournel-Gigleux, S.; Harding, D.; Burchell, B. Examination of the substrate specificity of cloned rat kidney phenol UDP-glucuronyltransferase expressed in COS-7 cells. <i>  Mol. Pharmacol. </i><b>1988</b><i>, 34</i>, 638−642.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1988&pages=642&author=M.+R.+Jackson&author=S.+Fournel-Gigleux&author=D.+Harding&author=B+Burchell&title=Examination+of+the+substrate+specificity+of+cloned+rat+kidney+phenol+UDP-glucuronyltransferase+expressed+in+COS-7+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DM.%2BR.%26atitle%3DExamination%2520of%2520the%2520substrate%2520specificity%2520of%2520cloned%2520rat%2520kidney%2520phenol%2520UDP-glucuronyltransferase%2520expressed%2520in%2520COS-7%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1988%26volume%3D34%26spage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm020397cb00033_1"><span><span class="NLM_string-ref">Sorich, M. J.; Smith, P. A.; McKinnon, R. A.; Miners, J. O. Pharmacophore and Quantitative Structure Activity Relationship Modeling of UDP-Glucuronosyltransferase 1A1 (UGT1A1) Substrates. <i>  Pharmacogenetics </i><b>2002</b><i>, 12</i> (8), 635−645.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=645&issue=8&author=M.+J.+Sorich&author=P.+A.+Smith&author=R.+A.+McKinnon&author=J.+O+Miners&title=Pharmacophore+and+Quantitative+Structure+Activity+Relationship+Modeling+of+UDP-Glucuronosyltransferase+1A1+%28UGT1A1%29+Substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSorich%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520and%2520Quantitative%2520Structure%2520Activity%2520Relationship%2520Modeling%2520of%2520UDP-Glucuronosyltransferase%25201A1%2520%2528UGT1A1%2529%2520Substrates%26jtitle%3DPharmacogenetics%26date%3D2002%26volume%3D12%26issue%3D8%26spage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm020397cb00034_1"><span><span class="NLM_string-ref">Todeschini, R., Consonni, V., Eds. <i>  Handbook of Molecular Descriptors</i>; Wiley-VCH:  Weinheim, Germany, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm020397cb00034/jm020397cb00034_1&amp;dbid=16&amp;doi=10.1021%2Fjm020397c&amp;key=10.1002%2F9783527613106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=R.+Todeschini&author=V.+Consonni&title=Handbook+of+Molecular+Descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00034_1&amp;dbid=16384&amp;doi=10.1002%2F9783527613106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527613106%26sid%3Dliteratum%253Aachs%26aulast%3DTodeschini%26aufirst%3DR.%26btitle%3DHandbook%2520of%2520Molecular%2520Descriptors%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm020397cb00035_1"><span><span class="NLM_string-ref">Gao, H. Application of BCUT metrics and genetic algorithm in binary QSAR analysis. <i>  J. Chem. Inf. Comput. Sci. </i><b>2001</b><i>, 41</i>, 402−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=407&author=H+Gao&title=Application+of+BCUT+metrics+and+genetic+algorithm+in+binary+QSAR+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH%26atitle%3DApplication%2520of%2520BCUT%2520metrics%2520and%2520genetic%2520algorithm%2520in%2520binary%2520QSAR%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm020397cb00036_1"><span><span class="NLM_string-ref">Vandrie, J. H.; Nugent, R. A. Addressing the Challenges Posed by Combinatorial Chemistry3d Databases, Pharmacophore Recognition and Beyond. <i>  SAR QSAR Environ. Res. </i><b>1998</b><i>, 9</i>, 1 ff.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&author=J.+H.+Vandrie&author=R.+A.+Nugent&title=Addressing+the+Challenges+Posed+by+Combinatorial+Chemistry3d+Databases%2C+Pharmacophore+Recognition+and+Beyond.+SAR+QSAR+Environ.+Res"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVandrie%26aufirst%3DJ.%2BH.%26btitle%3DAddressing%2520the%2520Challenges%2520Posed%2520by%2520Combinatorial%2520Chemistry3d%2520Databases%252C%2520Pharmacophore%2520Recognition%2520and%2520Beyond.%2520SAR%2520QSAR%2520Environ.%2520Res%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm020397cb00037_1"><span><span class="NLM_string-ref">Sprague, P. W.; Hoffmann, R. CATALYST Pharmacophore Models and Their Utility as Queries for Searching 3D Databases. In <i>Computer-Assisted Lead </i><i>Finding and Optimization:  Current Tools for Medicinal Chemistry</i>; Wiley-VCH:  Weinheim, Germany, 1997; pp 223−240.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=240&author=P.+W.+Sprague&author=R.+Hoffmann&title=Computer-Assisted+Lead+Finding+and+Optimization%3A%E2%80%89+Current+Tools+for+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSprague%26aufirst%3DP.%2BW.%26atitle%3DCATALYST%2520Pharmacophore%2520Models%2520and%2520Their%2520Utility%2520as%2520Queries%2520for%2520Searching%25203D%2520Databases%26btitle%3DComputer-Assisted%2520Lead%2520Finding%2520and%2520Optimization%253A%25E2%2580%2589%2520Current%2520Tools%2520for%2520Medicinal%2520Chemistry%26pub%3DWiley-VCH%26date%3D1997%26spage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm020397cb00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm020397cb00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm020397cb00038_1"><span><span class="NLM_string-ref">Cramer, R. D., 3rd; Patterson, D. E.; Bunce, J. D. Comparative Molecular Field Analysis (CoMFA). 1. Effect of Shape on Binding of Steroids to Carrier Proteins. <i>  J. Am. Chem. Soc. </i><b>1988</b><i>, 110</i>, 5959−5967</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=5967&author=R.+D.+Cramer&author=D.+E.+Patterson&author=J.+D+Bunce&title=Comparative+Molecular+Field+Analysis+%28CoMFA%29.+1.+Effect+of+Shape+on+Binding+of+Steroids+to+Carrier+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26atitle%3DComparative%2520Molecular%2520Field%2520Analysis%2520%2528CoMFA%2529.%25201.%2520Effect%2520of%2520Shape%2520on%2520Binding%2520of%2520Steroids%2520to%2520Carrier%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D5967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm020397cb00001':['jm020397cb00001_1'],'jm020397cb00002':['jm020397cb00002_1'],'jm020397cb00003':['jm020397cb00003_1'],'jm020397cb00004':['jm020397cb00004_1'],'jm020397cb00005':['jm020397cb00005_1'],'jm020397cb00006':['jm020397cb00006_1'],'jm020397cb00007':['jm020397cb00007_1'],'jm020397cb00008':['jm020397cb00008_1'],'jm020397cb00009':['jm020397cb00009_1'],'jm020397cb00010':['jm020397cb00010_1'],'jm020397cb00011':['jm020397cb00011_1'],'jm020397cb00012':['jm020397cb00012_1'],'jm020397cb00013':['jm020397cb00013_1'],'jm020397cb00014':['jm020397cb00014_1'],'jm020397cb00015':['jm020397cb00015_1'],'jm020397cb00016':['jm020397cb00016_1'],'jm020397cb00017':['jm020397cb00017_1'],'jm020397cb00018':['jm020397cb00018_1'],'jm020397cb00019':['jm020397cb00019_1'],'jm020397cb00020':['jm020397cb00020_1'],'jm020397cb00021':['jm020397cb00021_1'],'jm020397cb00022':['jm020397cb00022_1'],'jm020397cb00023':['jm020397cb00023_1'],'jm020397cb00024':['jm020397cb00024_1'],'jm020397cb00025':['jm020397cb00025_1'],'jm020397cb00026':['jm020397cb00026_1'],'jm020397cb00027':['jm020397cb00027_1'],'jm020397cb00028':['jm020397cb00028_1'],'jm020397cb00029':['jm020397cb00029_1'],'jm020397cb00030':['jm020397cb00030_1'],'jm020397cb00031':['jm020397cb00031_1'],'jm020397cb00032':['jm020397cb00032_1'],'jm020397cb00033':['jm020397cb00033_1'],'jm020397cb00034':['jm020397cb00034_1'],'jm020397cb00035':['jm020397cb00035_1'],'jm020397cb00036':['jm020397cb00036_1'],'jm020397cb00037':['jm020397cb00037_1'],'jm020397cb00038':['jm020397cb00038_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 50 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rashim  Singh</span>, <span class="hlFld-ContribAuthor ">Baojian  Wu</span>, <span class="hlFld-ContribAuthor ">Lan  Tang</span>, <span class="hlFld-ContribAuthor ">Zhongqiu  Liu</span> and <span class="hlFld-ContribAuthor ">Ming  Hu</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of the Position of Mono-O-glucuronide of Flavones and Flavonols by Analyzing Shift in Online UV Spectrum (λmax) Generated from an Online Diode Array Detector. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2010,</strong> <em>58 </em>
                                    (17)
                                     , 9384-9395. <a href="https://doi.org/10.1021/jf904561e" title="DOI URL">https://doi.org/10.1021/jf904561e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jf904561e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjf904561e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DIdentification%252Bof%252Bthe%252BPosition%252Bof%252BMono-O-glucuronide%252Bof%252BFlavones%252Band%252BFlavonols%252Bby%252BAnalyzing%252BShift%252Bin%252BOnline%252BUV%252BSpectrum%252B%252528%2525CE%2525BBmax%252529%252BGenerated%252Bfrom%252Ban%252BOnline%252BDiode%252BArray%252BDetector%26aulast%3DSingh%26aufirst%3DRashim%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D2010%26date%3D29122009%26date%3D24052010%26date%3D12072010%26date%3D05082010%26date%3D08092010%26volume%3D58%26issue%3D17%26spage%3D9384%26epage%3D9395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nikolay P.  Todorov,,   Ian L.  Alberts,,   Iwan J. P.  de Esch, and,   <span class="NLM_string-name hlFld-ContribAuthor">Philip M.  Dean</span>. </span><span class="cited-content_cbyCitation_article-title">QUASI:  A Novel Method for Simultaneous Superposition of Multiple Flexible Ligands and Virtual Screening Using Partial Similarity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (3)
                                     , 1007-1020. <a href="https://doi.org/10.1021/ci6003338" title="DOI URL">https://doi.org/10.1021/ci6003338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci6003338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci6003338%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DQUASI%25253A%2525E2%252580%252589%252BA%252BNovel%252BMethod%252Bfor%252BSimultaneous%252BSuperposition%252Bof%252BMultiple%252BFlexible%252BLigands%252Band%252BVirtual%252BScreening%252BUsing%252BPartial%252BSimilarity%26aulast%3DTodorov%26aufirst%3DNikolay%2BP.%26date%3D2007%26date%3D2007%26date%3D2006%26date%3D01%252008%25202006%2520Received%2520August%25201%252C%25202006%26volume%3D47%26issue%3D3%26spage%3D1007%26epage%3D1020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael J.  Sorich,,   Ross A.  McKinnon,,   John O.  Miners, and,   <span class="NLM_string-name hlFld-ContribAuthor">Paul A.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">The Importance of Local Chemical Structure for Chemical Metabolism by Human Uridine 5‘-Diphosphate−Glucuronosyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2006,</strong> <em>46 </em>
                                    (6)
                                     , 2692-2697. <a href="https://doi.org/10.1021/ci600248e" title="DOI URL">https://doi.org/10.1021/ci600248e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci600248e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci600248e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BImportance%252Bof%252BLocal%252BChemical%252BStructure%252Bfor%252BChemical%252BMetabolism%252Bby%252BHuman%252BUridine%252B5%2525E2%252580%252598-Diphosphate%2525E2%252588%252592Glucuronosyltransferase%26aulast%3DSorich%26aufirst%3DMichael%2BJ.%26date%3D2006%26date%3D2006%26date%3D2006%26date%3D20%252006%25202006%2520Received%2520June%252020%252C%25202006%26volume%3D46%26issue%3D6%26spage%3D2692%26epage%3D2697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael J.  Sorich,,   John O.  Miners,,   Ross A.  McKinnon,,   David A.  Winkler,,   Frank R.  Burden, and,   <span class="NLM_string-name hlFld-ContribAuthor">Paul A.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of Linear and Nonlinear Classification Algorithms for the Prediction of Drug and Chemical Metabolism by Human UDP-Glucuronosyltransferase Isoforms. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Computer Sciences</span><span> <strong>2003,</strong> <em>43 </em>
                                    (6)
                                     , 2019-2024. <a href="https://doi.org/10.1021/ci034108k" title="DOI URL">https://doi.org/10.1021/ci034108k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci034108k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci034108k%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Computer%2520Sciences%26atitle%3DComparison%252Bof%252BLinear%252Band%252BNonlinear%252BClassification%252BAlgorithms%252Bfor%252Bthe%252BPrediction%252Bof%252BDrug%252Band%252BChemical%252BMetabolism%252Bby%252BHuman%252BUDP-Glucuronosyltransferase%252BIsoforms%26aulast%3DSorich%26aufirst%3DMichael%2BJ.%26date%3D2003%26date%3D2003%26date%3D2003%26date%3D29%252005%25202003%2520Received%2520May%252029%252C%25202003%26volume%3D43%26issue%3D6%26spage%3D2019%26epage%3D2024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John O.  Miners</span>, <span class="hlFld-ContribAuthor ">Andrew  Rowland</span>, <span class="hlFld-ContribAuthor ">Jonathan J.  Novak</span>, <span class="hlFld-ContribAuthor ">Kimberly  Lapham</span>, <span class="hlFld-ContribAuthor ">Theunis C.  Goosen</span>. </span><span class="cited-content_cbyCitation_article-title">Evidence-based strategies for the characterisation of human drug and chemical glucuronidation in vitro and UDP-glucuronosyltransferase reaction phenotyping. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2021,</strong> <em>218 </em>, 107689. <a href="https://doi.org/10.1016/j.pharmthera.2020.107689" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2020.107689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2020.107689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2020.107689%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEvidence-based%252Bstrategies%252Bfor%252Bthe%252Bcharacterisation%252Bof%252Bhuman%252Bdrug%252Band%252Bchemical%252Bglucuronidation%252Bin%252Bvitro%252Band%252BUDP-glucuronosyltransferase%252Breaction%252Bphenotyping%26aulast%3DMiners%26aufirst%3DJohn%2BO.%26date%3D2021%26volume%3D218%26spage%3D107689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sungbo  Hwang</span>, <span class="hlFld-ContribAuthor ">Hyun Kil  Shin</span>, <span class="hlFld-ContribAuthor ">Seong Eun  Shin</span>, <span class="hlFld-ContribAuthor ">Myungwon  Seo</span>, <span class="hlFld-ContribAuthor ">Hyeon-Nae  Jeon</span>, <span class="hlFld-ContribAuthor ">Da-Eun  Yim</span>, <span class="hlFld-ContribAuthor ">Dong-Hyun  Kim</span>, <span class="hlFld-ContribAuthor ">Kyoung Tai  No</span>. </span><span class="cited-content_cbyCitation_article-title">PreMetabo: An in silico phase I and II drug metabolism prediction platform. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Pharmacokinetics</span><span> <strong>2020,</strong> <em>35 </em>
                                    (4)
                                     , 361-367. <a href="https://doi.org/10.1016/j.dmpk.2020.05.007" title="DOI URL">https://doi.org/10.1016/j.dmpk.2020.05.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dmpk.2020.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dmpk.2020.05.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Pharmacokinetics%26atitle%3DPreMetabo%25253A%252BAn%252Bin%252Bsilico%252Bphase%252BI%252Band%252BII%252Bdrug%252Bmetabolism%252Bprediction%252Bplatform%26aulast%3DHwang%26aufirst%3DSungbo%26date%3D2020%26volume%3D35%26issue%3D4%26spage%3D361%26epage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayse Gunes  Bayir</span>, <span class="hlFld-ContribAuthor ">Huriye Senay  Kiziltan</span>, <span class="hlFld-ContribAuthor ">Abdurrahim  Kocyigit</span>. </span><span class="cited-content_cbyCitation_article-title">Plant Family, Carvacrol, and Putative Protection in Gastric Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 3-18. <a href="https://doi.org/10.1016/B978-0-12-814468-8.00001-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-814468-8.00001-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-814468-8.00001-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-814468-8.00001-6%26sid%3Dliteratum%253Aachs%26atitle%3DPlant%252BFamily%25252C%252BCarvacrol%25252C%252Band%252BPutative%252BProtection%252Bin%252BGastric%252BCancer%26aulast%3DBayir%26aufirst%3DAyse%2BGunes%26date%3D2019%26spage%3D3%26epage%3D18%26pub%3DElsevier%26atitle%3DDietary%252BInterventions%252Bin%252BGastrointestinal%252BDiseases%26date%3D2019%26volume%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. Lakshmana  Prabu</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 1-46. <a href="https://doi.org/10.4018/978-1-5225-7326-5.ch001" title="DOI URL">https://doi.org/10.4018/978-1-5225-7326-5.ch001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4018/978-1-5225-7326-5.ch001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4018%2F978-1-5225-7326-5.ch001%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDiscovery%26aulast%3DPrabu%26aufirst%3DS.%2BLakshmana%26date%3D2019%26spage%3D1%26epage%3D46%26pub%3DIGI%2520Global%26atitle%3DComputer%252BApplications%252Bin%252BDrug%252BDiscovery%252Band%252BDevelopment%26aulast%3DPuratchikody%26aufirst%3DA.%26date%3D2019%26volume%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mehdi  Sharifi-Rad</span>, <span class="hlFld-ContribAuthor ">Elena Maria  Varoni</span>, <span class="hlFld-ContribAuthor ">Marcello  Iriti</span>, <span class="hlFld-ContribAuthor ">Miquel  Martorell</span>, <span class="hlFld-ContribAuthor ">William N.  Setzer</span>, <span class="hlFld-ContribAuthor ">María  del Mar Contreras</span>, <span class="hlFld-ContribAuthor ">Bahare  Salehi</span>, <span class="hlFld-ContribAuthor ">Azam  Soltani-Nejad</span>, <span class="hlFld-ContribAuthor ">Sadegh  Rajabi</span>, <span class="hlFld-ContribAuthor ">Mercedeh  Tajbakhsh</span>, <span class="hlFld-ContribAuthor ">Javad  Sharifi-Rad</span>. </span><span class="cited-content_cbyCitation_article-title">Carvacrol and human health: A comprehensive review. </span><span class="cited-content_cbyCitation_journal-name">Phytotherapy Research</span><span> <strong>2018,</strong> <em>32 </em>
                                    (9)
                                     , 1675-1687. <a href="https://doi.org/10.1002/ptr.6103" title="DOI URL">https://doi.org/10.1002/ptr.6103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ptr.6103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fptr.6103%26sid%3Dliteratum%253Aachs%26jtitle%3DPhytotherapy%2520Research%26atitle%3DCarvacrol%252Band%252Bhuman%252Bhealth%25253A%252BA%252Bcomprehensive%252Breview%26aulast%3DSharifi-Rad%26aufirst%3DMehdi%26date%3D2018%26date%3D2018%26volume%3D32%26issue%3D9%26spage%3D1675%26epage%3D1687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaokang  Wang</span>, <span class="hlFld-ContribAuthor ">Caiyun  Jiang</span>, <span class="hlFld-ContribAuthor ">Xiaosong  Wu</span>, <span class="hlFld-ContribAuthor ">Peng  Zou</span>, <span class="hlFld-ContribAuthor ">Zhufeng  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Substrate Selectivity for UDP-glucuronosyltransferase1A8 using the Pharmacophore Approach. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Pharmacology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (3)
                                     , 320-328. <a href="https://doi.org/10.3923/ijp.2018.320.328" title="DOI URL">https://doi.org/10.3923/ijp.2018.320.328</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3923/ijp.2018.320.328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3923%2Fijp.2018.320.328%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Pharmacology%26atitle%3DSubstrate%252BSelectivity%252Bfor%252BUDP-glucuronosyltransferase1A8%252Busing%252Bthe%252BPharmacophore%252BApproach%26aulast%3DWang%26aufirst%3DXiaokang%26date%3D2018%26volume%3D14%26issue%3D3%26spage%3D320%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryoichi  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Tsuyoshi  Yokoi</span>, <span class="hlFld-ContribAuthor ">Miki  Nakajima</span>. </span><span class="cited-content_cbyCitation_article-title">Structure and Protein–Protein Interactions of Human UDP-Glucuronosyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2016,</strong> <em>7 </em><a href="https://doi.org/10.3389/fphar.2016.00388" title="DOI URL">https://doi.org/10.3389/fphar.2016.00388</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2016.00388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2016.00388%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DStructure%252Band%252BProtein%2525E2%252580%252593Protein%252BInteractions%252Bof%252BHuman%252BUDP-Glucuronosyltransferases%26aulast%3DFujiwara%26aufirst%3DRyoichi%26date%3D2016%26date%3D2016%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul  Gangwal</span>, <span class="hlFld-ContribAuthor ">Mangesh  Damre</span>, <span class="hlFld-ContribAuthor ">Abhay  Sangamwar</span>. </span><span class="cited-content_cbyCitation_article-title">Overview and Recent Advances in QSAR Studies. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 1-32. <a href="https://doi.org/10.1201/b19853-2" title="DOI URL">https://doi.org/10.1201/b19853-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1201/b19853-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1201%2Fb19853-2%26sid%3Dliteratum%253Aachs%26atitle%3DOverview%252Band%252BRecent%252BAdvances%252Bin%252BQSAR%252BStudies%26aulast%3DGangwal%26aufirst%3DRahul%26date%3D2016%26date%3D2016%26spage%3D1%26epage%3D32%26pub%3DApple%2520Academic%2520Press%26atitle%3DChemometrics%252BApplications%252Band%252BResearch%26aulast%3DMercader%26aufirst%3DAndrew%26date%3D2016%26date%3D2016%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sivaranjani  Arivalagan</span>, <span class="hlFld-ContribAuthor ">Nisha Susan  Thomas</span>, <span class="hlFld-ContribAuthor ">Balaji  Chandrasekaran</span>, <span class="hlFld-ContribAuthor ">Vijay  Mani</span>, <span class="hlFld-ContribAuthor ">Aktarul Islam  Siddique</span>, <span class="hlFld-ContribAuthor ">Thayalan  Kuppsamy</span>, <span class="hlFld-ContribAuthor ">Nalini  Namasivayam</span>. </span><span class="cited-content_cbyCitation_article-title">Combined therapeutic efficacy of carvacrol and X-radiation against 1,2-dimethyl hydrazine-induced experimental rat colon carcinogenesis. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Biochemistry</span><span> <strong>2015,</strong> <em>410 </em>
                                    (1-2)
                                     , 37-54. <a href="https://doi.org/10.1007/s11010-015-2536-6" title="DOI URL">https://doi.org/10.1007/s11010-015-2536-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11010-015-2536-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11010-015-2536-6%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Biochemistry%26atitle%3DCombined%252Btherapeutic%252Befficacy%252Bof%252Bcarvacrol%252Band%252BX-radiation%252Bagainst%252B1%25252C2-dimethyl%252Bhydrazine-induced%252Bexperimental%252Brat%252Bcolon%252Bcarcinogenesis%26aulast%3DArivalagan%26aufirst%3DSivaranjani%26date%3D2015%26date%3D2015%26volume%3D410%26issue%3D1-2%26spage%3D37%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhufeng  Wu</span>, <span class="hlFld-ContribAuthor ">Xingwang  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhiguo  Ma</span>, <span class="hlFld-ContribAuthor ">Baojian  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Establishment of pharmacophore and VolSurf models to predict the substrates of UDP-glucuronosyltransferase1A3. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2015,</strong> <em>45 </em>
                                    (8)
                                     , 653-662. <a href="https://doi.org/10.3109/00498254.2015.1016136" title="DOI URL">https://doi.org/10.3109/00498254.2015.1016136</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/00498254.2015.1016136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F00498254.2015.1016136%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DEstablishment%252Bof%252Bpharmacophore%252Band%252BVolSurf%252Bmodels%252Bto%252Bpredict%252Bthe%252Bsubstrates%252Bof%252BUDP-glucuronosyltransferase1A3%26aulast%3DWu%26aufirst%3DZhufeng%26date%3D2015%26date%3D2015%26volume%3D45%26issue%3D8%26spage%3D653%26epage%3D662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zacharias E.  Suntres</span>, <span class="hlFld-ContribAuthor ">John  Coccimiglio</span>, <span class="hlFld-ContribAuthor ">Misagh  Alipour</span>. </span><span class="cited-content_cbyCitation_article-title">The Bioactivity and Toxicological Actions of Carvacrol. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Food Science and Nutrition</span><span> <strong>2015,</strong> <em>55 </em>
                                    (3)
                                     , 304-318. <a href="https://doi.org/10.1080/10408398.2011.653458" title="DOI URL">https://doi.org/10.1080/10408398.2011.653458</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10408398.2011.653458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10408398.2011.653458%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Food%2520Science%2520and%2520Nutrition%26atitle%3DThe%252BBioactivity%252Band%252BToxicological%252BActions%252Bof%252BCarvacrol%26aulast%3DSuntres%26aufirst%3DZacharias%2BE.%26date%3D2015%26date%3D2014%26volume%3D55%26issue%3D3%26spage%3D304%26epage%3D318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoyuki  Takaku</span>, <span class="hlFld-ContribAuthor ">Hirohisa  Nagahori</span>, <span class="hlFld-ContribAuthor ">Yoshihisa  Sogame</span>, <span class="hlFld-ContribAuthor ">Tatsuya  Takagi</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Structure–Activity Relationship Model for the Fetal–Maternal Blood Concentration Ratio of Chemicals in Humans. </span><span class="cited-content_cbyCitation_journal-name">Biological & Pharmaceutical Bulletin</span><span> <strong>2015,</strong> <em>38 </em>
                                    (6)
                                     , 930-934. <a href="https://doi.org/10.1248/bpb.b14-00883" title="DOI URL">https://doi.org/10.1248/bpb.b14-00883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/bpb.b14-00883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fbpb.b14-00883%26sid%3Dliteratum%253Aachs%26jtitle%3DBiological%2520%2526%2520Pharmaceutical%2520Bulletin%26atitle%3DQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BModel%252Bfor%252Bthe%252BFetal%2525E2%252580%252593Maternal%252BBlood%252BConcentration%252BRatio%252Bof%252BChemicals%252Bin%252BHumans%26aulast%3DTakaku%26aufirst%3DTomoyuki%26date%3D2015%26volume%3D38%26issue%3D6%26spage%3D930%26epage%3D934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oraphan  Phuangsawai</span>, <span class="hlFld-ContribAuthor ">Supa  Hannongbua</span>, <span class="hlFld-ContribAuthor ">Mathew Paul  Gleeson</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Structure–Activity Relationship (QSAR) Methods for the Prediction of Substrates, Inhibitors, and Inducers of Metabolic Enzymes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 319-350. <a href="https://doi.org/10.1002/9783527673261.ch13" title="DOI URL">https://doi.org/10.1002/9783527673261.ch13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527673261.ch13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527673261.ch13%26sid%3Dliteratum%253Aachs%26atitle%3DQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252B%252528QSAR%252529%252BMethods%252Bfor%252Bthe%252BPrediction%252Bof%252BSubstrates%25252C%252BInhibitors%25252C%252Band%252BInducers%252Bof%252BMetabolic%252BEnzymes%26aulast%3DPhuangsawai%26aufirst%3DOraphan%26date%3D2014%26date%3D2014%26spage%3D319%26epage%3D350%26pub%3DWiley%26atitle%3DDrug%252BMetabolism%252BPrediction%26aulast%3DKirchmair%26aufirst%3DJohannes%26date%3D2014%26date%3D2014%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Teresa  Kaserer</span>, <span class="hlFld-ContribAuthor ">Veronika  Temml</span>, <span class="hlFld-ContribAuthor ">Daniela  Schuster</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore‐Based Methods for Predicting the Inhibition and Induction of Metabolic Enzymes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 351-372. <a href="https://doi.org/10.1002/9783527673261.ch14" title="DOI URL">https://doi.org/10.1002/9783527673261.ch14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527673261.ch14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527673261.ch14%26sid%3Dliteratum%253Aachs%26atitle%3DPharmacophore%2525E2%252580%252590Based%252BMethods%252Bfor%252BPredicting%252Bthe%252BInhibition%252Band%252BInduction%252Bof%252BMetabolic%252BEnzymes%26aulast%3DKaserer%26aufirst%3DTeresa%26date%3D2014%26date%3D2014%26spage%3D351%26epage%3D372%26pub%3DWiley%26atitle%3DDrug%252BMetabolism%252BPrediction%26aulast%3DKirchmair%26aufirst%3DJohannes%26date%3D2014%26date%3D2014%26volume%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianlong  Peng</span>, <span class="hlFld-ContribAuthor ">Jing  Lu</span>, <span class="hlFld-ContribAuthor ">Qiancheng  Shen</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Weiliang  Zhu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">In silico site of metabolism prediction for human UGT-catalyzed reactions. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2014,</strong> <em>30 </em>
                                    (3)
                                     , 398-405. <a href="https://doi.org/10.1093/bioinformatics/btt681" title="DOI URL">https://doi.org/10.1093/bioinformatics/btt681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btt681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtt681%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DIn%252Bsilico%252Bsite%252Bof%252Bmetabolism%252Bprediction%252Bfor%252Bhuman%252BUGT-catalyzed%252Breactions%26aulast%3DPeng%26aufirst%3DJianlong%26date%3D2014%26date%3D2013%26volume%3D30%26issue%3D3%26spage%3D398%26epage%3D405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rui-Hua  Dong</span>, <span class="hlFld-ContribAuthor ">Zhong-Ze  Fang</span>, <span class="hlFld-ContribAuthor ">Liang-Liang  Zhu</span>, <span class="hlFld-ContribAuthor ">Guang-Bo  Ge</span>, <span class="hlFld-ContribAuthor ">Ling  Yang</span>, <span class="hlFld-ContribAuthor ">Ze-Yuan  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of UDP-glucuronosyltransferase isoforms involved in hepatic and intestinal glucuronidation of phytochemical carvacrol. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2012,</strong> <em>42 </em>
                                    (10)
                                     , 1009-1016. <a href="https://doi.org/10.3109/00498254.2012.682614" title="DOI URL">https://doi.org/10.3109/00498254.2012.682614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/00498254.2012.682614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F00498254.2012.682614%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DIdentification%252Bof%252BUDP-glucuronosyltransferase%252Bisoforms%252Binvolved%252Bin%252Bhepatic%252Band%252Bintestinal%252Bglucuronidation%252Bof%252Bphytochemical%252Bcarvacrol%26aulast%3DDong%26aufirst%3DRui-Hua%26date%3D2012%26date%3D2012%26volume%3D42%26issue%3D10%26spage%3D1009%26epage%3D1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong  Dong</span>, <span class="hlFld-ContribAuthor ">Baojian  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of UDP-Glucuronosyltransferase 1A10 Substrates Using the Volsurf Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical Sciences</span><span> <strong>2012,</strong> <em>101 </em>
                                    (9)
                                     , 3531-3539. <a href="https://doi.org/10.1002/jps.23100" title="DOI URL">https://doi.org/10.1002/jps.23100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jps.23100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjps.23100%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BUDP-Glucuronosyltransferase%252B1A10%252BSubstrates%252BUsing%252Bthe%252BVolsurf%252BApproach%26aulast%3DDong%26aufirst%3DDong%26date%3D2012%26volume%3D101%26issue%3D9%26spage%3D3531%26epage%3D3539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roland  Ako</span>, <span class="hlFld-ContribAuthor ">Dong  Dong</span>, <span class="hlFld-ContribAuthor ">Baojian  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR studies on UDP-glucuronosyltransferase 2B7 substrates using the pharmacophore and VolSurf approaches. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2012,</strong> <em>42 </em>
                                    (9)
                                     , 891-900. <a href="https://doi.org/10.3109/00498254.2012.675094" title="DOI URL">https://doi.org/10.3109/00498254.2012.675094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/00498254.2012.675094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F00498254.2012.675094%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3D3D-QSAR%252Bstudies%252Bon%252BUDP-glucuronosyltransferase%252B2B7%252Bsubstrates%252Busing%252Bthe%252Bpharmacophore%252Band%252BVolSurf%252Bapproaches%26aulast%3DAko%26aufirst%3DRoland%26date%3D2012%26date%3D2012%26volume%3D42%26issue%3D9%26spage%3D891%26epage%3D900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong  Dong</span>, <span class="hlFld-ContribAuthor ">Roland  Ako</span>, <span class="hlFld-ContribAuthor ">Ming  Hu</span>, <span class="hlFld-ContribAuthor ">Baojian  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding substrate selectivity of human UDP-glucuronosyltransferases through QSAR modeling and analysis of homologous enzymes. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2012,</strong> <em>42 </em>
                                    (8)
                                     , 808-820. <a href="https://doi.org/10.3109/00498254.2012.663515" title="DOI URL">https://doi.org/10.3109/00498254.2012.663515</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/00498254.2012.663515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F00498254.2012.663515%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DUnderstanding%252Bsubstrate%252Bselectivity%252Bof%252Bhuman%252BUDP-glucuronosyltransferases%252Bthrough%252BQSAR%252Bmodeling%252Band%252Banalysis%252Bof%252Bhomologous%252Benzymes%26aulast%3DDong%26aufirst%3DDong%26date%3D2012%26date%3D2012%26volume%3D42%26issue%3D8%26spage%3D808%26epage%3D820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Baojian  Wu</span>, <span class="hlFld-ContribAuthor ">Xiaoqiang  Wang</span>, <span class="hlFld-ContribAuthor ">Shuxing  Zhang</span>, <span class="hlFld-ContribAuthor ">Ming  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Accurate Prediction of Glucuronidation of Structurally Diverse Phenolics by Human UGT1A9 Using Combined Experimental and In Silico Approaches. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Research</span><span> <strong>2012,</strong> <em>29 </em>
                                    (6)
                                     , 1544-1561. <a href="https://doi.org/10.1007/s11095-012-0666-z" title="DOI URL">https://doi.org/10.1007/s11095-012-0666-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11095-012-0666-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11095-012-0666-z%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Research%26atitle%3DAccurate%252BPrediction%252Bof%252BGlucuronidation%252Bof%252BStructurally%252BDiverse%252BPhenolics%252Bby%252BHuman%252BUGT1A9%252BUsing%252BCombined%252BExperimental%252Band%252BIn%252BSilico%252BApproaches%26aulast%3DWu%26aufirst%3DBaojian%26date%3D2012%26date%3D2012%26volume%3D29%26issue%3D6%26spage%3D1544%26epage%3D1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert S.  Foti</span>, <span class="hlFld-ContribAuthor ">Michael B.  Fisher</span>. </span><span class="cited-content_cbyCitation_article-title">UDP-Glucuronosyltransferases: Pharmacogenetics, Functional Characterization, and Clinical Relevance. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,<a href="https://doi.org/10.1002/9780470921920.edm015" title="DOI URL">https://doi.org/10.1002/9780470921920.edm015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9780470921920.edm015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9780470921920.edm015%26sid%3Dliteratum%253Aachs%26atitle%3DUDP-Glucuronosyltransferases%25253A%252BPharmacogenetics%25252C%252BFunctional%252BCharacterization%25252C%252Band%252BClinical%252BRelevance%26aulast%3DFoti%26aufirst%3DRobert%2BS.%26date%3D2012%26date%3D2012%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26atitle%3DEncyclopedia%252Bof%252BDrug%252BMetabolism%252Band%252BInteractions%26aulast%3DLyubimov%26aufirst%3DAlexander%2BV.%26date%3D2011%26date%3D2011%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rui-Hua  Dong</span>, <span class="hlFld-ContribAuthor ">Zhong-Ze  Fang</span>, <span class="hlFld-ContribAuthor ">Liang-Liang  Zhu</span>, <span class="hlFld-ContribAuthor ">Si-Cheng  Liang</span>, <span class="hlFld-ContribAuthor ">Guang-Bo  Ge</span>, <span class="hlFld-ContribAuthor ">Ling  Yang</span>, <span class="hlFld-ContribAuthor ">Ze-Yuan  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of UDP-glucuronosyltransferases (UGTs) Inhibitory Properties of Carvacrol. </span><span class="cited-content_cbyCitation_journal-name">Phytotherapy Research</span><span> <strong>2012,</strong> <em>26 </em>
                                    (1)
                                     , 86-90. <a href="https://doi.org/10.1002/ptr.3525" title="DOI URL">https://doi.org/10.1002/ptr.3525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ptr.3525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fptr.3525%26sid%3Dliteratum%253Aachs%26jtitle%3DPhytotherapy%2520Research%26atitle%3DInvestigation%252Bof%252BUDP-glucuronosyltransferases%252B%252528UGTs%252529%252BInhibitory%252BProperties%252Bof%252BCarvacrol%26aulast%3DDong%26aufirst%3DRui-Hua%26date%3D2012%26date%3D2011%26volume%3D26%26issue%3D1%26spage%3D86%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Limin  Liao</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Li</span>, <span class="hlFld-ContribAuthor ">Bi  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Structure–Retention Relationship Study on Nitrogen-Containing Polycyclic Aromatic Compounds by Using Molecular Electronegativity Interaction Vector. </span><span class="cited-content_cbyCitation_journal-name">Chromatographia</span><span> <strong>2011,</strong> <em>74 </em>
                                    (5-6)
                                     , 451-457. <a href="https://doi.org/10.1007/s10337-011-2078-3" title="DOI URL">https://doi.org/10.1007/s10337-011-2078-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10337-011-2078-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10337-011-2078-3%26sid%3Dliteratum%253Aachs%26jtitle%3DChromatographia%26atitle%3DQuantitative%252BStructure%2525E2%252580%252593Retention%252BRelationship%252BStudy%252Bon%252BNitrogen-Containing%252BPolycyclic%252BAromatic%252BCompounds%252Bby%252BUsing%252BMolecular%252BElectronegativity%252BInteraction%252BVector%26aulast%3DLiao%26aufirst%3DLimin%26date%3D2011%26date%3D2011%26volume%3D74%26issue%3D5-6%26spage%3D451%26epage%3D457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. F.  Nasonov</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods and software in computer-aided combinatorial library design. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of General Chemistry</span><span> <strong>2010,</strong> <em>80 </em>
                                    (12)
                                     , 2541-2571. <a href="https://doi.org/10.1134/S1070363210120248" title="DOI URL">https://doi.org/10.1134/S1070363210120248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070363210120248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070363210120248%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520General%2520Chemistry%26atitle%3DComputational%252Bmethods%252Band%252Bsoftware%252Bin%252Bcomputer-aided%252Bcombinatorial%252Blibrary%252Bdesign%26aulast%3DNasonov%26aufirst%3DA.%2BF.%26date%3D2010%26date%3D2011%26volume%3D80%26issue%3D12%26spage%3D2541%26epage%3D2571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karine  Bourcier</span>, <span class="hlFld-ContribAuthor ">Ruth  Hyland</span>, <span class="hlFld-ContribAuthor ">Sarah  Kempshall</span>, <span class="hlFld-ContribAuthor ">Russell  Jones</span>, <span class="hlFld-ContribAuthor ">Jacqueline  Maximilien</span>, <span class="hlFld-ContribAuthor ">Nicola  Irvine</span>, <span class="hlFld-ContribAuthor ">Barry  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation into UDP-Glucuronosyltransferase (UGT) Enzyme Kinetics of Imidazole- and Triazole-Containing Antifungal Drugs in Human Liver Microsomes and Recombinant UGT Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2010,</strong> <em>38 </em>
                                    (6)
                                     , 923-929. <a href="https://doi.org/10.1124/dmd.109.030676" title="DOI URL">https://doi.org/10.1124/dmd.109.030676</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.109.030676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.109.030676%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DInvestigation%252Binto%252BUDP-Glucuronosyltransferase%252B%252528UGT%252529%252BEnzyme%252BKinetics%252Bof%252BImidazole-%252Band%252BTriazole-Containing%252BAntifungal%252BDrugs%252Bin%252BHuman%252BLiver%252BMicrosomes%252Band%252BRecombinant%252BUGT%252BEnzymes%26aulast%3DBourcier%26aufirst%3DKarine%26date%3D2010%26date%3D2010%26volume%3D38%26issue%3D6%26spage%3D923%26epage%3D929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacques  Magdalou</span>, <span class="hlFld-ContribAuthor ">Sylvie  Fournel-Gigleux</span>, <span class="hlFld-ContribAuthor ">Mohamed  Ouzzine</span>. </span><span class="cited-content_cbyCitation_article-title">Insights on membrane topology and structure/function of UDP-glucuronosyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2010,</strong> <em>42 </em>
                                    (1)
                                     , 159-166. <a href="https://doi.org/10.3109/03602530903209270" title="DOI URL">https://doi.org/10.3109/03602530903209270</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/03602530903209270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F03602530903209270%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DInsights%252Bon%252Bmembrane%252Btopology%252Band%252Bstructure%25252Ffunction%252Bof%252BUDP-glucuronosyltransferases%26aulast%3DMagdalou%26aufirst%3DJacques%26date%3D2010%26date%3D2009%26volume%3D42%26issue%3D1%26spage%3D159%26epage%3D166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Bibliography. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2009,</strong>,, 1-241. <a href="https://doi.org/10.1002/9783527628766.biblio" title="DOI URL">https://doi.org/10.1002/9783527628766.biblio</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527628766.biblio&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527628766.biblio%26sid%3Dliteratum%253Aachs%26atitle%3DBibliography%26date%3D2009%26date%3D2010%26spage%3D1%26epage%3D241%26pub%3DWiley%26atitle%3DMolecular%252BDescriptors%252Bfor%252BChemoinformatics%26aulast%3DTodeschini%26aufirst%3DRoberto%26date%3D2009%26date%3D2010%26volume%3D41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katriina  Itäaho</span>, <span class="hlFld-ContribAuthor ">Peter I.  Mackenzie</span>, <span class="hlFld-ContribAuthor ">Shin-ichi  Ikushiro</span>, <span class="hlFld-ContribAuthor ">John O.  Miners</span>, <span class="hlFld-ContribAuthor ">Moshe  Finel</span>. </span><span class="cited-content_cbyCitation_article-title">The Configuration of the 17-Hydroxy Group Variably Influences the Glucuronidation of β-Estradiol and Epiestradiol by Human UDP-Glucuronosyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2008,</strong> <em>36 </em>
                                    (11)
                                     , 2307-2315. <a href="https://doi.org/10.1124/dmd.108.022731" title="DOI URL">https://doi.org/10.1124/dmd.108.022731</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.108.022731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.108.022731%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DThe%252BConfiguration%252Bof%252Bthe%252B17-Hydroxy%252BGroup%252BVariably%252BInfluences%252Bthe%252BGlucuronidation%252Bof%252B%2525CE%2525B2-Estradiol%252Band%252BEpiestradiol%252Bby%252BHuman%252BUDP-Glucuronosyltransferases%26aulast%3DIt%25C3%25A4aho%26aufirst%3DKatriina%26date%3D2008%26date%3D2008%26volume%3D36%26issue%3D11%26spage%3D2307%26epage%3D2315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samuli-Petrus  Korhonen</span>, <span class="hlFld-ContribAuthor ">Kari  Tuppurainen</span>, <span class="hlFld-ContribAuthor ">Arja  Asikainen</span>, <span class="hlFld-ContribAuthor ">Reino  Laatikainen</span>, <span class="hlFld-ContribAuthor ">Mikael  Peräkylä</span>. </span><span class="cited-content_cbyCitation_article-title">SOMFA on Large Diverse Xenoestrogen Dataset: The Effect of Superposition Algorithms and External Regression Tools. </span><span class="cited-content_cbyCitation_journal-name">QSAR & Combinatorial Science</span><span> <strong>2007,</strong> <em>26 </em>
                                    (7)
                                     , 809-819. <a href="https://doi.org/10.1002/qsar.200610003" title="DOI URL">https://doi.org/10.1002/qsar.200610003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/qsar.200610003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fqsar.200610003%26sid%3Dliteratum%253Aachs%26jtitle%3DQSAR%2520%2526%2520Combinatorial%2520Science%26atitle%3DSOMFA%252Bon%252BLarge%252BDiverse%252BXenoestrogen%252BDataset%25253A%252BThe%252BEffect%252Bof%252BSuperposition%252BAlgorithms%252Band%252BExternal%252BRegression%252BTools%26aulast%3DKorhonen%26aufirst%3DSamuli-Petrus%26date%3D2007%26volume%3D26%26issue%3D7%26spage%3D809%26epage%3D819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Caron</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ermondi</span>, <span class="hlFld-ContribAuthor ">Bernard  Testa</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting the Oxidative Metabolism of Statins: An Application of the MetaSite® Algorithm. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Research</span><span> <strong>2007,</strong> <em>24 </em>
                                    (3)
                                     , 480-501. <a href="https://doi.org/10.1007/s11095-006-9199-7" title="DOI URL">https://doi.org/10.1007/s11095-006-9199-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11095-006-9199-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11095-006-9199-7%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Research%26atitle%3DPredicting%252Bthe%252BOxidative%252BMetabolism%252Bof%252BStatins%25253A%252BAn%252BApplication%252Bof%252Bthe%252BMetaSite%2525C2%2525AE%252BAlgorithm%26aulast%3DCaron%26aufirst%3DGiulia%26date%3D2007%26date%3D2007%26volume%3D24%26issue%3D3%26spage%3D480%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">B.  Testa</span>. </span><span class="cited-content_cbyCitation_article-title">Principles of Drug Metabolism 2: Hydrolysis and Conjugation Reactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 133-166. <a href="https://doi.org/10.1016/B0-08-045044-X/00120-6" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00120-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00120-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00120-6%26sid%3Dliteratum%253Aachs%26atitle%3DPrinciples%252Bof%252BDrug%252BMetabolism%252B2%25253A%252BHydrolysis%252Band%252BConjugation%252BReactions%26aulast%3DTesta%26aufirst%3DB.%26date%3D2007%26spage%3D133%26epage%3D166%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.J.  De Groot</span>, <span class="hlFld-ContribAuthor ">D.F.V.  Lewis</span>, <span class="hlFld-ContribAuthor ">S.  Modi</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Modeling and Quantitative Structure–Activity Relationship of Substrates and Inhibitors of Drug Metabolism Enzymes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 809-825. <a href="https://doi.org/10.1016/B0-08-045044-X/00149-8" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00149-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00149-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00149-8%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BModeling%252Band%252BQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252Bof%252BSubstrates%252Band%252BInhibitors%252Bof%252BDrug%252BMetabolism%252BEnzymes%26aulast%3DDe%2BGroot%26aufirst%3DM.J.%26date%3D2007%26spage%3D809%26epage%3D825%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J.-P.  Renaud</span>, <span class="hlFld-ContribAuthor ">D.  Moras</span>, <span class="hlFld-ContribAuthor ">J.-M.  Wurtz</span>. </span><span class="cited-content_cbyCitation_article-title">Nuclear Hormone Receptors: Insights for Drug Design from Structure and Modeling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 725-747. <a href="https://doi.org/10.1016/B0-08-045044-X/00273-X" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00273-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00273-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00273-X%26sid%3Dliteratum%253Aachs%26atitle%3DNuclear%252BHormone%252BReceptors%25253A%252BInsights%252Bfor%252BDrug%252BDesign%252Bfrom%252BStructure%252Band%252BModeling%26aulast%3DRenaud%26aufirst%3DJ.-P.%26date%3D2007%26spage%3D725%26epage%3D747%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Larry J.  Jolivette</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">Methods for Predicting Human Drug Metabolism. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 131-176. <a href="https://doi.org/10.1016/S0065-2423(06)43005-5" title="DOI URL">https://doi.org/10.1016/S0065-2423(06)43005-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S0065-2423(06)43005-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS0065-2423%2806%2943005-5%26sid%3Dliteratum%253Aachs%26atitle%3DMethods%252Bfor%252BPredicting%252BHuman%252BDrug%252BMetabolism%26aulast%3DJolivette%26aufirst%3DLarry%2BJ.%26date%3D2007%26spage%3D131%26epage%3D176%26pub%3DElsevier%26date%3D2007%26volume%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John O.  Miners</span>, <span class="hlFld-ContribAuthor ">Kathleen M.  Knights</span>, <span class="hlFld-ContribAuthor ">J. Brian  Houston</span>, <span class="hlFld-ContribAuthor ">Peter I.  Mackenzie</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro–in vivo correlation for drugs and other compounds eliminated by glucuronidation in humans: Pitfalls and promises. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2006,</strong> <em>71 </em>
                                    (11)
                                     , 1531-1539. <a href="https://doi.org/10.1016/j.bcp.2005.12.019" title="DOI URL">https://doi.org/10.1016/j.bcp.2005.12.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2005.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2005.12.019%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DIn%252Bvitro%2525E2%252580%252593in%252Bvivo%252Bcorrelation%252Bfor%252Bdrugs%252Band%252Bother%252Bcompounds%252Beliminated%252Bby%252Bglucuronidation%252Bin%252Bhumans%25253A%252BPitfalls%252Band%252Bpromises%26aulast%3DMiners%26aufirst%3DJohn%2BO.%26date%3D2006%26volume%3D71%26issue%3D11%26spage%3D1531%26epage%3D1539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.-P.  Korhonen</span>, <span class="hlFld-ContribAuthor ">K.  Tuppurainen</span>, <span class="hlFld-ContribAuthor ">R.  Laatikainen</span>, <span class="hlFld-ContribAuthor ">M.  Peräkylä</span>. </span><span class="cited-content_cbyCitation_article-title">Improving the performance of SOMFA by use of standard multivariate methods. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2005,</strong> <em>16 </em>
                                    (6)
                                     , 567-579. <a href="https://doi.org/10.1080/10659360500468419" title="DOI URL">https://doi.org/10.1080/10659360500468419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10659360500468419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10659360500468419%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DImproving%252Bthe%252Bperformance%252Bof%252BSOMFA%252Bby%252Buse%252Bof%252Bstandard%252Bmultivariate%252Bmethods%26aulast%3DKorhonen%26aufirst%3DS.-P.%26date%3D2005%26volume%3D16%26issue%3D6%26spage%3D567%26epage%3D579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean  Ekins</span>, <span class="hlFld-ContribAuthor ">Sergey  Andreyev</span>, <span class="hlFld-ContribAuthor ">Andy  Ryabov</span>, <span class="hlFld-ContribAuthor ">Eugene  Kirillov</span>, <span class="hlFld-ContribAuthor ">Eugene A  Rakhmatulin</span>, <span class="hlFld-ContribAuthor ">Andrej  Bugrim</span>, <span class="hlFld-ContribAuthor ">Tatiana  Nikolskaya</span>. </span><span class="cited-content_cbyCitation_article-title">Computational prediction of human drug metabolism. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Metabolism & Toxicology</span><span> <strong>2005,</strong> <em>1 </em>
                                    (2)
                                     , 303-324. <a href="https://doi.org/10.1517/17425255.1.2.303" title="DOI URL">https://doi.org/10.1517/17425255.1.2.303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17425255.1.2.303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17425255.1.2.303%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Metabolism%2520%2526%2520Toxicology%26atitle%3DComputational%252Bprediction%252Bof%252Bhuman%252Bdrug%252Bmetabolism%26aulast%3DEkins%26aufirst%3DSean%26date%3D2005%26date%3D2005%26volume%3D1%26issue%3D2%26spage%3D303%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bernard  Testa</span>, <span class="hlFld-ContribAuthor ">Anne-Loyse  Balmat</span>, <span class="hlFld-ContribAuthor ">Anthony  Long</span>, <span class="hlFld-ContribAuthor ">Philip  Judson</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Drug Metabolism - An Evaluation of the Expert SystemMETEOR. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biodiversity</span><span> <strong>2005,</strong> <em>2 </em>
                                    (7)
                                     , 872-885. <a href="https://doi.org/10.1002/cbdv.200590064" title="DOI URL">https://doi.org/10.1002/cbdv.200590064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbdv.200590064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbdv.200590064%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biodiversity%26atitle%3DPredicting%252BDrug%252BMetabolism%252B-%252BAn%252BEvaluation%252Bof%252Bthe%252BExpert%252BSystemMETEOR%26aulast%3DTesta%26aufirst%3DBernard%26date%3D2005%26volume%3D2%26issue%3D7%26spage%3D872%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">R. H  Lewinsky</span>, <span class="hlFld-ContribAuthor ">P. A  Smith</span>, <span class="hlFld-ContribAuthor ">P. I  Mackenzie</span>. </span><span class="cited-content_cbyCitation_article-title">Glucuronidation of bioflavonoids by human UGT1A10: structure–function relationships. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2005,</strong> <em>35 </em>
                                    (2)
                                     , 117-129. <a href="https://doi.org/10.1080/00498250400028189" title="DOI URL">https://doi.org/10.1080/00498250400028189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00498250400028189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00498250400028189%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DGlucuronidation%252Bof%252Bbioflavonoids%252Bby%252Bhuman%252BUGT1A10%25253A%252Bstructure%2525E2%252580%252593function%252Brelationships%26aulast%3DLewinsky%26aufirst%3DR.%2BH%26date%3D2005%26date%3D2008%26volume%3D35%26issue%3D2%26spage%3D117%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.A  Smith</span>, <span class="hlFld-ContribAuthor ">M.J  Sorich</span>, <span class="hlFld-ContribAuthor ">L.S.C  Low</span>, <span class="hlFld-ContribAuthor ">R.A  McKinnon</span>, <span class="hlFld-ContribAuthor ">J.O  Miners</span>. </span><span class="cited-content_cbyCitation_article-title">Towards integrated ADME prediction: past, present and future directions for modelling metabolism by UDP-glucuronosyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2004,</strong> <em>22 </em>
                                    (6)
                                     , 507-517. <a href="https://doi.org/10.1016/j.jmgm.2004.03.011" title="DOI URL">https://doi.org/10.1016/j.jmgm.2004.03.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2004.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2004.03.011%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DTowards%252Bintegrated%252BADME%252Bprediction%25253A%252Bpast%25252C%252Bpresent%252Band%252Bfuture%252Bdirections%252Bfor%252Bmodelling%252Bmetabolism%252Bby%252BUDP-glucuronosyltransferases%26aulast%3DSmith%26aufirst%3DP.A%26date%3D2004%26volume%3D22%26issue%3D6%26spage%3D507%26epage%3D517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting undesirable drug interactions with promiscuous proteins in silico. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2004,</strong> <em>9 </em>
                                    (6)
                                     , 276-285. <a href="https://doi.org/10.1016/S1359-6446(03)03008-3" title="DOI URL">https://doi.org/10.1016/S1359-6446(03)03008-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1359-6446(03)03008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1359-6446%2803%2903008-3%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DPredicting%252Bundesirable%252Bdrug%252Binteractions%252Bwith%252Bpromiscuous%252Bproteins%252Bin%252Bsilico%26aulast%3DEkins%26aufirst%3DSean%26date%3D2004%26volume%3D9%26issue%3D6%26spage%3D276%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. S.  Verkman</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery in academia. </span><span class="cited-content_cbyCitation_journal-name">American Journal of Physiology-Cell Physiology</span><span> <strong>2004,</strong> <em>286 </em>
                                    (3)
                                     , C465-C474. <a href="https://doi.org/10.1152/ajpcell.00397.2003" title="DOI URL">https://doi.org/10.1152/ajpcell.00397.2003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1152/ajpcell.00397.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1152%2Fajpcell.00397.2003%26sid%3Dliteratum%253Aachs%26jtitle%3DAmerican%2520Journal%2520of%2520Physiology-Cell%2520Physiology%26atitle%3DDrug%252Bdiscovery%252Bin%252Bacademia%26aulast%3DVerkman%26aufirst%3DA.%2BS.%26date%3D2004%26volume%3D286%26issue%3D3%26spage%3DC465%26epage%3DC474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John O.  Miners</span>, <span class="hlFld-ContribAuthor ">Paul A.  Smith</span>, <span class="hlFld-ContribAuthor ">Michael J.  Sorich</span>, <span class="hlFld-ContribAuthor ">Ross A.  McKinnon</span>, <span class="hlFld-ContribAuthor ">Peter I.  Mackenzie</span>. </span><span class="cited-content_cbyCitation_article-title">P
              REDICTING
              H
              UMAN
              D
              RUG
              G
              LUCURONIDATION
              P
              ARAMETERS
              : Application of In Vitro and In Silico Modeling Approaches. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Pharmacology and Toxicology</span><span> <strong>2004,</strong> <em>44 </em>
                                    (1)
                                     , 1-25. <a href="https://doi.org/10.1146/annurev.pharmtox.44.101802.121546" title="DOI URL">https://doi.org/10.1146/annurev.pharmtox.44.101802.121546</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev.pharmtox.44.101802.121546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev.pharmtox.44.101802.121546%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Pharmacology%2520and%2520Toxicology%26atitle%3DP%252BREDICTING%252BH%252BUMAN%252BD%252BRUG%252BG%252BLUCURONIDATION%252BP%252BARAMETERS%252B%25253A%252BApplication%252Bof%252BIn%252BVitro%252Band%252BIn%252BSilico%252BModeling%252BApproaches%26aulast%3DMiners%26aufirst%3DJohn%2BO.%26date%3D2004%26volume%3D44%26issue%3D1%26spage%3D1%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Sorich</span>, <span class="hlFld-ContribAuthor ">John O.  Miners</span>, <span class="hlFld-ContribAuthor ">Ross A.  McKinnon</span>, <span class="hlFld-ContribAuthor ">Paul A.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple Pharmacophores for the Investigation of Human UDP-Glucuronosyltransferase Isoform Substrate Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2004,</strong> <em>65 </em>
                                    (2)
                                     , 301-308. <a href="https://doi.org/10.1124/mol.65.2.301" title="DOI URL">https://doi.org/10.1124/mol.65.2.301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.65.2.301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.65.2.301%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DMultiple%252BPharmacophores%252Bfor%252Bthe%252BInvestigation%252Bof%252BHuman%252BUDP-Glucuronosyltransferase%252BIsoform%252BSubstrate%252BSelectivity%26aulast%3DSorich%26aufirst%3DMichael%2BJ.%26date%3D2004%26date%3D2004%26volume%3D65%26issue%3D2%26spage%3D301%26epage%3D308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maurice  Dickins</span>, <span class="hlFld-ContribAuthor ">Han  van de Waterbeemd</span>. </span><span class="cited-content_cbyCitation_article-title">Simulation models for drug disposition and drug interactions. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: BIOSILICO</span><span> <strong>2004,</strong> <em>2 </em>
                                    (1)
                                     , 38-45. <a href="https://doi.org/10.1016/S1741-8364(04)02388-1" title="DOI URL">https://doi.org/10.1016/S1741-8364(04)02388-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1741-8364(04)02388-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1741-8364%2804%2902388-1%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520BIOSILICO%26atitle%3DSimulation%252Bmodels%252Bfor%252Bdrug%252Bdisposition%252Band%252Bdrug%252Binteractions%26aulast%3DDickins%26aufirst%3DMaurice%26date%3D2004%26volume%3D2%26issue%3D1%26spage%3D38%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">PA  Smith</span>, <span class="hlFld-ContribAuthor ">MJ  Sorich</span>, <span class="hlFld-ContribAuthor ">RA  McKinnon</span>, <span class="hlFld-ContribAuthor ">JO  Miners</span>. </span><span class="cited-content_cbyCitation_article-title">In silico insights: Chemical and structural characteristics associated with uridine diphosphate-glucuronosyltransferase substrate selectivity. </span><span class="cited-content_cbyCitation_journal-name">Clinical and Experimental Pharmacology and Physiology</span><span> <strong>2003,</strong> <em>30 </em>
                                    (11)
                                     , 836-840. <a href="https://doi.org/10.1046/j.1440-1681.2003.03923.x" title="DOI URL">https://doi.org/10.1046/j.1440-1681.2003.03923.x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1046/j.1440-1681.2003.03923.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1046%2Fj.1440-1681.2003.03923.x%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520and%2520Experimental%2520Pharmacology%2520and%2520Physiology%26atitle%3DIn%252Bsilico%252Binsights%25253A%252BChemical%252Band%252Bstructural%252Bcharacteristics%252Bassociated%252Bwith%252Buridine%252Bdiphosphate-glucuronosyltransferase%252Bsubstrate%252Bselectivity%26aulast%3DSmith%26aufirst%3DPA%26date%3D2003%26volume%3D30%26issue%3D11%26spage%3D836%26epage%3D840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm020397cf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Structures and <i>K</i><sub>m,app</sub> values of the 24 UGT1A4 substrates in the training and test sets.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm020397cf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00002.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 (a) The “common features” pharmacophore model developed using the HipHop module of Catalyst. Cyan spheres represent hydrophobic regions, and purple represents the “glucuronidation” feature. (b) Eugenol mapped to the Hypogen pharmacophore. (c) Clozapine mapped to the Hypogen pharmacophore.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm020397cf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/medium/jm020397cf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00003.jpeg" id="rightTab-_i11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 (a) Pharmacophore-based 3D-QSAR model developed using the Hypogen module of Catalyst. Cyan spheres represent hydrophobic regions, and purple represents the “glucuronidation” feature. (b) Imipramine mapped to the Hypogen pharmacophore. (c) Benzidine mapped to the Hypogen pharmacophore.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-9/jm020397c/production/images/large/jm020397cf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm020397c&amp;id=jm020397cf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><a name="jm004600901626" id="jm004600901626"></a></div><p class="rightTab-references-count">
            This article references 38 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm020397cb00001"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00001_1"><span><span class="NLM_string-ref">Roberts, S. A. High-throughput screening approaches for investigating drug metabolism and pharmacokinetics [Review]. <i>  Xenobiotica </i><b>2001</b><i>, 31</i>, 557−589.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2001&pages=589&author=S.+A+Roberts&title=High-throughput+screening+approaches+for+investigating+drug+metabolism+and+pharmacokinetics+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DS.%2BA%26atitle%3DHigh-throughput%2520screening%2520approaches%2520for%2520investigating%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520%255BReview%255D%26jtitle%3DXenobiotica%26date%3D2001%26volume%3D31%26spage%3D589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00002"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00002_1"><span><span class="NLM_string-ref">Ekins, S.; Waller, C. L.; Swaan, P. W.; Cruciani, G.; Wrighton, S. A.; et al. Progress in predicting human ADME parameters in silico [Review]. <i>  J. </i><i>Pharmacol. Toxicol. Methods </i><b>2000</b><i>, 44</i>, 251−272.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2000&pages=272&author=S.+Ekins&author=C.+L.+Waller&author=P.+W.+Swaan&author=G.+Cruciani&author=S.+A.+Wrighton&title=Progress+in+predicting+human+ADME+parameters+in+silico+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DProgress%2520in%2520predicting%2520human%2520ADME%2520parameters%2520in%2520silico%2520%255BReview%255D%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2000%26volume%3D44%26spage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00003"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00003_1"><span><span class="NLM_string-ref">Miners, J. O.; Mackenzie, P. I. Drug glucuronidation in humans. <i>  Pharmacol. </i><i>Ther. </i><b>1991</b><i>, 51</i>, 347−369.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=369&author=J.+O.+Miners&author=P.+I+Mackenzie&title=Drug+glucuronidation+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiners%26aufirst%3DJ.%2BO.%26atitle%3DDrug%2520glucuronidation%2520in%2520humans%26jtitle%3DPharmacol.%2520Ther.%26date%3D1991%26volume%3D51%26spage%3D369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00004"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00004_1"><span><span class="NLM_string-ref">Wrighton, S. A.; Stevens, J. C. The human hepatic cytochromes P450 involved in drug metabolism. <i>  Crit. Rev. Toxicol. </i><b>1992</b><i>, 22</i>, 1−21.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1992&pages=21&author=S.+A.+Wrighton&author=J.+C+Stevens&title=The+human+hepatic+cytochromes+P450+involved+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWrighton%26aufirst%3DS.%2BA.%26atitle%3DThe%2520human%2520hepatic%2520cytochromes%2520P450%2520involved%2520in%2520drug%2520metabolism%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D1992%26volume%3D22%26spage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00005"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00005_1"><span><span class="NLM_string-ref">Lewis, D. F.; Dickins, M.; Eddershaw, P. J.; Tarbit, M. H.; Goldfarb, P. S. Cytochrome P450 substrate specificities, substrate structural templates and enzyme active site geometries. <i>  Drug Metab. Drug Interact. </i><b>1999</b><i>, 15</i>, 1−49.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=49&author=D.+F.+Lewis&author=M.+Dickins&author=P.+J.+Eddershaw&author=M.+H.+Tarbit&author=P.+S+Goldfarb&title=Cytochrome+P450+substrate+specificities%2C+substrate+structural+templates+and+enzyme+active+site+geometries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DD.%2BF.%26atitle%3DCytochrome%2520P450%2520substrate%2520specificities%252C%2520substrate%2520structural%2520templates%2520and%2520enzyme%2520active%2520site%2520geometries%26jtitle%3DDrug%2520Metab.%2520Drug%2520Interact.%26date%3D1999%26volume%3D15%26spage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00006"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00006_1"><span><span class="NLM_string-ref">Dai, R.; Pincus, M. R.; Friedman, F. K. Molecular modeling of mammalian cytochrome P450s [Review]. <i>  Cell. Mol. Life Sci. </i><b>2000</b><i>, 57</i>, 487−499.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=499&author=R.+Dai&author=M.+R.+Pincus&author=F.+K+Friedman&title=Molecular+modeling+of+mammalian+cytochrome+P450s+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DR.%26atitle%3DMolecular%2520modeling%2520of%2520mammalian%2520cytochrome%2520P450s%2520%255BReview%255D%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2000%26volume%3D57%26spage%3D499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00007"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00007_1"><span><span class="NLM_string-ref">Ekins, S.; De Groot, M. J.; Jones, J. P. Pharmacophore and three-dimensional quantitative structure activity relationship methods for modeling cytochrome P450 active sites [Review]. <i>  Drug Metab. Dispos. </i><b>2001</b><i>, 29</i>, 936−944.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=944&author=S.+Ekins&author=M.+J.+De+Groot&author=J.+P+Jones&title=Pharmacophore+and+three-dimensional+quantitative+structure+activity+relationship+methods+for+modeling+cytochrome+P450+active+sites+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DPharmacophore%2520and%2520three-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520methods%2520for%2520modeling%2520cytochrome%2520P450%2520active%2520sites%2520%255BReview%255D%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2001%26volume%3D29%26spage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00008"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00008_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three- and four-dimensional quantitative structure activity relationship analyses of cytochrome P-450 3A4 inhibitors. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 290</i>, 429−438.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=1999&pages=438&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three-+and+four-dimensional+quantitative+structure+activity+relationship+analyses+of+cytochrome+P-450+3A4+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-%2520and%2520four-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%2520of%2520cytochrome%2520P-450%25203A4%2520inhibitors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D290%26spage%3D438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00009"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00009_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three and four dimensional-quantitative structure activity relationship (3D/4D-QSAR) analyses of CYP2D6 inhibitors. <i>  Pharmacogenetics </i><b>1999</b><i>, 9</i>, 477−489.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=489&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three+and+four+dimensional-quantitative+structure+activity+relationship+%283D%2F4D-QSAR%29+analyses+of+CYP2D6+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree%2520and%2520four%2520dimensional-quantitative%2520structure%2520activity%2520relationship%2520%25283D%252F4D-QSAR%2529%2520analyses%2520of%2520CYP2D6%2520inhibitors%26jtitle%3DPharmacogenetics%26date%3D1999%26volume%3D9%26spage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm020397cb00010_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Binkley, S.; Gillespie, J. S.; Ring, B. J.; et al. Three- and four-dimensional-quantitative structure activity relationship (3D/4D-QSAR) analyses of CYP2C9 inhibitors. <i>  Drug Metab. Dispos. </i><b>2000</b><i>, 28</i>, 994−1002.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1002&author=S.+Ekins&author=G.+Bravi&author=S.+Binkley&author=J.+S.+Gillespie&author=B.+J.+Ring&title=Three-+and+four-dimensional-quantitative+structure+activity+relationship+%283D%2F4D-QSAR%29+analyses+of+CYP2C9+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-%2520and%2520four-dimensional-quantitative%2520structure%2520activity%2520relationship%2520%25283D%252F4D-QSAR%2529%2520analyses%2520of%2520CYP2C9%2520inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00011"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00011_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Ring, B. J.; Gillespie, T. A.; Gillespie, J. S.; et al. Three-dimensional quantitative structure activity relationship analyses of substrates for CYP2B6. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 288</i>, 21−29.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1999&pages=29&author=S.+Ekins&author=G.+Bravi&author=B.+J.+Ring&author=T.+A.+Gillespie&author=J.+S.+Gillespie&title=Three-dimensional+quantitative+structure+activity+relationship+analyses+of+substrates+for+CYP2B6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%2520of%2520substrates%2520for%2520CYP2B6%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D288%26spage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00012"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00012_1"><span><span class="NLM_string-ref">Ekins, S.; Bravi, G.; Wikel, J. H.; Wrighton, S. A. Three-dimensional-quantitative structure activity relationship analysis of cytochrome P-450 3A4 substrates. <i>  J. Pharmacol. Exp. Ther. </i><b>1999</b><i>, 291</i>, 424−433.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=1999&pages=433&author=S.+Ekins&author=G.+Bravi&author=J.+H.+Wikel&author=S.+A+Wrighton&title=Three-dimensional-quantitative+structure+activity+relationship+analysis+of+cytochrome+P-450+3A4+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional-quantitative%2520structure%2520activity%2520relationship%2520analysis%2520of%2520cytochrome%2520P-450%25203A4%2520substrates%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D291%26spage%3D433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00013"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00013_1"><span><span class="NLM_string-ref">Afzelius, L.; Zamora, I.; Ridderstrom, M.; Andersson, T. B.; Karlen, A.; et al. Competitive CYP2C9 inhibitors:  Enzyme inhibition studies, protein homology modeling, and three-dimensional quantitative structure−activity relationship analysis. <i>  Mol. Pharmacol. </i><b>2001</b><i>, 59</i>, 909−919.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2001&pages=919&author=L.+Afzelius&author=I.+Zamora&author=M.+Ridderstrom&author=T.+B.+Andersson&author=A.+Karlen&author=+Competitive&title=Enzyme+inhibition+studies%2C+protein+homology+modeling%2C+and+three-dimensional+quantitative+structure%E2%88%92activity+relationship+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAfzelius%26aufirst%3DL.%26atitle%3DEnzyme%2520inhibition%2520studies%252C%2520protein%2520homology%2520modeling%252C%2520and%2520three-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520analysis%26jtitle%3DMol.%2520Pharmacol.%26date%3D2001%26volume%3D59%26spage%3D919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00014"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00014_1"><span><span class="NLM_string-ref">Wang, Q.; Halpert, J. R. Combined three-dimensional quantitative structure−activity relationship analysis of cytochrome P450 2B6 substrates and protein homology modeling. <i>  Drug Metab. Dispos. </i><b>2002</b><i>, 30</i>, 86−95.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=95&author=Q.+Wang&author=J.+R+Halpert&title=Combined+three-dimensional+quantitative+structure%E2%88%92activity+relationship+analysis+of+cytochrome+P450+2B6+substrates+and+protein+homology+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26atitle%3DCombined%2520three-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520analysis%2520of%2520cytochrome%2520P450%25202B6%2520substrates%2520and%2520protein%2520homology%2520modeling%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00015"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00015_1"><span><span class="NLM_string-ref">Green, M. D.; Tephly, T. R. Glucuronidation of Amines and Hydroxylated Xenobiotics and Endobiotics Catalyzed by Expressed Human Ugt1.4 Protein. <i>  Drug Metab. Dispos. </i><b>1996</b><i>, 24</i>, 356−363.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1996&pages=363&author=M.+D.+Green&author=T.+R+Tephly&title=Glucuronidation+of+Amines+and+Hydroxylated+Xenobiotics+and+Endobiotics+Catalyzed+by+Expressed+Human+Ugt1.4+Protein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BD.%26atitle%3DGlucuronidation%2520of%2520Amines%2520and%2520Hydroxylated%2520Xenobiotics%2520and%2520Endobiotics%2520Catalyzed%2520by%2520Expressed%2520Human%2520Ugt1.4%2520Protein%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1996%26volume%3D24%26spage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00016"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00016_1"><span><span class="NLM_string-ref">Green, M. D.; Bishop, W. P.; Tephly, T. R. Expressed Human Ugt1.4 Protein Catalyzes the Formation of Quaternary Ammonium-Linked Glucuronides. <i>  Drug Metab. Dispos. </i><b>1995</b><i>, 23</i>, 299−302.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1995&pages=302&author=M.+D.+Green&author=W.+P.+Bishop&author=T.+R+Tephly&title=Expressed+Human+Ugt1.4+Protein+Catalyzes+the+Formation+of+Quaternary+Ammonium-Linked+Glucuronides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BD.%26atitle%3DExpressed%2520Human%2520Ugt1.4%2520Protein%2520Catalyzes%2520the%2520Formation%2520of%2520Quaternary%2520Ammonium-Linked%2520Glucuronides%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1995%26volume%3D23%26spage%3D302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00017"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00017_1"><span><span class="NLM_string-ref">Sprague, P. W. Automated Chemical Hypothesis Generation and Database Searching with Catalyst [Review]. <i>  Perspect. Drug Discovery Des. </i><b>1995</b><i>, 3</i>, 1−20.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=20&author=P.+W+Sprague&title=Automated+Chemical+Hypothesis+Generation+and+Database+Searching+with+Catalyst+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSprague%26aufirst%3DP.%2BW%26atitle%3DAutomated%2520Chemical%2520Hypothesis%2520Generation%2520and%2520Database%2520Searching%2520with%2520Catalyst%2520%255BReview%255D%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D1995%26volume%3D3%26spage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00018"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00018_1"><span><span class="NLM_string-ref">Robinson, D. D.; Winn, P. J.; Lyne, P. D.; Richards, W. G. Self-organizing molecular field analysis:  A tool for structure−activity studies. <i>  J. Med. </i><i>Chem. </i><b>1999</b><i>, 42</i>, 573−583.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=583&author=D.+D.+Robinson&author=P.+J.+Winn&author=P.+D.+Lyne&author=W.+G+Richards&title=Self-organizing+molecular+field+analysis%3A%E2%80%89+A+tool+for+structure%E2%88%92activity+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DD.%2BD.%26atitle%3DSelf-organizing%2520molecular%2520field%2520analysis%253A%25E2%2580%2589%2520A%2520tool%2520for%2520structure%25E2%2588%2592activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00019"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00019_1"><span><span class="NLM_string-ref">Whitley, D. C.; Ford, M. G.; Livingstone, D. J. Unsupervised forward selection:  a method for eliminating redundant variables. <i>  J. Chem. Inf. Comput. Sci. </i><b>2000</b><i>, 40</i>, 1160−1168.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=1168&author=D.+C.+Whitley&author=M.+G.+Ford&author=D.+J+Livingstone&title=Unsupervised+forward+selection%3A%E2%80%89+a+method+for+eliminating+redundant+variables"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DUnsupervised%2520forward%2520selection%253A%25E2%2580%2589%2520a%2520method%2520for%2520eliminating%2520redundant%2520variables%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2000%26volume%3D40%26spage%3D1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm020397cb00020_1"><span><span class="NLM_string-ref">Said, M.; Ziegler, J. C.; Magdalou, J.; Elass, A.; Vergoten, G. Inhibition of Bilirubin Udp-GlucuronosyltransferaseA Comparative Molecular Field Analysis (Comfa). <i>  Quant. Struct.</i><i>−</i><i>Act. Relat. </i><b>1996</b><i>, 15</i>, 382−388.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=388&author=M.+Said&author=J.+C.+Ziegler&author=J.+Magdalou&author=A.+Elass&author=G+Vergoten&title=Inhibition+of+Bilirubin+Udp-GlucuronosyltransferaseA+Comparative+Molecular+Field+Analysis+%28Comfa%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaid%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520Bilirubin%2520Udp-GlucuronosyltransferaseA%2520Comparative%2520Molecular%2520Field%2520Analysis%2520%2528Comfa%2529%26jtitle%3DQuant.%2520Struct.%25E2%2588%2592Act.%2520Relat.%26date%3D1996%26volume%3D15%26spage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00021"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00021_1"><span><span class="NLM_string-ref">Said, M.; Battaglia, E.; Elass, A.; Cano, V.; Ziegler, J. C.; et al. Mechanism of inhibition of rat liver bilirubin UDP-glucuronosyltransferase by triphenylalkyl derivatives. <i>  J. Biochem. Mol. Toxicol. </i><b>1998</b><i>, 12</i>, 19−27.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=27&author=M.+Said&author=E.+Battaglia&author=A.+Elass&author=V.+Cano&author=J.+C.+Ziegler&title=Mechanism+of+inhibition+of+rat+liver+bilirubin+UDP-glucuronosyltransferase+by+triphenylalkyl+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaid%26aufirst%3DM.%26atitle%3DMechanism%2520of%2520inhibition%2520of%2520rat%2520liver%2520bilirubin%2520UDP-glucuronosyltransferase%2520by%2520triphenylalkyl%2520derivatives%26jtitle%3DJ.%2520Biochem.%2520Mol.%2520Toxicol.%26date%3D1998%26volume%3D12%26spage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00022"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00022_1"><span><span class="NLM_string-ref">Naydenova, Z. G.; Grancharov, K. C.; Alargov, D. K.; Golovinsky, E. V.; Stanoeva, I. M.; et al. Inhibition of Udp-Glucuronosyltransferase by 5‘-<i>O</i>-Amino Acid and Oligopeptide Derivatives of Uridine-Structure−Activity Relationships. <i>  Z. Naturforsch., C:  J. Biosci. </i><b>1998</b><i>, 53</i>, 173−181.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&author=Z.+G.+Naydenova&author=K.+C.+Grancharov&author=D.+K.+Alargov&author=E.+V.+Golovinsky&author=I.+M.+Stanoeva&title=C%3A%E2%80%89+J.+Biosci."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DNaydenova%26aufirst%3DZ.%2BG.%26atitle%3DInhibition%2520of%2520Udp-Glucuronosyltransferase%2520by%25205%25E2%2580%2598-O-Amino%2520Acid%2520and%2520Oligopeptide%2520Derivatives%2520of%2520Uridine-Structure%25E2%2588%2592Activity%2520Relationships.%2520Z.%2520Naturforsch%26btitle%3DC%253A%25E2%2580%2589%2520J.%2520Biosci.%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00023"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00023_1"><span><span class="NLM_string-ref">Cupid, B. C.; Holmes, E.; Wilson, I. D.; Lindon, J. C.; Nicholson, J. K. Quantitative structure−metabolism relationships (QSMR) using computational chemistry:  pattern recognition analysis and statistical prediction of phase II conjugation reactions of substituted benzoic acids in the rat. <i>  Xenobiotica </i><b>1999</b><i>, 29</i>, 27−42.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1999&pages=42&author=B.+C.+Cupid&author=E.+Holmes&author=I.+D.+Wilson&author=J.+C.+Lindon&author=J.+K+Nicholson&title=Quantitative+structure%E2%88%92metabolism+relationships+%28QSMR%29+using+computational+chemistry%3A%E2%80%89+pattern+recognition+analysis+and+statistical+prediction+of+phase+II+conjugation+reactions+of+substituted+benzoic+acids+in+the+rat"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCupid%26aufirst%3DB.%2BC.%26atitle%3DQuantitative%2520structure%25E2%2588%2592metabolism%2520relationships%2520%2528QSMR%2529%2520using%2520computational%2520chemistry%253A%25E2%2580%2589%2520pattern%2520recognition%2520analysis%2520and%2520statistical%2520prediction%2520of%2520phase%2520II%2520conjugation%2520reactions%2520of%2520substituted%2520benzoic%2520acids%2520in%2520the%2520rat%26jtitle%3DXenobiotica%26date%3D1999%26volume%3D29%26spage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00024"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00024_1"><span><span class="NLM_string-ref">Ghauri, F. Y.; Blackledge, C. A.; Glen, R. C.; Sweatman, B. C.; Lindon, J. C.; et al. Quantitative structure−metabolism relationships for substituted benzoic acids in the rat. Computational chemistry, NMR spectroscopy and pattern recognition studies. <i>  Biochem. Pharmacol. </i><b>1992</b><i>, 44</i>, 1935−1946.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1992&pages=1946&author=F.+Y.+Ghauri&author=C.+A.+Blackledge&author=R.+C.+Glen&author=B.+C.+Sweatman&author=J.+C.+Lindon&title=Quantitative+structure%E2%88%92metabolism+relationships+for+substituted+benzoic+acids+in+the+rat.+Computational+chemistry%2C+NMR+spectroscopy+and+pattern+recognition+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhauri%26aufirst%3DF.%2BY.%26atitle%3DQuantitative%2520structure%25E2%2588%2592metabolism%2520relationships%2520for%2520substituted%2520benzoic%2520acids%2520in%2520the%2520rat.%2520Computational%2520chemistry%252C%2520NMR%2520spectroscopy%2520and%2520pattern%2520recognition%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1992%26volume%3D44%26spage%3D1946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00025"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00025_1"><span><span class="NLM_string-ref">Kim, K. H. Quantitative structure−activity relationships of the metabolism of drugs by uridine diphosphate glucuronosyltransferase. <i>  J. Pharm. Sci. </i><b>1991</b><i>, </i><i>80</i>, 966−970.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1991&pages=970&author=K.+H+Kim&title=Quantitative+structure%E2%88%92activity+relationships+of+the+metabolism+of+drugs+by+uridine+diphosphate+glucuronosyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH%26atitle%3DQuantitative%2520structure%25E2%2588%2592activity%2520relationships%2520of%2520the%2520metabolism%2520of%2520drugs%2520by%2520uridine%2520diphosphate%2520glucuronosyltransferase%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1991%26volume%3D80%26spage%3D970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00026"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00026_1"><span><span class="NLM_string-ref">Resetar, A.; Minick, D.; Spector, T. Glucuronidation of 3‘-azido-3‘-deoxythymidine catalyzed by human liver UDP-glucuronosyltransferase. Significance of nucleoside hydrophobicity and inhibition by xenobiotics. <i>  Biochem. Pharmacol. </i><b>1991</b><i>, 42</i>, 559−568.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1991&pages=568&author=A.+Resetar&author=D.+Minick&author=T+Spector&title=Glucuronidation+of+3%E2%80%98-azido-3%E2%80%98-deoxythymidine+catalyzed+by+human+liver+UDP-glucuronosyltransferase.+Significance+of+nucleoside+hydrophobicity+and+inhibition+by+xenobiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DResetar%26aufirst%3DA.%26atitle%3DGlucuronidation%2520of%25203%25E2%2580%2598-azido-3%25E2%2580%2598-deoxythymidine%2520catalyzed%2520by%2520human%2520liver%2520UDP-glucuronosyltransferase.%2520Significance%2520of%2520nucleoside%2520hydrophobicity%2520and%2520inhibition%2520by%2520xenobiotics%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1991%26volume%3D42%26spage%3D568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00027"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00027_1"><span><span class="NLM_string-ref">Yin, H. Q.; Bennett, G.; Jones, J. P. Mechanistic Studies of Uridine Diphosphate Glucuronosyltransferase. <i>  Chem.-Biol. Interact. </i><b>1994</b><i>, 90</i>, 47−58.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1994&pages=58&author=H.+Q.+Yin&author=G.+Bennett&author=J.+P+Jones&title=Mechanistic+Studies+of+Uridine+Diphosphate+Glucuronosyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DH.%2BQ.%26atitle%3DMechanistic%2520Studies%2520of%2520Uridine%2520Diphosphate%2520Glucuronosyltransferase%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D1994%26volume%3D90%26spage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00028"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00028_1"><span><span class="NLM_string-ref">Antonio, L.; Grillasca, J.; Taskinen, J.; Elovaara, E.; Burchell, B.; et al. Characterization of catechol glucuronidation in rat liver. <i>  Drug Metab. </i><i>Dispos. </i><b>2002</b><i>, 30</i>, 199−207.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=207&author=L.+Antonio&author=J.+Grillasca&author=J.+Taskinen&author=E.+Elovaara&author=B.+Burchell&title=Characterization+of+catechol+glucuronidation+in+rat+liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntonio%26aufirst%3DL.%26atitle%3DCharacterization%2520of%2520catechol%2520glucuronidation%2520in%2520rat%2520liver%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00029"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00029_1"><span><span class="NLM_string-ref">de Groot, M. J.; Ackland, M. J.; Horne, V. A.; Alex, A. A.; Jones, B. C. A novel approach to predicting P450 mediated drug metabolism. CYP2D6 catalyzed N-dealkylation reactions and qualitative metabolite predictions using a combined protein and pharmacophore model for CYP2D6. <i>  J. Med. </i><i>Chem. </i><b>1999</b><i>, 42</i>, 4062−4070.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4070&author=M.+J.+de+Groot&author=M.+J.+Ackland&author=V.+A.+Horne&author=A.+A.+Alex&author=B.+C+Jones&title=A+novel+approach+to+predicting+P450+mediated+drug+metabolism.+CYP2D6+catalyzed+N-dealkylation+reactions+and+qualitative+metabolite+predictions+using+a+combined+protein+and+pharmacophore+model+for+CYP2D6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DM.%2BJ.%26atitle%3DA%2520novel%2520approach%2520to%2520predicting%2520P450%2520mediated%2520drug%2520metabolism.%2520CYP2D6%2520catalyzed%2520N-dealkylation%2520reactions%2520and%2520qualitative%2520metabolite%2520predictions%2520using%2520a%2520combined%2520protein%2520and%2520pharmacophore%2520model%2520for%2520CYP2D6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm020397cb00030_1"><span><span class="NLM_string-ref">de Groot, M. J.; Ackland, M. J.; Horne, V. A.; Alex, A. A.; Jones, B. C. Novel approach to predicting P450-mediated drug metabolism:  Development of a combined protein and pharmacophore model for CYP2D6. <i>  J. Med. Chem. </i><b>1999</b><i>, 42</i>, 1515−1524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1524&author=M.+J.+de+Groot&author=M.+J.+Ackland&author=V.+A.+Horne&author=A.+A.+Alex&author=B.+C+Jones&title=Novel+approach+to+predicting+P450-mediated+drug+metabolism%3A%E2%80%89+Development+of+a+combined+protein+and+pharmacophore+model+for+CYP2D6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DM.%2BJ.%26atitle%3DNovel%2520approach%2520to%2520predicting%2520P450-mediated%2520drug%2520metabolism%253A%25E2%2580%2589%2520Development%2520of%2520a%2520combined%2520protein%2520and%2520pharmacophore%2520model%2520for%2520CYP2D6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00031"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00031_1"><span><span class="NLM_string-ref">Radominska-Pandya, A.; Czernik, P. J.; Little, J. M.; Battaglia, E.; Mackenzie, P. I. Structural and functional studies of UDP-glucuronosyltransferases [Review]. <i>  Drug Metab. Rev. </i><b>1999</b><i>, 31</i>, 817−899.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1999&pages=899&author=A.+Radominska-Pandya&author=P.+J.+Czernik&author=J.+M.+Little&author=E.+Battaglia&author=P.+I+Mackenzie&title=Structural+and+functional+studies+of+UDP-glucuronosyltransferases+%5BReview%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRadominska-Pandya%26aufirst%3DA.%26atitle%3DStructural%2520and%2520functional%2520studies%2520of%2520UDP-glucuronosyltransferases%2520%255BReview%255D%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D1999%26volume%3D31%26spage%3D899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00032"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00032_1"><span><span class="NLM_string-ref">Jackson, M. R.; Fournel-Gigleux, S.; Harding, D.; Burchell, B. Examination of the substrate specificity of cloned rat kidney phenol UDP-glucuronyltransferase expressed in COS-7 cells. <i>  Mol. Pharmacol. </i><b>1988</b><i>, 34</i>, 638−642.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1988&pages=642&author=M.+R.+Jackson&author=S.+Fournel-Gigleux&author=D.+Harding&author=B+Burchell&title=Examination+of+the+substrate+specificity+of+cloned+rat+kidney+phenol+UDP-glucuronyltransferase+expressed+in+COS-7+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DM.%2BR.%26atitle%3DExamination%2520of%2520the%2520substrate%2520specificity%2520of%2520cloned%2520rat%2520kidney%2520phenol%2520UDP-glucuronyltransferase%2520expressed%2520in%2520COS-7%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1988%26volume%3D34%26spage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00033"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00033_1"><span><span class="NLM_string-ref">Sorich, M. J.; Smith, P. A.; McKinnon, R. A.; Miners, J. O. Pharmacophore and Quantitative Structure Activity Relationship Modeling of UDP-Glucuronosyltransferase 1A1 (UGT1A1) Substrates. <i>  Pharmacogenetics </i><b>2002</b><i>, 12</i> (8), 635−645.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=645&issue=8&author=M.+J.+Sorich&author=P.+A.+Smith&author=R.+A.+McKinnon&author=J.+O+Miners&title=Pharmacophore+and+Quantitative+Structure+Activity+Relationship+Modeling+of+UDP-Glucuronosyltransferase+1A1+%28UGT1A1%29+Substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSorich%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520and%2520Quantitative%2520Structure%2520Activity%2520Relationship%2520Modeling%2520of%2520UDP-Glucuronosyltransferase%25201A1%2520%2528UGT1A1%2529%2520Substrates%26jtitle%3DPharmacogenetics%26date%3D2002%26volume%3D12%26issue%3D8%26spage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00034"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00034_1"><span><span class="NLM_string-ref">Todeschini, R., Consonni, V., Eds. <i>  Handbook of Molecular Descriptors</i>; Wiley-VCH:  Weinheim, Germany, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm020397cb00034/jm020397cb00034_1&amp;dbid=16&amp;doi=10.1021%2Fjm020397c&amp;key=10.1002%2F9783527613106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=R.+Todeschini&author=V.+Consonni&title=Handbook+of+Molecular+Descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00034_1&amp;dbid=16384&amp;doi=10.1002%2F9783527613106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527613106%26sid%3Dliteratum%253Aachs%26aulast%3DTodeschini%26aufirst%3DR.%26btitle%3DHandbook%2520of%2520Molecular%2520Descriptors%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00035"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00035_1"><span><span class="NLM_string-ref">Gao, H. Application of BCUT metrics and genetic algorithm in binary QSAR analysis. <i>  J. Chem. Inf. Comput. Sci. </i><b>2001</b><i>, 41</i>, 402−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=407&author=H+Gao&title=Application+of+BCUT+metrics+and+genetic+algorithm+in+binary+QSAR+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH%26atitle%3DApplication%2520of%2520BCUT%2520metrics%2520and%2520genetic%2520algorithm%2520in%2520binary%2520QSAR%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00036"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00036_1"><span><span class="NLM_string-ref">Vandrie, J. H.; Nugent, R. A. Addressing the Challenges Posed by Combinatorial Chemistry3d Databases, Pharmacophore Recognition and Beyond. <i>  SAR QSAR Environ. Res. </i><b>1998</b><i>, 9</i>, 1 ff.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&author=J.+H.+Vandrie&author=R.+A.+Nugent&title=Addressing+the+Challenges+Posed+by+Combinatorial+Chemistry3d+Databases%2C+Pharmacophore+Recognition+and+Beyond.+SAR+QSAR+Environ.+Res"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVandrie%26aufirst%3DJ.%2BH.%26btitle%3DAddressing%2520the%2520Challenges%2520Posed%2520by%2520Combinatorial%2520Chemistry3d%2520Databases%252C%2520Pharmacophore%2520Recognition%2520and%2520Beyond.%2520SAR%2520QSAR%2520Environ.%2520Res%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00037"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00037_1"><span><span class="NLM_string-ref">Sprague, P. W.; Hoffmann, R. CATALYST Pharmacophore Models and Their Utility as Queries for Searching 3D Databases. In <i>Computer-Assisted Lead </i><i>Finding and Optimization:  Current Tools for Medicinal Chemistry</i>; Wiley-VCH:  Weinheim, Germany, 1997; pp 223−240.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=240&author=P.+W.+Sprague&author=R.+Hoffmann&title=Computer-Assisted+Lead+Finding+and+Optimization%3A%E2%80%89+Current+Tools+for+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSprague%26aufirst%3DP.%2BW.%26atitle%3DCATALYST%2520Pharmacophore%2520Models%2520and%2520Their%2520Utility%2520as%2520Queries%2520for%2520Searching%25203D%2520Databases%26btitle%3DComputer-Assisted%2520Lead%2520Finding%2520and%2520Optimization%253A%25E2%2580%2589%2520Current%2520Tools%2520for%2520Medicinal%2520Chemistry%26pub%3DWiley-VCH%26date%3D1997%26spage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm020397cb00038"><div class="reference"><strong class="refLabel"><a href="#jm020397cb00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm020397cb00038_1"><span><span class="NLM_string-ref">Cramer, R. D., 3rd; Patterson, D. E.; Bunce, J. D. Comparative Molecular Field Analysis (CoMFA). 1. Effect of Shape on Binding of Steroids to Carrier Proteins. <i>  J. Am. Chem. Soc. </i><b>1988</b><i>, 110</i>, 5959−5967</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=5967&author=R.+D.+Cramer&author=D.+E.+Patterson&author=J.+D+Bunce&title=Comparative+Molecular+Field+Analysis+%28CoMFA%29.+1.+Effect+of+Shape+on+Binding+of+Steroids+to+Carrier+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm020397cb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26atitle%3DComparative%2520Molecular%2520Field%2520Analysis%2520%2528CoMFA%2529.%25201.%2520Effect%2520of%2520Shape%2520on%2520Binding%2520of%2520Steroids%2520to%2520Carrier%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D5967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm020397c&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2003.46.issue-9%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Fjm020397c" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm020397c" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                244KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67990910aa743c3a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
